
<html lang="en"     class="pb-page"  data-request-id="41fde3d9-bac9-467a-9afd-4901808c0dbe"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-19;article:article:10.1021/acs.jmedchem.0c00916;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Pyrazole-Based Lactate Dehydrogenase Inhibitors with Optimized Cell Activity and Pharmacokinetic Properties" /></meta><meta name="dc.Creator" content="Ganesha  Rai" /></meta><meta name="dc.Creator" content="Daniel J.  Urban" /></meta><meta name="dc.Creator" content="Bryan T.  Mott" /></meta><meta name="dc.Creator" content="Xin  Hu" /></meta><meta name="dc.Creator" content="Shyh-Ming  Yang" /></meta><meta name="dc.Creator" content="Gloria A.  Benavides" /></meta><meta name="dc.Creator" content="Michelle S.  Johnson" /></meta><meta name="dc.Creator" content="Giuseppe L.  Squadrito" /></meta><meta name="dc.Creator" content="Kyle R.  Brimacombe" /></meta><meta name="dc.Creator" content="Tobie D.  Lee" /></meta><meta name="dc.Creator" content="Dorian M.  Cheff" /></meta><meta name="dc.Creator" content="Hu  Zhu" /></meta><meta name="dc.Creator" content="Mark J.  Henderson" /></meta><meta name="dc.Creator" content="Katherine  Pohida" /></meta><meta name="dc.Creator" content="Gary A.  Sulikowski" /></meta><meta name="dc.Creator" content="David M.  Dranow" /></meta><meta name="dc.Creator" content="Md  Kabir" /></meta><meta name="dc.Creator" content="Pranav  Shah" /></meta><meta name="dc.Creator" content="Elias  Padilha" /></meta><meta name="dc.Creator" content="Dingyin  Tao" /></meta><meta name="dc.Creator" content="Yuhong  Fang" /></meta><meta name="dc.Creator" content="Plamen P.  Christov" /></meta><meta name="dc.Creator" content="Kwangho  Kim" /></meta><meta name="dc.Creator" content="Somnath  Jana" /></meta><meta name="dc.Creator" content="Pavan  Muttil" /></meta><meta name="dc.Creator" content="Tamara  Anderson" /></meta><meta name="dc.Creator" content="Nitesh K.  Kunda" /></meta><meta name="dc.Creator" content="Helen J.  Hathaway" /></meta><meta name="dc.Creator" content="Donna F.  Kusewitt" /></meta><meta name="dc.Creator" content="Nobu  Oshima" /></meta><meta name="dc.Creator" content="Murali  Cherukuri" /></meta><meta name="dc.Creator" content="Douglas R.  Davies" /></meta><meta name="dc.Creator" content="Jeffrey P.  Norenberg" /></meta><meta name="dc.Creator" content="Larry A.  Sklar" /></meta><meta name="dc.Creator" content="William J.  Moore" /></meta><meta name="dc.Creator" content="Chi V.  Dang" /></meta><meta name="dc.Creator" content="Gordon M.  Stott" /></meta><meta name="dc.Creator" content="Leonard  Neckers" /></meta><meta name="dc.Creator" content="Andrew J.  Flint" /></meta><meta name="dc.Creator" content="Victor M.  Darley-Usmar" /></meta><meta name="dc.Creator" content="Anton  Simeonov" /></meta><meta name="dc.Creator" content="Alex G.  Waterson" /></meta><meta name="dc.Creator" content="Ajit  Jadhav" /></meta><meta name="dc.Creator" content="Matthew D.  Hall" /></meta><meta name="dc.Creator" content="David J.  Maloney" /></meta><meta name="dc.Description" content="Lactate dehydrogenase (LDH) catalyzes the conversion of pyruvate to lactate, with concomitant oxidation of reduced nicotinamide adenine dinucleotide as the final step in the glycolytic pathway. Gly..." /></meta><meta name="Description" content="Lactate dehydrogenase (LDH) catalyzes the conversion of pyruvate to lactate, with concomitant oxidation of reduced nicotinamide adenine dinucleotide as the final step in the glycolytic pathway. Gly..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 9, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00916" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00916" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00916" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00916" /></link>
        
    
    

<title>Pyrazole-Based Lactate Dehydrogenase Inhibitors with Optimized Cell Activity and Pharmacokinetic Properties | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00916" /></meta><meta property="og:title" content="Pyrazole-Based Lactate Dehydrogenase Inhibitors with Optimized Cell Activity and Pharmacokinetic Properties" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0020.jpeg" /></meta><meta property="og:description" content="Lactate dehydrogenase (LDH) catalyzes the conversion of pyruvate to lactate, with concomitant oxidation of reduced nicotinamide adenine dinucleotide as the final step in the glycolytic pathway. Glycolysis plays an important role in the metabolic plasticity of cancer cells and has long been recognized as a potential therapeutic target. Thus, potent, selective inhibitors of LDH represent an attractive therapeutic approach. However, to date, pharmacological agents have failed to achieve significant target engagement in vivo, possibly because the protein is present in cells at very high concentrations. We report herein a lead optimization campaign focused on a pyrazole-based series of compounds, using structure-based design concepts, coupled with optimization of cellular potency, in vitro drug–target residence times, and in vivo PK properties, to identify first-in-class inhibitors that demonstrate LDH inhibition in vivo. The lead compounds, named NCATS-SM1440 (43) and NCATS-SM1441 (52), possess desirable attributes for further studying the effect of in vivo LDH inhibition." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00916"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00916">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00916&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00916&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00916&amp;href=/doi/10.1021/acs.jmedchem.0c00916" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 19</span><span class="cit-fg-pageRange">, 10984-11011</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/19" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00842" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00821" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Pyrazole-Based Lactate Dehydrogenase Inhibitors with Optimized Cell Activity and Pharmacokinetic Properties</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ganesha Rai</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ganesha Rai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#0a686b647e7f616b66667f6d4a676b636624646362246d657c"><span class="__cf_email__" data-cfemail="4123202f35342a202d2d3426012c20282d6f2f28296f262e37">[email protected]</span></a>. Phone: 301-827-1756. Fax: 301-217-5736.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ganesha++Rai">Ganesha Rai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9763-9641" title="Orcid link">http://orcid.org/0000-0001-9763-9641</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel J. Urban</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel J. Urban</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+J.++Urban">Daniel J. Urban</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bryan T. Mott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bryan T. Mott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bryan+T.++Mott">Bryan T. Mott</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Hu">Xin Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shyh-Ming Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shyh-Ming Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shyh-Ming++Yang">Shyh-Ming Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1928-136X" title="Orcid link">http://orcid.org/0000-0003-1928-136X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gloria A. Benavides</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gloria A. Benavides</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mitochondrial Medicine Laboratory, Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35294, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gloria+A.++Benavides">Gloria A. Benavides</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michelle S. Johnson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michelle S. Johnson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mitochondrial Medicine Laboratory, Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35294, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michelle+S.++Johnson">Michelle S. Johnson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Giuseppe L. Squadrito</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Giuseppe L. Squadrito</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mitochondrial Medicine Laboratory, Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35294, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe+L.++Squadrito">Giuseppe L. Squadrito</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kyle R. Brimacombe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kyle R. Brimacombe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kyle+R.++Brimacombe">Kyle R. Brimacombe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tobie D. Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tobie D. Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tobie+D.++Lee">Tobie D. Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dorian M. Cheff</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dorian M. Cheff</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dorian+M.++Cheff">Dorian M. Cheff</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hu Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hu Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hu++Zhu">Hu Zhu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark J. Henderson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark J. Henderson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark+J.++Henderson">Mark J. Henderson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8590-7338" title="Orcid link">http://orcid.org/0000-0001-8590-7338</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katherine Pohida</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katherine Pohida</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katherine++Pohida">Katherine Pohida</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gary A. Sulikowski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gary A. Sulikowski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vanderbilt Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gary+A.++Sulikowski">Gary A. Sulikowski</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1067-0767" title="Orcid link">http://orcid.org/0000-0002-1067-0767</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David M. Dranow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David M. Dranow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Beryllium Discovery Corp., 7869 Day Road West, Bainbridge Island, Washington 98110, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+M.++Dranow">David M. Dranow</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Md Kabir</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Md Kabir</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Md++Kabir">Md Kabir</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pranav Shah</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pranav Shah</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pranav++Shah">Pranav Shah</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elias Padilha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elias Padilha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elias++Padilha">Elias Padilha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dingyin Tao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dingyin Tao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dingyin++Tao">Dingyin Tao</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6531-2740" title="Orcid link">http://orcid.org/0000-0001-6531-2740</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuhong Fang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuhong Fang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuhong++Fang">Yuhong Fang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Plamen P. Christov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Plamen P. Christov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vanderbilt Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Plamen+P.++Christov">Plamen P. Christov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kwangho Kim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kwangho Kim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vanderbilt Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kwangho++Kim">Kwangho Kim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Somnath Jana</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Somnath Jana</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vanderbilt Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Somnath++Jana">Somnath Jana</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pavan Muttil</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pavan Muttil</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, University of New Mexico Health Sciences Center, Albuquerque, New Mexico 87131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pavan++Muttil">Pavan Muttil</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7636-8141" title="Orcid link">http://orcid.org/0000-0001-7636-8141</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tamara Anderson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tamara Anderson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, University of New Mexico Health Sciences Center, Albuquerque, New Mexico 87131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tamara++Anderson">Tamara Anderson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nitesh K. Kunda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nitesh K. Kunda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, University of New Mexico Health Sciences Center, Albuquerque, New Mexico 87131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nitesh+K.++Kunda">Nitesh K. Kunda</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3063-0375" title="Orcid link">http://orcid.org/0000-0002-3063-0375</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Helen J. Hathaway</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helen J. Hathaway</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, University of New Mexico Health Sciences Center, Albuquerque, New Mexico 87131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helen+J.++Hathaway">Helen J. Hathaway</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Donna F. Kusewitt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Donna F. Kusewitt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept of Pathology, University of New Mexico Cancer Center, Albuquerque, New Mexico 87131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Donna+F.++Kusewitt">Donna F. Kusewitt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nobu Oshima</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nobu Oshima</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, 9000 Rockville Pike, Bethesda, Maryland 20892, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nobu++Oshima">Nobu Oshima</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Murali Cherukuri</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Murali Cherukuri</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, 9000 Rockville Pike, Bethesda, Maryland 20892, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Murali++Cherukuri">Murali Cherukuri</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Douglas R. Davies</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Douglas R. Davies</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Beryllium Discovery Corp., 7869 Day Road West, Bainbridge Island, Washington 98110, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Douglas+R.++Davies">Douglas R. Davies</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeffrey P. Norenberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeffrey P. Norenberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, University of New Mexico Health Sciences Center, Albuquerque, New Mexico 87131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey+P.++Norenberg">Jeffrey P. Norenberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Larry A. Sklar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Larry A. Sklar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept of Pathology, University of New Mexico Cancer Center, Albuquerque, New Mexico 87131, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Larry+A.++Sklar">Larry A. Sklar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William J. Moore</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William J. Moore</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">NExT Program Support, Applied/Developmental Research Directorate, Leidos Biomedical Research, Inc., Frederick National Laboratory for Cancer Research, Frederick, Maryland 21702, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+J.++Moore">William J. Moore</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chi V. Dang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chi V. Dang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Abramson Cancer Center, Abramson Family Cancer Research Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</div><div class="loa-info-affiliations-info">Ludwig Institute for Cancer Research, New York, New York 10017, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chi+V.++Dang">Chi V. Dang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gordon M. Stott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gordon M. Stott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">NExT Program Support, Applied/Developmental Research Directorate, Leidos Biomedical Research, Inc., Frederick National Laboratory for Cancer Research, Frederick, Maryland 21702, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gordon+M.++Stott">Gordon M. Stott</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Leonard Neckers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Leonard Neckers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, 9000 Rockville Pike, Bethesda, Maryland 20892, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Leonard++Neckers">Leonard Neckers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew J. Flint</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew J. Flint</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">NExT Program Support, Applied/Developmental Research Directorate, Leidos Biomedical Research, Inc., Frederick National Laboratory for Cancer Research, Frederick, Maryland 21702, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+J.++Flint">Andrew J. Flint</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Victor M. Darley-Usmar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Victor M. Darley-Usmar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mitochondrial Medicine Laboratory, Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35294, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Victor+M.++Darley-Usmar">Victor M. Darley-Usmar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anton Simeonov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anton Simeonov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anton++Simeonov">Anton Simeonov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alex G. Waterson</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex G. Waterson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Vanderbilt Institute of Chemical Biology, Vanderbilt University, Nashville, Tennessee 37232, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a6c788d1c7d2c3d4d5c9c8e6f0c7c8c2c3d4c4cfcad288e3c2d3"><span class="__cf_email__" data-cfemail="f293dc8593869780819d9cb2a4939c969780909b9e86dcb79687">[email protected]</span></a>. Phone: 615-322-9971.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex+G.++Waterson">Alex G. Waterson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ajit Jadhav</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ajit Jadhav</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ajit++Jadhav">Ajit Jadhav</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew D. Hall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew D. Hall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Hall">Matthew D. Hall</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5073-442X" title="Orcid link">http://orcid.org/0000-0002-5073-442X</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">David J. Maloney</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David J. Maloney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#64200512012432011605080b1c4a070b09"><span class="__cf_email__" data-cfemail="622603140722340710030e0d1a4c010d0f">[email protected]</span></a>. Phone: 301-360-3502 ext. 3.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Maloney">David J. Maloney</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5607-4178" title="Orcid link">http://orcid.org/0000-0002-5607-4178</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00916&amp;href=/doi/10.1021%2Facs.jmedchem.0c00916" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 19</span><span class="cit-pageRange">, 10984–11011</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 9, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 May 2020</li><li><span class="item_label"><b>Published</b> online</span>9 September 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 October 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00916" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00916</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10984%26pageCount%3D28%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGanesha%2BRai%252C%2BDaniel%2BJ.%2BUrban%252C%2BBryan%2BT.%2BMott%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D19%26contentID%3Dacs.jmedchem.0c00916%26title%3DPyrazole-Based%2BLactate%2BDehydrogenase%2BInhibitors%2Bwith%2BOptimized%2BCell%2BActivity%2Band%2BPharmacokinetic%2BProperties%26numPages%3D28%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11011%26publicationDate%3DOctober%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00916"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1760</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00916" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Pyrazole-Based Lactate Dehydrogenase Inhibitors with Optimized Cell Activity and Pharmacokinetic Properties&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ganesha&quot;,&quot;last_name&quot;:&quot;Rai&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;J. Urban&quot;},{&quot;first_name&quot;:&quot;Bryan&quot;,&quot;last_name&quot;:&quot;T. Mott&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Shyh-Ming&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Gloria&quot;,&quot;last_name&quot;:&quot;A. Benavides&quot;},{&quot;first_name&quot;:&quot;Michelle&quot;,&quot;last_name&quot;:&quot;S. Johnson&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;L. Squadrito&quot;},{&quot;first_name&quot;:&quot;Kyle&quot;,&quot;last_name&quot;:&quot;R. Brimacombe&quot;},{&quot;first_name&quot;:&quot;Tobie&quot;,&quot;last_name&quot;:&quot;D. Lee&quot;},{&quot;first_name&quot;:&quot;Dorian&quot;,&quot;last_name&quot;:&quot;M. Cheff&quot;},{&quot;first_name&quot;:&quot;Hu&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;J. Henderson&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;Pohida&quot;},{&quot;first_name&quot;:&quot;Gary&quot;,&quot;last_name&quot;:&quot;A. Sulikowski&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;M. Dranow&quot;},{&quot;first_name&quot;:&quot;Md&quot;,&quot;last_name&quot;:&quot;Kabir&quot;},{&quot;first_name&quot;:&quot;Pranav&quot;,&quot;last_name&quot;:&quot;Shah&quot;},{&quot;first_name&quot;:&quot;Elias&quot;,&quot;last_name&quot;:&quot;Padilha&quot;},{&quot;first_name&quot;:&quot;Dingyin&quot;,&quot;last_name&quot;:&quot;Tao&quot;},{&quot;first_name&quot;:&quot;Yuhong&quot;,&quot;last_name&quot;:&quot;Fang&quot;},{&quot;first_name&quot;:&quot;Plamen&quot;,&quot;last_name&quot;:&quot;P. Christov&quot;},{&quot;first_name&quot;:&quot;Kwangho&quot;,&quot;last_name&quot;:&quot;Kim&quot;},{&quot;first_name&quot;:&quot;Somnath&quot;,&quot;last_name&quot;:&quot;Jana&quot;},{&quot;first_name&quot;:&quot;Pavan&quot;,&quot;last_name&quot;:&quot;Muttil&quot;},{&quot;first_name&quot;:&quot;Tamara&quot;,&quot;last_name&quot;:&quot;Anderson&quot;},{&quot;first_name&quot;:&quot;Nitesh&quot;,&quot;last_name&quot;:&quot;K. Kunda&quot;},{&quot;first_name&quot;:&quot;Helen&quot;,&quot;last_name&quot;:&quot;J. Hathaway&quot;},{&quot;first_name&quot;:&quot;Donna&quot;,&quot;last_name&quot;:&quot;F. Kusewitt&quot;},{&quot;first_name&quot;:&quot;Nobu&quot;,&quot;last_name&quot;:&quot;Oshima&quot;},{&quot;first_name&quot;:&quot;Murali&quot;,&quot;last_name&quot;:&quot;Cherukuri&quot;},{&quot;first_name&quot;:&quot;Douglas&quot;,&quot;last_name&quot;:&quot;R. Davies&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;P. Norenberg&quot;},{&quot;first_name&quot;:&quot;Larry&quot;,&quot;last_name&quot;:&quot;A. Sklar&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;J. Moore&quot;},{&quot;first_name&quot;:&quot;Chi&quot;,&quot;last_name&quot;:&quot;V. Dang&quot;},{&quot;first_name&quot;:&quot;Gordon&quot;,&quot;last_name&quot;:&quot;M. Stott&quot;},{&quot;first_name&quot;:&quot;Leonard&quot;,&quot;last_name&quot;:&quot;Neckers&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;J. Flint&quot;},{&quot;first_name&quot;:&quot;Victor&quot;,&quot;last_name&quot;:&quot;M. Darley-Usmar&quot;},{&quot;first_name&quot;:&quot;Anton&quot;,&quot;last_name&quot;:&quot;Simeonov&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;G. Waterson&quot;},{&quot;first_name&quot;:&quot;Ajit&quot;,&quot;last_name&quot;:&quot;Jadhav&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Hall&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Maloney&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;10984-11011&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00916&quot;},&quot;abstract&quot;:&quot;Lactate dehydrogenase (LDH) catalyzes the conversion of pyruvate to lactate, with concomitant oxidation of reduced nicotinamide adenine dinucleotide as the final step in the glycolytic pathway. Glycolysis plays an important role in the metabolic plasticity of cancer cells and has long been recognized as a potential therapeutic target. Thus, potent, selective inhibitors of LDH represent an attractive therapeutic approach. However, to date, pharmacological agents have failed to achieve significant target engagement in vivo, possibly because the protein is present in cells at very high concentrations. We report herein a lead optimization campaign focused on a pyrazole-based series of compounds, using structure-based design concepts, coupled with optimization of cellular potency, in vitro drug–target residence times, and in vivo PK properties, to identify first-in-class inhibitors that demonstrate LDH inhibition in vivo. The lead compounds, named NCATS-SM1440 (43) and NCATS-SM1441 (52), possess desirable attribut&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00916&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00916" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00916&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00916" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00916&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00916" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00916&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00916&amp;href=/doi/10.1021/acs.jmedchem.0c00916" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00916" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00916" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00916%26sid%3Dliteratum%253Aachs%26pmid%3D32902275%26genre%3Darticle%26aulast%3DRai%26date%3D2020%26atitle%3DPyrazole-Based%2BLactate%2BDehydrogenase%2BInhibitors%2Bwith%2BOptimized%2BCell%2BActivity%2Band%2BPharmacokinetic%2BProperties%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D19%26spage%3D10984%26epage%3D11011%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291456" title="Hydrocarbons">Hydrocarbons</a>,</li><li><a href="/action/doSearch?ConceptID=292078" title="Inorganic compounds">Inorganic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/19" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/jmcmar.2020.63.issue-19/20201008/jmcmar.2020.63.issue-19.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Lactate dehydrogenase (LDH) catalyzes the conversion of pyruvate to lactate, with concomitant oxidation of reduced nicotinamide adenine dinucleotide as the final step in the glycolytic pathway. Glycolysis plays an important role in the metabolic plasticity of cancer cells and has long been recognized as a potential therapeutic target. Thus, potent, selective inhibitors of LDH represent an attractive therapeutic approach. However, to date, pharmacological agents have failed to achieve significant target engagement <i>in vivo</i>, possibly because the protein is present in cells at very high concentrations. We report herein a lead optimization campaign focused on a pyrazole-based series of compounds, using structure-based design concepts, coupled with optimization of cellular potency, <i>in vitro</i> drug–target residence times, and <i>in vivo</i> PK properties, to identify first-in-class inhibitors that demonstrate LDH inhibition <i>in vivo</i>. The lead compounds, named <b>NCATS-SM1440</b> (<b>43</b>) and <b>NCATS-SM1441</b> (<b>52</b>), possess desirable attributes for further studying the effect of <i>in vivo</i> LDH inhibition.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36044" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36044" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cancer cells exhibit deregulated metabolic characteristics that are remarkably different from normal cells, demonstrating avidity for glucose uptake and catabolism, ultimately producing lactate and generating adenosine 5′-triphosphate (ATP) through the cytosolic, nonoxidative glycolytic pathway. Glycolysis can be the preferred nicotinamide adenine dinucleotide (NAD<sup>+</sup>) regeneration and energy production process for cancer cells, even in the presence of oxygen, rather than the more energy-efficient mitochondrial oxidative phosphorylation. This is referred to as aerobic glycolysis, and the preference for aerobic glycolysis is also known as the Warburg effect.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Although aerobic glycolysis is an inefficient way to generate ATP (compared with oxidative phosphorylation), the rate of ATP generation is rapid. In addition, it is hypothesized that rapidly proliferating cancer cells have adapted this approach to regenerate NAD<sup>+</sup> and to support the production of essential cellular building blocks such as amino acids, lipids, and nucleotides needed to support rapid cell growth.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Indeed, many noncancer cells use a combination of oxidative phosphorylation and glycolysis to achieve the needed metabolic plasticity to serve their biological functions. Although the molecular basis of aberrant cancer metabolism and its role in cancer development and progression have yet to be fully elucidated,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> the Warburg effect and the enzymes in glycolysis have long been recognized as potential targets for the selective killing of cancer cells.</div><div class="NLM_p">Many glycolytic enzymes are overexpressed in cancer cells, including the lactate dehydrogenase (LDH) enzymes A (LDHA) and B (LDHB).<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> LDH is a tetrameric protein composed of the products of <i>LDHA</i> (subunit M) and <i>LDHB</i> (subunit H) genes. The tetrameric combination of these gene products generates five LDH isoforms with different combinations of subunits depending on the cell type. The LDH5 (4 M) isoform is the primary form expressed in cancer cells,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> although other isoforms have also been reported. All LDH isoforms catalyze the last step in the glycolytic pathway converting pyruvate to lactate while regenerating NAD<sup>+</sup> from reduced nicotinamide adenine dinucleotide (NADH). The lactate produced is then secreted from cells <i>via</i> the monocarboxylate transporter proteins.</div><div class="NLM_p">Significant evidence exists to support the development of LDH inhibitors as a therapeutic option for cancer treatment. Genetic knockdown of <i>LDHA</i> has been shown to elicit cell death or delayed cell growth in cell lines from colorectal carcinoma (siRNA),<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Burkitt lymphoma (siRNA),<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> hepatocellular carcinoma (siRNA),<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> pancreatic cancer (shRNA),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and mouse mammary tumor cells (shRNA).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> For example, mouse mammary tumor cells lacking LDHA implanted as xenografts demonstrated dramatically reduced tumor growth.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In addition, in a genetically engineered mouse model of non-small-cell lung cancer, induced knockout of mouse LDHA led to the regression of established tumors without serious systemic toxicity.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> In contrast, LDHB knockdown has been reported not to significantly impact tumor cell survival.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">Although the therapeutic potential of LDHA inhibition appears to be substantial, the discovery and development of LDH inhibitors have proven to be challenging. For example, because of the micromolar concentrations of the LDH enzyme in cancer cells, an effective inhibitor will likely need to bind with exceptionally high affinity and also achieve high intracellular concentrations to enable a therapeutic level of target engagement. To date, no inhibitors of LDH which meet these criteria have been reported. The first reported LDH inhibitors came from academic groups (<i>e.g.</i>, FX-11<a onclick="showRef(event, 'cit13a'); return false;" href="javascript:void(0);" class="ref cit13a">(13a)</a> and NHI-2<a onclick="showRef(event, 'cit13b'); return false;" href="javascript:void(0);" class="ref cit13b">(13b)</a>) with efforts from biotech<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and pharmaceutical companies<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> being disclosed more recently. Molecules from GlaxoSmithKline (GSK)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and Genentech (GNE-140)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> have demonstrated <i>in vitro</i> inhibition of cellular lactate production and cytotoxicity in cancer cells, but the relatively poor pharmacokinetics of these compounds limited their usefulness for testing of the therapeutic hypothesis <i>in vivo</i>. More recently, LDH inhibitors (<i>e.g.</i>, “compound <b>24c</b>” in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) that show modest inhibition of MiaPaCa-2 xenograft growth have been described, but target engagement <i>in vivo</i> was not demonstrated.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative previously described LDH inhibitors and comparison with new leads <b>43 = NCATS-SM1440</b> and <b>52 = NCATS-SM1441</b>. <sup>a</sup>Named as NCI-006 in refs <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>. <sup>b</sup>Named as NCI-737 in ref <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We recently reported the identification of a pyrazole-based hit from quantitative high-throughput screening (qHTS) and used structure-based design to develop nanomolar inhibitors of LDH enzyme activity, exemplified by <b>1</b>.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Compound <b>1</b> inhibited LDH in highly glycolytic MiaPaCa-2 (human pancreatic cancer) and A673 (human Ewing’s sarcoma) cell lines, demonstrating sub-micromolar suppression of cellular lactate output and inhibition of cell growth. Although <b>1</b> showed favorable <i>in vitro</i> ADME properties (<i>e.g.</i>, microsomal stability and solubility), its PK profile was not suitable for <i>in vivo</i> use. Herein, we report the optimization of the pyrazole-based chemotype using binding constants (particularly off-rates or drug–target residence time) and cell-based activity to guide improvements in potency while also optimizing the pharmacokinetic properties. These efforts ultimately resulted in lead compounds <b>43</b> (<b>NCATS-SM1440</b>) and <b>52</b> (<b>NCATS-SM1441</b>) that have enabled studies of the effects of pharmacological LDH inhibition in animal models of Ewing’s sarcoma<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and human pancreatic cancers<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> as well as interrogation of a synergistic role in T cell-based immunotherapy.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51451" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51451" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A robust linear synthetic strategy was used for the optimization of the cyclopropylmethyl and sulfonamide regions (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>). The commercially available carboxylic acids <b>a–l</b> were reacted with 1,2,3-benzotriazole in the presence of thionyl chloride to afford acyl benzotriazole intermediates <b>Ia–l</b>. The reaction of these intermediates (<b>Ia–l</b>) with substituted acetophenones <b>IIa–d</b>, mediated by magnesium bromide ethyl etherate in the presence of Hunig’s base, afforded 1,3-diketone derivatives <b>IIIa–p</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Table S3</a>) in good yields. Subsequently, cesium carbonate mediated alkylation of 1,3-diketone intermediates <b>IIIa–p</b> with 4-(bromomethyl)benzenesulfonamide derivatives <b>IVa–c</b> in dimethyl sulfoxide (DMSO) at room temperature generated advanced intermediates <b>Va–r</b>, as described in our previous paper.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Initial attempts with tosic acid catalyzed cyclocondensation of 1,3-diketone derivatives <b>IIIa–p</b> with ethyl 2-hydrazinylthiazole-4-carboxylate hydrobromide generated a mixture of both pyrazole regioisomers. We previously reported that the 5-aryl regioisomer is inactive in LDHA assays, so formation of this regioisomer is undesired. The product ratio of the desired isomer was poor (≤10%), particularly when R<sub>1</sub> is an electron-donating alkyl group such as cyclopropylmethyl or cyclopropylethyl group which was found to be optimal for activity (<i>e.g.</i>, <b>Va–f</b>, <b>Vh–j</b>, and <b>Vn–r</b>). Therefore, it was neither efficient nor practical to use this cyclization protocol to synthesize the quantities of the advanced bromide-containing intermediates <b>VIIa–f</b>, <b>VIIh–j</b>, and <b>VIIn–r</b> needed for late-stage optimization efforts.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Intermediates <b>VIa–r</b> and Analogues <b>1–9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SOCl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 4 h, and 91–100%; (b) MgBr<sub>2</sub>·OEt<sub>2</sub>, iPr<sub>2</sub>NEt, CH<sub>2</sub>Cl<sub>2</sub>, 12 h, and 60–69%; (c) Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 1 h, and 55–83%; (d) (i) pyrrolidine (0.5 equiv), TsOH (0.5 equiv), EtOH, reflux, and 1–2 h and (ii) ethyl 2-hydrazinylthiazole-4-carboxylate·HBr and reflux overnight; and (e) LiOH, THF/MeOH/H<sub>2</sub>O, and 1 h.</p></p></figure><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Analogues <b>10–32</b> and <b>36–89</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) [P(<i>t</i>-Bu)<sub>3</sub>]Pd(crotyl)Cl, DABCO, dioxane, RT, and 1–12 h; (b) [DTBNpP]Pd(crotyl)Cl, DABCO, dioxane, 60 °C, and 12 h; (c) SPhosPd(crotyl)Cl, K<sub>3</sub>PO<sub>4</sub>, dioxane/H<sub>2</sub>O, 100 °C, and 0.5 h; (d) LiOH, THF/MeOH/H<sub>2</sub>O, and 1 h; and (e) CsF, THF–EtOH, RT, and 2 h.</p></p></figure><div class="NLM_p">The ratio of the two regioisomers produced in the reaction depends on the reactivity differences dictated by the electronic environment around the two keto groups. To improve the regioisomeric ratio, we envisioned utilizing these reactivity differences by blocking the undesired reactivity of the 1-keto group <i>via</i> initial formation of an enamine intermediate prior to the addition of the ethyl 2-hydrazinylthiazole-4-carboxylate. Extensive optimization identified improved conditions, in which heating 1,3-diketones <b>Va–r</b> in ethanol in the presence of 0.5 equiv of both pyrrolidine and tosic acid for 1–2 h at reflux, then adding ethyl 2-hydrazinylthiazole-4-carboxylate, and stirring at reflux overnight allowed us to obtain a ∼50:50 mixture of the two regioisomers. The desired isomer was isolated as the second peak using reversed-phase (C18) chromatography, as this regioisomer was slightly less polar [as judged by liquid chromatography mass spectrometry (LC–MS) analysis and according to our previous reports.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Further reaction optimization proved elusive, and the investigation of alternative routes to obtain exclusively the desired isomer were met with limited success. Thus, for all subsequent scale-up and analogue syntheses, we used the above procedure. Compound <b>1</b> and analogues <b>2–9</b> were obtained by hydrolysis of the intermediates <b>VIIa</b>, <b>VIIg–m</b>, and <b>VIIb–d</b>, respectively, using lithium hydroxide in ethanol and subsequent high-performance liquid chromatography (HPLC) purification.</div><div class="NLM_p">The synthesis of analogues <b>11–89</b> listed in <named-content content-type="anchor" rid="tbl1" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>–<named-content content-type="anchor" rid="tbl3" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a> was achieved utilizing either Sonogashira or Suzuki coupling of the advanced 3-bromo aryl intermediates <b>VIIe–f</b>, <b>VIIi–j</b>, and <b>VIIn–r</b>, followed by hydrolysis with LiOH (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). For the Sonogashira coupling, we developed a robust room-temperature, copper-free protocol catalyzed by commercially available monoligated precatalysts, [P(<i>t</i>-Bu)<sub>3</sub>] Pd(crotyl)Cl or [DTBNpP] Pd(crotyl)Cl, in the presence of 1,4-diazabicyclo[2.2.2]octane (DABCO) in dioxane.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> This method allowed us to quickly generate an alkyne library and explore the structure–activity relationship (SAR) around the biphenyl region with access to more complex structures. Moreover, the protocol requires only a minimal work-up that could further facilitate rapid analogue generation during our library synthesis and is amenable to providing large quantities of key intermediates (<i>e.g.</i>, <b>VIIIa–d</b>) to support advanced studies on top compounds. Using this method, analogues <b>14–16</b>, <b>20–32</b>, <b>36–43</b>, <b>48–49</b>, <b>51–52</b>, <b>74–75</b>, and <b>84–89</b> were synthesized directly from intermediates <b>VIIe–f</b>, <b>VIIi–j</b>, and <b>VIIn–r</b> with the corresponding commercially available terminal alkynes <b>IX</b> and the [P(<i>t</i>-Bu)<sub>3</sub>] Pd(crotyl)Cl catalyst in dioxane. The syntheses of analogues <b>33–35</b> are summarized in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. To rapidly explore the SAR with aryl or heteroaryl R substitutions, a slightly modified three-step protocol was used for analogues <b>44–47</b>, <b>53–67</b>, and <b>76–83</b>. The synthesis began with [P(<i>t</i>-Bu)<sub>3</sub>]Pd(crotyl)Cl-catalyzed Sonogashira coupling of intermediates <b>VIIe–f</b> and <b>VIIi–j</b> with trimethylsilylacetylene, followed by deprotection of the Trimethylsilyl (TMS) group with CsF to afford intermediates <b>VIIIa–d</b>. Further Sonogashira coupling of these alkyne intermediates with the corresponding aryl or heteroaryl bromides <b>X</b> under the same conditions, followed by hydrolysis with LiOH, furnished analogues <b>44–45</b>, <b>53–61</b>, <b>66–67</b>, and <b>76–79</b>. Analogue <b>11</b> (R = H) was obtained upon hydrolysis of intermediate <b>VIIIb</b> using LiOH in a methanol–water mixture. However, the use of the [P(<i>t</i>-Bu)<sub>3</sub>]Pd(crotyl)Cl catalyst under similar conditions failed to afford the isolable product in the case of more electron-deficient heteroaryl bromides, such as 1,3-thiazolyl or 1,3-oxazolyl bromides. Fortunately, switching to a more active catalyst, [DTBNpP]Pd(crotyl)Cl, with a slightly elevated reaction temperature (60 °C in dioxane) facilitated efficient product formation. Thus, the Sonogashira reaction of intermediates <b>VIIIa–d</b> with the corresponding 1,3-thiazolyl or 1,3-oxazolyl bromides under these conditions in the presence of DABCO, followed by hydrolysis, furnished analogues <b>46–47</b>, <b>62–65</b>, and <b>80–82</b>. Because of the gaseous nature and commercial unavailability of alkyne precursors, we used SPhos precatalyst-catalyzed Suzuki coupling of the corresponding alkynyl trifluoroboronates or pinacol esters, followed by hydrolysis protocol for alkyne analogues <b>12–13</b>, <b>17–19</b>, and <b>50</b>. Finally, Suzuki coupling of the corresponding alkenyl boronates or boronic acids with <b>VIIe–f</b>, using the SPhos Pd(crotyl)Cl precatalyst in the presence of potassium phosphate base at 100 °C and subsequent hydrolysis of the ester group, provided analogues <b>68–73</b>.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of Analogues <b>33–35</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Dess–Martin periodinane, DCM, RT, and 2 h; (b) NaN<sub>3</sub>, CF<sub>3</sub>SO<sub>3</sub>H, ACN, RT, and 12 h; (c) (CH<sub>3</sub>)<sub>3</sub>Sn–OH, dichloroethane, MW, 110 °C, and 1 h; (d) Deoxo-Fluor, DCM, RT, and 12 h; (e) P(<i>t</i>-Bu)<sub>3.</sub>HBF<sub>4</sub>, ([PdCl(allyl)]<sub>2</sub>, dioxane, 80 °C, and 4 h; and (f) LiOH, THF/MeOH/H<sub>2</sub>O, and 1 h.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical and Cellular LDH Inhibition of <b>2–10</b> with Comparator <b>1</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0010.gif" alt="" id="GRAPHIC-d7e1369-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0011.gif" alt="" id="gr9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values represent the half maximal (50%) inhibitory concentration as determined in the HTS assay using a dose response in the 1536-well format. (<i>n</i> = 2 for lactate and <i>n</i> = 3 for cytotoxicity, CETSA, and biochemical LDHA.)</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Efficacy < 20% with curve class 4; therefore, it should be considered as inactive.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Biochemical and Cellular LDH Inhibition of Analogues <b>11–49</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0012.gif" alt="" id="GRAPHIC-d7e1399-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0013.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values represent the half maximal (50%) inhibitory concentration as determined in the HTS assay using a dose response in the 1536-well format. (<i>n</i> = 2 for lactate and <i>n</i> = 3 for cytotoxicity, CETSA, and biochemical LDHA.)</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Biochemical and Cellular LDH Inhibition of Analogues <b>50–67</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0014.gif" alt="" id="GRAPHIC-d7e1424-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0015.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values represent the half maximal (50%) inhibitory concentration as determined in the HTS assay using a dose response in the 1536-well format. (<i>n</i> = 2 for lactate and <i>n</i> = 3 for cytotoxicity, CETSA, and biochemical LDHA.)</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Biochemical and Cellular LDH Inhibition of Compounds <b>68–73</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0016.gif" alt="" id="GRAPHIC-d7e1449-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0017.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values represent the half maximal (50%) inhibitory concentration as determined in the HTS assay using a dose response in the 1536-well format. (<i>n</i> = 2 for lactate and <i>n</i> = 3 for cytotoxicity, CETSA, and biochemical LDHA.)</p></div></div><div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43811" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43811" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In our previous work, we described structure-guided optimization of a weakly active qHTS hit into <b>1</b>, a potent, selective LDH inhibitor suitable for probing LDH function in cells.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Although <b>1</b> showed nanomolar inhibition of LDHA in the biochemical activity assay and was able to achieve reasonable cellular potency, additional optimization was required to identify highly potent molecules with PK properties suitable for <i>in vivo</i> studies. Here, we report a focused lead optimization effort that has identified compounds with good activity in highly glycolytic MiaPaCa-2 and A673 cell lines and with improved pharmacokinetic properties.</div><div class="NLM_p">We previously showed that the thiazole carboxylic acid moiety forms a critical interaction with the active site R168 and could not be replaced by other groups. The 4-benzyl sulfonamide group likewise forms critical hydrogen bond interactions with Asp140, Glu191, and Ile141 that contribute significantly to the potency. Further, the cyclopropylmethyl group forms an important π stacking interaction with Tyr238 and was found to be a key substituent for conferring cellular potency. However, previous modifications of the benzyl sulfonamide or replacements for the cyclopropylmethyl group had been limited, and thus, the potential to improve potency and residence time had not been fully explored. Additionally, in our previous cocrystal structure,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> the binding pose shows that the biphenyl does not fully occupy the hydrophobic pocket in this region, suggesting an opportunity to introduce other lipophilic groups in a region that appeared to be the most accommodating for further SAR optimization. Therefore, the lead optimization efforts described here have focused on these three regions of the compounds in this series.</div><div class="NLM_p">Based on our published crystal structures of compounds in this series bound to LDHA, the SAR campaign began with more extensive investigations of the cyclopropylmethyl group. Initially, a difluoro group was introduced into the cyclopropyl ring (<b>2</b>), anticipating an additional hydrogen bonding interaction with the phenol of residue Tyr238 (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Although analogue <b>2</b> maintained the biochemical potency (24 <i>vs</i> 25 nM), it showed reduced cellular potency and cytotoxicity compared to <b>1</b> (A673 lactate = 1423 <i>vs</i> 450 nM, A673 cytotox = 4642 <i>vs</i> 2450 nM; MiaPaCa-2 lactate = 1721 <i>vs</i> 636 nM, cytotox = 10,660 <i>vs</i> 8448 nM). To extend deeper into this binding site, the cyclopropylethyl (<b>3</b>) analogue was made. This group resulted in reduced biochemical LDHA activity and offered only marginal improvement in cellular potency. This trend appeared to be general, as it replicated across numerous matched pairs of analogues (see <b>43</b><i>vs</i><b>74</b>, <b>52</b><i>vs</i><b>75</b>, <b>54</b><i>vs</i><b>77</b>, <b>56</b><i>vs</i><b>78</b>, <b>57</b><i>vs</i><b>79</b>, <b>63</b><i>vs</i><b>80</b>, <b>64</b><i>vs</i><b>81</b>, <b>65</b><i>vs</i><b>82</b>, and <b>67</b><i>vs</i><b>83</b> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Replacing the cyclopropylmethyl group with other fluorinated alkyl groups (<b>4–6</b>) diminished potency in the biochemical assay and drastically reduced the cellular activity compared to <b>1</b>. Adding a branched substitution to the methylene group (<b>84</b>, <b>85</b>, and <b>89</b>) or adding substitution on the cyclopropyl ring (<b>86–88</b>) further reduced both biochemical and cellular potencies, relative to <b>75</b>. The primary focus was to gain cellular potency, yet moieties other than cyclopropylmethyl or cyclopropylethyl failed to improve the activity in both lactate and cytotoxicity cellular assays. Therefore, the preferred cyclopropylmethyl group was utilized in subsequent analogues. We next explored fluorine substitution at the 2- (<b>7</b>) or 3-positions (<b>8</b>) of the benzenesulfonamide ring. Analogue <b>7</b>, with a 2-fluorobenzene sulfonamide substitution, showed a similar biochemical potency to <b>1</b> and a slightly decreased activity in the lactate assay but exhibited an improvement in cytotoxicity. The 3-fluoro analogue, <b>8</b>, exhibited a significant decrease in activity across all assays. Initially, we were unable to rationalize these observations, as neither fluorine group appeared positioned to contribute an additional specific interaction with LDH, based on the previous crystal structure. However, crystal structures with 2-fluoro-containing analogues (<b>23</b>, <b>52</b><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) clearly show a preferred orientation of the ring, with fluorine occupying a position pointed into the protein and away from the space occupied by the biphenyl or alkyne substitutions. Based on the previously reported correlation between the binding kinetics and the cellular potency, we subsequently found that the 2-fluoro substitution improves the off-rate of the molecule (<named-content content-type="anchor" rid="tbl5" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>), which might explain the enhanced cytotoxic effects in MiaPaCa-2 and A673 cellular assays. Therefore, the 2-fluoro group was retained as an optimal substitution to evaluate further modifications of the lead molecule.</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of LDHA bound to inhibitor <b>23</b> (A, pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q0D">6Q0D</a>) and <b>52</b> (B, pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q13">6Q13</a>). LDHA is shown in ribbon (blue) and key residues in the active site are shown in green. Small-molecule inhibitors are shown in sticks with salmon- and magenta-colored carbons.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Biochemical and Cellular LDH Inhibition of Analogues <b>74–89</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0018.gif" alt="" id="GRAPHIC-d7e1693-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0019.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values represent the half maximal (50%) inhibitory concentration as determined in the HTS assay using a dose response in the 1536-well format. (<i>n</i> = 2 for lactate and <i>n</i> = 3 for cytotoxicity, CETSA, and biochemical LDHA.)</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. <i>k</i><sub>off</sub> and Residence Time (τ) Data Correlation to Cell Activity for Selected Analogues</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">A673 cells</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">MiaPaCa-2 cells</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char=".">lactate inh</th><th class="colsep0 rowsep0" align="center" char=".">cytotox</th><th class="colsep0 rowsep0" align="center" char=".">lactate inh</th><th class="colsep0 rowsep0" align="center" char=".">cytotox</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd.</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>D</sub> (nM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">residence time τ (h)</th><th class="colsep0 rowsep0" align="center" char=".">off rate<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a><i>k</i><sub>off</sub> (×10<sup>4</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">0.11</td><td class="colsep0 rowsep0" align="left">0.35</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">450</td><td class="colsep0 rowsep0" align="char" char=".">2454</td><td class="colsep0 rowsep0" align="char" char=".">636</td><td class="colsep0 rowsep0" align="char" char=".">8448</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char=".">0.0193</td><td class="colsep0 rowsep0" align="left">>277</td><td class="colsep0 rowsep0" align="char" char=".">0.01</td><td class="colsep0 rowsep0" align="char" char=".">509</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">333</td><td class="colsep0 rowsep0" align="char" char=".">314</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="char" char=".">0.1235</td><td class="colsep0 rowsep0" align="left">>277</td><td class="colsep0 rowsep0" align="char" char=".">0.01</td><td class="colsep0 rowsep0" align="char" char=".">487</td><td class="colsep0 rowsep0" align="char" char=".">939</td><td class="colsep0 rowsep0" align="char" char=".">424</td><td class="colsep0 rowsep0" align="char" char=".">848</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="char" char=".">0.0667</td><td class="colsep0 rowsep0" align="left">>277</td><td class="colsep0 rowsep0" align="char" char=".">0.01</td><td class="colsep0 rowsep0" align="char" char=".">873</td><td class="colsep0 rowsep0" align="char" char=".">119</td><td class="colsep0 rowsep0" align="char" char=".">403</td><td class="colsep0 rowsep0" align="char" char=".">257</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="char" char=".">0.0634</td><td class="colsep0 rowsep0" align="left">>277</td><td class="colsep0 rowsep0" align="char" char=".">0.01</td><td class="colsep0 rowsep0" align="char" char=".">760</td><td class="colsep0 rowsep0" align="char" char=".">105</td><td class="colsep0 rowsep0" align="char" char=".">367</td><td class="colsep0 rowsep0" align="char" char=".">347</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>D</sub> and <i>k</i><sub>off</sub> were determined <i>via</i> SPR on a Mass-1 instrument from Sierra Sensors and residence time τ was calculated as 1/<i>k</i><sub>off</sub> (s<sup>–1</sup>).</p></div></div></div><div class="NLM_p">Previous SAR<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> suggested that the 3-biphenyl moiety provided significant enzyme and cellular potency. However, this functional group failed to provide the pharmacokinetic profile needed for evaluation in <i>in vivo</i> studies. An initial assessment showed no clear SAR trend regarding the presence of a 4-fluoro group (<b>7</b> and <b>10</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Consequently, we employed both 4-fluoro and 4-H for further modifications of the 3-phenyl group in this region. In addition to extensive SAR in the biphenyl series, to be published in due course, we explored alkynes and alkenes as phenyl isosteres. We rationalized that alkynes, with a linear conformation and minimal steric bulk, could present the terminal groups to a hydrophobic region in the enzyme and potentially provide tighter binding. In an initial SAR exploration, a simple terminal alkyne <b>11</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) retained biochemical activity relative to <b>10</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), with only modest loss in potency toward lactate output (<2 fold) and cytotoxicity (<3 fold) in both cell lines. We thus explored a variety of substitutions on the alkyne to better understand the SAR in this hydrophobic space in LDH (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Initially, we explored the terminal alkyne region with numerous alkyl and cycloalkyl groups, which could occupy the hydrophobic pocket. Most changes were tolerated, with steadily improved cell-based activities noted as the size of the alkyl group increases (<i>e.g.</i>, <b>16–17</b> and <b>20</b>). Introducing electron-withdrawing polar groups, particularly terminal −CN (<b>33</b>) and −CF<sub>3</sub> groups (<b>34</b>) which are strong hydrogen bond acceptors, markedly reduced cellular activity. A similar trend of lower cell potency was observed for analogues <b>18</b>–<b>19</b> and <b>35–36</b>, which also possess electron-withdrawing groups capable of hydrogen bond interactions. Introducing cycloalkyl groups onto the terminal alkyne region resulted in very potent compounds in the biochemical assay, but the activity in cellular assays was more variable. Groups such as cyclopropyl (<b>21</b>), cyclobutyl (<b>22</b>), and cyclopentyl (<b>23</b> and <b>51</b>) showed promising potency increases in the cellular assays. When a methylene linker was introduced (<b>37</b><i>vs</i><b>21</b> and <b>38</b><i>vs</i><b>23</b>), the potency in the lactate inhibition assay increased or remained similar, while less potency was observed in the cytotoxicity assays. Interestingly, when the cycloalkyl groups contained hydrogen bond acceptor or donor atoms, biochemical potency was maintained, but cellular activities significantly dropped (<b>24–32</b> and <b>39–41</b> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), consistent with our previous observations.</div><div class="NLM_p">As the SAR indicated a preference for lipophilic groups at the terminal position of the alkyne, we synthesized analogue <b>42</b>, which replaced the terminal alkyl group with a 2-thienyl group. Though analogue <b>42</b> was less potent in the biochemical assay (34 nM), it showed improved cellular potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) in lactate (438 nM in A673 and 307 nM in MiaPaca-2) and cytotoxicity assays (265 nM in A673 and 268 nM in MiaPaca-2) compared to <b>10</b>. As an unsubstituted thienyl moiety could be metabolically labile,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> we incorporated a 5-methyl thienyl group, which retained a similar potency in the cellular assays (<b>43</b> and <b>52</b>, which are also named as <b>NCATS-SM1440</b> and <b>NCATS-SM1441</b>, respectively). Replacing the methyl group of thiophene with other alkyl groups (<b>45</b> and <b>55–57</b>) maintained the potency; however, bulkier substituents such as a <i>t</i>-butyl group decreased the potency (<b>45</b>). Moving the 5-methyl group to the 3-position of the thiophene resulted in a similar cellular potency (<b>53</b>), whereas switching to the 5-methylthien-3-yl substitution pattern (<b>54</b> and <b>77</b>) slightly decreased the activity. A 3,5- (<b>66</b>) or 2,5-dimethyl substitution pattern (<b>67</b> and <b>83</b>) on thiophene further improved the cellular potency. Interestingly, introduction of halogens onto 5-alkylthiophene significantly reduced the cellular potency (<b>58–60</b>). Replacing thiophene with similarly sized heterocycles was well tolerated. For instance, 5-methylfuran (<b>44</b> and <b>61</b>) showed similar inhibition of lactate production and cytotoxicity in both cell types compared to the thiophene analogues (<b>43</b> and <b>52</b>). Introducing methylthiazole (<b>46–47</b>, <b>63–65</b>, and <b>80–82</b>) or methyloxazole (<b>62</b>) also retained the biochemical and cellular potencies. The more hydrophilic methyl imidazole (<b>48</b>) or methylpyrazole (<b>49</b>) analogues showed good biochemical potency but less favorable cellular effects in lactate and cytotoxicity assays.</div><div class="NLM_p">We also examined the replacement of the alkyne linker (<b>68–73</b>, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) with an alkene motif. This slightly diminished the cellular potency despite low nanomolar biochemical activity. The alkene analogues <b>68</b> and <b>69</b> showed marginally inferior cellular potency compared to the corresponding alkyne analogues <b>52</b> and <b>23</b>. A similar trend was also observed for other matched pairs, including for alkene analogue <b>70</b> versus alkyne <b>21</b>, <b>71</b> versus <b>12</b>, and <b>72</b> versus <b>17</b>.</div><div class="NLM_p">Recalling the improved cellular potency of earlier analogues with the ethylcyclopropane pyrazole substitution, we synthesized analogues <b>74–83</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>) combining this substitution with optimal groups from the terminal alkyne region. As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, analogues <b>74–83</b> showed a noteworthy improvement in cellular potency when compared to their corresponding cyclopropylmethyl analogues mentioned above.</div><div class="NLM_p">In our previous report,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> we described several crystal structures with similar analogues bound to LDHA, establishing the binding mode of this series of compounds within the LDHA catalytic site and helping to guide our lead optimization campaign toward the identification of compounds with improved potency. As part of the optimization campaign, we obtained cocrystal structures of analogues <b>23</b> and <b>52</b> bound to LDHA in the presence of cofactor NADH (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Analogues <b>23</b> and <b>52</b> present similar binding poses as the previously reported molecules, with the carboxylate making a clearly defined salt bridge interaction with Arg168 and a hydrogen bond with Thr247. Additionally, the cyclopropylmethyl group and the cyclopropylethyl group both maintain a potential pseudo π–π interaction between the cyclopropyl group and active-site tyrosine Tyr238; this substitution was found to be vital for potent cellular activity. As demonstrated previously, the 4-benzyl sulfonamide motif makes well-defined H-bonding interactions with the side chains of Asp140 and Glu191 as well as with the main chain nitrogens of Asp140, Ile141, and Glu191 to confer potent biochemical and cellular activities. Further, the cyclopentylalkyne in analogue <b>23</b> and 5-methyl thienyl alkyne in analogue <b>52</b> presented similar binding poses compared to the biphenyl ortholog. The linear rigid confirmation of the alkynes enables the presentation of moieties that offer increased hydrophobic interactions with LDHA while offering lower lipophilicity compared with a phenyl ring. It is unclear from the crystal structure exactly how some of these variations improve cellular potency. For example, the 2-fluoro moiety on the benzenesulfonamide ring is oriented in the same way in both structures. Although this group does occupy a small subpocket in the enzyme, no specific interactions with the protein are evident. The changed binding kinetics that this substitution produces (<i>vide infra</i>) may instead result from subtle electronic modulation of the hydrogen bonding interactions, which ultimately provided better cellular potency. These efforts demonstrate that a multiparameter optimization strategy that utilizes a structure-based design combined with targeted improvement of residence time can elicit significant improvements in cellular potency, at least in the context of LDHA inhibitors.</div><div class="NLM_p">All of the synthesized analogues were profiled for their biochemical activity against both LDH isozymes, LDHA and LDHB, and several compounds were tested against a related “off-target” dehydrogenase, malate dehydrogenase (MDH). All analogues possess nearly identical potency against LDHB compared to LDHA (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Table S1</a>) and thus display a highly significant correlation coefficient (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Figure S1</a>). Recent reports allude to the need for a pan LDH inhibitor in order to elicit a cellular phenotype,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> highlighting a possible advantage of this profile. All the analogues showed little or no biochemical activity against MDH, with over 2500-fold selectivity for LDHA/LDHB (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Table S1</a>).</div><div class="NLM_p">While compounds inactive in the biochemical assay elicit no cellular activity, indicating the on-target nature of the observed effects, we noted only a marginal correlation between the biochemical LDHA inhibition and inhibition of cellular lactate production throughout this optimization campaign. This poor correlation may have hampered prior efforts to identify cell-active LDHA inhibitors. The unusual trend and poor correlation between biochemical and cellular potency may be due to the disparity in the concentration of LDHA in the biochemical assay (2 nM) versus the extremely high concentration of intracellular LDH (estimated to be in the range of 2–17 μM) in glycolytic tumor cells. Furthermore, the nature of the active site preferentially selects for highly lipophilic compounds with an acidic moiety such as carboxylic acid, which can introduce additional permeability and protein binding challenges that might reduce cellular efficacy and thus influence the correlation. Though structure-guided optimization was crucial in driving the cellular potency, it was also difficult to explain the drastically improved cytotoxic effects for certain analogues as a result of subtle structural changes during our optimization.</div><div class="NLM_p">To address such challenges, we reasoned that a highly potent compound with a long drug–target residence time might more effectively inhibit the cellular LDH activity. To better understand the reason for improved cellular activity emerging from relatively small structural changes, we complemented the structure-guided SAR with data from a surface plasmon resonance (SPR) binding assay to analyze the top cell-active compounds, in search of compounds with longer off-rates. A summary of SPR data and the corresponding cell data is outlined in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> for compounds <b>23</b>, <b>38</b>, <b>43</b>, and <b>52</b>. All analogues showed more potent binding affinities and longer off-rates (<i>k</i><sub>off</sub>) compared to <b>1</b>. Analysis of a larger data set across the series (including unpublished data) suggests that the compounds with long off-rates and residence times (τ) [calculated as 1/<i>k</i><sub>off</sub> (s<sup>–1</sup>)] offer high cellular potency in both the lactate and cytotoxicity assays. The most potent compounds exhibited off-rates that exceeded the detection limit of the assay, 10<sup>–6</sup> s<sup>–1</sup>. Comparing the data presented here with our previously published results suggests that the 2-fluoro group on the sulfonamide phenyl ring and the alkyne substitution both positively contribute to longer residence times, leading to improved binding affinity and more potent cellular activity.</div><div class="NLM_p">A good correlation was observed for the IC<sub>50</sub> values for our compounds in MiaPaCa-2 and A673 cell lines in both lactate secretion and cytotoxicity assays (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Figure S2</a>); however, the correlation between inhibition of lactate secretion and cytotoxic effect within the same cell line was poor (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Figure S3</a>). As such, a superior lactate inhibition profile did not always translate into improved cytotoxic effects for some compounds. The reason for such discrepancy is unclear but may reflect the differences in the two assay conditions. For example, in the lactate secretion assay, lactate output was measured after 30 min exposure to compound, while cytotoxicity was assessed after 48 h of incubation with compound. It is possible that the ability of compounds to display higher cytotoxic effects depends on a longer residence time along with higher lactate inhibition.</div><div class="NLM_p">Given the potential for the inhibitors to alter lactate production or viability due to unidentified off-target activity, we evaluated our LDHA inhibitors in a recently developed split Nano Luciferase (NLuc) high-throughput cellular thermal shift assay (SplitLuc-CETSA) to obtain concentration-dependent measurements of intracellular target engagement. We have previously demonstrated the applicability and performance of our SplitLuc CETSA assay to detect LDHA target engagement, in both 384- and 1536-well plate formats, for a set of well-validated inhibitors.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Accordingly, we evaluated the majority of analogues utilizing the SplitLuc-CETSA technique in HEK293T cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>), where LDHA was appended with a C-terminal 15-amino acid HiBiT fusion tag. A majority of the active compounds in the lactate assay showed cellular binding and thermal stabilization of LDHA in the SplitLuc assay. In general, we observed a good correlation between CETSA IC<sub>50</sub> and lactate IC<sub>50</sub> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Figure S4</a>), though we used different cell lines in the two assays. Our lead compounds <b>42</b> and <b>53</b>, along with many other compounds in the lactate assay, demonstrated stabilization of LDHA within cells. Though biochemically active compounds tended to show good CETSA activity, no clear correlation emerged between the cellular stabilization of LDHA <i>via</i> CETSA and the biochemical IC<sub>50</sub>. As we stated before, a similar disconnect was observed between biochemical and lactate IC<sub>50</sub>. As such biochemical LDHA screening appears to be a poor predictor of cellular effects in this series, the SplitLuc-CETSA cell-based screening platform provides a new avenue for HTS screening in a cellular context, particularly for LDHA.</div><div class="NLM_p">To assess the impact of LDH inhibitors on the glycolytic pathway, the glycolysis stress test (GST) was performed in A673 cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Glycolytic flux is assessed by measuring the extracellular acidification rate (ECAR) of the media that results from glycolysis-dependent proton production by the cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, after a stable baseline is recorded, the cells were treated with compounds (<b>43</b> and <b>52</b>-<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B) at increasing concentrations and the ECAR is measured for approximately 50 min before injection of glucose, which results in a substantial stimulation of glycolytic flux and proton production. After three further measurements, oligomycin is injected to inhibit mitochondrial oxidative phosphorylation, which results in a compensatory increase in ECAR that is then suppressed to basal levels by the hexokinase inhibitor 2-deoxyglucose. From this experiment, two glycolysis-dependent parameters can be obtained; the basal glucose-dependent ECAR (glycolysis <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C) and the maximal glycolytic capacity (shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). As anticipated, as LDH inhibition increases with the concentration of compounds <b>43</b> or <b>52</b>, the reserve biochemical capacity of LDH within the glycolytic pathway is exceeded, resulting in a depletion of NAD<sup>+</sup> and, ultimately, inhibition of the entire pathway, as evidenced by decreased ECAR.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. LDH inhibitor-dependent suppression of glycolytic flux in A673 cells. The GST was performed in A673 cells to measure the ECAR over time; cellular basal ECAR was measured and then compound <b>43</b> (panel A) or <b>52</b> (panel B) was injected at increasing concentrations. After 48 min, subsequent injections of glucose (10 mM; glycolysis), oligomycin (1 μg/mL) (O; reaching maximal glycolytic capacity), and 2-deoxyglucose (50 mM) (2-DG; inhibition of glycolysis) were made. (C) Quantification of the glycolysis (ECAR after glucose injection minus basal ECAR) and (D) quantification of the glycolytic capacity of the LDHA inhibitors, <b>43</b> and <b>52</b>, in a dose–response manner in comparison with compounds <b>1</b> and <b>23</b>. Data represent the mean ± standard error of the mean, <i>n</i> = 4–6 per group. All LDH inhibitors completely suppressed both basal and maximal glycolysis between 1 and 3 μM, with <b>43</b> and <b>52</b> being the most potent and <b>23</b> the least.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compounds in the series were assessed for their tier 1 ADME profile using stability in rat liver microsomes (RLM), PAMPA permeability, and aqueous solubility at pH 7.4 (data shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Table S2</a>). Most compounds showed good aqueous kinetic solubility, presumably enabled by the free carboxylic acid on the thiazole and the benzyl sulfonamide on the 4-position of the pyrazole. However, when additional hydrophilic groups are introduced in the alkyne region (e.g., <b>26</b>, <b>30</b>, <b>39</b>, <b>47–48</b>, <b>65</b>, and <b>80</b> in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Table S2</a>), the solubility significantly diminished. The RLM stability varied depending primarily on the structure of the terminal alkyne substitutions. Analogues with <i>n</i>-alkyl groups longer than 2-carbon linkers in the terminal alkyne group (<i>e.g.</i>, <b>14</b> and <b>15</b>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Table S2</a>) were metabolized quickly, likely because of CYP-mediated oxidation. This metabolism was blocked with <i>t</i>-butyl (<b>17</b>), <i>i</i>-Pr (<b>16</b>), hydroxy (<b>18</b>), or cyclobutyl (<b>22</b>) replacements (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Table S2</a>). As anticipated, a similar trend was observed for analogues <b>20</b> and <b>21</b>, with <i>iso</i>-butyl and cyclopropyl groups, respectively. Importantly, all the analogues with cycloalkyl or heterocycles on the terminal alkyne exhibited high microsomal stabilities of >30 min, the highest estimable <i>T</i><sub>1/2</sub> from a single-point (15 min) measurement in our assay. The PAMPA permeability was low [(1 – 10) × 10<sup>–6</sup> cm/s]<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> in most cases, presumably because of the presence of a carboxylic acid group. Our representative compounds <b>43</b> and <b>52</b> showed low permeability in the PAMPA-blood–brain barrier (BBB) assay (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). Notably, a combination of cyclopropylethyl group on the 5-position of pyrazole and methyl thiazole in the terminal alkyne (<b>80–82</b>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Table S2</a>) slightly improves the permeability. A broader assessment of the <i>in vitro</i> ADME profile for representative compounds <b>43</b> and <b>52</b> is summarized in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>. In time course studies, both representative compounds showed excellent multispecies metabolic profiles in hepatocytes (<i>T</i><sub>1/2</sub> > 150 min), cytosol (<i>T</i><sub>1/2</sub> > 120 min), plasma (<i>T</i><sub>1/2</sub> > 240 min), and liver microsomes (<i>T</i><sub>1/2</sub> > 120 min). The analogues showed very high plasma protein binding in mouse and human plasma, likely because of the presence of both carboxylic acid and sulfonamide moieties. Both compounds only showed CYP inhibition and induction at higher concentrations. Further, the compounds were not active in hERG (patch-clamp assay) and Ames profiling assays. No significant phase I and II metabolites were detected in our <i>in vitro</i> metabolic study even in the presence of GSH, which is consistent with the prolonged half-life in mouse hepatocytes. Further, both compounds showed less than 2-fold PXR and AhR activation at a 10 μM.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Summary of <i>in Vitro</i> ADME Profiles for <b>43</b> and <b>52</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" rowspan="2" align="center" valign="middle">profiling assays</th><th class="colsep0 rowsep0" align="center"><b>43</b></th><th class="colsep0 rowsep0" align="center"><b>52</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"><b>(NCATS-SM1440)</b></th><th class="colsep0 rowsep0" align="center"><b>(NCATS-SM1440)</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">liver microsomal <i>T</i><sub>1/2</sub> (human, mouse, and rat)</td><td class="colsep0 rowsep0" align="left">>120 min</td><td class="colsep0 rowsep0" align="left">>120 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">liver cytosolic <i>T</i><sub>1/2</sub> (human, mouse, and rat)</td><td class="colsep0 rowsep0" align="left">>120 min</td><td class="colsep0 rowsep0" align="left">>120 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">metabolic stability (hepatocytes-rat, dog, and mouse)</td><td class="colsep0 rowsep0" align="left">>150 min</td><td class="colsep0 rowsep0" align="left">>150 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">plasma stability <i>T</i><sub>1/2</sub></td><td class="colsep0 rowsep0" align="left">>240 min</td><td class="colsep0 rowsep0" align="left">>240 min</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAMPA permeability</td><td class="colsep0 rowsep0" align="left">10.4 × 10<sup>–6</sup> (cm/s)</td><td class="colsep0 rowsep0" align="left">2.2 × 10<sup>–6</sup> (cm/s)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAMPA-BBB permeability</td><td class="colsep0 rowsep0" align="left">16 × 10<sup>–6</sup> (cm/s)</td><td class="colsep0 rowsep0" align="left">9.6 × 10<sup>–6</sup> (cm/s)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility at pH 7.4</td><td class="colsep0 rowsep0" align="left">>70 μg/mL</td><td class="colsep0 rowsep0" align="left">>70 μg/mL</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">plasma protein binding (Fu = % fraction unbound)</td><td class="colsep0 rowsep0" align="left">0 (human), 0.7 (mouse)</td><td class="colsep0 rowsep0" align="left">0 (human), 4.4 (mouse)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP450 inhibition (isozyme)</td><td class="colsep0 rowsep0" align="left">2C8: 80% @10 μM, 2B6: 46% @10 μM, 2C9: 45% @10 μM</td><td class="colsep0 rowsep0" align="left">2C8: 7.5 μM, 2B6: 14 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">metabolite ID (<i>in vitro</i>)</td><td class="colsep0 rowsep0" align="left">no significant phase I or II metabolites</td><td class="colsep0 rowsep0" align="left">no significant phase I or II metabolites</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP induction</td><td class="colsep0 rowsep0" align="left">no induction at 10 μM</td><td class="colsep0 rowsep0" align="left">no induction at 10 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PXR and AhR Activation</td><td class="colsep0 rowsep0" align="left"><2-fold activation up to 10 μM</td><td class="colsep0 rowsep0" align="left"><2-fold activation up to 10 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reactive metabolite formation</td><td class="colsep0 rowsep0" align="left">no GSH adducts formed</td><td class="colsep0 rowsep0" align="left">no GSH adducts formed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hERG (patch clamp)</td><td class="colsep0 rowsep0" align="left">>10 μM</td><td class="colsep0 rowsep0" align="left">>10 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ames</td><td class="colsep0 rowsep0" align="left">negative</td><td class="colsep0 rowsep0" align="left">negative</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><p class="first last">Aqueous kinetic solubility (PBS buffer) and PAMPA permeability, liver microsomal & cytosolic stability studies, were conducted at NCATS. Mouse plasma stability studies were conducted at Pharmaron Inc. and involved five-time points. The microsomal stability data [mouse liver microsomes (MLM), human liver microsomes (HLM), and mouse hepatocytes] were conducted at QuintaraBio and represent the stability in the presence of NADPH and UDPGA. The parent compound was monitored at five-time points.</p></div></div></div><div class="NLM_p">Encouraged by the improved cellular potency and <i>in vitro</i> ADME profiles, we evaluated top compounds for mouse PK properties following IV and PO administration (summarized in <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). Representative analogues (<b>43–44</b>, <b>46–47</b>, <b>52</b>, <b>61</b>, <b>69</b>, and <b>74–75</b>) showed improved mouse PK properties compared to <b>1</b>. IV administration at 10 mg/kg for selected alkyne analogues showed a terminal half-life of ∼5 h or more, far superior to 0.85 h for <b>1</b>. Thus, both the use of the substituted alkyne as a phenyl isostere and adding 2-fluoro in the sulfonamide region seem to help improve the <i>in vivo</i> half-life. Moreover, most of these analogues, except for <b>46</b>, also showed reduced clearance. The clearance increased steadily as the steady-state volume of distribution (<i>V</i><sub>ss</sub>) increased, as noted for several matched pairs in the series of analogues <b>44</b>, <b>46–47</b>, <b>61</b>, and <b>69</b>. Despite the presence of the carboxylic acid functional group, favorable systemic exposure was achieved for several of these compounds, with plasma concentrations in the range of cellular IC<sub>50</sub> values following PO dosing at 40 or 50 mg/kg. Introducing more lipophilic groups, such as 5-methyl thienyl (<b>43</b> and <b>52</b>), 5-methyl furyl (<b>44</b> and <b>61</b>), or cyclopentyl (<b>69</b>) in the alkyne region, significantly improved the bioavailability (74% for <b>43</b>; 106% for <b>44</b>; 64% for <b>52</b>, and 83% for <b>69</b>) presumably because of higher lipophilicity leading to better permeability and absorption. Analogues <b>43</b> (AUC<sub>last</sub> = 31.39 μg/mL h and <i>C</i><sub>max</sub> of 5.26 μg/mL at 40 mg/kg) and <b>52</b> (AUC<sub>last</sub> = 53.54 μg/mL h and <i>C</i><sub>max</sub> of 8.83 μg/mL) possessed the best systemic exposure in this series. Relative to these molecules, replacing the cyclopropylmethyl group with cyclopropylethyl in the 5-position of the pyrazole and keeping 5-methyl thienyl in the alkyne region (analogues <b>76</b> and <b>77</b> with 37 and 42%, respectively) significantly reduced the bioavailability and systemic exposure. A similar trend was manifested for analogues <b>44</b> and <b>46</b> that contain methyl thiazole groups at the terminal alkyne. Overall, our SAR study shows that the 5-methylthienyl group in the alkyne region is key for improved pharmacokinetic properties. Thus, analogues <b>43</b> and <b>52</b> possess suitable PK profiles for evaluation in efficacy models.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Pharmacokinetic Profiles of Top Analogues in CD1 Mice<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">route</th><th class="colsep0 rowsep0" align="center" char=".">dose</th><th class="colsep0 rowsep0" align="center" char=".">Cl (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><a class="ref internalNav" href="#t8fn2" aria-label="b">b</a><i>C</i><sub>max</sub> (μg/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>last</sub> (μg/mL·h)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center"><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">16.1</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="char" char=".">41.79</td><td class="colsep0 rowsep0" align="char" char=".">10.57</td><td class="colsep0 rowsep0" align="char" char=".">1.79</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">2.78</td><td class="colsep0 rowsep0" align="char" char=".">5.26</td><td class="colsep0 rowsep0" align="char" char=".">31.39</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">73.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">75</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="char" char=".">4.77</td><td class="colsep0 rowsep0" align="char" char=".">2.32</td><td class="colsep0 rowsep0" align="char" char=".">10.3</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">10.7</td><td class="colsep0 rowsep0" align="char" char=".">1.26</td><td class="colsep0 rowsep0" align="char" char=".">13.52</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">106</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char=".">5.75</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">1.82</td><td class="colsep0 rowsep0" align="char" char=".">13.9</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">1.82</td><td class="colsep0 rowsep0" align="char" char=".">3.44</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">128</td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">3.32</td><td class="colsep0 rowsep0" align="char" char=".">1.36</td><td class="colsep0 rowsep0" align="char" char=".">17.9</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.54</td><td class="colsep0 rowsep0" align="char" char=".">2.03</td><td class="colsep0 rowsep0" align="char" char=".">2.65</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">10.4</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">33.82</td><td class="colsep0 rowsep0" align="char" char=".">16.53</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">2.6</td><td class="colsep0 rowsep0" align="char" char=".">8.83</td><td class="colsep0 rowsep0" align="char" char=".">53.54</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">72.3</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">4.57</td><td class="colsep0 rowsep0" align="char" char=".">2.28</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">2.28</td><td class="colsep0 rowsep0" align="char" char=".">10.17</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>69</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">74.7</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">6.44</td><td class="colsep0 rowsep0" align="char" char=".">2.81</td><td class="colsep0 rowsep0" align="char" char=".">13.3</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">9.22</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">74.7</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">35.33</td><td class="colsep0 rowsep0" align="char" char=".">6.65</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">4.73</td><td class="colsep0 rowsep0" align="char" char=".">12.24</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75</b></td><td class="colsep0 rowsep0" align="left">IV</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">18.6</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="char" char=".">58.23</td><td class="colsep0 rowsep0" align="char" char=".">9.6</td><td class="colsep0 rowsep0" align="char" char=".">1.47</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PO</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">6.85</td><td class="colsep0 rowsep0" align="char" char=".">20.03</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">42</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">Values calculated from drug concentration in plasma following IV or PO dosing. <i>n</i> = 3, 8 time points taken over 24 h. Compounds were formulated as a solution in PBS with 1.1 equiv of NaOH (final pH 7–8).</p></div><div class="footnote" id="t8fn2"><sup><sup>b</sup></sup><p class="last"><i>C</i><sub>max</sub> = <i>C</i><sub>0</sub> (<i>t</i> = 0) for IV administration. All pharmacokinetic studies were conducted at Pharmaron, Inc.</p></div></div></div><div class="NLM_p">Despite the long-standing interest in drugging LDH, efforts have thus far failed to achieve sub-micromolar cellular potency while also achieving optimal PK properties. Having identified the best-in-class lead compounds in terms of cellular potency and PK profile, we evaluated the top compounds in a mouse tumor model using A673 cells to demonstrate LDH inhibition <i>in vivo</i> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). An IV administration at 50 mg/kg of <b>43</b> and <b>52</b> significantly reduced the LDH activity in tumors analyzed at 1 and 6 h postdose. These two analogues have high exposure and low clearance rates compared to analogues <b>47</b>, <b>69</b>, and <b>74</b>. In mice dosed with <b>52</b>, 80% inhibition of tumor LDH activity was observed at 6 h postdose, compared to approximately 60% inhibition with <b>43</b>. Although both <b>43</b> and <b>52</b> possess good pharmacokinetic properties, <b>52</b> has a slightly higher <i>C</i><sub>max</sub> and lower clearance, which correlates with its superior <i>in vivo</i> LDH inhibition profile. <b>56</b> and <b>74</b> showed moderate inhibition of LDH activity in tumors, and <b>14</b>, <b>47</b>, <b>69</b>, and <b>71</b> have less effects. <b>56</b> suppressed tumor LDH activity below 40% of control at 1 h postdose but failed to maintain significant inhibition at 6 h possibly because of a higher clearance. The diminished activity of analogues <b>47</b>, <b>69</b>, and <b>74</b> can be attributed to a less optimal PK profile having reduced exposure and higher clearance compared to <b>43</b> and <b>52</b> (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). These data suggest an excellent correlation of <i>in vivo</i> LDH inhibition with the pharmacokinetic profile, which is vital to demonstrate sustained <i>in vivo</i> LDH inhibition. It is important to note that analogues <b>42</b> and <b>53</b> are the first LDH inhibitors to demonstrate sustained target engagement <i>in vivo</i>. Of note, a clear correlation is evident between the cellular concentrations necessary to affect glycolytic flux and the concentrations achieved <i>in vivo</i> at doses that demonstrate target engagement. Consistent with these data, both compounds showed efficacy in sarcoma<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and pancreatic<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> xenograft models at similar doses.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <i>In vivo</i> target engagement and tumor LDH activity upon administration of compounds in mice bearing A673 flank xenograft tumors. Mice received a single IV injection of LDHA inhibitors <b>14</b>, <b>43</b>, <b>47</b>, <b>52</b>, <b>56</b>, <b>69</b>, and <b>71</b> at 50 mg/kg. At the time of sacrifice, the samples of tumor and plasma were collected and flash-frozen in liquid nitrogen. Compound levels in plasma and tumor were determined by LC–MS/MS, and LDHA activity was measured in tumor lysates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to develop an LDH inhibitor suitable for <i>in vivo</i> experiments, a diverse set of parameters were used to inform design decisions, with a focus on improving cellular potency, establishing a long residence time in the binding pocket, and identifying optimal PK properties. In our case, structure-guided optimization coupled with long residence time helped to identify lead compounds with nanomolar cellular potency. Ultimately, improvement in PK attributes allowed us to demonstrate the first notable inhibition of LDH in tumors following dosing of mice harboring A673 xenografts. We identified critical structural features in each region of the molecule that contributed to improvements in the properties summarized in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. In short, the carboxylic acid on the thiazole ring and the 4-benzyl sulfonamide are keystone interactions critical for biochemical and cellular potency. Meanwhile, cyclopropylmethyl or cyclopropylethyl substitution on the pyrazole was crucial for cellular potency. The alkyne linker in the biphenyl region and the 2-fluoro group on benzenesulfonamide provide a longer off-rate that further improves the cellular activity. Heterocycles, such as 5-methylthienyl, 5-methylfuryl, or thiazolyl groups distal to the alkyne, contribute to improved cellular potency and, most importantly, improved <i>in vitro</i> ADME and <i>in vivo</i> PK properties.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SAR summary and essential structural moieties contributing to the cellular potency, binding affinity, and PK properties of analogue <b>52</b> (<b>NCATS-SM1441</b>) as an example.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24772" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24772" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Pharmaceutical companies and researchers in academia have invested significant efforts to identify small-molecule inhibitors of LDH. Such attempts appear to have shown limited success in discovering potent and drug-like inhibitors that can be utilized to validate LDH as a drug target <i>in vivo</i>, a necessary step toward advancing such a molecule toward the clinic. We describe a lead optimization campaign, aided by crystallography as well as the assessment of residence time, that has resulted in LDH inhibitors with improved cellular potency, <i>in vitro</i> ADME, and <i>in vivo</i> PK properties. We report the first LDH inhibitors showing direct target engagement in human tumor xenografts. Through a systematic SAR campaign, we identified optimal structural features in each region of the molecule that contributed to slow off-rates and improved PK properties, parameters which we found to be significant determinants of improved cellular potency and <i>in vivo</i> target engagement. We report compounds with enhanced cellular potency and properties that enable modulation of LDH activity <i>in vivo</i>, thus opening the door for studies to evaluate LDH as a pharmaceutical target in cancer models such as Ewing’s sarcoma<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> or pancreatic cancers.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22714" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22714" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Chemistry</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> General Methods</h4><div class="NLM_p last">All air- or moisture-sensitive reactions were performed under a positive pressure of nitrogen or argon with oven-dried glassware. Anhydrous solvents and bases such as dichloromethane (DCM), <i>N</i>,<i>N</i>-dimethylformamide, acetonitrile, ethanol, DMSO, dioxan, and DABCO were purchased from Sigma-Aldrich. Palladium catalysts were purchased from Johnson Matthey and used as such. Preparative purification was performed on a Waters semipreparative HPLC system using a Phenomenex Luna C18 column (5 μm, 30 × 75 mm) at a flow rate of 45 mL/min. The mobile phase consisted of acetonitrile and water (each containing 0.1% trifluoroacetic acid). A gradient of 10–50% acetonitrile over 8 min was used during the purification. Fraction collection was triggered by UV detection (220 nm). The analytical analysis was performed on an Agilent LC–MS (Agilent Technologies, Santa Clara, CA). Method 1: A 7 min gradient of 4–100% acetonitrile (containing 0.025% trifluoroacetic acid) in water (containing 0.05% trifluoroacetic acid) was used with an 8 min run time at a flow rate of 1 mL/min. A Phenomenex Luna C18 column (3 μm, 3 × 75 mm) was used at a temperature of 50 °C. Method 2: A 3 min gradient of 4–100% acetonitrile (containing 0.025% trifluoroacetic acid) in water (containing 0.05% trifluoroacetic acid) was used with a 4.5 min run time at a flow rate of 1 mL/min. A Phenomenex Gemini Phenyl column (3 μm, 3 × 100 mm) was used at a temperature of 50 °C. Purity determination was performed using an Agilent Diode Array Detector for both method 1 and method 2. The mass determination was performed using an Agilent 6130 mass spectrometer with electrospray ionization in the positive mode. <sup>1</sup>H NMR spectra were recorded on Varian 400 MHz spectrometers. Chemical shifts are reported in parts per million with an undeuterated solvent (DMSO-<i>d</i><sub>6</sub> at 2.49 ppm) as an internal standard for DMSO-<i>d</i><sub>6</sub> solutions. All of the analogues tested in the biological assays have purity greater than 95% based on both analytical methods. High-resolution mass spectrometry was performed on an Agilent 6210 time-of-flight LC–MS system. Confirmation of molecular formula was accomplished using electrospray ionization in the positive mode with the Agilent Masshunter software (version B.02).</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> General Procedure for the Synthesis of Acylbenzotriazole Derivatives <b>Ia–l</b> (Procedure A)</h4><div class="NLM_p last">These compounds were prepared as described previously.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> To the solution of 1<i>H</i>-benzo[<i>d</i>][1,2,3]triazole (476 g, 3995 mmol, 4 equiv) in DCM (600 mL) was added thionyl chloride (72.9 mL, 999 mmol, 1 equiv), and the reaction mixture was stirred at room temperature for 0.5 h. After cooling in an ice–water bath (for larger scale, the cooling was necessary because of the exothermic reaction. If the reaction mixture formed a thick precipitate that was difficult to stir, then more DCM was added), carboxylic acid (999 mmol, 1 equiv) was carefully added and the mixture was stirred for 6 h. The reaction was filtered, and the filter cake was washed with DCM. To the filtrate was added bicarbonate solution slowly, and the resulting mixture was stirred for 30 min and then transferred to a separatory funnel. The organic layer was subsequently washed with bicarbonate solution and brine solution. The organic layer was dried under sodium sulfate and concentrated to provide a thick oil. The crude product was purified on a CombiFlash system using a 340 g silica column eluting with 0–20% ethyl acetate in hexanes over 10 column volumes. The first peak was collected, concentrated, and dried to get pure products <b>Ia–l</b> as oil, which eventually solidifies into white solid in 91–100% yield.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> General Procedure for the Synthesis of 1-Aryl-3-substituted Propane-1,3-diones <b>III</b> (Procedure B)</h4><div class="NLM_p last">To a mixture of appropriate acetophenone <b>IIa–d</b> (51.0 mmol, 1 equiv), magnesium bromide diethyl etherate (32.9 g, 127 mmol, 2.5 equiv), and the corresponding acylbenzotriazole derivative <b>Ia–l</b> (76 mmol, 1.5 equiv in CH<sub>2</sub>Cl<sub>2</sub>) (100 mL) was added Hunig’s base (26.7 mL, 153 mmol, 3 equiv) slowly (cooling is necessary for large scale), and the mixture was stirred at rt for 12 h. The reaction mixture was cooled in an ice bath and quenched with 1 M HCl. The product was extracted with CH<sub>2</sub>Cl<sub>2</sub>, and the organic layer was subsequently washed with brine. After drying the organic layer with MgSO<sub>4</sub>, the crude product was purified on an Isco flash system using a 220 g gold column eluting with 0–30% ethyl acetate over 20 column volumes in hexanes to afford a yellow oil after removing the solvent in 60–69% yield required intermediates <b>III</b>.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> General Procedure for the Alkylation of 1-Aryl-3-substituted Propane-1,3-diones <b>Va–r</b> (Procedure C)</h4><div class="NLM_p last">1-Aryl-3-substituted propane-1,3-dione (35.9 mmol, 1 equiv) and cesium carbonate (14.05 g, 43.1 mmol, 1.2 equiv) in DMSO (25 mL) were stirred at room temperature for 10 min. The appropriate 4-(bromomethyl)benzenesulfonamide <b>IVa–c</b> (43.1 mmol, 1.2 equiv) was added in one portion, and the reaction mixture was further stirred at room temperature for another 1–2 h. The resulting mixture was diluted with large excess of ethyl acetate and filtered through Celite to remove any solid impurities. The filtrate was washed with saturated ammonium chloride (3×) and then with brine. The organic layer was dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified directly on a flash system using a 220 g silica column eluting with 20–60% ethyl acetate in hexanes over 16 column volumes to afford pure products <b>Va–r</b> in 55–83% yield.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> General Procedure for the Cyclization of 1-Aryl-2,3-disubstituted Propane-1,3-diones To Obtain <b>VIa–r</b> and <b>VIIa–r</b> (Procedure D)</h4><div class="NLM_p last">To a mixture of the appropriate 1-aryl-2, 3-disubstituted propane-1,3-dione (2.24 mmol, 1 equiv) in ethanol were added pyrrolidine (0.5 equiv) and tosic acid (0.5 equiv), and the mixture was heated at reflux for 1–2 h. The reaction mixture was removed from the heating block, and then ethyl 2-hydrazinylthiazole-4-carboxylate hydrogen bromide (0.600 g, 2.24 mmol, 1 equiv) was added in one portion. The reaction was again heated at reflux for 12 h. The reaction mixture was concentrated, and the residue was taken up in DCM and loaded to a prepacked silica loading cartridge. The product was purified on a flash system using a 100 g silica column eluting with 20–40% ethyl acetate in hexanes to get an inseparable 1:1 mixture of regioisomers in a combined yield of 77–83%. The product mixture was then taken up in DMSO and injected onto a C18 gold column and eluted with 60–100% ACN–water (contains 0.1% TFA). The second peak (the desired isomers <b>VIa–r</b>) was pooled, and most of the ACN was removed under reduced pressure. The solid formed was collected by filtration, washed with water, and air-dried to get pure compounds <b>VIa–r</b> in 21–36% yield.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> General Procedure for Sonogashira Coupling Using the P(<i>t</i>-Bu)<sub>3</sub>]Pd(crotyl)Cl Catalyst (Procedure E)</h4><div class="NLM_p last">A mixture of ethyl 2-(3-(3-bromo-4<i>H</i>/fluoro phenyl)-5-(substituted)-4-(3/4-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate (1 mmol), P(<i>t</i>-Bu)<sub>3</sub>]Pd(crotyl)Cl precatalyst Pd-162 (5 mol %), and DABCO (2 mmol, 2 equiv) in dioxan (0.5 M concentration) was bubbled with argon for 5 min, then alkyne (1.5 mmol, 1.5 equiv) was added, and the mixture was stirred at room temperature overnight. After completion of the reaction, a silica-bound palladium scavenger was added and the mixture was stirred at room temperature for 1 h, then diluted with ethyl acetate, and filtered through a pad of Celite. The filtrate was concentrated, and the residue was purified on an Isco flash system eluting with 20–40% ethyl acetate in hexanes over 20 column volumes to obtain pure products (NOTE: the above reaction also works with 10 mol % tri-<i>tert</i>-butylphosphonium tetrafluoroborate and 5 mol % allylpalladium chloride dimer under the same conditions).</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General Procedure for Sonogashira Coupling Using the [DTBNpP]Pd(crotyl)Cl Catalyst (Procedure F)</h4><div class="NLM_p last">A mixture of ethyl 2-(3-(3-bromo-4<i>H</i>/fluoro phenyl)-5-(substituted)-4-(3/4-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate (1 mmol), DTBNpP]Pd(crotyl)Cl precatalyst Pd-163, and DABCO (2 mmol, 2 equiv) in dioxane (0.5 M concentration) was bubbled with argon for 5 min, alkyne (1.5 mmol, 1.5 equiv) was added, and the reaction mixture was stirred at 60 °C overnight. After completion of the reaction, a silica-bound palladium scavenger was added and the mixture was stirred at room temperature for 1 h, then diluted with ethyl acetate, and filtered through a pad of Celite. The filtrate was concentrated, and the residue was purified on an Isco flash system eluting with 20–100% ethyl acetate in hexanes over 25 column volumes to obtain pure products.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> General Procedure for Suzuki Coupling (Procedure G)</h4><div class="NLM_p last">To a mixture of ethyl 2-(3-(3-bromo-4-substitutedphenyl)-5-(cyclopropylmethyl)-4-(3-substituted-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carbox-ylate (0.2 mmol, 1 equiv), potassium phosphate (0.4 mmol, 2 equiv), SPhos Pd(crotyl)Cl precatalyst Pd-172 (2.5 mol %), and the appropriate boronic acid/pinacol ester/trifluoroboronate in a Biotage microwave vial were added dioxane (2 mL) and water (0.5 mL). The reaction mixture was bubbled with argon for few minutes, then capped, and stirred at 100 °C in a preheated heating block for 0.5 h. Upon completion of the reaction, the reaction mixture was cooled and stirred with a metal scavenger for 1 h. The reaction mixture was then diluted with ethyl acetate and filtered through a pad of Celite. The filtrate was concentrated and purified on an Isco flash system using a silica column eluting with 20–40% ethyl acetate in hexanes to get pure products.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Procedure for the Hydrolysis of Esters To Synthesize Analogues <b>1–89</b> (Procedure H)</h4><div class="NLM_p">The corresponding ethyl ester intermediates were suspended in tetrahydrofuran (THF)–MeOH (3/2; 5 mL) and treated with 1.5 M solution (5 equiv) of aqueous lithium hydroxide. The reaction mixture was stirred for 1 h, and the solvent was removed by forced air and acidified with 1 M hydrochloric acid. The crude material was taken up in DMSO and purified on a preparative HPLC.</div><div id="sec5_1_9_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> 2-(3-([1,1′-Biphenyl]-3-yl)-5-((2,2-difluorocyclopropyl)methyl)-4-(4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>2</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, and H <i>via</i> intermediates <b>Ib</b> through <b>VIIg</b>. LC–MS retention time: (method 1) = 6.071 min and (method 2) = 3.483 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.77–7.67 (m, 4H), 7.63–7.50 (m, 2H), 7.49–7.41 (m, 4H), 7.39–7.28 (m, 5H), 4.19 (s, 2H), 3.45–3.35 (m, 2H), 2.27–2.11 (m, 1H), 1.56–1.30 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + Na)<sup>+</sup> calcd for C<sub>30</sub>H<sub>24</sub>F<sub>2</sub>N<sub>4</sub>NaO<sub>4</sub>S<sub>2</sub>, 629.1099; found, 629.1122.</div></div><div id="sec5_1_9_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> 2-(3-([1,1′-Biphenyl]-3-yl)-5-(2-cyclopropylethyl)-4-(4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>3</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, and H <i>via</i> intermediates <b>Ic</b> through <b>VIIh</b>. LC–MS retention time: (method 1) = 6.355 min and (method 2) = 3.598 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.12 (s, 1H), 8.28 (s, 1H), 7.79–7.65 (m, 4H), 7.61 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.57–7.48 (m, 1H), 7.52–7.40 (m, 4H), 7.44–7.33 (m, 2H), 7.38–7.29 (m, 4H), 4.18 (s, 2H), 3.30–3.22 (m, 2H), 1.45 (q, <i>J</i> = 7.3 Hz, 2H), 0.87–0.67 (m, 1H), 0.36–0.25 (m, 2H), 0.17–0.09 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.79, 152.52, 144.58, 144.09, 142.23, 140.38, 139.40, 132.38, 129.51, 129.01, 128.26, 127.73, 126.95, 126.57, 126.53, 126.00, 125.87, 125.68, 117.02, 33.32, 28.37, 24.81, 15.23, 15.20, 10.62, 4.19; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>29</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 585.1625; found, 585.1604.</div></div><div id="sec5_1_9_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> 2-(3-([1,1′-Biphenyl]-3-yl)-4-(4-sulfamoylbenzyl)-5-(2,2,2-trifluoroethyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>4</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, and H <i>via</i> intermediates <b>Id</b> through <b>VIIk</b>. LC–MS retention time: (method 1) = 5.991 min and (method 2) = 3.588 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.24 (s, 1H), 8.33 (s, 1H), 7.74–7.62 (m, 5H), 7.50 (td, <i>J</i> = 7.6, 0.7 Hz, 1H), 7.43 (d, <i>J</i> = 4.3 Hz, 4H), 7.39–7.28 (m, 5H), 4.69 (q, <i>J</i> = 10.5 Hz, 2H), 4.30 (s, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>22</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 599.1029; found, 599.1043.</div></div><div id="sec5_1_9_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> 2-(3-([1,1′-Biphenyl]-3-yl)-4-(4-sulfamoylbenzyl)-5-(1-(trifluoromethyl)cyclopropyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>5</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, and H <i>via</i> intermediates <b>Ie</b> through <b>VIIl</b>. LC–MS retention time: (method 1) = 6.031 min and (method 2) = 3.61 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.11 (s, 1H), 8.34 (s, 1H), 7.76–7.70 (m, 2H), 7.69–7.62 (m, 3H), 7.52–7.45 (m, 1H), 7.45–7.27 (m, 9H), 4.37 (s, 2H), 1.79–1.09 (m, 5H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>24</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 625.1186; found, 625.1215.</div></div><div id="sec5_1_9_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 2-(3-([1,1′-Biphenyl]-3-yl)-5-(2,2-difluorocyclopropyl)-4-(4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>6</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, and H <i>via</i> intermediates <b>If</b> through <b>VIIm</b>. LC–MS retention time: (method 1) = 5.858 min and (method 2) = 3.553 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.20–12.91 (m, 1H), 8.34 (s, 1H), 7.79–7.66 (m, 4H), 7.62–7.50 (m, 2H), 7.49–7.40 (m, 4H), 7.39–7.27 (m, 5H), 4.22 (s, 2H), 3.29–3.23 (m, 1H), 2.25–2.04 (m, 1H), 1.85–1.63 (m, 1H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)+ calcd for C<sub>29</sub>H<sub>23</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 593.1123; found, 593.114.</div></div><div id="sec5_1_9_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 2-(3-([1,1′-Biphenyl]-3-yl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>7</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, and H <i>via</i> intermediates <b>Ia</b> through <b>IVb</b> and <b>VIIb</b>. LC–MS retention time: (method 1) = 6.133 min and (method 2) = 3.641 min; 1H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.30 (s, 1H), 7.73–7.65 (m, 3H), 7.64–7.58 (m, 3H), 7.52 (t, <i>J</i> = 7.7 Hz, 1H), 7.49–7.41 (m, 4H), 7.40–7.33 (m, 1H), 7.26–7.07 (m, 2H), 4.20 (s, 2H), 3.19 (d, <i>J</i> = 6.9 Hz, 2H), 1.34–1.04 (m, 1H), 0.44–0.29 (m, 2H), 0.29–0.17 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>26</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 589.1374; found, 589.1363.</div></div><div id="sec5_1_9_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> 2-(3-([1,1′-Biphenyl]-3-yl)-5-(cyclopropylmethyl)-4-(2-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>8</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, and H <i>via</i> intermediates <b>Ia</b> through <b>IVc</b> and <b>VIIc</b>. LC–MS retention time: (method 1) = 6.234 min and (method 2) = 3.576 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.31 (s, 1H), 7.73–7.64 (m, 2H), 7.62–7.56 (m, 2H), 7.53 (ddd, <i>J</i> = 7.6, 4.2, 1.2 Hz, 2H), 7.51–7.40 (m, 6H), 7.40–7.33 (m, 1H), 7.21 (t, <i>J</i> = 7.8 Hz, 1H), 4.14 (s, 2H), 3.20 (d, <i>J</i> = 6.9 Hz, 2H), 1.27–1.06 (m, 1H), 0.40–0.31 (m, 2H), 0.27–0.20 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>26</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 589.1374; found, 589.1394.</div></div><div id="sec5_1_9_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 2-(5-(Cyclopropylmethyl)-3-(6-fluoro-[1,1′-biphenyl]-3-yl)-4-(4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>9</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, and H <i>via</i> intermediates <b>Ia</b> through <b>IVa</b> and <b>VIId</b>. LC–MS retention time: (method 1) = 6.297 min and (method 2) = 3.534 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.56 (s, 1H), 6.68 (s, 1H), 6.11 (d, <i>J</i> = 8.1 Hz, 2H), 6.03 (ddd, <i>J</i> = 8.7, 4.7, 2.3 Hz, 1H), 5.96 (dd, <i>J</i> = 7.6, 2.3 Hz, 1H), 5.88–5.66 (m, 10H), 2.56 (s, 2H), 1.56 (d, <i>J</i> = 6.9 Hz, 2H), −0.37 to −0.53 (m, 1H), −1.28 (dt, <i>J</i> = 8.5, 2.8 Hz, 2H), −1.35 to −1.42 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>26</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 589.1374; found, 589.1364.</div></div><div id="sec5_1_9_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(6-fluoro-[1,1′-biphenyl]-3-yl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>10</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.78 mmol) with phenylboronic acid utilizing the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.073 min and (method 2) = 3.54 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.30 (s, 1H), 7.72–7.62 (m, 2H), 7.60 (s, 2H), 7.55 (dd, <i>J</i> = 7.6, 2.3 Hz, 1H), 7.50–7.35 (m, 6H), 7.19 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.08 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.18 (s, 2H), 3.24–3.11 (m, 2H), 1.24–1.10 (m, 1H), 0.45–0.31 (m, 2H), 0.29–0.17 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.77, 161.19, 151.50, 144.67, 144.47, 134.41, 129.72, 128.70, 128.67, 128.64, 128.57, 128.53, 128.29, 128.14, 126.13, 123.64, 116.95, 116.72, 116.55, 116.45, 116.23, 28.23, 28.14, 15.18, 10.32, 4.49; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 607.128; found, 607.129.</div></div><div id="sec5_1_9_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 2-(5-(Cyclopropylmethyl)-3-(3-ethynyl-4-fluorophenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>11</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (3.14 mmol, 2 g) with trimethylsilylacetylene utilizing the general Sonogashira coupling procedure E. The product (3.05 mmol, 2 g) obtained was deprotected while stirring with CsF (0.51 g, 3.36 mmol, 1.1 equiv) in THF/ethanol (11 mL/4 mL) at room temperature for 2 h. The reaction mixture was diluted with ethyl acetate and filtered through a Celite pad. The crude product after removing the solvent was purified in an Isco flash system using a silica column eluting with 20–40% ethyl acetate in hexanes to get pure TMS deprotected product <b>VIIIb</b> in 90% yield over two steps. LC–MS retention time: (method 2) = 3.72 (M + H)<sup>+</sup> = 583. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.38 (d, <i>J</i> = 1.2 Hz, 1H), 7.75–7.60 (m, 3H), 7.57 (s, 2H), 7.39–7.31 (m, 1H), 7.14 (dd, <i>J</i> = 11.3, 1.7 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.55 (s, 1H), 4.32 (qd, <i>J</i> = 7.1, 1.1 Hz, 2H), 4.16 (s, 2H), 3.16 (d, <i>J</i> = 6.9 Hz, 2H), 1.32 (td, <i>J</i> = 7.1, 1.1 Hz, 2H), 1.23–1.08 (m, 1H), 0.39–0.31 (m, 2H), 0.25 (dt, <i>J</i> = 5.1, 1.4 Hz, 2H). A portion of <b>VIIIb</b> was hydrolyzed using general procedure H to obtain analogue <b>11</b>. LC–MS retention time: (method 1) = 5.653 min and (method 2) = 3.66 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.74–7.59 (m, 3H), 7.57 (s, 2H), 7.35 (t, <i>J</i> = 9.1 Hz, 1H), 7.14 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.55 (s, 1H), 4.16 (s, 2H), 3.16 (d, <i>J</i> = 6.9 Hz, 2H), 1.22–1.04 (m, 1H), 0.43–0.26 (m, 2H), 0.26–0.11 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>26</sub>H<sub>21</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 555.0967; found, 555.0966.</div></div><div id="sec5_1_9_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(prop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>12</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.314 mmol, 0.2 g) with potassium trifluoro(prop-1-yn-1-yl)borate (0.069 g, 0.471 mmol) using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.928 min and (method 2) = 3.518 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.65 (t, <i>J</i> = 7.9 Hz, 1H), 7.59 (d, <i>J</i> = 6.6 Hz, 3H), 7.53 (ddd, <i>J</i> = 8.6, 5.0, 2.3 Hz, 1H), 7.30 (dd, <i>J</i> = 9.5, 8.6 Hz, 1H), 7.14 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.14 (s, 2H), 3.16 (s, 2H), 2.09 (s, 3H), 1.20–1.06 (m, 1H), 0.39–0.29 (m, 2H), 0.25–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>23</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 569.1123; found, 569.1116.</div></div><div id="sec5_1_9_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 2-(3-(3-(But-1-yn-1-yl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>13</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with 2-(1-butyn-1-yl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (0.294 mmol, 0.053 g) using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.118 min and (method 2) = 3.605 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.00 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.60–7.47 (m, 4H), 7.34–7.23 (m, 1H), 7.15 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.14 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 2.45 (t, <i>J</i> = 7.5 Hz, 2H), 1.15 (s, 1H), 1.14–1.07 (m, 1H), 0.38–0.29 (m, 2H), 0.24–0.17 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 583.128; found, 583.1267.</div></div><div id="sec5_1_9_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(pent-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>14</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with 1-pentyne (0.294 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.246 min and (method 2) = 3.718 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.18 (s, 1H), 8.30 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.62–7.51 (m, 4H), 7.35–7.25 (m, 1H), 7.15 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.14 (s, 2H), 3.24–3.07 (m, 4H), 2.44 (t, <i>J</i> = 6.9 Hz, 2H), 1.56 (h, <i>J</i> = 7.2 Hz, 2H), 1.26–1.06 (m, 1H), 0.99 (t, <i>J</i> = 7.4 Hz, 3H), 0.39–0.29 (m, 2H), 0.27–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>29</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 597.1436; found, 597.144.</div></div><div id="sec5_1_9_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(hex-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>15</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with 1-hexyne (0.294 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.511 min and (method 2) = 3.644 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.18 (s, 1H), 8.30 (s, 1H), 7.65 (t, <i>J</i> = 7.9 Hz, 1H), 7.62–7.50 (m, 4H), 7.30 (dd, <i>J</i> = 9.4, 8.6 Hz, 1H), 7.15 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.14 (s, 2H), 3.23–3.10 (m, 3H), 2.46 (t, <i>J</i> = 6.9 Hz, 2H), 1.59–1.37 (m, 4H), 1.20–1.06 (m, 1H), 0.91 (t, <i>J</i> = 7.3 Hz, 3H), 0.38–0.29 (m, 2H), 0.25–0.16 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>29</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 611.1593; found, 611.1572.</div></div><div id="sec5_1_9_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(3-methylbut-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>16</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.235 mmol, 0.15 g) with 3-methyl-1-butyne (0.353 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.274 min and (method 2) = 3.563 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.60–7.50 (m, 4H), 7.30 (t, <i>J</i> = 9.0 Hz, 1H), 7.16 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.14 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 2.84 (hept, <i>J</i> = 6.8 Hz, 1H), 1.22 (d, <i>J</i> = 6.9 Hz, 6H), 1.14 (ddt, <i>J</i> = 9.9, 7.5, 4.8 Hz, 1H), 0.39–0.28 (m, 2H), 0.27–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>29</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 597.1436; found, 597.1424.</div></div><div id="sec5_1_9_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 2-(5-(Cyclopropylmethyl)-3-(3-(3,3-dimethylbut-1-yn-1-yl)-4-fluorophenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>17</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.314 mmol, 0.2 g) with 2-(3,3-dimethylbut-1-yn-1-yl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (0.471 mmol) utilizing the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.679 min and (method 2) = 3.754 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.58 (s, 2H), 7.62–7.53 (m, 1H), 7.50 (dd, <i>J</i> = 6.9, 2.3 Hz, 1H), 7.29 (dd, <i>J</i> = 9.4, 8.7 Hz, 1H), 7.17 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.15 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 1.29 (s, 9H), 1.20–1.07 (m, 1H), 0.39–0.29 (m, 2H), 0.27–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>29</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 611.1593; found, 611.1598.</div></div><div id="sec5_1_9_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(4-hydroxybut-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>18</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.235 mmol, 0.15 g) with potassium (4-((tert-butyldimethylsilyl)oxy)but-1-yn-1-yl)trifluoroborate (0.353 mmol) utilizing the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.211 min and (method 2) = 3.266 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.69–7.60 (m, 2H), 7.58 (s, 2H), 7.54 (ddd, <i>J</i> = 8.7, 5.0, 2.3 Hz, 1H), 7.30 (dd, <i>J</i> = 9.4, 8.7 Hz, 1H), 7.14 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.91 (d, <i>J</i> = 6.0 Hz, 1H), 4.15 (s, 2H), 3.59 (q, <i>J</i> = 6.5 Hz, 2H), 3.17 (d, <i>J</i> = 6.8 Hz, 2H), 2.60 (t, <i>J</i> = 6.8 Hz, 2H), 1.19–1.06 (m, 1H), 0.39–0.28 (m, 2H), 0.26–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 599.1229; found, 599.123.</div></div><div id="sec5_1_9_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(3-methoxyprop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>19</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.235 mmol, 0.15 g) with 2-(3-methoxyprop-1-yn-1-yl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (0.353 mmol) utilizing the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.782 min and (method 2) = 3.465 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.29 (s, 1H), 7.70–7.56 (m, 5H), 7.40–7.31 (m, 1H), 7.15 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.36 (s, 2H), 4.16 (s, 2H), 3.33 (s, 3H), 3.17 (d, <i>J</i> = 6.4 Hz, 2H), 1.32–1.01 (m, 1H), 0.39–0.28 (m, 2H), 0.26–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub> 599.1229; found, 599.1232.</div></div><div id="sec5_1_9_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(4-methylpent-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>20</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.235 mmol, 0.15 g) with isobutylacetylene (0.353 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.536 min and (method 2) = 3.787 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.18 (s, 1H), 8.30 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.61–7.52 (m, 4H), 7.31 (ddd, <i>J</i> = 9.3, 8.0, 1.2 Hz, 1H), 7.15 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.14 (s, 2H), 3.16 (d, <i>J</i> = 6.8 Hz, 2H), 2.36 (d, <i>J</i> = 6.4 Hz, 2H), 1.85 (dp, <i>J</i> = 13.1, 6.6 Hz, 1H), 1.19–1.08 (m, 1H), 0.99 (d, <i>J</i> = 6.7 Hz, 6H), 0.38–0.28 (m, 2H), 0.26–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>29</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 611.1593; found, 611.1608.</div></div><div id="sec5_1_9_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(4-methylpent-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>21</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.235 mmol, 0.15 g) with ethynylcyclopropane (0.353 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.026 min and (method 2) = 3.491 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.65 (t, <i>J</i> = 7.9 Hz, 1H), 7.60–7.49 (m, 4H), 7.28 (t, <i>J</i> = 9.0 Hz, 1H), 7.15 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.14 (s, 2H), 3.16 (d, <i>J</i> = 6.6 Hz, 2H), 1.59 (tt, <i>J</i> = 8.2, 5.0 Hz, 1H), 1.14 (ddt, <i>J</i> = 14.8, 7.7, 3.7 Hz, 1H), 0.98–0.86 (m, 2H), 0.81–0.72 (m, 2H), 0.39–0.28 (m, 2H), 0.31–0.18 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.75, 161.12, 159.28, 156.76, 150.92, 147.40, 147.33, 144.58, 144.45, 132.35, 129.40, 128.89, 128.80, 128.49, 126.17, 123.66, 116.76, 116.40, 116.18, 115.96, 111.92, 111.76, 100.05, 68.25, 28.10, 15.24, 15.08, 10.30, 8.65, 4.47; HRMS (ESI) <i>m</i>/<i>z</i>: (M + Na)<sup>+</sup> calcd for C<sub>29</sub>H<sub>24</sub>F<sub>2</sub>N<sub>4</sub>NaO<sub>4</sub>S<sub>2</sub>, 617.1099; found, 617.1126.</div></div><div id="sec5_1_9_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 2-(3-(3-(Cyclobutylethynyl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>22</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.275 mmol, 0.175 g) with ethynylcyclobutane (0.412 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.392 min and (method 2) = 3.698 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.60–7.51 (m, 4H), 7.30 (t, <i>J</i> = 8.9 Hz, 1H), 7.16 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.15 (s, 2H), 3.38–3.25 (m, 1H), 3.17 (d, <i>J</i> = 6.7 Hz, 2H), 2.38–2.25 (m, 2H), 2.21–2.06 (m, 2H), 2.05–1.81 (m, 2H), 1.14 (ddd, <i>J</i> = 12.7, 7.5, 5.0 Hz, 1H), 0.39–0.28 (m, 2H), 0.31–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + Na)<sup>+</sup> calcd for C<sub>30</sub>H<sub>26</sub>F<sub>2</sub>N<sub>4</sub>NaO<sub>4</sub>S<sub>2</sub>, 631.1256; found, 631.1285.</div></div><div id="sec5_1_9_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 2-(3-(3-(Cyclopentylethynyl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>23</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (3.14 mmol, 2 g) with cyclopentyl acetylene (4.39 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.319 min and (method 2) = 3.772 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.60–7.51 (m, 4H), 7.34–7.24 (m, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.14 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 2.89 (p, <i>J</i> = 7.2 Hz, 1H), 2.04–1.92 (m, 1H), 1.76–1.51 (m, 4H), 1.19–1.08 (m, 1H), 0.39–0.26 (m, 2H), 0.29–0.18 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.15, 161.75, 161.12, 159.31, 150.90, 147.33, 144.61, 144.46, 132.27, 129.56, 129.41, 128.94, 128.85, 128.52, 128.26, 126.17, 123.64, 116.70, 116.39, 116.22, 116.17, 116.00, 112.00, 72.79, 33.33, 30.08, 28.10, 24.64, 15.30, 15.23, 15.09, 10.30, 4.48; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>29</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 623.1593; found, 623.1584.</div></div><div id="sec5_1_9_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((1-fluorocyclopentyl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>24</b>)</h5><div class="NLM_p last">To a mixture of triethylamine trihydrofluoride (0.049 mL, 0.300 mmol) and tetraethylammonium (TEA) (0.021 mL, 0.150 mmol) in DCM (1 mL) at −78 °C was added XtalFluor-M<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> (0.055 g, 0.225 mmol), followed by ethyl 2-(5-(cyclopropylmethyl)-3-(4-fluoro-3-((1-hydroxycyclopentyl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate (prepared from the intermediate <b>VIIf</b> and 1-ethynyl-1-cyclopentanol using the Sonogashira coupling procedure E) (0.1 g, 0.150 mmol). The reaction mixture was allowed to attain room temperature and stirred overnight. The reaction was quenched with aqueous sodium bicarbonate solution and extracted twice using ethyl acetate. The organic layer was concentrated, and the crude product was purified in an Isco flash system eluting with 10–50% ethyl acetate in hexanes over 20 column volumes. The intermediate obtained was subsequently hydrolyzed using general procedure H. LC–MS retention time: (method 1) = 6.706 min and (method 2) = 3.739 min; (sufficient amount of material was not available to run NMR); LCMS <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>28</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub>S, 641.15; found, 641.2.</div></div><div id="sec5_1_9_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((1-hydroxycyclopentyl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>25</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.345 mmol, 0.22 g) with 1-ethynyl-1-cyclopentanol (0.449 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.576 min and (method 2) = 2.651 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.11 (s, 1H), 7.65 (t, <i>J</i> = 7.9 Hz, 1H), 7.62–7.53 (m, 2H), 7.43 (s, 2H), 7.36–7.27 (m, 1H), 7.15 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 5.41 (s, 1H), 4.15 (s, 2H), 3.20–3.13 (m, 2H), 1.97–1.62 (m, 8H), 1.12 (tt, <i>J</i> = 11.6, 4.6 Hz, 1H), 0.38–0.27 (m, 2H), 0.29–0.17 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + Na)<sup>+</sup> calcd for C<sub>31</sub>H<sub>28</sub>F<sub>2</sub>N<sub>4</sub>NaO<sub>5</sub>S<sub>2</sub> 661.1361; found, 661.1376.</div></div><div id="sec5_1_9_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((3-methyloxetan-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>26</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with 3-ethynyl-3-methyloxetane (0.235 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.598 min and (method 2) = 3.496 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.19 (s, 1H), 8.31 (s, 1H), 7.70–7.61 (m, 1H), 7.65–7.56 (m, 4H), 7.38–7.29 (m, 1H), 7.17 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.74 (d, <i>J</i> = 5.5 Hz, 2H), 4.45 (d, <i>J</i> = 5.6 Hz, 2H), 4.16 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 1.64 (s, 3H), 1.22–1.09 (m, 1H), 0.40–0.29 (m, 2H), 0.27–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub>S<sub>2</sub>, 625.1385; found, 625.1388.</div></div><div id="sec5_1_9_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((3-fluorooxetan-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>27</b>)</h5><div class="NLM_p last">To a mixture of triethylamine trihydrofluoride (0.458 mmol, 0.075 mL) and TEA (0.229 mmol, 0.032 mL) in DCM (1 mL) at −78 °C was added XtalFluor-M (0.344 mmol, 0.084 g), followed by ethyl 2-(5-(cyclopropylmethyl)-3-(4-fluoro-3-((3-hydroxyoxetan-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate (prepared from the intermediate <b>VIIf</b> and 3-ethynyloxetan-3-ol using the Sonogashira coupling procedure E) (0.125 g, 0.187 mmol). The reaction mixture was allowed to attain room temperature and stirred overnight. The reaction was quenched with aqueous sodium bicarbonate solution and extracted twice using ethyl acetate. The organic layer was concentrated, and the crude product was purified in an Isco flash system using a 12 g silica column eluting with 20–40% ethyl acetate in hexanes over 20 column volumes. The intermediate obtained was subsequently hydrolyzed using general procedure H. LC–MS retention time: (method 1) = 5.767 min and (method 2) = 3.413 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.19 (s, 1H), 8.32 (s, 1H), 7.78 (dd, <i>J</i> = 6.7, 2.3 Hz, 1H), 7.70 (ddd, <i>J</i> = 8.7, 5.1, 2.3 Hz, 1H), 7.64 (t, <i>J</i> = 7.9 Hz, 1H), 7.59 (s, 2H), 7.41 (t, <i>J</i> = 9.0 Hz, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 5.01–4.94 (m, 1H), 4.94–4.88 (m, 2H), 4.87 (dd, <i>J</i> = 7.9, 1.1 Hz, 1H), 4.17 (s, 2H), 3.18 (t, <i>J</i> = 6.1 Hz, 2H), 1.22–1.07 (m, 1H), 0.40–0.29 (m, 2H), 0.31–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>29</sub>H<sub>24</sub>F<sub>3</sub>N<sub>4</sub>O<sub>5</sub>S<sub>2</sub>, 629.1135; found, 629.1147.</div></div><div id="sec5_1_9_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((3-hydroxyoxetan-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>28</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.314 mmol, 0.2 g) with 1-3-ethynyloxetan-3-ol (0.392 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 4.877 min and (method 2) = 3.231 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (br, 1H), 8.20 (s, 1H), 7.72 (dd, <i>J</i> = 6.9, 2.3 Hz, 1H), 7.69–7.50 (m, 4H), 7.41–7.32 (m, 1H), 7.15 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.71 (s, 1H), 4.84–4.72 (m, 2H), 4.65–4.57 (m, 2H), 4.17 (s, 2H), 3.17 (d, <i>J</i> = 6.5 Hz, 2H), 1.20–1.06 (m, 1H), 0.40–0.29 (m, 2H), 0.26–0.17 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>29</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>6</sub>S<sub>2</sub>, 627.1178; found, 627.1181.</div></div><div id="sec5_1_9_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((tetrahydrofuran-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>29</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with 3-3-ethynyltetrahydrofuran (0.235 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.816 min and (method 2) = 3.478 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.61–7.55 (m, 4H), 7.35–7.28 (m, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.15 (s, 2H), 3.97 (dd, <i>J</i> = 8.1, 7.3 Hz, 1H), 3.88–3.71 (m, 2H), 3.60 (dd, <i>J</i> = 8.1, 6.5 Hz, 1H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 2.34–2.21 (m, 1H), 1.94 (ddt, <i>J</i> = 12.1, 7.8, 6.5 Hz, 1H), 1.14 (dddd, <i>J</i> = 13.1, 11.9, 5.1, 3.5 Hz, 1H), 0.40–0.31 (m, 2H), 0.28–0.16 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub>S<sub>2</sub>, 625.1385; found, 625.1377.</div></div><div id="sec5_1_9_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((3-fluorotetrahydrofuran-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>30</b>)</h5><div class="NLM_p last">To a mixture of triethylamine trihydrofluoride (0.374 mmol, 0.061 mL) and TEA (0.187 mmol, 0.026 mL) in DCM (1 mL) at −78 °C was added XtalFluor-M (0.280 mmol, 0.068 g), followed by ethyl 2-(5-(cyclopropylmethyl)-3-(4-fluoro-3-((3-hydroxytetrahydrofuran-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate (prepared from the intermediate <b>VIIf</b> and 3-ethynyloxetan-3-ol using the Sonogashira coupling procedure E) (0.125 g, 0.187 mmol). The reaction mixture was allowed to attain room temperature and stirred overnight. The reaction was quenched with aqueous sodium bicarbonate solution and extracted twice using ethyl acetate. The organic layer was concentrated, and the crude product was purified in an Isco flash system using a 12 g silica column eluting with 20–40% ethyl acetate in hexanes over 30 column volumes. The intermediate obtained was subsequently hydrolyzed using general procedure H. LC–MS retention time: (method 1) = 5.587 min and (method 2) = 3.489 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.19 (s, 1H), 8.31 (s, 1H), 7.72–7.62 (m, 3H), 7.59 (s, 2H), 7.39 (t, <i>J</i> = 9.3 Hz, 1H), 7.17 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.24–4.13 (m, 3H), 3.97 (dd, <i>J</i> = 8.5, 5.8 Hz, 2H), 3.87 (ddd, <i>J</i> = 32.4, 10.7, 0.5 Hz, 1H), 3.18 (d, <i>J</i> = 7.0 Hz, 2H), 2.68–2.52 (m, 1H), 2.49–2.30 (m, 1H), 1.14 (dddd, <i>J</i> = 13.4, 8.1, 4.9, 1.9 Hz, 1H), 0.41–0.31 (m, 2H), 0.27–0.17 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>26</sub>F<sub>3</sub>N<sub>4</sub>O<sub>5</sub>S<sub>2</sub>, 643.1291; found, 643.1288.</div></div><div id="sec5_1_9_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((3-hydroxytetrahydrofuran-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>31</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.392 mmol, 0.25 g) with 3-ethynyltetrahydrofuran-3-ol (0.471 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 4.929 min and (method 2) = 3.262 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.29 (s, 1H), 7.78–7.52 (m, 5H), 7.34 (t, <i>J</i> = 9.0 Hz, 1H), 7.15 (dd, <i>J</i> = 11.5, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 5.94 (s, 1H), 4.16 (s, 2H), 4.00–3.74 (m, 4H), 3.17 (dd, <i>J</i> = 5.9, 1.8 Hz, 2H), 2.28–2.15 (m, 2H), 1.19–1.06 (m, 1H), 0.37–0.30 (m, 2H), 0.30–0.14 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>6</sub>S<sub>2</sub>, 641.1332; found, 641.1335.</div></div><div id="sec5_1_9_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(piperidin-4-ylethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>32</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.627 mmol, 0.4 g) with <i>tert</i>-butyl 4-ethynylpiperidine-1-carboxylate (0.94 mmol) using the general Sonogashira coupling procedure E and subsequent Boc deprotection with TFA, followed by hydrolysis using general procedure H. LC–MS retention time: (method 1) = 4.466 min and (method 2) = 3.009 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.1 (br, 1H), 8.28 (s, 1H), 7.73–6.96 (m, 8H), 5.73 (s, 1H), 4.12 (s, 2H), 3.79 (s, 2H), 3.18 (dq, <i>J</i> = 16.7, 6.8, 5.5 Hz, 4H), 3.00 (qd, <i>J</i> = 8.8, 3.4 Hz, 4H), 2.40 (d, <i>J</i> = 6.8 Hz, 1H), 1.99 (ddq, <i>J</i> = 11.1, 6.9, 4.4, 3.8 Hz, 2H), 1.74 (ddq, <i>J</i> = 13.5, 9.1, 4.3 Hz, 2H), 1.09 (t, <i>J</i> = 6.7 Hz, 1H), 1.01–0.86 (m, 1H), 0.35–0.13 (m, 4H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub>S<sub>2</sub>, 638.1702; found, 638.1703.</div></div><div id="sec5_1_9_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 2-(3-(3-(Cyanoethynyl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>33</b>)</h5><div class="NLM_p">To a solution of ethyl 2-(5-(cyclopropylmethyl)-3-(4-fluoro-3-(3-hydroxyprop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate (0.5 g, 0.816 mmol, 1 equiv) in DCM (5 mL) was added Dess-Martin periodinane (0.519 g, 1.224 mmol, 1.5 equiv), and the reaction was stirred at room temperature for 2 h. The reaction was extracted with ethyl acetate, and the organic layer was washed with 1 M HCl and brine. The organic layer was concentrated, and the crude product was purified in an Isco flash system using a 12 g silica column eluting with 20–100% ethyl acetate in hexanes over 20 column volumes to obtain 260 mg of the intermediate ethyl 2-(5-(cyclopropylmethyl)-3-(4-fluoro-3-(3-oxoprop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate. LC–MS retention time: (method 1) = 5.721 min; <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> for C<sub>29</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub>S<sub>2</sub>, 611.2.</div><div class="NLM_p">To the above intermediate (0.1 g, 0.164 mmol, 1 equiv) and sodium azide (0.016 g, 0.246 mmol, 1.5 equiv) in ACN (1 mL) was added triflic acid (0.044 mL, 0.491 mmol, 3 equiv) at rt, and the mixture was then stirred overnight. The reaction was extracted with ethyl acetate, and the organic layer was washed with water, bicarbonate solution, and brine. The organic layer was concentrated, and the crude product was purified in an Isco flash system using a 4 g silica column eluting with 20–80% ethyl acetate in hexanes over 20 column volumes to obtain 32 mg of the intermediate ethyl 2-(3-(3-(cyanoethynyl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate. LC–MS retention time: (method 1) = 6.965 min; <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> for C<sub>29</sub>H<sub>24</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub>S<sub>2</sub>, 608.</div><div class="NLM_p last">The above intermediate ethyl 2-(3-(3-(cyanoethynyl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate (0.015 g, 0.025 mmol, 1 equiv) and hydroxytrimethylstannane (8.93 mg, 0.049 mmol, 2 equiv) in dichloroethane (1 mL) were heated in a microwave at 110 °C for 1 h. The solvent was removed by forced air, and the residue was taken up in 1 mL of DMSO. The crude product was purified in prep HPLC. LC–MS retention time: (method 1) = 5.721 min and (method 2) = 3.41 min; (sufficient amount of material was not available to run NMR); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>20</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub>S<sub>2</sub>, 580.0919; found, 580.0935.</div></div><div id="sec5_1_9_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(3,3,3-trifluoroprop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>34</b>)</h5><div class="NLM_p last">A mixture of <b>VIIf</b> (0.157 mmol, 0.1 g), tri(tert-butylphosphonium)tetrafluoroborate (0.016 mmol, 4.55 mg), and allylpalladium chloride dimer (7.84 μmol, 2.84 mg) in dioxane was bubbled with argon, and then tributyl(3,3,3-trifluoroprop-1-yn-1-yl)stannane (0.196 mmol, 0.083 g) was added. The vial was capped and stirred at 80 °C for 4 h. LC–MS showed only 15% conversion. [NOTE: a further attempt to modify the conditions did not improve the yield and the reaction did not reproduce well under the same conditions]. The reaction was diluted with DCM and stirred sequentially with KF and palladium scavenger. The product obtained after filtration was directly hydrolyzed using general procedure H to obtain a minute amount of the pure product. LC–MS retention time: (method 1) = 5.543 min and (method 2) = 3.484 min; (sufficient amount of material was not available to run NMR); LCMS <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>20</sub>F<sub>5</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 623.08; found: 623.1.</div></div><div id="sec5_1_9_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 2-(5-(Cyclopropylmethyl)-3-(3-(3,3-difluoroprop-1-yn-1-yl)-4-fluorophenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>35</b>)</h5><div class="NLM_p last">To a solution of ethyl 2-(5-(cyclopropylmethyl)-3-(4-fluoro-3-(3-oxoprop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate (0.115 mmol, 0.07 g) in DCM (3 mL) was added deoxofluor (0.229 mmol, l0.042 mL), and the mixture was stirred overnight at room temperature. The reaction was quenched with aqueous sodium bicarbonate solution and extracted twice using ethyl acetate. The organic layer was concentrated, and the crude product was purified in an Isco flash system using a 6 g silica column eluting with 20–40% ethyl acetate in hexanes over 20 column volumes. The intermediate obtained was subsequently hydrolyzed using general procedure H. LC–MS retention time: (method 1) = 5.896 min and (method 2) = 3.434 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.85 (dd, <i>J</i> = 6.7, 2.3 Hz, 1H), 7.74 (ddd, <i>J</i> = 8.7, 5.1, 2.3 Hz, 1H), 7.64 (t, <i>J</i> = 7.9 Hz, 1H), 7.57 (s, 2H), 7.44 (t, <i>J</i> = 9.1 Hz, 1H), 7.18–7.10 (m, 1H), 7.07–7.00 (m, 1H), 4.18 (s, 2H), 3.17 (d, <i>J</i> = 7.0 Hz, 2H), 1.21–1.06 (m, 1H), 0.39–0.28 (m, 2H), 0.27–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>21</sub>F<sub>4</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub> 605.0935; found, 605.0949.</div></div><div id="sec5_1_9_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(3-fluorobut-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>36</b>)</h5><div class="NLM_p last">Following the general procedure E, the reaction of <b>VIIf</b> (0.275 g, 0.431 mmol) with 2-trimethylsilyloxy-3-butyne (0.604 mmol, 0.105 mL) and subsequent deprotection of the TMS group with 2 equiv of K<sub>2</sub>CO<sub>3</sub> in methanol provided the intermediate ethyl 2-(5-(cyclopropylmethyl)-3-(4-fluoro-3-(3-hydroxybut-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylate. To a mixture of the above intermediate (0.207 mmol, 0.13 g), triethylamine trihydrofluoride (0.415 mmol, 0.068 mL), and TEA (0.207 mmol, 0.029 mL) in DCM (1 mL) was added XtalFluor-M (0.311 mmol, 0.076 g) at −78 °C, and the reaction mixture was stirred overnight at room temperature. The reaction was quenched with aqueous sodium bicarbonate solution and extracted twice using ethyl acetate. The organic layer was concentrated, and the crude product was purified in an Isco flash system using a 6 g silica column eluting with 20–40% ethyl acetate in hexanes over 20 column volumes. The intermediate obtained was subsequently hydrolyzed using general procedure H to afford a white solid. LC–MS retention time: (method 1) = 5.811 min and (method 2) = 3.519 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.30–13.07 (m, 1H), 8.31 (s, 1H), 7.71–7.62 (m, 3H), 7.58 (s, 2H), 7.38 (t, <i>J</i> = 8.9 Hz, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 5.83–5.53 (m, 1H), 4.16 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 1.62 (dd, <i>J</i> = 23.0, 6.6 Hz, 3H), 1.26–1.04 (m, 1H), 0.41–0.31 (m, 2H), 0.27–0.18 (m, 2H).; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>24</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 601.1186; found, 601.1207.</div></div><div id="sec5_1_9_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 2-(5-(Cyclopropylmethyl)-3-(3-(3-cyclopropylprop-1-yn-1-yl)-4-fluorophenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>37</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with prop-2-yn-1-ylcyclopropane (0.294 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.013 min and (method 2) = 3.671 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.61–7.51 (m, 4H), 7.31 (dd, <i>J</i> = 9.4, 8.5 Hz, 1H), 7.15 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.15 (s, 2H), 3.16 (d, <i>J</i> = 6.9 Hz, 2H), 2.53 (d, <i>J</i> = 5.9 Hz, 2H), 1.13 (ddt, <i>J</i> = 9.7, 7.8, 2.9 Hz, 1H), 1.07–0.94 (m, 1H), 0.54–0.42 (m, 2H), 0.39–0.27 (m, 2H), 0.30–0.18 (m, 4H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 609.1436; found, 609.1449.</div></div><div id="sec5_1_9_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 2-(3-(3-(3-Cyclopentylprop-1-yn-1-yl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>38</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with 1-3-cyclopentyl-1-propyne (0.294 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 8.045 min and (method 2) = 3.87 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.65 (t, <i>J</i> = 7.9 Hz, 1H), 7.58 (s, 2H), 7.61–7.51 (m, 2H), 7.30 (dd, <i>J</i> = 9.4, 8.6 Hz, 1H), 7.14 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 5.75 (s, 2H), 4.15 (s, 2H), 3.16 (d, <i>J</i> = 6.8 Hz, 2H), 2.47 (d, <i>J</i> = 6.7 Hz, 2H), 2.15–2.03 (m, 1H), 1.84–1.71 (m, 2H), 1.69–1.46 (m, 3H), 1.39–1.23 (m, 2H), 1.20–1.06 (m, 1H), 0.38–0.28 (m, 2H), 0.30–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>31</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 637.1749; found, 637.1761.</div></div><div id="sec5_1_9_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(3-(pyrrolidin-1-yl)prop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>39</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with 1-1-(prop-2-yn-1-yl)pyrrolidine HCl (0.235 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 4.49 min and (method 2) = 2.999 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.20 (s, 1H), 8.32 (s, 1H), 7.76–7.58 (m, 5H), 7.40 (t, <i>J</i> = 9.0 Hz, 1H), 7.14 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.40 (s, 2H), 4.16 (s, 2H), 3.28 (s, 6H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 1.96 (s, 4H), 1.27–1.04 (m, 1H), 0.39–0.28 (m, 2H), 0.27–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub>S<sub>2</sub>, 638.1702; found, 638.1692.</div></div><div id="sec5_1_9_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(3-(tetrahydrofuran-2-yl)prop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>40</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.196 mmol, 0.125 g) with 2-(prop-2-yn-1-yl)tetrahydrofuran (0.235 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.98 min and (method 2) = 3.412 min; (sufficient amount of material was not available to run NMR); HRMS (ESI) <i>m</i>/<i>z</i>: (M + Na)<sup>+</sup> calcd for C<sub>31</sub>H<sub>28</sub>F<sub>2</sub>N<sub>4</sub>NaO<sub>5</sub>S<sub>2</sub>, 661.1361; found, 661.137.</div></div><div id="sec5_1_9_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(3-morpholinoprop-1-yn-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>41</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> (0.627 mmol, 0.4 g) with 4-(prop-2-yn-1-yl)morpholine (1.882 mmol) using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 4.391 min and (method 2) = 3.008 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.84 (s, 1H), 8.18 (s, 1H), 7.65 (t, <i>J</i> = 7.9 Hz, 2H), 7.60 (s, 2H), 7.57–7.47 (m, 1H), 7.37 (t, <i>J</i> = 9.0 Hz, 1H), 7.13–7.01 (m, 2H), 3.83 (s, 2H), 3.68–3.38 (m, 4H), 2.68–2.32 (m, 6H), 0.98–0.91 (m, 1H), 0.43–0.40 (m, 2H), 0.15–0.11 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>F<sub>2</sub>N<sub>5</sub>O<sub>5</sub>S<sub>2</sub> 654.1651; found, 654.1672.</div></div><div id="sec5_1_9_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(thiophen-2-ylethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>42</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with either 4,4,5,5-tetramethyl-2-(thiophen-2-ylethynyl)-1,3,2-dioxaborolane following general Suzuki procedure G or 2-ethynylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.472 min and (method 2) = 3.687 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.18 (s, 1H), 8.31 (s, 1H), 7.80–7.71 (m, 2H), 7.70–7.60 (m, 2H), 7.58 (s, 2H), 7.49 (dd, <i>J</i> = 3.7, 1.2 Hz, 1H), 7.39 (dd, <i>J</i> = 9.4, 8.7 Hz, 1H), 7.21–7.13 (m, 2H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.18 (s, 2H), 3.18 (d, <i>J</i> = 7.0 Hz, 2H), 1.20–1.09 (m, 1H), 0.40–0.29 (m, 2H), 0.27–0.19 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 166.99, 161.75, 161.10, 160.26, 150.78, 144.62, 144.48, 133.54, 131.98, 130.07, 129.92, 129.55, 129.41, 128.63, 128.50, 127.97, 126.20, 123.71, 120.96, 116.89, 116.44, 116.24, 110.74, 85.57, 28.11, 15.16, 10.31, 4.47; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>23</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 637.0844; found, 637.0856.</div></div><div id="sec5_1_9_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(thiophen-2-ylethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>43</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with 2-ethynyl-5-methylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H (a detailed scale-up procedure is provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">Supporting Information</a>). LC–MS retention time: (method 1) = 6.133 min and (method 2) = 3.694 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (s, 1H), 8.29 (s, 1H), 7.71 (dd, <i>J</i> = 6.9, 2.3 Hz, 1H), 7.66–7.54 (m, 4H), 7.35 (dd, <i>J</i> = 9.4, 8.7 Hz, 1H), 7.26 (dd, <i>J</i> = 3.6, 0.5 Hz, 1H), 7.14 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.03 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.83 (dq, <i>J</i> = 3.6, 1.0 Hz, 1H), 4.15 (s, 2H), 3.15 (d, <i>J</i> = 6.9 Hz, 2H), 2.47–2.45 (m, 3H), 1.20–1.06 (m, 1H), 0.38–0.29 (m, 2H), 0.24–0.15 (m, 2H); 13C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.58, 160.94, 159.18, 150.73, 147.20, 147.12, 144.47, 144.44, 143.30, 133.59, 131.80, 129.81, 129.73, 129.50, 129.35, 128.51, 128.48, 128.39, 126.25, 125.90, 123.59, 123.56, 118.47, 116.75, 116.28, 116.23, 116.07, 116.02, 110.99, 84.86, 27.99, 14.90, 10.15, 4.34; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 651.1001; found, 651.1027.</div></div><div id="sec5_1_9_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((5-methylfuran-2-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>44</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIb</b> with 2-bromo-5-methylfuran using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.202 min and (method 2) = 3.593 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.23 (s, 1H), 8.30 (d, <i>J</i> = 3.2 Hz, 1H), 7.75 (dd, <i>J</i> = 6.8, 2.3 Hz, 1H), 7.70–7.58 (m, 4H), 7.39 (t, <i>J</i> = 9.0 Hz, 1H), 7.16 (d, <i>J</i> = 11.3 Hz, 1H), 7.05 (d, <i>J</i> = 8.1 Hz, 1H), 6.88 (d, <i>J</i> = 3.3 Hz, 1H), 6.29–6.23 (m, 1H), 4.17 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 2.33 (s, 3H), 1.14 (t, <i>J</i> = 7.6 Hz, 1H), 0.34 (dd, <i>J</i> = 8.0, 1.8 Hz, 2H), 0.23 (t, <i>J</i> = 4.9 Hz, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.00, 161.74, 161.10, 159.28, 156.76, 154.84, 150.77, 147.29, 144.61, 144.47, 133.57, 131.70, 130.13, 128.67, 128.49, 126.22, 123.73, 118.45, 116.89, 116.48, 116.43, 116.28, 116.22, 110.55, 110.40, 107.99, 85.97, 28.11, 28.02, 15.20, 15.19, 13.51, 10.30, 4.47; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub>S<sub>2</sub>, 635.1229; found, 635.124.</div></div><div id="sec5_1_9_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> 2-(3-(3-((5-(<i>tert</i>-Butyl)thiophen-2-yl)ethynyl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>45</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIb</b> with 2-bromo-5-(<i>tert</i>-butyl)thiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 7.143 min and (method 2) = 3.959 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.32 (s, 1H), 7.73 (dd, <i>J</i> = 6.9, 2.3 Hz, 1H), 7.70–7.58 (m, 2H), 7.57 (s, 2H), 7.42–7.33 (m, 1H), 7.29 (d, <i>J</i> = 3.8 Hz, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.92 (d, <i>J</i> = 3.7 Hz, 1H), 4.17 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 1.36 (s, 9H), 1.25–1.07 (m, 1H), 0.40–0.29 (m, 2H), 0.27–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>34</sub>H<sub>31</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub>S<sub>3</sub>, 693.147; found, 693.1484.</div></div><div id="sec5_1_9_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((2-methylthiazol-5-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>46</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIb</b> with 5-bromo-2-methylthiazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.093 min and (method 2) = 3.556 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.32 (s, 1H), 8.01 (s, 1H), 7.77 (dd, <i>J</i> = 6.8, 2.3 Hz, 1H), 7.70–7.61 (m, 2H), 7.58 (s, 2H), 7.40 (t, <i>J</i> = 9.0 Hz, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.18 (s, 2H), 3.18 (d, <i>J</i> = 6.8 Hz, 2H), 2.70 (s, 3H), 1.14 (s, 1H), 0.40–0.29 (m, 2H), 0.31–0.19 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.13, 161.75, 161.09, 160.28, 159.28, 156.76, 150.71, 147.44, 147.30, 144.63, 144.48, 132.09, 129.55, 129.40, 128.67, 128.50, 126.23, 123.74, 116.91, 116.52, 116.38, 116.31, 116.23, 110.35, 87.96, 84.71, 28.10, 19.00, 15.20, 10.31, 4.48; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>24</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub>S<sub>3</sub>, 652.0953; found, 652.097.</div></div><div id="sec5_1_9_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((5-methylthiazol-2-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>47</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIb</b> with 2-bromo-5-methylthiazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.063 min and (method 2) = 3.595 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.32 (s, 1H), 7.87 (dd, <i>J</i> = 6.8, 2.3 Hz, 1H), 7.73–7.62 (m, 3H), 7.58 (s, 2H), 7.43 (t, <i>J</i> = 9.0 Hz, 1H), 7.16 (d, <i>J</i> = 11.2 Hz, 1H), 7.06 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.19 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.52 (s, 3H), 1.14 (dh, <i>J</i> = 13.0, 6.6 Hz, 1H), 0.34 (dt, <i>J</i> = 8.3, 2.9 Hz, 2H), 0.23 (t, <i>J</i> = 4.9 Hz, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.16, 161.75, 161.09, 160.63, 156.75, 150.65, 147.35, 144.62, 144.57, 144.47, 142.38, 137.77, 132.38, 131.25, 131.16, 129.54, 128.81, 128.78, 128.49, 126.25, 123.75, 116.97, 116.63, 116.44, 116.23, 109.67, 109.51, 87.66, 85.43, 28.11, 27.99, 15.31, 15.28, 15.24, 15.21, 11.66, 10.31, 4.47; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>24</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub>S<sub>3</sub>, 652.0953; found, 652.097.</div></div><div id="sec5_1_9_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((1-methyl-1<i>H</i>-imidazole-5-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>48</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with 5-ethynyl-1-methyl-1<i>H</i>-imidazole using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 4.583 min and (method 2) = 2.868 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.32 (s, 1H), 7.88 (s, 1H), 7.72 (dd, <i>J</i> = 6.8, 2.3 Hz, 1H), 7.70–7.62 (m, 2H), 7.59 (s, 2H), 7.45–7.36 (m, 2H), 7.18 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.18 (s, 2H), 3.69 (d, <i>J</i> = 0.5 Hz, 3H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 1.22–1.07 (m, 1H), 0.40–0.29 (m, 2H), 0.27–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>25</sub>F<sub>2</sub>N<sub>6</sub>O<sub>4</sub>S<sub>2</sub>, 635.1344; found, 635.1345.</div></div><div id="sec5_1_9_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-((1-methyl-1<i>H</i>-pyrazol-4-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>49</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with 4-ethynyl-1-methyl-1<i>H</i>-pyrazole using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.607 min and (method 2) = 2.612 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.20 (s, 1H), 8.26 (s, 1H), 8.11 (d, <i>J</i> = 0.6 Hz, 1H), 7.72 (d, <i>J</i> = 0.7 Hz, 1H), 7.70–7.62 (m, 2H), 7.62–7.54 (m, 3H), 7.35 (dd, <i>J</i> = 9.4, 8.6 Hz, 1H), 7.17 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.16 (s, 2H), 3.86 (s, 3H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 1.30–1.06 (m, 1H), 0.39–0.30 (m, 2H), 0.33–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>25</sub>F<sub>2</sub>N<sub>6</sub>O<sub>4</sub>S<sub>2</sub>, 635.1344; found, 635.1345.</div></div><div id="sec5_1_9_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-(prop-1-yn-1-yl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>50</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIe</b> (0.202 mmol, 0.125 g) with potassium trifluoro(prop-1-yn-1-yl)borate (0.303 mmol) using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.653 min and (method 2) = 3.535 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.27 (s, 1H), 7.65 (t, <i>J</i> = 7.9 Hz, 1H), 7.59–7.52 (m, 3H), 7.52–7.45 (m, 1H), 7.42–7.32 (m, 2H), 7.14 (dd, <i>J</i> = 11.5, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.15 (s, 2H), 3.20–3.13 (m, 2H), 2.04 (s, 3H), 1.12 (dd, <i>J</i> = 12.2, 7.1 Hz, 1H), 0.38–0.28 (m, 2H), 0.30–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>24</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 551.1218; found, 551.1208.</div></div><div id="sec5_1_9_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> 2-(3-(3-(Cyclopentylethynyl)phenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>51</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIe</b> with cyclopentyl acetylene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.386 min and (method 2) = 3.68 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.18 (s, 1H), 8.31 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.58 (s, 2H), 7.55–7.47 (m, 2H), 7.41–7.32 (m, 2H), 7.16 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.05 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.15 (s, 2H), 3.17 (d, <i>J</i> = 6.8 Hz, 2H), 2.90–2.78 (m, 1H), 2.08–1.87 (m, 2H), 1.78–1.65 (m, 2H), 1.69–1.50 (m, 4H), 1.19–1.05 (m, 1H), 0.39–0.28 (m, 2H), 0.31–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 605.1687; found, 605.1707.</div></div><div id="sec5_1_9_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>52</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIe</b> with 2-ethynyl-5-methylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time = 6.402 min (M + H)<sup>+</sup> = 632. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.32 (d, <i>J</i> = 0.9 Hz, 1H), 7.71–7.62 (m, 2H), 7.58 (d, <i>J</i> = 6.6 Hz, 3H), 7.52 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.44 (t, <i>J</i> = 7.7 Hz, 1H), 7.24 (d, <i>J</i> = 3.6 Hz, 1H), 7.16 (d, <i>J</i> = 11.3 Hz, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.5 Hz, 1H), 6.83 (dt, <i>J</i> = 3.5, 1.2 Hz, 1H), 4.18 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.47 (d, <i>J</i> = 1.1 Hz, 3H), 1.21–1.09 (m, 1H), 0.35 (dt, <i>J</i> = 8.1, 2.8 Hz, 2H), 0.31–0.19 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.70, 161.11, 159.24, 156.73, 151.55, 147.42, 147.34, 144.53, 144.45, 142.72, 133.19, 132.20, 131.08, 129.80, 129.50, 129.36, 129.26, 128.46, 127.59, 126.24, 126.11, 123.67, 123.64, 122.51, 119.11, 116.86, 116.36, 116.14, 91.67, 83.60, 28.10, 15.01, 10.27, 4.45. HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>26</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 633.1095; found, 633.1086.</div></div><div id="sec5_1_9_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((3-methylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>53</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-3-methylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.488 min and (method 2) = 3.679 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.31 (s, 1H), 7.71–7.62 (m, 2H), 7.65–7.58 (m, 1H), 7.58–7.51 (m, 4H), 7.46 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.17 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.08 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 7.02 (dd, <i>J</i> = 5.1, 0.5 Hz, 1H), 4.19 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.31 (s, 3H), 1.20–1.07 (m, 1H), 0.39–0.29 (m, 2H), 0.31–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>26</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 633.1095; found, 633.1099.</div></div><div id="sec5_1_9_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-3-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>54</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 3-bromo-2-methylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.516 min and (method 2) = 3.691 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.71–7.61 (m, 3H), 7.58 (s, 2H), 7.58 (dt, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.50 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.44 (td, <i>J</i> = 7.6, 0.6 Hz, 1H), 7.18 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 6.95 (p, <i>J</i> = 1.1 Hz, 1H), 4.18 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.46 (d, <i>J</i> = 1.1 Hz, 3H), 1.22–1.08 (m, 1H), 0.40–0.29 (m, 2H), 0.27–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>26</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 633.1095; found, 633.1099.</div></div><div id="sec5_1_9_54" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-isopropylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>55</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-isopropylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 7.071 min and (method 2) = 3.835 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.31 (s, 1H), 7.71–7.62 (m, 2H), 7.62–7.55 (m, 1H), 7.56 (s, 2H), 7.51 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.45 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.25 (d, <i>J</i> = 3.6 Hz, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 6.88 (dd, <i>J</i> = 3.7, 1.0 Hz, 1H), 4.18 (s, 2H), 3.25–3.10 (m, 3H), 1.29 (d, <i>J</i> = 6.8 Hz, 6H), 1.21–1.09 (m, 1H), 0.40–0.29 (m, 2H), 0.31–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>33</sub>H<sub>30</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 661.1408; found, 661.1427.</div></div><div id="sec5_1_9_55" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 2-(5-(Cyclopropylmethyl)-3-(3-((5-cyclopropylthiophen-2-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>56</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-cyclopropylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.769 min and (method 2) = 3.771 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17–13.12 (m, 1H), 8.31 (s, 1H), 7.70–7.62 (m, 2H), 7.61–7.53 (m, 1H), 7.56 (s, 2H), 7.51 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.44 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.22 (d, <i>J</i> = 3.7 Hz, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.82 (dd, <i>J</i> = 3.7, 0.7 Hz, 1H), 4.18 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.17 (ttd, <i>J</i> = 8.3, 4.9, 0.7 Hz, 1H), 1.20–1.08 (m, 1H), 1.11–0.98 (m, 2H), 0.79–0.69 (m, 2H), 0.39–0.29 (m, 2H), 0.31–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>33</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 659.1251; found, 659.127.</div></div><div id="sec5_1_9_56" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> 2-(3-(3-((5-Cyclobutylthiophen-2-yl)ethynyl)phenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>57</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-cyclobutylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 7.119 min and (method 2) = 3.893 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.70–7.61 (m, 2H), 7.61–7.55 (m, 3H), 7.52 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.48–7.40 (m, 1H), 7.26 (d, <i>J</i> = 3.6 Hz, 1H), 7.16 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.88 (dd, <i>J</i> = 3.7, 0.9 Hz, 1H), 4.18 (s, 2H), 3.79–3.65 (m, 1H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.45–2.32 (m, 2H), 2.19–2.04 (m, 2H), 2.08–1.88 (m, 1H), 1.91–1.78 (m, 1H), 1.21–1.09 (m, 1H), 0.41–0.29 (m, 2H), 0.31–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>34</sub>H<sub>30</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 673.1408; found, 673.1391.</div></div><div id="sec5_1_9_57" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> 2-(3-(3-((5-Chlorothiophen-2-yl)ethynyl)phenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>58</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-chlorothiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.776 min and (method 2) = 3.844 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.31 (s, 1H), 7.74–7.52 (m, 6H), 7.46 (td, <i>J</i> = 7.8, 0.6 Hz, 1H), 7.34 (d, <i>J</i> = 3.9 Hz, 1H), 7.21–7.12 (m, 2H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.18 (s, 2H), 3.22–3.14 (m, 2H), 1.29–0.98 (m, 1H), 0.40–0.27 (m, 2H), 0.30–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>23</sub>ClFN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 653.0548; found, 653.0566.</div></div><div id="sec5_1_9_58" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> 2-(5-(Cyclopropylmethyl)-3-(3-((5-(difluoromethyl)thiophen-2-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>59</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-(difluoromethyl)thiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.315 min and (method 2) = 3.682 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.31 (s, 1H), 7.75 (td, <i>J</i> = 1.7, 0.6 Hz, 1H), 7.70–7.58 (m, 2H), 7.58 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.56 (s, 2H), 7.52–7.42 (m, 3H), 7.22–7.13 (m, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.19 (s, 2H), 3.19 (d, <i>J</i> = 6.9 Hz, 2H), 1.22–1.08 (m, 1H), 0.40–0.27 (m, 2H), 0.30–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>24</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 669.0906; found, 669.0909.</div></div><div id="sec5_1_9_59" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-(trifluoromethyl)thiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (60)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-(trifluoromethyl)thiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.897 min and (method 2) = 3.837 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.76 (dp, <i>J</i> = 3.4, 1.2 Hz, 2H), 7.70–7.61 (m, 2H), 7.64–7.57 (m, 1H), 7.56 (s, 2H), 7.54 (dq, <i>J</i> = 3.9, 1.3 Hz, 1H), 7.49 (td, <i>J</i> = 7.8, 0.6 Hz, 1H), 7.17 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.19 (s, 2H), 3.19 (d, <i>J</i> = 6.9 Hz, 2H), 1.21–1.09 (m, 1H), 0.40–0.29 (m, 2H), 0.32–0.20 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>23</sub>F<sub>4</sub>N<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 687.0812; found, 687.0807.</div></div><div id="sec5_1_9_60" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylfuran-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>61</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-methylfuran using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.261 min and (method 2) = 3.591 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.32 (s, 1H), 7.70 (td, <i>J</i> = 1.7, 0.6 Hz, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.62–7.56 (m, 1H), 7.57 (s, 2H), 7.53 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.46 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.81 (dd, <i>J</i> = 3.3, 0.6 Hz, 1H), 6.23 (dq, <i>J</i> = 3.1, 1.0 Hz, 1H), 4.18 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.32 (dd, <i>J</i> = 1.0, 0.5 Hz, 3H), 1.22–1.07 (m, 1H), 0.39–0.29 (m, 2H), 0.31–0.19 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.76, 161.16, 154.33, 151.53, 147.46, 144.59, 144.47, 134.00, 132.31, 131.05, 129.65, 129.39, 128.49, 127.88, 126.20, 123.73, 121.98, 117.70, 116.92, 116.41, 116.20, 107.85, 92.48, 80.50, 28.12, 15.34, 15.28, 15.19, 13.50, 10.30, 4.48; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>26</sub>FN<sub>4</sub>O<sub>5</sub>S<sub>2</sub>, 617.1323; found, 617.1341.</div></div><div id="sec5_1_9_61" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methyloxazol-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>62</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-methyloxazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.764 min and (method 2) = 3.499 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.32 (s, 1H), 7.85 (td, <i>J</i> = 1.7, 0.6 Hz, 1H), 7.72–7.61 (m, 3H), 7.56 (s, 2H), 7.51 (td, <i>J</i> = 7.8, 0.6 Hz, 1H), 7.15 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.10–7.03 (m, 2H), 4.20 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.37 (d, <i>J</i> = 1.2 Hz, 3H), 1.20–1.07 (m, 1H), 0.39–0.29 (m, 2H), 0.27–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>25</sub>FN<sub>5</sub>O<sub>5</sub>S<sub>2</sub>, 618.1276; found, 618.1261.</div></div><div id="sec5_1_9_62" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiazol-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>63</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-5-methylthiazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.875 min and (method 2) = 3.491 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.29 (s, 1H), 7.81 (td, <i>J</i> = 1.7, 0.6 Hz, 1H), 7.70–7.60 (m, 4H), 7.56 (s, 2H), 7.50 (td, <i>J</i> = 7.8, 0.6 Hz, 1H), 7.19–7.12 (m, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.20 (s, 2H), 3.28 (d, <i>J</i> = 10.6 Hz, 2H), 3.21–3.14 (m, 2H), 1.17–1.09 (m, 1H), 0.39–0.30 (m, 2H), 0.27–0.19 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.75, 161.14, 151.39, 147.36, 145.12, 144.61, 144.47, 142.14, 137.19, 132.41, 131.77, 130.37, 129.50, 128.78, 128.49, 126.22, 123.73, 121.04, 116.97, 116.41, 92.09, 83.00, 28.10, 15.19, 11.64, 10.31, 4.48; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>25</sub>FN<sub>5</sub>O<sub>4</sub>S<sub>3</sub>, 634.1047; found, 634.1056.</div></div><div id="sec5_1_9_63" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((2-methylthiazol-5-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>64</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 5-bromo-2-methylthiazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.804 min and (method 2) = 3.468 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.31 (s, 1H), 7.96 (s, 1H), 7.72 (d, <i>J</i> = 1.7 Hz, 1H), 7.73–7.59 (m, 2H), 7.59–7.52 (m, 3H), 7.47 (t, <i>J</i> = 7.8 Hz, 1H), 7.16 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.18 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.69 (s, 3H), 2.07 (s, 1H), 1.20–1.07 (m, 1H), 0.40–0.27 (m, 2H), 0.30–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>25</sub>FN<sub>5</sub>O<sub>4</sub>S<sub>3</sub>, 634.1047; found, 634.1046.</div></div><div id="sec5_1_9_64" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> 2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((2-methylthiazol-4-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>65</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 4-bromo-2-methylthiazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.741 min and (method 2) = 3.437 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.32 (s, 1H), 7.91 (s, 1H), 7.73 (t, <i>J</i> = 1.7 Hz, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.62–7.53 (m, 4H), 7.47 (t, <i>J</i> = 7.7 Hz, 1H), 7.16 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.19 (s, 2H), 3.18 (d, <i>J</i> = 6.9 Hz, 2H), 2.68 (s, 3H), 1.23–1.03 (m, 1H), 0.43–0.30 (m, 2H), 0.30–0.16 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>25</sub>FN<sub>5</sub>O<sub>4</sub>S<sub>3</sub>, 634.1047; found, 634.1041.</div></div><div id="sec5_1_9_65" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> 2-(5-(Cyclopropylmethyl)-3-(3-((3,5-dimethylthiophen-2-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>66</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 2-bromo-3,5-dimethylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.749 min and (method 2) = 3.834 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.31 (s, 1H), 7.70–7.63 (m, 2H), 7.60 (dt, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.56 (s, 2H), 7.51 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.45 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.17 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.72 (dt, <i>J</i> = 1.4, 0.7 Hz, 1H), 4.18 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 2.42 (d, <i>J</i> = 1.1 Hz, 3H), 2.23 (s, 3H), 1.19–1.07 (m, 1H), 0.39–0.31 (m, 2H), 0.26–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 647.1251; found, 647.1229.</div></div><div id="sec5_1_9_66" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> 2-(5-(Cyclopropylmethyl)-3-(3-((2,5-dimethylthiophen-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>67</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIa</b> with 3-bromo-2,5-dimethylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.787 min and (method 2) = 3.777 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.31 (s, 1H), 7.67 (t, <i>J</i> = 8.0 Hz, 1H), 7.64–7.57 (m, 2H), 7.58 (s, 2H), 7.51 (dt, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.45 (td, <i>J</i> = 7.6, 0.7 Hz, 1H), 7.18 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.08 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.77 (q, <i>J</i> = 1.2 Hz, 1H), 4.18 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 2.44 (d, <i>J</i> = 0.7 Hz, 3H), 2.38 (t, <i>J</i> = 0.9 Hz, 3H), 1.20–1.09 (m, 1H), 0.39–0.28 (m, 2H), 0.27–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 647.1251; found, 647.124.</div></div><div id="sec5_1_9_67" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> (<i>E</i>)-2-(5-(Cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-(2-(5-methylthiophen-2-yl)vinyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>68</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIe</b> with (E)-4,4,5,5-tetramethyl-2-(2-(5-methylthiophen-2-yl)vinyl)-1,3,2-dioxaborolane using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.619 min and (method 2) = 3.753 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (s, 1H), 8.30 (s, 1H), 7.70–7.62 (m, 2H), 7.58–7.51 (m, 3H), 7.43 (dt, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.38 (t, <i>J</i> = 7.6 Hz, 1H), 7.23 (dd, <i>J</i> = 16.2, 0.7 Hz, 1H), 7.17 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.08 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.99 (d, <i>J</i> = 3.5 Hz, 1H), 6.83–6.72 (m, 2H), 4.19 (s, 2H), 3.19 (d, <i>J</i> = 6.9 Hz, 2H), 2.45 (d, <i>J</i> = 1.1 Hz, 3H), 1.21–1.08 (m, 1H), 0.40–0.29 (m, 2H), 0.32–0.19 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.79, 161.24, 152.52, 144.46, 144.42, 140.00, 139.28, 137.16, 132.14, 129.15, 128.49, 127.52, 126.57, 126.40, 126.34, 126.11, 125.90, 124.93, 123.68, 122.90, 116.83, 116.37, 116.16, 28.13, 15.31, 10.35, 4.49; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 635.1251; found, 635.1258.</div></div><div id="sec5_1_9_68" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> (<i>E</i>)-2-(3-(3-(2-Cyclopentylvinyl)-4-fluorophenyl)-5-(cyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>69</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with (<i>E</i>)-(2-cyclopentylvinyl)boronic acid using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.444 min and (method 2) = 3.792 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.18 (s, 1H), 8.30 (s, 1H), 7.67 (t, <i>J</i> = 7.9 Hz, 1H), 7.58 (s, 2H), 7.60–7.54 (m, 1H), 7.49 (ddd, <i>J</i> = 8.6, 5.0, 2.3 Hz, 1H), 7.27–7.15 (m, 2H), 7.07 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.43 (dd, <i>J</i> = 16.0, 1.1 Hz, 1H), 6.10 (dd, <i>J</i> = 16.0, 8.0 Hz, 1H), 4.16 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 2.63–2.52 (m, 1H), 1.85–1.49 (m, 4H), 1.33 (ddt, <i>J</i> = 14.1, 11.9, 5.0 Hz, 2H), 1.22–1.07 (m, 1H), 0.39–0.28 (m, 2H), 0.31–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>31</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 625.1749; found, 625.1744.</div></div><div id="sec5_1_9_69" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> (<i>E</i>)-2-(5-(Cyclopropylmethyl)-3-(3-(2-cyclopropylvinyl)-4-fluorophenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>70</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with (<i>E</i>)-2-(2-cyclopropylvinyl)-4,4,5,5-tetramethyl-1,3,2-dioxaborolane using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.229 min and (method 2) = 3.694 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.59 (s, 2H), 7.53 (dd, <i>J</i> = 7.4, 2.3 Hz, 1H), 7.44 (ddd, <i>J</i> = 8.6, 5.0, 2.2 Hz, 1H), 7.25–7.12 (m, 2H), 7.05 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 6.53 (d, <i>J</i> = 15.9 Hz, 1H), 5.66 (dd, <i>J</i> = 15.9, 9.4 Hz, 1H), 4.15 (s, 2H), 3.16 (d, <i>J</i> = 6.7 Hz, 2H), 1.61 (dddd, <i>J</i> = 12.8, 9.4, 8.0, 4.7 Hz, 1H), 1.20–1.08 (m, 1H), 0.87–0.75 (m, 2H), 0.56–0.48 (m, 2H), 0.38–0.28 (m, 2H), 0.30–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>29</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 597.1436; found, 597.1425.</div></div><div id="sec5_1_9_70" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> (<i>E</i>)-2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(prop-1-en-1-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>71</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with (<i>E</i>)-prop-1-en-1-ylboronic acid using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.188 min and (method 2) = 3.502 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.17 (s, 1H), 8.30 (s, 1H), 7.67 (t, <i>J</i> = 7.9 Hz, 1H), 7.60 (s, 2H), 7.54 (dd, <i>J</i> = 7.4, 2.3 Hz, 1H), 7.45 (ddd, <i>J</i> = 8.6, 5.0, 2.3 Hz, 1H), 7.26–7.14 (m, 2H), 7.06 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.47 (dq, <i>J</i> = 15.9, 1.7 Hz, 1H), 6.13 (dq, <i>J</i> = 16.0, 6.6 Hz, 1H), 4.14 (s, 2H), 3.18 (d, <i>J</i> = 7.0 Hz, 2H), 1.85 (dd, <i>J</i> = 6.6, 1.7 Hz, 3H), 1.22–1.08 (m, 1H), 0.40–0.29 (m, 2H), 0.31–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 571.128; found, 571.1279.</div></div><div id="sec5_1_9_71" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> (<i>E</i>)-2-(5-(Cyclopropylmethyl)-3-(3-(3,3-dimethylbut-1-en-1-yl)-4-fluorophenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>72</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with (<i>E</i>)-(3,3-dimethylbut-1-en-1-yl)boronic acid using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.048 min and (method 2) = 3.696 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.15 (s, 1H), 8.30 (d, <i>J</i> = 0.8 Hz, 1H), 7.67 (t, <i>J</i> = 7.9 Hz, 1H), 7.58 (dd, <i>J</i> = 7.3, 2.3 Hz, 1H), 7.56 (s, 2H), 7.51 (ddd, <i>J</i> = 8.5, 5.0, 2.3 Hz, 1H), 7.28–7.15 (m, 2H), 7.07 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.36 (d, <i>J</i> = 16.4 Hz, 1H), 6.18 (d, <i>J</i> = 16.3 Hz, 1H), 4.17 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 1.24–1.10 (m, 1H), 1.07 (d, <i>J</i> = 1.0 Hz, 9H), 0.39–0.28 (m, 2H), 0.31–0.19 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>31</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 613.1749; found, 613.1728.</div></div><div id="sec5_1_9_72" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> 2-(5-(Cyclopropylmethyl)-3-(4-fluoro-3-(3,3,3-trifluoroprop-1-en-2-yl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>73</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIf</b> with 4,4,6,6-tetramethyl-2-(3,3,3-trifluoroprop-1-en-2-yl)-1,3,2-dioxaborinane using the general Suzuki coupling procedure G and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 5.911 min and (method 2) = 3.538 min; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.20 (s, 1H), 8.26 (s, 1H), 7.71–7.60 (m, 2H), 7.59 (s, 2H), 7.51 (dd, <i>J</i> = 7.2, 2.3 Hz, 1H), 7.39 (dd, <i>J</i> = 10.0, 8.6 Hz, 1H), 7.12 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.03 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.32 (dt, <i>J</i> = 2.1, 1.0 Hz, 1H), 5.97 (s, 1H), 4.14 (s, 2H), 3.17 (d, <i>J</i> = 6.9 Hz, 2H), 1.24–1.07 (m, 1H), 0.39–0.31 (m, 2H), 0.29–0.18 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>22</sub>F<sub>5</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub>, 625.0997; found, 625.099.</div></div><div id="sec5_1_9_73" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> 2-(5-(2-Cyclopropylethyl)-3-(4-fluoro-3-((5-methylthiophen-2-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>74</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIj</b> with 2-ethynyl-5-methylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.859 min and (method 2) = 3.779 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.12 (s, 1H), 8.29 (s, 1H), 7.73 (dd, <i>J</i> = 6.9, 2.3 Hz, 1H), 7.67 (t, <i>J</i> = 7.9 Hz, 1H), 7.61 (ddd, <i>J</i> = 8.7, 5.0, 2.3 Hz, 1H), 7.58 (s, 2H), 7.39 (dd, <i>J</i> = 9.4, 8.7 Hz, 1H), 7.29 (dd, <i>J</i> = 3.6, 0.5 Hz, 1H), 7.19 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.85 (dt, <i>J</i> = 3.4, 1.0 Hz, 1H), 4.17 (s, 2H), 3.29–3.20 (m, 2H), 2.48 (d, <i>J</i> = 1.1 Hz, 3H), 1.43 (q, <i>J</i> = 7.3 Hz, 2H), 0.73 (ddt, <i>J</i> = 10.2, 7.5, 3.7 Hz, 1H), 0.36–0.26 (m, 2H), 0.16–0.08 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.75, 160.93, 160.21, 159.32, 156.81, 150.73, 147.44, 147.36, 144.69, 144.57, 143.50, 133.80, 131.92, 130.01, 129.93, 129.60, 129.46, 128.59, 126.45, 125.98, 123.71, 118.51, 116.83, 116.46, 116.24, 110.94, 88.55, 84.99, 33.27, 27.98, 24.73, 15.07, 10.55, 4.18; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 665.1157; found, 665.1174.</div></div><div id="sec5_1_9_74" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> 2-(5-(2-Cyclopropylethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>75</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIi</b> with 2-ethynyl-5-methylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.713 min and (method 2) = 3.838 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.12 (s, 1H), 8.29 (s, 1H), 7.72–7.63 (m, 2H), 7.57 (s, 2H), 7.56 (ddt, <i>J</i> = 17.6, 7.7, 1.4 Hz, 2H), 7.46 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.24 (dd, <i>J</i> = 3.6, 0.5 Hz, 1H), 7.19 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.83 (dt, <i>J</i> = 3.4, 1.1 Hz, 1H), 4.17 (s, 2H), 3.29–3.21 (m, 2H), 2.47 (d, <i>J</i> = 1.1 Hz, 3H), 1.43 (q, <i>J</i> = 7.3 Hz, 2H), 0.81–0.67 (m, 1H), 0.36–0.26 (m, 2H), 0.16–0.06 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.79, 160.96, 159.32, 156.80, 151.49, 144.72, 144.66, 142.80, 133.26, 132.28, 131.14, 129.88, 129.57, 129.43, 129.37, 128.60, 127.71, 126.31, 125.86, 123.69, 122.57, 119.12, 116.85, 116.42, 116.21, 91.71, 83.65, 33.27, 28.06, 24.74, 15.06, 10.56, 4.18; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub> 647.1251; found, 647.1272.</div></div><div id="sec5_1_9_75" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> 2-(5-(2-Cyclopropylethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylfuran-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>76</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIc</b> with 2-bromo-5-methylfuran using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.449 min and (method 2) = 3.533 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.32 (s, 1H), 8.47 (s, 1H), 7.87 (d, <i>J</i> = 7.5 Hz, 2H), 7.77 (s, 3H), 7.75–7.62 (m, 2H), 7.37 (d, <i>J</i> = 11.2 Hz, 1H), 7.26 (d, <i>J</i> = 8.1 Hz, 1H), 6.99 (s, 1H), 6.41 (s, 1H), 4.36 (s, 2H), 2.69 (s, 2H), 1.67–1.56 (m, 2H), 0.93 (s, 1H), 0.50–0.31 (m, 4H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.76, 160.99, 154.34, 151.45, 147.44, 144.67, 144.58, 132.34, 131.06, 129.70, 129.44, 128.59, 127.95, 125.98, 123.73, 122.01, 117.70, 116.88, 116.43, 107.85, 92.48, 80.52, 33.26, 28.04, 24.75, 13.50, 10.56, 4.18; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>5</sub>S<sub>2</sub>, 631.148; found, 631.1486.</div></div><div id="sec5_1_9_76" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> 2-(5-(2-Cyclopropylethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-3-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>77</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIc</b> with 3-bromo-2-methylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.722 min and (method 2) = 3.751 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.13 (s, 1H), 8.29 (s, 1H), 7.72–7.61 (m, 3H), 7.59 (s, 2H), 7.57 (dt, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.55–7.49 (m, 1H), 7.53–7.42 (m, 1H), 7.20 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.08 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 6.95 (p, <i>J</i> = 1.1 Hz, 1H), 4.17 (s, 2H), 3.29–3.21 (m, 2H), 2.46 (d, <i>J</i> = 1.1 Hz, 3H), 1.44 (dt, <i>J</i> = 10.3, 7.1 Hz, 2H), 0.82–0.67 (m, 1H), 0.36–0.26 (m, 2H), 0.16–0.08 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 647.1251; found, 647.1237.</div></div><div id="sec5_1_9_77" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> 2-(5-(2-Cyclopropylethyl)-3-(3-((5-cyclopropylthiophen-2-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>78</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIc</b> with 2-bromo-5-cyclopropylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 8.339 min and (method 2) = 3.842 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.13 (s, 1H), 8.28 (s, 1H), 7.72–7.60 (m, 2H), 7.58 (s, 2H), 7.60–7.49 (m, 2H), 7.46 (t, <i>J</i> = 7.7 Hz, 1H), 7.25–7.15 (m, 2H), 7.07 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 6.82 (dd, <i>J</i> = 3.7, 0.7 Hz, 1H), 4.17 (s, 2H), 3.29–3.21 (m, 2H), 2.17 (tt, <i>J</i> = 8.3, 5.0 Hz, 1H), 1.43 (q, <i>J</i> = 7.3 Hz, 2H), 1.11–0.98 (m, 2H), 0.81–0.65 (m, 3H), 0.36–0.25 (m, 2H), 0.16–0.08 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>34</sub>H<sub>30</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 673.1408; found, 673.1392.</div></div><div id="sec5_1_9_78" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> 2-(3-(3-((5-Cyclobutylthiophen-2-yl)ethynyl)phenyl)-5-(2-cyclopropylethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>79</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIc</b> with 2-bromo-5-cyclobutylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 7.29 min and (method 2) = 4.251 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.13 (s, 1H), 8.28 (s, 1H), 7.72–7.64 (m, 2H), 7.62–7.54 (m, 3H), 7.53 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.46 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.26 (d, <i>J</i> = 3.6 Hz, 1H), 7.19 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.08 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.88 (dd, <i>J</i> = 3.7, 0.9 Hz, 1H), 4.17 (s, 2H), 3.72 (dqt, <i>J</i> = 9.0, 8.0, 1.0 Hz, 1H), 3.29–3.21 (m, 2H), 2.47–2.32 (m, 2H), 2.19–2.04 (m, 2H), 2.07–1.88 (m, 1H), 1.91–1.78 (m, 1H), 1.43 (q, <i>J</i> = 7.4 Hz, 2H), 0.74 (ddd, <i>J</i> = 12.6, 8.5, 5.0 Hz, 1H), 0.36–0.24 (m, 2H), 0.16–0.08 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>35</sub>H<sub>32</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 687.1564; found, 687.1558.</div></div><div id="sec5_1_9_79" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> 2-(5-(2-Cyclopropylethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiazol-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>80</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIc</b> with 2-bromo-5-methylthiazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.096 min and (method 2) = 3.581 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (s, 1H), 8.28 (s, 1H), 7.80 (td, <i>J</i> = 1.7, 0.6 Hz, 1H), 7.71–7.62 (m, 4H), 7.58 (s, 2H), 7.56–7.47 (m, 1H), 7.18 (dd, <i>J</i> = 11.4, 1.5 Hz, 1H), 7.08 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 4.19 (s, 2H), 3.25 (s, 2H), 2.51 (s, 3H), 1.43 (q, <i>J</i> = 7.5 Hz, 2H), 0.78–0.69 (m, 1H), 0.36–0.27 (m, 2H), 0.16–0.08 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>27</sub>FN<sub>5</sub>O<sub>4</sub>S<sub>3</sub>, 648.1204; found, 648.1202.</div></div><div id="sec5_1_9_80" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> 2-(5-(2-Cyclopropylethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((2-methylthiazol-5-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>81</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIc</b> with 5-bromo-2-methylthiazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.039 min and (method 2) = 3.566 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.13 (s, 1H), 8.28 (s, 1H), 7.96 (s, 1H), 7.74–7.54 (m, 7H), 7.48 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.19 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.18 (s, 2H), 3.24 (d, <i>J</i> = 8.2 Hz, 2H), 2.69 (s, 3H), 1.43 (q, <i>J</i> = 7.4 Hz, 2H), 0.74 (td, <i>J</i> = 7.5, 3.8 Hz, 1H), 0.36–0.27 (m, 2H), 0.16–0.08 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.55, 161.76, 160.97, 156.79, 151.40, 146.95, 144.69, 144.57, 132.33, 131.34, 130.07, 129.43, 128.58, 128.19, 125.98, 123.71, 122.00, 116.93, 116.88, 116.22, 94.67, 79.87, 33.27, 28.04, 24.74, 18.97, 10.56, 4.18; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>27</sub>FN<sub>5</sub>O<sub>4</sub>S<sub>3</sub>, 648.1204; found, 648.1191.</div></div><div id="sec5_1_9_81" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> 2-(5-(2-Cyclopropylethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((2-methylthiazol-4-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>82</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIc</b> with 4-bromo-2-methylthiazole using the general Sonogashira coupling procedure F and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.017 min and (method 2) = 3.538 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.13 (s, 1H), 8.30 (s, 1H), 7.92 (s, 1H), 7.73 (dt, <i>J</i> = 1.8, 1.0 Hz, 1H), 7.68 (t, <i>J</i> = 7.9 Hz, 1H), 7.59 (s, 2H), 7.64–7.54 (m, 2H), 7.53–7.44 (m, 1H), 7.19 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.08 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 4.18 (s, 2H), 3.29–3.21 (m, 2H), 2.68 (s, 3H), 1.43 (q, <i>J</i> = 7.3 Hz, 2H), 0.81–0.67 (m, 1H), 0.38–0.26 (m, 2H), 0.16–0.08 (m, 2H); <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.85, 161.76, 160.99, 151.47, 147.46, 144.68, 144.58, 132.34, 131.51, 130.19, 129.44, 129.43, 128.60, 128.05, 125.97, 124.77, 123.75, 122.17, 116.88, 116.44, 116.23, 87.46, 84.70, 33.26, 28.06, 24.76, 18.74, 10.57, 4.18; HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>27</sub>FN<sub>5</sub>O<sub>4</sub>S<sub>3</sub>, 648.1204; found, 648.1229.</div></div><div id="sec5_1_9_82" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> 2-(5-(2-Cyclopropylethyl)-3-(3-((2,5-dimethylthiophen-3-yl)ethynyl)phenyl)-4-(3-fluoro-4-sulfamoylbenzyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>83</b>)</h5><div class="NLM_p last">This compound was synthesized from coupling the advanced intermediate <b>VIIIc</b> with 3-bromo-2,5-dimethylthiophene using the general Sonogashira coupling procedure E and subsequent hydrolysis using general procedure H. LC–MS retention time: (method 1) = 6.982 min and (method 2) = 3.833 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.13 (s, 1H), 8.28 (s, 1H), 7.68 (t, <i>J</i> = 7.9 Hz, 1H), 7.64–7.56 (m, 2H), 7.59 (s, 2H), 7.52 (dt, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.46 (td, <i>J</i> = 7.6, 0.7 Hz, 1H), 7.21 (dd, <i>J</i> = 11.3, 1.6 Hz, 1H), 7.09 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.77 (q, <i>J</i> = 1.1 Hz, 1H), 4.17 (s, 2H), 3.29–3.20 (m, 2H), 2.44 (d, <i>J</i> = 0.7 Hz, 3H), 2.38 (t, <i>J</i> = 0.9 Hz, 3H), 1.44 (q, <i>J</i> = 7.3 Hz, 2H), 0.81–0.67 (m, 1H), 0.36–0.25 (m, 2H), 0.16–0.08 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>33</sub>H<sub>30</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 661.1408; found, 661.1414.</div></div><div id="sec5_1_9_83" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> 2-(5-(1-Cyclopropylethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>84</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, E, and H <i>via</i> intermediates <b>If</b> through <b>VIIm</b>. LC–MS retention time: (method 1) = 6.581 min and (method 2) = 3.785 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.14 (s, 1H), 8.32 (s, 1H), 7.73–7.63 (m, 2H), 7.59–7.49 (m, 4H), 7.44 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.23 (dd, <i>J</i> = 3.6, 0.5 Hz, 1H), 7.17 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.07 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.83 (dt, <i>J</i> = 3.4, 1.0 Hz, 1H), 4.25 (d, <i>J</i> = 3.2 Hz, 2H), 2.47 (d, <i>J</i> = 1.1 Hz, 3H), 1.37 (d, <i>J</i> = 7.2 Hz, 3H), 0.54–0.46 (m, 2H), 0.29–0.08 (m, 4H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 647.1251; found, 647.1268.</div></div><div id="sec5_1_9_84" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> 2-(5-(2-Cyclopropylpropan-2-yl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>85</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, E, and H <i>via</i> intermediates <b>Ig</b> through <b>VIIo</b>. LC–MS retention time: (method 1) = 6.425 min and (method 2) = 3.741 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.27 (s, 1H), 8.58 (s, 1H), 7.74 (t, <i>J</i> = 8.0 Hz, 1H), 7.59 (s, 2H), 7.57–7.47 (m, 2H), 7.47–7.36 (m, 2H), 7.27–7.15 (m, 2H), 7.09 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.81 (dt, <i>J</i> = 3.4, 1.1 Hz, 1H), 4.30 (s, 2H), 2.46 (dd, <i>J</i> = 1.1, 0.4 Hz, 3H), 1.28 (tt, <i>J</i> = 8.3, 5.8 Hz, 1H), 1.19 (s, 6H), 0.29–0.15 (m, 4H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>33</sub>H<sub>30</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 661.1408; found, 661.14.</div></div><div id="sec5_1_9_85" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> 2-(4-(3-Fluoro-4-sulfamoylbenzyl)-5-((1-methylcyclopropyl)methyl)-3-(3-((5-methylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>86</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, E, and H <i>via</i> intermediates <b>Ih</b> through <b>VIIn</b>. LC–MS retention time: (method 1) = 6.694 min and (method 2) = 3.751 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.16 (s, 1H), 8.32 (s, 1H), 7.70 (td, <i>J</i> = 1.7, 0.6 Hz, 1H), 7.64 (t, <i>J</i> = 7.9 Hz, 1H), 7.63–7.56 (m, 1H), 7.55 (s, 2H), 7.50 (dt, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.43 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.24 (dd, <i>J</i> = 3.6, 0.5 Hz, 1H), 7.17–7.00 (m, 2H), 6.83 (dt, <i>J</i> = 3.4, 1.1 Hz, 1H), 4.21 (s, 2H), 3.45 (s, 2H), 2.48 (d, <i>J</i> = 1.1 Hz, 3H), 1.00 (s, 3H), 0.37–0.26 (m, 2H), 0.20–0.08 (m, 2H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 647.1251; found, 647.127.</div></div><div id="sec5_1_9_86" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> 2-(4-(3-Fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-2-yl)ethynyl)phenyl)-5-(spiro[2.2]pentan-1-yl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>87</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, E, and H <i>via</i> intermediates <b>Ii</b> through <b>VIIp</b>. LC–MS retention time: (method 1) = 6.604 min and (method 2) = 3.769 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.19 (s, 1H), 8.33 (d, <i>J</i> = 4.1 Hz, 1H), 7.74–7.38 (m, 7H), 7.24 (ddd, <i>J</i> = 3.5, 2.7, 0.5 Hz, 1H), 7.20–7.01 (m, 2H), 6.83 (dq, <i>J</i> = 3.4, 1.1 Hz, 1H), 4.34–4.12 (m, 2H), 2.77–2.58 (m, 1H), 2.47 (s, 3H), 1.65–1.08 (m, 2H), 0.77–0.09 (m, 3H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>32</sub>H<sub>26</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 645.1095; found, 645.1123.</div></div><div id="sec5_1_9_87" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> 2-(5-([1,1′-Bi(cyclopropan)]-2-yl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>88</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, E, and H <i>via</i> intermediates <b>Iq</b> through <b>VIIk</b>. LC–MS retention time: (method 1) = 7.018 min and (method 2) = 3.871 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.13 (s, 1H), 8.30 (s, 1H), 7.74–7.65 (m, 2H), 7.62–7.52 (m, 4H), 7.47 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.24 (dd, <i>J</i> = 3.6, 0.5 Hz, 1H), 7.20 (dd, <i>J</i> = 11.4, 1.6 Hz, 1H), 7.08 (dd, <i>J</i> = 8.2, 1.6 Hz, 1H), 6.83 (dt, <i>J</i> = 3.4, 1.0 Hz, 1H), 4.14 (s, 2H), 2.78–2.63 (m, 1H), 2.47 (d, <i>J</i> = 1.1 Hz, 3H), 1.80 (ddq, <i>J</i> = 75.6, 14.6, 7.6 Hz, 2H), 0.78 (ddt, <i>J</i> = 13.1, 8.2, 4.2 Hz, 1H), 0.55 to −0.05 (m, 4H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>33</sub>H<sub>28</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 659.1251; found, 659.1282.</div></div><div id="sec5_1_9_88" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> 2-(5-(Dicyclopropylmethyl)-4-(3-fluoro-4-sulfamoylbenzyl)-3-(3-((5-methylthiophen-2-yl)ethynyl)phenyl)-1<i>H</i>-pyrazol-1-yl)thiazole-4-carboxylic Acid (<b>89</b>)</h5><div class="NLM_p last">This compound was synthesized using general procedures A, B, C, D, E, and H <i>via</i> intermediates <b>Il</b> through <b>VIIr</b>. LC–MS retention time: (method 1) = 6.68 min and (method 2) = 3.741 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.11 (s, 1H), 8.32 (s, 1H), 7.72–7.63 (m, 2H), 7.63–7.54 (m, 3H), 7.51 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.43 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 7.23 (dd, <i>J</i> = 3.6, 0.5 Hz, 1H), 7.18–7.11 (m, 1H), 7.05 (d, <i>J</i> = 7.8 Hz, 1H), 6.83 (dt, <i>J</i> = 3.4, 1.1 Hz, 1H), 4.26 (d, <i>J</i> = 11.1 Hz, 2H), 2.47 (d, <i>J</i> = 1.1 Hz, 3H), 1.96–1.14 (m, 3H), 0.73 to −0.19 (m, 8H); HRMS (ESI) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>34</sub>H<sub>30</sub>FN<sub>4</sub>O<sub>4</sub>S<sub>3</sub>, 673.1408; found, 673.1422.</div></div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> Biological Assays</h3><div class="NLM_p last">LDH and MDH biochemical assays, cellular lactate production assay, and cytotoxicity assays were performed as previously reported<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and also provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_002.csv" class="ext-link">Supporting Information</a>.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i117" class="anchor-spacer"></div><h3 class="article-section__title" id="_i117"> Split Nano Luciferase Cellular Thermal Shift Assay</h3><div class="NLM_p last">The SplitLuc CETSA was performed in 1536-well plates as previously described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Briefly, HEK293T cells were transiently transfected with plasmid encoding a CMV-driven LDHA open-reading frame with a carboxy-terminal 86b fusion tag (GS[HiBiT]GS). Cells were transfected in a T175 flask for 24 h using 2.3 × 10<sup>7</sup> cells, 52.5 μg of plasmid DNA, and 105 μL of Lipofectamine 2000. Cells were lifted, resuspended at 5 × 10<sup>5</sup> cells/mL (DPBS with CaCl<sub>2</sub> and MgCl<sub>2</sub> plus 1 g/L glucose), and reseeded into 1536-well cyclic olefin white plates (Aurora, cyclic olefin polymer, cat# EWB041000A) using a Multidrop Combi at 2500 cells per well (5 μL volume). Compounds (23 nL) were added to cells using a pin tool (Wako Automation) and incubated for 1 h at 37 °C. Plates were heated to 63.5 °C for 7.5 min using a custom-machined copper heat block fitted with an internal type-T thermocouple and controlled by a Watlow temperature controller. The plates were removed from the heat block and cooled to room temperature. One microliter of 6% NP40 was added per well and plates were incubated for 30 min to allow cell lysis, followed by addition of 3 μL of substrate containing 11S (final concentration 100 nM) and furimazine (final concentration 0.5×; from Promega 50× stock). The plates were centrifuged and analyzed for luminescence intensity using a ViewLux reader equipped with clear filters. Luminescence values were normalized to an unheated control sample.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i118" class="anchor-spacer"></div><h3 class="article-section__title" id="_i118"> Glycolytic Stress Test Assay</h3><div class="NLM_p last">A673 cells were cultured in Dulbecco’s modified Eagle’s medium (ATCC Catalog no. 302002) supplemented with fetal bovine serum (10%). The cells were plated into a XF96 cell culture microplate in the above-mentioned medium and maintained in a 5% CO<sub>2</sub> incubator at 37 °C for 24 h prior to the experiments. On the day of the assay, the compounds were diluted to the appropriate concentration in freshly prepared assay media (Seahorse basic DMEM with 2 mM glutamine, pH 7.4 at 37 °C). The media in the plate with cells were then changed to assay media and maintained in a non-CO<sub>2</sub> incubator at 37 °C for 1 h prior to the assay. The Seahorse XF GST was conducted by injecting the LDH inhibitors and then, at 48 min, subsequent injections of glucose (10 mM final concentration), oligomycin (1 μg/mL final concentration), and 2-deoxyglucose (2-DG; 50 mM final concentration) as described previously.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> PAMPA Permeability Assay</h3><div class="NLM_p last">The stirring double-sink PAMPA method patented by pION Inc. (Billerica, MA) was employed to determine the permeability of compounds <i>via</i> PAMPA passive diffusion. The PAMPA lipid membrane consisted of an artificial membrane of a proprietary lipid mixture and dodecane (Pion Inc.), optimized to predict gastrointestinal tract passive diffusion permeability, and immobilized on a plastic matrix of a 96-well “donor” filter plate placed above a 96-well “acceptor” plate. A pH 7.4 solution was used in both donor and acceptor wells. The test articles, stocked in 10 mM DMSO solutions, were diluted to 0.05 mM in aqueous buffer (pH 7.4), and the concentration of DMSO was 0.5% in the final solution. During the 30 min permeation period at room temperature, the test samples in the donor compartment were stirred using the Gutbox technology (Pion Inc.) to reduce the unstirred water layer. The test article concentrations in the donor and acceptor compartments were measured using a UV plate reader (Nano Quant, Infinite 200 PRO, Tecan Inc., Männedorf, Switzerland). Permeability calculations were performed using Pion Inc. software and were expressed in units of 10<sup>–6</sup> cm/s.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i120" class="anchor-spacer"></div><h3 class="article-section__title" id="_i120"> BBB-PAMPA Permeability Assay</h3><div class="NLM_p last">The double-sink PAMPA method was utilized for the BBB permeability assay as described previously.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The PAMPA-BBB-1 lipid (PN 110672, Pion) membrane consisted of 10% (w/v) porcine lipid brain extract dissolved in a proprietary phospholipid mixture and alkane. Like the PAMPA assay, test articles initially stocked at 10 mM DMSO solutions were diluted to 0.05 mM in an aqueous buffer (pH 7.4). The test samples were stirred using the Gutbox technology at a 60 μM aqueous boundary layer setting for 1 h at room temperature. The concentrations of compounds in the donor and acceptor compartments were measured against a reference and blank samples. The membrane permeability calculations were performed using PAMPA Evolution Software (version 3.8) from Pion Inc. and were expressed in units of 10<sup>–6</sup> cm/s.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i121" class="anchor-spacer"></div><h3 class="article-section__title" id="_i121"> Kinetic Solubility Test Assay</h3><div class="NLM_p last">Pion’s patented μSOL assay for kinetic solubility determination was used. In this assay, the classical saturation shake flask solubility method was adapted to a 96-well microtiter plate format, and a cosolvent method with <i>n</i>-propanol as the reference compound was utilized. Test compounds were prepared in 10 mM DMSO solutions (45 μL) and diluted with the cosolvent to a final drug concentration of 150 μM in the aqueous solution (pH 7.4). The samples were incubated at room temperature for 6 h to achieve equilibrium. The samples were then filtered to remove any precipitate formed. The concentration of the compound in the filtrate was measured by UV absorbance. A reference drug concentration of 17 μM was used for the quantitation of unknown drug concentration in the filtrate. Spectroscopically pure 1-propanol was used as a cosolvent to suppress precipitation in the reference solutions. The kinetic solubility (μg/mL) was calculated using the μSOL Evolution software.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122"> RLM Stability Assay</h3><div class="NLM_p last">Single time point microsomal stability was determined in a 96-well HTS format. Sample preparation was automated using a Tecan EVO 200 robot. A high-resolution LC–MS (Thermo Q Exactive) instrument was used to measure the percentage of the compound remaining after incubation using a previously described method.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Six standard controls were tested in each run: buspirone and propranolol (for short half-life), loperamide and diclofenac (for short to medium half-life), and carbamazepine and antipyrine (for long half-life). Briefly, the incubation consisted of 0.5 mg/mL microsomal protein, 1.0 μM drug concentration, and an NADPH regeneration system (containing 0.650 mM NADP<sup>+</sup>, 1.65 mM glucose 6-phosphate, 1.65 mM MgCl<sub>2</sub>, and 0.2 unit/mL G6PDH) in 100 mM phosphate buffer at pH 7.4. The incubation was carried out at 37 °C for 15 min. The reaction was quenched by adding 555 μL of acetonitrile (∼1:2 ratio) containing 0.28 μM albendazole (internal standard). Sample acquisition and data analysis were done using a previously described method.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i123" class="anchor-spacer"></div><h3 class="article-section__title" id="_i123"> Mouse Pharmacokinetic Studies</h3><div class="NLM_p last">All PK studies were conducted by Pharmaron. Male CD1 mice (sourced from Si Bei Fu Laboratory Animal Technology Co. Ltd), approximately 6–8 weeks of age and a weight of approximately 20–30 g, were dosed with compounds at 10 mg/kg (IV) or 40 or 50 mg/kg (PO). The formulation (0.1 M NaOH in PBS, adjusted with 1 N HCl to pH 7–8.5) was prepared on the day of dosing. Each dosing cohort had three mice and plasma was collected at 5 min, 15 min, 30 min, 1 h, 2 h, 4 h, 8 h, 12 h, and 24 h postdose for IV administration and 15 min, 30 min, 1 h, 2 h, 4 h, 8 h, 12 h, and 24 h for PO administration. Approximately 0.025 mL of blood was collected <i>via</i> the dorsal metatarsal vein at each time point. Blood samples were then transferred into plastic microcentrifuge tubes containing heparin-Na as an anticoagulant. The samples were then centrifuged at 4000<i>g</i> for 5 min at 4 °C to obtain plasma. Plasma samples were then stored in polypropylene tubes, quickly frozen, and maintained at −75 °C until analyzed by LC–MS/MS. The following pharmacokinetic parameters were measured: <i>T</i><sub>1/2</sub>, <i>C</i><sub>0</sub>, <i>C</i><sub>max</sub>, <i>T</i><sub>max</sub>, CL, <i>V</i><sub>d</sub>, AUC<sub>last</sub>, and <i>F</i>. Animals were also monitored during the in-life phase by once daily cage side observations, any adverse clinical signs are noted as part of the PK report.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i124" class="anchor-spacer"></div><h3 class="article-section__title" id="_i124"> <i>In Vivo</i> LDHA Activity in Tumor Mouse Model</h3><div class="NLM_p last">Two million A673 cells were injected SQ into each female athymic nude mouse from Taconic (CrTac:NCr-Foxn1nu). The mice were divided into four treatment groups of 16 mice each. Each group received a single IV injection of 50 mg/kg of an LDHA inhibitor. Four mice from each group were sacrificed at 1, 3, 6, and 24 h after dosing. The inhibitors were dissolved in a standard PBS-based vehicle. At the time of sacrifice, frozen samples of tumor and plasma were collected and compound levels were determined through LC–MS/MS measurement against standard curves of each compound. Frozen tumor samples were also collected for LDHA activity measurements as follows. Briefly, 10–50 mg of frozen tumors was pulverized in liquid nitrogen, followed by the addition of 10 volumes of PBS at pH 7.4 containing 0.1% Triton ×-100 and incubated on ice for 1 h. The samples were clarified by centrifugation and cleared tumor lysates (10–20 μg) were measured by UV–vis spectrometry for LDH activity in LDH assay buffer containing 10 mM sodium pyruvate (Sigma-Aldrich, St. Louis, MO). The reactions were initiated by the addition of 15 mM NADH (final concentration 0.3 mM) and monitored for the oxidation of NADH at 340 nM at 37 °C.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i125"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00916" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36483" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36483" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00916?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00916</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Correlation between LDHA and LDHB; correlation of A673 lactate versus MiaPaCa-2 lactate and A673 cytotoxicity versus MiaPaCa-2; correlation of A673 lactate versus A673 and MiaPaCa-2 lactate versus MiaPaCa-2; MiaPaCa-2 72h viability versus LDHA CETSA binding activity; original LCMS, proton, fluorine, carbon NMR for <b>43</b>, <b>52</b>, and key compounds; experimental procedures for representative scale up of lead compounds and key intermediates; and spectroscopic data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and PDB ID codes of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q0D">6Q0D</a> (<b>23</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q13">6Q13</a> (<b>52</b>). Authors will release the atomic coordinates and experimental data upon article publication (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf">jm0c00916_si_001.pdf (3.39 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_002.csv">jm0c00916_si_002.csv (17.95 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00916" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ganesha Rai</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9763-9641" title="Orcid link">http://orcid.org/0000-0001-9763-9641</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ee8c8f809a9b858f82829b89ae838f8782c0808786c0898198"><span class="__cf_email__" data-cfemail="d6b4b7b8a2a3bdb7babaa3b196bbb7bfbaf8b8bfbef8b1b9a0">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex G. Waterson</span> - <span class="hlFld-Affiliation affiliation">Vanderbilt
Institute of Chemical Biology, Vanderbilt
University, Nashville, Tennessee 37232, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#42236c3523362730312d2c0214232c262730202b2e366c072637"><span class="__cf_email__" data-cfemail="3958174e584d5c4b4a5657796f58575d5c4b5b50554d177c5d4c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Maloney</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5607-4178" title="Orcid link">http://orcid.org/0000-0002-5607-4178</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0c486d7a694c5a697e6d606374226f6361"><span class="__cf_email__" data-cfemail="8ecaeff8ebced8ebfcefe2e1f6a0ede1e3">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel J. Urban</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bryan T. Mott</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Hu</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shyh-Ming Yang</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1928-136X" title="Orcid link">http://orcid.org/0000-0003-1928-136X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gloria A. Benavides</span> - <span class="hlFld-Affiliation affiliation">Mitochondrial
Medicine Laboratory, Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35294, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michelle S. Johnson</span> - <span class="hlFld-Affiliation affiliation">Mitochondrial
Medicine Laboratory, Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35294, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe L. Squadrito</span> - <span class="hlFld-Affiliation affiliation">Mitochondrial
Medicine Laboratory, Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35294, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kyle R. Brimacombe</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobie D. Lee</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dorian M. Cheff</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hu Zhu</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark J. Henderson</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8590-7338" title="Orcid link">http://orcid.org/0000-0001-8590-7338</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine Pohida</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gary A. Sulikowski</span> - <span class="hlFld-Affiliation affiliation">Vanderbilt
Institute of Chemical Biology, Vanderbilt
University, Nashville, Tennessee 37232, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1067-0767" title="Orcid link">http://orcid.org/0000-0002-1067-0767</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David M. Dranow</span> - <span class="hlFld-Affiliation affiliation">Beryllium
Discovery Corp., 7869
Day Road West, Bainbridge Island, Washington 98110, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Md Kabir</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pranav Shah</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elias Padilha</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dingyin Tao</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6531-2740" title="Orcid link">http://orcid.org/0000-0001-6531-2740</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuhong Fang</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Plamen P. Christov</span> - <span class="hlFld-Affiliation affiliation">Vanderbilt
Institute of Chemical Biology, Vanderbilt
University, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kwangho Kim</span> - <span class="hlFld-Affiliation affiliation">Vanderbilt
Institute of Chemical Biology, Vanderbilt
University, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Somnath Jana</span> - <span class="hlFld-Affiliation affiliation">Vanderbilt
Institute of Chemical Biology, Vanderbilt
University, Nashville, Tennessee 37232, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pavan Muttil</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, University of New Mexico Health
Sciences Center, Albuquerque, New Mexico 87131, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7636-8141" title="Orcid link">http://orcid.org/0000-0001-7636-8141</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tamara Anderson</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, University of New Mexico Health
Sciences Center, Albuquerque, New Mexico 87131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nitesh K. Kunda</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, University of New Mexico Health
Sciences Center, Albuquerque, New Mexico 87131, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3063-0375" title="Orcid link">http://orcid.org/0000-0002-3063-0375</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helen J. Hathaway</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, University of New Mexico Health
Sciences Center, Albuquerque, New Mexico 87131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Donna F. Kusewitt</span> - <span class="hlFld-Affiliation affiliation">Dept
of Pathology, University of New Mexico Cancer
Center, Albuquerque, New Mexico 87131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nobu Oshima</span> - <span class="hlFld-Affiliation affiliation">Urologic
Oncology Branch, Center for Cancer Research, National Cancer Institute, 9000 Rockville Pike, Bethesda, Maryland 20892, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Murali Cherukuri</span> - <span class="hlFld-Affiliation affiliation">Urologic
Oncology Branch, Center for Cancer Research, National Cancer Institute, 9000 Rockville Pike, Bethesda, Maryland 20892, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Douglas R. Davies</span> - <span class="hlFld-Affiliation affiliation">Beryllium
Discovery Corp., 7869
Day Road West, Bainbridge Island, Washington 98110, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey P. Norenberg</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, University of New Mexico Health
Sciences Center, Albuquerque, New Mexico 87131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Larry A. Sklar</span> - <span class="hlFld-Affiliation affiliation">Dept
of Pathology, University of New Mexico Cancer
Center, Albuquerque, New Mexico 87131, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William J. Moore</span> - <span class="hlFld-Affiliation affiliation">NExT
Program Support, Applied/Developmental Research Directorate, Leidos
Biomedical Research, Inc., Frederick National
Laboratory for Cancer Research, Frederick, Maryland 21702, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chi V. Dang</span> - <span class="hlFld-Affiliation affiliation">Abramson
Cancer Center, Abramson Family Cancer Research Institute, Perelman
School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Ludwig
Institute for Cancer Research, New York, New York 10017, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gordon M. Stott</span> - <span class="hlFld-Affiliation affiliation">NExT
Program Support, Applied/Developmental Research Directorate, Leidos
Biomedical Research, Inc., Frederick National
Laboratory for Cancer Research, Frederick, Maryland 21702, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leonard Neckers</span> - <span class="hlFld-Affiliation affiliation">Urologic
Oncology Branch, Center for Cancer Research, National Cancer Institute, 9000 Rockville Pike, Bethesda, Maryland 20892, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew J. Flint</span> - <span class="hlFld-Affiliation affiliation">NExT
Program Support, Applied/Developmental Research Directorate, Leidos
Biomedical Research, Inc., Frederick National
Laboratory for Cancer Research, Frederick, Maryland 21702, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Victor M. Darley-Usmar</span> - <span class="hlFld-Affiliation affiliation">Mitochondrial
Medicine Laboratory, Department of Pathology, University of Alabama at Birmingham, Birmingham, Alabama 35294, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anton Simeonov</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ajit Jadhav</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Hall</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5073-442X" title="Orcid link">http://orcid.org/0000-0002-5073-442X</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>G.R. and D.J.U. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">All animal studies included as part of this manuscript were performed in accordance with institutional guidelines as defined by the Institutional Animal Care and Use Committee (IACUC).</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i129">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65473" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65473" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This project has been funded in whole or in part with Federal Funds from the National Cancer Institute, National Institutes of Health, under the contract no. HHSN261200800001E. The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government. G.R., D.J.U., B.T.M., X.H., K.R.B., S.-M.Y., T.D.L., D.M.C., M.J.H., D.T., Y.F., A.S., A.J., and D.J.M. also gratefully acknowledge the funding from the Intramural Research Program, National Center for Advancing Translational Sciences (NCATS), National Institutes of Health (NIH). The authors wish to thank Sam Michael and Richard Jones for automation support; Paul Shinn, Misha Itkin, Zina Itkin, and Danielle van Leer for the assistance with compound management; Christopher LeClair Heather Baker and Elizabeth Fernandez for analytical chemistry and purification support; and Xin Xu for <i>in vitro</i> ADME data. They would also like to thank Genentech for the generous donation of GNE140 for our studies, David Myszka at Biosensor Tools for conducting the SPR experiments, and Pharmaron Inc. for conducting the pharmacokinetic studies.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ACN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">LDH</td><td class="NLM_def"><p class="first last">lactate dehydrogenase</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">microwave</p></td></tr><tr><td class="NLM_term">NCGC</td><td class="NLM_def"><p class="first last">NCATS Chemical Genomics Center</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeability assay</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">qHTS</td><td class="NLM_def"><p class="first last">quantitative high-throughput screening</p></td></tr><tr><td class="NLM_term">SARs</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">TBAF</td><td class="NLM_def"><p class="first last">tetra-<i>n</i>-butylammonium fluoride</p></td></tr><tr><td class="NLM_term">TFAF</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i131">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 31 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warburg, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posener, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negelein, E.</span></span> <span> </span><span class="NLM_article-title">Ueber den stoffwechsel der tumoren</span>. <i>Biochem. Z.</i> <span class="NLM_year" style="font-weight: bold;">1924</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">344</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=1924&pages=319-344&author=O.+Warburgauthor=K.+Posenerauthor=E.+Negelein&title=Ueber+den+stoffwechsel+der+tumoren"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWarburg%26aufirst%3DO.%26aulast%3DPosener%26aufirst%3DK.%26aulast%3DNegelein%26aufirst%3DE.%26atitle%3DUeber%2520den%2520stoffwechsel%2520der%2520tumoren%26jtitle%3DBiochem.%2520Z.%26date%3D1924%26volume%3D152%26spage%3D319%26epage%3D344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Understanding the Warburg effect: the metabolic requirements of cell proliferation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>324</i></span>,  <span class="NLM_fpage">1029</span>– <span class="NLM_lpage">1033</span>, <span class="refDoi"> DOI: 10.1126/science.1160809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1126%2Fscience.1160809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=19460998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVKlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=2009&pages=1029-1033&author=M.+G.+Vander%0AHeidenauthor=L.+C.+Cantleyauthor=C.+B.+Thompson&title=Understanding+the+Warburg+effect%3A+the+metabolic+requirements+of+cell+proliferation&doi=10.1126%2Fscience.1160809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the Warburg Effect: The Metabolic Requirements of Cell Proliferation</span></div><div class="casAuthors">Vander Heiden, Matthew G.; Cantley, Lewis C.; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">324</span>
        (<span class="NLM_cas:issue">5930</span>),
    <span class="NLM_cas:pages">1029-1033</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  In contrast to normal differentiated cells, which rely primarily on mitochondrial oxidative phosphorylation to generate the energy needed for cellular processes, most cancer cells instead rely on aerobic glycolysis, a phenomenon termed "the Warburg effect".  Aerobic glycolysis is an inefficient way to generate ATP, however, and the advantage it confers to cancer cells has been unclear.  Here we propose that the metab. of cancer cells, and indeed all proliferating cells, is adapted to facilitate the uptake and incorporation of nutrients into the biomass (e.g., nucleotides, amino acids, and lipids) needed to produce a new cell.  Supporting this idea are recent studies showing that (i) several signaling pathways implicated in cell proliferation also regulate metabolic pathways that incorporate nutrients into biomass; and that (ii) certain cancer-assocd. mutations enable cancer cells to acquire and metabolize nutrients in a manner conducive to proliferation rather than efficient ATP prodn.  A better understanding of the mechanistic links between cellular metab. and growth control may ultimately lead to better treatments for human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGvLHGQ9TX77Vg90H21EOLACvtfcHk0lhjFoG9upbqHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVKlsbg%253D&md5=f927402a43cdca471c3f4874dcbeb943</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1160809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1160809%26sid%3Dliteratum%253Aachs%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DUnderstanding%2520the%2520Warburg%2520effect%253A%2520the%2520metabolic%2520requirements%2520of%2520cell%2520proliferation%26jtitle%3DScience%26date%3D2009%26volume%3D324%26spage%3D1029%26epage%3D1033%26doi%3D10.1126%2Fscience.1160809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Exploiting tumor metabolism: challenges for clinical translation</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">3648</span>– <span class="NLM_lpage">3651</span>, <span class="refDoi"> DOI: 10.1172/jci72391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1172%2FJCI72391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=23999437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVenu7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2013&pages=3648-3651&author=M.+G.+Vander%0AHeiden&title=Exploiting+tumor+metabolism%3A+challenges+for+clinical+translation&doi=10.1172%2Fjci72391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting tumor metabolism: challenges for clinical translation</span></div><div class="casAuthors">Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3648-3651</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review.  The metab. of cancer cells differs from most normal cells, but how to exploit this difference for patient benefit is incompletely understood.  Cancer cells require altered metab. to efficiently incorporate nutrients into biomass and support abnormal proliferation.  In addn., the survival of tumor cells outside of a normal tissue context requires adaptation of metab. to different microenvironments.  Some existing chemotherapies target metabolic enzymes, and there is a resurgent interest in developing new cancer drugs that interfere with metab.  Success with this approach depends on understanding why specific metabolic pathways are important for cancer cells, detg. how best to select patients, and developing technologies for monitoring patient response to therapies that target metabolic enzymes.  The articles in this Review series address these issues, with a focus on how altered metab. might influence tumor progression and how this knowledge might inform the use of new therapies targeting cancer metab.  Emerging biomarker strategies to guide drug development are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouHaPmfjSMjLVg90H21EOLACvtfcHk0lgxI7n8xVHtZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVenu7bO&md5=384f19e859ee2665ae545dcf1d937b6c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1172%2FJCI72391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI72391%26sid%3Dliteratum%253Aachs%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DExploiting%2520tumor%2520metabolism%253A%2520challenges%2520for%2520clinical%2520translation%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2013%26volume%3D123%26spage%3D3648%26epage%3D3651%26doi%3D10.1172%2Fjci72391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Z.</span></span> <span> </span><span class="NLM_article-title">Overexpression of lactate dehydrogenase-A in human intrahepatic cholangiocarcinoma: its implication for treatment</span>. <i>World J. Surg. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1186/1477-7819-12-78</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1186%2F1477-7819-12-78" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24679073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A280%3ADC%252BC2crosV2rsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=78-84&author=Y.+Yuauthor=M.+Liaoauthor=R.+Liuauthor=J.+Chenauthor=H.+Fengauthor=Z.+Fu&title=Overexpression+of+lactate+dehydrogenase-A+in+human+intrahepatic+cholangiocarcinoma%3A+its+implication+for+treatment&doi=10.1186%2F1477-7819-12-78"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of lactate dehydrogenase-A in human intrahepatic cholangiocarcinoma: its implication for treatment</span></div><div class="casAuthors">Yu Yaping; Liao Minqi; Liu Ruiwen; Chen Jian; Feng Hao; Fu Zan</div><div class="citationInfo"><span class="NLM_cas:title">World journal of surgical oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">78</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Previous studies have shown that lactate dehydrogenase-A (LDH-A) is strongly expressed in several malignancies, that LDH-A expression is associated with poor prognosis, and that LDH-A inhibition severely diminishes tumorigenicity.  However, little is known about the implications of LDH-A expression in intrahepatic cholangiocarcinoma.  The purpose of this study was to investigate the expression of LDH-A and to clarify its effect on intrahepatic cholangiocarcinoma.  METHODS:  We studied the expression of LDH-A in tissue samples from patients with intrahepatic cholangiocarcinoma (n = 54) using the ultrasensitive surfactant protein (S-P) immunohistochemical method.  We then inhibited LDH-A using small hairpin RNA (shRNA) in the cholangiocarcinoma cell line HuCCT-1 in vitro to study the role it plays in promoting growth and escaping apoptosis.  RESULTS:  We report that LDH-A was overexpressed in 52 of 54 (96%) paraffin-embedded cancer tissue samples and 0 of 54 para-carcinoma tissue samples.  Reduction of LDH-A by RNA interference (RNAi) inhibited cell growth and induced apoptosis in HuCCT-1 cells.  This result correlated with the elevation of cytoplasmic reactive oxygen species (ROS) levels.  CONCLUSIONS:  LDH-A expression is closely correlated with histopathological variables of intrahepatic cholangiocarcinoma, indicating that LDH-A may serve as a new treatment target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRH1LWvPIhPcQ8IxFYvehhofW6udTcc2eZr_dfjipOSabntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2crosV2rsg%253D%253D&md5=006604c27e3b3d6a1bea31204f737ef4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1186%2F1477-7819-12-78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1477-7819-12-78%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DFu%26aufirst%3DZ.%26atitle%3DOverexpression%2520of%2520lactate%2520dehydrogenase-A%2520in%2520human%2520intrahepatic%2520cholangiocarcinoma%253A%2520its%2520implication%2520for%2520treatment%26jtitle%3DWorld%2520J.%2520Surg.%2520Oncol.%26date%3D2014%26volume%3D12%26spage%3D78%26epage%3D84%26doi%3D10.1186%2F1477-7819-12-78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, W.</span></span> <span> </span><span class="NLM_article-title">Lactate dehydrogenase A is overexpressed in pancreatic cancer and promotes the growth of pancreatic cancer cells</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1523</span>– <span class="NLM_lpage">1530</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-0679-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1007%2Fs13277-013-0679-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=23404405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos1yktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=1523-1530&author=Y.+Rongauthor=W.+Wuauthor=X.+Niauthor=T.+Kuangauthor=D.+Jinauthor=D.+Wangauthor=W.+Lou&title=Lactate+dehydrogenase+A+is+overexpressed+in+pancreatic+cancer+and+promotes+the+growth+of+pancreatic+cancer+cells&doi=10.1007%2Fs13277-013-0679-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Lactate dehydrogenase A is overexpressed in pancreatic cancer and promotes the growth of pancreatic cancer cells</span></div><div class="casAuthors">Rong, Yefei; Wu, Wenchuan; Ni, Xiaoling; Kuang, Tiantao; Jin, Dayong; Wang, Dansong; Lou, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1523-1530</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The prognosis for pancreatic cancer is very poor, and developing new therapeutic strategies for this cancer is needed.  Recently, the Warburg effect (aerobic glycolysis) has attracted much attention for its function in the tumorigenesis.  Lactate dehydrogenase A (LDHA) executes the final step of aerobic glycolysis and has been reported to be involved in the tumor progression.  However, the function of LDHA in pancreatic cancer has not been studied.  Here, we found that the expression of LDHA was elevated in the clin. pancreatic cancer samples.  Forced expression of LDHA promoted the growth of pancreatic cancer cells, while knocking down the expression of LDHA inhibited cell growth dramatically.  Moreover, silencing the expression of LDHA inhibited the tumorigenicity of pancreatic cancer cells in vivo.  Mechanistically, knocking down the expression of LDHA activated apoptosis pathway.  Taken together, our study revealed the oncogenic role of LDHA in pancreatic cancer and suggested that LDHA might be a potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVt3oJsFE-ebVg90H21EOLACvtfcHk0lgxI7n8xVHtZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos1yktro%253D&md5=4719aab68b47f6c7229832b3425a1f0f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-0679-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-0679-1%26sid%3Dliteratum%253Aachs%26aulast%3DRong%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DKuang%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLou%26aufirst%3DW.%26atitle%3DLactate%2520dehydrogenase%2520A%2520is%2520overexpressed%2520in%2520pancreatic%2520cancer%2520and%2520promotes%2520the%2520growth%2520of%2520pancreatic%2520cancer%2520cells%26jtitle%3DTumor%2520Biol.%26date%3D2013%26volume%3D34%26spage%3D1523%26epage%3D1530%26doi%3D10.1007%2Fs13277-013-0679-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span> <span> </span><span class="NLM_article-title">Recent update on human lactate dehydrogenase enzyme 5 (hLDH5) inhibitors: a promising approach for cancer chemotherapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=487-496&author=R.+Raniauthor=V.+Kumar&title=Recent+update+on+human+lactate+dehydrogenase+enzyme+5+%28hLDH5%29+inhibitors%3A+a+promising+approach+for+cancer+chemotherapy&doi=10.1021%2Facs.jmedchem.5b00168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Update on Human Lactate Dehydrogenase Enzyme 5 (hLDH5) Inhibitors: A Promising Approach for Cancer Chemotherapy</span></div><div class="casAuthors">Rani, Reshma; Kumar, Vinit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">487-496</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human lactate dehydrogenase (hLDH5), a glycolytic enzyme responsible for the conversion of pyruvate to lactate coupled with oxidn. of NADH to NAD+, plays a crucial role in the promotion of glycolysis in invasive tumor cells.  Recently, hLDH5 has been considered a vital therapeutic target for invasive cancers.  Selective inhibition of hLDH5 using small mols. holds potential prospects for the treatment of cancer and assocd. diseases.  Consequently, significant progress has been made in the discovery of selective small-mol. hLDH5 inhibitors displaying remarkable inhibitory potencies.  The purpose of this review is to discuss briefly the roles of hLDH isoforms and to compile small hLDH5 inhibitors into groups based on their chem. classes and pharmacol. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptlkXXBAejKbVg90H21EOLACvtfcHk0lhFRDwvR8zpDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7bL&md5=dcf788f61e81e0420daa844c9896deb4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00168%26sid%3Dliteratum%253Aachs%26aulast%3DRani%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DV.%26atitle%3DRecent%2520update%2520on%2520human%2520lactate%2520dehydrogenase%2520enzyme%25205%2520%2528hLDH5%2529%2520inhibitors%253A%2520a%2520promising%2520approach%2520for%2520cancer%2520chemotherapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D487%26epage%3D496%26doi%3D10.1021%2Facs.jmedchem.5b00168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allison, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granchi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minutolo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, R. M.</span></span> <span> </span><span class="NLM_article-title">Identification of LDH-A as a therapeutic target for cancer cell killing via (i) p53/NAD(H)-dependent and (ii) p53-independent pathways</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e102</span> <span class="refDoi"> DOI: 10.1038/oncsis.2014.16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1038%2Foncsis.2014.16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24819061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvF2lsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&author=S.+J.+Allisonauthor=J.+R.+P.+Knightauthor=C.+Granchiauthor=R.+Raniauthor=F.+Minutoloauthor=J.+Milnerauthor=R.+M.+Phillips&title=Identification+of+LDH-A+as+a+therapeutic+target+for+cancer+cell+killing+via+%28i%29+p53%2FNAD%28H%29-dependent+and+%28ii%29+p53-independent+pathways&doi=10.1038%2Foncsis.2014.16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of LDH-A as a therapeutic target for cancer cell killing via (i) p53/NAD(H)-dependent and (ii) p53-independent pathways</span></div><div class="casAuthors">Allison, S. J.; Knight, J. R. P.; Granchi, C.; Rani, R.; Minutolo, F.; Milner, J.; Phillips, R. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e102</span>CODEN:
                <span class="NLM_cas:coden">ONCOCT</span>;
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most cancer cells use aerobic glycolysis to fuel their growth.  The enzyme lactate dehydrogenase-A (LDH-A) is key to cancer's glycolytic phenotype, catalyzing the regeneration of NAD (NAD+) from reduced NAD (NADH) necessary to sustain glycolysis.  As such, LDH-A is a promising target for anticancer therapy.  Here we ask if the tumor suppressor p53, a major regulator of cellular metab., influences the response of cancer cells to LDH-A suppression.  LDH-A knockdown by RNA interference (RNAi) induced cancer cell death in p53 wild-type, mutant and p53-null human cancer cell lines, indicating that endogenous LDH-A promotes cancer cell survival irresp. of cancer cell p53 status.  Unexpectedly, however, we uncovered a novel role for p53 in the regulation of cancer cell NAD+ and its reduced form NADH.  Thus, LDH-A silencing by RNAi, or its inhibition using a small-mol. inhibitor, resulted in a p53-dependent increase in the cancer cell ratio of NADH:NAD+.  This effect was specific for p53+/+ cancer cells and correlated with (i) reduced activity of NAD+-dependent deacetylase sirtuin 1 (SIRT1) and (ii) an increase in acetylated p53, a known target of SIRT1 deacetylation activity.  In addn., activation of the redox-sensitive anticancer drug EO9 was enhanced selectively in p53+/+ cancer cells, attributable to increased activity of NAD(P)H-dependent oxidoreductase NQO1 (NAD(P)H quinone oxidoreductase 1).  Suppressing LDH-A increased EO9-induced DNA damage in p53+/+ cancer cells, but importantly had no additive effect in non-cancer cells.  Our results identify a unique strategy by which the NADH/NAD+ cellular redox status can be modulated in a cancer-specific, p53-dependent manner and we show that this can impact upon the activity of important NAD(H)-dependent enzymes.  To summarize, this work indicates two distinct mechanisms by which suppressing LDH-A could potentially be used to kill cancer cells selectively, (i) through induction of apoptosis, irresp. of cancer cell p53 status and (ii) as a part of a combinatorial approach with redox-sensitive anticancer drugs via a novel p53/NAD(H)-dependent mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoshoz1s34Q6bVg90H21EOLACvtfcHk0lhFRDwvR8zpDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvF2lsr0%253D&md5=33778934ca3ace8b84c81ab9199d2ba2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Foncsis.2014.16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Foncsis.2014.16%26sid%3Dliteratum%253Aachs%26aulast%3DAllison%26aufirst%3DS.%2BJ.%26aulast%3DKnight%26aufirst%3DJ.%2BR.%2BP.%26aulast%3DGranchi%26aufirst%3DC.%26aulast%3DRani%26aufirst%3DR.%26aulast%3DMinutolo%26aufirst%3DF.%26aulast%3DMilner%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DR.%2BM.%26atitle%3DIdentification%2520of%2520LDH-A%2520as%2520a%2520therapeutic%2520target%2520for%2520cancer%2520cell%2520killing%2520via%2520%2528i%2529%2520p53%252FNAD%2528H%2529-dependent%2520and%2520%2528ii%2529%2520p53-independent%2520pathways%26jtitle%3DOncogenesis%26date%3D2014%26volume%3D3%26doi%3D10.1038%2Foncsis.2014.16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouw, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinavahi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deck, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royer, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Jagt, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semenza, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span> <span> </span><span class="NLM_article-title">Inhibition of lactate dehydrogenase A induces oxidative stress and inhibits tumor progression</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">2037</span>– <span class="NLM_lpage">2042</span>, <span class="refDoi"> DOI: 10.1073/pnas.0914433107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1073%2Fpnas.0914433107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=20133848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFCit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=2037-2042&author=A.+Leauthor=C.+R.+Cooperauthor=A.+M.+Gouwauthor=R.+Dinavahiauthor=A.+Maitraauthor=L.+M.+Deckauthor=R.+E.+Royerauthor=D.+L.+Vander+Jagtauthor=G.+L.+Semenzaauthor=C.+V.+Dang&title=Inhibition+of+lactate+dehydrogenase+A+induces+oxidative+stress+and+inhibits+tumor+progression&doi=10.1073%2Fpnas.0914433107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lactate dehydrogenase A induces oxidative stress and inhibits tumor progression</span></div><div class="casAuthors">Le, Anne; Cooper, Charles R.; Gouw, Arvin M.; Dinavahi, Ramani; Maitra, Anirban; Deck, Lorraine M.; Royer, Robert E.; Vander Jagt, David L.; Semenza, Gregg L.; Dang, Chi V.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2037-2042, S2037/1-S2037/12</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the result of genetic alterations and tumor hypoxia, many cancer cells avidly take up glucose and generate lactate through lactate dehydrogenase A (LDHA), which is encoded by a target gene of c-Myc and hypoxia-inducible factor (HIF-1).  Previous studies with redn. of LDHA expression indicate that LDHA is involved in tumor initiation, but its role in tumor maintenance and progression has not been established.  Furthermore, how redn. of LDHA expression by interference or antisense RNA inhibits tumorigenesis is not well understood.  Here, we report that redn. of LDHA by siRNA or its inhibition by a small-mol. inhibitor (FX11 [3-dihydroxy-6-methyl-7-(phenylmethyl)-4-propylnaphthalene-1-carboxylic acid]) reduced ATP levels and induced significant oxidative stress and cell death that could be partially reversed by the antioxidant N-acetylcysteine (NAC).  Furthermore, we document that FX11 inhibited the progression of sizable human lymphoma and pancreatic cancer xenografts.  When used in combination with the NAD+ synthesis inhibitor FK866, FX11 induced lymphoma regression.  Hence, inhibition of LDHA with FX11 is an achievable and tolerable treatment for LDHA-dependent tumors.  Our studies document a therapeutical approach to the Warburg effect and demonstrate that oxidative stress and metabolic phenotyping of cancers are crit. aspects of cancer biol. to consider for the therapeutical targeting of cancer energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1h1YikQKAn7Vg90H21EOLACvtfcHk0lhFRDwvR8zpDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFCit7g%253D&md5=714af2ec2bc79ffcba57d6987c52a87a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0914433107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0914433107%26sid%3Dliteratum%253Aachs%26aulast%3DLe%26aufirst%3DA.%26aulast%3DCooper%26aufirst%3DC.%2BR.%26aulast%3DGouw%26aufirst%3DA.%2BM.%26aulast%3DDinavahi%26aufirst%3DR.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DDeck%26aufirst%3DL.%2BM.%26aulast%3DRoyer%26aufirst%3DR.%2BE.%26aulast%3DVander%2BJagt%26aufirst%3DD.%2BL.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26aulast%3DDang%26aufirst%3DC.%2BV.%26atitle%3DInhibition%2520of%2520lactate%2520dehydrogenase%2520A%2520induces%2520oxidative%2520stress%2520and%2520inhibits%2520tumor%2520progression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D2037%26epage%3D2042%26doi%3D10.1073%2Fpnas.0914433107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span> <span> </span><span class="NLM_article-title">Knockdown of lactate dehydrogenase A suppresses tumor growth and metastasis of human hepatocellular carcinoma</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">3898</span>– <span class="NLM_lpage">3910</span>, <span class="refDoi"> DOI: 10.1111/j.1742-4658.2012.08748.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1111%2Fj.1742-4658.2012.08748.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=22897481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVehtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2012&pages=3898-3910&author=S.+L.+Shengauthor=J.+J.+Liuauthor=Y.+H.+Daiauthor=X.+G.+Sunauthor=X.+P.+Xiongauthor=G.+Huang&title=Knockdown+of+lactate+dehydrogenase+A+suppresses+tumor+growth+and+metastasis+of+human+hepatocellular+carcinoma&doi=10.1111%2Fj.1742-4658.2012.08748.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Knockdown of lactate dehydrogenase A suppresses tumor growth and metastasis of human hepatocellular carcinoma</span></div><div class="casAuthors">Sheng, Shi L.; Liu, Jian J.; Dai, Yun H.; Sun, Xiao G.; Xiong, Xiao P.; Huang, Gang</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3898-3910</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">In previous studies, lactate dehydrogenase A (LDHA) was identified as one of the leading genes that promote the proliferative and tumorigenic potential of malignancies.  However, less definitive evidence was reported in hepatocellular carcinoma (HCC) cells.  Furthermore, the role of LDHA in promoting metastasis of HCC, and its possible mechanism, is not clear.  In this study, RNA interference (RNAi) mediated by lentiviral vectors (which induce strong down-regulation of gene expression) was used to analyze the role of LDHA in tumor growth and metastasis in HCC.  We performed transient and stable RNAi knockdowns of LDHA in HCCLM3 cells, a line that over-expresses LDHA and has a high metastatic potential.  Our studies reveal that previously unidentified effects of LHDA may mediate tumor growth and metastatic effects in HCC.  First, HCC cell lines over-express LDHA.  Second, LDHA inhibition results in increased apoptosis via prodn. of reactive oxygen species in HCCLM3 cells.  Thus, LDHA knockdown resulted in significant redn. in metastatic potential in a xenograft mouse model.  Furthermore, we found that FAK,MMP-2, VEGF and E-cadherin proteins contribute to inhibitory effects on metastasis in HCC cells.  These studies have important implications for understanding the mechanisms by which LDHA promotes tumor growth and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ1-P-Ac3M97Vg90H21EOLACvtfcHk0lgLhfj9HsD8dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVehtLrL&md5=572370ddb4964a870be366d495cdb260</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2012.08748.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2012.08748.x%26sid%3Dliteratum%253Aachs%26aulast%3DSheng%26aufirst%3DS.%2BL.%26aulast%3DLiu%26aufirst%3DJ.%2BJ.%26aulast%3DDai%26aufirst%3DY.%2BH.%26aulast%3DSun%26aufirst%3DX.%2BG.%26aulast%3DXiong%26aufirst%3DX.%2BP.%26aulast%3DHuang%26aufirst%3DG.%26atitle%3DKnockdown%2520of%2520lactate%2520dehydrogenase%2520A%2520suppresses%2520tumor%2520growth%2520and%2520metastasis%2520of%2520human%2520hepatocellular%2520carcinoma%26jtitle%3DFEBS%2520J.%26date%3D2012%26volume%3D279%26spage%3D3898%26epage%3D3910%26doi%3D10.1111%2Fj.1742-4658.2012.08748.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purkey, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robarge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Nagro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pusapati, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salphati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanderbilt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daemen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T.</span></span> <span> </span><span class="NLM_article-title">Metabolic plasticity underpins innate and acquired resistance to LDHA inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">786</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1038%2Fnchembio.2143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=27479743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=779-786&author=A.+Boudreauauthor=H.+E.+Purkeyauthor=A.+Hitzauthor=K.+Robargeauthor=D.+Petersonauthor=S.+Labadieauthor=M.+Kwongauthor=R.+Hongauthor=M.+Gaoauthor=C.+Del+Nagroauthor=R.+Pusapatiauthor=S.+Maauthor=L.+Salphatiauthor=J.+Pangauthor=A.+Zhouauthor=T.+Laiauthor=Y.+Liauthor=Z.+Chenauthor=B.+Weiauthor=I.+Yenauthor=S.+Siderisauthor=M.+McClelandauthor=R.+Firesteinauthor=L.+Corsonauthor=A.+Vanderbiltauthor=S.+Williamsauthor=A.+Daemenauthor=M.+Belvinauthor=C.+Eigenbrotauthor=P.+K.+Jacksonauthor=S.+Malekauthor=G.+Hatzivassiliouauthor=D.+Sampathauthor=M.+Evangelistaauthor=T.+O%E2%80%99Brien&title=Metabolic+plasticity+underpins+innate+and+acquired+resistance+to+LDHA+inhibition&doi=10.1038%2Fnchembio.2143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic plasticity underpins innate and acquired resistance to LDHA inhibition</span></div><div class="casAuthors">Boudreau, Aaron; Purkey, Hans E.; Hitz, Anna; Robarge, Kirk; Peterson, David; Labadie, Sharada; Kwong, Mandy; Hong, Rebecca; Gao, Min; Del Nagro, Christopher; Pusapati, Raju; Ma, Shuguang; Salphati, Laurent; Pang, Jodie; Zhou, Aihe; Lai, Tommy; Li, Yingjie; Chen, Zhongguo; Wei, Binqing; Yen, Ivana; Sideris, Steve; McCleland, Mark; Firestein, Ron; Corson, Laura; Vanderbilt, Alex; Williams, Simon; Daemen, Anneleen; Belvin, Marcia; Eigenbrot, Charles; Jackson, Peter K.; Malek, Shiva; Hatzivassiliou, Georgia; Sampath, Deepak; Evangelista, Marie; O'Brien, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">779-786</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Metabolic reprogramming in tumors represents a potential therapeutic target.  Herein we used shRNA depletion and a novel lactate dehydrogenase (LDHA) inhibitor, GNE-140, to probe the role of LDHA in tumor growth in vitro and in vivo.  In MIA PaCa-2 human pancreatic cells, LDHA inhibition rapidly affected global metab., although cell death only occurred after 2 d of continuous LDHA inhibition.  Pancreatic cell lines that utilize oxidative phosphorylation (OXPHOS) rather than glycolysis were inherently resistant to GNE-140, but could be resensitized to GNE-140 with the OXPHOS inhibitor phenformin.  Acquired resistance to GNE-140 was driven by activation of the AMPK-mTOR-S6K signaling pathway, which led to increased OXPHOS, and inhibitors targeting this pathway could prevent resistance.  Thus, combining an LDHA inhibitor with compds. targeting the mitochondrial or AMPK-S6K signaling axis may not only broaden the clin. utility of LDHA inhibitors beyond glycolytically dependent tumors but also reduce the emergence of resistance to LDHA inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4wdebEpeIa7Vg90H21EOLACvtfcHk0lgLhfj9HsD8dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu73K&md5=28abbd7f174b422b7207a5f80352ce3e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2143%26sid%3Dliteratum%253Aachs%26aulast%3DBoudreau%26aufirst%3DA.%26aulast%3DPurkey%26aufirst%3DH.%2BE.%26aulast%3DHitz%26aufirst%3DA.%26aulast%3DRobarge%26aufirst%3DK.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DKwong%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DDel%2BNagro%26aufirst%3DC.%26aulast%3DPusapati%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DSalphati%26aufirst%3DL.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DA.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DMcCleland%26aufirst%3DM.%26aulast%3DFirestein%26aufirst%3DR.%26aulast%3DCorson%26aufirst%3DL.%26aulast%3DVanderbilt%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DS.%26aulast%3DDaemen%26aufirst%3DA.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DJackson%26aufirst%3DP.%2BK.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DEvangelista%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26atitle%3DMetabolic%2520plasticity%2520underpins%2520innate%2520and%2520acquired%2520resistance%2520to%2520LDHA%2520inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D779%26epage%3D786%26doi%3D10.1038%2Fnchembio.2143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Pierre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, P.</span></span> <span> </span><span class="NLM_article-title">Attenuation of LDH-A expression uncovers a link between glycolysis, mitochondrial physiology, and tumor maintenance</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">425</span>– <span class="NLM_lpage">434</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2006.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.ccr.2006.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=16766262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BD28Xmt1eksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2006&pages=425-434&author=V.+R.+Fantinauthor=J.+St-Pierreauthor=P.+Leder&title=Attenuation+of+LDH-A+expression+uncovers+a+link+between+glycolysis%2C+mitochondrial+physiology%2C+and+tumor+maintenance&doi=10.1016%2Fj.ccr.2006.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Attenuation of LDH-A expression uncovers a link between glycolysis, mitochondrial physiology, and tumor maintenance</span></div><div class="casAuthors">Fantin, Valeria R.; St-Pierre, Julie; Leder, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">425-434</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Alterations in cellular metab. are among the most consistent hallmarks of cancer.  Herein we have investigated the relationship between increased aerobic lactate prodn. and mitochondrial physiol. in tumor cells.  To diminish the ability of malignant cells to metabolize pyruvate to lactate, lactate dehydrogenase A (LDH-A) levels were knocked down by means of LDH-A short hairpin RNAs.  Redn. in LDH-A activity resulted in stimulation of mitochondrial respiration and decrease of mitochondrial membrane potential.  It also compromised the ability of these tumor cells to proliferate under hypoxia.  The tumorigenicity of the LDH-A-deficient cells was severely diminished, and this phenotype was reversed by complementation with the human ortholog LDH-A protein.  These results demonstrate that LDH-A plays a key role in tumor maintenance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDQMn1zZZKr7Vg90H21EOLACvtfcHk0ljj_EO-NILDKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xmt1eksLk%253D&md5=6ba9ae345a18d1b97a58a729d0bd35f6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2006.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2006.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DSt-Pierre%26aufirst%3DJ.%26aulast%3DLeder%26aufirst%3DP.%26atitle%3DAttenuation%2520of%2520LDH-A%2520expression%2520uncovers%2520a%2520link%2520between%2520glycolysis%252C%2520mitochondrial%2520physiology%252C%2520and%2520tumor%2520maintenance%26jtitle%3DCancer%2520Cell%26date%3D2006%26volume%3D9%26spage%3D425%26epage%3D434%26doi%3D10.1016%2Fj.ccr.2006.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanai, J.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kats, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhargava, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billiard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorkiewicz, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schatzman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousamra, M.,  2nd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandolfi, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukhatme, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, P.</span></span> <span> </span><span class="NLM_article-title">Targeting lactate dehydrogenase--a inhibits tumorigenesis and tumor progression in mouse models of lung cancer and impacts tumor-initiating cells</span>. <i>Cell Metabol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">809</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2014.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.cmet.2014.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24726384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVWrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=795-809&author=H.+Xieauthor=J.-i.+Hanaiauthor=J.-G.+Renauthor=L.+Katsauthor=K.+Burgessauthor=P.+Bhargavaauthor=S.+Signorettiauthor=J.+Billiardauthor=K.+J.+Duffyauthor=A.+Grantauthor=X.+Wangauthor=P.+K.+Lorkiewiczauthor=S.+Schatzmanauthor=M.+Bousamraauthor=A.+N.+Laneauthor=R.+M.+Higashiauthor=T.+W.+M.+Fanauthor=P.+P.+Pandolfiauthor=V.+P.+Sukhatmeauthor=P.+Seth&title=Targeting+lactate+dehydrogenase%2D%2Da+inhibits+tumorigenesis+and+tumor+progression+in+mouse+models+of+lung+cancer+and+impacts+tumor-initiating+cells&doi=10.1016%2Fj.cmet.2014.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Lactate Dehydrogenase-A Inhibits Tumorigenesis and Tumor Progression in Mouse Models of Lung Cancer and Impacts Tumor-Initiating Cells</span></div><div class="casAuthors">Xie, Han; Hanai, Jun-ichi; Ren, Jian-Guo; Kats, Lev; Burgess, Kerri; Bhargava, Parul; Signoretti, Sabina; Billiard, Julia; Duffy, Kevin J.; Grant, Aaron; Wang, Xiaoen; Lorkiewicz, Pawel K.; Schatzman, Sabrina; Bousamra, Michael; Lane, Andrew N.; Higashi, Richard M.; Fan, Teresa W. M.; Pandolfi, Pier Paolo; Sukhatme, Vikas P.; Seth, Pankaj</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">795-809</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lactate dehydrogenase-A (LDH-A) enzyme catalyzes the interconversion of pyruvate and lactate, is upregulated in human cancers, and is assocd. with aggressive tumor outcomes.  Here we use an inducible murine model and demonstrate that inactivation of LDH-A in mouse models of NSCLC driven by oncogenic K-RAS or EGFR leads to decreased tumorigenesis and disease regression in established tumors.  We also show that abrogation of LDH-A results in reprogramming of pyruvate metab., with decreased lactic fermn. in vitro, in vivo, and ex vivo.  This was accompanied by reactivation of mitochondrial function in vitro, but not in vivo or ex vivo.  Finally, using a specific small mol. LDH-A inhibitor, we demonstrated that LDH-A is essential for cancer-initiating cell survival and proliferation.  Thus, LDH-A can be a viable therapeutic target for NSCLC, including cancer stem cell-dependent drug-resistant tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNiWJNOeWwmLVg90H21EOLACvtfcHk0ljj_EO-NILDKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVWrtLY%253D&md5=d80624d60895f5a6c01106841ebd1af9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2014.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2014.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DH.%26aulast%3DHanai%26aufirst%3DJ.-i.%26aulast%3DRen%26aufirst%3DJ.-G.%26aulast%3DKats%26aufirst%3DL.%26aulast%3DBurgess%26aufirst%3DK.%26aulast%3DBhargava%26aufirst%3DP.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DBilliard%26aufirst%3DJ.%26aulast%3DDuffy%26aufirst%3DK.%2BJ.%26aulast%3DGrant%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLorkiewicz%26aufirst%3DP.%2BK.%26aulast%3DSchatzman%26aufirst%3DS.%26aulast%3DBousamra%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DA.%2BN.%26aulast%3DHigashi%26aufirst%3DR.%2BM.%26aulast%3DFan%26aufirst%3DT.%2BW.%2BM.%26aulast%3DPandolfi%26aufirst%3DP.%2BP.%26aulast%3DSukhatme%26aufirst%3DV.%2BP.%26aulast%3DSeth%26aufirst%3DP.%26atitle%3DTargeting%2520lactate%2520dehydrogenase--a%2520inhibits%2520tumorigenesis%2520and%2520tumor%2520progression%2520in%2520mouse%2520models%2520of%2520lung%2520cancer%2520and%2520impacts%2520tumor-initiating%2520cells%26jtitle%3DCell%2520Metabol.%26date%3D2014%26volume%3D19%26spage%3D795%26epage%3D809%26doi%3D10.1016%2Fj.cmet.2014.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouw, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinavahi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deck, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royer, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Jagt, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semenza, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span> <span> </span><span class="NLM_article-title">Inhibition of lactate dehydrogenase A induces oxidative stress and inhibits tumor progression</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">2037</span>– <span class="NLM_lpage">2042</span>, <span class="refDoi"> DOI: 10.1073/pnas.0914433107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1073%2Fpnas.0914433107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=20133848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFCit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=2037-2042&author=A.+Leauthor=C.+R.+Cooperauthor=A.+M.+Gouwauthor=R.+Dinavahiauthor=A.+Maitraauthor=L.+M.+Deckauthor=R.+E.+Royerauthor=D.+L.+Vander+Jagtauthor=G.+L.+Semenzaauthor=C.+V.+Dang&title=Inhibition+of+lactate+dehydrogenase+A+induces+oxidative+stress+and+inhibits+tumor+progression&doi=10.1073%2Fpnas.0914433107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lactate dehydrogenase A induces oxidative stress and inhibits tumor progression</span></div><div class="casAuthors">Le, Anne; Cooper, Charles R.; Gouw, Arvin M.; Dinavahi, Ramani; Maitra, Anirban; Deck, Lorraine M.; Royer, Robert E.; Vander Jagt, David L.; Semenza, Gregg L.; Dang, Chi V.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2037-2042, S2037/1-S2037/12</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the result of genetic alterations and tumor hypoxia, many cancer cells avidly take up glucose and generate lactate through lactate dehydrogenase A (LDHA), which is encoded by a target gene of c-Myc and hypoxia-inducible factor (HIF-1).  Previous studies with redn. of LDHA expression indicate that LDHA is involved in tumor initiation, but its role in tumor maintenance and progression has not been established.  Furthermore, how redn. of LDHA expression by interference or antisense RNA inhibits tumorigenesis is not well understood.  Here, we report that redn. of LDHA by siRNA or its inhibition by a small-mol. inhibitor (FX11 [3-dihydroxy-6-methyl-7-(phenylmethyl)-4-propylnaphthalene-1-carboxylic acid]) reduced ATP levels and induced significant oxidative stress and cell death that could be partially reversed by the antioxidant N-acetylcysteine (NAC).  Furthermore, we document that FX11 inhibited the progression of sizable human lymphoma and pancreatic cancer xenografts.  When used in combination with the NAD+ synthesis inhibitor FK866, FX11 induced lymphoma regression.  Hence, inhibition of LDHA with FX11 is an achievable and tolerable treatment for LDHA-dependent tumors.  Our studies document a therapeutical approach to the Warburg effect and demonstrate that oxidative stress and metabolic phenotyping of cancers are crit. aspects of cancer biol. to consider for the therapeutical targeting of cancer energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1h1YikQKAn7Vg90H21EOLACvtfcHk0ljj_EO-NILDKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFCit7g%253D&md5=714af2ec2bc79ffcba57d6987c52a87a</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0914433107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0914433107%26sid%3Dliteratum%253Aachs%26aulast%3DLe%26aufirst%3DA.%26aulast%3DCooper%26aufirst%3DC.%2BR.%26aulast%3DGouw%26aufirst%3DA.%2BM.%26aulast%3DDinavahi%26aufirst%3DR.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DDeck%26aufirst%3DL.%2BM.%26aulast%3DRoyer%26aufirst%3DR.%2BE.%26aulast%3DVander%2BJagt%26aufirst%3DD.%2BL.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26aulast%3DDang%26aufirst%3DC.%2BV.%26atitle%3DInhibition%2520of%2520lactate%2520dehydrogenase%2520A%2520induces%2520oxidative%2520stress%2520and%2520inhibits%2520tumor%2520progression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D2037%26epage%3D2042%26doi%3D10.1073%2Fpnas.0914433107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit13b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Granchi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacomelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuccinardi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannaccini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucacchini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">León, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giovannetti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palchaudhuri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvaresi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minutolo, F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-hydroxyindole-based inhibitors of human lactate dehydrogenase isoform A (LDH-A) as starvation agents against cancer cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1021/jm101007q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101007q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3MXitFWrtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1599-1612&author=C.+Granchiauthor=S.+Royauthor=C.+Giacomelliauthor=M.+Macchiaauthor=T.+Tuccinardiauthor=A.+Martinelliauthor=M.+Lanzaauthor=L.+Bettiauthor=G.+Giannacciniauthor=A.+Lucacchiniauthor=N.+Funelauthor=L.+G.+Le%C3%B3nauthor=E.+Giovannettiauthor=G.+J.+Petersauthor=R.+Palchaudhuriauthor=E.+C.+Calvaresiauthor=P.+J.+Hergenrotherauthor=F.+Minutolo&title=Discovery+of+N-hydroxyindole-based+inhibitors+of+human+lactate+dehydrogenase+isoform+A+%28LDH-A%29+as+starvation+agents+against+cancer+cells&doi=10.1021%2Fjm101007q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Hydroxyindole-Based Inhibitors of Human Lactate Dehydrogenase Isoform A (LDH-A) as Starvation Agents against Cancer Cells</span></div><div class="casAuthors">Granchi, Carlotta; Roy, Sarabindu; Giacomelli, Chiara; Macchia, Marco; Tuccinardi, Tiziano; Martinelli, Adriano; Lanza, Mario; Betti, Laura; Giannaccini, Gino; Lucacchini, Antonio; Funel, Nicola; Leon, Leticia G.; Giovannetti, Elisa; Peters, Godefridus J.; Palchaudhuri, Rahul; Calvaresi, Emilia C.; Hergenrother, Paul J.; Minutolo, Filippo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1599-1612</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Highly invasive tumor cells are characterized by a metabolic switch, known as the Warburg effect, from "normal" oxidative phosphorylation to increased glycolysis even under sufficiently oxygenated conditions.  This dependence on glycolysis also confers a growth advantage to cells present in hypoxic regions of the tumor.  One of the key enzymes involved in glycolysis, the muscle isoform of lactate dehydrogenase (LDH-A), is overexpressed by metastatic cancer cells and is linked to the vitality of tumors in hypoxia.  This enzyme may be considered as a potential target for new anticancer agents, since its inhibition cuts cancer energetic and anabolic supply, thus reducing the metastatic and invasive potential of cancer cells.  We have discovered new and efficient N-hydroxyindole-based inhibitors of LDH-A, which are isoform-selective (over LDH-B) and competitive with both the substrate (pyruvate) and the cofactor (NADH).  The antiproliferative activity of these compds. was confirmed on a series of cancer cell lines, and they proved to be particularly effective under hypoxic conditions.  Moreover, NMR expts. showed that these compds. are able to reduce the glucose-to-lactate conversion inside the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97RT2SmH8ZLVg90H21EOLACvtfcHk0li_mUS6kNzLYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXitFWrtrs%253D&md5=7fbe13667567f20567c23087cb66fbf5</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1021%2Fjm101007q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101007q%26sid%3Dliteratum%253Aachs%26aulast%3DGranchi%26aufirst%3DC.%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DGiacomelli%26aufirst%3DC.%26aulast%3DMacchia%26aufirst%3DM.%26aulast%3DTuccinardi%26aufirst%3DT.%26aulast%3DMartinelli%26aufirst%3DA.%26aulast%3DLanza%26aufirst%3DM.%26aulast%3DBetti%26aufirst%3DL.%26aulast%3DGiannaccini%26aufirst%3DG.%26aulast%3DLucacchini%26aufirst%3DA.%26aulast%3DFunel%26aufirst%3DN.%26aulast%3DLe%25C3%25B3n%26aufirst%3DL.%2BG.%26aulast%3DGiovannetti%26aufirst%3DE.%26aulast%3DPeters%26aufirst%3DG.%2BJ.%26aulast%3DPalchaudhuri%26aufirst%3DR.%26aulast%3DCalvaresi%26aufirst%3DE.%2BC.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26aulast%3DMinutolo%26aufirst%3DF.%26atitle%3DDiscovery%2520of%2520N-hydroxyindole-based%2520inhibitors%2520of%2520human%2520lactate%2520dehydrogenase%2520isoform%2520A%2520%2528LDH-A%2529%2520as%2520starvation%2520agents%2520against%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1599%26epage%3D1612%26doi%3D10.1021%2Fjm101007q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squillace, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Commodore, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenfield, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miret, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D. C.</span></span> <span> </span><span class="NLM_article-title">Fragment growing and linking lead to novel nanomolar lactate dehydrogenase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1023</span>– <span class="NLM_lpage">1040</span>, <span class="refDoi"> DOI: 10.1021/jm3014844</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3014844" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXms1OlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1023-1040&author=A.+Kohlmannauthor=S.+G.+Zechauthor=F.+Liauthor=T.+Zhouauthor=R.+M.+Squillaceauthor=L.+Commodoreauthor=M.+T.+Greenfieldauthor=X.+Luauthor=D.+P.+Millerauthor=W.-S.+Huangauthor=J.+Qiauthor=R.+M.+Thomasauthor=Y.+Wangauthor=S.+Zhangauthor=R.+Doddauthor=S.+Liuauthor=R.+Xuauthor=Y.+Xuauthor=J.+J.+Miretauthor=V.+Riveraauthor=T.+Clacksonauthor=W.+C.+Shakespeareauthor=X.+Zhuauthor=D.+C.+Dalgarno&title=Fragment+growing+and+linking+lead+to+novel+nanomolar+lactate+dehydrogenase+inhibitors&doi=10.1021%2Fjm3014844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Growing and Linking Lead to Novel Nanomolar Lactate Dehydrogenase Inhibitors</span></div><div class="casAuthors">Kohlmann, Anna; Zech, Stephan G.; Li, Feng; Zhou, Tianjun; Squillace, Rachel M.; Commodore, Lois; Greenfield, Matthew T.; Lu, Xiaohui; Miller, David P.; Huang, Wei-Sheng; Qi, Jiwei; Thomas, R. Mathew; Wang, Yihan; Zhang, Sen; Dodd, Rory; Liu, Shuangying; Xu, Rongsong; Xu, Yongjin; Miret, Juan J.; Rivera, Victor; Clackson, Tim; Shakespeare, William C.; Zhu, Xiaotian; Dalgarno, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1023-1040</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lactate dehydrogenase A (LDH-A) catalyzes the interconversion of lactate and pyruvate in the glycolysis pathway.  Cancer cells rely heavily on glycolysis instead of oxidative phosphorylation to generate ATP, a phenomenon known as the Warburg effect.  The inhibition of LDH-A by small mols. is therefore of interest for potential cancer treatments.  We describe the identification and optimization of LDH-A inhibitors by fragment-based drug discovery.  We applied ligand based NMR screening to identify low affinity fragments binding to LDH-A.  The dissocn. consts. (Kd) and enzyme inhibition (IC50) of fragment hits were measured by surface plasmon resonance (SPR) and enzyme assays, resp.  The binding modes of selected fragments were investigated by X-ray crystallog.  Fragment growing and linking, followed by chem. optimization, resulted in nanomolar LDH-A inhibitors that demonstrated stoichiometric binding to LDH-A.  Selected mols. inhibited lactate prodn. in cells, suggesting target-specific inhibition in cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pFkSoe2_ALVg90H21EOLACvtfcHk0li_mUS6kNzLYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXms1OlsA%253D%253D&md5=3a6b0b46318ae2d6c2a7c34e1575ada6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm3014844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3014844%26sid%3Dliteratum%253Aachs%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DZech%26aufirst%3DS.%2BG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DSquillace%26aufirst%3DR.%2BM.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DGreenfield%26aufirst%3DM.%2BT.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DMiller%26aufirst%3DD.%2BP.%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DDodd%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMiret%26aufirst%3DJ.%2BJ.%26aulast%3DRivera%26aufirst%3DV.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26atitle%3DFragment%2520growing%2520and%2520linking%2520lead%2520to%2520novel%2520nanomolar%2520lactate%2520dehydrogenase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1023%26epage%3D1040%26doi%3D10.1021%2Fjm3014844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchlow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrosek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogtherr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel lactate dehydrogenase A inhibitors by fragment-based lead generation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">3285</span>– <span class="NLM_lpage">3306</span>, <span class="refDoi"> DOI: 10.1021/jm201734r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201734r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvVSgtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3285-3306&author=R.+A.+Wardauthor=C.+Brassingtonauthor=A.+L.+Breezeauthor=A.+Caputoauthor=S.+Critchlowauthor=G.+Daviesauthor=L.+Goodwinauthor=G.+Hassallauthor=R.+Greenwoodauthor=G.+A.+Holdgateauthor=M.+Mrosekauthor=R.+A.+Normanauthor=S.+Pearsonauthor=J.+Tartauthor=J.+A.+Tuckerauthor=M.+Vogtherrauthor=D.+Whittakerauthor=J.+Wingfieldauthor=J.+Winterauthor=K.+Hudson&title=Design+and+synthesis+of+novel+lactate+dehydrogenase+A+inhibitors+by+fragment-based+lead+generation&doi=10.1021%2Fjm201734r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Lactate Dehydrogenase A Inhibitors by Fragment-Based Lead Generation</span></div><div class="casAuthors">Ward, Richard A.; Brassington, Claire; Breeze, Alexander L.; Caputo, Alessandro; Critchlow, Susan; Davies, Gareth; Goodwin, Louise; Hassall, Giles; Greenwood, Ryan; Holdgate, Geoffrey A.; Mrosek, Michael; Norman, Richard A.; Pearson, Stuart; Tart, Jonathan; Tucker, Julie A.; Vogtherr, Martin; Whittaker, David; Wingfield, Jonathan; Winter, Jon; Hudson, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3285-3306</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lactate dehydrogenase A (LDHA) catalyzes the conversion of pyruvate to lactate, utilizing NADH as a cofactor.  It has been identified as a potential therapeutic target in the area of cancer metab.  In this manuscript we report our progress using fragment-based lead generation (FBLG), assisted by x-ray crystallog. to develop small mol. LDHA inhibitors.  Fragment hits were identified through NMR and SPR screening and optimized into lead compds. with nanomolar binding affinities via fragment linking.  Also reported is their modification into cellular active compds. suitable for target validation work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH8QbdIUXNvbVg90H21EOLACvtfcHk0lgO4fLSI8jGSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvVSgtLs%253D&md5=d638e7a0db361c212ee084d5c05ee70c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm201734r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201734r%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DBrassington%26aufirst%3DC.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26aulast%3DCaputo%26aufirst%3DA.%26aulast%3DCritchlow%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DGoodwin%26aufirst%3DL.%26aulast%3DHassall%26aufirst%3DG.%26aulast%3DGreenwood%26aufirst%3DR.%26aulast%3DHoldgate%26aufirst%3DG.%2BA.%26aulast%3DMrosek%26aufirst%3DM.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DPearson%26aufirst%3DS.%26aulast%3DTart%26aufirst%3DJ.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DVogtherr%26aufirst%3DM.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWingfield%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DJ.%26aulast%3DHudson%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520lactate%2520dehydrogenase%2520A%2520inhibitors%2520by%2520fragment-based%2520lead%2520generation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3285%26epage%3D3306%26doi%3D10.1021%2Fjm201734r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Billiard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennison, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colón, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greshock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSurdy-Freed, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orband-Miller, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waitt, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooster, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, K. J.</span></span> <span> </span><span class="NLM_article-title">Quinoline 3-sulfonamides inhibit lactate dehydrogenase A and reverse aerobic glycolysis in cancer cells</span>. <i>Canc. Metabol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1186/2049-3002-1-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1186%2F2049-3002-1-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24280423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A280%3ADC%252BC2c3ntFCluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=19-36&author=J.+Billiardauthor=J.+B.+Dennisonauthor=J.+Briandauthor=R.+S.+Annanauthor=D.+Chaiauthor=M.+Col%C3%B3nauthor=C.+S.+Dodsonauthor=S.+A.+Gilbertauthor=J.+Greshockauthor=J.+Jingauthor=H.+Luauthor=J.+E.+McSurdy-Freedauthor=L.+A.+Orband-Millerauthor=G.+B.+Millsauthor=C.+J.+Quinnauthor=J.+L.+Schneckauthor=G.+F.+Scottauthor=A.+N.+Shawauthor=G.+M.+Waittauthor=R.+F.+Woosterauthor=K.+J.+Duffy&title=Quinoline+3-sulfonamides+inhibit+lactate+dehydrogenase+A+and+reverse+aerobic+glycolysis+in+cancer+cells&doi=10.1186%2F2049-3002-1-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Quinoline 3-sulfonamides inhibit lactate dehydrogenase A and reverse aerobic glycolysis in cancer cells</span></div><div class="casAuthors">Billiard Julia; Dennison Jennifer B; Briand Jacques; Annan Roland S; Chai Deping; Colon Mariela; Dodson Christopher S; Gilbert Seth A; Greshock Joel; Jing Junping; Lu Hong; McSurdy-Freed Jeanelle E; Orband-Miller Lisa A; Mills Gordon B; Quinn Chad J; Schneck Jessica L; Scott Gilbert F; Shaw Anthony N; Waitt Gregory M; Wooster Richard F; Duffy Kevin J</div><div class="citationInfo"><span class="NLM_cas:title">Cancer & metabolism</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>
        ISSN:<span class="NLM_cas:issn">2049-3002</span>.
    </div><div class="casAbstract">BACKGROUND:  Most normal cells in the presence of oxygen utilize glucose for mitochondrial oxidative phosphorylation.  In contrast, many cancer cells rapidly convert glucose to lactate in the cytosol, a process termed aerobic glycolysis.  This glycolytic phenotype is enabled by lactate dehydrogenase (LDH), which catalyzes the inter-conversion of pyruvate and lactate.  The purpose of this study was to identify and characterize potent and selective inhibitors of LDHA.  METHODS:  High throughput screening and lead optimization were used to generate inhibitors of LDHA enzymatic activity.  Effects of these inhibitors on metabolism were evaluated using cell-based lactate production, oxygen consumption, and 13C NMR spectroscopy assays.  Changes in comprehensive metabolic profile, cell proliferation, and apoptosis were assessed upon compound treatment.  RESULTS:  3-((3-carbamoyl-7-(3,5-dimethylisoxazol-4-yl)-6-methoxyquinolin-4-yl) amino) benzoic acid was identified as an NADH-competitive LDHA inhibitor.  Lead optimization yielded molecules with LDHA inhibitory potencies as low as 2 nM and 10 to 80-fold selectivity over LDHB.  Molecules in this family rapidly and profoundly inhibited lactate production rates in multiple cancer cell lines including hepatocellular and breast carcinomas.  Consistent with selective inhibition of LDHA, the most sensitive breast cancer cell lines to lactate inhibition in hypoxic conditions were cells with low expression of LDHB.  Our inhibitors increased rates of oxygen consumption in hepatocellular carcinoma cells at doses up to 3 microM, while higher concentrations directly inhibited mitochondrial function.  Analysis of more than 500 metabolites upon LDHA inhibition in Snu398 cells revealed that intracellular concentrations of glycolysis and citric acid cycle intermediates were increased, consistent with enhanced Krebs cycle activity and blockage of cytosolic glycolysis.  Treatment with these compounds also potentiated PKM2 activity and promoted apoptosis in Snu398 cells.  CONCLUSIONS:  Rapid chemical inhibition of LDHA by these quinoline 3-sulfonamids led to profound metabolic alterations and impaired cell survival in carcinoma cells making it a compelling strategy for treating solid tumors that rely on aerobic glycolysis for survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTY101Isa9tY-Lt9HypzAgQfW6udTcc2eYIR9emgsXXG7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3ntFCluw%253D%253D&md5=588288defc04ed3927e4c4fdb891d6cd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2F2049-3002-1-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2049-3002-1-19%26sid%3Dliteratum%253Aachs%26aulast%3DBilliard%26aufirst%3DJ.%26aulast%3DDennison%26aufirst%3DJ.%2BB.%26aulast%3DBriand%26aufirst%3DJ.%26aulast%3DAnnan%26aufirst%3DR.%2BS.%26aulast%3DChai%26aufirst%3DD.%26aulast%3DCol%25C3%25B3n%26aufirst%3DM.%26aulast%3DDodson%26aufirst%3DC.%2BS.%26aulast%3DGilbert%26aufirst%3DS.%2BA.%26aulast%3DGreshock%26aufirst%3DJ.%26aulast%3DJing%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DMcSurdy-Freed%26aufirst%3DJ.%2BE.%26aulast%3DOrband-Miller%26aufirst%3DL.%2BA.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DQuinn%26aufirst%3DC.%2BJ.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DScott%26aufirst%3DG.%2BF.%26aulast%3DShaw%26aufirst%3DA.%2BN.%26aulast%3DWaitt%26aufirst%3DG.%2BM.%26aulast%3DWooster%26aufirst%3DR.%2BF.%26aulast%3DDuffy%26aufirst%3DK.%2BJ.%26atitle%3DQuinoline%25203-sulfonamides%2520inhibit%2520lactate%2520dehydrogenase%2520A%2520and%2520reverse%2520aerobic%2520glycolysis%2520in%2520cancer%2520cells%26jtitle%3DCanc.%2520Metabol.%26date%3D2013%26volume%3D1%26spage%3D19%26epage%3D36%26doi%3D10.1186%2F2049-3002-1-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannetti, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderPorten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Q.</span></span> <span> </span><span class="NLM_article-title">Identification of 2-amino-5-aryl-pyrazines as inhibitors of human lactate dehydrogenase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5533</span>– <span class="NLM_lpage">5539</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.08.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.bmcl.2013.08.060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24012183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSrsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=5533-5539&author=B.+P.+Fauberauthor=P.+S.+Dragovichauthor=J.+Chenauthor=L.+B.+Corsonauthor=C.+Z.+Dingauthor=C.+Eigenbrotauthor=A.+M.+Giannettiauthor=T.+Hunsakerauthor=S.+Labadieauthor=Y.+Liuauthor=Y.+Liuauthor=S.+Malekauthor=D.+Petersonauthor=K.+Pittsauthor=S.+Siderisauthor=M.+Ultschauthor=E.+VanderPortenauthor=J.+Wangauthor=B.+Weiauthor=I.+Yenauthor=Q.+Yue&title=Identification+of+2-amino-5-aryl-pyrazines+as+inhibitors+of+human+lactate+dehydrogenase&doi=10.1016%2Fj.bmcl.2013.08.060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of 2-amino-5-aryl-pyrazines as inhibitors of human lactate dehydrogenase</span></div><div class="casAuthors">Fauber, Benjamin P.; Dragovich, Peter S.; Chen, Jinhua; Corson, Laura B.; Ding, Charles Z.; Eigenbrot, Charles; Giannetti, Anthony M.; Hunsaker, Thomas; Labadie, Sharada; Liu, Yichin; Liu, Yingchun; Malek, Shiva; Peterson, David; Pitts, Keith; Sideris, Steve; Ultsch, Mark; VanderPorten, Erica; Wang, Jing; Wei, BinQing; Yen, Ivana; Yue, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5533-5539</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A 2-amino-5-aryl-pyrazine, e.g. I [R = 2-O2CC6H4, 3-O2CC6H4, 4-O2CC6H4, etc.] was identified as an inhibitor of human lactate dehydrogenase A (LDHA) via a biochem. screening campaign.  Biochem. and biophys. expts. demonstrated that the compd. specifically interacted with human LDHA.  Structural variation of the screening hit resulted in improvements in LDHA biochem. inhibition and pharmacokinetic properties.  A crystal structure of an improved compd. bound to human LDHA was also obtained and it explained many of the obsd. structure-activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowVTwZ-VbwxrVg90H21EOLACvtfcHk0lgO4fLSI8jGSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSrsrvI&md5=80d750198bae55b872c58b771af64b9d</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.08.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.08.060%26sid%3Dliteratum%253Aachs%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCorson%26aufirst%3DL.%2BB.%26aulast%3DDing%26aufirst%3DC.%2BZ.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DHunsaker%26aufirst%3DT.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DPitts%26aufirst%3DK.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DVanderPorten%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DYue%26aufirst%3DQ.%26atitle%3DIdentification%2520of%25202-amino-5-aryl-pyrazines%2520as%2520inhibitors%2520of%2520human%2520lactate%2520dehydrogenase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D5533%26epage%3D5539%26doi%3D10.1016%2Fj.bmcl.2013.08.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit17b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannetti, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purkey, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderPorten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">Identification of substituted 2-thio-6-oxo-1,6-duhydropyrimidines as inhibitors of human lactate dehydrogenase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3186</span>– <span class="NLM_lpage">3194</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.bmcl.2013.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=23628333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvV2itr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3186-3194&author=P.+S.+Dragovichauthor=B.+P.+Fauberauthor=L.+B.+Corsonauthor=C.+Z.+Dingauthor=C.+Eigenbrotauthor=H.+Geauthor=A.+M.+Giannettiauthor=T.+Hunsakerauthor=S.+Labadieauthor=Y.+Liuauthor=S.+Malekauthor=B.+Panauthor=D.+Petersonauthor=K.+Pittsauthor=H.+E.+Purkeyauthor=S.+Siderisauthor=M.+Ultschauthor=E.+VanderPortenauthor=B.+Weiauthor=Q.+Xuauthor=I.+Yenauthor=Q.+Yueauthor=H.+Zhangauthor=X.+Zhang&title=Identification+of+substituted+2-thio-6-oxo-1%2C6-duhydropyrimidines+as+inhibitors+of+human+lactate+dehydrogenase&doi=10.1016%2Fj.bmcl.2013.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17bR"><div class="casContent"><span class="casTitleNuber">17b</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of substituted 2-thio-6-oxo-1,6-dihydropyrimidines as inhibitors of human lactate dehydrogenase</span></div><div class="casAuthors">Dragovich, Peter S.; Fauber, Benjamin P.; Corson, Laura B.; Ding, Charles Z.; Eigenbrot, Charles; Ge, HongXiu; Giannetti, Anthony M.; Hunsaker, Thomas; Labadie, Sharada; Liu, Yichin; Malek, Shiva; Pan, Borlan; Peterson, David; Pitts, Keith; Purkey, Hans E.; Sideris, Steve; Ultsch, Mark; VanderPorten, Erica; Wei, BinQing; Xu, Qing; Yen, Ivana; Yue, Qin; Zhang, Huihui; Zhang, Xuying</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3186-3194</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel 2-thio-6-oxo-1,6-dihydropyrimidine-contg. inhibitor of human lactate dehydrogenase (LDH) was identified by high-throughput screening (IC50 = 8.1 μM).  Biochem., surface plasmon resonance, and satn. transfer difference NMR expts. indicated that the compd. specifically assocd. with human LDHA in a manner that required simultaneous binding of the NADH co-factor.  Structural variation of the screening hit resulted in significant improvements in LDHA biochem. inhibition activity (best IC50 = 0.48 μM).  A crystal structure of an optimized compd. bound to human LDHA was obtained and explained many of the obsd. structure-activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e6oSsqyPHrVg90H21EOLACvtfcHk0lj3Xa-FYhMVQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvV2itr8%253D&md5=5367ac82c7f5583c8faaf899f966e621</span></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DCorson%26aufirst%3DL.%2BB.%26aulast%3DDing%26aufirst%3DC.%2BZ.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DHunsaker%26aufirst%3DT.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DPitts%26aufirst%3DK.%26aulast%3DPurkey%26aufirst%3DH.%2BE.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DVanderPorten%26aufirst%3DE.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DYue%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DIdentification%2520of%2520substituted%25202-thio-6-oxo-1%252C6-duhydropyrimidines%2520as%2520inhibitors%2520of%2520human%2520lactate%2520dehydrogenase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3186%26epage%3D3194%26doi%3D10.1016%2Fj.bmcl.2013.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span> <span> </span><span class="NLM_article-title">Development of novel human lactate dehydrogenase A inhibitors: high-throughput screening, synthesis, and biological evaluations</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.05.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.ejmech.2019.05.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=31129449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKqtbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=105-115&author=Y.+Zhouauthor=P.+Taoauthor=M.+Wangauthor=P.+Xuauthor=W.+Luauthor=P.+Leiauthor=Q.+You&title=Development+of+novel+human+lactate+dehydrogenase+A+inhibitors%3A+high-throughput+screening%2C+synthesis%2C+and+biological+evaluations&doi=10.1016%2Fj.ejmech.2019.05.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of novel human lactate dehydrogenase A inhibitors: high-throughput screening, synthesis, and biological evaluations</span></div><div class="casAuthors">Zhou, Yuan; Tao, Pingde; Wang, Meigui; Xu, Peng; Lu, Wei; Lei, Pan; You, Qiuyun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-115</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Human lactate dehydrogenase A (LDHA) plays a crit. role in the glycolytic process, making the enzyme an ideal of anti-cancer drug target.  Herein, we report the discovery of novel potent LDHA inhibitors by screening an inhouse library.  The hit-to-lead modification enabled us to identify compd. 24c(I), which inhibited LDHA activity with an EC50 value of 90 nM, and reduced MiaPaCa-2 cancer cell proliferation with an IC50 value of 2.1 μM.  In line with the in vitro anticancer activity, I suppressed the tumor growth at a dose of 10 mg/kg in a MiaPaCa-2 cells xenograft model, but with little effect to the mice wt.  Moreover, I strongly inhibited MiaPaCa-2 cell colonies formation, induced MiaPaCa-2 cell apoptosis, and arrested MiaPaCa-2 cell cycle at G2 phase.  In addn., the mitochondrial bioenergetics anal. suggested that I could reprogram cancer cell metabolic pathways from glycolysis to oxidn. phosphorylation, which verified by decreasing the extracellular acidification rates and lactate formation, and increasing oxygen consumption rate in cancer cell.  All these results indicate I is a promising metabolic modulator for the anticancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7PmbP3IKU5LVg90H21EOLACvtfcHk0lj3Xa-FYhMVQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKqtbnM&md5=f619bbf9198b20163a4a4601813a1af0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.05.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.05.033%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLei%26aufirst%3DP.%26aulast%3DYou%26aufirst%3DQ.%26atitle%3DDevelopment%2520of%2520novel%2520human%2520lactate%2520dehydrogenase%2520A%2520inhibitors%253A%2520high-throughput%2520screening%252C%2520synthesis%252C%2520and%2520biological%2520evaluations%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D177%26spage%3D105%26epage%3D115%26doi%3D10.1016%2Fj.ejmech.2019.05.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheff, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouznetsova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benavides, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuenstner, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luci, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dranow, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulikowski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flint, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darley-Usmar, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of potent, cell-active pyrazole-based inhibitors of lactate dehydrogenase (LDH)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9184</span>– <span class="NLM_lpage">9204</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00941</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00941" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslOkurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9184-9204&author=G.+Raiauthor=K.+R.+Brimacombeauthor=B.+T.+Mottauthor=D.+J.+Urbanauthor=X.+Huauthor=S.-M.+Yangauthor=T.+D.+Leeauthor=D.+M.+Cheffauthor=J.+Kouznetsovaauthor=G.+A.+Benavidesauthor=K.+Pohidaauthor=E.+J.+Kuenstnerauthor=D.+K.+Luciauthor=C.+M.+Lukacsauthor=D.+R.+Daviesauthor=D.+M.+Dranowauthor=H.+Zhuauthor=G.+Sulikowskiauthor=W.+J.+Mooreauthor=G.+M.+Stottauthor=A.+J.+Flintauthor=M.+D.+Hallauthor=V.+M.+Darley-Usmarauthor=L.+M.+Neckersauthor=C.+V.+Dangauthor=A.+G.+Watersonauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+J.+Maloney&title=Discovery+and+optimization+of+potent%2C+cell-active+pyrazole-based+inhibitors+of+lactate+dehydrogenase+%28LDH%29&doi=10.1021%2Facs.jmedchem.7b00941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Potent, Cell-Active Pyrazole-Based Inhibitors of Lactate Dehydrogenase (LDH)</span></div><div class="casAuthors">Rai, Ganesha; Brimacombe, Kyle R.; Mott, Bryan T.; Urban, Daniel J.; Hu, Xin; Yang, Shyh-Ming; Lee, Tobie D.; Cheff, Dorian M.; Kouznetsova, Jennifer; Benavides, Gloria A.; Pohida, Katie; Kuenstner, Eric J.; Luci, Diane K.; Lukacs, Christine M.; Davies, Douglas R.; Dranow, David M.; Zhu, Hu; Sulikowski, Gary; Moore, William J.; Stott, Gordon M.; Flint, Andrew J.; Hall, Matthew D.; Darley-Usmar, Victor M.; Neckers, Leonard M.; Dang, Chi V.; Waterson, Alex G.; Simeonov, Anton; Jadhav, Ajit; Maloney, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9184-9204</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the discovery and medicinal chem. optimization of a novel series of pyrazole-based inhibitors of human lactate dehydrogenase (LDH).  Utilization of a quant. high-throughput screening paradigm facilitated hit identification, while structure-based design and multiparameter optimization enabled the development of compds. with potent enzymic and cell-based inhibition of LDH enzymic activity.  Lead compds. such as 63 exhibit low nM inhibition of both LDHA and LDHB, submicromolar inhibition of lactate prodn., and inhibition of glycolysis in MiaPaCa2 pancreatic cancer and A673 sarcoma cells.  Moreover, robust target engagement of LDHA by lead compds. was demonstrated using the cellular thermal shift assay (CETSA), and drug-target residence time was detd. via SPR.  Anal. of these data suggests that drug-target residence time (off-rate) may be an important attribute to consider for obtaining potent cell-based inhibition of this cancer metab. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrejnE11w2acLVg90H21EOLACvtfcHk0lh4oKB4mgnQUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslOkurfP&md5=a2df6432e54403e18abc6078fc30e692</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00941%26sid%3Dliteratum%253Aachs%26aulast%3DRai%26aufirst%3DG.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DS.-M.%26aulast%3DLee%26aufirst%3DT.%2BD.%26aulast%3DCheff%26aufirst%3DD.%2BM.%26aulast%3DKouznetsova%26aufirst%3DJ.%26aulast%3DBenavides%26aufirst%3DG.%2BA.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DKuenstner%26aufirst%3DE.%2BJ.%26aulast%3DLuci%26aufirst%3DD.%2BK.%26aulast%3DLukacs%26aufirst%3DC.%2BM.%26aulast%3DDavies%26aufirst%3DD.%2BR.%26aulast%3DDranow%26aufirst%3DD.%2BM.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DSulikowski%26aufirst%3DG.%26aulast%3DMoore%26aufirst%3DW.%2BJ.%26aulast%3DStott%26aufirst%3DG.%2BM.%26aulast%3DFlint%26aufirst%3DA.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DDarley-Usmar%26aufirst%3DV.%2BM.%26aulast%3DNeckers%26aufirst%3DL.%2BM.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520potent%252C%2520cell-active%2520pyrazole-based%2520inhibitors%2520of%2520lactate%2520dehydrogenase%2520%2528LDH%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9184%26epage%3D9204%26doi%3D10.1021%2Facs.jmedchem.7b00941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issaq, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oshima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgart, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edessa, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benavides, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squadrito, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yohe, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowdy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Rodado, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shern, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larion, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helman, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishna, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darley-Usmar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heske, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting glycolysis through inhibition of lactate dehydrogenase impairs tumor growth in preclinical models of Ewing sarcoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">5060</span>– <span class="NLM_lpage">5073</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-19-0217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1158%2F0008-5472.CAN-19-0217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=31431459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslKrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=5060-5073&author=C.+Yeungauthor=A.+E.+Gibsonauthor=S.+H.+Issaqauthor=N.+Oshimaauthor=J.+T.+Baumgartauthor=L.+D.+Edessaauthor=G.+Raiauthor=D.+J.+Urbanauthor=M.+S.+Johnsonauthor=G.+A.+Benavidesauthor=G.+L.+Squadritoauthor=M.+E.+Yoheauthor=H.+Leiauthor=S.+Eldridgeauthor=J.+Hamreauthor=T.+Dowdyauthor=V.+Ruiz-Rodadoauthor=A.+Litaauthor=A.+Mendozaauthor=J.+F.+Shernauthor=M.+Larionauthor=L.+J.+Helmanauthor=G.+M.+Stottauthor=M.+C.+Krishnaauthor=M.+D.+Hallauthor=V.+Darley-Usmarauthor=L.+M.+Neckersauthor=C.+M.+Heske&title=Targeting+glycolysis+through+inhibition+of+lactate+dehydrogenase+impairs+tumor+growth+in+preclinical+models+of+Ewing+sarcoma&doi=10.1158%2F0008-5472.can-19-0217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting glycolysis through inhibition of lactate dehydrogenase impairs tumor growth in preclinical models of ewing sarcoma</span></div><div class="casAuthors">Yeung, Choh; Gibson, Anna E.; Issaq, Sameer H.; Oshima, Nobu; Baumgart, Joshua T.; Edessa, Leah D.; Rai, Ganesha; Urban, Daniel J.; Johnson, Michelle S.; Benavides, Gloria A.; Squadrito, Giuseppe L.; Yohe, Marielle E.; Lei, Haiyan; Eldridge, Sandy; Hamre, John, III; Dowdy, Tyrone; Ruiz-Rodado, Victor; Lita, Adrian; Mendoza, Arnulfo; Shern, Jack F.; Larion, Mioara; Helman, Lee J.; Stott, Gordon M.; Krishna, Murali C.; Hall, Matthew D.; Darley-Usmar, Victor; Neckers, Leonard M.; Heske, Christine M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5060-5073</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Altered cellular metab., including an increased dependence on aerobic glycolysis, is a hallmark of cancer.  Despite the fact that this observation was first made nearly a century ago, effective therapeutic targeting of glycolysis in cancer has remained elusive.  One potentially promising approach involves targeting the glycolytic enzyme lactate dehydrogenase (LDH), which is overexpressed and plays a crit. role in several cancers.  Here, we used a novel class of LDH inhibitors to demonstrate, for the first time, that Ewing sarcoma cells are exquisitely sensitive to inhibition of LDH.  EWS-FLI1, the oncogenic driver of Ewing sarcoma, regulated LDH A (LDHA) expression.  Genetic depletion of LDHA inhibited proliferation of Ewing sarcoma cells and induced apoptosis, phenocopying pharmacol. inhibition of LDH.  LDH inhibitors affected Ewing sarcoma cell viability both in vitro and in vivo by reducing glycolysis.  I.v. administration of LDH inhibitors resulted in the greatest intratumoral drug accumulation, inducing tumor cell death and reducing tumor growth.  The major dose-limiting toxicity obsd. was hemolysis, indicating that a narrow therapeutic window exists for these compds.  Taken together, these data suggest that targeting glycolysis through inhibition of LDH should be further investigated as a potential therapeutic approach for cancers such as Ewing sarcoma that exhibit oncogene-dependent expression of LDH and increased glycolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Bje4xjUjsbVg90H21EOLACvtfcHk0lh4oKB4mgnQUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslKrsrk%253D&md5=2a0ca91f802511e160d86ab4d33744ae</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-19-0217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-19-0217%26sid%3Dliteratum%253Aachs%26aulast%3DYeung%26aufirst%3DC.%26aulast%3DGibson%26aufirst%3DA.%2BE.%26aulast%3DIssaq%26aufirst%3DS.%2BH.%26aulast%3DOshima%26aufirst%3DN.%26aulast%3DBaumgart%26aufirst%3DJ.%2BT.%26aulast%3DEdessa%26aufirst%3DL.%2BD.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DM.%2BS.%26aulast%3DBenavides%26aufirst%3DG.%2BA.%26aulast%3DSquadrito%26aufirst%3DG.%2BL.%26aulast%3DYohe%26aufirst%3DM.%2BE.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DEldridge%26aufirst%3DS.%26aulast%3DHamre%26aufirst%3DJ.%26aulast%3DDowdy%26aufirst%3DT.%26aulast%3DRuiz-Rodado%26aufirst%3DV.%26aulast%3DLita%26aufirst%3DA.%26aulast%3DMendoza%26aufirst%3DA.%26aulast%3DShern%26aufirst%3DJ.%2BF.%26aulast%3DLarion%26aufirst%3DM.%26aulast%3DHelman%26aufirst%3DL.%2BJ.%26aulast%3DStott%26aufirst%3DG.%2BM.%26aulast%3DKrishna%26aufirst%3DM.%2BC.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DDarley-Usmar%26aufirst%3DV.%26aulast%3DNeckers%26aufirst%3DL.%2BM.%26aulast%3DHeske%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520glycolysis%2520through%2520inhibition%2520of%2520lactate%2520dehydrogenase%2520impairs%2520tumor%2520growth%2520in%2520preclinical%2520models%2520of%2520Ewing%2520sarcoma%26jtitle%3DCancer%2520Res.%26date%3D2019%26volume%3D79%26spage%3D5060%26epage%3D5073%26doi%3D10.1158%2F0008-5472.can-19-0217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Oshima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brender, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benavides, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squadrito, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crooks, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrimp, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moses, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusewitt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hathaway, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trepel, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklar, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norenberg, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linehan, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darley-Usmar, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishna, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L. M.</span></span> <span> </span><span class="NLM_article-title">Dynamic imaging of LDH inhibition in tumors reveals rapid in vivo metabolic rewiring and vulnerability to combination therapy</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1798</span>– <span class="NLM_lpage">1810</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2020.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.celrep.2020.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=32049011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFektb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=1798-1810&author=N.+Oshimaauthor=R.+Ishidaauthor=S.+Kishimotoauthor=K.+Beebeauthor=J.+R.+Brenderauthor=K.+Yamamotoauthor=D.+Urbanauthor=G.+Raiauthor=M.+S.+Johnsonauthor=G.+Benavidesauthor=G.+L.+Squadritoauthor=D.+Crooksauthor=J.+Jacksonauthor=A.+Joshiauthor=B.+T.+Mottauthor=J.+H.+Shrimpauthor=M.+A.+Mosesauthor=M.-J.+Leeauthor=A.+Yunoauthor=T.+D.+Leeauthor=X.+Huauthor=T.+Andersonauthor=D.+Kusewittauthor=H.+H.+Hathawayauthor=A.+Jadhavauthor=D.+Picardauthor=J.+B.+Trepelauthor=J.+B.+Mitchellauthor=G.+M.+Stottauthor=W.+Mooreauthor=A.+Simeonovauthor=L.+A.+Sklarauthor=J.+P.+Norenbergauthor=W.+M.+Linehanauthor=D.+J.+Maloneyauthor=C.+V.+Dangauthor=A.+G.+Watersonauthor=M.+Hallauthor=V.+M.+Darley-Usmarauthor=M.+C.+Krishnaauthor=L.+M.+Neckers&title=Dynamic+imaging+of+LDH+inhibition+in+tumors+reveals+rapid+in+vivo+metabolic+rewiring+and+vulnerability+to+combination+therapy&doi=10.1016%2Fj.celrep.2020.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21aR"><div class="casContent"><span class="casTitleNuber">21a</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Imaging of LDH Inhibition in Tumors Reveals Rapid In Vivo Metabolic Rewiring and Vulnerability to Combination Therapy</span></div><div class="casAuthors">Oshima, Nobu; Ishida, Ryo; Kishimoto, Shun; Beebe, Kristin; Brender, Jeffrey R.; Yamamoto, Kazutoshi; Urban, Daniel; Rai, Ganesha; Johnson, Michelle S.; Benavides, Gloria; Squadrito, Giuseppe L.; Crooks, Dan; Jackson, Joseph; Joshi, Abhinav; Mott, Bryan T.; Shrimp, Jonathan H.; Moses, Michael A.; Lee, Min-Jung; Yuno, Akira; Lee, Tobie D.; Hu, Xin; Anderson, Tamara; Kusewitt, Donna; Hathaway, Helen H.; Jadhav, Ajit; Picard, Didier; Trepel, Jane B.; Mitchell, James B.; Stott, Gordon M.; Moore, William; Simeonov, Anton; Sklar, Larry A.; Norenberg, Jeffrey P.; Linehan, W. Marston; Maloney, David J.; Dang, Chi V.; Waterson, Alex G.; Hall, Matthew; Darley-Usmar, Victor M.; Krishna, Murali C.; Neckers, Leonard M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1798-1810.e4</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The reliance of many cancers on aerobic glycolysis has stimulated efforts to develop lactate dehydrogenase (LDH) inhibitors.  However, despite significant efforts, LDH inhibitors (LDHi) with sufficient specificity and in vivo activity to det. whether LDH is a feasible drug target are lacking.  We describe an LDHi with potent, on-target, in vivo activity.  Using hyperpolarized magnetic resonance spectroscopic imaging (HP-MRSI), we demonstrate in vivo LDH inhibition in two glycolytic cancer models, MIA PaCa-2 and HT29, and we correlate depth and duration of LDH inhibition with direct anti-tumor activity.  HP-MRSI also reveals a metabolic rewiring that occurs in vivo within 30 min of LDH inhibition, wherein pyruvate in a tumor is redirected toward mitochondrial metab.  Using HP-MRSI, we show that inhibition of mitochondrial complex 1 rapidly redirects tumor pyruvate toward lactate.  Inhibition of both mitochondrial complex 1 and LDH suppresses metabolic plasticity, causing metabolic quiescence in vitro and tumor growth inhibition in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXo-hnKU_CkrVg90H21EOLACvtfcHk0likYncvkm-zDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFektb0%253D&md5=9cb03f50f7d5a21ee5a6b1c9b1dc2f0a</span></div><a href="/servlet/linkout?suffix=cit21a&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2020.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2020.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DOshima%26aufirst%3DN.%26aulast%3DIshida%26aufirst%3DR.%26aulast%3DKishimoto%26aufirst%3DS.%26aulast%3DBeebe%26aufirst%3DK.%26aulast%3DBrender%26aufirst%3DJ.%2BR.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DUrban%26aufirst%3DD.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DJohnson%26aufirst%3DM.%2BS.%26aulast%3DBenavides%26aufirst%3DG.%26aulast%3DSquadrito%26aufirst%3DG.%2BL.%26aulast%3DCrooks%26aufirst%3DD.%26aulast%3DJackson%26aufirst%3DJ.%26aulast%3DJoshi%26aufirst%3DA.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DShrimp%26aufirst%3DJ.%2BH.%26aulast%3DMoses%26aufirst%3DM.%2BA.%26aulast%3DLee%26aufirst%3DM.-J.%26aulast%3DYuno%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DT.%2BD.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DAnderson%26aufirst%3DT.%26aulast%3DKusewitt%26aufirst%3DD.%26aulast%3DHathaway%26aufirst%3DH.%2BH.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DPicard%26aufirst%3DD.%26aulast%3DTrepel%26aufirst%3DJ.%2BB.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%26aulast%3DStott%26aufirst%3DG.%2BM.%26aulast%3DMoore%26aufirst%3DW.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DSklar%26aufirst%3DL.%2BA.%26aulast%3DNorenberg%26aufirst%3DJ.%2BP.%26aulast%3DLinehan%26aufirst%3DW.%2BM.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DHall%26aufirst%3DM.%26aulast%3DDarley-Usmar%26aufirst%3DV.%2BM.%26aulast%3DKrishna%26aufirst%3DM.%2BC.%26aulast%3DNeckers%26aufirst%3DL.%2BM.%26atitle%3DDynamic%2520imaging%2520of%2520LDH%2520inhibition%2520in%2520tumors%2520reveals%2520rapid%2520in%2520vivo%2520metabolic%2520rewiring%2520and%2520vulnerability%2520to%2520combination%2520therapy%26jtitle%3DCell%2520Rep.%26date%3D2020%26volume%3D30%26spage%3D1798%26epage%3D1810%26doi%3D10.1016%2Fj.celrep.2020.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit21b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Quon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, L. B.</span></span> <span> </span><span class="NLM_article-title">Redox debt leads to metabolic bankruptcy in tumors</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">359</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2020.02.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.trecan.2020.02.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=32348730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFGqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=359-361&author=E.+Quonauthor=M.+L.+Hartauthor=L.+B.+Sullivan&title=Redox+debt+leads+to+metabolic+bankruptcy+in+tumors&doi=10.1016%2Fj.trecan.2020.02.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21bR"><div class="casContent"><span class="casTitleNuber">21b</span><div class="casTitle"><span class="NLM_cas:atitle">Redox Debt Leads to Metabolic Bankruptcy in Tumors</span></div><div class="casAuthors">Quon, Evan; Hart, Madeleine L.; Sullivan, Lucas B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">359-361</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lactate dehydrogenase (LDH) accounts for the fermentative component of aerobic glycolysis, a near ubiquitous metabolic alteration in cancer.  Recently, Oshima et al. developed a bioavailable LDH inhibitor that decreases tumor growth in mice and functions synergistically with mitochondrial respiration inhibitors.  These findings suggest a cooperative mechanism of action that targets redox homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmIKCGJVoNqrVg90H21EOLACvtfcHk0lj-CcQ2UecQNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFGqsbg%253D&md5=84b71c5b8130ea1cedaf315d8a5f6129</span></div><a href="/servlet/linkout?suffix=cit21b&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2020.02.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2020.02.012%26sid%3Dliteratum%253Aachs%26aulast%3DQuon%26aufirst%3DE.%26aulast%3DHart%26aufirst%3DM.%2BL.%26aulast%3DSullivan%26aufirst%3DL.%2BB.%26atitle%3DRedox%2520debt%2520leads%2520to%2520metabolic%2520bankruptcy%2520in%2520tumors%26jtitle%3DTrends%2520Cancer%26date%3D2020%26volume%3D6%26spage%3D359%26epage%3D361%26doi%3D10.1016%2Fj.trecan.2020.02.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Cañaveras, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gromer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spolski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenbergen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fioravanti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebina-Shibuya, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleck, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gattinoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, W. J.</span></span> <span> </span><span class="NLM_article-title">Lactate dehydrogenase inhibition synergizes with IL-21 to promote CD8<sup>+</sup> T cell stemness and antitumor immunity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">6047</span>, <span class="refDoi"> DOI: 10.1073/pnas.1920413117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1073%2Fpnas.1920413117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=32123114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1WltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=6047&author=D.+Hermansauthor=S.+Gautamauthor=J.+C.+Garc%C3%ADa-Ca%C3%B1averasauthor=D.+Gromerauthor=S.+Mitraauthor=R.+Spolskiauthor=P.+Liauthor=S.+Christensenauthor=R.+Nguyenauthor=J.-X.+Linauthor=J.+Ohauthor=N.+Duauthor=S.+Veenbergenauthor=J.+Fioravantiauthor=R.+Ebina-Shibuyaauthor=C.+Bleckauthor=L.+M.+Neckersauthor=J.+D.+Rabinowitzauthor=L.+Gattinoniauthor=W.+J.+Leonard&title=Lactate+dehydrogenase+inhibition+synergizes+with+IL-21+to+promote+CD8%2B+T+cell+stemness+and+antitumor+immunity&doi=10.1073%2Fpnas.1920413117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Lactate dehydrogenase inhibition synergizes with IL-21 to promote CD8+ T cell stemness and antitumor immunity</span></div><div class="casAuthors">Hermans, Dalton; Gautam, Sanjivan; Garcia-Canaveras, Juan C.; Gromer, Daniel; Mitra, Suman; Spolski, Rosanne; Li, Peng; Christensen, Stephen; Nguyen, Rosa; Lin, Jian-Xin; Oh, Jangsuk; Du, Ning; Veenbergen, Sharon; Fioravanti, Jessica; Ebina-Shibuya, Risa; Bleck, Christopher; Neckers, Leonard M.; Rabinowitz, Joshua D.; Gattinoni, Luca; Leonard, Warren J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6047-6055</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Interleukin (IL)-2 and IL-21 dichotomously shape CD8+ T cell differentiation.  IL-2 drives terminal differentiation, generating cells that are poorly effective against tumors, whereas IL-21 promotes stem cell memory T cells (TSCM) and antitumor responses.  Here we investigated the role of metabolic programming in the developmental differences induced by these cytokines.  IL-2 promoted effector-like metab. and aerobic glycolysis, robustly inducing lactate dehydrogenase (LDH) and lactate prodn., whereas IL-21 maintained a metabolically quiescent state dependent on oxidative phosphorylation.  LDH inhibition rewired IL-2-induced effects, promoting pyruvate entry into the tricarboxylic acid cycle and inhibiting terminal effector and exhaustion programs, including mRNA expression of members of the NR4A family of nuclear receptors, as well as Prdm1 and Xbp1.  While deletion of Ldha prevented development of cells with antitumor effector function, transient LDH inhibition enhanced the generation of memory cells capable of triggering robust antitumor responses after adoptive transfer.  LDH inhibition did not significantly affect IL-21-induced metab. but caused major transcriptomic changes, including the suppression of IL-21-induced exhaustion markers LAG3, PD1, 2B4, and TIM3.  LDH inhibition combined with IL-21 increased the formation of TSCM cells, resulting in more profound antitumor responses and prolonged host survival.  These findings indicate a pivotal role for LDH in modulating cytokine-mediated T cell differentiation and underscore the therapeutic potential of transiently inhibiting LDH during adoptive T cell-based immunotherapy, with an unanticipated cooperative antitumor effect of LDH inhibition and IL-21.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9FMceWMhkTLVg90H21EOLACvtfcHk0lj-CcQ2UecQNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1WltLc%253D&md5=a2305b1ae79995f18bd1d6b93ff153f2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1920413117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1920413117%26sid%3Dliteratum%253Aachs%26aulast%3DHermans%26aufirst%3DD.%26aulast%3DGautam%26aufirst%3DS.%26aulast%3DGarc%25C3%25ADa-Ca%25C3%25B1averas%26aufirst%3DJ.%2BC.%26aulast%3DGromer%26aufirst%3DD.%26aulast%3DMitra%26aufirst%3DS.%26aulast%3DSpolski%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DChristensen%26aufirst%3DS.%26aulast%3DNguyen%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DJ.-X.%26aulast%3DOh%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DN.%26aulast%3DVeenbergen%26aufirst%3DS.%26aulast%3DFioravanti%26aufirst%3DJ.%26aulast%3DEbina-Shibuya%26aufirst%3DR.%26aulast%3DBleck%26aufirst%3DC.%26aulast%3DNeckers%26aufirst%3DL.%2BM.%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26aulast%3DGattinoni%26aufirst%3DL.%26aulast%3DLeonard%26aufirst%3DW.%2BJ.%26atitle%3DLactate%2520dehydrogenase%2520inhibition%2520synergizes%2520with%2520IL-21%2520to%2520promote%2520CD8%252B%2520T%2520cell%2520stemness%2520and%2520antitumor%2520immunity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2020%26volume%3D117%26spage%3D6047%26doi%3D10.1073%2Fpnas.1920413117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span> <span> </span><span class="NLM_article-title">Room-temperature, copper-free Sonogashira reactions facilitated by air-stable, monoligated precatalyst [DTBNpP] Pd(crotyl)Cl</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">12985</span>– <span class="NLM_lpage">12998</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b01868</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b01868" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVOgs7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=12985-12998&author=K.+Pohidaauthor=D.+J.+Maloneyauthor=B.+T.+Mottauthor=G.+Rai&title=Room-temperature%2C+copper-free+Sonogashira+reactions+facilitated+by+air-stable%2C+monoligated+precatalyst+%5BDTBNpP%5D+Pd%28crotyl%29Cl&doi=10.1021%2Facsomega.8b01868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Room-Temperature, Copper-Free Sonogashira Reactions Facilitated by Air-Stable, Monoligated Precatalyst [DTBNpP] Pd(crotyl)Cl</span></div><div class="casAuthors">Pohida, Katherine; Maloney, David J.; Mott, Bryan T.; Rai, Ganesha</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">12985-12998</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel application of [DTBNpP] Pd(crotyl)Cl (DTBNpP = di-tert-butylneopentylphosphine) (P2), an air-stable, com.-available palladium precatalyst that allows rapid access to a mono-ligated state, has been identified for room temp., copper-free Sonogashira couplings of challenging aryl bromides and alkynes.  The mild reaction conditions with TMP in DMSO afford up to 97% yields, excellent functional group tolerability, and broad reaction compatibility with access to one-pot indole formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnQWpZwmXSYLVg90H21EOLACvtfcHk0ljLc6c9WqnVKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVOgs7nI&md5=d02e5a8e6f3084003ac25f4a2a28f18c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b01868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b01868%26sid%3Dliteratum%253Aachs%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DRai%26aufirst%3DG.%26atitle%3DRoom-temperature%252C%2520copper-free%2520Sonogashira%2520reactions%2520facilitated%2520by%2520air-stable%252C%2520monoligated%2520precatalyst%2520%255BDTBNpP%255D%2520Pd%2528crotyl%2529Cl%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D12985%26epage%3D12998%26doi%3D10.1021%2Facsomega.8b01868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh-Bakht, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. P.</span></span> <span> </span><span class="NLM_article-title">Biotransformation reactions of five-membered aromatic heterocyclic rings</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1021/tx015574b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx015574b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2002&pages=269-299&author=D.+K.+Dalvieauthor=A.+S.+Kalgutkarauthor=S.+C.+Khojasteh-Bakhtauthor=R.+S.+Obachauthor=J.+P.+O%E2%80%99Donnell&title=Biotransformation+reactions+of+five-membered+aromatic+heterocyclic+rings&doi=10.1021%2Ftx015574b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Biotransformation Reactions of Five-Membered Aromatic Heterocyclic Rings</span></div><div class="casAuthors">Dalvie, Deepak K.; Kalgutkar, Amit S.; Khojasteh-Bakht, S. Cyrus; Obach, R. Scott; O'Donnell, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-299</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 278 refs. is given on the biotransformation pathways of most commonly used 5-membered arom. heterocyclic rings.  The effect of physicochem. properties such as the electronic effects of heteroatoms, aromaticity, and acidity/basicity (pKa) of the rings are correlated on their biotransformation.  The generation of reactive metabolites following oxidn. or redn. of these heterocycles and the subsequent toxicol. consequences are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpNaEz19aehbVg90H21EOLACvtfcHk0ljLc6c9WqnVKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVWrsLc%253D&md5=e8e2dfd12772f9ff2289b173bdefa590</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Ftx015574b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx015574b%26sid%3Dliteratum%253Aachs%26aulast%3DDalvie%26aufirst%3DD.%2BK.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DKhojasteh-Bakht%26aufirst%3DS.%2BC.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BP.%26atitle%3DBiotransformation%2520reactions%2520of%2520five-membered%2520aromatic%2520heterocyclic%2520rings%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2002%26volume%3D15%26spage%3D269%26epage%3D299%26doi%3D10.1021%2Ftx015574b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Purkey, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robarge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPasquale, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salphati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, A.</span></span> <span> </span><span class="NLM_article-title">Cell active hydroxylactam inhibitors of human lactate dehydrogenase with oral bioavailability in mice</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">896</span>– <span class="NLM_lpage">901</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00190</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00190" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWlsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=896-901&author=H.+E.+Purkeyauthor=K.+Robargeauthor=J.+Chenauthor=Z.+Chenauthor=L.+B.+Corsonauthor=C.+Z.+Dingauthor=A.+G.+DiPasqualeauthor=P.+S.+Dragovichauthor=C.+Eigenbrotauthor=M.+Evangelistaauthor=B.+P.+Fauberauthor=Z.+Gaoauthor=H.+Geauthor=A.+Hitzauthor=Q.+Hoauthor=S.+S.+Labadieauthor=K.+W.+Laiauthor=W.+Liuauthor=Y.+Liuauthor=C.+Liauthor=S.+Maauthor=S.+Malekauthor=T.+O%E2%80%99Brienauthor=J.+Pangauthor=D.+Petersonauthor=L.+Salphatiauthor=S.+Siderisauthor=M.+Ultschauthor=B.+Weiauthor=I.+Yenauthor=Q.+Yueauthor=H.+Zhangauthor=A.+Zhou&title=Cell+active+hydroxylactam+inhibitors+of+human+lactate+dehydrogenase+with+oral+bioavailability+in+mice&doi=10.1021%2Facsmedchemlett.6b00190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Cell Active Hydroxylactam Inhibitors of Human Lactate Dehydrogenase with Oral Bioavailability in Mice</span></div><div class="casAuthors">Purkey, Hans E.; Robarge, Kirk; Chen, Jinhua; Chen, Zhongguo; Corson, Laura B.; Ding, Charles Z.; DiPasquale, Antonio G.; Dragovich, Peter S.; Eigenbrot, Charles; Evangelista, Marie; Fauber, Benjamin P.; Gao, Zhenting; Ge, Hongxiu; Hitz, Anna; Ho, Qunh; Labadie, Sharada S.; Lai, Kwong Wah; Liu, Wenfeng; Liu, Yajing; Li, Chiho; Ma, Shuguang; Malek, Shiva; OBrien, Thomas; Pang, Jodie; Peterson, David; Salphati, Laurent; Sideris, Steve; Ultsch, Mark; Wei, BinQing; Yen, Ivana; Yue, Qin; Zhang, Huihui; Zhou, Aihe</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">896-901</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of trisubstituted hydroxylactams was identified as potent enzymic and cellular inhibitors of human lactate dehydrogenase A.  Utilizing structure-based design and phys. property optimization, multiple inhibitors were discovered with <10 μM lactate IC50 in a MiaPaca2 cell line.  Optimization of the series led to 29, a potent cell active mol. (MiaPaca2 IC50 = 0.67 μM) that also possessed good exposure when dosed orally to mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp75UNOgf2VD7Vg90H21EOLACvtfcHk0ljF4WCSZwtmmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWlsbbK&md5=60598df7cf0499ac5c3e4345c53b3770</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00190%26sid%3Dliteratum%253Aachs%26aulast%3DPurkey%26aufirst%3DH.%2BE.%26aulast%3DRobarge%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DCorson%26aufirst%3DL.%2BB.%26aulast%3DDing%26aufirst%3DC.%2BZ.%26aulast%3DDiPasquale%26aufirst%3DA.%2BG.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DEvangelista%26aufirst%3DM.%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DHitz%26aufirst%3DA.%26aulast%3DHo%26aufirst%3DQ.%26aulast%3DLabadie%26aufirst%3DS.%2BS.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DSalphati%26aufirst%3DL.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DYue%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DA.%26atitle%3DCell%2520active%2520hydroxylactam%2520inhibitors%2520of%2520human%2520lactate%2520dehydrogenase%2520with%2520oral%2520bioavailability%2520in%2520mice%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D896%26epage%3D901%26doi%3D10.1021%2Facsmedchemlett.6b00190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asawa, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakharov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallgren, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klumpp-Thomas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coussens, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span> <span> </span><span class="NLM_article-title">A widely-applicable high-throughput cellular thermal shift assay (CETSA) using split Nano Luciferase</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">9472</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-27834-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1038%2Fs41598-018-27834-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=29930256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FgsFyktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=9472&author=N.+J.+Martinezauthor=R.+R.+Asawaauthor=M.+G.+Cyrauthor=A.+Zakharovauthor=D.+J.+Urbanauthor=J.+S.+Rothauthor=E.+Wallgrenauthor=C.+Klumpp-Thomasauthor=N.+P.+Coussensauthor=G.+Raiauthor=S.-M.+Yangauthor=M.+D.+Hallauthor=J.+J.+Maruganauthor=A.+Simeonovauthor=M.+J.+Henderson&title=A+widely-applicable+high-throughput+cellular+thermal+shift+assay+%28CETSA%29+using+split+Nano+Luciferase&doi=10.1038%2Fs41598-018-27834-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A widely-applicable high-throughput cellular thermal shift assay (CETSA) using split Nano Luciferase</span></div><div class="casAuthors">Martinez Natalia J; Asawa Rosita R; Cyr Matthew G; Zakharov Alexey; Urban Daniel J; Roth Jacob S; Wallgren Eric; Klumpp-Thomas Carleen; Coussens Nathan P; Rai Ganesha; Yang Shyh-Ming; Hall Matthew D; Marugan Juan J; Simeonov Anton; Henderson Mark J</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9472</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Assessment of the interactions between a drug and its protein target in a physiologically relevant cellular environment constitutes a major challenge in the pre-clinical drug discovery space.  The Cellular Thermal Shift Assay (CETSA) enables such an assessment by quantifying the changes in the thermal stability of proteins upon ligand binding in intact cells.  Here, we present the development and validation of a homogeneous, standardized, target-independent, and high-throughput (384- and 1536-well formats) CETSA platform that uses a split Nano Luciferase approach (SplitLuc CETSA).  The broad applicability of the assay was demonstrated for diverse targets, and its performance was compared with independent biochemical and cell-based readouts using a set of well-characterized inhibitors.  Moreover, we investigated the utility of the platform as a primary assay for high-throughput screening.  The SplitLuc CETSA presented here enables target engagement studies for medium and high-throughput applications.  Additionally, it provides a rapid assay development and screening platform for targets where phenotypic or other cell-based assays are not readily available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD30bRGadycx0LsK7uFuY7fW6udTcc2eY5IZ9elJ2Tbbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FgsFyktQ%253D%253D&md5=d4d1c56ac6d2cb79fe69911293cab32c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-27834-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-27834-y%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%26aufirst%3DN.%2BJ.%26aulast%3DAsawa%26aufirst%3DR.%2BR.%26aulast%3DCyr%26aufirst%3DM.%2BG.%26aulast%3DZakharov%26aufirst%3DA.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DRoth%26aufirst%3DJ.%2BS.%26aulast%3DWallgren%26aufirst%3DE.%26aulast%3DKlumpp-Thomas%26aufirst%3DC.%26aulast%3DCoussens%26aufirst%3DN.%2BP.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.-M.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26atitle%3DA%2520widely-applicable%2520high-throughput%2520cellular%2520thermal%2520shift%2520assay%2520%2528CETSA%2529%2520using%2520split%2520Nano%2520Luciferase%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D9472%26doi%3D10.1038%2Fs41598-018-27834-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Highly predictive and interpretable models for PAMPA permeability</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.bmc.2016.12.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=28082071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1266-1276&author=H.+Sunauthor=K.+Nguyenauthor=E.+Kernsauthor=Z.+Yanauthor=K.+R.+Yuauthor=P.+Shahauthor=A.+Jadhavauthor=X.+Xu&title=Highly+predictive+and+interpretable+models+for+PAMPA+permeability&doi=10.1016%2Fj.bmc.2016.12.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Highly predictive and interpretable models for PAMPA permeability</span></div><div class="casAuthors">Sun, Hongmao; Nguyen, Kimloan; Kerns, Edward; Yan, Zhengyin; Yu, Kyeong Ri; Shah, Pranav; Jadhav, Ajit; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1266-1276</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell membrane permeability is an important determinant for oral absorption and bioavailability of a drug mol.  An in silico model predicting drug permeability is described, which is built based on a large permeability dataset of 7488 compd. entries or 5435 structurally unique mols. measured by the same lab using parallel artificial membrane permeability assay (PAMPA).  On the basis of customized mol. descriptors, the support vector regression (SVR) model trained with 4071 compds. with quant. data is able to predict the remaining 1364 compds. with the qual. data with an area under the curve of receiver operating characteristic (AUC-ROC) of 0.90.  The support vector classification (SVC) model trained with half of the whole dataset comprised of both the quant. and the qual. data produced accurate predictions to the remaining data with the AUC-ROC of 0.88.  The results suggest that the developed SVR model is highly predictive and provides medicinal chemists a useful in silico tool to facilitate design and synthesis of novel compds. with optimal drug-like properties, and thus accelerate the lead optimization in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y8FhTLLqbrVg90H21EOLACvtfcHk0lirPvzmV9oWQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D&md5=2f6b60dfcd168b5bf837405ca52457b2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.049%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DHighly%2520predictive%2520and%2520interpretable%2520models%2520for%2520PAMPA%2520permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1266%26epage%3D1276%26doi%3D10.1016%2Fj.bmc.2016.12.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">L’Heureux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaulieu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bill, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaFlamme, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirmehrabi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadayon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tovell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couturier, M.</span></span> <span> </span><span class="NLM_article-title">Aminodifluorosulfinium salts: selective fluorination reagents with enhanced thermal stability and ease of handling</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3401</span>– <span class="NLM_lpage">3411</span>, <span class="refDoi"> DOI: 10.1021/jo100504x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo100504x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=3401-3411&author=A.+L%E2%80%99Heureuxauthor=F.+Beaulieuauthor=C.+Bennettauthor=D.+R.+Billauthor=S.+Claytonauthor=F.+LaFlammeauthor=M.+Mirmehrabiauthor=S.+Tadayonauthor=D.+Tovellauthor=M.+Couturier&title=Aminodifluorosulfinium+salts%3A+selective+fluorination+reagents+with+enhanced+thermal+stability+and+ease+of+handling&doi=10.1021%2Fjo100504x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Aminodifluorosulfinium Salts: Selective Fluorination Reagents with Enhanced Thermal Stability and Ease of Handling</span></div><div class="casAuthors">L'Heureux, Alexandre; Beaulieu, Francis; Bennett, Christopher; Bill, David R.; Clayton, Simon; La Flamme, Francois; Mirmehrabi, Mahmoud; Tadayon, Sam; Tovell, David; Couturier, Michel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3401-3411</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Diethylaminodifluorosulfinium tetrafluoroborate (XtalFluor-E) and morpholinodifluorosulfinium tetrafluoroborate (XtalFluor-M) are cryst. fluorinating agents that are more easily handled and significantly more stable than Deoxo-Fluor, DAST, and their analogs.  These reagents can be prepd. in a safer and more cost-efficient manner by avoiding the laborious and hazardous distn. of dialkylaminosulfur trifluorides.  Unlike DAST, Deoxo-Fluor, and Fluolead, XtalFluor reagents do not generate highly corrosive free-HF and therefore can be used in std. borosilicate vessels.  When used in conjunction with promoters such as Et3N·3HF, Et3N·2HF, or DBU, XtalFluor reagents effectively convert alcs. to alkyl fluorides and carbonyls to gem-difluorides.  These reagents are typically more selective than DAST and Deoxo-Fluor and exhibit superior performance by providing significantly less elimination side products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor511M96t3p7Vg90H21EOLACvtfcHk0lgN8msXK4dKeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqur0%253D&md5=3afb1f998f6e1a28a8559e76bab4dda1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo100504x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo100504x%26sid%3Dliteratum%253Aachs%26aulast%3DL%25E2%2580%2599Heureux%26aufirst%3DA.%26aulast%3DBeaulieu%26aufirst%3DF.%26aulast%3DBennett%26aufirst%3DC.%26aulast%3DBill%26aufirst%3DD.%2BR.%26aulast%3DClayton%26aufirst%3DS.%26aulast%3DLaFlamme%26aufirst%3DF.%26aulast%3DMirmehrabi%26aufirst%3DM.%26aulast%3DTadayon%26aufirst%3DS.%26aulast%3DTovell%26aufirst%3DD.%26aulast%3DCouturier%26aufirst%3DM.%26atitle%3DAminodifluorosulfinium%2520salts%253A%2520selective%2520fluorination%2520reagents%2520with%2520enhanced%2520thermal%2520stability%2520and%2520ease%2520of%2520handling%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2010%26volume%3D75%26spage%3D3401%26epage%3D3411%26doi%3D10.1021%2Fjo100504x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benavides, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancaster, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dell’Italia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darley-Usmar, V. M.</span></span> <span> </span><span class="NLM_article-title">Integration of cellular bioenergetics with mitochondrial quality control and autophagy</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>393</i></span>,  <span class="NLM_fpage">1485</span>– <span class="NLM_lpage">1512</span>, <span class="refDoi"> DOI: 10.1515/hsz-2012-0198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1515%2Fhsz-2012-0198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=23092819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXns1Wruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=393&publication_year=2012&pages=1485-1512&author=B.+G.+Hillauthor=G.+A.+Benavidesauthor=J.+R.+Lancasterauthor=S.+Ballingerauthor=L.+Dell%E2%80%99Italiaauthor=J.+Zhangauthor=V.+M.+Darley-Usmar&title=Integration+of+cellular+bioenergetics+with+mitochondrial+quality+control+and+autophagy&doi=10.1515%2Fhsz-2012-0198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of cellular bioenergetics with mitochondrial quality control and autophagy</span></div><div class="casAuthors">Hill, Bradford G.; Benavides, Gloria A.; Lancaster, Jack R., Jr.; Ballinger, Scott; Dell'Italia, Lou; Zhang, Jianhua; Darley-Usmar, Victor M.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">393</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1485-1512</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Bioenergetic dysfunction is emerging as a cornerstone for establishing a framework for understanding the pathophysiol. of cardiovascular disease, diabetes, cancer and neurodegeneration.  Recent advances in cellular bioenergetics have shown that many cells maintain a substantial bioenergetic reserve capacity, which is a prospective index of "healthy" mitochondrial populations.  The bioenergetics of the cell are likely regulated by energy requirements and substrate availability.  Addnl., the overall quality of the mitochondrial population and the relative abundance of mitochondria in cells and tissues also impinge on overall bioenergetic capacity and resistance to stress.  Because mitochondria are susceptible to damage mediated by reactive oxygen/nitrogen and lipid species, maintaining a "healthy" population of mitochondria through quality control mechanisms appears to be essential for cell survival under conditions of pathol. stress.  Accumulating evidence suggest that mitophagy is particularly important for preventing amplification of initial oxidative insults, which otherwise would further impair the respiratory chain or promote mutations in mitochondrial DNA (mtDNA).  The processes underlying the regulation of mitophagy depend on several factors, including the integrity of mtDNA, electron transport chain activity, and the interaction and regulation of the autophagic machinery.  The integration and interpretation of cellular bioenergetics in the context of mitochondrial quality control and genetics is the theme of this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_88kMLPXxJrVg90H21EOLACvtfcHk0lgN8msXK4dKeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXns1Wruw%253D%253D&md5=b0c74b21fe1b672050490cc98df5595b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1515%2Fhsz-2012-0198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fhsz-2012-0198%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DB.%2BG.%26aulast%3DBenavides%26aufirst%3DG.%2BA.%26aulast%3DLancaster%26aufirst%3DJ.%2BR.%26aulast%3DBallinger%26aufirst%3DS.%26aulast%3DDell%25E2%2580%2599Italia%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDarley-Usmar%26aufirst%3DV.%2BM.%26atitle%3DIntegration%2520of%2520cellular%2520bioenergetics%2520with%2520mitochondrial%2520quality%2520control%2520and%2520autophagy%26jtitle%3DBiol.%2520Chem.%26date%3D2012%26volume%3D393%26spage%3D1485%26epage%3D1512%26doi%3D10.1515%2Fhsz-2012-0198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsinman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsinman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avdeef, A.</span></span> <span> </span><span class="NLM_article-title">Physicochemical selectivity of the BBB microenvironment governing passive diffusion matching with a porcine brain lipid extract artificial membrane permeability model</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">363</span>, <span class="refDoi"> DOI: 10.1007/s11095-010-0280-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1007%2Fs11095-010-0280-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=20945153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yrs7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2011&pages=337-363&author=O.+Tsinmanauthor=K.+Tsinmanauthor=N.+Sunauthor=A.+Avdeef&title=Physicochemical+selectivity+of+the+BBB+microenvironment+governing+passive+diffusion+matching+with+a+porcine+brain+lipid+extract+artificial+membrane+permeability+model&doi=10.1007%2Fs11095-010-0280-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical Selectivity of the BBB Microenvironment Governing Passive Diffusion-Matching with a Porcine Brain Lipid Extract Artificial Membrane Permeability Model</span></div><div class="casAuthors">Tsinman, Oksana; Tsinman, Konstantin; Sun, Na; Avdeef, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-363</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: To mimic the physicochem. selectivity of the blood-brain barrier (BBB) and to predict its passive permeability using a PAMPA model based on porcine brain lipid ext. (PBLE 10%w/v in alkane).  Methods: Three PAMPA (BD pre-coated and PBLE with 2 different lipid vols.) models were tested with 108 drugs.  Abraham solvation descriptors were used to interpret the in vitro-in vivo correlation with 282 in situ brain perfusion measurements, spanning over 5 orders of magnitude.  An in combo PAMPA model was developed from combining measured PAMPA permeability with one H-bond descriptor.  Results: The in combo PAMPA predicted 93% of the variance of 197 largely efflux-inhibited in situ permeability training set.  The model was cross-validated by the "leave-many-out" procedure, with q2 = 0.92 ± 0.03.  The PAMPA models indicated the presence of paramembrane water channels.  Only the PBLE-based PAMPA-BBB model with sufficient lipid to fill all the internal pore space of the filter showed a wide dynamic range window, selectivity coeff. near 1, and was suitable for predicting BBB permeability.  Conclusion: BBB permeability can be predicted by in combo PAMPA.  Its speed and substantially lower cost, compared to in vivo measurements, make it an attractive first-pass screening method for BBB passive permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_ZUNSGMH-JbVg90H21EOLACvtfcHk0lhCEmKl0MueNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yrs7vF&md5=31afeb98ee1765bb812f0881e0fe4452</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs11095-010-0280-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-010-0280-x%26sid%3Dliteratum%253Aachs%26aulast%3DTsinman%26aufirst%3DO.%26aulast%3DTsinman%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DAvdeef%26aufirst%3DA.%26atitle%3DPhysicochemical%2520selectivity%2520of%2520the%2520BBB%2520microenvironment%2520governing%2520passive%2520diffusion%2520matching%2520with%2520a%2520porcine%2520brain%2520lipid%2520extract%2520artificial%2520membrane%2520permeability%2520model%26jtitle%3DPharm.%2520Res.%26date%3D2011%26volume%3D28%26spage%3D337%26epage%3D363%26doi%3D10.1007%2Fs11095-010-0280-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourassa, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D. M.</span></span> <span> </span><span class="NLM_article-title">Experimental design on single-time-point high-throughput microsomal stability assay</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1544</span>, <span class="refDoi"> DOI: 10.1002/jps.20076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1002%2Fjps.20076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=15124211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks12hsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2004&pages=1537-1544&author=L.+Diauthor=E.+H.+Kernsauthor=N.+Gaoauthor=S.+Q.+Liauthor=Y.+Huangauthor=J.+L.+Bourassaauthor=D.+M.+Huryn&title=Experimental+design+on+single-time-point+high-throughput+microsomal+stability+assay&doi=10.1002%2Fjps.20076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental design on single-time-point high-throughput microsomal stability assay</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.; Gao, Ning; Li, Susan Q.; Huang, Youping; Bourassa, Jim L.; Huryn, Donna M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1537-1544</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">An exptl. design for a single-time-point microsomal stability assay was evaluated as compared with multiple-time-point studies.  Results obtained from single-time-point expts. are in excellent agreement with those from multiple time points.  First-order reaction kinetics revealed rapid changes of predicted half-life from percent remaining of the parent compd. at the inflection points, suggesting a max. predictive limit for half-life.  Selection of the incubation time in single-time-point assays is important to obtain balanced information for stable and unstable compds.  A short incubation time (e.g., 5 min) is most useful for differentiating between unstable compds., which is beneficial to direct the synthetic efforts in projects with poor metabolic stability.  A long incubation time (e.g., 30 min) is more applicable to a compd. series with high metabolic stability.  For screening purposes, a moderate incubation time (e.g., 15 min) is recommended to achieve good resoln. and a sufficiently high max. predictive limit for half-life.  This study suggests that a single-time-point assay is sufficient for ranking compds. in early drug discovery.  It increases throughput and reduces turnaround time and cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdHPH7tWzrIrVg90H21EOLACvtfcHk0lhCEmKl0MueNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks12hsr4%253D&md5=45b3bfb054ad93e18655b988fdbbe89b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fjps.20076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.20076%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DS.%2BQ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DBourassa%26aufirst%3DJ.%2BL.%26aulast%3DHuryn%26aufirst%3DD.%2BM.%26atitle%3DExperimental%2520design%2520on%2520single-time-point%2520high-throughput%2520microsomal%2520stability%2520assay%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2004%26volume%3D93%26spage%3D1537%26epage%3D1544%26doi%3D10.1002%2Fjps.20076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13a','cit13b'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17a','cit17b'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21a','cit21b'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jinyue Ding, Rajesh Gumpena, Marc-Olivier Boily, Alexandre Caron, Oliver Chong, Jennifer H. Cox, Valerie Dumais, Samuel Gaudreault, Aaron H. Graff, Andrew King, John Knight, Renata Oballa, Jayakumar Surendradoss, Tim Tang, Joyce Wu, W. Todd Lowther, <span class="NLM_string-name hlFld-ContribAuthor">David A. Powell</span>. </span><span class="cited-content_cbyCitation_article-title">Dual Glycolate Oxidase/Lactate Dehydrogenase A Inhibitors for Primary Hyperoxaluria. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (7)
                                     , 1116-1123. <a href="https://doi.org/10.1021/acsmedchemlett.1c00196" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00196</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00196%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDual%252BGlycolate%252BOxidase%25252FLactate%252BDehydrogenase%252BA%252BInhibitors%252Bfor%252BPrimary%252BHyperoxaluria%26aulast%3DDing%26aufirst%3DJinyue%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D07042021%26date%3D18052021%26date%3D20052021%26volume%3D12%26issue%3D7%26spage%3D1116%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Plamen P.  Christov</span>, <span class="hlFld-ContribAuthor ">Kwangho  Kim</span>, <span class="hlFld-ContribAuthor ">Somnath  Jana</span>, <span class="hlFld-ContribAuthor ">Ian M.  Romaine</span>, <span class="hlFld-ContribAuthor ">Ganesha  Rai</span>, <span class="hlFld-ContribAuthor ">Bryan T.  Mott</span>, <span class="hlFld-ContribAuthor ">Alexander A.  Allweil</span>, <span class="hlFld-ContribAuthor ">Alexander  Lamers</span>, <span class="hlFld-ContribAuthor ">Kyle R.  Brimacombe</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Urban</span>, <span class="hlFld-ContribAuthor ">Tobie D.  Lee</span>, <span class="hlFld-ContribAuthor ">Xin  Hu</span>, <span class="hlFld-ContribAuthor ">Christine M.  Lukacs</span>, <span class="hlFld-ContribAuthor ">Douglas R.  Davies</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Hall</span>, <span class="hlFld-ContribAuthor ">Neal  Green</span>, <span class="hlFld-ContribAuthor ">William J.  Moore</span>, <span class="hlFld-ContribAuthor ">Gordon M.  Stott</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Flint</span>, <span class="hlFld-ContribAuthor ">David J.  Maloney</span>, <span class="hlFld-ContribAuthor ">Gary A.  Sulikowski</span>, <span class="hlFld-ContribAuthor ">Alex G.  Waterson</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of ether and aniline based inhibitors of lactate dehydrogenase. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>41 </em>, 127974. <a href="https://doi.org/10.1016/j.bmcl.2021.127974" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.127974</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.127974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.127974%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DOptimization%252Bof%252Bether%252Band%252Baniline%252Bbased%252Binhibitors%252Bof%252Blactate%252Bdehydrogenase%26aulast%3DChristov%26aufirst%3DPlamen%2BP.%26date%3D2021%26volume%3D41%26spage%3D127974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicholas R.  Perkons</span>, <span class="hlFld-ContribAuthor ">Omar  Johnson</span>, <span class="hlFld-ContribAuthor ">Gabrielle  Pilla</span>, <span class="hlFld-ContribAuthor ">Terence P. F.  Gade</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacodynamics and pharmacokinetics of hyperpolarized [1‐
              13
              C]‐pyruvate in a translational oncologic model. </span><span class="cited-content_cbyCitation_journal-name">NMR in Biomedicine</span><span> <strong>2021,</strong> <em>34 </em>
                                    (6)
                                     <a href="https://doi.org/10.1002/nbm.4502" title="DOI URL">https://doi.org/10.1002/nbm.4502</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/nbm.4502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fnbm.4502%26sid%3Dliteratum%253Aachs%26jtitle%3DNMR%2520in%2520Biomedicine%26atitle%3DPharmacodynamics%252Band%252Bpharmacokinetics%252Bof%252Bhyperpolarized%252B%25255B1%2525E2%252580%252590%252B13%252BC%25255D%2525E2%252580%252590pyruvate%252Bin%252Ba%252Btranslational%252Boncologic%252Bmodel%26aulast%3DPerkons%26aufirst%3DNicholas%2BR.%26date%3D2021%26date%3D2021%26volume%3D34%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria Dolores  Moya-Garzon</span>, <span class="hlFld-ContribAuthor ">Jose Antonio  Gomez-Vidal</span>, <span class="hlFld-ContribAuthor ">Alfonso  Alejo-Armijo</span>, <span class="hlFld-ContribAuthor ">Joaquin  Altarejos</span>, <span class="hlFld-ContribAuthor ">Juan Roberto  Rodriguez-Madoz</span>, <span class="hlFld-ContribAuthor ">Miguel Xavier  Fernandes</span>, <span class="hlFld-ContribAuthor ">Eduardo  Salido</span>, <span class="hlFld-ContribAuthor ">Sofia  Salido</span>, <span class="hlFld-ContribAuthor ">Monica  Diaz-Gavilan</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule-Based Enzyme Inhibitors in the Treatment of Primary Hyperoxalurias. </span><span class="cited-content_cbyCitation_journal-name">Journal of Personalized Medicine</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 74. <a href="https://doi.org/10.3390/jpm11020074" title="DOI URL">https://doi.org/10.3390/jpm11020074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/jpm11020074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fjpm11020074%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Personalized%2520Medicine%26atitle%3DSmall%252BMolecule-Based%252BEnzyme%252BInhibitors%252Bin%252Bthe%252BTreatment%252Bof%252BPrimary%252BHyperoxalurias%26aulast%3DMoya-Garzon%26aufirst%3DMaria%2BDolores%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative previously described LDH inhibitors and comparison with new leads <b>43 = NCATS-SM1440</b> and <b>52 = NCATS-SM1441</b>. <sup>a</sup>Named as NCI-006 in refs <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>. <sup>b</sup>Named as NCI-737 in ref <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of Intermediates <b>VIa–r</b> and Analogues <b>1–9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) SOCl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 4 h, and 91–100%; (b) MgBr<sub>2</sub>·OEt<sub>2</sub>, iPr<sub>2</sub>NEt, CH<sub>2</sub>Cl<sub>2</sub>, 12 h, and 60–69%; (c) Cs<sub>2</sub>CO<sub>3</sub>, DMSO, 1 h, and 55–83%; (d) (i) pyrrolidine (0.5 equiv), TsOH (0.5 equiv), EtOH, reflux, and 1–2 h and (ii) ethyl 2-hydrazinylthiazole-4-carboxylate·HBr and reflux overnight; and (e) LiOH, THF/MeOH/H<sub>2</sub>O, and 1 h.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Analogues <b>10–32</b> and <b>36–89</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) [P(<i>t</i>-Bu)<sub>3</sub>]Pd(crotyl)Cl, DABCO, dioxane, RT, and 1–12 h; (b) [DTBNpP]Pd(crotyl)Cl, DABCO, dioxane, 60 °C, and 12 h; (c) SPhosPd(crotyl)Cl, K<sub>3</sub>PO<sub>4</sub>, dioxane/H<sub>2</sub>O, 100 °C, and 0.5 h; (d) LiOH, THF/MeOH/H<sub>2</sub>O, and 1 h; and (e) CsF, THF–EtOH, RT, and 2 h.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of Analogues <b>33–35</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Dess–Martin periodinane, DCM, RT, and 2 h; (b) NaN<sub>3</sub>, CF<sub>3</sub>SO<sub>3</sub>H, ACN, RT, and 12 h; (c) (CH<sub>3</sub>)<sub>3</sub>Sn–OH, dichloroethane, MW, 110 °C, and 1 h; (d) Deoxo-Fluor, DCM, RT, and 12 h; (e) P(<i>t</i>-Bu)<sub>3.</sub>HBF<sub>4</sub>, ([PdCl(allyl)]<sub>2</sub>, dioxane, 80 °C, and 4 h; and (f) LiOH, THF/MeOH/H<sub>2</sub>O, and 1 h.</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of LDHA bound to inhibitor <b>23</b> (A, pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q0D">6Q0D</a>) and <b>52</b> (B, pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q13">6Q13</a>). LDHA is shown in ribbon (blue) and key residues in the active site are shown in green. Small-molecule inhibitors are shown in sticks with salmon- and magenta-colored carbons.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. LDH inhibitor-dependent suppression of glycolytic flux in A673 cells. The GST was performed in A673 cells to measure the ECAR over time; cellular basal ECAR was measured and then compound <b>43</b> (panel A) or <b>52</b> (panel B) was injected at increasing concentrations. After 48 min, subsequent injections of glucose (10 mM; glycolysis), oligomycin (1 μg/mL) (O; reaching maximal glycolytic capacity), and 2-deoxyglucose (50 mM) (2-DG; inhibition of glycolysis) were made. (C) Quantification of the glycolysis (ECAR after glucose injection minus basal ECAR) and (D) quantification of the glycolytic capacity of the LDHA inhibitors, <b>43</b> and <b>52</b>, in a dose–response manner in comparison with compounds <b>1</b> and <b>23</b>. Data represent the mean ± standard error of the mean, <i>n</i> = 4–6 per group. All LDH inhibitors completely suppressed both basal and maximal glycolysis between 1 and 3 μM, with <b>43</b> and <b>52</b> being the most potent and <b>23</b> the least.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <i>In vivo</i> target engagement and tumor LDH activity upon administration of compounds in mice bearing A673 flank xenograft tumors. Mice received a single IV injection of LDHA inhibitors <b>14</b>, <b>43</b>, <b>47</b>, <b>52</b>, <b>56</b>, <b>69</b>, and <b>71</b> at 50 mg/kg. At the time of sacrifice, the samples of tumor and plasma were collected and flash-frozen in liquid nitrogen. Compound levels in plasma and tumor were determined by LC–MS/MS, and LDHA activity was measured in tumor lysates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/medium/jm0c00916_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SAR summary and essential structural moieties contributing to the cellular potency, binding affinity, and PK properties of analogue <b>52</b> (<b>NCATS-SM1441</b>) as an example.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-19/acs.jmedchem.0c00916/20201001/images/large/jm0c00916_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00916&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i131">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18021" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18021" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 31 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warburg, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posener, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negelein, E.</span></span> <span> </span><span class="NLM_article-title">Ueber den stoffwechsel der tumoren</span>. <i>Biochem. Z.</i> <span class="NLM_year" style="font-weight: bold;">1924</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">344</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=1924&pages=319-344&author=O.+Warburgauthor=K.+Posenerauthor=E.+Negelein&title=Ueber+den+stoffwechsel+der+tumoren"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWarburg%26aufirst%3DO.%26aulast%3DPosener%26aufirst%3DK.%26aulast%3DNegelein%26aufirst%3DE.%26atitle%3DUeber%2520den%2520stoffwechsel%2520der%2520tumoren%26jtitle%3DBiochem.%2520Z.%26date%3D1924%26volume%3D152%26spage%3D319%26epage%3D344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">Understanding the Warburg effect: the metabolic requirements of cell proliferation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>324</i></span>,  <span class="NLM_fpage">1029</span>– <span class="NLM_lpage">1033</span>, <span class="refDoi"> DOI: 10.1126/science.1160809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1126%2Fscience.1160809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=19460998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVKlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=2009&pages=1029-1033&author=M.+G.+Vander%0AHeidenauthor=L.+C.+Cantleyauthor=C.+B.+Thompson&title=Understanding+the+Warburg+effect%3A+the+metabolic+requirements+of+cell+proliferation&doi=10.1126%2Fscience.1160809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the Warburg Effect: The Metabolic Requirements of Cell Proliferation</span></div><div class="casAuthors">Vander Heiden, Matthew G.; Cantley, Lewis C.; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">324</span>
        (<span class="NLM_cas:issue">5930</span>),
    <span class="NLM_cas:pages">1029-1033</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  In contrast to normal differentiated cells, which rely primarily on mitochondrial oxidative phosphorylation to generate the energy needed for cellular processes, most cancer cells instead rely on aerobic glycolysis, a phenomenon termed "the Warburg effect".  Aerobic glycolysis is an inefficient way to generate ATP, however, and the advantage it confers to cancer cells has been unclear.  Here we propose that the metab. of cancer cells, and indeed all proliferating cells, is adapted to facilitate the uptake and incorporation of nutrients into the biomass (e.g., nucleotides, amino acids, and lipids) needed to produce a new cell.  Supporting this idea are recent studies showing that (i) several signaling pathways implicated in cell proliferation also regulate metabolic pathways that incorporate nutrients into biomass; and that (ii) certain cancer-assocd. mutations enable cancer cells to acquire and metabolize nutrients in a manner conducive to proliferation rather than efficient ATP prodn.  A better understanding of the mechanistic links between cellular metab. and growth control may ultimately lead to better treatments for human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGvLHGQ9TX77Vg90H21EOLACvtfcHk0lg5gyTYNIO-wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVKlsbg%253D&md5=f927402a43cdca471c3f4874dcbeb943</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1160809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1160809%26sid%3Dliteratum%253Aachs%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DUnderstanding%2520the%2520Warburg%2520effect%253A%2520the%2520metabolic%2520requirements%2520of%2520cell%2520proliferation%26jtitle%3DScience%26date%3D2009%26volume%3D324%26spage%3D1029%26epage%3D1033%26doi%3D10.1126%2Fscience.1160809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Exploiting tumor metabolism: challenges for clinical translation</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">3648</span>– <span class="NLM_lpage">3651</span>, <span class="refDoi"> DOI: 10.1172/jci72391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1172%2FJCI72391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=23999437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVenu7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2013&pages=3648-3651&author=M.+G.+Vander%0AHeiden&title=Exploiting+tumor+metabolism%3A+challenges+for+clinical+translation&doi=10.1172%2Fjci72391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting tumor metabolism: challenges for clinical translation</span></div><div class="casAuthors">Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3648-3651</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review.  The metab. of cancer cells differs from most normal cells, but how to exploit this difference for patient benefit is incompletely understood.  Cancer cells require altered metab. to efficiently incorporate nutrients into biomass and support abnormal proliferation.  In addn., the survival of tumor cells outside of a normal tissue context requires adaptation of metab. to different microenvironments.  Some existing chemotherapies target metabolic enzymes, and there is a resurgent interest in developing new cancer drugs that interfere with metab.  Success with this approach depends on understanding why specific metabolic pathways are important for cancer cells, detg. how best to select patients, and developing technologies for monitoring patient response to therapies that target metabolic enzymes.  The articles in this Review series address these issues, with a focus on how altered metab. might influence tumor progression and how this knowledge might inform the use of new therapies targeting cancer metab.  Emerging biomarker strategies to guide drug development are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouHaPmfjSMjLVg90H21EOLACvtfcHk0ljIRrRheyn2Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVenu7bO&md5=384f19e859ee2665ae545dcf1d937b6c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1172%2FJCI72391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI72391%26sid%3Dliteratum%253Aachs%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DExploiting%2520tumor%2520metabolism%253A%2520challenges%2520for%2520clinical%2520translation%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2013%26volume%3D123%26spage%3D3648%26epage%3D3651%26doi%3D10.1172%2Fjci72391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Z.</span></span> <span> </span><span class="NLM_article-title">Overexpression of lactate dehydrogenase-A in human intrahepatic cholangiocarcinoma: its implication for treatment</span>. <i>World J. Surg. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1186/1477-7819-12-78</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1186%2F1477-7819-12-78" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24679073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A280%3ADC%252BC2crosV2rsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=78-84&author=Y.+Yuauthor=M.+Liaoauthor=R.+Liuauthor=J.+Chenauthor=H.+Fengauthor=Z.+Fu&title=Overexpression+of+lactate+dehydrogenase-A+in+human+intrahepatic+cholangiocarcinoma%3A+its+implication+for+treatment&doi=10.1186%2F1477-7819-12-78"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of lactate dehydrogenase-A in human intrahepatic cholangiocarcinoma: its implication for treatment</span></div><div class="casAuthors">Yu Yaping; Liao Minqi; Liu Ruiwen; Chen Jian; Feng Hao; Fu Zan</div><div class="citationInfo"><span class="NLM_cas:title">World journal of surgical oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">78</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Previous studies have shown that lactate dehydrogenase-A (LDH-A) is strongly expressed in several malignancies, that LDH-A expression is associated with poor prognosis, and that LDH-A inhibition severely diminishes tumorigenicity.  However, little is known about the implications of LDH-A expression in intrahepatic cholangiocarcinoma.  The purpose of this study was to investigate the expression of LDH-A and to clarify its effect on intrahepatic cholangiocarcinoma.  METHODS:  We studied the expression of LDH-A in tissue samples from patients with intrahepatic cholangiocarcinoma (n = 54) using the ultrasensitive surfactant protein (S-P) immunohistochemical method.  We then inhibited LDH-A using small hairpin RNA (shRNA) in the cholangiocarcinoma cell line HuCCT-1 in vitro to study the role it plays in promoting growth and escaping apoptosis.  RESULTS:  We report that LDH-A was overexpressed in 52 of 54 (96%) paraffin-embedded cancer tissue samples and 0 of 54 para-carcinoma tissue samples.  Reduction of LDH-A by RNA interference (RNAi) inhibited cell growth and induced apoptosis in HuCCT-1 cells.  This result correlated with the elevation of cytoplasmic reactive oxygen species (ROS) levels.  CONCLUSIONS:  LDH-A expression is closely correlated with histopathological variables of intrahepatic cholangiocarcinoma, indicating that LDH-A may serve as a new treatment target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRH1LWvPIhPcQ8IxFYvehhofW6udTcc2ea5fOEvCvZoI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2crosV2rsg%253D%253D&md5=006604c27e3b3d6a1bea31204f737ef4</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1186%2F1477-7819-12-78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1477-7819-12-78%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DFu%26aufirst%3DZ.%26atitle%3DOverexpression%2520of%2520lactate%2520dehydrogenase-A%2520in%2520human%2520intrahepatic%2520cholangiocarcinoma%253A%2520its%2520implication%2520for%2520treatment%26jtitle%3DWorld%2520J.%2520Surg.%2520Oncol.%26date%3D2014%26volume%3D12%26spage%3D78%26epage%3D84%26doi%3D10.1186%2F1477-7819-12-78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, W.</span></span> <span> </span><span class="NLM_article-title">Lactate dehydrogenase A is overexpressed in pancreatic cancer and promotes the growth of pancreatic cancer cells</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1523</span>– <span class="NLM_lpage">1530</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-0679-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1007%2Fs13277-013-0679-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=23404405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos1yktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=1523-1530&author=Y.+Rongauthor=W.+Wuauthor=X.+Niauthor=T.+Kuangauthor=D.+Jinauthor=D.+Wangauthor=W.+Lou&title=Lactate+dehydrogenase+A+is+overexpressed+in+pancreatic+cancer+and+promotes+the+growth+of+pancreatic+cancer+cells&doi=10.1007%2Fs13277-013-0679-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Lactate dehydrogenase A is overexpressed in pancreatic cancer and promotes the growth of pancreatic cancer cells</span></div><div class="casAuthors">Rong, Yefei; Wu, Wenchuan; Ni, Xiaoling; Kuang, Tiantao; Jin, Dayong; Wang, Dansong; Lou, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1523-1530</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The prognosis for pancreatic cancer is very poor, and developing new therapeutic strategies for this cancer is needed.  Recently, the Warburg effect (aerobic glycolysis) has attracted much attention for its function in the tumorigenesis.  Lactate dehydrogenase A (LDHA) executes the final step of aerobic glycolysis and has been reported to be involved in the tumor progression.  However, the function of LDHA in pancreatic cancer has not been studied.  Here, we found that the expression of LDHA was elevated in the clin. pancreatic cancer samples.  Forced expression of LDHA promoted the growth of pancreatic cancer cells, while knocking down the expression of LDHA inhibited cell growth dramatically.  Moreover, silencing the expression of LDHA inhibited the tumorigenicity of pancreatic cancer cells in vivo.  Mechanistically, knocking down the expression of LDHA activated apoptosis pathway.  Taken together, our study revealed the oncogenic role of LDHA in pancreatic cancer and suggested that LDHA might be a potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVt3oJsFE-ebVg90H21EOLACvtfcHk0ljIRrRheyn2Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos1yktro%253D&md5=4719aab68b47f6c7229832b3425a1f0f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-0679-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-0679-1%26sid%3Dliteratum%253Aachs%26aulast%3DRong%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DKuang%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLou%26aufirst%3DW.%26atitle%3DLactate%2520dehydrogenase%2520A%2520is%2520overexpressed%2520in%2520pancreatic%2520cancer%2520and%2520promotes%2520the%2520growth%2520of%2520pancreatic%2520cancer%2520cells%26jtitle%3DTumor%2520Biol.%26date%3D2013%26volume%3D34%26spage%3D1523%26epage%3D1530%26doi%3D10.1007%2Fs13277-013-0679-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span> <span> </span><span class="NLM_article-title">Recent update on human lactate dehydrogenase enzyme 5 (hLDH5) inhibitors: a promising approach for cancer chemotherapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=487-496&author=R.+Raniauthor=V.+Kumar&title=Recent+update+on+human+lactate+dehydrogenase+enzyme+5+%28hLDH5%29+inhibitors%3A+a+promising+approach+for+cancer+chemotherapy&doi=10.1021%2Facs.jmedchem.5b00168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Update on Human Lactate Dehydrogenase Enzyme 5 (hLDH5) Inhibitors: A Promising Approach for Cancer Chemotherapy</span></div><div class="casAuthors">Rani, Reshma; Kumar, Vinit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">487-496</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human lactate dehydrogenase (hLDH5), a glycolytic enzyme responsible for the conversion of pyruvate to lactate coupled with oxidn. of NADH to NAD+, plays a crucial role in the promotion of glycolysis in invasive tumor cells.  Recently, hLDH5 has been considered a vital therapeutic target for invasive cancers.  Selective inhibition of hLDH5 using small mols. holds potential prospects for the treatment of cancer and assocd. diseases.  Consequently, significant progress has been made in the discovery of selective small-mol. hLDH5 inhibitors displaying remarkable inhibitory potencies.  The purpose of this review is to discuss briefly the roles of hLDH isoforms and to compile small hLDH5 inhibitors into groups based on their chem. classes and pharmacol. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptlkXXBAejKbVg90H21EOLACvtfcHk0lhVPqjugyivRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7bL&md5=dcf788f61e81e0420daa844c9896deb4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00168%26sid%3Dliteratum%253Aachs%26aulast%3DRani%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DV.%26atitle%3DRecent%2520update%2520on%2520human%2520lactate%2520dehydrogenase%2520enzyme%25205%2520%2528hLDH5%2529%2520inhibitors%253A%2520a%2520promising%2520approach%2520for%2520cancer%2520chemotherapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D487%26epage%3D496%26doi%3D10.1021%2Facs.jmedchem.5b00168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allison, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granchi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minutolo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, R. M.</span></span> <span> </span><span class="NLM_article-title">Identification of LDH-A as a therapeutic target for cancer cell killing via (i) p53/NAD(H)-dependent and (ii) p53-independent pathways</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e102</span> <span class="refDoi"> DOI: 10.1038/oncsis.2014.16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1038%2Foncsis.2014.16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24819061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvF2lsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&author=S.+J.+Allisonauthor=J.+R.+P.+Knightauthor=C.+Granchiauthor=R.+Raniauthor=F.+Minutoloauthor=J.+Milnerauthor=R.+M.+Phillips&title=Identification+of+LDH-A+as+a+therapeutic+target+for+cancer+cell+killing+via+%28i%29+p53%2FNAD%28H%29-dependent+and+%28ii%29+p53-independent+pathways&doi=10.1038%2Foncsis.2014.16"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of LDH-A as a therapeutic target for cancer cell killing via (i) p53/NAD(H)-dependent and (ii) p53-independent pathways</span></div><div class="casAuthors">Allison, S. J.; Knight, J. R. P.; Granchi, C.; Rani, R.; Minutolo, F.; Milner, J.; Phillips, R. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e102</span>CODEN:
                <span class="NLM_cas:coden">ONCOCT</span>;
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most cancer cells use aerobic glycolysis to fuel their growth.  The enzyme lactate dehydrogenase-A (LDH-A) is key to cancer's glycolytic phenotype, catalyzing the regeneration of NAD (NAD+) from reduced NAD (NADH) necessary to sustain glycolysis.  As such, LDH-A is a promising target for anticancer therapy.  Here we ask if the tumor suppressor p53, a major regulator of cellular metab., influences the response of cancer cells to LDH-A suppression.  LDH-A knockdown by RNA interference (RNAi) induced cancer cell death in p53 wild-type, mutant and p53-null human cancer cell lines, indicating that endogenous LDH-A promotes cancer cell survival irresp. of cancer cell p53 status.  Unexpectedly, however, we uncovered a novel role for p53 in the regulation of cancer cell NAD+ and its reduced form NADH.  Thus, LDH-A silencing by RNAi, or its inhibition using a small-mol. inhibitor, resulted in a p53-dependent increase in the cancer cell ratio of NADH:NAD+.  This effect was specific for p53+/+ cancer cells and correlated with (i) reduced activity of NAD+-dependent deacetylase sirtuin 1 (SIRT1) and (ii) an increase in acetylated p53, a known target of SIRT1 deacetylation activity.  In addn., activation of the redox-sensitive anticancer drug EO9 was enhanced selectively in p53+/+ cancer cells, attributable to increased activity of NAD(P)H-dependent oxidoreductase NQO1 (NAD(P)H quinone oxidoreductase 1).  Suppressing LDH-A increased EO9-induced DNA damage in p53+/+ cancer cells, but importantly had no additive effect in non-cancer cells.  Our results identify a unique strategy by which the NADH/NAD+ cellular redox status can be modulated in a cancer-specific, p53-dependent manner and we show that this can impact upon the activity of important NAD(H)-dependent enzymes.  To summarize, this work indicates two distinct mechanisms by which suppressing LDH-A could potentially be used to kill cancer cells selectively, (i) through induction of apoptosis, irresp. of cancer cell p53 status and (ii) as a part of a combinatorial approach with redox-sensitive anticancer drugs via a novel p53/NAD(H)-dependent mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoshoz1s34Q6bVg90H21EOLACvtfcHk0lj4GRVPJD_0-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvF2lsr0%253D&md5=33778934ca3ace8b84c81ab9199d2ba2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Foncsis.2014.16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Foncsis.2014.16%26sid%3Dliteratum%253Aachs%26aulast%3DAllison%26aufirst%3DS.%2BJ.%26aulast%3DKnight%26aufirst%3DJ.%2BR.%2BP.%26aulast%3DGranchi%26aufirst%3DC.%26aulast%3DRani%26aufirst%3DR.%26aulast%3DMinutolo%26aufirst%3DF.%26aulast%3DMilner%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DR.%2BM.%26atitle%3DIdentification%2520of%2520LDH-A%2520as%2520a%2520therapeutic%2520target%2520for%2520cancer%2520cell%2520killing%2520via%2520%2528i%2529%2520p53%252FNAD%2528H%2529-dependent%2520and%2520%2528ii%2529%2520p53-independent%2520pathways%26jtitle%3DOncogenesis%26date%3D2014%26volume%3D3%26doi%3D10.1038%2Foncsis.2014.16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouw, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinavahi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deck, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royer, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Jagt, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semenza, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span> <span> </span><span class="NLM_article-title">Inhibition of lactate dehydrogenase A induces oxidative stress and inhibits tumor progression</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">2037</span>– <span class="NLM_lpage">2042</span>, <span class="refDoi"> DOI: 10.1073/pnas.0914433107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1073%2Fpnas.0914433107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=20133848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFCit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=2037-2042&author=A.+Leauthor=C.+R.+Cooperauthor=A.+M.+Gouwauthor=R.+Dinavahiauthor=A.+Maitraauthor=L.+M.+Deckauthor=R.+E.+Royerauthor=D.+L.+Vander+Jagtauthor=G.+L.+Semenzaauthor=C.+V.+Dang&title=Inhibition+of+lactate+dehydrogenase+A+induces+oxidative+stress+and+inhibits+tumor+progression&doi=10.1073%2Fpnas.0914433107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lactate dehydrogenase A induces oxidative stress and inhibits tumor progression</span></div><div class="casAuthors">Le, Anne; Cooper, Charles R.; Gouw, Arvin M.; Dinavahi, Ramani; Maitra, Anirban; Deck, Lorraine M.; Royer, Robert E.; Vander Jagt, David L.; Semenza, Gregg L.; Dang, Chi V.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2037-2042, S2037/1-S2037/12</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the result of genetic alterations and tumor hypoxia, many cancer cells avidly take up glucose and generate lactate through lactate dehydrogenase A (LDHA), which is encoded by a target gene of c-Myc and hypoxia-inducible factor (HIF-1).  Previous studies with redn. of LDHA expression indicate that LDHA is involved in tumor initiation, but its role in tumor maintenance and progression has not been established.  Furthermore, how redn. of LDHA expression by interference or antisense RNA inhibits tumorigenesis is not well understood.  Here, we report that redn. of LDHA by siRNA or its inhibition by a small-mol. inhibitor (FX11 [3-dihydroxy-6-methyl-7-(phenylmethyl)-4-propylnaphthalene-1-carboxylic acid]) reduced ATP levels and induced significant oxidative stress and cell death that could be partially reversed by the antioxidant N-acetylcysteine (NAC).  Furthermore, we document that FX11 inhibited the progression of sizable human lymphoma and pancreatic cancer xenografts.  When used in combination with the NAD+ synthesis inhibitor FK866, FX11 induced lymphoma regression.  Hence, inhibition of LDHA with FX11 is an achievable and tolerable treatment for LDHA-dependent tumors.  Our studies document a therapeutical approach to the Warburg effect and demonstrate that oxidative stress and metabolic phenotyping of cancers are crit. aspects of cancer biol. to consider for the therapeutical targeting of cancer energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1h1YikQKAn7Vg90H21EOLACvtfcHk0lj4GRVPJD_0-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFCit7g%253D&md5=714af2ec2bc79ffcba57d6987c52a87a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0914433107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0914433107%26sid%3Dliteratum%253Aachs%26aulast%3DLe%26aufirst%3DA.%26aulast%3DCooper%26aufirst%3DC.%2BR.%26aulast%3DGouw%26aufirst%3DA.%2BM.%26aulast%3DDinavahi%26aufirst%3DR.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DDeck%26aufirst%3DL.%2BM.%26aulast%3DRoyer%26aufirst%3DR.%2BE.%26aulast%3DVander%2BJagt%26aufirst%3DD.%2BL.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26aulast%3DDang%26aufirst%3DC.%2BV.%26atitle%3DInhibition%2520of%2520lactate%2520dehydrogenase%2520A%2520induces%2520oxidative%2520stress%2520and%2520inhibits%2520tumor%2520progression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D2037%26epage%3D2042%26doi%3D10.1073%2Fpnas.0914433107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span> <span> </span><span class="NLM_article-title">Knockdown of lactate dehydrogenase A suppresses tumor growth and metastasis of human hepatocellular carcinoma</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">3898</span>– <span class="NLM_lpage">3910</span>, <span class="refDoi"> DOI: 10.1111/j.1742-4658.2012.08748.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1111%2Fj.1742-4658.2012.08748.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=22897481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVehtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2012&pages=3898-3910&author=S.+L.+Shengauthor=J.+J.+Liuauthor=Y.+H.+Daiauthor=X.+G.+Sunauthor=X.+P.+Xiongauthor=G.+Huang&title=Knockdown+of+lactate+dehydrogenase+A+suppresses+tumor+growth+and+metastasis+of+human+hepatocellular+carcinoma&doi=10.1111%2Fj.1742-4658.2012.08748.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Knockdown of lactate dehydrogenase A suppresses tumor growth and metastasis of human hepatocellular carcinoma</span></div><div class="casAuthors">Sheng, Shi L.; Liu, Jian J.; Dai, Yun H.; Sun, Xiao G.; Xiong, Xiao P.; Huang, Gang</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3898-3910</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">In previous studies, lactate dehydrogenase A (LDHA) was identified as one of the leading genes that promote the proliferative and tumorigenic potential of malignancies.  However, less definitive evidence was reported in hepatocellular carcinoma (HCC) cells.  Furthermore, the role of LDHA in promoting metastasis of HCC, and its possible mechanism, is not clear.  In this study, RNA interference (RNAi) mediated by lentiviral vectors (which induce strong down-regulation of gene expression) was used to analyze the role of LDHA in tumor growth and metastasis in HCC.  We performed transient and stable RNAi knockdowns of LDHA in HCCLM3 cells, a line that over-expresses LDHA and has a high metastatic potential.  Our studies reveal that previously unidentified effects of LHDA may mediate tumor growth and metastatic effects in HCC.  First, HCC cell lines over-express LDHA.  Second, LDHA inhibition results in increased apoptosis via prodn. of reactive oxygen species in HCCLM3 cells.  Thus, LDHA knockdown resulted in significant redn. in metastatic potential in a xenograft mouse model.  Furthermore, we found that FAK,MMP-2, VEGF and E-cadherin proteins contribute to inhibitory effects on metastasis in HCC cells.  These studies have important implications for understanding the mechanisms by which LDHA promotes tumor growth and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ1-P-Ac3M97Vg90H21EOLACvtfcHk0lj4GRVPJD_0-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVehtLrL&md5=572370ddb4964a870be366d495cdb260</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2012.08748.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2012.08748.x%26sid%3Dliteratum%253Aachs%26aulast%3DSheng%26aufirst%3DS.%2BL.%26aulast%3DLiu%26aufirst%3DJ.%2BJ.%26aulast%3DDai%26aufirst%3DY.%2BH.%26aulast%3DSun%26aufirst%3DX.%2BG.%26aulast%3DXiong%26aufirst%3DX.%2BP.%26aulast%3DHuang%26aufirst%3DG.%26atitle%3DKnockdown%2520of%2520lactate%2520dehydrogenase%2520A%2520suppresses%2520tumor%2520growth%2520and%2520metastasis%2520of%2520human%2520hepatocellular%2520carcinoma%26jtitle%3DFEBS%2520J.%26date%3D2012%26volume%3D279%26spage%3D3898%26epage%3D3910%26doi%3D10.1111%2Fj.1742-4658.2012.08748.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purkey, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robarge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Nagro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pusapati, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salphati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanderbilt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daemen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T.</span></span> <span> </span><span class="NLM_article-title">Metabolic plasticity underpins innate and acquired resistance to LDHA inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">786</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1038%2Fnchembio.2143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=27479743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=779-786&author=A.+Boudreauauthor=H.+E.+Purkeyauthor=A.+Hitzauthor=K.+Robargeauthor=D.+Petersonauthor=S.+Labadieauthor=M.+Kwongauthor=R.+Hongauthor=M.+Gaoauthor=C.+Del+Nagroauthor=R.+Pusapatiauthor=S.+Maauthor=L.+Salphatiauthor=J.+Pangauthor=A.+Zhouauthor=T.+Laiauthor=Y.+Liauthor=Z.+Chenauthor=B.+Weiauthor=I.+Yenauthor=S.+Siderisauthor=M.+McClelandauthor=R.+Firesteinauthor=L.+Corsonauthor=A.+Vanderbiltauthor=S.+Williamsauthor=A.+Daemenauthor=M.+Belvinauthor=C.+Eigenbrotauthor=P.+K.+Jacksonauthor=S.+Malekauthor=G.+Hatzivassiliouauthor=D.+Sampathauthor=M.+Evangelistaauthor=T.+O%E2%80%99Brien&title=Metabolic+plasticity+underpins+innate+and+acquired+resistance+to+LDHA+inhibition&doi=10.1038%2Fnchembio.2143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic plasticity underpins innate and acquired resistance to LDHA inhibition</span></div><div class="casAuthors">Boudreau, Aaron; Purkey, Hans E.; Hitz, Anna; Robarge, Kirk; Peterson, David; Labadie, Sharada; Kwong, Mandy; Hong, Rebecca; Gao, Min; Del Nagro, Christopher; Pusapati, Raju; Ma, Shuguang; Salphati, Laurent; Pang, Jodie; Zhou, Aihe; Lai, Tommy; Li, Yingjie; Chen, Zhongguo; Wei, Binqing; Yen, Ivana; Sideris, Steve; McCleland, Mark; Firestein, Ron; Corson, Laura; Vanderbilt, Alex; Williams, Simon; Daemen, Anneleen; Belvin, Marcia; Eigenbrot, Charles; Jackson, Peter K.; Malek, Shiva; Hatzivassiliou, Georgia; Sampath, Deepak; Evangelista, Marie; O'Brien, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">779-786</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Metabolic reprogramming in tumors represents a potential therapeutic target.  Herein we used shRNA depletion and a novel lactate dehydrogenase (LDHA) inhibitor, GNE-140, to probe the role of LDHA in tumor growth in vitro and in vivo.  In MIA PaCa-2 human pancreatic cells, LDHA inhibition rapidly affected global metab., although cell death only occurred after 2 d of continuous LDHA inhibition.  Pancreatic cell lines that utilize oxidative phosphorylation (OXPHOS) rather than glycolysis were inherently resistant to GNE-140, but could be resensitized to GNE-140 with the OXPHOS inhibitor phenformin.  Acquired resistance to GNE-140 was driven by activation of the AMPK-mTOR-S6K signaling pathway, which led to increased OXPHOS, and inhibitors targeting this pathway could prevent resistance.  Thus, combining an LDHA inhibitor with compds. targeting the mitochondrial or AMPK-S6K signaling axis may not only broaden the clin. utility of LDHA inhibitors beyond glycolytically dependent tumors but also reduce the emergence of resistance to LDHA inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4wdebEpeIa7Vg90H21EOLACvtfcHk0lhAbPtuGfih6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu73K&md5=28abbd7f174b422b7207a5f80352ce3e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2143%26sid%3Dliteratum%253Aachs%26aulast%3DBoudreau%26aufirst%3DA.%26aulast%3DPurkey%26aufirst%3DH.%2BE.%26aulast%3DHitz%26aufirst%3DA.%26aulast%3DRobarge%26aufirst%3DK.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DKwong%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DDel%2BNagro%26aufirst%3DC.%26aulast%3DPusapati%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DSalphati%26aufirst%3DL.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DA.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DMcCleland%26aufirst%3DM.%26aulast%3DFirestein%26aufirst%3DR.%26aulast%3DCorson%26aufirst%3DL.%26aulast%3DVanderbilt%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DS.%26aulast%3DDaemen%26aufirst%3DA.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DJackson%26aufirst%3DP.%2BK.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DEvangelista%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26atitle%3DMetabolic%2520plasticity%2520underpins%2520innate%2520and%2520acquired%2520resistance%2520to%2520LDHA%2520inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D779%26epage%3D786%26doi%3D10.1038%2Fnchembio.2143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Pierre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, P.</span></span> <span> </span><span class="NLM_article-title">Attenuation of LDH-A expression uncovers a link between glycolysis, mitochondrial physiology, and tumor maintenance</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">425</span>– <span class="NLM_lpage">434</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2006.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.ccr.2006.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=16766262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BD28Xmt1eksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2006&pages=425-434&author=V.+R.+Fantinauthor=J.+St-Pierreauthor=P.+Leder&title=Attenuation+of+LDH-A+expression+uncovers+a+link+between+glycolysis%2C+mitochondrial+physiology%2C+and+tumor+maintenance&doi=10.1016%2Fj.ccr.2006.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Attenuation of LDH-A expression uncovers a link between glycolysis, mitochondrial physiology, and tumor maintenance</span></div><div class="casAuthors">Fantin, Valeria R.; St-Pierre, Julie; Leder, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">425-434</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Alterations in cellular metab. are among the most consistent hallmarks of cancer.  Herein we have investigated the relationship between increased aerobic lactate prodn. and mitochondrial physiol. in tumor cells.  To diminish the ability of malignant cells to metabolize pyruvate to lactate, lactate dehydrogenase A (LDH-A) levels were knocked down by means of LDH-A short hairpin RNAs.  Redn. in LDH-A activity resulted in stimulation of mitochondrial respiration and decrease of mitochondrial membrane potential.  It also compromised the ability of these tumor cells to proliferate under hypoxia.  The tumorigenicity of the LDH-A-deficient cells was severely diminished, and this phenotype was reversed by complementation with the human ortholog LDH-A protein.  These results demonstrate that LDH-A plays a key role in tumor maintenance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDQMn1zZZKr7Vg90H21EOLACvtfcHk0lhAbPtuGfih6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xmt1eksLk%253D&md5=6ba9ae345a18d1b97a58a729d0bd35f6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2006.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2006.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DSt-Pierre%26aufirst%3DJ.%26aulast%3DLeder%26aufirst%3DP.%26atitle%3DAttenuation%2520of%2520LDH-A%2520expression%2520uncovers%2520a%2520link%2520between%2520glycolysis%252C%2520mitochondrial%2520physiology%252C%2520and%2520tumor%2520maintenance%26jtitle%3DCancer%2520Cell%26date%3D2006%26volume%3D9%26spage%3D425%26epage%3D434%26doi%3D10.1016%2Fj.ccr.2006.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanai, J.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kats, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhargava, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billiard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorkiewicz, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schatzman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousamra, M.,  2nd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandolfi, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukhatme, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, P.</span></span> <span> </span><span class="NLM_article-title">Targeting lactate dehydrogenase--a inhibits tumorigenesis and tumor progression in mouse models of lung cancer and impacts tumor-initiating cells</span>. <i>Cell Metabol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">809</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2014.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.cmet.2014.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24726384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVWrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=795-809&author=H.+Xieauthor=J.-i.+Hanaiauthor=J.-G.+Renauthor=L.+Katsauthor=K.+Burgessauthor=P.+Bhargavaauthor=S.+Signorettiauthor=J.+Billiardauthor=K.+J.+Duffyauthor=A.+Grantauthor=X.+Wangauthor=P.+K.+Lorkiewiczauthor=S.+Schatzmanauthor=M.+Bousamraauthor=A.+N.+Laneauthor=R.+M.+Higashiauthor=T.+W.+M.+Fanauthor=P.+P.+Pandolfiauthor=V.+P.+Sukhatmeauthor=P.+Seth&title=Targeting+lactate+dehydrogenase%2D%2Da+inhibits+tumorigenesis+and+tumor+progression+in+mouse+models+of+lung+cancer+and+impacts+tumor-initiating+cells&doi=10.1016%2Fj.cmet.2014.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Lactate Dehydrogenase-A Inhibits Tumorigenesis and Tumor Progression in Mouse Models of Lung Cancer and Impacts Tumor-Initiating Cells</span></div><div class="casAuthors">Xie, Han; Hanai, Jun-ichi; Ren, Jian-Guo; Kats, Lev; Burgess, Kerri; Bhargava, Parul; Signoretti, Sabina; Billiard, Julia; Duffy, Kevin J.; Grant, Aaron; Wang, Xiaoen; Lorkiewicz, Pawel K.; Schatzman, Sabrina; Bousamra, Michael; Lane, Andrew N.; Higashi, Richard M.; Fan, Teresa W. M.; Pandolfi, Pier Paolo; Sukhatme, Vikas P.; Seth, Pankaj</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">795-809</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lactate dehydrogenase-A (LDH-A) enzyme catalyzes the interconversion of pyruvate and lactate, is upregulated in human cancers, and is assocd. with aggressive tumor outcomes.  Here we use an inducible murine model and demonstrate that inactivation of LDH-A in mouse models of NSCLC driven by oncogenic K-RAS or EGFR leads to decreased tumorigenesis and disease regression in established tumors.  We also show that abrogation of LDH-A results in reprogramming of pyruvate metab., with decreased lactic fermn. in vitro, in vivo, and ex vivo.  This was accompanied by reactivation of mitochondrial function in vitro, but not in vivo or ex vivo.  Finally, using a specific small mol. LDH-A inhibitor, we demonstrated that LDH-A is essential for cancer-initiating cell survival and proliferation.  Thus, LDH-A can be a viable therapeutic target for NSCLC, including cancer stem cell-dependent drug-resistant tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNiWJNOeWwmLVg90H21EOLACvtfcHk0lhCJW6zn5QegA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVWrtLY%253D&md5=d80624d60895f5a6c01106841ebd1af9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2014.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2014.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DH.%26aulast%3DHanai%26aufirst%3DJ.-i.%26aulast%3DRen%26aufirst%3DJ.-G.%26aulast%3DKats%26aufirst%3DL.%26aulast%3DBurgess%26aufirst%3DK.%26aulast%3DBhargava%26aufirst%3DP.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DBilliard%26aufirst%3DJ.%26aulast%3DDuffy%26aufirst%3DK.%2BJ.%26aulast%3DGrant%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLorkiewicz%26aufirst%3DP.%2BK.%26aulast%3DSchatzman%26aufirst%3DS.%26aulast%3DBousamra%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DA.%2BN.%26aulast%3DHigashi%26aufirst%3DR.%2BM.%26aulast%3DFan%26aufirst%3DT.%2BW.%2BM.%26aulast%3DPandolfi%26aufirst%3DP.%2BP.%26aulast%3DSukhatme%26aufirst%3DV.%2BP.%26aulast%3DSeth%26aufirst%3DP.%26atitle%3DTargeting%2520lactate%2520dehydrogenase--a%2520inhibits%2520tumorigenesis%2520and%2520tumor%2520progression%2520in%2520mouse%2520models%2520of%2520lung%2520cancer%2520and%2520impacts%2520tumor-initiating%2520cells%26jtitle%3DCell%2520Metabol.%26date%3D2014%26volume%3D19%26spage%3D795%26epage%3D809%26doi%3D10.1016%2Fj.cmet.2014.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouw, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinavahi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deck, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royer, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Jagt, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semenza, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span> <span> </span><span class="NLM_article-title">Inhibition of lactate dehydrogenase A induces oxidative stress and inhibits tumor progression</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">2037</span>– <span class="NLM_lpage">2042</span>, <span class="refDoi"> DOI: 10.1073/pnas.0914433107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1073%2Fpnas.0914433107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=20133848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFCit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=2037-2042&author=A.+Leauthor=C.+R.+Cooperauthor=A.+M.+Gouwauthor=R.+Dinavahiauthor=A.+Maitraauthor=L.+M.+Deckauthor=R.+E.+Royerauthor=D.+L.+Vander+Jagtauthor=G.+L.+Semenzaauthor=C.+V.+Dang&title=Inhibition+of+lactate+dehydrogenase+A+induces+oxidative+stress+and+inhibits+tumor+progression&doi=10.1073%2Fpnas.0914433107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lactate dehydrogenase A induces oxidative stress and inhibits tumor progression</span></div><div class="casAuthors">Le, Anne; Cooper, Charles R.; Gouw, Arvin M.; Dinavahi, Ramani; Maitra, Anirban; Deck, Lorraine M.; Royer, Robert E.; Vander Jagt, David L.; Semenza, Gregg L.; Dang, Chi V.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2037-2042, S2037/1-S2037/12</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the result of genetic alterations and tumor hypoxia, many cancer cells avidly take up glucose and generate lactate through lactate dehydrogenase A (LDHA), which is encoded by a target gene of c-Myc and hypoxia-inducible factor (HIF-1).  Previous studies with redn. of LDHA expression indicate that LDHA is involved in tumor initiation, but its role in tumor maintenance and progression has not been established.  Furthermore, how redn. of LDHA expression by interference or antisense RNA inhibits tumorigenesis is not well understood.  Here, we report that redn. of LDHA by siRNA or its inhibition by a small-mol. inhibitor (FX11 [3-dihydroxy-6-methyl-7-(phenylmethyl)-4-propylnaphthalene-1-carboxylic acid]) reduced ATP levels and induced significant oxidative stress and cell death that could be partially reversed by the antioxidant N-acetylcysteine (NAC).  Furthermore, we document that FX11 inhibited the progression of sizable human lymphoma and pancreatic cancer xenografts.  When used in combination with the NAD+ synthesis inhibitor FK866, FX11 induced lymphoma regression.  Hence, inhibition of LDHA with FX11 is an achievable and tolerable treatment for LDHA-dependent tumors.  Our studies document a therapeutical approach to the Warburg effect and demonstrate that oxidative stress and metabolic phenotyping of cancers are crit. aspects of cancer biol. to consider for the therapeutical targeting of cancer energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1h1YikQKAn7Vg90H21EOLACvtfcHk0lhCJW6zn5QegA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFCit7g%253D&md5=714af2ec2bc79ffcba57d6987c52a87a</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0914433107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0914433107%26sid%3Dliteratum%253Aachs%26aulast%3DLe%26aufirst%3DA.%26aulast%3DCooper%26aufirst%3DC.%2BR.%26aulast%3DGouw%26aufirst%3DA.%2BM.%26aulast%3DDinavahi%26aufirst%3DR.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DDeck%26aufirst%3DL.%2BM.%26aulast%3DRoyer%26aufirst%3DR.%2BE.%26aulast%3DVander%2BJagt%26aufirst%3DD.%2BL.%26aulast%3DSemenza%26aufirst%3DG.%2BL.%26aulast%3DDang%26aufirst%3DC.%2BV.%26atitle%3DInhibition%2520of%2520lactate%2520dehydrogenase%2520A%2520induces%2520oxidative%2520stress%2520and%2520inhibits%2520tumor%2520progression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D2037%26epage%3D2042%26doi%3D10.1073%2Fpnas.0914433107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit13b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Granchi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacomelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuccinardi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannaccini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucacchini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">León, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giovannetti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palchaudhuri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvaresi, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minutolo, F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-hydroxyindole-based inhibitors of human lactate dehydrogenase isoform A (LDH-A) as starvation agents against cancer cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1021/jm101007q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101007q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3MXitFWrtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1599-1612&author=C.+Granchiauthor=S.+Royauthor=C.+Giacomelliauthor=M.+Macchiaauthor=T.+Tuccinardiauthor=A.+Martinelliauthor=M.+Lanzaauthor=L.+Bettiauthor=G.+Giannacciniauthor=A.+Lucacchiniauthor=N.+Funelauthor=L.+G.+Le%C3%B3nauthor=E.+Giovannettiauthor=G.+J.+Petersauthor=R.+Palchaudhuriauthor=E.+C.+Calvaresiauthor=P.+J.+Hergenrotherauthor=F.+Minutolo&title=Discovery+of+N-hydroxyindole-based+inhibitors+of+human+lactate+dehydrogenase+isoform+A+%28LDH-A%29+as+starvation+agents+against+cancer+cells&doi=10.1021%2Fjm101007q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Hydroxyindole-Based Inhibitors of Human Lactate Dehydrogenase Isoform A (LDH-A) as Starvation Agents against Cancer Cells</span></div><div class="casAuthors">Granchi, Carlotta; Roy, Sarabindu; Giacomelli, Chiara; Macchia, Marco; Tuccinardi, Tiziano; Martinelli, Adriano; Lanza, Mario; Betti, Laura; Giannaccini, Gino; Lucacchini, Antonio; Funel, Nicola; Leon, Leticia G.; Giovannetti, Elisa; Peters, Godefridus J.; Palchaudhuri, Rahul; Calvaresi, Emilia C.; Hergenrother, Paul J.; Minutolo, Filippo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1599-1612</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Highly invasive tumor cells are characterized by a metabolic switch, known as the Warburg effect, from "normal" oxidative phosphorylation to increased glycolysis even under sufficiently oxygenated conditions.  This dependence on glycolysis also confers a growth advantage to cells present in hypoxic regions of the tumor.  One of the key enzymes involved in glycolysis, the muscle isoform of lactate dehydrogenase (LDH-A), is overexpressed by metastatic cancer cells and is linked to the vitality of tumors in hypoxia.  This enzyme may be considered as a potential target for new anticancer agents, since its inhibition cuts cancer energetic and anabolic supply, thus reducing the metastatic and invasive potential of cancer cells.  We have discovered new and efficient N-hydroxyindole-based inhibitors of LDH-A, which are isoform-selective (over LDH-B) and competitive with both the substrate (pyruvate) and the cofactor (NADH).  The antiproliferative activity of these compds. was confirmed on a series of cancer cell lines, and they proved to be particularly effective under hypoxic conditions.  Moreover, NMR expts. showed that these compds. are able to reduce the glucose-to-lactate conversion inside the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97RT2SmH8ZLVg90H21EOLACvtfcHk0lhCJW6zn5QegA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXitFWrtrs%253D&md5=7fbe13667567f20567c23087cb66fbf5</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1021%2Fjm101007q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101007q%26sid%3Dliteratum%253Aachs%26aulast%3DGranchi%26aufirst%3DC.%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DGiacomelli%26aufirst%3DC.%26aulast%3DMacchia%26aufirst%3DM.%26aulast%3DTuccinardi%26aufirst%3DT.%26aulast%3DMartinelli%26aufirst%3DA.%26aulast%3DLanza%26aufirst%3DM.%26aulast%3DBetti%26aufirst%3DL.%26aulast%3DGiannaccini%26aufirst%3DG.%26aulast%3DLucacchini%26aufirst%3DA.%26aulast%3DFunel%26aufirst%3DN.%26aulast%3DLe%25C3%25B3n%26aufirst%3DL.%2BG.%26aulast%3DGiovannetti%26aufirst%3DE.%26aulast%3DPeters%26aufirst%3DG.%2BJ.%26aulast%3DPalchaudhuri%26aufirst%3DR.%26aulast%3DCalvaresi%26aufirst%3DE.%2BC.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26aulast%3DMinutolo%26aufirst%3DF.%26atitle%3DDiscovery%2520of%2520N-hydroxyindole-based%2520inhibitors%2520of%2520human%2520lactate%2520dehydrogenase%2520isoform%2520A%2520%2528LDH-A%2529%2520as%2520starvation%2520agents%2520against%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1599%26epage%3D1612%26doi%3D10.1021%2Fjm101007q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squillace, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Commodore, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenfield, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miret, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D. C.</span></span> <span> </span><span class="NLM_article-title">Fragment growing and linking lead to novel nanomolar lactate dehydrogenase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1023</span>– <span class="NLM_lpage">1040</span>, <span class="refDoi"> DOI: 10.1021/jm3014844</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3014844" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXms1OlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1023-1040&author=A.+Kohlmannauthor=S.+G.+Zechauthor=F.+Liauthor=T.+Zhouauthor=R.+M.+Squillaceauthor=L.+Commodoreauthor=M.+T.+Greenfieldauthor=X.+Luauthor=D.+P.+Millerauthor=W.-S.+Huangauthor=J.+Qiauthor=R.+M.+Thomasauthor=Y.+Wangauthor=S.+Zhangauthor=R.+Doddauthor=S.+Liuauthor=R.+Xuauthor=Y.+Xuauthor=J.+J.+Miretauthor=V.+Riveraauthor=T.+Clacksonauthor=W.+C.+Shakespeareauthor=X.+Zhuauthor=D.+C.+Dalgarno&title=Fragment+growing+and+linking+lead+to+novel+nanomolar+lactate+dehydrogenase+inhibitors&doi=10.1021%2Fjm3014844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Growing and Linking Lead to Novel Nanomolar Lactate Dehydrogenase Inhibitors</span></div><div class="casAuthors">Kohlmann, Anna; Zech, Stephan G.; Li, Feng; Zhou, Tianjun; Squillace, Rachel M.; Commodore, Lois; Greenfield, Matthew T.; Lu, Xiaohui; Miller, David P.; Huang, Wei-Sheng; Qi, Jiwei; Thomas, R. Mathew; Wang, Yihan; Zhang, Sen; Dodd, Rory; Liu, Shuangying; Xu, Rongsong; Xu, Yongjin; Miret, Juan J.; Rivera, Victor; Clackson, Tim; Shakespeare, William C.; Zhu, Xiaotian; Dalgarno, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1023-1040</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lactate dehydrogenase A (LDH-A) catalyzes the interconversion of lactate and pyruvate in the glycolysis pathway.  Cancer cells rely heavily on glycolysis instead of oxidative phosphorylation to generate ATP, a phenomenon known as the Warburg effect.  The inhibition of LDH-A by small mols. is therefore of interest for potential cancer treatments.  We describe the identification and optimization of LDH-A inhibitors by fragment-based drug discovery.  We applied ligand based NMR screening to identify low affinity fragments binding to LDH-A.  The dissocn. consts. (Kd) and enzyme inhibition (IC50) of fragment hits were measured by surface plasmon resonance (SPR) and enzyme assays, resp.  The binding modes of selected fragments were investigated by X-ray crystallog.  Fragment growing and linking, followed by chem. optimization, resulted in nanomolar LDH-A inhibitors that demonstrated stoichiometric binding to LDH-A.  Selected mols. inhibited lactate prodn. in cells, suggesting target-specific inhibition in cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pFkSoe2_ALVg90H21EOLACvtfcHk0ljIApKWZefC1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXms1OlsA%253D%253D&md5=3a6b0b46318ae2d6c2a7c34e1575ada6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm3014844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3014844%26sid%3Dliteratum%253Aachs%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DZech%26aufirst%3DS.%2BG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DSquillace%26aufirst%3DR.%2BM.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DGreenfield%26aufirst%3DM.%2BT.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DMiller%26aufirst%3DD.%2BP.%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DDodd%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMiret%26aufirst%3DJ.%2BJ.%26aulast%3DRivera%26aufirst%3DV.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26atitle%3DFragment%2520growing%2520and%2520linking%2520lead%2520to%2520novel%2520nanomolar%2520lactate%2520dehydrogenase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1023%26epage%3D1040%26doi%3D10.1021%2Fjm3014844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchlow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrosek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogtherr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel lactate dehydrogenase A inhibitors by fragment-based lead generation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">3285</span>– <span class="NLM_lpage">3306</span>, <span class="refDoi"> DOI: 10.1021/jm201734r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201734r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvVSgtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3285-3306&author=R.+A.+Wardauthor=C.+Brassingtonauthor=A.+L.+Breezeauthor=A.+Caputoauthor=S.+Critchlowauthor=G.+Daviesauthor=L.+Goodwinauthor=G.+Hassallauthor=R.+Greenwoodauthor=G.+A.+Holdgateauthor=M.+Mrosekauthor=R.+A.+Normanauthor=S.+Pearsonauthor=J.+Tartauthor=J.+A.+Tuckerauthor=M.+Vogtherrauthor=D.+Whittakerauthor=J.+Wingfieldauthor=J.+Winterauthor=K.+Hudson&title=Design+and+synthesis+of+novel+lactate+dehydrogenase+A+inhibitors+by+fragment-based+lead+generation&doi=10.1021%2Fjm201734r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Lactate Dehydrogenase A Inhibitors by Fragment-Based Lead Generation</span></div><div class="casAuthors">Ward, Richard A.; Brassington, Claire; Breeze, Alexander L.; Caputo, Alessandro; Critchlow, Susan; Davies, Gareth; Goodwin, Louise; Hassall, Giles; Greenwood, Ryan; Holdgate, Geoffrey A.; Mrosek, Michael; Norman, Richard A.; Pearson, Stuart; Tart, Jonathan; Tucker, Julie A.; Vogtherr, Martin; Whittaker, David; Wingfield, Jonathan; Winter, Jon; Hudson, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3285-3306</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lactate dehydrogenase A (LDHA) catalyzes the conversion of pyruvate to lactate, utilizing NADH as a cofactor.  It has been identified as a potential therapeutic target in the area of cancer metab.  In this manuscript we report our progress using fragment-based lead generation (FBLG), assisted by x-ray crystallog. to develop small mol. LDHA inhibitors.  Fragment hits were identified through NMR and SPR screening and optimized into lead compds. with nanomolar binding affinities via fragment linking.  Also reported is their modification into cellular active compds. suitable for target validation work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH8QbdIUXNvbVg90H21EOLACvtfcHk0ljIApKWZefC1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvVSgtLs%253D&md5=d638e7a0db361c212ee084d5c05ee70c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm201734r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201734r%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DBrassington%26aufirst%3DC.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26aulast%3DCaputo%26aufirst%3DA.%26aulast%3DCritchlow%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DGoodwin%26aufirst%3DL.%26aulast%3DHassall%26aufirst%3DG.%26aulast%3DGreenwood%26aufirst%3DR.%26aulast%3DHoldgate%26aufirst%3DG.%2BA.%26aulast%3DMrosek%26aufirst%3DM.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DPearson%26aufirst%3DS.%26aulast%3DTart%26aufirst%3DJ.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DVogtherr%26aufirst%3DM.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWingfield%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DJ.%26aulast%3DHudson%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520lactate%2520dehydrogenase%2520A%2520inhibitors%2520by%2520fragment-based%2520lead%2520generation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3285%26epage%3D3306%26doi%3D10.1021%2Fjm201734r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Billiard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennison, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colón, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greshock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McSurdy-Freed, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orband-Miller, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waitt, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooster, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, K. J.</span></span> <span> </span><span class="NLM_article-title">Quinoline 3-sulfonamides inhibit lactate dehydrogenase A and reverse aerobic glycolysis in cancer cells</span>. <i>Canc. Metabol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1186/2049-3002-1-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1186%2F2049-3002-1-19" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24280423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A280%3ADC%252BC2c3ntFCluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=19-36&author=J.+Billiardauthor=J.+B.+Dennisonauthor=J.+Briandauthor=R.+S.+Annanauthor=D.+Chaiauthor=M.+Col%C3%B3nauthor=C.+S.+Dodsonauthor=S.+A.+Gilbertauthor=J.+Greshockauthor=J.+Jingauthor=H.+Luauthor=J.+E.+McSurdy-Freedauthor=L.+A.+Orband-Millerauthor=G.+B.+Millsauthor=C.+J.+Quinnauthor=J.+L.+Schneckauthor=G.+F.+Scottauthor=A.+N.+Shawauthor=G.+M.+Waittauthor=R.+F.+Woosterauthor=K.+J.+Duffy&title=Quinoline+3-sulfonamides+inhibit+lactate+dehydrogenase+A+and+reverse+aerobic+glycolysis+in+cancer+cells&doi=10.1186%2F2049-3002-1-19"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Quinoline 3-sulfonamides inhibit lactate dehydrogenase A and reverse aerobic glycolysis in cancer cells</span></div><div class="casAuthors">Billiard Julia; Dennison Jennifer B; Briand Jacques; Annan Roland S; Chai Deping; Colon Mariela; Dodson Christopher S; Gilbert Seth A; Greshock Joel; Jing Junping; Lu Hong; McSurdy-Freed Jeanelle E; Orband-Miller Lisa A; Mills Gordon B; Quinn Chad J; Schneck Jessica L; Scott Gilbert F; Shaw Anthony N; Waitt Gregory M; Wooster Richard F; Duffy Kevin J</div><div class="citationInfo"><span class="NLM_cas:title">Cancer & metabolism</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>
        ISSN:<span class="NLM_cas:issn">2049-3002</span>.
    </div><div class="casAbstract">BACKGROUND:  Most normal cells in the presence of oxygen utilize glucose for mitochondrial oxidative phosphorylation.  In contrast, many cancer cells rapidly convert glucose to lactate in the cytosol, a process termed aerobic glycolysis.  This glycolytic phenotype is enabled by lactate dehydrogenase (LDH), which catalyzes the inter-conversion of pyruvate and lactate.  The purpose of this study was to identify and characterize potent and selective inhibitors of LDHA.  METHODS:  High throughput screening and lead optimization were used to generate inhibitors of LDHA enzymatic activity.  Effects of these inhibitors on metabolism were evaluated using cell-based lactate production, oxygen consumption, and 13C NMR spectroscopy assays.  Changes in comprehensive metabolic profile, cell proliferation, and apoptosis were assessed upon compound treatment.  RESULTS:  3-((3-carbamoyl-7-(3,5-dimethylisoxazol-4-yl)-6-methoxyquinolin-4-yl) amino) benzoic acid was identified as an NADH-competitive LDHA inhibitor.  Lead optimization yielded molecules with LDHA inhibitory potencies as low as 2 nM and 10 to 80-fold selectivity over LDHB.  Molecules in this family rapidly and profoundly inhibited lactate production rates in multiple cancer cell lines including hepatocellular and breast carcinomas.  Consistent with selective inhibition of LDHA, the most sensitive breast cancer cell lines to lactate inhibition in hypoxic conditions were cells with low expression of LDHB.  Our inhibitors increased rates of oxygen consumption in hepatocellular carcinoma cells at doses up to 3 microM, while higher concentrations directly inhibited mitochondrial function.  Analysis of more than 500 metabolites upon LDHA inhibition in Snu398 cells revealed that intracellular concentrations of glycolysis and citric acid cycle intermediates were increased, consistent with enhanced Krebs cycle activity and blockage of cytosolic glycolysis.  Treatment with these compounds also potentiated PKM2 activity and promoted apoptosis in Snu398 cells.  CONCLUSIONS:  Rapid chemical inhibition of LDHA by these quinoline 3-sulfonamids led to profound metabolic alterations and impaired cell survival in carcinoma cells making it a compelling strategy for treating solid tumors that rely on aerobic glycolysis for survival.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTY101Isa9tY-Lt9HypzAgQfW6udTcc2eY5k7hvdaa4c7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3ntFCluw%253D%253D&md5=588288defc04ed3927e4c4fdb891d6cd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2F2049-3002-1-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2049-3002-1-19%26sid%3Dliteratum%253Aachs%26aulast%3DBilliard%26aufirst%3DJ.%26aulast%3DDennison%26aufirst%3DJ.%2BB.%26aulast%3DBriand%26aufirst%3DJ.%26aulast%3DAnnan%26aufirst%3DR.%2BS.%26aulast%3DChai%26aufirst%3DD.%26aulast%3DCol%25C3%25B3n%26aufirst%3DM.%26aulast%3DDodson%26aufirst%3DC.%2BS.%26aulast%3DGilbert%26aufirst%3DS.%2BA.%26aulast%3DGreshock%26aufirst%3DJ.%26aulast%3DJing%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DMcSurdy-Freed%26aufirst%3DJ.%2BE.%26aulast%3DOrband-Miller%26aufirst%3DL.%2BA.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DQuinn%26aufirst%3DC.%2BJ.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DScott%26aufirst%3DG.%2BF.%26aulast%3DShaw%26aufirst%3DA.%2BN.%26aulast%3DWaitt%26aufirst%3DG.%2BM.%26aulast%3DWooster%26aufirst%3DR.%2BF.%26aulast%3DDuffy%26aufirst%3DK.%2BJ.%26atitle%3DQuinoline%25203-sulfonamides%2520inhibit%2520lactate%2520dehydrogenase%2520A%2520and%2520reverse%2520aerobic%2520glycolysis%2520in%2520cancer%2520cells%26jtitle%3DCanc.%2520Metabol.%26date%3D2013%26volume%3D1%26spage%3D19%26epage%3D36%26doi%3D10.1186%2F2049-3002-1-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannetti, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderPorten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Q.</span></span> <span> </span><span class="NLM_article-title">Identification of 2-amino-5-aryl-pyrazines as inhibitors of human lactate dehydrogenase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5533</span>– <span class="NLM_lpage">5539</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.08.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.bmcl.2013.08.060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=24012183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSrsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=5533-5539&author=B.+P.+Fauberauthor=P.+S.+Dragovichauthor=J.+Chenauthor=L.+B.+Corsonauthor=C.+Z.+Dingauthor=C.+Eigenbrotauthor=A.+M.+Giannettiauthor=T.+Hunsakerauthor=S.+Labadieauthor=Y.+Liuauthor=Y.+Liuauthor=S.+Malekauthor=D.+Petersonauthor=K.+Pittsauthor=S.+Siderisauthor=M.+Ultschauthor=E.+VanderPortenauthor=J.+Wangauthor=B.+Weiauthor=I.+Yenauthor=Q.+Yue&title=Identification+of+2-amino-5-aryl-pyrazines+as+inhibitors+of+human+lactate+dehydrogenase&doi=10.1016%2Fj.bmcl.2013.08.060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of 2-amino-5-aryl-pyrazines as inhibitors of human lactate dehydrogenase</span></div><div class="casAuthors">Fauber, Benjamin P.; Dragovich, Peter S.; Chen, Jinhua; Corson, Laura B.; Ding, Charles Z.; Eigenbrot, Charles; Giannetti, Anthony M.; Hunsaker, Thomas; Labadie, Sharada; Liu, Yichin; Liu, Yingchun; Malek, Shiva; Peterson, David; Pitts, Keith; Sideris, Steve; Ultsch, Mark; VanderPorten, Erica; Wang, Jing; Wei, BinQing; Yen, Ivana; Yue, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5533-5539</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A 2-amino-5-aryl-pyrazine, e.g. I [R = 2-O2CC6H4, 3-O2CC6H4, 4-O2CC6H4, etc.] was identified as an inhibitor of human lactate dehydrogenase A (LDHA) via a biochem. screening campaign.  Biochem. and biophys. expts. demonstrated that the compd. specifically interacted with human LDHA.  Structural variation of the screening hit resulted in improvements in LDHA biochem. inhibition and pharmacokinetic properties.  A crystal structure of an improved compd. bound to human LDHA was also obtained and it explained many of the obsd. structure-activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowVTwZ-VbwxrVg90H21EOLACvtfcHk0lhk7TE9btdb6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSrsrvI&md5=80d750198bae55b872c58b771af64b9d</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.08.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.08.060%26sid%3Dliteratum%253Aachs%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCorson%26aufirst%3DL.%2BB.%26aulast%3DDing%26aufirst%3DC.%2BZ.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DHunsaker%26aufirst%3DT.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DPitts%26aufirst%3DK.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DVanderPorten%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DYue%26aufirst%3DQ.%26atitle%3DIdentification%2520of%25202-amino-5-aryl-pyrazines%2520as%2520inhibitors%2520of%2520human%2520lactate%2520dehydrogenase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D5533%26epage%3D5539%26doi%3D10.1016%2Fj.bmcl.2013.08.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit17b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannetti, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purkey, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderPorten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">Identification of substituted 2-thio-6-oxo-1,6-duhydropyrimidines as inhibitors of human lactate dehydrogenase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3186</span>– <span class="NLM_lpage">3194</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.bmcl.2013.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=23628333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvV2itr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3186-3194&author=P.+S.+Dragovichauthor=B.+P.+Fauberauthor=L.+B.+Corsonauthor=C.+Z.+Dingauthor=C.+Eigenbrotauthor=H.+Geauthor=A.+M.+Giannettiauthor=T.+Hunsakerauthor=S.+Labadieauthor=Y.+Liuauthor=S.+Malekauthor=B.+Panauthor=D.+Petersonauthor=K.+Pittsauthor=H.+E.+Purkeyauthor=S.+Siderisauthor=M.+Ultschauthor=E.+VanderPortenauthor=B.+Weiauthor=Q.+Xuauthor=I.+Yenauthor=Q.+Yueauthor=H.+Zhangauthor=X.+Zhang&title=Identification+of+substituted+2-thio-6-oxo-1%2C6-duhydropyrimidines+as+inhibitors+of+human+lactate+dehydrogenase&doi=10.1016%2Fj.bmcl.2013.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17bR"><div class="casContent"><span class="casTitleNuber">17b</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of substituted 2-thio-6-oxo-1,6-dihydropyrimidines as inhibitors of human lactate dehydrogenase</span></div><div class="casAuthors">Dragovich, Peter S.; Fauber, Benjamin P.; Corson, Laura B.; Ding, Charles Z.; Eigenbrot, Charles; Ge, HongXiu; Giannetti, Anthony M.; Hunsaker, Thomas; Labadie, Sharada; Liu, Yichin; Malek, Shiva; Pan, Borlan; Peterson, David; Pitts, Keith; Purkey, Hans E.; Sideris, Steve; Ultsch, Mark; VanderPorten, Erica; Wei, BinQing; Xu, Qing; Yen, Ivana; Yue, Qin; Zhang, Huihui; Zhang, Xuying</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3186-3194</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel 2-thio-6-oxo-1,6-dihydropyrimidine-contg. inhibitor of human lactate dehydrogenase (LDH) was identified by high-throughput screening (IC50 = 8.1 μM).  Biochem., surface plasmon resonance, and satn. transfer difference NMR expts. indicated that the compd. specifically assocd. with human LDHA in a manner that required simultaneous binding of the NADH co-factor.  Structural variation of the screening hit resulted in significant improvements in LDHA biochem. inhibition activity (best IC50 = 0.48 μM).  A crystal structure of an optimized compd. bound to human LDHA was obtained and explained many of the obsd. structure-activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e6oSsqyPHrVg90H21EOLACvtfcHk0lheEL5d4llrOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvV2itr8%253D&md5=5367ac82c7f5583c8faaf899f966e621</span></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DCorson%26aufirst%3DL.%2BB.%26aulast%3DDing%26aufirst%3DC.%2BZ.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DHunsaker%26aufirst%3DT.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DPitts%26aufirst%3DK.%26aulast%3DPurkey%26aufirst%3DH.%2BE.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DVanderPorten%26aufirst%3DE.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DYue%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DIdentification%2520of%2520substituted%25202-thio-6-oxo-1%252C6-duhydropyrimidines%2520as%2520inhibitors%2520of%2520human%2520lactate%2520dehydrogenase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3186%26epage%3D3194%26doi%3D10.1016%2Fj.bmcl.2013.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span> <span> </span><span class="NLM_article-title">Development of novel human lactate dehydrogenase A inhibitors: high-throughput screening, synthesis, and biological evaluations</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.05.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.ejmech.2019.05.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=31129449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVKqtbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=105-115&author=Y.+Zhouauthor=P.+Taoauthor=M.+Wangauthor=P.+Xuauthor=W.+Luauthor=P.+Leiauthor=Q.+You&title=Development+of+novel+human+lactate+dehydrogenase+A+inhibitors%3A+high-throughput+screening%2C+synthesis%2C+and+biological+evaluations&doi=10.1016%2Fj.ejmech.2019.05.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of novel human lactate dehydrogenase A inhibitors: high-throughput screening, synthesis, and biological evaluations</span></div><div class="casAuthors">Zhou, Yuan; Tao, Pingde; Wang, Meigui; Xu, Peng; Lu, Wei; Lei, Pan; You, Qiuyun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-115</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Human lactate dehydrogenase A (LDHA) plays a crit. role in the glycolytic process, making the enzyme an ideal of anti-cancer drug target.  Herein, we report the discovery of novel potent LDHA inhibitors by screening an inhouse library.  The hit-to-lead modification enabled us to identify compd. 24c(I), which inhibited LDHA activity with an EC50 value of 90 nM, and reduced MiaPaCa-2 cancer cell proliferation with an IC50 value of 2.1 μM.  In line with the in vitro anticancer activity, I suppressed the tumor growth at a dose of 10 mg/kg in a MiaPaCa-2 cells xenograft model, but with little effect to the mice wt.  Moreover, I strongly inhibited MiaPaCa-2 cell colonies formation, induced MiaPaCa-2 cell apoptosis, and arrested MiaPaCa-2 cell cycle at G2 phase.  In addn., the mitochondrial bioenergetics anal. suggested that I could reprogram cancer cell metabolic pathways from glycolysis to oxidn. phosphorylation, which verified by decreasing the extracellular acidification rates and lactate formation, and increasing oxygen consumption rate in cancer cell.  All these results indicate I is a promising metabolic modulator for the anticancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7PmbP3IKU5LVg90H21EOLACvtfcHk0lheEL5d4llrOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKqtbnM&md5=f619bbf9198b20163a4a4601813a1af0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.05.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.05.033%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLei%26aufirst%3DP.%26aulast%3DYou%26aufirst%3DQ.%26atitle%3DDevelopment%2520of%2520novel%2520human%2520lactate%2520dehydrogenase%2520A%2520inhibitors%253A%2520high-throughput%2520screening%252C%2520synthesis%252C%2520and%2520biological%2520evaluations%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D177%26spage%3D105%26epage%3D115%26doi%3D10.1016%2Fj.ejmech.2019.05.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheff, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouznetsova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benavides, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuenstner, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luci, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dranow, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulikowski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flint, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darley-Usmar, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of potent, cell-active pyrazole-based inhibitors of lactate dehydrogenase (LDH)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9184</span>– <span class="NLM_lpage">9204</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00941</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00941" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslOkurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9184-9204&author=G.+Raiauthor=K.+R.+Brimacombeauthor=B.+T.+Mottauthor=D.+J.+Urbanauthor=X.+Huauthor=S.-M.+Yangauthor=T.+D.+Leeauthor=D.+M.+Cheffauthor=J.+Kouznetsovaauthor=G.+A.+Benavidesauthor=K.+Pohidaauthor=E.+J.+Kuenstnerauthor=D.+K.+Luciauthor=C.+M.+Lukacsauthor=D.+R.+Daviesauthor=D.+M.+Dranowauthor=H.+Zhuauthor=G.+Sulikowskiauthor=W.+J.+Mooreauthor=G.+M.+Stottauthor=A.+J.+Flintauthor=M.+D.+Hallauthor=V.+M.+Darley-Usmarauthor=L.+M.+Neckersauthor=C.+V.+Dangauthor=A.+G.+Watersonauthor=A.+Simeonovauthor=A.+Jadhavauthor=D.+J.+Maloney&title=Discovery+and+optimization+of+potent%2C+cell-active+pyrazole-based+inhibitors+of+lactate+dehydrogenase+%28LDH%29&doi=10.1021%2Facs.jmedchem.7b00941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Potent, Cell-Active Pyrazole-Based Inhibitors of Lactate Dehydrogenase (LDH)</span></div><div class="casAuthors">Rai, Ganesha; Brimacombe, Kyle R.; Mott, Bryan T.; Urban, Daniel J.; Hu, Xin; Yang, Shyh-Ming; Lee, Tobie D.; Cheff, Dorian M.; Kouznetsova, Jennifer; Benavides, Gloria A.; Pohida, Katie; Kuenstner, Eric J.; Luci, Diane K.; Lukacs, Christine M.; Davies, Douglas R.; Dranow, David M.; Zhu, Hu; Sulikowski, Gary; Moore, William J.; Stott, Gordon M.; Flint, Andrew J.; Hall, Matthew D.; Darley-Usmar, Victor M.; Neckers, Leonard M.; Dang, Chi V.; Waterson, Alex G.; Simeonov, Anton; Jadhav, Ajit; Maloney, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9184-9204</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the discovery and medicinal chem. optimization of a novel series of pyrazole-based inhibitors of human lactate dehydrogenase (LDH).  Utilization of a quant. high-throughput screening paradigm facilitated hit identification, while structure-based design and multiparameter optimization enabled the development of compds. with potent enzymic and cell-based inhibition of LDH enzymic activity.  Lead compds. such as 63 exhibit low nM inhibition of both LDHA and LDHB, submicromolar inhibition of lactate prodn., and inhibition of glycolysis in MiaPaCa2 pancreatic cancer and A673 sarcoma cells.  Moreover, robust target engagement of LDHA by lead compds. was demonstrated using the cellular thermal shift assay (CETSA), and drug-target residence time was detd. via SPR.  Anal. of these data suggests that drug-target residence time (off-rate) may be an important attribute to consider for obtaining potent cell-based inhibition of this cancer metab. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrejnE11w2acLVg90H21EOLACvtfcHk0lheEL5d4llrOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslOkurfP&md5=a2df6432e54403e18abc6078fc30e692</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00941%26sid%3Dliteratum%253Aachs%26aulast%3DRai%26aufirst%3DG.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DS.-M.%26aulast%3DLee%26aufirst%3DT.%2BD.%26aulast%3DCheff%26aufirst%3DD.%2BM.%26aulast%3DKouznetsova%26aufirst%3DJ.%26aulast%3DBenavides%26aufirst%3DG.%2BA.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DKuenstner%26aufirst%3DE.%2BJ.%26aulast%3DLuci%26aufirst%3DD.%2BK.%26aulast%3DLukacs%26aufirst%3DC.%2BM.%26aulast%3DDavies%26aufirst%3DD.%2BR.%26aulast%3DDranow%26aufirst%3DD.%2BM.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DSulikowski%26aufirst%3DG.%26aulast%3DMoore%26aufirst%3DW.%2BJ.%26aulast%3DStott%26aufirst%3DG.%2BM.%26aulast%3DFlint%26aufirst%3DA.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DDarley-Usmar%26aufirst%3DV.%2BM.%26aulast%3DNeckers%26aufirst%3DL.%2BM.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520potent%252C%2520cell-active%2520pyrazole-based%2520inhibitors%2520of%2520lactate%2520dehydrogenase%2520%2528LDH%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9184%26epage%3D9204%26doi%3D10.1021%2Facs.jmedchem.7b00941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issaq, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oshima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgart, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edessa, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benavides, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squadrito, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yohe, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowdy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Rodado, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shern, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larion, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helman, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishna, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darley-Usmar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heske, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting glycolysis through inhibition of lactate dehydrogenase impairs tumor growth in preclinical models of Ewing sarcoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">5060</span>– <span class="NLM_lpage">5073</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-19-0217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1158%2F0008-5472.CAN-19-0217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=31431459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslKrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=5060-5073&author=C.+Yeungauthor=A.+E.+Gibsonauthor=S.+H.+Issaqauthor=N.+Oshimaauthor=J.+T.+Baumgartauthor=L.+D.+Edessaauthor=G.+Raiauthor=D.+J.+Urbanauthor=M.+S.+Johnsonauthor=G.+A.+Benavidesauthor=G.+L.+Squadritoauthor=M.+E.+Yoheauthor=H.+Leiauthor=S.+Eldridgeauthor=J.+Hamreauthor=T.+Dowdyauthor=V.+Ruiz-Rodadoauthor=A.+Litaauthor=A.+Mendozaauthor=J.+F.+Shernauthor=M.+Larionauthor=L.+J.+Helmanauthor=G.+M.+Stottauthor=M.+C.+Krishnaauthor=M.+D.+Hallauthor=V.+Darley-Usmarauthor=L.+M.+Neckersauthor=C.+M.+Heske&title=Targeting+glycolysis+through+inhibition+of+lactate+dehydrogenase+impairs+tumor+growth+in+preclinical+models+of+Ewing+sarcoma&doi=10.1158%2F0008-5472.can-19-0217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting glycolysis through inhibition of lactate dehydrogenase impairs tumor growth in preclinical models of ewing sarcoma</span></div><div class="casAuthors">Yeung, Choh; Gibson, Anna E.; Issaq, Sameer H.; Oshima, Nobu; Baumgart, Joshua T.; Edessa, Leah D.; Rai, Ganesha; Urban, Daniel J.; Johnson, Michelle S.; Benavides, Gloria A.; Squadrito, Giuseppe L.; Yohe, Marielle E.; Lei, Haiyan; Eldridge, Sandy; Hamre, John, III; Dowdy, Tyrone; Ruiz-Rodado, Victor; Lita, Adrian; Mendoza, Arnulfo; Shern, Jack F.; Larion, Mioara; Helman, Lee J.; Stott, Gordon M.; Krishna, Murali C.; Hall, Matthew D.; Darley-Usmar, Victor; Neckers, Leonard M.; Heske, Christine M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5060-5073</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Altered cellular metab., including an increased dependence on aerobic glycolysis, is a hallmark of cancer.  Despite the fact that this observation was first made nearly a century ago, effective therapeutic targeting of glycolysis in cancer has remained elusive.  One potentially promising approach involves targeting the glycolytic enzyme lactate dehydrogenase (LDH), which is overexpressed and plays a crit. role in several cancers.  Here, we used a novel class of LDH inhibitors to demonstrate, for the first time, that Ewing sarcoma cells are exquisitely sensitive to inhibition of LDH.  EWS-FLI1, the oncogenic driver of Ewing sarcoma, regulated LDH A (LDHA) expression.  Genetic depletion of LDHA inhibited proliferation of Ewing sarcoma cells and induced apoptosis, phenocopying pharmacol. inhibition of LDH.  LDH inhibitors affected Ewing sarcoma cell viability both in vitro and in vivo by reducing glycolysis.  I.v. administration of LDH inhibitors resulted in the greatest intratumoral drug accumulation, inducing tumor cell death and reducing tumor growth.  The major dose-limiting toxicity obsd. was hemolysis, indicating that a narrow therapeutic window exists for these compds.  Taken together, these data suggest that targeting glycolysis through inhibition of LDH should be further investigated as a potential therapeutic approach for cancers such as Ewing sarcoma that exhibit oncogene-dependent expression of LDH and increased glycolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Bje4xjUjsbVg90H21EOLACvtfcHk0lhzJpUTS_0R-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslKrsrk%253D&md5=2a0ca91f802511e160d86ab4d33744ae</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-19-0217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-19-0217%26sid%3Dliteratum%253Aachs%26aulast%3DYeung%26aufirst%3DC.%26aulast%3DGibson%26aufirst%3DA.%2BE.%26aulast%3DIssaq%26aufirst%3DS.%2BH.%26aulast%3DOshima%26aufirst%3DN.%26aulast%3DBaumgart%26aufirst%3DJ.%2BT.%26aulast%3DEdessa%26aufirst%3DL.%2BD.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DM.%2BS.%26aulast%3DBenavides%26aufirst%3DG.%2BA.%26aulast%3DSquadrito%26aufirst%3DG.%2BL.%26aulast%3DYohe%26aufirst%3DM.%2BE.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DEldridge%26aufirst%3DS.%26aulast%3DHamre%26aufirst%3DJ.%26aulast%3DDowdy%26aufirst%3DT.%26aulast%3DRuiz-Rodado%26aufirst%3DV.%26aulast%3DLita%26aufirst%3DA.%26aulast%3DMendoza%26aufirst%3DA.%26aulast%3DShern%26aufirst%3DJ.%2BF.%26aulast%3DLarion%26aufirst%3DM.%26aulast%3DHelman%26aufirst%3DL.%2BJ.%26aulast%3DStott%26aufirst%3DG.%2BM.%26aulast%3DKrishna%26aufirst%3DM.%2BC.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DDarley-Usmar%26aufirst%3DV.%26aulast%3DNeckers%26aufirst%3DL.%2BM.%26aulast%3DHeske%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520glycolysis%2520through%2520inhibition%2520of%2520lactate%2520dehydrogenase%2520impairs%2520tumor%2520growth%2520in%2520preclinical%2520models%2520of%2520Ewing%2520sarcoma%26jtitle%3DCancer%2520Res.%26date%3D2019%26volume%3D79%26spage%3D5060%26epage%3D5073%26doi%3D10.1158%2F0008-5472.can-19-0217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Oshima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brender, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benavides, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squadrito, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crooks, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrimp, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moses, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusewitt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hathaway, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trepel, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sklar, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norenberg, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linehan, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darley-Usmar, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishna, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L. M.</span></span> <span> </span><span class="NLM_article-title">Dynamic imaging of LDH inhibition in tumors reveals rapid in vivo metabolic rewiring and vulnerability to combination therapy</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1798</span>– <span class="NLM_lpage">1810</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2020.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.celrep.2020.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=32049011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFektb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=1798-1810&author=N.+Oshimaauthor=R.+Ishidaauthor=S.+Kishimotoauthor=K.+Beebeauthor=J.+R.+Brenderauthor=K.+Yamamotoauthor=D.+Urbanauthor=G.+Raiauthor=M.+S.+Johnsonauthor=G.+Benavidesauthor=G.+L.+Squadritoauthor=D.+Crooksauthor=J.+Jacksonauthor=A.+Joshiauthor=B.+T.+Mottauthor=J.+H.+Shrimpauthor=M.+A.+Mosesauthor=M.-J.+Leeauthor=A.+Yunoauthor=T.+D.+Leeauthor=X.+Huauthor=T.+Andersonauthor=D.+Kusewittauthor=H.+H.+Hathawayauthor=A.+Jadhavauthor=D.+Picardauthor=J.+B.+Trepelauthor=J.+B.+Mitchellauthor=G.+M.+Stottauthor=W.+Mooreauthor=A.+Simeonovauthor=L.+A.+Sklarauthor=J.+P.+Norenbergauthor=W.+M.+Linehanauthor=D.+J.+Maloneyauthor=C.+V.+Dangauthor=A.+G.+Watersonauthor=M.+Hallauthor=V.+M.+Darley-Usmarauthor=M.+C.+Krishnaauthor=L.+M.+Neckers&title=Dynamic+imaging+of+LDH+inhibition+in+tumors+reveals+rapid+in+vivo+metabolic+rewiring+and+vulnerability+to+combination+therapy&doi=10.1016%2Fj.celrep.2020.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21aR"><div class="casContent"><span class="casTitleNuber">21a</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Imaging of LDH Inhibition in Tumors Reveals Rapid In Vivo Metabolic Rewiring and Vulnerability to Combination Therapy</span></div><div class="casAuthors">Oshima, Nobu; Ishida, Ryo; Kishimoto, Shun; Beebe, Kristin; Brender, Jeffrey R.; Yamamoto, Kazutoshi; Urban, Daniel; Rai, Ganesha; Johnson, Michelle S.; Benavides, Gloria; Squadrito, Giuseppe L.; Crooks, Dan; Jackson, Joseph; Joshi, Abhinav; Mott, Bryan T.; Shrimp, Jonathan H.; Moses, Michael A.; Lee, Min-Jung; Yuno, Akira; Lee, Tobie D.; Hu, Xin; Anderson, Tamara; Kusewitt, Donna; Hathaway, Helen H.; Jadhav, Ajit; Picard, Didier; Trepel, Jane B.; Mitchell, James B.; Stott, Gordon M.; Moore, William; Simeonov, Anton; Sklar, Larry A.; Norenberg, Jeffrey P.; Linehan, W. Marston; Maloney, David J.; Dang, Chi V.; Waterson, Alex G.; Hall, Matthew; Darley-Usmar, Victor M.; Krishna, Murali C.; Neckers, Leonard M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1798-1810.e4</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The reliance of many cancers on aerobic glycolysis has stimulated efforts to develop lactate dehydrogenase (LDH) inhibitors.  However, despite significant efforts, LDH inhibitors (LDHi) with sufficient specificity and in vivo activity to det. whether LDH is a feasible drug target are lacking.  We describe an LDHi with potent, on-target, in vivo activity.  Using hyperpolarized magnetic resonance spectroscopic imaging (HP-MRSI), we demonstrate in vivo LDH inhibition in two glycolytic cancer models, MIA PaCa-2 and HT29, and we correlate depth and duration of LDH inhibition with direct anti-tumor activity.  HP-MRSI also reveals a metabolic rewiring that occurs in vivo within 30 min of LDH inhibition, wherein pyruvate in a tumor is redirected toward mitochondrial metab.  Using HP-MRSI, we show that inhibition of mitochondrial complex 1 rapidly redirects tumor pyruvate toward lactate.  Inhibition of both mitochondrial complex 1 and LDH suppresses metabolic plasticity, causing metabolic quiescence in vitro and tumor growth inhibition in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXo-hnKU_CkrVg90H21EOLACvtfcHk0lhzJpUTS_0R-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFektb0%253D&md5=9cb03f50f7d5a21ee5a6b1c9b1dc2f0a</span></div><a href="/servlet/linkout?suffix=cit21a&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2020.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2020.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DOshima%26aufirst%3DN.%26aulast%3DIshida%26aufirst%3DR.%26aulast%3DKishimoto%26aufirst%3DS.%26aulast%3DBeebe%26aufirst%3DK.%26aulast%3DBrender%26aufirst%3DJ.%2BR.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DUrban%26aufirst%3DD.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DJohnson%26aufirst%3DM.%2BS.%26aulast%3DBenavides%26aufirst%3DG.%26aulast%3DSquadrito%26aufirst%3DG.%2BL.%26aulast%3DCrooks%26aufirst%3DD.%26aulast%3DJackson%26aufirst%3DJ.%26aulast%3DJoshi%26aufirst%3DA.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DShrimp%26aufirst%3DJ.%2BH.%26aulast%3DMoses%26aufirst%3DM.%2BA.%26aulast%3DLee%26aufirst%3DM.-J.%26aulast%3DYuno%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DT.%2BD.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DAnderson%26aufirst%3DT.%26aulast%3DKusewitt%26aufirst%3DD.%26aulast%3DHathaway%26aufirst%3DH.%2BH.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DPicard%26aufirst%3DD.%26aulast%3DTrepel%26aufirst%3DJ.%2BB.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%26aulast%3DStott%26aufirst%3DG.%2BM.%26aulast%3DMoore%26aufirst%3DW.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DSklar%26aufirst%3DL.%2BA.%26aulast%3DNorenberg%26aufirst%3DJ.%2BP.%26aulast%3DLinehan%26aufirst%3DW.%2BM.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DHall%26aufirst%3DM.%26aulast%3DDarley-Usmar%26aufirst%3DV.%2BM.%26aulast%3DKrishna%26aufirst%3DM.%2BC.%26aulast%3DNeckers%26aufirst%3DL.%2BM.%26atitle%3DDynamic%2520imaging%2520of%2520LDH%2520inhibition%2520in%2520tumors%2520reveals%2520rapid%2520in%2520vivo%2520metabolic%2520rewiring%2520and%2520vulnerability%2520to%2520combination%2520therapy%26jtitle%3DCell%2520Rep.%26date%3D2020%26volume%3D30%26spage%3D1798%26epage%3D1810%26doi%3D10.1016%2Fj.celrep.2020.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit21b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Quon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, L. B.</span></span> <span> </span><span class="NLM_article-title">Redox debt leads to metabolic bankruptcy in tumors</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">359</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2020.02.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.trecan.2020.02.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=32348730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFGqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=359-361&author=E.+Quonauthor=M.+L.+Hartauthor=L.+B.+Sullivan&title=Redox+debt+leads+to+metabolic+bankruptcy+in+tumors&doi=10.1016%2Fj.trecan.2020.02.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21bR"><div class="casContent"><span class="casTitleNuber">21b</span><div class="casTitle"><span class="NLM_cas:atitle">Redox Debt Leads to Metabolic Bankruptcy in Tumors</span></div><div class="casAuthors">Quon, Evan; Hart, Madeleine L.; Sullivan, Lucas B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">359-361</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lactate dehydrogenase (LDH) accounts for the fermentative component of aerobic glycolysis, a near ubiquitous metabolic alteration in cancer.  Recently, Oshima et al. developed a bioavailable LDH inhibitor that decreases tumor growth in mice and functions synergistically with mitochondrial respiration inhibitors.  These findings suggest a cooperative mechanism of action that targets redox homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmIKCGJVoNqrVg90H21EOLACvtfcHk0lh9w-TnAYzjaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFGqsbg%253D&md5=84b71c5b8130ea1cedaf315d8a5f6129</span></div><a href="/servlet/linkout?suffix=cit21b&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2020.02.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2020.02.012%26sid%3Dliteratum%253Aachs%26aulast%3DQuon%26aufirst%3DE.%26aulast%3DHart%26aufirst%3DM.%2BL.%26aulast%3DSullivan%26aufirst%3DL.%2BB.%26atitle%3DRedox%2520debt%2520leads%2520to%2520metabolic%2520bankruptcy%2520in%2520tumors%26jtitle%3DTrends%2520Cancer%26date%3D2020%26volume%3D6%26spage%3D359%26epage%3D361%26doi%3D10.1016%2Fj.trecan.2020.02.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Cañaveras, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gromer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spolski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenbergen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fioravanti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebina-Shibuya, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleck, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gattinoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, W. J.</span></span> <span> </span><span class="NLM_article-title">Lactate dehydrogenase inhibition synergizes with IL-21 to promote CD8<sup>+</sup> T cell stemness and antitumor immunity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">6047</span>, <span class="refDoi"> DOI: 10.1073/pnas.1920413117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1073%2Fpnas.1920413117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=32123114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1WltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=6047&author=D.+Hermansauthor=S.+Gautamauthor=J.+C.+Garc%C3%ADa-Ca%C3%B1averasauthor=D.+Gromerauthor=S.+Mitraauthor=R.+Spolskiauthor=P.+Liauthor=S.+Christensenauthor=R.+Nguyenauthor=J.-X.+Linauthor=J.+Ohauthor=N.+Duauthor=S.+Veenbergenauthor=J.+Fioravantiauthor=R.+Ebina-Shibuyaauthor=C.+Bleckauthor=L.+M.+Neckersauthor=J.+D.+Rabinowitzauthor=L.+Gattinoniauthor=W.+J.+Leonard&title=Lactate+dehydrogenase+inhibition+synergizes+with+IL-21+to+promote+CD8%2B+T+cell+stemness+and+antitumor+immunity&doi=10.1073%2Fpnas.1920413117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Lactate dehydrogenase inhibition synergizes with IL-21 to promote CD8+ T cell stemness and antitumor immunity</span></div><div class="casAuthors">Hermans, Dalton; Gautam, Sanjivan; Garcia-Canaveras, Juan C.; Gromer, Daniel; Mitra, Suman; Spolski, Rosanne; Li, Peng; Christensen, Stephen; Nguyen, Rosa; Lin, Jian-Xin; Oh, Jangsuk; Du, Ning; Veenbergen, Sharon; Fioravanti, Jessica; Ebina-Shibuya, Risa; Bleck, Christopher; Neckers, Leonard M.; Rabinowitz, Joshua D.; Gattinoni, Luca; Leonard, Warren J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6047-6055</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Interleukin (IL)-2 and IL-21 dichotomously shape CD8+ T cell differentiation.  IL-2 drives terminal differentiation, generating cells that are poorly effective against tumors, whereas IL-21 promotes stem cell memory T cells (TSCM) and antitumor responses.  Here we investigated the role of metabolic programming in the developmental differences induced by these cytokines.  IL-2 promoted effector-like metab. and aerobic glycolysis, robustly inducing lactate dehydrogenase (LDH) and lactate prodn., whereas IL-21 maintained a metabolically quiescent state dependent on oxidative phosphorylation.  LDH inhibition rewired IL-2-induced effects, promoting pyruvate entry into the tricarboxylic acid cycle and inhibiting terminal effector and exhaustion programs, including mRNA expression of members of the NR4A family of nuclear receptors, as well as Prdm1 and Xbp1.  While deletion of Ldha prevented development of cells with antitumor effector function, transient LDH inhibition enhanced the generation of memory cells capable of triggering robust antitumor responses after adoptive transfer.  LDH inhibition did not significantly affect IL-21-induced metab. but caused major transcriptomic changes, including the suppression of IL-21-induced exhaustion markers LAG3, PD1, 2B4, and TIM3.  LDH inhibition combined with IL-21 increased the formation of TSCM cells, resulting in more profound antitumor responses and prolonged host survival.  These findings indicate a pivotal role for LDH in modulating cytokine-mediated T cell differentiation and underscore the therapeutic potential of transiently inhibiting LDH during adoptive T cell-based immunotherapy, with an unanticipated cooperative antitumor effect of LDH inhibition and IL-21.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9FMceWMhkTLVg90H21EOLACvtfcHk0lh9w-TnAYzjaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1WltLc%253D&md5=a2305b1ae79995f18bd1d6b93ff153f2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1920413117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1920413117%26sid%3Dliteratum%253Aachs%26aulast%3DHermans%26aufirst%3DD.%26aulast%3DGautam%26aufirst%3DS.%26aulast%3DGarc%25C3%25ADa-Ca%25C3%25B1averas%26aufirst%3DJ.%2BC.%26aulast%3DGromer%26aufirst%3DD.%26aulast%3DMitra%26aufirst%3DS.%26aulast%3DSpolski%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DChristensen%26aufirst%3DS.%26aulast%3DNguyen%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DJ.-X.%26aulast%3DOh%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DN.%26aulast%3DVeenbergen%26aufirst%3DS.%26aulast%3DFioravanti%26aufirst%3DJ.%26aulast%3DEbina-Shibuya%26aufirst%3DR.%26aulast%3DBleck%26aufirst%3DC.%26aulast%3DNeckers%26aufirst%3DL.%2BM.%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26aulast%3DGattinoni%26aufirst%3DL.%26aulast%3DLeonard%26aufirst%3DW.%2BJ.%26atitle%3DLactate%2520dehydrogenase%2520inhibition%2520synergizes%2520with%2520IL-21%2520to%2520promote%2520CD8%252B%2520T%2520cell%2520stemness%2520and%2520antitumor%2520immunity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2020%26volume%3D117%26spage%3D6047%26doi%3D10.1073%2Fpnas.1920413117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span> <span> </span><span class="NLM_article-title">Room-temperature, copper-free Sonogashira reactions facilitated by air-stable, monoligated precatalyst [DTBNpP] Pd(crotyl)Cl</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">12985</span>– <span class="NLM_lpage">12998</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b01868</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b01868" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVOgs7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=12985-12998&author=K.+Pohidaauthor=D.+J.+Maloneyauthor=B.+T.+Mottauthor=G.+Rai&title=Room-temperature%2C+copper-free+Sonogashira+reactions+facilitated+by+air-stable%2C+monoligated+precatalyst+%5BDTBNpP%5D+Pd%28crotyl%29Cl&doi=10.1021%2Facsomega.8b01868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Room-Temperature, Copper-Free Sonogashira Reactions Facilitated by Air-Stable, Monoligated Precatalyst [DTBNpP] Pd(crotyl)Cl</span></div><div class="casAuthors">Pohida, Katherine; Maloney, David J.; Mott, Bryan T.; Rai, Ganesha</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">12985-12998</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel application of [DTBNpP] Pd(crotyl)Cl (DTBNpP = di-tert-butylneopentylphosphine) (P2), an air-stable, com.-available palladium precatalyst that allows rapid access to a mono-ligated state, has been identified for room temp., copper-free Sonogashira couplings of challenging aryl bromides and alkynes.  The mild reaction conditions with TMP in DMSO afford up to 97% yields, excellent functional group tolerability, and broad reaction compatibility with access to one-pot indole formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnQWpZwmXSYLVg90H21EOLACvtfcHk0lh9w-TnAYzjaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVOgs7nI&md5=d02e5a8e6f3084003ac25f4a2a28f18c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b01868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b01868%26sid%3Dliteratum%253Aachs%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DRai%26aufirst%3DG.%26atitle%3DRoom-temperature%252C%2520copper-free%2520Sonogashira%2520reactions%2520facilitated%2520by%2520air-stable%252C%2520monoligated%2520precatalyst%2520%255BDTBNpP%255D%2520Pd%2528crotyl%2529Cl%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D12985%26epage%3D12998%26doi%3D10.1021%2Facsomega.8b01868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh-Bakht, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, J. P.</span></span> <span> </span><span class="NLM_article-title">Biotransformation reactions of five-membered aromatic heterocyclic rings</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1021/tx015574b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx015574b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2002&pages=269-299&author=D.+K.+Dalvieauthor=A.+S.+Kalgutkarauthor=S.+C.+Khojasteh-Bakhtauthor=R.+S.+Obachauthor=J.+P.+O%E2%80%99Donnell&title=Biotransformation+reactions+of+five-membered+aromatic+heterocyclic+rings&doi=10.1021%2Ftx015574b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Biotransformation Reactions of Five-Membered Aromatic Heterocyclic Rings</span></div><div class="casAuthors">Dalvie, Deepak K.; Kalgutkar, Amit S.; Khojasteh-Bakht, S. Cyrus; Obach, R. Scott; O'Donnell, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-299</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 278 refs. is given on the biotransformation pathways of most commonly used 5-membered arom. heterocyclic rings.  The effect of physicochem. properties such as the electronic effects of heteroatoms, aromaticity, and acidity/basicity (pKa) of the rings are correlated on their biotransformation.  The generation of reactive metabolites following oxidn. or redn. of these heterocycles and the subsequent toxicol. consequences are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpNaEz19aehbVg90H21EOLACvtfcHk0ljqgl82F4hQBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVWrsLc%253D&md5=e8e2dfd12772f9ff2289b173bdefa590</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Ftx015574b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx015574b%26sid%3Dliteratum%253Aachs%26aulast%3DDalvie%26aufirst%3DD.%2BK.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DKhojasteh-Bakht%26aufirst%3DS.%2BC.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DJ.%2BP.%26atitle%3DBiotransformation%2520reactions%2520of%2520five-membered%2520aromatic%2520heterocyclic%2520rings%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2002%26volume%3D15%26spage%3D269%26epage%3D299%26doi%3D10.1021%2Ftx015574b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Purkey, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robarge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPasquale, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salphati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sideris, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, A.</span></span> <span> </span><span class="NLM_article-title">Cell active hydroxylactam inhibitors of human lactate dehydrogenase with oral bioavailability in mice</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">896</span>– <span class="NLM_lpage">901</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00190</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00190" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWlsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=896-901&author=H.+E.+Purkeyauthor=K.+Robargeauthor=J.+Chenauthor=Z.+Chenauthor=L.+B.+Corsonauthor=C.+Z.+Dingauthor=A.+G.+DiPasqualeauthor=P.+S.+Dragovichauthor=C.+Eigenbrotauthor=M.+Evangelistaauthor=B.+P.+Fauberauthor=Z.+Gaoauthor=H.+Geauthor=A.+Hitzauthor=Q.+Hoauthor=S.+S.+Labadieauthor=K.+W.+Laiauthor=W.+Liuauthor=Y.+Liuauthor=C.+Liauthor=S.+Maauthor=S.+Malekauthor=T.+O%E2%80%99Brienauthor=J.+Pangauthor=D.+Petersonauthor=L.+Salphatiauthor=S.+Siderisauthor=M.+Ultschauthor=B.+Weiauthor=I.+Yenauthor=Q.+Yueauthor=H.+Zhangauthor=A.+Zhou&title=Cell+active+hydroxylactam+inhibitors+of+human+lactate+dehydrogenase+with+oral+bioavailability+in+mice&doi=10.1021%2Facsmedchemlett.6b00190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Cell Active Hydroxylactam Inhibitors of Human Lactate Dehydrogenase with Oral Bioavailability in Mice</span></div><div class="casAuthors">Purkey, Hans E.; Robarge, Kirk; Chen, Jinhua; Chen, Zhongguo; Corson, Laura B.; Ding, Charles Z.; DiPasquale, Antonio G.; Dragovich, Peter S.; Eigenbrot, Charles; Evangelista, Marie; Fauber, Benjamin P.; Gao, Zhenting; Ge, Hongxiu; Hitz, Anna; Ho, Qunh; Labadie, Sharada S.; Lai, Kwong Wah; Liu, Wenfeng; Liu, Yajing; Li, Chiho; Ma, Shuguang; Malek, Shiva; OBrien, Thomas; Pang, Jodie; Peterson, David; Salphati, Laurent; Sideris, Steve; Ultsch, Mark; Wei, BinQing; Yen, Ivana; Yue, Qin; Zhang, Huihui; Zhou, Aihe</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">896-901</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of trisubstituted hydroxylactams was identified as potent enzymic and cellular inhibitors of human lactate dehydrogenase A.  Utilizing structure-based design and phys. property optimization, multiple inhibitors were discovered with <10 μM lactate IC50 in a MiaPaca2 cell line.  Optimization of the series led to 29, a potent cell active mol. (MiaPaca2 IC50 = 0.67 μM) that also possessed good exposure when dosed orally to mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp75UNOgf2VD7Vg90H21EOLACvtfcHk0ljqgl82F4hQBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWlsbbK&md5=60598df7cf0499ac5c3e4345c53b3770</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00190%26sid%3Dliteratum%253Aachs%26aulast%3DPurkey%26aufirst%3DH.%2BE.%26aulast%3DRobarge%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DCorson%26aufirst%3DL.%2BB.%26aulast%3DDing%26aufirst%3DC.%2BZ.%26aulast%3DDiPasquale%26aufirst%3DA.%2BG.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DEvangelista%26aufirst%3DM.%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DHitz%26aufirst%3DA.%26aulast%3DHo%26aufirst%3DQ.%26aulast%3DLabadie%26aufirst%3DS.%2BS.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DPang%26aufirst%3DJ.%26aulast%3DPeterson%26aufirst%3DD.%26aulast%3DSalphati%26aufirst%3DL.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DYue%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DA.%26atitle%3DCell%2520active%2520hydroxylactam%2520inhibitors%2520of%2520human%2520lactate%2520dehydrogenase%2520with%2520oral%2520bioavailability%2520in%2520mice%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D896%26epage%3D901%26doi%3D10.1021%2Facsmedchemlett.6b00190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asawa, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakharov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallgren, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klumpp-Thomas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coussens, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span> <span> </span><span class="NLM_article-title">A widely-applicable high-throughput cellular thermal shift assay (CETSA) using split Nano Luciferase</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">9472</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-27834-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1038%2Fs41598-018-27834-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=29930256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FgsFyktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=9472&author=N.+J.+Martinezauthor=R.+R.+Asawaauthor=M.+G.+Cyrauthor=A.+Zakharovauthor=D.+J.+Urbanauthor=J.+S.+Rothauthor=E.+Wallgrenauthor=C.+Klumpp-Thomasauthor=N.+P.+Coussensauthor=G.+Raiauthor=S.-M.+Yangauthor=M.+D.+Hallauthor=J.+J.+Maruganauthor=A.+Simeonovauthor=M.+J.+Henderson&title=A+widely-applicable+high-throughput+cellular+thermal+shift+assay+%28CETSA%29+using+split+Nano+Luciferase&doi=10.1038%2Fs41598-018-27834-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A widely-applicable high-throughput cellular thermal shift assay (CETSA) using split Nano Luciferase</span></div><div class="casAuthors">Martinez Natalia J; Asawa Rosita R; Cyr Matthew G; Zakharov Alexey; Urban Daniel J; Roth Jacob S; Wallgren Eric; Klumpp-Thomas Carleen; Coussens Nathan P; Rai Ganesha; Yang Shyh-Ming; Hall Matthew D; Marugan Juan J; Simeonov Anton; Henderson Mark J</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9472</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Assessment of the interactions between a drug and its protein target in a physiologically relevant cellular environment constitutes a major challenge in the pre-clinical drug discovery space.  The Cellular Thermal Shift Assay (CETSA) enables such an assessment by quantifying the changes in the thermal stability of proteins upon ligand binding in intact cells.  Here, we present the development and validation of a homogeneous, standardized, target-independent, and high-throughput (384- and 1536-well formats) CETSA platform that uses a split Nano Luciferase approach (SplitLuc CETSA).  The broad applicability of the assay was demonstrated for diverse targets, and its performance was compared with independent biochemical and cell-based readouts using a set of well-characterized inhibitors.  Moreover, we investigated the utility of the platform as a primary assay for high-throughput screening.  The SplitLuc CETSA presented here enables target engagement studies for medium and high-throughput applications.  Additionally, it provides a rapid assay development and screening platform for targets where phenotypic or other cell-based assays are not readily available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD30bRGadycx0LsK7uFuY7fW6udTcc2ebGUQidotBIFrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FgsFyktQ%253D%253D&md5=d4d1c56ac6d2cb79fe69911293cab32c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-27834-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-27834-y%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%26aufirst%3DN.%2BJ.%26aulast%3DAsawa%26aufirst%3DR.%2BR.%26aulast%3DCyr%26aufirst%3DM.%2BG.%26aulast%3DZakharov%26aufirst%3DA.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DRoth%26aufirst%3DJ.%2BS.%26aulast%3DWallgren%26aufirst%3DE.%26aulast%3DKlumpp-Thomas%26aufirst%3DC.%26aulast%3DCoussens%26aufirst%3DN.%2BP.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.-M.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26atitle%3DA%2520widely-applicable%2520high-throughput%2520cellular%2520thermal%2520shift%2520assay%2520%2528CETSA%2529%2520using%2520split%2520Nano%2520Luciferase%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D9472%26doi%3D10.1038%2Fs41598-018-27834-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Highly predictive and interpretable models for PAMPA permeability</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1016%2Fj.bmc.2016.12.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=28082071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1266-1276&author=H.+Sunauthor=K.+Nguyenauthor=E.+Kernsauthor=Z.+Yanauthor=K.+R.+Yuauthor=P.+Shahauthor=A.+Jadhavauthor=X.+Xu&title=Highly+predictive+and+interpretable+models+for+PAMPA+permeability&doi=10.1016%2Fj.bmc.2016.12.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Highly predictive and interpretable models for PAMPA permeability</span></div><div class="casAuthors">Sun, Hongmao; Nguyen, Kimloan; Kerns, Edward; Yan, Zhengyin; Yu, Kyeong Ri; Shah, Pranav; Jadhav, Ajit; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1266-1276</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell membrane permeability is an important determinant for oral absorption and bioavailability of a drug mol.  An in silico model predicting drug permeability is described, which is built based on a large permeability dataset of 7488 compd. entries or 5435 structurally unique mols. measured by the same lab using parallel artificial membrane permeability assay (PAMPA).  On the basis of customized mol. descriptors, the support vector regression (SVR) model trained with 4071 compds. with quant. data is able to predict the remaining 1364 compds. with the qual. data with an area under the curve of receiver operating characteristic (AUC-ROC) of 0.90.  The support vector classification (SVC) model trained with half of the whole dataset comprised of both the quant. and the qual. data produced accurate predictions to the remaining data with the AUC-ROC of 0.88.  The results suggest that the developed SVR model is highly predictive and provides medicinal chemists a useful in silico tool to facilitate design and synthesis of novel compds. with optimal drug-like properties, and thus accelerate the lead optimization in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y8FhTLLqbrVg90H21EOLACvtfcHk0lh5ukH616XGcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D&md5=2f6b60dfcd168b5bf837405ca52457b2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.049%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DHighly%2520predictive%2520and%2520interpretable%2520models%2520for%2520PAMPA%2520permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1266%26epage%3D1276%26doi%3D10.1016%2Fj.bmc.2016.12.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">L’Heureux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaulieu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bill, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaFlamme, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirmehrabi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tadayon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tovell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couturier, M.</span></span> <span> </span><span class="NLM_article-title">Aminodifluorosulfinium salts: selective fluorination reagents with enhanced thermal stability and ease of handling</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3401</span>– <span class="NLM_lpage">3411</span>, <span class="refDoi"> DOI: 10.1021/jo100504x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo100504x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=3401-3411&author=A.+L%E2%80%99Heureuxauthor=F.+Beaulieuauthor=C.+Bennettauthor=D.+R.+Billauthor=S.+Claytonauthor=F.+LaFlammeauthor=M.+Mirmehrabiauthor=S.+Tadayonauthor=D.+Tovellauthor=M.+Couturier&title=Aminodifluorosulfinium+salts%3A+selective+fluorination+reagents+with+enhanced+thermal+stability+and+ease+of+handling&doi=10.1021%2Fjo100504x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Aminodifluorosulfinium Salts: Selective Fluorination Reagents with Enhanced Thermal Stability and Ease of Handling</span></div><div class="casAuthors">L'Heureux, Alexandre; Beaulieu, Francis; Bennett, Christopher; Bill, David R.; Clayton, Simon; La Flamme, Francois; Mirmehrabi, Mahmoud; Tadayon, Sam; Tovell, David; Couturier, Michel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3401-3411</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Diethylaminodifluorosulfinium tetrafluoroborate (XtalFluor-E) and morpholinodifluorosulfinium tetrafluoroborate (XtalFluor-M) are cryst. fluorinating agents that are more easily handled and significantly more stable than Deoxo-Fluor, DAST, and their analogs.  These reagents can be prepd. in a safer and more cost-efficient manner by avoiding the laborious and hazardous distn. of dialkylaminosulfur trifluorides.  Unlike DAST, Deoxo-Fluor, and Fluolead, XtalFluor reagents do not generate highly corrosive free-HF and therefore can be used in std. borosilicate vessels.  When used in conjunction with promoters such as Et3N·3HF, Et3N·2HF, or DBU, XtalFluor reagents effectively convert alcs. to alkyl fluorides and carbonyls to gem-difluorides.  These reagents are typically more selective than DAST and Deoxo-Fluor and exhibit superior performance by providing significantly less elimination side products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor511M96t3p7Vg90H21EOLACvtfcHk0ljck-mrzq4KGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqur0%253D&md5=3afb1f998f6e1a28a8559e76bab4dda1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo100504x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo100504x%26sid%3Dliteratum%253Aachs%26aulast%3DL%25E2%2580%2599Heureux%26aufirst%3DA.%26aulast%3DBeaulieu%26aufirst%3DF.%26aulast%3DBennett%26aufirst%3DC.%26aulast%3DBill%26aufirst%3DD.%2BR.%26aulast%3DClayton%26aufirst%3DS.%26aulast%3DLaFlamme%26aufirst%3DF.%26aulast%3DMirmehrabi%26aufirst%3DM.%26aulast%3DTadayon%26aufirst%3DS.%26aulast%3DTovell%26aufirst%3DD.%26aulast%3DCouturier%26aufirst%3DM.%26atitle%3DAminodifluorosulfinium%2520salts%253A%2520selective%2520fluorination%2520reagents%2520with%2520enhanced%2520thermal%2520stability%2520and%2520ease%2520of%2520handling%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2010%26volume%3D75%26spage%3D3401%26epage%3D3411%26doi%3D10.1021%2Fjo100504x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benavides, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancaster, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dell’Italia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darley-Usmar, V. M.</span></span> <span> </span><span class="NLM_article-title">Integration of cellular bioenergetics with mitochondrial quality control and autophagy</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>393</i></span>,  <span class="NLM_fpage">1485</span>– <span class="NLM_lpage">1512</span>, <span class="refDoi"> DOI: 10.1515/hsz-2012-0198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1515%2Fhsz-2012-0198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=23092819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3sXns1Wruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=393&publication_year=2012&pages=1485-1512&author=B.+G.+Hillauthor=G.+A.+Benavidesauthor=J.+R.+Lancasterauthor=S.+Ballingerauthor=L.+Dell%E2%80%99Italiaauthor=J.+Zhangauthor=V.+M.+Darley-Usmar&title=Integration+of+cellular+bioenergetics+with+mitochondrial+quality+control+and+autophagy&doi=10.1515%2Fhsz-2012-0198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of cellular bioenergetics with mitochondrial quality control and autophagy</span></div><div class="casAuthors">Hill, Bradford G.; Benavides, Gloria A.; Lancaster, Jack R., Jr.; Ballinger, Scott; Dell'Italia, Lou; Zhang, Jianhua; Darley-Usmar, Victor M.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">393</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1485-1512</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Bioenergetic dysfunction is emerging as a cornerstone for establishing a framework for understanding the pathophysiol. of cardiovascular disease, diabetes, cancer and neurodegeneration.  Recent advances in cellular bioenergetics have shown that many cells maintain a substantial bioenergetic reserve capacity, which is a prospective index of "healthy" mitochondrial populations.  The bioenergetics of the cell are likely regulated by energy requirements and substrate availability.  Addnl., the overall quality of the mitochondrial population and the relative abundance of mitochondria in cells and tissues also impinge on overall bioenergetic capacity and resistance to stress.  Because mitochondria are susceptible to damage mediated by reactive oxygen/nitrogen and lipid species, maintaining a "healthy" population of mitochondria through quality control mechanisms appears to be essential for cell survival under conditions of pathol. stress.  Accumulating evidence suggest that mitophagy is particularly important for preventing amplification of initial oxidative insults, which otherwise would further impair the respiratory chain or promote mutations in mitochondrial DNA (mtDNA).  The processes underlying the regulation of mitophagy depend on several factors, including the integrity of mtDNA, electron transport chain activity, and the interaction and regulation of the autophagic machinery.  The integration and interpretation of cellular bioenergetics in the context of mitochondrial quality control and genetics is the theme of this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_88kMLPXxJrVg90H21EOLACvtfcHk0ljck-mrzq4KGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXns1Wruw%253D%253D&md5=b0c74b21fe1b672050490cc98df5595b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1515%2Fhsz-2012-0198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fhsz-2012-0198%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DB.%2BG.%26aulast%3DBenavides%26aufirst%3DG.%2BA.%26aulast%3DLancaster%26aufirst%3DJ.%2BR.%26aulast%3DBallinger%26aufirst%3DS.%26aulast%3DDell%25E2%2580%2599Italia%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDarley-Usmar%26aufirst%3DV.%2BM.%26atitle%3DIntegration%2520of%2520cellular%2520bioenergetics%2520with%2520mitochondrial%2520quality%2520control%2520and%2520autophagy%26jtitle%3DBiol.%2520Chem.%26date%3D2012%26volume%3D393%26spage%3D1485%26epage%3D1512%26doi%3D10.1515%2Fhsz-2012-0198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsinman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsinman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avdeef, A.</span></span> <span> </span><span class="NLM_article-title">Physicochemical selectivity of the BBB microenvironment governing passive diffusion matching with a porcine brain lipid extract artificial membrane permeability model</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">363</span>, <span class="refDoi"> DOI: 10.1007/s11095-010-0280-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1007%2Fs11095-010-0280-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=20945153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yrs7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2011&pages=337-363&author=O.+Tsinmanauthor=K.+Tsinmanauthor=N.+Sunauthor=A.+Avdeef&title=Physicochemical+selectivity+of+the+BBB+microenvironment+governing+passive+diffusion+matching+with+a+porcine+brain+lipid+extract+artificial+membrane+permeability+model&doi=10.1007%2Fs11095-010-0280-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical Selectivity of the BBB Microenvironment Governing Passive Diffusion-Matching with a Porcine Brain Lipid Extract Artificial Membrane Permeability Model</span></div><div class="casAuthors">Tsinman, Oksana; Tsinman, Konstantin; Sun, Na; Avdeef, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-363</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: To mimic the physicochem. selectivity of the blood-brain barrier (BBB) and to predict its passive permeability using a PAMPA model based on porcine brain lipid ext. (PBLE 10%w/v in alkane).  Methods: Three PAMPA (BD pre-coated and PBLE with 2 different lipid vols.) models were tested with 108 drugs.  Abraham solvation descriptors were used to interpret the in vitro-in vivo correlation with 282 in situ brain perfusion measurements, spanning over 5 orders of magnitude.  An in combo PAMPA model was developed from combining measured PAMPA permeability with one H-bond descriptor.  Results: The in combo PAMPA predicted 93% of the variance of 197 largely efflux-inhibited in situ permeability training set.  The model was cross-validated by the "leave-many-out" procedure, with q2 = 0.92 ± 0.03.  The PAMPA models indicated the presence of paramembrane water channels.  Only the PBLE-based PAMPA-BBB model with sufficient lipid to fill all the internal pore space of the filter showed a wide dynamic range window, selectivity coeff. near 1, and was suitable for predicting BBB permeability.  Conclusion: BBB permeability can be predicted by in combo PAMPA.  Its speed and substantially lower cost, compared to in vivo measurements, make it an attractive first-pass screening method for BBB passive permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_ZUNSGMH-JbVg90H21EOLACvtfcHk0liLdNc3NDjdeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yrs7vF&md5=31afeb98ee1765bb812f0881e0fe4452</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs11095-010-0280-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-010-0280-x%26sid%3Dliteratum%253Aachs%26aulast%3DTsinman%26aufirst%3DO.%26aulast%3DTsinman%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DAvdeef%26aufirst%3DA.%26atitle%3DPhysicochemical%2520selectivity%2520of%2520the%2520BBB%2520microenvironment%2520governing%2520passive%2520diffusion%2520matching%2520with%2520a%2520porcine%2520brain%2520lipid%2520extract%2520artificial%2520membrane%2520permeability%2520model%26jtitle%3DPharm.%2520Res.%26date%3D2011%26volume%3D28%26spage%3D337%26epage%3D363%26doi%3D10.1007%2Fs11095-010-0280-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourassa, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D. M.</span></span> <span> </span><span class="NLM_article-title">Experimental design on single-time-point high-throughput microsomal stability assay</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1544</span>, <span class="refDoi"> DOI: 10.1002/jps.20076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=10.1002%2Fjps.20076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=15124211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks12hsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2004&pages=1537-1544&author=L.+Diauthor=E.+H.+Kernsauthor=N.+Gaoauthor=S.+Q.+Liauthor=Y.+Huangauthor=J.+L.+Bourassaauthor=D.+M.+Huryn&title=Experimental+design+on+single-time-point+high-throughput+microsomal+stability+assay&doi=10.1002%2Fjps.20076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental design on single-time-point high-throughput microsomal stability assay</span></div><div class="casAuthors">Di, Li; Kerns, Edward H.; Gao, Ning; Li, Susan Q.; Huang, Youping; Bourassa, Jim L.; Huryn, Donna M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1537-1544</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">An exptl. design for a single-time-point microsomal stability assay was evaluated as compared with multiple-time-point studies.  Results obtained from single-time-point expts. are in excellent agreement with those from multiple time points.  First-order reaction kinetics revealed rapid changes of predicted half-life from percent remaining of the parent compd. at the inflection points, suggesting a max. predictive limit for half-life.  Selection of the incubation time in single-time-point assays is important to obtain balanced information for stable and unstable compds.  A short incubation time (e.g., 5 min) is most useful for differentiating between unstable compds., which is beneficial to direct the synthetic efforts in projects with poor metabolic stability.  A long incubation time (e.g., 30 min) is more applicable to a compd. series with high metabolic stability.  For screening purposes, a moderate incubation time (e.g., 15 min) is recommended to achieve good resoln. and a sufficiently high max. predictive limit for half-life.  This study suggests that a single-time-point assay is sufficient for ranking compds. in early drug discovery.  It increases throughput and reduces turnaround time and cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdHPH7tWzrIrVg90H21EOLACvtfcHk0liLdNc3NDjdeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks12hsr4%253D&md5=45b3bfb054ad93e18655b988fdbbe89b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fjps.20076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.20076%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DS.%2BQ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DBourassa%26aufirst%3DJ.%2BL.%26aulast%3DHuryn%26aufirst%3DD.%2BM.%26atitle%3DExperimental%2520design%2520on%2520single-time-point%2520high-throughput%2520microsomal%2520stability%2520assay%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2004%26volume%3D93%26spage%3D1537%26epage%3D1544%26doi%3D10.1002%2Fjps.20076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q0D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q0D','PDB','6Q0D'); return false;">PDB: 6Q0D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q13" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q13','PDB','6Q13'); return false;">PDB: 6Q13</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i125"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00916">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_61085"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00916?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00916</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Correlation between LDHA and LDHB; correlation of A673 lactate versus MiaPaCa-2 lactate and A673 cytotoxicity versus MiaPaCa-2; correlation of A673 lactate versus A673 and MiaPaCa-2 lactate versus MiaPaCa-2; MiaPaCa-2 72h viability versus LDHA CETSA binding activity; original LCMS, proton, fluorine, carbon NMR for <b>43</b>, <b>52</b>, and key compounds; experimental procedures for representative scale up of lead compounds and key intermediates; and spectroscopic data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and PDB ID codes of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q0D">6Q0D</a> (<b>23</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Q13">6Q13</a> (<b>52</b>). Authors will release the atomic coordinates and experimental data upon article publication (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_001.pdf">jm0c00916_si_001.pdf (3.39 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00916/suppl_file/jm0c00916_si_002.csv">jm0c00916_si_002.csv (17.95 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00916&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-19%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00916%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00916" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799cd98ce203d5a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
