
<html lang="en"     class="pb-page"  data-request-id="d40242a3-9011-435f-8357-68e3d77d3539"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.5b01890;issue:issue:10.1021/jmcmar.2016.59.issue-10;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery and Characterization of a Highly Potent and Selective Aminopyrazoline-Based in Vivo Probe (BAY-598) for the Protein Lysine Methyltransferase SMYD2" /></meta><meta name="dc.Creator" content="Erik  Eggert" /></meta><meta name="dc.Creator" content="Roman C.  Hillig" /></meta><meta name="dc.Creator" content="Silke  Koehr" /></meta><meta name="dc.Creator" content="Detlef  Stöckigt" /></meta><meta name="dc.Creator" content="Jörg  Weiske" /></meta><meta name="dc.Creator" content="Naomi  Barak" /></meta><meta name="dc.Creator" content="Jeffrey  Mowat" /></meta><meta name="dc.Creator" content="Thomas  Brumby" /></meta><meta name="dc.Creator" content="Clara D.  Christ" /></meta><meta name="dc.Creator" content="Antonius  ter Laak" /></meta><meta name="dc.Creator" content="Tina  Lang" /></meta><meta name="dc.Creator" content="Amaury E.  Fernandez-Montalvan" /></meta><meta name="dc.Creator" content="Volker  Badock" /></meta><meta name="dc.Creator" content="Hilmar  Weinmann" /></meta><meta name="dc.Creator" content="Ingo V.  Hartung" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="Magdalena  Szewczyk" /></meta><meta name="dc.Creator" content="Steven  Kennedy" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Vijayaratnam  Santhakumar" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Timo  Stellfeld" /></meta><meta name="dc.Creator" content="Carlo  Stresemann" /></meta><meta name="dc.Description" content="Protein lysine methyltransferases have recently emerged as a new target class for the development of inhibitors that modulate gene transcription or signaling pathways. SET and MYND domain containin..." /></meta><meta name="Description" content="Protein lysine methyltransferases have recently emerged as a new target class for the development of inhibitors that modulate gene transcription or signaling pathways. SET and MYND domain containin..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 3, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01890" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01890" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01890" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01890" /></link>
        
    
    

<title>Discovery and Characterization of a Highly Potent and Selective Aminopyrazoline-Based in Vivo Probe (BAY-598) for the Protein Lysine Methyltransferase SMYD2 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01890" /></meta><meta property="og:title" content="Discovery and Characterization of a Highly Potent and Selective Aminopyrazoline-Based in Vivo Probe (BAY-598) for the Protein Lysine Methyltransferase SMYD2" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0006.jpeg" /></meta><meta property="og:description" content="Protein lysine methyltransferases have recently emerged as a new target class for the development of inhibitors that modulate gene transcription or signaling pathways. SET and MYND domain containing protein 2 (SMYD2) is a catalytic SET domain containing methyltransferase reported to monomethylate lysine residues on histone and nonhistone proteins. Although several studies have uncovered an important role of SMYD2 in promoting cancer by protein methylation, the biology of SMYD2 is far from being fully understood. Utilization of highly potent and selective chemical probes for target validation has emerged as a concept which circumvents possible limitations of knockdown experiments and, in particular, could result in an improved exploration of drug targets with a complex underlying biology. Here, we report the development of a potent, selective, and cell-active, substrate-competitive inhibitor of SMYD2, which is the first reported inhibitor suitable for in vivo target validation studies in rodents." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01890"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01890">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01890&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01890&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01890&amp;href=/doi/10.1021/acs.jmedchem.5b01890" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 4578-4600</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01863" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01891" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Characterization of a Highly Potent and Selective Aminopyrazoline-Based in Vivo Probe (BAY-598) for the Protein Lysine Methyltransferase SMYD2</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Erik++Eggert">Erik Eggert</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Roman+C.++Hillig">Roman C. Hillig</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Silke++Koehr">Silke Koehr</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Detlef++St%C3%B6ckigt">Detlef Stöckigt</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%B6rg++Weiske">Jörg Weiske</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Naomi++Barak">Naomi Barak</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey++Mowat">Jeffrey Mowat</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Brumby">Thomas Brumby</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Clara+D.++Christ">Clara D. Christ</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Antonius++ter+Laak">Antonius ter Laak</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tina++Lang">Tina Lang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Amaury+E.++Fernandez-Montalvan">Amaury E. Fernandez-Montalvan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Volker++Badock">Volker Badock</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hilmar++Weinmann">Hilmar Weinmann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ingo+V.++Hartung">Ingo V. Hartung</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Magdalena++Szewczyk">Magdalena Szewczyk</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven++Kennedy">Steven Kennedy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vijayaratnam++Santhakumar">Vijayaratnam Santhakumar</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Timo++Stellfeld">Timo Stellfeld</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Carlo++Stresemann">Carlo Stresemann</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Drug Discovery, BAYER Pharma AG, Muellerstrasse 178, 13353 Berlin, Germany</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">#</span> <span class="aff-text">Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">§</span> <span class="aff-text">Princess Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</span></div><div class="corresp-info"><strong>*</strong>For C.S.: phone, (+49)3046812866; E-mail, <a href="/cdn-cgi/l/email-protection#82e1e3f0eeedacf1f6f0e7f1e7efe3ececc2e0e3fbe7f0ace1edef"><span class="__cf_email__" data-cfemail="1e7d7f6c7271306d6a6c7b6d7b737f70705e7c7f677b6c307d7173">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For T.S.: phone, (+49)3046815727; E-mail, <a href="/cdn-cgi/l/email-protection#32465b5f5d1c4146575e5e54575e567250534b57401c515d5f"><span class="__cf_email__" data-cfemail="592d303436772a2d3c35353f3c353d193b38203c2b773a3634">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01890&amp;href=/doi/10.1021%2Facs.jmedchem.5b01890" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 4578–4600</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 13, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 December 2015</li><li><span class="item_label"><b>Published</b> online</span>3 May 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 May 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01890" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01890</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01890"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3848</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">38</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01890" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Characterization of a Highly Potent and Selective Aminopyrazoline-Based in Vivo Probe (BAY-598) for the Protein Lysine Methyltransferase SMYD2&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Erik&quot;,&quot;last_name&quot;:&quot;Eggert&quot;},{&quot;first_name&quot;:&quot;Roman&quot;,&quot;last_name&quot;:&quot;C. Hillig&quot;},{&quot;first_name&quot;:&quot;Silke&quot;,&quot;last_name&quot;:&quot;Koehr&quot;},{&quot;first_name&quot;:&quot;Detlef&quot;,&quot;last_name&quot;:&quot;Stöckigt&quot;},{&quot;first_name&quot;:&quot;Jörg&quot;,&quot;last_name&quot;:&quot;Weiske&quot;},{&quot;first_name&quot;:&quot;Naomi&quot;,&quot;last_name&quot;:&quot;Barak&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;Mowat&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Brumby&quot;},{&quot;first_name&quot;:&quot;Clara&quot;,&quot;last_name&quot;:&quot;D. Christ&quot;},{&quot;first_name&quot;:&quot;Antonius&quot;,&quot;last_name&quot;:&quot;ter Laak&quot;},{&quot;first_name&quot;:&quot;Tina&quot;,&quot;last_name&quot;:&quot;Lang&quot;},{&quot;first_name&quot;:&quot;Amaury&quot;,&quot;last_name&quot;:&quot;E. Fernandez-Montalvan&quot;},{&quot;first_name&quot;:&quot;Volker&quot;,&quot;last_name&quot;:&quot;Badock&quot;},{&quot;first_name&quot;:&quot;Hilmar&quot;,&quot;last_name&quot;:&quot;Weinmann&quot;},{&quot;first_name&quot;:&quot;Ingo&quot;,&quot;last_name&quot;:&quot;V. Hartung&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;Magdalena&quot;,&quot;last_name&quot;:&quot;Szewczyk&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;Kennedy&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Vijayaratnam&quot;,&quot;last_name&quot;:&quot;Santhakumar&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Timo&quot;,&quot;last_name&quot;:&quot;Stellfeld&quot;},{&quot;first_name&quot;:&quot;Carlo&quot;,&quot;last_name&quot;:&quot;Stresemann&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;4578-4600&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01890&quot;},&quot;abstract&quot;:&quot;Protein lysine methyltransferases have recently emerged as a new target class for the development of inhibitors that modulate gene transcription or signaling pathways. SET and MYND domain containing protein 2 (SMYD2) is a catalytic SET domain containing methyltransferase reported to monomethylate lysine residues on histone and nonhistone proteins. Although several studies have uncovered an important role of SMYD2 in promoting cancer by protein methylation, the biology of SMYD2 is far from being fully understood. Utilization of highly potent and selective chemical probes for target validation has emerged as a concept which circumvents possible limitations of knockdown experiments and, in particular, could result in an improved exploration of drug targets with a complex underlying biology. Here, we report the development of a potent, selective, and cell-active, substrate-competitive inhibitor of SMYD2, which is the first reported inhibitor suitable for in vivo target validation studies in rodents.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01890&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01890" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01890&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01890" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01890&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01890" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01890&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01890&amp;href=/doi/10.1021/acs.jmedchem.5b01890" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01890" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01890" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01890%26sid%3Dliteratum%253Aachs%26pmid%3D27075367%26genre%3Darticle%26aulast%3DEggert%26date%3D2016%26atitle%3DDiscovery%2Band%2BCharacterization%2Bof%2Ba%2BHighly%2BPotent%2Band%2BSelective%2BAminopyrazoline-Based%2Bin%2BVivo%2BProbe%2B%2528BAY-598%2529%2Bfor%2Bthe%2BProtein%2BLysine%2BMethyltransferase%2BSMYD2%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D10%26spage%3D4578%26epage%3D4600%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291304" title="Organic reactions">Organic reactions</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/jmcmar.2016.59.issue-10/20160526/jmcmar.2016.59.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein lysine methyltransferases have recently emerged as a new target class for the development of inhibitors that modulate gene transcription or signaling pathways. SET and MYND domain containing protein 2 (SMYD2) is a catalytic SET domain containing methyltransferase reported to monomethylate lysine residues on histone and nonhistone proteins. Although several studies have uncovered an important role of SMYD2 in promoting cancer by protein methylation, the biology of SMYD2 is far from being fully understood. Utilization of highly potent and selective chemical probes for target validation has emerged as a concept which circumvents possible limitations of knockdown experiments and, in particular, could result in an improved exploration of drug targets with a complex underlying biology. Here, we report the development of a potent, selective, and cell-active, substrate-competitive inhibitor of SMYD2, which is the first reported inhibitor suitable for in vivo target validation studies in rodents.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74304" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74304" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">SMYD2 is a catalytic SET domain containing protein methyltransferase reported to monomethylate lysine residues on histone and nonhistone proteins.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> SMYD2 has been proposed as a potential therapeutic target in cancer. Its overexpression has been reported in cancer cell lines as well as in esophageal squamous carcinoma, bladder carcinoma, gastric cancer, and pediatric acute lymphoblastic leukemia patients.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6">(2-6)</a> In these studies, SMYD2 overexpression often correlated with lower survival rate and was suggested to be a clinically relevant prognostic marker. Knockdown of SMYD2 in overexpressing ESCC, bladder, and gastric cancer cell line models significantly reduced cell proliferation.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Initially, SMYD2 was characterized as methylating H3 lysine 36<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and lysine 4 when interacting with HSP90a.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Methylation of histones by SMYD2 has been connected to increased transcription of genes involved in cell-cycle regulation, chromatin remodeling, and transcriptional regulation.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In addition, several studies have uncovered an important role of SMYD2 methylation activity toward nonhistone proteins closely connected to cancer. This is in line with the emerging concept that posttranslational methylation of nonhistone proteins (e.g., of transcription factors) by protein methyltransferases can also substantially alter protein function. Thereby, a regulatory role of lysine methylation can probably be extended to multiple cellular pathways besides transcriptional regulation and histones.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> So far, the best-characterized example of SMYD2 methylation of a nonhistone protein is the tumor suppressor transcription factor p53.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16">(11-16)</a> Transcriptional activity of p53 is inhibited by SMYD2-mediated posttranslational methylation at lysine 370 (K370).<a onclick="showRef(event, 'ref13 ref17'); return false;" href="javascript:void(0);" class="ref ref13 ref17">(13, 17)</a> The structural basis of p53 methylation by SMYD2 has been characterized by solving the crystal structure of a ternary complex with the cofactor product <i>S</i>-adenosylhomocysteine and a p53-derived substrate peptide.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> It has been proposed that methylation at K370 reduces the DNA-binding efficiency of p53 and subsequently prevents the transcriptional activation of p53 target genes.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In the same study, a knockdown of SMYD2 and treatment with doxorubicin led to an increase in p53-mediated cell-cycle arrest and apoptosis. In line with these observations, low SMYD2 gene expression was suggested as a predictive marker for an improved response to neoadjuvant chemotherapy in breast cancer patients.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Besides p53, several other proteins have been identified as SMYD2 substrates, including the estrogen receptor (ER),<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19, 20)</a> PARP1,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> retinoblastoma protein (Rb),<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> and HSP90.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> Mechanistically, methylation of HSP90 has been connected to the normal physiological role of SMYD2 in muscle biology,<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24, 25)</a> as well as in cancer.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> These studies indicate that SMYD2 has many substrates and various potentially tissue-specific physiological and pathogenic functions. SMYD2 therefore represents a very attractive target for further exploration in different disease-relevant models. Nevertheless, the biology of SMYD2 is still poorly understood and the molecular contribution of individual substrates to specific knockdown phenotypes remains largely unknown. For a more unbiased interpretation of biological experiments, fully profiled chemical probes can substantially contribute to preclinical target validation.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> Although first cellular-active probe inhibitors of SMYD2 have been described (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), there is a need for structurally orthogonal chemical probes to enable cross-validation studies and thereby rule out unspecific effects.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The publication of <b>1</b> (AZ505)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> was the first disclosure of a co-crystal structure of an inhibitor bound to SMYD2 and paved the way for further studies, leading to the discovery of <b>2</b> (LLY-507),<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and <b>3</b> (A-893),<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> which have reported significantly improved potency. Analysis of the respective cocrystal structures reveals that all three inhibitors bind in a similar fashion, occupying the same binding pockets. Furthermore, the cellular activity of the known inhibitors is limited, and no data about in vivo applicability have been published. Here we report the discovery of a potent and selective aminopyrazoline-based small-molecule inhibitor (<i>S</i>)-<b>4</b> (BAY-598).<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> We show that (<i>S</i>)-<b>4</b> has a distinctly different binding mode compared to previous inhibitors, utilizing a dichlorophenyl moiety as so far unprecedented chemical motif for addressing the methyl-lysine binding pocket of SMYD2. For the first time, we are presenting in vivo xenograft and DMPK data for a SMYD2 inhibitor. In addition to previously described inhibitors, (<i>S</i>)-<b>4</b> shows very potent cellular activity combined with reasonable DMPK properties (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Furthermore, we are indicating the potential that this inhibitor might offer to the field of protein methyltransferases in the quest to fully explore the underlying complex biology and therapeutic potential of SMYD2 by validating AHNAK protein<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> as a new cellular substrate.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0008.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Comparison of so far described SMYD2 inhibitors with (<i>S</i>)-<b>4</b> discovered in this study. Data from respective references as well as assay results from this study are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14440" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14440" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The potential link between SMYD2 and cancer motivated us to screen the Bayer compound collection, with the aim of identifying small-molecule inhibitors of the enzyme. To this end, a scintillation proximity assay (SPA) was set up using recombinant His-tagged SMYD2, a biotinylated p53-derived peptide substrate, and tritiated <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (<sup>3</sup>H-SAM) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). Of the three million compounds tested in a primary HTS, we identified more than 2300 confirmed hits which inhibited SMYD2 with IC<sub>50</sub> values below 15 μM. Among the multiple structural clusters and singletons in the hit list, several offered starting points with low micromolar potency and tractable chemical matter, and our attention was drawn to pyrazolines such as compound <b>5</b>. Initial hits and later derivatives of the pyrazoline series showed stabilizing effects in an SMYD2 thermal shift assay (TSA) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). In addition, binding of compound <b>6</b> to SMYD2 was validated by isothermal titration calorimetry (ITC), which indicated a submicromolar binding constant (<i>K</i><sub>d</sub> = 540 nM, <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C) and a high enthalpic contribution to the binding energy. As the latter reduces the likelihood of nonspecific off-target activities, the ITC data underscored the attractiveness of pyrazoline hit <b>5</b>.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a> It transpired that compounds of this structural series had been prepared during the course of an in-house program as antagonists for protease-activated receptor 1 (PAR1), a G-protein-coupled receptor which is highly expressed in platelets and plays an important role in thrombin signaling and platelet aggregation.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0009.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Identification and biophysical validation of the pyrazoline SMYD2 inhibitor series leading to (<i>S</i>)-<b>4</b>. (A) Scintillation proximity assay (SPA) screening approach leading to the pyrazoline hit cluster, with representative hit compound <b>5</b> with IC<sub>50</sub> = 1.7 μM. (B) Thermal shift assay (TSA) for SMYD2 protein stabilization for inhibitors of the aminopyrazoline series [●], with aminopyrazoline hit <b>5</b> (Δ<i>T</i><sub>m</sub> = 3.3 K) and (<i>S</i>)-<b>4</b> (Δ<i>T</i><sub>m</sub> = 7.1 K) highlighted. Stabilization correlates with SPA potency. (C) Isothermal titration calorimetry (ITC) of compound <b>6</b> indicating a submicromolar binding constant (<i>K</i><sub>d</sub> = 540 nM) and a high enthalpic contribution to the binding energy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Chemistry</h3><div class="NLM_p">The pyrazoline compounds of interest were synthesized according to previously described procedures.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Synthesis of the required intermediates <b>10a</b>–<b>f</b> started from the commercially available 2-amino-1-phenylethanones <b>7</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Protection of the amines as the allyl carbamates <b>8</b> was followed by Mannich reaction with formaldehyde and piperidine, and the resulting product mixture was treated with hydrazine monohydrate to install the pyrazoline moiety (compounds <b>9</b>). Subsequent reaction with diphenyl <i>N</i>-cyanocarbonimidate resulted in intermediates <b>10a</b>–<b>f</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0017.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyrazoline Intermediates <b>10a</b>–<b>f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) allyl chloroformate, K<sub>2</sub>CO<sub>3</sub> (1.5 M in water), DCM, rt, overnight; (b) formaldehyde (37 wt % in water), piperidine, EtOH, rt, overnight; (c) hydrazine monohydrate, EtOH, reflux; (d) diphenyl <i>N</i>-cyanocarbonimidate, <i>i</i>-PrOH, reflux, 1 h, then rt, overnight.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>5</b>, <b>6</b>, and <b>12</b>–<b>24</b> began with <b>10a</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Treatment of <b>10a</b> with the respective aniline or amine provided intermediates <b>11a</b>–<b>i</b>. Deprotection to the 4-aminopyrazolines and further elaboration led to the desired amides <b>5</b>, <b>6</b>, and <b>12</b>–<b>23</b> by a sequence of optional reductive N-alkylation and amide formation. For the hydroxyacetyl derivatives <b>5</b>, <b>6</b>, and <b>12</b>–<b>21</b>, the amide was formed by using acetoxyacetyl chloride, followed by treatment of the crude amide with methanolic potassium carbonate to remove the acetyl moiety. The amino acid amide <b>22</b> was prepared by a coupling with 9-fluorenylmethoxycarbonyl (Fmoc) protected glycine (HATU, NMM, DMF), followed by removal of the Fmoc group with piperidine. Amide formation with methoxyacetyl chloride gave compound <b>23</b>. Furthermore, oxoimidazolidine <b>24</b> was prepared by reaction of the N-unsubstituted 4-aminopyrazoline derivative with chloroacetyl chloride and treatment of the resulting chloroacetamide with urotropine.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0018.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>5</b>, <b>6</b>, and <b>12</b>–<b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) aniline derivative, <i>n</i>-BuLi, THF, −78 °C, or alkylamine, 1,4-dioxane, reflux; (b) 1,3-dimethylbarbituric acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF; (c) acetaldehyde, NaBH<sub>4</sub>, DCM, 0 °C; (d) (1) acetoxyacetyl chloride, aqueous NaHCO<sub>3</sub> solution, DCM, (2) K<sub>2</sub>CO<sub>3</sub>, MeOH; (e) skip (for compound <b>19</b>), or aldehyde derivative, NaBH<sub>4</sub>, MeOH, 0 °C; (f) Fmoc-glycine, HATU, NMM, DMF; (g) piperidine, DMF; (h) methoxyacetyl chloride, aqueous NaHCO<sub>3</sub> solution, DCM; (i) chloroacetyl chloride, <i>N</i>,<i>N</i>-diisopropylethylamine, DCM; (j) urotropine, EtOH.</p></p></figure><div class="NLM_p">Compounds <b>4</b> and <b>25</b>–<b>29</b> were prepared from intermediates <b>10b</b>–<b>f</b> by the addition of 3-(difluoromethoxy)aniline, followed by installation of the N-ethylated hydroxyacetamide as described above (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0019.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>4</b> and <b>25</b>–<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 3-(difluoromethoxy)aniline, <i>n</i>-BuLi, THF, −78 °C; (b) 1,3-dimethylbarbituric acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF; (c) acetaldehyde, NaBH<sub>4</sub>, MeOH, 0 °C; (d) (1) acetoxyacetyl chloride, aqueous NaHCO<sub>3</sub> solution, DCM, (2) K<sub>2</sub>CO<sub>3</sub>, MeOH.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>30</b>–<b>34</b> started from intermediate <b>10d</b> by addition of the respective aniline derivatives, followed by introduction of the N-ethylated hydroxyacetamide (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0020.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>30</b>–<b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) aniline derivative, <i>n</i>-BuLi, THF, −78 °C; (b) 1,3-dimethylbarbituric acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF; (c) acetaldehyde, NaBH<sub>4</sub>, MeOH, 0 °C; (d) (1) acetoxyacetyl chloride, aqueous NaHCO<sub>3</sub> solution, DCM, (2) K<sub>2</sub>CO<sub>3</sub>, MeOH.</p></p></figure><div class="NLM_p last">Compounds <b>4</b>, <b>28</b>, and <b>30</b>–<b>34</b> were separated into their enantiomers by preparative chiral HPLC or chiral supercritical fluid chromatography. For compounds <b>6</b> and <b>4</b>, the integrity of the pyrazoline stereocenter was tested. The compounds were stable to racemization in aqueous solution at pH 7, as well as in mouse and human plasma at 37 °C, for at least 48 h. However, clean racemization was obtained under basic conditions and microwave irradiation (DBU, THF, 90 °C).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Activity Relationships</h3><div class="NLM_p">Our initial structure–activity relationship (SAR) studies of 3-(4-chlorophenyl)pyrazoline derivatives with respect to SMYD2 inhibition are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>. For the carboximidamide N-substituent (R<sup>3</sup>), a meta-substituted phenyl seemed optimal, with the 3-(difluoromethoxy)phenyl derivative <b>6</b> displaying highest potency and binding efficiency<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (IC<sub>50</sub> = 0.8 μM, BEI = 12.5). Small para substituents are tolerated, such as with the 3-chloro-4-fluorophenyl, 3,4-dichlorophenyl, and 2,2-difluoro-1,3-benzodioxol-5-yl derivatives <b>5</b>, <b>12</b>, and <b>13</b> (IC<sub>50</sub> = 1.7, 3.3, and 2.2 μM, respectively). In contrast, the 4-(difluoromethoxy)phenyl derivative <b>14</b> is more than 15-fold less active than the corresponding meta-derivative <b>6</b>. The extended 3-chlorophenethyl derivative <b>15</b> showed no measurable activity (IC<sub>50</sub> > 20 μM), implying that the space within the relevant binding pocket is limited. Nonaromatic derivatives, such as 4,4,4-trifluorobutyl-substituted <b>16</b> or the cyclopropylethyl derivative <b>18</b>, are less potent than <b>6</b> (IC<sub>50</sub> = 6.9 and 8.3 μM for <b>16</b> and <b>18</b>, respectively). Surprisingly, the corresponding <i>n</i>-butyl derivative <b>17</b> is inactive (IC<sub>50</sub> > 20 μM). The SAR for PAR1 antagonism at the carboximidamide N-substituent appears to be rather flat: all compounds <b>5</b>, <b>6</b>, and <b>12</b>–<b>18</b> are potent PAR1 antagonists, with IC<sub>50</sub> values below 100 nM.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of 3-(4-Chlorophenyl)pyrazoline Derivatives: Variation of the Carboximidamide N-Substituent</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0021.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0022.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">SMYD2 thermal shift assay.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">Binding efficiency index: pIC<sub>50</sub>/MW × 1000.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></p></div></div><div></div></div><div class="NLM_p">With respect to the amide moiety, it became clear that the presence and orientation of a hydrogen-bond donor has a large impact on potency toward SMYD2 (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Variation of the amide <i>N</i>-alkyl substituents (R<sup>1</sup>) revealed that the <i>N</i>-ethyl derivative is preferred: secondary amide <b>19</b> (IC<sub>50</sub> = 10.9 μM) is about 10-fold less active and the <i>N</i>-propyl derivative <b>20</b> is also less potent than the corresponding <i>N</i>-ethyl derivative <b>6</b> (IC<sub>50</sub> = 0.8 μM), whereas the larger cyclopropylmethyl substituent in <b>21</b> resulted in a loss of potency (IC<sub>50</sub> > 20 μM). Replacement of the hydroxyacetyl in compound <b>12</b> (IC<sub>50</sub> = 3.3 μM) by an aminoacetyl group gave the equipotent derivative <b>22</b> (IC<sub>50</sub> = 2.8 μM). In contrast, the methoxyacetyl derivative <b>23</b>, lacking the hydrogen-bond donor, is inactive (IC<sub>50</sub> > 20 μM). In compound <b>24</b>, where the hydrogen-bond donor is fixed in an oxoimidazolidine ring, there is significantly reduced potency (IC<sub>50</sub> = 6 μM) relative to the <i>N</i>-ethyl derivative <b>6</b>. On the basis of the available amide derivatives, with BEI values in the same range (10–11), an improvement in the binding efficiency was not envisaged. Furthermore, alterations of the amide moiety did not offer a path forward to selectivity against PAR1. Compounds <b>22</b>, <b>23</b>, and <b>24</b> are in a similar potency range for PAR1 antagonism and greater than 10-fold more potent against PAR1 (IC<sub>50</sub> = 130, 30, and 100 nM, respectively) than SMYD2.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of 3-(4-Chlorophenyl)pyrazoline Derivatives: Variation of the Amide Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0023.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0001.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">SMYD2 thermal shift assay.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">Binding efficiency index: pIC<sub>50</sub>/MW × 1000.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></p></div></div><div></div></div><div class="NLM_p">At this stage, we selected compound <b>6</b> for co-crystal structure determination with SMYD2 based on its potency and promising biophysical properties. Compound <b>6</b> was soaked into crystals of SMYD2 grown in the presence of SAM. The crystal structure revealed that compound <b>6</b> binds into the substrate peptide binding pocket of SMYD2; the observed binding mode is consistent with the previously established SAR. There is a very good steric and electrostatic fit of <b>6</b> to the substrate binding site of SMYD2 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A,B). The pyrazoline and the NH of the carboximidamide form hydrogen bonds to Gly183. The 4-chlorophenyl substituent inserts into the lysine binding channel and is engaged in π-stacking interactions with Phe184 and Tyr240. There is a good fit of the 3-(difluoromethoxy)phenyl substituent into the adjacent hydrophobic pocket-1, with the difluoromethoxy group pointing into a hydrophobic subpocket. Although racemic <b>6</b> was used for soaking, the density maps reveal that only the <i>S</i>-enantiomer is bound. The <i>S</i> configuration at the pyrazoline provides an optimal exit vector for the hydroxyacetamide substituent, which occupies pocket-2 and forms two hydrogen bonds with Thr185 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). The <i>N</i>′-nitrile of the carboximidamide contributes to a water-bridged hydrogen bond with Ser196. On the basis of the crystal structure of compound <b>6</b>, we envisioned that further exploration of the aniline and amide moieties would not lead to a significant improvement of potency and binding efficiency. Therefore, we elected to focus on derivatives of the 4-chlorophenyl substituent, which were underrepresented in our compound library.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0010.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode of compound <b>6</b> and (<i>S</i>)-<b>4</b>. (A,B) Two different views of the co-crystal structure of <b>6</b> in complex with SMYD2. There is a good steric and electrostatic fit of <b>6</b> into the substrate binding site of SMYD2, and multiple hydrogen-bond interactions are involved. The lysine channel is occupied by the 4-chlorophenyl residue of the ligand, representing a novel lysine channel binding motif. In addition, pockets-1 and -2 are addressed. (C) Visualization of the WaterMap calculations suggesting an optimal displacement of water molecules by a 3,4-dichlorophenyl residue. Calculated water sites are shown as spheres and colored based on their free energy. Only sites within the lysine channel are shown. WaterMap results are based on the crystal structure of the monochloro derivative <b>6</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARF">5ARF</a>). The protein surface is depicted in gray. The modeled ligand structure is shown as colored sticks (chlorine, green; carbon, yellow; hydrogen, white; nitrogen, blue; oxygen, red; fluorine, light green). For clarity, protein residues and cofactors are not shown. (D) Overlay of the crystal structures of compound <b>6</b> (magenta) and (<i>S</i>)-<b>4</b> (yellow, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARG">5ARG</a>), showing nearly identical binding modes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">With the co-crystal structure in hand, we employed molecular modeling for the prioritization of derivatives. In particular, WaterMap calculations,<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> which estimate the position and the thermodynamic properties of water molecules in the ligand-free structure, suggested the introduction of a second substituent at the 3-position of the 4-chlorophenyl group. To test this hypothesis, a few derivatives were synthesized (see <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). The unsubstituted phenyl derivative <b>25</b> is inactive, while the 4-bromo derivative <b>26</b> (IC<sub>50</sub> = 1.1 μM) is as potent as the 4-chloro derivative <b>6</b>, indicating the importance of a hydrophobic substituent at the 4-position. As predicted by WaterMap calculations, introduction of a second substituent at the meta position, as exemplified by 3,4-dichloro derivative <b>4</b> (IC<sub>50</sub> = 0.08 μM, BEI = 13.5) and 4-chloro-3-methyl derivative <b>28</b> (IC<sub>50</sub> = 0.08 μM, BEI = 14), resulted in significantly improved potency (ca. 10-fold greater than <b>6</b>) and binding efficiency. As highlighted in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C, the two chloro substituents are colocated with two calculated water sites that have high free energy, suggesting an optimal water displacement by 3,4-dichlorophenyl and thus a lower binding free energy than for the 4-chlorophenyl or the unsubstituted phenyl derivative, consistent with the observed potency difference for these derivatives. However, introduction of a third substituent leads to a decrease in potency compared to the 3,4-disubstituted derivatives, as exemplified by 4-chloro-3,5-dimethyl derivative <b>27</b> (IC<sub>50</sub> = 0.57 μM). Compound <b>29</b>, with the bulkier 1,3-benzodioxole moiety, is also less potent (IC<sub>50</sub> = 4.2 μM), suggesting that the SAR at this position is rather steep.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of 3-Phenylpyrazoline Derivatives: Exploration of Phenyl Substituents</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0002.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0003.gif" alt="" id="GRAPHIC-d156e1308-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">SMYD2 thermal shift assay.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">Binding efficiency index: pIC<sub>50</sub>/MW × 1000.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></p></div></div><div></div></div><div class="NLM_p">Compound <b>4</b> was selected for further biological and crystallographic studies. The co-crystal was obtained by soaking (<i>S</i>)-<b>4</b> into crystals of SAM-bound SMYD2. Structure determination revealed that (<i>S</i>)-<b>4</b> features an almost identical binding mode as compound <b>6</b>, as evidenced in the overlay of both structures (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D). In line with the observed <i>S</i> configuration of compound <b>6</b> and (<i>S</i>)-<b>4</b>, we observed a greater than 50-fold difference in the IC<sub>50</sub> values of the two enantiomers of <b>4</b> [27 nM for (<i>S</i>)-<b>4</b> vs 1.7 μM for (<i>R</i>)-<b>4</b>, see <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>]. Introduction of the additional 3-chloro substituent also resulted in a dramatic decrease in the antagonistic effect on PAR1 [IC<sub>50</sub> = 1.7 μM for (<i>S</i>)-<b>4</b> and >30 μM for (<i>R</i>)-<b>4</b>]. Although (<i>S</i>)-<b>4</b> proved to be the active isomer for both SMYD2 inhibition and PAR1 antagonism, there is a greater than 50-fold selectivity for SMYD2 relative to PAR1.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. In Vitro Pharmacokinetic Properties of Potent SMYD2 Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0004.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0005.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">ee >95%.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">Rat hepatocytes.</p></div><div class="footnote" id="t4fn3"><sup>Table c</sup><p class="last">Permeability <i>P</i><sub>app</sub>(A–B)/efflux ratio.</p></div><div class="footnote" id="t4fn4"><sup>Table d</sup><p class="last">Aqueous solubility.</p></div></div><div></div></div><div class="NLM_p">The co-crystal structures of compound <b>6</b> and (<i>S</i>)-<b>4</b> revealed that the pyrazoline-based SMYD2 inhibitors feature a different binding mode to other recently reported inhibitors. Thus, <b>1</b><a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and <b>2</b><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A) have published IC<sub>50</sub> values of 0.12 μM and <15 nM, respectively, and like (<i>S</i>)-<b>4</b> they bind to the substrate binding site of SMYD2. The structures of the three inhibitors are superimposed in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B. All ligands occupy the lysine binding channel and the adjacent hydrophobic pocket-1; however, (<i>S</i>)-<b>4</b> employs pocket-2 via hydrogen-bond interactions with its hydroxyacetyl moiety (cf. <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A), which are not present in <b>1</b> or <b>2</b>. <b>1</b> and <b>2</b>, on the other hand, occupy a remote hydrophobic pocket-3 which is not addressed by (<i>S</i>)-<b>4</b>. <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C provides a view into the lysine binding channel (the respective motifs are highlighted in color on the structures in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). <b>2</b> binds with its <i>N</i>-alkylpyrrolidine moiety at this position, which is structurally closely related to the lysine side chain of a methylated substrate peptide. <b>1</b> and (<i>S</i>)-<b>4</b> address the lysine channel with aromatic substituents. It is noteworthy that the 3,4-dichlorophenyl group of <b>1</b> and of (<i>S</i>)-<b>4</b> bind to different positions.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0011.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of (<i>S</i>)-<b>4</b> (this work) with the recently reported SMYD2 inhibitors <b>1</b><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and <b>2</b>.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (A) Chemical structures of the three selected SMYD2 inhibitors. (B,C) Two different views of an overlay of (<i>S</i>)-<b>4</b> (yellow) with <b>1</b> (green) and <b>2</b> (blue). (<i>S</i>)-<b>4</b> has a distinct binding mode, addressing pocket-2 which is not occupied by <b>1</b> or <b>2</b>, while only <b>1</b> and <b>2</b> occupy the distant hydrophobic pocket-3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">On the basis of 3,4-dichlorophenyl as a novel lysine channel binding motif, we identified several pyrazoline derivatives as potent SMYD2 inhibitors (see <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Aiming to identify a candidate for in vivo experiments, we focused on the potent <i>S</i>-enantiomers of these derivatives, profiling them in pharmacokinetic assays in vitro. (<i>S</i>)-<b>4</b> showed moderate stability upon incubation with rat hepatocytes (CL<sub>blood</sub> = 2.5 L/h/kg) as well as moderate apparent permeability (34 nm/s) and a hint of active transport in the Caco2 assay (efflux ratio = 5). In comparison, the 4-chloro-3-methylphenyl derivative (<i>S</i>)-<b>28</b> had similar permeability and efflux (33 nm/s, efflux ratio = 7) and slightly lower metabolic stability in rat hepatocytes (CL<sub>blood</sub> = 2.8 L/h/kg). Compounds (<i>S</i>)-<b>30</b> and (<i>S</i>)-<b>32</b>–<b>34</b> exhibited high metabolic stability (CL<sub>blood</sub> = 1.8–0.3 L/h/kg); however, these derivatives have very low aqueous solubility (<5 mg/L), thereby limiting their suitability for in vivo experiments. On the other hand, the methoxyphenyl derivative (<i>S</i>)-<b>31</b> has high aqueous solubility (163 mg/L) and moderate permeability with a hint of active transport (62 nm/s, efflux ratio = 3) in the Caco2 assay; nevertheless, (<i>S</i>)-<b>31</b> displayed low stability upon incubation with rat hepatocytes (CL<sub>blood</sub> = 3.2 L/h/kg), and thus bioavailability is expected to be low. On the basis of these data, (<i>S</i>)-<b>4</b> was selected for further in vitro and in vivo studies. The in vivo pharmacokinetic properties of (<i>S</i>)-<b>4</b> were first evaluated by a single-dose administration of (<i>S</i>)-<b>4</b> at 0.4 mg/kg by iv bolus or 0.8 mg/kg po, respectively, to rats; there was moderate blood clearance (1.6 L/h/kg) and a low bioavailability of 24% (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Figure S1A</a>). These data prompted us to assess the exposure of (<i>S</i>)-<b>4</b> following oral administration. Hence, we treated mice with 10–100 mg/kg po qd which are well tolerated doses of (<i>S</i>)-<b>4</b>. As a result, unbound IC<sub>50</sub> [cellular methylation In-Cell Western (ICW) assay; see <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A–C] is covered for ∼9 to ∼12 h at steady state when 10 and 100 mg/kg, respectively, are administered (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Figure S1B</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Potency, Selectivity, and Inhibition Mode</h3><div class="NLM_p">(<i>S</i>)-<b>4</b> showed potent in vitro inhibition of SMYD2 with an IC<sub>50</sub> of 27 ± 7 nM in the biochemical SPA assay (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). To analyze the mode of inhibition and affinity of (<i>S</i>)-<b>4</b>, our SMYD2 protein preparation was characterized with respect to its apparent Michaelis–Menten constants for SAM and the p53 peptide substrate (data not shown). The <i>K</i><sub>m(app)</sub> values of 60 nM and 1 μM, respectively, were in excellent agreement with the constants previously reported using a similar assay.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Then, we performed IC<sub>50</sub> determinations at increasing concentrations of one substrate and a fixed amount of the other ([S] = <i>K</i><sub>m(app)</sub>) and applied the Cheng–Prusoff relationship<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> as described elsewhere.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Increasing the concentration of peptide substrate resulted in a linear increase in IC<sub>50</sub> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B, upper panel), as would be expected for a competitive mode of inhibition. Fitting these data to the corresponding Cheng–Prusoff competitive inhibition model revealed a <i>K</i><sub>i(app)</sub> of 8 ± 1 nM (SD). On the other hand, when SAM concentrations were titrated to saturation, we observed a decrease in IC<sub>50</sub> which converged to a constant value (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B, lower panel). This type of behavior toward SAM corresponds to an uncompetitive mode of inhibition. Consequently, the IC<sub>50</sub> vs [S]/<i>K</i><sub>m(app)</sub> plot fits best to the Cheng–Prusoff model for uncompetitive inhibition, yielding an inhibitor constant α<i>K</i><sub>i</sub><sub>(app)</sub> of 28 ± 3 nM (SD). Our data suggest that (<i>S</i>)-<b>4</b> is a peptide-competitive, SAM-uncompetitive inhibitor of SMYD2 methyltransferase activity, which preferably binds to the SMYD2–SAM substrate complex. Interestingly, SMYD2 has been reported to follow a random sequential Bi Bi mechanism of substrate binding,<a onclick="showRef(event, 'ref12 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref17">(12, 17)</a> but the uncompetitive mode of inhibition of (<i>S</i>)-<b>4</b> regarding the SAM cofactor suggests an ordered sequential Bi Bi mode of substrate binding, where SAM would be required to bind before the peptide substrate. Similar results have recently been described for inhibitors of the SAM-dependent arginine methyltransferase PRMT5.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0012.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (<i>S</i>)-<b>4</b> mode of inhibition and selectivity profile. (A) Activity in the scintillation proximity assay (SPA). IC<sub>50</sub> (<i>n</i> > 10) for SMYD2 inhibition = 26 ± 7 nM (representative inhibition curve shown). (B) IC<sub>50</sub> values obtained from SPA were plotted against the indicated substrate concentrations (represented as [substrate]/<i>K</i><sub>m(app)</sub>). Data were fitted to competitive and uncompetitive models of the Cheng–Prusoff equation.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41, 42)</a> Data points are the mean of eight replicates, error bars indicate 1 × SD. The data indicate that (<i>S</i>)-<b>4</b> is a peptide-competitive, SAM-uncompetitive inhibitor of SMYD2 methyltransferase activity. (C) Selectivity profile on a panel of 32 additional methyltransferases showing high selectivity of (<i>S</i>)-<b>4</b> for SMYD2. Only SMYD3 is weakly inhibited by (<i>S</i>)-<b>4</b> with <i>a</i> > 1 μM IC<sub>50</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">For further evaluation of selectivity, (<i>S</i>)-<b>4</b> was tested on a panel of 32 additional methyltransferases, including closely related family members SMYD3, SUV420H1, and SUV420H2. As a result, (<i>S</i>)-<b>4</b> displayed >100-fold selectivity for SMYD2, with very weak residual activity toward the closest related methyltransferase SMYD3 (IC<sub>50</sub> ∼ 3 μM) (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C). In addition to the methyltransferases, (<i>S</i>)-<b>4</b> was also profiled in the commercially available KINOMEscan (DiscoveRx)<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and LeadProfilingScreen (Eurofins Panlabs) assay panels to fully determine relative selectivity and specificity for kinases and other primary molecular targets, including several CNS targets. Overall, we were able to confirm the high selectivity and specificity of (<i>S</i>)-<b>4</b> for SMYD2 inhibition (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Tables S1 and S2</a>).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cellular Methylation Activity on p53</h3><div class="NLM_p">The ability of (<i>S</i>)-<b>4</b> to inhibit SMYD2 was tested by monitoring its effects on cellular p53 methylation using different cellular mechanistic assays. First, we generated a polyclonal antibody (SY46) for the specific detection of p53 monomethylation at lysine 370, as described elsewhere.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> This antibody was then tested on recombinant p53 protein which had been in vitro methylated by SMYD2 in a Western blot. Specificity for methylated p53 was confirmed by the exclusive detection of the in vitro methylated p53 protein isoforms, whereas the nonmethylated p53 was not detected (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A). Endogenous methylation of p53 protein was characterized by treatment of KYSE-150 esophageal cancer cells with increasing concentrations of (<i>S</i>)-<b>4</b> for 5 days. The KYSE-150 cell line model was selected based on a described SMYD2 gene amplification<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and a heterozygous R248Q mutation in p53 (COSMIC), leading to p53 protein accumulation without a stress stimulus. After treatment with (<i>S</i>)-<b>4</b>, a significant reduction of methylation was detected confirming that p53 is a cellular target of SMYD2-dependent methylation (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B). Nevertheless overall endogenous detection of p53 protein methylation led to weak signals, hence this method was not useful for the determination of a cellular IC<sub>50</sub>. Therefore, we additionally employed an established system with a transient FLAG-tagged SMYD2 and FLAG-tagged p53 overexpression in HEK293T cells as benchmark assay. This assay has been used previously to characterize the structurally unrelated SMYD2 inhibitor <b>2</b>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> As shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C, (<i>S</i>)-<b>4</b> showed a concentration-dependent decrease in p53 methylation without affecting p53 total protein levels. A cellular IC<sub>50</sub> of 58 nM was determined (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D), which confirms that (<i>S</i>)-<b>4</b> is the most potent cellular-active SMYD2 inhibitor known to date (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0013.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of cellular methylation of the tumor suppressor protein p53 by (<i>S</i>)-<b>4</b>. (A) A specific antibody directed against p53K370me1 (SY46) was generated and tested for specificity on recombinant p53 (rec. p53) which had been in vitro methylated by SMYD2, or nonmethylated. (B) Endogenous methylation of p53 protein was characterized by treatment of KYSE-150 esophageal cancer cells with increasing concentrations of (<i>S</i>)-<b>4</b> for 5 days (Co = control). (C) Cellular p53 methylation assay using transient FLAG-tagged SMYD2 and FLAG-tagged p53 overexpression in HEK293T cells. Increasing concentrations of (<i>S</i>)-<b>4</b> reduce the methylation of overexpressed p53 (for this assay p53K730me1-specific antibody kindly provided by Dr. Shelly Berger was used). (D) IC<sub>50</sub> determination in the cellular p53 methylation assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Characterization and Inhibition of SMYD2-Mediated AHNAK Methylation</h3><div class="NLM_p">To further characterize the effects of our aminopyrazoline-based inhibitors on the cellular methylation activity of SMYD2, we generated a polyclonal cell line derived from KYSE-150 with stable N-terminal 2xc-myc-tagged SMYD2 overexpression to maximize cellular methylation activity. In an immunofluorescence analysis, SMYD2 was localized mainly in the cytosol (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Figure S2A</a>), as reported by others.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Surprisingly, the antibody SY46 directed against methylated p53 showed a very strong signal specifically in the clones with SMYD2 overexpression in the immunochemistry staining not derived from p53 protein (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Figure S2B,C</a>). We validated the novel SMYD2 substrate giving rise to the strong immunochemistry staining signal by knockdown and overexpression experiments in additional cell lines to be derived from AHNAK protein<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Figure 2D–F</a>). AHNAK methylation has been very recently reported in a proteomics study by other as an additional methylation substrate of SMYD2, further confirming our results.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Importantly, we used the strong AHNAK methylation signal to set up an In Cell Western (ICW) assay for cellular optimization of our lead series (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). The methylation signal was reduced by treatment with increasing concentrations of (<i>S</i>)-<b>4</b>, derivatives of the aminopyrazoline series or <b>1</b> (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). In addition, (<i>S</i>)-<b>4</b> specifically reduced methylation of AHNAK without altering AHNAK protein expression in a Western blot (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B). Importantly, IC<sub>50</sub> values in the cellular ICW assay for aminopyrazolines correlated with potency in the scintillation proximity assay (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>C) and were also comparable to the p53 methylation assay (see <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D). Our results of the methylation of AHNAK and the recent identification of many additional methylation targets<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> clearly point toward additional roles beyond p53 regulation of SMYD2, and further studies supported by potent and selective inhibitors as (<i>S</i>)-<b>4</b> are needed to fully elucidate the underlying biology.</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0014.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. AHNAK is a novel substrate of SMYD2 and methylation can be inhibited by aminopyrazoline inhibitors. (A) In-Cell Western (ICW) assay. Immunofluorescence-based detection of AHNAK methylation. Each row represents a different inhibitor tested in increasing concentrations ranging from 39 nM up to 5 μM. (<i>S</i>)-<b>4</b> and <b>1</b> are highlighted. (B) Inhibition of AHNAK methylation in MDA-MB231 cells overexpressing SMYD2. Methylation signal overlaps with AHNAK protein detection. (<i>S</i>)-<b>4</b> inhibits concentration dependent the AHNAK methylation. (C) Good correlation of cellular activity (ICW) to biochemical activity (SPA) of (<i>S</i>)-<b>4</b> and pyrazoline cluster derivatives. (<i>S</i>)-<b>4</b> has a cellular IC<sub>50</sub> ∼ 60 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Characterization of (<i>S</i>)-4 in Proliferation and Apoptosis Assays</h3><div class="NLM_p">To explore the potential effects of (<i>S</i>)-<b>4</b> on proliferation, we tested a panel of 240 different cancer cell lines (OncoPanel 240/Eurofins Panlabs). Cell lines were long-term cultured with (<i>S</i>)-<b>4</b> for 10 days to allow for a sufficient translation of demethylation of SMYD2 substrates to potential antiproliferative effects, which takes 48–72 h (Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Figure S2C</a>). (<i>S</i>)-<b>4</b> exposure resulted in only limited responses in a subset of cell lines; an antiproliferative response with IC<sub>50</sub> < 10 μM was seen in 21 cell lines (∼9%), however, most cell lines (83%) did not reach 50% proliferation inhibition with 20 μM of (<i>S</i>)-<b>4</b> (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>A). In addition, there was no clear preference for a specific tissue origin of the responding cancer cell line. Thus, SMYD2 inhibition by (<i>S</i>)-<b>4</b> has only limited proliferation effects in a small subset of cancer cell lines under the employed conditions. On the basis of the observation that p53 protein is methylated by SMYD2, which should lead to suppression of apoptosis, we were additionally interested in the effects of (<i>S</i>)-<b>4</b> in combination with an apoptotic stimulus. KYSE-150, U2OS, and A2780 cell lines were pretreated with (<i>S</i>)-<b>4</b> or inactive derivative <b>25</b> (see <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) for 2 days (demethylation phase), followed by treatment with doxorubicin (apoptotic trigger). (<i>S</i>)-<b>4</b>, but not <b>25</b>, significantly improved caspase 3/7 activation in all three tested cell lines without inducing apoptosis alone (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B). Thus, SMYD2 inhibition can enhance apoptotic responses.</div><figure id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0015.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Proliferation and apoptosis effects of (<i>S</i>)-<b>4</b>. (A) A panel of 240 different cancer cell lines (OncoPanel 240/Eurofins Panlabs) was long-term cultured with (<i>S</i>)-<b>4</b> for 10 days and proliferation effects were determined. (<i>S</i>)-<b>4</b> showed proliferation effects with an IC<sub>50</sub> < 10 μM in a subset of cell lines. (B) Effects on apoptosis induction were determined by caspase 3/7 activation. KYSE-150, U2OS, and A2780 cell lines were pretreated with (<i>S</i>)-<b>4</b> or inactive derivative <b>25</b> (see <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) for 2 days, then apoptosis was induced by treatment with doxorubicin. (<i>S</i>)-<b>4</b>, but not <b>25</b>, significantly improving caspase 3/7 activation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> The First Chemical Probe Suitable for In Vivo Characterization of SMYD2 Inhibition</h3><div class="NLM_p">Functional validation of novel potential cancer targets such as the protein methyltransferase SMYD2 relies on appropriate model systems in vitro as well as in vivo. Additionally, chemical probe inhibitors also suitable for in vivo applications are highly desirable. Most reported activities of SMYD2 are not directly involved in survival signaling of cancer cells. Hence, in vitro proliferation assays may not adequately cover the full phenotype of SMYD2 inhibition, and more complex (in vivo) assays are thus required. The research work characterizing SMYD2 in heart and skeletal muscle cells by knockdown experiments in vivo<a onclick="showRef(event, 'ref15 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref15 ref24 ref25">(15, 24, 25)</a> clearly illustrates the complex underlying biology of SMYD2 and the necessity for chemical probes suitable for in vivo applications. Therefore, we were interested in establishing if SMYD2 methylation activity in tumor cells can be inhibited by (<i>S</i>)-<b>4</b> in vivo. To this end, mice bearing subcutaneous tumor xenografts (tumor tissue derived from the SMYD2-overexpressing KYSE-150 cell line) were treated orally with 10, 30, 70, or 100 mg/kg (<i>S</i>)-<b>4</b>, or vehicle (PEG 400/water 8:2), once daily for 3 days. After the treatment period, tumors were harvested and analyzed ex vivo for methylation of AHNAK by dot-blotting. For detection of the methylation signals, SY46 methylation antibody was used (see <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Figure S2A–F</a>). (<i>S</i>)-<b>4</b> significantly reduced the methylation with doses starting from 30 mg/kg, with most significant effects in the 100 mg/kg treated group (<i>P</i> < 0.001, Student’s <i>t</i> test) (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>A). Treatment with 10 mg/kg (<i>S</i>)-<b>4</b> resulted in no significant effect on the methylation level. Exposure at 10 mg/kg is close to the level of the cellular IC<sub>50</sub> for ∼9 h, which may indicate a need for an even higher exposure as the IC<sub>50</sub> to achieve in vivo effects on demethylation.</div><figure id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0016.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. In vivo characterization of (<i>S</i>)-<b>4</b>. (A) Demethylation of AHNAK was evaluated ex vivo. Average methylation signals ± standard deviation per group are shown. Mice (<i>n</i> = 12 per group) bearing subcutaneous tumor xenografts (tumor tissue derived from the SMYD2-overexpressing KYSE-150 cell line) were treated as indicated, then the tumors were analyzed for methylation signals by dot-blotting. (<i>S</i>)-<b>4</b> significantly reduced the methylation with doses starting from 30 mg/kg, with most significant effects in the 100 mg/kg group (<i>P</i> < 0.001, Student’s <i>t</i> test). (B) Tumor area graph summarizing the in vivo tumor efficacy study with the KYSE-150 xenograft model. Average tumor area ± standard error of mean per group is plotted as the mean. Treatment was started at day 4 after tumor inoculation (black arrow), and groups were treated as indicated. (C) Tumor weight graphs corresponding to the same experiment shown in (B). Average tumor weight are blotted as box plot. * Significant (<i>p</i> value <0.05) difference between vehicle control and treatment group (Dunn’s method). Group 2: one animal was excluded on the 14th treatment day due to animal welfare reasons (ulcerated tumor), tumor size 82 mm<sup>2</sup>. (D) Mouse body weight analysis. Average body weight per group throughout the experiment is plotted as the mean.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Then the KYSE-150 esophageal xenograft model was used to evaluate if the observed improved apoptosis induction in the in vitro setting after treatment with doxorubicin (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B) could translate to antitumor efficacy in vivo. Four groups of tumor-bearing mice were treated as follows: Group 1 (control group) was only treated with vehicle (Solutol/ethanol/water 1:1:8) iv qd and once at day 4 with the vehicle used for doxorubicin (saline) iv; group 2 was treated with (<i>S</i>)-<b>4</b> at 500 mg/kg po qd; group 3 was treated with 10 mg/kg doxorubicin iv once at day 4; group 4 was treated with a combination of (<i>S</i>)-<b>4</b> and doxorubicin. There was a slight reduction in area (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>B) and weight (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>C) of tumors from mice treated with the combination of (<i>S</i>)-<b>4</b> and doxorubicin relative to tumors from the control group. Combination treatment only resulted in a minor increase in treatment-related body weight loss (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>D). The combination treatment reached a T/C level (based on tumor weight) of 0.46, which is significant (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>C). Therefore, combination of a SMYD2 inhibitor with a chemotherapeutic agent resulted in reduced cancer cell growth in vivo. In comparison, the monotherapy groups treated with only doxorubicin or (<i>S</i>)-<b>4</b> showed no significant antitumor efficacy relative to the vehicle control group. These data are consistent with the observed limited cellular proliferation effects of (<i>S</i>)-<b>4</b> in the cell line panel (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>A) and indicate that, in contrast to SMYD2 knockdown,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> a catalytic inhibition may be insufficient to induce cell death in the KYSE-150 esophageal model. Furthermore, the monotherapeutic approach with doxorubicin did not result in any antitumor efficacy in the KYSE-150 xenograft model.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have identified (<i>S</i>)-<b>4</b> as a potent, selective, and cell-active, substrate-competitive inhibitor of SMYD2. Our data show that SMYD2 inhibition can enhance efficacy of doxorubicin in vivo, which confirms our in vitro observation of higher caspase 3/7 activation (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B). Our results (<i>S</i>)-<b>4</b> are also in agreement with an earlier study,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> where an increased apoptosis induction in cells with an SMYD2 knockdown was observed. Nonetheless, in our initial explorative in vivo study with (<i>S</i>)-<b>4</b>, effects on xenografted tumors were only moderate. In addition, high doses of (<i>S</i>)-<b>4</b> were needed in vitro as well as in vivo relative to the concentration needed to achieve effects on methylation. Therefore, we cannot exclude the possibility that additional, so-far unexplored activities of SMYD2 might be responsible for the observed effects. This again underlines the necessity to identify suitable chemical probes for more extensive target validation campaigns to fully explore the complex biology of SMYD2 and other targets. In this regard, (<i>S</i>)-<b>4</b> will be a highly valuable tool for the further exploration of SMYD2 biology, not only for in vitro but also for in vivo studies.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77333" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77333" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> General Procedures</h4><div class="NLM_p last">All reagents and solvents were used as purchased, unless otherwise specified. All final products were at least 95% pure, as determined by analytical HPLC.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Materials</h4><div class="NLM_p last">Intermediate <b>10a</b> and compounds <b>5</b>, <b>6</b>, and <b>12</b>–<b>24</b> were synthesized according to the methods described previously.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a><sup>1</sup>H NMR were determined to be >95%. <sup>1</sup>H NMR spectra were recorded on Bruker Avance III HD spectrometers operating at 300, 400, or 500 mHz. The chemical shifts (δ) reported are given in parts per million (ppm), and the coupling constants (<i>J</i>) are in hertz (Hz). The spin multiplicities are reported as s = singlet, br s = broad singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br = broad. The LC/MS analysis was performed on Waters Acquity UPLCMS SingleQuad with a Acquity UPLC BEH C18 column (1.7 μm, 50 mm × 2.1 mm) at 60 °C, using water +0.1 vol % formic acid (99%) and acetonitrile as mobile phase at a flow rate of 0.8 mL/min and a DAD detector (210–400 nm). LC/MS/MS was performed on a CTC PAL autosampler, an Agilent 1200 HPLC, and a ABSciex 4000 mass spectrometer.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Assignment of Stereochemistry</h4><div class="NLM_p last">For all separated enentiomers, it was assumed based on the co-crystal structure of <b>4</b>, that the active enantiomer (SMYD2 inhibition) has <i>S</i>-configuration.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Phenyl 4-{[(Allyloxy)carbonyl]amino}-3-(4-bromophenyl)-<i>N</i>-cyano-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidate (<b>10b</b>)</h4><div class="NLM_p last">Compound <b>10b</b> was prepared as described for <b>10d</b>. Starting from <b>7b</b> (37.7 g, 0.18 mol), <b>10b</b> was obtained as white solid (40.1 g, 49% over 4 steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 4.26–4.42 (m, 3H), 4.66 (dd, <i>J</i> = 13.3, 6.0 Hz, 1H), 4.74 (dd, <i>J</i> = 13.1, 5.8 Hz, 1H), 5.30 (d, <i>J</i> = 10.3 Hz, 1H), 5.40 (d, <i>J</i> = 16.8 Hz, 1H), 5.67 (td, <i>J</i> = 8.8, 4.5 Hz, 1H), 6.00 (ddt, <i>J</i> = 16.8, 11.0, 5.6 Hz, 1H), 7.02–7.11 (m, 2H), 7.19–7.34 (m, 5H), 7.58 (d, <i>J</i> = 8.5 Hz, 2H). LC-MS (ESI): <i>m</i>/<i>z</i> = 468.1/470.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Phenyl 4-{[(Allyloxy)carbonyl]amino}-3-(4-chloro-3,5-dimethylphenyl)-<i>N</i>-cyano-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidate (<b>10c</b>)</h4><div class="NLM_p last">Compound <b>10c</b> was prepared as described for <b>10d</b>, starting from <b>7c</b> (35 g, 0.15 mol) to obtain <b>10c</b> (29.3 g, 43% over 4 steps) as white solid. <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 2.32 (s, 6H), 4.03–4.18 (m, 1H), 4.39–4.61 (m, 2H), 5.06–5.27 (m, 2H), 5.49–5.66 (m, 1H), 5.79–5.94 (m, 1H), 5.75–5.95 (m, 1H), 7.26 (d, <i>J</i> = 7.8 Hz, 2H), 7.30–7.37 (m, 1H), 7.41–7.55 (m, 2H), 7.56–7.73 (m, 2H), 8.17 (d, <i>J</i> = 8.7 Hz, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 452.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Phenyl 4-{[(Allyloxy)carbonyl]amino}-<i>N</i>-cyano-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidate (<b>10d</b>)</h4><div class="NLM_p last">To a stirred solution of 2-amino-1-(3,4-dichlorophenyl)ethanone hydrochloride (1:1) (<b>7d</b>; 116 g, 0.480 mol) in water (500 mL) was added allyl chloroformate (56.5 mL, 0.530 mol) in DCM (800 mL). The reaction mixture was cooled to 0 °C, and K<sub>2</sub>CO<sub>3</sub> (207 g, 1.49 mol) in water (1 L) was added dropwise over 1 h. The mixture was allowed to warm to rt, stirred overnight, then diluted with DCM (500 mL). The organic phase was washed with saturated aqueous NH<sub>4</sub>Cl solution (400 mL) followed by brine (500 mL), then dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude mixture was purified by dry flash column chromatography (eluent: DCM/heptane 2:1, 3:1, 4:1; DCM; EtOAc) to yield allyl [2-(3,4-dichlorophenyl)-2-oxoethyl]carbamate (<b>8d</b>; 120 g, 87%) as a white crystalline solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 4.62 (d, <i>J</i> = 5.5 Hz, 2H), 4.68 (d, <i>J</i> = 4.6 Hz, 2H), 5.25 (dd, <i>J</i> = 1.4, 10.5 Hz, 1H), 5.35 (dd, <i>J</i> = 1.4, 16.9 Hz, 1H), 5.72 (br s, 1H), 5.94 (m, 1H), 7.60 (d, <i>J</i> = 8.2 Hz, 1H), 7.79 (dd, <i>J</i> = 2.1, 8.5 Hz, 1H), 8.06 (d, <i>J</i> = 1.8 Hz, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 288.1 [M + H]<sup>+</sup>. To a stirred suspension of <b>8d</b> (50.0 g, 0.174 mol) in EtOH (390 mL) was added formaldehyde solution (20 mL, 0.261 mol; 37 wt % in water) followed by the dropwise addition of piperidine (26 mL, 0.261 mol) in EtOH (130 mL) over 30 min. The reaction mixture was stirred overnight; TLC indicated consumption of <b>8d</b>. The volatiles were removed by evaporation to yield an orange oil which was not further purified. To a solution of this crude material in EtOH (480 mL) was added hydrazine monohydrate (29.6 mL, 0.609 mol), and the reaction mixture was heated to reflux for 2.5 h. Then the mixture was allowed to cool to rt, concentrated, and poured over ice-cooled saturated NH<sub>4</sub>Cl solution (300 mL). The crude product was extracted with EtOAc (1.5 L), and the combined organic layers were washed with brine (300 mL) then dried over MgSO<sub>4</sub>, filtered, and concentrated to yield allyl [3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>9d</b>; 50.0 g, 91%) as a pale-yellow solid. <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 3.24 (m, 1H), 3.59 (m, 1H), 4.39–4.54 (m, 2H), 5.08–5.25 (m, 3H), 5.79–5.90 (m, 1H), 7.52 (dd, <i>J</i> = 1.8, 8.3 Hz, 1H), 7.57 (br s, 1H), 7.59 (m, 1H), 7.68 (d, <i>J</i> = 1.8 Hz, 1H), 7.84 (d, <i>J</i> = 8.7 Hz, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 314.1 [M + H]<sup>+</sup>. To a stirred suspension of <b>9d</b> (50.0 g, 0.159 mol) in <i>i</i>-PrOH (860 mL) was added diphenyl <i>N</i>-cyanocarbonimidate (38.0 g, 0.159 mol). The reaction mixture was heated to reflux, at which point the suspension dissolved into solution; after a further 10 min at reflux, a white precipitate formed. The mixture was stirred at reflux for a further 1 h then allowed to slowly cool to rt overnight. The precipitate was collected by filtration and washed with Et<sub>2</sub>O (2 × 250 mL). The resulting white solid was allowed to dry to yield <b>10d</b> (48.6 g, 67%) as a white solid. <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 4.13 (apparent d, <i>J</i> = 8.2 Hz, 1H), 4.47 (m, 3H), 5.14 (m, 2H), 5.51–5.63 (m, 1H), 5.79–5.90 (m, 1H), 7.23 (d, <i>J</i> = 7.3 Hz, 2H), 7.30 (t, <i>J</i> = 7.8 Hz, 1H), 7.45 (m, 2H), 7.79 (br m, 2H), 7.97 (br s, 1H), 8.19 (d, <i>J</i> = 8.7 Hz, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 458.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Phenyl 4-{[(Allyloxy)carbonyl]amino}-3-(4-chloro-3-methylphenyl)-<i>N</i>-cyano-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidate (<b>10e</b>)</h4><div class="NLM_p last">Compound <b>10e</b> was prepared as described for <b>10d</b>, starting from <b>7e</b> (3.1 g, 13.9 mmol) to obtain <b>10e</b> (2.12 g, 34.4% over 4 steps) as white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 2.18 (s, 3H), 4.30 (d, <i>J</i> = 6.4 Hz, 2H), 4.64 (d, <i>J</i> = 6.0 Hz, 2H), 5.24 (d, <i>J</i> = 10.5 Hz, 1H), 5.34 (d, <i>J</i> = 16.9 Hz, 1H), 5.60–5.70 (m, 1H), 5.94 (ddt, <i>J</i> = 5.6, 10.9, 16.8 Hz, 1H), 6.89–7.35 (m, 7H), 7.48 (m, 1H), 7.54 (m, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 438.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Phenyl 4-{[(Allyloxy)carbonyl]amino}-3-(1,3-benzodioxol-5-yl)-<i>N</i>-cyano-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidate (<b>10f</b>)</h4><div class="NLM_p last">Compound <b>10f</b> was prepared as described for <b>10d</b>, starting from <b>7f</b> (12.0 g, 55.7 mmol) to obtain <b>10f</b> (9.1 g, 38% over 4 steps) as white solid. <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 4.05–4.08 (m, 1H), 4.43–4.47 (m, 3H), 5.1–5.2 (m, 2H), 5.50–5.55 (m, 1H), 5.80–5.90 (m, 1H), 6.08 (s, 2H), 7.05 (d, <i>J</i> = 7.8 Hz, 1H), 7.21–7.27 (m, 2H), 7.27–7.41 (m, 3H), 7.43–7.50 (m, 2H), 8.16 (d, <i>J</i> = 8.7 Hz, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 434.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> General Procedure for the Synthesis of Allyl [1-(<i>N</i>′-Cyano-<i>N</i>-arylcarbamimidoyl)-3-phenyl-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamates <b>11j</b>–<b>s</b></h4><div class="NLM_p">To a stirred solution of the respective aniline derivative (3 equiv) in anhydrous THF (2 mL/mmol) at −78 °C was added <i>n</i>-BuLi (3 equiv, 2 M in hexane) dropwise while maintaining the reaction temperature below −65 °C. The reaction mixture was stirred at −78 °C for 1 h, then the respective phenyl 4-{[(allyloxy)carbonyl]amino}-<i>N</i>-cyano-3-phenyl-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidate <b>10</b> (1 equiv) in anhydrous THF (30 mL/mmol) was added dropwise while maintaining the reaction temperature below −65 °C. The reaction mixture was stirred at −78 °C for 2 h, then slowly poured over saturated NH<sub>4</sub>Cl solution (30 mL/mmol based on <b>10</b>). The crude product was extracted into EtOAc (30 mL/mmol); the combined organic layers were washed with brine (15 mL/mmol), dried over MgSO<sub>4</sub>, filtered, and concentrated. The crude solid was precipitated from a minimum volume of EtOAc, collected by filtration, and washed with Et<sub>2</sub>O to give the corresponding intermediate <b>11</b>.</div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Allyl [3-(4-Bromophenyl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11j</b>)</h5><div class="NLM_p last">Compound <b>11j</b> was prepared from <b>10b</b> (5.0 g, 10.7 mmol) and 3-(difluoromethoxy)aniline. <b>11j</b> was not pure after chromatography and was used as such in further steps (3.8 g, purity 77%; UV, 51%). LC-MS (ESI): <i>m</i>/<i>z</i> = 533.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Allyl [3-(4-Chloro-3,5-dimethylphenyl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]-carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11k</b>)</h5><div class="NLM_p last">Compound <b>11k</b> was prepared from <b>10c</b> (10.0 g, 22.1 mmol) and 3-(difluoromethoxy)aniline. <b>11k</b> was obtained as white solid (9.1 g, 80%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 2.32 (s, 6H), 4.3–4.6 (m, 4H), 5.13 (dd, <i>J</i> = 1.1, 10.3 Hz, 1H), 5.19 (dd, <i>J</i> = 1.4, 17.4 Hz, 1H), 5.5–5.6 (m, 1H), 5.7–5.9 (m, 1H), 6.53 (t, <i>J</i> = 73.8 Hz, 1H), 6.87 (br d, <i>J</i> = 8.7 Hz, 1H), 7.1–7.3 (m, 4H), 7.50 (s, 2H), 8.35 (s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 517.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Allyl [1-{<i>N</i>′-Cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11l</b>)</h5><div class="NLM_p last">Compound <b>11l</b> was prepared from <b>10d</b> (20.0 g, 43.6 mmol) and 3-(difluoromethoxy)aniline. <b>11l</b> was obtained as white solid (21.2 g, 93%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 4.08 (dd, <i>J</i> = 4.6, 11.9 Hz, 1H), 4.36–4.53 (m, 3H), 5.11 (dd, <i>J</i> = 1.4, 10.5 Hz, 1H), 5.17 (dd, <i>J</i> = 1.4, 17.4 Hz, 1H), 5.50–5.59 (m, 1H), 5.77–5.90 (m, 1H), 6.99 (dd, <i>J</i> = 2.3, 8.2 Hz, 1H), 7.16 (m, 1H), 7.21 (t, <i>J</i> = 73.7 Hz, 1H), 7.22 (m, 1H), 7.39 (t, <i>J</i> = 8.3 Hz, 1H), 7.73–7.81 (m, 2H), 8.15 (d, <i>J</i> = 1.4 Hz, 1H), 8.17 (d, <i>J</i> = 8.7 Hz, 1H), 9.79 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 523.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Allyl [3-(4-Chloro-3-methylphenyl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]-carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11m</b>)</h5><div class="NLM_p last">Compound <b>11m</b> was prepared from <b>10e</b> (10.0 g, 22.8 mmol) and 3-(difluoromethoxy)aniline. <b>11m</b> was obtained as white solid (9.6 g, 84%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 2.33–2.40 (m, 3H), 4.03–4.15 (m, 1H), 4.41–4.56 (m, 3H), 5.13–5.36 (m, 2H), 5.53–5.63 (m, 1H), 5.82–5.94 (m, 1H), 7.00–7.08 (m, 1H), 7.19–7.29 (m, 2H), 7.25 (t, <i>J</i> = 74 Hz, 1H, partially masked by other peaks), 7.40–7.49 (m, 1H), 7.55 (d, <i>J</i> = 8.4 Hz, 1H), 7.73 (dd, <i>J</i> = 1.7, 8.2 Hz, 1H), 7.84–7.90 (m, 1H), 8.2 (br d, <i>J</i> = 8.6 Hz, 1H), 9.75 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 503.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Allyl [3-(1,3-Benzodioxol-5-yl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]-carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11n</b>)</h5><div class="NLM_p last">Compound <b>11n</b> was prepared from <b>10f</b> (2.0 g, 4.6 mmol) and 3-(difluoromethoxy)aniline. <b>11n</b> was obtained as white solid (1.7 g, 74%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 4.09 (dd, <i>J</i> = 4.6, 11.9 Hz, 1H), 4.43 (t, <i>J</i> = 11.2 Hz, 1H), 4.50 (d, <i>J</i> = 4.1 Hz, 2H), 5.16 (dd, <i>J</i> = 1.4, 10.5 Hz, 1H), 5.22 (dd, <i>J</i> = 1.6, 17.2 Hz, 1H), 5.49–5.60 (m, 1H), 5.82–5.93 (m, 1H), 6.11 (s, 2H), 6.95–7.09 (m, 2H), 7.16–7.32 (m, 4H), 7.39 (m, 1H), 7.61 (d, <i>J</i> = 1.4 Hz, 1H), 8.21 (d, <i>J</i> = 8.7 Hz, 1H), 9.63 (s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 499.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Allyl [1-{<i>N</i>′-Cyano-<i>N</i>-[3-(trifluoromethoxy)phenyl]carbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11o</b>)</h5><div class="NLM_p last">Compound <b>11o</b> was prepared from <b>10d</b> (10 g, 21.8 mmol) and 3-(trifluoromethoxy)aniline. <b>11o</b> was obtained as beige solid (7.4 g, 63%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 4.13 (dd, <i>J</i> = 4.8, 11.9 Hz, 1H), 4.42–4.56 (m, 3H), 5.12–5.25 (m, 2H), 5.50–5.63 (m, 1H), 5.83–5.94 (m, 1H), 7.19 (br d, <i>J</i> = 8.1 Hz, 1H), 7.35–7.44 (m, 2H), 7.46–7.56 (m, 1H), 7.77–7.85 (m, 2H), 8.13–8.24 (m, 2H), 9.89 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 540.8 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Allyl {1-[<i>N</i>′-Cyano-<i>N</i>-(3-methoxyphenyl)carbamimidoyl]-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl}carbamate (<b>11p</b>)</h5><div class="NLM_p last">Compound <b>11p</b> was prepared from <b>10d</b> (2.0 g, 4.4 mmol) and 3-methoxyaniline. <b>11p</b> was obtained as white solid (1.5 g, 71%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 3.3 (s, 3H, masked by water signal), 3.97–4.07 (m, 1H), 4.33–4.48 (m, 1H), 5.40 (s, 1H), 5.74–5.82 (m, 1H), 6.95 (br d, <i>J</i> = 7.1 Hz, 1H), 7.14–7.26 (m, 3H), 7.34–7.43 (m, 1H), 7.73 (d, <i>J</i> = 8.5 Hz, 1H), 7.81 (dd, <i>J</i> = 1.9, 8.5 Hz, 1H), 8.08 (s, 1H), 8.76 (br d, <i>J</i> = 8.8 Hz, 1H), 9.51 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 487.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Allyl [1-{<i>N</i>′-Cyano-<i>N</i>-[6-(difluoromethoxy)pyridin-2-yl]carbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11q</b>)</h5><div class="NLM_p last">Compound <b>11q</b> was prepared from <b>10d</b> (5.0 g, 10.9 mmol) and 6-(difluoromethoxy)pyridin-2-amine. <b>11q</b> was obtained as off-white solid (2.9 g, 43%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 3.95 (dd, <i>J</i> = 5.0, 12.4 Hz, 1H), 4.32 (t, <i>J</i> = 11.7 Hz, 1H), 4.4–4.5 (m, 2H), 5.13 (m, 2H), 5.51 (m, 1H), 5.84 (m, 1H), 6.82 (d, <i>J</i> = 8.2 Hz, 1H), 7.13 (d, <i>J</i> = 7.8 Hz, 1H), 7.58 (t, <i>J</i> = 72.8 Hz, 1H), 7.7–7.8 (m, 2H), 7.91 (t, <i>J</i> = 7.8 Hz, 1H), 8.05 (d, 1H), 8.18 (d, <i>J</i> = 1.8 Hz, 1H), 10.45 (s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 524.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Allyl [1-{<i>N</i>-[4-Chloro-3-(difluoromethoxy)phenyl]-<i>N</i>′-cyanocarbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11r</b>)</h5><div class="NLM_p last">Compound <b>11r</b> was prepared from <b>10d</b> (2.0 g, 4.4 mmol) and 4-chloro-3-(difluoromethoxy)aniline. <b>11r</b> was obtained as white solid (1.5 g, 62%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 4.01–4.17 (m, 1H), 4.42–4.56 (m, 3H), 5.12–5.25 (m, 2H), 5.53–5.64 (m, 1H), 5.82–5.94 (m, 1H), 7.28 (t, <i>J</i> = 73 Hz, 1H, partially masked by other peaks), 7.34 (dd, <i>J</i> = 2.4, 8.7 Hz, 1H), 7.42 (m, 1H), 7.61 (d, <i>J</i> = 8.6 Hz, 1H), 7.81 (s, 2H), 8.13–8.25 (m, 2H), 9.91 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 557.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Allyl [1-{<i>N</i>′-Cyano-<i>N</i>-[3-(difluoromethoxy)-5-fluorophenyl]carbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]carbamate (<b>11s</b>)</h5><div class="NLM_p last">Compound <b>11s</b> was prepared from <b>10d</b> (1.6 g, 3.4 mmol) and 3-(difluoromethoxy)-5-fluoroaniline. <b>11s</b> was obtained as white solid (0.8 g, 45%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 4.13 (dd, <i>J</i> = 4.6, 11.9 Hz, 1H), 4.41–4.53 (m, 3H), 5.10–5.25 (m, 2H), 5.51–5.63 (m, 1H), 5.78–5.94 (m, 1H), 6.96 (dd, <i>J</i> = 2.2, 9.7 Hz, 1H), 7.10 (d, <i>J</i> = 5.1 Hz, 1H), 7.18 (br d, <i>J</i> = 10.9 Hz, 1H), 7.29 (t, <i>J</i> = 73 Hz, 1H, partially masked by other peaks), 7.74–7.85 (m, 2H), 8.10–8.30 (m, 2H), 9.88 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 541.2 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(1-{<i>N</i>′-Cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-3-phenyl-4,5-dihydro-1<i>H</i>-pyrazol-4-yl)-<i>N</i>-ethyl-2-hydroxyacetamide (<b>25</b>) and <i>N</i>-[3-(4-Bromophenyl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>26</b>)</h4><div class="NLM_p">To a stirred solution of <b>11j</b> (3.8 g, 77%-UV, 5.5 mmol) in degassed THF (150 mL) was added 1,3-dimethylbarbituric acid (1.8 g, 11.0 mmol), followed by tetrakis(triphenylphosphine)palladium(0) [Pd(PPh<sub>3</sub>)<sub>4</sub>; 0.25 g, 0.22 mmol]. The reaction mixture was stirred under argon for 12 h, then cautiously quenched with saturated NaHCO<sub>3</sub> solution (150 mL) and extracted into EtOAc (150 mL). The organic layer was washed with brine (100 mL), then dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude material was purified by dry flash column chromatography (eluent: EtOAc, MeOH) to yield 4-amino-3-(4-bromophenyl)-<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidamide (2.4 g, 95%; UV, 92%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 2.38 (br s, 2H), 4.02 (dd, <i>J</i> = 4.4, 11.5 Hz, 1H), 4.37 (dd, <i>J</i> = 9.9, 11.4 Hz, 1H), 4.81 (dd, <i>J</i> = 4.5, 9.9 Hz, 1H), 6.98 (dd, <i>J</i> = 2.0, 8.1 Hz, 1H), 7.22 (t, <i>J</i> = 74.0 Hz, 1H), 7.20–7.23 (m, 1H), 7.27 (ddd, <i>J</i> = 0.8, 2.0, 8.1 Hz, 1H), 7.40 (t, <i>J</i> = 8.1 Hz, 1H), 7.64–7.70 (m, 2H), 7.98–8.03 (m, 2H), 9.62 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 449.3 [M + H]<sup>+</sup>. To the amine product (2.30 g, 5.1 mmol) in MeOH (70 mL) at rt was added acetaldehyde (0.27 g, 6.1 mmol). The reaction mixture was stirred at rt for 1 h, and at 40 °C for 1 h, then it was cooled to 0 °C and NaBH<sub>4</sub> (0.22 g, 5.7 mmol) was added in small portions. The mixture was stirred at rt for 1 h, then poured over saturated NaHCO<sub>3</sub> solution (10 mL). The volatiles were removed by evaporation, and the resulting aqueous slurry was extracted with EtOAc (2 × 10 mL). The combined organic layers were washed with brine (100 mL), then dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to yield a crude black oil. The crude material was purified by dry flash column chromatography (eluent: DCM, MeOH) to yield 3-(4-bromophenyl)-<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]-4-(ethylamino)-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidamide (1.11 g, 45%) as the main N-ethylation product. LC-MS (ESI): <i>m</i>/<i>z</i> = 477.1 [M + H]<sup>+</sup>. The corresponding debrominated derivative <i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]-4-(ethylamino)-3-phenyl-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidamide (0.41 g, 19%) was obtained as a byproduct. LC-MS (ESI): <i>m</i>/<i>z</i> = 398.3 [M + H]<sup>+</sup>. To the debrominated byproduct (410 mg, 1 mmol) in DCM (15 mL) were added Et<sub>3</sub>N (0.15 mL, 1.1 mmol), acetoxyacetyl chloride (0.12 mL, 1.1 mmol), and DMAP (126 mg, 1 mmol), and the mixture was stirred at rt for 1 h. EtOAc (15 mL) was added, and the mixture was washed with saturated aqueous NH<sub>4</sub>Cl solution (20 mL) and brine (20 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The crude product was purified by dry flash column chromatography (eluent: DCM, MeOH) to yield 504 mg of an impure intermediate, which was dissolved in MeOH (12 mL), K<sub>2</sub>CO<sub>3</sub> (140 mg, 1.01 mmol) was added, and the mixture was heated to reflux for 30 min. After cooling, to the reaction mixture was added saturated aqueous NH<sub>4</sub>Cl, and the mixture was extracted with EtOAc. The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated, and the residue purified by flash column chromatography (eluent: DCM, MeOH) and subsequent preparative HPLC to yield <i>N</i>-(1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-3-phenyl-4,5-dihydro-1<i>H</i>-pyrazol-4-yl)-<i>N</i>-ethyl-2-hydroxyacetamide (<b>25</b>; 50 mg, 10%). <sup>1</sup>H NMR (300 MHz, [D]<sub>6</sub>DMSO): δ 1.03 (br s, 3H), 3.16–3.33 (m, 2H), 3.92–4.25 (m, 3H), 4.47 (br t, <i>J</i> = 11.7 Hz, 1H), 4.75 (br t, <i>J</i> = 5.5 Hz, 1H), 6.94–7.03 (m, 1H), 7.19–7.30 (m, 3H), 7.36–7.52 (m, 4H), 7.75–7.84 (m, 2H), 9.77 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 457.2 [M + H]<sup>+</sup>.</div><div class="NLM_p last">To the main product from the N-ethylation step (1.1 g, 2.3 mmol) in DCM (30 mL) were added Et<sub>3</sub>N (0.34 mL, 2.4 mmol), acetoxyacetyl chloride (0.27 mL, 2.4 mmol), and DMAP (282 mg, 2.3 mmol) and the mixture was stirred at rt for 1 h. EtOAc (30 mL) was added, and the mixture was washed with saturated aqueous NH<sub>4</sub>Cl solution (40 mL) and brine (40 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The crude product was purified by dry flash column chromatography (eluent: DCM, MeOH) to yield 1.2 g of an impure intermediate. Then 200 mg of this intermediate were dissolved in MeOH (5 mL), K<sub>2</sub>CO<sub>3</sub> (48 mg, 0.35 mmol) was added, and the mixture was heated to reflux for 30 min. After cooling, the reaction mixture was concentrated and the residue was purified by HPLC to yield (<i>N</i>-[3-(4-bromophenyl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>26</b>; 50 mg, 24%). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 1.05 (br s, 3H), 3.27–3.36 (m, 2H), 3.98–4.18 (m, 3H), 4.48 (t, <i>J</i> = 11.7 Hz, 1H), 4.72 (t, <i>J</i> = 4.9 Hz, 1H), 7.00 (d, <i>J</i> = 7.8 Hz, 1H), 7.21 (s, 1H), 7.23 (t, <i>J</i> = 74.0 Hz, 1H), 7.26 (d, <i>J</i> = 8.3 Hz, 1H), 7.41 (t, <i>J</i> = 8.2 Hz, 1H), 7.67 (d, <i>J</i> = 8.5 Hz, 2H), 7.73 (d, <i>J</i> = 8.3 Hz, 2H), 9.78 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 535.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-[3-(4-Chloro-3,5-dimethylphenyl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]-carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> was prepared as described for <b>4</b>, starting from <b>11k</b> (4.9 g, 9.6 mmol). <b>27</b> was obtained as gray solid (1.6 g, 32% over 3 steps). <sup>1</sup>H NMR (300 MHz, [D]<sub>6</sub>DMSO): δ 1.05 (br s, 3H), 2.32 (s, 6H), 3.23–3.44 (m, 2H), 3.94–4.15 (m, 3H), 4.44 (br t, <i>J</i> = 11.7 Hz, 1H), 4.69 (br s, 1H), 6.92–7.02 (m, 1H), 7.14–7.27 (m, 2H), 7.33–7.43 (m, 1H), 7.56 (s, 2H), 9.73 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 518 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-[1-{<i>N</i>′-Cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>4</b>)</h4><div class="NLM_p">To a stirred solution of <b>11l</b> (14.2 g, 27.0 mmol) in degassed THF (370 mL) was added 1,3-dimethylbarbituric acid (17.0 g, 108 mmol), followed by Pd(PPh<sub>3</sub>)<sub>4</sub> (2.40 g, 2.16 mmol). The reaction mixture was stirred under argon for 15 min, then cautiously quenched with saturated NaHCO<sub>3</sub> solution (400 mL) and extracted into EtOAc (400 mL). The organic layer was washed with brine (200 mL), then dried over MgSO<sub>4</sub>, filtered, and concentrated. The crude material was purified by dry flash column chromatography (eluent: EtOAc/heptane 1:1, 2:1; EtOAc; MeOH/EtOAc 1:100) to yield 4-amino-<i>N</i>′-cyano-3-(3,4-dichlorophenyl)-<i>N</i>-[3-(difluoromethoxy)phenyl]-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidamide (9.3 g, 78%) as an orange oil. <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 3.95–4.02 (m, 1H, under EtOAc signal), 4.35 (m, 1H), 4.83 (dd, <i>J</i> = 4.6, 9.6 Hz, 1H), 7.01 (dd, <i>J</i> = 2.1, 8.0 Hz, 1H), 7.22 (t, <i>J</i> = 2.1 Hz, 1H), 7.24 (t, <i>J</i> = 73.7 Hz, 1H), 7.27 (m, 1H), 7.38 (t, <i>J</i> = 7.8 Hz, 1H), 7.48–7.61 (m, 1H), 7.72 (d, <i>J</i> = 8.2 Hz, 1H), 8.00 (dd, <i>J</i> = 2.1, 8.5 Hz, 1H), 8.31 (d, <i>J</i> = 1.8 Hz, 1H), 9.67 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 439.1 [M + H]<sup>+</sup>. To a stirred solution of the amine product (9.30 g, 21.2 mmol) in MeOH (170 mL) at 0 °C was added acetaldehyde (1.12 g, 25.4 mmol), followed by the portionwise addition of NaBH<sub>4</sub> (0.96 g, 25.4 mmol) over 20 min. The mixture was stirred for 30 min, then poured over saturated NaHCO<sub>3</sub> solution (100 mL). The volatiles were removed by evaporation, and the resulting aqueous slurry was extracted with EtOAc (2 × 100 mL). The combined organic layers were washed with brine (100 mL), then dried over MgSO<sub>4</sub>, filtered, and concentrated to yield a crude black oil. The crude material was purified by dry flash column chromatography (eluent: EtOAc/heptane 1:1, EtOAc) to give a black oil, which was triturated with Et<sub>2</sub>O to yield <i>N</i>′-cyano-3-(3,4-dichlorophenyl)-<i>N</i>-[3-(difluoromethoxy)phenyl]-4-(ethylamino)-4,5-dihydro-1<i>H</i>-pyrazole-1-carboximidamide (7.40 g, 75%) as a gray solid. <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 0.98 (t, <i>J</i> = 6.6 Hz, 3H), 2.40–2.64 (m, 2H, partially under DMSO signal), 4.16–4.27 (m, 2H), 4.83 (m, 1H), 6.98 (dd, <i>J</i> = 2.3, 8.2 Hz, 1H), 7.21 (t, <i>J</i> = 2.1 Hz, 1H), 7.25 (t, <i>J</i> = 73.7 Hz, 1H), 7.27 (dt, <i>J</i> = 1.0, 8.1 Hz, 1H), 7.42 (t, <i>J</i> = 8.2 Hz, 1H), 7.76 (d, <i>J</i> = 8.7 Hz, 1H), 8.01 (dd, <i>J</i> = 2.1, 8.5 Hz, 1H), 8.33 (d, <i>J</i> = 1.8 Hz, 1H), 9.71 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 467.2 [M + H]<sup>+</sup>. To this product (7.24 g, 15.5 mmol) in DCM (62 mL) was added saturated aqueous NaHCO<sub>3</sub> solution (72 mL). The biphasic mixture was stirred vigorously and cooled to 5 °C; acetoxyacetyl chloride (2.50 mL, 23.2 mmol) in DCM (10 mL) was added dropwise over 15 min. The mixture was stirred at 5 °C for 10 min, after which time LC analysis indicated total consumption of the starting material and only one major peak. The volatiles were removed by evaporation to yield an oily aqueous suspension to which K<sub>2</sub>CO<sub>3</sub> (4.28 g, 31.0 mmol) was added, followed by MeOH (110 mL). The mixture was brought to reflux for 5 min then allowed to cool to rt, whereupon the product precipitated as a white solid. The solid was collected by filtration, washed with water (50 mL) and Et<sub>2</sub>O (50 mL), and dried in vacuo to yield <b>4</b> (7.58 g, 93%) as a white powder. <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 0.98–1.10 (m, 3H), 3.17–3.42 (m, 2H, partially under water signal), 3.94–4.17 (m, 3H), 4.46 (t, <i>J</i> = 11.7 Hz, 1H), 4.75 (t, <i>J</i> = 5.7 Hz, 1H), 6.98 (dd, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.18 (br t, 1H), 7.22 (m, 1H), 7.23 (t, <i>J</i> = 8.2 Hz, 1H), 7.40 (t, 1H), 7.62 (dd, <i>J</i> = 8.2, 1.8 Hz, 1H), 7.7 (d, <i>J</i> = 8.2 Hz, 1H), 8.11 (d, <i>J</i> = 1.4 Hz, 1H), 9.85 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 525.0 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compound <b>4</b> was separated into its enantiomers by chiral SFC (instrument, Sepiatec Prep SFC 100; column, Chiralpak ID 5 μm, 250 mm × 20 mm; eluent, CO<sub>2</sub>/<i>i</i>-PrOH 7:3; flow, 80 mL/min; temperature, 40 °C): <i>t</i><sub>R</sub> (min) = 3.35–4.40 [(<i>S</i>)-<b>4</b>], 7.31–9.00 [(<i>R</i>)-<b>4</b>]. Enantiomer (<i>S</i>)-<b>4</b>: SFC (Chiralpak ID 5 μm, 100 mm × 4.6 mm, CO<sub>2</sub>/<i>i</i>-PrOH 7:3; flow, 4 mL/min; temperature, 37.5 °C) <i>t</i><sub>R</sub> (min) = 2.41. [α]<sub>D</sub> −102 (<i>c</i> 0.44, MeOH). Enantiomer (<i>R</i>)-<b>4</b>: SFC (Chiralpak ID 5 μm, 100 mm × 4.6 mm, CO<sub>2</sub>/<i>i</i>-PrOH 7:3; flow, 4 mL/min; temperature, 37.5 °C) <i>t</i><sub>R</sub> (min) = 5.66. [α]<sub>D</sub> +96 (<i>c</i> 0.25, MeOH).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>N</i>-[3-(4-Chloro-3-methylphenyl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>28</b>)</h4><div class="NLM_p">Compound <b>28</b> was prepared as described for <b>4</b>, starting from <b>11m</b> (3.4 g, 6.6 mmol). <b>28</b> was obtained as yellow solid (537 mg, 16% over 3 steps) <sup>1</sup>H NMR (500 MHz, [D]<sub>2</sub>DCM): δ 1.11 (br t, <i>J</i> = 6.8 Hz, 3H), 2.43 (s, 3H), 3.11–3.34 (m, 3H), 4.11–4.27 (m, 2H), 4.43–4.54 (m, 2H), 6.65 (t, <i>J</i> = 74.1 Hz, 1H), 7.00–7.08 (m, 1H), 7.32 (d, <i>J</i> = 7.3 Hz, 2H), 7.40–7.46 (m, 2H), 7.49 (dd, <i>J</i> = 10.1, 8.2 Hz, 1H), 7.72 (d, <i>J</i> = 1.3 Hz, 1H), 8.27 (s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 505.0 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compound <b>28</b> was separated into its enantiomers by chiral HPLC (instrument, Agilent Prep 1200; column, Chiralpak ID 5 μm, 250 × 30 mm; eluent, hexane/EtOH 7:3; flow, 50 mL/min): <i>t</i><sub>R</sub> (min) = 7.2–9.0 [(<i>R</i>)-<b>28</b>], 10.9–12.9 [(<i>S</i>)-<b>28</b>]. Enantiomer (<i>S</i>)-<b>28</b>: HPLC (Chiralpak ID 3 μm, 100 mm × 4.6 mm, hexane/EtOH 7:3, 1.0 mL/min) <i>t</i><sub>R</sub> (min) = 6.22. [α]<sub>D</sub> −96.8 (<i>c</i> 0.88, MeOH).</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i>-[3-(1,3-Benzodioxol-5-yl)-1-{<i>N</i>′-cyano-<i>N</i>-[3-(difluoromethoxy)phenyl]carbamimidoyl}-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>29</b>)</h4><div class="NLM_p last">Compound <b>29</b> was prepared as described for <b>4</b>, starting from <b>11n</b> (870 mg, 2.0 mmol). <b>29</b> was obtained as off-white solid (432 mg, 43% over 3 steps) <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 0.99 (br s, 3H), 3.08–3.31 (m, 2H), 3.94–4.18 (m, 3H), 4.41 (br t, <i>J</i> = 11.2 Hz, 1H), 4.75 (br s, 1H), 6.07 (br d, <i>J</i> = 2.3 Hz, 2H), 6.96 (br d, <i>J</i> = 8.2 Hz, 2H), 7.11–7.27 (m, 4H), 7.35–7.43 (m, 1H), 7.48–7.63 (m, 2H), 9.68 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 501.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>-[1-{<i>N</i>′-Cyano-<i>N</i>-[3-(trifluoromethoxy)phenyl]carbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>30</b>)</h4><div class="NLM_p">Compound <b>30</b> was prepared as described for compound <b>4</b>, starting from <b>11o</b> (385 mg, 0.71 mmol). <b>30</b> was obtained as white solid (244 mg, 66% over 3 steps). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 1.03 (t, <i>J</i> = 6.6 Hz, 3H), 3.3–3.5 (m, 2H), 3.9–4.1 (m, 3H), 4.4–4.5 (m, 1H), 4.7–4.8 (m, 1H), 7.15 (d, 1H), 7.3–7.8 (m, 6H), 8.09 (s, 1H), 9.88 (s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 542.9 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compound <b>30</b> was separated into its enantiomers by chiral SFC (instrument, Sepiatec Prep SFC100; column, Chiralpak IE 5 μm 250 × 20 mm; eluent, CO<sub>2</sub>/2-propanol 67/33; flow, 80 mL/min; temperature, 40 °C): <i>t</i><sub>R</sub> (min) = 4.8–5.9 [(<i>R</i>)-<b>30</b>], 8.6–10.3 [(<i>S</i>)-<b>30</b>]. Enantiomer (<i>S</i>)-<b>30</b>: SFC (Chiralpak IE 5 μm 100 mm × 4.6 mm, CO<sub>2</sub>/2-propanol 67/33; flow, 4.0 mL/min; temperature, 37.5 °C) <i>t</i><sub>R</sub> (min) = 5.24. [α]<sub>D</sub> −79 (<i>c</i> 0.83, MeOH).</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-{1-[<i>N</i>′-Cyano-<i>N</i>-(3-methoxyphenyl)carbamimidoyl]-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl}-<i>N</i>-ethyl-2-hydroxyacetamide (<b>31</b>)</h4><div class="NLM_p">Compound <b>31</b> was prepared as described for <b>4</b>, starting from <b>11p</b> (1.5 g, 3.1 mmol). <b>31</b> was obtained as yellow solid (325 mg, 21% over 3 steps). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 1.04 (br s, 3H), 3.14–3.27 (m, 1H), 3.74 (s, 3H), 3.99–4.14 (m, 3H), 4.38 (br t, <i>J</i> = 11.9 Hz, 1H), 4.72 (br t, <i>J</i> = 5.6 Hz, 1H), 5.85 (br s, 1H), 6.84 (m, 3H), 7.20 (br t, <i>J</i> = 8.5 Hz, 1H), 7.51–7.64 (m, 2H), 7.69–7.73 (m, 1H), 8.07 (br s, 1H), 9.75 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 489.2 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compound <b>31</b> was separated into its enantiomers by chiral HPLC (instrument, Agilent Prep 1200; column, Chiralpak IC 5 μm, 250 mm × 20 mm; eluent, MeOH/EtOH/Et<sub>2</sub>NH 50:50:0.1; flow, 20 mL/min; temperature, 23 °C): <i>t</i><sub>R</sub> (min) = 5.0–6.0 [(<i>S</i>)-<b>31</b>], 6.75–8.0 [(<i>R</i>)-<b>31</b>]. Enantiomer (<i>S</i>)-<b>31</b>: HPLC (Chiralpak IC 5 μm, 150 mm × 4.6 mm, MeOH/EtOH/Et<sub>2</sub>NH 50:50:0.1, 1.0 mL/min) <i>t</i><sub>R</sub> (min) = 2.65. [α]<sub>D</sub> −88.1 (<i>c</i> 1.0, MeOH).</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-[1-{<i>N</i>′-Cyano-<i>N</i>-[6-(difluoromethoxy)pyridin-2-yl]carbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>32</b>)</h4><div class="NLM_p">Compound <b>32</b> was prepared as described for <b>4</b>, starting from <b>11q</b> (1.9 g, 3.6 mmol). <b>32</b> was obtained as pale-yellow solid (558 mg, 29% over 3 steps). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 1.08 (br t, <i>J</i> = 7.1 Hz, 3H), 3.32–3.50 (m, 2H), 3.93–4.17 (m, 3H), 4.41 (br t, <i>J</i> = 11.7 Hz, 1H), 4.76 (br s, 1H), 6.84 (d, <i>J</i> = 8.2 Hz, 1H), 7.16 (br d, <i>J</i> = 8.2 Hz, 1H), 7.26 (t, <i>J</i> = 65.9 Hz, 1H), 7.63 (s, 1H), 7.75 (br d, <i>J</i> = 8.2 Hz, 1H), 7.94 (t, <i>J</i> = 8.0 Hz, 1H), 8.04 (s, 1H), 10.52 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 526.1 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compound <b>32</b> was separated into its enantiomers by chiral SFC (instrument, Sepiatec Prep SFC 100; column, Chiralpak ID 5 μm, 250 mm × 20 mm; eluent, CO<sub>2</sub>/<i>i</i>-PrOH 71:29; flow, 80 mL/min; temperature, 40 °C): <i>t</i><sub>R</sub> (min) = 3.5–5.0 [(<i>R</i>)-<b>32</b>], 5.0–7.0 [(<i>S</i>)-<b>32</b>]. Enantiomer (<i>S</i>)-<b>32</b>: SFC (Chiralpak ID 5 μm, 100 mm × 4.6 mm, CO<sub>2</sub>/<i>i</i>-PrOH 71:29, 4.0 mL/min) <i>t</i><sub>R</sub> (min) = 3.82. [α]<sub>D</sub> −56.2 (<i>c</i> 0.53, MeOH).</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-[1-{<i>N</i>-[4-Chloro-3-(difluoromethoxy)phenyl]-<i>N</i>′-cyanocarbamimidoyl}-3-(3,4-dichlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>33</b>)</h4><div class="NLM_p">Compound <b>33</b> was prepared as described for <b>4</b>, starting from <b>11r</b> (1.5 g, 2.7 mmol). <b>33</b> was obtained as yellow solid (18 mg, 12% over 3 steps). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 0.81 (br s, 1H), 0.99 (br s, 3H), 2.92 (br s, 0.4H), 3.01–3.29 (m, 3H), 3.94–4.33 (m, 6H), 4.40 (br s, 0.4H), 4.72 (t, <i>J</i> = 5.3 Hz, 1H), 5.13 (br s, 0.3H), 5.73 (br s, 0.4H), 5.94 (br s, 1H), 6.93–7.42 (m, 6H), 7.52–7.72 (m, 3H), 7.86–7.96 (m, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 559.2 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compound <b>33</b> was separated into its enantiomers by chiral SFC (instrument, Sepiatec Prep SFC 100; column, Chiralpak IC 5 μm, 250 mm × 20 mm; eluent, CO<sub>2</sub>/EtOH 7:3; flow, 80 mL/min; temperature, 40 °C): <i>t</i><sub>R</sub> (min) = 4.75–5.6 [(<i>S</i>)-<b>33</b>], 6.25–7.5 [(<i>R</i>)-<b>33</b>]. Enantiomer (<i>S</i>)-<b>33</b>: SFC (Chiralpak IC 5 μm, 100 mm × 4.6 mm, CO<sub>2</sub>/EtOH 7:3, 4.0 mL/min) <i>t</i><sub>R</sub> (min) = 2.99. [α]<sub>D</sub> −37.4 (<i>c</i> 1.0, DMSO).</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-[1-{<i>N</i>′-Cyano-<i>N</i>-[3-(difluoromethoxy)-5-fluorophenyl]carbamimidoyl}-3-(3,4-dichloro-phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-4-yl]-<i>N</i>-ethyl-2-hydroxyacetamide (<b>34</b>)</h4><div class="NLM_p">Compound <b>34</b> was prepared as described for <b>4</b>, starting from <b>11s</b> (415 mg, 0.8 mmol). <b>34</b> was obtained as white solid (65 mg, 15% over 3 steps). <sup>1</sup>H NMR (400 MHz, [D]<sub>6</sub>DMSO): δ 1.08 (br t, 3H), 3.39 (m, 1H), 4.00–4.20 (m, 3H), 4.50 (br t, <i>J</i> = 11.5 Hz, 1H), 4.77 (br t, <i>J</i> = 5.8 Hz, 1H), 5.59–6.16 (m, 1H), 6.95 (br d, <i>J</i> = 8.1 Hz, 1H), 7.10 (br s, 1H), 7.19 (br d, <i>J</i> = 10.4 Hz, 1H), 7.31 (t, <i>J</i> = 73.5 Hz, 1H), 7.66 (br d, <i>J</i> = 8.4 Hz, 1H), 7.77 (d, <i>J</i> = 8.4 Hz, 1H), 8.11 (s, 1H), 9.94 (br s, 1H). LC-MS (ESI): <i>m</i>/<i>z</i> = 543.2 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compound <b>34</b> was separated into its enantiomers by chiral HPLC (instrument, Agilent Prep 1200; column, Chiralpak ID 5 μm, 250 mm × 30 mm; eluent, hexane/EtOH/Et<sub>2</sub>NH 70:30:0.1; flow, 40 mL/min; temperature, 23 °C): <i>t</i><sub>R</sub> (min) = 6.5–7.7 [(<i>S</i>)-<b>34</b>], 8.1–9.7 [(<i>R</i>)-<b>34</b>]. Enantiomer (<i>S</i>)-<b>34</b>: HPLC (Chiralpak ID, 3 μm, 100 mm × 4.6 mm (hexane/EtOH/Et<sub>2</sub>NH 70:30:0.1, 1.0 mL/min) <i>t</i><sub>R</sub> (min) = 3.81. [α]<sub>D</sub> −38.1 (<i>c</i> 1.0, DMSO).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Biology</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Cell Culture</h4><div class="NLM_p last">Cell lines were obtained from the American Type Culture Collection (HEK293, U2OS), the German Collection of Microorganisms and Cell Cultures (KYSE-150), and the European Collection of Cell Cultures (A2780). They were maintained in the recommended cell culture media at 37 °C in 5% CO<sub>2</sub>.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Enzyme Activity and Inhibition Assays</h4><div class="NLM_p">SMYD2 enzyme kinetics and inhibitory activities of compounds were analyzed using a scintillation proximity assay (SPA) which measured methylation by the enzyme of the synthetic, biotinylated peptide Btn-Ahx-GSRAHSSHLKSKKGQSTSRH-amide (Biosyntan) derived from the C-terminal domain of p53. The SMYD2 full-length enzyme with an N-terminal 6xHis tag was expressed in <i>Escherichia coli</i> and purified by affinity chromatography on a Ni-NTA Sepharose column, followed by size-exclusion chromatography on a Superdex 200 16/60 column (GE Healthcare). Assays were conducted in 384-well microtiter plates in a buffer containing 50 mM Tris/HCl pH 9.0, 1 mM DTT, 0.01% (w/v) BSA, and 0.0022% (v/v) Pluronic, and a final volume of 5 μL. The SMYD2 concentration in the assay was 3 nM, while tritiated <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (<sup>3</sup>H-SAM) and the peptide substrate were present at 60 nM and 1 μM, respectively, to ensure “balanced” conditions.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Apparent Michaelis–Menten constants for SAM and the p53 peptide were determined by titrating one substrate to saturation at cosubstrate concentrations of 1 × <i>K</i><sub>m(app)</sub>. Enzyme kinetics were followed over 2 h by quenching the reactions as described above at time points 0, 5, 10, 15, 30, 60, 90, and 120 min. Compounds were tested in 11-point, 3.5-fold dilution series ranging from 0.1 nM to 20 μM. Reactions were run for 30 min and quenched by adding Streptavidin PS SPA imaging beads (PerkinElmer) to a concentration of 3.12 μg/μL and 25 μM “cold” SAM. The amount of product was evaluated using a Viewlux (PerkinElmer) CCD plate imaging device [emission filter 613/55]. The data were normalized using two sets of control wells for high (= enzyme reaction with DMSO instead of test compound = 0% = minimum inhibition) and low (= all assay components without enzyme = 100% = maximum inhibition) SMYD2 activity. IC<sub>50</sub> values were calculated by fitting the normalized inhibition data to a four-parameter logistic equation using either a Bayer proprietary tool or Genedata Screener analysis software.</div><div class="NLM_p">For mechanism-of-inhibition studies, IC<sub>50</sub> determinations were basically performed as described above, but incubation times were adjusted to guarantee kinetic linearity at each concentration of substrate analyzed. For peptide competition studies, IC<sub>50</sub> values were determined at six different concentrations of p53 peptide (0.5, 1, 2, 4, 6, and 12 μM). For SAM competition, 25, 50, 100, 200, 300, and 600 nM <sup>3</sup>H-SAM were used. In both cases, the other cosubstrate was used at a concentration of 1 × <i>K</i><sub>m(app)</sub>.</div><div class="NLM_p last">Calculation of <i>K</i><sub>i(app)</sub> and α<i>K</i><sub>i(app)</sub> using the corresponding Cheng–Prusoff equations:<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a><span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/jm-2015-01890m_m001.gif" alt="" /></img></span><span class="NLM_disp-formula" id="ueq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/jm-2015-01890m_m002.gif" alt="" /></img></span>where [S] = substrate concentration, <i>K</i><sub>m(app)</sub> = apparent Michaelis–Menten constant, <i>K</i><sub>i(app)</sub> = apparent inhibitor constant for binding to free enzyme, and α<i>K</i><sub>i(app)</sub> = apparent inhibitor constant for binding to enzyme–substrate complex.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Selectivity Assays</h4><div class="NLM_p last">The effect of (<i>S</i>)-<b>4</b> on the methyltransferase activity of G9A, EHMT1, SUV39H1, SUV39H2, SETDB1, SETD7, SETD8, SUV420H1, SUV420H2, PRMT1, PRMT3, PRMT6, PRMT8, PRDM9, SETD2, and SMYD3, and MLL1, MLL3, EZH2, and PRMT5 complexes, as well as DNMT1 and BCDIN3D, was assessed using the SPA method.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> For DNMT3<i>A</i>/3L, DNMT3<i>B</i>/3L, ASH1L, DOT1L, NSD1, NSD2, NSD3, METTL21A, and METTL21D, a trichloroacetic acid (TCA) precipitation method in filter plates were employed.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> PAR1 Assay</h4><div class="NLM_p last">The identification of PAR1 antagonistic effects and the quantification of those effects, as well as the resulting IC<sub>50</sub> values, were performed as described previously.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Biophysical Validation by ITC/TSA</h4><div class="NLM_p">Thermal shift assays (TSA) were carried out with the ThermoFluor system (Johnson & Johnson Pharmaceutical Research and Development). Melting curves were obtained at a protein concentration of 1.5 μM, 8xSypro Orange (Invitrogen), and a ligand concentration of 100 μM, using buffer containing 20 mM Tris pH 8.0, 100 mM NaCl, 8.5%. Scans were measured from 25 to 95 °C at a scanning rate of 2 °C/min. TSA data were analyzed using ThermoFluor++ (version 1.3.7) software.</div><div class="NLM_p last">Isothermal titration calorimetry (ITC) measurements were carried out using a MicroCal iTC200 system. In the experiments, compound (400 μM) was titrated into SMYD2 protein (67 μM) dissolved in 25 mM Tris pH 8.5, 50 mM NaCl, 0.0022% Pluronic F-127, and 2% DMSO at 25 °C. After an initial injection of 0.2 μL, which was not considered for data fitting, 10 injections of 4 μL each in 150 s intervals were performed. The stirring speed of the protein in the cell was 500 rpm. The protein concentration was determined from UV absorbance measurements at 280 nm. Data were evaluated using ORIGIN software, as supplied with the instrument.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Purification and Crystallization of SMYD2</h4><div class="NLM_p last">Recombinant human SMYD2 (UniProt Q9NRG4, amino acids 2–433) was expressed in insect cells (Sf9) containing an N-terminal TEV-cleavable 6xHis tag. Cell pellets were resuspended in lysis buffer (40 mM Tris pH 8, 500 mM NaCl, 0.1% IGEPAL, 5 mM imidazole, 1 mM DTT) supplemented with complete EDTA-free protease inhibitor tablets and 50 U/mL benzonase. The cell lysate was loaded onto a Ni-NTA column, eluted with imidazole, and concentrated using an ultracentrifugal filter unit. Then, SMYD2 was gel-filtrated on a Superdex S200 column equilibrated in 20 mM Tris pH 8, 100 mM NaCl, 5% glycerol, and 1 mM DTT. The 6xHis tag was cleaved with TEV protease in solution at 6 °C overnight. Uncleaved SMYD2 and TEV protease were separated from the cleaved product by a second Ni-NTA affinity step. The cleaved SMYD2 protein was further purified by a second gel-filtration step using a Superdex 200 column equilibrated in 20 mM Tris pH 8, 150 mM NaCl, 5% glycerol, and 1 mM Tris(2-carboxyethyl)phosphine (TCEP). The protein was concentrated to 15.5 mg/mL (313 μM) (UV–vis) using an ultracentrifugal filter unit and shock-frozen in liquid nitrogen. For crystallization, the cofactor SAM was added to a final concentration of 3.8 mM as follows: SAM stock solution (100 mM in DMSO, 1.2 μL) was added to concentrated SMYD2 solution (30 μL) and incubated at 4 °C for 2 h. Crystals grew at 20 °C within 3 d using the hanging drop method. Drops were made from the SMYD2/SAM solution (1 μL) and reservoir solution [20–24% (w/v) PEG 3350, 100 mM HEPES pH 7.0; 0.8 μL]. Thirty minutes after drop setup, a seed solution (0.2 μL) was added. The seed solution was made from SMYD2:SAM crystals (obtained with the same reservoir conditions in a previous experiment), which were crashed manually (using Seed Beads, Hampton Research), diluted in reservoir solution, shock-frozen, and stored at −80 °C. For complex formation, a crystal was transferred into a new drop reservoir solution (1.5 μL). A stock solution of the respective inhibitor (100 mM in DMSO) was 10-fold diluted with reservoir solution. Over the course of 2 h, this diluted stock solution (1.5 μL) was added in three steps of 0.5 μL to the drop containing the SMYD2:SAM crystal, resulting in a final concentration of 5 mM inhibitor in the soaking drop. The crystal was soaked in this drop at 20 °C for 4 days in the case of (<i>S</i>)-<b>4</b> and for 1 h for compound <b>6</b>.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Crystallographic Data Collection, Structure Determination, and Refinement</h4><div class="NLM_p last">The soaked crystal was briefly immersed in cryo buffer (0.1 M HEPES pH 7.0, 22% PEG 3350, 20% glycerol, 2 mM inhibitor) and shock-frozen in liquid nitrogen. Diffraction data sets were collected using beamline 14.1 at the Helmholtz-Zentrum Berlin at 100 K using a wavelength of 0.91841 Å and a PILATUS detector. The diffraction images were processed using the program XDS.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The crystals belonged to space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> with one molecule per asymmetric unit. The crystal form described here was first solved for an SMYD2:SAM crystal in the absence of an inhibitor, using the Molecular Replacement method (program Phaser<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> from the CCP4 program suite<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> and PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TG5">3TG5</a> as search model). The data sets described here were then solved by rigid body refinement using the SMYD2:SAM structure as starting model and the program Refmac<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> from the CCP4 program suite. 3D models for compound <b>6</b> and (<i>S</i>)-<b>4</b> were generated using the program Discovery Studio, and parameter files for crystallographic refinement and model building were generated using the software PRODRG.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> (<i>S</i>)-<b>4</b> was manually built into the electron-density maps using Coot,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> followed by several cycles of refinement with Refmac and rebuilding in Coot. For the data collection and refinement statistics, see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Table S3</a>.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> WaterMap Calculations</h4><div class="NLM_p">WaterMap calculations were based on the crystal structure of the 4-chlorophenyl derivative <b>6</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARF">5ARF</a>). The SMYD2–compound <b>6</b> complex structure was prepared using the Protein Preparation Wizard functionality in Maestro.<a onclick="showRef(event, 'ref54 ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56 ref57">(54-57)</a> Preparation involved assignment of bond orders, addition of hydrogens, creation of zero-order bonds to metals, deletion of water molecules beyond 5 Å from heteroatoms, assignment of protonation states according to pH 7.0, and optimization of the hydrogen-bonding network and restrained minimization. Here, and for all further minimizations and simulations, the OPLS 2005 force field<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58, 59)</a> was used.</div><div class="NLM_p last">Water sites and corresponding free energies were calculated using WaterMap.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> The calculation involved the following (default) settings: water molecules within 10 Å of the ligand were investigated, the ligand as well as any crystallographic water molecules were removed prior to simulation, the protein was truncated beyond 20 Å from the ligand, and the simulation time was 2 ns. In brief (see ref <a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">40</a> for details), the different stages of the calculation were solvation of the system in an orthorhombic box of TIP4P water<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> with a minimum distance between box edge and solute of 10 Å and a series of minimizations and short simulations to equilibrate the system, followed by a 2 ns production simulation. All nonwater heavy atoms were harmonically restrained during all minimizations and simulations using a force constant of 5 kcal mol<sup>–1</sup> Å<sup>–2</sup>. Coordinates were saved every 1.5 ps, yielding a total of 1334 snapshots for further analysis. Water molecules were subsequently clustered such that nonoverlapping spheres (i.e., hydration sites) with radius 1 Å were obtained. Thermodynamic properties of these sites (i.e., enthalpies, entropies, and thus free energies) were approximated as follows: Enthalpies were estimated as the difference between the average interaction energy of the water molecule with the rest of the system and the average interaction energy in bulk water. Entropies were estimated using inhomogeneous solvation theory.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Antibody Generation</h4><div class="NLM_p last">To detect SMYD2-mediated methylation, we used a lysine monomethylation specific rabbit polyclonal antibody (SY46). The antibody was generated (Eurogentec) against a p53 peptide containing the monomethylated K370 epitope, as described elsewhere.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Antibody has been purified against unmethylated p53 peptide. Using this antibody in cellular systems revealed that it also recognized methylated AHNAK, which is also methylated by SMYD2 (see Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">Figure S2</a>).</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> In Vitro Methylation of Recombinant p53</h4><div class="NLM_p last">Recombinant p53 protein (BD Pharmingen no. 556439) was incubated with full-length SMYD2 protein (in-house production) and SAM in reaction buffer (50 mM Tris pH 9, 1 mM DTT, 0.01% BSA, 0.0022% Pluronic) at 30 °C overnight.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Western Blot</h4><div class="NLM_p last">Cells were lysed in RIPA buffer Pierce (Thermo Fisher Scientific) with benzonase (Merck) and protease inhibitors (Roche Diagnostics). Proteins were separated by SDS-PAGE and blotted onto nitrocellulose membranes (Invitrogen). Membranes were blocked in 5% milk PBS-T (phosphate buffered saline with 0.1% Tween 20) and immunoprobed with antibodies raised against histone H3 (Cell Signaling Technology no. 4499S), p53 (BD Biosciences no. 554294), AHNAK (Santa Cruz Biotechnology, no. sc-390743, 1:500), SMYD2 (abcam, no. ab108217, 1:1000), beta-actin (abcam, no. ab8224, 1:1000), or with the methylation specific antibody SY46 (Eurogentec). The secondary antibodies used were goat antimouse/rabbit IRDye 800 CW (LI-COR Biosciences) and Alexa Fluor 680 goat antimouse/rabbit IgG. Bands were detected and quantified with Odyssey Fc Software (LI-COR Biosciences). Western blot assay for p53 methylation in HEK293T cells was performed as described elsewhere with an antibody directed against p53K370me1 kindly provided by Dr. Shelley Berger.<a onclick="showRef(event, 'ref13 ref28'); return false;" href="javascript:void(0);" class="ref ref13 ref28">(13, 28)</a></div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Caspase 3/7 Activation Assay</h4><div class="NLM_p last">Effects on apoptosis induction were measured with the Caspase-Glo 3/7 Assay System (Promega). In brief, 10000 cells/well were treated for 2 d with compound <b>25</b> or (<i>S</i>)-<b>4</b> (to allow for demethylation of p53. Then, caspase activation was induced by doxorubicin (Sigma) (1.25 μM for U2OS and A2780 cells, 2.5 μM for KYSE-150 cells) treatment for 24 h. The apoptotic index was determined by normalization of caspase 3/7 activation signals to proliferation effects, measured by alamarBlue (Thermo Fisher Scientific).</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Cell-Based Assay for the Detection of SMYD2Methylation Activity</h4><div class="NLM_p last">For the detection of SMYD2 cellular methylation activity, an In-Cell Western (ICW) assay was established. For the ICW, KYSE-150 cells stably transfected with a construct expressing wild-type N-terminal 2xc-myc-tagged SMYD2 (NCBI reference sequence: NP_064582.2) were used. For further detection and validation experiments of AHNAK methylation, we additionally generated stable HeLa and MDA-MB-231 cell lines using the same construct. For conducting the ICW assay, 5000 SMYD2-engineered KYSE-150 cells/well were seeded in 96-well plates (Sigma) and cultivated for 24 h at 37 °C in 5% CO<sub>2</sub>. Nontransfected KYSE-150 cells were used as a control for maximal inhibition of methylation activity. Cells were grown in 49% RPMI 1640 and 49% Ham’s F12 media supplemented with 2% heat-inactivated FCS. For the determination of SMYD2 inhibitory activity, cells were treated for 72 h in the presence of test compound (at a final concentration range of 3.9 × 10<sup>–8</sup> to 5 × 10<sup>–6</sup> M) or with DMSO. Media were removed, and 3.7% (w/v) formaldehyde in PBS was added for 20 min. After two washes with PBS, 0.25% (v/v) Triton X-100 in PBS was added for 15 min of permeabilization. After one wash with PBS, cells were blocked in 5% (w/v) nonfat dry milk in PBS for 1 h. Fixed cells were exposed to primary methylation antibody (SY46, 1:200) in 5% nonfat dry milk in PBS for 24 h. One row of cells on each plate was not exposed to methylation antibody (SY46) and was reserved for background control measurements. The wells were washed three times with PBS, then secondary IR800-conjugated antibody (LI-COR, no. 926–32211, 1:1000) and DNA-intercalating dye DRAQ5 (Thermo Fisher Scientific, no. 62251, 1:1000) were added for 3 h in blocking buffer. After five washes with PBS, the fluorescence in each well was measured on an Odyssey scanner (LI-COR) at 800 nm (SY46 methylation signal, 764 nm excitation) and 700 nm (DRAQ5 signal, 683 nm excitation). Fluorescence intensity was quantified and normalized to background and DRAQ5 signals. The normalized data were further analyzed by four-parameter logistic regression analysis using a Bayer proprietary tool to determine the IC<sub>50</sub> value for each tested compound. For IC<sub>50</sub> determinations, <i>C</i><sub>0</sub> (= no inhibition) was defined as the signal measured for the DMSO-treated controls. <i>C</i><sub>i</sub> (maximal inhibition) was defined as the signal measured for nontransfected KYSE-150 cells.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Proliferation Panel</h4><div class="NLM_p last">For the characterization of proliferation effects, the OncoPanel 240 (Eurofins Panlabs) was used.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Mouse Tumor Xenograft Model</h4><div class="NLM_p last">Animal experiments were conducted in accordance with the German animal welfare laws, approved by local authorities and in accordance with the ethical guidelines of Bayer AG. Eight-week-old female BALB/c nude mice obtained from Charles River Laboratories (Germany) were acclimated for at least for 24 h before tumor cell injection. KYSE-150 cells (4 × 10<sup>6</sup>) were resuspended in 100% Matrigel (100 μL) and injected subcutaneously into the right flank region of the mice; 4 days after tumor cell inoculation, mice were randomized into four treatment groups. Treatment was started at day 4 after tumor inoculation. Group 1 (<i>n</i> = 12) was treated with vehicle only (Solutol/ethanol/water 1:1:8) iv qd and once at day 4 with saline iv; group 2 (<i>n</i> = 6) was treated with 500 mg/kg (<i>S</i>)-<b>4</b> po qd; group 3 (<i>n</i> = 12) was treated with 10 mg/kg doxorubicin iv once at day 4; group 4 (<i>n</i> = 6) was treated with a combination of (<i>S</i>)-<b>4</b> and doxorubicin. Tumors were measured three times per week for 14 days. Tumor weight is assumed to be log normally distributed, thus all tumor weights were logarithmically transformed (base = 2) prior to statistical inference. One-sided Dunnett’s comparison of log 2 tumor weights of all treatment groups with the vehicle group were carried out on an overall significance level of 0.05. The results were transformed from the log 2 scale to the original scale so that the difference from the pairwise comparisons and the respective 95% confidence intervals calculated on the log 2 scale correspond to T/C ratios of geometric mean values plus respective 95% confidence intervals on the original tumor weight scale (mg). Data was analyzed with R 3.0.1.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Ex Vivo Methylation Detection by Dot Blot</h4><div class="NLM_p last">Eight-week-old female BALB/c nude mice obtained from Charles River Laboratories (Germany) were acclimated for at least for 24 h before tumor cell injection. SMYD2-engineered KYSE-150 cells (4 × 10<sup>6</sup>) were resuspended in 100% Matrigel (100 μL) and injected subcutaneously into the right flank region of the mice. Treatment was started when tumors reached a tumor area of 60–70 mm<sup>2</sup> (day 7 after inoculation). Mice (<i>n</i> = 12 per group) were treated with 10, 30, 70, or 100 mg/kg (<i>S</i>)-<b>4</b> po qd for 3 days, or vehicle (PEG 400/water 8:2) po qd for 3 days. After the treatment period, tumor samples were immediately frozen in liquid nitrogen and stored at −80 °C. Frozen tumors were mechanically homogenized using TissueLyser and stainless steel beads (Qiagen), and proteins were extracted as described for the Western blot method. Whole protein lysate (50 μg per sample) was transferred with the Dot-Blot system MiniFold-1 (Whatman) onto a nitrocellulose membrane (Invitrogen). Membrane were blocked in 5% milk PBS-T and immunoprobed with the SY46 methylation antibody. The secondary antibody used was goat antirabbit IRDye 800 CW (LI-COR, no. 926–32211, 1:1000). Bands were detected and quantified with Odyssey Fc software (LI-COR Biosciences).</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> In Vivo Rat PK</h4><div class="NLM_p last">Pharmacokinetic properties of (<i>S</i>)-<b>4</b> were determined by administering the test compound as indicated. Blood samples were withdrawn at different time points, and plasma was separated by centrifugation. The samples were analyzed by LC-MS/MS.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Metabolic Stability in Rat Hepatocytes</h4><div class="NLM_p last">Liver cells were distributed in Williams’ medium E containing 5% fetal calf serum (FCS) to glass vials at a density of 1.0 × 10<sup>6</sup> vital cells/mL. The test compound was added to a final concentration of 1 μM. During incubation, the hepatocyte suspensions were continuously shaken and aliquots were taken at 2, 8, 16, 30, 45, and 90 min, to which an equal volume of cold MeOH was immediately added. Samples were frozen at −20 °C overnight, then centrifuged (15 min, 3000 rpm), and the supernatant was analyzed with an Agilent 1200 HPLC system with LC-MS/MS detection. The half-life of a test compound was determined from the concentration–time plot. The intrinsic clearances were calculated from the half-life, together with additional parameters (liver blood flow, quantity of liver cells in vivo and in vitro). The hepatic in vivo blood clearance (CL) and the maximal oral bioavailability (<i>F</i><sub>max</sub>) were calculated using the following parameters: liver blood flow, 4.2 L/h/kg rat; specific liver weight, 32 g/kg rat body weight; liver cells in vivo, 1.1 × 10<sup>8</sup> cells/g liver; liver cells in vitro, 0.5 × 10<sup>6</sup>/mL.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Caco2 Drug Permeability Assay</h4><div class="NLM_p last">Caco-2 cells (purchased from DSMZ Braunschweig, Germany) were seeded at a density of 4.5 × 10<sup>4</sup> cell per well on 24-well insert plates, 0.4 μm pore size, and grown for 15 days in DMEM medium supplemented with 10% fetal bovine serum, 1% GlutaMAX (100×, GIBCO), 100 μg/mL penicillin, 100 μg/mL streptomycin (GIBCO), and 1% nonessential amino acids (100×). Cells were maintained at 37 °C in a humified 5% CO<sub>2</sub> atmosphere. Medium was changed every second to third day. Before running the permeation assay, the culture medium was replaced by an FCS-free Hepes-carbonate transport buffer (pH 7.2). For assessment of monolayer integrity, the transepithelial electrical resistance (TEER) was measured. Test compounds were dissolved in DMSO and added either to the apical or basolateral compartment in a final concentration of 2 μM. Before and after incubation at 37 °C, samples were taken from both compartments. Analysis of compound content was performed following precipitation with methanol and LC/MS/MS analysis. Permeability (<i>P</i><sub>app</sub>) was calculated in the apical to basolateral (A → B) and basolateral to apical (B → A) directions. The apparent permeability was calculated using following equation:<span class="NLM_disp-formula" id="ueq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/jm-2015-01890m_m003.gif" alt="" /></img></span>where <i>V</i><sub>r</sub> = volume of medium in the receiver chamber, <i>P</i><sub>0</sub> = measured peak area of the test drug in the donor chamber at <i>t</i> = 0, <i>S</i> = surface area of the monolayer, <i>P</i><sub>2</sub> = measured peak area of the test drug in the acceptor chamber after incubation for 2 h, and <i>t</i> = incubation time. The efflux ratio basolateral (B) to apical (A) was calculated by dividing <i>P</i><sub>app</sub> (B–A) by <i>P</i><sub>app</sub> (A–B). In addition, the compound recovery was calculated. Reference compounds were analyzed in parallel as assay controls. All samples were analyzed by LC/MS/MS.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01890">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43443" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43443" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01890" class="ext-link">10.1021/acs.jmedchem.5b01890</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figure and tables of PK and exposure data, selectivity data, and crystallographic data collection and refinement statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf">jm5b01890_si_001.pdf (1.28 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_002.csv">jm5b01890_si_002.csv (2.46 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factors for the described crystal structures have been deposited with the Protein Data Bank (PDB). The PDB accession codes are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARF">5ARF</a> (compound <b>6</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARG">5ARG</a> (<i>S</i>)-<b>4</b>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.5b01890" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57256" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57256" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timo Stellfeld</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#deaab7b3b1f0adaabbb2b2b8bbb2ba9ebcbfa7bbacf0bdb1b3"><span class="__cf_email__" data-cfemail="5a2e33373574292e3f36363c3f363e1a383b233f2874393537">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlo Stresemann</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ddbebcafb1b2f3aea9afb8aeb8b0bcb3b39dbfbca4b8aff3beb2b0"><span class="__cf_email__" data-cfemail="c0a3a1b2acafeeb3b4b2a5b3a5ada1aeae80a2a1b9a5b2eea3afad">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Erik Eggert</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roman C. Hillig</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Silke Koehr</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Detlef Stöckigt</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jörg Weiske</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Naomi Barak</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey Mowat</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Brumby</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Clara D. Christ</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antonius ter Laak</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tina Lang</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amaury E. Fernandez-Montalvan</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Volker Badock</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hilmar Weinmann</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ingo V. Hartung</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery, BAYER Pharma AG, Muellerstrasse
178, 13353 Berlin, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magdalena Szewczyk</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven Kennedy</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Pharmacology
and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vijayaratnam Santhakumar</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Princess
Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): E.E., R.C.H, S.K., D.S., J.W., N.B., J.M., T.B., C.D.C, A.t.L., T.L., A.E.F.-M, V.B., H. W., I.V.H., T.S., and C.S. are or have been employees and stockholders of Bayer AG.The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck & Co., Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, So Paulo Research Foundation-FAPESP, Takeda, and the Wellcome Trust..<br /></br></div></li></ul></div><div class="ack" id="ACK-d156e5514-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84681" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84681" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Peakdale Molecular Ltd. UK for excellent support in chemistry, Dr. Shelley Berger for providing p53 antibody, Beamline scientists at the Helmholtz-Zentrum Berlin for excellent facilities and support during data collection, moloX GmbH for crystallographic data collection, and Hanspeter Lauble for help with crystallographic refinement. The SGC is a registered charity (no. 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck & Co., Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and the Wellcome Trust.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i73" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i73"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i74" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i74"> Abbreviations Used</h2><tr><td class="NLM_term">SMYD2</td><td class="NLM_def"><p class="first last">SET and MYND domain containing 2, SET domain was first recognized as a conserved sequence in three <i>Drosophila melanogaster</i> proteins: a modifier of position–effect variegation, suppressor of variegation 3–9 (Su(var.)3–9) the Polycomb-group chromatin regulator Enhancer of zeste (E(z)), and the trithorax-group chromatin regulator trithorax (Trx)</p></td></tr><tr><td class="NLM_term">MYND domain</td><td class="NLM_def"><p class="first last">myeloid, Nervy, and DEAF-1</p></td></tr><tr><td class="NLM_term">ESCC</td><td class="NLM_def"><p class="first last">esophageal squamous carcinoma</p></td></tr><tr><td class="NLM_term">HSP90a</td><td class="NLM_def"><p class="first last">heat shock protein HSP 90-alpha 1</p></td></tr><tr><td class="NLM_term">PARP1</td><td class="NLM_def"><p class="first last">poly(ADP-ribose)-polymerase 1</p></td></tr><tr><td class="NLM_term">TSA</td><td class="NLM_def"><p class="first last">thermal shift assay</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">NCBI</td><td class="NLM_def"><p class="first last">National Center for Biotechnology Information</p></td></tr><tr><td class="NLM_term">CCD</td><td class="NLM_def"><p class="first last">charge-coupled device</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">(4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">FCS</td><td class="NLM_def"><p class="first last">fetal calf serum</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">PRMT5</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 5</p></td></tr><tr><td class="NLM_term">G9A</td><td class="NLM_def"><p class="first last">euchromatic histone-lysine <i>N</i>-methyltransferase 2</p></td></tr><tr><td class="NLM_term">EHMT1</td><td class="NLM_def"><p class="first last">euchromatic histone-lysine <i>N</i>-methyltransferase 1</p></td></tr><tr><td class="NLM_term">SUV39H1</td><td class="NLM_def"><p class="first last">suppressor of variegation 3–9 homologue 1</p></td></tr><tr><td class="NLM_term">SUV39H2</td><td class="NLM_def"><p class="first last">suppressor of variegation 3–9 homologue 2</p></td></tr><tr><td class="NLM_term">SETDB1</td><td class="NLM_def"><p class="first last">SET domain bifurcated 1</p></td></tr><tr><td class="NLM_term">SETD7</td><td class="NLM_def"><p class="first last">SET domain containing (lysine methyltransferase) 7</p></td></tr><tr><td class="NLM_term">SETD8</td><td class="NLM_def"><p class="first last">SET domain containing (lysine methyltransferase) 8</p></td></tr><tr><td class="NLM_term">SUV420H1</td><td class="NLM_def"><p class="first last">suppressor of variegation 4–20 homologue 1</p></td></tr><tr><td class="NLM_term">SUV420H2</td><td class="NLM_def"><p class="first last">suppressor of variegation 4–20 homologue 2</p></td></tr><tr><td class="NLM_term">PRMT1</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 1</p></td></tr><tr><td class="NLM_term">PRMT3</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 3</p></td></tr><tr><td class="NLM_term">PRMT6</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 6</p></td></tr><tr><td class="NLM_term">PRMT8</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 8</p></td></tr><tr><td class="NLM_term">PRDM9</td><td class="NLM_def"><p class="first last">PR domain 9</p></td></tr><tr><td class="NLM_term">SETD2</td><td class="NLM_def"><p class="first last">SET domain containing 2</p></td></tr><tr><td class="NLM_term">SMYD3</td><td class="NLM_def"><p class="first last">SET and MYND domain containing 3</p></td></tr><tr><td class="NLM_term">MLL1</td><td class="NLM_def"><p class="first last">lysine methyltransferase 2A</p></td></tr><tr><td class="NLM_term">MLL3</td><td class="NLM_def"><p class="first last">lysine methyltransferase 2C</p></td></tr><tr><td class="NLM_term">EZH2</td><td class="NLM_def"><p class="first last">enhancer of zeste 2 polycomb repressive complex 2 subunit</p></td></tr><tr><td class="NLM_term">DNMT1</td><td class="NLM_def"><p class="first last">DNA (cytosine-5-)-methyltransferase</p></td></tr><tr><td class="NLM_term">BCDIN3D</td><td class="NLM_def"><p class="first last">BCDIN3 domain containing RNA methyltransferase</p></td></tr><tr><td class="NLM_term">DNMT3A</td><td class="NLM_def"><p class="first last">DNA (cytosine-5-)-methyltransferase 3 alpha</p></td></tr><tr><td class="NLM_term">DNMT3B</td><td class="NLM_def"><p class="first last">DNA (cytosine-5-)-methyltransferase 3 beta</p></td></tr><tr><td class="NLM_term">DNMT3L</td><td class="NLM_def"><p class="first last">DNA (cytosine-5-)-methyltransferase 3-like</p></td></tr><tr><td class="NLM_term">ASH1L</td><td class="NLM_def"><p class="first last">ash1 (absent, small, or homeotic)-like</p></td></tr><tr><td class="NLM_term">DOT1L</td><td class="NLM_def"><p class="first last">DOT1 like histone H3K79 methyltransferase</p></td></tr><tr><td class="NLM_term">NSD1</td><td class="NLM_def"><p class="first last">nuclear receptor binding SET domain protein 1</p></td></tr><tr><td class="NLM_term">NSD2</td><td class="NLM_def"><p class="first last">Wolf–Hirschhorn syndrome candidate 1</p></td></tr><tr><td class="NLM_term">NSD3</td><td class="NLM_def"><p class="first last">Wolf–Hirschhorn syndrome candidate 1-like 1</p></td></tr><tr><td class="NLM_term">METTL21A</td><td class="NLM_def"><p class="first last">methyltransferase like 21A</p></td></tr><tr><td class="NLM_term">METTL21D</td><td class="NLM_def"><p class="first last">valosin containing protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">T/C</td><td class="NLM_def"><p class="first last">treatment/control group</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i75">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07580" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07580" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 61 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span> </span><span class="NLM_article-title">Emerging roles of lysine methylation on non-histone proteins</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">4257</span><span class="NLM_x">–</span> <span class="NLM_lpage">4272</span><span class="refDoi"> DOI: 10.1007/s00018-015-2001-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1007%2Fs00018-015-2001-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=26227335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kmu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2015&pages=4257-4272&author=X.+Zhangauthor=Y.+Huangauthor=X.+Shi&title=Emerging+roles+of+lysine+methylation+on+non-histone+proteins&doi=10.1007%2Fs00018-015-2001-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging roles of lysine methylation on non-histone proteins</span></div><div class="casAuthors">Zhang, Xi; Huang, Yaling; Shi, Xiaobing</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4257-4272</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Lysine methylation is a common posttranslational modification (PTM) of histones that is important for the epigenetic regulation of transcription and chromatin in eukaryotes.  Increasing evidence demonstrates that in addn. to histones, lysine methylation also occurs on various non-histone proteins, esp. transcription- and chromatin-regulating proteins.  In this review, we will briefly describe the histone lysine methyltransferases (KMTs) that have a broad spectrum of non-histone substrates.  We will use p53 and nuclear receptors, esp. estrogen receptor alpha, as examples to discuss the dynamic nature of non-histone protein lysine methylation, the writers, erasers, and readers of these modifications, and the crosstalk between lysine methylation and other PTMs in regulating the functions of the modified proteins.  Understanding the roles of lysine methylation in normal cells and during development will shed light on the complex biol. of diseases assocd. with the dysregulation of lysine methylation on both histones and non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUDZXq2gpdQLVg90H21EOLACvtfcHk0lgIY4BQQ9ylCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kmu7%252FK&md5=8a1a5243088f896012a2b0d368f6235e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fs00018-015-2001-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-015-2001-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DX.%26atitle%3DEmerging%2520roles%2520of%2520lysine%2520methylation%2520on%2520non-histone%2520proteins%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2015%26volume%3D72%26spage%3D4257%26epage%3D4272%26doi%3D10.1007%2Fs00018-015-2001-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Komatsu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imoto, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozaki, K.-i.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aiko, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshizumi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichikawa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otsuji, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inazawa, J.</span><span> </span><span class="NLM_article-title">Overexpression of SMYD2 relates to tumor cell proliferation and malignant outcome of esophageal squamous cell carcinoma</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1139</span><span class="NLM_x">–</span> <span class="NLM_lpage">1146</span><span class="refDoi"> DOI: 10.1093/carcin/bgp116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1093%2Fcarcin%2Fbgp116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19423649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1CisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1139-1146&author=S.+Komatsuauthor=I.+Imotoauthor=H.+Tsudaauthor=K.-i.+Kozakiauthor=T.+Muramatsuauthor=Y.+Shimadaauthor=S.+Aikoauthor=Y.+Yoshizumiauthor=D.+Ichikawaauthor=E.+Otsujiauthor=J.+Inazawa&title=Overexpression+of+SMYD2+relates+to+tumor+cell+proliferation+and+malignant+outcome+of+esophageal+squamous+cell+carcinoma&doi=10.1093%2Fcarcin%2Fbgp116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of SMYD2 relates to tumor cell proliferation and malignant outcome of esophageal squamous cell carcinoma</span></div><div class="casAuthors">Komatsu, Shuhei; Imoto, Issei; Tsuda, Hitoshi; Kozaki, Ken-ich; Muramatsu, Tomoki; Shimada, Yutaka; Aiko, Satoshi; Yoshizumi, Yutaka; Ichikawa, Daisuke; Otsuji, Eigo; Inazawa, Johji</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1139-1146</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although we have identified two putative targets, ATF3 and CENPF, for a frequently gained/amplified region around 1q32-q41 in esophageal squamous cell carcinoma (ESCC), it is possible that other amplification targets remain to be identified.  In this study, we tested whether SET and MYND domain-contg. protein 2 (SMYD2), located between those two genes and encoding a lysine methyltransferase for histone H3K36 and p53K370 that regulates transcription and inhibits transactivation activity, resp., acts as a cancer-promoting gene through activation/overexpression in ESCC.  Frequent overexpression of SMYD2 mRNA and protein was obsd. in KYSE150 cells with remarkable amplification at 1q32-41.1 and other ESCC cell lines (11/43 lines, 25.6%).  Overexpression of SMYD2 protein was frequently detected in primary tumor samples of ESCC (117/153 cases, 76.5%) as well and significantly correlated with gender, venous invasion, the pT category in the tumor-lymph node-metastases classification and status of recurrence.  Patients with SMYD2-overexpressing tumors had a worse overall rate of survival than those with non-expressing tumors, and SMYD2 positivity was independently assocd. with a worse outcome in the multivariate anal.  Knockdown of SMYD2 expression inhibited and ectopic overexpression of SMYD2 promoted the proliferation of ESCC cells in a TP53 mutation-independent but SMYD2 expression-dependent manner.  These findings suggest that SMYD2 plays an important role in tumor cell proliferation through its activation/overexpression and highlight its usefulness as a prognosticator and potential therapeutic target in ESCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGuuMnL7IjzbVg90H21EOLACvtfcHk0ljCrSAPJmPMkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1CisLs%253D&md5=7eb396b256c9ec2b182fd999796527b5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp116%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DS.%26aulast%3DImoto%26aufirst%3DI.%26aulast%3DTsuda%26aufirst%3DH.%26aulast%3DKozaki%26aufirst%3DK.-i.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DAiko%26aufirst%3DS.%26aulast%3DYoshizumi%26aufirst%3DY.%26aulast%3DIchikawa%26aufirst%3DD.%26aulast%3DOtsuji%26aufirst%3DE.%26aulast%3DInazawa%26aufirst%3DJ.%26atitle%3DOverexpression%2520of%2520SMYD2%2520relates%2520to%2520tumor%2520cell%2520proliferation%2520and%2520malignant%2520outcome%2520of%2520esophageal%2520squamous%2520cell%2520carcinoma%26jtitle%3DCarcinogenesis%26date%3D2009%26volume%3D30%26spage%3D1139%26epage%3D1146%26doi%3D10.1093%2Fcarcin%2Fbgp116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Komatsu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichikawa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirajima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawaguchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamae, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okajima, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konishi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiozaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imoto, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inazawa, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otsuji, E.</span><span> </span><span class="NLM_article-title">Overexpression of SMYD2 contributes to malignant outcome in gastric cancer</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">357</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/bjc.2014.543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fbjc.2014.543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25321194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOkt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=357-364&author=S.+Komatsuauthor=D.+Ichikawaauthor=S.+Hirajimaauthor=H.+Nagataauthor=Y.+Nishimuraauthor=T.+Kawaguchiauthor=M.+Miyamaeauthor=W.+Okajimaauthor=T.+Ohashiauthor=H.+Konishiauthor=A.+Shiozakiauthor=H.+Fujiwaraauthor=K.+Okamotoauthor=H.+Tsudaauthor=I.+Imotoauthor=J.+Inazawaauthor=E.+Otsuji&title=Overexpression+of+SMYD2+contributes+to+malignant+outcome+in+gastric+cancer&doi=10.1038%2Fbjc.2014.543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of SMYD2 contributes to malignant outcome in gastric cancer</span></div><div class="casAuthors">Komatsu, S.; Ichikawa, D.; Hirajima, S.; Nagata, H.; Nishimura, Y.; Kawaguchi, T.; Miyamae, M.; Okajima, W.; Ohashi, T.; Konishi, H.; Shiozaki, A.; Fujiwara, H.; Okamoto, K.; Tsuda, H.; Imoto, I.; Inazawa, J.; Otsuji, E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">357-364</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Background: SET and MYND domain-contg. protein 2 (SMYD2) is a lysine methyltransferase for histone H3, p53 and Rb and inhibits their transactivation activities.  In this study, we tested whether SMYD2 (1q42) acts as a cancer-promoting factor by being overexpressed in gastric cancer.  Methods: We analyzed 7 gastric cancer cell lines and 147 primary tumor samples of gastric cancer, which were curatively resected in our hospital.  Results: SET and MYND domain-contg. protein 2 was detected in these cell lines (five out of seven cell lines; 71.4%) and primary tumor samples (fifty-six out of one hundred and forty-seven cases; 38.1%).  Knockdown of SMYD2 using specific small interfering RNA inhibited proliferation, migration and invasion of SMYD2-overexpressing cells in a TP53 mutation-independent manner.  Overexpression of SMYD2 protein correlated with larger tumor size, more aggressive lymphatic invasion, deeper tumor invasion and higher rates of lymph node metastasis and recurrence.  Patients with SMYD2-overexpressing tumors had a worse overall rate of survival than those with non-expressing tumors (P=0.0073, log-rank test) in an intensity and proportion score-dependent manner.  Moreover, multivariate anal. demonstrated that SMYD2 was independently assocd. with worse outcome (P=0.0021, hazard ratio 4.25 (1.69-10.7)).  Conclusions: These findings suggest that SMYD2 has a crucial role in tumor cell proliferation by its overexpression and highlight its usefulness as a prognostic factor and potential therapeutic target in gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph37xZhp4Ll7Vg90H21EOLACvtfcHk0ljCrSAPJmPMkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOkt77O&md5=4c1b9afa2e324eedb0173eee4d65c884</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2014.543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2014.543%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DS.%26aulast%3DIchikawa%26aufirst%3DD.%26aulast%3DHirajima%26aufirst%3DS.%26aulast%3DNagata%26aufirst%3DH.%26aulast%3DNishimura%26aufirst%3DY.%26aulast%3DKawaguchi%26aufirst%3DT.%26aulast%3DMiyamae%26aufirst%3DM.%26aulast%3DOkajima%26aufirst%3DW.%26aulast%3DOhashi%26aufirst%3DT.%26aulast%3DKonishi%26aufirst%3DH.%26aulast%3DShiozaki%26aufirst%3DA.%26aulast%3DFujiwara%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DTsuda%26aufirst%3DH.%26aulast%3DImoto%26aufirst%3DI.%26aulast%3DInazawa%26aufirst%3DJ.%26aulast%3DOtsuji%26aufirst%3DE.%26atitle%3DOverexpression%2520of%2520SMYD2%2520contributes%2520to%2520malignant%2520outcome%2520in%2520gastric%2520cancer%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2015%26volume%3D112%26spage%3D357%26epage%3D364%26doi%3D10.1038%2Fbjc.2014.543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Cho, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaue, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">RB1 methylation by SMYD2 enhances cell cycle progression through an increase of RB1 phosphorylation</span> <span class="citation_source-journal">Neoplasia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">476</span><span class="NLM_x">–</span> <span class="NLM_lpage">486</span><span class="refDoi"> DOI: 10.1593/neo.12656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1593%2Fneo.12656" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=476-486&author=H.-S.+Choauthor=S.+Hayamiauthor=G.+Toyokawaauthor=K.+Maejimaauthor=Y.+Yamaneauthor=T.+Suzukiauthor=N.+Dohmaeauthor=M.+Kogureauthor=D.+Kangauthor=D.+E.+Nealauthor=B.+A.+J.+Ponderauthor=H.+Yamaueauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=RB1+methylation+by+SMYD2+enhances+cell+cycle+progression+through+an+increase+of+RB1+phosphorylation&doi=10.1593%2Fneo.12656"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1593%2Fneo.12656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1593%252Fneo.12656%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.-S.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DKogure%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DYamaue%26aufirst%3DH.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DRB1%2520methylation%2520by%2520SMYD2%2520enhances%2520cell%2520cycle%2520progression%2520through%2520an%2520increase%2520of%2520RB1%2520phosphorylation%26jtitle%3DNeoplasia%26date%3D2012%26volume%3D14%26spage%3D476%26epage%3D486%26doi%3D10.1593%2Fneo.12656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Sakamoto, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrade, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felipe, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoyama, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pittella Silva, F.</span><span> </span><span class="NLM_article-title">SMYD2 is highly expressed in pediatric acute lymphoblastic leukemia and constitutes a bad prognostic factor</span> <span class="citation_source-journal">Leuk. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">496</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1016/j.leukres.2014.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.leukres.2014.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=24631370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlWlu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2014&pages=496-502&author=L.+H.+Sakamotoauthor=R.+V.+Andradeauthor=M.+S.+Felipeauthor=A.+B.+Motoyamaauthor=F.+Pittella+Silva&title=SMYD2+is+highly+expressed+in+pediatric+acute+lymphoblastic+leukemia+and+constitutes+a+bad+prognostic+factor&doi=10.1016%2Fj.leukres.2014.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">SMYD2 is highly expressed in pediatric acute lymphoblastic leukemia and constitutes a bad prognostic factor</span></div><div class="casAuthors">Sakamoto, Luis Henrique Toshihiro; Andrade, Rosangela Vieira de; Felipe, Maria Sueli Soares; Motoyama, Andrea Barretto; Pittella Silva, Fabio</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">496-502</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Acute lymphoblastic leukemia (ALL) is the most common childhood malignancy.  Although several clin. characteristics can be assocd. with worse prognosis, more robust biol. markers still remains uncovered.  SMYD2, a member of SMYD protein family, regulates the activity of several proteins through methylation.  In this study, we performed quant. real time PCR to compare the expression of SMYD2 in 83 pediatric ALL patients and non-neoplastic bone marrow samples (BMS).  The study revealed that SMYD2 expression is altered in ALL BMS and its high expression was correlated with a bad prognosis.  Moreover, we also revealed that SMYD2 expression level significantly decreases in patients that respond to chemotherapy treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCUDTlFWqR_bVg90H21EOLACvtfcHk0lj4RpMBfOq2Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlWlu7s%253D&md5=001e2c0022c172b25ae457f0dc169879</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2014.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2014.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DL.%2BH.%26aulast%3DAndrade%26aufirst%3DR.%2BV.%26aulast%3DFelipe%26aufirst%3DM.%2BS.%26aulast%3DMotoyama%26aufirst%3DA.%2BB.%26aulast%3DPittella%2BSilva%26aufirst%3DF.%26atitle%3DSMYD2%2520is%2520highly%2520expressed%2520in%2520pediatric%2520acute%2520lymphoblastic%2520leukemia%2520and%2520constitutes%2520a%2520bad%2520prognostic%2520factor%26jtitle%3DLeuk.%2520Res.%26date%3D2014%26volume%3D38%26spage%3D496%26epage%3D502%26doi%3D10.1016%2Fj.leukres.2014.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Pimkhaokham, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imoto, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imamura, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amagasa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inazawa, J.</span><span> </span><span class="NLM_article-title">Nonrandom chromosomal imbalances in esophageal squamous cell carcinoma cell lines: possible involvement of the <i>ATF3</i> and <i>CENPF</i> genes in the 1q32 amplicon</span> <span class="citation_source-journal">Jpn. J. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">1126</span><span class="NLM_x">–</span> <span class="NLM_lpage">1133</span><span class="refDoi"> DOI: 10.1111/j.1349-7006.2000.tb00895.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1111%2Fj.1349-7006.2000.tb00895.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=11092977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVGrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2000&pages=1126-1133&author=A.+Pimkhaokhamauthor=Y.+Shimadaauthor=Y.+Fukudaauthor=N.+Kuriharaauthor=I.+Imotoauthor=Z.-Q.+Yangauthor=M.+Imamuraauthor=Y.+Nakamuraauthor=T.+Amagasaauthor=J.+Inazawa&title=Nonrandom+chromosomal+imbalances+in+esophageal+squamous+cell+carcinoma+cell+lines%3A+possible+involvement+of+the+ATF3+and+CENPF+genes+in+the+1q32+amplicon&doi=10.1111%2Fj.1349-7006.2000.tb00895.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Nonrandom chromosomal imbalances in esophageal squamous cell carcinoma cell lines: possible involvement of the ATF3 and CENPF genes in the 1q32 amplicon</span></div><div class="casAuthors">Pimkhaokham, Atiphan; Shimada, Yutaka; Fukuda, Yohji; Kurihara, Naoki; Imoto, Issei; Yang, Zeng-Quan; Imamura, Masayuki; Nakamura, Yusuke; Amagasa, Teruo; Inazawa, Johji</div><div class="citationInfo"><span class="NLM_cas:title">Japanese Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1126-1133</span>CODEN:
                <span class="NLM_cas:coden">JJCREP</span>;
        ISSN:<span class="NLM_cas:issn">0910-5050</span>.
    
            (<span class="NLM_cas:orgname">Japanese Cancer Association</span>)
        </div><div class="casAbstract">Using comparative genomic hybridization (CGH), the authors investigated copy no. aberrations in 29 esophageal squamous cell carcinoma (ESC) cell lines.  All lines displayed numerous chromosome imbalances.  The most frequent losses were obsd. on chromosome 18q (65.5%), Xp (48.3%), 3p (44.8%), 4q (44.8%), 8p (41.4%), 11q23-25 (34.5%) and 4p (27.6%), whereas the most common copy no. gains were noted at 8q (86.2%), 3q (82.8%), 5p (69%), 7p (69%), 20q (65.5%), 9q (55.2%), 11q (55.2%), 1q (48.3%), Xq (44.8%) and 18p (37.9%).  High-level gains (HLGs) were detected at 3q26 (9 cases), 8q23 (6 cases), 5p14-15 (6 cases), 18p11.2-11.3 (6 cases), 3q27-28 (5 cases), 5p13 (3 cases), 7p14-15 (3 cases), 20q12-13 (3 cases), 11q13 (3 cases), 14q21 (2 cases), 20p11.2 (2 cases), 13q32 (2 case), and 1q32 (1 case).  Among them, HLGs of 1q32 have been reported in other types of cancer, including glioblastoma and breast cancers.  The authors successfully narrowed down the smallest common amplicon involving 1q-gain to the genomic segment between D1S414 and D1S2860 by fluorescence in situ hybridization (FISH).  Southern and northern blot anal. clearly demonstrated that ATF3, human activating transcription factor-3 and CENPF, centromere protein F, mapped within this region, were significantly amplified and over-expressed in 1q32 amplicon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqneEUHUiLt-LVg90H21EOLACvtfcHk0lj4RpMBfOq2Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVGrsL0%253D&md5=6e514f7c3de4599fb56110b1b85f3012</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2000.tb00895.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2000.tb00895.x%26sid%3Dliteratum%253Aachs%26aulast%3DPimkhaokham%26aufirst%3DA.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DFukuda%26aufirst%3DY.%26aulast%3DKurihara%26aufirst%3DN.%26aulast%3DImoto%26aufirst%3DI.%26aulast%3DYang%26aufirst%3DZ.-Q.%26aulast%3DImamura%26aufirst%3DM.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DAmagasa%26aufirst%3DT.%26aulast%3DInazawa%26aufirst%3DJ.%26atitle%3DNonrandom%2520chromosomal%2520imbalances%2520in%2520esophageal%2520squamous%2520cell%2520carcinoma%2520cell%2520lines%253A%2520possible%2520involvement%2520of%2520the%2520ATF3%2520and%2520CENPF%2520genes%2520in%2520the%25201q32%2520amplicon%26jtitle%3DJpn.%2520J.%2520Cancer%2520Res.%26date%3D2000%26volume%3D91%26spage%3D1126%26epage%3D1133%26doi%3D10.1111%2Fj.1349-7006.2000.tb00895.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Brown, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottlieb, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, P. W.</span><span> </span><span class="NLM_article-title">Identification and characterization of Smyd2: a split SET/MYND domain-containing histone H3 lysine 36-specific methyltransferase that interacts with the Sin3 histone deacetylase complex</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">26</span><span class="refDoi"> DOI: 10.1186/1476-4598-5-26</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1186%2F1476-4598-5-26" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=16805913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A280%3ADC%252BD28vlslagtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=26&author=M.+A.+Brownauthor=R.+J.+Simsauthor=P.+D.+Gottliebauthor=P.+W.+Tucker&title=Identification+and+characterization+of+Smyd2%3A+a+split+SET%2FMYND+domain-containing+histone+H3+lysine+36-specific+methyltransferase+that+interacts+with+the+Sin3+histone+deacetylase+complex&doi=10.1186%2F1476-4598-5-26"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of Smyd2: a split SET/MYND domain-containing histone H3 lysine 36-specific methyltransferase that interacts with the Sin3 histone deacetylase complex</span></div><div class="casAuthors">Brown Mark A; Sims Robert J 3rd; Gottlieb Paul D; Tucker Philip W</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Disrupting the balance of histone lysine methylation alters the expression of genes involved in tumorigenesis including proto-oncogenes and cell cycle regulators.  Methylation of lysine residues is commonly catalyzed by a family of proteins that contain the SET domain.  Here, we report the identification and characterization of the SET domain-containing protein, Smyd2.  RESULTS:  Smyd2 mRNA is most highly expressed in heart and brain tissue, as demonstrated by northern analysis and in situ hybridization.  Over-expressed Smyd2 localizes to the cytoplasm and the nucleus in 293T cells.  Although accumulating evidence suggests that methylation of histone 3, lysine 36 (H3K36) is associated with actively transcribed genes, we show that the SET domain of Smyd2 mediates H3K36 dimethylation and that Smyd2 represses transcription from an SV40-luciferase reporter.  Smyd2 associates specifically with the Sin3A histone deacetylase complex, which was recently linked to H3K36 methylation within the coding regions of active genes in yeast.  Finally, we report that exogenous expression of Smyd2 suppresses cell proliferation.  CONCLUSION:  We propose that Sin3A-mediated deacetylation within the coding regions of active genes is directly linked to the histone methyltransferase activity of Smyd2.  Moreover, Smyd2 appears to restrain cell proliferation, likely through direct modulation of chromatin structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTT6a8vy2w9sbrIbt-tGZmxfW6udTcc2eas3nscdA8B0bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28vlslagtg%253D%253D&md5=af9e05d60f2cd0880783270911dc281f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-5-26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-5-26%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DM.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DGottlieb%26aufirst%3DP.%2BD.%26aulast%3DTucker%26aufirst%3DP.%2BW.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520Smyd2%253A%2520a%2520split%2520SET%252FMYND%2520domain-containing%2520histone%2520H3%2520lysine%252036-specific%2520methyltransferase%2520that%2520interacts%2520with%2520the%2520Sin3%2520histone%2520deacetylase%2520complex%26jtitle%3DMol.%2520Cancer%26date%3D2006%26volume%3D5%26spage%3D26%26doi%3D10.1186%2F1476-4598-5-26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Abu-Farha, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Madhoun, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elisma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skerjanc, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figeys, D.</span><span> </span><span class="NLM_article-title">The tale of two domains: proteomics and genomics analysis of SMYD2, a new histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">560</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span><span class="refDoi"> DOI: 10.1074/mcp.M700271-MCP200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1074%2Fmcp.M700271-MCP200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=18065756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFygt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=560-572&author=M.+Abu-Farhaauthor=J.-P.+Lambertauthor=A.+S.+Al-Madhounauthor=F.+Elismaauthor=I.+S.+Skerjancauthor=D.+Figeys&title=The+tale+of+two+domains%3A+proteomics+and+genomics+analysis+of+SMYD2%2C+a+new+histone+methyltransferase&doi=10.1074%2Fmcp.M700271-MCP200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The tale of two domains: proteomics and genomics analysis of SMYD2, a new histone methyltransferase</span></div><div class="casAuthors">Abu-Farha, Mohamed; Lambert, Jean-Philippe; Al-Madhoun, Ashraf S.; Elisma, Fred; Skerjanc, Ilona S.; Figeys, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">560-572</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9476</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Very little is known about SET- and MYND-contg. protein 2 (SMYD2), a member of the SMYD protein family.  However, the interest in better understanding the roles of SMYD2 has grown because of recent reports indicating that SMYD2 methylates p53 and histone H3.  In this study, we present a combined proteomics and genomics study of SMYD2 designed to elucidate its mol. roles.  We report the cytosolic and nuclear interactome of SMYD2 using a combination of immunopptn. coupled with high throughput MS, chromatin immunopptn. coupled with high throughput MS, and co-immunopptn. methods.  In particular, we report that SMYD2 interacted with HSP90α independently of the SET and MYND domains, with EBP41L3 through the MYND domain, and with p53 through the SET domain.  We demonstrated that the interaction of SMYD2 with HSP90α enhances SMYD2 histone methyltransferase activity and specificity for histone H3 at lysine 4 (H3K4) in vitro.  Interestingly histone H3K36 methyltransferase activity was independent of its interaction with HSP90α similar to LSD1 dependence on the androgen receptor.  We also showed that the SET domain is required for the methylation at H3K4.  We demonstrated using a modified chromatin immunopptn. protocol that the SMYD2 gain of function leads to an increase in H3K4 methylation in vivo, whereas no observable levels of H3K36 were detected.  We also report that the SMYD2 gain of function was correlated with the upregulation of 37 and down-regulation of four genes, the majority of which are involved in the cell cycle, chromatin remodeling, and transcriptional regulation.  TACC2 is one of the genes up-regulated as a result of SMYD2 gain of function.  Up-regulation of TACC2 by SMYD2 occurred as a result of SMYD2 binding to the TACC2 promoter where it methylates H3K4.  Furthermore the combination of the SMYD2 interactome with the gene expression data suggests that some of the genes regulated by SMYD2 are closely assocd. with SMYD2-interacting proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsnP4zSi0t2LVg90H21EOLACvtfcHk0li6zO7WFh0h1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFygt7w%253D&md5=0d3fe1f7c3be7472bb4d25c7ed5b93e6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M700271-MCP200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M700271-MCP200%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Farha%26aufirst%3DM.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DAl-Madhoun%26aufirst%3DA.%2BS.%26aulast%3DElisma%26aufirst%3DF.%26aulast%3DSkerjanc%26aufirst%3DI.%2BS.%26aulast%3DFigeys%26aufirst%3DD.%26atitle%3DThe%2520tale%2520of%2520two%2520domains%253A%2520proteomics%2520and%2520genomics%2520analysis%2520of%2520SMYD2%252C%2520a%2520new%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2008%26volume%3D7%26spage%3D560%26epage%3D572%26doi%3D10.1074%2Fmcp.M700271-MCP200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Hamamoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saloura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span> </span><span class="NLM_article-title">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1038/nrc3884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnrc3884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25614009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=110-124&author=R.+Hamamotoauthor=V.+Salouraauthor=Y.+Nakamura&title=Critical+roles+of+non-histone+protein+lysine+methylation+in+human+tumorigenesis&doi=10.1038%2Fnrc3884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span></div><div class="casAuthors">Hamamoto, Ryuji; Saloura, Vassiliki; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">110-124</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Several protein lysine methyltransferases and demethylases have been identified to have crit. roles in histone modification.  A large body of evidence has indicated that their dysregulation is involved in the development and progression of various diseases, including cancer, and these enzymes are now considered to be potential therapeutic targets.  Although most studies have focused on histone methylation, many reports have revealed that these enzymes also regulate the methylation dynamics of non-histone proteins such as p53, RB1 and STAT3 (signal transducer and activator of transcription 3), which have important roles in human tumorigenesis.  In this Review, we summarize the mol. functions of protein lysine methylation and its involvement in human cancer, with a particular focus on lysine methylation of non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0wn_Jk7jSGbVg90H21EOLACvtfcHk0li6zO7WFh0h1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D&md5=9d998a4efb3c1a069e14747341ceca5e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc3884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3884%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DSaloura%26aufirst%3DV.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DCritical%2520roles%2520of%2520non-histone%2520protein%2520lysine%2520methylation%2520in%2520human%2520tumorigenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D110%26epage%3D124%26doi%3D10.1038%2Fnrc3884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Benayoun, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veitia, R. A.</span><span> </span><span class="NLM_article-title">A post-translational modification code for transcription factors: sorting through a sea of signals</span> <span class="citation_source-journal">Trends Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">189</span><span class="NLM_x">–</span> <span class="NLM_lpage">197</span><span class="refDoi"> DOI: 10.1016/j.tcb.2009.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.tcb.2009.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19328693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVersbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=189-197&author=B.+A.+Benayounauthor=R.+A.+Veitia&title=A+post-translational+modification+code+for+transcription+factors%3A+sorting+through+a+sea+of+signals&doi=10.1016%2Fj.tcb.2009.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A post-translational modification code for transcription factors: sorting through a sea of signals</span></div><div class="casAuthors">Benayoun, Berenice A.; Veitia, Reiner A.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">189-197</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cellular responses to environmental or physiol. cues rely on transduction pathways that must ensure discrimination between different signals.  These cascades crosstalk and lead to a combinatorial regulation.  This often results in different combinations of post-translational modifications (PTMs) on target proteins, which might act as a mol. barcode.  Although appealing, the idea of the existence of such a code for transcription factors is debated.  Using general arguments and recent evidence, it is proposed that a PTM code is not only possible but necessary in the context of transcription factors regulating multiple processes.  Thus, the coding potential of PTM combinations should both provide a further layer of information integration from several transduction pathways and warrant highly specific cellular outputs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomvF_m-rjksbVg90H21EOLACvtfcHk0li6zO7WFh0h1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVersbs%253D&md5=cbc800258bc1a04d9bf2d67e1abddd0f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2009.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2009.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DBenayoun%26aufirst%3DB.%2BA.%26aulast%3DVeitia%26aufirst%3DR.%2BA.%26atitle%3DA%2520post-translational%2520modification%2520code%2520for%2520transcription%2520factors%253A%2520sorting%2520through%2520a%2520sea%2520of%2520signals%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D189%26epage%3D197%26doi%3D10.1016%2Fj.tcb.2009.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Fan, J.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, P.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, J.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.-N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M.</span><span> </span><span class="NLM_article-title">The selective activation of p53 target genes regulated by SMYD2 in BIX-01294 induced autophagy-related cell death</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0116782</span><span class="refDoi"> DOI: 10.1371/journal.pone.0116782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1371%2Fjournal.pone.0116782" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0116782&author=J.-D.+Fanauthor=P.-J.+Leiauthor=J.-Y.+Zhengauthor=X.+Wangauthor=S.+Liauthor=H.+Liuauthor=Y.-L.+Heauthor=Z.-N.+Wangauthor=G.+Weiauthor=X.+Zhangauthor=L.-Y.+Liauthor=M.+Wu&title=The+selective+activation+of+p53+target+genes+regulated+by+SMYD2+in+BIX-01294+induced+autophagy-related+cell+death&doi=10.1371%2Fjournal.pone.0116782"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0116782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0116782%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DJ.-D.%26aulast%3DLei%26aufirst%3DP.-J.%26aulast%3DZheng%26aufirst%3DJ.-Y.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DY.-L.%26aulast%3DWang%26aufirst%3DZ.-N.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.-Y.%26aulast%3DWu%26aufirst%3DM.%26atitle%3DThe%2520selective%2520activation%2520of%2520p53%2520target%2520genes%2520regulated%2520by%2520SMYD2%2520in%2520BIX-01294%2520induced%2520autophagy-related%2520cell%2520death%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0116782%26doi%3D10.1371%2Fjournal.pone.0116782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Arenas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span> </span><span class="NLM_article-title">Structural basis of substrate methylation and inhibition of SMYD2</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1016/j.str.2011.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.str.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21782458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1262-1273&author=A.+D.+Fergusonauthor=N.+A.+Larsenauthor=T.+Howardauthor=H.+Pollardauthor=I.+Greenauthor=C.+Grandeauthor=T.+Cheungauthor=R.+Garcia-Arenasauthor=S.+Cowenauthor=J.+Wuauthor=R.+Godinauthor=H.+Chenauthor=N.+Keen&title=Structural+basis+of+substrate+methylation+and+inhibition+of+SMYD2&doi=10.1016%2Fj.str.2011.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Methylation and Inhibition of SMYD2</span></div><div class="casAuthors">Ferguson, Andrew D.; Larsen, Nicholas A.; Howard, Tina; Pollard, Hannah; Green, Isabelle; Grande, Christie; Cheung, Tony; Garcia-Arenas, Renee; Cowen, Scott; Wu, Jiaquan; Godin, Robert; Chen, Huawei; Keen, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1262-1273</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases are important regulators of epigenetic signaling.  These enzymes catalyze the transfer of donor Me groups from S-adenosylmethionine to specific acceptor lysines on histones, leading to changes in chromatin structure and transcriptional regulation.  These enzymes also methylate nonhistone protein substrates, revealing an addnl. mechanism to regulate cellular physiol.  The oncogenic protein SMYD2 represses the functional activities of the tumor suppressor proteins p53 and Rb, making it an attractive drug target.  Here we report the discovery of AZ505, a potent and selective inhibitor of SMYD2 that was identified from a high throughput chem. screen.  We also present the crystal structures of SMYD2 with p53 substrate and product peptides, and notably, in complex with AZ505.  This substrate competitive inhibitor is bound in the peptide binding groove of SMYD2.  These results have implications for the development of SMYD2 inhibitors, and indicate the potential for developing novel therapies targeting this target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfo0RkV17Th7Vg90H21EOLACvtfcHk0ljVU8OG2FLrXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK&md5=31a08f46bffea75609e1c5bb25b7c610</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLarsen%26aufirst%3DN.%2BA.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGarcia-Arenas%26aufirst%3DR.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKeen%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520methylation%2520and%2520inhibition%2520of%2520SMYD2%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1262%26epage%3D1273%26doi%3D10.1016%2Fj.str.2011.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Burgos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Placek, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">Repression of p53 activity by Smyd2-mediated methylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">444</span><span class="NLM_x">, </span> <span class="NLM_fpage">629</span><span class="NLM_x">–</span> <span class="NLM_lpage">632</span><span class="refDoi"> DOI: 10.1038/nature05287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnature05287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=17108971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1KjurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=444&publication_year=2006&pages=629-632&author=J.+Huangauthor=L.+Perez-Burgosauthor=B.+J.+Placekauthor=R.+Senguptaauthor=M.+Richterauthor=J.+A.+Dorseyauthor=S.+Kubicekauthor=S.+Opravilauthor=T.+Jenuweinauthor=S.+L.+Berger&title=Repression+of+p53+activity+by+Smyd2-mediated+methylation&doi=10.1038%2Fnature05287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Repression of p53 activity by Smyd2-mediated methylation</span></div><div class="casAuthors">Huang, Jing; Perez-Burgos, Laura; Placek, Brandon J.; Sengupta, Roopsha; Richter, Mario; Dorsey, Jean A.; Kubicek, Stefan; Opravil, Susanne; Jenuwein, Thomas; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">444</span>
        (<span class="NLM_cas:issue">7119</span>),
    <span class="NLM_cas:pages">629-632</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Specific sites of lysine methylation on histones correlate with either activation or repression of transcription.  The tumor suppressor p53 (refs 4-7) is one of only a few non-histone proteins known to be regulated by lysine methylation.  Here we report a lysine methyltransferase, Smyd2, that methylates a previously unidentified site, Lys 370, in p53.  This methylation site, in contrast to the known site Lys 372, is repressing to p53-mediated transcriptional regulation.  Smyd2 helps to maintain low concns. of promoter-assocd. p53.  We show that reducing Smyd2 concns. by short interfering RNA enhances p53-mediated apoptosis.  We find that Set9-mediated methylation of Lys 372 inhibits Smyd2-mediated methylation of Lys 370, providing regulatory cross-talk between post-translational modifications.  In addn., we show that the inhibitory effect of Lys 372 methylation on Lys 370 methylation is caused, in part, by blocking the interaction between p53 and Smyd2.  Thus, similar to histones, p53 is subject to both activating and repressing lysine methylation.  These results also predict that Smyd2 may function as a putative oncogene by methylating p53 and repressing its tumor suppressive function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUh0EYS8bhMrVg90H21EOLACvtfcHk0ljVU8OG2FLrXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1KjurjP&md5=28ecb05d42c8631445e6563676406e32</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature05287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05287%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DPerez-Burgos%26aufirst%3DL.%26aulast%3DPlacek%26aufirst%3DB.%2BJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DDorsey%26aufirst%3DJ.%2BA.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DRepression%2520of%2520p53%2520activity%2520by%2520Smyd2-mediated%2520methylation%26jtitle%3DNature%26date%3D2006%26volume%3D444%26spage%3D629%26epage%3D632%26doi%3D10.1038%2Fnature05287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirinupong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span> </span><span class="NLM_article-title">Crystal structures of histone and p53 methyltransferase SmyD2 reveal a conformational flexibility of the autoinhibitory C-terminal domain</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e21640</span><span class="refDoi"> DOI: 10.1371/journal.pone.0021640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1371%2Fjournal.pone.0021640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21738746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVaitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e21640&author=Y.+Jiangauthor=N.+Sirinupongauthor=J.+Brunzelleauthor=Z.+Yang&title=Crystal+structures+of+histone+and+p53+methyltransferase+SmyD2+reveal+a+conformational+flexibility+of+the+autoinhibitory+C-terminal+domain&doi=10.1371%2Fjournal.pone.0021640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of histone and p53 methyltransferase SmyD2 reveal a conformational flexibility of the autoinhibitory C-terminal domain</span></div><div class="casAuthors">Jiang, Yuanyuan; Sirinupong, Nualpun; Brunzelle, Joseph; Yang, Zhe</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e21640</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Histone methyltransferase SmyD2 belongs to a new class of chromatin regulators that control gene expression in heart development and tumorigenesis.  In addn. to methylation of histone H3 Lys-4, SmyD2 can methylate non-histone targets including p53 and the retinoblastoma tumor suppressor.  The methyltransferase activity of SmyD proteins has been proposed to be regulated by autoinhibition via the intra- and inter-domain bending of the conserved C-terminal domain (CTD).  However, there has been no direct evidence of a conformational change in the CTD.  Here, the authors report 2 crystal structures of SmyD2 bound either to the cofactor product, S-adenosylhomocysteine, or to the inhibitor, sinefungin.  SmyD2 was found to have a 2-lobed structure with the active site located at the bottom of a deep crevice formed between the CTD and the catalytic domain.  By extensive engagement with the methyltransferase domain, the CTD stabilized the autoinhibited conformation of SmyD2 and restricted access to the catalytic site.  Unexpectedly, despite that the 2 SmyD2 structures were highly superimposable, significant differences were obsd. in the 1st 2 helixes of the CTDs: the 2 helixes were found to bend outward and move away from the catalytic domain to generate a less closed conformation in the sinefungin-bound structure.  Although the overall fold of the individual domains was structurally conserved among SmyD proteins, SmyD2 appeared to be a conformational "intermediate" between a close form of SmyD3 and an open form of SmyD1.  In addn., the structures revealed that the CTD was structurally similar to tetratricopeptide repeats (TPR), a motif through which many cochaperones bind to heat-shock protein Hsp90.  These results thus provide the 1st evidence for the intradomain flexibility of the TPR-like CTD, which may be important for the activation of SmyD proteins by Hsp90.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixUn8onz2U7Vg90H21EOLACvtfcHk0ljVU8OG2FLrXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosVaitbs%253D&md5=7507f0056e8de2c60bec871894bdaf59</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0021640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0021640%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DSirinupong%26aufirst%3DN.%26aulast%3DBrunzelle%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DCrystal%2520structures%2520of%2520histone%2520and%2520p53%2520methyltransferase%2520SmyD2%2520reveal%2520a%2520conformational%2520flexibility%2520of%2520the%2520autoinhibitory%2520C-terminal%2520domain%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De21640%26doi%3D10.1371%2Fjournal.pone.0021640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Sajjad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novoyatleva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vergarajauregui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troidl, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schermuly, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, H. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, F. B.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase Smyd2 suppresses p53-dependent cardiomyocyte apoptosis</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1843</span><span class="NLM_x">, </span> <span class="NLM_fpage">2556</span><span class="NLM_x">–</span> <span class="NLM_lpage">2562</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2014.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.bbamcr.2014.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25014164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1yksr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=2556-2562&author=A.+Sajjadauthor=T.+Novoyatlevaauthor=S.+Vergarajaureguiauthor=C.+Troidlauthor=R.+T.+Schermulyauthor=H.+O.+Tuckerauthor=F.+B.+Engel&title=Lysine+methyltransferase+Smyd2+suppresses+p53-dependent+cardiomyocyte+apoptosis&doi=10.1016%2Fj.bbamcr.2014.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methyltransferase Smyd2 suppresses p53-dependent cardiomyocyte apoptosis</span></div><div class="casAuthors">Sajjad, Amna; Novoyatleva, Tatyana; Vergarajauregui, Silvia; Troidl, Christian; Schermuly, Ralph T.; Tucker, Haley O.; Engel, Felix B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2556-2562</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Apoptosis, or programmed cell death, is an essential physiol. process for proper embryogenesis as well as for homeostasis during aging.  In addn., apoptosis is one of the major mechanisms causing cell loss in pathophysiol. conditions such as heart failure.  Thus, inhibition of apoptosis is an important approach for preventive and therapeutic strategies.  Here we show that the histone 3 lysine 4- and lysine 36-specific methyltransferase Smyd2 acts as an endogenous antagonistic player of p53-dependent cardiomyocyte apoptosis.  Smyd2 protein levels were significantly decreased in cardiomyocytes upon cobalt chloride-induced apoptosis or myocardial infarction, while p53 expression was enhanced.  SiRNA-mediated knockdown of Smyd2 in cultured cardiomyocytes further enhanced cobalt chloride-induced cardiomyocyte apoptosis.  In contrast, Smyd2 overexpression resulted in marked methylation of p53 and prevented its accumulation as well as apoptotic cell death in an Hsp90-independent manner.  Moreover, overexpression, of Smyd2, but not Smyd2Y240F lacking a Me transferase activity, significantly rescued CoCl2-induced apoptosis in H9c2 cardioblasts.  Finally, Smyd2 cardiomyocyte-specific deletion in vivo promoted apoptotic cell death upon myocardial infarction, which correlated with enhanced expression of p53 and pro-apoptotic Bax.  Collectively, our data indicate Smyd2 as a cardioprotective protein by methylating p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfw3CFurADyLVg90H21EOLACvtfcHk0ljKdWZA3qcSyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1yksr3J&md5=327a9367f05026a05ec527e18f71b26f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2014.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2014.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DSajjad%26aufirst%3DA.%26aulast%3DNovoyatleva%26aufirst%3DT.%26aulast%3DVergarajauregui%26aufirst%3DS.%26aulast%3DTroidl%26aufirst%3DC.%26aulast%3DSchermuly%26aufirst%3DR.%2BT.%26aulast%3DTucker%26aufirst%3DH.%2BO.%26aulast%3DEngel%26aufirst%3DF.%2BB.%26atitle%3DLysine%2520methyltransferase%2520Smyd2%2520suppresses%2520p53-dependent%2520cardiomyocyte%2520apoptosis%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26spage%3D2556%26epage%3D2562%26doi%3D10.1016%2Fj.bbamcr.2014.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">Structure of human SMYD2 protein reveals the basis of p53 tumor suppressor methylation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">38725</span><span class="NLM_x">–</span> <span class="NLM_lpage">38737</span><span class="refDoi"> DOI: 10.1074/jbc.M111.262410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1074%2Fjbc.M111.262410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21880715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlykurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=38725-38737&author=L.+Wangauthor=L.+Liauthor=H.+Zhangauthor=X.+Luoauthor=J.+Daiauthor=S.+Zhouauthor=J.+Guauthor=J.+Zhuauthor=P.+Atadjaauthor=C.+Luauthor=E.+Liauthor=K.+Zhao&title=Structure+of+human+SMYD2+protein+reveals+the+basis+of+p53+tumor+suppressor+methylation&doi=10.1074%2Fjbc.M111.262410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human SMYD2 Protein Reveals the Basis of p53 Tumor Suppressor Methylation</span></div><div class="casAuthors">Wang, Li; Li, Ling; Zhang, Hailong; Luo, Xiao; Dai, Jingquan; Zhou, Shaolian; Gu, Justin; Zhu, Jidong; Atadja, Peter; Lu, Chris; Li, En; Zhao, Kehao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">38725-38737, S38725/1-S38725/5</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">SMYD2 belongs to a subfamily of histone lysine methyltransferase and was recently identified to methylate tumor suppressor p53 and Rb.  Here we report that SMYD2 prefers to methylate p53 Lys-370 over histone substrates in vitro.  Consistently, the level of endogenous p53 Lys-370 monomethylation is significantly elevated when SMYD2 is overexpressed in vivo.  We have solved the high resoln. crystal structures of the full-length SMYD2 protein in binary complex with its cofactor S-adenosylmethionine and in ternary complex with cofactor product S-adenosylhomocysteine and p53 substrate peptide (residues 368-375), resp. p53 peptide binds to a deep pocket of the interface between catalytic SET(1-282) and C-terminal domain (CTD) with an unprecedented U-shaped conformation.  Subtle conformational change exists around the p53 binding site between the binary and ternary structures, in particular the tetratricopeptide repeat motif of the CTD.  In addn., a unique EDEE motif between the loop of anti-parallel β7 and β8 sheets of the SET core not only interacts with p53 substrate but also forms a hydrogen bond network with residues from CTD.  These observations suggest that the tetratricopeptide repeat and EDEE motif may play an important role in detg. p53 substrate binding specificity.  This is further verified by the findings that deletion of the CTD domain drastically reduces the methylation activity of SMYD2 to p53 protein.  Meanwhile, mutation of EDEE residues impairs both the binding and the enzymic activity of SMYD2 to p53 Lys-370.  These data together reveal the mol. basis of SMYD2 in specifically recognizing and regulating functions of p53 tumor suppressor through Lys-370 monomethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX6G4Yhf_H47Vg90H21EOLACvtfcHk0ljKdWZA3qcSyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlykurvI&md5=671857a22846de5d607e83a03dd52d37</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.262410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.262410%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DStructure%2520of%2520human%2520SMYD2%2520protein%2520reveals%2520the%2520basis%2520of%2520p53%2520tumor%2520suppressor%2520methylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D38725%26epage%3D38737%26doi%3D10.1074%2Fjbc.M111.262410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theriault, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Code, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birr, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span> </span><span class="NLM_article-title">Biochemical characterization of human SET and MYND domain-containing protein 2 methyltransferase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">6488</span><span class="NLM_x">–</span> <span class="NLM_lpage">6497</span><span class="refDoi"> DOI: 10.1021/bi200725p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200725p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=6488-6497&author=J.+Wuauthor=T.+Cheungauthor=C.+Grandeauthor=A.+D.+Fergusonauthor=X.+Zhuauthor=K.+Theriaultauthor=E.+Codeauthor=C.+Birrauthor=N.+Keenauthor=H.+Chen&title=Biochemical+characterization+of+human+SET+and+MYND+domain-containing+protein+2+methyltransferase&doi=10.1021%2Fbi200725p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fbi200725p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200725p%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DTheriault%26aufirst%3DK.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DBirr%26aufirst%3DC.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DBiochemical%2520characterization%2520of%2520human%2520SET%2520and%2520MYND%2520domain-containing%2520protein%25202%2520methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D6488%26epage%3D6497%26doi%3D10.1021%2Fbi200725p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Barros Filho, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katayama, M. L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brentani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abreu, A. P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Góes, J. C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brentani, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folgueira, M. A. A. K.</span><span> </span><span class="NLM_article-title">Gene trio signatures as molecular markers to predict response to doxorubicin cyclophosphamide neoadjuvant chemotherapy in breast cancer patients</span> <span class="citation_source-journal">Braz. J. Med. Biol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1225</span><span class="NLM_x">–</span> <span class="NLM_lpage">1231</span><span class="refDoi"> DOI: 10.1590/S0100-879X2010007500135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1590%2FS0100-879X2010007500135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21103787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivFehtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=1225-1231&author=M.+C.+Barros+Filhoauthor=M.+L.+H.+Katayamaauthor=H.+Brentaniauthor=A.+P.+S.+Abreuauthor=E.+M.+Barbosaauthor=C.+T.+Oliveiraauthor=J.+C.+S.+G%C3%B3esauthor=M.+M.+Brentaniauthor=M.+A.+A.+K.+Folgueira&title=Gene+trio+signatures+as+molecular+markers+to+predict+response+to+doxorubicin+cyclophosphamide+neoadjuvant+chemotherapy+in+breast+cancer+patients&doi=10.1590%2FS0100-879X2010007500135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Gene trio signatures as molecular markers to predict response to doxorubicin cyclophosphamide neoadjuvant chemotherapy in breast cancer patients</span></div><div class="casAuthors">Barros Filho, M. C.; Katayama, M. L. H.; Brentani, H.; Abreu, A. P. S.; Barbosa, E. M.; Oliveira, C. T.; Goes, J. C. S.; Brentani, M. M.; Folgueira, M. A. A. K.</div><div class="citationInfo"><span class="NLM_cas:title">Brazilian Journal of Medical and Biological Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1225-1231</span>CODEN:
                <span class="NLM_cas:coden">BJMRDK</span>;
        ISSN:<span class="NLM_cas:issn">0100-879X</span>.
    
            (<span class="NLM_cas:orgname">Associacao Brasileira de Divulgacao Cientifica</span>)
        </div><div class="casAbstract">In breast cancer patients submitted to neoadjuvant chemotherapy (4 cycles of doxorubicin and cyclophosphamide, AC), expression of groups of three genes (gene trio signatures) could distinguish responsive from non-responsive tumors, as demonstrated by cDNA microarray profiling in a previous study by our group.  In the current study, we detd. if the expression of the same genes would retain the predictive strength, when analyzed by a more accessible technique (real-time RT-PCR).  We evaluated 28 samples already analyzed by cDNA microarray, as a tech. validation procedure, and 14 tumors, as an independent biol. validation set.  All patients received neoadjuvant chemotherapy (4 AC).  Among five trio combinations previously identified, defined by nine genes individually investigated (BZRP, CLPTM1, MTSS1, NOTCH1, NUP210, PRSS11, RPL37A, SMYD2, and XLHSRF-1), the most accurate were established by RPL37A, XLHSRF-1 based trios, with NOTCH1 or NUP210.  Both trios correctly sepd. 86% of tumors (87% sensitivity and 80% specificity for predicting response), according to their response to chemotherapy (82% in a leave-one-out cross-validation method).  Using the pre-established features obtained by linear discriminant anal., 71% samples from the biol. validation set were also correctly classified by both trios (72% sensitivity; 66% specificity).  Furthermore, we explored other gene combinations to achieve a higher accuracy in the tech. validation group (as a training set).  A new trio, MTSS1, RPL37 and SMYD2, correctly classified 93% of samples from the tech. validation group (95% sensitivity and 80% specificity; 86% accuracy by the cross-validation method) and 79% from the biol. validation group (72% sensitivity and 100% specificity).  Therefore, the combined expression of MTSS1, RPL37 and SMYD2, as evaluated by real-time RT-PCR, is a potential candidate to predict response to neoadjuvant doxorubicin and cyclophosphamide in breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqViAm7L7uXCbVg90H21EOLACvtfcHk0lhL9OKYwLPKdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivFehtL8%253D&md5=71367c342457f24fc678e01854404e96</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1590%2FS0100-879X2010007500135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1590%252FS0100-879X2010007500135%26sid%3Dliteratum%253Aachs%26aulast%3DBarros%2BFilho%26aufirst%3DM.%2BC.%26aulast%3DKatayama%26aufirst%3DM.%2BL.%2BH.%26aulast%3DBrentani%26aufirst%3DH.%26aulast%3DAbreu%26aufirst%3DA.%2BP.%2BS.%26aulast%3DBarbosa%26aufirst%3DE.%2BM.%26aulast%3DOliveira%26aufirst%3DC.%2BT.%26aulast%3DG%25C3%25B3es%26aufirst%3DJ.%2BC.%2BS.%26aulast%3DBrentani%26aufirst%3DM.%2BM.%26aulast%3DFolgueira%26aufirst%3DM.%2BA.%2BA.%2BK.%26atitle%3DGene%2520trio%2520signatures%2520as%2520molecular%2520markers%2520to%2520predict%2520response%2520to%2520doxorubicin%2520cyclophosphamide%2520neoadjuvant%2520chemotherapy%2520in%2520breast%2520cancer%2520patients%26jtitle%3DBraz.%2520J.%2520Med.%2520Biol.%2520Res.%26date%3D2010%26volume%3D43%26spage%3D1225%26epage%3D1231%26doi%3D10.1590%2FS0100-879X2010007500135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trescott, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holcomb, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirinupong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span> </span><span class="NLM_article-title">Structural insights into estrogen receptor alpha methylation by histone methyltransferase SMYD2, a cellular event implicated in estrogen signaling regulation</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">426</span><span class="NLM_x">, </span> <span class="NLM_fpage">3413</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1016/j.jmb.2014.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.jmb.2014.02.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=3413-25&author=Y.+Jiangauthor=L.+Trescottauthor=J.+Holcombauthor=X.+Zhangauthor=J.+Brunzelleauthor=N.+Sirinupongauthor=X.+Shiauthor=Z.+Yang&title=Structural+insights+into+estrogen+receptor+alpha+methylation+by+histone+methyltransferase+SMYD2%2C+a+cellular+event+implicated+in+estrogen+signaling+regulation&doi=10.1016%2Fj.jmb.2014.02.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DTrescott%26aufirst%3DL.%26aulast%3DHolcomb%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBrunzelle%26aufirst%3DJ.%26aulast%3DSirinupong%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DStructural%2520insights%2520into%2520estrogen%2520receptor%2520alpha%2520methylation%2520by%2520histone%2520methyltransferase%2520SMYD2%252C%2520a%2520cellular%2520event%2520implicated%2520in%2520estrogen%2520signaling%2520regulation%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D3413%26epage%3D25%26doi%3D10.1016%2Fj.jmb.2014.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barton, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span> </span><span class="NLM_article-title">Regulation of estrogen receptor alpha by histone methyltransferase SMYD2-mediated protein methylation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">17284</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1073/pnas.1307959110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1073%2Fpnas.1307959110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=17284-9&author=X.+Zhangauthor=K.+Tanakaauthor=J.+Yanauthor=J.+Liauthor=D.+Pengauthor=Y.+Jiangauthor=Z.+Yangauthor=M.+C.+Bartonauthor=H.+Wenauthor=X.+Shi&title=Regulation+of+estrogen+receptor+alpha+by+histone+methyltransferase+SMYD2-mediated+protein+methylation&doi=10.1073%2Fpnas.1307959110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1307959110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1307959110%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DBarton%26aufirst%3DM.%2BC.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DX.%26atitle%3DRegulation%2520of%2520estrogen%2520receptor%2520alpha%2520by%2520histone%2520methyltransferase%2520SMYD2-mediated%2520protein%2520methylation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D17284%26epage%3D9%26doi%3D10.1073%2Fpnas.1307959110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Piao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">The histone methyltransferase SMYD2 methylates PARP1 and promotes poly(ADP-ribosyl)ation activity in cancer cells</span> <span class="citation_source-journal">Neoplasia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">264</span><span class="refDoi"> DOI: 10.1016/j.neo.2014.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.neo.2014.03.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=257-264&author=L.+Piaoauthor=D.+Kangauthor=T.+Suzukiauthor=A.+Masudaauthor=N.+Dohmaeauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=The+histone+methyltransferase+SMYD2+methylates+PARP1+and+promotes+poly%28ADP-ribosyl%29ation+activity+in+cancer+cells&doi=10.1016%2Fj.neo.2014.03.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.neo.2014.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neo.2014.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DPiao%26aufirst%3DL.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMasuda%26aufirst%3DA.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DThe%2520histone%2520methyltransferase%2520SMYD2%2520methylates%2520PARP1%2520and%2520promotes%2520poly%2528ADP-ribosyl%2529ation%2520activity%2520in%2520cancer%2520cells%26jtitle%3DNeoplasia%26date%3D2014%26volume%3D16%26spage%3D257%26epage%3D264%26doi%3D10.1016%2Fj.neo.2014.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Abu-Farha, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanouette, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elisma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tremblay, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figeys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J.-F.</span><span> </span><span class="NLM_article-title">Proteomic analyses of the SMYD family interactomes identify HSP90 as a novel target for SMYD2</span> <span class="citation_source-journal">J. Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">301</span><span class="NLM_x">–</span> <span class="NLM_lpage">308</span><span class="refDoi"> DOI: 10.1093/jmcb/mjr025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1093%2Fjmcb%2Fmjr025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=22028380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFagsL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=301-308&author=M.+Abu-Farhaauthor=S.+Lanouetteauthor=F.+Elismaauthor=V.+Tremblayauthor=J.+Butsonauthor=D.+Figeysauthor=J.-F.+Couture&title=Proteomic+analyses+of+the+SMYD+family+interactomes+identify+HSP90+as+a+novel+target+for+SMYD2&doi=10.1093%2Fjmcb%2Fmjr025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic analyses of the SMYD family interactomes identify HSP90 as a novel target for SMYD2</span></div><div class="casAuthors">Abu-Farha, Mohamed; Lanouette, Sylvain; Elisma, Fred; Tremblay, Veronique; Butson, Jeffery; Figeys, Daniel; Couture, Jean-Francois</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">301-308</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The SMYD (SET and MYND domain) family of lysine methyltransferases (KMTs) plays pivotal roles in various cellular processes, including gene expression regulation and DNA damage response.  Initially identified as genuine histone methyltransferases, specific members of this family have recently been shown to methylate non-histone proteins such as p53, VEGFR, and the retinoblastoma tumor suppressor (pRb).  To gain further functional insights into this family of KMTs, we generated the protein interaction network for three different human SMYD proteins (SMYD2, SMYD3, and SMYD5).  Characterization of each SMYD protein network revealed that they assoc. with both shared and unique sets of proteins.  Among those, we found that HSP90 and several of its co-chaperones interact specifically with the tetratrico peptide repeat (TPR)-contg. SMYD2 and SMYD3.  Moreover, using proteomic and biochem. techniques, we provide evidence that SMYD2 methylates K209 and K615 on HSP90 nucleotide-binding and dimerization domains, resp.  In addn., we found that each methylation site displays unique reactivity in regard to the presence of HSP90 co-chaperones, pH, and demethylation by the lysine amine oxidase LSD1, suggesting that alternative mechanisms control HSP90 methylation by SMYD2.  Altogether, this study highlights the ability of SMYD proteins to form unique protein complexes that may underlie their various biol. functions and the SMYD2-mediated methylation of the key mol. chaperone HSP90.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFwZSHL-gNyLVg90H21EOLACvtfcHk0lhL9OKYwLPKdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFagsL7J&md5=d3c5072e6958afc4f82935980ed4b880</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjr025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjr025%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Farha%26aufirst%3DM.%26aulast%3DLanouette%26aufirst%3DS.%26aulast%3DElisma%26aufirst%3DF.%26aulast%3DTremblay%26aufirst%3DV.%26aulast%3DButson%26aufirst%3DJ.%26aulast%3DFigeys%26aufirst%3DD.%26aulast%3DCouture%26aufirst%3DJ.-F.%26atitle%3DProteomic%2520analyses%2520of%2520the%2520SMYD%2520family%2520interactomes%2520identify%2520HSP90%2520as%2520a%2520novel%2520target%2520for%2520SMYD2%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Biol.%26date%3D2011%26volume%3D3%26spage%3D301%26epage%3D308%26doi%3D10.1093%2Fjmcb%2Fmjr025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Hamamoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakakido, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span> </span><span class="NLM_article-title">SMYD2-dependent HSP90 methylation promotes cancer cell proliferation by regulating the chaperone complex formation</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">351</span><span class="NLM_x">, </span> <span class="NLM_fpage">126</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1016/j.canlet.2014.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.canlet.2014.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=24880080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvVejtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2014&pages=126-133&author=R.+Hamamotoauthor=G.+Toyokawaauthor=M.+Nakakidoauthor=K.+Uedaauthor=Y.+Nakamura&title=SMYD2-dependent+HSP90+methylation+promotes+cancer+cell+proliferation+by+regulating+the+chaperone+complex+formation&doi=10.1016%2Fj.canlet.2014.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">SMYD2-dependent HSP90 methylation promotes cancer cell proliferation by regulating the chaperone complex formation</span></div><div class="casAuthors">Hamamoto, Ryuji; Toyokawa, Gouji; Nakakido, Makoto; Ueda, Koji; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126-133</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Heat shock protein 90 (HSP90) is a highly conserved mol. chaperone that facilitates the maturation of a wide range of proteins, and it has been recognized as a crucial facilitator of oncogene addiction and cancer cell survival.  Although HSP90 function is regulated by a variety of post-translational modifications, the physiol. significance of methylation has not fully been elucidated.  Here we demonstrate that HSP90AB1 is methylated by the histone methyltransferase SMYD2 and that it plays a crit. role in human carcinogenesis.  HSP90AB1 and SMYD2 can interact through the C-terminal region of HSP90AB1 and the SET domain of SMYD2.  Both in vitro and in vivo methyltransferase assays revealed that SMYD2 could methylate HSP90AB1 and mass spectrometry anal. indicated lysines 531 and 574 of HSP90AB1 to be methylated.  These methylation sites were shown to be important for the dimerization and chaperone complex formation of HSP90AB1.  Furthermore, methylated HSP90AB1 accelerated the proliferation of cancer cells.  Our study reveals a novel mechanism for human carcinogenesis via methylation of HSP90AB1 by SMYD2, and addnl. functional studies may assist in developing novel strategies for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhJR6mltsK1LVg90H21EOLACvtfcHk0liSsf5OVR7E5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvVejtLo%253D&md5=c2832d0fd7c14091ed8098a466f13122</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DNakakido%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DSMYD2-dependent%2520HSP90%2520methylation%2520promotes%2520cancer%2520cell%2520proliferation%2520by%2520regulating%2520the%2520chaperone%2520complex%2520formation%26jtitle%3DCancer%2520Lett.%26date%3D2014%26volume%3D351%26spage%3D126%26epage%3D133%26doi%3D10.1016%2Fj.canlet.2014.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Donlin, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andresen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Just, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappas, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, E. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zieseniss, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobenecker, M.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voelkel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chait, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregorio, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rottbauer, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linke, W. A.</span><span> </span><span class="NLM_article-title">Smyd2 controls cytoplasmic lysine methylation of Hsp90 and myofilament organization</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1101/gad.177758.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1101%2Fgad.177758.111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=114-119&author=L.+T.+Donlinauthor=C.+Andresenauthor=S.+Justauthor=E.+Rudenskyauthor=C.+T.+Pappasauthor=M.+Krugerauthor=E.+Y.+Jacobsauthor=A.+Ungerauthor=A.+Ziesenissauthor=M.-W.+Dobeneckerauthor=T.+Voelkelauthor=B.+T.+Chaitauthor=C.+C.+Gregorioauthor=W.+Rottbauerauthor=A.+Tarakhovskyauthor=W.+A.+Linke&title=Smyd2+controls+cytoplasmic+lysine+methylation+of+Hsp90+and+myofilament+organization&doi=10.1101%2Fgad.177758.111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1101%2Fgad.177758.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.177758.111%26sid%3Dliteratum%253Aachs%26aulast%3DDonlin%26aufirst%3DL.%2BT.%26aulast%3DAndresen%26aufirst%3DC.%26aulast%3DJust%26aufirst%3DS.%26aulast%3DRudensky%26aufirst%3DE.%26aulast%3DPappas%26aufirst%3DC.%2BT.%26aulast%3DKruger%26aufirst%3DM.%26aulast%3DJacobs%26aufirst%3DE.%2BY.%26aulast%3DUnger%26aufirst%3DA.%26aulast%3DZieseniss%26aufirst%3DA.%26aulast%3DDobenecker%26aufirst%3DM.-W.%26aulast%3DVoelkel%26aufirst%3DT.%26aulast%3DChait%26aufirst%3DB.%2BT.%26aulast%3DGregorio%26aufirst%3DC.%2BC.%26aulast%3DRottbauer%26aufirst%3DW.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DLinke%26aufirst%3DW.%2BA.%26atitle%3DSmyd2%2520controls%2520cytoplasmic%2520lysine%2520methylation%2520of%2520Hsp90%2520and%2520myofilament%2520organization%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D114%26epage%3D119%26doi%3D10.1101%2Fgad.177758.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Voelkel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andresen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Just, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rottbauer, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linke, W. A.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase Smyd2 regulates Hsp90-mediated protection of the sarcomeric titin springs and cardiac function</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1833</span><span class="NLM_x">, </span> <span class="NLM_fpage">812</span><span class="NLM_x">–</span> <span class="NLM_lpage">822</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2012.09.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.bbamcr.2012.09.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=23047121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ShtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1833&publication_year=2013&pages=812-822&author=T.+Voelkelauthor=C.+Andresenauthor=A.+Ungerauthor=S.+Justauthor=W.+Rottbauerauthor=W.+A.+Linke&title=Lysine+methyltransferase+Smyd2+regulates+Hsp90-mediated+protection+of+the+sarcomeric+titin+springs+and+cardiac+function&doi=10.1016%2Fj.bbamcr.2012.09.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methyltransferase Smyd2 regulates Hsp90-mediated protection of the sarcomeric titin springs and cardiac function</span></div><div class="casAuthors">Voelkel, Tobias; Andresen, Christian; Unger, Andreas; Just, Steffen; Rottbauer, Wolfgang; Linke, Wolfgang A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1833</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">812-822</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein lysine methylation controls gene expression and repair of DNA in the nucleus but also occurs in the cytoplasm, where the role of this posttranslational modification is less understood.  Members of the Smyd protein family of lysine methyltransferases are particularly abundant in the cytoplasm, with Smyd1 and Smyd2 being most highly expressed in the heart and in skeletal muscles.  Smyd1 is a crucial myogenic regulator with histone methyltransferase activity but also assocs. with myosin, which promotes sarcomere assembly.  Smyd2 methylates histones and non-histone proteins, such as the tumor suppressors, p53 and retinoblastoma protein, RB.  Smyd2 has an intriguing function in the cytoplasm of skeletal myocytes, where it methylates the chaperone Hsp90, thus promoting the interaction of a Smyd2-methyl-Hsp90 complex with the N2A-domain of titin.  This complex protects the sarcomeric I-band region and myocyte organization.  We briefly summarize some novel functions of Smyd family members, with a focus on Smyd2, and highlight their role in striated muscles and cytoplasmic actions.  We then provide exptl. evidence that Smyd2 is also important for cardiac function.  In the cytoplasm of cardiomyocytes, Smyd2 was found to assoc. with the sarcomeric I-band region at the titin N2A-domain.  Binding to N2A occurred in vitro and in yeast via N-terminal and extreme C-terminal regions of Smyd2.  Smyd2-knockdown in zebrafish using an antisense oligonucleotide morpholino approach strongly impaired cardiac performance.  We conclude that Smyd2 and presumably several other Smyd family members are lysine methyltransferases which have, next to their nuclear activity, specific regulatory functions in the cytoplasm of heart and skeletal muscle cells.  This article is part of a Special Issue entitled: Cardiomyocyte Biol.: Cardiac Pathways of Differentiation, Metab. and Contraction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAxKXSQCV-GbVg90H21EOLACvtfcHk0liSsf5OVR7E5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ShtrvJ&md5=9da8ecc40bcfe683ebe744404ebd348b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2012.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2012.09.012%26sid%3Dliteratum%253Aachs%26aulast%3DVoelkel%26aufirst%3DT.%26aulast%3DAndresen%26aufirst%3DC.%26aulast%3DUnger%26aufirst%3DA.%26aulast%3DJust%26aufirst%3DS.%26aulast%3DRottbauer%26aufirst%3DW.%26aulast%3DLinke%26aufirst%3DW.%2BA.%26atitle%3DLysine%2520methyltransferase%2520Smyd2%2520regulates%2520Hsp90-mediated%2520protection%2520of%2520the%2520sarcomeric%2520titin%2520springs%2520and%2520cardiac%2520function%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2013%26volume%3D1833%26spage%3D812%26epage%3D822%26doi%3D10.1016%2Fj.bbamcr.2012.09.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piatnitski Chekler, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span> </span><span class="NLM_article-title">Target validation using chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+Piatnitski+Cheklerauthor=L.+H.+Jones&title=Target+validation+using+chemical+probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0liSsf5OVR7E5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DPiatnitski%2BChekler%26aufirst%3DE.%2BL.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520validation%2520using%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buser-Doepner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederiksen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimshaw, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V. M</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller-Fahrnow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Hagan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundström, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo-Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walpole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lhuFjLLLPeIwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antonysamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emtage, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gheyi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">LLY-507, a cell-active, potent, and selective inhibitor of protein-lysine methyltransferase SMYD2</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">13641</span><span class="NLM_x">–</span> <span class="NLM_lpage">13653</span><span class="refDoi"> DOI: 10.1074/jbc.M114.626861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1074%2Fjbc.M114.626861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25825497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFGmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=13641-13653&author=H.+Nguyenauthor=A.+Allali-Hassaniauthor=S.+Antonysamyauthor=S.+Changauthor=L.+H.+Chenauthor=C.+Curtisauthor=S.+Emtageauthor=L.+Fanauthor=T.+Gheyiauthor=F.+Liauthor=S.+Liuauthor=J.+R.+Martinauthor=D.+Mendelauthor=J.+B.+Olsenauthor=L.+Pelletierauthor=T.+Shatsevaauthor=S.+Wuauthor=F.+F.+Zhangauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=R.+M.+Campbellauthor=B.+A.+Garciaauthor=D.+Barsyte-Lovejoyauthor=M.+Maderauthor=M.+Vedadi&title=LLY-507%2C+a+cell-active%2C+potent%2C+and+selective+inhibitor+of+protein-lysine+methyltransferase+SMYD2&doi=10.1074%2Fjbc.M114.626861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-507, a Cell-active, Potent, and Selective Inhibitor of Protein-lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Nguyen, Hannah; Allali-Hassani, Abdellah; Antonysamy, Stephen; Chang, Shawn; Chen, Lisa Hong; Curtis, Carmen; Emtage, Spencer; Fan, Li; Gheyi, Tarun; Li, Fengling; Liu, Shichong; Martin, Joseph R.; Mendel, David; Olsen, Jonathan B.; Pelletier, Laura; Shatseva, Tatiana; Wu, Song; Zhang, Feiyu Fred; Arrowsmith, Cheryl H.; Brown, Peter J.; Campbell, Robert M.; Garcia, Benjamin A.; Barsyte-Lovejoy, Dalia; Mader, Mary; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">13641-13653</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">SMYD2 is a lysine methyltransferase that catalyzes the monomethylation of several protein substrates including p53.  SMYD2 is overexpressed in a significant percentage of esophageal squamous primary carcinomas, and that overexpression correlates with poor patient survival.  However, the mechanism(s) by which SMYD2 promotes oncogenesis is not understood.  A small mol. probe for SMYD2 would allow for the pharmacol. dissection of this biol.  In this report, we disclose LLY-507, a cell-active, potent small mol. inhibitor of SMYD2.  LLY-507 is >100-fold selective for SMYD2 over a broad range of methyltransferase and non-methyltransferase targets.  A 1.63-Å resoln. crystal structure of SMYD2 in complex with LLY-507 shows the inhibitor binding in the substrate peptide binding pocket.  LLY-507 is active in cells as measured by redn. of SMYD2-induced monomethylation of p53 Lys370 at submicromolar concns.  We used LLY-507 to further test other potential roles of SMYD2.  Mass spectrometry-based proteomics showed that cellular global histone methylation levels were not significantly affected by SMYD2 inhibition with LLY-507, and subcellular fractionation studies indicate that SMYD2 is primarily cytoplasmic, suggesting that SMYD2 targets a very small subset of histones at specific chromatin loci and/or non-histone substrates.  Breast and liver cancers were identified through in silico data mining as tumor types that display amplification and/or overexpression of SMYD2.  LLY-507 inhibited the proliferation of several esophageal, liver, and breast cancer cell lines in a dose-dependent manner.  These findings suggest that LLY-507 serves as a valuable chem. probe to aid in the dissection of SMYD2 function in cancer and other biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb5tVOe0M2gLVg90H21EOLACvtfcHk0lhuFjLLLPeIwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFGmtrY%253D&md5=a75f59a900cfe7a6a04d35498caae78b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626861%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DCurtis%26aufirst%3DC.%26aulast%3DEmtage%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DJ.%2BR.%26aulast%3DMendel%26aufirst%3DD.%26aulast%3DOlsen%26aufirst%3DJ.%2BB.%26aulast%3DPelletier%26aufirst%3DL.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DF.%2BF.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DLLY-507%252C%2520a%2520cell-active%252C%2520potent%252C%2520and%2520selective%2520inhibitor%2520of%2520protein-lysine%2520methyltransferase%2520SMYD2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D13641%26epage%3D13653%26doi%3D10.1074%2Fjbc.M114.626861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Algire, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakob, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maag, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">Discovery of A-893, a new cell-active benzoxazinone inhibitor of lysine methyltransferase SMYD2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">695</span><span class="NLM_x">–</span> <span class="NLM_lpage">700</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00124</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00124" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsVSqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=695-700&author=R.+F.+Sweisauthor=Z.+Wangauthor=M.+Algireauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=G.+G.+Chiangauthor=J.+Guoauthor=C.+G.+Jakobauthor=S.+Kennedyauthor=F.+Liauthor=D.+Maagauthor=B.+Shawauthor=N.+B.+Soniauthor=M.+Vedadiauthor=W.+N.+Pappano&title=Discovery+of+A-893%2C+a+new+cell-active+benzoxazinone+inhibitor+of+lysine+methyltransferase+SMYD2&doi=10.1021%2Facsmedchemlett.5b00124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of A-893, A New Cell-Active Benzoxazinone Inhibitor of Lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Sweis, Ramzi F.; Wang, Zhi; Algire, Mikkel; Arrowsmith, Cheryl H.; Brown, Peter J.; Chiang, Gary G.; Guo, Jun; Jakob, Clarissa G.; Kennedy, Steven; Li, Fengling; Maag, David; Shaw, Bailin; Soni, Nirupama B.; Vedadi, Masoud; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">695-700</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A lack of useful small mol. tools has precluded thorough interrogation of the biol. function of SMYD2, a lysine methyltransferase with known tumor-suppressor substrates.  Systematic exploration of the structure-activity relationships of a previously known benzoxazinone compd. led to the synthesis of A-893 I, a potent and selective SMYD2 inhibitor (IC50: 2.8 nM).  A cocrystal structure reveals the origin of enhanced potency, and effective suppression of p53K370 methylation is obsd. in a lung carcinoma (A549) cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI0i3TmEl2DLVg90H21EOLACvtfcHk0ljTu22dgBJMuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsVSqtb8%253D&md5=4e37d0d524f1b1066a4ef7ba03dc4695</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00124%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DAlgire%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520of%2520A-893%252C%2520a%2520new%2520cell-active%2520benzoxazinone%2520inhibitor%2520of%2520lysine%2520methyltransferase%2520SMYD2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D695%26epage%3D700%26doi%3D10.1021%2Facsmedchemlett.5b00124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span>BAY-598 A selective chemical probe for SMYD2; <span class="NLM_publisher-name">Structural Genomics Consortium</span>: <span class="NLM_publisher-loc">Toronto</span>; <a href="http://www.thesgc.org/chemical-probes/BAY-598" class="extLink">http://www.thesgc.org/chemical-probes/BAY-598</a> (accessed Mar 9, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BAY-598+A+selective+chemical+probe+for+SMYD2%3B+Structural+Genomics+Consortium%3A+Toronto%3B+http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FBAY-598+%28accessed+Mar+9%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DBAY-598%2520A%2520selective%2520chemical%2520probe%2520for%2520SMYD2%26pub%3DStructural%2520Genomics%2520Consortium" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Reynoird, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazur, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stellfeld, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flores, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lofgren, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brambilla, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hainaut, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaznowska, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khatri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stresemann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, J.</span><span> </span><span class="NLM_article-title">Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">772</span><span class="NLM_x">–</span> <span class="NLM_lpage">785</span><span class="refDoi"> DOI: 10.1101/gad.275529.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1101%2Fgad.275529.115" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=772-785&author=N.+Reynoirdauthor=P.+K.+Mazurauthor=T.+Stellfeldauthor=N.+M.+Floresauthor=S.+M.+Lofgrenauthor=S.+M.+Carlsonauthor=E.+Brambillaauthor=P.+Hainautauthor=E.+B.+Kaznowskaauthor=C.+H.+Arrowsmithauthor=P.+Khatriauthor=C.+Stresemannauthor=O.+Gozaniauthor=J.+Sage&title=Coordination+of+stress+signals+by+the+lysine+methyltransferase+SMYD2+promotes+pancreatic+cancer&doi=10.1101%2Fgad.275529.115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1101%2Fgad.275529.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.275529.115%26sid%3Dliteratum%253Aachs%26aulast%3DReynoird%26aufirst%3DN.%26aulast%3DMazur%26aufirst%3DP.%2BK.%26aulast%3DStellfeld%26aufirst%3DT.%26aulast%3DFlores%26aufirst%3DN.%2BM.%26aulast%3DLofgren%26aufirst%3DS.%2BM.%26aulast%3DCarlson%26aufirst%3DS.%2BM.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DHainaut%26aufirst%3DP.%26aulast%3DKaznowska%26aufirst%3DE.%2BB.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKhatri%26aufirst%3DP.%26aulast%3DStresemann%26aufirst%3DC.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DSage%26aufirst%3DJ.%26atitle%3DCoordination%2520of%2520stress%2520signals%2520by%2520the%2520lysine%2520methyltransferase%2520SMYD2%2520promotes%2520pancreatic%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2016%26volume%3D30%26spage%3D772%26epage%3D785%26doi%3D10.1101%2Fgad.275529.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Hieda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukita, S.</span><span> </span><span class="NLM_article-title">A new high molecular mass protein showing unique localization in desmosomal plaque</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">1511</span><span class="NLM_x">–</span> <span class="NLM_lpage">1518</span><span class="refDoi"> DOI: 10.1083/jcb.109.4.1511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1083%2Fjcb.109.4.1511" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=1989&pages=1511-1518&author=Y.+Hiedaauthor=S.+Tsukitaauthor=S.+Tsukita&title=A+new+high+molecular+mass+protein+showing+unique+localization+in+desmosomal+plaque&doi=10.1083%2Fjcb.109.4.1511"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1083%2Fjcb.109.4.1511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.109.4.1511%26sid%3Dliteratum%253Aachs%26aulast%3DHieda%26aufirst%3DY.%26aulast%3DTsukita%26aufirst%3DS.%26aulast%3DTsukita%26aufirst%3DS.%26atitle%3DA%2520new%2520high%2520molecular%2520mass%2520protein%2520showing%2520unique%2520localization%2520in%2520desmosomal%2520plaque%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D1989%26volume%3D109%26spage%3D1511%26epage%3D1518%26doi%3D10.1083%2Fjcb.109.4.1511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Tarcsay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keserű, G. M.</span><span> </span><span class="NLM_article-title">Is there a link between selectivity and binding thermodynamics profiles?</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.drudis.2014.09.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25263698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=86-94&author=A.+Tarcsayauthor=G.+M.+Keser%C5%B1&title=Is+there+a+link+between+selectivity+and+binding+thermodynamics+profiles%3F&doi=10.1016%2Fj.drudis.2014.09.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Is there a link between selectivity and binding thermodynamics profiles?</span></div><div class="casAuthors">Tarcsay, Akos; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-94</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Thermodn. of ligand binding is influenced by the interplay between enthalpy and entropy contributions of the binding event.  The impact of these binding free energy components, however, is not limited to the primary target only.  Here, we investigate the relationship between binding thermodn. and selectivity profiles by combining publicly available data from broad off-target assay profiling and the corresponding thermodn. measurements.  Our anal. indicates that compds. binding their primary targets with higher entropy contributions tend to hit more off-targets compared with those ligands that demonstrated enthalpy-driven binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ3yzea7OANrVg90H21EOLACvtfcHk0ljTu22dgBJMuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt7rO&md5=4b5fe0131781069dc3a4883e8b66ec4e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.09.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3DA.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DIs%2520there%2520a%2520link%2520between%2520selectivity%2520and%2520binding%2520thermodynamics%2520profiles%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D86%26epage%3D94%26doi%3D10.1016%2Fj.drudis.2014.09.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Ladbury, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freire, E.</span><span> </span><span class="NLM_article-title">Adding calorimetric data to decision making in lead discovery: a hot tip</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1038/nrd3054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnrd3054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19960014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWjt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=23-27&author=J.+E.+Ladburyauthor=G.+Klebeauthor=E.+Freire&title=Adding+calorimetric+data+to+decision+making+in+lead+discovery%3A+a+hot+tip&doi=10.1038%2Fnrd3054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Adding calorimetric data to decision making in lead discovery: A hot tip</span></div><div class="casAuthors">Ladbury, John E.; Klebe, Gerhard; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-27</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recognition of the limitations of high-throughput screening approaches in the discovery of candidate drugs has reawakened interest in structure-based and other rational design methods.  Here, the authors describe how isothermal titrn. calorimetry can be used to obtain thermodn. data on the binding of compds. to protein targets.  The authors propose that these data - particularly the change in enthalpy - could provide a valuable, complementary addn. to established tools for selecting compds. in lead discovery and for aiding lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph3qvp2E8fbLVg90H21EOLACvtfcHk0lih2rJhsoNXog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWjt7bM&md5=e8446b6db983c35580e9010899eb40eb</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrd3054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3054%26sid%3Dliteratum%253Aachs%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DAdding%2520calorimetric%2520data%2520to%2520decision%2520making%2520in%2520lead%2520discovery%253A%2520a%2520hot%2520tip%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D23%26epage%3D27%26doi%3D10.1038%2Fnrd3054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Macfarlane, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seatter, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plevin, R.</span><span> </span><span class="NLM_article-title">Proteinase-activated receptors</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">245</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2001&pages=245-282&author=S.+R.+Macfarlaneauthor=M.+J.+Seatterauthor=T.+Kankeauthor=G.+D.+Hunterauthor=R.+Plevin&title=Proteinase-activated+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMacfarlane%26aufirst%3DS.%2BR.%26aulast%3DSeatter%26aufirst%3DM.%2BJ.%26aulast%3DKanke%26aufirst%3DT.%26aulast%3DHunter%26aufirst%3DG.%2BD.%26aulast%3DPlevin%26aufirst%3DR.%26atitle%3DProteinase-activated%2520receptors%26jtitle%3DPharmacol.%2520Rev.%26date%3D2001%26volume%3D53%26spage%3D245%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Leger, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covic, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuliopulos, A.</span><span> </span><span class="NLM_article-title">Protease-activated receptors in cardiovascular diseases</span> <span class="citation_source-journal">Circulation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">1070</span><span class="NLM_x">–</span> <span class="NLM_lpage">1077</span><span class="refDoi"> DOI: 10.1161/CIRCULATIONAHA.105.574830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1161%2FCIRCULATIONAHA.105.574830" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2006&pages=1070-1077&author=A.+J.+Legerauthor=L.+Covicauthor=A.+Kuliopulos&title=Protease-activated+receptors+in+cardiovascular+diseases&doi=10.1161%2FCIRCULATIONAHA.105.574830"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1161%2FCIRCULATIONAHA.105.574830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCULATIONAHA.105.574830%26sid%3Dliteratum%253Aachs%26aulast%3DLeger%26aufirst%3DA.%2BJ.%26aulast%3DCovic%26aufirst%3DL.%26aulast%3DKuliopulos%26aufirst%3DA.%26atitle%3DProtease-activated%2520receptors%2520in%2520cardiovascular%2520diseases%26jtitle%3DCirculation%26date%3D2006%26volume%3D114%26spage%3D1070%26epage%3D1077%26doi%3D10.1161%2FCIRCULATIONAHA.105.574830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Allerheiligen, S.; Brohm, D.; Diedrichs, N.; Hübsch, W.; Fröhlen, B.-N.; Gerdes, C.; Gnoth, M. J.; Perzborn, E.; Vöhringer, V.</span> (Bayer HeathCare AG) <span> </span><span class="NLM_article-title">Cyanoguanidine-substituted pyrazolines</span>. WO2006072350,<span class="NLM_x"> </span><span class="NLM_year">2006</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=S.+Allerheiligen&author=D.+Brohm&author=N.+Diedrichs&author=W.+H%C3%BCbsch&author=B.-N.+Fr%C3%B6hlen&author=C.+Gerdes&author=M.+J.+Gnoth&author=E.+Perzborn&author=V.+V%C3%B6hringer&title=Cyanoguanidine-substituted+pyrazolines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAllerheiligen%26aufirst%3DS.%26atitle%3DCyanoguanidine-substituted%2520pyrazolines%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Abad-Zapatero, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, J. T.</span><span> </span><span class="NLM_article-title">Ligand efficiency indices as guideposts for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">464</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03386-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2FS1359-6446%2805%2903386-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=15809192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A280%3ADC%252BD2M7nsF2iug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=464-469&author=C.+Abad-Zapateroauthor=J.+T.+Metz&title=Ligand+efficiency+indices+as+guideposts+for+drug+discovery&doi=10.1016%2FS1359-6446%2805%2903386-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency indices as guideposts for drug discovery</span></div><div class="casAuthors">Abad-Zapatero Cele; Metz James T</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">464-9</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSHqVfZXUq224qdXsLcGTEUfW6udTcc2eZpMROJc1_4Krntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M7nsF2iug%253D%253D&md5=1720f263a306122c86a57463e5a39287</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903386-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903386-6%26sid%3Dliteratum%253Aachs%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26atitle%3DLigand%2520efficiency%2520indices%2520as%2520guideposts%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D464%26epage%3D469%26doi%3D10.1016%2FS1359-6446%2805%2903386-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span>Schrödinger Suite 2012 Update 2, WaterMap, version 1.4.518; <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schr%C3%B6dinger+Suite+2012+Update+2%2C+WaterMap%2C+version+1.4.518%3B+Schr%C3%B6dinger+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%2520Suite%25202012%2520Update%25202%252C%2520WaterMap%252C%2520version%25201.4.518%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Young, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">808</span><span class="NLM_x">–</span> <span class="NLM_lpage">813</span><span class="refDoi"> DOI: 10.1073/pnas.0610202104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1073%2Fpnas.0610202104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=17204562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=808-813&author=T.+Youngauthor=R.+Abelauthor=B.+Kimauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Motifs+for+molecular+recognition+exploiting+hydrophobic+enclosure+in+protein-ligand+binding&doi=10.1073%2Fpnas.0610202104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span></div><div class="casAuthors">Young, Tom; Abel, Robert; Kim, Byungchan; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The thermodn. properties and phase behavior of water in confined regions can vary significantly from that obsd. in the bulk.  This is particularly true for systems in which the confinement is on the mol.-length scale.  In this study, we use mol. dynamics simulations and a powerful solvent anal. technique based on inhomogeneous solvation theory to investigate the properties of water mols. that solvate the confined regions of protein active sites.  Our simulations and anal. indicate that the solvation of protein active sites that are characterized by hydrophobic enclosure and correlated hydrogen bonds induce atypical entropic and enthalpic penalties of hydration.  These penalties apparently stabilize the protein-ligand complex with respect to the independently solvated ligand and protein, which leads to enhanced binding affinities.  Our anal. elucidates several challenging cases, including the super affinity of the streptavidin-biotin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWumQ61YTqILVg90H21EOLACvtfcHk0lg3YLMNcQsuiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D&md5=1109e3f48670cd910885f7fd527eb880</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0610202104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0610202104%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMotifs%2520for%2520molecular%2520recognition%2520exploiting%2520hydrophobic%2520enclosure%2520in%2520protein-ligand%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D808%26epage%3D813%26doi%3D10.1073%2Fpnas.0610202104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Cheng, Y.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (K1) and the concentration of inhibitor which causes 50% inhibition (I50) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.-C.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28K1%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lg3YLMNcQsuiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.-C.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528K1%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Mechanistic considerations in high-throughput screening</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.1016/S0003-2697(03)00346-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2FS0003-2697%2803%2900346-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2003&pages=1-12&author=R.+A.+Copeland&title=Mechanistic+considerations+in+high-throughput+screening&doi=10.1016%2FS0003-2697%2803%2900346-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FS0003-2697%2803%2900346-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0003-2697%252803%252900346-4%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DMechanistic%2520considerations%2520in%2520high-throughput%2520screening%26jtitle%3DAnal.%2520Biochem.%26date%3D2003%26volume%3D320%26spage%3D1%26epage%3D12%26doi%3D10.1016%2FS0003-2697%2803%2900346-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Chan-Penebre, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuplast, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraj, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stickland, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribich, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbash, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nurse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oza, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallagher, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lg3YLMNcQsuiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0ljCrbRAsB4eEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Olsen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span> </span><span class="NLM_article-title">Quantitative profiling of the activity of protein lysine methyltransferase SMYD2 using SILAC-based proteomics</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">892</span><span class="NLM_x">–</span> <span class="NLM_lpage">905</span><span class="refDoi"> DOI: 10.1074/mcp.M115.053280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1074%2Fmcp.M115.053280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=26750096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVGrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=892-905&author=J.+B.+Olsenauthor=X.+J.+Caoauthor=B.+Hanauthor=L.+H.+Chenauthor=A.+Horvathauthor=T.+I.+Richardsonauthor=R.+M.+Campbellauthor=B.+A.+Garciaauthor=H.+Nguyen&title=Quantitative+profiling+of+the+activity+of+protein+lysine+methyltransferase+SMYD2+using+SILAC-based+proteomics&doi=10.1074%2Fmcp.M115.053280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative profiling of the activity of protein lysine methyltransferase SMYD2 using SILAC-based proteomics</span></div><div class="casAuthors">Olsen, Jonathan B.; Cao, Xing-Jun; Han, Bomie; Chen, Lisa Hong; Horvath, Alexander; Richardson, Timothy I.; Campbell, Robert M.; Garcia, Benjamin A.; Nguyen, Hannah</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-905</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The significance of non-histone lysine methylation in cell biol. and human disease is an emerging area of research exploration.  The development of small mol. inhibitors that selectively and potently target enzymes that catalyze the addn. of methyl-groups to lysine residues, such as the protein lysine mono-methyltransferase SMYD2, is an active area of drug discovery.  Crit. to the accurate assessment of biol. function is the ability to identify target enzyme substrates and to define enzyme substrate specificity within the context of the cell.  Here, using stable isotopic labeling with amino acids in cell culture (SILAC) coupled with immunoaffinity enrichment of mono-methyl-lysine (Kme1) peptides and mass spectrometry, we report a comprehensive, large-scale proteomic study of lysine mono-methylation, comprising a total of 1032 Kme1 sites in esophageal squamous cell carcinoma (ESCC) cells and 1861 Kme1 sites in ESCC cells overexpressing SMYD2.  Among these Kme1 sites is a subset of 35 found to be potently down-regulated by both shRNA-mediated knockdown of SMYD2 and LLY-507, a selective small mol. inhibitor of SMYD2.  In addn., we report specific protein sequence motifs enriched in Kme1 sites that are directly regulated by endogenous SMYD2 activity, revealing that SMYD2 substrate specificity is more diverse than expected.  We further show direct activity of SMYD2 toward BTF3-K2, PDAP1-K126 as well as numerous sites within the repetitive units of two unique and exceptionally large proteins, AHNAK and AHNAK2.  Collectively, our findings provide quant. insights into the cellular activity and substrate recognition of SMYD2 as well as the global landscape and regulation of protein mono-methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIbAGzEvpHn7Vg90H21EOLACvtfcHk0ljCrbRAsB4eEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVGrurs%253D&md5=ebc0c9ca87e73f98b82cb1e3c10d1c71</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M115.053280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M115.053280%26sid%3Dliteratum%253Aachs%26aulast%3DOlsen%26aufirst%3DJ.%2BB.%26aulast%3DCao%26aufirst%3DX.%2BJ.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DHorvath%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DT.%2BI.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DNguyen%26aufirst%3DH.%26atitle%3DQuantitative%2520profiling%2520of%2520the%2520activity%2520of%2520protein%2520lysine%2520methyltransferase%2520SMYD2%2520using%2520SILAC-based%2520proteomics%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2016%26volume%3D15%26spage%3D892%26epage%3D905%26doi%3D10.1074%2Fmcp.M115.053280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Z.</span><span> </span><span class="NLM_article-title">Defining balanced conditions for inhibitor screening assays that target bisubstrate enzymes</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span><span class="refDoi"> DOI: 10.1177/1087057108328763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1177%2F1087057108328763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19196704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFWksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=111-120&author=J.+Yangauthor=R.+A.+Copelandauthor=Z.+Lai&title=Defining+balanced+conditions+for+inhibitor+screening+assays+that+target+bisubstrate+enzymes&doi=10.1177%2F1087057108328763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Defining balanced conditions for inhibitor screening assays that target bisubstrate enzymes</span></div><div class="casAuthors">Yang, Jingsong; Copeland, Robert A.; Lai, Zhihong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-120</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is a common mechanism for identifying lead compds. for drug discovery efforts.  Small mols. can inhibit enzymes by a variety of mechanisms, such as competitive, noncompetitive, and uncompetitive with respect to the substrate(s) of the catalytic reaction.  To optimize the chances of finding the broadest diversity of inhibitor modalities during screening, one must run assays under "balanced" conditions where the potency of inhibitors with various modes of action falls within a similar range.  When an enzyme reaction involves more than one substrate, the definition and assessment of the apparent potency of inhibitors (IC50), in relation to their true potency (Ki), can be nontrivial.  This article provides a theor. anal., on the basis of the Cheng-Prusoff derivation, of the IC50/Ki relationship of bisubstrate enzyme reactions following various sequential kinetic mechanisms, as well as the application and limitations of this information for defining optimal screening conditions for such enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM9O6ZD0r6l7Vg90H21EOLACvtfcHk0ljCrbRAsB4eEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFWksr4%253D&md5=076f5a6ad75c4fc0e01d59ca5ea53fe2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1177%2F1087057108328763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108328763%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DLai%26aufirst%3DZ.%26atitle%3DDefining%2520balanced%2520conditions%2520for%2520inhibitor%2520screening%2520assays%2520that%2520target%2520bisubstrate%2520enzymes%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D111%26epage%3D120%26doi%3D10.1177%2F1087057108328763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">1842</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2015.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.bbagen.2015.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=26002201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=1842-1848&author=M.+S.+Eramauthor=E.+Kuznetsovaauthor=F.+Liauthor=E.+Lima-Fernandesauthor=S.+Kennedyauthor=I.+Chauauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=Kinetic+characterization+of+human+histone+H3+lysine+36+methyltransferases%2C+ASH1L+and+SETD2&doi=10.1016%2Fj.bbagen.2015.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span></div><div class="casAuthors">Eram, Mohammad S.; Kuznetsova, Ekaterina; Li, Fengling; Lima-Fernandes, Evelyne; Kennedy, Steven; Chau, Irene; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1842-1848</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dysregulation of methylation of lysine 36 on histone H3 (H3K36) have been implicated in a variety of diseases including cancers.  ASH1L and SETD2 are two enzymes among others that catalyze H3K36 methylation.  H3K4 methylation has also been reported for ASH1L.  Radioactivity-based enzyme assays, Western and immunoblotting using specific antibodies and mol. modeling were used to characterize substrate specificity of ASH1L and SETD2.  Here we report on the assay development and kinetic characterization of ASH1L and SETD2 and their substrate specificities in vitro.  Both enzymes were active with recombinant nucleosome as substrate.  However, SETD2 but not ASH1L methylated histone peptides as well, indicating that the interaction of the basic post-SET extension with substrate may not be crit. for SETD2 activity.  Neither enzyme was active with nucleosome contg. a H3K36A mutation, indicating their specificity for H3K36.  Analyzing the methylation state of the products of ASH1L and SETD2 reactions also confirmed that both enzymes mono- and dimethylate H3K36 and are inactive with H3K4 as substrate, and that only SETD2 is able to trimethylate H3K36 in vitro.  We detd. the kinetic parameters for ASH1L and SETD2 activity, enabling screening for inhibitors that can be used to further investigate the roles of these two proteins in health and disease.  Both ASH1L and SETD2 are H3K36-specific methyltransferases but only SETD2 can trimethylate this mark.  The basic post-SET extension is crit. for ASH1L but not SETD2 activity.  We provide full kinetic characterization of ASH1L and SETD2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproBY7GkmOLrVg90H21EOLACvtfcHk0lg7WvH3UI40BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D&md5=64839475d5ee9cc2f20e6e71f8e8a6ea</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2015.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2015.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DKinetic%2520characterization%2520of%2520human%2520histone%2520H3%2520lysine%252036%2520methyltransferases%252C%2520ASH1L%2520and%2520SETD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D1842%26epage%3D1848%26doi%3D10.1016%2Fj.bbagen.2015.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lg7WvH3UI40BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lg7WvH3UI40BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lg8DvGzeKIdAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lg8DvGzeKIdAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Schuttelkopf, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Aalten, D. M.</span><span> </span><span class="NLM_article-title">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1355</span><span class="NLM_x">–</span> <span class="NLM_lpage">1363</span><span class="refDoi"> DOI: 10.1107/S0907444904011679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444904011679" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=15272157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A280%3ADC%252BD2czmslGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1355-1363&author=A.+W.+Schuttelkopfauthor=D.+M.+van+Aalten&title=PRODRG%3A+a+tool+for+high-throughput+crystallography+of+protein-ligand+complexes&doi=10.1107%2FS0907444904011679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span></div><div class="casAuthors">Schuttelkopf Alexander W; van Aalten Daan M F</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">Pt 8</span>),
    <span class="NLM_cas:pages">1355-63</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">The small-molecule topology generator PRODRG is described, which takes input from existing coordinates or various two-dimensional formats and automatically generates coordinates and molecular topologies suitable for X-ray refinement of protein-ligand complexes.  Test results are described for automatic generation of topologies followed by energy minimization for a subset of compounds from the Cambridge Structural Database, which shows that, within the limits of the empirical GROMOS87 force field used, structures with good geometries are generated.  X-ray refinement in X-PLOR/CNS, REFMAC and SHELX using PRODRG-generated topologies produces results comparable to refinement with topologies from the standard libraries.  However, tests with distorted starting coordinates show that PRODRG topologies perform better, both in terms of ligand geometry and of crystallographic R factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdNRIcoht5aXy-c-E1v-fmfW6udTcc2eZAd8xMPiJpuLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czmslGmtA%253D%253D&md5=1adce6cd2cacdace513a69efa33c4e93</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444904011679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904011679%26sid%3Dliteratum%253Aachs%26aulast%3DSchuttelkopf%26aufirst%3DA.%2BW.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%26atitle%3DPRODRG%253A%2520a%2520tool%2520for%2520high-throughput%2520crystallography%2520of%2520protein-ligand%2520complexes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D1355%26epage%3D1363%26doi%3D10.1107%2FS0907444904011679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lg8DvGzeKIdAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span>Epik, <span class="NLM_edition">version 2.3.518</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Epik%2C+version+2.3.518%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DEpik%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span>Impact, <span class="NLM_edition">version 5.8.518</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Impact%2C+version+5.8.518%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DImpact%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span>Prime, <span class="NLM_edition">version 3.1.518</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Prime%2C+version+3.1.518%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPrime%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span>Schrödinger Suite 2012 Update 2, Protein Preparation Wizard; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schr%C3%B6dinger+Suite+2012+Update+2%2C+Protein+Preparation+Wizard%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%2520Suite%25202012%2520Update%25202%252C%2520Protein%2520Preparation%2520Wizard%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span> </span><span class="NLM_article-title">The OPLS [optimized potentials for liquid simulations] potential functions for proteins, energy minimizations for crystals of cyclic peptides and crambin</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">1657</span><span class="NLM_x">–</span> <span class="NLM_lpage">1666</span><span class="refDoi"> DOI: 10.1021/ja00214a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00214a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=1657-1666&author=W.+L.+Jorgensenauthor=J.+Tirado-Rives&title=The+OPLS+%5Boptimized+potentials+for+liquid+simulations%5D+potential+functions+for+proteins%2C+energy+minimizations+for+crystals+of+cyclic+peptides+and+crambin&doi=10.1021%2Fja00214a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The OPLS [optimized potentials for liquid simulations] potential functions for proteins, energy minimizations for crystals of cyclic peptides and crambin</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1657-66</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A complete set of intermol. potential functions was developed for use in computer simulations of proteins in their native environment.  Parameters are reported for 25 peptide residues as well as the common neutral and charged terminal groups.  The potential functions have the simple Coulomb plus Lennard-Jones form and are compatible with the widely used models for water, TIP4P, TIP3P, and SPC.  The parameters were obtained and tested primarily in conjunction with Monte Carlo statistical mechanics simulations of 36 pure org. liqs. and numerous aq. solns. of org. ions representative of subunits in the side chains and backbones of proteins.  Bond stretch, angle bend, and torsional terms were adopted from the AMBER united-atom force field.  As reported here, further testing involved studies of conformational energy surfaces and optimizations of the crystal structures for 4 cyclic hexapeptides and a cyclic pentapeptide.  The av. root mean square deviation from the x-ray structures of the crystals is only 0.17 Å for the at. positions and 3% for the unit cell vols.  A more crit. test was then provided by performing energy minimizations for the complete crystal of the protein crambin, including 182 water mols. that were initially placed via a Monte Carlo simulation.  The resultant root mean square deviation for the non-H atoms is still only 0.17 Å and the variation in the errors for charged, polar, and nonpolar residues is small.  Significant improvement is apparent over the AMBER united-atom force field which was previously demonstrated to be superior to many alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ6an38QWApbVg90H21EOLACvtfcHk0lgtAyUGX5ZiOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D&md5=b6c901d8c295b3b37329a7faef527e12</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja00214a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00214a001%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DThe%2520OPLS%2520%255Boptimized%2520potentials%2520for%2520liquid%2520simulations%255D%2520potential%2520functions%2520for%2520proteins%252C%2520energy%2520minimizations%2520for%2520crystals%2520of%2520cyclic%2520peptides%2520and%2520crambin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D1657%26epage%3D1666%26doi%3D10.1021%2Fja00214a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maxwell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span> </span><span class="NLM_article-title">Development and testing of the OPLS all-atom force field on conformational energetics and properties of organic liquids</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">11225</span><span class="NLM_x">–</span> <span class="NLM_lpage">11236</span><span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+L.+Jorgensenauthor=D.+S.+Maxwellauthor=J.+Tirado-Rives&title=Development+and+testing+of+the+OPLS+all-atom+force+field+on+conformational+energetics+and+properties+of+organic+liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0lgtAyUGX5ZiOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DMaxwell%26aufirst%3DD.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520the%2520OPLS%2520all-atom%2520force%2520field%2520on%2520conformational%2520energetics%2520and%2520properties%2520of%2520organic%2520liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span><span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lgtAyUGX5ZiOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Lazaridis, T.</span><span> </span><span class="NLM_article-title">Inhomogeneous fluid approach to solvation thermodynamics. 1. Theory</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3531</span><span class="NLM_x">–</span> <span class="NLM_lpage">3541</span><span class="refDoi"> DOI: 10.1021/jp9723574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9723574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3531-3541&author=T.+Lazaridis&title=Inhomogeneous+fluid+approach+to+solvation+thermodynamics.+1.+Theory&doi=10.1021%2Fjp9723574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Inhomogeneous Fluid Approach to Solvation Thermodynamics. 1. Theory</span></div><div class="casAuthors">Lazaridis, Themis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3531-3541</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The solvation energy and entropy at infinite diln. consist of a solute-solvent term and a solvent reorganization term representing the contributions of changes in solvent structure upon solute insertion.  In the std., homogeneous treatment of solns., changes in solvent structure are expressed through derivs. of the homogeneous pair correlation function, which are very difficult to obtain by simulation.  Tractable expressions for the solvation energy and entropy are here derived by viewing the soln. as an inhomogeneous system with the solute fixed at a certain point.  The solvent reorganization terms in the inhomogeneous view contain two contributions: the local, "correlation" contributions, which are due to correlations between the solute and the solvent and dominate at high densities, and the nonlocal, "liberation" contributions, which are due to the effective diln. of the solvent caused by the thermal motion of the solute and dominate at low densities.  The liberation contributions are independent of the nature or size of the solute and depend only on the properties of the solvent.  For common liq. solvents the nonlocal terms are negligible and the solvation properties arise almost entirely from effects localized around the solute.  The new expressions are tested by calcns. of the solvent reorganization energy and entropy in ideal hard-sphere and Lennard-Jones mixts. (solute identical to solvent).  The solvent reorganization energy and entropy make distinct and significant contributions to the solvation free energy.  The theory can be applied to truly inhomogeneous systems as well as small solute solvation, thus providing a connection between interfacial phenomena and microscopic solvation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxEN_nWI8WfbVg90H21EOLACvtfcHk0livgnB0dEoDkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOltr8%253D&md5=99755c87ffc33abd5e7df89c9996a665</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjp9723574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9723574%26sid%3Dliteratum%253Aachs%26aulast%3DLazaridis%26aufirst%3DT.%26atitle%3DInhomogeneous%2520fluid%2520approach%2520to%2520solvation%2520thermodynamics.%25201.%2520Theory%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3531%26epage%3D3541%26doi%3D10.1021%2Fjp9723574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 38 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Hao Zhang, Ziyan Wei, Ai Hua Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Shouyun Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Access to Cyanoimines Enabled by Dual Photoredox/Copper-Catalyzed Cyanation of O-Acyl Oximes. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (18)
                                     , 7315-7320. <a href="https://doi.org/10.1021/acs.orglett.0c02659" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c02659</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c02659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c02659%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DAccess%252Bto%252BCyanoimines%252BEnabled%252Bby%252BDual%252BPhotoredox%25252FCopper-Catalyzed%252BCyanation%252Bof%252BO-Acyl%252BOximes%26aulast%3DZhang%26aufirst%3DHao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09082020%26date%3D31082020%26volume%3D22%26issue%3D18%26spage%3D7315%26epage%3D7320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Julien Lefranc, Volker Klaus Schulze, Roman Christian Hillig, Hans Briem, Florian Prinz, Anne Mengel, Tobias Heinrich, Jozsef Balint, Srinivasan Rengachari, Horst Irlbacher, Detlef Stöckigt, Ulf Bömer, Benjamin Bader, Stefan Nikolaus Gradl, Carl Friedrich Nising, Franz von Nussbaum, Dominik Mumberg, Daniel Panne, <span class="NLM_string-name hlFld-ContribAuthor">Antje Margret Wengner</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of BAY-985, a Highly Selective TBK1/IKKε Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 601-612. <a href="https://doi.org/10.1021/acs.jmedchem.9b01460" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01460</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01460%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BBAY-985%25252C%252Ba%252BHighly%252BSelective%252BTBK1%25252FIKK%2525CE%2525B5%252BInhibitor%26aulast%3DLefranc%26aufirst%3DJulien%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D02092019%26date%3D10012020%26date%3D20122019%26volume%3D63%26issue%3D2%26spage%3D601%26epage%3D612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alexandria
P. Taylor, Magdalena Swewczyk, Steven Kennedy, Viacheslav V. Trush, Hong Wu, Hong Zeng, Aiping Dong, Renato Ferreira de Freitas, John Tatlock, Robert A. Kumpf, Martin Wythes, Agustin Casimiro-Garcia, Rajiah Aldrin Denny, Mihir D. Parikh, Fengling Li, Dalia Barsyte-Lovejoy, Matthieu Schapira, Masoud Vedadi, Peter J. Brown, Cheryl H. Arrowsmith, <span class="NLM_string-name hlFld-ContribAuthor">Dafydd R. Owen</span>. </span><span class="cited-content_cbyCitation_article-title">Selective, Small-Molecule Co-Factor Binding Site Inhibition of a Su(var)3–9, Enhancer of Zeste, Trithorax Domain Containing Lysine Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (17)
                                     , 7669-7683. <a href="https://doi.org/10.1021/acs.jmedchem.9b00112" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00112</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00112%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%25252C%252BSmall-Molecule%252BCo-Factor%252BBinding%252BSite%252BInhibition%252Bof%252Ba%252BSu%252528var%2525293%2525E2%252580%2525939%25252C%252BEnhancer%252Bof%252BZeste%25252C%252BTrithorax%252BDomain%252BContaining%252BLysine%252BMethyltransferase%26aulast%3DTaylor%26aufirst%3DAlexandria%2BP.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D17012019%26date%3D03092019%26date%3D15082019%26volume%3D62%26issue%3D17%26spage%3D7669%26epage%3D7683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Graham
L. Simpson, Sophie M. Bertrand, Jennifer A. Borthwick, Nino Campobasso, Julien Chabanet, Susan Chen, Julia Coggins, Josh Cottom, Siegfried B. Christensen, Helen C. Dawson, Helen L. Evans, Andrew N. Hobbs, Xuan Hong, Biju Mangatt, Jordi Munoz-Muriedas, Allen Oliff, Donghui Qin, Paul Scott-Stevens, Paris Ward, Yoshiaki Washio, Jingsong Yang, <span class="NLM_string-name hlFld-ContribAuthor">Robert J. Young</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Optimization of Novel Small c-Abl Kinase Activators Using Fragment and HTS Methodologies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (4)
                                     , 2154-2171. <a href="https://doi.org/10.1021/acs.jmedchem.8b01872" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01872</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01872%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Band%252BOptimization%252Bof%252BNovel%252BSmall%252Bc-Abl%252BKinase%252BActivators%252BUsing%252BFragment%252Band%252BHTS%252BMethodologies%26aulast%3DSimpson%26aufirst%3DGraham%2BL.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D29112018%26date%3D11022019%26date%3D28012019%26volume%3D62%26issue%3D4%26spage%3D2154%26epage%3D2171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zakey
Yusuf Buuh, Zhigang Lyu, <span class="NLM_string-name hlFld-ContribAuthor">Rongsheng E. Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Interrogating the Roles of Post-Translational Modifications of Non-Histone Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (8)
                                     , 3239-3252. <a href="https://doi.org/10.1021/acs.jmedchem.6b01817" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01817</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01817%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DInterrogating%252Bthe%252BRoles%252Bof%252BPost-Translational%252BModifications%252Bof%252BNon-Histone%252BProteins%26aulast%3DBuuh%26aufirst%3DZakey%2BYusuf%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D11122016%26date%3D24052017%26date%3D15052017%26volume%3D61%26issue%3D8%26spage%3D3239%26epage%3D3252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Léa  Bouché</span>, <span class="hlFld-ContribAuthor ">Clara D.  Christ</span>, <span class="hlFld-ContribAuthor ">Stephan  Siegel</span>, <span class="hlFld-ContribAuthor ">Amaury E.  Fernández-Montalván</span>, <span class="hlFld-ContribAuthor ">Simon J.  Holton</span>, <span class="hlFld-ContribAuthor ">Oleg  Fedorov</span>, <span class="hlFld-ContribAuthor ">Antonius  ter Laak</span>, <span class="hlFld-ContribAuthor ">Tatsuo  Sugawara</span>, <span class="hlFld-ContribAuthor ">Detlef  Stöckigt</span>, <span class="hlFld-ContribAuthor ">Cynthia  Tallant</span>, <span class="hlFld-ContribAuthor ">James  Bennett</span>, <span class="hlFld-ContribAuthor ">Octovia  Monteiro</span>, <span class="hlFld-ContribAuthor ">Laura  Díaz-Sáez</span>, <span class="hlFld-ContribAuthor ">Paulina  Siejka</span>, <span class="hlFld-ContribAuthor ">Julia  Meier</span>, <span class="hlFld-ContribAuthor ">Vera  Pütter</span>, <span class="hlFld-ContribAuthor ">Jörg  Weiske</span>, <span class="hlFld-ContribAuthor ">Susanne  Müller</span>, <span class="hlFld-ContribAuthor ">Kilian V. M.  Huber</span>, <span class="hlFld-ContribAuthor ">Ingo V.  Hartung</span>, and <span class="hlFld-ContribAuthor ">Bernard  Haendler</span>  . </span><span class="cited-content_cbyCitation_article-title">Benzoisoquinolinediones as Potent and Selective Inhibitors of BRPF2 and TAF1/TAF1L Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (9)
                                     , 4002-4022. <a href="https://doi.org/10.1021/acs.jmedchem.7b00306" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00306</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00306%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBenzoisoquinolinediones%252Bas%252BPotent%252Band%252BSelective%252BInhibitors%252Bof%252BBRPF2%252Band%252BTAF1%25252FTAF1L%252BBromodomains%26aulast%3DBouch%25C3%25A9%26aufirst%3DL%25C3%25A9a%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D24022017%26date%3D01052017%26date%3D11052017%26date%3D12042017%26volume%3D60%26issue%3D9%26spage%3D4002%26epage%3D4022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Scott D.  Cowen</span>, <span class="hlFld-ContribAuthor ">Daniel  Russell</span>, <span class="hlFld-ContribAuthor ">Leslie A.  Dakin</span>, <span class="hlFld-ContribAuthor ">Huawei  Chen</span>, <span class="hlFld-ContribAuthor ">Nicholas A.  Larsen</span>, <span class="hlFld-ContribAuthor ">Robert  Godin</span>, <span class="hlFld-ContribAuthor ">Scott  Throner</span>, <span class="hlFld-ContribAuthor ">Xiaolan  Zheng</span>, <span class="hlFld-ContribAuthor ">Audrey  Molina</span>, <span class="hlFld-ContribAuthor ">Jiaquan  Wu</span>, <span class="hlFld-ContribAuthor ">Tony  Cheung</span>, <span class="hlFld-ContribAuthor ">Tina  Howard</span>, <span class="hlFld-ContribAuthor ">Renee  Garcia-Arenas</span>, <span class="hlFld-ContribAuthor ">Nicholas  Keen</span>, <span class="hlFld-ContribAuthor ">Christopher S.  Pendleton</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Pietenpol</span>, and <span class="hlFld-ContribAuthor ">Andrew D.  Ferguson</span>  . </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Biological Activity of Substrate Competitive SMYD2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (24)
                                     , 11079-11097. <a href="https://doi.org/10.1021/acs.jmedchem.6b01303" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01303</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01303%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BActivity%252Bof%252BSubstrate%252BCompetitive%252BSMYD2%252BInhibitors%26aulast%3DCowen%26aufirst%3DScott%2BD.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D04092016%26date%3D01122016%26date%3D22122016%26date%3D14112016%26volume%3D59%26issue%3D24%26spage%3D11079%26epage%3D11097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hazem  Ahmed</span>, <span class="hlFld-ContribAuthor ">Shili  Duan</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, and <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>  . </span><span class="cited-content_cbyCitation_article-title">An Integrative Proteomic Approach Identifies Novel Cellular SMYD2 Substrates. </span><span class="cited-content_cbyCitation_journal-name">Journal of Proteome Research</span><span> <strong>2016,</strong> <em>15 </em>
                                    (6)
                                     , 2052-2059. <a href="https://doi.org/10.1021/acs.jproteome.6b00220" title="DOI URL">https://doi.org/10.1021/acs.jproteome.6b00220</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jproteome.6b00220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jproteome.6b00220%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Proteome%2520Research%26atitle%3DAn%252BIntegrative%252BProteomic%252BApproach%252BIdentifies%252BNovel%252BCellular%252BSMYD2%252BSubstrates%26aulast%3DAhmed%26aufirst%3DHazem%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D13032016%26date%3D10052016%26date%3D03062016%26volume%3D15%26issue%3D6%26spage%3D2052%26epage%3D2059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arvind Kumar  Yadav</span>, <span class="hlFld-ContribAuthor ">Tiratha Raj  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Novel inhibitors design through structural investigations and simulation studies for human PKMTs (SMYD2) involved in cancer. </span><span class="cited-content_cbyCitation_journal-name">Molecular Simulation</span><span> <strong>2021,</strong> <em>16 </em>, 1-10. <a href="https://doi.org/10.1080/08927022.2021.1957882" title="DOI URL">https://doi.org/10.1080/08927022.2021.1957882</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/08927022.2021.1957882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F08927022.2021.1957882%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Simulation%26atitle%3DNovel%252Binhibitors%252Bdesign%252Bthrough%252Bstructural%252Binvestigations%252Band%252Bsimulation%252Bstudies%252Bfor%252Bhuman%252BPKMTs%252B%252528SMYD2%252529%252Binvolved%252Bin%252Bcancer%26aulast%3DYadav%26aufirst%3DArvind%2BKumar%26date%3D2021%26date%3D2021%26volume%3D16%26spage%3D1%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yunyun  Wang</span>, <span class="hlFld-ContribAuthor ">Wei  Shi</span>, <span class="hlFld-ContribAuthor ">Chenliang  Wu</span>, <span class="hlFld-ContribAuthor ">Lin  Wan</span>, <span class="hlFld-ContribAuthor ">Yuxun  Zhao</span>, <span class="hlFld-ContribAuthor ">Chenglong  Zhang</span>, <span class="hlFld-ContribAuthor ">Wen  Gu</span>, <span class="hlFld-ContribAuthor ">Shifa  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Pyrazole ring-containing isolongifolanone derivatives as potential CDK2 inhibitors: Evaluation of anticancer activity and investigation of action mechanism. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2021,</strong> <em>139 </em>, 111663. <a href="https://doi.org/10.1016/j.biopha.2021.111663" title="DOI URL">https://doi.org/10.1016/j.biopha.2021.111663</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2021.111663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2021.111663%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DPyrazole%252Bring-containing%252Bisolongifolanone%252Bderivatives%252Bas%252Bpotential%252BCDK2%252Binhibitors%25253A%252BEvaluation%252Bof%252Banticancer%252Bactivity%252Band%252Binvestigation%252Bof%252Baction%252Bmechanism%26aulast%3DWang%26aufirst%3DYunyun%26date%3D2021%26volume%3D139%26spage%3D111663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefan  Gradl</span>, <span class="hlFld-ContribAuthor ">Holger  Steuber</span>, <span class="hlFld-ContribAuthor ">Joerg  Weiske</span>, <span class="hlFld-ContribAuthor ">Magda M.  Szewczyk</span>, <span class="hlFld-ContribAuthor ">Norbert  Schmees</span>, <span class="hlFld-ContribAuthor ">Stephan  Siegel</span>, <span class="hlFld-ContribAuthor ">Detlef  Stoeckigt</span>, <span class="hlFld-ContribAuthor ">Clara D.  Christ</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Shawna  Organ</span>, <span class="hlFld-ContribAuthor ">Megha  Abbey</span>, <span class="hlFld-ContribAuthor ">Steven  Kennedy</span>, <span class="hlFld-ContribAuthor ">Irene  Chau</span>, <span class="hlFld-ContribAuthor ">Viacheslav  Trush</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Cheryl  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Manfred  Husemann</span>, <span class="hlFld-ContribAuthor ">Volker  Badock</span>, <span class="hlFld-ContribAuthor ">Marcus  Bauser</span>, <span class="hlFld-ContribAuthor ">Andrea  Haegebarth</span>, <span class="hlFld-ContribAuthor ">Ingo V.  Hartung</span>, <span class="hlFld-ContribAuthor ">Carlo  Stresemann</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of the SMYD3 Inhibitor BAY-6035 Using Thermal Shift Assay (TSA)-Based High-Throughput Screening. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>78 </em>, 247255522110194. <a href="https://doi.org/10.1177/24725552211019409" title="DOI URL">https://doi.org/10.1177/24725552211019409</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/24725552211019409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F24725552211019409%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DDiscovery%252Bof%252Bthe%252BSMYD3%252BInhibitor%252BBAY-6035%252BUsing%252BThermal%252BShift%252BAssay%252B%252528TSA%252529-Based%252BHigh-Throughput%252BScreening%26aulast%3DGradl%26aufirst%3DStefan%26date%3D2021%26date%3D2021%26volume%3D78%26spage%3D247255522110194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamakoti P.  Bhat</span>, <span class="hlFld-ContribAuthor ">H.  Ümit Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Or  Gozani</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics and beyond: targeting writers of protein lysine methylation to treat disease. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (4)
                                     , 265-286. <a href="https://doi.org/10.1038/s41573-020-00108-x" title="DOI URL">https://doi.org/10.1038/s41573-020-00108-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-020-00108-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-020-00108-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DEpigenetics%252Band%252Bbeyond%25253A%252Btargeting%252Bwriters%252Bof%252Bprotein%252Blysine%252Bmethylation%252Bto%252Btreat%252Bdisease%26aulast%3DBhat%26aufirst%3DKamakoti%2BP.%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D4%26spage%3D265%26epage%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongzhao  Wang</span>, <span class="hlFld-ContribAuthor ">Zijian  Qin</span>, <span class="hlFld-ContribAuthor ">Aixia  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Classification models and SAR analysis on CysLT1 receptor antagonists using machine learning algorithms. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2021,</strong> <em>69 </em><a href="https://doi.org/10.1007/s11030-020-10165-4" title="DOI URL">https://doi.org/10.1007/s11030-020-10165-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-020-10165-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-020-10165-4%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DClassification%252Bmodels%252Band%252BSAR%252Banalysis%252Bon%252BCysLT1%252Breceptor%252Bantagonists%252Busing%252Bmachine%252Blearning%252Balgorithms%26aulast%3DWang%26aufirst%3DHongzhao%26date%3D2021%26date%3D2021%26volume%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin  Zhang</span>, <span class="hlFld-ContribAuthor ">Liping  Liao</span>, <span class="hlFld-ContribAuthor ">Fan  Wu</span>, <span class="hlFld-ContribAuthor ">Fengcai  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhongya  Sun</span>, <span class="hlFld-ContribAuthor ">Haijun  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and structure–activity relationship studies of LLY-507 analogues as SMYD2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (22)
                                     , 127598. <a href="https://doi.org/10.1016/j.bmcl.2020.127598" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127598</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127598%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bstructure%2525E2%252580%252593activity%252Brelationship%252Bstudies%252Bof%252BLLY-507%252Banalogues%252Bas%252BSMYD2%252Binhibitors%26aulast%3DZhang%26aufirst%3DBin%26date%3D2020%26volume%3D30%26issue%3D22%26spage%3D127598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hope S.  Rugo</span>, <span class="hlFld-ContribAuthor ">Ira  Jacobs</span>, <span class="hlFld-ContribAuthor ">Shikhar  Sharma</span>, <span class="hlFld-ContribAuthor ">Frank  Scappaticci</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Paul</span>, <span class="hlFld-ContribAuthor ">Kristen  Jensen-Pergakes</span>, <span class="hlFld-ContribAuthor ">Gabriel G.  Malouf</span>. </span><span class="cited-content_cbyCitation_article-title">The Promise for Histone Methyltransferase Inhibitors for Epigenetic Therapy in Clinical Oncology: A Narrative Review. </span><span class="cited-content_cbyCitation_journal-name">Advances in Therapy</span><span> <strong>2020,</strong> <em>37 </em>
                                    (7)
                                     , 3059-3082. <a href="https://doi.org/10.1007/s12325-020-01379-x" title="DOI URL">https://doi.org/10.1007/s12325-020-01379-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12325-020-01379-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12325-020-01379-x%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvances%2520in%2520Therapy%26atitle%3DThe%252BPromise%252Bfor%252BHistone%252BMethyltransferase%252BInhibitors%252Bfor%252BEpigenetic%252BTherapy%252Bin%252BClinical%252BOncology%25253A%252BA%252BNarrative%252BReview%26aulast%3DRugo%26aufirst%3DHope%2BS.%26date%3D2020%26date%3D2020%26volume%3D37%26issue%3D7%26spage%3D3059%26epage%3D3082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dillon K.  Jarrell</span>, <span class="hlFld-ContribAuthor ">Kelly N.  Hassell</span>, <span class="hlFld-ContribAuthor ">Debbie C.  Crans</span>, <span class="hlFld-ContribAuthor ">Shari  Lanning</span>, <span class="hlFld-ContribAuthor ">Mark A.  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Characterizing the Role of SMYD2 in Mammalian Embryogenesis—Future Directions. </span><span class="cited-content_cbyCitation_journal-name">Veterinary Sciences</span><span> <strong>2020,</strong> <em>7 </em>
                                    (2)
                                     , 63. <a href="https://doi.org/10.3390/vetsci7020063" title="DOI URL">https://doi.org/10.3390/vetsci7020063</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/vetsci7020063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fvetsci7020063%26sid%3Dliteratum%253Aachs%26jtitle%3DVeterinary%2520Sciences%26atitle%3DCharacterizing%252Bthe%252BRole%252Bof%252BSMYD2%252Bin%252BMammalian%252BEmbryogenesis%2525E2%252580%252594Future%252BDirections%26aulast%3DJarrell%26aufirst%3DDillon%2BK.%26date%3D2020%26date%3D2020%26volume%3D7%26issue%3D2%26spage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laia  Richart</span>, <span class="hlFld-ContribAuthor ">Raphaël  Margueron</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging histone methyltransferases in cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>56 </em>, 51-62. <a href="https://doi.org/10.1016/j.cbpa.2019.11.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2019.11.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2019.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2019.11.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDrugging%252Bhistone%252Bmethyltransferases%252Bin%252Bcancer%26aulast%3DRichart%26aufirst%3DLaia%26date%3D2020%26volume%3D56%26spage%3D51%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Viviana  Cuartas</span>, <span class="hlFld-ContribAuthor ">Sara M.  Robledo</span>, <span class="hlFld-ContribAuthor ">Iván D.  Vélez</span>, <span class="hlFld-ContribAuthor ">María del Pilar  Crespo</span>, <span class="hlFld-ContribAuthor ">Maximiliano  Sortino</span>, <span class="hlFld-ContribAuthor ">Susana  Zacchino</span>, <span class="hlFld-ContribAuthor ">Manuel  Nogueras</span>, <span class="hlFld-ContribAuthor ">Justo  Cobo</span>, <span class="hlFld-ContribAuthor ">Yulieth  Upegui</span>, <span class="hlFld-ContribAuthor ">Tatiana  Pineda</span>, <span class="hlFld-ContribAuthor ">Lina  Yepes</span>, <span class="hlFld-ContribAuthor ">Braulio  Insuasty</span>. </span><span class="cited-content_cbyCitation_article-title">New thiazolyl‐pyrazoline derivatives bearing nitrogen mustard as potential antimicrobial and antiprotozoal agents. </span><span class="cited-content_cbyCitation_journal-name">Archiv der Pharmazie</span><span> <strong>2020,</strong> <em>353 </em>
                                    (5)
                                     <a href="https://doi.org/10.1002/ardp.201900351" title="DOI URL">https://doi.org/10.1002/ardp.201900351</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ardp.201900351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fardp.201900351%26sid%3Dliteratum%253Aachs%26jtitle%3DArchiv%2520der%2520Pharmazie%26atitle%3DNew%252Bthiazolyl%2525E2%252580%252590pyrazoline%252Bderivatives%252Bbearing%252Bnitrogen%252Bmustard%252Bas%252Bpotential%252Bantimicrobial%252Band%252Bantiprotozoal%252Bagents%26aulast%3DCuartas%26aufirst%3DViviana%26date%3D2020%26date%3D2020%26volume%3D353%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack‐Sjodin</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Proteins as Emerging Drug Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 423-448. <a href="https://doi.org/10.1002/9781118681121.ch18" title="DOI URL">https://doi.org/10.1002/9781118681121.ch18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch18%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BProteins%252Bas%252BEmerging%252BDrug%252BTargets%26aulast%3DBoriack%25E2%2580%2590Sjodin%26aufirst%3DP.%2BAnn%26date%3D2020%26date%3D2020%26spage%3D423%26epage%3D448%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sebastian  Scheer</span>, <span class="hlFld-ContribAuthor ">Suzanne  Ackloo</span>, <span class="hlFld-ContribAuthor ">Tiago S.  Medina</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Ward</span>, <span class="hlFld-ContribAuthor ">Andrew M.  Lewis</span>, <span class="hlFld-ContribAuthor ">Jeffrey P.  Northrop</span>, <span class="hlFld-ContribAuthor ">Paul L.  Richardson</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Yudao  Shen</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">David  McLeod</span>, <span class="hlFld-ContribAuthor ">Carlos A.  Zepeda-Velazquez</span>, <span class="hlFld-ContribAuthor ">Minkui  Luo</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Kilian V. M.  Huber</span>, <span class="hlFld-ContribAuthor ">Daniel D.  De Carvalho</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Colby  Zaph</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>. </span><span class="cited-content_cbyCitation_article-title">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2019,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-018-07905-4" title="DOI URL">https://doi.org/10.1038/s41467-018-07905-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DA%252Bchemical%252Bbiology%252Btoolbox%252Bto%252Bstudy%252Bprotein%252Bmethyltransferases%252Band%252Bepigenetic%252Bsignaling%26aulast%3DScheer%26aufirst%3DSebastian%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Yi</span>, <span class="hlFld-ContribAuthor ">Xue-Jun  Jiang</span>, <span class="hlFld-ContribAuthor ">Ze-Min  Fang</span>. </span><span class="cited-content_cbyCitation_article-title">Histone methyltransferase SMYD2: ubiquitous regulator of disease. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-019-0711-4" title="DOI URL">https://doi.org/10.1186/s13148-019-0711-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-019-0711-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-019-0711-4%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DHistone%252Bmethyltransferase%252BSMYD2%25253A%252Bubiquitous%252Bregulator%252Bof%252Bdisease%26aulast%3DYi%26aufirst%3DXin%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renato  Ferreira de Freitas</span>, <span class="hlFld-ContribAuthor ">Danton  Ivanochko</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>. </span><span class="cited-content_cbyCitation_article-title">Methyltransferase Inhibitors: Competing with, or Exploiting the Bound Cofactor. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (24)
                                     , 4492. <a href="https://doi.org/10.3390/molecules24244492" title="DOI URL">https://doi.org/10.3390/molecules24244492</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24244492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24244492%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMethyltransferase%252BInhibitors%25253A%252BCompeting%252Bwith%25252C%252Bor%252BExploiting%252Bthe%252BBound%252BCofactor%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DRenato%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D24%26spage%3D4492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kezia C. L.  Whatley</span>, <span class="hlFld-ContribAuthor ">Gilda  Padalino</span>, <span class="hlFld-ContribAuthor ">Helen  Whiteland</span>, <span class="hlFld-ContribAuthor ">Kathrin K.  Geyer</span>, <span class="hlFld-ContribAuthor ">Benjamin J.  Hulme</span>, <span class="hlFld-ContribAuthor ">Iain W.  Chalmers</span>, <span class="hlFld-ContribAuthor ">Josephine  Forde-Thomas</span>, <span class="hlFld-ContribAuthor ">Salvatore  Ferla</span>, <span class="hlFld-ContribAuthor ">Andrea  Brancale</span>, <span class="hlFld-ContribAuthor ">Karl F.  Hoffmann</span>, . </span><span class="cited-content_cbyCitation_article-title">The repositioning of epigenetic probes/inhibitors identifies new anti-schistosomal lead compounds and chemotherapeutic targets. </span><span class="cited-content_cbyCitation_journal-name">PLOS Neglected Tropical Diseases</span><span> <strong>2019,</strong> <em>13 </em>
                                    (11)
                                     , e0007693. <a href="https://doi.org/10.1371/journal.pntd.0007693" title="DOI URL">https://doi.org/10.1371/journal.pntd.0007693</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pntd.0007693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pntd.0007693%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Neglected%2520Tropical%2520Diseases%26atitle%3DThe%252Brepositioning%252Bof%252Bepigenetic%252Bprobes%25252Finhibitors%252Bidentifies%252Bnew%252Banti-schistosomal%252Blead%252Bcompounds%252Band%252Bchemotherapeutic%252Btargets%26aulast%3DWhatley%26aufirst%3DKezia%2BC.%2BL.%26date%3D2019%26date%3D2019%26volume%3D13%26issue%3D11%26spage%3De0007693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jodie  Hay</span>, <span class="hlFld-ContribAuthor ">Kathryn  Gilroy</span>, <span class="hlFld-ContribAuthor ">Camille  Huser</span>, <span class="hlFld-ContribAuthor ">Anna  Kilbey</span>, <span class="hlFld-ContribAuthor ">Alma  Mcdonald</span>, <span class="hlFld-ContribAuthor ">Amanda  MacCallum</span>, <span class="hlFld-ContribAuthor ">Ailsa  Holroyd</span>, <span class="hlFld-ContribAuthor ">Ewan  Cameron</span>, <span class="hlFld-ContribAuthor ">James C.  Neil</span>. </span><span class="cited-content_cbyCitation_article-title">Collaboration of MYC and RUNX2 in lymphoma simulates T‐cell receptor signaling and attenuates p53 pathway activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Biochemistry</span><span> <strong>2019,</strong> <em>120 </em>
                                    (10)
                                     , 18332-18345. <a href="https://doi.org/10.1002/jcb.29143" title="DOI URL">https://doi.org/10.1002/jcb.29143</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcb.29143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcb.29143%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Biochemistry%26atitle%3DCollaboration%252Bof%252BMYC%252Band%252BRUNX2%252Bin%252Blymphoma%252Bsimulates%252BT%2525E2%252580%252590cell%252Breceptor%252Bsignaling%252Band%252Battenuates%252Bp53%252Bpathway%252Bactivity%26aulast%3DHay%26aufirst%3DJodie%26date%3D2019%26date%3D2019%26volume%3D120%26issue%3D10%26spage%3D18332%26epage%3D18345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Dilworth</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein methylation: from chemical tools to precision medicines. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (15)
                                     , 2967-2985. <a href="https://doi.org/10.1007/s00018-019-03147-9" title="DOI URL">https://doi.org/10.1007/s00018-019-03147-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03147-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03147-9%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DTargeting%252Bprotein%252Bmethylation%25253A%252Bfrom%252Bchemical%252Btools%252Bto%252Bprecision%252Bmedicines%26aulast%3DDilworth%26aufirst%3DDavid%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D15%26spage%3D2967%26epage%3D2985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edoardo  Fabini</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Manoni</span>, <span class="hlFld-ContribAuthor ">Claudia  Ferroni</span>, <span class="hlFld-ContribAuthor ">Alberto Del  Rio</span>, <span class="hlFld-ContribAuthor ">Manuela  Bartolini</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule inhibitors of lysine methyltransferases SMYD2 and SMYD3: current trends. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (8)
                                     , 901-921. <a href="https://doi.org/10.4155/fmc-2018-0380" title="DOI URL">https://doi.org/10.4155/fmc-2018-0380</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0380%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSmall-molecule%252Binhibitors%252Bof%252Blysine%252Bmethyltransferases%252BSMYD2%252Band%252BSMYD3%25253A%252Bcurrent%252Btrends%26aulast%3DFabini%26aufirst%3DEdoardo%26date%3D2019%26volume%3D11%26issue%3D8%26spage%3D901%26epage%3D921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Small‐Molecule Inhibitors of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 201-220. <a href="https://doi.org/10.1002/9783527809257.ch9" title="DOI URL">https://doi.org/10.1002/9783527809257.ch9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch9%26sid%3Dliteratum%253Aachs%26atitle%3DSelective%252BSmall%2525E2%252580%252590Molecule%252BInhibitors%252Bof%252BProtein%252BMethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2019%26date%3D2019%26spage%3D201%26epage%3D220%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kevin  Dzobo</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenomics-Guided Drug Development: Recent Advances in Solving the Cancer Treatment “jigsaw puzzle”. </span><span class="cited-content_cbyCitation_journal-name">OMICS: A Journal of Integrative Biology</span><span> <strong>2019,</strong> <em>23 </em>
                                    (2)
                                     , 70-85. <a href="https://doi.org/10.1089/omi.2018.0206" title="DOI URL">https://doi.org/10.1089/omi.2018.0206</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/omi.2018.0206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fomi.2018.0206%26sid%3Dliteratum%253Aachs%26jtitle%3DOMICS%253A%2520A%2520Journal%2520of%2520Integrative%2520Biology%26atitle%3DEpigenomics-Guided%252BDrug%252BDevelopment%25253A%252BRecent%252BAdvances%252Bin%252BSolving%252Bthe%252BCancer%252BTreatment%252B%2525E2%252580%25259Cjigsaw%252Bpuzzle%2525E2%252580%25259D%26aulast%3DDzobo%26aufirst%3DKevin%26date%3D2019%26volume%3D23%26issue%3D2%26spage%3D70%26epage%3D85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Methyltransferases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 123-157. <a href="https://doi.org/10.1007/7355_2019_72" title="DOI URL">https://doi.org/10.1007/7355_2019_72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_72%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BMethyltransferases%252Band%252BTheir%252BInhibitors%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2019%26date%3D2019%26spage%3D123%26epage%3D157%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Shuichi  Mori</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Chemical Tools for the Regulation and Study of Protein Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1745-1759. <a href="https://doi.org/10.1002/tcr.201800034" title="DOI URL">https://doi.org/10.1002/tcr.201800034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800034%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DRecent%252BAdvances%252Bin%252BChemical%252BTools%252Bfor%252Bthe%252BRegulation%252Band%252BStudy%252Bof%252BProtein%252BLysine%252BMethyltransferases%26aulast%3DHirano%26aufirst%3DTomoya%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1745%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Evan M.  Cornett</span>, <span class="hlFld-ContribAuthor ">Bradley M.  Dickson</span>, <span class="hlFld-ContribAuthor ">Krzysztof  Krajewski</span>, <span class="hlFld-ContribAuthor ">Nicholas  Spellmon</span>, <span class="hlFld-ContribAuthor ">Andrew  Umstead</span>, <span class="hlFld-ContribAuthor ">Robert M.  Vaughan</span>, <span class="hlFld-ContribAuthor ">Kevin M.  Shaw</span>, <span class="hlFld-ContribAuthor ">Philip P.  Versluis</span>, <span class="hlFld-ContribAuthor ">Martis W.  Cowles</span>, <span class="hlFld-ContribAuthor ">Joseph  Brunzelle</span>, <span class="hlFld-ContribAuthor ">Zhe  Yang</span>, <span class="hlFld-ContribAuthor ">Irving E.  Vega</span>, <span class="hlFld-ContribAuthor ">Zu-Wen  Sun</span>, <span class="hlFld-ContribAuthor ">Scott B.  Rothbart</span>. </span><span class="cited-content_cbyCitation_article-title">A functional proteomics platform to reveal the sequence determinants of lysine methyltransferase substrate selectivity. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2018,</strong> <em>4 </em>
                                    (11)
                                     , eaav2623. <a href="https://doi.org/10.1126/sciadv.aav2623" title="DOI URL">https://doi.org/10.1126/sciadv.aav2623</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.aav2623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.aav2623%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DA%252Bfunctional%252Bproteomics%252Bplatform%252Bto%252Breveal%252Bthe%252Bsequence%252Bdeterminants%252Bof%252Blysine%252Bmethyltransferase%252Bsubstrate%252Bselectivity%26aulast%3DCornett%26aufirst%3DEvan%2BM.%26date%3D2018%26date%3D2018%26volume%3D4%26issue%3D11%26spage%3Deaav2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael J.  Thomenius</span>, <span class="hlFld-ContribAuthor ">Jennifer  Totman</span>, <span class="hlFld-ContribAuthor ">Darren  Harvey</span>, <span class="hlFld-ContribAuthor ">Lorna H.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Thomas V.  Riera</span>, <span class="hlFld-ContribAuthor ">Kat  Cosmopoulos</span>, <span class="hlFld-ContribAuthor ">Alexandra R.  Grassian</span>, <span class="hlFld-ContribAuthor ">Christine  Klaus</span>, <span class="hlFld-ContribAuthor ">Megan  Foley</span>, <span class="hlFld-ContribAuthor ">Elizabeth A.  Admirand</span>, <span class="hlFld-ContribAuthor ">Haris  Jahic</span>, <span class="hlFld-ContribAuthor ">Christina  Majer</span>, <span class="hlFld-ContribAuthor ">Tim  Wigle</span>, <span class="hlFld-ContribAuthor ">Suzanne L.  Jacques</span>, <span class="hlFld-ContribAuthor ">Jodi  Gureasko</span>, <span class="hlFld-ContribAuthor ">Dorothy  Brach</span>, <span class="hlFld-ContribAuthor ">Trupti  Lingaraj</span>, <span class="hlFld-ContribAuthor ">Kip  West</span>, <span class="hlFld-ContribAuthor ">Sherri  Smith</span>, <span class="hlFld-ContribAuthor ">Nathalie  Rioux</span>, <span class="hlFld-ContribAuthor ">Nigel J.  Waters</span>, <span class="hlFld-ContribAuthor ">Cuyue  Tang</span>, <span class="hlFld-ContribAuthor ">Alejandra  Raimondi</span>, <span class="hlFld-ContribAuthor ">Michael  Munchhof</span>, <span class="hlFld-ContribAuthor ">James E.  Mills</span>, <span class="hlFld-ContribAuthor ">Scott  Ribich</span>, <span class="hlFld-ContribAuthor ">Margaret  Porter Scott</span>, <span class="hlFld-ContribAuthor ">Kevin W.  Kuntz</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Mikel  Moyer</span>, <span class="hlFld-ContribAuthor ">Jesse J.  Smith</span>, <span class="hlFld-ContribAuthor ">Richard  Chesworth</span>, <span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>, <span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span>, . </span><span class="cited-content_cbyCitation_article-title">Small molecule inhibitors and CRISPR/Cas9 mutagenesis demonstrate that SMYD2 and SMYD3 activity are dispensable for autonomous cancer cell proliferation. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (6)
                                     , e0197372. <a href="https://doi.org/10.1371/journal.pone.0197372" title="DOI URL">https://doi.org/10.1371/journal.pone.0197372</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0197372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0197372%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DSmall%252Bmolecule%252Binhibitors%252Band%252BCRISPR%25252FCas9%252Bmutagenesis%252Bdemonstrate%252Bthat%252BSMYD2%252Band%252BSMYD3%252Bactivity%252Bare%252Bdispensable%252Bfor%252Bautonomous%252Bcancer%252Bcell%252Bproliferation%26aulast%3DThomenius%26aufirst%3DMichael%2BJ.%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D6%26spage%3De0197372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael J.  Morrison</span>, <span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Kerren K.  Swinger</span>, <span class="hlFld-ContribAuthor ">Tim J.  Wigle</span>, <span class="hlFld-ContribAuthor ">Dipti  Sadalge</span>, <span class="hlFld-ContribAuthor ">Kevin W.  Kuntz</span>, <span class="hlFld-ContribAuthor ">Margaret Porter  Scott</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Richard  Chesworth</span>, <span class="hlFld-ContribAuthor ">Kenneth W.  Duncan</span>, <span class="hlFld-ContribAuthor ">Darren M.  Harvey</span>, <span class="hlFld-ContribAuthor ">John W.  Lampe</span>, <span class="hlFld-ContribAuthor ">Lorna H.  Mitchell</span>, <span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>, . </span><span class="cited-content_cbyCitation_article-title">Identification of a peptide inhibitor for the histone methyltransferase WHSC1. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (5)
                                     , e0197082. <a href="https://doi.org/10.1371/journal.pone.0197082" title="DOI URL">https://doi.org/10.1371/journal.pone.0197082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0197082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0197082%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DIdentification%252Bof%252Ba%252Bpeptide%252Binhibitor%252Bfor%252Bthe%252Bhistone%252Bmethyltransferase%252BWHSC1%26aulast%3DMorrison%26aufirst%3DMichael%2BJ.%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3De0197082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in developing selective inhibitors of protein methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2017,</strong> <em>39 </em>, 100-108. <a href="https://doi.org/10.1016/j.cbpa.2017.06.013" title="DOI URL">https://doi.org/10.1016/j.cbpa.2017.06.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2017.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2017.06.013%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DRecent%252Bprogress%252Bin%252Bdeveloping%252Bselective%252Binhibitors%252Bof%252Bprotein%252Bmethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2017%26volume%3D39%26spage%3D100%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sophia Xiao  Pfister</span>, <span class="hlFld-ContribAuthor ">Alan  Ashworth</span>. </span><span class="cited-content_cbyCitation_article-title">Marked for death: targeting epigenetic changes in cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2017,</strong> <em>16 </em>
                                    (4)
                                     , 241-263. <a href="https://doi.org/10.1038/nrd.2016.256" title="DOI URL">https://doi.org/10.1038/nrd.2016.256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2016.256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2016.256%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DMarked%252Bfor%252Bdeath%25253A%252Btargeting%252Bepigenetic%252Bchanges%252Bin%252Bcancer%26aulast%3DPfister%26aufirst%3DSophia%2BXiao%26date%3D2017%26date%3D2017%26volume%3D16%26issue%3D4%26spage%3D241%26epage%3D263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.M.  Woster</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Therapeutics Targeting Epigenetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 297-328. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12325-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12325-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12325-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12325-X%26sid%3Dliteratum%253Aachs%26atitle%3DNovel%252BTherapeutics%252BTargeting%252BEpigenetics%26aulast%3DWoster%26aufirst%3DP.M.%26date%3D2017%26spage%3D297%26epage%3D328%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Inhibition of Protein Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2016,</strong> <em>23 </em>
                                    (9)
                                     , 1067-1076. <a href="https://doi.org/10.1016/j.chembiol.2016.07.014" title="DOI URL">https://doi.org/10.1016/j.chembiol.2016.07.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2016.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2016.07.014%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DChemical%252BInhibition%252Bof%252BProtein%252BMethyltransferases%26aulast%3DSchapira%26aufirst%3DMatthieu%26date%3D2016%26volume%3D23%26issue%3D9%26spage%3D1067%26epage%3D1076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0008.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Comparison of so far described SMYD2 inhibitors with (<i>S</i>)-<b>4</b> discovered in this study. Data from respective references as well as assay results from this study are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0009.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Identification and biophysical validation of the pyrazoline SMYD2 inhibitor series leading to (<i>S</i>)-<b>4</b>. (A) Scintillation proximity assay (SPA) screening approach leading to the pyrazoline hit cluster, with representative hit compound <b>5</b> with IC<sub>50</sub> = 1.7 μM. (B) Thermal shift assay (TSA) for SMYD2 protein stabilization for inhibitors of the aminopyrazoline series [●], with aminopyrazoline hit <b>5</b> (Δ<i>T</i><sub>m</sub> = 3.3 K) and (<i>S</i>)-<b>4</b> (Δ<i>T</i><sub>m</sub> = 7.1 K) highlighted. Stabilization correlates with SPA potency. (C) Isothermal titration calorimetry (ITC) of compound <b>6</b> indicating a submicromolar binding constant (<i>K</i><sub>d</sub> = 540 nM) and a high enthalpic contribution to the binding energy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0017.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyrazoline Intermediates <b>10a</b>–<b>f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) allyl chloroformate, K<sub>2</sub>CO<sub>3</sub> (1.5 M in water), DCM, rt, overnight; (b) formaldehyde (37 wt % in water), piperidine, EtOH, rt, overnight; (c) hydrazine monohydrate, EtOH, reflux; (d) diphenyl <i>N</i>-cyanocarbonimidate, <i>i</i>-PrOH, reflux, 1 h, then rt, overnight.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0018.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>5</b>, <b>6</b>, and <b>12</b>–<b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) aniline derivative, <i>n</i>-BuLi, THF, −78 °C, or alkylamine, 1,4-dioxane, reflux; (b) 1,3-dimethylbarbituric acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF; (c) acetaldehyde, NaBH<sub>4</sub>, DCM, 0 °C; (d) (1) acetoxyacetyl chloride, aqueous NaHCO<sub>3</sub> solution, DCM, (2) K<sub>2</sub>CO<sub>3</sub>, MeOH; (e) skip (for compound <b>19</b>), or aldehyde derivative, NaBH<sub>4</sub>, MeOH, 0 °C; (f) Fmoc-glycine, HATU, NMM, DMF; (g) piperidine, DMF; (h) methoxyacetyl chloride, aqueous NaHCO<sub>3</sub> solution, DCM; (i) chloroacetyl chloride, <i>N</i>,<i>N</i>-diisopropylethylamine, DCM; (j) urotropine, EtOH.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0019.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>4</b> and <b>25</b>–<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 3-(difluoromethoxy)aniline, <i>n</i>-BuLi, THF, −78 °C; (b) 1,3-dimethylbarbituric acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF; (c) acetaldehyde, NaBH<sub>4</sub>, MeOH, 0 °C; (d) (1) acetoxyacetyl chloride, aqueous NaHCO<sub>3</sub> solution, DCM, (2) K<sub>2</sub>CO<sub>3</sub>, MeOH.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0020.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>30</b>–<b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) aniline derivative, <i>n</i>-BuLi, THF, −78 °C; (b) 1,3-dimethylbarbituric acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, THF; (c) acetaldehyde, NaBH<sub>4</sub>, MeOH, 0 °C; (d) (1) acetoxyacetyl chloride, aqueous NaHCO<sub>3</sub> solution, DCM, (2) K<sub>2</sub>CO<sub>3</sub>, MeOH.</p></p></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0010.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode of compound <b>6</b> and (<i>S</i>)-<b>4</b>. (A,B) Two different views of the co-crystal structure of <b>6</b> in complex with SMYD2. There is a good steric and electrostatic fit of <b>6</b> into the substrate binding site of SMYD2, and multiple hydrogen-bond interactions are involved. The lysine channel is occupied by the 4-chlorophenyl residue of the ligand, representing a novel lysine channel binding motif. In addition, pockets-1 and -2 are addressed. (C) Visualization of the WaterMap calculations suggesting an optimal displacement of water molecules by a 3,4-dichlorophenyl residue. Calculated water sites are shown as spheres and colored based on their free energy. Only sites within the lysine channel are shown. WaterMap results are based on the crystal structure of the monochloro derivative <b>6</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARF">5ARF</a>). The protein surface is depicted in gray. The modeled ligand structure is shown as colored sticks (chlorine, green; carbon, yellow; hydrogen, white; nitrogen, blue; oxygen, red; fluorine, light green). For clarity, protein residues and cofactors are not shown. (D) Overlay of the crystal structures of compound <b>6</b> (magenta) and (<i>S</i>)-<b>4</b> (yellow, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARG">5ARG</a>), showing nearly identical binding modes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0011.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of (<i>S</i>)-<b>4</b> (this work) with the recently reported SMYD2 inhibitors <b>1</b><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and <b>2</b>.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (A) Chemical structures of the three selected SMYD2 inhibitors. (B,C) Two different views of an overlay of (<i>S</i>)-<b>4</b> (yellow) with <b>1</b> (green) and <b>2</b> (blue). (<i>S</i>)-<b>4</b> has a distinct binding mode, addressing pocket-2 which is not occupied by <b>1</b> or <b>2</b>, while only <b>1</b> and <b>2</b> occupy the distant hydrophobic pocket-3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0012.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (<i>S</i>)-<b>4</b> mode of inhibition and selectivity profile. (A) Activity in the scintillation proximity assay (SPA). IC<sub>50</sub> (<i>n</i> > 10) for SMYD2 inhibition = 26 ± 7 nM (representative inhibition curve shown). (B) IC<sub>50</sub> values obtained from SPA were plotted against the indicated substrate concentrations (represented as [substrate]/<i>K</i><sub>m(app)</sub>). Data were fitted to competitive and uncompetitive models of the Cheng–Prusoff equation.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41, 42)</a> Data points are the mean of eight replicates, error bars indicate 1 × SD. The data indicate that (<i>S</i>)-<b>4</b> is a peptide-competitive, SAM-uncompetitive inhibitor of SMYD2 methyltransferase activity. (C) Selectivity profile on a panel of 32 additional methyltransferases showing high selectivity of (<i>S</i>)-<b>4</b> for SMYD2. Only SMYD3 is weakly inhibited by (<i>S</i>)-<b>4</b> with <i>a</i> > 1 μM IC<sub>50</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0013.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibition of cellular methylation of the tumor suppressor protein p53 by (<i>S</i>)-<b>4</b>. (A) A specific antibody directed against p53K370me1 (SY46) was generated and tested for specificity on recombinant p53 (rec. p53) which had been in vitro methylated by SMYD2, or nonmethylated. (B) Endogenous methylation of p53 protein was characterized by treatment of KYSE-150 esophageal cancer cells with increasing concentrations of (<i>S</i>)-<b>4</b> for 5 days (Co = control). (C) Cellular p53 methylation assay using transient FLAG-tagged SMYD2 and FLAG-tagged p53 overexpression in HEK293T cells. Increasing concentrations of (<i>S</i>)-<b>4</b> reduce the methylation of overexpressed p53 (for this assay p53K730me1-specific antibody kindly provided by Dr. Shelly Berger was used). (D) IC<sub>50</sub> determination in the cellular p53 methylation assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0014.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. AHNAK is a novel substrate of SMYD2 and methylation can be inhibited by aminopyrazoline inhibitors. (A) In-Cell Western (ICW) assay. Immunofluorescence-based detection of AHNAK methylation. Each row represents a different inhibitor tested in increasing concentrations ranging from 39 nM up to 5 μM. (<i>S</i>)-<b>4</b> and <b>1</b> are highlighted. (B) Inhibition of AHNAK methylation in MDA-MB231 cells overexpressing SMYD2. Methylation signal overlaps with AHNAK protein detection. (<i>S</i>)-<b>4</b> inhibits concentration dependent the AHNAK methylation. (C) Good correlation of cellular activity (ICW) to biochemical activity (SPA) of (<i>S</i>)-<b>4</b> and pyrazoline cluster derivatives. (<i>S</i>)-<b>4</b> has a cellular IC<sub>50</sub> ∼ 60 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0015.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Proliferation and apoptosis effects of (<i>S</i>)-<b>4</b>. (A) A panel of 240 different cancer cell lines (OncoPanel 240/Eurofins Panlabs) was long-term cultured with (<i>S</i>)-<b>4</b> for 10 days and proliferation effects were determined. (<i>S</i>)-<b>4</b> showed proliferation effects with an IC<sub>50</sub> < 10 μM in a subset of cell lines. (B) Effects on apoptosis induction were determined by caspase 3/7 activation. KYSE-150, U2OS, and A2780 cell lines were pretreated with (<i>S</i>)-<b>4</b> or inactive derivative <b>25</b> (see <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) for 2 days, then apoptosis was induced by treatment with doxorubicin. (<i>S</i>)-<b>4</b>, but not <b>25</b>, significantly improving caspase 3/7 activation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/medium/jm-2015-01890m_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0016.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. In vivo characterization of (<i>S</i>)-<b>4</b>. (A) Demethylation of AHNAK was evaluated ex vivo. Average methylation signals ± standard deviation per group are shown. Mice (<i>n</i> = 12 per group) bearing subcutaneous tumor xenografts (tumor tissue derived from the SMYD2-overexpressing KYSE-150 cell line) were treated as indicated, then the tumors were analyzed for methylation signals by dot-blotting. (<i>S</i>)-<b>4</b> significantly reduced the methylation with doses starting from 30 mg/kg, with most significant effects in the 100 mg/kg group (<i>P</i> < 0.001, Student’s <i>t</i> test). (B) Tumor area graph summarizing the in vivo tumor efficacy study with the KYSE-150 xenograft model. Average tumor area ± standard error of mean per group is plotted as the mean. Treatment was started at day 4 after tumor inoculation (black arrow), and groups were treated as indicated. (C) Tumor weight graphs corresponding to the same experiment shown in (B). Average tumor weight are blotted as box plot. * Significant (<i>p</i> value <0.05) difference between vehicle control and treatment group (Dunn’s method). Group 2: one animal was excluded on the 14th treatment day due to animal welfare reasons (ulcerated tumor), tumor size 82 mm<sup>2</sup>. (D) Mouse body weight analysis. Average body weight per group throughout the experiment is plotted as the mean.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.5b01890/20160621/images/large/jm-2015-01890m_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01890&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i75">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10300" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10300" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 61 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span> </span><span class="NLM_article-title">Emerging roles of lysine methylation on non-histone proteins</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">4257</span><span class="NLM_x">–</span> <span class="NLM_lpage">4272</span><span class="refDoi"> DOI: 10.1007/s00018-015-2001-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1007%2Fs00018-015-2001-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=26227335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kmu7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2015&pages=4257-4272&author=X.+Zhangauthor=Y.+Huangauthor=X.+Shi&title=Emerging+roles+of+lysine+methylation+on+non-histone+proteins&doi=10.1007%2Fs00018-015-2001-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging roles of lysine methylation on non-histone proteins</span></div><div class="casAuthors">Zhang, Xi; Huang, Yaling; Shi, Xiaobing</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4257-4272</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Lysine methylation is a common posttranslational modification (PTM) of histones that is important for the epigenetic regulation of transcription and chromatin in eukaryotes.  Increasing evidence demonstrates that in addn. to histones, lysine methylation also occurs on various non-histone proteins, esp. transcription- and chromatin-regulating proteins.  In this review, we will briefly describe the histone lysine methyltransferases (KMTs) that have a broad spectrum of non-histone substrates.  We will use p53 and nuclear receptors, esp. estrogen receptor alpha, as examples to discuss the dynamic nature of non-histone protein lysine methylation, the writers, erasers, and readers of these modifications, and the crosstalk between lysine methylation and other PTMs in regulating the functions of the modified proteins.  Understanding the roles of lysine methylation in normal cells and during development will shed light on the complex biol. of diseases assocd. with the dysregulation of lysine methylation on both histones and non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUDZXq2gpdQLVg90H21EOLACvtfcHk0lh4P-_H6kX84A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kmu7%252FK&md5=8a1a5243088f896012a2b0d368f6235e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2Fs00018-015-2001-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-015-2001-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DX.%26atitle%3DEmerging%2520roles%2520of%2520lysine%2520methylation%2520on%2520non-histone%2520proteins%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2015%26volume%3D72%26spage%3D4257%26epage%3D4272%26doi%3D10.1007%2Fs00018-015-2001-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Komatsu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imoto, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozaki, K.-i.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aiko, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshizumi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichikawa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otsuji, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inazawa, J.</span><span> </span><span class="NLM_article-title">Overexpression of SMYD2 relates to tumor cell proliferation and malignant outcome of esophageal squamous cell carcinoma</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1139</span><span class="NLM_x">–</span> <span class="NLM_lpage">1146</span><span class="refDoi"> DOI: 10.1093/carcin/bgp116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1093%2Fcarcin%2Fbgp116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19423649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1CisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1139-1146&author=S.+Komatsuauthor=I.+Imotoauthor=H.+Tsudaauthor=K.-i.+Kozakiauthor=T.+Muramatsuauthor=Y.+Shimadaauthor=S.+Aikoauthor=Y.+Yoshizumiauthor=D.+Ichikawaauthor=E.+Otsujiauthor=J.+Inazawa&title=Overexpression+of+SMYD2+relates+to+tumor+cell+proliferation+and+malignant+outcome+of+esophageal+squamous+cell+carcinoma&doi=10.1093%2Fcarcin%2Fbgp116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of SMYD2 relates to tumor cell proliferation and malignant outcome of esophageal squamous cell carcinoma</span></div><div class="casAuthors">Komatsu, Shuhei; Imoto, Issei; Tsuda, Hitoshi; Kozaki, Ken-ich; Muramatsu, Tomoki; Shimada, Yutaka; Aiko, Satoshi; Yoshizumi, Yutaka; Ichikawa, Daisuke; Otsuji, Eigo; Inazawa, Johji</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1139-1146</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although we have identified two putative targets, ATF3 and CENPF, for a frequently gained/amplified region around 1q32-q41 in esophageal squamous cell carcinoma (ESCC), it is possible that other amplification targets remain to be identified.  In this study, we tested whether SET and MYND domain-contg. protein 2 (SMYD2), located between those two genes and encoding a lysine methyltransferase for histone H3K36 and p53K370 that regulates transcription and inhibits transactivation activity, resp., acts as a cancer-promoting gene through activation/overexpression in ESCC.  Frequent overexpression of SMYD2 mRNA and protein was obsd. in KYSE150 cells with remarkable amplification at 1q32-41.1 and other ESCC cell lines (11/43 lines, 25.6%).  Overexpression of SMYD2 protein was frequently detected in primary tumor samples of ESCC (117/153 cases, 76.5%) as well and significantly correlated with gender, venous invasion, the pT category in the tumor-lymph node-metastases classification and status of recurrence.  Patients with SMYD2-overexpressing tumors had a worse overall rate of survival than those with non-expressing tumors, and SMYD2 positivity was independently assocd. with a worse outcome in the multivariate anal.  Knockdown of SMYD2 expression inhibited and ectopic overexpression of SMYD2 promoted the proliferation of ESCC cells in a TP53 mutation-independent but SMYD2 expression-dependent manner.  These findings suggest that SMYD2 plays an important role in tumor cell proliferation through its activation/overexpression and highlight its usefulness as a prognosticator and potential therapeutic target in ESCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGuuMnL7IjzbVg90H21EOLACvtfcHk0lh4P-_H6kX84A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1CisLs%253D&md5=7eb396b256c9ec2b182fd999796527b5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp116%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DS.%26aulast%3DImoto%26aufirst%3DI.%26aulast%3DTsuda%26aufirst%3DH.%26aulast%3DKozaki%26aufirst%3DK.-i.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DAiko%26aufirst%3DS.%26aulast%3DYoshizumi%26aufirst%3DY.%26aulast%3DIchikawa%26aufirst%3DD.%26aulast%3DOtsuji%26aufirst%3DE.%26aulast%3DInazawa%26aufirst%3DJ.%26atitle%3DOverexpression%2520of%2520SMYD2%2520relates%2520to%2520tumor%2520cell%2520proliferation%2520and%2520malignant%2520outcome%2520of%2520esophageal%2520squamous%2520cell%2520carcinoma%26jtitle%3DCarcinogenesis%26date%3D2009%26volume%3D30%26spage%3D1139%26epage%3D1146%26doi%3D10.1093%2Fcarcin%2Fbgp116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Komatsu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichikawa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirajima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawaguchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamae, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okajima, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konishi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiozaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuda, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imoto, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inazawa, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otsuji, E.</span><span> </span><span class="NLM_article-title">Overexpression of SMYD2 contributes to malignant outcome in gastric cancer</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">357</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/bjc.2014.543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fbjc.2014.543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25321194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOkt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=357-364&author=S.+Komatsuauthor=D.+Ichikawaauthor=S.+Hirajimaauthor=H.+Nagataauthor=Y.+Nishimuraauthor=T.+Kawaguchiauthor=M.+Miyamaeauthor=W.+Okajimaauthor=T.+Ohashiauthor=H.+Konishiauthor=A.+Shiozakiauthor=H.+Fujiwaraauthor=K.+Okamotoauthor=H.+Tsudaauthor=I.+Imotoauthor=J.+Inazawaauthor=E.+Otsuji&title=Overexpression+of+SMYD2+contributes+to+malignant+outcome+in+gastric+cancer&doi=10.1038%2Fbjc.2014.543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of SMYD2 contributes to malignant outcome in gastric cancer</span></div><div class="casAuthors">Komatsu, S.; Ichikawa, D.; Hirajima, S.; Nagata, H.; Nishimura, Y.; Kawaguchi, T.; Miyamae, M.; Okajima, W.; Ohashi, T.; Konishi, H.; Shiozaki, A.; Fujiwara, H.; Okamoto, K.; Tsuda, H.; Imoto, I.; Inazawa, J.; Otsuji, E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">357-364</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Background: SET and MYND domain-contg. protein 2 (SMYD2) is a lysine methyltransferase for histone H3, p53 and Rb and inhibits their transactivation activities.  In this study, we tested whether SMYD2 (1q42) acts as a cancer-promoting factor by being overexpressed in gastric cancer.  Methods: We analyzed 7 gastric cancer cell lines and 147 primary tumor samples of gastric cancer, which were curatively resected in our hospital.  Results: SET and MYND domain-contg. protein 2 was detected in these cell lines (five out of seven cell lines; 71.4%) and primary tumor samples (fifty-six out of one hundred and forty-seven cases; 38.1%).  Knockdown of SMYD2 using specific small interfering RNA inhibited proliferation, migration and invasion of SMYD2-overexpressing cells in a TP53 mutation-independent manner.  Overexpression of SMYD2 protein correlated with larger tumor size, more aggressive lymphatic invasion, deeper tumor invasion and higher rates of lymph node metastasis and recurrence.  Patients with SMYD2-overexpressing tumors had a worse overall rate of survival than those with non-expressing tumors (P=0.0073, log-rank test) in an intensity and proportion score-dependent manner.  Moreover, multivariate anal. demonstrated that SMYD2 was independently assocd. with worse outcome (P=0.0021, hazard ratio 4.25 (1.69-10.7)).  Conclusions: These findings suggest that SMYD2 has a crucial role in tumor cell proliferation by its overexpression and highlight its usefulness as a prognostic factor and potential therapeutic target in gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph37xZhp4Ll7Vg90H21EOLACvtfcHk0lhij1ygy-kvQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOkt77O&md5=4c1b9afa2e324eedb0173eee4d65c884</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2014.543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2014.543%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DS.%26aulast%3DIchikawa%26aufirst%3DD.%26aulast%3DHirajima%26aufirst%3DS.%26aulast%3DNagata%26aufirst%3DH.%26aulast%3DNishimura%26aufirst%3DY.%26aulast%3DKawaguchi%26aufirst%3DT.%26aulast%3DMiyamae%26aufirst%3DM.%26aulast%3DOkajima%26aufirst%3DW.%26aulast%3DOhashi%26aufirst%3DT.%26aulast%3DKonishi%26aufirst%3DH.%26aulast%3DShiozaki%26aufirst%3DA.%26aulast%3DFujiwara%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DK.%26aulast%3DTsuda%26aufirst%3DH.%26aulast%3DImoto%26aufirst%3DI.%26aulast%3DInazawa%26aufirst%3DJ.%26aulast%3DOtsuji%26aufirst%3DE.%26atitle%3DOverexpression%2520of%2520SMYD2%2520contributes%2520to%2520malignant%2520outcome%2520in%2520gastric%2520cancer%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2015%26volume%3D112%26spage%3D357%26epage%3D364%26doi%3D10.1038%2Fbjc.2014.543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Cho, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogure, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaue, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">RB1 methylation by SMYD2 enhances cell cycle progression through an increase of RB1 phosphorylation</span> <span class="citation_source-journal">Neoplasia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">476</span><span class="NLM_x">–</span> <span class="NLM_lpage">486</span><span class="refDoi"> DOI: 10.1593/neo.12656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1593%2Fneo.12656" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=476-486&author=H.-S.+Choauthor=S.+Hayamiauthor=G.+Toyokawaauthor=K.+Maejimaauthor=Y.+Yamaneauthor=T.+Suzukiauthor=N.+Dohmaeauthor=M.+Kogureauthor=D.+Kangauthor=D.+E.+Nealauthor=B.+A.+J.+Ponderauthor=H.+Yamaueauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=RB1+methylation+by+SMYD2+enhances+cell+cycle+progression+through+an+increase+of+RB1+phosphorylation&doi=10.1593%2Fneo.12656"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1593%2Fneo.12656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1593%252Fneo.12656%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.-S.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DKogure%26aufirst%3DM.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DYamaue%26aufirst%3DH.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DRB1%2520methylation%2520by%2520SMYD2%2520enhances%2520cell%2520cycle%2520progression%2520through%2520an%2520increase%2520of%2520RB1%2520phosphorylation%26jtitle%3DNeoplasia%26date%3D2012%26volume%3D14%26spage%3D476%26epage%3D486%26doi%3D10.1593%2Fneo.12656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Sakamoto, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrade, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felipe, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoyama, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pittella Silva, F.</span><span> </span><span class="NLM_article-title">SMYD2 is highly expressed in pediatric acute lymphoblastic leukemia and constitutes a bad prognostic factor</span> <span class="citation_source-journal">Leuk. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">496</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1016/j.leukres.2014.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.leukres.2014.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=24631370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktlWlu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2014&pages=496-502&author=L.+H.+Sakamotoauthor=R.+V.+Andradeauthor=M.+S.+Felipeauthor=A.+B.+Motoyamaauthor=F.+Pittella+Silva&title=SMYD2+is+highly+expressed+in+pediatric+acute+lymphoblastic+leukemia+and+constitutes+a+bad+prognostic+factor&doi=10.1016%2Fj.leukres.2014.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">SMYD2 is highly expressed in pediatric acute lymphoblastic leukemia and constitutes a bad prognostic factor</span></div><div class="casAuthors">Sakamoto, Luis Henrique Toshihiro; Andrade, Rosangela Vieira de; Felipe, Maria Sueli Soares; Motoyama, Andrea Barretto; Pittella Silva, Fabio</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">496-502</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Acute lymphoblastic leukemia (ALL) is the most common childhood malignancy.  Although several clin. characteristics can be assocd. with worse prognosis, more robust biol. markers still remains uncovered.  SMYD2, a member of SMYD protein family, regulates the activity of several proteins through methylation.  In this study, we performed quant. real time PCR to compare the expression of SMYD2 in 83 pediatric ALL patients and non-neoplastic bone marrow samples (BMS).  The study revealed that SMYD2 expression is altered in ALL BMS and its high expression was correlated with a bad prognosis.  Moreover, we also revealed that SMYD2 expression level significantly decreases in patients that respond to chemotherapy treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCUDTlFWqR_bVg90H21EOLACvtfcHk0lhij1ygy-kvQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktlWlu7s%253D&md5=001e2c0022c172b25ae457f0dc169879</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2014.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2014.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DL.%2BH.%26aulast%3DAndrade%26aufirst%3DR.%2BV.%26aulast%3DFelipe%26aufirst%3DM.%2BS.%26aulast%3DMotoyama%26aufirst%3DA.%2BB.%26aulast%3DPittella%2BSilva%26aufirst%3DF.%26atitle%3DSMYD2%2520is%2520highly%2520expressed%2520in%2520pediatric%2520acute%2520lymphoblastic%2520leukemia%2520and%2520constitutes%2520a%2520bad%2520prognostic%2520factor%26jtitle%3DLeuk.%2520Res.%26date%3D2014%26volume%3D38%26spage%3D496%26epage%3D502%26doi%3D10.1016%2Fj.leukres.2014.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Pimkhaokham, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurihara, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imoto, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.-Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imamura, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amagasa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inazawa, J.</span><span> </span><span class="NLM_article-title">Nonrandom chromosomal imbalances in esophageal squamous cell carcinoma cell lines: possible involvement of the <i>ATF3</i> and <i>CENPF</i> genes in the 1q32 amplicon</span> <span class="citation_source-journal">Jpn. J. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">1126</span><span class="NLM_x">–</span> <span class="NLM_lpage">1133</span><span class="refDoi"> DOI: 10.1111/j.1349-7006.2000.tb00895.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1111%2Fj.1349-7006.2000.tb00895.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=11092977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVGrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2000&pages=1126-1133&author=A.+Pimkhaokhamauthor=Y.+Shimadaauthor=Y.+Fukudaauthor=N.+Kuriharaauthor=I.+Imotoauthor=Z.-Q.+Yangauthor=M.+Imamuraauthor=Y.+Nakamuraauthor=T.+Amagasaauthor=J.+Inazawa&title=Nonrandom+chromosomal+imbalances+in+esophageal+squamous+cell+carcinoma+cell+lines%3A+possible+involvement+of+the+ATF3+and+CENPF+genes+in+the+1q32+amplicon&doi=10.1111%2Fj.1349-7006.2000.tb00895.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Nonrandom chromosomal imbalances in esophageal squamous cell carcinoma cell lines: possible involvement of the ATF3 and CENPF genes in the 1q32 amplicon</span></div><div class="casAuthors">Pimkhaokham, Atiphan; Shimada, Yutaka; Fukuda, Yohji; Kurihara, Naoki; Imoto, Issei; Yang, Zeng-Quan; Imamura, Masayuki; Nakamura, Yusuke; Amagasa, Teruo; Inazawa, Johji</div><div class="citationInfo"><span class="NLM_cas:title">Japanese Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1126-1133</span>CODEN:
                <span class="NLM_cas:coden">JJCREP</span>;
        ISSN:<span class="NLM_cas:issn">0910-5050</span>.
    
            (<span class="NLM_cas:orgname">Japanese Cancer Association</span>)
        </div><div class="casAbstract">Using comparative genomic hybridization (CGH), the authors investigated copy no. aberrations in 29 esophageal squamous cell carcinoma (ESC) cell lines.  All lines displayed numerous chromosome imbalances.  The most frequent losses were obsd. on chromosome 18q (65.5%), Xp (48.3%), 3p (44.8%), 4q (44.8%), 8p (41.4%), 11q23-25 (34.5%) and 4p (27.6%), whereas the most common copy no. gains were noted at 8q (86.2%), 3q (82.8%), 5p (69%), 7p (69%), 20q (65.5%), 9q (55.2%), 11q (55.2%), 1q (48.3%), Xq (44.8%) and 18p (37.9%).  High-level gains (HLGs) were detected at 3q26 (9 cases), 8q23 (6 cases), 5p14-15 (6 cases), 18p11.2-11.3 (6 cases), 3q27-28 (5 cases), 5p13 (3 cases), 7p14-15 (3 cases), 20q12-13 (3 cases), 11q13 (3 cases), 14q21 (2 cases), 20p11.2 (2 cases), 13q32 (2 case), and 1q32 (1 case).  Among them, HLGs of 1q32 have been reported in other types of cancer, including glioblastoma and breast cancers.  The authors successfully narrowed down the smallest common amplicon involving 1q-gain to the genomic segment between D1S414 and D1S2860 by fluorescence in situ hybridization (FISH).  Southern and northern blot anal. clearly demonstrated that ATF3, human activating transcription factor-3 and CENPF, centromere protein F, mapped within this region, were significantly amplified and over-expressed in 1q32 amplicon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqneEUHUiLt-LVg90H21EOLACvtfcHk0lhij1ygy-kvQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVGrsL0%253D&md5=6e514f7c3de4599fb56110b1b85f3012</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2000.tb00895.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2000.tb00895.x%26sid%3Dliteratum%253Aachs%26aulast%3DPimkhaokham%26aufirst%3DA.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DFukuda%26aufirst%3DY.%26aulast%3DKurihara%26aufirst%3DN.%26aulast%3DImoto%26aufirst%3DI.%26aulast%3DYang%26aufirst%3DZ.-Q.%26aulast%3DImamura%26aufirst%3DM.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DAmagasa%26aufirst%3DT.%26aulast%3DInazawa%26aufirst%3DJ.%26atitle%3DNonrandom%2520chromosomal%2520imbalances%2520in%2520esophageal%2520squamous%2520cell%2520carcinoma%2520cell%2520lines%253A%2520possible%2520involvement%2520of%2520the%2520ATF3%2520and%2520CENPF%2520genes%2520in%2520the%25201q32%2520amplicon%26jtitle%3DJpn.%2520J.%2520Cancer%2520Res.%26date%3D2000%26volume%3D91%26spage%3D1126%26epage%3D1133%26doi%3D10.1111%2Fj.1349-7006.2000.tb00895.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Brown, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottlieb, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, P. W.</span><span> </span><span class="NLM_article-title">Identification and characterization of Smyd2: a split SET/MYND domain-containing histone H3 lysine 36-specific methyltransferase that interacts with the Sin3 histone deacetylase complex</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">26</span><span class="refDoi"> DOI: 10.1186/1476-4598-5-26</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1186%2F1476-4598-5-26" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=16805913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A280%3ADC%252BD28vlslagtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=26&author=M.+A.+Brownauthor=R.+J.+Simsauthor=P.+D.+Gottliebauthor=P.+W.+Tucker&title=Identification+and+characterization+of+Smyd2%3A+a+split+SET%2FMYND+domain-containing+histone+H3+lysine+36-specific+methyltransferase+that+interacts+with+the+Sin3+histone+deacetylase+complex&doi=10.1186%2F1476-4598-5-26"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of Smyd2: a split SET/MYND domain-containing histone H3 lysine 36-specific methyltransferase that interacts with the Sin3 histone deacetylase complex</span></div><div class="casAuthors">Brown Mark A; Sims Robert J 3rd; Gottlieb Paul D; Tucker Philip W</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Disrupting the balance of histone lysine methylation alters the expression of genes involved in tumorigenesis including proto-oncogenes and cell cycle regulators.  Methylation of lysine residues is commonly catalyzed by a family of proteins that contain the SET domain.  Here, we report the identification and characterization of the SET domain-containing protein, Smyd2.  RESULTS:  Smyd2 mRNA is most highly expressed in heart and brain tissue, as demonstrated by northern analysis and in situ hybridization.  Over-expressed Smyd2 localizes to the cytoplasm and the nucleus in 293T cells.  Although accumulating evidence suggests that methylation of histone 3, lysine 36 (H3K36) is associated with actively transcribed genes, we show that the SET domain of Smyd2 mediates H3K36 dimethylation and that Smyd2 represses transcription from an SV40-luciferase reporter.  Smyd2 associates specifically with the Sin3A histone deacetylase complex, which was recently linked to H3K36 methylation within the coding regions of active genes in yeast.  Finally, we report that exogenous expression of Smyd2 suppresses cell proliferation.  CONCLUSION:  We propose that Sin3A-mediated deacetylation within the coding regions of active genes is directly linked to the histone methyltransferase activity of Smyd2.  Moreover, Smyd2 appears to restrain cell proliferation, likely through direct modulation of chromatin structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTT6a8vy2w9sbrIbt-tGZmxfW6udTcc2eb_qSZ47jUyRbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28vlslagtg%253D%253D&md5=af9e05d60f2cd0880783270911dc281f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-5-26&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-5-26%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DM.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DGottlieb%26aufirst%3DP.%2BD.%26aulast%3DTucker%26aufirst%3DP.%2BW.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520Smyd2%253A%2520a%2520split%2520SET%252FMYND%2520domain-containing%2520histone%2520H3%2520lysine%252036-specific%2520methyltransferase%2520that%2520interacts%2520with%2520the%2520Sin3%2520histone%2520deacetylase%2520complex%26jtitle%3DMol.%2520Cancer%26date%3D2006%26volume%3D5%26spage%3D26%26doi%3D10.1186%2F1476-4598-5-26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Abu-Farha, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Madhoun, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elisma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skerjanc, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figeys, D.</span><span> </span><span class="NLM_article-title">The tale of two domains: proteomics and genomics analysis of SMYD2, a new histone methyltransferase</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">560</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span><span class="refDoi"> DOI: 10.1074/mcp.M700271-MCP200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1074%2Fmcp.M700271-MCP200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=18065756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFygt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=560-572&author=M.+Abu-Farhaauthor=J.-P.+Lambertauthor=A.+S.+Al-Madhounauthor=F.+Elismaauthor=I.+S.+Skerjancauthor=D.+Figeys&title=The+tale+of+two+domains%3A+proteomics+and+genomics+analysis+of+SMYD2%2C+a+new+histone+methyltransferase&doi=10.1074%2Fmcp.M700271-MCP200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The tale of two domains: proteomics and genomics analysis of SMYD2, a new histone methyltransferase</span></div><div class="casAuthors">Abu-Farha, Mohamed; Lambert, Jean-Philippe; Al-Madhoun, Ashraf S.; Elisma, Fred; Skerjanc, Ilona S.; Figeys, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">560-572</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9476</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Very little is known about SET- and MYND-contg. protein 2 (SMYD2), a member of the SMYD protein family.  However, the interest in better understanding the roles of SMYD2 has grown because of recent reports indicating that SMYD2 methylates p53 and histone H3.  In this study, we present a combined proteomics and genomics study of SMYD2 designed to elucidate its mol. roles.  We report the cytosolic and nuclear interactome of SMYD2 using a combination of immunopptn. coupled with high throughput MS, chromatin immunopptn. coupled with high throughput MS, and co-immunopptn. methods.  In particular, we report that SMYD2 interacted with HSP90α independently of the SET and MYND domains, with EBP41L3 through the MYND domain, and with p53 through the SET domain.  We demonstrated that the interaction of SMYD2 with HSP90α enhances SMYD2 histone methyltransferase activity and specificity for histone H3 at lysine 4 (H3K4) in vitro.  Interestingly histone H3K36 methyltransferase activity was independent of its interaction with HSP90α similar to LSD1 dependence on the androgen receptor.  We also showed that the SET domain is required for the methylation at H3K4.  We demonstrated using a modified chromatin immunopptn. protocol that the SMYD2 gain of function leads to an increase in H3K4 methylation in vivo, whereas no observable levels of H3K36 were detected.  We also report that the SMYD2 gain of function was correlated with the upregulation of 37 and down-regulation of four genes, the majority of which are involved in the cell cycle, chromatin remodeling, and transcriptional regulation.  TACC2 is one of the genes up-regulated as a result of SMYD2 gain of function.  Up-regulation of TACC2 by SMYD2 occurred as a result of SMYD2 binding to the TACC2 promoter where it methylates H3K4.  Furthermore the combination of the SMYD2 interactome with the gene expression data suggests that some of the genes regulated by SMYD2 are closely assocd. with SMYD2-interacting proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsnP4zSi0t2LVg90H21EOLACvtfcHk0lj8ULfGqKBSDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFygt7w%253D&md5=0d3fe1f7c3be7472bb4d25c7ed5b93e6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M700271-MCP200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M700271-MCP200%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Farha%26aufirst%3DM.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DAl-Madhoun%26aufirst%3DA.%2BS.%26aulast%3DElisma%26aufirst%3DF.%26aulast%3DSkerjanc%26aufirst%3DI.%2BS.%26aulast%3DFigeys%26aufirst%3DD.%26atitle%3DThe%2520tale%2520of%2520two%2520domains%253A%2520proteomics%2520and%2520genomics%2520analysis%2520of%2520SMYD2%252C%2520a%2520new%2520histone%2520methyltransferase%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2008%26volume%3D7%26spage%3D560%26epage%3D572%26doi%3D10.1074%2Fmcp.M700271-MCP200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Hamamoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saloura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span> </span><span class="NLM_article-title">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1038/nrc3884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnrc3884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25614009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=110-124&author=R.+Hamamotoauthor=V.+Salouraauthor=Y.+Nakamura&title=Critical+roles+of+non-histone+protein+lysine+methylation+in+human+tumorigenesis&doi=10.1038%2Fnrc3884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Critical roles of non-histone protein lysine methylation in human tumorigenesis</span></div><div class="casAuthors">Hamamoto, Ryuji; Saloura, Vassiliki; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">110-124</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Several protein lysine methyltransferases and demethylases have been identified to have crit. roles in histone modification.  A large body of evidence has indicated that their dysregulation is involved in the development and progression of various diseases, including cancer, and these enzymes are now considered to be potential therapeutic targets.  Although most studies have focused on histone methylation, many reports have revealed that these enzymes also regulate the methylation dynamics of non-histone proteins such as p53, RB1 and STAT3 (signal transducer and activator of transcription 3), which have important roles in human tumorigenesis.  In this Review, we summarize the mol. functions of protein lysine methylation and its involvement in human cancer, with a particular focus on lysine methylation of non-histone proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0wn_Jk7jSGbVg90H21EOLACvtfcHk0lj8ULfGqKBSDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVartrs%253D&md5=9d998a4efb3c1a069e14747341ceca5e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc3884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3884%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DSaloura%26aufirst%3DV.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DCritical%2520roles%2520of%2520non-histone%2520protein%2520lysine%2520methylation%2520in%2520human%2520tumorigenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D110%26epage%3D124%26doi%3D10.1038%2Fnrc3884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Benayoun, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veitia, R. A.</span><span> </span><span class="NLM_article-title">A post-translational modification code for transcription factors: sorting through a sea of signals</span> <span class="citation_source-journal">Trends Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">189</span><span class="NLM_x">–</span> <span class="NLM_lpage">197</span><span class="refDoi"> DOI: 10.1016/j.tcb.2009.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.tcb.2009.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19328693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVersbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=189-197&author=B.+A.+Benayounauthor=R.+A.+Veitia&title=A+post-translational+modification+code+for+transcription+factors%3A+sorting+through+a+sea+of+signals&doi=10.1016%2Fj.tcb.2009.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A post-translational modification code for transcription factors: sorting through a sea of signals</span></div><div class="casAuthors">Benayoun, Berenice A.; Veitia, Reiner A.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">189-197</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cellular responses to environmental or physiol. cues rely on transduction pathways that must ensure discrimination between different signals.  These cascades crosstalk and lead to a combinatorial regulation.  This often results in different combinations of post-translational modifications (PTMs) on target proteins, which might act as a mol. barcode.  Although appealing, the idea of the existence of such a code for transcription factors is debated.  Using general arguments and recent evidence, it is proposed that a PTM code is not only possible but necessary in the context of transcription factors regulating multiple processes.  Thus, the coding potential of PTM combinations should both provide a further layer of information integration from several transduction pathways and warrant highly specific cellular outputs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomvF_m-rjksbVg90H21EOLACvtfcHk0liFpQ0WIxEp1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVersbs%253D&md5=cbc800258bc1a04d9bf2d67e1abddd0f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2009.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2009.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DBenayoun%26aufirst%3DB.%2BA.%26aulast%3DVeitia%26aufirst%3DR.%2BA.%26atitle%3DA%2520post-translational%2520modification%2520code%2520for%2520transcription%2520factors%253A%2520sorting%2520through%2520a%2520sea%2520of%2520signals%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D189%26epage%3D197%26doi%3D10.1016%2Fj.tcb.2009.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Fan, J.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, P.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, J.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.-N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M.</span><span> </span><span class="NLM_article-title">The selective activation of p53 target genes regulated by SMYD2 in BIX-01294 induced autophagy-related cell death</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0116782</span><span class="refDoi"> DOI: 10.1371/journal.pone.0116782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1371%2Fjournal.pone.0116782" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0116782&author=J.-D.+Fanauthor=P.-J.+Leiauthor=J.-Y.+Zhengauthor=X.+Wangauthor=S.+Liauthor=H.+Liuauthor=Y.-L.+Heauthor=Z.-N.+Wangauthor=G.+Weiauthor=X.+Zhangauthor=L.-Y.+Liauthor=M.+Wu&title=The+selective+activation+of+p53+target+genes+regulated+by+SMYD2+in+BIX-01294+induced+autophagy-related+cell+death&doi=10.1371%2Fjournal.pone.0116782"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0116782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0116782%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DJ.-D.%26aulast%3DLei%26aufirst%3DP.-J.%26aulast%3DZheng%26aufirst%3DJ.-Y.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DY.-L.%26aulast%3DWang%26aufirst%3DZ.-N.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.-Y.%26aulast%3DWu%26aufirst%3DM.%26atitle%3DThe%2520selective%2520activation%2520of%2520p53%2520target%2520genes%2520regulated%2520by%2520SMYD2%2520in%2520BIX-01294%2520induced%2520autophagy-related%2520cell%2520death%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0116782%26doi%3D10.1371%2Fjournal.pone.0116782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Arenas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span> </span><span class="NLM_article-title">Structural basis of substrate methylation and inhibition of SMYD2</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1262</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1016/j.str.2011.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.str.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21782458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1262-1273&author=A.+D.+Fergusonauthor=N.+A.+Larsenauthor=T.+Howardauthor=H.+Pollardauthor=I.+Greenauthor=C.+Grandeauthor=T.+Cheungauthor=R.+Garcia-Arenasauthor=S.+Cowenauthor=J.+Wuauthor=R.+Godinauthor=H.+Chenauthor=N.+Keen&title=Structural+basis+of+substrate+methylation+and+inhibition+of+SMYD2&doi=10.1016%2Fj.str.2011.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Substrate Methylation and Inhibition of SMYD2</span></div><div class="casAuthors">Ferguson, Andrew D.; Larsen, Nicholas A.; Howard, Tina; Pollard, Hannah; Green, Isabelle; Grande, Christie; Cheung, Tony; Garcia-Arenas, Renee; Cowen, Scott; Wu, Jiaquan; Godin, Robert; Chen, Huawei; Keen, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1262-1273</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases are important regulators of epigenetic signaling.  These enzymes catalyze the transfer of donor Me groups from S-adenosylmethionine to specific acceptor lysines on histones, leading to changes in chromatin structure and transcriptional regulation.  These enzymes also methylate nonhistone protein substrates, revealing an addnl. mechanism to regulate cellular physiol.  The oncogenic protein SMYD2 represses the functional activities of the tumor suppressor proteins p53 and Rb, making it an attractive drug target.  Here we report the discovery of AZ505, a potent and selective inhibitor of SMYD2 that was identified from a high throughput chem. screen.  We also present the crystal structures of SMYD2 with p53 substrate and product peptides, and notably, in complex with AZ505.  This substrate competitive inhibitor is bound in the peptide binding groove of SMYD2.  These results have implications for the development of SMYD2 inhibitors, and indicate the potential for developing novel therapies targeting this target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfo0RkV17Th7Vg90H21EOLACvtfcHk0liFpQ0WIxEp1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fK&md5=31a08f46bffea75609e1c5bb25b7c610</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLarsen%26aufirst%3DN.%2BA.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DPollard%26aufirst%3DH.%26aulast%3DGreen%26aufirst%3DI.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGarcia-Arenas%26aufirst%3DR.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGodin%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKeen%26aufirst%3DN.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520methylation%2520and%2520inhibition%2520of%2520SMYD2%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1262%26epage%3D1273%26doi%3D10.1016%2Fj.str.2011.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Burgos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Placek, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">Repression of p53 activity by Smyd2-mediated methylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">444</span><span class="NLM_x">, </span> <span class="NLM_fpage">629</span><span class="NLM_x">–</span> <span class="NLM_lpage">632</span><span class="refDoi"> DOI: 10.1038/nature05287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnature05287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=17108971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1KjurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=444&publication_year=2006&pages=629-632&author=J.+Huangauthor=L.+Perez-Burgosauthor=B.+J.+Placekauthor=R.+Senguptaauthor=M.+Richterauthor=J.+A.+Dorseyauthor=S.+Kubicekauthor=S.+Opravilauthor=T.+Jenuweinauthor=S.+L.+Berger&title=Repression+of+p53+activity+by+Smyd2-mediated+methylation&doi=10.1038%2Fnature05287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Repression of p53 activity by Smyd2-mediated methylation</span></div><div class="casAuthors">Huang, Jing; Perez-Burgos, Laura; Placek, Brandon J.; Sengupta, Roopsha; Richter, Mario; Dorsey, Jean A.; Kubicek, Stefan; Opravil, Susanne; Jenuwein, Thomas; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">444</span>
        (<span class="NLM_cas:issue">7119</span>),
    <span class="NLM_cas:pages">629-632</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Specific sites of lysine methylation on histones correlate with either activation or repression of transcription.  The tumor suppressor p53 (refs 4-7) is one of only a few non-histone proteins known to be regulated by lysine methylation.  Here we report a lysine methyltransferase, Smyd2, that methylates a previously unidentified site, Lys 370, in p53.  This methylation site, in contrast to the known site Lys 372, is repressing to p53-mediated transcriptional regulation.  Smyd2 helps to maintain low concns. of promoter-assocd. p53.  We show that reducing Smyd2 concns. by short interfering RNA enhances p53-mediated apoptosis.  We find that Set9-mediated methylation of Lys 372 inhibits Smyd2-mediated methylation of Lys 370, providing regulatory cross-talk between post-translational modifications.  In addn., we show that the inhibitory effect of Lys 372 methylation on Lys 370 methylation is caused, in part, by blocking the interaction between p53 and Smyd2.  Thus, similar to histones, p53 is subject to both activating and repressing lysine methylation.  These results also predict that Smyd2 may function as a putative oncogene by methylating p53 and repressing its tumor suppressive function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUh0EYS8bhMrVg90H21EOLACvtfcHk0liFpQ0WIxEp1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1KjurjP&md5=28ecb05d42c8631445e6563676406e32</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature05287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05287%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DPerez-Burgos%26aufirst%3DL.%26aulast%3DPlacek%26aufirst%3DB.%2BJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DDorsey%26aufirst%3DJ.%2BA.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DRepression%2520of%2520p53%2520activity%2520by%2520Smyd2-mediated%2520methylation%26jtitle%3DNature%26date%3D2006%26volume%3D444%26spage%3D629%26epage%3D632%26doi%3D10.1038%2Fnature05287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirinupong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span> </span><span class="NLM_article-title">Crystal structures of histone and p53 methyltransferase SmyD2 reveal a conformational flexibility of the autoinhibitory C-terminal domain</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e21640</span><span class="refDoi"> DOI: 10.1371/journal.pone.0021640</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1371%2Fjournal.pone.0021640" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21738746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVaitbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e21640&author=Y.+Jiangauthor=N.+Sirinupongauthor=J.+Brunzelleauthor=Z.+Yang&title=Crystal+structures+of+histone+and+p53+methyltransferase+SmyD2+reveal+a+conformational+flexibility+of+the+autoinhibitory+C-terminal+domain&doi=10.1371%2Fjournal.pone.0021640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of histone and p53 methyltransferase SmyD2 reveal a conformational flexibility of the autoinhibitory C-terminal domain</span></div><div class="casAuthors">Jiang, Yuanyuan; Sirinupong, Nualpun; Brunzelle, Joseph; Yang, Zhe</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e21640</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Histone methyltransferase SmyD2 belongs to a new class of chromatin regulators that control gene expression in heart development and tumorigenesis.  In addn. to methylation of histone H3 Lys-4, SmyD2 can methylate non-histone targets including p53 and the retinoblastoma tumor suppressor.  The methyltransferase activity of SmyD proteins has been proposed to be regulated by autoinhibition via the intra- and inter-domain bending of the conserved C-terminal domain (CTD).  However, there has been no direct evidence of a conformational change in the CTD.  Here, the authors report 2 crystal structures of SmyD2 bound either to the cofactor product, S-adenosylhomocysteine, or to the inhibitor, sinefungin.  SmyD2 was found to have a 2-lobed structure with the active site located at the bottom of a deep crevice formed between the CTD and the catalytic domain.  By extensive engagement with the methyltransferase domain, the CTD stabilized the autoinhibited conformation of SmyD2 and restricted access to the catalytic site.  Unexpectedly, despite that the 2 SmyD2 structures were highly superimposable, significant differences were obsd. in the 1st 2 helixes of the CTDs: the 2 helixes were found to bend outward and move away from the catalytic domain to generate a less closed conformation in the sinefungin-bound structure.  Although the overall fold of the individual domains was structurally conserved among SmyD proteins, SmyD2 appeared to be a conformational "intermediate" between a close form of SmyD3 and an open form of SmyD1.  In addn., the structures revealed that the CTD was structurally similar to tetratricopeptide repeats (TPR), a motif through which many cochaperones bind to heat-shock protein Hsp90.  These results thus provide the 1st evidence for the intradomain flexibility of the TPR-like CTD, which may be important for the activation of SmyD proteins by Hsp90.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixUn8onz2U7Vg90H21EOLACvtfcHk0licgURSUpeq5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosVaitbs%253D&md5=7507f0056e8de2c60bec871894bdaf59</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0021640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0021640%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DSirinupong%26aufirst%3DN.%26aulast%3DBrunzelle%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DCrystal%2520structures%2520of%2520histone%2520and%2520p53%2520methyltransferase%2520SmyD2%2520reveal%2520a%2520conformational%2520flexibility%2520of%2520the%2520autoinhibitory%2520C-terminal%2520domain%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De21640%26doi%3D10.1371%2Fjournal.pone.0021640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Sajjad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novoyatleva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vergarajauregui, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troidl, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schermuly, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, H. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, F. B.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase Smyd2 suppresses p53-dependent cardiomyocyte apoptosis</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1843</span><span class="NLM_x">, </span> <span class="NLM_fpage">2556</span><span class="NLM_x">–</span> <span class="NLM_lpage">2562</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2014.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.bbamcr.2014.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25014164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1yksr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1843&publication_year=2014&pages=2556-2562&author=A.+Sajjadauthor=T.+Novoyatlevaauthor=S.+Vergarajaureguiauthor=C.+Troidlauthor=R.+T.+Schermulyauthor=H.+O.+Tuckerauthor=F.+B.+Engel&title=Lysine+methyltransferase+Smyd2+suppresses+p53-dependent+cardiomyocyte+apoptosis&doi=10.1016%2Fj.bbamcr.2014.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methyltransferase Smyd2 suppresses p53-dependent cardiomyocyte apoptosis</span></div><div class="casAuthors">Sajjad, Amna; Novoyatleva, Tatyana; Vergarajauregui, Silvia; Troidl, Christian; Schermuly, Ralph T.; Tucker, Haley O.; Engel, Felix B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1843</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2556-2562</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Apoptosis, or programmed cell death, is an essential physiol. process for proper embryogenesis as well as for homeostasis during aging.  In addn., apoptosis is one of the major mechanisms causing cell loss in pathophysiol. conditions such as heart failure.  Thus, inhibition of apoptosis is an important approach for preventive and therapeutic strategies.  Here we show that the histone 3 lysine 4- and lysine 36-specific methyltransferase Smyd2 acts as an endogenous antagonistic player of p53-dependent cardiomyocyte apoptosis.  Smyd2 protein levels were significantly decreased in cardiomyocytes upon cobalt chloride-induced apoptosis or myocardial infarction, while p53 expression was enhanced.  SiRNA-mediated knockdown of Smyd2 in cultured cardiomyocytes further enhanced cobalt chloride-induced cardiomyocyte apoptosis.  In contrast, Smyd2 overexpression resulted in marked methylation of p53 and prevented its accumulation as well as apoptotic cell death in an Hsp90-independent manner.  Moreover, overexpression, of Smyd2, but not Smyd2Y240F lacking a Me transferase activity, significantly rescued CoCl2-induced apoptosis in H9c2 cardioblasts.  Finally, Smyd2 cardiomyocyte-specific deletion in vivo promoted apoptotic cell death upon myocardial infarction, which correlated with enhanced expression of p53 and pro-apoptotic Bax.  Collectively, our data indicate Smyd2 as a cardioprotective protein by methylating p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfw3CFurADyLVg90H21EOLACvtfcHk0licgURSUpeq5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1yksr3J&md5=327a9367f05026a05ec527e18f71b26f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2014.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2014.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DSajjad%26aufirst%3DA.%26aulast%3DNovoyatleva%26aufirst%3DT.%26aulast%3DVergarajauregui%26aufirst%3DS.%26aulast%3DTroidl%26aufirst%3DC.%26aulast%3DSchermuly%26aufirst%3DR.%2BT.%26aulast%3DTucker%26aufirst%3DH.%2BO.%26aulast%3DEngel%26aufirst%3DF.%2BB.%26atitle%3DLysine%2520methyltransferase%2520Smyd2%2520suppresses%2520p53-dependent%2520cardiomyocyte%2520apoptosis%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2014%26volume%3D1843%26spage%3D2556%26epage%3D2562%26doi%3D10.1016%2Fj.bbamcr.2014.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">Structure of human SMYD2 protein reveals the basis of p53 tumor suppressor methylation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">38725</span><span class="NLM_x">–</span> <span class="NLM_lpage">38737</span><span class="refDoi"> DOI: 10.1074/jbc.M111.262410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1074%2Fjbc.M111.262410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21880715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlykurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=38725-38737&author=L.+Wangauthor=L.+Liauthor=H.+Zhangauthor=X.+Luoauthor=J.+Daiauthor=S.+Zhouauthor=J.+Guauthor=J.+Zhuauthor=P.+Atadjaauthor=C.+Luauthor=E.+Liauthor=K.+Zhao&title=Structure+of+human+SMYD2+protein+reveals+the+basis+of+p53+tumor+suppressor+methylation&doi=10.1074%2Fjbc.M111.262410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human SMYD2 Protein Reveals the Basis of p53 Tumor Suppressor Methylation</span></div><div class="casAuthors">Wang, Li; Li, Ling; Zhang, Hailong; Luo, Xiao; Dai, Jingquan; Zhou, Shaolian; Gu, Justin; Zhu, Jidong; Atadja, Peter; Lu, Chris; Li, En; Zhao, Kehao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">38725-38737, S38725/1-S38725/5</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">SMYD2 belongs to a subfamily of histone lysine methyltransferase and was recently identified to methylate tumor suppressor p53 and Rb.  Here we report that SMYD2 prefers to methylate p53 Lys-370 over histone substrates in vitro.  Consistently, the level of endogenous p53 Lys-370 monomethylation is significantly elevated when SMYD2 is overexpressed in vivo.  We have solved the high resoln. crystal structures of the full-length SMYD2 protein in binary complex with its cofactor S-adenosylmethionine and in ternary complex with cofactor product S-adenosylhomocysteine and p53 substrate peptide (residues 368-375), resp. p53 peptide binds to a deep pocket of the interface between catalytic SET(1-282) and C-terminal domain (CTD) with an unprecedented U-shaped conformation.  Subtle conformational change exists around the p53 binding site between the binary and ternary structures, in particular the tetratricopeptide repeat motif of the CTD.  In addn., a unique EDEE motif between the loop of anti-parallel β7 and β8 sheets of the SET core not only interacts with p53 substrate but also forms a hydrogen bond network with residues from CTD.  These observations suggest that the tetratricopeptide repeat and EDEE motif may play an important role in detg. p53 substrate binding specificity.  This is further verified by the findings that deletion of the CTD domain drastically reduces the methylation activity of SMYD2 to p53 protein.  Meanwhile, mutation of EDEE residues impairs both the binding and the enzymic activity of SMYD2 to p53 Lys-370.  These data together reveal the mol. basis of SMYD2 in specifically recognizing and regulating functions of p53 tumor suppressor through Lys-370 monomethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX6G4Yhf_H47Vg90H21EOLACvtfcHk0licgURSUpeq5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlykurvI&md5=671857a22846de5d607e83a03dd52d37</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.262410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.262410%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DStructure%2520of%2520human%2520SMYD2%2520protein%2520reveals%2520the%2520basis%2520of%2520p53%2520tumor%2520suppressor%2520methylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D38725%26epage%3D38737%26doi%3D10.1074%2Fjbc.M111.262410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grande, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theriault, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Code, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birr, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span> </span><span class="NLM_article-title">Biochemical characterization of human SET and MYND domain-containing protein 2 methyltransferase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">6488</span><span class="NLM_x">–</span> <span class="NLM_lpage">6497</span><span class="refDoi"> DOI: 10.1021/bi200725p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200725p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=6488-6497&author=J.+Wuauthor=T.+Cheungauthor=C.+Grandeauthor=A.+D.+Fergusonauthor=X.+Zhuauthor=K.+Theriaultauthor=E.+Codeauthor=C.+Birrauthor=N.+Keenauthor=H.+Chen&title=Biochemical+characterization+of+human+SET+and+MYND+domain-containing+protein+2+methyltransferase&doi=10.1021%2Fbi200725p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fbi200725p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200725p%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DGrande%26aufirst%3DC.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DTheriault%26aufirst%3DK.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DBirr%26aufirst%3DC.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DBiochemical%2520characterization%2520of%2520human%2520SET%2520and%2520MYND%2520domain-containing%2520protein%25202%2520methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D6488%26epage%3D6497%26doi%3D10.1021%2Fbi200725p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Barros Filho, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katayama, M. L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brentani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abreu, A. P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Góes, J. C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brentani, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folgueira, M. A. A. K.</span><span> </span><span class="NLM_article-title">Gene trio signatures as molecular markers to predict response to doxorubicin cyclophosphamide neoadjuvant chemotherapy in breast cancer patients</span> <span class="citation_source-journal">Braz. J. Med. Biol. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1225</span><span class="NLM_x">–</span> <span class="NLM_lpage">1231</span><span class="refDoi"> DOI: 10.1590/S0100-879X2010007500135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1590%2FS0100-879X2010007500135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21103787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivFehtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=1225-1231&author=M.+C.+Barros+Filhoauthor=M.+L.+H.+Katayamaauthor=H.+Brentaniauthor=A.+P.+S.+Abreuauthor=E.+M.+Barbosaauthor=C.+T.+Oliveiraauthor=J.+C.+S.+G%C3%B3esauthor=M.+M.+Brentaniauthor=M.+A.+A.+K.+Folgueira&title=Gene+trio+signatures+as+molecular+markers+to+predict+response+to+doxorubicin+cyclophosphamide+neoadjuvant+chemotherapy+in+breast+cancer+patients&doi=10.1590%2FS0100-879X2010007500135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Gene trio signatures as molecular markers to predict response to doxorubicin cyclophosphamide neoadjuvant chemotherapy in breast cancer patients</span></div><div class="casAuthors">Barros Filho, M. C.; Katayama, M. L. H.; Brentani, H.; Abreu, A. P. S.; Barbosa, E. M.; Oliveira, C. T.; Goes, J. C. S.; Brentani, M. M.; Folgueira, M. A. A. K.</div><div class="citationInfo"><span class="NLM_cas:title">Brazilian Journal of Medical and Biological Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1225-1231</span>CODEN:
                <span class="NLM_cas:coden">BJMRDK</span>;
        ISSN:<span class="NLM_cas:issn">0100-879X</span>.
    
            (<span class="NLM_cas:orgname">Associacao Brasileira de Divulgacao Cientifica</span>)
        </div><div class="casAbstract">In breast cancer patients submitted to neoadjuvant chemotherapy (4 cycles of doxorubicin and cyclophosphamide, AC), expression of groups of three genes (gene trio signatures) could distinguish responsive from non-responsive tumors, as demonstrated by cDNA microarray profiling in a previous study by our group.  In the current study, we detd. if the expression of the same genes would retain the predictive strength, when analyzed by a more accessible technique (real-time RT-PCR).  We evaluated 28 samples already analyzed by cDNA microarray, as a tech. validation procedure, and 14 tumors, as an independent biol. validation set.  All patients received neoadjuvant chemotherapy (4 AC).  Among five trio combinations previously identified, defined by nine genes individually investigated (BZRP, CLPTM1, MTSS1, NOTCH1, NUP210, PRSS11, RPL37A, SMYD2, and XLHSRF-1), the most accurate were established by RPL37A, XLHSRF-1 based trios, with NOTCH1 or NUP210.  Both trios correctly sepd. 86% of tumors (87% sensitivity and 80% specificity for predicting response), according to their response to chemotherapy (82% in a leave-one-out cross-validation method).  Using the pre-established features obtained by linear discriminant anal., 71% samples from the biol. validation set were also correctly classified by both trios (72% sensitivity; 66% specificity).  Furthermore, we explored other gene combinations to achieve a higher accuracy in the tech. validation group (as a training set).  A new trio, MTSS1, RPL37 and SMYD2, correctly classified 93% of samples from the tech. validation group (95% sensitivity and 80% specificity; 86% accuracy by the cross-validation method) and 79% from the biol. validation group (72% sensitivity and 100% specificity).  Therefore, the combined expression of MTSS1, RPL37 and SMYD2, as evaluated by real-time RT-PCR, is a potential candidate to predict response to neoadjuvant doxorubicin and cyclophosphamide in breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqViAm7L7uXCbVg90H21EOLACvtfcHk0lhdFT72WD6hBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivFehtL8%253D&md5=71367c342457f24fc678e01854404e96</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1590%2FS0100-879X2010007500135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1590%252FS0100-879X2010007500135%26sid%3Dliteratum%253Aachs%26aulast%3DBarros%2BFilho%26aufirst%3DM.%2BC.%26aulast%3DKatayama%26aufirst%3DM.%2BL.%2BH.%26aulast%3DBrentani%26aufirst%3DH.%26aulast%3DAbreu%26aufirst%3DA.%2BP.%2BS.%26aulast%3DBarbosa%26aufirst%3DE.%2BM.%26aulast%3DOliveira%26aufirst%3DC.%2BT.%26aulast%3DG%25C3%25B3es%26aufirst%3DJ.%2BC.%2BS.%26aulast%3DBrentani%26aufirst%3DM.%2BM.%26aulast%3DFolgueira%26aufirst%3DM.%2BA.%2BA.%2BK.%26atitle%3DGene%2520trio%2520signatures%2520as%2520molecular%2520markers%2520to%2520predict%2520response%2520to%2520doxorubicin%2520cyclophosphamide%2520neoadjuvant%2520chemotherapy%2520in%2520breast%2520cancer%2520patients%26jtitle%3DBraz.%2520J.%2520Med.%2520Biol.%2520Res.%26date%3D2010%26volume%3D43%26spage%3D1225%26epage%3D1231%26doi%3D10.1590%2FS0100-879X2010007500135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trescott, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holcomb, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirinupong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span> </span><span class="NLM_article-title">Structural insights into estrogen receptor alpha methylation by histone methyltransferase SMYD2, a cellular event implicated in estrogen signaling regulation</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">426</span><span class="NLM_x">, </span> <span class="NLM_fpage">3413</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1016/j.jmb.2014.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.jmb.2014.02.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=3413-25&author=Y.+Jiangauthor=L.+Trescottauthor=J.+Holcombauthor=X.+Zhangauthor=J.+Brunzelleauthor=N.+Sirinupongauthor=X.+Shiauthor=Z.+Yang&title=Structural+insights+into+estrogen+receptor+alpha+methylation+by+histone+methyltransferase+SMYD2%2C+a+cellular+event+implicated+in+estrogen+signaling+regulation&doi=10.1016%2Fj.jmb.2014.02.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DTrescott%26aufirst%3DL.%26aulast%3DHolcomb%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBrunzelle%26aufirst%3DJ.%26aulast%3DSirinupong%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DStructural%2520insights%2520into%2520estrogen%2520receptor%2520alpha%2520methylation%2520by%2520histone%2520methyltransferase%2520SMYD2%252C%2520a%2520cellular%2520event%2520implicated%2520in%2520estrogen%2520signaling%2520regulation%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D3413%26epage%3D25%26doi%3D10.1016%2Fj.jmb.2014.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barton, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span> </span><span class="NLM_article-title">Regulation of estrogen receptor alpha by histone methyltransferase SMYD2-mediated protein methylation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">17284</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1073/pnas.1307959110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1073%2Fpnas.1307959110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=17284-9&author=X.+Zhangauthor=K.+Tanakaauthor=J.+Yanauthor=J.+Liauthor=D.+Pengauthor=Y.+Jiangauthor=Z.+Yangauthor=M.+C.+Bartonauthor=H.+Wenauthor=X.+Shi&title=Regulation+of+estrogen+receptor+alpha+by+histone+methyltransferase+SMYD2-mediated+protein+methylation&doi=10.1073%2Fpnas.1307959110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1307959110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1307959110%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DBarton%26aufirst%3DM.%2BC.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DX.%26atitle%3DRegulation%2520of%2520estrogen%2520receptor%2520alpha%2520by%2520histone%2520methyltransferase%2520SMYD2-mediated%2520protein%2520methylation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D17284%26epage%3D9%26doi%3D10.1073%2Fpnas.1307959110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Piao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuda, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">The histone methyltransferase SMYD2 methylates PARP1 and promotes poly(ADP-ribosyl)ation activity in cancer cells</span> <span class="citation_source-journal">Neoplasia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">264</span><span class="refDoi"> DOI: 10.1016/j.neo.2014.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.neo.2014.03.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=257-264&author=L.+Piaoauthor=D.+Kangauthor=T.+Suzukiauthor=A.+Masudaauthor=N.+Dohmaeauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=The+histone+methyltransferase+SMYD2+methylates+PARP1+and+promotes+poly%28ADP-ribosyl%29ation+activity+in+cancer+cells&doi=10.1016%2Fj.neo.2014.03.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.neo.2014.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neo.2014.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DPiao%26aufirst%3DL.%26aulast%3DKang%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMasuda%26aufirst%3DA.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DThe%2520histone%2520methyltransferase%2520SMYD2%2520methylates%2520PARP1%2520and%2520promotes%2520poly%2528ADP-ribosyl%2529ation%2520activity%2520in%2520cancer%2520cells%26jtitle%3DNeoplasia%26date%3D2014%26volume%3D16%26spage%3D257%26epage%3D264%26doi%3D10.1016%2Fj.neo.2014.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Abu-Farha, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanouette, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elisma, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tremblay, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figeys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J.-F.</span><span> </span><span class="NLM_article-title">Proteomic analyses of the SMYD family interactomes identify HSP90 as a novel target for SMYD2</span> <span class="citation_source-journal">J. Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">301</span><span class="NLM_x">–</span> <span class="NLM_lpage">308</span><span class="refDoi"> DOI: 10.1093/jmcb/mjr025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1093%2Fjmcb%2Fmjr025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=22028380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFagsL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=301-308&author=M.+Abu-Farhaauthor=S.+Lanouetteauthor=F.+Elismaauthor=V.+Tremblayauthor=J.+Butsonauthor=D.+Figeysauthor=J.-F.+Couture&title=Proteomic+analyses+of+the+SMYD+family+interactomes+identify+HSP90+as+a+novel+target+for+SMYD2&doi=10.1093%2Fjmcb%2Fmjr025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic analyses of the SMYD family interactomes identify HSP90 as a novel target for SMYD2</span></div><div class="casAuthors">Abu-Farha, Mohamed; Lanouette, Sylvain; Elisma, Fred; Tremblay, Veronique; Butson, Jeffery; Figeys, Daniel; Couture, Jean-Francois</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">301-308</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The SMYD (SET and MYND domain) family of lysine methyltransferases (KMTs) plays pivotal roles in various cellular processes, including gene expression regulation and DNA damage response.  Initially identified as genuine histone methyltransferases, specific members of this family have recently been shown to methylate non-histone proteins such as p53, VEGFR, and the retinoblastoma tumor suppressor (pRb).  To gain further functional insights into this family of KMTs, we generated the protein interaction network for three different human SMYD proteins (SMYD2, SMYD3, and SMYD5).  Characterization of each SMYD protein network revealed that they assoc. with both shared and unique sets of proteins.  Among those, we found that HSP90 and several of its co-chaperones interact specifically with the tetratrico peptide repeat (TPR)-contg. SMYD2 and SMYD3.  Moreover, using proteomic and biochem. techniques, we provide evidence that SMYD2 methylates K209 and K615 on HSP90 nucleotide-binding and dimerization domains, resp.  In addn., we found that each methylation site displays unique reactivity in regard to the presence of HSP90 co-chaperones, pH, and demethylation by the lysine amine oxidase LSD1, suggesting that alternative mechanisms control HSP90 methylation by SMYD2.  Altogether, this study highlights the ability of SMYD proteins to form unique protein complexes that may underlie their various biol. functions and the SMYD2-mediated methylation of the key mol. chaperone HSP90.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFwZSHL-gNyLVg90H21EOLACvtfcHk0lidSzq2kL4v_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFagsL7J&md5=d3c5072e6958afc4f82935980ed4b880</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjr025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjr025%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Farha%26aufirst%3DM.%26aulast%3DLanouette%26aufirst%3DS.%26aulast%3DElisma%26aufirst%3DF.%26aulast%3DTremblay%26aufirst%3DV.%26aulast%3DButson%26aufirst%3DJ.%26aulast%3DFigeys%26aufirst%3DD.%26aulast%3DCouture%26aufirst%3DJ.-F.%26atitle%3DProteomic%2520analyses%2520of%2520the%2520SMYD%2520family%2520interactomes%2520identify%2520HSP90%2520as%2520a%2520novel%2520target%2520for%2520SMYD2%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Biol.%26date%3D2011%26volume%3D3%26spage%3D301%26epage%3D308%26doi%3D10.1093%2Fjmcb%2Fmjr025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Hamamoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakakido, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span> </span><span class="NLM_article-title">SMYD2-dependent HSP90 methylation promotes cancer cell proliferation by regulating the chaperone complex formation</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">351</span><span class="NLM_x">, </span> <span class="NLM_fpage">126</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1016/j.canlet.2014.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.canlet.2014.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=24880080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvVejtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2014&pages=126-133&author=R.+Hamamotoauthor=G.+Toyokawaauthor=M.+Nakakidoauthor=K.+Uedaauthor=Y.+Nakamura&title=SMYD2-dependent+HSP90+methylation+promotes+cancer+cell+proliferation+by+regulating+the+chaperone+complex+formation&doi=10.1016%2Fj.canlet.2014.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">SMYD2-dependent HSP90 methylation promotes cancer cell proliferation by regulating the chaperone complex formation</span></div><div class="casAuthors">Hamamoto, Ryuji; Toyokawa, Gouji; Nakakido, Makoto; Ueda, Koji; Nakamura, Yusuke</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126-133</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Heat shock protein 90 (HSP90) is a highly conserved mol. chaperone that facilitates the maturation of a wide range of proteins, and it has been recognized as a crucial facilitator of oncogene addiction and cancer cell survival.  Although HSP90 function is regulated by a variety of post-translational modifications, the physiol. significance of methylation has not fully been elucidated.  Here we demonstrate that HSP90AB1 is methylated by the histone methyltransferase SMYD2 and that it plays a crit. role in human carcinogenesis.  HSP90AB1 and SMYD2 can interact through the C-terminal region of HSP90AB1 and the SET domain of SMYD2.  Both in vitro and in vivo methyltransferase assays revealed that SMYD2 could methylate HSP90AB1 and mass spectrometry anal. indicated lysines 531 and 574 of HSP90AB1 to be methylated.  These methylation sites were shown to be important for the dimerization and chaperone complex formation of HSP90AB1.  Furthermore, methylated HSP90AB1 accelerated the proliferation of cancer cells.  Our study reveals a novel mechanism for human carcinogenesis via methylation of HSP90AB1 by SMYD2, and addnl. functional studies may assist in developing novel strategies for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhJR6mltsK1LVg90H21EOLACvtfcHk0lidSzq2kL4v_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvVejtLo%253D&md5=c2832d0fd7c14091ed8098a466f13122</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2014.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2014.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DNakakido%26aufirst%3DM.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26atitle%3DSMYD2-dependent%2520HSP90%2520methylation%2520promotes%2520cancer%2520cell%2520proliferation%2520by%2520regulating%2520the%2520chaperone%2520complex%2520formation%26jtitle%3DCancer%2520Lett.%26date%3D2014%26volume%3D351%26spage%3D126%26epage%3D133%26doi%3D10.1016%2Fj.canlet.2014.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Donlin, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andresen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Just, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappas, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, E. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zieseniss, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobenecker, M.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voelkel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chait, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregorio, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rottbauer, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linke, W. A.</span><span> </span><span class="NLM_article-title">Smyd2 controls cytoplasmic lysine methylation of Hsp90 and myofilament organization</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1101/gad.177758.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1101%2Fgad.177758.111" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=114-119&author=L.+T.+Donlinauthor=C.+Andresenauthor=S.+Justauthor=E.+Rudenskyauthor=C.+T.+Pappasauthor=M.+Krugerauthor=E.+Y.+Jacobsauthor=A.+Ungerauthor=A.+Ziesenissauthor=M.-W.+Dobeneckerauthor=T.+Voelkelauthor=B.+T.+Chaitauthor=C.+C.+Gregorioauthor=W.+Rottbauerauthor=A.+Tarakhovskyauthor=W.+A.+Linke&title=Smyd2+controls+cytoplasmic+lysine+methylation+of+Hsp90+and+myofilament+organization&doi=10.1101%2Fgad.177758.111"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1101%2Fgad.177758.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.177758.111%26sid%3Dliteratum%253Aachs%26aulast%3DDonlin%26aufirst%3DL.%2BT.%26aulast%3DAndresen%26aufirst%3DC.%26aulast%3DJust%26aufirst%3DS.%26aulast%3DRudensky%26aufirst%3DE.%26aulast%3DPappas%26aufirst%3DC.%2BT.%26aulast%3DKruger%26aufirst%3DM.%26aulast%3DJacobs%26aufirst%3DE.%2BY.%26aulast%3DUnger%26aufirst%3DA.%26aulast%3DZieseniss%26aufirst%3DA.%26aulast%3DDobenecker%26aufirst%3DM.-W.%26aulast%3DVoelkel%26aufirst%3DT.%26aulast%3DChait%26aufirst%3DB.%2BT.%26aulast%3DGregorio%26aufirst%3DC.%2BC.%26aulast%3DRottbauer%26aufirst%3DW.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DLinke%26aufirst%3DW.%2BA.%26atitle%3DSmyd2%2520controls%2520cytoplasmic%2520lysine%2520methylation%2520of%2520Hsp90%2520and%2520myofilament%2520organization%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D114%26epage%3D119%26doi%3D10.1101%2Fgad.177758.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Voelkel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andresen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Just, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rottbauer, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linke, W. A.</span><span> </span><span class="NLM_article-title">Lysine methyltransferase Smyd2 regulates Hsp90-mediated protection of the sarcomeric titin springs and cardiac function</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1833</span><span class="NLM_x">, </span> <span class="NLM_fpage">812</span><span class="NLM_x">–</span> <span class="NLM_lpage">822</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2012.09.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.bbamcr.2012.09.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=23047121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ShtrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1833&publication_year=2013&pages=812-822&author=T.+Voelkelauthor=C.+Andresenauthor=A.+Ungerauthor=S.+Justauthor=W.+Rottbauerauthor=W.+A.+Linke&title=Lysine+methyltransferase+Smyd2+regulates+Hsp90-mediated+protection+of+the+sarcomeric+titin+springs+and+cardiac+function&doi=10.1016%2Fj.bbamcr.2012.09.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methyltransferase Smyd2 regulates Hsp90-mediated protection of the sarcomeric titin springs and cardiac function</span></div><div class="casAuthors">Voelkel, Tobias; Andresen, Christian; Unger, Andreas; Just, Steffen; Rottbauer, Wolfgang; Linke, Wolfgang A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1833</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">812-822</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein lysine methylation controls gene expression and repair of DNA in the nucleus but also occurs in the cytoplasm, where the role of this posttranslational modification is less understood.  Members of the Smyd protein family of lysine methyltransferases are particularly abundant in the cytoplasm, with Smyd1 and Smyd2 being most highly expressed in the heart and in skeletal muscles.  Smyd1 is a crucial myogenic regulator with histone methyltransferase activity but also assocs. with myosin, which promotes sarcomere assembly.  Smyd2 methylates histones and non-histone proteins, such as the tumor suppressors, p53 and retinoblastoma protein, RB.  Smyd2 has an intriguing function in the cytoplasm of skeletal myocytes, where it methylates the chaperone Hsp90, thus promoting the interaction of a Smyd2-methyl-Hsp90 complex with the N2A-domain of titin.  This complex protects the sarcomeric I-band region and myocyte organization.  We briefly summarize some novel functions of Smyd family members, with a focus on Smyd2, and highlight their role in striated muscles and cytoplasmic actions.  We then provide exptl. evidence that Smyd2 is also important for cardiac function.  In the cytoplasm of cardiomyocytes, Smyd2 was found to assoc. with the sarcomeric I-band region at the titin N2A-domain.  Binding to N2A occurred in vitro and in yeast via N-terminal and extreme C-terminal regions of Smyd2.  Smyd2-knockdown in zebrafish using an antisense oligonucleotide morpholino approach strongly impaired cardiac performance.  We conclude that Smyd2 and presumably several other Smyd family members are lysine methyltransferases which have, next to their nuclear activity, specific regulatory functions in the cytoplasm of heart and skeletal muscle cells.  This article is part of a Special Issue entitled: Cardiomyocyte Biol.: Cardiac Pathways of Differentiation, Metab. and Contraction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAxKXSQCV-GbVg90H21EOLACvtfcHk0lidSzq2kL4v_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ShtrvJ&md5=9da8ecc40bcfe683ebe744404ebd348b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2012.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2012.09.012%26sid%3Dliteratum%253Aachs%26aulast%3DVoelkel%26aufirst%3DT.%26aulast%3DAndresen%26aufirst%3DC.%26aulast%3DUnger%26aufirst%3DA.%26aulast%3DJust%26aufirst%3DS.%26aulast%3DRottbauer%26aufirst%3DW.%26aulast%3DLinke%26aufirst%3DW.%2BA.%26atitle%3DLysine%2520methyltransferase%2520Smyd2%2520regulates%2520Hsp90-mediated%2520protection%2520of%2520the%2520sarcomeric%2520titin%2520springs%2520and%2520cardiac%2520function%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2013%26volume%3D1833%26spage%3D812%26epage%3D822%26doi%3D10.1016%2Fj.bbamcr.2012.09.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piatnitski Chekler, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span> </span><span class="NLM_article-title">Target validation using chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+Piatnitski+Cheklerauthor=L.+H.+Jones&title=Target+validation+using+chemical+probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0liO7O8PnsKBBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DPiatnitski%2BChekler%26aufirst%3DE.%2BL.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520validation%2520using%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buser-Doepner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederiksen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimshaw, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V. M</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller-Fahrnow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Hagan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundström, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo-Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walpole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0liO7O8PnsKBBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Nguyen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antonysamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emtage, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gheyi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">LLY-507, a cell-active, potent, and selective inhibitor of protein-lysine methyltransferase SMYD2</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">13641</span><span class="NLM_x">–</span> <span class="NLM_lpage">13653</span><span class="refDoi"> DOI: 10.1074/jbc.M114.626861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1074%2Fjbc.M114.626861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25825497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFGmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=13641-13653&author=H.+Nguyenauthor=A.+Allali-Hassaniauthor=S.+Antonysamyauthor=S.+Changauthor=L.+H.+Chenauthor=C.+Curtisauthor=S.+Emtageauthor=L.+Fanauthor=T.+Gheyiauthor=F.+Liauthor=S.+Liuauthor=J.+R.+Martinauthor=D.+Mendelauthor=J.+B.+Olsenauthor=L.+Pelletierauthor=T.+Shatsevaauthor=S.+Wuauthor=F.+F.+Zhangauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=R.+M.+Campbellauthor=B.+A.+Garciaauthor=D.+Barsyte-Lovejoyauthor=M.+Maderauthor=M.+Vedadi&title=LLY-507%2C+a+cell-active%2C+potent%2C+and+selective+inhibitor+of+protein-lysine+methyltransferase+SMYD2&doi=10.1074%2Fjbc.M114.626861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-507, a Cell-active, Potent, and Selective Inhibitor of Protein-lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Nguyen, Hannah; Allali-Hassani, Abdellah; Antonysamy, Stephen; Chang, Shawn; Chen, Lisa Hong; Curtis, Carmen; Emtage, Spencer; Fan, Li; Gheyi, Tarun; Li, Fengling; Liu, Shichong; Martin, Joseph R.; Mendel, David; Olsen, Jonathan B.; Pelletier, Laura; Shatseva, Tatiana; Wu, Song; Zhang, Feiyu Fred; Arrowsmith, Cheryl H.; Brown, Peter J.; Campbell, Robert M.; Garcia, Benjamin A.; Barsyte-Lovejoy, Dalia; Mader, Mary; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">13641-13653</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">SMYD2 is a lysine methyltransferase that catalyzes the monomethylation of several protein substrates including p53.  SMYD2 is overexpressed in a significant percentage of esophageal squamous primary carcinomas, and that overexpression correlates with poor patient survival.  However, the mechanism(s) by which SMYD2 promotes oncogenesis is not understood.  A small mol. probe for SMYD2 would allow for the pharmacol. dissection of this biol.  In this report, we disclose LLY-507, a cell-active, potent small mol. inhibitor of SMYD2.  LLY-507 is >100-fold selective for SMYD2 over a broad range of methyltransferase and non-methyltransferase targets.  A 1.63-Å resoln. crystal structure of SMYD2 in complex with LLY-507 shows the inhibitor binding in the substrate peptide binding pocket.  LLY-507 is active in cells as measured by redn. of SMYD2-induced monomethylation of p53 Lys370 at submicromolar concns.  We used LLY-507 to further test other potential roles of SMYD2.  Mass spectrometry-based proteomics showed that cellular global histone methylation levels were not significantly affected by SMYD2 inhibition with LLY-507, and subcellular fractionation studies indicate that SMYD2 is primarily cytoplasmic, suggesting that SMYD2 targets a very small subset of histones at specific chromatin loci and/or non-histone substrates.  Breast and liver cancers were identified through in silico data mining as tumor types that display amplification and/or overexpression of SMYD2.  LLY-507 inhibited the proliferation of several esophageal, liver, and breast cancer cell lines in a dose-dependent manner.  These findings suggest that LLY-507 serves as a valuable chem. probe to aid in the dissection of SMYD2 function in cancer and other biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb5tVOe0M2gLVg90H21EOLACvtfcHk0liO7O8PnsKBBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFGmtrY%253D&md5=a75f59a900cfe7a6a04d35498caae78b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626861%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DCurtis%26aufirst%3DC.%26aulast%3DEmtage%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DJ.%2BR.%26aulast%3DMendel%26aufirst%3DD.%26aulast%3DOlsen%26aufirst%3DJ.%2BB.%26aulast%3DPelletier%26aufirst%3DL.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DF.%2BF.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DLLY-507%252C%2520a%2520cell-active%252C%2520potent%252C%2520and%2520selective%2520inhibitor%2520of%2520protein-lysine%2520methyltransferase%2520SMYD2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D13641%26epage%3D13653%26doi%3D10.1074%2Fjbc.M114.626861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Sweis, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Algire, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakob, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maag, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soni, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappano, W. N.</span><span> </span><span class="NLM_article-title">Discovery of A-893, a new cell-active benzoxazinone inhibitor of lysine methyltransferase SMYD2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">695</span><span class="NLM_x">–</span> <span class="NLM_lpage">700</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00124</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00124" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsVSqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=695-700&author=R.+F.+Sweisauthor=Z.+Wangauthor=M.+Algireauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=G.+G.+Chiangauthor=J.+Guoauthor=C.+G.+Jakobauthor=S.+Kennedyauthor=F.+Liauthor=D.+Maagauthor=B.+Shawauthor=N.+B.+Soniauthor=M.+Vedadiauthor=W.+N.+Pappano&title=Discovery+of+A-893%2C+a+new+cell-active+benzoxazinone+inhibitor+of+lysine+methyltransferase+SMYD2&doi=10.1021%2Facsmedchemlett.5b00124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of A-893, A New Cell-Active Benzoxazinone Inhibitor of Lysine Methyltransferase SMYD2</span></div><div class="casAuthors">Sweis, Ramzi F.; Wang, Zhi; Algire, Mikkel; Arrowsmith, Cheryl H.; Brown, Peter J.; Chiang, Gary G.; Guo, Jun; Jakob, Clarissa G.; Kennedy, Steven; Li, Fengling; Maag, David; Shaw, Bailin; Soni, Nirupama B.; Vedadi, Masoud; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">695-700</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A lack of useful small mol. tools has precluded thorough interrogation of the biol. function of SMYD2, a lysine methyltransferase with known tumor-suppressor substrates.  Systematic exploration of the structure-activity relationships of a previously known benzoxazinone compd. led to the synthesis of A-893 I, a potent and selective SMYD2 inhibitor (IC50: 2.8 nM).  A cocrystal structure reveals the origin of enhanced potency, and effective suppression of p53K370 methylation is obsd. in a lung carcinoma (A549) cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI0i3TmEl2DLVg90H21EOLACvtfcHk0liYP2OjIejZXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsVSqtb8%253D&md5=4e37d0d524f1b1066a4ef7ba03dc4695</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00124%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DAlgire%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520of%2520A-893%252C%2520a%2520new%2520cell-active%2520benzoxazinone%2520inhibitor%2520of%2520lysine%2520methyltransferase%2520SMYD2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D695%26epage%3D700%26doi%3D10.1021%2Facsmedchemlett.5b00124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span>BAY-598 A selective chemical probe for SMYD2; <span class="NLM_publisher-name">Structural Genomics Consortium</span>: <span class="NLM_publisher-loc">Toronto</span>; <a href="http://www.thesgc.org/chemical-probes/BAY-598" class="extLink">http://www.thesgc.org/chemical-probes/BAY-598</a> (accessed Mar 9, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BAY-598+A+selective+chemical+probe+for+SMYD2%3B+Structural+Genomics+Consortium%3A+Toronto%3B+http%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FBAY-598+%28accessed+Mar+9%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DBAY-598%2520A%2520selective%2520chemical%2520probe%2520for%2520SMYD2%26pub%3DStructural%2520Genomics%2520Consortium" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Reynoird, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazur, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stellfeld, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flores, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lofgren, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brambilla, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hainaut, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaznowska, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khatri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stresemann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, J.</span><span> </span><span class="NLM_article-title">Coordination of stress signals by the lysine methyltransferase SMYD2 promotes pancreatic cancer</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">772</span><span class="NLM_x">–</span> <span class="NLM_lpage">785</span><span class="refDoi"> DOI: 10.1101/gad.275529.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1101%2Fgad.275529.115" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=772-785&author=N.+Reynoirdauthor=P.+K.+Mazurauthor=T.+Stellfeldauthor=N.+M.+Floresauthor=S.+M.+Lofgrenauthor=S.+M.+Carlsonauthor=E.+Brambillaauthor=P.+Hainautauthor=E.+B.+Kaznowskaauthor=C.+H.+Arrowsmithauthor=P.+Khatriauthor=C.+Stresemannauthor=O.+Gozaniauthor=J.+Sage&title=Coordination+of+stress+signals+by+the+lysine+methyltransferase+SMYD2+promotes+pancreatic+cancer&doi=10.1101%2Fgad.275529.115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1101%2Fgad.275529.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.275529.115%26sid%3Dliteratum%253Aachs%26aulast%3DReynoird%26aufirst%3DN.%26aulast%3DMazur%26aufirst%3DP.%2BK.%26aulast%3DStellfeld%26aufirst%3DT.%26aulast%3DFlores%26aufirst%3DN.%2BM.%26aulast%3DLofgren%26aufirst%3DS.%2BM.%26aulast%3DCarlson%26aufirst%3DS.%2BM.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DHainaut%26aufirst%3DP.%26aulast%3DKaznowska%26aufirst%3DE.%2BB.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKhatri%26aufirst%3DP.%26aulast%3DStresemann%26aufirst%3DC.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DSage%26aufirst%3DJ.%26atitle%3DCoordination%2520of%2520stress%2520signals%2520by%2520the%2520lysine%2520methyltransferase%2520SMYD2%2520promotes%2520pancreatic%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2016%26volume%3D30%26spage%3D772%26epage%3D785%26doi%3D10.1101%2Fgad.275529.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Hieda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukita, S.</span><span> </span><span class="NLM_article-title">A new high molecular mass protein showing unique localization in desmosomal plaque</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">1511</span><span class="NLM_x">–</span> <span class="NLM_lpage">1518</span><span class="refDoi"> DOI: 10.1083/jcb.109.4.1511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1083%2Fjcb.109.4.1511" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=1989&pages=1511-1518&author=Y.+Hiedaauthor=S.+Tsukitaauthor=S.+Tsukita&title=A+new+high+molecular+mass+protein+showing+unique+localization+in+desmosomal+plaque&doi=10.1083%2Fjcb.109.4.1511"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1083%2Fjcb.109.4.1511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.109.4.1511%26sid%3Dliteratum%253Aachs%26aulast%3DHieda%26aufirst%3DY.%26aulast%3DTsukita%26aufirst%3DS.%26aulast%3DTsukita%26aufirst%3DS.%26atitle%3DA%2520new%2520high%2520molecular%2520mass%2520protein%2520showing%2520unique%2520localization%2520in%2520desmosomal%2520plaque%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D1989%26volume%3D109%26spage%3D1511%26epage%3D1518%26doi%3D10.1083%2Fjcb.109.4.1511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Tarcsay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keserű, G. M.</span><span> </span><span class="NLM_article-title">Is there a link between selectivity and binding thermodynamics profiles?</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.drudis.2014.09.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25263698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=86-94&author=A.+Tarcsayauthor=G.+M.+Keser%C5%B1&title=Is+there+a+link+between+selectivity+and+binding+thermodynamics+profiles%3F&doi=10.1016%2Fj.drudis.2014.09.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Is there a link between selectivity and binding thermodynamics profiles?</span></div><div class="casAuthors">Tarcsay, Akos; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-94</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Thermodn. of ligand binding is influenced by the interplay between enthalpy and entropy contributions of the binding event.  The impact of these binding free energy components, however, is not limited to the primary target only.  Here, we investigate the relationship between binding thermodn. and selectivity profiles by combining publicly available data from broad off-target assay profiling and the corresponding thermodn. measurements.  Our anal. indicates that compds. binding their primary targets with higher entropy contributions tend to hit more off-targets compared with those ligands that demonstrated enthalpy-driven binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ3yzea7OANrVg90H21EOLACvtfcHk0liYP2OjIejZXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Kgt7rO&md5=4b5fe0131781069dc3a4883e8b66ec4e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.09.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3DA.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DIs%2520there%2520a%2520link%2520between%2520selectivity%2520and%2520binding%2520thermodynamics%2520profiles%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D86%26epage%3D94%26doi%3D10.1016%2Fj.drudis.2014.09.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Ladbury, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freire, E.</span><span> </span><span class="NLM_article-title">Adding calorimetric data to decision making in lead discovery: a hot tip</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1038/nrd3054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnrd3054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19960014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWjt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=23-27&author=J.+E.+Ladburyauthor=G.+Klebeauthor=E.+Freire&title=Adding+calorimetric+data+to+decision+making+in+lead+discovery%3A+a+hot+tip&doi=10.1038%2Fnrd3054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Adding calorimetric data to decision making in lead discovery: A hot tip</span></div><div class="casAuthors">Ladbury, John E.; Klebe, Gerhard; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-27</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recognition of the limitations of high-throughput screening approaches in the discovery of candidate drugs has reawakened interest in structure-based and other rational design methods.  Here, the authors describe how isothermal titrn. calorimetry can be used to obtain thermodn. data on the binding of compds. to protein targets.  The authors propose that these data - particularly the change in enthalpy - could provide a valuable, complementary addn. to established tools for selecting compds. in lead discovery and for aiding lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph3qvp2E8fbLVg90H21EOLACvtfcHk0lhaf8w_wJ2Qgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWjt7bM&md5=e8446b6db983c35580e9010899eb40eb</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrd3054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3054%26sid%3Dliteratum%253Aachs%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DAdding%2520calorimetric%2520data%2520to%2520decision%2520making%2520in%2520lead%2520discovery%253A%2520a%2520hot%2520tip%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D23%26epage%3D27%26doi%3D10.1038%2Fnrd3054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Macfarlane, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seatter, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plevin, R.</span><span> </span><span class="NLM_article-title">Proteinase-activated receptors</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">245</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2001&pages=245-282&author=S.+R.+Macfarlaneauthor=M.+J.+Seatterauthor=T.+Kankeauthor=G.+D.+Hunterauthor=R.+Plevin&title=Proteinase-activated+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMacfarlane%26aufirst%3DS.%2BR.%26aulast%3DSeatter%26aufirst%3DM.%2BJ.%26aulast%3DKanke%26aufirst%3DT.%26aulast%3DHunter%26aufirst%3DG.%2BD.%26aulast%3DPlevin%26aufirst%3DR.%26atitle%3DProteinase-activated%2520receptors%26jtitle%3DPharmacol.%2520Rev.%26date%3D2001%26volume%3D53%26spage%3D245%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Leger, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covic, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuliopulos, A.</span><span> </span><span class="NLM_article-title">Protease-activated receptors in cardiovascular diseases</span> <span class="citation_source-journal">Circulation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x">, </span> <span class="NLM_fpage">1070</span><span class="NLM_x">–</span> <span class="NLM_lpage">1077</span><span class="refDoi"> DOI: 10.1161/CIRCULATIONAHA.105.574830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1161%2FCIRCULATIONAHA.105.574830" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2006&pages=1070-1077&author=A.+J.+Legerauthor=L.+Covicauthor=A.+Kuliopulos&title=Protease-activated+receptors+in+cardiovascular+diseases&doi=10.1161%2FCIRCULATIONAHA.105.574830"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1161%2FCIRCULATIONAHA.105.574830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCULATIONAHA.105.574830%26sid%3Dliteratum%253Aachs%26aulast%3DLeger%26aufirst%3DA.%2BJ.%26aulast%3DCovic%26aufirst%3DL.%26aulast%3DKuliopulos%26aufirst%3DA.%26atitle%3DProtease-activated%2520receptors%2520in%2520cardiovascular%2520diseases%26jtitle%3DCirculation%26date%3D2006%26volume%3D114%26spage%3D1070%26epage%3D1077%26doi%3D10.1161%2FCIRCULATIONAHA.105.574830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Allerheiligen, S.; Brohm, D.; Diedrichs, N.; Hübsch, W.; Fröhlen, B.-N.; Gerdes, C.; Gnoth, M. J.; Perzborn, E.; Vöhringer, V.</span> (Bayer HeathCare AG) <span> </span><span class="NLM_article-title">Cyanoguanidine-substituted pyrazolines</span>. WO2006072350,<span class="NLM_x"> </span><span class="NLM_year">2006</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=S.+Allerheiligen&author=D.+Brohm&author=N.+Diedrichs&author=W.+H%C3%BCbsch&author=B.-N.+Fr%C3%B6hlen&author=C.+Gerdes&author=M.+J.+Gnoth&author=E.+Perzborn&author=V.+V%C3%B6hringer&title=Cyanoguanidine-substituted+pyrazolines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAllerheiligen%26aufirst%3DS.%26atitle%3DCyanoguanidine-substituted%2520pyrazolines%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Abad-Zapatero, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metz, J. T.</span><span> </span><span class="NLM_article-title">Ligand efficiency indices as guideposts for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">464</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span><span class="refDoi"> DOI: 10.1016/S1359-6446(05)03386-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2FS1359-6446%2805%2903386-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=15809192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A280%3ADC%252BD2M7nsF2iug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=464-469&author=C.+Abad-Zapateroauthor=J.+T.+Metz&title=Ligand+efficiency+indices+as+guideposts+for+drug+discovery&doi=10.1016%2FS1359-6446%2805%2903386-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency indices as guideposts for drug discovery</span></div><div class="casAuthors">Abad-Zapatero Cele; Metz James T</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">464-9</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSHqVfZXUq224qdXsLcGTEUfW6udTcc2eZAKSYZcnEwPrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M7nsF2iug%253D%253D&md5=1720f263a306122c86a57463e5a39287</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903386-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903386-6%26sid%3Dliteratum%253Aachs%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26atitle%3DLigand%2520efficiency%2520indices%2520as%2520guideposts%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D464%26epage%3D469%26doi%3D10.1016%2FS1359-6446%2805%2903386-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span>Schrödinger Suite 2012 Update 2, WaterMap, version 1.4.518; <span class="NLM_publisher-name">Schrödinger LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schr%C3%B6dinger+Suite+2012+Update+2%2C+WaterMap%2C+version+1.4.518%3B+Schr%C3%B6dinger+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%2520Suite%25202012%2520Update%25202%252C%2520WaterMap%252C%2520version%25201.4.518%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Young, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berne, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">808</span><span class="NLM_x">–</span> <span class="NLM_lpage">813</span><span class="refDoi"> DOI: 10.1073/pnas.0610202104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1073%2Fpnas.0610202104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=17204562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=808-813&author=T.+Youngauthor=R.+Abelauthor=B.+Kimauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Motifs+for+molecular+recognition+exploiting+hydrophobic+enclosure+in+protein-ligand+binding&doi=10.1073%2Fpnas.0610202104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span></div><div class="casAuthors">Young, Tom; Abel, Robert; Kim, Byungchan; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The thermodn. properties and phase behavior of water in confined regions can vary significantly from that obsd. in the bulk.  This is particularly true for systems in which the confinement is on the mol.-length scale.  In this study, we use mol. dynamics simulations and a powerful solvent anal. technique based on inhomogeneous solvation theory to investigate the properties of water mols. that solvate the confined regions of protein active sites.  Our simulations and anal. indicate that the solvation of protein active sites that are characterized by hydrophobic enclosure and correlated hydrogen bonds induce atypical entropic and enthalpic penalties of hydration.  These penalties apparently stabilize the protein-ligand complex with respect to the independently solvated ligand and protein, which leads to enhanced binding affinities.  Our anal. elucidates several challenging cases, including the super affinity of the streptavidin-biotin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWumQ61YTqILVg90H21EOLACvtfcHk0lglxlHNGUN0Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D&md5=1109e3f48670cd910885f7fd527eb880</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0610202104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0610202104%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMotifs%2520for%2520molecular%2520recognition%2520exploiting%2520hydrophobic%2520enclosure%2520in%2520protein-ligand%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D808%26epage%3D813%26doi%3D10.1073%2Fpnas.0610202104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Cheng, Y.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (K1) and the concentration of inhibitor which causes 50% inhibition (I50) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.-C.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28K1%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lglxlHNGUN0Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.-C.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528K1%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span> </span><span class="NLM_article-title">Mechanistic considerations in high-throughput screening</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.1016/S0003-2697(03)00346-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2FS0003-2697%2803%2900346-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2003&pages=1-12&author=R.+A.+Copeland&title=Mechanistic+considerations+in+high-throughput+screening&doi=10.1016%2FS0003-2697%2803%2900346-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2FS0003-2697%2803%2900346-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0003-2697%252803%252900346-4%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DMechanistic%2520considerations%2520in%2520high-throughput%2520screening%26jtitle%3DAnal.%2520Biochem.%26date%3D2003%26volume%3D320%26spage%3D1%26epage%3D12%26doi%3D10.1016%2FS0003-2697%2803%2900346-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Chan-Penebre, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuplast, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraj, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stickland, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribich, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbash, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nurse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oza, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallagher, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lglxlHNGUN0Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lijx2CnF7RSDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Olsen, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, H.</span><span> </span><span class="NLM_article-title">Quantitative profiling of the activity of protein lysine methyltransferase SMYD2 using SILAC-based proteomics</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">892</span><span class="NLM_x">–</span> <span class="NLM_lpage">905</span><span class="refDoi"> DOI: 10.1074/mcp.M115.053280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1074%2Fmcp.M115.053280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=26750096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVGrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=892-905&author=J.+B.+Olsenauthor=X.+J.+Caoauthor=B.+Hanauthor=L.+H.+Chenauthor=A.+Horvathauthor=T.+I.+Richardsonauthor=R.+M.+Campbellauthor=B.+A.+Garciaauthor=H.+Nguyen&title=Quantitative+profiling+of+the+activity+of+protein+lysine+methyltransferase+SMYD2+using+SILAC-based+proteomics&doi=10.1074%2Fmcp.M115.053280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative profiling of the activity of protein lysine methyltransferase SMYD2 using SILAC-based proteomics</span></div><div class="casAuthors">Olsen, Jonathan B.; Cao, Xing-Jun; Han, Bomie; Chen, Lisa Hong; Horvath, Alexander; Richardson, Timothy I.; Campbell, Robert M.; Garcia, Benjamin A.; Nguyen, Hannah</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-905</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The significance of non-histone lysine methylation in cell biol. and human disease is an emerging area of research exploration.  The development of small mol. inhibitors that selectively and potently target enzymes that catalyze the addn. of methyl-groups to lysine residues, such as the protein lysine mono-methyltransferase SMYD2, is an active area of drug discovery.  Crit. to the accurate assessment of biol. function is the ability to identify target enzyme substrates and to define enzyme substrate specificity within the context of the cell.  Here, using stable isotopic labeling with amino acids in cell culture (SILAC) coupled with immunoaffinity enrichment of mono-methyl-lysine (Kme1) peptides and mass spectrometry, we report a comprehensive, large-scale proteomic study of lysine mono-methylation, comprising a total of 1032 Kme1 sites in esophageal squamous cell carcinoma (ESCC) cells and 1861 Kme1 sites in ESCC cells overexpressing SMYD2.  Among these Kme1 sites is a subset of 35 found to be potently down-regulated by both shRNA-mediated knockdown of SMYD2 and LLY-507, a selective small mol. inhibitor of SMYD2.  In addn., we report specific protein sequence motifs enriched in Kme1 sites that are directly regulated by endogenous SMYD2 activity, revealing that SMYD2 substrate specificity is more diverse than expected.  We further show direct activity of SMYD2 toward BTF3-K2, PDAP1-K126 as well as numerous sites within the repetitive units of two unique and exceptionally large proteins, AHNAK and AHNAK2.  Collectively, our findings provide quant. insights into the cellular activity and substrate recognition of SMYD2 as well as the global landscape and regulation of protein mono-methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIbAGzEvpHn7Vg90H21EOLACvtfcHk0lijx2CnF7RSDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVGrurs%253D&md5=ebc0c9ca87e73f98b82cb1e3c10d1c71</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M115.053280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M115.053280%26sid%3Dliteratum%253Aachs%26aulast%3DOlsen%26aufirst%3DJ.%2BB.%26aulast%3DCao%26aufirst%3DX.%2BJ.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DHorvath%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DT.%2BI.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DNguyen%26aufirst%3DH.%26atitle%3DQuantitative%2520profiling%2520of%2520the%2520activity%2520of%2520protein%2520lysine%2520methyltransferase%2520SMYD2%2520using%2520SILAC-based%2520proteomics%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2016%26volume%3D15%26spage%3D892%26epage%3D905%26doi%3D10.1074%2Fmcp.M115.053280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Z.</span><span> </span><span class="NLM_article-title">Defining balanced conditions for inhibitor screening assays that target bisubstrate enzymes</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span><span class="refDoi"> DOI: 10.1177/1087057108328763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1177%2F1087057108328763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19196704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFWksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=111-120&author=J.+Yangauthor=R.+A.+Copelandauthor=Z.+Lai&title=Defining+balanced+conditions+for+inhibitor+screening+assays+that+target+bisubstrate+enzymes&doi=10.1177%2F1087057108328763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Defining balanced conditions for inhibitor screening assays that target bisubstrate enzymes</span></div><div class="casAuthors">Yang, Jingsong; Copeland, Robert A.; Lai, Zhihong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-120</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is a common mechanism for identifying lead compds. for drug discovery efforts.  Small mols. can inhibit enzymes by a variety of mechanisms, such as competitive, noncompetitive, and uncompetitive with respect to the substrate(s) of the catalytic reaction.  To optimize the chances of finding the broadest diversity of inhibitor modalities during screening, one must run assays under "balanced" conditions where the potency of inhibitors with various modes of action falls within a similar range.  When an enzyme reaction involves more than one substrate, the definition and assessment of the apparent potency of inhibitors (IC50), in relation to their true potency (Ki), can be nontrivial.  This article provides a theor. anal., on the basis of the Cheng-Prusoff derivation, of the IC50/Ki relationship of bisubstrate enzyme reactions following various sequential kinetic mechanisms, as well as the application and limitations of this information for defining optimal screening conditions for such enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM9O6ZD0r6l7Vg90H21EOLACvtfcHk0lijx2CnF7RSDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFWksr4%253D&md5=076f5a6ad75c4fc0e01d59ca5ea53fe2</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1177%2F1087057108328763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108328763%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DLai%26aufirst%3DZ.%26atitle%3DDefining%2520balanced%2520conditions%2520for%2520inhibitor%2520screening%2520assays%2520that%2520target%2520bisubstrate%2520enzymes%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D111%26epage%3D120%26doi%3D10.1177%2F1087057108328763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Eram, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lima-Fernandes, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gen. Subj.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1850</span><span class="NLM_x">, </span> <span class="NLM_fpage">1842</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span><span class="refDoi"> DOI: 10.1016/j.bbagen.2015.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1016%2Fj.bbagen.2015.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=26002201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=1842-1848&author=M.+S.+Eramauthor=E.+Kuznetsovaauthor=F.+Liauthor=E.+Lima-Fernandesauthor=S.+Kennedyauthor=I.+Chauauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadi&title=Kinetic+characterization+of+human+histone+H3+lysine+36+methyltransferases%2C+ASH1L+and+SETD2&doi=10.1016%2Fj.bbagen.2015.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic characterization of human histone H3 lysine 36 methyltransferases, ASH1L and SETD2</span></div><div class="casAuthors">Eram, Mohammad S.; Kuznetsova, Ekaterina; Li, Fengling; Lima-Fernandes, Evelyne; Kennedy, Steven; Chau, Irene; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1842-1848</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Dysregulation of methylation of lysine 36 on histone H3 (H3K36) have been implicated in a variety of diseases including cancers.  ASH1L and SETD2 are two enzymes among others that catalyze H3K36 methylation.  H3K4 methylation has also been reported for ASH1L.  Radioactivity-based enzyme assays, Western and immunoblotting using specific antibodies and mol. modeling were used to characterize substrate specificity of ASH1L and SETD2.  Here we report on the assay development and kinetic characterization of ASH1L and SETD2 and their substrate specificities in vitro.  Both enzymes were active with recombinant nucleosome as substrate.  However, SETD2 but not ASH1L methylated histone peptides as well, indicating that the interaction of the basic post-SET extension with substrate may not be crit. for SETD2 activity.  Neither enzyme was active with nucleosome contg. a H3K36A mutation, indicating their specificity for H3K36.  Analyzing the methylation state of the products of ASH1L and SETD2 reactions also confirmed that both enzymes mono- and dimethylate H3K36 and are inactive with H3K4 as substrate, and that only SETD2 is able to trimethylate H3K36 in vitro.  We detd. the kinetic parameters for ASH1L and SETD2 activity, enabling screening for inhibitors that can be used to further investigate the roles of these two proteins in health and disease.  Both ASH1L and SETD2 are H3K36-specific methyltransferases but only SETD2 can trimethylate this mark.  The basic post-SET extension is crit. for ASH1L but not SETD2 activity.  We provide full kinetic characterization of ASH1L and SETD2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproBY7GkmOLrVg90H21EOLACvtfcHk0li3jOEj2tW9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVSmsL8%253D&md5=64839475d5ee9cc2f20e6e71f8e8a6ea</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2015.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2015.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DKinetic%2520characterization%2520of%2520human%2520histone%2520H3%2520lysine%252036%2520methyltransferases%252C%2520ASH1L%2520and%2520SETD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D1842%26epage%3D1848%26doi%3D10.1016%2Fj.bbagen.2015.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0li3jOEj2tW9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0li3jOEj2tW9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lil1U1JMffAOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lil1U1JMffAOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Schuttelkopf, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Aalten, D. M.</span><span> </span><span class="NLM_article-title">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1355</span><span class="NLM_x">–</span> <span class="NLM_lpage">1363</span><span class="refDoi"> DOI: 10.1107/S0907444904011679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444904011679" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=15272157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A280%3ADC%252BD2czmslGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1355-1363&author=A.+W.+Schuttelkopfauthor=D.+M.+van+Aalten&title=PRODRG%3A+a+tool+for+high-throughput+crystallography+of+protein-ligand+complexes&doi=10.1107%2FS0907444904011679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">PRODRG: a tool for high-throughput crystallography of protein-ligand complexes</span></div><div class="casAuthors">Schuttelkopf Alexander W; van Aalten Daan M F</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">Pt 8</span>),
    <span class="NLM_cas:pages">1355-63</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">The small-molecule topology generator PRODRG is described, which takes input from existing coordinates or various two-dimensional formats and automatically generates coordinates and molecular topologies suitable for X-ray refinement of protein-ligand complexes.  Test results are described for automatic generation of topologies followed by energy minimization for a subset of compounds from the Cambridge Structural Database, which shows that, within the limits of the empirical GROMOS87 force field used, structures with good geometries are generated.  X-ray refinement in X-PLOR/CNS, REFMAC and SHELX using PRODRG-generated topologies produces results comparable to refinement with topologies from the standard libraries.  However, tests with distorted starting coordinates show that PRODRG topologies perform better, both in terms of ligand geometry and of crystallographic R factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdNRIcoht5aXy-c-E1v-fmfW6udTcc2eZ5UUyWcfJuULntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czmslGmtA%253D%253D&md5=1adce6cd2cacdace513a69efa33c4e93</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444904011679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904011679%26sid%3Dliteratum%253Aachs%26aulast%3DSchuttelkopf%26aufirst%3DA.%2BW.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%26atitle%3DPRODRG%253A%2520a%2520tool%2520for%2520high-throughput%2520crystallography%2520of%2520protein-ligand%2520complexes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D1355%26epage%3D1363%26doi%3D10.1107%2FS0907444904011679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljs8kk7rA_7sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span>Epik, <span class="NLM_edition">version 2.3.518</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Epik%2C+version+2.3.518%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DEpik%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span>Impact, <span class="NLM_edition">version 5.8.518</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Impact%2C+version+5.8.518%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DImpact%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span>Prime, <span class="NLM_edition">version 3.1.518</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Prime%2C+version+3.1.518%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPrime%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span>Schrödinger Suite 2012 Update 2, Protein Preparation Wizard; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schr%C3%B6dinger+Suite+2012+Update+2%2C+Protein+Preparation+Wizard%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%2520Suite%25202012%2520Update%25202%252C%2520Protein%2520Preparation%2520Wizard%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span> </span><span class="NLM_article-title">The OPLS [optimized potentials for liquid simulations] potential functions for proteins, energy minimizations for crystals of cyclic peptides and crambin</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">1657</span><span class="NLM_x">–</span> <span class="NLM_lpage">1666</span><span class="refDoi"> DOI: 10.1021/ja00214a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00214a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=1657-1666&author=W.+L.+Jorgensenauthor=J.+Tirado-Rives&title=The+OPLS+%5Boptimized+potentials+for+liquid+simulations%5D+potential+functions+for+proteins%2C+energy+minimizations+for+crystals+of+cyclic+peptides+and+crambin&doi=10.1021%2Fja00214a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">The OPLS [optimized potentials for liquid simulations] potential functions for proteins, energy minimizations for crystals of cyclic peptides and crambin</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1657-66</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A complete set of intermol. potential functions was developed for use in computer simulations of proteins in their native environment.  Parameters are reported for 25 peptide residues as well as the common neutral and charged terminal groups.  The potential functions have the simple Coulomb plus Lennard-Jones form and are compatible with the widely used models for water, TIP4P, TIP3P, and SPC.  The parameters were obtained and tested primarily in conjunction with Monte Carlo statistical mechanics simulations of 36 pure org. liqs. and numerous aq. solns. of org. ions representative of subunits in the side chains and backbones of proteins.  Bond stretch, angle bend, and torsional terms were adopted from the AMBER united-atom force field.  As reported here, further testing involved studies of conformational energy surfaces and optimizations of the crystal structures for 4 cyclic hexapeptides and a cyclic pentapeptide.  The av. root mean square deviation from the x-ray structures of the crystals is only 0.17 Å for the at. positions and 3% for the unit cell vols.  A more crit. test was then provided by performing energy minimizations for the complete crystal of the protein crambin, including 182 water mols. that were initially placed via a Monte Carlo simulation.  The resultant root mean square deviation for the non-H atoms is still only 0.17 Å and the variation in the errors for charged, polar, and nonpolar residues is small.  Significant improvement is apparent over the AMBER united-atom force field which was previously demonstrated to be superior to many alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ6an38QWApbVg90H21EOLACvtfcHk0ljSXdshKADJPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D&md5=b6c901d8c295b3b37329a7faef527e12</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja00214a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00214a001%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DThe%2520OPLS%2520%255Boptimized%2520potentials%2520for%2520liquid%2520simulations%255D%2520potential%2520functions%2520for%2520proteins%252C%2520energy%2520minimizations%2520for%2520crystals%2520of%2520cyclic%2520peptides%2520and%2520crambin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D1657%26epage%3D1666%26doi%3D10.1021%2Fja00214a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maxwell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span> </span><span class="NLM_article-title">Development and testing of the OPLS all-atom force field on conformational energetics and properties of organic liquids</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">11225</span><span class="NLM_x">–</span> <span class="NLM_lpage">11236</span><span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+L.+Jorgensenauthor=D.+S.+Maxwellauthor=J.+Tirado-Rives&title=Development+and+testing+of+the+OPLS+all-atom+force+field+on+conformational+energetics+and+properties+of+organic+liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0ljSXdshKADJPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DMaxwell%26aufirst%3DD.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520the%2520OPLS%2520all-atom%2520force%2520field%2520on%2520conformational%2520energetics%2520and%2520properties%2520of%2520organic%2520liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandrasekhar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madura, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Impey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, M. L.</span><span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">926</span><span class="NLM_x">–</span> <span class="NLM_lpage">935</span><span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0ljSXdshKADJPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Lazaridis, T.</span><span> </span><span class="NLM_article-title">Inhomogeneous fluid approach to solvation thermodynamics. 1. Theory</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3531</span><span class="NLM_x">–</span> <span class="NLM_lpage">3541</span><span class="refDoi"> DOI: 10.1021/jp9723574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp9723574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3531-3541&author=T.+Lazaridis&title=Inhomogeneous+fluid+approach+to+solvation+thermodynamics.+1.+Theory&doi=10.1021%2Fjp9723574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Inhomogeneous Fluid Approach to Solvation Thermodynamics. 1. Theory</span></div><div class="casAuthors">Lazaridis, Themis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3531-3541</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The solvation energy and entropy at infinite diln. consist of a solute-solvent term and a solvent reorganization term representing the contributions of changes in solvent structure upon solute insertion.  In the std., homogeneous treatment of solns., changes in solvent structure are expressed through derivs. of the homogeneous pair correlation function, which are very difficult to obtain by simulation.  Tractable expressions for the solvation energy and entropy are here derived by viewing the soln. as an inhomogeneous system with the solute fixed at a certain point.  The solvent reorganization terms in the inhomogeneous view contain two contributions: the local, "correlation" contributions, which are due to correlations between the solute and the solvent and dominate at high densities, and the nonlocal, "liberation" contributions, which are due to the effective diln. of the solvent caused by the thermal motion of the solute and dominate at low densities.  The liberation contributions are independent of the nature or size of the solute and depend only on the properties of the solvent.  For common liq. solvents the nonlocal terms are negligible and the solvation properties arise almost entirely from effects localized around the solute.  The new expressions are tested by calcns. of the solvent reorganization energy and entropy in ideal hard-sphere and Lennard-Jones mixts. (solute identical to solvent).  The solvent reorganization energy and entropy make distinct and significant contributions to the solvation free energy.  The theory can be applied to truly inhomogeneous systems as well as small solute solvation, thus providing a connection between interfacial phenomena and microscopic solvation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxEN_nWI8WfbVg90H21EOLACvtfcHk0liBVp2GCoNAkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOltr8%253D&md5=99755c87ffc33abd5e7df89c9996a665</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjp9723574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp9723574%26sid%3Dliteratum%253Aachs%26aulast%3DLazaridis%26aufirst%3DT.%26atitle%3DInhomogeneous%2520fluid%2520approach%2520to%2520solvation%2520thermodynamics.%25201.%2520Theory%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3531%26epage%3D3541%26doi%3D10.1021%2Fjp9723574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARF','PDB','5ARF'); return false;">PDB: 5ARF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARG','PDB','5ARG'); return false;">PDB: 5ARG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TG5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3TG5','PDB','3TG5'); return false;">PDB: 3TG5</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i70"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01890">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_55616"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01890">10.1021/acs.jmedchem.5b01890</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figure and tables of PK and exposure data, selectivity data, and crystallographic data collection and refinement statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_001.pdf">jm5b01890_si_001.pdf (1.28 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01890/suppl_file/jm5b01890_si_002.csv">jm5b01890_si_002.csv (2.46 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factors for the described crystal structures have been deposited with the Protein Data Bank (PDB). The PDB accession codes are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARF">5ARF</a> (compound <b>6</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ARG">5ARG</a> (<i>S</i>)-<b>4</b>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01890&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01890%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-10%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01890" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991ff90e723c76","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
