
<html lang="en"     class="pb-page"  data-request-id="197f8648-d1c6-4aef-8a05-2af3d6250139"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c02177;requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-10;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Re-emerging Aspartic Protease Targets: Examining Cryptococcus neoformans Major Aspartyl Peptidase 1 as a Target for Antifungal Drug Discovery" /></meta><meta name="dc.Creator" content="Robin  Kryštůfek" /></meta><meta name="dc.Creator" content="Pavel  Šácha" /></meta><meta name="dc.Creator" content="Jana  Starková" /></meta><meta name="dc.Creator" content="Jiří  Brynda" /></meta><meta name="dc.Creator" content="Martin  Hradilek" /></meta><meta name="dc.Creator" content="Eva  Tloušt’ová" /></meta><meta name="dc.Creator" content="Justyna  Grzymska" /></meta><meta name="dc.Creator" content="Wioletta  Rut" /></meta><meta name="dc.Creator" content="Michael J.  Boucher" /></meta><meta name="dc.Creator" content="Marcin  Drąg" /></meta><meta name="dc.Creator" content="Pavel  Majer" /></meta><meta name="dc.Creator" content="Miroslav  Hájek" /></meta><meta name="dc.Creator" content="Pavlína  Řezáčová" /></meta><meta name="dc.Creator" content="Hiten D.  Madhani" /></meta><meta name="dc.Creator" content="Charles S.  Craik" /></meta><meta name="dc.Creator" content="Jan  Konvalinka" /></meta><meta name="dc.Description" content="Cryptococcosis is an invasive infection that accounts for 15% of AIDS-related fatalities. Still, treating cryptococcosis remains a significant challenge due to the poor availability of effective an..." /></meta><meta name="Description" content="Cryptococcosis is an invasive infection that accounts for 15% of AIDS-related fatalities. Still, treating cryptococcosis remains a significant challenge due to the poor availability of effective an..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 18, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02177" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02177" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02177" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02177" /></link>
        
    
    

<title>Re-emerging Aspartic Protease Targets: Examining Cryptococcus neoformans Major Aspartyl Peptidase 1 as a Target for Antifungal Drug Discovery | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02177" /></meta><meta property="og:title" content="Re-emerging Aspartic Protease Targets: Examining Cryptococcus neoformans Major Aspartyl Peptidase 1 as a Target for Antifungal Drug Discovery" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0011.jpeg" /></meta><meta property="og:description" content="Cryptococcosis is an invasive infection that accounts for 15% of AIDS-related fatalities. Still, treating cryptococcosis remains a significant challenge due to the poor availability of effective antifungal therapies and emergence of drug resistance. Interestingly, protease inhibitor components of antiretroviral therapy regimens have shown some clinical benefits in these opportunistic infections. We investigated Major aspartyl peptidase 1 (May1), a secreted Cryptococcus neoformans protease, as a possible target for the development of drugs that act against both fungal and retroviral aspartyl proteases. Here, we describe the biochemical characterization of May1, present its high-resolution X-ray structure, and provide its substrate specificity analysis. Through combinatorial screening of 11,520 compounds, we identified a potent inhibitor of May1 and HIV protease. This dual-specificity inhibitor exhibits antifungal activity in yeast culture, low cytotoxicity, and low off-target activity against host proteases and could thus serve as a lead compound for further development of May1 and HIV protease inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02177"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02177">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02177&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02177&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02177&amp;href=/doi/10.1021/acs.jmedchem.0c02177" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 6706-6719</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02065" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02202" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Re-emerging Aspartic Protease Targets: Examining <i>Cryptococcus neoformans</i> Major Aspartyl Peptidase 1 as a Target for Antifungal Drug Discovery</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Robin Kryštůfek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robin Kryštůfek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div><div class="loa-info-affiliations-info">Department of Physical and Macromolecular Chemistry, Faculty of Science, Charles UniversityHlavova 8, Prague 2 12843, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robin++Kry%C5%A1t%C5%AFfek">Robin Kryštůfek</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4096-1159" title="Orcid link">https://orcid.org/0000-0003-4096-1159</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pavel Šácha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pavel Šácha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div><div class="loa-info-affiliations-info">Department of Biochemistry, Faculty of Science, Charles UniversityHlavova 8, Prague 2 12843, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pavel++%C5%A0%C3%A1cha">Pavel Šácha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jana Starková</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jana Starková</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jana++Starkov%C3%A1">Jana Starková</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiří Brynda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiří Brynda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div><div class="loa-info-affiliations-info">Institute of Molecular Genetics of the Czech Academy of Sciences, Vídeňská 1083, Prague 4 14220, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ji%C5%99%C3%AD++Brynda">Jiří Brynda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Hradilek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Hradilek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Hradilek">Martin Hradilek</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Eva Tloušt’ová</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eva Tloušt’ová</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eva++Tlou%C5%A1t%E2%80%99ov%C3%A1">Eva Tloušt’ová</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Justyna Grzymska</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Justyna Grzymska</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology, Wybrzeze Wyspianskiego 27, Wroclaw 50-370, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Justyna++Grzymska">Justyna Grzymska</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wioletta Rut</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wioletta Rut</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology, Wybrzeze Wyspianskiego 27, Wroclaw 50-370, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wioletta++Rut">Wioletta Rut</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael J. Boucher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael J. Boucher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Biophysics, University of California, San Francisco, UCSF Genentech Hall, 600 16th St Rm N374, San Francisco, California 94158, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+J.++Boucher">Michael J. Boucher</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marcin Drąg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcin Drąg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology and Bioimaging, Wroclaw University of Science and Technology, Wybrzeze Wyspianskiego 27, Wroclaw 50-370, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcin++Dr%C4%85g">Marcin Drąg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pavel Majer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pavel Majer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pavel++Majer">Pavel Majer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Miroslav Hájek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Miroslav Hájek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Miroslav++H%C3%A1jek">Miroslav Hájek</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pavlína Řezáčová</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pavlína Řezáčová</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div><div class="loa-info-affiliations-info">Institute of Molecular Genetics of the Czech Academy of Sciences, Vídeňská 1083, Prague 4 14220, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pavl%C3%ADna++%C5%98ez%C3%A1%C4%8Dov%C3%A1">Pavlína Řezáčová</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hiten D. Madhani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hiten D. Madhani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Biophysics, University of California, San Francisco, UCSF Genentech Hall, 600 16th St Rm N374, San Francisco, California 94158, United States</div><div class="loa-info-affiliations-info">Chan-Zuckerberg Biohub, 499 Illinois Street, San Francisco, California 94158, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hiten+D.++Madhani">Hiten D. Madhani</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Charles S. Craik</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charles S. Craik</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, University of California San Francisco, UCSF Genentech Hall, 600 16th St Rm S512, San Francisco, California 94158, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charles+S.++Craik">Charles S. Craik</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7704-9185" title="Orcid link">https://orcid.org/0000-0001-7704-9185</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jan Konvalinka</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jan Konvalinka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</div><div class="loa-info-affiliations-info">Department of Biochemistry, Faculty of Science, Charles UniversityHlavova 8, Prague 2 12843, Czech Republic</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3e555150485f527e4b515d565c105d5f4d105d44"><span class="__cf_email__" data-cfemail="ff949091899e93bf8a909c979dd19c9e8cd19c85">[email protected]</span></a>. Phone: +420 220 183 218.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Konvalinka">Jan Konvalinka</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0695-9266" title="Orcid link">https://orcid.org/0000-0003-0695-9266</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02177&amp;href=/doi/10.1021%2Facs.jmedchem.0c02177" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 6706–6719</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 18, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 December 2020</li><li><span class="item_label"><b>Published</b> online</span>18 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02177" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02177</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc"><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nc-1612971093307.svg" alt="NC: Only noncommercial uses of the work are permitted" /></img><span><strong>NC: </strong>Only noncommercial uses of the work are permitted</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nd-1612971093637.svg" alt="ND: No derivatives or adaptations of the work are permitted" /></img><span><strong>ND: </strong>No derivatives or adaptations of the work are permitted</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02177"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1290</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02177" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Re-emerging Aspartic Protease Targets: Examining Cryptococcus neoformans Major Aspartyl Peptidase 1 as a Target for Antifungal Drug Discovery&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Robin&quot;,&quot;last_name&quot;:&quot;Kryštůfek&quot;},{&quot;first_name&quot;:&quot;Pavel&quot;,&quot;last_name&quot;:&quot;Šácha&quot;},{&quot;first_name&quot;:&quot;Jana&quot;,&quot;last_name&quot;:&quot;Starková&quot;},{&quot;first_name&quot;:&quot;Jiří&quot;,&quot;last_name&quot;:&quot;Brynda&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Hradilek&quot;},{&quot;first_name&quot;:&quot;Eva&quot;,&quot;last_name&quot;:&quot;Tloušt’ová&quot;},{&quot;first_name&quot;:&quot;Justyna&quot;,&quot;last_name&quot;:&quot;Grzymska&quot;},{&quot;first_name&quot;:&quot;Wioletta&quot;,&quot;last_name&quot;:&quot;Rut&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;J. Boucher&quot;},{&quot;first_name&quot;:&quot;Marcin&quot;,&quot;last_name&quot;:&quot;Drąg&quot;},{&quot;first_name&quot;:&quot;Pavel&quot;,&quot;last_name&quot;:&quot;Majer&quot;},{&quot;first_name&quot;:&quot;Miroslav&quot;,&quot;last_name&quot;:&quot;Hájek&quot;},{&quot;first_name&quot;:&quot;Pavlína&quot;,&quot;last_name&quot;:&quot;Řezáčová&quot;},{&quot;first_name&quot;:&quot;Hiten&quot;,&quot;last_name&quot;:&quot;D. Madhani&quot;},{&quot;first_name&quot;:&quot;Charles&quot;,&quot;last_name&quot;:&quot;S. Craik&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Konvalinka&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;6706-6719&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02177&quot;},&quot;abstract&quot;:&quot;Cryptococcosis is an invasive infection that accounts for 15% of AIDS-related fatalities. Still, treating cryptococcosis remains a significant challenge due to the poor availability of effective antifungal therapies and emergence of drug resistance. Interestingly, protease inhibitor components of antiretroviral therapy regimens have shown some clinical benefits in these opportunistic infections. We investigated Major aspartyl peptidase 1 (May1), a secreted Cryptococcus neoformans protease, as a possible target for the development of drugs that act against both fungal and retroviral aspartyl proteases. Here, we describe the biochemical characterization of May1, present its high-resolution X-ray structure, and provide its substrate specificity analysis. Through combinatorial screening of 11,520 compounds, we identified a potent inhibitor of May1 and HIV protease. This dual-specificity inhibitor exhibits antifungal activity in yeast culture, low cytotoxicity, and low off-target activity against host proteases an&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02177&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02177" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02177&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02177" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02177&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02177" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02177&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02177&amp;href=/doi/10.1021/acs.jmedchem.0c02177" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02177" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02177" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02177%26sid%3Dliteratum%253Aachs%26pmid%3D34006103%26genre%3Darticle%26aulast%3DKry%25C5%25A1t%25C5%25AFfek%26date%3D2021%26atitle%3DRe-emerging%2BAspartic%2BProtease%2BTargets%253A%2BExamining%2BCryptococcus%2Bneoformans%2BMajor%2BAspartyl%2BPeptidase%2B1%2Bas%2Ba%2BTarget%2Bfor%2BAntifungal%2BDrug%2BDiscovery%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D10%26spage%3D6706%26epage%3D6719%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292432" title="Fungi">Fungi</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/jmcmar.2021.64.issue-10/20210527/jmcmar.2021.64.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Cryptococcosis is an invasive infection that accounts for 15% of AIDS-related fatalities. Still, treating cryptococcosis remains a significant challenge due to the poor availability of effective antifungal therapies and emergence of drug resistance. Interestingly, protease inhibitor components of antiretroviral therapy regimens have shown some clinical benefits in these opportunistic infections. We investigated Major aspartyl peptidase 1 (May1), a secreted <i>Cryptococcus neoformans</i> protease, as a possible target for the development of drugs that act against both fungal and retroviral aspartyl proteases. Here, we describe the biochemical characterization of May1, present its high-resolution X-ray structure, and provide its substrate specificity analysis. Through combinatorial screening of 11,520 compounds, we identified a potent inhibitor of May1 and HIV protease. This dual-specificity inhibitor exhibits antifungal activity in yeast culture, low cytotoxicity, and low off-target activity against host proteases and could thus serve as a lead compound for further development of May1 and HIV protease inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15347" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15347" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><i>Cryptococcus neoformans</i>, along with <i>Cryptococcus gattii</i>, is one of the two major basidiomycetes yeasts responsible for cryptococcosis,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> a life-threatening systemic mycosis that frequently occurs in immunosuppressed patients, chiefly in cases of HIV infection.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Cryptococcosis, typically environmentally acquired by inhalation of basidiospores,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> can lead to fatal cryptococcal meningitis even in immunocompetent patients.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a></div><div class="NLM_p">More than 1 million cases of cryptococcosis are reported each year worldwide, with an estimated annual death toll of 181,100,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> accounting for 15% of AIDS-related fatalities.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The World Health Organization recommends combination therapy with amphotericin B and flucytosine followed by fluconazole for management of cryptococcosis.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> However, this approach has been hampered by shortages of key regimen components (recently fluconazole<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a>), the general inaccessibility of effective therapeutics in developing areas,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and a rapid increase in incidence of multidrug-resistant pathogenic fungi.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> Cryptococcus mutants resistant to individual components of the regimen—amphotericin B,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> flucytosine,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and fluconazole<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> —have been identified, and the emergence of complete cross-resistance is a possible threat.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p">As these medications represent the three chief classes of approved antifungals currently used in clinical practice,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> advances in development of novel antifungals are of utmost importance. Focus should be placed on the large population of HIV patients with Cryptococcus coinfections to maximize the availability of treatment regimens and minimize the patients’ already high pill burden.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p">One potential solution is the development of HIV-1 protease (HIV-1 Pr) inhibitors that also target vital fungal proteases, which often serve critical roles in survival, reproduction, and host tissue invasion.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> We set out to characterize <i>C. neoformans</i> Major aspartyl peptidase 1 (May1,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> CnAP1<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a>) as a possible target for antifungal therapy,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> search for novel aspartyl protease inhibitor motifs, and examine their structure–activity relationship based on X-ray structures of May1 alone and in complex with the model inhibitor pepstatin A.</div><div class="NLM_p">May1 is one of many proteases in the <i>C. neoformans</i> secreted proteome, members of which serve diverse functions, including virulence modulation,<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> host tissue invasion,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> dissemination to the central nervous system,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and nitrogen assimilation.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> May1 is a secreted fungal pepsin-like acidic endopeptidase<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> with a broad substrate preference for hydrolysis between hydrophobic residues required for high-density growth under acidic conditions. Experiments in a mouse inhalation model of infection established that May1 is required for fungal virulence.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><div class="NLM_p last">The HIV-1 Pr inhibitors saquinavir, darunavir, ritonavir, and indinavir can affect production of important <i>C. neoformans</i> virulence factors.<a onclick="showRef(event, 'ref25 ref31'); return false;" href="javascript:void(0);" class="ref ref25 ref31">(25,31)</a> Similarly, the protease inhibitors indinavir, ritonavir, and pepstatin A inhibit growth of <i>Candida albicans</i> under nitrogen-limited conditions.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> People with HIV who have had cryptococcal meningitis have a lower risk of cryptococcosis relapse while taking antiretroviral therapy (ART) regimens including protease inhibitors, even without direct antifungal maintenance.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> A similar phenomenon has been hypothesized for <i>C. albicans</i> infections.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> These observations formed our basic rationale for exploring May1 as a possible secondary target of components of HIV-1 Pr inhibitor regimens as well as a direct target for antifungal drug development.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56687" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56687" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Expression and Characterization of Recombinant May1</h3><div class="NLM_p">Full-length <i>C. neoformans</i> var. <i>grubii</i> H99 May1 is a 489 amino acid long, multi-domain enzyme encoded by the gene cnap1 (CNAG_05872). To prepare a recombinant protein for subsequent biochemical and structural characterization, we designed a series of <i>C. neoformans</i> May1 constructs with N/C-terminal purification tags for recombinant expression in <i>Drosophila</i> S2 cells. The constructs span different regions of the protein according to the predicted architecture<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. The 17 N-terminal amino acids, forming a putative native secretion signal,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> were replaced with a secretion signal compatible with the <i>Drosophila</i> expression system.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Diagram of constructs used to identify May1 architecture and optimize expression strategy. Residue numbers indicate construct boundaries. The first 17 N-terminal amino acids, forming a putative native secretion signal, were replaced with a secretion signal compatible with the S2 expression system (not shown for clarity). Beveled rectangles denote positions of affinity tags used for purification.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We evaluated enzymatic activity using a fluorometric assay with the internally quenched fluorogenic substrate IQ-2 [AMC-GSPAF*LAK(DNP)dR-NH<sub>2</sub>; May1 cleaves between the phenylalanine and leucine residues.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Activity assays using IQ-2 confirmed that May1 is activated upon autoproteolytic cleavage of the prodomain<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and that the presence of the prodomain in the construct is necessary to retain activity. Interestingly, mass spectrometry analysis indicated promiscuous autoproteolytic cleavages from residue 76 to residue 79 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Figure S1a</a>). The presence of the C-terminal domain (downstream from residue 434) decreased the enzymatic activity. This domain was cleaved at residue 456 at the pH optimum of the protease, hinting at a possible regulatory or other function. The C-terminal cleavage site does not appear to be directly sequence-dependent, as constructs with different tags and multiple adaptor sequences (Strep-tag, Avi-tag, and His-tag) all underwent autocatalytic processing. The C-terminal purification tag required a thorough sequence optimization, which finally yielded a construct with a stable C-terminal biotinylation site (Avi-tag; construct 3 in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><div class="NLM_p">To obtain a sufficient quantity and purity of May1 for biochemical characterization, we conducted large-scale expression of construct 3 [May1(17-434)-Avi]. The recombinant protein was purified by affinity chromatography and size exclusion chromatography. Overall, 3 L of S2 culture media yielded 15 mg of monodisperse recombinant enzyme that was >95% pure according to SDS-PAGE (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) with an apparent molecular weight of 45 kDa. The mass spectrum shows multiple peaks with a dominant mass of 40,830 Da (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Figure S1b</a>). This value is larger than the theoretically largest possible average molecular weight of 40,145 Da (residues 77-434, including C-terminal tag and its biotinylation), which together with the broad peak distribution indicates that the protein undergoes glycosylation. The enzyme concentration and purity were further evaluated by amino acid analysis and active site titration.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Analysis of May1(17-434)-Avi expression and purification. (a) Silver-stained SDS-PAGE of samples collected throughout the purification process. MED, crude S2 media; PER, permeate from media concentration; LOAD, concentrated filtrate; FT, flow-through; W1-3, wash fractions; E1-4, elution fractions; POOL, pooled concentrated elution fractions; MTX, matrix binding control. (b) Western blot analysis with the same layout as in a visualized using Streptavidin-800CW (LI-COR, 1:15,000). (c) Silver-stained SDS-PAGE of concentrated pooled fractions after size exclusion chromatography.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Once the expression protocol was established, kinetic assay conditions were optimized to enable the design of a combinatorial screening assay. May1 exhibits optimal activity at pH 5.0, consistent with other members of the pepsin-like aspartyl protease family.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Ionic strength does not affect enzyme activity in the near-isotonic range (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Figure S2</a>). The enzyme does not significantly lose activity in <0.5% Triton X-100 and Tween-20 or <5% dimethyl sulfoxide (DMSO) (data not shown).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Substrate Specificity of May1</h3><div class="NLM_p">May1 exhibits a substrate preference for aromatic and positively charged residues at the P3–P1 and P1′ positions,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> like many proteases of the pepsin-like family.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> To gain broader structural insight into the substrate specificity of the enzyme, we evaluated its ability to cleave an internally quenched fluorescent substrate library with a general structure of ACC-GKSLGFLVAK(DNP)G-NH<sub>2</sub>. In the presumed P4–P1′ positions, we incorporated natural and unnatural amino acids (ACC-GK-P4-P3-P2-P1-P1′VAK(DNP)G-NH<sub>2</sub>). Library screening revealed that phenylalanine and its derivatives (Phe-3F, Phe-3Cl) and Ala(2-thienyl) were preferred amino acids at the P1′ position. Hydrophobic residues were favored at the P1 position [relative activity: Ile (100%), Phg (90%), Val (63%), Chg (38%), Abu (34%), Nle (32%)] (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Substrate specificity profile of May1 at the P4–P1′ positions presented as a heat map. L and D rows indicate the amino acid enantiomer at a given position. Dashed lines indicate substrate cleavage sites detected by LC–MS, with the most abundant N-proximal Phe*Leu site selected as a residue numbering reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Positively charged amino acids were preferred in the S2 pocket [Arg (100%), Lys (89%), Orn (65%)]. This pocket also was able to accommodate polar, uncharged residues including hSer, Cit, Thr, and Ser. May1 favored hydrophobic amino acids at the P3 and P4 positions. Interestingly, in addition to the expected processing at P1–P1′ (Phe–Leu), we also observed substrate cleavage at P1′–P2′ (Leu–Val), suggesting some degree of catalytic promiscuity.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> May1 Inhibition by Candidate and Clinically Used Protease Inhibitors</h3><div class="NLM_p">To further investigate the biochemical basis of reports that HIV-1 Pr inhibitor regimens lower relapse rates of cryptococcal meningitis<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and to explore a possible link of this phenomenon to May1 activity, we screened all protease inhibitors currently used in ART regimens<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> to assess their inhibitory activity against May1 and validate previously published results.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><div class="NLM_p last">Only two such protease inhibitors exhibited detectable inhibitory activity against May1: amprenavir (<i>K</i><sub>i</sub> = 750 ± 320 nM) and ritonavir (<i>K</i><sub>i</sub> = 1.8 ± 0.5 μM). We also screened an extended panel of candidates and discontinued HIV-1 Pr inhibitors. CGP53437 (discontinued at phase I, Novartis<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>), GS-8374 (discontinued at a preclinical stage, Gilead<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a>), and brecanavir (discontinued at phase II, GlaxoSmithKline<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a>) inhibited May1 with <i>K</i><sub>i</sub> values of 13 ± 10, 170 ± 10, and 350 ± 20 nM, respectively. The relatively high <i>K</i><sub>i</sub> values of amprenavir and ritonavir do not support the notion that May1 inhibition is a major biochemical mechanism underlying the beneficial effect of HIV-1 Pr inhibitors in preventing cryptococcal meningitis, suggesting an alternative mode of action for these protease inhibitors.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Crystal Structure of May1</h3><div class="NLM_p">We crystallized a May1 construct spanning residues 17 to 434 with a C-terminal Avi tag both in free form and in complex with pepstatin A (PepA), an aspartyl protease inhibitor reported to inhibit May1.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> We verified that May1(17-434)-Avi is inhibited by both PepA and acetyl pepstatin with <i>K</i><sub>i</sub> values of 1.3 ± 0.1 and 3.3 ± 0.1 nM, respectively.</div><div class="NLM_p">Diffraction data for May1 and the May1–PepA complex were collected to maximal resolutions of 1.75 and 1.8 Å, respectively. The May1 crystal structure was determined by molecular replacement using the structure of human renin<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> as a model. The centered orthorhombic crystal contained 63% solvent and one protein molecule per asymmetric unit, representing a monomeric biological unit. Residues 86-435 were modeled into the electron density map and numbered 7-356 starting with the N-terminus generated upon cleavage at position 79. The structure revealed that the boundary of the pepsin-like domain is shifted by four residues toward the N-terminus compared to the predicted one (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). No electron density was observed for N-terminal residues 80–85 or the C-terminal Avi tag; residue 356 in the crystal structure represents a cloning artifact. Additional residues that were not modeled in the May1–PepA crystallographic model include residues 226–236 that belong to a surface-exposed loop with poor electron density. Both structures contain a glycosylated asparagine at position 187, modeled as GlcNAc-β(1 → 4)GlcNAc-β-Asn187.</div><div class="NLM_p">As previously predicted by homology modeling,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> May1 adopts a renin fold, which is a predominantly β-sheet conformation characteristic of the aspartyl proteinase family.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The substrate binding cleft and active site are at the junction of two structurally similar domains of approximately equal size.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The catalytic residues, Asp40 and Asp238, are located centrally in the cleft (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) with the carboxyl side chains and surrounding main chain scaffolding related by an approximate twofold interdomain axis.<a onclick="showRef(event, 'ref43 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref43 ref45 ref46">(43,45,46)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Crystal structure of recombinant May1 in free form (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5H">6R5H</a>). The catalytic Asp residues are highlighted as magenta sticks. (b) Superposition of May1 (green) with human renin (magenta, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2REN">2REN</a><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Structural comparison with models deposited in the Protein Data Bank using the program DALI<a onclick="showRef(event, 'ref47 ref58'); return false;" href="javascript:void(0);" class="ref ref47 ref58">(47,48)</a> identified various aspartyl proteases as structural homologues of May1, with human renin being the closest homologue (the RMSD for Cα superposition of 232 residues was 1.5 Å). We also identified structural similarity to HIV-1 Pr<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(49)</a> despite its low sequence identity of 13% (the RMSD for Cα superposition of 175 residues was 3.2 Å).</div><div class="NLM_p">PepA binds in the substrate binding groove of May1, and its central statine residue with a non-scissile bond acts as a tetrahedral transition-state mimic. The first five residues of the PepA molecule were modeled into a well-defined continuous electron density map. The ambiguous electron density map for the C-terminal part of the inhibitor can be explained by two equally occupied conformations of the terminal Sta residue (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). The May1 structure does not change significantly upon PepA binding; the RMSD for superposition of the free and complexed May1 structures was 0.129 Å.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. PepA binding to May1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R6A">6R6A</a>). (a) Top view of the binding pose of PepA in the May1 active site. The 2<i>Fo</i>-<i>Fc</i> map (contoured at 1.5σ) is shown in blue; catalytic aspartates (sticks with carbons colored cyan) interact with the central statine residue. Residues contributing to PepA binding in the S4–S3 subsites are indicated; those forming polar interactions are underlined. (b) Ligand–protein interaction diagram of the May1–PepA complex generated by LigPlot.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(50)</a> Residues engaged in polar ligand interactions (green dashed lines) are shown; numbers represent distances between hydrogen bond donor and acceptor in Å. (c) Model of N-terminally carboxybenzylated phenylstatine superimposed with PepA. Points of attachment for the R1 and R2 substituents are indicated by arrows. The protein is represented by its solvent accessible surface area colored by electrostatic potential (red for negative, blue for positive). Residues available to form polar interactions with the P1′ substituent are shown as sticks and labeled, and possible interactions are indicated by dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">PepA binds to May1 in an extended conformation occupying the S4–S2′ substrate binding subsites. The hydroxyl group of the central statine moiety forms two hydrogen bonds with the side chains of the two catalytic aspartates (Asp40 and Asp238). Additional hydrogen bonding interactions involve the main chain atoms of Gly42 (O), Gly84 (N), Thr82 (O), Ser85 (N), and Thr242 (N) and the side chain atoms of Ser85 (OG), Tyr207 (OH), Thr241 (OG), and Thr242 (OG). In addition to this hydrogen bonding network, the inhibitor forms extensive van der Waals interactions with May1 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b).</div><div class="NLM_p last">The binding pose of PepA in the active site was used to validate N-terminally carboxybenzylated phenylstatine as a suitable scaffold for production of an inhibitor library. We constructed a model by altering PepA substituents in the P2, P1, and P1′ positions and fitting them into the corresponding enzyme subsites. The model indicated that the active site of May1 is able to accommodate bulky benzene substituents in the P2 and P1 positions and polar substitutions in P1′ (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c). Moreover, we identified potential hydrogen bond donors and acceptors for interaction with P1′ polar groups (OH or NH<sub>2</sub>). The model also showed that the binding pose allows for attachment of an R2 substituent that can be extended into the S3′ subsite (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Inhibitor Motif Identification Using a Combinatorial Approach</h3><div class="NLM_p">Our first approach to lead identification involved screening a library of candidates comprising N-terminally carboxybenzylated<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(51)</a> (Z) aromatic peptide bond mimetics followed by a single amino acid and ending with a degenerate amino acid amide assembled by the split and mix method. We used the known peptide bond mimetics phenylnorstatine (Pns<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(52)</a>), 3-amino-4-phenylbutanoic acid (Apb<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(53)</a>), phenylstatine (Pst<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(54)</a>), a Phe–Phe moiety with a reduced peptide bond (Phe–CH<sub>2</sub>–NH–Phe<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(55)</a>) and a Phe–Phe moiety analogue with a hydroxyethylamine mimetic linkage (Phe–HEA–Phe<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(56)</a>). For the two attached amino acids, we used a selection of 48 amino acid building blocks, including all proteinogenic amino acids except cysteine, 16 corresponding D-enantiomers, and 13 synthetic amino acid derivatives.<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(57,58)</a></div><div class="NLM_p">To conduct the screening, we developed an automated assay using the internally quenched fluorescent substrate IQ-2. This method allowed us to evaluate the relative activity of individual compound mixtures expressed as a fluorescence percentage ratio between a control reaction with no inhibitor and the test compound mixture (500 nM per compound) using the PepA-inhibited reaction as a background control. This assay, combined with our combinatorial approach to inhibitor screening, allowed us to perform duplicate evaluation of 11,520 compounds in two 384-well plates to assess pharmacophore preference in the P2′ position (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Linear statin inhibitor structures and residue preference heatmaps. L and D rows indicate the amino acid enantiomer at a given position. (a) Initial screening of the N-proximal site with a degenerate C-terminal residue. A final concentration of 500 nM per individual compound was used in library screening. (b) Deconvolution of selected libraries with favorable N-proximal residues based on initial screening. Single-compound deconvolution experiments were conducted with a 1 μM final concentration of the test compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The results of this screening were cross-referenced with the binding preferences of HIV-1 Pr for similar statin derivatives<a onclick="showRef(event, 'ref52 ref56'); return false;" href="javascript:void(0);" class="ref ref52 ref56">(53,57)</a> to determine which libraries to further deconvolute to resolve the P3′ position preference and ultimately obtain a consensus inhibitor structure. HIV-1 Pr shared a preference for P2′ <span class="smallcaps smallerCapital">l</span>-Glu and <span class="smallcaps smallerCapital">l-</span>Gln in the Pst library class, <span class="smallcaps smallerCapital">l-</span>Glu in the Phe–HEA–Phe class, and Aib in the Pns class. These four library mixtures were deconvoluted by screening the 48 individual mixture components (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b). Based on the results of this preliminary screen, we determined <i>K</i><sub>i</sub> values of the selected compounds presumed to inhibit at concentrations in the nanomolar range (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compound <b>13a</b> (Z-Pst–<span class="smallcaps smallerCapital">l-</span>Glu–Hph–NH<sub>2</sub>) had the most favorable binding to May1 with <i>K</i><sub>i</sub> = 12 ± 1 nM and was therefore selected as the lead compound for further analysis.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Selected May1 Inhibitors Derived from an N-Terminally Carboxybenzylated Phenylstatine Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0010.gif" alt="" id="GRAPHIC-d7e872-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">May1 <i>K</i><sub>i</sub> [nM]</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">P2′</th><th class="colsep0 rowsep0" align="center" char="±">P1′: <span class="smallcaps smallerCapital">L</span>-Glu (a)</th><th class="colsep0 rowsep0" align="center">P1′: <span class="smallcaps smallerCapital">L</span>-Gln (b)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">l-Abu</td><td class="colsep0 rowsep0" align="char" char="±">760 ± 50</td><td class="colsep0 rowsep0" align="left">1200 ± 100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">l-Nva</td><td class="colsep0 rowsep0" align="char" char="±">410 ± 60</td><td class="colsep0 rowsep0" align="left">470 ± 40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">l-Val</td><td class="colsep0 rowsep0" align="char" char="±">82 ± 12</td><td class="colsep0 rowsep0" align="left">120 ± 20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">l-Nle</td><td class="colsep0 rowsep0" align="char" char="±">180 ± 20</td><td class="colsep0 rowsep0" align="left">170 ± 10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">l-Leu</td><td class="colsep0 rowsep0" align="char" char="±">160 ± 10</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">l-Ile</td><td class="colsep0 rowsep0" align="char" char="±">410 ± 60</td><td class="colsep0 rowsep0" align="left">240 ± 30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">l-Thr</td><td class="colsep0 rowsep0" align="char" char="±">250 ± 20</td><td class="colsep0 rowsep0" align="left">1300 ± 200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">l-Met</td><td class="colsep0 rowsep0" align="char" char="±">280 ± 20</td><td class="colsep0 rowsep0" align="left">290 ± 40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">l-Trp</td><td class="colsep0 rowsep0" align="char" char="±">340 ± 30</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left"><span class="smallcaps smallerCapital"><u class="uu">l</u></span>-Cha</td><td class="colsep0 rowsep0" align="char" char="±">160 ± 20</td><td class="colsep0 rowsep0" align="left">400 ± 60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left"><span class="smallcaps smallerCapital"><u class="uu">l</u></span>-Phg</td><td class="colsep0 rowsep0" align="char" char="±">210 ± 20</td><td class="colsep0 rowsep0" align="left">360 ± 40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left"><span class="smallcaps smallerCapital"><u class="uu">l</u></span>-Phe</td><td class="colsep0 rowsep0" align="char" char="±">390 ± 50</td><td class="colsep0 rowsep0" align="left">320 ± 30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left"><span class="smallcaps smallerCapital"><u class="uu">l</u></span>-Hph</td><td class="colsep0 rowsep0" align="char" char="±">12 ± 1</td><td class="colsep0 rowsep0" align="left">64 ± 7</td></tr></tbody></table></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Off-Target Activity of Lead Compound Z-Pst–<span class="smallcaps smallerCapital">l</span>-Glu–Hph–NH<sub>2</sub></h3><div class="NLM_p">Off-target activity is a major concern in the design of protease inhibitors. Lead compound <b>13a</b> was thus investigated for <i>in vitro</i> off-target activity against several relevant host aspartyl proteases—human pepsin, renin, and cathepsins D and E (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In addition to inhibiting target protease May1, <b>13a</b> was a potent, subnanomolar inhibitor of wild-type HIV-1 Pr.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. <i>In Vitro</i> Off-Target Profile of Lead Compound Z-Pst–<span class="smallcaps smallerCapital">l</span>-Glu–Hph–NH<sub>2</sub></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">May1 <i>C. neoformans</i></th><th class="colsep0 rowsep0" align="center">HIV-1 Pr <i>wild-type</i></th><th class="colsep0 rowsep0" align="center" char="±">Pepsin <i>porcine</i></th><th class="colsep0 rowsep0" align="center" char="±">Renin <i>human</i></th><th class="colsep0 rowsep0" align="center">Cathepsin D <i>human</i></th><th class="colsep0 rowsep0" align="center">Cathepsin E <i>human</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>K</i><sub>i</sub> 12 ± 1 nM</td><td class="colsep0 rowsep0" align="left">32 ± 5 pM<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(57)</a></td><td class="colsep0 rowsep0" align="char" char="±">1.6 ± 0.3 μM</td><td class="colsep0 rowsep0" align="char" char="±">19 ± 2 μM</td><td class="colsep0 rowsep0" align="left">620 ± 50 nM</td><td class="colsep0 rowsep0" align="left">440 ± 70 nM</td></tr></tbody></table></div></div><div class="NLM_p last">While the lead compound inhibited both cathepsins D and E at higher concentrations, the overall cytotoxic effect of the compound was very low, with CC<sub>50</sub> higher than 80 μM for multiple cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Table S2, Figure S3</a>). As exemplified by experiments with the HepG2 cell line, the cytotoxicity is on par with or lower than that of clinically used protease inhibitors, including darunavir, amprenavir, and indinavir.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Activity of Lead Compound against Cultured <i>C. neoformans</i> Yeast</h3><div class="NLM_p">Next, we assessed whether May1 inhibitors identified here were effective against cultured <i>C. neoformans</i> yeast. Previous work showed that May1 activity is required for acidic pH tolerance of yeast grown in minimal medium.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> May1 deficiency manifested as a culture saturation density phenotype wherein <i>may1</i>Δ or pepstatin A-treated yeast grew normally during logarithmic growth but achieved lower terminal densities during the stationary phase after the medium had acidified.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> To identify compounds with activity against May1 in an <i>in vivo</i> setting, we treated yeast with a panel of inhibitors that had displayed a range of <i>in vitro</i> efficacies against May1 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a). After 48 h of culture, yeast treated with compounds <b>3a</b>, <b>4a</b>, and <b>13a</b> exhibited dose-dependent reductions in terminal culture density, with 50 μM <b>13a</b> treatment nearly phenocopying the <i>may1</i>Δ mutant (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Evaluation of May1 inhibitor activity against cultured <i>C. neoformans</i> yeast. (a) Terminal saturation densities of yeast cultured for 48 h in YNB minimal medium containing May1 inhibitors or a 0.5% DMSO vehicle. Dashed lines represent terminal OD<sub>600</sub>’s of the DMSO-treated H99 and <i>may1</i>Δ controls. Error bars represent standard deviation of the mean from triplicate cultures. (b) Growth curves of H99 and <i>may1</i>Δ yeast treated with lead compound <b>13a</b>, negative control compound <b>1b</b>, or a 0.5% DMSO vehicle control in either standard YNB medium (left) or YNB medium buffered to pH 6.5 (right). Error bars represent standard deviation of the mean from triplicate cultures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">If <b>13a</b> activity is selective for May1, we expected that (1) <b>13a</b>-treated yeast should grow similarly to untreated yeast during the logarithmic phase; (2) <b>13a</b>-treatment should not further inhibit the growth or terminal saturation density of <i>may1</i>Δ yeast, which are already null for <i>may1</i>; and (3) <b>13a</b>-treatment should not affect the growth of yeast cultured in medium buffered to pH 6.5, which negates the need for May1 activity.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Consistent with <b>13a</b> being a specific inhibitor of May1, <b>13a</b>-treated yeast achieved similar densities to <b>1b</b>- or DMSO-treated controls during the logarithmic phase growth despite failing to attain the same terminal density at saturation (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b, left). Additionally, <b>13a</b>-treatment of <i>may1</i>Δ yeast did not further reduce culture densities during either the logarithmic or the stationary phase (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b, left), and buffering the medium to pH 6.5 eliminated the saturation density phenotype of both <i>may1</i>Δ and <b>13a</b>-treated yeast (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b, right). Altogether, these data indicate that compound <b>13a</b> is a selective May1 inhibitor in the context of cultured <i>C. neoformans</i>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28016" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28016" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">May1 has a high degree of similarity to other members of the aspartyl protease family, and we identified aspartyl proteases including endohiapepsin,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a><i>Rhizomucor miehei</i> aspartyl proteinase,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and candidapepsin<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> as its closest structural homologues. Although the May1 sequence is distinct from that of HIV-1 Pr (13% similarity), structural similarity between the two enzymes is high, with an RMSD for superposition of 175 residues of 3.2 Å (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a). Comparison of the substrate binding clefts and binding poses of pepstatin inhibitors revealed additional similarities (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b,c).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Comparison of May1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5H">6R5H</a>) with HIV-1 Pr (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2hvp">2hvp</a><a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a>). (a) Overall superposition of May1 (green) with HIV-1 Pr (maroon). Catalytic aspartates are highlighted in stick representation. (b) Top view into the May1 active site. The protein is represented by the solvent accessible surface area colored by electrostatic potential; PepA is represented as sticks with carbon atoms colored green; the inhibitor pose in HIV-1 Pr is shown with black lines. (c) Top view into the HIV-1 Pr active site. The protein is represented by solvent accessible surface area colored by electrostatic potential; acetyl-pepstatin is represented as sticks with carbon atoms colored yellow; the PepA pose in May1 is shown with black lines. In b and c, flap regions covering the active site are omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Inhibitors bind to the active sites of both enzymes in a comparable manner, especially for the P1 to P4 substituents (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b,c). On the other hand, we observed variation in the positions of P1′, P2′, and P3′ substituents. May1 binding sites on the prime side form a rather wide groove that could accommodate larger P2′ and P3′ substituents, such as Hph in Z-Pst–<span class="smallcaps smallerCapital">l</span>-Glu–Hph–NH<sub>2</sub>.</div><div class="NLM_p">Internally quenched fluorescent substrate libraries containing natural and unnatural amino acids are a convenient chemical tool for investigating the substrate specificity of endopeptidases.<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(63,64)</a> Inclusion of unnatural amino acids with diverse chemical structures enables precise examination of the binding pocket architecture. We confirmed the preference of May1 for large aromatic and hydrophobic residues in the P1–P1′ positions, as expected for an aspartyl protease, and observed favorable binding of positively charged amino acids at the P2 subsite. These findings are in agreement with the promiscuous GIT*R*R*K*RAN prodomain cleavage sites identified by mass spectrometry (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Figure S1a</a>) and follow general preferences for aspartyl protease sequence specificity.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(65)</a> The observed preference for hydrophobic and positively charged amino acids potentially implicates May1 in cleavage of lamin, fibronectin, and elastin,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(66)</a> further reinforcing the link between May1 and extracellular matrix degradation.</div><div class="NLM_p">Next, we set out to identify a potential dual-specificity inhibitor of HIV-1 Pr and May1. Pepstatin is a prototypical aspartyl protease inhibitor with picomolar binding affinity toward cathepsin D-like enzymes.<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(63,64)</a> We and others have previously shown that pepstatin derivatives can serve as a useful scaffold for design of highly potent inhibitors of HIV-1 Pr and cathepsin D-like enzymes.<a onclick="showRef(event, 'ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref67 ref68">(67,68)</a> Therefore, we based our inhibition specificity screen on pepstatin-like scaffolds (phenylstatine and phenylnorstatine) and complemented them with other peptidomimetics such as Apb, a reduced peptide bond, and a hydroxyethylamine peptide bond isostere. Libraries containing peptidomimetics other than Apb showed inhibitory activity against May1. Libraries containing Pns and Phe–CH<sub>2</sub>–NH–Phe mimetics with <span class="smallcaps smallerCapital">l</span>-amino acid enantiomers at the P2′ position were generally good inhibitors, with no strong preference for a single amino acid class. Only peptides with orientation-changing residues (<i>D</i>-enantiomers, Aib, Pro, Tic) were disfavored. Phe–HEA–Phe libraries had a narrower preference for charged amino acids and shared the trend of favoring P2′ <span class="smallcaps smallerCapital">l</span>-Glu rather than <span class="smallcaps smallerCapital">l</span>-Asp with the Phe–CH<sub>2</sub>–NH–Phe class. Curiously, Phe–HEA–Phe and Pns libraries were more tolerant of Aib substitution, which was also the dominant residue in the Pns class. The observed preference for hydrophobic and charged residues at the P2′ position in the Pst, Phe–HEA–Phe, and Phe–CH<sub>2</sub>–NH–Phe mimetic classes is not in complete agreement with a recently published May1 substrate specificity analysis,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> which reports hydrophobic and aromatic residues being favored in P1′–P3′, with Asn and Gln being disfavored in P2′ presumably due to the difference in the libraries used to probe the specificity.</div><div class="NLM_p">Deconvolution screening of the Pns–Aib and Phe–HEA–Phe libraries did not yield any major hits. However, these libraries shared a common preference for some hydrophobic and aromatic residues in the P3′ position with the Pst–<span class="smallcaps smallerCapital">l</span>-Glu and Pst–<span class="smallcaps smallerCapital">l</span>-Gln libraries, which showed an almost exclusive preference for this class of residues. Most importantly, Pst–<span class="smallcaps smallerCapital">l</span>-Glu and Pst–<span class="smallcaps smallerCapital">l</span>-Gln inhibitors with aromatic residues at the P3′ position were favored, similarly as for HIV-1 Pr inhibitors of these classes.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(53)</a> Both Pst–<span class="smallcaps smallerCapital">l</span>-Glu and Pst–<span class="smallcaps smallerCapital">l</span>-Gln exhibited the strongest inhibitory effect with <span class="smallcaps smallerCapital">l</span>-homophenylalanine (Hph) at the terminal P3′ position, which we determined to be the most favorable residue among the 48 assayed blocks.</div><div class="NLM_p last">The lead structure obtained by combinatorial screening—<b>13a</b>—is a potent inhibitor of May1 with <i>K</i><sub>i</sub> = 12 ± 1 nM, and its activity profile in yeast culture indicates that its mode of antifungal action is driven specifically by MayI inhibition. It also inhibits wild-type HIV-1 Pr with <i>K</i><sub>i</sub> = 32 ± 5 pM and three drug resistance-associated mutants with <i>K</i><sub>i</sub> values ranging from 24 to 320 pM.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(57)</a> Compound <b>13a</b> also showed remarkable selectivity between May1 and human renin, its closest structural homologue. This can be attributed to the distinct renin inhibitor preference.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Renin inhibition favors non-aromatic transition-state analogues preceded by nonpolar residues,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> whereas <b>13a</b> as a May1 inhibitor possesses a phenylstatine analogue preceded by a charged Glu residue. This lead compound therefore exemplifies a common inhibitor of both HIV-1 Pr and May1 with a favorable <i>in vitro</i> cytotoxicity and anti-fungal activity profile.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24311" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24311" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this work, we structurally and functionally characterized <i>C. neoformans</i> May1, a secreted fungal protease. Using structure-guided combinatorial screening, we identified a common inhibitor of both May1 and HIV-1 Pr. With a favorable cytotoxicity profile, low <i>in vitro</i> off-target activity, and demonstrated activity in yeast culture, this compound could serve as a scaffold for further inhibitor development.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24083" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24083" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Expression of Recombinant May1</h3><div class="NLM_p last">An established protocol for the expression and purification of proteins in Drosophila S2 cells<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> was used to prepare the May1(17-434)-Avi construct for kinetics and crystallization. Full-length cnap1 gene cDNA encoding May1 (CNAG_05872) generated <i>in silico</i> in pUC57 plasmid was purchased from Genscript. The 17-434 fragment was PCR amplified using primers 5′-AAAAAGATCTGCACCAAGCAACAGCGTC-3′ and 5′-AAAAGCGGCCGCCAGTTCAGCGAATCCGAT-3′, which include <i>Bgl</i>II and <i>Not</i>I restriction sites. The amplified fragment was purified by gel extraction and digested with <i>Bgl</i>II and <i>Not</i>I (NEB R0144, R0189) per the manufacturer’s protocol. The cleaved fragment was again purified and ligated using T4 DNA ligase (NEB M0202) into an analogously digested and purified pMT_BiP_CAvi plasmid fragment. The ligation reaction mixture was transformed into TOP10 competent <i>Escherichia coli</i> (Invitrogen C404010), and the sequence identity of the resulting plasmid DNA was verified by sequencing. Following transfection of S2 cells using calcium phosphate and selection of an expression clone (300 μg/mL hygromycin-B), a large-scale expression in 3 L of serum-free media (Thermo Fisher Scientific 21720) was carried out. The expressed construct was concentrated and purified using affinity chromatography followed by size exclusion chromatography. Briefly, crude media was concentrated using tangential flow filtration (Labscale TFF), yielding a 50-fold concentrate and permeate, which was discarded. The concentrate dialyzed against 100 mM Tris, pH 7.2, 150 mM NaCl (Buffer W) was purified using Streptavidin Mutein Matrix (Roche 3708152001) in a cold room. The flow-through fraction was collected and used for subsequent purification (a total of 4 times per expression run). The enzyme was eluted using 2 mM biotin in Buffer W. Elution fractions were pooled, concentrated, and exchanged into 50 mM sodium acetate, pH 5.0, 50 mM sodium chloride (Buffer P) by ultrafiltration to avoid loss of activity outside the pH optimum and then purified by Superdex 75 gel filtration on a GE ÄKTA Explorer FPLC system. Size exclusion chromatography yielded a monodisperse protein preparation as verified by dynamic light scattering (Malvern Zetasizer Nano ZS, data not shown). Overall, 3 L of S2 culture yielded 15 mg of >95% pure (SDS-PAGE) monodisperse recombinant May1. Enzyme concentration and purity was further evaluated by amino acid analysis and active site titration. Protein mass was evaluated using Bruker Daltonics UltrafleXtreme a MALDI-TOF mass spectrometer.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Combinatorial Library Testing</h3><div class="NLM_p last">Assays were conducted in 384-well plates with transparent flat bottoms (Greiner 781097) in a stop-point format. Assay mixtures consisted of 15 μL of 50 mM sodium acetate, pH 5.0, 50 mM sodium chloride, 0.05% Tween-20 (Buffer R) with 20 μM IQ-2 substrate [AMC-GSPAFLAK(DNP)dR-NH<sub>2</sub>]. Test compounds were dispensed by a LabCyte Echo 550 acoustic dispensing system with a final DMSO content of 0.8% (v/v). For library mixtures, a final concentration of 500 nM of each individual compound was used; for single-compound deconvolution experiments, a 1 μM final concentration was used. Reactions conducted at 37 °C were started by sequential addition of 5 ng May1 in 10 μL of Buffer R to each well by an EnSpire Multilabel Reader and stopped after 20 min by addition of 12 μL of 50 μM pepstatin A in Buffer R. Reaction fluorescence (excitation at 328 nm, emission at 393 nm) was analyzed using a plate reader (Tecan Infinite M1000). Each plate contained two columns of positive control wells containing 1 μM pepstatin A and negative controls with DMSO to determine <i>Z</i>′ and a vertical signal shift.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> <i>K</i><sub>i</sub> Determination</h3><div class="NLM_p last">The combinatorial library screening assay was adjusted to contain a threefold dilution series of inhibitors with 10 points in total designed to be centered around the expected <i>K</i><sub>i</sub> determined from preliminary assays. <i>K</i><sub>i</sub> values were determined by nonlinear regression analysis using the Williams–Morrison equation (GraphPad Software GraphPad Prism 6) and applying the Cheng-Prusoff model for correction based on the <i>K</i><sub>m</sub> value of the IQ-2 substrate (21 ± 4 μM), as previously described.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Inhibition curves are included in Figure S4 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Active Site Titration</h3><div class="NLM_p last">The fraction of active protease in our enzyme preparations was determined by assessing 20 μM IQ-2 substrate cleavage efficiency in the presence of 50 nM May1 protease in Buffer P and titrating with increasing amounts of PepA (<i>K</i><sub>i</sub> = 1.3 nM). In a linear regression model of the percentage of protease activity against the PepA concentration, the <i>x</i>-axis intercept represents the concentration of active protease present in the sample. The active site concentrations of all preparations were close to the respective protein concentrations determined by amino acid analysis, indicating that the enzyme preparations were >99% active (data not shown).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Determining the May1 pH Optimum</h3><div class="NLM_p last">Fluorogenic activity assays were conducted using 20 μM IQ-2 with 50 ng of May1(17-434)-Avi in 40 mM phosphate-borate-acetate Britton–Robinson buffer<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> from pH 2.0 to 13.0 with steps of 0.5. The activity maximum was further resolved with a series of buffers with pH values ranging from 4.0 to 6.0 with steps of 0.2.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Substrate Specificity Screening</h3><div class="NLM_p last">To obtain a substrate specificity matrix, the peptide ACC-GKSLGFLVAK(DNP)G-NH<sub>2</sub> was modified by incorporating natural and unnatural amino acids in the SLGFL sequence to yield five internally quenched fluorescent substrate sublibraries (P1-4 and P1′), each consisting of 85–91 compounds. These sublibraries were screened at 2 μM with 2 nM May1(17-434)-Avi100 in Buffer P (1% v/v DMSO) in a final volume of 100 μL. Substrate specificity screening was conducted in an opaque 96-well plate (Costar, Corning). The enzyme was incubated in buffer P for 15 min at 37 °C prior to substrate addition, and substrate hydrolysis was monitored by fluorescence measurement on a SpectraMax Gemini XPS Microplate Reader (Molecular Devices) in kinetic mode (excitation at 355 nm, emission at 460 nm) for 15 min, using only the linear portion of the hydrolysis curve to calculate the final reaction velocity (relative fluorescent units per second, RFU/s). Screening of libraries was performed in duplicate, and the resulting average value was used to calculate May1 specificity matrices. Every sublibrary was scaled to 100% for the amino acid with the highest RFU/s value.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Assays for Off-Target Activity</h3><div class="NLM_p last">Selected lead inhibitors were screened for off-target activity against pepsin (porcine, Sigma-Aldrich P7012), renin (human, Biovision 6300), cathepsin D (human, Athens Research & Technology 16-12-030104), and cathepsin E (human, Biovision 7842). We followed established protocols with the substrates BSA–bromophenol blue<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> (formal <i>K</i><sub>m</sub> = 80 μM) for pepsin, H-R-E(EDANS)-IHPFHLVIHT-K(DABCYL)-R-OH<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> (<i>K</i><sub>m</sub> = 2.5 μM, Anaspec AS-62022) for renin, and ACC-GKPILFFRLK(DNP)-(dR)-NH<sub>2</sub><a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> (<i>K</i><sub>m,CatD</sub> = 3.7 μM, <i>K</i><sub>m,CatE</sub> = 3.3 μM, Anaspec AS-61793) for cathepsins D and E. Inhibition curves are included in Figure S5 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Yeast Culture Activity Assay</h3><div class="NLM_p last"><i>C. neoformans</i> strains H99 (CM018) and <i>may1</i>Δ (CM1383)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> were cultured in Yeast Nitrogen Base (YNB) medium containing 2% glucose. To assess May1 inhibitors for reduction of saturation culture density, overnight cultures were diluted to OD<sub>600</sub>’s of 0.1 the presence of 0.5% DMSO vehicle or 0.1, 1.0, 10, or 50 μM compound. Cultures were incubated at 30 °C on a tube rotator, and OD<sub>600</sub>’s were measured after 48 h. For full growth curves, logarithmically growing cultures were pelleted and resuspended to OD<sub>600</sub>’s of 0.1 in either unbuffered YNB or YNB containing 50 mM MES, pH 6.5. Cultures were treated with 50 μM <b>13a</b>, 50 μM <b>1b</b>, or 0.5% DMSO vehicle and were incubated at 30 °C on a tube rotator. OD<sub>600</sub>’s were measured at 2 h intervals for 12 h and after 24 and 48 h of growth.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Crystallization and X-ray Data Collection</h3><div class="NLM_p">Diffraction-quality crystals of native May1 were obtained at 25 °C using the hanging-drop vapor diffusion technique by mixing equal volumes of May1(17-434)-Avi (83 mg/mL) in Buffer P with reservoir solution composed of 200 mM lithium sulfate, 45% (v/v) PEG-400, 100 mM sodium acetate, pH 4.5. After 7 days, crystals used for X-ray data collection were harvested, flash-frozen in liquid nitrogen with the reservoir solution as self-cryoprotectant, and stored. These conditions yielded crystals of space group <i>C</i>222<sub>1</sub> (<i>a</i> = 97.4 Å, <i>b</i> = 112.1 Å, <i>c</i> = 91.2 Å, α = β = γ = 90°) diffracting up to 1.75 Ε. Co-crystallization of May1 with PepA was performed using the same technique. The May1–inhibitor complex was formed by incubating May1(17-434)-Avi protein (1 mg/mL) in Buffer P with PepA (100 μM final concentration) at 4 °C for 1 h. The complex was then concentrated, and excess inhibitor was removed by ultrafiltration using an Amicon 3K MWCO centrifugal device. Diffraction quality crystals were obtained at 25 °C by mixing equal volumes of the concentrated May1–PepA complex with the same reservoir solution as above. Drops were streak-seeded with previously obtained native May1 crystal seeds. After 4–9 days, the crystals were flash-frozen in liquid nitrogen and stored for X-ray data collection.</div><div class="NLM_p last">Diffraction data for the May1–PepA complex were collected at 100 K on an in-house MicroMax-007 HF Microfocus X-ray generator with a VariMax VHF ArcSec confocal optical system (Rigaku, Japan), an AFC11 partial four-axis goniometer (Rigaku, Japan), a PILATUS 300 K detector (Dectris, Switzerland), and a Cryostream 800 cryocooling system (Oxford Cryosystems, England). Diffraction data for free May1 were collected at 100 K on MX 14.1 operated by the Joint Berlin MX-Laboratory at the BESSY II electron-storage ring in Berlin-Adlershof, Germany.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Diffraction data were processed using XDS.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> The crystal parameters and data collection and refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Table S1</a>.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Structure Determination, Model Building, and Refinement</h3><div class="NLM_p last">The structure of the May1–PepA complex was solved by molecular replacement with CCP4 Molrep<a onclick="showRef(event, 'ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref78 ref79">(78,79)</a> using the structure of human renin (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2REN">2REN</a>)<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and the native May1 structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5H">6R5H</a>, this work) as search models. The initial models were refined through several cycles of manual building using Coot and automated refinement with CCP4 REFMAC5.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> Visualization of structural data was performed in PyMOL 0.99rc6, and 2D diagrams summarizing molecular interaction between inhibitors and May1 were prepared using LigPlot.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(50)</a> Atomic coordinates and structure factors were deposited into the Protein Data Bank under codes 6R5H and 6R6A for free May1 and the May1–PepA complex, respectively.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Organic Synthesis</h3><div class="NLM_p">Synthesis and characterization of linear statin combinatorial libraries have been previously described.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(58)</a> Briefly, libraries were prepared on a Rink amide linker resin using an automated Fmoc/HOBt/DIC SPPS strategy. Degenerate positions were introduced by the split and mix method using 48 amino acid building blocks and five peptidomimetic building blocks with carboxybenzylated N-termini. Deconvoluted libraries and individual assayed compounds were synthesized as previously described.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(57)</a> Internally quenched fluorescent substrate libraries were prepared using standard SPPS as previously described.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a></div><div class="NLM_p"><sup>1</sup>H and <sup>13</sup>C NMR spectra were measured using a Bruker AVANCE III HD 400 with broadband PRODIGY cryo-probe. High-resolution ESI mass spectra were recorded using a hybrid FT mass spectrometer with an Orbitrap mass analyzer (LTQ Orbitrap XL, Thermo Fisher Scientific). Nominal ESI mass and purity of the compounds were assessed on an analytical Agilent 6230 Accurate-Mass TOF liquid chromatograph–mass spectrometer (flow rate of 0.3 mL min<sup>–1</sup>, invariable gradient from 2 to 100% acetonitrile in 8 min, 0.1% formic acid) with a Waters ACQUITY UPLC BEH C18 Column (1.7 μm, 100 × 2.1 mm).</div><div class="NLM_p">The compounds were purified by preparative HPLC on an Agilent 1200 Series instrument equipped with a G1365D DAD detector and Vydac 218TP101522 C18 Prep Column (10–15 μm, 22 × 250 mm) using invariable 2 to 100% gradient of CH<sub>3</sub>OH/H<sub>2</sub>O with 0.05% TFA for 90 min, 222 nm detection, and flow rate of 7 mL min<sup>–1</sup>.</div><div class="NLM_p">Compounds with high activity against May1 (<b>3a</b>, <b>3b</b>, <b>4a</b>, <b>4b</b>, <b>13a</b>, and <b>13b</b>) were selected for thorough characterization by <sup>1</sup>H and <sup>13</sup>C NMR. Their LC traces are included in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">Supporting Information</a> Figure S6. The following compounds were under 95% purity: <b>2a</b>, <b>5a</b>, <b>6a</b>, <b>8a</b>, <b>9a</b>, <b>11a</b>, <b>12a</b>, <b>1b</b>, <b>2b</b>, <b>4b</b>, <b>6b</b>, <b>7b</b>, <b>8b</b>, <b>10b</b>, <b>11b</b>, and <b>12b</b>. Compounds <b>1a</b>, <b>3a</b>, <b>4a</b>, <b>7a</b>, <b>10a</b>, <b>13a</b>, <b>3b</b>, and <b>13b</b> were ≥95% pure.</div><div class="NLM_p">Compound <b>1a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-1-oxobutan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 557.26 [M + H]+, HRMS <i>m</i>/<i>z</i>: 579.24182 [M + Na]<sup>+</sup> calcd C<sub>28</sub>H<sub>36</sub>O<sub>8</sub>N<sub>4</sub>Na<sup>+</sup>, 579.24254. HPLC: rt 6.47 min (95.1%).</div><div class="NLM_p">Compound <b>2a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-1-oxopentan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 571.28 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 593.25778 [M + Na]<sup>+</sup> calcd C<sub>29</sub>H<sub>38</sub>O<sub>8</sub>N<sub>4</sub>Na<sup>+</sup>, 593.25819. HPLC: rt 6.34 min (93.6%).</div><div class="NLM_p">Compound <b>3a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-3-methyl-1-oxobutan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 571.28 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.09 (s, 1H), 8.04 (d, <i>J</i> = 7.8 Hz, 1H), 7.62 (d, <i>J</i> = 8.9 Hz, 1H), 7.37 (s, 1H), 7.35–7.24 (m, 5H), 7.24 (s, 1H), 7.23–7.15 (m, 5H), 7.12 (d, <i>J</i> = 9.2 Hz, 1H), 5.00 (d, <i>J</i> = 13.0 Hz, 1H), 4.90 (d, <i>J</i> = 12.8 Hz, 1H), 4.29 (td, <i>J</i> = 8.3, 5.3 Hz, 1H), 4.09 (dd, <i>J</i> = 8.9, 6.4 Hz, 1H), 3.96–3.92 (m, 1H), 3.77–3.65 (m, 2H), 2.87 (dd, <i>J</i> = 13.7, 4.5 Hz, 1H), 2.60 (dd, <i>J</i> = 13.6, 10.0 Hz, 1H), 2.34–2.17 (m, 4H), 2.00–1.82 (m, 2H), 1.79–1.67 (m, 1H), 0.82 (dd, <i>J</i> = 13.1, 6.7 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.55, 173.24, 171.64, 171.60, 156.43, 139.94, 137.71, 129.56 (2C), 128.74 (2C), 128.52 (2C), 128.02, 127.69 (2C), 126.33, 69.32, 65.43, 57.79, 57.38, 52.58, 49.06, 36.11, 30.87, 30.70, 27.71, 19.75, 18.32. HRMS <i>m</i>/<i>z</i>: 571.27579 [M + H]<sup>+</sup> calcd C<sub>29</sub>H<sub>39</sub>O<sub>8</sub>N<sub>4</sub><sup>+</sup>, 571.27624. HPLC: rt 6.92 min (98.9%).</div><div class="NLM_p">Compound <b>4a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-1-oxohexan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 585.31 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.05 (s, 1H), 8.09 (d, <i>J</i> = 7.0 Hz, 1H), 7.83 (d, <i>J</i> = 8.1 Hz, 1H), 7.32 (s, 1H), 7.32–7.23 (m, 5H), 7.23 (s, 1H), 7.23–7.16 (m, 5H), 7.14 (d, <i>J</i> = 9.3 Hz, 1H), 5.02 (d, <i>J</i> = 12.9 Hz, 1H), 4.88 (d, <i>J</i> = 12.9 Hz, 1H), 4.16 (td, <i>J</i> = 8.3, 5.4 Hz, 1H), 4.09 (dt, <i>J</i> = 8.8, 4.5 Hz, 1H), 3.99–3.90 (m, 1H), 3.77–3.56 (m, 2H), 2.83 (dd, <i>J</i> = 13.7, 4.5 Hz, 1H), 2.64 (dd, <i>J</i> = 13.8, 10.3 Hz, 1H), 2.34–2.22 (m, 4H), 1.93–1.70 (m, 2H), 1.73–1.57 (m, 1H), 1.49–1.38 (m, 1H), 1.26–1.12 (m, 4H), 0.79 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.52, 174.15, 171.88, 171.59, 156.57, 139.86, 137.69, 129.59 (2C), 128.73 (2C), 128.50 (2C), 128.00, 127.62 (2C), 126.30, 69.48, 65.44, 57.08, 53.04, 52.84, 49.07, 36.56, 31.58, 30.57, 28.02, 27.53, 22.20, 14.31. HRMS <i>m</i>/<i>z</i>: 585.29153 [M + H]<sup>+</sup> calcd C<sub>30</sub>H<sub>41</sub>O<sub>8</sub>N<sub>4</sub><sup>+</sup>, 585.29189. HPLC: rt 7.02 min (96.2%).</div><div class="NLM_p">Compound <b>5a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-4-methyl-1-oxopentan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 585.30 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 607.27309 [M + Na]<sup>+</sup> calcd C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>8</sub>Na<sup>+</sup>, 607.27384. HPLC: rt 6.79 min (94.4%).</div><div class="NLM_p">Compound <b>6a</b> ((<i>S</i>)-5-(((2<i>S</i>,3<i>S</i>)-1-amino-3-methyl-1-oxopentan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 585.30 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 607.27311 [M + Na]<sup>+</sup> calcd C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>8</sub>Na<sup>+</sup>, 607.27384. HPLC: rt 6.81 min (92.2%).</div><div class="NLM_p">Compound <b>7a</b> ((<i>S</i>)-5-(((2<i>S</i>,3<i>R</i>)-1-amino-3-hydroxy-1-oxobutan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 573.26 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 595.23689 [M + Na]<sup>+</sup> calcd C<sub>28</sub>H<sub>36</sub>N<sub>4</sub>O<sub>9</sub>Na<sup>+</sup>, 595.23745. HPLC: rt 6.36 min (99.5%).</div><div class="NLM_p">Compound <b>8a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-4-(methylthio)-1-oxobutan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 603.25 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 603.24782 [M + H]<sup>+</sup> calcd C<sub>29</sub>H<sub>39</sub>N<sub>4</sub>O<sub>8</sub>S<sup>+</sup>, 603.24831. HPLC: rt 6.63 min (71.6%).</div><div class="NLM_p">Compound <b>9a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-3-(1H-indol-3-yl)-1-oxopropan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 658.29 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 680.26867 [M + Na]<sup>+</sup> calcd C<sub>35</sub>H<sub>39</sub>N<sub>5</sub>O<sub>8</sub>Na<sup>+</sup>, 680.26908. HPLC: rt 6.86 min (74.1%).</div><div class="NLM_p">Compound <b>10a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-3-cyclohexyl-1-oxopropan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 625.33 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 647.30450 [M + Na]<sup>+</sup> calcd C<sub>33</sub>H<sub>44</sub>N<sub>4</sub>O<sub>8</sub>Na<sup>+</sup>, 647.30514. HPLC: rt 7.12 min (95.1%).</div><div class="NLM_p">Compound <b>11a</b> ((<i>S</i>)-5-(((<i>S</i>)-2-amino-2-oxo-1-phenylethyl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 605.27 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 627.24160 [M + Na]<sup>+</sup> calcd C<sub>32</sub>H<sub>36</sub>N<sub>4</sub>O<sub>8</sub>Na<sup>+</sup>, 627.24254. HPLC: rt 6.79 min (93.1%).</div><div class="NLM_p">Compound <b>12a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-1-oxo-3-phenylpropan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 619.28 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 641.25726 [M + Na]<sup>+</sup> calcd C<sub>33</sub>H<sub>38</sub>N<sub>4</sub>O<sub>8</sub>Na<sup>+</sup>, 641.25819. HPLC: rt 6.81 min (94.8%).</div><div class="NLM_p">Compound <b>13a</b> ((<i>S</i>)-5-(((<i>S</i>)-1-amino-1-oxo-4-phenylbutan-2-yl)amino)-4-((3<i>S</i>,4<i>S</i>)-4-(((benzyloxy)carbonyl)amino)-3-hydroxy-5-phenylpentanamido)-5-oxopentanoic acid) analytical data MS <i>m</i>/<i>z</i>: 633.32 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.10 (s, 1H), 8.16 (d, <i>J</i> = 6.7 Hz, 1H), 8.00 (d, <i>J</i> = 8.0 Hz, 1H), 7.31 (s, 1H), 7.30–7.22 (m, 5H), 7.22 (s, 1H), 7.22–7.11 (m, 10H), 7.10 (d, <i>J</i> = 6.4 Hz, 1H), 5.02 (d, <i>J</i> = 12.1 Hz, 1H), 4.87 (d, <i>J</i> = 12.9 Hz, 1H), 4.21–4.14 (m, 2H), 4.00–3.93 (m, 1H), 3.77–3.66 (m, 2H), 2.83 (dd, <i>J</i> = 13.7, 4.4 Hz, 1H), 2.68–2.60 (m, 1H), 2.48–2.40 (m, 2H), 2.36–2.26 (m, 4H), 1.97–1.86 (m, 2H), 1.86–1.70 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.51, 173.97, 172.04, 171.80, 156.63, 141.77, 139.83, 137.65, 129.60 (2C), 128.72 (2C + 2C, overlap), 128.49 (2C), 127.99 (2C), 127.61 (2C + 1C, overlap), 126.30, 126.23, 69.58, 65.48, 57.01, 53.32, 52.57, 49.06, 36.70, 33.46, 31.93, 30.62, 27.38. HRMS <i>m</i>/<i>z</i>: 633.29153 [M + H]<sup>+</sup> calcd C<sub>34</sub>H<sub>41</sub>O<sub>8</sub>N<sub>4</sub><sup>+</sup>, 633.29189. HPLC: rt 7.25 min (98.3%).</div><div class="NLM_p">Compound <b>1b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-1-amino-1-oxobutan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 556.28 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 578.25753 [M + Na]<sup>+</sup> calcd C<sub>28</sub>H<sub>37</sub>N<sub>5</sub>O<sub>7</sub>Na<sup>+</sup>, 578.25852. HPLC: rt 6.34 min (93.6%).</div><div class="NLM_p">Compound <b>2b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-1-amino-1-oxopentan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 570.30 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 592.27344 [M + Na]<sup>+</sup> calcd C<sub>29</sub>H<sub>39</sub>N<sub>5</sub>O<sub>7</sub>Na<sup>+</sup>, 592.27417. HPLC: rt 6.49 min (88.1%).</div><div class="NLM_p">Compound <b>3b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-1-amino-3-methyl-1-oxobutan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 570.31 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.06 (d, <i>J</i> = 7.7 Hz, 1H), 7.62 (d, <i>J</i> = 8.9 Hz, 1H), 7.36 (s, 1H), 7.35–7.24 (m, 5H), 7.24 (s, 1H), 7.24–7.16 (m, 5H), 7.13 (d, <i>J</i> = 9.2 Hz, 1H), 7.04 (s, 1H), 6.76 (s, 1H), 5.01 (d, <i>J</i> = 12.9 Hz, 1H), 4.90 (d, <i>J</i> = 12.8 Hz, 1H), 4.25 (td, <i>J</i> = 8.3, 5.5 Hz, 1H), 4.09 (dd, <i>J</i> = 8.9, 6.3 Hz, 1H), 3.99–3.88 (m, 2H), 3.72 (t, <i>J</i> = 3.7 Hz, 1H), 2.87 (dd, <i>J</i> = 13.6, 4.4 Hz, 1H), 2.60 (dd, <i>J</i> = 13.7, 10.0 Hz, 1H), 2.34–2.17 (m, 2H), 2.17–2.02 (m, 2H), 2.01–1.82 (m, 2H), 1.77–1.65 (m, 1H), 0.82 (dd, <i>J</i> = 13.2, 6.8 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.33, 173.28, 171.80, 171.56, 156.44, 139.95, 137.71, 129.56 (2C), 128.74 (2C), 128.52 (2C), 128.02, 127.69 (2C), 126.33, 69.34, 65.43, 57.74, 57.37, 52.97, 49.06, 36.10, 32.06, 30.90, 28.29, 19.76, 18.30. HRMS <i>m</i>/<i>z</i>: 570.29189 [M + H]<sup>+</sup> calcd C<sub>29</sub>H<sub>40</sub>O<sub>7</sub>N<sub>5</sub><sup>+</sup>, 570.29223. HPLC: rt 6.73 min (95.0%).</div><div class="NLM_p">Compound <b>4b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-1-amino-1-oxohexan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 584.33 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.14 (d, <i>J</i> = 6.9 Hz, 1H), 7.82 (d, <i>J</i> = 8.1 Hz, 1H), 7.32 (s, 1H), 7.32–7.23 (m, 5H), 7.23 (s, 1H), 7.23–7.16 (m, 5H), 7.14 (d, <i>J</i> = 9.3 Hz, 1H), 7.03 (s, 1H), 6.78 (s, 1H), 5.02 (d, <i>J</i> = 12.9 Hz, 1H), 4.88 (d, <i>J</i> = 13.0 Hz, 1H), 4.17–4.05 (m, 2H), 3.96 (t, <i>J</i> = 6.8 Hz, 1H), 3.77–3.67 (m, 2H), 2.83 (dd, <i>J</i> = 13.7, 4.4 Hz, 1H), 2.65 (dd, <i>J</i> = 13.8, 10.3 Hz, 1H), 2.32–2.24 (m, 2H), 2.20–2.04 (m, 2H), 1.92–1.71 (m, 2H), 1.69–1.59 (m, 1H), 1.49–1.38 (m, 1H), 1.25–1.13 (m, 4H), 0.79 (t, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.33, 174.22, 171.84, 171.77, 156.58, 139.85, 137.69, 129.60 (2C), 128.73 (2C), 128.50 (2C), 128.00, 127.62 (2C), 126.30, 69.50, 65.44, 57.05, 53.47, 52.85, 49.07, 36.56, 31.98, 31.59, 28.10, 28.02, 22.21, 14.31. HRMS <i>m</i>/<i>z</i>: 584.30756 [M + H]<sup>+</sup> calcd C<sub>30</sub>H<sub>42</sub>O<sub>7</sub>N<sub>5</sub><sup>+</sup>, 584.30788. HPLC: rt 6.88 min (92.1%).</div><div class="NLM_p">Compound <b>6b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((2<i>S</i>,3<i>S</i>)-1-amino-3-methyl-1-oxopentan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 584.31 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 606.28878 [M + Na]<sup>+</sup> calcd C<sub>30</sub>H<sub>41</sub>N<sub>5</sub>O<sub>7</sub>Na<sup>+</sup>, 606.28982. HPLC: rt 6.65 min (88.2%).</div><div class="NLM_p">Compound <b>7b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((2<i>S</i>,3<i>R</i>)-1-amino-3-hydroxy-1-oxobutan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 572.28 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 594.25243 [M + Na]<sup>+</sup> calcd C<sub>28</sub>H<sub>37</sub>N<sub>5</sub>O<sub>8</sub>Na<sup>+</sup>, 594.25343. HPLC: rt 6.25 min (92.1%).</div><div class="NLM_p">Compound <b>8b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-1-amino-4-(methylthio)-1-oxobutan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 602.27 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 602.26362 [M + H]<sup>+</sup> calcd C<sub>29</sub>H<sub>40</sub>N<sub>5</sub>O<sub>7</sub>S<sup>+</sup>, 602.26430. HPLC: rt 6.51 min (82.2%).</div><div class="NLM_p">Compound <b>10b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-1-amino-3-cyclohexyl-1-oxopropan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 624.35 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 646.32006 [M + Na]<sup>+</sup> calcd C<sub>33</sub>H<sub>45</sub>N<sub>5</sub>O<sub>7</sub>Na<sup>+</sup>, 646.32112. HPLC: rt 7.00 min (84.2%).</div><div class="NLM_p">Compound <b>11b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-2-amino-2-oxo-1-phenylethyl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 604.28 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 626.25753 [M + Na]<sup>+</sup> calcd C<sub>32</sub>H<sub>37</sub>N<sub>5</sub>O<sub>7</sub>Na<sup>+</sup>, 626.25852. HPLC: rt 6.64 min (87.4%).</div><div class="NLM_p">Compound <b>12b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-1-amino-1-oxo-3-phenylpropan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 618.30 [M + H]<sup>+</sup>, HRMS <i>m</i>/<i>z</i>: 640.27388 [M + Na]<sup>+</sup> calcd C<sub>33</sub>H<sub>39</sub>N<sub>5</sub>O<sub>7</sub>Na<sup>+</sup>, 640.27417. HPLC: rt 6.71 min (85.3%).</div><div class="NLM_p last">Compound <b>13b</b> (benzyl ((2<i>S</i>,3<i>S</i>)-5-(((<i>S</i>)-5-amino-1-(((<i>S</i>)-1-amino-1-oxo-4-phenylbutan-2-yl)amino)-1,5-dioxopentan-2-yl)amino)-3-hydroxy-5-oxo-1-phenylpentan-2-yl)carbamate) analytical data MS <i>m</i>/<i>z</i>: 632.33 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.22 (d, <i>J</i> = 6.6 Hz, 1H), 8.00 (d, <i>J</i> = 8.1 Hz, 1H), 7.31 (s, 1H), 7.30–7.23 (m, 5H), 7.22 (s, 1H), 7.22–7.11 (m, 10H), 7.11 (s, 1H), 7.10 (d, <i>J</i> = 6.7 Hz, 1H), 6.80 (s, 1H), 4.99 (d, <i>J</i> = 12.9 Hz, 1H), 4.87 (d, <i>J</i> = 12.9 Hz, 1H), 4.18–4.05 (m, 2H), 3.98 (td, <i>J</i> = 6.7, 2.6 Hz, 1H), 3.76–3.68 (m, 2H), 2.83 (dd, <i>J</i> = 13.7, 4.4 Hz, 1H), 2.66 (dd, <i>J</i> = 13.6, 10.3 Hz, 1H), 2.47–2.39 (m, 2H), 2.32 (d, <i>J</i> = 6.6 Hz, 2H), 2.24–2.09 (m, 2H), 1.99–1.85 (m, 2H), 1.85–1.68 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.33, 174.05, 172.01, 171.98, 156.65, 141.79, 139.82, 137.65, 129.61 (2C), 128.72 (2C + 2C, overlap), 128.49 (2C), 128.00 (2C), 127.61 (2C + 1C, overlap), 126.31, 126.22, 69.60, 65.49, 56.98, 53.78, 52.56, 49.07, 36.72, 33.47, 31.98, 31.93, 27.93. HRMS <i>m</i>/<i>z</i>: 632.30772 [M + H]<sup>+</sup> calcd C<sub>34</sub>H<sub>42</sub>O<sub>7</sub>N<sub>5</sub><sup>+</sup>, 632.30788. HPLC: rt 7.07 min (96.7%).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02177" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27461" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27461" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02177?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02177</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Prodomain processing, structure determination, and cytotoxicity assays MayI SI (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">MayI SMILES (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf">jm0c02177_si_001.pdf (1.26 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_002.csv">jm0c02177_si_002.csv (3.02 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB code for MayI with bound PepA is 6R6A. MayI in apo-enzyme form is under 6R5H.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02177" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43605" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43605" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Konvalinka</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Faculty of Science, Charles
UniversityHlavova 8, Prague 2 12843, Czech Republic</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0695-9266" title="Orcid link">https://orcid.org/0000-0003-0695-9266</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#97fcf8f9e1f6fbd7e2f8f4fff5b9f4f6e4b9f4ed"><span class="__cf_email__" data-cfemail="ed8682839b8c81ad98828e858fc38e8c9ec38e97">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robin Kryštůfek</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Physical and Macromolecular Chemistry, Faculty of Science, Charles UniversityHlavova 8, Prague 2 12843, Czech Republic</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4096-1159" title="Orcid link">https://orcid.org/0000-0003-4096-1159</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pavel Šácha</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Faculty of Science, Charles
UniversityHlavova 8, Prague 2 12843, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jana Starková</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiří Brynda</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Molecular Genetics of the Czech Academy of Sciences, Vídeňská 1083, Prague 4 14220, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Hradilek</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eva Tloušt’ová</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justyna Grzymska</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology and Bioimaging, Wroclaw
University of Science and Technology, Wybrzeze Wyspianskiego 27, Wroclaw 50-370, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wioletta Rut</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology and Bioimaging, Wroclaw
University of Science and Technology, Wybrzeze Wyspianskiego 27, Wroclaw 50-370, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael J. Boucher</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry & Biophysics, University
of California, San Francisco, UCSF Genentech Hall, 600 16th St Rm N374, San
Francisco, California 94158, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcin Drąg</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology and Bioimaging, Wroclaw
University of Science and Technology, Wybrzeze Wyspianskiego 27, Wroclaw 50-370, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pavel Majer</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Miroslav Hájek</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pavlína Řezáčová</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry of the Czech Academy of Sciences, Flemingovo n. 2, Prague 6 16610, Czech Republic</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Molecular Genetics of the Czech Academy of Sciences, Vídeňská 1083, Prague 4 14220, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hiten D. Madhani</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry & Biophysics, University
of California, San Francisco, UCSF Genentech Hall, 600 16th St Rm N374, San
Francisco, California 94158, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Chan-Zuckerberg
Biohub, 499 Illinois
Street, San Francisco, California 94158, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charles S. Craik</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, University
of California San Francisco, UCSF Genentech Hall, 600 16th St Rm S512, San
Francisco, California 94158, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7704-9185" title="Orcid link">https://orcid.org/0000-0001-7704-9185</a></span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>The study was supported by the EU Regional Development Fund (OP RDE) Project Chemical biology for drugging undruggable targets (J.K., ChemBioDrug, CZ.02.1.01/0.0/0.0/16_019/0000729) and by National Institutes of Health grants T32HL007185 (M.J.B.), F32AI152270 (M.J.B.), R01AI100272 (H.D.M.), and P50GM082250 (C.S.C.). H.D.M. is an Investigator of the Chan-Zuckerberg Biohub.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The authors will release the atomic coordinates and experimental data upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23757" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23757" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank Karolína Šrámková for technical support throughout the duration of the study.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AMC</td><td class="NLM_def"><p class="first last">7-amino-4-methyl-3-coumarinylacetic acid</p></td></tr><tr><td class="NLM_term">ACC</td><td class="NLM_def"><p class="first last">7-amino-4-carbamoylmethylcoumarin</p></td></tr><tr><td class="NLM_term">K(DNP)</td><td class="NLM_def"><p class="first last"><i>N</i>ε-((2,4-dinitrophenyl)-<span class="smallcaps smallerCapital">l</span>-lysine)</p></td></tr><tr><td class="NLM_term">HIV-1 Pr</td><td class="NLM_def"><p class="first last">HIV-1 protease</p></td></tr><tr><td class="NLM_term">ART</td><td class="NLM_def"><p class="first last">antiretroviral therapy</p></td></tr><tr><td class="NLM_term">Pns</td><td class="NLM_def"><p class="first last">phenylnorstatine</p></td></tr><tr><td class="NLM_term">Apb</td><td class="NLM_def"><p class="first last">3-amino-4-phenylbutanoic acid</p></td></tr><tr><td class="NLM_term">Pst</td><td class="NLM_def"><p class="first last">phenylstatine</p></td></tr><tr><td class="NLM_term">HEA</td><td class="NLM_def"><p class="first last">hydroxethylamine mimetic linkage</p></td></tr><tr><td class="NLM_term">Abu</td><td class="NLM_def"><p class="first last">2-aminobutyric acid</p></td></tr><tr><td class="NLM_term">Nva</td><td class="NLM_def"><p class="first last">norvaline</p></td></tr><tr><td class="NLM_term">Cha</td><td class="NLM_def"><p class="first last">3-cyclohexylalanine</p></td></tr><tr><td class="NLM_term">Phg</td><td class="NLM_def"><p class="first last">phenylglycine</p></td></tr><tr><td class="NLM_term">Hph</td><td class="NLM_def"><p class="first last">homophenylalanine</p></td></tr><tr><td class="NLM_term">Aib</td><td class="NLM_def"><p class="first last">2-aminoisobutyric acid</p></td></tr><tr><td class="NLM_term">Tic</td><td class="NLM_def"><p class="first last">1,2,3,4-tetrahydroisoquinoline-3-carboxylic acid</p></td></tr><tr><td class="NLM_term">Pal</td><td class="NLM_def"><p class="first last">pyridylalanine</p></td></tr><tr><td class="NLM_term">Thi</td><td class="NLM_def"><p class="first last">β-thienylalanine</p></td></tr><tr><td class="NLM_term">Hyp</td><td class="NLM_def"><p class="first last">4-hydroxyproline</p></td></tr><tr><td class="NLM_term">hSer</td><td class="NLM_def"><p class="first last">homoserine</p></td></tr><tr><td class="NLM_term">PepA</td><td class="NLM_def"><p class="first last">pepstatin A</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07573" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07573" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 81 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon-Chung, K. J.</span>; <span class="NLM_string-name">Boekhout, T.</span>; <span class="NLM_string-name">Wickes, B. L.</span>; <span class="NLM_string-name">Fell, J. W.</span></span> <span> </span><span class="NLM_article-title">Systematics of the Genus Cryptococcus and its Type Species C. Neoformans</span>.  <i>Cryptococcus</i>; <span class="NLM_publisher-name">ASM Press</span>, <span class="NLM_year">2011</span>; Vol.  <span class="NLM_volume">1</span>; pp  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">15</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=3-15&author=K.+J.+Kwon-Chung&author=T.+Boekhout&author=B.+L.+Wickes&author=J.+W.+Fell&title=Cryptococcus"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKwon-Chung%26aufirst%3DK.%2BJ.%26atitle%3DSystematics%2520of%2520the%2520Genus%2520Cryptococcus%2520and%2520its%2520Type%2520Species%2520C.%2520Neoformans%26btitle%3DCryptococcus%26pub%3DASM%2520Press%26date%3D2011%26volume%3D1%26spage%3D3%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaworowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, S. R.</span></span> <span> </span><span class="NLM_article-title">HIV and Co-infections</span>. <i>Immunol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>254</i></span>,  <span class="NLM_fpage">114</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1111/imr.12063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1111%2Fimr.12063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=23772618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFyit7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=254&publication_year=2013&pages=114-142&author=C.+C.+Changauthor=M.+Craneauthor=J.+Zhouauthor=M.+Minaauthor=J.+J.+Postauthor=B.+A.+Cameronauthor=A.+R.+Lloydauthor=A.+Jaworowskiauthor=M.+A.+Frenchauthor=S.+R.+Lewin&title=HIV+and+Co-infections&doi=10.1111%2Fimr.12063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">HIV and co-infections</span></div><div class="casAuthors">Chang, Christina C.; Crane, Megan; Zhou, Jingling; Mina, Michael; Post, Jeffrey J.; Cameron, Barbara A.; Lloyd, Andrew R.; Jaworowski, Anthony; French, Martyn A.; Lewin, Sharon R.</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">254</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">114-142, 29 pp.</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Despite significant redns. in morbidity and mortality secondary to availability of effective combination anti-retroviral therapy (cART), human immunodeficiency virus (HIV) infection still accounts for 1.5 million deaths annually.  The majority of deaths occur in sub-Saharan Africa where rates of opportunistic co-infections are disproportionately high.  In this review, we discuss the immunopathogenesis of five common infections that cause significant morbidity in HIV-infected patients globally.  These include co-infection with Mycobacterium tuberculosis, Cryptococcus neoformans, hepatitis B virus, hepatitis C virus, and Plasmodium falciparum.  Specifically, we review the natural history of each co-infection in the setting of HIV, the specific immune defects induced by HIV, the effects of cART on the immune response to the co-infection, the pathogenesis of immune restoration disease (IRD) assocd. with each infection, and advances in the areas of prevention of each co-infection via vaccination.  Finally, we discuss the opportunities and gaps in knowledge for future research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqGCu0bX3blrVg90H21EOLACvtfcHk0lg3F6D9SuuzKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFyit7zP&md5=609076465668935e01c1e3172db10462</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fimr.12063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fimr.12063%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.%2BC.%26aulast%3DCrane%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DMina%26aufirst%3DM.%26aulast%3DPost%26aufirst%3DJ.%2BJ.%26aulast%3DCameron%26aufirst%3DB.%2BA.%26aulast%3DLloyd%26aufirst%3DA.%2BR.%26aulast%3DJaworowski%26aufirst%3DA.%26aulast%3DFrench%26aufirst%3DM.%2BA.%26aulast%3DLewin%26aufirst%3DS.%2BR.%26atitle%3DHIV%2520and%2520Co-infections%26jtitle%3DImmunol.%2520Rev.%26date%3D2013%26volume%3D254%26spage%3D114%26epage%3D142%26doi%3D10.1111%2Fimr.12063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jarvis, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, T. S.</span></span> <span> </span><span class="NLM_article-title">HIV-associated Cryptococcal Meningitis</span>. <i>AIDS</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2119</span>– <span class="NLM_lpage">2129</span>, <span class="refDoi"> DOI: 10.1097/qad.0b013e3282a4a64d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1097%2FQAD.0b013e3282a4a64d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=18090038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD2sjlsV2ktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=2119-2129&author=J.+N.+Jarvisauthor=T.+S.+Harrison&title=HIV-associated+Cryptococcal+Meningitis&doi=10.1097%2Fqad.0b013e3282a4a64d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-associated cryptococcal meningitis</span></div><div class="casAuthors">Jarvis Joseph N; Harrison Thomas S</div><div class="citationInfo"><span class="NLM_cas:title">AIDS (London, England)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2119-29</span>
        ISSN:<span class="NLM_cas:issn">0269-9370</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkLKSquk7kKnthwrmtH4OBfW6udTcc2eZetz6BIZ6w07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjlsV2ktw%253D%253D&md5=e536a9b693287f84a59abb1348999f3b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1097%2FQAD.0b013e3282a4a64d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQAD.0b013e3282a4a64d%26sid%3Dliteratum%253Aachs%26aulast%3DJarvis%26aufirst%3DJ.%2BN.%26aulast%3DHarrison%26aufirst%3DT.%2BS.%26atitle%3DHIV-associated%2520Cryptococcal%2520Meningitis%26jtitle%3DAIDS%26date%3D2007%26volume%3D21%26spage%3D2119%26epage%3D2129%26doi%3D10.1097%2Fqad.0b013e3282a4a64d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsueh, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geunes-Boyer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, J.</span></span> <span> </span><span class="NLM_article-title">Spores as Infectious Propagules of Cryptococcus Neoformans</span>. <i>Infect. Immun.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">4345</span>– <span class="NLM_lpage">4355</span>, <span class="refDoi"> DOI: 10.1128/iai.00542-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FIAI.00542-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=19620339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Ghu7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2009&pages=4345-4355&author=R.+Velagapudiauthor=Y.-P.+Hsuehauthor=S.+Geunes-Boyerauthor=J.+R.+Wrightauthor=J.+Heitman&title=Spores+as+Infectious+Propagules+of+Cryptococcus+Neoformans&doi=10.1128%2Fiai.00542-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Spores as infectious propagules of Cryptococcus neoformans</span></div><div class="casAuthors">Velagapudi, Rajesh; Hsueh, Yen-Ping; Geunes-Boyer, Scarlett; Wright, Jo Rae; Heitman, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Infection and Immunity</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4345-4355</span>CODEN:
                <span class="NLM_cas:coden">INFIBR</span>;
        ISSN:<span class="NLM_cas:issn">0019-9567</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cryptococcus neoformans and Cryptococcus gattii are closely related pathogenic fungi that cause pneumonia and meningitis in both immunocompromised and immunocompetent hosts and are a significant global infectious disease risk.  Both species are found in the environment and are acquired via inhalation, leading to an initial pulmonary infection.  The infectious propagule is unknown but is hypothesized to be small desiccated yeast cells or spores produced by sexual reprodn. (opposite- or same-sex mating).  Here we characterize the morphol., germination properties, and virulence of spores.  A comparative morphol. anal. of hyphae and spores produced by opposite-sex mating, same-sex mating, and self-fertile diploid strains was conducted by SEM, yielding insight into hyphal/basidial morphol. and spore size, structure, and surface properties.  Spores isolated by microdissection were found to readily germinate even on water agarose medium.  Thus, nutritional signals do not appear to be required to stimulate spore germination, and as-yet-unknown environmental factors may normally constrain germination in nature.  As few as 500 CFU of a spore-enriched infectious inoculum (∼95% spores) of serotype A C. neoformans var. grubii were fully virulent (100% lethal infection) in both a murine inhalation virulence model and the invertebrate model host Galleria mellonella.  In contrast to a previous report on C. neoformans var. neoformans, spores of C. neoformans var. grubii were not more infectious than yeast cells.  Mol. anal. of isolates recovered from tissues of infected mice (lung, spleen, and brain) provides evidence for infection and dissemination by recombinant spore products.  These studies provide a detailed morphol. and physiol. anal. of the spore and document that spores can serve as infectious propagules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSnUo4GlO8UbVg90H21EOLACvtfcHk0lgQrApxBnu4zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Ghu7rN&md5=37f0b4cd4af6ad76b307773e7b0af7bc</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FIAI.00542-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FIAI.00542-09%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DR.%26aulast%3DHsueh%26aufirst%3DY.-P.%26aulast%3DGeunes-Boyer%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DJ.%2BR.%26aulast%3DHeitman%26aufirst%3DJ.%26atitle%3DSpores%2520as%2520Infectious%2520Propagules%2520of%2520Cryptococcus%2520Neoformans%26jtitle%3DInfect.%2520Immun.%26date%3D2009%26volume%3D77%26spage%3D4345%26epage%3D4355%26doi%3D10.1128%2Fiai.00542-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, H. I.</span></span> <span> </span><span class="NLM_article-title">Not Your “Typical Patient”: Cryptococcal Meningitis in an Immunocompetent Patient</span>. <i>J. Neurosci. Nurs.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">144</span>– <span class="NLM_lpage">148</span>, <span class="refDoi"> DOI: 10.1097/01376517-200506000-00005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1097%2F01376517-200506000-00005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=16001819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD2MzkvFyguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2005&pages=144-148&author=H.+I.+Thompson&title=Not+Your+%E2%80%9CTypical+Patient%E2%80%9D%3A+Cryptococcal+Meningitis+in+an+Immunocompetent+Patient&doi=10.1097%2F01376517-200506000-00005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Not your "typical patient": cryptococcal meningitis in an immunocompetent patient</span></div><div class="casAuthors">Thompson Hilaire J</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of neuroscience nursing : journal of the American Association of Neuroscience Nurses</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">144-8</span>
        ISSN:<span class="NLM_cas:issn">0888-0395</span>.
    </div><div class="casAbstract">Meningitis, when caused by the fungal mycoses Cryptococcus neoformans, is normally seen in immuno-compromised hosts.  However, immunocompetent patients are also susceptible to cryptococcal meningitis (CM).  In patients with an intact immune system, CM usually presents with the typical signs and symptoms of meningitis: fever, stiff neck, and headache.  Major implications for the primary and advanced practice nursing plans of care for CM patients include a thorough history and physical exam, early diagnosis and treatment, and an individualized plan of care focused on minimizing sequelae and side effects of treatment and maximizing functional recovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQLU3fjt9OUP36oPH0mYWL1fW6udTcc2eZetz6BIZ6w07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzkvFyguw%253D%253D&md5=9ffdbce31dc09be1b7c873bb041b85f9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1097%2F01376517-200506000-00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F01376517-200506000-00005%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DH.%2BI.%26atitle%3DNot%2520Your%2520%25E2%2580%259CTypical%2520Patient%25E2%2580%259D%253A%2520Cryptococcal%2520Meningitis%2520in%2520an%2520Immunocompetent%2520Patient%26jtitle%3DJ.%2520Neurosci.%2520Nurs.%26date%3D2005%26volume%3D37%26spage%3D144%26epage%3D148%26doi%3D10.1097%2F01376517-200506000-00005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lui, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tso, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockram, C. S.</span></span> <span> </span><span class="NLM_article-title">Cryptococcosis in Apparently Immunocompetent Patients</span>. <i>Q. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1093/qjmed/hcl014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fqjmed%2Fhcl014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2006&pages=143-151&author=G.+Luiauthor=N.+Leeauthor=M.+Ipauthor=K.+W.+Choiauthor=Y.+K.+Tsoauthor=E.+Lamauthor=S.+Chauauthor=R.+Laiauthor=C.+S.+Cockram&title=Cryptococcosis+in+Apparently+Immunocompetent+Patients&doi=10.1093%2Fqjmed%2Fhcl014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fqjmed%2Fhcl014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fqjmed%252Fhcl014%26sid%3Dliteratum%253Aachs%26aulast%3DLui%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DN.%26aulast%3DIp%26aufirst%3DM.%26aulast%3DChoi%26aufirst%3DK.%2BW.%26aulast%3DTso%26aufirst%3DY.%2BK.%26aulast%3DLam%26aufirst%3DE.%26aulast%3DChau%26aufirst%3DS.%26aulast%3DLai%26aufirst%3DR.%26aulast%3DCockram%26aufirst%3DC.%2BS.%26atitle%3DCryptococcosis%2520in%2520Apparently%2520Immunocompetent%2520Patients%26jtitle%3DQ.%2520J.%2520Med.%26date%3D2006%26volume%3D99%26spage%3D143%26epage%3D151%26doi%3D10.1093%2Fqjmed%2Fhcl014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pappas, P. G.</span></span> <span> </span><span class="NLM_article-title">Cryptococcal Infections in Non-HIV-infected Patients</span>. <i>Trans. Am. Clin. Climatol. Assoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">79</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=23874010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC3sfislamsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2013&pages=61-79&author=P.+G.+Pappas&title=Cryptococcal+Infections+in+Non-HIV-infected+Patients"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Cryptococcal infections in non-HIV-infected patients</span></div><div class="casAuthors">Pappas Peter G</div><div class="citationInfo"><span class="NLM_cas:title">Transactions of the American Clinical and Climatological Association</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">61-79</span>
        ISSN:<span class="NLM_cas:issn">0065-7778</span>.
    </div><div class="casAbstract">Infections due to Cryptococcus species occur globally and in a wide variety of hosts, ranging from those who are severely immunosuppressed to those who have phenotypically "normal" immune systems.  Approximately 1 million cases of cryptococcosis occur throughout the world, and is it estimated that there are 650,000 associated deaths annually.  Most of these cases occur among patients with advanced HIV disease, but a growing number occur among solid organ transplant recipients and others receiving exogenous immunosuppression, patients with innate and acquired immunodeficiency, and otherwise immunologically normal hosts.  Much of our recent knowledge is solely derived from clinical experience over the last 2 to 3 decades of cryptococcosis among HIV-infected patients.  However, based on recent observations, it is clear that there are substantial differences in the epidemiology, clinical features, approaches to therapy, and outcome when comparing HIV-infected to non-HIV-infected individuals who have cryptococcosis.  If one carefully examines cryptococcosis in the three largest subgroups of patients based on host immune status, specifically, those with HIV, solid organ transplant recipients, and those who are non-HIV, non-transplant (NHNT) infected persons, then one can observe very different risks for infection, varied clinical presentations, long-term complications, mortality, and approaches to therapy.  This article focuses on cryptococcosis in the non-HIV-infected patient, including a brief review of ongoing events in the Pacific Northwest of the United States and Canada relative to the outbreak of Cryptococcus gattii infections among a largely immunologically normal population, and highlights some of the key insights and questions which have emerged as a result of these important new observations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqArOx8qswLsKL0FlyGHDofW6udTcc2eY1Gk9eoELrybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfislamsQ%253D%253D&md5=44b3401b3e9707d6a071bd96a558f729</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPappas%26aufirst%3DP.%2BG.%26atitle%3DCryptococcal%2520Infections%2520in%2520Non-HIV-infected%2520Patients%26jtitle%3DTrans.%2520Am.%2520Clin.%2520Climatol.%2520Assoc.%26date%3D2013%26volume%3D124%26spage%3D61%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajasingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarvis, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govender, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiller, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denning, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loyse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulware, D. R.</span></span> <span> </span><span class="NLM_article-title">Global Burden of Disease of HIV-associated Cryptococcal Meningitis: an Updated Analysis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">873</span>– <span class="NLM_lpage">881</span>, <span class="refDoi"> DOI: 10.1016/s1473-3099(17)30243-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2FS1473-3099%2817%2930243-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28483415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC1crlt1Sjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=873-881&author=R.+Rajasinghamauthor=R.+M.+Smithauthor=B.+J.+Parkauthor=J.+N.+Jarvisauthor=N.+P.+Govenderauthor=T.+M.+Chillerauthor=D.+W.+Denningauthor=A.+Loyseauthor=D.+R.+Boulware&title=Global+Burden+of+Disease+of+HIV-associated+Cryptococcal+Meningitis%3A+an+Updated+Analysis&doi=10.1016%2Fs1473-3099%2817%2930243-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Global burden of disease of HIV-associated cryptococcal meningitis: an updated analysis</span></div><div class="casAuthors">Rajasingham Radha; Smith Rachel M; Park Benjamin J; Chiller Tom M; Jarvis Joseph N; Govender Nelesh P; Denning David W; Loyse Angela; Boulware David R</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">873-881</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Cryptococcus is the most common cause of meningitis in adults living with HIV in sub-Saharan Africa.  Global burden estimates are crucial to guide prevention strategies and to determine treatment needs, and we aimed to provide an updated estimate of global incidence of HIV-associated cryptococcal disease.  METHODS:  We used 2014 Joint UN Programme on HIV and AIDS estimates of adults (aged >15 years) with HIV and antiretroviral therapy (ART) coverage.  Estimates of CD4 less than 100 cells per μL, virological failure incidence, and loss to follow-up were from published multinational cohorts in low-income and middle-income countries.  We calculated those at risk for cryptococcal infection, specifically those with CD4 less than 100 cells/μL not on ART, and those with CD4 less than 100 cells per μL on ART but lost to follow-up or with virological failure.  Cryptococcal antigenaemia prevalence by country was derived from 46 studies globally.  Based on cryptococcal antigenaemia prevalence in each country and region, we estimated the annual numbers of people who are developing and dying from cryptococcal meningitis.  FINDINGS:  We estimated an average global cryptococcal antigenaemia prevalence of 6·0% (95% CI 5·8-6·2) among people with a CD4 cell count of less than 100 cells per μL, with 278 000 (95% CI 195 500-340 600) people positive for cryptococcal antigen globally and 223 100 (95% CI 150 600-282 400) incident cases of cryptococcal meningitis globally in 2014.  Sub-Saharan Africa accounted for 73% of the estimated cryptococcal meningitis cases in 2014 (162 500 cases [95% CI 113 600-193 900]).  Annual global deaths from cryptococcal meningitis were estimated at 181 100 (95% CI 119 400-234 300), with 135 900 (75%; [95% CI 93 900-163 900]) deaths in sub-Saharan Africa.  Globally, cryptococcal meningitis was responsible for 15% of AIDS-related deaths (95% CI 10-19).  INTERPRETATION:  Our analysis highlights the substantial ongoing burden of HIV-associated cryptococcal disease, primarily in sub-Saharan Africa.  Cryptococcal meningitis is a metric of HIV treatment programme failure; timely HIV testing and rapid linkage to care remain an urgent priority.  FUNDING:  None.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6kthTNuMfLqNc3HjJRoFAfW6udTcc2eY1Gk9eoELrybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crlt1Sjug%253D%253D&md5=85268904374cb498fc2d92ac35cd4aee</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2817%2930243-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252817%252930243-8%26sid%3Dliteratum%253Aachs%26aulast%3DRajasingham%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DR.%2BM.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DJarvis%26aufirst%3DJ.%2BN.%26aulast%3DGovender%26aufirst%3DN.%2BP.%26aulast%3DChiller%26aufirst%3DT.%2BM.%26aulast%3DDenning%26aufirst%3DD.%2BW.%26aulast%3DLoyse%26aufirst%3DA.%26aulast%3DBoulware%26aufirst%3DD.%2BR.%26atitle%3DGlobal%2520Burden%2520of%2520Disease%2520of%2520HIV-associated%2520Cryptococcal%2520Meningitis%253A%2520an%2520Updated%2520Analysis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2017%26volume%3D17%26spage%3D873%26epage%3D881%26doi%3D10.1016%2Fs1473-3099%2817%2930243-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong-James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meintjes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G. D.</span></span> <span> </span><span class="NLM_article-title">A Neglected Epidemic: Fungal Infections in HIV/AIDS</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">127</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2014.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.tim.2014.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=24530175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFehurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=120-127&author=D.+Armstrong-Jamesauthor=G.+Meintjesauthor=G.+D.+Brown&title=A+Neglected+Epidemic%3A+Fungal+Infections+in+HIV%2FAIDS&doi=10.1016%2Fj.tim.2014.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A neglected epidemic: fungal infections in HIV/AIDS</span></div><div class="casAuthors">Armstrong-James, Darius; Meintjes, Graeme; Brown, Gordon D.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">120-127</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Invasive fungal infections (IFIs) are a major cause of HIV-related mortality globally.  Despite widespread rollout of combined antiretroviral therapy, there are still up to 1 million deaths annually from IFIs, accounting for 50% of all AIDS-related death.  A historic failure to focus efforts on the IFIs that kill so many HIV patients has led to fundamental flaws in the management of advanced HIV infection.  This review, based on the EMBO AIDS-Related Mycoses Workshop in Cape Town in July 2013, summarizes the current state of the-art in AIDS-related mycoses, and the key action points required to improve outcomes from these devastating infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCS714GKJsybVg90H21EOLACvtfcHk0lgcbQba8bcybg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFehurg%253D&md5=e00bfe12c6824e9b55f3fdbfa09e3514</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2014.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2014.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DArmstrong-James%26aufirst%3DD.%26aulast%3DMeintjes%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DG.%2BD.%26atitle%3DA%2520Neglected%2520Epidemic%253A%2520Fungal%2520Infections%2520in%2520HIV%252FAIDS%26jtitle%3DTrends%2520Microbiol.%26date%3D2014%26volume%3D22%26spage%3D120%26epage%3D127%26doi%3D10.1016%2Fj.tim.2014.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">World Health Organization</span>.  <i>Guidelines for the Diagnosis, Prevention
and Management of Cryptococcal Disease in HIV-Infected Adults, Adolescents
and Children: Supplement to the 2016 Consolidated Guidelines on the
Use of Antiretroviral Drugs for Treating and Preventing HIV Infection</i>; <span class="NLM_publisher-name">World Health Organization</span>: <span class="NLM_publisher-loc">Geneva</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=World+Health+Organization&title=Guidelines+for+the+Diagnosis%2C+Prevention%0Aand+Management+of+Cryptococcal+Disease+in+HIV-Infected+Adults%2C+Adolescents%0Aand+Children%3A+Supplement+to+the+2016+Consolidated+Guidelines+on+the%0AUse+of+Antiretroviral+Drugs+for+Treating+and+Preventing+HIV+Infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DGuidelines%2520for%2520the%2520Diagnosis%252C%2520Prevention%250Aand%2520Management%2520of%2520Cryptococcal%2520Disease%2520in%2520HIV-Infected%2520Adults%252C%2520Adolescents%250Aand%2520Children%253A%2520Supplement%2520to%2520the%25202016%2520Consolidated%2520Guidelines%2520on%2520the%250AUse%2520of%2520Antiretroviral%2520Drugs%2520for%2520Treating%2520and%2520Preventing%2520HIV%2520Infection%26pub%3DWorld%2520Health%2520Organization%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span> </span><span class="NLM_article-title">Fluconazole Injection Current Drug Shortages</span>. <a href="https://www.ashp.org/Drug-Shortages/Current-Shortages/Drug-Shortage-Detail.aspx?id=318" class="extLink">https://www.ashp.org/Drug-Shortages/Current-Shortages/Drug-Shortage-Detail.aspx?id=318</a> (accessed Mar 19, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Fluconazole+Injection+Current+Drug+Shortages.+https%3A%2F%2Fwww.ashp.org%2FDrug-Shortages%2FCurrent-Shortages%2FDrug-Shortage-Detail.aspx%3Fid%3D318+%28accessed+Mar+19%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DFluconazole%2520Injection%2520Current%2520Drug%2520Shortages" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sloan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedicoat, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalloo, D. G.</span></span> <span> </span><span class="NLM_article-title">Treatment of Cryptococcal Meningitis in Resource Limited Settings</span>. <i>Curr. Opin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">463</span>, <span class="refDoi"> DOI: 10.1097/qco.0b013e32832fa214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1097%2FQCO.0b013e32832fa214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=19587589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD1MngtF2ktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2009&pages=455-463&author=D.+J.+Sloanauthor=M.+J.+Dedicoatauthor=D.+G.+Lalloo&title=Treatment+of+Cryptococcal+Meningitis+in+Resource+Limited+Settings&doi=10.1097%2Fqco.0b013e32832fa214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of cryptococcal meningitis in resource limited settings</span></div><div class="casAuthors">Sloan Derek J; Dedicoat Martin J; Lalloo David G</div><div class="citationInfo"><span class="NLM_cas:title">Current opinion in infectious diseases</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">455-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PURPOSE OF REVIEW:  Cryptococcal meningitis most commonly occurs in advanced HIV.  Although diminishing in the developed world with antiretroviral therapy (ART), it remains a major problem in resource-limited settings.  ART rollout will improve long-term HIV survival if opportunistic infections are effectively treated.  Considering cryptococcal meningitis in that context, this review addresses excess morbidity and mortality in developing countries, treatment in areas of limited drug availability and challenges posed by combined anticryptococcal and HIV therapy.  RECENT FINDINGS:  From Early Fungicidal Activity (EFA) studies, amphotericin B-flucytosine is best induction therapy but often unavailable; high dose amphotericin B monotherapy may be feasible in some settings.  Where fluconazole is the only option, higher doses are more fungicidal.  Serum cryptococcal antigen testing may identify patients at highest disease risk and primary prophylaxis is effective; the clinical role of such interventions needs to be established.  Timing of ART introduction remains controversial; early initiation risks Immune Reconstitution Disease (IRD) delays may increase mortality.  SUMMARY:  Amphotericin B based treatment is appropriate where possible.  More studies are needed to optimize fluconazole monotherapy doses.  Other research priorities include management of raised intracranial pressure, appropriate ART initiation and IRD treatment.  Studies should focus on developing countries where problems are greatest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTE316TRMI4BZHZzFRCXxVKfW6udTcc2eYHNbx9o021_bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MngtF2ktA%253D%253D&md5=5e0b9541ed4d82fadec3eca2489b8f6f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1097%2FQCO.0b013e32832fa214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQCO.0b013e32832fa214%26sid%3Dliteratum%253Aachs%26aulast%3DSloan%26aufirst%3DD.%2BJ.%26aulast%3DDedicoat%26aufirst%3DM.%2BJ.%26aulast%3DLalloo%26aufirst%3DD.%2BG.%26atitle%3DTreatment%2520of%2520Cryptococcal%2520Meningitis%2520in%2520Resource%2520Limited%2520Settings%26jtitle%3DCurr.%2520Opin.%2520Infect.%2520Dis.%26date%3D2009%26volume%3D22%26spage%3D455%26epage%3D463%26doi%3D10.1097%2Fqco.0b013e32832fa214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiederhold, N. P.</span></span> <span> </span><span class="NLM_article-title">Antifungal Resistance: Current Trends and Future Strategies to Combat</span>. <i>Infect. Drug Resist.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.2147/idr.s124918</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.2147%2FIDR.S124918" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28919789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFags7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=249-259&author=N.+P.+Wiederhold&title=Antifungal+Resistance%3A+Current+Trends+and+Future+Strategies+to+Combat&doi=10.2147%2Fidr.s124918"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Antifungal resistance: current trends and future strategies to combat</span></div><div class="casAuthors">Wiederhold, Nathan P.</div><div class="citationInfo"><span class="NLM_cas:title">Infection and Drug Resistance</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">249-259</span>CODEN:
                <span class="NLM_cas:coden">IDRNAV</span>;
        ISSN:<span class="NLM_cas:issn">1178-6973</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Antifungal resistance represents a major clin. challenge to clinicians responsible for treating invasive fungal infections due to the limited arsenal of systemically available anti-fungal agents.  In addn. current drugs may be limited by drug-drug interactions and serious adverse effects/toxicities that prevent their prolonged use or dosage escalation.  Fluconazole resistance is of particular concern in non-Candida albicans species due to the increased incidence of infections caused by these species in different geog. locations worldwide and the elevated prevalence of resistance to this commonly used azole in many institutions.  C. glabrata resistance to the echinocandins has also been documented to be rising in several US institutions, and a higher percentage of these isolates may also be azole resistant.  Azole resistance in Aspergillus fumigatus due to clin. and environmental exposure to this class of agents has also been found worldwide, and these isolates can cause invasive infections with high mortality rates.  In addn., several species of Aspergillus, and other molds, including Scedosporium and Fusarium species, have reduced susceptibility or pan-resistance to clin. available antifungals.  Various investigational antifungals are currently in preclin. or clin. development, including several of them that have the potential to overcome resistance obsd. against the azoles and the echinocandins.  These include agents that also target ergosterol and β-glucan biosynthesis, as well as compds. with novel mechanisms of action that may also overcome the limitations of currently available antifungal classes, including both resistance and adverse effects/toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJkplFm02SHbVg90H21EOLACvtfcHk0lggmBSlHWwVhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFags7Y%253D&md5=f187c81d848df8d551b5edd1f362ae13</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2147%2FIDR.S124918&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FIDR.S124918%26sid%3Dliteratum%253Aachs%26aulast%3DWiederhold%26aufirst%3DN.%2BP.%26atitle%3DAntifungal%2520Resistance%253A%2520Current%2520Trends%2520and%2520Future%2520Strategies%2520to%2520Combat%26jtitle%3DInfect.%2520Drug%2520Resist.%26date%3D2017%26volume%3D10%26spage%3D249%26epage%3D259%26doi%3D10.2147%2Fidr.s124918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caplan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowen, L. E.</span></span> <span> </span><span class="NLM_article-title">Molecular Evolution of Antifungal Drug Resistance</span>. <i>Annu. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">775</span>, <span class="refDoi"> DOI: 10.1146/annurev-micro-030117-020345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1146%2Fannurev-micro-030117-020345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28886681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKlurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=753-775&author=N.+Robbinsauthor=T.+Caplanauthor=L.+E.+Cowen&title=Molecular+Evolution+of+Antifungal+Drug+Resistance&doi=10.1146%2Fannurev-micro-030117-020345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Evolution of Antifungal Drug Resistance</span></div><div class="casAuthors">Robbins, Nicole; Caplan, Tavia; Cowen, Leah E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">753-775</span>CODEN:
                <span class="NLM_cas:coden">ARMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4227</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The fungal pathogens Candida albicans, Cryptococcus neoformans, and Aspergillus fumigatus have transitioned from a rare curiosity to a leading cause of human mortality.  The management of infections caused by these organisms is intimately dependent on the efficacy of antifungal agents; however, fungi that are resistant to these treatments are regularly isolated in the clinic, impeding our ability to control infections.  Given the significant impact fungal pathogens have on human health, it is imperative to understand the mol. mechanisms that govern antifungal drug resistance.  This review describes our current knowledge of the mechanisms by which antifungal drug resistance evolves in exptl. populations and clin. settings.  We explore current antifungal treatment options and discuss promising strategies to impede the evolution of drug resistance.  By tackling antifungal drug resistance as an evolutionary problem, there is potential to improve the utility of current treatments and accelerate the development of novel therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf_rS6B3dA4bVg90H21EOLACvtfcHk0lggmBSlHWwVhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKlurvE&md5=2eb2fbfc778bce94065a35e0bd9bea3b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev-micro-030117-020345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-micro-030117-020345%26sid%3Dliteratum%253Aachs%26aulast%3DRobbins%26aufirst%3DN.%26aulast%3DCaplan%26aufirst%3DT.%26aulast%3DCowen%26aufirst%3DL.%2BE.%26atitle%3DMolecular%2520Evolution%2520of%2520Antifungal%2520Drug%2520Resistance%26jtitle%3DAnnu.%2520Rev.%2520Microbiol.%26date%3D2017%26volume%3D71%26spage%3D753%26epage%3D775%26doi%3D10.1146%2Fannurev-micro-030117-020345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joseph-Horne, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeffler, R. S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollomon, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, S. L.</span></span> <span> </span><span class="NLM_article-title">Amphotericin B resistant isolates ofCryptococcus neoformanswithout alteration in sterol biosynthesis</span>. <i>J. Med. Vet. Mycol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1080/02681219680000381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1080%2F02681219680000381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8803806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADyaK28zps1OjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1996&pages=223-225&author=T.+Joseph-Horneauthor=R.+S.+T.+Loefflerauthor=D.+W.+Hollomonauthor=S.+L.+Kelly&title=Amphotericin+B+resistant+isolates+ofCryptococcus+neoformanswithout+alteration+in+sterol+biosynthesis&doi=10.1080%2F02681219680000381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Amphotericin B resistant isolates of Cryptococcus neoformans without alteration in sterol biosynthesis</span></div><div class="casAuthors">Joseph-Horne T; Loeffler R S; Hollomon D W; Kelly S L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medical and veterinary mycology : bi-monthly publication of the International Society for Human and Animal Mycology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">223-5</span>
        ISSN:<span class="NLM_cas:issn">0268-1218</span>.
    </div><div class="casAbstract">Amphotericin B resistant mutants of Cryptococcus neoformans were isolated accumulating mainly ergosterol.  Cross-resistance to azole antifungals was not observed.  Together with previous data this indicates that at least three categories of amphotericin B resistance can arise: sterol mutants, amphotericin B and azole cross-resistant mutants and amphotericin B resistant mutants with no azole cross-resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_b8G2NhYRWntlzo9xastyfW6udTcc2ebvqUT50Kb6sbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK28zps1OjsA%253D%253D&md5=70212d684742b588fce910b5e1a313f6</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1080%2F02681219680000381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F02681219680000381%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph-Horne%26aufirst%3DT.%26aulast%3DLoeffler%26aufirst%3DR.%2BS.%2BT.%26aulast%3DHollomon%26aufirst%3DD.%2BW.%26aulast%3DKelly%26aufirst%3DS.%2BL.%26atitle%3DAmphotericin%2520B%2520resistant%2520isolates%2520ofCryptococcus%2520neoformanswithout%2520alteration%2520in%2520sterol%2520biosynthesis%26jtitle%3DJ.%2520Med.%2520Vet.%2520Mycol.%26date%3D1996%26volume%3D34%26spage%3D223%26epage%3D225%26doi%3D10.1080%2F02681219680000381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loyse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dromer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lortholary, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, T. S.</span></span> <span> </span><span class="NLM_article-title">Flucytosine and Cryptococcosis: Time to Urgently Address the Worldwide Accessibility of a 50-year-old Antifungal</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">2435</span>– <span class="NLM_lpage">2444</span>, <span class="refDoi"> DOI: 10.1093/jac/dkt221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fjac%2Fdkt221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=23788479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CmurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2013&pages=2435-2444&author=A.+Loyseauthor=F.+Dromerauthor=J.+Dayauthor=O.+Lortholaryauthor=T.+S.+Harrison&title=Flucytosine+and+Cryptococcosis%3A+Time+to+Urgently+Address+the+Worldwide+Accessibility+of+a+50-year-old+Antifungal&doi=10.1093%2Fjac%2Fdkt221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Flucytosine and cryptococcosis: time to urgently address the worldwide accessibility of a 50-year-old antifungal</span></div><div class="casAuthors">Loyse, Angela; Dromer, Francoise; Day, Jeremy; Lortholary, Olivier; Harrison, Thomas S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2435-2444</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Current, widely accepted guidelines for the management of HIV-assocd. cryptococcal meningoencephalitis (CM) recommend amphotericin B combined with flucytosine (5-FC) for ≥2 wk as the initial induction treatment of choice.  However, access to flucytosine in Africa and Asia, where disease burden is greatest, is inadequate at present.  While research into identifying effective and well-tolerated antifungal combinations that do not contain flucytosine continues, an ever-increasing body of evidence from in vitro, in vivo, and clin. studies points to the benefits of flucytosine in the treatment of CM in both i.v. combinations with amphotericin B and oral combinations with high-dose fluconazole.  This article provides an up-to-date review of this evidence, and the current issues and challenges regarding increasing access to this key component of combination antifungal therapy for cryptococcosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeuqIQy-uf5rVg90H21EOLACvtfcHk0lg5fe4sC1XYWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CmurzP&md5=bb025d2005441ba2f42ea6ebddb20354</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkt221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkt221%26sid%3Dliteratum%253Aachs%26aulast%3DLoyse%26aufirst%3DA.%26aulast%3DDromer%26aufirst%3DF.%26aulast%3DDay%26aufirst%3DJ.%26aulast%3DLortholary%26aufirst%3DO.%26aulast%3DHarrison%26aufirst%3DT.%2BS.%26atitle%3DFlucytosine%2520and%2520Cryptococcosis%253A%2520Time%2520to%2520Urgently%2520Address%2520the%2520Worldwide%2520Accessibility%2520of%2520a%252050-year-old%2520Antifungal%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2013%26volume%3D68%26spage%3D2435%26epage%3D2444%26doi%3D10.1093%2Fjac%2Fdkt221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, T.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-H.</span></span> <span> </span><span class="NLM_article-title">Increasing Trend of Fluconazole-non-susceptible Cryptococcus Neoformans in Patients with Invasive Cryptococcosis: a 12-year Longitudinal Study</span>. <i>BMC Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">277</span>, <span class="refDoi"> DOI: 10.1186/s12879-015-1023-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1186%2Fs12879-015-1023-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=26194004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FmtFaisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=277&author=Y.-C.+Chenauthor=T.-Y.+Changauthor=J.-W.+Liuauthor=F.-J.+Chenauthor=C.-C.+Chienauthor=C.-H.+Leeauthor=C.-H.+Lu&title=Increasing+Trend+of+Fluconazole-non-susceptible+Cryptococcus+Neoformans+in+Patients+with+Invasive+Cryptococcosis%3A+a+12-year+Longitudinal+Study&doi=10.1186%2Fs12879-015-1023-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing trend of fluconazole-non-susceptible Cryptococcus neoformans in patients with invasive cryptococcosis: a 12-year longitudinal study</span></div><div class="casAuthors">Chen Yi-Chun; Chang Tzu-Yao; Liu Jien-Wei; Chen Fang-Ju; Lee Chen-Hsiang; Liu Jien-Wei; Lee Chen-Hsiang; Lu Cheng-Hsien; Chien Chun-Chih; Lu Cheng-Hsien; Lu Cheng-Hsien</div><div class="citationInfo"><span class="NLM_cas:title">BMC infectious diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">277</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  This study aimed to investigate the rate of fluconazole-non-susceptible Cryptococcus neoformans in Southern Taiwan for the period 2001-2012 and analyze the risk factors for acquiring it among patients with invasive cryptococcosis.  METHODS:  All enrolled strains were isolated from blood or cerebrospinal fluid samples of the included patients.  If a patient had multiple positive results for C. neoformans, only the first instance was enrolled.  Susceptibility testing was performed using the Clinical and Laboratory Standards Institutes M27-A3 broth micro-dilution method.  The MIC interpretative criteria for susceptibility to fluconazole were ≤ 8 μg/ml.  A total of 89 patients were included.  Patients (n = 59) infected by fluconazole-susceptible strains were compared with those (n = 30) infected by non-susceptible strains.  The patients' demographic and clinical characteristics were analyzed.  RESULTS:  The rate of fluconazole-non-susceptible C. neoformans in the study period significantly increased over time (p < 0.001).  The C. neoformans isolated in 2011-2012 (odds ratio: 10.68; 95 % confidence interval: 2.87-39.74; p < 0.001) was an independent predictive factor for the acquisition of fluconazole-non-susceptible C. neoformans.  CONCLUSIONS:  The rate of fluconazole-non-susceptible C. neoformans has significantly increased recently.  Continuous and large-scale anti-fungal susceptibility tests for C. neoformans are warranted to confirm this trend.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS21nkQL0Hl0xwLZxei1HN8fW6udTcc2eb53VVIzQmFk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FmtFaisw%253D%253D&md5=5fccae94bfa4f33266b719949438a274</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2Fs12879-015-1023-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12879-015-1023-8%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.-C.%26aulast%3DChang%26aufirst%3DT.-Y.%26aulast%3DLiu%26aufirst%3DJ.-W.%26aulast%3DChen%26aufirst%3DF.-J.%26aulast%3DChien%26aufirst%3DC.-C.%26aulast%3DLee%26aufirst%3DC.-H.%26aulast%3DLu%26aufirst%3DC.-H.%26atitle%3DIncreasing%2520Trend%2520of%2520Fluconazole-non-susceptible%2520Cryptococcus%2520Neoformans%2520in%2520Patients%2520with%2520Invasive%2520Cryptococcosis%253A%2520a%252012-year%2520Longitudinal%2520Study%26jtitle%3DBMC%2520Infect.%2520Dis.%26date%3D2015%26volume%3D15%26spage%3D277%26doi%3D10.1186%2Fs12879-015-1023-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanglard, D.</span></span> <span> </span><span class="NLM_article-title">Emerging Threats in Antifungal-Resistant Fungal Pathogens</span>. <i>Front. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">11</span>, <span class="refDoi"> DOI: 10.3389/fmed.2016.00011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.3389%2Ffmed.2016.00011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC28fjvFOnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=11&author=D.+Sanglard&title=Emerging+Threats+in+Antifungal-Resistant+Fungal+Pathogens&doi=10.3389%2Ffmed.2016.00011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging Threats in Antifungal-Resistant Fungal Pathogens</span></div><div class="casAuthors">Sanglard Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11</span>
        ISSN:<span class="NLM_cas:issn">2296-858X</span>.
    </div><div class="casAbstract">The use of antifungal drugs in the therapy of fungal diseases can lead to the development of antifungal resistance.  Resistance has been described for virtually all antifungal agents in diverse pathogens, including Candida and Aspergillus species.  The majority of resistance mechanisms have also been elucidated at the molecular level in these pathogens.  Drug resistance genes and genome mutations have been identified.  Therapeutic choices are limited for the control of fungal diseases, and it is tempting to combine several drugs to achieve better therapeutic efficacy.  In the recent years, several novel resistance patterns have been observed, including antifungal resistance originating from environmental sources in Aspergillus fumigatus and the emergence of simultaneous resistance to different antifungal classes (multidrug resistance) in different Candida species.  This review will summarize these current trends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTpI6xsgJyoezjk2McQZ6XmfW6udTcc2eb53VVIzQmFk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28fjvFOnsA%253D%253D&md5=35b70007e870f369abaacc240a9b699d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3389%2Ffmed.2016.00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmed.2016.00011%26sid%3Dliteratum%253Aachs%26aulast%3DSanglard%26aufirst%3DD.%26atitle%3DEmerging%2520Threats%2520in%2520Antifungal-Resistant%2520Fungal%2520Pathogens%26jtitle%3DFront.%2520Med.%26date%3D2016%26volume%3D3%26spage%3D11%26doi%3D10.3389%2Ffmed.2016.00011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scorzoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Paula E Silva, A. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcos, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assato, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Melo, W. C. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa-Orlandi, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendes-Giannini, M. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fusco-Almeida, A. M.</span></span> <span> </span><span class="NLM_article-title">Antifungal Therapy: New Advances in the Understanding and Treatment of Mycosis</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>08</i></span>,  <span class="NLM_fpage">36</span>, <span class="refDoi"> DOI: 10.3389/fmicb.2017.00036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.3389%2Ffmicb.2017.00036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC1c3jslyhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=08&publication_year=2017&pages=36&author=L.+Scorzoniauthor=A.+C.+A.+de+Paula+E+Silvaauthor=C.+M.+Marcosauthor=P.+A.+Assatoauthor=W.+C.+M.+A.+de+Meloauthor=H.+C.+de+Oliveiraauthor=C.+B.+Costa-Orlandiauthor=M.+J.+S.+Mendes-Gianniniauthor=A.+M.+Fusco-Almeida&title=Antifungal+Therapy%3A+New+Advances+in+the+Understanding+and+Treatment+of+Mycosis&doi=10.3389%2Ffmicb.2017.00036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Antifungal Therapy: New Advances in the Understanding and Treatment of Mycosis</span></div><div class="casAuthors">Scorzoni Liliana; de Paula E Silva Ana C A; Marcos Caroline M; Assato Patricia A; de Melo Wanessa C M A; de Oliveira Haroldo C; Costa-Orlandi Caroline B; Mendes-Giannini Maria J S; Fusco-Almeida Ana M</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">The high rates of morbidity and mortality caused by fungal infections are associated with the current limited antifungal arsenal and the high toxicity of the compounds.  Additionally, identifying novel drug targets is challenging because there are many similarities between fungal and human cells.  The most common antifungal targets include fungal RNA synthesis and cell wall and membrane components, though new antifungal targets are being investigated.  Nonetheless, fungi have developed resistance mechanisms, such as overexpression of e ux pump proteins and biofilm formation, emphasizing the importance of understanding these mechanisms.  To address these problems, different approaches to preventing and treating fungal diseases are described in this review, with a focus on the resistance mechanisms of fungi, with the goal of developing efficient strategies to overcoming and preventing resistance as well as new advances in antifungal therapy.  Due to the limited antifungal arsenal, researchers have sought to improve treatment via different approaches, and the synergistic effect obtained by the combination of antifungals contributes to reducing toxicity and could be an alternative for treatment.  Another important issue is the development of new formulations for antifungal agents, and interest in nanoparticles as new types of carriers of antifungal drugs has increased.  In addition, modifications to the chemical structures of traditional antifungals have improved their activity and pharmacokinetic parameters.  Moreover, a different approach to preventing and treating fungal diseases is immunotherapy, which involves different mechanisms, such as vaccines, activation of the immune response and inducing the production of host antimicrobial molecules.  Finally, the use of a mini-host has been encouraging for in vivo testing because these animal models demonstrate a good correlation with the mammalian model; they also increase the speediness of as well as facilitate the preliminary testing of new antifungal agents.  In general, many years are required from discovery of a new antifungal to clinical use.  However, the development of new antifungal strategies will reduce the therapeutic time and/or increase the quality of life of patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR79GlNDVo7nGgRQsDqrjW_fW6udTcc2eb53VVIzQmFk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c3jslyhsw%253D%253D&md5=ac4a31dd5a3d11a5a45cf999b348ece7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2017.00036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2017.00036%26sid%3Dliteratum%253Aachs%26aulast%3DScorzoni%26aufirst%3DL.%26aulast%3Dde%2BPaula%2BE%2BSilva%26aufirst%3DA.%2BC.%2BA.%26aulast%3DMarcos%26aufirst%3DC.%2BM.%26aulast%3DAssato%26aufirst%3DP.%2BA.%26aulast%3Dde%2BMelo%26aufirst%3DW.%2BC.%2BM.%2BA.%26aulast%3Dde%2BOliveira%26aufirst%3DH.%2BC.%26aulast%3DCosta-Orlandi%26aufirst%3DC.%2BB.%26aulast%3DMendes-Giannini%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DFusco-Almeida%26aufirst%3DA.%2BM.%26atitle%3DAntifungal%2520Therapy%253A%2520New%2520Advances%2520in%2520the%2520Understanding%2520and%2520Treatment%2520of%2520Mycosis%26jtitle%3DFront.%2520Microbiol.%26date%3D2017%26volume%3D08%26spage%3D36%26doi%3D10.3389%2Ffmicb.2017.00036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakanjako, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colebunders, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutinho, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamya, M. R.</span></span> <span> </span><span class="NLM_article-title">Strategies to Optimize HIV Treatment Outcomes in Resource-limited Settings</span>. <i>AIDS Rev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">189</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=19940945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD1MjotlaqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=179-189&author=D.+Nakanjakoauthor=R.+Colebundersauthor=A.+G.+Coutinhoauthor=M.+R.+Kamya&title=Strategies+to+Optimize+HIV+Treatment+Outcomes+in+Resource-limited+Settings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to optimize HIV treatment outcomes in resource-limited settings</span></div><div class="casAuthors">Nakanjako Damalie; Colebunders Robert; Coutinho Alex G; Kamya Moses R</div><div class="citationInfo"><span class="NLM_cas:title">AIDS reviews</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">179-89</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although the availability of antiretroviral therapy has increased rapidly to reach over three million people in low- and middle-income countries, coverage remains low as only 31% of people in need were receiving antiretroviral therapy in sub-Saharan Africa.  Antiretroviral therapy scale-up needs to continue to grow exponentially to meet the need for universal access and keep pace with or exceed the new HIV infections.  This calls for strategies that will have the greatest impact on the reduction of opportunistic infections, toxicities, and early mortality after antiretroviral therapy initiation as well improve adherence, clinical, immunological, and virologic responses, patient retention in antiretroviral therapy programs, and overall quality of life of people living with HIV/AIDS.  Expanding antiretroviral therapy to all those eligible requires evidence-based decisions about how, when, and where expansion should occur.  In this article we highlight some of the strategies that have optimized HIV treatment outcomes within the constraints of limited resources in sub-Saharan Africa.  Key strategies to optimize HIV treatment outcomes include, i) scaling up HIV testing to identify all in need of HIV treatment, ii) strengthening the links between HIV diagnosis and comprehensive HIV/AIDS care, iii) timely initiation of antiretroviral therapy, iv) optimal diagnosis and treatment of opportunistic infections and comorbidities, v) investing in laboratory tests to support clinical monitoring of patients on antiretroviral therapy, vi) maximizing adherence to antiretroviral medication and retention of patients in HIV/AIDS care, viii) improving the health infrastructure, and increasing the human resources to handle the growing numbers of people in need of HIV treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSO9DJxprTKtoIRAdBLKY41fW6udTcc2eYnPiKOr-jhArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MjotlaqtQ%253D%253D&md5=ed9d8026355aeb4bce4bef3f248e1af2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNakanjako%26aufirst%3DD.%26aulast%3DColebunders%26aufirst%3DR.%26aulast%3DCoutinho%26aufirst%3DA.%2BG.%26aulast%3DKamya%26aufirst%3DM.%2BR.%26atitle%3DStrategies%2520to%2520Optimize%2520HIV%2520Treatment%2520Outcomes%2520in%2520Resource-limited%2520Settings%26jtitle%3DAIDS%2520Rev.%26date%3D2009%26volume%3D11%26spage%3D179%26epage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levitz, S. M.</span></span> <span> </span><span class="NLM_article-title">Virulence Factors of Medically Important Fungi</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1128/cmr.9.4.469</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FCMR.9.4.469" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8894347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK28XmsFOksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=469-488&author=L.+H.+Hoganauthor=B.+S.+Kleinauthor=S.+M.+Levitz&title=Virulence+Factors+of+Medically+Important+Fungi&doi=10.1128%2Fcmr.9.4.469"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Virulence factors of medically important fungi</span></div><div class="casAuthors">Hogan, Laura H.; Klein, Bruce S.; Levitz, Stuart M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-488</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review with 326 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8P7URMyPg27Vg90H21EOLACvtfcHk0lgLJa-5MKadBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmsFOksrc%253D&md5=70097820426f47aaa5f629d95a99cd59</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FCMR.9.4.469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.9.4.469%26sid%3Dliteratum%253Aachs%26aulast%3DHogan%26aufirst%3DL.%2BH.%26aulast%3DKlein%26aufirst%3DB.%2BS.%26aulast%3DLevitz%26aufirst%3DS.%2BM.%26atitle%3DVirulence%2520Factors%2520of%2520Medically%2520Important%2520Fungi%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D1996%26volume%3D9%26spage%3D469%26epage%3D488%26doi%3D10.1128%2Fcmr.9.4.469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandujano-González, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa-Tanaca, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anducho-Reyes, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercado-Flores, Y.</span></span> <span> </span><span class="NLM_article-title">Secreted Fungal Aspartic Proteases: A review</span>. <i>Rev. Iberoam. Micol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1016/j.riam.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.riam.2015.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=27137097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC28blsVCjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=76-82&author=V.+Mandujano-Gonz%C3%A1lezauthor=L.+Villa-Tanacaauthor=M.+A.+Anducho-Reyesauthor=Y.+Mercado-Flores&title=Secreted+Fungal+Aspartic+Proteases%3A+A+review&doi=10.1016%2Fj.riam.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Secreted fungal aspartic proteases: A review</span></div><div class="casAuthors">Mandujano-Gonzalez Virginia; Anducho-Reyes Miguel Angel; Villa-Tanaca Lourdes; Mercado-Flores Yuridia</div><div class="citationInfo"><span class="NLM_cas:title">Revista iberoamericana de micologia</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">76-82</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The aspartic proteases, also called aspartyl and aspartate proteases or acid proteases (E.C.3.4.23), belong to the endopeptidase family and are characterized by the conserved sequence Asp-Gly-Thr at the active site.  These enzymes are found in a wide variety of microorganisms in which they perform important functions related to nutrition and pathogenesis.  In addition, their high activity and stability at acid pH make them attractive for industrial application in the food industry; specifically, they are used as milk-coagulating agents in cheese production or serve to improve the taste of some foods.  This review presents an analysis of the characteristics and properties of secreted microbial aspartic proteases and their potential for commercial application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTQMcnhOfS5GQxyojvcjibhfW6udTcc2eYnPiKOr-jhArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28blsVCjtw%253D%253D&md5=1ff57df1ed1fcec9a731668c1b9e2cb8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.riam.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.riam.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DMandujano-Gonz%25C3%25A1lez%26aufirst%3DV.%26aulast%3DVilla-Tanaca%26aufirst%3DL.%26aulast%3DAnducho-Reyes%26aufirst%3DM.%2BA.%26aulast%3DMercado-Flores%26aufirst%3DY.%26atitle%3DSecreted%2520Fungal%2520Aspartic%2520Proteases%253A%2520A%2520review%26jtitle%3DRev.%2520Iberoam.%2520Micol.%26date%3D2016%26volume%3D33%26spage%3D76%26epage%3D82%26doi%3D10.1016%2Fj.riam.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumesic, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donoghue, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Greca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhani, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">Integrated Activity and Genetic Profiling of Secreted Peptidases in Cryptococcus Neoformans Reveals an Aspartyl Peptidase Required for Low pH Survival and Virulence</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1006051</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1006051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1371%2Fjournal.ppat.1006051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=27977806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktVOhu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=S.+C.+Clarkeauthor=P.+A.+Dumesicauthor=C.+M.+Homerauthor=A.+J.+O%E2%80%99Donoghueauthor=F.+La+Grecaauthor=L.+Pallovaauthor=P.+Majerauthor=H.+D.+Madhaniauthor=C.+S.+Craik&title=Integrated+Activity+and+Genetic+Profiling+of+Secreted+Peptidases+in+Cryptococcus+Neoformans+Reveals+an+Aspartyl+Peptidase+Required+for+Low+pH+Survival+and+Virulence&doi=10.1371%2Fjournal.ppat.1006051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated activity and genetic profiling of secreted peptidases in Cryptococcus neoformans reveals an aspartyl peptidase required for low pH survival and virulence</span></div><div class="casAuthors">Clarke, Starlynn C.; Dumesic, Phillip A.; Homer, Christina M.; O'Donoghue, Anthony J.; La Greca, Florencia; Pallova, Lenka; Majer, Pavel; Madhani, Hiten D.; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1006051/1-e1006051/30</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The opportunistic fungal pathogen Cryptococcus neoformans is a major cause of mortality in immunocompromised individuals, resulting in more than 600,000 deaths per yr.  Many human fungal pathogens secrete peptidases that influence virulence, but in most cases the substrate specificity and regulation of these enzymes remains poorly understood.  The paucity of such information is a roadblock to our understanding of the biol. functions of peptidases and whether or not these enzymes are viable therapeutic targets.  We report here an unbiased anal. of secreted peptidase activity and specificity in C. neoformans using a mass spectrometry-based substrate profiling strategy and subsequent functional investigations.  Our initial studies revealed that global peptidase activity and specificity are dramatically altered by environmental conditions.  To uncover the substrate preferences of individual enzymes and interrogate their biol. functions, we constructed and profiled a ten-member gene deletion collection of candidate secreted peptidases.  Through this deletion approach, we characterized the substrate specificity of three peptidases within the context of the C. neoformans secretome, including an enzyme known to be important for fungal entry into the brain.  We selected a previously uncharacterized peptidase, which we term Major aspartyl peptidase 1 (May1), for detailed study due to its substantial contribution to extracellular proteolytic activity.  Based on the preference of May1 for proteolysis between hydrophobic amino acids, we screened a focused library of aspartyl peptidase inhibitors and identified four high-affinity antagonists.  Finally, we tested may1Δ strains in a mouse model of C. neoformans infection and found that strains lacking this enzyme are significantly attenuated for virulence.  Our study reveals the secreted peptidase activity and specificity of an important human fungal pathogen, identifies responsible enzymes through genetic tests of their function, and demonstrates how this information can guide the development of high affinity small mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUf62D_2DdfLVg90H21EOLACvtfcHk0lhpqJ7ibgocCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktVOhu7k%253D&md5=b196b08575b78973bee0c22fe12caf27</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1006051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1006051%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DS.%2BC.%26aulast%3DDumesic%26aufirst%3DP.%2BA.%26aulast%3DHomer%26aufirst%3DC.%2BM.%26aulast%3DO%25E2%2580%2599Donoghue%26aufirst%3DA.%2BJ.%26aulast%3DLa%2BGreca%26aufirst%3DF.%26aulast%3DPallova%26aufirst%3DL.%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DMadhani%26aufirst%3DH.%2BD.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DIntegrated%2520Activity%2520and%2520Genetic%2520Profiling%2520of%2520Secreted%2520Peptidases%2520in%2520Cryptococcus%2520Neoformans%2520Reveals%2520an%2520Aspartyl%2520Peptidase%2520Required%2520for%2520Low%2520pH%2520Survival%2520and%2520Virulence%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1006051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orsi, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibellini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esposito, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peppoloni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mussini, C.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of an aspartyl protease fromCryptococcus neoformans</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>581</i></span>,  <span class="NLM_fpage">3882</span>– <span class="NLM_lpage">3886</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2007.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.febslet.2007.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=17651737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotlSlsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=581&publication_year=2007&pages=3882-3886&author=M.+Pintiauthor=C.+F.+Orsiauthor=L.+Gibelliniauthor=R.+Espositoauthor=A.+Cossarizzaauthor=E.+Blasiauthor=S.+Peppoloniauthor=C.+Mussini&title=Identification+and+characterization+of+an+aspartyl+protease+fromCryptococcus+neoformans&doi=10.1016%2Fj.febslet.2007.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of an aspartyl protease from Cryptococcus neoformans</span></div><div class="casAuthors">Pinti, Marcello; Orsi, Carlotta Francesca; Gibellini, Lara; Esposito, Roberto; Cossarizza, Andrea; Blasi, Elisabetta; Peppoloni, Samuele; Mussini, Cristina</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">581</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3882-3886</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cryptococcosis, caused by Cryptococcus neoformans, is an invasive infection often occurring in AIDS patients.  Potent therapy against HIV, which includes protease inhibitors (PIs), has beneficial effects also on opportunistic infections by pathogens such as C. neoformans and C. albicans.  PIs inhibit growth of C. albicans by affecting the activity of its aspartyl proteases.  We identified, cloned and sequenced a cDNA from C. neoformans encoding for a putative aspartyl protease (CnAP1), and the corresponding genomic region.  The gene cnap1 codifies for a protein of 505 aa, with a canonical aspartyl protease structure.  We purified the recombinant protein and analyzed its activity in the presence of PIs (Indinavir, Lopinavir, Ritonavir), but did not evidence any inhibition of protease activity.  The transcriptional level of cnap1 in C. neoformans is const. in different media.  The absence of any inhibition activity by PIs suggests that other targets for PIs might exist in C. neoformans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeyYtH3G4rBrVg90H21EOLACvtfcHk0lhpqJ7ibgocCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotlSlsrs%253D&md5=ab205d4d01730c3197995c477d7e38a3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2007.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2007.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DPinti%26aufirst%3DM.%26aulast%3DOrsi%26aufirst%3DC.%2BF.%26aulast%3DGibellini%26aufirst%3DL.%26aulast%3DEsposito%26aufirst%3DR.%26aulast%3DCossarizza%26aufirst%3DA.%26aulast%3DBlasi%26aufirst%3DE.%26aulast%3DPeppoloni%26aufirst%3DS.%26aulast%3DMussini%26aufirst%3DC.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520an%2520aspartyl%2520protease%2520fromCryptococcus%2520neoformans%26jtitle%3DFEBS%2520Lett.%26date%3D2007%26volume%3D581%26spage%3D3882%26epage%3D3886%26doi%3D10.1016%2Fj.febslet.2007.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidrim, J. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdigão-Neto, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordeiro, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brilhante, R. S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leite, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teixeira, C. E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitas, R. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesquita, J. R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves, M. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, M. F. G.</span></span> <span> </span><span class="NLM_article-title">Viral Protease Inhibitors Affect the Production of Virulence Factors in Cryptococcus Neoformans</span>. <i>Can. J. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">932</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1139/w2012-075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1139%2Fw2012-075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=22716223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2012&pages=932-936&author=J.+J.+C.+Sidrimauthor=L.+V.+Perdig%C3%A3o-Netoauthor=R.+A.+Cordeiroauthor=R.+S.+N.+Brilhanteauthor=J.+J.+G.+Leiteauthor=C.+E.+C.+Teixeiraauthor=A.+J.+Monteiroauthor=R.+M.+F.+Freitasauthor=J.+F.+Ribeiroauthor=J.+R.+L.+Mesquitaauthor=M.+V.+F.+Gon%C3%A7alvesauthor=M.+F.+G.+Rocha&title=Viral+Protease+Inhibitors+Affect+the+Production+of+Virulence+Factors+in+Cryptococcus+Neoformans&doi=10.1139%2Fw2012-075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Viral protease inhibitors affect the production of virulence factors in Cryptococcus neoformans</span></div><div class="casAuthors">Sidrim, J. J. C.; Perdigao-Neto, L. V.; Cordeiro, R. A.; Brilhante, R. S. N.; Leite, J. J. G.; Teixeira, C. E. C.; Monteiro, A. J.; Freitas, R. M. F.; Ribeiro, J. F.; Mesquita, J. R. L.; Goncalves, M. V. F.; Rocha, M. F. G.</div><div class="citationInfo"><span class="NLM_cas:title">Canadian Journal of Microbiology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">932-936</span>CODEN:
                <span class="NLM_cas:coden">CJMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0008-4166</span>.
    
            (<span class="NLM_cas:orgname">Canadian Science Publishing</span>)
        </div><div class="casAbstract">The effects of the protease inhibitors saquinavir, darunavir, ritonavir, and indinavir on growth inhibition, protease and phospholipase activities, as well as capsule thickness of Cryptococcus neoformans were investigated.  Viral protease inhibitors did not reduce fungal growth when tested in concns. ranging from 0.001 to 1.000 mg/L.  A tendency toward increasing phospholipase activity was obsd. with the highest tested drug concn. in a strain-specific pattern.  However, these drugs reduced protease activity as well as capsule prodn.  Our results confirm a previous finding that antiretroviral drugs affect the prodn. of important virulence factors of C. neoformans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJe8ljIdPt-LVg90H21EOLACvtfcHk0lifOHYZ_usGLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGhtLg%253D&md5=9dc87065ed3e3139462a75383c075623</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1139%2Fw2012-075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fw2012-075%26sid%3Dliteratum%253Aachs%26aulast%3DSidrim%26aufirst%3DJ.%2BJ.%2BC.%26aulast%3DPerdig%25C3%25A3o-Neto%26aufirst%3DL.%2BV.%26aulast%3DCordeiro%26aufirst%3DR.%2BA.%26aulast%3DBrilhante%26aufirst%3DR.%2BS.%2BN.%26aulast%3DLeite%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DTeixeira%26aufirst%3DC.%2BE.%2BC.%26aulast%3DMonteiro%26aufirst%3DA.%2BJ.%26aulast%3DFreitas%26aufirst%3DR.%2BM.%2BF.%26aulast%3DRibeiro%26aufirst%3DJ.%2BF.%26aulast%3DMesquita%26aufirst%3DJ.%2BR.%2BL.%26aulast%3DGon%25C3%25A7alves%26aufirst%3DM.%2BV.%2BF.%26aulast%3DRocha%26aufirst%3DM.%2BF.%2BG.%26atitle%3DViral%2520Protease%2520Inhibitors%2520Affect%2520the%2520Production%2520of%2520Virulence%2520Factors%2520in%2520Cryptococcus%2520Neoformans%26jtitle%3DCan.%2520J.%2520Microbiol.%26date%3D2012%26volume%3D58%26spage%3D932%26epage%3D936%26doi%3D10.1139%2Fw2012-075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadevall, A.</span></span> <span> </span><span class="NLM_article-title">Virulence-Associated Enzymes of Cryptococcus Neoformans</span>. <i>Eukaryotic Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1173</span>– <span class="NLM_lpage">1185</span>, <span class="refDoi"> DOI: 10.1128/ec.00103-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FEC.00103-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=26453651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2qsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=1173-1185&author=F.+Almeidaauthor=J.+M.+Wolfauthor=A.+Casadevall&title=Virulence-Associated+Enzymes+of+Cryptococcus+Neoformans&doi=10.1128%2Fec.00103-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Virulence-associated enzymes of Cryptococcus neoformans</span></div><div class="casAuthors">Almeida, Fausto; Wolf, Julie M.; Casadevall, Arturo</div><div class="citationInfo"><span class="NLM_cas:title">Eukaryotic Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1173-1185</span>CODEN:
                <span class="NLM_cas:coden">ECUEA2</span>;
        ISSN:<span class="NLM_cas:issn">1535-9786</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Enzymes play key roles in fungal pathogenesis.  Manipulation of enzyme expression or activity can significantly alter the infection process, and enzyme expression profiles can be a hallmark of disease.  Hence, enzymes are worthy targets for better understanding pathogenesis and identifying new options for combating fungal infections.  Advances in genomics, proteomics, transcriptomics, and mass spectrometry have enabled the identification and characterization of new fungal enzymes.  This review focuses on recent developments in the virulence-assocd. enzymes from Cryptococcus neoformans.  The enzymic suite of C. neoformans has evolved for environmental survival, but several of these enzymes play a dual role in colonizing the mammalian host.  We also discuss new therapeutic and diagnostic strategies that could be based on the underlying enzymol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu9Edib2HgurVg90H21EOLACvtfcHk0lifOHYZ_usGLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2qsb4%253D&md5=789b1ca217f73079e96f40cc81f89b61</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FEC.00103-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FEC.00103-15%26sid%3Dliteratum%253Aachs%26aulast%3DAlmeida%26aufirst%3DF.%26aulast%3DWolf%26aufirst%3DJ.%2BM.%26aulast%3DCasadevall%26aufirst%3DA.%26atitle%3DVirulence-Associated%2520Enzymes%2520of%2520Cryptococcus%2520Neoformans%26jtitle%3DEukaryotic%2520Cell%26date%3D2015%26volume%3D14%26spage%3D1173%26epage%3D1185%26doi%3D10.1128%2Fec.00103-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casadevall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coelho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alanio, A.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Cryptococcus Neoformans-mediated Host Damage</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">855</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2018.00855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.3389%2Ffimmu.2018.00855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=29760698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Gls7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=855&author=A.+Casadevallauthor=C.+Coelhoauthor=A.+Alanio&title=Mechanisms+of+Cryptococcus+Neoformans-mediated+Host+Damage&doi=10.3389%2Ffimmu.2018.00855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Cryptococcus neoformans-mediated host damage</span></div><div class="casAuthors">Casadevall, Arturo; Coelho, Carolina; Alanio, Alexandre</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">855/1-855/8</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Cryptococcus neoformans is not usually considered a cytotoxic fungal pathogen but there is considerable evidence that this microbe can damage host cells and tissues.  In this essay, we review the evidence that C. neoformans damages host cells and note that the mechanisms involved are diverse.  We consider C. neoformans-mediated host damage at the mol., cellular, tissue, and organism level.  Direct mechanisms of cytotoxicity include lytic exocytosis, organelle dysfunction, phagolysosomal membrane damage, and cytoskeletal alterations.  Cytotoxicity contributes to pathogenesis by inter- fering with immune effector cell function and disrupting endothelial barriers thus allowing dissemination.  When C. neoformans-mediated and immune-mediated host damage is sufficient to affect homeostasis, cryptococcosis occurs at the organism level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppkLowe53xLbVg90H21EOLACvtfcHk0lifOHYZ_usGLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Gls7vI&md5=cbc33b6eee40097f5e6ce3d3ef04699d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2018.00855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2018.00855%26sid%3Dliteratum%253Aachs%26aulast%3DCasadevall%26aufirst%3DA.%26aulast%3DCoelho%26aufirst%3DC.%26aulast%3DAlanio%26aufirst%3DA.%26atitle%3DMechanisms%2520of%2520Cryptococcus%2520Neoformans-mediated%2520Host%2520Damage%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D9%26spage%3D855%26doi%3D10.3389%2Ffimmu.2018.00855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhrig, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Na Pombejra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamklang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bautos, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelli, A.</span></span> <span> </span><span class="NLM_article-title">Invasion of the Central Nervous System by Cryptococcus Neoformans Requires a Secreted Fungal Metalloprotease</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e01101</span> <span class="refDoi"> DOI: 10.1128/mBio.01101-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FmBio.01101-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=24895304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1OlsLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&author=K.+Vuauthor=R.+Thamauthor=J.+P.+Uhrigauthor=G.+R.+Thompsonauthor=S.+Na+Pombejraauthor=M.+Jamklangauthor=J.+M.+Bautosauthor=A.+Gelli&title=Invasion+of+the+Central+Nervous+System+by+Cryptococcus+Neoformans+Requires+a+Secreted+Fungal+Metalloprotease&doi=10.1128%2FmBio.01101-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Invasion of the central nervous system by Cryptococcus neoformans requires a secreted fungal metalloprotease</span></div><div class="casAuthors">Vu, Kiem; Tham, Rick; Uhrig, John P.; Thompson, George R., III; Pombejra, Sarisa Na; Jamklang, Mantana; Bautos, Jennifer M.; Gelli, Angie</div><div class="citationInfo"><span class="NLM_cas:title">mBio</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e01101/1-e01101/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">MBIOCL</span>;
        ISSN:<span class="NLM_cas:issn">2150-7511</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cryptococcus spp. cause life-threatening fungal infection of the central nervous system (CNS), predominantly in patients with a compromised immune system.  Why Cryptococcus neoformans has this remarkable tropism for the CNS is not clear.  Recent research on cerebral pathogenesis of C. neoformans revealed a predominantly transcellular migration of cryptococci across the brain endothelium; however, the identities of key fungal virulence factors that function specifically to invade the CNS remain unresolved.  Here we found that a novel, secreted metalloprotease (Mpr1) that we identified in the extracellular proteome of C. neoformans (CnMpr1) is required for establishing fungal disease in the CNS.  Mpr1 belongs to a poorly characterized M36 class of fungalysins that are expressed in only some fungal species.  A strain of C. neoformans lacking the gene encoding Mpr1 (mpr1Δ) failed to breach the endothelium in an in vitro model of the human blood-brain barrier (BBB).  A mammalian host infected with the mpr1Δ null strain demonstrated significant improvement in survival due to a reduced brain fungal burden and lacked the brain pathol. commonly assocd. with cryptococcal disease.  The in vivo studies further indicate that Mpr1 is not required for fungal dissemination and Mpr1 likely targets the brain endothelium specifically.  Remarkably, the sole expression of CnMPR1 in Saccharomyces cerevisiae resulted in a robust migration of yeast cells across the brain endothelium, demonstrating Mpr1's specific activity in breaching the BBB and suggesting that Mpr1 may function independently of the hyaluronic acid-CD44 pathway.  This distinct role for Mpr1 may develop into innovative treatment options and facilitate a brain-specific drug delivery platform.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrGoQEhu8PB7Vg90H21EOLACvtfcHk0lj8Hha8Wg9CZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1OlsLfL&md5=e10ebba9af88ef9ea711d66fc1b49335</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1128%2FmBio.01101-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.01101-14%26sid%3Dliteratum%253Aachs%26aulast%3DVu%26aufirst%3DK.%26aulast%3DTham%26aufirst%3DR.%26aulast%3DUhrig%26aufirst%3DJ.%2BP.%26aulast%3DThompson%26aufirst%3DG.%2BR.%26aulast%3DNa%2BPombejra%26aufirst%3DS.%26aulast%3DJamklang%26aufirst%3DM.%26aulast%3DBautos%26aufirst%3DJ.%2BM.%26aulast%3DGelli%26aufirst%3DA.%26atitle%3DInvasion%2520of%2520the%2520Central%2520Nervous%2520System%2520by%2520Cryptococcus%2520Neoformans%2520Requires%2520a%2520Secreted%2520Fungal%2520Metalloprotease%26jtitle%3DmBio%26date%3D2014%26volume%3D5%26doi%3D10.1128%2FmBio.01101-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito-Kuwa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ninomiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidotto, V.</span></span> <span> </span><span class="NLM_article-title">Extracellular proteolytic activity of Cryptococcus neoformans</span>. <i>Mycopathologia</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1007/bf01138475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1007%2FBF01138475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=7739728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK2MXmsV2gsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=1994&pages=143-150&author=S.+Aokiauthor=S.+Ito-Kuwaauthor=K.+Nakamuraauthor=J.+Katoauthor=K.+Ninomiyaauthor=V.+Vidotto&title=Extracellular+proteolytic+activity+of+Cryptococcus+neoformans&doi=10.1007%2Fbf01138475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular proteolytic activity of Cryptococcus neoformans</span></div><div class="casAuthors">Aoki, Shigeji; Ito-Kuwa, Shoko; Nakamura, Kenjirou; Kato, Joji; Ninomiya, Kazunori; Vidotto, Valerio</div><div class="citationInfo"><span class="NLM_cas:title">Mycopathologia</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">143-50</span>CODEN:
                <span class="NLM_cas:coden">MYCPAH</span>;
        ISSN:<span class="NLM_cas:issn">0301-486X</span>.
    </div><div class="casAbstract">Eight strains of Cryptococcus neoformans var. neoformans isolated from AIDS patients in the Infectious Disease Institute, University of Turin, Italy, were examd. for growth and extracellular proteolytic activity in culture with solid and liq. media.  All of the strains grew well on Yeast Carbon Base (YCB) agar medium supplemented with both 0.1% (w/v) bovine serum albumin (BSA) and 0.01% (w/v) polypeptone (Pp), and produced a clear proteolytic zone around their colonies, whereas they exhibited less growth and proteolytic activity on YCB medium supplemented with BSA alone.  Strain #8 with a strong proteolytic activity was cultured in three different liq. media.  Its growth was limited in YCB medium supplemented with 0.1% BSA, but was moderate in that with 0.01% Pp.  Enhanced growth was supported by the addn. of both BSA and Pp to the YCB medium.  The relative value of the final cellular yields obtained with the above YCB-0.1% BSA, YCB-0.01% Pp and YCB-0.1% BSA-0.01% Pp media was approx. 1:10:20.  In the culture with YCB medium contg. both BSA and Pp, a rapid decrease in the amt. of BSA was demonstrated by a spectrophotometric assay and gel electrophoresis of the culture supernatant after the log-to-stationary phase.  The proteolytic activity in the culture supernatant became detectable after the log phase when tested with skim milk agarose plates.  These results show that Cr. neoformans var. neoformans is able to secrete protease and to utilize protein as a source of nitrogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVkUqt-jc0B7Vg90H21EOLACvtfcHk0ljRko4uNFXfhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmsV2gsb4%253D&md5=c3306de7efdd241b62c7afc4d5952fe0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2FBF01138475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01138475%26sid%3Dliteratum%253Aachs%26aulast%3DAoki%26aufirst%3DS.%26aulast%3DIto-Kuwa%26aufirst%3DS.%26aulast%3DNakamura%26aufirst%3DK.%26aulast%3DKato%26aufirst%3DJ.%26aulast%3DNinomiya%26aufirst%3DK.%26aulast%3DVidotto%26aufirst%3DV.%26atitle%3DExtracellular%2520proteolytic%2520activity%2520of%2520Cryptococcus%2520neoformans%26jtitle%3DMycopathologia%26date%3D1994%26volume%3D128%26spage%3D143%26epage%3D150%26doi%3D10.1007%2Fbf01138475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadevall, A.</span></span> <span> </span><span class="NLM_article-title">Extracellular Proteinase Activity of Cryptococcus Neoformans</span>. <i>Clin. Diagn. Lab. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1128/cdli.3.5.570-574.1996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FCDLI.3.5.570-574.1996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8877137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK28XmtFWjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1996&pages=570-574&author=L.+C.+Chenauthor=E.+S.+Blankauthor=A.+Casadevall&title=Extracellular+Proteinase+Activity+of+Cryptococcus+Neoformans&doi=10.1128%2Fcdli.3.5.570-574.1996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular proteinase activity of Cryptococcus neoformans</span></div><div class="casAuthors">Chen, Lin-Chi; Blank, Erika S.; Casadevall, Arturo</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Diagnostic Laboratory Immunology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">570-574</span>CODEN:
                <span class="NLM_cas:coden">CDIMEN</span>;
        ISSN:<span class="NLM_cas:issn">1071-412X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Extracellular proteinase activity was studies for eight strains of Cryptococcus neoformans var. neoformans and two strains of Cryptococcus neoformans var. gattii.  Proteinase activity was measured by protein agar clearance, azoalbumin hydrolysis, gelatin liquefaction, and protein substrate polyacrylamide gel electrophoresis.  All strains of C. neoformans produced extracellular proteolytic activity.  Maximal extracellular proteinase activity in supernatants of C. neoformans cultures was assocd. with late logarithmic- and stationary-phase cultures.  C. neoformans was able to utilize murine IgG1, bovine IgG, and human complement factor 5 for growth in media contg. these proteins as the sole sources of carbon and nitrogen, suggesting a capacity to degrade immunol. important proteins.  Protein substrate polyacrylamide gel electrophoresis revealed several bands with proteolytic activity at apparent mol. masses of 200, 100, and 50 kDa.  The results confirm the existence of extracellular proteinase activity for C. neoformans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocstBEeaOWsrVg90H21EOLACvtfcHk0lhhjEFMl4VGQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtFWjs7c%253D&md5=163674442c972568204ea6991082ce2d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1128%2FCDLI.3.5.570-574.1996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCDLI.3.5.570-574.1996%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%2BC.%26aulast%3DBlank%26aufirst%3DE.%2BS.%26aulast%3DCasadevall%26aufirst%3DA.%26atitle%3DExtracellular%2520Proteinase%2520Activity%2520of%2520Cryptococcus%2520Neoformans%26jtitle%3DClin.%2520Diagn.%2520Lab.%2520Immunol.%26date%3D1996%26volume%3D3%26spage%3D570%26epage%3D574%26doi%3D10.1128%2Fcdli.3.5.570-574.1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blasi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombari, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francesca Orsi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troiano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esposito, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peppoloni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mussini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neglia, R.</span></span> <span> </span><span class="NLM_article-title">The human immunodeficiency virus (HIV) protease inhibitor indinavir directly affects the opportunistic fungal pathogenCryptococcus neoformans</span>. <i>FEMS Immunol. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/j.femsim.2004.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.femsim.2004.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=15364103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFSktbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2004&pages=187-195&author=E.+Blasiauthor=B.+Colombariauthor=C.+Francesca+Orsiauthor=M.+Pintiauthor=L.+Troianoauthor=A.+Cossarizzaauthor=R.+Espositoauthor=S.+Peppoloniauthor=C.+Mussiniauthor=R.+Neglia&title=The+human+immunodeficiency+virus+%28HIV%29+protease+inhibitor+indinavir+directly+affects+the+opportunistic+fungal+pathogenCryptococcus+neoformans&doi=10.1016%2Fj.femsim.2004.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The human immunodeficiency virus (HIV) protease inhibitor indinavir directly affects the opportunistic fungal pathogen Cryptococcus neoformans</span></div><div class="casAuthors">Blasi, Elisabetta; Colombari, Bruna; Orsi, Carlotta Francesca; Pinti, Marcello; Troiano, Leonarda; Cossarizza, Andrea; Esposito, Roberto; Peppoloni, Samuela; Mussini, Cristina; Neglia, Rachele</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Immunology and Medical Microbiology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-195</span>CODEN:
                <span class="NLM_cas:coden">FIMIEV</span>;
        ISSN:<span class="NLM_cas:issn">0928-8244</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Highly active antiretroviral therapy (HAART), that includes human immunodeficiency virus (HIV) protease inhibitors (PIs), has been remarkably efficacious including against some opportunistic infections.  In this report we investigated the effect(s) of the PI indinavir on protease activity by Cryptococcus neoformans, an opportunistic fungal pathogen responsible for recurrent meningoencephalitis in AIDS patients.  Indinavir was also tested for potential effects on other parameters, such as fungal viability, growth ability and susceptibility to immune effector cells.  It was found that indinavir impaired cryptococcal protease activity in a time- and dose-dependent fashion.  The phenomenon was similarly detectable in ATCC/lab. strains and clin. isolates.  C. neoformans growth rate was also significantly reduced upon exposure to indinavir, while fungal viability was not affected and mitochondrial toxicity not detected.  Furthermore, as assessed by an in vitro infection model, indinavir significantly and consistently augmented C. neoformans susceptibility to microglial cell-mediated phagocytosis and killing.  Overall, by providing the first evidence that indinavir directly affects C. neoformans, these data add new in vitro insights on the wide-spectrum efficacy of PIs, further arguing for the clin. relevance of HAART against opportunistic infections in AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolSYsj5AJkybVg90H21EOLACvtfcHk0lhhjEFMl4VGQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFSktbk%253D&md5=1b0f7a5f45c8c5107b8bf18e704cf928</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.femsim.2004.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.femsim.2004.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DBlasi%26aufirst%3DE.%26aulast%3DColombari%26aufirst%3DB.%26aulast%3DFrancesca%2BOrsi%26aufirst%3DC.%26aulast%3DPinti%26aufirst%3DM.%26aulast%3DTroiano%26aufirst%3DL.%26aulast%3DCossarizza%26aufirst%3DA.%26aulast%3DEsposito%26aufirst%3DR.%26aulast%3DPeppoloni%26aufirst%3DS.%26aulast%3DMussini%26aufirst%3DC.%26aulast%3DNeglia%26aufirst%3DR.%26atitle%3DThe%2520human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520protease%2520inhibitor%2520indinavir%2520directly%2520affects%2520the%2520opportunistic%2520fungal%2520pathogenCryptococcus%2520neoformans%26jtitle%3DFEMS%2520Immunol.%2520Med.%2520Microbiol.%26date%3D2004%26volume%3D42%26spage%3D187%26epage%3D195%26doi%3D10.1016%2Fj.femsim.2004.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bernardis, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torosantucci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumbarello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cauda, R.</span></span> <span> </span><span class="NLM_article-title">In Vitro and In Vivo Anticandidal Activity of Human Immunodeficiency Virus Protease Inhibitors</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>180</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1086/314871</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1086%2F314871" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10395861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK1MXlt1ygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=1999&pages=448-453&author=A.+Cassoneauthor=F.+De+Bernardisauthor=A.+Torosantucciauthor=E.+Tacconelliauthor=M.+Tumbarelloauthor=R.+Cauda&title=In+Vitro+and+In+Vivo+Anticandidal+Activity+of+Human+Immunodeficiency+Virus+Protease+Inhibitors&doi=10.1086%2F314871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro and in vivo anticandidal activity of human immunodeficiency virus protease inhibitors</span></div><div class="casAuthors">Cassone, Antonio; De Bernardis, Flavia; Torosantucci, Antonella; Tacconelli, Evelina; Tumbarello, Mario; Cauda, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">180</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-453</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">Highly active antiretroviral therapy that includes human immunodeficiency virus (HIV) aspartyl protease inhibitors (PIs) causes a decline in the incidence of some opportunistic infections in AIDS, and this decline is currently attributed to the restoration of specific immunity.  The effect of two PIs (indinavir and ritonavir) on the enzymic activity of a secretory aspartyl protease (Sap) of Candida albicans (a major agent of mucosal disease in HIV-infected subjects) and on growth and exptl. pathogenicity of this fungus was evaluated.  Both PIs strongly (≥90%) and concn.-dependently (0.1-10 μM) inhibited Sap activity and prodn.  They also reduced Candida growth in a nitrogen-limited, Sap-expression-dependent growth medium and exerted a therapeutic effect in an exptl. model of vaginal candidiasis, with an efficacy comparable to that of fluconazole.  Thus, besides the expected immunorestoration, patients receiving PI therapy may benefit from a direct anticandidal activity of these drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzWKcfIy7H-bVg90H21EOLACvtfcHk0lhhjEFMl4VGQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlt1ygtr4%253D&md5=f1660f40251968002ac0b533ad6b16eb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1086%2F314871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F314871%26sid%3Dliteratum%253Aachs%26aulast%3DCassone%26aufirst%3DA.%26aulast%3DDe%2BBernardis%26aufirst%3DF.%26aulast%3DTorosantucci%26aufirst%3DA.%26aulast%3DTacconelli%26aufirst%3DE.%26aulast%3DTumbarello%26aufirst%3DM.%26aulast%3DCauda%26aufirst%3DR.%26atitle%3DIn%2520Vitro%2520and%2520In%2520Vivo%2520Anticandidal%2520Activity%2520of%2520Human%2520Immunodeficiency%2520Virus%2520Protease%2520Inhibitors%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D1999%26volume%3D180%26spage%3D448%26epage%3D453%26doi%3D10.1086%2F314871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mussini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pezzotti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Quiros, J. C. L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cinque, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borghi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domingo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cahn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Luca, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">d’Arminio
Monforte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nwokolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negroni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacchetti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antinori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazzarin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esposito, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antinori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aberg, J. A.</span></span> <span> </span><span class="NLM_article-title">Discontinuation of Maintenance Therapy for Cryptococcal Meningitis in Patients with AIDS Treated with Highly Active Antiretroviral Therapy: an International Observational Study</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1086/381261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1086%2F381261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=14765351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD2c%252Fmtlaguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2004&pages=565-571&author=C.+Mussiniauthor=P.+Pezzottiauthor=J.+M.+Miroauthor=E.+Martinezauthor=J.+C.+L.+B.+de+Quirosauthor=P.+Cinqueauthor=V.+Borghiauthor=A.+Bediniauthor=P.+Domingoauthor=P.+Cahnauthor=P.+Bossiauthor=A.+de+Lucaauthor=A.+d%E2%80%99Arminio%0AMonforteauthor=M.+Nelsonauthor=N.+Nwokoloauthor=S.+Helouauthor=R.+Negroniauthor=G.+Jacchettiauthor=S.+Antinoriauthor=A.+Lazzarinauthor=A.+Cossarizzaauthor=R.+Espositoauthor=A.+Antinoriauthor=J.+A.+Aberg&title=Discontinuation+of+Maintenance+Therapy+for+Cryptococcal+Meningitis+in+Patients+with+AIDS+Treated+with+Highly+Active+Antiretroviral+Therapy%3A+an+International+Observational+Study&doi=10.1086%2F381261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discontinuation of maintenance therapy for cryptococcal meningitis in patients with AIDS treated with highly active antiretroviral therapy: an international observational study</span></div><div class="casAuthors">Mussini Cristina; Pezzotti Patrizio; Miro Jose M; Martinez Esteban; de Quiros Juan Carlos Lopez Bernaldo; Cinque Paola; Borghi Vanni; Bedini Andrea; Domingo Pere; Cahn Pedro; Bossi Philippe; de Luca Andrea; d'Arminio Monforte Antonella; Nelson Mark; Nwokolo Nneka; Helou Silvia; Negroni Ricardo; Jacchetti Gaia; Antinori Spinello; Lazzarin Adriano; Cossarizza Andrea; Esposito Roberto; Antinori Andrea; Aberg Judith A</div><div class="citationInfo"><span class="NLM_cas:title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">565-71</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We conducted a retrospective, multicenter study evaluating the safety of discontinuing maintenance therapy for cryptococcal meningitis after immune reconstitution.  Inclusion criteria were a previous definitive diagnosis of cryptococcal meningitis, a CD4 cell count of >100 cells/microL while receiving highly active antiretroviral therapy (HAART), and the subsequent discontinuation of maintenance therapy for cryptococcal meningitis.  The primary end point was relapse of cryptococcal disease.  As of July 2002, 100 patients were enrolled.  When maintenance therapy was discontinued, the median CD4 cell count was 259 cells/microL and the median plasma virus load was <2.30 log10 copies/mL, and serum cryptococcal antigen was undetectable in 56 patients.  During a median follow-up period of 28.4 months (range, 6.7-64.5; 262 person-years), 4 events were observed (incidence, 1.53 events per 100 person-years; 95% confidence interval, 0.42-3.92).  Three of these patients had a CD4 cell count of >100 cells/microL and a positive serum cryptococcal antigen test result during the recurrent episode.  In conclusion, discontinuation of maintenance therapy for cryptococcal meningitis is safe if the CD4 cell count increases to >100 cells/microL while receiving HAART.  Recurrent cryptococcal infection should be suspected in patients whose serum cryptococcal antigen test results revert back to positive after discontinuation of maintenance therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnmn6Ub2HVWPIW7hznt0zpfW6udTcc2eaAkGec1stHdrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c%252Fmtlaguw%253D%253D&md5=bf61cc060d032bafa195eff03773a52e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1086%2F381261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F381261%26sid%3Dliteratum%253Aachs%26aulast%3DMussini%26aufirst%3DC.%26aulast%3DPezzotti%26aufirst%3DP.%26aulast%3DMiro%26aufirst%3DJ.%2BM.%26aulast%3DMartinez%26aufirst%3DE.%26aulast%3Dde%2BQuiros%26aufirst%3DJ.%2BC.%2BL.%2BB.%26aulast%3DCinque%26aufirst%3DP.%26aulast%3DBorghi%26aufirst%3DV.%26aulast%3DBedini%26aufirst%3DA.%26aulast%3DDomingo%26aufirst%3DP.%26aulast%3DCahn%26aufirst%3DP.%26aulast%3DBossi%26aufirst%3DP.%26aulast%3Dde%2BLuca%26aufirst%3DA.%26aulast%3Dd%25E2%2580%2599Arminio%2BMonforte%26aufirst%3DA.%26aulast%3DNelson%26aufirst%3DM.%26aulast%3DNwokolo%26aufirst%3DN.%26aulast%3DHelou%26aufirst%3DS.%26aulast%3DNegroni%26aufirst%3DR.%26aulast%3DJacchetti%26aufirst%3DG.%26aulast%3DAntinori%26aufirst%3DS.%26aulast%3DLazzarin%26aufirst%3DA.%26aulast%3DCossarizza%26aufirst%3DA.%26aulast%3DEsposito%26aufirst%3DR.%26aulast%3DAntinori%26aufirst%3DA.%26aulast%3DAberg%26aufirst%3DJ.%2BA.%26atitle%3DDiscontinuation%2520of%2520Maintenance%2520Therapy%2520for%2520Cryptococcal%2520Meningitis%2520in%2520Patients%2520with%2520AIDS%2520Treated%2520with%2520Highly%2520Active%2520Antiretroviral%2520Therapy%253A%2520an%2520International%2520Observational%2520Study%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2004%26volume%3D38%26spage%3D565%26epage%3D571%26doi%3D10.1086%2F381261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santos, A. L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braga-Silva, L. A.</span></span> <span> </span><span class="NLM_article-title">Aspartic Protease Inhibitors: Effective Drugs Against the Human Fungal Pathogen Candida Albicans</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">162</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.2174%2F138955713804484802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=23256485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFertLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=155-162&author=A.+L.+S.+Santosauthor=L.+A.+Braga-Silva&title=Aspartic+Protease+Inhibitors%3A+Effective+Drugs+Against+the+Human+Fungal+Pathogen+Candida+Albicans"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Aspartic protease inhibitors: effective drugs against the human fungal pathogen Candida albicans</span></div><div class="casAuthors">Santos, Andre L. S.; Braga-Silva, Lys A.</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">155-162</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Candida albicans can invade humans and may lead to mucosal and skin infections or to deep-seated mycoses of almost all inner organs, esp. in immunocompromised patients.  In this context, both the host immune status and the ability of C. albicans to modulate the expression of its virulence factors are relevant aspects that drive the candidal susceptibility or resistance; in this last case, culminating in the establishment of successful infection known as candidiasis.  C. albicans possesses a potent armamentarium consisting of several virulence mols. that help the fungal cells to escape from the host immune responses.  There is no doubt that the secretion of aspartic proteases, designated as Saps, is one of the major virulence attributes produced by C. albicans cells, since these hydrolytic enzymes participate in a wide range of fungal physiol. processes as well as in different facets of the fungal-host interactions.  For these reasons, Saps clearly hold promise as new potential drug targets.  Corroborating this hypothesis, the introduction of anti-human immunodeficiency virus drugs of the aspartic protease inhibitor-type (HIV PIs) have emerged as new agents for the inhibition of Saps.  The introduction of HIV PIs has revolutionized the treatment of HIV disease, reducing the opportunistic infections, esp. candidiasis.  The attenuation of candidal infections in HIV-infected individuals might not solely has not only resulted from improved immunol. status, but also as a result of direct inhibition of C. albicans Saps as well as the blockage of several biol. processes controlled by these proteolytic enzymes.  The present article will discuss the updates on the functional implications of HIV PIs on the development of candidiasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxxfn1gpzdZbVg90H21EOLACvtfcHk0liFm4U8w2sgFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFertLg%253D&md5=55fdb44e3f14cedeeecae3c5f7008180</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F138955713804484802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138955713804484802%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DA.%2BL.%2BS.%26aulast%3DBraga-Silva%26aufirst%3DL.%2BA.%26atitle%3DAspartic%2520Protease%2520Inhibitors%253A%2520Effective%2520Drugs%2520Against%2520the%2520Human%2520Fungal%2520Pathogen%2520Candida%2520Albicans%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26spage%3D155%26epage%3D162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sigrist, C. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gattiker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falquet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bairoch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bucher, P.</span></span> <span> </span><span class="NLM_article-title">PROSITE: a Documented Database Using Patterns and Profiles as Motif Descriptors</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1093/bib/3.3.265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fbib%2F3.3.265" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=12230035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsFOhtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=265-274&author=C.+J.+A.+Sigristauthor=L.+Ceruttiauthor=N.+Huloauthor=A.+Gattikerauthor=L.+Falquetauthor=M.+Pagniauthor=A.+Bairochauthor=P.+Bucher&title=PROSITE%3A+a+Documented+Database+Using+Patterns+and+Profiles+as+Motif+Descriptors&doi=10.1093%2Fbib%2F3.3.265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROSITE: A documented database using patterns and profiles as motif descriptors</span></div><div class="casAuthors">Sigrist, Christian J. A.; Cerutti, Lorenzo; Hulo, Nicolas; Gattiker, Alexandre; Falquet, Laurent; Pagni, Marco; Bairoch, Amos; Bucher, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">265-274</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1467-5463</span>.
    
            (<span class="NLM_cas:orgname">Henry Stewart Publications</span>)
        </div><div class="casAbstract">The main characteristics and the techniques of family and domain identification used by PROSITE are described.  Among the various databases dedicated to the identification protein families and domains, PROSITE is the first one created and has continuously evolved since.  PROSITE currently consists of a large collection of biol. meaningful motifs that are described as patterns or profiles, and linked to documentation briefly describing the protein family or domain they are designed to detect.  The close relation of PROSITE with the SWISS-PROT protein database allows the evaluation of the sensitivity and specificity of the FROSITE motifs and their periodic reviewing.  In return, PROSITE is used to help annotate SWISS-PROT entries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZjk2fEuPFM7Vg90H21EOLACvtfcHk0liFm4U8w2sgFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsFOhtbc%253D&md5=6385337d47f40fd1b16f63a0cc7b0320</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fbib%2F3.3.265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252F3.3.265%26sid%3Dliteratum%253Aachs%26aulast%3DSigrist%26aufirst%3DC.%2BJ.%2BA.%26aulast%3DCerutti%26aufirst%3DL.%26aulast%3DHulo%26aufirst%3DN.%26aulast%3DGattiker%26aufirst%3DA.%26aulast%3DFalquet%26aufirst%3DL.%26aulast%3DPagni%26aufirst%3DM.%26aulast%3DBairoch%26aufirst%3DA.%26aulast%3DBucher%26aufirst%3DP.%26atitle%3DPROSITE%253A%2520a%2520Documented%2520Database%2520Using%2520Patterns%2520and%2520Profiles%2520as%2520Motif%2520Descriptors%26jtitle%3DBriefings%2520Bioinf.%26date%3D2002%26volume%3D3%26spage%3D265%26epage%3D274%26doi%3D10.1093%2Fbib%2F3.3.265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Heijne, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, H.</span></span> <span> </span><span class="NLM_article-title">SignalP 4.0: Discriminating Signal Peptides from Transmembrane Regions</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">786</span>, <span class="refDoi"> DOI: 10.1038/nmeth.1701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1038%2Fnmeth.1701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=21959131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1CrtrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2011&pages=785-786&author=T.+N.+Petersenauthor=S.+Brunakauthor=G.+von+Heijneauthor=H.+Nielsen&title=SignalP+4.0%3A+Discriminating+Signal+Peptides+from+Transmembrane+Regions&doi=10.1038%2Fnmeth.1701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">SignalP 4.0: discriminating signal peptides from transmembrane regions</span></div><div class="casAuthors">Petersen, Thomas Nordahl; Brunak, Soeren; von Heijne, Gunnar; Nielsen, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">785-786</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">SignalP 4.0 is a purely neural network-based method designed to discriminate between signal peptides and transmembrane regions.  Evaluation of prediction performance of SignalP 4.0 showed that it was the best signal-peptide predictor for all three organism types, namely eukaryotes, Gram-pos. bacteria and Gram-neg. bacteria.  This comes at a price, however, because SignalP 4.0 was not in all cases as good as SignalP 3.0 according to cleavage-site sensitivity or signal-peptide correlation when there are no transmembrane proteins present.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoapAuMBGaJXbVg90H21EOLACvtfcHk0lj4yLv19ujC_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1CrtrbL&md5=543e761a8f7509a1cdc1209c1d08b6c9</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1701%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DT.%2BN.%26aulast%3DBrunak%26aufirst%3DS.%26aulast%3Dvon%2BHeijne%26aufirst%3DG.%26aulast%3DNielsen%26aufirst%3DH.%26atitle%3DSignalP%25204.0%253A%2520Discriminating%2520Signal%2520Peptides%2520from%2520Transmembrane%2520Regions%26jtitle%3DNat.%2520Methods%26date%3D2011%26volume%3D8%26spage%3D785%26epage%3D786%26doi%3D10.1038%2Fnmeth.1701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, B. M.</span></span> <span> </span><span class="NLM_article-title">Structure and Mechanism of the Pepsin-like Family of Aspartic Peptidases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4431</span>– <span class="NLM_lpage">4458</span>, <span class="refDoi"> DOI: 10.1021/cr010167q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr010167q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFWhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2002&pages=4431-4458&author=B.+M.+Dunn&title=Structure+and+Mechanism+of+the+Pepsin-like+Family+of+Aspartic+Peptidases&doi=10.1021%2Fcr010167q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and mechanism of the pepsin-like family of aspartic peptidases</span></div><div class="casAuthors">Dunn, Ben M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4431-4458</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Topics discussed include the cloning and expression of new aspartic endopeptidases, structural aspects of aspartic peptidases, the N-terminal prosegment region of aspartic peptidases, active site engineering, substrate and inhibitor design, and synthetic and naturally occurring inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU2C94Lnni_LVg90H21EOLACvtfcHk0lj4yLv19ujC_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFWhtLk%253D&md5=0e6b1f501111075667ea562d9d3ca132</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcr010167q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr010167q%26sid%3Dliteratum%253Aachs%26aulast%3DDunn%26aufirst%3DB.%2BM.%26atitle%3DStructure%2520and%2520Mechanism%2520of%2520the%2520Pepsin-like%2520Family%2520of%2520Aspartic%2520Peptidases%26jtitle%3DChem.%2520Rev.%26date%3D2002%26volume%3D102%26spage%3D4431%26epage%3D4458%26doi%3D10.1021%2Fcr010167q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarla, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.-R.</span></span> <span> </span><span class="NLM_article-title">HAART in HIV/AIDS Treatments: Future Trends</span>. <i>Infect. Disord.: Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.2174/1871526517666170505122800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.2174%2F1871526517666170505122800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28474549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVCqt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=15-22&author=D.-Y.+Luauthor=H.-Y.+Wuauthor=N.+S.+Yarlaauthor=B.+Xuauthor=J.+Dingauthor=T.-R.+Lu&title=HAART+in+HIV%2FAIDS+Treatments%3A+Future+Trends&doi=10.2174%2F1871526517666170505122800"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">HAART in HIV/AIDS Treatments: Future Trends</span></div><div class="casAuthors">Lu, Da-Yong; Wu, Hong-Ying; Yarla, Nagendra Sastry; Xu, Bin; Ding, Jian; Lu, Ting-Ren</div><div class="citationInfo"><span class="NLM_cas:title">Infectious Disorders: Drug Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-22</span>CODEN:
                <span class="NLM_cas:coden">IDDTAD</span>;
        ISSN:<span class="NLM_cas:issn">1871-5265</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  AIDS (acquired immune deficient syndrome) is a deadly human viral infectious disease caused by HIV (human immune-deficient virus) infection.  Almost every AIDS patient losses his/her life before mid 1990s.  AIDS was once the 1st disease killer in US (1993).  After one decade hard work, antiviral drug cocktails-high active anti-retroviral therapy (HAART) have been invented for almost all HIV infection treatments.  Due to the invention of HAART, 80-90% HIV/AIDS patients still effectively response to HAART for deadly AIDS episode controls and life saving.  Yet, this type of HIV therapeutics is incurable.  HIV/AIDS patients need to take HAART medications regularly and even life-long.  To counteract this therapeutic drawback, more revolutionary efforts (different angles of therapeutic modes/attempts) are urgently needed.  In this article, the major progresses and drawbacks of HIV/AIDS chemotherapy (HAART) to HIV/AIDS patients have been discussed.  Future trends (updating pathogenesis study, next generations of drug developments, new drug target discovery, different scientific disciplinary and so on) are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6WbPH3DgABrVg90H21EOLACvtfcHk0liaCb72-Uzcww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntVCqt78%253D&md5=26ffa17208e500e760b6d1dfde1ffcdc</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2174%2F1871526517666170505122800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1871526517666170505122800%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DD.-Y.%26aulast%3DWu%26aufirst%3DH.-Y.%26aulast%3DYarla%26aufirst%3DN.%2BS.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.-R.%26atitle%3DHAART%2520in%2520HIV%252FAIDS%2520Treatments%253A%2520Future%2520Trends%26jtitle%3DInfect.%2520Disord.%253A%2520Drug%2520Targets%26date%3D2018%26volume%3D18%26spage%3D15%26epage%3D22%26doi%3D10.2174%2F1871526517666170505122800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alteri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cozens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faessler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimkait, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazdins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poncioni, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roesel, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span> <span> </span><span class="NLM_article-title">CGP 53437, an Orally Bioavailable Inhibitor of Human Immunodeficiency Virus Type 1 Protease with Potent Antiviral Activity</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2087</span>– <span class="NLM_lpage">2092</span>, <span class="refDoi"> DOI: 10.1128/aac.37.10.2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FAAC.37.10.2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8257128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK2cXpsVygsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1993&pages=2087-2092&author=E.+Alteriauthor=G.+Boldauthor=R.+Cozensauthor=A.+Faesslerauthor=T.+Klimkaitauthor=M.+Langauthor=J.+Lazdinsauthor=B.+Poncioniauthor=J.+L.+Roeselauthor=P.+Schneider&title=CGP+53437%2C+an+Orally+Bioavailable+Inhibitor+of+Human+Immunodeficiency+Virus+Type+1+Protease+with+Potent+Antiviral+Activity&doi=10.1128%2Faac.37.10.2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">CGP 53437, an orally bioavailable inhibitor of human immunodeficiency virus type 1 protease with potent antiviral activity</span></div><div class="casAuthors">Alteri, E.; Bold, G.; Cozens, R.; Faessler, A.; Klimkait, T.; Lang, M.; Lazdins, J.; Poncioni, B.; Roesel, J. L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2087-92</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    </div><div class="casAbstract">CGP 53437 is a peptidomimetic inhibitor of human immunodeficiency virus type 1 proteinase contg. a hydroxyethylene isostere.  The compd. inhibited recombinant human immunodeficiency virus-1 proteinase with a Ki of 0.2 nM.  The inhibition const. vs. human cathepsin D and human cathepsin E was 4 nM.  Human pepsin and gastricsin were inhibited with Kis of 8 and 500 nM, resp., and human renin was inhibited with a Ki of 190 μM.  The replication of human immunodeficiency virus-1/LAV, human immunodeficiency virus-1/Z-84, and human immunodeficiency virus-1/pLAI was inhibited with a 90% ED of 0.1 μM in acutely infected MT-2 cells.  The 50% cytotoxic dose was 100 μM.  Similar antiviral activity was obsd. when the compd. was added up to 10 h after infection.  At the effective concn., processing of Gag precursor protein p55 was greatly reduced, confirming an action on the late stage of the virus life cycle, as expected.  The efficacy of the inhibitor was also demonstrated by using primary human peripheral blood lymphocytes infected with the human immunodeficiency virus-1/LAV strain, low-passage clin. isolates obtained from human immunodeficiency virus-1-seropos. individuals (including a zidovudine-resistant strain), and human immunodeficiency virus-2/ROD.  In these cells, CGP 53437 delayed the onset of human immunodeficiency virus replication in a dose-dependent fashion (substantial effects with concns. of ≥0.1 μM) as long as the inhibitor was maintained in the culture.  CGP 53437 was orally bioavailable in mice.  Concns. in plasma 10-fold in excess of the in vitro antiviral 90% ED could be sustained for several h after oral application of 120 mg/kg.  Therefore, CGP 53437 has the potential to be a therapeutically useful anti-human immunodeficiency virus agent for the treatment of AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_5zVOGNIqIrVg90H21EOLACvtfcHk0liaCb72-Uzcww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXpsVygsg%253D%253D&md5=9e0976269a28a2f995c850987e4f931d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FAAC.37.10.2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.37.10.2087%26sid%3Dliteratum%253Aachs%26aulast%3DAlteri%26aufirst%3DE.%26aulast%3DBold%26aufirst%3DG.%26aulast%3DCozens%26aufirst%3DR.%26aulast%3DFaessler%26aufirst%3DA.%26aulast%3DKlimkait%26aufirst%3DT.%26aulast%3DLang%26aufirst%3DM.%26aulast%3DLazdins%26aufirst%3DJ.%26aulast%3DPoncioni%26aufirst%3DB.%26aulast%3DRoesel%26aufirst%3DJ.%2BL.%26aulast%3DSchneider%26aufirst%3DP.%26atitle%3DCGP%252053437%252C%2520an%2520Orally%2520Bioavailable%2520Inhibitor%2520of%2520Human%2520Immunodeficiency%2520Virus%2520Type%25201%2520Protease%2520with%2520Potent%2520Antiviral%2520Activity%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1993%26volume%3D37%26spage%3D2087%26epage%3D2092%26doi%3D10.1128%2Faac.37.10.2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callebaut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stray, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavitt, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priskich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span> <span> </span><span class="NLM_article-title">In Vitro Characterization of GS-8374, a Novel Phosphonate-Containing Inhibitor of HIV-1 Protease with a Favorable Resistance Profile</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1366</span>– <span class="NLM_lpage">1376</span>, <span class="refDoi"> DOI: 10.1128/aac.01183-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FAAC.01183-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=21245449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFyntbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=1366-1376&author=C.+Callebautauthor=K.+Strayauthor=L.+Tsaiauthor=M.+Williamsauthor=Z.-Y.+Yangauthor=C.+Cannizzaroauthor=S.+A.+Leavittauthor=X.+Liuauthor=K.+Wangauthor=B.+P.+Murrayauthor=A.+Mulatoauthor=M.+Hatadaauthor=T.+Priskichauthor=N.+Parkinauthor=S.+Swaminathanauthor=W.+Leeauthor=G.-X.+Heauthor=L.+Xuauthor=T.+Cihlar&title=In+Vitro+Characterization+of+GS-8374%2C+a+Novel+Phosphonate-Containing+Inhibitor+of+HIV-1+Protease+with+a+Favorable+Resistance+Profile&doi=10.1128%2Faac.01183-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro characterization of GS-8374, a novel phosphonate-containing inhibitor of HIV-1 protease with a favorable resistance profile</span></div><div class="casAuthors">Callebaut, Christian; Stray, Kirsten; Tsai, Luong; Williams, Matt; Yang, Zheng-Yu; Cannizzaro, Carina; Leavitt, Stephanie A.; Liu, Xiaohong; Wang, Kelly; Murray, Bernard P.; Mulato, Andrew; Hatada, Marcos; Priskich, Tina; Parkin, Neil; Swaminathan, Swami; Lee, William; He, Gong-Xin; Xu, Lianhong; Cihlar, Tomas</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1366-1376</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">GS-8374 is a novel bis-tetrahydrofuran HIV-1 protease (PR) inhibitor (PI) with a unique diethylphosphonate moiety.  It was selected from a series of analogs contg. various di(alkyl)phosphonate substitutions connected via a linker to the para position of a P-1 Ph ring.  GS-8374 inhibits HIV-1 PR with high potency (Ki = 8.1 pM) and with no known effect on host proteases.  Kinetic and thermodn. anal. of GS-8374 binding to PR demonstrated an extremely slow off rate for the inhibitor and favorable contributions of both the enthalpic and entropic components to the total free binding energy.  GS-8374 showed potent antiretroviral activity in T-cell lines, primary CD4+ T cells (50% effective concn. [EC50] = 3.4 to 11.5 nM), and macrophages (EC50 = 25.5 nM) and exhibited low cytotoxicity in multiple human cell types.  The antiviral potency of GS-8374 was only moderately affected by human serum protein binding, and its combination with multiple approved antiretrovirals showed synergistic effects.  When it was tested in a PhenoSense assay against a panel of 24 patient-derived viruses with high-level PI resistance, GS-8374 showed lower mean EC50s and lower fold resistance than any of the clin. approved PIs.  Similar to other PIs, in vitro hepatic microsomal metab. of GS-8374 was efficiently blocked by ritonavir, suggesting a potential for effective pharmacokinetic boosting in vivo.  In summary, results from this broad in vitro pharmacol. profiling indicate that GS-8374 is a promising candidate to be further assessed as a new antiretroviral agent with potential for clin. efficacy in both treatment-naive and -experienced patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC6gBbIaR7irVg90H21EOLACvtfcHk0lh49HNeXJG_Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFyntbo%253D&md5=d4f8da97ca6d462349a282997d08b096</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1128%2FAAC.01183-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01183-10%26sid%3Dliteratum%253Aachs%26aulast%3DCallebaut%26aufirst%3DC.%26aulast%3DStray%26aufirst%3DK.%26aulast%3DTsai%26aufirst%3DL.%26aulast%3DWilliams%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DZ.-Y.%26aulast%3DCannizzaro%26aufirst%3DC.%26aulast%3DLeavitt%26aufirst%3DS.%2BA.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DMurray%26aufirst%3DB.%2BP.%26aulast%3DMulato%26aufirst%3DA.%26aulast%3DHatada%26aufirst%3DM.%26aulast%3DPriskich%26aufirst%3DT.%26aulast%3DParkin%26aufirst%3DN.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DG.-X.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DCihlar%26aufirst%3DT.%26atitle%3DIn%2520Vitro%2520Characterization%2520of%2520GS-8374%252C%2520a%2520Novel%2520Phosphonate-Containing%2520Inhibitor%2520of%2520HIV-1%2520Protease%2520with%2520a%2520Favorable%2520Resistance%2520Profile%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2011%26volume%3D55%26spage%3D1366%26epage%3D1376%26doi%3D10.1128%2Faac.01183-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furfine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaltenstein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St. Clair, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanlon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boone, L.</span></span> <span> </span><span class="NLM_article-title">In Vitro Antiviral Activity of the Novel, Tyrosyl-Based Human Immunodeficiency Virus (HIV) Type 1 Protease Inhibitor Brecanavir (GW640385) in Combination with Other Antiretrovirals and against a Panel of Protease Inhibitor-Resistant HIV</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3147</span>– <span class="NLM_lpage">3154</span>, <span class="refDoi"> DOI: 10.1128/aac.00401-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FAAC.00401-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=17620375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVais73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=3147-3154&author=R.+Hazenauthor=R.+Harveyauthor=R.+Ferrisauthor=C.+Craigauthor=P.+Yatesauthor=P.+Griffinauthor=J.+Millerauthor=I.+Kaldorauthor=J.+Rayauthor=V.+Samanoauthor=E.+Furfineauthor=A.+Spaltensteinauthor=M.+Haleauthor=R.+Tungauthor=M.+St.+Clairauthor=M.+Hanlonauthor=L.+Boone&title=In+Vitro+Antiviral+Activity+of+the+Novel%2C+Tyrosyl-Based+Human+Immunodeficiency+Virus+%28HIV%29+Type+1+Protease+Inhibitor+Brecanavir+%28GW640385%29+in+Combination+with+Other+Antiretrovirals+and+against+a+Panel+of+Protease+Inhibitor-Resistant+HIV&doi=10.1128%2Faac.00401-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity of the novel, tyrosyl-based human immunodeficiency virus (HIV) type 1 protease inhibitor brecanavir (GW640385) in combination with other antiretrovirals and against a panel of protease inhibitor-resistant HIV</span></div><div class="casAuthors">Hazen, Richard; Harvey, Robert; Ferris, Robert; Craig, Charles; Yates, Phillip; Griffin, Philip; Miller, John; Kaldor, Istvan; Ray, John; Samano, Vincente; Furfine, Eric; Spaltenstein, Andrew; Hale, Michael; Tung, Roger; St. Clair, Marty; Hanlon, Mary; Boone, Lawrence</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3147-3154</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Brecanavir, a novel tyrosyl-based arylsulfonamide, high-affinity, human immunodeficiency virus type 1 (HIV-1) protease inhibitor (PI), has been evaluated for anti-HIV activity in several in vitro assays.  Preclin. assessment of brecanavir indicated that this compd. potently inhibited HIV-1 in cell culture assays with 50% effective concns. (EC50s) of 0.2 to 0.53 nM and was equally active against HIV strains utilizing either the CXCR4 or CCR5 coreceptor, as was found with other PIs.  The presence of up to 40% human serum decreased the anti-HIV-1 activity of brecanavir by 5.2-fold, but under these conditions the compd. retained single-digit nanomolar EC50s.  When brecanavir was tested in combination with nucleoside reverse transcriptase inhibitors, the antiviral activity of brecanavir was synergistic with the effects of stavudine and additive to the effects of zidovudine, tenofovir, dideoxycytidine, didanosine, adefovir, abacavir, lamivudine, and emtricitabine.  Brecanavir was synergistic with the nonnucleoside reverse transcriptase inhibitor nevirapine or delavirdine and was additive to the effects of efavirenz.  In combination with other PIs, brecanavir was additive to the activities of indinavir, lopinavir, nelfinavir, ritonavir, amprenavir, saquinavir, and atazanavir.  Clin. HIV isolates from PI-experienced patients were evaluated for sensitivity to brecanavir and other PIs in a recombinant virus assay.  Brecanavir had a <5-fold increase in EC50s against 80% of patient isolates tested and had a greater mean in vitro potency than amprenavir, indinavir, lopinavir, atazanavir, tipranavir, and darunavir.  Brecanavir is by a substantial margin the most potent and broadly active antiviral agent among the PIs tested in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqplFzzXvQJ9rVg90H21EOLACvtfcHk0lgi-MR_IAY12g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVais73F&md5=a84fb4b36143fb6c9e92db4895ad1d9d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1128%2FAAC.00401-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00401-07%26sid%3Dliteratum%253Aachs%26aulast%3DHazen%26aufirst%3DR.%26aulast%3DHarvey%26aufirst%3DR.%26aulast%3DFerris%26aufirst%3DR.%26aulast%3DCraig%26aufirst%3DC.%26aulast%3DYates%26aufirst%3DP.%26aulast%3DGriffin%26aufirst%3DP.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DKaldor%26aufirst%3DI.%26aulast%3DRay%26aufirst%3DJ.%26aulast%3DSamano%26aufirst%3DV.%26aulast%3DFurfine%26aufirst%3DE.%26aulast%3DSpaltenstein%26aufirst%3DA.%26aulast%3DHale%26aufirst%3DM.%26aulast%3DTung%26aufirst%3DR.%26aulast%3DSt.%2BClair%26aufirst%3DM.%26aulast%3DHanlon%26aufirst%3DM.%26aulast%3DBoone%26aufirst%3DL.%26atitle%3DIn%2520Vitro%2520Antiviral%2520Activity%2520of%2520the%2520Novel%252C%2520Tyrosyl-Based%2520Human%2520Immunodeficiency%2520Virus%2520%2528HIV%2529%2520Type%25201%2520Protease%2520Inhibitor%2520Brecanavir%2520%2528GW640385%2529%2520in%2520Combination%2520with%2520Other%2520Antiretrovirals%2520and%2520against%2520a%2520Panel%2520of%2520Protease%2520Inhibitor-Resistant%2520HIV%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D3147%26epage%3D3154%26doi%3D10.1128%2Faac.00401-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sielecki, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayakawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carilli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewicki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M.</span></span> <span> </span><span class="NLM_article-title">Structure of Recombinant Human Renin, a Target for Cardiovascular-active Drugs, at 2.5 A resolution</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>243</i></span>,  <span class="NLM_fpage">1346</span>– <span class="NLM_lpage">1351</span>, <span class="refDoi"> DOI: 10.1126/science.2493678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1126%2Fscience.2493678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=2493678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL1MXhs12htbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=243&publication_year=1989&pages=1346-1351&author=A.+R.+Sieleckiauthor=K.+Hayakawaauthor=M.+Fujinagaauthor=M.+Murphyauthor=M.+Fraserauthor=A.+Muirauthor=C.+Carilliauthor=J.+Lewickiauthor=J.+Baxterauthor=M.+James&title=Structure+of+Recombinant+Human+Renin%2C+a+Target+for+Cardiovascular-active+Drugs%2C+at+2.5+A+resolution&doi=10.1126%2Fscience.2493678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of recombinant human renin, a target for cardiovascular-active drugs, at 2.5 Å resolution</span></div><div class="casAuthors">Sielecki, Anita R.; Hayakawa, Koto; Fujinaga, Masao; Murphy, Michael E. P.; Fraser, Marie; Muir, Alastaiar K.; Carilli, Cynthia T.; Lewicki, John A.; Baxter, John D.; James, Michael N. G.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">243</span>
        (<span class="NLM_cas:issue">4896</span>),
    <span class="NLM_cas:pages">1346-51</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The x-ray crystal structure of recombinant human renin was detd.  Mol. dynamics techniques that included crystallog. data as a restraint were used to improve an initial model based on porcine pepsinogen.  The present agreement factor for data at 8.0-2.5 Å is 0.236.  Some of the surface loops are poorly detd., and these disordered regions border a 30 Å wide solvent channel.  Comparison of renin with other aspartyl proteinases shows that, although the structural cores and active sites are highly conserved, surface residues, some of which are crit. for specificity, vary greatly (≤10 Å).  Knowledge of the actual structure, as opposed to the use of models based on related enzymes, should facilitate the design of renin inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfLdkSwz5asrVg90H21EOLACvtfcHk0lhjQlVREkoi5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhs12htbo%253D&md5=2ac6057c5421181be26cd23e55eb1c5a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.2493678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2493678%26sid%3Dliteratum%253Aachs%26aulast%3DSielecki%26aufirst%3DA.%2BR.%26aulast%3DHayakawa%26aufirst%3DK.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DMurphy%26aufirst%3DM.%26aulast%3DFraser%26aufirst%3DM.%26aulast%3DMuir%26aufirst%3DA.%26aulast%3DCarilli%26aufirst%3DC.%26aulast%3DLewicki%26aufirst%3DJ.%26aulast%3DBaxter%26aufirst%3DJ.%26aulast%3DJames%26aufirst%3DM.%26atitle%3DStructure%2520of%2520Recombinant%2520Human%2520Renin%252C%2520a%2520Target%2520for%2520Cardiovascular-active%2520Drugs%252C%2520at%25202.5%2520A%2520resolution%26jtitle%3DScience%26date%3D1989%26volume%3D243%26spage%3D1346%26epage%3D1351%26doi%3D10.1126%2Fscience.2493678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, M. N. G.</span>; <span class="NLM_string-name">Sielecki, A. R.</span></span> <span> </span><span class="NLM_article-title">Aspartic Proteinases and their Catalytic Pathway</span>. In  <i>Biological Macromolecules and Assemblies, Vol. 3 Active Sites of Enzymes</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurnak, F. A.</span>, <span class="NLM_string-name">McPherson, A.</span></span>, Eds.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1987</span>; pp  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">482</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1987&pages=413-482&author=M.+N.+G.+James&author=A.+R.+Sieleckiauthor=F.+A.+Jurnak&author=A.+McPherson&title=Biological+Macromolecules+and+Assemblies%2C+Vol.+3+Active+Sites+of+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DM.%2BN.%2BG.%26atitle%3DAspartic%2520Proteinases%2520and%2520their%2520Catalytic%2520Pathway%26btitle%3DBiological%2520Macromolecules%2520and%2520Assemblies%252C%2520Vol.%25203%2520Active%2520Sites%2520of%2520Enzymes%26aulast%3DJurnak%26aufirst%3DF.%2BA.%26pub%3DWiley%26date%3D1987%26spage%3D413%26epage%3D482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M. N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Structural Evidence for Gene Duplication in the Evolution of the Acid Proteases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">618</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1038/271618a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1038%2F271618a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=24179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaE1cXhvVGhtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1978&pages=618-621&author=J.+Tangauthor=M.+N.+G.+Jamesauthor=I.+N.+Hsuauthor=J.+A.+Jenkinsauthor=T.+L.+Blundell&title=Structural+Evidence+for+Gene+Duplication+in+the+Evolution+of+the+Acid+Proteases&doi=10.1038%2F271618a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structural evidence for gene duplication in the evolution of the acid proteases</span></div><div class="casAuthors">Tang, J.; James, M. N. G.; Hsu, I. N.; Jenkins, J. A.; Blundell, T. L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">5646</span>),
    <span class="NLM_cas:pages">618-21</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">X-ray studies of acid proteases (I) indicated a bilobal structure with a well-defined active-site cleft.  An intramol. 2-fold symmetry axis related 2 topol. similar domains and the active-site residues.  I may have evolved by gene duplication of an ancestral protein of ∼150 residues, having a fold similar to that of one lobe of pepsin.  The 2 subunits may then have evolved divergently so that the carboxyl groups of the 2 aspartyl residues developed different pKs and a mechanism of hydrolytic cleavage.  Gene fusion would be selectively advantageous in assuring the correct assocn. of dissimilar subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopTbDD0MPpC7Vg90H21EOLACvtfcHk0lg1QA52TB6fsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXhvVGhtbg%253D&md5=cb6501707d35e0fbecb6921ad2c0c590</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2F271618a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F271618a0%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DJames%26aufirst%3DM.%2BN.%2BG.%26aulast%3DHsu%26aufirst%3DI.%2BN.%26aulast%3DJenkins%26aufirst%3DJ.%2BA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520Evidence%2520for%2520Gene%2520Duplication%2520in%2520the%2520Evolution%2520of%2520the%2520Acid%2520Proteases%26jtitle%3DNature%26date%3D1978%26volume%3D271%26spage%3D618%26epage%3D621%26doi%3D10.1038%2F271618a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sewell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLachlan, A. D.</span></span> <span> </span><span class="NLM_article-title">Four-fold Structural Repeat in the Acid Proteases</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1016/0005-2795(79)90194-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2F0005-2795%2879%2990194-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=44681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaE1MXlslSkur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=1979&pages=24-31&author=T.+L.+Blundellauthor=B.+T.+Sewellauthor=A.+D.+McLachlan&title=Four-fold+Structural+Repeat+in+the+Acid+Proteases&doi=10.1016%2F0005-2795%2879%2990194-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Four-fold structural repeat in the acid proteases</span></div><div class="casAuthors">Blundell, T. L.; Sewell, B. T.; McLachlan, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Protein Structure</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-31</span>CODEN:
                <span class="NLM_cas:coden">BBPTBH</span>;
        ISSN:<span class="NLM_cas:issn">0005-2795</span>.
    </div><div class="casAbstract">The observation that the acid proteases contain 2 structurally equiv. lobes related by a dyad through the active center was extended to show that in endothiapepsin (Endothia acid protease) each lobe contains 2 similar halves related by a further local dyad.  In lobe 1, 22 pairs of α-carbons are equiv. with a root mean square deviation of 1.92 Å.  In lobe 2, 17 pairs match within 2.31 Å.  Convergent evolution or gene quadruplication may have occurred.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGmUx9-sOCG7Vg90H21EOLACvtfcHk0lg1QA52TB6fsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXlslSkur4%253D&md5=b8596d402106e37ea343b7812738d2b6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2F0005-2795%2879%2990194-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2795%252879%252990194-6%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DSewell%26aufirst%3DB.%2BT.%26aulast%3DMcLachlan%26aufirst%3DA.%2BD.%26atitle%3DFour-fold%2520Structural%2520Repeat%2520in%2520the%2520Acid%2520Proteases%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1979%26volume%3D580%26spage%3D24%26epage%3D31%26doi%3D10.1016%2F0005-2795%2879%2990194-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, M. N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sielecki, A. R.</span></span> <span> </span><span class="NLM_article-title">Structure and refinement of penicillopepsin at 1.8 Å resolution</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1016/0022-2836(83)90008-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2F0022-2836%2883%2990008-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=6341600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL3sXht1Wqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=1983&pages=299-361&author=M.+N.+G.+Jamesauthor=A.+R.+Sielecki&title=Structure+and+refinement+of+penicillopepsin+at+1.8+%C3%85+resolution&doi=10.1016%2F0022-2836%2883%2990008-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and refinement of penicillopepsin at 1.8 Å resolution</span></div><div class="casAuthors">James, Michael N. G.; Sielecki, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">299-361</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Penicillopepsin (I), the aspartyl protease from Penicillium janthinellum, had its mol. structure refined by a restrained-parameter least-squares procedure at 1.8 Å resoln.  The av. at. thermal vibration parameter (B) for the atoms of I is 14.5 Å2.  One detg. factor of this low av. B value is the large central hydrophobic core, in which there are 2 prominent clusters of arom. residues, one of 9, the other of 7 residues.  The N- and C-terminal domains of I display an approx. 2-fold symmetry: 70 residue pairs are topol. equiv., related by a rotation of 177° and a translation of 1.2 Å.  Anal. of the secondary structural features of the mol. reveals nonlinear H bonding.  The av. at. B-factors of the residues in the active site region of the mol. are 5-8 Å2, confirming the proposal that conformational mobility of the active site residues has no role in the enzymic mechanism.  However, conformational mobility of neighboring regions of the mol., e.g., the flap contg. tyrosine-75, is verified by the high B-factors for those residues.  The positions of 319 solvent sites/asym. unit were selected from difference electron d. maps and refined.  Thirteen were classified as internal, and several of these may have key roles during catalysis.  The intermol. packing involves many protein-protein contacts (66 residues) with a large no. of solvent mols. involved in bridging between polar residues at the contact surface.  I mols. resembled an approx. hexagonal close-packing of spheres with each mol. having 12 nearest neighbors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd_uOSJ-pTlLVg90H21EOLACvtfcHk0lj-x6nchkAw8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXht1Wqu7o%253D&md5=0e042c2bf7fb1475465df0aed6dc111f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2883%2990008-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252883%252990008-6%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DM.%2BN.%2BG.%26aulast%3DSielecki%26aufirst%3DA.%2BR.%26atitle%3DStructure%2520and%2520refinement%2520of%2520penicillopepsin%2520at%25201.8%2520%25C3%2585%2520resolution%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1983%26volume%3D163%26spage%3D299%26epage%3D361%26doi%3D10.1016%2F0022-2836%2883%2990008-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenström, P.</span></span> <span> </span><span class="NLM_article-title">Dali Server: Conservation Mapping in 3D</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">W545</span>– <span class="NLM_lpage">W549</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fnar%2Fgkq366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=20457744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVSqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W545-W549&author=L.+Holmauthor=P.+Rosenstr%C3%B6m&title=Dali+Server%3A+Conservation+Mapping+in+3D&doi=10.1093%2Fnar%2Fgkq366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Dali server: conservation mapping in 3D</span></div><div class="casAuthors">Holm, Liisa; Rosenstrom, Paivi</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W545-W549</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Our web site (http://ekhidna.biocenter.helsinki.fi/daliserver) runs the Dali program for protein structure comparison.  The web site consists of three parts: (i) the Dali server compares newly solved structures against structures in the Protein Data Bank (PDB), (ii) the Dali database allows browsing precomputed structural neighborhoods and (iii) the pairwise comparison generates suboptimal alignments for a pair of structures.  Each part has its own query form and a common format for the results page.  The inputs are either PDB identifiers or novel structures uploaded by the user.  The results pages are hyperlinked to aid interactive anal.  The web interface is simple and easy to use.  The key purpose of interactive anal. is to check whether conserved residues line up in multiple structural alignments and how conserved residues and ligands cluster together in multiple structure superimpositions.  In favorable cases, protein structure comparison can lead to evolutionary discoveries not detected by sequence anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdTA8Qcz4NHLVg90H21EOLACvtfcHk0lini-5rJrB3vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVSqsr4%253D&md5=b7832262058d6a4f7d362f4ce565a666</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq366%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DRosenstr%25C3%25B6m%26aufirst%3DP.%26atitle%3DDali%2520Server%253A%2520Conservation%2520Mapping%2520in%25203D%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW545%26epage%3DW549%26doi%3D10.1093%2Fnar%2Fgkq366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laakso, L. M.</span></span> <span> </span><span class="NLM_article-title">Dali Server Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">355</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fnar%2Fgkw357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=351-355&author=L.+Holmauthor=L.+M.+Laakso&title=Dali+Server+Update&doi=10.1093%2Fnar%2Fgkw357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Dali server update</span></div><div class="casAuthors">Holm, Liisa; Laakso, Laura M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W351-W355</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Dali server is a network service for comparing protein structures in 3D.  In favorable cases, comparing 3D structures may reveal biol. interesting similarities that are not detectable by comparing sequences.  The Dali server has been running in various places for over 20 years and is used routinely by crystallographers on newly solved structures.  The latest update of the server provides enhanced analytics for the study of sequence and structure conservation.  The server performs three types of structure comparisons: (i) Protein Data Bank (PDB) search compares one query structure against those in the PDB and returns a list of similar structures; (ii) pairwise comparison compares one query structure against a list of structures specified by the user; and (iii) all against all structure comparison returns a structural similarity matrix, a dendrogram and a multidimensional scaling projection of a set of structures specified by the user.  Structural superimpositions are visualized using the Java-free WebGL viewer PV.  The structural alignment view is enhanced by sequence similarity searches against Uniprot.  The combined structure-sequence alignment information is compressed to a stack of aligned sequence logos.  In the stack, each structure is structurally aligned to the query protein and represented by a sequence logo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9jpe1HGTYrLVg90H21EOLACvtfcHk0lgQbs5_8uVizA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrrO&md5=cd2b5fabf6a01a4b35f350342ae7ddc2</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw357%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DLaakso%26aufirst%3DL.%2BM.%26atitle%3DDali%2520Server%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D351%26epage%3D355%26doi%3D10.1093%2Fnar%2Fgkw357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Navia, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, P. M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeever, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leu, C.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimbach, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herber, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigal, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darke, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, J. P.</span></span> <span> </span><span class="NLM_article-title">Three-dimensional Structure of Aspartyl Protease from Human Immunodeficiency Virus HIV-1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>337</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1038/337615a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1038%2F337615a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=2645523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL1MXhs12hsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=337&publication_year=1989&pages=615-620&author=M.+A.+Naviaauthor=P.+M.+D.+Fitzgeraldauthor=B.+M.+McKeeverauthor=C.-T.+Leuauthor=J.+C.+Heimbachauthor=W.+K.+Herberauthor=I.+S.+Sigalauthor=P.+L.+Darkeauthor=J.+P.+Springer&title=Three-dimensional+Structure+of+Aspartyl+Protease+from+Human+Immunodeficiency+Virus+HIV-1&doi=10.1038%2F337615a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Three-dimensional structure of aspartyl protease from human immunodeficiency virus HIV-1</span></div><div class="casAuthors">Navia, Manuel A.; Fitzgerald, Paula M. D.; McKeever, Brian M.; Leu, Chih Tai; Heimbach, Jill C.; Herber, Wayne K.; Sigal, Irving S.; Darke, Paul L.; Springer, James P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">6208</span>),
    <span class="NLM_cas:pages">615-20</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The crystal structure of the protease of the human immunodeficiency virus type 1 (HIV-1), which releases structural proteins and enzymes from viral polyprotein products, has been detd. to 3 Å resoln.  Large regions of the protease dimer, including the active site, have structural homol. to the family of microbial aspartyl proteases.  The structure suggests a mechanism for the autoproteolytic release of protease and a role in the control of virus maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrORd1OnAtyRrVg90H21EOLACvtfcHk0lgQbs5_8uVizA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhs12hsLs%253D&md5=7cfe3d1c0f65a7b0c6280ac860c68781</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2F337615a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F337615a0%26sid%3Dliteratum%253Aachs%26aulast%3DNavia%26aufirst%3DM.%2BA.%26aulast%3DFitzgerald%26aufirst%3DP.%2BM.%2BD.%26aulast%3DMcKeever%26aufirst%3DB.%2BM.%26aulast%3DLeu%26aufirst%3DC.-T.%26aulast%3DHeimbach%26aufirst%3DJ.%2BC.%26aulast%3DHerber%26aufirst%3DW.%2BK.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DDarke%26aufirst%3DP.%2BL.%26aulast%3DSpringer%26aufirst%3DJ.%2BP.%26atitle%3DThree-dimensional%2520Structure%2520of%2520Aspartyl%2520Protease%2520from%2520Human%2520Immunodeficiency%2520Virus%2520HIV-1%26jtitle%3DNature%26date%3D1989%26volume%3D337%26spage%3D615%26epage%3D620%26doi%3D10.1038%2F337615a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swindells, M. B.</span></span> <span> </span><span class="NLM_article-title">LigPlot+: Multiple Ligand-protein Interaction Diagrams for Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2778</span>– <span class="NLM_lpage">2786</span>, <span class="refDoi"> DOI: 10.1021/ci200227u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200227u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2778-2786&author=R.+A.+Laskowskiauthor=M.+B.+Swindells&title=LigPlot%2B%3A+Multiple+Ligand-protein+Interaction+Diagrams+for+Drug+Discovery&doi=10.1021%2Fci200227u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span></div><div class="casAuthors">Laskowski, Roman A.; Swindells, Mark B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2778-2786</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe a graphical system for automatically generating multiple 2D diagrams of ligand-protein interactions from 3D coordinates.  The diagrams portray the hydrogen-bond interaction patterns and hydrophobic contacts between the ligand(s) and the main-chain or side-chain elements of the protein.  The system is able to plot, in the same orientation, related sets of ligand-protein interactions.  This facilitates popular research tasks, such as analyzing a series of small mols. binding to the same protein target, a single ligand binding to homologous proteins, or the completely general case where both protein and ligand change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYuWvYH7Nel7Vg90H21EOLACvtfcHk0lgd6mFJ-Vnq5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP&md5=576e59ad4f0c4304d01c6c80301ecf39</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci200227u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200227u%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DSwindells%26aufirst%3DM.%2BB.%26atitle%3DLigPlot%252B%253A%2520Multiple%2520Ligand-protein%2520Interaction%2520Diagrams%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2778%26epage%3D2786%26doi%3D10.1021%2Fci200227u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, A. J.</span>; <span class="NLM_string-name">Rawlings, N. D.</span>; <span class="NLM_string-name">Woessner, J. F.</span></span> <i>Handbook of Proteolytic Enzymes</i>; <span class="NLM_publisher-name">Elsevier Academic Press</span>: <span class="NLM_publisher-loc">Amsterdam</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=A.+J.+Barrett&author=N.+D.+Rawlings&author=J.+F.+Woessner&title=Handbook+of+Proteolytic+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DA.%2BJ.%26btitle%3DHandbook%2520of%2520Proteolytic%2520Enzymes%26pub%3DElsevier%2520Academic%2520Press%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kisanuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span> <span> </span><span class="NLM_article-title">KNI-102, a Novel Tripeptide HIV Protease Inhibitor Containing Allophenylnorstatine as a Transition-state Mimic</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">3088</span>– <span class="NLM_lpage">3090</span>, <span class="refDoi"> DOI: 10.1248/cpb.39.3088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1248%2Fcpb.39.3088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1799953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK38XhsVegsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1991&pages=3088-3090&author=T.+Mimotoauthor=J.+Imaiauthor=S.+Tanakaauthor=N.+Hattoriauthor=S.+Kisanukiauthor=K.+Akajiauthor=Y.+Kiso&title=KNI-102%2C+a+Novel+Tripeptide+HIV+Protease+Inhibitor+Containing+Allophenylnorstatine+as+a+Transition-state+Mimic&doi=10.1248%2Fcpb.39.3088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">KNI-102, a novel tripeptide HIV protease inhibitor containing allophenylnorstatine as a transition-state mimic</span></div><div class="casAuthors">Mimoto, Tsutomu; Imai, Junya; Tanaka, Shigeki; Hattori, Naoko; Kisanuki, Sumitsugu; Akaji, Kenichi; Kiso, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3088-90</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    </div><div class="casAbstract">HIV-1 protease inhibitors contg. allophenylnorstatine[Apns; (2S,3S)-3-amino-2-hydroxy-4-phenylbutyric acid]-Pro (syn diastereomer) as a transition-state mimic were established to be potent and highly selective.  Z-Asn-Apns-Pro-NHBut (KNI-102) is the only tripeptide exhibiting substantial anti-HIV activity and may be of min. size for potent, selective inhibition of HIV protease.  Ready availability due to its simple chem. structure and stability should make it valuable for studies of the development of metabolically stable anti-AIDS drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3IUEjcsc-gLVg90H21EOLACvtfcHk0lglhOUFWFE7BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhsVegsbw%253D&md5=3e32a98a7b2bfe77f33897b23349f1df</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1248%2Fcpb.39.3088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.39.3088%26sid%3Dliteratum%253Aachs%26aulast%3DMimoto%26aufirst%3DT.%26aulast%3DImai%26aufirst%3DJ.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DHattori%26aufirst%3DN.%26aulast%3DKisanuki%26aufirst%3DS.%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DKiso%26aufirst%3DY.%26atitle%3DKNI-102%252C%2520a%2520Novel%2520Tripeptide%2520HIV%2520Protease%2520Inhibitor%2520Containing%2520Allophenylnorstatine%2520as%2520a%2520Transition-state%2520Mimic%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1991%26volume%3D39%26spage%3D3088%26epage%3D3090%26doi%3D10.1248%2Fcpb.39.3088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hradilek, M.</span>; <span class="NLM_string-name">Rinnová, M.</span>; <span class="NLM_string-name">Bařinka, C.</span>; <span class="NLM_string-name">Souček, M.</span>; <span class="NLM_string-name">Konvalinka, J.</span></span> <span> </span><span class="NLM_article-title">Analysis of Substrate Specificity of HIV Protease Species</span>. In  <i>Peptides for the New Millennium: Proceedings of the 16th American Peptide Symposium June 26–July 1, 1999, Minneapolis, Minnesota, U.S.A.</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, G. B.</span>, <span class="NLM_string-name">Tam, J. P.</span>, <span class="NLM_string-name">Barany, G.</span></span>, Eds.; <span class="NLM_publisher-name">Springer Netherlands</span>: <span class="NLM_publisher-loc">Dordrecht</span>, <span class="NLM_year">2002</span>; pp  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">475</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&pages=474-475&author=M.+Hradilek&author=M.+Rinnov%C3%A1&author=C.+Ba%C5%99inka&author=M.+Sou%C4%8Dek&author=J.+Konvalinkaauthor=G.+B.+Fields&author=J.+P.+Tam&author=G.+Barany&title=Analysis+of+Substrate+Specificity+of+HIV+Protease+Species"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DHradilek%26aufirst%3DM.%26atitle%3DAnalysis%2520of%2520Substrate%2520Specificity%2520of%2520HIV%2520Protease%2520Species%26jtitle%3DPeptides%2520for%2520the%2520New%2520Millennium%253A%2520Proceedings%2520of%2520the%252016th%2520American%2520Peptide%2520Symposium%2520June%252026%25E2%2580%2593July%25201%252C%25201999%252C%2520Minneapolis%252C%2520Minnesota%252C%2520U.S.A.%26aulast%3DFields%26aufirst%3DG.%2BB.%26pub%3DSpringer%2520Netherlands%26date%3D2002%26spage%3D474%26epage%3D475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dreyer, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaszek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandler, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakhoury, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magaard, V. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickler, J. E.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Human Immunodeficiency Virus 1 Protease In Vitro: Rational Design of Substrate Analogue Inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">9752</span>– <span class="NLM_lpage">9756</span>, <span class="refDoi"> DOI: 10.1073/pnas.86.24.9752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1073%2Fpnas.86.24.9752" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=2690072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3cXktVSitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=9752-9756&author=G.+B.+Dreyerauthor=B.+W.+Metcalfauthor=T.+A.+Tomaszekauthor=T.+J.+Carrauthor=A.+C.+Chandlerauthor=L.+Hylandauthor=S.+A.+Fakhouryauthor=V.+W.+Magaardauthor=M.+L.+Mooreauthor=J.+E.+Strickler&title=Inhibition+of+Human+Immunodeficiency+Virus+1+Protease+In+Vitro%3A+Rational+Design+of+Substrate+Analogue+Inhibitors&doi=10.1073%2Fpnas.86.24.9752"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of human immunodeficiency virus 1 protease in vitro:  rational design of substrate analog inhibitors</span></div><div class="casAuthors">Dreyer, Geoffrey B.; Metcalf, Brian W.; Tomaszek, Thaddeus A., Jr.; Carr, Thomas J.; Chandler, Arthur C., III; Hyland, Lawrence; Fakhoury, Stephen A.; Magaard, Victoria W.; Moore, Michael L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9752-6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Inhibitors of the protease from human immunodeficiency virus 1 (HIV-1) were designed, synthesized, and kinetically characterized.  Analogs of a heptapeptide substrate of HIV-1 protease with sequence similar to the p17-p24 cleavage site in the natural substrate, Pr55gag, were synthesized in which the scissile dipeptide bond was replaced with bonds from 6 categories of stable mimics of an aspartic proteolysis transition state of intermediate.  These mimics included an analog of statine, hydroxyethylene isosteres, 2 categories of phosphinic acids, a reduced amide isostere, and an α,α-difluoroketone.  The resulting peptide analogs were linear-competitive inhibitors of purified recombinant HIV-1 protease with inhibition consts. ranging from 18 nM to 40 μM depending on the type of inhibitor.  A truncated inhibitor, an analog of a hexapeptide, retained full inhibitory potency.  The most potent inhibitors, contg. the hydroxyethylene isostere, effectively blocked the proteolytic processing of a recombinant form of Pr55gag by HIV-1 protease in a cell-free assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrini6UHzRXwLVg90H21EOLACvtfcHk0liYV753iMGOUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktVSitrw%253D&md5=0e40a620c040dd6b2a322bb077fdc13c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1073%2Fpnas.86.24.9752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.86.24.9752%26sid%3Dliteratum%253Aachs%26aulast%3DDreyer%26aufirst%3DG.%2BB.%26aulast%3DMetcalf%26aufirst%3DB.%2BW.%26aulast%3DTomaszek%26aufirst%3DT.%2BA.%26aulast%3DCarr%26aufirst%3DT.%2BJ.%26aulast%3DChandler%26aufirst%3DA.%2BC.%26aulast%3DHyland%26aufirst%3DL.%26aulast%3DFakhoury%26aufirst%3DS.%2BA.%26aulast%3DMagaard%26aufirst%3DV.%2BW.%26aulast%3DMoore%26aufirst%3DM.%2BL.%26aulast%3DStrickler%26aufirst%3DJ.%2BE.%26atitle%3DInhibition%2520of%2520Human%2520Immunodeficiency%2520Virus%25201%2520Protease%2520In%2520Vitro%253A%2520Rational%2520Design%2520of%2520Substrate%2520Analogue%2520Inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1989%26volume%3D86%26spage%3D9752%26epage%3D9756%26doi%3D10.1073%2Fpnas.86.24.9752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stehliková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregorová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souček, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreánsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fábrys, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Štrop, P.</span></span> <span> </span><span class="NLM_article-title">Reduced-bond Tight-binding Inhibitors of HIV-1 Protease Fine Tuning of the Enzyme Subsite Specificity</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/0014-5793(92)80010-e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2F0014-5793%2892%2980010-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1544426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK38XktVShtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=1992&pages=9-13&author=J.+Urbanauthor=J.+Konvalinkaauthor=J.+Stehlikov%C3%A1author=E.+Gregorov%C3%A1author=P.+Majerauthor=M.+Sou%C4%8Dekauthor=M.+Andre%C3%A1nskyauthor=M.+F%C3%A1brysauthor=P.+%C5%A0trop&title=Reduced-bond+Tight-binding+Inhibitors+of+HIV-1+Protease+Fine+Tuning+of+the+Enzyme+Subsite+Specificity&doi=10.1016%2F0014-5793%2892%2980010-e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Reduced-bond tight-binding inhibitors of HIV-1 protease.  Fine tuning of the enzyme subsite specificity</span></div><div class="casAuthors">Urban, Jan; Konvalinka, Jan; Stehlikova, Jana; Gregorova, Eva; Majer, Pavel; Soucek, Milan; Andreansky, Martin; Fabry, Milan; Strop, Petr</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-13</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    </div><div class="casAbstract">Truncation of a peptide substrate in the N-terminus and replacement of its scissile amide bone with a non-cleavable reduced bond results in a potent inhibitor of HIV-1 protease.  A series of such inhibitors has been synthesized, and S2-S3' subsites of the protease binding cleft mapped.  The S2 pocket requires bulky Boc or PIV groups, large arom. Phe residues are preferred in P1 and P1' and Glu in P2'.  The S3' pocket prefers Phe over small Ala or Val.  Introduction of a Glu residue into the P2' position yields a tight-binding inhibitor of HIV-1 protease, Boc-Phe-[CH2-NH]-Phe-Glu-Phe-OMe, with a subnanomolar inhibition const.  The relevant peptide derived from the same amino acid sequence binds to the protease with a Ki of 110 nM, thus still demonstrating a good fit of the amino acid residues into the protease binding pockets and also the importance of the flexibility of P1-P1' linkage for proper binding.  A new type of peptide bond mimetic, N-hydroxylamine-CH2-N(OH)-, has been synthesized.  Binding of hydroxylamino inhibitor of HIV-1 protease is further improved with respect to reduced-bond inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqh234jOpbX7Vg90H21EOLACvtfcHk0lhugW9VePbAiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XktVShtbc%253D&md5=06567fdfb5a1e4de897f41bc1f98a275</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2F0014-5793%2892%2980010-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-5793%252892%252980010-E%26sid%3Dliteratum%253Aachs%26aulast%3DUrban%26aufirst%3DJ.%26aulast%3DKonvalinka%26aufirst%3DJ.%26aulast%3DStehlikov%25C3%25A1%26aufirst%3DJ.%26aulast%3DGregorov%25C3%25A1%26aufirst%3DE.%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DSou%25C4%258Dek%26aufirst%3DM.%26aulast%3DAndre%25C3%25A1nsky%26aufirst%3DM.%26aulast%3DF%25C3%25A1brys%26aufirst%3DM.%26aulast%3D%25C5%25A0trop%26aufirst%3DP.%26atitle%3DReduced-bond%2520Tight-binding%2520Inhibitors%2520of%2520HIV-1%2520Protease%2520Fine%2520Tuning%2520of%2520the%2520Enzyme%2520Subsite%2520Specificity%26jtitle%3DFEBS%2520Lett.%26date%3D1992%26volume%3D298%26spage%3D9%26epage%3D13%26doi%3D10.1016%2F0014-5793%2892%2980010-e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rich, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J. V. N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houseman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malkovsky, M.</span></span> <span> </span><span class="NLM_article-title">New Hydroxyethylamine HIV Protease Inhibitors that Suppress Viral Replication</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">3803</span>– <span class="NLM_lpage">3812</span>, <span class="refDoi"> DOI: 10.1021/jm00099a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00099a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK38XlvFags7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1992&pages=3803-3812&author=D.+H.+Richauthor=J.+V.+N.+V.+Prasadauthor=C.+Q.+Sunauthor=J.+Greenauthor=R.+Muellerauthor=K.+Housemanauthor=D.+MacKenzieauthor=M.+Malkovsky&title=New+Hydroxyethylamine+HIV+Protease+Inhibitors+that+Suppress+Viral+Replication&doi=10.1021%2Fjm00099a008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">New hydroxyethylamine HIV protease inhibitors that suppress viral replication</span></div><div class="casAuthors">Rich, Daniel H.; Prasad, J. V. N. Vara; Sun, Chong Qing; Green, Jeremy; Mueller, Richard; Houseman, Kathryn; MacKenzie, Debra; Malkovsky, Miroslav</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3803-12</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The synthesis of analogs of Ac-Ser-Leu-Asn-[Phe-HEA-Pro]-Ile-Val-OMe (JG-365; Phe-HEA-Pro = hydroxyethylamine transition state analog I), a tight-binding inhibitor of human immunodeficiency virus protease (HIVP), are reported.  Systematic modification of the P3 and P3' regions of the inhibitors has led to smaller HIVP inhibitors that inhibit viral replication in HIV-infected and simian immunodeficiency virus (SIV)-infected cell cultures.  Six aliph. and arom. derivs. were prepd. by replacing residues in the P3 regions of Boc-Leu-Asn-[Phe-HEA-Pro]-Ile-Val-OMe (Boc = Me3CO2C).  Arom. side chains at P3 gave better inhibitors than aliph. side chains.  The better inhibitors in this series contained a β-naphthylalanine or a biphenyl unit at P3.  A second series of HIVP inhibitors were obtained by converting the P3 group into acyl groups.  R-Asn-[Phe-HEA-Pro]-Ile-Phe-OMe (R = PhCH2O2C, 2-quinolinylcarbonyl) are potent HIVP inhibitors with Ki values equal to 1.0 and 0.1 nM, resp.  The inhibition consts. were det. by using the continuous fluorometric assay developed by M. V. Toth and G. R. Marshall (1990).  The activities of the protease inhibitors for inhibition of SIV replication were detd. in vitro using CEMx174 cells.  Inhibition of HIV infection was detd. essentially as reported by R. Pauwels, et. al. (1988).  The anti-HIV assay was carried out in culture using CEM cells (a CD4+ lymphocyte line) infected with virus strain HTLV-IIIb with a multiplicity of infection of 0.1.  Several analogs inhibited the cytopathic effect at concns. of 0.1-0.8 μg/mL.  These results establish that good inhibitors of HIV protease that inhibit viral replication in infected lymphocytes in in vitro cell assays can be obtained from JG-365 when the Ac-Ser-Leu unit is replaced by arom. acyl derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFgN6BYEYXb7Vg90H21EOLACvtfcHk0lgHWuwokXrdRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlvFags7Y%253D&md5=4fd3643c608b8ebdff13563757868d23</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm00099a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00099a008%26sid%3Dliteratum%253Aachs%26aulast%3DRich%26aufirst%3DD.%2BH.%26aulast%3DPrasad%26aufirst%3DJ.%2BV.%2BN.%2BV.%26aulast%3DSun%26aufirst%3DC.%2BQ.%26aulast%3DGreen%26aufirst%3DJ.%26aulast%3DMueller%26aufirst%3DR.%26aulast%3DHouseman%26aufirst%3DK.%26aulast%3DMacKenzie%26aufirst%3DD.%26aulast%3DMalkovsky%26aufirst%3DM.%26atitle%3DNew%2520Hydroxyethylamine%2520HIV%2520Protease%2520Inhibitors%2520that%2520Suppress%2520Viral%2520Replication%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1992%26volume%3D35%26spage%3D3803%26epage%3D3812%26doi%3D10.1021%2Fjm00099a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rinnová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hradilek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souček, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimkait, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span> <span> </span><span class="NLM_article-title">A Picomolar Inhibitor of Resistant Strains of Human Immunodeficiency Virus Protease Identified by a Combinatorial Approach</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>382</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1006/abbi.2000.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1006%2Fabbi.2000.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=11051093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD3cXms1eis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2000&pages=22-30&author=M.+Rinnov%C3%A1author=M.+Hradilekauthor=C.+Ba%C5%99inkaauthor=J.+Weberauthor=M.+Sou%C4%8Dekauthor=J.+Vondr%C3%A1%C5%A1ekauthor=T.+Klimkaitauthor=J.+Konvalinka&title=A+Picomolar+Inhibitor+of+Resistant+Strains+of+Human+Immunodeficiency+Virus+Protease+Identified+by+a+Combinatorial+Approach&doi=10.1006%2Fabbi.2000.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Picomolar Inhibitor of Resistant Strains of Human Immunodeficiency Virus Protease Identified by a Combinatorial Approach</span></div><div class="casAuthors">Rinnova, Marketa; Hradilek, Martin; Barinka, Cyril; Weber, Jan; Soucek, Milan; Vondrasek, Jiri; Klimkait, Thomas; Konvalinka, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-30</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">To identify inhibitors of various drug-resistant forms of the human immunodeficiency virus protease (HIV PR), we have designed and synthesized pseudopeptide libraries with a general structure Z-mimetic-Aa1-Aa2-NH2.  Five different chemistries for peptide bond replacement have been employed and the resulting five individual sublibraries tested with the HIV PR and its drug-resistant mutants.  Each mutant contains amino acid substitutions that have previously been shown to be assocd. with resistance to protease inhibitors, including Ritonavir, Indinavir, and Saquinavir.  We have mapped the subsite preferences of resistant HIV PR species with the aim of selecting a pluripotent pharmaceutical lead.  All of the enzyme species in this study manifest clear preference for an l-Glu residue in the P2' position.  Slight, but significant, differences in P3' subsite specificity among individual resistant PR species have been documented.  We have identified three compds., combining the most favorable features of the inhibitor array, that exhibit low-nanomolar or picomolar Ki values for all three mutant PR species tested.  (c) 2000 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqswzXAL7fJMLVg90H21EOLACvtfcHk0lhEozwKmt4OQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXms1eis7Y%253D&md5=5f9e73d0fc7a0dcba305b75f770966ff</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1006%2Fabbi.2000.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.2000.2017%26sid%3Dliteratum%253Aachs%26aulast%3DRinnov%25C3%25A1%26aufirst%3DM.%26aulast%3DHradilek%26aufirst%3DM.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DSou%25C4%258Dek%26aufirst%3DM.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26aulast%3DKlimkait%26aufirst%3DT.%26aulast%3DKonvalinka%26aufirst%3DJ.%26atitle%3DA%2520Picomolar%2520Inhibitor%2520of%2520Resistant%2520Strains%2520of%2520Human%2520Immunodeficiency%2520Virus%2520Protease%2520Identified%2520by%2520a%2520Combinatorial%2520Approach%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2000%26volume%3D382%26spage%3D22%26epage%3D30%26doi%3D10.1006%2Fabbi.2000.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hradilek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souček, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Library of HIV Proteases Inhibitors</span>. <i>Collect. Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1135/css199903079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1135%2Fcss199903079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVGls7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1999&pages=79-81&author=M.+Hradilekauthor=M.+Rinnovaauthor=C.+Barinkaauthor=M.+Sou%C4%8Dekauthor=J.+Konvalinka&title=Synthesis+of+Library+of+HIV+Proteases+Inhibitors&doi=10.1135%2Fcss199903079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of library of HIV proteases inhibitors</span></div><div class="casAuthors">Hradilek, Martin; Rinnova, Marketa; Barinka, Cyril; Soucek, Milan; Konvalinka, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Collection Symposium Series</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">Biologically Active Peptides</span>),
    <span class="NLM_cas:pages">79-81</span>CODEN:
                <span class="NLM_cas:coden">CSYSFN</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Institute of Organic Chemistry and Biochemistry, Academy of Sciences of the Czech Republic</span>)
        </div><div class="casAbstract">A symposium report.  We prepd. 288 mixts. of pseudopeptides, each contg. 48 chem. individuals.  Six nonhydrolyzable peptidomimetic groups were combined with 48 different α-amino acids by solid phase method, using Fmoc/tert-Bu protection.  Inhibitory activity of the pseudopeptides was tested against natural HIV protease and its mutants resistant to clin. employed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooQhJFgV3TWbVg90H21EOLACvtfcHk0lgvP7_gINQv6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVGls7Y%253D&md5=15bbe3134d8e5f710cf367e9ce0b48f9</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1135%2Fcss199903079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1135%252Fcss199903079%26sid%3Dliteratum%253Aachs%26aulast%3DHradilek%26aufirst%3DM.%26aulast%3DRinnova%26aufirst%3DM.%26aulast%3DBarinka%26aufirst%3DC.%26aulast%3DSou%25C4%258Dek%26aufirst%3DM.%26aulast%3DKonvalinka%26aufirst%3DJ.%26atitle%3DSynthesis%2520of%2520Library%2520of%2520HIV%2520Proteases%2520Inhibitors%26jtitle%3DCollect.%2520Symp.%2520Ser.%26date%3D1999%26volume%3D3%26spage%3D79%26epage%3D81%26doi%3D10.1135%2Fcss199903079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foundling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmings, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szelke, M.</span></span> <span> </span><span class="NLM_article-title">The Structure of a Synthetic Pepsin Inhibitor Complexed with Endothiapepsin</span>. <i>Eur. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1111/j.1432-1033.1987.tb13600.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1111%2Fj.1432-1033.1987.tb13600.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=3119339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL2sXmtlajtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=1987&pages=215-221&author=J.+Cooperauthor=S.+Foundlingauthor=A.+Hemmingsauthor=T.+Blundellauthor=D.+M.+Jonesauthor=A.+Hallettauthor=M.+Szelke&title=The+Structure+of+a+Synthetic+Pepsin+Inhibitor+Complexed+with+Endothiapepsin&doi=10.1111%2Fj.1432-1033.1987.tb13600.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of a synthetic pepsin inhibitor complexed with endothiapepsin</span></div><div class="casAuthors">Cooper, Jon; Foundling, Steve; Hemmings, Andrew; Blundell, Tom; Jones, D. Michael; Hallett, Allan; Szelke, Michael</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">215-21</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    </div><div class="casAbstract">The conformation of a synthetic polypeptide inhibitor bound to the active site of the fungal aspartic proteinase endothiapepsin (EC 3.4.23.6) was detd. by x-ray diffraction at 0.20-nm resoln. and refined to an agreement factor of 0.20.  The inhibitor, Pro-Thr-Glu-Phe-R-Phe-Arg-Glu (R = -CH2NH-), is based on a chromogenic substrate of pepsin (EC 3.4.23.1).  It has, in place of the scissile bond, a reduced peptide group which is resistant to hydrolysis and mimics the tetrahedral transition state.  The inhibitor binds in an extended conformation with the reduced bond close to the essential aspartate side chains of the enzyme.  The H bonds and hydrophobic interactions between the enzyme and the inhibitor do not induce large conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCDX_hOQ0ne7Vg90H21EOLACvtfcHk0lg5b4e4LkIT_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXmtlajtb4%253D&md5=b02c449599a73fee98bb412a1e71c7c0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1111%2Fj.1432-1033.1987.tb13600.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1432-1033.1987.tb13600.x%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DJ.%26aulast%3DFoundling%26aufirst%3DS.%26aulast%3DHemmings%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DD.%2BM.%26aulast%3DHallett%26aufirst%3DA.%26aulast%3DSzelke%26aufirst%3DM.%26atitle%3DThe%2520Structure%2520of%2520a%2520Synthetic%2520Pepsin%2520Inhibitor%2520Complexed%2520with%2520Endothiapepsin%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1987%26volume%3D169%26spage%3D215%26epage%3D221%26doi%3D10.1111%2Fj.1432-1033.1987.tb13600.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quail, J. W.</span></span> <span> </span><span class="NLM_article-title">Structure of theRhizomucor mieheiaspartic proteinase complexed with the inhibitor pepstatin A at 2.7 Å resolution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">625</span>– <span class="NLM_lpage">630</span>, <span class="refDoi"> DOI: 10.1107/s0907444998013961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0907444998013961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10089458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK1MXivVeru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1999&pages=625-630&author=J.+Yangauthor=J.+W.+Quail&title=Structure+of+theRhizomucor+mieheiaspartic+proteinase+complexed+with+the+inhibitor+pepstatin+A+at+2.7+%C3%85+resolution&doi=10.1107%2Fs0907444998013961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Rhizomucor miehei aspartic proteinase complexed with the inhibitor pepstatin A at 2.7 Å resolution</span></div><div class="casAuthors">Yang, Jian; Quail, J. Wilson</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">D55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">625-630</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">Crystals of Rhizomucor miehei aspartic proteinase (mucorpepsin) (I) complexed with pepstatin A (II) grew in orthorhombic space group P212121 and were isomorphous to native I crystals.  The unit-cell dimensions were a = 41.52, b = 50.82, and c = 172.71 Å.  There was 1 I-II complex per asym. unit.  The structure of the I-II complex was refined to a crystallog. R value of 19.3% and an Rfree value of 28.0% at 2.7 Å resoln.  A II mol. fit into the large substrate-binding cleft between the 2 domains of I in an extended conformation up to the Ala residue at the P2' position.  The dipeptide analog statine residue at the P3'-P4' position formed an inverse γ-turn (P3'-P1') with the statine residue at the P1-P1' position and its Leu side-chain bound back into the S1' subsite.  II interacted with the residues of the substrate-binding pocket by both H-bonds and hydrophobic interactions.  The OH group of the statine residue at the P1-P1' position formed H-bonds with both catalytic Asp residues (Asp-38 and Asp-237).  This conformation mimicked the expected transition state of the enzyme-substrate interaction.  The binding of II to the enzyme did not produce large distortions of the active site.  No domain movement was obsd. compared with the native enzyme structure.  However, the surface-flap region (residues 82-88) underwent a conformational change, moving toward the inhibitor and becoming rigid owing to the formation of H-bonds with the inhibitor.  B-factor calcns. of the 2 domains suggested that the C-terminal domain becomes more rigid in the complex than in the native structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXdoEjdtcNJ7Vg90H21EOLACvtfcHk0lg5b4e4LkIT_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivVeru7g%253D&md5=ed3f3c209942d09ba1b65793d0e57e5b</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444998013961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998013961%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DQuail%26aufirst%3DJ.%2BW.%26atitle%3DStructure%2520of%2520theRhizomucor%2520mieheiaspartic%2520proteinase%2520complexed%2520with%2520the%2520inhibitor%2520pepstatin%2520A%2520at%25202.7%2520%25C3%2585%2520resolution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1999%26volume%3D55%26spage%3D625%26epage%3D630%26doi%3D10.1107%2Fs0907444998013961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruge, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelsang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monod, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korting, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maskos, K.</span></span> <span> </span><span class="NLM_article-title">X-ray structures of Sap1 and Sap5: Structural Comparison of the Secreted Aspartic Proteinases from Candida Albicans</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1308</span>– <span class="NLM_lpage">1319</span>, <span class="refDoi"> DOI: 10.1002/prot.22021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1002%2Fprot.22021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=18384081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2008&pages=1308-1319&author=C.+Borelliauthor=E.+Rugeauthor=J.+H.+Leeauthor=M.+Schallerauthor=A.+Vogelsangauthor=M.+Monodauthor=H.+C.+Kortingauthor=R.+Huberauthor=K.+Maskos&title=X-ray+structures+of+Sap1+and+Sap5%3A+Structural+Comparison+of+the+Secreted+Aspartic+Proteinases+from+Candida+Albicans&doi=10.1002%2Fprot.22021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray structures of Sap1 and Sap5: structural comparison of the secreted aspartic proteinases from Candida albicans</span></div><div class="casAuthors">Borelli, Claudia; Ruge, Elisabeth; Lee, Jung Hoon; Schaller, Martin; Vogelsang, Alexandra; Monod, Michel; Korting, Hans Christian; Huber, Robert; Maskos, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1308-1319</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Proteolytic activity is an important virulence factor for Candida albicans.  It is attributed to the family of the secreted aspartic proteinases (Saps) from C. albicans with a min. of 10 members.  Saps show controlled expression and regulation for the individual stages of the infection process.  Distinct isoenzymes can be responsible for adherence and tissue damage of local infections, while others cause systemic diseases.  Earlier, only the structures of Sap2 and Sap3 were known.  In our research, we have now succeeded in solving the X-ray crystal structures of the apoenzyme of Sap1 and Sap5 in complex with pepstatin A (pep A) at 2.05 and 2.5 Å resoln., resp.  With the structure of Sap1, we have completed the set of structures of isoenzyme subgroup Sap1-3.  Of subgroup Sap4-6, the structure of the enzyme Sap5 is the first structure that has been described up to now.  This facilitates comparison of structural details as well as inhibitor binding modes among the different subgroup members.  Structural anal. reveals a highly conserved overall secondary structure of Sap1-3 and Sap5.  However, Sap5 clearly differs from Sap1-3 by its electrostatic overall charge as well as through structural conformation of its entrance to the active site cleft.  Design of inhibitors specific for Sap5 should conc. on the S4 and S3 pockets, which significantly differ from Sap1-3 in size and electrostatic charge.  Both Sap1 and Sap5 seem to play a major part in superficial Candida infections.  Detn. of the isoenzymes' structures can contribute to the development of new Sap-specific inhibitors for the treatment of superficial infections with a structure-based drug design program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPr1xjkXUPprVg90H21EOLACvtfcHk0limesW47ZoFPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCrtbzP&md5=512e20e202e3aea0c095ead9fe069115</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fprot.22021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22021%26sid%3Dliteratum%253Aachs%26aulast%3DBorelli%26aufirst%3DC.%26aulast%3DRuge%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DSchaller%26aufirst%3DM.%26aulast%3DVogelsang%26aufirst%3DA.%26aulast%3DMonod%26aufirst%3DM.%26aulast%3DKorting%26aufirst%3DH.%2BC.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DMaskos%26aufirst%3DK.%26atitle%3DX-ray%2520structures%2520of%2520Sap1%2520and%2520Sap5%253A%2520Structural%2520Comparison%2520of%2520the%2520Secreted%2520Aspartic%2520Proteinases%2520from%2520Candida%2520Albicans%26jtitle%3DProteins%26date%3D2008%26volume%3D72%26spage%3D1308%26epage%3D1319%26doi%3D10.1002%2Fprot.22021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeever, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanMiddlesworth, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimbach, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leu, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herber, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darke, P. L.</span></span> <span> </span><span class="NLM_article-title">Crystallographic Analysis of a Complex Between Human Immunodeficiency Virus Type 1 Protease and Acetyl-pepstatin at 2.0-A Resolution</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>265</i></span>,  <span class="NLM_fpage">14209</span>– <span class="NLM_lpage">14219</span>, <span class="refDoi"> DOI: 10.1016/s0021-9258(18)77288-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2FS0021-9258%2818%2977288-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=2201682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3cXls1Oqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=14209-14219&author=P.+M.+Fitzgeraldauthor=B.+M.+McKeeverauthor=J.+F.+VanMiddlesworthauthor=J.+P.+Springerauthor=J.+C.+Heimbachauthor=C.+T.+Leuauthor=W.+K.+Herberauthor=R.+A.+Dixonauthor=P.+L.+Darke&title=Crystallographic+Analysis+of+a+Complex+Between+Human+Immunodeficiency+Virus+Type+1+Protease+and+Acetyl-pepstatin+at+2.0-A+Resolution&doi=10.1016%2Fs0021-9258%2818%2977288-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic analysis of a complex between human immunodeficiency virus type 1 protease and acetyl-pepstatin at 2.0-Å resolution</span></div><div class="casAuthors">Fitzgerald, Paula M. D.; McKeever, Brian M.; VanMiddlesworth, Jody F.; Springer, James P.; Heimbach, Jill C.; Leu, Chih Tai; Herber, Wayne K.; Dixon, Richard A. F.; Darke, Paul L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">14209-19</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">The mode of binding of acetyl-pepstatin to the protease from the human immunodeficiency virus type 1 (HIV-1) has been detd. by x-ray diffraction anal.  Crystals of an acetyl-pepstatin-HIV-1 protease complex were obtained in space group P21212 (unit cell dimensions a = 58.39 Å, b = 86.70 Å, c = 46.27 Å) by pptn. with NaCl.  The structure was phased by mol. replacement methods, and a model for the structure was refined using diffraction data to 2.0 Å resoln. (R = 0.176 for 12,901 reflections with I > σ(I); deviation of bond distances from ideal values = 0.018 Å; 172 solvent mols. included).  The structure of the protein in the complex has been compared with the structure of the enzyme without the ligand.  A core of 44 amino acids in each monomer, including residues in the active site and residues at the dimer interface, remains unchanged on binding of the inhibitor (root mean square deviation of α-C positions = 0.39 Å).  The remaining 55 residues in each monomer undergo substantial rearrangement, with the most dramatic changes occurring at residues 44-57 (these residues comprise the so-called flaps of the enzyme).  The flaps interact with one another and with the inhibitor so as to largely preserve the 2-fold symmetry of the protein.  The inhibitor is bound in 2 approx. sym. orientations.  In both orientations the peptidyl backbone of the inhibitor is extended; a network of H bonds is formed between the inhibitor and the main body of the protein as well as between the inhibitor and the flaps.  Hydrophobic side chains of residues in the body of the protein form partial binding sites for the side chains of the inhibitor; hydrophobic side chains of residues in the flaps complete these binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRZgaPlbh_ibVg90H21EOLACvtfcHk0limesW47ZoFPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXls1Oqsrs%253D&md5=7f8331534e65f8bb17d4acde995dfbfa</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2FS0021-9258%2818%2977288-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0021-9258%252818%252977288-8%26sid%3Dliteratum%253Aachs%26aulast%3DFitzgerald%26aufirst%3DP.%2BM.%26aulast%3DMcKeever%26aufirst%3DB.%2BM.%26aulast%3DVanMiddlesworth%26aufirst%3DJ.%2BF.%26aulast%3DSpringer%26aufirst%3DJ.%2BP.%26aulast%3DHeimbach%26aufirst%3DJ.%2BC.%26aulast%3DLeu%26aufirst%3DC.%2BT.%26aulast%3DHerber%26aufirst%3DW.%2BK.%26aulast%3DDixon%26aufirst%3DR.%2BA.%26aulast%3DDarke%26aufirst%3DP.%2BL.%26atitle%3DCrystallographic%2520Analysis%2520of%2520a%2520Complex%2520Between%2520Human%2520Immunodeficiency%2520Virus%2520Type%25201%2520Protease%2520and%2520Acetyl-pepstatin%2520at%25202.0-A%2520Resolution%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D14209%26epage%3D14219%26doi%3D10.1016%2Fs0021-9258%2818%2977288-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poręba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drąg, M.</span></span> <span> </span><span class="NLM_article-title">Selective Substrates and Activity-based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5222</span>– <span class="NLM_lpage">5234</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVags7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5222-5234&author=W.+Rutauthor=M.+Por%C4%99baauthor=P.+Kasperkiewiczauthor=S.+J.+Snipasauthor=M.+Dr%C4%85g&title=Selective+Substrates+and+Activity-based+Probes+for+Imaging+of+the+Human+Constitutive+20S+Proteasome+in+Cells+and+Blood+Samples&doi=10.1021%2Facs.jmedchem.8b00026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Substrates and Activity-Based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span></div><div class="casAuthors">Rut, Wioletta; Poreba, Marcin; Kasperkiewicz, Paulina; Snipas, Scott J.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5222-5234</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proteasome is an enzyme complex crit. for maintaining protein homeostasis.  Perturbed proteasome function leads to pathologies including cancer and autoimmune and neurodegenerative disease.  Therefore, the proteasome constitutes an excellent mol. target for pharmaceutical development.  Here, using the HyCoSuL approach, the authors designed and synthesized novel and selective fluorogenic substrates for each of these three constitutive 20S proteasome activities and applied them to assess inhibition of proteasome subunits by MG-132 and a clin. used inhibitor bortezomib.  The authors' results confirm the utility of designed substrates in biochem. assays.  Furthermore, selective peptide sequences obtained in this manner were used to construct fluorophore-labeled activity-based probes and then used to detect each constitutive 20S proteasome subunit simultaneously in lysates of HEK-293F cells and red blood cells.  Overall, the authors describe a simple and rapid method useful to measure constitutive 20S proteasome activity in whole human blood samples that could enable early diagnosis of pathol. states assocd. with aberrantly upregulated proteasome activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrd0FSTAesirVg90H21EOLACvtfcHk0lgQqq2OcOfPXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVags7vO&md5=1fb97b492a7f3f25d6659035f5bf4b14</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00026%26sid%3Dliteratum%253Aachs%26aulast%3DRut%26aufirst%3DW.%26aulast%3DPor%25C4%2599ba%26aufirst%3DM.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DDr%25C4%2585g%26aufirst%3DM.%26atitle%3DSelective%2520Substrates%2520and%2520Activity-based%2520Probes%2520for%2520Imaging%2520of%2520the%2520Human%2520Constitutive%252020S%2520Proteasome%2520in%2520Cells%2520and%2520Blood%2520Samples%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5222%26epage%3D5234%26doi%3D10.1021%2Facs.jmedchem.8b00026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunde, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihelic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Counter Selection Substrate Library Strategy for Developing Specific Protease Substrates and Probes</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1023</span>– <span class="NLM_lpage">1035</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.chembiol.2016.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=27478158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1023-1035&author=M.+Porebaauthor=R.+Solbergauthor=W.+Rutauthor=N.+N.+Lundeauthor=P.+Kasperkiewiczauthor=S.+J.+Snipasauthor=M.+Mihelicauthor=D.+Turkauthor=B.+Turkauthor=G.+S.+Salvesenauthor=M.+Drag&title=Counter+Selection+Substrate+Library+Strategy+for+Developing+Specific+Protease+Substrates+and+Probes&doi=10.1016%2Fj.chembiol.2016.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Counter Selection Substrate Library Strategy for Developing Specific Protease Substrates and Probes</span></div><div class="casAuthors">Poreba, Marcin; Solberg, Rigmor; Rut, Wioletta; Lunde, Ngoc Nguyen; Kasperkiewicz, Paulina; Snipas, Scott J.; Mihelic, Marko; Turk, Dusan; Turk, Boris; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1023-1035</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Legumain (AEP) is a lysosomal cysteine protease that was first characterized in leguminous seeds and later discovered in higher eukaryotes.  AEP upregulation is linked to a no. of diseases including inflammation, arteriosclerosis, and tumorigenesis.  Thus this protease is an excellent mol. target for the development of new chem. markers.  We deployed a hybrid combinatorial substrate library (HyCoSuL) approach to obtain P1-Asp fluorogenic substrates and biotin-labeled inhibitors that targeted legumain.  Since this approach led to probes that were also recognized by caspases, we introduced a Counter Selection Substrate Library (CoSeSuL) approach that biases the peptidic scaffold against caspases, thus delivering highly selective legumain probes.  The selectivity of these tools was validated using M38L and HEK293 cells.  We also propose that the CoSeSuL methodol. can be considered as a general principle in the design of selective probes for other protease families where selectivity is difficult to achieve by conventional sequence-based profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtgQyo_YeLk7Vg90H21EOLACvtfcHk0lgQqq2OcOfPXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnN&md5=96a607ad2ac4bbcb35c3dd106aad0688</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSolberg%26aufirst%3DR.%26aulast%3DRut%26aufirst%3DW.%26aulast%3DLunde%26aufirst%3DN.%2BN.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DMihelic%26aufirst%3DM.%26aulast%3DTurk%26aufirst%3DD.%26aulast%3DTurk%26aufirst%3DB.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DCounter%2520Selection%2520Substrate%2520Library%2520Strategy%2520for%2520Developing%2520Specific%2520Protease%2520Substrates%2520and%2520Probes%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D1023%26epage%3D1035%26doi%3D10.1016%2Fj.chembiol.2016.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, B. M.</span></span> <span> </span><span class="NLM_article-title">Substrate Specificity and Inhibitors of Aspartic Proteinases</span>. <i>Scand. J. Clin. Lab. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.3109/00365519209104651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1080%2F00365519209104651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3sXjsV2ktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1992&pages=23-30&author=J.+Kayauthor=B.+M.+Dunn&title=Substrate+Specificity+and+Inhibitors+of+Aspartic+Proteinases&doi=10.3109%2F00365519209104651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate specificity and inhibitors of aspartic proteinases</span></div><div class="casAuthors">Kay, John; Dunn, B. M.</div><div class="citationInfo"><span class="NLM_cas:title">Scandinavian Journal of Clinical and Laboratory Investigation, Supplement</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">210</span>),
    <span class="NLM_cas:pages">23-30</span>CODEN:
                <span class="NLM_cas:coden">SCLSAH</span>;
        ISSN:<span class="NLM_cas:issn">0085-591X</span>.
    </div><div class="casAbstract">Aspartic proteinases of vertebrate, fungal and retroviral origin have been characterized by utilization of chromogenic peptide substrates and synthetic inhibitors as probes.  By this means, the mol. topog. of sub-sites within the active site cleft of individual enzymes has been elucidated.  With suitable selection of residues to occupy each sub-site, the design of effective inhibitors specifically targetted against individual aspartic proteinases has become feasible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojIg7wIZCs0LVg90H21EOLACvtfcHk0lgQqq2OcOfPXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXjsV2ktw%253D%253D&md5=8939a05cc45ca7f786c7d9b0c3e39f8a</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1080%2F00365519209104651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00365519209104651%26sid%3Dliteratum%253Aachs%26aulast%3DKay%26aufirst%3DJ.%26aulast%3DDunn%26aufirst%3DB.%2BM.%26atitle%3DSubstrate%2520Specificity%2520and%2520Inhibitors%2520of%2520Aspartic%2520Proteinases%26jtitle%3DScand.%2520J.%2520Clin.%2520Lab.%2520Invest.%26date%3D1992%26volume%3D52%26spage%3D23%26epage%3D30%26doi%3D10.3109%2F00365519209104651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werb, Z.</span></span> <span> </span><span class="NLM_article-title">Extracellular Matrix Degradation and Remodeling in Development and Disease</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">a005058</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a005058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1101%2Fcshperspect.a005058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=21917992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVKrt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=a005058&author=P.+Luauthor=K.+Takaiauthor=V.+M.+Weaverauthor=Z.+Werb&title=Extracellular+Matrix+Degradation+and+Remodeling+in+Development+and+Disease&doi=10.1101%2Fcshperspect.a005058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular matrix degradation and remodeling in development and disease</span></div><div class="casAuthors">Lu, Pengfei; Takai, Ken; Weaver, Valerie M.; Werb, Zena</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">a005058/1-a005058/24</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The extracellular matrix (ECM) serves diverse functions and is a major component of the cellular microenvironment.  The ECM is a highly dynamic structure, constantly undergoing a remodeling process where ECM components are deposited, degraded, or otherwise modified.  ECM dynamics are indispensable during restructuring of tissue architecture.  ECM remodeling is an important mechanism whereby cell differentiation can be regulated, including processes such as the establishment and maintenance of stem cell niches, branching morphogenesis, angiogenesis, bone remodeling, and wound repair.  In contrast, abnormal ECM dynamics lead to deregulated cell proliferation and invasion, failure of cell death, and loss of cell differentiation, resulting in congenital defects and pathol. processes including tissue fibrosis and cancer.  Understanding the mechanisms of ECM remodeling and its regulation, therefore, is essential for developing new therapeutic interventions for diseases and novel strategies for tissue engineering and regenerative medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpu5HZYPalvrVg90H21EOLACvtfcHk0lhG8oZ0_Elcjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVKrt78%253D&md5=d59a5d55420420b4cbbb731286ecd780</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a005058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a005058%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DP.%26aulast%3DTakai%26aufirst%3DK.%26aulast%3DWeaver%26aufirst%3DV.%2BM.%26aulast%3DWerb%26aufirst%3DZ.%26atitle%3DExtracellular%2520Matrix%2520Degradation%2520and%2520Remodeling%2520in%2520Development%2520and%2520Disease%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2011%26volume%3D3%26spage%3Da005058%26doi%3D10.1101%2Fcshperspect.a005058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregorová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stehlíková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreánsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedlácek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strop, P.</span></span> <span> </span><span class="NLM_article-title">Specificity Mapping of HIV-1 Protease by Reduced Bond Inhibitors</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1006/abbi.1993.1314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1006%2Fabbi.1993.1314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8323274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3sXlsFWlt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=1993&pages=1-8&author=P.+Majerauthor=J.+Urbanauthor=E.+Gregorov%C3%A1author=J.+Konvalinkaauthor=P.+Novekauthor=J.+Stehl%C3%ADkov%C3%A1author=M.+Andre%C3%A1nskyauthor=J.+Sedl%C3%A1cekauthor=P.+Strop&title=Specificity+Mapping+of+HIV-1+Protease+by+Reduced+Bond+Inhibitors&doi=10.1006%2Fabbi.1993.1314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity mapping of HIV-1 protease by reduced bond inhibitors</span></div><div class="casAuthors">Majer, Pavel; Urban, Jan; Gregorova, Eva; Konvalinka, Jan; Novek, Petr; Stehlikova, Jana; Andreansky, Martin; Sedlacek, Juraj; Strop, Petr</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">A series of 47 N-truncated reduced bond inhibitors, systematically modified at individual positions (P1, P'1, P'2, P'3, and P'4), were synthesized and used to map the subsite preferences of HIV-1 protease.  The tight binding inhibitor of HIV-1 protease t-butoxycarbonyl-Phe-[Ch2NH]Phe-Glu-Phe-NH2 (Ki = 0.2 nM) was chosen as the parent structure for further modifications.  The P'2 glutamic acid was found to fit well into the S'2 subsite of the protease.  The conformational restriction of any phenylalanine residue or satn. of more than one phenylalanine side chain in P'1 or P'3 lead is to a large Ki increase.  Introduction of tyrosine in the P1 position improves the binding by an order of magnitude.  The S'4 subsite of the protease was shown to accommodate large structural changes in the inhibitor at this position.  Therefore P'4 may serve as an ideal region for further modification in order to improve bioavailability of this type of compd.  An improved method of direct comparison of tight binding inhibitors with subnanomolar Ki values has been described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMg912DjALIrVg90H21EOLACvtfcHk0lhG8oZ0_Elcjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsFWlt7Y%253D&md5=770e121ac372cb4025c68fe96c97da4d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1006%2Fabbi.1993.1314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.1993.1314%26sid%3Dliteratum%253Aachs%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DUrban%26aufirst%3DJ.%26aulast%3DGregorov%25C3%25A1%26aufirst%3DE.%26aulast%3DKonvalinka%26aufirst%3DJ.%26aulast%3DNovek%26aufirst%3DP.%26aulast%3DStehl%25C3%25ADkov%25C3%25A1%26aufirst%3DJ.%26aulast%3DAndre%25C3%25A1nsky%26aufirst%3DM.%26aulast%3DSedl%25C3%25A1cek%26aufirst%3DJ.%26aulast%3DStrop%26aufirst%3DP.%26atitle%3DSpecificity%2520Mapping%2520of%2520HIV-1%2520Protease%2520by%2520Reduced%2520Bond%2520Inhibitors%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1993%26volume%3D304%26spage%3D1%26epage%3D8%26doi%3D10.1006%2Fabbi.1993.1314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Houštecká, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadzima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanfrlík, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hánová, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertlíková-Kaiserová, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepšík, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smrčina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure–Activity Relationship and Binding Mode Analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1576</span>– <span class="NLM_lpage">1596</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01351</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01351" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVOgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=1576-1596&author=R.+Hou%C5%A1teck%C3%A1author=M.+Hadzimaauthor=J.+Fanfrl%C3%ADkauthor=J.+Bryndaauthor=L.+Pallov%C3%A1author=I.+H%C3%A1nov%C3%A1author=H.+Mertl%C3%ADkov%C3%A1-Kaiserov%C3%A1author=M.+Lep%C5%A1%C3%ADkauthor=M.+Hornauthor=M.+Smr%C4%8Dinaauthor=P.+Majerauthor=M.+Mare%C5%A1&title=Biomimetic+Macrocyclic+Inhibitors+of+Human+Cathepsin+D%3A+Structure%E2%80%93Activity+Relationship+and+Binding+Mode+Analysis&doi=10.1021%2Facs.jmedchem.9b01351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure-Activity Relationship and Binding Mode Analysis</span></div><div class="casAuthors">Houstecka, Radka; Hadzima, Martin; Fanfrlik, Jindrich; Brynda, Jiri; Pallova, Lenka; Hanova, Iva; Mertlikova-Kaiserova, Helena; Lepsik, Martin; Horn, Martin; Smrcina, Martin; Majer, Pavel; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1576-1596</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human cathepsin D (CatD), a pepsin-family aspartic protease, plays an important role in tumor progression and metastasis.  Here, we report the development of biomimetic inhibitors of CatD as novel tools for regulation of this therapeutic target.  We designed a macrocyclic scaffold to mimic the spatial conformation of the minimal pseudo-dipeptide binding motif of pepstatin A, a microbial oligopeptide inhibitor, in the CatD active site.  A library of more than 30 macrocyclic peptidomimetic inhibitors was employed for scaffold optimization, mapping of subsite interactions, and profiling of inhibitor selectivity.  Furthermore, we solved high-resoln. crystal structures of three macrocyclic inhibitors with low nanomolar or subnanomolar potency in complex with CatD and detd. their binding mode using quantum chem. calcns.  The study provides a new structural template and functional profile that can be exploited for design of potential chemotherapeutics that specifically inhibit CatD and related aspartic proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaOxpVGiZrGLVg90H21EOLACvtfcHk0liyW9F3yxky0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVOgsrY%253D&md5=5c5d311485db722abf1ac78b0a640273</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01351%26sid%3Dliteratum%253Aachs%26aulast%3DHou%25C5%25A1teck%25C3%25A1%26aufirst%3DR.%26aulast%3DHadzima%26aufirst%3DM.%26aulast%3DFanfrl%25C3%25ADk%26aufirst%3DJ.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DPallov%25C3%25A1%26aufirst%3DL.%26aulast%3DH%25C3%25A1nov%25C3%25A1%26aufirst%3DI.%26aulast%3DMertl%25C3%25ADkov%25C3%25A1-Kaiserov%25C3%25A1%26aufirst%3DH.%26aulast%3DLep%25C5%25A1%25C3%25ADk%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DSmr%25C4%258Dina%26aufirst%3DM.%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DBiomimetic%2520Macrocyclic%2520Inhibitors%2520of%2520Human%2520Cathepsin%2520D%253A%2520Structure%25E2%2580%2593Activity%2520Relationship%2520and%2520Binding%2520Mode%2520Analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D1576%26epage%3D1596%26doi%3D10.1021%2Facs.jmedchem.9b01351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiering, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedrani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maibaum, J.</span></span> <span> </span><span class="NLM_article-title">Direct Renin Inhibitors as a New Therapy for Hypertension</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">7490</span>– <span class="NLM_lpage">7520</span>, <span class="refDoi"> DOI: 10.1021/jm901885s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901885s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKjsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=7490-7520&author=R.+L.+Webbauthor=N.+Schieringauthor=R.+Sedraniauthor=J.+Maibaum&title=Direct+Renin+Inhibitors+as+a+New+Therapy+for+Hypertension&doi=10.1021%2Fjm901885s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Direct Renin Inhibitors as a New Therapy for Hypertension</span></div><div class="casAuthors">Webb, Randy L.; Schiering, Nikolaus; Sedrani, Richard; Maibaum, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7490-7520</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The objective of this review is to highlight key events and recent advances in the evolution of nonpeptide peptidomimetic direct renin inhibitors (DRIs).  Special emphasis will be placed on the successful structure-based strategies that enabled the discovery of several novel classes of DRIs.  The potential limitations assocd. with these inhibitors and opportunities for future directions of renin inhibitor design will be discussed from a chem. and biol. perspective.  Lessons learned will also be discussed in an effort to provide a unique perspective on the development of aliskiren.  With its introduction, it is now very important to distinguish this mechanism of action from indirect means of inhibiting renin.  In this regard, ongoing clin. trials designed to demonstrate a clear benefit, beyond an effect on blood pressure, by redn. in morbidity and mortality are of paramount importance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRBUB_cukx3LVg90H21EOLACvtfcHk0liyW9F3yxky0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKjsb3E&md5=54b73ed33f69d2ad788c6f5839a54a7e</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm901885s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901885s%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DR.%2BL.%26aulast%3DSchiering%26aufirst%3DN.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DMaibaum%26aufirst%3DJ.%26atitle%3DDirect%2520Renin%2520Inhibitors%2520as%2520a%2520New%2520Therapy%2520for%2520Hypertension%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D7490%26epage%3D7520%26doi%3D10.1021%2Fjm901885s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tykvart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šácha, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knedlík, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubkowski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span> <span> </span><span class="NLM_article-title">Efficient and Versatile One-step Affinity Purification of In Vivo Biotinylated Proteins: Expression, Characterization and Structure Analysis of Recombinant Human Glutamate Carboxypeptidase II</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1016/j.pep.2011.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.pep.2011.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=22178733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2012&pages=106-115&author=J.+Tykvartauthor=P.+%C5%A0%C3%A1chaauthor=C.+Ba%C5%99inkaauthor=T.+Knedl%C3%ADkauthor=J.+Starkov%C3%A1author=J.+Lubkowskiauthor=J.+Konvalinka&title=Efficient+and+Versatile+One-step+Affinity+Purification+of+In+Vivo+Biotinylated+Proteins%3A+Expression%2C+Characterization+and+Structure+Analysis+of+Recombinant+Human+Glutamate+Carboxypeptidase+II&doi=10.1016%2Fj.pep.2011.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient and versatile one-step affinity purification of in vivo biotinylated proteins: Expression, characterization and structure analysis of recombinant human glutamate carboxypeptidase II</span></div><div class="casAuthors">Tykvart, J.; Sacha, P.; Barinka, C.; Knedlik, T.; Starkova, J.; Lubkowski, J.; Konvalinka, J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">106-115</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Affinity purifn. is a useful approach for purifn. of recombinant proteins.  Eukaryotic expression systems have become more frequently used at the expense of prokaryotic systems since they afford recombinant eukaryotic proteins with post-translational modifications similar or identical to the native ones.  Here, a one-step affinity purifn. set-up suitable for the purifn. of secreted proteins is presented.  The set-up is based on the interaction between biotin and mutated streptavidin.  Drosophila Schneider 2 cells are chosen as the expression host, and a biotin acceptor peptide is used as an affinity tag.  This tag is biotinylated by Escherichia coli biotin-protein ligase in vivo.  It was detd. that localization of the ligase within the ER led to the most effective in vivo biotinylation of the secreted proteins.  A protocol was optimized for large-scale expression and purifn. of AviTEV-tagged recombinant human glutamate carboxypeptidase II (Avi-GCPII) with milligram yields per L of culture.  The 3D structure of Avi-GCPII was also detd. by X-ray crystallog. and the enzymic characteristics of the protein were compared to those of its non-tagged variant.  These expts. confirmed that AviTEV tag does not affect the biophys. properties of its fused partner.  The purifn. approach, developed here, provides not only a sufficient amt. of highly homogeneous protein but also specifically and effectively biotinylates a target protein and thus enables its subsequent visualization or immobilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo57Mui9FOUXbVg90H21EOLACvtfcHk0liyW9F3yxky0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWgs7c%253D&md5=d731679432bae009c380ace62a6d3893</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2011.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2011.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DTykvart%26aufirst%3DJ.%26aulast%3D%25C5%25A0%25C3%25A1cha%26aufirst%3DP.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DKnedl%25C3%25ADk%26aufirst%3DT.%26aulast%3DStarkov%25C3%25A1%26aufirst%3DJ.%26aulast%3DLubkowski%26aufirst%3DJ.%26aulast%3DKonvalinka%26aufirst%3DJ.%26atitle%3DEfficient%2520and%2520Versatile%2520One-step%2520Affinity%2520Purification%2520of%2520In%2520Vivo%2520Biotinylated%2520Proteins%253A%2520Expression%252C%2520Characterization%2520and%2520Structure%2520Analysis%2520of%2520Recombinant%2520Human%2520Glutamate%2520Carboxypeptidase%2520II%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2012%26volume%3D82%26spage%3D106%26epage%3D115%26doi%3D10.1016%2Fj.pep.2011.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yung-Chi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the Inhibition Constant (KI) and the Concentration of Inhibitor which Causes 50 per cent Inhibition (I50) of an Enzymatic Reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+Inhibition+Constant+%28KI%29+and+the+Concentration+of+Inhibitor+which+Causes+50+per+cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520Inhibition%2520Constant%2520%2528KI%2529%2520and%2520the%2520Concentration%2520of%2520Inhibitor%2520which%2520Causes%252050%2520per%2520cent%2520Inhibition%2520%2528I50%2529%2520of%2520an%2520Enzymatic%2520Reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Britton, H. T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. A.</span></span> <span> </span><span class="NLM_article-title">CXCVIII.-Universal Buffer Solutions and the Dissociation Constant of Veronal</span>. <i>J. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1931</span>,  <span class="NLM_fpage">1456</span>– <span class="NLM_lpage">1462</span>, <span class="refDoi"> DOI: 10.1039/jr9310001456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1039%2FJR9310001456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaA3MXkt1aqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1931&pages=1456-1462&author=H.+T.+S.+Brittonauthor=R.+A.+Robinson&title=CXCVIII.-Universal+Buffer+Solutions+and+the+Dissociation+Constant+of+Veronal&doi=10.1039%2Fjr9310001456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Universal buffer solutions and the dissociation constant of veronal</span></div><div class="casAuthors">Britton, Hubert T. S.; Robinson, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society</span>
        (<span class="NLM_cas:date">1931</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1456-62</span>CODEN:
                <span class="NLM_cas:coden">JCSOA9</span>;
        ISSN:<span class="NLM_cas:issn">0368-1769</span>.
    </div><div class="casAbstract">The so-called "universal" buffer solns. contg. H3PO4, AcOH (or phenylacetic acid) and boric acids show a graph that is by no means a straight line when the pH is detd. after each small portion of added NaOH.  The use of a linear interpolation formula, therefore, may lead to results which are as much as 0.2 pH in error.  A better universal buffer soln. can be prepd. from citric acid, H3PO4, veronal and H3BO3 with 0.2 N solns. of each.  The dissocn. const. of veronal (diethylbarbituric acid) is 7.96.  The buffer mixt. of the 4 acids gives rise to 7 stages of dissocn., each corresponding to an acid concn. of 0.02857 N.  The soln. contains KH2PO4, citric acid, veronal, H3BO3 and HCl dissolved in water and dild. to 1 1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozqK1uPsKrR7Vg90H21EOLACvtfcHk0li_uY2SQ-4Kaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaA3MXkt1aqtg%253D%253D&md5=265f63ca520e32cfac52dc2324ae6ddf</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1039%2FJR9310001456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FJR9310001456%26sid%3Dliteratum%253Aachs%26aulast%3DBritton%26aufirst%3DH.%2BT.%2BS.%26aulast%3DRobinson%26aufirst%3DR.%2BA.%26atitle%3DCXCVIII.-Universal%2520Buffer%2520Solutions%2520and%2520the%2520Dissociation%2520Constant%2520of%2520Veronal%26jtitle%3DJ.%2520Chem.%2520Soc.%26date%3D1931%26spage%3D1456%26epage%3D1462%26doi%3D10.1039%2Fjr9310001456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gray, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billings, J. A.</span></span> <span> </span><span class="NLM_article-title">Kinetic Assay of Human Pepsin with Albumin-bromphenol Blue as Substrate</span>. <i>Clin. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1093/clinchem/29.3.447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fclinchem%2F29.3.447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=6402323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL3sXhsFKrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1983&pages=447-451&author=S.+P.+Grayauthor=J.+A.+Billings&title=Kinetic+Assay+of+Human+Pepsin+with+Albumin-bromphenol+Blue+as+Substrate&doi=10.1093%2Fclinchem%2F29.3.447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic assay of human pepsin with albumin-bromphenol blue as substrate</span></div><div class="casAuthors">Gray, Stephen P.; Billings, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">447-51</span>CODEN:
                <span class="NLM_cas:coden">CLCHAU</span>;
        ISSN:<span class="NLM_cas:issn">0009-9147</span>.
    </div><div class="casAbstract">A novel substrate, albumin complexed with bromophenol blue, has been developed for the assay of human gastric juice pepsin by a kinetic method in the Cobas centrifugal analyzer.  The action of pepsin on the complex degrades the albumin and releases the dye.  The change in the color of the substrate is a zero-order reaction.  Human and porcine pepsin have different Km values with the new substrate.  This kinetic method has a throughput of 28 tests in ∼10 min and good precision.  Other advantages include anal. in homogeneous soln. (thereby eliminating the need to sep. substrate and products), lack of interference from bilirubin or phenol red, and the expression of pepsin activity in IUB enzyme units.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSannvJ_B-Z7Vg90H21EOLACvtfcHk0lhq1CmZsYyuHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhsFKrtr0%253D&md5=c5a1cadde3bb74092f4c5e2f8384f230</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fclinchem%2F29.3.447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fclinchem%252F29.3.447%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DS.%2BP.%26aulast%3DBillings%26aufirst%3DJ.%2BA.%26atitle%3DKinetic%2520Assay%2520of%2520Human%2520Pepsin%2520with%2520Albumin-bromphenol%2520Blue%2520as%2520Substrate%26jtitle%3DClin.%2520Chem.%26date%3D1983%26volume%3D29%26spage%3D447%26epage%3D451%26doi%3D10.1093%2Fclinchem%2F29.3.447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holzman, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubbins, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pederson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krafft, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. T.</span></span> <span> </span><span class="NLM_article-title">Recombinant Human Prorenin from CHO Cells: Expression and Purification</span>. <i>J. Protein Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1007/bf01024761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1007%2FBF01024761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1963533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3MXps1Wqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1990&pages=663-672&author=T.+F.+Holzmanauthor=C.+C.+Chungauthor=R.+Edaljiauthor=D.+A.+Eganauthor=E.+J.+Gubbinsauthor=A.+Rueterauthor=G.+Howardauthor=L.+K.+Yangauthor=T.+M.+Pedersonauthor=G.+A.+Krafftauthor=G.+T.+Wang&title=Recombinant+Human+Prorenin+from+CHO+Cells%3A+Expression+and+Purification&doi=10.1007%2Fbf01024761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Recombinant human prorenin from CHO cells:  expression and purification</span></div><div class="casAuthors">Holzman, Thomas F.; Chung, Christine C.; Edalji, Rohinton; Egan, David A.; Gubbins, Earl J.; Rueter, Annemarie; Howard, Gail; Yang, Lana K.; Pederson, Terry M.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Protein Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">663-72</span>CODEN:
                <span class="NLM_cas:coden">JPCHD2</span>;
        ISSN:<span class="NLM_cas:issn">0277-8033</span>.
    </div><div class="casAbstract">The gene for human preprorenin was obtained from total RNA prepd. from primary human chorion cells.  An expression vector was constructed contg. an SV40 early promoter, a human preprorenin cDNA, bovine growth hormone poly-A addn. signal, and a dihydrofolate reductase (dhfr) expression cassette.  This vector was inserted into the DXB 11 Chinese hamster ovary (CHO) cell line.  The recombinant protein was exported by CHO cells into the tissue culture media.  At harvest the prorenin levels ranged from ∼1-5 mg/L.  For protein isolation the cell culture supernatants were processed by filtration, concn., dialysis, and batch extn.  Preparative-scale isolation of prorenin was accomplished using blue-dye chromatog. and size-exclusion chromatog.  The isolated prorenin yielded a single SDS-gel band with Mr ∼ 40,000.  The proprotein was characterized with respect to N-terminal sequence and N-linked sugar compn.  Trypsin-activated renin prepd. from the proprotein was characterized with respect to N-terminal sequence and pH-activity profile.  Enzyme activity was measured with a newly developed fluorogenic peptide substrate contg. the P6-P'3 sequence of human angiotensinogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFuzdaHY7v7Vg90H21EOLACvtfcHk0lhq1CmZsYyuHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXps1Wqtw%253D%253D&md5=6e4eb32ddfb7bcb68b13ef2b25f46f41</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1007%2FBF01024761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01024761%26sid%3Dliteratum%253Aachs%26aulast%3DHolzman%26aufirst%3DT.%2BF.%26aulast%3DChung%26aufirst%3DC.%2BC.%26aulast%3DEdalji%26aufirst%3DR.%26aulast%3DEgan%26aufirst%3DD.%2BA.%26aulast%3DGubbins%26aufirst%3DE.%2BJ.%26aulast%3DRueter%26aufirst%3DA.%26aulast%3DHoward%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DL.%2BK.%26aulast%3DPederson%26aufirst%3DT.%2BM.%26aulast%3DKrafft%26aufirst%3DG.%2BA.%26aulast%3DWang%26aufirst%3DG.%2BT.%26atitle%3DRecombinant%2520Human%2520Prorenin%2520from%2520CHO%2520Cells%253A%2520Expression%2520and%2520Purification%26jtitle%3DJ.%2520Protein%2520Chem.%26date%3D1990%26volume%3D9%26spage%3D663%26epage%3D672%26doi%3D10.1007%2Fbf01024761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kageyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akamine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kominami, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchiyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span> <span> </span><span class="NLM_article-title">Characterization of New Fluorogenic Substrates for the Rapid and Sensitive Assay of Cathepsin E and Cathepsin D</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1143</span>, <span class="refDoi"> DOI: 10.1093/oxfordjournals.jbchem.a022396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Foxfordjournals.jbchem.a022396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10348917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK1MXlsVygsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1999&pages=1137-1143&author=Y.+Yasudaauthor=T.+Kageyamaauthor=A.+Akamineauthor=M.+Shibataauthor=E.+Kominamiauthor=Y.+Uchiyamaauthor=K.+Yamamoto&title=Characterization+of+New+Fluorogenic+Substrates+for+the+Rapid+and+Sensitive+Assay+of+Cathepsin+E+and+Cathepsin+D&doi=10.1093%2Foxfordjournals.jbchem.a022396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of new fluorogenic substrates for the rapid and sensitive assay of cathepsin E and cathepsin D</span></div><div class="casAuthors">Yasuda, Yoshiyuki; Kageyama, Takashi; Akamine, Akifumi; Shibata, Masahiro; Kominami, Eiki; Uchiyama, Yasuo; Yamamoto, Kenji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1137-1143</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">Cathepsin E and cathepsin D are two major intracellular aspartic proteinases implicated in the physiol. and pathol. degrdn. of intra- and extracellular proteins.  In this study, we designed and constructed highly sensitive synthetic decapeptide substrates for assays of cathepsins E and D based on the known sequence specificities of their cleavage sites.  These substrates contain a highly fluorescent (7-methoxycoumarin-4-yl)acetyl (MOCAc) moiety and a quenching 2,4-dinitrophenyl (Dnp) group.  When the Phe-Phe bond is cleaved, the fluorescence at an excitation wavelength of 328 nm and emission wavelength of 393 increases due to diminished quenching resulting from the sepn. of the fluorescent and quenching moieties.  The first substrate, MOCAc-Gly-Lys-Pro-Ile-Leu-Phe-Phe-Arg-Leu-Lys(Dnp)γ-NH2, in which the Lys-Pro combination at positions P5 and P4 was designed for specific interaction with cathepsin E, is hydrolyzed equally well by cathepsins E and D (kcat/Km = 10.9 μM-1·s-1 for cathepsin E and 15.6 μM-1·s-1 for cathepsin D).  A very acidic pH optimum of 4.0 was obtained for both enzymes.  The second substrate, MOCAc-Gly-Lys-Pro-Ile-Ile-Phe-Phe-Arg-Leu-Lys(Dnp)γ-NH2, in which the isoleucine residue at position P2 was meant to increase the specificity for cathepsin E, is also hydrolyzed equally by both enzymes (kcat/Km = 12.2 μM-1·s-1 for cathepsin E and 16.3 μM-1·s-1 for cathepsin D).  The kcat/Km values for both substrates are greater than those for the best substrates for cathepsins E and D described so far.  Unfortunately, each substrate shows little discrimination between cathepsin E and cathepsin D, suggesting that amino acids at positions far from the cleavage site are important for discrimination between the two enzymes.  However, in combination with aspartic proteinase inhibitors, such as pepstatin A and Ascaris pepsin inhibitor, these substrates enable a rapid and sensitive detn. of the precise levels of cathepsins E and D in crude cell exts. of various tissues and cells.  Thus these substrates represent a potentially valuable tool for routine assays and for mechanistic studies on cathepsins E and D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhuyRvnqRnebVg90H21EOLACvtfcHk0ljnTxMxQR2lqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlsVygsr4%253D&md5=23d0e8c891cf0ed5c38fa1191d18ec9a</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.jbchem.a022396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.jbchem.a022396%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DY.%26aulast%3DKageyama%26aufirst%3DT.%26aulast%3DAkamine%26aufirst%3DA.%26aulast%3DShibata%26aufirst%3DM.%26aulast%3DKominami%26aufirst%3DE.%26aulast%3DUchiyama%26aufirst%3DY.%26aulast%3DYamamoto%26aufirst%3DK.%26atitle%3DCharacterization%2520of%2520New%2520Fluorogenic%2520Substrates%2520for%2520the%2520Rapid%2520and%2520Sensitive%2520Assay%2520of%2520Cathepsin%2520E%2520and%2520Cathepsin%2520D%26jtitle%3DJ.%2520Biochem.%26date%3D1999%26volume%3D125%26spage%3D1137%26epage%3D1143%26doi%3D10.1093%2Foxfordjournals.jbchem.a022396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">The MX Beamlines BL14.1-3 at BESSY II</span>. <i>J. Large-scale Res. Facil.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">47</span>, <span class="refDoi"> DOI: 10.17815/jlsrf-2-64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.17815%2Fjlsrf-2-64" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=47&author=M.+Gerlachauthor=U.+Muellerauthor=M.+S.+Weiss&title=The+MX+Beamlines+BL14.1-3+at+BESSY+II&doi=10.17815%2Fjlsrf-2-64"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.17815%2Fjlsrf-2-64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.17815%252Fjlsrf-2-64%26sid%3Dliteratum%253Aachs%26aulast%3DGerlach%26aufirst%3DM.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DThe%2520MX%2520Beamlines%2520BL14.1-3%2520at%2520BESSY%2520II%26jtitle%3DJ.%2520Large-scale%2520Res.%2520Facil.%26date%3D2016%26volume%3D2%26spage%3D47%26doi%3D10.17815%2Fjlsrf-2-64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, Scaling, Space-group Assignment and Post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+Scaling%2C+Space-group+Assignment+and+Post-refinement&doi=10.1107%2Fs0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0liS1FfXF_n19g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520Scaling%252C%2520Space-group%2520Assignment%2520and%2520Post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2Fs0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of theCCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+theCCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0liS1FfXF_n19g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520theCCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an Automated Program for Molecular Replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/s0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+Automated+Program+for+Molecular+Replacement&doi=10.1107%2Fs0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgXTzUdKeLdFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520Automated%2520Program%2520for%2520Molecular%2520Replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2Fs0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 Dictionary: Organization of Prior Chemical Knowledge and Guidelines for its Use</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2184</span>– <span class="NLM_lpage">2195</span>, <span class="refDoi"> DOI: 10.1107/s0907444904023510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0907444904023510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=15572771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2184-2195&author=A.+A.+Vaginauthor=R.+A.+Steinerauthor=A.+A.+Lebedevauthor=L.+Pottertonauthor=S.+McNicholasauthor=F.+Longauthor=G.+N.+Murshudov&title=REFMAC5+Dictionary%3A+Organization+of+Prior+Chemical+Knowledge+and+Guidelines+for+its+Use&doi=10.1107%2Fs0907444904023510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 dictionary: organization of prior chemical knowledge and guidelines for its use</span></div><div class="casAuthors">Vagin, Alexei A.; Steiner, Roberto A.; Lebedev, Andrey A.; Potterton, Liz; McNicholas, Stuart; Long, Fei; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2184-2195</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">One of the most important aspects of macromol. structure refinement is the use of prior chem. knowledge.  Bond lengths, bond angles and other chem. properties are used in restrained refinement as subsidiary conditions.  This contribution describes the organization and some aspects of the use of the flexible and human/machine-readable dictionary of prior chem. knowledge used by the max.-likelihood macromol.-refinement program REFMAC5.  The dictionary stores information about monomers which represent the constitutive building blocks of biol. macromols. (amino acids, nucleic acids and saccharides) and about numerous org./inorg. compds. commonly found in macromol. crystallog.  It also describes the modifications the building blocks undergo as a result of chem. reactions and the links required for polymer formation.  More than 2000 monomer entries, 100 modification entries and 200 link entries are currently available.  Algorithms and tools for updating and adding new entries to the dictionary have also been developed and are presented here.  In many cases, the REFMAC5 dictionary allows entirely automatic generation of restraints within REFMAC5 refinement runs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRjDd-k3YHGrVg90H21EOLACvtfcHk0ljC0IFR5cn60Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73F&md5=005910422c5fb72fe2434c39b5321bf4</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1107%2FS0907444904023510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904023510%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPotterton%26aufirst%3DL.%26aulast%3DMcNicholas%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DREFMAC5%2520Dictionary%253A%2520Organization%2520of%2520Prior%2520Chemical%2520Knowledge%2520and%2520Guidelines%2520for%2520its%2520Use%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2184%26epage%3D2195%26doi%3D10.1107%2Fs0907444904023510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szalek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowska-Wlodarczyk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaczmarek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Highly Sensitive and Adaptable Fluorescence-quenched Pair Discloses the Substrate Specificity Profiles in Diverse Protease Families</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">43135</span>, <span class="refDoi"> DOI: 10.1038/srep43135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1038%2Fsrep43135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28230157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVGjtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=43135&author=M.+Porebaauthor=A.+Szalekauthor=W.+Rutauthor=P.+Kasperkiewiczauthor=I.+Rutkowska-Wlodarczykauthor=S.+J.+Snipasauthor=Y.+Itohauthor=D.+Turkauthor=B.+Turkauthor=C.+M.+Overallauthor=L.+Kaczmarekauthor=G.+S.+Salvesenauthor=M.+Drag&title=Highly+Sensitive+and+Adaptable+Fluorescence-quenched+Pair+Discloses+the+Substrate+Specificity+Profiles+in+Diverse+Protease+Families&doi=10.1038%2Fsrep43135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Highly sensitive and adaptable fluorescence-quenched pair discloses the substrate specificity profiles in diverse protease families</span></div><div class="casAuthors">Poreba, Marcin; Szalek, Aleksandra; Rut, Wioletta; Kasperkiewicz, Paulina; Rutkowska-Wlodarczyk, Izabela; Snipas, Scott J.; Itoh, Yoshifumi; Turk, Dusan; Turk, Boris; Overall, Christopher M.; Kaczmarek, Leszek; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43135</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Internally quenched fluorescent (IQF) peptide substrates originating from FRET (Forster Resonance Energy Transfer) are powerful tool for examg. the activity and specificity of proteases, and a variety of donor/acceptor pairs are extensively used to design individual substrates and combinatorial libraries.  We developed a highly sensitive and adaptable donor/acceptor pair that can be used to investigate the substrate specificity of cysteine proteases, serine proteases and metalloproteinases.  This novel pair comprises 7-amino-4-carbamoylmethylcoumarin (ACC) as the fluorophore and 2,4-dinitrophenyl-lysine (Lys(DNP)) as the quencher.  Using caspase-3, caspase-7, caspase-8, neutrophil elastase, legumain, and two matrix metalloproteinases (MMP2 and MMP9), we demonstrated that substrates contg. ACC/Lys(DNP) exhibit 7 to 10 times higher sensitivity than conventional 7-methoxy-coumarin-4-yl acetic acid (MCA)/Lys(DNP) substrates; thus, substantially lower amts. of substrate and enzyme can be used for each assay.  We therefore propose that the ACC/Lys(DNP) pair can be considered a novel and sensitive scaffold for designing substrates for any group of endopeptidases.  We further demonstrate that IQF substrates contg. unnatural amino acids can be used to investigate protease activities/specificities for peptides contg. post-translationally modified amino acids.  Finally, we used IQF substrates to re-investigate the P1-Asp characteristic of caspases, thus demonstrating that some human caspases can also hydrolyze substrates after glutamic acid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe_raSFJMVd7Vg90H21EOLACvtfcHk0lheKkHFtEa_1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsVGjtLs%253D&md5=acc661ca9d9b4f2f0e65d3d317b9290f</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fsrep43135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep43135%26sid%3Dliteratum%253Aachs%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSzalek%26aufirst%3DA.%26aulast%3DRut%26aufirst%3DW.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DRutkowska-Wlodarczyk%26aufirst%3DI.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTurk%26aufirst%3DD.%26aulast%3DTurk%26aufirst%3DB.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26aulast%3DKaczmarek%26aufirst%3DL.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DHighly%2520Sensitive%2520and%2520Adaptable%2520Fluorescence-quenched%2520Pair%2520Discloses%2520the%2520Substrate%2520Specificity%2520Profiles%2520in%2520Diverse%2520Protease%2520Families%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D43135%26doi%3D10.1038%2Fsrep43135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref58':['cit58'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref65':['cit65'],'ref66':['cit66'],'ref63':['cit63'],'ref64':['cit64'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Diagram of constructs used to identify May1 architecture and optimize expression strategy. Residue numbers indicate construct boundaries. The first 17 N-terminal amino acids, forming a putative native secretion signal, were replaced with a secretion signal compatible with the S2 expression system (not shown for clarity). Beveled rectangles denote positions of affinity tags used for purification.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Analysis of May1(17-434)-Avi expression and purification. (a) Silver-stained SDS-PAGE of samples collected throughout the purification process. MED, crude S2 media; PER, permeate from media concentration; LOAD, concentrated filtrate; FT, flow-through; W1-3, wash fractions; E1-4, elution fractions; POOL, pooled concentrated elution fractions; MTX, matrix binding control. (b) Western blot analysis with the same layout as in a visualized using Streptavidin-800CW (LI-COR, 1:15,000). (c) Silver-stained SDS-PAGE of concentrated pooled fractions after size exclusion chromatography.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Substrate specificity profile of May1 at the P4–P1′ positions presented as a heat map. L and D rows indicate the amino acid enantiomer at a given position. Dashed lines indicate substrate cleavage sites detected by LC–MS, with the most abundant N-proximal Phe*Leu site selected as a residue numbering reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Crystal structure of recombinant May1 in free form (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5H">6R5H</a>). The catalytic Asp residues are highlighted as magenta sticks. (b) Superposition of May1 (green) with human renin (magenta, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2REN">2REN</a><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. PepA binding to May1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R6A">6R6A</a>). (a) Top view of the binding pose of PepA in the May1 active site. The 2<i>Fo</i>-<i>Fc</i> map (contoured at 1.5σ) is shown in blue; catalytic aspartates (sticks with carbons colored cyan) interact with the central statine residue. Residues contributing to PepA binding in the S4–S3 subsites are indicated; those forming polar interactions are underlined. (b) Ligand–protein interaction diagram of the May1–PepA complex generated by LigPlot.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(50)</a> Residues engaged in polar ligand interactions (green dashed lines) are shown; numbers represent distances between hydrogen bond donor and acceptor in Å. (c) Model of N-terminally carboxybenzylated phenylstatine superimposed with PepA. Points of attachment for the R1 and R2 substituents are indicated by arrows. The protein is represented by its solvent accessible surface area colored by electrostatic potential (red for negative, blue for positive). Residues available to form polar interactions with the P1′ substituent are shown as sticks and labeled, and possible interactions are indicated by dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Linear statin inhibitor structures and residue preference heatmaps. L and D rows indicate the amino acid enantiomer at a given position. (a) Initial screening of the N-proximal site with a degenerate C-terminal residue. A final concentration of 500 nM per individual compound was used in library screening. (b) Deconvolution of selected libraries with favorable N-proximal residues based on initial screening. Single-compound deconvolution experiments were conducted with a 1 μM final concentration of the test compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Evaluation of May1 inhibitor activity against cultured <i>C. neoformans</i> yeast. (a) Terminal saturation densities of yeast cultured for 48 h in YNB minimal medium containing May1 inhibitors or a 0.5% DMSO vehicle. Dashed lines represent terminal OD<sub>600</sub>’s of the DMSO-treated H99 and <i>may1</i>Δ controls. Error bars represent standard deviation of the mean from triplicate cultures. (b) Growth curves of H99 and <i>may1</i>Δ yeast treated with lead compound <b>13a</b>, negative control compound <b>1b</b>, or a 0.5% DMSO vehicle control in either standard YNB medium (left) or YNB medium buffered to pH 6.5 (right). Error bars represent standard deviation of the mean from triplicate cultures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/medium/jm0c02177_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Comparison of May1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5H">6R5H</a>) with HIV-1 Pr (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2hvp">2hvp</a><a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a>). (a) Overall superposition of May1 (green) with HIV-1 Pr (maroon). Catalytic aspartates are highlighted in stick representation. (b) Top view into the May1 active site. The protein is represented by the solvent accessible surface area colored by electrostatic potential; PepA is represented as sticks with carbon atoms colored green; the inhibitor pose in HIV-1 Pr is shown with black lines. (c) Top view into the HIV-1 Pr active site. The protein is represented by solvent accessible surface area colored by electrostatic potential; acetyl-pepstatin is represented as sticks with carbon atoms colored yellow; the PepA pose in May1 is shown with black lines. In b and c, flap regions covering the active site are omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.0c02177/20210520/images/large/jm0c02177_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02177&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41358" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41358" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 81 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon-Chung, K. J.</span>; <span class="NLM_string-name">Boekhout, T.</span>; <span class="NLM_string-name">Wickes, B. L.</span>; <span class="NLM_string-name">Fell, J. W.</span></span> <span> </span><span class="NLM_article-title">Systematics of the Genus Cryptococcus and its Type Species C. Neoformans</span>.  <i>Cryptococcus</i>; <span class="NLM_publisher-name">ASM Press</span>, <span class="NLM_year">2011</span>; Vol.  <span class="NLM_volume">1</span>; pp  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">15</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=3-15&author=K.+J.+Kwon-Chung&author=T.+Boekhout&author=B.+L.+Wickes&author=J.+W.+Fell&title=Cryptococcus"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKwon-Chung%26aufirst%3DK.%2BJ.%26atitle%3DSystematics%2520of%2520the%2520Genus%2520Cryptococcus%2520and%2520its%2520Type%2520Species%2520C.%2520Neoformans%26btitle%3DCryptococcus%26pub%3DASM%2520Press%26date%3D2011%26volume%3D1%26spage%3D3%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Post, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaworowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, S. R.</span></span> <span> </span><span class="NLM_article-title">HIV and Co-infections</span>. <i>Immunol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>254</i></span>,  <span class="NLM_fpage">114</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1111/imr.12063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1111%2Fimr.12063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=23772618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFyit7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=254&publication_year=2013&pages=114-142&author=C.+C.+Changauthor=M.+Craneauthor=J.+Zhouauthor=M.+Minaauthor=J.+J.+Postauthor=B.+A.+Cameronauthor=A.+R.+Lloydauthor=A.+Jaworowskiauthor=M.+A.+Frenchauthor=S.+R.+Lewin&title=HIV+and+Co-infections&doi=10.1111%2Fimr.12063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">HIV and co-infections</span></div><div class="casAuthors">Chang, Christina C.; Crane, Megan; Zhou, Jingling; Mina, Michael; Post, Jeffrey J.; Cameron, Barbara A.; Lloyd, Andrew R.; Jaworowski, Anthony; French, Martyn A.; Lewin, Sharon R.</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">254</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">114-142, 29 pp.</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Despite significant redns. in morbidity and mortality secondary to availability of effective combination anti-retroviral therapy (cART), human immunodeficiency virus (HIV) infection still accounts for 1.5 million deaths annually.  The majority of deaths occur in sub-Saharan Africa where rates of opportunistic co-infections are disproportionately high.  In this review, we discuss the immunopathogenesis of five common infections that cause significant morbidity in HIV-infected patients globally.  These include co-infection with Mycobacterium tuberculosis, Cryptococcus neoformans, hepatitis B virus, hepatitis C virus, and Plasmodium falciparum.  Specifically, we review the natural history of each co-infection in the setting of HIV, the specific immune defects induced by HIV, the effects of cART on the immune response to the co-infection, the pathogenesis of immune restoration disease (IRD) assocd. with each infection, and advances in the areas of prevention of each co-infection via vaccination.  Finally, we discuss the opportunities and gaps in knowledge for future research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqGCu0bX3blrVg90H21EOLACvtfcHk0lh9pbT5DVO1Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFyit7zP&md5=609076465668935e01c1e3172db10462</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fimr.12063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fimr.12063%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.%2BC.%26aulast%3DCrane%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DMina%26aufirst%3DM.%26aulast%3DPost%26aufirst%3DJ.%2BJ.%26aulast%3DCameron%26aufirst%3DB.%2BA.%26aulast%3DLloyd%26aufirst%3DA.%2BR.%26aulast%3DJaworowski%26aufirst%3DA.%26aulast%3DFrench%26aufirst%3DM.%2BA.%26aulast%3DLewin%26aufirst%3DS.%2BR.%26atitle%3DHIV%2520and%2520Co-infections%26jtitle%3DImmunol.%2520Rev.%26date%3D2013%26volume%3D254%26spage%3D114%26epage%3D142%26doi%3D10.1111%2Fimr.12063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jarvis, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, T. S.</span></span> <span> </span><span class="NLM_article-title">HIV-associated Cryptococcal Meningitis</span>. <i>AIDS</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2119</span>– <span class="NLM_lpage">2129</span>, <span class="refDoi"> DOI: 10.1097/qad.0b013e3282a4a64d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1097%2FQAD.0b013e3282a4a64d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=18090038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD2sjlsV2ktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=2119-2129&author=J.+N.+Jarvisauthor=T.+S.+Harrison&title=HIV-associated+Cryptococcal+Meningitis&doi=10.1097%2Fqad.0b013e3282a4a64d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-associated cryptococcal meningitis</span></div><div class="casAuthors">Jarvis Joseph N; Harrison Thomas S</div><div class="citationInfo"><span class="NLM_cas:title">AIDS (London, England)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2119-29</span>
        ISSN:<span class="NLM_cas:issn">0269-9370</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkLKSquk7kKnthwrmtH4OBfW6udTcc2eaKA0qX_NCsWbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjlsV2ktw%253D%253D&md5=e536a9b693287f84a59abb1348999f3b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1097%2FQAD.0b013e3282a4a64d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQAD.0b013e3282a4a64d%26sid%3Dliteratum%253Aachs%26aulast%3DJarvis%26aufirst%3DJ.%2BN.%26aulast%3DHarrison%26aufirst%3DT.%2BS.%26atitle%3DHIV-associated%2520Cryptococcal%2520Meningitis%26jtitle%3DAIDS%26date%3D2007%26volume%3D21%26spage%3D2119%26epage%3D2129%26doi%3D10.1097%2Fqad.0b013e3282a4a64d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsueh, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geunes-Boyer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, J.</span></span> <span> </span><span class="NLM_article-title">Spores as Infectious Propagules of Cryptococcus Neoformans</span>. <i>Infect. Immun.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">4345</span>– <span class="NLM_lpage">4355</span>, <span class="refDoi"> DOI: 10.1128/iai.00542-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FIAI.00542-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=19620339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Ghu7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2009&pages=4345-4355&author=R.+Velagapudiauthor=Y.-P.+Hsuehauthor=S.+Geunes-Boyerauthor=J.+R.+Wrightauthor=J.+Heitman&title=Spores+as+Infectious+Propagules+of+Cryptococcus+Neoformans&doi=10.1128%2Fiai.00542-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Spores as infectious propagules of Cryptococcus neoformans</span></div><div class="casAuthors">Velagapudi, Rajesh; Hsueh, Yen-Ping; Geunes-Boyer, Scarlett; Wright, Jo Rae; Heitman, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">Infection and Immunity</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4345-4355</span>CODEN:
                <span class="NLM_cas:coden">INFIBR</span>;
        ISSN:<span class="NLM_cas:issn">0019-9567</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cryptococcus neoformans and Cryptococcus gattii are closely related pathogenic fungi that cause pneumonia and meningitis in both immunocompromised and immunocompetent hosts and are a significant global infectious disease risk.  Both species are found in the environment and are acquired via inhalation, leading to an initial pulmonary infection.  The infectious propagule is unknown but is hypothesized to be small desiccated yeast cells or spores produced by sexual reprodn. (opposite- or same-sex mating).  Here we characterize the morphol., germination properties, and virulence of spores.  A comparative morphol. anal. of hyphae and spores produced by opposite-sex mating, same-sex mating, and self-fertile diploid strains was conducted by SEM, yielding insight into hyphal/basidial morphol. and spore size, structure, and surface properties.  Spores isolated by microdissection were found to readily germinate even on water agarose medium.  Thus, nutritional signals do not appear to be required to stimulate spore germination, and as-yet-unknown environmental factors may normally constrain germination in nature.  As few as 500 CFU of a spore-enriched infectious inoculum (∼95% spores) of serotype A C. neoformans var. grubii were fully virulent (100% lethal infection) in both a murine inhalation virulence model and the invertebrate model host Galleria mellonella.  In contrast to a previous report on C. neoformans var. neoformans, spores of C. neoformans var. grubii were not more infectious than yeast cells.  Mol. anal. of isolates recovered from tissues of infected mice (lung, spleen, and brain) provides evidence for infection and dissemination by recombinant spore products.  These studies provide a detailed morphol. and physiol. anal. of the spore and document that spores can serve as infectious propagules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSnUo4GlO8UbVg90H21EOLACvtfcHk0lhkjkoG0cmLSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Ghu7rN&md5=37f0b4cd4af6ad76b307773e7b0af7bc</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FIAI.00542-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FIAI.00542-09%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DR.%26aulast%3DHsueh%26aufirst%3DY.-P.%26aulast%3DGeunes-Boyer%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DJ.%2BR.%26aulast%3DHeitman%26aufirst%3DJ.%26atitle%3DSpores%2520as%2520Infectious%2520Propagules%2520of%2520Cryptococcus%2520Neoformans%26jtitle%3DInfect.%2520Immun.%26date%3D2009%26volume%3D77%26spage%3D4345%26epage%3D4355%26doi%3D10.1128%2Fiai.00542-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, H. I.</span></span> <span> </span><span class="NLM_article-title">Not Your “Typical Patient”: Cryptococcal Meningitis in an Immunocompetent Patient</span>. <i>J. Neurosci. Nurs.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">144</span>– <span class="NLM_lpage">148</span>, <span class="refDoi"> DOI: 10.1097/01376517-200506000-00005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1097%2F01376517-200506000-00005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=16001819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD2MzkvFyguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2005&pages=144-148&author=H.+I.+Thompson&title=Not+Your+%E2%80%9CTypical+Patient%E2%80%9D%3A+Cryptococcal+Meningitis+in+an+Immunocompetent+Patient&doi=10.1097%2F01376517-200506000-00005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Not your "typical patient": cryptococcal meningitis in an immunocompetent patient</span></div><div class="casAuthors">Thompson Hilaire J</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of neuroscience nursing : journal of the American Association of Neuroscience Nurses</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">144-8</span>
        ISSN:<span class="NLM_cas:issn">0888-0395</span>.
    </div><div class="casAbstract">Meningitis, when caused by the fungal mycoses Cryptococcus neoformans, is normally seen in immuno-compromised hosts.  However, immunocompetent patients are also susceptible to cryptococcal meningitis (CM).  In patients with an intact immune system, CM usually presents with the typical signs and symptoms of meningitis: fever, stiff neck, and headache.  Major implications for the primary and advanced practice nursing plans of care for CM patients include a thorough history and physical exam, early diagnosis and treatment, and an individualized plan of care focused on minimizing sequelae and side effects of treatment and maximizing functional recovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQLU3fjt9OUP36oPH0mYWL1fW6udTcc2ebtm4uhv_Wfprntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MzkvFyguw%253D%253D&md5=9ffdbce31dc09be1b7c873bb041b85f9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1097%2F01376517-200506000-00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F01376517-200506000-00005%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DH.%2BI.%26atitle%3DNot%2520Your%2520%25E2%2580%259CTypical%2520Patient%25E2%2580%259D%253A%2520Cryptococcal%2520Meningitis%2520in%2520an%2520Immunocompetent%2520Patient%26jtitle%3DJ.%2520Neurosci.%2520Nurs.%26date%3D2005%26volume%3D37%26spage%3D144%26epage%3D148%26doi%3D10.1097%2F01376517-200506000-00005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lui, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tso, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockram, C. S.</span></span> <span> </span><span class="NLM_article-title">Cryptococcosis in Apparently Immunocompetent Patients</span>. <i>Q. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1093/qjmed/hcl014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fqjmed%2Fhcl014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2006&pages=143-151&author=G.+Luiauthor=N.+Leeauthor=M.+Ipauthor=K.+W.+Choiauthor=Y.+K.+Tsoauthor=E.+Lamauthor=S.+Chauauthor=R.+Laiauthor=C.+S.+Cockram&title=Cryptococcosis+in+Apparently+Immunocompetent+Patients&doi=10.1093%2Fqjmed%2Fhcl014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fqjmed%2Fhcl014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fqjmed%252Fhcl014%26sid%3Dliteratum%253Aachs%26aulast%3DLui%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DN.%26aulast%3DIp%26aufirst%3DM.%26aulast%3DChoi%26aufirst%3DK.%2BW.%26aulast%3DTso%26aufirst%3DY.%2BK.%26aulast%3DLam%26aufirst%3DE.%26aulast%3DChau%26aufirst%3DS.%26aulast%3DLai%26aufirst%3DR.%26aulast%3DCockram%26aufirst%3DC.%2BS.%26atitle%3DCryptococcosis%2520in%2520Apparently%2520Immunocompetent%2520Patients%26jtitle%3DQ.%2520J.%2520Med.%26date%3D2006%26volume%3D99%26spage%3D143%26epage%3D151%26doi%3D10.1093%2Fqjmed%2Fhcl014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pappas, P. G.</span></span> <span> </span><span class="NLM_article-title">Cryptococcal Infections in Non-HIV-infected Patients</span>. <i>Trans. Am. Clin. Climatol. Assoc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">79</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=23874010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC3sfislamsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2013&pages=61-79&author=P.+G.+Pappas&title=Cryptococcal+Infections+in+Non-HIV-infected+Patients"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Cryptococcal infections in non-HIV-infected patients</span></div><div class="casAuthors">Pappas Peter G</div><div class="citationInfo"><span class="NLM_cas:title">Transactions of the American Clinical and Climatological Association</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">61-79</span>
        ISSN:<span class="NLM_cas:issn">0065-7778</span>.
    </div><div class="casAbstract">Infections due to Cryptococcus species occur globally and in a wide variety of hosts, ranging from those who are severely immunosuppressed to those who have phenotypically "normal" immune systems.  Approximately 1 million cases of cryptococcosis occur throughout the world, and is it estimated that there are 650,000 associated deaths annually.  Most of these cases occur among patients with advanced HIV disease, but a growing number occur among solid organ transplant recipients and others receiving exogenous immunosuppression, patients with innate and acquired immunodeficiency, and otherwise immunologically normal hosts.  Much of our recent knowledge is solely derived from clinical experience over the last 2 to 3 decades of cryptococcosis among HIV-infected patients.  However, based on recent observations, it is clear that there are substantial differences in the epidemiology, clinical features, approaches to therapy, and outcome when comparing HIV-infected to non-HIV-infected individuals who have cryptococcosis.  If one carefully examines cryptococcosis in the three largest subgroups of patients based on host immune status, specifically, those with HIV, solid organ transplant recipients, and those who are non-HIV, non-transplant (NHNT) infected persons, then one can observe very different risks for infection, varied clinical presentations, long-term complications, mortality, and approaches to therapy.  This article focuses on cryptococcosis in the non-HIV-infected patient, including a brief review of ongoing events in the Pacific Northwest of the United States and Canada relative to the outbreak of Cryptococcus gattii infections among a largely immunologically normal population, and highlights some of the key insights and questions which have emerged as a result of these important new observations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqArOx8qswLsKL0FlyGHDofW6udTcc2eaLU4c0q7V89bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfislamsQ%253D%253D&md5=44b3401b3e9707d6a071bd96a558f729</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPappas%26aufirst%3DP.%2BG.%26atitle%3DCryptococcal%2520Infections%2520in%2520Non-HIV-infected%2520Patients%26jtitle%3DTrans.%2520Am.%2520Clin.%2520Climatol.%2520Assoc.%26date%3D2013%26volume%3D124%26spage%3D61%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajasingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarvis, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govender, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiller, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denning, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loyse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulware, D. R.</span></span> <span> </span><span class="NLM_article-title">Global Burden of Disease of HIV-associated Cryptococcal Meningitis: an Updated Analysis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">873</span>– <span class="NLM_lpage">881</span>, <span class="refDoi"> DOI: 10.1016/s1473-3099(17)30243-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2FS1473-3099%2817%2930243-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28483415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC1crlt1Sjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=873-881&author=R.+Rajasinghamauthor=R.+M.+Smithauthor=B.+J.+Parkauthor=J.+N.+Jarvisauthor=N.+P.+Govenderauthor=T.+M.+Chillerauthor=D.+W.+Denningauthor=A.+Loyseauthor=D.+R.+Boulware&title=Global+Burden+of+Disease+of+HIV-associated+Cryptococcal+Meningitis%3A+an+Updated+Analysis&doi=10.1016%2Fs1473-3099%2817%2930243-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Global burden of disease of HIV-associated cryptococcal meningitis: an updated analysis</span></div><div class="casAuthors">Rajasingham Radha; Smith Rachel M; Park Benjamin J; Chiller Tom M; Jarvis Joseph N; Govender Nelesh P; Denning David W; Loyse Angela; Boulware David R</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">873-881</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Cryptococcus is the most common cause of meningitis in adults living with HIV in sub-Saharan Africa.  Global burden estimates are crucial to guide prevention strategies and to determine treatment needs, and we aimed to provide an updated estimate of global incidence of HIV-associated cryptococcal disease.  METHODS:  We used 2014 Joint UN Programme on HIV and AIDS estimates of adults (aged >15 years) with HIV and antiretroviral therapy (ART) coverage.  Estimates of CD4 less than 100 cells per μL, virological failure incidence, and loss to follow-up were from published multinational cohorts in low-income and middle-income countries.  We calculated those at risk for cryptococcal infection, specifically those with CD4 less than 100 cells/μL not on ART, and those with CD4 less than 100 cells per μL on ART but lost to follow-up or with virological failure.  Cryptococcal antigenaemia prevalence by country was derived from 46 studies globally.  Based on cryptococcal antigenaemia prevalence in each country and region, we estimated the annual numbers of people who are developing and dying from cryptococcal meningitis.  FINDINGS:  We estimated an average global cryptococcal antigenaemia prevalence of 6·0% (95% CI 5·8-6·2) among people with a CD4 cell count of less than 100 cells per μL, with 278 000 (95% CI 195 500-340 600) people positive for cryptococcal antigen globally and 223 100 (95% CI 150 600-282 400) incident cases of cryptococcal meningitis globally in 2014.  Sub-Saharan Africa accounted for 73% of the estimated cryptococcal meningitis cases in 2014 (162 500 cases [95% CI 113 600-193 900]).  Annual global deaths from cryptococcal meningitis were estimated at 181 100 (95% CI 119 400-234 300), with 135 900 (75%; [95% CI 93 900-163 900]) deaths in sub-Saharan Africa.  Globally, cryptococcal meningitis was responsible for 15% of AIDS-related deaths (95% CI 10-19).  INTERPRETATION:  Our analysis highlights the substantial ongoing burden of HIV-associated cryptococcal disease, primarily in sub-Saharan Africa.  Cryptococcal meningitis is a metric of HIV treatment programme failure; timely HIV testing and rapid linkage to care remain an urgent priority.  FUNDING:  None.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6kthTNuMfLqNc3HjJRoFAfW6udTcc2eaLU4c0q7V89bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crlt1Sjug%253D%253D&md5=85268904374cb498fc2d92ac35cd4aee</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2817%2930243-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252817%252930243-8%26sid%3Dliteratum%253Aachs%26aulast%3DRajasingham%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DR.%2BM.%26aulast%3DPark%26aufirst%3DB.%2BJ.%26aulast%3DJarvis%26aufirst%3DJ.%2BN.%26aulast%3DGovender%26aufirst%3DN.%2BP.%26aulast%3DChiller%26aufirst%3DT.%2BM.%26aulast%3DDenning%26aufirst%3DD.%2BW.%26aulast%3DLoyse%26aufirst%3DA.%26aulast%3DBoulware%26aufirst%3DD.%2BR.%26atitle%3DGlobal%2520Burden%2520of%2520Disease%2520of%2520HIV-associated%2520Cryptococcal%2520Meningitis%253A%2520an%2520Updated%2520Analysis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2017%26volume%3D17%26spage%3D873%26epage%3D881%26doi%3D10.1016%2Fs1473-3099%2817%2930243-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong-James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meintjes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G. D.</span></span> <span> </span><span class="NLM_article-title">A Neglected Epidemic: Fungal Infections in HIV/AIDS</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">127</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2014.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.tim.2014.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=24530175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFehurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=120-127&author=D.+Armstrong-Jamesauthor=G.+Meintjesauthor=G.+D.+Brown&title=A+Neglected+Epidemic%3A+Fungal+Infections+in+HIV%2FAIDS&doi=10.1016%2Fj.tim.2014.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A neglected epidemic: fungal infections in HIV/AIDS</span></div><div class="casAuthors">Armstrong-James, Darius; Meintjes, Graeme; Brown, Gordon D.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">120-127</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Invasive fungal infections (IFIs) are a major cause of HIV-related mortality globally.  Despite widespread rollout of combined antiretroviral therapy, there are still up to 1 million deaths annually from IFIs, accounting for 50% of all AIDS-related death.  A historic failure to focus efforts on the IFIs that kill so many HIV patients has led to fundamental flaws in the management of advanced HIV infection.  This review, based on the EMBO AIDS-Related Mycoses Workshop in Cape Town in July 2013, summarizes the current state of the-art in AIDS-related mycoses, and the key action points required to improve outcomes from these devastating infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCS714GKJsybVg90H21EOLACvtfcHk0lhqiK5uttOTOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFehurg%253D&md5=e00bfe12c6824e9b55f3fdbfa09e3514</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2014.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2014.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DArmstrong-James%26aufirst%3DD.%26aulast%3DMeintjes%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DG.%2BD.%26atitle%3DA%2520Neglected%2520Epidemic%253A%2520Fungal%2520Infections%2520in%2520HIV%252FAIDS%26jtitle%3DTrends%2520Microbiol.%26date%3D2014%26volume%3D22%26spage%3D120%26epage%3D127%26doi%3D10.1016%2Fj.tim.2014.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">World Health Organization</span>.  <i>Guidelines for the Diagnosis, Prevention
and Management of Cryptococcal Disease in HIV-Infected Adults, Adolescents
and Children: Supplement to the 2016 Consolidated Guidelines on the
Use of Antiretroviral Drugs for Treating and Preventing HIV Infection</i>; <span class="NLM_publisher-name">World Health Organization</span>: <span class="NLM_publisher-loc">Geneva</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=World+Health+Organization&title=Guidelines+for+the+Diagnosis%2C+Prevention%0Aand+Management+of+Cryptococcal+Disease+in+HIV-Infected+Adults%2C+Adolescents%0Aand+Children%3A+Supplement+to+the+2016+Consolidated+Guidelines+on+the%0AUse+of+Antiretroviral+Drugs+for+Treating+and+Preventing+HIV+Infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DGuidelines%2520for%2520the%2520Diagnosis%252C%2520Prevention%250Aand%2520Management%2520of%2520Cryptococcal%2520Disease%2520in%2520HIV-Infected%2520Adults%252C%2520Adolescents%250Aand%2520Children%253A%2520Supplement%2520to%2520the%25202016%2520Consolidated%2520Guidelines%2520on%2520the%250AUse%2520of%2520Antiretroviral%2520Drugs%2520for%2520Treating%2520and%2520Preventing%2520HIV%2520Infection%26pub%3DWorld%2520Health%2520Organization%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span> </span><span class="NLM_article-title">Fluconazole Injection Current Drug Shortages</span>. <a href="https://www.ashp.org/Drug-Shortages/Current-Shortages/Drug-Shortage-Detail.aspx?id=318" class="extLink">https://www.ashp.org/Drug-Shortages/Current-Shortages/Drug-Shortage-Detail.aspx?id=318</a> (accessed Mar 19, 2021).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Fluconazole+Injection+Current+Drug+Shortages.+https%3A%2F%2Fwww.ashp.org%2FDrug-Shortages%2FCurrent-Shortages%2FDrug-Shortage-Detail.aspx%3Fid%3D318+%28accessed+Mar+19%2C+2021%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DFluconazole%2520Injection%2520Current%2520Drug%2520Shortages" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sloan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedicoat, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalloo, D. G.</span></span> <span> </span><span class="NLM_article-title">Treatment of Cryptococcal Meningitis in Resource Limited Settings</span>. <i>Curr. Opin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">463</span>, <span class="refDoi"> DOI: 10.1097/qco.0b013e32832fa214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1097%2FQCO.0b013e32832fa214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=19587589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD1MngtF2ktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2009&pages=455-463&author=D.+J.+Sloanauthor=M.+J.+Dedicoatauthor=D.+G.+Lalloo&title=Treatment+of+Cryptococcal+Meningitis+in+Resource+Limited+Settings&doi=10.1097%2Fqco.0b013e32832fa214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of cryptococcal meningitis in resource limited settings</span></div><div class="casAuthors">Sloan Derek J; Dedicoat Martin J; Lalloo David G</div><div class="citationInfo"><span class="NLM_cas:title">Current opinion in infectious diseases</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">455-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PURPOSE OF REVIEW:  Cryptococcal meningitis most commonly occurs in advanced HIV.  Although diminishing in the developed world with antiretroviral therapy (ART), it remains a major problem in resource-limited settings.  ART rollout will improve long-term HIV survival if opportunistic infections are effectively treated.  Considering cryptococcal meningitis in that context, this review addresses excess morbidity and mortality in developing countries, treatment in areas of limited drug availability and challenges posed by combined anticryptococcal and HIV therapy.  RECENT FINDINGS:  From Early Fungicidal Activity (EFA) studies, amphotericin B-flucytosine is best induction therapy but often unavailable; high dose amphotericin B monotherapy may be feasible in some settings.  Where fluconazole is the only option, higher doses are more fungicidal.  Serum cryptococcal antigen testing may identify patients at highest disease risk and primary prophylaxis is effective; the clinical role of such interventions needs to be established.  Timing of ART introduction remains controversial; early initiation risks Immune Reconstitution Disease (IRD) delays may increase mortality.  SUMMARY:  Amphotericin B based treatment is appropriate where possible.  More studies are needed to optimize fluconazole monotherapy doses.  Other research priorities include management of raised intracranial pressure, appropriate ART initiation and IRD treatment.  Studies should focus on developing countries where problems are greatest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTE316TRMI4BZHZzFRCXxVKfW6udTcc2ebl3qaL5SjxB7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MngtF2ktA%253D%253D&md5=5e0b9541ed4d82fadec3eca2489b8f6f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1097%2FQCO.0b013e32832fa214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQCO.0b013e32832fa214%26sid%3Dliteratum%253Aachs%26aulast%3DSloan%26aufirst%3DD.%2BJ.%26aulast%3DDedicoat%26aufirst%3DM.%2BJ.%26aulast%3DLalloo%26aufirst%3DD.%2BG.%26atitle%3DTreatment%2520of%2520Cryptococcal%2520Meningitis%2520in%2520Resource%2520Limited%2520Settings%26jtitle%3DCurr.%2520Opin.%2520Infect.%2520Dis.%26date%3D2009%26volume%3D22%26spage%3D455%26epage%3D463%26doi%3D10.1097%2Fqco.0b013e32832fa214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiederhold, N. P.</span></span> <span> </span><span class="NLM_article-title">Antifungal Resistance: Current Trends and Future Strategies to Combat</span>. <i>Infect. Drug Resist.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.2147/idr.s124918</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.2147%2FIDR.S124918" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28919789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFags7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=249-259&author=N.+P.+Wiederhold&title=Antifungal+Resistance%3A+Current+Trends+and+Future+Strategies+to+Combat&doi=10.2147%2Fidr.s124918"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Antifungal resistance: current trends and future strategies to combat</span></div><div class="casAuthors">Wiederhold, Nathan P.</div><div class="citationInfo"><span class="NLM_cas:title">Infection and Drug Resistance</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">249-259</span>CODEN:
                <span class="NLM_cas:coden">IDRNAV</span>;
        ISSN:<span class="NLM_cas:issn">1178-6973</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Antifungal resistance represents a major clin. challenge to clinicians responsible for treating invasive fungal infections due to the limited arsenal of systemically available anti-fungal agents.  In addn. current drugs may be limited by drug-drug interactions and serious adverse effects/toxicities that prevent their prolonged use or dosage escalation.  Fluconazole resistance is of particular concern in non-Candida albicans species due to the increased incidence of infections caused by these species in different geog. locations worldwide and the elevated prevalence of resistance to this commonly used azole in many institutions.  C. glabrata resistance to the echinocandins has also been documented to be rising in several US institutions, and a higher percentage of these isolates may also be azole resistant.  Azole resistance in Aspergillus fumigatus due to clin. and environmental exposure to this class of agents has also been found worldwide, and these isolates can cause invasive infections with high mortality rates.  In addn., several species of Aspergillus, and other molds, including Scedosporium and Fusarium species, have reduced susceptibility or pan-resistance to clin. available antifungals.  Various investigational antifungals are currently in preclin. or clin. development, including several of them that have the potential to overcome resistance obsd. against the azoles and the echinocandins.  These include agents that also target ergosterol and β-glucan biosynthesis, as well as compds. with novel mechanisms of action that may also overcome the limitations of currently available antifungal classes, including both resistance and adverse effects/toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJkplFm02SHbVg90H21EOLACvtfcHk0ljl8QZPjVMDew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFags7Y%253D&md5=f187c81d848df8d551b5edd1f362ae13</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2147%2FIDR.S124918&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FIDR.S124918%26sid%3Dliteratum%253Aachs%26aulast%3DWiederhold%26aufirst%3DN.%2BP.%26atitle%3DAntifungal%2520Resistance%253A%2520Current%2520Trends%2520and%2520Future%2520Strategies%2520to%2520Combat%26jtitle%3DInfect.%2520Drug%2520Resist.%26date%3D2017%26volume%3D10%26spage%3D249%26epage%3D259%26doi%3D10.2147%2Fidr.s124918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caplan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowen, L. E.</span></span> <span> </span><span class="NLM_article-title">Molecular Evolution of Antifungal Drug Resistance</span>. <i>Annu. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">775</span>, <span class="refDoi"> DOI: 10.1146/annurev-micro-030117-020345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1146%2Fannurev-micro-030117-020345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28886681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKlurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=753-775&author=N.+Robbinsauthor=T.+Caplanauthor=L.+E.+Cowen&title=Molecular+Evolution+of+Antifungal+Drug+Resistance&doi=10.1146%2Fannurev-micro-030117-020345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Evolution of Antifungal Drug Resistance</span></div><div class="casAuthors">Robbins, Nicole; Caplan, Tavia; Cowen, Leah E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">753-775</span>CODEN:
                <span class="NLM_cas:coden">ARMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4227</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The fungal pathogens Candida albicans, Cryptococcus neoformans, and Aspergillus fumigatus have transitioned from a rare curiosity to a leading cause of human mortality.  The management of infections caused by these organisms is intimately dependent on the efficacy of antifungal agents; however, fungi that are resistant to these treatments are regularly isolated in the clinic, impeding our ability to control infections.  Given the significant impact fungal pathogens have on human health, it is imperative to understand the mol. mechanisms that govern antifungal drug resistance.  This review describes our current knowledge of the mechanisms by which antifungal drug resistance evolves in exptl. populations and clin. settings.  We explore current antifungal treatment options and discuss promising strategies to impede the evolution of drug resistance.  By tackling antifungal drug resistance as an evolutionary problem, there is potential to improve the utility of current treatments and accelerate the development of novel therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf_rS6B3dA4bVg90H21EOLACvtfcHk0ljl8QZPjVMDew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKlurvE&md5=2eb2fbfc778bce94065a35e0bd9bea3b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev-micro-030117-020345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-micro-030117-020345%26sid%3Dliteratum%253Aachs%26aulast%3DRobbins%26aufirst%3DN.%26aulast%3DCaplan%26aufirst%3DT.%26aulast%3DCowen%26aufirst%3DL.%2BE.%26atitle%3DMolecular%2520Evolution%2520of%2520Antifungal%2520Drug%2520Resistance%26jtitle%3DAnnu.%2520Rev.%2520Microbiol.%26date%3D2017%26volume%3D71%26spage%3D753%26epage%3D775%26doi%3D10.1146%2Fannurev-micro-030117-020345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joseph-Horne, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeffler, R. S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollomon, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, S. L.</span></span> <span> </span><span class="NLM_article-title">Amphotericin B resistant isolates ofCryptococcus neoformanswithout alteration in sterol biosynthesis</span>. <i>J. Med. Vet. Mycol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1080/02681219680000381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1080%2F02681219680000381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8803806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADyaK28zps1OjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1996&pages=223-225&author=T.+Joseph-Horneauthor=R.+S.+T.+Loefflerauthor=D.+W.+Hollomonauthor=S.+L.+Kelly&title=Amphotericin+B+resistant+isolates+ofCryptococcus+neoformanswithout+alteration+in+sterol+biosynthesis&doi=10.1080%2F02681219680000381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Amphotericin B resistant isolates of Cryptococcus neoformans without alteration in sterol biosynthesis</span></div><div class="casAuthors">Joseph-Horne T; Loeffler R S; Hollomon D W; Kelly S L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medical and veterinary mycology : bi-monthly publication of the International Society for Human and Animal Mycology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">223-5</span>
        ISSN:<span class="NLM_cas:issn">0268-1218</span>.
    </div><div class="casAbstract">Amphotericin B resistant mutants of Cryptococcus neoformans were isolated accumulating mainly ergosterol.  Cross-resistance to azole antifungals was not observed.  Together with previous data this indicates that at least three categories of amphotericin B resistance can arise: sterol mutants, amphotericin B and azole cross-resistant mutants and amphotericin B resistant mutants with no azole cross-resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_b8G2NhYRWntlzo9xastyfW6udTcc2easxHsU3Ccp1Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK28zps1OjsA%253D%253D&md5=70212d684742b588fce910b5e1a313f6</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1080%2F02681219680000381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F02681219680000381%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph-Horne%26aufirst%3DT.%26aulast%3DLoeffler%26aufirst%3DR.%2BS.%2BT.%26aulast%3DHollomon%26aufirst%3DD.%2BW.%26aulast%3DKelly%26aufirst%3DS.%2BL.%26atitle%3DAmphotericin%2520B%2520resistant%2520isolates%2520ofCryptococcus%2520neoformanswithout%2520alteration%2520in%2520sterol%2520biosynthesis%26jtitle%3DJ.%2520Med.%2520Vet.%2520Mycol.%26date%3D1996%26volume%3D34%26spage%3D223%26epage%3D225%26doi%3D10.1080%2F02681219680000381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loyse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dromer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lortholary, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, T. S.</span></span> <span> </span><span class="NLM_article-title">Flucytosine and Cryptococcosis: Time to Urgently Address the Worldwide Accessibility of a 50-year-old Antifungal</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">2435</span>– <span class="NLM_lpage">2444</span>, <span class="refDoi"> DOI: 10.1093/jac/dkt221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fjac%2Fdkt221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=23788479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CmurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2013&pages=2435-2444&author=A.+Loyseauthor=F.+Dromerauthor=J.+Dayauthor=O.+Lortholaryauthor=T.+S.+Harrison&title=Flucytosine+and+Cryptococcosis%3A+Time+to+Urgently+Address+the+Worldwide+Accessibility+of+a+50-year-old+Antifungal&doi=10.1093%2Fjac%2Fdkt221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Flucytosine and cryptococcosis: time to urgently address the worldwide accessibility of a 50-year-old antifungal</span></div><div class="casAuthors">Loyse, Angela; Dromer, Francoise; Day, Jeremy; Lortholary, Olivier; Harrison, Thomas S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2435-2444</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Current, widely accepted guidelines for the management of HIV-assocd. cryptococcal meningoencephalitis (CM) recommend amphotericin B combined with flucytosine (5-FC) for ≥2 wk as the initial induction treatment of choice.  However, access to flucytosine in Africa and Asia, where disease burden is greatest, is inadequate at present.  While research into identifying effective and well-tolerated antifungal combinations that do not contain flucytosine continues, an ever-increasing body of evidence from in vitro, in vivo, and clin. studies points to the benefits of flucytosine in the treatment of CM in both i.v. combinations with amphotericin B and oral combinations with high-dose fluconazole.  This article provides an up-to-date review of this evidence, and the current issues and challenges regarding increasing access to this key component of combination antifungal therapy for cryptococcosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeuqIQy-uf5rVg90H21EOLACvtfcHk0ljcBQgsUmG1rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CmurzP&md5=bb025d2005441ba2f42ea6ebddb20354</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkt221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkt221%26sid%3Dliteratum%253Aachs%26aulast%3DLoyse%26aufirst%3DA.%26aulast%3DDromer%26aufirst%3DF.%26aulast%3DDay%26aufirst%3DJ.%26aulast%3DLortholary%26aufirst%3DO.%26aulast%3DHarrison%26aufirst%3DT.%2BS.%26atitle%3DFlucytosine%2520and%2520Cryptococcosis%253A%2520Time%2520to%2520Urgently%2520Address%2520the%2520Worldwide%2520Accessibility%2520of%2520a%252050-year-old%2520Antifungal%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2013%26volume%3D68%26spage%3D2435%26epage%3D2444%26doi%3D10.1093%2Fjac%2Fdkt221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, T.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-H.</span></span> <span> </span><span class="NLM_article-title">Increasing Trend of Fluconazole-non-susceptible Cryptococcus Neoformans in Patients with Invasive Cryptococcosis: a 12-year Longitudinal Study</span>. <i>BMC Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">277</span>, <span class="refDoi"> DOI: 10.1186/s12879-015-1023-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1186%2Fs12879-015-1023-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=26194004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FmtFaisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=277&author=Y.-C.+Chenauthor=T.-Y.+Changauthor=J.-W.+Liuauthor=F.-J.+Chenauthor=C.-C.+Chienauthor=C.-H.+Leeauthor=C.-H.+Lu&title=Increasing+Trend+of+Fluconazole-non-susceptible+Cryptococcus+Neoformans+in+Patients+with+Invasive+Cryptococcosis%3A+a+12-year+Longitudinal+Study&doi=10.1186%2Fs12879-015-1023-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing trend of fluconazole-non-susceptible Cryptococcus neoformans in patients with invasive cryptococcosis: a 12-year longitudinal study</span></div><div class="casAuthors">Chen Yi-Chun; Chang Tzu-Yao; Liu Jien-Wei; Chen Fang-Ju; Lee Chen-Hsiang; Liu Jien-Wei; Lee Chen-Hsiang; Lu Cheng-Hsien; Chien Chun-Chih; Lu Cheng-Hsien; Lu Cheng-Hsien</div><div class="citationInfo"><span class="NLM_cas:title">BMC infectious diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">277</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  This study aimed to investigate the rate of fluconazole-non-susceptible Cryptococcus neoformans in Southern Taiwan for the period 2001-2012 and analyze the risk factors for acquiring it among patients with invasive cryptococcosis.  METHODS:  All enrolled strains were isolated from blood or cerebrospinal fluid samples of the included patients.  If a patient had multiple positive results for C. neoformans, only the first instance was enrolled.  Susceptibility testing was performed using the Clinical and Laboratory Standards Institutes M27-A3 broth micro-dilution method.  The MIC interpretative criteria for susceptibility to fluconazole were ≤ 8 μg/ml.  A total of 89 patients were included.  Patients (n = 59) infected by fluconazole-susceptible strains were compared with those (n = 30) infected by non-susceptible strains.  The patients' demographic and clinical characteristics were analyzed.  RESULTS:  The rate of fluconazole-non-susceptible C. neoformans in the study period significantly increased over time (p < 0.001).  The C. neoformans isolated in 2011-2012 (odds ratio: 10.68; 95 % confidence interval: 2.87-39.74; p < 0.001) was an independent predictive factor for the acquisition of fluconazole-non-susceptible C. neoformans.  CONCLUSIONS:  The rate of fluconazole-non-susceptible C. neoformans has significantly increased recently.  Continuous and large-scale anti-fungal susceptibility tests for C. neoformans are warranted to confirm this trend.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS21nkQL0Hl0xwLZxei1HN8fW6udTcc2ebnTOKtVuBKdbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FmtFaisw%253D%253D&md5=5fccae94bfa4f33266b719949438a274</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2Fs12879-015-1023-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12879-015-1023-8%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.-C.%26aulast%3DChang%26aufirst%3DT.-Y.%26aulast%3DLiu%26aufirst%3DJ.-W.%26aulast%3DChen%26aufirst%3DF.-J.%26aulast%3DChien%26aufirst%3DC.-C.%26aulast%3DLee%26aufirst%3DC.-H.%26aulast%3DLu%26aufirst%3DC.-H.%26atitle%3DIncreasing%2520Trend%2520of%2520Fluconazole-non-susceptible%2520Cryptococcus%2520Neoformans%2520in%2520Patients%2520with%2520Invasive%2520Cryptococcosis%253A%2520a%252012-year%2520Longitudinal%2520Study%26jtitle%3DBMC%2520Infect.%2520Dis.%26date%3D2015%26volume%3D15%26spage%3D277%26doi%3D10.1186%2Fs12879-015-1023-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanglard, D.</span></span> <span> </span><span class="NLM_article-title">Emerging Threats in Antifungal-Resistant Fungal Pathogens</span>. <i>Front. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">11</span>, <span class="refDoi"> DOI: 10.3389/fmed.2016.00011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.3389%2Ffmed.2016.00011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC28fjvFOnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=11&author=D.+Sanglard&title=Emerging+Threats+in+Antifungal-Resistant+Fungal+Pathogens&doi=10.3389%2Ffmed.2016.00011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging Threats in Antifungal-Resistant Fungal Pathogens</span></div><div class="casAuthors">Sanglard Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11</span>
        ISSN:<span class="NLM_cas:issn">2296-858X</span>.
    </div><div class="casAbstract">The use of antifungal drugs in the therapy of fungal diseases can lead to the development of antifungal resistance.  Resistance has been described for virtually all antifungal agents in diverse pathogens, including Candida and Aspergillus species.  The majority of resistance mechanisms have also been elucidated at the molecular level in these pathogens.  Drug resistance genes and genome mutations have been identified.  Therapeutic choices are limited for the control of fungal diseases, and it is tempting to combine several drugs to achieve better therapeutic efficacy.  In the recent years, several novel resistance patterns have been observed, including antifungal resistance originating from environmental sources in Aspergillus fumigatus and the emergence of simultaneous resistance to different antifungal classes (multidrug resistance) in different Candida species.  This review will summarize these current trends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTpI6xsgJyoezjk2McQZ6XmfW6udTcc2eYyd7HdbkucOLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28fjvFOnsA%253D%253D&md5=35b70007e870f369abaacc240a9b699d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3389%2Ffmed.2016.00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmed.2016.00011%26sid%3Dliteratum%253Aachs%26aulast%3DSanglard%26aufirst%3DD.%26atitle%3DEmerging%2520Threats%2520in%2520Antifungal-Resistant%2520Fungal%2520Pathogens%26jtitle%3DFront.%2520Med.%26date%3D2016%26volume%3D3%26spage%3D11%26doi%3D10.3389%2Ffmed.2016.00011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scorzoni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Paula E Silva, A. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcos, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assato, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Melo, W. C. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa-Orlandi, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendes-Giannini, M. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fusco-Almeida, A. M.</span></span> <span> </span><span class="NLM_article-title">Antifungal Therapy: New Advances in the Understanding and Treatment of Mycosis</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>08</i></span>,  <span class="NLM_fpage">36</span>, <span class="refDoi"> DOI: 10.3389/fmicb.2017.00036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.3389%2Ffmicb.2017.00036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC1c3jslyhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=08&publication_year=2017&pages=36&author=L.+Scorzoniauthor=A.+C.+A.+de+Paula+E+Silvaauthor=C.+M.+Marcosauthor=P.+A.+Assatoauthor=W.+C.+M.+A.+de+Meloauthor=H.+C.+de+Oliveiraauthor=C.+B.+Costa-Orlandiauthor=M.+J.+S.+Mendes-Gianniniauthor=A.+M.+Fusco-Almeida&title=Antifungal+Therapy%3A+New+Advances+in+the+Understanding+and+Treatment+of+Mycosis&doi=10.3389%2Ffmicb.2017.00036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Antifungal Therapy: New Advances in the Understanding and Treatment of Mycosis</span></div><div class="casAuthors">Scorzoni Liliana; de Paula E Silva Ana C A; Marcos Caroline M; Assato Patricia A; de Melo Wanessa C M A; de Oliveira Haroldo C; Costa-Orlandi Caroline B; Mendes-Giannini Maria J S; Fusco-Almeida Ana M</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">36</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">The high rates of morbidity and mortality caused by fungal infections are associated with the current limited antifungal arsenal and the high toxicity of the compounds.  Additionally, identifying novel drug targets is challenging because there are many similarities between fungal and human cells.  The most common antifungal targets include fungal RNA synthesis and cell wall and membrane components, though new antifungal targets are being investigated.  Nonetheless, fungi have developed resistance mechanisms, such as overexpression of e ux pump proteins and biofilm formation, emphasizing the importance of understanding these mechanisms.  To address these problems, different approaches to preventing and treating fungal diseases are described in this review, with a focus on the resistance mechanisms of fungi, with the goal of developing efficient strategies to overcoming and preventing resistance as well as new advances in antifungal therapy.  Due to the limited antifungal arsenal, researchers have sought to improve treatment via different approaches, and the synergistic effect obtained by the combination of antifungals contributes to reducing toxicity and could be an alternative for treatment.  Another important issue is the development of new formulations for antifungal agents, and interest in nanoparticles as new types of carriers of antifungal drugs has increased.  In addition, modifications to the chemical structures of traditional antifungals have improved their activity and pharmacokinetic parameters.  Moreover, a different approach to preventing and treating fungal diseases is immunotherapy, which involves different mechanisms, such as vaccines, activation of the immune response and inducing the production of host antimicrobial molecules.  Finally, the use of a mini-host has been encouraging for in vivo testing because these animal models demonstrate a good correlation with the mammalian model; they also increase the speediness of as well as facilitate the preliminary testing of new antifungal agents.  In general, many years are required from discovery of a new antifungal to clinical use.  However, the development of new antifungal strategies will reduce the therapeutic time and/or increase the quality of life of patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR79GlNDVo7nGgRQsDqrjW_fW6udTcc2eYyd7HdbkucOLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c3jslyhsw%253D%253D&md5=ac4a31dd5a3d11a5a45cf999b348ece7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2017.00036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2017.00036%26sid%3Dliteratum%253Aachs%26aulast%3DScorzoni%26aufirst%3DL.%26aulast%3Dde%2BPaula%2BE%2BSilva%26aufirst%3DA.%2BC.%2BA.%26aulast%3DMarcos%26aufirst%3DC.%2BM.%26aulast%3DAssato%26aufirst%3DP.%2BA.%26aulast%3Dde%2BMelo%26aufirst%3DW.%2BC.%2BM.%2BA.%26aulast%3Dde%2BOliveira%26aufirst%3DH.%2BC.%26aulast%3DCosta-Orlandi%26aufirst%3DC.%2BB.%26aulast%3DMendes-Giannini%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DFusco-Almeida%26aufirst%3DA.%2BM.%26atitle%3DAntifungal%2520Therapy%253A%2520New%2520Advances%2520in%2520the%2520Understanding%2520and%2520Treatment%2520of%2520Mycosis%26jtitle%3DFront.%2520Microbiol.%26date%3D2017%26volume%3D08%26spage%3D36%26doi%3D10.3389%2Ffmicb.2017.00036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakanjako, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colebunders, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutinho, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamya, M. R.</span></span> <span> </span><span class="NLM_article-title">Strategies to Optimize HIV Treatment Outcomes in Resource-limited Settings</span>. <i>AIDS Rev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">189</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=19940945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD1MjotlaqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=179-189&author=D.+Nakanjakoauthor=R.+Colebundersauthor=A.+G.+Coutinhoauthor=M.+R.+Kamya&title=Strategies+to+Optimize+HIV+Treatment+Outcomes+in+Resource-limited+Settings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to optimize HIV treatment outcomes in resource-limited settings</span></div><div class="casAuthors">Nakanjako Damalie; Colebunders Robert; Coutinho Alex G; Kamya Moses R</div><div class="citationInfo"><span class="NLM_cas:title">AIDS reviews</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">179-89</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although the availability of antiretroviral therapy has increased rapidly to reach over three million people in low- and middle-income countries, coverage remains low as only 31% of people in need were receiving antiretroviral therapy in sub-Saharan Africa.  Antiretroviral therapy scale-up needs to continue to grow exponentially to meet the need for universal access and keep pace with or exceed the new HIV infections.  This calls for strategies that will have the greatest impact on the reduction of opportunistic infections, toxicities, and early mortality after antiretroviral therapy initiation as well improve adherence, clinical, immunological, and virologic responses, patient retention in antiretroviral therapy programs, and overall quality of life of people living with HIV/AIDS.  Expanding antiretroviral therapy to all those eligible requires evidence-based decisions about how, when, and where expansion should occur.  In this article we highlight some of the strategies that have optimized HIV treatment outcomes within the constraints of limited resources in sub-Saharan Africa.  Key strategies to optimize HIV treatment outcomes include, i) scaling up HIV testing to identify all in need of HIV treatment, ii) strengthening the links between HIV diagnosis and comprehensive HIV/AIDS care, iii) timely initiation of antiretroviral therapy, iv) optimal diagnosis and treatment of opportunistic infections and comorbidities, v) investing in laboratory tests to support clinical monitoring of patients on antiretroviral therapy, vi) maximizing adherence to antiretroviral medication and retention of patients in HIV/AIDS care, viii) improving the health infrastructure, and increasing the human resources to handle the growing numbers of people in need of HIV treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSO9DJxprTKtoIRAdBLKY41fW6udTcc2eYxuGRjWv_OO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MjotlaqtQ%253D%253D&md5=ed9d8026355aeb4bce4bef3f248e1af2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNakanjako%26aufirst%3DD.%26aulast%3DColebunders%26aufirst%3DR.%26aulast%3DCoutinho%26aufirst%3DA.%2BG.%26aulast%3DKamya%26aufirst%3DM.%2BR.%26atitle%3DStrategies%2520to%2520Optimize%2520HIV%2520Treatment%2520Outcomes%2520in%2520Resource-limited%2520Settings%26jtitle%3DAIDS%2520Rev.%26date%3D2009%26volume%3D11%26spage%3D179%26epage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levitz, S. M.</span></span> <span> </span><span class="NLM_article-title">Virulence Factors of Medically Important Fungi</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1128/cmr.9.4.469</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FCMR.9.4.469" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8894347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK28XmsFOksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=469-488&author=L.+H.+Hoganauthor=B.+S.+Kleinauthor=S.+M.+Levitz&title=Virulence+Factors+of+Medically+Important+Fungi&doi=10.1128%2Fcmr.9.4.469"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Virulence factors of medically important fungi</span></div><div class="casAuthors">Hogan, Laura H.; Klein, Bruce S.; Levitz, Stuart M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-488</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review with 326 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8P7URMyPg27Vg90H21EOLACvtfcHk0lgBKuXKOgReOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmsFOksrc%253D&md5=70097820426f47aaa5f629d95a99cd59</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FCMR.9.4.469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.9.4.469%26sid%3Dliteratum%253Aachs%26aulast%3DHogan%26aufirst%3DL.%2BH.%26aulast%3DKlein%26aufirst%3DB.%2BS.%26aulast%3DLevitz%26aufirst%3DS.%2BM.%26atitle%3DVirulence%2520Factors%2520of%2520Medically%2520Important%2520Fungi%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D1996%26volume%3D9%26spage%3D469%26epage%3D488%26doi%3D10.1128%2Fcmr.9.4.469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandujano-González, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa-Tanaca, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anducho-Reyes, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercado-Flores, Y.</span></span> <span> </span><span class="NLM_article-title">Secreted Fungal Aspartic Proteases: A review</span>. <i>Rev. Iberoam. Micol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1016/j.riam.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.riam.2015.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=27137097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BC28blsVCjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=76-82&author=V.+Mandujano-Gonz%C3%A1lezauthor=L.+Villa-Tanacaauthor=M.+A.+Anducho-Reyesauthor=Y.+Mercado-Flores&title=Secreted+Fungal+Aspartic+Proteases%3A+A+review&doi=10.1016%2Fj.riam.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Secreted fungal aspartic proteases: A review</span></div><div class="casAuthors">Mandujano-Gonzalez Virginia; Anducho-Reyes Miguel Angel; Villa-Tanaca Lourdes; Mercado-Flores Yuridia</div><div class="citationInfo"><span class="NLM_cas:title">Revista iberoamericana de micologia</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">76-82</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The aspartic proteases, also called aspartyl and aspartate proteases or acid proteases (E.C.3.4.23), belong to the endopeptidase family and are characterized by the conserved sequence Asp-Gly-Thr at the active site.  These enzymes are found in a wide variety of microorganisms in which they perform important functions related to nutrition and pathogenesis.  In addition, their high activity and stability at acid pH make them attractive for industrial application in the food industry; specifically, they are used as milk-coagulating agents in cheese production or serve to improve the taste of some foods.  This review presents an analysis of the characteristics and properties of secreted microbial aspartic proteases and their potential for commercial application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTQMcnhOfS5GQxyojvcjibhfW6udTcc2eYnt3nBNdsrp7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28blsVCjtw%253D%253D&md5=1ff57df1ed1fcec9a731668c1b9e2cb8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.riam.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.riam.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DMandujano-Gonz%25C3%25A1lez%26aufirst%3DV.%26aulast%3DVilla-Tanaca%26aufirst%3DL.%26aulast%3DAnducho-Reyes%26aufirst%3DM.%2BA.%26aulast%3DMercado-Flores%26aufirst%3DY.%26atitle%3DSecreted%2520Fungal%2520Aspartic%2520Proteases%253A%2520A%2520review%26jtitle%3DRev.%2520Iberoam.%2520Micol.%26date%3D2016%26volume%3D33%26spage%3D76%26epage%3D82%26doi%3D10.1016%2Fj.riam.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumesic, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donoghue, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Greca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhani, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">Integrated Activity and Genetic Profiling of Secreted Peptidases in Cryptococcus Neoformans Reveals an Aspartyl Peptidase Required for Low pH Survival and Virulence</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1006051</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1006051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1371%2Fjournal.ppat.1006051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=27977806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktVOhu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=S.+C.+Clarkeauthor=P.+A.+Dumesicauthor=C.+M.+Homerauthor=A.+J.+O%E2%80%99Donoghueauthor=F.+La+Grecaauthor=L.+Pallovaauthor=P.+Majerauthor=H.+D.+Madhaniauthor=C.+S.+Craik&title=Integrated+Activity+and+Genetic+Profiling+of+Secreted+Peptidases+in+Cryptococcus+Neoformans+Reveals+an+Aspartyl+Peptidase+Required+for+Low+pH+Survival+and+Virulence&doi=10.1371%2Fjournal.ppat.1006051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated activity and genetic profiling of secreted peptidases in Cryptococcus neoformans reveals an aspartyl peptidase required for low pH survival and virulence</span></div><div class="casAuthors">Clarke, Starlynn C.; Dumesic, Phillip A.; Homer, Christina M.; O'Donoghue, Anthony J.; La Greca, Florencia; Pallova, Lenka; Majer, Pavel; Madhani, Hiten D.; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e1006051/1-e1006051/30</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The opportunistic fungal pathogen Cryptococcus neoformans is a major cause of mortality in immunocompromised individuals, resulting in more than 600,000 deaths per yr.  Many human fungal pathogens secrete peptidases that influence virulence, but in most cases the substrate specificity and regulation of these enzymes remains poorly understood.  The paucity of such information is a roadblock to our understanding of the biol. functions of peptidases and whether or not these enzymes are viable therapeutic targets.  We report here an unbiased anal. of secreted peptidase activity and specificity in C. neoformans using a mass spectrometry-based substrate profiling strategy and subsequent functional investigations.  Our initial studies revealed that global peptidase activity and specificity are dramatically altered by environmental conditions.  To uncover the substrate preferences of individual enzymes and interrogate their biol. functions, we constructed and profiled a ten-member gene deletion collection of candidate secreted peptidases.  Through this deletion approach, we characterized the substrate specificity of three peptidases within the context of the C. neoformans secretome, including an enzyme known to be important for fungal entry into the brain.  We selected a previously uncharacterized peptidase, which we term Major aspartyl peptidase 1 (May1), for detailed study due to its substantial contribution to extracellular proteolytic activity.  Based on the preference of May1 for proteolysis between hydrophobic amino acids, we screened a focused library of aspartyl peptidase inhibitors and identified four high-affinity antagonists.  Finally, we tested may1Δ strains in a mouse model of C. neoformans infection and found that strains lacking this enzyme are significantly attenuated for virulence.  Our study reveals the secreted peptidase activity and specificity of an important human fungal pathogen, identifies responsible enzymes through genetic tests of their function, and demonstrates how this information can guide the development of high affinity small mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUf62D_2DdfLVg90H21EOLACvtfcHk0lj8-8NMIq1E1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktVOhu7k%253D&md5=b196b08575b78973bee0c22fe12caf27</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1006051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1006051%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DS.%2BC.%26aulast%3DDumesic%26aufirst%3DP.%2BA.%26aulast%3DHomer%26aufirst%3DC.%2BM.%26aulast%3DO%25E2%2580%2599Donoghue%26aufirst%3DA.%2BJ.%26aulast%3DLa%2BGreca%26aufirst%3DF.%26aulast%3DPallova%26aufirst%3DL.%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DMadhani%26aufirst%3DH.%2BD.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DIntegrated%2520Activity%2520and%2520Genetic%2520Profiling%2520of%2520Secreted%2520Peptidases%2520in%2520Cryptococcus%2520Neoformans%2520Reveals%2520an%2520Aspartyl%2520Peptidase%2520Required%2520for%2520Low%2520pH%2520Survival%2520and%2520Virulence%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1006051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orsi, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibellini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esposito, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peppoloni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mussini, C.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of an aspartyl protease fromCryptococcus neoformans</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>581</i></span>,  <span class="NLM_fpage">3882</span>– <span class="NLM_lpage">3886</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2007.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.febslet.2007.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=17651737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotlSlsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=581&publication_year=2007&pages=3882-3886&author=M.+Pintiauthor=C.+F.+Orsiauthor=L.+Gibelliniauthor=R.+Espositoauthor=A.+Cossarizzaauthor=E.+Blasiauthor=S.+Peppoloniauthor=C.+Mussini&title=Identification+and+characterization+of+an+aspartyl+protease+fromCryptococcus+neoformans&doi=10.1016%2Fj.febslet.2007.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of an aspartyl protease from Cryptococcus neoformans</span></div><div class="casAuthors">Pinti, Marcello; Orsi, Carlotta Francesca; Gibellini, Lara; Esposito, Roberto; Cossarizza, Andrea; Blasi, Elisabetta; Peppoloni, Samuele; Mussini, Cristina</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">581</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3882-3886</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cryptococcosis, caused by Cryptococcus neoformans, is an invasive infection often occurring in AIDS patients.  Potent therapy against HIV, which includes protease inhibitors (PIs), has beneficial effects also on opportunistic infections by pathogens such as C. neoformans and C. albicans.  PIs inhibit growth of C. albicans by affecting the activity of its aspartyl proteases.  We identified, cloned and sequenced a cDNA from C. neoformans encoding for a putative aspartyl protease (CnAP1), and the corresponding genomic region.  The gene cnap1 codifies for a protein of 505 aa, with a canonical aspartyl protease structure.  We purified the recombinant protein and analyzed its activity in the presence of PIs (Indinavir, Lopinavir, Ritonavir), but did not evidence any inhibition of protease activity.  The transcriptional level of cnap1 in C. neoformans is const. in different media.  The absence of any inhibition activity by PIs suggests that other targets for PIs might exist in C. neoformans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeyYtH3G4rBrVg90H21EOLACvtfcHk0lj8-8NMIq1E1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotlSlsrs%253D&md5=ab205d4d01730c3197995c477d7e38a3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2007.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2007.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DPinti%26aufirst%3DM.%26aulast%3DOrsi%26aufirst%3DC.%2BF.%26aulast%3DGibellini%26aufirst%3DL.%26aulast%3DEsposito%26aufirst%3DR.%26aulast%3DCossarizza%26aufirst%3DA.%26aulast%3DBlasi%26aufirst%3DE.%26aulast%3DPeppoloni%26aufirst%3DS.%26aulast%3DMussini%26aufirst%3DC.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520an%2520aspartyl%2520protease%2520fromCryptococcus%2520neoformans%26jtitle%3DFEBS%2520Lett.%26date%3D2007%26volume%3D581%26spage%3D3882%26epage%3D3886%26doi%3D10.1016%2Fj.febslet.2007.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidrim, J. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdigão-Neto, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordeiro, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brilhante, R. S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leite, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teixeira, C. E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitas, R. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesquita, J. R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves, M. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, M. F. G.</span></span> <span> </span><span class="NLM_article-title">Viral Protease Inhibitors Affect the Production of Virulence Factors in Cryptococcus Neoformans</span>. <i>Can. J. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">932</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1139/w2012-075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1139%2Fw2012-075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=22716223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2012&pages=932-936&author=J.+J.+C.+Sidrimauthor=L.+V.+Perdig%C3%A3o-Netoauthor=R.+A.+Cordeiroauthor=R.+S.+N.+Brilhanteauthor=J.+J.+G.+Leiteauthor=C.+E.+C.+Teixeiraauthor=A.+J.+Monteiroauthor=R.+M.+F.+Freitasauthor=J.+F.+Ribeiroauthor=J.+R.+L.+Mesquitaauthor=M.+V.+F.+Gon%C3%A7alvesauthor=M.+F.+G.+Rocha&title=Viral+Protease+Inhibitors+Affect+the+Production+of+Virulence+Factors+in+Cryptococcus+Neoformans&doi=10.1139%2Fw2012-075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Viral protease inhibitors affect the production of virulence factors in Cryptococcus neoformans</span></div><div class="casAuthors">Sidrim, J. J. C.; Perdigao-Neto, L. V.; Cordeiro, R. A.; Brilhante, R. S. N.; Leite, J. J. G.; Teixeira, C. E. C.; Monteiro, A. J.; Freitas, R. M. F.; Ribeiro, J. F.; Mesquita, J. R. L.; Goncalves, M. V. F.; Rocha, M. F. G.</div><div class="citationInfo"><span class="NLM_cas:title">Canadian Journal of Microbiology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">932-936</span>CODEN:
                <span class="NLM_cas:coden">CJMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0008-4166</span>.
    
            (<span class="NLM_cas:orgname">Canadian Science Publishing</span>)
        </div><div class="casAbstract">The effects of the protease inhibitors saquinavir, darunavir, ritonavir, and indinavir on growth inhibition, protease and phospholipase activities, as well as capsule thickness of Cryptococcus neoformans were investigated.  Viral protease inhibitors did not reduce fungal growth when tested in concns. ranging from 0.001 to 1.000 mg/L.  A tendency toward increasing phospholipase activity was obsd. with the highest tested drug concn. in a strain-specific pattern.  However, these drugs reduced protease activity as well as capsule prodn.  Our results confirm a previous finding that antiretroviral drugs affect the prodn. of important virulence factors of C. neoformans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJe8ljIdPt-LVg90H21EOLACvtfcHk0lj8-8NMIq1E1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGhtLg%253D&md5=9dc87065ed3e3139462a75383c075623</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1139%2Fw2012-075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fw2012-075%26sid%3Dliteratum%253Aachs%26aulast%3DSidrim%26aufirst%3DJ.%2BJ.%2BC.%26aulast%3DPerdig%25C3%25A3o-Neto%26aufirst%3DL.%2BV.%26aulast%3DCordeiro%26aufirst%3DR.%2BA.%26aulast%3DBrilhante%26aufirst%3DR.%2BS.%2BN.%26aulast%3DLeite%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DTeixeira%26aufirst%3DC.%2BE.%2BC.%26aulast%3DMonteiro%26aufirst%3DA.%2BJ.%26aulast%3DFreitas%26aufirst%3DR.%2BM.%2BF.%26aulast%3DRibeiro%26aufirst%3DJ.%2BF.%26aulast%3DMesquita%26aufirst%3DJ.%2BR.%2BL.%26aulast%3DGon%25C3%25A7alves%26aufirst%3DM.%2BV.%2BF.%26aulast%3DRocha%26aufirst%3DM.%2BF.%2BG.%26atitle%3DViral%2520Protease%2520Inhibitors%2520Affect%2520the%2520Production%2520of%2520Virulence%2520Factors%2520in%2520Cryptococcus%2520Neoformans%26jtitle%3DCan.%2520J.%2520Microbiol.%26date%3D2012%26volume%3D58%26spage%3D932%26epage%3D936%26doi%3D10.1139%2Fw2012-075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadevall, A.</span></span> <span> </span><span class="NLM_article-title">Virulence-Associated Enzymes of Cryptococcus Neoformans</span>. <i>Eukaryotic Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1173</span>– <span class="NLM_lpage">1185</span>, <span class="refDoi"> DOI: 10.1128/ec.00103-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FEC.00103-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=26453651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2qsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=1173-1185&author=F.+Almeidaauthor=J.+M.+Wolfauthor=A.+Casadevall&title=Virulence-Associated+Enzymes+of+Cryptococcus+Neoformans&doi=10.1128%2Fec.00103-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Virulence-associated enzymes of Cryptococcus neoformans</span></div><div class="casAuthors">Almeida, Fausto; Wolf, Julie M.; Casadevall, Arturo</div><div class="citationInfo"><span class="NLM_cas:title">Eukaryotic Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1173-1185</span>CODEN:
                <span class="NLM_cas:coden">ECUEA2</span>;
        ISSN:<span class="NLM_cas:issn">1535-9786</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Enzymes play key roles in fungal pathogenesis.  Manipulation of enzyme expression or activity can significantly alter the infection process, and enzyme expression profiles can be a hallmark of disease.  Hence, enzymes are worthy targets for better understanding pathogenesis and identifying new options for combating fungal infections.  Advances in genomics, proteomics, transcriptomics, and mass spectrometry have enabled the identification and characterization of new fungal enzymes.  This review focuses on recent developments in the virulence-assocd. enzymes from Cryptococcus neoformans.  The enzymic suite of C. neoformans has evolved for environmental survival, but several of these enzymes play a dual role in colonizing the mammalian host.  We also discuss new therapeutic and diagnostic strategies that could be based on the underlying enzymol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu9Edib2HgurVg90H21EOLACvtfcHk0ljSbCDaqjVHow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2qsb4%253D&md5=789b1ca217f73079e96f40cc81f89b61</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FEC.00103-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FEC.00103-15%26sid%3Dliteratum%253Aachs%26aulast%3DAlmeida%26aufirst%3DF.%26aulast%3DWolf%26aufirst%3DJ.%2BM.%26aulast%3DCasadevall%26aufirst%3DA.%26atitle%3DVirulence-Associated%2520Enzymes%2520of%2520Cryptococcus%2520Neoformans%26jtitle%3DEukaryotic%2520Cell%26date%3D2015%26volume%3D14%26spage%3D1173%26epage%3D1185%26doi%3D10.1128%2Fec.00103-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casadevall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coelho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alanio, A.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Cryptococcus Neoformans-mediated Host Damage</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">855</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2018.00855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.3389%2Ffimmu.2018.00855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=29760698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Gls7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=855&author=A.+Casadevallauthor=C.+Coelhoauthor=A.+Alanio&title=Mechanisms+of+Cryptococcus+Neoformans-mediated+Host+Damage&doi=10.3389%2Ffimmu.2018.00855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Cryptococcus neoformans-mediated host damage</span></div><div class="casAuthors">Casadevall, Arturo; Coelho, Carolina; Alanio, Alexandre</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">855/1-855/8</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Cryptococcus neoformans is not usually considered a cytotoxic fungal pathogen but there is considerable evidence that this microbe can damage host cells and tissues.  In this essay, we review the evidence that C. neoformans damages host cells and note that the mechanisms involved are diverse.  We consider C. neoformans-mediated host damage at the mol., cellular, tissue, and organism level.  Direct mechanisms of cytotoxicity include lytic exocytosis, organelle dysfunction, phagolysosomal membrane damage, and cytoskeletal alterations.  Cytotoxicity contributes to pathogenesis by inter- fering with immune effector cell function and disrupting endothelial barriers thus allowing dissemination.  When C. neoformans-mediated and immune-mediated host damage is sufficient to affect homeostasis, cryptococcosis occurs at the organism level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppkLowe53xLbVg90H21EOLACvtfcHk0ljSbCDaqjVHow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Gls7vI&md5=cbc33b6eee40097f5e6ce3d3ef04699d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2018.00855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2018.00855%26sid%3Dliteratum%253Aachs%26aulast%3DCasadevall%26aufirst%3DA.%26aulast%3DCoelho%26aufirst%3DC.%26aulast%3DAlanio%26aufirst%3DA.%26atitle%3DMechanisms%2520of%2520Cryptococcus%2520Neoformans-mediated%2520Host%2520Damage%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D9%26spage%3D855%26doi%3D10.3389%2Ffimmu.2018.00855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhrig, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Na Pombejra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamklang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bautos, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelli, A.</span></span> <span> </span><span class="NLM_article-title">Invasion of the Central Nervous System by Cryptococcus Neoformans Requires a Secreted Fungal Metalloprotease</span>. <i>mBio</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e01101</span> <span class="refDoi"> DOI: 10.1128/mBio.01101-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FmBio.01101-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=24895304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1OlsLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&author=K.+Vuauthor=R.+Thamauthor=J.+P.+Uhrigauthor=G.+R.+Thompsonauthor=S.+Na+Pombejraauthor=M.+Jamklangauthor=J.+M.+Bautosauthor=A.+Gelli&title=Invasion+of+the+Central+Nervous+System+by+Cryptococcus+Neoformans+Requires+a+Secreted+Fungal+Metalloprotease&doi=10.1128%2FmBio.01101-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Invasion of the central nervous system by Cryptococcus neoformans requires a secreted fungal metalloprotease</span></div><div class="casAuthors">Vu, Kiem; Tham, Rick; Uhrig, John P.; Thompson, George R., III; Pombejra, Sarisa Na; Jamklang, Mantana; Bautos, Jennifer M.; Gelli, Angie</div><div class="citationInfo"><span class="NLM_cas:title">mBio</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e01101/1-e01101/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">MBIOCL</span>;
        ISSN:<span class="NLM_cas:issn">2150-7511</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cryptococcus spp. cause life-threatening fungal infection of the central nervous system (CNS), predominantly in patients with a compromised immune system.  Why Cryptococcus neoformans has this remarkable tropism for the CNS is not clear.  Recent research on cerebral pathogenesis of C. neoformans revealed a predominantly transcellular migration of cryptococci across the brain endothelium; however, the identities of key fungal virulence factors that function specifically to invade the CNS remain unresolved.  Here we found that a novel, secreted metalloprotease (Mpr1) that we identified in the extracellular proteome of C. neoformans (CnMpr1) is required for establishing fungal disease in the CNS.  Mpr1 belongs to a poorly characterized M36 class of fungalysins that are expressed in only some fungal species.  A strain of C. neoformans lacking the gene encoding Mpr1 (mpr1Δ) failed to breach the endothelium in an in vitro model of the human blood-brain barrier (BBB).  A mammalian host infected with the mpr1Δ null strain demonstrated significant improvement in survival due to a reduced brain fungal burden and lacked the brain pathol. commonly assocd. with cryptococcal disease.  The in vivo studies further indicate that Mpr1 is not required for fungal dissemination and Mpr1 likely targets the brain endothelium specifically.  Remarkably, the sole expression of CnMPR1 in Saccharomyces cerevisiae resulted in a robust migration of yeast cells across the brain endothelium, demonstrating Mpr1's specific activity in breaching the BBB and suggesting that Mpr1 may function independently of the hyaluronic acid-CD44 pathway.  This distinct role for Mpr1 may develop into innovative treatment options and facilitate a brain-specific drug delivery platform.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrGoQEhu8PB7Vg90H21EOLACvtfcHk0lgnfLu4mKcQfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1OlsLfL&md5=e10ebba9af88ef9ea711d66fc1b49335</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1128%2FmBio.01101-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmBio.01101-14%26sid%3Dliteratum%253Aachs%26aulast%3DVu%26aufirst%3DK.%26aulast%3DTham%26aufirst%3DR.%26aulast%3DUhrig%26aufirst%3DJ.%2BP.%26aulast%3DThompson%26aufirst%3DG.%2BR.%26aulast%3DNa%2BPombejra%26aufirst%3DS.%26aulast%3DJamklang%26aufirst%3DM.%26aulast%3DBautos%26aufirst%3DJ.%2BM.%26aulast%3DGelli%26aufirst%3DA.%26atitle%3DInvasion%2520of%2520the%2520Central%2520Nervous%2520System%2520by%2520Cryptococcus%2520Neoformans%2520Requires%2520a%2520Secreted%2520Fungal%2520Metalloprotease%26jtitle%3DmBio%26date%3D2014%26volume%3D5%26doi%3D10.1128%2FmBio.01101-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito-Kuwa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ninomiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidotto, V.</span></span> <span> </span><span class="NLM_article-title">Extracellular proteolytic activity of Cryptococcus neoformans</span>. <i>Mycopathologia</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1007/bf01138475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1007%2FBF01138475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=7739728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK2MXmsV2gsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=1994&pages=143-150&author=S.+Aokiauthor=S.+Ito-Kuwaauthor=K.+Nakamuraauthor=J.+Katoauthor=K.+Ninomiyaauthor=V.+Vidotto&title=Extracellular+proteolytic+activity+of+Cryptococcus+neoformans&doi=10.1007%2Fbf01138475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular proteolytic activity of Cryptococcus neoformans</span></div><div class="casAuthors">Aoki, Shigeji; Ito-Kuwa, Shoko; Nakamura, Kenjirou; Kato, Joji; Ninomiya, Kazunori; Vidotto, Valerio</div><div class="citationInfo"><span class="NLM_cas:title">Mycopathologia</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">143-50</span>CODEN:
                <span class="NLM_cas:coden">MYCPAH</span>;
        ISSN:<span class="NLM_cas:issn">0301-486X</span>.
    </div><div class="casAbstract">Eight strains of Cryptococcus neoformans var. neoformans isolated from AIDS patients in the Infectious Disease Institute, University of Turin, Italy, were examd. for growth and extracellular proteolytic activity in culture with solid and liq. media.  All of the strains grew well on Yeast Carbon Base (YCB) agar medium supplemented with both 0.1% (w/v) bovine serum albumin (BSA) and 0.01% (w/v) polypeptone (Pp), and produced a clear proteolytic zone around their colonies, whereas they exhibited less growth and proteolytic activity on YCB medium supplemented with BSA alone.  Strain #8 with a strong proteolytic activity was cultured in three different liq. media.  Its growth was limited in YCB medium supplemented with 0.1% BSA, but was moderate in that with 0.01% Pp.  Enhanced growth was supported by the addn. of both BSA and Pp to the YCB medium.  The relative value of the final cellular yields obtained with the above YCB-0.1% BSA, YCB-0.01% Pp and YCB-0.1% BSA-0.01% Pp media was approx. 1:10:20.  In the culture with YCB medium contg. both BSA and Pp, a rapid decrease in the amt. of BSA was demonstrated by a spectrophotometric assay and gel electrophoresis of the culture supernatant after the log-to-stationary phase.  The proteolytic activity in the culture supernatant became detectable after the log phase when tested with skim milk agarose plates.  These results show that Cr. neoformans var. neoformans is able to secrete protease and to utilize protein as a source of nitrogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVkUqt-jc0B7Vg90H21EOLACvtfcHk0lgnfLu4mKcQfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmsV2gsb4%253D&md5=c3306de7efdd241b62c7afc4d5952fe0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2FBF01138475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01138475%26sid%3Dliteratum%253Aachs%26aulast%3DAoki%26aufirst%3DS.%26aulast%3DIto-Kuwa%26aufirst%3DS.%26aulast%3DNakamura%26aufirst%3DK.%26aulast%3DKato%26aufirst%3DJ.%26aulast%3DNinomiya%26aufirst%3DK.%26aulast%3DVidotto%26aufirst%3DV.%26atitle%3DExtracellular%2520proteolytic%2520activity%2520of%2520Cryptococcus%2520neoformans%26jtitle%3DMycopathologia%26date%3D1994%26volume%3D128%26spage%3D143%26epage%3D150%26doi%3D10.1007%2Fbf01138475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadevall, A.</span></span> <span> </span><span class="NLM_article-title">Extracellular Proteinase Activity of Cryptococcus Neoformans</span>. <i>Clin. Diagn. Lab. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1128/cdli.3.5.570-574.1996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FCDLI.3.5.570-574.1996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8877137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK28XmtFWjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1996&pages=570-574&author=L.+C.+Chenauthor=E.+S.+Blankauthor=A.+Casadevall&title=Extracellular+Proteinase+Activity+of+Cryptococcus+Neoformans&doi=10.1128%2Fcdli.3.5.570-574.1996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular proteinase activity of Cryptococcus neoformans</span></div><div class="casAuthors">Chen, Lin-Chi; Blank, Erika S.; Casadevall, Arturo</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Diagnostic Laboratory Immunology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">570-574</span>CODEN:
                <span class="NLM_cas:coden">CDIMEN</span>;
        ISSN:<span class="NLM_cas:issn">1071-412X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Extracellular proteinase activity was studies for eight strains of Cryptococcus neoformans var. neoformans and two strains of Cryptococcus neoformans var. gattii.  Proteinase activity was measured by protein agar clearance, azoalbumin hydrolysis, gelatin liquefaction, and protein substrate polyacrylamide gel electrophoresis.  All strains of C. neoformans produced extracellular proteolytic activity.  Maximal extracellular proteinase activity in supernatants of C. neoformans cultures was assocd. with late logarithmic- and stationary-phase cultures.  C. neoformans was able to utilize murine IgG1, bovine IgG, and human complement factor 5 for growth in media contg. these proteins as the sole sources of carbon and nitrogen, suggesting a capacity to degrade immunol. important proteins.  Protein substrate polyacrylamide gel electrophoresis revealed several bands with proteolytic activity at apparent mol. masses of 200, 100, and 50 kDa.  The results confirm the existence of extracellular proteinase activity for C. neoformans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocstBEeaOWsrVg90H21EOLACvtfcHk0ljsA9H_1N3Gkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtFWjs7c%253D&md5=163674442c972568204ea6991082ce2d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1128%2FCDLI.3.5.570-574.1996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCDLI.3.5.570-574.1996%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%2BC.%26aulast%3DBlank%26aufirst%3DE.%2BS.%26aulast%3DCasadevall%26aufirst%3DA.%26atitle%3DExtracellular%2520Proteinase%2520Activity%2520of%2520Cryptococcus%2520Neoformans%26jtitle%3DClin.%2520Diagn.%2520Lab.%2520Immunol.%26date%3D1996%26volume%3D3%26spage%3D570%26epage%3D574%26doi%3D10.1128%2Fcdli.3.5.570-574.1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blasi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombari, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francesca Orsi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troiano, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esposito, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peppoloni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mussini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neglia, R.</span></span> <span> </span><span class="NLM_article-title">The human immunodeficiency virus (HIV) protease inhibitor indinavir directly affects the opportunistic fungal pathogenCryptococcus neoformans</span>. <i>FEMS Immunol. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/j.femsim.2004.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.femsim.2004.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=15364103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFSktbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2004&pages=187-195&author=E.+Blasiauthor=B.+Colombariauthor=C.+Francesca+Orsiauthor=M.+Pintiauthor=L.+Troianoauthor=A.+Cossarizzaauthor=R.+Espositoauthor=S.+Peppoloniauthor=C.+Mussiniauthor=R.+Neglia&title=The+human+immunodeficiency+virus+%28HIV%29+protease+inhibitor+indinavir+directly+affects+the+opportunistic+fungal+pathogenCryptococcus+neoformans&doi=10.1016%2Fj.femsim.2004.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The human immunodeficiency virus (HIV) protease inhibitor indinavir directly affects the opportunistic fungal pathogen Cryptococcus neoformans</span></div><div class="casAuthors">Blasi, Elisabetta; Colombari, Bruna; Orsi, Carlotta Francesca; Pinti, Marcello; Troiano, Leonarda; Cossarizza, Andrea; Esposito, Roberto; Peppoloni, Samuela; Mussini, Cristina; Neglia, Rachele</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Immunology and Medical Microbiology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">187-195</span>CODEN:
                <span class="NLM_cas:coden">FIMIEV</span>;
        ISSN:<span class="NLM_cas:issn">0928-8244</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Highly active antiretroviral therapy (HAART), that includes human immunodeficiency virus (HIV) protease inhibitors (PIs), has been remarkably efficacious including against some opportunistic infections.  In this report we investigated the effect(s) of the PI indinavir on protease activity by Cryptococcus neoformans, an opportunistic fungal pathogen responsible for recurrent meningoencephalitis in AIDS patients.  Indinavir was also tested for potential effects on other parameters, such as fungal viability, growth ability and susceptibility to immune effector cells.  It was found that indinavir impaired cryptococcal protease activity in a time- and dose-dependent fashion.  The phenomenon was similarly detectable in ATCC/lab. strains and clin. isolates.  C. neoformans growth rate was also significantly reduced upon exposure to indinavir, while fungal viability was not affected and mitochondrial toxicity not detected.  Furthermore, as assessed by an in vitro infection model, indinavir significantly and consistently augmented C. neoformans susceptibility to microglial cell-mediated phagocytosis and killing.  Overall, by providing the first evidence that indinavir directly affects C. neoformans, these data add new in vitro insights on the wide-spectrum efficacy of PIs, further arguing for the clin. relevance of HAART against opportunistic infections in AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolSYsj5AJkybVg90H21EOLACvtfcHk0lgRCdig6Dv8Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFSktbk%253D&md5=1b0f7a5f45c8c5107b8bf18e704cf928</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.femsim.2004.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.femsim.2004.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DBlasi%26aufirst%3DE.%26aulast%3DColombari%26aufirst%3DB.%26aulast%3DFrancesca%2BOrsi%26aufirst%3DC.%26aulast%3DPinti%26aufirst%3DM.%26aulast%3DTroiano%26aufirst%3DL.%26aulast%3DCossarizza%26aufirst%3DA.%26aulast%3DEsposito%26aufirst%3DR.%26aulast%3DPeppoloni%26aufirst%3DS.%26aulast%3DMussini%26aufirst%3DC.%26aulast%3DNeglia%26aufirst%3DR.%26atitle%3DThe%2520human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520protease%2520inhibitor%2520indinavir%2520directly%2520affects%2520the%2520opportunistic%2520fungal%2520pathogenCryptococcus%2520neoformans%26jtitle%3DFEMS%2520Immunol.%2520Med.%2520Microbiol.%26date%3D2004%26volume%3D42%26spage%3D187%26epage%3D195%26doi%3D10.1016%2Fj.femsim.2004.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bernardis, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torosantucci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumbarello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cauda, R.</span></span> <span> </span><span class="NLM_article-title">In Vitro and In Vivo Anticandidal Activity of Human Immunodeficiency Virus Protease Inhibitors</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>180</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1086/314871</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1086%2F314871" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10395861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK1MXlt1ygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=1999&pages=448-453&author=A.+Cassoneauthor=F.+De+Bernardisauthor=A.+Torosantucciauthor=E.+Tacconelliauthor=M.+Tumbarelloauthor=R.+Cauda&title=In+Vitro+and+In+Vivo+Anticandidal+Activity+of+Human+Immunodeficiency+Virus+Protease+Inhibitors&doi=10.1086%2F314871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro and in vivo anticandidal activity of human immunodeficiency virus protease inhibitors</span></div><div class="casAuthors">Cassone, Antonio; De Bernardis, Flavia; Torosantucci, Antonella; Tacconelli, Evelina; Tumbarello, Mario; Cauda, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">180</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-453</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">Highly active antiretroviral therapy that includes human immunodeficiency virus (HIV) aspartyl protease inhibitors (PIs) causes a decline in the incidence of some opportunistic infections in AIDS, and this decline is currently attributed to the restoration of specific immunity.  The effect of two PIs (indinavir and ritonavir) on the enzymic activity of a secretory aspartyl protease (Sap) of Candida albicans (a major agent of mucosal disease in HIV-infected subjects) and on growth and exptl. pathogenicity of this fungus was evaluated.  Both PIs strongly (≥90%) and concn.-dependently (0.1-10 μM) inhibited Sap activity and prodn.  They also reduced Candida growth in a nitrogen-limited, Sap-expression-dependent growth medium and exerted a therapeutic effect in an exptl. model of vaginal candidiasis, with an efficacy comparable to that of fluconazole.  Thus, besides the expected immunorestoration, patients receiving PI therapy may benefit from a direct anticandidal activity of these drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzWKcfIy7H-bVg90H21EOLACvtfcHk0lgRCdig6Dv8Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlt1ygtr4%253D&md5=f1660f40251968002ac0b533ad6b16eb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1086%2F314871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F314871%26sid%3Dliteratum%253Aachs%26aulast%3DCassone%26aufirst%3DA.%26aulast%3DDe%2BBernardis%26aufirst%3DF.%26aulast%3DTorosantucci%26aufirst%3DA.%26aulast%3DTacconelli%26aufirst%3DE.%26aulast%3DTumbarello%26aufirst%3DM.%26aulast%3DCauda%26aufirst%3DR.%26atitle%3DIn%2520Vitro%2520and%2520In%2520Vivo%2520Anticandidal%2520Activity%2520of%2520Human%2520Immunodeficiency%2520Virus%2520Protease%2520Inhibitors%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D1999%26volume%3D180%26spage%3D448%26epage%3D453%26doi%3D10.1086%2F314871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mussini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pezzotti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Quiros, J. C. L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cinque, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borghi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domingo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cahn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Luca, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">d’Arminio
Monforte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nwokolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negroni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacchetti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antinori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazzarin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esposito, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antinori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aberg, J. A.</span></span> <span> </span><span class="NLM_article-title">Discontinuation of Maintenance Therapy for Cryptococcal Meningitis in Patients with AIDS Treated with Highly Active Antiretroviral Therapy: an International Observational Study</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1086/381261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1086%2F381261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=14765351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A280%3ADC%252BD2c%252Fmtlaguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2004&pages=565-571&author=C.+Mussiniauthor=P.+Pezzottiauthor=J.+M.+Miroauthor=E.+Martinezauthor=J.+C.+L.+B.+de+Quirosauthor=P.+Cinqueauthor=V.+Borghiauthor=A.+Bediniauthor=P.+Domingoauthor=P.+Cahnauthor=P.+Bossiauthor=A.+de+Lucaauthor=A.+d%E2%80%99Arminio%0AMonforteauthor=M.+Nelsonauthor=N.+Nwokoloauthor=S.+Helouauthor=R.+Negroniauthor=G.+Jacchettiauthor=S.+Antinoriauthor=A.+Lazzarinauthor=A.+Cossarizzaauthor=R.+Espositoauthor=A.+Antinoriauthor=J.+A.+Aberg&title=Discontinuation+of+Maintenance+Therapy+for+Cryptococcal+Meningitis+in+Patients+with+AIDS+Treated+with+Highly+Active+Antiretroviral+Therapy%3A+an+International+Observational+Study&doi=10.1086%2F381261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discontinuation of maintenance therapy for cryptococcal meningitis in patients with AIDS treated with highly active antiretroviral therapy: an international observational study</span></div><div class="casAuthors">Mussini Cristina; Pezzotti Patrizio; Miro Jose M; Martinez Esteban; de Quiros Juan Carlos Lopez Bernaldo; Cinque Paola; Borghi Vanni; Bedini Andrea; Domingo Pere; Cahn Pedro; Bossi Philippe; de Luca Andrea; d'Arminio Monforte Antonella; Nelson Mark; Nwokolo Nneka; Helou Silvia; Negroni Ricardo; Jacchetti Gaia; Antinori Spinello; Lazzarin Adriano; Cossarizza Andrea; Esposito Roberto; Antinori Andrea; Aberg Judith A</div><div class="citationInfo"><span class="NLM_cas:title">Clinical infectious diseases : an official publication of the Infectious Diseases Society of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">565-71</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We conducted a retrospective, multicenter study evaluating the safety of discontinuing maintenance therapy for cryptococcal meningitis after immune reconstitution.  Inclusion criteria were a previous definitive diagnosis of cryptococcal meningitis, a CD4 cell count of >100 cells/microL while receiving highly active antiretroviral therapy (HAART), and the subsequent discontinuation of maintenance therapy for cryptococcal meningitis.  The primary end point was relapse of cryptococcal disease.  As of July 2002, 100 patients were enrolled.  When maintenance therapy was discontinued, the median CD4 cell count was 259 cells/microL and the median plasma virus load was <2.30 log10 copies/mL, and serum cryptococcal antigen was undetectable in 56 patients.  During a median follow-up period of 28.4 months (range, 6.7-64.5; 262 person-years), 4 events were observed (incidence, 1.53 events per 100 person-years; 95% confidence interval, 0.42-3.92).  Three of these patients had a CD4 cell count of >100 cells/microL and a positive serum cryptococcal antigen test result during the recurrent episode.  In conclusion, discontinuation of maintenance therapy for cryptococcal meningitis is safe if the CD4 cell count increases to >100 cells/microL while receiving HAART.  Recurrent cryptococcal infection should be suspected in patients whose serum cryptococcal antigen test results revert back to positive after discontinuation of maintenance therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQnmn6Ub2HVWPIW7hznt0zpfW6udTcc2eZBE6NPUJVhDbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c%252Fmtlaguw%253D%253D&md5=bf61cc060d032bafa195eff03773a52e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1086%2F381261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F381261%26sid%3Dliteratum%253Aachs%26aulast%3DMussini%26aufirst%3DC.%26aulast%3DPezzotti%26aufirst%3DP.%26aulast%3DMiro%26aufirst%3DJ.%2BM.%26aulast%3DMartinez%26aufirst%3DE.%26aulast%3Dde%2BQuiros%26aufirst%3DJ.%2BC.%2BL.%2BB.%26aulast%3DCinque%26aufirst%3DP.%26aulast%3DBorghi%26aufirst%3DV.%26aulast%3DBedini%26aufirst%3DA.%26aulast%3DDomingo%26aufirst%3DP.%26aulast%3DCahn%26aufirst%3DP.%26aulast%3DBossi%26aufirst%3DP.%26aulast%3Dde%2BLuca%26aufirst%3DA.%26aulast%3Dd%25E2%2580%2599Arminio%2BMonforte%26aufirst%3DA.%26aulast%3DNelson%26aufirst%3DM.%26aulast%3DNwokolo%26aufirst%3DN.%26aulast%3DHelou%26aufirst%3DS.%26aulast%3DNegroni%26aufirst%3DR.%26aulast%3DJacchetti%26aufirst%3DG.%26aulast%3DAntinori%26aufirst%3DS.%26aulast%3DLazzarin%26aufirst%3DA.%26aulast%3DCossarizza%26aufirst%3DA.%26aulast%3DEsposito%26aufirst%3DR.%26aulast%3DAntinori%26aufirst%3DA.%26aulast%3DAberg%26aufirst%3DJ.%2BA.%26atitle%3DDiscontinuation%2520of%2520Maintenance%2520Therapy%2520for%2520Cryptococcal%2520Meningitis%2520in%2520Patients%2520with%2520AIDS%2520Treated%2520with%2520Highly%2520Active%2520Antiretroviral%2520Therapy%253A%2520an%2520International%2520Observational%2520Study%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2004%26volume%3D38%26spage%3D565%26epage%3D571%26doi%3D10.1086%2F381261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santos, A. L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braga-Silva, L. A.</span></span> <span> </span><span class="NLM_article-title">Aspartic Protease Inhibitors: Effective Drugs Against the Human Fungal Pathogen Candida Albicans</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">162</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.2174%2F138955713804484802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=23256485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFertLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=155-162&author=A.+L.+S.+Santosauthor=L.+A.+Braga-Silva&title=Aspartic+Protease+Inhibitors%3A+Effective+Drugs+Against+the+Human+Fungal+Pathogen+Candida+Albicans"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Aspartic protease inhibitors: effective drugs against the human fungal pathogen Candida albicans</span></div><div class="casAuthors">Santos, Andre L. S.; Braga-Silva, Lys A.</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">155-162</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Candida albicans can invade humans and may lead to mucosal and skin infections or to deep-seated mycoses of almost all inner organs, esp. in immunocompromised patients.  In this context, both the host immune status and the ability of C. albicans to modulate the expression of its virulence factors are relevant aspects that drive the candidal susceptibility or resistance; in this last case, culminating in the establishment of successful infection known as candidiasis.  C. albicans possesses a potent armamentarium consisting of several virulence mols. that help the fungal cells to escape from the host immune responses.  There is no doubt that the secretion of aspartic proteases, designated as Saps, is one of the major virulence attributes produced by C. albicans cells, since these hydrolytic enzymes participate in a wide range of fungal physiol. processes as well as in different facets of the fungal-host interactions.  For these reasons, Saps clearly hold promise as new potential drug targets.  Corroborating this hypothesis, the introduction of anti-human immunodeficiency virus drugs of the aspartic protease inhibitor-type (HIV PIs) have emerged as new agents for the inhibition of Saps.  The introduction of HIV PIs has revolutionized the treatment of HIV disease, reducing the opportunistic infections, esp. candidiasis.  The attenuation of candidal infections in HIV-infected individuals might not solely has not only resulted from improved immunol. status, but also as a result of direct inhibition of C. albicans Saps as well as the blockage of several biol. processes controlled by these proteolytic enzymes.  The present article will discuss the updates on the functional implications of HIV PIs on the development of candidiasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxxfn1gpzdZbVg90H21EOLACvtfcHk0ljcVGYuPcO-jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFertLg%253D&md5=55fdb44e3f14cedeeecae3c5f7008180</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F138955713804484802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138955713804484802%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DA.%2BL.%2BS.%26aulast%3DBraga-Silva%26aufirst%3DL.%2BA.%26atitle%3DAspartic%2520Protease%2520Inhibitors%253A%2520Effective%2520Drugs%2520Against%2520the%2520Human%2520Fungal%2520Pathogen%2520Candida%2520Albicans%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26spage%3D155%26epage%3D162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sigrist, C. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gattiker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falquet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bairoch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bucher, P.</span></span> <span> </span><span class="NLM_article-title">PROSITE: a Documented Database Using Patterns and Profiles as Motif Descriptors</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1093/bib/3.3.265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fbib%2F3.3.265" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=12230035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsFOhtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=265-274&author=C.+J.+A.+Sigristauthor=L.+Ceruttiauthor=N.+Huloauthor=A.+Gattikerauthor=L.+Falquetauthor=M.+Pagniauthor=A.+Bairochauthor=P.+Bucher&title=PROSITE%3A+a+Documented+Database+Using+Patterns+and+Profiles+as+Motif+Descriptors&doi=10.1093%2Fbib%2F3.3.265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PROSITE: A documented database using patterns and profiles as motif descriptors</span></div><div class="casAuthors">Sigrist, Christian J. A.; Cerutti, Lorenzo; Hulo, Nicolas; Gattiker, Alexandre; Falquet, Laurent; Pagni, Marco; Bairoch, Amos; Bucher, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">265-274</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1467-5463</span>.
    
            (<span class="NLM_cas:orgname">Henry Stewart Publications</span>)
        </div><div class="casAbstract">The main characteristics and the techniques of family and domain identification used by PROSITE are described.  Among the various databases dedicated to the identification protein families and domains, PROSITE is the first one created and has continuously evolved since.  PROSITE currently consists of a large collection of biol. meaningful motifs that are described as patterns or profiles, and linked to documentation briefly describing the protein family or domain they are designed to detect.  The close relation of PROSITE with the SWISS-PROT protein database allows the evaluation of the sensitivity and specificity of the FROSITE motifs and their periodic reviewing.  In return, PROSITE is used to help annotate SWISS-PROT entries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZjk2fEuPFM7Vg90H21EOLACvtfcHk0lgEUwNznsJdNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsFOhtbc%253D&md5=6385337d47f40fd1b16f63a0cc7b0320</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fbib%2F3.3.265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252F3.3.265%26sid%3Dliteratum%253Aachs%26aulast%3DSigrist%26aufirst%3DC.%2BJ.%2BA.%26aulast%3DCerutti%26aufirst%3DL.%26aulast%3DHulo%26aufirst%3DN.%26aulast%3DGattiker%26aufirst%3DA.%26aulast%3DFalquet%26aufirst%3DL.%26aulast%3DPagni%26aufirst%3DM.%26aulast%3DBairoch%26aufirst%3DA.%26aulast%3DBucher%26aufirst%3DP.%26atitle%3DPROSITE%253A%2520a%2520Documented%2520Database%2520Using%2520Patterns%2520and%2520Profiles%2520as%2520Motif%2520Descriptors%26jtitle%3DBriefings%2520Bioinf.%26date%3D2002%26volume%3D3%26spage%3D265%26epage%3D274%26doi%3D10.1093%2Fbib%2F3.3.265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Heijne, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, H.</span></span> <span> </span><span class="NLM_article-title">SignalP 4.0: Discriminating Signal Peptides from Transmembrane Regions</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">786</span>, <span class="refDoi"> DOI: 10.1038/nmeth.1701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1038%2Fnmeth.1701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=21959131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1CrtrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2011&pages=785-786&author=T.+N.+Petersenauthor=S.+Brunakauthor=G.+von+Heijneauthor=H.+Nielsen&title=SignalP+4.0%3A+Discriminating+Signal+Peptides+from+Transmembrane+Regions&doi=10.1038%2Fnmeth.1701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">SignalP 4.0: discriminating signal peptides from transmembrane regions</span></div><div class="casAuthors">Petersen, Thomas Nordahl; Brunak, Soeren; von Heijne, Gunnar; Nielsen, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">785-786</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">SignalP 4.0 is a purely neural network-based method designed to discriminate between signal peptides and transmembrane regions.  Evaluation of prediction performance of SignalP 4.0 showed that it was the best signal-peptide predictor for all three organism types, namely eukaryotes, Gram-pos. bacteria and Gram-neg. bacteria.  This comes at a price, however, because SignalP 4.0 was not in all cases as good as SignalP 3.0 according to cleavage-site sensitivity or signal-peptide correlation when there are no transmembrane proteins present.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoapAuMBGaJXbVg90H21EOLACvtfcHk0lgEUwNznsJdNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1CrtrbL&md5=543e761a8f7509a1cdc1209c1d08b6c9</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1701%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DT.%2BN.%26aulast%3DBrunak%26aufirst%3DS.%26aulast%3Dvon%2BHeijne%26aufirst%3DG.%26aulast%3DNielsen%26aufirst%3DH.%26atitle%3DSignalP%25204.0%253A%2520Discriminating%2520Signal%2520Peptides%2520from%2520Transmembrane%2520Regions%26jtitle%3DNat.%2520Methods%26date%3D2011%26volume%3D8%26spage%3D785%26epage%3D786%26doi%3D10.1038%2Fnmeth.1701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, B. M.</span></span> <span> </span><span class="NLM_article-title">Structure and Mechanism of the Pepsin-like Family of Aspartic Peptidases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4431</span>– <span class="NLM_lpage">4458</span>, <span class="refDoi"> DOI: 10.1021/cr010167q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr010167q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFWhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2002&pages=4431-4458&author=B.+M.+Dunn&title=Structure+and+Mechanism+of+the+Pepsin-like+Family+of+Aspartic+Peptidases&doi=10.1021%2Fcr010167q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and mechanism of the pepsin-like family of aspartic peptidases</span></div><div class="casAuthors">Dunn, Ben M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4431-4458</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Topics discussed include the cloning and expression of new aspartic endopeptidases, structural aspects of aspartic peptidases, the N-terminal prosegment region of aspartic peptidases, active site engineering, substrate and inhibitor design, and synthetic and naturally occurring inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU2C94Lnni_LVg90H21EOLACvtfcHk0li9tLxHCmDZ6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFWhtLk%253D&md5=0e6b1f501111075667ea562d9d3ca132</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcr010167q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr010167q%26sid%3Dliteratum%253Aachs%26aulast%3DDunn%26aufirst%3DB.%2BM.%26atitle%3DStructure%2520and%2520Mechanism%2520of%2520the%2520Pepsin-like%2520Family%2520of%2520Aspartic%2520Peptidases%26jtitle%3DChem.%2520Rev.%26date%3D2002%26volume%3D102%26spage%3D4431%26epage%3D4458%26doi%3D10.1021%2Fcr010167q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarla, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.-R.</span></span> <span> </span><span class="NLM_article-title">HAART in HIV/AIDS Treatments: Future Trends</span>. <i>Infect. Disord.: Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.2174/1871526517666170505122800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.2174%2F1871526517666170505122800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28474549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVCqt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=15-22&author=D.-Y.+Luauthor=H.-Y.+Wuauthor=N.+S.+Yarlaauthor=B.+Xuauthor=J.+Dingauthor=T.-R.+Lu&title=HAART+in+HIV%2FAIDS+Treatments%3A+Future+Trends&doi=10.2174%2F1871526517666170505122800"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">HAART in HIV/AIDS Treatments: Future Trends</span></div><div class="casAuthors">Lu, Da-Yong; Wu, Hong-Ying; Yarla, Nagendra Sastry; Xu, Bin; Ding, Jian; Lu, Ting-Ren</div><div class="citationInfo"><span class="NLM_cas:title">Infectious Disorders: Drug Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-22</span>CODEN:
                <span class="NLM_cas:coden">IDDTAD</span>;
        ISSN:<span class="NLM_cas:issn">1871-5265</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  AIDS (acquired immune deficient syndrome) is a deadly human viral infectious disease caused by HIV (human immune-deficient virus) infection.  Almost every AIDS patient losses his/her life before mid 1990s.  AIDS was once the 1st disease killer in US (1993).  After one decade hard work, antiviral drug cocktails-high active anti-retroviral therapy (HAART) have been invented for almost all HIV infection treatments.  Due to the invention of HAART, 80-90% HIV/AIDS patients still effectively response to HAART for deadly AIDS episode controls and life saving.  Yet, this type of HIV therapeutics is incurable.  HIV/AIDS patients need to take HAART medications regularly and even life-long.  To counteract this therapeutic drawback, more revolutionary efforts (different angles of therapeutic modes/attempts) are urgently needed.  In this article, the major progresses and drawbacks of HIV/AIDS chemotherapy (HAART) to HIV/AIDS patients have been discussed.  Future trends (updating pathogenesis study, next generations of drug developments, new drug target discovery, different scientific disciplinary and so on) are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6WbPH3DgABrVg90H21EOLACvtfcHk0li9tLxHCmDZ6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntVCqt78%253D&md5=26ffa17208e500e760b6d1dfde1ffcdc</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2174%2F1871526517666170505122800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1871526517666170505122800%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DD.-Y.%26aulast%3DWu%26aufirst%3DH.-Y.%26aulast%3DYarla%26aufirst%3DN.%2BS.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.-R.%26atitle%3DHAART%2520in%2520HIV%252FAIDS%2520Treatments%253A%2520Future%2520Trends%26jtitle%3DInfect.%2520Disord.%253A%2520Drug%2520Targets%26date%3D2018%26volume%3D18%26spage%3D15%26epage%3D22%26doi%3D10.2174%2F1871526517666170505122800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alteri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cozens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faessler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimkait, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazdins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poncioni, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roesel, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span> <span> </span><span class="NLM_article-title">CGP 53437, an Orally Bioavailable Inhibitor of Human Immunodeficiency Virus Type 1 Protease with Potent Antiviral Activity</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2087</span>– <span class="NLM_lpage">2092</span>, <span class="refDoi"> DOI: 10.1128/aac.37.10.2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FAAC.37.10.2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8257128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK2cXpsVygsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1993&pages=2087-2092&author=E.+Alteriauthor=G.+Boldauthor=R.+Cozensauthor=A.+Faesslerauthor=T.+Klimkaitauthor=M.+Langauthor=J.+Lazdinsauthor=B.+Poncioniauthor=J.+L.+Roeselauthor=P.+Schneider&title=CGP+53437%2C+an+Orally+Bioavailable+Inhibitor+of+Human+Immunodeficiency+Virus+Type+1+Protease+with+Potent+Antiviral+Activity&doi=10.1128%2Faac.37.10.2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">CGP 53437, an orally bioavailable inhibitor of human immunodeficiency virus type 1 protease with potent antiviral activity</span></div><div class="casAuthors">Alteri, E.; Bold, G.; Cozens, R.; Faessler, A.; Klimkait, T.; Lang, M.; Lazdins, J.; Poncioni, B.; Roesel, J. L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2087-92</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    </div><div class="casAbstract">CGP 53437 is a peptidomimetic inhibitor of human immunodeficiency virus type 1 proteinase contg. a hydroxyethylene isostere.  The compd. inhibited recombinant human immunodeficiency virus-1 proteinase with a Ki of 0.2 nM.  The inhibition const. vs. human cathepsin D and human cathepsin E was 4 nM.  Human pepsin and gastricsin were inhibited with Kis of 8 and 500 nM, resp., and human renin was inhibited with a Ki of 190 μM.  The replication of human immunodeficiency virus-1/LAV, human immunodeficiency virus-1/Z-84, and human immunodeficiency virus-1/pLAI was inhibited with a 90% ED of 0.1 μM in acutely infected MT-2 cells.  The 50% cytotoxic dose was 100 μM.  Similar antiviral activity was obsd. when the compd. was added up to 10 h after infection.  At the effective concn., processing of Gag precursor protein p55 was greatly reduced, confirming an action on the late stage of the virus life cycle, as expected.  The efficacy of the inhibitor was also demonstrated by using primary human peripheral blood lymphocytes infected with the human immunodeficiency virus-1/LAV strain, low-passage clin. isolates obtained from human immunodeficiency virus-1-seropos. individuals (including a zidovudine-resistant strain), and human immunodeficiency virus-2/ROD.  In these cells, CGP 53437 delayed the onset of human immunodeficiency virus replication in a dose-dependent fashion (substantial effects with concns. of ≥0.1 μM) as long as the inhibitor was maintained in the culture.  CGP 53437 was orally bioavailable in mice.  Concns. in plasma 10-fold in excess of the in vitro antiviral 90% ED could be sustained for several h after oral application of 120 mg/kg.  Therefore, CGP 53437 has the potential to be a therapeutically useful anti-human immunodeficiency virus agent for the treatment of AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_5zVOGNIqIrVg90H21EOLACvtfcHk0liOve3KA4mp4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXpsVygsg%253D%253D&md5=9e0976269a28a2f995c850987e4f931d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1128%2FAAC.37.10.2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.37.10.2087%26sid%3Dliteratum%253Aachs%26aulast%3DAlteri%26aufirst%3DE.%26aulast%3DBold%26aufirst%3DG.%26aulast%3DCozens%26aufirst%3DR.%26aulast%3DFaessler%26aufirst%3DA.%26aulast%3DKlimkait%26aufirst%3DT.%26aulast%3DLang%26aufirst%3DM.%26aulast%3DLazdins%26aufirst%3DJ.%26aulast%3DPoncioni%26aufirst%3DB.%26aulast%3DRoesel%26aufirst%3DJ.%2BL.%26aulast%3DSchneider%26aufirst%3DP.%26atitle%3DCGP%252053437%252C%2520an%2520Orally%2520Bioavailable%2520Inhibitor%2520of%2520Human%2520Immunodeficiency%2520Virus%2520Type%25201%2520Protease%2520with%2520Potent%2520Antiviral%2520Activity%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1993%26volume%3D37%26spage%3D2087%26epage%3D2092%26doi%3D10.1128%2Faac.37.10.2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callebaut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stray, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavitt, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priskich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span> <span> </span><span class="NLM_article-title">In Vitro Characterization of GS-8374, a Novel Phosphonate-Containing Inhibitor of HIV-1 Protease with a Favorable Resistance Profile</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1366</span>– <span class="NLM_lpage">1376</span>, <span class="refDoi"> DOI: 10.1128/aac.01183-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FAAC.01183-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=21245449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFyntbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=1366-1376&author=C.+Callebautauthor=K.+Strayauthor=L.+Tsaiauthor=M.+Williamsauthor=Z.-Y.+Yangauthor=C.+Cannizzaroauthor=S.+A.+Leavittauthor=X.+Liuauthor=K.+Wangauthor=B.+P.+Murrayauthor=A.+Mulatoauthor=M.+Hatadaauthor=T.+Priskichauthor=N.+Parkinauthor=S.+Swaminathanauthor=W.+Leeauthor=G.-X.+Heauthor=L.+Xuauthor=T.+Cihlar&title=In+Vitro+Characterization+of+GS-8374%2C+a+Novel+Phosphonate-Containing+Inhibitor+of+HIV-1+Protease+with+a+Favorable+Resistance+Profile&doi=10.1128%2Faac.01183-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro characterization of GS-8374, a novel phosphonate-containing inhibitor of HIV-1 protease with a favorable resistance profile</span></div><div class="casAuthors">Callebaut, Christian; Stray, Kirsten; Tsai, Luong; Williams, Matt; Yang, Zheng-Yu; Cannizzaro, Carina; Leavitt, Stephanie A.; Liu, Xiaohong; Wang, Kelly; Murray, Bernard P.; Mulato, Andrew; Hatada, Marcos; Priskich, Tina; Parkin, Neil; Swaminathan, Swami; Lee, William; He, Gong-Xin; Xu, Lianhong; Cihlar, Tomas</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1366-1376</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">GS-8374 is a novel bis-tetrahydrofuran HIV-1 protease (PR) inhibitor (PI) with a unique diethylphosphonate moiety.  It was selected from a series of analogs contg. various di(alkyl)phosphonate substitutions connected via a linker to the para position of a P-1 Ph ring.  GS-8374 inhibits HIV-1 PR with high potency (Ki = 8.1 pM) and with no known effect on host proteases.  Kinetic and thermodn. anal. of GS-8374 binding to PR demonstrated an extremely slow off rate for the inhibitor and favorable contributions of both the enthalpic and entropic components to the total free binding energy.  GS-8374 showed potent antiretroviral activity in T-cell lines, primary CD4+ T cells (50% effective concn. [EC50] = 3.4 to 11.5 nM), and macrophages (EC50 = 25.5 nM) and exhibited low cytotoxicity in multiple human cell types.  The antiviral potency of GS-8374 was only moderately affected by human serum protein binding, and its combination with multiple approved antiretrovirals showed synergistic effects.  When it was tested in a PhenoSense assay against a panel of 24 patient-derived viruses with high-level PI resistance, GS-8374 showed lower mean EC50s and lower fold resistance than any of the clin. approved PIs.  Similar to other PIs, in vitro hepatic microsomal metab. of GS-8374 was efficiently blocked by ritonavir, suggesting a potential for effective pharmacokinetic boosting in vivo.  In summary, results from this broad in vitro pharmacol. profiling indicate that GS-8374 is a promising candidate to be further assessed as a new antiretroviral agent with potential for clin. efficacy in both treatment-naive and -experienced patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC6gBbIaR7irVg90H21EOLACvtfcHk0liOve3KA4mp4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFyntbo%253D&md5=d4f8da97ca6d462349a282997d08b096</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1128%2FAAC.01183-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01183-10%26sid%3Dliteratum%253Aachs%26aulast%3DCallebaut%26aufirst%3DC.%26aulast%3DStray%26aufirst%3DK.%26aulast%3DTsai%26aufirst%3DL.%26aulast%3DWilliams%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DZ.-Y.%26aulast%3DCannizzaro%26aufirst%3DC.%26aulast%3DLeavitt%26aufirst%3DS.%2BA.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DMurray%26aufirst%3DB.%2BP.%26aulast%3DMulato%26aufirst%3DA.%26aulast%3DHatada%26aufirst%3DM.%26aulast%3DPriskich%26aufirst%3DT.%26aulast%3DParkin%26aufirst%3DN.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DG.-X.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DCihlar%26aufirst%3DT.%26atitle%3DIn%2520Vitro%2520Characterization%2520of%2520GS-8374%252C%2520a%2520Novel%2520Phosphonate-Containing%2520Inhibitor%2520of%2520HIV-1%2520Protease%2520with%2520a%2520Favorable%2520Resistance%2520Profile%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2011%26volume%3D55%26spage%3D1366%26epage%3D1376%26doi%3D10.1128%2Faac.01183-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furfine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaltenstein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St. Clair, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanlon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boone, L.</span></span> <span> </span><span class="NLM_article-title">In Vitro Antiviral Activity of the Novel, Tyrosyl-Based Human Immunodeficiency Virus (HIV) Type 1 Protease Inhibitor Brecanavir (GW640385) in Combination with Other Antiretrovirals and against a Panel of Protease Inhibitor-Resistant HIV</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3147</span>– <span class="NLM_lpage">3154</span>, <span class="refDoi"> DOI: 10.1128/aac.00401-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1128%2FAAC.00401-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=17620375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVais73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=3147-3154&author=R.+Hazenauthor=R.+Harveyauthor=R.+Ferrisauthor=C.+Craigauthor=P.+Yatesauthor=P.+Griffinauthor=J.+Millerauthor=I.+Kaldorauthor=J.+Rayauthor=V.+Samanoauthor=E.+Furfineauthor=A.+Spaltensteinauthor=M.+Haleauthor=R.+Tungauthor=M.+St.+Clairauthor=M.+Hanlonauthor=L.+Boone&title=In+Vitro+Antiviral+Activity+of+the+Novel%2C+Tyrosyl-Based+Human+Immunodeficiency+Virus+%28HIV%29+Type+1+Protease+Inhibitor+Brecanavir+%28GW640385%29+in+Combination+with+Other+Antiretrovirals+and+against+a+Panel+of+Protease+Inhibitor-Resistant+HIV&doi=10.1128%2Faac.00401-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity of the novel, tyrosyl-based human immunodeficiency virus (HIV) type 1 protease inhibitor brecanavir (GW640385) in combination with other antiretrovirals and against a panel of protease inhibitor-resistant HIV</span></div><div class="casAuthors">Hazen, Richard; Harvey, Robert; Ferris, Robert; Craig, Charles; Yates, Phillip; Griffin, Philip; Miller, John; Kaldor, Istvan; Ray, John; Samano, Vincente; Furfine, Eric; Spaltenstein, Andrew; Hale, Michael; Tung, Roger; St. Clair, Marty; Hanlon, Mary; Boone, Lawrence</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3147-3154</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Brecanavir, a novel tyrosyl-based arylsulfonamide, high-affinity, human immunodeficiency virus type 1 (HIV-1) protease inhibitor (PI), has been evaluated for anti-HIV activity in several in vitro assays.  Preclin. assessment of brecanavir indicated that this compd. potently inhibited HIV-1 in cell culture assays with 50% effective concns. (EC50s) of 0.2 to 0.53 nM and was equally active against HIV strains utilizing either the CXCR4 or CCR5 coreceptor, as was found with other PIs.  The presence of up to 40% human serum decreased the anti-HIV-1 activity of brecanavir by 5.2-fold, but under these conditions the compd. retained single-digit nanomolar EC50s.  When brecanavir was tested in combination with nucleoside reverse transcriptase inhibitors, the antiviral activity of brecanavir was synergistic with the effects of stavudine and additive to the effects of zidovudine, tenofovir, dideoxycytidine, didanosine, adefovir, abacavir, lamivudine, and emtricitabine.  Brecanavir was synergistic with the nonnucleoside reverse transcriptase inhibitor nevirapine or delavirdine and was additive to the effects of efavirenz.  In combination with other PIs, brecanavir was additive to the activities of indinavir, lopinavir, nelfinavir, ritonavir, amprenavir, saquinavir, and atazanavir.  Clin. HIV isolates from PI-experienced patients were evaluated for sensitivity to brecanavir and other PIs in a recombinant virus assay.  Brecanavir had a <5-fold increase in EC50s against 80% of patient isolates tested and had a greater mean in vitro potency than amprenavir, indinavir, lopinavir, atazanavir, tipranavir, and darunavir.  Brecanavir is by a substantial margin the most potent and broadly active antiviral agent among the PIs tested in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqplFzzXvQJ9rVg90H21EOLACvtfcHk0lghKZC0a3d2Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVais73F&md5=a84fb4b36143fb6c9e92db4895ad1d9d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1128%2FAAC.00401-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00401-07%26sid%3Dliteratum%253Aachs%26aulast%3DHazen%26aufirst%3DR.%26aulast%3DHarvey%26aufirst%3DR.%26aulast%3DFerris%26aufirst%3DR.%26aulast%3DCraig%26aufirst%3DC.%26aulast%3DYates%26aufirst%3DP.%26aulast%3DGriffin%26aufirst%3DP.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DKaldor%26aufirst%3DI.%26aulast%3DRay%26aufirst%3DJ.%26aulast%3DSamano%26aufirst%3DV.%26aulast%3DFurfine%26aufirst%3DE.%26aulast%3DSpaltenstein%26aufirst%3DA.%26aulast%3DHale%26aufirst%3DM.%26aulast%3DTung%26aufirst%3DR.%26aulast%3DSt.%2BClair%26aufirst%3DM.%26aulast%3DHanlon%26aufirst%3DM.%26aulast%3DBoone%26aufirst%3DL.%26atitle%3DIn%2520Vitro%2520Antiviral%2520Activity%2520of%2520the%2520Novel%252C%2520Tyrosyl-Based%2520Human%2520Immunodeficiency%2520Virus%2520%2528HIV%2529%2520Type%25201%2520Protease%2520Inhibitor%2520Brecanavir%2520%2528GW640385%2529%2520in%2520Combination%2520with%2520Other%2520Antiretrovirals%2520and%2520against%2520a%2520Panel%2520of%2520Protease%2520Inhibitor-Resistant%2520HIV%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D3147%26epage%3D3154%26doi%3D10.1128%2Faac.00401-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sielecki, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayakawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carilli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewicki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M.</span></span> <span> </span><span class="NLM_article-title">Structure of Recombinant Human Renin, a Target for Cardiovascular-active Drugs, at 2.5 A resolution</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>243</i></span>,  <span class="NLM_fpage">1346</span>– <span class="NLM_lpage">1351</span>, <span class="refDoi"> DOI: 10.1126/science.2493678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1126%2Fscience.2493678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=2493678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL1MXhs12htbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=243&publication_year=1989&pages=1346-1351&author=A.+R.+Sieleckiauthor=K.+Hayakawaauthor=M.+Fujinagaauthor=M.+Murphyauthor=M.+Fraserauthor=A.+Muirauthor=C.+Carilliauthor=J.+Lewickiauthor=J.+Baxterauthor=M.+James&title=Structure+of+Recombinant+Human+Renin%2C+a+Target+for+Cardiovascular-active+Drugs%2C+at+2.5+A+resolution&doi=10.1126%2Fscience.2493678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of recombinant human renin, a target for cardiovascular-active drugs, at 2.5 Å resolution</span></div><div class="casAuthors">Sielecki, Anita R.; Hayakawa, Koto; Fujinaga, Masao; Murphy, Michael E. P.; Fraser, Marie; Muir, Alastaiar K.; Carilli, Cynthia T.; Lewicki, John A.; Baxter, John D.; James, Michael N. G.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">243</span>
        (<span class="NLM_cas:issue">4896</span>),
    <span class="NLM_cas:pages">1346-51</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The x-ray crystal structure of recombinant human renin was detd.  Mol. dynamics techniques that included crystallog. data as a restraint were used to improve an initial model based on porcine pepsinogen.  The present agreement factor for data at 8.0-2.5 Å is 0.236.  Some of the surface loops are poorly detd., and these disordered regions border a 30 Å wide solvent channel.  Comparison of renin with other aspartyl proteinases shows that, although the structural cores and active sites are highly conserved, surface residues, some of which are crit. for specificity, vary greatly (≤10 Å).  Knowledge of the actual structure, as opposed to the use of models based on related enzymes, should facilitate the design of renin inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfLdkSwz5asrVg90H21EOLACvtfcHk0lghKZC0a3d2Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhs12htbo%253D&md5=2ac6057c5421181be26cd23e55eb1c5a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.2493678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2493678%26sid%3Dliteratum%253Aachs%26aulast%3DSielecki%26aufirst%3DA.%2BR.%26aulast%3DHayakawa%26aufirst%3DK.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DMurphy%26aufirst%3DM.%26aulast%3DFraser%26aufirst%3DM.%26aulast%3DMuir%26aufirst%3DA.%26aulast%3DCarilli%26aufirst%3DC.%26aulast%3DLewicki%26aufirst%3DJ.%26aulast%3DBaxter%26aufirst%3DJ.%26aulast%3DJames%26aufirst%3DM.%26atitle%3DStructure%2520of%2520Recombinant%2520Human%2520Renin%252C%2520a%2520Target%2520for%2520Cardiovascular-active%2520Drugs%252C%2520at%25202.5%2520A%2520resolution%26jtitle%3DScience%26date%3D1989%26volume%3D243%26spage%3D1346%26epage%3D1351%26doi%3D10.1126%2Fscience.2493678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, M. N. G.</span>; <span class="NLM_string-name">Sielecki, A. R.</span></span> <span> </span><span class="NLM_article-title">Aspartic Proteinases and their Catalytic Pathway</span>. In  <i>Biological Macromolecules and Assemblies, Vol. 3 Active Sites of Enzymes</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurnak, F. A.</span>, <span class="NLM_string-name">McPherson, A.</span></span>, Eds.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1987</span>; pp  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">482</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1987&pages=413-482&author=M.+N.+G.+James&author=A.+R.+Sieleckiauthor=F.+A.+Jurnak&author=A.+McPherson&title=Biological+Macromolecules+and+Assemblies%2C+Vol.+3+Active+Sites+of+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DM.%2BN.%2BG.%26atitle%3DAspartic%2520Proteinases%2520and%2520their%2520Catalytic%2520Pathway%26btitle%3DBiological%2520Macromolecules%2520and%2520Assemblies%252C%2520Vol.%25203%2520Active%2520Sites%2520of%2520Enzymes%26aulast%3DJurnak%26aufirst%3DF.%2BA.%26pub%3DWiley%26date%3D1987%26spage%3D413%26epage%3D482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M. N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Structural Evidence for Gene Duplication in the Evolution of the Acid Proteases</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">618</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1038/271618a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1038%2F271618a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=24179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaE1cXhvVGhtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1978&pages=618-621&author=J.+Tangauthor=M.+N.+G.+Jamesauthor=I.+N.+Hsuauthor=J.+A.+Jenkinsauthor=T.+L.+Blundell&title=Structural+Evidence+for+Gene+Duplication+in+the+Evolution+of+the+Acid+Proteases&doi=10.1038%2F271618a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structural evidence for gene duplication in the evolution of the acid proteases</span></div><div class="casAuthors">Tang, J.; James, M. N. G.; Hsu, I. N.; Jenkins, J. A.; Blundell, T. L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">5646</span>),
    <span class="NLM_cas:pages">618-21</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">X-ray studies of acid proteases (I) indicated a bilobal structure with a well-defined active-site cleft.  An intramol. 2-fold symmetry axis related 2 topol. similar domains and the active-site residues.  I may have evolved by gene duplication of an ancestral protein of ∼150 residues, having a fold similar to that of one lobe of pepsin.  The 2 subunits may then have evolved divergently so that the carboxyl groups of the 2 aspartyl residues developed different pKs and a mechanism of hydrolytic cleavage.  Gene fusion would be selectively advantageous in assuring the correct assocn. of dissimilar subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopTbDD0MPpC7Vg90H21EOLACvtfcHk0lhydyrI67weZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXhvVGhtbg%253D&md5=cb6501707d35e0fbecb6921ad2c0c590</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2F271618a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F271618a0%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DJames%26aufirst%3DM.%2BN.%2BG.%26aulast%3DHsu%26aufirst%3DI.%2BN.%26aulast%3DJenkins%26aufirst%3DJ.%2BA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520Evidence%2520for%2520Gene%2520Duplication%2520in%2520the%2520Evolution%2520of%2520the%2520Acid%2520Proteases%26jtitle%3DNature%26date%3D1978%26volume%3D271%26spage%3D618%26epage%3D621%26doi%3D10.1038%2F271618a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sewell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLachlan, A. D.</span></span> <span> </span><span class="NLM_article-title">Four-fold Structural Repeat in the Acid Proteases</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1016/0005-2795(79)90194-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2F0005-2795%2879%2990194-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=44681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaE1MXlslSkur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=1979&pages=24-31&author=T.+L.+Blundellauthor=B.+T.+Sewellauthor=A.+D.+McLachlan&title=Four-fold+Structural+Repeat+in+the+Acid+Proteases&doi=10.1016%2F0005-2795%2879%2990194-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Four-fold structural repeat in the acid proteases</span></div><div class="casAuthors">Blundell, T. L.; Sewell, B. T.; McLachlan, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Protein Structure</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-31</span>CODEN:
                <span class="NLM_cas:coden">BBPTBH</span>;
        ISSN:<span class="NLM_cas:issn">0005-2795</span>.
    </div><div class="casAbstract">The observation that the acid proteases contain 2 structurally equiv. lobes related by a dyad through the active center was extended to show that in endothiapepsin (Endothia acid protease) each lobe contains 2 similar halves related by a further local dyad.  In lobe 1, 22 pairs of α-carbons are equiv. with a root mean square deviation of 1.92 Å.  In lobe 2, 17 pairs match within 2.31 Å.  Convergent evolution or gene quadruplication may have occurred.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGmUx9-sOCG7Vg90H21EOLACvtfcHk0lhydyrI67weZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1MXlslSkur4%253D&md5=b8596d402106e37ea343b7812738d2b6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2F0005-2795%2879%2990194-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2795%252879%252990194-6%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DSewell%26aufirst%3DB.%2BT.%26aulast%3DMcLachlan%26aufirst%3DA.%2BD.%26atitle%3DFour-fold%2520Structural%2520Repeat%2520in%2520the%2520Acid%2520Proteases%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1979%26volume%3D580%26spage%3D24%26epage%3D31%26doi%3D10.1016%2F0005-2795%2879%2990194-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, M. N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sielecki, A. R.</span></span> <span> </span><span class="NLM_article-title">Structure and refinement of penicillopepsin at 1.8 Å resolution</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1016/0022-2836(83)90008-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2F0022-2836%2883%2990008-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=6341600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL3sXht1Wqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=1983&pages=299-361&author=M.+N.+G.+Jamesauthor=A.+R.+Sielecki&title=Structure+and+refinement+of+penicillopepsin+at+1.8+%C3%85+resolution&doi=10.1016%2F0022-2836%2883%2990008-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and refinement of penicillopepsin at 1.8 Å resolution</span></div><div class="casAuthors">James, Michael N. G.; Sielecki, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">299-361</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Penicillopepsin (I), the aspartyl protease from Penicillium janthinellum, had its mol. structure refined by a restrained-parameter least-squares procedure at 1.8 Å resoln.  The av. at. thermal vibration parameter (B) for the atoms of I is 14.5 Å2.  One detg. factor of this low av. B value is the large central hydrophobic core, in which there are 2 prominent clusters of arom. residues, one of 9, the other of 7 residues.  The N- and C-terminal domains of I display an approx. 2-fold symmetry: 70 residue pairs are topol. equiv., related by a rotation of 177° and a translation of 1.2 Å.  Anal. of the secondary structural features of the mol. reveals nonlinear H bonding.  The av. at. B-factors of the residues in the active site region of the mol. are 5-8 Å2, confirming the proposal that conformational mobility of the active site residues has no role in the enzymic mechanism.  However, conformational mobility of neighboring regions of the mol., e.g., the flap contg. tyrosine-75, is verified by the high B-factors for those residues.  The positions of 319 solvent sites/asym. unit were selected from difference electron d. maps and refined.  Thirteen were classified as internal, and several of these may have key roles during catalysis.  The intermol. packing involves many protein-protein contacts (66 residues) with a large no. of solvent mols. involved in bridging between polar residues at the contact surface.  I mols. resembled an approx. hexagonal close-packing of spheres with each mol. having 12 nearest neighbors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd_uOSJ-pTlLVg90H21EOLACvtfcHk0lhydyrI67weZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXht1Wqu7o%253D&md5=0e042c2bf7fb1475465df0aed6dc111f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2883%2990008-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252883%252990008-6%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DM.%2BN.%2BG.%26aulast%3DSielecki%26aufirst%3DA.%2BR.%26atitle%3DStructure%2520and%2520refinement%2520of%2520penicillopepsin%2520at%25201.8%2520%25C3%2585%2520resolution%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1983%26volume%3D163%26spage%3D299%26epage%3D361%26doi%3D10.1016%2F0022-2836%2883%2990008-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenström, P.</span></span> <span> </span><span class="NLM_article-title">Dali Server: Conservation Mapping in 3D</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">W545</span>– <span class="NLM_lpage">W549</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fnar%2Fgkq366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=20457744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVSqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W545-W549&author=L.+Holmauthor=P.+Rosenstr%C3%B6m&title=Dali+Server%3A+Conservation+Mapping+in+3D&doi=10.1093%2Fnar%2Fgkq366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Dali server: conservation mapping in 3D</span></div><div class="casAuthors">Holm, Liisa; Rosenstrom, Paivi</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W545-W549</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Our web site (http://ekhidna.biocenter.helsinki.fi/daliserver) runs the Dali program for protein structure comparison.  The web site consists of three parts: (i) the Dali server compares newly solved structures against structures in the Protein Data Bank (PDB), (ii) the Dali database allows browsing precomputed structural neighborhoods and (iii) the pairwise comparison generates suboptimal alignments for a pair of structures.  Each part has its own query form and a common format for the results page.  The inputs are either PDB identifiers or novel structures uploaded by the user.  The results pages are hyperlinked to aid interactive anal.  The web interface is simple and easy to use.  The key purpose of interactive anal. is to check whether conserved residues line up in multiple structural alignments and how conserved residues and ligands cluster together in multiple structure superimpositions.  In favorable cases, protein structure comparison can lead to evolutionary discoveries not detected by sequence anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdTA8Qcz4NHLVg90H21EOLACvtfcHk0ljgtfAYm7L6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVSqsr4%253D&md5=b7832262058d6a4f7d362f4ce565a666</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq366%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DRosenstr%25C3%25B6m%26aufirst%3DP.%26atitle%3DDali%2520Server%253A%2520Conservation%2520Mapping%2520in%25203D%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW545%26epage%3DW549%26doi%3D10.1093%2Fnar%2Fgkq366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laakso, L. M.</span></span> <span> </span><span class="NLM_article-title">Dali Server Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">355</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fnar%2Fgkw357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=351-355&author=L.+Holmauthor=L.+M.+Laakso&title=Dali+Server+Update&doi=10.1093%2Fnar%2Fgkw357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Dali server update</span></div><div class="casAuthors">Holm, Liisa; Laakso, Laura M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W351-W355</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Dali server is a network service for comparing protein structures in 3D.  In favorable cases, comparing 3D structures may reveal biol. interesting similarities that are not detectable by comparing sequences.  The Dali server has been running in various places for over 20 years and is used routinely by crystallographers on newly solved structures.  The latest update of the server provides enhanced analytics for the study of sequence and structure conservation.  The server performs three types of structure comparisons: (i) Protein Data Bank (PDB) search compares one query structure against those in the PDB and returns a list of similar structures; (ii) pairwise comparison compares one query structure against a list of structures specified by the user; and (iii) all against all structure comparison returns a structural similarity matrix, a dendrogram and a multidimensional scaling projection of a set of structures specified by the user.  Structural superimpositions are visualized using the Java-free WebGL viewer PV.  The structural alignment view is enhanced by sequence similarity searches against Uniprot.  The combined structure-sequence alignment information is compressed to a stack of aligned sequence logos.  In the stack, each structure is structurally aligned to the query protein and represented by a sequence logo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9jpe1HGTYrLVg90H21EOLACvtfcHk0ljgtfAYm7L6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrrO&md5=cd2b5fabf6a01a4b35f350342ae7ddc2</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw357%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DLaakso%26aufirst%3DL.%2BM.%26atitle%3DDali%2520Server%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D351%26epage%3D355%26doi%3D10.1093%2Fnar%2Fgkw357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Navia, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, P. M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeever, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leu, C.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimbach, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herber, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigal, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darke, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, J. P.</span></span> <span> </span><span class="NLM_article-title">Three-dimensional Structure of Aspartyl Protease from Human Immunodeficiency Virus HIV-1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>337</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1038/337615a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1038%2F337615a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=2645523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL1MXhs12hsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=337&publication_year=1989&pages=615-620&author=M.+A.+Naviaauthor=P.+M.+D.+Fitzgeraldauthor=B.+M.+McKeeverauthor=C.-T.+Leuauthor=J.+C.+Heimbachauthor=W.+K.+Herberauthor=I.+S.+Sigalauthor=P.+L.+Darkeauthor=J.+P.+Springer&title=Three-dimensional+Structure+of+Aspartyl+Protease+from+Human+Immunodeficiency+Virus+HIV-1&doi=10.1038%2F337615a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Three-dimensional structure of aspartyl protease from human immunodeficiency virus HIV-1</span></div><div class="casAuthors">Navia, Manuel A.; Fitzgerald, Paula M. D.; McKeever, Brian M.; Leu, Chih Tai; Heimbach, Jill C.; Herber, Wayne K.; Sigal, Irving S.; Darke, Paul L.; Springer, James P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">6208</span>),
    <span class="NLM_cas:pages">615-20</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The crystal structure of the protease of the human immunodeficiency virus type 1 (HIV-1), which releases structural proteins and enzymes from viral polyprotein products, has been detd. to 3 Å resoln.  Large regions of the protease dimer, including the active site, have structural homol. to the family of microbial aspartyl proteases.  The structure suggests a mechanism for the autoproteolytic release of protease and a role in the control of virus maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrORd1OnAtyRrVg90H21EOLACvtfcHk0lj2w21r2zGtaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhs12hsLs%253D&md5=7cfe3d1c0f65a7b0c6280ac860c68781</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2F337615a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F337615a0%26sid%3Dliteratum%253Aachs%26aulast%3DNavia%26aufirst%3DM.%2BA.%26aulast%3DFitzgerald%26aufirst%3DP.%2BM.%2BD.%26aulast%3DMcKeever%26aufirst%3DB.%2BM.%26aulast%3DLeu%26aufirst%3DC.-T.%26aulast%3DHeimbach%26aufirst%3DJ.%2BC.%26aulast%3DHerber%26aufirst%3DW.%2BK.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DDarke%26aufirst%3DP.%2BL.%26aulast%3DSpringer%26aufirst%3DJ.%2BP.%26atitle%3DThree-dimensional%2520Structure%2520of%2520Aspartyl%2520Protease%2520from%2520Human%2520Immunodeficiency%2520Virus%2520HIV-1%26jtitle%3DNature%26date%3D1989%26volume%3D337%26spage%3D615%26epage%3D620%26doi%3D10.1038%2F337615a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swindells, M. B.</span></span> <span> </span><span class="NLM_article-title">LigPlot+: Multiple Ligand-protein Interaction Diagrams for Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2778</span>– <span class="NLM_lpage">2786</span>, <span class="refDoi"> DOI: 10.1021/ci200227u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200227u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2778-2786&author=R.+A.+Laskowskiauthor=M.+B.+Swindells&title=LigPlot%2B%3A+Multiple+Ligand-protein+Interaction+Diagrams+for+Drug+Discovery&doi=10.1021%2Fci200227u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">LigPlot+: Multiple Ligand-Protein Interaction Diagrams for Drug Discovery</span></div><div class="casAuthors">Laskowski, Roman A.; Swindells, Mark B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2778-2786</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe a graphical system for automatically generating multiple 2D diagrams of ligand-protein interactions from 3D coordinates.  The diagrams portray the hydrogen-bond interaction patterns and hydrophobic contacts between the ligand(s) and the main-chain or side-chain elements of the protein.  The system is able to plot, in the same orientation, related sets of ligand-protein interactions.  This facilitates popular research tasks, such as analyzing a series of small mols. binding to the same protein target, a single ligand binding to homologous proteins, or the completely general case where both protein and ligand change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYuWvYH7Nel7Vg90H21EOLACvtfcHk0lj2w21r2zGtaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1KjtrzP&md5=576e59ad4f0c4304d01c6c80301ecf39</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci200227u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200227u%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DSwindells%26aufirst%3DM.%2BB.%26atitle%3DLigPlot%252B%253A%2520Multiple%2520Ligand-protein%2520Interaction%2520Diagrams%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2778%26epage%3D2786%26doi%3D10.1021%2Fci200227u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, A. J.</span>; <span class="NLM_string-name">Rawlings, N. D.</span>; <span class="NLM_string-name">Woessner, J. F.</span></span> <i>Handbook of Proteolytic Enzymes</i>; <span class="NLM_publisher-name">Elsevier Academic Press</span>: <span class="NLM_publisher-loc">Amsterdam</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=A.+J.+Barrett&author=N.+D.+Rawlings&author=J.+F.+Woessner&title=Handbook+of+Proteolytic+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DA.%2BJ.%26btitle%3DHandbook%2520of%2520Proteolytic%2520Enzymes%26pub%3DElsevier%2520Academic%2520Press%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kisanuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span> <span> </span><span class="NLM_article-title">KNI-102, a Novel Tripeptide HIV Protease Inhibitor Containing Allophenylnorstatine as a Transition-state Mimic</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">3088</span>– <span class="NLM_lpage">3090</span>, <span class="refDoi"> DOI: 10.1248/cpb.39.3088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1248%2Fcpb.39.3088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1799953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK38XhsVegsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1991&pages=3088-3090&author=T.+Mimotoauthor=J.+Imaiauthor=S.+Tanakaauthor=N.+Hattoriauthor=S.+Kisanukiauthor=K.+Akajiauthor=Y.+Kiso&title=KNI-102%2C+a+Novel+Tripeptide+HIV+Protease+Inhibitor+Containing+Allophenylnorstatine+as+a+Transition-state+Mimic&doi=10.1248%2Fcpb.39.3088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">KNI-102, a novel tripeptide HIV protease inhibitor containing allophenylnorstatine as a transition-state mimic</span></div><div class="casAuthors">Mimoto, Tsutomu; Imai, Junya; Tanaka, Shigeki; Hattori, Naoko; Kisanuki, Sumitsugu; Akaji, Kenichi; Kiso, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3088-90</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    </div><div class="casAbstract">HIV-1 protease inhibitors contg. allophenylnorstatine[Apns; (2S,3S)-3-amino-2-hydroxy-4-phenylbutyric acid]-Pro (syn diastereomer) as a transition-state mimic were established to be potent and highly selective.  Z-Asn-Apns-Pro-NHBut (KNI-102) is the only tripeptide exhibiting substantial anti-HIV activity and may be of min. size for potent, selective inhibition of HIV protease.  Ready availability due to its simple chem. structure and stability should make it valuable for studies of the development of metabolically stable anti-AIDS drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3IUEjcsc-gLVg90H21EOLACvtfcHk0licnjfGtrWBsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhsVegsbw%253D&md5=3e32a98a7b2bfe77f33897b23349f1df</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1248%2Fcpb.39.3088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.39.3088%26sid%3Dliteratum%253Aachs%26aulast%3DMimoto%26aufirst%3DT.%26aulast%3DImai%26aufirst%3DJ.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DHattori%26aufirst%3DN.%26aulast%3DKisanuki%26aufirst%3DS.%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DKiso%26aufirst%3DY.%26atitle%3DKNI-102%252C%2520a%2520Novel%2520Tripeptide%2520HIV%2520Protease%2520Inhibitor%2520Containing%2520Allophenylnorstatine%2520as%2520a%2520Transition-state%2520Mimic%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1991%26volume%3D39%26spage%3D3088%26epage%3D3090%26doi%3D10.1248%2Fcpb.39.3088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hradilek, M.</span>; <span class="NLM_string-name">Rinnová, M.</span>; <span class="NLM_string-name">Bařinka, C.</span>; <span class="NLM_string-name">Souček, M.</span>; <span class="NLM_string-name">Konvalinka, J.</span></span> <span> </span><span class="NLM_article-title">Analysis of Substrate Specificity of HIV Protease Species</span>. In  <i>Peptides for the New Millennium: Proceedings of the 16th American Peptide Symposium June 26–July 1, 1999, Minneapolis, Minnesota, U.S.A.</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, G. B.</span>, <span class="NLM_string-name">Tam, J. P.</span>, <span class="NLM_string-name">Barany, G.</span></span>, Eds.; <span class="NLM_publisher-name">Springer Netherlands</span>: <span class="NLM_publisher-loc">Dordrecht</span>, <span class="NLM_year">2002</span>; pp  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">475</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&pages=474-475&author=M.+Hradilek&author=M.+Rinnov%C3%A1&author=C.+Ba%C5%99inka&author=M.+Sou%C4%8Dek&author=J.+Konvalinkaauthor=G.+B.+Fields&author=J.+P.+Tam&author=G.+Barany&title=Analysis+of+Substrate+Specificity+of+HIV+Protease+Species"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DHradilek%26aufirst%3DM.%26atitle%3DAnalysis%2520of%2520Substrate%2520Specificity%2520of%2520HIV%2520Protease%2520Species%26jtitle%3DPeptides%2520for%2520the%2520New%2520Millennium%253A%2520Proceedings%2520of%2520the%252016th%2520American%2520Peptide%2520Symposium%2520June%252026%25E2%2580%2593July%25201%252C%25201999%252C%2520Minneapolis%252C%2520Minnesota%252C%2520U.S.A.%26aulast%3DFields%26aufirst%3DG.%2BB.%26pub%3DSpringer%2520Netherlands%26date%3D2002%26spage%3D474%26epage%3D475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dreyer, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaszek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandler, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakhoury, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magaard, V. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickler, J. E.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Human Immunodeficiency Virus 1 Protease In Vitro: Rational Design of Substrate Analogue Inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">9752</span>– <span class="NLM_lpage">9756</span>, <span class="refDoi"> DOI: 10.1073/pnas.86.24.9752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1073%2Fpnas.86.24.9752" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=2690072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3cXktVSitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=9752-9756&author=G.+B.+Dreyerauthor=B.+W.+Metcalfauthor=T.+A.+Tomaszekauthor=T.+J.+Carrauthor=A.+C.+Chandlerauthor=L.+Hylandauthor=S.+A.+Fakhouryauthor=V.+W.+Magaardauthor=M.+L.+Mooreauthor=J.+E.+Strickler&title=Inhibition+of+Human+Immunodeficiency+Virus+1+Protease+In+Vitro%3A+Rational+Design+of+Substrate+Analogue+Inhibitors&doi=10.1073%2Fpnas.86.24.9752"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of human immunodeficiency virus 1 protease in vitro:  rational design of substrate analog inhibitors</span></div><div class="casAuthors">Dreyer, Geoffrey B.; Metcalf, Brian W.; Tomaszek, Thaddeus A., Jr.; Carr, Thomas J.; Chandler, Arthur C., III; Hyland, Lawrence; Fakhoury, Stephen A.; Magaard, Victoria W.; Moore, Michael L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9752-6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Inhibitors of the protease from human immunodeficiency virus 1 (HIV-1) were designed, synthesized, and kinetically characterized.  Analogs of a heptapeptide substrate of HIV-1 protease with sequence similar to the p17-p24 cleavage site in the natural substrate, Pr55gag, were synthesized in which the scissile dipeptide bond was replaced with bonds from 6 categories of stable mimics of an aspartic proteolysis transition state of intermediate.  These mimics included an analog of statine, hydroxyethylene isosteres, 2 categories of phosphinic acids, a reduced amide isostere, and an α,α-difluoroketone.  The resulting peptide analogs were linear-competitive inhibitors of purified recombinant HIV-1 protease with inhibition consts. ranging from 18 nM to 40 μM depending on the type of inhibitor.  A truncated inhibitor, an analog of a hexapeptide, retained full inhibitory potency.  The most potent inhibitors, contg. the hydroxyethylene isostere, effectively blocked the proteolytic processing of a recombinant form of Pr55gag by HIV-1 protease in a cell-free assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrini6UHzRXwLVg90H21EOLACvtfcHk0lgo5y0LLMGXFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktVSitrw%253D&md5=0e40a620c040dd6b2a322bb077fdc13c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1073%2Fpnas.86.24.9752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.86.24.9752%26sid%3Dliteratum%253Aachs%26aulast%3DDreyer%26aufirst%3DG.%2BB.%26aulast%3DMetcalf%26aufirst%3DB.%2BW.%26aulast%3DTomaszek%26aufirst%3DT.%2BA.%26aulast%3DCarr%26aufirst%3DT.%2BJ.%26aulast%3DChandler%26aufirst%3DA.%2BC.%26aulast%3DHyland%26aufirst%3DL.%26aulast%3DFakhoury%26aufirst%3DS.%2BA.%26aulast%3DMagaard%26aufirst%3DV.%2BW.%26aulast%3DMoore%26aufirst%3DM.%2BL.%26aulast%3DStrickler%26aufirst%3DJ.%2BE.%26atitle%3DInhibition%2520of%2520Human%2520Immunodeficiency%2520Virus%25201%2520Protease%2520In%2520Vitro%253A%2520Rational%2520Design%2520of%2520Substrate%2520Analogue%2520Inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1989%26volume%3D86%26spage%3D9752%26epage%3D9756%26doi%3D10.1073%2Fpnas.86.24.9752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stehliková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregorová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souček, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreánsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fábrys, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Štrop, P.</span></span> <span> </span><span class="NLM_article-title">Reduced-bond Tight-binding Inhibitors of HIV-1 Protease Fine Tuning of the Enzyme Subsite Specificity</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/0014-5793(92)80010-e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2F0014-5793%2892%2980010-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1544426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK38XktVShtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=1992&pages=9-13&author=J.+Urbanauthor=J.+Konvalinkaauthor=J.+Stehlikov%C3%A1author=E.+Gregorov%C3%A1author=P.+Majerauthor=M.+Sou%C4%8Dekauthor=M.+Andre%C3%A1nskyauthor=M.+F%C3%A1brysauthor=P.+%C5%A0trop&title=Reduced-bond+Tight-binding+Inhibitors+of+HIV-1+Protease+Fine+Tuning+of+the+Enzyme+Subsite+Specificity&doi=10.1016%2F0014-5793%2892%2980010-e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Reduced-bond tight-binding inhibitors of HIV-1 protease.  Fine tuning of the enzyme subsite specificity</span></div><div class="casAuthors">Urban, Jan; Konvalinka, Jan; Stehlikova, Jana; Gregorova, Eva; Majer, Pavel; Soucek, Milan; Andreansky, Martin; Fabry, Milan; Strop, Petr</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-13</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    </div><div class="casAbstract">Truncation of a peptide substrate in the N-terminus and replacement of its scissile amide bone with a non-cleavable reduced bond results in a potent inhibitor of HIV-1 protease.  A series of such inhibitors has been synthesized, and S2-S3' subsites of the protease binding cleft mapped.  The S2 pocket requires bulky Boc or PIV groups, large arom. Phe residues are preferred in P1 and P1' and Glu in P2'.  The S3' pocket prefers Phe over small Ala or Val.  Introduction of a Glu residue into the P2' position yields a tight-binding inhibitor of HIV-1 protease, Boc-Phe-[CH2-NH]-Phe-Glu-Phe-OMe, with a subnanomolar inhibition const.  The relevant peptide derived from the same amino acid sequence binds to the protease with a Ki of 110 nM, thus still demonstrating a good fit of the amino acid residues into the protease binding pockets and also the importance of the flexibility of P1-P1' linkage for proper binding.  A new type of peptide bond mimetic, N-hydroxylamine-CH2-N(OH)-, has been synthesized.  Binding of hydroxylamino inhibitor of HIV-1 protease is further improved with respect to reduced-bond inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqh234jOpbX7Vg90H21EOLACvtfcHk0lh9Arv2dAwiEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XktVShtbc%253D&md5=06567fdfb5a1e4de897f41bc1f98a275</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2F0014-5793%2892%2980010-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-5793%252892%252980010-E%26sid%3Dliteratum%253Aachs%26aulast%3DUrban%26aufirst%3DJ.%26aulast%3DKonvalinka%26aufirst%3DJ.%26aulast%3DStehlikov%25C3%25A1%26aufirst%3DJ.%26aulast%3DGregorov%25C3%25A1%26aufirst%3DE.%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DSou%25C4%258Dek%26aufirst%3DM.%26aulast%3DAndre%25C3%25A1nsky%26aufirst%3DM.%26aulast%3DF%25C3%25A1brys%26aufirst%3DM.%26aulast%3D%25C5%25A0trop%26aufirst%3DP.%26atitle%3DReduced-bond%2520Tight-binding%2520Inhibitors%2520of%2520HIV-1%2520Protease%2520Fine%2520Tuning%2520of%2520the%2520Enzyme%2520Subsite%2520Specificity%26jtitle%3DFEBS%2520Lett.%26date%3D1992%26volume%3D298%26spage%3D9%26epage%3D13%26doi%3D10.1016%2F0014-5793%2892%2980010-e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rich, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J. V. N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houseman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malkovsky, M.</span></span> <span> </span><span class="NLM_article-title">New Hydroxyethylamine HIV Protease Inhibitors that Suppress Viral Replication</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">3803</span>– <span class="NLM_lpage">3812</span>, <span class="refDoi"> DOI: 10.1021/jm00099a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00099a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK38XlvFags7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1992&pages=3803-3812&author=D.+H.+Richauthor=J.+V.+N.+V.+Prasadauthor=C.+Q.+Sunauthor=J.+Greenauthor=R.+Muellerauthor=K.+Housemanauthor=D.+MacKenzieauthor=M.+Malkovsky&title=New+Hydroxyethylamine+HIV+Protease+Inhibitors+that+Suppress+Viral+Replication&doi=10.1021%2Fjm00099a008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">New hydroxyethylamine HIV protease inhibitors that suppress viral replication</span></div><div class="casAuthors">Rich, Daniel H.; Prasad, J. V. N. Vara; Sun, Chong Qing; Green, Jeremy; Mueller, Richard; Houseman, Kathryn; MacKenzie, Debra; Malkovsky, Miroslav</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3803-12</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The synthesis of analogs of Ac-Ser-Leu-Asn-[Phe-HEA-Pro]-Ile-Val-OMe (JG-365; Phe-HEA-Pro = hydroxyethylamine transition state analog I), a tight-binding inhibitor of human immunodeficiency virus protease (HIVP), are reported.  Systematic modification of the P3 and P3' regions of the inhibitors has led to smaller HIVP inhibitors that inhibit viral replication in HIV-infected and simian immunodeficiency virus (SIV)-infected cell cultures.  Six aliph. and arom. derivs. were prepd. by replacing residues in the P3 regions of Boc-Leu-Asn-[Phe-HEA-Pro]-Ile-Val-OMe (Boc = Me3CO2C).  Arom. side chains at P3 gave better inhibitors than aliph. side chains.  The better inhibitors in this series contained a β-naphthylalanine or a biphenyl unit at P3.  A second series of HIVP inhibitors were obtained by converting the P3 group into acyl groups.  R-Asn-[Phe-HEA-Pro]-Ile-Phe-OMe (R = PhCH2O2C, 2-quinolinylcarbonyl) are potent HIVP inhibitors with Ki values equal to 1.0 and 0.1 nM, resp.  The inhibition consts. were det. by using the continuous fluorometric assay developed by M. V. Toth and G. R. Marshall (1990).  The activities of the protease inhibitors for inhibition of SIV replication were detd. in vitro using CEMx174 cells.  Inhibition of HIV infection was detd. essentially as reported by R. Pauwels, et. al. (1988).  The anti-HIV assay was carried out in culture using CEM cells (a CD4+ lymphocyte line) infected with virus strain HTLV-IIIb with a multiplicity of infection of 0.1.  Several analogs inhibited the cytopathic effect at concns. of 0.1-0.8 μg/mL.  These results establish that good inhibitors of HIV protease that inhibit viral replication in infected lymphocytes in in vitro cell assays can be obtained from JG-365 when the Ac-Ser-Leu unit is replaced by arom. acyl derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFgN6BYEYXb7Vg90H21EOLACvtfcHk0lh9Arv2dAwiEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlvFags7Y%253D&md5=4fd3643c608b8ebdff13563757868d23</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm00099a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00099a008%26sid%3Dliteratum%253Aachs%26aulast%3DRich%26aufirst%3DD.%2BH.%26aulast%3DPrasad%26aufirst%3DJ.%2BV.%2BN.%2BV.%26aulast%3DSun%26aufirst%3DC.%2BQ.%26aulast%3DGreen%26aufirst%3DJ.%26aulast%3DMueller%26aufirst%3DR.%26aulast%3DHouseman%26aufirst%3DK.%26aulast%3DMacKenzie%26aufirst%3DD.%26aulast%3DMalkovsky%26aufirst%3DM.%26atitle%3DNew%2520Hydroxyethylamine%2520HIV%2520Protease%2520Inhibitors%2520that%2520Suppress%2520Viral%2520Replication%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1992%26volume%3D35%26spage%3D3803%26epage%3D3812%26doi%3D10.1021%2Fjm00099a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rinnová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hradilek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souček, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimkait, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span> <span> </span><span class="NLM_article-title">A Picomolar Inhibitor of Resistant Strains of Human Immunodeficiency Virus Protease Identified by a Combinatorial Approach</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>382</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1006/abbi.2000.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1006%2Fabbi.2000.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=11051093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD3cXms1eis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2000&pages=22-30&author=M.+Rinnov%C3%A1author=M.+Hradilekauthor=C.+Ba%C5%99inkaauthor=J.+Weberauthor=M.+Sou%C4%8Dekauthor=J.+Vondr%C3%A1%C5%A1ekauthor=T.+Klimkaitauthor=J.+Konvalinka&title=A+Picomolar+Inhibitor+of+Resistant+Strains+of+Human+Immunodeficiency+Virus+Protease+Identified+by+a+Combinatorial+Approach&doi=10.1006%2Fabbi.2000.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Picomolar Inhibitor of Resistant Strains of Human Immunodeficiency Virus Protease Identified by a Combinatorial Approach</span></div><div class="casAuthors">Rinnova, Marketa; Hradilek, Martin; Barinka, Cyril; Weber, Jan; Soucek, Milan; Vondrasek, Jiri; Klimkait, Thomas; Konvalinka, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-30</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">To identify inhibitors of various drug-resistant forms of the human immunodeficiency virus protease (HIV PR), we have designed and synthesized pseudopeptide libraries with a general structure Z-mimetic-Aa1-Aa2-NH2.  Five different chemistries for peptide bond replacement have been employed and the resulting five individual sublibraries tested with the HIV PR and its drug-resistant mutants.  Each mutant contains amino acid substitutions that have previously been shown to be assocd. with resistance to protease inhibitors, including Ritonavir, Indinavir, and Saquinavir.  We have mapped the subsite preferences of resistant HIV PR species with the aim of selecting a pluripotent pharmaceutical lead.  All of the enzyme species in this study manifest clear preference for an l-Glu residue in the P2' position.  Slight, but significant, differences in P3' subsite specificity among individual resistant PR species have been documented.  We have identified three compds., combining the most favorable features of the inhibitor array, that exhibit low-nanomolar or picomolar Ki values for all three mutant PR species tested.  (c) 2000 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqswzXAL7fJMLVg90H21EOLACvtfcHk0ljB3O-sMA7DPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXms1eis7Y%253D&md5=5f9e73d0fc7a0dcba305b75f770966ff</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1006%2Fabbi.2000.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.2000.2017%26sid%3Dliteratum%253Aachs%26aulast%3DRinnov%25C3%25A1%26aufirst%3DM.%26aulast%3DHradilek%26aufirst%3DM.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DSou%25C4%258Dek%26aufirst%3DM.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26aulast%3DKlimkait%26aufirst%3DT.%26aulast%3DKonvalinka%26aufirst%3DJ.%26atitle%3DA%2520Picomolar%2520Inhibitor%2520of%2520Resistant%2520Strains%2520of%2520Human%2520Immunodeficiency%2520Virus%2520Protease%2520Identified%2520by%2520a%2520Combinatorial%2520Approach%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2000%26volume%3D382%26spage%3D22%26epage%3D30%26doi%3D10.1006%2Fabbi.2000.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hradilek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souček, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Library of HIV Proteases Inhibitors</span>. <i>Collect. Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1135/css199903079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1135%2Fcss199903079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVGls7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1999&pages=79-81&author=M.+Hradilekauthor=M.+Rinnovaauthor=C.+Barinkaauthor=M.+Sou%C4%8Dekauthor=J.+Konvalinka&title=Synthesis+of+Library+of+HIV+Proteases+Inhibitors&doi=10.1135%2Fcss199903079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of library of HIV proteases inhibitors</span></div><div class="casAuthors">Hradilek, Martin; Rinnova, Marketa; Barinka, Cyril; Soucek, Milan; Konvalinka, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Collection Symposium Series</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">Biologically Active Peptides</span>),
    <span class="NLM_cas:pages">79-81</span>CODEN:
                <span class="NLM_cas:coden">CSYSFN</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Institute of Organic Chemistry and Biochemistry, Academy of Sciences of the Czech Republic</span>)
        </div><div class="casAbstract">A symposium report.  We prepd. 288 mixts. of pseudopeptides, each contg. 48 chem. individuals.  Six nonhydrolyzable peptidomimetic groups were combined with 48 different α-amino acids by solid phase method, using Fmoc/tert-Bu protection.  Inhibitory activity of the pseudopeptides was tested against natural HIV protease and its mutants resistant to clin. employed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooQhJFgV3TWbVg90H21EOLACvtfcHk0ljB3O-sMA7DPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVGls7Y%253D&md5=15bbe3134d8e5f710cf367e9ce0b48f9</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1135%2Fcss199903079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1135%252Fcss199903079%26sid%3Dliteratum%253Aachs%26aulast%3DHradilek%26aufirst%3DM.%26aulast%3DRinnova%26aufirst%3DM.%26aulast%3DBarinka%26aufirst%3DC.%26aulast%3DSou%25C4%258Dek%26aufirst%3DM.%26aulast%3DKonvalinka%26aufirst%3DJ.%26atitle%3DSynthesis%2520of%2520Library%2520of%2520HIV%2520Proteases%2520Inhibitors%26jtitle%3DCollect.%2520Symp.%2520Ser.%26date%3D1999%26volume%3D3%26spage%3D79%26epage%3D81%26doi%3D10.1135%2Fcss199903079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foundling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmings, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szelke, M.</span></span> <span> </span><span class="NLM_article-title">The Structure of a Synthetic Pepsin Inhibitor Complexed with Endothiapepsin</span>. <i>Eur. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1111/j.1432-1033.1987.tb13600.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1111%2Fj.1432-1033.1987.tb13600.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=3119339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL2sXmtlajtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=1987&pages=215-221&author=J.+Cooperauthor=S.+Foundlingauthor=A.+Hemmingsauthor=T.+Blundellauthor=D.+M.+Jonesauthor=A.+Hallettauthor=M.+Szelke&title=The+Structure+of+a+Synthetic+Pepsin+Inhibitor+Complexed+with+Endothiapepsin&doi=10.1111%2Fj.1432-1033.1987.tb13600.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">The structure of a synthetic pepsin inhibitor complexed with endothiapepsin</span></div><div class="casAuthors">Cooper, Jon; Foundling, Steve; Hemmings, Andrew; Blundell, Tom; Jones, D. Michael; Hallett, Allan; Szelke, Michael</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">215-21</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    </div><div class="casAbstract">The conformation of a synthetic polypeptide inhibitor bound to the active site of the fungal aspartic proteinase endothiapepsin (EC 3.4.23.6) was detd. by x-ray diffraction at 0.20-nm resoln. and refined to an agreement factor of 0.20.  The inhibitor, Pro-Thr-Glu-Phe-R-Phe-Arg-Glu (R = -CH2NH-), is based on a chromogenic substrate of pepsin (EC 3.4.23.1).  It has, in place of the scissile bond, a reduced peptide group which is resistant to hydrolysis and mimics the tetrahedral transition state.  The inhibitor binds in an extended conformation with the reduced bond close to the essential aspartate side chains of the enzyme.  The H bonds and hydrophobic interactions between the enzyme and the inhibitor do not induce large conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCDX_hOQ0ne7Vg90H21EOLACvtfcHk0lhr8YypPrbITw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXmtlajtb4%253D&md5=b02c449599a73fee98bb412a1e71c7c0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1111%2Fj.1432-1033.1987.tb13600.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1432-1033.1987.tb13600.x%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DJ.%26aulast%3DFoundling%26aufirst%3DS.%26aulast%3DHemmings%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DD.%2BM.%26aulast%3DHallett%26aufirst%3DA.%26aulast%3DSzelke%26aufirst%3DM.%26atitle%3DThe%2520Structure%2520of%2520a%2520Synthetic%2520Pepsin%2520Inhibitor%2520Complexed%2520with%2520Endothiapepsin%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1987%26volume%3D169%26spage%3D215%26epage%3D221%26doi%3D10.1111%2Fj.1432-1033.1987.tb13600.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quail, J. W.</span></span> <span> </span><span class="NLM_article-title">Structure of theRhizomucor mieheiaspartic proteinase complexed with the inhibitor pepstatin A at 2.7 Å resolution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">625</span>– <span class="NLM_lpage">630</span>, <span class="refDoi"> DOI: 10.1107/s0907444998013961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0907444998013961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10089458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK1MXivVeru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1999&pages=625-630&author=J.+Yangauthor=J.+W.+Quail&title=Structure+of+theRhizomucor+mieheiaspartic+proteinase+complexed+with+the+inhibitor+pepstatin+A+at+2.7+%C3%85+resolution&doi=10.1107%2Fs0907444998013961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Rhizomucor miehei aspartic proteinase complexed with the inhibitor pepstatin A at 2.7 Å resolution</span></div><div class="casAuthors">Yang, Jian; Quail, J. Wilson</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">D55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">625-630</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">Crystals of Rhizomucor miehei aspartic proteinase (mucorpepsin) (I) complexed with pepstatin A (II) grew in orthorhombic space group P212121 and were isomorphous to native I crystals.  The unit-cell dimensions were a = 41.52, b = 50.82, and c = 172.71 Å.  There was 1 I-II complex per asym. unit.  The structure of the I-II complex was refined to a crystallog. R value of 19.3% and an Rfree value of 28.0% at 2.7 Å resoln.  A II mol. fit into the large substrate-binding cleft between the 2 domains of I in an extended conformation up to the Ala residue at the P2' position.  The dipeptide analog statine residue at the P3'-P4' position formed an inverse γ-turn (P3'-P1') with the statine residue at the P1-P1' position and its Leu side-chain bound back into the S1' subsite.  II interacted with the residues of the substrate-binding pocket by both H-bonds and hydrophobic interactions.  The OH group of the statine residue at the P1-P1' position formed H-bonds with both catalytic Asp residues (Asp-38 and Asp-237).  This conformation mimicked the expected transition state of the enzyme-substrate interaction.  The binding of II to the enzyme did not produce large distortions of the active site.  No domain movement was obsd. compared with the native enzyme structure.  However, the surface-flap region (residues 82-88) underwent a conformational change, moving toward the inhibitor and becoming rigid owing to the formation of H-bonds with the inhibitor.  B-factor calcns. of the 2 domains suggested that the C-terminal domain becomes more rigid in the complex than in the native structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXdoEjdtcNJ7Vg90H21EOLACvtfcHk0lhr8YypPrbITw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivVeru7g%253D&md5=ed3f3c209942d09ba1b65793d0e57e5b</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444998013961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998013961%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DQuail%26aufirst%3DJ.%2BW.%26atitle%3DStructure%2520of%2520theRhizomucor%2520mieheiaspartic%2520proteinase%2520complexed%2520with%2520the%2520inhibitor%2520pepstatin%2520A%2520at%25202.7%2520%25C3%2585%2520resolution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1999%26volume%3D55%26spage%3D625%26epage%3D630%26doi%3D10.1107%2Fs0907444998013961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruge, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelsang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monod, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korting, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maskos, K.</span></span> <span> </span><span class="NLM_article-title">X-ray structures of Sap1 and Sap5: Structural Comparison of the Secreted Aspartic Proteinases from Candida Albicans</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1308</span>– <span class="NLM_lpage">1319</span>, <span class="refDoi"> DOI: 10.1002/prot.22021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1002%2Fprot.22021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=18384081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2008&pages=1308-1319&author=C.+Borelliauthor=E.+Rugeauthor=J.+H.+Leeauthor=M.+Schallerauthor=A.+Vogelsangauthor=M.+Monodauthor=H.+C.+Kortingauthor=R.+Huberauthor=K.+Maskos&title=X-ray+structures+of+Sap1+and+Sap5%3A+Structural+Comparison+of+the+Secreted+Aspartic+Proteinases+from+Candida+Albicans&doi=10.1002%2Fprot.22021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray structures of Sap1 and Sap5: structural comparison of the secreted aspartic proteinases from Candida albicans</span></div><div class="casAuthors">Borelli, Claudia; Ruge, Elisabeth; Lee, Jung Hoon; Schaller, Martin; Vogelsang, Alexandra; Monod, Michel; Korting, Hans Christian; Huber, Robert; Maskos, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1308-1319</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Proteolytic activity is an important virulence factor for Candida albicans.  It is attributed to the family of the secreted aspartic proteinases (Saps) from C. albicans with a min. of 10 members.  Saps show controlled expression and regulation for the individual stages of the infection process.  Distinct isoenzymes can be responsible for adherence and tissue damage of local infections, while others cause systemic diseases.  Earlier, only the structures of Sap2 and Sap3 were known.  In our research, we have now succeeded in solving the X-ray crystal structures of the apoenzyme of Sap1 and Sap5 in complex with pepstatin A (pep A) at 2.05 and 2.5 Å resoln., resp.  With the structure of Sap1, we have completed the set of structures of isoenzyme subgroup Sap1-3.  Of subgroup Sap4-6, the structure of the enzyme Sap5 is the first structure that has been described up to now.  This facilitates comparison of structural details as well as inhibitor binding modes among the different subgroup members.  Structural anal. reveals a highly conserved overall secondary structure of Sap1-3 and Sap5.  However, Sap5 clearly differs from Sap1-3 by its electrostatic overall charge as well as through structural conformation of its entrance to the active site cleft.  Design of inhibitors specific for Sap5 should conc. on the S4 and S3 pockets, which significantly differ from Sap1-3 in size and electrostatic charge.  Both Sap1 and Sap5 seem to play a major part in superficial Candida infections.  Detn. of the isoenzymes' structures can contribute to the development of new Sap-specific inhibitors for the treatment of superficial infections with a structure-based drug design program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPr1xjkXUPprVg90H21EOLACvtfcHk0lhr8YypPrbITw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCrtbzP&md5=512e20e202e3aea0c095ead9fe069115</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fprot.22021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22021%26sid%3Dliteratum%253Aachs%26aulast%3DBorelli%26aufirst%3DC.%26aulast%3DRuge%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DSchaller%26aufirst%3DM.%26aulast%3DVogelsang%26aufirst%3DA.%26aulast%3DMonod%26aufirst%3DM.%26aulast%3DKorting%26aufirst%3DH.%2BC.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DMaskos%26aufirst%3DK.%26atitle%3DX-ray%2520structures%2520of%2520Sap1%2520and%2520Sap5%253A%2520Structural%2520Comparison%2520of%2520the%2520Secreted%2520Aspartic%2520Proteinases%2520from%2520Candida%2520Albicans%26jtitle%3DProteins%26date%3D2008%26volume%3D72%26spage%3D1308%26epage%3D1319%26doi%3D10.1002%2Fprot.22021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeever, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanMiddlesworth, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimbach, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leu, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herber, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darke, P. L.</span></span> <span> </span><span class="NLM_article-title">Crystallographic Analysis of a Complex Between Human Immunodeficiency Virus Type 1 Protease and Acetyl-pepstatin at 2.0-A Resolution</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>265</i></span>,  <span class="NLM_fpage">14209</span>– <span class="NLM_lpage">14219</span>, <span class="refDoi"> DOI: 10.1016/s0021-9258(18)77288-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2FS0021-9258%2818%2977288-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=2201682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3cXls1Oqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=14209-14219&author=P.+M.+Fitzgeraldauthor=B.+M.+McKeeverauthor=J.+F.+VanMiddlesworthauthor=J.+P.+Springerauthor=J.+C.+Heimbachauthor=C.+T.+Leuauthor=W.+K.+Herberauthor=R.+A.+Dixonauthor=P.+L.+Darke&title=Crystallographic+Analysis+of+a+Complex+Between+Human+Immunodeficiency+Virus+Type+1+Protease+and+Acetyl-pepstatin+at+2.0-A+Resolution&doi=10.1016%2Fs0021-9258%2818%2977288-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic analysis of a complex between human immunodeficiency virus type 1 protease and acetyl-pepstatin at 2.0-Å resolution</span></div><div class="casAuthors">Fitzgerald, Paula M. D.; McKeever, Brian M.; VanMiddlesworth, Jody F.; Springer, James P.; Heimbach, Jill C.; Leu, Chih Tai; Herber, Wayne K.; Dixon, Richard A. F.; Darke, Paul L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">14209-19</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">The mode of binding of acetyl-pepstatin to the protease from the human immunodeficiency virus type 1 (HIV-1) has been detd. by x-ray diffraction anal.  Crystals of an acetyl-pepstatin-HIV-1 protease complex were obtained in space group P21212 (unit cell dimensions a = 58.39 Å, b = 86.70 Å, c = 46.27 Å) by pptn. with NaCl.  The structure was phased by mol. replacement methods, and a model for the structure was refined using diffraction data to 2.0 Å resoln. (R = 0.176 for 12,901 reflections with I > σ(I); deviation of bond distances from ideal values = 0.018 Å; 172 solvent mols. included).  The structure of the protein in the complex has been compared with the structure of the enzyme without the ligand.  A core of 44 amino acids in each monomer, including residues in the active site and residues at the dimer interface, remains unchanged on binding of the inhibitor (root mean square deviation of α-C positions = 0.39 Å).  The remaining 55 residues in each monomer undergo substantial rearrangement, with the most dramatic changes occurring at residues 44-57 (these residues comprise the so-called flaps of the enzyme).  The flaps interact with one another and with the inhibitor so as to largely preserve the 2-fold symmetry of the protein.  The inhibitor is bound in 2 approx. sym. orientations.  In both orientations the peptidyl backbone of the inhibitor is extended; a network of H bonds is formed between the inhibitor and the main body of the protein as well as between the inhibitor and the flaps.  Hydrophobic side chains of residues in the body of the protein form partial binding sites for the side chains of the inhibitor; hydrophobic side chains of residues in the flaps complete these binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRZgaPlbh_ibVg90H21EOLACvtfcHk0lha0sjfWfDjwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXls1Oqsrs%253D&md5=7f8331534e65f8bb17d4acde995dfbfa</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2FS0021-9258%2818%2977288-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0021-9258%252818%252977288-8%26sid%3Dliteratum%253Aachs%26aulast%3DFitzgerald%26aufirst%3DP.%2BM.%26aulast%3DMcKeever%26aufirst%3DB.%2BM.%26aulast%3DVanMiddlesworth%26aufirst%3DJ.%2BF.%26aulast%3DSpringer%26aufirst%3DJ.%2BP.%26aulast%3DHeimbach%26aufirst%3DJ.%2BC.%26aulast%3DLeu%26aufirst%3DC.%2BT.%26aulast%3DHerber%26aufirst%3DW.%2BK.%26aulast%3DDixon%26aufirst%3DR.%2BA.%26aulast%3DDarke%26aufirst%3DP.%2BL.%26atitle%3DCrystallographic%2520Analysis%2520of%2520a%2520Complex%2520Between%2520Human%2520Immunodeficiency%2520Virus%2520Type%25201%2520Protease%2520and%2520Acetyl-pepstatin%2520at%25202.0-A%2520Resolution%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D14209%26epage%3D14219%26doi%3D10.1016%2Fs0021-9258%2818%2977288-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poręba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drąg, M.</span></span> <span> </span><span class="NLM_article-title">Selective Substrates and Activity-based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5222</span>– <span class="NLM_lpage">5234</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVags7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5222-5234&author=W.+Rutauthor=M.+Por%C4%99baauthor=P.+Kasperkiewiczauthor=S.+J.+Snipasauthor=M.+Dr%C4%85g&title=Selective+Substrates+and+Activity-based+Probes+for+Imaging+of+the+Human+Constitutive+20S+Proteasome+in+Cells+and+Blood+Samples&doi=10.1021%2Facs.jmedchem.8b00026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Substrates and Activity-Based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span></div><div class="casAuthors">Rut, Wioletta; Poreba, Marcin; Kasperkiewicz, Paulina; Snipas, Scott J.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5222-5234</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proteasome is an enzyme complex crit. for maintaining protein homeostasis.  Perturbed proteasome function leads to pathologies including cancer and autoimmune and neurodegenerative disease.  Therefore, the proteasome constitutes an excellent mol. target for pharmaceutical development.  Here, using the HyCoSuL approach, the authors designed and synthesized novel and selective fluorogenic substrates for each of these three constitutive 20S proteasome activities and applied them to assess inhibition of proteasome subunits by MG-132 and a clin. used inhibitor bortezomib.  The authors' results confirm the utility of designed substrates in biochem. assays.  Furthermore, selective peptide sequences obtained in this manner were used to construct fluorophore-labeled activity-based probes and then used to detect each constitutive 20S proteasome subunit simultaneously in lysates of HEK-293F cells and red blood cells.  Overall, the authors describe a simple and rapid method useful to measure constitutive 20S proteasome activity in whole human blood samples that could enable early diagnosis of pathol. states assocd. with aberrantly upregulated proteasome activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrd0FSTAesirVg90H21EOLACvtfcHk0lha0sjfWfDjwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVags7vO&md5=1fb97b492a7f3f25d6659035f5bf4b14</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00026%26sid%3Dliteratum%253Aachs%26aulast%3DRut%26aufirst%3DW.%26aulast%3DPor%25C4%2599ba%26aufirst%3DM.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DDr%25C4%2585g%26aufirst%3DM.%26atitle%3DSelective%2520Substrates%2520and%2520Activity-based%2520Probes%2520for%2520Imaging%2520of%2520the%2520Human%2520Constitutive%252020S%2520Proteasome%2520in%2520Cells%2520and%2520Blood%2520Samples%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5222%26epage%3D5234%26doi%3D10.1021%2Facs.jmedchem.8b00026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunde, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihelic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Counter Selection Substrate Library Strategy for Developing Specific Protease Substrates and Probes</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1023</span>– <span class="NLM_lpage">1035</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.chembiol.2016.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=27478158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1023-1035&author=M.+Porebaauthor=R.+Solbergauthor=W.+Rutauthor=N.+N.+Lundeauthor=P.+Kasperkiewiczauthor=S.+J.+Snipasauthor=M.+Mihelicauthor=D.+Turkauthor=B.+Turkauthor=G.+S.+Salvesenauthor=M.+Drag&title=Counter+Selection+Substrate+Library+Strategy+for+Developing+Specific+Protease+Substrates+and+Probes&doi=10.1016%2Fj.chembiol.2016.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Counter Selection Substrate Library Strategy for Developing Specific Protease Substrates and Probes</span></div><div class="casAuthors">Poreba, Marcin; Solberg, Rigmor; Rut, Wioletta; Lunde, Ngoc Nguyen; Kasperkiewicz, Paulina; Snipas, Scott J.; Mihelic, Marko; Turk, Dusan; Turk, Boris; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1023-1035</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Legumain (AEP) is a lysosomal cysteine protease that was first characterized in leguminous seeds and later discovered in higher eukaryotes.  AEP upregulation is linked to a no. of diseases including inflammation, arteriosclerosis, and tumorigenesis.  Thus this protease is an excellent mol. target for the development of new chem. markers.  We deployed a hybrid combinatorial substrate library (HyCoSuL) approach to obtain P1-Asp fluorogenic substrates and biotin-labeled inhibitors that targeted legumain.  Since this approach led to probes that were also recognized by caspases, we introduced a Counter Selection Substrate Library (CoSeSuL) approach that biases the peptidic scaffold against caspases, thus delivering highly selective legumain probes.  The selectivity of these tools was validated using M38L and HEK293 cells.  We also propose that the CoSeSuL methodol. can be considered as a general principle in the design of selective probes for other protease families where selectivity is difficult to achieve by conventional sequence-based profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtgQyo_YeLk7Vg90H21EOLACvtfcHk0lhhtyT58fZokw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnN&md5=96a607ad2ac4bbcb35c3dd106aad0688</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSolberg%26aufirst%3DR.%26aulast%3DRut%26aufirst%3DW.%26aulast%3DLunde%26aufirst%3DN.%2BN.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DMihelic%26aufirst%3DM.%26aulast%3DTurk%26aufirst%3DD.%26aulast%3DTurk%26aufirst%3DB.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DCounter%2520Selection%2520Substrate%2520Library%2520Strategy%2520for%2520Developing%2520Specific%2520Protease%2520Substrates%2520and%2520Probes%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D1023%26epage%3D1035%26doi%3D10.1016%2Fj.chembiol.2016.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, B. M.</span></span> <span> </span><span class="NLM_article-title">Substrate Specificity and Inhibitors of Aspartic Proteinases</span>. <i>Scand. J. Clin. Lab. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.3109/00365519209104651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1080%2F00365519209104651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3sXjsV2ktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1992&pages=23-30&author=J.+Kayauthor=B.+M.+Dunn&title=Substrate+Specificity+and+Inhibitors+of+Aspartic+Proteinases&doi=10.3109%2F00365519209104651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate specificity and inhibitors of aspartic proteinases</span></div><div class="casAuthors">Kay, John; Dunn, B. M.</div><div class="citationInfo"><span class="NLM_cas:title">Scandinavian Journal of Clinical and Laboratory Investigation, Supplement</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">210</span>),
    <span class="NLM_cas:pages">23-30</span>CODEN:
                <span class="NLM_cas:coden">SCLSAH</span>;
        ISSN:<span class="NLM_cas:issn">0085-591X</span>.
    </div><div class="casAbstract">Aspartic proteinases of vertebrate, fungal and retroviral origin have been characterized by utilization of chromogenic peptide substrates and synthetic inhibitors as probes.  By this means, the mol. topog. of sub-sites within the active site cleft of individual enzymes has been elucidated.  With suitable selection of residues to occupy each sub-site, the design of effective inhibitors specifically targetted against individual aspartic proteinases has become feasible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojIg7wIZCs0LVg90H21EOLACvtfcHk0lhhtyT58fZokw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXjsV2ktw%253D%253D&md5=8939a05cc45ca7f786c7d9b0c3e39f8a</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1080%2F00365519209104651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00365519209104651%26sid%3Dliteratum%253Aachs%26aulast%3DKay%26aufirst%3DJ.%26aulast%3DDunn%26aufirst%3DB.%2BM.%26atitle%3DSubstrate%2520Specificity%2520and%2520Inhibitors%2520of%2520Aspartic%2520Proteinases%26jtitle%3DScand.%2520J.%2520Clin.%2520Lab.%2520Invest.%26date%3D1992%26volume%3D52%26spage%3D23%26epage%3D30%26doi%3D10.3109%2F00365519209104651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werb, Z.</span></span> <span> </span><span class="NLM_article-title">Extracellular Matrix Degradation and Remodeling in Development and Disease</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">a005058</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a005058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1101%2Fcshperspect.a005058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=21917992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktVKrt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=a005058&author=P.+Luauthor=K.+Takaiauthor=V.+M.+Weaverauthor=Z.+Werb&title=Extracellular+Matrix+Degradation+and+Remodeling+in+Development+and+Disease&doi=10.1101%2Fcshperspect.a005058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular matrix degradation and remodeling in development and disease</span></div><div class="casAuthors">Lu, Pengfei; Takai, Ken; Weaver, Valerie M.; Werb, Zena</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">a005058/1-a005058/24</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The extracellular matrix (ECM) serves diverse functions and is a major component of the cellular microenvironment.  The ECM is a highly dynamic structure, constantly undergoing a remodeling process where ECM components are deposited, degraded, or otherwise modified.  ECM dynamics are indispensable during restructuring of tissue architecture.  ECM remodeling is an important mechanism whereby cell differentiation can be regulated, including processes such as the establishment and maintenance of stem cell niches, branching morphogenesis, angiogenesis, bone remodeling, and wound repair.  In contrast, abnormal ECM dynamics lead to deregulated cell proliferation and invasion, failure of cell death, and loss of cell differentiation, resulting in congenital defects and pathol. processes including tissue fibrosis and cancer.  Understanding the mechanisms of ECM remodeling and its regulation, therefore, is essential for developing new therapeutic interventions for diseases and novel strategies for tissue engineering and regenerative medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpu5HZYPalvrVg90H21EOLACvtfcHk0lhhtyT58fZokw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktVKrt78%253D&md5=d59a5d55420420b4cbbb731286ecd780</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a005058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a005058%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DP.%26aulast%3DTakai%26aufirst%3DK.%26aulast%3DWeaver%26aufirst%3DV.%2BM.%26aulast%3DWerb%26aufirst%3DZ.%26atitle%3DExtracellular%2520Matrix%2520Degradation%2520and%2520Remodeling%2520in%2520Development%2520and%2520Disease%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2011%26volume%3D3%26spage%3Da005058%26doi%3D10.1101%2Fcshperspect.a005058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregorová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stehlíková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreánsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedlácek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strop, P.</span></span> <span> </span><span class="NLM_article-title">Specificity Mapping of HIV-1 Protease by Reduced Bond Inhibitors</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1006/abbi.1993.1314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1006%2Fabbi.1993.1314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=8323274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3sXlsFWlt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=1993&pages=1-8&author=P.+Majerauthor=J.+Urbanauthor=E.+Gregorov%C3%A1author=J.+Konvalinkaauthor=P.+Novekauthor=J.+Stehl%C3%ADkov%C3%A1author=M.+Andre%C3%A1nskyauthor=J.+Sedl%C3%A1cekauthor=P.+Strop&title=Specificity+Mapping+of+HIV-1+Protease+by+Reduced+Bond+Inhibitors&doi=10.1006%2Fabbi.1993.1314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity mapping of HIV-1 protease by reduced bond inhibitors</span></div><div class="casAuthors">Majer, Pavel; Urban, Jan; Gregorova, Eva; Konvalinka, Jan; Novek, Petr; Stehlikova, Jana; Andreansky, Martin; Sedlacek, Juraj; Strop, Petr</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">A series of 47 N-truncated reduced bond inhibitors, systematically modified at individual positions (P1, P'1, P'2, P'3, and P'4), were synthesized and used to map the subsite preferences of HIV-1 protease.  The tight binding inhibitor of HIV-1 protease t-butoxycarbonyl-Phe-[Ch2NH]Phe-Glu-Phe-NH2 (Ki = 0.2 nM) was chosen as the parent structure for further modifications.  The P'2 glutamic acid was found to fit well into the S'2 subsite of the protease.  The conformational restriction of any phenylalanine residue or satn. of more than one phenylalanine side chain in P'1 or P'3 lead is to a large Ki increase.  Introduction of tyrosine in the P1 position improves the binding by an order of magnitude.  The S'4 subsite of the protease was shown to accommodate large structural changes in the inhibitor at this position.  Therefore P'4 may serve as an ideal region for further modification in order to improve bioavailability of this type of compd.  An improved method of direct comparison of tight binding inhibitors with subnanomolar Ki values has been described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMg912DjALIrVg90H21EOLACvtfcHk0ljY8S9IMrKKOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsFWlt7Y%253D&md5=770e121ac372cb4025c68fe96c97da4d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1006%2Fabbi.1993.1314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.1993.1314%26sid%3Dliteratum%253Aachs%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DUrban%26aufirst%3DJ.%26aulast%3DGregorov%25C3%25A1%26aufirst%3DE.%26aulast%3DKonvalinka%26aufirst%3DJ.%26aulast%3DNovek%26aufirst%3DP.%26aulast%3DStehl%25C3%25ADkov%25C3%25A1%26aufirst%3DJ.%26aulast%3DAndre%25C3%25A1nsky%26aufirst%3DM.%26aulast%3DSedl%25C3%25A1cek%26aufirst%3DJ.%26aulast%3DStrop%26aufirst%3DP.%26atitle%3DSpecificity%2520Mapping%2520of%2520HIV-1%2520Protease%2520by%2520Reduced%2520Bond%2520Inhibitors%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1993%26volume%3D304%26spage%3D1%26epage%3D8%26doi%3D10.1006%2Fabbi.1993.1314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Houštecká, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadzima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanfrlík, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hánová, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertlíková-Kaiserová, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepšík, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smrčina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure–Activity Relationship and Binding Mode Analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1576</span>– <span class="NLM_lpage">1596</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01351</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01351" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVOgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=1576-1596&author=R.+Hou%C5%A1teck%C3%A1author=M.+Hadzimaauthor=J.+Fanfrl%C3%ADkauthor=J.+Bryndaauthor=L.+Pallov%C3%A1author=I.+H%C3%A1nov%C3%A1author=H.+Mertl%C3%ADkov%C3%A1-Kaiserov%C3%A1author=M.+Lep%C5%A1%C3%ADkauthor=M.+Hornauthor=M.+Smr%C4%8Dinaauthor=P.+Majerauthor=M.+Mare%C5%A1&title=Biomimetic+Macrocyclic+Inhibitors+of+Human+Cathepsin+D%3A+Structure%E2%80%93Activity+Relationship+and+Binding+Mode+Analysis&doi=10.1021%2Facs.jmedchem.9b01351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure-Activity Relationship and Binding Mode Analysis</span></div><div class="casAuthors">Houstecka, Radka; Hadzima, Martin; Fanfrlik, Jindrich; Brynda, Jiri; Pallova, Lenka; Hanova, Iva; Mertlikova-Kaiserova, Helena; Lepsik, Martin; Horn, Martin; Smrcina, Martin; Majer, Pavel; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1576-1596</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human cathepsin D (CatD), a pepsin-family aspartic protease, plays an important role in tumor progression and metastasis.  Here, we report the development of biomimetic inhibitors of CatD as novel tools for regulation of this therapeutic target.  We designed a macrocyclic scaffold to mimic the spatial conformation of the minimal pseudo-dipeptide binding motif of pepstatin A, a microbial oligopeptide inhibitor, in the CatD active site.  A library of more than 30 macrocyclic peptidomimetic inhibitors was employed for scaffold optimization, mapping of subsite interactions, and profiling of inhibitor selectivity.  Furthermore, we solved high-resoln. crystal structures of three macrocyclic inhibitors with low nanomolar or subnanomolar potency in complex with CatD and detd. their binding mode using quantum chem. calcns.  The study provides a new structural template and functional profile that can be exploited for design of potential chemotherapeutics that specifically inhibit CatD and related aspartic proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaOxpVGiZrGLVg90H21EOLACvtfcHk0ljY8S9IMrKKOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVOgsrY%253D&md5=5c5d311485db722abf1ac78b0a640273</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01351%26sid%3Dliteratum%253Aachs%26aulast%3DHou%25C5%25A1teck%25C3%25A1%26aufirst%3DR.%26aulast%3DHadzima%26aufirst%3DM.%26aulast%3DFanfrl%25C3%25ADk%26aufirst%3DJ.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DPallov%25C3%25A1%26aufirst%3DL.%26aulast%3DH%25C3%25A1nov%25C3%25A1%26aufirst%3DI.%26aulast%3DMertl%25C3%25ADkov%25C3%25A1-Kaiserov%25C3%25A1%26aufirst%3DH.%26aulast%3DLep%25C5%25A1%25C3%25ADk%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DSmr%25C4%258Dina%26aufirst%3DM.%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DBiomimetic%2520Macrocyclic%2520Inhibitors%2520of%2520Human%2520Cathepsin%2520D%253A%2520Structure%25E2%2580%2593Activity%2520Relationship%2520and%2520Binding%2520Mode%2520Analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D1576%26epage%3D1596%26doi%3D10.1021%2Facs.jmedchem.9b01351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiering, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedrani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maibaum, J.</span></span> <span> </span><span class="NLM_article-title">Direct Renin Inhibitors as a New Therapy for Hypertension</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">7490</span>– <span class="NLM_lpage">7520</span>, <span class="refDoi"> DOI: 10.1021/jm901885s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901885s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKjsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=7490-7520&author=R.+L.+Webbauthor=N.+Schieringauthor=R.+Sedraniauthor=J.+Maibaum&title=Direct+Renin+Inhibitors+as+a+New+Therapy+for+Hypertension&doi=10.1021%2Fjm901885s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Direct Renin Inhibitors as a New Therapy for Hypertension</span></div><div class="casAuthors">Webb, Randy L.; Schiering, Nikolaus; Sedrani, Richard; Maibaum, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7490-7520</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The objective of this review is to highlight key events and recent advances in the evolution of nonpeptide peptidomimetic direct renin inhibitors (DRIs).  Special emphasis will be placed on the successful structure-based strategies that enabled the discovery of several novel classes of DRIs.  The potential limitations assocd. with these inhibitors and opportunities for future directions of renin inhibitor design will be discussed from a chem. and biol. perspective.  Lessons learned will also be discussed in an effort to provide a unique perspective on the development of aliskiren.  With its introduction, it is now very important to distinguish this mechanism of action from indirect means of inhibiting renin.  In this regard, ongoing clin. trials designed to demonstrate a clear benefit, beyond an effect on blood pressure, by redn. in morbidity and mortality are of paramount importance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRBUB_cukx3LVg90H21EOLACvtfcHk0lhxHmHD4JUvwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKjsb3E&md5=54b73ed33f69d2ad788c6f5839a54a7e</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm901885s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901885s%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DR.%2BL.%26aulast%3DSchiering%26aufirst%3DN.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DMaibaum%26aufirst%3DJ.%26atitle%3DDirect%2520Renin%2520Inhibitors%2520as%2520a%2520New%2520Therapy%2520for%2520Hypertension%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D7490%26epage%3D7520%26doi%3D10.1021%2Fjm901885s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tykvart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šácha, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bařinka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knedlík, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubkowski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span> <span> </span><span class="NLM_article-title">Efficient and Versatile One-step Affinity Purification of In Vivo Biotinylated Proteins: Expression, Characterization and Structure Analysis of Recombinant Human Glutamate Carboxypeptidase II</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1016/j.pep.2011.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2Fj.pep.2011.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=22178733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2012&pages=106-115&author=J.+Tykvartauthor=P.+%C5%A0%C3%A1chaauthor=C.+Ba%C5%99inkaauthor=T.+Knedl%C3%ADkauthor=J.+Starkov%C3%A1author=J.+Lubkowskiauthor=J.+Konvalinka&title=Efficient+and+Versatile+One-step+Affinity+Purification+of+In+Vivo+Biotinylated+Proteins%3A+Expression%2C+Characterization+and+Structure+Analysis+of+Recombinant+Human+Glutamate+Carboxypeptidase+II&doi=10.1016%2Fj.pep.2011.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient and versatile one-step affinity purification of in vivo biotinylated proteins: Expression, characterization and structure analysis of recombinant human glutamate carboxypeptidase II</span></div><div class="casAuthors">Tykvart, J.; Sacha, P.; Barinka, C.; Knedlik, T.; Starkova, J.; Lubkowski, J.; Konvalinka, J.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">106-115</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Affinity purifn. is a useful approach for purifn. of recombinant proteins.  Eukaryotic expression systems have become more frequently used at the expense of prokaryotic systems since they afford recombinant eukaryotic proteins with post-translational modifications similar or identical to the native ones.  Here, a one-step affinity purifn. set-up suitable for the purifn. of secreted proteins is presented.  The set-up is based on the interaction between biotin and mutated streptavidin.  Drosophila Schneider 2 cells are chosen as the expression host, and a biotin acceptor peptide is used as an affinity tag.  This tag is biotinylated by Escherichia coli biotin-protein ligase in vivo.  It was detd. that localization of the ligase within the ER led to the most effective in vivo biotinylation of the secreted proteins.  A protocol was optimized for large-scale expression and purifn. of AviTEV-tagged recombinant human glutamate carboxypeptidase II (Avi-GCPII) with milligram yields per L of culture.  The 3D structure of Avi-GCPII was also detd. by X-ray crystallog. and the enzymic characteristics of the protein were compared to those of its non-tagged variant.  These expts. confirmed that AviTEV tag does not affect the biophys. properties of its fused partner.  The purifn. approach, developed here, provides not only a sufficient amt. of highly homogeneous protein but also specifically and effectively biotinylates a target protein and thus enables its subsequent visualization or immobilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo57Mui9FOUXbVg90H21EOLACvtfcHk0lh73qbfogiNSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWgs7c%253D&md5=d731679432bae009c380ace62a6d3893</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.pep.2011.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pep.2011.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DTykvart%26aufirst%3DJ.%26aulast%3D%25C5%25A0%25C3%25A1cha%26aufirst%3DP.%26aulast%3DBa%25C5%2599inka%26aufirst%3DC.%26aulast%3DKnedl%25C3%25ADk%26aufirst%3DT.%26aulast%3DStarkov%25C3%25A1%26aufirst%3DJ.%26aulast%3DLubkowski%26aufirst%3DJ.%26aulast%3DKonvalinka%26aufirst%3DJ.%26atitle%3DEfficient%2520and%2520Versatile%2520One-step%2520Affinity%2520Purification%2520of%2520In%2520Vivo%2520Biotinylated%2520Proteins%253A%2520Expression%252C%2520Characterization%2520and%2520Structure%2520Analysis%2520of%2520Recombinant%2520Human%2520Glutamate%2520Carboxypeptidase%2520II%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2012%26volume%3D82%26spage%3D106%26epage%3D115%26doi%3D10.1016%2Fj.pep.2011.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yung-Chi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the Inhibition Constant (KI) and the Concentration of Inhibitor which Causes 50 per cent Inhibition (I50) of an Enzymatic Reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+Inhibition+Constant+%28KI%29+and+the+Concentration+of+Inhibitor+which+Causes+50+per+cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520Inhibition%2520Constant%2520%2528KI%2529%2520and%2520the%2520Concentration%2520of%2520Inhibitor%2520which%2520Causes%252050%2520per%2520cent%2520Inhibition%2520%2528I50%2529%2520of%2520an%2520Enzymatic%2520Reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Britton, H. T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. A.</span></span> <span> </span><span class="NLM_article-title">CXCVIII.-Universal Buffer Solutions and the Dissociation Constant of Veronal</span>. <i>J. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1931</span>,  <span class="NLM_fpage">1456</span>– <span class="NLM_lpage">1462</span>, <span class="refDoi"> DOI: 10.1039/jr9310001456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1039%2FJR9310001456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaA3MXkt1aqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1931&pages=1456-1462&author=H.+T.+S.+Brittonauthor=R.+A.+Robinson&title=CXCVIII.-Universal+Buffer+Solutions+and+the+Dissociation+Constant+of+Veronal&doi=10.1039%2Fjr9310001456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Universal buffer solutions and the dissociation constant of veronal</span></div><div class="casAuthors">Britton, Hubert T. S.; Robinson, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society</span>
        (<span class="NLM_cas:date">1931</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1456-62</span>CODEN:
                <span class="NLM_cas:coden">JCSOA9</span>;
        ISSN:<span class="NLM_cas:issn">0368-1769</span>.
    </div><div class="casAbstract">The so-called "universal" buffer solns. contg. H3PO4, AcOH (or phenylacetic acid) and boric acids show a graph that is by no means a straight line when the pH is detd. after each small portion of added NaOH.  The use of a linear interpolation formula, therefore, may lead to results which are as much as 0.2 pH in error.  A better universal buffer soln. can be prepd. from citric acid, H3PO4, veronal and H3BO3 with 0.2 N solns. of each.  The dissocn. const. of veronal (diethylbarbituric acid) is 7.96.  The buffer mixt. of the 4 acids gives rise to 7 stages of dissocn., each corresponding to an acid concn. of 0.02857 N.  The soln. contains KH2PO4, citric acid, veronal, H3BO3 and HCl dissolved in water and dild. to 1 1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozqK1uPsKrR7Vg90H21EOLACvtfcHk0lh73qbfogiNSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaA3MXkt1aqtg%253D%253D&md5=265f63ca520e32cfac52dc2324ae6ddf</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1039%2FJR9310001456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FJR9310001456%26sid%3Dliteratum%253Aachs%26aulast%3DBritton%26aufirst%3DH.%2BT.%2BS.%26aulast%3DRobinson%26aufirst%3DR.%2BA.%26atitle%3DCXCVIII.-Universal%2520Buffer%2520Solutions%2520and%2520the%2520Dissociation%2520Constant%2520of%2520Veronal%26jtitle%3DJ.%2520Chem.%2520Soc.%26date%3D1931%26spage%3D1456%26epage%3D1462%26doi%3D10.1039%2Fjr9310001456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gray, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billings, J. A.</span></span> <span> </span><span class="NLM_article-title">Kinetic Assay of Human Pepsin with Albumin-bromphenol Blue as Substrate</span>. <i>Clin. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1093/clinchem/29.3.447</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Fclinchem%2F29.3.447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=6402323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaL3sXhsFKrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1983&pages=447-451&author=S.+P.+Grayauthor=J.+A.+Billings&title=Kinetic+Assay+of+Human+Pepsin+with+Albumin-bromphenol+Blue+as+Substrate&doi=10.1093%2Fclinchem%2F29.3.447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic assay of human pepsin with albumin-bromphenol blue as substrate</span></div><div class="casAuthors">Gray, Stephen P.; Billings, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">447-51</span>CODEN:
                <span class="NLM_cas:coden">CLCHAU</span>;
        ISSN:<span class="NLM_cas:issn">0009-9147</span>.
    </div><div class="casAbstract">A novel substrate, albumin complexed with bromophenol blue, has been developed for the assay of human gastric juice pepsin by a kinetic method in the Cobas centrifugal analyzer.  The action of pepsin on the complex degrades the albumin and releases the dye.  The change in the color of the substrate is a zero-order reaction.  Human and porcine pepsin have different Km values with the new substrate.  This kinetic method has a throughput of 28 tests in ∼10 min and good precision.  Other advantages include anal. in homogeneous soln. (thereby eliminating the need to sep. substrate and products), lack of interference from bilirubin or phenol red, and the expression of pepsin activity in IUB enzyme units.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSannvJ_B-Z7Vg90H21EOLACvtfcHk0ljH8buqazdkLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhsFKrtr0%253D&md5=c5a1cadde3bb74092f4c5e2f8384f230</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fclinchem%2F29.3.447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fclinchem%252F29.3.447%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DS.%2BP.%26aulast%3DBillings%26aufirst%3DJ.%2BA.%26atitle%3DKinetic%2520Assay%2520of%2520Human%2520Pepsin%2520with%2520Albumin-bromphenol%2520Blue%2520as%2520Substrate%26jtitle%3DClin.%2520Chem.%26date%3D1983%26volume%3D29%26spage%3D447%26epage%3D451%26doi%3D10.1093%2Fclinchem%2F29.3.447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holzman, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubbins, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pederson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krafft, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. T.</span></span> <span> </span><span class="NLM_article-title">Recombinant Human Prorenin from CHO Cells: Expression and Purification</span>. <i>J. Protein Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1007/bf01024761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1007%2FBF01024761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1963533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK3MXps1Wqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1990&pages=663-672&author=T.+F.+Holzmanauthor=C.+C.+Chungauthor=R.+Edaljiauthor=D.+A.+Eganauthor=E.+J.+Gubbinsauthor=A.+Rueterauthor=G.+Howardauthor=L.+K.+Yangauthor=T.+M.+Pedersonauthor=G.+A.+Krafftauthor=G.+T.+Wang&title=Recombinant+Human+Prorenin+from+CHO+Cells%3A+Expression+and+Purification&doi=10.1007%2Fbf01024761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Recombinant human prorenin from CHO cells:  expression and purification</span></div><div class="casAuthors">Holzman, Thomas F.; Chung, Christine C.; Edalji, Rohinton; Egan, David A.; Gubbins, Earl J.; Rueter, Annemarie; Howard, Gail; Yang, Lana K.; Pederson, Terry M.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Protein Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">663-72</span>CODEN:
                <span class="NLM_cas:coden">JPCHD2</span>;
        ISSN:<span class="NLM_cas:issn">0277-8033</span>.
    </div><div class="casAbstract">The gene for human preprorenin was obtained from total RNA prepd. from primary human chorion cells.  An expression vector was constructed contg. an SV40 early promoter, a human preprorenin cDNA, bovine growth hormone poly-A addn. signal, and a dihydrofolate reductase (dhfr) expression cassette.  This vector was inserted into the DXB 11 Chinese hamster ovary (CHO) cell line.  The recombinant protein was exported by CHO cells into the tissue culture media.  At harvest the prorenin levels ranged from ∼1-5 mg/L.  For protein isolation the cell culture supernatants were processed by filtration, concn., dialysis, and batch extn.  Preparative-scale isolation of prorenin was accomplished using blue-dye chromatog. and size-exclusion chromatog.  The isolated prorenin yielded a single SDS-gel band with Mr ∼ 40,000.  The proprotein was characterized with respect to N-terminal sequence and N-linked sugar compn.  Trypsin-activated renin prepd. from the proprotein was characterized with respect to N-terminal sequence and pH-activity profile.  Enzyme activity was measured with a newly developed fluorogenic peptide substrate contg. the P6-P'3 sequence of human angiotensinogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFuzdaHY7v7Vg90H21EOLACvtfcHk0ljH8buqazdkLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXps1Wqtw%253D%253D&md5=6e4eb32ddfb7bcb68b13ef2b25f46f41</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1007%2FBF01024761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01024761%26sid%3Dliteratum%253Aachs%26aulast%3DHolzman%26aufirst%3DT.%2BF.%26aulast%3DChung%26aufirst%3DC.%2BC.%26aulast%3DEdalji%26aufirst%3DR.%26aulast%3DEgan%26aufirst%3DD.%2BA.%26aulast%3DGubbins%26aufirst%3DE.%2BJ.%26aulast%3DRueter%26aufirst%3DA.%26aulast%3DHoward%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DL.%2BK.%26aulast%3DPederson%26aufirst%3DT.%2BM.%26aulast%3DKrafft%26aufirst%3DG.%2BA.%26aulast%3DWang%26aufirst%3DG.%2BT.%26atitle%3DRecombinant%2520Human%2520Prorenin%2520from%2520CHO%2520Cells%253A%2520Expression%2520and%2520Purification%26jtitle%3DJ.%2520Protein%2520Chem.%26date%3D1990%26volume%3D9%26spage%3D663%26epage%3D672%26doi%3D10.1007%2Fbf01024761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kageyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akamine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kominami, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchiyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span> <span> </span><span class="NLM_article-title">Characterization of New Fluorogenic Substrates for the Rapid and Sensitive Assay of Cathepsin E and Cathepsin D</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1143</span>, <span class="refDoi"> DOI: 10.1093/oxfordjournals.jbchem.a022396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1093%2Foxfordjournals.jbchem.a022396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10348917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK1MXlsVygsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=1999&pages=1137-1143&author=Y.+Yasudaauthor=T.+Kageyamaauthor=A.+Akamineauthor=M.+Shibataauthor=E.+Kominamiauthor=Y.+Uchiyamaauthor=K.+Yamamoto&title=Characterization+of+New+Fluorogenic+Substrates+for+the+Rapid+and+Sensitive+Assay+of+Cathepsin+E+and+Cathepsin+D&doi=10.1093%2Foxfordjournals.jbchem.a022396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of new fluorogenic substrates for the rapid and sensitive assay of cathepsin E and cathepsin D</span></div><div class="casAuthors">Yasuda, Yoshiyuki; Kageyama, Takashi; Akamine, Akifumi; Shibata, Masahiro; Kominami, Eiki; Uchiyama, Yasuo; Yamamoto, Kenji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1137-1143</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">Cathepsin E and cathepsin D are two major intracellular aspartic proteinases implicated in the physiol. and pathol. degrdn. of intra- and extracellular proteins.  In this study, we designed and constructed highly sensitive synthetic decapeptide substrates for assays of cathepsins E and D based on the known sequence specificities of their cleavage sites.  These substrates contain a highly fluorescent (7-methoxycoumarin-4-yl)acetyl (MOCAc) moiety and a quenching 2,4-dinitrophenyl (Dnp) group.  When the Phe-Phe bond is cleaved, the fluorescence at an excitation wavelength of 328 nm and emission wavelength of 393 increases due to diminished quenching resulting from the sepn. of the fluorescent and quenching moieties.  The first substrate, MOCAc-Gly-Lys-Pro-Ile-Leu-Phe-Phe-Arg-Leu-Lys(Dnp)γ-NH2, in which the Lys-Pro combination at positions P5 and P4 was designed for specific interaction with cathepsin E, is hydrolyzed equally well by cathepsins E and D (kcat/Km = 10.9 μM-1·s-1 for cathepsin E and 15.6 μM-1·s-1 for cathepsin D).  A very acidic pH optimum of 4.0 was obtained for both enzymes.  The second substrate, MOCAc-Gly-Lys-Pro-Ile-Ile-Phe-Phe-Arg-Leu-Lys(Dnp)γ-NH2, in which the isoleucine residue at position P2 was meant to increase the specificity for cathepsin E, is also hydrolyzed equally by both enzymes (kcat/Km = 12.2 μM-1·s-1 for cathepsin E and 16.3 μM-1·s-1 for cathepsin D).  The kcat/Km values for both substrates are greater than those for the best substrates for cathepsins E and D described so far.  Unfortunately, each substrate shows little discrimination between cathepsin E and cathepsin D, suggesting that amino acids at positions far from the cleavage site are important for discrimination between the two enzymes.  However, in combination with aspartic proteinase inhibitors, such as pepstatin A and Ascaris pepsin inhibitor, these substrates enable a rapid and sensitive detn. of the precise levels of cathepsins E and D in crude cell exts. of various tissues and cells.  Thus these substrates represent a potentially valuable tool for routine assays and for mechanistic studies on cathepsins E and D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhuyRvnqRnebVg90H21EOLACvtfcHk0liUFPHklkVPiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlsVygsr4%253D&md5=23d0e8c891cf0ed5c38fa1191d18ec9a</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.jbchem.a022396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.jbchem.a022396%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DY.%26aulast%3DKageyama%26aufirst%3DT.%26aulast%3DAkamine%26aufirst%3DA.%26aulast%3DShibata%26aufirst%3DM.%26aulast%3DKominami%26aufirst%3DE.%26aulast%3DUchiyama%26aufirst%3DY.%26aulast%3DYamamoto%26aufirst%3DK.%26atitle%3DCharacterization%2520of%2520New%2520Fluorogenic%2520Substrates%2520for%2520the%2520Rapid%2520and%2520Sensitive%2520Assay%2520of%2520Cathepsin%2520E%2520and%2520Cathepsin%2520D%26jtitle%3DJ.%2520Biochem.%26date%3D1999%26volume%3D125%26spage%3D1137%26epage%3D1143%26doi%3D10.1093%2Foxfordjournals.jbchem.a022396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">The MX Beamlines BL14.1-3 at BESSY II</span>. <i>J. Large-scale Res. Facil.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">47</span>, <span class="refDoi"> DOI: 10.17815/jlsrf-2-64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.17815%2Fjlsrf-2-64" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=47&author=M.+Gerlachauthor=U.+Muellerauthor=M.+S.+Weiss&title=The+MX+Beamlines+BL14.1-3+at+BESSY+II&doi=10.17815%2Fjlsrf-2-64"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.17815%2Fjlsrf-2-64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.17815%252Fjlsrf-2-64%26sid%3Dliteratum%253Aachs%26aulast%3DGerlach%26aufirst%3DM.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DThe%2520MX%2520Beamlines%2520BL14.1-3%2520at%2520BESSY%2520II%26jtitle%3DJ.%2520Large-scale%2520Res.%2520Facil.%26date%3D2016%26volume%3D2%26spage%3D47%26doi%3D10.17815%2Fjlsrf-2-64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, Scaling, Space-group Assignment and Post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+Scaling%2C+Space-group+Assignment+and+Post-refinement&doi=10.1107%2Fs0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0liUFPHklkVPiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520Scaling%252C%2520Space-group%2520Assignment%2520and%2520Post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2Fs0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of theCCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+theCCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljhJ5GWgyE8NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520theCCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an Automated Program for Molecular Replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1107/s0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+Automated+Program+for+Molecular+Replacement&doi=10.1107%2Fs0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0ljhJ5GWgyE8NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520Automated%2520Program%2520for%2520Molecular%2520Replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2Fs0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 Dictionary: Organization of Prior Chemical Knowledge and Guidelines for its Use</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2184</span>– <span class="NLM_lpage">2195</span>, <span class="refDoi"> DOI: 10.1107/s0907444904023510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1107%2FS0907444904023510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=15572771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2184-2195&author=A.+A.+Vaginauthor=R.+A.+Steinerauthor=A.+A.+Lebedevauthor=L.+Pottertonauthor=S.+McNicholasauthor=F.+Longauthor=G.+N.+Murshudov&title=REFMAC5+Dictionary%3A+Organization+of+Prior+Chemical+Knowledge+and+Guidelines+for+its+Use&doi=10.1107%2Fs0907444904023510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 dictionary: organization of prior chemical knowledge and guidelines for its use</span></div><div class="casAuthors">Vagin, Alexei A.; Steiner, Roberto A.; Lebedev, Andrey A.; Potterton, Liz; McNicholas, Stuart; Long, Fei; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2184-2195</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">One of the most important aspects of macromol. structure refinement is the use of prior chem. knowledge.  Bond lengths, bond angles and other chem. properties are used in restrained refinement as subsidiary conditions.  This contribution describes the organization and some aspects of the use of the flexible and human/machine-readable dictionary of prior chem. knowledge used by the max.-likelihood macromol.-refinement program REFMAC5.  The dictionary stores information about monomers which represent the constitutive building blocks of biol. macromols. (amino acids, nucleic acids and saccharides) and about numerous org./inorg. compds. commonly found in macromol. crystallog.  It also describes the modifications the building blocks undergo as a result of chem. reactions and the links required for polymer formation.  More than 2000 monomer entries, 100 modification entries and 200 link entries are currently available.  Algorithms and tools for updating and adding new entries to the dictionary have also been developed and are presented here.  In many cases, the REFMAC5 dictionary allows entirely automatic generation of restraints within REFMAC5 refinement runs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRjDd-k3YHGrVg90H21EOLACvtfcHk0lhzw48arNx-IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73F&md5=005910422c5fb72fe2434c39b5321bf4</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1107%2FS0907444904023510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904023510%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPotterton%26aufirst%3DL.%26aulast%3DMcNicholas%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DREFMAC5%2520Dictionary%253A%2520Organization%2520of%2520Prior%2520Chemical%2520Knowledge%2520and%2520Guidelines%2520for%2520its%2520Use%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2184%26epage%3D2195%26doi%3D10.1107%2Fs0907444904023510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szalek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowska-Wlodarczyk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overall, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaczmarek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Highly Sensitive and Adaptable Fluorescence-quenched Pair Discloses the Substrate Specificity Profiles in Diverse Protease Families</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">43135</span>, <span class="refDoi"> DOI: 10.1038/srep43135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=10.1038%2Fsrep43135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=28230157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVGjtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=43135&author=M.+Porebaauthor=A.+Szalekauthor=W.+Rutauthor=P.+Kasperkiewiczauthor=I.+Rutkowska-Wlodarczykauthor=S.+J.+Snipasauthor=Y.+Itohauthor=D.+Turkauthor=B.+Turkauthor=C.+M.+Overallauthor=L.+Kaczmarekauthor=G.+S.+Salvesenauthor=M.+Drag&title=Highly+Sensitive+and+Adaptable+Fluorescence-quenched+Pair+Discloses+the+Substrate+Specificity+Profiles+in+Diverse+Protease+Families&doi=10.1038%2Fsrep43135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Highly sensitive and adaptable fluorescence-quenched pair discloses the substrate specificity profiles in diverse protease families</span></div><div class="casAuthors">Poreba, Marcin; Szalek, Aleksandra; Rut, Wioletta; Kasperkiewicz, Paulina; Rutkowska-Wlodarczyk, Izabela; Snipas, Scott J.; Itoh, Yoshifumi; Turk, Dusan; Turk, Boris; Overall, Christopher M.; Kaczmarek, Leszek; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43135</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Internally quenched fluorescent (IQF) peptide substrates originating from FRET (Forster Resonance Energy Transfer) are powerful tool for examg. the activity and specificity of proteases, and a variety of donor/acceptor pairs are extensively used to design individual substrates and combinatorial libraries.  We developed a highly sensitive and adaptable donor/acceptor pair that can be used to investigate the substrate specificity of cysteine proteases, serine proteases and metalloproteinases.  This novel pair comprises 7-amino-4-carbamoylmethylcoumarin (ACC) as the fluorophore and 2,4-dinitrophenyl-lysine (Lys(DNP)) as the quencher.  Using caspase-3, caspase-7, caspase-8, neutrophil elastase, legumain, and two matrix metalloproteinases (MMP2 and MMP9), we demonstrated that substrates contg. ACC/Lys(DNP) exhibit 7 to 10 times higher sensitivity than conventional 7-methoxy-coumarin-4-yl acetic acid (MCA)/Lys(DNP) substrates; thus, substantially lower amts. of substrate and enzyme can be used for each assay.  We therefore propose that the ACC/Lys(DNP) pair can be considered a novel and sensitive scaffold for designing substrates for any group of endopeptidases.  We further demonstrate that IQF substrates contg. unnatural amino acids can be used to investigate protease activities/specificities for peptides contg. post-translationally modified amino acids.  Finally, we used IQF substrates to re-investigate the P1-Asp characteristic of caspases, thus demonstrating that some human caspases can also hydrolyze substrates after glutamic acid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe_raSFJMVd7Vg90H21EOLACvtfcHk0lhzw48arNx-IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsVGjtLs%253D&md5=acc661ca9d9b4f2f0e65d3d317b9290f</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fsrep43135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep43135%26sid%3Dliteratum%253Aachs%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSzalek%26aufirst%3DA.%26aulast%3DRut%26aufirst%3DW.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DRutkowska-Wlodarczyk%26aufirst%3DI.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTurk%26aufirst%3DD.%26aulast%3DTurk%26aufirst%3DB.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26aulast%3DKaczmarek%26aufirst%3DL.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DHighly%2520Sensitive%2520and%2520Adaptable%2520Fluorescence-quenched%2520Pair%2520Discloses%2520the%2520Substrate%2520Specificity%2520Profiles%2520in%2520Diverse%2520Protease%2520Families%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D43135%26doi%3D10.1038%2Fsrep43135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R5H','PDB','6R5H'); return false;">PDB: 6R5H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2REN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2REN','PDB','2REN'); return false;">PDB: 2REN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R6A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R6A','PDB','6R6A'); return false;">PDB: 6R6A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2hvp" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2hvp','PDB','2hvp'); return false;">PDB: 2hvp</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02177">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_38515"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02177?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02177</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Prodomain processing, structure determination, and cytotoxicity assays MayI SI (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf">PDF</a>)</p></li><li><p class="inline">MayI SMILES (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_001.pdf">jm0c02177_si_001.pdf (1.26 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02177/suppl_file/jm0c02177_si_002.csv">jm0c02177_si_002.csv (3.02 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB code for MayI with bound PepA is 6R6A. MayI in apo-enzyme form is under 6R5H.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02177&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02177%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-10%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02177" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997f424d372440","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
