
<html lang="en"     class="pb-page"  data-request-id="79a864b1-b8c9-44dc-b9d8-775eec802870"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c01768;issue:issue:10.1021/jmcmar.2021.64.issue-3;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Small-Molecule Antagonists of the PWWP Domain of NSD2" /></meta><meta name="dc.Creator" content="Renato  Ferreira de Freitas" /></meta><meta name="dc.Creator" content="Yanli  Liu" /></meta><meta name="dc.Creator" content="Magdalena M.  Szewczyk" /></meta><meta name="dc.Creator" content="Naimee  Mehta" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="David  McLeod" /></meta><meta name="dc.Creator" content="Carlos  Zepeda-Velázquez" /></meta><meta name="dc.Creator" content="David  Dilworth" /></meta><meta name="dc.Creator" content="Ronan P.  Hanley" /></meta><meta name="dc.Creator" content="Elisa  Gibson" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Rima  Al-Awar" /></meta><meta name="dc.Creator" content="Lindsey I.  James" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="Jinrong  Min" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Matthieu  Schapira" /></meta><meta name="dc.Creator" content="Abdellah  Allali-Hassani" /></meta><meta name="dc.Description" content="Increased activity of the lysine methyltransferase NSD2 driven by translocation and activating mutations is associated with multiple myeloma and acute lymphoblastic leukemia, but no NSD2-targeting ..." /></meta><meta name="Description" content="Increased activity of the lysine methyltransferase NSD2 driven by translocation and activating mutations is associated with multiple myeloma and acute lymphoblastic leukemia, but no NSD2-targeting ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 1, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01768" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01768" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01768" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01768" /></link>
        
    
    

<title>Discovery of Small-Molecule Antagonists of the PWWP Domain of NSD2 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01768" /></meta><meta property="og:title" content="Discovery of Small-Molecule Antagonists of the PWWP Domain of NSD2" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0015.jpeg" /></meta><meta property="og:description" content="Increased activity of the lysine methyltransferase NSD2 driven by translocation and activating mutations is associated with multiple myeloma and acute lymphoblastic leukemia, but no NSD2-targeting chemical probe has been reported to date. Here, we present the first antagonists that block the protein–protein interaction between the N-terminal PWWP domain of NSD2 and H3K36me2. Using virtual screening and experimental validation, we identified the small-molecule antagonist 3f, which binds to the NSD2-PWWP1 domain with a Kd of 3.4 μM and abrogates histone H3K36me2 binding to the PWWP1 domain in cells. This study establishes an alternative approach to targeting NSD2 and provides a small-molecule antagonist that can be further optimized into a chemical probe to better understand the cellular function of this protein." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01768"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01768">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01768&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01768&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01768&amp;href=/doi/10.1021/acs.jmedchem.0c01768" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1584-1592</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01537" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01774" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Small-Molecule Antagonists of the PWWP Domain of NSD2</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Renato Ferreira de Freitas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Renato Ferreira de Freitas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Renato++Ferreira+de+Freitas">Renato Ferreira de Freitas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanli Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanli Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanli++Liu">Yanli Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Magdalena M. Szewczyk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Magdalena M. Szewczyk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Magdalena+M.++Szewczyk">Magdalena M. Szewczyk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Naimee Mehta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Naimee Mehta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Naimee++Mehta">Naimee Mehta</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fengling Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fengling Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David McLeod</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David McLeod</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery Program, Ontario Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++McLeod">David McLeod</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carlos Zepeda-Velázquez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carlos Zepeda-Velázquez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery Program, Ontario Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carlos++Zepeda-Vel%C3%A1zquez">Carlos Zepeda-Velázquez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Dilworth</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Dilworth</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Dilworth">David Dilworth</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ronan P. Hanley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ronan P. Hanley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ronan+P.++Hanley">Ronan P. Hanley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elisa Gibson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisa Gibson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisa++Gibson">Elisa Gibson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter J. Brown</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter J. Brown</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rima Al-Awar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rima Al-Awar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery Program, Ontario Institute for Cancer Research, Toronto, Ontario M5G 0A3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rima++Al-Awar">Rima Al-Awar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lindsey I. James</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lindsey I. James</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lindsey+I.++James">Lindsey I. James</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6034-7116" title="Orcid link">http://orcid.org/0000-0002-6034-7116</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheryl H. Arrowsmith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheryl H. Arrowsmith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Princess Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dalia Barsyte-Lovejoy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dalia Barsyte-Lovejoy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology & Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jinrong Min</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jinrong Min</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Physiology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jinrong++Min">Jinrong Min</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masoud Vedadi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masoud Vedadi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology & Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthieu Schapira</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthieu Schapira</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology & Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#076a6673736f6e62722974646f66776e7566477273687568697368296466"><span class="__cf_email__" data-cfemail="caa7abbebea2a3afbfe4b9a9a2abbaa3b8ab8abfbea5b8a5a4bea5e4a9ab">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthieu++Schapira">Matthieu Schapira</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1047-3309" title="Orcid link">http://orcid.org/0000-0002-1047-3309</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Abdellah Allali-Hassani</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Abdellah Allali-Hassani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#8beaeae7e7eae7e2e3eaf8f8eae5e2cbe2e5e8f2ffeea5e8e4e6"><span class="__cf_email__" data-cfemail="e382828f8f828f8a8b829090828d8aa38a8d809a9786cd808c8e">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Abdellah++Allali-Hassani">Abdellah Allali-Hassani</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01768&amp;href=/doi/10.1021%2Facs.jmedchem.0c01768" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1584–1592</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 1, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 October 2020</li><li><span class="item_label"><b>Published</b> online</span>1 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 February 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01768" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01768</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1584%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRenato%2BFerreira%2Bde%2BFreitas%252C%2BYanli%2BLiu%252C%2BMagdalena%2BM.%2BSzewczyk%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D3%26contentID%3Dacs.jmedchem.0c01768%26title%3DDiscovery%2Bof%2BSmall-Molecule%2BAntagonists%2Bof%2Bthe%2BPWWP%2BDomain%2Bof%2BNSD2%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1592%26publicationDate%3DFebruary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01768"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1539</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01768" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Small-Molecule Antagonists of the PWWP Domain of NSD2&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Renato&quot;,&quot;last_name&quot;:&quot;Ferreira de Freitas&quot;},{&quot;first_name&quot;:&quot;Yanli&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Magdalena&quot;,&quot;last_name&quot;:&quot;M. Szewczyk&quot;},{&quot;first_name&quot;:&quot;Naimee&quot;,&quot;last_name&quot;:&quot;Mehta&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;McLeod&quot;},{&quot;first_name&quot;:&quot;Carlos&quot;,&quot;last_name&quot;:&quot;Zepeda-Velázquez&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Dilworth&quot;},{&quot;first_name&quot;:&quot;Ronan&quot;,&quot;last_name&quot;:&quot;P. Hanley&quot;},{&quot;first_name&quot;:&quot;Elisa&quot;,&quot;last_name&quot;:&quot;Gibson&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Rima&quot;,&quot;last_name&quot;:&quot;Al-Awar&quot;},{&quot;first_name&quot;:&quot;Lindsey&quot;,&quot;last_name&quot;:&quot;I. James&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;Jinrong&quot;,&quot;last_name&quot;:&quot;Min&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Matthieu&quot;,&quot;last_name&quot;:&quot;Schapira&quot;},{&quot;first_name&quot;:&quot;Abdellah&quot;,&quot;last_name&quot;:&quot;Allali-Hassani&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;1584-1592&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01768&quot;},&quot;abstract&quot;:&quot;Increased activity of the lysine methyltransferase NSD2 driven by translocation and activating mutations is associated with multiple myeloma and acute lymphoblastic leukemia, but no NSD2-targeting chemical probe has been reported to date. Here, we present the first antagonists that block the protein–protein interaction between the N-terminal PWWP domain of NSD2 and H3K36me2. Using virtual screening and experimental validation, we identified the small-molecule antagonist 3f, which binds to the NSD2-PWWP1 domain with a Kd of 3.4 μM and abrogates histone H3K36me2 binding to the PWWP1 domain in cells. This study establishes an alternative approach to targeting NSD2 and provides a small-molecule antagonist that can be further optimized into a chemical probe to better understand the cellular function of this protein.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01768&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01768" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01768&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01768" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01768&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01768" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01768&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01768&amp;href=/doi/10.1021/acs.jmedchem.0c01768" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01768" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01768" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01768%26sid%3Dliteratum%253Aachs%26pmid%3D33522809%26genre%3Darticle%26aulast%3DFerreira%2Bde%2BFreitas%26date%3D2021%26atitle%3DDiscovery%2Bof%2BSmall-Molecule%2BAntagonists%2Bof%2Bthe%2BPWWP%2BDomain%2Bof%2BNSD2%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D3%26spage%3D1584%26epage%3D1592%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/jmcmar.2021.64.issue-3/20210211/jmcmar.2021.64.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Increased activity of the lysine methyltransferase NSD2 driven by translocation and activating mutations is associated with multiple myeloma and acute lymphoblastic leukemia, but no NSD2-targeting chemical probe has been reported to date. Here, we present the first antagonists that block the protein–protein interaction between the N-terminal PWWP domain of NSD2 and H3K36me2. Using virtual screening and experimental validation, we identified the small-molecule antagonist <b>3f</b>, which binds to the NSD2-PWWP1 domain with a <i>K</i><sub>d</sub> of 3.4 μM and abrogates histone H3K36me2 binding to the PWWP1 domain in cells. This study establishes an alternative approach to targeting NSD2 and provides a small-molecule antagonist that can be further optimized into a chemical probe to better understand the cellular function of this protein.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57071" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57071" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">NSD2 (nuclear receptor-binding SET domain-containing 2, also known as WHSC1 and MMSET) is a protein lysine methyltransferase that belongs to the NSD family, which also includes NSD1 and NSD3, and that predominantly mono- and dimethylates lysine 36 of histone 3 (H3K36).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> NSD2 is an oncoprotein that is aberrantly expressed, amplified, or somatically mutated in multiple types of cancer.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Notably, the t(4;14) NSD2 translocation in multiple myeloma and the hyperactivating NSD2 mutation E1099K in a subset of pediatric acute lymphoblastic leukemia result in altered chromatin methylation that drives oncogenesis.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a></div><div class="NLM_p">While NSD2 is an attractive therapeutic target, efforts to target the catalytic SET domain with small-molecule inhibitors have so far met with little success,<a onclick="showRef(event, 'ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9">(6−9)</a> and only recently was the first selective inhibitor of an NSD family protein reported, a compound that binds covalently to the catalytic site of NSD1.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Aside from the catalytic domain, NSD2 has multiple protein–protein interaction domains that may be clinically relevant, including plant homeodomain and PWWP (proline–tryptophan–tryptophan–proline) domains (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> The N-terminal PWWP domain of NSD2 (NSD2-PWWP1) binds H3K36me2, presumably through a conserved aromatic cage composed of three orthogonally positioned aromatic side chains (Y233, W236, F266) that can engage in cation−π and hydrophobic interactions with the ammonium group of the methylated lysine;<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> the F266A mutation at the aromatic cage destabilizes the chromatin occupancy of full-length NSD2 and inhibits cancer cell proliferation, but without significantly affecting H3K36 dimethylation.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Small molecules selectively targeting the aromatic cage of NSD2-PWWP1 would be valuable chemical tools to probe the therapeutic relevance of this domain in NSD2-driven tumors or to design NSD2-targeting PROTACs.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Domain architecture of NSD2 and chemical evolution of NSD2-PWWP1 antagonists. Receptor-based virtual screening followed by target and scaffold hopping led to the development of <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In this study, we report the first chemical antagonists targeting the NSD2-PWWP1:H3K36me2 interaction in biophysical and cellular assays. X-ray crystallography established the structural determinants for ligand binding and provided crucial information for future ligand optimization. To our knowledge, the present study is the first successful example of using a virtual screening approach to discover antagonists that directly and specifically target a PWWP domain.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23451" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23451" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Structure-Based Discovery of NSD2-PWWP1 Ligands</h3><div class="NLM_p last">At the onset of this project, no structure of NSD2 PWWP domains was available. Of the PWWP domain structures available from the Protein Data Bank (PDB), that of the chromatin factor ZMYND11 had a structurally well-defined methyl-lysine binding pocket and was identified as a good candidate for a PWWP-focused virtual screening campaign. A library of ∼2 million commercial compounds was therefore screened virtually against the PWWP domain of ZMYND11.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> A total of 39 compounds were purchased and experimentally screened using differential static light scattering (DSLS—<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Table S1</a>). Since no hit was confirmed for ZMYND11, we took a target class approach and screened the purchased compound set against other members of the PWWP family. Compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) displayed a stabilization effect on the N-terminal PWWP domain of NSD2, increasing its melting temperature by 4 °C at 400 μM. The interaction of <b>1</b> with NSD2-PWWP1 was further confirmed by surface plasmon resonance (SPR), which yielded a dissociation constant (<i>K</i><sub>d</sub>) of 41 ± 8 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Figure S1a</a>). Given the poor solubility of <b>1</b>, a reverse isothermal titration calorimetry (ITC) titration was carried out by titrating a concentrated NSD2-PWWP1 solution (1 mM) into the sample cell containing a solution of compound <b>1</b> at 40 μM, which produced a <i>K</i><sub>d</sub> of 8.9 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Figure S1b</a>). Compound <b>1</b> did not bind six other PWWP-containing proteins when tested at 400 μM using DSLS (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Table S2</a>), indicating that this compound is selective for NSD2-PWWP1. Compound <b>1</b> contains two chiral centers but was found to be a single diastereoisomer where both enantiomers bound the target equipotently (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Figures S2–S4</a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Ligand-Guided Optimization Leads to Improved Compounds</h3><div class="NLM_p">The encouraging potency and selectivity of <b>1</b> prompted us to further explore this scaffold. However, no analogues were commercially available, and only a limited number of building blocks could be purchased for analogue synthesis. We therefore sought to identify a new series of antagonists that would be more tractable for follow-up medicinal chemistry efforts. Since no structure of the NSD2-PWWP1 domain had been reported, we used ligand-based scaffold hopping approaches.</div><div class="NLM_p">First, a low-energy 3D conformation of compound <b>1</b> was generated with the computational chemistry suite ICM<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and used as a reference to screen a virtual library of ∼8 million commercially available compounds with the shape-matching tool ROCS (OpenEye).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Second, we used FTrees (BioSolveIT)<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> to perform a 2D similarity search against the same library. This approach uses a fuzzy similarity searching, ignoring the three-dimensional structure and chirality of the reference. This feature was helpful in this case, as at the time, we knew neither the bioactive conformation of <b>1</b> nor the correct stereochemistry of the two chiral centers. In the third approach, we performed a substructure search using Filter (OpenEye).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Finally, based on the structure of <b>1</b>, a series of scaffold-hopping candidates were manually designed and the closest commercial compounds selected. The atomic property field<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> alignment method implemented in ICM was used to flexibly align the top hits of each approach to the lowest energy conformation of <b>1</b> used in the ROCS screen. Visual analysis and clustering with LibMCS (ChemAxon)<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> led to the selection of 24 compounds (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Table S1</a>).</div><div class="NLM_p">Of the 24 compounds tested, only one hit (compound <b>2</b>) showed a modest stabilization (Δ<i>T</i><sub>agg</sub> = 1.9 °C) of NSD2-PWWP1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The binding of <b>2</b> was further confirmed by SPR with a <i>K</i><sub>d</sub> of 175 μM. Although <b>2</b> was weaker than the initial hit, this new scaffold allowed us to perform a structure–activity relationship (SAR) by catalog, as several analogues were commercially available. From a substructure search, 34 analogues of <b>2</b> were purchased and tested (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Table S1</a>). The initial SAR showed that removing the chlorine group improved potency (<b>2a</b>), while replacing it with a methoxy group had no significant effect (<b>2b</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The chlorophenyl group could be replaced with other heterocycles without a significant effect on potency (<b>2c–2e</b>) but removing the nitrile or replacing it with groups lacking a hydrogen bond acceptor abolished activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>: <b>2h</b> vs <b>2a</b>, <b>2i</b> vs <b>2</b>), suggesting that the nitrile group was forming a specific interaction with the protein.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SARs of the Analogues of Compound <b>2</b> and Close Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0006.gif" alt="" id="GRAPHIC-d7e636-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0007.gif" alt="" id="gr5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Compound concentration was 500 μM.</p></div></div><div></div></div><div class="NLM_p">On the basis of the preliminary SAR provided by <b>2</b> and its close derivatives, we set out to identify novel chemical templates that would preserve the two phenyl groups and the nitrile of <b>2</b> using FTrees. The seven compounds selected from this exercise (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Table S1</a>) included the breakthrough antagonist <b>3</b>, which binds NSD2-PWWP1 with a <i>K</i><sub>d</sub> of 7 μM by SPR and shows strong protein stabilization (Δ<i>T</i><sub>agg</sub> = 5.7 °C) by DSLS (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Compound <b>3</b> is 6 and 25-times more potent than <b>1</b> and <b>2</b>, respectively, is chemically more accessible than the previous scaffolds as it has no chiral center, and can easily be derivatized.</div><div class="NLM_p">To further understand the molecular basis for the increased potency of <b>3</b>, we tested 40 analogues that were either purchased or synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">S1</a>). The initial SAR demonstrated that the cyclopropyl group is an essential structural element, since compounds where it is absent (<b>3b</b>) or is replaced with smaller (<b>3a</b>) or larger (<b>3c</b>—<b>3e</b>) groups are much weaker or completely inactive (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Moreover, replacing the fluorophenyl group with thiophene (<b>3f</b>) or tetrahydrofuran (THF) (<b>3g</b>) resulted in similar potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). When tested against 10 PWWP domains by DSLS, compound <b>3f</b> only bound NSD2-PWWP1 and was found to be about 6 times more potent than the initial compound <b>1</b> (SPR <i>K</i><sub>d</sub> = 7 ± 3 μM) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). <b>3f</b> did not bind to NSD2-PWWP1 mutants where aromatic cage residues Y233 or F266 were mutated to alanine, indicating binding at the methyl-lysine binding site (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Figure S5</a>). It was previously shown that F266A antagonizes association of NSD2 to chromatin, indicating that <b>3f</b> binds at a site that contributes to chromatin engagement.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> In agreement with a recent observation that high dimethyl sulfoxide (DMSO) concentrations can limit the potency of PWWP ligands,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> the <i>K</i><sub>d</sub> value of <b>3f</b> was measured as 3.4 ± 0.4 μM in an SPR experiment with 0.5% DMSO (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Figure S6</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>3f</b> (structure shown in the center) selectively binds NSD2 PWWP1. (A) Treatment with <b>3f</b> increases the melting temperature of NSD2-PWWP1 in a DSLS assay. (B) <b>3f</b> binds NSD2-PWWP1 in an SPR assay. (C) <b>3f</b> only affects the melting temperature of NSD2-PWWP1 in a panel of 10 PWWP domains at 100 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the Analogues of Compound <b>3</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0008.gif" alt="" id="GRAPHIC-d7e790-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0009.gif" alt="" id="gr6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Compound concentration was 500 μM.</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Crystal Structure Confirms Binding at the PWWP Aromatic Cage</h3><div class="NLM_p">To support follow-up chemistry focused on this chemical series, we solved the crystal structure of NSD2-PWWP1 in complex with compound <b>3f</b> at a resolution of 2.4 Å (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Table S3, and Figure S7</a>). Examination of the complex structure revealed the molecular basis for the high affinity of the antagonist and confirmed that it was binding in the Kme reader aromatic cage. The cyclopropyl ring is bound in a narrow and deep lipophilic pocket formed by the aromatic cage residues (Y233, W236, and F266) and V230, while a tightly coordinated water molecule is found at the bottom of the pocket. This rationalizes previous SAR showing that compounds lacking the cyclopropyl ring are inactive, as they do not fill the hydrophobic pocket. Conversely, compounds featuring larger rings such as cyclohexane (<b>3e</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) or even an iso-propyl group (<b>3c</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) probably clash with the amino acids of the aromatic cage (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Figure S8</a>). The cyanophenyl group is enclosed in a pocket formed by W236, G268, D269, A270, E272, L318, and Q321. The nitrile group is completely shielded from the solvent and the nitrogen accepts a hydrogen bond from the amide nitrogen of A270. This supports our previous SAR showing that compounds without the nitrile group are weak or inactive. In addition, the carbonyl oxygen of <b>3f</b> forms a hydrogen bond with the side chain of Y233 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Finally, the thiophene ring binds within a hydrophobic pocket defined by V230 and A274, and partially exposed to the solvent, which is in agreement with the fact that chemical modifications are tolerated at this end of the molecule.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of the NSD2-PWWP1 domain in complex with <b>3f</b> [PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UE6">6UE6</a>]. Binding pocket residues (gray) and the antagonist (cyan) are displayed as sticks and hydrogen bonds as magenta dashed lines. Aromatic cage residues are colored in green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Compared to the recently published structure of NSD2-PWWP1 in complex with DNA (PBD ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VC8">5VC8</a>), the loop residues G268, D269, A270, and P271 connecting the β3 and β4 strands undergo significant conformational changes when the ligand binds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">Figure S9</a>). Also, <b>3f</b> induces a conformational change of Y233 and E272, opening-up the aromatic cage which was occluded in the apo structure.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Compound <b>3f</b> Engages NSD2-PWWP1 in Cells</h3><div class="NLM_p">Cellular target engagement and displacement of NSD2-PWWP1 from histone H3.3 were tested using a NanoBRET assay measuring the displacement of the NanoLuc-tagged NSD2-PWWP1 domain from Halo-tagged histone H3.3. Encouragingly, we found that compound <b>3f</b> decreased the interaction of NSD2-PWWP1 but not NSD3-PWWP1 with histone 3 in cells in a dose-dependent manner with an IC<sub>50</sub> of 17.3 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>3f</b> disengages NSD2 PWWP1 from histone H3 in cells. Mean-corrected NanoBRET ratios (mBU) are shown for the interaction between NSD2 (orange) and NSD3 (green) with increasing doses of compound <b>3f</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The first potent antagonist for a PWWP domain was recently reported against NSD3-PWWP1.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Our results further support the idea that PWWP domains represent a chemically tractable target class. PWWP domains are often found in multi-modular proteins involved in transcriptional regulation or DNA repair,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and pharmacological targeting of PWWP domains could provide an avenue to deregulate the function of these proteins. PWWP ligands could also serve as chemical handles toward the development of PROTACs that catalyze the proteasomal degradation of PWWP-containing proteins.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42661" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42661" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, using a combination of receptor-based virtual screening and ligand-based scaffold hopping, we identified chemically tractable small-molecule antagonists targeting the NSD2-PWWP1 domain with low micromolar affinity. A crystal structure showed that compound <b>3f</b> occupies the Kme binding aromatic cage of NSD2-PWWP1 and <b>3f</b> inhibits histone H3K36me2 binding in cells. This study establishes an alternative approach to targeting NSD2 and reveals a class of compounds that we hope to further optimize into a high-quality chemical probe.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62235" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62235" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Computational Methods</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Docking</h4><div class="NLM_p">The X-ray structure of the PWWP domain of ZMYND11 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N4I">4N4I</a>)<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> was prepared with PrepWizard (Schrodinger, New York) using the standard protocol, including the addition of hydrogens, the assignment of bond order, assessment of the correct protonation states, and a restrained minimization using the OPLS-AA 2005 force field. Receptor grids were calculated at the centroid of the trimethyllysine (M3L) with the option to dock ligands of similar size.</div><div class="NLM_p last">A lead-like library of ∼2 million compounds was prepared with LigPrep (Schrodinger, New York). The resulting library was then docked using the virtual screening workflow using Glide HTVS in the first stage, followed by Glide SP in the second stage (Schrodinger, New York). After removing the compounds with a docking score worse than −7 kcal/mol and that do not contain a tertiary amine, the remaining 18k compounds were docked with Glide XP. Next, compounds with a docking score better than −8 kcal/mol or with a score better than −6 kcal/mol and two hydrogen bonds with nearby residues (R317, F311, H313, N316, W319, H314, S340) were selected for visual inspection. This step selected 431 compounds, which were rescored with the GBSA method (AMBER 12, UCSF). The compounds were clustered and the ones with GBSA > −15 kcal/mol were removed. Finally, after a visual inspection, 40 compounds were selected to be purchased.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> ROCS Search</h4><div class="NLM_p last">A library of ∼8 million compounds was processed with the Omega software (OpenEye Scientific Software, Santa Fe) to generate 200 lowest energy conformations for each compound. Then, ROCS (OpenEye Scientific Software, Santa Fe) was used to perform a 3D shape-based search against the resulting library using a low-energy 3D conformation of <b>1</b> generated by ICM (Molsoft, San Diego) as the query. The compounds were ranked by the TanimotoCombo score, which is a combination of the shape and color similarities, and the top 7500 were saved for analysis.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> FTrees Search</h4><div class="NLM_p last">The 2D structure of <b>1</b> was used as the reference to search the virtual library described before using the Ftrees (Feature Trees) software (BioSolveIT GmbH). Feature trees are descriptors that represent the molecule as a reduced graph. The graph encodes functional groups and rings to single nodes. The similarity between two molecules is calculated by comparing their associated feature trees by superposing (matching) similar subtrees onto each other. The top 1980 compounds with the highest similarity with compound 1 were selected for further analysis.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Substructure Search</h4><div class="NLM_p last">The software Filter (OpenEye Scientific Software, Sant Fe) was used to perform a substructure search against the virtual library using a substructure of <b>1</b> as the query. The substructure used in the search encoded in SMARTS was NC(═[OX1])C([ar5,ar6])[N;!$(N═C);!$(N═N);!$(N-[!#6;!#1]);!$(N–C═[O,N,S])][CX4][ar5,ar6]. This searched resulted in 1041 compounds.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">Purified NSD2-PWWP1 was dialyzed in ITC buffer (20 mM Tris, pH 7.5, and 100 mM NaCl). Protein at 1 mM was injected into the sample cell containing about 300 μL of 40 μM <b>1</b> with 1% DMSO. ITC titrations were performed on a Nano ITC from TA Instruments (New Castle, DE) at 25 °C by using 4 μL injections with a total of 12 injections. The concentration of DMSO was adjusted to 1% for NSD2-PWWP1 solution. Data were fitted with a one-binding-site model using Nano Analyze software.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Differential Static Light Scattering</h3><div class="NLM_p last">Experiments were carried out by determining the effect of compounds on the thermal stability by DSLS using StarGazer (from Harbinger). All PWWP domains for selectivity experiments were used at a final concentration of 0.2 mg/mL in a buffer consisting of 0.1 M 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES) (pH 7.5) and 150 mM NaCl. As described previously,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> this method assesses ligand binding through ligand-induced stability where protein aggregation/denaturation is measured while heating the sample from 25 to 85 °C at 1 °C/min in a 50 μL volume (covered with 50 μL of mineral oil to prevent evaporation) in a clear-bottom 384-well plate (from Nunc). Aggregation was monitored by the increase of scattered light using CCD camera detection. Pixel intensities were integrated using image analysis software, plotted against temperature and data was then fitted to a Boltzmann sigmoid function to obtain the aggregation temperature (<i>T</i><sub>agg</sub>) from the midpoint of the transition.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Surface Plasmon Resonance</h3><div class="NLM_p last">SPR studies were performed using a Biacore T200 (GE Health Sciences Inc.) at 20 °C. Biotinylated NSD2-PWWP1 was captured onto one flow cell of a streptavidin-conjugated SA chip at approximately 6000 response units (according to the manufacturer’s protocol), while another flow cell was left empty for reference subtraction. Compound <b>3f</b> was tested at 200 μM as the highest concentration and dilution factor of 0.5 in HBS–EP buffer (20 mM HEPES pH 7.4, 150 mM NaCl, 3 mM ethylenediaminetetraacetic acid, 0.05% Tween-20) was used to yield 10 concentrations. Experiments were performed using the same buffer with 5% DMSO in single cycle kinetics with 60 s contact time and a dissociation time of 120 s at a flow rate of 75 μL/min. Kinetic curve fittings and <i>K</i><sub>d</sub> value calculations were done with a 1:1 binding model using the Biacore T200 Evaluation software (GE Health Sciences Inc.). All compounds that were evaluated in biochemical and biophysical assays had >95% purity as determined by <sup>1</sup>H NMR and liquid chromatography/mass spectrometry (LC–MS).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Crystal Structure</h3><div class="NLM_p last">The NSD2 fragment (aa 211-350) containing the first human PWWP domain of NSD2 was cloned into a pET28a-MHL vector with a His-tag in its N-terminus for affinity purification. The recombinant plasmid was transformed in the <i>Escherichia coli</i> BL21 (DE3), and the target protein overexpression was induced by 0.2 mM IPTG at 18 °C overnight. The cells were harvested and lysed by sonication. The cell lysis was first purified by Ni-NTA, and the eluted protein was treated by tobacco etch virus to remove the His-tag by dialysis against dialysis buffer containing 20 mM Tris pH 7.5, 150 mM NaCl, 1 mM dithiothreitol (DTT). The sample was then purified by gel filtration chromatography using a Superdex 75 column in a buffer containing 20 mM Tris, pH 7.5, 150 mM NaCl, 1 mM DTT. The peak fractions from the gel filtration column were pooled and concentrated to 20 mg/mL for crystallization. The purified protein was mixed with the ligand at a molar ratio of 1:3 and cocrystallized using the sitting drop vapor diffusion method at 18 °C. The crystals were obtained in a buffer containing 20% PEG 3350, 0.2 M KSCN. Crystals were soaked in a cryoprotectant consisting of 100% reservoir solution and 15% glycerol for data collection. X-ray diffraction data for NSD2 + <b>3f</b> was collected at 100k at beamline 08ID-1 of Canadian Light Source (CLS). The data set was processed using the XDS<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> suite. The structures of NSD2 + <b>3f</b> were solved by molecular replacement using PHASER<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> with PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VC8">5VC8</a> as the search template. Graphics program COOT<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> was used for model building and visualization. Geometry restraints for the compound refinement were prepared with GRADE (Global Phasing Ltd.) developed at Global Phasing Ltd. Restrained refinement and validation using BUSTER (Global Phasing Ltd.) and MOLPROBITY,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> respectively.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> NanoBRET Assay</h3><div class="NLM_p last">U2Os cells were plated in 12-well plates (1 × 10<sup>5</sup>/well) in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (FBS), penicillin (100 U/mL), and streptomycin (100 μg/mL). 4 h after plating, cells were transfected with 1 μg of histone H3.3-HaloTagFusion Vector DNA (Promega) + 0.1 μg of NSD2 PWWP1—NanoLuc Fusion Vector DNA (C- terminal) (Promega) using X-tremeGENE HP DNA Transfection Reagent (Sigma) following manufacturer’s instructions. Next day, cells were trypsinized, spun down, and resuspended in DMEM/F12 (no phenol red) supplemented with 4% FBS, penicillin (100 U/mL), and streptomycin (100 μg/mL) at a density of 1.1 × 10<sup>5</sup>/mL. Cells were divided into two pools. To the first pool, 1 μL/mL DMEM/F12 (no phenol red) of HaloTag NanoBRET 618 Ligand (Promega) was added and to the second pool DMSO. Cells were plated (90 μL/well) in 96-well plates (white, 655083, Greiner Bio One). 10× concentrated compound and DMSO control were prepared in DMEM/F12 and added to cells (10 μL/well). Next day, 25 μL of NanoBRET Nano-GloR Substrate (Promega) solution in DMEM/F12 (10 μL/mL) was added to each well. Cells were shaken for 30 s, and donor emission at 450 nm (filter: 450 nm/BP 80 nm) and acceptor emission at 618 nm (filter: 610 nm/LP) were measured within 10 min of substrate addition using a CLARIOstar microplate reader (Mandel). Mean-corrected NanoBRET ratios (mBU) were determined by subtracting the mean of the 618/460 signal from cells without NanoBRET 618 Ligand x 1000 from the mean of the 618/460 signal from cells with NanoBRET 618 Ligand × 1000.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Synthesis and Separation of All Four Isomers of Compound <b>1</b></h3><div class="NLM_p">All reagents were purchased from commercial vendors and used without further purification. Volatiles were removed under reduced pressure by rotary evaporation or by using the V-10 solvent evaporator system by Biotage. Very high boiling point (6000 rpm, 0 mbar, 56 °C), mixed volatile (7000 rpm, 30 mbar, 36 °C), and volatile (6000 rpm, 30 mbar, 36 °C) methods were used to evaporate solvents. The yields given refer to chromatographically purified and spectroscopically pure compounds. Compounds were purified using a Biotage Isolera One system by normal phase chromatography using Biotage SNAP KP-Sil or Sfär Silica D columns (part no: FSKO-1107/FSRD-0445) or by reverse-phase chromatography using Biotage SNAP KP-C18-HS or Sfär C18 D columns (Part no.: FSLO-1118/FSUD-040). If additional purification was required, compounds were purified by solid-phase extraction (SPE) using Biotage Isolute Flash SCX-2 cation exchange cartridges (part no: 532-0050-C and 456-0200-D). Products were washed with two cartridge volumes of MeOH and eluted with a solution of MeOH and NH<sub>4</sub>OH (9:1 v/v). Preparative chromatography was carried out using a Waters 2767 injector with the collector attached to PDA UV/Vis and SQD mass detectors. An XSelect CSH Prep C18 5 μm OBD 19 mm × 100 mm (part no: 186005421) or XSelect CSH Prep C18 5 μm 10 mm × 100 mm (part no: 186005415) column was used for purification. Final compounds were dried using the Labconco Benchtop FreeZone Freeze-Dry System (4.5 L model). <sup>1</sup>H and proton-decoupled <sup>19</sup>F NMRs were recorded on a Bruker AVANCE-III 500 MHz spectrometer at ambient temperature. Residual protons of CDCl<sub>3</sub>, DMSO-<i>d</i><sub>6</sub>, and CD<sub>3</sub>OD solvents were used as internal references. Spectral data are reported as follows: chemical shift (δ in ppm), multiplicity (br = broad, s = singlet, d = doublet, dd = doublet of doublets, m = multiplet), coupling constants (<i>J</i> in Hz), and proton integration. Compound purity was determined by UV absorbance at 254 nm during tandem LC–MS using a Waters Acquity separations module. All final compounds had a purity of ≥95% as determined using this method. Low-resolution mass spectrometry was conducted in positive ion mode using a Waters Acquity SQD mass spectrometer [electrospray ionization (ESI) source] fitted with a PDA detector. Mobile phase A consisted of 0.1% formic acid in water, while mobile phase B consisted of 0.1% formic acid in acetonitrile. Column 1 (method 1): Acquity UPLC CSH C18 (2.1 × 50 mm, 130 Å, 1.7 μm. Part no. 186005296). The gradient went from 90% to 5% mobile phase A over 1.8 min, maintained at 5% for 0.5 min, and then increased to 90% over 0.2 min for a total run time of 3 min. The column was used with the temperature maintained at 25 °C.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0010.gif" alt="" id="dgr1" /></img></div><div class="NLM_p">A solution of (<i>R</i>)-2-(4-chlorophenyl)azetidine, hydrochloride (67.5 mg, 0.331 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (121 μL, 0.694 mmol) in acetonitrile (0.5 mL) was added dropwise to methyl 2-bromo-2-(4-cyanophenyl)acetate (84 mg, 0.331 mmol) in acetonitrile (1.18 mL) at 0 °C. The mixture was allowed to warm to room temperature and stirred for 5 h. All volatiles were then removed under reduced pressure, and the residue was partitioned between EtOAc and brine. The combined organic layers were then dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated under reduced pressure, and column chromatographed (silica gel, hexanes/EtOAc, 10:0 to 7:3 v/v) to afford methyl 2-((<i>R</i>)-2-(4-chlorophenyl)azetidin-1-yl)-2-(4-cyanophenyl)acetate (72 mg, 64% yield) as a yellow oil. LC–MS method 1, RT = 1.84 and 2.04 min, MS (ESI): <i>m</i>/<i>z</i> = 341.3 [M + 1]<sup>+</sup>, purity (UV<sup>254</sup>) = 99%</div><div class="NLM_p">To the above ester (72 mg, 0.211 mmol) in wet MeOH (10 mL) was added potassium hydroxide (59 mg, 1.06 mmol), and the reaction was stirred at room temperature overnight. All volatiles were evaporated, and the residue was column chromatographed (RP-C18, H<sub>2</sub>O (0.1% v/v FA)/MeCN, 98:2 to 10:90 v/v) to afford 2-2<i>R</i>-((4-chlorophenyl)azetidin-1-yl)-2-(4-cyanophenyl)acetic acid formic acid salt (39 mg, 50% yield) as a white solid. LC–MS method 1, RT = 1.36 and 1.39 min, MS (ESI): <i>m</i>/<i>z</i> = 327.4 [M + 1]<sup>+</sup>, purity (UV<sup>254</sup>) = 99%</div><div class="NLM_p">To the above acid salt in dimethylformamide (5 mL) were added <i>N</i>,<i>N</i>-diisopropylethylamine (121 μL, 0.694 mmol) and 1-[bis(dimethylamino)methylene]-1H-1,2,3-triazolo[4,5-b]pyridinium 3-oxide hexafluorophosphate (69.6 mg, 0.183 mmol) at room temperature. To this mixture was added, with vigorous stirring, ammonium chloride (26.1 mg, 0.488 mmol) in one portion, and the reaction was stirred for 1 h. Water (20 mL) was added to quench the reaction, and the aqueous layer was extracted with EtOAc (3 × 10 mL) to give the desired compound as a mixture of diastereomers. LC–MS method 1, RT = 1.52 min, MS (ESI): <i>m</i>/<i>z</i> = 326.4 [M + 1]<sup>+</sup>, purity (UV<sup>254</sup>) = 99%.</div><div class="NLM_p">This synthetic mixture was separated by preparative chiral SFC using an AD-H column (2 × 25 cm) using 35% MeOH/CO<sub>2</sub> as the eluant at a flow rate of 70 mL·min<sup>–1</sup> and 100 bar pressure. Detection by UV at 220 nm. Injection volume was 1.0 mL of 5 mg·mL<sup>–1</sup> in MeOH/dichloromethane. Analytical chiral SFC (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) used AD-H column (0.46 × 25 cm) with 300% MeOH/CO<sub>2</sub> as the eluant at a flow rate of 3 mL·min<sup>–1</sup> and 120 bar pressure. Detection by UV at 220 nm.</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 2<i>R</i>-(2-(4-Chlorophenyl)azetidin-1-yl)-2<i>R</i>-(4-cyanophenyl)acetamide (29 mg)</h4><div class="NLM_p last">Chiral SFC retention time 2.6 min. <sup>1</sup>H NMR (500 MHz, MeOD): δ 7.38 (d, <i>J</i> = 8.3 Hz, 2H), 7.33 (d, <i>J</i> = 8.4 Hz, 2H), 7.12 (d, <i>J</i> = 8.6 Hz, 2H), 7.08 (d, <i>J</i> = 8.6 Hz, 2H), 4.11 (t, <i>J</i> = 8.2 Hz, 1H), 4.07 (s, 1H), 3.73–3.69 (m, 1H), 3.13 (dt, <i>J</i> = 9.3, 8.0 Hz, 1H), 2.36 (dtd, <i>J</i> = 10.3, 8.0, 2.3 Hz, 1H), 2.24–2.11 (m, 1H).</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 2<i>R</i>-(2-(4-Chlorophenyl)azetidin-1-yl)-2<i>S</i>-(4-cyanophenyl)acetamide (13 mg)</h4><div class="NLM_p last">Chiral SFC retention time 4.1 min. <sup>1</sup>H NMR (500 MHz, MeOD): δ 7.72 (d, <i>J</i> = 8.3 Hz, 2H), 7.63 (d, <i>J</i> = 8.3 Hz, 2H), 7.54 (d, <i>J</i> = 8.4 Hz, 2H), 7.34 (d, <i>J</i> = 8.4 Hz, 2H), 4.27 (t, <i>J</i> = 8.3 Hz, 1H), 4.11 (s, 1H), 3.27–3.20 (m, 1H), 2.89–2.80 (m, 1H), 2.32 (dtd, <i>J</i> = 10.4, 7.9, 2.4 Hz, 1H), 2.25–2.15 (m, 1H).</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Similarly Prepared Were the Diastereoisomers from (<i>S</i>)-2-(4-Chlorophenyl)azetidine</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 2<i>S</i>-(2-(4-Chlorophenyl)azetidin-1-yl)-2<i>S</i>-(4-cyanophenyl)acetamide (29 mg)</h4><div class="NLM_p last">Chiral SFC retention time 3.1 min. <sup>1</sup>H NMR (500 MHz, MeOD): δ 7.36 (d, <i>J</i> = 8.3 Hz, 2H), 7.31 (d, <i>J</i> = 8.4 Hz, 2H), 7.10 (d, <i>J</i> = 8.6 Hz, 2H), 7.07 (d, <i>J</i> = 8.6 Hz, 2H), 4.09 (t, <i>J</i> = 8.2 Hz, 1H), 4.05 (s, 1H), 3.72–3.66 (m, 1H), 3.11 (dt, <i>J</i> = 9.4, 8.0 Hz, 1H), 2.34 (dtd, <i>J</i> = 10.3, 8.0, 2.3 Hz, 1H), 2.20–2.10 (m, 1H).</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 2<i>S</i>-(2-(4-Chlorophenyl)azetidin-1-yl)-2<i>R</i>-(4-cyanophenyl)acetamide (7 mg)</h4><div class="NLM_p last">Chiral SFC retention time 4.9 min. <sup>1</sup>H NMR (500 MHz, MeOD): δ 7.72 (d, <i>J</i> = 8.4 Hz, 2H), 7.63 (d, <i>J</i> = 8.3 Hz, 2H), 7.54 (d, <i>J</i> = 8.4 Hz, 2H), 7.34 (d, <i>J</i> = 8.4 Hz, 2H), 4.27 (t, <i>J</i> = 8.3 Hz, 1H), 4.11 (s, 1H), 3.28–3.20 (m, 1H), 2.89–2.80 (m, 1H), 2.32 (dtd, <i>J</i> = 10.4, 8.0, 2.4 Hz, 1H), 2.24–2.15 (m, 1H).</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Chemical Synthesis of Compounds <b>3c</b> and <b>3d</b></h3><div class="NLM_p last">All starting materials were commercially procured and were used without further purification unless specified. Analytical LC–MS data for all compounds were acquired using an Agilent 6110 Series system with the UV detector set to 220 nm. Samples were injected (<10 μL) onto an Agilent Eclipse Plus 4.6 × 50 mm, 1.8 μm, C18 column at room temperature. Mobile phases A (H<sub>2</sub>O + 0.1% acetic acid) and B (MeOH + 0.1% acetic acid) were used with a linear gradient from 10 to 100% B in 5.0 min, followed by a flush at 100% B for another 2 min with a flow rate of 1.0 mL/min. Mass spectroscopy (MS) data were acquired in positive ion mode using an Agilent 6110 single quadrupole mass spectrometer with an ESI source. Nuclear magnetic resonance (NMR) spectra were recorded on a Varian Mercury spectrometer at 400 MHz for proton (<sup>1</sup>H NMR); chemical shifts are reported in ppm (δ) relative to residual protons in deuterated solvent peaks. Normal phase column chromatography was performed with a Teledyne Isco CombiFlashRf using silica RediSepRf columns with the UV detector set to 220 nm and 254 nm. The mobile phases used are indicated for each compound. Reverse phase column chromatography was performed with a Teledyne Isco CombiFlashRf 200 using C18 RediSepRf Gold columns with the UV detector set to 220 nm and 254 nm. Mobile phases of A (H<sub>2</sub>O + 0.1% TFA) and B (MeCN) were used with default column gradients. Preparative HPLC was performed using an Agilent Prep 1200 series with the UV detector set to 220 nm and 254 nm. Samples were injected onto a Phenomenex Luna 250 × 30 mm, 5 μm, C18 column at room temperature. Mobile phases of A (H<sub>2</sub>O + 0.1% TFA) and B (MeOH or MeCN) were used with a flow rate of 40 mL/min. A general gradient of 0–15 min increasing from 10 to 100% B followed by a 100% B flush for another 5 min was used. Small variations in this purification method were made as needed to achieve ideal separation for each compound. Analytical LC–MS (at 254 nm) and NMR were used to establish the purity of targeted compounds. All compounds that were evaluated in biochemical and biophysical assays had >95% purity as determined by <sup>1</sup>H NMR and LC–MS.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Synthesis of Compound <b>3c</b></h3><div class="NLM_p"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0011.gif" alt="" id="dgr2" /></img>To a flask equipped with a stir bar were added 4-fluorobenzaldehyde (0.25 g, 0.21 mL, 1 equiv, 2.0 mmol) and methanol (10 mL). The flask was sealed with a septum, and isopropylamine (0.59 g, 0.86 mL, 5 equiv, 10 mmol) was introduced <i>via</i> a syringe. The mixture was stirred at room temperature for 3 h to allow for imine formation. Next, the flask was cooled in an ice bath, and NaBH<sub>4</sub> (0.15 g, 2 equiv, 4.0 mmol) was added in one portion. The reaction was allowed to come to room temperature with stirring overnight. The next day, the reaction was diluted with water and extracted three times with ethyl acetate. The combined organic layers were washed three times with water and once with brine, then dried over sodium sulfate, and concentrated to a clear oil. The oil was taken up in ether (10 mL) and cooled in an ice bath, and TFA (0.34 g, 0.23 mL, 1.5 equiv, 3.0 mmol) was added with vigorous stirring. The voluminous white precipitate formed was filtered off, rinsed with additional ether, and air-dried to provide <i>N</i>-(4-fluorobenzyl)propan-2-amine 2,2,2-trifluoroacetate (111.5 mg, 396.4 μmol, 20%) as a fluffy white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.54 (dd, <i>J</i> = 8.6, 5.4 Hz, 2H), 7.20 (t, <i>J</i> = 8.7 Hz, 2H), 4.20 (s, 2H), 3.44 (hept, <i>J</i> = 6.5 Hz, 1H), 1.39 (d, <i>J</i> = 6.6 Hz, 6H). LC–MS (ESI, +ve mode) expected <i>m</i>/<i>z</i> for C<sub>10</sub>H<sub>15</sub>FN [M + H]<sup>+</sup> 168.12, found 168.20.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0012.gif" alt="" id="dgr3" /></img></div><div class="NLM_p">To a vial containing <i>N</i>-(4-fluorobenzyl)propan-2-amine 2,2,2-trifluoroacetate (75 mg, 1.1 equiv, 0.27 mmol) were added acetonitrile (1 mL) and triethylamine (74 mg, 0.10 mL, 3 equiv, 0.73 mmol). The vial was cooled in an ice bath, and 4-cyanobenzoyl chloride (40 mg, 1 equiv, 0.24 mmol) was added in one portion. The reaction was allowed to come to room temperature with stirring overnight. The next day, the reaction was diluted with water and extracted three times with ethyl acetate. The combined organic layers were washed once with 0.5 M citric acid, once with water, once with saturated sodium bicarbonate, and once with brine, dried over sodium sulfate, and concentrated to a white solid. Normal phase chromatography over silica gel (0-50% ethyl acetate in hexanes) afforded compound <b>3c</b> (47 mg, 0.16 mmol, 65%) as a clear residue that solidified on standing to a white solid.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) complicated by rotamers. δ 7.79–7.65 (br s, 2H), 7.65–7.38 (br s, 2H), 7.37–7.27 (br s, 2H), 7.05–6.93 (m, 2H), 4.62–4.26 (br m, 2H), 4.05–3.81 (br s, 1H), 1.33–1.01 (br s, 6H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 170.3, 162.0 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 245.1 Hz), 141.6, 134.6, 132.7, 129.0, 127.0, 118.2, 115.5 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 21.7 Hz), 113.3, 51.1, 43.2, 21.6. <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>): δ −114.6 (minor rotamer), −115.7 (major rotamer). LC–MS (ESI, +ve mode) expected <i>m</i>/<i>z</i> for C<sub>18</sub>H<sub>18</sub>FN<sub>2</sub>O [M + H]<sup>+</sup> 297.14, found 297.10.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Synthesis of Compound <b>3d</b></h3><div class="NLM_p"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0013.gif" alt="" id="dgr4" /></img>To a flame-dried microwave vial equipped with a stir bar were added 4-fluorobenzaldehyde (100 mg, 86.4 μL, 0.806 mmol) and cyclobutanamine (57.3 mg, 68.8 μL, 0.806 mmol), followed by acetic acid and THF (ratio 1:4; 4mL). The sealed vial was irradiated in the microwave at 60 °C (250 W) for 10 min. Upon cooling, SiliaBond cyanoborohydride was added (50.6 mg, 0.806 mmol) and stirred at room temperature for 10 min. The mixture was irradiated in the microwave at 120 °C (250 W) for 10 min. The crude mixture was filtered, concentrated under vacuum, and used in the next step without further purification.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0014.gif" alt="" id="dgr5" /></img></div><div class="NLM_p">To a flame-dried microwave vial equipped with a stir bar were added the <i>N</i>-(4-fluorobenzyl)cyclobutanamine synthesized previously, 4-cyanobenzoyl chloride (100 mg, 0.604 mmol), and acetonitrile (2 mL). The sealed vial was irradiated in the microwave at 120 °C for 60 min (250 W). Once the reaction was complete, the volatiles were evaporated. The crude mixture was first purified by normal phase gradient column chromatography (0–100% ethyl acetate in hexanes). The desired fractions were concentrated, redissolved in methanol (1 mL), and then purified further by preparative-HPLC [(H<sub>2</sub>O + 0.1% TFA)/MeCN] to afford compound <b>3d</b> (56.9 mg, 184 μmol, 30% over two steps) as a colorless gum.</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) complicated by rotamers. δ 7.81–7.59 (m, 2H), 7.57–7.38 (m, 2H), 7.23 (br s, 2H), 7.07–6.95 (m, 2H), 4.90–4.00 (multiple br signals, 3H), 2.18–1.33 (multiple br signals, 6H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 162.08 (d, <sup>1</sup><i>J</i><sub>CF</sub> = 245.7 Hz), 141.27, 134.02, 132.49, 128.32, 127.44, 118.21, 115.73 (d, <sup>2</sup><i>J</i><sub>CF</sub> = 21.5 Hz), 113.60, 77.48, 77.16, 76.84, 53.71, 44.71, 29.43, 14.60. NB—carbonyl carbon is too weak to be detected. <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>): δ −115.21 (minor rotamer), −116.12 (major rotamer). NB—trace TFA in spectrum. LC–MS (ESI, +ve mode) expected <i>m/z</i> for C<sub>19</sub>H<sub>18</sub>FN<sub>2</sub>O [M + H]<sup>+</sup> 309.14, found 309.20</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01768" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88580" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88580" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01768?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01768</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Experimental details, 1H NMR spectra of synthesized compounds, 3D docked conformations of virtual screening hits, and compound purity: analytical spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular string formula and activity of all compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">X-ray crystallographic data and refinement statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf">jm0c01768_si_001.pdf (3.53 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_002.pdb">jm0c01768_si_002.pdb (411.73 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_003.csv">jm0c01768_si_003.csv (3.02 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01768" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84628" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84628" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthieu Schapira</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1047-3309" title="Orcid link">http://orcid.org/0000-0002-1047-3309</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9ef3ffeaeaf6f7fbebb0edfdf6ffeef7ecffdeebeaf1ecf1f0eaf1b0fdff"><span class="__cf_email__" data-cfemail="523f3326263a3b37277c21313a33223b20331227263d203d3c263d7c3133">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abdellah Allali-Hassani</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span>Present Address:
                        Incyte, 1801 Augustine Cut-Off, Wilmington, DE 19803, USA</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5031313c3c313c3938312323313e3910393e332924357e333f3d"><span class="__cf_email__" data-cfemail="751414191914191c1d140606141b1c351c1b160c01105b161a18">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renato Ferreira de Freitas</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span>Present Address:
                        Centro de Ciências Naturais e Humanas, Universidade Federal do ABC, Rua Arcturus 3, São Bernardo do Campo, SP 09606-070, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanli Liu</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span>Present Address:
                        College of Pharmaceutical Sciences, Soochow University, Suzhou, Jiangsu 215123, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magdalena M. Szewczyk</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Naimee Mehta</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David McLeod</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlos Zepeda-Velázquez</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Dilworth</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ronan P. Hanley</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisa Gibson</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rima Al-Awar</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lindsey I. James</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrative Chemical Biology and Drug Discovery, Division of Chemical
Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6034-7116" title="Orcid link">http://orcid.org/0000-0002-6034-7116</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Princess
Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinrong Min</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Physiology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.F.d.F., Y.L., and A.A.-H. equal contributors. Experimental design: R.F.d.F., A.A.-H., M.S., M.V., N.M., L.I.J., Y.L., J.M., M.M.S., D.D., D.B.-L.; data generation: R.F.d.F., A.A.-H., N.M., Y.L., F.L., D.M., M.M.S., D.D., R.P.H., E.G., D.M., C.Z.-V.; data analysis: R.F.d.F., A.A.-H., M.S., M.V., N.M., L.I.J., M.M.S., D.D., D.B.-L., P.J.B., C.H.A.; supervision: M.S., M.V., J.M., L.I.J., D.B.-L., C.Z.-V.; manuscript writing: R.F.d.F., M.S., A.A.-H., D.D., M.V., L.I.J.; funding: C.H.A., L.I.J., R.A.-A.; manuscript review: all.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63565" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63565" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The SGC is a registered charity (no. 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada through Ontario Genomics Institute, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck & Co., Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and the Wellcome Trust. We received additional funding from the Collaborative Research and Training Experience grant (Aled Edwards 432008-2013) from the Natural Sciences and Engineering Research Council of Canada. We thank BioSolveIT GmbH for supplying an evaluation copy of the Ftrees. We are grateful to OpenEye Scientific Software, Inc. for providing us with an academic license for their software. M.S. gratefully acknowledges funding from NSERC (grants RGPIN-2019-04416 and ALLRP 555329-20). This work was further supported by the National Cancer Institute, NIH (grant R01CA242305 to L.I.J.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term"><i>K</i><sub>d</sub></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">H3K36</td><td class="NLM_def"><p class="first last">lysine 36 of histone 3</p></td></tr><tr><td class="NLM_term">DSLS</td><td class="NLM_def"><p class="first last">differential static light scattering</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">Δ<i>T</i><sub>agg</sub></td><td class="NLM_def"><p class="first last">difference in aggregation temperature</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33681" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33681" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 26 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zee, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kioi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauring, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span> <span> </span><span class="NLM_article-title">NSD2 Links Dimethylation of Histone H3 at Lysine 36 to Oncogenic Programming</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2011.08.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1016%2Fj.molcel.2011.08.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=22099308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2ku7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=609-620&author=A.+J.+Kuoauthor=P.+Cheungauthor=K.+Chenauthor=B.+M.+Zeeauthor=M.+Kioiauthor=J.+Lauringauthor=Y.+Xiauthor=B.+H.+Parkauthor=X.+Shiauthor=B.+A.+Garciaauthor=W.+Liauthor=O.+Gozani&title=NSD2+Links+Dimethylation+of+Histone+H3+at+Lysine+36+to+Oncogenic+Programming&doi=10.1016%2Fj.molcel.2011.08.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">NSD2 Links Dimethylation of Histone H3 at Lysine 36 to Oncogenic Programming</span></div><div class="casAuthors">Kuo, Alex J.; Cheung, Peggie; Chen, Kaifu; Zee, Barry M.; Kioi, Mitomu; Lauring, Josh; Xi, Yuanxin; Park, Ben Ho; Shi, Xiaobing; Garcia, Benjamin A.; Li, Wei; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-620</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The histone lysine methyltransferase NSD2 (MMSET/WHSC1) is implicated in diverse diseases and commonly overexpressed in multiple myeloma due to a recurrent t(4;14) chromosomal translocation.  However, the precise catalytic activity of NSD2 is obscure, preventing progress in understanding how this enzyme influences chromatin biol. and myeloma pathogenesis.  Here, we show that dimethylation of histone H3 at lysine 36 (H3K36me2) is the principal chromatin-regulatory activity of NSD2.  Catalysis of H3K36me2 by NSD2 is sufficient for gene activation.  In t(4;14)-pos. myeloma cells, the normal genome-wide and gene-specific distribution of H3K36me2 is obliterated, creating a chromatin landscape that selects for a transcription profile favorable for myelomagenesis.  Catalytically active NSD2 confers xenograft tumor formation upon t(4;14)-neg. cells and promotes oncogenic transformation of primary cells in an H3K36me2-dependent manner.  Together, our findings establish H3K36me2 as the primary product generated by NSD2 and demonstrate that genomic disorganization of this canonical chromatin mark by NSD2 initiates oncogenic programming.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJbJf7X8UfurVg90H21EOLACvtfcHk0lhii5e34NWnsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2ku7vL&md5=f023db790f506bcb9472bce31dd2e48a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2011.08.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2011.08.042%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DA.%2BJ.%26aulast%3DCheung%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DZee%26aufirst%3DB.%2BM.%26aulast%3DKioi%26aufirst%3DM.%26aulast%3DLauring%26aufirst%3DJ.%26aulast%3DXi%26aufirst%3DY.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DNSD2%2520Links%2520Dimethylation%2520of%2520Histone%2520H3%2520at%2520Lysine%252036%2520to%2520Oncogenic%2520Programming%26jtitle%3DMol.%2520Cell%26date%3D2011%26volume%3D44%26spage%3D609%26epage%3D620%26doi%3D10.1016%2Fj.molcel.2011.08.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vougiouklakis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saloura, V.</span></span> <span> </span><span class="NLM_article-title">The NSD Family of Protein Methyltransferases in Human Cancer</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.2217/epi.15.32</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.2217%2Fepi.15.32" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=25942451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cqsb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=863-874&author=T.+Vougiouklakisauthor=R.+Hamamotoauthor=Y.+Nakamuraauthor=V.+Saloura&title=The+NSD+Family+of+Protein+Methyltransferases+in+Human+Cancer&doi=10.2217%2Fepi.15.32"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The NSD family of protein methyltransferases in human cancer</span></div><div class="casAuthors">Vougiouklakis, Theodore; Hamamoto, Ryuji; Nakamura, Yusuke; Saloura, Vassiliki</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">863-874</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  The NSD family of protein lysine methyltransferases consists of NSD1, NSD2/WHSC1/MMSET and NSD3/WHSC1L1.  NSD2 haploinsufficiency causes Wolf-Hirschhorn syndrome, while NSD1 mutations lead to the Sotos syndrome.  Recently, a no. of studies showed that the NSD methyltransferases were overexpressed, amplified or somatically mutated in multiple types of cancer, suggesting their crit. role in cancer.  These enzymes methylate specific lysine residues on histone tails and their dysfunction results in epigenomic aberrations which play a fundamental role in oncogenesis.  Furthermore, NSD1 was also reported to methylate a nonhistone protein substrate, RELA/p65 subunit of NF-κB, implying its regulatory function through nonhistone methylation pathways.  In this review, we summarize the current research regarding the role of the NSD family proteins in cancer and underline their potential as targets for novel cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUEZ0yw91nA7Vg90H21EOLACvtfcHk0lglGaAhO8MWWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cqsb%252FK&md5=68993bcfd787c4391e65ae2aaba2ecdd</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2217%2Fepi.15.32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.15.32%26sid%3Dliteratum%253Aachs%26aulast%3DVougiouklakis%26aufirst%3DT.%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DSaloura%26aufirst%3DV.%26atitle%3DThe%2520NSD%2520Family%2520of%2520Protein%2520Methyltransferases%2520in%2520Human%2520Cancer%26jtitle%3DEpigenomics%26date%3D2015%26volume%3D7%26spage%3D863%26epage%3D874%26doi%3D10.2217%2Fepi.15.32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larratt, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mant, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belch, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilarski, L. M.</span></span> <span> </span><span class="NLM_article-title">In multiple myeloma, t(4;14)(p16;q32) is an adverse prognostic factor irrespective of FGFR3 expression</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">1520</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.1182/blood-2002-06-1675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1182%2Fblood-2002-06-1675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=12393535" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1agu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2003&pages=1520-1529&author=J.+J.+Keatsauthor=T.+Reimanauthor=C.+A.+Maxwellauthor=B.+J.+Taylorauthor=L.+M.+Larrattauthor=M.+J.+Mantauthor=A.+R.+Belchauthor=L.+M.+Pilarski&title=In+multiple+myeloma%2C+t%284%3B14%29%28p16%3Bq32%29+is+an+adverse+prognostic+factor+irrespective+of+FGFR3+expression&doi=10.1182%2Fblood-2002-06-1675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">In multiple myeloma, t(4;14)(p16;q32) is an adverse prognostic factor irrespective of FGFR3 expression</span></div><div class="casAuthors">Keats, Jonathan J.; Reiman, Tony; Maxwell, Christopher A.; Taylor, Brian J.; Larratt, Loree M.; Mant, Michael J.; Belch, Andrew R.; Pilarski, Linda M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1520-1529</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">This study analyzed the frequency and clin. significance of t(4;14)(p16;q32) in multiple myeloma (MM) among 208 patients with MM and 52 patients with monoclonal gammopathy of undetd. significance (MGUS); diagnosed between 1994 and 2001.  Patients with the translocation were identified using reverse transcription-polymerase chain reaction (RT-PCR) to detect hybrid Ig heavy chain (IgH)-MMSET transcripts from the der(4) chromosome.  We found 31 (14.9%) t(4;14)+ MM patients and 1 (1.9%) t(4;14)+ MGUS patient.  IgH-MMSET hybrid transcripts were detected in bone marrow (BM) and blood.  Breakpoint anal. revealed that 67.7% of t(4;14)+ patients expressed hybrid transcripts potentially encoding full-length MMSET, whereas the remainder lacked one or more amino terminal exons.  Expression of fibroblast growth factor receptor 3 (FGFR3), presumptively dysregulated on der(14), was detected by RT-PCR in only 23 of 31 (74%) patients with t(4;14)+ MM.  Patients lacking FGFR3 expression also lacked detectable der(14) products.  Longitudinal anal. of 53 MM patients with multiple BM and blood samples showed that, over time, BM from t(4;14)+ patients remained pos. and that t(4;14)- patients did not acquire the translocation.  IgH-MMSET hybrid transcripts and FGFR3 transcripts disappeared from blood during response to therapy.  No correlation was obsd. between the occurrence of t(4;14) and known prognostic indicators.  However, we find the t(4;14) translocation predicts for poor survival (P = .006; median, 644 days vs 1288 days; hazard ratio [HR], 2.0), even in FGFR3 nonexpressors (P = .003).  The presence of t(4;14) is also predictive of poor response to first-line chemotherapy (P = .05).  These results indicate a significant clin. impact of the t(4;14) translocation in MM that is independent of FGFR3 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2jr5_p4QDcLVg90H21EOLACvtfcHk0lglGaAhO8MWWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1agu7c%253D&md5=868219f535ec8ff62c7ca660f07cc1c4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1182%2Fblood-2002-06-1675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2002-06-1675%26sid%3Dliteratum%253Aachs%26aulast%3DKeats%26aufirst%3DJ.%2BJ.%26aulast%3DReiman%26aufirst%3DT.%26aulast%3DMaxwell%26aufirst%3DC.%2BA.%26aulast%3DTaylor%26aufirst%3DB.%2BJ.%26aulast%3DLarratt%26aufirst%3DL.%2BM.%26aulast%3DMant%26aufirst%3DM.%2BJ.%26aulast%3DBelch%26aufirst%3DA.%2BR.%26aulast%3DPilarski%26aufirst%3DL.%2BM.%26atitle%3DIn%2520multiple%2520myeloma%252C%2520t%25284%253B14%2529%2528p16%253Bq32%2529%2520is%2520an%2520adverse%2520prognostic%2520factor%2520irrespective%2520of%2520FGFR3%2520expression%26jtitle%3DBlood%26date%3D2003%26volume%3D101%26spage%3D1520%26epage%3D1529%26doi%3D10.1182%2Fblood-2002-06-1675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huether, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhang, H.-e. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Englund, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert McDonald, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumount, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caponigro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barretina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullighan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downing, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span> <span> </span><span class="NLM_article-title">Global Chromatin Profiling Reveals NSD2 Mutations in Pediatric Acute Lymphoblastic Leukemia</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1386</span>– <span class="NLM_lpage">1391</span>, <span class="refDoi"> DOI: 10.1038/ng.2777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fng.2777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=24076604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFaksLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=1386-1391&author=J.+D.+Jaffeauthor=Y.+Wangauthor=H.+M.+Chanauthor=J.+Zhangauthor=R.+Huetherauthor=G.+V.+Kryukovauthor=H.-e.+C.+Bhangauthor=J.+E.+Taylorauthor=M.+Huauthor=N.+P.+Englundauthor=F.+Yanauthor=Z.+Wangauthor=E.+Robert+McDonaldauthor=L.+Weiauthor=J.+Maauthor=J.+Eastonauthor=Z.+Yuauthor=R.+deBeaumountauthor=V.+Gibajaauthor=K.+Venkatesanauthor=R.+Schlegelauthor=W.+R.+Sellersauthor=N.+Keenauthor=J.+Liuauthor=G.+Caponigroauthor=J.+Barretinaauthor=V.+G.+Cookeauthor=C.+Mullighanauthor=S.+A.+Carrauthor=J.+R.+Downingauthor=L.+A.+Garrawayauthor=F.+Stegmeier&title=Global+Chromatin+Profiling+Reveals+NSD2+Mutations+in+Pediatric+Acute+Lymphoblastic+Leukemia&doi=10.1038%2Fng.2777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Global chromatin profiling reveals NSD2 mutations in pediatric acute lymphoblastic leukemia</span></div><div class="casAuthors">Jaffe, Jacob D.; Wang, Yan; Chan, Ho Man; Zhang, Jinghui; Huether, Robert; Kryukov, Gregory V.; Bhang, Hyo-eun C.; Taylor, Jordan E.; Hu, Min; Englund, Nathan P.; Yan, Feng; Wang, Zhaofu; McDonald, E. Robert; Wei, Lei; Ma, Jing; Easton, John; Yu, Zhengtian; de Beaumount, Rosalie; Gibaja, Veronica; Venkatesan, Kavitha; Schlegel, Robert; Sellers, William R.; Keen, Nicholas; Liu, Jun; Caponigro, Giordano; Barretina, Jordi; Cooke, Vesselina G.; Mullighan, Charles; Carr, Steven A.; Downing, James R.; Garraway, Levi A.; Stegmeier, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1386-1391</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic dysregulation is an emerging hallmark of cancers.  We developed a high-information-content mass spectrometry approach to profile global histone modifications in human cancers.  When applied to 115 lines from the Cancer Cell Line Encyclopedia, this approach identified distinct mol. chromatin signatures.  One signature was characterized by increased histone 3 lysine 36 (H3K36) dimethylation, exhibited by several lines harboring translocations in NSD2, which encodes a methyltransferase.  A previously unknown NSD2 p.  Glu1099Lys (p.E1099K) variant was identified in nontranslocated acute lymphoblastic leukemia (ALL) cell lines sharing this signature.  Ectopic expression of the variant induced a chromatin signature characteristic of NSD2 hyperactivation and promoted transformation.  NSD2 knockdown selectively inhibited the proliferation of NSD2-mutant lines and impaired the in vivo growth of an NSD2-mutant ALL xenograft.  Sequencing anal. of >1,000 pediatric cancer genomes identified the NSD2 p.  E1099K alteration in 14% of t(12;21) ETV6-RUNX1-contg. ALLs.  These findings identify NSD2 as a potential therapeutic target for pediatric ALL and provide a general framework for the functional annotation of cancer epigenomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzi5jFEwRNMrVg90H21EOLACvtfcHk0lh4KSdoIZdwbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFaksLjE&md5=386178a58572ae3180297c3d92d012fa</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fng.2777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2777%26sid%3Dliteratum%253Aachs%26aulast%3DJaffe%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChan%26aufirst%3DH.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHuether%26aufirst%3DR.%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DBhang%26aufirst%3DH.-e.%2BC.%26aulast%3DTaylor%26aufirst%3DJ.%2BE.%26aulast%3DHu%26aufirst%3DM.%26aulast%3DEnglund%26aufirst%3DN.%2BP.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DRobert%2BMcDonald%26aufirst%3DE.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DEaston%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DdeBeaumount%26aufirst%3DR.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DSchlegel%26aufirst%3DR.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCaponigro%26aufirst%3DG.%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DCooke%26aufirst%3DV.%2BG.%26aulast%3DMullighan%26aufirst%3DC.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DDowning%26aufirst%3DJ.%2BR.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DStegmeier%26aufirst%3DF.%26atitle%3DGlobal%2520Chromatin%2520Profiling%2520Reveals%2520NSD2%2520Mutations%2520in%2520Pediatric%2520Acute%2520Lymphoblastic%2520Leukemia%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D1386%26epage%3D1391%26doi%3D10.1038%2Fng.2777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oyer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezponda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allegretta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okot-Kotber, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackerell, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelleher, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, R.</span></span> <span> </span><span class="NLM_article-title">Point Mutation E1099K in MMSET/NSD2 Enhances Its Methyltranferase Activity and Leads to Altered Global Chromatin Methylation in Lymphoid Malignancies</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">198</span>– <span class="NLM_lpage">201</span>, <span class="refDoi"> DOI: 10.1038/leu.2013.204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fleu.2013.204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=23823660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVejsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=198-201&author=J.+A.+Oyerauthor=X.+Huangauthor=Y.+Zhengauthor=J.+Shimauthor=T.+Ezpondaauthor=Z.+Carpenterauthor=M.+Allegrettaauthor=C.+I.+Okot-Kotberauthor=J.+P.+Patelauthor=A.+Melnickauthor=R.+L.+Levineauthor=A.+Ferrandoauthor=A.+D.+Mackerellauthor=N.+L.+Kelleherauthor=J.+D.+Lichtauthor=R.+Popovic&title=Point+Mutation+E1099K+in+MMSET%2FNSD2+Enhances+Its+Methyltranferase+Activity+and+Leads+to+Altered+Global+Chromatin+Methylation+in+Lymphoid+Malignancies&doi=10.1038%2Fleu.2013.204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Point mutation E1099K in MMSET/NSD2 enhances its methyltranferase activity and leads to altered global chromatin methylation in lymphoid malignancies</span></div><div class="casAuthors">Oyer, J. A.; Huang, X.; Zheng, Y.; Shim, J.; Ezponda, T.; Carpenter, Z.; Allegretta, M.; Okot-Kotber, C. I.; Patel, J. P.; Melnick, A.; Levine, R. L.; Ferrando, A.; MacKerell, A. D.; Kelleher, N. L.; Licht, J. D.; Popovic, R.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">198-201</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acute lymphoblastic leukemia (ALL) is a neoplasm of lymphocytic progenitors and represents the most common malignancy in children.  The disease is frequently characterized by large scale chromosomal alterations including hyper- or hypodiploidy, translocations, deletions and insertions.  We report on a novel recurrent mutation affecting the catalytic domain of MMSET protein.  This mutation is most predominant in pediatric lymphoid malignancies, although its involvement in other tumors cannot be ruled out.  Our data suggest that, similar to t(4;14) translocations in multiple myeloma, E1099K mutation within MMSET enhances global H3K36 dimethylationin lymphoid cells.  This is likely due to a charge alteration within the histone-binding pocket of MMSET and putative disruption of histone-enzyme ionic interactions.  In addn., E1099K mutation and aberrantly high H3K36 methylation lead to a genome-wide decrease in H3K27me3, possibly affecting expression of a no. of genes important for normal lymphoid development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr682-2ULMT7Vg90H21EOLACvtfcHk0lh4KSdoIZdwbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVejsg%253D%253D&md5=c6ce681c77a444ac44b9a66f8855034c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fleu.2013.204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2013.204%26sid%3Dliteratum%253Aachs%26aulast%3DOyer%26aufirst%3DJ.%2BA.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DEzponda%26aufirst%3DT.%26aulast%3DCarpenter%26aufirst%3DZ.%26aulast%3DAllegretta%26aufirst%3DM.%26aulast%3DOkot-Kotber%26aufirst%3DC.%2BI.%26aulast%3DPatel%26aufirst%3DJ.%2BP.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DFerrando%26aufirst%3DA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DKelleher%26aufirst%3DN.%2BL.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DPopovic%26aufirst%3DR.%26atitle%3DPoint%2520Mutation%2520E1099K%2520in%2520MMSET%252FNSD2%2520Enhances%2520Its%2520Methyltranferase%2520Activity%2520and%2520Leads%2520to%2520Altered%2520Global%2520Chromatin%2520Methylation%2520in%2520Lymphoid%2520Malignancies%26jtitle%3DLeukemia%26date%3D2014%26volume%3D28%26spage%3D198%26epage%3D201%26doi%3D10.1038%2Fleu.2013.204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span>; <span class="NLM_string-name">Lnu, S.</span>; <span class="NLM_string-name">Cao, Q.</span>; <span class="NLM_string-name">Asangani, I.</span></span> <span> </span><span class="NLM_article-title">Compositions and Methods for Inhibiting MMSET</span>. U.S. Patent <span class="NLM_patent">20,110,207,198 A1</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=A.+M.+Chinnaiyan&author=S.+Lnu&author=Q.+Cao&author=I.+Asangani&title=Compositions+and+Methods+for+Inhibiting+MMSET"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DCompositions%2520and%2520Methods%2520for%2520Inhibiting%2520MMSET%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morishita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mevius, D. E. H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Luccio, E.</span></span> <span> </span><span class="NLM_article-title">BIX-01294 Inhibits Oncoproteins NSD1, NSD2 and NSD3</span>. <i>Med. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2038</span>– <span class="NLM_lpage">2047</span>, <span class="refDoi"> DOI: 10.1007/s00044-017-1909-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1007%2Fs00044-017-1909-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Gjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2017&pages=2038-2047&author=M.+Morishitaauthor=D.+E.+H.+F.+Meviusauthor=Y.+Shenauthor=S.+Zhaoauthor=E.+di+Luccio&title=BIX-01294+Inhibits+Oncoproteins+NSD1%2C+NSD2+and+NSD3&doi=10.1007%2Fs00044-017-1909-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">BIX-01294 inhibits oncoproteins NSD1, NSD2 and NSD3</span></div><div class="casAuthors">Morishita, Masayo; Mevius, Damiaan E. H. F.; Shen, Yunpeng; Zhao, Shuyu; di Luccio, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2038-2047</span>CODEN:
                <span class="NLM_cas:coden">MCREEB</span>;
        ISSN:<span class="NLM_cas:issn">1054-2523</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">NSD1, NSD2/MMSET/WHSC1, and NSD3/WHSC1L1 make up the nuclear receptor-binding Su(var)3-9, Enhancer-of-zeste and Trithorax domain family of histone methyltransferases, which are essential for regulating the chromatin.  Abnormalities in histone methyltransferases function are increasingly being found to be assocd. with numerous pathol. conditions, including carcinogenesis and tumor progression.  NSD1, NSD2, and NSD3 are oncoproteins aberrantly expressed in numerous cancers in which selective inhibition may offer therapeutic opportunities, esp. in cases of conditions with poor prognoses such as multiple myeloma.  Histone methyltransferase inhibitors are scarce and selective inhibitors are being explored.  NSD inhibitors are urgently needed.  BIX-01294 is a G9a-like protein/G9a histone methyltransferase inhibitor commonly used to modulate H3K9 methylation in the context of cell reprogramming and cancer biol. with in vitro IC50 values of 0.7 and 1.9 μM, resp.  Since the catalytic Su(var)3-9, Enhancer-of-zeste and Trithorax domain of the NSDs is highly related to that of G9a-like protein and G9a, we investigated the potentially differential NSDs inhibition by BIX-01294.  In this study, we identified BIX-01294 as an NSD in vitro hit compd. that differentially inhibits H3K36 methylation by NSD1, NSD2, and NSD3 with IC50 values of 40 ∼ 112 μM.  Furthermore, we investigated the mol. basis of inhibition by BIX-01294 on the NSDs and discuss the prospects of BIX-01294 derivs. for selective NSD inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr29-siRHAlArVg90H21EOLACvtfcHk0lh4KSdoIZdwbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Gjsbo%253D&md5=e7e463c62224cf3afdd199414bb51564</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00044-017-1909-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00044-017-1909-7%26sid%3Dliteratum%253Aachs%26aulast%3DMorishita%26aufirst%3DM.%26aulast%3DMevius%26aufirst%3DD.%2BE.%2BH.%2BF.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3Ddi%2BLuccio%26aufirst%3DE.%26atitle%3DBIX-01294%2520Inhibits%2520Oncoproteins%2520NSD1%252C%2520NSD2%2520and%2520NSD3%26jtitle%3DMed.%2520Chem.%2520Res.%26date%3D2017%26volume%3D26%26spage%3D2038%26epage%3D2047%26doi%3D10.1007%2Fs00044-017-1909-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tisi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamanini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pathuri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hold, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holding, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, A. C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acklam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drouin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmer, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newbatt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westlake, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Structure of the Epigenetic Oncogene MMSET and Inhibition byN-Alkyl Sinefungin Derivatives</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3093</span>– <span class="NLM_lpage">3105</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00308</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00308" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWltrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=3093-3105&author=D.+Tisiauthor=E.+Chiarparinauthor=E.+Tamaniniauthor=P.+Pathuriauthor=J.+E.+Coyleauthor=A.+Holdauthor=F.+P.+Holdingauthor=N.+Aminauthor=A.+C.+L.+Martinauthor=S.+J.+Richauthor=V.+Berdiniauthor=J.+Yonauthor=P.+Acklamauthor=R.+Burkeauthor=L.+Drouinauthor=J.+E.+Harmerauthor=F.+Jeganathanauthor=R.+L.+M.+van+Montfortauthor=Y.+Newbattauthor=M.+Tortoriciauthor=M.+Westlakeauthor=A.+Woodauthor=S.+Hoelderauthor=T.+D.+Heightman&title=Structure+of+the+Epigenetic+Oncogene+MMSET+and+Inhibition+byN-Alkyl+Sinefungin+Derivatives&doi=10.1021%2Facschembio.6b00308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Epigenetic Oncogene MMSET and Inhibition by N-Alkyl Sinefungin Derivatives</span></div><div class="casAuthors">Tisi, Dominic; Chiarparin, Elisabetta; Tamanini, Emiliano; Pathuri, Puja; Coyle, Joseph E.; Hold, Adam; Holding, Finn P.; Amin, Nader; Martin, Agnes C. L.; Rich, Sharna J.; Berdini, Valerio; Yon, Jeff; Acklam, Paul; Burke, Rosemary; Drouin, Ludovic; Harmer, Jenny E.; Jeganathan, Fiona; van Montfort, Rob L. M.; Newbatt, Yvette; Tortorici, Marcello; Westlake, Maura; Wood, Amy; Hoelder, Swen; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3093-3105</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The NSD subfamily of lysine Me transferases are compelling oncol. targets due to the recent characterization of gain-of-function mutations and translocations in several hematol. cancers.  To date these proteins have proven intractable to small mol. inhibition.  Here the authors present initial efforts to identify inhibitors of MMSET (aka NSD2 or WHSC1) using soln. phase and crystal structural methods.  Based on 2D NMR expts. comparing NSD1 and MMSET structural mobility, the authors designed an MMSET construct with five point mutations in the N-terminal helix of its SET domain for crystn. expts. and elucidated the structure of the mutant MMSET SET domain at 2.1 Å resoln.  Both NSD1 and MMSET crystal systems proved resistant to soaking or co-crystallog. with inhibitors.  However, use of the close homolog SETD2 as a structural surrogate supported the design and characterization of N-alkyl sinefungin derivs. which showed low micromolar inhibition against both SETD2 and MMSET.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi3BsBci-LlLVg90H21EOLACvtfcHk0lhYrlDz-aYKyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWltrrM&md5=b12447cc07001711080004eb39a4ab02</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00308%26sid%3Dliteratum%253Aachs%26aulast%3DTisi%26aufirst%3DD.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DTamanini%26aufirst%3DE.%26aulast%3DPathuri%26aufirst%3DP.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DHold%26aufirst%3DA.%26aulast%3DHolding%26aufirst%3DF.%2BP.%26aulast%3DAmin%26aufirst%3DN.%26aulast%3DMartin%26aufirst%3DA.%2BC.%2BL.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DYon%26aufirst%3DJ.%26aulast%3DAcklam%26aufirst%3DP.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DDrouin%26aufirst%3DL.%26aulast%3DHarmer%26aufirst%3DJ.%2BE.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DNewbatt%26aufirst%3DY.%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DWestlake%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DA.%26aulast%3DHoelder%26aufirst%3DS.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DStructure%2520of%2520the%2520Epigenetic%2520Oncogene%2520MMSET%2520and%2520Inhibition%2520byN-Alkyl%2520Sinefungin%2520Derivatives%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D3093%26epage%3D3105%26doi%3D10.1021%2Facschembio.6b00308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coussens, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horiuchi, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheff, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K. C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal-Nag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span> <span> </span><span class="NLM_article-title">High-throughput screening with nucleosome substrate identifies small-molecule inhibitors of the human histone lysine methyltransferase NSD2</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">13750</span>– <span class="NLM_lpage">13765</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra118.004274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1074%2Fjbc.RA118.004274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=29945974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1altrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=13750-13765&author=N.+P.+Coussensauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=O.+W.+Leeauthor=K.+Y.+Horiuchiauthor=Y.+Wangauthor=Q.+Chenauthor=E.+Kuznetsovaauthor=J.+Wuauthor=S.+Chakkaauthor=D.+M.+Cheffauthor=K.+C.-C.+Chengauthor=P.+Shinnauthor=K.+R.+Brimacombeauthor=M.+Shenauthor=A.+Simeonovauthor=M.+Lal-Nagauthor=H.+Maauthor=A.+Jadhavauthor=M.+D.+Hall&title=High-throughput+screening+with+nucleosome+substrate+identifies+small-molecule+inhibitors+of+the+human+histone+lysine+methyltransferase+NSD2&doi=10.1074%2Fjbc.ra118.004274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput screening with nucleosome substrate identifies small-molecule inhibitors of the human histone lysine methyltransferase NSD2</span></div><div class="casAuthors">Coussens, Nathan P.; Kales, Stephen C.; Henderson, Mark J.; Lee, Olivia W.; Horiuchi, Kurumi Y.; Wang, Yuren; Chen, Qing; Kuznetsova, Ekaterina; Wu, Jianghong; Chakka, Sirisha; Cheff, Dorian M.; Chih-Chien Cheng, Ken; Shinn, Paul; Brimacombe, Kyle R.; Shen, Min; Simeonov, Anton; Lal-Nag, Madhu; Ma, Haiching; Jadhav, Ajit; Hall, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">13750-13765</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The histone lysine methyltransferase nuclear receptor-binding SET domain protein 2 (NSD2, also known as WHSC1/MMSET) is an epigenetic modifier and is thought to play a driving role in oncogenesis.  Both NSD2 overexpression and point mutations that increase its catalytic activity are assocd. with several human cancers.  Although NSD2 is an attractive therapeutic target, no potent, selective, and bioactive small mol. inhibitors of NSD2 have been reported to date, possibly due to the challenges of developing high-throughput assays for NSD2.  Here, to establish a platform for the discovery and development of selective NSD2 inhibitors, we optimized and implemented multiple assays.  We performed quant. high-throughput screening with full-length WT NSD2 and a nucleosome substrate against a diverse collection of bioactive small mols. comprising 16,251 compds.  We further interrogated 174 inhibitory compds. identified in the primary screen with orthogonal and counter assays and with activity assays based on the clin. relevant NSD2 variants E1099K and T1150A.  We selected five confirmed inhibitors for follow-up, which included a radiolabeled validation assay, surface plasmon resonance studies, methyltransferase profiling, and histone methylation in cells.  We found that all five NSD2 inhibitors bind the catalytic SET domain and one exhibited apparent activity in cells, validating the workflow and providing a template for identifying selective NSD2 inhibitors.  In summary, we have established a robust discovery pipeline for identifying potent NSD2 inhibitors from small-mol. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8fq_D6LH0brVg90H21EOLACvtfcHk0ljJch4mq7rcdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1altrzN&md5=e2676702321d6ba99968b06125336eb9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA118.004274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA118.004274%26sid%3Dliteratum%253Aachs%26aulast%3DCoussens%26aufirst%3DN.%2BP.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DO.%2BW.%26aulast%3DHoriuchi%26aufirst%3DK.%2BY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DChakka%26aufirst%3DS.%26aulast%3DCheff%26aufirst%3DD.%2BM.%26aulast%3DCheng%26aufirst%3DK.%2BC.-C.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DLal-Nag%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DM.%2BD.%26atitle%3DHigh-throughput%2520screening%2520with%2520nucleosome%2520substrate%2520identifies%2520small-molecule%2520inhibitors%2520of%2520the%2520human%2520histone%2520lysine%2520methyltransferase%2520NSD2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D13750%26epage%3D13765%26doi%3D10.1074%2Fjbc.ra118.004274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndoj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Alonso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolaidis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogawski, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potopnyk, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulis, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Covalent Inhibition of NSD1 Histone Methyltransferase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1403</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0626-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fs41589-020-0626-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=32868895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWgtLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1403-1410&author=H.+Huangauthor=C.+A.+Howardauthor=S.+Zariauthor=H.+J.+Choauthor=S.+Shuklaauthor=H.+Liauthor=J.+Ndojauthor=P.+Gonz%C3%A1lez-Alonsoauthor=C.+Nikolaidisauthor=J.+Abbottauthor=D.+S.+Rogawskiauthor=M.+A.+Potopnykauthor=K.+Kempinskaauthor=H.+Miaoauthor=T.+Purohitauthor=A.+Hendersonauthor=A.+Mappauthor=M.+L.+Sulisauthor=A.+Ferrandoauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Covalent+Inhibition+of+NSD1+Histone+Methyltransferase&doi=10.1038%2Fs41589-020-0626-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibition of NSD1 histone methyltransferase</span></div><div class="casAuthors">Huang, Huang; Howard, Christina A.; Zari, Sergei; Cho, Hyo Je; Shukla, Shirish; Li, Hao; Ndoj, Juliano; Gonzalez-Alonso, Paula; Nikolaidis, Caroline; Abbott, Joshua; Rogawski, David S.; Potopnyk, Mykhaylo A.; Kempinska, Katarzyna; Miao, Hongzhi; Purohit, Trupta; Henderson, Andrew; Mapp, Anna; Sulis, Maria Luisa; Ferrando, Adolfo; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1403-1410</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The nuclear receptor-binding SET domain (NSD) family of histone methyltransferases is assocd. with various malignancies, including aggressive acute leukemia with NUP98-NSD1 translocation.  While NSD proteins represent attractive drug targets, their catalytic SET domains exist in autoinhibited conformation, presenting notable challenges for inhibitor development.  Here, we employed a fragment-based screening strategy followed by chem. optimization, which resulted in the development of the first-in-class irreversible small-mol. inhibitors of the nuclear receptor-binding SET domain protein 1 (NSD1) SET domain.  The crystal structure of NSD1 in complex with covalently bound ligand reveals a conformational change in the autoinhibitory loop of the SET domain and formation of a channel-like pocket suitable for targeting with small mols.  Our covalent lead-compd. BT5-demonstrates on-target activity in NUP98-NSD1 leukemia cells, including inhibition of histone H3 lysine 36 dimethylation and downregulation of target genes, and impaired colony formation in an NUP98-NSD1 patient sample.  This study will facilitate the development of the next generation of potent and selective inhibitors of the NSD histone methyltransferases. [graphic not available: see fulltext].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryjJt6oKWCx7Vg90H21EOLACvtfcHk0ljJch4mq7rcdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWgtLjJ&md5=1551bed7c9607e30ca76f6941370230b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0626-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0626-6%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DHoward%26aufirst%3DC.%2BA.%26aulast%3DZari%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DH.%2BJ.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DNdoj%26aufirst%3DJ.%26aulast%3DGonz%25C3%25A1lez-Alonso%26aufirst%3DP.%26aulast%3DNikolaidis%26aufirst%3DC.%26aulast%3DAbbott%26aufirst%3DJ.%26aulast%3DRogawski%26aufirst%3DD.%2BS.%26aulast%3DPotopnyk%26aufirst%3DM.%2BA.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DHenderson%26aufirst%3DA.%26aulast%3DMapp%26aufirst%3DA.%26aulast%3DSulis%26aufirst%3DM.%2BL.%26aulast%3DFerrando%26aufirst%3DA.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DCovalent%2520Inhibition%2520of%2520NSD1%2520Histone%2520Methyltransferase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D1403%26epage%3D1410%26doi%3D10.1038%2Fs41589-020-0626-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Englund, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span> <span> </span><span class="NLM_article-title">NSD2 Is Recruited through Its PHD Domain to Oncogenic Gene Loci to Drive Multiple Myeloma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6277</span>– <span class="NLM_lpage">6288</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-13-1000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1158%2F0008-5472.CAN-13-1000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=23980095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6277-6288&author=Z.+Huangauthor=H.+Wuauthor=S.+Chuaiauthor=F.+Xuauthor=F.+Yanauthor=N.+Englundauthor=Z.+Wangauthor=H.+Zhangauthor=M.+Fangauthor=Y.+Wangauthor=J.+Guauthor=M.+Zhangauthor=T.+Yangauthor=K.+Zhaoauthor=Y.+Yuauthor=J.+Daiauthor=W.+Yiauthor=S.+Zhouauthor=Q.+Liauthor=J.+Wuauthor=J.+Liuauthor=X.+Wuauthor=H.+Chanauthor=C.+Luauthor=P.+Atadjaauthor=E.+Liauthor=Y.+Wangauthor=M.+Hu&title=NSD2+Is+Recruited+through+Its+PHD+Domain+to+Oncogenic+Gene+Loci+to+Drive+Multiple+Myeloma&doi=10.1158%2F0008-5472.can-13-1000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">NSD2 Is Recruited through Its PHD Domain to Oncogenic Gene Loci to Drive Multiple Myeloma</span></div><div class="casAuthors">Huang, Zheng; Wu, Haiping; Chuai, Shannon; Xu, Fiona; Yan, Feng; Englund, Nathan; Wang, Zhaofu; Zhang, Hailong; Fang, Ming; Wang, Youzhen; Gu, Justin; Zhang, Man; Yang, Teddy; Zhao, Kehao; Yu, Yanyan; Dai, Jingquan; Yi, Wei; Zhou, Shaolian; Li, Qian; Wu, Jing; Liu, Jun; Wu, Xu; Chan, Homan; Lu, Chris; Atadja, Peter; Li, En; Wang, Yan; Hu, Min</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6277-6288</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone lysine methyltransferase NSD2 (WHSC1/MMSET) is overexpressed frequently in multiple myeloma due to the t(4;14) translocation assocd. with 15% to 20% of cases of this disease.  NSD2 has been found to be involved in myelomagenesis, suggesting it may offer a novel therapeutic target.  Here we show that NSD2 methyltransferase activity is crucial for clonogenicity, adherence, and proliferation of multiple myeloma cells on bone marrow stroma in vitro and that NSD2 is required for tumorigenesis of t(4;14)+ but not t(4;14)- multiple myeloma cells in vivo.  The PHD domains in NSD2 were important for its cellular activity and biol. function through recruiting NSD2 to its oncogenic target genes and driving their transcriptional activation.  By strengthening its disease linkage and deepening insights into its mechanism of action, this study provides a strategy to therapeutically target NSD2 in multiple myeloma patients with a t(4;14) translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp55o4Nm1KVybVg90H21EOLACvtfcHk0ljJch4mq7rcdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7P&md5=96474c8dce30666d3b0f8f3dfc29b70b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1000%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DChuai%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DEnglund%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DYi%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DM.%26atitle%3DNSD2%2520Is%2520Recruited%2520through%2520Its%2520PHD%2520Domain%2520to%2520Oncogenic%2520Gene%2520Loci%2520to%2520Drive%2520Multiple%2520Myeloma%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6277%26epage%3D6288%26doi%3D10.1158%2F0008-5472.can-13-1000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sankaran, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span> <span> </span><span class="NLM_article-title">A PWWP Domain of Histone-Lysine N-Methyltransferase NSD2 Binds to Dimethylated Lys-36 of Histone H3 and Regulates NSD2 Function at Chromatin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">8465</span>– <span class="NLM_lpage">8474</span>, <span class="refDoi"> DOI: 10.1074/jbc.m116.720748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1074%2Fjbc.M116.720748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=26912663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVyjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=8465-8474&author=S.+M.+Sankaranauthor=A.+W.+Wilkinsonauthor=J.+E.+Eliasauthor=O.+Gozani&title=A+PWWP+Domain+of+Histone-Lysine+N-Methyltransferase+NSD2+Binds+to+Dimethylated+Lys-36+of+Histone+H3+and+Regulates+NSD2+Function+at+Chromatin&doi=10.1074%2Fjbc.m116.720748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A PWWP Domain of Histone-Lysine N-Methyltransferase NSD2 Binds to Dimethylated Lys-36 of Histone H3 and Regulates NSD2 Function at Chromatin</span></div><div class="casAuthors">Sankaran, Saumya M.; Wilkinson, Alex W.; Elias, Joshua E.; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8465-8474</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The readout of histone modifications plays a crit. role in chromatin-regulated processes.  Dimethylation at Lys-36 on histone H3 (H3K36me2) is assocd. with actively transcribed genes, and global up-regulation of this modification is assocd. with several cancers.  However, the mol. mechanism by which H3K36me2 is sensed and transduced to downstream biol. outcomes remains unclear.  Here we identify a PWWP domain within the histone lysine methyltransferase and oncoprotein NSD2 that preferentially binds to nucleosomes contg. H3K36me2.  In cells, the NSD2 PWWP domain interaction with H3K36me2 plays a role in stabilizing NSD2 at chromatin.  Furthermore, NSD2's ability to induce global increases in H3K36me2 via its enzymic activity, and consequently promote cellular proliferation, is compromised by mutations within the PWWP domain that specifically abrogate H3K36me2-recognition.  Together, our results identify a pivotal role for NSD2 binding to its catalytic product in regulating its cellular functions, and suggest a model for how this interaction may facilitate epigenetic spreading and propagation of H3K36me2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosUBgHKrvHG7Vg90H21EOLACvtfcHk0lhDhSwguUWo3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVyjs7g%253D&md5=385cbc77875bec389b01e2e93b7a76d5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.720748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.720748%26sid%3Dliteratum%253Aachs%26aulast%3DSankaran%26aufirst%3DS.%2BM.%26aulast%3DWilkinson%26aufirst%3DA.%2BW.%26aulast%3DElias%26aufirst%3DJ.%2BE.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DA%2520PWWP%2520Domain%2520of%2520Histone-Lysine%2520N-Methyltransferase%2520NSD2%2520Binds%2520to%2520Dimethylated%2520Lys-36%2520of%2520Histone%2520H3%2520and%2520Regulates%2520NSD2%2520Function%2520at%2520Chromatin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D8465%26epage%3D8474%26doi%3D10.1074%2Fjbc.m116.720748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structure and Function of the Nucleosome-Binding PWWP Domain</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1016%2Fj.tibs.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=25277115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyhtrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=536-547&author=S.+Qinauthor=J.+Min&title=Structure+and+Function+of+the+Nucleosome-Binding+PWWP+Domain&doi=10.1016%2Fj.tibs.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of the nucleosome-binding PWWP domain</span></div><div class="casAuthors">Qin, Su; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">536-547</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  PWWP domain-contg. proteins are often involved in chromatin-assocd. biol. processes, such as transcriptional regulation and DNA repair, and recent studies have shown that the PWWP domain specifies chromatin localization.  Mutations in the PWWP domain, a 100-150 amino acid motif, have been linked to various human diseases, emphasizing its importance.  Structural studies reveal that PWWP domains possess a conserved arom. cage for histone methyl-lysine recognition, and synergistically bind both histone and DNA, which contributes to their nucleosome-binding ability and chromatin localization.  Furthermore, the PWWP domain often cooperates with other histone and DNA 'reader' or 'modifier' domains to evoke crosstalk between various epigenetic marks.  Here, we discuss these recent advances in understanding the structure and function of the PWWP domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2W0zaGem7yLVg90H21EOLACvtfcHk0lhDhSwguUWo3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyhtrzF&md5=56fa8f9dd279d0f41a6d52c95b987336</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DS.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520Function%2520of%2520the%2520Nucleosome-Binding%2520PWWP%2520Domain%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2014%26volume%3D39%26spage%3D536%26epage%3D547%26doi%3D10.1016%2Fj.tibs.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span> <span> </span><span class="NLM_article-title">ZMYND11 Links Histone H3.3K36me3 to Transcription Elongation and Tumour Suppression</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1038/nature13045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fnature13045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=24590075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlahsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=263-268&author=H.+Wenauthor=Y.+Liauthor=Y.+Xiauthor=S.+Jiangauthor=S.+Strattonauthor=D.+Pengauthor=K.+Tanakaauthor=Y.+Renauthor=Z.+Xiaauthor=J.+Wuauthor=B.+Liauthor=M.+C.+Bartonauthor=W.+Liauthor=H.+Liauthor=X.+Shi&title=ZMYND11+Links+Histone+H3.3K36me3+to+Transcription+Elongation+and+Tumour+Suppression&doi=10.1038%2Fnature13045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">ZMYND11 links histone H3.3K36me3 to transcription elongation and tumour suppression</span></div><div class="casAuthors">Wen, Hong; Li, Yuanyuan; Xi, Yuanxin; Jiang, Shiming; Stratton, Sabrina; Peng, Danni; Tanaka, Kaori; Ren, Yongfeng; Xia, Zheng; Wu, Jun; Li, Bing; Barton, Michelle C.; Li, Wei; Li, Haitao; Shi, Xiaobing</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">263-268</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recognition of modified histones by 'reader' proteins plays a crit. role in the regulation of chromatin.  H3K36 trimethylation (H3K36me3) is deposited onto the nucleosomes in the transcribed regions after RNA polymerase II elongation.  In yeast, this mark in turn recruits epigenetic regulators to reset the chromatin to a relatively repressive state, thus suppressing cryptic transcription.  However, much less is known about the role of H3K36me3 in transcription regulation in mammals.  This is further complicated by the transcription-coupled incorporation of the histone variant H3.3 in gene bodies.  Here we show that the candidate tumor suppressor ZMYND11 specifically recognizes H3K36me3 on H3.3 (H3.3K36me3) and regulates RNA polymerase II elongation.  Structural studies show that in addn. to the trimethyl-lysine binding by an arom. cage within the PWWP domain, the H3.3-dependent recognition is mediated by the encapsulation of the H3.3-specific 'Ser 31' residue in a composite pocket formed by the tandem bromo-PWWP domains of ZMYND11.  Chromatin immunopptn. followed by sequencing shows a genome-wide co-localization of ZMYND11 with H3K36me3 and H3.3 in gene bodies, and its occupancy requires the pre-deposition of H3.3K36me3.  Although ZMYND11 is assocd. with highly expressed genes, it functions as an unconventional transcription co-repressor by modulating RNA polymerase II at the elongation stage.  ZMYND11 is crit. for the repression of a transcriptional program that is essential for tumor cell growth; low expression levels of ZMYND11 in breast cancer patients correlate with worse prognosis.  Consistently, overexpression of ZMYND11 suppresses cancer cell growth in vitro and tumor formation in mice.  Together, this study identifies ZMYND11 as an H3.3-specific reader of H3K36me3 that links the histone-variant-mediated transcription elongation control to tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8IktYGHqR5rVg90H21EOLACvtfcHk0lhDhSwguUWo3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlahsLY%253D&md5=b3d469704c7ce1567a4682dafbab3b0d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature13045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13045%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXi%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DStratton%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DD.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DXia%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DBarton%26aufirst%3DM.%2BC.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DX.%26atitle%3DZMYND11%2520Links%2520Histone%2520H3.3K36me3%2520to%2520Transcription%2520Elongation%2520and%2520Tumour%2520Suppression%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D263%26epage%3D268%26doi%3D10.1038%2Fnature13045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span> <span> </span><span class="NLM_article-title">ICM?A new method for protein modeling and design: Applications to docking and structure prediction from the distorted native conformation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1002/jcc.540150503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1002%2Fjcc.540150503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=488-506&author=R.+Abagyanauthor=M.+Totrovauthor=D.+Kuznetsov&title=ICM%3FA+new+method+for+protein+modeling+and+design%3A+Applications+to+docking+and+structure+prediction+from+the+distorted+native+conformation&doi=10.1002%2Fjcc.540150503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">ICM - a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim; Kuznetsov, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient methodol., further referred to as internal coordinate modeling (ICM), for versatile modeling operations and global energy optimization on arbitrary fixed multimol. systems is described.  It is aimed at protein structure prediction, homol. modeling, mol. docking, NMR structure detn., and protein design.  The method uses and further develops a previously introduced approach to model biomol. structures in which bond lengths, bond angles, and torsion angles are considered as independent variables, any subset of them being fixed.  Here the authors simplify and generalize the basic description of the system, introduce the variable dihedral phase angle, and allow arbitrary connections of the mols. and conventional definition of the torsion angles.  Algorithms for calcn. of energy derivs. with respect to internal variables in the topol. tree of the system and for rapid evaluation of accessible surface are presented.  Multidimensional variables restraints are proposed to represent the statistical information about the torsion angle distributions in proteins.  To incorporate complex energy terms as solvation energy and electrostatics into a structure prediction procedure, a "double-energy" Monte Carlo minimization procedure in which these terms are omitted during the minimization state of the random step and included for the comparison with the previous conformation in a Markov chain is proposed and justified.  The ICM method is applied successfully to a mol. docking problem.  The procedure finds the correct parallel arrangement of 2 rigid helixes from a leucine zipper domain as the lowest-energy conformation (0.5 Å root mean square, RMS, deviation from the native structure) starting from completely random configuration.  Structures with antiparallel helixes or helixes staggered by one helix turn had energies higher by about 7 or 9 kcal/mol, resp.  Soft docking was also attempted.  A docking procedure allowing side-chain flexibility also converged to the parallel configuration starting from the helixes optimized individually.  To justify an internal coordinate approach to the structure prediction as opposed to a Cartesian one, energy hypersurfaces around the native structure of the squash seeds trypsin inhibitor were studied.  Torsion angle minimization from the optimal conformation randomly distorted up to the RMS deviation of 2.2 Å or angular RMS deviation of 10° restored the native conformation in most cases.  In contrast, Cartesian coordinate minimization did not reach the min. from deviations as small as 0.3 Å or 2°.  The authors conclude that the most promising detailed approach to the protein-folding problem would consist of some coarse global sampling strategy combined with the local energy minimization in the torsion coordinate space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kscl2--38LVg90H21EOLACvtfcHk0lid8LTyF7ZQRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D&md5=5f1a1836429860ded9012dab112c9c47</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540150503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540150503%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DD.%26atitle%3DICM%253FA%2520new%2520method%2520for%2520protein%2520modeling%2520and%2520design%253A%2520Applications%2520to%2520docking%2520and%2520structure%2520prediction%2520from%2520the%2520distorted%2520native%2520conformation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D488%26epage%3D506%26doi%3D10.1002%2Fjcc.540150503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span> <i>ROCS 3.2.2.2</i>; <span class="NLM_publisher-name">OpenEye Scientific
Software</span>: <span class="NLM_publisher-loc">Santa Fe, New Mexico</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+ROCS+3.2.2.2%3B+OpenEye+Scientific%0ASoftware%3A+Santa+Fe%2C+New+Mexico."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DROCS%25203.2.2.2%26pub%3DOpenEye%2520Scientific%250ASoftware" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span> <i>FTrees, 3.4</i>; <span class="NLM_publisher-name">BioSolveIT GmbH</span>: <span class="NLM_publisher-loc">Sankt Augustin, Germany</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+FTrees%2C+3.4%3B+BioSolveIT+GmbH%3A+Sankt+Augustin%2C+Germany."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DFTrees%252C%25203.4%26pub%3DBioSolveIT%2520GmbH" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span> <i>Filter, 2.4</i>; <span class="NLM_publisher-name">OpenEye Scientific
Software</span>: <span class="NLM_publisher-loc">Santa Fe, New Mexico</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Filter%2C+2.4%3B+OpenEye+Scientific%0ASoftware%3A+Santa+Fe%2C+New+Mexico."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DFilter%252C%25202.4%26pub%3DOpenEye%2520Scientific%250ASoftware" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span> <span> </span><span class="NLM_article-title">Atomic Property Fields: Generalized 3D Pharmacophoric Potential for Automated Ligand Superposition, Pharmacophore Elucidation and 3D QSAR</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00605.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1111%2Fj.1747-0285.2007.00605.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=18069986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvF2lsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=15-27&author=M.+Totrov&title=Atomic+Property+Fields%3A+Generalized+3D+Pharmacophoric+Potential+for+Automated+Ligand+Superposition%2C+Pharmacophore+Elucidation+and+3D+QSAR&doi=10.1111%2Fj.1747-0285.2007.00605.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Atomic property fields: generalized 3D pharmacophoric potential for automated ligand superposition, pharmacophore elucidation and 3D QSAR</span></div><div class="casAuthors">Totrov, Maxim</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-27</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The at. property fields (APF) concept is introduced as a continuous, multi-component 3D potential that reflects preferences for various at. properties at each point in space.  At. property field-based approaches to several key problems in the field of ligand structure-based rational drug discovery and design are investigated.  The superposition of ligands on one or multiple mol. templates is performed by Monte-Carlo minimization in the at. property fields potentials combined with std. force-field energy.  The approach is extended to multiple flexible ligand alignments using an iterative procedure, Self-Consistent at. Property Fields by Optimization (SCAPFOld).  The application of at. property fields and SCAPFOld for virtual ligand screening and 3D Quantitive Structure-Activity Relationship (QSAR) is tested on published benchmarks.  The new methods are shown to perform competitively in comparison to current state-of-the-art methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQFMXqW3d8ErVg90H21EOLACvtfcHk0lid8LTyF7ZQRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvF2lsL8%253D&md5=d66ce22225d4d9dfeed56196af4025d1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00605.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00605.x%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DAtomic%2520Property%2520Fields%253A%2520Generalized%25203D%2520Pharmacophoric%2520Potential%2520for%2520Automated%2520Ligand%2520Superposition%252C%2520Pharmacophore%2520Elucidation%2520and%25203D%2520QSAR%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2008%26volume%3D71%26spage%3D15%26epage%3D27%26doi%3D10.1111%2Fj.1747-0285.2007.00605.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span> <i>LibMCS, 16.6.13.0</i>; <span class="NLM_publisher-name">ChemAxon</span>: <span class="NLM_publisher-loc">Budapest,
Hungary</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LibMCS%2C+16.6.13.0%3B+ChemAxon%3A+Budapest%2C%0AHungary."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLibMCS%252C%252016.6.13.0%26pub%3DChemAxon" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Böttcher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dilworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumüller, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schleicher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronczki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mischerikow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böse, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hörmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krammer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corcokovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häring, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müllauer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of a Chemical Probe for the PWWP1 Domain of NSD3</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">829</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0310-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fs41589-019-0310-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=31285596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A280%3ADC%252BB3MzltlKnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=822-829&author=J.+B%C3%B6ttcherauthor=D.+Dilworthauthor=U.+Reiserauthor=R.+A.+Neum%C3%BCllerauthor=M.+Schleicherauthor=M.+Petronczkiauthor=M.+Zeebauthor=N.+Mischerikowauthor=A.+Allali-Hassaniauthor=M.+M.+Szewczykauthor=F.+Liauthor=S.+Kennedyauthor=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=P.+J.+Brownauthor=K.+V.+M.+Huberauthor=C.+M.+Rogersauthor=C.+I.+Wellsauthor=O.+Fedorovauthor=K.+Rumpelauthor=A.+Zoephelauthor=M.+Mayerauthor=T.+Wunbergauthor=D.+B%C3%B6seauthor=S.+Zahnauthor=H.+Arnhofauthor=H.+Bergerauthor=C.+Reiserauthor=A.+H%C3%B6rmannauthor=T.+Krammerauthor=M.+Corcokovicauthor=B.+Sharpsauthor=S.+Winklerauthor=D.+H%C3%A4ringauthor=X.-L.+Cockcroftauthor=J.+E.+Fuchsauthor=B.+M%C3%BCllauerauthor=A.+Weiss-Puxbaumauthor=T.+Gerstbergerauthor=G.+Boehmeltauthor=C.+R.+Vakocauthor=C.+H.+Arrowsmithauthor=M.+Pearsonauthor=D.+B.+McConnell&title=Fragment-Based+Discovery+of+a+Chemical+Probe+for+the+PWWP1+Domain+of+NSD3&doi=10.1038%2Fs41589-019-0310-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based discovery of a chemical probe for the PWWP1 domain of NSD3</span></div><div class="casAuthors">Bottcher Jark; Reiser Ulrich; Neumuller Ralph A; Schleicher Michael; Petronczki Mark; Mischerikow Nikolai; Rumpel Klaus; Zoephel Andreas; Mayer Moriz; Wunberg Tobias; Bose Dietrich; Zahn Stephan; Arnhof Heribert; Berger Helmut; Reiser Christoph; Hormann Alexandra; Krammer Teresa; Corcokovic Maja; Sharps Bernadette; Winkler Sandra; Haring Daniela; Cockcroft Xiao-Ling; Fuchs Julian E; Mullauer Barbara; Weiss-Puxbaum Alexander; Gerstberger Thomas; Boehmelt Guido; Pearson Mark; McConnell Darryl B; Dilworth David; Allali-Hassani Abdellah; Szewczyk Magdalena M; Li Fengling; Kennedy Steven; Vedadi Masoud; Barsyte-Lovejoy Dalia; Brown Peter J; Arrowsmith Cheryl H; Zeeb Markus; Vedadi Masoud; Huber Kilian V M; Rogers Catherine M; Fedorov Oleg; Huber Kilian V M; Rogers Catherine M; Fedorov Oleg; Wells Carrow I; Vakoc Christopher R; Arrowsmith Cheryl H</div><div class="citationInfo"><span class="NLM_cas:title">Nature chemical biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">822-829</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Here, we report the fragment-based discovery of BI-9321, a potent, selective and cellular active antagonist of the NSD3-PWWP1 domain.  The human NSD3 protein is encoded by the WHSC1L1 gene located in the 8p11-p12 amplicon, frequently amplified in breast and squamous lung cancer.  Recently, it was demonstrated that the PWWP1 domain of NSD3 is required for the viability of acute myeloid leukemia cells.  To further elucidate the relevance of NSD3 in cancer biology, we developed a chemical probe, BI-9321, targeting the methyl-lysine binding site of the PWWP1 domain with sub-micromolar in vitro activity and cellular target engagement at 1 μM.  As a single agent, BI-9321 downregulates Myc messenger RNA expression and reduces proliferation in MOLM-13 cells.  This first-in-class chemical probe BI-9321, together with the negative control BI-9466, will greatly facilitate the elucidation of the underexplored biological function of PWWP domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAYYvdYFXCxzEwO064Z95rfW6udTcc2eYwkGb9211gkrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzltlKnuw%253D%253D&md5=c28ad6cd575641c001addacd729c293f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0310-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0310-x%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6ttcher%26aufirst%3DJ.%26aulast%3DDilworth%26aufirst%3DD.%26aulast%3DReiser%26aufirst%3DU.%26aulast%3DNeum%25C3%25BCller%26aufirst%3DR.%2BA.%26aulast%3DSchleicher%26aufirst%3DM.%26aulast%3DPetronczki%26aufirst%3DM.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DMischerikow%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DWunberg%26aufirst%3DT.%26aulast%3DB%25C3%25B6se%26aufirst%3DD.%26aulast%3DZahn%26aufirst%3DS.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DReiser%26aufirst%3DC.%26aulast%3DH%25C3%25B6rmann%26aufirst%3DA.%26aulast%3DKrammer%26aufirst%3DT.%26aulast%3DCorcokovic%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DH%25C3%25A4ring%26aufirst%3DD.%26aulast%3DCockcroft%26aufirst%3DX.-L.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DM%25C3%25BCllauer%26aufirst%3DB.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26atitle%3DFragment-Based%2520Discovery%2520of%2520a%2520Chemical%2520Probe%2520for%2520the%2520PWWP1%2520Domain%2520of%2520NSD3%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D822%26epage%3D829%26doi%3D10.1038%2Fs41589-019-0310-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awrey, D. E.</span></span> <span> </span><span class="NLM_article-title">Screening for Ligands Using a Generic and High-Throughput Light-Scattering-Based Assay</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">940</span>– <span class="NLM_lpage">948</span>, <span class="refDoi"> DOI: 10.1177/1087057106294699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1177%2F1087057106294699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=17092916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFOltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=940-948&author=G.+A.+Senisterraauthor=E.+Markinauthor=K.+Yamazakiauthor=R.+Huiauthor=M.+Vedadiauthor=D.+E.+Awrey&title=Screening+for+Ligands+Using+a+Generic+and+High-Throughput+Light-Scattering-Based+Assay&doi=10.1177%2F1087057106294699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Screening for ligands using a generic and high-throughput light-scattering-based assay</span></div><div class="casAuthors">Senisterra, Guillermo A.; Markin, Eugene; Yamazaki, Ken; Hui, Raymond; Vedadi, Masoud; Awrey, Donald E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">940-948</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Rapid identification of small mols. that interact with protein targets using a generic screening method greatly facilitates the development of therapeutic agents.  The authors describe a novel method for performing homogeneous biophys. assays in a high-throughput format.  The use of light scattering as a method to evaluate protein stability during thermal denaturation in a 384-well format yields a robust assay with a low frequency of false positives.  This novel method leads to the identification of interacting small mols. without the addn. of extraneous fluorescent probes.  The anal. and interpretation of data is rapid, with sensitivity for protein stability comparable to differential scanning calorimetry.  The authors propose potential uses in drug discovery, structural genomics, and functional genomics as a method to evaluate small-mol. interactions, identify natural cofactors that stabilize target proteins, and identify natural substrates and products for previously uncharacterized protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodvwx9dxxNrrVg90H21EOLACvtfcHk0lhj4IuteA2tDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFOltA%253D%253D&md5=cfa9fee21f6a5a98330038fcc2856e0d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1177%2F1087057106294699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106294699%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DMarkin%26aufirst%3DE.%26aulast%3DYamazaki%26aufirst%3DK.%26aulast%3DHui%26aufirst%3DR.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAwrey%26aufirst%3DD.%2BE.%26atitle%3DScreening%2520for%2520Ligands%2520Using%2520a%2520Generic%2520and%2520High-Throughput%2520Light-Scattering-Based%2520Assay%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D940%26epage%3D948%26doi%3D10.1177%2F1087057106294699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhj4IuteA2tDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phasercrystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1107%2Fs0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phasercrystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0li5ctWwaBEIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2Fs0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhasercrystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0li5ctWwaBEIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MOLPROBITY: Structure Validation and All-Atom Contact Analysis for Nucleic Acids and Their Complexes</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">W615</span>– <span class="NLM_lpage">W619</span>, <span class="refDoi"> DOI: 10.1093/nar/gkh398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1093%2Fnar%2Fgkh398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=15215462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W615-W619&author=I.+W.+Davisauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MOLPROBITY%3A+Structure+Validation+and+All-Atom+Contact+Analysis+for+Nucleic+Acids+and+Their+Complexes&doi=10.1093%2Fnar%2Fgkh398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">MolPROBITY: Structure validation and all-atom contact analysis for nucleic acids and their complexes</span></div><div class="casAuthors">Davis, Ian W.; Murray, Laura Weston; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W615-W619</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">MolProbity is a general-purpose web service offering quality validation for three-dimensional (3D) structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact anal. of any steric problems within the mols. and can calc. and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addn. and full optimization of all hydrogen atoms, both polar and nonpolar.  The results are reported in multiple forms: as overall numeric scores, as lists, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KING viewer.  This service is available free to all users at http://kinemage.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9I64e-HQbnbVg90H21EOLACvtfcHk0li5ctWwaBEIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D&md5=2050f7acf49743addde69c5ce9c9e2e4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh398%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMOLPROBITY%253A%2520Structure%2520Validation%2520and%2520All-Atom%2520Contact%2520Analysis%2520for%2520Nucleic%2520Acids%2520and%2520Their%2520Complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW615%26epage%3DW619%26doi%3D10.1093%2Fnar%2Fgkh398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Domain architecture of NSD2 and chemical evolution of NSD2-PWWP1 antagonists. Receptor-based virtual screening followed by target and scaffold hopping led to the development of <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>3f</b> (structure shown in the center) selectively binds NSD2 PWWP1. (A) Treatment with <b>3f</b> increases the melting temperature of NSD2-PWWP1 in a DSLS assay. (B) <b>3f</b> binds NSD2-PWWP1 in an SPR assay. (C) <b>3f</b> only affects the melting temperature of NSD2-PWWP1 in a panel of 10 PWWP domains at 100 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of the NSD2-PWWP1 domain in complex with <b>3f</b> [PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UE6">6UE6</a>]. Binding pocket residues (gray) and the antagonist (cyan) are displayed as sticks and hydrogen bonds as magenta dashed lines. Aromatic cage residues are colored in green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/medium/jm0c01768_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>3f</b> disengages NSD2 PWWP1 from histone H3 in cells. Mean-corrected NanoBRET ratios (mBU) are shown for the interaction between NSD2 (orange) and NSD3 (green) with increasing doses of compound <b>3f</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01768/20210216/images/large/jm0c01768_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01768&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47675" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47675" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 26 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zee, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kioi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauring, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span> <span> </span><span class="NLM_article-title">NSD2 Links Dimethylation of Histone H3 at Lysine 36 to Oncogenic Programming</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2011.08.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1016%2Fj.molcel.2011.08.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=22099308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2ku7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=609-620&author=A.+J.+Kuoauthor=P.+Cheungauthor=K.+Chenauthor=B.+M.+Zeeauthor=M.+Kioiauthor=J.+Lauringauthor=Y.+Xiauthor=B.+H.+Parkauthor=X.+Shiauthor=B.+A.+Garciaauthor=W.+Liauthor=O.+Gozani&title=NSD2+Links+Dimethylation+of+Histone+H3+at+Lysine+36+to+Oncogenic+Programming&doi=10.1016%2Fj.molcel.2011.08.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">NSD2 Links Dimethylation of Histone H3 at Lysine 36 to Oncogenic Programming</span></div><div class="casAuthors">Kuo, Alex J.; Cheung, Peggie; Chen, Kaifu; Zee, Barry M.; Kioi, Mitomu; Lauring, Josh; Xi, Yuanxin; Park, Ben Ho; Shi, Xiaobing; Garcia, Benjamin A.; Li, Wei; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-620</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The histone lysine methyltransferase NSD2 (MMSET/WHSC1) is implicated in diverse diseases and commonly overexpressed in multiple myeloma due to a recurrent t(4;14) chromosomal translocation.  However, the precise catalytic activity of NSD2 is obscure, preventing progress in understanding how this enzyme influences chromatin biol. and myeloma pathogenesis.  Here, we show that dimethylation of histone H3 at lysine 36 (H3K36me2) is the principal chromatin-regulatory activity of NSD2.  Catalysis of H3K36me2 by NSD2 is sufficient for gene activation.  In t(4;14)-pos. myeloma cells, the normal genome-wide and gene-specific distribution of H3K36me2 is obliterated, creating a chromatin landscape that selects for a transcription profile favorable for myelomagenesis.  Catalytically active NSD2 confers xenograft tumor formation upon t(4;14)-neg. cells and promotes oncogenic transformation of primary cells in an H3K36me2-dependent manner.  Together, our findings establish H3K36me2 as the primary product generated by NSD2 and demonstrate that genomic disorganization of this canonical chromatin mark by NSD2 initiates oncogenic programming.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJbJf7X8UfurVg90H21EOLACvtfcHk0lirURnuTmDi-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2ku7vL&md5=f023db790f506bcb9472bce31dd2e48a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2011.08.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2011.08.042%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DA.%2BJ.%26aulast%3DCheung%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DZee%26aufirst%3DB.%2BM.%26aulast%3DKioi%26aufirst%3DM.%26aulast%3DLauring%26aufirst%3DJ.%26aulast%3DXi%26aufirst%3DY.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DNSD2%2520Links%2520Dimethylation%2520of%2520Histone%2520H3%2520at%2520Lysine%252036%2520to%2520Oncogenic%2520Programming%26jtitle%3DMol.%2520Cell%26date%3D2011%26volume%3D44%26spage%3D609%26epage%3D620%26doi%3D10.1016%2Fj.molcel.2011.08.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vougiouklakis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saloura, V.</span></span> <span> </span><span class="NLM_article-title">The NSD Family of Protein Methyltransferases in Human Cancer</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.2217/epi.15.32</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.2217%2Fepi.15.32" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=25942451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cqsb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=863-874&author=T.+Vougiouklakisauthor=R.+Hamamotoauthor=Y.+Nakamuraauthor=V.+Saloura&title=The+NSD+Family+of+Protein+Methyltransferases+in+Human+Cancer&doi=10.2217%2Fepi.15.32"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The NSD family of protein methyltransferases in human cancer</span></div><div class="casAuthors">Vougiouklakis, Theodore; Hamamoto, Ryuji; Nakamura, Yusuke; Saloura, Vassiliki</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">863-874</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  The NSD family of protein lysine methyltransferases consists of NSD1, NSD2/WHSC1/MMSET and NSD3/WHSC1L1.  NSD2 haploinsufficiency causes Wolf-Hirschhorn syndrome, while NSD1 mutations lead to the Sotos syndrome.  Recently, a no. of studies showed that the NSD methyltransferases were overexpressed, amplified or somatically mutated in multiple types of cancer, suggesting their crit. role in cancer.  These enzymes methylate specific lysine residues on histone tails and their dysfunction results in epigenomic aberrations which play a fundamental role in oncogenesis.  Furthermore, NSD1 was also reported to methylate a nonhistone protein substrate, RELA/p65 subunit of NF-κB, implying its regulatory function through nonhistone methylation pathways.  In this review, we summarize the current research regarding the role of the NSD family proteins in cancer and underline their potential as targets for novel cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUEZ0yw91nA7Vg90H21EOLACvtfcHk0lizF57Kbi2vTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cqsb%252FK&md5=68993bcfd787c4391e65ae2aaba2ecdd</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2217%2Fepi.15.32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.15.32%26sid%3Dliteratum%253Aachs%26aulast%3DVougiouklakis%26aufirst%3DT.%26aulast%3DHamamoto%26aufirst%3DR.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DSaloura%26aufirst%3DV.%26atitle%3DThe%2520NSD%2520Family%2520of%2520Protein%2520Methyltransferases%2520in%2520Human%2520Cancer%26jtitle%3DEpigenomics%26date%3D2015%26volume%3D7%26spage%3D863%26epage%3D874%26doi%3D10.2217%2Fepi.15.32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larratt, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mant, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belch, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilarski, L. M.</span></span> <span> </span><span class="NLM_article-title">In multiple myeloma, t(4;14)(p16;q32) is an adverse prognostic factor irrespective of FGFR3 expression</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">1520</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.1182/blood-2002-06-1675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1182%2Fblood-2002-06-1675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=12393535" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1agu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2003&pages=1520-1529&author=J.+J.+Keatsauthor=T.+Reimanauthor=C.+A.+Maxwellauthor=B.+J.+Taylorauthor=L.+M.+Larrattauthor=M.+J.+Mantauthor=A.+R.+Belchauthor=L.+M.+Pilarski&title=In+multiple+myeloma%2C+t%284%3B14%29%28p16%3Bq32%29+is+an+adverse+prognostic+factor+irrespective+of+FGFR3+expression&doi=10.1182%2Fblood-2002-06-1675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">In multiple myeloma, t(4;14)(p16;q32) is an adverse prognostic factor irrespective of FGFR3 expression</span></div><div class="casAuthors">Keats, Jonathan J.; Reiman, Tony; Maxwell, Christopher A.; Taylor, Brian J.; Larratt, Loree M.; Mant, Michael J.; Belch, Andrew R.; Pilarski, Linda M.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1520-1529</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">This study analyzed the frequency and clin. significance of t(4;14)(p16;q32) in multiple myeloma (MM) among 208 patients with MM and 52 patients with monoclonal gammopathy of undetd. significance (MGUS); diagnosed between 1994 and 2001.  Patients with the translocation were identified using reverse transcription-polymerase chain reaction (RT-PCR) to detect hybrid Ig heavy chain (IgH)-MMSET transcripts from the der(4) chromosome.  We found 31 (14.9%) t(4;14)+ MM patients and 1 (1.9%) t(4;14)+ MGUS patient.  IgH-MMSET hybrid transcripts were detected in bone marrow (BM) and blood.  Breakpoint anal. revealed that 67.7% of t(4;14)+ patients expressed hybrid transcripts potentially encoding full-length MMSET, whereas the remainder lacked one or more amino terminal exons.  Expression of fibroblast growth factor receptor 3 (FGFR3), presumptively dysregulated on der(14), was detected by RT-PCR in only 23 of 31 (74%) patients with t(4;14)+ MM.  Patients lacking FGFR3 expression also lacked detectable der(14) products.  Longitudinal anal. of 53 MM patients with multiple BM and blood samples showed that, over time, BM from t(4;14)+ patients remained pos. and that t(4;14)- patients did not acquire the translocation.  IgH-MMSET hybrid transcripts and FGFR3 transcripts disappeared from blood during response to therapy.  No correlation was obsd. between the occurrence of t(4;14) and known prognostic indicators.  However, we find the t(4;14) translocation predicts for poor survival (P = .006; median, 644 days vs 1288 days; hazard ratio [HR], 2.0), even in FGFR3 nonexpressors (P = .003).  The presence of t(4;14) is also predictive of poor response to first-line chemotherapy (P = .05).  These results indicate a significant clin. impact of the t(4;14) translocation in MM that is independent of FGFR3 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2jr5_p4QDcLVg90H21EOLACvtfcHk0lizF57Kbi2vTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1agu7c%253D&md5=868219f535ec8ff62c7ca660f07cc1c4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1182%2Fblood-2002-06-1675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2002-06-1675%26sid%3Dliteratum%253Aachs%26aulast%3DKeats%26aufirst%3DJ.%2BJ.%26aulast%3DReiman%26aufirst%3DT.%26aulast%3DMaxwell%26aufirst%3DC.%2BA.%26aulast%3DTaylor%26aufirst%3DB.%2BJ.%26aulast%3DLarratt%26aufirst%3DL.%2BM.%26aulast%3DMant%26aufirst%3DM.%2BJ.%26aulast%3DBelch%26aufirst%3DA.%2BR.%26aulast%3DPilarski%26aufirst%3DL.%2BM.%26atitle%3DIn%2520multiple%2520myeloma%252C%2520t%25284%253B14%2529%2528p16%253Bq32%2529%2520is%2520an%2520adverse%2520prognostic%2520factor%2520irrespective%2520of%2520FGFR3%2520expression%26jtitle%3DBlood%26date%3D2003%26volume%3D101%26spage%3D1520%26epage%3D1529%26doi%3D10.1182%2Fblood-2002-06-1675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huether, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhang, H.-e. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Englund, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert McDonald, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumount, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caponigro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barretina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullighan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downing, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span> <span> </span><span class="NLM_article-title">Global Chromatin Profiling Reveals NSD2 Mutations in Pediatric Acute Lymphoblastic Leukemia</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1386</span>– <span class="NLM_lpage">1391</span>, <span class="refDoi"> DOI: 10.1038/ng.2777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fng.2777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=24076604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFaksLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=1386-1391&author=J.+D.+Jaffeauthor=Y.+Wangauthor=H.+M.+Chanauthor=J.+Zhangauthor=R.+Huetherauthor=G.+V.+Kryukovauthor=H.-e.+C.+Bhangauthor=J.+E.+Taylorauthor=M.+Huauthor=N.+P.+Englundauthor=F.+Yanauthor=Z.+Wangauthor=E.+Robert+McDonaldauthor=L.+Weiauthor=J.+Maauthor=J.+Eastonauthor=Z.+Yuauthor=R.+deBeaumountauthor=V.+Gibajaauthor=K.+Venkatesanauthor=R.+Schlegelauthor=W.+R.+Sellersauthor=N.+Keenauthor=J.+Liuauthor=G.+Caponigroauthor=J.+Barretinaauthor=V.+G.+Cookeauthor=C.+Mullighanauthor=S.+A.+Carrauthor=J.+R.+Downingauthor=L.+A.+Garrawayauthor=F.+Stegmeier&title=Global+Chromatin+Profiling+Reveals+NSD2+Mutations+in+Pediatric+Acute+Lymphoblastic+Leukemia&doi=10.1038%2Fng.2777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Global chromatin profiling reveals NSD2 mutations in pediatric acute lymphoblastic leukemia</span></div><div class="casAuthors">Jaffe, Jacob D.; Wang, Yan; Chan, Ho Man; Zhang, Jinghui; Huether, Robert; Kryukov, Gregory V.; Bhang, Hyo-eun C.; Taylor, Jordan E.; Hu, Min; Englund, Nathan P.; Yan, Feng; Wang, Zhaofu; McDonald, E. Robert; Wei, Lei; Ma, Jing; Easton, John; Yu, Zhengtian; de Beaumount, Rosalie; Gibaja, Veronica; Venkatesan, Kavitha; Schlegel, Robert; Sellers, William R.; Keen, Nicholas; Liu, Jun; Caponigro, Giordano; Barretina, Jordi; Cooke, Vesselina G.; Mullighan, Charles; Carr, Steven A.; Downing, James R.; Garraway, Levi A.; Stegmeier, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1386-1391</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic dysregulation is an emerging hallmark of cancers.  We developed a high-information-content mass spectrometry approach to profile global histone modifications in human cancers.  When applied to 115 lines from the Cancer Cell Line Encyclopedia, this approach identified distinct mol. chromatin signatures.  One signature was characterized by increased histone 3 lysine 36 (H3K36) dimethylation, exhibited by several lines harboring translocations in NSD2, which encodes a methyltransferase.  A previously unknown NSD2 p.  Glu1099Lys (p.E1099K) variant was identified in nontranslocated acute lymphoblastic leukemia (ALL) cell lines sharing this signature.  Ectopic expression of the variant induced a chromatin signature characteristic of NSD2 hyperactivation and promoted transformation.  NSD2 knockdown selectively inhibited the proliferation of NSD2-mutant lines and impaired the in vivo growth of an NSD2-mutant ALL xenograft.  Sequencing anal. of >1,000 pediatric cancer genomes identified the NSD2 p.  E1099K alteration in 14% of t(12;21) ETV6-RUNX1-contg. ALLs.  These findings identify NSD2 as a potential therapeutic target for pediatric ALL and provide a general framework for the functional annotation of cancer epigenomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzi5jFEwRNMrVg90H21EOLACvtfcHk0lhjLX4P6AFXqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFaksLjE&md5=386178a58572ae3180297c3d92d012fa</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fng.2777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2777%26sid%3Dliteratum%253Aachs%26aulast%3DJaffe%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChan%26aufirst%3DH.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DHuether%26aufirst%3DR.%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DBhang%26aufirst%3DH.-e.%2BC.%26aulast%3DTaylor%26aufirst%3DJ.%2BE.%26aulast%3DHu%26aufirst%3DM.%26aulast%3DEnglund%26aufirst%3DN.%2BP.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DRobert%2BMcDonald%26aufirst%3DE.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DEaston%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DdeBeaumount%26aufirst%3DR.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DSchlegel%26aufirst%3DR.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCaponigro%26aufirst%3DG.%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DCooke%26aufirst%3DV.%2BG.%26aulast%3DMullighan%26aufirst%3DC.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DDowning%26aufirst%3DJ.%2BR.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DStegmeier%26aufirst%3DF.%26atitle%3DGlobal%2520Chromatin%2520Profiling%2520Reveals%2520NSD2%2520Mutations%2520in%2520Pediatric%2520Acute%2520Lymphoblastic%2520Leukemia%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D1386%26epage%3D1391%26doi%3D10.1038%2Fng.2777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oyer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezponda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allegretta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okot-Kotber, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackerell, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelleher, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, R.</span></span> <span> </span><span class="NLM_article-title">Point Mutation E1099K in MMSET/NSD2 Enhances Its Methyltranferase Activity and Leads to Altered Global Chromatin Methylation in Lymphoid Malignancies</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">198</span>– <span class="NLM_lpage">201</span>, <span class="refDoi"> DOI: 10.1038/leu.2013.204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fleu.2013.204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=23823660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVejsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=198-201&author=J.+A.+Oyerauthor=X.+Huangauthor=Y.+Zhengauthor=J.+Shimauthor=T.+Ezpondaauthor=Z.+Carpenterauthor=M.+Allegrettaauthor=C.+I.+Okot-Kotberauthor=J.+P.+Patelauthor=A.+Melnickauthor=R.+L.+Levineauthor=A.+Ferrandoauthor=A.+D.+Mackerellauthor=N.+L.+Kelleherauthor=J.+D.+Lichtauthor=R.+Popovic&title=Point+Mutation+E1099K+in+MMSET%2FNSD2+Enhances+Its+Methyltranferase+Activity+and+Leads+to+Altered+Global+Chromatin+Methylation+in+Lymphoid+Malignancies&doi=10.1038%2Fleu.2013.204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Point mutation E1099K in MMSET/NSD2 enhances its methyltranferase activity and leads to altered global chromatin methylation in lymphoid malignancies</span></div><div class="casAuthors">Oyer, J. A.; Huang, X.; Zheng, Y.; Shim, J.; Ezponda, T.; Carpenter, Z.; Allegretta, M.; Okot-Kotber, C. I.; Patel, J. P.; Melnick, A.; Levine, R. L.; Ferrando, A.; MacKerell, A. D.; Kelleher, N. L.; Licht, J. D.; Popovic, R.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">198-201</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acute lymphoblastic leukemia (ALL) is a neoplasm of lymphocytic progenitors and represents the most common malignancy in children.  The disease is frequently characterized by large scale chromosomal alterations including hyper- or hypodiploidy, translocations, deletions and insertions.  We report on a novel recurrent mutation affecting the catalytic domain of MMSET protein.  This mutation is most predominant in pediatric lymphoid malignancies, although its involvement in other tumors cannot be ruled out.  Our data suggest that, similar to t(4;14) translocations in multiple myeloma, E1099K mutation within MMSET enhances global H3K36 dimethylationin lymphoid cells.  This is likely due to a charge alteration within the histone-binding pocket of MMSET and putative disruption of histone-enzyme ionic interactions.  In addn., E1099K mutation and aberrantly high H3K36 methylation lead to a genome-wide decrease in H3K27me3, possibly affecting expression of a no. of genes important for normal lymphoid development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr682-2ULMT7Vg90H21EOLACvtfcHk0lhjLX4P6AFXqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVejsg%253D%253D&md5=c6ce681c77a444ac44b9a66f8855034c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fleu.2013.204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2013.204%26sid%3Dliteratum%253Aachs%26aulast%3DOyer%26aufirst%3DJ.%2BA.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DEzponda%26aufirst%3DT.%26aulast%3DCarpenter%26aufirst%3DZ.%26aulast%3DAllegretta%26aufirst%3DM.%26aulast%3DOkot-Kotber%26aufirst%3DC.%2BI.%26aulast%3DPatel%26aufirst%3DJ.%2BP.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DFerrando%26aufirst%3DA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DKelleher%26aufirst%3DN.%2BL.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DPopovic%26aufirst%3DR.%26atitle%3DPoint%2520Mutation%2520E1099K%2520in%2520MMSET%252FNSD2%2520Enhances%2520Its%2520Methyltranferase%2520Activity%2520and%2520Leads%2520to%2520Altered%2520Global%2520Chromatin%2520Methylation%2520in%2520Lymphoid%2520Malignancies%26jtitle%3DLeukemia%26date%3D2014%26volume%3D28%26spage%3D198%26epage%3D201%26doi%3D10.1038%2Fleu.2013.204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span>; <span class="NLM_string-name">Lnu, S.</span>; <span class="NLM_string-name">Cao, Q.</span>; <span class="NLM_string-name">Asangani, I.</span></span> <span> </span><span class="NLM_article-title">Compositions and Methods for Inhibiting MMSET</span>. U.S. Patent <span class="NLM_patent">20,110,207,198 A1</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=A.+M.+Chinnaiyan&author=S.+Lnu&author=Q.+Cao&author=I.+Asangani&title=Compositions+and+Methods+for+Inhibiting+MMSET"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DCompositions%2520and%2520Methods%2520for%2520Inhibiting%2520MMSET%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morishita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mevius, D. E. H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Luccio, E.</span></span> <span> </span><span class="NLM_article-title">BIX-01294 Inhibits Oncoproteins NSD1, NSD2 and NSD3</span>. <i>Med. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2038</span>– <span class="NLM_lpage">2047</span>, <span class="refDoi"> DOI: 10.1007/s00044-017-1909-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1007%2Fs00044-017-1909-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Gjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2017&pages=2038-2047&author=M.+Morishitaauthor=D.+E.+H.+F.+Meviusauthor=Y.+Shenauthor=S.+Zhaoauthor=E.+di+Luccio&title=BIX-01294+Inhibits+Oncoproteins+NSD1%2C+NSD2+and+NSD3&doi=10.1007%2Fs00044-017-1909-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">BIX-01294 inhibits oncoproteins NSD1, NSD2 and NSD3</span></div><div class="casAuthors">Morishita, Masayo; Mevius, Damiaan E. H. F.; Shen, Yunpeng; Zhao, Shuyu; di Luccio, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2038-2047</span>CODEN:
                <span class="NLM_cas:coden">MCREEB</span>;
        ISSN:<span class="NLM_cas:issn">1054-2523</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">NSD1, NSD2/MMSET/WHSC1, and NSD3/WHSC1L1 make up the nuclear receptor-binding Su(var)3-9, Enhancer-of-zeste and Trithorax domain family of histone methyltransferases, which are essential for regulating the chromatin.  Abnormalities in histone methyltransferases function are increasingly being found to be assocd. with numerous pathol. conditions, including carcinogenesis and tumor progression.  NSD1, NSD2, and NSD3 are oncoproteins aberrantly expressed in numerous cancers in which selective inhibition may offer therapeutic opportunities, esp. in cases of conditions with poor prognoses such as multiple myeloma.  Histone methyltransferase inhibitors are scarce and selective inhibitors are being explored.  NSD inhibitors are urgently needed.  BIX-01294 is a G9a-like protein/G9a histone methyltransferase inhibitor commonly used to modulate H3K9 methylation in the context of cell reprogramming and cancer biol. with in vitro IC50 values of 0.7 and 1.9 μM, resp.  Since the catalytic Su(var)3-9, Enhancer-of-zeste and Trithorax domain of the NSDs is highly related to that of G9a-like protein and G9a, we investigated the potentially differential NSDs inhibition by BIX-01294.  In this study, we identified BIX-01294 as an NSD in vitro hit compd. that differentially inhibits H3K36 methylation by NSD1, NSD2, and NSD3 with IC50 values of 40 ∼ 112 μM.  Furthermore, we investigated the mol. basis of inhibition by BIX-01294 on the NSDs and discuss the prospects of BIX-01294 derivs. for selective NSD inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr29-siRHAlArVg90H21EOLACvtfcHk0lg3-N7MCAJOEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Gjsbo%253D&md5=e7e463c62224cf3afdd199414bb51564</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00044-017-1909-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00044-017-1909-7%26sid%3Dliteratum%253Aachs%26aulast%3DMorishita%26aufirst%3DM.%26aulast%3DMevius%26aufirst%3DD.%2BE.%2BH.%2BF.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3Ddi%2BLuccio%26aufirst%3DE.%26atitle%3DBIX-01294%2520Inhibits%2520Oncoproteins%2520NSD1%252C%2520NSD2%2520and%2520NSD3%26jtitle%3DMed.%2520Chem.%2520Res.%26date%3D2017%26volume%3D26%26spage%3D2038%26epage%3D2047%26doi%3D10.1007%2Fs00044-017-1909-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tisi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamanini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pathuri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hold, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holding, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, A. C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acklam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drouin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmer, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newbatt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westlake, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Structure of the Epigenetic Oncogene MMSET and Inhibition byN-Alkyl Sinefungin Derivatives</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3093</span>– <span class="NLM_lpage">3105</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00308</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00308" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWltrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=3093-3105&author=D.+Tisiauthor=E.+Chiarparinauthor=E.+Tamaniniauthor=P.+Pathuriauthor=J.+E.+Coyleauthor=A.+Holdauthor=F.+P.+Holdingauthor=N.+Aminauthor=A.+C.+L.+Martinauthor=S.+J.+Richauthor=V.+Berdiniauthor=J.+Yonauthor=P.+Acklamauthor=R.+Burkeauthor=L.+Drouinauthor=J.+E.+Harmerauthor=F.+Jeganathanauthor=R.+L.+M.+van+Montfortauthor=Y.+Newbattauthor=M.+Tortoriciauthor=M.+Westlakeauthor=A.+Woodauthor=S.+Hoelderauthor=T.+D.+Heightman&title=Structure+of+the+Epigenetic+Oncogene+MMSET+and+Inhibition+byN-Alkyl+Sinefungin+Derivatives&doi=10.1021%2Facschembio.6b00308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Epigenetic Oncogene MMSET and Inhibition by N-Alkyl Sinefungin Derivatives</span></div><div class="casAuthors">Tisi, Dominic; Chiarparin, Elisabetta; Tamanini, Emiliano; Pathuri, Puja; Coyle, Joseph E.; Hold, Adam; Holding, Finn P.; Amin, Nader; Martin, Agnes C. L.; Rich, Sharna J.; Berdini, Valerio; Yon, Jeff; Acklam, Paul; Burke, Rosemary; Drouin, Ludovic; Harmer, Jenny E.; Jeganathan, Fiona; van Montfort, Rob L. M.; Newbatt, Yvette; Tortorici, Marcello; Westlake, Maura; Wood, Amy; Hoelder, Swen; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3093-3105</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The NSD subfamily of lysine Me transferases are compelling oncol. targets due to the recent characterization of gain-of-function mutations and translocations in several hematol. cancers.  To date these proteins have proven intractable to small mol. inhibition.  Here the authors present initial efforts to identify inhibitors of MMSET (aka NSD2 or WHSC1) using soln. phase and crystal structural methods.  Based on 2D NMR expts. comparing NSD1 and MMSET structural mobility, the authors designed an MMSET construct with five point mutations in the N-terminal helix of its SET domain for crystn. expts. and elucidated the structure of the mutant MMSET SET domain at 2.1 Å resoln.  Both NSD1 and MMSET crystal systems proved resistant to soaking or co-crystallog. with inhibitors.  However, use of the close homolog SETD2 as a structural surrogate supported the design and characterization of N-alkyl sinefungin derivs. which showed low micromolar inhibition against both SETD2 and MMSET.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi3BsBci-LlLVg90H21EOLACvtfcHk0lhxzEA6HRjAuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWltrrM&md5=b12447cc07001711080004eb39a4ab02</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00308%26sid%3Dliteratum%253Aachs%26aulast%3DTisi%26aufirst%3DD.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DTamanini%26aufirst%3DE.%26aulast%3DPathuri%26aufirst%3DP.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DHold%26aufirst%3DA.%26aulast%3DHolding%26aufirst%3DF.%2BP.%26aulast%3DAmin%26aufirst%3DN.%26aulast%3DMartin%26aufirst%3DA.%2BC.%2BL.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DYon%26aufirst%3DJ.%26aulast%3DAcklam%26aufirst%3DP.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DDrouin%26aufirst%3DL.%26aulast%3DHarmer%26aufirst%3DJ.%2BE.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DNewbatt%26aufirst%3DY.%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DWestlake%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DA.%26aulast%3DHoelder%26aufirst%3DS.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DStructure%2520of%2520the%2520Epigenetic%2520Oncogene%2520MMSET%2520and%2520Inhibition%2520byN-Alkyl%2520Sinefungin%2520Derivatives%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D3093%26epage%3D3105%26doi%3D10.1021%2Facschembio.6b00308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coussens, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horiuchi, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheff, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K. C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lal-Nag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span> <span> </span><span class="NLM_article-title">High-throughput screening with nucleosome substrate identifies small-molecule inhibitors of the human histone lysine methyltransferase NSD2</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">13750</span>– <span class="NLM_lpage">13765</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra118.004274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1074%2Fjbc.RA118.004274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=29945974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1altrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=13750-13765&author=N.+P.+Coussensauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=O.+W.+Leeauthor=K.+Y.+Horiuchiauthor=Y.+Wangauthor=Q.+Chenauthor=E.+Kuznetsovaauthor=J.+Wuauthor=S.+Chakkaauthor=D.+M.+Cheffauthor=K.+C.-C.+Chengauthor=P.+Shinnauthor=K.+R.+Brimacombeauthor=M.+Shenauthor=A.+Simeonovauthor=M.+Lal-Nagauthor=H.+Maauthor=A.+Jadhavauthor=M.+D.+Hall&title=High-throughput+screening+with+nucleosome+substrate+identifies+small-molecule+inhibitors+of+the+human+histone+lysine+methyltransferase+NSD2&doi=10.1074%2Fjbc.ra118.004274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput screening with nucleosome substrate identifies small-molecule inhibitors of the human histone lysine methyltransferase NSD2</span></div><div class="casAuthors">Coussens, Nathan P.; Kales, Stephen C.; Henderson, Mark J.; Lee, Olivia W.; Horiuchi, Kurumi Y.; Wang, Yuren; Chen, Qing; Kuznetsova, Ekaterina; Wu, Jianghong; Chakka, Sirisha; Cheff, Dorian M.; Chih-Chien Cheng, Ken; Shinn, Paul; Brimacombe, Kyle R.; Shen, Min; Simeonov, Anton; Lal-Nag, Madhu; Ma, Haiching; Jadhav, Ajit; Hall, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">13750-13765</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The histone lysine methyltransferase nuclear receptor-binding SET domain protein 2 (NSD2, also known as WHSC1/MMSET) is an epigenetic modifier and is thought to play a driving role in oncogenesis.  Both NSD2 overexpression and point mutations that increase its catalytic activity are assocd. with several human cancers.  Although NSD2 is an attractive therapeutic target, no potent, selective, and bioactive small mol. inhibitors of NSD2 have been reported to date, possibly due to the challenges of developing high-throughput assays for NSD2.  Here, to establish a platform for the discovery and development of selective NSD2 inhibitors, we optimized and implemented multiple assays.  We performed quant. high-throughput screening with full-length WT NSD2 and a nucleosome substrate against a diverse collection of bioactive small mols. comprising 16,251 compds.  We further interrogated 174 inhibitory compds. identified in the primary screen with orthogonal and counter assays and with activity assays based on the clin. relevant NSD2 variants E1099K and T1150A.  We selected five confirmed inhibitors for follow-up, which included a radiolabeled validation assay, surface plasmon resonance studies, methyltransferase profiling, and histone methylation in cells.  We found that all five NSD2 inhibitors bind the catalytic SET domain and one exhibited apparent activity in cells, validating the workflow and providing a template for identifying selective NSD2 inhibitors.  In summary, we have established a robust discovery pipeline for identifying potent NSD2 inhibitors from small-mol. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8fq_D6LH0brVg90H21EOLACvtfcHk0liNHqcjyAHf0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1altrzN&md5=e2676702321d6ba99968b06125336eb9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA118.004274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA118.004274%26sid%3Dliteratum%253Aachs%26aulast%3DCoussens%26aufirst%3DN.%2BP.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DO.%2BW.%26aulast%3DHoriuchi%26aufirst%3DK.%2BY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DChakka%26aufirst%3DS.%26aulast%3DCheff%26aufirst%3DD.%2BM.%26aulast%3DCheng%26aufirst%3DK.%2BC.-C.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DLal-Nag%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DM.%2BD.%26atitle%3DHigh-throughput%2520screening%2520with%2520nucleosome%2520substrate%2520identifies%2520small-molecule%2520inhibitors%2520of%2520the%2520human%2520histone%2520lysine%2520methyltransferase%2520NSD2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D13750%26epage%3D13765%26doi%3D10.1074%2Fjbc.ra118.004274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndoj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Alonso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolaidis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogawski, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potopnyk, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempinska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulis, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Covalent Inhibition of NSD1 Histone Methyltransferase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1403</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0626-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fs41589-020-0626-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=32868895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWgtLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1403-1410&author=H.+Huangauthor=C.+A.+Howardauthor=S.+Zariauthor=H.+J.+Choauthor=S.+Shuklaauthor=H.+Liauthor=J.+Ndojauthor=P.+Gonz%C3%A1lez-Alonsoauthor=C.+Nikolaidisauthor=J.+Abbottauthor=D.+S.+Rogawskiauthor=M.+A.+Potopnykauthor=K.+Kempinskaauthor=H.+Miaoauthor=T.+Purohitauthor=A.+Hendersonauthor=A.+Mappauthor=M.+L.+Sulisauthor=A.+Ferrandoauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Covalent+Inhibition+of+NSD1+Histone+Methyltransferase&doi=10.1038%2Fs41589-020-0626-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibition of NSD1 histone methyltransferase</span></div><div class="casAuthors">Huang, Huang; Howard, Christina A.; Zari, Sergei; Cho, Hyo Je; Shukla, Shirish; Li, Hao; Ndoj, Juliano; Gonzalez-Alonso, Paula; Nikolaidis, Caroline; Abbott, Joshua; Rogawski, David S.; Potopnyk, Mykhaylo A.; Kempinska, Katarzyna; Miao, Hongzhi; Purohit, Trupta; Henderson, Andrew; Mapp, Anna; Sulis, Maria Luisa; Ferrando, Adolfo; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1403-1410</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The nuclear receptor-binding SET domain (NSD) family of histone methyltransferases is assocd. with various malignancies, including aggressive acute leukemia with NUP98-NSD1 translocation.  While NSD proteins represent attractive drug targets, their catalytic SET domains exist in autoinhibited conformation, presenting notable challenges for inhibitor development.  Here, we employed a fragment-based screening strategy followed by chem. optimization, which resulted in the development of the first-in-class irreversible small-mol. inhibitors of the nuclear receptor-binding SET domain protein 1 (NSD1) SET domain.  The crystal structure of NSD1 in complex with covalently bound ligand reveals a conformational change in the autoinhibitory loop of the SET domain and formation of a channel-like pocket suitable for targeting with small mols.  Our covalent lead-compd. BT5-demonstrates on-target activity in NUP98-NSD1 leukemia cells, including inhibition of histone H3 lysine 36 dimethylation and downregulation of target genes, and impaired colony formation in an NUP98-NSD1 patient sample.  This study will facilitate the development of the next generation of potent and selective inhibitors of the NSD histone methyltransferases. [graphic not available: see fulltext].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryjJt6oKWCx7Vg90H21EOLACvtfcHk0liNHqcjyAHf0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWgtLjJ&md5=1551bed7c9607e30ca76f6941370230b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0626-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0626-6%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DHoward%26aufirst%3DC.%2BA.%26aulast%3DZari%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DH.%2BJ.%26aulast%3DShukla%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DNdoj%26aufirst%3DJ.%26aulast%3DGonz%25C3%25A1lez-Alonso%26aufirst%3DP.%26aulast%3DNikolaidis%26aufirst%3DC.%26aulast%3DAbbott%26aufirst%3DJ.%26aulast%3DRogawski%26aufirst%3DD.%2BS.%26aulast%3DPotopnyk%26aufirst%3DM.%2BA.%26aulast%3DKempinska%26aufirst%3DK.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DHenderson%26aufirst%3DA.%26aulast%3DMapp%26aufirst%3DA.%26aulast%3DSulis%26aufirst%3DM.%2BL.%26aulast%3DFerrando%26aufirst%3DA.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DCovalent%2520Inhibition%2520of%2520NSD1%2520Histone%2520Methyltransferase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D1403%26epage%3D1410%26doi%3D10.1038%2Fs41589-020-0626-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Englund, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span> <span> </span><span class="NLM_article-title">NSD2 Is Recruited through Its PHD Domain to Oncogenic Gene Loci to Drive Multiple Myeloma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6277</span>– <span class="NLM_lpage">6288</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-13-1000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1158%2F0008-5472.CAN-13-1000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=23980095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6277-6288&author=Z.+Huangauthor=H.+Wuauthor=S.+Chuaiauthor=F.+Xuauthor=F.+Yanauthor=N.+Englundauthor=Z.+Wangauthor=H.+Zhangauthor=M.+Fangauthor=Y.+Wangauthor=J.+Guauthor=M.+Zhangauthor=T.+Yangauthor=K.+Zhaoauthor=Y.+Yuauthor=J.+Daiauthor=W.+Yiauthor=S.+Zhouauthor=Q.+Liauthor=J.+Wuauthor=J.+Liuauthor=X.+Wuauthor=H.+Chanauthor=C.+Luauthor=P.+Atadjaauthor=E.+Liauthor=Y.+Wangauthor=M.+Hu&title=NSD2+Is+Recruited+through+Its+PHD+Domain+to+Oncogenic+Gene+Loci+to+Drive+Multiple+Myeloma&doi=10.1158%2F0008-5472.can-13-1000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">NSD2 Is Recruited through Its PHD Domain to Oncogenic Gene Loci to Drive Multiple Myeloma</span></div><div class="casAuthors">Huang, Zheng; Wu, Haiping; Chuai, Shannon; Xu, Fiona; Yan, Feng; Englund, Nathan; Wang, Zhaofu; Zhang, Hailong; Fang, Ming; Wang, Youzhen; Gu, Justin; Zhang, Man; Yang, Teddy; Zhao, Kehao; Yu, Yanyan; Dai, Jingquan; Yi, Wei; Zhou, Shaolian; Li, Qian; Wu, Jing; Liu, Jun; Wu, Xu; Chan, Homan; Lu, Chris; Atadja, Peter; Li, En; Wang, Yan; Hu, Min</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6277-6288</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone lysine methyltransferase NSD2 (WHSC1/MMSET) is overexpressed frequently in multiple myeloma due to the t(4;14) translocation assocd. with 15% to 20% of cases of this disease.  NSD2 has been found to be involved in myelomagenesis, suggesting it may offer a novel therapeutic target.  Here we show that NSD2 methyltransferase activity is crucial for clonogenicity, adherence, and proliferation of multiple myeloma cells on bone marrow stroma in vitro and that NSD2 is required for tumorigenesis of t(4;14)+ but not t(4;14)- multiple myeloma cells in vivo.  The PHD domains in NSD2 were important for its cellular activity and biol. function through recruiting NSD2 to its oncogenic target genes and driving their transcriptional activation.  By strengthening its disease linkage and deepening insights into its mechanism of action, this study provides a strategy to therapeutically target NSD2 in multiple myeloma patients with a t(4;14) translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp55o4Nm1KVybVg90H21EOLACvtfcHk0liNHqcjyAHf0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7P&md5=96474c8dce30666d3b0f8f3dfc29b70b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1000%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DChuai%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DEnglund%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DYi%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DM.%26atitle%3DNSD2%2520Is%2520Recruited%2520through%2520Its%2520PHD%2520Domain%2520to%2520Oncogenic%2520Gene%2520Loci%2520to%2520Drive%2520Multiple%2520Myeloma%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6277%26epage%3D6288%26doi%3D10.1158%2F0008-5472.can-13-1000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sankaran, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span> <span> </span><span class="NLM_article-title">A PWWP Domain of Histone-Lysine N-Methyltransferase NSD2 Binds to Dimethylated Lys-36 of Histone H3 and Regulates NSD2 Function at Chromatin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">8465</span>– <span class="NLM_lpage">8474</span>, <span class="refDoi"> DOI: 10.1074/jbc.m116.720748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1074%2Fjbc.M116.720748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=26912663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVyjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=8465-8474&author=S.+M.+Sankaranauthor=A.+W.+Wilkinsonauthor=J.+E.+Eliasauthor=O.+Gozani&title=A+PWWP+Domain+of+Histone-Lysine+N-Methyltransferase+NSD2+Binds+to+Dimethylated+Lys-36+of+Histone+H3+and+Regulates+NSD2+Function+at+Chromatin&doi=10.1074%2Fjbc.m116.720748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A PWWP Domain of Histone-Lysine N-Methyltransferase NSD2 Binds to Dimethylated Lys-36 of Histone H3 and Regulates NSD2 Function at Chromatin</span></div><div class="casAuthors">Sankaran, Saumya M.; Wilkinson, Alex W.; Elias, Joshua E.; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8465-8474</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The readout of histone modifications plays a crit. role in chromatin-regulated processes.  Dimethylation at Lys-36 on histone H3 (H3K36me2) is assocd. with actively transcribed genes, and global up-regulation of this modification is assocd. with several cancers.  However, the mol. mechanism by which H3K36me2 is sensed and transduced to downstream biol. outcomes remains unclear.  Here we identify a PWWP domain within the histone lysine methyltransferase and oncoprotein NSD2 that preferentially binds to nucleosomes contg. H3K36me2.  In cells, the NSD2 PWWP domain interaction with H3K36me2 plays a role in stabilizing NSD2 at chromatin.  Furthermore, NSD2's ability to induce global increases in H3K36me2 via its enzymic activity, and consequently promote cellular proliferation, is compromised by mutations within the PWWP domain that specifically abrogate H3K36me2-recognition.  Together, our results identify a pivotal role for NSD2 binding to its catalytic product in regulating its cellular functions, and suggest a model for how this interaction may facilitate epigenetic spreading and propagation of H3K36me2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosUBgHKrvHG7Vg90H21EOLACvtfcHk0ljL2jnSCEe-Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVyjs7g%253D&md5=385cbc77875bec389b01e2e93b7a76d5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.720748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.720748%26sid%3Dliteratum%253Aachs%26aulast%3DSankaran%26aufirst%3DS.%2BM.%26aulast%3DWilkinson%26aufirst%3DA.%2BW.%26aulast%3DElias%26aufirst%3DJ.%2BE.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DA%2520PWWP%2520Domain%2520of%2520Histone-Lysine%2520N-Methyltransferase%2520NSD2%2520Binds%2520to%2520Dimethylated%2520Lys-36%2520of%2520Histone%2520H3%2520and%2520Regulates%2520NSD2%2520Function%2520at%2520Chromatin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D8465%26epage%3D8474%26doi%3D10.1074%2Fjbc.m116.720748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structure and Function of the Nucleosome-Binding PWWP Domain</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2014.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1016%2Fj.tibs.2014.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=25277115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyhtrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=536-547&author=S.+Qinauthor=J.+Min&title=Structure+and+Function+of+the+Nucleosome-Binding+PWWP+Domain&doi=10.1016%2Fj.tibs.2014.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of the nucleosome-binding PWWP domain</span></div><div class="casAuthors">Qin, Su; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">536-547</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  PWWP domain-contg. proteins are often involved in chromatin-assocd. biol. processes, such as transcriptional regulation and DNA repair, and recent studies have shown that the PWWP domain specifies chromatin localization.  Mutations in the PWWP domain, a 100-150 amino acid motif, have been linked to various human diseases, emphasizing its importance.  Structural studies reveal that PWWP domains possess a conserved arom. cage for histone methyl-lysine recognition, and synergistically bind both histone and DNA, which contributes to their nucleosome-binding ability and chromatin localization.  Furthermore, the PWWP domain often cooperates with other histone and DNA 'reader' or 'modifier' domains to evoke crosstalk between various epigenetic marks.  Here, we discuss these recent advances in understanding the structure and function of the PWWP domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2W0zaGem7yLVg90H21EOLACvtfcHk0ljL2jnSCEe-Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyhtrzF&md5=56fa8f9dd279d0f41a6d52c95b987336</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2014.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2014.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DS.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520Function%2520of%2520the%2520Nucleosome-Binding%2520PWWP%2520Domain%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2014%26volume%3D39%26spage%3D536%26epage%3D547%26doi%3D10.1016%2Fj.tibs.2014.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span> <span> </span><span class="NLM_article-title">ZMYND11 Links Histone H3.3K36me3 to Transcription Elongation and Tumour Suppression</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1038/nature13045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fnature13045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=24590075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlahsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=263-268&author=H.+Wenauthor=Y.+Liauthor=Y.+Xiauthor=S.+Jiangauthor=S.+Strattonauthor=D.+Pengauthor=K.+Tanakaauthor=Y.+Renauthor=Z.+Xiaauthor=J.+Wuauthor=B.+Liauthor=M.+C.+Bartonauthor=W.+Liauthor=H.+Liauthor=X.+Shi&title=ZMYND11+Links+Histone+H3.3K36me3+to+Transcription+Elongation+and+Tumour+Suppression&doi=10.1038%2Fnature13045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">ZMYND11 links histone H3.3K36me3 to transcription elongation and tumour suppression</span></div><div class="casAuthors">Wen, Hong; Li, Yuanyuan; Xi, Yuanxin; Jiang, Shiming; Stratton, Sabrina; Peng, Danni; Tanaka, Kaori; Ren, Yongfeng; Xia, Zheng; Wu, Jun; Li, Bing; Barton, Michelle C.; Li, Wei; Li, Haitao; Shi, Xiaobing</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">263-268</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recognition of modified histones by 'reader' proteins plays a crit. role in the regulation of chromatin.  H3K36 trimethylation (H3K36me3) is deposited onto the nucleosomes in the transcribed regions after RNA polymerase II elongation.  In yeast, this mark in turn recruits epigenetic regulators to reset the chromatin to a relatively repressive state, thus suppressing cryptic transcription.  However, much less is known about the role of H3K36me3 in transcription regulation in mammals.  This is further complicated by the transcription-coupled incorporation of the histone variant H3.3 in gene bodies.  Here we show that the candidate tumor suppressor ZMYND11 specifically recognizes H3K36me3 on H3.3 (H3.3K36me3) and regulates RNA polymerase II elongation.  Structural studies show that in addn. to the trimethyl-lysine binding by an arom. cage within the PWWP domain, the H3.3-dependent recognition is mediated by the encapsulation of the H3.3-specific 'Ser 31' residue in a composite pocket formed by the tandem bromo-PWWP domains of ZMYND11.  Chromatin immunopptn. followed by sequencing shows a genome-wide co-localization of ZMYND11 with H3K36me3 and H3.3 in gene bodies, and its occupancy requires the pre-deposition of H3.3K36me3.  Although ZMYND11 is assocd. with highly expressed genes, it functions as an unconventional transcription co-repressor by modulating RNA polymerase II at the elongation stage.  ZMYND11 is crit. for the repression of a transcriptional program that is essential for tumor cell growth; low expression levels of ZMYND11 in breast cancer patients correlate with worse prognosis.  Consistently, overexpression of ZMYND11 suppresses cancer cell growth in vitro and tumor formation in mice.  Together, this study identifies ZMYND11 as an H3.3-specific reader of H3K36me3 that links the histone-variant-mediated transcription elongation control to tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8IktYGHqR5rVg90H21EOLACvtfcHk0ljbTykvVyhfoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlahsLY%253D&md5=b3d469704c7ce1567a4682dafbab3b0d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature13045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13045%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXi%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DStratton%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DD.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DXia%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DBarton%26aufirst%3DM.%2BC.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DX.%26atitle%3DZMYND11%2520Links%2520Histone%2520H3.3K36me3%2520to%2520Transcription%2520Elongation%2520and%2520Tumour%2520Suppression%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D263%26epage%3D268%26doi%3D10.1038%2Fnature13045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span> <span> </span><span class="NLM_article-title">ICM?A new method for protein modeling and design: Applications to docking and structure prediction from the distorted native conformation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1002/jcc.540150503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1002%2Fjcc.540150503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=488-506&author=R.+Abagyanauthor=M.+Totrovauthor=D.+Kuznetsov&title=ICM%3FA+new+method+for+protein+modeling+and+design%3A+Applications+to+docking+and+structure+prediction+from+the+distorted+native+conformation&doi=10.1002%2Fjcc.540150503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">ICM - a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim; Kuznetsov, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient methodol., further referred to as internal coordinate modeling (ICM), for versatile modeling operations and global energy optimization on arbitrary fixed multimol. systems is described.  It is aimed at protein structure prediction, homol. modeling, mol. docking, NMR structure detn., and protein design.  The method uses and further develops a previously introduced approach to model biomol. structures in which bond lengths, bond angles, and torsion angles are considered as independent variables, any subset of them being fixed.  Here the authors simplify and generalize the basic description of the system, introduce the variable dihedral phase angle, and allow arbitrary connections of the mols. and conventional definition of the torsion angles.  Algorithms for calcn. of energy derivs. with respect to internal variables in the topol. tree of the system and for rapid evaluation of accessible surface are presented.  Multidimensional variables restraints are proposed to represent the statistical information about the torsion angle distributions in proteins.  To incorporate complex energy terms as solvation energy and electrostatics into a structure prediction procedure, a "double-energy" Monte Carlo minimization procedure in which these terms are omitted during the minimization state of the random step and included for the comparison with the previous conformation in a Markov chain is proposed and justified.  The ICM method is applied successfully to a mol. docking problem.  The procedure finds the correct parallel arrangement of 2 rigid helixes from a leucine zipper domain as the lowest-energy conformation (0.5 Å root mean square, RMS, deviation from the native structure) starting from completely random configuration.  Structures with antiparallel helixes or helixes staggered by one helix turn had energies higher by about 7 or 9 kcal/mol, resp.  Soft docking was also attempted.  A docking procedure allowing side-chain flexibility also converged to the parallel configuration starting from the helixes optimized individually.  To justify an internal coordinate approach to the structure prediction as opposed to a Cartesian one, energy hypersurfaces around the native structure of the squash seeds trypsin inhibitor were studied.  Torsion angle minimization from the optimal conformation randomly distorted up to the RMS deviation of 2.2 Å or angular RMS deviation of 10° restored the native conformation in most cases.  In contrast, Cartesian coordinate minimization did not reach the min. from deviations as small as 0.3 Å or 2°.  The authors conclude that the most promising detailed approach to the protein-folding problem would consist of some coarse global sampling strategy combined with the local energy minimization in the torsion coordinate space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kscl2--38LVg90H21EOLACvtfcHk0ljbTykvVyhfoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D&md5=5f1a1836429860ded9012dab112c9c47</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540150503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540150503%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DD.%26atitle%3DICM%253FA%2520new%2520method%2520for%2520protein%2520modeling%2520and%2520design%253A%2520Applications%2520to%2520docking%2520and%2520structure%2520prediction%2520from%2520the%2520distorted%2520native%2520conformation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D488%26epage%3D506%26doi%3D10.1002%2Fjcc.540150503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span> <i>ROCS 3.2.2.2</i>; <span class="NLM_publisher-name">OpenEye Scientific
Software</span>: <span class="NLM_publisher-loc">Santa Fe, New Mexico</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+ROCS+3.2.2.2%3B+OpenEye+Scientific%0ASoftware%3A+Santa+Fe%2C+New+Mexico."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DROCS%25203.2.2.2%26pub%3DOpenEye%2520Scientific%250ASoftware" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span> <i>FTrees, 3.4</i>; <span class="NLM_publisher-name">BioSolveIT GmbH</span>: <span class="NLM_publisher-loc">Sankt Augustin, Germany</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+FTrees%2C+3.4%3B+BioSolveIT+GmbH%3A+Sankt+Augustin%2C+Germany."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DFTrees%252C%25203.4%26pub%3DBioSolveIT%2520GmbH" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span> <i>Filter, 2.4</i>; <span class="NLM_publisher-name">OpenEye Scientific
Software</span>: <span class="NLM_publisher-loc">Santa Fe, New Mexico</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Filter%2C+2.4%3B+OpenEye+Scientific%0ASoftware%3A+Santa+Fe%2C+New+Mexico."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DFilter%252C%25202.4%26pub%3DOpenEye%2520Scientific%250ASoftware" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span> <span> </span><span class="NLM_article-title">Atomic Property Fields: Generalized 3D Pharmacophoric Potential for Automated Ligand Superposition, Pharmacophore Elucidation and 3D QSAR</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00605.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1111%2Fj.1747-0285.2007.00605.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=18069986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvF2lsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2008&pages=15-27&author=M.+Totrov&title=Atomic+Property+Fields%3A+Generalized+3D+Pharmacophoric+Potential+for+Automated+Ligand+Superposition%2C+Pharmacophore+Elucidation+and+3D+QSAR&doi=10.1111%2Fj.1747-0285.2007.00605.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Atomic property fields: generalized 3D pharmacophoric potential for automated ligand superposition, pharmacophore elucidation and 3D QSAR</span></div><div class="casAuthors">Totrov, Maxim</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-27</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The at. property fields (APF) concept is introduced as a continuous, multi-component 3D potential that reflects preferences for various at. properties at each point in space.  At. property field-based approaches to several key problems in the field of ligand structure-based rational drug discovery and design are investigated.  The superposition of ligands on one or multiple mol. templates is performed by Monte-Carlo minimization in the at. property fields potentials combined with std. force-field energy.  The approach is extended to multiple flexible ligand alignments using an iterative procedure, Self-Consistent at. Property Fields by Optimization (SCAPFOld).  The application of at. property fields and SCAPFOld for virtual ligand screening and 3D Quantitive Structure-Activity Relationship (QSAR) is tested on published benchmarks.  The new methods are shown to perform competitively in comparison to current state-of-the-art methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQFMXqW3d8ErVg90H21EOLACvtfcHk0ljbTykvVyhfoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvF2lsL8%253D&md5=d66ce22225d4d9dfeed56196af4025d1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00605.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00605.x%26sid%3Dliteratum%253Aachs%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DAtomic%2520Property%2520Fields%253A%2520Generalized%25203D%2520Pharmacophoric%2520Potential%2520for%2520Automated%2520Ligand%2520Superposition%252C%2520Pharmacophore%2520Elucidation%2520and%25203D%2520QSAR%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2008%26volume%3D71%26spage%3D15%26epage%3D27%26doi%3D10.1111%2Fj.1747-0285.2007.00605.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span> <i>LibMCS, 16.6.13.0</i>; <span class="NLM_publisher-name">ChemAxon</span>: <span class="NLM_publisher-loc">Budapest,
Hungary</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+LibMCS%2C+16.6.13.0%3B+ChemAxon%3A+Budapest%2C%0AHungary."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLibMCS%252C%252016.6.13.0%26pub%3DChemAxon" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Böttcher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dilworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumüller, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schleicher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronczki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mischerikow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böse, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hörmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krammer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corcokovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häring, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müllauer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of a Chemical Probe for the PWWP1 Domain of NSD3</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">829</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0310-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1038%2Fs41589-019-0310-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=31285596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A280%3ADC%252BB3MzltlKnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=822-829&author=J.+B%C3%B6ttcherauthor=D.+Dilworthauthor=U.+Reiserauthor=R.+A.+Neum%C3%BCllerauthor=M.+Schleicherauthor=M.+Petronczkiauthor=M.+Zeebauthor=N.+Mischerikowauthor=A.+Allali-Hassaniauthor=M.+M.+Szewczykauthor=F.+Liauthor=S.+Kennedyauthor=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=P.+J.+Brownauthor=K.+V.+M.+Huberauthor=C.+M.+Rogersauthor=C.+I.+Wellsauthor=O.+Fedorovauthor=K.+Rumpelauthor=A.+Zoephelauthor=M.+Mayerauthor=T.+Wunbergauthor=D.+B%C3%B6seauthor=S.+Zahnauthor=H.+Arnhofauthor=H.+Bergerauthor=C.+Reiserauthor=A.+H%C3%B6rmannauthor=T.+Krammerauthor=M.+Corcokovicauthor=B.+Sharpsauthor=S.+Winklerauthor=D.+H%C3%A4ringauthor=X.-L.+Cockcroftauthor=J.+E.+Fuchsauthor=B.+M%C3%BCllauerauthor=A.+Weiss-Puxbaumauthor=T.+Gerstbergerauthor=G.+Boehmeltauthor=C.+R.+Vakocauthor=C.+H.+Arrowsmithauthor=M.+Pearsonauthor=D.+B.+McConnell&title=Fragment-Based+Discovery+of+a+Chemical+Probe+for+the+PWWP1+Domain+of+NSD3&doi=10.1038%2Fs41589-019-0310-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based discovery of a chemical probe for the PWWP1 domain of NSD3</span></div><div class="casAuthors">Bottcher Jark; Reiser Ulrich; Neumuller Ralph A; Schleicher Michael; Petronczki Mark; Mischerikow Nikolai; Rumpel Klaus; Zoephel Andreas; Mayer Moriz; Wunberg Tobias; Bose Dietrich; Zahn Stephan; Arnhof Heribert; Berger Helmut; Reiser Christoph; Hormann Alexandra; Krammer Teresa; Corcokovic Maja; Sharps Bernadette; Winkler Sandra; Haring Daniela; Cockcroft Xiao-Ling; Fuchs Julian E; Mullauer Barbara; Weiss-Puxbaum Alexander; Gerstberger Thomas; Boehmelt Guido; Pearson Mark; McConnell Darryl B; Dilworth David; Allali-Hassani Abdellah; Szewczyk Magdalena M; Li Fengling; Kennedy Steven; Vedadi Masoud; Barsyte-Lovejoy Dalia; Brown Peter J; Arrowsmith Cheryl H; Zeeb Markus; Vedadi Masoud; Huber Kilian V M; Rogers Catherine M; Fedorov Oleg; Huber Kilian V M; Rogers Catherine M; Fedorov Oleg; Wells Carrow I; Vakoc Christopher R; Arrowsmith Cheryl H</div><div class="citationInfo"><span class="NLM_cas:title">Nature chemical biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">822-829</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Here, we report the fragment-based discovery of BI-9321, a potent, selective and cellular active antagonist of the NSD3-PWWP1 domain.  The human NSD3 protein is encoded by the WHSC1L1 gene located in the 8p11-p12 amplicon, frequently amplified in breast and squamous lung cancer.  Recently, it was demonstrated that the PWWP1 domain of NSD3 is required for the viability of acute myeloid leukemia cells.  To further elucidate the relevance of NSD3 in cancer biology, we developed a chemical probe, BI-9321, targeting the methyl-lysine binding site of the PWWP1 domain with sub-micromolar in vitro activity and cellular target engagement at 1 μM.  As a single agent, BI-9321 downregulates Myc messenger RNA expression and reduces proliferation in MOLM-13 cells.  This first-in-class chemical probe BI-9321, together with the negative control BI-9466, will greatly facilitate the elucidation of the underexplored biological function of PWWP domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAYYvdYFXCxzEwO064Z95rfW6udTcc2eYEsHNsI_j-vrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzltlKnuw%253D%253D&md5=c28ad6cd575641c001addacd729c293f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0310-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0310-x%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6ttcher%26aufirst%3DJ.%26aulast%3DDilworth%26aufirst%3DD.%26aulast%3DReiser%26aufirst%3DU.%26aulast%3DNeum%25C3%25BCller%26aufirst%3DR.%2BA.%26aulast%3DSchleicher%26aufirst%3DM.%26aulast%3DPetronczki%26aufirst%3DM.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DMischerikow%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DWunberg%26aufirst%3DT.%26aulast%3DB%25C3%25B6se%26aufirst%3DD.%26aulast%3DZahn%26aufirst%3DS.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DReiser%26aufirst%3DC.%26aulast%3DH%25C3%25B6rmann%26aufirst%3DA.%26aulast%3DKrammer%26aufirst%3DT.%26aulast%3DCorcokovic%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DH%25C3%25A4ring%26aufirst%3DD.%26aulast%3DCockcroft%26aufirst%3DX.-L.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DM%25C3%25BCllauer%26aufirst%3DB.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26atitle%3DFragment-Based%2520Discovery%2520of%2520a%2520Chemical%2520Probe%2520for%2520the%2520PWWP1%2520Domain%2520of%2520NSD3%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D822%26epage%3D829%26doi%3D10.1038%2Fs41589-019-0310-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awrey, D. E.</span></span> <span> </span><span class="NLM_article-title">Screening for Ligands Using a Generic and High-Throughput Light-Scattering-Based Assay</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">940</span>– <span class="NLM_lpage">948</span>, <span class="refDoi"> DOI: 10.1177/1087057106294699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1177%2F1087057106294699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=17092916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFOltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=940-948&author=G.+A.+Senisterraauthor=E.+Markinauthor=K.+Yamazakiauthor=R.+Huiauthor=M.+Vedadiauthor=D.+E.+Awrey&title=Screening+for+Ligands+Using+a+Generic+and+High-Throughput+Light-Scattering-Based+Assay&doi=10.1177%2F1087057106294699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Screening for ligands using a generic and high-throughput light-scattering-based assay</span></div><div class="casAuthors">Senisterra, Guillermo A.; Markin, Eugene; Yamazaki, Ken; Hui, Raymond; Vedadi, Masoud; Awrey, Donald E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">940-948</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Rapid identification of small mols. that interact with protein targets using a generic screening method greatly facilitates the development of therapeutic agents.  The authors describe a novel method for performing homogeneous biophys. assays in a high-throughput format.  The use of light scattering as a method to evaluate protein stability during thermal denaturation in a 384-well format yields a robust assay with a low frequency of false positives.  This novel method leads to the identification of interacting small mols. without the addn. of extraneous fluorescent probes.  The anal. and interpretation of data is rapid, with sensitivity for protein stability comparable to differential scanning calorimetry.  The authors propose potential uses in drug discovery, structural genomics, and functional genomics as a method to evaluate small-mol. interactions, identify natural cofactors that stabilize target proteins, and identify natural substrates and products for previously uncharacterized protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodvwx9dxxNrrVg90H21EOLACvtfcHk0lhD6Augp5CPbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFOltA%253D%253D&md5=cfa9fee21f6a5a98330038fcc2856e0d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1177%2F1087057106294699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106294699%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DMarkin%26aufirst%3DE.%26aulast%3DYamazaki%26aufirst%3DK.%26aulast%3DHui%26aufirst%3DR.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DAwrey%26aufirst%3DD.%2BE.%26atitle%3DScreening%2520for%2520Ligands%2520Using%2520a%2520Generic%2520and%2520High-Throughput%2520Light-Scattering-Based%2520Assay%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D940%26epage%3D948%26doi%3D10.1177%2F1087057106294699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhD6Augp5CPbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phasercrystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1107%2Fs0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phasercrystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhD6Augp5CPbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2Fs0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhasercrystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lg9AVOVwkZjQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MOLPROBITY: Structure Validation and All-Atom Contact Analysis for Nucleic Acids and Their Complexes</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">W615</span>– <span class="NLM_lpage">W619</span>, <span class="refDoi"> DOI: 10.1093/nar/gkh398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=10.1093%2Fnar%2Fgkh398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=15215462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=W615-W619&author=I.+W.+Davisauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MOLPROBITY%3A+Structure+Validation+and+All-Atom+Contact+Analysis+for+Nucleic+Acids+and+Their+Complexes&doi=10.1093%2Fnar%2Fgkh398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">MolPROBITY: Structure validation and all-atom contact analysis for nucleic acids and their complexes</span></div><div class="casAuthors">Davis, Ian W.; Murray, Laura Weston; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W615-W619</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">MolProbity is a general-purpose web service offering quality validation for three-dimensional (3D) structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact anal. of any steric problems within the mols. and can calc. and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addn. and full optimization of all hydrogen atoms, both polar and nonpolar.  The results are reported in multiple forms: as overall numeric scores, as lists, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KING viewer.  This service is available free to all users at http://kinemage.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9I64e-HQbnbVg90H21EOLACvtfcHk0lg9AVOVwkZjQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFKmsLY%253D&md5=2050f7acf49743addde69c5ce9c9e2e4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh398%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMOLPROBITY%253A%2520Structure%2520Validation%2520and%2520All-Atom%2520Contact%2520Analysis%2520for%2520Nucleic%2520Acids%2520and%2520Their%2520Complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DW615%26epage%3DW619%26doi%3D10.1093%2Fnar%2Fgkh398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UE6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UE6','PDB','6UE6'); return false;">PDB: 6UE6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VC8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VC8','PDB','5VC8'); return false;">PDB: 5VC8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N4I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N4I','PDB','4N4I'); return false;">PDB: 4N4I</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i31"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01768">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_81521"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01768?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01768</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Experimental details, 1H NMR spectra of synthesized compounds, 3D docked conformations of virtual screening hits, and compound purity: analytical spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular string formula and activity of all compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_002.pdb">PDB</a>)</p></li><li><p class="inline">X-ray crystallographic data and refinement statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_001.pdf">jm0c01768_si_001.pdf (3.53 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_002.pdb">jm0c01768_si_002.pdb (411.73 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01768/suppl_file/jm0c01768_si_003.csv">jm0c01768_si_003.csv (3.02 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01768&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01768%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-3%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01768" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799244c4bf63db8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
