
<html lang="en"     class="pb-page"  data-request-id="be55c3bc-fc7a-4e42-8dc0-960e4e2a8a2b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2014.57.issue-23;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/jm5015633;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Analysis of Subpocket Selectivity and Identification of Potent Selective Inhibitors for Matriptase and Matriptase-2" /></meta><meta name="dc.Creator" content="Dominic  Duchêne" /></meta><meta name="dc.Creator" content="Eloïc  Colombo" /></meta><meta name="dc.Creator" content="Antoine  Désilets" /></meta><meta name="dc.Creator" content="Pierre-Luc  Boudreault" /></meta><meta name="dc.Creator" content="Richard  Leduc" /></meta><meta name="dc.Creator" content="Eric  Marsault" /></meta><meta name="dc.Creator" content="Rafael  Najmanovich" /></meta><meta name="dc.Description" content="We studied the factors affecting the selectivity of peptidomimetic inhibitors of the highly homologous proteases matriptase and matriptase-2 across subpockets using docking simulations. We observed..." /></meta><meta name="Description" content="We studied the factors affecting the selectivity of peptidomimetic inhibitors of the highly homologous proteases matriptase and matriptase-2 across subpockets using docking simulations. We observed..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 21, 2014" /></meta><meta name="dc.Type" content="rapid-communication" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm5015633" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2014 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm5015633" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm5015633" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm5015633" /></link>
        
    
    

<title>Analysis of Subpocket Selectivity and Identification of Potent Selective Inhibitors for Matriptase and Matriptase-2 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm5015633" /></meta><meta property="og:title" content="Analysis of Subpocket Selectivity and Identification of Potent Selective Inhibitors for Matriptase and Matriptase-2" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0004.jpeg" /></meta><meta property="og:description" content="We studied the factors affecting the selectivity of peptidomimetic inhibitors of the highly homologous proteases matriptase and matriptase-2 across subpockets using docking simulations. We observed that the farther away a subpocket is located from the catalytic site, the more pronounced its role in selectivity. As a result of our exhaustive virtual screening, we biochemically validated novel potent and selective inhibitors of both enzymes." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm5015633"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm5015633">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm5015633&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm5015633&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm5015633&amp;href=/doi/10.1021/jm5015633" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2014</span><span class="cit-fg-volume">, 57</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 10198-10204</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/57/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm501375c" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Brief Article</span></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Analysis of Subpocket Selectivity and Identification of Potent Selective Inhibitors for Matriptase and Matriptase-2</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dominic++Duch%C3%AAne">Dominic Duchêne</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elo%C3%AFc++Colombo">Eloïc Colombo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Antoine++D%C3%A9silets">Antoine Désilets</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Pierre-Luc++Boudreault">Pierre-Luc Boudreault</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Leduc">Richard Leduc</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Eric++Marsault">Eric Marsault</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rafael++Najmanovich">Rafael Najmanovich</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Departments of Biochemistry and <sup>‡</sup>Pharmacology, Faculty of Medicine and Health Sciences, Université de Sherbrooke, 3001 12e Avenue Nord, Sherbrooke, Quebec J1H 5N4, Canada</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4735262126222b6929262d2a262928312e242f0732342f2235253528282c22692426"><span class="__cf_email__" data-cfemail="dfadbeb9bebab3f1b1beb5b2beb1b0a9b6bcb79faaacb7baadbdadb0b0b4baf1bcbe">[email protected]</span></a>. Phone: +1 (819) 820-6868 Ext. 12374.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm5015633&amp;href=/doi/10.1021%2Fjm5015633" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2014</span></span><span class="cit-volume">, 57</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 10198–10204</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 11, 2014</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 September 2014</li><li><span class="item_label"><b>Published</b> online</span>21 November 2014</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 December 2014</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm5015633" title="DOI URL">https://doi.org/10.1021/jm5015633</a></div><div class="article_header-article-copyright"><strong>Copyright © 2014 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm5015633"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2215</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">27</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm5015633" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Analysis of Subpocket Selectivity and Identification of Potent Selective Inhibitors for Matriptase and Matriptase-2&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dominic&quot;,&quot;last_name&quot;:&quot;Duchêne&quot;},{&quot;first_name&quot;:&quot;Eloïc&quot;,&quot;last_name&quot;:&quot;Colombo&quot;},{&quot;first_name&quot;:&quot;Antoine&quot;,&quot;last_name&quot;:&quot;Désilets&quot;},{&quot;first_name&quot;:&quot;Pierre-Luc&quot;,&quot;last_name&quot;:&quot;Boudreault&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Leduc&quot;},{&quot;first_name&quot;:&quot;Eric&quot;,&quot;last_name&quot;:&quot;Marsault&quot;},{&quot;first_name&quot;:&quot;Rafael&quot;,&quot;last_name&quot;:&quot;Najmanovich&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2014&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;57&quot;,&quot;pages&quot;:&quot;10198-10204&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm5015633&quot;},&quot;abstract&quot;:&quot;We studied the factors affecting the selectivity of peptidomimetic inhibitors of the highly homologous proteases matriptase and matriptase-2 across subpockets using docking simulations. We observed that the farther away a subpocket is located from the catalytic site, the more pronounced its role in selectivity. As a result of our exhaustive virtual screening, we biochemically validated novel potent and selective inhibitors of both enzymes.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5015633&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5015633" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5015633&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5015633" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5015633&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5015633" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm5015633&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5015633&amp;href=/doi/10.1021/jm5015633" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm5015633" /></input><a href="/doi/pdf/10.1021/jm5015633" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (1 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm5015633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm5015633%26sid%3Dliteratum%253Aachs%26pmid%3D25387268%26genre%3Darticle%26aulast%3DDuch%25C3%25AAne%26date%3D2014%26atitle%3DAnalysis%2Bof%2BSubpocket%2BSelectivity%2Band%2BIdentification%2Bof%2BPotent%2BSelective%2BInhibitors%2Bfor%2BMatriptase%2Band%2BMatriptase-2%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D57%26issue%3D23%26spage%3D10198%26epage%3D10204%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292420" title="Entropy">Entropy</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/57/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jmcmar.2014.57.issue-23/20141211/jmcmar.2014.57.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/medium/jm-2014-015633_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5015633&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We studied the factors affecting the selectivity of peptidomimetic inhibitors of the highly homologous proteases matriptase and matriptase-2 across subpockets using docking simulations. We observed that the farther away a subpocket is located from the catalytic site, the more pronounced its role in selectivity. As a result of our exhaustive virtual screening, we biochemically validated novel potent and selective inhibitors of both enzymes.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03174" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03174" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Proteases possess a broad range of functions in cellular processes, ranging from nonspecific digestive enzymes, maturation of peptidic hormone precursors or growth factors, tissue remodelling, or activation of specific zymogens. Therefore, deregulation or mutations of proteases can have critical consequences on physiological processes. Proteolytic enzymes have been classified in different families depending on their catalytic mechanism and homology.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Serine proteases account for almost one-third of all proteases.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The S1 serine protease family contains the catalytic triad histidine, aspartate, and serine. We focused on the S1A subfamily, which includes members of the novel family of type II transmembrane serine proteases (TTSPs).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">TTSPs are mostly found at the plasma membrane with an intracellular N-terminal domain, a transmembrane domain, and an exposed extracellular stem region composed of up to 11 structural domains terminated by a C-terminal catalytic domain.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> TTSPs are further divided into four subfamilies: the human airway trypsin-like protease/differentially expressed in squamous cell carcinoma (HAT/DESC), hepsin/transmembrane protease/serine (TMPRSS), corin, and matriptase. Matriptase (also known as ST-14 for suppressor of tumorigenicity 14 protein) is one of the most studied members of his subfamily, followed by matriptase-2 (also known as TMPRSS6).</div><div class="NLM_p">Matriptase is expressed in epithelial cells as a zymogen<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> and needs to be activated. It undergoes spontaneous hydrolysis of a peptide bond at the Gly<sup>149</sup> in the SEA domain. The protease is then autocatalytically processed and activated following cleavage at Arg<sup>614</sup> within the RQAR<sup>614</sup>-VVGG activation sequence.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> In physiological conditions, matriptase-2 is exclusively expressed in the liver initially under zymogen form. Matriptase-2 undergoes autocatalysis at Arg<sup>576</sup> within the PSSR<sup>576</sup>-IVGG sequence located in the consensus activation site of its pro-domain.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">A tight regulation of matriptase by its endogenous inhibitor (such as the hepatocyte activation inhibitor 1, HAI-1), among others, plays a crucial role in placental development, neural tube closure,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and epithelial barrier homeostasis.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In past years, several studies have shown that a deregulation of matriptase can lead to various pathologies. Matriptase is thought to be implicated in osteoarthritis by initiating and inducting cartilage destruction.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Additionally, a mutation in matriptase has been shown to be implicated in autosomal recessive ichtyosis with hypotrichosis (ARIH),<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> a rare skin disease. Several studies have also shown that matriptase is upregulated in different types of epithelial tumors<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10-12)</a> but not mesenchymal-derived tumors. It has also been shown that matriptase can cause a malignant transformation in mice when orthotopically overexpressed.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Finally, it has been demonstrated that matriptase has the ability to cleave and activate the hemagglutinin precursor of the H1<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> and H9 subtypes<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> of influenza virus, which is an essential step in viral replication. Accordingly, treatment with matriptase inhibitors has been shown to reduce viral replication.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a></div><div class="NLM_p">Matriptase-2, on the other hand, is important for iron regulation.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> The protease negatively regulates the expression of hepcidin, an important regulator of iron homeostasis<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> via cleavage of hemojuvelin from plasma membranes.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> A number of studies have shown that mutations in the <i>TMPRSS6</i> gene lead to iron refractory iron deficiency anemia (IRIDA).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> At the other side of the spectrum, iron overload is a condition defined by an increase in total body of iron of more than 5 g caused by a number of factors and leading to different illnesses.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Thus, modulation of matriptase-2 activity could be exploited as a therapy in these conditions.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a></div><div class="NLM_p">Considering these pathologies, both matriptase and matriptase-2 appear to be relevant therapeutic targets for a number of diseases. As a result, in recent years, several groups have reported different classes of inhibitors via diverse strategies<a onclick="showRef(event, 'ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31">(23-31)</a> including virtual screening.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Among these, our group exploited the autocatalytic cleavage sequence of matriptase, flanked by a ketobenzothiazole serine trap, to produce high potency matriptase inhibitors.<a onclick="showRef(event, 'ref15 ref28'); return false;" href="javascript:void(0);" class="ref ref15 ref28">(15, 28)</a></div><div class="NLM_p">The optimization of a high selectivity profile against other proteases of the same family is often a major challenge when developing protease inhibitors. To achieve this goal, the molecule must exploit differences between the target binding site and those of other similar proteases.</div><div class="NLM_p last">We report herein the use of docking simulations to tackle the problem of selectivity between matriptase and matriptase-2 inhibitors. Toward this end, we performed a large scale docking experiment with 8000 different peptidomimetics against both proteases. The compounds are of the same type as our previously developed inhibitor<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> containing a tetrapeptide with arginine in position P<sub>1</sub> linked to a ketobenzothiazole serine trap group. The 20 natural amino acids were used to create all 8000 possible combinations at positions P<sub>4</sub>, P<sub>3</sub>, and P<sub>2</sub>. Our docking results led to the identification of new potent matriptase-2 inhibitors with improved selectivity against matriptase, as well as novel matriptase inhibitors selective against matriptase-2. The docking results provided a tool to analyze selectivity between matriptase and matriptase-2, shedding light into the factors affecting molecular recognition in these two very similar proteases.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42162" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42162" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Because there is no reported crystallographic structure of matriptase-2, a homology model was built using MODELER<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and GROMACS<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> as described in the <a class="ref internalNav" href="#sec3" aria-label="Methods">Methods</a> section. The model is highly similar to the structure of matriptase (0.77 Å RMSD), which was used as a template (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NCL">3NCL</a>) given the high level (44%) of sequence identity of the two enzymes. In the substrate binding subpockets, only five residues differ between the two enzymes: Ser<sup>800</sup> to Ala<sup>757</sup> (subpocket S1), Phe<sup>708</sup> to His<sup>665</sup> (S2–S4), Tyr<sup>755</sup> to Glu<sup>712</sup> (S3), Phe<sup>706</sup> to Asp<sup>663</sup> (S4), and Asp<sup>828</sup> to Leu<sup>785</sup> (S4), respectively, in matriptase and matriptase-2 (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information (SI), Figure 1">Supporting Information (SI), Figure 1</a>). The coordinates of the homology model are provided as <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>.</div><div class="NLM_p">It has been previously shown that protein flexibility plays a key role in docking simulations, especially in cross-docking studies and homology models.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Our docking method FlexAID<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> uses a probabilistic approach to account for rotamer changes upon ligand binding on-the-fly during each simulation using a rotamer library.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Accordingly, we included flexibility in various residues of the binding site. Residues Asp<sup>705</sup>, Phe<sup>708</sup>, Tyr<sup>755</sup>, Gln<sup>782</sup>, Gln<sup>783</sup>, Gln<sup>802</sup>, and Asp<sup>828</sup> of matriptase were found to be in at least one different rotamer conformation or able to undergo rotamer changes without producing steric clashes. The above residues and the corresponding residues in matriptase-2 (Asp<sup>663</sup>, His<sup>665</sup>, Glu<sup>712</sup>, Tyr<sup>739</sup>, Gln<sup>740</sup>, Gln<sup>759</sup>, and Leu<sup>828</sup>) were also set as flexible.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Virtual Screening</h3><div class="NLM_p">We have previously reported a peptidomimetic compound<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> with a <i>K</i><sub>i</sub> of 0.011 ± 0.004 nM for matriptase<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and 300-fold selectivity against matriptase-2 (sequence RQAR in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). We performed docking simulations with a library of 8000 different compounds representing all the possible permutations of natural amino acids in positions P4, P3, and P2 (inset Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), with the goal of identifying potent and selective inhibitors for matriptase-2 and to better rationalize the structural tenants of potency and selectivity between the two proteases. Additionally, it has been shown that arginine in P<sub>1</sub> is the favored amino acid in matriptase and matriptase-2,<a onclick="showRef(event, 'ref28 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref28 ref37 ref38">(28, 37, 38)</a> for this reason the P<sub>1</sub> arginine moiety was kept constant. Given the probabilistic nature of the genetic algorithm based search procedure used in FlexAID, we made 20 independent docking simulations covering 10<sup>6</sup> energy evaluations. This choice of parameters provided 93% accuracy in the prediction of binding poses in the presence of side chain flexibility in the target.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Each binding pose was rescored with MM/GBSA,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and the best one out of the 20 simulations was kept for further analysis. MM/GBSA rescoring was shown to yield better results than most scoring functions in finding the correct binding conformation and binding free energy.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The MM/GBSA scores of all 8000 compounds docked against matriptase and matriptase-2 can be found in <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Specificity and Selectivity of Compounds for Matriptase and Matriptase-2</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="±"><i>K</i><sub>i</sub> (nM)</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">rank</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">SEQ</th><th class="colsep0 rowsep0" align="center" char="±">M<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="±">M2<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">S<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">M</th><th class="colsep0 rowsep0" align="center" char=".">M2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RQAR</td><td class="colsep0 rowsep0" align="char" char="±">0.011<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.004<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char="±">3.30<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 1.00</td><td class="colsep0 rowsep0" align="char" char=".">300</td><td class="colsep0 rowsep0" align="char" char=".">2301</td><td class="colsep0 rowsep0" align="char" char=".">4033</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RQPR</td><td class="colsep0 rowsep0" align="char" char="±">0.061<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.004</td><td class="colsep0 rowsep0" align="char" char="±">7.80<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.90</td><td class="colsep0 rowsep0" align="char" char=".">128</td><td class="colsep0 rowsep0" align="char" char=".">104</td><td class="colsep0 rowsep0" align="char" char=".">2312</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RQYR</td><td class="colsep0 rowsep0" align="char" char="±">0.18<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">0.74<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">678</td><td class="colsep0 rowsep0" align="char" char=".">2754</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RQFR</td><td class="colsep0 rowsep0" align="char" char="±">0.23<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">1.40<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.10</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">721</td><td class="colsep0 rowsep0" align="char" char=".">1168</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RYAR</td><td class="colsep0 rowsep0" align="char" char="±">0.14<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">6.30<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 1.00</td><td class="colsep0 rowsep0" align="char" char=".">45</td><td class="colsep0 rowsep0" align="char" char=".">725</td><td class="colsep0 rowsep0" align="char" char=".">2298</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RRAR</td><td class="colsep0 rowsep0" align="char" char="±">0.72<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">15.40<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> ± 2.80</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">102</td><td class="colsep0 rowsep0" align="char" char=".">179</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LQAR</td><td class="colsep0 rowsep0" align="char" char="±">0.15<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">13.47<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> ± 2.66</td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="char" char=".">659</td><td class="colsep0 rowsep0" align="char" char=".">908</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SQAR</td><td class="colsep0 rowsep0" align="char" char="±">0.092<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.023</td><td class="colsep0 rowsep0" align="char" char="±">34.80<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> ± 13.70</td><td class="colsep0 rowsep0" align="char" char=".">378</td><td class="colsep0 rowsep0" align="char" char=".">4028</td><td class="colsep0 rowsep0" align="char" char=".">3879</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WRER</td><td class="colsep0 rowsep0" align="char" char="±">14.67<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 1.87</td><td class="colsep0 rowsep0" align="char" char="±">46.78<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> ± 2.54</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">4184</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RNPR</td><td class="colsep0 rowsep0" align="char" char="±">0.91<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">47.77<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> ± 5.90</td><td class="colsep0 rowsep0" align="char" char=".">49</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">3690</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KNAR</td><td class="colsep0 rowsep0" align="char" char="±">0.45<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">24.84<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> ± 4.13</td><td class="colsep0 rowsep0" align="char" char=".">62</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">4084</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WCYR</td><td class="colsep0 rowsep0" align="char" char="±">7.98<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 1.27</td><td class="colsep0 rowsep0" align="char" char="±">1.03<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.29</td><td class="colsep0 rowsep0" align="char" char=".">0.13</td><td class="colsep0 rowsep0" align="char" char=".">83</td><td class="colsep0 rowsep0" align="char" char=".">6561</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">YYVR</td><td class="colsep0 rowsep0" align="char" char="±">31.54<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> ± 7.14</td><td class="colsep0 rowsep0" align="char" char="±">2.56<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.20</td><td class="colsep0 rowsep0" align="char" char=".">0.08</td><td class="colsep0 rowsep0" align="char" char=".">4227</td><td class="colsep0 rowsep0" align="char" char=".">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LWWR</td><td class="colsep0 rowsep0" align="char" char="±">3.22<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.37</td><td class="colsep0 rowsep0" align="char" char="±">1.98<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.18</td><td class="colsep0 rowsep0" align="char" char=".">0.63</td><td class="colsep0 rowsep0" align="char" char=".">3244</td><td class="colsep0 rowsep0" align="char" char=".">31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RLSR</td><td class="colsep0 rowsep0" align="char" char="±">1.40<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.21</td><td class="colsep0 rowsep0" align="char" char="±">2.48<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.38</td><td class="colsep0 rowsep0" align="char" char=".">1.79</td><td class="colsep0 rowsep0" align="char" char=".">4741</td><td class="colsep0 rowsep0" align="char" char=".">52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">YKAR</td><td class="colsep0 rowsep0" align="char" char="±">0.89<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 0.29</td><td class="colsep0 rowsep0" align="char" char="±">20.79<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> ± 2.77</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">6226</td><td class="colsep0 rowsep0" align="char" char=".">71</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Matriptase.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Matriptase-2.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Selectivity defined as <i>K</i><sub>i,M2</sub>/<i>K</i><sub>i,M</sub>.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">Tight-binding inhibitor</p></div><div class="footnote" id="t1fn5"><sup>e</sup><p class="last">Classical inhibitor</p></div></div></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/medium/jm-2014-015633_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Comparison of experimental and computed average preferences at position P<sub>3</sub> in IP<sub>3</sub>XR. Each point represents the average of all other 19 possibilities in position P<sub>2</sub>. Colors represent amino acid types: aromatic (orange), positively (blue) or negatively (red) charged, polar (green), and hydrophobic (gray). The inset shows the general structure of the compounds used in this study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5015633&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Agreement between Docking and Experimental Results</h3><div class="NLM_p">Recently, Wysocka et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> reported the substrate preferences of matriptase-2 using a fluorescence functional assay deconvoluting mixtures of fluorescent tetrapeptides. In particular, the authors measured the average activity of matriptase-2 for arrays of molecules of the form IP<sub>3</sub>XR for 19 different natural amino acids (all but Cys) in position P<sub>3</sub> averaged over the same 19 possibilities in position P<sub>2</sub> denoted by X. We extracted the fluorescence intensity values per second as reported by the authors (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B in Wysocka et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>), then calculated the average MM/GBSA <i>Z</i>-scores over the same 19 amino acids for each possibility at position P<sub>3</sub> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).</div><div class="NLM_p last">We obtained a statistically significant linear regression with <i>R</i> = 0.71 (<i>p</i>-value 2.16 × 10<sup>–3</sup>). Somewhat surprisingly, while Gln and Ala behave in our docking simulations like other polar or hydrophobic residues, respectively, these two residues give no signal for Wysocka et al.,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> irrespective of the amino acids present in position P<sub>2</sub>. Lastly, Arg is the preferred residue on average at position P<sub>3</sub> in IP<sub>3</sub>XR but second best after Trp in our docking simulations. Considering the outlier nature of Arg and the uncertainty of the experimental results for Gln and Ala in P<sub>3</sub> as several compounds with these amino acids in position P<sub>3</sub> are active,<a onclick="showRef(event, 'ref28 ref37'); return false;" href="javascript:void(0);" class="ref ref28 ref37">(28, 37)</a> these three amino acids were not taken in consideration when calculating the above linear regression. The above correlation represents the agreement between 361 in silico and experimental results. The high correlation gives us confidence in the quality of the docking simulations.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Average Subpocket Substrate Preferences</h3><div class="NLM_p last">To determine the binding preferences for matriptase and matriptase-2 in each subpocket, we looked at the average MM/GBSA <i>Z</i>-score for the 400 compounds with a conserved residue at each position in P<sub>4</sub>P<sub>3</sub>P<sub>2</sub>R. For example, for a given residue at position P<sub>4</sub>, we averaged over all 400 combinations at positions P<sub>3</sub> and P<sub>2</sub>, or with a given residue at positions P<sub>3</sub> or P<sub>2</sub> instead and averaging over the remaining two positions in each case. For both matriptase and matriptase-2, we observed differences in the preferences across positions within each protein, as well as between proteins for given positions (<a class="ref internalNav" href="#notes-1" aria-label="SI, Figures 2 and 3">SI, Figures 2 and 3</a>, respectively). The observed amino acid preferences within P<sub>4</sub> (<a class="ref internalNav" href="#notes-1" aria-label="SI, Figure 3">SI, Figure 3</a>) are in agreement with the in vitro results of Wysocka et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Our results also agree with those of Béliveau et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> for the residues in P<sub>4</sub> (Arg, Ala, Glu, Leu, and Tyr) within P<sub>4</sub>QAR. However, our prediction that Arg is the preferred choice at P<sub>4</sub> agrees with that of Béliveau et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> while Wysocka et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> find Ile (second best choice in our study) as the best amino acid at that position. For position P<sub>3</sub>, Wysocka et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> only tested compounds with Ile in P<sub>4</sub>, for which we have excellent agreement as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. Our results also generally agree with those of Béliveau et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> who tested Glu, Ala, Leu, Tyr, and Arg as part of RP<sub>3</sub>AR. For position P<sub>2</sub>, Wisocka et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> showed a large preference for Ala and to a smaller extent for Met. In our case, a preference for Lys and Arg is similar to the results obtained by Béliveau et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Differences in Substrate Preferences between Matriptase and Matriptase-2</h3><div class="NLM_p">The above average preferences can mask interesting differences that become apparent when looking at the preferred sequences of each protein. We compared the top 10% scoring compounds for matriptase and matriptase-2 and observed that there are distinct preferences among these (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/medium/jm-2014-015633_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Analysis of subpocket preferences for top scoring compounds. The top scoring 800 compounds for matriptase (blue) and matriptase-2 (red) are shown in a parallel plot. Each line connecting the three subpocket-binding residues represents one compound. At each position, the color gradient corresponds to the fraction of top compounds that use the given amino acid at the particular position, white means equally distributed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5015633&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">At each position a number of amino acids is more readily accepted in matriptase or matriptase-2. For example, in position P<sub>4</sub>, Lys, Leu, Phe, and Trp are more prominent in top scoring compounds in matriptase. Differences are also clear for P<sub>3</sub> but less clear for P<sub>2</sub>. In a recent study, Maurer et al.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> mutated matriptase-2, reintroducing in its binding site four of the five amino acids observed in matriptase (all but D663F in S4) and analyzed their effect on binding a number of small-molecule inhibitors showing the importance of these mutations in the selectivity of compounds. The preference for hydrophobic and aromatic residues in P4 observed for matriptase in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> is in agreement with the results of Maurer et al.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and shows how the added hydrophobicity brought about by the particular amino acids in matriptase (Ser<sup>800</sup>, Phe<sup>708</sup>, Tyr<sup>755</sup>, Phe<sup>706</sup>, and Asp<sup>828</sup>) compared to the respective residues in matriptase-2 (Ala<sup>757</sup>, His<sup>665</sup>, Glu<sup>712</sup>, Asp<sup>663</sup>, and Leu<sup>785</sup>) may account for the observed differences in preferences.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Shannon Entropy of Subpockets and Druggability</h3><div class="NLM_p">The same analysis with a parallel plot can be performed to compare the top and bottom scoring compounds for matriptase (<a class="ref internalNav" href="#notes-1" aria-label="SI, Figure 4">SI, Figure 4</a>) and matriptase-2 (<a class="ref internalNav" href="#notes-1" aria-label="SI, Figure 5">SI, Figure 5</a>). We observed a larger variability in average MM/GBSA <i>Z</i>-scores for amino acids in P<sub>4</sub> than in P<sub>2</sub>. These results point to differences across subpockets in terms of the effect of different amino acids with respect to discriminating top from bottom scoring compounds. This suggests that the S<sub>4</sub> subpocket has a higher potential to be exploited in drug design than S<sub>2</sub> as different choices in the compounds at position P<sub>4</sub> lead to differences in activity while in P<sub>2</sub> the choice of amino acid has less effect.</div><div class="NLM_p">Shannon entropy<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> provides a way to measure a specificity profile for each subpocket. Ranging from 0 to 1, the value determines the tolerance for different amino acids at a given position. A subpocket with Shannon entropy 0 can only bind one specific amino acid (high selectivity for this amino acids), while a Shannon entropy of 1 means no preferences toward the 20 amino acids. To evaluate the binding selectivity of each subpocket, we clustered the 8000 compounds using MM/GBSA <i>Z</i>-score similarities. For each cluster, we calculated the Shannon entropy, as described by Fuchs et al.,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> for the molecules in each cluster at each position. In Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> we show such plot for matriptase-2.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/medium/jm-2014-015633_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Shannon entropy of different compound clusters based on <i>Z</i>-score differences across different subpockets. The number of compounds in each cluster is shown next to the <i>Z</i>-score outer ring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5015633&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The three inner rings represent Shannon entropy values for each cluster at each respective subpocket as marked.</div><div class="NLM_p last">Molecule clusters split early into one cluster (11 o’clock arm) with good scoring molecules (outer ring) and the rest that splits into a cluster with poor scoring molecules (1 o’clock arm) and the remaining with average to poor scores. For all clusters, we have a decrease of Shannon entropy from P<sub>2</sub> to P<sub>4</sub>, with the most pronounced decreases in Shannon entropy observed for good scoring clusters in the 11 o’clock arm of top scoring molecules. Similar results are obtained for matriptase (<a class="ref internalNav" href="#notes-1" aria-label="SI, Figure 6">SI, Figure 6</a>). These results show that for compounds with similar scores, the two proteases tend to be less restrictive and can accommodate more amino acids at position P<sub>2</sub>, while there are fewer possible amino acids accepted at positions P<sub>3</sub> and P<sub>4</sub>. In summary, we observe differences between matriptase and matriptase-2 in terms of their amino acid tolerances at each subpocket. Position P<sub>4</sub> followed by P<sub>3</sub> and P<sub>2</sub>, respectively, have lower Shannon entropy, thus choices of amino acids at position P<sub>4</sub> and P<sub>3</sub> have a higher impact than at position P<sub>2</sub>. Fuchs et al.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> introduced the concept of information entropy above and discussed it in the context of experimental substrate promiscuity. Our results agree with those of Fuchs et al. but provide a more complete picture given the additional data available computationally. Fuchs observes that different protease families have different patterns of Shannon entropy variations (promiscuity) across subpockets. In the present case, by using MM/GBSA scores we observe that both matriptase and matriptase-2, two members of the TTSP SA1 matriptase subfamily, behave similarly, with Shannon entropy decreasing as distance from the catalytic site increases. In other words, within this subfamily, our results suggest that the larger the distance from the catalytic site, the higher the ability of choosing selectivity-conferring amino acids in the nonprimed direction. The differences between matriptase and matriptase-2 come about despite the fact that the existing amino acid differences between the two binding sites are distributed throughout all subpockets.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Selective Inhibitors for Matriptase</h3><div class="NLM_p">To develop more insight on the contribution of different positions in binding and further validate our virtual screening approach, we experimentally tested the activity of a number of compounds with good MM/GBSA scores for matriptase containing a single alteration compared to RQAR. Seven compounds were selected and synthesized, with single alterations in P<sub>2</sub> (RQPR, RQYR, and RQFR), P<sub>3</sub> (RYAR and RRAR), and P<sub>4</sub> (LQAR and SQAR) relative to RQAR. All compounds were tested for in vitro inhibition of matriptase and matriptase-2.</div><div class="NLM_p">In all cases, the compounds have higher specificity for matriptase (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Among these compounds we find two new compounds, RQPR and SQAR that have very low <i>K</i><sub>i</sub> for matriptase and 128- or 378-fold selectivity defined as the ratio of the <i>K</i><sub>i</sub> for matriptase-2 by that of matriptase.</div><div class="NLM_p last">Another four compounds (WRER, RNPR, KNAR, and WCYR) were selected entirely based on having better MM/GBSA ranks for matriptase (top 10%) compared to matriptase-2 (bottom 50%). Two of these compounds are selective for matriptase against matriptase-2 with a selectivity of 49 for RNPR and 60 for KNAR. While their <i>K</i><sub>i</sub> are slightly worse than RQAR, they are comparable in both specificity and selectivity to many compounds related to RQAR by single alterations and provide novel venues for chemical modifications in the pursuit of lead compounds.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Selective Inhibitors for Matriptase-2</h3><div class="NLM_p">Four compounds (YYVR, LWWR, RLSR, and YKAR) with predicted higher specificity for matriptase-2 (top 10% rank) than for matriptase (bottom 50% rank) were experimentally tested. Interestingly, we found one compound (YYVR) that is 13 times more selective for matriptase-2 than matriptase and a second compound (LWWR) with 2-fold selectivity. One compound selected for matriptase is 8 times more selective for matriptase-2 (WCYR). YYVR and WCYR along with recent data on recognition of substrate residues in the P and P′ positions by these proteases<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> will serve as starting points to develop selective compounds of the TTSP family.</div><div class="NLM_p last">Docking methods are generally applied to three distinct but inter-related tasks. Namely, the prediction of binding modes (the structure of the ligand-protein complex), the discrimination of binders from nonbinders (virtual screening), and last, the prediction of binding affinities. Docking methods are notoriously bad at tackling satisfactorily these three problems simultaneously.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> It is therefore interesting to note that there is a qualitative agreement (as shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) between virtual screening ranks and binding affinities as well as quantitative agreement between averaged MM/GBSA scores and activities (as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). In conclusion, the virtual screening led to the identification of potent and selective inhibitors of matriptase and matriptase-2 despite a high level of homology between the two binding pockets and provided insights into the subpocket preferences of each enzyme.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56674" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56674" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Preparation of Ligand Structures</h3><div class="NLM_p last">The 3D coordinates of the ligands with correct chirality were generated with Open Babel 2.3.0<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> using the SMARTS input format. The coordinates were exported in PDB format. The orientation of the ketobenzothiazole and the arginine in P<sub>1</sub> were fixed in a conformation mimicking that found in the thrombin inhibitor (0ZE) in PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1B5G">1B5G</a> for every compound.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Preparation of Protein Targets</h3><div class="NLM_p last">The structure of the protease domain of matriptase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NCL">3NCL</a>) was obtained from the RCSB Protein Data Bank.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Hydrogen atoms and water molecules were removed from the structure. A homology model of matriptase-2 was built with MODELER.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Twenty different models were generated using basic parameters. The conformation with the best DOPE<a onclick="showRef(event, 'ref33 ref49'); return false;" href="javascript:void(0);" class="ref ref33 ref49">(33, 49)</a> score was used.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Docking Parameters</h3><div class="NLM_p last">Molecules were docked in each protein with the small-molecule protein docking program FlexAID.<a onclick="showRef(event, 'ref36 ref39'); return false;" href="javascript:void(0);" class="ref ref36 ref39">(36, 39)</a> FlexAID uses a genetic algorithm to optimize the relative position of the ligand with respect to the protein as well as dihedral angles of the ligand and flexible residues (optimized using an exhaustive search with a rotamer library). The population size and number of generations used in the genetic algorithm were both set at 1000 for a total of 10<sup>6</sup> energy evaluations. The ligands were set as semiflexible. Arg in P<sub>1</sub> and the ketobenzothiazole group were set as rigid as well as all the peptide bonds. All side chain dihedral bonds were set as flexible. The carbon making the reversible covalent bond with the protein was fixed at a distance of 3.0 ± 0.1 Å of the catalytic Ser<sup>805</sup> to emulate the effect that the covalent bond would have in constraining the position of the ketobenzothiazole group within the binding site. Whereas this procedure only approximates covalent binding as it ignores the influence of developing a covalent bond on the structure of the binding site, it has been shown to be successful in a number of protein families and types of covalent ligands.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Target flexibility was restricted to binding-site side chains. Flexible residues were located based on the superimposition and detection of residues seen in different conformations in the binding sites of other matriptase structures: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EAW">1EAW</a> with bovine pancreatic trypsin inhibitor (BPTI), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EAX">1EAX</a> with the small molecule inhibitor benzamidine, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GV6">2GV6</a> with the small molecule inhibitor CJ-730, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GV7">2GV7</a> with the small molecule inhibitor CJ-672, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BN9">3BN9</a> with a noncompetitive FAB antibody inhibitor. For each of the 8000 docked compounds, 20 different simulations were performed using our volunteer-based grid computing platform <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="c6889481868ea9aba3">[email protected]</a> using the BOINC platform.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Each docking result was rescored with the MM/GBSA method as described below. FlexAID and the NRGsuite PyMOL plugin to perform and view on-the-fly docking simulations for the Window, Linux, and MacOS platforms can be obtained free of charge at <a href="http://bcb.med.usherbrooke.ca/flexaid" class="extLink">http://bcb.med.usherbrooke.ca/flexaid</a>.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> MM/GBSA Rescoring of Docking Conformations</h3><div class="NLM_p last">All protein–ligand complexes were first energy-minimized with GROMACS<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> 4.5.1 using the conjugate gradient algorithm in a TIP3P water box. For the proteins, AMBER03 force field parameters were used. For the ligands, general AMBER force field generated in Antechamber and converted to GROMACS formats by ACPYPE<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> were used. Partial charges were calculated with the Partial Equalization of Orbital Electronegativities (PEOE) algorithm by Gaisteiger and Marsilli.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> All poses were then rescored with MM/GBSA using the HCT model effective Born radii.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Synthesis of Inhibitors</h3><div class="NLM_p last">Inhibitors were synthesized using a combination of solid phase and solution synthesis methodologies. Intermediate <b>2</b> (<a class="ref internalNav" href="#notes-1" aria-label="SI, Scheme 1">SI, Scheme 1</a>) was prepared as reported earlier.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Analogues <b>3a</b>–<b>s</b> (<a class="ref internalNav" href="#notes-1" aria-label="SI, Scheme 2">SI, Scheme 2</a>) were synthesized by standard solid phase peptide synthesis (SPPS) using the 4-chlorotrityl chloride resin.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Briefly, C-terminal anchoring of the first amino acid on the resin was performed in dichloromethane (DCM) in the presence of diisopropylethylamine (DIPEA). Tripeptide synthesis were pursued with standard Fmoc protected amino acid coupling, deprotected in basic conditions (20% piperidine/DMF) and in order to selectively couple the modified arginine <b>2</b> (<a class="ref internalNav" href="#notes-1" aria-label="SI, Scheme 1">SI, Scheme 1</a>), last coupling was done using N-Boc-protected amino acid. Analogues <b>3a</b>–<b>s</b> were cleaved from resin under mild acidic conditions (DCM/TFE or HFIP, see note in <a class="ref internalNav" href="#notes-1" aria-label="SI, Scheme 2">SI, Scheme 2</a>). Tripeptides <b>3a</b>–<b>s</b> and intermediate <b>2</b> were engaged in peptidic coupling reaction, followed by oxidization and acidolysis to give tetrapeptide compounds <b>5a</b>–<b>s</b> based on the synthetic strategy described by Colombo et al.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Generally, compounds were obtained as a 3:1 mixture of epimers at the P1 position,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> which were purified and separated by reverse-phase preparative high-performance liquid chromatography (prep-HPLC) above 95% purity (UV). Compound purity and identity were confirmed by UPLC-MS.</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Inhibition Assays of Matriptase and Matriptase-2</h3><div class="NLM_p last">Active recombinant human matriptase (amino acids 596–655) and matriptase-2 (amino acids 78–811) were expressed and purified as previously described.<a onclick="showRef(event, 'ref27 ref37'); return false;" href="javascript:void(0);" class="ref ref27 ref37">(27, 37)</a> Proteases were active-site titrated with 4-methylumbelliferyl-<i>p</i>-guanidino benzoate (MUGB) prior to inhibition assays. Enzymatic assays and <i>K</i><sub>i</sub> determination were performed at room temperature in an assay buffer containing 50 mM Tris-HCl, 15 mM NaCl, and 500 at pH 7.4 as described previously.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Then 1 nM of protease was added to a reaction buffer containing 0 nM, 10 nM, or 10 mM of inhibitors and 100 mM of a fluorogenic substrate (Boc-Gln-Ala-Arg-AMC), and proteolytic activity was monitored by measuring the release of fluorescence (excitation, 360 nm; emission, 441 nm) in a FLX800 TBE microplate reader (Bio-Tek Instruments, Winooski, VT, USA). If substantial inhibition occurred using a ratio <i>I</i>/<i>E</i> = 10, compound were treated as tight-binding inhibitors, and if inhibition occurs only at <i>I</i>/<i>E</i> > 10, compounds were treated as classical reversible inhibitors. For <i>K</i><sub>i</sub> determination of tight-binding inhibitors, plots of enzyme velocity as a function of inhibitor concentration were fitted by nonlinear regression analysis to the Morrison <i>K</i><sub>i</sub> equation.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> For <i>K</i><sub>i</sub> determination of classical reversible inhibitors, plots of enzyme velocity as a function of substrate concentration at several inhibitor concentrations were fitted by nonlinear regression to equations describing different model of reversible inhibition (competitive, uncompetitive, noncompetitive, and mixed model), and the preferred model was used for <i>K</i><sub>i</sub> determination. All assays were performed at least three times in duplicates, and data are presented as mean ± standard deviation (SD). Nonlinear regression and statistical analysis were performed using GraphPad Prism version 6.02 for Windows (GraphPad Software, San Diego, CA, USA).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18"><a href="/doi/suppl/10.1021/jm5015633">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Solution and solid-phase synthesis, materials and equipment; synthetic procedures; chemistry; progress curves; docking results for the 8000 compounds (CSV) and the coordinates of the model structure of matriptase-2 (PDB). This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5015633/suppl_file/jm5015633_si_001.pdf">jm5015633_si_001.pdf (11.49 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5015633/suppl_file/jm5015633_si_002.csv">jm5015633_si_002.csv (338.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5015633/suppl_file/jm5015633_si_003.pdb">jm5015633_si_003.pdb (287.99 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm5015633" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rafael Najmanovich</span> - <span class="hlFld-Affiliation affiliation">†Departments of Biochemistry and ‡Pharmacology, Faculty of Medicine and Health
Sciences, Université de Sherbrooke, 3001 12e Avenue Nord, Sherbrooke, Quebec J1H 5N4, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d1a3b0b7b0b4bdffbfb0bbbcb0bfbea7b8b2b991a4a2b9b4a3b3a3bebebab4ffb2b0"><span class="__cf_email__" data-cfemail="3c4e5d5a5d595012525d56515d52534a555f547c494f54594e5e4e53535759125f5d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominic Duchêne</span> - <span class="hlFld-Affiliation affiliation">†Departments of Biochemistry and ‡Pharmacology, Faculty of Medicine and Health
Sciences, Université de Sherbrooke, 3001 12e Avenue Nord, Sherbrooke, Quebec J1H 5N4, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eloïc Colombo</span> - <span class="hlFld-Affiliation affiliation">†Departments of Biochemistry and ‡Pharmacology, Faculty of Medicine and Health
Sciences, Université de Sherbrooke, 3001 12e Avenue Nord, Sherbrooke, Quebec J1H 5N4, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antoine Désilets</span> - <span class="hlFld-Affiliation affiliation">†Departments of Biochemistry and ‡Pharmacology, Faculty of Medicine and Health
Sciences, Université de Sherbrooke, 3001 12e Avenue Nord, Sherbrooke, Quebec J1H 5N4, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pierre-Luc Boudreault</span> - <span class="hlFld-Affiliation affiliation">†Departments of Biochemistry and ‡Pharmacology, Faculty of Medicine and Health
Sciences, Université de Sherbrooke, 3001 12e Avenue Nord, Sherbrooke, Quebec J1H 5N4, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Leduc</span> - <span class="hlFld-Affiliation affiliation">†Departments of Biochemistry and ‡Pharmacology, Faculty of Medicine and Health
Sciences, Université de Sherbrooke, 3001 12e Avenue Nord, Sherbrooke, Quebec J1H 5N4, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eric Marsault</span> - <span class="hlFld-Affiliation affiliation">†Departments of Biochemistry and ‡Pharmacology, Faculty of Medicine and Health
Sciences, Université de Sherbrooke, 3001 12e Avenue Nord, Sherbrooke, Quebec J1H 5N4, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>D.D., E.C., A.D., R.L., E.M., and R.J.N. designed the experiments, D.D., E.C., A.D., and P.L.B. performed the experiments. D.D., E.C., A.D., and R.J.N. wrote the manuscript. All authors reviewed and gave approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d4994e1546-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44403" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44403" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge the input of Francis Gaudreault in the use of FlexAID for the docking simulations and all <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="420c1005020a2d2f27">[email protected]</a> volunteers for their CPU time donated to this project. R.J.N., E.M., and R.L. are part of CR-CHUS and the Institute of Pharmacology of Sherbrooke. R.J.N. and E.M. are members of PROTEO (the Québec network for research on protein function, structure and engineering). R.J.N. is a member of GRASP (Groupe de Recherche Axé sur la Structure des Protéines). This work was supported by the Canadian Institutes for Health Research (CIHR). E.C. was partly founded by a Proteo student fellowship.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i21" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i21"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i22" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i22"> Abbreviations Used</h2><tr><td class="NLM_term">MMGB/SA</td><td class="NLM_def"><p class="first last">Molecular Mechanics/Generalized Born Surface Area</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34013" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34013" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Rawlings, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, A.</span><span> </span><span class="NLM_article-title">MEROPS: the Database of Proteolytic Enzymes, Their Substrates and Inhibitors</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D503</span><span class="NLM_x">–</span> <span class="NLM_lpage">D509</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1093%2Fnar%2Fgkt953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=24157837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Cj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D503-D509&author=N.+D.+Rawlingsauthor=M.+Wallerauthor=A.+J.+Barrettauthor=A.+Bateman&title=MEROPS%3A+the+Database+of+Proteolytic+Enzymes%2C+Their+Substrates+and+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MEROPS: the database of proteolytic enzymes, their substrates and inhibitors</span></div><div class="casAuthors">Rawlings, Neil D.; Waller, Matthew; Barrett, Alan J.; Bateman, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D503-D509</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Peptidases, their substrates and inhibitors are of great relevance to biol., medicine and biotechnol.  The MEROPS database (http://merops.sanger.ac.uk) aims to fulfill the need for an integrated source of information about these.  The database has hierarchical classifications in which homologous sets of peptidases and protein inhibitors are grouped into protein species, which are grouped into families, which are in turn grouped into clans.  Recent developments include the following.  A community annotation project has been instigated in which acknowledged experts are invited to contribute summaries for peptidases.  Software has been written to provide an Internet-based data entry form.  Contributors are acknowledged on the relevant web page.  A new display showing the intron/exon structures of eukaryote peptidase genes and the phasing of the junctions has been implemented.  It is now possible to filter the list of peptidases from a completely sequenced bacterial genome for a particular strain of the organism.  The MEROPS filing pipeline has been altered to circumvent the restrictions imposed on non-interactive blastp searches, and a HMMER search using specially generated alignments to maximize the distribution of organisms returned in the search results has been added.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmCK1nNJsPmrVg90H21EOLACvtfcHk0lhMPJUzX6O-pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Cj&md5=97e3fe5e7e2e09bd1971efcd425d22bc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt953%26sid%3Dliteratum%253Aachs%26aulast%3DRawlings%26aufirst%3DN.%2BD.%26aulast%3DWaller%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DA.%2BJ.%26aulast%3DBateman%26aufirst%3DA.%26atitle%3DMEROPS%253A%2520the%2520Database%2520of%2520Proteolytic%2520Enzymes%252C%2520Their%2520Substrates%2520and%2520Inhibitors%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD503%26epage%3DD509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Page, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Cera, E.</span><span> </span><span class="NLM_article-title">Serine Peptidases: Classification, Structure and Function</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1220</span><span class="NLM_x">–</span> <span class="NLM_lpage">1236</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1007%2Fs00018-008-7565-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=18259688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFynsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2008&pages=1220-1236&author=M.+J.+Pageauthor=E.+Di+Cera&title=Serine+Peptidases%3A+Classification%2C+Structure+and+Function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Serine peptidases: Classification, structure and function</span></div><div class="casAuthors">Page, M. J.; Di Cera, E.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">1220-1236</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Serine proteases play key roles in human health and disease and their biochem. properties have shaped the mol. evolution of these processes.  Of known proteolytic enzymes, the serine protease family is the major cornerstone of the vertebrate degradome; over 1/3 of all known proteolytic enzymes are serine proteases.  The authors describe the known diversity of serine proteases with respect to structure and function.  In the protease database MEROPS, >26,000 serine proteases are grouped into 13 clans and 40 families.  Particular emphasis is placed on the S1 protease family, the trypsins, which underwent the most predominant genetic expansion yielding the enzymes responsible for vital processes in man such as digestion, blood coagulation, fibrinolysis, development, fertilization, apoptosis, and immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoigIRqJJCserVg90H21EOLACvtfcHk0lhMPJUzX6O-pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFynsLs%253D&md5=cf9fa1b377a106e5ae42a7c2edea789d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs00018-008-7565-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-008-7565-9%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BJ.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DSerine%2520Peptidases%253A%2520Classification%252C%2520Structure%2520and%2520Function%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2008%26volume%3D65%26spage%3D1220%26epage%3D1236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Bugge, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antalis, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span> </span><span class="NLM_article-title">Type II Transmembrane Serine Proteases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">23177</span><span class="NLM_x">–</span> <span class="NLM_lpage">23181</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1074%2Fjbc.R109.021006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=19487698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWqtrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=23177-23181&author=T.+H.+Buggeauthor=T.+M.+Antalisauthor=Q.+Wu&title=Type+II+Transmembrane+Serine+Proteases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Type II transmembrane serine proteases</span></div><div class="casAuthors">Bugge, Thomas H.; Antalis, Toni M.; Wu, Qingyu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">23177-23181</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Anal. of genome and expressed sequence tag data bases at the turn of the millennium unveiled a new protease family named the type II transmembrane serine proteases (TTSPs).  Since then, the no. of known TTSPs has more than doubled, and more importantly, the understanding of the physiol. functions of individual TTSPs and their contribution to human disease has greatly increased.  Progress has also been made in identifying mol. substrates and endogenous inhibitors.  Here, the authors summarize current knowledge of the rapidly advancing TTSP field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg6e9fDiyCZbVg90H21EOLACvtfcHk0liF6vXinotr3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWqtrrL&md5=90a98263918e10142083c497307834b2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R109.021006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R109.021006%26sid%3Dliteratum%253Aachs%26aulast%3DBugge%26aufirst%3DT.%2BH.%26aulast%3DAntalis%26aufirst%3DT.%2BM.%26aulast%3DWu%26aufirst%3DQ.%26atitle%3DType%2520II%2520Transmembrane%2520Serine%2520Proteases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D23177%26epage%3D23181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Oberst, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickson, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.-Y.</span><span> </span><span class="NLM_article-title">The Activation of Matriptase Requires Its Noncatalytic Domains, Serine Protease Domain, and Its Cognate Inhibitor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">26773</span><span class="NLM_x">–</span> <span class="NLM_lpage">26779</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=26773-26779&author=M.+D.+Oberstauthor=C.+A.+Williamsauthor=R.+B.+Dicksonauthor=M.+D.+Johnsonauthor=C.-Y.+Lin&title=The+Activation+of+Matriptase+Requires+Its+Noncatalytic+Domains%2C+Serine+Protease+Domain%2C+and+Its+Cognate+Inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOberst%26aufirst%3DM.%2BD.%26aulast%3DWilliams%26aufirst%3DC.%2BA.%26aulast%3DDickson%26aufirst%3DR.%2BB.%26aulast%3DJohnson%26aufirst%3DM.%2BD.%26aulast%3DLin%26aufirst%3DC.-Y.%26atitle%3DThe%2520Activation%2520of%2520Matriptase%2520Requires%2520Its%2520Noncatalytic%2520Domains%252C%2520Serine%2520Protease%2520Domain%252C%2520and%2520Its%2520Cognate%2520Inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D26773%26epage%3D26779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Jiang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuo, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span> </span><span class="NLM_article-title"><i>N</i>-Glycosylation Is Required for Matriptase-2 Autoactivation and Ectodomain Shedding</span> <span class="citation_source-journal">J. Biol. Chemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">19500</span><span class="NLM_x">–</span> <span class="NLM_lpage">19507</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1074%2Fjbc.M114.555110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=24867957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCktLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=19500-19507&author=J.+Jiangauthor=J.+Yangauthor=P.+Fengauthor=B.+Zuoauthor=N.+Dongauthor=Q.+Wuauthor=Y.+He&title=N-Glycosylation+Is+Required+for+Matriptase-2+Autoactivation+and+Ectodomain+Shedding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">N-Glycosylation is required for matriptase-2 autoactivation and ectodomain shedding</span></div><div class="casAuthors">Jiang, Jiang; Yang, Jianfeng; Feng, Ping; Zuo, Bin; Dong, Ningzheng; Wu, Qingyu; He, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">19500-19507</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Matriptase-2 (I) is a hepatic membrane serine protease that regulates Fe homeostasis.  Defects in I cause Fe deficiency anemia.  In cells, I is synthesized as a zymogen.  To date, how I expression and activation are regulated remains poorly understood.  Here, the authors expressed human I in HEK293 and hepatic BEL-7402, SMMC-7721, and QGY-7703 cells.  By labeling cell surface proteins and Western anal., the authors examd. I cell surface expression, zymogen activation, and ectodomain shedding.  The results showed that I was activated on the cell surface but not intracellularly.  Activated I underwent ectodomain shedding, producing sol. fragments in the conditioned medium.  By testing inactive mutants, R576A and S762A, the authors found that I activation and shedding were mediated by its own catalytic activity and that the one-chain form of I had little activity in ectodomain shedding.  The authors made addnl. I mutants (N136Q, N184Q, N216Q, N338Q, N433Q, N453Q, and N518Q), in which each of the predicted N-glycosylation sites was mutated.  All of these mutants were expressed on the cell surface.  However, mutants N216Q, N453Q, and N518Q, but not the other mutants, had impaired zymogen activation and ectodomain shedding.  These results indicated that N-glycans at specific sites are crit. for I activation.  Together, these data provide new insights into the cell surface expression, zymogen activation, and ectodomain shedding of I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2g18hlGd6rVg90H21EOLACvtfcHk0liF6vXinotr3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCktLrE&md5=7564171896db7b44c9da0ecbe29193f1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.555110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.555110%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DP.%26aulast%3DZuo%26aufirst%3DB.%26aulast%3DDong%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%26atitle%3DN-Glycosylation%2520Is%2520Required%2520for%2520Matriptase-2%2520Autoactivation%2520and%2520Ectodomain%2520Shedding%26jtitle%3DJ.%2520Biol.%2520Chemistry%26date%3D2014%26volume%3D289%26spage%3D19500%26epage%3D19507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Szabo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobson, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christoph, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">List, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugge, T. H.</span><span> </span><span class="NLM_article-title">Regulation of Cell Surface Protease Matriptase by HAI2 Is Essential for Placental Development, Neural Tube Closure and Embryonic Survival in Mice</span> <span class="citation_source-journal">Development</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">2653</span><span class="NLM_x">–</span> <span class="NLM_lpage">2663</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=2653-2663&author=R.+Szaboauthor=J.+P.+Hobsonauthor=K.+Christophauthor=P.+Kosaauthor=K.+Listauthor=T.+H.+Bugge&title=Regulation+of+Cell+Surface+Protease+Matriptase+by+HAI2+Is+Essential+for+Placental+Development%2C+Neural+Tube+Closure+and+Embryonic+Survival+in+Mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSzabo%26aufirst%3DR.%26aulast%3DHobson%26aufirst%3DJ.%2BP.%26aulast%3DChristoph%26aufirst%3DK.%26aulast%3DKosa%26aufirst%3DP.%26aulast%3DList%26aufirst%3DK.%26aulast%3DBugge%26aufirst%3DT.%2BH.%26atitle%3DRegulation%2520of%2520Cell%2520Surface%2520Protease%2520Matriptase%2520by%2520HAI2%2520Is%2520Essential%2520for%2520Placental%2520Development%252C%2520Neural%2520Tube%2520Closure%2520and%2520Embryonic%2520Survival%2520in%2520Mice%26jtitle%3DDevelopment%26date%3D2009%26volume%3D136%26spage%3D2653%26epage%3D2663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">List, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haudenschild, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szabo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaim, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelholm, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Behrendt, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugge, T. H.</span><span> </span><span class="NLM_article-title">Matriptase/MT-SP1 Is Required for Postnatal Survival, Epidermal Barrier Function, Hair Follicle Development, and Thymic Homeostasis</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">3765</span><span class="NLM_x">–</span> <span class="NLM_lpage">3779</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=3765-3779&author=K.+Listauthor=C.+C.+Haudenschildauthor=R.+Szaboauthor=W.+Chenauthor=S.+M.+Wahlauthor=W.+Swaimauthor=L.+H.+Engelholmauthor=N.+Behrendtauthor=T.+H.+Bugge&title=Matriptase%2FMT-SP1+Is+Required+for+Postnatal+Survival%2C+Epidermal+Barrier+Function%2C+Hair+Follicle+Development%2C+and+Thymic+Homeostasis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DList%26aufirst%3DK.%26aulast%3DHaudenschild%26aufirst%3DC.%2BC.%26aulast%3DSzabo%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWahl%26aufirst%3DS.%2BM.%26aulast%3DSwaim%26aufirst%3DW.%26aulast%3DEngelholm%26aufirst%3DL.%2BH.%26aulast%3DBehrendt%26aufirst%3DN.%26aulast%3DBugge%26aufirst%3DT.%2BH.%26atitle%3DMatriptase%252FMT-SP1%2520Is%2520Required%2520for%2520Postnatal%2520Survival%252C%2520Epidermal%2520Barrier%2520Function%252C%2520Hair%2520Follicle%2520Development%252C%2520and%2520Thymic%2520Homeostasis%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D3765%26epage%3D3779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Milner, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swingler, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelso, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donell, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cawston, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrell, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plevin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowan, A. D.</span><span> </span><span class="NLM_article-title">Matriptase Is a Novel Initiator of Cartilage Matrix Degradation in Osteoarthritis</span> <span class="citation_source-journal">Arthritis Rheum.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">1955</span><span class="NLM_x">–</span> <span class="NLM_lpage">1966</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=20506309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1aqsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=1955-1966&author=J.+M.+Milnerauthor=A.+Patelauthor=R.+K.+Davidsonauthor=T.+E.+Swinglerauthor=A.+D%C3%A9siletsauthor=D.+A.+Youngauthor=E.+B.+Kelsoauthor=S.+T.+Donellauthor=T.+E.+Cawstonauthor=I.+M.+Clarkauthor=W.+R.+Ferrellauthor=R.+Plevinauthor=J.+C.+Lockhartauthor=R.+Leducauthor=A.+D.+Rowan&title=Matriptase+Is+a+Novel+Initiator+of+Cartilage+Matrix+Degradation+in+Osteoarthritis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Matriptase is a novel initiator of cartilage matrix degradation in osteoarthritis</span></div><div class="casAuthors">Milner, Jennifer M.; Patel, Amit; Davidson, Rose K.; Swingler, Tracey E.; Desilets, Antoine; Young, David A.; Kelso, Elizabeth B.; Donell, Simon T.; Cawston, Tim E.; Clark, Ian M.; Ferrell, William R.; Plevin, Robin; Lockhart, John C.; Leduc, Richard; Rowan, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatism</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1955-1966</span>CODEN:
                <span class="NLM_cas:coden">ARHEAW</span>;
        ISSN:<span class="NLM_cas:issn">0004-3591</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Objective. Increasing evidence implicates serine proteinases in pathol. tissue turnover.  The aim of this study was to assess the role of the transmembrane serine proteinase matriptase in cartilage destruction in osteoarthritis (OA).  Methods. Serine proteinase gene expression in femoral head cartilage obtained from either patients with hip OA or patients with fracture to the neck of the femur (NOF) was assessed using a low-d. array.  The effect of matriptase on collagen breakdown was detd. in cartilage degrdn. models, while the effect on matrix metalloproteinase (MMP) expression was analyzed by real-time polymerase chain reaction.  ProMMP processing was detd. using sodium dodecyl sulfate-polyacrylamide gel electrophoresis/N-terminal sequencing, while its ability to activate proteinase-activated receptor 2 (PAR-2) was detd. using a synovial perfusion assay in mice.  Results. Matriptase gene expression was significantly elevated in OA cartilage compared with NOF cartilage, and matriptase was immunolocalized to OA chondrocytes.  We showed that matriptase activated proMMP-1 and processed proMMP-3 to its fully active form.  Exogenous matriptase significantly enhanced cytokine-stimulated cartilage collagenolysis, while matriptase alone caused significant collagenolysis from OA cartilage, which was metalloproteinase-dependent.  Matriptase also induced MMP-1, MMP-3, and MMP-13 gene expression.  Synovial perfusion data confirmed that matriptase activates PAR-2, and we demonstrated that matriptase-dependent enhancement of collagenolysis from OA cartilage is blocked by PAR-2 inhibition.  Conclusion. Elevated matriptase expression in OA and the ability of matriptase to activate selective proMMPs as well as induce collagenase expression make this serine proteinase a key initiator and inducer of cartilage destruction in OA.  We propose that the indirect effects of matriptase are mediated by PAR-2, and a more detailed understanding of these mechanisms may highlight important new therapeutic targets for OA treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGNctphoRE3bVg90H21EOLACvtfcHk0lh_h7QGs9Pmng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1aqsL%252FL&md5=af039e6d95a18d1b9877eaa9d75b0672</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMilner%26aufirst%3DJ.%2BM.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDavidson%26aufirst%3DR.%2BK.%26aulast%3DSwingler%26aufirst%3DT.%2BE.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DD.%2BA.%26aulast%3DKelso%26aufirst%3DE.%2BB.%26aulast%3DDonell%26aufirst%3DS.%2BT.%26aulast%3DCawston%26aufirst%3DT.%2BE.%26aulast%3DClark%26aufirst%3DI.%2BM.%26aulast%3DFerrell%26aufirst%3DW.%2BR.%26aulast%3DPlevin%26aufirst%3DR.%26aulast%3DLockhart%26aufirst%3DJ.%2BC.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DRowan%26aufirst%3DA.%2BD.%26atitle%3DMatriptase%2520Is%2520a%2520Novel%2520Initiator%2520of%2520Cartilage%2520Matrix%2520Degradation%2520in%2520Osteoarthritis%26jtitle%3DArthritis%2520Rheum.%26date%3D2010%26volume%3D62%26spage%3D1955%26epage%3D1966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Basel-Vanagaite, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Attia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishida-Yamamoto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rainshtein, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ben Amitai, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lurie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasmanik-Chor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Indelman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zvulunov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saban, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magal, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprecher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shohat, M.</span><span> </span><span class="NLM_article-title">Autosomal Recessive Ichthyosis with Hypotrichosis Caused by a Mutation in ST14, Encoding Type II Transmembrane Serine Protease Matriptase</span> <span class="citation_source-journal">Am. J. Hum. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1086%2F512487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=17273967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1Wksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2007&pages=467-477&author=L.+Basel-Vanagaiteauthor=R.+Attiaauthor=A.+Ishida-Yamamotoauthor=L.+Rainshteinauthor=D.+Ben+Amitaiauthor=R.+Lurieauthor=M.+Pasmanik-Chorauthor=M.+Indelmanauthor=A.+Zvulunovauthor=S.+Sabanauthor=N.+Magalauthor=E.+Sprecherauthor=M.+Shohat&title=Autosomal+Recessive+Ichthyosis+with+Hypotrichosis+Caused+by+a+Mutation+in+ST14%2C+Encoding+Type+II+Transmembrane+Serine+Protease+Matriptase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Autosomal recessive ichthyosis with hypotrichosis caused by a mutation in ST14, encoding type II transmembrane serine protease matriptase</span></div><div class="casAuthors">Basel-Vanagaite, Lina; Attia, Revital; Ishida-Yamamoto, Akemi; Rainshtein, Limor; Ben Amitai, Dan; Lurie, Raziel; Pasmanik-Chor, Metsada; Indelman, Margarita; Zvulunov, Alex; Saban, Shirley; Magal, Nurit; Sprecher, Eli; Shohat, Mordechai</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">467-477</span>CODEN:
                <span class="NLM_cas:coden">AJHGAG</span>;
        ISSN:<span class="NLM_cas:issn">0002-9297</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">In this article, we describe a novel autosomal recessive ichthyosis with hypotrichosis syndrome, characterized by congenital ichthyosis assocd. with abnormal hair.  Using homozygosity mapping, we mapped the disease locus to 11q24.3-q25.  We screened the ST14 gene, which encodes matriptase, since transplantation of skin from matriptase-/--knockout mice onto adult athymic nude mice has been shown elsewhere to result in an ichthyosislike phenotype assocd. with almost complete absence of erupted pelage hairs.  Mutation anal. revealed a missense mutation, G827R, in the highly conserved peptidase S1-S6 domain.  Marked skin hyperkeratosis due to impaired degrdn. of the stratum corneum corneodesmosomes was obsd. in the affected individuals, which suggests that matriptase plays a significant role in epidermal desquamation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTeYW7jyml8rVg90H21EOLACvtfcHk0lh_h7QGs9Pmng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1Wksr4%253D&md5=ee1d1f822e3a4b767f196cff5635e86b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1086%2F512487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F512487%26sid%3Dliteratum%253Aachs%26aulast%3DBasel-Vanagaite%26aufirst%3DL.%26aulast%3DAttia%26aufirst%3DR.%26aulast%3DIshida-Yamamoto%26aufirst%3DA.%26aulast%3DRainshtein%26aufirst%3DL.%26aulast%3DBen%2BAmitai%26aufirst%3DD.%26aulast%3DLurie%26aufirst%3DR.%26aulast%3DPasmanik-Chor%26aufirst%3DM.%26aulast%3DIndelman%26aufirst%3DM.%26aulast%3DZvulunov%26aufirst%3DA.%26aulast%3DSaban%26aufirst%3DS.%26aulast%3DMagal%26aufirst%3DN.%26aulast%3DSprecher%26aufirst%3DE.%26aulast%3DShohat%26aufirst%3DM.%26atitle%3DAutosomal%2520Recessive%2520Ichthyosis%2520with%2520Hypotrichosis%2520Caused%2520by%2520a%2520Mutation%2520in%2520ST14%252C%2520Encoding%2520Type%2520II%2520Transmembrane%2520Serine%2520Protease%2520Matriptase%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2007%26volume%3D80%26spage%3D467%26epage%3D477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Bergum, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">List, K.</span><span> </span><span class="NLM_article-title">Loss of the Matriptase Inhibitor HAI-2 During Prostate Cancer Progression</span> <span class="citation_source-journal">Prostate</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">1422</span><span class="NLM_x">–</span> <span class="NLM_lpage">1428</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=1422-1428&author=C.+Bergumauthor=K.+List&title=Loss+of+the+Matriptase+Inhibitor+HAI-2+During+Prostate+Cancer+Progression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBergum%26aufirst%3DC.%26aulast%3DList%26aufirst%3DK.%26atitle%3DLoss%2520of%2520the%2520Matriptase%2520Inhibitor%2520HAI-2%2520During%2520Prostate%2520Cancer%2520Progression%26jtitle%3DProstate%26date%3D2010%26volume%3D70%26spage%3D1422%26epage%3D1428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Jin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyama, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirosaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higashi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickson, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitamura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyazaki, K.</span><span> </span><span class="NLM_article-title">Matriptase Activates Stromelysin (MMP-3) and Promotes Tumor Growth and Angiogenesis</span> <span class="citation_source-journal">Cancer Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1327</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1111%2Fj.1349-7006.2006.00328.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=16999819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1OgurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2006&pages=1327-1334&author=X.+Jinauthor=M.+Yagiauthor=N.+Akiyamaauthor=T.+Hirosakiauthor=S.+Higashiauthor=C.-Y.+Linauthor=R.+B.+Dicksonauthor=H.+Kitamuraauthor=K.+Miyazaki&title=Matriptase+Activates+Stromelysin+%28MMP-3%29+and+Promotes+Tumor+Growth+and+Angiogenesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Matriptase activates stromelysin (MMP-3) and promotes tumor growth and angiogenesis</span></div><div class="casAuthors">Jin, Xinlian; Yagi, Motoki; Akiyama, Nagisa; Hirosaki, Tomomi; Higashi, Shouichi; Lin, Chen-Yong; Dickson, Robert B.; Kitamura, Hitoshi; Miyazaki, Kaoru</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1327-1334</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1347-9032</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">Matriptase/MT-SP1, a type II membrane serine protease widely expressed in normal epithelial cells and human carcinoma cells, is thought to be involved in cancer progression.  To clarify this possibility, we overexpressed exogenous matriptase in the human stomach cancer cell line AZ521.  In vitro, the matriptase transfectant (Mat-AZ521) and the control transfectant (Mock-AZ521) showed a similar growth rate, although the satn. cell d. was significantly higher with the Mat-AZ521.  When implanted into nude mice s.c. or i.p., Mat-AZ521 cells grew faster and produced much larger solid tumors than Mock-AZ521 cells.  The overexpression of matriptase in AZ521 cells shortened the survival time of tumor-bearing mice.  Histol. anal. showed that both the no. and the size of blood vessels in tumor tissues were significantly higher in the Mat-AZ521 tumors than the Mock-AZ521 ones.  Moreover, it was found that purified matriptase activated one of the important matrix metalloproteinases, stromelysin (MMP-3).  These results suggest the possibility that the matriptase-dependent activation of MMP-3, as well as the direct activity of matriptase, promotes tumor growth and angiogenesis by enhancing extracellular matrix degrdn. in tumor cell microenvironments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEauNcAfWLhbVg90H21EOLACvtfcHk0liNNa1p9Fy-DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1OgurrI&md5=07758d9ce9d8a0de78e08f0f1bceca75</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2006.00328.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2006.00328.x%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DX.%26aulast%3DYagi%26aufirst%3DM.%26aulast%3DAkiyama%26aufirst%3DN.%26aulast%3DHirosaki%26aufirst%3DT.%26aulast%3DHigashi%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.-Y.%26aulast%3DDickson%26aufirst%3DR.%2BB.%26aulast%3DKitamura%26aufirst%3DH.%26aulast%3DMiyazaki%26aufirst%3DK.%26atitle%3DMatriptase%2520Activates%2520Stromelysin%2520%2528MMP-3%2529%2520and%2520Promotes%2520Tumor%2520Growth%2520and%2520Angiogenesis%26jtitle%3DCancer%2520Sci.%26date%3D2006%26volume%3D97%26spage%3D1327%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Köbel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalloger, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyd, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinney, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowen, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ionescu, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajput, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prentice, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swenerton, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilks, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntsman, D.</span><span> </span><span class="NLM_article-title">Ovarian Carcinoma Subtypes Are Different Diseases: Implications for Biomarker Studies</span> <span class="citation_source-journal">PLoS Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e232</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=e232&author=M.+K%C3%B6belauthor=S.+E.+Kallogerauthor=N.+Boydauthor=S.+McKinneyauthor=E.+Mehlauthor=C.+Palmerauthor=S.+Leungauthor=N.+J.+Bowenauthor=D.+N.+Ionescuauthor=A.+Rajputauthor=L.+M.+Prenticeauthor=D.+Millerauthor=J.+Santosauthor=K.+Swenertonauthor=C.+B.+Gilksauthor=D.+Huntsman&title=Ovarian+Carcinoma+Subtypes+Are+Different+Diseases%3A+Implications+for+Biomarker+Studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6bel%26aufirst%3DM.%26aulast%3DKalloger%26aufirst%3DS.%2BE.%26aulast%3DBoyd%26aufirst%3DN.%26aulast%3DMcKinney%26aufirst%3DS.%26aulast%3DMehl%26aufirst%3DE.%26aulast%3DPalmer%26aufirst%3DC.%26aulast%3DLeung%26aufirst%3DS.%26aulast%3DBowen%26aufirst%3DN.%2BJ.%26aulast%3DIonescu%26aufirst%3DD.%2BN.%26aulast%3DRajput%26aufirst%3DA.%26aulast%3DPrentice%26aufirst%3DL.%2BM.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DSantos%26aufirst%3DJ.%26aulast%3DSwenerton%26aufirst%3DK.%26aulast%3DGilks%26aufirst%3DC.%2BB.%26aulast%3DHuntsman%26aufirst%3DD.%26atitle%3DOvarian%2520Carcinoma%2520Subtypes%2520Are%2520Different%2520Diseases%253A%2520Implications%2520for%2520Biomarker%2520Studies%26jtitle%3DPLoS%2520Med.%26date%3D2008%26volume%3D5%26spage%3De232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">List, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szabo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bey, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molinolo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugge, T. H.</span><span> </span><span class="NLM_article-title">Epithelial Integrity Is Maintained by a Matriptase-Dependent Proteolytic Pathway</span> <span class="citation_source-journal">Am. J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">175</span><span class="NLM_x">, </span> <span class="NLM_fpage">1453</span><span class="NLM_x">–</span> <span class="NLM_lpage">1463</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.2353%2Fajpath.2009.090240" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=19717635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlWiur%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2009&pages=1453-1463&author=K.+Listauthor=P.+Kosaauthor=R.+Szaboauthor=A.+L.+Beyauthor=C.+B.+Wangauthor=A.+Molinoloauthor=T.+H.+Bugge&title=Epithelial+Integrity+Is+Maintained+by+a+Matriptase-Dependent+Proteolytic+Pathway"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Epithelial integrity is maintained by a matriptase-dependent proteolytic pathway</span></div><div class="casAuthors">List, Karin; Kosa, Peter; Szabo, Roman; Bey, Alexandra L.; Wang, Chao Becky; Molinolo, Alfredo; Bugge, Thomas H.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1453-1463</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">A pericellular proteolytic pathway initiated by the transmembrane serine protease matriptase plays a crit. role in the terminal differentiation of epidermal tissues.  Matriptase is constitutively expressed in multiple other epithelia, suggesting a putative role of this membrane serine protease in general epithelial homeostasis.  Here we generated mice with conditional deletion of the St14 gene, encoding matriptase, and show that matriptase indeed is essential for the maintenance of multiple types of epithelia in the mouse.  Thus, embryonic or postnatal ablation of St14 in epithelial tissues of diverse origin and function caused severe organ dysfunction, which was often assocd. with increased permeability, loss of tight junction function, mislocation of tight junction-assocd. proteins, and generalized epithelial demise.  The study reveals that the homeostasis of multiple simple and stratified epithelia is matriptase-dependent, and provides an important animal model for the exploration of this membrane serine protease in a range of physiol. and pathol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYPCXcYcgd4rVg90H21EOLACvtfcHk0ljhoauhP3UmSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlWiur%252FK&md5=2cc86115b8ac59b1a7636558d02fecae</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2009.090240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2009.090240%26sid%3Dliteratum%253Aachs%26aulast%3DList%26aufirst%3DK.%26aulast%3DKosa%26aufirst%3DP.%26aulast%3DSzabo%26aufirst%3DR.%26aulast%3DBey%26aufirst%3DA.%2BL.%26aulast%3DWang%26aufirst%3DC.%2BB.%26aulast%3DMolinolo%26aufirst%3DA.%26aulast%3DBugge%26aufirst%3DT.%2BH.%26atitle%3DEpithelial%2520Integrity%2520Is%2520Maintained%2520by%2520a%2520Matriptase-Dependent%2520Proteolytic%2520Pathway%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2009%26volume%3D175%26spage%3D1453%26epage%3D1463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Hamilton, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gludish, D. W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, G. R.</span><span> </span><span class="NLM_article-title">Cleavage Activation of the Human-Adapted Influenza Virus Subtypes by Matriptase Reveals Both Subtype and Strain Specificities</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">10579</span><span class="NLM_x">–</span> <span class="NLM_lpage">10586</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1128%2FJVI.00306-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=22811538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOmtb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=10579-10586&author=B.+S.+Hamiltonauthor=D.+W.+J.+Gludishauthor=G.+R.+Whittaker&title=Cleavage+Activation+of+the+Human-Adapted+Influenza+Virus+Subtypes+by+Matriptase+Reveals+Both+Subtype+and+Strain+Specificities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cleavage activation of the human-adapted influenza virus subtypes by matriptase reveals both subtype and strain specificities</span></div><div class="casAuthors">Hamilton, Brian S.; Gludish, David W. J.; Whittaker, Gary R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10579-10586</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cleavage activation of the hemagglutinin (HA) precursor is an essential step in the influenza virus replication cycle that is driven by host cell proteases.  HA cleavage activation is required for virus-endosome membrane fusion and the subsequent release of the influenza virus genome into the cytoplasm.  Previous studies have detd. that HA cleavage is most likely driven by either membrane-bound or extracellular trypsin-like proteases that reside in the respiratory tract.  However, there is still uncertainty regarding which proteases are crit. for HA cleavage in vivo.  Therefore, further investigation of HA cleavage activation is needed in order to gain insight into the crit. proteases involved.  Matriptase is a member of the type II transmembrane serine protease family that is highly expressed in a membrane-bound form throughout the respiratory tract.  One feature of matriptase is that, once activated, the catalytic domain is secreted into the extracellular space and so serves as a functional extracellular protease.  In this study, it was detd. that the secreted, catalytic domain of matriptase has the ability to cleave and activate HA from the influenza virus H1 subtype but not the H2 and H3 subtypes.  Furthermore, matriptase selectively cleaved the HA of particular strains within the H1 subtype, revealing both subtype and H1 strain specificity.  Matriptase was also found to activate thrombolytic zymogens that have been shown to cleave and activate the influenza virus HA.  The data demonstrate that matriptase has the ability to cleave HA directly or indirectly by activating HA-cleaving zymogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXLuhD8xGHLLVg90H21EOLACvtfcHk0ljhoauhP3UmSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOmtb3L&md5=d82d35942ca30280356c6d35a518f3f4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FJVI.00306-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00306-12%26sid%3Dliteratum%253Aachs%26aulast%3DHamilton%26aufirst%3DB.%2BS.%26aulast%3DGludish%26aufirst%3DD.%2BW.%2BJ.%26aulast%3DWhittaker%26aufirst%3DG.%2BR.%26atitle%3DCleavage%2520Activation%2520of%2520the%2520Human-Adapted%2520Influenza%2520Virus%2520Subtypes%2520by%2520Matriptase%2520Reveals%2520Both%2520Subtype%2520and%2520Strain%2520Specificities%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D10579%26epage%3D10586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Beaulieu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gravel, É.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cloutier, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marois, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verreault, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsault, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M. V.</span><span> </span><span class="NLM_article-title">Matriptase Proteolytically Activates Influenza Virus and Promotes Multicycle Replication in the Human Airway Epithelium</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">4237</span><span class="NLM_x">–</span> <span class="NLM_lpage">4251</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1128%2FJVI.03005-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23365447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotVyqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=4237-4251&author=A.+Beaulieuauthor=%C3%89.+Gravelauthor=A.+Cloutierauthor=I.+Maroisauthor=E.+Colomboauthor=A.+D%C3%A9siletsauthor=C.+Verreaultauthor=R.+Leducauthor=E.+Marsaultauthor=M.+V.+Richter&title=Matriptase+Proteolytically+Activates+Influenza+Virus+and+Promotes+Multicycle+Replication+in+the+Human+Airway+Epithelium"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Matriptase proteolytically activates influenza virus and promotes multicycle replication in the human airway epithelium</span></div><div class="casAuthors">Beaulieu, Alexandre; Gravel, Emilie; Cloutier, Alexandre; Marois, Isabelle; Colombo, Eloic; Desilets, Antoine; Verreault, Catherine; Leduc, Richard; Marsault, Eric; Richter, Martin V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4237-4251</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Influenza viruses do not encode any proteases and must rely on host proteases for the proteolytic activation of their surface hemagglutinin proteins to fuse with the infected host cells.  Recent progress in the understanding of human proteases responsible for influenza virus hemagglutinin activation led to the identification of members of the type II transmembrane serine proteases TMPRSS2 and TMPRSS4 and human airway trypsin-like protease; however, none proved to be the sole enzyme responsible for hemagglutinin cleavage.  The authors identify and characterize matriptase as an influenza virus-activating protease capable of supporting multicycle viral replication in the human respiratory epithelium.  Using confocal microscopy, the authors found matriptase to colocalize with hemagglutinin at the apical surface of human epithelial cells and within endosomes, and the sol. form of the protease was able to specifically cleave hemagglutinins from H1 virus, but not from H2 and H3 viruses, in a broad pH range.  Small interfering RNA (siRNA) knockdown of matriptase in human bronchial epithelial cells significantly blocked influenza virus replication in these cells.  Lastly, the authors provide a selective, slow, tight-binding inhibitor of matriptase that significantly reduces viral replication (by 1.5 log) of H1N1 influenza virus, including the 2009 pandemic virus.  The authors' study establishes a three-pronged model for the action of matriptase: activation of incoming viruses in the extracellular space in its shed form, upon viral attachment or exit in its membrane-bound and/or shed forms at the apical surface of epithelial cells, and within endosomes by its membrane-bound form where viral fusion takes place.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWmz0DwP1-qLVg90H21EOLACvtfcHk0ljhoauhP3UmSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotVyqsb4%253D&md5=caaa3417ca7693113dcdb07b219bd68b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJVI.03005-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03005-12%26sid%3Dliteratum%253Aachs%26aulast%3DBeaulieu%26aufirst%3DA.%26aulast%3DGravel%26aufirst%3D%25C3%2589.%26aulast%3DCloutier%26aufirst%3DA.%26aulast%3DMarois%26aufirst%3DI.%26aulast%3DColombo%26aufirst%3DE.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DVerreault%26aufirst%3DC.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DRichter%26aufirst%3DM.%2BV.%26atitle%3DMatriptase%2520Proteolytically%2520Activates%2520Influenza%2520Virus%2520and%2520Promotes%2520Multicycle%2520Replication%2520in%2520the%2520Human%2520Airway%2520Epithelium%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D4237%26epage%3D4251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Baron, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarnow, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayoli-Nüssle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schilling, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwalm, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinmetzer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garten, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klenk, H.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Böttcher-Friebertshäuser, E.</span><span> </span><span class="NLM_article-title">Matriptase, HAT, and TMPRSS2 Activate the Hemagglutinin of H9N2 Influenza a Viruses</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">1811</span><span class="NLM_x">–</span> <span class="NLM_lpage">1820</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1128%2FJVI.02320-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23192872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltV2itLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=1811-1820&author=J.+Baronauthor=C.+Tarnowauthor=D.+Mayoli-N%C3%BCssleauthor=E.+Schillingauthor=D.+Meyerauthor=M.+Hammamiauthor=F.+Schwalmauthor=T.+Steinmetzerauthor=Y.+Guanauthor=W.+Gartenauthor=H.-D.+Klenkauthor=E.+B%C3%B6ttcher-Friebertsh%C3%A4user&title=Matriptase%2C+HAT%2C+and+TMPRSS2+Activate+the+Hemagglutinin+of+H9N2+Influenza+a+Viruses"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Matriptase, HAT, and TMPRSS2 activate the hemagglutinin of H9N2 influenza A viruses</span></div><div class="casAuthors">Baron, Joanna; Tarnow, Carolin; Mayoli-Nuessle, Deborah; Schilling, Eva; Hammami, Maya; Schwalm, Folker; Steinmetzer, Torsten; Guan, Yi; Garten, Wolfgang; Klenk, Hans-Dieter; Boettcher-Friebertshaeuser, Eva</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1811-1820</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Influenza A viruses of the subtype H9N2 circulate worldwide and have become highly prevalent in poultry in many countries.  Moreover, they are occasionally transmitted to humans, raising concern about their pandemic potential.  Influenza virus infectivity requires cleavage of the surface glycoprotein hemagglutinin (HA) at a distinct cleavage site by host cell proteases.  H9N2 viruses vary remarkably in the amino acid sequence at the cleavage site, and many isolates from Asia and the Middle East possess the multibasic motifs R-S-S-R and R-S-R-R, but are not activated by furin.  Here, we investigated proteolytic activation of the early H9N2 isolate A/turkey/Wisconsin/1/66 (H9-Wisc) and two recent Asian isolates, A/quail/Shantou/782/00 (H9-782) and A/quail/Shantou/2061/00 (H9-2061), contg. mono-, di-, and tribasic HA cleavage sites, resp.  All H9N2 isolates were activated by human proteases TMPRSS2 (transmembrane protease, serine S1 member 2) and HAT (human airway trypsin-like protease).  Interestingly, H9-782 and H9-2061 were also activated by matriptase, a protease widely expressed in most epithelia with high expression levels in the kidney.  Nephrotropism of H9N2 viruses has been obsd. in chickens, and here we found that H9-782 and H9-2061 were proteolytically activated in canine kidney (MDCK-II) and chicken embryo kidney (CEK) cells, whereas H9-Wisc was not.  Virus activation was inhibited by peptide-mimetic inhibitors of matriptase, strongly suggesting that matriptase is responsible for HA cleavage in these kidney cells.  Our data demonstrate that H9N2 viruses with R-S-S-R or R-S-R-R cleavage sites are activated by matriptase in addn. to HAT and TMPRSS2 and, therefore, can be activated in a wide range of tissues what may affect virus spread, tissue tropism and pathogenicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYEOrvA-4zH7Vg90H21EOLACvtfcHk0ljJ4PVcJ53PDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltV2itLg%253D&md5=2bb0516ece577cbbce5ab3130bbb4eed</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1128%2FJVI.02320-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02320-12%26sid%3Dliteratum%253Aachs%26aulast%3DBaron%26aufirst%3DJ.%26aulast%3DTarnow%26aufirst%3DC.%26aulast%3DMayoli-N%25C3%25BCssle%26aufirst%3DD.%26aulast%3DSchilling%26aufirst%3DE.%26aulast%3DMeyer%26aufirst%3DD.%26aulast%3DHammami%26aufirst%3DM.%26aulast%3DSchwalm%26aufirst%3DF.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DGuan%26aufirst%3DY.%26aulast%3DGarten%26aufirst%3DW.%26aulast%3DKlenk%26aufirst%3DH.-D.%26aulast%3DB%25C3%25B6ttcher-Friebertsh%25C3%25A4user%26aufirst%3DE.%26atitle%3DMatriptase%252C%2520HAT%252C%2520and%2520TMPRSS2%2520Activate%2520the%2520Hemagglutinin%2520of%2520H9N2%2520Influenza%2520a%2520Viruses%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D1811%26epage%3D1820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Ramsay, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooper, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folgueras, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">López-Otín, C.</span><span> </span><span class="NLM_article-title">Matriptase-2 (TMPRSS6): A Proteolytic Regulator of Iron Homeostasis</span> <span class="citation_source-journal">Haematologica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">840</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2009&pages=840-849&author=A.+J.+Ramsayauthor=J.+D.+Hooperauthor=A.+R.+Folguerasauthor=G.+Velascoauthor=C.+L%C3%B3pez-Ot%C3%ADn&title=Matriptase-2+%28TMPRSS6%29%3A+A+Proteolytic+Regulator+of+Iron+Homeostasis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRamsay%26aufirst%3DA.%2BJ.%26aulast%3DHooper%26aufirst%3DJ.%2BD.%26aulast%3DFolgueras%26aufirst%3DA.%2BR.%26aulast%3DVelasco%26aufirst%3DG.%26aulast%3DL%25C3%25B3pez-Ot%25C3%25ADn%26aufirst%3DC.%26atitle%3DMatriptase-2%2520%2528TMPRSS6%2529%253A%2520A%2520Proteolytic%2520Regulator%2520of%2520Iron%2520Homeostasis%26jtitle%3DHaematologica%26date%3D2009%26volume%3D94%26spage%3D840%26epage%3D849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">She, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelbart, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truksa, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khovananth, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mudd, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moresco, E. M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beutler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beutler, B.</span><span> </span><span class="NLM_article-title">The Serine Protease TMPRSS6 Is Required to Sense Iron Deficiency</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">1088</span><span class="NLM_x">–</span> <span class="NLM_lpage">1092</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2008&pages=1088-1092&author=X.+Duauthor=E.+Sheauthor=T.+Gelbartauthor=J.+Truksaauthor=P.+Leeauthor=Y.+Xiaauthor=K.+Khovananthauthor=S.+Muddauthor=N.+Mannauthor=E.+M.+Y.+Morescoauthor=E.+Beutlerauthor=B.+Beutler&title=The+Serine+Protease+TMPRSS6+Is+Required+to+Sense+Iron+Deficiency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DX.%26aulast%3DShe%26aufirst%3DE.%26aulast%3DGelbart%26aufirst%3DT.%26aulast%3DTruksa%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DXia%26aufirst%3DY.%26aulast%3DKhovananth%26aufirst%3DK.%26aulast%3DMudd%26aufirst%3DS.%26aulast%3DMann%26aufirst%3DN.%26aulast%3DMoresco%26aufirst%3DE.%2BM.%2BY.%26aulast%3DBeutler%26aufirst%3DE.%26aulast%3DBeutler%26aufirst%3DB.%26atitle%3DThe%2520Serine%2520Protease%2520TMPRSS6%2520Is%2520Required%2520to%2520Sense%2520Iron%2520Deficiency%26jtitle%3DScience%26date%3D2008%26volume%3D320%26spage%3D1088%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Silvestri, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Domenico, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaplan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camaschella, C.</span><span> </span><span class="NLM_article-title">The Serine Protease Matriptase-2 (TMPRSS6) Inhibits Hepcidin Activation by Cleaving Membrane Hemojuvelin</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">502</span><span class="NLM_x">–</span> <span class="NLM_lpage">511</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1016%2Fj.cmet.2008.09.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=18976966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsV2qtbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=502-511&author=L.+Silvestriauthor=A.+Paganiauthor=A.+Naiauthor=I.+De+Domenicoauthor=J.+Kaplanauthor=C.+Camaschella&title=The+Serine+Protease+Matriptase-2+%28TMPRSS6%29+Inhibits+Hepcidin+Activation+by+Cleaving+Membrane+Hemojuvelin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The serine protease matriptase-2 (TMPRSS6) inhibits hepcidin activation by cleaving membrane hemojuvelin</span></div><div class="casAuthors">Silvestri, Laura; Pagani, Alessia; Nai, Antonella; De Domenico, Ivana; Kaplan, Jerry; Camaschella, Clara</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">502-511</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The liver peptide, hepcidin (I), regulates body Fe, is up-regulated in Fe overload and inflammation, and is down-regulated in Fe deficiency/hypoxia.  The transmembrane serine protease, matriptase-2, inhibits the I response and its mutational inactivation causes Fe-deficient anemia in mice and humans.  Here, the authors confirmed the inhibitory effect of matriptase-2 on I promoter; they showed that matriptase-2 lacking the serine protease domain, identified in the anemic Mask mouse (matriptase-2MASK), was fully inactive and that mutant R774C found in patients with genetic Fe deficiency had decreased inhibitory activity.  Matriptase-2 cleaved hemojuvelin (II), a regulator of I, on plasma membranes; matriptase-2MASK showed no cleavage activity and the human mutant showed only partial cleavage capacity.  Matriptase-2 interacted with II through the ectodomain since the interaction was conserved in matriptase-2MASK.  The expression of matriptase-2 mutants in zebrafish resulted in anemia, confirming the matriptase-2 role in Fe metab. and its interaction with II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgJ2IQkeHGILVg90H21EOLACvtfcHk0ljJ4PVcJ53PDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsV2qtbnP&md5=c99d24bb2935383d62094942f1ce6289</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2008.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2008.09.012%26sid%3Dliteratum%253Aachs%26aulast%3DSilvestri%26aufirst%3DL.%26aulast%3DPagani%26aufirst%3DA.%26aulast%3DNai%26aufirst%3DA.%26aulast%3DDe%2BDomenico%26aufirst%3DI.%26aulast%3DKaplan%26aufirst%3DJ.%26aulast%3DCamaschella%26aufirst%3DC.%26atitle%3DThe%2520Serine%2520Protease%2520Matriptase-2%2520%2528TMPRSS6%2529%2520Inhibits%2520Hepcidin%2520Activation%2520by%2520Cleaving%2520Membrane%2520Hemojuvelin%26jtitle%3DCell%2520Metab.%26date%3D2008%26volume%3D8%26spage%3D502%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Silvestri, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillem, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toutain, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kannengiesser, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaumont, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camaschella, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandchamp, B.</span><span> </span><span class="NLM_article-title">Molecular Mechanisms of the Defective Hepcidin Inhibition in TMPRSS6 Mutations Associated with Iron-Refractory Iron Deficiency Anemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">5605</span><span class="NLM_x">–</span> <span class="NLM_lpage">5608</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=5605-5608&author=L.+Silvestriauthor=F.+Guillemauthor=A.+Paganiauthor=A.+Naiauthor=C.+Oudinauthor=M.+Silvaauthor=F.+Toutainauthor=C.+Kannengiesserauthor=C.+Beaumontauthor=C.+Camaschellaauthor=B.+Grandchamp&title=Molecular+Mechanisms+of+the+Defective+Hepcidin+Inhibition+in+TMPRSS6+Mutations+Associated+with+Iron-Refractory+Iron+Deficiency+Anemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSilvestri%26aufirst%3DL.%26aulast%3DGuillem%26aufirst%3DF.%26aulast%3DPagani%26aufirst%3DA.%26aulast%3DNai%26aufirst%3DA.%26aulast%3DOudin%26aufirst%3DC.%26aulast%3DSilva%26aufirst%3DM.%26aulast%3DToutain%26aufirst%3DF.%26aulast%3DKannengiesser%26aufirst%3DC.%26aulast%3DBeaumont%26aufirst%3DC.%26aulast%3DCamaschella%26aufirst%3DC.%26aulast%3DGrandchamp%26aufirst%3DB.%26atitle%3DMolecular%2520Mechanisms%2520of%2520the%2520Defective%2520Hepcidin%2520Inhibition%2520in%2520TMPRSS6%2520Mutations%2520Associated%2520with%2520Iron-Refractory%2520Iron%2520Deficiency%2520Anemia%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26spage%3D5605%26epage%3D5608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Finberg, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heeney, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campagna, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aydinok, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartman, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayo, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuel, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strouse, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markianos, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fleming, M. D.</span><span> </span><span class="NLM_article-title">Mutations in TMPRSS6 Cause Iron-Refractory Iron Deficiency Anemia (IRIDA)</span> <span class="citation_source-journal">Nature Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">571</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1038%2Fng.130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=18408718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltFylu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2008&pages=569-571&author=K.+E.+Finbergauthor=M.+M.+Heeneyauthor=D.+R.+Campagnaauthor=Y.+Aydinokauthor=H.+A.+Pearsonauthor=K.+R.+Hartmanauthor=M.+M.+Mayoauthor=S.+M.+Samuelauthor=J.+J.+Strouseauthor=K.+Markianosauthor=N.+C.+Andrewsauthor=M.+D.+Fleming&title=Mutations+in+TMPRSS6+Cause+Iron-Refractory+Iron+Deficiency+Anemia+%28IRIDA%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in TMPRSS6 cause iron-refractory iron deficiency anemia (IRIDA)</span></div><div class="casAuthors">Finberg, Karin E.; Heeney, Matthew M.; Campagna, Dean R.; Aydinok, Yesim; Pearson, Howard A.; Hartman, Kip R.; Mayo, Mary M.; Samuel, Stewart M.; Strouse, John J.; Markianos, Kyriacos; Andrews, Nancy C.; Fleming, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">569-571</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Iron deficiency is usually attributed to chronic blood loss or inadequate dietary intake.  Here, we show that iron deficiency anemia refractory to oral iron therapy can be caused by germline mutations in TMPRSS6, which encodes a type II transmembrane serine protease produced by the liver that regulates the expression of the systemic iron regulatory hormone hepcidin.  These findings demonstrate that TMPRSS6 is essential for normal systemic iron homeostasis in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN5cCMWgwdt7Vg90H21EOLACvtfcHk0ljq9wXccvIYDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXltFylu7g%253D&md5=8fc8784914e1b566034d913fa9578da2</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fng.130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.130%26sid%3Dliteratum%253Aachs%26aulast%3DFinberg%26aufirst%3DK.%2BE.%26aulast%3DHeeney%26aufirst%3DM.%2BM.%26aulast%3DCampagna%26aufirst%3DD.%2BR.%26aulast%3DAydinok%26aufirst%3DY.%26aulast%3DPearson%26aufirst%3DH.%2BA.%26aulast%3DHartman%26aufirst%3DK.%2BR.%26aulast%3DMayo%26aufirst%3DM.%2BM.%26aulast%3DSamuel%26aufirst%3DS.%2BM.%26aulast%3DStrouse%26aufirst%3DJ.%2BJ.%26aulast%3DMarkianos%26aufirst%3DK.%26aulast%3DAndrews%26aufirst%3DN.%2BC.%26aulast%3DFleming%26aufirst%3DM.%2BD.%26atitle%3DMutations%2520in%2520TMPRSS6%2520Cause%2520Iron-Refractory%2520Iron%2520Deficiency%2520Anemia%2520%2528IRIDA%2529%26jtitle%3DNature%2520Genet.%26date%3D2008%26volume%3D40%26spage%3D569%26epage%3D571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Kew, M. C.</span><span> </span><span class="NLM_article-title">Hepatic Iron Overload and Hepatocellular Carcinoma</span> <span class="citation_source-journal">Liver Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=31-40&author=M.+C.+Kew&title=Hepatic+Iron+Overload+and+Hepatocellular+Carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKew%26aufirst%3DM.%2BC.%26atitle%3DHepatic%2520Iron%2520Overload%2520and%2520Hepatocellular%2520Carcinoma%26jtitle%3DLiver%2520Cancer%26date%3D2014%26volume%3D3%26spage%3D31%26epage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Riba, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rausa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorosina, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cittaro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia Manteiga, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinelli-Boneschi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stupka, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camaschella, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silvestri, L.</span><span> </span><span class="NLM_article-title">A Strong Anti-Inflammatory Signature Revealed by Liver Transcription Profiling of Tmprss6–/– Mice</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e69694</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e69694&author=M.+Ribaauthor=M.+Rausaauthor=M.+Sorosinaauthor=D.+Cittaroauthor=J.+M.+Garcia+Manteigaauthor=A.+Naiauthor=A.+Paganiauthor=F.+Martinelli-Boneschiauthor=E.+Stupkaauthor=C.+Camaschellaauthor=L.+Silvestri&title=A+Strong+Anti-Inflammatory+Signature+Revealed+by+Liver+Transcription+Profiling+of+Tmprss6%E2%80%93%2F%E2%80%93+Mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRiba%26aufirst%3DM.%26aulast%3DRausa%26aufirst%3DM.%26aulast%3DSorosina%26aufirst%3DM.%26aulast%3DCittaro%26aufirst%3DD.%26aulast%3DGarcia%2BManteiga%26aufirst%3DJ.%2BM.%26aulast%3DNai%26aufirst%3DA.%26aulast%3DPagani%26aufirst%3DA.%26aulast%3DMartinelli-Boneschi%26aufirst%3DF.%26aulast%3DStupka%26aufirst%3DE.%26aulast%3DCamaschella%26aufirst%3DC.%26aulast%3DSilvestri%26aufirst%3DL.%26atitle%3DA%2520Strong%2520Anti-Inflammatory%2520Signature%2520Revealed%2520by%2520Liver%2520Transcription%2520Profiling%2520of%2520Tmprss6%25E2%2580%2593%252F%25E2%2580%2593%2520Mice%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De69694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Maurer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gütschow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stirnberg, M.</span><span> </span><span class="NLM_article-title">Hepatocyte Growth Factor Activator Inhibitor Type 2 (HAI-2) Modulates Hepcidin Expression by Inhibiting the Cell Surface Protease Matriptase-2</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">450</span><span class="NLM_x">, </span> <span class="NLM_fpage">583</span><span class="NLM_x">–</span> <span class="NLM_lpage">593</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2013&pages=583-593&author=E.+Maurerauthor=M.+G%C3%BCtschowauthor=M.+Stirnberg&title=Hepatocyte+Growth+Factor+Activator+Inhibitor+Type+2+%28HAI-2%29+Modulates+Hepcidin+Expression+by+Inhibiting+the+Cell+Surface+Protease+Matriptase-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DE.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DStirnberg%26aufirst%3DM.%26atitle%3DHepatocyte%2520Growth%2520Factor%2520Activator%2520Inhibitor%2520Type%25202%2520%2528HAI-2%2529%2520Modulates%2520Hepcidin%2520Expression%2520by%2520Inhibiting%2520the%2520Cell%2520Surface%2520Protease%2520Matriptase-2%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D450%26spage%3D583%26epage%3D593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Galkin, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madison, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agus, D. B.</span><span> </span><span class="NLM_article-title">CVS-3983, a Selective Matriptase Inhibitor, Suppresses the Growth of Androgen Independent Prostate Tumor Xenografts</span> <span class="citation_source-journal">Prostate</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">228</span><span class="NLM_x">–</span> <span class="NLM_lpage">235</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2004&pages=228-235&author=A.+V.+Galkinauthor=L.+Mullenauthor=W.+D.+Foxauthor=J.+Brownauthor=D.+Duncanauthor=O.+Morenoauthor=E.+L.+Madisonauthor=D.+B.+Agus&title=CVS-3983%2C+a+Selective+Matriptase+Inhibitor%2C+Suppresses+the+Growth+of+Androgen+Independent+Prostate+Tumor+Xenografts"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGalkin%26aufirst%3DA.%2BV.%26aulast%3DMullen%26aufirst%3DL.%26aulast%3DFox%26aufirst%3DW.%2BD.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DDuncan%26aufirst%3DD.%26aulast%3DMoreno%26aufirst%3DO.%26aulast%3DMadison%26aufirst%3DE.%2BL.%26aulast%3DAgus%26aufirst%3DD.%2BB.%26atitle%3DCVS-3983%252C%2520a%2520Selective%2520Matriptase%2520Inhibitor%252C%2520Suppresses%2520the%2520Growth%2520of%2520Androgen%2520Independent%2520Prostate%2520Tumor%2520Xenografts%26jtitle%3DProstate%26date%3D2004%26volume%3D61%26spage%3D228%26epage%3D235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Stoop, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, C. S.</span><span> </span><span class="NLM_article-title">Engineering of a Macromolecular Scaffold to Develop Specific Protease Inhibitors</span> <span class="citation_source-journal">Nature Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1063</span><span class="NLM_x">–</span> <span class="NLM_lpage">1068</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1038%2Fnbt860" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=12923547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmslCrsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2003&pages=1063-1068&author=A.+A.+Stoopauthor=C.+S.+Craik&title=Engineering+of+a+Macromolecular+Scaffold+to+Develop+Specific+Protease+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering of a macromolecular scaffold to develop specific protease inhibitors</span></div><div class="casAuthors">Stoop, A. Allart; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1063-1068</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The specific inhibition of serine proteases, which are crucial switches in many physiol. important processes, is of value both for basic research and for therapeutic applications.  Ecotin, a potent macromol. inhibitor of serine proteases of the S1A family, presents an attractive scaffold to engineer specific protease inhibitors because of its large inhibitor-protease interface.  Using synthetic shuffling in combination with a restricted tetranomial diversity, we created ecotin libraries that are mutated at all 20 amino acid residues in the binding interface.  The efficacy of these libraries was demonstrated against the serine protease plasma kallikrein (Pkal).  Competitive phage display selection yielded a Pkal inhibitor with an apparent dissocn. equil. const. (Ki*) of 11 pM, whereas Ki* values for related proteases (such as Factor Xa (FXa), Factor XIa (FXIa), urokinase-type plasminogen activator (uPA), thrombin, and membrane-type serine protease 1 (MT-SP1)) were four to seven orders of magnitude higher.  The adaptability of the scaffold was demonstrated by the isolation of inhibitors to two addnl. serine proteases, MT-SP1/matriptase and Factor XIIa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMZvKZ8D3zgLVg90H21EOLACvtfcHk0lgkJCuCUyiTNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmslCrsr4%253D&md5=f267555e4282067173015d20ae38202a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnbt860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt860%26sid%3Dliteratum%253Aachs%26aulast%3DStoop%26aufirst%3DA.%2BA.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DEngineering%2520of%2520a%2520Macromolecular%2520Scaffold%2520to%2520Develop%2520Specific%2520Protease%2520Inhibitors%26jtitle%3DNature%2520Biotechnol.%26date%3D2003%26volume%3D21%26spage%3D1063%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longpré, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaulieu, M.-E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span> </span><span class="NLM_article-title">Inhibition of Human Matriptase by Eglin C Variants</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">580</span><span class="NLM_x">, </span> <span class="NLM_fpage">2227</span><span class="NLM_x">–</span> <span class="NLM_lpage">2232</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2006&pages=2227-2232&author=A.+D%C3%A9siletsauthor=J.-M.+Longpr%C3%A9author=M.-E.+Beaulieuauthor=R.+Leduc&title=Inhibition+of+Human+Matriptase+by+Eglin+C+Variants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DLongpr%25C3%25A9%26aufirst%3DJ.-M.%26aulast%3DBeaulieu%26aufirst%3DM.-E.%26aulast%3DLeduc%26aufirst%3DR.%26atitle%3DInhibition%2520of%2520Human%2520Matriptase%2520by%2520Eglin%2520C%2520Variants%26jtitle%3DFEBS%2520Lett.%26date%3D2006%26volume%3D580%26spage%3D2227%26epage%3D2232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Colombo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duchêne, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chagnon, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsault, E.</span><span> </span><span class="NLM_article-title">Design and Synthesis of Potent, Selective Inhibitors of Matriptase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">530</span><span class="NLM_x">–</span> <span class="NLM_lpage">534</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml3000534" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=530-534&author=E.+Colomboauthor=A.+D%C3%A9siletsauthor=D.+Duch%C3%AAneauthor=F.+Chagnonauthor=R.+J.+Najmanovichauthor=R.+Leducauthor=E.+Marsault&title=Design+and+Synthesis+of+Potent%2C+Selective+Inhibitors+of+Matriptase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fml3000534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3000534%26sid%3Dliteratum%253Aachs%26aulast%3DColombo%26aufirst%3DE.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DDuch%25C3%25AAne%26aufirst%3DD.%26aulast%3DChagnon%26aufirst%3DF.%26aulast%3DNajmanovich%26aufirst%3DR.%2BJ.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DMarsault%26aufirst%3DE.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Potent%252C%2520Selective%2520Inhibitors%2520of%2520Matriptase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D530%26epage%3D534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Quimbar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malik, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sommerhoff, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaas, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundhuber, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, N. L.</span><span> </span><span class="NLM_article-title">High-Affinity Cyclic Peptide Matriptase Inhibitors</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">13885</span><span class="NLM_x">–</span> <span class="NLM_lpage">13896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1074%2Fjbc.M113.460030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23548907" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFWit70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=13885-13896&author=P.+Quimbarauthor=U.+Malikauthor=C.+P.+Sommerhoffauthor=Q.+Kaasauthor=L.+Y.+Chanauthor=Y.-H.+Huangauthor=M.+Grundhuberauthor=K.+Dunseauthor=D.+J.+Craikauthor=M.+A.+Andersonauthor=N.+L.+Daly&title=High-Affinity+Cyclic+Peptide+Matriptase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity Cyclic Peptide Matriptase Inhibitors</span></div><div class="casAuthors">Quimbar, Pedro; Malik, Uru; Sommerhoff, Christian P.; Kaas, Quentin; Chan, Lai Y.; Huang, Yen-Hua; Grundhuber, Maresa; Dunse, Kerry; Craik, David J.; Anderson, Marilyn A.; Daly, Norelle L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">13885-13896</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The type II transmembrane serine protease matriptase is a key activator of multiple signaling pathways assocd. with cell proliferation and modification of the extracellular matrix.  Deregulated matriptase activity correlates with a no. of diseases, including cancer and hence highly selective matriptase inhibitors may have therapeutic potential.  The plant-derived cyclic peptide, sunflower trypsin inhibitor-1 (SFTI-1), is a promising drug scaffold with potent matriptase inhibitory activity.  In the current study we have analyzed the structure-activity relationships of SFTI-1 and Momordica cochinchinensis trypsin inhibitor-II (MCoTI-II), a structurally divergent trypsin inhibitor from Momordica cochinchinensis that also contains a cyclic backbone.  We show that MCoTI-II is a significantly more potent matriptase inhibitor than SFTI-1 and that all alanine mutants of both peptides, generated using positional scanning mutagenesis, have decreased trypsin affinity, whereas several mutations either maintain or result in enhanced matriptase inhibitory activity.  These intriguing results were used to design one of the most potent matriptase inhibitors known to date with a 290 pm equil. dissocn. const., and provide the first indication on how to modulate affinity for matriptase over trypsin in cyclic peptides.  This information might be useful for the design of more selective and therapeutically relevant inhibitors of matriptase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySN0T6sDSFrVg90H21EOLACvtfcHk0lgkJCuCUyiTNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFWit70%253D&md5=4e537a35f93128a6120052e865d2c4a2</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.460030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.460030%26sid%3Dliteratum%253Aachs%26aulast%3DQuimbar%26aufirst%3DP.%26aulast%3DMalik%26aufirst%3DU.%26aulast%3DSommerhoff%26aufirst%3DC.%2BP.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DChan%26aufirst%3DL.%2BY.%26aulast%3DHuang%26aufirst%3DY.-H.%26aulast%3DGrundhuber%26aufirst%3DM.%26aulast%3DDunse%26aufirst%3DK.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DAnderson%26aufirst%3DM.%2BA.%26aulast%3DDaly%26aufirst%3DN.%2BL.%26atitle%3DHigh-Affinity%2520Cyclic%2520Peptide%2520Matriptase%2520Inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D13885%26epage%3D13896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Fittler, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avrutina, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glotzbach, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Empting, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolmar, H.</span><span> </span><span class="NLM_article-title">Combinatorial Tuning of Peptidic Drug Candidates: High-Affinity Matriptase Inhibitors Through Incremental Structure-Guided Optimization</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1848</span><span class="NLM_x">–</span> <span class="NLM_lpage">1857</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1039%2Fc3ob27469a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23361403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVChtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=1848-1857&author=H.+Fittlerauthor=O.+Avrutinaauthor=B.+Glotzbachauthor=M.+Emptingauthor=H.+Kolmar&title=Combinatorial+Tuning+of+Peptidic+Drug+Candidates%3A+High-Affinity+Matriptase+Inhibitors+Through+Incremental+Structure-Guided+Optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial tuning of peptidic drug candidates: high-affinity matriptase inhibitors through incremental structure-guided optimization</span></div><div class="casAuthors">Fittler, Heiko; Avrutina, Olga; Glotzbach, Bernhard; Empting, Martin; Kolmar, Harald</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1848-1857</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Herein we report a convenient strategy for the development of novel, highly-potent peptidic inhibitors of the trypsin-like serine protease matriptase based on the monocyclic variant of the sunflower trypsin inhibitor-1 (SFTI-1[1,14]).  We screened SFTI-1[1,14] variants possessing incremental modifications of the parent peptide for beneficial binding properties.  This compd. library comprising 6 peptides and 16 triazole-contg. peptidomimetics was established via structure-guided rational design and synthesized using a divergent strategy employing "copper-click" chem.  The most favorable amino acid substitutions were combined in one framework yielding potent SFTI-1-derived matriptase inhibitor-1 (SDMI-1) [H-Gly-Arg-c(Cys-Thr-Lys-Ser-Ile-Pro-Pro-Arg-Cys)-His-Pro-Asp-OH] and the truncated dodecapeptide variant (SDMI-2) [H-Gly-Arg-c(Cys-Thr-Lys-Ser-Ile-Pro-Pro-Arg-Cys)-His-NH2] with single-digit nanomolar inhibition consts.  In silico studies indicated that the improved matriptase affinity compared to the parent peptide is caused by the successful establishment of addnl. favorable proton donor-acceptor interactions between basic inhibitor side chains and acidic residues on the surface of the target enzyme.  SDMI-1 and 2 are potent inhibitors of the pharmaceutically relevant protease matriptase at a near physiol. pH and, thus, may find applications in therapy or diagnostics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHBE3Tv17SrVg90H21EOLACvtfcHk0ljGDx17m1uLjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVChtbo%253D&md5=fa964b6e8c7889abd9e9385cd95afeff</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2Fc3ob27469a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob27469a%26sid%3Dliteratum%253Aachs%26aulast%3DFittler%26aufirst%3DH.%26aulast%3DAvrutina%26aufirst%3DO.%26aulast%3DGlotzbach%26aufirst%3DB.%26aulast%3DEmpting%26aufirst%3DM.%26aulast%3DKolmar%26aufirst%3DH.%26atitle%3DCombinatorial%2520Tuning%2520of%2520Peptidic%2520Drug%2520Candidates%253A%2520High-Affinity%2520Matriptase%2520Inhibitors%2520Through%2520Incremental%2520Structure-Guided%2520Optimization%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D1848%26epage%3D1857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Goswami, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghadiyaram, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlfahrt, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistla, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagaraj, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satyam, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subbarao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palakurthy, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gopinath, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnamurthy, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikonen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moilanen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanya, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallio, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramachandra, M.</span><span> </span><span class="NLM_article-title">Structure-Guided Discovery of 1,3,5 Tri-Substituted Benzenes as Potent and Selective Matriptase Inhibitors Exhibiting in Vivo Antitumor Efficacy</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3187</span><span class="NLM_x">–</span> <span class="NLM_lpage">3203</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1016%2Fj.bmc.2014.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=24794746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1Gnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=3187-3203&author=R.+Goswamiauthor=S.+Mukherjeeauthor=C.+Ghadiyaramauthor=G.+Wohlfahrtauthor=R.+K.+Sistlaauthor=J.+Nagarajauthor=L.+K.+Satyamauthor=K.+Subbaraoauthor=R.+K.+Palakurthyauthor=S.+Gopinathauthor=N.+R.+Krishnamurthyauthor=T.+Ikonenauthor=A.+Moilanenauthor=H.+S.+Subramanyaauthor=P.+Kallioauthor=M.+Ramachandra&title=Structure-Guided+Discovery+of+1%2C3%2C5+Tri-Substituted+Benzenes+as+Potent+and+Selective+Matriptase+Inhibitors+Exhibiting+in+Vivo+Antitumor+Efficacy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided discovery of 1,3,5 tri-substituted benzenes as potent and selective matriptase inhibitors exhibiting in vivo antitumor efficacy</span></div><div class="casAuthors">Goswami, Rajeev; Mukherjee, Subhendu; Ghadiyaram, Chakshusmathi; Wohlfahrt, Gerd; Sistla, Ramesh K.; Nagaraj, Jwala; Satyam, Leena K.; Subbarao, Krishnaprasad; Palakurthy, Rajendra K.; Gopinath, Sreevalsam; Krishnamurthy, Narasimha R.; Ikonen, Tarja; Moilanen, Anu; Subramanya, Hosahalli S.; Kallio, Pekka; Ramachandra, Murali</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3187-3203</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Matriptase is a serine protease implicated in cancer invasion and metastasis.  Expression of matriptase is frequently dysregulated in human cancers and matriptase has been reported to activate latent growth factors such as hepatocyte growth factor/scatter factor, and proteases such as urokinase plasminogen activator suggesting that matriptase inhibitors could have therapeutic potential in treatment of cancer.  Here we report a structure-based approach which led to the discovery of selective and potent matriptase inhibitors with benzene as central core having 1,3,5 tri-substitution pattern.  X-ray crystallog. of one of the potent analogs in complex with matriptase revealed strong hydrogen bonding and salt-bridge interactions in the S1 pocket, as well as strong CH-π contacts between the P2/P4 cyclohexyl and Trp215 side-chain.  An addnl. interaction of the pendant amine at cyclohexyl with Gln175 side-chain results in substantial improvement in matriptase inhibition and selectivity against other related serine proteases.  Compds. I and II showed tumor growth inhibition in a s.c. DU-145 prostate cancer mouse model.  These compds. could be useful as tools to further explore the biol. of matriptase as a drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAfx9aewmxYLVg90H21EOLACvtfcHk0ljGDx17m1uLjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1Gnsbo%253D&md5=4c57601d086acd229ec05fb49b6775b1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DGoswami%26aufirst%3DR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DGhadiyaram%26aufirst%3DC.%26aulast%3DWohlfahrt%26aufirst%3DG.%26aulast%3DSistla%26aufirst%3DR.%2BK.%26aulast%3DNagaraj%26aufirst%3DJ.%26aulast%3DSatyam%26aufirst%3DL.%2BK.%26aulast%3DSubbarao%26aufirst%3DK.%26aulast%3DPalakurthy%26aufirst%3DR.%2BK.%26aulast%3DGopinath%26aufirst%3DS.%26aulast%3DKrishnamurthy%26aufirst%3DN.%2BR.%26aulast%3DIkonen%26aufirst%3DT.%26aulast%3DMoilanen%26aufirst%3DA.%26aulast%3DSubramanya%26aufirst%3DH.%2BS.%26aulast%3DKallio%26aufirst%3DP.%26aulast%3DRamachandra%26aufirst%3DM.%26atitle%3DStructure-Guided%2520Discovery%2520of%25201%252C3%252C5%2520Tri-Substituted%2520Benzenes%2520as%2520Potent%2520and%2520Selective%2520Matriptase%2520Inhibitors%2520Exhibiting%2520in%2520Vivo%2520Antitumor%2520Efficacy%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D3187%26epage%3D3203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Sisay, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinmetzer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stirnberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gütschow, M.</span><span> </span><span class="NLM_article-title">Identification of the First Low-Molecular-Weight Inhibitors of Matriptase-2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5523</span><span class="NLM_x">–</span> <span class="NLM_lpage">5535</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100183e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCis7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5523-5535&author=M.+T.+Sisayauthor=T.+Steinmetzerauthor=M.+Stirnbergauthor=E.+Maurerauthor=M.+Hammamiauthor=J.+Bajorathauthor=M.+G%C3%BCtschow&title=Identification+of+the+First+Low-Molecular-Weight+Inhibitors+of+Matriptase-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of the First Low-Molecular-Weight Inhibitors of Matriptase-2</span></div><div class="casAuthors">Sisay, Mihiret Tekeste; Steinmetzer, Torsten; Stirnberg, Marit; Maurer, Eva; Hammami, Maya; Bajorath, Juergen; Guetschow, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5523-5535</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As recently discovered, matriptase-2, a type II transmembrane serine protease, plays a crucial role in body iron homeostasis by down-regulating hepcidin expression, which results in increased iron levels.  Thus, matriptase-2 represents a novel target for the development of enzyme inhibitors potentially useful for the treatment of systemic iron overload (hemochromatosis).  A comparative three-dimensional model of the catalytic domain of matriptase-2 was generated and utilized for structure-based virtual screening in combination with similarity searching and knowledge-based compd. design.  Two N-protected dipeptide amides contg. a 4-amidinobenzylamide as P1 residue (compds. 1 and 3) were identified as the first small mol. inhibitors of matriptase-2 with Ki values of 170 and 460 nM, resp.  An inhibitor of the closely related protease matriptase (compd. 2, Ki = 220 nM), with more than 50-fold selectivity over matriptase-2, was also identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZuPBqKmx3SbVg90H21EOLACvtfcHk0ljGDx17m1uLjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCis7w%253D&md5=f4599613f7ca1b4356a420ecf0a15a3b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm100183e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100183e%26sid%3Dliteratum%253Aachs%26aulast%3DSisay%26aufirst%3DM.%2BT.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DStirnberg%26aufirst%3DM.%26aulast%3DMaurer%26aufirst%3DE.%26aulast%3DHammami%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520the%2520First%2520Low-Molecular-Weight%2520Inhibitors%2520of%2520Matriptase-2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5523%26epage%3D5535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Eswar, N.; Webb, B.; Marti-Renom, M. A.; Madhusudhan, M. S.; Eramian, D.; Shen, M.-Y.; Pieper, U.; Sali, A.</span><span> </span><span class="NLM_article-title">Comparative Protein Structure Modeling Using Modeller</span>.  <span class="citation_source-book">Current Protocols in Bioinformatics</span>; <span class="NLM_contrib-group">Baxevanis, A. D.</span>, Ed.: <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">, </span>Chapter 5, Unit 5.6–5.6.30.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1002%2F0471250953.bi0506s15" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=N.+Eswar&author=B.+Webb&author=M.+A.+Marti-Renom&author=M.+S.+Madhusudhan&author=D.+Eramian&author=M.-Y.+Shen&author=U.+Pieper&author=A.+Saliauthor=A.+D.+Baxevanis&title=Current+Protocols+in+Bioinformatics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2F0471250953.bi0506s15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471250953.bi0506s15%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26atitle%3DComparative%2520Protein%2520Structure%2520Modeling%2520Using%2520Modeller%26btitle%3DCurrent%2520Protocols%2520in%2520Bioinformatics%26aulast%3DBaxevanis%26aufirst%3DA.%2BD.%26pub%3DWiley%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Pronk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Páll, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjelkmar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apostolov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasson, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span> </span><span class="NLM_article-title">GROMACS 4.5: a High-Throughput and Highly Parallel Open Source Molecular Simulation Toolkit</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">845</span><span class="NLM_x">–</span> <span class="NLM_lpage">854</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1093%2Fbioinformatics%2Fbtt055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23407358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=845-854&author=S.+Pronkauthor=S.+P%C3%A1llauthor=R.+Schulzauthor=P.+Larssonauthor=P.+Bjelkmarauthor=R.+Apostolovauthor=M.+R.+Shirtsauthor=J.+C.+Smithauthor=P.+M.+Kassonauthor=D.+van+der+Spoelauthor=B.+Hessauthor=E.+Lindahl&title=GROMACS+4.5%3A+a+High-Throughput+and+Highly+Parallel+Open+Source+Molecular+Simulation+Toolkit"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span></div><div class="casAuthors">Pronk, Sander; Pall, Szilard; Schulz, Roland; Larsson, Per; Bjelkmar, Paer; Apostolov, Rossen; Shirts, Michael R.; Smith, Jeremy C.; Kasson, Peter M.; van der Spoel, David; Hess, Berk; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">845-854</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Mol. simulation has historically been a low-throughput technique, but faster computers and increasing amts. of genomic and structural data are changing this by enabling large-scale automated simulation of, for instance, many conformers or mutants of biomols. with or without a range of ligands.  At the same time, advances in performance and scaling now make it possible to model complex biomol. interaction and function in a manner directly testable by expt.  These applications share a need for fast and efficient software that can be deployed on massive scale in clusters, web servers, distributed computing or cloud resources.  Results: Here, we present a range of new simulation algorithms and features developed during the past 4 years, leading up to the GROMACS 4.5 software package.  The software now automatically handles wide classes of biomols., such as proteins, nucleic acids and lipids, and comes with all commonly used force fields for these mols. built-in.  GROMACS supports several implicit solvent models, as well as new free-energy algorithms, and the software now uses multithreading for efficient parallelization even on low-end systems, including windows-based workstations.  Together with hand-tuned assembly kernels and state-of-the-art parallelization, this provides extremely high performance and cost efficiency for high-throughput as well as massively parallel simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71lQgYMm017Vg90H21EOLACvtfcHk0ljxBqb8Pk3NNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D&md5=4b25fd6ab4e33725ae56b5da63f4ad68</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt055%26sid%3Dliteratum%253Aachs%26aulast%3DPronk%26aufirst%3DS.%26aulast%3DP%25C3%25A1ll%26aufirst%3DS.%26aulast%3DSchulz%26aufirst%3DR.%26aulast%3DLarsson%26aufirst%3DP.%26aulast%3DBjelkmar%26aufirst%3DP.%26aulast%3DApostolov%26aufirst%3DR.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DKasson%26aufirst%3DP.%2BM.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGROMACS%25204.5%253A%2520a%2520High-Throughput%2520and%2520Highly%2520Parallel%2520Open%2520Source%2520Molecular%2520Simulation%2520Toolkit%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D845%26epage%3D854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Gaudreault, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chartier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R. J.</span><span> </span><span class="NLM_article-title">Side-Chain Rotamer Changes Upon Ligand Binding: Common, Crucial, Correlate with Entropy and Rearrange Hydrogen Bonding</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">i423</span><span class="NLM_x">–</span> <span class="NLM_lpage">i430</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1093%2Fbioinformatics%2Fbts395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=22962462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlantLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=i423-i430&author=F.+Gaudreaultauthor=M.+Chartierauthor=R.+J.+Najmanovich&title=Side-Chain+Rotamer+Changes+Upon+Ligand+Binding%3A+Common%2C+Crucial%2C+Correlate+with+Entropy+and+Rearrange+Hydrogen+Bonding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Side-chain rotamer changes upon ligand binding: Common, crucial, correlate with entropy and rearrange hydrogen bonding</span></div><div class="casAuthors">Gaudreault, Francis; Chartier, Matthieu; Najmanovich, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">i423-i430</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Protein movements form a continuum from large domain rearrangements (including folding and restructuring) to side-chain rotamer changes and small rearrangements.  Understanding side-chain flexibility upon binding is important for understanding mol. recognition events and predicting ligand binding.  Methods: In the present work, we developed a well-curated non-redundant dataset of 188 proteins in pairs of structures in the Apo (unbound) and Holo (bound) forms to study the extent and the factors that guide side-chain rotamer changes upon binding.  Results: Our anal. shows that side-chain rotamer changes are widespread with only 10% of binding sites displaying no conformational changes.  Overall, at most five rotamer changes account for the obsd. movements in 90% of the cases.  Furthermore, rotamer changes are essential in 32% of flexible binding sites.  The different amino acids have a 11-fold difference in their probability to undergo changes.  Side-chain flexibility represents an intrinsic property of amino acids as it correlates well with configurational entropy differences.  Furthermore, on av. b-factors and solvent accessible surface areas can discriminate flexible side-chains in the Apo form.  Finally, there is a rearrangement of the hydrogen-bonding network upon binding primarily with a loss of H-bonds with water mols. and a gain of H-bonds with protein residues for flexible residues.  Interestingly, only 25% of side chains capable of forming H-bonds do so with the ligand upon binding.  In terms of drug design, this last result shows that there is a large no. of potential interactions that may be exploited to modulate the specificity and sensitivity of inhibitors.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="e89a898e898d84c6868982858986879e818b80a89b808d9a8a9a8787838dc68b89c6">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoerwRd3_qiYrVg90H21EOLACvtfcHk0ljxBqb8Pk3NNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlantLfO&md5=1cb775e36034683efd17908930e4639d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts395%26sid%3Dliteratum%253Aachs%26aulast%3DGaudreault%26aufirst%3DF.%26aulast%3DChartier%26aufirst%3DM.%26aulast%3DNajmanovich%26aufirst%3DR.%2BJ.%26atitle%3DSide-Chain%2520Rotamer%2520Changes%2520Upon%2520Ligand%2520Binding%253A%2520Common%252C%2520Crucial%252C%2520Correlate%2520with%2520Entropy%2520and%2520Rearrange%2520Hydrogen%2520Bonding%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3Di423%26epage%3Di430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Najmanovich, R. J.</span><span> </span><span class="NLM_article-title">Side Chain Flexibility Upon Ligand Binding: Docking Predictions and Statistical Analysis</span>. Ph.D. Thesis. <span class="NLM_publisher-name">Feinberg Graduate School, Weizmann Institute of Sciences</span><i>arXiv</i>:<i>13014564</i>,<span class="NLM_x"> </span><span class="NLM_year">,2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=%2C2004&author=R.+J.+Najmanovich&title=Side+Chain+Flexibility+Upon+Ligand+Binding%3A+Docking+Predictions+and+Statistical+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNajmanovich%26aufirst%3DR.%2BJ.%26atitle%3DSide%2520Chain%2520Flexibility%2520Upon%2520Ligand%2520Binding%253A%2520Docking%2520Predictions%2520and%2520Statistical%2520Analysis%26pub%3DFeinberg%2520Graduate%2520School%252C%2520Weizmann%2520Institute%2520of%2520Sciences%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Béliveau, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span> </span><span class="NLM_article-title">Probing the Substrate Specificities of Matriptase, Matriptase-2, Hepsin and DESC1 with Internally Quenched Fluorescent Peptides</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">2213</span><span class="NLM_x">–</span> <span class="NLM_lpage">2226</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2009&pages=2213-2226&author=F.+B%C3%A9liveauauthor=A.+D%C3%A9siletsauthor=R.+Leduc&title=Probing+the+Substrate+Specificities+of+Matriptase%2C+Matriptase-2%2C+Hepsin+and+DESC1+with+Internally+Quenched+Fluorescent+Peptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25A9liveau%26aufirst%3DF.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DLeduc%26aufirst%3DR.%26atitle%3DProbing%2520the%2520Substrate%2520Specificities%2520of%2520Matriptase%252C%2520Matriptase-2%252C%2520Hepsin%2520and%2520DESC1%2520with%2520Internally%2520Quenched%2520Fluorescent%2520Peptides%26jtitle%3DFEBS%2520J.%26date%3D2009%26volume%3D276%26spage%3D2213%26epage%3D2226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Wysocka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruba, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miecznikowska, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popow-Stellmaszyk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gütschow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stirnberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furtmann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolka, K.</span><span> </span><span class="NLM_article-title">Substrate Specificity of Human Matriptase-2</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">127</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2014&pages=121-127&author=M.+Wysockaauthor=N.+Grubaauthor=A.+Miecznikowskaauthor=J.+Popow-Stellmaszykauthor=M.+G%C3%BCtschowauthor=M.+Stirnbergauthor=N.+Furtmannauthor=J.+Bajorathauthor=A.+Lesnerauthor=K.+Rolka&title=Substrate+Specificity+of+Human+Matriptase-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWysocka%26aufirst%3DM.%26aulast%3DGruba%26aufirst%3DN.%26aulast%3DMiecznikowska%26aufirst%3DA.%26aulast%3DPopow-Stellmaszyk%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DStirnberg%26aufirst%3DM.%26aulast%3DFurtmann%26aufirst%3DN.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DLesner%26aufirst%3DA.%26aulast%3DRolka%26aufirst%3DK.%26atitle%3DSubstrate%2520Specificity%2520of%2520Human%2520Matriptase-2%26jtitle%3DBiochimie%26date%3D2014%26volume%3D97%26spage%3D121%26epage%3D127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Gaudreault, F.; Najmanovich, R. J.</span><span> </span><span class="NLM_article-title">FlexAID: Revisiting Docking on Non Native-Complex Structures</span>. Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Gaudreault%2C+F.%3B+Najmanovich%2C+R.+J.+FlexAID%3A+Revisiting+Docking+on+Non+Native-Complex+Structures.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DGaudreault%26aufirst%3DF.%26atitle%3DFlexAID%253A%2520Revisiting%2520Docking%2520on%2520Non%2520Native-Complex%2520Structures" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Hou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">Assessing the Performance of the MM/PBSA and MM/GBSA Methods. 1. The Accuracy of Binding Free Energy Calculations Based on Molecular Dynamics Simulations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100275a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=69-82&author=T.+Houauthor=J.+Wangauthor=Y.+Liauthor=W.+Wang&title=Assessing+the+Performance+of+the+MM%2FPBSA+and+MM%2FGBSA+Methods.+1.+The+Accuracy+of+Binding+Free+Energy+Calculations+Based+on+Molecular+Dynamics+Simulations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the Performance of the MM/PBSA and MM/GBSA Methods. 1. The Accuracy of Binding Free Energy Calculations Based on Molecular Dynamics Simulations</span></div><div class="casAuthors">Hou, Tingjun; Wang, Junmei; Li, Youyong; Wang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-82</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Mol. Mechanics/Poisson-Boltzmann Surface Area (MM/PBSA) and the Mol. Mechanics/Generalized Born Surface Area (MM/GBSA) methods calc. binding free energies for macromols. by combining mol. mechanics calcns. and continuum solvation models.  To systematically evaluate the performance of these methods, the authors report here an extensive study of 59 ligands interacting with six different proteins.  First, the authors explored the effects of the length of the mol. dynamics (MD) simulation, ranging from 400 to 4800 ps, and the solute dielec. const. (1, 2, or 4) on the binding free energies predicted by MM/PBSA.  The following three important conclusions could be obsd.: (1) MD simulation length has an obvious impact on the predictions, and longer MD simulation is not always necessary to achieve better predictions.  (2) The predictions are quite sensitive to the solute dielec. const., and this parameter should be carefully detd. according to the characteristics of the protein/ligand binding interface.  (3) Conformational entropy often show large fluctuations in MD trajectories, and a large no. of snapshots are necessary to achieve stable predictions.  Next, the authors evaluated the accuracy of the binding free energies calcd. by three Generalized Born (GB) models.  The authors found that the GB model developed by Onufriev and Case was the most successful model in ranking the binding affinities of the studied inhibitors.  Finally, the authors evaluated the performance of MM/GBSA and MM/PBSA in predicting binding free energies.  The authors' results showed that MM/PBSA performed better in calcg. abs., but not necessarily relative, binding free energies than MM/GBSA.  Considering its computational efficiency, MM/GBSA can serve as a powerful tool in drug design, where correct ranking of inhibitors is often emphasized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_kxNWs2s3grVg90H21EOLACvtfcHk0ljcTg7YgznAcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsrrK&md5=b9f5b7fab70a6cc76b115d202d718f87</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci100275a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100275a%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DAssessing%2520the%2520Performance%2520of%2520the%2520MM%252FPBSA%2520and%2520MM%252FGBSA%2520Methods.%25201.%2520The%2520Accuracy%2520of%2520Binding%2520Free%2520Energy%2520Calculations%2520Based%2520on%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D69%26epage%3D82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Hou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">Assessing the Performance of the Molecular Mechanics/Poisson Boltzmann Surface Area and Molecular Mechanics/Generalized Born Surface Area Methods. II. The Accuracy of Ranking Poses Generated From Docking</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">866</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1002%2Fjcc.21666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=20949517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3MXitleisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=866-877&author=T.+Houauthor=J.+Wangauthor=Y.+Liauthor=W.+Wang&title=Assessing+the+Performance+of+the+Molecular+Mechanics%2FPoisson+Boltzmann+Surface+Area+and+Molecular+Mechanics%2FGeneralized+Born+Surface+Area+Methods.+II.+The+Accuracy+of+Ranking+Poses+Generated+From+Docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the performance of the molecular mechanics/Poisson Boltzmann surface area and molecular mechanics/generalized Born surface area methods. II: The accuracy of ranking poses generated from docking</span></div><div class="casAuthors">Hou, Ting-Jun; Wang, Jun-Mei; Li, You-Yong; Wang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">866-877</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In mol. docking, it is challenging to develop a scoring function that is accurate to conduct high-throughput screenings.  Most scoring functions implemented in popular docking software packages were developed with many approxns. for computational efficiency, which sacrifices the accuracy of prediction.  With advanced technol. and powerful computational hardware nowadays, it is feasible to use rigorous scoring functions, such as mol. mechanics/Poisson Boltzmann surface area (MM/PBSA) and mol. mechanics/generalized Born surface area (MM/GBSA) in mol. docking studies.  Here, the authors systematically investigated the performance of MM/PBSA and MM/GBSA to identify the correct binding conformations and predict the binding free energies for 98 protein-ligand complexes.  Comparison studies showed that MM/GBSA (69.4%) outperformed MM/PBSA (45.5%) and many popular scoring functions to identify the correct binding conformations.  Moreover, the authors found that mol. dynamics simulations are necessary for some systems to identify the correct binding conformations.  Based on the authors' results, the authors proposed the guideline for MM/GBSA to predict the binding conformations.  The authors then tested the performance of MM/GBSA and MM/PBSA to reproduce the binding free energies of the 98 protein-ligand complexes.  The best prediction of MM/GBSA model with internal dielec. const. 2.0, produced a Spearman's correlation coeff. of 0.66, which is better than MM/PBSA (0.49) and almost all scoring functions used in mol. docking.  In summary, MM/GBSA performs well for both binding pose predictions and binding free-energy estns. and is efficient to re-score the top-hit poses produced by other less-accurate scoring functions.  © 2010 Wiley Periodicals, Inc.  J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6V0ra4bAVq7Vg90H21EOLACvtfcHk0ljcTg7YgznAcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXitleisrw%253D&md5=fd42a4da52a20e7dfcbfff62e746513e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21666%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DAssessing%2520the%2520Performance%2520of%2520the%2520Molecular%2520Mechanics%252FPoisson%2520Boltzmann%2520Surface%2520Area%2520and%2520Molecular%2520Mechanics%252FGeneralized%2520Born%2520Surface%2520Area%2520Methods.%2520II.%2520The%2520Accuracy%2520of%2520Ranking%2520Poses%2520Generated%2520From%2520Docking%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D866%26epage%3D877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Maurer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sisay, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stirnberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinmetzer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gütschow, M.</span><span> </span><span class="NLM_article-title">Insights Into Matriptase-2 Substrate Binding and Inhibition Mechanisms by Analyzing Active-Site-Mutated Variants</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">68</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=68-72&author=E.+Maurerauthor=M.+T.+Sisayauthor=M.+Stirnbergauthor=T.+Steinmetzerauthor=J.+Bajorathauthor=M.+G%C3%BCtschow&title=Insights+Into+Matriptase-2+Substrate+Binding+and+Inhibition+Mechanisms+by+Analyzing+Active-Site-Mutated+Variants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DE.%26aulast%3DSisay%26aufirst%3DM.%2BT.%26aulast%3DStirnberg%26aufirst%3DM.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26atitle%3DInsights%2520Into%2520Matriptase-2%2520Substrate%2520Binding%2520and%2520Inhibition%2520Mechanisms%2520by%2520Analyzing%2520Active-Site-Mutated%2520Variants%26jtitle%3DChemMedChem%26date%3D2012%26volume%3D7%26spage%3D68%26epage%3D72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Shannon, C. E.</span><span> </span><span class="NLM_article-title">A Mathematical Theory of Communication</span> <span class="citation_source-journal">Bell System Tech. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1948</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">423</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1002%2Fj.1538-7305.1948.tb01338.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1948&pages=379-423&author=C.+E.+Shannon&title=A+Mathematical+Theory+of+Communication"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fj.1538-7305.1948.tb01338.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1538-7305.1948.tb01338.x%26sid%3Dliteratum%253Aachs%26aulast%3DShannon%26aufirst%3DC.%2BE.%26atitle%3DA%2520Mathematical%2520Theory%2520of%2520Communication%26jtitle%3DBell%2520System%2520Tech.%2520J.%26date%3D1948%26volume%3D27%26spage%3D379%26epage%3D423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Fuchs, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Grafenstein, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margreiter, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzer, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallnoefer, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liedl, K. R.</span><span> </span><span class="NLM_article-title">Cleavage Entropy as Quantitative Measure of Protease Specificity</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003007</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003007&author=J.+E.+Fuchsauthor=S.+von+Grafensteinauthor=R.+G.+Huberauthor=M.+A.+Margreiterauthor=G.+M.+Spitzerauthor=H.+G.+Wallnoeferauthor=K.+R.+Liedl&title=Cleavage+Entropy+as+Quantitative+Measure+of+Protease+Specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3Dvon%2BGrafenstein%26aufirst%3DS.%26aulast%3DHuber%26aufirst%3DR.%2BG.%26aulast%3DMargreiter%26aufirst%3DM.%2BA.%26aulast%3DSpitzer%26aufirst%3DG.%2BM.%26aulast%3DWallnoefer%26aufirst%3DH.%2BG.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DCleavage%2520Entropy%2520as%2520Quantitative%2520Measure%2520of%2520Protease%2520Specificity%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3De1003007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Barré, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dufour, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckhard, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kappelhoff, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Béliveau, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overall, C. M.</span><span> </span><span class="NLM_article-title">Cleavage Specificity Analysis of Six Type II Transmembrane Serine Proteases (TTSPs) Using PICS with Proteome-Derived Peptide Libraries</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e105984</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e105984&author=O.+Barr%C3%A9author=A.+Dufourauthor=U.+Eckhardauthor=R.+Kappelhoffauthor=F.+B%C3%A9liveauauthor=R.+Leducauthor=C.+M.+Overall&title=Cleavage+Specificity+Analysis+of+Six+Type+II+Transmembrane+Serine+Proteases+%28TTSPs%29+Using+PICS+with+Proteome-Derived+Peptide+Libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBarr%25C3%25A9%26aufirst%3DO.%26aulast%3DDufour%26aufirst%3DA.%26aulast%3DEckhard%26aufirst%3DU.%26aulast%3DKappelhoff%26aufirst%3DR.%26aulast%3DB%25C3%25A9liveau%26aufirst%3DF.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26atitle%3DCleavage%2520Specificity%2520Analysis%2520of%2520Six%2520Type%2520II%2520Transmembrane%2520Serine%2520Proteases%2520%2528TTSPs%2529%2520Using%2520PICS%2520with%2520Proteome-Derived%2520Peptide%2520Libraries%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De105984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Huang, S.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grinter, S. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, X.</span><span> </span><span class="NLM_article-title">Scoring Functions and Their Evaluation Methods for Protein–Ligand Docking: Recent Advances and Future Directions</span> <span class="citation_source-journal">Phys. Chem. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">12899</span><span class="NLM_x">–</span> <span class="NLM_lpage">12908</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=12899-12908&author=S.-Y.+Huangauthor=S.+Z.+Grinterauthor=X.+Zou&title=Scoring+Functions+and+Their+Evaluation+Methods+for+Protein%E2%80%93Ligand+Docking%3A+Recent+Advances+and+Future+Directions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.-Y.%26aulast%3DGrinter%26aufirst%3DS.%2BZ.%26aulast%3DZou%26aufirst%3DX.%26atitle%3DScoring%2520Functions%2520and%2520Their%2520Evaluation%2520Methods%2520for%2520Protein%25E2%2580%2593Ligand%2520Docking%253A%2520Recent%2520Advances%2520and%2520Future%2520Directions%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D12899%26epage%3D12908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">O’Boyle, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeersch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, G. R.</span><span> </span><span class="NLM_article-title">Open Babel: an Open Chemical Toolbox</span> <span class="citation_source-journal">J. Cheminform.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=21982300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A280%3ADC%252BC3Mbjt1elsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+an+Open+Chemical+Toolbox"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Open Babel: An open chemical toolbox</span></div><div class="casAuthors">O'Boyle Noel M; Banck Michael; James Craig A; Morley Chris; Vandermeersch Tim; Hutchison Geoffrey R</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cheminformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  A frequent problem in computational modeling is the interconversion of chemical structures between different formats.  While standard interchange formats exist (for example, Chemical Markup Language) and de facto standards have arisen (for example, SMILES format), the need to interconvert formats is a continuing problem due to the multitude of different application areas for chemistry data, differences in the data stored by different formats (0D versus 3D, for example), and competition between software along with a lack of vendor-neutral formats.  RESULTS:  We discuss, for the first time, Open Babel, an open-source chemical toolbox that speaks the many languages of chemical data.  Open Babel version 2.3 interconverts over 110 formats.  The need to represent such a wide variety of chemical and molecular data requires a library that implements a wide range of cheminformatics algorithms, from partial charge assignment and aromaticity detection, to bond order perception and canonicalization.  We detail the implementation of Open Babel, describe key advances in the 2.3 release, and outline a variety of uses both in terms of software products and scientific research, including applications far beyond simple format interconversion.  CONCLUSIONS:  Open Babel presents a solution to the proliferation of multiple chemical file formats.  In addition, it provides a variety of useful utilities from conformer searching and 2D depiction, to filtering, batch conversion, and substructure and similarity searching.  For developers, it can be used as a programming library to handle chemical data in areas such as organic chemistry, drug design, materials science, and computational chemistry.  It is freely available under an open-source license from http://openbabel.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAiR-F6gwEoGLFM_lbR1VmfW6udTcc2eaMzzzOm9BOD7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mbjt1elsQ%253D%253D&md5=b3c770a7e1bacab8afe04f91da9c2c1e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520an%2520Open%2520Chemical%2520Toolbox%26jtitle%3DJ.%2520Cheminform.%26date%3D2011%26volume%3D3%26spage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucl. Acids. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lg5L_r7VXmwTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucl.%2520Acids.%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Shen, M.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Statistical Potential for Assessment and Prediction of Protein Structures</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">2507</span><span class="NLM_x">–</span> <span class="NLM_lpage">2524</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1110%2Fps.062416606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=17075131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFygsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2006&pages=2507-2524&author=M.-Y.+Shenauthor=A.+Sali&title=Statistical+Potential+for+Assessment+and+Prediction+of+Protein+Structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical potential for assessment and prediction of protein structures</span></div><div class="casAuthors">Shen, Min-Yi; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2507-2524</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Protein structures in the Protein Data Bank provide a wealth of data about the interactions that det. the native states of proteins.  Using the probability theory, the authors derive an at. distance-dependent statistical potential from a sample of native structures that does not depend on any adjustable parameters (Discrete Optimized Protein Energy, or DOPE).  DOPE is based on an improved ref. state that corresponds to noninteracting atoms in a homogeneous sphere with the radius dependent on a sample native structure; it thus accounts for the finite and spherical shape of the native structures.  The DOPE potential was extd. from a nonredundant set of 1472 crystallog. structures.  The authors tested DOPE and five other scoring functions by the detection of the native state among six multiple target decoy sets, the correlation between the score and model error, and the identification of the most accurate nonnative structure in the decoy set.  For all decoy sets, DOPE is the best performing function in terms of all criteria, except for a tie in one criterion for one decoy set.  To facilitate its use in various applications, such as model assessment, loop modeling, and fitting into cryo-electron microscopy mass d. maps combined with comparative protein structure modeling, DOPE was incorporated into the modeling package MODELLER-8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz7bz4vq-p_7Vg90H21EOLACvtfcHk0lg5L_r7VXmwTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFygsrfO&md5=f4e7ef56cd100164e347103a6888ee8e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1110%2Fps.062416606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.062416606%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DM.-Y.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStatistical%2520Potential%2520for%2520Assessment%2520and%2520Prediction%2520of%2520Protein%2520Structures%26jtitle%3DProtein%2520Sci.%26date%3D2006%26volume%3D15%26spage%3D2507%26epage%3D2524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">London, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cimermančič, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Covalent Docking of Large Libraries for the Discovery of Chemical Probes</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1066</span><span class="NLM_x">–</span> <span class="NLM_lpage">1072</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1038%2Fnchembio.1666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=25344815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A280%3ADC%252BC2M3jt1Wmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1066-1072&author=N.+Londonauthor=R.+M.+Millerauthor=S.+Krishnanauthor=K.+Uchidaauthor=J.+J.+Irwinauthor=O.+Eidamauthor=L.+Giboldauthor=P.+Cimerman%C4%8Di%C4%8Dauthor=R.+Bonnetauthor=B.+K.+Shoichetauthor=J.+Taunton&title=Covalent+Docking+of+Large+Libraries+for+the+Discovery+of+Chemical+Probes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking of large libraries for the discovery of chemical probes</span></div><div class="casAuthors">London Nir; Eidam Oliv; Miller Rand M; Krishnan Shyam; Uchida Kenji; Taunton Jack; Irwin John J; Shoichet Brian K; Gibold Lucie; Cimermancic Peter; Bonnet Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature chemical biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1066-72</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chemical probes that form a covalent bond with a protein target often show enhanced selectivity, potency and utility for biological studies.  Despite these advantages, protein-reactive compounds are usually avoided in high-throughput screening campaigns.  Here we describe a general method (DOCKovalent) for screening large virtual libraries of electrophilic small molecules.  We apply this method prospectively to discover reversible covalent fragments that target distinct protein nucleophiles, including the catalytic serine of AmpC β-lactamase and noncatalytic cysteines in RSK2, MSK1 and JAK3 kinases.  We identify submicromolar to low-nanomolar hits with high ligand efficiency, cellular activity and selectivity, including what are to our knowledge the first reported reversible covalent inhibitors of JAK3.  Crystal structures of inhibitor complexes with AmpC and RSK2 confirm the docking predictions and guide further optimization.  As covalent virtual screening may have broad utility for the rapid discovery of chemical probes, we have made the method freely available through an automated web server (http://covalent.docking.org/).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREZM0Uo7YPEWK_SFaaT9INfW6udTcc2ea4YKMR5HEeW7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3jt1Wmuw%253D%253D&md5=721636ba69de480e84a31c2b0679d382</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1666%26sid%3Dliteratum%253Aachs%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DGibold%26aufirst%3DL.%26aulast%3DCimerman%25C4%258Di%25C4%258D%26aufirst%3DP.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DCovalent%2520Docking%2520of%2520Large%2520Libraries%2520for%2520the%2520Discovery%2520of%2520Chemical%2520Probes%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1066%26epage%3D1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Anderson, D. P.</span><span> </span><span class="NLM_article-title">BOINC: a System for Public-Resource Computing and Storage. </span>. In <span class="NLM_conf-name">Fifth IEEE/ACM International Workshop on Grid Computing</span>, <span class="NLM_conf-loc">Pittsburgh, PA</span>, <span class="NLM_conf-date">November 8, 2004</span>; <span class="NLM_publisher-name">IEEE</span>: <span class="NLM_publisher-loc">Los Angeles</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=4-10&conference=Fifth+IEEE%2FACM+International+Workshop+on+Grid+Computing&author=D.+P.+Anderson&title=BOINC%3A+a+System+for+Public-Resource+Computing+and+Storage.+"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BP.%26atitle%3DBOINC%253A%2520a%2520System%2520for%2520Public-Resource%2520Computing%2520and%2520Storage.%2520%26pub%3DIEEE%26date%3D2004%26spage%3D4%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Van Der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">GROMACS: Fast, Flexible, and Free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+C.+Berendsen&title=GROMACS%3A+Fast%2C+Flexible%2C+and+Free"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0ljHH-hO7_bpTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DGROMACS%253A%2520Fast%252C%2520Flexible%252C%2520and%2520Free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Sousa da Silva, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vranken, W. F.</span><span> </span><span class="NLM_article-title">ACPYPE—AnteChamber PYthon Parser interfacE</span> <span class="citation_source-journal">BMC Res. Notes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">367</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1186%2F1756-0500-5-367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A280%3ADC%252BC38fitlWjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=367&author=A.+W.+Sousa+da+Silvaauthor=W.+F.+Vranken&title=ACPYPE%E2%80%94AnteChamber+PYthon+Parser+interfacE"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">ACPYPE - AnteChamber PYthon Parser interfacE</span></div><div class="casAuthors">Sousa da Silva Alan W; Vranken Wim F</div><div class="citationInfo"><span class="NLM_cas:title">BMC research notes</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">367</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  ACPYPE (or AnteChamber PYthon Parser interfacE) is a wrapper script around the ANTECHAMBER software that simplifies the generation of small molecule topologies and parameters for a variety of molecular dynamics programmes like GROMACS, CHARMM and CNS.  It is written in the Python programming language and was developed as a tool for interfacing with other Python based applications such as the CCPN software suite (for NMR data analysis) and ARIA (for structure calculations from NMR data).  ACPYPE is open source code, under GNU GPL v3, and is available as a stand-alone application at http://www.ccpn.ac.uk/acpype and as a web portal application at http://webapps.ccpn.ac.uk/acpype.  FINDINGS:  We verified the topologies generated by ACPYPE in three ways: by comparing with default AMBER topologies for standard amino acids; by generating and verifying topologies for a large set of ligands from the PDB; and by recalculating the structures for 5 protein-ligand complexes from the PDB.  CONCLUSIONS:  ACPYPE is a tool that simplifies the automatic generation of topology and parameters in different formats for different molecular mechanics programmes, including calculation of partial charges, while being object oriented for integration with other applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbj1v3HcWoNJZoeNcnHOQ9fW6udTcc2ea4YKMR5HEeW7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fitlWjtg%253D%253D&md5=60f750ba98c975374b351a39a1fa0ec4</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2F1756-0500-5-367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-0500-5-367%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%2Bda%2BSilva%26aufirst%3DA.%2BW.%26aulast%3DVranken%26aufirst%3DW.%2BF.%26atitle%3DACPYPE%25E2%2580%2594AnteChamber%2520PYthon%2520Parser%2520interfacE%26jtitle%3DBMC%2520Res.%2520Notes%26date%3D2012%26volume%3D5%26spage%3D367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsili, M.</span><span> </span><span class="NLM_article-title">Iterative Partial Equalization of Orbital Electronegativity—A Rapid Access to Atomic Charges</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">3219</span><span class="NLM_x">–</span> <span class="NLM_lpage">3228</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1980&pages=3219-3228&author=J.+Gasteigerauthor=M.+Marsili&title=Iterative+Partial+Equalization+of+Orbital+Electronegativity%E2%80%94A+Rapid+Access+to+Atomic+Charges"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DMarsili%26aufirst%3DM.%26atitle%3DIterative%2520Partial%2520Equalization%2520of%2520Orbital%2520Electronegativity%25E2%2580%2594A%2520Rapid%2520Access%2520to%2520Atomic%2520Charges%26jtitle%3DTetrahedron%26date%3D1980%26volume%3D36%26spage%3D3219%26epage%3D3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Costanzo, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almond, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hecker, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schott, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabut, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrade-Gordon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corcoran, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giardino, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauffman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Garavilla, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haertlein, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maryanoff, B. E.</span><span> </span><span class="NLM_article-title">In-Depth Study of Tripeptide-Based Alpha-Ketoheterocycles as Inhibitors of Thrombin. Effective Utilization of the S1′ Subsite and Its Implications to Structure-Based Drug Design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1984</span><span class="NLM_x">–</span> <span class="NLM_lpage">2008</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0303857" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1984-2008&author=M.+J.+Costanzoauthor=H.+R.+Almondauthor=L.+R.+Heckerauthor=M.+R.+Schottauthor=S.+C.+Yabutauthor=H.-C.+Zhangauthor=P.+Andrade-Gordonauthor=T.+W.+Corcoranauthor=E.+C.+Giardinoauthor=J.+A.+Kauffmanauthor=J.+M.+Lewisauthor=L.+de+Garavillaauthor=B.+J.+Haertleinauthor=B.+E.+Maryanoff&title=In-Depth+Study+of+Tripeptide-Based+Alpha-Ketoheterocycles+as+Inhibitors+of+Thrombin.+Effective+Utilization+of+the+S1%E2%80%B2+Subsite+and+Its+Implications+to+Structure-Based+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm0303857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303857%26sid%3Dliteratum%253Aachs%26aulast%3DCostanzo%26aufirst%3DM.%2BJ.%26aulast%3DAlmond%26aufirst%3DH.%2BR.%26aulast%3DHecker%26aufirst%3DL.%2BR.%26aulast%3DSchott%26aufirst%3DM.%2BR.%26aulast%3DYabut%26aufirst%3DS.%2BC.%26aulast%3DZhang%26aufirst%3DH.-C.%26aulast%3DAndrade-Gordon%26aufirst%3DP.%26aulast%3DCorcoran%26aufirst%3DT.%2BW.%26aulast%3DGiardino%26aufirst%3DE.%2BC.%26aulast%3DKauffman%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DJ.%2BM.%26aulast%3Dde%2BGaravilla%26aufirst%3DL.%26aulast%3DHaertlein%26aufirst%3DB.%2BJ.%26aulast%3DMaryanoff%26aufirst%3DB.%2BE.%26atitle%3DIn-Depth%2520Study%2520of%2520Tripeptide-Based%2520Alpha-Ketoheterocycles%2520as%2520Inhibitors%2520of%2520Thrombin.%2520Effective%2520Utilization%2520of%2520the%2520S1%25E2%2580%25B2%2520Subsite%2520and%2520Its%2520Implications%2520to%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1984%26epage%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Williams, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, J. F.</span><span> </span><span class="NLM_article-title">The Kinetics of Reversible Tight-Binding Inhibition</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1979</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">437</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1016%2F0076-6879%2879%2963019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=502865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1979&pages=437-467&author=J.+W.+Williamsauthor=J.+F.+Morrison&title=The+Kinetics+of+Reversible+Tight-Binding+Inhibition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of reversible tight-binding inhibition</span></div><div class="casAuthors">Williams, Jeffrey W.; Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">Enzyme Kinet. Mech., Part A</span>),
    <span class="NLM_cas:pages">437-67</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">In addn. to a general discussion of the kinetics of reversible tight-binding inhibitors, which exert their effect at a concn. comparable to that of the enzyme, and for which steady-state theory is used, exptl. results are given of the tight-binding inhibition of creatine kinase by Eu3+ (actually Eu-ADP) under steady-state conditions.  The inhibition of creatine kinase by fixed concns. of Eu3+ in the presence of varying concns. of Mg-ADP indicated that the inhibition is competitive, suggesting that Eu-ADP acts as a structural analog of Mg-ADP.  The inhibition was noncompetitive with respect to creatine phosphate.  The dissocn. consts. for binding of Eu-ADP to the free enzyme and the enzyme-phosphocreatine complex were 1.5 and 2.2 nM, resp., as detd. from Ki values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwV3voXt27Vg90H21EOLACvtfcHk0ljUxsDtWeIL1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D&md5=a770f91b51661c9502d0aca44a24e1c7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2879%2963019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252879%252963019-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BW.%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520Kinetics%2520of%2520Reversible%2520Tight-Binding%2520Inhibition%26jtitle%3DMethods%2520Enzymol.%26date%3D1979%26volume%3D63%26spage%3D437%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 27 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Philippe Sarret</span>. </span><span class="cited-content_cbyCitation_article-title">Éric Marsault (1971–2021): A Legacy through the Prism of Relationship Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5221-5224. <a href="https://doi.org/10.1021/acs.jmedchem.1c00481" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00481</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00481%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D%2525C3%252589ric%252BMarsault%252B%2525281971%2525E2%252580%2525932021%252529%25253A%252BA%252BLegacy%252Bthrough%252Bthe%252BPrism%252Bof%252BRelationship%252BChemistry%26aulast%3DSarret%26aufirst%3DPhilippe%26date%3D2021%26date%3D2021%26date%3D24032021%26volume%3D64%26issue%3D9%26spage%3D5221%26epage%3D5224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Vishnu
C. Damalanka, Zhenfu Han, Partha Karmakar, Anthony J. O’Donoghue, Florencia La Greca, Tommy Kim, Shishir M. Pant, Jonathan Helander, Juha Klefström, Charles S. Craik, <span class="NLM_string-name hlFld-ContribAuthor">James W. Janetka</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Selective Matriptase and Hepsin Serine Protease Inhibitors: Useful Chemical Tools for Cancer Cell Biology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (2)
                                     , 480-490. <a href="https://doi.org/10.1021/acs.jmedchem.8b01536" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01536</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01536&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01536%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSelective%252BMatriptase%252Band%252BHepsin%252BSerine%252BProtease%252BInhibitors%25253A%252BUseful%252BChemical%252BTools%252Bfor%252BCancer%252BCell%252BBiology%26aulast%3DDamalanka%26aufirst%3DVishnu%2BC.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D02102018%26date%3D04012019%26date%3D20122018%26volume%3D62%26issue%3D2%26spage%3D480%26epage%3D490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Birgit J.  Waldner</span>, <span class="hlFld-ContribAuthor ">Julian E.  Fuchs</span>, <span class="hlFld-ContribAuthor ">Roland G.  Huber</span>, <span class="hlFld-ContribAuthor ">Susanne  von Grafenstein</span>, <span class="hlFld-ContribAuthor ">Michael  Schauperl</span>, <span class="hlFld-ContribAuthor ">Christian  Kramer</span>, and <span class="hlFld-ContribAuthor ">Klaus R.  Liedl</span>  . </span><span class="cited-content_cbyCitation_article-title">Quantitative Correlation of Conformational Binding Enthalpy with Substrate Specificity of Serine Proteases. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2016,</strong> <em>120 </em>
                                    (2)
                                     , 299-308. <a href="https://doi.org/10.1021/acs.jpcb.5b10637" title="DOI URL">https://doi.org/10.1021/acs.jpcb.5b10637</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.5b10637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.5b10637%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DQuantitative%252BCorrelation%252Bof%252BConformational%252BBinding%252BEnthalpy%252Bwith%252BSubstrate%252BSpecificity%252Bof%252BSerine%252BProteases%26aulast%3DWaldner%26aufirst%3DBirgit%2BJ.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D30102015%26date%3D28122015%26date%3D11012016%26date%3D21012016%26date%3D28122015%26volume%3D120%26issue%3D2%26spage%3D299%26epage%3D308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaoyuan  Liu</span>, <span class="hlFld-ContribAuthor ">Zhiyang  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuan  Zhao</span>, <span class="hlFld-ContribAuthor ">Chaojie  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Atomic-Level Investigation of Reactant Recognition Mechanism and Thermodynamic Property in Glucosamine 6-Phosphate Deaminase Catalysis. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.737492" title="DOI URL">https://doi.org/10.3389/fchem.2021.737492</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.737492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.737492%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DAtomic-Level%252BInvestigation%252Bof%252BReactant%252BRecognition%252BMechanism%252Band%252BThermodynamic%252BProperty%252Bin%252BGlucosamine%252B6-Phosphate%252BDeaminase%252BCatalysis%26aulast%3DZhang%26aufirst%3DXiao%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qi  Lu</span>, <span class="hlFld-ContribAuthor ">Xiao-Tong  Sun</span>, <span class="hlFld-ContribAuthor ">Yao-Lei  Kong</span>, <span class="hlFld-ContribAuthor ">Jin-Chuan  Liu</span>, <span class="hlFld-ContribAuthor ">Bo  Chen</span>, <span class="hlFld-ContribAuthor ">Jian-Quan  Weng</span>. </span><span class="cited-content_cbyCitation_article-title">C2-arylacylation of 2H-benzothiazoles with methyl arenes via Selectfluor oxidation. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2021,</strong> <em>75 </em>, 153184. <a href="https://doi.org/10.1016/j.tetlet.2021.153184" title="DOI URL">https://doi.org/10.1016/j.tetlet.2021.153184</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2021.153184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2021.153184%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DC2-arylacylation%252Bof%252B2H-benzothiazoles%252Bwith%252Bmethyl%252Barenes%252Bvia%252BSelectfluor%252Boxidation%26aulast%3DLu%26aufirst%3DQi%26date%3D2021%26volume%3D75%26spage%3D153184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexandre  Murza</span>, <span class="hlFld-ContribAuthor ">Sébastien P.  Dion</span>, <span class="hlFld-ContribAuthor ">Pierre-Luc  Boudreault</span>, <span class="hlFld-ContribAuthor ">Antoine  Désilets</span>, <span class="hlFld-ContribAuthor ">Richard  Leduc</span>, <span class="hlFld-ContribAuthor ">Éric  Marsault</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of type II transmembrane serine proteases in the treatment of diseases of the respiratory tract – A review of patent literature. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2020,</strong> <em>30 </em>
                                    (11)
                                     , 807-824. <a href="https://doi.org/10.1080/13543776.2020.1817390" title="DOI URL">https://doi.org/10.1080/13543776.2020.1817390</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2020.1817390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2020.1817390%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DInhibitors%252Bof%252Btype%252BII%252Btransmembrane%252Bserine%252Bproteases%252Bin%252Bthe%252Btreatment%252Bof%252Bdiseases%252Bof%252Bthe%252Brespiratory%252Btract%252B%2525E2%252580%252593%252BA%252Breview%252Bof%252Bpatent%252Bliterature%26aulast%3DMurza%26aufirst%3DAlexandre%26date%3D2020%26date%3D2020%26volume%3D30%26issue%3D11%26spage%3D807%26epage%3D824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">François  Béliveau</span>, <span class="hlFld-ContribAuthor ">Aarti  Tarkar</span>, <span class="hlFld-ContribAuthor ">Sébastien P.  Dion</span>, <span class="hlFld-ContribAuthor ">Antoine  Désilets</span>, <span class="hlFld-ContribAuthor ">Mariana Gabriela  Ghinet</span>, <span class="hlFld-ContribAuthor ">Pierre-Luc  Boudreault</span>, <span class="hlFld-ContribAuthor ">Catherine  St-Georges</span>, <span class="hlFld-ContribAuthor ">Éric  Marsault</span>, <span class="hlFld-ContribAuthor ">Daniel  Paone</span>, <span class="hlFld-ContribAuthor ">Jon  Collins</span>, <span class="hlFld-ContribAuthor ">Colin H.  Macphee</span>, <span class="hlFld-ContribAuthor ">Nino  Campobasso</span>, <span class="hlFld-ContribAuthor ">Arthur  Groy</span>, <span class="hlFld-ContribAuthor ">Josh  Cottom</span>, <span class="hlFld-ContribAuthor ">Michael  Ouellette</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Pope</span>, <span class="hlFld-ContribAuthor ">Richard  Leduc</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Development of TMPRSS6 Inhibitors Modulating Hepcidin Levels in Human Hepatocytes. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (11)
                                     , 1559-1572.e9. <a href="https://doi.org/10.1016/j.chembiol.2019.09.004" title="DOI URL">https://doi.org/10.1016/j.chembiol.2019.09.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2019.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2019.09.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DDiscovery%252Band%252BDevelopment%252Bof%252BTMPRSS6%252BInhibitors%252BModulating%252BHepcidin%252BLevels%252Bin%252BHuman%252BHepatocytes%26aulast%3DB%25C3%25A9liveau%26aufirst%3DFran%25C3%25A7ois%26date%3D2019%26volume%3D26%26issue%3D11%26spage%3D1559%26epage%3D1572.e9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bing  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiwen  Dai</span>, <span class="hlFld-ContribAuthor ">Ziyang  Bao</span>, <span class="hlFld-ContribAuthor ">Qing  Mao</span>, <span class="hlFld-ContribAuthor ">Yulin  Duan</span>, <span class="hlFld-ContribAuthor ">Yuwei  Yang</span>, <span class="hlFld-ContribAuthor ">Shaojie  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the subpocket in xanthine oxidase: Design, synthesis, and biological evaluation of 2-[4-alkoxy-3-(1H-tetrazol-1-yl) phenyl]-6-oxo-1,6-dihydropyrimidine-5-carboxylic acid derivatives. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>181 </em>, 111559. <a href="https://doi.org/10.1016/j.ejmech.2019.07.062" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.07.062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.07.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.07.062%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bthe%252Bsubpocket%252Bin%252Bxanthine%252Boxidase%25253A%252BDesign%25252C%252Bsynthesis%25252C%252Band%252Bbiological%252Bevaluation%252Bof%252B2-%25255B4-alkoxy-3-%2525281H-tetrazol-1-yl%252529%252Bphenyl%25255D-6-oxo-1%25252C6-dihydropyrimidine-5-carboxylic%252Bacid%252Bderivatives%26aulast%3DZhang%26aufirst%3DBing%26date%3D2019%26volume%3D181%26spage%3D111559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke  Zuo</span>, <span class="hlFld-ContribAuthor ">Yingying  Qi</span>, <span class="hlFld-ContribAuthor ">Cai  Yuan</span>, <span class="hlFld-ContribAuthor ">Longguang  Jiang</span>, <span class="hlFld-ContribAuthor ">Peng  Xu</span>, <span class="hlFld-ContribAuthor ">Jianping  Hu</span>, <span class="hlFld-ContribAuthor ">Mingdong  Huang</span>, <span class="hlFld-ContribAuthor ">Jinyu  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Specifically targeting cancer proliferation and metastasis processes: the development of matriptase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cancer and Metastasis Reviews</span><span> <strong>2019,</strong> <em>38 </em>
                                    (3)
                                     , 507-524. <a href="https://doi.org/10.1007/s10555-019-09802-8" title="DOI URL">https://doi.org/10.1007/s10555-019-09802-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10555-019-09802-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10555-019-09802-8%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520and%2520Metastasis%2520Reviews%26atitle%3DSpecifically%252Btargeting%252Bcancer%252Bproliferation%252Band%252Bmetastasis%252Bprocesses%25253A%252Bthe%252Bdevelopment%252Bof%252Bmatriptase%252Binhibitors%26aulast%3DZuo%26aufirst%3DKe%26date%3D2019%26date%3D2019%26volume%3D38%26issue%3D3%26spage%3D507%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tonghui  Huang</span>, <span class="hlFld-ContribAuthor ">Xin  Wu</span>, <span class="hlFld-ContribAuthor ">Yongbo  Yu</span>, <span class="hlFld-ContribAuthor ">Lin  An</span>, <span class="hlFld-ContribAuthor ">Xiaoxing  Yin</span>. </span><span class="cited-content_cbyCitation_article-title">A convenient synthesis of 2-acyl benzothiazoles/thiazoles from benzothiazole/thiazole and N,N'-carbonyldiimidazole activated carboxylic acids. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2019,</strong> <em>60 </em>
                                    (25)
                                     , 1667-1670. <a href="https://doi.org/10.1016/j.tetlet.2019.05.043" title="DOI URL">https://doi.org/10.1016/j.tetlet.2019.05.043</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2019.05.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2019.05.043%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DA%252Bconvenient%252Bsynthesis%252Bof%252B2-acyl%252Bbenzothiazoles%25252Fthiazoles%252Bfrom%252Bbenzothiazole%25252Fthiazole%252Band%252BN%25252CN%252527-carbonyldiimidazole%252Bactivated%252Bcarboxylic%252Bacids%26aulast%3DHuang%26aufirst%3DTonghui%26date%3D2019%26volume%3D60%26issue%3D25%26spage%3D1667%26epage%3D1670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vishnu C  Damalanka</span>, <span class="hlFld-ContribAuthor ">James W  Janetka</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress on inhibitors of the type II transmembrane serine proteases, hepsin, matriptase and matriptase-2. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (7)
                                     , 743-769. <a href="https://doi.org/10.4155/fmc-2018-0446" title="DOI URL">https://doi.org/10.4155/fmc-2018-0446</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0446%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bon%252Binhibitors%252Bof%252Bthe%252Btype%252BII%252Btransmembrane%252Bserine%252Bproteases%25252C%252Bhepsin%25252C%252Bmatriptase%252Band%252Bmatriptase-2%26aulast%3DDamalanka%26aufirst%3DVishnu%2BC%26date%3D2019%26volume%3D11%26issue%3D7%26spage%3D743%26epage%3D769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nai  She</span>, <span class="hlFld-ContribAuthor ">Yuan  Zhao</span>, <span class="hlFld-ContribAuthor ">Jingjing  Hao</span>, <span class="hlFld-ContribAuthor ">Songqiang  Xie</span>, <span class="hlFld-ContribAuthor ">Chaojie  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Uridine diphosphate release mechanism in O-N-acetylglucosamine (O-GlcNAc) transferase catalysis. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2019,</strong> <em>1863 </em>
                                    (3)
                                     , 609-622. <a href="https://doi.org/10.1016/j.bbagen.2018.12.005" title="DOI URL">https://doi.org/10.1016/j.bbagen.2018.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2018.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2018.12.005%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DUridine%252Bdiphosphate%252Brelease%252Bmechanism%252Bin%252BO-N-acetylglucosamine%252B%252528O-GlcNAc%252529%252Btransferase%252Bcatalysis%26aulast%3DShe%26aufirst%3DNai%26date%3D2019%26volume%3D1863%26issue%3D3%26spage%3D609%26epage%3D622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Agata  Gitlin-Domagalska</span>, <span class="hlFld-ContribAuthor ">Martin  Mangold</span>, <span class="hlFld-ContribAuthor ">Dawid  Dębowski</span>, <span class="hlFld-ContribAuthor ">Natalia  Ptaszyńska</span>, <span class="hlFld-ContribAuthor ">Anna  Łęgowska</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Krzysztof  Rolka</span>. </span><span class="cited-content_cbyCitation_article-title">Matriptase-2: monitoring and inhibiting its proteolytic activity. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2018,</strong> <em>10 </em>
                                    (23)
                                     , 2745-2761. <a href="https://doi.org/10.4155/fmc-2018-0346" title="DOI URL">https://doi.org/10.4155/fmc-2018-0346</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0346%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DMatriptase-2%25253A%252Bmonitoring%252Band%252Binhibiting%252Bits%252Bproteolytic%252Bactivity%26aulast%3DGitlin-Domagalska%26aufirst%3DAgata%26date%3D2018%26volume%3D10%26issue%3D23%26spage%3D2745%26epage%3D2761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Mangold</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Marit  Stirnberg</span>. </span><span class="cited-content_cbyCitation_article-title">A Short Peptide Inhibitor as an Activity-Based Probe for Matriptase-2. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2018,</strong> <em>11 </em>
                                    (2)
                                     , 49. <a href="https://doi.org/10.3390/ph11020049" title="DOI URL">https://doi.org/10.3390/ph11020049</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph11020049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph11020049%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DA%252BShort%252BPeptide%252BInhibitor%252Bas%252Ban%252BActivity-Based%252BProbe%252Bfor%252BMatriptase-2%26aulast%3DMangold%26aufirst%3DMartin%26date%3D2018%26date%3D2018%26volume%3D11%26issue%3D2%26spage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Louis-Philippe  Morency</span>, <span class="hlFld-ContribAuthor ">Francis  Gaudreault</span>, <span class="hlFld-ContribAuthor ">Rafael  Najmanovich</span>. </span><span class="cited-content_cbyCitation_article-title">Applications of the NRGsuite and the Molecular Docking Software FlexAID in Computational Drug Discovery and Design. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 367-388. <a href="https://doi.org/10.1007/978-1-4939-7756-7_18" title="DOI URL">https://doi.org/10.1007/978-1-4939-7756-7_18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-7756-7_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-7756-7_18%26sid%3Dliteratum%253Aachs%26atitle%3DApplications%252Bof%252Bthe%252BNRGsuite%252Band%252Bthe%252BMolecular%252BDocking%252BSoftware%252BFlexAID%252Bin%252BComputational%252BDrug%252BDiscovery%252Band%252BDesign%26aulast%3DMorency%26aufirst%3DLouis-Philippe%26date%3D2018%26date%3D2018%26spage%3D367%26epage%3D388%26pub%3DSpringer%2520New%2520York%26atitle%3DComputational%252BDrug%252BDiscovery%252Band%252BDesign%26aulast%3DGore%26aufirst%3DMohini%26date%3D2018%26volume%3D1762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Torsten  Steinmetzer</span>, <span class="hlFld-ContribAuthor ">Kornelia  Hardes</span>. </span><span class="cited-content_cbyCitation_article-title">The Antiviral Potential of Host Protease Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 279-325. <a href="https://doi.org/10.1007/978-3-319-75474-1_11" title="DOI URL">https://doi.org/10.1007/978-3-319-75474-1_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-75474-1_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-75474-1_11%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BAntiviral%252BPotential%252Bof%252BHost%252BProtease%252BInhibitors%26aulast%3DSteinmetzer%26aufirst%3DTorsten%26date%3D2018%26date%3D2018%26spage%3D279%26epage%3D325%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DActivation%252Bof%252BViruses%252Bby%252BHost%252BProteases%26aulast%3DB%25C3%25B6ttcher-Friebertsh%25C3%25A4user%26aufirst%3DEva%26date%3D2018%26volume%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Häußler</span>, <span class="hlFld-ContribAuthor ">Anna-Christina  Schulz-Fincke</span>, <span class="hlFld-ContribAuthor ">Anna-Madeleine  Beckmann</span>, <span class="hlFld-ContribAuthor ">Aline  Keils</span>, <span class="hlFld-ContribAuthor ">Erik  Gilberg</span>, <span class="hlFld-ContribAuthor ">Martin  Mangold</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>, <span class="hlFld-ContribAuthor ">Marit  Stirnberg</span>, <span class="hlFld-ContribAuthor ">Torsten  Steinmetzer</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>. </span><span class="cited-content_cbyCitation_article-title">A Fluorescent-Labeled Phosphono Bisbenzguanidine As an Activity-Based Probe for Matriptase. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2017,</strong> <em>23 </em>
                                    (22)
                                     , 5205-5209. <a href="https://doi.org/10.1002/chem.201700319" title="DOI URL">https://doi.org/10.1002/chem.201700319</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201700319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201700319%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DA%252BFluorescent-Labeled%252BPhosphono%252BBisbenzguanidine%252BAs%252Ban%252BActivity-Based%252BProbe%252Bfor%252BMatriptase%26aulast%3DH%25C3%25A4u%25C3%259Fler%26aufirst%3DDaniela%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D22%26spage%3D5205%26epage%3D5209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catherine  St-Georges</span>, <span class="hlFld-ContribAuthor ">Antoine  Désilets</span>, <span class="hlFld-ContribAuthor ">François  Béliveau</span>, <span class="hlFld-ContribAuthor ">Mariana  Ghinet</span>, <span class="hlFld-ContribAuthor ">Sébastien P.  Dion</span>, <span class="hlFld-ContribAuthor ">Éloic  Colombo</span>, <span class="hlFld-ContribAuthor ">Pierre-Luc  Boudreault</span>, <span class="hlFld-ContribAuthor ">Rafael J.  Najmanovich</span>, <span class="hlFld-ContribAuthor ">Richard  Leduc</span>, <span class="hlFld-ContribAuthor ">Éric  Marsault</span>. </span><span class="cited-content_cbyCitation_article-title">Modulating the selectivity of matriptase-2 inhibitors with unnatural amino acids. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>129 </em>, 110-123. <a href="https://doi.org/10.1016/j.ejmech.2017.02.006" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.02.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.02.006%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DModulating%252Bthe%252Bselectivity%252Bof%252Bmatriptase-2%252Binhibitors%252Bwith%252Bunnatural%252Bamino%252Bacids%26aulast%3DSt-Georges%26aufirst%3DCatherine%26date%3D2017%26volume%3D129%26spage%3D110%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erzsebet  Pászti-Gere</span>, <span class="hlFld-ContribAuthor ">Réka Fanni  Barna</span>, <span class="hlFld-ContribAuthor ">Gabriella  Ujhelyi</span>, <span class="hlFld-ContribAuthor ">Torsten  Steinmetzer</span>. </span><span class="cited-content_cbyCitation_article-title">Interaction exists between matriptase inhibitors and intestinal epithelial cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2016,</strong> <em>31 </em>
                                    (5)
                                     , 736-741. <a href="https://doi.org/10.3109/14756366.2015.1060483" title="DOI URL">https://doi.org/10.3109/14756366.2015.1060483</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/14756366.2015.1060483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F14756366.2015.1060483%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DInteraction%252Bexists%252Bbetween%252Bmatriptase%252Binhibitors%252Band%252Bintestinal%252Bepithelial%252Bcells%26aulast%3DP%25C3%25A1szti-Gere%26aufirst%3DErzsebet%26date%3D2016%26date%3D2015%26volume%3D31%26issue%3D5%26spage%3D736%26epage%3D741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna-Madeleine  Beckmann</span>, <span class="hlFld-ContribAuthor ">Erik  Gilberg</span>, <span class="hlFld-ContribAuthor ">Susanne  Gattner</span>, <span class="hlFld-ContribAuthor ">Tien L.  Huang</span>, <span class="hlFld-ContribAuthor ">Jean Jacques  Vanden Eynde</span>, <span class="hlFld-ContribAuthor ">Annie  Mayence</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>, <span class="hlFld-ContribAuthor ">Marit  Stirnberg</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of bisbenzamidines as inhibitors for matriptase-2. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>26 </em>
                                    (15)
                                     , 3741-3745. <a href="https://doi.org/10.1016/j.bmcl.2016.05.071" title="DOI URL">https://doi.org/10.1016/j.bmcl.2016.05.071</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2016.05.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2016.05.071%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DEvaluation%252Bof%252Bbisbenzamidines%252Bas%252Binhibitors%252Bfor%252Bmatriptase-2%26aulast%3DBeckmann%26aufirst%3DAnna-Madeleine%26date%3D2016%26volume%3D26%26issue%3D15%26spage%3D3741%26epage%3D3745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Häußler</span>, <span class="hlFld-ContribAuthor ">Martin  Mangold</span>, <span class="hlFld-ContribAuthor ">Norbert  Furtmann</span>, <span class="hlFld-ContribAuthor ">Annett  Braune</span>, <span class="hlFld-ContribAuthor ">Michael  Blaut</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>, <span class="hlFld-ContribAuthor ">Marit  Stirnberg</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphono Bisbenzguanidines as Irreversible Dipeptidomimetic Inhibitors and Activity-Based Probes of Matriptase-2. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2016,</strong> <em>22 </em>
                                    (25)
                                     , 8525-8535. <a href="https://doi.org/10.1002/chem.201600206" title="DOI URL">https://doi.org/10.1002/chem.201600206</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201600206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201600206%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DPhosphono%252BBisbenzguanidines%252Bas%252BIrreversible%252BDipeptidomimetic%252BInhibitors%252Band%252BActivity-Based%252BProbes%252Bof%252BMatriptase-2%26aulast%3DH%25C3%25A4u%25C3%259Fler%26aufirst%3DDaniela%26date%3D2016%26date%3D2016%26volume%3D22%26issue%3D25%26spage%3D8525%26epage%3D8535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna-Madeleine  Beckmann</span>, <span class="hlFld-ContribAuthor ">Eva  Maurer</span>, <span class="hlFld-ContribAuthor ">Verena  Lülsdorff</span>, <span class="hlFld-ContribAuthor ">Annika  Wilms</span>, <span class="hlFld-ContribAuthor ">Norbert  Furtmann</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Marit  Stirnberg</span>. </span><span class="cited-content_cbyCitation_article-title">En Route to New Therapeutic Options for Iron Overload Diseases: Matriptase-2 as a Target for Kunitz-Type Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2016,</strong> <em>17 </em>
                                    (7)
                                     , 595-604. <a href="https://doi.org/10.1002/cbic.201500651" title="DOI URL">https://doi.org/10.1002/cbic.201500651</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201500651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201500651%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DEn%252BRoute%252Bto%252BNew%252BTherapeutic%252BOptions%252Bfor%252BIron%252BOverload%252BDiseases%25253A%252BMatriptase-2%252Bas%252Ba%252BTarget%252Bfor%252BKunitz-Type%252BInhibitors%26aulast%3DBeckmann%26aufirst%3DAnna-Madeleine%26date%3D2016%26date%3D2016%26volume%3D17%26issue%3D7%26spage%3D595%26epage%3D604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhenfu  Han</span>, <span class="hlFld-ContribAuthor ">Peter K. W.  Harris</span>, <span class="hlFld-ContribAuthor ">Partha  Karmakar</span>, <span class="hlFld-ContribAuthor ">Tommy  Kim</span>, <span class="hlFld-ContribAuthor ">Ben Y.  Owusu</span>, <span class="hlFld-ContribAuthor ">Scott A.  Wildman</span>, <span class="hlFld-ContribAuthor ">Lidija  Klampfer</span>, <span class="hlFld-ContribAuthor ">James W.  Janetka</span>. </span><span class="cited-content_cbyCitation_article-title">α-Ketobenzothiazole Serine Protease Inhibitors of Aberrant HGF/c-MET and MSP/RON Kinase Pathway Signaling in Cancer. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (6)
                                     , 585-599. <a href="https://doi.org/10.1002/cmdc.201500600" title="DOI URL">https://doi.org/10.1002/cmdc.201500600</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500600%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3D%2525CE%2525B1-Ketobenzothiazole%252BSerine%252BProtease%252BInhibitors%252Bof%252BAberrant%252BHGF%25252Fc-MET%252Band%252BMSP%25252FRON%252BKinase%252BPathway%252BSignaling%252Bin%252BCancer%26aulast%3DHan%26aufirst%3DZhenfu%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D6%26spage%3D585%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Polya  Roydeva</span>, <span class="hlFld-ContribAuthor ">Anna-Madeleine  Beckmann</span>, <span class="hlFld-ContribAuthor ">Marit  Stirnberg</span>, <span class="hlFld-ContribAuthor ">Jožko  Cesar</span>, <span class="hlFld-ContribAuthor ">Danijel  Kikelj</span>, <span class="hlFld-ContribAuthor ">Janez  Ilaš</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>. </span><span class="cited-content_cbyCitation_article-title">3,1-Benzothiazines, 1,4-Benzodioxines and 1,4-Benzoxazines as Inhibitors of Matriptase-2: Outcome of a Focused Screening Approach. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2016,</strong> <em>9 </em>
                                    (1)
                                     , 2. <a href="https://doi.org/10.3390/ph9010002" title="DOI URL">https://doi.org/10.3390/ph9010002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph9010002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph9010002%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3D3%25252C1-Benzothiazines%25252C%252B1%25252C4-Benzodioxines%252Band%252B1%25252C4-Benzoxazines%252Bas%252BInhibitors%252Bof%252BMatriptase-2%25253A%252BOutcome%252Bof%252Ba%252BFocused%252BScreening%252BApproach%26aulast%3DRoydeva%26aufirst%3DPolya%26date%3D2016%26date%3D2016%26volume%3D9%26issue%3D1%26spage%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Norbert  Furtmann</span>, <span class="hlFld-ContribAuthor ">Daniela  Häußler</span>, <span class="hlFld-ContribAuthor ">Tamara  Scheidt</span>, <span class="hlFld-ContribAuthor ">Marit  Stirnberg</span>, <span class="hlFld-ContribAuthor ">Torsten  Steinmetzer</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>. </span><span class="cited-content_cbyCitation_article-title">Limiting the Number of Potential Binding Modes by Introducing Symmetry into Ligands: Structure-Based Design of Inhibitors for Trypsin-Like Serine Proteases. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2016,</strong> <em>22 </em>
                                    (2)
                                     , 610-625. <a href="https://doi.org/10.1002/chem.201503534" title="DOI URL">https://doi.org/10.1002/chem.201503534</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201503534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201503534%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DLimiting%252Bthe%252BNumber%252Bof%252BPotential%252BBinding%252BModes%252Bby%252BIntroducing%252BSymmetry%252Binto%252BLigands%25253A%252BStructure-Based%252BDesign%252Bof%252BInhibitors%252Bfor%252BTrypsin-Like%252BSerine%252BProteases%26aulast%3DFurtmann%26aufirst%3DNorbert%26date%3D2016%26date%3D2015%26volume%3D22%26issue%3D2%26spage%3D610%26epage%3D625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francis  Gaudreault</span>, <span class="hlFld-ContribAuthor ">Louis-Philippe  Morency</span>, <span class="hlFld-ContribAuthor ">Rafael J.  Najmanovich</span>. </span><span class="cited-content_cbyCitation_article-title">NRGsuite: a PyMOL plugin to perform docking simulations in real time using FlexAID. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2015,</strong> <em>9 </em>, btv458. <a href="https://doi.org/10.1093/bioinformatics/btv458" title="DOI URL">https://doi.org/10.1093/bioinformatics/btv458</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btv458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtv458%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DNRGsuite%25253A%252Ba%252BPyMOL%252Bplugin%252Bto%252Bperform%252Bdocking%252Bsimulations%252Bin%252Breal%252Btime%252Busing%252BFlexAID%26aulast%3DGaudreault%26aufirst%3DFrancis%26date%3D2015%26date%3D2015%26volume%3D9%26spage%3Dbtv458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Agata  Gitlin</span>, <span class="hlFld-ContribAuthor ">Dawid  Dębowski</span>, <span class="hlFld-ContribAuthor ">Natalia  Karna</span>, <span class="hlFld-ContribAuthor ">Anna  Łęgowska</span>, <span class="hlFld-ContribAuthor ">Marit  Stirnberg</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Krzysztof  Rolka</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Matriptase-2 Based on the Trypsin Inhibitor SFTI-1. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2015,</strong> <em>16 </em>
                                    (11)
                                     , 1601-1607. <a href="https://doi.org/10.1002/cbic.201500200" title="DOI URL">https://doi.org/10.1002/cbic.201500200</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201500200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201500200%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DInhibitors%252Bof%252BMatriptase-2%252BBased%252Bon%252Bthe%252BTrypsin%252BInhibitor%252BSFTI-1%26aulast%3DGitlin%26aufirst%3DAgata%26date%3D2015%26date%3D2015%26volume%3D16%26issue%3D11%26spage%3D1601%26epage%3D1607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/medium/jm-2014-015633_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5015633&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/medium/jm-2014-015633_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Comparison of experimental and computed average preferences at position P<sub>3</sub> in IP<sub>3</sub>XR. Each point represents the average of all other 19 possibilities in position P<sub>2</sub>. Colors represent amino acid types: aromatic (orange), positively (blue) or negatively (red) charged, polar (green), and hydrophobic (gray). The inset shows the general structure of the compounds used in this study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5015633&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/medium/jm-2014-015633_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Analysis of subpocket preferences for top scoring compounds. The top scoring 800 compounds for matriptase (blue) and matriptase-2 (red) are shown in a parallel plot. Each line connecting the three subpocket-binding residues represents one compound. At each position, the color gradient corresponds to the fraction of top compounds that use the given amino acid at the particular position, white means equally distributed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5015633&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/medium/jm-2014-015633_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Shannon entropy of different compound clusters based on <i>Z</i>-score differences across different subpockets. The number of compounds in each cluster is shown next to the <i>Z</i>-score outer ring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-23/jm5015633/20141205/images/large/jm-2014-015633_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5015633&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Rawlings, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, A.</span><span> </span><span class="NLM_article-title">MEROPS: the Database of Proteolytic Enzymes, Their Substrates and Inhibitors</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">D503</span><span class="NLM_x">–</span> <span class="NLM_lpage">D509</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1093%2Fnar%2Fgkt953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=24157837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Cj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D503-D509&author=N.+D.+Rawlingsauthor=M.+Wallerauthor=A.+J.+Barrettauthor=A.+Bateman&title=MEROPS%3A+the+Database+of+Proteolytic+Enzymes%2C+Their+Substrates+and+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">MEROPS: the database of proteolytic enzymes, their substrates and inhibitors</span></div><div class="casAuthors">Rawlings, Neil D.; Waller, Matthew; Barrett, Alan J.; Bateman, Alex</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D503-D509</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Peptidases, their substrates and inhibitors are of great relevance to biol., medicine and biotechnol.  The MEROPS database (http://merops.sanger.ac.uk) aims to fulfill the need for an integrated source of information about these.  The database has hierarchical classifications in which homologous sets of peptidases and protein inhibitors are grouped into protein species, which are grouped into families, which are in turn grouped into clans.  Recent developments include the following.  A community annotation project has been instigated in which acknowledged experts are invited to contribute summaries for peptidases.  Software has been written to provide an Internet-based data entry form.  Contributors are acknowledged on the relevant web page.  A new display showing the intron/exon structures of eukaryote peptidase genes and the phasing of the junctions has been implemented.  It is now possible to filter the list of peptidases from a completely sequenced bacterial genome for a particular strain of the organism.  The MEROPS filing pipeline has been altered to circumvent the restrictions imposed on non-interactive blastp searches, and a HMMER search using specially generated alignments to maximize the distribution of organisms returned in the search results has been added.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmCK1nNJsPmrVg90H21EOLACvtfcHk0lgjzvt4ugufzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Cj&md5=97e3fe5e7e2e09bd1971efcd425d22bc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt953%26sid%3Dliteratum%253Aachs%26aulast%3DRawlings%26aufirst%3DN.%2BD.%26aulast%3DWaller%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DA.%2BJ.%26aulast%3DBateman%26aufirst%3DA.%26atitle%3DMEROPS%253A%2520the%2520Database%2520of%2520Proteolytic%2520Enzymes%252C%2520Their%2520Substrates%2520and%2520Inhibitors%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD503%26epage%3DD509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Page, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Cera, E.</span><span> </span><span class="NLM_article-title">Serine Peptidases: Classification, Structure and Function</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1220</span><span class="NLM_x">–</span> <span class="NLM_lpage">1236</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1007%2Fs00018-008-7565-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=18259688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFynsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2008&pages=1220-1236&author=M.+J.+Pageauthor=E.+Di+Cera&title=Serine+Peptidases%3A+Classification%2C+Structure+and+Function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Serine peptidases: Classification, structure and function</span></div><div class="casAuthors">Page, M. J.; Di Cera, E.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">1220-1236</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Serine proteases play key roles in human health and disease and their biochem. properties have shaped the mol. evolution of these processes.  Of known proteolytic enzymes, the serine protease family is the major cornerstone of the vertebrate degradome; over 1/3 of all known proteolytic enzymes are serine proteases.  The authors describe the known diversity of serine proteases with respect to structure and function.  In the protease database MEROPS, >26,000 serine proteases are grouped into 13 clans and 40 families.  Particular emphasis is placed on the S1 protease family, the trypsins, which underwent the most predominant genetic expansion yielding the enzymes responsible for vital processes in man such as digestion, blood coagulation, fibrinolysis, development, fertilization, apoptosis, and immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoigIRqJJCserVg90H21EOLACvtfcHk0lgjzvt4ugufzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFynsLs%253D&md5=cf9fa1b377a106e5ae42a7c2edea789d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs00018-008-7565-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-008-7565-9%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BJ.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DSerine%2520Peptidases%253A%2520Classification%252C%2520Structure%2520and%2520Function%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2008%26volume%3D65%26spage%3D1220%26epage%3D1236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Bugge, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antalis, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span> </span><span class="NLM_article-title">Type II Transmembrane Serine Proteases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">23177</span><span class="NLM_x">–</span> <span class="NLM_lpage">23181</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1074%2Fjbc.R109.021006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=19487698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWqtrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=23177-23181&author=T.+H.+Buggeauthor=T.+M.+Antalisauthor=Q.+Wu&title=Type+II+Transmembrane+Serine+Proteases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Type II transmembrane serine proteases</span></div><div class="casAuthors">Bugge, Thomas H.; Antalis, Toni M.; Wu, Qingyu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">23177-23181</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Anal. of genome and expressed sequence tag data bases at the turn of the millennium unveiled a new protease family named the type II transmembrane serine proteases (TTSPs).  Since then, the no. of known TTSPs has more than doubled, and more importantly, the understanding of the physiol. functions of individual TTSPs and their contribution to human disease has greatly increased.  Progress has also been made in identifying mol. substrates and endogenous inhibitors.  Here, the authors summarize current knowledge of the rapidly advancing TTSP field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg6e9fDiyCZbVg90H21EOLACvtfcHk0lhmlyt3gO0Vfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWqtrrL&md5=90a98263918e10142083c497307834b2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R109.021006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R109.021006%26sid%3Dliteratum%253Aachs%26aulast%3DBugge%26aufirst%3DT.%2BH.%26aulast%3DAntalis%26aufirst%3DT.%2BM.%26aulast%3DWu%26aufirst%3DQ.%26atitle%3DType%2520II%2520Transmembrane%2520Serine%2520Proteases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D23177%26epage%3D23181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Oberst, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickson, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.-Y.</span><span> </span><span class="NLM_article-title">The Activation of Matriptase Requires Its Noncatalytic Domains, Serine Protease Domain, and Its Cognate Inhibitor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">26773</span><span class="NLM_x">–</span> <span class="NLM_lpage">26779</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=26773-26779&author=M.+D.+Oberstauthor=C.+A.+Williamsauthor=R.+B.+Dicksonauthor=M.+D.+Johnsonauthor=C.-Y.+Lin&title=The+Activation+of+Matriptase+Requires+Its+Noncatalytic+Domains%2C+Serine+Protease+Domain%2C+and+Its+Cognate+Inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOberst%26aufirst%3DM.%2BD.%26aulast%3DWilliams%26aufirst%3DC.%2BA.%26aulast%3DDickson%26aufirst%3DR.%2BB.%26aulast%3DJohnson%26aufirst%3DM.%2BD.%26aulast%3DLin%26aufirst%3DC.-Y.%26atitle%3DThe%2520Activation%2520of%2520Matriptase%2520Requires%2520Its%2520Noncatalytic%2520Domains%252C%2520Serine%2520Protease%2520Domain%252C%2520and%2520Its%2520Cognate%2520Inhibitor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D26773%26epage%3D26779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Jiang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuo, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span> </span><span class="NLM_article-title"><i>N</i>-Glycosylation Is Required for Matriptase-2 Autoactivation and Ectodomain Shedding</span> <span class="citation_source-journal">J. Biol. Chemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">19500</span><span class="NLM_x">–</span> <span class="NLM_lpage">19507</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1074%2Fjbc.M114.555110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=24867957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCktLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=19500-19507&author=J.+Jiangauthor=J.+Yangauthor=P.+Fengauthor=B.+Zuoauthor=N.+Dongauthor=Q.+Wuauthor=Y.+He&title=N-Glycosylation+Is+Required+for+Matriptase-2+Autoactivation+and+Ectodomain+Shedding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">N-Glycosylation is required for matriptase-2 autoactivation and ectodomain shedding</span></div><div class="casAuthors">Jiang, Jiang; Yang, Jianfeng; Feng, Ping; Zuo, Bin; Dong, Ningzheng; Wu, Qingyu; He, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">19500-19507</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Matriptase-2 (I) is a hepatic membrane serine protease that regulates Fe homeostasis.  Defects in I cause Fe deficiency anemia.  In cells, I is synthesized as a zymogen.  To date, how I expression and activation are regulated remains poorly understood.  Here, the authors expressed human I in HEK293 and hepatic BEL-7402, SMMC-7721, and QGY-7703 cells.  By labeling cell surface proteins and Western anal., the authors examd. I cell surface expression, zymogen activation, and ectodomain shedding.  The results showed that I was activated on the cell surface but not intracellularly.  Activated I underwent ectodomain shedding, producing sol. fragments in the conditioned medium.  By testing inactive mutants, R576A and S762A, the authors found that I activation and shedding were mediated by its own catalytic activity and that the one-chain form of I had little activity in ectodomain shedding.  The authors made addnl. I mutants (N136Q, N184Q, N216Q, N338Q, N433Q, N453Q, and N518Q), in which each of the predicted N-glycosylation sites was mutated.  All of these mutants were expressed on the cell surface.  However, mutants N216Q, N453Q, and N518Q, but not the other mutants, had impaired zymogen activation and ectodomain shedding.  These results indicated that N-glycans at specific sites are crit. for I activation.  Together, these data provide new insights into the cell surface expression, zymogen activation, and ectodomain shedding of I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2g18hlGd6rVg90H21EOLACvtfcHk0lhmlyt3gO0Vfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCktLrE&md5=7564171896db7b44c9da0ecbe29193f1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.555110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.555110%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DP.%26aulast%3DZuo%26aufirst%3DB.%26aulast%3DDong%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%26atitle%3DN-Glycosylation%2520Is%2520Required%2520for%2520Matriptase-2%2520Autoactivation%2520and%2520Ectodomain%2520Shedding%26jtitle%3DJ.%2520Biol.%2520Chemistry%26date%3D2014%26volume%3D289%26spage%3D19500%26epage%3D19507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Szabo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobson, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christoph, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">List, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugge, T. H.</span><span> </span><span class="NLM_article-title">Regulation of Cell Surface Protease Matriptase by HAI2 Is Essential for Placental Development, Neural Tube Closure and Embryonic Survival in Mice</span> <span class="citation_source-journal">Development</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">2653</span><span class="NLM_x">–</span> <span class="NLM_lpage">2663</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=2653-2663&author=R.+Szaboauthor=J.+P.+Hobsonauthor=K.+Christophauthor=P.+Kosaauthor=K.+Listauthor=T.+H.+Bugge&title=Regulation+of+Cell+Surface+Protease+Matriptase+by+HAI2+Is+Essential+for+Placental+Development%2C+Neural+Tube+Closure+and+Embryonic+Survival+in+Mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSzabo%26aufirst%3DR.%26aulast%3DHobson%26aufirst%3DJ.%2BP.%26aulast%3DChristoph%26aufirst%3DK.%26aulast%3DKosa%26aufirst%3DP.%26aulast%3DList%26aufirst%3DK.%26aulast%3DBugge%26aufirst%3DT.%2BH.%26atitle%3DRegulation%2520of%2520Cell%2520Surface%2520Protease%2520Matriptase%2520by%2520HAI2%2520Is%2520Essential%2520for%2520Placental%2520Development%252C%2520Neural%2520Tube%2520Closure%2520and%2520Embryonic%2520Survival%2520in%2520Mice%26jtitle%3DDevelopment%26date%3D2009%26volume%3D136%26spage%3D2653%26epage%3D2663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">List, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haudenschild, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szabo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaim, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelholm, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Behrendt, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugge, T. H.</span><span> </span><span class="NLM_article-title">Matriptase/MT-SP1 Is Required for Postnatal Survival, Epidermal Barrier Function, Hair Follicle Development, and Thymic Homeostasis</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">3765</span><span class="NLM_x">–</span> <span class="NLM_lpage">3779</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=3765-3779&author=K.+Listauthor=C.+C.+Haudenschildauthor=R.+Szaboauthor=W.+Chenauthor=S.+M.+Wahlauthor=W.+Swaimauthor=L.+H.+Engelholmauthor=N.+Behrendtauthor=T.+H.+Bugge&title=Matriptase%2FMT-SP1+Is+Required+for+Postnatal+Survival%2C+Epidermal+Barrier+Function%2C+Hair+Follicle+Development%2C+and+Thymic+Homeostasis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DList%26aufirst%3DK.%26aulast%3DHaudenschild%26aufirst%3DC.%2BC.%26aulast%3DSzabo%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWahl%26aufirst%3DS.%2BM.%26aulast%3DSwaim%26aufirst%3DW.%26aulast%3DEngelholm%26aufirst%3DL.%2BH.%26aulast%3DBehrendt%26aufirst%3DN.%26aulast%3DBugge%26aufirst%3DT.%2BH.%26atitle%3DMatriptase%252FMT-SP1%2520Is%2520Required%2520for%2520Postnatal%2520Survival%252C%2520Epidermal%2520Barrier%2520Function%252C%2520Hair%2520Follicle%2520Development%252C%2520and%2520Thymic%2520Homeostasis%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D3765%26epage%3D3779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Milner, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swingler, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelso, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donell, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cawston, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrell, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plevin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowan, A. D.</span><span> </span><span class="NLM_article-title">Matriptase Is a Novel Initiator of Cartilage Matrix Degradation in Osteoarthritis</span> <span class="citation_source-journal">Arthritis Rheum.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">1955</span><span class="NLM_x">–</span> <span class="NLM_lpage">1966</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=20506309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1aqsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=1955-1966&author=J.+M.+Milnerauthor=A.+Patelauthor=R.+K.+Davidsonauthor=T.+E.+Swinglerauthor=A.+D%C3%A9siletsauthor=D.+A.+Youngauthor=E.+B.+Kelsoauthor=S.+T.+Donellauthor=T.+E.+Cawstonauthor=I.+M.+Clarkauthor=W.+R.+Ferrellauthor=R.+Plevinauthor=J.+C.+Lockhartauthor=R.+Leducauthor=A.+D.+Rowan&title=Matriptase+Is+a+Novel+Initiator+of+Cartilage+Matrix+Degradation+in+Osteoarthritis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Matriptase is a novel initiator of cartilage matrix degradation in osteoarthritis</span></div><div class="casAuthors">Milner, Jennifer M.; Patel, Amit; Davidson, Rose K.; Swingler, Tracey E.; Desilets, Antoine; Young, David A.; Kelso, Elizabeth B.; Donell, Simon T.; Cawston, Tim E.; Clark, Ian M.; Ferrell, William R.; Plevin, Robin; Lockhart, John C.; Leduc, Richard; Rowan, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatism</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1955-1966</span>CODEN:
                <span class="NLM_cas:coden">ARHEAW</span>;
        ISSN:<span class="NLM_cas:issn">0004-3591</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Objective. Increasing evidence implicates serine proteinases in pathol. tissue turnover.  The aim of this study was to assess the role of the transmembrane serine proteinase matriptase in cartilage destruction in osteoarthritis (OA).  Methods. Serine proteinase gene expression in femoral head cartilage obtained from either patients with hip OA or patients with fracture to the neck of the femur (NOF) was assessed using a low-d. array.  The effect of matriptase on collagen breakdown was detd. in cartilage degrdn. models, while the effect on matrix metalloproteinase (MMP) expression was analyzed by real-time polymerase chain reaction.  ProMMP processing was detd. using sodium dodecyl sulfate-polyacrylamide gel electrophoresis/N-terminal sequencing, while its ability to activate proteinase-activated receptor 2 (PAR-2) was detd. using a synovial perfusion assay in mice.  Results. Matriptase gene expression was significantly elevated in OA cartilage compared with NOF cartilage, and matriptase was immunolocalized to OA chondrocytes.  We showed that matriptase activated proMMP-1 and processed proMMP-3 to its fully active form.  Exogenous matriptase significantly enhanced cytokine-stimulated cartilage collagenolysis, while matriptase alone caused significant collagenolysis from OA cartilage, which was metalloproteinase-dependent.  Matriptase also induced MMP-1, MMP-3, and MMP-13 gene expression.  Synovial perfusion data confirmed that matriptase activates PAR-2, and we demonstrated that matriptase-dependent enhancement of collagenolysis from OA cartilage is blocked by PAR-2 inhibition.  Conclusion. Elevated matriptase expression in OA and the ability of matriptase to activate selective proMMPs as well as induce collagenase expression make this serine proteinase a key initiator and inducer of cartilage destruction in OA.  We propose that the indirect effects of matriptase are mediated by PAR-2, and a more detailed understanding of these mechanisms may highlight important new therapeutic targets for OA treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGNctphoRE3bVg90H21EOLACvtfcHk0lhVk_qPUwZhfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1aqsL%252FL&md5=af039e6d95a18d1b9877eaa9d75b0672</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMilner%26aufirst%3DJ.%2BM.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDavidson%26aufirst%3DR.%2BK.%26aulast%3DSwingler%26aufirst%3DT.%2BE.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DD.%2BA.%26aulast%3DKelso%26aufirst%3DE.%2BB.%26aulast%3DDonell%26aufirst%3DS.%2BT.%26aulast%3DCawston%26aufirst%3DT.%2BE.%26aulast%3DClark%26aufirst%3DI.%2BM.%26aulast%3DFerrell%26aufirst%3DW.%2BR.%26aulast%3DPlevin%26aufirst%3DR.%26aulast%3DLockhart%26aufirst%3DJ.%2BC.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DRowan%26aufirst%3DA.%2BD.%26atitle%3DMatriptase%2520Is%2520a%2520Novel%2520Initiator%2520of%2520Cartilage%2520Matrix%2520Degradation%2520in%2520Osteoarthritis%26jtitle%3DArthritis%2520Rheum.%26date%3D2010%26volume%3D62%26spage%3D1955%26epage%3D1966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Basel-Vanagaite, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Attia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishida-Yamamoto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rainshtein, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ben Amitai, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lurie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasmanik-Chor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Indelman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zvulunov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saban, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magal, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprecher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shohat, M.</span><span> </span><span class="NLM_article-title">Autosomal Recessive Ichthyosis with Hypotrichosis Caused by a Mutation in ST14, Encoding Type II Transmembrane Serine Protease Matriptase</span> <span class="citation_source-journal">Am. J. Hum. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1086%2F512487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=17273967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit1Wksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2007&pages=467-477&author=L.+Basel-Vanagaiteauthor=R.+Attiaauthor=A.+Ishida-Yamamotoauthor=L.+Rainshteinauthor=D.+Ben+Amitaiauthor=R.+Lurieauthor=M.+Pasmanik-Chorauthor=M.+Indelmanauthor=A.+Zvulunovauthor=S.+Sabanauthor=N.+Magalauthor=E.+Sprecherauthor=M.+Shohat&title=Autosomal+Recessive+Ichthyosis+with+Hypotrichosis+Caused+by+a+Mutation+in+ST14%2C+Encoding+Type+II+Transmembrane+Serine+Protease+Matriptase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Autosomal recessive ichthyosis with hypotrichosis caused by a mutation in ST14, encoding type II transmembrane serine protease matriptase</span></div><div class="casAuthors">Basel-Vanagaite, Lina; Attia, Revital; Ishida-Yamamoto, Akemi; Rainshtein, Limor; Ben Amitai, Dan; Lurie, Raziel; Pasmanik-Chor, Metsada; Indelman, Margarita; Zvulunov, Alex; Saban, Shirley; Magal, Nurit; Sprecher, Eli; Shohat, Mordechai</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">467-477</span>CODEN:
                <span class="NLM_cas:coden">AJHGAG</span>;
        ISSN:<span class="NLM_cas:issn">0002-9297</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">In this article, we describe a novel autosomal recessive ichthyosis with hypotrichosis syndrome, characterized by congenital ichthyosis assocd. with abnormal hair.  Using homozygosity mapping, we mapped the disease locus to 11q24.3-q25.  We screened the ST14 gene, which encodes matriptase, since transplantation of skin from matriptase-/--knockout mice onto adult athymic nude mice has been shown elsewhere to result in an ichthyosislike phenotype assocd. with almost complete absence of erupted pelage hairs.  Mutation anal. revealed a missense mutation, G827R, in the highly conserved peptidase S1-S6 domain.  Marked skin hyperkeratosis due to impaired degrdn. of the stratum corneum corneodesmosomes was obsd. in the affected individuals, which suggests that matriptase plays a significant role in epidermal desquamation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTeYW7jyml8rVg90H21EOLACvtfcHk0lhVk_qPUwZhfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit1Wksr4%253D&md5=ee1d1f822e3a4b767f196cff5635e86b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1086%2F512487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F512487%26sid%3Dliteratum%253Aachs%26aulast%3DBasel-Vanagaite%26aufirst%3DL.%26aulast%3DAttia%26aufirst%3DR.%26aulast%3DIshida-Yamamoto%26aufirst%3DA.%26aulast%3DRainshtein%26aufirst%3DL.%26aulast%3DBen%2BAmitai%26aufirst%3DD.%26aulast%3DLurie%26aufirst%3DR.%26aulast%3DPasmanik-Chor%26aufirst%3DM.%26aulast%3DIndelman%26aufirst%3DM.%26aulast%3DZvulunov%26aufirst%3DA.%26aulast%3DSaban%26aufirst%3DS.%26aulast%3DMagal%26aufirst%3DN.%26aulast%3DSprecher%26aufirst%3DE.%26aulast%3DShohat%26aufirst%3DM.%26atitle%3DAutosomal%2520Recessive%2520Ichthyosis%2520with%2520Hypotrichosis%2520Caused%2520by%2520a%2520Mutation%2520in%2520ST14%252C%2520Encoding%2520Type%2520II%2520Transmembrane%2520Serine%2520Protease%2520Matriptase%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2007%26volume%3D80%26spage%3D467%26epage%3D477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Bergum, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">List, K.</span><span> </span><span class="NLM_article-title">Loss of the Matriptase Inhibitor HAI-2 During Prostate Cancer Progression</span> <span class="citation_source-journal">Prostate</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">1422</span><span class="NLM_x">–</span> <span class="NLM_lpage">1428</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=1422-1428&author=C.+Bergumauthor=K.+List&title=Loss+of+the+Matriptase+Inhibitor+HAI-2+During+Prostate+Cancer+Progression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBergum%26aufirst%3DC.%26aulast%3DList%26aufirst%3DK.%26atitle%3DLoss%2520of%2520the%2520Matriptase%2520Inhibitor%2520HAI-2%2520During%2520Prostate%2520Cancer%2520Progression%26jtitle%3DProstate%26date%3D2010%26volume%3D70%26spage%3D1422%26epage%3D1428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Jin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyama, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirosaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higashi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dickson, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitamura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyazaki, K.</span><span> </span><span class="NLM_article-title">Matriptase Activates Stromelysin (MMP-3) and Promotes Tumor Growth and Angiogenesis</span> <span class="citation_source-journal">Cancer Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1327</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1111%2Fj.1349-7006.2006.00328.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=16999819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1OgurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2006&pages=1327-1334&author=X.+Jinauthor=M.+Yagiauthor=N.+Akiyamaauthor=T.+Hirosakiauthor=S.+Higashiauthor=C.-Y.+Linauthor=R.+B.+Dicksonauthor=H.+Kitamuraauthor=K.+Miyazaki&title=Matriptase+Activates+Stromelysin+%28MMP-3%29+and+Promotes+Tumor+Growth+and+Angiogenesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Matriptase activates stromelysin (MMP-3) and promotes tumor growth and angiogenesis</span></div><div class="casAuthors">Jin, Xinlian; Yagi, Motoki; Akiyama, Nagisa; Hirosaki, Tomomi; Higashi, Shouichi; Lin, Chen-Yong; Dickson, Robert B.; Kitamura, Hitoshi; Miyazaki, Kaoru</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1327-1334</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1347-9032</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">Matriptase/MT-SP1, a type II membrane serine protease widely expressed in normal epithelial cells and human carcinoma cells, is thought to be involved in cancer progression.  To clarify this possibility, we overexpressed exogenous matriptase in the human stomach cancer cell line AZ521.  In vitro, the matriptase transfectant (Mat-AZ521) and the control transfectant (Mock-AZ521) showed a similar growth rate, although the satn. cell d. was significantly higher with the Mat-AZ521.  When implanted into nude mice s.c. or i.p., Mat-AZ521 cells grew faster and produced much larger solid tumors than Mock-AZ521 cells.  The overexpression of matriptase in AZ521 cells shortened the survival time of tumor-bearing mice.  Histol. anal. showed that both the no. and the size of blood vessels in tumor tissues were significantly higher in the Mat-AZ521 tumors than the Mock-AZ521 ones.  Moreover, it was found that purified matriptase activated one of the important matrix metalloproteinases, stromelysin (MMP-3).  These results suggest the possibility that the matriptase-dependent activation of MMP-3, as well as the direct activity of matriptase, promotes tumor growth and angiogenesis by enhancing extracellular matrix degrdn. in tumor cell microenvironments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEauNcAfWLhbVg90H21EOLACvtfcHk0liIofMGFta8hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1OgurrI&md5=07758d9ce9d8a0de78e08f0f1bceca75</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2006.00328.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2006.00328.x%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DX.%26aulast%3DYagi%26aufirst%3DM.%26aulast%3DAkiyama%26aufirst%3DN.%26aulast%3DHirosaki%26aufirst%3DT.%26aulast%3DHigashi%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.-Y.%26aulast%3DDickson%26aufirst%3DR.%2BB.%26aulast%3DKitamura%26aufirst%3DH.%26aulast%3DMiyazaki%26aufirst%3DK.%26atitle%3DMatriptase%2520Activates%2520Stromelysin%2520%2528MMP-3%2529%2520and%2520Promotes%2520Tumor%2520Growth%2520and%2520Angiogenesis%26jtitle%3DCancer%2520Sci.%26date%3D2006%26volume%3D97%26spage%3D1327%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Köbel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalloger, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyd, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinney, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowen, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ionescu, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajput, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prentice, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swenerton, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilks, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntsman, D.</span><span> </span><span class="NLM_article-title">Ovarian Carcinoma Subtypes Are Different Diseases: Implications for Biomarker Studies</span> <span class="citation_source-journal">PLoS Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e232</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=e232&author=M.+K%C3%B6belauthor=S.+E.+Kallogerauthor=N.+Boydauthor=S.+McKinneyauthor=E.+Mehlauthor=C.+Palmerauthor=S.+Leungauthor=N.+J.+Bowenauthor=D.+N.+Ionescuauthor=A.+Rajputauthor=L.+M.+Prenticeauthor=D.+Millerauthor=J.+Santosauthor=K.+Swenertonauthor=C.+B.+Gilksauthor=D.+Huntsman&title=Ovarian+Carcinoma+Subtypes+Are+Different+Diseases%3A+Implications+for+Biomarker+Studies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6bel%26aufirst%3DM.%26aulast%3DKalloger%26aufirst%3DS.%2BE.%26aulast%3DBoyd%26aufirst%3DN.%26aulast%3DMcKinney%26aufirst%3DS.%26aulast%3DMehl%26aufirst%3DE.%26aulast%3DPalmer%26aufirst%3DC.%26aulast%3DLeung%26aufirst%3DS.%26aulast%3DBowen%26aufirst%3DN.%2BJ.%26aulast%3DIonescu%26aufirst%3DD.%2BN.%26aulast%3DRajput%26aufirst%3DA.%26aulast%3DPrentice%26aufirst%3DL.%2BM.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DSantos%26aufirst%3DJ.%26aulast%3DSwenerton%26aufirst%3DK.%26aulast%3DGilks%26aufirst%3DC.%2BB.%26aulast%3DHuntsman%26aufirst%3DD.%26atitle%3DOvarian%2520Carcinoma%2520Subtypes%2520Are%2520Different%2520Diseases%253A%2520Implications%2520for%2520Biomarker%2520Studies%26jtitle%3DPLoS%2520Med.%26date%3D2008%26volume%3D5%26spage%3De232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">List, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szabo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bey, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Molinolo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugge, T. H.</span><span> </span><span class="NLM_article-title">Epithelial Integrity Is Maintained by a Matriptase-Dependent Proteolytic Pathway</span> <span class="citation_source-journal">Am. J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">175</span><span class="NLM_x">, </span> <span class="NLM_fpage">1453</span><span class="NLM_x">–</span> <span class="NLM_lpage">1463</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.2353%2Fajpath.2009.090240" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=19717635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlWiur%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2009&pages=1453-1463&author=K.+Listauthor=P.+Kosaauthor=R.+Szaboauthor=A.+L.+Beyauthor=C.+B.+Wangauthor=A.+Molinoloauthor=T.+H.+Bugge&title=Epithelial+Integrity+Is+Maintained+by+a+Matriptase-Dependent+Proteolytic+Pathway"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Epithelial integrity is maintained by a matriptase-dependent proteolytic pathway</span></div><div class="casAuthors">List, Karin; Kosa, Peter; Szabo, Roman; Bey, Alexandra L.; Wang, Chao Becky; Molinolo, Alfredo; Bugge, Thomas H.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1453-1463</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">A pericellular proteolytic pathway initiated by the transmembrane serine protease matriptase plays a crit. role in the terminal differentiation of epidermal tissues.  Matriptase is constitutively expressed in multiple other epithelia, suggesting a putative role of this membrane serine protease in general epithelial homeostasis.  Here we generated mice with conditional deletion of the St14 gene, encoding matriptase, and show that matriptase indeed is essential for the maintenance of multiple types of epithelia in the mouse.  Thus, embryonic or postnatal ablation of St14 in epithelial tissues of diverse origin and function caused severe organ dysfunction, which was often assocd. with increased permeability, loss of tight junction function, mislocation of tight junction-assocd. proteins, and generalized epithelial demise.  The study reveals that the homeostasis of multiple simple and stratified epithelia is matriptase-dependent, and provides an important animal model for the exploration of this membrane serine protease in a range of physiol. and pathol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYPCXcYcgd4rVg90H21EOLACvtfcHk0liIofMGFta8hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlWiur%252FK&md5=2cc86115b8ac59b1a7636558d02fecae</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2009.090240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2009.090240%26sid%3Dliteratum%253Aachs%26aulast%3DList%26aufirst%3DK.%26aulast%3DKosa%26aufirst%3DP.%26aulast%3DSzabo%26aufirst%3DR.%26aulast%3DBey%26aufirst%3DA.%2BL.%26aulast%3DWang%26aufirst%3DC.%2BB.%26aulast%3DMolinolo%26aufirst%3DA.%26aulast%3DBugge%26aufirst%3DT.%2BH.%26atitle%3DEpithelial%2520Integrity%2520Is%2520Maintained%2520by%2520a%2520Matriptase-Dependent%2520Proteolytic%2520Pathway%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2009%26volume%3D175%26spage%3D1453%26epage%3D1463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Hamilton, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gludish, D. W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, G. R.</span><span> </span><span class="NLM_article-title">Cleavage Activation of the Human-Adapted Influenza Virus Subtypes by Matriptase Reveals Both Subtype and Strain Specificities</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">10579</span><span class="NLM_x">–</span> <span class="NLM_lpage">10586</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1128%2FJVI.00306-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=22811538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOmtb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=10579-10586&author=B.+S.+Hamiltonauthor=D.+W.+J.+Gludishauthor=G.+R.+Whittaker&title=Cleavage+Activation+of+the+Human-Adapted+Influenza+Virus+Subtypes+by+Matriptase+Reveals+Both+Subtype+and+Strain+Specificities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cleavage activation of the human-adapted influenza virus subtypes by matriptase reveals both subtype and strain specificities</span></div><div class="casAuthors">Hamilton, Brian S.; Gludish, David W. J.; Whittaker, Gary R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10579-10586</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cleavage activation of the hemagglutinin (HA) precursor is an essential step in the influenza virus replication cycle that is driven by host cell proteases.  HA cleavage activation is required for virus-endosome membrane fusion and the subsequent release of the influenza virus genome into the cytoplasm.  Previous studies have detd. that HA cleavage is most likely driven by either membrane-bound or extracellular trypsin-like proteases that reside in the respiratory tract.  However, there is still uncertainty regarding which proteases are crit. for HA cleavage in vivo.  Therefore, further investigation of HA cleavage activation is needed in order to gain insight into the crit. proteases involved.  Matriptase is a member of the type II transmembrane serine protease family that is highly expressed in a membrane-bound form throughout the respiratory tract.  One feature of matriptase is that, once activated, the catalytic domain is secreted into the extracellular space and so serves as a functional extracellular protease.  In this study, it was detd. that the secreted, catalytic domain of matriptase has the ability to cleave and activate HA from the influenza virus H1 subtype but not the H2 and H3 subtypes.  Furthermore, matriptase selectively cleaved the HA of particular strains within the H1 subtype, revealing both subtype and H1 strain specificity.  Matriptase was also found to activate thrombolytic zymogens that have been shown to cleave and activate the influenza virus HA.  The data demonstrate that matriptase has the ability to cleave HA directly or indirectly by activating HA-cleaving zymogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXLuhD8xGHLLVg90H21EOLACvtfcHk0ljAhisphGRX1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlOmtb3L&md5=d82d35942ca30280356c6d35a518f3f4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1128%2FJVI.00306-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00306-12%26sid%3Dliteratum%253Aachs%26aulast%3DHamilton%26aufirst%3DB.%2BS.%26aulast%3DGludish%26aufirst%3DD.%2BW.%2BJ.%26aulast%3DWhittaker%26aufirst%3DG.%2BR.%26atitle%3DCleavage%2520Activation%2520of%2520the%2520Human-Adapted%2520Influenza%2520Virus%2520Subtypes%2520by%2520Matriptase%2520Reveals%2520Both%2520Subtype%2520and%2520Strain%2520Specificities%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D10579%26epage%3D10586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Beaulieu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gravel, É.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cloutier, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marois, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verreault, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsault, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M. V.</span><span> </span><span class="NLM_article-title">Matriptase Proteolytically Activates Influenza Virus and Promotes Multicycle Replication in the Human Airway Epithelium</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">4237</span><span class="NLM_x">–</span> <span class="NLM_lpage">4251</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1128%2FJVI.03005-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23365447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotVyqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=4237-4251&author=A.+Beaulieuauthor=%C3%89.+Gravelauthor=A.+Cloutierauthor=I.+Maroisauthor=E.+Colomboauthor=A.+D%C3%A9siletsauthor=C.+Verreaultauthor=R.+Leducauthor=E.+Marsaultauthor=M.+V.+Richter&title=Matriptase+Proteolytically+Activates+Influenza+Virus+and+Promotes+Multicycle+Replication+in+the+Human+Airway+Epithelium"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Matriptase proteolytically activates influenza virus and promotes multicycle replication in the human airway epithelium</span></div><div class="casAuthors">Beaulieu, Alexandre; Gravel, Emilie; Cloutier, Alexandre; Marois, Isabelle; Colombo, Eloic; Desilets, Antoine; Verreault, Catherine; Leduc, Richard; Marsault, Eric; Richter, Martin V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4237-4251</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Influenza viruses do not encode any proteases and must rely on host proteases for the proteolytic activation of their surface hemagglutinin proteins to fuse with the infected host cells.  Recent progress in the understanding of human proteases responsible for influenza virus hemagglutinin activation led to the identification of members of the type II transmembrane serine proteases TMPRSS2 and TMPRSS4 and human airway trypsin-like protease; however, none proved to be the sole enzyme responsible for hemagglutinin cleavage.  The authors identify and characterize matriptase as an influenza virus-activating protease capable of supporting multicycle viral replication in the human respiratory epithelium.  Using confocal microscopy, the authors found matriptase to colocalize with hemagglutinin at the apical surface of human epithelial cells and within endosomes, and the sol. form of the protease was able to specifically cleave hemagglutinins from H1 virus, but not from H2 and H3 viruses, in a broad pH range.  Small interfering RNA (siRNA) knockdown of matriptase in human bronchial epithelial cells significantly blocked influenza virus replication in these cells.  Lastly, the authors provide a selective, slow, tight-binding inhibitor of matriptase that significantly reduces viral replication (by 1.5 log) of H1N1 influenza virus, including the 2009 pandemic virus.  The authors' study establishes a three-pronged model for the action of matriptase: activation of incoming viruses in the extracellular space in its shed form, upon viral attachment or exit in its membrane-bound and/or shed forms at the apical surface of epithelial cells, and within endosomes by its membrane-bound form where viral fusion takes place.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWmz0DwP1-qLVg90H21EOLACvtfcHk0ljAhisphGRX1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotVyqsb4%253D&md5=caaa3417ca7693113dcdb07b219bd68b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJVI.03005-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03005-12%26sid%3Dliteratum%253Aachs%26aulast%3DBeaulieu%26aufirst%3DA.%26aulast%3DGravel%26aufirst%3D%25C3%2589.%26aulast%3DCloutier%26aufirst%3DA.%26aulast%3DMarois%26aufirst%3DI.%26aulast%3DColombo%26aufirst%3DE.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DVerreault%26aufirst%3DC.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DRichter%26aufirst%3DM.%2BV.%26atitle%3DMatriptase%2520Proteolytically%2520Activates%2520Influenza%2520Virus%2520and%2520Promotes%2520Multicycle%2520Replication%2520in%2520the%2520Human%2520Airway%2520Epithelium%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D4237%26epage%3D4251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Baron, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarnow, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayoli-Nüssle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schilling, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwalm, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinmetzer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garten, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klenk, H.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Böttcher-Friebertshäuser, E.</span><span> </span><span class="NLM_article-title">Matriptase, HAT, and TMPRSS2 Activate the Hemagglutinin of H9N2 Influenza a Viruses</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">1811</span><span class="NLM_x">–</span> <span class="NLM_lpage">1820</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1128%2FJVI.02320-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23192872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltV2itLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=1811-1820&author=J.+Baronauthor=C.+Tarnowauthor=D.+Mayoli-N%C3%BCssleauthor=E.+Schillingauthor=D.+Meyerauthor=M.+Hammamiauthor=F.+Schwalmauthor=T.+Steinmetzerauthor=Y.+Guanauthor=W.+Gartenauthor=H.-D.+Klenkauthor=E.+B%C3%B6ttcher-Friebertsh%C3%A4user&title=Matriptase%2C+HAT%2C+and+TMPRSS2+Activate+the+Hemagglutinin+of+H9N2+Influenza+a+Viruses"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Matriptase, HAT, and TMPRSS2 activate the hemagglutinin of H9N2 influenza A viruses</span></div><div class="casAuthors">Baron, Joanna; Tarnow, Carolin; Mayoli-Nuessle, Deborah; Schilling, Eva; Hammami, Maya; Schwalm, Folker; Steinmetzer, Torsten; Guan, Yi; Garten, Wolfgang; Klenk, Hans-Dieter; Boettcher-Friebertshaeuser, Eva</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1811-1820</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Influenza A viruses of the subtype H9N2 circulate worldwide and have become highly prevalent in poultry in many countries.  Moreover, they are occasionally transmitted to humans, raising concern about their pandemic potential.  Influenza virus infectivity requires cleavage of the surface glycoprotein hemagglutinin (HA) at a distinct cleavage site by host cell proteases.  H9N2 viruses vary remarkably in the amino acid sequence at the cleavage site, and many isolates from Asia and the Middle East possess the multibasic motifs R-S-S-R and R-S-R-R, but are not activated by furin.  Here, we investigated proteolytic activation of the early H9N2 isolate A/turkey/Wisconsin/1/66 (H9-Wisc) and two recent Asian isolates, A/quail/Shantou/782/00 (H9-782) and A/quail/Shantou/2061/00 (H9-2061), contg. mono-, di-, and tribasic HA cleavage sites, resp.  All H9N2 isolates were activated by human proteases TMPRSS2 (transmembrane protease, serine S1 member 2) and HAT (human airway trypsin-like protease).  Interestingly, H9-782 and H9-2061 were also activated by matriptase, a protease widely expressed in most epithelia with high expression levels in the kidney.  Nephrotropism of H9N2 viruses has been obsd. in chickens, and here we found that H9-782 and H9-2061 were proteolytically activated in canine kidney (MDCK-II) and chicken embryo kidney (CEK) cells, whereas H9-Wisc was not.  Virus activation was inhibited by peptide-mimetic inhibitors of matriptase, strongly suggesting that matriptase is responsible for HA cleavage in these kidney cells.  Our data demonstrate that H9N2 viruses with R-S-S-R or R-S-R-R cleavage sites are activated by matriptase in addn. to HAT and TMPRSS2 and, therefore, can be activated in a wide range of tissues what may affect virus spread, tissue tropism and pathogenicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYEOrvA-4zH7Vg90H21EOLACvtfcHk0lhP1jpjcAMHWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltV2itLg%253D&md5=2bb0516ece577cbbce5ab3130bbb4eed</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1128%2FJVI.02320-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02320-12%26sid%3Dliteratum%253Aachs%26aulast%3DBaron%26aufirst%3DJ.%26aulast%3DTarnow%26aufirst%3DC.%26aulast%3DMayoli-N%25C3%25BCssle%26aufirst%3DD.%26aulast%3DSchilling%26aufirst%3DE.%26aulast%3DMeyer%26aufirst%3DD.%26aulast%3DHammami%26aufirst%3DM.%26aulast%3DSchwalm%26aufirst%3DF.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DGuan%26aufirst%3DY.%26aulast%3DGarten%26aufirst%3DW.%26aulast%3DKlenk%26aufirst%3DH.-D.%26aulast%3DB%25C3%25B6ttcher-Friebertsh%25C3%25A4user%26aufirst%3DE.%26atitle%3DMatriptase%252C%2520HAT%252C%2520and%2520TMPRSS2%2520Activate%2520the%2520Hemagglutinin%2520of%2520H9N2%2520Influenza%2520a%2520Viruses%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D1811%26epage%3D1820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Ramsay, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooper, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folgueras, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">López-Otín, C.</span><span> </span><span class="NLM_article-title">Matriptase-2 (TMPRSS6): A Proteolytic Regulator of Iron Homeostasis</span> <span class="citation_source-journal">Haematologica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">840</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2009&pages=840-849&author=A.+J.+Ramsayauthor=J.+D.+Hooperauthor=A.+R.+Folguerasauthor=G.+Velascoauthor=C.+L%C3%B3pez-Ot%C3%ADn&title=Matriptase-2+%28TMPRSS6%29%3A+A+Proteolytic+Regulator+of+Iron+Homeostasis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRamsay%26aufirst%3DA.%2BJ.%26aulast%3DHooper%26aufirst%3DJ.%2BD.%26aulast%3DFolgueras%26aufirst%3DA.%2BR.%26aulast%3DVelasco%26aufirst%3DG.%26aulast%3DL%25C3%25B3pez-Ot%25C3%25ADn%26aufirst%3DC.%26atitle%3DMatriptase-2%2520%2528TMPRSS6%2529%253A%2520A%2520Proteolytic%2520Regulator%2520of%2520Iron%2520Homeostasis%26jtitle%3DHaematologica%26date%3D2009%26volume%3D94%26spage%3D840%26epage%3D849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">She, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelbart, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truksa, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khovananth, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mudd, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moresco, E. M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beutler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beutler, B.</span><span> </span><span class="NLM_article-title">The Serine Protease TMPRSS6 Is Required to Sense Iron Deficiency</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">1088</span><span class="NLM_x">–</span> <span class="NLM_lpage">1092</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2008&pages=1088-1092&author=X.+Duauthor=E.+Sheauthor=T.+Gelbartauthor=J.+Truksaauthor=P.+Leeauthor=Y.+Xiaauthor=K.+Khovananthauthor=S.+Muddauthor=N.+Mannauthor=E.+M.+Y.+Morescoauthor=E.+Beutlerauthor=B.+Beutler&title=The+Serine+Protease+TMPRSS6+Is+Required+to+Sense+Iron+Deficiency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DX.%26aulast%3DShe%26aufirst%3DE.%26aulast%3DGelbart%26aufirst%3DT.%26aulast%3DTruksa%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DXia%26aufirst%3DY.%26aulast%3DKhovananth%26aufirst%3DK.%26aulast%3DMudd%26aufirst%3DS.%26aulast%3DMann%26aufirst%3DN.%26aulast%3DMoresco%26aufirst%3DE.%2BM.%2BY.%26aulast%3DBeutler%26aufirst%3DE.%26aulast%3DBeutler%26aufirst%3DB.%26atitle%3DThe%2520Serine%2520Protease%2520TMPRSS6%2520Is%2520Required%2520to%2520Sense%2520Iron%2520Deficiency%26jtitle%3DScience%26date%3D2008%26volume%3D320%26spage%3D1088%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Silvestri, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Domenico, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaplan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camaschella, C.</span><span> </span><span class="NLM_article-title">The Serine Protease Matriptase-2 (TMPRSS6) Inhibits Hepcidin Activation by Cleaving Membrane Hemojuvelin</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">502</span><span class="NLM_x">–</span> <span class="NLM_lpage">511</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1016%2Fj.cmet.2008.09.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=18976966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsV2qtbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=502-511&author=L.+Silvestriauthor=A.+Paganiauthor=A.+Naiauthor=I.+De+Domenicoauthor=J.+Kaplanauthor=C.+Camaschella&title=The+Serine+Protease+Matriptase-2+%28TMPRSS6%29+Inhibits+Hepcidin+Activation+by+Cleaving+Membrane+Hemojuvelin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The serine protease matriptase-2 (TMPRSS6) inhibits hepcidin activation by cleaving membrane hemojuvelin</span></div><div class="casAuthors">Silvestri, Laura; Pagani, Alessia; Nai, Antonella; De Domenico, Ivana; Kaplan, Jerry; Camaschella, Clara</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">502-511</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The liver peptide, hepcidin (I), regulates body Fe, is up-regulated in Fe overload and inflammation, and is down-regulated in Fe deficiency/hypoxia.  The transmembrane serine protease, matriptase-2, inhibits the I response and its mutational inactivation causes Fe-deficient anemia in mice and humans.  Here, the authors confirmed the inhibitory effect of matriptase-2 on I promoter; they showed that matriptase-2 lacking the serine protease domain, identified in the anemic Mask mouse (matriptase-2MASK), was fully inactive and that mutant R774C found in patients with genetic Fe deficiency had decreased inhibitory activity.  Matriptase-2 cleaved hemojuvelin (II), a regulator of I, on plasma membranes; matriptase-2MASK showed no cleavage activity and the human mutant showed only partial cleavage capacity.  Matriptase-2 interacted with II through the ectodomain since the interaction was conserved in matriptase-2MASK.  The expression of matriptase-2 mutants in zebrafish resulted in anemia, confirming the matriptase-2 role in Fe metab. and its interaction with II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgJ2IQkeHGILVg90H21EOLACvtfcHk0lhP1jpjcAMHWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsV2qtbnP&md5=c99d24bb2935383d62094942f1ce6289</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2008.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2008.09.012%26sid%3Dliteratum%253Aachs%26aulast%3DSilvestri%26aufirst%3DL.%26aulast%3DPagani%26aufirst%3DA.%26aulast%3DNai%26aufirst%3DA.%26aulast%3DDe%2BDomenico%26aufirst%3DI.%26aulast%3DKaplan%26aufirst%3DJ.%26aulast%3DCamaschella%26aufirst%3DC.%26atitle%3DThe%2520Serine%2520Protease%2520Matriptase-2%2520%2528TMPRSS6%2529%2520Inhibits%2520Hepcidin%2520Activation%2520by%2520Cleaving%2520Membrane%2520Hemojuvelin%26jtitle%3DCell%2520Metab.%26date%3D2008%26volume%3D8%26spage%3D502%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Silvestri, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillem, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toutain, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kannengiesser, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaumont, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camaschella, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandchamp, B.</span><span> </span><span class="NLM_article-title">Molecular Mechanisms of the Defective Hepcidin Inhibition in TMPRSS6 Mutations Associated with Iron-Refractory Iron Deficiency Anemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">5605</span><span class="NLM_x">–</span> <span class="NLM_lpage">5608</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=5605-5608&author=L.+Silvestriauthor=F.+Guillemauthor=A.+Paganiauthor=A.+Naiauthor=C.+Oudinauthor=M.+Silvaauthor=F.+Toutainauthor=C.+Kannengiesserauthor=C.+Beaumontauthor=C.+Camaschellaauthor=B.+Grandchamp&title=Molecular+Mechanisms+of+the+Defective+Hepcidin+Inhibition+in+TMPRSS6+Mutations+Associated+with+Iron-Refractory+Iron+Deficiency+Anemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSilvestri%26aufirst%3DL.%26aulast%3DGuillem%26aufirst%3DF.%26aulast%3DPagani%26aufirst%3DA.%26aulast%3DNai%26aufirst%3DA.%26aulast%3DOudin%26aufirst%3DC.%26aulast%3DSilva%26aufirst%3DM.%26aulast%3DToutain%26aufirst%3DF.%26aulast%3DKannengiesser%26aufirst%3DC.%26aulast%3DBeaumont%26aufirst%3DC.%26aulast%3DCamaschella%26aufirst%3DC.%26aulast%3DGrandchamp%26aufirst%3DB.%26atitle%3DMolecular%2520Mechanisms%2520of%2520the%2520Defective%2520Hepcidin%2520Inhibition%2520in%2520TMPRSS6%2520Mutations%2520Associated%2520with%2520Iron-Refractory%2520Iron%2520Deficiency%2520Anemia%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26spage%3D5605%26epage%3D5608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Finberg, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heeney, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campagna, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aydinok, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartman, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayo, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samuel, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strouse, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markianos, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fleming, M. D.</span><span> </span><span class="NLM_article-title">Mutations in TMPRSS6 Cause Iron-Refractory Iron Deficiency Anemia (IRIDA)</span> <span class="citation_source-journal">Nature Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">571</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1038%2Fng.130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=18408718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltFylu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2008&pages=569-571&author=K.+E.+Finbergauthor=M.+M.+Heeneyauthor=D.+R.+Campagnaauthor=Y.+Aydinokauthor=H.+A.+Pearsonauthor=K.+R.+Hartmanauthor=M.+M.+Mayoauthor=S.+M.+Samuelauthor=J.+J.+Strouseauthor=K.+Markianosauthor=N.+C.+Andrewsauthor=M.+D.+Fleming&title=Mutations+in+TMPRSS6+Cause+Iron-Refractory+Iron+Deficiency+Anemia+%28IRIDA%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in TMPRSS6 cause iron-refractory iron deficiency anemia (IRIDA)</span></div><div class="casAuthors">Finberg, Karin E.; Heeney, Matthew M.; Campagna, Dean R.; Aydinok, Yesim; Pearson, Howard A.; Hartman, Kip R.; Mayo, Mary M.; Samuel, Stewart M.; Strouse, John J.; Markianos, Kyriacos; Andrews, Nancy C.; Fleming, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">569-571</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Iron deficiency is usually attributed to chronic blood loss or inadequate dietary intake.  Here, we show that iron deficiency anemia refractory to oral iron therapy can be caused by germline mutations in TMPRSS6, which encodes a type II transmembrane serine protease produced by the liver that regulates the expression of the systemic iron regulatory hormone hepcidin.  These findings demonstrate that TMPRSS6 is essential for normal systemic iron homeostasis in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN5cCMWgwdt7Vg90H21EOLACvtfcHk0ljcX85qvwqBqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXltFylu7g%253D&md5=8fc8784914e1b566034d913fa9578da2</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fng.130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.130%26sid%3Dliteratum%253Aachs%26aulast%3DFinberg%26aufirst%3DK.%2BE.%26aulast%3DHeeney%26aufirst%3DM.%2BM.%26aulast%3DCampagna%26aufirst%3DD.%2BR.%26aulast%3DAydinok%26aufirst%3DY.%26aulast%3DPearson%26aufirst%3DH.%2BA.%26aulast%3DHartman%26aufirst%3DK.%2BR.%26aulast%3DMayo%26aufirst%3DM.%2BM.%26aulast%3DSamuel%26aufirst%3DS.%2BM.%26aulast%3DStrouse%26aufirst%3DJ.%2BJ.%26aulast%3DMarkianos%26aufirst%3DK.%26aulast%3DAndrews%26aufirst%3DN.%2BC.%26aulast%3DFleming%26aufirst%3DM.%2BD.%26atitle%3DMutations%2520in%2520TMPRSS6%2520Cause%2520Iron-Refractory%2520Iron%2520Deficiency%2520Anemia%2520%2528IRIDA%2529%26jtitle%3DNature%2520Genet.%26date%3D2008%26volume%3D40%26spage%3D569%26epage%3D571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Kew, M. C.</span><span> </span><span class="NLM_article-title">Hepatic Iron Overload and Hepatocellular Carcinoma</span> <span class="citation_source-journal">Liver Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=31-40&author=M.+C.+Kew&title=Hepatic+Iron+Overload+and+Hepatocellular+Carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKew%26aufirst%3DM.%2BC.%26atitle%3DHepatic%2520Iron%2520Overload%2520and%2520Hepatocellular%2520Carcinoma%26jtitle%3DLiver%2520Cancer%26date%3D2014%26volume%3D3%26spage%3D31%26epage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Riba, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rausa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorosina, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cittaro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia Manteiga, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinelli-Boneschi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stupka, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camaschella, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silvestri, L.</span><span> </span><span class="NLM_article-title">A Strong Anti-Inflammatory Signature Revealed by Liver Transcription Profiling of Tmprss6–/– Mice</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e69694</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e69694&author=M.+Ribaauthor=M.+Rausaauthor=M.+Sorosinaauthor=D.+Cittaroauthor=J.+M.+Garcia+Manteigaauthor=A.+Naiauthor=A.+Paganiauthor=F.+Martinelli-Boneschiauthor=E.+Stupkaauthor=C.+Camaschellaauthor=L.+Silvestri&title=A+Strong+Anti-Inflammatory+Signature+Revealed+by+Liver+Transcription+Profiling+of+Tmprss6%E2%80%93%2F%E2%80%93+Mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRiba%26aufirst%3DM.%26aulast%3DRausa%26aufirst%3DM.%26aulast%3DSorosina%26aufirst%3DM.%26aulast%3DCittaro%26aufirst%3DD.%26aulast%3DGarcia%2BManteiga%26aufirst%3DJ.%2BM.%26aulast%3DNai%26aufirst%3DA.%26aulast%3DPagani%26aufirst%3DA.%26aulast%3DMartinelli-Boneschi%26aufirst%3DF.%26aulast%3DStupka%26aufirst%3DE.%26aulast%3DCamaschella%26aufirst%3DC.%26aulast%3DSilvestri%26aufirst%3DL.%26atitle%3DA%2520Strong%2520Anti-Inflammatory%2520Signature%2520Revealed%2520by%2520Liver%2520Transcription%2520Profiling%2520of%2520Tmprss6%25E2%2580%2593%252F%25E2%2580%2593%2520Mice%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De69694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Maurer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gütschow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stirnberg, M.</span><span> </span><span class="NLM_article-title">Hepatocyte Growth Factor Activator Inhibitor Type 2 (HAI-2) Modulates Hepcidin Expression by Inhibiting the Cell Surface Protease Matriptase-2</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">450</span><span class="NLM_x">, </span> <span class="NLM_fpage">583</span><span class="NLM_x">–</span> <span class="NLM_lpage">593</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2013&pages=583-593&author=E.+Maurerauthor=M.+G%C3%BCtschowauthor=M.+Stirnberg&title=Hepatocyte+Growth+Factor+Activator+Inhibitor+Type+2+%28HAI-2%29+Modulates+Hepcidin+Expression+by+Inhibiting+the+Cell+Surface+Protease+Matriptase-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DE.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DStirnberg%26aufirst%3DM.%26atitle%3DHepatocyte%2520Growth%2520Factor%2520Activator%2520Inhibitor%2520Type%25202%2520%2528HAI-2%2529%2520Modulates%2520Hepcidin%2520Expression%2520by%2520Inhibiting%2520the%2520Cell%2520Surface%2520Protease%2520Matriptase-2%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D450%26spage%3D583%26epage%3D593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Galkin, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreno, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madison, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agus, D. B.</span><span> </span><span class="NLM_article-title">CVS-3983, a Selective Matriptase Inhibitor, Suppresses the Growth of Androgen Independent Prostate Tumor Xenografts</span> <span class="citation_source-journal">Prostate</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">228</span><span class="NLM_x">–</span> <span class="NLM_lpage">235</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2004&pages=228-235&author=A.+V.+Galkinauthor=L.+Mullenauthor=W.+D.+Foxauthor=J.+Brownauthor=D.+Duncanauthor=O.+Morenoauthor=E.+L.+Madisonauthor=D.+B.+Agus&title=CVS-3983%2C+a+Selective+Matriptase+Inhibitor%2C+Suppresses+the+Growth+of+Androgen+Independent+Prostate+Tumor+Xenografts"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGalkin%26aufirst%3DA.%2BV.%26aulast%3DMullen%26aufirst%3DL.%26aulast%3DFox%26aufirst%3DW.%2BD.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DDuncan%26aufirst%3DD.%26aulast%3DMoreno%26aufirst%3DO.%26aulast%3DMadison%26aufirst%3DE.%2BL.%26aulast%3DAgus%26aufirst%3DD.%2BB.%26atitle%3DCVS-3983%252C%2520a%2520Selective%2520Matriptase%2520Inhibitor%252C%2520Suppresses%2520the%2520Growth%2520of%2520Androgen%2520Independent%2520Prostate%2520Tumor%2520Xenografts%26jtitle%3DProstate%26date%3D2004%26volume%3D61%26spage%3D228%26epage%3D235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Stoop, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, C. S.</span><span> </span><span class="NLM_article-title">Engineering of a Macromolecular Scaffold to Develop Specific Protease Inhibitors</span> <span class="citation_source-journal">Nature Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1063</span><span class="NLM_x">–</span> <span class="NLM_lpage">1068</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1038%2Fnbt860" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=12923547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmslCrsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2003&pages=1063-1068&author=A.+A.+Stoopauthor=C.+S.+Craik&title=Engineering+of+a+Macromolecular+Scaffold+to+Develop+Specific+Protease+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering of a macromolecular scaffold to develop specific protease inhibitors</span></div><div class="casAuthors">Stoop, A. Allart; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1063-1068</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The specific inhibition of serine proteases, which are crucial switches in many physiol. important processes, is of value both for basic research and for therapeutic applications.  Ecotin, a potent macromol. inhibitor of serine proteases of the S1A family, presents an attractive scaffold to engineer specific protease inhibitors because of its large inhibitor-protease interface.  Using synthetic shuffling in combination with a restricted tetranomial diversity, we created ecotin libraries that are mutated at all 20 amino acid residues in the binding interface.  The efficacy of these libraries was demonstrated against the serine protease plasma kallikrein (Pkal).  Competitive phage display selection yielded a Pkal inhibitor with an apparent dissocn. equil. const. (Ki*) of 11 pM, whereas Ki* values for related proteases (such as Factor Xa (FXa), Factor XIa (FXIa), urokinase-type plasminogen activator (uPA), thrombin, and membrane-type serine protease 1 (MT-SP1)) were four to seven orders of magnitude higher.  The adaptability of the scaffold was demonstrated by the isolation of inhibitors to two addnl. serine proteases, MT-SP1/matriptase and Factor XIIa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMZvKZ8D3zgLVg90H21EOLACvtfcHk0li8wWyyHFIJ4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmslCrsr4%253D&md5=f267555e4282067173015d20ae38202a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnbt860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt860%26sid%3Dliteratum%253Aachs%26aulast%3DStoop%26aufirst%3DA.%2BA.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DEngineering%2520of%2520a%2520Macromolecular%2520Scaffold%2520to%2520Develop%2520Specific%2520Protease%2520Inhibitors%26jtitle%3DNature%2520Biotechnol.%26date%3D2003%26volume%3D21%26spage%3D1063%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longpré, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaulieu, M.-E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span> </span><span class="NLM_article-title">Inhibition of Human Matriptase by Eglin C Variants</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">580</span><span class="NLM_x">, </span> <span class="NLM_fpage">2227</span><span class="NLM_x">–</span> <span class="NLM_lpage">2232</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2006&pages=2227-2232&author=A.+D%C3%A9siletsauthor=J.-M.+Longpr%C3%A9author=M.-E.+Beaulieuauthor=R.+Leduc&title=Inhibition+of+Human+Matriptase+by+Eglin+C+Variants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DLongpr%25C3%25A9%26aufirst%3DJ.-M.%26aulast%3DBeaulieu%26aufirst%3DM.-E.%26aulast%3DLeduc%26aufirst%3DR.%26atitle%3DInhibition%2520of%2520Human%2520Matriptase%2520by%2520Eglin%2520C%2520Variants%26jtitle%3DFEBS%2520Lett.%26date%3D2006%26volume%3D580%26spage%3D2227%26epage%3D2232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Colombo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duchêne, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chagnon, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsault, E.</span><span> </span><span class="NLM_article-title">Design and Synthesis of Potent, Selective Inhibitors of Matriptase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">530</span><span class="NLM_x">–</span> <span class="NLM_lpage">534</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml3000534" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=530-534&author=E.+Colomboauthor=A.+D%C3%A9siletsauthor=D.+Duch%C3%AAneauthor=F.+Chagnonauthor=R.+J.+Najmanovichauthor=R.+Leducauthor=E.+Marsault&title=Design+and+Synthesis+of+Potent%2C+Selective+Inhibitors+of+Matriptase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fml3000534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3000534%26sid%3Dliteratum%253Aachs%26aulast%3DColombo%26aufirst%3DE.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DDuch%25C3%25AAne%26aufirst%3DD.%26aulast%3DChagnon%26aufirst%3DF.%26aulast%3DNajmanovich%26aufirst%3DR.%2BJ.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DMarsault%26aufirst%3DE.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Potent%252C%2520Selective%2520Inhibitors%2520of%2520Matriptase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D530%26epage%3D534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Quimbar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malik, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sommerhoff, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaas, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grundhuber, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craik, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, N. L.</span><span> </span><span class="NLM_article-title">High-Affinity Cyclic Peptide Matriptase Inhibitors</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">13885</span><span class="NLM_x">–</span> <span class="NLM_lpage">13896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1074%2Fjbc.M113.460030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23548907" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFWit70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=13885-13896&author=P.+Quimbarauthor=U.+Malikauthor=C.+P.+Sommerhoffauthor=Q.+Kaasauthor=L.+Y.+Chanauthor=Y.-H.+Huangauthor=M.+Grundhuberauthor=K.+Dunseauthor=D.+J.+Craikauthor=M.+A.+Andersonauthor=N.+L.+Daly&title=High-Affinity+Cyclic+Peptide+Matriptase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">High-affinity Cyclic Peptide Matriptase Inhibitors</span></div><div class="casAuthors">Quimbar, Pedro; Malik, Uru; Sommerhoff, Christian P.; Kaas, Quentin; Chan, Lai Y.; Huang, Yen-Hua; Grundhuber, Maresa; Dunse, Kerry; Craik, David J.; Anderson, Marilyn A.; Daly, Norelle L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">13885-13896</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The type II transmembrane serine protease matriptase is a key activator of multiple signaling pathways assocd. with cell proliferation and modification of the extracellular matrix.  Deregulated matriptase activity correlates with a no. of diseases, including cancer and hence highly selective matriptase inhibitors may have therapeutic potential.  The plant-derived cyclic peptide, sunflower trypsin inhibitor-1 (SFTI-1), is a promising drug scaffold with potent matriptase inhibitory activity.  In the current study we have analyzed the structure-activity relationships of SFTI-1 and Momordica cochinchinensis trypsin inhibitor-II (MCoTI-II), a structurally divergent trypsin inhibitor from Momordica cochinchinensis that also contains a cyclic backbone.  We show that MCoTI-II is a significantly more potent matriptase inhibitor than SFTI-1 and that all alanine mutants of both peptides, generated using positional scanning mutagenesis, have decreased trypsin affinity, whereas several mutations either maintain or result in enhanced matriptase inhibitory activity.  These intriguing results were used to design one of the most potent matriptase inhibitors known to date with a 290 pm equil. dissocn. const., and provide the first indication on how to modulate affinity for matriptase over trypsin in cyclic peptides.  This information might be useful for the design of more selective and therapeutically relevant inhibitors of matriptase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySN0T6sDSFrVg90H21EOLACvtfcHk0lgdZ0fwFAxMiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFWit70%253D&md5=4e537a35f93128a6120052e865d2c4a2</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.460030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.460030%26sid%3Dliteratum%253Aachs%26aulast%3DQuimbar%26aufirst%3DP.%26aulast%3DMalik%26aufirst%3DU.%26aulast%3DSommerhoff%26aufirst%3DC.%2BP.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DChan%26aufirst%3DL.%2BY.%26aulast%3DHuang%26aufirst%3DY.-H.%26aulast%3DGrundhuber%26aufirst%3DM.%26aulast%3DDunse%26aufirst%3DK.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DAnderson%26aufirst%3DM.%2BA.%26aulast%3DDaly%26aufirst%3DN.%2BL.%26atitle%3DHigh-Affinity%2520Cyclic%2520Peptide%2520Matriptase%2520Inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D13885%26epage%3D13896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Fittler, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avrutina, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glotzbach, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Empting, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolmar, H.</span><span> </span><span class="NLM_article-title">Combinatorial Tuning of Peptidic Drug Candidates: High-Affinity Matriptase Inhibitors Through Incremental Structure-Guided Optimization</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1848</span><span class="NLM_x">–</span> <span class="NLM_lpage">1857</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1039%2Fc3ob27469a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23361403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVChtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=1848-1857&author=H.+Fittlerauthor=O.+Avrutinaauthor=B.+Glotzbachauthor=M.+Emptingauthor=H.+Kolmar&title=Combinatorial+Tuning+of+Peptidic+Drug+Candidates%3A+High-Affinity+Matriptase+Inhibitors+Through+Incremental+Structure-Guided+Optimization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial tuning of peptidic drug candidates: high-affinity matriptase inhibitors through incremental structure-guided optimization</span></div><div class="casAuthors">Fittler, Heiko; Avrutina, Olga; Glotzbach, Bernhard; Empting, Martin; Kolmar, Harald</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1848-1857</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Herein we report a convenient strategy for the development of novel, highly-potent peptidic inhibitors of the trypsin-like serine protease matriptase based on the monocyclic variant of the sunflower trypsin inhibitor-1 (SFTI-1[1,14]).  We screened SFTI-1[1,14] variants possessing incremental modifications of the parent peptide for beneficial binding properties.  This compd. library comprising 6 peptides and 16 triazole-contg. peptidomimetics was established via structure-guided rational design and synthesized using a divergent strategy employing "copper-click" chem.  The most favorable amino acid substitutions were combined in one framework yielding potent SFTI-1-derived matriptase inhibitor-1 (SDMI-1) [H-Gly-Arg-c(Cys-Thr-Lys-Ser-Ile-Pro-Pro-Arg-Cys)-His-Pro-Asp-OH] and the truncated dodecapeptide variant (SDMI-2) [H-Gly-Arg-c(Cys-Thr-Lys-Ser-Ile-Pro-Pro-Arg-Cys)-His-NH2] with single-digit nanomolar inhibition consts.  In silico studies indicated that the improved matriptase affinity compared to the parent peptide is caused by the successful establishment of addnl. favorable proton donor-acceptor interactions between basic inhibitor side chains and acidic residues on the surface of the target enzyme.  SDMI-1 and 2 are potent inhibitors of the pharmaceutically relevant protease matriptase at a near physiol. pH and, thus, may find applications in therapy or diagnostics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHBE3Tv17SrVg90H21EOLACvtfcHk0lgdZ0fwFAxMiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVChtbo%253D&md5=fa964b6e8c7889abd9e9385cd95afeff</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2Fc3ob27469a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob27469a%26sid%3Dliteratum%253Aachs%26aulast%3DFittler%26aufirst%3DH.%26aulast%3DAvrutina%26aufirst%3DO.%26aulast%3DGlotzbach%26aufirst%3DB.%26aulast%3DEmpting%26aufirst%3DM.%26aulast%3DKolmar%26aufirst%3DH.%26atitle%3DCombinatorial%2520Tuning%2520of%2520Peptidic%2520Drug%2520Candidates%253A%2520High-Affinity%2520Matriptase%2520Inhibitors%2520Through%2520Incremental%2520Structure-Guided%2520Optimization%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D1848%26epage%3D1857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Goswami, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghadiyaram, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlfahrt, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistla, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagaraj, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satyam, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subbarao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palakurthy, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gopinath, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnamurthy, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikonen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moilanen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanya, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallio, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramachandra, M.</span><span> </span><span class="NLM_article-title">Structure-Guided Discovery of 1,3,5 Tri-Substituted Benzenes as Potent and Selective Matriptase Inhibitors Exhibiting in Vivo Antitumor Efficacy</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3187</span><span class="NLM_x">–</span> <span class="NLM_lpage">3203</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1016%2Fj.bmc.2014.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=24794746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1Gnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=3187-3203&author=R.+Goswamiauthor=S.+Mukherjeeauthor=C.+Ghadiyaramauthor=G.+Wohlfahrtauthor=R.+K.+Sistlaauthor=J.+Nagarajauthor=L.+K.+Satyamauthor=K.+Subbaraoauthor=R.+K.+Palakurthyauthor=S.+Gopinathauthor=N.+R.+Krishnamurthyauthor=T.+Ikonenauthor=A.+Moilanenauthor=H.+S.+Subramanyaauthor=P.+Kallioauthor=M.+Ramachandra&title=Structure-Guided+Discovery+of+1%2C3%2C5+Tri-Substituted+Benzenes+as+Potent+and+Selective+Matriptase+Inhibitors+Exhibiting+in+Vivo+Antitumor+Efficacy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided discovery of 1,3,5 tri-substituted benzenes as potent and selective matriptase inhibitors exhibiting in vivo antitumor efficacy</span></div><div class="casAuthors">Goswami, Rajeev; Mukherjee, Subhendu; Ghadiyaram, Chakshusmathi; Wohlfahrt, Gerd; Sistla, Ramesh K.; Nagaraj, Jwala; Satyam, Leena K.; Subbarao, Krishnaprasad; Palakurthy, Rajendra K.; Gopinath, Sreevalsam; Krishnamurthy, Narasimha R.; Ikonen, Tarja; Moilanen, Anu; Subramanya, Hosahalli S.; Kallio, Pekka; Ramachandra, Murali</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3187-3203</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Matriptase is a serine protease implicated in cancer invasion and metastasis.  Expression of matriptase is frequently dysregulated in human cancers and matriptase has been reported to activate latent growth factors such as hepatocyte growth factor/scatter factor, and proteases such as urokinase plasminogen activator suggesting that matriptase inhibitors could have therapeutic potential in treatment of cancer.  Here we report a structure-based approach which led to the discovery of selective and potent matriptase inhibitors with benzene as central core having 1,3,5 tri-substitution pattern.  X-ray crystallog. of one of the potent analogs in complex with matriptase revealed strong hydrogen bonding and salt-bridge interactions in the S1 pocket, as well as strong CH-π contacts between the P2/P4 cyclohexyl and Trp215 side-chain.  An addnl. interaction of the pendant amine at cyclohexyl with Gln175 side-chain results in substantial improvement in matriptase inhibition and selectivity against other related serine proteases.  Compds. I and II showed tumor growth inhibition in a s.c. DU-145 prostate cancer mouse model.  These compds. could be useful as tools to further explore the biol. of matriptase as a drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAfx9aewmxYLVg90H21EOLACvtfcHk0lgdZ0fwFAxMiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1Gnsbo%253D&md5=4c57601d086acd229ec05fb49b6775b1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DGoswami%26aufirst%3DR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DGhadiyaram%26aufirst%3DC.%26aulast%3DWohlfahrt%26aufirst%3DG.%26aulast%3DSistla%26aufirst%3DR.%2BK.%26aulast%3DNagaraj%26aufirst%3DJ.%26aulast%3DSatyam%26aufirst%3DL.%2BK.%26aulast%3DSubbarao%26aufirst%3DK.%26aulast%3DPalakurthy%26aufirst%3DR.%2BK.%26aulast%3DGopinath%26aufirst%3DS.%26aulast%3DKrishnamurthy%26aufirst%3DN.%2BR.%26aulast%3DIkonen%26aufirst%3DT.%26aulast%3DMoilanen%26aufirst%3DA.%26aulast%3DSubramanya%26aufirst%3DH.%2BS.%26aulast%3DKallio%26aufirst%3DP.%26aulast%3DRamachandra%26aufirst%3DM.%26atitle%3DStructure-Guided%2520Discovery%2520of%25201%252C3%252C5%2520Tri-Substituted%2520Benzenes%2520as%2520Potent%2520and%2520Selective%2520Matriptase%2520Inhibitors%2520Exhibiting%2520in%2520Vivo%2520Antitumor%2520Efficacy%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D3187%26epage%3D3203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Sisay, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinmetzer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stirnberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammami, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gütschow, M.</span><span> </span><span class="NLM_article-title">Identification of the First Low-Molecular-Weight Inhibitors of Matriptase-2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5523</span><span class="NLM_x">–</span> <span class="NLM_lpage">5535</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100183e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCis7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5523-5535&author=M.+T.+Sisayauthor=T.+Steinmetzerauthor=M.+Stirnbergauthor=E.+Maurerauthor=M.+Hammamiauthor=J.+Bajorathauthor=M.+G%C3%BCtschow&title=Identification+of+the+First+Low-Molecular-Weight+Inhibitors+of+Matriptase-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of the First Low-Molecular-Weight Inhibitors of Matriptase-2</span></div><div class="casAuthors">Sisay, Mihiret Tekeste; Steinmetzer, Torsten; Stirnberg, Marit; Maurer, Eva; Hammami, Maya; Bajorath, Juergen; Guetschow, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5523-5535</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As recently discovered, matriptase-2, a type II transmembrane serine protease, plays a crucial role in body iron homeostasis by down-regulating hepcidin expression, which results in increased iron levels.  Thus, matriptase-2 represents a novel target for the development of enzyme inhibitors potentially useful for the treatment of systemic iron overload (hemochromatosis).  A comparative three-dimensional model of the catalytic domain of matriptase-2 was generated and utilized for structure-based virtual screening in combination with similarity searching and knowledge-based compd. design.  Two N-protected dipeptide amides contg. a 4-amidinobenzylamide as P1 residue (compds. 1 and 3) were identified as the first small mol. inhibitors of matriptase-2 with Ki values of 170 and 460 nM, resp.  An inhibitor of the closely related protease matriptase (compd. 2, Ki = 220 nM), with more than 50-fold selectivity over matriptase-2, was also identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZuPBqKmx3SbVg90H21EOLACvtfcHk0lhvcYQWycjP6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCis7w%253D&md5=f4599613f7ca1b4356a420ecf0a15a3b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm100183e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100183e%26sid%3Dliteratum%253Aachs%26aulast%3DSisay%26aufirst%3DM.%2BT.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DStirnberg%26aufirst%3DM.%26aulast%3DMaurer%26aufirst%3DE.%26aulast%3DHammami%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520the%2520First%2520Low-Molecular-Weight%2520Inhibitors%2520of%2520Matriptase-2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5523%26epage%3D5535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Eswar, N.; Webb, B.; Marti-Renom, M. A.; Madhusudhan, M. S.; Eramian, D.; Shen, M.-Y.; Pieper, U.; Sali, A.</span><span> </span><span class="NLM_article-title">Comparative Protein Structure Modeling Using Modeller</span>.  <span class="citation_source-book">Current Protocols in Bioinformatics</span>; <span class="NLM_contrib-group">Baxevanis, A. D.</span>, Ed.: <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">, </span>Chapter 5, Unit 5.6–5.6.30.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1002%2F0471250953.bi0506s15" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=N.+Eswar&author=B.+Webb&author=M.+A.+Marti-Renom&author=M.+S.+Madhusudhan&author=D.+Eramian&author=M.-Y.+Shen&author=U.+Pieper&author=A.+Saliauthor=A.+D.+Baxevanis&title=Current+Protocols+in+Bioinformatics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2F0471250953.bi0506s15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471250953.bi0506s15%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26atitle%3DComparative%2520Protein%2520Structure%2520Modeling%2520Using%2520Modeller%26btitle%3DCurrent%2520Protocols%2520in%2520Bioinformatics%26aulast%3DBaxevanis%26aufirst%3DA.%2BD.%26pub%3DWiley%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Pronk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Páll, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjelkmar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apostolov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasson, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span> </span><span class="NLM_article-title">GROMACS 4.5: a High-Throughput and Highly Parallel Open Source Molecular Simulation Toolkit</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">845</span><span class="NLM_x">–</span> <span class="NLM_lpage">854</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1093%2Fbioinformatics%2Fbtt055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=23407358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=845-854&author=S.+Pronkauthor=S.+P%C3%A1llauthor=R.+Schulzauthor=P.+Larssonauthor=P.+Bjelkmarauthor=R.+Apostolovauthor=M.+R.+Shirtsauthor=J.+C.+Smithauthor=P.+M.+Kassonauthor=D.+van+der+Spoelauthor=B.+Hessauthor=E.+Lindahl&title=GROMACS+4.5%3A+a+High-Throughput+and+Highly+Parallel+Open+Source+Molecular+Simulation+Toolkit"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span></div><div class="casAuthors">Pronk, Sander; Pall, Szilard; Schulz, Roland; Larsson, Per; Bjelkmar, Paer; Apostolov, Rossen; Shirts, Michael R.; Smith, Jeremy C.; Kasson, Peter M.; van der Spoel, David; Hess, Berk; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">845-854</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Mol. simulation has historically been a low-throughput technique, but faster computers and increasing amts. of genomic and structural data are changing this by enabling large-scale automated simulation of, for instance, many conformers or mutants of biomols. with or without a range of ligands.  At the same time, advances in performance and scaling now make it possible to model complex biomol. interaction and function in a manner directly testable by expt.  These applications share a need for fast and efficient software that can be deployed on massive scale in clusters, web servers, distributed computing or cloud resources.  Results: Here, we present a range of new simulation algorithms and features developed during the past 4 years, leading up to the GROMACS 4.5 software package.  The software now automatically handles wide classes of biomols., such as proteins, nucleic acids and lipids, and comes with all commonly used force fields for these mols. built-in.  GROMACS supports several implicit solvent models, as well as new free-energy algorithms, and the software now uses multithreading for efficient parallelization even on low-end systems, including windows-based workstations.  Together with hand-tuned assembly kernels and state-of-the-art parallelization, this provides extremely high performance and cost efficiency for high-throughput as well as massively parallel simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71lQgYMm017Vg90H21EOLACvtfcHk0lhvcYQWycjP6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D&md5=4b25fd6ab4e33725ae56b5da63f4ad68</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt055%26sid%3Dliteratum%253Aachs%26aulast%3DPronk%26aufirst%3DS.%26aulast%3DP%25C3%25A1ll%26aufirst%3DS.%26aulast%3DSchulz%26aufirst%3DR.%26aulast%3DLarsson%26aufirst%3DP.%26aulast%3DBjelkmar%26aufirst%3DP.%26aulast%3DApostolov%26aufirst%3DR.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DKasson%26aufirst%3DP.%2BM.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGROMACS%25204.5%253A%2520a%2520High-Throughput%2520and%2520Highly%2520Parallel%2520Open%2520Source%2520Molecular%2520Simulation%2520Toolkit%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D845%26epage%3D854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Gaudreault, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chartier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R. J.</span><span> </span><span class="NLM_article-title">Side-Chain Rotamer Changes Upon Ligand Binding: Common, Crucial, Correlate with Entropy and Rearrange Hydrogen Bonding</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">i423</span><span class="NLM_x">–</span> <span class="NLM_lpage">i430</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1093%2Fbioinformatics%2Fbts395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=22962462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlantLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=i423-i430&author=F.+Gaudreaultauthor=M.+Chartierauthor=R.+J.+Najmanovich&title=Side-Chain+Rotamer+Changes+Upon+Ligand+Binding%3A+Common%2C+Crucial%2C+Correlate+with+Entropy+and+Rearrange+Hydrogen+Bonding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Side-chain rotamer changes upon ligand binding: Common, crucial, correlate with entropy and rearrange hydrogen bonding</span></div><div class="casAuthors">Gaudreault, Francis; Chartier, Matthieu; Najmanovich, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">i423-i430</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Protein movements form a continuum from large domain rearrangements (including folding and restructuring) to side-chain rotamer changes and small rearrangements.  Understanding side-chain flexibility upon binding is important for understanding mol. recognition events and predicting ligand binding.  Methods: In the present work, we developed a well-curated non-redundant dataset of 188 proteins in pairs of structures in the Apo (unbound) and Holo (bound) forms to study the extent and the factors that guide side-chain rotamer changes upon binding.  Results: Our anal. shows that side-chain rotamer changes are widespread with only 10% of binding sites displaying no conformational changes.  Overall, at most five rotamer changes account for the obsd. movements in 90% of the cases.  Furthermore, rotamer changes are essential in 32% of flexible binding sites.  The different amino acids have a 11-fold difference in their probability to undergo changes.  Side-chain flexibility represents an intrinsic property of amino acids as it correlates well with configurational entropy differences.  Furthermore, on av. b-factors and solvent accessible surface areas can discriminate flexible side-chains in the Apo form.  Finally, there is a rearrangement of the hydrogen-bonding network upon binding primarily with a loss of H-bonds with water mols. and a gain of H-bonds with protein residues for flexible residues.  Interestingly, only 25% of side chains capable of forming H-bonds do so with the ligand upon binding.  In terms of drug design, this last result shows that there is a large no. of potential interactions that may be exploited to modulate the specificity and sensitivity of inhibitors.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="0775666166626b2969666d6a666968716e646f47746f6275657568686c6229646629">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoerwRd3_qiYrVg90H21EOLACvtfcHk0ljKa4UxBHwXIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlantLfO&md5=1cb775e36034683efd17908930e4639d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts395&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts395%26sid%3Dliteratum%253Aachs%26aulast%3DGaudreault%26aufirst%3DF.%26aulast%3DChartier%26aufirst%3DM.%26aulast%3DNajmanovich%26aufirst%3DR.%2BJ.%26atitle%3DSide-Chain%2520Rotamer%2520Changes%2520Upon%2520Ligand%2520Binding%253A%2520Common%252C%2520Crucial%252C%2520Correlate%2520with%2520Entropy%2520and%2520Rearrange%2520Hydrogen%2520Bonding%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3Di423%26epage%3Di430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Najmanovich, R. J.</span><span> </span><span class="NLM_article-title">Side Chain Flexibility Upon Ligand Binding: Docking Predictions and Statistical Analysis</span>. Ph.D. Thesis. <span class="NLM_publisher-name">Feinberg Graduate School, Weizmann Institute of Sciences</span><i>arXiv</i>:<i>13014564</i>,<span class="NLM_x"> </span><span class="NLM_year">,2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=%2C2004&author=R.+J.+Najmanovich&title=Side+Chain+Flexibility+Upon+Ligand+Binding%3A+Docking+Predictions+and+Statistical+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNajmanovich%26aufirst%3DR.%2BJ.%26atitle%3DSide%2520Chain%2520Flexibility%2520Upon%2520Ligand%2520Binding%253A%2520Docking%2520Predictions%2520and%2520Statistical%2520Analysis%26pub%3DFeinberg%2520Graduate%2520School%252C%2520Weizmann%2520Institute%2520of%2520Sciences%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Béliveau, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Désilets, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span> </span><span class="NLM_article-title">Probing the Substrate Specificities of Matriptase, Matriptase-2, Hepsin and DESC1 with Internally Quenched Fluorescent Peptides</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">2213</span><span class="NLM_x">–</span> <span class="NLM_lpage">2226</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2009&pages=2213-2226&author=F.+B%C3%A9liveauauthor=A.+D%C3%A9siletsauthor=R.+Leduc&title=Probing+the+Substrate+Specificities+of+Matriptase%2C+Matriptase-2%2C+Hepsin+and+DESC1+with+Internally+Quenched+Fluorescent+Peptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25A9liveau%26aufirst%3DF.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DLeduc%26aufirst%3DR.%26atitle%3DProbing%2520the%2520Substrate%2520Specificities%2520of%2520Matriptase%252C%2520Matriptase-2%252C%2520Hepsin%2520and%2520DESC1%2520with%2520Internally%2520Quenched%2520Fluorescent%2520Peptides%26jtitle%3DFEBS%2520J.%26date%3D2009%26volume%3D276%26spage%3D2213%26epage%3D2226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Wysocka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruba, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miecznikowska, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popow-Stellmaszyk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gütschow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stirnberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furtmann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolka, K.</span><span> </span><span class="NLM_article-title">Substrate Specificity of Human Matriptase-2</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">121</span><span class="NLM_x">–</span> <span class="NLM_lpage">127</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2014&pages=121-127&author=M.+Wysockaauthor=N.+Grubaauthor=A.+Miecznikowskaauthor=J.+Popow-Stellmaszykauthor=M.+G%C3%BCtschowauthor=M.+Stirnbergauthor=N.+Furtmannauthor=J.+Bajorathauthor=A.+Lesnerauthor=K.+Rolka&title=Substrate+Specificity+of+Human+Matriptase-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWysocka%26aufirst%3DM.%26aulast%3DGruba%26aufirst%3DN.%26aulast%3DMiecznikowska%26aufirst%3DA.%26aulast%3DPopow-Stellmaszyk%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DStirnberg%26aufirst%3DM.%26aulast%3DFurtmann%26aufirst%3DN.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DLesner%26aufirst%3DA.%26aulast%3DRolka%26aufirst%3DK.%26atitle%3DSubstrate%2520Specificity%2520of%2520Human%2520Matriptase-2%26jtitle%3DBiochimie%26date%3D2014%26volume%3D97%26spage%3D121%26epage%3D127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Gaudreault, F.; Najmanovich, R. J.</span><span> </span><span class="NLM_article-title">FlexAID: Revisiting Docking on Non Native-Complex Structures</span>. Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Gaudreault%2C+F.%3B+Najmanovich%2C+R.+J.+FlexAID%3A+Revisiting+Docking+on+Non+Native-Complex+Structures.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DGaudreault%26aufirst%3DF.%26atitle%3DFlexAID%253A%2520Revisiting%2520Docking%2520on%2520Non%2520Native-Complex%2520Structures" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Hou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">Assessing the Performance of the MM/PBSA and MM/GBSA Methods. 1. The Accuracy of Binding Free Energy Calculations Based on Molecular Dynamics Simulations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100275a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=69-82&author=T.+Houauthor=J.+Wangauthor=Y.+Liauthor=W.+Wang&title=Assessing+the+Performance+of+the+MM%2FPBSA+and+MM%2FGBSA+Methods.+1.+The+Accuracy+of+Binding+Free+Energy+Calculations+Based+on+Molecular+Dynamics+Simulations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the Performance of the MM/PBSA and MM/GBSA Methods. 1. The Accuracy of Binding Free Energy Calculations Based on Molecular Dynamics Simulations</span></div><div class="casAuthors">Hou, Tingjun; Wang, Junmei; Li, Youyong; Wang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-82</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Mol. Mechanics/Poisson-Boltzmann Surface Area (MM/PBSA) and the Mol. Mechanics/Generalized Born Surface Area (MM/GBSA) methods calc. binding free energies for macromols. by combining mol. mechanics calcns. and continuum solvation models.  To systematically evaluate the performance of these methods, the authors report here an extensive study of 59 ligands interacting with six different proteins.  First, the authors explored the effects of the length of the mol. dynamics (MD) simulation, ranging from 400 to 4800 ps, and the solute dielec. const. (1, 2, or 4) on the binding free energies predicted by MM/PBSA.  The following three important conclusions could be obsd.: (1) MD simulation length has an obvious impact on the predictions, and longer MD simulation is not always necessary to achieve better predictions.  (2) The predictions are quite sensitive to the solute dielec. const., and this parameter should be carefully detd. according to the characteristics of the protein/ligand binding interface.  (3) Conformational entropy often show large fluctuations in MD trajectories, and a large no. of snapshots are necessary to achieve stable predictions.  Next, the authors evaluated the accuracy of the binding free energies calcd. by three Generalized Born (GB) models.  The authors found that the GB model developed by Onufriev and Case was the most successful model in ranking the binding affinities of the studied inhibitors.  Finally, the authors evaluated the performance of MM/GBSA and MM/PBSA in predicting binding free energies.  The authors' results showed that MM/PBSA performed better in calcg. abs., but not necessarily relative, binding free energies than MM/GBSA.  Considering its computational efficiency, MM/GBSA can serve as a powerful tool in drug design, where correct ranking of inhibitors is often emphasized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_kxNWs2s3grVg90H21EOLACvtfcHk0ljKa4UxBHwXIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyrsrrK&md5=b9f5b7fab70a6cc76b115d202d718f87</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci100275a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100275a%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DAssessing%2520the%2520Performance%2520of%2520the%2520MM%252FPBSA%2520and%2520MM%252FGBSA%2520Methods.%25201.%2520The%2520Accuracy%2520of%2520Binding%2520Free%2520Energy%2520Calculations%2520Based%2520on%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D69%26epage%3D82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Hou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">Assessing the Performance of the Molecular Mechanics/Poisson Boltzmann Surface Area and Molecular Mechanics/Generalized Born Surface Area Methods. II. The Accuracy of Ranking Poses Generated From Docking</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">866</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1002%2Fjcc.21666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=20949517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BC3MXitleisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=866-877&author=T.+Houauthor=J.+Wangauthor=Y.+Liauthor=W.+Wang&title=Assessing+the+Performance+of+the+Molecular+Mechanics%2FPoisson+Boltzmann+Surface+Area+and+Molecular+Mechanics%2FGeneralized+Born+Surface+Area+Methods.+II.+The+Accuracy+of+Ranking+Poses+Generated+From+Docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing the performance of the molecular mechanics/Poisson Boltzmann surface area and molecular mechanics/generalized Born surface area methods. II: The accuracy of ranking poses generated from docking</span></div><div class="casAuthors">Hou, Ting-Jun; Wang, Jun-Mei; Li, You-Yong; Wang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">866-877</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In mol. docking, it is challenging to develop a scoring function that is accurate to conduct high-throughput screenings.  Most scoring functions implemented in popular docking software packages were developed with many approxns. for computational efficiency, which sacrifices the accuracy of prediction.  With advanced technol. and powerful computational hardware nowadays, it is feasible to use rigorous scoring functions, such as mol. mechanics/Poisson Boltzmann surface area (MM/PBSA) and mol. mechanics/generalized Born surface area (MM/GBSA) in mol. docking studies.  Here, the authors systematically investigated the performance of MM/PBSA and MM/GBSA to identify the correct binding conformations and predict the binding free energies for 98 protein-ligand complexes.  Comparison studies showed that MM/GBSA (69.4%) outperformed MM/PBSA (45.5%) and many popular scoring functions to identify the correct binding conformations.  Moreover, the authors found that mol. dynamics simulations are necessary for some systems to identify the correct binding conformations.  Based on the authors' results, the authors proposed the guideline for MM/GBSA to predict the binding conformations.  The authors then tested the performance of MM/GBSA and MM/PBSA to reproduce the binding free energies of the 98 protein-ligand complexes.  The best prediction of MM/GBSA model with internal dielec. const. 2.0, produced a Spearman's correlation coeff. of 0.66, which is better than MM/PBSA (0.49) and almost all scoring functions used in mol. docking.  In summary, MM/GBSA performs well for both binding pose predictions and binding free-energy estns. and is efficient to re-score the top-hit poses produced by other less-accurate scoring functions.  © 2010 Wiley Periodicals, Inc.  J Comput Chem, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6V0ra4bAVq7Vg90H21EOLACvtfcHk0lgnsYznrOzktQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXitleisrw%253D&md5=fd42a4da52a20e7dfcbfff62e746513e</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21666%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DAssessing%2520the%2520Performance%2520of%2520the%2520Molecular%2520Mechanics%252FPoisson%2520Boltzmann%2520Surface%2520Area%2520and%2520Molecular%2520Mechanics%252FGeneralized%2520Born%2520Surface%2520Area%2520Methods.%2520II.%2520The%2520Accuracy%2520of%2520Ranking%2520Poses%2520Generated%2520From%2520Docking%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D866%26epage%3D877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Maurer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sisay, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stirnberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinmetzer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gütschow, M.</span><span> </span><span class="NLM_article-title">Insights Into Matriptase-2 Substrate Binding and Inhibition Mechanisms by Analyzing Active-Site-Mutated Variants</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">68</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=68-72&author=E.+Maurerauthor=M.+T.+Sisayauthor=M.+Stirnbergauthor=T.+Steinmetzerauthor=J.+Bajorathauthor=M.+G%C3%BCtschow&title=Insights+Into+Matriptase-2+Substrate+Binding+and+Inhibition+Mechanisms+by+Analyzing+Active-Site-Mutated+Variants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DE.%26aulast%3DSisay%26aufirst%3DM.%2BT.%26aulast%3DStirnberg%26aufirst%3DM.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26atitle%3DInsights%2520Into%2520Matriptase-2%2520Substrate%2520Binding%2520and%2520Inhibition%2520Mechanisms%2520by%2520Analyzing%2520Active-Site-Mutated%2520Variants%26jtitle%3DChemMedChem%26date%3D2012%26volume%3D7%26spage%3D68%26epage%3D72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Shannon, C. E.</span><span> </span><span class="NLM_article-title">A Mathematical Theory of Communication</span> <span class="citation_source-journal">Bell System Tech. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1948</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">423</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1002%2Fj.1538-7305.1948.tb01338.x" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1948&pages=379-423&author=C.+E.+Shannon&title=A+Mathematical+Theory+of+Communication"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fj.1538-7305.1948.tb01338.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1538-7305.1948.tb01338.x%26sid%3Dliteratum%253Aachs%26aulast%3DShannon%26aufirst%3DC.%2BE.%26atitle%3DA%2520Mathematical%2520Theory%2520of%2520Communication%26jtitle%3DBell%2520System%2520Tech.%2520J.%26date%3D1948%26volume%3D27%26spage%3D379%26epage%3D423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Fuchs, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Grafenstein, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margreiter, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzer, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallnoefer, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liedl, K. R.</span><span> </span><span class="NLM_article-title">Cleavage Entropy as Quantitative Measure of Protease Specificity</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003007</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003007&author=J.+E.+Fuchsauthor=S.+von+Grafensteinauthor=R.+G.+Huberauthor=M.+A.+Margreiterauthor=G.+M.+Spitzerauthor=H.+G.+Wallnoeferauthor=K.+R.+Liedl&title=Cleavage+Entropy+as+Quantitative+Measure+of+Protease+Specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3Dvon%2BGrafenstein%26aufirst%3DS.%26aulast%3DHuber%26aufirst%3DR.%2BG.%26aulast%3DMargreiter%26aufirst%3DM.%2BA.%26aulast%3DSpitzer%26aufirst%3DG.%2BM.%26aulast%3DWallnoefer%26aufirst%3DH.%2BG.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DCleavage%2520Entropy%2520as%2520Quantitative%2520Measure%2520of%2520Protease%2520Specificity%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3De1003007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Barré, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dufour, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckhard, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kappelhoff, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Béliveau, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leduc, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overall, C. M.</span><span> </span><span class="NLM_article-title">Cleavage Specificity Analysis of Six Type II Transmembrane Serine Proteases (TTSPs) Using PICS with Proteome-Derived Peptide Libraries</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e105984</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e105984&author=O.+Barr%C3%A9author=A.+Dufourauthor=U.+Eckhardauthor=R.+Kappelhoffauthor=F.+B%C3%A9liveauauthor=R.+Leducauthor=C.+M.+Overall&title=Cleavage+Specificity+Analysis+of+Six+Type+II+Transmembrane+Serine+Proteases+%28TTSPs%29+Using+PICS+with+Proteome-Derived+Peptide+Libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBarr%25C3%25A9%26aufirst%3DO.%26aulast%3DDufour%26aufirst%3DA.%26aulast%3DEckhard%26aufirst%3DU.%26aulast%3DKappelhoff%26aufirst%3DR.%26aulast%3DB%25C3%25A9liveau%26aufirst%3DF.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DOverall%26aufirst%3DC.%2BM.%26atitle%3DCleavage%2520Specificity%2520Analysis%2520of%2520Six%2520Type%2520II%2520Transmembrane%2520Serine%2520Proteases%2520%2528TTSPs%2529%2520Using%2520PICS%2520with%2520Proteome-Derived%2520Peptide%2520Libraries%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De105984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Huang, S.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grinter, S. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, X.</span><span> </span><span class="NLM_article-title">Scoring Functions and Their Evaluation Methods for Protein–Ligand Docking: Recent Advances and Future Directions</span> <span class="citation_source-journal">Phys. Chem. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">12899</span><span class="NLM_x">–</span> <span class="NLM_lpage">12908</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=12899-12908&author=S.-Y.+Huangauthor=S.+Z.+Grinterauthor=X.+Zou&title=Scoring+Functions+and+Their+Evaluation+Methods+for+Protein%E2%80%93Ligand+Docking%3A+Recent+Advances+and+Future+Directions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.-Y.%26aulast%3DGrinter%26aufirst%3DS.%2BZ.%26aulast%3DZou%26aufirst%3DX.%26atitle%3DScoring%2520Functions%2520and%2520Their%2520Evaluation%2520Methods%2520for%2520Protein%25E2%2580%2593Ligand%2520Docking%253A%2520Recent%2520Advances%2520and%2520Future%2520Directions%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D12899%26epage%3D12908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">O’Boyle, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeersch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, G. R.</span><span> </span><span class="NLM_article-title">Open Babel: an Open Chemical Toolbox</span> <span class="citation_source-journal">J. Cheminform.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=21982300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A280%3ADC%252BC3Mbjt1elsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+an+Open+Chemical+Toolbox"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Open Babel: An open chemical toolbox</span></div><div class="casAuthors">O'Boyle Noel M; Banck Michael; James Craig A; Morley Chris; Vandermeersch Tim; Hutchison Geoffrey R</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cheminformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  A frequent problem in computational modeling is the interconversion of chemical structures between different formats.  While standard interchange formats exist (for example, Chemical Markup Language) and de facto standards have arisen (for example, SMILES format), the need to interconvert formats is a continuing problem due to the multitude of different application areas for chemistry data, differences in the data stored by different formats (0D versus 3D, for example), and competition between software along with a lack of vendor-neutral formats.  RESULTS:  We discuss, for the first time, Open Babel, an open-source chemical toolbox that speaks the many languages of chemical data.  Open Babel version 2.3 interconverts over 110 formats.  The need to represent such a wide variety of chemical and molecular data requires a library that implements a wide range of cheminformatics algorithms, from partial charge assignment and aromaticity detection, to bond order perception and canonicalization.  We detail the implementation of Open Babel, describe key advances in the 2.3 release, and outline a variety of uses both in terms of software products and scientific research, including applications far beyond simple format interconversion.  CONCLUSIONS:  Open Babel presents a solution to the proliferation of multiple chemical file formats.  In addition, it provides a variety of useful utilities from conformer searching and 2D depiction, to filtering, batch conversion, and substructure and similarity searching.  For developers, it can be used as a programming library to handle chemical data in areas such as organic chemistry, drug design, materials science, and computational chemistry.  It is freely available under an open-source license from http://openbabel.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAiR-F6gwEoGLFM_lbR1VmfW6udTcc2ea416Tzp-jS6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mbjt1elsQ%253D%253D&md5=b3c770a7e1bacab8afe04f91da9c2c1e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520an%2520Open%2520Chemical%2520Toolbox%26jtitle%3DJ.%2520Cheminform.%26date%3D2011%26volume%3D3%26spage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucl. Acids. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lj_3sn4pEySjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucl.%2520Acids.%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Shen, M.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Statistical Potential for Assessment and Prediction of Protein Structures</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">2507</span><span class="NLM_x">–</span> <span class="NLM_lpage">2524</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1110%2Fps.062416606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=17075131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFygsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2006&pages=2507-2524&author=M.-Y.+Shenauthor=A.+Sali&title=Statistical+Potential+for+Assessment+and+Prediction+of+Protein+Structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical potential for assessment and prediction of protein structures</span></div><div class="casAuthors">Shen, Min-Yi; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2507-2524</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Protein structures in the Protein Data Bank provide a wealth of data about the interactions that det. the native states of proteins.  Using the probability theory, the authors derive an at. distance-dependent statistical potential from a sample of native structures that does not depend on any adjustable parameters (Discrete Optimized Protein Energy, or DOPE).  DOPE is based on an improved ref. state that corresponds to noninteracting atoms in a homogeneous sphere with the radius dependent on a sample native structure; it thus accounts for the finite and spherical shape of the native structures.  The DOPE potential was extd. from a nonredundant set of 1472 crystallog. structures.  The authors tested DOPE and five other scoring functions by the detection of the native state among six multiple target decoy sets, the correlation between the score and model error, and the identification of the most accurate nonnative structure in the decoy set.  For all decoy sets, DOPE is the best performing function in terms of all criteria, except for a tie in one criterion for one decoy set.  To facilitate its use in various applications, such as model assessment, loop modeling, and fitting into cryo-electron microscopy mass d. maps combined with comparative protein structure modeling, DOPE was incorporated into the modeling package MODELLER-8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz7bz4vq-p_7Vg90H21EOLACvtfcHk0lj_3sn4pEySjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFygsrfO&md5=f4e7ef56cd100164e347103a6888ee8e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1110%2Fps.062416606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.062416606%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DM.-Y.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStatistical%2520Potential%2520for%2520Assessment%2520and%2520Prediction%2520of%2520Protein%2520Structures%26jtitle%3DProtein%2520Sci.%26date%3D2006%26volume%3D15%26spage%3D2507%26epage%3D2524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">London, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibold, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cimermančič, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Covalent Docking of Large Libraries for the Discovery of Chemical Probes</span> <span class="citation_source-journal">Nature Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1066</span><span class="NLM_x">–</span> <span class="NLM_lpage">1072</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1038%2Fnchembio.1666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=25344815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A280%3ADC%252BC2M3jt1Wmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1066-1072&author=N.+Londonauthor=R.+M.+Millerauthor=S.+Krishnanauthor=K.+Uchidaauthor=J.+J.+Irwinauthor=O.+Eidamauthor=L.+Giboldauthor=P.+Cimerman%C4%8Di%C4%8Dauthor=R.+Bonnetauthor=B.+K.+Shoichetauthor=J.+Taunton&title=Covalent+Docking+of+Large+Libraries+for+the+Discovery+of+Chemical+Probes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking of large libraries for the discovery of chemical probes</span></div><div class="casAuthors">London Nir; Eidam Oliv; Miller Rand M; Krishnan Shyam; Uchida Kenji; Taunton Jack; Irwin John J; Shoichet Brian K; Gibold Lucie; Cimermancic Peter; Bonnet Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature chemical biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1066-72</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chemical probes that form a covalent bond with a protein target often show enhanced selectivity, potency and utility for biological studies.  Despite these advantages, protein-reactive compounds are usually avoided in high-throughput screening campaigns.  Here we describe a general method (DOCKovalent) for screening large virtual libraries of electrophilic small molecules.  We apply this method prospectively to discover reversible covalent fragments that target distinct protein nucleophiles, including the catalytic serine of AmpC β-lactamase and noncatalytic cysteines in RSK2, MSK1 and JAK3 kinases.  We identify submicromolar to low-nanomolar hits with high ligand efficiency, cellular activity and selectivity, including what are to our knowledge the first reported reversible covalent inhibitors of JAK3.  Crystal structures of inhibitor complexes with AmpC and RSK2 confirm the docking predictions and guide further optimization.  As covalent virtual screening may have broad utility for the rapid discovery of chemical probes, we have made the method freely available through an automated web server (http://covalent.docking.org/).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREZM0Uo7YPEWK_SFaaT9INfW6udTcc2eYXobUkDoHctrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3jt1Wmuw%253D%253D&md5=721636ba69de480e84a31c2b0679d382</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1666%26sid%3Dliteratum%253Aachs%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DGibold%26aufirst%3DL.%26aulast%3DCimerman%25C4%258Di%25C4%258D%26aufirst%3DP.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DCovalent%2520Docking%2520of%2520Large%2520Libraries%2520for%2520the%2520Discovery%2520of%2520Chemical%2520Probes%26jtitle%3DNature%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1066%26epage%3D1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Anderson, D. P.</span><span> </span><span class="NLM_article-title">BOINC: a System for Public-Resource Computing and Storage. </span>. In <span class="NLM_conf-name">Fifth IEEE/ACM International Workshop on Grid Computing</span>, <span class="NLM_conf-loc">Pittsburgh, PA</span>, <span class="NLM_conf-date">November 8, 2004</span>; <span class="NLM_publisher-name">IEEE</span>: <span class="NLM_publisher-loc">Los Angeles</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=4-10&conference=Fifth+IEEE%2FACM+International+Workshop+on+Grid+Computing&author=D.+P.+Anderson&title=BOINC%3A+a+System+for+Public-Resource+Computing+and+Storage.+"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DD.%2BP.%26atitle%3DBOINC%253A%2520a%2520System%2520for%2520Public-Resource%2520Computing%2520and%2520Storage.%2520%26pub%3DIEEE%26date%3D2004%26spage%3D4%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Van Der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">GROMACS: Fast, Flexible, and Free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+C.+Berendsen&title=GROMACS%3A+Fast%2C+Flexible%2C+and+Free"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0liQthk2Jwf5pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DGROMACS%253A%2520Fast%252C%2520Flexible%252C%2520and%2520Free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Sousa da Silva, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vranken, W. F.</span><span> </span><span class="NLM_article-title">ACPYPE—AnteChamber PYthon Parser interfacE</span> <span class="citation_source-journal">BMC Res. Notes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">367</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1186%2F1756-0500-5-367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A280%3ADC%252BC38fitlWjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=367&author=A.+W.+Sousa+da+Silvaauthor=W.+F.+Vranken&title=ACPYPE%E2%80%94AnteChamber+PYthon+Parser+interfacE"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">ACPYPE - AnteChamber PYthon Parser interfacE</span></div><div class="casAuthors">Sousa da Silva Alan W; Vranken Wim F</div><div class="citationInfo"><span class="NLM_cas:title">BMC research notes</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">367</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  ACPYPE (or AnteChamber PYthon Parser interfacE) is a wrapper script around the ANTECHAMBER software that simplifies the generation of small molecule topologies and parameters for a variety of molecular dynamics programmes like GROMACS, CHARMM and CNS.  It is written in the Python programming language and was developed as a tool for interfacing with other Python based applications such as the CCPN software suite (for NMR data analysis) and ARIA (for structure calculations from NMR data).  ACPYPE is open source code, under GNU GPL v3, and is available as a stand-alone application at http://www.ccpn.ac.uk/acpype and as a web portal application at http://webapps.ccpn.ac.uk/acpype.  FINDINGS:  We verified the topologies generated by ACPYPE in three ways: by comparing with default AMBER topologies for standard amino acids; by generating and verifying topologies for a large set of ligands from the PDB; and by recalculating the structures for 5 protein-ligand complexes from the PDB.  CONCLUSIONS:  ACPYPE is a tool that simplifies the automatic generation of topology and parameters in different formats for different molecular mechanics programmes, including calculation of partial charges, while being object oriented for integration with other applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbj1v3HcWoNJZoeNcnHOQ9fW6udTcc2eYXobUkDoHctrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fitlWjtg%253D%253D&md5=60f750ba98c975374b351a39a1fa0ec4</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2F1756-0500-5-367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-0500-5-367%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%2Bda%2BSilva%26aufirst%3DA.%2BW.%26aulast%3DVranken%26aufirst%3DW.%2BF.%26atitle%3DACPYPE%25E2%2580%2594AnteChamber%2520PYthon%2520Parser%2520interfacE%26jtitle%3DBMC%2520Res.%2520Notes%26date%3D2012%26volume%3D5%26spage%3D367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsili, M.</span><span> </span><span class="NLM_article-title">Iterative Partial Equalization of Orbital Electronegativity—A Rapid Access to Atomic Charges</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">3219</span><span class="NLM_x">–</span> <span class="NLM_lpage">3228</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1980&pages=3219-3228&author=J.+Gasteigerauthor=M.+Marsili&title=Iterative+Partial+Equalization+of+Orbital+Electronegativity%E2%80%94A+Rapid+Access+to+Atomic+Charges"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DMarsili%26aufirst%3DM.%26atitle%3DIterative%2520Partial%2520Equalization%2520of%2520Orbital%2520Electronegativity%25E2%2580%2594A%2520Rapid%2520Access%2520to%2520Atomic%2520Charges%26jtitle%3DTetrahedron%26date%3D1980%26volume%3D36%26spage%3D3219%26epage%3D3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Costanzo, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almond, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hecker, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schott, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabut, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrade-Gordon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corcoran, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giardino, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauffman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Garavilla, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haertlein, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maryanoff, B. E.</span><span> </span><span class="NLM_article-title">In-Depth Study of Tripeptide-Based Alpha-Ketoheterocycles as Inhibitors of Thrombin. Effective Utilization of the S1′ Subsite and Its Implications to Structure-Based Drug Design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1984</span><span class="NLM_x">–</span> <span class="NLM_lpage">2008</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0303857" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1984-2008&author=M.+J.+Costanzoauthor=H.+R.+Almondauthor=L.+R.+Heckerauthor=M.+R.+Schottauthor=S.+C.+Yabutauthor=H.-C.+Zhangauthor=P.+Andrade-Gordonauthor=T.+W.+Corcoranauthor=E.+C.+Giardinoauthor=J.+A.+Kauffmanauthor=J.+M.+Lewisauthor=L.+de+Garavillaauthor=B.+J.+Haertleinauthor=B.+E.+Maryanoff&title=In-Depth+Study+of+Tripeptide-Based+Alpha-Ketoheterocycles+as+Inhibitors+of+Thrombin.+Effective+Utilization+of+the+S1%E2%80%B2+Subsite+and+Its+Implications+to+Structure-Based+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm0303857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303857%26sid%3Dliteratum%253Aachs%26aulast%3DCostanzo%26aufirst%3DM.%2BJ.%26aulast%3DAlmond%26aufirst%3DH.%2BR.%26aulast%3DHecker%26aufirst%3DL.%2BR.%26aulast%3DSchott%26aufirst%3DM.%2BR.%26aulast%3DYabut%26aufirst%3DS.%2BC.%26aulast%3DZhang%26aufirst%3DH.-C.%26aulast%3DAndrade-Gordon%26aufirst%3DP.%26aulast%3DCorcoran%26aufirst%3DT.%2BW.%26aulast%3DGiardino%26aufirst%3DE.%2BC.%26aulast%3DKauffman%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DJ.%2BM.%26aulast%3Dde%2BGaravilla%26aufirst%3DL.%26aulast%3DHaertlein%26aufirst%3DB.%2BJ.%26aulast%3DMaryanoff%26aufirst%3DB.%2BE.%26atitle%3DIn-Depth%2520Study%2520of%2520Tripeptide-Based%2520Alpha-Ketoheterocycles%2520as%2520Inhibitors%2520of%2520Thrombin.%2520Effective%2520Utilization%2520of%2520the%2520S1%25E2%2580%25B2%2520Subsite%2520and%2520Its%2520Implications%2520to%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1984%26epage%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Williams, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, J. F.</span><span> </span><span class="NLM_article-title">The Kinetics of Reversible Tight-Binding Inhibition</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1979</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">437</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm5015633&amp;key=10.1016%2F0076-6879%2879%2963019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm5015633&amp;key=502865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm5015633&amp;key=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1979&pages=437-467&author=J.+W.+Williamsauthor=J.+F.+Morrison&title=The+Kinetics+of+Reversible+Tight-Binding+Inhibition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of reversible tight-binding inhibition</span></div><div class="casAuthors">Williams, Jeffrey W.; Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">Enzyme Kinet. Mech., Part A</span>),
    <span class="NLM_cas:pages">437-67</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">In addn. to a general discussion of the kinetics of reversible tight-binding inhibitors, which exert their effect at a concn. comparable to that of the enzyme, and for which steady-state theory is used, exptl. results are given of the tight-binding inhibition of creatine kinase by Eu3+ (actually Eu-ADP) under steady-state conditions.  The inhibition of creatine kinase by fixed concns. of Eu3+ in the presence of varying concns. of Mg-ADP indicated that the inhibition is competitive, suggesting that Eu-ADP acts as a structural analog of Mg-ADP.  The inhibition was noncompetitive with respect to creatine phosphate.  The dissocn. consts. for binding of Eu-ADP to the free enzyme and the enzyme-phosphocreatine complex were 1.5 and 2.2 nM, resp., as detd. from Ki values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwV3voXt27Vg90H21EOLACvtfcHk0lhPWs2QUJqnqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D&md5=a770f91b51661c9502d0aca44a24e1c7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2879%2963019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252879%252963019-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BW.%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520Kinetics%2520of%2520Reversible%2520Tight-Binding%2520Inhibition%26jtitle%3DMethods%2520Enzymol.%26date%3D1979%26volume%3D63%26spage%3D437%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NCL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NCL','PDB','3NCL'); return false;">PDB: 3NCL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1B5G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1B5G','PDB','1B5G'); return false;">PDB: 1B5G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EAW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EAW','PDB','1EAW'); return false;">PDB: 1EAW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EAX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EAX','PDB','1EAX'); return false;">PDB: 1EAX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GV6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GV6','PDB','2GV6'); return false;">PDB: 2GV6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GV7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GV7','PDB','2GV7'); return false;">PDB: 2GV7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BN9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BN9','PDB','3BN9'); return false;">PDB: 3BN9</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i18"><a href="/doi/suppl/10.1021/jm5015633">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_30144"></div></div></div></div></div><hr /></hr><p class="last">Solution and solid-phase synthesis, materials and equipment; synthetic procedures; chemistry; progress curves; docking results for the 8000 compounds (CSV) and the coordinates of the model structure of matriptase-2 (PDB). This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5015633/suppl_file/jm5015633_si_001.pdf">jm5015633_si_001.pdf (11.49 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5015633/suppl_file/jm5015633_si_002.csv">jm5015633_si_002.csv (338.86 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5015633/suppl_file/jm5015633_si_003.pdb">jm5015633_si_003.pdb (287.99 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm5015633&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2014.57.issue-23%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Fjm5015633%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm5015633" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                1MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679957a13c443c5e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
