
<html lang="en"     class="pb-page"  data-request-id="7c25f5b1-81a4-4eca-99bb-a41194875b64"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2017.60.issue-10;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b00313;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of a B-Cell Lymphoma 6 Protein–Protein Interaction Inhibitor by a Biophysics-Driven Fragment-Based Approach" /></meta><meta name="dc.Creator" content="Yusuke  Kamada" /></meta><meta name="dc.Creator" content="Nozomu  Sakai" /></meta><meta name="dc.Creator" content="Satoshi  Sogabe" /></meta><meta name="dc.Creator" content="Koh  Ida" /></meta><meta name="dc.Creator" content="Hideyuki  Oki" /></meta><meta name="dc.Creator" content="Kotaro  Sakamoto" /></meta><meta name="dc.Creator" content="Weston  Lane" /></meta><meta name="dc.Creator" content="Gyorgy  Snell" /></meta><meta name="dc.Creator" content="Motoo  Iida" /></meta><meta name="dc.Creator" content="Yasuhiro  Imaeda" /></meta><meta name="dc.Creator" content="Junichi  Sakamoto" /></meta><meta name="dc.Creator" content="Junji  Matsui" /></meta><meta name="dc.Description" content="B-cell lymphoma 6 (BCL6) is a transcriptional factor that expresses in lymphocytes and regulates the differentiation and proliferation of lymphocytes. Therefore, BCL6 is a therapeutic target for au..." /></meta><meta name="Description" content="B-cell lymphoma 6 (BCL6) is a transcriptional factor that expresses in lymphocytes and regulates the differentiation and proliferation of lymphocytes. Therefore, BCL6 is a therapeutic target for au..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 11, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00313" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00313" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00313" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00313" /></link>
        
    
    

<title>Discovery of a B-Cell Lymphoma 6 Protein–Protein Interaction Inhibitor by a Biophysics-Driven Fragment-Based Approach | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00313" /></meta><meta property="og:title" content="Discovery of a B-Cell Lymphoma 6 Protein–Protein Interaction Inhibitor by a Biophysics-Driven Fragment-Based Approach" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0013.jpeg" /></meta><meta property="og:description" content="B-cell lymphoma 6 (BCL6) is a transcriptional factor that expresses in lymphocytes and regulates the differentiation and proliferation of lymphocytes. Therefore, BCL6 is a therapeutic target for autoimmune diseases and cancer treatment. This report presents the discovery of BCL6–corepressor interaction inhibitors by using a biophysics-driven fragment-based approach. Using the surface plasmon resonance (SPR)-based fragment screening, we successfully identified fragment 1 (SPR KD = 1200 μM, ligand efficiency (LE) = 0.28), a competitive binder to the natural ligand BCoR peptide. Moreover, we elaborated 1 into the more potent compound 7 (SPR KD = 0.078 μM, LE = 0.37, cell-free protein–protein interaction (PPI) IC50 = 0.48 μM (ELISA), cellular PPI IC50 = 8.6 μM (M2H)) by a structure-based design and structural integration with a second high-throughput screening hit." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00313"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00313">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00313&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00313&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00313&amp;href=/doi/10.1021/acs.jmedchem.7b00313" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 4358-4368</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00301" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00328" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a B-Cell Lymphoma 6 Protein–Protein Interaction Inhibitor by a Biophysics-Driven Fragment-Based Approach</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yusuke++Kamada">Yusuke Kamada</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-3701-2075" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nozomu++Sakai">Nozomu Sakai</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Satoshi++Sogabe">Satoshi Sogabe</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Koh++Ida">Koh Ida</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hideyuki++Oki">Hideyuki Oki</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kotaro++Sakamoto">Kotaro Sakamoto</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Weston++Lane">Weston Lane</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gyorgy++Snell">Gyorgy Snell</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Motoo++Iida">Motoo Iida</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yasuhiro++Imaeda">Yasuhiro Imaeda</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Junichi++Sakamoto">Junichi Sakamoto</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Junji++Matsui">Junji Matsui</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Takeda California Inc., 10410 Science Center Dr., San Diego, California 92121, United States</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#463f3335332d23682d272b272227770632272d2322276825292b"><span class="__cf_email__" data-cfemail="324b47414759571c59535f53565303724653595756531c515d5f">[email protected]</span></a>. Tel: +81-0466-32-2782.</div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#325c5d485d5f471c415359535b724653595756531c515d5f"><span class="__cf_email__" data-cfemail="95fbfaeffaf8e0bbe6f4fef4fcd5e1f4fef0f1f4bbf6faf8">[email protected]</span></a>. Tel: +81-0466-32-1273.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00313&amp;href=/doi/10.1021%2Facs.jmedchem.7b00313" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 4358–4368</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 4, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 February 2017</li><li><span class="item_label"><b>Published</b> online</span>11 May 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 May 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00313" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00313</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4358%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYusuke%2BKamada%252C%2BNozomu%2BSakai%252C%2BSatoshi%2BSogabe%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D10%26contentID%3Dacs.jmedchem.7b00313%26title%3DDiscovery%2Bof%2Ba%2BB-Cell%2BLymphoma%2B6%2BProtein%25E2%2580%2593Protein%2BInteraction%2BInhibitor%2Bby%2Ba%2BBiophysics-Driven%2BFragment-Based%2BApproach%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4368%26publicationDate%3DMay%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00313"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3097</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">22</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00313" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a B-Cell Lymphoma 6 Protein–Protein Interaction Inhibitor by a Biophysics-Driven Fragment-Based Approach&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yusuke&quot;,&quot;last_name&quot;:&quot;Kamada&quot;},{&quot;first_name&quot;:&quot;Nozomu&quot;,&quot;last_name&quot;:&quot;Sakai&quot;},{&quot;first_name&quot;:&quot;Satoshi&quot;,&quot;last_name&quot;:&quot;Sogabe&quot;},{&quot;first_name&quot;:&quot;Koh&quot;,&quot;last_name&quot;:&quot;Ida&quot;},{&quot;first_name&quot;:&quot;Hideyuki&quot;,&quot;last_name&quot;:&quot;Oki&quot;},{&quot;first_name&quot;:&quot;Kotaro&quot;,&quot;last_name&quot;:&quot;Sakamoto&quot;},{&quot;first_name&quot;:&quot;Weston&quot;,&quot;last_name&quot;:&quot;Lane&quot;},{&quot;first_name&quot;:&quot;Gyorgy&quot;,&quot;last_name&quot;:&quot;Snell&quot;},{&quot;first_name&quot;:&quot;Motoo&quot;,&quot;last_name&quot;:&quot;Iida&quot;},{&quot;first_name&quot;:&quot;Yasuhiro&quot;,&quot;last_name&quot;:&quot;Imaeda&quot;},{&quot;first_name&quot;:&quot;Junichi&quot;,&quot;last_name&quot;:&quot;Sakamoto&quot;},{&quot;first_name&quot;:&quot;Junji&quot;,&quot;last_name&quot;:&quot;Matsui&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;4358-4368&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00313&quot;},&quot;abstract&quot;:&quot;B-cell lymphoma 6 (BCL6) is a transcriptional factor that expresses in lymphocytes and regulates the differentiation and proliferation of lymphocytes. Therefore, BCL6 is a therapeutic target for autoimmune diseases and cancer treatment. This report presents the discovery of BCL6–corepressor interaction inhibitors by using a biophysics-driven fragment-based approach. Using the surface plasmon resonance (SPR)-based fragment screening, we successfully identified fragment 1 (SPR KD = 1200 μM, ligand efficiency (LE) = 0.28), a competitive binder to the natural ligand BCoR peptide. Moreover, we elaborated 1 into the more potent compound 7 (SPR KD = 0.078 μM, LE = 0.37, cell-free protein–protein interaction (PPI) IC50 = 0.48 μM (ELISA), cellular PPI IC50 = 8.6 μM (M2H)) by a structure-based design and structural integration with a second high-throughput screening hit.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00313&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00313" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00313&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00313" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00313&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00313" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00313&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00313&amp;href=/doi/10.1021/acs.jmedchem.7b00313" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00313" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00313" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00313%26sid%3Dliteratum%253Aachs%26pmid%3D28471657%26genre%3Darticle%26aulast%3DKamada%26date%3D2017%26atitle%3DDiscovery%2Bof%2Ba%2BB-Cell%2BLymphoma%2B6%2BProtein%25E2%2580%2593Protein%2BInteraction%2BInhibitor%2Bby%2Ba%2BBiophysics-Driven%2BFragment-Based%2BApproach%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D10%26spage%3D4358%26epage%3D4368%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291917" title="Pyrimidine">Pyrimidine</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/jmcmar.2017.60.issue-10/20170525/jmcmar.2017.60.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">B-cell lymphoma 6 (BCL6) is a transcriptional factor that expresses in lymphocytes and regulates the differentiation and proliferation of lymphocytes. Therefore, BCL6 is a therapeutic target for autoimmune diseases and cancer treatment. This report presents the discovery of BCL6–corepressor interaction inhibitors by using a biophysics-driven fragment-based approach. Using the surface plasmon resonance (SPR)-based fragment screening, we successfully identified fragment <b>1</b> (SPR <i>K</i><sub>D</sub> = 1200 μM, ligand efficiency (LE) = 0.28), a competitive binder to the natural ligand BCoR peptide. Moreover, we elaborated <b>1</b> into the more potent compound <b>7</b> (SPR <i>K</i><sub>D</sub> = 0.078 μM, LE = 0.37, cell-free protein–protein interaction (PPI) IC<sub>50</sub> = 0.48 μM (ELISA), cellular PPI IC<sub>50</sub> = 8.6 μM (M2H)) by a structure-based design and structural integration with a second high-throughput screening hit.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44445" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44445" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">B-cell lymphoma 6 (BCL6) is a transcriptional factor that belongs to the bric-a-brac, tramtrack, broad complex/poxvirus zinc finger (BTB/POZ) family of proteins. It possesses BTB, RD2, and zinc finger domains and interacts with three corepressors, i.e., BCoR, SMRT, and NCoR.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> It drives germinal center B-cell formation and differentiation of T lymphocytes.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5">(2-5)</a> It is also involved in the differentiation and proliferation of diffuse large B-cell lymphomas.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> Consequently, it is thought to be an effective therapeutic target for the treatment of autoimmune diseases and cancer.</div><div class="NLM_p">The crystal structure of the BCL6 BTB domain (BCL6<sup>BTB</sup>) complexed with the SMRT peptide has already been reported.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The crystal structure of the cocomplex with the BCL6 inhibitor 79-6 has also been solved.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> These ligands bind in the lateral groove of the BCL6<sup>BTB</sup> homodimer. Parekh et al. have suggested that the lateral groove of the BCL6<sup>BTB</sup> homodimer could be an excellent therapeutic target to develop effective small-molecule inhibitors.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Several compounds and peptides, other than 79-6, have been reported as BCL6<sup>BTB</sup> inhibitors, including RI-BPI with a <i>K</i><sub>D</sub> of ∼10 μM, Rifamycin SV with a <i>K</i><sub>D</sub> of ∼1 mM, and FX1 with a <i>K</i><sub>D</sub> of 7 μM, all of which have binding affinities weaker than 1 μM (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10-12)</a> Recently, we reported the peptide F1324, which inhibits the BCL6–BCoR interaction with a <i>K</i><sub>D</sub> of 0.57 nM (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Peptide F1324 also binds to the lateral groove of the BCL6<sup>BTB</sup> homodimer. These data suggest that the lateral groove of the BCL6<sup>BTB</sup> homodimer could be a potent site for inhibitors to bind with high affinity.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known BCL6 inhibitors. <sup><i>a</i></sup>Determined by microscale thermophoresis.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a><sup><i>b</i></sup>Determined by Fluorescence polarization assay.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a><sup><i>c</i></sup>Determined by SPR.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a><sup><i>d</i></sup>All <span class="smallcaps smallerCapital">d</span>-amino acids; TAT means cell penetrating peptide sequence, and Fu means fusogenic peptide sequence.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a><sup><i>e</i></sup>Determined by DLBCL cell growth inhibition.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Fragment-based drug discovery (FBDD) has been widely practiced in both industry and academia to identify compounds with good physiological profiles, new scaffolds, and new binding sites.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> The starting point of this approach is to identify molecule fragments with low affinity using high-concentration screening and then to improve their affinity by linking or growing them.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> Biophysical assays (surface plasmon resonance (SPR), nuclear magnetic resonance (NMR), thermal shift assays, and so on) have been used for the FBDD approach because they are more sensitive than biochemical assays.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> Of these, SPR biosensing is the most cost-effective (low-protein consumption and quick assay development) and provides high-content information on binding affinity and kinetics and identification of pan-assay interference compounds (PAINS).<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> Thus, SPR biosensing assays are often used for the FBDD approach.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25">(22-25)</a> Protein–protein interactions (PPIs) represent a challenging target for the biophysics-driven FBDD approach because the targets, like BCL6, do not possess a deep binding pocket.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26-29)</a></div><div class="NLM_p last">In this study, SPR-based fragment screening was performed against BCL6 to discover BCL6–cofactor interaction inhibitors. As a result, we successfully identified triazine fragment <b>1</b>, which binds to the lateral groove of the BCL6<sup>BTB</sup> homodimer where BCoR also binds. Furthermore, fragment <b>1</b> was optimized into the more potent inhibitor <b>7</b>, which exhibits cellular activity and two-digit nanomolar binding affinity, by structure-based drug design using X-ray analysis and structural integration with a second high-throughput screening (HTS) hit. This process represents a highly successful fragment-assisted drug discovery approach (FADD).<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22012" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22012" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Protein Preparation</h3><div class="NLM_p last">To identify BCL6 inhibitors that disrupt the interaction between BCL6 and cofactors, we attempted to prepare BCL6<sup>BTB</sup> cofactors that bind to the lateral groove of the BCL6<sup>BTB</sup> homodimer. BCL6<sup>BTB</sup> is approximately 15 kDa in size, but despite its small size, its domain contains five cysteine residues. We speculated that the high proportion of Cys residues would cause instability, which was reinforced by the fact that the co-crystal structure of BCL6<sup>BTB</sup> has been obtained using the Cys mutant (Ala5-Glu129, C8Q, C67R, and C84N) to prevent aggregation.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In addition, we needed to consider that the high concentration (mM) required for SPR screening can cause false positives. In an attempt to mitigate this tendency for false positives, we decided to use the Cys-mutation protein for SPR-based screening.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> SPR Assay Development</h3><div class="NLM_p">For our SPR-based screening, we used a Biacore 4000 high-throughput instrument with five detection areas for protein immobilization and four flow cells for compound injection. This parallel immobilization enables the evaluation of selectivity and nonspecific binding simultaneously.<a onclick="showRef(event, 'ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34">(32-34)</a> We also screened against both wild-type (wt) and mutant (mt) BCL6<sup>BTB</sup> by SPR. <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A,B shows the layout of the sensor chip used in this screening. Avi-tagged wt BCL6<sup>BTB</sup> and avi-tagged mt BCL6<sup>BTB</sup> were captured onto spots 1 and 2, respectively, by NeutrAvidin. In addition, wt BCL6<sup>BTB</sup> was covalently immobilized onto spot 5 by an amine coupling reaction (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1">Figure S1</a>). This configuration enabled us to evaluate binding to captured wt BCL6<sup>BTB</sup>, captured mt BCL6<sup>BTB</sup>, coupled wt BCL6<sup>BTB</sup>, and coupled NeutrAvidin.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Layout of the sensor chip. (A) Four independent flow cells (Fc), each with five detection spots. (B) Protein immobilized in each flow cell. Binding responses of captured wt BCL6<sup>BTB</sup>, captured mt BCL6<sup>BTB</sup>, coupled wt BCL6<sup>BTB</sup>, and NeutrAvidin are detected on spots 1–3, spots 2–3, spots 5–4, and spots 3–4, respectively. Sensorgrams of BCoR peptide binding to (C) captured wt BCL6<sup>BTB</sup>, (D) captured mt BCL6<sup>BTB</sup>, (E) coupled wt BCL6<sup>BTB</sup>, and (F) NeutrAvidin. Lower graphs indicate the fit plots of the response measured at equilibrium plotted against BCoR peptide concentration. Top concentration is 100 μM; dilution step is 2-fold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To verify the activity of each immobilized BCL6<sup>BTB</sup>, we evaluated the binding of the BCoR peptide as a positive control (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C–F). The BCoR peptide successfully binds with each BCL6<sup>BTB</sup> in a dose-dependent manner. The <i>K</i><sub>D</sub> values for the BCoR peptide with each BCL6<sup>BTB</sup> are in the two-digit micromolar range. Since there are no significant differences in the binding affinities among the different BCL6<sup>BTB</sup>s, the mutation of the Cys residues is not thought to affect the interaction between BCL6 and BCoR, and we concluded that each BCL6<sup>BTB</sup> is immobilized while retaining its activity.</div><div class="NLM_p">For performing high-quality SPR-based screening, maintaining the stability of the immobilized protein is important.<a onclick="showRef(event, 'ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37">(35-37)</a> Therefore, selection of a suitable buffer is critical to retain stability. Compared with that used in biochemical assays, the concentration of dimethyl sulfoxide (DMSO) required for SPR-based screening is generally higher to solubilize the fragments at high concentration. Accordingly, the influence of DMSO on the activities of the immobilized BCL6<sup>BTB</sup>s was evaluated (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>). The <i>K</i><sub>D</sub> and <i>R</i><sub>max</sub> values for the BCoR peptide are unaffected by the concentration of DMSO. Moreover both wt BCL6<sup>BTB</sup> and mt BCL6<sup>BTB</sup> are stable for 36 h (500 injections) under buffer conditions of 1% or 5% DMSO. Therefore, we decided to perform screening with a 5% DMSO buffer to ensure a 1 mM solubility for the fragments.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Affinities Between BCoR Peptide and BCL6<sup>BTB</sup>s</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">wt BCL6<sup>BTB</sup></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">mt BCL6<sup>BTB</sup></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">DMSO</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>D</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub>max</sub> (RU)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>D</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub>max</sub> (RU)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">1%</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">245</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">311</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">3%</td><td class="colsep0 rowsep0" align="char" char=".">9.3</td><td class="colsep0 rowsep0" align="char" char=".">242</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">302</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">5%</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">243</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">304</td></tr></tbody></table></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Primary Fragment Screening</h3><div class="NLM_p">In a primary SPR-based screening of our in-house fragment library, we were able to obtain the screening data for captured wt BCL6<sup>BTB</sup>, captured mt BCL6<sup>BTB</sup>, coupled wt BCL6<sup>BTB</sup>, and NeutrAvidin simultaneously using the Biacore 4000 apparatus. In the screening with the captured mt BCL6<sup>BTB</sup>, the sensorgrams against most of the fragments show a reasonable box-shape, indicating fast binding (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Conversely, captured wt BCL6<sup>BTB</sup> presents many plots that indicate slow binding (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). These data suggest that many fragments reversibly reacted with the Cys residues on wt BCL6 due to a concentration as high as 1 mM. Regarding the immobilization method, the binding responses of the positive control (BCoR) to coupled wt BCL6<sup>BTB</sup> decrease depending on the injection number, while the binding responses of the positive control to each captured BCL6<sup>BTB</sup> are stable (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A–C). These results indicate that captured mt BCL6<sup>BTB</sup> is optimal for stable screening.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. SPR-based screening of 1494 fragments tested against (A) captured wt BCL6<sup>BTB</sup>, (B) captured mt BCL6<sup>BTB</sup>, (C) coupled wt BCL6<sup>BTB</sup>, and (D) NeutrAvidin. The color of each plot indicates fast binding (blue-circle), slow binding (red-circle), and 100 μM BCoR peptide as a positive control (gray-square). (E) Venn diagram showing the number of fragments selected at each BCL6<sup>BTB</sup> screening and the overlap of fragments found by each screening.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In the primary screening, we identified 266 fragments for captured wt BCL6<sup>BTB</sup>, 256 fragments for captured mt BCL6<sup>BTB</sup>, 273 fragments for coupled wt BCL6<sup>BTB</sup>, and 50 fragments for NeutrAvidin. The Venn diagram in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>E summarizes the results of this fragment screening. A total of 64 compounds bind to all the BCL6<sup>BTB</sup>s, hence these fragments were selected as the first priority for the follow-up dose–response tests.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Discovery and Characterization of Fragment <b>1</b></h3><div class="NLM_p">To confirm the reproducibility and dose-responsivity of the 64 fragments, we performed dose-titration experiments. All the fragments show binding responses in a dose-dependent and reversible manner (data not shown). Next, we performed saturation transfer difference-nuclear magnetic resonance (STD-NMR) experiments to prioritize the 64 fragments. Seven out of the 64 fragments exhibit binding to BCL6 (data not shown). Of seven binding fragments, we focused on fragment <b>1</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A), which shows binding to BCL6 by STD-NMR (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B) and furthermore competition with the BCoR peptide by SPR competition experiments (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). The <i>K</i><sub>D</sub> value for <b>1</b> was determined to be 1.2 mM by SPR, and its ligand efficiency (LE) was calculated to be 0.28 from its <i>K</i><sub>D</sub> value. An LE of at least 0.29 is necessary to develop an orally available candidate.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Therefore, <b>1</b> is a good starting point for elaboration into potent BCL6 inhibitors.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Characterization of fragment <b>1</b>. (A) Chemical structure of fragment <b>1</b>. (B) STD-NMR experiments. <sup>1</sup>H spectra of fragment <b>1</b> for off-resonance (black line) and on-resonance (red line), and the STD spectrum of fragment <b>1</b> (difference). (C) SPR competition experiments. Sensorgrams of fragment <b>1</b> to mt BCL6<sup>BTB</sup> in the absence or presence of 100 μM BCoR peptide. Top concentration is 1 mM; dilution step is 2-fold. (D) Co-crystal structure of fragment <b>1</b> in complex with BCL6<sup>BTB</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">For further optimization of fragment <b>1</b>, we attempted to determine its binding mode by determining the co-crystal structure of <b>1</b> with wt BCL6<sup>BTB</sup>. Co-crystals of <b>1</b> with BCL6<sup>BTB</sup> were obtained by soaking into unliganded BCL6<sup>BTB</sup> crystals in which the biological homodimer is related by crystallographic two-fold symmetry. Electron density indicated that <b>1</b> binds to the lateral groove at the dimer interface and exhibits alternate conformations of the aminotriazine moiety (<a class="ref internalNav" href="#fig4" aria-label="Figures 4">Figures 4</a> and <a class="ref internalNav" href="#notes-1" aria-label="S4">S4</a>). The linker nitrogen forms a hydrogen bond with the main-chain oxygen of Met51. Comparison with the binding mode of 79-6 shows that <b>1</b> occupies the same site with a similar interaction pattern (data not shown). Therefore, we evaluated the BCL6 binding affinities of several pyrimidine derivatives that are structurally related to <b>1</b>.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Activity Relationships of Compounds Related to Fragment <b>1</b></h3><div class="NLM_p"><a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> summarizes the structure–activity relationships of the pyrimidine derivatives along with the original fragment <b>1</b>. Pyrimidine derivative <b>2</b> exhibits a weak binding potency (<i>K</i><sub>D</sub> = 3000 μM), while the introduction of fluorine (<b>3</b>) or chlorine (<b>4</b>) atoms onto the 5-position of compound <b>2</b> results in significantly improved binding affinities (<i>K</i><sub>D</sub> = 180 and 68 μM, respectively). Chloropyrimidine <b>4</b> exhibits good LE (0.38) and may be a promising compound for further optimization with the aim to develop more potent BCL6 inhibitors.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure–Activity Relationships of the Pyrimidine Derivatives</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0010.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center">Z</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>D</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">LE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char=".">1200</td><td class="colsep0 rowsep0" align="char" char=".">0.28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="char" char=".">3000</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="char" char=".">180</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last"><i>K</i><sub>D</sub> values are reported as the arithmetic mean of at least two separate runs (<i>n</i> = 2).</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Structural Integration with a Second HTS Hit</h3><div class="NLM_p">Simultaneously, an independent HTS campaign for BCL6 inhibitors identified the weak BCL6 inhibitor <b>5</b> (<i>K</i><sub>D</sub> = 88 μM, LE = 0.16), which also has a pyrimidine core structure, as shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>; compound <b>5</b> was discovered by the HTS campaign using the ELISA monitoring the interaction between the BCL6<sup>BTB</sup> and the BCoR peptide. The structural resemblance of <b>5</b> to <b>4</b> prompted us to integrate these two structures, forming the “hybrid compounds” shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>. Co-crystal structure determination of compounds <b>4</b> and <b>5</b> with BCL6<sup>BTB</sup> confirmed that these compounds exhibit the same binding modes as that of the initial triazine fragment <b>1</b> (<a class="ref internalNav" href="#fig6" aria-label="Figures 6">Figures 6</a> and <a class="ref internalNav" href="#notes-1" aria-label="S4">S4</a>). As expected, the two aromatic rings connected by the amine linker in the two structures significantly overlap. The co-crystal structure of <b>4</b> reveals that the chlorine atom at the 5-position of the pyrimidine core occupies a small lipophilic region of BCL6<sup>BTB</sup>. The binding mode information on the HTS-derived pyrimidine derivative <b>5</b> shows that the carbonyl oxygen of the right-hand-side cyclic amide moiety interacts with Glu115 of BCL6<sup>BTB</sup>. Therefore, based on the structural superposition of <b>5</b> with <b>4</b>, the hybrid compounds <b>6</b> and <b>7</b> were designed and synthesized (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). We selected a simple unsubstituted pyridinylmethyl substituent for compound <b>7</b> while taking synthetic tractability into account.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Consolidation of the fragment-based inhibitor <b>4</b> with hit compound <b>5</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structural superposition of the complex structures with compounds <b>4</b> (orange) and <b>5</b> (yellow).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. BCL6 Binding Affinities of the Hybrid-Type Compounds</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0011.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0012.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last"><i>K</i><sub>D</sub> values are reported as the arithmetic mean of at least two separate runs (<i>n</i> = 2).</p></div></div><div></div></div><div class="NLM_p">The pyrimidine derivatives <b>6</b> and <b>7</b> exhibit significantly improved BCL6 binding affinities (<i>K</i><sub>D</sub> = 9.3 and 0.078 μM, respectively) compared to those of the fragment derivative <b>4</b> and HTS hit <b>5</b>. To confirm the binding modes, the co-crystal structures of compounds <b>6</b> and <b>7</b> were determined as well as those of the other compounds (<a class="ref internalNav" href="#fig7" aria-label="Figures 7">Figures 7</a>A and <a class="ref internalNav" href="#notes-1" aria-label="S4">S4</a>). The electron density map shows that the pyridine moiety of compound <b>7</b> is exposed to solvent and mostly disordered. Both compounds bind to BCL6<sup>BTB</sup> in the same binding mode as those for compounds <b>1</b>, <b>4</b>, and <b>5</b>. The linker nitrogen forms a hydrogen bond with the main-chain oxygen of Met51, and the carbonyl oxygen of the cyclic amide moiety interacts with Glu115. The reasons for the significant improvement in binding affinity by introduction of the pyrimidine ring are unclear because no specific structural features are observed.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Co-crystal structure of compound <b>7</b> in complex with BCL6<sup>BTB</sup>. (B) Chemical structure of <b>7</b>. (C) Equilibrium plot and sensorgrams (insert) of <b>7</b> binding to BCL6. (D, E) PPI inhibitory activities in cell-free (ELISA) and cellular (M2H) assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As compound <b>7</b> exhibits two-digit nanomolar binding affinity along with good LE (0.37), we also evaluated the BCL6−BCoR PPI inhibitory activity of <b>7</b> in cell-free and cellular assays (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B–E). To evaluate the PPI inhibitory activities in cells, a previously reported M2H assay was used.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Briefly, HEK293T cells were transfected with the GAL4-responsive reporter plasmid, the bait expression plasmid (GAL4 DNA-binding domain fused to wt BCL6, Ala5-Glu129), and the VB expression plasmid (VP16 activation domain fused to BCoR, Leu112-Ala753). Then, the PPI activities were evaluated by monitoring luciferase activities in the transfected HEK293 cell lysates. Compound <b>7</b> exhibits potent cell-free PPI inhibitory activity (ELISA IC<sub>50</sub> = 0.48 μM) along with moderate cellular potency (M2H IC<sub>50</sub> = 8.6 μM). Taken together, compound <b>7</b> is a promising BCL6 inhibitor candidate for further exploration. The total process of the fragment-based approach was summarized in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>. Scaffold hopping of the initial triazine fragment yielded the pyrimidine fragment <b>2</b>, which enabled an introduction of a substituent occupying the small lipophilic region of BCL6<sup>BTB</sup>. Structure-based modification of the resulting compound <b>4</b> along with the consolidation of HTS hit <b>5</b> led to the discovery of the cell-active compound <b>7</b>. The HTS hit <b>5</b> had a poor ligand efficiency (LE = 0.16), and its complex chemical structure made its modification rather difficult. Conversely, the fragment-derived compound <b>4</b> was a very good step for further modification due to its good ligand efficiency (LE = 0.38) and small molecular size.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Summary of the FBDD approach for identification of the compound <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemistry</h3><div class="NLM_p">Synthetic methods for pyrimidine derivatives are shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The reaction of the 4-chloropyrimidines <b>8</b>–<b>11</b> with anilines <b>12</b> and <b>13</b> gives the corresponding 4-anilinopyrimidines <b>2</b>, <b>6</b>, <b>14</b>–<b>16</b>. Amination of <b>14</b>–<b>16</b> with amines <b>17</b> and <b>18</b> at the 2-position of the pyrimidine derivatives gives the di- or trisubstituted pyrimidine derivatives <b>3</b>, <b>4</b>, and <b>7</b>.</div><figure id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Procedure for the Pyrimidine Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26400" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26400" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">To identify novel and promising starting points for the discovery of potent BCL6 inhibitors, we adopted an FBDD approach, which represents a challenge given that the target is a PPI. We identified 64 novel binding fragments using SPR-based fragment screening, and among them, fragment <b>1</b> showed moderate LE (0.28) and competition with the BCoR peptide. Moreover, BCL6 binding affinities of several structurally related pyrimidine derivatives were also measured, and we identified the more potent fragment-like pyrimidine derivative <b>4</b> (<i>K</i><sub>D</sub> = 68 μM). Consolidation of the structural information on compound <b>4</b> with the independently obtained HTS hit <b>5</b> led to the identification of the potent BCL6 inhibitor <b>7</b> (<i>K</i><sub>D</sub> = 0.078 μM, LE = 0.37), which is >15,000-fold more potent than the initial fragment <b>1</b>. Compound <b>7</b> also exhibited an efficacy in cell-free and cellular PPI assays (ELISA IC<sub>50</sub> = 480 nM, M2H IC<sub>50</sub> = 8.6 μM). These findings suggest that the combination of biophysics-driven FBDD, SBDD, and FADD is a promising strategy for hit identification and lead generation against challenging targets such as PPIs.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41500" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41500" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Preparation of BCL6</h3><div class="NLM_p last">The fragments of His, avi, SUMO (LifeSensors), Flag, Tobacco Etch Virus (TEV) protease recognition sequence, and BCL6<sup>BTB</sup> (human BCL6 from 5 to 129 aa) were amplified by the polymerase chain reaction (PCR) and ligated into pET21 vector (Merck). The mt BCL6<sup>BTB</sup> (C8Q, C67R, C84N) was constructed by the overlap PCR method. The proteins were expressed in <i>E. coli</i> BL21 (DE3) (NIPPON GENE) and partially biotinylated on avi-tag by endogenous BirA. The proteins were purified using Ni-NTA (QIAGEN) and Superdex200 (GE healthcare) columns. For Flag-tagged wt BCL6<sup>BTB</sup>, SUMO was digested by ULP1 (LifeSensors). To remove ULP1, His-avi-SUMO, and uncleaved protein, the digested solution was passed through an Ni-NTA column in 50 mM Tris-HCl pH 8.0, 300 mM NaCl, 5% glycerol, and 1 mM DTT. To increase the purity, ion-exchange chromatography (monoQ, GE Healthcare) was conducted. The purified proteins were concentrated to 2 mg/mL for storage at −80 °C.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Fragment Library</h3><div class="NLM_p last">The library contained 1494 fragments. All fragments in the library had a clogP < 3.5, the number of hydrogen-bond acceptors and donors were <6 and <3, respectively, and rotatable bonds were <3. Heavy atom count ranged between 8 and 15. The molecular weight of the fragments was <350 Da (average 180 Da), and the average of the aromatic rings was 1.5. Fragments were dissolved at 200 mM in 100% DMSO and stored at −30 °C.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> SPR</h3><div class="NLM_p">SPR biosensing experiments were performed at 22 °C on Biacore 4000 and Biacore S200 instruments equipped with SeriesS CM5 and SA sensor chips (GE healthcare).</div><div class="NLM_p">HBS-P+ (10 mM Hepes pH 7.4, 150 mM NaCl, 0.05% Surfactant P20, GE healthcare) supplemented with 1 mM DTT was used as the running buffer for immobilization. NeutrAvidin (Thermo Fisher Scientific Inc.) was covalently coupled onto spots 1, 2, and 3 of a CM5 sensor chip following the standard amine coupling procedure according to the manufacturer’s instructions. Typical immobilization levels of NeutrAvidin ranged from 10,000 to 13,000 resonance units (RUs). Subsequently, avi-tagged wt BCL6<sup>BTB</sup> and avi-tagged mt BCL6<sup>BTB</sup> were injected to spots 1 and 2, respectively. The surfaces were blocked by injecting biocytin (Thermofisher) to spots 1, 2, and 3. Approximately 13,000 RUs of each BCL6<sup>BTB</sup> were captured by NeutrAvidin. Furthermore, wt BCL6<sup>BTB</sup> was immobilized on spot 5 via the standard amine coupling procedure, resulting in immobilization levels of around 4,000 RUs.</div><div class="NLM_p">For the interaction studies, binding experiments were performed in 20 mM Tris, pH 8.0, 150 mM NaCl, 0.01% Surfactant P20, 1 mM DTT, and 5% DMSO. Different concentration sample solutions were injected for 30 or 60 s at a flow rate of 30 or 50 μL/min, and the dissociation was thereafter followed for up to 30, 60, or 150 s. Data processing and analysis were performed using the Biacore 4000 and Biacore S200 evaluation software (GE healthcare). Solvent correction was included as described in the Biacore software handbook. Sensorgrams were double referenced prior to fitting the concentration series to a steady-state affinity model. The dissociation constant <i>K</i><sub>D</sub> was calculated using the equation:<span class="NLM_disp-formula"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/jm-2017-003132_m001.gif" alt="" /></img></span>where <i>R</i><sub>max</sub>, <i>R</i>, and <i>C</i> correspond to the sample binding capacity of the surface (RU), the normalized response of test sample (RU), and the concentration of the test solution (M), respectively. LE was calculated using the equation:<span class="NLM_disp-formula"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/jm-2017-003132_m002.gif" alt="" /></img></span>where Δ<i>G</i>, HA, <i>R</i>, and <i>T</i> correspond to the Gibbs free energy (kcal/mol), heavy atom count, the ideal gas constant (1.987 × 10<sup>–3</sup> kcal/K/mol) and the temperature in Kelvin (K).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div><div class="NLM_p">For the primary screen, Takeda’s fragment library was diluted to 1 mM in the running buffer and injected for 30 s. Then, the dissociation was followed for 30 s. Every 33rd cycle, a positive control containing 100 μM BCoR peptide ((Arg498-Pro514)-Lys, acetyl-RSEIISTAPSSWVVPPK-OH, chemically synthesized by Toray Research Center) and running buffer as a blank were injected. Data processing and analysis were performed using the Biacore 4000 evaluation software. Solvent correction was included as described in the Biacore 4000 software handbook. The binding responses were normalized by the positive control and the molecular weight of each sample. Fragments with a normalized binding response that exceeded median + median absolute deviation were selected as primary hits. As for NeutrAvidin, fragments with a normalized response that was more than 90% against their theoretical <i>R</i><sub>max</sub> were selected.</div><div class="NLM_p last">For competition studies, data were acquired for test compounds binding to BCL6 in the presence of 100 μM BCoR peptide in the running buffer and samples. When a compound binds to the same site as the BCoR peptide, the response for the compound decreases.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> X-ray Crystallography</h3><div class="NLM_p last">Crystals of unliganded BCL6<sup>BTB</sup> for soaking experiments were obtained as described previously.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Crystallization was by vapor-diffusion using the sitting-drop method from 0.1 M Bis-Tris pH 6.5, 0.7 M potassium/sodium tartrate at 20 °C. To generate protein–ligand complexes, crystals were typically soaked for 2 h in a reservoir solution containing 1 mM ligand(s). Prior to data collection, crystals were immersed in the reservoir solution with the addition of 30% glycerol as a cryoprotectant and were flash-frozen in liquid nitrogen. Diffraction data were collected from a single crystal using the CCD detector Quantum 315 (ADSC) at beamlines 5.0.2 and 5.0.3 of the Advanced Light Source (Berkeley) under a 100 K nitrogen cryostream. The data were reduced and scaled with HKL2000.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The structures were solved by the molecular replacement method with Molrep<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> of the CCP4 software suite<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> using the BCL6<sup>BTB</sup> structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R28">1R28</a>) as a search model. The structures were refined through an iterative procedure utilizing REFMAC,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> followed by model building in COOT.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The final models were validated using Molprobity.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Crystallographic processing and refinement statistics are summarized in <a class="ref internalNav" href="#notes-1" aria-label="Table S1">Table S1</a>. All structural figures were generated using PyMOL (Schrödinger).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> STD-NMR</h3><div class="NLM_p last"><sup>1</sup>H NMR spectra were recorded using a 3 mm NMR tube on a 600 MHz Avance spectrometer (Bruker Biospin) equipped with a 5 mm TCI cryoprobe. All NMR samples were prepared in 50 mM phosphate D<sub>2</sub>O buffer at pH 7.4 containing 150 mM NaCl, 1 mM DTT-<i>d</i><sub>10</sub>, and 0.2% DMSO-<i>d</i><sub>6</sub> in D<sub>2</sub>O solvent. The STD spectrum was acquired from a sample containing 400 μM of fragment <b>1</b> and 20 μM BCL6<sup>BTB</sup> at 15 °C with 32 scans with on- and off-resonance saturation frequencies of 0.4 and 30 ppm, respectively, and a total saturation time of 2.0 s. The decrease in signal intensity for on-resonance irradiation results from the transfer of saturation from BCL6<sup>BTB</sup> to fragment <b>1</b>. The STD spectrum of fragment <b>1</b> was obtained by subtracting the on-resonance spectrum from the off-resonance spectrum. A positive signal in the STD spectrum indicates binding of fragment <b>1</b> to BCL6<sup>BTB</sup>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> ELISA</h3><div class="NLM_p last">Biotinylation of the ε-amino group of the C-terminus Lys on the BCoR peptide was carried out with the Biotin-(AC<sub>5</sub>)<sub>2</sub> Sulfo-OSu (DOJINDO) according to the protocol recommended by the manufacturer. The wells of a Nunc Maxisorp microplate (460–518) were coated with streptavidin (SA) (Wako) and were blocked with phosphate-buffered saline (PBS) that contained 1.0% Casein. The biotinylated BCoR peptide was captured by the SA, and a wt BCL6<sup>BTB</sup> (0.5 nM) solution in PBS that contained 0.05% Tween20 and 1 mM DTT (PBST) was added to the wells. After washing with PBST, bound BCL6<sup>BTB</sup> was detected using horseradish peroxidase (HRP)-conjugated anti-FLAG antibody (Sigma). The amounts of HRP in the wells were measured using a chemical luminescent regent (Thermo Fisher Scientific Inc.).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Percent inhibition was calculated based on wells without BCL6<sup>BTB</sup> as a high control and without compound as a low control.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> M2H</h3><div class="NLM_p last">The assay was performed according to a procedure described previously.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The vectors pGL4.35, pBind, and pACT were obtained from Promega Corp. As template DNA, human BCL6 cDNA was isolated by PCR from a human skeletal muscle cDNA library (TAKARA Bio), and human BCoR cDNA was purchased from GeneCopoeia Inc. Each cDNA fragment was granted a restriction site by PCR and digested with restriction enzymes to insert into pBIND or pACT, respectively. M2H was performed in HEK293T cells that were transfected with the reporter constructs pGL4.35 containing GAL4 special response element of firefly luciferase (9×GAL4UAS); pBIND/GAL4-BCL6 (Ala5-Glu129); and pACT/VP16-BCoR (Leu112-Ala753) by Fugene HD (Promega). The transfected cells were seeded at 1 × 10<sup>4</sup> cells/15 μL/well on 384-well plates (CORNNING) in Dulbecco’s modified Eagle’s medium that contained 10% fetal bovine serum. After incubation for 20 h at 37 °C under 5% CO<sub>2</sub>, cells were lysed to measure luciferase activity using the Bright-Glo luciferase assay system (Promega).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Percent inhibition was calculated based on the well without transfection of pBIND/GAL-BCL6 (Ala5-Glu129) as the high control and the well without the compound as a low control.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemistry</h3><div class="NLM_p"><sup>1</sup>H NMR spectra were obtained on a Bruker AVANCE II (300 MHz) spectrometer. Chemical shifts for <sup>1</sup>H NMR are reported in parts per million (ppm) downfield from tetramethylsilane (δ) as an internal standard in deuterated solvent, and the coupling constants (<i>J</i>) are given in hertz (Hz). The following abbreviations are used for spin multiplicity: s = singlet, d = doublet, t = triplet, q = quartet, dd = doublet of doublets, dt = doublet of triplets, m = multiplet, and brs = broad singlet. Reaction progress was monitored by thin-layer chromatography (TLC) analysis on silica gel 60 F<sub>254</sub> plates (Merck) or an NH TLC plate (Fuji Silysia Chemical Ltd.). Column chromatography was carried out on a silica gel column (Chromatorex NH-DM1020, 100–200 mesh; Fuji Silysia Chemical, Ltd.) or on Purif-Pack (SI ϕ 60 μM or NH ϕ 60 μM; Fuji Silysia Chemical, Ltd.). Preparative HPLC (prep-HPLC) purification was performed on a Waters 2525 equipped with a L-column2 ODS (20 × 150 mm i.d., 5 μm particle size, CERI, Japan), eluting with 0.1% TFA in distilled water (Mobile phase A) and 0.1% TFA in acetonitrile (Mobile phase B) using the following elution gradient: 20% B to 55% B over 7 min followed by 100% B isocratic over 4 min at a flow rate of 20 mL/min. The products were detected by monitoring the mass spectra recorded using a Waters ZQ2000 with electrospray ionization. The purity of all compounds used in biological studies was determined to be ≥95% by elemental analysis or HPLC-MS analysis. Low-resolution mass spectra were acquired using a Shimadzu UFLC/MS (Shimazu LC-20AD/LCMS-2020) in electron spray ionization mode (ESI+). The column used was an L-column 2 ODS (3.0 × 50 mm i.d., 3 μm; CERI, Japan) at a temperature of 40 °C and a flow rate of 1.2 or 1.5 mL/min. Condition 1: Mobile phases A and B under acidic conditions were 0.05% TFA in water and 0.05% TFA in MeCN, respectively. The proportion of mobile phase B was increased linearly from 5% to 90% over 0.9 min and maintained at 90% over the next 1.1 min. Condition 2: Mobile phases A and B under neutral conditions were a mixture of 5 mmol/L AcONH<sub>4</sub> and MeCN (9:1, v/v) and a mixture of 5 mmol/L AcONH<sub>4</sub> and MeCN (1:9, v/v), respectively. The proportion of mobile phase B was increased linearly from 5% to 90% over 0.9 min and maintained at 90% over the next 1.1 min. Elemental analyses were carried out by Sumika Chemical Analysis Service, Ltd. All commercially available solvents and reagents were used without further purification. Yields were not optimized. Compounds <b>1</b> and <b>5</b> are commercially available.</div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i><sup>4</sup>-Phenylpyrimidine-2,4-diamine (<b>2</b>)</h4><div class="NLM_p last">A mixture of 4-chloropyrimidin-2-amine (100 mg, 0.77 mmol), aniline (108 mg, 1.16 mmol), and conc. HCl (1 drop) in <i>i-</i>PrOH (1.0 mL) was heated at 80 °C for 2 h. The reaction mixture was diluted with saturated aqueous NaHCO<sub>3</sub> solution and extracted with EtOAc. The organic layer was concentrated, and the residue was purified by column chromatography (NH silica gel, eluted with 50% to 100% EtOAc in hexane). The desired fractions were concentrated, and the residue was triturated with hexane-EtOAc to give compound <b>2</b> (98 mg, 69%) as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 6.00 (1H, brs), 6.20 (2H, brs), 6.94 (1H, brs), 7.27 (2H, brs), 7.53–8.00 (3H, m), 9.08 (1H, brs). MS <i>m</i>/<i>z</i> 187.1 (M + H)<sup>+</sup>. Anal. calcd for C<sub>10</sub>H<sub>10</sub>N<sub>4</sub>: C, 64.50; H, 5.41; N, 30.09. Found: C, 64.30; H, 5.56; N, 30.21.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 5-Fluoro-<i>N</i><sup>4</sup>-phenylpyrimidine-2,4-diamine (<b>3</b>)</h4><div class="NLM_p last">A mixture of compound <b>14</b> (63 mg, 0.28 mmol) and 28% aqueous ammonia solution (2.5 mL, 37.2 mmol) in <i>i-</i>PrOH (2.5 mL) was heated at 150 °C under microwave irradiation for 4 h. The reaction mixture was concentrated, and the residue was purified by prep-HPLC. The desired fraction was concentrated and neutralized with saturated aqueous NaHCO<sub>3</sub> solution. The precipitated solid was collected, washed with H<sub>2</sub>O and IPE, air-dried, and dried in vacuo to give <b>3</b> (9.1 mg, 16%) as a colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 6.18 (2H, s), 7.01 (1H, t, <i>J</i> = 7.1 Hz), 7.28 (2H, t, <i>J</i> = 7.7 Hz), 7.80–7.89 (3H, m), 9.04 (1H, s). MS <i>m</i>/<i>z</i> 205.1 (M + H)<sup>+</sup>. Anal. calcd for C<sub>10</sub>H<sub>9</sub>FN<sub>4</sub>·0.1 H<sub>2</sub>O: C, 58.30; H, 4.50; N, 27.20. Found: C, 58.33; H, 4.67; N, 27.32.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 5-Chloro-<i>N</i><sup>4</sup>-phenylpyrimidine-2,4-diamine (<b>4</b>)</h4><div class="NLM_p last">A mixture of <b>15</b> (100 mg, 0.42 mmol) and 2 M ammonia <i>i-</i>PrOH solution (5.0 mL, 10 mmol) was heated at 150 °C under microwave irradiation for 4 h. The reaction mixture was concentrated, and the residue was purified by prep-HPLC. The desired fraction was concentrated and neutralized with saturated aqueous NaHCO<sub>3</sub> solution. The precipitated solids were collected, washed with H<sub>2</sub>O and IPE, and dried in vacuo to give <b>4</b> (23 mg, 25%) as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 6.41 (2H, s), 7.01–7.08 (1H, m), 7.26–7.33 (2H, m), 7.74 (2H, d, <i>J</i> = 8.1 Hz), 7.91 (1H, s), 8.45 (1H, s); MS <i>m</i>/<i>z</i> 221.1 (M + H)<sup>+</sup>. HPLC analysis conditions 1: <i>R</i><sub>t</sub> = 0.65 min (100% purity); 2: <i>R</i><sub>t</sub> = 0.89 min (100% purity).</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 5-((5-Chloropyrimidin-4-yl)amino)-1,3-dihydro-2<i>H</i>-indol-2-one (<b>6</b>)</h4><div class="NLM_p last">A mixture of 4,5-dichloropyrimidine (119 mg, 0.80 mmol), 5-aminoindolin-2-one (119 mg, 0.80 mmol), and TEA (0.133 mL, 0.96 mmol) in DMSO (1.0 mL) was stirred at room temperature for 14 h. The reaction mixture was diluted with a 9:1 mixture of H<sub>2</sub>O and EtOH (10 mL). The precipitated solid was collected, washed with H<sub>2</sub>O, and dried in vacuo to give <b>6</b> (95 mg, 46%) as a pale brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 3.49 (2H, s), 6.80 (1H, d, <i>J</i> = 8.2 Hz), 7.25–7.36 (1H, m), 7.40–7.49 (1H, m), 8.39 (1H, s), 8.42 (1H, s), 9.02 (1H, s), 10.36 (1H, s). MS <i>m</i>/<i>z</i> 261.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>12</sub>H<sub>9</sub>ClN<sub>4</sub>O·0.8 H<sub>2</sub>O: C, 52.39; H, 3.88; N, 20.37. Found: C, 52.32; H, 4.12; N, 20.30.</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 5-((5-Chloro-2-((pyridin-3-ylmethyl)amino)pyrimidin-4-yl)amino)-1,3-dihydro-2<i>H</i>-indol-2-one (<b>7</b>)</h4><div class="NLM_p last">A mixture of <b>16</b> (120 mg, 0.41 mmol), pyridin-3-ylmethanamine (132 mg, 1.22 mmol), and TEA (49 mg, 0.49 mmol) in DMSO (1.0 mL) was heated at 95 °C for 3 h. The reaction mixture was directly purified by column chromatography (silica gel, eluted with 50% to 100% EtOAc in hexane, followed by 0% to 20% MeOH in EtOAc) to give <b>7</b> (34 mg, 22%) as a yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 3.40 (2H, brs), 4.37 (2H, brs), 6.62–6.77 (1H, m), 7.13–7.71 (5H, m), 7.92 (1H, s), 8.37–8.52 (3H, m), 10.30 (1H, s). MS <i>m</i>/<i>z</i> 367.0 (M + H)<sup>+</sup>. Anal. calcd for C<sub>18</sub>H<sub>15</sub>ClN<sub>6</sub>O·0.1–H<sub>2</sub>O·0.1–EtOAc: C, 58.56; H, 4.27; N, 22.27. Found: C, 58.40; H, 4.52; N, 22.12. HPLC analysis conditions 1: <i>R</i><sub>t</sub> = 0.54 min (100% purity); conditions 2: <i>R</i><sub>t</sub> = 0.80 min (97.3% purity).</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2-Chloro-5-fluoro-<i>N</i>-phenylpyrimidin-4-amine (<b>14</b>)</h4><div class="NLM_p last">A mixture of 2,4-dichloro-5-fluoropyrimidine (500 mg, 2.99 mmol), aniline (0.287 mL, 3.14 mmol), and TEA (333 mg, 3.29 mmol) in <i>i-</i>PrOH (10 mL) was stirred at room temperature for 3 h. The reaction mixture was diluted with hexane, and the insoluble materials were removed by filtration. The filtrate was concentrated, and the resulting solid was washed with water and dried in vacuo to give <b>14</b> (530 mg, 79%) as a colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 7.15 (1H, t, <i>J</i> = 7.2 Hz), 7.69 (2H, d, <i>J</i> = 8.2 Hz), 8.32 (1H, d, <i>J</i> = 3.5 Hz), 9.99 (1H, s). MS <i>m</i>/<i>z</i> 224.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2,5-Dichloro-<i>N</i>-phenylpyrimidin-4-amine (<b>15</b>)</h4><div class="NLM_p last">A mixture of 2,4,5-trichloropyrimidine (1025 mg, 5.59 mmol), aniline (520 mg, 5.59 mmol), and TEA (565 mg, 5.59 mmol) in <i>i-</i>PrOH (20 mL) was heated at reflux for 14 h. The mixture was poured into H<sub>2</sub>O and extracted with EtOAc. The organic layer was washed with H<sub>2</sub>O and brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo. The residue was purified by column chromatography (silica gel, eluted with 5% to 50% EtOAc in hexane) to give <b>15</b> (900 mg, 67%) as a colorless oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 7.20 (1H, s), 7.40 (2H, s), 7.58 (2H, d, <i>J</i> = 8.2 Hz), 8.38 (1H, s), 9.52 (1H, s). MS <i>m</i>/<i>z</i> 240.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_8_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 5-((2,5-Dichloropyrimidin-4-yl)amino)indolin-2-one (<b>16</b>)</h4><div class="NLM_p last">A mixture of 2,4,5-trichloropyrimidine (0.50 g, 2.73 mmol) and 5-aminoindolin-2-one (0.40 g, 2.73 mmol) in DMSO (1.0 mL) was added with TEA (0.379 mL, 2.73 mmol) at room temperature, and the resultant mixture was stirred at room temperature for 2 h. The resulting solution was diluted with H<sub>2</sub>O/EtOH (5:1, 10 mL), and the precipitated solid was collected by filtration. The obtained solid was washed with H<sub>2</sub>O and dried in vacuo to give <b>16</b> (634 mg, 79%) as a pale brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 3.51 (2H, s), 6.82 (1H, d, <i>J</i> = 8.2 Hz), 7.24–7.30 (1H, m), 7.34 (1H, s), 8.31 (1H, s), 9.37–9.49 (1H, m), 10.33–10.51 (1H, m). MS <i>m</i>/<i>z</i> 295.1 (M + H)<sup>+</sup>.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00313">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54223" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54223" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00313" class="ext-link">10.1021/acs.jmedchem.7b00313</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">SPR sensorgrams for parallel immobilization, evaluation of stability and DMSO effect, SPR sensorgrams and equilibrium plots for compounds <b>2</b>–<b>6</b>, electron density OMIT map, data collection and refinement statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00313/suppl_file/jm7b00313_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00313/suppl_file/jm7b00313_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00313/suppl_file/jm7b00313_si_001.pdf">jm7b00313_si_001.pdf (480.79 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00313/suppl_file/jm7b00313_si_002.csv">jm7b00313_si_002.csv (0.31 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atom coordinates and structure factors have been deposited in the Protein Data Bank with accession codes: 5X4M (BCL6/<b>1</b>), 5X4N (BCL6/<b>4</b>), 5X4O (BCL6/<b>5</b>), 5X4P (BCL6/<b>6</b>), and 5X4Q (BCL6/<b>7</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b00313" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28126" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28126" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yusuke Kamada</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3701-2075" title="Orcid link">http://orcid.org/0000-0003-3701-2075</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f28b8781879997dc99939f939693c3b2869399979693dc919d9f"><span class="__cf_email__" data-cfemail="abd2ded8dec0ce85c0cac6cacfca9aebdfcac0cecfca85c8c4c6">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nozomu Sakai</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e58b8a9f8a8890cb96848e848ca591848e808184cb868a88"><span class="__cf_email__" data-cfemail="c6a8a9bca9abb3e8b5a7ada7af86b2a7ada3a2a7e8a5a9ab">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Satoshi Sogabe</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Koh Ida</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hideyuki Oki</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kotaro Sakamoto</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weston Lane</span> - <span class="hlFld-Affiliation affiliation">Takeda
California Inc., 10410
Science Center Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gyorgy Snell</span> - <span class="hlFld-Affiliation affiliation">Takeda
California Inc., 10410
Science Center Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Motoo Iida</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yasuhiro Imaeda</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junichi Sakamoto</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junji Matsui</span> - <span class="hlFld-Affiliation affiliation">Takeda
Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.K. and N.S. contributed equally to this work and wrote most of this paper. Y.K. discovered and analyzed the fragments and their relatives by SPR. N.S. designed and synthesized the compounds. S.S., W.L., and G.S. conducted crystal structure analysis and wrote a part of this paper. K.I. prepared the recombinant proteins. H.O. and M.I. conducted NMR experiments. K.S. conducted ELISA and M2H assays. Y.I., J.S., and J.M. supervised and supported this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d92e2251-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20882" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20882" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank Takamichi Muraki for biochemical experiments, Haihong Wu, Yuuka Hirao, Yoshifumi Nishimura, and Hideaki Shimojo for STD-NMR experiments, Shugo Ohno and Derek Cole for fragment library and compounds supply, and Kengo Okada and Takashi Ito for suggestions, help, and support to this work. We also acknowledge Takeshi Yamamoto, Takeshi Yasui, Takafumi Takai, Yoshihisa Nakada, Takashi Ichikawa, Takayuki Tatamiya, and Masashi Iwata for their kind suggestions. We also thank the staff of the Berkeley Center for Structural Biology who operated the ALS beamlines 5.0.2 and 5.0.3. Berkeley Center for Structural Biology is supported in part by the U.S. National Institutes of Health and U.S. National Institute of General Medical Sciences. The Advanced Light Source is supported by the Director, Office of Science, Office of Basic Energy Sciences of the U.S. Department of Energy under contract no. DE-AC02-05CH11231.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i37" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i37"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i38" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i38"> Abbreviations Used</h2><tr><td class="NLM_term">BCL6</td><td class="NLM_def"><p class="first last">B-cell lymphoma 6</p></td></tr><tr><td class="NLM_term">BCoR</td><td class="NLM_def"><p class="first last">BCL6 corepressor</p></td></tr><tr><td class="NLM_term">SMRT</td><td class="NLM_def"><p class="first last">silencing mediator of retinoic acid and thyroid hormone receptor</p></td></tr><tr><td class="NLM_term">NCoR</td><td class="NLM_def"><p class="first last">nuclear receptor corepressor</p></td></tr><tr><td class="NLM_term">M2H</td><td class="NLM_def"><p class="first last">mammalian two hybrid assay</p></td></tr><tr><td class="NLM_term">Et<sub>3</sub>N</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">IPE</td><td class="NLM_def"><p class="first last">diisopropyl ether</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term"><i>i</i>-PrOH</td><td class="NLM_def"><p class="first last">2-propanol</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TEV</td><td class="NLM_def"><p class="first last">tobacco etch virus</p></td></tr><tr><td class="NLM_term">RU</td><td class="NLM_def"><p class="first last">resonance unit</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45766" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45766" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Huynh, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischle, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardwell, V. J.</span><span> </span><span class="NLM_article-title">BCoR, a novel corepressor involved in BCL-6 repression</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1810</span><span class="NLM_x">–</span> <span class="NLM_lpage">1823</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10898795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD3cXltFCru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2000&pages=1810-1823&author=K.+D.+Huynhauthor=W.+Fischleauthor=E.+Verdinauthor=V.+J.+Bardwell&title=BCoR%2C+a+novel+corepressor+involved+in+BCL-6+repression"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">BCoR, a novel corepressor involved in BCL-6 repression</span></div><div class="casAuthors">Huynh, Khanh D.; Fischle, Wolfgang; Verdin, Eric; Bardwell, Vivian J.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1810-1823</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">BCL-6 encodes a POZ/zinc finger transcriptional repressor that is required for germinal center formation and may influence apoptosis.  Aberrant expression of BCL-6 due to chromosomal translocations is implicated in certain subtypes of non-Hodgkin's lymphoma.  The POZ domains of BCL-6 and several other POZ proteins interact with corepressors N-CoR and SMRT.  Here the authors identify and characterize a novel corepressor BCoR (BCL-6 interacting corepressor), which is expressed ubiquitously in human tissues.  BCoR can function as a corepressor when tethered to DNA and, when overexpressed, can potentiate BCL-6 repression.  Specific class I and II histone deacetylases (HDACs) interact in vivo with BCoR, suggesting that BCoR may functionally link these two classes of HDACs.  Strikingly, BCoR interacts selectively with the POZ domain of BCL-6 but not with eight other POZ proteins tested, including PLZF.  Addnl., interactions between the BCL-6 POZ domain and SMRT, N-CoR, and BCoR are mutually exclusive.  The specificity of the BCL-6/BCoR interaction suggests that BCoR may have a role in BCL-6-assocd. lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd5QFf1IeYsrVg90H21EOLACvtfcHk0ljXzJLBpepMmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXltFCru78%253D&md5=07cb1649b94629d05e5550d5cb0bcd22</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuynh%26aufirst%3DK.%2BD.%26aulast%3DFischle%26aufirst%3DW.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DBardwell%26aufirst%3DV.%2BJ.%26atitle%3DBCoR%252C%2520a%2520novel%2520corepressor%2520involved%2520in%2520BCL-6%2520repression%26jtitle%3DGenes%2520Dev.%26date%3D2000%26volume%3D14%26spage%3D1810%26epage%3D1823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Dent, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaffer, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staudt, L. M.</span><span> </span><span class="NLM_article-title">Control of inflammation, cytokine expression, and germinal center formation by BCL-6</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">589</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1126/science.276.5312.589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1126%2Fscience.276.5312.589" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=589-592&author=A.+L.+Dentauthor=A.+L.+Shafferauthor=X.+Yuauthor=D.+Allmanauthor=L.+M.+Staudt&title=Control+of+inflammation%2C+cytokine+expression%2C+and+germinal+center+formation+by+BCL-6&doi=10.1126%2Fscience.276.5312.589"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.276.5312.589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.276.5312.589%26sid%3Dliteratum%253Aachs%26aulast%3DDent%26aufirst%3DA.%2BL.%26aulast%3DShaffer%26aufirst%3DA.%2BL.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DAllman%26aufirst%3DD.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DControl%2520of%2520inflammation%252C%2520cytokine%2520expression%252C%2520and%2520germinal%2520center%2520formation%2520by%2520BCL-6%26jtitle%3DScience%26date%3D1997%26volume%3D276%26spage%3D589%26epage%3D592%26doi%3D10.1126%2Fscience.276.5312.589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Linterman, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinuesa, C. G.</span><span> </span><span class="NLM_article-title">Signals that influence T follicular helper cell differentiation and function</span> <span class="citation_source-journal">Semin. Immunopathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">196</span><span class="refDoi"> DOI: 10.1007/s00281-009-0194-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1007%2Fs00281-009-0194-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=20107805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVejt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2010&pages=183-196&author=M.+A.+Lintermanauthor=C.+G.+Vinuesa&title=Signals+that+influence+T+follicular+helper+cell+differentiation+and+function&doi=10.1007%2Fs00281-009-0194-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Signals that influence T follicular helper cell differentiation and function</span></div><div class="casAuthors">Linterman, Michelle A.; Vinuesa, Carola G.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Immunopathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-196</span>CODEN:
                <span class="NLM_cas:coden">SIEMB7</span>;
        ISSN:<span class="NLM_cas:issn">1863-2297</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Follicular helper T cells have recently emerged as a sep. CD4+ T helper lineage specialised in provision of help to B cells.  They develop independently from Th1, Th2 and Th17 cells and are crit. for humoral immunity, including the generation of long-lived and high affinity plasma cells and memory cells crucial for long-term protection against infections.  A stepwise differentiation program has emerged in which T cell receptor (TCR) signalling strength, CD28-mediated costimulation, B cell-derived inducible costimulator ligand signals, induction of c-maf and actions of cytokines, including interleukin (IL)-6 and IL-21, lead to upregulation of the transcriptional repressor B cell lymphoma 6 (Bcl-6) that drives T follicular helper (Tfh) cell differentiation.  Bcl-6 turns on a repression program that targets Blimp-1, transcriptional regulators of other helper lineages and microRNAs.  Their concerted actions modulate expression of chemokine receptors, surface mols. and cytokines crit. for follicular homing and B cell helper functions.  Here, we review the nature of Tfh cells providing help to B cells during the two phases of B cell activation that occur in the outer T zone and, for some B cells, in germinal centers (GC).  Recent insights into the signalling events that drive terminal differentiation of Tfh cells crit. for selecting somatically mutated GC B cells and the consequences of Tfh dysregulation for immunodeficiency and autoimmune pathol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDLrmacGcZ5rVg90H21EOLACvtfcHk0ljXzJLBpepMmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVejt7s%253D&md5=94a7bdf3d5a19f8490f41806de033f73</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2Fs00281-009-0194-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00281-009-0194-z%26sid%3Dliteratum%253Aachs%26aulast%3DLinterman%26aufirst%3DM.%2BA.%26aulast%3DVinuesa%26aufirst%3DC.%2BG.%26atitle%3DSignals%2520that%2520influence%2520T%2520follicular%2520helper%2520cell%2520differentiation%2520and%2520function%26jtitle%3DSemin.%2520Immunopathol.%26date%3D2010%26volume%3D32%26spage%3D183%26epage%3D196%26doi%3D10.1007%2Fs00281-009-0194-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Nurieva, R. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matskevitch, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, C.</span><span> </span><span class="NLM_article-title">Bcl6 mediates the development of T follicular helper cells</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">1005</span><span class="refDoi"> DOI: 10.1126/science.1176676</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1126%2Fscience.1176676" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=19628815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtValsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=1001-1005&author=R.+I.+Nurievaauthor=Y.+Chungauthor=G.+J.+Martinezauthor=X.+O.+Yangauthor=S.+Tanakaauthor=T.+D.+Matskevitchauthor=Y.+H.+Wangauthor=C.+Dong&title=Bcl6+mediates+the+development+of+T+follicular+helper+cells&doi=10.1126%2Fscience.1176676"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bcl6 Mediates the Development of T Follicular Helper Cells</span></div><div class="casAuthors">Nurieva, Roza I.; Chung, Yeonseok; Martinez, Gustavo J.; Yang, Xuexian O.; Tanaka, Shinya; Matskevitch, Tatyana D.; Wang, Yi-Hong; Dong, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5943</span>),
    <span class="NLM_cas:pages">1001-1005</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A fundamental function of CD4+ helper T (TH) cells is the regulation of B cell-mediated humoral immunity.  Development of T follicular helper (TFH) cells that provide help to B cells is mediated by the cytokines interleukin-6 and interleukin-21 but is independent of TH1, TH2, and TH17 effector cell lineages.  Here, the authors characterize the function of Bcl6, a transcription factor selectively expressed in TFH cells.  Bcl6 expression is regulated by interleukin-6 and interleukin-21.  Bcl6 overexpression induced TFH-related gene expression and inhibited other TH lineage cell differentiation in a DNA binding-dependent manner.  Moreover, Bcl6 deficiency in T cells resulted in impaired TFH cell development and germinal center reactions, and altered prodn. of other effector T cell subsets.  The authors' data thus illustrate that Bcl6 is required for programming of TFH cell generation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr25TQ6m1zfs7Vg90H21EOLACvtfcHk0lhxVbs03POl7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtValsr3F&md5=3921d8d2b28d9d4ddec78f8033600fab</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1176676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1176676%26sid%3Dliteratum%253Aachs%26aulast%3DNurieva%26aufirst%3DR.%2BI.%26aulast%3DChung%26aufirst%3DY.%26aulast%3DMartinez%26aufirst%3DG.%2BJ.%26aulast%3DYang%26aufirst%3DX.%2BO.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DMatskevitch%26aufirst%3DT.%2BD.%26aulast%3DWang%26aufirst%3DY.%2BH.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DBcl6%2520mediates%2520the%2520development%2520of%2520T%2520follicular%2520helper%2520cells%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D1001%26epage%3D1005%26doi%3D10.1126%2Fscience.1176676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Xin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, R.</span><span> </span><span class="NLM_article-title">RNA interference targeting Bcl-6 ameliorates experimental autoimmune myasthenia gravis in mice</span> <span class="citation_source-journal">Mol. Cell. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">85</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span><span class="refDoi"> DOI: 10.1016/j.mcn.2013.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.mcn.2013.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=24361642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlCgs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=85-94&author=N.+Xinauthor=L.+Fuauthor=Z.+Shaoauthor=M.+Guoauthor=X.+Zhangauthor=Y.+Zhangauthor=C.+Douauthor=S.+Zhengauthor=X.+Shenauthor=Y.+Yaoauthor=J.+Wangauthor=J.+Wangauthor=G.+Cuiauthor=Y.+Liuauthor=D.+Gengauthor=C.+Xiaoauthor=Z.+Zhangauthor=R.+Dong&title=RNA+interference+targeting+Bcl-6+ameliorates+experimental+autoimmune+myasthenia+gravis+in+mice&doi=10.1016%2Fj.mcn.2013.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">RNA interference targeting Bcl-6 ameliorates experimental autoimmune myasthenia gravis in mice</span></div><div class="casAuthors">Xin, Ning; Fu, Linlin; Shao, Zhen; Guo, Mingfeng; Zhang, Xiuying; Zhang, Yong; Dou, Changxin; Zheng, Shuangshuang; Shen, Xia; Yao, Yuanhu; Wang, Jiao; Wang, Jinhua; Cui, Guiyun; Liu, Yonghai; Geng, Deqin; Xiao, Chenghua; Zhang, Zunsheng; Dong, Ruiguo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Neuroscience</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">85-94</span>CODEN:
                <span class="NLM_cas:coden">MOCNED</span>;
        ISSN:<span class="NLM_cas:issn">1044-7431</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Follicular helper T (Tfh) cells are dedicated to providing help to B cells and are strongly assocd. with antibody-mediated autoimmune disease.  B cell lymphoma 6 (Bcl-6) is a key transcription factor of Tfh cells, and IL-21 is known to be a crit. cytokine produced by Tfh cells.  We silenced Bcl-6 gene expression using RNA interference (RNAi) delivered by a lentiviral vector, to evaluate the therapeutic role of Bcl-6 short hairpin RNAs (shRNAs) in exptl. autoimmune myasthenia gravis (EAMG).  Our data demonstrate that CD4(+)CXCR5(+)PD-1(+) Tfh cells, Bcl-6 and IL-21 were significantly increased in EAMG mice, compared with controls.  In addn., we found that frequencies of Tfh cells were pos. correlated with the levels of serum anti-AChR Ab.  In-vivo transduction of lenti-siRNA-Bcl6 ameliorates the severity of ongoing EAMG with decreased Tfh cells, Bcl-6 and IL-21 expression, and leads to decreased anti-AChR antibody levels.  Furthermore, we found that siRNA knockdown of Bcl-6 expression increases the expression of Th1(IFN-γ, T-bet) and Th2 markers (IL-4 and GATA3), but failed to alter the expression of Th17-related markers (RORγt, IL-17) and Treg markers (FoxP3).  Our study suggests that Tfh cells contribute to the antibody prodn. and could be one of the most important T cell subsets responsible for development and progression of EAMG or MG.  Bcl-6 provides a promising therapeutic target for immunotherapy not only for MG, but also for other antibody-mediated autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHKM7UyeZWh7Vg90H21EOLACvtfcHk0lhxVbs03POl7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlCgs70%253D&md5=a7167314504da585f9ef8625bfdaa458</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.mcn.2013.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mcn.2013.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DXin%26aufirst%3DN.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDou%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGeng%26aufirst%3DD.%26aulast%3DXiao%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DDong%26aufirst%3DR.%26atitle%3DRNA%2520interference%2520targeting%2520Bcl-6%2520ameliorates%2520experimental%2520autoimmune%2520myasthenia%2520gravis%2520in%2520mice%26jtitle%3DMol.%2520Cell.%2520Neurosci.%26date%3D2014%26volume%3D58%26spage%3D85%26epage%3D94%26doi%3D10.1016%2Fj.mcn.2013.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Parekh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prive, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span> </span><span class="NLM_article-title">Therapeutic targeting of the BCL6 oncogene for diffuse large B-cell lymphomas</span> <span class="citation_source-journal">Leuk. Lymphoma</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">874</span><span class="NLM_x">–</span> <span class="NLM_lpage">882</span><span class="refDoi"> DOI: 10.1080/10428190801895345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1080%2F10428190801895345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=18452090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsFaqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2008&pages=874-882&author=S.+Parekhauthor=G.+Priveauthor=A.+Melnick&title=Therapeutic+targeting+of+the+BCL6+oncogene+for+diffuse+large+B-cell+lymphomas&doi=10.1080%2F10428190801895345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the BCL6 oncogene for diffuse large B-cell lymphomas</span></div><div class="casAuthors">Parekh, Samir; Prive, Gilbert; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia & Lymphoma</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">874-882</span>CODEN:
                <span class="NLM_cas:coden">LELYEA</span>;
        ISSN:<span class="NLM_cas:issn">1042-8194</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  BCL6 is a transcriptional repressor often expressed constitutively in diffuse large B-cell lymphomas (DLBCL) due to mutations of its genomic locus.  BCL6 mediates aberrant survival, proliferation, genomic instability and differentiation blockade in DLBCL cells.  The biochem. study of BCL6 mediated gene repression has provided the basis for design of agents that inhibit BCL6 and kill lymphoma cells.  The repressor activity of the BCL6 BTB domain is particularly well defined from the structural standpoint.  Design of inhibitors targeting BCL6 BTB domain protein interaction surfaces appears to be an effective approach, which reactivates important BCL6 target genes and readily kills DLBCL cells.  Targeting other domains of BCL6 or using histone deacetylase inhibitors to overcome BCL6 mediated repression may also be useful.  Recent studies in DLBCL transcriptional signatures have revealed a subset of DLBCLs that are particularly dependent on BCL6 to maintain their survival and these patients could be candidates for clin. trials of BCL6 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsFT8NRceCBrVg90H21EOLACvtfcHk0lhxVbs03POl7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsFaqsrs%253D&md5=c528a71af6ba67489199c3bb1f6960fb</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1080%2F10428190801895345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10428190801895345%26sid%3Dliteratum%253Aachs%26aulast%3DParekh%26aufirst%3DS.%26aulast%3DPrive%26aufirst%3DG.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DTherapeutic%2520targeting%2520of%2520the%2520BCL6%2520oncogene%2520for%2520diffuse%2520large%2520B-cell%2520lymphomas%26jtitle%3DLeuk.%2520Lymphoma%26date%3D2008%26volume%3D49%26spage%3D874%26epage%3D882%26doi%3D10.1080%2F10428190801895345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Duy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurtz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shojaee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerchietti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaminathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klemm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kweon, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nahar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braig, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herzog, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jumaa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeffler, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heisterkamp, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graeber, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muschen, M.</span><span> </span><span class="NLM_article-title">BCL6 enables Ph+ acute lymphoblastic leukaemia cells to survive BCR-ABL1 kinase inhibition</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">473</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">388</span><span class="refDoi"> DOI: 10.1038/nature09883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1038%2Fnature09883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=21593872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtleltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=473&publication_year=2011&pages=384-388&author=C.+Duyauthor=C.+Hurtzauthor=S.+Shojaeeauthor=L.+Cerchiettiauthor=H.+Gengauthor=S.+Swaminathanauthor=L.+Klemmauthor=S.+M.+Kweonauthor=R.+Naharauthor=M.+Braigauthor=E.+Parkauthor=Y.+M.+Kimauthor=W.+K.+Hofmannauthor=S.+Herzogauthor=H.+Jumaaauthor=H.+P.+Koefflerauthor=J.+J.+Yuauthor=N.+Heisterkampauthor=T.+G.+Graeberauthor=H.+Wuauthor=B.+H.+Yeauthor=A.+Melnickauthor=M.+Muschen&title=BCL6+enables+Ph%2B+acute+lymphoblastic+leukaemia+cells+to+survive+BCR-ABL1+kinase+inhibition&doi=10.1038%2Fnature09883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">BCL6 enables Ph+ acute lymphoblastic leukaemia cells to survive BCR-ABL1 kinase inhibition</span></div><div class="casAuthors">Duy, Cihangir; Hurtz, Christian; Shojaee, Seyedmehdi; Cerchietti, Leandro; Geng, Huimin; Swaminathan, Srividya; Klemm, Lars; Kweon, Soo-mi; Nahar, Rahul; Braig, Melanie; Park, Eugene; Kim, Yong-mi; Hofmann, Wolf-Karsten; Herzog, Sebastian; Jumaa, Hassan; Koeffler, H. Phillip; Yu, J. Jessica; Heisterkamp, Nora; Graeber, Thomas G.; Wu, Hong; Ye, B. Hilda; Melnick, Ari; Mueschen, Markus</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">473</span>
        (<span class="NLM_cas:issue">7347</span>),
    <span class="NLM_cas:pages">384-388</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tyrosine kinase inhibitors (TKIs) are widely used to treat patients with leukemia driven by BCR-ABL1 (ref. 1) and other oncogenic tyrosine kinases.  Recent efforts have focused on developing more potent TKIs that also inhibit mutant tyrosine kinases.  However, even effective TKIs typically fail to eradicate leukemia-initiating cells (LICs), which often cause recurrence of leukemia after initially successful treatment.  Here we report the discovery of a novel mechanism of drug resistance, which is based on protective feedback signaling of leukemia cells in response to treatment with TKI.  We identify BCL6 as a central component of this drug-resistance pathway and demonstrate that targeted inhibition of BCL6 leads to eradication of drug-resistant and leukemia-initiating subclones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofwJ2MLGsWSLVg90H21EOLACvtfcHk0ljCrfrXq1YB5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtleltb4%253D&md5=24762f79c26b679694cd6fd524afed67</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature09883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09883%26sid%3Dliteratum%253Aachs%26aulast%3DDuy%26aufirst%3DC.%26aulast%3DHurtz%26aufirst%3DC.%26aulast%3DShojaee%26aufirst%3DS.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DKlemm%26aufirst%3DL.%26aulast%3DKweon%26aufirst%3DS.%2BM.%26aulast%3DNahar%26aufirst%3DR.%26aulast%3DBraig%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DY.%2BM.%26aulast%3DHofmann%26aufirst%3DW.%2BK.%26aulast%3DHerzog%26aufirst%3DS.%26aulast%3DJumaa%26aufirst%3DH.%26aulast%3DKoeffler%26aufirst%3DH.%2BP.%26aulast%3DYu%26aufirst%3DJ.%2BJ.%26aulast%3DHeisterkamp%26aufirst%3DN.%26aulast%3DGraeber%26aufirst%3DT.%2BG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DYe%26aufirst%3DB.%2BH.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DMuschen%26aufirst%3DM.%26atitle%3DBCL6%2520enables%2520Ph%252B%2520acute%2520lymphoblastic%2520leukaemia%2520cells%2520to%2520survive%2520BCR-ABL1%2520kinase%2520inhibition%26jtitle%3DNature%26date%3D2011%26volume%3D473%26spage%3D384%26epage%3D388%26doi%3D10.1038%2Fnature09883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Ahmad, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lax, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouchard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiang, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Licht, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prive, G. G.</span><span> </span><span class="NLM_article-title">Mechanism of SMRT corepressor recruitment by the BCL6 BTB domain</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1551</span><span class="NLM_x">–</span> <span class="NLM_lpage">1564</span><span class="refDoi"> DOI: 10.1016/S1097-2765(03)00454-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2FS1097-2765%2803%2900454-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=14690607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsVOksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=1551-1564&author=K.+F.+Ahmadauthor=A.+Melnickauthor=S.+Laxauthor=D.+Bouchardauthor=J.+Liuauthor=C.+L.+Kiangauthor=S.+Mayerauthor=S.+Takahashiauthor=J.+D.+Lichtauthor=G.+G.+Prive&title=Mechanism+of+SMRT+corepressor+recruitment+by+the+BCL6+BTB+domain&doi=10.1016%2FS1097-2765%2803%2900454-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of SMRT corepressor recruitment by the BCL6 BTB domain</span></div><div class="casAuthors">Ahmad, K. Farid; Melnick, Ari; Lax, Stuart; Bouchard, Denis; Liu, Jun; Kiang, Chih-li; Mayer, Sebastian; Takahashi, Shinichiro; Licht, Jonathan D.; Prive, Gilbert G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1551-1564</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BCL6 encodes a transcription factor that represses genes necessary for the terminal differentiation of lymphocytes within germinal centers, and the mis-regulated expression of this factor is strongly implicated in several types of B cell lymphoma.  The homodimeric BTB domain of BCL6 (also known as the POZ domain) is required for the repression activity of the protein and interacts directly with the SMRT and N-CoR corepressors that are found within large multiprotein histone deacetylase-contg. complexes.  We have identified a 17 residue fragment from SMRT that binds to the BCL6 BTB domain, and detd. the crystal structure of the complex to 2.2 Å.  Two SMRT fragments bind sym. to the BCL6 BTB homodimer and, in combination with biochem. and in vivo data, the structure provides insight into the basis of transcriptional repression by this crit. B cell lymphoma protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGmXOfYEe7Z7Vg90H21EOLACvtfcHk0ljCrfrXq1YB5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsVOksA%253D%253D&md5=1120b558b43f6714104eb8bed425f44b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2803%2900454-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252803%252900454-4%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DK.%2BF.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DLax%26aufirst%3DS.%26aulast%3DBouchard%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DKiang%26aufirst%3DC.%2BL.%26aulast%3DMayer%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DPrive%26aufirst%3DG.%2BG.%26atitle%3DMechanism%2520of%2520SMRT%2520corepressor%2520recruitment%2520by%2520the%2520BCL6%2520BTB%2520domain%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D12%26spage%3D1551%26epage%3D1564%26doi%3D10.1016%2FS1097-2765%2803%2900454-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Cerchietti, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghetu, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da Silva, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunting, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fares, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prive, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span> </span><span class="NLM_article-title">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">400</span><span class="NLM_x">–</span> <span class="NLM_lpage">411</span><span class="refDoi"> DOI: 10.1016/j.ccr.2009.12.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.ccr.2009.12.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=20385364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=400-411&author=L.+C.+Cerchiettiauthor=A.+F.+Ghetuauthor=X.+Zhuauthor=G.+F.+Da+Silvaauthor=S.+Zhongauthor=M.+Matthewsauthor=K.+L.+Buntingauthor=J.+M.+Poloauthor=C.+Faresauthor=C.+H.+Arrowsmithauthor=S.+N.+Yangauthor=M.+Garciaauthor=A.+Coopauthor=A.+D.+Mackerellauthor=G.+G.+Priveauthor=A.+Melnick&title=A+small-molecule+inhibitor+of+BCL6+kills+DLBCL+cells+in+vitro+and+in+vivo&doi=10.1016%2Fj.ccr.2009.12.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Ghetu, Alexandru F.; Zhu, Xiao; Da Silva, Gustavo F.; Zhong, Shijun; Matthews, Marilyn; Bunting, Karen L.; Polo, Jose M.; Fares, Christophe; Arrowsmith, Cheryl H.; Yang, Shao Ning; Garcia, Monica; Coop, Andrew; MacKerell, Alexander D., Jr.; Prive, Gilbert G.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">400-411</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most frequently involved oncogene in diffuse large B cell lymphoma (DLBCL).  We combined computer-aided drug design with functional assays to identify low-mol.-wt. compds. that bind to the corepressor binding groove of the BCL6 BTB domain.  One such compd. disrupted BCL6/corepressor complexes in vitro and in vivo, and was obsd. by X-ray crystallog. and NMR to bind the crit. site within the BTB groove.  This compd. could induce expression of BCL6 target genes and kill BCL6-pos. DLBCL cell lines.  In xenotransplantation expts., the compd. was nontoxic and potently suppressed DLBCL tumors in vivo.  The compd. also killed primary DLBCLs from human patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdcScjNPjhiLVg90H21EOLACvtfcHk0li5w7VffPOF6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D&md5=64f30a813c26394b69da6de95991f8dd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.12.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.12.050%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DGhetu%26aufirst%3DA.%2BF.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDa%2BSilva%26aufirst%3DG.%2BF.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DMatthews%26aufirst%3DM.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DFares%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DGarcia%26aufirst%3DM.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DPrive%26aufirst%3DG.%2BG.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520small-molecule%2520inhibitor%2520of%2520BCL6%2520kills%2520DLBCL%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26spage%3D400%26epage%3D411%26doi%3D10.1016%2Fj.ccr.2009.12.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Cardenas, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beguelin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teater, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldstein, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oswald, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaknovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbott, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elemento, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerchietti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cierpicki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A. M.</span><span> </span><span class="NLM_article-title">Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">3351</span><span class="NLM_x">–</span> <span class="NLM_lpage">3362</span><span class="refDoi"> DOI: 10.1172/JCI85795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1172%2FJCI85795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=27482887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BC2s3mvFagsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2016&pages=3351-3362&author=M.+G.+Cardenasauthor=W.+Yuauthor=W.+Beguelinauthor=M.+R.+Teaterauthor=H.+Gengauthor=R.+L.+Goldsteinauthor=E.+Oswaldauthor=K.+Hatziauthor=S.+N.+Yangauthor=J.+Cohenauthor=R.+Shaknovichauthor=K.+Vanommeslaegheauthor=H.+Chengauthor=D.+Liangauthor=H.+J.+Choauthor=J.+Abbottauthor=W.+Tamauthor=W.+Duauthor=J.+P.+Leonardauthor=O.+Elementoauthor=L.+Cerchiettiauthor=T.+Cierpickiauthor=F.+Xueauthor=A.+D.+MacKerellauthor=A.+M.+Melnick&title=Rationally+designed+BCL6+inhibitors+target+activated+B+cell+diffuse+large+B+cell+lymphoma&doi=10.1172%2FJCI85795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</span></div><div class="casAuthors">Cardenas Mariano G; Yu Wenbo; Beguelin Wendy; Teater Matthew R; Geng Huimin; Goldstein Rebecca L; Oswald Erin; Hatzi Katerina; Yang Shao-Ning; Cohen Joanna; Shaknovich Rita; Vanommeslaeghe Kenno; Cheng Huimin; Liang Dongdong; Cho Hyo Je; Abbott Joshua; Tam Wayne; Du Wei; Leonard John P; Elemento Olivier; Cerchietti Leandro; Cierpicki Tomasz; Xue Fengtian; MacKerell Alexander D Jr; Melnick Ari M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3351-62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Diffuse large B cell lymphomas (DLBCLs) arise from proliferating B cells transiting different stages of the germinal center reaction.  In activated B cell DLBCLs (ABC-DLBCLs), a class of DLBCLs that respond poorly to current therapies, chromosomal translocations and amplification lead to constitutive expression of the B cell lymphoma 6 (BCL6) oncogene.  The role of BCL6 in maintaining these lymphomas has not been investigated.  Here, we designed small-molecule inhibitors that display higher affinity for BCL6 than its endogenous corepressor ligands to evaluate their therapeutic efficacy for targeting ABC-DLBCL.  We used an in silico drug design functional-group mapping approach called SILCS to create a specific BCL6 inhibitor called FX1 that has 10-fold greater potency than endogenous corepressors and binds an essential region of the BCL6 lateral groove.  FX1 disrupted formation of the BCL6 repression complex, reactivated BCL6 target genes, and mimicked the phenotype of mice engineered to express BCL6 with corepressor binding site mutations.  Low doses of FX1 induced regression of established tumors in mice bearing DLBCL xenografts.  Furthermore, FX1 suppressed ABC-DLBCL cells in vitro and in vivo, as well as primary human ABC-DLBCL specimens ex vivo.  These findings indicate that ABC-DLBCL is a BCL6-dependent disease that can be targeted by rationally designed inhibitors that exceed the binding affinity of natural BCL6 ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAVUzYiXt5ZL5XHngOXiV_fW6udTcc2eY65XIGeSwYsbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3mvFagsw%253D%253D&md5=90a313fd7adce1922773212f0960bad7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1172%2FJCI85795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI85795%26sid%3Dliteratum%253Aachs%26aulast%3DCardenas%26aufirst%3DM.%2BG.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBeguelin%26aufirst%3DW.%26aulast%3DTeater%26aufirst%3DM.%2BR.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DGoldstein%26aufirst%3DR.%2BL.%26aulast%3DOswald%26aufirst%3DE.%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DCohen%26aufirst%3DJ.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DD.%26aulast%3DCho%26aufirst%3DH.%2BJ.%26aulast%3DAbbott%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DW.%26aulast%3DDu%26aufirst%3DW.%26aulast%3DLeonard%26aufirst%3DJ.%2BP.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DXue%26aufirst%3DF.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26atitle%3DRationally%2520designed%2520BCL6%2520inhibitors%2520target%2520activated%2520B%2520cell%2520diffuse%2520large%2520B%2520cell%2520lymphoma%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2016%26volume%3D126%26spage%3D3351%26epage%3D3362%26doi%3D10.1172%2FJCI85795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Cerchietti, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaknovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chadburn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowdy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span> </span><span class="NLM_article-title">A peptomimetic inhibitor of BCL6 with potent antilymphoma effects in vitro and in vivo</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">3397</span><span class="NLM_x">–</span> <span class="NLM_lpage">3405</span><span class="refDoi"> DOI: 10.1182/blood-2008-07-168773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1182%2Fblood-2008-07-168773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=18927431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslCkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=3397-3405&author=L.+C.+Cerchiettiauthor=S.+N.+Yangauthor=R.+Shaknovichauthor=K.+Hatziauthor=J.+M.+Poloauthor=A.+Chadburnauthor=S.+F.+Dowdyauthor=A.+Melnick&title=A+peptomimetic+inhibitor+of+BCL6+with+potent+antilymphoma+effects+in+vitro+and+in+vivo&doi=10.1182%2Fblood-2008-07-168773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A peptomimetic inhibitor of BCL6 with potent antilymphoma effects in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Yang, Shao Ning; Shaknovich, Rita; Hatzi, Katerina; Polo, Jose M.; Chadburn, Amy; Dowdy, Steven F.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3397-3405</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most commonly involved oncogene in diffuse large B-cell lymphomas (DLBCLs).  BCL6 lymphomagenic activity is dependent on its ability to recruit corepressor proteins to a unique binding site on its N-terminal BTB domain.  A recombinant peptide fragment of the SMRT (silencing mediator for retinoid and thyroid hormone receptor) corepressor that blocks this site can inhibit BCL6 biol. functions.  Shortening and conversion of this peptide to D-amino acid and retro configuration as well as the addn. of a fusogenic motif yielded a far more potent and stable BCL6 inhibitor that still retained the specificity of the original SMRT fragment.  Like the L-peptide, retroinverso BCL6 peptide inhibitor (RI-BPI) selectively killed BCR rather than OxPhos-type DLBCL cells.  The RI-BPI could recapitulate the failure to form germinal centers seen in BCL6 null mice yet was nontoxic and non-immunogenic even when administered for up to 52 wk.  RI-BPI showed superior duration of tissue penetration and could accordingly powerfully suppress the growth of human DLBCLs xenografts in a dose-dependent manner.  Finally, RI-BPI could kill primary human DLBCL cells but had no effect on normal lymphoid tissue or other tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxsckch1D41rVg90H21EOLACvtfcHk0lg3FBvQHq4SeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslCkurw%253D&md5=8a12df08b153ba0712b5d06f9bc550c2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1182%2Fblood-2008-07-168773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2008-07-168773%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DChadburn%26aufirst%3DA.%26aulast%3DDowdy%26aufirst%3DS.%2BF.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520peptomimetic%2520inhibitor%2520of%2520BCL6%2520with%2520potent%2520antilymphoma%2520effects%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26spage%3D3397%26epage%3D3405%26doi%3D10.1182%2Fblood-2008-07-168773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Evans, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goult, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairall, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamieson, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko Ferrigno, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwabe, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, S. D.</span><span> </span><span class="NLM_article-title">The ansamycin antibiotic, rifamycin SV, inhibits BCL6 transcriptional repression and forms a complex with the BCL6-BTB/POZ domain</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e90889</span><span class="refDoi"> DOI: 10.1371/journal.pone.0090889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1371%2Fjournal.pone.0090889" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e90889&author=S.+E.+Evansauthor=B.+T.+Goultauthor=L.+Fairallauthor=A.+G.+Jamiesonauthor=P.+Ko+Ferrignoauthor=R.+Fordauthor=J.+W.+Schwabeauthor=S.+D.+Wagner&title=The+ansamycin+antibiotic%2C+rifamycin+SV%2C+inhibits+BCL6+transcriptional+repression+and+forms+a+complex+with+the+BCL6-BTB%2FPOZ+domain&doi=10.1371%2Fjournal.pone.0090889"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0090889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0090889%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DS.%2BE.%26aulast%3DGoult%26aufirst%3DB.%2BT.%26aulast%3DFairall%26aufirst%3DL.%26aulast%3DJamieson%26aufirst%3DA.%2BG.%26aulast%3DKo%2BFerrigno%26aufirst%3DP.%26aulast%3DFord%26aufirst%3DR.%26aulast%3DSchwabe%26aufirst%3DJ.%2BW.%26aulast%3DWagner%26aufirst%3DS.%2BD.%26atitle%3DThe%2520ansamycin%2520antibiotic%252C%2520rifamycin%2520SV%252C%2520inhibits%2520BCL6%2520transcriptional%2520repression%2520and%2520forms%2520a%2520complex%2520with%2520the%2520BCL6-BTB%252FPOZ%2520domain%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De90889%26doi%3D10.1371%2Fjournal.pone.0090889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Sakamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sogabe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakai, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asano, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imaeda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakamoto, J. I.</span><span> </span><span class="NLM_article-title">Discovery of high-affinity BCL6-binding peptide and its structure-activity relationship</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">482</span><span class="NLM_x">, </span> <span class="NLM_fpage">310</span><span class="NLM_x">–</span> <span class="NLM_lpage">316</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2016.11.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.bbrc.2016.11.060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=27856253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGgt7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2017&pages=310-316&author=K.+Sakamotoauthor=S.+Sogabeauthor=Y.+Kamadaauthor=N.+Sakaiauthor=K.+Asanoauthor=M.+Yoshimatsuauthor=K.+Idaauthor=Y.+Imaedaauthor=J.+I.+Sakamoto&title=Discovery+of+high-affinity+BCL6-binding+peptide+and+its+structure-activity+relationship&doi=10.1016%2Fj.bbrc.2016.11.060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of high-affinity BCL6-binding peptide and its structure-activity relationship</span></div><div class="casAuthors">Sakamoto, Kotaro; Sogabe, Satoshi; Kamada, Yusuke; Sakai, Nozomu; Asano, Kouhei; Yoshimatsu, Mie; Ida, Kou; Imaeda, Yasuhiro; Sakamoto, Jun-ichi</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">310-316</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">B cell lymphoma 6 (BCL6) is a transcriptional repressor that interacts with its corepressors BcoR and SMRT.  Since this protein-protein interaction (PPI) induces activation and differentiation of B lymphocytes, BCL6 has been an attractive drug target for potential autoimmune disease treatments.  Here we report a novel BCL6 inhibitory peptide, F1324 (Ac-LWYTDIRMSWRVP-OH), which we discovered using phage display technol.; we also discuss this peptide's structure-activity relationship (SAR).  For BCL6(5-129) binding, KD and IC50 values of F1324 were 0.57 nM and 1 nM according to the results of an SPR anal. and cell-free ELISA assay, resp.  In contrast, BcoR(Arg498-514Pro) and SMRT(Leu1422-Arg1438) exhibited relatively weak micromole-order binding to BCL6.  Furthermore, Fusion protein AcGFP-F1324 transiently expressed in HEK293T cells inhibited intracellular PPI in cell-based M2H assay.  By examn. of the truncation and fragmentation of F1324, the C-terminal sequence WRVP, which is similar to the BcoR(509-512) sequence WVVP, was identified as being crit. for BCL6 binding.  In addn., subsequent single-crystal X-ray diffraction anal. of F1324/BCL6(5-129) complex revealed that the high affinity of F1324 was caused by effective interaction of its side chains while its main chain structure was similar to that of BcoR(Arg498-514Pro).  To our knowledge, F1324 is the strongest BCL6-binding peptide yet reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7xCi3L_jL-bVg90H21EOLACvtfcHk0lg3FBvQHq4SeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGgt7nJ&md5=a543acd613f9b63efa9c814a75e6b5f6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.11.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.11.060%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DSakai%26aufirst%3DN.%26aulast%3DAsano%26aufirst%3DK.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DIda%26aufirst%3DK.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DSakamoto%26aufirst%3DJ.%2BI.%26atitle%3DDiscovery%2520of%2520high-affinity%2520BCL6-binding%2520peptide%2520and%2520its%2520structure-activity%2520relationship%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2017%26volume%3D482%26spage%3D310%26epage%3D316%26doi%3D10.1016%2Fj.bbrc.2016.11.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahnke, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lg3FBvQHq4SeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Ludlow, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saini, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tickle, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Detection of secondary binding sites in proteins using fragment screening</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">15910</span><span class="NLM_x">–</span> <span class="NLM_lpage">15915</span><span class="refDoi"> DOI: 10.1073/pnas.1518946112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1073%2Fpnas.1518946112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=26655740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15910-15915&author=R.+F.+Ludlowauthor=M.+L.+Verdonkauthor=H.+K.+Sainiauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Detection+of+secondary+binding+sites+in+proteins+using+fragment+screening&doi=10.1073%2Fpnas.1518946112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of secondary binding sites in proteins using fragment screening</span></div><div class="casAuthors">Ludlow, R. Frederick; Verdonk, Marcel L.; Saini, Harpreet K.; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">15910-15915</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Proteins need to be tightly regulated as they control biol. processes in most normal cellular functions.  The precise mechanisms of regulation are rarely completely understood but can involve binding of endogenous ligands and/or partner proteins at specific locations on a protein that can modulate function.  Often, these addnl. secondary binding sites appear sep. to the primary binding site, which, e.g., for an enzyme may bind a substrate.  In previous work, the authors uncovered several examples in which secondary binding sites were discovered on proteins using fragment screening approaches.  In each case, the authors were able to establish that the newly identified secondary binding site was biol. relevant as it was able to modulate function by the binding of a small mol.  Here, the authors investigated how often secondary binding sites are located on proteins by analyzing 24 protein targets for which they performed a fragment screen using x-ray crystallog.  The anal. showed that, surprisingly, the majority of proteins contained secondary binding sites based on their ability to bind fragments.  Furthermore, sequence anal. of these previously unknown sites indicated high conservation, which suggested that they may have a biol. function, perhaps via an allosteric mechanism.  Comparing the physicochem. properties of the secondary sites with known primary ligand-binding sites also showed broad similarities indicating that many of the secondary sites may be druggable in nature with small mols. that could provide new opportunities to modulate potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ITF1w230fbVg90H21EOLACvtfcHk0ligr0Z9Hc32Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP&md5=9e5f15595263dc049c713d5a3f5912a1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1518946112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1518946112%26sid%3Dliteratum%253Aachs%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DSaini%26aufirst%3DH.%2BK.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520secondary%2520binding%2520sites%2520in%2520proteins%2520using%2520fragment%2520screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D15910%26epage%3D15915%26doi%3D10.1073%2Fpnas.1518946112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Jordan, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittington, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartberger, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sickmier, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, T.</span><span> </span><span class="NLM_article-title">Fragment-linking approach using (19)F NMR spectroscopy to obtain highly potent and selective inhibitors of beta-secretase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">3732</span><span class="NLM_x">–</span> <span class="NLM_lpage">3749</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01917</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01917" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFeitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3732-3749&author=J.+B.+Jordanauthor=D.+A.+Whittingtonauthor=M.+D.+Bartbergerauthor=E.+A.+Sickmierauthor=K.+Chenauthor=Y.+Chengauthor=T.+Judd&title=Fragment-linking+approach+using+%2819%29F+NMR+spectroscopy+to+obtain+highly+potent+and+selective+inhibitors+of+beta-secretase&doi=10.1021%2Facs.jmedchem.5b01917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Linking Approach Using 19F NMR Spectroscopy To Obtain Highly Potent and Selective Inhibitors of β-Secretase</span></div><div class="casAuthors">Jordan, John B.; Whittington, Douglas A.; Bartberger, Michael D.; Sickmier, E. Allen; Chen, Kui; Cheng, Yuan; Judd, Ted</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3732-3749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) has become a widely used tool in small-mol. drug discovery efforts.  One of the most commonly used biophys. methods in detecting weak binding of fragments is NMR (NMR) spectroscopy.  In particular, FBDD performed with 19F NMR-based methods has been shown to provide several advantages over 1H NMR using traditional magnetization-transfer and/or two-dimensional methods.  Here, we demonstrate the utility and power of 19F-based fragment screening by detailing the identification of a second-site fragment through 19F NMR screening that binds to a specific pocket of the aspartic acid protease, β-secretase (BACE-1).  The identification of this second-site fragment allowed the undertaking of a fragment-linking approach, which ultimately yielded a mol. exhibiting a more than 360-fold increase in potency while maintaining reasonable ligand efficiency and gaining much improved selectivity over cathepsin-D (CatD).  X-ray crystallog. studies of the mols. demonstrated that the linked fragments exhibited binding modes consistent with those predicted from the targeted screening approach, through-space NMR data, and mol. modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq95M4k5aaQurVg90H21EOLACvtfcHk0ligr0Z9Hc32Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFeitr8%253D&md5=6bd6021938c555d8c7083549e4b49a2e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01917%26sid%3Dliteratum%253Aachs%26aulast%3DJordan%26aufirst%3DJ.%2BB.%26aulast%3DWhittington%26aufirst%3DD.%2BA.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DSickmier%26aufirst%3DE.%2BA.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DJudd%26aufirst%3DT.%26atitle%3DFragment-linking%2520approach%2520using%2520%252819%2529F%2520NMR%2520spectroscopy%2520to%2520obtain%2520highly%2520potent%2520and%2520selective%2520inhibitors%2520of%2520beta-secretase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3732%26epage%3D3749%26doi%3D10.1021%2Facs.jmedchem.5b01917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Kohlmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zech, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Squillace, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Commodore, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenfield, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodd, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miret, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakespeare, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalgarno, D. C.</span><span> </span><span class="NLM_article-title">Fragment growing and linking lead to novel nanomolar lactate dehydrogenase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1023</span><span class="NLM_x">–</span> <span class="NLM_lpage">1040</span><span class="refDoi"> DOI: 10.1021/jm3014844</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3014844" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3sXms1OlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1023-1040&author=A.+Kohlmannauthor=S.+G.+Zechauthor=F.+Liauthor=T.+Zhouauthor=R.+M.+Squillaceauthor=L.+Commodoreauthor=M.+T.+Greenfieldauthor=X.+Luauthor=D.+P.+Millerauthor=W.+S.+Huangauthor=J.+Qiauthor=R.+M.+Thomasauthor=Y.+Wangauthor=S.+Zhangauthor=R.+Doddauthor=S.+Liuauthor=R.+Xuauthor=Y.+Xuauthor=J.+J.+Miretauthor=V.+Riveraauthor=T.+Clacksonauthor=W.+C.+Shakespeareauthor=X.+Zhuauthor=D.+C.+Dalgarno&title=Fragment+growing+and+linking+lead+to+novel+nanomolar+lactate+dehydrogenase+inhibitors&doi=10.1021%2Fjm3014844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Growing and Linking Lead to Novel Nanomolar Lactate Dehydrogenase Inhibitors</span></div><div class="casAuthors">Kohlmann, Anna; Zech, Stephan G.; Li, Feng; Zhou, Tianjun; Squillace, Rachel M.; Commodore, Lois; Greenfield, Matthew T.; Lu, Xiaohui; Miller, David P.; Huang, Wei-Sheng; Qi, Jiwei; Thomas, R. Mathew; Wang, Yihan; Zhang, Sen; Dodd, Rory; Liu, Shuangying; Xu, Rongsong; Xu, Yongjin; Miret, Juan J.; Rivera, Victor; Clackson, Tim; Shakespeare, William C.; Zhu, Xiaotian; Dalgarno, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1023-1040</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lactate dehydrogenase A (LDH-A) catalyzes the interconversion of lactate and pyruvate in the glycolysis pathway.  Cancer cells rely heavily on glycolysis instead of oxidative phosphorylation to generate ATP, a phenomenon known as the Warburg effect.  The inhibition of LDH-A by small mols. is therefore of interest for potential cancer treatments.  We describe the identification and optimization of LDH-A inhibitors by fragment-based drug discovery.  We applied ligand based NMR screening to identify low affinity fragments binding to LDH-A.  The dissocn. consts. (Kd) and enzyme inhibition (IC50) of fragment hits were measured by surface plasmon resonance (SPR) and enzyme assays, resp.  The binding modes of selected fragments were investigated by X-ray crystallog.  Fragment growing and linking, followed by chem. optimization, resulted in nanomolar LDH-A inhibitors that demonstrated stoichiometric binding to LDH-A.  Selected mols. inhibited lactate prodn. in cells, suggesting target-specific inhibition in cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pFkSoe2_ALVg90H21EOLACvtfcHk0ljQNBjb38pbcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXms1OlsA%253D%253D&md5=3a6b0b46318ae2d6c2a7c34e1575ada6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm3014844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3014844%26sid%3Dliteratum%253Aachs%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DZech%26aufirst%3DS.%2BG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DSquillace%26aufirst%3DR.%2BM.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DGreenfield%26aufirst%3DM.%2BT.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DMiller%26aufirst%3DD.%2BP.%26aulast%3DHuang%26aufirst%3DW.%2BS.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DDodd%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMiret%26aufirst%3DJ.%2BJ.%26aulast%3DRivera%26aufirst%3DV.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26atitle%3DFragment%2520growing%2520and%2520linking%2520lead%2520to%2520novel%2520nanomolar%2520lactate%2520dehydrogenase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1023%26epage%3D1040%26doi%3D10.1021%2Fjm3014844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Larsson, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Then, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panicker, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sangthongpitag, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendharkar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tai, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheong, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulsen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alam, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span> </span><span class="NLM_article-title">Fragment-based ligand design of novel potent inhibitors of tankyrases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">4497</span><span class="NLM_x">–</span> <span class="NLM_lpage">4508</span><span class="refDoi"> DOI: 10.1021/jm400211f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400211f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFartr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=4497-4508&author=E.+A.+Larssonauthor=A.+Janssonauthor=F.+M.+Ngauthor=S.+W.+Thenauthor=R.+Panickerauthor=B.+Liuauthor=K.+Sangthongpitagauthor=V.+Pendharkarauthor=S.+J.+Taiauthor=J.+Hillauthor=C.+Danauthor=S.+Y.+Hoauthor=W.+W.+Cheongauthor=A.+Poulsenauthor=S.+Blanchardauthor=G.+R.+Linauthor=J.+Alamauthor=T.+H.+Kellerauthor=P.+Nordlund&title=Fragment-based+ligand+design+of+novel+potent+inhibitors+of+tankyrases&doi=10.1021%2Fjm400211f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based ligand design of novel potent inhibitors of tankyrases</span></div><div class="casAuthors">Larsson, E. Andreas; Jansson, Anna; Ng, Fui Mee; Then, Siew Wen; Panicker, Resmi; Liu, Boping; Sangthongpitag, Kanda; Pendharkar, Vishal; Tai, Shi Jing; Hill, Jeffrey; Dan, Chen; Ho, Soo Yei; Cheong, Wei Wen; Poulsen, Anders; Blanchard, Stephanie; Lin, Grace Ruiting; Alam, Jenefer; Keller, Thomas H.; Nordlund, Par</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4497-4508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tankyrases constitute potential drug targets for cancer and myelin-degrading diseases.  We have applied a structure- and biophysics-driven fragment-based ligand design strategy to discover a novel family of potent inhibitors for human tankyrases.  Biophys. screening based on a thermal shift assay identified highly efficient fragments binding in the nicotinamide-binding site, a local hot spot for fragment binding.  Evolution of the fragment hit 4-methyl-1,2-dihydroquinolin-2-one (2) along its 7-vector yields dramatic affinity improvements in the first cycle of expansion.  A crystal structure of 7-(2-fluorophenyl)-4-methylquinolin-2(1H)-one (11) reveals that the nonplanar compd. extends with its fluorine atom into a pocket, which coincides with a region of the active site where structural differences are seen between tankyrases and other poly(ADP-ribose) polymerase (PARP) family members.  A further cycle of optimization yielded compds. with affinities and IC50 values in the low nanomolar range and with good soly., PARP selectivity, and ligand efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQxampX4n-xLVg90H21EOLACvtfcHk0ljQNBjb38pbcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFartr4%253D&md5=4cd39f8715bfc51807129bb04da1e6f0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm400211f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400211f%26sid%3Dliteratum%253Aachs%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DJansson%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DF.%2BM.%26aulast%3DThen%26aufirst%3DS.%2BW.%26aulast%3DPanicker%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSangthongpitag%26aufirst%3DK.%26aulast%3DPendharkar%26aufirst%3DV.%26aulast%3DTai%26aufirst%3DS.%2BJ.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DHo%26aufirst%3DS.%2BY.%26aulast%3DCheong%26aufirst%3DW.%2BW.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DBlanchard%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DG.%2BR.%26aulast%3DAlam%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DFragment-based%2520ligand%2520design%2520of%2520novel%2520potent%2520inhibitors%2520of%2520tankyrases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D4497%26epage%3D4508%26doi%3D10.1021%2Fjm400211f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Wielens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headey, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulder, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolezal, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deadman, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalmers, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peat, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scanlon, M. J.</span><span> </span><span class="NLM_article-title">Parallel screening of low molecular weight fragment libraries: do differences in methodology affect hit identification?</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">147</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1177/1087057112465979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057112465979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=23139382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BC3s7itFGluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=147-159&author=J.+Wielensauthor=S.+J.+Headeyauthor=D.+I.+Rhodesauthor=R.+J.+Mulderauthor=O.+Dolezalauthor=J.+J.+Deadmanauthor=J.+Newmanauthor=D.+K.+Chalmersauthor=M.+W.+Parkerauthor=T.+S.+Peatauthor=M.+J.+Scanlon&title=Parallel+screening+of+low+molecular+weight+fragment+libraries%3A+do+differences+in+methodology+affect+hit+identification%3F&doi=10.1177%2F1087057112465979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel screening of low molecular weight fragment libraries: do differences in methodology affect hit identification?</span></div><div class="casAuthors">Wielens Jerome; Headey Stephen J; Rhodes David I; Mulder Roger J; Dolezal Olan; Deadman John J; Newman Janet; Chalmers David K; Parker Michael W; Peat Thomas S; Scanlon Martin J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">147-59</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment screening is becoming widely accepted as a technique to identify hit compounds for the development of novel lead compounds.  In neighboring laboratories, we have recently, and independently, performed a fragment screening campaign on the HIV-1 integrase core domain (IN) using similar commercially purchased fragment libraries.  The two campaigns used different screening methods for the preliminary identification of fragment hits; one used saturation transfer difference nuclear magnetic resonance spectroscopy (STD-NMR), and the other used surface plasmon resonance (SPR) spectroscopy.  Both initial screens were followed by X-ray crystallography.  Using the STD-NMR/X-ray approach, 15 IN/fragment complexes were identified, whereas the SPR/X-ray approach found 6 complexes.  In this article, we compare the approaches that were taken by each group and the results obtained, and we look at what factors could potentially influence the final results.  We find that despite using different approaches with little overlap of initial hits, both approaches identified binding sites on IN that provided a basis for fragment-based lead discovery and further lead development.  Comparison of hits identified in the two studies highlights a key role for both the conditions under which fragment binding is measured and the criteria selected to classify hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaYsG71Pu6x669lAbTj9rGfW6udTcc2eZkHwR8av4kXbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7itFGluw%253D%253D&md5=9061e308aecbffc35d50a29b5b682283</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1087057112465979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112465979%26sid%3Dliteratum%253Aachs%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DHeadey%26aufirst%3DS.%2BJ.%26aulast%3DRhodes%26aufirst%3DD.%2BI.%26aulast%3DMulder%26aufirst%3DR.%2BJ.%26aulast%3DDolezal%26aufirst%3DO.%26aulast%3DDeadman%26aufirst%3DJ.%2BJ.%26aulast%3DNewman%26aufirst%3DJ.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DPeat%26aufirst%3DT.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DParallel%2520screening%2520of%2520low%2520molecular%2520weight%2520fragment%2520libraries%253A%2520do%2520differences%2520in%2520methodology%2520affect%2520hit%2520identification%253F%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D147%26epage%3D159%26doi%3D10.1177%2F1087057112465979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browner, M. F.</span><span> </span><span class="NLM_article-title">Surface plasmon resonance based assay for the detection and characterization of promiscuous inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">580</span><span class="refDoi"> DOI: 10.1021/jm700952v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700952v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=574-580&author=A.+M.+Giannettiauthor=B.+D.+Kochauthor=M.+F.+Browner&title=Surface+plasmon+resonance+based+assay+for+the+detection+and+characterization+of+promiscuous+inhibitors&doi=10.1021%2Fjm700952v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Plasmon Resonance Based Assay for the Detection and Characterization of Promiscuous Inhibitors</span></div><div class="casAuthors">Giannetti, Anthony M.; Koch, Bruce D.; Browner, Michelle F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-580</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous binders achieve enzyme inhibition using a nonspecific aggregation-type binding mechanism to proteins.  These compds. are a source of false-pos. hits in biochem. inhibition assays and should be removed from screening hit lists because they are not good candidates to initiate medicinal chem. programs.  The authors introduce a robust approach to identify these mols. early in the lead generation process using real time surface plasmon resonance based biosensors to observe the behavior of the binding interactions between promiscuous compds. and proteins.  Furthermore, the time resoln. of the assay reveals a no. of distinct mechanisms that promiscuous compds. employ to inhibit enzyme function and indicate that the type of mechanism can vary depending on the protein target.  A classification scheme for these compds. is presented that can be used to rapidly characterize the hits from high-throughput screens and eliminate compds. with a nonspecific mechanism of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVTh2kYyROf7Vg90H21EOLACvtfcHk0lhtsRkImARifw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D&md5=b2869c5be1cd6f455503f8e812ed6f23</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm700952v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700952v%26sid%3Dliteratum%253Aachs%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DKoch%26aufirst%3DB.%2BD.%26aulast%3DBrowner%26aufirst%3DM.%2BF.%26atitle%3DSurface%2520plasmon%2520resonance%2520based%2520assay%2520for%2520the%2520detection%2520and%2520characterization%2520of%2520promiscuous%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D574%26epage%3D580%26doi%3D10.1021%2Fjm700952v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Retra, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irth, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Muijlwijk-Koezen, J. E.</span><span> </span><span class="NLM_article-title">Surface plasmon resonance biosensor analysis as a useful tool in FBDD</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e181</span><span class="NLM_x">–</span> <span class="NLM_lpage">e187</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2010.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.ddtec.2010.11.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=e181-e187&author=K.+Retraauthor=H.+Irthauthor=J.+E.+van+Muijlwijk-Koezen&title=Surface+plasmon+resonance+biosensor+analysis+as+a+useful+tool+in+FBDD&doi=10.1016%2Fj.ddtec.2010.11.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2010.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2010.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DRetra%26aufirst%3DK.%26aulast%3DIrth%26aufirst%3DH.%26aulast%3Dvan%2BMuijlwijk-Koezen%26aufirst%3DJ.%2BE.%26atitle%3DSurface%2520plasmon%2520resonance%2520biosensor%2520analysis%2520as%2520a%2520useful%2520tool%2520in%2520FBDD%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2010%26volume%3D7%26spage%3De181%26epage%3De187%26doi%3D10.1016%2Fj.ddtec.2010.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Chimenti, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulfer, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neitz, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renslo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arkin, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, M. J.</span><span> </span><span class="NLM_article-title">A fragment-based ligand screen against part of a large protein machine: the ND1 domains of the AAA+ ATPase p97/VCP</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">788</span><span class="NLM_x">–</span> <span class="NLM_lpage">800</span><span class="refDoi"> DOI: 10.1177/1087057115570550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057115570550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=25690569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsb3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=788-800&author=M.+S.+Chimentiauthor=S.+L.+Bulferauthor=R.+J.+Neitzauthor=A.+R.+Rensloauthor=M.+P.+Jacobsonauthor=T.+L.+Jamesauthor=M.+R.+Arkinauthor=M.+J.+Kelly&title=A+fragment-based+ligand+screen+against+part+of+a+large+protein+machine%3A+the+ND1+domains+of+the+AAA%2B+ATPase+p97%2FVCP&doi=10.1177%2F1087057115570550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A fragment-based ligand screen against part of a large protein machine: the ND1 domains of the AAA+ ATPase p97/VCP</span></div><div class="casAuthors">Chimenti, Michael S.; Bulfer, Stacie L.; Neitz, R. Jeffrey; Renslo, Adam R.; Jacobson, Matthew P.; James, Thomas L.; Arkin, Michelle R.; Kelly, Mark J. S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">788-800</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The ubiquitous AAA+ ATPase p97 functions as a dynamic mol. machine driving several cellular processes.  It is essential in regulating protein homeostasis, and it represents a potential drug target for cancer, particularly when there is a greater reliance on the endoplasmic reticulum-assocd. protein degrdn. pathway and ubiquitin-proteasome pathway to degrade an overabundance of secreted proteins.  Here, we report a case study for using fragment-based ligand design approaches against this large and dynamic hexamer, which has multiple potential binding sites for small mols.  A screen of a fragment library was conducted by surface plasmon resonance (SPR) and followed up by NMR (NMR), two complementary biophys. techniques.  Virtual screening was also carried out to examine possible binding sites for the exptl. hits and evaluate the potential utility of fragment docking for this target.  Out of this effort, 13 fragments were discovered that showed reversible binding with affinities between 140 μM and 1 mM, binding stoichiometries of 1:1 or 2:1, and good ligand efficiencies.  Structural data for fragment-protein interactions were obtained with residue-specific [U-2H] 13CH3-methyl-labeling NMR strategies, and these data were compared to poses from docking.  The combination of virtual screening, SPR, and NMR enabled us to find and validate a no. of interesting fragment hits and allowed us to gain an understanding of the structural nature of fragment binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdiXW9LiaobbVg90H21EOLACvtfcHk0lg__OxSJlafRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsb3M&md5=42c1d64fd37316597c350f772a4c1028</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1177%2F1087057115570550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115570550%26sid%3Dliteratum%253Aachs%26aulast%3DChimenti%26aufirst%3DM.%2BS.%26aulast%3DBulfer%26aufirst%3DS.%2BL.%26aulast%3DNeitz%26aufirst%3DR.%2BJ.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DJames%26aufirst%3DT.%2BL.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DKelly%26aufirst%3DM.%2BJ.%26atitle%3DA%2520fragment-based%2520ligand%2520screen%2520against%2520part%2520of%2520a%2520large%2520protein%2520machine%253A%2520the%2520ND1%2520domains%2520of%2520the%2520AAA%252B%2520ATPase%2520p97%252FVCP%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D20%26spage%3D788%26epage%3D800%26doi%3D10.1177%2F1087057115570550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skelton, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bravo, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bair, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crocker, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzner-Toste, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liederer, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuen, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dragovich, P. S.</span><span> </span><span class="NLM_article-title">Fragment-based identification of amides derived from trans-2-(pyridin-3-yl)cyclopropanecarboxylic acid as potent inhibitors of human nicotinamide phosphoribosyltransferase (NAMPT)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">770</span><span class="NLM_x">–</span> <span class="NLM_lpage">792</span><span class="refDoi"> DOI: 10.1021/jm4015108</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4015108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFWitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=770-792&author=A.+M.+Giannettiauthor=X.+Zhengauthor=N.+J.+Skeltonauthor=W.+Wangauthor=B.+J.+Bravoauthor=K.+W.+Bairauthor=T.+Baumeisterauthor=E.+Chengauthor=L.+Crockerauthor=Y.+Fengauthor=J.+Gunzner-Tosteauthor=Y.+C.+Hoauthor=R.+Huaauthor=B.+M.+Liedererauthor=Y.+Liuauthor=X.+Maauthor=T.+O%E2%80%99Brienauthor=J.+Oehauthor=D.+Sampathauthor=Y.+Shenauthor=C.+Wangauthor=L.+Wangauthor=H.+Wuauthor=Y.+Xiaoauthor=P.+W.+Yuenauthor=M.+Zakauthor=G.+Zhaoauthor=Q.+Zhaoauthor=P.+S.+Dragovich&title=Fragment-based+identification+of+amides+derived+from+trans-2-%28pyridin-3-yl%29cyclopropanecarboxylic+acid+as+potent+inhibitors+of+human+nicotinamide+phosphoribosyltransferase+%28NAMPT%29&doi=10.1021%2Fjm4015108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based identification of amides derived from trans-2-(pyridin-3-yl)cyclopropanecarboxylic acid as potent inhibitors of human nicotinamide phosphoribosyltransferase (NAMPT)</span></div><div class="casAuthors">Giannetti, Anthony M.; Zheng, Xiaozhang; Skelton, Nicholas J.; Wang, Weiru; Bravo, Brandon J.; Bair, Kenneth W.; Baumeister, Timm; Cheng, Eric; Crocker, Lisa; Feng, Yezhen; Gunzner-Toste, Janet; Ho, Yen-Ching; Hua, Rongbao; Liederer, Bianca M.; Liu, Yongbo; Ma, Xiaolei; O'Brien, Thomas; Oeh, Jason; Sampath, Deepak; Shen, Youming; Wang, Chengcheng; Wang, Leslie; Wu, Hongxing; Xiao, Yang; Yuen, Po-wai; Zak, Mark; Zhao, Guiling; Zhao, Qiang; Dragovich, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">770-792</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Potent, trans-2-(pyridin-3-yl)-cyclopropane-carboxamide-contg. inhibitors of the human nicotinamide phosphoribosyl-transferase (NAMPT) enzyme were identified using fragment-based screening and structure-based design techniques.  Multiple crystal structures were obtained of initial fragment leads, and this structural information was utilized to improve the biochem. and cell-based potency of the assocd. mols.  Many of the optimized compds. exhibited nanomolar antiproliferative activities against human tumor lines in in vitro cell culture expts.  In a key example, a fragment lead (13, KD = 51 μM) was elaborated into a potent NAMPT inhibitor (39, NAMPT IC50 = 0.0051 μM, A2780 cell culture IC50 = 0.000 49 μM) which demonstrated encouraging in vivo efficacy in an HT-1080 mouse xenograft tumor model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGDYM9bbug1rVg90H21EOLACvtfcHk0lgqJlEEBQ-lgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFWitQ%253D%253D&md5=cf1e87e52ef0641e48465a44d5b3347c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm4015108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4015108%26sid%3Dliteratum%253Aachs%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DBravo%26aufirst%3DB.%2BJ.%26aulast%3DBair%26aufirst%3DK.%2BW.%26aulast%3DBaumeister%26aufirst%3DT.%26aulast%3DCheng%26aufirst%3DE.%26aulast%3DCrocker%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DGunzner-Toste%26aufirst%3DJ.%26aulast%3DHo%26aufirst%3DY.%2BC.%26aulast%3DHua%26aufirst%3DR.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DOeh%26aufirst%3DJ.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DYuen%26aufirst%3DP.%2BW.%26aulast%3DZak%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26atitle%3DFragment-based%2520identification%2520of%2520amides%2520derived%2520from%2520trans-2-%2528pyridin-3-yl%2529cyclopropanecarboxylic%2520acid%2520as%2520potent%2520inhibitors%2520of%2520human%2520nicotinamide%2520phosphoribosyltransferase%2520%2528NAMPT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D770%26epage%3D792%26doi%3D10.1021%2Fjm4015108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Hendrick, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willems, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irving, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davenport, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reeves, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffron, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bazin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turney, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, P. J.</span><span> </span><span class="NLM_article-title">Identification and investigation of novel binding fragments in the fatty acid binding protein 6 (FABP6)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">8094</span><span class="NLM_x">–</span> <span class="NLM_lpage">8102</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00869</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00869" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSht7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8094-8102&author=A.+G.+Hendrickauthor=I.+Mullerauthor=H.+Willemsauthor=P.+M.+Leonardauthor=S.+Irvingauthor=R.+Davenportauthor=T.+Itoauthor=J.+Reevesauthor=S.+Wrightauthor=V.+Allenauthor=S.+Wilkinsonauthor=H.+Heffronauthor=R.+Bazinauthor=J.+Turneyauthor=P.+J.+Mitchell&title=Identification+and+investigation+of+novel+binding+fragments+in+the+fatty+acid+binding+protein+6+%28FABP6%29&doi=10.1021%2Facs.jmedchem.6b00869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Investigation of Novel Binding Fragments in the Fatty Acid Binding Protein 6 (FABP6)</span></div><div class="casAuthors">Hendrick, Alan G.; Muller, Ilka; Willems, Henriette; Leonard, Philip M.; Irving, Steve; Davenport, Richard; Ito, Takashi; Reeves, Jenny; Wright, Susanne; Allen, Vivienne; Wilkinson, Stephen; Heffron, Helen; Bazin, Richard; Turney, Jennifer; Mitchell, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8094-8102</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fatty acid binding protein 6 (FABP6) is a potential drug discovery target, which, if inhibited, may have a therapeutic benefit for the treatment of diabetes.  Currently, there are no published inhibitors of FABP6, and with the target believed to be amenable to fragment-based drug discovery, a structurally enabled program was initiated.  This program successfully identified fragment hits using the surface plasmon resonance (SPR) platform.  Several hits were validated with SAR and were found to be displaced by the natural ligand taurocholate.  We report the first crystal structure of human FABP6 in the unbound form, in complex with cholate, and with one of the key fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYHJ_ox17KQbVg90H21EOLACvtfcHk0lgqJlEEBQ-lgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSht7%252FL&md5=f98fa9c2cf870f1ad2fac5b9b361ae40</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00869%26sid%3Dliteratum%253Aachs%26aulast%3DHendrick%26aufirst%3DA.%2BG.%26aulast%3DMuller%26aufirst%3DI.%26aulast%3DWillems%26aufirst%3DH.%26aulast%3DLeonard%26aufirst%3DP.%2BM.%26aulast%3DIrving%26aufirst%3DS.%26aulast%3DDavenport%26aufirst%3DR.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DReeves%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DV.%26aulast%3DWilkinson%26aufirst%3DS.%26aulast%3DHeffron%26aufirst%3DH.%26aulast%3DBazin%26aufirst%3DR.%26aulast%3DTurney%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DP.%2BJ.%26atitle%3DIdentification%2520and%2520investigation%2520of%2520novel%2520binding%2520fragments%2520in%2520the%2520fatty%2520acid%2520binding%2520protein%25206%2520%2528FABP6%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8094%26epage%3D8102%26doi%3D10.1021%2Facs.jmedchem.6b00869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Lund, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christopeit, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guttormsen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leiros, H. K.</span><span> </span><span class="NLM_article-title">Screening and design of inhibitor scaffolds for the antibiotic resistance oxacillinase-48 (OXA-48) through surface plasmon resonance screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">5542</span><span class="NLM_x">–</span> <span class="NLM_lpage">5554</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00660</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00660" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5542-5554&author=B.+A.+Lundauthor=T.+Christopeitauthor=Y.+Guttormsenauthor=A.+Bayerauthor=H.+K.+Leiros&title=Screening+and+design+of+inhibitor+scaffolds+for+the+antibiotic+resistance+oxacillinase-48+%28OXA-48%29+through+surface+plasmon+resonance+screening&doi=10.1021%2Facs.jmedchem.6b00660"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Screening and Design of Inhibitor Scaffolds for the Antibiotic Resistance Oxacillinase-48 (OXA-48) through Surface Plasmon Resonance Screening</span></div><div class="casAuthors">Lund, Bjarte Aarmo; Christopeit, Tony; Guttormsen, Yngve; Bayer, Annette; Leiros, Hanna-Kirsti S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5542-5554</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The spread of antibiotic resistant bacteria is a global threat that shakes the foundations of modern healthcare.  β-Lactamases are enzymes that confer resistance to β-lactam antibiotics in bacteria, and there is a crit. need for new inhibitors of these enzymes for combination therapy together with an antibiotic.  With this in mind, we have screened a library of 490 fragments to identify starting points for the development of new inhibitors of the class D β-lactamase oxacillinase-48 (OXA-48) through surface plasmon resonance (SPR), dose-rate inhibition assays, and X-ray crystallog.  Furthermore, we have uncovered structure-activity relationships and used alternate conformations from a crystallog. structure to grow a fragment into a more potent compd. with a KD of 50 μM and an IC50 of 18 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHGpwAMa7NrbVg90H21EOLACvtfcHk0lgqJlEEBQ-lgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWiurc%253D&md5=2a20589a324252b22d7558e5a7068367</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00660%26sid%3Dliteratum%253Aachs%26aulast%3DLund%26aufirst%3DB.%2BA.%26aulast%3DChristopeit%26aufirst%3DT.%26aulast%3DGuttormsen%26aufirst%3DY.%26aulast%3DBayer%26aufirst%3DA.%26aulast%3DLeiros%26aufirst%3DH.%2BK.%26atitle%3DScreening%2520and%2520design%2520of%2520inhibitor%2520scaffolds%2520for%2520the%2520antibiotic%2520resistance%2520oxacillinase-48%2520%2528OXA-48%2529%2520through%2520surface%2520plasmon%2520resonance%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5542%26epage%3D5554%26doi%3D10.1021%2Facs.jmedchem.6b00660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wixted, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffiths-Jones, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hearn, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rich, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saxty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willems, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolford, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cottom, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kou, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonchuk, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldser, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolognese, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podolin, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callahan, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerns, J. K.</span><span> </span><span class="NLM_article-title">Monoacidic inhibitors of the kelch-like ECH-associated protein 1: nuclear factor erythroid 2-related factor 2 (KEAP1:NRF2) protein–protein interaction with high cell potency identified by fragment-based discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">3991</span><span class="NLM_x">–</span> <span class="NLM_lpage">4006</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+Willemsauthor=A.+J.+Woolfordauthor=J.+E.+Cottomauthor=J.+P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+inhibitors+of+the+kelch-like+ECH-associated+protein+1%3A+nuclear+factor+erythroid+2-related+factor+2+%28KEAP1%3ANRF2%29+protein%E2%80%93protein+interaction+with+high+cell+potency+identified+by+fragment-based+discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0liYIg2g4oKETQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.%2BP.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520inhibitors%2520of%2520the%2520kelch-like%2520ECH-associated%2520protein%25201%253A%2520nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%253ANRF2%2529%2520protein%25E2%2580%2593protein%2520interaction%2520with%2520high%2520cell%2520potency%2520identified%2520by%2520fragment-based%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Bower, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannifer, A.</span><span> </span><span class="NLM_article-title">Using fragment-based technologies to target protein–protein interactions</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4685</span><span class="NLM_x">–</span> <span class="NLM_lpage">4696</span><span class="refDoi"> DOI: 10.2174/138161212802651689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.2174%2F138161212802651689" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4685-4696&author=J.+F.+Bowerauthor=A.+Pannifer&title=Using+fragment-based+technologies+to+target+protein%E2%80%93protein+interactions&doi=10.2174%2F138161212802651689"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.2174%2F138161212802651689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212802651689%26sid%3Dliteratum%253Aachs%26aulast%3DBower%26aufirst%3DJ.%2BF.%26aulast%3DPannifer%26aufirst%3DA.%26atitle%3DUsing%2520fragment-based%2520technologies%2520to%2520target%2520protein%25E2%2580%2593protein%2520interactions%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D4685%26epage%3D4696%26doi%3D10.2174%2F138161212802651689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Wartchow, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podlaski, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K. S.</span><span> </span><span class="NLM_article-title">Biosensor-based small molecule fragment screening with biolayer interferometry</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.1007/s10822-011-9439-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1007%2Fs10822-011-9439-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=21660516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSqtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=669-676&author=C.+A.+Wartchowauthor=F.+Podlaskiauthor=S.+Liauthor=K.+Rowanauthor=X.+Zhangauthor=D.+Markauthor=K.+S.+Huang&title=Biosensor-based+small+molecule+fragment+screening+with+biolayer+interferometry&doi=10.1007%2Fs10822-011-9439-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-based small molecule fragment screening with biolayer interferometry</span></div><div class="casAuthors">Wartchow, Charles A.; Podlaski, Frank; Li, Shirley; Rowan, Karen; Zhang, Xiaolei; Mark, David; Huang, Kuo-Sen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Biosensor-based fragment screening is a valuable tool in the drug discovery process.  This method is advantageous over many biochem. methods because primary hits can be distinguished from non-specific or non-ideal interactions by examg. binding profiles and responses, resulting in reduced false-pos. rates.  Biolayer interferometry (BLI), a technique that measures changes in an interference pattern generated from visible light reflected from an optical layer and a biolayer contg. proteins of interest, is a relatively new method for monitoring small mol. interactions.  The BLI format is based on a disposable sensor that is immersed in 96-well or 384-well plates.  BLI has been validated for small mol. detection and fragment screening with model systems and well-characterized targets where affinity consts. and binding profiles are generally similar to those obtained with surface plasmon resonance (SPR).  Screens with challenging targets involved in protein-protein interactions including BCL-2, JNK1, and eIF4E were performed with a fragment library of 6,500 compds., and hit rates were compared for these targets.  For eIF4E, a protein contg. a PPI site and a nucleotide binding site, results from a BLI fragment screen were compared to results obtained in biochem. HTS screens.  Overlapping hits were obsd. for the PPI site, and hits unique to the BLI screen were identified.  Hit assessments with SPR and BLI are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnIPrJFqAln7Vg90H21EOLACvtfcHk0liYIg2g4oKETQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSqtbbP&md5=c8c3bcca03e2def50ddcaf0ed8325060</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9439-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9439-8%26sid%3Dliteratum%253Aachs%26aulast%3DWartchow%26aufirst%3DC.%2BA.%26aulast%3DPodlaski%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DRowan%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMark%26aufirst%3DD.%26aulast%3DHuang%26aufirst%3DK.%2BS.%26atitle%3DBiosensor-based%2520small%2520molecule%2520fragment%2520screening%2520with%2520biolayer%2520interferometry%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26spage%3D669%26epage%3D676%26doi%3D10.1007%2Fs10822-011-9439-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Yoshimura, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyafusa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, K.</span><span> </span><span class="NLM_article-title">Identification of small-molecule inhibitors of the human S100B-p53 interaction and evaluation of their activity in human melanoma cells</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1109</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1016/j.bmc.2012.12.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.bmc.2012.12.042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1109-1115&author=C.+Yoshimuraauthor=T.+Miyafusaauthor=K.+Tsumoto&title=Identification+of+small-molecule+inhibitors+of+the+human+S100B-p53+interaction+and+evaluation+of+their+activity+in+human+melanoma+cells&doi=10.1016%2Fj.bmc.2012.12.042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.12.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.12.042%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimura%26aufirst%3DC.%26aulast%3DMiyafusa%26aufirst%3DT.%26aulast%3DTsumoto%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520small-molecule%2520inhibitors%2520of%2520the%2520human%2520S100B-p53%2520interaction%2520and%2520evaluation%2520of%2520their%2520activity%2520in%2520human%2520melanoma%2520cells%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D1109%26epage%3D1115%26doi%3D10.1016%2Fj.bmc.2012.12.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Whittaker, M.</span><span> </span><span class="NLM_article-title">Picking up the pieces with FBDD or FADD: invest early for future success</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">624</span><span class="refDoi"> DOI: 10.1016/j.drudis.2009.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.drudis.2009.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=19486947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BD1Mvmt1Citg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=623-624&author=M.+Whittaker&title=Picking+up+the+pieces+with+FBDD+or+FADD%3A+invest+early+for+future+success&doi=10.1016%2Fj.drudis.2009.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Picking up the pieces with FBDD or FADD: invest early for future success</span></div><div class="casAuthors">Whittaker Mark</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">623-4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRrARZ0UGwnHu98rul4nhSdfW6udTcc2eZdbzNobcoyf7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mvmt1Citg%253D%253D&md5=1e8b209f593cf74a642ea9f7c3289d42</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2009.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2009.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DWhittaker%26aufirst%3DM.%26atitle%3DPicking%2520up%2520the%2520pieces%2520with%2520FBDD%2520or%2520FADD%253A%2520invest%2520early%2520for%2520future%2520success%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D623%26epage%3D624%26doi%3D10.1016%2Fj.drudis.2009.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Varnes, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geschwindner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmquist, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forst, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dekker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, J. S.</span><span> </span><span class="NLM_article-title">Fragment-assisted hit investigation involving integrated HTS and fragment screening: Application to the identification of phosphodiesterase 10A (PDE10A) inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.10.100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.bmcl.2015.10.100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=26597534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVKqt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=197-202&author=J.+G.+Varnesauthor=S.+Geschwindnerauthor=C.+R.+Holmquistauthor=J.+Forstauthor=X.+Wangauthor=N.+Dekkerauthor=C.+W.+Scottauthor=G.+Tianauthor=M.+W.+Woodauthor=J.+S.+Albert&title=Fragment-assisted+hit+investigation+involving+integrated+HTS+and+fragment+screening%3A+Application+to+the+identification+of+phosphodiesterase+10A+%28PDE10A%29+inhibitors&doi=10.1016%2Fj.bmcl.2015.10.100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-assisted hit investigation involving integrated HTS and fragment screening: Application to the identification of phosphodiesterase 10A (PDE10A) inhibitors</span></div><div class="casAuthors">Varnes, Jeffrey G.; Geschwindner, Stefan; Holmquist, Christopher R.; Forst, Janet; Wang, Xia; Dekker, Niek; Scott, Clay W.; Tian, Gaochao; Wood, Michael W.; Albert, Jeffrey S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">197-202</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Fragment-based drug design (FBDD) relies on direct elaboration of fragment hits and typically requires high resoln. structural information to guide optimization.  In fragment-assisted drug discovery (FADD), fragments provide information to guide selection and design but do not serve as starting points for elaboration.  The authors describe FADD and high-throughput screening (HTS) campaign strategies conducted in parallel against PDE10A where fragment hit co-crystallog. was not available.  The fragment screen led to prioritized fragment hits (IC50's ∼500 μM), which were used to generate a hypothetical core scaffold.  Application of this scaffold as a filter to HTS output afforded a 4 μM hit, which, after prepn. of a small no. of analogs, was elaborated into a 16 nM lead.  This approach highlights the strength of FADD, as fragment methods were applied despite the absence of co-crystallog. information to efficiently identify a lead compd. for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoya6D4nTvSu7Vg90H21EOLACvtfcHk0lgQ0trbdNBEPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVKqt7jO&md5=17cdaef2373e4a9e693dae2983d7dc07</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.10.100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.10.100%26sid%3Dliteratum%253Aachs%26aulast%3DVarnes%26aufirst%3DJ.%2BG.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DHolmquist%26aufirst%3DC.%2BR.%26aulast%3DForst%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDekker%26aufirst%3DN.%26aulast%3DScott%26aufirst%3DC.%2BW.%26aulast%3DTian%26aufirst%3DG.%26aulast%3DWood%26aufirst%3DM.%2BW.%26aulast%3DAlbert%26aufirst%3DJ.%2BS.%26atitle%3DFragment-assisted%2520hit%2520investigation%2520involving%2520integrated%2520HTS%2520and%2520fragment%2520screening%253A%2520Application%2520to%2520the%2520identification%2520of%2520phosphodiesterase%252010A%2520%2528PDE10A%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D197%26epage%3D202%26doi%3D10.1016%2Fj.bmcl.2015.10.100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Elinder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geitmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gossas, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallblad, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winquist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordstrom, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamalainen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danielson, U. H.</span><span> </span><span class="NLM_article-title">Experimental validation of a fragment library for lead discovery using SPR biosensor technology</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1177/1087057110389038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057110389038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=21149860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsl2gs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=15-25&author=M.+Elinderauthor=M.+Geitmannauthor=T.+Gossasauthor=P.+Kallbladauthor=J.+Winquistauthor=H.+Nordstromauthor=M.+Hamalainenauthor=U.+H.+Danielson&title=Experimental+validation+of+a+fragment+library+for+lead+discovery+using+SPR+biosensor+technology&doi=10.1177%2F1087057110389038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental validation of a fragment library for lead discovery using SPR biosensor technology</span></div><div class="casAuthors">Elinder, Malin; Geitmann, Matthis; Gossas, Thomas; Kaellblad, Per; Winquist, Johan; Nordstroem, Helena; Haemaelaeinen, Markku; Danielson, U. Helena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-25</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A new fragment library for lead discovery has been designed and exptl. validated for use in surface plasmon resonance (SPR) biosensor-based screening.  The 930 compds. in the library were selected from 4.6 million com. available compds. using a series of physicochem. and medicinal chem. filters.  They were screened against 3 prototypical drug targets: HIV-1 protease, thrombin and carbonic anhydrase, and a nontarget: human serum albumin.  Compd. soly. was not a problem under the conditions used for screening.  The high sensitivity of the sensor surfaces allowed the detection of interactions for 35% to 97% of the fragments, depending on the target protein.  None of the fragments was promiscuous (i.e., interacted with a stoichiometry ≥5:1 with all 4 proteins), and only 2 compds. dissocd. slowly from all 4 proteins.  The use of several targets proved valuable since several compds. would have been disqualified from the library on the grounds of promiscuity if fewer target proteins had been used.  The exptl. procedure allowed an efficient evaluation and exploration of the new fragment library and confirmed that the new library is suitable for SPR biosensor-based screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMQe_fTV57lrVg90H21EOLACvtfcHk0lgAK2sTKpNLFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsl2gs7s%253D&md5=a49b6811a04299589c02852318b01aae</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1177%2F1087057110389038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057110389038%26sid%3Dliteratum%253Aachs%26aulast%3DElinder%26aufirst%3DM.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DGossas%26aufirst%3DT.%26aulast%3DKallblad%26aufirst%3DP.%26aulast%3DWinquist%26aufirst%3DJ.%26aulast%3DNordstrom%26aufirst%3DH.%26aulast%3DHamalainen%26aufirst%3DM.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26atitle%3DExperimental%2520validation%2520of%2520a%2520fragment%2520library%2520for%2520lead%2520discovery%2520using%2520SPR%2520biosensor%2520technology%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2011%26volume%3D16%26spage%3D15%26epage%3D25%26doi%3D10.1177%2F1087057110389038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Hamalainen, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhukov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivarsson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fex, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottfries, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjorsne, M.</span><span> </span><span class="NLM_article-title">Label-free primary screening and affinity ranking of fragment libraries using parallel analysis of protein panels</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">202</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1177/1087057108314651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057108314651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=18270366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BD1c7pt1Ojtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=202-209&author=M.+D.+Hamalainenauthor=A.+Zhukovauthor=M.+Ivarssonauthor=T.+Fexauthor=J.+Gottfriesauthor=R.+Karlssonauthor=M.+Bjorsne&title=Label-free+primary+screening+and+affinity+ranking+of+fragment+libraries+using+parallel+analysis+of+protein+panels&doi=10.1177%2F1087057108314651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Label-free primary screening and affinity ranking of fragment libraries using parallel analysis of protein panels</span></div><div class="casAuthors">Hamalainen Markku D; Zhukov Andrei; Ivarsson Maria; Fex Tomas; Gottfries Johan; Karlsson Robert; Bjorsne Magnus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">202-9</span>
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    </div><div class="casAbstract">The authors present fragment screening data obtained using a label-free parallel analysis approach where the binding of fragment library compounds to 4 different target proteins can be screened simultaneously using surface plasmon resonance detection.  They suggest this method as a first step in fragment screening to identify and select binders, reducing the demanding requirements on subsequent X-ray or nuclear magnetic resonance studies, and as a valuable "clean-up" tool to eliminate unwanted promiscuous binders from libraries.  A small directed fragment library of known thrombin binders and a general 500-compound fragment library were used in this study.  Thrombin, blocked thrombin, carbonic anhydrase, and glutathione-S-transferase were immobilized on the sensor chip surface, and the direct binding of the fragments was studied in real time.  Only 12 microg of each protein is needed for screening of a 3000-compound fragment library.  For screening, a binding site-blocked target as reference facilitates the identification of binding site-selective hits and the signals from other reference proteins for the elimination of false positives.  The scope and limitations of this screening approach are discussed for both target-directed and general fragment libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSjH4rWZRY1vhBrI9DTGjC0fW6udTcc2eal7lWsRI-m6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7pt1Ojtg%253D%253D&md5=52fd220da141e9347513d73ac18c87eb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1177%2F1087057108314651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108314651%26sid%3Dliteratum%253Aachs%26aulast%3DHamalainen%26aufirst%3DM.%2BD.%26aulast%3DZhukov%26aufirst%3DA.%26aulast%3DIvarsson%26aufirst%3DM.%26aulast%3DFex%26aufirst%3DT.%26aulast%3DGottfries%26aufirst%3DJ.%26aulast%3DKarlsson%26aufirst%3DR.%26aulast%3DBjorsne%26aufirst%3DM.%26atitle%3DLabel-free%2520primary%2520screening%2520and%2520affinity%2520ranking%2520of%2520fragment%2520libraries%2520using%2520parallel%2520analysis%2520of%2520protein%2520panels%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2008%26volume%3D13%26spage%3D202%26epage%3D209%26doi%3D10.1177%2F1087057108314651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Nordstrom, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gossas, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamalainen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallblad, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nystrom, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallberg, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danielson, U. H.</span><span> </span><span class="NLM_article-title">Identification of MMP-12 inhibitors by using biosensor-based screening of a fragment library</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3449</span><span class="NLM_x">–</span> <span class="NLM_lpage">3459</span><span class="refDoi"> DOI: 10.1021/jm8000289</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8000289" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3449-3459&author=H.+Nordstromauthor=T.+Gossasauthor=M.+Hamalainenauthor=P.+Kallbladauthor=S.+Nystromauthor=H.+Wallbergauthor=U.+H.+Danielson&title=Identification+of+MMP-12+inhibitors+by+using+biosensor-based+screening+of+a+fragment+library&doi=10.1021%2Fjm8000289"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm8000289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8000289%26sid%3Dliteratum%253Aachs%26aulast%3DNordstrom%26aufirst%3DH.%26aulast%3DGossas%26aufirst%3DT.%26aulast%3DHamalainen%26aufirst%3DM.%26aulast%3DKallblad%26aufirst%3DP.%26aulast%3DNystrom%26aufirst%3DS.%26aulast%3DWallberg%26aufirst%3DH.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26atitle%3DIdentification%2520of%2520MMP-12%2520inhibitors%2520by%2520using%2520biosensor-based%2520screening%2520of%2520a%2520fragment%2520library%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3449%26epage%3D3459%26doi%3D10.1021%2Fjm8000289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Navratilova, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macdonald, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathias, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span> </span><span class="NLM_article-title">Biosensor-based approach to the identification of protein kinase ligands with dual-site modes of action</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1177/1087057111422746</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057111422746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=22068706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVKltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=183-193&author=I.+Navratilovaauthor=G.+Macdonaldauthor=C.+Robinsonauthor=S.+Hughesauthor=J.+Mathiasauthor=C.+Phillipsauthor=A.+Cook&title=Biosensor-based+approach+to+the+identification+of+protein+kinase+ligands+with+dual-site+modes+of+action&doi=10.1177%2F1087057111422746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-based approach to the identification of protein kinase ligands with dual-site modes of action</span></div><div class="casAuthors">Navratilova, Iva; MacDonald, Graeme; Robinson, Colin; Hughes, Samantha; Mathias, John; Phillips, Chris; Cook, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-193</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The authors have used a surface plasmon resonance (SPR)-based biosensor approach to identify and characterize compds. with a unique binding mode to protein kinases.  Biacore was used to characterize hits from an enzymic high-throughput screen of the Tec family tyrosine kinase, IL2-inducible T cell kinase (ITK).  Complex binding kinetics was obsd. for some compds., which led to identification of compds. that bound simultaneously at both the ATP binding site and a second, allosteric site on ITK.  The presence of the second binding site was confirmed by X-ray crystallog.  The second site is located in the N-terminal lobe of the protein kinase catalytic domain, adjacent to but distinct from the ATP site.  To enable rapid optimization of binding properties, a competition-based Biacore assay has been developed to successfully identify second site noncompetitive binders that have been confirmed by X-ray crystallog. studies.  The authors have found that SPR technol. is a key method for rapid identification of compds. with dual-site modes of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_O18si5LWPLVg90H21EOLACvtfcHk0liPH4pIIVOMUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtVKltL8%253D&md5=8860bb955a27b7ba6b167725224c9e60</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1177%2F1087057111422746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422746%26sid%3Dliteratum%253Aachs%26aulast%3DNavratilova%26aufirst%3DI.%26aulast%3DMacdonald%26aufirst%3DG.%26aulast%3DRobinson%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%26aulast%3DMathias%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DA.%26atitle%3DBiosensor-based%2520approach%2520to%2520the%2520identification%2520of%2520protein%2520kinase%2520ligands%2520with%2520dual-site%2520modes%2520of%2520action%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D183%26epage%3D193%26doi%3D10.1177%2F1087057111422746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Nordin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jungnelius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, O. P.</span><span> </span><span class="NLM_article-title">Kinetic studies of small molecule interactions with protein kinases using biosensor technology</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">340</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">368</span><span class="refDoi"> DOI: 10.1016/j.ab.2005.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.ab.2005.02.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=2005&pages=359-368&author=H.+Nordinauthor=M.+Jungneliusauthor=R.+Karlssonauthor=O.+P.+Karlsson&title=Kinetic+studies+of+small+molecule+interactions+with+protein+kinases+using+biosensor+technology&doi=10.1016%2Fj.ab.2005.02.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DNordin%26aufirst%3DH.%26aulast%3DJungnelius%26aufirst%3DM.%26aulast%3DKarlsson%26aufirst%3DR.%26aulast%3DKarlsson%26aufirst%3DO.%2BP.%26atitle%3DKinetic%2520studies%2520of%2520small%2520molecule%2520interactions%2520with%2520protein%2520kinases%2520using%2520biosensor%2520technology%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D340%26spage%3D359%26epage%3D368%26doi%3D10.1016%2Fj.ab.2005.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Stenlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frostell-Karlsson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, O. P.</span><span> </span><span class="NLM_article-title">Studies of small molecule interactions with protein phosphatases using biosensor technology</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">353</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">–</span> <span class="NLM_lpage">225</span><span class="refDoi"> DOI: 10.1016/j.ab.2006.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.ab.2006.03.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2006&pages=217-225&author=P.+Stenlundauthor=A.+Frostell-Karlssonauthor=O.+P.+Karlsson&title=Studies+of+small+molecule+interactions+with+protein+phosphatases+using+biosensor+technology&doi=10.1016%2Fj.ab.2006.03.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2006.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2006.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DStenlund%26aufirst%3DP.%26aulast%3DFrostell-Karlsson%26aufirst%3DA.%26aulast%3DKarlsson%26aufirst%3DO.%2BP.%26atitle%3DStudies%2520of%2520small%2520molecule%2520interactions%2520with%2520protein%2520phosphatases%2520using%2520biosensor%2520technology%26jtitle%3DAnal.%2520Biochem.%26date%3D2006%26volume%3D353%26spage%3D217%26epage%3D225%26doi%3D10.1016%2Fj.ab.2006.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eb2jKBUDbHwBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0liEaOeFQueoPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Perspicace, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlatter, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, W.</span><span> </span><span class="NLM_article-title">Fragment-based screening using surface plasmon resonance technology</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">349</span><span class="refDoi"> DOI: 10.1177/1087057109332595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057109332595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=19403917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFygu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=337-349&author=S.+Perspicaceauthor=D.+Bannerauthor=J.+Benzauthor=F.+Mullerauthor=D.+Schlatterauthor=W.+Huber&title=Fragment-based+screening+using+surface+plasmon+resonance+technology&doi=10.1177%2F1087057109332595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based screening using surface plasmon resonance technology</span></div><div class="casAuthors">Perspicace, Samantha; Banner, David; Benz, Jorg; Muller, Francis; Schlatter, Daniel; Huber, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-349</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Surface plasmon resonance (SPR) technol. has emerged as a new and powerful technique to investigate the interaction between low-mol.-wt. mols. and target proteins.  In the present work, the authors assemble from a large compd. collection a library of 2226 mols. (fragments having low mol. wts. between 100 and 300 Da) to screen them for binding to chymase, a serine protease.  Both the active chymase and a zymogen-like form of the protein were used in parallel to distinguish between specific and unspecific binding.  The relative ligand-binding activity of the immobilized protein was periodically measured with a ref. compd.  The screening expts. were performed at 25 °C at a fragment concn. of 200 μM in the presence of 2% DMSO.  Applying the filter cascade, affinity-selectivity-competition (competition with ref. compds. and cross-competition with fragments), 80 compds. show up as pos. screening hits.  Competition expts. between fragments show that they bind to different parts of the active site.  Of 36 fragments co-crystd. for X-ray studies, 12 could be located in the active site of the protein.  These results validate the authors' library and demonstrate that the application of SPR technol. as a filter in fragment screening can be achieved successfully.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmmLjEQrEFlLVg90H21EOLACvtfcHk0liEaOeFQueoPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFygu74%253D&md5=c2bd577576c10adb933eae83a79c9f4c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1177%2F1087057109332595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109332595%26sid%3Dliteratum%253Aachs%26aulast%3DPerspicace%26aufirst%3DS.%26aulast%3DBanner%26aufirst%3DD.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DF.%26aulast%3DSchlatter%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DW.%26atitle%3DFragment-based%2520screening%2520using%2520surface%2520plasmon%2520resonance%2520technology%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D337%26epage%3D349%26doi%3D10.1177%2F1087057109332595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Otwinowski, Z.; Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. In  <span class="citation_source-book">Macromolecular Crystallography, part A</span>; <span class="NLM_contrib-group">Carter, C. W. J.; Sweet, R. M.</span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=C.+W.+J.+Carter&author=R.+M.+Sweet&title=Macromolecular+Crystallography%2C+part+A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMacromolecular%2520Crystallography%252C%2520part%2520A%26aulast%3DCarter%26aufirst%3DC.%2BW.%2BJ.%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: An automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span><span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+An+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgeil_ZTckDMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520An%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>.  <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span>. <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=15299374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BD2czpsV2gug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+CCP4+suite%3A+programs+for+protein+crystallography.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+1994%2C+50%2C+760%E2%80%93763.+10.1107%2FS0907444994003112."target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The CCP4 suite: programs for protein crystallography</span></div><div class="casAuthors">Anonymous</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">760-3</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">The CCP4 (Collaborative Computational Project, number 4) program suite is a collection of programs and associated data and subroutine libraries which can be used for macromolecular structure determination by X-ray crystallography.  The suite is designed to be flexible, allowing users a number of methods of achieving their aims and so there may be more than one program to cover each function.  The programs are written mainly in standard Fortran77.  They are from a wide variety of sources but are connected by standard data file formats.  The package has been ported to all the major platforms under both Unix and VMS.  The suite is distributed by anonymous ftp from Daresbury Laboratory and is widely used throughout the world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ1Q-skJYIZ2IYPsCkJIw00fW6udTcc2eYhjFIdnlpWmLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czpsV2gug%253D%253D&md5=92b759927d6071ea3f20c08b78ad1eb2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skubak, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebedev, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steiner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span> </span><span class="NLM_article-title">REFMAC5 for the refinement of macromolecular crystal structures</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skubakauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0liwPi3FYnXtQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkubak%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liwPi3FYnXtQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhebQLZl9rGAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 22 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yong Ai, Lucia Hwang, Alexander D. MacKerell Jr., Ari Melnick, <span class="NLM_string-name hlFld-ContribAuthor">Fengtian Xue</span>. </span><span class="cited-content_cbyCitation_article-title">Progress toward B-Cell Lymphoma 6 BTB Domain Inhibitors for the Treatment of Diffuse Large B-Cell Lymphoma and Beyond. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4333-4358. <a href="https://doi.org/10.1021/acs.jmedchem.0c01686" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01686</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01686%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProgress%252Btoward%252BB-Cell%252BLymphoma%252B6%252BBTB%252BDomain%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BDiffuse%252BLarge%252BB-Cell%252BLymphoma%252Band%252BBeyond%26aulast%3DAi%26aufirst%3DYong%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D25092020%26date%3D12042021%26volume%3D64%26issue%3D8%26spage%3D4333%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Natesh Singh, Ludovic Chaput, <span class="NLM_string-name hlFld-ContribAuthor">Bruno O. Villoutreix</span>. </span><span class="cited-content_cbyCitation_article-title">Fast Rescoring Protocols to Improve the Performance of Structure-Based Virtual Screening Performed on Protein–Protein Interfaces. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3910-3934. <a href="https://doi.org/10.1021/acs.jcim.0c00545" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00545</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00545%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFast%252BRescoring%252BProtocols%252Bto%252BImprove%252Bthe%252BPerformance%252Bof%252BStructure-Based%252BVirtual%252BScreening%252BPerformed%252Bon%252BProtein%2525E2%252580%252593Protein%252BInterfaces%26aulast%3DSingh%26aufirst%3DNatesh%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16052020%26date%3D11082020%26date%3D03082020%26volume%3D60%26issue%3D8%26spage%3D3910%26epage%3D3934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mingxing Teng, Scott B. Ficarro, Hojong Yoon, Jianwei Che, Jing Zhou, Eric S. Fischer, Jarrod A. Marto, Tinghu Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Nathanael S. Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Rationally Designed Covalent BCL6 Inhibitor That Targets a Tyrosine Residue in the Homodimer Interface. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (6)
                                     , 1269-1273. <a href="https://doi.org/10.1021/acsmedchemlett.0c00111" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00111%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DRationally%252BDesigned%252BCovalent%252BBCL6%252BInhibitor%252BThat%252BTargets%252Ba%252BTyrosine%252BResidue%252Bin%252Bthe%252BHomodimer%252BInterface%26aulast%3DTeng%26aufirst%3DMingxing%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03032020%26date%3D03042020%26date%3D07042020%26date%3D03042020%26volume%3D11%26issue%3D6%26spage%3D1269%26epage%3D1273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter R. Curran, Chris J. Radoux, Mihaela D. Smilova, Richard A. Sykes, Alicia P. Higueruelo, Anthony R. Bradley, Brian D. Marsden, David R. Spring, Tom L. Blundell, Andrew R. Leach, William R. Pitt, <span class="NLM_string-name hlFld-ContribAuthor">Jason C. Cole</span>. </span><span class="cited-content_cbyCitation_article-title">Hotspots API: A Python Package for the Detection of Small Molecule Binding Hotspots and Application to Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 1911-1916. <a href="https://doi.org/10.1021/acs.jcim.9b00996" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00996</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00996%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHotspots%252BAPI%25253A%252BA%252BPython%252BPackage%252Bfor%252Bthe%252BDetection%252Bof%252BSmall%252BMolecule%252BBinding%252BHotspots%252Band%252BApplication%252Bto%252BStructure-Based%252BDrug%252BDesign%26aulast%3DCurran%26aufirst%3DPeter%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D28102019%26date%3D02042020%26date%3D24032020%26volume%3D60%26issue%3D4%26spage%3D1911%26epage%3D1916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Benjamin R. Bellenie, Kwai-Ming J. Cheung, Ana Varela, Olivier A. Pierrat, Gavin W. Collie, Gary M. Box, Michael D. Bright, Sharon Gowan, Angela Hayes, Matthew J. Rodrigues, Kartika N. Shetty, Michael Carter, Owen A. Davis, Alan T. Henley, Paolo Innocenti, Louise D. Johnson, Manjuan Liu, Selby de Klerk, Yann-Vaï Le Bihan, Matthew G. Lloyd, P. Craig McAndrew, Erald Shehu, Rachel Talbot, Hannah L. Woodward, Rosemary Burke, Vladimir Kirkin, Rob L. M. van Montfort, Florence I. Raynaud, Olivia W. Rossanese, <span class="NLM_string-name hlFld-ContribAuthor">Swen Hoelder</span>. </span><span class="cited-content_cbyCitation_article-title">Achieving In Vivo Target Depletion through the Discovery and Optimization of Benzimidazolone BCL6 Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (8)
                                     , 4047-4068. <a href="https://doi.org/10.1021/acs.jmedchem.9b02076" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02076</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02076%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAchieving%252BIn%252BVivo%252BTarget%252BDepletion%252Bthrough%252Bthe%252BDiscovery%252Band%252BOptimization%252Bof%252BBenzimidazolone%252BBCL6%252BDegraders%26aulast%3DBellenie%26aufirst%3DBenjamin%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D13122019%26date%3D10042020%26volume%3D63%26issue%3D8%26spage%3D4047%26epage%3D4068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Weikai Guo, Yajing Xing, Qiansen Zhang, Jiuqing Xie, Dongxia Huang, Haijun Gu, Peng He, Miaoran Zhou, Shifen Xu, Xiufeng Pang, Mingyao Liu, Zhengfang Yi, <span class="NLM_string-name hlFld-ContribAuthor">Yihua Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Evaluation of B-Cell Lymphoma 6 Inhibitors of N-Phenyl-4-pyrimidinamine Derivatives Bearing Potent Activities against Tumor Growth. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 676-695. <a href="https://doi.org/10.1021/acs.jmedchem.9b01618" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01618</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01618%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252BB-Cell%252BLymphoma%252B6%252BInhibitors%252Bof%252BN-Phenyl-4-pyrimidinamine%252BDerivatives%252BBearing%252BPotent%252BActivities%252Bagainst%252BTumor%252BGrowth%26aulast%3DGuo%26aufirst%3DWeikai%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D29092019%26date%3D13012020%26date%3D02012020%26volume%3D63%26issue%3D2%26spage%3D676%26epage%3D695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul N. Mortenson, Daniel A. Erlanson, Iwan J. P. de Esch, Wolfgang Jahnke, <span class="NLM_string-name hlFld-ContribAuthor">Christopher N. Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2017. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (8)
                                     , 3857-3872. <a href="https://doi.org/10.1021/acs.jmedchem.8b01472" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01472</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01472%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2017%26aulast%3DMortenson%26aufirst%3DPaul%2BN.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D21092018%26date%3D21112018%26volume%3D62%26issue%3D8%26spage%3D3857%26epage%3D3872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">William McCoull, Tony Cheung, Erica Anderson, Peter Barton, Jonathan Burgess, Kate Byth, Qing Cao, M. Paola Castaldi, Huawei Chen, Elisabetta Chiarparin, Rodrigo J. Carbajo, Erin Code, Suzanna Cowan, Paul R. Davey, Andrew D. Ferguson, Shaun Fillery, Nathan O. Fuller, Ning Gao, David Hargreaves, Martin R. Howard, Jun Hu, Aarti Kawatkar, Paul D. Kemmitt, Elisabetta Leo, Daniel M. Molina, Nichole O’Connell, Philip Petteruti, Timothy Rasmusson, Piotr Raubo, Philip B. Rawlins, Piero Ricchiuto, Graeme R. Robb, Monica Schenone, Michael J. Waring, Michael Zinda, Stephen Fawell, <span class="NLM_string-name hlFld-ContribAuthor">David M. Wilson</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Novel B-Cell Lymphoma 6 (BCL6) PROTAC To Provide Insight into Small Molecule Targeting of BCL6. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (11)
                                     , 3131-3141. <a href="https://doi.org/10.1021/acschembio.8b00698" title="DOI URL">https://doi.org/10.1021/acschembio.8b00698</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.8b00698%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDevelopment%252Bof%252Ba%252BNovel%252BB-Cell%252BLymphoma%252B6%252B%252528BCL6%252529%252BPROTAC%252BTo%252BProvide%252BInsight%252Binto%252BSmall%252BMolecule%252BTargeting%252Bof%252BBCL6%26aulast%3DMcCoull%26aufirst%3DWilliam%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D27072018%26date%3D27092018%26date%3D17102018%26volume%3D13%26issue%3D11%26spage%3D3131%26epage%3D3141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Huimin Cheng, Brian M. Linhares, Wenbo Yu, Mariano G. Cardenas, Yong Ai, Wenjuan Jiang, Alyssa Winkler, Sandra Cohen, Ari Melnick, Alexander MacKerell, Jr., Tomasz Cierpicki, <span class="NLM_string-name hlFld-ContribAuthor">Fengtian Xue</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Thiourea-Based Inhibitors of the B-Cell Lymphoma 6 BTB Domain via NMR-Based Fragment Screening and Computer-Aided Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (17)
                                     , 7573-7588. <a href="https://doi.org/10.1021/acs.jmedchem.8b00040" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00040</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00040%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BThiourea-Based%252BInhibitors%252Bof%252Bthe%252BB-Cell%252BLymphoma%252B6%252BBTB%252BDomain%252Bvia%252BNMR-Based%252BFragment%252BScreening%252Band%252BComputer-Aided%252BDrug%252BDesign%26aulast%3DCheng%26aufirst%3DHuimin%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D10012018%26date%3D17072018%26date%3D03072018%26volume%3D61%26issue%3D17%26spage%3D7573%26epage%3D7588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barbara  Orth</span>, <span class="hlFld-ContribAuthor ">Bodo  Sander</span>, <span class="hlFld-ContribAuthor ">Andreas  Möglich</span>, <span class="hlFld-ContribAuthor ">Kay  Diederichs</span>, <span class="hlFld-ContribAuthor ">Martin  Eilers</span>, <span class="hlFld-ContribAuthor ">Sonja  Lorenz</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of an atypical interaction site in the BTB domain of the MYC-interacting zinc-finger protein 1. </span><span class="cited-content_cbyCitation_journal-name">Structure</span><span> <strong>2021,</strong> <em>66 </em><a href="https://doi.org/10.1016/j.str.2021.06.005" title="DOI URL">https://doi.org/10.1016/j.str.2021.06.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.str.2021.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.str.2021.06.005%26sid%3Dliteratum%253Aachs%26jtitle%3DStructure%26atitle%3DIdentification%252Bof%252Ban%252Batypical%252Binteraction%252Bsite%252Bin%252Bthe%252BBTB%252Bdomain%252Bof%252Bthe%252BMYC-interacting%252Bzinc-finger%252Bprotein%252B1%26aulast%3DOrth%26aufirst%3DBarbara%26date%3D2021%26volume%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiawei  Guo</span>, <span class="hlFld-ContribAuthor ">Yanan  Liu</span>, <span class="hlFld-ContribAuthor ">Jing  Lv</span>, <span class="hlFld-ContribAuthor ">Bin  Zou</span>, <span class="hlFld-ContribAuthor ">Zhi  Chen</span>, <span class="hlFld-ContribAuthor ">Kun  Li</span>, <span class="hlFld-ContribAuthor ">Juanjuan  Feng</span>, <span class="hlFld-ContribAuthor ">Zhenyu  Cai</span>, <span class="hlFld-ContribAuthor ">Lai  Wei</span>, <span class="hlFld-ContribAuthor ">Mingyao  Liu</span>, <span class="hlFld-ContribAuthor ">Xiufeng  Pang</span>. </span><span class="cited-content_cbyCitation_article-title">BCL6 confers KRAS-mutant non–small-cell lung cancer resistance to BET inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Clinical Investigation</span><span> <strong>2021,</strong> <em>131 </em>
                                    (1)
                                     <a href="https://doi.org/10.1172/JCI133090" title="DOI URL">https://doi.org/10.1172/JCI133090</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1172/JCI133090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1172%2FJCI133090%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Clinical%2520Investigation%26atitle%3DBCL6%252Bconfers%252BKRAS-mutant%252Bnon%2525E2%252580%252593small-cell%252Blung%252Bcancer%252Bresistance%252Bto%252BBET%252Binhibitors%26aulast%3DGuo%26aufirst%3DJiawei%26date%3D2021%26date%3D2021%26volume%3D131%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Charline  Kieffer</span>, <span class="hlFld-ContribAuthor ">Jean Pierre  Jourdan</span>, <span class="hlFld-ContribAuthor ">Marie  Jouanne</span>, <span class="hlFld-ContribAuthor ">Anne Sophie  Voisin-Chiret</span>. </span><span class="cited-content_cbyCitation_article-title">Noncellular screening for the discovery of protein–protein interaction modulators. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1592-1603. <a href="https://doi.org/10.1016/j.drudis.2020.07.012" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.07.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.07.012%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DNoncellular%252Bscreening%252Bfor%252Bthe%252Bdiscovery%252Bof%252Bprotein%2525E2%252580%252593protein%252Binteraction%252Bmodulators%26aulast%3DKieffer%26aufirst%3DCharline%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1592%26epage%3D1603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sofia  Piepoli</span>, <span class="hlFld-ContribAuthor ">Aaron Oliver  Alt</span>, <span class="hlFld-ContribAuthor ">Canan  Atilgan</span>, <span class="hlFld-ContribAuthor ">Erika Jazmin  Mancini</span>, <span class="hlFld-ContribAuthor ">Batu  Erman</span>. </span><span class="cited-content_cbyCitation_article-title">Structural analysis of the PATZ1 BTB domain homodimer. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section D Structural Biology</span><span> <strong>2020,</strong> <em>76 </em>
                                    (6)
                                     , 581-593. <a href="https://doi.org/10.1107/S2059798320005355" title="DOI URL">https://doi.org/10.1107/S2059798320005355</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2059798320005355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2059798320005355%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520D%2520Structural%2520Biology%26atitle%3DStructural%252Banalysis%252Bof%252Bthe%252BPATZ1%252BBTB%252Bdomain%252Bhomodimer%26aulast%3DPiepoli%26aufirst%3DSofia%26date%3D2020%26date%3D2020%26volume%3D76%26issue%3D6%26spage%3D581%26epage%3D593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefanie  Schlager</span>, <span class="hlFld-ContribAuthor ">Carina  Salomon</span>, <span class="hlFld-ContribAuthor ">Sabine  Olt</span>, <span class="hlFld-ContribAuthor ">Christoph  Albrecht</span>, <span class="hlFld-ContribAuthor ">Anja  Ebert</span>, <span class="hlFld-ContribAuthor ">Oliver  Bergner</span>, <span class="hlFld-ContribAuthor ">Johannes  Wachter</span>, <span class="hlFld-ContribAuthor ">Francesca  Trapani</span>, <span class="hlFld-ContribAuthor ">Daniel  Gerlach</span>, <span class="hlFld-ContribAuthor ">Tilman  Voss</span>, <span class="hlFld-ContribAuthor ">Anna  Traunbauer</span>, <span class="hlFld-ContribAuthor ">Julian  Jude</span>, <span class="hlFld-ContribAuthor ">Matthias  Hinterndorfer</span>, <span class="hlFld-ContribAuthor ">Martina  Minnich</span>, <span class="hlFld-ContribAuthor ">Norbert  Schweifer</span>, <span class="hlFld-ContribAuthor ">Sophia M.  Blake</span>, <span class="hlFld-ContribAuthor ">Vittoria  Zinzalla</span>, <span class="hlFld-ContribAuthor ">Barbara  Drobits</span>, <span class="hlFld-ContribAuthor ">Darryl B.  McConnell</span>, <span class="hlFld-ContribAuthor ">Norbert  Kraut</span>, <span class="hlFld-ContribAuthor ">Mark  Pearson</span>, <span class="hlFld-ContribAuthor ">Johannes  Zuber</span>, <span class="hlFld-ContribAuthor ">Manfred  Koegl</span>. </span><span class="cited-content_cbyCitation_article-title">Inducible knock-out of BCL6 in lymphoma cells results in tumor stasis. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 875-890. <a href="https://doi.org/10.18632/oncotarget.27506" title="DOI URL">https://doi.org/10.18632/oncotarget.27506</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.27506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.27506%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DInducible%252Bknock-out%252Bof%252BBCL6%252Bin%252Blymphoma%252Bcells%252Bresults%252Bin%252Btumor%252Bstasis%26aulast%3DSchlager%26aufirst%3DStefanie%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D9%26spage%3D875%26epage%3D890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhengyu  Han</span>, <span class="hlFld-ContribAuthor ">Gang  Liu</span>, <span class="hlFld-ContribAuthor ">Rui  Wang</span>, <span class="hlFld-ContribAuthor ">Xiu-Qin  Dong</span>, <span class="hlFld-ContribAuthor ">Xumu  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Highly efficient Ir-catalyzed asymmetric hydrogenation of benzoxazinones and derivatives with a Brønsted acid cocatalyst. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2019,</strong> <em>10 </em>
                                    (15)
                                     , 4328-4333. <a href="https://doi.org/10.1039/C8SC05797D" title="DOI URL">https://doi.org/10.1039/C8SC05797D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8SC05797D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8SC05797D%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DHighly%252Befficient%252BIr-catalyzed%252Basymmetric%252Bhydrogenation%252Bof%252Bbenzoxazinones%252Band%252Bderivatives%252Bwith%252Ba%252BBr%2525C3%2525B8nsted%252Bacid%252Bcocatalyst%26aulast%3DHan%26aufirst%3DZhengyu%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D15%26spage%3D4328%26epage%3D4333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng  Li</span>, <span class="hlFld-ContribAuthor ">Huaqing  Liu</span>, <span class="hlFld-ContribAuthor ">Yanmeng  Li</span>, <span class="hlFld-ContribAuthor ">Yan  Wang</span>, <span class="hlFld-ContribAuthor ">Lin  Zhao</span>, <span class="hlFld-ContribAuthor ">Hui  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">miR‑339‑5p inhibits lung adenocarcinoma invasion and migration by directly targeting BCL6. </span><span class="cited-content_cbyCitation_journal-name">Oncology Letters</span><span> <strong>2018,</strong> <em> </em><a href="https://doi.org/10.3892/ol.2018.9376" title="DOI URL">https://doi.org/10.3892/ol.2018.9376</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/ol.2018.9376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2Fol.2018.9376%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Letters%26atitle%3DmiR%2525E2%252580%252591339%2525E2%252580%2525915p%252Binhibits%252Blung%252Badenocarcinoma%252Binvasion%252Band%252Bmigration%252Bby%252Bdirectly%252Btargeting%252BBCL6%26aulast%3DLi%26aufirst%3DPeng%26date%3D2018%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eliot L.  Osher</span>, <span class="hlFld-ContribAuthor ">Francisco  Castillo</span>, <span class="hlFld-ContribAuthor ">Nagarajan  Elumalai</span>, <span class="hlFld-ContribAuthor ">Michael J.  Waring</span>, <span class="hlFld-ContribAuthor ">Garry  Pairaudeau</span>, <span class="hlFld-ContribAuthor ">Ali  Tavassoli</span>. </span><span class="cited-content_cbyCitation_article-title">A genetically selected cyclic peptide inhibitor of BCL6 homodimerization. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (11)
                                     , 3034-3038. <a href="https://doi.org/10.1016/j.bmc.2018.03.012" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.03.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.03.012%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DA%252Bgenetically%252Bselected%252Bcyclic%252Bpeptide%252Binhibitor%252Bof%252BBCL6%252Bhomodimerization%26aulast%3DOsher%26aufirst%3DEliot%2BL.%26date%3D2018%26volume%3D26%26issue%3D11%26spage%3D3034%26epage%3D3038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mélanie  Lambert</span>, <span class="hlFld-ContribAuthor ">Samy  Jambon</span>, <span class="hlFld-ContribAuthor ">Sabine  Depauw</span>, <span class="hlFld-ContribAuthor ">Marie-Hélène  David-Cordonnier</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Transcription Factors for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (6)
                                     , 1479. <a href="https://doi.org/10.3390/molecules23061479" title="DOI URL">https://doi.org/10.3390/molecules23061479</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23061479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23061479%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DTargeting%252BTranscription%252BFactors%252Bfor%252BCancer%252BTreatment%26aulast%3DLambert%26aufirst%3DM%25C3%25A9lanie%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D6%26spage%3D1479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rebecca J  Leeman-Neill</span>, <span class="hlFld-ContribAuthor ">Govind  Bhagat</span>. </span><span class="cited-content_cbyCitation_article-title">BCL6 as a therapeutic target for lymphoma. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2018,</strong> <em>22 </em>
                                    (2)
                                     , 143-152. <a href="https://doi.org/10.1080/14728222.2018.1420782" title="DOI URL">https://doi.org/10.1080/14728222.2018.1420782</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14728222.2018.1420782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14728222.2018.1420782%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DBCL6%252Bas%252Ba%252Btherapeutic%252Btarget%252Bfor%252Blymphoma%26aulast%3DLeeman-Neill%26aufirst%3DRebecca%2BJ%26date%3D2018%26date%3D2018%26volume%3D22%26issue%3D2%26spage%3D143%26epage%3D152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dan  Chen</span>, <span class="hlFld-ContribAuthor ">Xiao-Qing  Xiong</span>, <span class="hlFld-ContribAuthor ">Ying-Hao  Zang</span>, <span class="hlFld-ContribAuthor ">Ying  Tong</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>, <span class="hlFld-ContribAuthor ">Qi  Chen</span>, <span class="hlFld-ContribAuthor ">Yue-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Xing-Ya  Gao</span>, <span class="hlFld-ContribAuthor ">Yu-Ming  Kang</span>, <span class="hlFld-ContribAuthor ">Guo-Qing  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">BCL6 attenuates renal inflammation via negative regulation of NLRP3 transcription. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Disease</span><span> <strong>2017,</strong> <em>8 </em>
                                    (10)
                                     , e3156-e3156. <a href="https://doi.org/10.1038/cddis.2017.567" title="DOI URL">https://doi.org/10.1038/cddis.2017.567</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/cddis.2017.567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fcddis.2017.567%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Disease%26atitle%3DBCL6%252Battenuates%252Brenal%252Binflammation%252Bvia%252Bnegative%252Bregulation%252Bof%252BNLRP3%252Btranscription%26aulast%3DChen%26aufirst%3DDan%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D10%26spage%3De3156%26epage%3De3156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nina  Kerres</span>, <span class="hlFld-ContribAuthor ">Steffen  Steurer</span>, <span class="hlFld-ContribAuthor ">Stefanie  Schlager</span>, <span class="hlFld-ContribAuthor ">Gerd  Bader</span>, <span class="hlFld-ContribAuthor ">Helmut  Berger</span>, <span class="hlFld-ContribAuthor ">Maureen  Caligiuri</span>, <span class="hlFld-ContribAuthor ">Christian  Dank</span>, <span class="hlFld-ContribAuthor ">John R.  Engen</span>, <span class="hlFld-ContribAuthor ">Peter  Ettmayer</span>, <span class="hlFld-ContribAuthor ">Bernhard  Fischerauer</span>, <span class="hlFld-ContribAuthor ">Gerlinde  Flotzinger</span>, <span class="hlFld-ContribAuthor ">Daniel  Gerlach</span>, <span class="hlFld-ContribAuthor ">Thomas  Gerstberger</span>, <span class="hlFld-ContribAuthor ">Teresa  Gmaschitz</span>, <span class="hlFld-ContribAuthor ">Peter  Greb</span>, <span class="hlFld-ContribAuthor ">Bingsong  Han</span>, <span class="hlFld-ContribAuthor ">Elizabeth  Heyes</span>, <span class="hlFld-ContribAuthor ">Roxana E.  Iacob</span>, <span class="hlFld-ContribAuthor ">Dirk  Kessler</span>, <span class="hlFld-ContribAuthor ">Heike  Kölle</span>, <span class="hlFld-ContribAuthor ">Lyne  Lamarre</span>, <span class="hlFld-ContribAuthor ">David R.  Lancia</span>, <span class="hlFld-ContribAuthor ">Simon  Lucas</span>, <span class="hlFld-ContribAuthor ">Moriz  Mayer</span>, <span class="hlFld-ContribAuthor ">Katharina  Mayr</span>, <span class="hlFld-ContribAuthor ">Nikolai  Mischerikow</span>, <span class="hlFld-ContribAuthor ">Katja  Mück</span>, <span class="hlFld-ContribAuthor ">Christoph  Peinsipp</span>, <span class="hlFld-ContribAuthor ">Oliver  Petermann</span>, <span class="hlFld-ContribAuthor ">Ulrich  Reiser</span>, <span class="hlFld-ContribAuthor ">Dorothea  Rudolph</span>, <span class="hlFld-ContribAuthor ">Klaus  Rumpel</span>, <span class="hlFld-ContribAuthor ">Carina  Salomon</span>, <span class="hlFld-ContribAuthor ">Dirk  Scharn</span>, <span class="hlFld-ContribAuthor ">Renate  Schnitzer</span>, <span class="hlFld-ContribAuthor ">Andreas  Schrenk</span>, <span class="hlFld-ContribAuthor ">Norbert  Schweifer</span>, <span class="hlFld-ContribAuthor ">Diane  Thompson</span>, <span class="hlFld-ContribAuthor ">Elisabeth  Traxler</span>, <span class="hlFld-ContribAuthor ">Roland  Varecka</span>, <span class="hlFld-ContribAuthor ">Tilman  Voss</span>, <span class="hlFld-ContribAuthor ">Alexander  Weiss-Puxbaum</span>, <span class="hlFld-ContribAuthor ">Sandra  Winkler</span>, <span class="hlFld-ContribAuthor ">Xiaozhang  Zheng</span>, <span class="hlFld-ContribAuthor ">Andreas  Zoephel</span>, <span class="hlFld-ContribAuthor ">Norbert  Kraut</span>, <span class="hlFld-ContribAuthor ">Darryl  McConnell</span>, <span class="hlFld-ContribAuthor ">Mark  Pearson</span>, <span class="hlFld-ContribAuthor ">Manfred  Koegl</span>. </span><span class="cited-content_cbyCitation_article-title">Chemically Induced Degradation of the Oncogenic Transcription Factor BCL6. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2017,</strong> <em>20 </em>
                                    (12)
                                     , 2860-2875. <a href="https://doi.org/10.1016/j.celrep.2017.08.081" title="DOI URL">https://doi.org/10.1016/j.celrep.2017.08.081</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2017.08.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2017.08.081%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DChemically%252BInduced%252BDegradation%252Bof%252Bthe%252BOncogenic%252BTranscription%252BFactor%252BBCL6%26aulast%3DKerres%26aufirst%3DNina%26date%3D2017%26volume%3D20%26issue%3D12%26spage%3D2860%26epage%3D2875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takeshi  Yasui</span>, <span class="hlFld-ContribAuthor ">Takeshi  Yamamoto</span>, <span class="hlFld-ContribAuthor ">Nozomu  Sakai</span>, <span class="hlFld-ContribAuthor ">Kouhei  Asano</span>, <span class="hlFld-ContribAuthor ">Takafumi  Takai</span>, <span class="hlFld-ContribAuthor ">Yayoi  Yoshitomi</span>, <span class="hlFld-ContribAuthor ">Melinda  Davis</span>, <span class="hlFld-ContribAuthor ">Terufumi  Takagi</span>, <span class="hlFld-ContribAuthor ">Kotaro  Sakamoto</span>, <span class="hlFld-ContribAuthor ">Satoshi  Sogabe</span>, <span class="hlFld-ContribAuthor ">Yusuke  Kamada</span>, <span class="hlFld-ContribAuthor ">Weston  Lane</span>, <span class="hlFld-ContribAuthor ">Gyorgy  Snell</span>, <span class="hlFld-ContribAuthor ">Masashi  Iwata</span>, <span class="hlFld-ContribAuthor ">Masayuki  Goto</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Inooka</span>, <span class="hlFld-ContribAuthor ">Jun-ichi  Sakamoto</span>, <span class="hlFld-ContribAuthor ">Yoshihisa  Nakada</span>, <span class="hlFld-ContribAuthor ">Yasuhiro  Imaeda</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a novel B-cell lymphoma 6 (BCL6)–corepressor interaction inhibitor by utilizing structure-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (17)
                                     , 4876-4886. <a href="https://doi.org/10.1016/j.bmc.2017.07.037" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.07.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.07.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.07.037%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bnovel%252BB-cell%252Blymphoma%252B6%252B%252528BCL6%252529%2525E2%252580%252593corepressor%252Binteraction%252Binhibitor%252Bby%252Butilizing%252Bstructure-based%252Bdrug%252Bdesign%26aulast%3DYasui%26aufirst%3DTakeshi%26date%3D2017%26volume%3D25%26issue%3D17%26spage%3D4876%26epage%3D4886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known BCL6 inhibitors. <sup><i>a</i></sup>Determined by microscale thermophoresis.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a><sup><i>b</i></sup>Determined by Fluorescence polarization assay.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a><sup><i>c</i></sup>Determined by SPR.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a><sup><i>d</i></sup>All <span class="smallcaps smallerCapital">d</span>-amino acids; TAT means cell penetrating peptide sequence, and Fu means fusogenic peptide sequence.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a><sup><i>e</i></sup>Determined by DLBCL cell growth inhibition.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Layout of the sensor chip. (A) Four independent flow cells (Fc), each with five detection spots. (B) Protein immobilized in each flow cell. Binding responses of captured wt BCL6<sup>BTB</sup>, captured mt BCL6<sup>BTB</sup>, coupled wt BCL6<sup>BTB</sup>, and NeutrAvidin are detected on spots 1–3, spots 2–3, spots 5–4, and spots 3–4, respectively. Sensorgrams of BCoR peptide binding to (C) captured wt BCL6<sup>BTB</sup>, (D) captured mt BCL6<sup>BTB</sup>, (E) coupled wt BCL6<sup>BTB</sup>, and (F) NeutrAvidin. Lower graphs indicate the fit plots of the response measured at equilibrium plotted against BCoR peptide concentration. Top concentration is 100 μM; dilution step is 2-fold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. SPR-based screening of 1494 fragments tested against (A) captured wt BCL6<sup>BTB</sup>, (B) captured mt BCL6<sup>BTB</sup>, (C) coupled wt BCL6<sup>BTB</sup>, and (D) NeutrAvidin. The color of each plot indicates fast binding (blue-circle), slow binding (red-circle), and 100 μM BCoR peptide as a positive control (gray-square). (E) Venn diagram showing the number of fragments selected at each BCL6<sup>BTB</sup> screening and the overlap of fragments found by each screening.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Characterization of fragment <b>1</b>. (A) Chemical structure of fragment <b>1</b>. (B) STD-NMR experiments. <sup>1</sup>H spectra of fragment <b>1</b> for off-resonance (black line) and on-resonance (red line), and the STD spectrum of fragment <b>1</b> (difference). (C) SPR competition experiments. Sensorgrams of fragment <b>1</b> to mt BCL6<sup>BTB</sup> in the absence or presence of 100 μM BCoR peptide. Top concentration is 1 mM; dilution step is 2-fold. (D) Co-crystal structure of fragment <b>1</b> in complex with BCL6<sup>BTB</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Consolidation of the fragment-based inhibitor <b>4</b> with hit compound <b>5</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structural superposition of the complex structures with compounds <b>4</b> (orange) and <b>5</b> (yellow).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Co-crystal structure of compound <b>7</b> in complex with BCL6<sup>BTB</sup>. (B) Chemical structure of <b>7</b>. (C) Equilibrium plot and sensorgrams (insert) of <b>7</b> binding to BCL6. (D, E) PPI inhibitory activities in cell-free (ELISA) and cellular (M2H) assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Summary of the FBDD approach for identification of the compound <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/medium/jm-2017-003132_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Procedure for the Pyrimidine Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-10/acs.jmedchem.7b00313/20170519/images/large/jm-2017-003132_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00313&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i39">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12153" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12153" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Huynh, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischle, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardwell, V. J.</span><span> </span><span class="NLM_article-title">BCoR, a novel corepressor involved in BCL-6 repression</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1810</span><span class="NLM_x">–</span> <span class="NLM_lpage">1823</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10898795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD3cXltFCru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2000&pages=1810-1823&author=K.+D.+Huynhauthor=W.+Fischleauthor=E.+Verdinauthor=V.+J.+Bardwell&title=BCoR%2C+a+novel+corepressor+involved+in+BCL-6+repression"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">BCoR, a novel corepressor involved in BCL-6 repression</span></div><div class="casAuthors">Huynh, Khanh D.; Fischle, Wolfgang; Verdin, Eric; Bardwell, Vivian J.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1810-1823</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">BCL-6 encodes a POZ/zinc finger transcriptional repressor that is required for germinal center formation and may influence apoptosis.  Aberrant expression of BCL-6 due to chromosomal translocations is implicated in certain subtypes of non-Hodgkin's lymphoma.  The POZ domains of BCL-6 and several other POZ proteins interact with corepressors N-CoR and SMRT.  Here the authors identify and characterize a novel corepressor BCoR (BCL-6 interacting corepressor), which is expressed ubiquitously in human tissues.  BCoR can function as a corepressor when tethered to DNA and, when overexpressed, can potentiate BCL-6 repression.  Specific class I and II histone deacetylases (HDACs) interact in vivo with BCoR, suggesting that BCoR may functionally link these two classes of HDACs.  Strikingly, BCoR interacts selectively with the POZ domain of BCL-6 but not with eight other POZ proteins tested, including PLZF.  Addnl., interactions between the BCL-6 POZ domain and SMRT, N-CoR, and BCoR are mutually exclusive.  The specificity of the BCL-6/BCoR interaction suggests that BCoR may have a role in BCL-6-assocd. lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd5QFf1IeYsrVg90H21EOLACvtfcHk0lh5TkxqTd_Zyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXltFCru78%253D&md5=07cb1649b94629d05e5550d5cb0bcd22</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuynh%26aufirst%3DK.%2BD.%26aulast%3DFischle%26aufirst%3DW.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DBardwell%26aufirst%3DV.%2BJ.%26atitle%3DBCoR%252C%2520a%2520novel%2520corepressor%2520involved%2520in%2520BCL-6%2520repression%26jtitle%3DGenes%2520Dev.%26date%3D2000%26volume%3D14%26spage%3D1810%26epage%3D1823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Dent, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaffer, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staudt, L. M.</span><span> </span><span class="NLM_article-title">Control of inflammation, cytokine expression, and germinal center formation by BCL-6</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">589</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1126/science.276.5312.589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1126%2Fscience.276.5312.589" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=589-592&author=A.+L.+Dentauthor=A.+L.+Shafferauthor=X.+Yuauthor=D.+Allmanauthor=L.+M.+Staudt&title=Control+of+inflammation%2C+cytokine+expression%2C+and+germinal+center+formation+by+BCL-6&doi=10.1126%2Fscience.276.5312.589"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.276.5312.589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.276.5312.589%26sid%3Dliteratum%253Aachs%26aulast%3DDent%26aufirst%3DA.%2BL.%26aulast%3DShaffer%26aufirst%3DA.%2BL.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DAllman%26aufirst%3DD.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DControl%2520of%2520inflammation%252C%2520cytokine%2520expression%252C%2520and%2520germinal%2520center%2520formation%2520by%2520BCL-6%26jtitle%3DScience%26date%3D1997%26volume%3D276%26spage%3D589%26epage%3D592%26doi%3D10.1126%2Fscience.276.5312.589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Linterman, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinuesa, C. G.</span><span> </span><span class="NLM_article-title">Signals that influence T follicular helper cell differentiation and function</span> <span class="citation_source-journal">Semin. Immunopathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">196</span><span class="refDoi"> DOI: 10.1007/s00281-009-0194-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1007%2Fs00281-009-0194-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=20107805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVejt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2010&pages=183-196&author=M.+A.+Lintermanauthor=C.+G.+Vinuesa&title=Signals+that+influence+T+follicular+helper+cell+differentiation+and+function&doi=10.1007%2Fs00281-009-0194-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Signals that influence T follicular helper cell differentiation and function</span></div><div class="casAuthors">Linterman, Michelle A.; Vinuesa, Carola G.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Immunopathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-196</span>CODEN:
                <span class="NLM_cas:coden">SIEMB7</span>;
        ISSN:<span class="NLM_cas:issn">1863-2297</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Follicular helper T cells have recently emerged as a sep. CD4+ T helper lineage specialised in provision of help to B cells.  They develop independently from Th1, Th2 and Th17 cells and are crit. for humoral immunity, including the generation of long-lived and high affinity plasma cells and memory cells crucial for long-term protection against infections.  A stepwise differentiation program has emerged in which T cell receptor (TCR) signalling strength, CD28-mediated costimulation, B cell-derived inducible costimulator ligand signals, induction of c-maf and actions of cytokines, including interleukin (IL)-6 and IL-21, lead to upregulation of the transcriptional repressor B cell lymphoma 6 (Bcl-6) that drives T follicular helper (Tfh) cell differentiation.  Bcl-6 turns on a repression program that targets Blimp-1, transcriptional regulators of other helper lineages and microRNAs.  Their concerted actions modulate expression of chemokine receptors, surface mols. and cytokines crit. for follicular homing and B cell helper functions.  Here, we review the nature of Tfh cells providing help to B cells during the two phases of B cell activation that occur in the outer T zone and, for some B cells, in germinal centers (GC).  Recent insights into the signalling events that drive terminal differentiation of Tfh cells crit. for selecting somatically mutated GC B cells and the consequences of Tfh dysregulation for immunodeficiency and autoimmune pathol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDLrmacGcZ5rVg90H21EOLACvtfcHk0lj0XwCBSI74gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVejt7s%253D&md5=94a7bdf3d5a19f8490f41806de033f73</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2Fs00281-009-0194-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00281-009-0194-z%26sid%3Dliteratum%253Aachs%26aulast%3DLinterman%26aufirst%3DM.%2BA.%26aulast%3DVinuesa%26aufirst%3DC.%2BG.%26atitle%3DSignals%2520that%2520influence%2520T%2520follicular%2520helper%2520cell%2520differentiation%2520and%2520function%26jtitle%3DSemin.%2520Immunopathol.%26date%3D2010%26volume%3D32%26spage%3D183%26epage%3D196%26doi%3D10.1007%2Fs00281-009-0194-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Nurieva, R. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matskevitch, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, C.</span><span> </span><span class="NLM_article-title">Bcl6 mediates the development of T follicular helper cells</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">1005</span><span class="refDoi"> DOI: 10.1126/science.1176676</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1126%2Fscience.1176676" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=19628815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtValsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=1001-1005&author=R.+I.+Nurievaauthor=Y.+Chungauthor=G.+J.+Martinezauthor=X.+O.+Yangauthor=S.+Tanakaauthor=T.+D.+Matskevitchauthor=Y.+H.+Wangauthor=C.+Dong&title=Bcl6+mediates+the+development+of+T+follicular+helper+cells&doi=10.1126%2Fscience.1176676"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Bcl6 Mediates the Development of T Follicular Helper Cells</span></div><div class="casAuthors">Nurieva, Roza I.; Chung, Yeonseok; Martinez, Gustavo J.; Yang, Xuexian O.; Tanaka, Shinya; Matskevitch, Tatyana D.; Wang, Yi-Hong; Dong, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5943</span>),
    <span class="NLM_cas:pages">1001-1005</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A fundamental function of CD4+ helper T (TH) cells is the regulation of B cell-mediated humoral immunity.  Development of T follicular helper (TFH) cells that provide help to B cells is mediated by the cytokines interleukin-6 and interleukin-21 but is independent of TH1, TH2, and TH17 effector cell lineages.  Here, the authors characterize the function of Bcl6, a transcription factor selectively expressed in TFH cells.  Bcl6 expression is regulated by interleukin-6 and interleukin-21.  Bcl6 overexpression induced TFH-related gene expression and inhibited other TH lineage cell differentiation in a DNA binding-dependent manner.  Moreover, Bcl6 deficiency in T cells resulted in impaired TFH cell development and germinal center reactions, and altered prodn. of other effector T cell subsets.  The authors' data thus illustrate that Bcl6 is required for programming of TFH cell generation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr25TQ6m1zfs7Vg90H21EOLACvtfcHk0lhLKqOxGLPpNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtValsr3F&md5=3921d8d2b28d9d4ddec78f8033600fab</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1176676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1176676%26sid%3Dliteratum%253Aachs%26aulast%3DNurieva%26aufirst%3DR.%2BI.%26aulast%3DChung%26aufirst%3DY.%26aulast%3DMartinez%26aufirst%3DG.%2BJ.%26aulast%3DYang%26aufirst%3DX.%2BO.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DMatskevitch%26aufirst%3DT.%2BD.%26aulast%3DWang%26aufirst%3DY.%2BH.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DBcl6%2520mediates%2520the%2520development%2520of%2520T%2520follicular%2520helper%2520cells%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D1001%26epage%3D1005%26doi%3D10.1126%2Fscience.1176676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Xin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, R.</span><span> </span><span class="NLM_article-title">RNA interference targeting Bcl-6 ameliorates experimental autoimmune myasthenia gravis in mice</span> <span class="citation_source-journal">Mol. Cell. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">85</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span><span class="refDoi"> DOI: 10.1016/j.mcn.2013.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.mcn.2013.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=24361642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlCgs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=85-94&author=N.+Xinauthor=L.+Fuauthor=Z.+Shaoauthor=M.+Guoauthor=X.+Zhangauthor=Y.+Zhangauthor=C.+Douauthor=S.+Zhengauthor=X.+Shenauthor=Y.+Yaoauthor=J.+Wangauthor=J.+Wangauthor=G.+Cuiauthor=Y.+Liuauthor=D.+Gengauthor=C.+Xiaoauthor=Z.+Zhangauthor=R.+Dong&title=RNA+interference+targeting+Bcl-6+ameliorates+experimental+autoimmune+myasthenia+gravis+in+mice&doi=10.1016%2Fj.mcn.2013.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">RNA interference targeting Bcl-6 ameliorates experimental autoimmune myasthenia gravis in mice</span></div><div class="casAuthors">Xin, Ning; Fu, Linlin; Shao, Zhen; Guo, Mingfeng; Zhang, Xiuying; Zhang, Yong; Dou, Changxin; Zheng, Shuangshuang; Shen, Xia; Yao, Yuanhu; Wang, Jiao; Wang, Jinhua; Cui, Guiyun; Liu, Yonghai; Geng, Deqin; Xiao, Chenghua; Zhang, Zunsheng; Dong, Ruiguo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Neuroscience</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">85-94</span>CODEN:
                <span class="NLM_cas:coden">MOCNED</span>;
        ISSN:<span class="NLM_cas:issn">1044-7431</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Follicular helper T (Tfh) cells are dedicated to providing help to B cells and are strongly assocd. with antibody-mediated autoimmune disease.  B cell lymphoma 6 (Bcl-6) is a key transcription factor of Tfh cells, and IL-21 is known to be a crit. cytokine produced by Tfh cells.  We silenced Bcl-6 gene expression using RNA interference (RNAi) delivered by a lentiviral vector, to evaluate the therapeutic role of Bcl-6 short hairpin RNAs (shRNAs) in exptl. autoimmune myasthenia gravis (EAMG).  Our data demonstrate that CD4(+)CXCR5(+)PD-1(+) Tfh cells, Bcl-6 and IL-21 were significantly increased in EAMG mice, compared with controls.  In addn., we found that frequencies of Tfh cells were pos. correlated with the levels of serum anti-AChR Ab.  In-vivo transduction of lenti-siRNA-Bcl6 ameliorates the severity of ongoing EAMG with decreased Tfh cells, Bcl-6 and IL-21 expression, and leads to decreased anti-AChR antibody levels.  Furthermore, we found that siRNA knockdown of Bcl-6 expression increases the expression of Th1(IFN-γ, T-bet) and Th2 markers (IL-4 and GATA3), but failed to alter the expression of Th17-related markers (RORγt, IL-17) and Treg markers (FoxP3).  Our study suggests that Tfh cells contribute to the antibody prodn. and could be one of the most important T cell subsets responsible for development and progression of EAMG or MG.  Bcl-6 provides a promising therapeutic target for immunotherapy not only for MG, but also for other antibody-mediated autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHKM7UyeZWh7Vg90H21EOLACvtfcHk0lhLKqOxGLPpNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlCgs70%253D&md5=a7167314504da585f9ef8625bfdaa458</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.mcn.2013.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mcn.2013.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DXin%26aufirst%3DN.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDou%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGeng%26aufirst%3DD.%26aulast%3DXiao%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DDong%26aufirst%3DR.%26atitle%3DRNA%2520interference%2520targeting%2520Bcl-6%2520ameliorates%2520experimental%2520autoimmune%2520myasthenia%2520gravis%2520in%2520mice%26jtitle%3DMol.%2520Cell.%2520Neurosci.%26date%3D2014%26volume%3D58%26spage%3D85%26epage%3D94%26doi%3D10.1016%2Fj.mcn.2013.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Parekh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prive, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span> </span><span class="NLM_article-title">Therapeutic targeting of the BCL6 oncogene for diffuse large B-cell lymphomas</span> <span class="citation_source-journal">Leuk. Lymphoma</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">874</span><span class="NLM_x">–</span> <span class="NLM_lpage">882</span><span class="refDoi"> DOI: 10.1080/10428190801895345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1080%2F10428190801895345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=18452090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsFaqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2008&pages=874-882&author=S.+Parekhauthor=G.+Priveauthor=A.+Melnick&title=Therapeutic+targeting+of+the+BCL6+oncogene+for+diffuse+large+B-cell+lymphomas&doi=10.1080%2F10428190801895345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic targeting of the BCL6 oncogene for diffuse large B-cell lymphomas</span></div><div class="casAuthors">Parekh, Samir; Prive, Gilbert; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia & Lymphoma</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">874-882</span>CODEN:
                <span class="NLM_cas:coden">LELYEA</span>;
        ISSN:<span class="NLM_cas:issn">1042-8194</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  BCL6 is a transcriptional repressor often expressed constitutively in diffuse large B-cell lymphomas (DLBCL) due to mutations of its genomic locus.  BCL6 mediates aberrant survival, proliferation, genomic instability and differentiation blockade in DLBCL cells.  The biochem. study of BCL6 mediated gene repression has provided the basis for design of agents that inhibit BCL6 and kill lymphoma cells.  The repressor activity of the BCL6 BTB domain is particularly well defined from the structural standpoint.  Design of inhibitors targeting BCL6 BTB domain protein interaction surfaces appears to be an effective approach, which reactivates important BCL6 target genes and readily kills DLBCL cells.  Targeting other domains of BCL6 or using histone deacetylase inhibitors to overcome BCL6 mediated repression may also be useful.  Recent studies in DLBCL transcriptional signatures have revealed a subset of DLBCLs that are particularly dependent on BCL6 to maintain their survival and these patients could be candidates for clin. trials of BCL6 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsFT8NRceCBrVg90H21EOLACvtfcHk0lio4i70ySxGFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsFaqsrs%253D&md5=c528a71af6ba67489199c3bb1f6960fb</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1080%2F10428190801895345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10428190801895345%26sid%3Dliteratum%253Aachs%26aulast%3DParekh%26aufirst%3DS.%26aulast%3DPrive%26aufirst%3DG.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DTherapeutic%2520targeting%2520of%2520the%2520BCL6%2520oncogene%2520for%2520diffuse%2520large%2520B-cell%2520lymphomas%26jtitle%3DLeuk.%2520Lymphoma%26date%3D2008%26volume%3D49%26spage%3D874%26epage%3D882%26doi%3D10.1080%2F10428190801895345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Duy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurtz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shojaee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerchietti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaminathan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klemm, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kweon, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nahar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braig, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herzog, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jumaa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeffler, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heisterkamp, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graeber, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muschen, M.</span><span> </span><span class="NLM_article-title">BCL6 enables Ph+ acute lymphoblastic leukaemia cells to survive BCR-ABL1 kinase inhibition</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">473</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">388</span><span class="refDoi"> DOI: 10.1038/nature09883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1038%2Fnature09883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=21593872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtleltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=473&publication_year=2011&pages=384-388&author=C.+Duyauthor=C.+Hurtzauthor=S.+Shojaeeauthor=L.+Cerchiettiauthor=H.+Gengauthor=S.+Swaminathanauthor=L.+Klemmauthor=S.+M.+Kweonauthor=R.+Naharauthor=M.+Braigauthor=E.+Parkauthor=Y.+M.+Kimauthor=W.+K.+Hofmannauthor=S.+Herzogauthor=H.+Jumaaauthor=H.+P.+Koefflerauthor=J.+J.+Yuauthor=N.+Heisterkampauthor=T.+G.+Graeberauthor=H.+Wuauthor=B.+H.+Yeauthor=A.+Melnickauthor=M.+Muschen&title=BCL6+enables+Ph%2B+acute+lymphoblastic+leukaemia+cells+to+survive+BCR-ABL1+kinase+inhibition&doi=10.1038%2Fnature09883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">BCL6 enables Ph+ acute lymphoblastic leukaemia cells to survive BCR-ABL1 kinase inhibition</span></div><div class="casAuthors">Duy, Cihangir; Hurtz, Christian; Shojaee, Seyedmehdi; Cerchietti, Leandro; Geng, Huimin; Swaminathan, Srividya; Klemm, Lars; Kweon, Soo-mi; Nahar, Rahul; Braig, Melanie; Park, Eugene; Kim, Yong-mi; Hofmann, Wolf-Karsten; Herzog, Sebastian; Jumaa, Hassan; Koeffler, H. Phillip; Yu, J. Jessica; Heisterkamp, Nora; Graeber, Thomas G.; Wu, Hong; Ye, B. Hilda; Melnick, Ari; Mueschen, Markus</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">473</span>
        (<span class="NLM_cas:issue">7347</span>),
    <span class="NLM_cas:pages">384-388</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tyrosine kinase inhibitors (TKIs) are widely used to treat patients with leukemia driven by BCR-ABL1 (ref. 1) and other oncogenic tyrosine kinases.  Recent efforts have focused on developing more potent TKIs that also inhibit mutant tyrosine kinases.  However, even effective TKIs typically fail to eradicate leukemia-initiating cells (LICs), which often cause recurrence of leukemia after initially successful treatment.  Here we report the discovery of a novel mechanism of drug resistance, which is based on protective feedback signaling of leukemia cells in response to treatment with TKI.  We identify BCL6 as a central component of this drug-resistance pathway and demonstrate that targeted inhibition of BCL6 leads to eradication of drug-resistant and leukemia-initiating subclones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofwJ2MLGsWSLVg90H21EOLACvtfcHk0lio4i70ySxGFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtleltb4%253D&md5=24762f79c26b679694cd6fd524afed67</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature09883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09883%26sid%3Dliteratum%253Aachs%26aulast%3DDuy%26aufirst%3DC.%26aulast%3DHurtz%26aufirst%3DC.%26aulast%3DShojaee%26aufirst%3DS.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DKlemm%26aufirst%3DL.%26aulast%3DKweon%26aufirst%3DS.%2BM.%26aulast%3DNahar%26aufirst%3DR.%26aulast%3DBraig%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DKim%26aufirst%3DY.%2BM.%26aulast%3DHofmann%26aufirst%3DW.%2BK.%26aulast%3DHerzog%26aufirst%3DS.%26aulast%3DJumaa%26aufirst%3DH.%26aulast%3DKoeffler%26aufirst%3DH.%2BP.%26aulast%3DYu%26aufirst%3DJ.%2BJ.%26aulast%3DHeisterkamp%26aufirst%3DN.%26aulast%3DGraeber%26aufirst%3DT.%2BG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DYe%26aufirst%3DB.%2BH.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DMuschen%26aufirst%3DM.%26atitle%3DBCL6%2520enables%2520Ph%252B%2520acute%2520lymphoblastic%2520leukaemia%2520cells%2520to%2520survive%2520BCR-ABL1%2520kinase%2520inhibition%26jtitle%3DNature%26date%3D2011%26volume%3D473%26spage%3D384%26epage%3D388%26doi%3D10.1038%2Fnature09883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Ahmad, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lax, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouchard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiang, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Licht, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prive, G. G.</span><span> </span><span class="NLM_article-title">Mechanism of SMRT corepressor recruitment by the BCL6 BTB domain</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1551</span><span class="NLM_x">–</span> <span class="NLM_lpage">1564</span><span class="refDoi"> DOI: 10.1016/S1097-2765(03)00454-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2FS1097-2765%2803%2900454-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=14690607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsVOksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=1551-1564&author=K.+F.+Ahmadauthor=A.+Melnickauthor=S.+Laxauthor=D.+Bouchardauthor=J.+Liuauthor=C.+L.+Kiangauthor=S.+Mayerauthor=S.+Takahashiauthor=J.+D.+Lichtauthor=G.+G.+Prive&title=Mechanism+of+SMRT+corepressor+recruitment+by+the+BCL6+BTB+domain&doi=10.1016%2FS1097-2765%2803%2900454-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of SMRT corepressor recruitment by the BCL6 BTB domain</span></div><div class="casAuthors">Ahmad, K. Farid; Melnick, Ari; Lax, Stuart; Bouchard, Denis; Liu, Jun; Kiang, Chih-li; Mayer, Sebastian; Takahashi, Shinichiro; Licht, Jonathan D.; Prive, Gilbert G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1551-1564</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BCL6 encodes a transcription factor that represses genes necessary for the terminal differentiation of lymphocytes within germinal centers, and the mis-regulated expression of this factor is strongly implicated in several types of B cell lymphoma.  The homodimeric BTB domain of BCL6 (also known as the POZ domain) is required for the repression activity of the protein and interacts directly with the SMRT and N-CoR corepressors that are found within large multiprotein histone deacetylase-contg. complexes.  We have identified a 17 residue fragment from SMRT that binds to the BCL6 BTB domain, and detd. the crystal structure of the complex to 2.2 Å.  Two SMRT fragments bind sym. to the BCL6 BTB homodimer and, in combination with biochem. and in vivo data, the structure provides insight into the basis of transcriptional repression by this crit. B cell lymphoma protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGmXOfYEe7Z7Vg90H21EOLACvtfcHk0liJmGNf87mf6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsVOksA%253D%253D&md5=1120b558b43f6714104eb8bed425f44b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2803%2900454-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252803%252900454-4%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DK.%2BF.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DLax%26aufirst%3DS.%26aulast%3DBouchard%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DKiang%26aufirst%3DC.%2BL.%26aulast%3DMayer%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DPrive%26aufirst%3DG.%2BG.%26atitle%3DMechanism%2520of%2520SMRT%2520corepressor%2520recruitment%2520by%2520the%2520BCL6%2520BTB%2520domain%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D12%26spage%3D1551%26epage%3D1564%26doi%3D10.1016%2FS1097-2765%2803%2900454-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Cerchietti, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghetu, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da Silva, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunting, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fares, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prive, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span> </span><span class="NLM_article-title">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">400</span><span class="NLM_x">–</span> <span class="NLM_lpage">411</span><span class="refDoi"> DOI: 10.1016/j.ccr.2009.12.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.ccr.2009.12.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=20385364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=400-411&author=L.+C.+Cerchiettiauthor=A.+F.+Ghetuauthor=X.+Zhuauthor=G.+F.+Da+Silvaauthor=S.+Zhongauthor=M.+Matthewsauthor=K.+L.+Buntingauthor=J.+M.+Poloauthor=C.+Faresauthor=C.+H.+Arrowsmithauthor=S.+N.+Yangauthor=M.+Garciaauthor=A.+Coopauthor=A.+D.+Mackerellauthor=G.+G.+Priveauthor=A.+Melnick&title=A+small-molecule+inhibitor+of+BCL6+kills+DLBCL+cells+in+vitro+and+in+vivo&doi=10.1016%2Fj.ccr.2009.12.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Ghetu, Alexandru F.; Zhu, Xiao; Da Silva, Gustavo F.; Zhong, Shijun; Matthews, Marilyn; Bunting, Karen L.; Polo, Jose M.; Fares, Christophe; Arrowsmith, Cheryl H.; Yang, Shao Ning; Garcia, Monica; Coop, Andrew; MacKerell, Alexander D., Jr.; Prive, Gilbert G.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">400-411</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most frequently involved oncogene in diffuse large B cell lymphoma (DLBCL).  We combined computer-aided drug design with functional assays to identify low-mol.-wt. compds. that bind to the corepressor binding groove of the BCL6 BTB domain.  One such compd. disrupted BCL6/corepressor complexes in vitro and in vivo, and was obsd. by X-ray crystallog. and NMR to bind the crit. site within the BTB groove.  This compd. could induce expression of BCL6 target genes and kill BCL6-pos. DLBCL cell lines.  In xenotransplantation expts., the compd. was nontoxic and potently suppressed DLBCL tumors in vivo.  The compd. also killed primary DLBCLs from human patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdcScjNPjhiLVg90H21EOLACvtfcHk0lgW3y-kpMlL8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D&md5=64f30a813c26394b69da6de95991f8dd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.12.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.12.050%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DGhetu%26aufirst%3DA.%2BF.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDa%2BSilva%26aufirst%3DG.%2BF.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DMatthews%26aufirst%3DM.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DFares%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DGarcia%26aufirst%3DM.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DPrive%26aufirst%3DG.%2BG.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520small-molecule%2520inhibitor%2520of%2520BCL6%2520kills%2520DLBCL%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26spage%3D400%26epage%3D411%26doi%3D10.1016%2Fj.ccr.2009.12.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Cardenas, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beguelin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teater, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldstein, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oswald, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaknovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbott, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elemento, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerchietti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cierpicki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A. M.</span><span> </span><span class="NLM_article-title">Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">3351</span><span class="NLM_x">–</span> <span class="NLM_lpage">3362</span><span class="refDoi"> DOI: 10.1172/JCI85795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1172%2FJCI85795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=27482887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BC2s3mvFagsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2016&pages=3351-3362&author=M.+G.+Cardenasauthor=W.+Yuauthor=W.+Beguelinauthor=M.+R.+Teaterauthor=H.+Gengauthor=R.+L.+Goldsteinauthor=E.+Oswaldauthor=K.+Hatziauthor=S.+N.+Yangauthor=J.+Cohenauthor=R.+Shaknovichauthor=K.+Vanommeslaegheauthor=H.+Chengauthor=D.+Liangauthor=H.+J.+Choauthor=J.+Abbottauthor=W.+Tamauthor=W.+Duauthor=J.+P.+Leonardauthor=O.+Elementoauthor=L.+Cerchiettiauthor=T.+Cierpickiauthor=F.+Xueauthor=A.+D.+MacKerellauthor=A.+M.+Melnick&title=Rationally+designed+BCL6+inhibitors+target+activated+B+cell+diffuse+large+B+cell+lymphoma&doi=10.1172%2FJCI85795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</span></div><div class="casAuthors">Cardenas Mariano G; Yu Wenbo; Beguelin Wendy; Teater Matthew R; Geng Huimin; Goldstein Rebecca L; Oswald Erin; Hatzi Katerina; Yang Shao-Ning; Cohen Joanna; Shaknovich Rita; Vanommeslaeghe Kenno; Cheng Huimin; Liang Dongdong; Cho Hyo Je; Abbott Joshua; Tam Wayne; Du Wei; Leonard John P; Elemento Olivier; Cerchietti Leandro; Cierpicki Tomasz; Xue Fengtian; MacKerell Alexander D Jr; Melnick Ari M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3351-62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Diffuse large B cell lymphomas (DLBCLs) arise from proliferating B cells transiting different stages of the germinal center reaction.  In activated B cell DLBCLs (ABC-DLBCLs), a class of DLBCLs that respond poorly to current therapies, chromosomal translocations and amplification lead to constitutive expression of the B cell lymphoma 6 (BCL6) oncogene.  The role of BCL6 in maintaining these lymphomas has not been investigated.  Here, we designed small-molecule inhibitors that display higher affinity for BCL6 than its endogenous corepressor ligands to evaluate their therapeutic efficacy for targeting ABC-DLBCL.  We used an in silico drug design functional-group mapping approach called SILCS to create a specific BCL6 inhibitor called FX1 that has 10-fold greater potency than endogenous corepressors and binds an essential region of the BCL6 lateral groove.  FX1 disrupted formation of the BCL6 repression complex, reactivated BCL6 target genes, and mimicked the phenotype of mice engineered to express BCL6 with corepressor binding site mutations.  Low doses of FX1 induced regression of established tumors in mice bearing DLBCL xenografts.  Furthermore, FX1 suppressed ABC-DLBCL cells in vitro and in vivo, as well as primary human ABC-DLBCL specimens ex vivo.  These findings indicate that ABC-DLBCL is a BCL6-dependent disease that can be targeted by rationally designed inhibitors that exceed the binding affinity of natural BCL6 ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAVUzYiXt5ZL5XHngOXiV_fW6udTcc2eZJH_H70zpdZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3mvFagsw%253D%253D&md5=90a313fd7adce1922773212f0960bad7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1172%2FJCI85795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI85795%26sid%3Dliteratum%253Aachs%26aulast%3DCardenas%26aufirst%3DM.%2BG.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBeguelin%26aufirst%3DW.%26aulast%3DTeater%26aufirst%3DM.%2BR.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DGoldstein%26aufirst%3DR.%2BL.%26aulast%3DOswald%26aufirst%3DE.%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DCohen%26aufirst%3DJ.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DD.%26aulast%3DCho%26aufirst%3DH.%2BJ.%26aulast%3DAbbott%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DW.%26aulast%3DDu%26aufirst%3DW.%26aulast%3DLeonard%26aufirst%3DJ.%2BP.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DXue%26aufirst%3DF.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26atitle%3DRationally%2520designed%2520BCL6%2520inhibitors%2520target%2520activated%2520B%2520cell%2520diffuse%2520large%2520B%2520cell%2520lymphoma%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2016%26volume%3D126%26spage%3D3351%26epage%3D3362%26doi%3D10.1172%2FJCI85795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Cerchietti, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaknovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chadburn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowdy, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span> </span><span class="NLM_article-title">A peptomimetic inhibitor of BCL6 with potent antilymphoma effects in vitro and in vivo</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">3397</span><span class="NLM_x">–</span> <span class="NLM_lpage">3405</span><span class="refDoi"> DOI: 10.1182/blood-2008-07-168773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1182%2Fblood-2008-07-168773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=18927431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslCkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=3397-3405&author=L.+C.+Cerchiettiauthor=S.+N.+Yangauthor=R.+Shaknovichauthor=K.+Hatziauthor=J.+M.+Poloauthor=A.+Chadburnauthor=S.+F.+Dowdyauthor=A.+Melnick&title=A+peptomimetic+inhibitor+of+BCL6+with+potent+antilymphoma+effects+in+vitro+and+in+vivo&doi=10.1182%2Fblood-2008-07-168773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A peptomimetic inhibitor of BCL6 with potent antilymphoma effects in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Yang, Shao Ning; Shaknovich, Rita; Hatzi, Katerina; Polo, Jose M.; Chadburn, Amy; Dowdy, Steven F.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3397-3405</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most commonly involved oncogene in diffuse large B-cell lymphomas (DLBCLs).  BCL6 lymphomagenic activity is dependent on its ability to recruit corepressor proteins to a unique binding site on its N-terminal BTB domain.  A recombinant peptide fragment of the SMRT (silencing mediator for retinoid and thyroid hormone receptor) corepressor that blocks this site can inhibit BCL6 biol. functions.  Shortening and conversion of this peptide to D-amino acid and retro configuration as well as the addn. of a fusogenic motif yielded a far more potent and stable BCL6 inhibitor that still retained the specificity of the original SMRT fragment.  Like the L-peptide, retroinverso BCL6 peptide inhibitor (RI-BPI) selectively killed BCR rather than OxPhos-type DLBCL cells.  The RI-BPI could recapitulate the failure to form germinal centers seen in BCL6 null mice yet was nontoxic and non-immunogenic even when administered for up to 52 wk.  RI-BPI showed superior duration of tissue penetration and could accordingly powerfully suppress the growth of human DLBCLs xenografts in a dose-dependent manner.  Finally, RI-BPI could kill primary human DLBCL cells but had no effect on normal lymphoid tissue or other tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxsckch1D41rVg90H21EOLACvtfcHk0li6mnoLzZOgEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslCkurw%253D&md5=8a12df08b153ba0712b5d06f9bc550c2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1182%2Fblood-2008-07-168773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2008-07-168773%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DChadburn%26aufirst%3DA.%26aulast%3DDowdy%26aufirst%3DS.%2BF.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520peptomimetic%2520inhibitor%2520of%2520BCL6%2520with%2520potent%2520antilymphoma%2520effects%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26spage%3D3397%26epage%3D3405%26doi%3D10.1182%2Fblood-2008-07-168773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Evans, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goult, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairall, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamieson, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko Ferrigno, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwabe, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, S. D.</span><span> </span><span class="NLM_article-title">The ansamycin antibiotic, rifamycin SV, inhibits BCL6 transcriptional repression and forms a complex with the BCL6-BTB/POZ domain</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e90889</span><span class="refDoi"> DOI: 10.1371/journal.pone.0090889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1371%2Fjournal.pone.0090889" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e90889&author=S.+E.+Evansauthor=B.+T.+Goultauthor=L.+Fairallauthor=A.+G.+Jamiesonauthor=P.+Ko+Ferrignoauthor=R.+Fordauthor=J.+W.+Schwabeauthor=S.+D.+Wagner&title=The+ansamycin+antibiotic%2C+rifamycin+SV%2C+inhibits+BCL6+transcriptional+repression+and+forms+a+complex+with+the+BCL6-BTB%2FPOZ+domain&doi=10.1371%2Fjournal.pone.0090889"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0090889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0090889%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DS.%2BE.%26aulast%3DGoult%26aufirst%3DB.%2BT.%26aulast%3DFairall%26aufirst%3DL.%26aulast%3DJamieson%26aufirst%3DA.%2BG.%26aulast%3DKo%2BFerrigno%26aufirst%3DP.%26aulast%3DFord%26aufirst%3DR.%26aulast%3DSchwabe%26aufirst%3DJ.%2BW.%26aulast%3DWagner%26aufirst%3DS.%2BD.%26atitle%3DThe%2520ansamycin%2520antibiotic%252C%2520rifamycin%2520SV%252C%2520inhibits%2520BCL6%2520transcriptional%2520repression%2520and%2520forms%2520a%2520complex%2520with%2520the%2520BCL6-BTB%252FPOZ%2520domain%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De90889%26doi%3D10.1371%2Fjournal.pone.0090889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Sakamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sogabe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakai, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asano, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imaeda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakamoto, J. I.</span><span> </span><span class="NLM_article-title">Discovery of high-affinity BCL6-binding peptide and its structure-activity relationship</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">482</span><span class="NLM_x">, </span> <span class="NLM_fpage">310</span><span class="NLM_x">–</span> <span class="NLM_lpage">316</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2016.11.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.bbrc.2016.11.060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=27856253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGgt7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2017&pages=310-316&author=K.+Sakamotoauthor=S.+Sogabeauthor=Y.+Kamadaauthor=N.+Sakaiauthor=K.+Asanoauthor=M.+Yoshimatsuauthor=K.+Idaauthor=Y.+Imaedaauthor=J.+I.+Sakamoto&title=Discovery+of+high-affinity+BCL6-binding+peptide+and+its+structure-activity+relationship&doi=10.1016%2Fj.bbrc.2016.11.060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of high-affinity BCL6-binding peptide and its structure-activity relationship</span></div><div class="casAuthors">Sakamoto, Kotaro; Sogabe, Satoshi; Kamada, Yusuke; Sakai, Nozomu; Asano, Kouhei; Yoshimatsu, Mie; Ida, Kou; Imaeda, Yasuhiro; Sakamoto, Jun-ichi</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">482</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">310-316</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">B cell lymphoma 6 (BCL6) is a transcriptional repressor that interacts with its corepressors BcoR and SMRT.  Since this protein-protein interaction (PPI) induces activation and differentiation of B lymphocytes, BCL6 has been an attractive drug target for potential autoimmune disease treatments.  Here we report a novel BCL6 inhibitory peptide, F1324 (Ac-LWYTDIRMSWRVP-OH), which we discovered using phage display technol.; we also discuss this peptide's structure-activity relationship (SAR).  For BCL6(5-129) binding, KD and IC50 values of F1324 were 0.57 nM and 1 nM according to the results of an SPR anal. and cell-free ELISA assay, resp.  In contrast, BcoR(Arg498-514Pro) and SMRT(Leu1422-Arg1438) exhibited relatively weak micromole-order binding to BCL6.  Furthermore, Fusion protein AcGFP-F1324 transiently expressed in HEK293T cells inhibited intracellular PPI in cell-based M2H assay.  By examn. of the truncation and fragmentation of F1324, the C-terminal sequence WRVP, which is similar to the BcoR(509-512) sequence WVVP, was identified as being crit. for BCL6 binding.  In addn., subsequent single-crystal X-ray diffraction anal. of F1324/BCL6(5-129) complex revealed that the high affinity of F1324 was caused by effective interaction of its side chains while its main chain structure was similar to that of BcoR(Arg498-514Pro).  To our knowledge, F1324 is the strongest BCL6-binding peptide yet reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7xCi3L_jL-bVg90H21EOLACvtfcHk0ljolPGY_iVU8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGgt7nJ&md5=a543acd613f9b63efa9c814a75e6b5f6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.11.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.11.060%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DSakai%26aufirst%3DN.%26aulast%3DAsano%26aufirst%3DK.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DIda%26aufirst%3DK.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DSakamoto%26aufirst%3DJ.%2BI.%26atitle%3DDiscovery%2520of%2520high-affinity%2520BCL6-binding%2520peptide%2520and%2520its%2520structure-activity%2520relationship%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2017%26volume%3D482%26spage%3D310%26epage%3D316%26doi%3D10.1016%2Fj.bbrc.2016.11.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahnke, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0ljolPGY_iVU8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Ludlow, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saini, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tickle, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Detection of secondary binding sites in proteins using fragment screening</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">15910</span><span class="NLM_x">–</span> <span class="NLM_lpage">15915</span><span class="refDoi"> DOI: 10.1073/pnas.1518946112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1073%2Fpnas.1518946112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=26655740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15910-15915&author=R.+F.+Ludlowauthor=M.+L.+Verdonkauthor=H.+K.+Sainiauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Detection+of+secondary+binding+sites+in+proteins+using+fragment+screening&doi=10.1073%2Fpnas.1518946112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of secondary binding sites in proteins using fragment screening</span></div><div class="casAuthors">Ludlow, R. Frederick; Verdonk, Marcel L.; Saini, Harpreet K.; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">15910-15915</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Proteins need to be tightly regulated as they control biol. processes in most normal cellular functions.  The precise mechanisms of regulation are rarely completely understood but can involve binding of endogenous ligands and/or partner proteins at specific locations on a protein that can modulate function.  Often, these addnl. secondary binding sites appear sep. to the primary binding site, which, e.g., for an enzyme may bind a substrate.  In previous work, the authors uncovered several examples in which secondary binding sites were discovered on proteins using fragment screening approaches.  In each case, the authors were able to establish that the newly identified secondary binding site was biol. relevant as it was able to modulate function by the binding of a small mol.  Here, the authors investigated how often secondary binding sites are located on proteins by analyzing 24 protein targets for which they performed a fragment screen using x-ray crystallog.  The anal. showed that, surprisingly, the majority of proteins contained secondary binding sites based on their ability to bind fragments.  Furthermore, sequence anal. of these previously unknown sites indicated high conservation, which suggested that they may have a biol. function, perhaps via an allosteric mechanism.  Comparing the physicochem. properties of the secondary sites with known primary ligand-binding sites also showed broad similarities indicating that many of the secondary sites may be druggable in nature with small mols. that could provide new opportunities to modulate potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ITF1w230fbVg90H21EOLACvtfcHk0ljYPYsJZt_Z3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP&md5=9e5f15595263dc049c713d5a3f5912a1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1518946112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1518946112%26sid%3Dliteratum%253Aachs%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DSaini%26aufirst%3DH.%2BK.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520secondary%2520binding%2520sites%2520in%2520proteins%2520using%2520fragment%2520screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D15910%26epage%3D15915%26doi%3D10.1073%2Fpnas.1518946112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Jordan, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittington, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartberger, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sickmier, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judd, T.</span><span> </span><span class="NLM_article-title">Fragment-linking approach using (19)F NMR spectroscopy to obtain highly potent and selective inhibitors of beta-secretase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">3732</span><span class="NLM_x">–</span> <span class="NLM_lpage">3749</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01917</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01917" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFeitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3732-3749&author=J.+B.+Jordanauthor=D.+A.+Whittingtonauthor=M.+D.+Bartbergerauthor=E.+A.+Sickmierauthor=K.+Chenauthor=Y.+Chengauthor=T.+Judd&title=Fragment-linking+approach+using+%2819%29F+NMR+spectroscopy+to+obtain+highly+potent+and+selective+inhibitors+of+beta-secretase&doi=10.1021%2Facs.jmedchem.5b01917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Linking Approach Using 19F NMR Spectroscopy To Obtain Highly Potent and Selective Inhibitors of β-Secretase</span></div><div class="casAuthors">Jordan, John B.; Whittington, Douglas A.; Bartberger, Michael D.; Sickmier, E. Allen; Chen, Kui; Cheng, Yuan; Judd, Ted</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3732-3749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) has become a widely used tool in small-mol. drug discovery efforts.  One of the most commonly used biophys. methods in detecting weak binding of fragments is NMR (NMR) spectroscopy.  In particular, FBDD performed with 19F NMR-based methods has been shown to provide several advantages over 1H NMR using traditional magnetization-transfer and/or two-dimensional methods.  Here, we demonstrate the utility and power of 19F-based fragment screening by detailing the identification of a second-site fragment through 19F NMR screening that binds to a specific pocket of the aspartic acid protease, β-secretase (BACE-1).  The identification of this second-site fragment allowed the undertaking of a fragment-linking approach, which ultimately yielded a mol. exhibiting a more than 360-fold increase in potency while maintaining reasonable ligand efficiency and gaining much improved selectivity over cathepsin-D (CatD).  X-ray crystallog. studies of the mols. demonstrated that the linked fragments exhibited binding modes consistent with those predicted from the targeted screening approach, through-space NMR data, and mol. modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq95M4k5aaQurVg90H21EOLACvtfcHk0ljT1UWRZEnWUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFeitr8%253D&md5=6bd6021938c555d8c7083549e4b49a2e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01917%26sid%3Dliteratum%253Aachs%26aulast%3DJordan%26aufirst%3DJ.%2BB.%26aulast%3DWhittington%26aufirst%3DD.%2BA.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DSickmier%26aufirst%3DE.%2BA.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DJudd%26aufirst%3DT.%26atitle%3DFragment-linking%2520approach%2520using%2520%252819%2529F%2520NMR%2520spectroscopy%2520to%2520obtain%2520highly%2520potent%2520and%2520selective%2520inhibitors%2520of%2520beta-secretase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3732%26epage%3D3749%26doi%3D10.1021%2Facs.jmedchem.5b01917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Kohlmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zech, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Squillace, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Commodore, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenfield, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodd, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miret, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clackson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shakespeare, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalgarno, D. C.</span><span> </span><span class="NLM_article-title">Fragment growing and linking lead to novel nanomolar lactate dehydrogenase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1023</span><span class="NLM_x">–</span> <span class="NLM_lpage">1040</span><span class="refDoi"> DOI: 10.1021/jm3014844</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3014844" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3sXms1OlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1023-1040&author=A.+Kohlmannauthor=S.+G.+Zechauthor=F.+Liauthor=T.+Zhouauthor=R.+M.+Squillaceauthor=L.+Commodoreauthor=M.+T.+Greenfieldauthor=X.+Luauthor=D.+P.+Millerauthor=W.+S.+Huangauthor=J.+Qiauthor=R.+M.+Thomasauthor=Y.+Wangauthor=S.+Zhangauthor=R.+Doddauthor=S.+Liuauthor=R.+Xuauthor=Y.+Xuauthor=J.+J.+Miretauthor=V.+Riveraauthor=T.+Clacksonauthor=W.+C.+Shakespeareauthor=X.+Zhuauthor=D.+C.+Dalgarno&title=Fragment+growing+and+linking+lead+to+novel+nanomolar+lactate+dehydrogenase+inhibitors&doi=10.1021%2Fjm3014844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Growing and Linking Lead to Novel Nanomolar Lactate Dehydrogenase Inhibitors</span></div><div class="casAuthors">Kohlmann, Anna; Zech, Stephan G.; Li, Feng; Zhou, Tianjun; Squillace, Rachel M.; Commodore, Lois; Greenfield, Matthew T.; Lu, Xiaohui; Miller, David P.; Huang, Wei-Sheng; Qi, Jiwei; Thomas, R. Mathew; Wang, Yihan; Zhang, Sen; Dodd, Rory; Liu, Shuangying; Xu, Rongsong; Xu, Yongjin; Miret, Juan J.; Rivera, Victor; Clackson, Tim; Shakespeare, William C.; Zhu, Xiaotian; Dalgarno, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1023-1040</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lactate dehydrogenase A (LDH-A) catalyzes the interconversion of lactate and pyruvate in the glycolysis pathway.  Cancer cells rely heavily on glycolysis instead of oxidative phosphorylation to generate ATP, a phenomenon known as the Warburg effect.  The inhibition of LDH-A by small mols. is therefore of interest for potential cancer treatments.  We describe the identification and optimization of LDH-A inhibitors by fragment-based drug discovery.  We applied ligand based NMR screening to identify low affinity fragments binding to LDH-A.  The dissocn. consts. (Kd) and enzyme inhibition (IC50) of fragment hits were measured by surface plasmon resonance (SPR) and enzyme assays, resp.  The binding modes of selected fragments were investigated by X-ray crystallog.  Fragment growing and linking, followed by chem. optimization, resulted in nanomolar LDH-A inhibitors that demonstrated stoichiometric binding to LDH-A.  Selected mols. inhibited lactate prodn. in cells, suggesting target-specific inhibition in cancer cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pFkSoe2_ALVg90H21EOLACvtfcHk0ljT1UWRZEnWUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXms1OlsA%253D%253D&md5=3a6b0b46318ae2d6c2a7c34e1575ada6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm3014844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3014844%26sid%3Dliteratum%253Aachs%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DZech%26aufirst%3DS.%2BG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DSquillace%26aufirst%3DR.%2BM.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DGreenfield%26aufirst%3DM.%2BT.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DMiller%26aufirst%3DD.%2BP.%26aulast%3DHuang%26aufirst%3DW.%2BS.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DR.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DDodd%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMiret%26aufirst%3DJ.%2BJ.%26aulast%3DRivera%26aufirst%3DV.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26atitle%3DFragment%2520growing%2520and%2520linking%2520lead%2520to%2520novel%2520nanomolar%2520lactate%2520dehydrogenase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1023%26epage%3D1040%26doi%3D10.1021%2Fjm3014844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Larsson, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Then, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panicker, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sangthongpitag, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendharkar, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tai, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheong, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulsen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alam, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span> </span><span class="NLM_article-title">Fragment-based ligand design of novel potent inhibitors of tankyrases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">4497</span><span class="NLM_x">–</span> <span class="NLM_lpage">4508</span><span class="refDoi"> DOI: 10.1021/jm400211f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400211f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsFartr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=4497-4508&author=E.+A.+Larssonauthor=A.+Janssonauthor=F.+M.+Ngauthor=S.+W.+Thenauthor=R.+Panickerauthor=B.+Liuauthor=K.+Sangthongpitagauthor=V.+Pendharkarauthor=S.+J.+Taiauthor=J.+Hillauthor=C.+Danauthor=S.+Y.+Hoauthor=W.+W.+Cheongauthor=A.+Poulsenauthor=S.+Blanchardauthor=G.+R.+Linauthor=J.+Alamauthor=T.+H.+Kellerauthor=P.+Nordlund&title=Fragment-based+ligand+design+of+novel+potent+inhibitors+of+tankyrases&doi=10.1021%2Fjm400211f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based ligand design of novel potent inhibitors of tankyrases</span></div><div class="casAuthors">Larsson, E. Andreas; Jansson, Anna; Ng, Fui Mee; Then, Siew Wen; Panicker, Resmi; Liu, Boping; Sangthongpitag, Kanda; Pendharkar, Vishal; Tai, Shi Jing; Hill, Jeffrey; Dan, Chen; Ho, Soo Yei; Cheong, Wei Wen; Poulsen, Anders; Blanchard, Stephanie; Lin, Grace Ruiting; Alam, Jenefer; Keller, Thomas H.; Nordlund, Par</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4497-4508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tankyrases constitute potential drug targets for cancer and myelin-degrading diseases.  We have applied a structure- and biophysics-driven fragment-based ligand design strategy to discover a novel family of potent inhibitors for human tankyrases.  Biophys. screening based on a thermal shift assay identified highly efficient fragments binding in the nicotinamide-binding site, a local hot spot for fragment binding.  Evolution of the fragment hit 4-methyl-1,2-dihydroquinolin-2-one (2) along its 7-vector yields dramatic affinity improvements in the first cycle of expansion.  A crystal structure of 7-(2-fluorophenyl)-4-methylquinolin-2(1H)-one (11) reveals that the nonplanar compd. extends with its fluorine atom into a pocket, which coincides with a region of the active site where structural differences are seen between tankyrases and other poly(ADP-ribose) polymerase (PARP) family members.  A further cycle of optimization yielded compds. with affinities and IC50 values in the low nanomolar range and with good soly., PARP selectivity, and ligand efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQxampX4n-xLVg90H21EOLACvtfcHk0lje8G0Rkvj5bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsFartr4%253D&md5=4cd39f8715bfc51807129bb04da1e6f0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm400211f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400211f%26sid%3Dliteratum%253Aachs%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DJansson%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DF.%2BM.%26aulast%3DThen%26aufirst%3DS.%2BW.%26aulast%3DPanicker%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSangthongpitag%26aufirst%3DK.%26aulast%3DPendharkar%26aufirst%3DV.%26aulast%3DTai%26aufirst%3DS.%2BJ.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DHo%26aufirst%3DS.%2BY.%26aulast%3DCheong%26aufirst%3DW.%2BW.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DBlanchard%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DG.%2BR.%26aulast%3DAlam%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DFragment-based%2520ligand%2520design%2520of%2520novel%2520potent%2520inhibitors%2520of%2520tankyrases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D4497%26epage%3D4508%26doi%3D10.1021%2Fjm400211f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Wielens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headey, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulder, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolezal, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deadman, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalmers, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peat, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scanlon, M. J.</span><span> </span><span class="NLM_article-title">Parallel screening of low molecular weight fragment libraries: do differences in methodology affect hit identification?</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">147</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1177/1087057112465979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057112465979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=23139382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BC3s7itFGluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=147-159&author=J.+Wielensauthor=S.+J.+Headeyauthor=D.+I.+Rhodesauthor=R.+J.+Mulderauthor=O.+Dolezalauthor=J.+J.+Deadmanauthor=J.+Newmanauthor=D.+K.+Chalmersauthor=M.+W.+Parkerauthor=T.+S.+Peatauthor=M.+J.+Scanlon&title=Parallel+screening+of+low+molecular+weight+fragment+libraries%3A+do+differences+in+methodology+affect+hit+identification%3F&doi=10.1177%2F1087057112465979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel screening of low molecular weight fragment libraries: do differences in methodology affect hit identification?</span></div><div class="casAuthors">Wielens Jerome; Headey Stephen J; Rhodes David I; Mulder Roger J; Dolezal Olan; Deadman John J; Newman Janet; Chalmers David K; Parker Michael W; Peat Thomas S; Scanlon Martin J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">147-59</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment screening is becoming widely accepted as a technique to identify hit compounds for the development of novel lead compounds.  In neighboring laboratories, we have recently, and independently, performed a fragment screening campaign on the HIV-1 integrase core domain (IN) using similar commercially purchased fragment libraries.  The two campaigns used different screening methods for the preliminary identification of fragment hits; one used saturation transfer difference nuclear magnetic resonance spectroscopy (STD-NMR), and the other used surface plasmon resonance (SPR) spectroscopy.  Both initial screens were followed by X-ray crystallography.  Using the STD-NMR/X-ray approach, 15 IN/fragment complexes were identified, whereas the SPR/X-ray approach found 6 complexes.  In this article, we compare the approaches that were taken by each group and the results obtained, and we look at what factors could potentially influence the final results.  We find that despite using different approaches with little overlap of initial hits, both approaches identified binding sites on IN that provided a basis for fragment-based lead discovery and further lead development.  Comparison of hits identified in the two studies highlights a key role for both the conditions under which fragment binding is measured and the criteria selected to classify hits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaYsG71Pu6x669lAbTj9rGfW6udTcc2eZJ1ickHHamT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7itFGluw%253D%253D&md5=9061e308aecbffc35d50a29b5b682283</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1087057112465979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057112465979%26sid%3Dliteratum%253Aachs%26aulast%3DWielens%26aufirst%3DJ.%26aulast%3DHeadey%26aufirst%3DS.%2BJ.%26aulast%3DRhodes%26aufirst%3DD.%2BI.%26aulast%3DMulder%26aufirst%3DR.%2BJ.%26aulast%3DDolezal%26aufirst%3DO.%26aulast%3DDeadman%26aufirst%3DJ.%2BJ.%26aulast%3DNewman%26aufirst%3DJ.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26aulast%3DParker%26aufirst%3DM.%2BW.%26aulast%3DPeat%26aufirst%3DT.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DParallel%2520screening%2520of%2520low%2520molecular%2520weight%2520fragment%2520libraries%253A%2520do%2520differences%2520in%2520methodology%2520affect%2520hit%2520identification%253F%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D147%26epage%3D159%26doi%3D10.1177%2F1087057112465979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browner, M. F.</span><span> </span><span class="NLM_article-title">Surface plasmon resonance based assay for the detection and characterization of promiscuous inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">580</span><span class="refDoi"> DOI: 10.1021/jm700952v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700952v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=574-580&author=A.+M.+Giannettiauthor=B.+D.+Kochauthor=M.+F.+Browner&title=Surface+plasmon+resonance+based+assay+for+the+detection+and+characterization+of+promiscuous+inhibitors&doi=10.1021%2Fjm700952v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Plasmon Resonance Based Assay for the Detection and Characterization of Promiscuous Inhibitors</span></div><div class="casAuthors">Giannetti, Anthony M.; Koch, Bruce D.; Browner, Michelle F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-580</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous binders achieve enzyme inhibition using a nonspecific aggregation-type binding mechanism to proteins.  These compds. are a source of false-pos. hits in biochem. inhibition assays and should be removed from screening hit lists because they are not good candidates to initiate medicinal chem. programs.  The authors introduce a robust approach to identify these mols. early in the lead generation process using real time surface plasmon resonance based biosensors to observe the behavior of the binding interactions between promiscuous compds. and proteins.  Furthermore, the time resoln. of the assay reveals a no. of distinct mechanisms that promiscuous compds. employ to inhibit enzyme function and indicate that the type of mechanism can vary depending on the protein target.  A classification scheme for these compds. is presented that can be used to rapidly characterize the hits from high-throughput screens and eliminate compds. with a nonspecific mechanism of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVTh2kYyROf7Vg90H21EOLACvtfcHk0ljMQSHDcXeQQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D&md5=b2869c5be1cd6f455503f8e812ed6f23</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm700952v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700952v%26sid%3Dliteratum%253Aachs%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DKoch%26aufirst%3DB.%2BD.%26aulast%3DBrowner%26aufirst%3DM.%2BF.%26atitle%3DSurface%2520plasmon%2520resonance%2520based%2520assay%2520for%2520the%2520detection%2520and%2520characterization%2520of%2520promiscuous%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D574%26epage%3D580%26doi%3D10.1021%2Fjm700952v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Retra, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irth, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Muijlwijk-Koezen, J. E.</span><span> </span><span class="NLM_article-title">Surface plasmon resonance biosensor analysis as a useful tool in FBDD</span> <span class="citation_source-journal">Drug Discovery Today: Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e181</span><span class="NLM_x">–</span> <span class="NLM_lpage">e187</span><span class="refDoi"> DOI: 10.1016/j.ddtec.2010.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.ddtec.2010.11.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=e181-e187&author=K.+Retraauthor=H.+Irthauthor=J.+E.+van+Muijlwijk-Koezen&title=Surface+plasmon+resonance+biosensor+analysis+as+a+useful+tool+in+FBDD&doi=10.1016%2Fj.ddtec.2010.11.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2010.11.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2010.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DRetra%26aufirst%3DK.%26aulast%3DIrth%26aufirst%3DH.%26aulast%3Dvan%2BMuijlwijk-Koezen%26aufirst%3DJ.%2BE.%26atitle%3DSurface%2520plasmon%2520resonance%2520biosensor%2520analysis%2520as%2520a%2520useful%2520tool%2520in%2520FBDD%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2010%26volume%3D7%26spage%3De181%26epage%3De187%26doi%3D10.1016%2Fj.ddtec.2010.11.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Chimenti, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulfer, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neitz, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renslo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arkin, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, M. J.</span><span> </span><span class="NLM_article-title">A fragment-based ligand screen against part of a large protein machine: the ND1 domains of the AAA+ ATPase p97/VCP</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">788</span><span class="NLM_x">–</span> <span class="NLM_lpage">800</span><span class="refDoi"> DOI: 10.1177/1087057115570550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057115570550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=25690569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsb3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=788-800&author=M.+S.+Chimentiauthor=S.+L.+Bulferauthor=R.+J.+Neitzauthor=A.+R.+Rensloauthor=M.+P.+Jacobsonauthor=T.+L.+Jamesauthor=M.+R.+Arkinauthor=M.+J.+Kelly&title=A+fragment-based+ligand+screen+against+part+of+a+large+protein+machine%3A+the+ND1+domains+of+the+AAA%2B+ATPase+p97%2FVCP&doi=10.1177%2F1087057115570550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A fragment-based ligand screen against part of a large protein machine: the ND1 domains of the AAA+ ATPase p97/VCP</span></div><div class="casAuthors">Chimenti, Michael S.; Bulfer, Stacie L.; Neitz, R. Jeffrey; Renslo, Adam R.; Jacobson, Matthew P.; James, Thomas L.; Arkin, Michelle R.; Kelly, Mark J. S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">788-800</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The ubiquitous AAA+ ATPase p97 functions as a dynamic mol. machine driving several cellular processes.  It is essential in regulating protein homeostasis, and it represents a potential drug target for cancer, particularly when there is a greater reliance on the endoplasmic reticulum-assocd. protein degrdn. pathway and ubiquitin-proteasome pathway to degrade an overabundance of secreted proteins.  Here, we report a case study for using fragment-based ligand design approaches against this large and dynamic hexamer, which has multiple potential binding sites for small mols.  A screen of a fragment library was conducted by surface plasmon resonance (SPR) and followed up by NMR (NMR), two complementary biophys. techniques.  Virtual screening was also carried out to examine possible binding sites for the exptl. hits and evaluate the potential utility of fragment docking for this target.  Out of this effort, 13 fragments were discovered that showed reversible binding with affinities between 140 μM and 1 mM, binding stoichiometries of 1:1 or 2:1, and good ligand efficiencies.  Structural data for fragment-protein interactions were obtained with residue-specific [U-2H] 13CH3-methyl-labeling NMR strategies, and these data were compared to poses from docking.  The combination of virtual screening, SPR, and NMR enabled us to find and validate a no. of interesting fragment hits and allowed us to gain an understanding of the structural nature of fragment binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdiXW9LiaobbVg90H21EOLACvtfcHk0ljMQSHDcXeQQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsb3M&md5=42c1d64fd37316597c350f772a4c1028</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1177%2F1087057115570550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115570550%26sid%3Dliteratum%253Aachs%26aulast%3DChimenti%26aufirst%3DM.%2BS.%26aulast%3DBulfer%26aufirst%3DS.%2BL.%26aulast%3DNeitz%26aufirst%3DR.%2BJ.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DJames%26aufirst%3DT.%2BL.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DKelly%26aufirst%3DM.%2BJ.%26atitle%3DA%2520fragment-based%2520ligand%2520screen%2520against%2520part%2520of%2520a%2520large%2520protein%2520machine%253A%2520the%2520ND1%2520domains%2520of%2520the%2520AAA%252B%2520ATPase%2520p97%252FVCP%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D20%26spage%3D788%26epage%3D800%26doi%3D10.1177%2F1087057115570550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skelton, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bravo, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bair, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crocker, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzner-Toste, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liederer, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuen, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dragovich, P. S.</span><span> </span><span class="NLM_article-title">Fragment-based identification of amides derived from trans-2-(pyridin-3-yl)cyclopropanecarboxylic acid as potent inhibitors of human nicotinamide phosphoribosyltransferase (NAMPT)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">770</span><span class="NLM_x">–</span> <span class="NLM_lpage">792</span><span class="refDoi"> DOI: 10.1021/jm4015108</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4015108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFWitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=770-792&author=A.+M.+Giannettiauthor=X.+Zhengauthor=N.+J.+Skeltonauthor=W.+Wangauthor=B.+J.+Bravoauthor=K.+W.+Bairauthor=T.+Baumeisterauthor=E.+Chengauthor=L.+Crockerauthor=Y.+Fengauthor=J.+Gunzner-Tosteauthor=Y.+C.+Hoauthor=R.+Huaauthor=B.+M.+Liedererauthor=Y.+Liuauthor=X.+Maauthor=T.+O%E2%80%99Brienauthor=J.+Oehauthor=D.+Sampathauthor=Y.+Shenauthor=C.+Wangauthor=L.+Wangauthor=H.+Wuauthor=Y.+Xiaoauthor=P.+W.+Yuenauthor=M.+Zakauthor=G.+Zhaoauthor=Q.+Zhaoauthor=P.+S.+Dragovich&title=Fragment-based+identification+of+amides+derived+from+trans-2-%28pyridin-3-yl%29cyclopropanecarboxylic+acid+as+potent+inhibitors+of+human+nicotinamide+phosphoribosyltransferase+%28NAMPT%29&doi=10.1021%2Fjm4015108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based identification of amides derived from trans-2-(pyridin-3-yl)cyclopropanecarboxylic acid as potent inhibitors of human nicotinamide phosphoribosyltransferase (NAMPT)</span></div><div class="casAuthors">Giannetti, Anthony M.; Zheng, Xiaozhang; Skelton, Nicholas J.; Wang, Weiru; Bravo, Brandon J.; Bair, Kenneth W.; Baumeister, Timm; Cheng, Eric; Crocker, Lisa; Feng, Yezhen; Gunzner-Toste, Janet; Ho, Yen-Ching; Hua, Rongbao; Liederer, Bianca M.; Liu, Yongbo; Ma, Xiaolei; O'Brien, Thomas; Oeh, Jason; Sampath, Deepak; Shen, Youming; Wang, Chengcheng; Wang, Leslie; Wu, Hongxing; Xiao, Yang; Yuen, Po-wai; Zak, Mark; Zhao, Guiling; Zhao, Qiang; Dragovich, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">770-792</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Potent, trans-2-(pyridin-3-yl)-cyclopropane-carboxamide-contg. inhibitors of the human nicotinamide phosphoribosyl-transferase (NAMPT) enzyme were identified using fragment-based screening and structure-based design techniques.  Multiple crystal structures were obtained of initial fragment leads, and this structural information was utilized to improve the biochem. and cell-based potency of the assocd. mols.  Many of the optimized compds. exhibited nanomolar antiproliferative activities against human tumor lines in in vitro cell culture expts.  In a key example, a fragment lead (13, KD = 51 μM) was elaborated into a potent NAMPT inhibitor (39, NAMPT IC50 = 0.0051 μM, A2780 cell culture IC50 = 0.000 49 μM) which demonstrated encouraging in vivo efficacy in an HT-1080 mouse xenograft tumor model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGDYM9bbug1rVg90H21EOLACvtfcHk0lgj5ArWLomRDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFWitQ%253D%253D&md5=cf1e87e52ef0641e48465a44d5b3347c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm4015108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4015108%26sid%3Dliteratum%253Aachs%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DBravo%26aufirst%3DB.%2BJ.%26aulast%3DBair%26aufirst%3DK.%2BW.%26aulast%3DBaumeister%26aufirst%3DT.%26aulast%3DCheng%26aufirst%3DE.%26aulast%3DCrocker%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DGunzner-Toste%26aufirst%3DJ.%26aulast%3DHo%26aufirst%3DY.%2BC.%26aulast%3DHua%26aufirst%3DR.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DOeh%26aufirst%3DJ.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DYuen%26aufirst%3DP.%2BW.%26aulast%3DZak%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26atitle%3DFragment-based%2520identification%2520of%2520amides%2520derived%2520from%2520trans-2-%2528pyridin-3-yl%2529cyclopropanecarboxylic%2520acid%2520as%2520potent%2520inhibitors%2520of%2520human%2520nicotinamide%2520phosphoribosyltransferase%2520%2528NAMPT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D770%26epage%3D792%26doi%3D10.1021%2Fjm4015108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Hendrick, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willems, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irving, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davenport, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reeves, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffron, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bazin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turney, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, P. J.</span><span> </span><span class="NLM_article-title">Identification and investigation of novel binding fragments in the fatty acid binding protein 6 (FABP6)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">8094</span><span class="NLM_x">–</span> <span class="NLM_lpage">8102</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00869</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00869" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSht7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8094-8102&author=A.+G.+Hendrickauthor=I.+Mullerauthor=H.+Willemsauthor=P.+M.+Leonardauthor=S.+Irvingauthor=R.+Davenportauthor=T.+Itoauthor=J.+Reevesauthor=S.+Wrightauthor=V.+Allenauthor=S.+Wilkinsonauthor=H.+Heffronauthor=R.+Bazinauthor=J.+Turneyauthor=P.+J.+Mitchell&title=Identification+and+investigation+of+novel+binding+fragments+in+the+fatty+acid+binding+protein+6+%28FABP6%29&doi=10.1021%2Facs.jmedchem.6b00869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Investigation of Novel Binding Fragments in the Fatty Acid Binding Protein 6 (FABP6)</span></div><div class="casAuthors">Hendrick, Alan G.; Muller, Ilka; Willems, Henriette; Leonard, Philip M.; Irving, Steve; Davenport, Richard; Ito, Takashi; Reeves, Jenny; Wright, Susanne; Allen, Vivienne; Wilkinson, Stephen; Heffron, Helen; Bazin, Richard; Turney, Jennifer; Mitchell, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8094-8102</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fatty acid binding protein 6 (FABP6) is a potential drug discovery target, which, if inhibited, may have a therapeutic benefit for the treatment of diabetes.  Currently, there are no published inhibitors of FABP6, and with the target believed to be amenable to fragment-based drug discovery, a structurally enabled program was initiated.  This program successfully identified fragment hits using the surface plasmon resonance (SPR) platform.  Several hits were validated with SAR and were found to be displaced by the natural ligand taurocholate.  We report the first crystal structure of human FABP6 in the unbound form, in complex with cholate, and with one of the key fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYHJ_ox17KQbVg90H21EOLACvtfcHk0lgU6OmUfVQmUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSht7%252FL&md5=f98fa9c2cf870f1ad2fac5b9b361ae40</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00869%26sid%3Dliteratum%253Aachs%26aulast%3DHendrick%26aufirst%3DA.%2BG.%26aulast%3DMuller%26aufirst%3DI.%26aulast%3DWillems%26aufirst%3DH.%26aulast%3DLeonard%26aufirst%3DP.%2BM.%26aulast%3DIrving%26aufirst%3DS.%26aulast%3DDavenport%26aufirst%3DR.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DReeves%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DV.%26aulast%3DWilkinson%26aufirst%3DS.%26aulast%3DHeffron%26aufirst%3DH.%26aulast%3DBazin%26aufirst%3DR.%26aulast%3DTurney%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DP.%2BJ.%26atitle%3DIdentification%2520and%2520investigation%2520of%2520novel%2520binding%2520fragments%2520in%2520the%2520fatty%2520acid%2520binding%2520protein%25206%2520%2528FABP6%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8094%26epage%3D8102%26doi%3D10.1021%2Facs.jmedchem.6b00869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Lund, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christopeit, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guttormsen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leiros, H. K.</span><span> </span><span class="NLM_article-title">Screening and design of inhibitor scaffolds for the antibiotic resistance oxacillinase-48 (OXA-48) through surface plasmon resonance screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">5542</span><span class="NLM_x">–</span> <span class="NLM_lpage">5554</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00660</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00660" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5542-5554&author=B.+A.+Lundauthor=T.+Christopeitauthor=Y.+Guttormsenauthor=A.+Bayerauthor=H.+K.+Leiros&title=Screening+and+design+of+inhibitor+scaffolds+for+the+antibiotic+resistance+oxacillinase-48+%28OXA-48%29+through+surface+plasmon+resonance+screening&doi=10.1021%2Facs.jmedchem.6b00660"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Screening and Design of Inhibitor Scaffolds for the Antibiotic Resistance Oxacillinase-48 (OXA-48) through Surface Plasmon Resonance Screening</span></div><div class="casAuthors">Lund, Bjarte Aarmo; Christopeit, Tony; Guttormsen, Yngve; Bayer, Annette; Leiros, Hanna-Kirsti S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5542-5554</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The spread of antibiotic resistant bacteria is a global threat that shakes the foundations of modern healthcare.  β-Lactamases are enzymes that confer resistance to β-lactam antibiotics in bacteria, and there is a crit. need for new inhibitors of these enzymes for combination therapy together with an antibiotic.  With this in mind, we have screened a library of 490 fragments to identify starting points for the development of new inhibitors of the class D β-lactamase oxacillinase-48 (OXA-48) through surface plasmon resonance (SPR), dose-rate inhibition assays, and X-ray crystallog.  Furthermore, we have uncovered structure-activity relationships and used alternate conformations from a crystallog. structure to grow a fragment into a more potent compd. with a KD of 50 μM and an IC50 of 18 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHGpwAMa7NrbVg90H21EOLACvtfcHk0lgU6OmUfVQmUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWiurc%253D&md5=2a20589a324252b22d7558e5a7068367</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00660%26sid%3Dliteratum%253Aachs%26aulast%3DLund%26aufirst%3DB.%2BA.%26aulast%3DChristopeit%26aufirst%3DT.%26aulast%3DGuttormsen%26aufirst%3DY.%26aulast%3DBayer%26aufirst%3DA.%26aulast%3DLeiros%26aufirst%3DH.%2BK.%26atitle%3DScreening%2520and%2520design%2520of%2520inhibitor%2520scaffolds%2520for%2520the%2520antibiotic%2520resistance%2520oxacillinase-48%2520%2528OXA-48%2529%2520through%2520surface%2520plasmon%2520resonance%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5542%26epage%3D5554%26doi%3D10.1021%2Facs.jmedchem.6b00660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wixted, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffiths-Jones, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hearn, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rich, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saxty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willems, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolford, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cottom, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kou, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonchuk, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldser, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanchez, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolognese, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podolin, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callahan, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerns, J. K.</span><span> </span><span class="NLM_article-title">Monoacidic inhibitors of the kelch-like ECH-associated protein 1: nuclear factor erythroid 2-related factor 2 (KEAP1:NRF2) protein–protein interaction with high cell potency identified by fragment-based discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">3991</span><span class="NLM_x">–</span> <span class="NLM_lpage">4006</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00228</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00228" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3991-4006&author=T.+G.+Daviesauthor=W.+E.+Wixtedauthor=J.+E.+Coyleauthor=C.+Griffiths-Jonesauthor=K.+Hearnauthor=R.+McMenaminauthor=D.+Nortonauthor=S.+J.+Richauthor=C.+Richardsonauthor=G.+Saxtyauthor=H.+M.+Willemsauthor=A.+J.+Woolfordauthor=J.+E.+Cottomauthor=J.+P.+Kouauthor=J.+G.+Yonchukauthor=H.+G.+Feldserauthor=Y.+Sanchezauthor=J.+P.+Foleyauthor=B.+J.+Bologneseauthor=G.+Loganauthor=P.+L.+Podolinauthor=H.+Yanauthor=J.+F.+Callahanauthor=T.+D.+Heightmanauthor=J.+K.+Kerns&title=Monoacidic+inhibitors+of+the+kelch-like+ECH-associated+protein+1%3A+nuclear+factor+erythroid+2-related+factor+2+%28KEAP1%3ANRF2%29+protein%E2%80%93protein+interaction+with+high+cell+potency+identified+by+fragment-based+discovery&doi=10.1021%2Facs.jmedchem.6b00228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Monoacidic Inhibitors of the Kelch-like ECH-Associated Protein 1: Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1:NRF2) Protein-Protein Interaction with High Cell Potency Identified by Fragment-Based Discovery</span></div><div class="casAuthors">Davies, Thomas G.; Wixted, William E.; Coyle, Joseph E.; Griffiths-Jones, Charlotte; Hearn, Keisha; McMenamin, Rachel; Norton, David; Rich, Sharna J.; Richardson, Caroline; Saxty, Gordon; Willems, Henriette M. G.; Woolford, Alison J.-A.; Cottom, Joshua E.; Kou, Jen-Pyng; Yonchuk, John G.; Feldser, Heidi G.; Sanchez, Yolanda; Foley, Joseph P.; Bolognese, Brian J.; Logan, Gregory; Podolin, Patricia L.; Yan, Hongxing; Callahan, James F.; Heightman, Tom D.; Kerns, Jeffrey K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3991-4006</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KEAP1 is the key regulator of the NRF2-mediated cytoprotective response, and increasingly recognized as a target for diseases involving oxidative stress.  Pharmacol. intervention has focused on mols. that decrease NRF2-ubiquitination through covalent modification of KEAP1 cysteine residues, but such electrophilic compds. lack selectivity and may be assocd. with off-target toxicity.  We report here the first use of a fragment-based approach to directly target the KEAP1 Kelch-NRF2 interaction.  X-ray crystallog. screening identified three distinct "hot-spots" for fragment binding within the NRF2 binding pocket of KEAP1, allowing progression of a weak fragment hit to mols. with nanomolar affinity for KEAP1 while maintaining drug-like properties.  This work resulted in a promising lead compd. (I) which exhibits tight and selective binding to KEAP1, and activates the NRF2 antioxidant response in cellular and in vivo models, thereby providing a high quality chem. probe to explore the therapeutic potential of disrupting the KEAP1-NRF2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq-gfBTuYreLVg90H21EOLACvtfcHk0lgU6OmUfVQmUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Gjtr0%253D&md5=ae42cc9f06d14b340cb3a8f5d6c157a7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00228%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DGriffiths-Jones%26aufirst%3DC.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DNorton%26aufirst%3DD.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWillems%26aufirst%3DH.%2BM.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26aulast%3DCottom%26aufirst%3DJ.%2BE.%26aulast%3DKou%26aufirst%3DJ.%2BP.%26aulast%3DYonchuk%26aufirst%3DJ.%2BG.%26aulast%3DFeldser%26aufirst%3DH.%2BG.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DFoley%26aufirst%3DJ.%2BP.%26aulast%3DBolognese%26aufirst%3DB.%2BJ.%26aulast%3DLogan%26aufirst%3DG.%26aulast%3DPodolin%26aufirst%3DP.%2BL.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DKerns%26aufirst%3DJ.%2BK.%26atitle%3DMonoacidic%2520inhibitors%2520of%2520the%2520kelch-like%2520ECH-associated%2520protein%25201%253A%2520nuclear%2520factor%2520erythroid%25202-related%2520factor%25202%2520%2528KEAP1%253ANRF2%2529%2520protein%25E2%2580%2593protein%2520interaction%2520with%2520high%2520cell%2520potency%2520identified%2520by%2520fragment-based%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3991%26epage%3D4006%26doi%3D10.1021%2Facs.jmedchem.6b00228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Bower, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannifer, A.</span><span> </span><span class="NLM_article-title">Using fragment-based technologies to target protein–protein interactions</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4685</span><span class="NLM_x">–</span> <span class="NLM_lpage">4696</span><span class="refDoi"> DOI: 10.2174/138161212802651689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.2174%2F138161212802651689" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4685-4696&author=J.+F.+Bowerauthor=A.+Pannifer&title=Using+fragment-based+technologies+to+target+protein%E2%80%93protein+interactions&doi=10.2174%2F138161212802651689"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.2174%2F138161212802651689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212802651689%26sid%3Dliteratum%253Aachs%26aulast%3DBower%26aufirst%3DJ.%2BF.%26aulast%3DPannifer%26aufirst%3DA.%26atitle%3DUsing%2520fragment-based%2520technologies%2520to%2520target%2520protein%25E2%2580%2593protein%2520interactions%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D4685%26epage%3D4696%26doi%3D10.2174%2F138161212802651689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Wartchow, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podlaski, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K. S.</span><span> </span><span class="NLM_article-title">Biosensor-based small molecule fragment screening with biolayer interferometry</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.1007/s10822-011-9439-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1007%2Fs10822-011-9439-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=21660516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSqtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=669-676&author=C.+A.+Wartchowauthor=F.+Podlaskiauthor=S.+Liauthor=K.+Rowanauthor=X.+Zhangauthor=D.+Markauthor=K.+S.+Huang&title=Biosensor-based+small+molecule+fragment+screening+with+biolayer+interferometry&doi=10.1007%2Fs10822-011-9439-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-based small molecule fragment screening with biolayer interferometry</span></div><div class="casAuthors">Wartchow, Charles A.; Podlaski, Frank; Li, Shirley; Rowan, Karen; Zhang, Xiaolei; Mark, David; Huang, Kuo-Sen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Biosensor-based fragment screening is a valuable tool in the drug discovery process.  This method is advantageous over many biochem. methods because primary hits can be distinguished from non-specific or non-ideal interactions by examg. binding profiles and responses, resulting in reduced false-pos. rates.  Biolayer interferometry (BLI), a technique that measures changes in an interference pattern generated from visible light reflected from an optical layer and a biolayer contg. proteins of interest, is a relatively new method for monitoring small mol. interactions.  The BLI format is based on a disposable sensor that is immersed in 96-well or 384-well plates.  BLI has been validated for small mol. detection and fragment screening with model systems and well-characterized targets where affinity consts. and binding profiles are generally similar to those obtained with surface plasmon resonance (SPR).  Screens with challenging targets involved in protein-protein interactions including BCL-2, JNK1, and eIF4E were performed with a fragment library of 6,500 compds., and hit rates were compared for these targets.  For eIF4E, a protein contg. a PPI site and a nucleotide binding site, results from a BLI fragment screen were compared to results obtained in biochem. HTS screens.  Overlapping hits were obsd. for the PPI site, and hits unique to the BLI screen were identified.  Hit assessments with SPR and BLI are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnIPrJFqAln7Vg90H21EOLACvtfcHk0lhkv30kdLUUPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSqtbbP&md5=c8c3bcca03e2def50ddcaf0ed8325060</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9439-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9439-8%26sid%3Dliteratum%253Aachs%26aulast%3DWartchow%26aufirst%3DC.%2BA.%26aulast%3DPodlaski%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DRowan%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMark%26aufirst%3DD.%26aulast%3DHuang%26aufirst%3DK.%2BS.%26atitle%3DBiosensor-based%2520small%2520molecule%2520fragment%2520screening%2520with%2520biolayer%2520interferometry%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26spage%3D669%26epage%3D676%26doi%3D10.1007%2Fs10822-011-9439-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Yoshimura, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyafusa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, K.</span><span> </span><span class="NLM_article-title">Identification of small-molecule inhibitors of the human S100B-p53 interaction and evaluation of their activity in human melanoma cells</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1109</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1016/j.bmc.2012.12.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.bmc.2012.12.042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1109-1115&author=C.+Yoshimuraauthor=T.+Miyafusaauthor=K.+Tsumoto&title=Identification+of+small-molecule+inhibitors+of+the+human+S100B-p53+interaction+and+evaluation+of+their+activity+in+human+melanoma+cells&doi=10.1016%2Fj.bmc.2012.12.042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.12.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.12.042%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimura%26aufirst%3DC.%26aulast%3DMiyafusa%26aufirst%3DT.%26aulast%3DTsumoto%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520small-molecule%2520inhibitors%2520of%2520the%2520human%2520S100B-p53%2520interaction%2520and%2520evaluation%2520of%2520their%2520activity%2520in%2520human%2520melanoma%2520cells%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D1109%26epage%3D1115%26doi%3D10.1016%2Fj.bmc.2012.12.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Whittaker, M.</span><span> </span><span class="NLM_article-title">Picking up the pieces with FBDD or FADD: invest early for future success</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">624</span><span class="refDoi"> DOI: 10.1016/j.drudis.2009.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.drudis.2009.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=19486947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BD1Mvmt1Citg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=623-624&author=M.+Whittaker&title=Picking+up+the+pieces+with+FBDD+or+FADD%3A+invest+early+for+future+success&doi=10.1016%2Fj.drudis.2009.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Picking up the pieces with FBDD or FADD: invest early for future success</span></div><div class="casAuthors">Whittaker Mark</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">13-14</span>),
    <span class="NLM_cas:pages">623-4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRrARZ0UGwnHu98rul4nhSdfW6udTcc2eaHtGsaj6TcJLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mvmt1Citg%253D%253D&md5=1e8b209f593cf74a642ea9f7c3289d42</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2009.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2009.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DWhittaker%26aufirst%3DM.%26atitle%3DPicking%2520up%2520the%2520pieces%2520with%2520FBDD%2520or%2520FADD%253A%2520invest%2520early%2520for%2520future%2520success%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D623%26epage%3D624%26doi%3D10.1016%2Fj.drudis.2009.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Varnes, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geschwindner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmquist, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forst, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dekker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albert, J. S.</span><span> </span><span class="NLM_article-title">Fragment-assisted hit investigation involving integrated HTS and fragment screening: Application to the identification of phosphodiesterase 10A (PDE10A) inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.10.100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.bmcl.2015.10.100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=26597534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVKqt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=197-202&author=J.+G.+Varnesauthor=S.+Geschwindnerauthor=C.+R.+Holmquistauthor=J.+Forstauthor=X.+Wangauthor=N.+Dekkerauthor=C.+W.+Scottauthor=G.+Tianauthor=M.+W.+Woodauthor=J.+S.+Albert&title=Fragment-assisted+hit+investigation+involving+integrated+HTS+and+fragment+screening%3A+Application+to+the+identification+of+phosphodiesterase+10A+%28PDE10A%29+inhibitors&doi=10.1016%2Fj.bmcl.2015.10.100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-assisted hit investigation involving integrated HTS and fragment screening: Application to the identification of phosphodiesterase 10A (PDE10A) inhibitors</span></div><div class="casAuthors">Varnes, Jeffrey G.; Geschwindner, Stefan; Holmquist, Christopher R.; Forst, Janet; Wang, Xia; Dekker, Niek; Scott, Clay W.; Tian, Gaochao; Wood, Michael W.; Albert, Jeffrey S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">197-202</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Fragment-based drug design (FBDD) relies on direct elaboration of fragment hits and typically requires high resoln. structural information to guide optimization.  In fragment-assisted drug discovery (FADD), fragments provide information to guide selection and design but do not serve as starting points for elaboration.  The authors describe FADD and high-throughput screening (HTS) campaign strategies conducted in parallel against PDE10A where fragment hit co-crystallog. was not available.  The fragment screen led to prioritized fragment hits (IC50's ∼500 μM), which were used to generate a hypothetical core scaffold.  Application of this scaffold as a filter to HTS output afforded a 4 μM hit, which, after prepn. of a small no. of analogs, was elaborated into a 16 nM lead.  This approach highlights the strength of FADD, as fragment methods were applied despite the absence of co-crystallog. information to efficiently identify a lead compd. for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoya6D4nTvSu7Vg90H21EOLACvtfcHk0ljTmJTKwDoqKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVKqt7jO&md5=17cdaef2373e4a9e693dae2983d7dc07</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.10.100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.10.100%26sid%3Dliteratum%253Aachs%26aulast%3DVarnes%26aufirst%3DJ.%2BG.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DHolmquist%26aufirst%3DC.%2BR.%26aulast%3DForst%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDekker%26aufirst%3DN.%26aulast%3DScott%26aufirst%3DC.%2BW.%26aulast%3DTian%26aufirst%3DG.%26aulast%3DWood%26aufirst%3DM.%2BW.%26aulast%3DAlbert%26aufirst%3DJ.%2BS.%26atitle%3DFragment-assisted%2520hit%2520investigation%2520involving%2520integrated%2520HTS%2520and%2520fragment%2520screening%253A%2520Application%2520to%2520the%2520identification%2520of%2520phosphodiesterase%252010A%2520%2528PDE10A%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D197%26epage%3D202%26doi%3D10.1016%2Fj.bmcl.2015.10.100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Elinder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geitmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gossas, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallblad, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winquist, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordstrom, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamalainen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danielson, U. H.</span><span> </span><span class="NLM_article-title">Experimental validation of a fragment library for lead discovery using SPR biosensor technology</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1177/1087057110389038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057110389038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=21149860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsl2gs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=15-25&author=M.+Elinderauthor=M.+Geitmannauthor=T.+Gossasauthor=P.+Kallbladauthor=J.+Winquistauthor=H.+Nordstromauthor=M.+Hamalainenauthor=U.+H.+Danielson&title=Experimental+validation+of+a+fragment+library+for+lead+discovery+using+SPR+biosensor+technology&doi=10.1177%2F1087057110389038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental validation of a fragment library for lead discovery using SPR biosensor technology</span></div><div class="casAuthors">Elinder, Malin; Geitmann, Matthis; Gossas, Thomas; Kaellblad, Per; Winquist, Johan; Nordstroem, Helena; Haemaelaeinen, Markku; Danielson, U. Helena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-25</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A new fragment library for lead discovery has been designed and exptl. validated for use in surface plasmon resonance (SPR) biosensor-based screening.  The 930 compds. in the library were selected from 4.6 million com. available compds. using a series of physicochem. and medicinal chem. filters.  They were screened against 3 prototypical drug targets: HIV-1 protease, thrombin and carbonic anhydrase, and a nontarget: human serum albumin.  Compd. soly. was not a problem under the conditions used for screening.  The high sensitivity of the sensor surfaces allowed the detection of interactions for 35% to 97% of the fragments, depending on the target protein.  None of the fragments was promiscuous (i.e., interacted with a stoichiometry ≥5:1 with all 4 proteins), and only 2 compds. dissocd. slowly from all 4 proteins.  The use of several targets proved valuable since several compds. would have been disqualified from the library on the grounds of promiscuity if fewer target proteins had been used.  The exptl. procedure allowed an efficient evaluation and exploration of the new fragment library and confirmed that the new library is suitable for SPR biosensor-based screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMQe_fTV57lrVg90H21EOLACvtfcHk0ljTmJTKwDoqKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsl2gs7s%253D&md5=a49b6811a04299589c02852318b01aae</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1177%2F1087057110389038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057110389038%26sid%3Dliteratum%253Aachs%26aulast%3DElinder%26aufirst%3DM.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DGossas%26aufirst%3DT.%26aulast%3DKallblad%26aufirst%3DP.%26aulast%3DWinquist%26aufirst%3DJ.%26aulast%3DNordstrom%26aufirst%3DH.%26aulast%3DHamalainen%26aufirst%3DM.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26atitle%3DExperimental%2520validation%2520of%2520a%2520fragment%2520library%2520for%2520lead%2520discovery%2520using%2520SPR%2520biosensor%2520technology%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2011%26volume%3D16%26spage%3D15%26epage%3D25%26doi%3D10.1177%2F1087057110389038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Hamalainen, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhukov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivarsson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fex, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottfries, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjorsne, M.</span><span> </span><span class="NLM_article-title">Label-free primary screening and affinity ranking of fragment libraries using parallel analysis of protein panels</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">202</span><span class="NLM_x">–</span> <span class="NLM_lpage">209</span><span class="refDoi"> DOI: 10.1177/1087057108314651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057108314651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=18270366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BD1c7pt1Ojtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=202-209&author=M.+D.+Hamalainenauthor=A.+Zhukovauthor=M.+Ivarssonauthor=T.+Fexauthor=J.+Gottfriesauthor=R.+Karlssonauthor=M.+Bjorsne&title=Label-free+primary+screening+and+affinity+ranking+of+fragment+libraries+using+parallel+analysis+of+protein+panels&doi=10.1177%2F1087057108314651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Label-free primary screening and affinity ranking of fragment libraries using parallel analysis of protein panels</span></div><div class="casAuthors">Hamalainen Markku D; Zhukov Andrei; Ivarsson Maria; Fex Tomas; Gottfries Johan; Karlsson Robert; Bjorsne Magnus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">202-9</span>
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    </div><div class="casAbstract">The authors present fragment screening data obtained using a label-free parallel analysis approach where the binding of fragment library compounds to 4 different target proteins can be screened simultaneously using surface plasmon resonance detection.  They suggest this method as a first step in fragment screening to identify and select binders, reducing the demanding requirements on subsequent X-ray or nuclear magnetic resonance studies, and as a valuable "clean-up" tool to eliminate unwanted promiscuous binders from libraries.  A small directed fragment library of known thrombin binders and a general 500-compound fragment library were used in this study.  Thrombin, blocked thrombin, carbonic anhydrase, and glutathione-S-transferase were immobilized on the sensor chip surface, and the direct binding of the fragments was studied in real time.  Only 12 microg of each protein is needed for screening of a 3000-compound fragment library.  For screening, a binding site-blocked target as reference facilitates the identification of binding site-selective hits and the signals from other reference proteins for the elimination of false positives.  The scope and limitations of this screening approach are discussed for both target-directed and general fragment libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSjH4rWZRY1vhBrI9DTGjC0fW6udTcc2eamLEzpwVlQkLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7pt1Ojtg%253D%253D&md5=52fd220da141e9347513d73ac18c87eb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1177%2F1087057108314651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108314651%26sid%3Dliteratum%253Aachs%26aulast%3DHamalainen%26aufirst%3DM.%2BD.%26aulast%3DZhukov%26aufirst%3DA.%26aulast%3DIvarsson%26aufirst%3DM.%26aulast%3DFex%26aufirst%3DT.%26aulast%3DGottfries%26aufirst%3DJ.%26aulast%3DKarlsson%26aufirst%3DR.%26aulast%3DBjorsne%26aufirst%3DM.%26atitle%3DLabel-free%2520primary%2520screening%2520and%2520affinity%2520ranking%2520of%2520fragment%2520libraries%2520using%2520parallel%2520analysis%2520of%2520protein%2520panels%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2008%26volume%3D13%26spage%3D202%26epage%3D209%26doi%3D10.1177%2F1087057108314651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Nordstrom, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gossas, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamalainen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kallblad, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nystrom, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallberg, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danielson, U. H.</span><span> </span><span class="NLM_article-title">Identification of MMP-12 inhibitors by using biosensor-based screening of a fragment library</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3449</span><span class="NLM_x">–</span> <span class="NLM_lpage">3459</span><span class="refDoi"> DOI: 10.1021/jm8000289</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8000289" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3449-3459&author=H.+Nordstromauthor=T.+Gossasauthor=M.+Hamalainenauthor=P.+Kallbladauthor=S.+Nystromauthor=H.+Wallbergauthor=U.+H.+Danielson&title=Identification+of+MMP-12+inhibitors+by+using+biosensor-based+screening+of+a+fragment+library&doi=10.1021%2Fjm8000289"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm8000289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8000289%26sid%3Dliteratum%253Aachs%26aulast%3DNordstrom%26aufirst%3DH.%26aulast%3DGossas%26aufirst%3DT.%26aulast%3DHamalainen%26aufirst%3DM.%26aulast%3DKallblad%26aufirst%3DP.%26aulast%3DNystrom%26aufirst%3DS.%26aulast%3DWallberg%26aufirst%3DH.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26atitle%3DIdentification%2520of%2520MMP-12%2520inhibitors%2520by%2520using%2520biosensor-based%2520screening%2520of%2520a%2520fragment%2520library%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3449%26epage%3D3459%26doi%3D10.1021%2Fjm8000289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Navratilova, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macdonald, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathias, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span> </span><span class="NLM_article-title">Biosensor-based approach to the identification of protein kinase ligands with dual-site modes of action</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1177/1087057111422746</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057111422746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=22068706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVKltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=183-193&author=I.+Navratilovaauthor=G.+Macdonaldauthor=C.+Robinsonauthor=S.+Hughesauthor=J.+Mathiasauthor=C.+Phillipsauthor=A.+Cook&title=Biosensor-based+approach+to+the+identification+of+protein+kinase+ligands+with+dual-site+modes+of+action&doi=10.1177%2F1087057111422746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-based approach to the identification of protein kinase ligands with dual-site modes of action</span></div><div class="casAuthors">Navratilova, Iva; MacDonald, Graeme; Robinson, Colin; Hughes, Samantha; Mathias, John; Phillips, Chris; Cook, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-193</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The authors have used a surface plasmon resonance (SPR)-based biosensor approach to identify and characterize compds. with a unique binding mode to protein kinases.  Biacore was used to characterize hits from an enzymic high-throughput screen of the Tec family tyrosine kinase, IL2-inducible T cell kinase (ITK).  Complex binding kinetics was obsd. for some compds., which led to identification of compds. that bound simultaneously at both the ATP binding site and a second, allosteric site on ITK.  The presence of the second binding site was confirmed by X-ray crystallog.  The second site is located in the N-terminal lobe of the protein kinase catalytic domain, adjacent to but distinct from the ATP site.  To enable rapid optimization of binding properties, a competition-based Biacore assay has been developed to successfully identify second site noncompetitive binders that have been confirmed by X-ray crystallog. studies.  The authors have found that SPR technol. is a key method for rapid identification of compds. with dual-site modes of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_O18si5LWPLVg90H21EOLACvtfcHk0lhwu3eOAyLSAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtVKltL8%253D&md5=8860bb955a27b7ba6b167725224c9e60</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1177%2F1087057111422746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422746%26sid%3Dliteratum%253Aachs%26aulast%3DNavratilova%26aufirst%3DI.%26aulast%3DMacdonald%26aufirst%3DG.%26aulast%3DRobinson%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%26aulast%3DMathias%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DA.%26atitle%3DBiosensor-based%2520approach%2520to%2520the%2520identification%2520of%2520protein%2520kinase%2520ligands%2520with%2520dual-site%2520modes%2520of%2520action%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D183%26epage%3D193%26doi%3D10.1177%2F1087057111422746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Nordin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jungnelius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, O. P.</span><span> </span><span class="NLM_article-title">Kinetic studies of small molecule interactions with protein kinases using biosensor technology</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">340</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">368</span><span class="refDoi"> DOI: 10.1016/j.ab.2005.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.ab.2005.02.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=340&publication_year=2005&pages=359-368&author=H.+Nordinauthor=M.+Jungneliusauthor=R.+Karlssonauthor=O.+P.+Karlsson&title=Kinetic+studies+of+small+molecule+interactions+with+protein+kinases+using+biosensor+technology&doi=10.1016%2Fj.ab.2005.02.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DNordin%26aufirst%3DH.%26aulast%3DJungnelius%26aufirst%3DM.%26aulast%3DKarlsson%26aufirst%3DR.%26aulast%3DKarlsson%26aufirst%3DO.%2BP.%26atitle%3DKinetic%2520studies%2520of%2520small%2520molecule%2520interactions%2520with%2520protein%2520kinases%2520using%2520biosensor%2520technology%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D340%26spage%3D359%26epage%3D368%26doi%3D10.1016%2Fj.ab.2005.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Stenlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frostell-Karlsson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, O. P.</span><span> </span><span class="NLM_article-title">Studies of small molecule interactions with protein phosphatases using biosensor technology</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">353</span><span class="NLM_x">, </span> <span class="NLM_fpage">217</span><span class="NLM_x">–</span> <span class="NLM_lpage">225</span><span class="refDoi"> DOI: 10.1016/j.ab.2006.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2Fj.ab.2006.03.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2006&pages=217-225&author=P.+Stenlundauthor=A.+Frostell-Karlssonauthor=O.+P.+Karlsson&title=Studies+of+small+molecule+interactions+with+protein+phosphatases+using+biosensor+technology&doi=10.1016%2Fj.ab.2006.03.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2006.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2006.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DStenlund%26aufirst%3DP.%26aulast%3DFrostell-Karlsson%26aufirst%3DA.%26aulast%3DKarlsson%26aufirst%3DO.%2BP.%26atitle%3DStudies%2520of%2520small%2520molecule%2520interactions%2520with%2520protein%2520phosphatases%2520using%2520biosensor%2520technology%26jtitle%3DAnal.%2520Biochem.%26date%3D2006%26volume%3D353%26spage%3D217%26epage%3D225%26doi%3D10.1016%2Fj.ab.2006.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eZrVMYElIXrjbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0liDEPTwqtbNqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Perspicace, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlatter, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, W.</span><span> </span><span class="NLM_article-title">Fragment-based screening using surface plasmon resonance technology</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">349</span><span class="refDoi"> DOI: 10.1177/1087057109332595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1177%2F1087057109332595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=19403917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFygu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=337-349&author=S.+Perspicaceauthor=D.+Bannerauthor=J.+Benzauthor=F.+Mullerauthor=D.+Schlatterauthor=W.+Huber&title=Fragment-based+screening+using+surface+plasmon+resonance+technology&doi=10.1177%2F1087057109332595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based screening using surface plasmon resonance technology</span></div><div class="casAuthors">Perspicace, Samantha; Banner, David; Benz, Jorg; Muller, Francis; Schlatter, Daniel; Huber, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-349</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Surface plasmon resonance (SPR) technol. has emerged as a new and powerful technique to investigate the interaction between low-mol.-wt. mols. and target proteins.  In the present work, the authors assemble from a large compd. collection a library of 2226 mols. (fragments having low mol. wts. between 100 and 300 Da) to screen them for binding to chymase, a serine protease.  Both the active chymase and a zymogen-like form of the protein were used in parallel to distinguish between specific and unspecific binding.  The relative ligand-binding activity of the immobilized protein was periodically measured with a ref. compd.  The screening expts. were performed at 25 °C at a fragment concn. of 200 μM in the presence of 2% DMSO.  Applying the filter cascade, affinity-selectivity-competition (competition with ref. compds. and cross-competition with fragments), 80 compds. show up as pos. screening hits.  Competition expts. between fragments show that they bind to different parts of the active site.  Of 36 fragments co-crystd. for X-ray studies, 12 could be located in the active site of the protein.  These results validate the authors' library and demonstrate that the application of SPR technol. as a filter in fragment screening can be achieved successfully.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmmLjEQrEFlLVg90H21EOLACvtfcHk0liDEPTwqtbNqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFygu74%253D&md5=c2bd577576c10adb933eae83a79c9f4c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1177%2F1087057109332595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109332595%26sid%3Dliteratum%253Aachs%26aulast%3DPerspicace%26aufirst%3DS.%26aulast%3DBanner%26aufirst%3DD.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DF.%26aulast%3DSchlatter%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DW.%26atitle%3DFragment-based%2520screening%2520using%2520surface%2520plasmon%2520resonance%2520technology%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D337%26epage%3D349%26doi%3D10.1177%2F1087057109332595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Otwinowski, Z.; Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. In  <span class="citation_source-book">Macromolecular Crystallography, part A</span>; <span class="NLM_contrib-group">Carter, C. W. J.; Sweet, R. M.</span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=C.+W.+J.+Carter&author=R.+M.+Sweet&title=Macromolecular+Crystallography%2C+part+A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMacromolecular%2520Crystallography%252C%2520part%2520A%26aulast%3DCarter%26aufirst%3DC.%2BW.%2BJ.%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: An automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span><span class="refDoi"> DOI: 10.1107/S0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+An+automated+program+for+molecular+replacement&doi=10.1107%2FS0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0liDEPTwqtbNqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520An%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2FS0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>.  <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span>. <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=15299374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A280%3ADC%252BD2czpsV2gug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+CCP4+suite%3A+programs+for+protein+crystallography.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+1994%2C+50%2C+760%E2%80%93763.+10.1107%2FS0907444994003112."target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The CCP4 suite: programs for protein crystallography</span></div><div class="casAuthors">Anonymous</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">760-3</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">The CCP4 (Collaborative Computational Project, number 4) program suite is a collection of programs and associated data and subroutine libraries which can be used for macromolecular structure determination by X-ray crystallography.  The suite is designed to be flexible, allowing users a number of methods of achieving their aims and so there may be more than one program to cover each function.  The programs are written mainly in standard Fortran77.  They are from a wide variety of sources but are connected by standard data file formats.  The package has been ported to all the major platforms under both Unix and VMS.  The suite is distributed by anonymous ftp from Daresbury Laboratory and is widely used throughout the world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ1Q-skJYIZ2IYPsCkJIw00fW6udTcc2ebm4cnaosAaLLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czpsV2gug%253D%253D&md5=92b759927d6071ea3f20c08b78ad1eb2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skubak, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebedev, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steiner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span> </span><span class="NLM_article-title">REFMAC5 for the refinement of macromolecular crystal structures</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skubakauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lip9wKnpogUBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkubak%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lip9wKnpogUBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhkUsC22_VPRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R28" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R28','PDB','1R28'); return false;">PDB: 1R28</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i32"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00313">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_58175"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00313">10.1021/acs.jmedchem.7b00313</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">SPR sensorgrams for parallel immobilization, evaluation of stability and DMSO effect, SPR sensorgrams and equilibrium plots for compounds <b>2</b>–<b>6</b>, electron density OMIT map, data collection and refinement statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00313/suppl_file/jm7b00313_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00313/suppl_file/jm7b00313_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00313/suppl_file/jm7b00313_si_001.pdf">jm7b00313_si_001.pdf (480.79 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00313/suppl_file/jm7b00313_si_002.csv">jm7b00313_si_002.csv (0.31 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atom coordinates and structure factors have been deposited in the Protein Data Bank with accession codes: 5X4M (BCL6/<b>1</b>), 5X4N (BCL6/<b>4</b>), 5X4O (BCL6/<b>5</b>), 5X4P (BCL6/<b>6</b>), and 5X4Q (BCL6/<b>7</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-10%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00313%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00313" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679925c308a33dca","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
