
<html lang="en"     class="pb-page"  data-request-id="4937ec8d-8fd7-44b9-a468-95f616742bc6"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2013.56.issue-22;pageGroup:string:Publication Pages;article:article:10.1021/jm401334s;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains" /></meta><meta name="dc.Creator" content="Guangtao  Zhang" /></meta><meta name="dc.Creator" content="Alexander N.  Plotnikov" /></meta><meta name="dc.Creator" content="Elena  Rusinova" /></meta><meta name="dc.Creator" content="Tong  Shen" /></meta><meta name="dc.Creator" content="Keita  Morohashi" /></meta><meta name="dc.Creator" content="Jennifer  Joshua" /></meta><meta name="dc.Creator" content="Lei  Zeng" /></meta><meta name="dc.Creator" content="Shiraz  Mujtaba" /></meta><meta name="dc.Creator" content="Michael  Ohlmeyer" /></meta><meta name="dc.Creator" content="Ming-Ming  Zhou" /></meta><meta name="dc.Description" content="BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor ..." /></meta><meta name="Description" content="BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 11, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm401334s" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm401334s" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm401334s" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm401334s" /></link>
        
    
    

<title>Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm401334s" /></meta><meta property="og:title" content="Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0011.jpeg" /></meta><meta property="og:description" content="BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation. BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation. Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors. Here, we report our structure-guided development of diazobenzene-based small-molecule inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site. Our lead compound, MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estimated Ki of 30–50 nM), with preference for the first bromodomain over the second. We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed production of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages. MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biological functions of the two bromodomains of BRD4 in gene expression." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm401334s"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm401334s">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm401334s&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm401334s&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm401334s&amp;href=/doi/10.1021/jm401334s" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 9251-9264</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm401333u" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm401340p" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Guangtao++Zhang">Guangtao Zhang</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alexander+N.++Plotnikov">Alexander N. Plotnikov</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elena++Rusinova">Elena Rusinova</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tong++Shen">Tong Shen</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Keita++Morohashi">Keita Morohashi</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer++Joshua">Jennifer Joshua</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lei++Zeng">Lei Zeng</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shiraz++Mujtaba">Shiraz Mujtaba</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Ohlmeyer">Michael Ohlmeyer</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ming-Ming++Zhou">Ming-Ming Zhou</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Department of Structural and Chemical Biology, Icahn School of Medicine at Mount Sinai, 1425 Madison Avenue, New York, New York 10029, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 212-659-8652; Fax: 212-849-2456; E-mail: <a href="/cdn-cgi/l/email-protection#167b7f78713b7b7f7871386c7e7963567b65657b38737263"><span class="__cf_email__" data-cfemail="412c282f266c2c282f266f3b292e34012c32322c6f242534">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm401334s&amp;href=/doi/10.1021%2Fjm401334s" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 9251–9264</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 21, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>2 September 2013</li><li><span class="item_label"><b>Published</b> online</span>11 November 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 November 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm401334s" title="DOI URL">https://doi.org/10.1021/jm401334s</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9251%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGuangtao%2BZhang%252C%2BAlexander%2BN.%2BPlotnikov%252C%2BElena%2BRusinova%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D22%26contentID%3Djm401334s%26title%3DStructure-Guided%2BDesign%2Bof%2BPotent%2BDiazobenzene%2BInhibitors%2Bfor%2Bthe%2BBET%2BBromodomains%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9264%26publicationDate%3DNovember%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm401334s"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3461</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">86</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm401334s" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Guangtao&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;N. Plotnikov&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Rusinova&quot;},{&quot;first_name&quot;:&quot;Tong&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Keita&quot;,&quot;last_name&quot;:&quot;Morohashi&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;Joshua&quot;},{&quot;first_name&quot;:&quot;Lei&quot;,&quot;last_name&quot;:&quot;Zeng&quot;},{&quot;first_name&quot;:&quot;Shiraz&quot;,&quot;last_name&quot;:&quot;Mujtaba&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Ohlmeyer&quot;},{&quot;first_name&quot;:&quot;Ming-Ming&quot;,&quot;last_name&quot;:&quot;Zhou&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;9251-9264&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm401334s&quot;},&quot;abstract&quot;:&quot;BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation. BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation. Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors. Here, we report our structure-guided development of diazobenzene-based small-molecule inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site. Our lead compound, MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estimated Ki of 30–50 nM), with preference for the first bromodomain over the second. We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-direc&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm401334s&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm401334s" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm401334s&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm401334s" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm401334s&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm401334s" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm401334s&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm401334s&amp;href=/doi/10.1021/jm401334s" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm401334s" /></input><a href="/doi/pdf/10.1021/jm401334s" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26pmid%3D24144283%26genre%3Darticle%26aulast%3DZhang%26date%3D2013%26atitle%3DStructure-Guided%2BDesign%2Bof%2BPotent%2BDiazobenzene%2BInhibitors%2Bfor%2Bthe%2BBET%2BBromodomains%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D22%26spage%3D9251%26epage%3D9264%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291909" title="Immunology">Immunology</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jmcmar.2013.56.issue-22/production/jmcmar.2013.56.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation. BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation. Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors. Here, we report our structure-guided development of diazobenzene-based small-molecule inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site. Our lead compound, MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estimated <i>K</i><sub>i</sub> of 30–50 nM), with preference for the first bromodomain over the second. We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed production of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages. MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biological functions of the two bromodomains of BRD4 in gene expression.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07571" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07571" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Human DNA is tightly packaged into chromatin by wrapping around the core histone octamer of nucleosomes. The N- and C-termini of the core histones, protruding out from the nucleosome, are subject to a wide variety of post-translational amino acid modifications including acetylation, methylation, phosphorylation, ubiquitinylation, and SUMOylation.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> These chemical modifications of the histones, in combination with DNA modifications, function in a synergistic fashion to regulate gene activation or silencing in chromatin. The ε-N-acetylation of lysine neutralizes positive charges on histones, weakening their interactions with negatively charged DNA. Site-specific lysine acetylation also plays an active role in transitioning chromatin into a relaxed state and directing the recruitment of the gene transcriptional machinery complex for gene activation.</div><div class="NLM_p">The evolutionarily conserved bromodomain (BrD) serves as the acetyl-lysine binding domain<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> to regulate gene activation in chromatin. The 61 human bromodomains embedded in 46 proteins<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> are divided into eight subfamilies with distinctive features based on sequence similarities.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> One major group is BET (bromodomain and extra-terminal domain) proteins composed of BRD2, BRD3, BRD4, and BRDT, each of which contains two tandem bromodomains (BrD1 and BrD2). All bromodomains share a conserved left-handed helix-bundle that is made out of four α helices, named αZ, αA, αB, and αC, respectively. The interhelical loop regions, known as the ZA and BC loops, form the acetyl-lysine binding pocket that is located at one end of the helix bundle. The amino acid residues in the acetyl-lysine binding pocket are highly conserved, with over 90% sequence identity between the two bromodomains in each BET proteins. Of these is a highly conserved Asn residue that is essential for lysine-acetylated histone recognition by forming a hydrogen bond to the acetyl amide group of the acetylated lysine.</div><div class="NLM_p">BRD4, arguably the most extensively characterized BET protein, has been implicated in functions in a wide array of human disorders including cancer,<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6-8)</a> obesity,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> kidney disease,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> lung fibrosis,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and other inflammatory diseases.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> There is growing evidence that the two bromodomains of BRD4 have different biological functions.<a onclick="showRef(event, 'ref10 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref12 ref13">(10, 12, 13)</a> Despite several potent BET inhibitors reported in recent studies,<a onclick="showRef(event, 'ref6 ref7 ref10 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref10 ref14 ref15 ref16 ref17">(6, 7, 10, 14-17)</a> there is still no small-molecule inhibitor shown to be capable of differentiating between the two bromodomains within any individual BET protein. Developing such a selective inhibitor is a challenging task because of the extremely high sequence identity of these bromodomains, particularly at their acetyl-lysine binding pockets.</div><div class="NLM_p last">In this study, we report the structure-guided development of a new class of potent and selective diazobenzene-based small-molecule inhibitors for the BET bromodomains. Our ligand design started with a diazobenzene compound MS120, originally discovered as an inhibitor for the CBP BrD.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Guided by our new structural insights into recognition of the diazobenzenes by both CBP and BRD4 BrDs, we designed specific chemical modifications to acquire lead selectivity toward BRD4 BrDs and conducted extensive structure–activity relationship studies. Our lead compound MS436 exhibits potent affinity of an estimated <i>K</i><sub>i</sub> of 30–50 nM for BRD4 BrD1 and a 10-fold selectivity over the BrD2, which is achieved through a unique set of water-mediated intermolecular interactions. We further demonstrated cellular efficacy of our lead diazobenzene inhibitors in blocking BRD4 transcriptional activity in NF-κB-directed production of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages. We expect that this new class of bromodomain inhibitors will facilitate further mechanistic investigation of the biological functions of the two bromodomains of BRD4 in gene activation in human biology and disease.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54112" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54112" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Structure-Guided Design of Diazobenzenes as BET Bromodomain Inhibitors</h3><div class="NLM_p">We have previously reported MS120 (compound <b>1</b>, Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), (<i>E</i>)-5-((2-amino-4-hydroxy-5-methylphenyl)diazenyl)-2,4-dimethylbenzenesulfonic acid, as a small-molecule inhibitor for the bromodomain of HAT coactivator CBP.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> This compound shows modest activity as assessed in a fluorescent polarization assay toward BRD4 BrD1. The NMR solution structure of the CBP BrD/MS120 complex (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L84">2L84</a>) reveals that MS120 binds at the acetyl-lysine binding pocket with the phenolic hydroxyl group, forming a hydrogen bond with the amide nitrogen on the side chain of Asn1168 of the BrD. This Asn residue is critically important in acetyl-lysine binding and is highly conserved in the bromodomain family.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The overall molecule fits in the narrow groove constituted by the interhelical ZA and BC loops in which both the sulfonate and the amino groups are locked by a pair of electrostatic interactions with the guanidinium group of Arg1173 in the BC loop, a unique residue in the CBP BrD (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A). Our structure-based sequence analysis suggested chemical modifications to engineer this diazobenzene scaffold compound to be selective inhibitors for the BET bromodomains. For example, additional π–π interactions may be established between the ligand and Trp81 in the BRD4 BrD1, which corresponds to Leu1109 of CBP. The acetyl-lysine binding pocket of the BRD4 BrD1 is larger in volume than that of the CBP BrD, thus allowing further modifications to build target selectivity.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure-guided development of diazobenzene-based BrD inhibitors. (A) Three-dimensional solution structure of the BrD inhibitor MS120 (yellow) bound to the CBP BrD (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L84">2L84</a>). The key amino acid residues in the acetyl-lysine binding site are shown in sticks. (B) Structure–activity relationship table illustrating binding affinity of a select number of diazobenzene-based BrD inhibitors to the two BrDs of BRD4. CLogP values were calculated using ChemBioDraw Ultra 12.0.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemical Synthesis of Diazobenezene-Based Bromodomain Inhibitors</h3><div class="NLM_p last">On the basis of the structure-guided design, we first extended the diazobenzene by adding a ring to benzenesulfonate (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B). In this case, we constructed the extended molecule based on <b>2</b>, a MS120 analogue, which itself showed a 2- to 3-fold improvement in binding affinity to the BRD4 BrDs over MS120 (<b>1</b>). The synthetic schemes and procedures are described in detail in the <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a>. The binding affinity of these newly synthesized compounds for various bromodomain proteins was assessed using a fluorescence anisotropy binding assay as described previously.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Specifically, <b>3</b>, an analogue of <b>2</b>, inherited the sulfonyl group at the meta position and extended the system by coupling to 2-aminopyridine. <b>3</b> retained the affinity of <b>2</b>, indicating extension at the meta position is tolerated. Interestingly, as the sulfonamide was repositioned to the para position, the affinity for the BRD4 BrD1 was improved by over 6-fold (MS435, <b>4</b> vs <b>3</b>). However, when we attempted trifluoromethyl-benzylamine as an alternative ring system with an extra carbon in the linker (<b>5</b> in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B and <b>35</b> and <b>36</b> in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 1">Supporting Information Table 1</a>), the inhibitory capacity deteriorated drastically. Therefore, we adopted the optimized 2-aminopyridine as a model system as we continued our lead-optimization campaign.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Structure–Activity Relationship of the C Ring</h3><div class="NLM_p">Encouraged by the initial results, we commenced an investigation of the structure–activity relationship (SAR) on the C ring toward BRD4 BrD1 (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Hydrogen on R<sub>5</sub> is required; substitution of H with a methyl group resulted in a 9-fold loss in activity (<b>6</b> vs MS435). Substitution at this position possibly causes a steric collision with Pro82 (see below). Although substitutions at R<sub>6</sub> and R<sub>8</sub> are generally not tolerated (<b>7</b> and <b>8</b> vs MS435), substitutions at R<sub>7</sub> are well tolerated. In particular, electron-withdrawing groups at R<sub>7</sub>, such as the carboxyl group in compound <b>9</b>, displayed activities with about a 3-fold higher affinity than MS435. Chlorine substitution at R<sub>7</sub> (<b>10</b>) was tolerated; however, when bromine was introduced instead (<b>11</b>), the inhibitory activity toward BRD4 BrD1 declined by 10-fold. The nitrogen on the C ring is preferred, and corresponding carbon analogues displayed lower affinity (for example, <b>11</b> or <b>12</b> vs <b>13</b>, <b>14</b>, or <b>15</b>; <b>16</b> vs <b>17</b>). Notably, we observed that the nitrogen in the C ring resulted in a 13-fold improvement in binding affinity to BRD4 BrD1, whereas there was a 3-fold increase for BRD4 BrD2, thus yielding an overall 4-fold binding selectivity between these two BrDs (<b>12</b> vs <b>13</b>). Nonsubstituted 2-aminopyridine was one of the best pieces in the C ring. Considering its simplicity, stability, and potential impact on the selectivity between the BrD1 and BrD2 of BRD4, we decided to choose 2-aminopyridine as a conserved moiety as we continued to optimize the A and B rings.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure–Activity Relationship of C Ring Analogues of Diazobenzenes</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0007.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0008.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">CLogP values were calculated using ChemBioDraw Ultra 12.0.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structure–Activity Relationship of the A and B Rings</h3><div class="NLM_p">Substitutions on the B ring, such as a methoxyl group, did not lead to tighter binding (<b>18</b> vs MS435) (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Analysis of the crystal structure of MS435 bound to BRD4 BrD1 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C) revealed that the hydroxyl group on the phenol ring was hydrogen bonded to the conserved Asn140. Hence, we primarily concentrated on the optimization of the A ring and built a focused library of over 30 diazobenzene compounds.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure–Activity Relationship of the A or B Ring Analogues of Diazobenzenes</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0009.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0010.gif" alt="" id="GRAPHIC-d1715e459-autogenerated" /></img><div></div></div><div class="NLM_p">We first examined the importance of the hydroxyl group on the A ring. When it was replaced with a methoxy (<b>37</b>) or an acetamide group (<b>38</b>) (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 1">Supporting Information Table 1</a>), the resulting compounds completely lost binding activity, confirming our structural analysis. Detailed study further revealed that bulky groups such as <i>t</i>-butyl and isopropyl at R<sub>1</sub> and R<sub>3</sub> are not tolerated. When one bulky group was introduced (<b>39</b> and <b>40</b> in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 1">Supporting Information Table 1</a>), the affinity decreased about 10-fold as compared to MS435; when two bulky groups were introduced simultaneously (<b>41</b> and <b>42</b>), the compounds almost completely lost their activities (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 1">Supporting Information Table 1</a>). The limited space in the acetyl-lysine binding pocket simply cannot accommodate large functional groups. For the same reason, bicyclic (<b>43</b>–<b>46</b>) systems did not show good activities toward BRD4 BrD1. Flexible substitutions such as aminomethyl and hydroxymethyl were also not tolerated (<b>47</b>–<b>49</b> in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 1">Supporting Information Table 1</a>). However, halides, especially bromides, are generally tolerated, such as in compounds <b>19</b>–<b>25</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). It is worth pointing out that symmetrically bisubstituted phenols appear to outperform their monosubstituted counterparts, suggesting that steric hindrance constrains the rotational freedom between the A ring and diazo linker, locking the substituted A ring in a more favorable conformation. Strong electron-withdrawing groups, such as CF<sub>3</sub> (<b>26</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), likely weakens the hydrogen bond between phenolic OH and Asn140 and is therefore not preferred.</div><div class="NLM_p last">Notably, MS436 (<b>27</b>) is the most potent compound of this set of diazobenzene compounds, as determined in an in vitro fluorescent anisotropy assay, with a <i>K</i><sub>i</sub> value of better than 85 nM for BRD4 BrD1 (limited by the binding affinity of the assay probe; the estimated <i>K</i><sub>i</sub> is 30–50 nM) and an approximately 10-fold selectivity for BrD1 over BrD2 of BRD4 (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). This is the first reported low-nanomolar affinity BrD inhibitor with clear selectivity between these two BrDs of BRD4 as far as we know. MS436 is 10 times more potent than model compound MS435 toward BRD4 BrD1. Interestingly, MS267 (<b>28</b>), a hybrid compound incorporating the key features of MS435 and MS436, exhibited about a 2-fold reduction in potency compared to MS436, although it is still much better than MS435. In addition, placing a Cl at the ortho position with respect to the hydroxyl group in the A ring in another hybrid compound, MS363 (<b>29</b>), showed a 2-fold decrease in binding affinity for both BrDs of BRD4. Nevertheless, both MS267 and MS363 maintained the selectivity for BRD4 BrD1 over BrD2. Notably, a change of the methyl group at R<sub>3</sub> in MS436 to methoxy (<b>30</b>) resulted in a 15-fold decrease in the binding affinity to either BRD4 BrD1 or BrD2. This explains the importance of this methyl group in mimicking the methyl group of acetyl-lysine in a biological ligand. Finally, our endeavors to employ heterocycles in the A ring were unfruitful. Introduction of a nitrogen atom into the A ring aromatic system led to a 3-fold decrease in the affinity (<b>31</b> and <b>32</b> vs MS435).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Structural Basis for Ligand Selectivity</h3><div class="NLM_p">We solved three high-resolution crystal structures of BRD4 BrD1 bound to MS436, MS435, or MS267 to determine the molecular basis of the binding affinity and selectivity of these lead BrD inhibitors (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A–F). In a similar orientation to that of MS120 in the CBP BrD, these much improved diazobenzene inhibitors bind across the acetyl-lysine binding pocket in the ZA and BC loops in BRD4 BrD1. The hydroxyl group in the A ring forms a hydrogen bond to the amide of the conserved Asn140 and another water (W1)-mediated hydrogen bond to the phenoxyl group for Tyr97 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C–F). In addition, the second nitrogen in the diazo linker is engaged in a water (W8)-mediated hydrogen bond to the backbone carbonyl oxygen of the conserved Pro82 that plays an important structural role in many BrDs.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The core diazobenzene moiety is sandwiched between hydrophobic walls formed by Val87, Leu92, Leu94, and Tyr139 on one side and Pro82, Phe83 and Ile146 on the other. Furthermore, the pyridine ring (C ring), particularly in MS435 and MS267, forms π–π interactions with the side-chain idol of Trp81. Collectively, these intermolecular interactions explain the diazobenzene scaffold as a preferred platform as BrD inhibitors for BRD4.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Molecular basis of the BRD4 BrD1 recognition of the lead diazobenzene-based BrD inhibitors. (A) Three-dimensional crystal structure of MS436 (yellow) bound to BRD4-BrD1, depicted in a ribbon diagram. Key amino acid residues at the acetyl-lysine binding site are shown in sticks. Ligand MS436 is color-coded by atom type. (B) Electrostatic surface representation of BRD4-BrD1 bound to MS436. (C) Superimposition of MS436, MS435, and MS267 when bound to BRD4-BrD1. Only the MS436-bound protein structure is shown. Side chains of the key residues involved in ligand binding are depicted in sticks and color-coded by atom type. The bound water molecules are shown as spheres in magenta. (D–F) Schematic diagrams highlighting key interactions in the BRD4 BrD1 recognition of MS436, MS435, and MS267, respectively. Water molecules are shown in magenta spheres, and hydrogen bonds are drawn in dashed line. The figure was generated using the program LIGPLOT.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Notably, besides five water molecules (W1–W5) commonly observed in BrDs and stably located at the bottom of the acetyl-lysine binding pocket,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> we observed several more bound water molecules that are intimately engaged in ligand recognition with these diazobenzene inhibitors, particularly for MS436 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C). Specifically, in addition to W8, which facilitates a water-mediated hydrogen bond between one nitrogen atom of the diazo linker and carbonyl oxygen of Pro82 as described above for all three BrD inhibitors, MS436-bound structure reveals three additional water molecules. For example, W6 bridges hydrogen-bonding interactions between the amino group of the A ring and carbonyl oxygen of the conserved Asn140; the former is engaged in a hydrogen bond with another surface-exposed bound water, W7. In addition, W9 establishes a set of hydrogen-bonding interactions between the sulfonamide oxygen in the B–C ring linker and the side-chain amine of Lys91. The latter is one of very few unique residues in the BrD1 of BRD4; the corresponding residue in the BrD2 is Ala384. Notably, when the 2-aminopyridine sulfonamide (C ring) connectivity to the B ring was moved to meta position to the diazo linker, we observed a 10- or 3-fold reduction in binding affinity to BRD4 BrD1 and BrD2, respectively (<b>51</b> vs MS436, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 1">Supporting Information Table 1</a>). Finally, W6, W7, and W9 are missing in the MS435-bound structure, whereas only W7 and W9 are present in the MS267 structure.</div><div class="NLM_p last">The protein structures of BRD4 BrD1 bound to these diazobenzene inhibitors are nearly identical. Superimposition of these structures reveals that MS435 and MS267 are bound in an almost identical position and orientation, whereas MS436 is upward shifted, anchoring on the hydroxyl group of the A ring, which is positioned at nearly same position in the protein with all three ligands (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C). This shift of MS436 resulted in an increased distance between the pyridine ring (C ring) and Trp81 as compared to that in MS435 or MS267 but yielded additional hydrogen-bonding interactions involving W6, W7, and W9 as compared to MS435 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>D vs <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>E,F). In addition, W8 and W9 in the MS436-bound structure are repositioned accordingly to maintain hydrogen bonds between the protein residues and the ligand as described above. Most notably, the key hydrogen bond formed between the hydroxyl group in the A ring and the amide of Asn140 is reduced to 2.8 Å in MS436 from 3.0 Å in MS435 and MS267. Taken together, these structural insights explain the detailed molecular basis for the potent binding affinity of MS436 that is superior to MS435 and MS267 as a selective BrD inhibitor for BrD1 over BrD2 of BRD4.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Activity of Diazobenzene Compounds against the Bromodomain Panel</h3><div class="NLM_p">We profiled four lead diazobenzene-based BET BrD inhibitors, MS435, MS436, MS267, and MS363, against a panel of bromodomains that represent different subgroups of the human bromodomain family (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). All four inhibitors preferentially bind to the BrDs of BRD4 and BRD3 over the other BrDs tested. Notably, MS436 activity toward CBP BrD is four times better than initial compound MS120. It is worth mentioning that these diazobenzene compounds have a different bromodomain activity profile from diazepine-based BrD inhibitors such as JQ1,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> MS417,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and I-BET.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Specifically, MS267 showed binding affinity <i>K</i><sub>i</sub> values of <1.43, <1.70, and 2.26 μM for the BrDs of BRD7, BPTF, and BAZ2b, respectively. MS435 showed a <i>K</i><sub>i</sub> value of <6.56 μM for SMARCA4 BrD. These unique activity profiles make them potentially useful probes for studies of these non-BET BrDs.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Lead BrD Inhibitors against a Panel of BrDs</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">BrD protein</th><th class="colsep0 rowsep0" align="center" char="."><b>27</b> (MS436)<br /></br><i>K</i><sub>i</sub>, μM</th><th class="colsep0 rowsep0" align="center" char="."><b>4</b> (MS435)<br /></br><i>K</i><sub>i</sub>, μM</th><th class="colsep0 rowsep0" align="center" char="."><b>28</b> (MS267)<br /></br><i>K</i><sub>i</sub>, μM</th><th class="colsep0 rowsep0" align="center" char="."><b>29</b> (MS363)<br /></br><i>K</i><sub>i</sub>, μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4-BrD1<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="."><0.085</td><td class="colsep0 rowsep0" align="char" char=".">0.91</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4-BrD2<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">4.30</td><td class="colsep0 rowsep0" align="char" char=".">0.45</td><td class="colsep0 rowsep0" align="char" char=".">0.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3-BrD1<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">0.10</td><td class="colsep0 rowsep0" align="char" char=".">1.64</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3-BrD2<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">0.14</td><td class="colsep0 rowsep0" align="char" char=".">2.48</td><td class="colsep0 rowsep0" align="char" char=".">0.22</td><td class="colsep0 rowsep0" align="char" char=".">0.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CBP BrD<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">2.18</td><td class="colsep0 rowsep0" align="char" char=".">7.15</td><td class="colsep0 rowsep0" align="char" char=".">5.12</td><td class="colsep0 rowsep0" align="char" char=".">5.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PCAF<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">5.52</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><3.34</td><td class="colsep0 rowsep0" align="char" char="."><3.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD7<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">2.72</td><td class="colsep0 rowsep0" align="char" char=".">3.54</td><td class="colsep0 rowsep0" align="char" char="."><1.43</td><td class="colsep0 rowsep0" align="char" char="."><1.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BPTF<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">6.06</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."><1.7</td><td class="colsep0 rowsep0" align="char" char=".">1.62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BAZ2b<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">3.29</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">2.26</td><td class="colsep0 rowsep0" align="char" char=".">3.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMARCA4<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">7.97</td><td class="colsep0 rowsep0" align="char" char="."><6.56</td><td class="colsep0 rowsep0" align="char" char="."><6.56</td><td class="colsep0 rowsep0" align="char" char="."><6.56</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn0"><div class="footnote" id="t3fn0"><p class="first last">Note: Assay probe used:</p></div><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">MS574;</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">MS226;</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last">MS239 (see details in <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a>).</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Physiochemical Properties of the Top Diazobenzene Bromodomain Inhibitors</h3><div class="NLM_p">We evaluated the physicochemical properties, including CLogP and lipophilic ligand efficiency (LLE),<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> of our top four diazobenzene-based bromodomain inhibitors, MS435, MS436, MS267, and MS363, to understand further their potential in cellular study. Of these, MS436 shows the best physiochemical properties (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). As suggested in recent toxicology studies, compounds with CLogP <3 have much lower potential adverse effects in vivo.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> MS436 has CLogP of 2.65, thus belonging to the low-risk category. MS436 strikes a good balance between lipophilicity and potency. The lipophilic ligand efficiency of MS436 (LLE = 4.42) is also well above the other diazobenzene analogues and is in line with the most probable LLE distribution (2–6) as reported in a recent statistical analysis of successful drug molecules.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Physiochemical Properties of Compounds against BRD4-BrD1</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>i</sub>, μM</th><th class="colsep0 rowsep0" align="center" char=".">p<i>K</i><sub>i</sub></th><th class="colsep0 rowsep0" align="center" char=".">CLogP</th><th class="colsep0 rowsep0" align="center" char=".">lipophilic ligand efficiency</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="center">MS436</td><td class="colsep0 rowsep0" align="char" char="."><0.0845</td><td class="colsep0 rowsep0" align="char" char=".">7.07</td><td class="colsep0 rowsep0" align="char" char=".">2.65</td><td class="colsep0 rowsep0" align="char" char=".">>4.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="center">MS435</td><td class="colsep0 rowsep0" align="char" char=".">0.91</td><td class="colsep0 rowsep0" align="char" char=".">6.04</td><td class="colsep0 rowsep0" align="char" char=".">4.26</td><td class="colsep0 rowsep0" align="char" char=".">1.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="center">MS267</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td><td class="colsep0 rowsep0" align="char" char=".">6.82</td><td class="colsep0 rowsep0" align="char" char=".">3.05</td><td class="colsep0 rowsep0" align="char" char=".">3.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="center">MS363</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td><td class="colsep0 rowsep0" align="char" char=".">6.77</td><td class="colsep0 rowsep0" align="char" char=".">3.17</td><td class="colsep0 rowsep0" align="char" char=".">3.60</td></tr></tbody></table></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Activities of Diazobenzene Compounds in Murine Macrophage RAW264.7 Cells</h3><div class="NLM_p">These four lead diazobenzene bromodomain inhibitors exhibited little observable cytotoxicity on cell growth and proliferation, as determined in an MTT assay in murine macrophage RAW264.7 cells. As shown by this MTT study, the cell viability was fairly stable with these bromodomain inhibitors at concentrations up to 100 μM (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A). We next evaluated the cellular efficacy of these bromodomain inhibitors in blocking BRD4 functions in gene transcription. Nitric oxide synthase (NOS) catalyzes a stoichiometric production of nitric oxide (NO) from <span class="smallcaps smallerCapital">l</span>-arginine in cells. NO production in macrophages is mediated by inducible nitric oxide synthase (iNOS), which is regulated by the NF-κB pathway.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> As shown in our recent study, NF-κB transcriptional activity for target-gene activation is dependent upon its lysine-acetylation-mediated interactions with BRD4,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> which recruits the activated NF-κB to its target gene sites by binding to diacetylated histone H4, particularly H4K5ac/K8ac. Indeed, we observed that treatment with the diazobenzene compounds in a dose-dependent manner blocked NF-κB-directed NO production in RAW264.7 cells upon LPS stimulation (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B). In agreement with their in vitro binding affinity to the BRD4 BrD1, MS436 showed a more profound inhibitory activity on NO production than MS435. Consistent with the fact that the BET proteins are functionally vital for macrophage inflammatory responses,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> these diazobenzene bromodomain inhibitors effectively block LPS-induced proinflammatory cytokine interleukin (IL)-6 expression in the macrophage cells, as illustrated in an ELISA assay (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C), of which MS436 and MS363 showed the most profound inhibitory activity, which was similar to their activity in the NO inhibition study. Taken together, these results demonstrate that these lead diazobenzene bromodomain inhibitors, particularly MS436, can effectively modulate BRD4 functions in gene transcriptional activation in cells.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of effects of the diazobenzene BrD inhibitors on BRD4 function in gene transcription. (A) Effects of cell viability on the BrD inhibitors in murine macrophage RAW264.7 cells. Mitochondrial respiration, an indicator of cell viability, was assayed by the mitochondrial-dependent reduction of MTT to formazan. (B) Effects of the BrD inhibitors on LPS-induced IL-6 secretion in murine macrophage RAW264.7 cells. (C) Effects of the BrD inhibitors on LPS-induced NO production in murine macrophage RAW264.7 cells. (D) Estimated efficiency of the four lead BrD inhibitors in the cellular assays as described in panels B and C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81385" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81385" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, we report our structure-guided development of a new class of diazobenzene-based small-molecule inhibitors for the BET bromodomains. MS436 is the best inhibitor yielded from our extensive lead optimization using a combined medicinal chemistry and structure–activity relationship study. We improved the affinity of the diazobenzene compounds toward BRD4 BrD1 by over 100-fold. MS436 has an estimated <i>K</i><sub>i</sub> of 30–50 nM for BRD4 BrD1, with a 10-fold selectivity over BrD2. These lead diazobenzene compounds possess preferable druglike properties. We further demonstrated the cellular efficacy of four lead diazobenzene BrD inhibitors in inhibiting BRD4 transcriptional activity in LPS-activated, NF-κB-directed production of nitric oxide and proinflammatory cytokine IL-6 in murine macrophage RAW264.7 cells. To our knowledge, MS436 is the first low-nanomolar small-molecule bromodomain inhibitor that is selective between the two structurally highly similar BrDs of BRD4. It compares favorably to the other recently reported BRD4 BrD inhibitors such as 3,4-dihydro-3methyl-2(1<i>H</i>)-quinazolinones<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and 3,5-dimethoxyl-isoxazole,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> which do not show selectivity between the two BrDs of BRD4.</div><div class="NLM_p">Our detailed high-resolution crystal structural analysis of the BRD4 BrD1 bound to the lead diazobenzene ligands reveals that the residues in the acetyl-lysine binding pocket adopt a rather rigid conformation that is not influenced significantly upon binding to different small-molecule ligands or the acetylated-lysine in a histone peptide (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Surprisingly, we found that a set of structurally bound water molecules work together to direct ligand recognition through bridging multiple water-mediated hydrogen-bonding interactions between the key conserved residues and a bound ligand. It is interesting to note that through such versatile and cooperative activity, bound water molecules play a direct role in ligand recognition by the bromodomains. These observations likely reflect the dynamic and transient nature of acetyl-lysine binding by the bromodomains, which is necessary for directing the protein–protein interactions required for gene transcriptional activation on-demand and in an ordered fashion in the context of chromatin.</div><div class="NLM_p last">Unlike other bromodomains of the HAT coactivators PCAF and CBP/p300, the BET bromodomains distinctly prefer to interact with multilysine acetylation sites in histones and transcription-associated proteins. Growing evidence from our studies and others suggest that the two bromodomains of BRD4 are engaged in different molecular functions for the control of gene transcriptional activation in chromatin.<a onclick="showRef(event, 'ref10 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref12 ref13">(10, 12, 13)</a> Specifically, the first bromodomain of BRD4 binds to hyper-acetylated histone H4 by recognizing H4K5ac/K8ac, a dual-acetylation mark that signals for gene activation, whereas the second bromodomain functions to recruit activated and lysine-acetylated transcription factors to target gene promoter and enhancer sites as well as to bridge triacetylated cyclin T1 of p-TEFb to RNA polymerase II to establish an activated transcriptional-machinery complex for productive transcriptional elongation. Our new diazobenzene-based bromodomain inhibitors that are potent and selective for these two BrDs of BRD4, particularly MS436, and should facilitate further mechanistic investigation of specific functions of the two bromodomains of BRD4 in various physiological and pathophysiological contexts. Such studies are needed to guide the future development of novel and safe small-molecule BRD4 inhibitors as new targeted epigenetic therapies to treat human disorders including cancer and inflammation.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Protein Preparation</h3><div class="NLM_p last">Expression and purification of the recombinant bromodomains of various transcriptional proteins in poly-His tagged form were performed using a procedure described previously.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The protein was purified by affinity chromatography on a nickel-IDA column (Invitrogen) followed by the removal of the poly-His tag by thrombin cleavage.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Fluorescence Anisotropy Binding Assay</h3><div class="NLM_p last">Binding affinity of the newly synthesized diazobenzene compounds for various bromodoamins was assessed in a fluorescence anisotropy competition assay using fluorescein-labeled MS417 (MS574) as an assay probe for BET BrDs, as described previously.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> For non-BET BrDs, different fluorescein-labeled probes (unpublished data, M.-M.Z.), were used as indicated in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. Competition experiments were performed with a BrD protein (0.25–1 μM) and the fluorescent probe (80 nM) and increasing concentration of unlabeled competing ligand in a PBS buffer (pH 7.4) in total volume of 80 μL. Measurements were obtained after a 1 h incubation of the fluorescent ligand and the protein at 25 °C with a Safire 2 microplate reader (Tecan, Research Triangle Park, NC). In a competition-binding assay, the fluorescent ligand concentration was ≤2<i> K</i><sub>d</sub>, and the protein concentration was set such that 50–80% of fluorescent ligand is bound. The dissociation constant of a competing ligand was calculated with the correction to Cheng–Prussoff equation introduced by Nicolovska-Coleska and colleagues.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Assuming a one-site competitive-binding model, the equation used to calculate <i>K</i><sub>i</sub> values from IC<sub>50</sub> values recovered from fitting data using Prism is<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/jm-2013-01334s_m001.gif" alt="" /></img></span>where [I<sub>50</sub>] is the concentration of free inhibitor at 50% inhibition, [L<sub>50</sub>], the concentration of free labeled ligand at 50% inhibition, and [P<sub>0</sub>], the concentration of free protein at 0% inhibition. Note that <i>K</i><sub>d</sub> for each protein–probe pair is the limit of resolvable <i>K</i><sub>i</sub> in a competition assay.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Protein Crystallization, X-ray Diffraction Data Collection, and Structure Determination</h3><div class="NLM_p last">Purified BRD4-BrD1 protein (14 mg/mL) was mixed with a diazobenzene BrD inhibitor at a 1:10 molar ratio of protein/ligand. The complex was crystallized using the sitting-drop vapor-diffusion method at 20 °C by mixing 1 μL of protein solution with 1 μL of the reservoir solution containing 15–30% PEG 4000, 0.2 M MgCl<sub>2</sub>, and 0.1 M Tris-HCl, pH 8.5. Crystals were soaked in the corresponding mother liquor supplemented with 20% ethylene glycerol as a cryoprotectant before freezing in liquid nitrogen. X-ray diffraction data were collected at 100 K at beamline X6A of the National Synchrotron Light Source (NSLS) at Brookhaven National Laboratory. Data were processed using the HKL-2000 suite.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The BRD4-BD1 structure was solved by molecular replacement using the program MOLREP,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and the structure refinement was done using the program Refmac.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Graphics program COOT<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> was used for model building and visualization. Crystal diffraction data and refinement statistics for the structure are displayed in <a class="ref internalNav" href="#notes-1" aria-label="Table S2">Table S2</a> (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cell Viability Study of Murine Macrophage Cells</h3><div class="NLM_p last">Murine macrophage RAW264.7 cells were plated at a density of 1 × 10<sup>4</sup> cells per well in a 96-well plate and incubated at 37 °C for 18 h. The cells were then treated with the diazobenzene bromodomain inhibitors up to 100 μM for 24 h. At the end of the 24 h incubation, 10 μL of the MTT solution (4 mg/mL) was added to each well, and the plates were incubated at 37 °C for 4 h. The supernatants were then removed, and the cells were solubilized in 100 μL of 100% DMSO. The diazobenzene compounds were first dissolved in DMSO and then diluted with culture medium to concentrations that ranged from 0.28 to 50 000 nM. The final concentration of DMSO was adjusted to 0.05% (v/v). The extent of the reduction was measured by the absorbance at 570/630 nm using an EnVison 2104 multilabel reader (PerkinElmer, Inc., Waltham, MA).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Assessing LPS-Induced IL-6 and Nitric Oxide Levels in Murine Macrophage Cells</h3><div class="NLM_p last">Murine macrophage RAW264.7 cells were cultivated in DMEM (HyClone, Logan, UT) supplemented with 10% FBS (HyClone) at 37 °C in a humidified atmosphere of 5% CO<sub>2</sub>. Cells in 96-well plates (0.1 mL, 3 × 10<sup>5</sup> cells/mL) were treated with lead diazobenzene inhibitors. After 30 min, all supernatants were removed, and the cells were treated with LPS (1 μg/mL) (Sigma-Aldrich Chemical Co., St. Louis, MO) and lead diazobenzene inhibitors. After 24 h, the supernatant was collected and measured using a mouse IL-6 ELISA assay kit (Thermo Scientific, Pittsburgh, PA). The lead diazobenzene bromodomain inhibitors were first dissolved in DMSO (Sigma-Aldrich Chemical Co.) and then diluted with culture medium to concentrations that ranged from 0.28 to 50 000 nM. The final concentration of DMSO was adjusted to 0.05% (v/v). The assay was measured by an absorption reading at 570 nm using an EnVison 2104 multilabel reader (PerkinElmer, Inc.). For assessing LPS-induced nitric oxide release, nitrite production was measured by spectrophotometry at 520 nm using an EnVison 2104 multilabel reader (PerkinElmer, Inc.). Each experiment was performed in triplicate and plotted using Prism 5.0 (GraphPad Software, Inc.). The curve-fitting equation used was log(inhibitor) versus response – variable slope (four parameters).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Chemical Synthesis</h3><div class="NLM_p">The core of the diazobenzene compounds was constructed by azo coupling<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> of the diazonium chloride intermediate <b>2</b> with appropriately functionalized phenol (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> For C ring diazo analogues, 4-aminobenzenesulfonyl chloride precursors were first coupled to functionalized aromatic amines. The protective groups of the aromatic amines were either unmasked by sodium hydroxide aqueous solution or reduced with iron in ammonium chloride aqueous solution. Key intermediate <b>A</b> was accessed by treating the anilines with glacial acetic acid and isoamyl nitrite sequentially. Subsequent azo coupling with 2,6-dimethylphenol in the presence of potassium carbonate gave a variety of C ring diazobenzene analogues. A ring diazobenzene analogues were accessed through a simpler procedure. Sulfapyridine was treated with concentrated hydrogen chloride, and the resulting diazonium chloride reacted readily with various phenols under basic conditions to give diversified A ring diazobenzene analogues (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>).</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Scheme for C Ring Analogues of Diazobenzene</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. General Synthesis of a Ring Analogues of Diazobenzene</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">All nonaqueous reactions were carried out in oven-dried glassware under an atmosphere of argon. All solvents were purchased in anhydrous from Acros Organics and used without further purification. Automatic chromatography was performed on a Biotage Isolera system equipped with a variable wavelength detector and a fraction collector using a Biotage SNAP cartridge KP-Sil 10 g. Analytical thin-layer chromatography (TLC) was performed employing Sigma-Aldrich 250 μm 60F-254 silica plates. The plates were visualized either by exposure to UV light, staining with iodine-impregnated silica gel, or staining with ceric ammonium molybdate (CAM). Preparative TLC was performed employing Silicycle 1000 μm SiliaPlate Prep silica plates. LCMS analysis was conducted on an Agilent Technologies G1969A high-resolution API-TOF mass spectrometer attached to an Agilent Technologies 1200 HPLC system. Samples were ionized by electrospray ionization (ESI) in positive mode. Chromatography was performed on a 2.1 × 150 mm Zorbax 300SB-C18 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). The temperature of the column was held at 50 °C for the entire analysis. NMR spectra were acquired on a Bruker DRX-600 spectrometer at 600 MHz for <sup>1</sup>H and 150 MHz for <sup>13</sup>C. Chemical shifts are expressed in parts per million downfield from tetramethylsilane (TMS) using either TMS or the solvent resonance as an internal standard (TMS, <sup>1</sup>H: 0 ppm; chloroform, <sup>13</sup>C: 77.0 ppm; DMSO-<i>d</i><sub>6</sub>, <sup>1</sup>H: 2.50 ppm; <sup>13</sup>C: 39.5 ppm; methanol-<i>d</i><sub>4</sub>, <sup>1</sup>H: 3.31 ppm; <sup>13</sup>C: 49.0 ppm). Data are reported as follows: chemical shift, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br = broad), integration, and coupling constant. IR spectra were obtained on a Bruker TENSOR 27 series FT-IR spectrometer equipped with a Diamond ATR. Melting point was measured on an OptiMelt automatic melting point system MPA100 from Stanford Research System.</div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General Experimental Procedures</h4><div id="sec4_6_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> <i>(E)-</i>5-((2-Amino-4-hydroxy-5-methylphenyl)diazenyl)-2,4-dimethylbenzenesulfonic acid (<b>1</b>)</h5><div class="NLM_p last">A 100 mL round-bottomed flask was charged with concentrated HCl (5 mL) and crushed ice (1 g) and cooled to 0 °C. To this flask were added 5-amino-2,4-xylene-sulfonic acid (230 mg, 1.15 mmol, 1.0 equiv) and NaNO<sub>2</sub> (1 N, 1 mL, 1.15 mmol, 1 equiv). The mixture was stirred at 0 °C for 2 h. 5-Amino-2-methylphenol (155 mg, 1.26 mmol, 1.2 equiv) was dissolved in a 10% NaOH(aq) solution (20 mL, 50 equiv). The previously prepared yellow color diazonium ion was added dropwise under argon over 10 min. The pH of the solution was maintained between 8 and 10 with the addition of small amount of 10% NaOH(aq). The solution was allowed to stir at 0 °C for 1 h and was quenched with 1 N HCl and adjusted to pH 7. Filtration provided the product as a fine red powder (335 mg, 95%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.20 (br s, 1H), 7.97 (s, 1H), 7.48 (s, 1H), 7.11 (s, 1H), 6.41 (s, 1H), 2.51 (s, 3H), 2.48 (s, 3H), 2.07 (s, 3H). <sup>13</sup>C NMR (150 MHz, MeOD-<i>d</i><sub>4</sub>) δ 155.5, 148.0, 144.2, 141.5, 139.4, 139.2, 138.6, 134.0, 119.3, 117.5, 116.8, 114.4, 19.6, 16.0, 15.9. MS (<i>m</i>/<i>z</i>): calcd for C<sub>15</sub>H<sub>18</sub>N<sub>3</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 336.10; found, 336.10. Purity >95%, <i>t</i><sub>R</sub> = 4.4 min.</div></div><div id="sec4_6_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> (<i>E</i>)-5-(3,5-Dimethyl-4-hydroxyphenylazo)-2,4-dimethylbenzenesulfonic acid (<b>2</b>)</h5><div class="NLM_p last">Following a similar procedure as compound <b>1</b>, compound <b>2</b> was obtained as a yellowish solid (74%). <sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.33 (s, 1H), 7.71 (s, 2H), 7.40 (s, 1H), 2.84 (s, 3H), 2.82 (s, 3H), 2.45 (s, 6H). <sup>13</sup>C NMR (150 MHz, MeOD-<i>d</i><sub>4</sub>) δ 151.1, 143.4, 137.7, 135.1, 133.7, 132.6, 130.4, 129.5, 110.5, 108.5, 29.3, 22.6, 21.1. MS (<i>m</i>/<i>z</i>): calcd for C<sub>16</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 335.11; found, 335.13.</div></div><div id="sec4_6_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> <i>(E)-</i>3-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>3</b>)</h5><div class="NLM_p last">A 250 mL round-bottomed flask was charged with 4-nitrobenzenesulfonyl chloride (500.0 mg, 2.26 mmol, 1.0 equiv) dissolved in DCM (40 mL). The solution was cooled to 0 °C. 2-Aminopyridine (212 mg, 2.26 mmol, 1.0 equiv) was dissolved in DCM (5 mL) and added carefully at 0 °C dropwise over 20 min. After 1 h, the solution was gradually warmed to rt and stirred overnight. The reaction was quenched by water (30.0 mL). The organic layer was washed sequentially with water and brine and dried over MgSO<sub>4</sub>. The volatiles were removed under reduced pressure. The crude product was recrystallized in a mixture of DCM and MeOH. Pure product <b>A</b> appeared as a pale-yellow powder (410.0 mg, 65%). A 100 mL pressure tube was charged with compound <b>A</b> (410.0 mg, 1.47 mmol, 1.0 equiv) and was dissolved in a 1:1 (v/v) MeOH/H<sub>2</sub>O mixture (20 mL). To this mixture were added iron (240 mg, 4.33 mmol, 2.95 equiv) and NH<sub>4</sub>Cl(aq) (400 mg, 7.34 mmol, 5.0 equiv). The mixture was heated at 70 °C for 4 h. The reaction mixture was cooled and filtered through a Celite pad to remove the inorganic residues. The pad was washed (10 mL × 3) with acetone. The combined filtrate was concentrated in vacuo. DI water (40 mL) was added, and the pH was adjusted to basic by the addition of NaHCO<sub>3</sub>. The solid was filtered and dried. Pure product <b>B</b> appeared as gray powder (250 mg, 68%). A 100 mL round-bottomed flask was charged with <b>B</b> (100.0 mg, 0.41 mmol, 1.0 equiv) and concentrated HCl (0.16 mL, 0.41 mmol, 1.0 equiv). The mixture was dissolved in a MeOH/ACN mixture 1.5 mL/1.5 mL. The reaction solution was stirred at 0 °C for 15 min. Isoamyl nitrite (55 μL, 0.41 mmol, 1.0 equiv) was added dropwise under argon over 15 min. The bright-yellow solution was stirred at 0 °C for 1 h and gradually warmed to rt for 1 h. 2,6-Dimethylphenol (50.0 mg, 0.41 mmol, 1.0 equiv) and K<sub>2</sub>CO<sub>3</sub> (280.6 mg, 2.03 mmol, 5.0 equiv) were premixed and deoxygenized by argon for 15 min. The previously prepared amber color diazonium ion <b>C</b> was added dropwise under argon. At the end of the addition, the pH of the solution was maintained between 8 and 10 using K<sub>2</sub>CO<sub>3</sub>. The solution was stirred at 0 °C for 1 h and then at rt overnight. The resultant solution was adjusted to pH 1. The solid was filtered and dried. Pure product <b>3</b> appeared as an orange powder (134.1 mg, 86%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.29 (br s, 1H), 8.18 (s, 1H), 7.98 (d, <i>J</i> = 7.8 Hz, 1H), 7.97–7.92 (m, 2H), 7.76 (dt, <i>J</i> = 8.4, 1.5 Hz, 1H), 7.70 (t, <i>J</i> = 7.8 Hz, 1H), 7.59 (s, 2H), 7.20 (d, <i>J</i> = 9.0 Hz, 1H), 6.84 (t, <i>J</i> = 6.0 Hz, 1H), 2.24 (s, 6H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.9, 156.8, 155.2, 148.0, 146.9, 144.5, 133.5, 131.4, 130.8, 129.8, 128.3, 127.0, 122.3, 121.4, 117.7, 19.9. MS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 383.11; found, 383.11. Purity >99%, <i>t</i><sub>R</sub> = 5.0 min.</div></div><div id="sec4_6_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> <i>(E)-</i>4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>4</b>, MS435)</h5><div class="NLM_p last">Following a similar procedure as compound <b>3</b>, compound <b>4</b> was prepared as an orange powder (80%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.2 (br s, 1H), 9.33 (br s, 1H), 8.00 (d, <i>J</i> = 8.4 Hz, 1H), 7.98–7.93 (m, 2H), 7.87 (d, <i>J</i> = 8.4 Hz, 2H), 7.74 (t, <i>J</i> = 8.4 Hz, 1H), 7.57 (s, 2H), 7.20 (d, <i>J</i> = 9.0 Hz, 1H), 6.85 (t, <i>J</i> = 6.6 Hz, 1H), 2.14 (s, 6H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.3, 154.3, 154.0, 145.4, 143.7, 141.8, 141.6, 128.6, 125.6, 124.3, 122.8, 120.0, 114.8, 17.1. MS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 383.11; found, 383.11. Purity >99%, <i>t</i><sub>R</sub> = 2.1 min. (Chromatography was performed on a 2.1 × 50 mm Acquity BEH C-18 column using acetonitrile as solvent A and water containing 0.025% TFA as solvent B at a flow rate of 0.5 mL/min. The gradient program was as follows: 90% B (0–0.5 min), 90–10% B (0.5–3 min), and 10% B (3–6 min).)</div></div><div id="sec4_6_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> (<i>E</i>)-4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-(4-(trifluoromethyl)benzyl)benzenesulfonamide (<b>5</b>)</h5><div class="NLM_p"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 7.93 (d, <i>J</i> = 8.4 Hz, 2H), 7.89 (d, <i>J</i> = 8.4 Hz, 2H), 7.62 (s, 2H), 7.54 (d, <i>J</i> = 8.4 Hz, 2H), 7.44 (d, <i>J</i> = 8.4 Hz, 2H), 4.20 (s, 2H), 2.30 (s, 6H). <sup>13</sup>C NMR (150 MHz, MeOD-<i>d</i><sub>4</sub>) δ 160.5, 157.7, 148.6, 144.7, 144.0, 132.1, 131.9, 130.8, 130.5, 127.9, 127.6 (d), 126.6, 125.1, 48.7, 18.1. IR (neat): ν 3498 (−OH), 3247 (−NH−), 1616 (−C═N−), 1591 (−N═N−), 1382 (−SO<sub>2</sub>−). MS (<i>m</i>/<i>z</i>): calcd for C<sub>22</sub>H<sub>21</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 464.12; found, 464.12. Purity >99%, <i>t</i><sub>R</sub> = 6.5 min. mp 195.7 °C.</div><div class="NLM_p last">The following compounds were synthesized following Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>.</div></div><div id="sec4_6_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> (<i>E</i>)-4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-methyl-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>6</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.16–8.12 (m, 1H), 8.10 (d, <i>J</i> = 8.4 Hz, 2H), 7.67 (d, <i>J</i> = 8.4 Hz, 2H), 7.63 (s, 2H), 7.04 (t, <i>J</i> = 6.6 Hz, 1H), 6.86 (d, <i>J</i> = 7.2 Hz, 1H), 6.63 (t, <i>J</i> = 7.2 Hz, 1H), 2.30 (s, 6H), 2.18 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.1, 156.9, 155.6, 148.3, 146.7, 143.1, 140.9, 130.7, 127.8, 126.8, 122.1, 115.1, 114.1, 31.5, 18.1. MS (<i>m</i>/<i>z</i>): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 397.13; found, 397.13. Yield: 31%, purity >99%, <i>t</i><sub>R</sub> = 4.8 min.</div></div><div id="sec4_6_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> (<i>E</i>)-4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-N-(3-(trifluoromethyl)phenyl)benzenesulfonamide (<b>7</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.90 (m, 4H), 7.65 (s, 2H), 7.38 (m, 3H), 7.29 (br s, 1H), 6.82 (s, 1H), 5.10 (s, 1H), 2.34 (s, 6H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>19</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 450.11; found, 450.11. Yield: 68%, purity >99%, <i>t</i><sub>R</sub> = 6.5 min.</div></div><div id="sec4_6_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> (<i>E</i>)-<i>N</i>-(4-Chloropyridin-2-yl)-4-((4-hydroxy-3,5-dimethylphenyl)diazenyl)benzenesulfonamide (<b>8</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.39 (br s, 1H), 8.18 (d, <i>J</i> = 1.8 Hz, 1H), 8.04 (d, <i>J</i> = 7.8 Hz, 2H), 7.89 (d, <i>J</i> = 7.8 Hz, 2H), 7.79 (dd, <i>J</i> = 9.0, 1.8 Hz, 1H), 7.56 (s, 2H), 7.08 (d, <i>J</i> = 9.0 Hz, 1H), 7.01 (d, <i>J</i> = 5.6 Hz, 1H), 2.23 (s, 6H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.2, 157.7, 153.2, 149.3, 148.2, 143.9, 141.7, 131.8, 128.3, 127.1, 125.6, 121.8, 116.7, 19.8. MS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>18</sub>ClN<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 417.08; found, 417.08. Yield: 87%, purity >99%, <i>t</i><sub>R</sub> = 5.3 min.</div></div><div id="sec4_6_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> (<i>E</i>)-6-(4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)phenylsulfonamido)nicotinic Acid (<b>9</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.31 (br s, 1H), 8.24–8.14 (m, 2H), 8.03 (d, <i>J</i> = 8.4 Hz, 2H), 7.88 (d, <i>J</i> = 8.4 Hz, 2H), 7.72 (s, 1H), 7.57 (s, 2H), 7.50 (s, 1H), 7.27 (d, <i>J</i> = 4.8 Hz, 2H), 2.24 (s, 6H). <sup>13</sup>C NMR (150 MHz, MeOD-<i>d</i><sub>4</sub>) δ 169.0, 161.2, 157.3, 156.1, 148.1, 145.0, 131.4, 128.4, 127.1, 125.5, 122.3, 117.7, 114.8, 19.9. MS (<i>m</i>/<i>z</i>): calcd for C<sub>20</sub>H<sub>18</sub>N<sub>4</sub>O<sub>5</sub>S [M]<sup>+</sup>, 426.11; found, 426.11. Yield: 24%, purity >99%, <i>t</i><sub>R</sub> = 5.1 min.</div></div><div id="sec4_6_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> (<i>E</i>)-<i>N</i>-(5-Chloropyridin-2-yl)-4-((4-hydroxy-3,5-dimethylphenyl)diazenyl)benzenesulfonamide (<b>10</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.36 (br s, 1H), 8.03 (d, <i>J</i> = 7.8 Hz, 4H), 7.89 (d, <i>J</i> = 7.8 Hz, 2H), 7.57 (s, 2H), 7.16 (s, 1H), 7.01 (d, <i>J</i> = 5.6 Hz, 1H), 2.24 (s, 6H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.1, 157.4, 156.6, 149.2, 148.2, 145.1, 131.7, 131.3, 128.3, 127.1, 125.6, 119.9, 115.8, 19.8. MS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>18</sub>ClN<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 417.08; found, 417.07. Yield: 87%, purity >99%, <i>t</i><sub>R</sub> = 5.3 min.</div></div><div id="sec4_6_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> (<i>E</i>)-<i>N</i>-(5-Bromopyridin-2-yl)-4-((4-hydroxy-3,5-dimethylphenyl)diazenyl)benzenesulfonamide (<b>11</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 7.89–7.80 (m, 4H), 7.58 (s, 2H), 7.36 (d, <i>J</i> = 8.4 Hz, 2H), 7.04 (d, <i>J</i> = 8.4 Hz, 2H), 2.27 (s, 6H). <sup>13</sup>C NMR (150 MHz, MeOD-d<sub>4</sub>) δ 160.6, 157.9, 148.5, 142.6, 139.6, 134.6, 130.7, 127.5, 126.6, 125.5, 125.0, 120.2, 18.1. MS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>18</sub>BrN<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 462.03; found, 462.00. Yield: 55%, purity >99%, <i>t</i><sub>R</sub> = 5.6 min.</div></div><div id="sec4_6_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> (<i>E</i>)-4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-(5-methoxypyridin-2-yl)benzenesulfonamide (<b>12</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.85 (s, 1H), 9.35 (s, 1H), 7.99 (d, <i>J</i> = 8.4 Hz, 2H), 7.92–7.85 (m, 3H), 7.59 (s, 2H), 7.37 (m, 1H), 7.11 (m, 1H), 3.73 (s, 3H), 2.26 (s, 6H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 413.13; found, 413.12. Yield: 57%, purity >99%, <i>t</i><sub>R</sub> = 6.2 min.</div></div><div id="sec4_6_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> (<i>E</i>)-4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-(4-methoxyphenyl)benzenesulfonamide (<b>13</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.86 (d, <i>J</i> = 8.4 Hz, 2H), 7.79 (d, <i>J</i> = 8.4 Hz, 2H), 7.65 (s, 2H), 6.98 (d, <i>J</i> = 9.0 Hz, 2H), 6.77 (d, <i>J</i> = 9.0 Hz, 1H), 6.28 (s, 1H), 5.10 (s, 1H), 3.76 (s, 3H), 2.34 (s, 6H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>22</sub>N<sub>3</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 412.13; found, 412.13. Yield: 57%, purity >99%, <i>t</i><sub>R</sub> = 6.3 min.</div></div><div id="sec4_6_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> (<i>E</i>)-4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-(4-(trifluoromethyl)phenyl)benzenesulfonamide (<b>14</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.02 (s, 1H), 9.38 (s, 1H), 7.98 (d, <i>J</i> = 8.4 Hz, 2H), 7.92 (d, <i>J</i> = 8.4 Hz, 2H), 7.63 (d, <i>J</i> = 8.4 Hz, 2H), 7.59 (s, 2H), 7.31 (d, <i>J</i> = 8.4 Hz, 2H), 2.26 (s, 6H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>19</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 450.11; found, 450.13. Purity >95%, <i>t</i><sub>R</sub> = 6.5 min.</div></div><div id="sec4_6_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> (<i>E</i>)-4-(4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)phenylsulfonamido)benzamide (<b>15</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.81 (br s, 1H), 8.36 (d, <i>J</i> = 8.4 Hz, 1H), 8.05 (d, <i>J</i> = 8.4 Hz, 1H), 8.00 (d, <i>J</i> = 8.4 Hz, 1H), 7.92 (d, <i>J</i> = 8.4 Hz, 1H), 7.84–7.77 (m, 2H), 7.75–7.66 (m, 2H), 7.28–7.20 (m, 2H), 7.14 (d, <i>J</i> = 8.4 Hz, 2H), 2.48 (s, 6H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.3, 160.0, 153.0, 149.5, 143.5, 142.6, 132.8, 132.0, 131.3, 130.9, 128.6, 126.8, 122.1, 121.8, 16.5. MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 425.13; found, 425.14. Yield: 24%, purity >99%, <i>t</i><sub>R</sub> = 5.4 min.</div></div><div id="sec4_6_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> (<i>E</i>)-N-(5-Fluoropyridin-2-yl)-4-((4-hydroxy-3,5-dimethylphenyl)diazenyl)benzenesulfonamide (<b>16</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.47 (br s, 1H), 8.15 (d, <i>J</i> = 2.4 Hz, 1H), 8.02 (d, <i>J</i> = 7.8 Hz, 2H), 7.88 (d, <i>J</i> = 7.8 Hz, 2H), 7.64 (dt, <i>J</i> = 7.8, 2.4 Hz, 1H), 7.56 (s, 2H), 7.11 (dd, <i>J</i> = 7.8, 2.4 Hz, 1H), 2.23 (s, 6H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.2, 159.9, 158.3, 157.6, 150.9, 148.1, 144.0, 131.7, 129.4, 128.3, 127.1, 125.6, 117.1, 19.8. MS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 401.11; found, 401.08. Yield: 78%, purity >99%, <i>t</i><sub>R</sub> = 5.3 min.</div></div><div id="sec4_6_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> (<i>E</i>)-N-(4-Fluorophenyl)-4-((4-hydroxy-3,5-dimethylphenyl)diazenyl)benzenesulfonamide (<b>17</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 10.31 (br s, 1H), 9.40 (br s, 1H), 8.00–7.79 (m, 4H), 7.59 (s, 2H), 7.21–7.07 (m, 4H), 2.25 (s, 6H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>20</sub>H<sub>19</sub>FN<sub>3</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 400.11; found, 400.12. Yield: 76%, purity >99%, <i>t</i><sub>R</sub> = 6.4 min.</div></div><div id="sec4_6_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> (<i>E</i>)-4-((4-Hydroxy-3,5-dimethylphenyl)diazenyl)-2-methoxy-<i>N</i>-(pyridin-2-yl)benzenesulfon-amide (<b>18</b>)</h5><div class="NLM_p"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.31 (br s, 1H), 8.00 (d, <i>J</i> = 8.4 Hz, 1H), 7.96 (d, <i>J</i> = 4.8 Hz, 1H), 7.70 (t, <i>J</i> = 7.8 Hz, 1H), 7.58 (s, 2H), 7.47–7.41 (m, 2H), 7.18 (d, <i>J</i> = 8.4 Hz, 1H), 6.85 (t, <i>J</i> = 6.6 Hz, 1H), 3.8 (s, 3H), 2.24 (s, 6H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.6, 158.7, 156.6 (m), 148.0, 143.2, 140.9, 133.9, 131.4, 128.4, 127.1, 122.3, 117.5, 117.0, 114.9, 114.1, 31.5, 18.1. MS (<i>m</i>/<i>z</i>): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 413.13; found, 413.13. Yield: 31%, purity >99%, <i>t</i><sub>R</sub> = 4.8 min.</div><div class="NLM_p last">The following compounds were synthesized following Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>.</div></div><div id="sec4_6_1_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> (<i>E</i>)-4-((3,5-Dichloro-4-hydroxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>19</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.08 (d, <i>J</i> = 8.4 Hz, 1H), 8.02–7.94 (m, 3H), 7.93 (s, 2H), 7.74 (t, <i>J</i> = 7.8 Hz, 2H), 7.30 (d, <i>J</i> = 9.0 Hz, 1H), 6.89 (t, <i>J</i> = 6.6 Hz, 1H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.0, 156.5, 156.0, 146.7, 146.5, 142.9, 142.5, 131.3, 131.0, 126.8 (m), 125.9, 118.1, 115.7. MS (<i>m</i>/<i>z</i>): calcd for C<sub>17</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 423.01, 425.01; found, 423.01, 425.01. Yield: 75%, purity >95%, <i>t</i><sub>R</sub> = 5.9 min.</div></div><div id="sec4_6_1_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> (<i>E</i>)-4-((3-Bromo-4-hydroxy-5-methylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>20</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.12 (s, 1H), 8.13–7.98 (m, 3H), 7.97–7.87 (m, 3H), 7.83–7.72 (m, 2H), 7.35–7.10 (m, 1H), 6.95–6.75 (m, 1H), 2.34 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.9, 157.8, 156.6, 153.9, 148.6, 147.2, 144.5, 132.6, 131.0, 130.9, 130.3, 128.3, 125.8, 124.9, 124.5, 123.4, 117.7, 115.8, 20.1. MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>16</sub>BrN<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 447.01 and 449.01; found, 446.99, 448.98. Yield: 94%, purity >99%, <i>t</i><sub>R</sub> = 6.0 min.</div></div><div id="sec4_6_1_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> (<i>E</i>)-4-((3-Chloro-4-hydroxy-5-methylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>21</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.22 (s, 1H), 9.08 (s, 1H), 8.02 (d, <i>J</i> = 8.4 Hz, 2H), 8.00–7.94 (m, 1H), 7.91 (d, <i>J</i> = 8.4 Hz, 2H), 7.79–7.68 (m, 3H), 7.21 (d, <i>J</i> = 8.4 Hz, 1H), 6.85 (t, <i>J</i> = 6.6 Hz, 1H), 2.30 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.9, 156.6, 153.9, 148.1, 147.2, 144.4, 132.6, 131.0, 130.9, 130.3, 128.3, 125.8, 124.9, 124.5, 123.4, 117.7, 115.8, 20.1. MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>16</sub>ClN<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 403.06; found, 403.05. Purity >99%, <i>t</i><sub>R</sub> = 6.0 min.</div></div><div id="sec4_6_1_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> (<i>E</i>)-4-((3-Bromo-4-hydroxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>22</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.10 (s, 1H), 8.00–7.93 (m, 1H), 7.92 (d, <i>J</i> = 8.4 Hz, 2H), 7.82 (d, <i>J</i> = 9.0 Hz, 1H), 7.73 (t, <i>J</i> = 7.2 Hz, 2H), 7.29 (d, <i>J</i> = 8.4 Hz, 1H), 7.00 (d, <i>J</i> = 9.0 Hz, 1H), 6.88 (t, <i>J</i> = 6.0 Hz, 1H). <sup>13</sup>C NMR (150 MHz, MeOD-<i>d</i><sub>4</sub>) δ 158.8, 154.1, 153.9, 145.8, 144.2, 141.7, 128.3, 126.9, 126.3, 123.1, 117.0, 114.9 (br), 111.2. MS (<i>m</i>/<i>z</i>): calcd for C<sub>17</sub>H<sub>14</sub>BrN<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 432.99 and 434.99; found, 432.99 and 434.99. Yield: 80%, purity >95%, <i>t</i><sub>R</sub> = 5.9 min.</div></div><div id="sec4_6_1_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> (<i>E</i>)-4-((3-Chloro-4-hydroxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>23</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.34 (s, 1H), 8.09 (d, <i>J</i> = 8.4 Hz, 1H), 8.03 (d, <i>J</i> = 7.8 Hz, 2H), 7.95–7.89 (m, 3H), 7.87–7.81 (m, 2H), 7.77 (m, 1H), 7.24 (d, <i>J</i> = 9.0 Hz, 1H), 7.20 (d, <i>J</i> = 9.0 Hz, 1H), 6.86 (s, 1H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>17</sub>H<sub>14</sub>ClN<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 389.05; found, 389.05. Yield: 100%, purity >99%, <i>t</i><sub>R</sub> = 5.9 min.</div></div><div id="sec4_6_1_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> (<i>E</i>)-4-((4-Hydroxy-3-methylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>24</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.05 (d, <i>J</i> = 8.4 Hz, 2H), 7.95 (d, <i>J</i> = 5.3 Hz, 1H), 7.89 (d, <i>J</i> = 8.4 Hz, 2H), 7.77–7.70 (m, 2H), 7.68 (d, <i>J</i> = 8.4 Hz, 1H), 7.29 (d, <i>J</i> = 8.7 Hz, 1H), 6.94–6.87 (m, 2H), 2.26 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.0, 156.6, 155.9, 147.9, 144.9, 144.0, 143.0, 129.9, 127.5, 127.3, 125.7, 124.3, 117.4, 116.8, 116.7, 17.3. IR (neat): ν 3326 (−OH), 1641 (−C═N−), 1593 (−N═N−), 1396 (−SO<sub>2</sub>−). MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>17</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 369.10; found, 369.11. Purity >99%, <i>t</i><sub>R</sub> = 5.7 min. mp 248.1 °C.</div></div><div id="sec4_6_1_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> (<i>E</i>)-4-((3,5-Dibromo-4-hydroxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>25</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.12 (s, 1H), 8.08 (d, <i>J</i> = 8.4 Hz, 2H), 8.00–7.92 (m, 4H), 7.74 (t, <i>J</i> = 7.8 Hz, 2H), 7.30 (d, <i>J</i> = 9.0 Hz, 2H), 6.85 (t, <i>J</i> = 6.6 Hz, 1H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.6, 157.8, 157.2, 146.1, 143.0, 141.3, 140.8, 139.8, 131.3, 130.7, 130.4, 126.2, 124.0, 117.9, 115.5. MS (<i>m</i>/<i>z</i>): calcd for C<sub>17</sub>H<sub>13</sub>Br<sub>2</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 510.90, 512.90, 514.90; found, 510.90, 512.90, 514.90. Yield: 47%, purity >99%, <i>t</i><sub>R</sub> = 6.0 min.</div></div><div id="sec4_6_1_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (<i>E</i>)-4-((4-Hydroxy-3-(trifluoromethyl)phenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>26</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.17–8.13 (m, 1H), 8.08 (d, <i>J</i> = 9.0 Hz, 2H), 8.05 (d, <i>J</i> = 8.4 Hz, 1H), 8.03–7.95 (m, 3H), 7.74 (t, <i>J</i> = 7.8 Hz, 1H), 7.30 (d, <i>J</i> = 8.4 Hz, 2H), 7.09(d, <i>J</i> = 9.0 Hz, 1H), 6.88 (t, <i>J</i> = 6.6 Hz, 1H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>14</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 423.07; found, 423.08. Yield: 39%, purity >99%, <i>t</i><sub>R</sub> = 5.8 min.</div></div><div id="sec4_6_1_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (<i>E</i>)-4-((2-Amino-4-hydroxy-5-methylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>27</b>, MS436)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.28 (br s, 1H), 7.98 (m, 1H), 7.90 (d, <i>J</i> = 8.4 Hz, 2H), 7.82 (d, <i>J</i> = 8.4 Hz, 2H), 7.71 (t, <i>J</i> = 7.2 Hz, 2H), 7.37 (s, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 1H), 7.11 (br s, 2H), 6.85 (t, <i>J</i> = 6.0 Hz, 1H), 6.22 (s, 1H), 2.00 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.0, 155.4, 153.7, 147.0, 141.0, 138.2, 133.2, 131.3, 128.7, 128.2, 121.9, 115.4, 114.4, 103.4, 100.4, 15.7. MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>18</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 384.11; found, 384.12. Yield: 87%, purity >99%, <i>t</i><sub>R</sub> = 5.6 min.</div></div><div id="sec4_6_1_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (<i>E</i>)-4-((2-Amino-4-hydroxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>28</b>, MS267)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.01 (d, <i>J</i> = 4.8 Hz, 1H), 7.85 (d, <i>J</i> = 8.4 Hz, 2H), 7.76 (d, <i>J</i> = 8.4 Hz, 2H), 7.70 (t, <i>J</i> = 7.8 Hz, 1H), 7.46 (s, 1H), 7.15 (d, <i>J</i> = 7.8 Hz, 1H), 6.86 (t, <i>J</i> = 6.0 Hz, 1H), 3.90–3.70 (m, 4H), 2.00 (s, 3H), 1.89 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.2, 154.9 (br), 153.0 (br), 146.8, 143.5, 140.1 (br), 134.7, 131.2, 122.9, 120.7, 119.0, 116.8, 116.4, 109.9, 19.9, 12.7. MS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>20</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 398.13; found, 398.12. Yield: 89%, purity >99%, <i>t</i><sub>R</sub> = 5.4 min.</div></div><div id="sec4_6_1_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (<i>E</i>)-4-((2-Amino-3-chloro-4-hydroxy-5-methylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>29</b>, MS363)</h5><div class="NLM_p last">A 50 mL round-bottomed flask was charged with sulfapyridine (100.0 mg, 0.40 mmol, 1.0 equiv) and concentrated HCl (87.5 mg, 160 μL, 2.40 mmol, 5.98 equiv). The mixture was dissolved in a MeOH/ACN mixture (3 mL/3 mL). The solution was cooled to 0 °C and stirred for 15 min. Isoamyl nitrite (47.0 mg, 54 μL, 0.40 mmol, 1.0 equiv) was added dropwise under argon over 10 min. The solution was stirred at 0 °C for 45 min. Meanwhile, to another 50 mL round-bottomed flask were added 3-amino-2-chloro-6-cresol (63.0 mg, 0.40 mmol, 1.0 equiv) and potassium carbonate (276.3 mg, 2.0 mmol, 5.0 equiv). To this mixture were added methanol (1.0 mL) and DI H<sub>2</sub>O (8.0 mL). The solution was deoxygenated for 15 min by argon. The resultant solution was cooled to 0 °C. The previously prepared amber color diazonium ion was added dropwise under argon over 15 min. At the end of the addition, the pH of the solution was maintained between 8 and 10. The solution was allowed to stir at 0 °C for 1 h and then quenched with 1 N HCl to reach pH 1. The product was filtered and dried under vacuum. The pure product appeared as a fine red powder (167.0 mg, 99%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.51 (s, 1H), 8.04 (s, 1H), 7.97–7.78 (m, 3H), 7.78–7.62 (m, 3H), 7.53 (s, 1H), 7.15 (s, 1H), 6.89 (s, 1H), 6.73 (br s, 2H), 1.98 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.1, 156.1, 155.4, 146.8, 146.3, 143.3, 142.0, 133.5, 132.0, 131.3, 130.6, 119.0, 116.7, 115.9, 115.7, 115.4, 107.4, 19.7. MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>17</sub>ClN<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 418.07; found, 418.08. Purity >99%, <i>t</i><sub>R</sub> = 5.5 min.</div></div><div id="sec4_6_1_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (<i>E</i>)-4-((2-Amino-4-hydroxy-5-methoxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>30</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.00 (d, <i>J</i> = 4.8 Hz, 1H), 7.88 (d, <i>J</i> = 8.4 Hz, 2H), 7.66 (dt, <i>J</i> = 7.2 Hz, 1.2, 2H), 7.55 (d, <i>J</i> = 8.4 Hz, 2H), 7.20–7.15 (m, 1H), 6.93 (t, <i>J</i> = 6.6 Hz, 1H), 6.89–6.84 (m, 2H), 6.76 (s, 1H), 3.84 (s, 3H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>18</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 400.11; found, 400.11. Yield: 15%, purity >99%, <i>t</i><sub>R</sub> = 5.0 min.</div></div><div id="sec4_6_1_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> <i>(E)-</i>4-((5-Hydroxy-4,6-dimethylpyridin-2-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>31</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>): δ 8.11 (d, <i>J</i> = 8.5 Hz, 2H), 8.04 (d, <i>J</i> = 8.4 Hz, 2H), 7.94 (d, <i>J</i> = 5.3 Hz, 1H), 7.78–7.74 (m, 2H), 7.32 (d, <i>J</i> = 8.8 Hz, 1H), 6.89 (t, <i>J</i> = 6.4 Hz, 1H), 2.59 (s, 3H), 2.39 (s, 3H). HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>18</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 384.113; found, 384.112. Purity >96%, <i>t</i><sub>R</sub> = 4.3 min.</div></div><div id="sec4_6_1_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> <i>(E)-</i>4-((5-Hydroxy-3,4,6-trimethylpyridin-2-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>32</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>): δ 8.11 (d, <i>J</i> = 8.2 Hz, 2H), 8.01 (d, <i>J</i> = 4.9 Hz, 2H), 7.94 (d, <i>J</i> = 5.3 Hz, 1H), 7.75 (t, <i>J</i> = 7.9 Hz, 1H), 7.31 (d, <i>J</i> = 8.8 Hz, 1H), 6.89 (t, <i>J</i> = 6.4 Hz, 1H), 2.70 (s, 3H), 2.62 (s, 3H), 2.38 (s, 3H). HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>20</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 398.128; found, 398.127. Purity >90%, <i>t</i><sub>R</sub> = 4.7 min.</div></div><div id="sec4_6_1_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> (<i>E</i>)-4-((3,4-Dihydroxy-5-methylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>33</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.25 (s, 1H), 9.95 (s, 1H), 8.70 (s, 1H), 8.20–7.95 (m, 3H), 7.90 (d, <i>J</i> = 8.4 Hz, 2H), 7.82–7.70 (m, 2H), 7.30–7.13 (m, 2H), 6.98–6.80 (m, 1H), 2.22 (s, 3H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>17</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 385.10; found, 389.05. Yield: 79%, purity >99%, <i>t</i><sub>R</sub> = 5.5 min.</div></div><div id="sec4_6_1_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> (<i>E</i>)-4-((3,4-Dihydroxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>34</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.11–7.96 (m, 3H), 7.87 (d, <i>J</i> = 7.8 Hz, 2H), 7.77–7.70 (m, 2H), 7.39 (d, <i>J</i> = 8.4 Hz, 1H), 7.34 (s, 1H), 7.27–7.10 (m, 2H), 6.93 (d, <i>J</i> = 8.4 Hz, 1H), 6.91–6.80 (m, 2H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>17</sub>H<sub>15</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 371.08; found, 371.10. Yield: 65%, purity >99%, <i>t</i><sub>R</sub> = 5.4 min.</div></div><div id="sec4_6_1_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> (<i>E</i>)-4-((4-Hydroxy-3-methylphenyl)diazenyl)-<i>N</i>-(4-(trifluoromethyl)benzyl)benzenesulfonamide (<b>35</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 7.94 (d, <i>J</i> = 8.4 Hz, 2H), 7.91 (d, <i>J</i> = 8.4 Hz, 2H), 7.76 (d, <i>J</i> = 1.9 Hz, 1H), 7.70 (dd, <i>J</i> = 7.8, 1.9 Hz, 1H), 7.54 (d, <i>J</i> = 8.4 Hz, 2H), 7.44 (d, <i>J</i> = 8.4 Hz, 2H), 6.90 (d, <i>J</i> = 7.8 Hz, 1H), 4.21 (s, 2H), 2.28 (s, 3H). <sup>13</sup>C NMR (150 MHz, MeOD-<i>d</i><sub>4</sub>) δ 162.7, 157.7, 148.7, 144.6, 144.0, 132.1, 131.9, 130.8, 130.5, 128.2, 128.0, 127.6, 126.3, 125.1, 117.2, 48.7, 17.6. IR (neat): ν 3442 (−OH), 3268 (−NH−), 1615 (−C═N−), 1591 (−N═N−), 1377 (−SO<sub>2</sub>−). MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>19</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 450.11; found, 450.11. Purity >99%, <i>t</i><sub>R</sub> = 6.5 min. mp 181.8 °C.</div></div><div id="sec4_6_1_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> (<i>E</i>)-4-((2-Amino-4-hydroxy-5-methylphenyl)diazenyl)-<i>N</i>-(4-(trifluoromethyl)benzyl)benzenesulfonamide (<b>36</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.22 (s, 1H), 8.30 (t, <i>J</i> = 3.9 Hz, 1H), 7.90–7.80 (m, 4H), 7.64 (d, <i>J</i> = 8.4 Hz, 2H), 7.49 (d, <i>J</i> = 8.4 Hz, 2H), 7.41 (s, 1H), 7.19 (br s, 2H), 6.23 (s, 1H), 4.17 (d, <i>J</i> = 3.9 Hz, 2H), 2.02 (s, 3H). IR (neat): ν 3479 (−OH), 3233 (−NH−), 1631 (−C═N−), 1596 (−N═N−), 1389 (−SO<sub>2</sub>−). MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>20</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 465.12; found, 465.12. Purity >99%, <i>t</i><sub>R</sub> = 6.3 min. mp 171.7 °C.</div></div><div id="sec4_6_1_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> (<i>E</i>)-4-((4-Methoxy-3,5-dimethylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>37</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.10–7.96 (m, 2H), 7.92–7.85 (m, 2H), 7.80–7.68 (m, 1H), 7.16–7.02 (m, 2H), 6.80–6.68 (m, 1H), 4.11 (s, 3H), 2.13 (s, 6H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 397.13; found, 397.15. Yield: 3%, purity >99%, <i>t</i><sub>R</sub> = 6.2 min.</div></div><div id="sec4_6_1_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> (<i>E</i>)-N-(2,6-Dimethyl-4-((4-(<i>N</i>-(pyridin-2-yl)sulfamoyl)phenyl)diazenyl)phenyl)acetamide (<b>38</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.23 (s, 1H), 7.85–7.67 (m, 2H), 7.67–7.30 (m, 3H), 7.20–6.90 (m, 1H), 6.85–6.60 (m, 1H), 2.12 (s, 6H), 2.03 (s, 3H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>22</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 446.14; found, 446.13. Yield: 5%, purity >99%, <i>t</i><sub>R</sub> = 5.0 min.</div></div><div id="sec4_6_1_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> (<i>E</i>)-4-((4-Hydroxy-3-isopropylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>39</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.46 (s, 1H), 8.08–7.99 (m, 3H), 7.91 (d, <i>J</i> = 7.8 Hz, 2H), 7.76 (m, 2H), 7.67 (d, <i>J</i> = 8.4 Hz, 2H), 7.22 (m, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 6.86 (m, 1H), 3.25 (m, 1H), 1.22 (d, <i>J</i> = 7.2 Hz, 6H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 397.13; found, 397.13. Yield: 24%, purity >99%, <i>t</i><sub>R</sub> = 6.0 min.</div></div><div id="sec4_6_1_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> (<i>E</i>)-4-((3-(<i>tert</i>-Butyl)-4-hydroxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>40</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.51 (s, 1H), 8.20–7.97 (m, 2H), 7.90 (d, <i>J</i> = 8.4 Hz, 2H), 7.84–7.65 (m, 4H), 7.40–7.10 (m, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 1.40 (s, 9H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>23</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 411.15; found, 411.14. Yield: 20%, purity >99%, <i>t</i><sub>R</sub> = 6.2 min.</div></div><div id="sec4_6_1_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> (<i>E</i>)-4-((4-Hydroxy-3,5-diisopropylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>41</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.07 (d, <i>J</i> = 8.4 Hz, 2H), 8.01–7.96 (m, 1H), 7.93 (d, <i>J</i> = 8.4 Hz, 2H), 7.74 (t, <i>J</i> = 7.8 Hz, 2H), 7.70 (s, 2H), 7.30 (d, <i>J</i> = 8.4 Hz, 2H), 6.89 (t, <i>J</i> = 6.6 Hz, 1H), 3.45–3.60 (m, 2H), 1.28 (d, <i>J</i> = 7.2 Hz, 12H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>23</sub>H<sub>27</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 439.18; found, 439.22. Yield: 16%, purity >99%, <i>t</i><sub>R</sub> = 6.3 min.</div></div><div id="sec4_6_1_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> (<i>E</i>)-4-((3,5-di-<i>tert</i>-Butyl-4-hydroxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>42</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>) δ 8.01 (d, <i>J</i> = 6.6 Hz, 2H), 7.95 (d, <i>J</i> = 8.4 Hz, 1H), 7.73 (s, 2H), 7.57 (t, <i>J</i> = 7.2 Hz, 1H), 7.52 (d, <i>J</i> = 8.4 Hz, 2H), 7.29 (d, <i>J</i> = 9.0 Hz, 2H), 6.89 (t, <i>J</i> = 6.6 Hz, 1H), 1.37 (s, 9H), 1.31 (s, 9H). <sup>13</sup>C NMR (150 MHz, MeOD-<i>d</i><sub>4</sub>) δ 156.6, 156.4, 153.6, 144.6, 144.5, 143.6, 136.6, 135.0, 131.3, 129.3, 126.8 (m), 125.9, 118.0, 117.5, 116.5, 37.1, 31.5, 31.3. MS (<i>m</i>/<i>z</i>): calcd for C<sub>25</sub>H<sub>31</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 467.21; found, 467.21. Yield: 2%, purity >99%, <i>t</i><sub>R</sub> = 6.8 min.</div></div><div id="sec4_6_1_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> (<i>E</i>)-4-((4-Hydroxynaphthalen-1-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>43</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.13 (s, 1H), 8.96–8.88 (m, 1H), 8.55–8.43 (m, 1H), 8.37–8.20 (m, 1H), 8.16–8.00 (m, 2H), 7.99–7.84 (m, 1H), 7.83–7.70 (m, 2H), 7.69–7.54 (m, 1H), 7.53–7.40 (m, 1H), 7.40–7.28 (m, 1H), 7.27–6.99 (m, 1H), 6.97–6.69 (m, 1H), 5.75 (s, 1H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>17</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 405.10; found, 405.12. Yield: 67%, purity >99%, <i>t</i><sub>R</sub> = 5.0 min.</div></div><div id="sec4_6_1_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> <i>(E)-</i>4-((4-Hydroxy-5,6,7,8-tetrahydronaphthalen-1-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>44</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>): δ 8.04 (d, <i>J</i> = 8.6 Hz, 2H), 7.95 (d, <i>J</i> = 5.1 Hz, 1H), 7.87 (d, <i>J</i> = 8.5 Hz, 2H), 7.73 (td, <i>J</i> = 8.0, 1.9 Hz, 1H), 7.49 (d, <i>J</i> = 8.8 Hz, 1H), 7.28 (d, <i>J</i> = 8.8 Hz, 1H), 6.88 (t, <i>J</i> = 6.4 Hz, 1H), 6.65 (d, <i>J</i> = 8.8 Hz, 1H), 3.31–3.25 (m, 2H), 2.99 (s, 1H), 2.86 (s, 1H), 2.73–2.67 (m, 2H), 1.89–1.83 (m, 4H). HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 409.133; found, 409.129. Purity >99%, <i>t</i><sub>R</sub> = 5.1 min.</div></div><div id="sec4_6_1_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> <i>(E)-</i>4-((5-Hydroxy-1,2,3,4-tetrahydroquinolin-8-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>45</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.03–7.95 (m, 1H), 7.87 (d, <i>J</i> = 8.5 Hz, 2H), 7.71 (td, <i>J</i> = 8.1, 1.9 Hz, 1H), 7.61 (d, <i>J</i> = 8.1 Hz, 2H), 7.15 (d, <i>J</i> = 7.9 Hz, 1H), 7.04 (d, <i>J</i> = 8.9 Hz, 1H), 6.86 (t, <i>J</i> = 6.4 Hz, 1H), 6.30 (d, <i>J</i> = 8.9 Hz, 1H), 3.32–3.24 (m, 2H), 3.16–3.14 (m, 2H), 1.76 (qt, <i>J</i> = 6.6 Hz, 2H). HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>20</sub>H<sub>20</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 410.128; found, 410.126. Purity >96%, <i>t</i><sub>R</sub> = 4.7 min.</div></div><div id="sec4_6_1_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> <i>(E)-</i>4-((7-Hydroxy-2,2-dimethyl-2,3-dihydrobenzofuran-4-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>46</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>): δ 8.05 (d, <i>J</i> = 8.5 Hz, 2H), 7.95 (d, <i>J</i> = 4.8 Hz, 1H), 7.88 (d, <i>J</i> = 8.5 Hz, 2H), 7.74 (td, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.37 (d, <i>J</i> = 8.6 Hz, 1H), 7.28 (d, <i>J</i> = 8.8 Hz, 1H), 6.89 (t, <i>J</i> = 6.4 Hz, 1H), 6.80 (d, <i>J</i> = 8.6 Hz, 1H), 3.42 (s, 2H), 1.52 (s, 6H). HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>21</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 425.128; found, 425.124. Purity >99%, <i>t</i><sub>R</sub> = 4.8 min.</div></div><div id="sec4_6_1_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> <i>(E)-</i>4-((5-Hydroxy-3-(hydroxymethyl)-4,6-dimethylpyridin-2-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>47</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>): δ 8.10 (d, <i>J</i> = 8.5 Hz, 2H), 8.03 (d, <i>J</i> = 8.4 Hz, 2H), 7.95 (d, <i>J</i> = 5.1 Hz, 1H), 7.76 (ddd, <i>J</i> = 8.9, 7.2, 1.8 Hz, 1H), 7.31 (d, <i>J</i> = 8.9 Hz, 1H), 6.89 (t, <i>J</i> = 6.5 Hz, 1H), 5.18 (d, <i>J</i> = 0.8 Hz, 2H), 2.57 (s, 3H), 2.46 (s, 3H). HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>20</sub>N<sub>5</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 414.123; found, 414.121. Purity >95%, <i>t</i><sub>R</sub> = 4.0 min.</div></div><div id="sec4_6_1_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> <i>(E)-</i>4-((4-(Aminomethyl)-5-hydroxy-3-(hydroxymethyl)-6-methylpyridin-2-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>48</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, MeOD-<i>d</i><sub>4</sub>): δ 8.04 (d, <i>J</i> = 7.1 Hz, 2H), 7.96 (d, <i>J</i> = 5.0 Hz, 1H), 7.92–7.79 (m, 2H), 7.74 (td, <i>J</i> = 8.0, 1.9 Hz, 1H), 7.28 (d, <i>J</i> = 8.7 Hz, 1H), 6.89 (t, <i>J</i> = 6.4 Hz, 1H), 5.17 (s, 2H), 4.28 (s, 2H), 2.51 (s, 3H). HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>21</sub>N<sub>6</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 429.135; found, 429.136. Purity >95%, <i>t</i><sub>R</sub> = 2.2 min.</div></div><div id="sec4_6_1_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> <i>(E)-</i>4-((4-Formyl-5-hydroxy-3-(hydroxymethyl)-6-methylpyridin-2-yl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>49</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.99 (d, <i>J</i> = 4.4 Hz, 1H), 7.85 (d, <i>J</i> = 8.6 Hz, 2H), 7.64 (dd, <i>J</i> = 17.8, 8.0 Hz, 3H), 7.08 (d, <i>J</i> = 8.5 Hz, 1H), 6.79 (t, <i>J</i> = 5.8 Hz, 1H), 6.11 (s, 2H), 5.14 (d, <i>J</i> = 15.1 Hz, 1H), 4.95 (d, <i>J</i> = 15.2 Hz, 1H), 2.23 (s, 3H). HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>19</sub>H<sub>18</sub>N<sub>5</sub>O<sub>5</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 428.103; found, 428.105. Purity >96%, <i>t</i><sub>R</sub> = 4.0 min.</div></div><div id="sec4_6_1_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> (<i>E</i>)-4-((4-Hydroxy-3-methoxyphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>50</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.19 (s, 1H), 8.09–7.95 (m, 3H), 7.92 (d, <i>J</i> = 9.0 Hz, 2H), 7.77 (t, <i>J</i> = 6.0 Hz, 1H), 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.46 (s, 1H), 7.24 (d, <i>J</i> = 9.0 Hz, 1H), 7.22 (m, 1H), 7.00 (d, <i>J</i> = 8.4 Hz, 1H), 6.87 (s, 1H), 3.83 (s, 3H). MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>17</sub>N<sub>4</sub>O<sub>4</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 385.10; found, 385.05. Yield: 60%, purity >99%, <i>t</i><sub>R</sub> = 5.9 min.</div></div><div id="sec4_6_1_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> (<i>E</i>)-3-((2-Amino-4-hydroxy-5-methylphenyl)diazenyl)-<i>N</i>-(pyridin-2-yl)benzenesulfonamide (<b>51</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.15 (s, 1H), 8.04–7.92 (m, 2H), 7.76 (d, <i>J</i> = 7.8 Hz, 1H), 7.73 (t, <i>J</i> = 7.8 Hz, 1H), 7.59 (t, <i>J</i> = 7.8 Hz, 1H), 7.44 (s, 1H), 7.20 (d, <i>J</i> = 8.4 Hz, 1H), 6.84 (t, <i>J</i> = 6.0 Hz, 1H), 6.29 (s, 1H), 2.02 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.6, 156.6, 154.9, 149.9, 146.5, 144.2, 140.1, 133.8, 133.1, 131.1, 128.6, 127.8, 121.9, 119.9, 118.3, 117.5, 103.5, 18.5. MS (<i>m</i>/<i>z</i>): calcd for C<sub>18</sub>H<sub>18</sub>N<sub>5</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>, 384.11; found, 384.14. Yield: 86%, purity >99%, <i>t</i><sub>R</sub> = 4.7 min.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77"><a href="/doi/suppl/10.1021/jm401334s">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43884" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43884" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Additional diazobenzene inhibitors and crystallography data and refinement statistics for BRD4–BrD1/MS436, BRD4–BrD1/MS435, and BRD4–BrD1/MS267. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm401334s/suppl_file/jm401334s_si_001.pdf">jm401334s_si_001.pdf (312.87 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm401334s" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15715" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15715" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ming-Ming Zhou</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bbd6d2d5dc96d6d2d5dc95c1d3d4cefbd6c8c8d695dedfce"><span class="__cf_email__" data-cfemail="cba6a2a5ace6a6a2a5ace5b1a3a4be8ba6b8b8a6e5aeafbe">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guangtao Zhang</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander N. Plotnikov</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Rusinova</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tong Shen</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Keita Morohashi</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer Joshua</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei Zeng</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shiraz Mujtaba</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Ohlmeyer</span> - <span class="hlFld-Affiliation affiliation">Department of Structural
and Chemical Biology, Icahn School of Medicine
at Mount Sinai, 1425
Madison Avenue, New York, New York 10029, United
States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>G.T.Z. and M.-M. Z. conceived and designed the experiments for this project. G.T.Z. and M.O. designed the synthetic schemes, and G.T.Z performed the chemical synthesis. J.J. prepared the protein samples and carried out crystallization. A.N.P. solved the crystal structures. E.R., K.M., and L.Z. performed the biochemical binding study. T.S. and S.M. performed the cell biology study. G.T.Z. and M.-M.Z. wrote the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d1715e4822-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81251" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81251" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge the use of the NMR facility at the New York Structural Biology Center and thank the staff at the X6A beamline of the National Synchrotron Light Sources at the Brookhaven National Laboratory for facilitating the X-ray data collection. This work was supported in part by research grants from the National Institutes of Health (to M.-M.Z.).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i80" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i80"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i81" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i81"> Abbreviations Used</h2><tr><td class="NLM_term">BrD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">NOS</td><td class="NLM_def"><p class="first last">nitric oxide synthase</p></td></tr><tr><td class="NLM_term">iNOS</td><td class="NLM_def"><p class="first last">inducible nitric oxide synthase</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i82">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27153" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27153" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 32 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0ljqdUIqunkXwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Dhalluin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aggarwal, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">399</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.+M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0ljqdUIqunkXwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Sanchez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">The role of human bromodomains in chromatin biology and gene transcription</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">659</span><span class="NLM_x">–</span> <span class="NLM_lpage">665</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2009&pages=659-665&author=R.+Sanchezauthor=M.+M.+Zhou&title=The+role+of+human+bromodomains+in+chromatin+biology+and+gene+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DThe%2520role%2520of%2520human%2520bromodomains%2520in%2520chromatin%2520biology%2520and%2520gene%2520transcription%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2009%26volume%3D12%26spage%3D659%26epage%3D665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkmer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0ljqdUIqunkXwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7346</span><span class="NLM_x">–</span> <span class="NLM_lpage">7359</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lh_Bui7JF1yXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+C.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lh_Bui7JF1yXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lgr1Jy3oHut7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappaport, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sison, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magoon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blatt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wunderlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johns, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chicas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulloy, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valent, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">524</span><span class="NLM_x">–</span> <span class="NLM_lpage">528</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+W.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lgr1Jy3oHut7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%2BW.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Belkina, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lgr1Jy3oHut7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plotnikov, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusinova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerona-Nevarro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moshkina, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshua, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohlmeyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">Down-regulation of NF-kappaB transcriptional activity in HIV-associated kidney disease by BRD4 inhibition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">28840</span><span class="NLM_x">–</span> <span class="NLM_lpage">28851</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=28840-28851&author=G.+Zhangauthor=R.+Liuauthor=Y.+Zhongauthor=A.+N.+Plotnikovauthor=W.+Zhangauthor=L.+Zengauthor=E.+Rusinovaauthor=G.+Gerona-Nevarroauthor=N.+Moshkinaauthor=J.+Joshuaauthor=P.+Y.+Chuangauthor=M.+Ohlmeyerauthor=J.+C.+Heauthor=M.+M.+Zhou&title=Down-regulation+of+NF-kappaB+transcriptional+activity+in+HIV-associated+kidney+disease+by+BRD4+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DGerona-Nevarro%26aufirst%3DG.%26aulast%3DMoshkina%26aufirst%3DN.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DChuang%26aufirst%3DP.%2BY.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DJ.%2BC.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DDown-regulation%2520of%2520NF-kappaB%2520transcriptional%2520activity%2520in%2520HIV-associated%2520kidney%2520disease%2520by%2520BRD4%2520inhibition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D28840%26epage%3D28851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Tang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deguzman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apparsundaram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuentes, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demartino, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyagi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrido, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hogaboam, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denton, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budd, D. C.</span><span> </span><span class="NLM_article-title">Assessment of Brd4 inhibition in idiopathic pulmonary fibrosis lung fibroblasts and in vivo models of lung fibrosis</span> <span class="citation_source-journal">Am. J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">183</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">479</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.ajpath.2013.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFynur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2013&pages=470-479&author=X.+Tangauthor=R.+Pengauthor=J.+E.+Phillipsauthor=J.+Deguzmanauthor=Y.+Renauthor=S.+Apparsundaramauthor=Q.+Luoauthor=C.+M.+Bauerauthor=M.+E.+Fuentesauthor=J.+A.+Demartinoauthor=G.+Tyagiauthor=R.+Garridoauthor=C.+M.+Hogaboamauthor=C.+P.+Dentonauthor=A.+M.+Holmesauthor=C.+Kitsonauthor=C.+S.+Stevensonauthor=D.+C.+Budd&title=Assessment+of+Brd4+inhibition+in+idiopathic+pulmonary+fibrosis+lung+fibroblasts+and+in+vivo+models+of+lung+fibrosis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Brd4 Inhibition in Idiopathic Pulmonary Fibrosis Lung Fibroblasts and in Vivo Models of Lung Fibrosis</span></div><div class="casAuthors">Tang, Xiaoyan; Peng, Ruoqi; Phillips, Jonathan E.; Deguzman, Jeremy; Ren, Yonglin; Apparsundaram, Subramanium; Luo, Qi; Bauer, Carla M.; Fuentes, Maria E.; DeMartino, Julie A.; Tyagi, Gaurav; Garrido, Rosario; Hogaboam, Cory M.; Denton, Christopher P.; Holmes, Alan M.; Kitson, Christopher; Stevenson, Christopher S.; Budd, David C.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">470-479</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Idiopathic pulmonary fibrosis (IPF) is a chronic lung disease of high unmet medical need.  Although bromodomain (Brd) and extra terminal domain isoforms have recently been implicated in mediating inflammatory and oncol. indications, their roles in lung fibrosis have not been comprehensively assessed.  We investigated the role of Brd on the profibrotic responses of lung fibroblasts (LFs) in patients with rapidly progressing IPF and a mouse bleomycin model of lung fibrosis.  The enhanced migration, proliferation, and IL-6 release obsd. in LFs from patients with rapidly progressing IPF are attenuated by pharmacol. inhibition of Brd4.  These changes are accompanied by enhanced histone H4 lysine5 acetylation and assocn. of Brd4 with genes involved in the profibrotic responses in IPF LFs as demonstrated using chromatin immunopptn. and quant. PCR.  Oral administration of 200 mg/kg per day Brd4 inhibitor JQ1 in a therapeutic dosing regimen substantially attenuated lung fibrosis induced by bleomycin in C57BL/6 mice.  In conclusion, this study shows that the Brd4 inhibitor JQ1, administered in a therapeutic dosage, is capable of inhibiting the profibrotic effects of IPF LFs and attenuates bleomycin-induced lung fibrosis in mice.  These results suggest that Brd4 inhibitors may represent a novel therapy for the treatment of rapidly progressing IPF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeyA6Tfi6PObVg90H21EOLACvtfcHk0lg_0xBOSf4ndw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFynur7I&md5=8a7b094630ce44df6cd1b2b9de6cc6ec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2013.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2013.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DJ.%2BE.%26aulast%3DDeguzman%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DApparsundaram%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DBauer%26aufirst%3DC.%2BM.%26aulast%3DFuentes%26aufirst%3DM.%2BE.%26aulast%3DDemartino%26aufirst%3DJ.%2BA.%26aulast%3DTyagi%26aufirst%3DG.%26aulast%3DGarrido%26aufirst%3DR.%26aulast%3DHogaboam%26aufirst%3DC.%2BM.%26aulast%3DDenton%26aufirst%3DC.%2BP.%26aulast%3DHolmes%26aufirst%3DA.%2BM.%26aulast%3DKitson%26aufirst%3DC.%26aulast%3DStevenson%26aufirst%3DC.%2BS.%26aulast%3DBudd%26aufirst%3DD.%2BC.%26atitle%3DAssessment%2520of%2520Brd4%2520inhibition%2520in%2520idiopathic%2520pulmonary%2520fibrosis%2520lung%2520fibroblasts%2520and%2520in%2520vivo%2520models%2520of%2520lung%2520fibrosis%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2013%26volume%3D183%26spage%3D470%26epage%3D479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. F.</span><span> </span><span class="NLM_article-title">Brd4 coactivates transcriptional activation of NF-kappaB via specific binding to acetylated RelA</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1375</span><span class="NLM_x">–</span> <span class="NLM_lpage">1387</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&author=B.+Huangauthor=X.+D.+Yangauthor=M.+M.+Zhouauthor=K.+Ozatoauthor=L.+F.+Chen&title=Brd4+coactivates+transcriptional+activation+of+NF-kappaB+via+specific+binding+to+acetylated+RelA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0lg_0xBOSf4ndw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.%2BD.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBrd4%2520coactivates%2520transcriptional%2520activation%2520of%2520NF-kappaB%2520via%2520specific%2520binding%2520to%2520acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D1375%26epage%3D1387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Schroder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaehlcke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bisgrove, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnolzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, M.</span><span> </span><span class="NLM_article-title">Two-pronged binding with bromodomain-containing protein 4 liberates positive transcription elongation factor b from inactive ribonucleoprotein complexes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">1090</span><span class="NLM_x">–</span> <span class="NLM_lpage">1099</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=1090-1099&author=S.+Schroderauthor=S.+Choauthor=L.+Zengauthor=Q.+Zhangauthor=K.+Kaehlckeauthor=L.+Makauthor=J.+Lauauthor=D.+Bisgroveauthor=M.+Schnolzerauthor=E.+Verdinauthor=M.+M.+Zhouauthor=M.+Ott&title=Two-pronged+binding+with+bromodomain-containing+protein+4+liberates+positive+transcription+elongation+factor+b+from+inactive+ribonucleoprotein+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchroder%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DBisgrove%26aufirst%3DD.%26aulast%3DSchnolzer%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DTwo-pronged%2520binding%2520with%2520bromodomain-containing%2520protein%25204%2520liberates%2520positive%2520transcription%2520elongation%2520factor%2520b%2520from%2520inactive%2520ribonucleoprotein%2520complexes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D1090%26epage%3D1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olcina, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritchie, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammond, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">Optimization of 3,5-dimethylisoxazole derivatives as potent bromodomain ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">3217</span><span class="NLM_x">–</span> <span class="NLM_lpage">3227</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm301588r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3217-3227&author=D.+S.+Hewingsauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=S.+Martinauthor=S.+Picaudauthor=A.+Tumberauthor=C.+Wellsauthor=M.+M.+Olcinaauthor=K.+Freemanauthor=A.+Gillauthor=A.+J.+Ritchieauthor=D.+W.+Sheppardauthor=A.+J.+Russellauthor=E.+M.+Hammondauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=Optimization+of+3%2C5-dimethylisoxazole+derivatives+as+potent+bromodomain+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm301588r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301588r%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DOlcina%26aufirst%3DM.%2BM.%26aulast%3DFreeman%26aufirst%3DK.%26aulast%3DGill%26aufirst%3DA.%26aulast%3DRitchie%26aufirst%3DA.%2BJ.%26aulast%3DSheppard%26aufirst%3DD.%2BW.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DHammond%26aufirst%3DE.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DOptimization%2520of%25203%252C5-dimethylisoxazole%2520derivatives%2520as%2520potent%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3217%26epage%3D3227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Seal, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamotte, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donche, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouillot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellibert, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krysa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCleary, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioja, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span> </span><span class="NLM_article-title">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2968</span><span class="NLM_x">–</span> <span class="NLM_lpage">2972</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.bmcl.2012.02.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=22437115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2968-2972&author=J.+Sealauthor=Y.+Lamotteauthor=F.+Doncheauthor=A.+Bouillotauthor=O.+Mirguetauthor=F.+Gellibertauthor=E.+Nicodemeauthor=G.+Krysaauthor=J.+Kirilovskyauthor=S.+Beinkeauthor=S.+McClearyauthor=I.+Riojaauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=L.+Gordonauthor=T.+Lewisauthor=A.+L.+Walkerauthor=L.+Cutlerauthor=D.+Lugoauthor=D.+M.+Wilsonauthor=J.+Witheringtonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Identification+of+a+novel+series+of+BET+family+bromodomain+inhibitors%3A+Binding+mode+and+profile+of+I-BET151+%28GSK1210151A%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span></div><div class="casAuthors">Seal, Jonathan; Lamotte, Yann; Donche, Frederic; Bouillot, Anne; Mirguet, Olivier; Gellibert, Francoise; Nicodeme, Edwige; Krysa, Gael; Kirilovsky, Jorge; Beinke, Soren; McCleary, Scott; Rioja, Inma; Bamborough, Paul; Chung, Chun-Wa; Gordon, Laurie; Lewis, Toni; Walker, Ann L.; Cutler, Leanne; Lugo, David; Wilson, David M.; Witherington, Jason; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2968-2972</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of quinoline isoxazole BET family bromodomain inhibitors are discussed.  Crystallog. is used to illustrate binding modes and rationalize their SAR.  One member, I-BET151 (GSK1210151A), shows good oral bioavailability in both the rat and minipig as well as demonstrating efficient suppression of bacterial induced inflammation and sepsis in a murine in vivo endotoxemia model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJKtxBCr-pRbVg90H21EOLACvtfcHk0lge99O9CY81fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D&md5=a8037c89bcb93948d663208bd1ebfd55</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.041%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DIdentification%2520of%2520a%2520novel%2520series%2520of%2520BET%2520family%2520bromodomain%2520inhibitors%253A%2520Binding%2520mode%2520and%2520profile%2520of%2520I-BET151%2520%2528GSK1210151A%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2968%26epage%3D2972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giotopoulos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huthmacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gudgin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auger, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doehner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">529</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+P.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lge99O9CY81fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diallo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodacre, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seal, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodrow, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 2: Optimization of phenylisoxazole sulfonamides</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">587</span><span class="NLM_x">–</span> <span class="NLM_lpage">596</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201283q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=587-596&author=P.+Bamboroughauthor=H.+Dialloauthor=J.+D.+Goodacreauthor=L.+Gordonauthor=A.+Lewisauthor=J.+T.+Sealauthor=D.+M.+Wilsonauthor=M.+D.+Woodrowauthor=C.+W.+Chung&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+2%3A+Optimization+of+phenylisoxazole+sulfonamides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 2: Optimization of Phenylisoxazole Sulfonamides</span></div><div class="casAuthors">Bamborough, Paul; Diallo, Hawa; Goodacre, Jonathan D.; Gordon, Laurie; Lewis, Antonia; Seal, Jonathan T.; Wilson, David M.; Woodrow, Michael D.; Chung, Chun-wa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">587-596</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic reader modules that regulate gene transcription through their recognition of acetyl-lysine modified histone tails.  Inhibitors of this protein-protein interaction have the potential to modulate multiple diseases as demonstrated by the profound anti-inflammatory and antiproliferative effects of a recently disclosed class of BET compds.  While these compds. were discovered using phenotypic assays, here we present a highly efficient alternative approach to find new chem. templates, exploiting the abundant structural knowledge that exists for this target class.  A Ph di-Me isoxazole chemotype resulting from a focused fragment screen has been rapidly optimized through structure-based design, leading to a sulfonamide series showing anti-inflammatory activity in cellular assays.  This proof-of-principle expt. demonstrates the tractability of the BET family and bromodomain target class to fragment-based hit discovery and structure-based lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr6lsic9qjJrVg90H21EOLACvtfcHk0lgT7OOc1nUdeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF&md5=e73b16b3fddf88b43b037264b000b276</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm201283q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201283q%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DGoodacre%26aufirst%3DJ.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWoodrow%26aufirst%3DM.%2BD.%26aulast%3DChung%26aufirst%3DC.%2BW.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25202%253A%2520Optimization%2520of%2520phenylisoxazole%2520sulfonamides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D587%26epage%3D596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Borah, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mujtaba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karakikes, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moshkina, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerona-Navarro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajjar, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">A small molecule binding to the coactivator CREB-binding protein blocks apoptosis in cardiomyocytes</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.chembiol.2010.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=21513889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=531-541&author=J.+C.+Borahauthor=S.+Mujtabaauthor=I.+Karakikesauthor=L.+Zengauthor=M.+Mullerauthor=J.+Patelauthor=N.+Moshkinaauthor=K.+Morohashiauthor=W.+J.+Zhangauthor=G.+Gerona-Navarroauthor=R.+J.+Hajjarauthor=M.+M.+Zhou&title=A+small+molecule+binding+to+the+coactivator+CREB-binding+protein+blocks+apoptosis+in+cardiomyocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Binding to the Coactivator CREB-Binding Protein Blocks Apoptosis in Cardiomyocytes</span></div><div class="casAuthors">Borah, Jagat C.; Mujtaba, Shiraz; Karakikes, Ioannis; Zeng, Lei; Muller, Michaela; Patel, Jigneshkumar; Moshkina, Natasha; Morohashi, Keita; Zhang, Weijia; Gerona-Navarro, Guillermo; Hajjar, Roger J.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-541</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: As a master transcription factor in cellular responses to external stress, tumor suppressor p53 is tightly regulated.  Excessive p53 activity during myocardial ischemia causes irreversible cellular injury and cardiomyocyte death.  P53 activation is dependent on lysine acetylation by the lysine acetyltransferase and transcriptional coactivator CREB-binding protein (CBP) and on acetylation-directed CBP recruitment for p53 target gene expression.  Here, we report a small mol. ischemin, developed with a structure-guided approach to inhibit the acetyl-lysine binding activity of the bromodomain of CBP.  We show that ischemin alters post-translational modifications on p53 and histones, inhibits p53 interaction with CBP and transcriptional activity in cells, and prevents apoptosis in ischemic cardiomyocytes.  Our study suggests small mol. modulation of acetylation-mediated interactions in gene transcription as a new approach to therapeutic interventions of human disorders such as myocardial ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHKOaQAGSYTbVg90H21EOLACvtfcHk0lgT7OOc1nUdeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D&md5=0c1d7500e1fff40bc335d1b10ac3c27a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DBorah%26aufirst%3DJ.%2BC.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DKarakikes%26aufirst%3DI.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DMoshkina%26aufirst%3DN.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DW.%2BJ.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DHajjar%26aufirst%3DR.%2BJ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DA%2520small%2520molecule%2520binding%2520to%2520the%2520coactivator%2520CREB-binding%2520protein%2520blocks%2520apoptosis%2520in%2520cardiomyocytes%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D531%26epage%3D541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Perola, E.</span><span> </span><span class="NLM_article-title">An analysis of the binding efficiencies of drugs and their leads in successful drug discovery programs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2986</span><span class="NLM_x">–</span> <span class="NLM_lpage">2997</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100118x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlWqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2986-2997&author=E.+Perola&title=An+analysis+of+the+binding+efficiencies+of+drugs+and+their+leads+in+successful+drug+discovery+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">An Analysis of the Binding Efficiencies of Drugs and their Leads in Successful Drug Discovery Programs</span></div><div class="casAuthors">Perola, Emanuele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2986-2997</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In order to investigate the evolution of binding efficiency in successful drug discovery programs, a data set of 60 lead/drug pairs with known binding affinities has been compiled and analyzed.  Low-end thresholds for the binding efficiencies of viable leads and drugs have been derived.  On av., the drugs in the set are significantly larger and more potent but have similar lipophilicity relative to their originating leads, suggesting that the ability to maintain low levels of lipophilicity while increasing mol. wt. is one of the keys to a successful drug discovery program.  A no. of examples demonstrate that large increases in binding efficiency from leads to more elaborate drugs sharing the same scaffold can be achieved.  The importance of dissecting a lead structure to identify the most efficient fragments and the option of sacrificing binding efficiency to optimize other properties are discussed, and relevant examples are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRqzYFBuX2Q7Vg90H21EOLACvtfcHk0ljP78kVf7wz_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlWqu7o%253D&md5=a7eaad792f1518d093588951df64d22f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm100118x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100118x%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26atitle%3DAn%2520analysis%2520of%2520the%2520binding%2520efficiencies%2520of%2520drugs%2520and%2520their%2520leads%2520in%2520successful%2520drug%2520discovery%2520programs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2986%26epage%3D2997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Tarcsay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">Contributions of molecular properties to drug promiscuity miniperspective</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1789</span><span class="NLM_x">–</span> <span class="NLM_lpage">1795</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm301514n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1789-1795&author=A.+Tarcsayauthor=G.+M.+Keseru&title=Contributions+of+molecular+properties+to+drug+promiscuity+miniperspective"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm301514n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301514n%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3DA.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DContributions%2520of%2520molecular%2520properties%2520to%2520drug%2520promiscuity%2520miniperspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1789%26epage%3D1795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Hughes, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeCrescenzo, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devraj, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellsworth, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fobian, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibbs, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilles, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krieger-Burke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loesel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wager, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whiteley, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Physiochemical drug properties associated with in vivo toxicological outcomes</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4872</span><span class="NLM_x">–</span> <span class="NLM_lpage">4875</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.bmcl.2008.07.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=18691886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOnsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4872-4875&author=J.+D.+Hughesauthor=J.+Blaggauthor=D.+A.+Priceauthor=S.+Baileyauthor=G.+A.+DeCrescenzoauthor=R.+V.+Devrajauthor=E.+Ellsworthauthor=Y.+M.+Fobianauthor=M.+E.+Gibbsauthor=R.+W.+Gillesauthor=N.+Greeneauthor=E.+Huangauthor=T.+Krieger-Burkeauthor=J.+Loeselauthor=T.+Wagerauthor=L.+Whiteleyauthor=Y.+Zhang&title=Physiochemical+drug+properties+associated+with+in+vivo+toxicological+outcomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Physiochemical drug properties associated with in vivo toxicological outcomes</span></div><div class="casAuthors">Hughes, Jason D.; Blagg, Julian; Price, David A.; Bailey, Simon; DeCrescenzo, Gary A.; Devraj, Rajesh V.; Ellsworth, Edmund; Fobian, Yvette M.; Gibbs, Michael E.; Gilles, Richard W.; Greene, Nigel; Huang, Enoch; Krieger-Burke, Teresa; Loesel, Jens; Wager, Travis; Whiteley, Larry; Zhang, Yao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4872-4875</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Relationships between physicochem. drug properties and toxicity were inferred from a data set consisting of animal in vivo toleration (IVT) studies on 245 preclin. Pfizer compds.; an increased likelihood of toxic events was found for less polar, more lipophilic compds.  This trend held across a wide range of types of toxicity and across a broad swath of chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPMmGEpVF0mbVg90H21EOLACvtfcHk0ljP78kVf7wz_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOnsbjE&md5=80ea5ced359112a363f38b3545cd7646</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.07.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.07.071%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DJ.%2BD.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DBailey%26aufirst%3DS.%26aulast%3DDeCrescenzo%26aufirst%3DG.%2BA.%26aulast%3DDevraj%26aufirst%3DR.%2BV.%26aulast%3DEllsworth%26aufirst%3DE.%26aulast%3DFobian%26aufirst%3DY.%2BM.%26aulast%3DGibbs%26aufirst%3DM.%2BE.%26aulast%3DGilles%26aufirst%3DR.%2BW.%26aulast%3DGreene%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DE.%26aulast%3DKrieger-Burke%26aufirst%3DT.%26aulast%3DLoesel%26aufirst%3DJ.%26aulast%3DWager%26aufirst%3DT.%26aulast%3DWhiteley%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DPhysiochemical%2520drug%2520properties%2520associated%2520with%2520in%2520vivo%2520toxicological%2520outcomes%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4872%26epage%3D4875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Lee, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, N. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, M. S.</span><span> </span><span class="NLM_article-title">NF-kappa B-dependent expression of nitric oxide synthase is required for membrane fusion of chick embryonic myoblasts</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">324</span><span class="NLM_x">, </span> <span class="NLM_fpage">237</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=1997&pages=237-242&author=K.+H.+Leeauthor=D.+G.+Kimauthor=N.+Y.+Shinauthor=W.+K.+Songauthor=H.+Kwonauthor=C.+H.+Chungauthor=M.+S.+Kang&title=NF-kappa+B-dependent+expression+of+nitric+oxide+synthase+is+required+for+membrane+fusion+of+chick+embryonic+myoblasts"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DShin%26aufirst%3DN.%2BY.%26aulast%3DSong%26aufirst%3DW.%2BK.%26aulast%3DKwon%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%2BH.%26aulast%3DKang%26aufirst%3DM.%2BS.%26atitle%3DNF-kappa%2520B-dependent%2520expression%2520of%2520nitric%2520oxide%2520synthase%2520is%2520required%2520for%2520membrane%2520fusion%2520of%2520chick%2520embryonic%2520myoblasts%26jtitle%3DBiochem.%2520J.%26date%3D1997%26volume%3D324%26spage%3D237%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Belkina, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikolajczyk, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span> </span><span class="NLM_article-title">BET protein function is required for inflammation: Brd2 genetic disruption and BET inhibitor JQ1 impair mouse macrophage inflammatory responses</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">190</span><span class="NLM_x">, </span> <span class="NLM_fpage">3670</span><span class="NLM_x">–</span> <span class="NLM_lpage">3678</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670-3678&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+protein+function+is+required+for+inflammation%3A+Brd2+genetic+disruption+and+BET+inhibitor+JQ1+impair+mouse+macrophage+inflammatory+responses"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-α while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0ljP78kVf7wz_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520protein%2520function%2520is%2520required%2520for%2520inflammation%253A%2520Brd2%2520genetic%2520disruption%2520and%2520BET%2520inhibitor%2520JQ1%2520impair%2520mouse%2520macrophage%2520inflammatory%2520responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26spage%3D3670%26epage%3D3678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bish, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Primiano, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ralph, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciammetta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trzupek, J. D.</span><span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra C-terminal bromodomain inhibition through optimization of a fragment-derived hit</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9831</span><span class="NLM_x">–</span> <span class="NLM_lpage">9837</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+C-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0lgjq-dSICSieQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520C-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Nikolovska-Coleska, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roller, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajewski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0lgjq-dSICSieQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D261%26epage%3D273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Otwinowski, Z.; Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. In  <span class="citation_source-book">Methods in Enzymology</span>; <span class="NLM_contrib-group">Carter; C. W.,  Jr.</span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=+Carter&author=C.+W.+&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMethods%2520in%2520Enzymology%26aulast%3DCarter%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A. .</span><span> </span><span class="NLM_article-title">MOLREP: An automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+.+Teplyakov&title=MOLREP%3A+An+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lhPwv-OLvdbAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%2B.%26atitle%3DMOLREP%253A%2520An%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhPwv-OLvdbAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: Model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhPwv-OLvdbAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Hunger, K.; Mischke, P.; Rieper, W.</span><span> </span><span class="NLM_article-title">Azo dyes</span>. In  <span class="citation_source-book">Ullmann’s Encyclopedia of Industrial Chemistry</span>; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2000</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1002%2F14356007.a03_245" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=K.+Hunger&author=P.+Mischke&author=W.+Rieper&title=Ullmann%E2%80%99s+Encyclopedia+of+Industrial+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2F14356007.a03_245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F14356007.a03_245%26sid%3Dliteratum%253Aachs%26aulast%3DHunger%26aufirst%3DK.%26atitle%3DAzo%2520dyes%26btitle%3DUllmann%25E2%2580%2599s%2520Encyclopedia%2520of%2520Industrial%2520Chemistry%26pub%3DWiley-VCH%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Zhou, M. M.; Ohlmeyer, M.; Mujtaba, S.; Plotnikov, A.; Kastrinsky, D.; Zhang, G.; Borah, J.</span><span> </span><span class="NLM_article-title">Inhibitors of bromodomains as modulators of gene expression</span>. Patent WO2012116170,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=M.+M.+Zhou&author=M.+Ohlmeyer&author=S.+Mujtaba&author=A.+Plotnikov&author=D.+Kastrinsky&author=G.+Zhang&author=J.+Borah&title=Inhibitors+of+bromodomains+as+modulators+of+gene+expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DInhibitors%2520of%2520bromodomains%2520as%2520modulators%2520of%2520gene%2520expression%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Wallace, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J.</span><span> </span><span class="NLM_article-title">LIGPLOT: A program to generate schematic diagrams of protein-ligand interactions</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1093%2Fprotein%2F8.2.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=7630882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADyaK2MXltVCgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1995&pages=127-134&author=A.+Wallaceauthor=R.+Laskowskiauthor=J.+Thornton&title=LIGPLOT%3A+A+program+to+generate+schematic+diagrams+of+protein-ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions</span></div><div class="casAuthors">Wallace, Andrew C.; Laskowski, Roman A.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">127-34</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The LIGPLOT program automatically generates schematic 2-D representations of protein-ligand complexes from std. Protein Data Bank file input.  The output is a color, or black-and-white, PostScript file giving a simple and informative representation of the intermol. interactions and their strengths, including hydrogen bonds, hydrophobic interactions and atom accessibilities.  The program is completely general for any ligand and can also be used to show other types of interaction in proteins and nucleic acids.  It was designed to facilitate the rapid inspection of many enzyme complexes, but has found many other applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp9SrWw0klVbVg90H21EOLACvtfcHk0ljsE-Wb3J--TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXltVCgtLc%253D&md5=ed74e39b0cd6be4b2b8907d3ff6d9619</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.2.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.2.127%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DA.%26aulast%3DLaskowski%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%26atitle%3DLIGPLOT%253A%2520A%2520program%2520to%2520generate%2520schematic%2520diagrams%2520of%2520protein-ligand%2520interactions%26jtitle%3DProtein%2520Eng.%26date%3D1995%26volume%3D8%26spage%3D127%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 86 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Lina Humbeck, Jette Pretzel, Saskia Spitzer, <span class="NLM_string-name hlFld-ContribAuthor">Oliver Koch</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of an Unexpected Similarity in Ligand Binding between BRD4 and PPARγ. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1255-1265. <a href="https://doi.org/10.1021/acschembio.1c00323" title="DOI URL">https://doi.org/10.1021/acschembio.1c00323</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00323%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDiscovery%252Bof%252Ban%252BUnexpected%252BSimilarity%252Bin%252BLigand%252BBinding%252Bbetween%252BBRD4%252Band%252BPPAR%2525CE%2525B3%26aulast%3DHumbeck%26aufirst%3DLina%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D30042021%26date%3D07062021%26date%3D28062021%26volume%3D16%26issue%3D7%26spage%3D1255%26epage%3D1265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Helen E. Aylott, Stephen J. Atkinson, Paul Bamborough, Anna Bassil, Chun-wa Chung, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Cassie Messenger, Darren Mitchell, Alexander Phillipou, Alex Preston, Rab K. Prinjha, Francesco Rianjongdee, Inmaculada Rioja, Jonathan T. Seal, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Template-Hopping Approach Leads to Potent, Selective, and Highly Soluble Bromo and Extraterminal Domain (BET) Second Bromodomain (BD2) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (6)
                                     , 3249-3281. <a href="https://doi.org/10.1021/acs.jmedchem.0c02156" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02156</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02156%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTemplate-Hopping%252BApproach%252BLeads%252Bto%252BPotent%25252C%252BSelective%25252C%252Band%252BHighly%252BSoluble%252BBromo%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitors%26aulast%3DAylott%26aufirst%3DHelen%2BE.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D04032021%26volume%3D64%26issue%3D6%26spage%3D3249%26epage%3D3281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jean-Rémy Marchand, Tim Knehans, Amedeo Caflisch, <span class="NLM_string-name hlFld-ContribAuthor">Andreas Vitalis</span>. </span><span class="cited-content_cbyCitation_article-title">An ABSINTH-Based Protocol for Predicting Binding Affinities between Proteins and Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 5188-5202. <a href="https://doi.org/10.1021/acs.jcim.0c00558" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00558</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00558%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAn%252BABSINTH-Based%252BProtocol%252Bfor%252BPredicting%252BBinding%252BAffinities%252Bbetween%252BProteins%252Band%252BSmall%252BMolecules%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19052020%26date%3D23092020%26date%3D08092020%26volume%3D60%26issue%3D10%26spage%3D5188%26epage%3D5202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher R. Wellaway, Paul Bamborough, Sharon G. Bernard, Chun-wa Chung, Peter D. Craggs, Leanne Cutler, Emmanuel H. Demont, John P. Evans, Laurie Gordon, Bhumika Karamshi, Antonia J. Lewis, Matthew J. Lindon, Darren J. Mitchell, Inmaculada Rioja, Peter E. Soden, Simon Taylor, Robert J. Watson, Rob Willis, James M. Woolven, Beata S. Wyspiańska, William J. Kerr, <span class="NLM_string-name hlFld-ContribAuthor">Rab K. Prinjha</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9020-9044. <a href="https://doi.org/10.1021/acs.jmedchem.0c00566" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00566</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ba%252BBromodomain%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BInhibitor%252BSelective%252Bfor%252Bthe%252BN-Terminal%252BBromodomains%252BThat%252BRetains%252Ban%252BAnti-inflammatory%252Band%252BAntiproliferative%252BPhenotype%26aulast%3DWellaway%26aufirst%3DChristopher%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06042020%26date%3D03082020%26volume%3D63%26issue%3D17%26spage%3D9020%26epage%3D9044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert J. Watson, Paul Bamborough, Heather Barnett, Chun-wa Chung, Rob Davis, Laurie Gordon, Paola Grandi, Massimo Petretich, Alex Phillipou, Rab K. Prinjha, Inmaculada Rioja, Peter Soden, Thilo Werner, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9045-9069. <a href="https://doi.org/10.1021/acs.jmedchem.0c00614" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00614</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DGSK789%25253A%252BA%252BSelective%252BInhibitor%252Bof%252Bthe%252BFirst%252BBromodomains%252B%252528BD1%252529%252Bof%252Bthe%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BProteins%26aulast%3DWatson%26aufirst%3DRobert%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14042020%26date%3D04082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9045%26epage%3D9069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan T. Seal, Stephen J. Atkinson, Helen Aylott, Paul Bamborough, Chun-wa Chung, Royston C. B. Copley, Laurie Gordon, Paola Grandi, James R. J. Gray, Lee A. Harrison, Thomas G. Hayhow, Matthew Lindon, Cassie Messenger, Anne-Marie Michon, Darren Mitchell, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Simon Taylor, Ian D. Wall, Robert J. Watson, James M. Woolven, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9093-9126. <a href="https://doi.org/10.1021/acs.jmedchem.0c00796" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00796</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BOptimization%252Bof%252Ba%252BNovel%25252C%252BWeak%252BBromo%252Band%252BExtra%252BTerminal%252BDomain%252B%252528BET%252529%252BBromodomain%252BFragment%252BLigand%252Bto%252Ba%252BPotent%252Band%252BSelective%252BSecond%252BBromodomain%252B%252528BD2%252529%252BInhibitor%26aulast%3DSeal%26aufirst%3DJonathan%2BT.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11052020%26date%3D30082020%26date%3D23072020%26volume%3D63%26issue%3D17%26spage%3D9093%26epage%3D9126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">George S. Sheppard, Le Wang, Steven D. Fidanze, Lisa A. Hasvold, Dachun Liu, John K. Pratt, Chang H. Park, Kenton Longenecker, Wei Qiu, Maricel Torrent, Peter J. Kovar, Mai Bui, Emily Faivre, Xiaoli Huang, Xiaoyu Lin, Denise Wilcox, Lu Zhang, Yu Shen, Daniel H. Albert, Terrance J. Magoc, Ganesh Rajaraman, Warren M. Kati, <span class="NLM_string-name hlFld-ContribAuthor">Keith F. McDaniel</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5585-5623. <a href="https://doi.org/10.1021/acs.jmedchem.0c00628" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00628</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BN-Ethyl-4-%25255B2-%2525284-fluoro-2%25252C6-dimethyl-phenoxy%252529-5-%2525281-hydroxy-1-methyl-ethyl%252529phenyl%25255D-6-methyl-7-oxo-1H-pyrrolo%25255B2%25252C3-c%25255Dpyridine-2-carboxamide%252B%252528ABBV-744%252529%25252C%252Ba%252BBET%252BBromodomain%252BInhibitor%252Bwith%252BSelectivity%252Bfor%252Bthe%252BSecond%252BBromodomain%26aulast%3DSheppard%26aufirst%3DGeorge%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D16042020%26date%3D07052020%26date%3D23042020%26volume%3D63%26issue%3D10%26spage%3D5585%26epage%3D5623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher R. Wellaway, Dominique Amans, Paul Bamborough, Heather Barnett, Rino A. Bit, Jack A. Brown, Neil R. Carlson, Chun-wa Chung, Anthony W. J. Cooper, Peter D. Craggs, Robert P. Davis, Tony W. Dean, John P. Evans, Laurie Gordon, Isobel L. Harada, David J. Hirst, Philip G. Humphreys, Katherine L. Jones, Antonia J. Lewis, Matthew J. Lindon, Dave Lugo, Mahnoor Mahmood, Scott McCleary, Patricia Medeiros, Darren J. Mitchell, Michael O’Sullivan, Armelle Le Gall, Vipulkumar K. Patel, Chris Patten, Darren L. Poole, Rishi R. Shah, Jane E. Smith, Kayleigh A. J. Stafford, Pamela J. Thomas, Mythily Vimal, Ian D. Wall, Robert J. Watson, Natalie Wellaway, Gang Yao, <span class="NLM_string-name hlFld-ContribAuthor">Rab K. Prinjha</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 714-746. <a href="https://doi.org/10.1021/acs.jmedchem.9b01670" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01670</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01670%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BBromodomain%252Band%252BExtraterminal%252BInhibitor%252Bwith%252Ba%252BLow%252BPredicted%252BHuman%252BDose%252Bthrough%252BSynergistic%252BUse%252Bof%252BEncoded%252BLibrary%252BTechnology%252Band%252BFragment%252BScreening%26aulast%3DWellaway%26aufirst%3DChristopher%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D08102019%26date%3D06012020%26volume%3D63%26issue%3D2%26spage%3D714%26epage%3D746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fei Jiang, Qinghua Hu, Zhimin Zhang, Hongmei Li, Huili Li, Dewei Zhang, Hanwen Li, Yu Ma, Jingjing Xu, Haifang Chen, Yong Cui, Yanle Zhi, Yanmin Zhang, Junyu Xu, Jiapeng Zhu, Tao Lu, <span class="NLM_string-name hlFld-ContribAuthor">Yadong Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (24)
                                     , 11080-11107. <a href="https://doi.org/10.1021/acs.jmedchem.9b01010" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BBenzo%25255Bcd%25255Dindol-2%2525281H%252529-ones%252Band%252BPyrrolo%25255B4%25252C3%25252C2-de%25255Dquinolin-2%2525281H%252529-ones%252Bas%252BBromodomain%252Band%252BExtra-Terminal%252BDomain%252B%252528BET%252529%252BInhibitors%252Bwith%252BSelectivity%252Bfor%252Bthe%252BFirst%252BBromodomain%252Bwith%252BPotential%252BHigh%252BEfficiency%252Bagainst%252BAcute%252BGouty%252BArthritis%26aulast%3DJiang%26aufirst%3DFei%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26062019%26date%3D13122019%26date%3D30112019%26volume%3D62%26issue%3D24%26spage%3D11080%26epage%3D11107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Carolyn
N. Paulson, Xianghong Guan, Alex M. Ayoub, Alice Chan, Rezaul M. Karim, William C. K. Pomerantz, Ernst Schönbrunn, Gunda I. Georg, <span class="NLM_string-name hlFld-ContribAuthor">Jon E. Hawkinson</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Characterization of a Fluorescence Polarization Pan-BET Bromodomain Probe. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (12)
                                     , 1223-1229. <a href="https://doi.org/10.1021/acsmedchemlett.8b00380" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00380</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00380%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BCharacterization%252Bof%252Ba%252BFluorescence%252BPolarization%252BPan-BET%252BBromodomain%252BProbe%26aulast%3DPaulson%26aufirst%3DCarolyn%2BN.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D16082018%26date%3D31102018%26date%3D09112018%26date%3D31102018%26volume%3D9%26issue%3D12%26spage%3D1223%26epage%3D1229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anand Divakaran, Siva K. Talluri, Alex M. Ayoub, Neeraj K. Mishra, Huarui Cui, John C. Widen, Norbert Berndt, Jin-Yi Zhu, Angela S. Carlson, Joseph J. Topczewski, Ernst K. Schonbrunn, Daniel A. Harki, <span class="NLM_string-name hlFld-ContribAuthor">William C. K. Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase–Bromodomain Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (20)
                                     , 9316-9334. <a href="https://doi.org/10.1021/acs.jmedchem.8b01248" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMolecular%252BBasis%252Bfor%252Bthe%252BN-Terminal%252BBromodomain-and-Extra-Terminal-Family%252BSelectivity%252Bof%252Ba%252BDual%252BKinase%2525E2%252580%252593Bromodomain%252BInhibitor%26aulast%3DDivakaran%26aufirst%3DAnand%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D07082018%26date%3D16102018%26date%3D25092018%26volume%3D61%26issue%3D20%26spage%3D9316%26epage%3D9334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Chong Qin, Yang Hu, Bing Zhou, Ester Fernandez-Salas, Chao-Yie Yang, Liu Liu, Donna McEachern, Sally Przybranowski, Mi Wang, Jeanne Stuckey, Jennifer Meagher, Longchuan Bai, Zhuo Chen, Mei Lin, Jiuling Yang, Danya N. Ziazadeh, Fuming Xu, Jiantao Hu, Weiguo Xiang, Liyue Huang, Siwei Li, Bo Wen, Duxin Sun, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6685-6704. <a href="https://doi.org/10.1021/acs.jmedchem.8b00506" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00506</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BQCA570%252Bas%252Ban%252BExceptionally%252BPotent%252Band%252BEfficacious%252BProteolysis%252BTargeting%252BChimera%252B%252528PROTAC%252529%252BDegrader%252Bof%252Bthe%252BBromodomain%252Band%252BExtra-Terminal%252B%252528BET%252529%252BProteins%252BCapable%252Bof%252BInducing%252BComplete%252Band%252BDurable%252BTumor%252BRegression%26aulast%3DQin%26aufirst%3DChong%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D30032018%26date%3D18072018%26volume%3D61%26issue%3D15%26spage%3D6685%26epage%3D6704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert P. Law, Stephen J. Atkinson, Paul Bamborough, Chun-wa Chung, Emmanuel H. Demont, Laurie J. Gordon, Matthew Lindon, Rab K. Prinjha, Allan J. B. Watson, <span class="NLM_string-name hlFld-ContribAuthor">David J. Hirst</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (10)
                                     , 4317-4334. <a href="https://doi.org/10.1021/acs.jmedchem.7b01666" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01666</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BTetrahydroquinoxalines%252Bas%252BBromodomain%252Band%252BExtra-Terminal%252BDomain%252B%252528BET%252529%252BInhibitors%252Bwith%252BSelectivity%252Bfor%252Bthe%252BSecond%252BBromodomain%26aulast%3DLaw%26aufirst%3DRobert%2BP.%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D11112017%26date%3D03052018%26date%3D15042018%26volume%3D61%26issue%3D10%26spage%3D4317%26epage%3D4334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bryce K.  Allen</span>, <span class="hlFld-ContribAuthor ">Saurabh  Mehta</span>, <span class="hlFld-ContribAuthor ">Stuart W. J.  Ember</span>, <span class="hlFld-ContribAuthor ">Jin-Yi  Zhu</span>, <span class="hlFld-ContribAuthor ">Ernst  Schönbrunn</span>, <span class="hlFld-ContribAuthor ">Nagi G.  Ayad</span>, and <span class="hlFld-ContribAuthor ">Stephan C.  Schürer</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of a Novel Class of BRD4 Inhibitors by Computational Screening and Binding Simulations. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (8)
                                     , 4760-4771. <a href="https://doi.org/10.1021/acsomega.7b00553" title="DOI URL">https://doi.org/10.1021/acsomega.7b00553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00553%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DIdentification%252Bof%252Ba%252BNovel%252BClass%252Bof%252BBRD4%252BInhibitors%252Bby%252BComputational%252BScreening%252Band%252BBinding%252BSimulations%26aulast%3DAllen%26aufirst%3DBryce%2BK.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D04052017%26date%3D27072017%26date%3D31082017%26volume%3D2%26issue%3D8%26spage%3D4760%26epage%3D4771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Xing</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Yulan  Wang</span>, <span class="hlFld-ContribAuthor ">Yiqian  Xie</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhe  Shi</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, and <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>  . </span><span class="cited-content_cbyCitation_article-title">Machine-Learning-Assisted Approach for Discovering Novel Inhibitors Targeting Bromodomain-Containing Protein 4. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (7)
                                     , 1677-1690. <a href="https://doi.org/10.1021/acs.jcim.7b00098" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00098%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMachine-Learning-Assisted%252BApproach%252Bfor%252BDiscovering%252BNovel%252BInhibitors%252BTargeting%252BBromodomain-Containing%252BProtein%252B4%26aulast%3DXing%26aufirst%3DJing%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D18022017%26date%3D10072017%26date%3D24072017%26date%3D21062017%26volume%3D57%26issue%3D7%26spage%3D1677%26epage%3D1690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alex M.  Ayoub</span>, <span class="hlFld-ContribAuthor ">Laura M. L.  Hawk</span>, <span class="hlFld-ContribAuthor ">Ryan J.  Herzig</span>, <span class="hlFld-ContribAuthor ">Jiewei  Jiang</span>, <span class="hlFld-ContribAuthor ">Andrea J.  Wisniewski</span>, <span class="hlFld-ContribAuthor ">Clifford T.  Gee</span>, <span class="hlFld-ContribAuthor ">Peiliang  Zhao</span>, <span class="hlFld-ContribAuthor ">Jin-Yi  Zhu</span>, <span class="hlFld-ContribAuthor ">Norbert  Berndt</span>, <span class="hlFld-ContribAuthor ">Nana K.  Offei-Addo</span>, <span class="hlFld-ContribAuthor ">Thomas G.  Scott</span>, <span class="hlFld-ContribAuthor ">Jun  Qi</span>, <span class="hlFld-ContribAuthor ">James E.  Bradner</span>, <span class="hlFld-ContribAuthor ">Timothy R.  Ward</span>, <span class="hlFld-ContribAuthor ">Ernst  Schönbrunn</span>, <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span>, and <span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>  . </span><span class="cited-content_cbyCitation_article-title">BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (12)
                                     , 4805-4817. <a href="https://doi.org/10.1021/acs.jmedchem.6b01336" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01336%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBET%252BBromodomain%252BInhibitors%252Bwith%252BOne-Step%252BSynthesis%252BDiscovered%252Bfrom%252BVirtual%252BScreen%26aulast%3DAyoub%26aufirst%3DAlex%2BM.%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D05092016%26date%3D07062017%26date%3D22062017%26date%3D23052017%26volume%3D60%26issue%3D12%26spage%3D4805%26epage%3D4817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhiqing  Liu</span>, <span class="hlFld-ContribAuthor ">Pingyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Haiying  Chen</span>, <span class="hlFld-ContribAuthor ">Eric A.  Wold</span>, <span class="hlFld-ContribAuthor ">Bing  Tian</span>, <span class="hlFld-ContribAuthor ">Allan R.  Brasier</span>, and <span class="hlFld-ContribAuthor ">Jia  Zhou</span>  . </span><span class="cited-content_cbyCitation_article-title">Drug Discovery Targeting Bromodomain-Containing Protein 4. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (11)
                                     , 4533-4558. <a href="https://doi.org/10.1021/acs.jmedchem.6b01761" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01761</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDrug%252BDiscovery%252BTargeting%252BBromodomain-Containing%252BProtein%252B4%26aulast%3DLiu%26aufirst%3DZhiqing%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D02122016%26date%3D02032017%26date%3D08062017%26date%3D14022017%26volume%3D60%26issue%3D11%26spage%3D4533%26epage%3D4558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xavier  Gómez-Santacana</span>, <span class="hlFld-ContribAuthor ">Silvia  Pittolo</span>, <span class="hlFld-ContribAuthor ">Xavier  Rovira</span>, <span class="hlFld-ContribAuthor ">Marc  Lopez</span>, <span class="hlFld-ContribAuthor ">Charleine  Zussy</span>, <span class="hlFld-ContribAuthor ">James A. R.  Dalton</span>, <span class="hlFld-ContribAuthor ">Adèle  Faucherre</span>, <span class="hlFld-ContribAuthor ">Chris  Jopling</span>, <span class="hlFld-ContribAuthor ">Jean-Philippe  Pin</span>, <span class="hlFld-ContribAuthor ">Francisco  Ciruela</span>, <span class="hlFld-ContribAuthor ">Cyril  Goudet</span>, <span class="hlFld-ContribAuthor ">Jesús  Giraldo</span>, <span class="hlFld-ContribAuthor ">Pau  Gorostiza</span>, and <span class="hlFld-ContribAuthor ">Amadeu  Llebaria</span>  . </span><span class="cited-content_cbyCitation_article-title">Illuminating Phenylazopyridines To Photoswitch Metabotropic Glutamate Receptors: From the Flask to the Animals. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2017,</strong> <em>3 </em>
                                    (1)
                                     , 81-91. <a href="https://doi.org/10.1021/acscentsci.6b00353" title="DOI URL">https://doi.org/10.1021/acscentsci.6b00353</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.6b00353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.6b00353%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DIlluminating%252BPhenylazopyridines%252BTo%252BPhotoswitch%252BMetabotropic%252BGlutamate%252BReceptors%25253A%252BFrom%252Bthe%252BFlask%252Bto%252Bthe%252BAnimals%26aulast%3DG%25C3%25B3mez-Santacana%26aufirst%3DXavier%26date%3D2017%26date%3D2016%26date%3D2016%26date%3D15112016%26date%3D19122016%26date%3D25012017%26volume%3D3%26issue%3D1%26spage%3D81%26epage%3D91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven G.  Smith</span> and <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>  . </span><span class="cited-content_cbyCitation_article-title">The Bromodomain: A New Target in Emerging Epigenetic Medicine. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2016,</strong> <em>11 </em>
                                    (3)
                                     , 598-608. <a href="https://doi.org/10.1021/acschembio.5b00831" title="DOI URL">https://doi.org/10.1021/acschembio.5b00831</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DThe%252BBromodomain%25253A%252BA%252BNew%252BTarget%252Bin%252BEmerging%252BEpigenetic%252BMedicine%26aulast%3DSmith%26aufirst%3DSteven%2BG.%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D09102015%26date%3D23112015%26date%3D03122015%26date%3D18032016%26date%3D23112015%26volume%3D11%26issue%3D3%26spage%3D598%26epage%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalie H.  Theodoulou</span>, <span class="hlFld-ContribAuthor ">Paul  Bamborough</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Bannister</span>, <span class="hlFld-ContribAuthor ">Isabelle  Becher</span>, <span class="hlFld-ContribAuthor ">Rino A.  Bit</span>, <span class="hlFld-ContribAuthor ">Ka Hing  Che</span>, <span class="hlFld-ContribAuthor ">Chun-wa  Chung</span>, <span class="hlFld-ContribAuthor ">Antje  Dittmann</span>, <span class="hlFld-ContribAuthor ">Gerard  Drewes</span>, <span class="hlFld-ContribAuthor ">David H.  Drewry</span>, <span class="hlFld-ContribAuthor ">Laurie  Gordon</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>, <span class="hlFld-ContribAuthor ">Melanie  Leveridge</span>, <span class="hlFld-ContribAuthor ">Matthew  Lindon</span>, <span class="hlFld-ContribAuthor ">Anne-Marie  Michon</span>, <span class="hlFld-ContribAuthor ">Judit  Molnar</span>, <span class="hlFld-ContribAuthor ">Samuel C.  Robson</span>, <span class="hlFld-ContribAuthor ">Nicholas C. O.  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Tony  Kouzarides</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, and <span class="hlFld-ContribAuthor ">Philip G.  Humphreys</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1425-1439. <a href="https://doi.org/10.1021/acs.jmedchem.5b00256" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BI-BRD9%25252C%252Ba%252BSelective%252BCell%252BActive%252BChemical%252BProbe%252Bfor%252BBromodomain%252BContaining%252BProtein%252B9%252BInhibition%26aulast%3DTheodoulou%26aufirst%3DNatalie%2BH.%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D13022015%26date%3D30042015%26date%3D25022016%26date%3D09042015%26volume%3D59%26issue%3D4%26spage%3D1425%26epage%3D1439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias G. J.  Baud</span>, <span class="hlFld-ContribAuthor ">Enrique  Lin-Shiao</span>, <span class="hlFld-ContribAuthor ">Michael  Zengerle</span>, <span class="hlFld-ContribAuthor ">Cynthia  Tallant</span>, and <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>  . </span><span class="cited-content_cbyCitation_article-title">New Synthetic Routes to Triazolo-benzodiazepine Analogues: Expanding the Scope of the Bump-and-Hole Approach for Selective Bromo and Extra-Terminal (BET) Bromodomain Inhibition. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1492-1500. <a href="https://doi.org/10.1021/acs.jmedchem.5b01135" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01135</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01135%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNew%252BSynthetic%252BRoutes%252Bto%252BTriazolo-benzodiazepine%252BAnalogues%25253A%252BExpanding%252Bthe%252BScope%252Bof%252Bthe%252BBump-and-Hole%252BApproach%252Bfor%252BSelective%252BBromo%252Band%252BExtra-Terminal%252B%252528BET%252529%252BBromodomain%252BInhibition%26aulast%3DBaud%26aufirst%3DMatthias%2BG.%2BJ.%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D20072015%26date%3D01102015%26date%3D25022016%26date%3D14092015%26volume%3D59%26issue%3D4%26spage%3D1492%26epage%3D1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">F. Anthony  Romero</span>, <span class="hlFld-ContribAuthor ">Alexander M.  Taylor</span>, <span class="hlFld-ContribAuthor ">Terry D.  Crawford</span>, <span class="hlFld-ContribAuthor ">Vickie  Tsui</span>, <span class="hlFld-ContribAuthor ">Alexandre  Côté</span>, and <span class="hlFld-ContribAuthor ">Steven  Magnuson</span>  . </span><span class="cited-content_cbyCitation_article-title">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1271-1298. <a href="https://doi.org/10.1021/acs.jmedchem.5b01514" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01514</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDisrupting%252BAcetyl-Lysine%252BRecognition%25253A%252BProgress%252Bin%252Bthe%252BDevelopment%252Bof%252BBromodomain%252BInhibitors%26aulast%3DRomero%26aufirst%3DF.%2BAnthony%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D29092015%26date%3D01122015%26date%3D25022016%26date%3D16112015%26volume%3D59%26issue%3D4%26spage%3D1271%26epage%3D1298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangtao  Zhang</span>, <span class="hlFld-ContribAuthor ">Steven G.  Smith</span>, and <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Chemical Inhibitors of Human Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2015,</strong> <em>115 </em>
                                    (21)
                                     , 11625-11668. <a href="https://doi.org/10.1021/acs.chemrev.5b00205" title="DOI URL">https://doi.org/10.1021/acs.chemrev.5b00205</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.5b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.5b00205%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DDiscovery%252Bof%252BChemical%252BInhibitors%252Bof%252BHuman%252BBromodomains%26aulast%3DZhang%26aufirst%3DGuangtao%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D06042015%26date%3D23102015%26date%3D11112015%26volume%3D115%26issue%3D21%26spage%3D11625%26epage%3D11668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Brand</span>, <span class="hlFld-ContribAuthor ">Angelina M.  Measures</span>, <span class="hlFld-ContribAuthor ">Brian G.  Wilson</span>, <span class="hlFld-ContribAuthor ">Wilian A.  Cortopassi</span>, <span class="hlFld-ContribAuthor ">Rikki  Alexander</span>, <span class="hlFld-ContribAuthor ">Matthias  Höss</span>, <span class="hlFld-ContribAuthor ">David S.  Hewings</span>, <span class="hlFld-ContribAuthor ">Timothy P. C.  Rooney</span>, <span class="hlFld-ContribAuthor ">Robert S.  Paton</span>, and <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>  . </span><span class="cited-content_cbyCitation_article-title">Small Molecule Inhibitors of Bromodomain–Acetyl-lysine Interactions. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2015,</strong> <em>10 </em>
                                    (1)
                                     , 22-39. <a href="https://doi.org/10.1021/cb500996u" title="DOI URL">https://doi.org/10.1021/cb500996u</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb500996u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb500996u%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DSmall%252BMolecule%252BInhibitors%252Bof%252BBromodomain%2525E2%252580%252593Acetyl-lysine%252BInteractions%26aulast%3DBrand%26aufirst%3DMichael%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D05122014%26date%3D30122014%26date%3D16012015%26date%3D30122014%26volume%3D10%26issue%3D1%26spage%3D22%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laia  Richart</span>, <span class="hlFld-ContribAuthor ">François-Clément  Bidard</span>, <span class="hlFld-ContribAuthor ">Raphaël  Margueron</span>. </span><span class="cited-content_cbyCitation_article-title">Enhancer rewiring in tumors: an opportunity for therapeutic intervention. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2021,</strong> <em>40 </em>
                                    (20)
                                     , 3475-3491. <a href="https://doi.org/10.1038/s41388-021-01793-7" title="DOI URL">https://doi.org/10.1038/s41388-021-01793-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-021-01793-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-021-01793-7%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DEnhancer%252Brewiring%252Bin%252Btumors%25253A%252Ban%252Bopportunity%252Bfor%252Btherapeutic%252Bintervention%26aulast%3DRichart%26aufirst%3DLaia%26date%3D2021%26date%3D2021%26volume%3D40%26issue%3D20%26spage%3D3475%26epage%3D3491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammed  Ghiboub</span>, <span class="hlFld-ContribAuthor ">Ahmed M. I.  Elfiky</span>, <span class="hlFld-ContribAuthor ">Menno P. J.  de Winther</span>, <span class="hlFld-ContribAuthor ">Nicola R.  Harker</span>, <span class="hlFld-ContribAuthor ">David F.  Tough</span>, <span class="hlFld-ContribAuthor ">Wouter J.  de Jonge</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Targeting of Epigenetic Readers and Histone Deacetylases in Autoimmune and Inflammatory Diseases: Recent Advances and Future Perspectives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Personalized Medicine</span><span> <strong>2021,</strong> <em>11 </em>
                                    (5)
                                     , 336. <a href="https://doi.org/10.3390/jpm11050336" title="DOI URL">https://doi.org/10.3390/jpm11050336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/jpm11050336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fjpm11050336%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Personalized%2520Medicine%26atitle%3DSelective%252BTargeting%252Bof%252BEpigenetic%252BReaders%252Band%252BHistone%252BDeacetylases%252Bin%252BAutoimmune%252Band%252BInflammatory%252BDiseases%25253A%252BRecent%252BAdvances%252Band%252BFuture%252BPerspectives%26aulast%3DGhiboub%26aufirst%3DMohammed%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D5%26spage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giovanna  Spatari</span>, <span class="hlFld-ContribAuthor ">Alessandro  Allegra</span>, <span class="hlFld-ContribAuthor ">Mariella  Carrieri</span>, <span class="hlFld-ContribAuthor ">Giovanni  Pioggia</span>, <span class="hlFld-ContribAuthor ">Sebastiano  Gangemi</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Effects of Benzene in Hematologic Neoplasms: The Altered Gene Expression. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (10)
                                     , 2392. <a href="https://doi.org/10.3390/cancers13102392" title="DOI URL">https://doi.org/10.3390/cancers13102392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13102392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13102392%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DEpigenetic%252BEffects%252Bof%252BBenzene%252Bin%252BHematologic%252BNeoplasms%25253A%252BThe%252BAltered%252BGene%252BExpression%26aulast%3DSpatari%26aufirst%3DGiovanna%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D10%26spage%3D2392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shiliang  Wu</span>, <span class="hlFld-ContribAuthor ">Lifei  Wang</span>, <span class="hlFld-ContribAuthor ">Lulu  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaoyan  Xu</span>, <span class="hlFld-ContribAuthor ">Juan  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular dynamics insights into binding selectivity of inhibitors toward BRD4 and CBP. </span><span class="cited-content_cbyCitation_journal-name">Chemical Physics Letters</span><span> <strong>2021,</strong> <em>769 </em>, 138435. <a href="https://doi.org/10.1016/j.cplett.2021.138435" title="DOI URL">https://doi.org/10.1016/j.cplett.2021.138435</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cplett.2021.138435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cplett.2021.138435%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Physics%2520Letters%26atitle%3DMolecular%252Bdynamics%252Binsights%252Binto%252Bbinding%252Bselectivity%252Bof%252Binhibitors%252Btoward%252BBRD4%252Band%252BCBP%26aulast%3DWu%26aufirst%3DShiliang%26date%3D2021%26volume%3D769%26spage%3D138435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tian-bang  Wu</span>, <span class="hlFld-ContribAuthor ">Qiu-ping  Xiang</span>, <span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Chun  Wu</span>, <span class="hlFld-ContribAuthor ">Cheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Mao-feng  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhao-xuan  Liu</span>, <span class="hlFld-ContribAuthor ">Yan  Zhang</span>, <span class="hlFld-ContribAuthor ">Lin-jiu  Xiao</span>, <span class="hlFld-ContribAuthor ">Yong  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Y06014 is a selective BET inhibitor for the treatment of prostate cancer. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2021,</strong> <em>69 </em><a href="https://doi.org/10.1038/s41401-021-00614-7" title="DOI URL">https://doi.org/10.1038/s41401-021-00614-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41401-021-00614-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41401-021-00614-7%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DY06014%252Bis%252Ba%252Bselective%252BBET%252Binhibitor%252Bfor%252Bthe%252Btreatment%252Bof%252Bprostate%252Bcancer%26aulast%3DWu%26aufirst%3DTian-bang%26date%3D2021%26date%3D2021%26volume%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ewelina  Kulikowski</span>, <span class="hlFld-ContribAuthor ">Brooke D.  Rakai</span>, <span class="hlFld-ContribAuthor ">Norman C. W.  Wong</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of bromodomain and extra‐terminal proteins for treating multiple human diseases. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2021,</strong> <em>41 </em>
                                    (1)
                                     , 223-245. <a href="https://doi.org/10.1002/med.21730" title="DOI URL">https://doi.org/10.1002/med.21730</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21730%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DInhibitors%252Bof%252Bbromodomain%252Band%252Bextra%2525E2%252580%252590terminal%252Bproteins%252Bfor%252Btreating%252Bmultiple%252Bhuman%252Bdiseases%26aulast%3DKulikowski%26aufirst%3DEwelina%26date%3D2021%26date%3D2020%26volume%3D41%26issue%3D1%26spage%3D223%26epage%3D245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mandakini B.  Singh</span>, <span class="hlFld-ContribAuthor ">Gregory C.  Sartor</span>. </span><span class="cited-content_cbyCitation_article-title">BET bromodomains as novel epigenetic targets for brain health and disease. </span><span class="cited-content_cbyCitation_journal-name">Neuropharmacology</span><span> <strong>2020,</strong> <em>181 </em>, 108306. <a href="https://doi.org/10.1016/j.neuropharm.2020.108306" title="DOI URL">https://doi.org/10.1016/j.neuropharm.2020.108306</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neuropharm.2020.108306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neuropharm.2020.108306%26sid%3Dliteratum%253Aachs%26jtitle%3DNeuropharmacology%26atitle%3DBET%252Bbromodomains%252Bas%252Bnovel%252Bepigenetic%252Btargets%252Bfor%252Bbrain%252Bhealth%252Band%252Bdisease%26aulast%3DSingh%26aufirst%3DMandakini%2BB.%26date%3D2020%26volume%3D181%26spage%3D108306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jessica Y.  Hsu</span>, <span class="hlFld-ContribAuthor ">Jennifer L.  Major</span>, <span class="hlFld-ContribAuthor ">Andrew S.  Riching</span>, <span class="hlFld-ContribAuthor ">Rwik  Sen</span>, <span class="hlFld-ContribAuthor ">Julie  Pires da Silva</span>, <span class="hlFld-ContribAuthor ">Rushita A.  Bagchi</span>. </span><span class="cited-content_cbyCitation_article-title">Beyond the genome: challenges and potential for epigenetics-driven therapeutic approaches in pulmonary arterial hypertension. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry and Cell Biology</span><span> <strong>2020,</strong> <em>98 </em>
                                    (6)
                                     , 631-646. <a href="https://doi.org/10.1139/bcb-2020-0039" title="DOI URL">https://doi.org/10.1139/bcb-2020-0039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1139/bcb-2020-0039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1139%2Fbcb-2020-0039%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%2520and%2520Cell%2520Biology%26atitle%3DBeyond%252Bthe%252Bgenome%25253A%252Bchallenges%252Band%252Bpotential%252Bfor%252Bepigenetics-driven%252Btherapeutic%252Bapproaches%252Bin%252Bpulmonary%252Barterial%252Bhypertension%26aulast%3DHsu%26aufirst%3DJessica%2BY.%26date%3D2020%26volume%3D98%26issue%3D6%26spage%3D631%26epage%3D646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Massimo  Petretich</span>, <span class="hlFld-ContribAuthor ">Emmanuel H.  Demont</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>. </span><span class="cited-content_cbyCitation_article-title">Domain-selective targeting of BET proteins in cancer and immunological diseases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 184-193. <a href="https://doi.org/10.1016/j.cbpa.2020.02.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.02.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.02.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDomain-selective%252Btargeting%252Bof%252BBET%252Bproteins%252Bin%252Bcancer%252Band%252Bimmunological%252Bdiseases%26aulast%3DPetretich%26aufirst%3DMassimo%26date%3D2020%26volume%3D57%26spage%3D184%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.L.  Wu</span>, <span class="hlFld-ContribAuthor ">L.F.  Wang</span>, <span class="hlFld-ContribAuthor ">H.B.  Sun</span>, <span class="hlFld-ContribAuthor ">W.  Wang</span>, <span class="hlFld-ContribAuthor ">Y.X.  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing molecular mechanism of inhibitor bindings to bromodomain-containing protein 4 based on molecular dynamics simulations and principal component analysis. </span><span class="cited-content_cbyCitation_journal-name">SAR and QSAR in Environmental Research</span><span> <strong>2020,</strong> <em>31 </em>
                                    (7)
                                     , 547-570. <a href="https://doi.org/10.1080/1062936X.2020.1777584" title="DOI URL">https://doi.org/10.1080/1062936X.2020.1777584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/1062936X.2020.1777584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F1062936X.2020.1777584%26sid%3Dliteratum%253Aachs%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26atitle%3DProbing%252Bmolecular%252Bmechanism%252Bof%252Binhibitor%252Bbindings%252Bto%252Bbromodomain-containing%252Bprotein%252B4%252Bbased%252Bon%252Bmolecular%252Bdynamics%252Bsimulations%252Band%252Bprincipal%252Bcomponent%252Banalysis%26aulast%3DWu%26aufirst%3DS.L.%26date%3D2020%26date%3D2020%26volume%3D31%26issue%3D7%26spage%3D547%26epage%3D570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arun  Padmanabhan</span>, <span class="hlFld-ContribAuthor ">Saptarsi M.  Haldar</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging transcription in heart failure. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physiology</span><span> <strong>2020,</strong> <em>598 </em>
                                    (14)
                                     , 3005-3014. <a href="https://doi.org/10.1113/JP276745" title="DOI URL">https://doi.org/10.1113/JP276745</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1113/JP276745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1113%2FJP276745%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physiology%26atitle%3DDrugging%252Btranscription%252Bin%252Bheart%252Bfailure%26aulast%3DPadmanabhan%26aufirst%3DArun%26date%3D2020%26date%3D2019%26volume%3D598%26issue%3D14%26spage%3D3005%26epage%3D3014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Durga  Khandekar</span>, <span class="hlFld-ContribAuthor ">Venkataswarup  Tiriveedhi</span>. </span><span class="cited-content_cbyCitation_article-title">Role of BET Inhibitors in Triple Negative Breast Cancers. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2020,</strong> <em>12 </em>
                                    (4)
                                     , 784. <a href="https://doi.org/10.3390/cancers12040784" title="DOI URL">https://doi.org/10.3390/cancers12040784</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers12040784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers12040784%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DRole%252Bof%252BBET%252BInhibitors%252Bin%252BTriple%252BNegative%252BBreast%252BCancers%26aulast%3DKhandekar%26aufirst%3DDurga%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D4%26spage%3D784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yoel  Rodríguez</span>, <span class="hlFld-ContribAuthor ">Guillermo  Gerona‐Navarro</span>, <span class="hlFld-ContribAuthor ">Roman  Osman</span>, <span class="hlFld-ContribAuthor ">Ming‐Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">In silico design and molecular basis for the selectivity of Olinone toward the first over the second bromodomain of BRD4. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2020,</strong> <em>88 </em>
                                    (3)
                                     , 414-430. <a href="https://doi.org/10.1002/prot.25818" title="DOI URL">https://doi.org/10.1002/prot.25818</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.25818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.25818%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DIn%252Bsilico%252Bdesign%252Band%252Bmolecular%252Bbasis%252Bfor%252Bthe%252Bselectivity%252Bof%252BOlinone%252Btoward%252Bthe%252Bfirst%252Bover%252Bthe%252Bsecond%252Bbromodomain%252Bof%252BBRD4%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DYoel%26date%3D2020%26date%3D2019%26volume%3D88%26issue%3D3%26spage%3D414%26epage%3D430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando D.  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Current advances on the development of BET inhibitors: insights from computational methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 127-180. <a href="https://doi.org/10.1016/bs.apcsb.2020.06.002" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.06.002%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252Badvances%252Bon%252Bthe%252Bdevelopment%252Bof%252BBET%252Binhibitors%25253A%252Binsights%252Bfrom%252Bcomputational%252Bmethods%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DFernando%2BD.%26date%3D2020%26spage%3D127%26epage%3D180%26pub%3DElsevier%26date%3D2020%26volume%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deheng  Chen</span>, <span class="hlFld-ContribAuthor ">Tian  Lu</span>, <span class="hlFld-ContribAuthor ">Ziqin  Yan</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Feilong  Zhou</span>, <span class="hlFld-ContribAuthor ">Xilin  Lyu</span>, <span class="hlFld-ContribAuthor ">Biling  Xu</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Yujun  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>182 </em>, 111633. <a href="https://doi.org/10.1016/j.ejmech.2019.111633" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111633</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111633%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%25252C%252Bstructural%252Binsight%25252C%252Band%252Bbioactivities%252Bof%252BBY27%252Bas%252Ba%252Bselective%252Binhibitor%252Bof%252Bthe%252Bsecond%252Bbromodomains%252Bof%252BBET%252Bproteins%26aulast%3DChen%26aufirst%3DDeheng%26date%3D2019%26volume%3D182%26spage%3D111633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Padmaja  Wakchaure</span>, <span class="hlFld-ContribAuthor ">Ravichandiran  Velayutham</span>, <span class="hlFld-ContribAuthor ">Kuldeep K.  Roy</span>. </span><span class="cited-content_cbyCitation_article-title">Structure investigation, enrichment analysis and structure-based repurposing of FDA-approved drugs as inhibitors of BET-BRD4. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2019,</strong> <em>37 </em>
                                    (12)
                                     , 3048-3057. <a href="https://doi.org/10.1080/07391102.2018.1507838" title="DOI URL">https://doi.org/10.1080/07391102.2018.1507838</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2018.1507838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2018.1507838%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DStructure%252Binvestigation%25252C%252Benrichment%252Banalysis%252Band%252Bstructure-based%252Brepurposing%252Bof%252BFDA-approved%252Bdrugs%252Bas%252Binhibitors%252Bof%252BBET-BRD4%26aulast%3DWakchaure%26aufirst%3DPadmaja%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D12%26spage%3D3048%26epage%3D3057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susu  Zhang</span>, <span class="hlFld-ContribAuthor ">Yue  Zhao</span>, <span class="hlFld-ContribAuthor ">Tiffany M.  Heaster</span>, <span class="hlFld-ContribAuthor ">Melissa A.  Fischer</span>, <span class="hlFld-ContribAuthor ">Kristy R.  Stengel</span>, <span class="hlFld-ContribAuthor ">Xiaofan  Zhou</span>, <span class="hlFld-ContribAuthor ">Haley  Ramsey</span>, <span class="hlFld-ContribAuthor ">Ming‐Ming  Zhou</span>, <span class="hlFld-ContribAuthor ">Michael R.  Savona</span>, <span class="hlFld-ContribAuthor ">Melissa C.  Skala</span>, <span class="hlFld-ContribAuthor ">Scott W.  Hiebert</span>. </span><span class="cited-content_cbyCitation_article-title">BET inhibitors reduce cell size and induce reversible cell cycle arrest in AML. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Biochemistry</span><span> <strong>2019,</strong> <em>120 </em>
                                    (5)
                                     , 7309-7322. <a href="https://doi.org/10.1002/jcb.28005" title="DOI URL">https://doi.org/10.1002/jcb.28005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcb.28005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcb.28005%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Biochemistry%26atitle%3DBET%252Binhibitors%252Breduce%252Bcell%252Bsize%252Band%252Binduce%252Breversible%252Bcell%252Bcycle%252Barrest%252Bin%252BAML%26aulast%3DZhang%26aufirst%3DSusu%26date%3D2019%26date%3D2018%26volume%3D120%26issue%3D5%26spage%3D7309%26epage%3D7322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mehrosh  Pervaiz</span>, <span class="hlFld-ContribAuthor ">Pankaj  Mishra</span>, <span class="hlFld-ContribAuthor ">Stefan  Günther</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomains: Promising Targets for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 347-381. <a href="https://doi.org/10.1002/9783527809257.ch13" title="DOI URL">https://doi.org/10.1002/9783527809257.ch13</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch13%26sid%3Dliteratum%253Aachs%26atitle%3DBromodomains%25253A%252BPromising%252BTargets%252Bfor%252BDrug%252BDiscovery%26aulast%3DPervaiz%26aufirst%3DMehrosh%26date%3D2019%26date%3D2019%26spage%3D347%26epage%3D381%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>, <span class="hlFld-ContribAuthor ">Jorden A.  Johnson</span>, <span class="hlFld-ContribAuthor ">Peter D.  Ycas</span>. </span><span class="cited-content_cbyCitation_article-title">Applied Biophysics for Bromodomain Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 287-337. <a href="https://doi.org/10.1007/7355_2019_79" title="DOI URL">https://doi.org/10.1007/7355_2019_79</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_79%26sid%3Dliteratum%253Aachs%26atitle%3DApplied%252BBiophysics%252Bfor%252BBromodomain%252BDrug%252BDiscovery%26aulast%3DPomerantz%26aufirst%3DWilliam%2BC.%2BK.%26date%3D2019%26date%3D2019%26spage%3D287%26epage%3D337%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thangavel  Vaijayanthi</span>, <span class="hlFld-ContribAuthor ">Ganesh N.  Pandian</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Sugiyama</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Control System of Epigenetics. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1833-1853. <a href="https://doi.org/10.1002/tcr.201800067" title="DOI URL">https://doi.org/10.1002/tcr.201800067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800067%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DChemical%252BControl%252BSystem%252Bof%252BEpigenetics%26aulast%3DVaijayanthi%26aufirst%3DThangavel%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1833%26epage%3D1853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alex  Dickson</span>. </span><span class="cited-content_cbyCitation_article-title">Mapping the Ligand Binding Landscape. </span><span class="cited-content_cbyCitation_journal-name">Biophysical Journal</span><span> <strong>2018,</strong> <em>115 </em>
                                    (9)
                                     , 1707-1719. <a href="https://doi.org/10.1016/j.bpj.2018.09.021" title="DOI URL">https://doi.org/10.1016/j.bpj.2018.09.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bpj.2018.09.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bpj.2018.09.021%26sid%3Dliteratum%253Aachs%26jtitle%3DBiophysical%2520Journal%26atitle%3DMapping%252Bthe%252BLigand%252BBinding%252BLandscape%26aulast%3DDickson%26aufirst%3DAlex%26date%3D2018%26volume%3D115%26issue%3D9%26spage%3D1707%26epage%3D1719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunyan  Ren</span>, <span class="hlFld-ContribAuthor ">Guangtao  Zhang</span>, <span class="hlFld-ContribAuthor ">Fangbin  Han</span>, <span class="hlFld-ContribAuthor ">Shibo  Fu</span>, <span class="hlFld-ContribAuthor ">Yingdi  Cao</span>, <span class="hlFld-ContribAuthor ">Fan  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiang  Zhang</span>, <span class="hlFld-ContribAuthor ">Jamel  Meslamani</span>, <span class="hlFld-ContribAuthor ">Yaoyao  Xu</span>, <span class="hlFld-ContribAuthor ">Donglei  Ji</span>, <span class="hlFld-ContribAuthor ">Lingling  Cao</span>, <span class="hlFld-ContribAuthor ">Qian  Zhou</span>, <span class="hlFld-ContribAuthor ">Ka-lung  Cheung</span>, <span class="hlFld-ContribAuthor ">Rajal  Sharma</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Zhengzi  Yi</span>, <span class="hlFld-ContribAuthor ">Weijia  Zhang</span>, <span class="hlFld-ContribAuthor ">Martin J.  Walsh</span>, <span class="hlFld-ContribAuthor ">Lei  Zeng</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Spatially constrained tandem bromodomain inhibition bolsters sustained repression of BRD4 transcriptional activity for TNBC cell growth. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2018,</strong> <em>115 </em>
                                    (31)
                                     , 7949-7954. <a href="https://doi.org/10.1073/pnas.1720000115" title="DOI URL">https://doi.org/10.1073/pnas.1720000115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1720000115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1720000115%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DSpatially%252Bconstrained%252Btandem%252Bbromodomain%252Binhibition%252Bbolsters%252Bsustained%252Brepression%252Bof%252BBRD4%252Btranscriptional%252Bactivity%252Bfor%252BTNBC%252Bcell%252Bgrowth%26aulast%3DRen%26aufirst%3DChunyan%26date%3D2018%26date%3D2018%26volume%3D115%26issue%3D31%26spage%3D7949%26epage%3D7954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqian  Xue</span>, <span class="hlFld-ContribAuthor ">Yan  Zhang</span>, <span class="hlFld-ContribAuthor ">Chao  Wang</span>, <span class="hlFld-ContribAuthor ">Maofeng  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiuping  Xiang</span>, <span class="hlFld-ContribAuthor ">Junjian  Wang</span>, <span class="hlFld-ContribAuthor ">Anhui  Wang</span>, <span class="hlFld-ContribAuthor ">Chenchang  Li</span>, <span class="hlFld-ContribAuthor ">Cheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Lingjiao  Zou</span>, <span class="hlFld-ContribAuthor ">Rui  Wang</span>, <span class="hlFld-ContribAuthor ">Shuang  Wu</span>, <span class="hlFld-ContribAuthor ">Yongzhi  Lu</span>, <span class="hlFld-ContribAuthor ">Hongwu  Chen</span>, <span class="hlFld-ContribAuthor ">Ke  Ding</span>, <span class="hlFld-ContribAuthor ">Guohui  Li</span>, <span class="hlFld-ContribAuthor ">Yong  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Benzoxazinone-containing 3,5-dimethylisoxazole derivatives as BET bromodomain inhibitors for treatment of castration-resistant prostate cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>152 </em>, 542-559. <a href="https://doi.org/10.1016/j.ejmech.2018.04.034" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.04.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.04.034%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBenzoxazinone-containing%252B3%25252C5-dimethylisoxazole%252Bderivatives%252Bas%252BBET%252Bbromodomain%252Binhibitors%252Bfor%252Btreatment%252Bof%252Bcastration-resistant%252Bprostate%252Bcancer%26aulast%3DXue%26aufirst%3DXiaoqian%26date%3D2018%26volume%3D152%26spage%3D542%26epage%3D559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nagakumar  Bharatham</span>, <span class="hlFld-ContribAuthor ">Peter J.  Slavish</span>, <span class="hlFld-ContribAuthor ">William R.  Shadrick</span>, <span class="hlFld-ContribAuthor ">Brandon M.  Young</span>, <span class="hlFld-ContribAuthor ">Anang A.  Shelat</span>. </span><span class="cited-content_cbyCitation_article-title">The role of ZA channel water-mediated interactions in the design of bromodomain-selective BET inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2018,</strong> <em>81 </em>, 197-210. <a href="https://doi.org/10.1016/j.jmgm.2018.03.005" title="DOI URL">https://doi.org/10.1016/j.jmgm.2018.03.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2018.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2018.03.005%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DThe%252Brole%252Bof%252BZA%252Bchannel%252Bwater-mediated%252Binteractions%252Bin%252Bthe%252Bdesign%252Bof%252Bbromodomain-selective%252BBET%252Binhibitors%26aulast%3DBharatham%26aufirst%3DNagakumar%26date%3D2018%26volume%3D81%26spage%3D197%26epage%3D210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhiqing  Liu</span>, <span class="hlFld-ContribAuthor ">Bing  Tian</span>, <span class="hlFld-ContribAuthor ">Haiying  Chen</span>, <span class="hlFld-ContribAuthor ">Pingyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Allan R.  Brasier</span>, <span class="hlFld-ContribAuthor ">Jia  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of potent and selective BRD4 inhibitors capable of blocking TLR3-induced acute airway inflammation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>151 </em>, 450-461. <a href="https://doi.org/10.1016/j.ejmech.2018.04.006" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.04.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.04.006%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bpotent%252Band%252Bselective%252BBRD4%252Binhibitors%252Bcapable%252Bof%252Bblocking%252BTLR3-induced%252Bacute%252Bairway%252Binflammation%26aulast%3DLiu%26aufirst%3DZhiqing%26date%3D2018%26volume%3D151%26spage%3D450%26epage%3D461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bing  Tian</span>, <span class="hlFld-ContribAuthor ">Zhiqing  Liu</span>, <span class="hlFld-ContribAuthor ">Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Hong  Sun</span>, <span class="hlFld-ContribAuthor ">Yingxin  Zhao</span>, <span class="hlFld-ContribAuthor ">Maki  Wakamiya</span>, <span class="hlFld-ContribAuthor ">Haiying  Chen</span>, <span class="hlFld-ContribAuthor ">Erik  Rytting</span>, <span class="hlFld-ContribAuthor ">Jia  Zhou</span>, <span class="hlFld-ContribAuthor ">Allan R.  Brasier</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Antagonists of the Bronchiolar Epithelial NF-κB-Bromodomain-Containing Protein 4 Pathway in Viral-Induced Airway Inflammation. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2018,</strong> <em>23 </em>
                                    (4)
                                     , 1138-1151. <a href="https://doi.org/10.1016/j.celrep.2018.03.106" title="DOI URL">https://doi.org/10.1016/j.celrep.2018.03.106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2018.03.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2018.03.106%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DSelective%252BAntagonists%252Bof%252Bthe%252BBronchiolar%252BEpithelial%252BNF-%2525CE%2525BAB-Bromodomain-Containing%252BProtein%252B4%252BPathway%252Bin%252BViral-Induced%252BAirway%252BInflammation%26aulast%3DTian%26aufirst%3DBing%26date%3D2018%26volume%3D23%26issue%3D4%26spage%3D1138%26epage%3D1151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoyi  Yan</span>, <span class="hlFld-ContribAuthor ">Manzhou  Hou</span>, <span class="hlFld-ContribAuthor ">Jiang  Luo</span>, <span class="hlFld-ContribAuthor ">Chunlan  Pu</span>, <span class="hlFld-ContribAuthor ">Xueyan  Hou</span>, <span class="hlFld-ContribAuthor ">Suke  Lan</span>, <span class="hlFld-ContribAuthor ">Rui  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore-based virtual screening, molecular docking, molecular dynamics simulation, and biological evaluation for the discovery of novel BRD4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2018,</strong> <em>91 </em>
                                    (2)
                                     , 478-490. <a href="https://doi.org/10.1111/cbdd.13109" title="DOI URL">https://doi.org/10.1111/cbdd.13109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13109%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DPharmacophore-based%252Bvirtual%252Bscreening%25252C%252Bmolecular%252Bdocking%25252C%252Bmolecular%252Bdynamics%252Bsimulation%25252C%252Band%252Bbiological%252Bevaluation%252Bfor%252Bthe%252Bdiscovery%252Bof%252Bnovel%252BBRD4%252Binhibitors%26aulast%3DYan%26aufirst%3DGuoyi%26date%3D2018%26date%3D2017%26volume%3D91%26issue%3D2%26spage%3D478%26epage%3D490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anastasios  Stathis</span>, <span class="hlFld-ContribAuthor ">Francesco  Bertoni</span>. </span><span class="cited-content_cbyCitation_article-title">BET Proteins as Targets for Anticancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Cancer Discovery</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     , 24-36. <a href="https://doi.org/10.1158/2159-8290.CD-17-0605" title="DOI URL">https://doi.org/10.1158/2159-8290.CD-17-0605</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/2159-8290.CD-17-0605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F2159-8290.CD-17-0605%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Discovery%26atitle%3DBET%252BProteins%252Bas%252BTargets%252Bfor%252BAnticancer%252BTreatment%26aulast%3DStathis%26aufirst%3DAnastasios%26date%3D2018%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D24%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Utkarsh  Raj</span>, <span class="hlFld-ContribAuthor ">Himansu  Kumar</span>, <span class="hlFld-ContribAuthor ">Pritish Kumar  Varadwaj</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking and dynamics simulation study of flavonoids as BET bromodomain inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2017,</strong> <em>35 </em>
                                    (11)
                                     , 2351-2362. <a href="https://doi.org/10.1080/07391102.2016.1217276" title="DOI URL">https://doi.org/10.1080/07391102.2016.1217276</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2016.1217276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2016.1217276%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DMolecular%252Bdocking%252Band%252Bdynamics%252Bsimulation%252Bstudy%252Bof%252Bflavonoids%252Bas%252BBET%252Bbromodomain%252Binhibitors%26aulast%3DRaj%26aufirst%3DUtkarsh%26date%3D2017%26date%3D2016%26volume%3D35%26issue%3D11%26spage%3D2351%26epage%3D2362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nilesh  Zaware</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical modulators for epigenome reader domains as emerging epigenetic therapies for cancer and inflammation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2017,</strong> <em>39 </em>, 116-125. <a href="https://doi.org/10.1016/j.cbpa.2017.06.012" title="DOI URL">https://doi.org/10.1016/j.cbpa.2017.06.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2017.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2017.06.012%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DChemical%252Bmodulators%252Bfor%252Bepigenome%252Breader%252Bdomains%252Bas%252Bemerging%252Bepigenetic%252Btherapies%252Bfor%252Bcancer%252Band%252Binflammation%26aulast%3DZaware%26aufirst%3DNilesh%26date%3D2017%26volume%3D39%26spage%3D116%26epage%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yali  Xu</span>, <span class="hlFld-ContribAuthor ">Christopher R.  Vakoc</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Cancer Cells with BET Bromodomain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cold Spring Harbor Perspectives in Medicine</span><span> <strong>2017,</strong> <em>7 </em>
                                    (7)
                                     , a026674. <a href="https://doi.org/10.1101/cshperspect.a026674" title="DOI URL">https://doi.org/10.1101/cshperspect.a026674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/cshperspect.a026674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fcshperspect.a026674%26sid%3Dliteratum%253Aachs%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspectives%2520in%2520Medicine%26atitle%3DTargeting%252BCancer%252BCells%252Bwith%252BBET%252BBromodomain%252BInhibitors%26aulast%3DXu%26aufirst%3DYali%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D7%26spage%3Da026674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Montserrat  Pérez-Salvia</span>, <span class="hlFld-ContribAuthor ">Manel  Esteller</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain inhibitors and cancer therapy: From structures to applications. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 323-339. <a href="https://doi.org/10.1080/15592294.2016.1265710" title="DOI URL">https://doi.org/10.1080/15592294.2016.1265710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2016.1265710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2016.1265710%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DBromodomain%252Binhibitors%252Band%252Bcancer%252Btherapy%25253A%252BFrom%252Bstructures%252Bto%252Bapplications%26aulast%3DP%25C3%25A9rez-Salvia%26aufirst%3DMontserrat%26date%3D2017%26date%3D2016%26volume%3D12%26issue%3D5%26spage%3D323%26epage%3D339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Holger  Greschik</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Thomas  Günther</span>. </span><span class="cited-content_cbyCitation_article-title">Selective targeting of epigenetic reader domains. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 449-463. <a href="https://doi.org/10.1080/17460441.2017.1303474" title="DOI URL">https://doi.org/10.1080/17460441.2017.1303474</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2017.1303474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2017.1303474%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DSelective%252Btargeting%252Bof%252Bepigenetic%252Breader%252Bdomains%26aulast%3DGreschik%26aufirst%3DHolger%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3D449%26epage%3D463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhongya  Sun</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhifeng  Chen</span>, <span class="hlFld-ContribAuthor ">Yiqian  Xie</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Limin  Chen</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel BRD4 inhibitors by high-throughput screening, crystallography, and cell-based assays. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (9)
                                     , 2003-2009. <a href="https://doi.org/10.1016/j.bmcl.2017.03.012" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.03.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.03.012%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bnovel%252BBRD4%252Binhibitors%252Bby%252Bhigh-throughput%252Bscreening%25252C%252Bcrystallography%25252C%252Band%252Bcell-based%252Bassays%26aulast%3DSun%26aufirst%3DZhongya%26date%3D2017%26volume%3D27%26issue%3D9%26spage%3D2003%26epage%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Victoria  Jeffers</span>, <span class="hlFld-ContribAuthor ">Chunlin  Yang</span>, <span class="hlFld-ContribAuthor ">Sherri  Huang</span>, <span class="hlFld-ContribAuthor ">William J.  Sullivan</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomains in Protozoan Parasites: Evolution, Function, and Opportunities for Drug Development. </span><span class="cited-content_cbyCitation_journal-name">Microbiology and Molecular Biology Reviews</span><span> <strong>2017,</strong> <em>81 </em>
                                    (1)
                                     <a href="https://doi.org/10.1128/MMBR.00047-16" title="DOI URL">https://doi.org/10.1128/MMBR.00047-16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/MMBR.00047-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FMMBR.00047-16%26sid%3Dliteratum%253Aachs%26jtitle%3DMicrobiology%2520and%2520Molecular%2520Biology%2520Reviews%26atitle%3DBromodomains%252Bin%252BProtozoan%252BParasites%25253A%252BEvolution%25252C%252BFunction%25252C%252Band%252BOpportunities%252Bfor%252BDrug%252BDevelopment%26aulast%3DJeffers%26aufirst%3DVictoria%26date%3D2017%26volume%3D81%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Forest H.  Andrews</span>, <span class="hlFld-ContribAuthor ">Alok R.  Singh</span>, <span class="hlFld-ContribAuthor ">Shweta  Joshi</span>, <span class="hlFld-ContribAuthor ">Cassandra A.  Smith</span>, <span class="hlFld-ContribAuthor ">Guillermo A.  Morales</span>, <span class="hlFld-ContribAuthor ">Joseph R.  Garlich</span>, <span class="hlFld-ContribAuthor ">Donald L.  Durden</span>, <span class="hlFld-ContribAuthor ">Tatiana G.  Kutateladze</span>. </span><span class="cited-content_cbyCitation_article-title">Dual-activity PI3K–BRD4 inhibitor for the orthogonal inhibition of MYC to block tumor growth and metastasis. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2017,</strong> <em>114 </em>
                                    (7)
                                     , E1072-E1080. <a href="https://doi.org/10.1073/pnas.1613091114" title="DOI URL">https://doi.org/10.1073/pnas.1613091114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1613091114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1613091114%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DDual-activity%252BPI3K%2525E2%252580%252593BRD4%252Binhibitor%252Bfor%252Bthe%252Borthogonal%252Binhibition%252Bof%252BMYC%252Bto%252Bblock%252Btumor%252Bgrowth%252Band%252Bmetastasis%26aulast%3DAndrews%26aufirst%3DForest%2BH.%26date%3D2017%26date%3D2017%26volume%3D114%26issue%3D7%26spage%3DE1072%26epage%3DE1080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.M.  Woster</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Therapeutics Targeting Epigenetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 297-328. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12325-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12325-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12325-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12325-X%26sid%3Dliteratum%253Aachs%26atitle%3DNovel%252BTherapeutics%252BTargeting%252BEpigenetics%26aulast%3DWoster%26aufirst%3DP.M.%26date%3D2017%26spage%3D297%26epage%3D328%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunyan  Cheng</span>, <span class="hlFld-ContribAuthor ">Hongjuan  Diao</span>, <span class="hlFld-ContribAuthor ">Fan  Zhang</span>, <span class="hlFld-ContribAuthor ">Yongheng  Wang</span>, <span class="hlFld-ContribAuthor ">Kai  Wang</span>, <span class="hlFld-ContribAuthor ">Ruibo  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Deciphering the mechanisms of selective inhibition for the tandem BD1/BD2 in the BET-bromodomain family. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2017,</strong> <em>19 </em>
                                    (35)
                                     , 23934-23941. <a href="https://doi.org/10.1039/C7CP04608A" title="DOI URL">https://doi.org/10.1039/C7CP04608A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7CP04608A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7CP04608A%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DDeciphering%252Bthe%252Bmechanisms%252Bof%252Bselective%252Binhibition%252Bfor%252Bthe%252Btandem%252BBD1%25252FBD2%252Bin%252Bthe%252BBET-bromodomain%252Bfamily%26aulast%3DCheng%26aufirst%3DChunyan%26date%3D2017%26date%3D2017%26volume%3D19%26issue%3D35%26spage%3D23934%26epage%3D23941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Peiqi  Wang</span>, <span class="hlFld-ContribAuthor ">Bihui  Zhou</span>, <span class="hlFld-ContribAuthor ">Jianyou  Shi</span>, <span class="hlFld-ContribAuthor ">Jie  Liu</span>, <span class="hlFld-ContribAuthor ">Xiangrong  Li</span>, <span class="hlFld-ContribAuthor ">Limei  Fan</span>, <span class="hlFld-ContribAuthor ">Yaxin  Zheng</span>, <span class="hlFld-ContribAuthor ">Liang  Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of 4,5-dihydro-benzodiazepinone derivatives as a new chemical series of BRD4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>121 </em>, 294-299. <a href="https://doi.org/10.1016/j.ejmech.2016.05.057" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.05.057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.05.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.05.057%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252B4%25252C5-dihydro-benzodiazepinone%252Bderivatives%252Bas%252Ba%252Bnew%252Bchemical%252Bseries%252Bof%252BBRD4%252Binhibitors%26aulast%3DLi%26aufirst%3DJie%26date%3D2016%26volume%3D121%26spage%3D294%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joeva J.  Barrow</span>, <span class="hlFld-ContribAuthor ">Eduardo  Balsa</span>, <span class="hlFld-ContribAuthor ">Francisco  Verdeguer</span>, <span class="hlFld-ContribAuthor ">Clint D.J.  Tavares</span>, <span class="hlFld-ContribAuthor ">Meghan S.  Soustek</span>, <span class="hlFld-ContribAuthor ">Louis R.  Hollingsworth</span>, <span class="hlFld-ContribAuthor ">Mark  Jedrychowski</span>, <span class="hlFld-ContribAuthor ">Rutger  Vogel</span>, <span class="hlFld-ContribAuthor ">Joao A.  Paulo</span>, <span class="hlFld-ContribAuthor ">Jan  Smeitink</span>, <span class="hlFld-ContribAuthor ">Steve P.  Gygi</span>, <span class="hlFld-ContribAuthor ">John  Doench</span>, <span class="hlFld-ContribAuthor ">David E.  Root</span>, <span class="hlFld-ContribAuthor ">Pere  Puigserver</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain Inhibitors Correct Bioenergetic Deficiency Caused by Mitochondrial Disease Complex I Mutations. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2016,</strong> <em>64 </em>
                                    (1)
                                     , 163-175. <a href="https://doi.org/10.1016/j.molcel.2016.08.023" title="DOI URL">https://doi.org/10.1016/j.molcel.2016.08.023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2016.08.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2016.08.023%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DBromodomain%252BInhibitors%252BCorrect%252BBioenergetic%252BDeficiency%252BCaused%252Bby%252BMitochondrial%252BDisease%252BComplex%252BI%252BMutations%26aulast%3DBarrow%26aufirst%3DJoeva%25C2%25A0J.%26date%3D2016%26volume%3D64%26issue%3D1%26spage%3D163%26epage%3D175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carles  Galdeano</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (13)
                                     , 1655-1680. <a href="https://doi.org/10.4155/fmc-2016-0059" title="DOI URL">https://doi.org/10.4155/fmc-2016-0059</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0059%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSelectivity%252Bon-target%252Bof%252Bbromodomain%252Bchemical%252Bprobes%252Bby%252Bstructure-guided%252Bmedicinal%252Bchemistry%252Band%252Bchemical%252Bbiology%26aulast%3DGaldeano%26aufirst%3DCarles%26date%3D2016%26volume%3D8%26issue%3D13%26spage%3D1655%26epage%3D1680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yue  Zhao</span>, <span class="hlFld-ContribAuthor ">Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Pankaj  Acharya</span>, <span class="hlFld-ContribAuthor ">Kristy R.  Stengel</span>, <span class="hlFld-ContribAuthor ">Quanhu  Sheng</span>, <span class="hlFld-ContribAuthor ">Xiaofan  Zhou</span>, <span class="hlFld-ContribAuthor ">Hojoong  Kwak</span>, <span class="hlFld-ContribAuthor ">Melissa A.  Fischer</span>, <span class="hlFld-ContribAuthor ">James E.  Bradner</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Strickland</span>, <span class="hlFld-ContribAuthor ">Sanjay R.  Mohan</span>, <span class="hlFld-ContribAuthor ">Michael R.  Savona</span>, <span class="hlFld-ContribAuthor ">Bryan J.  Venters</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>, <span class="hlFld-ContribAuthor ">John T.  Lis</span>, <span class="hlFld-ContribAuthor ">Scott W.  Hiebert</span>. </span><span class="cited-content_cbyCitation_article-title">High-Resolution Mapping of RNA Polymerases Identifies Mechanisms of Sensitivity and Resistance to BET Inhibitors in t(8;21) AML. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2016,</strong> <em>16 </em>
                                    (7)
                                     , 2003-2016. <a href="https://doi.org/10.1016/j.celrep.2016.07.032" title="DOI URL">https://doi.org/10.1016/j.celrep.2016.07.032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2016.07.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2016.07.032%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DHigh-Resolution%252BMapping%252Bof%252BRNA%252BPolymerases%252BIdentifies%252BMechanisms%252Bof%252BSensitivity%252Band%252BResistance%252Bto%252BBET%252BInhibitors%252Bin%252Bt%2525288%25253B21%252529%252BAML%26aulast%3DZhao%26aufirst%3DYue%26date%3D2016%26volume%3D16%26issue%3D7%26spage%3D2003%26epage%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jun-lan  Yu</span>, <span class="hlFld-ContribAuthor ">Tian-tian  Chen</span>, <span class="hlFld-ContribAuthor ">Chen  Zhou</span>, <span class="hlFld-ContribAuthor ">Fu-lin  Lian</span>, <span class="hlFld-ContribAuthor ">Xu-long  Tang</span>, <span class="hlFld-ContribAuthor ">Yi  Wen</span>, <span class="hlFld-ContribAuthor ">Jing-kang  Shen</span>, <span class="hlFld-ContribAuthor ">Ye-chun  Xu</span>, <span class="hlFld-ContribAuthor ">Bing  Xiong</span>, <span class="hlFld-ContribAuthor ">Nai-xia  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">NMR-based platform for fragment-based lead discovery used in screening BRD4-targeted compounds. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2016,</strong> <em>37 </em>
                                    (7)
                                     , 984-993. <a href="https://doi.org/10.1038/aps.2016.19" title="DOI URL">https://doi.org/10.1038/aps.2016.19</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/aps.2016.19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Faps.2016.19%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DNMR-based%252Bplatform%252Bfor%252Bfragment-based%252Blead%252Bdiscovery%252Bused%252Bin%252Bscreening%252BBRD4-targeted%252Bcompounds%26aulast%3DYu%26aufirst%3DJun-lan%26date%3D2016%26date%3D2016%26volume%3D37%26issue%3D7%26spage%3D984%26epage%3D993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena  Ferri</span>, <span class="hlFld-ContribAuthor ">Carlo  Petosa</span>, <span class="hlFld-ContribAuthor ">Charles E.  McKenna</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomains: Structure, function and pharmacology of inhibition. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2016,</strong> <em>106 </em>, 1-18. <a href="https://doi.org/10.1016/j.bcp.2015.12.005" title="DOI URL">https://doi.org/10.1016/j.bcp.2015.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2015.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2015.12.005%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DBromodomains%25253A%252BStructure%25252C%252Bfunction%252Band%252Bpharmacology%252Bof%252Binhibition%26aulast%3DFerri%26aufirst%3DElena%26date%3D2016%26volume%3D106%26spage%3D1%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalie H.  Theodoulou</span>, <span class="hlFld-ContribAuthor ">Nicholas C. O.  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Philip G.  Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Progress in the Development of non-BET Bromodomain Chemical Probes. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (5)
                                     , 477-487. <a href="https://doi.org/10.1002/cmdc.201500540" title="DOI URL">https://doi.org/10.1002/cmdc.201500540</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DProgress%252Bin%252Bthe%252BDevelopment%252Bof%252Bnon-BET%252BBromodomain%252BChemical%252BProbes%26aulast%3DTheodoulou%26aufirst%3DNatalie%2BH.%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D5%26spage%3D477%26epage%3D487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.G.  Smith</span>, <span class="hlFld-ContribAuthor ">M.-M.  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">The Bromodomain as an Acetyl-Lysine Reader Domain. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 97-111. <a href="https://doi.org/10.1016/B978-0-12-802389-1.00005-8" title="DOI URL">https://doi.org/10.1016/B978-0-12-802389-1.00005-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802389-1.00005-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802389-1.00005-8%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BBromodomain%252Bas%252Ban%252BAcetyl-Lysine%252BReader%2525C2%2525A0Domain%26aulast%3DSmith%26aufirst%3DS.G.%26date%3D2016%26spage%3D97%26epage%3D111%26pub%3DElsevier%26atitle%3DChromatin%252BSignaling%252Band%252BDiseases%26date%3D2016%26volume%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Estelle  Léonard</span>, <span class="hlFld-ContribAuthor ">Floriane  Mangin</span>, <span class="hlFld-ContribAuthor ">Carole  Villette</span>, <span class="hlFld-ContribAuthor ">Muriel  Billamboz</span>, <span class="hlFld-ContribAuthor ">Christophe  Len</span>. </span><span class="cited-content_cbyCitation_article-title">Azobenzenes and catalysis. </span><span class="cited-content_cbyCitation_journal-name">Catalysis Science & Technology</span><span> <strong>2016,</strong> <em>6 </em>
                                    (2)
                                     , 379-398. <a href="https://doi.org/10.1039/C4CY01597E" title="DOI URL">https://doi.org/10.1039/C4CY01597E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4CY01597E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4CY01597E%26sid%3Dliteratum%253Aachs%26jtitle%3DCatalysis%2520Science%2520%2526%2520Technology%26atitle%3DAzobenzenes%252Band%252Bcatalysis%26aulast%3DL%25C3%25A9onard%26aufirst%3DEstelle%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D2%26spage%3D379%26epage%3D398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weilong  Yao</span>, <span class="hlFld-ContribAuthor ">Ping  Yue</span>, <span class="hlFld-ContribAuthor ">Fadlo R.  Khuri</span>, <span class="hlFld-ContribAuthor ">Shi-Yong  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">The BET bromodomain inhibitor, JQ1, facilitates c-FLIP degradation and enhances TRAIL-induced apoptosis independent of BRD4 and c-Myc inhibition. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2015,</strong> <em>6 </em>
                                    (33)
                                     , 34669-34679. <a href="https://doi.org/10.18632/oncotarget.5785" title="DOI URL">https://doi.org/10.18632/oncotarget.5785</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.5785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.5785%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DThe%252BBET%252Bbromodomain%252Binhibitor%25252C%252BJQ1%25252C%252Bfacilitates%252Bc-FLIP%252Bdegradation%252Band%252Benhances%252BTRAIL-induced%252Bapoptosis%252Bindependent%252Bof%252BBRD4%252Band%252Bc-Myc%252Binhibition%26aulast%3DYao%26aufirst%3DWeilong%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D33%26spage%3D34669%26epage%3D34679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Minoru  Tanaka</span>, <span class="hlFld-ContribAuthor ">Justin M  Roberts</span>, <span class="hlFld-ContribAuthor ">Jun  Qi</span>, <span class="hlFld-ContribAuthor ">James E  Bradner</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of emerging epigenetic targets for cancer therapy: a patent review (2010–2014). </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutical Patent Analyst</span><span> <strong>2015,</strong> <em>4 </em>
                                    (4)
                                     , 261-284. <a href="https://doi.org/10.4155/ppa.15.16" title="DOI URL">https://doi.org/10.4155/ppa.15.16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/ppa.15.16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Fppa.15.16%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutical%2520Patent%2520Analyst%26atitle%3DInhibitors%252Bof%252Bemerging%252Bepigenetic%252Btargets%252Bfor%252Bcancer%252Btherapy%25253A%252Ba%252Bpatent%252Breview%252B%2525282010%2525E2%252580%2525932014%252529%26aulast%3DTanaka%26aufirst%3DMinoru%26date%3D2015%26volume%3D4%26issue%3D4%26spage%3D261%26epage%3D284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marie  Jung</span>, <span class="hlFld-ContribAuthor ">Kathy A  Gelato</span>, <span class="hlFld-ContribAuthor ">Amaury  Fernández-Montalván</span>, <span class="hlFld-ContribAuthor ">Stephan  Siegel</span>, <span class="hlFld-ContribAuthor ">Bernard  Haendler</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting BET bromodomains for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (3)
                                     , 487-501. <a href="https://doi.org/10.2217/epi.14.91" title="DOI URL">https://doi.org/10.2217/epi.14.91</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi.14.91&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi.14.91%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DTargeting%252BBET%252Bbromodomains%252Bfor%252Bcancer%252Btreatment%26aulast%3DJung%26aufirst%3DMarie%26date%3D2015%26volume%3D7%26issue%3D3%26spage%3D487%26epage%3D501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael C.  Hewitt</span>, <span class="hlFld-ContribAuthor ">Yves  Leblanc</span>, <span class="hlFld-ContribAuthor ">Victor S.  Gehling</span>, <span class="hlFld-ContribAuthor ">Rishi G.  Vaswani</span>, <span class="hlFld-ContribAuthor ">Alexandre  Côté</span>, <span class="hlFld-ContribAuthor ">Christopher G.  Nasveschuk</span>, <span class="hlFld-ContribAuthor ">Alexander M.  Taylor</span>, <span class="hlFld-ContribAuthor ">Jean-Christophe  Harmange</span>, <span class="hlFld-ContribAuthor ">James E.  Audia</span>, <span class="hlFld-ContribAuthor ">Eneida  Pardo</span>, <span class="hlFld-ContribAuthor ">Rich  Cummings</span>, <span class="hlFld-ContribAuthor ">Shivangi  Joshi</span>, <span class="hlFld-ContribAuthor ">Peter  Sandy</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Mertz</span>, <span class="hlFld-ContribAuthor ">Robert J.  Sims</span>, <span class="hlFld-ContribAuthor ">Louise  Bergeron</span>, <span class="hlFld-ContribAuthor ">Barbara M.  Bryant</span>, <span class="hlFld-ContribAuthor ">Steve  Bellon</span>, <span class="hlFld-ContribAuthor ">Florence  Poy</span>, <span class="hlFld-ContribAuthor ">Hariharan  Jayaram</span>, <span class="hlFld-ContribAuthor ">Yong  Tang</span>, <span class="hlFld-ContribAuthor ">Brian K.  Albrecht</span>. </span><span class="cited-content_cbyCitation_article-title">Development of methyl isoxazoleazepines as inhibitors of BET. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2015,</strong> <em>25 </em>
                                    (9)
                                     , 1842-1848. <a href="https://doi.org/10.1016/j.bmcl.2015.03.045" title="DOI URL">https://doi.org/10.1016/j.bmcl.2015.03.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2015.03.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2015.03.045%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDevelopment%252Bof%252Bmethyl%252Bisoxazoleazepines%252Bas%252Binhibitors%252Bof%252BBET%26aulast%3DHewitt%26aufirst%3DMichael%2BC.%26date%3D2015%26volume%3D25%26issue%3D9%26spage%3D1842%26epage%3D1848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Karthik  Dhananjayan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Docking Study Characterization of Rare Flavonoids at the Nac-Binding Site of the First Bromodomain of BRD4 (BRD4 BD1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Cancer Research</span><span> <strong>2015,</strong> <em>2015 </em>, 1-15. <a href="https://doi.org/10.1155/2015/762716" title="DOI URL">https://doi.org/10.1155/2015/762716</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2015/762716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2015%2F762716%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cancer%2520Research%26atitle%3DMolecular%252BDocking%252BStudy%252BCharacterization%252Bof%252BRare%252BFlavonoids%252Bat%252Bthe%252BNac-Binding%252BSite%252Bof%252Bthe%252BFirst%252BBromodomain%252Bof%252BBRD4%252B%252528BRD4%252BBD1%252529%26aulast%3DDhananjayan%26aufirst%3DKarthik%26date%3D2015%26volume%3D2015%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stuti  Mehta</span>, <span class="hlFld-ContribAuthor ">Kate L  Jeffrey</span>. </span><span class="cited-content_cbyCitation_article-title">Beyond receptors and signaling: epigenetic factors in the regulation of innate immunity. </span><span class="cited-content_cbyCitation_journal-name">Immunology & Cell Biology</span><span> <strong>2015,</strong> <em>93 </em>
                                    (3)
                                     , 233-244. <a href="https://doi.org/10.1038/icb.2014.101" title="DOI URL">https://doi.org/10.1038/icb.2014.101</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/icb.2014.101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Ficb.2014.101%26sid%3Dliteratum%253Aachs%26jtitle%3DImmunology%2520%2526%2520Cell%2520Biology%26atitle%3DBeyond%252Breceptors%252Band%252Bsignaling%25253A%252Bepigenetic%252Bfactors%252Bin%252Bthe%252Bregulation%252Bof%252Binnate%252Bimmunity%26aulast%3DMehta%26aufirst%3DStuti%26date%3D2015%26date%3D2015%26volume%3D93%26issue%3D3%26spage%3D233%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven G.  Smith</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">The Bromodomain as the Acetyl-Lysine Binding Domain in Gene Transcription. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 1-26. <a href="https://doi.org/10.1007/978-3-319-18102-8_1" title="DOI URL">https://doi.org/10.1007/978-3-319-18102-8_1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-18102-8_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-18102-8_1%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BBromodomain%252Bas%252Bthe%252BAcetyl-Lysine%252BBinding%252BDomain%252Bin%252BGene%252BTranscription%26aulast%3DSmith%26aufirst%3DSteven%2BG.%26date%3D2015%26date%3D2015%26spage%3D1%26epage%3D26%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DHistone%252BRecognition%26aulast%3DZhou%26aufirst%3DMing-Ming%26date%3D2015%26volume%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David S.  Hewings</span>, <span class="hlFld-ContribAuthor ">Timothy P.C.  Rooney</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging Epigenetic Therapies—Bromodomain Ligands. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 495-524. <a href="https://doi.org/10.1016/B978-0-12-800206-3.00022-7" title="DOI URL">https://doi.org/10.1016/B978-0-12-800206-3.00022-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-800206-3.00022-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-800206-3.00022-7%26sid%3Dliteratum%253Aachs%26atitle%3DEmerging%252BEpigenetic%252BTherapies%2525E2%252580%252594Bromodomain%252BLigands%26aulast%3DHewings%26aufirst%3DDavid%2BS.%26date%3D2015%26spage%3D495%26epage%3D524%26pub%3DElsevier%26atitle%3DEpigenetic%252BCancer%252BTherapy%26date%3D2015%26volume%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Ran</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Kejun  Liu</span>, <span class="hlFld-ContribAuthor ">Yi  Lu</span>, <span class="hlFld-ContribAuthor ">Huifang  Li</span>, <span class="hlFld-ContribAuthor ">Jinxing  Xu</span>, <span class="hlFld-ContribAuthor ">Xiao  Xiong</span>, <span class="hlFld-ContribAuthor ">Yanmin  Zhang</span>, <span class="hlFld-ContribAuthor ">Anyang  Xu</span>, <span class="hlFld-ContribAuthor ">Shuai  Lu</span>, <span class="hlFld-ContribAuthor ">Haichun  Liu</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Yadong  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Insight into the key interactions of bromodomain inhibitors based on molecular docking, interaction fingerprinting, molecular dynamics and binding free energy calculation. </span><span class="cited-content_cbyCitation_journal-name">Molecular BioSystems</span><span> <strong>2015,</strong> <em>11 </em>
                                    (5)
                                     , 1295-1304. <a href="https://doi.org/10.1039/C4MB00723A" title="DOI URL">https://doi.org/10.1039/C4MB00723A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MB00723A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MB00723A%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520BioSystems%26atitle%3DInsight%252Binto%252Bthe%252Bkey%252Binteractions%252Bof%252Bbromodomain%252Binhibitors%252Bbased%252Bon%252Bmolecular%252Bdocking%25252C%252Binteraction%252Bfingerprinting%25252C%252Bmolecular%252Bdynamics%252Band%252Bbinding%252Bfree%252Benergy%252Bcalculation%26aulast%3DRan%26aufirst%3DTing%26date%3D2015%26date%3D2015%26volume%3D11%26issue%3D5%26spage%3D1295%26epage%3D1304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isidro  Cortés-Ciriano</span>, <span class="hlFld-ContribAuthor ">Qurrat Ul  Ain</span>, <span class="hlFld-ContribAuthor ">Vigneshwari  Subramanian</span>, <span class="hlFld-ContribAuthor ">Eelke B.  Lenselink</span>, <span class="hlFld-ContribAuthor ">Oscar  Méndez-Lucio</span>, <span class="hlFld-ContribAuthor ">Adriaan P.  IJzerman</span>, <span class="hlFld-ContribAuthor ">Gerd  Wohlfahrt</span>, <span class="hlFld-ContribAuthor ">Peteris  Prusis</span>, <span class="hlFld-ContribAuthor ">Thérèse E.  Malliavin</span>, <span class="hlFld-ContribAuthor ">Gerard J. P.  van Westen</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Polypharmacology modelling using proteochemometrics (PCM): recent methodological developments, applications to target families, and future prospects. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (1)
                                     , 24-50. <a href="https://doi.org/10.1039/C4MD00216D" title="DOI URL">https://doi.org/10.1039/C4MD00216D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00216D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00216D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DPolypharmacology%252Bmodelling%252Busing%252Bproteochemometrics%252B%252528PCM%252529%25253A%252Brecent%252Bmethodological%252Bdevelopments%25252C%252Bapplications%252Bto%252Btarget%252Bfamilies%25252C%252Band%252Bfuture%252Bprospects%26aulast%3DCort%25C3%25A9s-Ciriano%26aufirst%3DIsidro%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D1%26spage%3D24%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raffaella  Di Micco</span>, <span class="hlFld-ContribAuthor ">Barbara  Fontanals-Cirera</span>, <span class="hlFld-ContribAuthor ">Vivien  Low</span>, <span class="hlFld-ContribAuthor ">Panagiotis  Ntziachristos</span>, <span class="hlFld-ContribAuthor ">Stephanie K.  Yuen</span>, <span class="hlFld-ContribAuthor ">Claudia D.  Lovell</span>, <span class="hlFld-ContribAuthor ">Igor  Dolgalev</span>, <span class="hlFld-ContribAuthor ">Yoshiya  Yonekubo</span>, <span class="hlFld-ContribAuthor ">Guangtao  Zhang</span>, <span class="hlFld-ContribAuthor ">Elena  Rusinova</span>, <span class="hlFld-ContribAuthor ">Guillermo  Gerona-Navarro</span>, <span class="hlFld-ContribAuthor ">Marta  Cañamero</span>, <span class="hlFld-ContribAuthor ">Michael  Ohlmeyer</span>, <span class="hlFld-ContribAuthor ">Iannis  Aifantis</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>, <span class="hlFld-ContribAuthor ">Aristotelis  Tsirigos</span>, <span class="hlFld-ContribAuthor ">Eva  Hernando</span>. </span><span class="cited-content_cbyCitation_article-title">Control of Embryonic Stem Cell Identity by BRD4-Dependent Transcriptional Elongation of Super-Enhancer-Associated Pluripotency Genes. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2014,</strong> <em>9 </em>
                                    (1)
                                     , 234-247. <a href="https://doi.org/10.1016/j.celrep.2014.08.055" title="DOI URL">https://doi.org/10.1016/j.celrep.2014.08.055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2014.08.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2014.08.055%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DControl%252Bof%252BEmbryonic%252BStem%252BCell%252BIdentity%252Bby%252BBRD4-Dependent%252BTranscriptional%252BElongation%252Bof%252BSuper-Enhancer-Associated%252BPluripotency%252BGenes%26aulast%3DDi%25C2%25A0Micco%26aufirst%3DRaffaella%26date%3D2014%26volume%3D9%26issue%3D1%26spage%3D234%26epage%3D247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitrios  Spiliotopoulos</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Dynamics Simulations of Bromodomains Reveal Binding-Site Flexibility and Multiple Binding Modes of the Natural Ligand Acetyl-Lysine. </span><span class="cited-content_cbyCitation_journal-name">Israel Journal of Chemistry</span><span> <strong>2014,</strong> <em>54 </em>
                                    (8-9)
                                     , 1084-1092. <a href="https://doi.org/10.1002/ijch.201400009" title="DOI URL">https://doi.org/10.1002/ijch.201400009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ijch.201400009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fijch.201400009%26sid%3Dliteratum%253Aachs%26jtitle%3DIsrael%2520Journal%2520of%2520Chemistry%26atitle%3DMolecular%252BDynamics%252BSimulations%252Bof%252BBromodomains%252BReveal%252BBinding-Site%252BFlexibility%252Band%252BMultiple%252BBinding%252BModes%252Bof%252Bthe%252BNatural%252BLigand%252BAcetyl-Lysine%26aulast%3DSpiliotopoulos%26aufirst%3DDimitrios%26date%3D2014%26date%3D2014%26volume%3D54%26issue%3D8-9%26spage%3D1084%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roberto  Sanchez</span>, <span class="hlFld-ContribAuthor ">Jamel  Meslamani</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">The bromodomain: From epigenome reader to druggable target. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</span><span> <strong>2014,</strong> <em>1839 </em>
                                    (8)
                                     , 676-685. <a href="https://doi.org/10.1016/j.bbagrm.2014.03.011" title="DOI URL">https://doi.org/10.1016/j.bbagrm.2014.03.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagrm.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagrm.2014.03.011%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Gene%2520Regulatory%2520Mechanisms%26atitle%3DThe%252Bbromodomain%25253A%252BFrom%252Bepigenome%252Breader%252Bto%252Bdruggable%252Btarget%26aulast%3DSanchez%26aufirst%3DRoberto%26date%3D2014%26volume%3D1839%26issue%3D8%26spage%3D676%26epage%3D685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mar  Gacias</span>, <span class="hlFld-ContribAuthor ">Guillermo  Gerona-Navarro</span>, <span class="hlFld-ContribAuthor ">Alexander N.  Plotnikov</span>, <span class="hlFld-ContribAuthor ">Guangtao  Zhang</span>, <span class="hlFld-ContribAuthor ">Lei  Zeng</span>, <span class="hlFld-ContribAuthor ">Jasbir  Kaur</span>, <span class="hlFld-ContribAuthor ">Gregory  Moy</span>, <span class="hlFld-ContribAuthor ">Elena  Rusinova</span>, <span class="hlFld-ContribAuthor ">Yoel  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Bridget  Matikainen</span>, <span class="hlFld-ContribAuthor ">Adam  Vincek</span>, <span class="hlFld-ContribAuthor ">Jennifer  Joshua</span>, <span class="hlFld-ContribAuthor ">Patrizia  Casaccia</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biology</span><span> <strong>2014,</strong> <em>21 </em>
                                    (7)
                                     , 841-854. <a href="https://doi.org/10.1016/j.chembiol.2014.05.009" title="DOI URL">https://doi.org/10.1016/j.chembiol.2014.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2014.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biology%26atitle%3DSelective%252BChemical%252BModulation%252Bof%252BGene%252BTranscription%252BFavors%252BOligodendrocyte%252BLineage%252BProgression%26aulast%3DGacias%26aufirst%3DMar%26date%3D2014%26volume%3D21%26issue%3D7%26spage%3D841%26epage%3D854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Panagis  Filippakopoulos</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2014,</strong> <em>13 </em>
                                    (5)
                                     , 337-356. <a href="https://doi.org/10.1038/nrd4286" title="DOI URL">https://doi.org/10.1038/nrd4286</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd4286%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DTargeting%252Bbromodomains%25253A%252Bepigenetic%252Breaders%252Bof%252Blysine%252Bacetylation%26aulast%3DFilippakopoulos%26aufirst%3DPanagis%26date%3D2014%26date%3D2014%26volume%3D13%26issue%3D5%26spage%3D337%26epage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure-guided development of diazobenzene-based BrD inhibitors. (A) Three-dimensional solution structure of the BrD inhibitor MS120 (yellow) bound to the CBP BrD (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L84">2L84</a>). The key amino acid residues in the acetyl-lysine binding site are shown in sticks. (B) Structure–activity relationship table illustrating binding affinity of a select number of diazobenzene-based BrD inhibitors to the two BrDs of BRD4. CLogP values were calculated using ChemBioDraw Ultra 12.0.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Molecular basis of the BRD4 BrD1 recognition of the lead diazobenzene-based BrD inhibitors. (A) Three-dimensional crystal structure of MS436 (yellow) bound to BRD4-BrD1, depicted in a ribbon diagram. Key amino acid residues at the acetyl-lysine binding site are shown in sticks. Ligand MS436 is color-coded by atom type. (B) Electrostatic surface representation of BRD4-BrD1 bound to MS436. (C) Superimposition of MS436, MS435, and MS267 when bound to BRD4-BrD1. Only the MS436-bound protein structure is shown. Side chains of the key residues involved in ligand binding are depicted in sticks and color-coded by atom type. The bound water molecules are shown as spheres in magenta. (D–F) Schematic diagrams highlighting key interactions in the BRD4 BrD1 recognition of MS436, MS435, and MS267, respectively. Water molecules are shown in magenta spheres, and hydrogen bonds are drawn in dashed line. The figure was generated using the program LIGPLOT.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Characterization of effects of the diazobenzene BrD inhibitors on BRD4 function in gene transcription. (A) Effects of cell viability on the BrD inhibitors in murine macrophage RAW264.7 cells. Mitochondrial respiration, an indicator of cell viability, was assayed by the mitochondrial-dependent reduction of MTT to formazan. (B) Effects of the BrD inhibitors on LPS-induced IL-6 secretion in murine macrophage RAW264.7 cells. (C) Effects of the BrD inhibitors on LPS-induced NO production in murine macrophage RAW264.7 cells. (D) Estimated efficiency of the four lead BrD inhibitors in the cellular assays as described in panels B and C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Scheme for C Ring Analogues of Diazobenzene</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/medium/jm-2013-01334s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. General Synthesis of a Ring Analogues of Diazobenzene</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-22/jm401334s/production/images/large/jm-2013-01334s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm401334s&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i82">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77418" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77418" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 32 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lgC9baTouwlow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Dhalluin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aggarwal, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">399</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.+M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0ljrEoR1DUMZ8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Sanchez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">The role of human bromodomains in chromatin biology and gene transcription</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">659</span><span class="NLM_x">–</span> <span class="NLM_lpage">665</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2009&pages=659-665&author=R.+Sanchezauthor=M.+M.+Zhou&title=The+role+of+human+bromodomains+in+chromatin+biology+and+gene+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DThe%2520role%2520of%2520human%2520bromodomains%2520in%2520chromatin%2520biology%2520and%2520gene%2520transcription%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2009%26volume%3D12%26spage%3D659%26epage%3D665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkmer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">149</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0ljrEoR1DUMZ8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7346</span><span class="NLM_x">–</span> <span class="NLM_lpage">7359</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lhgJWBOFxpmlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+C.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lhgJWBOFxpmlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lhgJWBOFxpmlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappaport, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sison, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magoon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blatt, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wunderlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johns, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chicas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulloy, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valent, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">524</span><span class="NLM_x">–</span> <span class="NLM_lpage">528</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+W.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lhQb75zbdFXVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%2BW.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Belkina, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">477</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lhQb75zbdFXVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plotnikov, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusinova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerona-Nevarro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moshkina, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshua, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohlmeyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">Down-regulation of NF-kappaB transcriptional activity in HIV-associated kidney disease by BRD4 inhibition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">28840</span><span class="NLM_x">–</span> <span class="NLM_lpage">28851</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=28840-28851&author=G.+Zhangauthor=R.+Liuauthor=Y.+Zhongauthor=A.+N.+Plotnikovauthor=W.+Zhangauthor=L.+Zengauthor=E.+Rusinovaauthor=G.+Gerona-Nevarroauthor=N.+Moshkinaauthor=J.+Joshuaauthor=P.+Y.+Chuangauthor=M.+Ohlmeyerauthor=J.+C.+Heauthor=M.+M.+Zhou&title=Down-regulation+of+NF-kappaB+transcriptional+activity+in+HIV-associated+kidney+disease+by+BRD4+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DGerona-Nevarro%26aufirst%3DG.%26aulast%3DMoshkina%26aufirst%3DN.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DChuang%26aufirst%3DP.%2BY.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DJ.%2BC.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DDown-regulation%2520of%2520NF-kappaB%2520transcriptional%2520activity%2520in%2520HIV-associated%2520kidney%2520disease%2520by%2520BRD4%2520inhibition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D28840%26epage%3D28851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Tang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deguzman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apparsundaram, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuentes, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demartino, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyagi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrido, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hogaboam, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denton, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budd, D. C.</span><span> </span><span class="NLM_article-title">Assessment of Brd4 inhibition in idiopathic pulmonary fibrosis lung fibroblasts and in vivo models of lung fibrosis</span> <span class="citation_source-journal">Am. J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">183</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">479</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.ajpath.2013.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFynur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2013&pages=470-479&author=X.+Tangauthor=R.+Pengauthor=J.+E.+Phillipsauthor=J.+Deguzmanauthor=Y.+Renauthor=S.+Apparsundaramauthor=Q.+Luoauthor=C.+M.+Bauerauthor=M.+E.+Fuentesauthor=J.+A.+Demartinoauthor=G.+Tyagiauthor=R.+Garridoauthor=C.+M.+Hogaboamauthor=C.+P.+Dentonauthor=A.+M.+Holmesauthor=C.+Kitsonauthor=C.+S.+Stevensonauthor=D.+C.+Budd&title=Assessment+of+Brd4+inhibition+in+idiopathic+pulmonary+fibrosis+lung+fibroblasts+and+in+vivo+models+of+lung+fibrosis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Brd4 Inhibition in Idiopathic Pulmonary Fibrosis Lung Fibroblasts and in Vivo Models of Lung Fibrosis</span></div><div class="casAuthors">Tang, Xiaoyan; Peng, Ruoqi; Phillips, Jonathan E.; Deguzman, Jeremy; Ren, Yonglin; Apparsundaram, Subramanium; Luo, Qi; Bauer, Carla M.; Fuentes, Maria E.; DeMartino, Julie A.; Tyagi, Gaurav; Garrido, Rosario; Hogaboam, Cory M.; Denton, Christopher P.; Holmes, Alan M.; Kitson, Christopher; Stevenson, Christopher S.; Budd, David C.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">470-479</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Idiopathic pulmonary fibrosis (IPF) is a chronic lung disease of high unmet medical need.  Although bromodomain (Brd) and extra terminal domain isoforms have recently been implicated in mediating inflammatory and oncol. indications, their roles in lung fibrosis have not been comprehensively assessed.  We investigated the role of Brd on the profibrotic responses of lung fibroblasts (LFs) in patients with rapidly progressing IPF and a mouse bleomycin model of lung fibrosis.  The enhanced migration, proliferation, and IL-6 release obsd. in LFs from patients with rapidly progressing IPF are attenuated by pharmacol. inhibition of Brd4.  These changes are accompanied by enhanced histone H4 lysine5 acetylation and assocn. of Brd4 with genes involved in the profibrotic responses in IPF LFs as demonstrated using chromatin immunopptn. and quant. PCR.  Oral administration of 200 mg/kg per day Brd4 inhibitor JQ1 in a therapeutic dosing regimen substantially attenuated lung fibrosis induced by bleomycin in C57BL/6 mice.  In conclusion, this study shows that the Brd4 inhibitor JQ1, administered in a therapeutic dosage, is capable of inhibiting the profibrotic effects of IPF LFs and attenuates bleomycin-induced lung fibrosis in mice.  These results suggest that Brd4 inhibitors may represent a novel therapy for the treatment of rapidly progressing IPF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeyA6Tfi6PObVg90H21EOLACvtfcHk0lh5YFhMhcmRLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFynur7I&md5=8a7b094630ce44df6cd1b2b9de6cc6ec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2013.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2013.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DJ.%2BE.%26aulast%3DDeguzman%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DApparsundaram%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DBauer%26aufirst%3DC.%2BM.%26aulast%3DFuentes%26aufirst%3DM.%2BE.%26aulast%3DDemartino%26aufirst%3DJ.%2BA.%26aulast%3DTyagi%26aufirst%3DG.%26aulast%3DGarrido%26aufirst%3DR.%26aulast%3DHogaboam%26aufirst%3DC.%2BM.%26aulast%3DDenton%26aufirst%3DC.%2BP.%26aulast%3DHolmes%26aufirst%3DA.%2BM.%26aulast%3DKitson%26aufirst%3DC.%26aulast%3DStevenson%26aufirst%3DC.%2BS.%26aulast%3DBudd%26aufirst%3DD.%2BC.%26atitle%3DAssessment%2520of%2520Brd4%2520inhibition%2520in%2520idiopathic%2520pulmonary%2520fibrosis%2520lung%2520fibroblasts%2520and%2520in%2520vivo%2520models%2520of%2520lung%2520fibrosis%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2013%26volume%3D183%26spage%3D470%26epage%3D479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozato, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. F.</span><span> </span><span class="NLM_article-title">Brd4 coactivates transcriptional activation of NF-kappaB via specific binding to acetylated RelA</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1375</span><span class="NLM_x">–</span> <span class="NLM_lpage">1387</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&author=B.+Huangauthor=X.+D.+Yangauthor=M.+M.+Zhouauthor=K.+Ozatoauthor=L.+F.+Chen&title=Brd4+coactivates+transcriptional+activation+of+NF-kappaB+via+specific+binding+to+acetylated+RelA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0lh5YFhMhcmRLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.%2BD.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBrd4%2520coactivates%2520transcriptional%2520activation%2520of%2520NF-kappaB%2520via%2520specific%2520binding%2520to%2520acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D1375%26epage%3D1387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Schroder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaehlcke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bisgrove, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnolzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, M.</span><span> </span><span class="NLM_article-title">Two-pronged binding with bromodomain-containing protein 4 liberates positive transcription elongation factor b from inactive ribonucleoprotein complexes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">1090</span><span class="NLM_x">–</span> <span class="NLM_lpage">1099</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=1090-1099&author=S.+Schroderauthor=S.+Choauthor=L.+Zengauthor=Q.+Zhangauthor=K.+Kaehlckeauthor=L.+Makauthor=J.+Lauauthor=D.+Bisgroveauthor=M.+Schnolzerauthor=E.+Verdinauthor=M.+M.+Zhouauthor=M.+Ott&title=Two-pronged+binding+with+bromodomain-containing+protein+4+liberates+positive+transcription+elongation+factor+b+from+inactive+ribonucleoprotein+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchroder%26aufirst%3DS.%26aulast%3DCho%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DBisgrove%26aufirst%3DD.%26aulast%3DSchnolzer%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DTwo-pronged%2520binding%2520with%2520bromodomain-containing%2520protein%25204%2520liberates%2520positive%2520transcription%2520elongation%2520factor%2520b%2520from%2520inactive%2520ribonucleoprotein%2520complexes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D1090%26epage%3D1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olcina, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritchie, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammond, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">Optimization of 3,5-dimethylisoxazole derivatives as potent bromodomain ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">3217</span><span class="NLM_x">–</span> <span class="NLM_lpage">3227</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm301588r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3217-3227&author=D.+S.+Hewingsauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=S.+Martinauthor=S.+Picaudauthor=A.+Tumberauthor=C.+Wellsauthor=M.+M.+Olcinaauthor=K.+Freemanauthor=A.+Gillauthor=A.+J.+Ritchieauthor=D.+W.+Sheppardauthor=A.+J.+Russellauthor=E.+M.+Hammondauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=Optimization+of+3%2C5-dimethylisoxazole+derivatives+as+potent+bromodomain+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm301588r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301588r%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DOlcina%26aufirst%3DM.%2BM.%26aulast%3DFreeman%26aufirst%3DK.%26aulast%3DGill%26aufirst%3DA.%26aulast%3DRitchie%26aufirst%3DA.%2BJ.%26aulast%3DSheppard%26aufirst%3DD.%2BW.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DHammond%26aufirst%3DE.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DOptimization%2520of%25203%252C5-dimethylisoxazole%2520derivatives%2520as%2520potent%2520bromodomain%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3217%26epage%3D3227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Seal, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamotte, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donche, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouillot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellibert, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krysa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCleary, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioja, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span> </span><span class="NLM_article-title">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2968</span><span class="NLM_x">–</span> <span class="NLM_lpage">2972</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.bmcl.2012.02.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=22437115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2968-2972&author=J.+Sealauthor=Y.+Lamotteauthor=F.+Doncheauthor=A.+Bouillotauthor=O.+Mirguetauthor=F.+Gellibertauthor=E.+Nicodemeauthor=G.+Krysaauthor=J.+Kirilovskyauthor=S.+Beinkeauthor=S.+McClearyauthor=I.+Riojaauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=L.+Gordonauthor=T.+Lewisauthor=A.+L.+Walkerauthor=L.+Cutlerauthor=D.+Lugoauthor=D.+M.+Wilsonauthor=J.+Witheringtonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Identification+of+a+novel+series+of+BET+family+bromodomain+inhibitors%3A+Binding+mode+and+profile+of+I-BET151+%28GSK1210151A%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span></div><div class="casAuthors">Seal, Jonathan; Lamotte, Yann; Donche, Frederic; Bouillot, Anne; Mirguet, Olivier; Gellibert, Francoise; Nicodeme, Edwige; Krysa, Gael; Kirilovsky, Jorge; Beinke, Soren; McCleary, Scott; Rioja, Inma; Bamborough, Paul; Chung, Chun-Wa; Gordon, Laurie; Lewis, Toni; Walker, Ann L.; Cutler, Leanne; Lugo, David; Wilson, David M.; Witherington, Jason; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2968-2972</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of quinoline isoxazole BET family bromodomain inhibitors are discussed.  Crystallog. is used to illustrate binding modes and rationalize their SAR.  One member, I-BET151 (GSK1210151A), shows good oral bioavailability in both the rat and minipig as well as demonstrating efficient suppression of bacterial induced inflammation and sepsis in a murine in vivo endotoxemia model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJKtxBCr-pRbVg90H21EOLACvtfcHk0lisCoFK5SGNaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D&md5=a8037c89bcb93948d663208bd1ebfd55</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.041%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DIdentification%2520of%2520a%2520novel%2520series%2520of%2520BET%2520family%2520bromodomain%2520inhibitors%253A%2520Binding%2520mode%2520and%2520profile%2520of%2520I-BET151%2520%2528GSK1210151A%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2968%26epage%3D2972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Dawson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giotopoulos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitski, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huthmacher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gudgin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auger, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doehner, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">529</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+P.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lg1Q8bsedno1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diallo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodacre, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seal, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodrow, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 2: Optimization of phenylisoxazole sulfonamides</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">587</span><span class="NLM_x">–</span> <span class="NLM_lpage">596</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201283q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=587-596&author=P.+Bamboroughauthor=H.+Dialloauthor=J.+D.+Goodacreauthor=L.+Gordonauthor=A.+Lewisauthor=J.+T.+Sealauthor=D.+M.+Wilsonauthor=M.+D.+Woodrowauthor=C.+W.+Chung&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+2%3A+Optimization+of+phenylisoxazole+sulfonamides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 2: Optimization of Phenylisoxazole Sulfonamides</span></div><div class="casAuthors">Bamborough, Paul; Diallo, Hawa; Goodacre, Jonathan D.; Gordon, Laurie; Lewis, Antonia; Seal, Jonathan T.; Wilson, David M.; Woodrow, Michael D.; Chung, Chun-wa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">587-596</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic reader modules that regulate gene transcription through their recognition of acetyl-lysine modified histone tails.  Inhibitors of this protein-protein interaction have the potential to modulate multiple diseases as demonstrated by the profound anti-inflammatory and antiproliferative effects of a recently disclosed class of BET compds.  While these compds. were discovered using phenotypic assays, here we present a highly efficient alternative approach to find new chem. templates, exploiting the abundant structural knowledge that exists for this target class.  A Ph di-Me isoxazole chemotype resulting from a focused fragment screen has been rapidly optimized through structure-based design, leading to a sulfonamide series showing anti-inflammatory activity in cellular assays.  This proof-of-principle expt. demonstrates the tractability of the BET family and bromodomain target class to fragment-based hit discovery and structure-based lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr6lsic9qjJrVg90H21EOLACvtfcHk0lgeB5CF2wT-mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF&md5=e73b16b3fddf88b43b037264b000b276</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm201283q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201283q%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DGoodacre%26aufirst%3DJ.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWoodrow%26aufirst%3DM.%2BD.%26aulast%3DChung%26aufirst%3DC.%2BW.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25202%253A%2520Optimization%2520of%2520phenylisoxazole%2520sulfonamides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D587%26epage%3D596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Borah, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mujtaba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karakikes, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moshkina, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerona-Navarro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajjar, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M. M.</span><span> </span><span class="NLM_article-title">A small molecule binding to the coactivator CREB-binding protein blocks apoptosis in cardiomyocytes</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">531</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.chembiol.2010.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=21513889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=531-541&author=J.+C.+Borahauthor=S.+Mujtabaauthor=I.+Karakikesauthor=L.+Zengauthor=M.+Mullerauthor=J.+Patelauthor=N.+Moshkinaauthor=K.+Morohashiauthor=W.+J.+Zhangauthor=G.+Gerona-Navarroauthor=R.+J.+Hajjarauthor=M.+M.+Zhou&title=A+small+molecule+binding+to+the+coactivator+CREB-binding+protein+blocks+apoptosis+in+cardiomyocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Binding to the Coactivator CREB-Binding Protein Blocks Apoptosis in Cardiomyocytes</span></div><div class="casAuthors">Borah, Jagat C.; Mujtaba, Shiraz; Karakikes, Ioannis; Zeng, Lei; Muller, Michaela; Patel, Jigneshkumar; Moshkina, Natasha; Morohashi, Keita; Zhang, Weijia; Gerona-Navarro, Guillermo; Hajjar, Roger J.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-541</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: As a master transcription factor in cellular responses to external stress, tumor suppressor p53 is tightly regulated.  Excessive p53 activity during myocardial ischemia causes irreversible cellular injury and cardiomyocyte death.  P53 activation is dependent on lysine acetylation by the lysine acetyltransferase and transcriptional coactivator CREB-binding protein (CBP) and on acetylation-directed CBP recruitment for p53 target gene expression.  Here, we report a small mol. ischemin, developed with a structure-guided approach to inhibit the acetyl-lysine binding activity of the bromodomain of CBP.  We show that ischemin alters post-translational modifications on p53 and histones, inhibits p53 interaction with CBP and transcriptional activity in cells, and prevents apoptosis in ischemic cardiomyocytes.  Our study suggests small mol. modulation of acetylation-mediated interactions in gene transcription as a new approach to therapeutic interventions of human disorders such as myocardial ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHKOaQAGSYTbVg90H21EOLACvtfcHk0lgeB5CF2wT-mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D&md5=0c1d7500e1fff40bc335d1b10ac3c27a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DBorah%26aufirst%3DJ.%2BC.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DKarakikes%26aufirst%3DI.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DMoshkina%26aufirst%3DN.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DW.%2BJ.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DHajjar%26aufirst%3DR.%2BJ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DA%2520small%2520molecule%2520binding%2520to%2520the%2520coactivator%2520CREB-binding%2520protein%2520blocks%2520apoptosis%2520in%2520cardiomyocytes%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D531%26epage%3D541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Perola, E.</span><span> </span><span class="NLM_article-title">An analysis of the binding efficiencies of drugs and their leads in successful drug discovery programs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2986</span><span class="NLM_x">–</span> <span class="NLM_lpage">2997</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100118x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlWqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2986-2997&author=E.+Perola&title=An+analysis+of+the+binding+efficiencies+of+drugs+and+their+leads+in+successful+drug+discovery+programs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">An Analysis of the Binding Efficiencies of Drugs and their Leads in Successful Drug Discovery Programs</span></div><div class="casAuthors">Perola, Emanuele</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2986-2997</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In order to investigate the evolution of binding efficiency in successful drug discovery programs, a data set of 60 lead/drug pairs with known binding affinities has been compiled and analyzed.  Low-end thresholds for the binding efficiencies of viable leads and drugs have been derived.  On av., the drugs in the set are significantly larger and more potent but have similar lipophilicity relative to their originating leads, suggesting that the ability to maintain low levels of lipophilicity while increasing mol. wt. is one of the keys to a successful drug discovery program.  A no. of examples demonstrate that large increases in binding efficiency from leads to more elaborate drugs sharing the same scaffold can be achieved.  The importance of dissecting a lead structure to identify the most efficient fragments and the option of sacrificing binding efficiency to optimize other properties are discussed, and relevant examples are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRqzYFBuX2Q7Vg90H21EOLACvtfcHk0lhcBhlreDf2EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlWqu7o%253D&md5=a7eaad792f1518d093588951df64d22f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm100118x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100118x%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26atitle%3DAn%2520analysis%2520of%2520the%2520binding%2520efficiencies%2520of%2520drugs%2520and%2520their%2520leads%2520in%2520successful%2520drug%2520discovery%2520programs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2986%26epage%3D2997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Tarcsay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">Contributions of molecular properties to drug promiscuity miniperspective</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1789</span><span class="NLM_x">–</span> <span class="NLM_lpage">1795</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm301514n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1789-1795&author=A.+Tarcsayauthor=G.+M.+Keseru&title=Contributions+of+molecular+properties+to+drug+promiscuity+miniperspective"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm301514n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301514n%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3DA.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DContributions%2520of%2520molecular%2520properties%2520to%2520drug%2520promiscuity%2520miniperspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1789%26epage%3D1795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Hughes, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeCrescenzo, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devraj, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellsworth, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fobian, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibbs, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilles, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krieger-Burke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loesel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wager, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whiteley, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Physiochemical drug properties associated with in vivo toxicological outcomes</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4872</span><span class="NLM_x">–</span> <span class="NLM_lpage">4875</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.bmcl.2008.07.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=18691886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOnsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4872-4875&author=J.+D.+Hughesauthor=J.+Blaggauthor=D.+A.+Priceauthor=S.+Baileyauthor=G.+A.+DeCrescenzoauthor=R.+V.+Devrajauthor=E.+Ellsworthauthor=Y.+M.+Fobianauthor=M.+E.+Gibbsauthor=R.+W.+Gillesauthor=N.+Greeneauthor=E.+Huangauthor=T.+Krieger-Burkeauthor=J.+Loeselauthor=T.+Wagerauthor=L.+Whiteleyauthor=Y.+Zhang&title=Physiochemical+drug+properties+associated+with+in+vivo+toxicological+outcomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Physiochemical drug properties associated with in vivo toxicological outcomes</span></div><div class="casAuthors">Hughes, Jason D.; Blagg, Julian; Price, David A.; Bailey, Simon; DeCrescenzo, Gary A.; Devraj, Rajesh V.; Ellsworth, Edmund; Fobian, Yvette M.; Gibbs, Michael E.; Gilles, Richard W.; Greene, Nigel; Huang, Enoch; Krieger-Burke, Teresa; Loesel, Jens; Wager, Travis; Whiteley, Larry; Zhang, Yao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4872-4875</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Relationships between physicochem. drug properties and toxicity were inferred from a data set consisting of animal in vivo toleration (IVT) studies on 245 preclin. Pfizer compds.; an increased likelihood of toxic events was found for less polar, more lipophilic compds.  This trend held across a wide range of types of toxicity and across a broad swath of chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPMmGEpVF0mbVg90H21EOLACvtfcHk0lhcBhlreDf2EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOnsbjE&md5=80ea5ced359112a363f38b3545cd7646</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.07.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.07.071%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DJ.%2BD.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DBailey%26aufirst%3DS.%26aulast%3DDeCrescenzo%26aufirst%3DG.%2BA.%26aulast%3DDevraj%26aufirst%3DR.%2BV.%26aulast%3DEllsworth%26aufirst%3DE.%26aulast%3DFobian%26aufirst%3DY.%2BM.%26aulast%3DGibbs%26aufirst%3DM.%2BE.%26aulast%3DGilles%26aufirst%3DR.%2BW.%26aulast%3DGreene%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DE.%26aulast%3DKrieger-Burke%26aufirst%3DT.%26aulast%3DLoesel%26aufirst%3DJ.%26aulast%3DWager%26aufirst%3DT.%26aulast%3DWhiteley%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DPhysiochemical%2520drug%2520properties%2520associated%2520with%2520in%2520vivo%2520toxicological%2520outcomes%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4872%26epage%3D4875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Lee, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, N. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, M. S.</span><span> </span><span class="NLM_article-title">NF-kappa B-dependent expression of nitric oxide synthase is required for membrane fusion of chick embryonic myoblasts</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">324</span><span class="NLM_x">, </span> <span class="NLM_fpage">237</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=1997&pages=237-242&author=K.+H.+Leeauthor=D.+G.+Kimauthor=N.+Y.+Shinauthor=W.+K.+Songauthor=H.+Kwonauthor=C.+H.+Chungauthor=M.+S.+Kang&title=NF-kappa+B-dependent+expression+of+nitric+oxide+synthase+is+required+for+membrane+fusion+of+chick+embryonic+myoblasts"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DKim%26aufirst%3DD.%2BG.%26aulast%3DShin%26aufirst%3DN.%2BY.%26aulast%3DSong%26aufirst%3DW.%2BK.%26aulast%3DKwon%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%2BH.%26aulast%3DKang%26aufirst%3DM.%2BS.%26atitle%3DNF-kappa%2520B-dependent%2520expression%2520of%2520nitric%2520oxide%2520synthase%2520is%2520required%2520for%2520membrane%2520fusion%2520of%2520chick%2520embryonic%2520myoblasts%26jtitle%3DBiochem.%2520J.%26date%3D1997%26volume%3D324%26spage%3D237%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Belkina, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikolajczyk, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span> </span><span class="NLM_article-title">BET protein function is required for inflammation: Brd2 genetic disruption and BET inhibitor JQ1 impair mouse macrophage inflammatory responses</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">190</span><span class="NLM_x">, </span> <span class="NLM_fpage">3670</span><span class="NLM_x">–</span> <span class="NLM_lpage">3678</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670-3678&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+protein+function+is+required+for+inflammation%3A+Brd2+genetic+disruption+and+BET+inhibitor+JQ1+impair+mouse+macrophage+inflammatory+responses"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-α while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0lgQUabGDx3kYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520protein%2520function%2520is%2520required%2520for%2520inflammation%253A%2520Brd2%2520genetic%2520disruption%2520and%2520BET%2520inhibitor%2520JQ1%2520impair%2520mouse%2520macrophage%2520inflammatory%2520responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26spage%3D3670%26epage%3D3678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bish, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Primiano, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ralph, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sciammetta, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trzupek, J. D.</span><span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra C-terminal bromodomain inhibition through optimization of a fragment-derived hit</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9831</span><span class="NLM_x">–</span> <span class="NLM_lpage">9837</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+C-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0lgQUabGDx3kYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520C-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Nikolovska-Coleska, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roller, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajewski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0lj8IdtA0oMstQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D261%26epage%3D273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Otwinowski, Z.; Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. In  <span class="citation_source-book">Methods in Enzymology</span>; <span class="NLM_contrib-group">Carter; C. W.,  Jr.</span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=+Carter&author=C.+W.+&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26btitle%3DMethods%2520in%2520Enzymology%26aulast%3DCarter%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A. .</span><span> </span><span class="NLM_article-title">MOLREP: An automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+.+Teplyakov&title=MOLREP%3A+An+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lj8IdtA0oMstQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%2B.%26atitle%3DMOLREP%253A%2520An%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lj8IdtA0oMstQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: Model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lj8IdtA0oMstQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Hunger, K.; Mischke, P.; Rieper, W.</span><span> </span><span class="NLM_article-title">Azo dyes</span>. In  <span class="citation_source-book">Ullmann’s Encyclopedia of Industrial Chemistry</span>; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2000</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1002%2F14356007.a03_245" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=K.+Hunger&author=P.+Mischke&author=W.+Rieper&title=Ullmann%E2%80%99s+Encyclopedia+of+Industrial+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2F14356007.a03_245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F14356007.a03_245%26sid%3Dliteratum%253Aachs%26aulast%3DHunger%26aufirst%3DK.%26atitle%3DAzo%2520dyes%26btitle%3DUllmann%25E2%2580%2599s%2520Encyclopedia%2520of%2520Industrial%2520Chemistry%26pub%3DWiley-VCH%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Zhou, M. M.; Ohlmeyer, M.; Mujtaba, S.; Plotnikov, A.; Kastrinsky, D.; Zhang, G.; Borah, J.</span><span> </span><span class="NLM_article-title">Inhibitors of bromodomains as modulators of gene expression</span>. Patent WO2012116170,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=M.+M.+Zhou&author=M.+Ohlmeyer&author=S.+Mujtaba&author=A.+Plotnikov&author=D.+Kastrinsky&author=G.+Zhang&author=J.+Borah&title=Inhibitors+of+bromodomains+as+modulators+of+gene+expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DInhibitors%2520of%2520bromodomains%2520as%2520modulators%2520of%2520gene%2520expression%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Wallace, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J.</span><span> </span><span class="NLM_article-title">LIGPLOT: A program to generate schematic diagrams of protein-ligand interactions</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm401334s&amp;key=10.1093%2Fprotein%2F8.2.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm401334s&amp;key=7630882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm401334s&amp;key=1%3ACAS%3A528%3ADyaK2MXltVCgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1995&pages=127-134&author=A.+Wallaceauthor=R.+Laskowskiauthor=J.+Thornton&title=LIGPLOT%3A+A+program+to+generate+schematic+diagrams+of+protein-ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions</span></div><div class="casAuthors">Wallace, Andrew C.; Laskowski, Roman A.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">127-34</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The LIGPLOT program automatically generates schematic 2-D representations of protein-ligand complexes from std. Protein Data Bank file input.  The output is a color, or black-and-white, PostScript file giving a simple and informative representation of the intermol. interactions and their strengths, including hydrogen bonds, hydrophobic interactions and atom accessibilities.  The program is completely general for any ligand and can also be used to show other types of interaction in proteins and nucleic acids.  It was designed to facilitate the rapid inspection of many enzyme complexes, but has found many other applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp9SrWw0klVbVg90H21EOLACvtfcHk0lgTQqA9n6R9NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXltVCgtLc%253D&md5=ed74e39b0cd6be4b2b8907d3ff6d9619</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.2.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.2.127%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DA.%26aulast%3DLaskowski%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%26atitle%3DLIGPLOT%253A%2520A%2520program%2520to%2520generate%2520schematic%2520diagrams%2520of%2520protein-ligand%2520interactions%26jtitle%3DProtein%2520Eng.%26date%3D1995%26volume%3D8%26spage%3D127%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L84" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2L84','PDB','2L84'); return false;">PDB: 2L84</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i77"><a href="/doi/suppl/10.1021/jm401334s">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_66528"></div></div></div></div></div><hr /></hr><p class="last">Additional diazobenzene inhibitors and crystallography data and refinement statistics for BRD4–BrD1/MS436, BRD4–BrD1/MS435, and BRD4–BrD1/MS267. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm401334s/suppl_file/jm401334s_si_001.pdf">jm401334s_si_001.pdf (312.87 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm401334s&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-22%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Fjm401334s%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm401334s" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679995a588c13cb8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
