
<html lang="en"     class="pb-page"  data-request-id="f6b6ba8a-584d-4de3-9c93-4098cebaeec4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c00021;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-10"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Optimization of Potent ATAD2 and CECR2 Bromodomain Inhibitors with an Atypical Binding Mode" /></meta><meta name="dc.Creator" content="Simon C. C.  Lucas" /></meta><meta name="dc.Creator" content="Stephen J.  Atkinson" /></meta><meta name="dc.Creator" content="Paul  Bamborough" /></meta><meta name="dc.Creator" content="Heather  Barnett" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Laurie  Gordon" /></meta><meta name="dc.Creator" content="Darren J.  Mitchell" /></meta><meta name="dc.Creator" content="Alexander  Phillipou" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Creator" content="Robert J.  Sheppard" /></meta><meta name="dc.Creator" content="Nicholas C. O.  Tomkinson" /></meta><meta name="dc.Creator" content="Robert J.  Watson" /></meta><meta name="dc.Creator" content="Emmanuel H.  Demont" /></meta><meta name="dc.Description" content="Most bromodomain inhibitors mimic the interactions of the natural acetylated lysine (KAc) histone substrate through key interactions with conserved asparagine and tyrosine residues within the bindi..." /></meta><meta name="Description" content="Most bromodomain inhibitors mimic the interactions of the natural acetylated lysine (KAc) histone substrate through key interactions with conserved asparagine and tyrosine residues within the bindi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 22, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00021" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00021" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00021" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00021" /></link>
        
    
    

<title>Optimization of Potent ATAD2 and CECR2 Bromodomain Inhibitors with an Atypical Binding Mode | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00021" /></meta><meta property="og:title" content="Optimization of Potent ATAD2 and CECR2 Bromodomain Inhibitors with an Atypical Binding Mode" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0027.jpeg" /></meta><meta property="og:description" content="Most bromodomain inhibitors mimic the interactions of the natural acetylated lysine (KAc) histone substrate through key interactions with conserved asparagine and tyrosine residues within the binding pocket. Herein we report the optimization of a series of phenyl sulfonamides that exhibit a novel mode of binding to non-bromodomain and extra terminal domain (non-BET) bromodomains through displacement of a normally conserved network of four water molecules. Starting from an initial hit molecule, we report its divergent optimization toward the ATPase family AAA domain containing 2 (ATAD2) and cat eye syndrome chromosome region, candidate 2 (CECR2) domains. This work concludes with the identification of (R)-55 (GSK232), a highly selective, cellularly penetrant CECR2 inhibitor with excellent physicochemical properties." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00021"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00021">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00021&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00021&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00021&amp;href=/doi/10.1021/acs.jmedchem.0c00021" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 5212-5241</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02153" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00035" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Optimization of Potent ATAD2 and CECR2 Bromodomain Inhibitors with an Atypical Binding Mode</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Simon C. C. Lucas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon C. C. Lucas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div><div class="loa-info-affiliations-info">WestCHEM, Department of Pure and Applied Chemistry, University of Strathclyde, Thomas Graham Building, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon+C.+C.++Lucas">Simon C. C. Lucas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephen J. Atkinson</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen J. Atkinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#88fbfcedf8e0ede6a6e2a6e9fce3e1e6fbe7e6c8effbe3a6ebe7e5"><span class="__cf_email__" data-cfemail="32414657425a575c1c581c5346595b5c415d5c725541591c515d5f">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+J.++Atkinson">Stephen J. Atkinson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3636-3674" title="Orcid link">http://orcid.org/0000-0003-3636-3674</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul Bamborough</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Bamborough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational & Structural Chemistry, Molecular Discovery Research, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Bamborough">Paul Bamborough</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heather Barnett</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heather Barnett</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Flexible Discovery Unit, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heather++Barnett">Heather Barnett</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chun-wa Chung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun-wa Chung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational & Structural Chemistry, Molecular Discovery Research, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2480-3110" title="Orcid link">http://orcid.org/0000-0002-2480-3110</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laurie Gordon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laurie Gordon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biological Sciences, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laurie++Gordon">Laurie Gordon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darren J. Mitchell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darren J. Mitchell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darren+J.++Mitchell">Darren J. Mitchell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexander Phillipou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexander Phillipou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biological Sciences, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexander++Phillipou">Alexander Phillipou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rab K. Prinjha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rab K. Prinjha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert J. Sheppard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Sheppard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Sheppard">Robert J. Sheppard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas C. O. Tomkinson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas C. O. Tomkinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WestCHEM, Department of Pure and Applied Chemistry, University of Strathclyde, Thomas Graham Building, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas+C.+O.++Tomkinson">Nicholas C. O. Tomkinson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5509-0133" title="Orcid link">http://orcid.org/0000-0002-5509-0133</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert J. Watson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Watson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Watson">Robert J. Watson</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Emmanuel H. Demont</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emmanuel H. Demont</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emmanuel+H.++Demont">Emmanuel H. Demont</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7307-3129" title="Orcid link">http://orcid.org/0000-0001-7307-3129</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00021&amp;href=/doi/10.1021%2Facs.jmedchem.0c00021" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 5212–5241</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 22, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 January 2020</li><li><span class="item_label"><b>Published</b> online</span>22 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00021" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00021</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5212%26pageCount%3D30%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSimon%2BC.%2BC.%2BLucas%252C%2BStephen%2BJ.%2BAtkinson%252C%2BPaul%2BBamborough%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D10%26contentID%3Dacs.jmedchem.0c00021%26title%3DOptimization%2Bof%2BPotent%2BATAD2%2Band%2BCECR2%2BBromodomain%2BInhibitors%2Bwith%2Ban%2BAtypical%2BBinding%2BMode%26numPages%3D30%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5241%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00021"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1488</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00021" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Optimization of Potent ATAD2 and CECR2 Bromodomain Inhibitors with an Atypical Binding Mode&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;C. C. Lucas&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;J. Atkinson&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Bamborough&quot;},{&quot;first_name&quot;:&quot;Heather&quot;,&quot;last_name&quot;:&quot;Barnett&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Laurie&quot;,&quot;last_name&quot;:&quot;Gordon&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;J. Mitchell&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;Phillipou&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Sheppard&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;C. O. Tomkinson&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Watson&quot;},{&quot;first_name&quot;:&quot;Emmanuel&quot;,&quot;last_name&quot;:&quot;H. Demont&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;5212-5241&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00021&quot;},&quot;abstract&quot;:&quot;Most bromodomain inhibitors mimic the interactions of the natural acetylated lysine (KAc) histone substrate through key interactions with conserved asparagine and tyrosine residues within the binding pocket. Herein we report the optimization of a series of phenyl sulfonamides that exhibit a novel mode of binding to non-bromodomain and extra terminal domain (non-BET) bromodomains through displacement of a normally conserved network of four water molecules. Starting from an initial hit molecule, we report its divergent optimization toward the ATPase family AAA domain containing 2 (ATAD2) and cat eye syndrome chromosome region, candidate 2 (CECR2) domains. This work concludes with the identification of (R)-55 (GSK232), a highly selective, cellularly penetrant CECR2 inhibitor with excellent physicochemical properties.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00021&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00021" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00021&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00021" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00021&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00021" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00021&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00021&amp;href=/doi/10.1021/acs.jmedchem.0c00021" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00021" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00021" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00021%26sid%3Dliteratum%253Aachs%26pmid%3D32321240%26genre%3Darticle%26aulast%3DLucas%26date%3D2020%26atitle%3DOptimization%2Bof%2BPotent%2BATAD2%2Band%2BCECR2%2BBromodomain%2BInhibitors%2Bwith%2Ban%2BAtypical%2BBinding%2BMode%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D10%26spage%3D5212%26epage%3D5241%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291651" title="Anions">Anions</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290716" title="Sulfones">Sulfones</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/jmcmar.2020.63.issue-10/20200528/jmcmar.2020.63.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Most bromodomain inhibitors mimic the interactions of the natural acetylated lysine (KAc) histone substrate through key interactions with conserved asparagine and tyrosine residues within the binding pocket. Herein we report the optimization of a series of phenyl sulfonamides that exhibit a novel mode of binding to non-bromodomain and extra terminal domain (non-BET) bromodomains through displacement of a normally conserved network of four water molecules. Starting from an initial hit molecule, we report its divergent optimization toward the ATPase family AAA domain containing 2 (ATAD2) and cat eye syndrome chromosome region, candidate 2 (CECR2) domains. This work concludes with the identification of <b>(</b><i><b>R</b></i><b>)-55</b> (GSK232), a highly selective, cellularly penetrant CECR2 inhibitor with excellent physicochemical properties.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24766" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24766" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The bromodomain and extra-terminal domain (BET) family of bromodomain containing proteins is a well-established therapeutic target for small molecule inhibition, leading to efficacy in both oncology and immunoinflammation indications.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> It has been shown that BET proteins organize on superenhancer and enhancer regions and profoundly regulate gene expression relevant to a range of diseases.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Indeed, many small molecule BET bromodomain inhibitors, which block the interaction of BET with histones and chromatin, are currently progressing through clinical trials.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> However, the BET family represents just 8 of the 61 known human bromodomains; therefore, it is logical to propose that other bromodomains may control the expression of genes crucial to other diseases.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> Despite this promising hypothesis, the pharmacological relevance of inhibiting these bromodomains using small molecules is still largely unknown. Therefore, before investment in a full drug discovery program, these targets need to be fully validated.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12 ref13">(8−13)</a> Work in our group has recently been focused on developing tool molecules for the ATPase family AAA domain containing 2 (ATAD2) and cat eye syndrome chromosome region, candidate 2 (CECR2) bromodomains, with a particular focus on novel chemotypes.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a></div><div class="NLM_p">ATAD2 is a promising oncogene that is overexpressed in multiple, unrelated human cancers.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> Protein knockdown experiments have linked ATAD2 to cell survival, proliferation, and migration pathways.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> CECR2 is a bromodomain containing transcription factor that is involved in neurulation through the chromatin remodeling complex SNF2L.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> It has also been implicated in oncology using a functional genomics approach and is involved in the formation of γ-H2AX foci in response to DNA double strand breaks.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a></div><div class="NLM_p">The ATAD2 and CECR2 bromodomains are exciting targets for drug discovery, and consequently, inhibitors for both bromodomains are starting to appear in the literature. Computational modeling had previously predicted the ATAD2 bromodomain to be among the least tractable of the bromodomain family; however, work by our group among others has shown that the optimization of submicromolar inhibitors is possible.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> The first small molecule inhibitor (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) published by our group was naphthyridone <b>1a</b>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Despite showing relatively modest activity compared to the leading BET inhibitors (ATAD2 pIC<sub>50</sub> = 5.9), this was a significant step forward against what was predicted to be one of the least tractable bromodomains.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Naphthyridone <b>1a</b> was optimized to afford <b>1b</b> (GSK8814), a cell-permeable and selective tool molecule.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Recently, an isoform selective ATAD2 inhibitor, <b>2</b> (BAY-850) was published.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> This showed excellent ATAD2 potency and no measurable inhibition of other bromodomains. The compound displayed a unique binding mode whereby it bound to a dimer of ATAD2 as determined by mass spectrometry.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Small molecule inhibitors of ATAD2 and CECR2 with KAc mimetics shown in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There are currently three reported, selective, small molecule inhibitors of CECR2 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The pan-BRD inhibitor bromosporine <b>3</b> is a potent inhibitor of CECR2 with a p<i>K</i><sub>d</sub> of 8.0 but lacks selectivity over other BRDs.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The first selective CECR2 inhibitor was <b>4</b> (NVS-CECR2-1), which is a highly selective tool molecule.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> It has nanomolar activity in an isothermal calorimetry (ITC) assay (p<i>K</i><sub>d</sub> = 7.1) and displays robust activity in cellular fluorescence recovery after photobleaching (FRAP) and nanobioluminescence resonance energy tranfer (nanoBRET) assays. However, the authors describe the compound as being poorly soluble. The poor solubility of this chemotype was addressed through a collaborative effort between the SGC and Takeda which delivered <b>5</b> (TP-238) as a dual CECR2/bromodomain PHD finger transcription factor (BTPF) tool compound.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Crawford et al. then showed that modest selectivity for different BRDs could be achieved by changing the N-substituent of a series of pyrrolopyridines.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> In CECR2 the allyl substituent <b>6a</b> induces a small lipophilic pocket adjacent to the water nework, by moving the gatekeeper Tyr. This was further optimized into <b>6b</b> (GNE-886), which has a CECR2 TR-FRET pIC<sub>50</sub> of 7.8 equating to a 100-fold selectivity window over the closest bromodomains.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The authors demonstrated cellular permeability and target engagement, identifying it as a suitable tool molecule.</div><div class="NLM_p">While tool molecules have been identified for these bromodomains, fully validating any results observed with these tools will rely on the availability of additional molecules that have differentiated chemotypes.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> We recently disclosed the output of a high throughput screen (HTS) that identified a series of phenyl sulfonamides as ATAD2 inhibitors.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Phenyl sulfonamide <b>(</b><i><b>rac</b></i><b>)-7</b> was identified as a suitable starting point for optimization into a chemical tool for both ATAD2 and CECR2 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). It showed activity against ATAD2 (pIC<sub>50</sub> = 6.3) and CECR2 (pIC<sub>50</sub> = 5.6) using TR-FRET assays. As in our earlier work on the naphthyridone series, we moved from a peptide to a ligand displacement TR-FRET assay due to improved assay robustness.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Since the data were concordant (e.g., <b>(</b><i><b>rac</b></i><b>)-7</b>, 6.5 vs 6.3), the ligand displacement assay will be used herein. Importantly, <b>(</b><i><b>rac</b></i><b>)-7</b> showed little affinity for either bromodomain of BRD4 (pIC<sub>50</sub> ≤ 4.7), used in this work as a representitive example of the BET family due to the high sequence homology between family members (for full BET data on <b>(</b><i><b>rac</b></i><b>)-7</b>, <b>(</b><i><b>S,S</b></i><b>)-32</b>, and <b>(</b><i><b>R</b></i><b>)-55</b> see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_001.pdf" class="ext-link">Supporting Information</a>). High BET selectivity (>100-fold) is desirable due to the strong phenotype observed with BET inhibitors. Phenyl sulfonamide <b>(</b><i><b>rac</b></i><b>)-7</b> was in a good physicochemical space as defined by ChromLogD (a chromatographic measure of lipophilicity, measured at 4.8), the property forecast index (PFI (ChromLogD + no. Ar), 5.8), and clogP (2.8).<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> The series had high kinetic solubility as measured by chemiluminescence nitrogen detection (CLND) solubility (93 μg mL<sup>–1</sup>) and good permeability through an artificial lipid membrane (200 nm s<sup>–1</sup>).<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39−41)</a> Charged aerosol detection (CAD) solubility was also used in this work, and an internal analysis has shown that this is equivalent to CLND (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_001.pdf" class="ext-link">Supporting Information</a>).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The aim of this work was to generate cell active chemical probes, and optimization of the physicochemical properties was considered important to maximize the probability of success of good cell permeation. We stipulated targets for solubility of >100 μg mL<sup>–1</sup> and permeability of >100 nm s<sup>–1</sup> with the knowledge that one would likely compensate for the other if both of these were not achievable.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Hit Phenyl Sulfonamide Identified from a HTS against ATAD2</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0011.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center"><b>(</b><i><b>rac</b></i><b>)-7</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATAD2 (peptide) pIC<sub>50</sub> (<i>n</i>)</td><td class="colsep0 rowsep0" align="left">6.5 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATAD2 (ligand) pIC<sub>50</sub> (<i>n</i>)</td><td class="colsep0 rowsep0" align="left">6.3 (5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LE/LLE<sub>at</sub></td><td class="colsep0 rowsep0" align="left">0.25/0.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CECR2 pIC<sub>50</sub> (<i>n</i>)/Sel</td><td class="colsep0 rowsep0" align="left">5.6 (2)/5×</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD1 pIC<sub>50</sub> (<i>n</i>)/Sel</td><td class="colsep0 rowsep0" align="left">4.3 (1)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a>/100×</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2 pIC<sub>50</sub> (<i>n</i>)/Sel</td><td class="colsep0 rowsep0" align="left">4.7 (3)/40×</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChromLogD/PFI/clogP</td><td class="colsep0 rowsep0" align="left">4.8/5.8/2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLND (μg mL<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AMP (nm s<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">200</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 2) test occasions.</p></div></div></div><div class="NLM_p">As discussed previously, the phenyl sulfonamides, such as <b>(</b><i><b>rac</b></i><b>)-8</b>, bind to the bromodomain of ATAD2 in a noncanonical fashion, completely displacing the normally conserved water network from the base of the binding pocket (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The total removal of the water network has been observed for family VIII ligands,<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43−45)</a> which grand canonical Monte Carlo simulations predicted to have the least stable of the BRD water networks.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> ATAD2 was also predicted to have a high energy water network, consistent with our observations that these are displaced by this series. Conversely, the water network in CECR2 was shown to be among the most stable, and so the CECR2 pIC<sub>50</sub> of 5.6 for <b>(</b><i><b>rac</b></i><b>)-8</b> is harder to reconcile. For ATAD2 binding of <b>(</b><i><b>rac</b></i><b>)-8</b>, one of the sulfonamide oxygens replaces the interactions of water W1 and makes a hydrogen bond to Y1021. Additionally, the characteristic 110.5° angle of the sulfonamide positions the piperidine into a position where it is able to make lipophilic contacts with V1018, Y1063, and I1074. Initial SAR studies revealed that a halide <i>ortho</i> to the sulfonamide was optimal for potency, bromide typically being 5-fold more potent than chloride. In the ATAD2 crystal structure, this bromide overlaps with the position of the W2 water in the conserved water network and forms a halogen bond to I1056. Previous quantum mechanical analysis of the binding interactions of this series found a relatively small contribution to the binding energy from the halogen bond, with instead the increase in binding affinity likely due to contibutions from multiple interactions with other nearby residues.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Truncation of the hydantoin back to an ethylamide resulted in a 50-fold reduced potency, suggesting a significant binding contribution from this moiety.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> This can be rationalized by the two binding interactions of the hydantoin carbonyl groups with R1007 and the backbone NH of D1014 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). However, it should be noted that multiple R1007 conformations were observed in different structures, indicating a degree of flexibility for this residue. Furthermore, the hydantoin NH did not appear to pick up any specific interactions with the protein.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) X-ray crystal structure of <b>(</b><i><b>rac</b></i><b>)-8</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55">6S55</a><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a>) overlaid with KAc binding in ATAD2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QSP">4QSP</a><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>). The density fitted the <i>S</i>-enantiomer better, and so this was modeled. The conserved network of waters (W1–W4) at the base of the binding pocket are displaced by the phenyl sulfonamide ligand. (B) Key contacts between the ligand (<b>(</b><i><b>rac</b></i><b>)-8</b>) and ATAD2 are shown as yellow dashes. A halogen bond between the aryl bromide and the backbone carbonyl group of I1056 is present at the base of the KAc recognition site. One of the sulfonamide oxygens makes a H-bond to Y1021, and the two carbonyl groups of the hydantoin make H-bonds to R1007 and D1014. (C) Alternative view of (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The potency of <b>(</b><i><b>rac</b></i><b>)-7</b> against ATAD2 and its selectivity over the BET family (>40-fold), combined with the good physicochemical properties of this molecule and the novel binding mode of the phenyl sulfonamides, made it an attractive starting point for our investigation. Additionally, potency for CECR2 presented the opportunity for divergent optimization. Within this manuscript we report the optimization of <b>(</b><i><b>rac</b></i><b>)-7</b> into potent inhibitors of ATAD2 and CECR2, respectively.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35145" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35145" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Optimization toward an ATAD2 Inhibitor</h3><div class="NLM_p">As previously disclosed during the discovery of this template, preliminary SAR generation efforts had focused on the aryl substituent and the hydantoin vector. The binding contribution of the sulfonamide substituent was not evaluated at this stage.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As a starting point for SAR expansion toward ATAD2 potency and selectivity, piperidine <b>10</b> was either truncated or expanded to the five- and seven-membered analogues (<b>9</b> and <b>11</b>, respectively, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). For synthetic feasibility, these were prepared with a chloroaryl substituent and an unfunctionalized hydantoin. Pyrrolidine <b>9</b> was the most potent (pIC<sub>50</sub> = 6.0) and ligand efficient of the three heterocycles against ATAD2, which was attributed to a good shape complementarity of the substituent with the lipophilic pocket (formed by the BC loop) of the protein. Truncation of the heterocycle to dimethyl sulfonamide <b>12</b> led to a significant decrease in potency (pIC<sub>50</sub> = 4.9) against ATAD2, a trend mirrored in CECR2 (pIC<sub>50</sub> = 4.1). As the pyrrolidine was the optimal size, functionalization of the ring was then investigated. A methyl substituent at the 3- position of the pyrrolidine (<b>13</b>) was tolerated, but no increase in ATAD2 potency or selectivity over CECR2 was observed for either enantiomer. While both enantiomers of the 3-methyl substituent were well tolerated, <i>gem</i>-dimethyl <b>14</b> showed a 0.5 log unit decrease in potency, suggesting the steric demands of this analogue are not accommodated by the binding pocket. Interestingly, cyclopropyl <b>15</b> was comparable with the des-methyl and monomethyl analogues (<b>9</b> and <b>(</b><i><b>S</b></i><b>)-13</b>); however, it had a slightly lower ligand efficiency. Bis-substitution at the 3- and 4-position was then investigated. The <i>trans</i>-configuration (<b>(</b><i><b>rac</b></i><b>)-16</b>) gave a 0.5 log unit increase in potency compared to <b>(</b><i><b>S</b></i><b>)-13</b>. This included a slight increase in LE but a small decrease in LLE<sub>at</sub>. Conversely, the <i>cis</i>-configuration <b>17</b> was poorly tolerated and less potent (pIC<sub>50</sub> = 5.7) than the monomethyl examples. Extending the 3-position substituent to ethyl <b>(</b><i><b>rac</b></i><b>)-18</b> or ether <b>(</b><i><b>rac</b></i><b>)-19</b> was tolerated, but neither offered any improvements in terms of potency or ligand efficiency compared to <b>9</b> and <b>(</b><i><b>S</b></i><b>)-13</b>. Cyclopropyl <b>(</b><i><b>rac</b></i><b>)-20</b> was notable for its increased CECR2 potency vs <b>(</b><i><b>S</b></i><b>)-13</b> and was now equipotent against the two proteins. Substitution at the 2-position to afford pyrrolidine <b>(</b><i><b>R</b></i><b>)-21</b> was less well tolerated, with a drop in pIC<sub>50</sub> to 5.6. The more substantive change of a sulfonamide to a cyclohexyl sulfone (<b>22</b>, entry 7, pIC<sub>50</sub> < 4.0) was not tolerated in accordance with the predicted importance of the directionality confered by the sulfonamide (vide supra). Likewise, sulfonylimidazole <b>23</b> was inactive against ATAD2 and CECR2 at the concentrations tested. All of the examples tested showed no appreciable affinity for BRD4 BD1, maintained a moderate bias for ATAD2 over CECR2, and showed desirable levels of solubility (≥116 μg mL<sup>–1</sup>). While no increases in ATAD2 selectivity were observed, pyrrolidines <b>9</b>, <b>(</b><i><b>S</b></i><b>)-13</b>, and <b>(</b><i><b>rac</b></i><b>)-16</b> were identified as the optimal sulfonamides due to their potency and ligand efficiency.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Exploration of the Sulfonamide Vector</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0013.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0014.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Also tested <4.0 (<i>n</i> = 1) test occasions.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 2) test occasions.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">CAD solubility.</p></div></div><div></div></div><div class="NLM_p">Attention then turned to the amide vector, which directed the hydantoin moiety toward R1007 and D1014 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The HTS had identified the hydantoin as an effective substituent, but it was important to fully explore the SAR of this region to further elicidate the differences in ATAD2 and CECR2 binding (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As before, the aryl chloride was used for synthetic feasibility and the more optimal pyrrolidine sulfonamide was employed. As expected, the unsubstituted hydantoin <b>9</b> showed good potency (pIC<sub>50</sub> = 6.0), though it was characterized by a low artificial membrane permeability (AMP < 3 nm s<sup>–1</sup>). This is most likely due to the low lipophilicity of this molecule. The methyl ((<i><b>rac</b></i>)<b>-24</b>) and dimethyl (<b>25</b>) analogues increased permeability further and maintained a similar ATAD2 potency (pIC<sub>50</sub> = 5.9 and 5.8, respectively). Here, α substitution of the H-bond donor NH increases lipophilicity by 0.4–0.8 log units, allowing it to more easily pass though the hydrophobic portion of membranes. The geminally substituted <b>(</b><i><b>rac</b></i><b>)-26</b> further raised the predicted permeability, achieving a desirable value of 100 nm s<sup>–1</sup>, and also maintained potency at ATAD2. Interestingly, 2,4-oxazolidinedione <b>27</b> (pIC<sub>50</sub> = 5.9) was also well tolerated, supporting the crystallographic observation that the hydantoin NH was not making any specific interactions. This was further supported by pyrrolidinedione <b>28</b>, which also maintained a similar potency of 5.9 (cf. hydantoins <b>9</b>, <b>24</b>–<b>26</b>). However, further simplification of the hydantoin substituent to 2-pyrrolidinone <b>29</b> saw a large decrease in potency (pIC<sub>50</sub> = 4.3). This shows, in accord with the crystallography (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), that both carbonyl groups are important for binding to ATAD2. In fact, 2-pyrrolidinone <b>29</b> offered little difference in potency compared to truncated acetamide <b>30</b> (pIC<sub>50</sub> = 4.5). Pleasingly, <b>24</b>–<b>30</b> maintained good solubility (>94 μg mL<sup>–1</sup>), and none of the compounds tested showed any appreciable affinity for the BET family. Unfortunately, potency for CECR2 largely tracked with ATAD2 and selectivity between the two bromodomains was not forthcoming. In summary, hydantoin <b>(</b><i><b>rac</b></i><b>)-26</b> offered the highest potency for ATAD2 combined with the best balance of physicochemical properties.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Investigation of the Hydantoin Substituent</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0015.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0016.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">(C) denotes selectivity for CECR2.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Also tested <4.0 (<i>n</i> = 2) test occasions.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 1) test occasions.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 4) test occasions.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">CLND solubility.</p></div></div><div></div></div><div class="NLM_p">If the SAR of the sulfonamide and amide groups were additive, the combination of the three most potent sulfonamide groups (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), the 3-methyl-3-cyclopropyl hydantoin (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), and bromide should afford optimal ATAD2 inhibitors (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Unfortunately, <b>(</b><i><b>S</b></i><b>)-31</b> did not show the expected increase in potency when the pyrrolidine was combined with the other substituents. As the optimization was carried out using Cl analogues, the nonadditivity of the SAR could arise from a subtle shift in binding position when moving to the larger Br atom. However, addition of the 3-methyl substituent increased potency by 0.3–0.5 log units, leading to a pIC<sub>50</sub> of 7.2 for the most potent diastereomer (<b>(</b><i><b>S,S</b></i><b>)-32</b>). Phenyl sulfonamide <b>(</b><i><b>S,S</b></i><b>)-32</b> maintained the excellent solubility and physicochemical profile of the starting hit and had a 5-fold bias for ATAD2 over CECR2. This profile was a marginal improvement over the hydantoin epimer <b>(</b><i><b>R,S</b></i><b>)-32</b>, and consequently, <b>(</b><i><b>S,S</b></i><b>)-32</b> was nominated for further profiling. In agreement with previous SAR, <i>trans</i>-3,4-dimethylpyrrolidine <b>33</b> increased affinity for ATAD2 (pIC<sub>50</sub> = 7.6). However, <b>33</b> was less soluble (47 μg mL<sup>–1</sup>), potentially driven by the increased lipophilicity of this analogue.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Combination of Hydantoin Substituent and Sulfonamide Vector</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0017.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0018.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Tested as a diastereomeric mixture of hydantoin epimers with unknown <i>trans</i>-stereochemistry of the pyrrolidine substituents.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">CAD solubility.</p></div></div><div></div></div><div class="NLM_p">A crystal structure of <b>(</b><i><b>R,S</b></i><b>)-32</b> was resolved in ATAD2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B). The binding mode of <b>(</b><i><b>R,S</b></i><b>)-32</b> was consistent with that of the initial hit (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The aromatic bromide sits deep within the binding pocket and displaces the conserved water molecules in ATAD2. The (<i>S</i>)-methylpyrrolidine group occupies a lipophilic pocket surrounded by the conserved Y1021, N1064, and the gatekeeper I1074 residues. The hydantoin carbonyl groups can then be seen making the previously observed H-bonding interactions with the backbone NH of D1014 and R1007. The methyl and cyclopropyl groups, which were added to improve permeability, do not appear to pick up any specific contacts, in agreement with the biochemical data generated.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YB4">6YB4</a>) of <b>(</b><i><b>R,S</b></i><b>)-32</b> (teal) in ATAD2 (silver) and a docking model of <b>(</b><i><b>S,S</b></i><b>)-32</b> (lime) in CECR2 (lilac, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXB">3NXB</a><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a>): (A) pose showing <b>(</b><i><b>R,S</b></i><b>)-32</b> sitting deep within the binding pocket, where the ligand displaces the normally conserved water network in the KAc recognition site; (B) rotated pose showing the hydantoin carbonyl groups making hydrogen bonds to both R1007 and D1014 and one of the sulfonamide oxygens hydrogen bonding to Y1021; (C) pose showing there is space for <b>(</b><i><b>S,S</b></i><b>)-32</b> to displace the waters in CECR2 and make key contacts with M506 and Y471; (D) rotated pose showing that <b>(</b><i><b>S,S</b></i><b>)-32</b> makes a H-bond to D464 NH but does not make any specific interactions with the WPF shelf in CECR2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><b>(</b><i><b>S,S</b></i><b>)-32</b> represented the most optimized ATAD2 inhibitor to date, boasting excellent potency and a novel mode of binding (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). To assess the inherent selectivity of this series of inhibitors, the wider bromodomain selectivity of <b>(</b><i><b>S,S</b></i><b>)-32</b> was investigated, first, using in house TR-FRET assays and then DiscoverX’s full BROMO<i>scan</i> panel (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Pleasingly, <b>(</b><i><b>S,S</b></i><b>)-32</b> was >200-fold selectivity over the BET family of bromodomains, using BRD4 as a representitive example. The high potency of <b>(</b><i><b>S,S</b></i><b>)-32</b> was confirmed in the BROMO<i>scan</i> ATAD2A (p<i>K</i><sub>d</sub> = 8.0) and ATAD2B (p<i>K</i><sub>d</sub> = 7.6) assays. Significant potency was observed for seven other bromodomains: bromodomain adjacent to zinc finger domain protein 2A (BAZ2A, p<i>K</i><sub>d</sub> = 7.8), BRD1 (p<i>K</i><sub>d</sub> = 6.9), bromodomain and PHD finger-containing protein 3 (BRPF3, p<i>K</i><sub>d</sub> = 6.9), CECR2 (p<i>K</i><sub>d</sub> = 7.6), general control nonderepressible 5 (GCN5L2, p<i>K</i><sub>d</sub> = 6.6), TATA-binding protein-associated factor 1 (TAF1(2), p<i>K</i><sub>d</sub> = 7.8), and TAF1-like (TAF1L(2), p<i>K</i><sub>d</sub> = 7.6) bromodomains. Greater than 30-fold selectivity was observed for all other bromodomains tested against. Interestingly, the stability of the water network in these bromodomains has previously been predicted to vary greatly; therefore, this level of off-target activity was suprising.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> It may suggest that displacement of the canonical waters is a viable approach for multiple targets in the future. Cellular target engagement was demonstrated using a CECR2 nanoBRET assay, in which a fluorescent bromosporine ligand was competitively displaced from CECR2 in HEK293 cells.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a><b>(</b><i><b>S,S</b></i><b>)-32</b> showed activity (pIC<sub>50</sub> = 6.6) in concordance with its CECR2 potency, demonstrating its ability to engage a cellular target. Overall <b>(</b><i><b>S,S</b></i><b>)-32</b> was identified as a potent ATAD2 inhibitor with good physicochemical profile and evidence of cellular target engagement. Despite the residual activity against other bromodomains, given the historical difficulties with achieving highly potent ATAD2 inhibitors, the unique binding mode and novel chemotype will make <b>(</b><i><b>S,S</b></i><b>)-32</b> (GSK388) a useful addition to the ATAD2 chemical toolbox when used in conjuction with other differentially selective probes.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Profile of <b>(</b><i><b>S,S</b></i><b>)-32</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0019.gif" alt="" id="fx8" /></img><div></div></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. DiscoverX’s BROMO<i>scan</i> results of <b>(</b><i><b>S</b></i><b>)-32</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Optimization toward a CECR2 Selective Inhibitor</h3><div class="NLM_p">Having optimized our phenyl sulfonamide series for ATAD2 potency, it was intriguing that potency at CECR2 (and indeed 7 other bromodomains) had not been reduced more significantly. Since we did not have a CECR2 crystal system, docking of <b>(</b><i><b>S,S</b></i><b>)</b><i><b>-</b></i><b>32</b> in the apo structure of CECR2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXB">3NXB</a><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a>) was solved to aid understanding of the differences between the two proteins (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D). Simulation suggested that the water network in CECR2 is more stable than in ATAD2;<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> however the binding pocket of CECR2 is able to accommodate <b>(</b><i><b>S,S</b></i><b>)</b><i><b>-</b></i><b>32</b> when all its water is removed. Considering the trends in SAR, we believe that the phenyl sulfonamide ligands bind to both bromodomains with displacement of the canonical waters in a similar manner. In the CECR2 model, one of the sulfonamide oxygens hydrogen-bonds to Y471 and the aryl bromide forms a halogen bond to the carbonyl group of M506 (the CECR2 residue analogous to I1056 in ATAD2). The (<i>S</i>)-methylpyrrolidine then occupies a lipophilic pocket formed of Y471, N514, and Y520. When compared with the crystal structure of ATAD2, several notable residue changes were apparent (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). First, the I1074 gatekeeper residue in ATAD2 is replaced by Y520 in CECR2. In ATAD2 the (RVF) shelf region is formed from R1007, V1008, and F1009, compared to W457, P458, and F459 in CECR2 which form a WPF shelf. It was hypothesized that these residue changes could be exploited to obtain enhanced CECR2 potency and selectivity.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structure of <b>(</b><i><b>R,S</b></i><b>)-32</b> (lime) in ATAD2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YB4">6YB4</a>) with point changes between the CECR2 (lilac, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXB">3NXB</a><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a>) and ATAD2 (gray) domains annotated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In ATAD2, the hydantoin makes H-bonds using both carbonyl groups: one to the backbone NH of D1014 and the other to the flexible side chain of R1007 on the RVF shelf (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B). In CECR2, according to the model, one of the hydantoin carbonyl groups can still make a H-bond to the backbone NH of D464. However, there is no suitable donor for the other carbonyl group on the WPF region due to the residue change from R1007 to W457. SAR work had shown that the two carbonyl groups were vital for ATAD2 potency and that moderate selectivity for CECR2 could be gained through removal of one of them, albeit with a significant reduction in potency (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Therefore, a strategy was adopted that aimed to build out toward the WPF residues with a substituent that could interact favorably with W457 in CECR2.</div><div class="NLM_p">Aromatic rings have been shown to form π-stacking interactions with the WPF shelf in the BET family.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> While the shelf in CECR2 is blocked by the gatekeeper Y520, it was hypothesized that substituents such as benzimidazolinone <b>(</b><i><b>S</b></i><b>)-34</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>) might still interact with W457 while maintaining a H-bond with the backbone NH of D464. This would also remove one of the carbonyl groups which had appeared crucial for ATAD2 activity, hopefully leading to an increase in selectivity. Pleasingly, benzimidazolinone <b>(</b><i><b>S</b></i><b>)-34</b> maintained potency at CECR2 (pIC<sub>50</sub> = 6.3) relative to hydantoin <b>(</b><i><b>S,S</b></i><b>)-32</b> while activity at ATAD2 dropped 20-fold. The corresponding benzoxazolidine <b>(</b><i><b>S</b></i><b>)-35</b> showed a further 10-fold increase in potency toward CECR2 (pIC<sub>50</sub> = 7.2) and 5-fold selectivity over ATAD2. There was only a marginal increase in LLE<sub>at</sub> for <b>(</b><i><b>S</b></i><b>)-35</b> relative to <b>(</b><i><b>S,S</b></i><b>)-32</b>, due to the raised lipophilicity of this analogue. Compound <b>(</b><i><b>S</b></i><b>)-35</b> was also more potent at BRD4 BD1 (pIC<sub>50</sub> = 5.0) although it still maintained a 250-fold selectivity window. On the basis of previously discussed ATAD2 crystallography and CECR2 modeling (vide supra), it was hypothesized that the carbamate in <b>(</b><i><b>S</b></i><b>)-35</b> retained a H-bonding interaction with both proteins. Benzotriazole <b>(</b><i><b>S</b></i><b>)-36</b> and benzisoxazole <b>(</b><i><b>S</b></i><b>)-37</b> were designed to explore whether this interaction could be maintained when the H-bond acceptor was located within an aromatic ring. Both compounds showed good potency for CECR2 (pIC<sub>50</sub> = 6.7 and 6.6, respectively) and maintained a bias over ATAD2 and high selectivity over BRD4 BD1. Removal of one of the heteroatoms to give benzofuran <b>(</b><i><b>S</b></i><b>)-38</b> showed a 32-fold drop-off in CECR2 potency, suggesting that a H-bond acceptor at the 2-position was crucial. Unfortunately, compounds <b>34</b>–<b>38</b> showed reduced solubility (≤25 μg mL<sup>–1</sup>), most likely due to their greater lipophilicity and aromaticity. As a result, none of the compounds tested were ideal tool molecules.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Targeting the WPF Shelf of CECR2</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0020.gif" alt="" id="fx9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0021.gif" alt="" id="fx10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">(A) denotes selectivity for ATAD2 over CECR2.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 1) test occasions.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">CLND solubility.</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last">Also tested <0 (<i>n</i> = 1) test occasions.</p></div></div><div></div></div><div class="NLM_p">While the use of an aromatic amide substituent increased potency and selectivity at CECR2, it was unclear whether this was indeed the optimal substituent. To investigate the requirement for the phenyl ring of the benzotriazole, <b>(</b><i><b>S</b></i><b>)-36</b> was truncated. Unsubstituted triazole <b>(</b><i><b>S</b></i><b>)-39</b> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>) showed weak CECR2 potency (cf. benzotriazole <b>(</b><i><b>S</b></i><b>)-36</b>) but maintained ligand efficiency compared to the leading biaryls (<b>34</b>–<b>37</b>). It was therefore logical to investigate alternative substituents to look for improved physicochemical properties while maintaining the potency of benzotriazole <b>(</b><i><b>S</b></i><b>)-36</b>. Dimethyltriazole <b>(</b><i><b>S</b></i><b>)-40</b> showed increased CECR2 inhibition (pIC<sub>50</sub> = 6.0) while maintaining the ligand efficiency and ATAD2 selectivity of <b>(</b><i><b>S</b></i><b>)-39</b>. This suggested that the aromatic ring could be replaced by sp<sup>3</sup> groups with similar overall efficiency. A diversity orientated approach that initially looked at a range of substitution at the 4-position was undertaken, and the key examples are discussed herein. Neither a propyl nor ether group increased affinity for CECR2 (<b>(</b><i><b>S</b></i><b>)-41</b> and <b>(</b><i><b>S</b></i><b>)-42</b>), but the installation of a methylene linked dimethylamine afforded <b>(</b><i><b>S</b></i><b>)-43</b> (pIC<sub>50</sub> = 6.6), a potent CECR2 inhibitor that was 5-fold selective over ATAD2. <b>(</b><i><b>S</b></i><b>)-43</b> was equipotent with benzotriazole <b>(</b><i><b>S</b></i><b>)-36</b> but significantly less lipophilic and highly soluble (≥234 μg mL<sup>–1</sup>). The same substituents at the 5-position were then prepared, and interestingly, all three analogues (<b>44</b>–<b>46</b>) showed increased potency for CECR2 and improved selectivity over ATAD2. In particular, dimethylamine <b>(</b><i><b>S</b></i><b>)-46</b> displayed 50-fold selectivity for CECR2 (pIC<sub>50</sub> = 7.1). Importantly, it was also soluble (≥219 μg mL<sup>–1</sup>) and therefore demonstrated, for the first time, that desirable CECR2 potency and selectivity were possible with compounds with a suitable physicochemical profile.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Investigating Triazole Substitution</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0022.gif" alt="" id="fx11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0023.gif" alt="" id="fx12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 1) test occasions.</p></div></div><div></div></div><div class="NLM_p">Building on the success of the tertiary amine derivatives and the tolerance of substitution at both the 4- and 5- positions, we wanted to explore the effect of conformationally restricting the dimethylamine group. To do this, a range of triazolopiperidines (<b>47</b>–<b>53</b>, <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>) were prepared. <b>(</b><i><b>S</b></i><b>)-46</b> had afforded higher potency relative to its regioisomer, <b>(</b><i><b>S</b></i><b>)-43</b> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). Therefore, conformational restriction of this amine was investigated first. Interestingly, triazolopiperidine <b>(</b><i><b>S</b></i><b>)-47</b> was 0.5 log units less potent at CECR2 (pIC<sub>50</sub> = 6.6) relative to dimethylamine <b>(</b><i><b>S</b></i><b>)-46</b>, suggesting that conformational restriction did not maintain the dimethylamine in its favored binding conformation. Regioisomeric triazolopiperidine <b>(</b><i><b>S</b></i><b>)-48</b> was prepared and pleasingly was both highly potent (CECR2 pIC<sub>50</sub> = 7.3) and 40-fold selective for CECR2. Additionally, <b>(</b><i><b>S</b></i><b>)-48</b> had a low ChromLogD (3.7) and was highly soluble (≥211 μg mL<sup>–1</sup>). The methyl group could be extended to give ethylamine <b>(</b><i><b>S</b></i><b>)-49</b> which had a comparable profile to <b>(</b><i><b>S</b></i><b>)-48</b> albeit with a slight increase in ChromLogD. Branching of the alkyl group gave isopropylamine <b>(</b><i><b>S</b></i><b>)-50</b> which was also highly potent at CECR2 (pIC<sub>50</sub> = 7.5), equating to 63-fold selectivity over ATAD2. Despite the increased lipophilicity compared to <b>(</b><i><b>S</b></i><b>)-48</b>, <b>(</b><i><b>S</b></i><b>)-49</b> and <b>(</b><i><b>S</b></i><b>)-50</b> were still highly soluble (≥238 μg mL<sup>–1</sup>). In an effort to further understand the tolerated substitution of the amine, trifluoroethylamine <b>(</b><i><b>S</b></i><b>)-51</b> was tested. A 0.5 log unit reduction in CECR2 potency was observed relative to ethylamine <b>(</b><i><b>S</b></i><b>)-49</b>. The increased lipophilicity of <b>(</b><i><b>S</b></i><b>)-51</b> was also detrimental to the solubility (49 μg mL<sup>–1</sup>). Acylation of the amine gave acetamide <b>(</b><i><b>S</b></i><b>)-52</b> which was potent at CECR2 (pIC<sub>50</sub> = 7.0) but was only 16-fold selective over ATAD2. The unsubstituted triazolopiperidine <b>(</b><i><b>S</b></i><b>)-53</b> was poorly tolerated in CECR2 (pIC<sub>50</sub> = 6.6) but still maintained a 10-fold selectivity window. Overall, while there seemed scope to extend from the -NMe to larger substituents, <b>(</b><i><b>S</b></i><b>)-48</b> already exhibited a highly promising profile worthy of further investigation.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Investigation of the SAR of Triazolopiperidines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0024.gif" alt="" id="fx13" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0025.gif" alt="" id="fx14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">Also tested <4.3 (<i>n</i> = 1) test occasions.</p></div></div><div></div></div><div class="NLM_p">Having identified triazolopiperidines as suitable CECR2 selective inhibitors, the sulfonamide vector was revisited with respect to CECR2. As discussed previously, 3-cyclopropylpyrrolidine <b>(</b><i><b>rac</b></i><b>)-20</b> was 6-fold more potent at CECR2 relative to (<i>S</i>)-methylpyrrolidine <b>(</b><i><b>S</b></i><b>)-13</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This sulfonamide was therefore prepared with the optimized triazolopiperidine. Due to the predicted increase in both potency and lipophilicity, both the chloride and bromide aryl substituents were prepared (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>). The enantiomers were separated by preparative chiral HPLC and identified by vibrational circular dichroism. As predicted, the switch from methyl to cyclopropyl led to a 0.9 log unit increase in potency for the most active enantiomer (<b>(</b><i><b>R</b></i><b>)-54</b>) relative to <b>(</b><i><b>S</b></i><b>)-48</b>. Interestingly, the opposite enantiomer was 20-fold less potent at CECR2, a trend that was also observed for <b>(</b><i><b>R</b></i><b>)-55</b> vs <b>(</b><i><b>S</b></i><b>)-55</b>. This is significant as both enantiomers of methylpyrrolidine (<b>(</b><i><b>R</b></i><b>)-13</b> and <b>(</b><i><b>S</b></i><b>)-13</b>) were equipotent at CECR2 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). It is likely that the increased size of the cyclopropyl group means that the pocket is more accommodating to one enantiomer than the other. Pleasingly, despite the increased lipophilicity (ChromLogD = 4.4), <b>(</b><i><b>R</b></i><b>)-54</b> maintained high solubility (177 μg mL<sup>–1</sup>). In line with previous SAR,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> the chloride analogue was 0.5 log units less potent than the bromide. However, in combination with the cyclopropylpyrrolidine, <b>(</b><i><b>R</b></i><b>)-55</b> had a high CECR2 pIC<sub>50</sub> of 7.7 and was 160-fold selective over ATAD2, 1300-fold selective over BRD4 BD1, and highly soluble (207 μg mL<sup>–1</sup>). Both <b>(</b><i><b>R</b></i><b>)-54</b> and <b>(</b><i><b>R</b></i><b>)-55</b> were therefore nominated for further profiling of their wider bromodomain selectivity and cellular target engagement (vide infra).</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Combining Sulfonamide SAR with an Optimized Triazolopiperidine</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0026.gif" alt="" id="fx15" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">CECR2 pIC<sub>50</sub> (<i>n</i>)</th><th class="colsep0 rowsep0" align="center">CECR2 LE/LLE<sub>at</sub></th><th class="colsep0 rowsep0" align="center">ATAD2 pIC<sub>50</sub> (<i>n</i>)/Sel</th><th class="colsep0 rowsep0" align="center">BRD4 BD1 pIC<sub>50</sub> (<i>n</i>)</th><th class="colsep0 rowsep0" align="center" char=".">ChromLogD</th><th class="colsep0 rowsep0" align="center" char=".">CAD Sol. (μg mL<sup>–1</sup>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>R</b></i><b>)-54</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">8.2 (3)</td><td class="colsep0 rowsep0" align="left">0.36/0.38</td><td class="colsep0 rowsep0" align="left">5.7 (4)/320×</td><td class="colsep0 rowsep0" align="left">4.9 (2)</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">177</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>S</b></i><b>)-54</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">7.4 (3)</td><td class="colsep0 rowsep0" align="left">0.32/0.35</td><td class="colsep0 rowsep0" align="left">5.5 (4)/79×</td><td class="colsep0 rowsep0" align="left">4.6( 2)</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char=".">248</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>R</b></i><b>)-55</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">7.7 (4)</td><td class="colsep0 rowsep0" align="left">0.35/0.38</td><td class="colsep0 rowsep0" align="left">5.5 (2)/160×</td><td class="colsep0 rowsep0" align="left">4.6 (4)</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">207</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>S</b></i><b>)-55</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">6.9 (3)</td><td class="colsep0 rowsep0" align="left">0.30/0.32</td><td class="colsep0 rowsep0" align="left">5.2 (4)/50×</td><td class="colsep0 rowsep0" align="left">4.4 (2)</td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">≥312</td></tr></tbody></table></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry</h3><div class="NLM_p">The phenyl sulfonamides were typically constructed using the chemistry detailed in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> which shows the synthesis of <b>(</b><i><b>S,S</b></i><b>)-32</b>. The commercially available sulfonyl chloride <b>56</b> was reacted with (<i>S</i>)-methylpyrrolidine under basic conditions to afford sulfonamide <b>(</b><i><b>S</b></i><b>)-57</b> in 90% yield. The nitro group was reduced to aniline <b>(</b><i><b>S</b></i><b>)-58</b> using iron and ammonium chloride in 98% yield. An amide coupling with the commercially available racemic carboxylic acid then introduced the hydantoin substituent to give <b>(</b><i><b>S,S</b></i><b>)-32</b> after separation of the two diastereoisomers.</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>(</b><i><b>S,S</b></i><b>)-32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: a) (<i>S</i>)-methylpyrrolidine hydrochloride (1.0 equiv), DIPEA (2.2 equiv), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 1 h, 90%; (b) iron (5.0 equiv), NH<sub>4</sub>Cl (1.5 equiv), EtOH/water (3:1), 70 °C, 2 h, 98%; (c) (±)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetic acid (1.1 equiv), DIPEA (3.0 equiv), T3P (50% w/w in EtOAc, 1.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 73%, then chiral purification.</p></p></figure><div class="NLM_p">High throughput array chemistry was utilized to explore the sulfonamide vector. In this case sulfonyl chloride <b>64</b> was synthesized as a late stage intermediate (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The aromatic nitro group of <b>59</b> was reduced using iron and ammonium chloride to afford aniline <b>60</b> in 88% yield. This was then acylated using 2-chloroacetyl chloride to afford alkyl chloride <b>61</b> in 88% yield. Subsequent alkylation using the commercial hydantoin under basic conditions gave <b>62</b> as a single regioisomer. A palladium (Pd<sub>2</sub>dba<sub>3</sub>, xantphos) catalyzed thioetherification with benzylthiol gave sulfide <b>63</b> in good yield,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> which was then oxidized using 1,3-dichloro-5,5-dimethylimidazolidine-2,4-dione to afford sulfonyl chloride <b>64</b> in five steps with 16% overall yield.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> This could then be used in a series of DIPEA mediated sulfonamidations to synthesize the desired products (<b>9</b>–<b>21</b>, <b>23</b>).</div><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of a Late Stage Sulfonyl Chloride for Use in a Sulfonamide Array<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) iron (3.0 equiv), NH<sub>4</sub>Cl (1.5 equiv), EtOH/water (3:1), 70 °C, 16 h, 88%; (b) 2-chloroacetyl chloride (1.0 equiv), DIPEA (1.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 1 h, 88%; (c) hydantoin (1.2 equiv), K<sub>2</sub>CO<sub>3</sub> (3.0 equiv), DMF, 70 °C, 1 h, 79%; (d) benzylthiol (1.5 equiv), xantphos (20 mol %), Pd<sub>2</sub>dba<sub>3</sub> (10 mol %), DIPEA (3.0 equiv), 1,4-dioxane/DMF (3:2), 100 °C, 1 h, 77%; (e) 1,3-dichloro-5,5-dimethylimidazolidine-2,4-dione (2.0 equiv), water, AcOH, MeCN, rt, 2 h, 35%; (f) amine (2.0 equiv), DIPEA (3.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h, 10–87%.</p></p></figure><div class="NLM_p">In our exploration of the triazole template, click chemistry was employed to regioselectively synthesize the desired analogues (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Late-stage azide <b>(</b><i><b>S</b></i><b>)-65</b> was prepared from aniline <b>(</b><i><b>S</b></i><b>)-58</b> and 2-azidoacetic acid using a HATU mediated amide coupling in 94% yield. A copper catalyzed click reaction with the desired alkyne gave 4-substituted triazoles <b>41</b>–<b>43</b> in up to 59% yield.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Alternatively, a ruthenium catalyzed click reaction gave the 5-substituted triazoles <b>40</b> and <b>44</b>–<b>46</b> in up to 62% yield.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Triazoles <b>40</b>–<b>46</b> Using Either Copper or Ruthenium Catalyzed Click Chemistry<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2-azidoacetic acid (1.0 equiv), HATU (1.2 equiv), DIPEA (3.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 94%; (b) alkyne (1.5 equiv), Cp*RuCl(PPh<sub>3</sub>)<sub>2</sub> (3 mol %), THF, 65 °C, 2 h, 22–82%; (c) alkyne (1.1 equiv), CuSO<sub>4</sub>·5H<sub>2</sub>O (5 mol %), sodium ascorbate (0.15 equiv), CH<sub>2</sub>Cl<sub>2</sub>/water (1:1), rt, 16 h, 49–59%; (d) alkyne (2.0 equiv), Cu(OAc)<sub>2</sub>·H<sub>2</sub>O (10 mol %), sodium ascorbate (0.15 equiv), MeOH, 100 °C, microwave, 30 min, 42%.</p></p></figure><div class="NLM_p">The chemistry to generate the CECR2 selective triazolopiperidine <b>(</b><i><b>R</b></i><b>)-55</b> was more complex, as it involved synthesis of the triazolopyridine coupling partner <b>69</b> in the first instance (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Starting from the commercially available pyridine <b>66</b>, an S<sub>N</sub>Ar using <i>tert-</i>butyl glycinate gave ester <b>67</b>. Reduction of the nitro group to the aniline was facilitated using the conditions outlined previously (cf. <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>), and reaction of <b>68</b> with sodium nitrite under acidic conditions gave the desired triazolopyridine <b>69</b> in 37% yield.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> This was hydrolyzed under acidic (HCl, 4 M in 1,4-dioxane) conditions and coupled to aniline <b>(</b><i><b>rac</b></i><b>)-72</b> using HATU and DIPEA to afford intermediate <b>(</b><i><b>rac</b></i><b>)-73</b>. Aniline <b>(</b><i><b>rac</b></i><b>)-72</b> was prepared from sulfonyl chloride <b>70</b> using chemistry discussed previously (vide supra). The pyridine nitrogen was quaternized using methyl iodide before the intermediate pyridinium ion was reduced using sodium borohydride to afford <b>(</b><i><b>R</b></i><b>)-55</b>.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div><figure id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>(</b><i><b>R</b></i><b>)-55</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert-</i>butyl glycinate (1.1 equiv), TEA (1.0 equiv), EtOH, reflux, 6 h, 33%; (b) iron (5.0 equiv), NH<sub>4</sub>Cl (1.5 equiv), EtOH/water (3:1), 70 °C, 2 h, 91%; (c) NaNO<sub>2</sub> (1.2 equiv), AcOH/water (1:1), 0–80 °C, 37%; (d) 3-cyclopropylpyrrolidine (1.0 equiv), DIPEA (2.2 equiv), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 2 h, 90%; (e) iron (3.0 equiv), NH<sub>4</sub>Cl (1.5 equiv), 3:1 EtOH/water, 70 °C, 2 h, 98%; (f) HCl (4 M in dioxane, 8.3 equiv), 40 °C, overnight, then HATU (1.2 equiv), DIPEA (3.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 79%; (g) MeI (1.0 equiv), MeCN, 80 °C, 2 h, then NaBH<sub>4</sub> (2.2 equiv), 1:1 MeOH/water, rt, 73%.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Profiling of the Lead Compounds</h3><div class="NLM_p">Phenyl sulfonamides <b>(</b><i><b>R</b></i><b>)-54</b> and <b>(</b><i><b>R</b></i><b>)-55</b> had been identified for further profiling as the most potent and ATAD2 selective CECR2 analogues (<a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>). Cellular target engagement was again demonstrated using a CECR2 nanoBRET assay.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Both <b>(</b><i><b>R</b></i><b>)-54</b> and <b>(</b><i><b>R</b></i><b>)-55</b> showed nanomolar activity (pIC<sub>50</sub> = 7.4 and 7.3, respectively). Their wider bromodomain selectivity was then investigated. Both <b>(</b><i><b>R</b></i><b>)-54</b> and <b>(</b><i><b>R</b></i><b>)-55</b> displayed excellent selectivity over BRD4 BD1 (>1300-fold) and BD2 (>500-fold). They were then initially submitted to DiscoverX’s BROMO<i>scan</i> panel at a single 10 μM concentration. This identified GCN5L2 and TAF1(2) as potential off-targets. Therefore, concentration–response curves were generated for these bromodomains along with CECR2. A TAF1(2) TR-FRET assay had previously been established in our laboratories, and data were generated for both compounds. <b>(</b><i><b>R</b></i><b>)-54</b> retained CECR2 potency in the DiscoverX assay (p<i>K</i><sub>d</sub> = 8.0) but showed <30-fold selectivity over both GCN5L2 and TAF1(2). Interestingly, the TAF1(2) TR-FRET data showed 130-fold selectivity for CECR2, a marked discrepancy between the two data sources (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_001.pdf" class="ext-link">Supporting Information</a>). Encouragingly, <b>(</b><i><b>R</b></i><b>)-55</b> showed an improved selectivity profile against both GCN5L2 (63-fold) and TAF1(2) (25-fold). The TR-FRET data generated also showed an increased window for CECR2 over TAF1(2) (50-fold). Due to the excellent selectivity profile of <b>(</b><i><b>R</b></i><b>)-55</b> (GSK232) against key off-target bromodomains and confirmed cellular target engagement, it was nominated as a chemical tool for the CECR2 bromodomain and was assessed in the full BROMO<i>scan</i> panel which is visualized on the phylogenetic tree (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). As identified throughout this work, the closest off-targets were ATAD2 (79-fold selective), ATAD2B (63-fold selective), GCN5L2 (63-fold selective), TAF1(2) (25-fold selective), and TAF1L(2) (40-fold selective). Pleasingly >100-fold selectivity (p<i>K</i><sub>d</sub> < 6.1) was observed against all other bromodomains tested (for a full table of <i>K</i><sub>d</sub> values, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_001.pdf" class="ext-link">Table S2, Supporting Information</a>).</div><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. Profiles of CECR2 Tools <b>(</b><i><b>R</b></i><b>)-54</b> and <b>(</b><i><b>R</b></i><b>)-55</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0012.gif" alt="" id="fx16" /></img><div></div></div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. DiscoverX’s BROMO<i>scan</i> results of <b>(</b><i><b>R</b></i><b>)-55</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66853" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66853" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have optimized a series of phenyl sulfonamide bromodomain inhibitors, initially identified from a HTS against ATAD2, which displace the conserved water network from the KAc binding site. Optimization of this template for ATAD2 potency/selectivity was achieved through an iterative medicinal chemistry approach, aided by crystallography, to deliver <b>(</b><i><b>S,S</b></i><b>)-32</b> (GSK388) as a potent ATAD2 inhibitor with >200-fold selectivity over the BET family, which will be a valuable addition to the ATAD2 literature. Interestingly, <b>(</b><i><b>S,S</b></i><b>)-32</b> inhibited not only ATAD2, where the stability of the water network is predicted to be among the least stable, but also CECR2, where the water network is predicted to be more stable. Differences in the RVF/WPF shelf regions of ATAD2 and CECR2 were then exploited to secure high CECR2 selectivity, culminating in <b>(</b><i><b>R</b></i><b>)-55</b> (GSK232). This was a highly potent ligand with >30-fold selectivity over all other bromodomains tested and >500-fold selectivity over the BET family. <b>(</b><i><b>R</b></i><b>)-55</b> had an excellent physicochemical profile and was highly soluble. Cellular target engagement was demonstrated using a nanoBRET assay, and as such, <b>(</b><i><b>R</b></i><b>)-55</b> is a high quality chemical probe.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Experimental</h3><div class="NLM_p last">Unless otherwise stated, all reactions were carried out under an atmosphere of nitrogen in heat or oven-dried glassware and anhydrous solvent. Solvents and reagents were purchased from commercial suppliers and used as received. Reactions were monitored by thin layer chromatography (TLC) or liquid chromatography–mass spectrometry (LCMS). TLC was carried out on glass or aluminum-backed 60 silica plates coated with UV<sub>254</sub> fluorescent indicator. Spots were visualized using UV light (254 or 365 nm) or alkaline KMnO<sub>4</sub> solution, followed by gentle heating. LCMS analysis was carried out on a Waters Acquity UPLC instrument equipped with a BEH column (50 mm × 2.1 mm, 1.7 μm packing diameter) and Waters micromass ZQ MS using alternate-scan positive and negative electrospray. Analytes were detected as a summed UV wavelength of 210–350 nm. Flash column chromatography was carried out using Biotage SP4 or Isolera One apparatus with SNAP silica cartridges. Mass directed automatic purification (MDAP) was carried out using a Waters ZQ MS using alternate-scan positive and negative electrospray and a summed UV wavelength of 210–350 nm. NMR spectra were recorded at ambient temperature (unless otherwise stated) using standard pulse methods on any of the following spectrometers and signal frequencies: Bruker AV-400 (<sup>1</sup>H = 400 MHz, <sup>13</sup>C = 101 MHz,), Bruker AV-600 (<sup>1</sup>H = 600 MHz, <sup>13</sup>C = 150 MHz,), Bruker DPX-250 spectrometer at 250 MHz, Varian INOVA spectrometer at 300 MHz. Chemical shifts are referenced to trimethylsilane (TMS) or the residual solvent peak and are reported in ppm. Coupling constants are quoted to the nearest 0.1 Hz, and multiplicities are given by the following abbreviations and combinations thereof: s (singlet), δ (doublet), t (triplet), q (quartet), quin (quintet), sxt (sextet), m (multiplet), br (broad). IR spectra were obtained on a PerkinElmer Spectrum 1 machine. Optical rotation of chiral products was measured using a Jasco P1030 polarimeter. Melting point analysis was carried out using a Stuart SMP40 melting point apparatus. Liquid chromatography–high resolution mass spectra (HRMS) were recorded on a Micromass Q-Tof Ultima hybrid quadrupole time-of-flight mass spectrometer, with analytes separated on an Agilent 1100 liquid chromatograph equipped with a Phenomenex Luna C18(2) reversed phase column (100 mm × 2.1 mm, 3 μm packing diameter). Purity of synthesized compounds was determined by LCMS analysis. All compounds for biological testing were >95% pure.</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Synthesis of <b>9</b> to <b>23</b>. 3-Bromo-4-chloroaniline (<b>60</b>)</h3><div class="NLM_p last">2-Bromo-1-chloro-4-nitrobenzene <b>59</b> (10.00 g, 42.3 mmol) was suspended in EtOH (30 mL) and water (8.5 mL). Ammonium chloride (3.39 g, 63.4 mmol) and iron (7.09 g, 127 mmol) were added, and the reaction was heated to 70 °C for 16 h. The reaction mixture was cooled and filtered through a plug of Celite, washing with MeOH (30 mL). The filtrate was concentrated <i>in vacuo</i> and the residue partitioned between CH<sub>2</sub>Cl<sub>2</sub> and sat. aq NaHCO<sub>3</sub> (25 mL each). The aq layer was re-extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 25 mL). The combined organics were eluted through a hydrophobic frit and concentrated <i>in vacuo</i> to give the crude product as an orange solid. The crude product was purified by silica chromatography, eluting with 0–50% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to give 3-bromo-4-chloroaniline <b>60</b> (7.65 g, 37.1 mmol, 88% yield) as a yellow solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.01 min; <i>m</i>/<i>z</i> = 206.0. HRMS (C<sub>6</sub>H<sub>5</sub>BrClN): [M + H]<sup>+</sup> calculated 205.9367, found 205.9369. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d,</i> 400 MHz): δ (ppm) 7.19 (d, <i>J</i> = 8.6 Hz, 1H), 6.95 (d, <i>J</i> = 2.7 Hz, 1H), 6.56 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 3.70 (br s, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 146.0, 130.6, 123.1, 122.6, 119.5, 115.2. Mp 79.6–82.6 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3312, 2938, 1708, 1456, 568.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>N</i>-(3-Bromo-4-chlorophenyl)-2-chloroacetamide (<b>61</b>)</h3><div class="NLM_p last">DIPEA (3.37 mL, 19.4 mmol) was added to 3-bromo-4-chloroaniline <b>60</b> (4.00 g, 19.4 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (40 mL). 2-Chloroacetyl chloride (1.54 mL, 19.4 mmol) was added, and the reaction was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (20 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The organic layers were filtered through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–50% (3:1 EtOH:EtOAc)/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(3-bromo-4-chlorophenyl)-2-chloroacetamide <b>61</b> (4.85 g, 17.1 mmol, 88% yield) as a yellow solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.10 min; <i>m</i>/<i>z</i> = 281.9. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d,</i> 400 MHz): δ (ppm) 8.21 (br s, 1H), 7.93 (d, <i>J</i> = 2.4 Hz, 1H), 7.49 (dd, <i>J</i> = 8.6, 2.4 Hz, 1H), 7.44 (d, <i>J</i> = 8.6 Hz, 1H), 4.20 (s, 2H).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>N</i>-(3-Bromo-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>62</b>)</h3><div class="NLM_p last">Imidazolidine-2,4-dione (2.04 g, 20.4 mmol) was added to <i>N</i>-(3-bromo-4-chlorophenyl)-2-chloroacetamide <b>61</b> (4.80 g, 17.0 mmol) and K<sub>2</sub>CO<sub>3</sub> (7.03 g, 50.9 mmol) in DMF (15 mL) at rt under nitrogen. The resulting solution was stirred at 70 °C for 1 h before being cooled to rt. The reaction was diluted with CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and washed with water (2 × 20 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified using silica chromatography, eluting with 0–100% (3:1 EtOAc:EtOH)/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(3-bromo-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>62</b> (4.62 g, 13.3 mmol, 79% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.86 min; <i>m</i>/<i>z</i> = 348.0. HRMS (C<sub>11</sub>H<sub>9</sub>BrClN<sub>3</sub>O<sub>3</sub>): [M + H]<sup>+</sup> calculated 345.9589, found 345.9598. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub> 400 MHz): δ (ppm) 10.49 (s, 1H), 8.16 (s, 1H), 8.07 (d, <i>J</i> = 2.4 Hz, 1H), 7.55–7.59 (m, 1H), 7.49–7.50 (m, 1H), 4.18 (s, 2H), 4.01 (s, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 172.3, 165.9, 157.5, 139.0, 131.1, 127.5, 124.0, 121.8, 120.2, 46.6, 41.3. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3303, 3235, 1700, 1674, 1587, 1523, 1455, 1143, 929, 827, 733, 683.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>-(3-(Benzylthio)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>63</b>)</h3><div class="NLM_p last">To a solution of <i>N</i>-(3-bromo-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>62</b> (4.60 g, 13.27 mmol) and DIPEA (6.94 mL, 39.8 mmol) in 1,4-dioxane (25 mL) and DMF (17 mL) at 100 °C was added xantphos (1.54 g, 2.65 mmol) and Pd<sub>2</sub>dba<sub>3</sub> (1.22 g, 1.327 mmol). Phenylmethanethiol (1.87 mL, 15.93 mmol) was added dropwise. After 1 h, the reaction was allowed to cool and then partitioned between sat. aq NaHCO<sub>3</sub> (20 mL) and EtOAc (20 mL). A precipitate formed, and this was filtered off to give <i>N</i>-(3-(benzylthio)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>63</b> (4.00 g, 10.26 mmol, 77% yield) as a green solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.07 min; <i>m</i>/<i>z</i> = 390.1. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.42 (s, 1H), 8.16 (s, 1H), 7.79 (d, <i>J</i> = 2.2 Hz, 1H), 7.21–7.48 (m, 7 H), 4.22 (s, 2H), 4.19 (s, 2H), 4.02 (s, 2H).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 2-Chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzene-1-sulfonyl Chloride (<b>64</b>)</h3><div class="NLM_p last">To a suspension of <i>N</i>-(3-(benzylthio)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>63</b> (4.00 g, 10.3 mmol) in MeCN (75 mL), water (3.5 mL), and AcOH (5.0 mL) was added 1,3-dichloro-5,5-dimethylimidazolidine-2,4-dione (4.04 g, 20.5 mmol). The resulting solution was stirred at rt for 2 h. The solution was concentrated <i>in vacuo</i> and redissolved in CH<sub>2</sub>Cl<sub>2</sub> (50 mL), washing with sat. aq NaHCO<sub>3</sub> (50 mL). The organic layer was passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–100% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford 2-chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzene-1-sulfonyl chloride <b>64</b> (1.30 g, 3.55 mmol, 35% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.50 min; <i>m</i>/<i>z</i> = 364.2. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.54 (d, <i>J</i> = 2.5 Hz, 1H), 7.97 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.73 (d, <i>J</i> = 8.8 Hz, 1H), 4.36 (s, 2H), 4.09 (s, 2H); amide NH and hydantoin NH not visible. <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 172.3, 165.4, 157.5, 145.9, 137.1, 131.1, 125.2, 120.9, 120.4, 46.6, 41.2.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> General Procedure A: Sulfonamide Coupling</h3><div class="NLM_p last">A stock solution of 2-chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzene-1-sulfonyl chloride <b>64</b> (330 g, 0.90 mmol)) and DIPEA (0.468 mL, 2.700 mmol) dissolved in CH<sub>2</sub>Cl<sub>2</sub> (4.5 mL) was prepared and aliquoted (0.5 mL) across into 9 vessels containing preweighed monomers (0.200 mmol). Stirrer bars were added, the reaction vials were capped, and the reaction mixtures were stirred at rt for 4 h. Solvent was removed from all samples. The samples were purified by MDAP (HPH). The solvent was dried under a stream of nitrogen in the Radleys blowdown apparatus to give the required product sulfonamides listed below.</div></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i>-(4-Chloro-3-((3((dimethylamino)methyl)pyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-ioxoimidazolidin-1-yl)acetamide (<b>9</b>)</h3><div class="NLM_p last">General procedure A was followed to afford <i>N</i>-(4-chloro-3-((3((dimethylamino)methyl)pyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-ioxoimidazolidin-1-yl)acetamide <b>9</b> (19 mg, 0.011 mmol, 47% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.76 min; <i>m</i>/<i>z</i> = 401.0 (100% pure). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.67 (s, 1H), 8.27 (d, <i>J</i> = 2.5 Hz, 1H), 8.18 (s, 1H), 7.80 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 7.64 (d, <i>J</i> = 8.6 Hz, 1H), 4.21 (s, 2H), 4.03 (d, <i>J</i> = 1.0 Hz, 2H), 3.28–3.31 (m, 4H), 1.84 (dt, <i>J</i> = 6.9, 3.3 Hz, 4H).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>N</i>-(4-Chloro-3-(piperidin-1-ylsulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>10</b>)</h3><div class="NLM_p last">2-Chloro-<i>N</i>-(4-chloro-3-(piperidin-1-ylsulfonyl)phenyl)acetamide (200 mg, 0.569 mmol), imidazolidine-2,4-dione (57.0 mg, 0.569 mmol), and K<sub>2</sub>CO<sub>3</sub> (157 mg, 1.139 mmol) were dissolved in acetone (10 mL). The resulting solution was heated to 80 °C under microwave irradiation for 30 min. The reaction was cooled to rt, diluted with water (20 mL), and extracted with CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The organic phase was washed with water (25 mL) and saturated brine (25 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to afford the crude product. The crude material was purified by preparatory TLC, eluting with 70% EtOAc/pet. ether. Fractions containing the desired product were further purified by prep-HPLC using an Atlantis T3 (250 mm × 19 mm) column, eluting with 35% (0.1% TFA in water)/MeCN. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(piperidin-1-ylsulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>10</b> (23 mg, 0.055 mmol, 10%) as an off white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.88 min, <i>m</i>/<i>z</i> = 415.3 (100% pure). HRMS (C<sub>16</sub>H<sub>19</sub>ClN<sub>4</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 415.0843, found 415.0840. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.76 (s, 1H), 8.28 (d, <i>J</i> = 2.6 Hz, 1H), 8.18 (s, 1H), 7.81 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 7.64 (d, <i>J</i> = 8.6 Hz, 1H), 4.22 (s, 2H), 4.03 (s, 2H), 3.17 (t, <i>J</i> = 5.3 Hz, 4H), 1.41–1.60 (m, 6H).</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i>-(3-(Azepan-1-ylsulfonyl)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>11</b>)</h3><div class="NLM_p last">General procedure A was followed to afford <i>N</i>-(3-(azepan-1-ylsulfonyl)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>11</b> (52 mg, 0.121 mmol, 57% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.98 min; <i>m</i>/<i>z</i> = 429.1 (100% pure). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 600 MHz): δ (ppm) 10.64–10.68 (m, 1H), 8.23–8.28 (m, 1H), 8.16–8.21 (m, 1H), 7.73–7.79 (m, 1H), 7.58–7.66 (m, 1H), 4.20 (s, 2H), 4.02 (s, 2H), 3.30–3.33 (m, 4H), 1.62–1.70 (m, 4H), 1.50–1.59 (m, 4H).</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-(4-Chloro-3-(<i>N</i>,<i>N</i>-dimethylsulfamoyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>12</b>)</h3><div class="NLM_p last">2-Chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzenesulfonyl chloride <b>64</b> (40 mg, 0.109 mmol) was taken up in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) under nitrogen and cooled in an ice bath. DIPEA (0.042 mL, 0.240 mmol) was added and the reaction stirred for 5 min before dimethylamine (2 M in THF, 0.057 mL, 0.115 mmol) was added, and the reaction was stirred for 1 h. The reaction mixture was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and washed with 2 M HCl aq (10 mL), then filtered through a hydrophobic frit and concentrated <i>in</i><i>vacuo</i> to give the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(<i>N</i>,<i>N</i>-dimethylsulfamoyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>12</b> (14 mg, 0.037 mmol, 34% yield) as a white solid. LCMS (formic ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.73 min; <i>m</i>/<i>z</i> = 375.2. HRMS (C<sub>13</sub>H<sub>15</sub>ClN<sub>4</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 375.0530, found 375.0521. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.73 (s, 1H), 8.25 (d, <i>J</i> = 2.7 Hz, 1H), 8.17 (s, 1H), 7.80 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.64 (d, <i>J</i> = 8.8 Hz, 1H), 4.21 (s, 2H), 4.02 (s, 2H), 2.81 (s, 6H).</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (<i>S</i>)-<i>N</i>-(4-Chloro-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-13</b>)</h3><div class="NLM_p last">To a solution of 2-chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzene-1-sulfonyl chloride <b>64</b> (70 mg, 0.191 mmol) and DIPEA (0.080 mL, 0.459 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (4 mL) was added (<i>S</i>)-3-methylpyrrolidine, hydrochloride (27.9 mg, 0.229 mmol). The resulting solution was stirred at rt for 30 min. After this time the reaction mixture was concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by MDAP (HPH). The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-chloro-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-13</b> (77 mg, 0.186 mmol, 87%) as a white solid. LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.86 min, <i>m</i>/<i>z</i> = 415.3 (99% pure). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.69 (s, 1H), 8.27 (d, <i>J</i> = 2.7 Hz, 1H), 8.21 (s, 1H), 7.75–7.84 (m, 1H), 7.65 (d, <i>J</i> = 8.8 Hz, 1H), 4.21 (s, 2H), 4.03 (s, 2H), 3.45 (d, <i>J</i> = 6.8 Hz, 2H), 3.23–3.30 (m, 1H), 2.76–2.87 (m, <i>J</i> = 7.8 Hz, 1H), 2.17–2.30 (m, 1H), 1.94–2.04 (m, 1H), 1.42–1.55 (m, 1H), 0.94 (d, <i>J</i> = 6.6 Hz, 3H).</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> (<i>R</i>)-<i>N</i>-(4-Chloro-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>R</b></i><b>)-13</b>)</h3><div class="NLM_p last">General procedure A was followed to afford (<i>R</i>)-<i>N</i>-(4-chloro-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>R</b></i><b>)-13</b> (39 mg, 0.095 mmol, 45%) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.89 min, <i>m</i>/<i>z</i> = 415.1; data consistent with enantiomer.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i>-(4-Chloro-3-((3,3-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>14</b>)</h3><div class="NLM_p last">DIPEA (0.080 mL, 0.459 mmol) and 3,3-dimethylpyrrolidine hydrochloride (29 mg, 0.210 mmol) were added to 2-chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzene-1-sulfonyl chloride (70 mg, 0.191 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2.5 mL) at 0 °C under nitrogen. The resulting solution was stirred at rt for 30 min. After this time the reaction mixture was concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by MDAP (formic). The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-((3,3-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>14</b> (37 mg, 0.087 mmol, 46% yield) as a colorless oil. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.89 min, <i>m</i>/<i>z</i> = 429.3. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.32 (d, <i>J</i> = 2.7 Hz, 1H), 7.80 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.56 (d, <i>J</i> = 8.8 Hz, 1H), 4.34 (s, 2H), 4.08 (s, 2H), 3.52 (t, <i>J</i> = 7.1 Hz, 2H), 3.13 (s, 2H), 1.74 (t, <i>J</i> = 7.1 Hz, 2H), 1.06 (s, 6H); amide NH and hydantoin NH not visible.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i>-(3-(5-Azaspiro[2.4]heptan-5-ylsulfonyl)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>15</b>)</h3><div class="NLM_p last">5-Azaspiro[2.4]heptane hydrochloride (38 mg, 0.281 mmol) was added to a solution of 2-chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzene-1-sulfonyl chloride <b>64</b> (86 mg, 0.234 mmol) and DIPEA (0.098 mL, 0.561 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL). The resulting solution was stirred at rt for 1 h. After this time the reaction mixture was concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by MDAP (HPH). The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(3-(5-azaspiro[2.4]heptan-5-ylsulfonyl)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>15</b> (74 mg, 0.173 mmol, 70%) as a cream solid. LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.88 min, <i>m</i>/<i>z</i> = 427.3. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.68 (s, 1H), 8.29 (d, <i>J</i> = 2.7 Hz, 1H), 8.19 (s, 1H), 7.79 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 7.66 (d, <i>J</i> = 8.6 Hz, 1H), 4.21 (s, 2H), 4.02 (s, 2H), 3.47 (t, <i>J</i> = 6.8 Hz, 2H), 3.20 (s, 2H), 1.79 (t, <i>J</i> = 6.8 Hz, 2H), 0.45–0.62 (m, 4H).</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-(4-Chloro-3-(((3<i>R</i>,4<i>S</i>)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>17</b>) and <i>N</i>-(4-Chloro-3-(((3<i>S</i>,4<i>S</i>)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>rac</b></i><b>)-16</b>)</h3><div class="NLM_p">3,4-Dimethylpyrrolidine hydrochloride (59 mg, 0.435 mmol) and DIPEA (110 mg, 0.851 mmol) were added to a solution of 2-chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzene-1-sulfonyl chloride <b>64</b> (209 mg, 0.342 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (4 mL) at 0 °C under nitrogen. The solution was stirred at rt for 1 h while allowed to warm to rt. The reaction was diluted with CH<sub>2</sub>Cl<sub>2</sub> (4 mL) and washed with sat. aq sodium carbonate (4 mL) followed by 10% citric acid in H<sub>2</sub>O solution (4 mL). The combined organic layers were combined, passed through a hydrophobic frit, and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by MDAP (HPH). The fractions containing the desired product were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-((3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (77 mg, 0.171 mmol, 50% yield) as a white powder and a (2.7:1) mixture of <i>cis</i> and <i>trans</i> diastereomers. The diastereomeric mixture was dissolved in EtOH (5 mL) and injected onto the column (column, 250 mm × 20 mm Chiralpak AD-H, 5 μm), eluting with 25% (EtOH + 0.2% isopropylamine)/(heptane + 0.2% isopropylamine), flow rate = 40 mL min<sup>–1</sup>, detection wavelength 250 nm. The pure fractions from peak 1 were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(((3<i>R</i>,4<i>S</i>)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>17</b> (36 mg, 0.080 mmol, 23%) as a white solid, and the pure fractions from peak 2 were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(((3<i>S</i>,4<i>S</i>)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-16</b> (8 mg, 0.018 mmol, 5%) as a mixture of <i>trans</i> enantiomers.</div><div id="sec4_3_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(4-Chloro-3-(((3<i>R</i>,4<i>S</i>)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>17</b>)</h4><div class="NLM_p last">LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.92 min, <i>m</i>/<i>z</i> = 428.3. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.32 (d, <i>J</i> = 2.4 Hz, 1H), 7.81 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 7.57 (d, <i>J</i> = 8.6 Hz, 1H), 4.34 (s, 2H), 4.08 (s, 2H), 3.53 (dd, <i>J</i> = 9.5, 6.0 Hz, 2H), 3.12 (dd, <i>J</i> = 9.5, 6.0 Hz, 2H), 2.26–2.38 (m, 2H), 0.92 (d, <i>J</i> = 6.8, 6H); amide NH and hydantoin NH not visible.</div></div><div id="sec4_3_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(4-Chloro-3-(((3<i>S</i>,4<i>S</i>)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>(<i>rac</i>)-16</b>)</h4><div class="NLM_p last">LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.93 min, <i>m</i>/<i>z</i> = 428.3. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.31 (d, <i>J</i> = 2.4 Hz, 1H), 7.80 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 7.57 (d, <i>J</i> = 8.6 Hz, 1H), 4.34 (s, 2H), 4.08 (s, 2H), 3.62–3.71 (m, 2H), 2.89–3.00 (m, 2H), 1.73–1.85 (m, 2H), 1.02 (d, <i>J</i> = 6.1 Hz, 6H); amide NH and hydantoin NH not visible.</div></div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>N</i>-(4-Chloro-3-((3-ethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>rac</b></i><b>)-18</b>)</h3><div class="NLM_p last">General procedure A was followed to afford <i>N</i>-(4-chloro-3-((3-ethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-18</b> (44 mg, 0.102 mmol, 48%) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.96 min, <i>m</i>/<i>z</i> = 429.1. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 600 MHz): δ (ppm) 10.68 (s, 1H), 8.27 (d, <i>J</i> = 2.5 Hz, 1H), 8.19 (s, 1H), 7.79 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 7.64 (d, <i>J</i> = 8.7 Hz, 1H), 4.20 (s, 2H), 4.02 (s, 2H), 3.45–3.49 (m, 1H), 3.38–3.43 (m, 1H), 3.22–3.28 (m, 1H), 2.83–2.89 (m, 1H), 1.96–2.09 (m, 2H), 1.43–1.55 (m, 1H), 1.23–1.35 (m, 2H), 0.83 (t, <i>J</i> = 7.6 Hz, 3H).</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>-(4-Chloro-3-((3-(methoxymethyl)pyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>rac</b></i><b>)-19</b>)</h3><div class="NLM_p last">General procedure A was followed to afford <i>N</i>-(4-chloro-3-((3-(methoxymethyl)pyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-19</b> (4 mg, 0.009 mmol, 4%) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.82 min, <i>m</i>/<i>z</i> = 445.1. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 500 MHz): δ (ppm) 10.74 (s, 1H), 8.28 (d, <i>J</i> = 2.7 Hz, 1H), 8.21 (s, 1H), 7.80 (dd, <i>J</i> = 8.6, 2.3 Hz, 1H), 7.66 (d, <i>J</i> = 8.8 Hz, 1H), 4.21 (s, 2H), 4.03 (s, 2H), 3.23–3.32 (m, 3H), 3.12–3.22 (m, 5H), 3.00–3.09 (m, 1H), 2.38–2.49 (m, 1H), 1.89–1.99 (m, 1H), 1.55–1.67 (m, 1H).</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> (<i>rac</i>)-<i>N</i>-(4-Chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>rac</b></i><b>)-20</b>)</h3><div class="NLM_p last">2-Chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzenesulfonyl chloride <b>64</b> (50 mg, 0.137 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at rt under nitrogen. DIPEA (0.072 mL, 0.410 mmol) was added followed by 3-cyclopropylpyrrolidine (18 mg, 0.164 mmol), and the resulting solution was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (2 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 2 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-<i>N</i>-(4-chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-20</b> (19 mg, 0.043 mmol, 32% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.96 min, <i>m</i>/<i>z</i> = 441.1. HRMS (C<sub>18</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 441.0999, found 441.1001. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.32 (d, <i>J</i> = 2.5 Hz, 1H), 7.78 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 7.56 (d, <i>J</i> = 8.6 Hz, 1H), 4.34 (s, 2H), 4.08 (s, 2H), 3.52–3.65 (m, 2H), 3.34–3.42 (m, 1H), 3.12–3.19 (m, 1H), 2.06–2.11 (m, 1H), 1.71–1.86 (m, 1H), 1.57–1.69 (m, 1H), 0.61–0.74 (m, 1H), 0.40–0.51 (m, 2H), 0.11–0.19 (m, 2H); amide NH and hydantoin NH not visible. <sup>13</sup>C NMR (MeOD-<i>d</i><sub>4</sub>, 101 MHz): δ (ppm) 172.4, 165.9, 158.0, 137.4, 136.9, 132.3, 125.9, 124.2, 122.2, 52.4, 46.2, 44.2, 40.5, 30.9, 26.6, 12.6, 2.6, 2.3. Mp 113.0–114.7 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3309, 2938, 1708, 1455, 1149, 581.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> (<i>R</i>)-<i>N</i>-(4-Chloro-3-((2-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>R</b></i><b>)-21</b>)</h3><div class="NLM_p last">2-Chloro-5-(2-(2,5-dioxoimidazolidin-1-yl) acetamido)benzene-1-sulfonyl chloride <b>64</b> (106 mg, 0.174 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) and cooled to 0 °C. (<i>R</i>)-2-Methylpyrrolidine, hydrochloride (25 mg, 0.208 mmol) and DIPEA (54 mg, 0.417 mmol) were then added. The reaction mixture was stirred for 30 min while allowed to warm to rt. The reaction was then concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by MDAP (HPH). Pure fractions were concentrated <i>in vacuo</i> to afford (<i>R</i>)-<i>N</i>-(4-chloro-3-((2-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>R</b></i><b>)-21</b> (54 mg, 0.129 mmol, 75% yield) as a white solid. LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.84 min, <i>m</i>/<i>z</i> = 415.3 (98% pure). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.34 (d, <i>J</i> = 2.4 Hz, 1H), 7.77 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.51–7.59 (m, 1H), 4.35 (s, 2H), 4.08 (s, 2H), 3.98–4.05 (m, 1H), 3.41–3.51 (m, 2H), 1.91–2.12 (m, 2H), 1.76–1.87 (m, 1H), 1.57–1.68 (m, 1H), 1.15–1.26 (m, 3H); 2 amide NHs not visible.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> <i>N</i>-(4-Chloro-3-(cyclohexylthio)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>74</b>)</h3><div class="NLM_p last">To a solution of <i>N</i>-(3-bromo-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>62</b> (100 mg, 0.29 mmol) and DIPEA (0.15 mL, 0.87 mmol) in 1,4-dioxane (3 mL) and DMF (2 mL) at 100 °C were added xantphos (33 mg, 0.06 mmol and Pd<sub>2</sub>dba<sub>3</sub> (26 mg, 0.03 mmol). Cyclohexanethiol (0.04 mL, 0.29 mmol) was added dropwise and the reaction stirred for 1 h. The reaction was allowed to cool before quenching with sat. aq Na<sub>2</sub>CO<sub>3</sub> (50 mL) and extracting with CH<sub>2</sub>Cl<sub>2</sub> (3 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by MDAP (formic). The solvent was evaporated <i>in vacuo</i> to give <i>N</i>-(4-chloro-3-(cyclohexylthio)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>73</b> (46 mg, 0.12 mmol, 42% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.15; <i>m</i>/<i>z</i> = 382.1. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d,</i> 400 MHz): δ (ppm) 8.33 (br s, 1H) 8.12 (s, 1H) 7.69 (s, 1H), 7.64 (s, 1H) 7.02 (d, <i>J</i> = 2.0 Hz, 1H) 4.30 (s, 2H) 4.08 (s, 2H) 1.99 (d, <i>J</i> = 5.1 Hz, 1H) 1.78 (dd, <i>J</i> = 8.7, 3.8 Hz, 2H) 1.58–1.65 (m, 1H) 1.22–1.50 (m, 7 H);</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> <i>N</i>-(4-Chloro-3-(cyclohexylsulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>22</b>)</h3><div class="NLM_p last"><i>m</i>CPBA (46 mg, 0.27 mmol) was added to <i>N-</i>(4-chloro-3-(cyclohexylthio)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>74</b> (46 mg, 0.12 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) at rt under nitrogen. The resulting solution was stirred at rt for 16 h. The reaction was quenched with sat. aq Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL). The organic layer was washed with sat. aq NaHCO<sub>3</sub>, passed through a hydrophobic frit, and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by MDAP (formic). The solvent was evaporated in <i>vacuo</i> to give <i>N</i>-(4-chloro-3-(cyclohexylsulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>22</b> (23 mg, 0.06 mmol, 46% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.92 min; <i>m</i>/<i>z</i> = 414.1 (97% pure). HRMS (C<sub>17</sub>H<sub>21</sub>ClN<sub>3</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 414.0885, found 414.0880. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d,</i> 400 MHz): δ (ppm) 9.28 (s, 1H) 7.95–8.08 (m, 2H) 7.42 (d, <i>J</i> = 8.8 Hz, 1H) 6.63 (s, 1H) 4.35 (s, 2H) 4.06 (s, 2H) 3.43–3.57 (m, 1H) 1.81–1.97 (m, 4 H) 1.69 (d, <i>J</i> = 9.8 Hz, 1H) 1.41–1.58 (m, 2H) 1.12–1.34 (m, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 171.6, 165.3, 157.9, 137.3, 135.0, 132.6, 126.9, 125.7, 123.0, 61.5, 46.9, 41.5, 25.1, 25.0, 24.9.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> <i>N</i>-(3-((1<i>H</i>-Imidazol-1-yl)sulfonyl)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>23</b>)</h3><div class="NLM_p last">1<i>H</i>-Imidazole (11.15 mg, 0.164 mmol) was added to 2-chloro-5-(2-(2,5-dioxoimidazolidin-1-yl)acetamido)benzene-1-sulfonyl chloride <b>64</b> (60 mg, 0.164 mmol) and DIPEA (0.086 mL, 0.492 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at rt under nitrogen. The resulting solution was stirred at rt for 2 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 10 mL). The combined organics were concentrated <i>in</i><i>vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(3-((1<i>H</i>-imidazol-1-yl)sulfonyl)-4-chlorophenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide <b>23</b> (45 mg, 0.113 mmol, 69% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.71 min; <i>m</i>/<i>z</i> = 398.2. HRMS (C<sub>14</sub>H<sub>12</sub>ClN<sub>5</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 398.0326, found 398.0327. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.85 (s, 1H), 8.58 (d, <i>J</i> = 2.7 Hz, 1H), 8.36–8.42 (m, 1H), 8.20 (s, 1H), 7.90 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.64–7.74 (m, 2H), 7.16 (dd, <i>J</i> = 1.6, 0.9 Hz, 1H), 4.23 (s, 2H), 4.04 (s, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 172.3, 166.3, 157.4, 138.9, 138.6, 134.9, 133.7, 131.5, 126.9, 125.2, 121.7, 119.1, 46.6, 41.3. Mp 149.3–152.8 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2951, 1715, 1608, 1288, 1251, 773.</div></div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Synthesis of <b>(</b><i><b>rac</b></i><b>)-24</b> to <b>30</b>. 1-((2-Chloro-5-nitrophenyl)sulfonyl)pyrrolidine (<b>75</b>)</h3><div class="NLM_p last">2-Chloro-5-nitrobenzenesulfonyl chloride (1.20 g, 4.69 mmol) was taken up in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) at 0 °C under nitrogen. DIPEA (1.80 mL, 10.31 mmol) was added and the reaction stirred for 5 min before pyrrolidine (0.39 mL, 4.69 mmol) was added. The reaction was stirred for 2 h at 0 °C after which time the reaction mixture was quenched with sat. aq NaHCO<sub>3</sub> (50 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 50 mL). The combined organics were filtered through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–40% EtOAc/cyclohexane. The pure fractions were concentrated <i>in</i><i>vacuo</i> to afford 1-((2-chloro-5-nitrophenyl)sulfonyl)pyrrolidine <b>75</b> (1.14 g, 3.92 mmol, 84% yield) as a yellow solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.06, does not ionize. HRMS (C<sub>10</sub>H<sub>11</sub>ClN<sub>2</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 291.0206, found 291.0204. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.91 (d, <i>J</i> = 2.7 Hz, 1H), 8.33 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 1H), 3.41–3.54 (m, 4H), 1.88–2.02 (m, 4H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 146.2, 139.4, 139.0, 133.2, 127.4, 126.7, 48.1, 25.8. Mp 95.5–101.5 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3105, 2970, 1601, 1524, 1337, 603.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 4-Chloro-3-(pyrrolidin-1-ylsulfonyl)aniline (<b>76</b>)</h3><div class="NLM_p last">1-((2-Chloro-5-nitrophenyl)sulfonyl)pyrrolidine <b>75</b> (1.00 g, 3.44 mmol), ammonium chloride (0.28 g, 5.16 mmol), and iron (0.96 g, 17.20 mmol) were dissolved in a 3:1 mixture of EtOH (5.0 mL) and water (1.7 mL). The resulting solution was heated to 70 °C for 2 h. The reaction was allowed to cool then filtered through a plug of Celite, washing with MeOH (20 mL). The resulting solution was concentrated <i>in vacuo</i> and then partitioned between sat. aq NaHCO<sub>3</sub> (10 mL) and CH<sub>2</sub>Cl<sub>2</sub> (10 mL). The layers were separated, and the aq layer was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–100% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford 4-chloro-3-(pyrrolidin-1-ylsulfonyl)aniline <b>76</b> (743 mg, 2.85 mmol, 83% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.88 min, <i>m</i>/<i>z</i> = 261.2. HRMS (C<sub>10</sub>H<sub>13</sub>ClN<sub>2</sub>O<sub>2</sub>S): [M + H]<sup>+</sup> calculated 261.0465, found 261.0464. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 7.36–7.43 (m, 1H), 7.25 (d, <i>J</i> = 8.6 Hz, 1H), 6.63–6.79 (m, 1H), 3.94 (br s, 2H), 3.36–3.46 (m, 4H), 1.82–1.98 (m, 4H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 145.5, 137.0, 132.6, 119.9, 119.2, 118.0, 47.7, 25.8. Mp 114.2–116.5 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3369, 2981, 1616, 1471, 1331, 1158, 586.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 2-Chloro-<i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)acetamide (<b>77</b>)</h3><div class="NLM_p last">DIPEA (0.200 mL, 1.151 mmol) was added to 4-chloro-3-(pyrrolidin-1-ylsulfonyl)aniline <b>76</b> (100 mg, 0.384 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) at 0 °C. 2-Chloroacetyl chloride (0.037 mL, 0.460 mmol) was added, and the reaction was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 10 mL). The organic layers were filtered through a hydrophobic frit and concentrated in <i>vacuo</i> to afford 2-chloro-<i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)acetamide <b>77</b> (103 mg, 0.305 mmol, 80% yield) as a brown oil. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.98 min, <i>m</i>/<i>z</i> = 337.1, 339.1. HRMS (C<sub>12</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 337.0180, found 337.0176. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.66 (br s, 1H), 8.09 (d, <i>J</i> = 2.5 Hz, 1H), 8.02 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 7.51 (d, <i>J</i> = 8.6 Hz, 1H), 4.23 (s, 2H), 3.43–3.48 (m, 4H), 1.92–1.97 (m, 4H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 164.4, 137.5, 135.9, 132.8, 127.5, 124.6, 122.9, 47.9, 42.8, 25.8. Mp 113.2–119.5 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3358, 2978, 1714, 1582, 1465, 1145, 530.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> (<i>rac</i>)-<i>N</i>-(4-Chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>rac</b></i><b>)-24</b>)</h3><div class="NLM_p last">2-Chloro-<i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)acetamide <b>77</b> (100 mg, 0.297 mmol), (<i>rac</i>)-5-methylimidazolidine-2,4-dione (37 mg, 0.326 mmol), and K<sub>2</sub>CO<sub>3</sub> (53 mg, 0.386 mmol) were dissolved in acetone (1 mL). The resulting solution was heated to 70 °C and stirred for 1 h. The reaction was allowed to cool before quenching with sat. aq NaHCO<sub>3</sub> (10 mL) and extracting with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-<i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-24</b> (82 mg, 0.198 mmol, 67% yield). LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.82 min, <i>m</i>/<i>z</i> = 415.3. HRMS (C<sub>16</sub>H<sub>19</sub>ClN<sub>4</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 415.0843, found 415.0844. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 8.41 (s, 1H), 8.30–8.37 (m, 1H), 8.27 (d, <i>J</i> = 2.7 Hz, 1H), 7.80 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.63 (d, <i>J</i> = 8.8 Hz, 1H), 4.14–4.24 (m, 3H), 3.24–3.35 (m, 4H), 1.79–1.88 (m, 4H), 1.30 (d, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 175.4, 166.0, 156.4, 138.3, 136.8, 133.2, 124.8, 124.3, 121.7, 52.8, 48.1, 41.2, 25.6, 17.8. Mp 130.9–134.9 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3321, 2981, 1702, 1454, 1152, 588.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> <i>N</i>-(4-Chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(4,4-dimethyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>25</b>)</h3><div class="NLM_p last">4-Chloro-3-(pyrrolidin-1-ylsulfonyl)aniline <b>76</b> (50 mg, 0.192 mmol), 2-(4,4-dimethyl-2,5-dioxoimidazolidin-1-yl)acetic acid (39 mg, 0.211 mmol), and HATU (87 mg, 0.230 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at rt under air. DIPEA (0.10 mL, 0.575 mmol) was added, and the resulting solution was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(4,4-dimethyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>24</b> (57 mg, 0.133 mmol, 69% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.87 min, <i>m</i>/<i>z</i> = 429.3. HRMS (C<sub>17</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 429.0993, found 429.0995. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.82 (s, 1H), 8.08 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.96 (d, <i>J</i> = 2.5 Hz, 1H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 6.03 (s, 1H), 4.40 (s, 2H), 3.38–3.46 (m, 4H), 1.90–1.95 (m, 4H), 1.53 (s, 6H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 177.0, 164.7, 156.0, 136.8, 132.6, 132.7, 126.4, 124.5, 122.6, 59.7, 47.9, 41.6, 25.7, 25.0. Mp 264.2–267.2 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3344, 2980, 1714, 1531, 1450, 1304, 595.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> (<i>rac</i>)-<i>N</i>-(4-Chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>rac</b></i><b>)-26</b>)</h3><div class="NLM_p last">HATU (538 mg, 1.414 mmol) was added to a solution of (<i>rac</i>)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetic acid (250 mg, 1.178 mmol) in DMF (5 mL) at 0 °C under nitrogen followed by the addition of DIPEA (0.617 mL, 3.53 mmol). The reaction mixture was stirred at 0 °C for 10 min, before 4-chloro-3-(pyrrolidin-1-ylsulfonyl)aniline <b>76</b> (338 mg, 1.296 mmol) was added. The reaction mixture was stirred at rt for 12 h. The reaction was then poured into ice–water and extracted with EtOAc (3 × 50 mL). The organic layer was washed with water (50 mL) and brine solution (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography eluting with 0–50% EtOAc/hexanes. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-<i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-26</b> (165 mg, 0.355 mmol, 30% yield) as an off-white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.95 min, <i>m</i>/<i>z</i> = 455.3. HRMS (C<sub>19</sub>H<sub>23</sub>ClN<sub>4</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 455.1156, found 455.1160. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.74 (s, 1H), 8.25 (d, <i>J</i> = 2.4 Hz, 1H), 8.17 (s, 1H), 7.75–7.84 (m, 1H), 7.64 (d, <i>J</i> = 8.8 Hz, 1H), 4.20 (s, 2H), 3.28–3.32 (m, 4H), 1.80–1.90 (m, 4H), 1.40 (s, 3H), 1.11–1.20 (m, 1H), 0.41–0.49 (m, 2H), 0.27–0.39 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 176.8, 165.8, 156.0, 138.3, 136.9, 133.2, 124.7, 124.2, 121.4, 61.0, 48.1, 41.0, 25.6, 23.6, 17.4, 0.9, 0.4.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> <i>N</i>-(4-Chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(2,4-dioxooxazolidin-3-yl)acetamide (<b>27</b>)</h3><div class="NLM_p last">2-Chloro-<i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)acetamide <b>77</b> (50 mg, 0.148 mmol) and potassium 2,4-dioxooxazolidin-3-ide (25 mg, 0.178 mmol) were dissolved in DMF (0.5 mL). The resulting solution was heated to 100 °C and stirred for 2 h. The reaction was allowed to cool, diluted with water (5 mL), and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(2,4-dioxooxazolidin-3-yl)acetamide <b>27</b> (33 mg, 0.082 mmol, 55% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.89 min, <i>m</i>/<i>z</i> = 402.2. HRMS (C<sub>15</sub>H<sub>16</sub>ClN<sub>3</sub>O<sub>6</sub>S): [M + H]<sup>+</sup> calculated 402.0527, found 402.0523. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.75 (s, 1H), 8.26 (d, <i>J</i> = 2.7 Hz, 1H), 7.80 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.66 (d, <i>J</i> = 8.8 Hz, 1H), 5.04 (s, 2H), 4.32 (s, 2H), 3.28–3.33 (m, 4H), 1.82–1.87 (m, 4H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 171.4, 165.0, 156.2, 138.0, 136.9, 133.2, 125.1, 124.5, 121.9, 69.2, 48.1, 42.6, 25.6. Mp 230.6–233.1 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3343, 2980, 1745, 1683, 1446, 1157, 534.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> <i>N</i>-(4-Chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(2,5-dioxopyrrolidin-1-yl)acetamide (<b>28</b>)</h3><div class="NLM_p last">4-Chloro-3-(pyrrolidin-1-ylsulfonyl)aniline <b>76</b> (50 mg, 0.192 mmol), 2-(2,5-dioxopyrrolidin-1-yl)acetic acid (33 mg, 0.211 mmol), and HATU (87 mg, 0.230 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at rt under air. DIPEA (0.100 mL, 0.575 mmol) was added, and the resulting solution was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(2,5-dioxopyrrolidin-1-yl)acetamide <b>28</b> (40 mg, 0.100 mmol, 52% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.85 min, <i>m</i>/<i>z</i> = 400.3. HRMS (C<sub>16</sub>H<sub>18</sub>ClN<sub>3</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 400.0734, found 400.0733. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.76 (s, 1H), 8.06 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.96 (d, <i>J</i> = 2.7 Hz, 1H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 4.41 (s, 2H), 3.37–3.46 (m, 4H), 2.86 (s, 4H), 1.89–1.95 (m, 4H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 176.9, 164.3, 136.8, 136.6, 132.7, 126.5, 124.6, 122.6, 47.9, 41.7, 28.3, 25.7. Mp 195.5–198.5 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3317, 2972, 1708, 1682, 1529, 1421, 1100, 586, 533.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> <i>N</i>-(4-Chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(2-oxopyrrolidin-1-yl)acetamide (<b>29</b>)</h3><div class="NLM_p last">To a solution of 4-chloro-3-(pyrrolidin-1-ylsulfonyl)aniline <b>76</b> (200 mg, 0.767 mmol) and 2-(2-oxopyrrolidin-1-yl)acetic acid (110 mg, 0.767 mmol) in 1,4-dioxane (7 mL) at rt under nitrogen were added DIPEA (0.402 mL, 2.301 mmol) and T3P (50% w/w in EtOAc, 0.686 mL, 1.151 mmol). The resulting solution was stirred at 85 °C for 16 h. The reaction was allowed to cool, diluted with water (50 mL), and extracted with CH<sub>2</sub>Cl<sub>2</sub> (50 mL). The organic phase was washed with water (50 mL) and brine (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated <i>in vacuo</i> to give the crude product. The crude material was purified by silica chromatography, eluting with 80% EtOAc/pet. ether. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(2-oxopyrrolidin-1-yl)acetamide <b>29</b> (51 mg, 0.127 mmol, 17% yield) as an off white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.84 min, <i>m</i>/<i>z</i> = 386.3. HRMS (C<sub>16</sub>H<sub>20</sub>ClN<sub>3</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 386.0941, found 386.0942. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.51 (s, 1H), 8.30 (d, <i>J</i> = 2.4 Hz, 1H), 7.82 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 7.63 (d, <i>J</i> = 8.6 Hz, 1H), 4.06 (s, 2H), 3.45 (t, <i>J</i> = 7.1 Hz, 2H), 3.27–3.33 (m, 4H), 2.28 (t, <i>J</i> = 8.2 Hz, 2H), 1.99 (quin, <i>J</i> = 7.6 Hz, 2H), 1.80–1.89 (m, 4H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 175.1, 167.7, 138.4, 136.8, 133.1, 124.6, 124.3, 121.8, 48.1, 47.9, 46.1, 30.4, 25.6, 18.1;</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> <i>N</i>-(4-Chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)acetamide (<b>30</b>)</h3><div class="NLM_p last">4-Chloro-3-(pyrrolidin-1-ylsulfonyl)aniline <b>76</b> (50 mg, 0.192 mmol) and DIPEA (0.100 mL, 0.575 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (1 mL). Acetic anhydride (0.018 mL, 0.192 mmol) was added, and the resulting solution was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-chloro-3-(pyrrolidin-1-ylsulfonyl)phenyl)acetamide <b>30</b> (32 mg, 0.106 mmol, 55% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.86 min, <i>m</i>/<i>z</i> = 303.1. HRMS (C<sub>12</sub>H<sub>15</sub>ClN<sub>2</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 303.0570, found 303.0571. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.09–8.18 (m, 1H), 7.91–7.99 (m, 2H), 7.48 (d, <i>J</i> = 8.8 Hz, 1H), 3.40–3.47 (m, 4H), 2.23 (s, 3H), 1.89–1.98 (m, 4H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 169.4, 139.0, 136.8, 133.0, 124.1, 124.0, 121.2, 48.1, 25.6, 24.5. Mp 179.5–181.3 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3336, 2981, 1693, 1593, 1527, 1464, 1326, 1156, 595, 542.</div></div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Synthesis of <b>(</b><i><b>S</b></i><b>)-31</b> to <b>33</b>. 1-((2-Bromo-5-nitrophenyl)sulfonyl)pyrrolidine (<b>78</b>)</h3><div class="NLM_p last">2-Bromo-5-nitrobenzene-1-sulfonyl chloride (150 mg, 0.50 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) under nitrogen at 5 °C. DIPEA (0.19 mL, 1.10 mmol) was added and the reaction stirred for 5 min before pyrrolidine (0.04 mL, 0.50 mmol) was added. The reaction was stirred for 1 h and allowed to warm to rt. The reaction mixture was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 10 mL). The combined organics were washed with 2 M HCl aq (10 mL), then filtered through a hydrophobic frit and concentrated <i>in vacuo</i> to give 1-((2-bromo-5-nitrophenyl)sulfonyl)pyrrolidine <b>78</b> (160 mg, 0.48 mmol, 96% yield) as a brown solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.10 min; does not ionize. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): δ (ppm) 8.91 (d, <i>J</i> = 2.7 Hz, 1H) 8.21 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H) 7.95 (d, <i>J</i> = 8.6 Hz, 1H) 3.42–3.56 (m, 4 H) 1.93–2.01 (m, 4 H).</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 4-Bromo-3-(pyrrolidin-1-ylsulfonyl)aniline (<b>79</b>)</h3><div class="NLM_p last">1-((2-Bromo-5-nitrophenyl)sulfonyl)pyrrolidine <b>78</b> (160 mg, 0.48 mmol) was suspended in EtOH (5.0 mL) and water (1.3 mL). Ammonium chloride (38 mg, 0.72 mmol) and iron (133 mg, 2.39 mmol) were added, and the reaction was heated to 70 °C. After 1.5 h the reaction was allowed to cool to rt and was filtered through a Celite cartridge (10 g), washing with MeOH (30 mL). The filtrate was concentrated <i>in vacuo</i> and the residue partitioned between CH<sub>2</sub>Cl<sub>2</sub> and sat. aq NaHCO<sub>3</sub> (25 mL each). The aq layer was re-extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 25 mL). The combined organics were eluted through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–50% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to give 4-bromo-3-(pyrrolidin-1-ylsulfonyl)aniline <b>79</b> (142 mg, 0.47 mmol, 97% yield) as a yellow oil. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.92 min; <i>m</i>/<i>z</i> = 305.0, 307.0. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 7.37–7.49 (m, 2H) 6.66 (dd, <i>J</i> = 8.4, 2.8 Hz, 1H) 3.92 (br s, 2H) 3.32–3.46 (m, 4 H) 1.83–1.97 (m, 4 H).</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> (<i>S*</i>)-<i>N</i>-(4-Bromo-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-31</b>)</h3><div class="NLM_p last">4-Bromo-3-(pyrrolidin-1-ylsulfonyl)aniline <b>79</b> (200 mg, 0.655 mmol), (<i>rac</i>)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetic acid (153 mg, 0.721 mmol), and HATU (299 mg, 0.786 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) at rt under air. DIPEA (0.343 mL, 1.966 mmol) was added, and the resulting solution was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (20 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford the racemic product. The racemate was dissolved in EtOH (1.5 mL) and injected onto a chiral HPLC column (column, 250 mm × 20 mm Chiralcel OJ-H, 5 μm). This was eluted with 30% (EtOH + 0.2% isopropylamine)/(heptane + 0.2% isopropylamine), flow rate = 40 mL min<sup>–1</sup>, detection wavelength 280 nm. The pure fractions from peak 2 were concentrated <i>in vacuo</i> to afford (<i>S</i>*)-<i>N</i>-(4-bromo-3-(pyrrolidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (56 mg, 0.112 mmol, 17% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.96 min, <i>m</i>/<i>z</i> = 499.3, 501.3 (100% pure). <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.78 (s, 1H), 7.95–8.01 (m, 2H), 7.65 (d, <i>J</i> = 9.3 Hz, 1H), 5.83 (s, 1H), 4.37 (s, 2H), 3.38–3.46 (m, 4H), 1.94 (dt, <i>J</i> = 6.9, 3.3 Hz, 4H), 1.56 (s, 3H), 1.22–1.33 (m, 1H), 0.56–0.66 (m, 1H), 0.45–0.54 (m, 2H), 0.34–0.43 (m, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 10 1 MHz): δ (ppm) 176.8, 165.8, 156.0, 138.8, 138.8, 136.7, 124.2, 121.5, 112.6, 63.3, 61.0, 48.2, 25.6, 23.6, 17.4, 0.9. Chiral LC: 4.6 mm × 250 mm Chiralcel OJ-H column, 30% (EtOH + 0.2% isopropylamine)/(heptane + 0.2% isopropylamine), <i>t</i><sub>R</sub> = 22.489 min; er 99:1.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> (<i>S</i>)-1-((2-Bromo-5-nitrophenyl)sulfonyl)-3-methylpyrrolidine (<b>(</b><i><b>S</b></i><b>)-57</b>)</h3><div class="NLM_p last">2-Bromo-5-nitrobenzene-1-sulfonyl chloride <b>56</b> (1.70 g, 5.66 mmol) was taken up in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) under nitrogen and cooled in an ice bath. DIPEA (2.17 mL, 12.45 mmol) was added and the reaction stirred for 5 min before (<i>S</i>)-3-methylpyrrolidine hydrochloride (0.69 g, 5.66 mmol) was added. The reaction was stirred for 1 h before warming to rt. The reaction mixture was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The combined organics were washed with 2 M HCl aq (10 mL), filtered through a hydrophobic frit, and concentrated <i>in vacuo</i> to give the crude product. The crude product was purified by silica chromatography, eluting with 0–40% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-1-((2-bromo-5-nitrophenyl)sulfonyl)-3-methylpyrrolidine <b>(</b><i><b>S</b></i><b>)-57</b> (1.77 g, 5.08 mmol, 90% yield) as a yellow solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.17 min, <i>m</i>/<i>z</i> = 349.1, 351.1. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.89 (d, <i>J</i> = 2.7 Hz, 1H), 8.21 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.95 (d, <i>J</i> = 8.8 Hz, 1H), 3.56–3.67 (m, 2H), 3.45 (ddd, <i>J</i> = 9.5, 8.6, 7.1 Hz, 1H), 2.98 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.28–2.43 (m, 1H), 2.09 (dtd, <i>J</i> = 12.7, 6.6, 3.9 Hz, 1H), 1.53–1.67 (m, 1H), 1.07 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 146.8, 141.2, 127.7, 127.2, 126.6, 126.1, 59.6, 54.8, 47.9, 33.7, 17.4. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3000, 1595, 1528, 1331, 1153, 1125, 1057, 1022, 880, 842, 774, 739, 697, 670.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> (<i>S</i>)-4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline (<b>(</b><i><b>S</b></i><b>)-58</b>)</h3><div class="NLM_p last">(<i>S</i>)-1-((2-Bromo-5-nitrophenyl)sulfonyl)-3-methylpyrrolidine <b>(</b><i><b>S</b></i><b>)-57</b> (2.92 g, 8.36 mmol), ammonium chloride (0.67 g, 12.54 mmol), and iron (2.36 g, 41.8 mmol) were dissolved in a 3:1 mixture of EtOH (20 mL) and water (7 mL). The resulting solution was heated to 70 °C for 2 h. The reaction was allowed to cool, then filtered through a plug of Celite, washing with MeOH (50 mL). The resulting solution was concentrated <i>in vacuo</i> and then partitioned with sat. aq NaHCO<sub>3</sub> (20 mL) and CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The layers were separated and the aq portion was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford (<i>S</i>)-4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (2.62 g, 8.21 mmol, 98% yield) as an orange gum. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.00 min, <i>m</i>/<i>z</i> = 319.1, 321.1. HRMS (C<sub>11</sub>H<sub>15</sub>BrN<sub>2</sub>O<sub>2</sub>S): [M + H]<sup>+</sup> calculated 319.0116, found 319.0114. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 7.41–7.49 (m, 2H), 6.67 (dd, <i>J</i> = 8.4, 2.8 Hz, 1H), 3.92 (br s, 2H), 3.51–3.62 (m, 2H), 3.33–3.44 (m, <i>J</i> = 9.5, 8.6, 7.1 Hz, 1H), 2.88–2.96 (m, 1H), 2.26–2.38 (m, <i>J</i> = 14.9, 8.5, 6.8, 6.8 Hz, 1H), 1.99–2.09 (m, 1H), 1.50–1.57 (m, 1H), 1.05 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 146.0, 138.8, 136.0, 119.4, 118.4, 106.8, 54.4, 47.5, 33.8, 33.6, 17.5. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3467, 3371, 1591, 1468, 1328, 1142, 584.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> <i>N</i>-(4-Bromo-3-(((<i>S</i>)-3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-((<i>R</i>)-4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>(<i>R</i>,<i>S</i>)-32</b> and <i>N</i>-(4-Bromo-3-(((<i>S</i>)-3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-((<i>S</i>)-4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>S,S</b></i><b>)-32</b></h3><div class="NLM_p">(<i>S</i>)-4-Bromo-3-((3-methylpyrrol idin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (265 mg, 0.830 mmol) was taken up in CH<sub>2</sub>Cl<sub>2</sub> (3 mL). (<i>rac</i>)-2-(4-Cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetic acid (194 mg, 0.913 mmol) and DIPEA (0.435 mL, 2.490 mmol) were added. T3P (50% w/w in EtOAc, 1.438 mL, 0.830 mmol) was then added and the reaction stirred at rt for 2 h. The reaction was concentrated <i>in vacuo</i>, and the residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> and sat. aq NaHCO<sub>3</sub> (15 mL each). The aqueous layer was re-extracted with CH<sub>2</sub>Cl<sub>2</sub> (15 mL) and the combined organics were eluted through a hydrophobic frit and then concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 5–50% (3:1 EtOAc:EtOH)/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to give the <i>N</i>-(4-bromo-3-(((<i>S</i>)-3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (326 mg, 0.604 mmol, 73%) as a mixture of diastereomers (1:1). The pure fractions were concentrated <i>in vacuo</i> to afford the racemic product. The racemate was dissolved in EtOH (5 mL) and injected onto a chiral column (column, 250 mm × 20 mm Chiralcpak AD-H, 5 μm). This was eluted with 20% (EtOH + 0.2% isopropylamine)/(heptane + 0.2% isopropylamine), flow rate = 42.5 mL min<sup>–1</sup>, detection wavelength 280 nm. The pure fractions from peak 1 were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-bromo-3-(((<i>S</i>)-3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-((<i>R</i>)-4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>R,S</b></i><b>)-32</b> (134 mg, 0.248 mmol, 30% yield), and the pure fractions from peak 2 were concentrated <i>in vacuo</i> to afford <i>N</i>-(4-bromo-3-(((<i>S</i>)-3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-((<i>S</i>)-4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>(</b><i><b>S,S</b></i><b>)-32</b> (129 mg, 0.238 mmol, 29%) as a cream solid.</div><div id="sec4_5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-(4-Bromo-3-(((<i>S</i>)-3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-((<i>R</i>)-4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>(<i>R</i>,<i>S</i>)-32</b></h4><div class="NLM_p last">LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.08 min, <i>m</i>/<i>z</i> = 513.3, 515.3. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.57 (s, 1H), 7.94–8.03 (m, 2H), 7.61–7.70 (m, 1H), 5.59 (s, 1H), 4.37 (s, 2H), 3.53–3.63 (m, 2H), 3.41 (td, <i>J</i> = 9.0, 6.8 Hz, 1H), 2.94 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.26–2.41 (m, 1H), 2.00–2.13 (m, 1H), 1.58–1.62 (m, 1H), 1.57 (s, 3H), 1.28–1.32 (m, 1H), 1.01–1.09 (m, 3H), 0.58–0.69 (m, 1H), 0.45–0.57 (m, 2H), 0.33–0.43 (m, 1H). Chiral LC: 4.6 mm × 250 mm Chiralpak AD-H column, 20% (EtOH + 0.2% isopropylamine)/(heptane + 0.2% isopropylamine), <i>t</i><sub>R</sub> = 21.224 min; er 99:1.</div></div><div id="sec4_5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>N</i>-(4-Bromo-3-(((<i>S</i>)-3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-((<i>S</i>)-4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>(<i>S</i>,<i>S</i>)-32</b></h4><div class="NLM_p last">LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.04 min, <i>m</i>/<i>z</i> = 513.2, 515.2 (100% pure). HRMS (C<sub>20</sub>H<sub>25</sub>BrN<sub>4</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 513.0807, found 513.0804. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.70 (s, 1H), 8.28 (d, <i>J</i> = 2.7 Hz, 1H), 8.16 (s, 1H), 7.77–7.84 (m, 1H), 7.70 (dd, <i>J</i> = 8.7, 2.6 Hz, 1H), 4.19 (s, 2H), 3.40–3.52 (m, 2H), 3.31–3.35 (m, 1H), 2.84 (dd, <i>J</i> = 9.4, 7.7 Hz, 1H), 2.22–2.35 (m, 1H), 1.95–2.07 (m, 1H), 1.52 (dq, <i>J</i> = 12.3, 8.3 Hz, 1H), 1.40 (s, 3H), 1.10–1.20 (m, 1H), 0.93–1.01 (m, 3H), 0.40–0.49 (m, 2H), 0.26–0.39 (m, 2H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 176.8, 165.8, 156.0, 138.8, 138.8, 136.7, 124.2, 121.5, 112.6, 61.0, 54.7, 47.9, 41.1, 33.6, 33.4, 23.6, 17.6, 17.4, 0.9, 0.4. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3295, 1712, 1601, 1536, 1457, 1307. Mp 218.8–220.5 °C. Chiral LC: 4.6 mm × 250 mm Chiralpak AD-H column, 20% (EtOH + 0.2% isopropylamine)/(heptane + 0.2% isopropylamine), <i>t</i><sub>R</sub> = 23.301 min; er 95:1.</div></div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> (3<i>S</i>*,4<i>S</i>*)-1-((2-Bromo-5-nitrophenyl)sulfonyl)-3,4-dimethylpyrrolidine (<b>80</b>)</h3><div class="NLM_p last">3,4-Dimethylpyrrolidine hydrochloride (0.824 g, 6.08 mmol, ratio ((<i>S,R</i>)-<i>cis</i>:(<i>S</i>,<i>S</i>)/(<i>R</i>,<i>R</i>)-<i>trans</i> (racemic mixture), 2.7:1) was dissolved in dichloromethane (20 mL) at 0 °C, and DIPEA (2.3 mL, 13.17 mmol) was added. 2-Bromo-5-nitrobenzene-1-sulfonyl chloride (1.64 g, 5.46 mmol) was added, and the reaction mixture was stirred and allowed to warm to rt. After 1 h, the reaction mixture was diluted with DCM (10 mL) and washed with a mix of sat. sodium bicarbonate solution (∼20 mL) and brine (∼10 mL). The organic layer was collected and dried by passing through a hydrophobic frit. The solvent was evaporated <i>in vacuo</i> to give the crude product (2.455 g). This was loaded onto a SNAP 100 g silica column in the minimum DCM and eluted with a gradient of 4–19% ethyl acetate/cyclohexane. The appropriate fractions were combined and the solvent was evaporated <i>in vacuo</i> to give 1-((2-bromo-5-nitrophenyl)sulfonyl)-3,4-dimethylpyrrolidine (1.54 g, 4.03 mmol, 73.8% yield) as a yellow solid and a mixture of isomers. 1.269 g of the product was submitted for preparative chiral separation as follows: The sample was dissolved in ethanol (∼40 mL). This was heated during purification using a hot plate (plate temp 60–90 °C). Manual injections (1.2 mL) were made via a Rheodyne valve onto a Chiralpak AS-H column (250 mm × 30 mm, 5 μm); flow rate 42.5 mL/min (pressure, 48 bar); detection, UV DAD (280 nm (bandwidth 140 nm, reference 400 nm (bandwidth 100 nm)), with a mobile phase A of hexane (containing 0.2% v/v isopropylamine) and mobile phase B of ethanol (containing 0.2% v/v isopropylamine) with an isocratic elution method of 95:5 mobile phase A:mobile phase B over 20 min per run. The <i>trans</i>-isomer fractions (as a racemic mixture) were collected between 13.5 and 14.5 min, and the <i>cis</i>-isomer fractions were collected between 16 and 18 min. The <i>trans</i>-isomer fractions were concentrated and further purified to provide desired single enantiomers as follows: the residue was dissolved in ethanol (12 mL). This was kept in solution during purification by use of hot plate (plate temp 60–80 °C). Manual injections (0.4–0.5 mL) were made via a Rheodyne valve onto a Chiralcel OD-H (250 mm × 30 mm, 5 μm) column; flow rate, 42.5 mL/min (pressure, 82 bar); detection, UV DAD (280 nm (bandwidth 140 nm, reference 400 nm (bandwidth 100 nm)), with a mobile phase of hexane:ethanol:isopropylamine of 1000:30:0.2 (premixed) over 30 min per run. The first eluting <i>trans</i>-isomer was collected between 21.5 and 23 min and the second eluting isomer between 23.5 and 25.5 min. Fraction solutions were combined then evaporated <i>in vacuo</i> to provide the first eluting <i>trans-</i>isomer, (3<i>S</i>*,4<i>S</i>*)-1-((2-bromo-5-nitrophenyl)sulfonyl)-3,4-dimethylpyrrolidine, <b>80</b> (124 mg) and the second eluting <i>trans</i>-isomer (123 mg). The first eluting isomer (<b>80</b>) was used in the subsequent reaction, and the absolute stereochemistry was not determined. <b>80</b>: LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.21 min, no <i>m</i>/<i>z</i> at correct mass due to lack of ionization. Chiral HPLC: (conditions, column, Chiralcel OD-H (250 mm × 4.6 mm, 5 μm); flow rate, 1 mL/min; detection, UV DAD (280 nm (bandwidth 140 nm, reference 400 nm (bandwidth 100 nm)); mobile phase A, hexane (containing 0.2% v/v isopropylamine); mobile phase B, ethanol (containing 0.2% v/v isopropylamine); run time 25 min), <i>t</i><sub>R</sub> = 17.8 min, >99% ee.</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 4-Bromo-3-(((3<i>S</i>*,4<i>S</i>*)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)aniline (<b>81</b>)</h3><div class="NLM_p last">(3<i>S</i>*,4<i>S</i>*)-1-((2-Bromo-5-nitrophenyl)sulfonyl)-3,4-dimethylpyrrolidine, <b>80</b> (121 mg, 0.333 mmol) was suspended in ethanol (3 mL) and water (1 mL). Iron powder 325 mesh (96.9 mg, 1.735 mmol) and ammonium chloride (50 mg, 0.935 mmol) were added, and the reaction mixture was sonicated for 2 min. The reaction mixture was then heated to 80 °C under N<sub>2</sub> for 2.5 h. The reaction mixture was then filtered through Celite (2.5 g cartridge) which was prewashed with MeOH. The retentate was then washed with MeOH (∼60 mL). The filtrate was collected, and 1 M HCl in Et<sub>2</sub>O (0.8 mL) added. The solvent and excess HCl were evaporated <i>in vacuo</i>, and the product then dissolved in MeOH (∼10 mL) and loaded onto a 10 g SCX cartridge which was prewashed with MeOH. The cartidge was then washed with MeOH (∼60 mL) followed by 2 M ammonia in MeOH (∼60 mL). The basic wash was concentrated <i>in vacuo</i> to 4-bromo-3-((3,4-dimethylpyrrolidin-1-yl)sulfonyl)aniline, <b>81</b> (109.9 mg, 0.313 mmol, 94% yield) as a pale yellow solid, with resolved unknown <i>trans</i> stereochemistry of the pyrrolidine substituents. LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.09 min, <i>m</i>/<i>z</i> = 333.1, 335.1; (100% pure). <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 7.40–7.49 (m, 2H), 6.67 (dd, <i>J</i> = 8.3, 2.9 Hz, 1H), 3.93 (br s, 2H), 3.62–3.74 (m, 2H), 2.92–3.01 (m, 2H), 1.72–1.89 (m, 2H), 1.01 (d, <i>J</i> = 6.4 Hz, 6H).</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> <i>N</i>-(4-Bromo-3-(((3<i>S</i>*,4<i>S</i>*)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>33</b>)</h3><div class="NLM_p last">To 2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetic acid (52 mg, 0.243 mmol) were added DIPEA (0.085 mL, 0.486 mmol) and T3P (50% w/w, 309 mg, 0.486 mmol). 4-Bromo-3-(((3<i>S</i>*,4<i>S</i>*)-3,4-dimethylpyrrolidin-1-yl)sulfonyl)aniline, <b>81</b> (54 mg, 0.162 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) was added and the resulting solution stirred at rt for 2 h. The reaction was left to stand overnight and then concentrated <i>in vacuo</i>. The crude product was purified by silica chromatography, eluting with 5–50% (3:1 EtOAc:EtOH)/cyclohexane. The appropriate fractions were concentrated <i>in vacuo</i> to give <i>N</i>-(4-bromo-3-((3,4-dimethylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide <b>33</b> (70 mg, 0.126 mmol, 78% yield) as a cream solid as a diastereomeric mixture of hydantoin epimers with resolved unknown <i>trans</i> stereochemistry of the pyrrolidine substituents. LCMS (HPH, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.10 min, <i>m</i>/<i>z</i> = 527.2, 529.2. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.50 (s, 1H), 7.92–8.05 (m, 2H), 7.66 (d, <i>J</i> = 9.0 Hz, 1H), 5.54 (s, 1H), 4.37 (s, 2H), 3.69 (dd, <i>J</i> = 9.4, 7.0 Hz, 2H), 2.98 (t, <i>J</i> = 9.4 Hz, 2H), 1.76–1.89 (m, 2H), 1.57 (s, 3H), 1.25–1.31 (m, 1H), 1.02 (d, <i>J</i> = 6.1 Hz, 6H), 0.58–0.67 (m, 1H), 0.45–0.56 (m, 2H), 0.33–0.42 (m, 1H); diastereotopic protons not visible.</div></div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Synthesis of <b>(</b><i><b>S</b></i><b>)-34</b> to <b>(</b><i><b>S</b></i><b>)-39</b>. (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-chloroacetamide (<b>(</b><i><b>S</b></i><b>)-82</b>)</h3><div class="NLM_p last">DIPEA (0.273 mL, 1.566 mmol) was added to (<i>S</i>)-4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (500 mg, 1.566 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL). 2-Chloroacetyl chloride (0.125 mL, 1.566 mmol) was added, and the reaction was stirred at rt for 1 h. The reaction was quenched with sat. NaHCO<sub>3</sub> (20 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The organic layers were filtered through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. This was purified by silica chromatography, eluting with 0–50% (3:1 EtOH:EtOAc)/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-chloroacetamide <b>(</b><i><b>S</b></i><b>)-82</b> (554 mg, 1.400 mmol, 89% yield) as a yellow oil. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.08 min, <i>m</i>/<i>z</i> = 395.3, 397.3. HRMS (C<sub>13</sub>H<sub>16</sub>BrClN<sub>2</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 394.9832, found 394.9830. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.49 (br s, 1H), 8.10 (d, <i>J</i> = 2.7 Hz, 1H), 7.93 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.73 (d, <i>J</i> = 8.8 Hz, 1H), 4.22 (s, 2H), 3.56–3.64 (m, 2H), 3.43 (td, <i>J</i> = 9.0, 7.2 Hz, 1H), 2.96 (dd, <i>J</i> = 9.0, 8.1 Hz, 1H), 2.28–2.43 (m, 1H), 2.02–2.12 (m, 1H), 1.53–1.61 (m, 1H), 1.07 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 164.3, 139.3, 136.5, 136.3, 124.5, 123.1, 115.1, 54.6, 47.7, 42.8, 33.8, 33.6, 17.4. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3301, 2961, 1718, 1532, 1460, 1150, 591.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-34</b>)</h3><div class="NLM_p last">(<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-chloroacetamide <b>(</b><i><b>S</b></i><b>)-82</b> (50 mg, 0.126 mmol), 1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (19 mg, 0.139 mmol), and K<sub>2</sub>CO<sub>3</sub> (23 mg, 0.164 mmol) were dissolved in acetone (1 mL). The resulting solution was heated to 70 °C and stirred for 1 h. The reaction was allowed to cool before quenching with sat. aq NaHCO<sub>3</sub> (10 mL) and extracting with CH<sub>2</sub>Cl<sub>2</sub> (2 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-34</b> (7 mg, 0.014 mmol, 11% yield) as an off-white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.03 min, <i>m</i>/<i>z</i> = 493.2, 495.2. HRMS (C<sub>20</sub>H<sub>21</sub>BrN<sub>4</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 493.0545, found 493.0544. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.33 (d, <i>J</i> = 2.4 Hz, 1H), 7.69–7.79 (m, 2H), 7.06–7.15 (m, 4H), 4.76 (s, 2H), 3.49–3.60 (m, 2H), 3.35–3.43 (m, 1H), 2.92 (dd, <i>J</i> = 9.4, 7.7 Hz, 1H), 2.25–2.39 (m, 1H), 1.99–2.12 (m, 1H), 1.57 (dd, <i>J</i> = 12.3, 8.4 Hz, 1H), 1.03 (d, <i>J</i> = 6.8 Hz, 3H); amide NH and benzoimidazolone NH not visible.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2-oxobenzo[d]oxazol-3(2<i>H</i>)-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-35</b>)</h3><div class="NLM_p last">(<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-chloroacetamide <b>(</b><i><b>S</b></i><b>)-82</b> (50 mg, 0.126 mmol), benzo[<i>d</i>]oxazol-2(3<i>H</i>)-one (20 mg, 0.152 mmol), and K<sub>2</sub>CO<sub>3</sub> (23 mg, 0.164 mmol) were dissolved in acetone (1 mL). The resulting solution was heated to 60 °C and stirred for 1 h. The reaction was allowed to cool before quenching with sat. aq NaHCO<sub>3</sub> (10 mL) and extracting with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(2-oxobenzo[<i>d</i>]oxazol-3(2<i>H</i>)-yl)acetamide <b>(</b><i><b>S</b></i><b>)-35</b> (35 mg, 0.071 mmol, 56% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.15 min, <i>m</i>/<i>z</i> = 394.0, 396.0. HRMS (C<sub>20</sub>H<sub>20</sub>BrN<sub>3</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calculated 494.0385, found 494.0384. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.95 (s, 1H), 8.11 (d, <i>J</i> = 2.7 Hz, 1H), 7.94 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.64 (d, <i>J</i> = 8.8 Hz, 1H), 7.13–7.27 (m, 3H), 7.10 (dd, <i>J</i> = 7.5, 1.6 Hz, 1H), 4.73 (s, 2H), 3.52–3.63 (m, 2H), 3.40 (td, <i>J</i> = 9.0, 7.1 Hz, 1H), 2.92 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.27–2.38 (m, 1H), 1.98–2.10 (m, 1H), 1.57 (dq, <i>J</i> = 12.3, 8.5 Hz, 1H), 1.04 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 164.5, 155.1, 142.7, 138.6, 137.2, 136.3, 130.8, 124.7, 124.4, 123.2, 123.0, 114.3, 110.4, 109.1, 54.6, 47.8, 45.7, 33.8, 33.6, 17.4.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> (<i>S</i>)-2-(1<i>H</i>-Benzo[<i>d</i>][1,2,3]triazol-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(</b><i><b>S</b></i><b>)-36</b>)</h3><div class="NLM_p last">(<i>S</i>)-4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (50 mg, 0.157 mmol), 2-(1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-1-yl)acetic acid (28 mg, 0.157 mmol), and HATU (89 mg, 0.235 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at rt under air. DIPEA (0.082 mL, 0.470 mmol) was added, and the resulting solution was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-2-(1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-36</b> (59 mg, 0.123 mmol, 79% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.13 min, <i>m</i>/<i>z</i> = 478.1, 480.1. HRMS (C<sub>19</sub>H<sub>20</sub>BrN<sub>5</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 478.0548, found 478.0551. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 11.02 (s, 1H), 8.30 (d, <i>J</i> = 2.8 Hz, 1H), 8.07 (d, <i>J</i> = 8.3 Hz, 1H), 7.80–7.89 (m, 2H), 7.70–7.77 (m, 1H), 7.57 (ddd, <i>J</i> = 8.2, 7.1, 1.0 Hz, 1H), 7.43 (ddd, <i>J</i> = 8.2, 7.1, 1.0 Hz, 1H), 5.74 (s, 2H), 3.37–3.50 (m, 2H), 3.23–3.33 (m, 1H), 2.82 (dd, <i>J</i> = 9.3, 7.8 Hz, 1H), 2.24 (dq, <i>J</i> = 14.7, 7.2 Hz, 1H), 1.90–2.03 (m, 1H), 1.48 (dq, <i>J</i> = 12.3, 8.3 Hz, 1H), 0.93 (d, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 165.0, 145.0, 138.3, 138.1, 136.2, 133.8, 127.4, 124.0, 123.9, 121.6, 119.0, 112.5, 110.9, 54.2, 50.4, 47.3, 33.0, 32.8, 17.1.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> (<i>S</i>)-2-(Benzo[<i>d</i>]isoxazol-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(</b><i><b>S</b></i><b>)-37</b>)</h3><div class="NLM_p last">(<i>S</i>)-4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (50 mg, 0.157 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (3 mL). 2-(Benzo[<i>d</i>]isoxazol-3-yl)acetic acid (28 mg, 0.157 mmol) and DIPEA (0.082 mL, 0.470 mmol) were added, and the solution was stirred at rt for 5 min. T3P (50% w/w in EtOAc, 1.438 mL, 0.157 mmol) was added, and the reaction was stirred at rt for 1 h. The reaction was concentrated <i>in vacuo</i>, and the residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> and sat. NaHCO<sub>3</sub> (15 mL each). The aq layer was re-extracted with CH<sub>2</sub>Cl<sub>2</sub> (15 mL). The combined organics were eluted through a hydrophobic frit and then concentrated <i>in</i><i>vacuo</i> to give the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in</i><i>vacuo</i> to afford (<i>S</i>)-2-(benzo[<i>d</i>]isoxazol-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-37</b> (41 mg, 0.086 mmol, 55% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.21 min; <i>m</i>/<i>z</i> = 478.1, 480.1. HRMS (C<sub>20</sub>H<sub>20</sub>BrN<sub>3</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 478.0436, found 478.0432. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.91 (br s, 1H), 8.33 (d, <i>J</i> = 2.4 Hz, 1H), 7.92 (d, <i>J</i> = 8.1 Hz, 1H), 7.81 (d, <i>J</i> = 8.6 Hz, 1H), 7.72–7.78 (m, 2H), 7.67 (ddd, <i>J</i> = 7.5, 7.5, 1.2 Hz, 1H), 7.41 (dd, <i>J</i> = 7.5, 7.5 Hz, 1H), 4.22 (s, 2H), 3.38–3.51 (m, 2H), 3.25–3.29 (m, 1H), 2.79–2.86 (m, 1H), 2.26 (dq, <i>J</i> = 14.5, 7.2 Hz, 1H), 1.94–2.06 (m, 1H), 1.50 (dq, <i>J</i> = 12.2, 8.3 Hz, 1H), 0.95 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 167.0, 162.9, 154.3, 139.0, 138.8, 136.6, 130.8, 124.4, 124.2, 123.2, 121.9, 121.9, 112.7, 110.1, 54.7, 47.8, 33.7, 33.5, 33.4, 17.6. Mp 61.3–64.9 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3326, 2961, 1674, 1583, 1522, 1459, 1380, 1322, 1141, 750.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> (<i>S</i>)-2-(Benzofuran-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(</b><i><b>S</b></i><b>)-38</b>)</h3><div class="NLM_p last">(<i>S</i>)-4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (50 mg, 0.157 mmol) and 2-(benzofuran-3-yl)acetic acid (33 mg, 0.188 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at rt under air. DIPEA (0.082 mL, 0.470 mmol) was added, followed by T3P (50% w/w in EtOAc, 0.131 mL, 0.219 mmol), and the resulting solution was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-2-(benzofuran-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-38</b> (48 mg, 0.101 mmol, 64% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.27 min, <i>m</i>/<i>z</i> = 477.5, 479.5. HRMS (C<sub>21</sub>H<sub>21</sub>BrN<sub>2</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 477.0484, found 477.0491. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.04 (s, 1H), 7.98 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 7.90 (d, <i>J</i> = 2.5 Hz, 1H), 7.73 (s, 1H), 7.61–7.66 (m, 2H), 7.53 (d, <i>J</i> = 8.1 Hz, 1H), 7.35 (td, <i>J</i> = 7.7, 1.3 Hz, 1H), 7.24–7.31 (m, 1H), 3.84 (s, 2H), 3.48–3.58 (m, 2H), 3.36 (td, <i>J</i> = 9.0, 7.2 Hz, 1H), 2.89 (dd, <i>J</i> = 9.0, 8.2 Hz, 1H), 2.20–2.35 (m, 1H), 1.97–2.06 (m, 1H), 1.53 (dq, <i>J</i> = 12.3, 8.6 Hz, 1H), 1.01 (d, <i>J</i> = 6.5 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 168.5, 155.5, 143.4, 138.6, 137.5, 136.2, 127.2, 125.0, 124.7, 123.1, 122.8, 119.6, 113.9, 113.2, 111.8, 54.6, 47.7, 33.8, 33.6, 32.8, 17.4. Mp 134.0–138.5 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3338, 2964, 1673, 1583, 1521, 1453, 1143, 746, 587.</div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(1<i>H</i>-1,2,3-triazol-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-39</b>)</h3><div class="NLM_p last">(<i>S</i>)-4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (40 mg, 0.125 mmol), 2-(1<i>H</i>-1,2,3-triazol-1-yl)acetic acid, hydrochloride (25 mg, 0.150 mmol), and HATU (72 mg, 0.188 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) at rt under air. DIPEA (0.066 mL, 0.376 mmol) was added, and the resulting solution was stirred at rt for 2 h. The reaction was quenched by the addition of sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(1<i>H</i>-1,2,3-triazol-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-39</b> (36 mg, 0.084 mmol, 67% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.94 min, <i>m</i>/<i>z</i> = 428.2, 430.2. HRMS (C<sub>15</sub>H<sub>18</sub>BrN<sub>5</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 428.0397, found 428.0394. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.33 (s, 1H), 8.23 (d, <i>J</i> = 2.5 Hz, 1H), 7.92 (s, 1H), 7.86 (s, 1H), 7.81 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 7.65 (d, <i>J</i> = 8.6 Hz, 1H), 5.40 (s, 2H), 3.52–3.61 (m, 2H), 3.33–3.44 (m, 1H), 2.91 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.24–2.35 (m, 1H), 1.98–2.09 (m, 1H), 1.55 (dq, <i>J</i> = 12.4, 8.5 Hz, 1H), 1.03 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.8, 138.7, 137.0, 136.2, 134.0, 126.1, 124.8, 123.3, 114.6, 54.6, 53.2, 47.7, 33.8, 33.6, 17.4. Mp 65.8–71.1 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2970, 1704, 1585, 1531, 1459, 1142, 840, 557.</div></div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Synthesis of <b>(</b><i><b>S</b></i><b>)-39</b> to <b>(</b><i><b>S</b></i><b>)-46</b>. (<i>S</i>)-2-Azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(</b><i><b>S</b></i><b>)-65</b>)</h3><div class="NLM_p last">(<i>S</i>)-4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (500 mg, 1.566 mmol), 2-azidoacetic acid (158 mg, 1.566 mmol), and HATU (715 mg, 1.880 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) at rt under air. DIPEA (0.819 mL, 4.70 mmol) was added, and the resulting solution was stirred at rt for 2 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (20 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–100% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-2-azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-65</b> (590 mg, 1.467 mmol, 94% yield) as a yellow oil. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.08 min, <i>m</i>/<i>z</i> = 400.1, 402.1. HRMS (C<sub>13</sub>H<sub>16</sub>BrN<sub>5</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 402.0232, found 402.0231. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.39 (br s, 1H), 8.09 (d, <i>J</i> = 2.7 Hz, 1H), 7.94 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.71 (d, <i>J</i> = 8.8 Hz, 1H), 4.17 (s, 2H), 3.54–3.64 (m, 2H), 3.36–3.47 (m, 1H), 2.95 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.35 (ddd, <i>J</i> = 14.9, 6.6, 2.2 Hz, 1H), 2.03–2.12 (m, 1H), 1.59 (dq, <i>J</i> = 12.3, 8.5 Hz, 1H), 1.06 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 165.1, 139.2, 136.6, 136.3, 124.5, 123.0, 114.7, 54.6, 52.9, 47.7, 33.8, 33.6, 17.4. Mp 80.5–83.2 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3339, 2960, 2097, 1715, 1598, 1530, 1146, 618, 586, 538.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> ((<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4,5-dimethyl-1<i>H</i>-1,2,3-triazol-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-40</b>)</h3><div class="NLM_p last">(<i>S</i>)-2-Azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-65</b> (50 mg, 0.124 mmol), but-2-yne (10 mg, 0.186 mmol), and Cp*RuCl(PPh<sub>3</sub>)<sub>2</sub> (3 mg, 3.73 μmol) were dissolved in THF (1 mL) at rt under nitrogen. The resulting solution was stirred at 65 °C for 2 h. The reaction was quenched with water (2 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 2 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford ((<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4,5-dimethyl-1<i>H</i>-1,2,3-triazol-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-40</b> (35 mg, 0.077 mmol, 62% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.99 min, <i>m</i>/<i>z</i> = 456.2, 458.2. HRMS (C<sub>17</sub>H<sub>22</sub>BrN<sub>5</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 456.0705, found 456.0705. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.44 (s, 1H), 8.20 (d, <i>J</i> = 2.7 Hz, 1H), 7.84 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.65 (d, <i>J</i> = 8.8 Hz, 1H), 5.22 (s, 2H), 3.52–3.61 (m, 2H), 3.35–3.44 (m, 1H), 2.92 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.26–2.37 (m, 7H), 1.99–2.09 (m, 1H), 1.49–1.62 (m, 1H), 1.04 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 164.1, 141.2, 138.9, 137.2, 136.1, 131.3, 124.7, 123.3, 114.5, 54.6, 51.4, 47.7, 33.8, 33.6, 17.4, 10.3, 7.9.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-propyl-1<i>H</i>-1,2,3-triazol-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-41</b>)</h3><div class="NLM_p last">(<i>S</i>)-2-Azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-65</b> (50 mg, 0.124 mmol), pent-1-yne (0.03 mL, 0.249 mmol), sodium ascorbate (4 mg, 0.019 mmol), and Cu(OAc)<sub>2</sub>.H<sub>2</sub>O (3 mg, 0.012 mmol) were dissolved in degassed MeOH (1 mL). The resulting solution was stirred at 100 °C under microwave irradiation for 30 min. The reaction was diluted with water (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-propyl-1<i>H</i>-1,2,3-triazol-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-41</b> (26 mg, 0.055 mmol, 45% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.09 min, <i>m</i>/<i>z</i> = 470.2, 472.2. HRMS (C<sub>18</sub>H<sub>24</sub>BrN<sub>5</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 470.0861, found 470.0863. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.87 (s, 1H), 8.14 (d, <i>J</i> = 2.7 Hz, 1H), 7.82 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 7.67 (d, <i>J</i> = 8.6 Hz, 1H), 7.54 (s, 1H), 5.24 (s, 2H), 3.50–3.63 (m, 2H), 3.40 (td, <i>J</i> = 9.0, 7.1 Hz, 1H), 2.93 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.75 (t, <i>J</i> = 7.6 Hz, 2H), 2.26–2.39 (m, 1H), 2.01–2.11 (m, 1H), 1.71–1.78 (m, 3H), 1.50–1.61 (m, 1H), 0.97–1.07 (m, 5H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.3, 149.0, 139.0, 136.7, 136.2, 124.7, 123.3, 122.7, 114.9, 54.6, 53.4, 47.7, 33.8, 33.6, 27.6, 22.5, 17.4, 13.8. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3276, 2961, 1705, 1586, 1531, 1459, 1143, 650, 536.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-(methoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-42</b>)</h3><div class="NLM_p last">(<i>S</i>)-2-Azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-65</b> (50 mg, 0.124 mmol), 3-methoxyprop-1-yne (10 mg, 0.137 mmol), CuSO<sub>4</sub>·5H<sub>2</sub>O (2 mg, 6.21 μmol), and sodium ascorbate (4 mg, 0.019 mmol) were dissolved in a mixture of CH<sub>2</sub>Cl<sub>2</sub> (1.0 mL) and water (1.0 mL). The resulting solution was stirred at rt for 16 h. The reaction was diluted with water (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-(methoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-42</b> (29 mg, 0.061 mmol, 49% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.96 min, <i>m</i>/<i>z</i> = 472.2, 474.2. HRMS (C<sub>17</sub>H<sub>22</sub>BrN<sub>5</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 472.0654, found 472.0658. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.17 (s, 1H), 8.19 (d, <i>J</i> = 2.7 Hz, 1H), 7.81–7.92 (m, 2H), 7.67 (d, <i>J</i> = 8.6 Hz, 1H), 5.33 (s, 2H), 4.64 (s, 2H), 3.53–3.62 (m, 2H), 3.44 (s, 3H), 3.36–3.43 (m, 1H), 2.93 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.28–2.39 (m, 1H), 2.00–2.10 (m, 1H), 1.51–1.64 (m, 1H), 1.04 (d, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.7, 145.6, 138.8, 137.0, 136.2, 124.8, 124.5, 123.3, 114.6, 65.8, 58.5, 54.6, 53.3, 47.8, 33.8, 33.6, 17.4. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2964, 1706, 1586, 1532, 1459, 1139, 620, 536.</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-((dimethylamino)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)acetamide, Formic Acid Salt (<b>(</b><i><b>S</b></i><b>)-43</b>)</h3><div class="NLM_p last">(<i>S</i>)-2-Azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-65</b> (50 mg, 0.124 mmol), <i>N</i>,<i>N</i>-dimethylprop-2-yn-1-amine (0.02 mL, 0.137 mmol), CuSO<sub>4</sub>·5H<sub>2</sub>O (2 mg, 6.21 μmol), and sodium ascorbate (4 mg, 0.019 mmol) were dissolved in a mixture of CH<sub>2</sub>Cl<sub>2</sub> (1.0 mL) and water (1.0 mL). The resulting solution was stirred at rt for 16 h. The reaction was diluted with water (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4-((dimethylamino)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)acetamide, formic acid salt <b>(</b><i><b>S</b></i><b>)-43</b> (39 mg, 0.073 mmol, 59% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.60 min, <i>m</i>/<i>z</i> = 485.2, 487.2. HRMS (C<sub>18</sub>H<sub>25</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 485.0970, found 485.0975. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 10.61 (br s, 1H), 8.48 (s, 1H), 8.24 (d, <i>J</i> = 2.7 Hz, 1H), 7.81 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.61 (d, <i>J</i> = 8.8 Hz, 1H), 5.36 (s, 2H), 4.21 (s, 2H), 3.46–3.58 (m, 2H), 3.36 (td, <i>J</i> = 8.9, 7.1 Hz, 1H), 2.89 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.73 (s, 6H), 2.23–2.36 (m, 1H), 1.96–2.08 (m, 1H), 1.54 (dq, <i>J</i> = 12.3, 8.5 Hz, 1H), 1.02 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.6, 139.0, 137.4, 136.3, 134.7, 133.9, 124.5, 122.7, 114.3, 54.5, 52.6, 51.3, 47.6, 45.0, 33.8, 33.6, 17.4.</div></div><div id="sec4_7_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-propyl-1<i>H</i>-1,2,3-triazol-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-44</b>)</h3><div class="NLM_p last">(<i>S</i>)-2-Azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-65</b> (40 mg, 0.099 mmol), pent-1-yne (0.02 mL, 0.149 mmol), and Cp*RuCl(PPh<sub>3</sub>)<sub>2</sub> (2 mg, 2.98 μmol) were dissolved in THF (1 mL) at rt under nitrogen. The resulting solution was stirred at 65 °C for 2 h. The reaction was quenched with water (2 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 2 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-propyl-1<i>H</i>-1,2,3-triazol-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-44</b> (24 mg, 0.051 mmol, 51% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.08 min, <i>m</i>/<i>z</i> = 470.3, 472.3. HRMS (C<sub>18</sub>H<sub>24</sub>BrN<sub>5</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 470.0861, found 470.0865. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 10.94 (s, 1H), 8.30 (d, <i>J</i> = 2.6 Hz, 1H), 7.83 (d, <i>J</i> = 8.6 Hz, 1H), 7.73 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 7.57 (s, 1H), 5.29 (s, 2H), 3.39–3.52 (m, 2H), 3.30–3.34 (m, 1H), 2.84 (dd, <i>J</i> = 9.4, 7.5 Hz, 1H), 2.63 (t, <i>J</i> = 7.5 Hz, 2H), 2.18–2.34 (m, 1H), 1.93–2.07 (m, 1H), 1.63–1.64 (m, 2H), 1.51 (dq, <i>J</i> = 12.3, 8.3 Hz, 1H), 0.89–1.03 (m, 6H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 165.4, 139.1, 138.9, 138.6, 136.7, 131.9, 124.4, 121.9, 113.0, 54.7, 50.6, 47.8, 33.6, 33.4, 24.7, 21.3, 17.6, 14.0. Mp 145.0–150.0 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2970, 1693, 1531, 1461, 584.</div></div><div id="sec4_7_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-(methoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-45</b>)</h3><div class="NLM_p last">(<i>S</i>)-2-Azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (50 mg, 0.124 mmol), 3-methoxyprop-1-yne <b>(</b><i><b>S</b></i><b>)-65</b> (13 mg, 0.186 mmol), and Cp*RuCl(PPh<sub>3</sub>)<sub>2</sub> (3 mg, 3.73 μmol) were dissolved in THF (1 mL) at rt under nitrogen. The resulting solution was stirred at 65 °C for 18 h. The reaction was quenched with water (2 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 2 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-(methoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-45</b> (13 mg, 0.028 mmol, 22% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.97 min, <i>m</i>/<i>z</i> = 472.2, 474.2. HRMS (C<sub>17</sub>H<sub>22</sub>BrN<sub>5</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 472.0654, found 472.0651. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.78 (s, 1H), 8.10 (d, <i>J</i> = 2.7 Hz, 1H), 7.87 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 7.75 (s, 1H), 7.68 (d, <i>J</i> = 8.6 Hz, 1H), 5.34 (s, 2H), 4.65 (s, 2H), 3.53–3.63 (m, 2H), 3.37–3.46 (m, 4H), 2.94 (dd, <i>J</i> = 9.4, 7.9 Hz, 1H), 2.27–2.40 (m, 1H), 2.01–2.11 (m, 1H), 1.52–1.64 (m, 1H), 1.05 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.8, 139.0, 136.9, 136.2, 134.6, 134.1, 124.7, 123.1, 114.7, 62.6, 58.7, 54.6, 52.0, 47.7, 33.8, 33.6, 17.4. Mp 63.1–69.9 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2970, 1699, 1585, 1532, 1460, 1153, 585.</div></div><div id="sec4_7_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-((dimethylamino)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)acetamide, Formic Acid Salt (<b>(</b><i><b>S</b></i><b>)-46</b>)</h3><div class="NLM_p last">(<i>S</i>)-2-Azido-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-65</b> (150 mg, 0.373 mmol), <i>N</i>,<i>N</i>-dimethylprop-2-yn-1-amine (47 mg, 0.559 mmol), and Cp*RuCl(PPh<sub>3</sub>)<sub>2</sub> (9 mg, 0.011 mmol) were dissolved in THF (3 mL) at rt under nitrogen. The resulting solution was stirred at 65 °C for 2 h. The reaction was quenched with water (2 mL) and extracted with CH<sub>2</sub>Cl<sub>c</sub> (2 × 2 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–25% EtOH/EtOAc. The pure fractions were concentrated in <i>vacuo</i> to afford the second crude product. The second crude was purified by HPH MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-((dimethylamino)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-46</b> (82 mg, 0.169 mmol, 45% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.63 min, <i>m</i>/<i>z</i> = 485.3, 487.2. HRMS (C<sub>18</sub>H<sub>25</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 485.0970, found 485.0969. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.28 (br s, 1H), 7.76–7.84 (m, 2H), 7.64 (d, <i>J</i> = 8.6 Hz, 1H), 7.28 (s, 1H), 5.41 (s, 2H), 3.95 (s, 2H), 3.49–3.59 (m, 2H), 3.37 (td, <i>J</i> = 9.0, 7.1 Hz, 1H), 2.90 (dd, <i>J</i> = 9.0, 7.8 Hz, 1H), 2.54 (s, 6H), 2.25–2.38 (m, 1H), 1.98–2.10 (m, 1H), 1.55 (dd, <i>J</i> = 12.5, 8.6 Hz, 1H), 1.04 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 164.1, 138.9, 137.4, 136.2, 135.2, 132.4, 124.6, 122.9, 114.3, 54.5, 52.1, 50.2, 47.6, 44.2, 33.8, 33.6, 17.4.</div></div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Synthesis of <b>(</b><i><b>S</b></i><b>)-47</b> to <b>(</b><i><b>S</b></i><b>)-53</b>. (<i>S</i>)-2-(1<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(<i>S</i>)-83</b>) and (<i>S</i>)-2-(3<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-3-yl)<i>-N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(<i>S</i>)-84</b>)</h3><div id="sec4_8_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Method 1</h4><div class="NLM_p last">(<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-chloroacetamide <b>(</b><i><b>S</b></i><b>)-82</b> (350 mg, 0.885 mmol), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridine (127 mg, 1.061 mmol), and K<sub>2</sub>CO<sub>3</sub> (159 mg, 1.150 mmol) were dissolved in acetone (3 mL). The resulting solution was heated to 80 °C and stirred for 12 h. The reaction was allowed to cool before quenching with sat. aq NaHCO<sub>3</sub> (10 mL) and extracting with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by TFA MDAP. Fractions containing the desired products were collected separately and concentrated <i>in vacuo</i> to afford (<i>S</i>)-2-(3<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-3-yl)<i>-N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (59 mg, 0.123 mmol, 14% yield) <b>(</b><i><b>S</b></i><b>)-84</b> as a white solid which required further purification. This was repurified using formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-83</b> (67 mg, 0.140 mmol, 16% yield) as a white solid.</div></div><div id="sec4_8_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> Method 2: Synthesis of (<i>S</i>)-2-(1<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(</b><i><b>S</b></i><b>)</b><i><b>-</b></i><b>83</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl 2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetate (176 mg, 0.752 mmol) was dissolved in HCl (4 M in dioxane, 1.566 mL, 6.27 mmol) and stirred at 40 °C overnight. The reaction was concentrated <i>in vacuo</i> to afford the crude acid. (<i>S</i>)-4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>S</b></i><b>)-58</b> (200 mg, 0.627 mmol) dissolved in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was added to the crude acid. HATU (286 mg, 0.752 mmol) and DIPEA (0.328 mL, 1.880 mmol) were added, and the resulting solution was stirred at rt for 2 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–50% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-83</b> (220 mg, 0.459 mmol, 73% yield) as a cream solid.</div></div><div id="sec4_8_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> (<i>S</i>)-2-(1<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>((<i>S</i>)-83)</b></h4><div class="NLM_p last">LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.91 min, <i>m</i>/<i>z</i> = 479.2, 481.2. HRMS (C<sub>18</sub>H<sub>19</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 479.0501, found 479.0500. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 9.89–10.00 (m, 1H), 8.67–8.80 (m, 1H), 8.38 (d, <i>J</i> = 2.5 Hz, 2H), 7.76 (d, <i>J</i> = 8.7 1H), 7.67 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 5.93 (s, 2H), 3.47–3.57 (m, 2H), 3.34–3.41 (m, 1H), 2.90 (dd, <i>J</i> = 9.3, 7.8 Hz, 1H), 2.24–2.36 (m, 1H), 2.00–2.04 (m, 1H), 1.47–1.61 (m, 1H), 1.01 (d, <i>J</i> = 6.6 Hz, 3H); amide NH not visible. <sup>13</sup>C NMR (MeOD-<i>d</i><sub>4</sub>, 101 MHz): δ (ppm) 164.0, 142.1, 140.9, 140.0, 138.9, 137.7, 136.8, 136.0, 124.2, 122.3, 113.6, 109.5, 54.3, 50.9, 47.4, 33.6, 33.1, 16.2. Mp 141.5–143.0 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3572, 2957, 1704, 1586, 1610, 1462, 1316, 1138, 818, 618.</div></div><div id="sec4_8_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> (<i>S</i>)-2-(3<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>((<i>S</i>)-84)</b></h4><div class="NLM_p last">LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.98 min, <i>m</i>/<i>z</i> = 479.2, 481.2. HRMS (C<sub>18</sub>H<sub>19</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 479.0501, found 479.0501. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ (ppm) 11.09 (s, 1H), 9.47 (d, <i>J</i> = 1.5 Hz, 1H), 8.55 (d, <i>J</i> = 5.9 Hz, 1H), 8.31 (d, <i>J</i> = 2.7 Hz, 1H), 8.16 (dd, <i>J</i> = 5.9, 1.5 Hz, 1H), 7.84 (d, <i>J</i> = 8.7 Hz, 1H), 7.73 (dd, <i>J</i> = 8.7, 2.7 Hz, 1H), 5.91 (s, 2H), 3.38–3.50 (m, 2H), 3.23–3.33 (m, 1H), 2.82 (dd, <i>J</i> = 9.4, 7.7 Hz, 1H), 2.18–2.31 (m, 1H), 1.93–2.03 (m, 1H), 1.48 (dq, <i>J</i> = 12.2, 8.3 Hz, 1H), 0.94 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 164.7, 148.2, 141.2, 138.3, 138.1, 136.7, 136.3, 131.1, 124.0, 121.6, 113.3, 112.6, 54.2, 51.3, 47.3, 33.1, 32.8, 17.1.</div></div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-3<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-3-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-47</b>)</h3><div class="NLM_p last">Iodomethane (6.23 μL, 0.100 mmol) was added to (<i>S</i>)-2-(<i>3H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-3-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-84</b> (40 mg, 0.083 mmol) in MeCN (2 mL) at rt under air. The resulting solution was refluxed at 80 °C for 8 h. The reaction mixture was allowed to cool and concentrated <i>in vacuo</i> to afford the methylpyridinium intermediate. The residue was dissolved in a 1:1 mixture of MeOH (1 mL) and water (1 mL) before NaBH<sub>4</sub> (7 mg, 0.184 mmol) was added, and the resulting solution was stirred at rt for 30 min. The reaction was quenched with water (5 mL) and extracted with EtOAc (3 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by HPH MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-<i>3H</i>-[1,2,3]triazol [4,5-<i>c</i>]pyridin-3-yl) acetamide <b>(</b><i><b>S</b></i><b>)-47</b> (12 mg, 0.025 mmol, 29% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.66 min, <i>m</i>/<i>z</i> = 497.2, 499.2. HRMS (C<sub>19</sub>H<sub>25</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 497.0969, found 497.0970. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.27 (br s, 1H), 8.20 (d, <i>J</i> = 2.7 Hz, 1H), 7.86 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.67 (d, <i>J</i> = 8.8 Hz, 1H), 5.18 (s, 2H), 3.66 (s, 2H), 3.52–3.63 (m, 2H), 3.35–3.46 (m, 1H), 2.89–2.99 (m, 3H), 2.76–2.86 (m, 2H), 2.54 (s, 3H), 2.26–2.39 (m, 1H), 1.99–2.11 (m, 1H), 1.58 (dq, <i>J</i> = 12.3, 8.5 Hz, 1H), 1.05 (d, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.6, 142.2, 139.0, 137.0, 136.2, 132.6, 124.7, 123.3, 114.7, 54.6, 52.1, 51.6, 49.5, 47.7, 45.1, 33.8, 33.6, 22.2, 17.4. Mp 106.5–109.5 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2970, 1705, 1585, 1534, 1460, 1148, 621, 589.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-48</b>)</h3><div class="NLM_p last">Iodomethane (8 μL, 0.125 mmol) was added to (<i>S</i>)-2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-83</b> (50 mg, 0.104 mmol) in MeCN (2 mL) at rt under air. The resulting solution was refluxed at 80 °C for 4 h. The reaction was allowed to cool and concentrated <i>in vacuo</i> to afford the methylpyridinium intermediate. The residue was dissolved in a 1:1 mixture of MeOH (1 mL) and water (1 mL) before NaBH<sub>4</sub> (9 mg, 0.184 mmol) was added, and the resulting solution was stirred at rt for 30 min. The reaction was quenched with water (5 mL) and extracted with EtOAc (3 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by HPH MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-48</b> (20 mg, 0.039 mmol, 38% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.62 min, <i>m</i>/<i>z</i> = 497.2, 499.2. HRMS (C<sub>19</sub>H<sub>24</sub>BrN<sub>5</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 497.0967, found 497.0964. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.36 (d, <i>J</i> = 2.7 Hz, 1H), 7.65–7.80 (m, 2H), 5.29 (s, 2H), 3.66 (s, 2H), 3.49–3.62 (m, 2H), 3.39 (ddd, <i>J</i> = 9.6, 8.4, 7.2 Hz, 1H), 2.82–2.96 (m, 5H), 2.51–2.57 (m, 3H), 2.27–2.40 (m, 1H), 1.99–2.12 (m, 3H), 1.51–1.65 (m, 1H), 1.04 (d, <i>J</i> = 6.6 Hz, 3H); amide NH not visible. <sup>13</sup>C NMR (MeOD-<i>d</i><sub>4</sub>, 101 MHz): δ (ppm) 164.5, 141.2, 139.0, 137.8, 136.0, 132.4, 124.0, 122.1, 113.5, 54.3, 51.1, 50.6, 50.2, 47.4, 43.6, 33.6, 33.1, 19.7, 16.3. Mp 71.2–75.2 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2970, 1704, 1586, 1532, 1459, 1304, 1143, 620, 587.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-ethyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-49</b>)</h3><div class="NLM_p last">Iodoethane (33 mg, 0.209 mmol) was added to (<i>S</i>)-2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-83</b> (100 mg, 0.209 mmol) in MeCN (2 mL) at rt under air. The resulting solution was refluxed for 18 h. The reaction was allowed to cool and concentrated <i>in vacuo</i> to afford the ethylpyridinium intermediate. The intermediate was dissolved in a 1:1 mixture of MeOH (1 mL) and water (1 mL), NaBH<sub>4</sub> (17 mg, 0.459 mmol) was added, and the resulting solution was stirred at rt for 30 min. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by HPH MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-ethyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo [4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-49</b> (21 mg, 0.041 mmol, 20% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.63 min, <i>m</i>/<i>z</i> = 511.3, 513.3. HRMS (C<sub>20</sub>H<sub>27</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 511.1114, found 511.1125. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.75 (br s, 1H), 8.22 (d, <i>J</i> = 2.7 Hz, 1H), 7.78 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.62 (d, <i>J</i> = 8.8 Hz, 1H), 5.20 (s, 2H), 3.78 (s, 2H), 3.48–3.58 (m, 2H), 3.34–3.41 (m, 1H), 2.82–2.96 (m, 5H), 2.76 (q, <i>J</i> = 7.3 Hz, 2H), 2.23–2.36 (m, 1H), 1.96–2.07 (m, 1H), 1.54 (dd, <i>J</i> = 12.5, 8.6 Hz, 1H), 1.18–1.25 (m, 3H), 1.02 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 164.0, 142.0, 138.8, 137.3, 136.1, 132.6, 124.7, 123.1, 114.3, 54.6, 51.2, 49.3, 48.7, 47.7, 33.7, 33.6, 20.3, 17.4, 12.2, 1.9. Mp 88.0–90.4 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3262, 2965, 1704, 1586, 1533, 1459.</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> <i>tert</i>-Butyl (3-Nitropyridin-4-yl)glycinate (<b>67</b>)</h3><div class="NLM_p last">4-Chloro-3-nitropyridine <b>66</b> (3.00 g, 18.92 mmol) and <i>tert</i>-butyl glycinate, hydrochloride (3.33 g, 19.87 mmol) were dissolved in EtOH (20 mL). TEA (2.64 mL, 18.92 mmol) was added and the reaction refluxed for 6 h. The reaction was allowed to cool before concentrating <i>in vacuo</i>. The residue was partitioned between sat. aq NaHCO<sub>3</sub> (20 mL) and CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The layers were separated, and the aq layer was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude was purified by silica chromatography, eluting with 0–80% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford <i>tert</i>-butyl (3-nitropyridin-4-yl)glycinate <b>67</b> (1.58 g, 6.24 mmol, 33% yield) as a yellow solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.61 min, <i>m</i>/<i>z</i> = 254.2. HRMS (C<sub>11</sub>H<sub>15</sub>N<sub>3</sub>O<sub>4</sub>): [M + H]<sup>+</sup> calculated 254.1141, found 254.1136. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.27 (s, 1H), 8.59 (s, 1H), 8.36 (d, <i>J</i> = 6.1 Hz, 1H), 6.58 (d, <i>J</i> = 6.1 Hz, 1H), 4.02 (d, <i>J</i> = 5.1 Hz, 2H), 1.55 (s, 9H). <sup>13</sup>C NMR (DMSO<i>-d</i><sub>6</sub>, 101 MHz): δ (ppm) 168.6, 153.3, 149.0, 148.5, 130.0, 109.6, 82.3, 44.9, 28.2. Mp 91.2–93.3 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3346, 1727, 1623, 1355, 1164, 826, 550.</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> <i>tert</i>-Butyl (3-Aminopyridin-4-yl)glycinate (<b>68</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl (3-nitropyridin-4-yl)glycinate <b>67</b> (1.50 g, 5.92 mmol) was suspended in EtOH (30 mL) and water (8 mL). Ammonium chloride (0.48 g, 8.88 mmol) and iron (1.65 g, 29.6 mmol) were added, and the reaction was heated to 70 °C for 2 h. The reaction mixture was filtered through a 10 g Celite cartridge washing with MeOH (30 mL). The filtrate was concentrated <i>in vacuo</i> and the residue partitioned between CH<sub>2</sub>Cl<sub>2</sub> and water (25 mL each). The aq layer was re-extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 25 mL). The combined organics were eluted through a hydrophobic frit and concentrated <i>in vacuo</i> to give <i>tert</i>-butyl (3-aminopyridin-4-yl)glycinate <b>68</b> (1.21 g, 5.42 mmol, 91% yield) as an orange gum. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.39 min, <i>m</i>/<i>z</i> = 224.3. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.00 (d, <i>J</i> = 5.4 Hz, 1H), 7.95 (s, 1H), 6.37 (d, <i>J</i> = 5.4 Hz, 1H), 4.67–4.76 (m, 1H), 3.86 (d, <i>J</i> = 5.1 Hz, 2H), 3.18 (s, 2H), 1.53 (s, 9H).</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> <i>tert</i>-Butyl 2-(1<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-1-yl)acetate (<b>69</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl (3-aminopyridin-4-yl)glycinate <b>68</b> (1.00 g, 4.48 mmol) was suspended in water (10 mL) at 0 °C under air. AcOH (10 mL) was added, followed by sodium nitrite (0.371 g, 5.37 mmol), and the resulting suspension was then placed in a preheated stirrer hot plate and stirred at 80 °C for 10 min before it was allowed to cool to rt. The reaction was diluted with water (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–100% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford <i>tert</i>-butyl 2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetate <b>69</b> (390 mg, 1.665 mmol, 37% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.68 min, <i>m</i>/<i>z</i> = 235.2. HRMS (C<sub>11</sub>H<sub>14</sub>N<sub>4</sub>O<sub>2</sub>): [M + H]<sup>+</sup> calculated 235.1195, found 235.1192. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.51 (d, <i>J</i> = 1.5 Hz, 1H), 8.62 (d, <i>J</i> = 5.9 Hz, 1H), 7.44 (dd, <i>J</i> = 5.9, 1.5 Hz, 1H), 5.36 (s, 2H), 1.48 (s, 9H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 101 MHz): δ (ppm) 166.4, 145.2, 144.3, 142.9, 137.5, 106.1, 83.2, 49.8, 28.0. Mp 96.7–98.2 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2978, 1745, 1607, 1369, 1234, 1154, 1081.</div></div><div id="sec4_8_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-53</b>)</h3><div id="sec4_8_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> Method 1</h4><div class="NLM_p last">(<i>S</i>)-2-(1<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-83</b> (30 mg, 0.063 mmol) was dissolved in a 1:1:1 mixture of MeOH, EtOH, and AcOH (12 mL). The resulting solution was hydrogenated over Rh/C on the H-cube apparatus with a flow rate of 1 mL min<sup>–1</sup>. The apparatus was set up to allow the reaction mixture to cycle through the H-cube for 24 h. The reaction mixture was then concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by HPH MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (S)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-53</b> (9 mg, 0.018 mmol, 29% yield) as a cream solid.</div></div><div id="sec4_8_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Method 2</h4><div class="NLM_p last">(<i>S</i>)-2-(1<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-83</b> (750 mg, 1.565 mmol) and 5% Pt/C B103032-5 JM26 (75 mg, 1.565 mmol) were dissolved in MeOH (3.6 mL) and AcOH (0.1 mL). The resulting solution was placed in a HP ChemSCAN reactor and stirred at 50 °C under 240 psi of hydrogen for 48 h. The reaction was filtered through Celite, washing with MeOH, and concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-53</b> (500 mg, 1.034 mmol, 66% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.61 min, <i>m</i>/<i>z</i> = 483.2, 485.2. HRMS (C<sub>18</sub>H<sub>23</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 483.0814, found 483.0814. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.31 (d, <i>J</i> = 2.5 Hz, 1H), 7.82 (d, <i>J</i> = 8.7 Hz, 1H), 7.73 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 5.28 (s, 2H), 3.95–4.00 (m, 2H), 3.50–3.60 (m, 2H), 3.37 (s, 2H), 3.12 (s, 2H), 2.88–2.97 (m, 1H), 2.76–2.82 (m, 2H), 2.28–2.40 (m, 1H), 2.02–2.13 (m, 1H), 1.52–1.65 (m, 1H), 1.04 (d, <i>J</i> = 6.6 Hz, 3H); amide NH not visible. <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.9, 142.7, 138.9, 137.1, 136.2, 132.9, 124.7, 123.2, 114.5, 54.6, 51.1, 47.7, 42.4, 42.3, 33.8, 33.6, 21.6, 17.4. Mp 140.0–144.8 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2959, 1701, 1586, 1533, 1459, 1301, 1149, 621.</div></div></div><div id="sec4_8_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-isopropyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-50</b>)</h3><div class="NLM_p last">(<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-53</b> (35 mg, 0.072 mmol) and K<sub>2</sub>CO<sub>3</sub> (30 mg, 0.217 mmol) were dissolved in acetone (1 mL). 2-Iodopropane (18 mg, 0.109 mmol) was added and the resulting solution stirred at 80 °C for 4 h. A further 1 equiv. of 2-iodopropane was added, and the resulting solution stirred at 80 °C for a further 4 h. The reaction was allowed to cool, diluted with water (2 mL), and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 2 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by HPH MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-isopropyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-50</b> (22 mg, 0.042 mmol, 58% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.64 min, <i>m</i>/<i>z</i> = 525.3, 527.3. HRMS (C<sub>21</sub>H<sub>29</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 525.1283, found 525.1280. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.36 (d, <i>J</i> = 2.7 Hz, 1H), 7.77 (s, 1H), 7.70–7.75 (m, 1H), 3.78 (s, 2H), 3.50–3.59 (m, 2H), 3.36–3.44 (m, 1H), 3.01–3.10 (m, 1H), 2.93 (d, <i>J</i> = 5.1 Hz, 3H), 2.81–2.87 (m, 2H), 2.29–2.40 (m, 1H), 2.08–2.12 (m, 1H), 2.02–2.04 (m, 1H), 1.52–1.64 (m, 1H), 1.17–1.22 (m, 7H), 1.05 (d, <i>J</i> = 6.6 Hz, 3H); amide NH not visible. <sup>13</sup>C NMR (MeOD-<i>d</i><sub>4</sub>, 101 MHz): δ (ppm) 164.6, 142.0, 139.0, 137.8, 136.0, 133.1, 124.1, 122.3, 116.7, 56.9, 54.3, 53.8, 50.2, 45.4, 44.1, 33.7, 33.0, 20.4, 17.2, 16.2.</div></div><div id="sec4_8_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-(2,2,2-trifluoroethyl)-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-51</b>)</h3><div class="NLM_p last">(<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-53</b> (40 mg, 0.083 mmol) was dissolved in THF (1 mL) at rt under nitrogen. The resulting solution was heating to 70 °C, followed by sequential addition of phenylsilane (0.02 mL, 0.165 mmol) and TFA (0.01 mL, 0.145 mmol). The resulting solution was stirred at 70 °C for 1 h. The reaction was allowed to cool, quenched with sat. aq NaHCO<sub>3</sub> (2 mL), and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 2 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-(2,2,2-trifluoroethyl)-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-51</b> (20 mg, 0.035 mmol, 43% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.10 min, <i>m</i>/<i>z</i> = 565.0, 567.0. HRMS (C<sub>20</sub>H<sub>24</sub>BrF<sub>3</sub>N<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 565.0844, found 565.0842. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ (ppm) 8.36 (d, <i>J</i> = 2.5 Hz, 1H), 7.78 (d, <i>J</i> = 8.6 Hz, 1H), 7.72 (dd, <i>J</i> = 8.6, 2.4 Hz, 1H), 5.29 (s, 2H), 3.94 (s, 2H), 3.51–3.61 (m, 2H), 3.34–3.43 (m, <i>J</i> = 9.8 Hz, 3H), 3.11 (t, <i>J</i> = 5.9 Hz, 2H), 2.93 (dd, <i>J</i> = 9.5, 7.8 Hz, 1H), 2.85 (t, <i>J</i> = 5.7 Hz, 2H), 2.29–2.39 (m, 1H), 2.06–2.11 (m, 1H), 1.53–1.64 (m, 1H), 1.05 (d, <i>J</i> = 6.6 Hz, 3H); amide NH not visible. <sup>13</sup>C NMR (MeOD-<i>d</i><sub>4</sub>, 101 MHz): δ (ppm) 165.0, 141.0, 138.9, 137.7, 135.9, 132.7, 124.3, 122.2, 113.7, 56.2, 55.8, 54.4, 50.2, 49.7, 49.1, 33.6, 33.2, 26.6, 19.8, 16.3; <sup>19</sup>F NMR (MeOD-<i>d</i><sub>4</sub>, 376 MHz): δ (ppm) −71.50.</div></div><div id="sec4_8_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> (<i>S</i>)-2-(5-Acetyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(</b><i><b>S</b></i><b>)-52</b>)</h3><div class="NLM_p last">(<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-methylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-53</b> (35 mg, 0.072 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (1 mL) at rt under nitrogen. DIPEA (0.038 mL, 0.217 mmol) was added followed by Ac<sub>2</sub>O (10 μL, 0.109 mmol). The resulting solution was stirred at rt for 1 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (2 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 2 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by formic MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>S</i>)-2-(5-acetyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-methy lpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>S</b></i><b>)-52</b> (20 mg, 0.039 mmol, 53% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.89 min, <i>m</i>/<i>z</i> = 525.2, 527.2. HRMS (C<sub>20</sub>H<sub>25</sub>BrN<sub>6</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 525.0920, found 525.0920.. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 393 K): δ (ppm) 8.29 (d, <i>J</i> = 2.7 Hz, 1H), 7.78 (d, <i>J</i> = 8.6 Hz, 1H), 7.71 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 4.64 (s, 2H), 3.79 (t, <i>J</i> = 5.7 Hz, 2H), 3.42–3.55 (m, 2H), 3.29–3.39 (m, 1H), 2.89 (dd, <i>J</i> = 9.7, 7.2 Hz, 1H), 2.81 (t, <i>J</i> = 5.7 Hz, 4H), 2.31 (dd, <i>J</i> = 14.3, 7.2 Hz, 1H), 2.12 (s, 3H), 1.99–2.09 (m, 1H), 1.52 (dd, <i>J</i> = 12.2, 7.8 Hz, 1H), 1.01 (d, <i>J</i> = 6.8 Hz, 3H); amide NH not visible.</div></div><div id="sec4_9_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> Synthesis of <b>(</b><i><b>R</b></i><b>)-54</b> to <b>(</b><i><b>S</b></i><b>)-55</b>. (<i>rac</i>)-1-((2-Bromo-5-nitrophenyl)sulfonyl)-3-cyclopropylpyrrolidine (<b>(</b><i><b>rac</b></i><b>)-85</b>)</h3><div class="NLM_p last">2-Bromo-5-nitrobenzenesulfonyl chloride <b>56</b> (1.30 g, 4.33 mmol) was taken up in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) under nitrogen and cooled in an ice bath. DIPEA (1.66 mL, 9.52 mmol) was added and the reaction stirred for 5 min before (<i>rac</i>)-3-cyclopropylpyrrolidine (0.48 g, 4.33 mmol) was added. The reaction was stirred for 2 h at 0 °C. The reaction mixture was quenched with sat. NaHCO<sub>3</sub> (50 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 50 mL). The combined organics were filtered through a hydrophobic frit and concentrated <i>in</i><i>vacuo</i> to give the crude product. This was purified by silica chromatography, eluting with 0–30% cyclohexane/EtOAc. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-1-((2-bromo-5-nitrophenyl)sulfonyl)-3-cyclopropylpyrrolidine <b>(</b><i><b>rac</b></i><b>)-85</b> (1.38 g, 3.68 mmol, 85% yield) as a yellow gum. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.24 min, <i>m</i>/<i>z</i> = 375.2, 377.2. HRMS (C<sub>13</sub>H<sub>15</sub>BrN<sub>2</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 375.0014, found 375.0013. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.91 (d, <i>J</i> = 2.8 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 2.8 Hz, 1H), 7.96 (d, <i>J</i> = 8.6 Hz, 1H), 3.58–3.68 (m, 2H), 3.41–3.52 (m, 1H), 3.21 (dd, <i>J</i> = 9.5, 7.6 Hz, 1H), 2.04–2.16 (m, 1H), 1.81 (dq, <i>J</i> = 12.3, 8.4 Hz, 1H), 1.55–1.71 (m, 1H), 0.62–0.77 (m, 1H), 0.43–0.55 (m, 2H), 0.08–0.23 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 146.8, 141.2, 136.9, 127.7, 127.2, 126.6, 53.0, 47.9, 44.4, 31.5, 13.0, 3.7, 3.4. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3103, 1597, 1526, 1341, 1161, 880, 606.</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> (<i>rac</i>)-4-Bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)aniline (<b>(</b><i><b>rac</b></i><b>)-86</b>)</h3><div class="NLM_p last">(<i>rac</i>)-1-((2-Bromo-5-nitrophenyl)sulfonyl)-3-cyclopropylpyrrolidine <b>(</b><i><b>rac</b></i><b>)-85</b> (1.30 g, 3.46 mmol), ammonium chloride (0.28 g, 5.20 mmol), and iron (0.58 g, 10.39 mmol) were dissolved in EtOH (5.0 mL) and water (1.7 mL). The resulting solution was heated to 70 °C for 2 h. The reaction was allowed to cool and then filtered through a plug of Celite, washing with MeOH (20 mL). The resulting solution was concentrated <i>in vacuo</i> and then partitioned between sat. aq NaHCO<sub>3</sub> (20 mL) and CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The layers were separated, and the aq layer was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–100% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-4-bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>rac</b></i><b>)-86</b> (1.19 g, 3.45 mmol, 99% yield) as an orange gum. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.10 min, <i>m</i>/<i>z</i> = 345.2, 347.2. HRMS (C<sub>13</sub>H<sub>17</sub>BrN<sub>2</sub>O<sub>2</sub>S): [M + H]<sup>+</sup> calculated 345.0272, found 345.0270. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 7.40–7.48 (m, 2H), 6.67 (dd, <i>J</i> = 8.4, 2.8 Hz, 1H), 3.94 (br s, 2H), 3.51–3.63 (m, 2H), 3.39 (ddd, <i>J</i> = 9.5, 8.2, 7.2 Hz, 1H), 3.15 (dd, <i>J</i> = 9.4, 7.7 Hz, 1H), 1.97–2.11 (m, 1H), 1.69–1.82 (m, 1H), 1.54–1.66 (m, 1H), 0.62–0.75 (m, 1H), 0.39–0.54 (m, 2H), 0.07–0.20 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 146.0, 138.8, 136.0, 119.4, 118.4, 106.9, 52.5, 47.4, 44.3, 31.5, 13.2, 3.6, 3.3. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3374, 1591, 1462, 1312, 1156, 544.</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i104" class="anchor-spacer"></div><h3 class="article-section__title" id="_i104"> (<i>rac</i>)-2-(1<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide (<b>(</b><i><b>rac</b></i><b>)-87</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl 2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetate <b>69</b> (163 mg, 0.695 mmol) was dissolved in HCl (4 M in dioxane, 1.45 mL, 5.79 mmol) and stirred at 40 °C for 16 h. The reaction mixture was concentrated <i>in vacuo</i> to afford the crude acid. (<i>rac</i>)-4-Bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>rac</b></i><b>)-86</b> (200 mg, 0.579 mmol), and HATU (264 mg, 0.695 mmol) were added, and the mixture was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (10 mL). DIPEA (0.304 mL, 1.738 mmol) was added, and the resulting solution was stirred at rt for 2 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–50% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>rac</b></i><b>)-87</b> (222 mg, 0.439 mmol, 76% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.97 min, <i>m</i>/<i>z</i> = 505.2, 507.2. HRMS (C<sub>20</sub>H<sub>21</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 505.0657, found 505.0658. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.54 (d, <i>J</i> = 0.7 Hz, 1H), 9.06 (s, 1H), 8.67 (d, <i>J</i> = 4.5 Hz, 1H), 8.12 (d, <i>J</i> = 2.7 Hz, 1H), 7.99 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 7.59–7.71 (m, 2H), 5.64 (s, 2H), 3.55–3.65 (m, 2H), 3.38–3.47 (m, 1H), 3.17 (dd, <i>J</i> = 9.4, 7.7 Hz, 1H), 2.04–2.13 (m, 1H), 1.80 (dq, <i>J</i> = 12.3, 8.4 Hz, 1H), 1.57–1.69 (m, 1H), 0.61–0.73 (m, 1H), 0.40–0.53 (m, 2H), 0.07–0.19 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.5, 145.3, 144.7, 138.5, 137.4, 137.0, 136.5, 125.1, 125.0, 123.2, 114.7, 104.9, 52.9, 51.4, 47.9, 44.4, 31.4, 13.0, 3.7, 3.4. Mp 74.3–77.8 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3207, 1701, 1587, 1387, 1158, 833, 556.</div></div><div id="sec4_9_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> (<i>rac</i>)-<i>N</i>-(4-Bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>rac</b></i><b>)-54</b>)</h3><div class="NLM_p last">Iodomethane (42 mg, 0.297 mmol) was added to (<i>rac</i>)-2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>rac</b></i><b>)-87</b> (150 mg, 0.297 mmol) in MeCN (2 mL) at rt under air. The resulting solution was refluxed at 80 °C for 2 h. The reaction was allowed to cool and concentrated <i>in vacuo</i> to afford the methylpyridinium intermediate. The residue was dissolved in MeOH (1 mL) and water (1 mL), before NaBH<sub>4</sub> (25 mg, 0.653 mmol) was added, and the resulting solution was stirred at rt for 30 min. The reaction was quenched with water (5 mL) and extracted with EtOAc (3 × 5 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by HPH MDAP. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-<i>N</i>-(4-bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-54</b> (67 mg, 0.128 mmol, 43% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.69 min, <i>m</i>/<i>z</i> = 523.2, 525.2. HRMS (C<sub>21</sub>H<sub>27</sub>BrN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 523.1127, found 523.1124. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.23 (s, 1H), 8.18 (d, <i>J</i> = 2.7 Hz, 1H), 7.83 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.66 (d, <i>J</i> = 8.8 Hz, 1H), 5.18 (s, 2H), 3.68 (s, 2H), 3.53–3.62 (m, 2H), 3.35–3.43 (m, 1H), 3.15 (dd, <i>J</i> = 9.5, 7.6 Hz, 1H), 2.78–2.87 (m, 4H), 2.54 (s, 3H), 2.03–2.09 (m, 1H), 1.76 (dq, <i>J</i> = 12.3, 8.4 Hz, 1H), 1.54–1.66 (m, 1H), 0.61–0.72 (m, 1H), 0.40–0.52 (m, 2H), 0.08–0.17 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.7, 143.0, 138.9, 137.0, 136.2, 132.4, 124.7, 123.3, 114.7, 52.8, 51.4, 51.4, 51.4, 47.7, 45.2, 44.3, 31.4, 20.6, 13.1, 3.7, 3.4.</div></div><div id="sec4_9_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> (<i>R</i>)-<i>N</i>-(4-Bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>R</b></i><b>)-54</b>) and (<i>S</i>)-<i>N</i>-(4-Bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>S</b></i><b>)-54</b>)</h3><div class="NLM_p last">(<i>rac</i>)-<i>N</i>-(4-bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phen yl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-54</b> was dissolved in EtOH (5 mL) and injected onto the column (column, 250 mm × 20 mm Chiralpak IG, 5 μm), eluting with 95% (EtOH + 0.2% isopropylamine)/CH<sub>2</sub>Cl<sub>2</sub>, flow rate = 15 mL min<sup>–1</sup>, detection wavelength 215 nm. The pure fractions from peak 1 were concentrated <i>in vacuo</i> to afford (<i>R</i>)-<i>N</i>-(4-bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazol o[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>R</b></i><b>)-54</b> (20 mg, 0.038 mmol) as a white solid, and the pure fractions from peak 1 were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-bromo-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]tria zolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-54</b> (24 mg, 0.046 mmol) as white solids. Data consistent with racemate (<b>(</b><i><b>rac</b></i><b>)-54</b>). Chiral LC: 4.6 mm × 250 mm Chiralpak IG column, 80% (EtOH + 2% isopropylamine)/heptane. <b>(</b><i><b>R</b></i><b>)-54</b>: <i>t</i><sub>R</sub> = 9.785 min; er > 99:1. <b>(</b><i><b>S</b></i><b>)-54</b>: <i>t</i><sub>R</sub> = 12.045 min; er > 99:1.</div></div><div id="sec4_9_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> (<i>rac</i>)-1-((2-Chloro-5-nitrophenyl)sulfonyl)-3-cyclopropylpyrrolidine (<b>(</b><i><b>rac</b></i><b>)-71</b>)</h3><div class="NLM_p last">2-Chloro-5-nitrobenzenesulfonyl chloride <b>70</b> (1.10 g, 4.30 mmol) was taken up in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) under nitrogen and cooled to 0 °C. DIPEA (1.65 mL, 9.45 mmol) was added and the reaction stirred for 5 min before (<i>rac</i>)-3-cyclopropylpyrrolidine (0.48 g, 4.30 mmol) was added. The reaction was stirred at 0 °C for 2 h. The reaction mixture was quenched with sat. NaHCO<sub>3</sub> (50 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 50 mL). The combined organics were filtered through a hydrophobic frit and concentrated <i>in vacuo</i> to give the crude product. The crude product was purified by silica chromatography, eluting with 0–40% cyclohexane/EtOAc. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-1-((2-chloro-5-nitrophenyl)sulfonyl)-3-cyclopropylpyrrolidine <b>(</b><i><b>rac</b></i><b>)-71</b> (1.28 g, 3.87 mmol, 90% yield) as a yellow gum. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.23 min, <i>m</i>/<i>z</i> = 331.2. HRMS (C<sub>13</sub>H<sub>15</sub>ClN<sub>2</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calculated 331.0519, found 331.0510. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.92 (d, <i>J</i> = 2.7 Hz, 1H), 8.33 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.71 (d, <i>J</i> = 8.8 Hz, 1H), 3.58–3.69 (m, 2H), 3.45 (ddd, <i>J</i> = 9.7, 8.3, 7.2 Hz, 1H), 3.21 (dd, <i>J</i> = 9.5, 7.8 Hz, 1H), 2.03–2.15 (m, 1H), 1.80 (dq, <i>J</i> = 12.5, 8.3 Hz, 1H), 1.60–1.69 (m, 1H), 0.63–0.76 (m, 1H), 0.43–0.55 (m, 2H), 0.08–0.22 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 146.2, 139.4, 139.0, 133.2, 127.4, 126.7, 52.9, 47.8, 44.4, 31.5, 13.0, 3.7, 3.4. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3098, 1601, 1524, 1344, 1162, 884.</div></div><div id="sec4_9_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> (<i>rac</i>)-4-Chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)aniline (<b>(</b><i><b>rac</b></i><b>)-72</b>)</h3><div class="NLM_p last">(<i>rac</i>)-1-((2-Chloro-5-nitrophenyl)sulfonyl)-3-cyclopropylpyrrolidine <b>(</b><i><b>rac</b></i><b>)-71</b> (1.25 g, 3.78 mmol), ammonium chloride (0.30 g, 5.67 mmol), and iron (0.63 g, 11.34 mmol) were dissolved in EtOH (5.0 mL) and water (1.7 mL). The resulting solution was heated to 70 °C for 2 h. The reaction was allowed to cool then filtered through a plug of Celite, washing with MeOH (20 mL). The resulting solution was concentrated <i>in vacuo</i> and then partitioned between sat. aq NaHCO<sub>3</sub> (20 mL) and CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The layers were separated, and the aq layer was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford (<i>rac</i>)-4-chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>rac</b></i><b>)-72</b> (1.11 g, 3.69 mmol, 98% yield) as a yellow gum. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 1.07 min, <i>m</i>/<i>z</i> = 301.1. HRMS (C<sub>13</sub>H<sub>17</sub>ClN<sub>2</sub>O<sub>2</sub>S): [M + H]<sup>+</sup> calculated 301.0778, found 301.0771. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 7.41 (d, <i>J</i> = 2.7 Hz, 1H), 7.26 (d, <i>J</i> = 8.5 Hz, 1H), 6.75 (dd, <i>J</i> = 8.5, 2.7 Hz, 1H), 3.91 (br s, 2H), 3.53–3.63 (m, 2H), 3.39 (ddd, <i>J</i> = 9.5, 8.3, 7.1 Hz, 1H), 3.15 (dd, <i>J</i> = 9.5, 7.7 Hz, 1H), 1.98–2.09 (m, 1H), 1.74 (dq, <i>J</i> = 12.3, 8.4 Hz, 1H), 1.56–1.61 (m, 1H), 0.60–0.74 (m, 1H), 0.41–0.52 (m, 2H), 0.08–0.18 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 145.4, 137.1, 132.5, 120.1, 119.2, 118.0, 52.5, 47.4, 44.3, 31.5, 13.1, 3.6, 3.3. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3375, 1596, 1468, 1323, 1159, 590.</div></div><div id="sec4_9_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> (<i>rac</i>)-2-(1<i>H</i>-[1,2,3]Triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>rac</b></i><b>)-73</b></h3><div class="NLM_p last"><i>tert-</i>Butyl 2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetate <b>(</b><i><b>rac</b></i><b>)-69</b> (187 mg, 0.798 mmol) was dissolved in HCl (4 M in dioxane, 1.66 mL, 6.65 mmol) and stirred at rt for 16 h. The reaction mixture was concentrated <i>in vacuo</i> to afford the crude acid. (<i>rac</i>)-4-Chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)aniline <b>(</b><i><b>rac</b></i><b>)-72</b> (200 mg, 0.665 mmol) and HATU (303 mg, 0.798 mmol) were added, and the mixture was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (10 mL). DIPEA (0.348 mL, 1.995 mmol) was added, and the resulting solution was stirred at rt for 2 h. The reaction was quenched with sat. aq NaHCO<sub>3</sub> (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford the crude product. The crude product was purified by silica chromatography, eluting with 0–100% EtOAc/cyclohexane. The pure fractions were concentrated <i>in vacuo</i> to afford (<i>rac</i>)-2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>rac</b></i><b>)-73</b> (241 mg, 0.523 mmol, 79% yield) as a cream solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.98 min, <i>m</i>/<i>z</i> = 461.3. HRMS (C<sub>20</sub>H<sub>21</sub>ClN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 461.1163, found 461.1161. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 9.45 (d, <i>J</i> = 1.1 Hz, 1H), 9.16 (s, 1H), 8.58 (d, <i>J</i> = 6.0 Hz, 1H), 8.25 (d, <i>J</i> = 2.7 Hz, 1H), 7.82 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.65 (dd, <i>J</i> = 6.0, 1.1 Hz, 1H), 7.43 (d, <i>J</i> = 8.8 Hz, 1H), 5.67 (s, 2H), 3.51–3.58 (m, 2H), 3.30–3.39 (m, 1H), 3.12 (dd, <i>J</i> = 9.7, 7.7 Hz, 1H), 1.97–2.06 (m, 1H), 1.73 (dd, <i>J</i> = 12.5, 8.6 Hz, 1H), 1.51–1.63 (m, 1H), 0.55–0.67 (m, 1H), 0.37–0.48 (m, 2H), 0.05–0.12 (m, 2H). Mp 63.7–68.8 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 3008, 1590, 1465, 1156, 839, 556.</div></div><div id="sec4_9_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> (<i>rac</i>)-<i>N</i>-(4-Chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>rac</b></i><b>)-55</b>)</h3><div class="NLM_p last">Iodomethane (0.16 mL, 2.169 mmol) was added to (<i>rac</i>)-2-(1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)-<i>N</i>-(4-chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)acetamide <b>(</b><i><b>rac</b></i><b>)-73</b> (1.00 g, 2.169 mmol) in MeCN (20 mL) at rt under air. The resulting solution was refluxed at 80 °C for 2 h. The reaction was allowed to cool and concentrated <i>in vacuo</i> to afford the methylpyridinium intermediate. The residue was dissolved in MeOH (10 mL) and water (10 mL) before NaBH<sub>4</sub> (0.18 g, 4.77 mmol) was added and the resulting solution stirred at rt for 30 min. The reaction was quenched with water (20 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated <i>in vacuo</i> to afford (<i>rac</i>)-<i>N</i>-(4-chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazol o[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-55</b> (762 mg, 1.591 mmol, 73% yield) as a white solid. LCMS (formic, ES<sup>+</sup>) <i>t</i><sub>R</sub> = 0.68 min, <i>m</i>/<i>z</i> = 479.3. HRMS (C<sub>21</sub>H<sub>27</sub>ClN<sub>6</sub>O<sub>3</sub>S): [M + H]<sup>+</sup> calculated 479.1632, found 479.1629. <sup>1</sup>H NMR (CDCl<sub>3</sub>-<i>d</i>, 400 MHz): δ (ppm) 8.98 (s, 1H), 8.13 (d, <i>J</i> = 2.7 Hz, 1H), 7.91 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.46 (d, <i>J</i> = 8.8 Hz, 1H), 5.16 (s, 2H), 3.70 (s, 2H), 3.53–3.62 (m, 2H), 3.39 (ddd, <i>J</i> = 9.5, 8.3, 7.2 Hz, 1H), 3.15 (dd, <i>J</i> = 9.5, 7.8 Hz, 1H), 2.80–2.89 (m, 4H), 2.56 (s, 3H), 1.99–2.10 (m, 1H), 1.76 (dq, <i>J</i> = 12.2, 8.4 Hz, 1H), 1.56–1.66 (m, 1H), 0.60–0.71 (m, 1H), 0.41–0.52 (m, 2H), 0.09–0.18 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>-<i>d</i>, 101 MHz): δ (ppm) 163.6, 143.0, 137.3, 136.1, 132.6, 132.2, 127.3, 124.7, 123.1, 77.2, 52.7, 51.4, 51.4, 47.6, 45.1, 44.3, 31.4, 20.6, 13.0, 3.7, 3.3. Mp 91.8–97.8 °C. IR ν<sub>max</sub> (cm<sup>–1</sup>) 2950, 1705, 1535, 1464, 1157, 591.</div></div><div id="sec4_9_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> (<i>R</i>)-<i>N</i>-(4-Chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide (<b>(</b><i><b>R</b></i><b>)-55</b>) and (<i>S</i>)-<i>N</i>-(4-Chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-55</b>)</h3><div class="NLM_p last">(<i>rac</i>)-<i>N</i>-(4-Chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5 -methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>rac</b></i><b>)-55</b> was dissolved in EtOH (1 mL) and injected onto the column (column, 250 mm × 20 mm Chiralpak IG, 5 μm), eluting with 50% (EtOH + 0.2% isopropylamine)/(heptane + 0.2% isopropylamine), flow rate = 15 mL min<sup>–1</sup>, detection wavelength 215 nm. The pure fractions from peak 1 were concentrated <i>in vacuo</i> to afford (<i>R</i>)-<i>N</i>-(4-chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazol o[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>R</b></i><b>)-55</b> (16 mg, 0.033 mmol) as a white solid, and the pure fractions from peak 2 were concentrated <i>in vacuo</i> to afford (<i>S</i>)-<i>N</i>-(4-chloro-3-((3-cyclopropylpyrrolidin-1-yl)sulfonyl)phenyl)-2-(5-methyl-4,5,6,7-tetrahydro-1<i>H</i>-[1,2,3]triazolo[4,5-<i>c</i>]pyridin-1-yl)acetamide <b>(</b><i><b>S</b></i><b>)-55</b> (15 mg, 0.031 mmol) as a white solid. Data consistent with racemate (<b>(</b><i><b>rac</b></i><b>)-55</b>). Chiral LC: 4.6 mm × 250 mm Chiralpak IG column, 80% (EtOH + 2% isopropylamine)/heptane. <b>(</b><i><b>R</b></i><b>)-55</b>: <i>t</i><sub>R</sub> = 17.354 min; er > 99:1. <b>(</b><i><b>S</b></i><b>)-55</b>: <i>t</i><sub>R</sub> = 23.739 min; er > 99:1.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> ATAD2 Synthetic Ligand Competition TR-FRET Binding Assay</h3><div class="NLM_p last">Compounds were titrated from 10 mM in 100% DMSO, and 100 nL was transferred to a low-volume black 384-well microtiter plate using a Labcyte Echo 555. A Thermo Scientific Multidrop Micro was used to dispense 5 μL of 5 nM FLAG-6His-Tev-ATAD2(981–1121) in 50 mM Hepes, 150 mM NaCl, 5% glycerol, 1 mM CHAPS, and 1 mM DTT, pH 7.4, in the presence of 100 nM Alexa Fluor 488-labeled ligand. After equilibrating for 30 min in the dark at rt, the ATAD2 protein–fluorescent ligand interaction was detected using TR-FRET following a 5 μL addition of 1.5 nM Lanthascreen Elite Tb-anti His antibody (Invitrogen, PV5863) in assay buffer. Time-resolved fluorescence (TRF) was then detected on a TRF laser equipped PerkinElmer Envision multimode plate reader (excitation = 337 nm; emission 1 = 520 nm; emission 2 = 495 nm; dual wavelength bias dichroic = 400, 505 nm). TR-FRET ratio was calculated using the following equation: ratio = ((acceptor fluorescence at 520 nm)/(donor fluorescence at 495 nm)) × 1000. TR FRET ratio data were normalized to a mean of 16 replicates per microtiter plate of both 10 μM proprietary ATAD2 inhibitor and 100% DMSO controls. IC<sub>50</sub> values were determined for each of the compounds tested by fitting the fluorescence ratio data to a four parameter curve fit of the following form:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_m001.gif" alt="" id="_i113" /></img></span>where “<i>a</i>” is the minimum, “<i>b</i>” is the Hill slope, “<i>c</i>” is the pIC<sub>50</sub> ,and “<i>d</i>” is the maximum.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> CECR2 Synthetic Ligand Competition TR-FRET Binding Assay<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></h3><div class="NLM_p last">Compounds were titrated from 10 mM in 100% DMSO, and 100 nL was transferred to a low-volume black 384-well microtiter plate using a Labcyte Echo 555. A Thermo Scientific Multidrop Micro was used to dispense 5 μL of 6H-FLAG-Tev-CECR2(424–543) in 50 mM Hepes, 150 mM NaCl, 5% glycerol, 1 mM CHAPS, and 1 mM DTT, pH 7.4, in the presence of 200 nM Alexa Fluor 647-labeled GSK tracer ligand. After equilibrating for 15 min at rt, the CECR2 protein–fluorescent ligand interaction was detected using TR-FRET following a 5 μL addition of 3 nM Eu-W1024-labeled anti-6xHis antibody in assay buffer. Time-resolved fluorescence (TRF) was then detected on a TRF laser equipped PerkinElmer Envision multimode plate reader (excitation = 315 nm; emission 1 = 665 nm; emission 2 = 615 nm). TR-FRET ratio was calculated using the following equation: ratio = ((acceptor fluorescence at 665 nm)/(donor fluorescence at 615 nm)) × 1000. TR FRET ratio data were normalized to a mean of 16 replicates per microtiter plate of both no protein and 100% DMSO controls. IC<sub>50</sub> values were determined for each of the compounds tested by fitting the fluorescence ratio data to a four parameter curve fit of the following form.<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_m002.gif" alt="" id="_i115" /></img></span>where “<i>a</i>” is the minimum, “<i>b</i>” is the Hill slope, “<i>c</i>” is the pIC<sub>50</sub>, and “<i>d</i>” is the maximum.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> BRD4 Mutant TR-FRET Assay<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></h3><div class="NLM_p last">The binding of the compounds to BRD4 was assessed using a mutated protein to detect differential binding to binding domain 1 (BD1) on the bromodomain containing protein. The single residue mutation in the binding domain 2 (BD2) acetyl lysine binding pocket (Y390A) greatly lowers the affinity of the fluoroligand for the mutated BD2 domain (>1000-fold selective for the nonmutated domain). Therefore, in the final assay conditions, binding of the fluoroligand to the mutated BD2 domain cannot be detected and subsequently the assay is suitable to determine the binding of compounds to the single nonmutated BD1 bromodomain.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i117" class="anchor-spacer"></div><h3 class="article-section__title" id="_i117"> Protocol for BRD4 BD1 Ligand FRET Assay</h3><div class="NLM_p last">All assay components were dissolved in buffer composition of 50 mM HEPES, pH 7.4, 50 mM NaCl, 5% glycerol, 1 mM DTT, and 1 mM CHAPS. The final concentration of BRD4 protein (1–477, Y390A) was at 10 nM and the Alexa Fluor 647 ligand was at <i>K</i><sub>d</sub> (∼50 nM for BRD4). These components were premixed, and 5 μL of this reaction mixture was added to all wells containing 50 nL of various concentrations of test compound or DMSO vehicle (0.5% DMSO final) in Greiner 384 well black low volume microtiter plates and incubated in dark for 30 min at rt. Detection reagents were prepared in assay buffer by diluting Eu-W1024 anti-6xHis antibody (AD0111 PerkinElmer) to 1.5 nM FAC. 5 μL of this solution was then added to all wells. The plates were read on the Envision reader, and the donor and acceptor counts were determined. From this, the ratio of acceptor/donor was calculated (λ<sub>ex</sub> = 337 nm, λ<sub>em</sub> donor = 615 nm, λ<sub>em</sub> acceptor = 665 nm) and used for data analysis. All data were normalized to the robust mean of 16 high and 16 low control wells on each plate. A four parameter curve fit of the following form was then applied.<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_m003.gif" alt="" id="_i118" /></img></span>where “<i>a</i>” is the minimum, “<i>b</i>” is the Hill slope, “<i>c</i>” is the pIC<sub>50</sub>, and “<i>d</i>” is the maximum.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> CECR2 NanoBRET Tracer Assay<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></h3><div class="NLM_p">Compounds were titrated 1:4 from 10 mM stock solutions in 100% DMSO, with 40 nL transferred per well to a white 384-well TC treated microplate (Nunc 164610, Thermo Fisher Scientific, USA) using a Labcyte Echo 555 (Labcyte Inc., USA). All other liquid handling was performed using a Thermo Scientific Multidrop (Thermo Fisher Scientific, USA) unless otherwise stated. HEK293-wt cells were seeded into a Corning 6-well microplate at 8 × 10<sup>5</sup> cells per well and left to attach for 4–6 h at 37 °C, 5% CO<sub>2</sub> prior to transfection. The cells were then transfected with the following transfection mix (per well): 92 μL of OptimMEM, 0.2 μg of CECR2-NanoLuc (obtained from Promega), 8 μL of FuGENE HD.</div><div class="NLM_p last">Following an overnight incubation at 37 °C, 5% CO<sub>2</sub>, cells were washed with PBS and then detached by trypsinization using TrypLE Express. Cells were then centrifuged at 1500 rpm for 5 min, the supernatant was discarded, and the cell pellet was resuspended in OptiMEM + 4% US FBS at 2 × 10<sup>5</sup> cells/mL. BSP-BODIPY was then added to the cell solution at a final assay concentration (FAC) of 150 nM. 40 nL of the cell suspension was dispensed per well into the prestamped compound microplates and then incubated for 4–6 h at 37 °C, 5% CO<sub>2</sub>. This time point was chosen to give the optimal signal:background, data not shown. NanoLuc substrate solution was prepared by performing 1:100 dilution in OptiMEM. 10 uL of this solution was added per well, and plates were centrifuged at 1000 rpm for 1 min and shaken at 1000 rpm for 1 min followed immediately by data capture. All data were normalized using <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>, to 16 high (DMSO) and 16 low (with literature compound 40<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a>) control wells on each microplate. pIC<sub>50</sub> determination was performed by plotting the ratioed (3), normalized data against the BET inhibitor concentration and applying a four-parameter logistic fit as described in <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i120" class="anchor-spacer"></div><h3 class="article-section__title" id="_i120"> Data Analysis</h3><div class="NLM_p">All high throughput data analysis was carried out in IDBS ActivityBase XE.</div><div class="NLM_p last">NanoBRET tracer data binding were fitted using the following four parameter logistic fit, <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>,<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_m004.gif" alt="" id="_i121" /></img><span class="labelSpan">(1)</span></span>where “<i>a</i>” is the minimum, “<i>b</i>” is the Hill slope, “<i>c</i>” is the pIC<sub>50</sub>, and “<i>d</i>” is the maximum. Assay data were normalized to microplate controls using the following equation, <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>,<span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_m005.gif" alt="" id="_i122" /></img><span class="labelSpan">(2)</span></span>where “<i>a</i>” is the raw compound data, “<i>b</i>” is the low control raw data, and “<i>c</i>” is the high control raw data. Data generated using TR-FRET assays were normalized to the lanthanide donor fluorescence using the following equation, <a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>,<span class="NLM_disp-formula" id="eq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_m006.gif" alt="" id="_i123" /></img><span class="labelSpan">(3)</span></span>where “<i>x</i>” is milliBRET units (mBu), “<i>a</i>” is the acceptor fluorescence at 610 nm, and “<i>d</i>” is the donor fluorescence at 480 nm.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i124" class="anchor-spacer"></div><h3 class="article-section__title" id="_i124"> Physicochemical Properties</h3><div class="NLM_p last">Permeability across a lipid membrane, chromatographic log <i>D</i> at pH 7.4, and solubility by precipitation into saline were measured using published protocols.<a onclick="showRef(event, 'ref27 ref28 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref59 ref60">(27,28,59,60)</a></div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i125" class="anchor-spacer"></div><h3 class="article-section__title" id="_i125"> <i>In Silico</i> CECR2 Docking</h3><div class="NLM_p last">The crystal structure of CECR2 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXB">3NXB</a>) was used as the starting point for modeling in Maestro (  <cite><i>Schrödinger
Release 2018-3: Maestro</i></cite>; Schrödinger
LLC: New York, <span class="NLM_year">2018</span>). The protein preparation tool was used to remove all solvent molecules, add missing side chains, add hydrogens (PROPKA pH 7.0), and minimize the site under the OPLS3 force field with heavy atoms restrained to 0.3 Å RMSD. The structure of ATAD2 bound to <b>(</b><i><b>rac</b></i><b>)-8</b> (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55">6S55</a>) was superimposed and the 1-bromo-2-hydrosulfonylbenzene fragment of the ligand extracted and merged into the CECR2 structure, where it was used as a template constraint during Glide docking.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i126"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00021" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73100" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73100" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00021?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00021</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">BRD2,3 and T TR-FRET data and DiscoverX BROMO<i>scan</i> bromodomain profiling of key compounds, a comparison of CLND and CAD solubility measurements, correlation between TR-FRET and DiscoverX BROMO<i>scan</i> TAF1(2) data, X-ray crystallographic data, chemistry experimental procedures, and selected NMR and LCMS spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Model of <b>(</b><i><b>S,S</b></i><b>)-32</b> docked into CECR2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_001.pdf">jm0c00021_si_001.pdf (753.09 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_002.csv">jm0c00021_si_002.csv (9.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_003.pdb">jm0c00021_si_003.pdb (56.6 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The crystal structure of ATAD2 bound <b>(</b><i><b>R,S</b></i><b>)-32</b> has been deposited in the Protein Data Bank as <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YB4">6YB4</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00021" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34432" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34432" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen J. Atkinson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3636-3674" title="Orcid link">http://orcid.org/0000-0003-3636-3674</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9be8effeebf3fef5b5f1b5faeff0f2f5e8f4f5dbfce8f0b5f8f4f6"><span class="__cf_email__" data-cfemail="5a292e3f2a323f347430743b2e3133342935341a3d293174393537">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon C. C. Lucas</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, 
    , , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">WestCHEM,
Department of Pure and Applied Chemistry, University of Strathclyde, Thomas Graham Building, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</span>; 
    <span>Present Address:
                        S.C.C.L.: Discovery Sciences, AstraZeneca, Cambridge, UK</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Bamborough</span> - <span class="hlFld-Affiliation affiliation">Computational & Structural
Chemistry, Molecular Discovery Research, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heather Barnett</span> - <span class="hlFld-Affiliation affiliation">Flexible Discovery
Unit, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">Computational & Structural
Chemistry, Molecular Discovery Research, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2480-3110" title="Orcid link">http://orcid.org/0000-0002-2480-3110</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurie Gordon</span> - <span class="hlFld-Affiliation affiliation">Biological Sciences, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren J. Mitchell</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander Phillipou</span> - <span class="hlFld-Affiliation affiliation">Biological Sciences, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Sheppard</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span>; 
    <span>Present Address:
                        R.J.S.: Medicinal Chemistry, Research and Early Development, Cardiovascular, Renal, and Metabolism, BioPharmaceuticals R&D, AstraZeneca, Gothenburg 431 50, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas C. O. Tomkinson</span> - <span class="hlFld-Affiliation affiliation">WestCHEM,
Department of Pure and Applied Chemistry, University of Strathclyde, Thomas Graham Building, 295 Cathedral Street, Glasgow G1 1XL, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5509-0133" title="Orcid link">http://orcid.org/0000-0002-5509-0133</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Watson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emmanuel H. Demont</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , ,
and GlaxoSmithKline Medicines Research Centre, Gunnels Wood Road, Stevenage SG1 2NY, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7307-3129" title="Orcid link">http://orcid.org/0000-0001-7307-3129</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors except S.C.C.L. and N.C.O.T. were GlaxoSmithKline full-time employees when this study was performed.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e13133-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i131">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56346" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56346" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">S.C.C.L. is grateful to GlaxoSmithKline R&D and the EPSRC via Prosperity Partnership EP/S035990/1 for Ph.D. studentship funding. Chiral analysis and separation were carried out by Eric Hortense, Richard Briers, and Steve Jackson. We also thank Sean Lynn and Stephen Richards for assistance with NMR spectra and Emily Lowndes for crystallization support.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AMP</td><td class="NLM_def"><p class="first last">artificial membrane permeability</p></td></tr><tr><td class="NLM_term">ATAD2</td><td class="NLM_def"><p class="first last">ATPase family AAA domain containing 2</p></td></tr><tr><td class="NLM_term">BD1</td><td class="NLM_def"><p class="first last">first (N-terminal) bromodomain</p></td></tr><tr><td class="NLM_term">BD2</td><td class="NLM_def"><p class="first last">second (C-terminal) bromodomain</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal domain</p></td></tr><tr><td class="NLM_term">BRD2/3/4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 2/3/4</p></td></tr><tr><td class="NLM_term">BTPF</td><td class="NLM_def"><p class="first last">bromodomain PHD finger transcription factor</p></td></tr><tr><td class="NLM_term">CAD</td><td class="NLM_def"><p class="first last">charged aerosol detection</p></td></tr><tr><td class="NLM_term">CECR2</td><td class="NLM_def"><p class="first last">cat eye chromosome region, candidate 2</p></td></tr><tr><td class="NLM_term">ChromLogD</td><td class="NLM_def"><p class="first last">chromatographic log <i>D</i></p></td></tr><tr><td class="NLM_term">CLND</td><td class="NLM_def"><p class="first last">charged luminescence nitrogen detection</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectra</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE<sub>at</sub></td><td class="NLM_def"><p class="first last">lipophilic ligand efficiency, Astex</p></td></tr><tr><td class="NLM_term">MDAP</td><td class="NLM_def"><p class="first last">mass directed automatic purification</p></td></tr><tr><td class="NLM_term">RVF</td><td class="NLM_def"><p class="first last">arginine–valine–phenylalanine</p></td></tr><tr><td class="NLM_term">Sel</td><td class="NLM_def"><p class="first last">selectivity</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved-fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">WPF</td><td class="NLM_def"><p class="first last">tryptophan–proline–phenylalanine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i133">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70717" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70717" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, M. C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Histone Readers and Cancer</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">2003</span>– <span class="NLM_lpage">2010</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2016.11.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.jmb.2016.11.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=27890782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOisbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2003-2010&issue=13&author=A.+K.+Jainauthor=M.+C.+Barton&title=Bromodomain+Histone+Readers+and+Cancer&doi=10.1016%2Fj.jmb.2016.11.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain histone readers and cancer</span></div><div class="casAuthors">Jain, Abhinav K.; Barton, Michelle C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2003-2010</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation of histone proteins is a fundamental post-translational modification that regulates chromatin structure and plays an important role in gene transcription.  Aberrant levels of histone lysine acetylation are assocd. with the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are structurally conserved modules present in transcription-assocd. proteins that are termed "reader" proteins.  Bromodomain-contg. reader proteins are part of multiprotein complexes that regulate transcription programs, which are often assocd. with profound phenotypic changes.  Many bromodomain-contg. proteins are aberrantly expressed in diseases, as best studied in cancers, where bromodomain proteins impact the expression of oncogenes and anti-apoptotic proteins.  Thus, bromodomain readers of histone acetylation have emerged as attractive targets for cancer drug discovery, prompting immense interest in epigenetic-focused, medicinal chem. to develop structurally guided chem. probes of bromodomains.  Here, we describe bromodomain-contg. proteins with defined roles in cancer and highlight recent progress in the development of bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPuEwpy7dAA7Vg90H21EOLACvtfcHk0liABLNwVPratA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOisbbI&md5=2d1d3efa959f66eeb8789b48daf96325</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.11.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.11.020%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DM.%2BC.%26atitle%3DBromodomain%2520Histone%2520Readers%2520and%2520Cancer%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26issue%3D13%26spage%3D2003%26epage%3D2010%26doi%3D10.1016%2Fj.jmb.2016.11.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting Bromodomains: Epigenetic Readers of Lysine Acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+Bromodomains%3A+Epigenetic+Readers+of+Lysine+Acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liABLNwVPratA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520Bromodomains%253A%2520Epigenetic%2520Readers%2520of%2520Lysine%2520Acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=598-608&issue=3&author=S.+G.+Smithauthor=M.-M.+Zhou&title=The+Bromodomain%3A+A+New+Target+in+Emerging+Epigenetic+Medicine&doi=10.1021%2Facschembio.5b00831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span></div><div class="casAuthors">Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-608</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved protein modular domain found in many chromatin- and transcription-assocd. proteins that has the ability to recognize acetylated lysine residues.  This activity allows bromodomains to play a vital role in many acetylation-mediated protein-protein interactions in the cell, ranging from substrate recruitment for histone acetyltransferases (HATs) to aiding in multiple-protein complex assembly for gene transcriptional activation or suppression in chromatin.  In recent years, considerable efforts have been made to develop chem. inhibitors of these bromodomains in an effort to probe their cellular functions.  Potent and selective inhibitors have been extensively developed for one group of the bromodomain family termed BET proteins that consist of tandem bromodomains followed by an extra terminal domain.  Drug developers are actively designing inhibitors of other bromodomains and working to move the most successful inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQF2Qjn_iuLVg90H21EOLACvtfcHk0lhI4VoIbCfXkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ&md5=dd34fc591550ff1528e6dccd1d04f1c1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DThe%2520Bromodomain%253A%2520A%2520New%2520Target%2520in%2520Emerging%2520Epigenetic%2520Medicine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26issue%3D3%26spage%3D598%26epage%3D608%26doi%3D10.1021%2Facschembio.5b00831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovén, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.cell.2013.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=23582323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=320-334&issue=2&author=J.+Lov%C3%A9nauthor=H.+A.+Hokeauthor=C.+Y.+Linauthor=A.+Lauauthor=D.+A.+Orlandoauthor=C.+R.+Vakocauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Selective+Inhibition+of+Tumor+Oncogenes+by+Disruption+of+Super-Enhancers&doi=10.1016%2Fj.cell.2013.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span></div><div class="casAuthors">Loven, Jakob; Hoke, Heather A.; Lin, Charles Y.; Lau, Ashley; Orlando, David A.; Vakoc, Christopher R.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-334</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells.  Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM).  BRD4 and Mediator were found to co-occupy thousands of enhancers assocd. with active genes.  They also co-occupied a small set of exceptionally large super-enhancers assocd. with genes that feature prominently in MM biol., including the MYC oncogene.  Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC.  Super-enhancers were found at key oncogenic drivers in many other tumor cells.  These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-NqFYuGMqrVg90H21EOLACvtfcHk0lhI4VoIbCfXkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D&md5=66b6ccb5b8a6039832910443fd8b4664</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DLov%25C3%25A9n%26aufirst%3DJ.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520Inhibition%2520of%2520Tumor%2520Oncogenes%2520by%2520Disruption%2520of%2520Super-Enhancers%26jtitle%3DCell%26date%3D2013%26volume%3D153%26issue%3D2%26spage%3D320%26epage%3D334%26doi%3D10.1016%2Fj.cell.2013.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-F.</span></span> <span> </span><span class="NLM_article-title">Brd4 Coactivates Transcriptional Activation of NF-κB via Specific Binding to Acetylated RelA</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1375</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1128/MCB.01365-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&issue=5&author=B.+Huangauthor=X.-D.+Yangauthor=M.-M.+Zhouauthor=K.+Ozatoauthor=L.-F.+Chen&title=Brd4+Coactivates+Transcriptional+Activation+of+NF-%CE%BAB+via+Specific+Binding+to+Acetylated+RelA&doi=10.1128%2FMCB.01365-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0liHhqN_Qhnxfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.-D.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DBrd4%2520Coactivates%2520Transcriptional%2520Activation%2520of%2520NF-%25CE%25BAB%2520via%2520Specific%2520Binding%2520to%2520Acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26issue%3D5%26spage%3D1375%26epage%3D1387%26doi%3D10.1128%2FMCB.01365-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Clinical Progress and Pharmacology of Small Molecule Bromodomain Inhibitors</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2016.05.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.cbpa.2016.05.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=27295577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=58-66&author=N.+H.+Theodoulouauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Clinical+Progress+and+Pharmacology+of+Small+Molecule+Bromodomain+Inhibitors&doi=10.1016%2Fj.cbpa.2016.05.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58-66</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains have emerged as an exciting target class for drug discovery over the past decade.  Research has primarily focused on the bromodomain and extra terminal (BET) family of bromodomains, which has led to the development of multiple small mol. inhibitors and an increasing no. of clin. assets.  The excitement centered on the clin. potential of BET inhibition has stimulated intense interest in the broader family and the growing no. of non-BET bromodomain chem. probes has facilitated phenotypic investigations, implicating these targets in a variety of disease pathways including cancer, inflammation, embryonic development and neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEE0jCnUuyk7Vg90H21EOLACvtfcHk0liHhqN_Qhnxfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D&md5=4bcd42755b2dd3cebd7fae417e69e83f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2016.05.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2016.05.028%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DClinical%2520Progress%2520and%2520Pharmacology%2520of%2520Small%2520Molecule%2520Bromodomain%2520Inhibitors%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D33%26spage%3D58%26epage%3D66%26doi%3D10.1016%2Fj.cbpa.2016.05.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Scaling the Druggability Landscape of Human Bromodomains, a New Class of Drug Targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">7342</span>– <span class="NLM_lpage">7345</span>, <span class="refDoi"> DOI: 10.1021/jm3011977</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3011977" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1yhsLnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7342-7345&issue=17&author=G.+Zhangauthor=R.+Sanchezauthor=M.+Zhou&title=Scaling+the+Druggability+Landscape+of+Human+Bromodomains%2C+a+New+Class+of+Drug+Targets&doi=10.1021%2Fjm3011977"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling the Druggability Landscape of Human Bromodomains, a New Class of Drug Targets</span></div><div class="casAuthors">Zhang, Guangtao; Sanchez, Roberto; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7342-7345</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Gene transcriptional activation of the human genome in response to physiol. and environmental stimuli requires chromatin structure changes defined by enzymes that modify chromatin and directed by proteins that interact with chromatin in a modification-sensitive manner.  This highly complex system operates with a large no. of chem. modifications on chromatin (both DNA and histones) and transcription-assocd. proteins.  Of these, lysine acetylation functions to facilitate chromatin opening and productive transcriptional machinery assembly required for gene activation.  These activities are directed by the acetyl-lysine binding activity of the bromodomain (BrD), a fundamental mol. mechanism for gene transcriptional activation that was discovered in the structural biol. study of the histone acetyltransferase (HAT) transcriptional coactivator PCAF.  As a parallel in drug discovery that is moving from highly druggable targets such as kinases and GPCRs to more challenging PPI targets, rapidly growing ligand design efforts for bromodomains are expected to progress from targeting a few druggable bromodomains to more difficult ones.  The outcome of these studies will undoubtedly enable us to validate members of this new class of drug targets and develop more effective targeted epigenetic therapies for human diseases including cancer and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq7hhNrw3hGLVg90H21EOLACvtfcHk0lj0M8RZSsCJ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1yhsLnK&md5=d2b5afff493388e946e8753a35883d2b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm3011977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3011977%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DScaling%2520the%2520Druggability%2520Landscape%2520of%2520Human%2520Bromodomains%252C%2520a%2520New%2520Class%2520of%2520Drug%2520Targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26issue%3D17%26spage%3D7342%26epage%3D7345%26doi%3D10.1021%2Fjm3011977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Chemical Probes and Inhibitors of Bromodomains Outside the BET Family</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1039/C6MD00373G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1039%2FC6MD00373G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=29170712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2246-2264&issue=12&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=D.+A.+Hayauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Chemical+Probes+and+Inhibitors+of+Bromodomains+Outside+the+BET+Family&doi=10.1039%2FC6MD00373G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes and inhibitors of bromodomains outside the BET family</span></div><div class="casAuthors">Moustakim, Moses; Clark, Peter G. K.; Hay, Duncan A.; Dixon, Darren J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2246-2264</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last five years, the development of inhibitors of bromodomains has emerged as an area of intensive worldwide research.  Emerging evidence has implicated a no. of non-BET bromodomains in the onset and progression of diseases such as cancer, HIV infection and inflammation.  The development and use of small mol. chem. probes has been fundamental to pre-clin. evaluation of bromodomains as targets.  Recent efforts are described highlighting the development of potent, selective and cell active non-BET bromodomain inhibitors and their therapeutic potential.  Over half of typical bromodomains now have reported ligands, but those with atypical binding site residues remain resistant to chem. probe discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYJvhj3Z0Ry7Vg90H21EOLACvtfcHk0lj0M8RZSsCJ2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF&md5=df4c5718b15ff7d401a0cd9974abc236</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2FC6MD00373G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00373G%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DChemical%2520Probes%2520and%2520Inhibitors%2520of%2520Bromodomains%2520Outside%2520the%2520BET%2520Family%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26issue%3D12%26spage%3D2246%26epage%3D2264%26doi%3D10.1039%2FC6MD00373G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Progress in the Development of non-BET Bromodomain Chemical Probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2Fcmdc.201500540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=26749027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=477-487&issue=5&author=N.+H.+Theodoulouauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Progress+in+the+Development+of+non-BET+Bromodomain+Chemical+Probes&doi=10.1002%2Fcmdc.201500540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">477-487</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain and extra terminal (BET) family of bromodomains have been the focus of extensive research, leading to the development of many potent, selective chem. probes and recent clin. assets.  The profound biol. assocd. with BET bromodomain inhibition has provided a convincing rationale for targeting bromodomains for the treatment of disease.  However, the BET family represents just eight of the at least 56 human bromodomains identified to date.  Until recently, there has been significantly less interest in non-BET bromodomains, leaving a vast area of research and the majority of this new target class yet to be thoroughly investigated.  It has been widely reported that several non-BET bromodomain contg. proteins are assocd. with various diseases including cancer and HIV.  Therefore, the development of chem. probes for non-BET bromodomains will facilitate elucidation of their precise biol. roles and potentially lead to the development of new medicines.  This review summarises the progress made towards the development of non-BET bromodomain chem. probes to date.  In addn., we highlight the potential for future work in this new and exciting area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYLj_VyHOS5bVg90H21EOLACvtfcHk0lgHmM-cpkfVIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D&md5=1e1339bf09a10bde8621c18344172fa6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DProgress%2520in%2520the%2520Development%2520of%2520non-BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26issue%3D5%26spage%3D477%26epage%3D487%26doi%3D10.1002%2Fcmdc.201500540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span> <span> </span><span class="NLM_article-title">Target (In)Validation: A Critical, Sometimes Unheralded, Role of Modern Medicinal Chemistry</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">618</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovVSnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=618-621&issue=6&author=R.+F.+Sweis&title=Target+%28In%29Validation%3A+A+Critical%2C+Sometimes+Unheralded%2C+Role+of+Modern+Medicinal+Chemistry&doi=10.1021%2Facsmedchemlett.5b00183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Target (In)Validation: A Critical, Sometimes Unheralded, Role of Modern Medicinal Chemistry</span></div><div class="casAuthors">Sweis, Ramzi F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">618-621</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review. Small mol. drug discovery commonly ventures into previously unknown and unexplored target space.  For such programs, an important role of medicinal chem. is to generate mols. that enable the most reliable conclusions from a preclin. target validation/invalidation study.  Multiple facets of chem. that provide the most rigorous results for such an expt. are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZCtopNE1bl7Vg90H21EOLACvtfcHk0lgHmM-cpkfVIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovVSnu7o%253D&md5=0da4f3e6e2ead39de6e16219d20c118a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00183%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26atitle%3DTarget%2520%2528In%2529Validation%253A%2520A%2520Critical%252C%2520Sometimes%2520Unheralded%252C%2520Role%2520of%2520Modern%2520Medicinal%2520Chemistry%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26issue%3D6%26spage%3D618%26epage%3D621%26doi%3D10.1021%2Facsmedchemlett.5b00183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Bromodomains: Are Readers Right for Epigenetic Therapy?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">691</span>– <span class="NLM_lpage">694</span>, <span class="refDoi"> DOI: 10.1021/ml300221t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300221t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFKgu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=691-694&issue=9&author=S.+J.+Conway&title=Bromodomains%3A+Are+Readers+Right+for+Epigenetic+Therapy%3F&doi=10.1021%2Fml300221t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains: Are Readers Right for Epigenetic Therapy?</span></div><div class="casAuthors">Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">691-694</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  There is intense interest in the development of small mol. inhibitors of the acetyl-lysine-reading bromodomain protein module.  These inhibitors represent a way of interfering therapeutically in epigenetic processes, and there are currently two bromodomain inhibitors in clin. trials.  The success of these compds. rests on safety aspects of epigenetic target modulation being addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLmhkWiev2rLVg90H21EOLACvtfcHk0ljcUv2J1ZmxNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFKgu7nL&md5=e3ac85cb4b639616b829295e866da801</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fml300221t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300221t%26sid%3Dliteratum%253Aachs%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DBromodomains%253A%2520Are%2520Readers%2520Right%2520for%2520Epigenetic%2520Therapy%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26issue%3D9%26spage%3D691%26epage%3D694%26doi%3D10.1021%2Fml300221t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&issue=4&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+Acetyl-Lysine+Recognition%3A+Progress+in+the+Development+of+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0ljcUv2J1ZmxNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520Acetyl-Lysine%2520Recognition%253A%2520Progress%2520in%2520the%2520Development%2520of%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Advancements in the Development of non-BET Bromodomain Chemical Probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2Fcmdc.201800738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=30624862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=362-385&issue=4&author=M.+A.+Cleggauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Advancements+in+the+Development+of+non-BET+Bromodomain+Chemical+Probes&doi=10.1002%2Fcmdc.201800738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Advancements in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">362-385</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extra terminal (BET) family of bromodomain-contg. proteins (BCPs) have been the subject of extensive research over the past decade, resulting in a plethora of high-quality chem. probes for their tandem bromodomains.  In turn, these chem. probes have helped reveal the profound biol. role of the BET bromodomains and their role in disease, ultimately leading to a no. of mols. in active clin. development.  However, the BET subfamily represents just 8/61 of the known human bromodomains, and attention has now expanded to the biol. role of the remaining 53 non-BET bromodomains.  Rapid growth of this research area has been accompanied by a greater understanding of the requirements for an effective bromodomain chem. probe and has led to a no. of new non-BET bromodomain chem. probes being developed.  Advances since Dec. 2015 are discussed, highlighting the strengths/caveats of each mol., and the value they add toward validating the non-BET bromodomains as tractable therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_U9uvpaCEd7Vg90H21EOLACvtfcHk0ljcUv2J1ZmxNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D&md5=a93505759fe45ffd0352de2f0e17885a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DAdvancements%2520in%2520the%2520Development%2520of%2520non-BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26issue%3D4%26spage%3D362%26epage%3D385%26doi%3D10.1002%2Fcmdc.201800738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Target Validation using Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&issue=4&author=M.+E.+Bunnageauthor=E.+L.+P.+Cheklerauthor=L.+H.+Jones&title=Target+Validation+using+Chemical+Probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0li-o7SlfgmvGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520Validation%2520using%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D4%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">The Art of the Chemical Probe</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">159</span>, <span class="refDoi"> DOI: 10.1038/nchembio.296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159&author=S.+V.+Frye&title=The+Art+of+the+Chemical+Probe&doi=10.1038%2Fnchembio.296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0li-o7SlfgmvGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520Art%2520of%2520the%2520Chemical%2520Probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26doi%3D10.1038%2Fnchembio.296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Small Molecules and their Role in Effective Preclinical Target Validation</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">1579</span>– <span class="NLM_lpage">1582</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.4155%2Ffmc-2017-0121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=28828889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFensbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1579-1582&issue=14&author=M.+A.+Cleggauthor=N.+C.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Small+Molecules+and+their+Role+in+Effective+Preclinical+Target+Validation&doi=10.4155%2Ffmc-2017-0121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules and their role in effective preclinical target validation</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1579-1582</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUfI_qHeoQ0rVg90H21EOLACvtfcHk0ljG4F370Dik1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFensbfM&md5=b8fea6af5ce3bf5c61e1a58c445404b8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0121%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DSmall%2520Molecules%2520and%2520their%2520Role%2520in%2520Effective%2520Preclinical%2520Target%2520Validation%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26issue%3D14%26spage%3D1579%26epage%3D1582%26doi%3D10.4155%2Ffmc-2017-0121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-W.</span></span> <span> </span><span class="NLM_article-title">ANCCA, an Estrogen-Regulated AAA+ ATPase Coactivator for ERa, is Required for Coregulator Occupancy and Chromatin Modification</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span> (<span class="NLM_issue">46</span>),  <span class="NLM_fpage">18067</span>– <span class="NLM_lpage">18072</span>, <span class="refDoi"> DOI: 10.1073/pnas.0705814104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1073%2Fpnas.0705814104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=17998543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=18067-18072&issue=46&author=J.+X.+Zouauthor=A.+S.+Revenkoauthor=L.+B.+Liauthor=A.+T.+Gemoauthor=H.-W.+Chen&title=ANCCA%2C+an+Estrogen-Regulated+AAA%2B+ATPase+Coactivator+for+ERa%2C+is+Required+for+Coregulator+Occupancy+and+Chromatin+Modification&doi=10.1073%2Fpnas.0705814104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERα, is required for coregulator occupancy and chromatin modification</span></div><div class="casAuthors">Zou, June X.; Revenko, Alexey S.; Li, Li B.; Gemo, Abigael T.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18067-18072</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">AAA+ proteins play crucial roles in diverse biol. processes via their ATPase-driven remodeling of macromol. complexes.  Here the authors report the authors' identification of an evolutionarily conserved AAA+ protein, ANCCA/Pro 2000, endowed with a bromodomain that is strongly induced by estrogen in human breast cancer cells and is a direct target of protooncogene ACTR/AlB1/SRC-3.  The authors found that ANCCA assocs. directly with estrogen-bound estrogen receptor (ER) α and ACTR.  It is selectively recruited, upon estrogen stimulation, to a subset of ERα target genes including cyclin D1, c-myc, and E2F1 and is required for their estrogen-induced expression as well as breast cancer cell proliferation.  Further studies indicate that ANCCA binds and hydrolyzes ATP and is crit. for recruitment of coregulator CBP and histone hyperacetylation at the ER target chromatin.  Moreover, mutations at the ATP binding motifs rendered ANCCA defective as a coactivator in mediating estrogen induction of gene expression.  Together, the authors' findings reveal an unexpected layer of regulatory mechanism in hormone signaling mediated by ANCCA and suggest that hormone-induced assembly of transcriptional coregulator complexes at chromatin is a process facilitated by AAA+ ATPase proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NFjjiBVoRbVg90H21EOLACvtfcHk0ljG4F370Dik1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO&md5=618fcc431bf4b0a7589213fea55732ca</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0705814104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0705814104%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DLi%26aufirst%3DL.%2BB.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DH.-W.%26atitle%3DANCCA%252C%2520an%2520Estrogen-Regulated%2520AAA%252B%2520ATPase%2520Coactivator%2520for%2520ERa%252C%2520is%2520Required%2520for%2520Coregulator%2520Occupancy%2520and%2520Chromatin%2520Modification%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26issue%3D46%26spage%3D18067%26epage%3D18072%26doi%3D10.1073%2Fpnas.0705814104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lestrat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoffier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virolle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Functional Characterization of ATAD2 as a New Cancer/Testis Factor and a Predictor of Poor Prognosis in Breast and Lung Cancers</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">37</span>),  <span class="NLM_fpage">5171</span>– <span class="NLM_lpage">5181</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fonc.2010.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=20581866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=5171-5181&issue=37&author=C.+Caronauthor=C.+Lestratauthor=S.+Marsalauthor=E.+Escoffierauthor=S.+Curtetauthor=V.+Virolleauthor=P.+Barbryauthor=A.+Debernardiauthor=C.+Brambillaauthor=E.+Brambillaauthor=S.+Rousseauxauthor=S.+Khochbin&title=Functional+Characterization+of+ATAD2+as+a+New+Cancer%2FTestis+Factor+and+a+Predictor+of+Poor+Prognosis+in+Breast+and+Lung+Cancers&doi=10.1038%2Fonc.2010.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span></div><div class="casAuthors">Caron, C.; Lestrat, C.; Marsal, S.; Escoffier, E.; Curtet, S.; Virolle, V.; Barbry, P.; Debernardi, A.; Brambilla, C.; Brambilla, E.; Rousseaux, S.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5171-5181</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells frequently express genes normally active in male germ cells.  ATAD2 is one of them encoding a conserved factor harboring an AAA type ATPase domain and a bromodomain.  We show here that ATAD2 is highly expressed in testis as well as in many cancers of different origins and that its high expression is a strong predictor of rapid mortality in lung and breast cancers.  These observations suggest that ATAD2 acts on upstream and basic cellular processes to enhance oncogenesis in a variety of unrelated cell types.  Accordingly, our functional studies show that ATAD2 controls chromatin dynamics, genome transcriptional activities and apoptotic cell response.  We could also highlight some of the important intrinsic properties of its two regulatory domains, including a functional cross-talk between the AAA ATPase domain and the bromodomain.  Altogether, these data indicate that ATAD2 overexpression in somatic cells, by acting on basic properties of chromatin, may contribute to malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotoD3fqS5d4LVg90H21EOLACvtfcHk0lgnYY-TS7o4vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D&md5=1eaa99ca9fd31e5a486bedd4d0f6178d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.259%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DLestrat%26aufirst%3DC.%26aulast%3DMarsal%26aufirst%3DS.%26aulast%3DEscoffier%26aufirst%3DE.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVirolle%26aufirst%3DV.%26aulast%3DBarbry%26aufirst%3DP.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DBrambilla%26aufirst%3DC.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DFunctional%2520Characterization%2520of%2520ATAD2%2520as%2520a%2520New%2520Cancer%252FTestis%2520Factor%2520and%2520a%2520Predictor%2520of%2520Poor%2520Prognosis%2520in%2520Breast%2520and%2520Lung%2520Cancers%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26issue%3D37%26spage%3D5171%26epage%3D5181%26doi%3D10.1038%2Fonc.2010.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boussouar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidikia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morozumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Malignant Genome Reprogramming by ATAD2</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1829</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2013.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.bbagrm.2013.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1829&publication_year=2013&pages=1010-1014&issue=10&author=F.+Boussouarauthor=M.+Jamshidikiaauthor=Y.+Morozumiauthor=S.+Rousseauxauthor=S.+Khochbin&title=Malignant+Genome+Reprogramming+by+ATAD2&doi=10.1016%2Fj.bbagrm.2013.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Malignant genome reprogramming by ATAD2</span></div><div class="casAuthors">Boussouar, Faycal; Jamshidikia, Mahya; Morozumi, Yuichi; Rousseaux, Sophie; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1829</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1010-1014</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Unscheduled expression of crit. cellular regulators could be central to malignant genome reprogramming and tumor establishment.  One such factor appears to be ATAD2, a remarkably conserved protein normally predominantly expressed in germ cells but almost systematically over-expressed in a variety of unrelated cancers.  The presence of a bromodomain adjacent to an AAA type ATPase domain, points to ATAD2 as a factor preliminarily acting on chromatin structure and function.  Accordingly, ATAD2 has been shown to cooperate with a series of transcription factors and chromatin modifiers to regulate specific set of genes.  Here we discuss our knowledge on ATAD2 to evaluate its role as a cancer driver and its value as a new anti-cancer target.  Upon its activation, ATAD2 through its interaction with defined transcription factors, initiates a loop of transcriptional stimulation of target genes, including ATAD2 itself, leading to enhanced cell proliferation and resistance to apoptosis in an ATAD2-dependent manner.  Approaches aiming at neutralizing ATAD2 activity in cancer, including the use of small mol. inhibitors of its two "druggable" domains, AAA ATPase and bromodomain, could become part of a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp824JMo6Dp4rVg90H21EOLACvtfcHk0lgnYY-TS7o4vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF&md5=cba5991a2dc5f96f522e1104ae4e4889</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2013.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2013.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DMalignant%2520Genome%2520Reprogramming%2520by%2520ATAD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2013%26volume%3D1829%26issue%3D10%26spage%3D1010%26epage%3D1014%26doi%3D10.1016%2Fj.bbagrm.2013.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banting, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barak, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ames, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnham, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kardel, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooch, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godbout, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermid, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span> <span> </span><span class="NLM_article-title">CECR2, a Protein Involved in Neurulation, Forms a Novel Chromatin Remodeling Complex with SNF2L</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddi048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1093%2Fhmg%2Fddi048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=15640247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtV2qs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=513-524&issue=4&author=G.+S.+Bantingauthor=O.+Barakauthor=T.+M.+Amesauthor=A.+C.+Burnhamauthor=M.+D.+Kardelauthor=N.+S.+Coochauthor=C.+E.+Davidsonauthor=R.+Godboutauthor=H.+E.+McDermidauthor=R.+Shiekhattar&title=CECR2%2C+a+Protein+Involved+in+Neurulation%2C+Forms+a+Novel+Chromatin+Remodeling+Complex+with+SNF2L&doi=10.1093%2Fhmg%2Fddi048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">CECR2, a protein involved in neurulation, forms a novel chromatin remodeling complex with SNF2L</span></div><div class="casAuthors">Banting, Graham S.; Barak, Orr; Ames, Tanya M.; Burnham, Amanda C.; Kardel, Melanie D.; Cooch, Neil S.; Davidson, Courtney E.; Godbout, Roseline; McDermid, Heather E.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">513-524</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Chromatin remodeling complexes play crit. roles in development.  Here we describe a transcription factor, CECR2, which is involved in neurulation and chromatin remodeling.  CECR2 shows complex alternative splicing, but all variants contain DDT and bromodomain motifs.  A mutant mouse line was generated from an embryonic stem cell line contg. a genetrap within Cecr2.  Reporter gene expression demonstrated Cecr2 expression to be predominantly neural in the embryo.  Mice homozygous for the Cecr2 genetrap-induced mutation show a high penetrance of the neural tube defect exencephaly, the human equiv. of anencephaly, in a strain-dependent fashion.  Biochem. isolation of CECR2 revealed the presence of this protein as a component of a novel heterodimeric complex termed CECR2-contg. remodeling factor (CERF).  CERF comprises CECR2 and the ATP-dependent chromatin remodeler SNF2L, a mammalian ISWI ortholog expressed predominantly in the central nervous system.  CERF is capable of remodeling chromatin in vitro and displays an ATP hydrolyzing activity that is stimulated by nucleosomes.  Together, these data identify a novel chromatin remodeling complex with a crit. role in neurulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVKYqBa3GTk7Vg90H21EOLACvtfcHk0ljrCKpzHNYNYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtV2qs7g%253D&md5=88583cf64ca64c079bb5b88b1e3d9c51</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddi048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddi048%26sid%3Dliteratum%253Aachs%26aulast%3DBanting%26aufirst%3DG.%2BS.%26aulast%3DBarak%26aufirst%3DO.%26aulast%3DAmes%26aufirst%3DT.%2BM.%26aulast%3DBurnham%26aufirst%3DA.%2BC.%26aulast%3DKardel%26aufirst%3DM.%2BD.%26aulast%3DCooch%26aufirst%3DN.%2BS.%26aulast%3DDavidson%26aufirst%3DC.%2BE.%26aulast%3DGodbout%26aufirst%3DR.%26aulast%3DMcDermid%26aufirst%3DH.%2BE.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DCECR2%252C%2520a%2520Protein%2520Involved%2520in%2520Neurulation%252C%2520Forms%2520a%2520Novel%2520Chromatin%2520Remodeling%2520Complex%2520with%2520SNF2L%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2005%26volume%3D14%26issue%3D4%26spage%3D513%26epage%3D524%26doi%3D10.1093%2Fhmg%2Fddi048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toyoshima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howie, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imakura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annis, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frazier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loboda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsley, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandori, C.</span></span> <span> </span><span class="NLM_article-title">Functional Genomics Identifies Therapeutic Targets for MYC-Driven Cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">9545</span>– <span class="NLM_lpage">9550</span>, <span class="refDoi"> DOI: 10.1073/pnas.1121119109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1073%2Fpnas.1121119109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=22623531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38Xptlahs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=9545-9550&issue=24&author=M.+Toyoshimaauthor=H.+L.+Howieauthor=M.+Imakuraauthor=R.+M.+Walshauthor=J.+E.+Annisauthor=A.+N.+Changauthor=J.+Frazierauthor=B.+N.+Chauauthor=A.+Lobodaauthor=P.+S.+Linsleyauthor=M.+A.+Clearyauthor=J.+R.+Parkauthor=C.+Grandori&title=Functional+Genomics+Identifies+Therapeutic+Targets+for+MYC-Driven+Cancer&doi=10.1073%2Fpnas.1121119109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Functional genomics identifies therapeutic targets for MYC-driven cancer</span></div><div class="casAuthors">Toyoshima, Masafumi; Howie, Heather L.; Imakura, Maki; Walsh, Ryan M.; Annis, James E.; Chang, Aaron N.; Frazier, Jason; Chau, Nelson; Loboda, Andrey; Linsley, Peter S.; Cleary, Michele A.; Park, Julie R.; Grandori, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9545-9550, S9545/1-S9545/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">MYC oncogene family members are broadly implicated in human cancers, yet are considered "undruggable" as they encode transcription factors.  MYC also carries out essential functions in proliferative tissues, suggesting that its inhibition could cause severe side effects.  We elected to identify synthetic lethal interactions with c-MYC overexpression (MYC-SL) in a collection of ∼3,300 druggable genes, using high-throughput siRNA screening.  Of 49 genes selected for follow-up, 48 were confirmed by independent retesting and approx. one-third selectively induced accumulation of DNA damage, consistent with enrichment in DNA-repair genes by functional annotation.  In addn., genes involved in histone acetylation and transcriptional elongation, such as TRRAP and BRD4, were identified, indicating that the screen revealed known MYC-assocd. pathways.  For in vivo validation we selected CSNK1e, a kinase whose expression correlated with MYCN amplification in neuroblastoma (an established MYC-driven cancer).  Using RNAi and available small-mol. inhibitors, we confirmed that inhibition of CSNK1e halted growth of MYCN-amplified neuroblastoma xenografts.  CSNK1e had previously been implicated in the regulation of developmental pathways and circadian rhythms, whereas our data provide a previously unknown link with oncogenic MYC.  Furthermore, expression of CSNKle correlated with c-MYC and its transcriptional signature in other human cancers, indicating potential broad therapeutic implications of targeting CSNK1e function.  In summary, through a functional genomics approach, pathways essential in the context of oncogenic MYC but not to normal cells were identified, thus revealing a rich therapeutic space linked to a previously "undruggable" oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr60vylCRVnA7Vg90H21EOLACvtfcHk0ljrCKpzHNYNYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xptlahs7Y%253D&md5=32a3095f78c37c4a3a7b8878089d6f77</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1121119109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1121119109%26sid%3Dliteratum%253Aachs%26aulast%3DToyoshima%26aufirst%3DM.%26aulast%3DHowie%26aufirst%3DH.%2BL.%26aulast%3DImakura%26aufirst%3DM.%26aulast%3DWalsh%26aufirst%3DR.%2BM.%26aulast%3DAnnis%26aufirst%3DJ.%2BE.%26aulast%3DChang%26aufirst%3DA.%2BN.%26aulast%3DFrazier%26aufirst%3DJ.%26aulast%3DChau%26aufirst%3DB.%2BN.%26aulast%3DLoboda%26aufirst%3DA.%26aulast%3DLinsley%26aufirst%3DP.%2BS.%26aulast%3DCleary%26aufirst%3DM.%2BA.%26aulast%3DPark%26aufirst%3DJ.%2BR.%26aulast%3DGrandori%26aufirst%3DC.%26atitle%3DFunctional%2520Genomics%2520Identifies%2520Therapeutic%2520Targets%2520for%2520MYC-Driven%2520Cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D24%26spage%3D9545%26epage%3D9550%26doi%3D10.1073%2Fpnas.1121119109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, J.</span></span> <span> </span><span class="NLM_article-title">Genome-Wide Screen of Human Bromodomain-Containing Proteins identifies Cecr2 as a Novel DNA Damage Response Protein</span>. <i>Mol. Cells</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1007/s10059-012-0112-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1007%2Fs10059-012-0112-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=22699752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFSqsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2012&pages=85-91&issue=1&author=S.-K.+Leeauthor=E.-J.+Parkauthor=H.-S.+Leeauthor=Y.+S.+Leeauthor=J.+Kwon&title=Genome-Wide+Screen+of+Human+Bromodomain-Containing+Proteins+identifies+Cecr2+as+a+Novel+DNA+Damage+Response+Protein&doi=10.1007%2Fs10059-012-0112-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide screen of human bromodomain-containing proteins identifies Cecr2 as a novel DNA damage response protein</span></div><div class="casAuthors">Lee, Seul-Ki; Park, Eun-Jung; Lee, Han-Sae; Lee, Ye Seul; Kwon, Jongbum</div><div class="citationInfo"><span class="NLM_cas:title">Molecules and Cells</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-91</span>CODEN:
                <span class="NLM_cas:coden">MOCEEK</span>;
        ISSN:<span class="NLM_cas:issn">1016-8478</span>.
    
            (<span class="NLM_cas:orgname">Korean Society for Molecular and Cellular Biology</span>)
        </div><div class="casAbstract">The formation of γ-H2AX foci after DNA double strand breaks (DSBs) is crucial for the cellular response to this lethal DNA damage.  We previously have shown that BRG1, a chromatin remodeling enzyme, facilitates DSB repair by stimulating γ-H2AX formation, and this function of BRG1 requires the binding of BRGI to acetylated histone H3 on γ-H2AX-contg. nucleosomes using its bromodomain (BRD), a protein module that specifically recognizes acetyl-Lys moieties.  We also have shown that the BRD of BRG1, when ectopically expressed in cells, functions as a dominant neg. inhibitor of the BRG1 activity to stimulate γ-H2AX and DSB repair.  Here, we found that BRDs from a select group of proteins have no such activity, suggesting that the γ-H2AX inhibition activity of BRG1 BRD is specific.  This finding led us to search for more BRDs that exhibit γ-H2AX inhibition activity in the hope of finding addnl. BRD-contg. proteins involved in DNA damage responses.  We screened a total of 52 individual BRDs present in 38 human BRD-contg. proteins, comprising 93% of all human BRDs.  We identified the BRD of cat eye syndrome chromosome region candidate 2 (Cecr2), which recently was shown to form a novel chromatin remodeling complex with unknown cellular functions, as having a strong γ-H2AX inhibition activity.  This activity of Cecr2 BRD is specific because it depends on the chromatin binding affinity of Cecr2 BRD.  Small interfering RNA knockdown expts. showed that Cecr2 is important for γ-H2AX formation and DSB repair.  Therefore, our genomewide screen identifies Cecr2 as a novel DNA damage response protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEipOhHZMKBLVg90H21EOLACvtfcHk0lgweurEwLuROg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFSqsrjO&md5=bf381774000bd4e65dfcc1033c1b1a29</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2Fs10059-012-0112-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10059-012-0112-4%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.-K.%26aulast%3DPark%26aufirst%3DE.-J.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DLee%26aufirst%3DY.%2BS.%26aulast%3DKwon%26aufirst%3DJ.%26atitle%3DGenome-Wide%2520Screen%2520of%2520Human%2520Bromodomain-Containing%2520Proteins%2520identifies%2520Cecr2%2520as%2520a%2520Novel%2520DNA%2520Damage%2520Response%2520Protein%26jtitle%3DMol.%2520Cells%26date%3D2012%26volume%3D34%26issue%3D1%26spage%3D85%26epage%3D91%26doi%3D10.1007%2Fs10059-012-0112-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&issue=17&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+Analysis+and+Structural+Classification+of+Bromodomain+Acetyl-lysine+Binding+Sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lgweurEwLuROg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520Analysis%2520and%2520Structural%2520Classification%2520of%2520Bromodomain%2520Acetyl-lysine%2520Binding%2520Sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26issue%3D17%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, H. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ATAD2 in Cancer: A Pharmacologically Challenging but Tractable Target</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1406921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1080%2F14728222.2018.1406921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=29148850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=85-96&issue=1&author=M.+Hussainauthor=Y.+Zhouauthor=Y.+Songauthor=H.+M.+A.+Hameedauthor=H.+Jiangauthor=Y.+Tuauthor=J.+Zhang&title=ATAD2+in+Cancer%3A+A+Pharmacologically+Challenging+but+Tractable+Target&doi=10.1080%2F14728222.2018.1406921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 in cancer: a pharmacologically challenging but tractable target</span></div><div class="casAuthors">Hussain, Muzammal; Zhou, Yang; Song, Yu; Hameed, H. M. Adnan; Jiang, Hao; Tu, Yaoquan; Zhang, Jiancun</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-96</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  ATAD2 protein is an emerging oncogene that has strongly been linked to the etiol. of multiple advanced human cancers.  Therapeutically, despite the fact that genetic suppression/knockdown studies have validated it as a compelling drug target for future therapeutic development, recent druggability assessment data suggest that direct targeting of ATAD2's bromodomain (BRD) may be a very challenging task.  ATAD2's BRD has been predicted as a 'difficult to drug' or 'least druggable' target due to the concern that its binding pocket, and the areas around it, seem to be unfeasible for ligand binding.: In this review, after shedding light on the multifaceted roles of ATAD2 in normal physiol. as well as in cancer-etiol., we discuss tech. challenges rendered by ATAD2's BRD active site and the recent drug discovery efforts to find small mol. inhibitors against it.: The identification of a novel low-nanomolar semi-permeable chem. probe against ATAD2's BRD by recent drug discovery campaign has demonstrated it to be a pharmacol. tractable target.  Nevertheless, the development of high quality bioavailable inhibitors against ATAD2 is still a pending task.  Moreover, ATAD2 may also potentially be utilized as a promising target for future development of RNAi-based therapy to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0bdigfdvMjbVg90H21EOLACvtfcHk0lhXXQ4mb8gLkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN&md5=5d0ff7c936bb9eff125efeeaa4d275d1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1406921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1406921%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DHameed%26aufirst%3DH.%2BM.%2BA.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DTu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DATAD2%2520in%2520Cancer%253A%2520A%2520Pharmacologically%2520Challenging%2520but%2520Tractable%2520Target%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26issue%3D1%26spage%3D85%26epage%3D96%26doi%3D10.1080%2F14728222.2018.1406921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Approaches Towards Identification of Fragments for the Low-Druggability ATAD2 Bromodomain</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1848</span>, <span class="refDoi"> DOI: 10.1039/C4MD00237G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1039%2FC4MD00237G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGqtr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1843-1848&issue=12&author=A.+Chaikuadauthor=A.+M.+Petrosauthor=O.+Fedorovauthor=J.+Xuauthor=S.+Knapp&title=Structure-Based+Approaches+Towards+Identification+of+Fragments+for+the+Low-Druggability+ATAD2+Bromodomain&doi=10.1039%2FC4MD00237G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based approaches towards identification of fragments for the low-druggability ATAD2 bromodomain</span></div><div class="casAuthors">Chaikuad, Apirat; Petros, Andrew M.; Fedorov, Oleg; Xu, Jing; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1843-1848</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The transcriptional co-regulator ATAD2 is a prognostic marker for patient survival in many cancers.  ATAD2 harbours a bromodomain which may offer an opportunity for pharmacol. intervention, but its shallow, polar binding surface makes the development of inhibitors challenging.  Here we optimized crystal transfer/soaking conditions enabling crystallog. fragment screening.  We describe nine crystal structures of fragments including thymidine, a novel acetyl-lysine mimetic ligand and the evaluation of the binding properties of the identified fragments using NMR chem. shift perturbation expts.  The presented binding modes offer chem. starting points for the development of more potent ATAD2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqBr9VBOWv17Vg90H21EOLACvtfcHk0lhXXQ4mb8gLkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGqtr3F&md5=52a30d24141c11b63efbe8b5b0ecb04c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1039%2FC4MD00237G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00237G%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-Based%2520Approaches%2520Towards%2520Identification%2520of%2520Fragments%2520for%2520the%2520Low-Druggability%2520ATAD2%2520Bromodomain%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1843%26epage%3D1848%26doi%3D10.1039%2FC4MD00237G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmons, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouzon, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5649</span>– <span class="NLM_lpage">5673</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5649-5673&issue=14&author=E.+H.+Demontauthor=C.-w.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=C.+Wellawayauthor=N.+Barrettauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=A.+J.+Emmonsauthor=E.+J.+Jonesauthor=B.+V.+Karamshiauthor=K.+Lockeauthor=D.+J.+Mitchellauthor=B.+H.+Mouzonauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watsonauthor=P.+Bamborough&title=Fragment-Based+Discovery+of+Low-Micromolar+ATAD2+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Demont, Emmanuel H.; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Wellaway, Chris; Barrett, Nathalie; Bridges, Angela M.; Craggs, Peter D.; Diallo, Hawa; Dixon, David P.; Douault, Clement; Emmons, Amanda J.; Jones, Emma J.; Karamshi, Bhumika V.; Locke, Kelly; Mitchell, Darren J.; Mouzon, Bernadette H.; Prinjha, Rab K.; Roberts, Andy D.; Sheppard, Robert J.; Watson, Robert J.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5649-5673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Overexpression of ATAD2 (ATPase family, AAA domain contg. 2) has been linked to disease severity and progression in a wide range of cancers, and is implicated in the regulation of several drivers of cancer growth.  Little is known of the dependence of these effects upon the ATAD2 bromodomain, which has been categorized as among the least tractable of its class.  The absence of any potent, selective inhibitors limits clear understanding of the therapeutic potential of the bromodomain.  Here, the authors describe the discovery of a hit from a fragment-based targeted array.  Optimization of this produced the first known micromolar inhibitors of the ATAD2 bromodomain, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5pJPrVexJbVg90H21EOLACvtfcHk0lg8HUwOzwNPpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP&md5=c133ce439365cd4e4110230827a7a0a4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00772%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DN.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DEmmons%26aufirst%3DA.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%2BV.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DMouzon%26aufirst%3DB.%2BH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-Based%2520Discovery%2520of%2520Low-Micromolar%2520ATAD2%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D14%26spage%3D5649%26epage%3D5673%26doi%3D10.1021%2Facs.jmedchem.5b00772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&issue=15&author=P.+Bamboroughauthor=C.-w.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-Based+Optimization+of+Naphthyridones+into+Potent+ATAD2+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0lg8HUwOzwNPpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-Based%2520Optimization%2520of%2520Naphthyridones%2520into%2520Potent%2520ATAD2%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D15%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe for the ATAD2 Bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">11382</span>– <span class="NLM_lpage">11386</span>, <span class="refDoi"> DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&issue=38&author=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+Chemical+Probe+for+the+ATAD2+Bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0lgHdUVAwo2ChQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Chemical%2520Probe%2520for%2520the%2520ATAD2%2520Bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26issue%3D38%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Díaz-Sáez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&issue=11&author=A.+E.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+St%C3%B6ckigtauthor=A.+ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+D%C3%ADaz-S%C3%A1ezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=J.+H%C3%BCbnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Isoform-Selective+ATAD2+Chemical+Probe+with+Novel+Chemical+Structure+and+Unusual+Mode+of+Action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span></div><div class="casAuthors">Fernandez-Montalvan, Amaury E.; Berger, Markus; Kuropka, Benno; Koo, Seong Joo; Badock, Volker; Weiske, Joerg; Puetter, Vera; Holton, Simon J.; Stoeckigt, Detlef; ter Laak, Antonius; Centrella, Paolo A.; Clark, Matthew A.; Dumelin, Christoph E.; Sigel, Eric A.; Soutter, Holly H.; Troast, Dawn M.; Zhang, Ying; Cuozzo, John W.; Keefe, Anthony D.; Roche, Didier; Rodeschini, Vincent; Chaikuad, Apirat; Diaz-Saez, Laura; Bennett, James M.; Fedorov, Oleg; Huber, Kilian V. M.; Huebner, Jan; Weinmann, Hilmar; Hartung, Ingo V.; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2730-2736</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 (ANCCA) is an epigenetic regulator and transcriptional cofactor, whose overexpression has been linked to the progress of various cancer types.  Here, we report a DNA-encoded library screen leading to the discovery of BAY-850, a potent and isoform selective inhibitor that specifically induces ATAD2 bromodomain dimerization and prevents interactions with acetylated histones in vitro, as well as with chromatin in cells.  These features qualify BAY-850 as a chem. probe to explore ATAD2 biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCB3nyG9YvubVg90H21EOLACvtfcHk0lgHdUVAwo2ChQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL&md5=1f09000a7d7e60e30540fbbae117acfe</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DSt%25C3%25B6ckigt%26aufirst%3DD.%26aulast%3Dter%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DD%25C3%25ADaz-S%25C3%25A1ez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DH%25C3%25BCbner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DIsoform-Selective%2520ATAD2%2520Chemical%2520Probe%2520with%2520Novel%2520Chemical%2520Structure%2520and%2520Unusual%2520Mode%2520of%2520Action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D11%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous Targeting of Bromodomains by Bromosporine Identifies BET Proteins as Master Regulators of Primary Transcription Response in Leukemia</span>. <i>Science Advances</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">e1600760</span>, <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=e1600760&issue=10&author=S.+Picaudauthor=K.+Leonardsauthor=J.-P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.-Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.-C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+Targeting+of+Bromodomains+by+Bromosporine+Identifies+BET+Proteins+as+Master+Regulators+of+Primary+Transcription+Response+in+Leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0lhKoVt5ncCdiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.-Y.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520Targeting%2520of%2520Bromodomains%2520by%2520Bromosporine%2520Identifies%2520BET%2520Proteins%2520as%2520Master%2520Regulators%2520of%2520Primary%2520Transcription%2520Response%2520in%2520Leukemia%26jtitle%3DScience%2520Advances%26date%3D2016%26volume%3D2%26issue%3D10%26spage%3De1600760%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><a href="https://www.thesgc.org/chemical-probes/NVS-1" class="extLink">https://www.thesgc.org/chemical-probes/NVS-1</a>. (accessed Mar 22, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FNVS-1.+%28accessed+Mar+22%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><a href="https://www.thesgc.org/chemical-probes/TP-238" class="extLink">https://www.thesgc.org/chemical-probes/TP-238</a>. (accessed Mar 22, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FTP-238.+%28accessed+Mar+22%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&issue=11&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=Diving+into+the+Water%3A+Inducible+Binding+Conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+Bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0lgiVq9006iDhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520into%2520the%2520Water%253A%2520Inducible%2520Binding%2520Conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D11%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bommi-Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">GNE-886: A Potent and Selective Inhibitor of the Cat Eye Syndrome Chromosome Region Candidate 2 Bromodomain (CECR2)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">737</span>– <span class="NLM_lpage">741</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00132</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00132" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=737-741&issue=7&author=T.+D.+Crawfordauthor=J.+E.+Audiaauthor=S.+Bellonauthor=D.+J.+Burdickauthor=A.+Bommi-Reddyauthor=A.+C%C3%B4t%C3%A9author=R.+T.+Cummingsauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=Y.+Jiangauthor=S.+Joshiauthor=J.+R.+Kieferauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=A.+Neissauthor=E.+Pardoauthor=F.+A.+Romeroauthor=P.+Sandyauthor=R.+J.+Simsauthor=Y.+Tangauthor=A.+M.+Taylorauthor=V.+Tsuiauthor=J.+Wangauthor=S.+Wangauthor=Y.+Wangauthor=Z.+Xuauthor=L.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=A.+G.+Cochran&title=GNE-886%3A+A+Potent+and+Selective+Inhibitor+of+the+Cat+Eye+Syndrome+Chromosome+Region+Candidate+2+Bromodomain+%28CECR2%29&doi=10.1021%2Facsmedchemlett.7b00132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">GNE-886: A Potent and Selective Inhibitor of the Cat Eye Syndrome Chromosome Region Candidate 2 Bromodomain (CECR2)</span></div><div class="casAuthors">Crawford, Terry D.; Audia, James E.; Bellon, Steve; Burdick, Daniel J.; Bommi-Reddy, Archana; Cote, Alexandre; Cummings, Richard T.; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael; Huang, Hon-Ren; Jayaram, Hariharan; Jiang, Ying; Joshi, Shivangi; Kiefer, James R.; Murray, Jeremy; Nasveschuk, Christopher G.; Neiss, Arianne; Pardo, Eneida; Romero, F. Anthony; Sandy, Peter; Sims, Robert J.; Tang, Yong; Taylor, Alexander M.; Tsui, Vickie; Wang, Jian; Wang, Shumei; Wang, Yongyun; Xu, Zhaowu; Zawadzke, Laura; Zhu, Xiaoqin; Albrecht, Brian K.; Magnuson, Steven R.; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">737-741</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. function of bromodomains, epigenetic readers of acetylated lysine residues, remains largely unknown.  Herein we report our efforts to discover a potent and selective inhibitor of the bromodomain of cat eye syndrome chromosome region candidate 2 (CECR2).  Screening of our internal medicinal chem. collection led to the identification of a pyrrolopyridone chem. lead, and subsequent structure-based drug design led to a potent and selective CECR2 bromodomain inhibitor (GNE-886) suitable for use as an in vitro tool compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxtzb-gQeFAbVg90H21EOLACvtfcHk0ljrRxsIPexBpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrw%253D&md5=f92e17923136477fc36edefd3bdd4998</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00132%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DBommi-Reddy%26aufirst%3DA.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DGNE-886%253A%2520A%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520the%2520Cat%2520Eye%2520Syndrome%2520Chromosome%2520Region%2520Candidate%25202%2520Bromodomain%2520%2528CECR2%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26issue%3D7%26spage%3D737%26epage%3D741%26doi%3D10.1021%2Facsmedchemlett.7b00132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogány, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">7506</span>– <span class="NLM_lpage">7525</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFeqsL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7506-7525&issue=16&author=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=D.+P.+Dixonauthor=P.+Francisauthor=R.+C.+Furzeauthor=P.+Grandiauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=K.+Lockeauthor=S.+C.+C.+Lucasauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=P.+Pog%C3%A1nyauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=A.+M.+Roaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watson&title=A+Qualified+Success%3A+Discovery+of+a+New+Series+of+ATAD2+Bromodomain+Inhibitors+with+a+Novel+Binding+Mode+Using+High-Throughput+Screening+and+Hit+Qualification&doi=10.1021%2Facs.jmedchem.9b00673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Bridges, Angela M.; Craggs, Peter D.; Dixon, David P.; Francis, Peter; Furze, Rebecca C.; Grandi, Paola; Jones, Emma J.; Karamshi, Bhumika; Locke, Kelly; Lucas, Simon C. C.; Michon, Anne-Marie; Mitchell, Darren J.; Pogany, Peter; Prinjha, Rab K.; Rau, Christina; Roa, Ana Maria; Roberts, Andrew D.; Sheppard, Robert J.; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7506-7525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class.  Here, we describe the discovery of a new class of inhibitors by high-throughput screening and show how the difficulties encountered in establishing a screening triage capable of finding progressible hits were overcome by data-driven optimization.  Despite the prevalence of nonspecific hits and an exceptionally low progressible hit rate (0.001%), our optimized hit qualification strategy employing orthogonal biophys. methods enabled us to identify a single active series.  The compds. have a novel ATAD2 binding mode with noncanonical features including the displacement of all conserved water mols. within the active site and a halogen-bonding interaction.  In addn. to reporting this new series and preliminary structure-activity relationship, we demonstrate the value of diversity screening to complement the knowledge-based approach used in our previous ATAD2 work.  We also exemplify tactics that can increase the chance of success when seeking new chem. starting points for novel and less-tractable targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomDZyktXHrhbVg90H21EOLACvtfcHk0ljrRxsIPexBpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFeqsL%252FJ&md5=8132ccb7f63c3300bd173c42cd2ac026</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00673%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DLucas%26aufirst%3DS.%2BC.%2BC.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPog%25C3%25A1ny%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRoa%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Qualified%2520Success%253A%2520Discovery%2520of%2520a%2520New%2520Series%2520of%2520ATAD2%2520Bromodomain%2520Inhibitors%2520with%2520a%2520Novel%2520Binding%2520Mode%2520Using%2520High-Throughput%2520Screening%2520and%2520Hit%2520Qualification%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D16%26spage%3D7506%26epage%3D7525%26doi%3D10.1021%2Facs.jmedchem.9b00673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting Physical in Drug Discovery: A Contemporary Perspective on Solubility and Hydrophobicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">648</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&issue=15&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+Physical+in+Drug+Discovery%3A+A+Contemporary+Perspective+on+Solubility+and+Hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and ∼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0ljJMCFXHUubqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520Physical%2520in%2520Drug%2520Discovery%253A%2520A%2520Contemporary%2520Perspective%2520on%2520Solubility%2520and%2520Hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26issue%3D15%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting Physical in Drug Discovery II: The Impact of Chromatographic Hydrophobicity Measurements and Aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&issue=17&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+Physical+in+Drug+Discovery+II%3A+The+Impact+of+Chromatographic+Hydrophobicity+Measurements+and+Aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0ljJMCFXHUubqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520Physical%2520in%2520Drug%2520Discovery%2520II%253A%2520The%2520Impact%2520of%2520Chromatographic%2520Hydrophobicity%2520Measurements%2520and%2520Aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26issue%3D17%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palovich, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. J.</span></span> <span> </span><span class="NLM_article-title">Quality Guidelines for Oral Drug Candidates: Dose, Solubility and Lipophilicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1719</span>– <span class="NLM_lpage">1727</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.drudis.2016.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=27423371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1aisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1719-1727&issue=10&author=M.+K.+Baylissauthor=J.+Butlerauthor=P.+L.+Feldmanauthor=D.+V.+S.+Greenauthor=P.+D.+Leesonauthor=M.+R.+Palovichauthor=A.+J.+Taylor&title=Quality+Guidelines+for+Oral+Drug+Candidates%3A+Dose%2C+Solubility+and+Lipophilicity&doi=10.1016%2Fj.drudis.2016.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Quality guidelines for oral drug candidates: dose, solubility and lipophilicity</span></div><div class="casAuthors">Bayliss, Martin K.; Butler, James; Feldman, Paul L.; Green, Darren V. S.; Leeson, Paul D.; Palovich, Michael R.; Taylor, Anthony J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1719-1727</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In an attempt to seek increased understanding of compd. attributes that influence successful drug pipeline progression, GlaxoSmithKline's portfolio of oral candidates was compared with ref. sets of marketed oral drugs.  The approach differs from other attrition studies by explicitly focusing on choosing 'the right compd.' by applying relevant, exptl. derived properties.  The anal. led to four proposed compd. quality categories, created by combining specific criteria for three measures: dose, soly. and the property forecast index, a composite measure of lipophilicity using chromatog. detd. Log D and aromaticity.  The 'three properties' provide benchmarked guidelines for project teams to use when seeking and selecting clin. candidates, because they reflect the property distribution of marketed oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRMpjV3EZ_4rVg90H21EOLACvtfcHk0lgeu5cXMGf1FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1aisb%252FK&md5=d9d145b1ee399b6ad54cdc55fff43c48</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DBayliss%26aufirst%3DM.%2BK.%26aulast%3DButler%26aufirst%3DJ.%26aulast%3DFeldman%26aufirst%3DP.%2BL.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DPalovich%26aufirst%3DM.%2BR.%26aulast%3DTaylor%26aufirst%3DA.%2BJ.%26atitle%3DQuality%2520Guidelines%2520for%2520Oral%2520Drug%2520Candidates%253A%2520Dose%252C%2520Solubility%2520and%2520Lipophilicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26issue%3D10%26spage%3D1719%26epage%3D1727%26doi%3D10.1016%2Fj.drudis.2016.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pease, J. H.</span></span> <span> </span><span class="NLM_article-title">A High Throughput Solubility Assay for Drug Discovery using Microscale Shake-Flask and Rapid UHPLC–UV–CLND Quantification</span>. <i>J. Pharm. Biomed. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1016/j.jpba.2016.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.jpba.2016.01.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=26855285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGhurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2016&pages=126-140&author=B.+Linauthor=J.+H.+Pease&title=A+High+Throughput+Solubility+Assay+for+Drug+Discovery+using+Microscale+Shake-Flask+and+Rapid+UHPLC%E2%80%93UV%E2%80%93CLND+Quantification&doi=10.1016%2Fj.jpba.2016.01.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A high throughput solubility assay for drug discovery using microscale shake-flask and rapid UHPLC-UV-CLND quantification</span></div><div class="casAuthors">Lin, Baiwei; Pease, Joseph H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">126-140</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The rapid detn. of key phys. properties of lead compds. is essential to the drug discovery process.  Soly. is one of the most important properties since good soly. is needed not only for obtaining reliable in vitro and in vivo assay results in early discovery but also to ensure sufficient concn. of the drug being in circulation to get the desired therapeutic exposure at the target of interest.  In order for medicinal chemists to tune soly. of lead compds., a rapid assay is needed to provide soly. data that is accurate and predictive so that it can be reliably used for designing the next generation of compds. with improved properties.  To ensure speed and data quality, we developed a high throughput soly. assay that utilizes a single calibration UHPLC-UV-CLND method and a 24 h shake-flask format for rapid quantification.  A set of 46 model compds. was used to demonstrate that the method is accurate, reproducible and predictive.  Here we present development of the assay, including evaluation of quantification method, filtration membranes, equil. times, DMSO concns., and buffer conditions.  A comparison of thermodn. soly. results to our high throughput 24 h shake-flask soly. assay results is also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpojPpYm9G6NrVg90H21EOLACvtfcHk0lgeu5cXMGf1FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGhurg%253D&md5=98415728ed8254d173301d5f2c85e3f8</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.jpba.2016.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpba.2016.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DB.%26aulast%3DPease%26aufirst%3DJ.%2BH.%26atitle%3DA%2520High%2520Throughput%2520Solubility%2520Assay%2520for%2520Drug%2520Discovery%2520using%2520Microscale%2520Shake-Flask%2520and%2520Rapid%2520UHPLC%25E2%2580%2593UV%25E2%2580%2593CLND%2520Quantification%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2016%26volume%3D122%26spage%3D126%26epage%3D140%26doi%3D10.1016%2Fj.jpba.2016.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kestranek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenberger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Placko, S.</span></span> <span> </span><span class="NLM_article-title">Chemiluminescent Nitrogen Detection (CLND) to Measure Kinetic Aqueous Solubility</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch130145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2F9780470559277.ch130145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=24391097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A280%3ADC%252BC2czitF2itw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=269-280&author=A.+Kestranekauthor=A.+Chervenakauthor=J.+Longenbergerauthor=S.+Placko&title=Chemiluminescent+Nitrogen+Detection+%28CLND%29+to+Measure+Kinetic+Aqueous+Solubility&doi=10.1002%2F9780470559277.ch130145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chemiluminescent nitrogen detection (CLND) to measure kinetic aqueous solubility</span></div><div class="casAuthors">Kestranek Aimee; Chervenak Andrew; Longenberger Justin; Placko Steven</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in chemical biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">269-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Solubility is the dose-limiting property for in vitro studies, and therefore is a critical physicochemical property to measure in drug discovery.  Solubility data can be used to guide lead optimization, troubleshoot erratic bioassay results, and identify potential downstream liabilities such as insufficient solubility for bioassays or oral bioavailability.  Typically, early in vitro studies are performed using library compounds prepared as dimethylsulfoxide (DMSO) stock solutions, resulting in in vitro test solutions containing DMSO at low concentration (<5% v/v).  Since DMSO can affect the apparent solubility, it is desirable to obtain solubility data under conditions mimicking the in vitro study.  Kinetic solubility (from DMSO stock solutions) is often preferred over thermodynamic solubility (from dry powder) in early drug discovery.  The protocols in this article describe a general procedure for assessing kinetic aqueous solubility of early drug discovery compounds using a miniaturized shake flask method with chemiluminescent nitrogen detection (CLND).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQa5zXGR-PJU_SAs_PE1QvifW6udTcc2eZulft2DyGdqbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2czitF2itw%253D%253D&md5=d63d6d57c4ff0934379d3c37f82eb1a0</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch130145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch130145%26sid%3Dliteratum%253Aachs%26aulast%3DKestranek%26aufirst%3DA.%26aulast%3DChervenak%26aufirst%3DA.%26aulast%3DLongenberger%26aufirst%3DJ.%26aulast%3DPlacko%26aufirst%3DS.%26atitle%3DChemiluminescent%2520Nitrogen%2520Detection%2520%2528CLND%2529%2520to%2520Measure%2520Kinetic%2520Aqueous%2520Solubility%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D5%26spage%3D269%26epage%3D280%26doi%3D10.1002%2F9780470559277.ch130145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span> <span> </span><span class="NLM_article-title">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&issue=7&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+High+Throughput+Screening%3A+Parallel+Artificial+Membrane+Permeation+Assay+in+the+Description+of+Passive+Absorption+Processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0lgrr1r8Cj432w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520High%2520Throughput%2520Screening%253A%2520Parallel%2520Artificial%2520Membrane%2520Permeation%2520Assay%2520in%2520the%2520Description%2520of%2520Passive%2520Absorption%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26issue%3D7%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollerton, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besley, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boughtflower, B. J.</span></span> <span> </span><span class="NLM_article-title">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1772</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.6b04060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b04060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2017&pages=1772-1777&issue=3&author=M.+W.+Robinsonauthor=A.+P.+Hillauthor=S.+A.+Readshawauthor=J.+C.+Hollertonauthor=R.+J.+Uptonauthor=S.+M.+Lynnauthor=S.+C.+Besleyauthor=B.+J.+Boughtflower&title=Use+of+Calculated+Physicochemical+Properties+to+Enhance+Quantitative+Response+When+Using+Charged+Aerosol+Detection&doi=10.1021%2Facs.analchem.6b04060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span></div><div class="casAuthors">Robinson, Max W.; Hill, Alan P.; Readshaw, Simon A.; Hollerton, John C.; Upton, Richard J.; Lynn, Sean M.; Besley, Steve C.; Boughtflower, Bob J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1772-1777</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Universal quant. detection without the need for analyte ref. stds. would offer substantial benefits in many areas of anal. science.  The quant. capability of high performance liq. chromatog. (HPLC) with charged aerosol detection (CAD) was investigated for 50 compds. with a wide range of phys. and chem. properties.  It is widely believed that CAD is a mass detector.  Quantification of the 50 compds. using a generic calibrant and mass calibration achieved an av. error of 11.4% relative to 1H NMR.  Correction factors are proposed that est. the relative surface area of particles in the detector, taking into account the effects of the d. and charge of analytes.  Performing these corrections and quantifying with surface area calibration, rather than mass, shows considerably improved linearity and uniformity of detection, reducing the av. error relative to 1H NMR to 7.1%.  The accuracy of CAD quantification was most significantly improved for highly dense compds., with traditional mass calibration showing an av. error of 34.7% and the newly proposed surface area calibration showing an av. error of 5.8%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxWCD0M2yQRLVg90H21EOLACvtfcHk0lhXPGSXEgUM4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK&md5=46c11b7527157b76f1fbc8eaf5cef75a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b04060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b04060%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DM.%2BW.%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DReadshaw%26aufirst%3DS.%2BA.%26aulast%3DHollerton%26aufirst%3DJ.%2BC.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26aulast%3DLynn%26aufirst%3DS.%2BM.%26aulast%3DBesley%26aufirst%3DS.%2BC.%26aulast%3DBoughtflower%26aufirst%3DB.%2BJ.%26atitle%3DUse%2520of%2520Calculated%2520Physicochemical%2520Properties%2520to%2520Enhance%2520Quantitative%2520Response%2520When%2520Using%2520Charged%2520Aerosol%2520Detection%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26issue%3D3%26spage%3D1772%26epage%3D1777%26doi%3D10.1021%2Facs.analchem.6b04060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Myrianthopoulos, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaboriaud-Kolar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grigoriou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownlee, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanior, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mexia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagratuni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terpos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaltsounis, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downs, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikros, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of a Selective Ligand for the Switch/Sucrose Nonfermenting-Related Bromodomains of Polybromo Protein-1 by the Use of Virtual Screening and Hydration Analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8787</span>– <span class="NLM_lpage">8803</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8787-8803&issue=19&author=V.+Myrianthopoulosauthor=N.+Gaboriaud-Kolarauthor=C.+Tallantauthor=M.-L.+Hallauthor=S.+Grigoriouauthor=P.+M.+Brownleeauthor=O.+Fedorovauthor=C.+Rogersauthor=D.+Heidenreichauthor=M.+Waniorauthor=N.+Drososauthor=N.+Mexiaauthor=P.+Savitskyauthor=T.+Bagratuniauthor=E.+Kastritisauthor=E.+Terposauthor=P.+Filippakopoulosauthor=S.+M%C3%BCllerauthor=A.-L.+Skaltsounisauthor=J.+A.+Downsauthor=S.+Knappauthor=E.+Mikros&title=Discovery+and+Optimization+of+a+Selective+Ligand+for+the+Switch%2FSucrose+Nonfermenting-Related+Bromodomains+of+Polybromo+Protein-1+by+the+Use+of+Virtual+Screening+and+Hydration+Analysis&doi=10.1021%2Facs.jmedchem.6b00355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of a Selective Ligand for the Switch/Sucrose Nonfermenting-Related Bromodomains of Polybromo Protein-1 by the Use of Virtual Screening and Hydration Analysis</span></div><div class="casAuthors">Myrianthopoulos, Vassilios; Gaboriaud-Kolar, Nicolas; Tallant, Cynthia; Hall, Michelle-Lynn; Grigoriou, Stylianos; Brownlee, Peter Moore; Fedorov, Oleg; Rogers, Catherine; Heidenreich, David; Wanior, Marek; Drosos, Nikolaos; Mexia, Nikitia; Savitsky, Pavel; Bagratuni, Tina; Kastritis, Efstathios; Terpos, Evangelos; Filippakopoulos, Panagis; Muller, Susanne; Skaltsounis, Alexios-Leandros; Downs, Jessica Ann; Knapp, Stefan; Mikros, Emmanuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8787-8803</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic interaction domains currently recognized as emerging drug targets for development of anticancer or anti-inflammatory agents.  In this study, development of a selective ligand of the fifth BRD of Polybromo protein-1 (PB1(5)) related to Switch/Sucrose Non-Fermenting (SWI/SNF) chromatin remodeling complexes is presented.  A compd. collection was evaluated by consensus virtual screening and a hit was identified.  The biophys. study of protein-ligand interactions was performed using x-ray crystallog. and isothermal titrn. calorimetry.  Collective data supported the hypothesis that affinity improvement could be achieved by enhancing interactions of the complex with the solvent.  The derived SAR along with free energy calcns. and a consensus hydration anal. using WaterMap and SZmap algorithms guided rational design of a set of novel analogs.  The most potent analog demonstrated high affinity of 3.3 μM and an excellent selectivity profile, thus comprising a promising lead for the development of chem. probes targeting PB1(5).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrMsChShl81rVg90H21EOLACvtfcHk0lhXPGSXEgUM4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtLrL&md5=026dcc048ad31b7bf37c9a41b7562018</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00355%26sid%3Dliteratum%253Aachs%26aulast%3DMyrianthopoulos%26aufirst%3DV.%26aulast%3DGaboriaud-Kolar%26aufirst%3DN.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DHall%26aufirst%3DM.-L.%26aulast%3DGrigoriou%26aufirst%3DS.%26aulast%3DBrownlee%26aufirst%3DP.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DWanior%26aufirst%3DM.%26aulast%3DDrosos%26aufirst%3DN.%26aulast%3DMexia%26aufirst%3DN.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DBagratuni%26aufirst%3DT.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DTerpos%26aufirst%3DE.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSkaltsounis%26aufirst%3DA.-L.%26aulast%3DDowns%26aufirst%3DJ.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMikros%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520a%2520Selective%2520Ligand%2520for%2520the%2520Switch%252FSucrose%2520Nonfermenting-Related%2520Bromodomains%2520of%2520Polybromo%2520Protein-1%2520by%2520the%2520Use%2520of%2520Virtual%2520Screening%2520and%2520Hydration%2520Analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D19%26spage%3D8787%26epage%3D8803%26doi%3D10.1021%2Facs.jmedchem.6b00355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherell, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siejka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span> <span> </span><span class="NLM_article-title">Identification and Development of 2,3-Dihydropyrrolo[1,2-a]quinazolin-5(1H)-one Inhibitors Targeting Bromodomains within the Switch/Sucrose Nonfermenting Complex</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">5095</span>– <span class="NLM_lpage">5101</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5095-5101&issue=10&author=C.+L.+Sutherellauthor=C.+Tallantauthor=O.+P.+Monteiroauthor=C.+Yappauthor=J.+E.+Fuchsauthor=O.+Fedorovauthor=P.+Siejkaauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=J.+D.+Brentonauthor=P.+E.+Brennanauthor=S.+V.+Ley&title=Identification+and+Development+of+2%2C3-Dihydropyrrolo%5B1%2C2-a%5Dquinazolin-5%281H%29-one+Inhibitors+Targeting+Bromodomains+within+the+Switch%2FSucrose+Nonfermenting+Complex&doi=10.1021%2Facs.jmedchem.5b01997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Development of 2,3-Dihydropyrrolo[1,2-a]quinazolin-5(1H)-one Inhibitors Targeting Bromodomains within the Switch/Sucrose Nonfermenting Complex</span></div><div class="casAuthors">Sutherell, Charlotte L.; Tallant, Cynthia; Monteiro, Octovia P.; Yapp, Clarence; Fuchs, Julian E.; Fedorov, Oleg; Siejka, Paulina; Muller, Suzanne; Knapp, Stefan; Brenton, James D.; Brennan, Paul E.; Ley, Steven V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5095-5101</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain contg. proteins PB1, SMARCA4, and SMARCA2 are important components of SWI/SNF chromatin remodeling complexes.  We identified bromodomain inhibitors that target these proteins and display unusual binding modes involving water displacement from the KAc binding site.  The best compd. (I) binds the fifth bromodomain of PB1 with a KD of 124 nM, SMARCA2B and SMARCA4 with KD values of 262 and 417 nM, resp., and displays excellent selectivity over bromodomains other than PB1, SMARCA2, and SMARCA4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfnWlpl3TimLVg90H21EOLACvtfcHk0lglfFF8gI3Jzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D&md5=a27afa2c97bd74462a09c9ec231f61ef</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01997%26sid%3Dliteratum%253Aachs%26aulast%3DSutherell%26aufirst%3DC.%2BL.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSiejka%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DLey%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520and%2520Development%2520of%25202%252C3-Dihydropyrrolo%255B1%252C2-a%255Dquinazolin-5%25281H%2529-one%2520Inhibitors%2520Targeting%2520Bromodomains%2520within%2520the%2520Switch%252FSucrose%2520Nonfermenting%2520Complex%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D10%26spage%3D5095%26epage%3D5101%26doi%3D10.1021%2Facs.jmedchem.5b01997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castex, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Selective Targeting of the BRG/PB1 Bromodomains Impairs Embryonic and Trophoblast Stem Cell Maintenance</span>. <i>Science Advances</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">e1500723</span>, <span class="refDoi"> DOI: 10.1126/sciadv.1500723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1126%2Fsciadv.1500723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=26702435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=e1500723&issue=10&author=O.+Fedorovauthor=J.+Castexauthor=C.+Tallantauthor=D.+R.+Owenauthor=S.+Martinauthor=M.+Aldeghiauthor=O.+Monteiroauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=J.+D.+Trzupekauthor=B.+S.+Gerstenbergerauthor=C.+Bountraauthor=D.+Willmannauthor=C.+Wellsauthor=M.+Philpottauthor=C.+Rogersauthor=P.+C.+Bigginauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=R.+Sch%C3%BCleauthor=T.+G%C3%BCntherauthor=S.+Knappauthor=S.+M%C3%BCller&title=Selective+Targeting+of+the+BRG%2FPB1+Bromodomains+Impairs+Embryonic+and+Trophoblast+Stem+Cell+Maintenance&doi=10.1126%2Fsciadv.1500723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance</span></div><div class="casAuthors">Fedorov, Oleg; Castex, Josefina; Tallant, Cynthia; Owen, Dafydd R.; Martin, Sarah; Aldeghi, Matteo; Monteiro, Octovia; Filippakopoulos, Panagis; Picaud, Sarah; Trzupek, John D.; Gerstenberger, Brian S.; Bountra, Chas; Willmann, Dominica; Wells, Christopher; Philpott, Martin; Rogers, Catherine; Biggin, Philip C.; Brennan, Paul E.; Bunnage, Mark E.; Schule, Roland; Gunther, Thomas; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1500723/1-e1500723/11</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF [also called Brg/Brahma-assocd. factors (BAFs)] are evolutionarily conserved chromatin remodeling complexes regulating gene transcription programs during development and stem cell differentiation.  BAF complexes contain an ATP (ATP)-driven remodeling enzyme (either BRG1 or BRM) and multiple protein interaction domains including bromodomains, an evolutionary conserved acetyl lysine-dependent protein interaction motif that recruits transcriptional regulators to acetylated chromatin.  We report a potent and cell active protein interaction inhibitor, PFI-3, that selectively binds to essential BAF bromodomains.  The high specificity of PFI-3 was achieved on the basis of a novel binding mode of a salicylic acid head group that led to the replacement of water mols. typically maintained in other bromodomain inhibitor complexes.  We show that exposure of embryonic stem cells to PFI-3 led to deprivation of stemness and deregulated lineage specification.  Furthermore, differentiation of trophoblast stem cells in the presence of PFI-3 was markedly enhanced.  The data present a key function of BAF bromodomains in stem cell maintenance and differentiation, introducing a novel versatile chem. probe for studies on acetylation-dependent cellular processes controlled by BAF remodeling complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiGyzl98dGOLVg90H21EOLACvtfcHk0ljl0KfB9pehHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D&md5=52757e667cc5afeb4c3a151fc8ef3442</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500723%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DCastex%26aufirst%3DJ.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DSelective%2520Targeting%2520of%2520the%2520BRG%252FPB1%2520Bromodomains%2520Impairs%2520Embryonic%2520and%2520Trophoblast%2520Stem%2520Cell%2520Maintenance%26jtitle%3DScience%2520Advances%26date%3D2015%26volume%3D1%26issue%3D10%26spage%3De1500723%26doi%3D10.1126%2Fsciadv.1500723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Analysis of Water Stability in Bromodomain Binding Pockets with Grand Canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=29863194" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=19&issue=1&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-Scale+Analysis+of+Water+Stability+in+Bromodomain+Binding+Pockets+with+Grand+Canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-Scale%2520Analysis%2520of%2520Water%2520Stability%2520in%2520Bromodomain%2520Binding%2520Pockets%2520with%2520Grand%2520Canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26issue%3D1%26spage%3D19%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&issue=1&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0liC2_7VzMqKyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26issue%3D1%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET—A Novel BRET Platform for the Analysis of Protein–Protein Interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&issue=8&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+M%C3%A9ndezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET%E2%80%94A+Novel+BRET+Platform+for+the+Analysis+of+Protein%E2%80%93Protein+Interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">NanoBRET-A Novel BRET Platform for the Analysis of Protein-Protein Interactions</span></div><div class="casAuthors">Machleidt, Thomas; Woodroofe, Carolyn C.; Schwinn, Marie K.; Mendez, Jacqui; Robers, Matthew B.; Zimmerman, Kris; Otto, Paul; Daniels, Danette L.; Kirkland, Thomas A.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1797-1804</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dynamic interactions between proteins comprise a key mechanism for temporal control of cellular function and thus hold promise for development of novel drug therapies.  It remains tech. challenging, however, to quant. characterize these interactions within the biol. relevant context of living cells.  Although, bioluminescence resonance energy transfer (BRET) has often been used for this purpose, its general applicability has been hindered by limited sensitivity and dynamic range.  We have addressed this by combining an extremely bright luciferase (Nanoluc) with a means for tagging intracellular proteins with a long-wavelength fluorophore (HaloTag).  The small size (19 kDa), high emission intensity, and relatively narrow spectrum (460 nm peak intensity) make Nanoluc luciferase well suited as an energy donor.  By selecting an efficient red-emitting fluorophore (635 nm peak intensity) for attachment onto the HaloTag, an overall spectral sepn. exceeding 175 nm was achieved.  This combination of greater light intensity with improved spectral resoln. results in substantially increased detection sensitivity and dynamic range over current BRET technologies.  Enhanced performance is demonstrated using several established model systems, as well as the ability to image BRET in individual cells.  The capabilities are further exhibited in a novel assay developed for analyzing the interactions of bromodomain proteins with chromatin in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-h9HlReNabVg90H21EOLACvtfcHk0lgL371r0C5oeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D&md5=9f5f773b6cd14c40074a2ffbd6472008</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET%25E2%2580%2594A%2520Novel%2520BRET%2520Platform%2520for%2520the%2520Analysis%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of Chemical Inhibitors of Human Bromodomains</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">11625</span>– <span class="NLM_lpage">11668</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00205</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00205" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=11625-11668&issue=21&author=G.+Zhangauthor=S.+G.+Smithauthor=M.+Zhou&title=Discovery+of+Chemical+Inhibitors+of+Human+Bromodomains&doi=10.1021%2Facs.chemrev.5b00205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Chemical Inhibitors of Human Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11625-11668</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-guided drug design strategy has played an important role in the recent discovery and development of bromodomain inhibitors that has yielded a multitude of chemotypes as described in this review.  Going forward, we expect that this structure-based approach will continue to guide medicinal chemists in lead optimization of bromodomain inhibitors and facilitate the efforts to develop new inhibitors for less druggable bromodomains.  Collectively, these emerging new chem. inhibitors will empower our ongoing investigation to better understand the roles of bromodomain proteins in control of gene transcription in human biol. and disease pathol.  These small-mol. inhibitors can also be developed into new epigenetic therapies for new effective treatments of various human diseases including cancer and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHbsgfBguOabVg90H21EOLACvtfcHk0lgL371r0C5oeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP&md5=4014c771384b4a768a80644af22c4485</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00205%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520Chemical%2520Inhibitors%2520of%2520Human%2520Bromodomains%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26issue%3D21%26spage%3D11625%26epage%3D11668%26doi%3D10.1021%2Facs.chemrev.5b00205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mispelaere-Canivet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spindler, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beslin, P.</span></span> <span> </span><span class="NLM_article-title">Pd2(dba)3/Xantphos-Catalyzed Cross-Coupling of Thiols and Aryl Bromides/Triflates</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">5253</span>– <span class="NLM_lpage">5259</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2005.03.078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.tet.2005.03.078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFWgs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=5253-5259&issue=22&author=C.+Mispelaere-Canivetauthor=J.-F.+Spindlerauthor=S.+Perrioauthor=P.+Beslin&title=Pd2%28dba%293%2FXantphos-Catalyzed+Cross-Coupling+of+Thiols+and+Aryl+Bromides%2FTriflates&doi=10.1016%2Fj.tet.2005.03.078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Pd2(dba)3/Xantphos-catalyzed cross-coupling of thiols and aryl bromides/triflates</span></div><div class="casAuthors">Mispelaere-Canivet, Clotilde; Spindler, Jean-Francis; Perrio, Stephane; Beslin, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5253-5259</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The cross-coupling of aliph. and arom. thiols and aryl halides or triflates mediated by a Pd2(dba)3/Xantphos catalytic system in refluxing xylene afforded the corresponding aryl thioethers in good to excellent yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfbcJkhwf7l7Vg90H21EOLACvtfcHk0lgqxwdX6BFZuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFWgs7k%253D&md5=54027b2333f6deb1df375bd85b36ea71</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2005.03.078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2005.03.078%26sid%3Dliteratum%253Aachs%26aulast%3DMispelaere-Canivet%26aufirst%3DC.%26aulast%3DSpindler%26aufirst%3DJ.-F.%26aulast%3DPerrio%26aufirst%3DS.%26aulast%3DBeslin%26aufirst%3DP.%26atitle%3DPd2%2528dba%25293%252FXantphos-Catalyzed%2520Cross-Coupling%2520of%2520Thiols%2520and%2520Aryl%2520Bromides%252FTriflates%26jtitle%3DTetrahedron%26date%3D2005%26volume%3D61%26issue%3D22%26spage%3D5253%26epage%3D5259%26doi%3D10.1016%2Fj.tet.2005.03.078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pu, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christesen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ku, Y.-Y.</span></span> <span> </span><span class="NLM_article-title">A Simple and Highly Effective Oxidative Chlorination Protocol for the Preparation of Arenesulfonyl Chlorides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">418</span>– <span class="NLM_lpage">421</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2009.11.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.tetlet.2009.11.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=418-421&issue=2&author=Y.-M.+Puauthor=A.+Christesenauthor=Y.-Y.+Ku&title=A+Simple+and+Highly+Effective+Oxidative+Chlorination+Protocol+for+the+Preparation+of+Arenesulfonyl+Chlorides&doi=10.1016%2Fj.tetlet.2009.11.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A simple and highly effective oxidative chlorination protocol for the preparation of arenesulfonyl chlorides</span></div><div class="casAuthors">Pu, Yu-Ming; Christesen, Alan; Ku, Yi-Yin</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">418-421</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">2,4-Dichloro-5,5-dimethylhydantoin (DCDMH) was found to be a mild and efficient reagent for the direct oxidative conversion of sulfur compds. to the corresponding arenesulfonyl chlorides in good to excellent yields through oxidative chlorination.  The method is suitable for many types of sulfur substrates (thiols, disulfides, and benzylic sulfides).  The overall process is simple, practical, and it provides convenient access to a variety of aryl or heteroarylsulfonyl chlorides.  The mild reaction conditions and the broad substrate scope render this method attractive and complementary to existing syntheses of aryl or heteroarylsulfonyl chlorides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAa01G0iIlebVg90H21EOLACvtfcHk0lgqxwdX6BFZuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsrvO&md5=99e2389d65e31bc57a2a3c740b59df98</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2009.11.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2009.11.047%26sid%3Dliteratum%253Aachs%26aulast%3DPu%26aufirst%3DY.-M.%26aulast%3DChristesen%26aufirst%3DA.%26aulast%3DKu%26aufirst%3DY.-Y.%26atitle%3DA%2520Simple%2520and%2520Highly%2520Effective%2520Oxidative%2520Chlorination%2520Protocol%2520for%2520the%2520Preparation%2520of%2520Arenesulfonyl%2520Chlorides%26jtitle%3DTetrahedron%2520Lett.%26date%3D2010%26volume%3D51%26issue%3D2%26spage%3D418%26epage%3D421%26doi%3D10.1016%2Fj.tetlet.2009.11.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. S.</span></span> <span> </span><span class="NLM_article-title">A New Solvent System for Efficient Synthesis of 1,2,3-Triazoles</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">29</span>),  <span class="NLM_fpage">5105</span>– <span class="NLM_lpage">5109</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2006.05.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.tetlet.2006.05.079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD28Xmt1Ghtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2006&pages=5105-5109&issue=29&author=B.-Y.+Leeauthor=S.+R.+Parkauthor=H.+B.+Jeonauthor=K.+S.+Kim&title=A+New+Solvent+System+for+Efficient+Synthesis+of+1%2C2%2C3-Triazoles&doi=10.1016%2Fj.tetlet.2006.05.079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">A new solvent system for efficient synthesis of 1,2,3-triazoles</span></div><div class="casAuthors">Lee, Bo-Young; Park, So Ra; Jeon, Heung Bae; Kim, Kwan Soo</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">5105-5109</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The use of CH2Cl2 as a co-solvent with H2O in the copper(I)-catalyzed 1,3-dipolar cycloaddn. of org. azides and alkynes increased reaction rates and provided the corresponding 1,2,3-triazoles in excellent yields compared to other org. co-solvent systems.  Moreover, we applied this discovery to prep. the functional multivalent compd. in an excellent yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8pPjYb6ZERrVg90H21EOLACvtfcHk0lgqxwdX6BFZuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xmt1Ghtb0%253D&md5=8f452666e79d95c509209a1557910710</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2006.05.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2006.05.079%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DB.-Y.%26aulast%3DPark%26aufirst%3DS.%2BR.%26aulast%3DJeon%26aufirst%3DH.%2BB.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DA%2520New%2520Solvent%2520System%2520for%2520Efficient%2520Synthesis%2520of%25201%252C2%252C3-Triazoles%26jtitle%3DTetrahedron%2520Lett.%26date%3D2006%26volume%3D47%26issue%3D29%26spage%3D5105%26epage%3D5109%26doi%3D10.1016%2Fj.tetlet.2006.05.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boren, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokin, V. V.</span></span> <span> </span><span class="NLM_article-title">Ruthenium-Catalyzed Azide–Alkyne Cycloaddition: Scope and Mechanism</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span> (<span class="NLM_issue">28</span>),  <span class="NLM_fpage">8923</span>– <span class="NLM_lpage">8930</span>, <span class="refDoi"> DOI: 10.1021/ja0749993</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0749993" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1Wjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=8923-8930&issue=28&author=B.+C.+Borenauthor=S.+Narayanauthor=L.+K.+Rasmussenauthor=L.+Zhangauthor=H.+Zhaoauthor=Z.+Linauthor=G.+Jiaauthor=V.+V.+Fokin&title=Ruthenium-Catalyzed+Azide%E2%80%93Alkyne+Cycloaddition%3A+Scope+and+Mechanism&doi=10.1021%2Fja0749993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Ruthenium-Catalyzed Azide-Alkyne Cycloaddition: Scope and Mechanism</span></div><div class="casAuthors">Boren, Brant C.; Narayan, Sridhar; Rasmussen, Lars K.; Zhang, Li; Zhao, Haitao; Lin, Zhenyang; Jia, Guochen; Fokin, Valery V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">8923-8930</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic activity of ruthenium(II) complexes in azide-alkyne cycloaddns. were evaluated.  The [Cp*RuCl] complexes, such as Cp*RuCl(PPh3)2, Cp*RuCl(COD), and Cp*RuCl(NBD), were among the most effective catalysts.  In the presence of catalytic Cp*RuCl(PPh3)2 or Cp*RuCl(COD), primary and secondary azides react with a broad range of terminal alkynes contg. a range of functionalities selectively producing 1,5-disubstituted 1,2,3-triazoles; tertiary azides were significantly less reactive.  Both complexes also promote the cycloaddn. reactions of org. azides with internal alkynes, providing access to fully-substituted 1,2,3-triazoles.  The ruthenium-catalyzed azide-alkyne cycloaddn. (RuAAC) appears to proceed via oxidative coupling of the azide and alkyne reactants to give a six-membered ruthenacycle intermediate, in which the first new carbon-nitrogen bond is formed between the more electroneg. carbon of the alkyne and the terminal, electrophilic nitrogen of the azide.  This step is followed by reductive elimination, which forms the triazole product.  DFT calcns. support this mechanistic proposal and indicate that the reductive elimination step is rate-detg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_rfIeBuSgUbVg90H21EOLACvtfcHk0lhFzIj91SkaIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1Wjtr8%253D&md5=8b5ba21060ad9d6984bba3e2587aa44d</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja0749993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0749993%26sid%3Dliteratum%253Aachs%26aulast%3DBoren%26aufirst%3DB.%2BC.%26aulast%3DNarayan%26aufirst%3DS.%26aulast%3DRasmussen%26aufirst%3DL.%2BK.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DJia%26aufirst%3DG.%26aulast%3DFokin%26aufirst%3DV.%2BV.%26atitle%3DRuthenium-Catalyzed%2520Azide%25E2%2580%2593Alkyne%2520Cycloaddition%253A%2520Scope%2520and%2520Mechanism%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26issue%3D28%26spage%3D8923%26epage%3D8930%26doi%3D10.1021%2Fja0749993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alekseev, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurkin, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yurovskaya, M. A.</span></span> <span> </span><span class="NLM_article-title">Use of the Graebe-Ullmann Reaction in the Synthesis of 8-Methyl-γ-carboline and Isomeric Aromatic Aza-γ-carbolines</span>. <i>Chem. Heterocycl. Compd.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1235</span>– <span class="NLM_lpage">1250</span>, <span class="refDoi"> DOI: 10.1007/s10593-012-1127-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1007%2Fs10593-012-1127-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslWqs7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=1235-1250&issue=8&author=R.+S.+Alekseevauthor=A.+V.+Kurkinauthor=M.+A.+Yurovskaya&title=Use+of+the+Graebe-Ullmann+Reaction+in+the+Synthesis+of+8-Methyl-%CE%B3-carboline+and+Isomeric+Aromatic+Aza-%CE%B3-carbolines&doi=10.1007%2Fs10593-012-1127-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Use of the Graebe-Ullmann reaction in the synthesis of 8-methyl-γ-carboline and isomeric aromatic aza-γ-carbolines</span></div><div class="casAuthors">Alekseev, R. S.; Kurkin, A. V.; Yurovskaya, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Heterocyclic Compounds (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1235-1250</span>CODEN:
                <span class="NLM_cas:coden">CHCCAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-3122</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Two variants of the Graebe-Ullmann reaction were used to obtain 8-methyl-5H-pyrido[4,3-b]indole (8-methyl-γ-carboline) and the conditions for this reaction were optimized.  The feasibility of this method was studied for the synthesis of a series of isomeric arom. aza-γ-carbolines from the corresponding 1-(pyridyl)-1H-1,2,3-triazolo[4,5-c]pyridines under thermal and microwave irradn. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUZ7wIiDj_37Vg90H21EOLACvtfcHk0lhFzIj91SkaIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslWqs7vF&md5=f9d69d5c1d822fc801f2c60607d4e0f6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs10593-012-1127-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10593-012-1127-7%26sid%3Dliteratum%253Aachs%26aulast%3DAlekseev%26aufirst%3DR.%2BS.%26aulast%3DKurkin%26aufirst%3DA.%2BV.%26aulast%3DYurovskaya%26aufirst%3DM.%2BA.%26atitle%3DUse%2520of%2520the%2520Graebe-Ullmann%2520Reaction%2520in%2520the%2520Synthesis%2520of%25208-Methyl-%25CE%25B3-carboline%2520and%2520Isomeric%2520Aromatic%2520Aza-%25CE%25B3-carbolines%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2012%26volume%3D48%26issue%3D8%26spage%3D1235%26epage%3D1250%26doi%3D10.1007%2Fs10593-012-1127-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yutilov, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolyar, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramyants, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izotova, N. P.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Ticlopidine Analogs Based on Spinaceamine and 2-Azaspinaceamine</span>. <i>Pharm. Chem. J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">243</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1023/A:1025382623339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1023%2FA%3A1025382623339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVGnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2003&pages=243-245&issue=5&author=Y.+M.+Yutilovauthor=N.+N.+Smolyarauthor=M.+G.+Abramyantsauthor=N.+P.+Izotova&title=Synthesis+of+Ticlopidine+Analogs+Based+on+Spinaceamine+and+2-Azaspinaceamine&doi=10.1023%2FA%3A1025382623339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Ticlopidine Analogs Based on Spinaceamine and 2-Azaspinaceamine</span></div><div class="casAuthors">Yutilov, Yu. M.; Smolyar, N. N.; Abramyants, M. G.; Izotova, N. P.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Chemistry Journal (Translation of Khimiko-Farmatsevticheskii Zhurnal)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">243-245</span>CODEN:
                <span class="NLM_cas:coden">PCJOAU</span>;
        ISSN:<span class="NLM_cas:issn">0091-150X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Consultants Bureau</span>)
        </div><div class="casAbstract">A series of ticlopidine analogs contg. imidazole or triazoles instead if the thiophene nucleus were prepd. from imidazopyridines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohmi6jGWLb5rVg90H21EOLACvtfcHk0liUYj8s9NxXOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVGnsr8%253D&md5=ff3a8e2120199b6f84751e2e65f3a5c8</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1023%2FA%3A1025382623339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1025382623339%26sid%3Dliteratum%253Aachs%26aulast%3DYutilov%26aufirst%3DY.%2BM.%26aulast%3DSmolyar%26aufirst%3DN.%2BN.%26aulast%3DAbramyants%26aufirst%3DM.%2BG.%26aulast%3DIzotova%26aufirst%3DN.%2BP.%26atitle%3DSynthesis%2520of%2520Ticlopidine%2520Analogs%2520Based%2520on%2520Spinaceamine%2520and%25202-Azaspinaceamine%26jtitle%3DPharm.%2520Chem.%2520J.%26date%3D2003%26volume%3D37%26issue%3D5%26spage%3D243%26epage%3D245%26doi%3D10.1023%2FA%3A1025382623339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lay, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carver, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenhough, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatfield, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brierley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span> <span> </span><span class="NLM_article-title">Cellular Target Engagement Approaches to Monitor Epigenetic Reader Domain Interactions</span>. <i>SLAS DISCOVERY: Advancing the Science of Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1177/2472555219896278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1177%2F2472555219896278" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=163-175&issue=2&author=A.+N.+Phillipouauthor=C.+S.+Layauthor=C.+E.+Carverauthor=C.+Messengerauthor=J.+P.+Evansauthor=A.+J.+Lewisauthor=L.+J.+Gordonauthor=M.+Mahmoodauthor=L.+A.+Greenhoughauthor=D.+Sammonauthor=A.+T.+Chengauthor=S.+Chakrabortyauthor=E.+J.+Jonesauthor=S.+C.+C.+Lucasauthor=K.+M.+Gatfieldauthor=D.+J.+Brierleyauthor=P.+D.+Craggs&title=Cellular+Target+Engagement+Approaches+to+Monitor+Epigenetic+Reader+Domain+Interactions&doi=10.1177%2F2472555219896278"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1177%2F2472555219896278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555219896278%26sid%3Dliteratum%253Aachs%26aulast%3DPhillipou%26aufirst%3DA.%2BN.%26aulast%3DLay%26aufirst%3DC.%2BS.%26aulast%3DCarver%26aufirst%3DC.%2BE.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DMahmood%26aufirst%3DM.%26aulast%3DGreenhough%26aufirst%3DL.%2BA.%26aulast%3DSammon%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DA.%2BT.%26aulast%3DChakraborty%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLucas%26aufirst%3DS.%2BC.%2BC.%26aulast%3DGatfield%26aufirst%3DK.%2BM.%26aulast%3DBrierley%26aufirst%3DD.%2BJ.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26atitle%3DCellular%2520Target%2520Engagement%2520Approaches%2520to%2520Monitor%2520Epigenetic%2520Reader%2520Domain%2520Interactions%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26date%3D2020%26volume%3D25%26issue%3D2%26spage%3D163%26epage%3D175%26doi%3D10.1177%2F2472555219896278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the BET Family Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3838</span>, <span class="refDoi"> DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&issue=11&author=C.-w.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.-M.+G.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=A.-B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+BET+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0ljWnx-mRMAJqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520BET%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D11%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molnar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1425-1439&issue=4&author=N.+H.+Theodoulouauthor=P.+Bamboroughauthor=A.+J.+Bannisterauthor=I.+Becherauthor=R.+A.+Bitauthor=K.+H.+Cheauthor=C.-w.+Chungauthor=A.+Dittmannauthor=G.+Drewesauthor=D.+H.+Drewryauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Leveridgeauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+Molnarauthor=S.+C.+Robsonauthor=N.+C.+O.+Tomkinsonauthor=T.+Kouzaridesauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Discovery+of+I-BRD9%2C+a+Selective+Cell+Active+Chemical+Probe+for+Bromodomain+Containing+Protein+9+Inhibition&doi=10.1021%2Facs.jmedchem.5b00256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span></div><div class="casAuthors">Theodoulou, Natalie H.; Bamborough, Paul; Bannister, Andrew J.; Becher, Isabelle; Bit, Rino A.; Che, Ka Hing; Chung, Chun-wa; Dittmann, Antje; Drewes, Gerard; Drewry, David H.; Gordon, Laurie; Grandi, Paola; Leveridge, Melanie; Lindon, Matthew; Michon, Anne-Marie; Molnar, Judit; Robson, Samuel C.; Tomkinson, Nicholas C. O.; Kouzarides, Tony; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1425-1439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetylation of histone lysine residues is one of the most well-studied post-translational modifications of chromatin, selectively recognized by bromodomain "reader" modules.  Inhibitors of the bromodomain and extra terminal domain (BET) family of bromodomains have shown profound anticancer and anti-inflammatory properties, generating much interest in targeting other bromodomain-contg. proteins for disease treatment.  Herein, we report the discovery of I-BRD9, the first selective cellular chem. probe for bromodomain-contg. protein 9 (BRD9).  I-BRD9 was identified through structure-based design, leading to greater than 700-fold selectivity over the BET family and 200-fold over the highly homologous bromodomain-contg. protein 7 (BRD7).  I-BRD9 was used to identify genes regulated by BRD9 in Kasumi-1 cells involved in oncol. and immune response pathways and to the best of our knowledge, represents the first selective tool compd. available to elucidate the cellular phenotype of BRD9 bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmjhksIDyTrVg90H21EOLACvtfcHk0ljWnx-mRMAJqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D&md5=4e8f6aa664797fe54772b64c81cf2a83</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMolnar%26aufirst%3DJ.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520I-BRD9%252C%2520a%2520Selective%2520Cell%2520Active%2520Chemical%2520Probe%2520for%2520Bromodomain%2520Containing%2520Protein%25209%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1425%26epage%3D1439%26doi%3D10.1021%2Facs.jmedchem.5b00256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camurri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaramella, A.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">3716</span>– <span class="NLM_lpage">3722</span>, <span class="refDoi"> DOI: 10.1021/ac001388j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac001388j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=3716-3722&issue=15&author=G.+Camurriauthor=A.+Zaramella&title=High-Throughput+Liquid+Chromatography%2FMass+Spectrometry+Method+for+the+Determination+of+the+Chromatographic+Hydrophobicity+Index&doi=10.1021%2Fac001388j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span></div><div class="casAuthors">Camurri, Giulio; Zaramella, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3716-3722</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast gradient reversed-phase liq. chromatog. (LC) method, using an acetonitrile gradient was developed to det. the chromatog. hydrophobicity index (CHI), as reported by Valco et al. (Anal. Chem. 1997, 69, 2022-2029).The anal. method provides retention times, based on UV detection at two different wavelengths, which then are converted into CHI values after calibration with a set of test compds.  The CHI of each compd. is measured at three different pH values, 2.0, 7.4, and 10.5; so using an 8-min gradient at each pH value one compd. can be analyzed in ∼24 min.  The aim of this work is to improve the throughput of the CHI screening using a LC/MS approach, so the application of the LC/MS technique is an extension of the LC/UV approach previously reported by Valco et al.  This approach allows contemporary injection of N compds. into the LC/MS system, the retention time of each compd. can be then extd. from the selected ion recording chromatograms.  The throughput of the existing screening method could be increased by N times, where N is the no. of compds. injected, so only three runs are needed to det. the CHI at three different pH values for a set of N compds.  The highest value of N depends on the total no. of channels that can be monitored simultaneously; 32 channels were used.  This LC/MS method was tested for a no. of com. products analyzed as mixts., and data obtained were compared with those coming from the classical LC/UV approach.  In the same way, the method was tested for a no. of compds. assocd. with two GlaxoWellcome projects in the antibacterial area.  The LC/MS method can be successfully applied for analyzing compds. in mixts. and for compds. with poor UV absorption, which cannot be analyzed with the std. LC/UV method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpqYuIDpy5rVg90H21EOLACvtfcHk0lipUJLtR2TQdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D&md5=8773c87e08ba7ddce3c2d679360a6220</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fac001388j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac001388j%26sid%3Dliteratum%253Aachs%26aulast%3DCamurri%26aufirst%3DG.%26aulast%3DZaramella%26aufirst%3DA.%26atitle%3DHigh-Throughput%2520Liquid%2520Chromatography%252FMass%2520Spectrometry%2520Method%2520for%2520the%2520Determination%2520of%2520the%2520Chromatographic%2520Hydrophobicity%2520Index%26jtitle%3DAnal.%2520Chem.%26date%3D2001%26volume%3D73%26issue%3D15%26spage%3D3716%26epage%3D3722%26doi%3D10.1021%2Fac001388j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunhuck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D. P.</span></span> <span> </span><span class="NLM_article-title">Fast Gradient HPLC Method to Determine Compounds Binding to Human Serum Albumin. Relationships with Octanol/Water and Immobilized Artificial Membrane Lipophilicity</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2236</span>– <span class="NLM_lpage">2248</span>, <span class="refDoi"> DOI: 10.1002/jps.10494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2Fjps.10494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=14603509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=2236-2248&issue=11&author=K.+Valkoauthor=S.+Nunhuckauthor=C.+Bevanauthor=M.+H.+Abrahamauthor=D.+P.+Reynolds&title=Fast+Gradient+HPLC+Method+to+Determine+Compounds+Binding+to+Human+Serum+Albumin.+Relationships+with+Octanol%2FWater+and+Immobilized+Artificial+Membrane+Lipophilicity&doi=10.1002%2Fjps.10494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fast gradient HPLC method to determine compounds binding to human serum albumin. Relationships with octanol/water and immobilized artificial membrane lipophilicity</span></div><div class="casAuthors">Valko, Klara; Nunhuck, Shenaz; Bevan, Chris; Abraham, Michael H.; Reynolds, Derek P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2236-2248</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A fast gradient HPLC method (cycle time 15 min) has been developed to det. Human Serum Albumin (HSA) binding of discovery compds. using chem. bonded protein stationary phases.  The HSA binding values were derived from the gradient retention times that were converted to the logarithm of the equil. consts. (logK HSA) using data from a calibration set of mols.  The method has been validated using literature plasma protein binding data of 68 known drug mols.  The method is fully automated, and has been used for lead optimization in more than 20 company projects.  The HSA binding data obtained for more than 4000 compds. were suitable to set up global and project specific quant. structure binding relationships that helped compd. design in early drug discovery.  The obtained HSA binding of known drug mols. were compared to the Immobilizd Artificial Membrane binding data (CHI IAM) obtained by our previously described HPLC-based method.  The solvation equation approach has been used to characterize the normal binding ability of HSA, and this relationship shows that compd. lipophilicity is a significant factor.  It was found that the selectivity of the "baseline" lipophilicity governing HSA binding, membrane interaction, and octanol/water partition are very similar.  However, the effect of the presence of pos. or neg. charges have very different effects.  It was found that neg. charged compds. bind more strongly to HSA than it would be expected from the lipophilicity of the ionized species at pH 7.4.  Several compds. showed stronger HSA binding than can be expected from their lipophilicity alone, and comparison between predicted and exptl. binding affinity allows the identification of compds. that have good complementarities with any of the known binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1LUBLdv-t7Vg90H21EOLACvtfcHk0lipUJLtR2TQdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D&md5=14eb1e0f0ad616846a2b5d1428133fa9</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjps.10494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10494%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DNunhuck%26aufirst%3DS.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DReynolds%26aufirst%3DD.%2BP.%26atitle%3DFast%2520Gradient%2520HPLC%2520Method%2520to%2520Determine%2520Compounds%2520Binding%2520to%2520Human%2520Serum%2520Albumin.%2520Relationships%2520with%2520Octanol%252FWater%2520and%2520Immobilized%2520Artificial%2520Membrane%2520Lipophilicity%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2003%26volume%3D92%26issue%3D11%26spage%3D2236%26epage%3D2248%26doi%3D10.1002%2Fjps.10494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Huarui Cui, Angela S. Carlson, Mary A. Schleiff, Anand Divakaran, Jorden A. Johnson, Caroline R. Buchholz, Huda Zahid, Nora R. Vail, Ke Shi, Hideki Aihara, Daniel A. Harki, Grover P. Miller, Joseph J. Topczewski, <span class="NLM_string-name hlFld-ContribAuthor">William C. K. Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">4-Methyl-1,2,3-Triazoles as N-Acetyl-Lysine Mimics Afford Potent BET Bromodomain Inhibitors with Improved Selectivity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 10497-10511. <a href="https://doi.org/10.1021/acs.jmedchem.1c00933" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00933</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00933%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D4-Methyl-1%25252C2%25252C3-Triazoles%252Bas%252BN-Acetyl-Lysine%252BMimics%252BAfford%252BPotent%252BBET%252BBromodomain%252BInhibitors%252Bwith%252BImproved%252BSelectivity%26aulast%3DCui%26aufirst%3DHuarui%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D24052021%26date%3D08072021%26volume%3D64%26issue%3D14%26spage%3D10497%26epage%3D10511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gang  Zhang</span>, <span class="hlFld-ContribAuthor ">Shan  Li</span>, <span class="hlFld-ContribAuthor ">Kai-Wen  Cheng</span>, <span class="hlFld-ContribAuthor ">Tsui-Fen  Chou</span>. </span><span class="cited-content_cbyCitation_article-title">AAA ATPases as therapeutic targets: Structure, functions, and small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>219 </em>, 113446. <a href="https://doi.org/10.1016/j.ejmech.2021.113446" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113446</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113446%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAAA%252BATPases%252Bas%252Btherapeutic%252Btargets%25253A%252BStructure%25252C%252Bfunctions%25252C%252Band%252Bsmall-molecule%252Binhibitors%26aulast%3DZhang%26aufirst%3DGang%26date%3D2021%26volume%3D219%26spage%3D113446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Virgyl  Camberlein</span>, <span class="hlFld-ContribAuthor ">Nicolas  Kraupner</span>, <span class="hlFld-ContribAuthor ">Nour  Bou Karroum</span>, <span class="hlFld-ContribAuthor ">Emmanuelle  Lipka</span>, <span class="hlFld-ContribAuthor ">Rebecca  Deprez-Poulain</span>, <span class="hlFld-ContribAuthor ">Benoit  Deprez</span>, <span class="hlFld-ContribAuthor ">Damien  Bosc</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-component reaction for the preparation of 1,5-disubstituted 1,2,3-triazoles by in-situ generation of azides and nickel-catalyzed azide-alkyne cycloaddition. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2021,</strong> <em>73 </em>, 153131. <a href="https://doi.org/10.1016/j.tetlet.2021.153131" title="DOI URL">https://doi.org/10.1016/j.tetlet.2021.153131</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2021.153131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2021.153131%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DMulti-component%252Breaction%252Bfor%252Bthe%252Bpreparation%252Bof%252B1%25252C5-disubstituted%252B1%25252C2%25252C3-triazoles%252Bby%252Bin-situ%252Bgeneration%252Bof%252Bazides%252Band%252Bnickel-catalyzed%252Bazide-alkyne%252Bcycloaddition%26aulast%3DCamberlein%26aufirst%3DVirgyl%26date%3D2021%26volume%3D73%26spage%3D153131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Small molecule inhibitors of ATAD2 and CECR2 with KAc mimetics shown in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) X-ray crystal structure of <b>(</b><i><b>rac</b></i><b>)-8</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55">6S55</a><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a>) overlaid with KAc binding in ATAD2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QSP">4QSP</a><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>). The density fitted the <i>S</i>-enantiomer better, and so this was modeled. The conserved network of waters (W1–W4) at the base of the binding pocket are displaced by the phenyl sulfonamide ligand. (B) Key contacts between the ligand (<b>(</b><i><b>rac</b></i><b>)-8</b>) and ATAD2 are shown as yellow dashes. A halogen bond between the aryl bromide and the backbone carbonyl group of I1056 is present at the base of the KAc recognition site. One of the sulfonamide oxygens makes a H-bond to Y1021, and the two carbonyl groups of the hydantoin make H-bonds to R1007 and D1014. (C) Alternative view of (B).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YB4">6YB4</a>) of <b>(</b><i><b>R,S</b></i><b>)-32</b> (teal) in ATAD2 (silver) and a docking model of <b>(</b><i><b>S,S</b></i><b>)-32</b> (lime) in CECR2 (lilac, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXB">3NXB</a><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a>): (A) pose showing <b>(</b><i><b>R,S</b></i><b>)-32</b> sitting deep within the binding pocket, where the ligand displaces the normally conserved water network in the KAc recognition site; (B) rotated pose showing the hydantoin carbonyl groups making hydrogen bonds to both R1007 and D1014 and one of the sulfonamide oxygens hydrogen bonding to Y1021; (C) pose showing there is space for <b>(</b><i><b>S,S</b></i><b>)-32</b> to displace the waters in CECR2 and make key contacts with M506 and Y471; (D) rotated pose showing that <b>(</b><i><b>S,S</b></i><b>)-32</b> makes a H-bond to D464 NH but does not make any specific interactions with the WPF shelf in CECR2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. DiscoverX’s BROMO<i>scan</i> results of <b>(</b><i><b>S</b></i><b>)-32</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structure of <b>(</b><i><b>R,S</b></i><b>)-32</b> (lime) in ATAD2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YB4">6YB4</a>) with point changes between the CECR2 (lilac, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXB">3NXB</a><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a>) and ATAD2 (gray) domains annotated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>(</b><i><b>S,S</b></i><b>)-32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: a) (<i>S</i>)-methylpyrrolidine hydrochloride (1.0 equiv), DIPEA (2.2 equiv), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 1 h, 90%; (b) iron (5.0 equiv), NH<sub>4</sub>Cl (1.5 equiv), EtOH/water (3:1), 70 °C, 2 h, 98%; (c) (±)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetic acid (1.1 equiv), DIPEA (3.0 equiv), T3P (50% w/w in EtOAc, 1.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 73%, then chiral purification.</p></p></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of a Late Stage Sulfonyl Chloride for Use in a Sulfonamide Array<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) iron (3.0 equiv), NH<sub>4</sub>Cl (1.5 equiv), EtOH/water (3:1), 70 °C, 16 h, 88%; (b) 2-chloroacetyl chloride (1.0 equiv), DIPEA (1.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 1 h, 88%; (c) hydantoin (1.2 equiv), K<sub>2</sub>CO<sub>3</sub> (3.0 equiv), DMF, 70 °C, 1 h, 79%; (d) benzylthiol (1.5 equiv), xantphos (20 mol %), Pd<sub>2</sub>dba<sub>3</sub> (10 mol %), DIPEA (3.0 equiv), 1,4-dioxane/DMF (3:2), 100 °C, 1 h, 77%; (e) 1,3-dichloro-5,5-dimethylimidazolidine-2,4-dione (2.0 equiv), water, AcOH, MeCN, rt, 2 h, 35%; (f) amine (2.0 equiv), DIPEA (3.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 4 h, 10–87%.</p></p></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Triazoles <b>40</b>–<b>46</b> Using Either Copper or Ruthenium Catalyzed Click Chemistry<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2-azidoacetic acid (1.0 equiv), HATU (1.2 equiv), DIPEA (3.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 94%; (b) alkyne (1.5 equiv), Cp*RuCl(PPh<sub>3</sub>)<sub>2</sub> (3 mol %), THF, 65 °C, 2 h, 22–82%; (c) alkyne (1.1 equiv), CuSO<sub>4</sub>·5H<sub>2</sub>O (5 mol %), sodium ascorbate (0.15 equiv), CH<sub>2</sub>Cl<sub>2</sub>/water (1:1), rt, 16 h, 49–59%; (d) alkyne (2.0 equiv), Cu(OAc)<sub>2</sub>·H<sub>2</sub>O (10 mol %), sodium ascorbate (0.15 equiv), MeOH, 100 °C, microwave, 30 min, 42%.</p></p></figure><figure data-id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>(</b><i><b>R</b></i><b>)-55</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert-</i>butyl glycinate (1.1 equiv), TEA (1.0 equiv), EtOH, reflux, 6 h, 33%; (b) iron (5.0 equiv), NH<sub>4</sub>Cl (1.5 equiv), EtOH/water (3:1), 70 °C, 2 h, 91%; (c) NaNO<sub>2</sub> (1.2 equiv), AcOH/water (1:1), 0–80 °C, 37%; (d) 3-cyclopropylpyrrolidine (1.0 equiv), DIPEA (2.2 equiv), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C, 2 h, 90%; (e) iron (3.0 equiv), NH<sub>4</sub>Cl (1.5 equiv), 3:1 EtOH/water, 70 °C, 2 h, 98%; (f) HCl (4 M in dioxane, 8.3 equiv), 40 °C, overnight, then HATU (1.2 equiv), DIPEA (3.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, rt, 2 h, 79%; (g) MeI (1.0 equiv), MeCN, 80 °C, 2 h, then NaBH<sub>4</sub> (2.2 equiv), 1:1 MeOH/water, rt, 73%.</p></p></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/medium/jm0c00021_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. DiscoverX’s BROMO<i>scan</i> results of <b>(</b><i><b>R</b></i><b>)-55</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00021/20200520/images/large/jm0c00021_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00021&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i133">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71738" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71738" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, M. C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain Histone Readers and Cancer</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">2003</span>– <span class="NLM_lpage">2010</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2016.11.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.jmb.2016.11.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=27890782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOisbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2003-2010&issue=13&author=A.+K.+Jainauthor=M.+C.+Barton&title=Bromodomain+Histone+Readers+and+Cancer&doi=10.1016%2Fj.jmb.2016.11.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain histone readers and cancer</span></div><div class="casAuthors">Jain, Abhinav K.; Barton, Michelle C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2003-2010</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation of histone proteins is a fundamental post-translational modification that regulates chromatin structure and plays an important role in gene transcription.  Aberrant levels of histone lysine acetylation are assocd. with the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are structurally conserved modules present in transcription-assocd. proteins that are termed "reader" proteins.  Bromodomain-contg. reader proteins are part of multiprotein complexes that regulate transcription programs, which are often assocd. with profound phenotypic changes.  Many bromodomain-contg. proteins are aberrantly expressed in diseases, as best studied in cancers, where bromodomain proteins impact the expression of oncogenes and anti-apoptotic proteins.  Thus, bromodomain readers of histone acetylation have emerged as attractive targets for cancer drug discovery, prompting immense interest in epigenetic-focused, medicinal chem. to develop structurally guided chem. probes of bromodomains.  Here, we describe bromodomain-contg. proteins with defined roles in cancer and highlight recent progress in the development of bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPuEwpy7dAA7Vg90H21EOLACvtfcHk0li_VaStI47bjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOisbbI&md5=2d1d3efa959f66eeb8789b48daf96325</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2016.11.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2016.11.020%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DM.%2BC.%26atitle%3DBromodomain%2520Histone%2520Readers%2520and%2520Cancer%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26issue%3D13%26spage%3D2003%26epage%3D2010%26doi%3D10.1016%2Fj.jmb.2016.11.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting Bromodomains: Epigenetic Readers of Lysine Acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+Bromodomains%3A+Epigenetic+Readers+of+Lysine+Acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lgXLaD2IbudJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520Bromodomains%253A%2520Epigenetic%2520Readers%2520of%2520Lysine%2520Acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=598-608&issue=3&author=S.+G.+Smithauthor=M.-M.+Zhou&title=The+Bromodomain%3A+A+New+Target+in+Emerging+Epigenetic+Medicine&doi=10.1021%2Facschembio.5b00831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span></div><div class="casAuthors">Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-608</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved protein modular domain found in many chromatin- and transcription-assocd. proteins that has the ability to recognize acetylated lysine residues.  This activity allows bromodomains to play a vital role in many acetylation-mediated protein-protein interactions in the cell, ranging from substrate recruitment for histone acetyltransferases (HATs) to aiding in multiple-protein complex assembly for gene transcriptional activation or suppression in chromatin.  In recent years, considerable efforts have been made to develop chem. inhibitors of these bromodomains in an effort to probe their cellular functions.  Potent and selective inhibitors have been extensively developed for one group of the bromodomain family termed BET proteins that consist of tandem bromodomains followed by an extra terminal domain.  Drug developers are actively designing inhibitors of other bromodomains and working to move the most successful inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQF2Qjn_iuLVg90H21EOLACvtfcHk0lgXLaD2IbudJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ&md5=dd34fc591550ff1528e6dccd1d04f1c1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DThe%2520Bromodomain%253A%2520A%2520New%2520Target%2520in%2520Emerging%2520Epigenetic%2520Medicine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26issue%3D3%26spage%3D598%26epage%3D608%26doi%3D10.1021%2Facschembio.5b00831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovén, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.cell.2013.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=23582323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=320-334&issue=2&author=J.+Lov%C3%A9nauthor=H.+A.+Hokeauthor=C.+Y.+Linauthor=A.+Lauauthor=D.+A.+Orlandoauthor=C.+R.+Vakocauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Selective+Inhibition+of+Tumor+Oncogenes+by+Disruption+of+Super-Enhancers&doi=10.1016%2Fj.cell.2013.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span></div><div class="casAuthors">Loven, Jakob; Hoke, Heather A.; Lin, Charles Y.; Lau, Ashley; Orlando, David A.; Vakoc, Christopher R.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-334</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells.  Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM).  BRD4 and Mediator were found to co-occupy thousands of enhancers assocd. with active genes.  They also co-occupied a small set of exceptionally large super-enhancers assocd. with genes that feature prominently in MM biol., including the MYC oncogene.  Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC.  Super-enhancers were found at key oncogenic drivers in many other tumor cells.  These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-NqFYuGMqrVg90H21EOLACvtfcHk0ljMekAWJeByng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D&md5=66b6ccb5b8a6039832910443fd8b4664</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DLov%25C3%25A9n%26aufirst%3DJ.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520Inhibition%2520of%2520Tumor%2520Oncogenes%2520by%2520Disruption%2520of%2520Super-Enhancers%26jtitle%3DCell%26date%3D2013%26volume%3D153%26issue%3D2%26spage%3D320%26epage%3D334%26doi%3D10.1016%2Fj.cell.2013.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-F.</span></span> <span> </span><span class="NLM_article-title">Brd4 Coactivates Transcriptional Activation of NF-κB via Specific Binding to Acetylated RelA</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1375</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1128/MCB.01365-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&issue=5&author=B.+Huangauthor=X.-D.+Yangauthor=M.-M.+Zhouauthor=K.+Ozatoauthor=L.-F.+Chen&title=Brd4+Coactivates+Transcriptional+Activation+of+NF-%CE%BAB+via+Specific+Binding+to+Acetylated+RelA&doi=10.1128%2FMCB.01365-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0ljMekAWJeByng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.-D.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DBrd4%2520Coactivates%2520Transcriptional%2520Activation%2520of%2520NF-%25CE%25BAB%2520via%2520Specific%2520Binding%2520to%2520Acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26issue%3D5%26spage%3D1375%26epage%3D1387%26doi%3D10.1128%2FMCB.01365-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Clinical Progress and Pharmacology of Small Molecule Bromodomain Inhibitors</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2016.05.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.cbpa.2016.05.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=27295577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=58-66&author=N.+H.+Theodoulouauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Clinical+Progress+and+Pharmacology+of+Small+Molecule+Bromodomain+Inhibitors&doi=10.1016%2Fj.cbpa.2016.05.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58-66</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains have emerged as an exciting target class for drug discovery over the past decade.  Research has primarily focused on the bromodomain and extra terminal (BET) family of bromodomains, which has led to the development of multiple small mol. inhibitors and an increasing no. of clin. assets.  The excitement centered on the clin. potential of BET inhibition has stimulated intense interest in the broader family and the growing no. of non-BET bromodomain chem. probes has facilitated phenotypic investigations, implicating these targets in a variety of disease pathways including cancer, inflammation, embryonic development and neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEE0jCnUuyk7Vg90H21EOLACvtfcHk0lhCYCye3fUIjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D&md5=4bcd42755b2dd3cebd7fae417e69e83f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2016.05.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2016.05.028%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DClinical%2520Progress%2520and%2520Pharmacology%2520of%2520Small%2520Molecule%2520Bromodomain%2520Inhibitors%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D33%26spage%3D58%26epage%3D66%26doi%3D10.1016%2Fj.cbpa.2016.05.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Scaling the Druggability Landscape of Human Bromodomains, a New Class of Drug Targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">7342</span>– <span class="NLM_lpage">7345</span>, <span class="refDoi"> DOI: 10.1021/jm3011977</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3011977" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1yhsLnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7342-7345&issue=17&author=G.+Zhangauthor=R.+Sanchezauthor=M.+Zhou&title=Scaling+the+Druggability+Landscape+of+Human+Bromodomains%2C+a+New+Class+of+Drug+Targets&doi=10.1021%2Fjm3011977"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling the Druggability Landscape of Human Bromodomains, a New Class of Drug Targets</span></div><div class="casAuthors">Zhang, Guangtao; Sanchez, Roberto; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7342-7345</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Gene transcriptional activation of the human genome in response to physiol. and environmental stimuli requires chromatin structure changes defined by enzymes that modify chromatin and directed by proteins that interact with chromatin in a modification-sensitive manner.  This highly complex system operates with a large no. of chem. modifications on chromatin (both DNA and histones) and transcription-assocd. proteins.  Of these, lysine acetylation functions to facilitate chromatin opening and productive transcriptional machinery assembly required for gene activation.  These activities are directed by the acetyl-lysine binding activity of the bromodomain (BrD), a fundamental mol. mechanism for gene transcriptional activation that was discovered in the structural biol. study of the histone acetyltransferase (HAT) transcriptional coactivator PCAF.  As a parallel in drug discovery that is moving from highly druggable targets such as kinases and GPCRs to more challenging PPI targets, rapidly growing ligand design efforts for bromodomains are expected to progress from targeting a few druggable bromodomains to more difficult ones.  The outcome of these studies will undoubtedly enable us to validate members of this new class of drug targets and develop more effective targeted epigenetic therapies for human diseases including cancer and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq7hhNrw3hGLVg90H21EOLACvtfcHk0lhCYCye3fUIjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1yhsLnK&md5=d2b5afff493388e946e8753a35883d2b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm3011977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3011977%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DScaling%2520the%2520Druggability%2520Landscape%2520of%2520Human%2520Bromodomains%252C%2520a%2520New%2520Class%2520of%2520Drug%2520Targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26issue%3D17%26spage%3D7342%26epage%3D7345%26doi%3D10.1021%2Fjm3011977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moustakim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Chemical Probes and Inhibitors of Bromodomains Outside the BET Family</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2246</span>– <span class="NLM_lpage">2264</span>, <span class="refDoi"> DOI: 10.1039/C6MD00373G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1039%2FC6MD00373G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=29170712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2246-2264&issue=12&author=M.+Moustakimauthor=P.+G.+K.+Clarkauthor=D.+A.+Hayauthor=D.+J.+Dixonauthor=P.+E.+Brennan&title=Chemical+Probes+and+Inhibitors+of+Bromodomains+Outside+the+BET+Family&doi=10.1039%2FC6MD00373G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes and inhibitors of bromodomains outside the BET family</span></div><div class="casAuthors">Moustakim, Moses; Clark, Peter G. K.; Hay, Duncan A.; Dixon, Darren J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2246-2264</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last five years, the development of inhibitors of bromodomains has emerged as an area of intensive worldwide research.  Emerging evidence has implicated a no. of non-BET bromodomains in the onset and progression of diseases such as cancer, HIV infection and inflammation.  The development and use of small mol. chem. probes has been fundamental to pre-clin. evaluation of bromodomains as targets.  Recent efforts are described highlighting the development of potent, selective and cell active non-BET bromodomain inhibitors and their therapeutic potential.  Over half of typical bromodomains now have reported ligands, but those with atypical binding site residues remain resistant to chem. probe discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYJvhj3Z0Ry7Vg90H21EOLACvtfcHk0liK9OcfgXfkSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhsbbF&md5=df4c5718b15ff7d401a0cd9974abc236</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2FC6MD00373G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00373G%26sid%3Dliteratum%253Aachs%26aulast%3DMoustakim%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DChemical%2520Probes%2520and%2520Inhibitors%2520of%2520Bromodomains%2520Outside%2520the%2520BET%2520Family%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26issue%3D12%26spage%3D2246%26epage%3D2264%26doi%3D10.1039%2FC6MD00373G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Progress in the Development of non-BET Bromodomain Chemical Probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2Fcmdc.201500540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=26749027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=477-487&issue=5&author=N.+H.+Theodoulouauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Progress+in+the+Development+of+non-BET+Bromodomain+Chemical+Probes&doi=10.1002%2Fcmdc.201500540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">477-487</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain and extra terminal (BET) family of bromodomains have been the focus of extensive research, leading to the development of many potent, selective chem. probes and recent clin. assets.  The profound biol. assocd. with BET bromodomain inhibition has provided a convincing rationale for targeting bromodomains for the treatment of disease.  However, the BET family represents just eight of the at least 56 human bromodomains identified to date.  Until recently, there has been significantly less interest in non-BET bromodomains, leaving a vast area of research and the majority of this new target class yet to be thoroughly investigated.  It has been widely reported that several non-BET bromodomain contg. proteins are assocd. with various diseases including cancer and HIV.  Therefore, the development of chem. probes for non-BET bromodomains will facilitate elucidation of their precise biol. roles and potentially lead to the development of new medicines.  This review summarises the progress made towards the development of non-BET bromodomain chem. probes to date.  In addn., we highlight the potential for future work in this new and exciting area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYLj_VyHOS5bVg90H21EOLACvtfcHk0liK9OcfgXfkSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xks1ehtA%253D%253D&md5=1e1339bf09a10bde8621c18344172fa6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500540%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DProgress%2520in%2520the%2520Development%2520of%2520non-BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26issue%3D5%26spage%3D477%26epage%3D487%26doi%3D10.1002%2Fcmdc.201500540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span> <span> </span><span class="NLM_article-title">Target (In)Validation: A Critical, Sometimes Unheralded, Role of Modern Medicinal Chemistry</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">618</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovVSnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=618-621&issue=6&author=R.+F.+Sweis&title=Target+%28In%29Validation%3A+A+Critical%2C+Sometimes+Unheralded%2C+Role+of+Modern+Medicinal+Chemistry&doi=10.1021%2Facsmedchemlett.5b00183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Target (In)Validation: A Critical, Sometimes Unheralded, Role of Modern Medicinal Chemistry</span></div><div class="casAuthors">Sweis, Ramzi F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">618-621</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review. Small mol. drug discovery commonly ventures into previously unknown and unexplored target space.  For such programs, an important role of medicinal chem. is to generate mols. that enable the most reliable conclusions from a preclin. target validation/invalidation study.  Multiple facets of chem. that provide the most rigorous results for such an expt. are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZCtopNE1bl7Vg90H21EOLACvtfcHk0liK9OcfgXfkSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovVSnu7o%253D&md5=0da4f3e6e2ead39de6e16219d20c118a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00183%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26atitle%3DTarget%2520%2528In%2529Validation%253A%2520A%2520Critical%252C%2520Sometimes%2520Unheralded%252C%2520Role%2520of%2520Modern%2520Medicinal%2520Chemistry%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26issue%3D6%26spage%3D618%26epage%3D621%26doi%3D10.1021%2Facsmedchemlett.5b00183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Bromodomains: Are Readers Right for Epigenetic Therapy?</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">691</span>– <span class="NLM_lpage">694</span>, <span class="refDoi"> DOI: 10.1021/ml300221t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300221t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFKgu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=691-694&issue=9&author=S.+J.+Conway&title=Bromodomains%3A+Are+Readers+Right+for+Epigenetic+Therapy%3F&doi=10.1021%2Fml300221t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains: Are Readers Right for Epigenetic Therapy?</span></div><div class="casAuthors">Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">691-694</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  There is intense interest in the development of small mol. inhibitors of the acetyl-lysine-reading bromodomain protein module.  These inhibitors represent a way of interfering therapeutically in epigenetic processes, and there are currently two bromodomain inhibitors in clin. trials.  The success of these compds. rests on safety aspects of epigenetic target modulation being addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLmhkWiev2rLVg90H21EOLACvtfcHk0lgIEb59gT-F0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFKgu7nL&md5=e3ac85cb4b639616b829295e866da801</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fml300221t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300221t%26sid%3Dliteratum%253Aachs%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DBromodomains%253A%2520Are%2520Readers%2520Right%2520for%2520Epigenetic%2520Therapy%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26issue%3D9%26spage%3D691%26epage%3D694%26doi%3D10.1021%2Fml300221t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&issue=4&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+Acetyl-Lysine+Recognition%3A+Progress+in+the+Development+of+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lgIEb59gT-F0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520Acetyl-Lysine%2520Recognition%253A%2520Progress%2520in%2520the%2520Development%2520of%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Advancements in the Development of non-BET Bromodomain Chemical Probes</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2Fcmdc.201800738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=30624862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=362-385&issue=4&author=M.+A.+Cleggauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Advancements+in+the+Development+of+non-BET+Bromodomain+Chemical+Probes&doi=10.1002%2Fcmdc.201800738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Advancements in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">362-385</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extra terminal (BET) family of bromodomain-contg. proteins (BCPs) have been the subject of extensive research over the past decade, resulting in a plethora of high-quality chem. probes for their tandem bromodomains.  In turn, these chem. probes have helped reveal the profound biol. role of the BET bromodomains and their role in disease, ultimately leading to a no. of mols. in active clin. development.  However, the BET subfamily represents just 8/61 of the known human bromodomains, and attention has now expanded to the biol. role of the remaining 53 non-BET bromodomains.  Rapid growth of this research area has been accompanied by a greater understanding of the requirements for an effective bromodomain chem. probe and has led to a no. of new non-BET bromodomain chem. probes being developed.  Advances since Dec. 2015 are discussed, highlighting the strengths/caveats of each mol., and the value they add toward validating the non-BET bromodomains as tractable therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_U9uvpaCEd7Vg90H21EOLACvtfcHk0ljp-0e5cqJ6FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D&md5=a93505759fe45ffd0352de2f0e17885a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DAdvancements%2520in%2520the%2520Development%2520of%2520non-BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26issue%3D4%26spage%3D362%26epage%3D385%26doi%3D10.1002%2Fcmdc.201800738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Target Validation using Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&issue=4&author=M.+E.+Bunnageauthor=E.+L.+P.+Cheklerauthor=L.+H.+Jones&title=Target+Validation+using+Chemical+Probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0ljp-0e5cqJ6FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520Validation%2520using%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D4%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span> <span> </span><span class="NLM_article-title">The Art of the Chemical Probe</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">159</span>, <span class="refDoi"> DOI: 10.1038/nchembio.296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159&author=S.+V.+Frye&title=The+Art+of+the+Chemical+Probe&doi=10.1038%2Fnchembio.296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0lgjCF_i0KKphA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520Art%2520of%2520the%2520Chemical%2520Probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26doi%3D10.1038%2Fnchembio.296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Small Molecules and their Role in Effective Preclinical Target Validation</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">1579</span>– <span class="NLM_lpage">1582</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.4155%2Ffmc-2017-0121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=28828889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFensbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1579-1582&issue=14&author=M.+A.+Cleggauthor=N.+C.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Small+Molecules+and+their+Role+in+Effective+Preclinical+Target+Validation&doi=10.4155%2Ffmc-2017-0121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules and their role in effective preclinical target validation</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1579-1582</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUfI_qHeoQ0rVg90H21EOLACvtfcHk0lgjCF_i0KKphA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFensbfM&md5=b8fea6af5ce3bf5c61e1a58c445404b8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0121%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DSmall%2520Molecules%2520and%2520their%2520Role%2520in%2520Effective%2520Preclinical%2520Target%2520Validation%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26issue%3D14%26spage%3D1579%26epage%3D1582%26doi%3D10.4155%2Ffmc-2017-0121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-W.</span></span> <span> </span><span class="NLM_article-title">ANCCA, an Estrogen-Regulated AAA+ ATPase Coactivator for ERa, is Required for Coregulator Occupancy and Chromatin Modification</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span> (<span class="NLM_issue">46</span>),  <span class="NLM_fpage">18067</span>– <span class="NLM_lpage">18072</span>, <span class="refDoi"> DOI: 10.1073/pnas.0705814104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1073%2Fpnas.0705814104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=17998543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=18067-18072&issue=46&author=J.+X.+Zouauthor=A.+S.+Revenkoauthor=L.+B.+Liauthor=A.+T.+Gemoauthor=H.-W.+Chen&title=ANCCA%2C+an+Estrogen-Regulated+AAA%2B+ATPase+Coactivator+for+ERa%2C+is+Required+for+Coregulator+Occupancy+and+Chromatin+Modification&doi=10.1073%2Fpnas.0705814104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERα, is required for coregulator occupancy and chromatin modification</span></div><div class="casAuthors">Zou, June X.; Revenko, Alexey S.; Li, Li B.; Gemo, Abigael T.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18067-18072</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">AAA+ proteins play crucial roles in diverse biol. processes via their ATPase-driven remodeling of macromol. complexes.  Here the authors report the authors' identification of an evolutionarily conserved AAA+ protein, ANCCA/Pro 2000, endowed with a bromodomain that is strongly induced by estrogen in human breast cancer cells and is a direct target of protooncogene ACTR/AlB1/SRC-3.  The authors found that ANCCA assocs. directly with estrogen-bound estrogen receptor (ER) α and ACTR.  It is selectively recruited, upon estrogen stimulation, to a subset of ERα target genes including cyclin D1, c-myc, and E2F1 and is required for their estrogen-induced expression as well as breast cancer cell proliferation.  Further studies indicate that ANCCA binds and hydrolyzes ATP and is crit. for recruitment of coregulator CBP and histone hyperacetylation at the ER target chromatin.  Moreover, mutations at the ATP binding motifs rendered ANCCA defective as a coactivator in mediating estrogen induction of gene expression.  Together, the authors' findings reveal an unexpected layer of regulatory mechanism in hormone signaling mediated by ANCCA and suggest that hormone-induced assembly of transcriptional coregulator complexes at chromatin is a process facilitated by AAA+ ATPase proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NFjjiBVoRbVg90H21EOLACvtfcHk0lgjCF_i0KKphA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO&md5=618fcc431bf4b0a7589213fea55732ca</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0705814104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0705814104%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DLi%26aufirst%3DL.%2BB.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DH.-W.%26atitle%3DANCCA%252C%2520an%2520Estrogen-Regulated%2520AAA%252B%2520ATPase%2520Coactivator%2520for%2520ERa%252C%2520is%2520Required%2520for%2520Coregulator%2520Occupancy%2520and%2520Chromatin%2520Modification%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26issue%3D46%26spage%3D18067%26epage%3D18072%26doi%3D10.1073%2Fpnas.0705814104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lestrat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoffier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virolle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Functional Characterization of ATAD2 as a New Cancer/Testis Factor and a Predictor of Poor Prognosis in Breast and Lung Cancers</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">37</span>),  <span class="NLM_fpage">5171</span>– <span class="NLM_lpage">5181</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fonc.2010.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=20581866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=5171-5181&issue=37&author=C.+Caronauthor=C.+Lestratauthor=S.+Marsalauthor=E.+Escoffierauthor=S.+Curtetauthor=V.+Virolleauthor=P.+Barbryauthor=A.+Debernardiauthor=C.+Brambillaauthor=E.+Brambillaauthor=S.+Rousseauxauthor=S.+Khochbin&title=Functional+Characterization+of+ATAD2+as+a+New+Cancer%2FTestis+Factor+and+a+Predictor+of+Poor+Prognosis+in+Breast+and+Lung+Cancers&doi=10.1038%2Fonc.2010.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span></div><div class="casAuthors">Caron, C.; Lestrat, C.; Marsal, S.; Escoffier, E.; Curtet, S.; Virolle, V.; Barbry, P.; Debernardi, A.; Brambilla, C.; Brambilla, E.; Rousseaux, S.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5171-5181</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells frequently express genes normally active in male germ cells.  ATAD2 is one of them encoding a conserved factor harboring an AAA type ATPase domain and a bromodomain.  We show here that ATAD2 is highly expressed in testis as well as in many cancers of different origins and that its high expression is a strong predictor of rapid mortality in lung and breast cancers.  These observations suggest that ATAD2 acts on upstream and basic cellular processes to enhance oncogenesis in a variety of unrelated cell types.  Accordingly, our functional studies show that ATAD2 controls chromatin dynamics, genome transcriptional activities and apoptotic cell response.  We could also highlight some of the important intrinsic properties of its two regulatory domains, including a functional cross-talk between the AAA ATPase domain and the bromodomain.  Altogether, these data indicate that ATAD2 overexpression in somatic cells, by acting on basic properties of chromatin, may contribute to malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotoD3fqS5d4LVg90H21EOLACvtfcHk0linbqrXkx0bjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D&md5=1eaa99ca9fd31e5a486bedd4d0f6178d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.259%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DLestrat%26aufirst%3DC.%26aulast%3DMarsal%26aufirst%3DS.%26aulast%3DEscoffier%26aufirst%3DE.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVirolle%26aufirst%3DV.%26aulast%3DBarbry%26aufirst%3DP.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DBrambilla%26aufirst%3DC.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DFunctional%2520Characterization%2520of%2520ATAD2%2520as%2520a%2520New%2520Cancer%252FTestis%2520Factor%2520and%2520a%2520Predictor%2520of%2520Poor%2520Prognosis%2520in%2520Breast%2520and%2520Lung%2520Cancers%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26issue%3D37%26spage%3D5171%26epage%3D5181%26doi%3D10.1038%2Fonc.2010.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boussouar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidikia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morozumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Malignant Genome Reprogramming by ATAD2</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1829</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2013.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.bbagrm.2013.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1829&publication_year=2013&pages=1010-1014&issue=10&author=F.+Boussouarauthor=M.+Jamshidikiaauthor=Y.+Morozumiauthor=S.+Rousseauxauthor=S.+Khochbin&title=Malignant+Genome+Reprogramming+by+ATAD2&doi=10.1016%2Fj.bbagrm.2013.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Malignant genome reprogramming by ATAD2</span></div><div class="casAuthors">Boussouar, Faycal; Jamshidikia, Mahya; Morozumi, Yuichi; Rousseaux, Sophie; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1829</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1010-1014</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Unscheduled expression of crit. cellular regulators could be central to malignant genome reprogramming and tumor establishment.  One such factor appears to be ATAD2, a remarkably conserved protein normally predominantly expressed in germ cells but almost systematically over-expressed in a variety of unrelated cancers.  The presence of a bromodomain adjacent to an AAA type ATPase domain, points to ATAD2 as a factor preliminarily acting on chromatin structure and function.  Accordingly, ATAD2 has been shown to cooperate with a series of transcription factors and chromatin modifiers to regulate specific set of genes.  Here we discuss our knowledge on ATAD2 to evaluate its role as a cancer driver and its value as a new anti-cancer target.  Upon its activation, ATAD2 through its interaction with defined transcription factors, initiates a loop of transcriptional stimulation of target genes, including ATAD2 itself, leading to enhanced cell proliferation and resistance to apoptosis in an ATAD2-dependent manner.  Approaches aiming at neutralizing ATAD2 activity in cancer, including the use of small mol. inhibitors of its two "druggable" domains, AAA ATPase and bromodomain, could become part of a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp824JMo6Dp4rVg90H21EOLACvtfcHk0linbqrXkx0bjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVClsrnF&md5=cba5991a2dc5f96f522e1104ae4e4889</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2013.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2013.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DMalignant%2520Genome%2520Reprogramming%2520by%2520ATAD2%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2013%26volume%3D1829%26issue%3D10%26spage%3D1010%26epage%3D1014%26doi%3D10.1016%2Fj.bbagrm.2013.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banting, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barak, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ames, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnham, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kardel, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooch, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godbout, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermid, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiekhattar, R.</span></span> <span> </span><span class="NLM_article-title">CECR2, a Protein Involved in Neurulation, Forms a Novel Chromatin Remodeling Complex with SNF2L</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddi048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1093%2Fhmg%2Fddi048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=15640247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtV2qs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=513-524&issue=4&author=G.+S.+Bantingauthor=O.+Barakauthor=T.+M.+Amesauthor=A.+C.+Burnhamauthor=M.+D.+Kardelauthor=N.+S.+Coochauthor=C.+E.+Davidsonauthor=R.+Godboutauthor=H.+E.+McDermidauthor=R.+Shiekhattar&title=CECR2%2C+a+Protein+Involved+in+Neurulation%2C+Forms+a+Novel+Chromatin+Remodeling+Complex+with+SNF2L&doi=10.1093%2Fhmg%2Fddi048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">CECR2, a protein involved in neurulation, forms a novel chromatin remodeling complex with SNF2L</span></div><div class="casAuthors">Banting, Graham S.; Barak, Orr; Ames, Tanya M.; Burnham, Amanda C.; Kardel, Melanie D.; Cooch, Neil S.; Davidson, Courtney E.; Godbout, Roseline; McDermid, Heather E.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">513-524</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Chromatin remodeling complexes play crit. roles in development.  Here we describe a transcription factor, CECR2, which is involved in neurulation and chromatin remodeling.  CECR2 shows complex alternative splicing, but all variants contain DDT and bromodomain motifs.  A mutant mouse line was generated from an embryonic stem cell line contg. a genetrap within Cecr2.  Reporter gene expression demonstrated Cecr2 expression to be predominantly neural in the embryo.  Mice homozygous for the Cecr2 genetrap-induced mutation show a high penetrance of the neural tube defect exencephaly, the human equiv. of anencephaly, in a strain-dependent fashion.  Biochem. isolation of CECR2 revealed the presence of this protein as a component of a novel heterodimeric complex termed CECR2-contg. remodeling factor (CERF).  CERF comprises CECR2 and the ATP-dependent chromatin remodeler SNF2L, a mammalian ISWI ortholog expressed predominantly in the central nervous system.  CERF is capable of remodeling chromatin in vitro and displays an ATP hydrolyzing activity that is stimulated by nucleosomes.  Together, these data identify a novel chromatin remodeling complex with a crit. role in neurulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVKYqBa3GTk7Vg90H21EOLACvtfcHk0lgwzZMudySgZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtV2qs7g%253D&md5=88583cf64ca64c079bb5b88b1e3d9c51</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddi048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddi048%26sid%3Dliteratum%253Aachs%26aulast%3DBanting%26aufirst%3DG.%2BS.%26aulast%3DBarak%26aufirst%3DO.%26aulast%3DAmes%26aufirst%3DT.%2BM.%26aulast%3DBurnham%26aufirst%3DA.%2BC.%26aulast%3DKardel%26aufirst%3DM.%2BD.%26aulast%3DCooch%26aufirst%3DN.%2BS.%26aulast%3DDavidson%26aufirst%3DC.%2BE.%26aulast%3DGodbout%26aufirst%3DR.%26aulast%3DMcDermid%26aufirst%3DH.%2BE.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DCECR2%252C%2520a%2520Protein%2520Involved%2520in%2520Neurulation%252C%2520Forms%2520a%2520Novel%2520Chromatin%2520Remodeling%2520Complex%2520with%2520SNF2L%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2005%26volume%3D14%26issue%3D4%26spage%3D513%26epage%3D524%26doi%3D10.1093%2Fhmg%2Fddi048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toyoshima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howie, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imakura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annis, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frazier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loboda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsley, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandori, C.</span></span> <span> </span><span class="NLM_article-title">Functional Genomics Identifies Therapeutic Targets for MYC-Driven Cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">9545</span>– <span class="NLM_lpage">9550</span>, <span class="refDoi"> DOI: 10.1073/pnas.1121119109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1073%2Fpnas.1121119109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=22623531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38Xptlahs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=9545-9550&issue=24&author=M.+Toyoshimaauthor=H.+L.+Howieauthor=M.+Imakuraauthor=R.+M.+Walshauthor=J.+E.+Annisauthor=A.+N.+Changauthor=J.+Frazierauthor=B.+N.+Chauauthor=A.+Lobodaauthor=P.+S.+Linsleyauthor=M.+A.+Clearyauthor=J.+R.+Parkauthor=C.+Grandori&title=Functional+Genomics+Identifies+Therapeutic+Targets+for+MYC-Driven+Cancer&doi=10.1073%2Fpnas.1121119109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Functional genomics identifies therapeutic targets for MYC-driven cancer</span></div><div class="casAuthors">Toyoshima, Masafumi; Howie, Heather L.; Imakura, Maki; Walsh, Ryan M.; Annis, James E.; Chang, Aaron N.; Frazier, Jason; Chau, Nelson; Loboda, Andrey; Linsley, Peter S.; Cleary, Michele A.; Park, Julie R.; Grandori, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9545-9550, S9545/1-S9545/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">MYC oncogene family members are broadly implicated in human cancers, yet are considered "undruggable" as they encode transcription factors.  MYC also carries out essential functions in proliferative tissues, suggesting that its inhibition could cause severe side effects.  We elected to identify synthetic lethal interactions with c-MYC overexpression (MYC-SL) in a collection of ∼3,300 druggable genes, using high-throughput siRNA screening.  Of 49 genes selected for follow-up, 48 were confirmed by independent retesting and approx. one-third selectively induced accumulation of DNA damage, consistent with enrichment in DNA-repair genes by functional annotation.  In addn., genes involved in histone acetylation and transcriptional elongation, such as TRRAP and BRD4, were identified, indicating that the screen revealed known MYC-assocd. pathways.  For in vivo validation we selected CSNK1e, a kinase whose expression correlated with MYCN amplification in neuroblastoma (an established MYC-driven cancer).  Using RNAi and available small-mol. inhibitors, we confirmed that inhibition of CSNK1e halted growth of MYCN-amplified neuroblastoma xenografts.  CSNK1e had previously been implicated in the regulation of developmental pathways and circadian rhythms, whereas our data provide a previously unknown link with oncogenic MYC.  Furthermore, expression of CSNKle correlated with c-MYC and its transcriptional signature in other human cancers, indicating potential broad therapeutic implications of targeting CSNK1e function.  In summary, through a functional genomics approach, pathways essential in the context of oncogenic MYC but not to normal cells were identified, thus revealing a rich therapeutic space linked to a previously "undruggable" oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr60vylCRVnA7Vg90H21EOLACvtfcHk0lgwzZMudySgZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xptlahs7Y%253D&md5=32a3095f78c37c4a3a7b8878089d6f77</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1121119109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1121119109%26sid%3Dliteratum%253Aachs%26aulast%3DToyoshima%26aufirst%3DM.%26aulast%3DHowie%26aufirst%3DH.%2BL.%26aulast%3DImakura%26aufirst%3DM.%26aulast%3DWalsh%26aufirst%3DR.%2BM.%26aulast%3DAnnis%26aufirst%3DJ.%2BE.%26aulast%3DChang%26aufirst%3DA.%2BN.%26aulast%3DFrazier%26aufirst%3DJ.%26aulast%3DChau%26aufirst%3DB.%2BN.%26aulast%3DLoboda%26aufirst%3DA.%26aulast%3DLinsley%26aufirst%3DP.%2BS.%26aulast%3DCleary%26aufirst%3DM.%2BA.%26aulast%3DPark%26aufirst%3DJ.%2BR.%26aulast%3DGrandori%26aufirst%3DC.%26atitle%3DFunctional%2520Genomics%2520Identifies%2520Therapeutic%2520Targets%2520for%2520MYC-Driven%2520Cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D24%26spage%3D9545%26epage%3D9550%26doi%3D10.1073%2Fpnas.1121119109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, J.</span></span> <span> </span><span class="NLM_article-title">Genome-Wide Screen of Human Bromodomain-Containing Proteins identifies Cecr2 as a Novel DNA Damage Response Protein</span>. <i>Mol. Cells</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1007/s10059-012-0112-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1007%2Fs10059-012-0112-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=22699752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFSqsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2012&pages=85-91&issue=1&author=S.-K.+Leeauthor=E.-J.+Parkauthor=H.-S.+Leeauthor=Y.+S.+Leeauthor=J.+Kwon&title=Genome-Wide+Screen+of+Human+Bromodomain-Containing+Proteins+identifies+Cecr2+as+a+Novel+DNA+Damage+Response+Protein&doi=10.1007%2Fs10059-012-0112-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide screen of human bromodomain-containing proteins identifies Cecr2 as a novel DNA damage response protein</span></div><div class="casAuthors">Lee, Seul-Ki; Park, Eun-Jung; Lee, Han-Sae; Lee, Ye Seul; Kwon, Jongbum</div><div class="citationInfo"><span class="NLM_cas:title">Molecules and Cells</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-91</span>CODEN:
                <span class="NLM_cas:coden">MOCEEK</span>;
        ISSN:<span class="NLM_cas:issn">1016-8478</span>.
    
            (<span class="NLM_cas:orgname">Korean Society for Molecular and Cellular Biology</span>)
        </div><div class="casAbstract">The formation of γ-H2AX foci after DNA double strand breaks (DSBs) is crucial for the cellular response to this lethal DNA damage.  We previously have shown that BRG1, a chromatin remodeling enzyme, facilitates DSB repair by stimulating γ-H2AX formation, and this function of BRG1 requires the binding of BRGI to acetylated histone H3 on γ-H2AX-contg. nucleosomes using its bromodomain (BRD), a protein module that specifically recognizes acetyl-Lys moieties.  We also have shown that the BRD of BRG1, when ectopically expressed in cells, functions as a dominant neg. inhibitor of the BRG1 activity to stimulate γ-H2AX and DSB repair.  Here, we found that BRDs from a select group of proteins have no such activity, suggesting that the γ-H2AX inhibition activity of BRG1 BRD is specific.  This finding led us to search for more BRDs that exhibit γ-H2AX inhibition activity in the hope of finding addnl. BRD-contg. proteins involved in DNA damage responses.  We screened a total of 52 individual BRDs present in 38 human BRD-contg. proteins, comprising 93% of all human BRDs.  We identified the BRD of cat eye syndrome chromosome region candidate 2 (Cecr2), which recently was shown to form a novel chromatin remodeling complex with unknown cellular functions, as having a strong γ-H2AX inhibition activity.  This activity of Cecr2 BRD is specific because it depends on the chromatin binding affinity of Cecr2 BRD.  Small interfering RNA knockdown expts. showed that Cecr2 is important for γ-H2AX formation and DSB repair.  Therefore, our genomewide screen identifies Cecr2 as a novel DNA damage response protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEipOhHZMKBLVg90H21EOLACvtfcHk0ljGH-MA0AGL4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFSqsrjO&md5=bf381774000bd4e65dfcc1033c1b1a29</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2Fs10059-012-0112-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10059-012-0112-4%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.-K.%26aulast%3DPark%26aufirst%3DE.-J.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DLee%26aufirst%3DY.%2BS.%26aulast%3DKwon%26aufirst%3DJ.%26atitle%3DGenome-Wide%2520Screen%2520of%2520Human%2520Bromodomain-Containing%2520Proteins%2520identifies%2520Cecr2%2520as%2520a%2520Novel%2520DNA%2520Damage%2520Response%2520Protein%26jtitle%3DMol.%2520Cells%26date%3D2012%26volume%3D34%26issue%3D1%26spage%3D85%26epage%3D91%26doi%3D10.1007%2Fs10059-012-0112-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&issue=17&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+Analysis+and+Structural+Classification+of+Bromodomain+Acetyl-lysine+Binding+Sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0ljGH-MA0AGL4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520Analysis%2520and%2520Structural%2520Classification%2520of%2520Bromodomain%2520Acetyl-lysine%2520Binding%2520Sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26issue%3D17%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, H. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ATAD2 in Cancer: A Pharmacologically Challenging but Tractable Target</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1406921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1080%2F14728222.2018.1406921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=29148850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=85-96&issue=1&author=M.+Hussainauthor=Y.+Zhouauthor=Y.+Songauthor=H.+M.+A.+Hameedauthor=H.+Jiangauthor=Y.+Tuauthor=J.+Zhang&title=ATAD2+in+Cancer%3A+A+Pharmacologically+Challenging+but+Tractable+Target&doi=10.1080%2F14728222.2018.1406921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 in cancer: a pharmacologically challenging but tractable target</span></div><div class="casAuthors">Hussain, Muzammal; Zhou, Yang; Song, Yu; Hameed, H. M. Adnan; Jiang, Hao; Tu, Yaoquan; Zhang, Jiancun</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-96</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  ATAD2 protein is an emerging oncogene that has strongly been linked to the etiol. of multiple advanced human cancers.  Therapeutically, despite the fact that genetic suppression/knockdown studies have validated it as a compelling drug target for future therapeutic development, recent druggability assessment data suggest that direct targeting of ATAD2's bromodomain (BRD) may be a very challenging task.  ATAD2's BRD has been predicted as a 'difficult to drug' or 'least druggable' target due to the concern that its binding pocket, and the areas around it, seem to be unfeasible for ligand binding.: In this review, after shedding light on the multifaceted roles of ATAD2 in normal physiol. as well as in cancer-etiol., we discuss tech. challenges rendered by ATAD2's BRD active site and the recent drug discovery efforts to find small mol. inhibitors against it.: The identification of a novel low-nanomolar semi-permeable chem. probe against ATAD2's BRD by recent drug discovery campaign has demonstrated it to be a pharmacol. tractable target.  Nevertheless, the development of high quality bioavailable inhibitors against ATAD2 is still a pending task.  Moreover, ATAD2 may also potentially be utilized as a promising target for future development of RNAi-based therapy to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0bdigfdvMjbVg90H21EOLACvtfcHk0ljMZNPFjuz5hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN&md5=5d0ff7c936bb9eff125efeeaa4d275d1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1406921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1406921%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DHameed%26aufirst%3DH.%2BM.%2BA.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DTu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DATAD2%2520in%2520Cancer%253A%2520A%2520Pharmacologically%2520Challenging%2520but%2520Tractable%2520Target%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26issue%3D1%26spage%3D85%26epage%3D96%26doi%3D10.1080%2F14728222.2018.1406921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Approaches Towards Identification of Fragments for the Low-Druggability ATAD2 Bromodomain</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1848</span>, <span class="refDoi"> DOI: 10.1039/C4MD00237G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1039%2FC4MD00237G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGqtr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1843-1848&issue=12&author=A.+Chaikuadauthor=A.+M.+Petrosauthor=O.+Fedorovauthor=J.+Xuauthor=S.+Knapp&title=Structure-Based+Approaches+Towards+Identification+of+Fragments+for+the+Low-Druggability+ATAD2+Bromodomain&doi=10.1039%2FC4MD00237G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based approaches towards identification of fragments for the low-druggability ATAD2 bromodomain</span></div><div class="casAuthors">Chaikuad, Apirat; Petros, Andrew M.; Fedorov, Oleg; Xu, Jing; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1843-1848</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The transcriptional co-regulator ATAD2 is a prognostic marker for patient survival in many cancers.  ATAD2 harbours a bromodomain which may offer an opportunity for pharmacol. intervention, but its shallow, polar binding surface makes the development of inhibitors challenging.  Here we optimized crystal transfer/soaking conditions enabling crystallog. fragment screening.  We describe nine crystal structures of fragments including thymidine, a novel acetyl-lysine mimetic ligand and the evaluation of the binding properties of the identified fragments using NMR chem. shift perturbation expts.  The presented binding modes offer chem. starting points for the development of more potent ATAD2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqBr9VBOWv17Vg90H21EOLACvtfcHk0ljMZNPFjuz5hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGqtr3F&md5=52a30d24141c11b63efbe8b5b0ecb04c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1039%2FC4MD00237G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00237G%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-Based%2520Approaches%2520Towards%2520Identification%2520of%2520Fragments%2520for%2520the%2520Low-Druggability%2520ATAD2%2520Bromodomain%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1843%26epage%3D1848%26doi%3D10.1039%2FC4MD00237G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmons, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouzon, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5649</span>– <span class="NLM_lpage">5673</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5649-5673&issue=14&author=E.+H.+Demontauthor=C.-w.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=C.+Wellawayauthor=N.+Barrettauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=A.+J.+Emmonsauthor=E.+J.+Jonesauthor=B.+V.+Karamshiauthor=K.+Lockeauthor=D.+J.+Mitchellauthor=B.+H.+Mouzonauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watsonauthor=P.+Bamborough&title=Fragment-Based+Discovery+of+Low-Micromolar+ATAD2+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Demont, Emmanuel H.; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Wellaway, Chris; Barrett, Nathalie; Bridges, Angela M.; Craggs, Peter D.; Diallo, Hawa; Dixon, David P.; Douault, Clement; Emmons, Amanda J.; Jones, Emma J.; Karamshi, Bhumika V.; Locke, Kelly; Mitchell, Darren J.; Mouzon, Bernadette H.; Prinjha, Rab K.; Roberts, Andy D.; Sheppard, Robert J.; Watson, Robert J.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5649-5673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Overexpression of ATAD2 (ATPase family, AAA domain contg. 2) has been linked to disease severity and progression in a wide range of cancers, and is implicated in the regulation of several drivers of cancer growth.  Little is known of the dependence of these effects upon the ATAD2 bromodomain, which has been categorized as among the least tractable of its class.  The absence of any potent, selective inhibitors limits clear understanding of the therapeutic potential of the bromodomain.  Here, the authors describe the discovery of a hit from a fragment-based targeted array.  Optimization of this produced the first known micromolar inhibitors of the ATAD2 bromodomain, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5pJPrVexJbVg90H21EOLACvtfcHk0liguHPrIkhLJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP&md5=c133ce439365cd4e4110230827a7a0a4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00772%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DN.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DEmmons%26aufirst%3DA.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%2BV.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DMouzon%26aufirst%3DB.%2BH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-Based%2520Discovery%2520of%2520Low-Micromolar%2520ATAD2%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D14%26spage%3D5649%26epage%3D5673%26doi%3D10.1021%2Facs.jmedchem.5b00772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&issue=15&author=P.+Bamboroughauthor=C.-w.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.-M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-Based+Optimization+of+Naphthyridones+into+Potent+ATAD2+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0liguHPrIkhLJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-Based%2520Optimization%2520of%2520Naphthyridones%2520into%2520Potent%2520ATAD2%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D15%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe for the ATAD2 Bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">11382</span>– <span class="NLM_lpage">11386</span>, <span class="refDoi"> DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&issue=38&author=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+Chemical+Probe+for+the+ATAD2+Bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0lihcf4JrxoJDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Chemical%2520Probe%2520for%2520the%2520ATAD2%2520Bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26issue%3D38%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Díaz-Sáez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&issue=11&author=A.+E.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+St%C3%B6ckigtauthor=A.+ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+D%C3%ADaz-S%C3%A1ezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=J.+H%C3%BCbnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Isoform-Selective+ATAD2+Chemical+Probe+with+Novel+Chemical+Structure+and+Unusual+Mode+of+Action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span></div><div class="casAuthors">Fernandez-Montalvan, Amaury E.; Berger, Markus; Kuropka, Benno; Koo, Seong Joo; Badock, Volker; Weiske, Joerg; Puetter, Vera; Holton, Simon J.; Stoeckigt, Detlef; ter Laak, Antonius; Centrella, Paolo A.; Clark, Matthew A.; Dumelin, Christoph E.; Sigel, Eric A.; Soutter, Holly H.; Troast, Dawn M.; Zhang, Ying; Cuozzo, John W.; Keefe, Anthony D.; Roche, Didier; Rodeschini, Vincent; Chaikuad, Apirat; Diaz-Saez, Laura; Bennett, James M.; Fedorov, Oleg; Huber, Kilian V. M.; Huebner, Jan; Weinmann, Hilmar; Hartung, Ingo V.; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2730-2736</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 (ANCCA) is an epigenetic regulator and transcriptional cofactor, whose overexpression has been linked to the progress of various cancer types.  Here, we report a DNA-encoded library screen leading to the discovery of BAY-850, a potent and isoform selective inhibitor that specifically induces ATAD2 bromodomain dimerization and prevents interactions with acetylated histones in vitro, as well as with chromatin in cells.  These features qualify BAY-850 as a chem. probe to explore ATAD2 biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCB3nyG9YvubVg90H21EOLACvtfcHk0lihcf4JrxoJDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL&md5=1f09000a7d7e60e30540fbbae117acfe</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DSt%25C3%25B6ckigt%26aufirst%3DD.%26aulast%3Dter%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DD%25C3%25ADaz-S%25C3%25A1ez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DH%25C3%25BCbner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DIsoform-Selective%2520ATAD2%2520Chemical%2520Probe%2520with%2520Novel%2520Chemical%2520Structure%2520and%2520Unusual%2520Mode%2520of%2520Action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D11%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous Targeting of Bromodomains by Bromosporine Identifies BET Proteins as Master Regulators of Primary Transcription Response in Leukemia</span>. <i>Science Advances</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">e1600760</span>, <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=e1600760&issue=10&author=S.+Picaudauthor=K.+Leonardsauthor=J.-P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.-Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.-C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+Targeting+of+Bromodomains+by+Bromosporine+Identifies+BET+Proteins+as+Master+Regulators+of+Primary+Transcription+Response+in+Leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0ljgGeLgy78VYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.-Y.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520Targeting%2520of%2520Bromodomains%2520by%2520Bromosporine%2520Identifies%2520BET%2520Proteins%2520as%2520Master%2520Regulators%2520of%2520Primary%2520Transcription%2520Response%2520in%2520Leukemia%26jtitle%3DScience%2520Advances%26date%3D2016%26volume%3D2%26issue%3D10%26spage%3De1600760%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><a href="https://www.thesgc.org/chemical-probes/NVS-1" class="extLink">https://www.thesgc.org/chemical-probes/NVS-1</a>. (accessed Mar 22, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FNVS-1.+%28accessed+Mar+22%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><a href="https://www.thesgc.org/chemical-probes/TP-238" class="extLink">https://www.thesgc.org/chemical-probes/TP-238</a>. (accessed Mar 22, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.thesgc.org%2Fchemical-probes%2FTP-238.+%28accessed+Mar+22%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&issue=11&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=Diving+into+the+Water%3A+Inducible+Binding+Conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+Bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0lgKGE3R311ikQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520into%2520the%2520Water%253A%2520Inducible%2520Binding%2520Conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D11%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bommi-Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">GNE-886: A Potent and Selective Inhibitor of the Cat Eye Syndrome Chromosome Region Candidate 2 Bromodomain (CECR2)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">737</span>– <span class="NLM_lpage">741</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00132</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00132" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=737-741&issue=7&author=T.+D.+Crawfordauthor=J.+E.+Audiaauthor=S.+Bellonauthor=D.+J.+Burdickauthor=A.+Bommi-Reddyauthor=A.+C%C3%B4t%C3%A9author=R.+T.+Cummingsauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+Hewittauthor=H.-R.+Huangauthor=H.+Jayaramauthor=Y.+Jiangauthor=S.+Joshiauthor=J.+R.+Kieferauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=A.+Neissauthor=E.+Pardoauthor=F.+A.+Romeroauthor=P.+Sandyauthor=R.+J.+Simsauthor=Y.+Tangauthor=A.+M.+Taylorauthor=V.+Tsuiauthor=J.+Wangauthor=S.+Wangauthor=Y.+Wangauthor=Z.+Xuauthor=L.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=A.+G.+Cochran&title=GNE-886%3A+A+Potent+and+Selective+Inhibitor+of+the+Cat+Eye+Syndrome+Chromosome+Region+Candidate+2+Bromodomain+%28CECR2%29&doi=10.1021%2Facsmedchemlett.7b00132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">GNE-886: A Potent and Selective Inhibitor of the Cat Eye Syndrome Chromosome Region Candidate 2 Bromodomain (CECR2)</span></div><div class="casAuthors">Crawford, Terry D.; Audia, James E.; Bellon, Steve; Burdick, Daniel J.; Bommi-Reddy, Archana; Cote, Alexandre; Cummings, Richard T.; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael; Huang, Hon-Ren; Jayaram, Hariharan; Jiang, Ying; Joshi, Shivangi; Kiefer, James R.; Murray, Jeremy; Nasveschuk, Christopher G.; Neiss, Arianne; Pardo, Eneida; Romero, F. Anthony; Sandy, Peter; Sims, Robert J.; Tang, Yong; Taylor, Alexander M.; Tsui, Vickie; Wang, Jian; Wang, Shumei; Wang, Yongyun; Xu, Zhaowu; Zawadzke, Laura; Zhu, Xiaoqin; Albrecht, Brian K.; Magnuson, Steven R.; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">737-741</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. function of bromodomains, epigenetic readers of acetylated lysine residues, remains largely unknown.  Herein we report our efforts to discover a potent and selective inhibitor of the bromodomain of cat eye syndrome chromosome region candidate 2 (CECR2).  Screening of our internal medicinal chem. collection led to the identification of a pyrrolopyridone chem. lead, and subsequent structure-based drug design led to a potent and selective CECR2 bromodomain inhibitor (GNE-886) suitable for use as an in vitro tool compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxtzb-gQeFAbVg90H21EOLACvtfcHk0lh54oULsRpbWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Shsrw%253D&md5=f92e17923136477fc36edefd3bdd4998</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00132%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DBommi-Reddy%26aufirst%3DA.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DGNE-886%253A%2520A%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520the%2520Cat%2520Eye%2520Syndrome%2520Chromosome%2520Region%2520Candidate%25202%2520Bromodomain%2520%2528CECR2%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26issue%3D7%26spage%3D737%26epage%3D741%26doi%3D10.1021%2Facsmedchemlett.7b00132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogány, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">7506</span>– <span class="NLM_lpage">7525</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFeqsL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7506-7525&issue=16&author=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=D.+P.+Dixonauthor=P.+Francisauthor=R.+C.+Furzeauthor=P.+Grandiauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=K.+Lockeauthor=S.+C.+C.+Lucasauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=P.+Pog%C3%A1nyauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=A.+M.+Roaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watson&title=A+Qualified+Success%3A+Discovery+of+a+New+Series+of+ATAD2+Bromodomain+Inhibitors+with+a+Novel+Binding+Mode+Using+High-Throughput+Screening+and+Hit+Qualification&doi=10.1021%2Facs.jmedchem.9b00673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Bridges, Angela M.; Craggs, Peter D.; Dixon, David P.; Francis, Peter; Furze, Rebecca C.; Grandi, Paola; Jones, Emma J.; Karamshi, Bhumika; Locke, Kelly; Lucas, Simon C. C.; Michon, Anne-Marie; Mitchell, Darren J.; Pogany, Peter; Prinjha, Rab K.; Rau, Christina; Roa, Ana Maria; Roberts, Andrew D.; Sheppard, Robert J.; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7506-7525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class.  Here, we describe the discovery of a new class of inhibitors by high-throughput screening and show how the difficulties encountered in establishing a screening triage capable of finding progressible hits were overcome by data-driven optimization.  Despite the prevalence of nonspecific hits and an exceptionally low progressible hit rate (0.001%), our optimized hit qualification strategy employing orthogonal biophys. methods enabled us to identify a single active series.  The compds. have a novel ATAD2 binding mode with noncanonical features including the displacement of all conserved water mols. within the active site and a halogen-bonding interaction.  In addn. to reporting this new series and preliminary structure-activity relationship, we demonstrate the value of diversity screening to complement the knowledge-based approach used in our previous ATAD2 work.  We also exemplify tactics that can increase the chance of success when seeking new chem. starting points for novel and less-tractable targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomDZyktXHrhbVg90H21EOLACvtfcHk0lh54oULsRpbWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFeqsL%252FJ&md5=8132ccb7f63c3300bd173c42cd2ac026</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00673%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DLucas%26aufirst%3DS.%2BC.%2BC.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPog%25C3%25A1ny%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRoa%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520Qualified%2520Success%253A%2520Discovery%2520of%2520a%2520New%2520Series%2520of%2520ATAD2%2520Bromodomain%2520Inhibitors%2520with%2520a%2520Novel%2520Binding%2520Mode%2520Using%2520High-Throughput%2520Screening%2520and%2520Hit%2520Qualification%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D16%26spage%3D7506%26epage%3D7525%26doi%3D10.1021%2Facs.jmedchem.9b00673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting Physical in Drug Discovery: A Contemporary Perspective on Solubility and Hydrophobicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">648</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&issue=15&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+Physical+in+Drug+Discovery%3A+A+Contemporary+Perspective+on+Solubility+and+Hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and ∼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0liS9HhNR_4CrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520Physical%2520in%2520Drug%2520Discovery%253A%2520A%2520Contemporary%2520Perspective%2520on%2520Solubility%2520and%2520Hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26issue%3D15%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting Physical in Drug Discovery II: The Impact of Chromatographic Hydrophobicity Measurements and Aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&issue=17&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+Physical+in+Drug+Discovery+II%3A+The+Impact+of+Chromatographic+Hydrophobicity+Measurements+and+Aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0liS9HhNR_4CrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520Physical%2520in%2520Drug%2520Discovery%2520II%253A%2520The%2520Impact%2520of%2520Chromatographic%2520Hydrophobicity%2520Measurements%2520and%2520Aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26issue%3D17%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palovich, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. J.</span></span> <span> </span><span class="NLM_article-title">Quality Guidelines for Oral Drug Candidates: Dose, Solubility and Lipophilicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1719</span>– <span class="NLM_lpage">1727</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.drudis.2016.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=27423371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1aisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1719-1727&issue=10&author=M.+K.+Baylissauthor=J.+Butlerauthor=P.+L.+Feldmanauthor=D.+V.+S.+Greenauthor=P.+D.+Leesonauthor=M.+R.+Palovichauthor=A.+J.+Taylor&title=Quality+Guidelines+for+Oral+Drug+Candidates%3A+Dose%2C+Solubility+and+Lipophilicity&doi=10.1016%2Fj.drudis.2016.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Quality guidelines for oral drug candidates: dose, solubility and lipophilicity</span></div><div class="casAuthors">Bayliss, Martin K.; Butler, James; Feldman, Paul L.; Green, Darren V. S.; Leeson, Paul D.; Palovich, Michael R.; Taylor, Anthony J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1719-1727</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In an attempt to seek increased understanding of compd. attributes that influence successful drug pipeline progression, GlaxoSmithKline's portfolio of oral candidates was compared with ref. sets of marketed oral drugs.  The approach differs from other attrition studies by explicitly focusing on choosing 'the right compd.' by applying relevant, exptl. derived properties.  The anal. led to four proposed compd. quality categories, created by combining specific criteria for three measures: dose, soly. and the property forecast index, a composite measure of lipophilicity using chromatog. detd. Log D and aromaticity.  The 'three properties' provide benchmarked guidelines for project teams to use when seeking and selecting clin. candidates, because they reflect the property distribution of marketed oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRMpjV3EZ_4rVg90H21EOLACvtfcHk0lixx--3yMqTXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1aisb%252FK&md5=d9d145b1ee399b6ad54cdc55fff43c48</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DBayliss%26aufirst%3DM.%2BK.%26aulast%3DButler%26aufirst%3DJ.%26aulast%3DFeldman%26aufirst%3DP.%2BL.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DPalovich%26aufirst%3DM.%2BR.%26aulast%3DTaylor%26aufirst%3DA.%2BJ.%26atitle%3DQuality%2520Guidelines%2520for%2520Oral%2520Drug%2520Candidates%253A%2520Dose%252C%2520Solubility%2520and%2520Lipophilicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26issue%3D10%26spage%3D1719%26epage%3D1727%26doi%3D10.1016%2Fj.drudis.2016.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pease, J. H.</span></span> <span> </span><span class="NLM_article-title">A High Throughput Solubility Assay for Drug Discovery using Microscale Shake-Flask and Rapid UHPLC–UV–CLND Quantification</span>. <i>J. Pharm. Biomed. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">140</span>, <span class="refDoi"> DOI: 10.1016/j.jpba.2016.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.jpba.2016.01.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=26855285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGhurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2016&pages=126-140&author=B.+Linauthor=J.+H.+Pease&title=A+High+Throughput+Solubility+Assay+for+Drug+Discovery+using+Microscale+Shake-Flask+and+Rapid+UHPLC%E2%80%93UV%E2%80%93CLND+Quantification&doi=10.1016%2Fj.jpba.2016.01.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A high throughput solubility assay for drug discovery using microscale shake-flask and rapid UHPLC-UV-CLND quantification</span></div><div class="casAuthors">Lin, Baiwei; Pease, Joseph H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">126-140</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The rapid detn. of key phys. properties of lead compds. is essential to the drug discovery process.  Soly. is one of the most important properties since good soly. is needed not only for obtaining reliable in vitro and in vivo assay results in early discovery but also to ensure sufficient concn. of the drug being in circulation to get the desired therapeutic exposure at the target of interest.  In order for medicinal chemists to tune soly. of lead compds., a rapid assay is needed to provide soly. data that is accurate and predictive so that it can be reliably used for designing the next generation of compds. with improved properties.  To ensure speed and data quality, we developed a high throughput soly. assay that utilizes a single calibration UHPLC-UV-CLND method and a 24 h shake-flask format for rapid quantification.  A set of 46 model compds. was used to demonstrate that the method is accurate, reproducible and predictive.  Here we present development of the assay, including evaluation of quantification method, filtration membranes, equil. times, DMSO concns., and buffer conditions.  A comparison of thermodn. soly. results to our high throughput 24 h shake-flask soly. assay results is also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpojPpYm9G6NrVg90H21EOLACvtfcHk0lixx--3yMqTXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGhurg%253D&md5=98415728ed8254d173301d5f2c85e3f8</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.jpba.2016.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpba.2016.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DB.%26aulast%3DPease%26aufirst%3DJ.%2BH.%26atitle%3DA%2520High%2520Throughput%2520Solubility%2520Assay%2520for%2520Drug%2520Discovery%2520using%2520Microscale%2520Shake-Flask%2520and%2520Rapid%2520UHPLC%25E2%2580%2593UV%25E2%2580%2593CLND%2520Quantification%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2016%26volume%3D122%26spage%3D126%26epage%3D140%26doi%3D10.1016%2Fj.jpba.2016.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kestranek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenberger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Placko, S.</span></span> <span> </span><span class="NLM_article-title">Chemiluminescent Nitrogen Detection (CLND) to Measure Kinetic Aqueous Solubility</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch130145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2F9780470559277.ch130145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=24391097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A280%3ADC%252BC2czitF2itw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=269-280&author=A.+Kestranekauthor=A.+Chervenakauthor=J.+Longenbergerauthor=S.+Placko&title=Chemiluminescent+Nitrogen+Detection+%28CLND%29+to+Measure+Kinetic+Aqueous+Solubility&doi=10.1002%2F9780470559277.ch130145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chemiluminescent nitrogen detection (CLND) to measure kinetic aqueous solubility</span></div><div class="casAuthors">Kestranek Aimee; Chervenak Andrew; Longenberger Justin; Placko Steven</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in chemical biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">269-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Solubility is the dose-limiting property for in vitro studies, and therefore is a critical physicochemical property to measure in drug discovery.  Solubility data can be used to guide lead optimization, troubleshoot erratic bioassay results, and identify potential downstream liabilities such as insufficient solubility for bioassays or oral bioavailability.  Typically, early in vitro studies are performed using library compounds prepared as dimethylsulfoxide (DMSO) stock solutions, resulting in in vitro test solutions containing DMSO at low concentration (<5% v/v).  Since DMSO can affect the apparent solubility, it is desirable to obtain solubility data under conditions mimicking the in vitro study.  Kinetic solubility (from DMSO stock solutions) is often preferred over thermodynamic solubility (from dry powder) in early drug discovery.  The protocols in this article describe a general procedure for assessing kinetic aqueous solubility of early drug discovery compounds using a miniaturized shake flask method with chemiluminescent nitrogen detection (CLND).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQa5zXGR-PJU_SAs_PE1QvifW6udTcc2eYEo6XPvciRgbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2czitF2itw%253D%253D&md5=d63d6d57c4ff0934379d3c37f82eb1a0</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch130145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch130145%26sid%3Dliteratum%253Aachs%26aulast%3DKestranek%26aufirst%3DA.%26aulast%3DChervenak%26aufirst%3DA.%26aulast%3DLongenberger%26aufirst%3DJ.%26aulast%3DPlacko%26aufirst%3DS.%26atitle%3DChemiluminescent%2520Nitrogen%2520Detection%2520%2528CLND%2529%2520to%2520Measure%2520Kinetic%2520Aqueous%2520Solubility%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D5%26spage%3D269%26epage%3D280%26doi%3D10.1002%2F9780470559277.ch130145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span> <span> </span><span class="NLM_article-title">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&issue=7&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+High+Throughput+Screening%3A+Parallel+Artificial+Membrane+Permeation+Assay+in+the+Description+of+Passive+Absorption+Processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0lgZWOD21zPa5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520High%2520Throughput%2520Screening%253A%2520Parallel%2520Artificial%2520Membrane%2520Permeation%2520Assay%2520in%2520the%2520Description%2520of%2520Passive%2520Absorption%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26issue%3D7%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollerton, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besley, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boughtflower, B. J.</span></span> <span> </span><span class="NLM_article-title">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1772</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.6b04060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b04060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2017&pages=1772-1777&issue=3&author=M.+W.+Robinsonauthor=A.+P.+Hillauthor=S.+A.+Readshawauthor=J.+C.+Hollertonauthor=R.+J.+Uptonauthor=S.+M.+Lynnauthor=S.+C.+Besleyauthor=B.+J.+Boughtflower&title=Use+of+Calculated+Physicochemical+Properties+to+Enhance+Quantitative+Response+When+Using+Charged+Aerosol+Detection&doi=10.1021%2Facs.analchem.6b04060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span></div><div class="casAuthors">Robinson, Max W.; Hill, Alan P.; Readshaw, Simon A.; Hollerton, John C.; Upton, Richard J.; Lynn, Sean M.; Besley, Steve C.; Boughtflower, Bob J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1772-1777</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Universal quant. detection without the need for analyte ref. stds. would offer substantial benefits in many areas of anal. science.  The quant. capability of high performance liq. chromatog. (HPLC) with charged aerosol detection (CAD) was investigated for 50 compds. with a wide range of phys. and chem. properties.  It is widely believed that CAD is a mass detector.  Quantification of the 50 compds. using a generic calibrant and mass calibration achieved an av. error of 11.4% relative to 1H NMR.  Correction factors are proposed that est. the relative surface area of particles in the detector, taking into account the effects of the d. and charge of analytes.  Performing these corrections and quantifying with surface area calibration, rather than mass, shows considerably improved linearity and uniformity of detection, reducing the av. error relative to 1H NMR to 7.1%.  The accuracy of CAD quantification was most significantly improved for highly dense compds., with traditional mass calibration showing an av. error of 34.7% and the newly proposed surface area calibration showing an av. error of 5.8%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxWCD0M2yQRLVg90H21EOLACvtfcHk0lgZWOD21zPa5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK&md5=46c11b7527157b76f1fbc8eaf5cef75a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b04060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b04060%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DM.%2BW.%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DReadshaw%26aufirst%3DS.%2BA.%26aulast%3DHollerton%26aufirst%3DJ.%2BC.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26aulast%3DLynn%26aufirst%3DS.%2BM.%26aulast%3DBesley%26aufirst%3DS.%2BC.%26aulast%3DBoughtflower%26aufirst%3DB.%2BJ.%26atitle%3DUse%2520of%2520Calculated%2520Physicochemical%2520Properties%2520to%2520Enhance%2520Quantitative%2520Response%2520When%2520Using%2520Charged%2520Aerosol%2520Detection%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26issue%3D3%26spage%3D1772%26epage%3D1777%26doi%3D10.1021%2Facs.analchem.6b04060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Myrianthopoulos, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaboriaud-Kolar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grigoriou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownlee, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanior, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mexia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagratuni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastritis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terpos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaltsounis, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downs, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikros, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of a Selective Ligand for the Switch/Sucrose Nonfermenting-Related Bromodomains of Polybromo Protein-1 by the Use of Virtual Screening and Hydration Analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">8787</span>– <span class="NLM_lpage">8803</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8787-8803&issue=19&author=V.+Myrianthopoulosauthor=N.+Gaboriaud-Kolarauthor=C.+Tallantauthor=M.-L.+Hallauthor=S.+Grigoriouauthor=P.+M.+Brownleeauthor=O.+Fedorovauthor=C.+Rogersauthor=D.+Heidenreichauthor=M.+Waniorauthor=N.+Drososauthor=N.+Mexiaauthor=P.+Savitskyauthor=T.+Bagratuniauthor=E.+Kastritisauthor=E.+Terposauthor=P.+Filippakopoulosauthor=S.+M%C3%BCllerauthor=A.-L.+Skaltsounisauthor=J.+A.+Downsauthor=S.+Knappauthor=E.+Mikros&title=Discovery+and+Optimization+of+a+Selective+Ligand+for+the+Switch%2FSucrose+Nonfermenting-Related+Bromodomains+of+Polybromo+Protein-1+by+the+Use+of+Virtual+Screening+and+Hydration+Analysis&doi=10.1021%2Facs.jmedchem.6b00355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of a Selective Ligand for the Switch/Sucrose Nonfermenting-Related Bromodomains of Polybromo Protein-1 by the Use of Virtual Screening and Hydration Analysis</span></div><div class="casAuthors">Myrianthopoulos, Vassilios; Gaboriaud-Kolar, Nicolas; Tallant, Cynthia; Hall, Michelle-Lynn; Grigoriou, Stylianos; Brownlee, Peter Moore; Fedorov, Oleg; Rogers, Catherine; Heidenreich, David; Wanior, Marek; Drosos, Nikolaos; Mexia, Nikitia; Savitsky, Pavel; Bagratuni, Tina; Kastritis, Efstathios; Terpos, Evangelos; Filippakopoulos, Panagis; Muller, Susanne; Skaltsounis, Alexios-Leandros; Downs, Jessica Ann; Knapp, Stefan; Mikros, Emmanuel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8787-8803</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic interaction domains currently recognized as emerging drug targets for development of anticancer or anti-inflammatory agents.  In this study, development of a selective ligand of the fifth BRD of Polybromo protein-1 (PB1(5)) related to Switch/Sucrose Non-Fermenting (SWI/SNF) chromatin remodeling complexes is presented.  A compd. collection was evaluated by consensus virtual screening and a hit was identified.  The biophys. study of protein-ligand interactions was performed using x-ray crystallog. and isothermal titrn. calorimetry.  Collective data supported the hypothesis that affinity improvement could be achieved by enhancing interactions of the complex with the solvent.  The derived SAR along with free energy calcns. and a consensus hydration anal. using WaterMap and SZmap algorithms guided rational design of a set of novel analogs.  The most potent analog demonstrated high affinity of 3.3 μM and an excellent selectivity profile, thus comprising a promising lead for the development of chem. probes targeting PB1(5).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrMsChShl81rVg90H21EOLACvtfcHk0liAE-uoBfLXyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtLrL&md5=026dcc048ad31b7bf37c9a41b7562018</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00355%26sid%3Dliteratum%253Aachs%26aulast%3DMyrianthopoulos%26aufirst%3DV.%26aulast%3DGaboriaud-Kolar%26aufirst%3DN.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DHall%26aufirst%3DM.-L.%26aulast%3DGrigoriou%26aufirst%3DS.%26aulast%3DBrownlee%26aufirst%3DP.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DWanior%26aufirst%3DM.%26aulast%3DDrosos%26aufirst%3DN.%26aulast%3DMexia%26aufirst%3DN.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DBagratuni%26aufirst%3DT.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DTerpos%26aufirst%3DE.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSkaltsounis%26aufirst%3DA.-L.%26aulast%3DDowns%26aufirst%3DJ.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMikros%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520a%2520Selective%2520Ligand%2520for%2520the%2520Switch%252FSucrose%2520Nonfermenting-Related%2520Bromodomains%2520of%2520Polybromo%2520Protein-1%2520by%2520the%2520Use%2520of%2520Virtual%2520Screening%2520and%2520Hydration%2520Analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D19%26spage%3D8787%26epage%3D8803%26doi%3D10.1021%2Facs.jmedchem.6b00355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherell, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siejka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenton, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span> <span> </span><span class="NLM_article-title">Identification and Development of 2,3-Dihydropyrrolo[1,2-a]quinazolin-5(1H)-one Inhibitors Targeting Bromodomains within the Switch/Sucrose Nonfermenting Complex</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">5095</span>– <span class="NLM_lpage">5101</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5095-5101&issue=10&author=C.+L.+Sutherellauthor=C.+Tallantauthor=O.+P.+Monteiroauthor=C.+Yappauthor=J.+E.+Fuchsauthor=O.+Fedorovauthor=P.+Siejkaauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=J.+D.+Brentonauthor=P.+E.+Brennanauthor=S.+V.+Ley&title=Identification+and+Development+of+2%2C3-Dihydropyrrolo%5B1%2C2-a%5Dquinazolin-5%281H%29-one+Inhibitors+Targeting+Bromodomains+within+the+Switch%2FSucrose+Nonfermenting+Complex&doi=10.1021%2Facs.jmedchem.5b01997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Development of 2,3-Dihydropyrrolo[1,2-a]quinazolin-5(1H)-one Inhibitors Targeting Bromodomains within the Switch/Sucrose Nonfermenting Complex</span></div><div class="casAuthors">Sutherell, Charlotte L.; Tallant, Cynthia; Monteiro, Octovia P.; Yapp, Clarence; Fuchs, Julian E.; Fedorov, Oleg; Siejka, Paulina; Muller, Suzanne; Knapp, Stefan; Brenton, James D.; Brennan, Paul E.; Ley, Steven V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5095-5101</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain contg. proteins PB1, SMARCA4, and SMARCA2 are important components of SWI/SNF chromatin remodeling complexes.  We identified bromodomain inhibitors that target these proteins and display unusual binding modes involving water displacement from the KAc binding site.  The best compd. (I) binds the fifth bromodomain of PB1 with a KD of 124 nM, SMARCA2B and SMARCA4 with KD values of 262 and 417 nM, resp., and displays excellent selectivity over bromodomains other than PB1, SMARCA2, and SMARCA4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfnWlpl3TimLVg90H21EOLACvtfcHk0liAE-uoBfLXyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmsl2rt7s%253D&md5=a27afa2c97bd74462a09c9ec231f61ef</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01997%26sid%3Dliteratum%253Aachs%26aulast%3DSutherell%26aufirst%3DC.%2BL.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSiejka%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrenton%26aufirst%3DJ.%2BD.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DLey%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520and%2520Development%2520of%25202%252C3-Dihydropyrrolo%255B1%252C2-a%255Dquinazolin-5%25281H%2529-one%2520Inhibitors%2520Targeting%2520Bromodomains%2520within%2520the%2520Switch%252FSucrose%2520Nonfermenting%2520Complex%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D10%26spage%3D5095%26epage%3D5101%26doi%3D10.1021%2Facs.jmedchem.5b01997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castex, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Selective Targeting of the BRG/PB1 Bromodomains Impairs Embryonic and Trophoblast Stem Cell Maintenance</span>. <i>Science Advances</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">e1500723</span>, <span class="refDoi"> DOI: 10.1126/sciadv.1500723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1126%2Fsciadv.1500723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=26702435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=e1500723&issue=10&author=O.+Fedorovauthor=J.+Castexauthor=C.+Tallantauthor=D.+R.+Owenauthor=S.+Martinauthor=M.+Aldeghiauthor=O.+Monteiroauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=J.+D.+Trzupekauthor=B.+S.+Gerstenbergerauthor=C.+Bountraauthor=D.+Willmannauthor=C.+Wellsauthor=M.+Philpottauthor=C.+Rogersauthor=P.+C.+Bigginauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=R.+Sch%C3%BCleauthor=T.+G%C3%BCntherauthor=S.+Knappauthor=S.+M%C3%BCller&title=Selective+Targeting+of+the+BRG%2FPB1+Bromodomains+Impairs+Embryonic+and+Trophoblast+Stem+Cell+Maintenance&doi=10.1126%2Fsciadv.1500723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of the BRG/PB1 bromodomains impairs embryonic and trophoblast stem cell maintenance</span></div><div class="casAuthors">Fedorov, Oleg; Castex, Josefina; Tallant, Cynthia; Owen, Dafydd R.; Martin, Sarah; Aldeghi, Matteo; Monteiro, Octovia; Filippakopoulos, Panagis; Picaud, Sarah; Trzupek, John D.; Gerstenberger, Brian S.; Bountra, Chas; Willmann, Dominica; Wells, Christopher; Philpott, Martin; Rogers, Catherine; Biggin, Philip C.; Brennan, Paul E.; Bunnage, Mark E.; Schule, Roland; Gunther, Thomas; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1500723/1-e1500723/11</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF [also called Brg/Brahma-assocd. factors (BAFs)] are evolutionarily conserved chromatin remodeling complexes regulating gene transcription programs during development and stem cell differentiation.  BAF complexes contain an ATP (ATP)-driven remodeling enzyme (either BRG1 or BRM) and multiple protein interaction domains including bromodomains, an evolutionary conserved acetyl lysine-dependent protein interaction motif that recruits transcriptional regulators to acetylated chromatin.  We report a potent and cell active protein interaction inhibitor, PFI-3, that selectively binds to essential BAF bromodomains.  The high specificity of PFI-3 was achieved on the basis of a novel binding mode of a salicylic acid head group that led to the replacement of water mols. typically maintained in other bromodomain inhibitor complexes.  We show that exposure of embryonic stem cells to PFI-3 led to deprivation of stemness and deregulated lineage specification.  Furthermore, differentiation of trophoblast stem cells in the presence of PFI-3 was markedly enhanced.  The data present a key function of BAF bromodomains in stem cell maintenance and differentiation, introducing a novel versatile chem. probe for studies on acetylation-dependent cellular processes controlled by BAF remodeling complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiGyzl98dGOLVg90H21EOLACvtfcHk0lgXRHemFzWVgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovFaktLw%253D&md5=52757e667cc5afeb4c3a151fc8ef3442</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1500723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1500723%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DCastex%26aufirst%3DJ.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DSelective%2520Targeting%2520of%2520the%2520BRG%252FPB1%2520Bromodomains%2520Impairs%2520Embryonic%2520and%2520Trophoblast%2520Stem%2520Cell%2520Maintenance%26jtitle%3DScience%2520Advances%26date%3D2015%26volume%3D1%26issue%3D10%26spage%3De1500723%26doi%3D10.1126%2Fsciadv.1500723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Analysis of Water Stability in Bromodomain Binding Pockets with Grand Canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">19</span>, <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=29863194" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=19&issue=1&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-Scale+Analysis+of+Water+Stability+in+Bromodomain+Binding+Pockets+with+Grand+Canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-Scale%2520Analysis%2520of%2520Water%2520Stability%2520in%2520Bromodomain%2520Binding%2520Pockets%2520with%2520Grand%2520Canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26issue%3D1%26spage%3D19%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&issue=1&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0ljniz9rfz3XXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26issue%3D1%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET—A Novel BRET Platform for the Analysis of Protein–Protein Interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&issue=8&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+M%C3%A9ndezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET%E2%80%94A+Novel+BRET+Platform+for+the+Analysis+of+Protein%E2%80%93Protein+Interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">NanoBRET-A Novel BRET Platform for the Analysis of Protein-Protein Interactions</span></div><div class="casAuthors">Machleidt, Thomas; Woodroofe, Carolyn C.; Schwinn, Marie K.; Mendez, Jacqui; Robers, Matthew B.; Zimmerman, Kris; Otto, Paul; Daniels, Danette L.; Kirkland, Thomas A.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1797-1804</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dynamic interactions between proteins comprise a key mechanism for temporal control of cellular function and thus hold promise for development of novel drug therapies.  It remains tech. challenging, however, to quant. characterize these interactions within the biol. relevant context of living cells.  Although, bioluminescence resonance energy transfer (BRET) has often been used for this purpose, its general applicability has been hindered by limited sensitivity and dynamic range.  We have addressed this by combining an extremely bright luciferase (Nanoluc) with a means for tagging intracellular proteins with a long-wavelength fluorophore (HaloTag).  The small size (19 kDa), high emission intensity, and relatively narrow spectrum (460 nm peak intensity) make Nanoluc luciferase well suited as an energy donor.  By selecting an efficient red-emitting fluorophore (635 nm peak intensity) for attachment onto the HaloTag, an overall spectral sepn. exceeding 175 nm was achieved.  This combination of greater light intensity with improved spectral resoln. results in substantially increased detection sensitivity and dynamic range over current BRET technologies.  Enhanced performance is demonstrated using several established model systems, as well as the ability to image BRET in individual cells.  The capabilities are further exhibited in a novel assay developed for analyzing the interactions of bromodomain proteins with chromatin in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-h9HlReNabVg90H21EOLACvtfcHk0ljniz9rfz3XXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D&md5=9f5f773b6cd14c40074a2ffbd6472008</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET%25E2%2580%2594A%2520Novel%2520BRET%2520Platform%2520for%2520the%2520Analysis%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of Chemical Inhibitors of Human Bromodomains</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">11625</span>– <span class="NLM_lpage">11668</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00205</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00205" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=11625-11668&issue=21&author=G.+Zhangauthor=S.+G.+Smithauthor=M.+Zhou&title=Discovery+of+Chemical+Inhibitors+of+Human+Bromodomains&doi=10.1021%2Facs.chemrev.5b00205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Chemical Inhibitors of Human Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11625-11668</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-guided drug design strategy has played an important role in the recent discovery and development of bromodomain inhibitors that has yielded a multitude of chemotypes as described in this review.  Going forward, we expect that this structure-based approach will continue to guide medicinal chemists in lead optimization of bromodomain inhibitors and facilitate the efforts to develop new inhibitors for less druggable bromodomains.  Collectively, these emerging new chem. inhibitors will empower our ongoing investigation to better understand the roles of bromodomain proteins in control of gene transcription in human biol. and disease pathol.  These small-mol. inhibitors can also be developed into new epigenetic therapies for new effective treatments of various human diseases including cancer and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHbsgfBguOabVg90H21EOLACvtfcHk0lgiOmy8DhA6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP&md5=4014c771384b4a768a80644af22c4485</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00205%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520Chemical%2520Inhibitors%2520of%2520Human%2520Bromodomains%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26issue%3D21%26spage%3D11625%26epage%3D11668%26doi%3D10.1021%2Facs.chemrev.5b00205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mispelaere-Canivet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spindler, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beslin, P.</span></span> <span> </span><span class="NLM_article-title">Pd2(dba)3/Xantphos-Catalyzed Cross-Coupling of Thiols and Aryl Bromides/Triflates</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">5253</span>– <span class="NLM_lpage">5259</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2005.03.078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.tet.2005.03.078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktFWgs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=5253-5259&issue=22&author=C.+Mispelaere-Canivetauthor=J.-F.+Spindlerauthor=S.+Perrioauthor=P.+Beslin&title=Pd2%28dba%293%2FXantphos-Catalyzed+Cross-Coupling+of+Thiols+and+Aryl+Bromides%2FTriflates&doi=10.1016%2Fj.tet.2005.03.078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Pd2(dba)3/Xantphos-catalyzed cross-coupling of thiols and aryl bromides/triflates</span></div><div class="casAuthors">Mispelaere-Canivet, Clotilde; Spindler, Jean-Francis; Perrio, Stephane; Beslin, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5253-5259</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The cross-coupling of aliph. and arom. thiols and aryl halides or triflates mediated by a Pd2(dba)3/Xantphos catalytic system in refluxing xylene afforded the corresponding aryl thioethers in good to excellent yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfbcJkhwf7l7Vg90H21EOLACvtfcHk0lgiOmy8DhA6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktFWgs7k%253D&md5=54027b2333f6deb1df375bd85b36ea71</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2005.03.078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2005.03.078%26sid%3Dliteratum%253Aachs%26aulast%3DMispelaere-Canivet%26aufirst%3DC.%26aulast%3DSpindler%26aufirst%3DJ.-F.%26aulast%3DPerrio%26aufirst%3DS.%26aulast%3DBeslin%26aufirst%3DP.%26atitle%3DPd2%2528dba%25293%252FXantphos-Catalyzed%2520Cross-Coupling%2520of%2520Thiols%2520and%2520Aryl%2520Bromides%252FTriflates%26jtitle%3DTetrahedron%26date%3D2005%26volume%3D61%26issue%3D22%26spage%3D5253%26epage%3D5259%26doi%3D10.1016%2Fj.tet.2005.03.078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pu, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christesen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ku, Y.-Y.</span></span> <span> </span><span class="NLM_article-title">A Simple and Highly Effective Oxidative Chlorination Protocol for the Preparation of Arenesulfonyl Chlorides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">418</span>– <span class="NLM_lpage">421</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2009.11.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.tetlet.2009.11.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=418-421&issue=2&author=Y.-M.+Puauthor=A.+Christesenauthor=Y.-Y.+Ku&title=A+Simple+and+Highly+Effective+Oxidative+Chlorination+Protocol+for+the+Preparation+of+Arenesulfonyl+Chlorides&doi=10.1016%2Fj.tetlet.2009.11.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A simple and highly effective oxidative chlorination protocol for the preparation of arenesulfonyl chlorides</span></div><div class="casAuthors">Pu, Yu-Ming; Christesen, Alan; Ku, Yi-Yin</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">418-421</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">2,4-Dichloro-5,5-dimethylhydantoin (DCDMH) was found to be a mild and efficient reagent for the direct oxidative conversion of sulfur compds. to the corresponding arenesulfonyl chlorides in good to excellent yields through oxidative chlorination.  The method is suitable for many types of sulfur substrates (thiols, disulfides, and benzylic sulfides).  The overall process is simple, practical, and it provides convenient access to a variety of aryl or heteroarylsulfonyl chlorides.  The mild reaction conditions and the broad substrate scope render this method attractive and complementary to existing syntheses of aryl or heteroarylsulfonyl chlorides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAa01G0iIlebVg90H21EOLACvtfcHk0lgiOmy8DhA6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsrvO&md5=99e2389d65e31bc57a2a3c740b59df98</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2009.11.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2009.11.047%26sid%3Dliteratum%253Aachs%26aulast%3DPu%26aufirst%3DY.-M.%26aulast%3DChristesen%26aufirst%3DA.%26aulast%3DKu%26aufirst%3DY.-Y.%26atitle%3DA%2520Simple%2520and%2520Highly%2520Effective%2520Oxidative%2520Chlorination%2520Protocol%2520for%2520the%2520Preparation%2520of%2520Arenesulfonyl%2520Chlorides%26jtitle%3DTetrahedron%2520Lett.%26date%3D2010%26volume%3D51%26issue%3D2%26spage%3D418%26epage%3D421%26doi%3D10.1016%2Fj.tetlet.2009.11.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, B.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. S.</span></span> <span> </span><span class="NLM_article-title">A New Solvent System for Efficient Synthesis of 1,2,3-Triazoles</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">29</span>),  <span class="NLM_fpage">5105</span>– <span class="NLM_lpage">5109</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2006.05.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1016%2Fj.tetlet.2006.05.079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD28Xmt1Ghtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2006&pages=5105-5109&issue=29&author=B.-Y.+Leeauthor=S.+R.+Parkauthor=H.+B.+Jeonauthor=K.+S.+Kim&title=A+New+Solvent+System+for+Efficient+Synthesis+of+1%2C2%2C3-Triazoles&doi=10.1016%2Fj.tetlet.2006.05.079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">A new solvent system for efficient synthesis of 1,2,3-triazoles</span></div><div class="casAuthors">Lee, Bo-Young; Park, So Ra; Jeon, Heung Bae; Kim, Kwan Soo</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">5105-5109</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The use of CH2Cl2 as a co-solvent with H2O in the copper(I)-catalyzed 1,3-dipolar cycloaddn. of org. azides and alkynes increased reaction rates and provided the corresponding 1,2,3-triazoles in excellent yields compared to other org. co-solvent systems.  Moreover, we applied this discovery to prep. the functional multivalent compd. in an excellent yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8pPjYb6ZERrVg90H21EOLACvtfcHk0lgo8GdX7VKXeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xmt1Ghtb0%253D&md5=8f452666e79d95c509209a1557910710</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2006.05.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2006.05.079%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DB.-Y.%26aulast%3DPark%26aufirst%3DS.%2BR.%26aulast%3DJeon%26aufirst%3DH.%2BB.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DA%2520New%2520Solvent%2520System%2520for%2520Efficient%2520Synthesis%2520of%25201%252C2%252C3-Triazoles%26jtitle%3DTetrahedron%2520Lett.%26date%3D2006%26volume%3D47%26issue%3D29%26spage%3D5105%26epage%3D5109%26doi%3D10.1016%2Fj.tetlet.2006.05.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boren, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokin, V. V.</span></span> <span> </span><span class="NLM_article-title">Ruthenium-Catalyzed Azide–Alkyne Cycloaddition: Scope and Mechanism</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span> (<span class="NLM_issue">28</span>),  <span class="NLM_fpage">8923</span>– <span class="NLM_lpage">8930</span>, <span class="refDoi"> DOI: 10.1021/ja0749993</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0749993" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1Wjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=8923-8930&issue=28&author=B.+C.+Borenauthor=S.+Narayanauthor=L.+K.+Rasmussenauthor=L.+Zhangauthor=H.+Zhaoauthor=Z.+Linauthor=G.+Jiaauthor=V.+V.+Fokin&title=Ruthenium-Catalyzed+Azide%E2%80%93Alkyne+Cycloaddition%3A+Scope+and+Mechanism&doi=10.1021%2Fja0749993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Ruthenium-Catalyzed Azide-Alkyne Cycloaddition: Scope and Mechanism</span></div><div class="casAuthors">Boren, Brant C.; Narayan, Sridhar; Rasmussen, Lars K.; Zhang, Li; Zhao, Haitao; Lin, Zhenyang; Jia, Guochen; Fokin, Valery V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">8923-8930</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic activity of ruthenium(II) complexes in azide-alkyne cycloaddns. were evaluated.  The [Cp*RuCl] complexes, such as Cp*RuCl(PPh3)2, Cp*RuCl(COD), and Cp*RuCl(NBD), were among the most effective catalysts.  In the presence of catalytic Cp*RuCl(PPh3)2 or Cp*RuCl(COD), primary and secondary azides react with a broad range of terminal alkynes contg. a range of functionalities selectively producing 1,5-disubstituted 1,2,3-triazoles; tertiary azides were significantly less reactive.  Both complexes also promote the cycloaddn. reactions of org. azides with internal alkynes, providing access to fully-substituted 1,2,3-triazoles.  The ruthenium-catalyzed azide-alkyne cycloaddn. (RuAAC) appears to proceed via oxidative coupling of the azide and alkyne reactants to give a six-membered ruthenacycle intermediate, in which the first new carbon-nitrogen bond is formed between the more electroneg. carbon of the alkyne and the terminal, electrophilic nitrogen of the azide.  This step is followed by reductive elimination, which forms the triazole product.  DFT calcns. support this mechanistic proposal and indicate that the reductive elimination step is rate-detg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_rfIeBuSgUbVg90H21EOLACvtfcHk0lgo8GdX7VKXeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1Wjtr8%253D&md5=8b5ba21060ad9d6984bba3e2587aa44d</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja0749993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0749993%26sid%3Dliteratum%253Aachs%26aulast%3DBoren%26aufirst%3DB.%2BC.%26aulast%3DNarayan%26aufirst%3DS.%26aulast%3DRasmussen%26aufirst%3DL.%2BK.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DJia%26aufirst%3DG.%26aulast%3DFokin%26aufirst%3DV.%2BV.%26atitle%3DRuthenium-Catalyzed%2520Azide%25E2%2580%2593Alkyne%2520Cycloaddition%253A%2520Scope%2520and%2520Mechanism%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26issue%3D28%26spage%3D8923%26epage%3D8930%26doi%3D10.1021%2Fja0749993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alekseev, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurkin, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yurovskaya, M. A.</span></span> <span> </span><span class="NLM_article-title">Use of the Graebe-Ullmann Reaction in the Synthesis of 8-Methyl-γ-carboline and Isomeric Aromatic Aza-γ-carbolines</span>. <i>Chem. Heterocycl. Compd.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1235</span>– <span class="NLM_lpage">1250</span>, <span class="refDoi"> DOI: 10.1007/s10593-012-1127-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1007%2Fs10593-012-1127-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslWqs7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=1235-1250&issue=8&author=R.+S.+Alekseevauthor=A.+V.+Kurkinauthor=M.+A.+Yurovskaya&title=Use+of+the+Graebe-Ullmann+Reaction+in+the+Synthesis+of+8-Methyl-%CE%B3-carboline+and+Isomeric+Aromatic+Aza-%CE%B3-carbolines&doi=10.1007%2Fs10593-012-1127-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Use of the Graebe-Ullmann reaction in the synthesis of 8-methyl-γ-carboline and isomeric aromatic aza-γ-carbolines</span></div><div class="casAuthors">Alekseev, R. S.; Kurkin, A. V.; Yurovskaya, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Heterocyclic Compounds (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1235-1250</span>CODEN:
                <span class="NLM_cas:coden">CHCCAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-3122</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Two variants of the Graebe-Ullmann reaction were used to obtain 8-methyl-5H-pyrido[4,3-b]indole (8-methyl-γ-carboline) and the conditions for this reaction were optimized.  The feasibility of this method was studied for the synthesis of a series of isomeric arom. aza-γ-carbolines from the corresponding 1-(pyridyl)-1H-1,2,3-triazolo[4,5-c]pyridines under thermal and microwave irradn. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUZ7wIiDj_37Vg90H21EOLACvtfcHk0liPWBgrZwkoag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslWqs7vF&md5=f9d69d5c1d822fc801f2c60607d4e0f6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs10593-012-1127-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10593-012-1127-7%26sid%3Dliteratum%253Aachs%26aulast%3DAlekseev%26aufirst%3DR.%2BS.%26aulast%3DKurkin%26aufirst%3DA.%2BV.%26aulast%3DYurovskaya%26aufirst%3DM.%2BA.%26atitle%3DUse%2520of%2520the%2520Graebe-Ullmann%2520Reaction%2520in%2520the%2520Synthesis%2520of%25208-Methyl-%25CE%25B3-carboline%2520and%2520Isomeric%2520Aromatic%2520Aza-%25CE%25B3-carbolines%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2012%26volume%3D48%26issue%3D8%26spage%3D1235%26epage%3D1250%26doi%3D10.1007%2Fs10593-012-1127-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yutilov, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolyar, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramyants, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izotova, N. P.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Ticlopidine Analogs Based on Spinaceamine and 2-Azaspinaceamine</span>. <i>Pharm. Chem. J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">243</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1023/A:1025382623339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1023%2FA%3A1025382623339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVGnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2003&pages=243-245&issue=5&author=Y.+M.+Yutilovauthor=N.+N.+Smolyarauthor=M.+G.+Abramyantsauthor=N.+P.+Izotova&title=Synthesis+of+Ticlopidine+Analogs+Based+on+Spinaceamine+and+2-Azaspinaceamine&doi=10.1023%2FA%3A1025382623339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Ticlopidine Analogs Based on Spinaceamine and 2-Azaspinaceamine</span></div><div class="casAuthors">Yutilov, Yu. M.; Smolyar, N. N.; Abramyants, M. G.; Izotova, N. P.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Chemistry Journal (Translation of Khimiko-Farmatsevticheskii Zhurnal)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">243-245</span>CODEN:
                <span class="NLM_cas:coden">PCJOAU</span>;
        ISSN:<span class="NLM_cas:issn">0091-150X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Consultants Bureau</span>)
        </div><div class="casAbstract">A series of ticlopidine analogs contg. imidazole or triazoles instead if the thiophene nucleus were prepd. from imidazopyridines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohmi6jGWLb5rVg90H21EOLACvtfcHk0liPWBgrZwkoag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVGnsr8%253D&md5=ff3a8e2120199b6f84751e2e65f3a5c8</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1023%2FA%3A1025382623339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1025382623339%26sid%3Dliteratum%253Aachs%26aulast%3DYutilov%26aufirst%3DY.%2BM.%26aulast%3DSmolyar%26aufirst%3DN.%2BN.%26aulast%3DAbramyants%26aufirst%3DM.%2BG.%26aulast%3DIzotova%26aufirst%3DN.%2BP.%26atitle%3DSynthesis%2520of%2520Ticlopidine%2520Analogs%2520Based%2520on%2520Spinaceamine%2520and%25202-Azaspinaceamine%26jtitle%3DPharm.%2520Chem.%2520J.%26date%3D2003%26volume%3D37%26issue%3D5%26spage%3D243%26epage%3D245%26doi%3D10.1023%2FA%3A1025382623339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lay, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carver, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenhough, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatfield, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brierley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span> <span> </span><span class="NLM_article-title">Cellular Target Engagement Approaches to Monitor Epigenetic Reader Domain Interactions</span>. <i>SLAS DISCOVERY: Advancing the Science of Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1177/2472555219896278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1177%2F2472555219896278" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=163-175&issue=2&author=A.+N.+Phillipouauthor=C.+S.+Layauthor=C.+E.+Carverauthor=C.+Messengerauthor=J.+P.+Evansauthor=A.+J.+Lewisauthor=L.+J.+Gordonauthor=M.+Mahmoodauthor=L.+A.+Greenhoughauthor=D.+Sammonauthor=A.+T.+Chengauthor=S.+Chakrabortyauthor=E.+J.+Jonesauthor=S.+C.+C.+Lucasauthor=K.+M.+Gatfieldauthor=D.+J.+Brierleyauthor=P.+D.+Craggs&title=Cellular+Target+Engagement+Approaches+to+Monitor+Epigenetic+Reader+Domain+Interactions&doi=10.1177%2F2472555219896278"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1177%2F2472555219896278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555219896278%26sid%3Dliteratum%253Aachs%26aulast%3DPhillipou%26aufirst%3DA.%2BN.%26aulast%3DLay%26aufirst%3DC.%2BS.%26aulast%3DCarver%26aufirst%3DC.%2BE.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DMahmood%26aufirst%3DM.%26aulast%3DGreenhough%26aufirst%3DL.%2BA.%26aulast%3DSammon%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DA.%2BT.%26aulast%3DChakraborty%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLucas%26aufirst%3DS.%2BC.%2BC.%26aulast%3DGatfield%26aufirst%3DK.%2BM.%26aulast%3DBrierley%26aufirst%3DD.%2BJ.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26atitle%3DCellular%2520Target%2520Engagement%2520Approaches%2520to%2520Monitor%2520Epigenetic%2520Reader%2520Domain%2520Interactions%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26date%3D2020%26volume%3D25%26issue%3D2%26spage%3D163%26epage%3D175%26doi%3D10.1177%2F2472555219896278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the BET Family Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3838</span>, <span class="refDoi"> DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&issue=11&author=C.-w.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.-M.+G.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=A.-B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+BET+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lheoyomtPsb5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520BET%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26issue%3D11%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molnar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1425-1439&issue=4&author=N.+H.+Theodoulouauthor=P.+Bamboroughauthor=A.+J.+Bannisterauthor=I.+Becherauthor=R.+A.+Bitauthor=K.+H.+Cheauthor=C.-w.+Chungauthor=A.+Dittmannauthor=G.+Drewesauthor=D.+H.+Drewryauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Leveridgeauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+Molnarauthor=S.+C.+Robsonauthor=N.+C.+O.+Tomkinsonauthor=T.+Kouzaridesauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Discovery+of+I-BRD9%2C+a+Selective+Cell+Active+Chemical+Probe+for+Bromodomain+Containing+Protein+9+Inhibition&doi=10.1021%2Facs.jmedchem.5b00256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span></div><div class="casAuthors">Theodoulou, Natalie H.; Bamborough, Paul; Bannister, Andrew J.; Becher, Isabelle; Bit, Rino A.; Che, Ka Hing; Chung, Chun-wa; Dittmann, Antje; Drewes, Gerard; Drewry, David H.; Gordon, Laurie; Grandi, Paola; Leveridge, Melanie; Lindon, Matthew; Michon, Anne-Marie; Molnar, Judit; Robson, Samuel C.; Tomkinson, Nicholas C. O.; Kouzarides, Tony; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1425-1439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetylation of histone lysine residues is one of the most well-studied post-translational modifications of chromatin, selectively recognized by bromodomain "reader" modules.  Inhibitors of the bromodomain and extra terminal domain (BET) family of bromodomains have shown profound anticancer and anti-inflammatory properties, generating much interest in targeting other bromodomain-contg. proteins for disease treatment.  Herein, we report the discovery of I-BRD9, the first selective cellular chem. probe for bromodomain-contg. protein 9 (BRD9).  I-BRD9 was identified through structure-based design, leading to greater than 700-fold selectivity over the BET family and 200-fold over the highly homologous bromodomain-contg. protein 7 (BRD7).  I-BRD9 was used to identify genes regulated by BRD9 in Kasumi-1 cells involved in oncol. and immune response pathways and to the best of our knowledge, represents the first selective tool compd. available to elucidate the cellular phenotype of BRD9 bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmjhksIDyTrVg90H21EOLACvtfcHk0ljb_V5cngQ4TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D&md5=4e8f6aa664797fe54772b64c81cf2a83</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMolnar%26aufirst%3DJ.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520I-BRD9%252C%2520a%2520Selective%2520Cell%2520Active%2520Chemical%2520Probe%2520for%2520Bromodomain%2520Containing%2520Protein%25209%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D4%26spage%3D1425%26epage%3D1439%26doi%3D10.1021%2Facs.jmedchem.5b00256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camurri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaramella, A.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">3716</span>– <span class="NLM_lpage">3722</span>, <span class="refDoi"> DOI: 10.1021/ac001388j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac001388j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=3716-3722&issue=15&author=G.+Camurriauthor=A.+Zaramella&title=High-Throughput+Liquid+Chromatography%2FMass+Spectrometry+Method+for+the+Determination+of+the+Chromatographic+Hydrophobicity+Index&doi=10.1021%2Fac001388j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Liquid Chromatography/Mass Spectrometry Method for the Determination of the Chromatographic Hydrophobicity Index</span></div><div class="casAuthors">Camurri, Giulio; Zaramella, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3716-3722</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A fast gradient reversed-phase liq. chromatog. (LC) method, using an acetonitrile gradient was developed to det. the chromatog. hydrophobicity index (CHI), as reported by Valco et al. (Anal. Chem. 1997, 69, 2022-2029).The anal. method provides retention times, based on UV detection at two different wavelengths, which then are converted into CHI values after calibration with a set of test compds.  The CHI of each compd. is measured at three different pH values, 2.0, 7.4, and 10.5; so using an 8-min gradient at each pH value one compd. can be analyzed in ∼24 min.  The aim of this work is to improve the throughput of the CHI screening using a LC/MS approach, so the application of the LC/MS technique is an extension of the LC/UV approach previously reported by Valco et al.  This approach allows contemporary injection of N compds. into the LC/MS system, the retention time of each compd. can be then extd. from the selected ion recording chromatograms.  The throughput of the existing screening method could be increased by N times, where N is the no. of compds. injected, so only three runs are needed to det. the CHI at three different pH values for a set of N compds.  The highest value of N depends on the total no. of channels that can be monitored simultaneously; 32 channels were used.  This LC/MS method was tested for a no. of com. products analyzed as mixts., and data obtained were compared with those coming from the classical LC/UV approach.  In the same way, the method was tested for a no. of compds. assocd. with two GlaxoWellcome projects in the antibacterial area.  The LC/MS method can be successfully applied for analyzing compds. in mixts. and for compds. with poor UV absorption, which cannot be analyzed with the std. LC/UV method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpqYuIDpy5rVg90H21EOLACvtfcHk0ljb_V5cngQ4TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvVCgsbY%253D&md5=8773c87e08ba7ddce3c2d679360a6220</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fac001388j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac001388j%26sid%3Dliteratum%253Aachs%26aulast%3DCamurri%26aufirst%3DG.%26aulast%3DZaramella%26aufirst%3DA.%26atitle%3DHigh-Throughput%2520Liquid%2520Chromatography%252FMass%2520Spectrometry%2520Method%2520for%2520the%2520Determination%2520of%2520the%2520Chromatographic%2520Hydrophobicity%2520Index%26jtitle%3DAnal.%2520Chem.%26date%3D2001%26volume%3D73%26issue%3D15%26spage%3D3716%26epage%3D3722%26doi%3D10.1021%2Fac001388j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunhuck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D. P.</span></span> <span> </span><span class="NLM_article-title">Fast Gradient HPLC Method to Determine Compounds Binding to Human Serum Albumin. Relationships with Octanol/Water and Immobilized Artificial Membrane Lipophilicity</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2236</span>– <span class="NLM_lpage">2248</span>, <span class="refDoi"> DOI: 10.1002/jps.10494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=10.1002%2Fjps.10494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=14603509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2003&pages=2236-2248&issue=11&author=K.+Valkoauthor=S.+Nunhuckauthor=C.+Bevanauthor=M.+H.+Abrahamauthor=D.+P.+Reynolds&title=Fast+Gradient+HPLC+Method+to+Determine+Compounds+Binding+to+Human+Serum+Albumin.+Relationships+with+Octanol%2FWater+and+Immobilized+Artificial+Membrane+Lipophilicity&doi=10.1002%2Fjps.10494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fast gradient HPLC method to determine compounds binding to human serum albumin. Relationships with octanol/water and immobilized artificial membrane lipophilicity</span></div><div class="casAuthors">Valko, Klara; Nunhuck, Shenaz; Bevan, Chris; Abraham, Michael H.; Reynolds, Derek P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2236-2248</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A fast gradient HPLC method (cycle time 15 min) has been developed to det. Human Serum Albumin (HSA) binding of discovery compds. using chem. bonded protein stationary phases.  The HSA binding values were derived from the gradient retention times that were converted to the logarithm of the equil. consts. (logK HSA) using data from a calibration set of mols.  The method has been validated using literature plasma protein binding data of 68 known drug mols.  The method is fully automated, and has been used for lead optimization in more than 20 company projects.  The HSA binding data obtained for more than 4000 compds. were suitable to set up global and project specific quant. structure binding relationships that helped compd. design in early drug discovery.  The obtained HSA binding of known drug mols. were compared to the Immobilizd Artificial Membrane binding data (CHI IAM) obtained by our previously described HPLC-based method.  The solvation equation approach has been used to characterize the normal binding ability of HSA, and this relationship shows that compd. lipophilicity is a significant factor.  It was found that the selectivity of the "baseline" lipophilicity governing HSA binding, membrane interaction, and octanol/water partition are very similar.  However, the effect of the presence of pos. or neg. charges have very different effects.  It was found that neg. charged compds. bind more strongly to HSA than it would be expected from the lipophilicity of the ionized species at pH 7.4.  Several compds. showed stronger HSA binding than can be expected from their lipophilicity alone, and comparison between predicted and exptl. binding affinity allows the identification of compds. that have good complementarities with any of the known binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1LUBLdv-t7Vg90H21EOLACvtfcHk0lgDjnOof79CSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFCrs7Y%253D&md5=14eb1e0f0ad616846a2b5d1428133fa9</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjps.10494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.10494%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DNunhuck%26aufirst%3DS.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DReynolds%26aufirst%3DD.%2BP.%26atitle%3DFast%2520Gradient%2520HPLC%2520Method%2520to%2520Determine%2520Compounds%2520Binding%2520to%2520Human%2520Serum%2520Albumin.%2520Relationships%2520with%2520Octanol%252FWater%2520and%2520Immobilized%2520Artificial%2520Membrane%2520Lipophilicity%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2003%26volume%3D92%26issue%3D11%26spage%3D2236%26epage%3D2248%26doi%3D10.1002%2Fjps.10494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55','PDB','6S55'); return false;">PDB: 6S55</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QSP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QSP','PDB','4QSP'); return false;">PDB: 4QSP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YB4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YB4','PDB','6YB4'); return false;">PDB: 6YB4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXB','PDB','3NXB'); return false;">PDB: 3NXB</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i126"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00021">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_42657"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00021?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00021</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">BRD2,3 and T TR-FRET data and DiscoverX BROMO<i>scan</i> bromodomain profiling of key compounds, a comparison of CLND and CAD solubility measurements, correlation between TR-FRET and DiscoverX BROMO<i>scan</i> TAF1(2) data, X-ray crystallographic data, chemistry experimental procedures, and selected NMR and LCMS spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_002.csv">CSV</a>)</p></li><li><p class="inline">Model of <b>(</b><i><b>S,S</b></i><b>)-32</b> docked into CECR2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_001.pdf">jm0c00021_si_001.pdf (753.09 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_002.csv">jm0c00021_si_002.csv (9.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00021/suppl_file/jm0c00021_si_003.pdb">jm0c00021_si_003.pdb (56.6 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The crystal structure of ATAD2 bound <b>(</b><i><b>R,S</b></i><b>)-32</b> has been deposited in the Protein Data Bank as <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YB4">6YB4</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00021&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00021%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-10" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00021" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d084397b3ca6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
