
<html lang="en"     class="pb-page"  data-request-id="e25f2624-6575-4d94-afdf-9b15e6e2b16b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00673;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-16;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification" /></meta><meta name="dc.Creator" content="Paul  Bamborough" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Emmanuel H.  Demont" /></meta><meta name="dc.Creator" content="Angela M.  Bridges" /></meta><meta name="dc.Creator" content="Peter D.  Craggs" /></meta><meta name="dc.Creator" content="David P.  Dixon" /></meta><meta name="dc.Creator" content="Peter  Francis" /></meta><meta name="dc.Creator" content="Rebecca C.  Furze" /></meta><meta name="dc.Creator" content="Paola  Grandi" /></meta><meta name="dc.Creator" content="Emma J.  Jones" /></meta><meta name="dc.Creator" content="Bhumika  Karamshi" /></meta><meta name="dc.Creator" content="Kelly  Locke" /></meta><meta name="dc.Creator" content="Simon C. C.  Lucas" /></meta><meta name="dc.Creator" content="Anne-Marie  Michon" /></meta><meta name="dc.Creator" content="Darren J.  Mitchell" /></meta><meta name="dc.Creator" content="Peter  Pogány" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Creator" content="Christina  Rau" /></meta><meta name="dc.Creator" content="Ana Maria  Roa" /></meta><meta name="dc.Creator" content="Andrew D.  Roberts" /></meta><meta name="dc.Creator" content="Robert J.  Sheppard" /></meta><meta name="dc.Creator" content="Robert J.  Watson" /></meta><meta name="dc.Description" content="The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class. Here, we describe the discovery of a new class of inhibitors by high-throughput screening and..." /></meta><meta name="Description" content="The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class. Here, we describe the discovery of a new class of inhibitors by high-throughput screening and..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 9, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00673" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00673" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00673" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00673" /></link>
        
    
    

<title>A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00673" /></meta><meta property="og:title" content="A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0015.jpeg" /></meta><meta property="og:description" content="The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class. Here, we describe the discovery of a new class of inhibitors by high-throughput screening and show how the difficulties encountered in establishing a screening triage capable of finding progressible hits were overcome by data-driven optimization. Despite the prevalence of nonspecific hits and an exceptionally low progressible hit rate (0.001%), our optimized hit qualification strategy employing orthogonal biophysical methods enabled us to identify a single active series. The compounds have a novel ATAD2 binding mode with noncanonical features including the displacement of all conserved water molecules within the active site and a halogen-bonding interaction. In addition to reporting this new series and preliminary structure–activity relationship, we demonstrate the value of diversity screening to complement the knowledge-based approach used in our previous ATAD2 work. We also exemplify tactics that can increase the chance of success when seeking new chemical starting points for novel and less-tractable targets." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00673"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00673">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00673&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00673&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00673&amp;href=/doi/10.1021/acs.jmedchem.9b00673" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 7506-7525</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00625" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00591" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Paul Bamborough</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Bamborough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#7f0f1e0a13511e511d1e121d100d100a18173f180c14511c1012"><span class="__cf_email__" data-cfemail="3848594d541659165a59555a574a574d5f50785f4b53165b5755">[email protected]</span></a> (P.B.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Bamborough">Paul Bamborough</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chun-wa Chung</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun-wa Chung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#cdaea5b8a3e0baace3a5e3aea5b8a3aa8daabea6e3aea2a0"><span class="__cf_email__" data-cfemail="3f5c574a5112485e1157115c574a51587f584c54115c5052">[email protected]</span></a> (C.-W.C).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Emmanuel H. Demont</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emmanuel H. Demont</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emmanuel+H.++Demont">Emmanuel H. Demont</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Angela M. Bridges</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Angela M. Bridges</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Angela+M.++Bridges">Angela M. Bridges</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter D. Craggs</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter D. Craggs</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+D.++Craggs">Peter D. Craggs</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David P. Dixon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David P. Dixon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+P.++Dixon">David P. Dixon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Francis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Francis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Francis">Peter Francis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rebecca C. Furze</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rebecca C. Furze</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Research Unit, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rebecca+C.++Furze">Rebecca C. Furze</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paola Grandi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Grandi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cellzome, Meyerhofstrasse 1, Heidelberg 69117, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Grandi">Paola Grandi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Emma J. Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emma J. Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emma+J.++Jones">Emma J. Jones</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bhumika Karamshi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bhumika Karamshi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bhumika++Karamshi">Bhumika Karamshi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kelly Locke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kelly Locke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kelly++Locke">Kelly Locke</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simon C. C. Lucas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon C. C. Lucas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon+C.+C.++Lucas">Simon C. C. Lucas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anne-Marie Michon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne-Marie Michon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cellzome, Meyerhofstrasse 1, Heidelberg 69117, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne-Marie++Michon">Anne-Marie Michon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darren J. Mitchell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darren J. Mitchell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darren+J.++Mitchell">Darren J. Mitchell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Pogány</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Pogány</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Pog%C3%A1ny">Peter Pogány</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rab K. Prinjha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rab K. Prinjha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Research Unit, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christina Rau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christina Rau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cellzome, Meyerhofstrasse 1, Heidelberg 69117, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christina++Rau">Christina Rau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ana Maria Roa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ana Maria Roa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GlaxoSmithKline Tres Cantos, 28760 Tres Cantos, Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ana+Maria++Roa">Ana Maria Roa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew D. Roberts</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew D. Roberts</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+D.++Roberts">Andrew D. Roberts</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert J. Sheppard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Sheppard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Research Unit, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Sheppard">Robert J. Sheppard</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Robert J. Watson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Watson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Science and Technology, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Watson">Robert J. Watson</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00673&amp;href=/doi/10.1021%2Facs.jmedchem.9b00673" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 7506–7525</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 9, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 April 2019</li><li><span class="item_label"><b>Published</b> online</span>9 August 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 August 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00673" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00673</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7506%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPaul%2BBamborough%252C%2BChun-wa%2BChung%252C%2BEmmanuel%2BH.%2BDemont%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D16%26contentID%3Dacs.jmedchem.9b00673%26title%3DA%2BQualified%2BSuccess%253A%2BDiscovery%2Bof%2Ba%2BNew%2BSeries%2Bof%2BATAD2%2BBromodomain%2BInhibitors%2Bwith%2Ba%2BNovel%2BBinding%2BMode%2BUsing%2BHigh-Throughput%2BScreening%2Band%2BHit%2BQualification%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7525%26publicationDate%3DAugust%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00673"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2325</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00673" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A Qualified Success: Discovery of a New Series of ATAD2 Bromodomain Inhibitors with a Novel Binding Mode Using High-Throughput Screening and Hit Qualification&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Bamborough&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Emmanuel&quot;,&quot;last_name&quot;:&quot;H. Demont&quot;},{&quot;first_name&quot;:&quot;Angela&quot;,&quot;last_name&quot;:&quot;M. Bridges&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;D. Craggs&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;P. Dixon&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Francis&quot;},{&quot;first_name&quot;:&quot;Rebecca&quot;,&quot;last_name&quot;:&quot;C. Furze&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Grandi&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;J. Jones&quot;},{&quot;first_name&quot;:&quot;Bhumika&quot;,&quot;last_name&quot;:&quot;Karamshi&quot;},{&quot;first_name&quot;:&quot;Kelly&quot;,&quot;last_name&quot;:&quot;Locke&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;C. C. Lucas&quot;},{&quot;first_name&quot;:&quot;Anne-Marie&quot;,&quot;last_name&quot;:&quot;Michon&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;J. Mitchell&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Pogány&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;},{&quot;first_name&quot;:&quot;Christina&quot;,&quot;last_name&quot;:&quot;Rau&quot;},{&quot;first_name&quot;:&quot;Ana&quot;,&quot;last_name&quot;:&quot;Maria Roa&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;D. Roberts&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Sheppard&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Watson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7506-7525&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00673&quot;},&quot;abstract&quot;:&quot;The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class. Here, we describe the discovery of a new class of inhibitors by high-throughput screening and show how the difficulties encountered in establishing a screening triage capable of finding progressible hits were overcome by data-driven optimization. Despite the prevalence of nonspecific hits and an exceptionally low progressible hit rate (0.001%), our optimized hit qualification strategy employing orthogonal biophysical methods enabled us to identify a single active series. The compounds have a novel ATAD2 binding mode with noncanonical features including the displacement of all conserved water molecules within the active site and a halogen-bonding interaction. In addition to reporting this new series and preliminary structure–activity relationship, we demonstrate the value of diversity screening to complement the knowledge-based approach used in our previous ATAD2 work. We also exemplify tactics that can incr&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00673&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00673" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00673&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00673" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00673&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00673" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00673&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00673&amp;href=/doi/10.1021/acs.jmedchem.9b00673" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00673" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00673" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00673%26sid%3Dliteratum%253Aachs%26pmid%3D31398032%26genre%3Darticle%26aulast%3DBamborough%26date%3D2019%26atitle%3DA%2BQualified%2BSuccess%253A%2BDiscovery%2Bof%2Ba%2BNew%2BSeries%2Bof%2BATAD2%2BBromodomain%2BInhibitors%2Bwith%2Ba%2BNovel%2BBinding%2BMode%2BUsing%2BHigh-Throughput%2BScreening%2Band%2BHit%2BQualification%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D16%26spage%3D7506%26epage%3D7525%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291651" title="Anions">Anions</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292119" title="Surface plasmon resonance">Surface plasmon resonance</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/jmcmar.2019.62.issue-16/20190822/jmcmar.2019.62.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The bromodomain of ATAD2 has proved to be one of the least-tractable proteins within this target class. Here, we describe the discovery of a new class of inhibitors by high-throughput screening and show how the difficulties encountered in establishing a screening triage capable of finding progressible hits were overcome by data-driven optimization. Despite the prevalence of nonspecific hits and an exceptionally low progressible hit rate (0.001%), our optimized hit qualification strategy employing orthogonal biophysical methods enabled us to identify a single active series. The compounds have a novel ATAD2 binding mode with noncanonical features including the displacement of all conserved water molecules within the active site and a halogen-bonding interaction. In addition to reporting this new series and preliminary structure–activity relationship, we demonstrate the value of diversity screening to complement the knowledge-based approach used in our previous ATAD2 work. We also exemplify tactics that can increase the chance of success when seeking new chemical starting points for novel and less-tractable targets.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35524" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35524" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Many drug discovery efforts focus on families or classes that have been studied before and are believed to be tractable to small-molecule inhibition (for example, kinases and G protein-coupled receptors). Although these are more likely to be druggable targets, it can still be challenging to find progressible hits against individual members of established ligandable protein classes. This is especially true if the aim is to find novel modes of action, perhaps to overcome issues of selectivity within the protein family. When novelty is a key objective, approaches that rely on prior knowledge of important ligand binding features (virtual screening, machine learning/quantitative structure–activity relationship (SAR) models, etc.), may be of limited utility. High-throughput screening (HTS), in which large and well-curated diverse compound libraries are screened, is unbiased by expectations and gives greater opportunities to find novel starting points and modes of action.</div><div class="NLM_p">The quality of chemical starting points is a significant factor that influences the chance of success of a drug discovery project. The importance of the physicochemical properties of leads and of remaining within druglike space during optimization is well established.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Sometimes, starting points for medicinal chemistry are suboptimal because no good small-molecule binding sites exist or the sites that do exist have a strong preference for chemotypes that are difficult to develop into drugs. These may be inevitable consequences of the choice of target. However, the quality of leads may also be suboptimal for avoidable reasons, for example, by poor choices made in the design and execution of the screening cascade. In such cases, good starting points may be hidden among a sea of assay hits operating through unprogressible mechanisms. Such mechanisms may be highly assay-dependent and include phenomena such as optical interference, polystoichiometry or aggregation, redox recycling, covalent modification, etc.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Substructural motifs associated with an increased probability of assay interference can provide a warning flag, but these only cover a fraction of all possible unprogressible mechanisms of inhibition.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">HTS assays are routinely optimized to ensure reproducibility, to yield the expected pharmacology with known control compounds, and to produce a low enough hit rate to avoid overwhelming downstream assays. As it is well known that all screening assays have artefactual positives, confidence to initiate a medicinal chemistry optimization effort has often been gained through a hit disqualification triage process (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), in which compounds with unwanted modes of action are sequentially removed from the hit list by selected computational and experimental counterscreens. The type and proportion of mechanistically unprogressible hits can be very reagent- and assay-dependent, so the most effective choice of counterscreens may be unclear at the start of a project. Furthermore, the number of hits acting through such mechanisms is often underestimated, and in our experience there are many cases where these greatly outnumber hits where the activity results from desirable mechanisms. When combined with low biological tractability, numerous unprogressible hits must be invalidated before any progressible ones are found. This takes time and effort, can undermine confidence in the target, and may even result in program termination.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Summary of key messages, and schematic of disqualification (left) vs qualification (right) triage approaches to hit confirmation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The fragment-based drug design community often employs an alternative positive selection strategy. Faced with the expensive and time-consuming optimization of weak hits from high-concentration screens, it is common to progress only those that come from the intersection of orthogonal biophysical screening methods. These techniques can now be used to test thousands of compounds, a scale compatible with the number of compounds typically tested in dose–response after an HTS.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Here, we demonstrate the power of using confirmation (orthogonal) screens after HTS, rather than counterscreens—a strategy of hit qualification, as opposed to hit disqualification (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Using a positive inclusion process, and deemphasizing assays intended to exclude possible unwanted mechanisms, we avoid unwarranted assumptions about what those mechanisms might be. Since compounds are progressed only if they demonstrate desirable attributes, after each assay, our knowledge of and confidence in the remaining hits increases. For example, direct interaction between the compound and target protein can be probed by label-free biophysical approaches such as NMR, surface plasmon resonance (SPR), affinity selection mass spectroscopy, or thermal shift assays (TSAs).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Similarly, sensible binding stoichiometry can be confirmed by native mass spectrometry, SPR, or isothermal calorimetry (ITC). This approach is particularly useful for less-tractable targets where the proportion of developable hits is small and the number of potential undesirable mechanisms is large. We also show the value of the routine use of hit qualification approaches before the screen itself to ensure the entire triage is fit for purpose.</div><div class="NLM_p">We illustrate the application of hit qualification here using the bromodomain of ATAD2 as an example. ATAD2 or AAA nuclear coregulator cancer-associated protein (ANCCA) is one of over 40 human bromodomain-containing proteins (BRDs). Highly expressed in a wide variety of unrelated cancers, increased expression levels correlate with poor prognosis and disease recurrence in patients.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10">(7−10)</a> Downregulation of its expression using siRNA has implicated ATAD2 in pathways including apoptosis, cell survival, proliferation, and migration.<a onclick="showRef(event, 'ref7 ref9 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref9 ref11 ref12 ref13">(7,9,11−13)</a> Drug discovery efforts for oncology have so far focused on the ATAD2 bromodomain rather than its ATPase domain,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> as bromodomains, especially the bromodomain and extraterminal (BET) subfamily, are generally considered to be tractable to most screening approaches.<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15−17)</a></div><div class="NLM_p">Compared to the more extensively studied BET family proteins, relatively few inhibitors of ATAD2 have so far been reported (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26">(18−26)</a> Although the KAc binding site of ATAD2 retains the features of typical bromodomains, computational predictions suggest that it is less amenable to small-molecule inhibition than others in the family.<a onclick="showRef(event, 'ref20 ref27'); return false;" href="javascript:void(0);" class="ref ref20 ref27">(20,27)</a> In our hands, ATAD2 has indeed been less tractable than other bromodomains, resulting in lower qualified hit rates from bromodomain-targeted and diverse fragment sets.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Some literature ATAD2 inhibitors and affinities reported using various techniques.<a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26">(18−26)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We have previously reported the discovery of a series of naphthyridone ATAD2 inhibitors. Screening a library of compounds designed to interact with the conserved acetyl-lysine (KAc) site of the bromodomain family led to GSK8814 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20−22)</a> This knowledge-based fragment approach drew upon historical and structural understanding of bromodomains and their inhibitors. Here, we will describe our concurrent attempt to find alternative, more unusual classes of ATAD2 bromodomain inhibitor by screening a large, diverse compound collection. We discuss the many challenges we faced and how they were diagnosed and overcome by data-driven choices, including our hit qualification triage. This led to the discovery of a single potent series out of the 1.7 million compounds screened.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21847" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21847" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Primary Screening Assay</h3><div class="NLM_p">At the start of the project, no tool compounds for ATAD2 were known, and the expression and purification of full-length ATAD2 had not been reported. A short bromodomain-containing construct (construct 1, ATAD2 981–1108) from the Structural Genomics Consortium formed the basis of the <sup>15</sup>N–<sup>1</sup>H heteronuclear single quantum coherence (HSQC) NMR and X-ray crystallography systems we developed.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Unfortunately, in early assay development trials, this protein showed temperature-dependent but reversible precipitation and batch-to-batch aggregation, which raised concerns about the suitability of this construct for high-throughput screening. Reasoning that neighboring regions of the sequence could be important for domain stability, we expressed and purified a longer sequence, 6H-Flag-tev-ATAD2 950–1148 (construct 2). This indeed provided a more robust reagent for SPR binding studies for both fragments and lead-sized compounds.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> When used to screen panels of modified histone peptides for binding partners (AltaBioscience), a 21-residue triacetylated histone peptide was identified with an affinity of ∼50 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S1a,b</a>, Supporting Information). This interaction was confirmed by isothermal titration calorimetry (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S1c</a>, Supporting Information). This peptide was subsequently used to configure an ATAD2 time-resolved fluorescence (TRF) energy transfer (TR-FRET) binding assay.</div><div class="NLM_p">A typical HTS triage is shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a. Before carrying out the ATAD2 screen, the reproducibility and effectiveness of every stage of the triage were assessed by passing a robustness set (a panel of 1408 diverse compounds representative of the wider screening collection, filtered to remove known frequent hitters and pan-assay interference compounds)<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> through the cascade from top to bottom. This set is large and diverse enough to provide an estimate of the expected number of compounds passing through all stages from the full HTS and to provide confidence that the capacity of the lower-throughput assays will not be overwhelmed. As the robustness set compounds were not designed to interact with ATAD2, our expectation was that most hits were likely to act through unprogressible mechanisms and therefore, perhaps counterintuitively, a low hit rate is highly desirable.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Typical HTS process at GSK. Following the primary screen at a single compound concentration and digital correction for plate patterns, typically ∼20 000 compounds are selected, replated, and the assay repeated in duplicate. If a second assay is available, either an orthogonal or an interference screen may be run in parallel. Following confirmation, a few thousand reproducible hits are normally progressed for full-curve IC<sub>50</sub> determination and then into hit qualification. Computational approaches (e.g., clustering, property prediction) may be applied at various stages, although over-reliance on this risks losing progressible hits. If the number of hits greatly exceeds compound-handling capacity (for example, in situations with a high, reproducible hit rate acting through nonprogressible mechanisms), then performing in silico filtering without losing the few progressible hits may be particularly challenging. (B) Refined ATAD2 HTS triage process as described below.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In reality, the robustness set gave a high hit rate in the 6H-Flag-tev-ATAD2 950–1148/peptide displacement TR-FRET assay. When screened at 10 μM, 4.1% of compounds were classed as active, at a high threshold of 50% inhibition (defined using a cutoff of 3 standard deviations (SDs)). If this translated into HTS, 70 000 hits would be generated, making it highly challenging to select a subset for progression through the lower-throughput triage stages. It was not viable to reduce the hit rate simply by raising the activity threshold: achieving even a modest reduction to a 2% hit rate would require a cutoff of nearly 80% inhibition, leading to the exclusion of many potentially potent hits. In general, raising the cutoff is not advisable because (a) single-concentration HTS inhibition may not extrapolate well to IC<sub>50</sub> curves and (b) the mode of action is undetermined at this stage and it is likely that progressible hits would be lost. A high primary hit rate is especially problematic if the primary screen is reproducible, in which case the confirmation stage is unlikely to deliver significant attrition (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a).</div><div class="NLM_p">Compound <b>1</b> was one representative hit from the robustness set, which was reproducibly active in dose–response in the TR-FRET assay with an IC<sub>50</sub> of ∼30 nM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). Noting its structural similarity to bromosporine (ATAD2 <i>K</i><sub>d</sub> 44 μM), we expected that <b>1</b> was more likely to show confirmed binding in hit qualification assays than other structurally novel hits.<a onclick="showRef(event, 'ref26 ref30'); return false;" href="javascript:void(0);" class="ref ref26 ref30">(26,30)</a> However, like the other hits from the robustness set, it showed no detectable binding to the bromodomain in direct binding assays such as <sup>15</sup>N–<sup>1</sup>H HSQC NMR (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b), suggesting that its TR-FRET activity must arise through an unwanted mechanism. In this case, chemical intuition was no substitute for direct experimental validation as a reliable way to select hits with desirable modes of action.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) TR-FRET dose–response curve for <b>1</b> in the 6H-Flag-tev-ATAD2(950–1148) peptide displacement assay. Compound <b>1</b> did not give fitted curves up to 10 μM in the optimized flag-6H-tev-ATAD2(981–1121) assay (data not shown). (B) <sup>15</sup>N–<sup>1</sup>H HSQC NMR spectrum (black in the presence of 100 μM <b>1</b> and red without) showing lack of evidence for compound binding.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">From the robustness set data, we concluded that although the longer 6H-Flag-tev-ATAD2 950–1148 protein was well behaved in some respects, it produced a high level of unprogressible positives that could not be confirmed by other means. Attempts to remove these unwanted hits by modifying assay conditions were unsuccessful (data not shown); so, we therefore turned to make alternative, more robust bromodomain constructs.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Alternative ATAD2 Bromodomain Constructs</h3><div class="NLM_p">Alternative ATAD2 bromodomain-containing constructs were designed using bioinformatics analysis of human and orthologue sequences, informed by the X-ray crystal structure of the 981–1108 core bromodomain module [Protein Data Bank (PDB) code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DAI">3DAI</a>] and limited proteolysis of 6H-Flag-tev-ATAD2 950–1148 (constructs 3–15, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S1</a>, Supporting Information).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Following small-scale expression trials and purification, eight constructs yielded soluble proteins (constructs 3–10, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) with good integrity and purity, as confirmed by peptide mass fingerprint and liquid chromatography/mass spectrometry (LC/MS). These were characterized by a range of approaches. An assessment of helical content and stability was made using thermal melting via circular dichroism (CD). CD spectra consistent with the expected bromodomain four-helical bundle were obtained (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S2</a>, Supporting Information). Most constructs had similar melting temperatures, while a designed hexamutant (construct 4) was more stable and another (construct 8) was biphasic. We note in passing that the design of the hexamutant construct was partly intended to remove cysteine residues, which we speculated could influence the stability of the bromodomain, including the neighboring pair Cys1057/Cys1079, an idea that has recently been discussed by others.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. TR-FRET Peptide Competition Binding Assay Results from Compounds Screened at 10 μM against Constructs 2–10 (See Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S1</a> for Full Data)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0016.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">CD summarizes the thermal melting curves in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S2</a>, Supporting Information. The robustness cutoff is the % inhibition cutoff required to give a 2% hit rate from a set of 1408 diverse compounds. The bromodomain toolset <i>r</i><sup>2</sup> column reports the correlation between <i>n</i> = 2 replicates from a bromodomain-targeted set of 998 compounds. Green = positive result; amber = warning; red = negative result.</p></div></div><div></div></div><div class="NLM_p">In parallel, the ability of each protein to bind unlabeled acetyl-lysine histone H4 peptide was confirmed by competition with biotinylated peptide in TR-FRET assays (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S3</a>, Supporting Information). These peptide displacement assays were then used to screen the 1408 compound robustness set at a 10 μM compound concentration. As mentioned above, this should contain few if any genuine ATAD2 binding molecules, so a low hit rate was desirable. Seven of the eight additional constructs 3–10 showed lower hit rates than construct 2 (6H-Flag-tev-ATAD2 950–1148; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">S4</a>). Three of the new constructs (3–5) had higher hit rates than the others and so were removed from further consideration. Interestingly, there was no agreement in the hits identified from the robustness set even between the well-behaved constructs 6 and 8–10 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">S5</a>), consistent with our assumption that these are unlikely to be specific progressible inhibitors.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. % inhibition responses at 10 μM from the diverse robustness set for ATAD2 construct 6 against (A) construct 2 and (B) construct 10. (C) Reproducibility of the 996 targeted bromodomain compounds screened in duplicate against ATAD2 construct 6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Constructs 6 and 10 also showed TR-FRET dose–response curves for three presumed tool compounds found in parallel activities and believed to be genuine inhibitors as these were consistently active in biophysical assays. We therefore tested a larger knowledge-based set of 996 compounds resembling known bromodomain inhibitors (the “bromodomain toolset”) at 10 μM against constructs 6 and 10 and found that construct 6, Flag-6H-tev-ATAD2 981–1121, gave more reproducible results than construct 10 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">S6</a>).</div><div class="NLM_p last">Since construct 6 also had a lower-predicted protein consumption, we opted to use this for our primary TR-FRET screen. The optimized form of this assay was subsequently used to identify our previously reported naphthyridinone series.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> HTS and Hit Qualification</h3><div class="NLM_p">A summary of the outcome of the TR-FRET HTS and triage is shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B. Compounds (1.7 million) were screened, and statistical plate pattern correction was carried out.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Hits were defined as those with inhibition greater than 3SD from the mean (∼29% inhibition) giving 9441 hits (a hit rate of 0.6%). Because of the small number, all could be progressed to confirmation by TR-FRET. The more potent and chemically attractive compounds were taken straight to dose–response, while the more borderline hits went first for duplicate retest at 10 μM. Most of the 3354 compounds screened in dose–response showed little concentration dependence and low maximum inhibition and so did not give fittable curves (IC<sub>50</sub> > 30 μM). Compounds (428) showing full or partial inhibition were progressed into hit qualification by orthogonal biophysical binding assays.</div><div class="NLM_p">Two independent assays to monitor the direct binding of compounds to the ATAD2 bromodomain were developed in parallel, as we were unsure whether either would be successful, and, in fact, each proved to be complementary to the other. One was <sup>15</sup>N–<sup>1</sup>H HSQC NMR using tag-cleaved <sup>15</sup>N ATAD2 981–1108, which we have described previously.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> After testing at a single concentration (100 μM), each compound was categorized as either binding or nonbinding, and any effects on protein aggregation were noted. The other technology was surface plasmon resonance (SPR), in which hits were tested in concentration–response up to 100 μM. Compounds were classed as binding, nonbinding, or linear binding (a nonsaturating, polystoichiometric response where the calculated <i>K</i><sub>d</sub> was greater than the highest assay concentration). The ability of SPR to screen multiple proteins simultaneously was also useful, as it enabled us to determine differential interaction with wild-type ATAD2 over a mutant bromodomain that lacks the essential acetyl-lysine binding motif (Y1021A/N1064A). The optimal construct for SPR was found to be construct 2 (6H-Flag-tev-ATAD2 950–1148). Construct 6 (6H-Flag-tev-ATAD2 981–1121) was also attempted, but this immobilized protein did not show saturating binding or differences between wild-type and mutant ATAD2 for tool compounds (data not shown).</div><div class="NLM_p">The vast majority of the HTS hits did not show progressible binding behavior in the biophysical assays. Only 26 of the 428 hits showed saturating binding behavior by SPR (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The others either did not bind at all or were linear binders. When the same 428 hits were classified independently by NMR, only 17 showed evidence of binding at 100 μM. Example SPR and NMR results for a compound showing saturating binding and for a linear binding hit are shown in Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figures S7 and S8</a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Hit Qualification Results from Biophysical Assays Showing the Number of Compounds Classified as Binding/Nonbinding by <sup>15</sup>N–<sup>1</sup>H HSQC NMR and as Binding/Linear Binding/Nonbinding by SPR</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0019.gif" alt="" id="GRAPHIC-d7e746-autogenerated" /></img><div></div></div><div class="NLM_p">Reassuringly, there was very good agreement between the NMR and SPR results, with 401/428 hits classified as either nonbinding or polystoichiometric by both techniques. Only one compound classified as binding by NMR failed to show saturating binding by SPR, and only 10 hits exhibiting saturating binding by SPR failed to show binding by NMR. The discrepant compounds had weak biochemical activity (TR-FRET IC<sub>50</sub> > 30 μM), so they may lie on the borderline of detectability within the two assays at the highest concentration tested (100 μM).</div><div class="NLM_p">Furthermore, for every linear SPR hit, the binding response with mutant ATAD2 was similar to that with wild type, suggesting that they did not bind at the acetyl-lysine site (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). This could mean that these compounds bind in an allosteric site, but more likely they may interact with the SPR chip or with sticky patches on the protein in a nonspecific manner. In contrast, all but one of the compounds classed as binders by NMR showed preferential SPR binding to wild-type over mutant ATAD2 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>), even if not all of these reached saturation at 100 μM.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. SPR binding responses for all hits for mutant ATAD2 (<i>X</i> axis) vs wild-type (<i>Y</i> axis). Hits were analyzed at 11 μM and signals normalized to correct for differences due to molecular weight. Hits are colored according to their <sup>15</sup>N–<sup>1</sup>H HSQC NMR binding class (red = binding, blue = nonbinding). All hits showing preferential binding to wild-type over mutant ATAD2 are the same as those that show binding by NMR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For additional characterization and to confirm binding to full-length endogenous ATAD2, we employed the bromosphere assay.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> This is a chemoproteomic assay measuring the competitive inhibition of endogenous ATAD2 from Hut-78 chromatin lysate using a bead-linked ATAD2 bromodomain partner, in this case, an acetylated lysine histone peptide. Thirty-four hits were chosen to test, mostly ones that were positive in the TR-FRET assay but nonbinding by biophysics. All compounds tested were inactive in the bromosphere assay, except for three, which were positive by SPR and NMR (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The three active compounds have lower potency in the bromosphere assay than in TR-FRET or SPR, possibly because of nonspecific binding to other components of the lysate or reduced affinity for full-length ATAD2.</div><div class="NLM_p">In summary, after extensive characterization, 16 hits from the 1.7 million screened were confirmed to be specific, saturating binders by both NMR and SPR. All of them belonged to a single series of phenylsulfonamides, which had been purchased several years previously to populate the HTS collection. An example, <b>2</b>, and its summarized biophysical data are shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>. Summary data for the rest of the hits is given in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Biophysical binding data for compound <b>2</b>. (A) SPR sensorgram and concentration−response curves for mutant (red) and wild-type (black) ATAD2 bromodomains. (B) <sup>15</sup>N–<sup>1</sup>H HSQC NMR spectrum (black in the presence of 100 μM <b>2</b> and red without).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. ATAD2 and BRD4 Potencies for Phenylsulfonamides Directly from the HTS</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0012.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Some results below the curve-fitting threshold could not be included in mean and SD. In these cases, the numbers included in the mean/number of times tested are both shown.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. ATAD2 SPR, <sup>15</sup>N–<sup>1</sup>H HSQC NMR, and Bromosphere Confirmation of HTS Hits</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center">SPR saturating binding</th><th class="colsep0 rowsep0" align="center">SPR p<i>K</i><sub>d</sub></th><th class="colsep0 rowsep0" align="center">NMR</th><th class="colsep0 rowsep0" align="center" char="±">bromosphere pIC<sub>50</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>2</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">5.7</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±">4.8 ± 0.02 (2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>3</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±">4.8 ± 0.05 (2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>4</b></td><td class="colsep0 rowsep0" align="left">yes but poor curve fit</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>5</b></td><td class="colsep0 rowsep0" align="left">yes but poor curve fit</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>6</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>7</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.8</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>8</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.8</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>9</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>10</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.8</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>11</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.9</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>12</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.7</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>13</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>14</b></td><td class="colsep0 rowsep0" align="left">yes but poor curve fit</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>15</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>16</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.5</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±">4.4 ± 0.09 (2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>17</b></td><td class="colsep0 rowsep0" align="left">Y</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">binding</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr></tbody></table></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Structure–Activity Relationships</h3><div class="NLM_p">The most potent qualified hits had single-digit micromolar IC<sub>50</sub> (pIC<sub>50</sub> 5.0–6.0) in the ATAD2 TR-FRET assay (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Selectivity against the BET family bromodomains is desirable to avoid the extensive pharmacology associated with their inhibition, and this was measured using TR-FRET fluoroligand displacement assays in mutant tandem BET bromodomains.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Below 50 μM, none of the phenylsulfonamides showed significant binding to BRD4, which we use as a representative of the rest of the BET family (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>; full data in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S2</a>, Supporting Information). ATAD2 SPR dissociation constants were consistent with the ATAD2 TR-FRET IC<sub>50</sub>s (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_p">The hits contained little variation at the R1 position, with only chloride and methyl substituents represented. The chloride analogues mostly had better potency; for example, chloride <b>2</b> is about 10-fold more potent than its direct methyl analogue <b>12</b>. Variants at the R2 position included simple cyclic and symmetrically disubstituted acyclic amines, with piperidine the most heavily represented. In contrast, a variety of larger chemically distinct substituents were tolerated at the R3 position.</div><div class="NLM_p">As the most potent example, we chose <b>2</b> for further investigation. Chiral separation generated two undefined stereoisomers, <b>18</b> and <b>19</b>, whose ATAD2 potency was indistinguishable from racemate <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). For the initial investigation of chemistry, we therefore opted to leave the racemic 4-methyl 4-cyclopropyl hydantoin group at R3, as well as the piperidine at R2 unchanged, and varied the R1 position by introducing small modifications.</div><div class="NLM_p">The chemistry to access the noncommercial compounds discussed in this article is described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Synthesis of the inhibitors started from sulfonyl chlorides <b>24</b>, which in most cases were commercially available (X = Cl, Br, Me, F) or easily accessible by chlorosulfonylation of the para-substituted nitrobenzine (X = ethyl, Cl).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The formation of the sulfonamide followed by reduction of the nitro group with Fe(0) gave aniline <b>25</b>, which could then be coupled with commercially available acid <b>26</b> to give compound <b>23</b> or with propionyl chloride to give compounds <b>29</b>, <b>34</b>, and <b>35</b>. It was then possible to obtain the trifluoromethyl derivatives <b>20</b> and <b>28</b> from bromide <b>23</b> or <b>35</b>, respectively, using methyl fluorosulfonyl difluoroacetate.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Phenol <b>30</b>, aniline <b>31</b>, or nitrile <b>32</b> could all be obtained from bromide <b>35</b> using either palladium- or copper-catalyzed chemistry.</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0011.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Phenysulfonamide Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) piperidine, NEt<sub>3</sub> or <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to room temperature, 37–97%; (b) Fe(0), NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O, 70 °C, 67–74%; (c) <b>26</b>, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU), DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 63%; (d) C<sub>2</sub>H<sub>5</sub>COCl, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to room temperature, 39–76%; (e) CH<sub>3</sub>CF<sub>2</sub>COOSO<sub>2</sub>F, CuI, dimethylformamide (DMF), 80 °C, 10–15%; (f) KOH, Pd<sub>2</sub>(dba)<sub>3</sub>, <i>t</i>BuXPhos, dioxane/H<sub>2</sub>O, 100 °C, 21%; (g) <span class="smallcaps smallerCapital">l</span>-proline, CuI, K<sub>2</sub>CO<sub>3</sub>, 25% w/w aqueous ammonia, 110 °C, 22%; (h) CuCN, <i>N</i>-methyl-2-pyrrolidone (NMP), 160 °C, 62%.</p></p></figure><div class="NLM_p">The R1 = CF<sub>3</sub> derivative (<b>20</b>) was roughly equipotent to the methyl analogue <b>12</b> but less active than chloride <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Increasing the size of the R1 group to methoxy, <b>21</b>, abolished measurable binding, as did replacing R1 with hydrogen (<b>22</b>), highlighting the tight size restriction of groups at R1. Only the bromide analogue <b>23</b> had improved potency over the chloride, by approximately 5-fold, producing the first sub-micromolar lead compound in this series (pIC<sub>50</sub> 6.5, IC<sub>50</sub> 300 nM). All of these R1 analogues had minimal BRD4 potency. <b>23</b>, the most potent ATAD2 compound for which the selectivity window was most measurable, achieved >100-fold selectivity over BRD4 BD1 and 60-fold over BRD4 BD2. Binding to BRD2, BRD3, and BRDT bromodomains was also negligible (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S2</a>, Supporting Information).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. ATAD2 and BRD4 Potencies for Compounds <b>18</b>–<b>23</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0013.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">stereochemistry</th><th class="colsep0 rowsep0" align="center">ATAD2 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">BRD4 BD1 pIC<sub>50</sub></th><th class="colsep0 rowsep0" align="center">BRD4 BD2 pIC<sub>50</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>2</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">racemic</td><td class="colsep0 rowsep0" align="left">5.9 ± 0.08 (12)</td><td class="colsep0 rowsep0" align="left"><4.3 (4)</td><td class="colsep0 rowsep0" align="left">4.3 (1/4<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>18</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">enantiomer 1</td><td class="colsep0 rowsep0" align="left">5.9 ± 0.04 (4)</td><td class="colsep0 rowsep0" align="left">5.0 ± 0.39 (2/3<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>)</td><td class="colsep0 rowsep0" align="left">4.5 ± 0.06 (2/3<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>19</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">enantiomer 2</td><td class="colsep0 rowsep0" align="left">5.9 ± 0.09 (4)</td><td class="colsep0 rowsep0" align="left"><4.3 (3)</td><td class="colsep0 rowsep0" align="left"><4.3 (3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>12</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">racemic</td><td class="colsep0 rowsep0" align="left">4.9 ± 0.22 (5/6<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>)</td><td class="colsep0 rowsep0" align="left"><4.3 (2)</td><td class="colsep0 rowsep0" align="left"><4.3 (2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>20</b></td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">racemic</td><td class="colsep0 rowsep0" align="left">5.0 ± 0.08 (8)</td><td class="colsep0 rowsep0" align="left"><4.3 (2)</td><td class="colsep0 rowsep0" align="left">4.6 (1/2<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>21</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">racemic</td><td class="colsep0 rowsep0" align="left"><4.0 (4)</td><td class="colsep0 rowsep0" align="left"><4.3 (1)</td><td class="colsep0 rowsep0" align="left"><4.3 (1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>22</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">racemic</td><td class="colsep0 rowsep0" align="left"><4.0 (2)</td><td class="colsep0 rowsep0" align="left"><4.3 (4)</td><td class="colsep0 rowsep0" align="left">4.5 (1/4<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>23</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">racemic</td><td class="colsep0 rowsep0" align="left">6.5 ± 0.15 (8)</td><td class="colsep0 rowsep0" align="left">4.3 (1/3<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a>)</td><td class="colsep0 rowsep0" align="left">4.7 ± 0.08 (3)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Some results below the curve-fitting threshold could not be included in mean and SD. In these cases, the numbers included in the mean/number of times tested are both shown.</p></div></div></div><div class="NLM_p">Returning to <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, the activity of compounds such as <b>3</b>, <b>8</b>, and <b>16</b> suggested that the hydantoin R3 substituent may not be critical. We therefore prepared a small number of heavily truncated fragments to assess the baseline activity of the minimal phenylsulfonamide core and simultaneously to test additional small R1 substituents, as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The minimal chloride fragment <b>27</b> was significantly less potent than parent <b>2</b> but retained some measurable activity (pIC<sub>50</sub> ∼ 4.1, <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Fragments could be tested at concentrations up to 1 mM, enabling us to conclude that most chlorine replacements (<b>28</b>–<b>34</b>) resulted in the loss of activity, consistent with the results from the larger scaffold in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. Compound <b>27</b> gave partial responses at the concentrations tested (46–55% response at 100 μM), whereas the others showed little response in the 100 μM to 1 mM range on all test occasions. As before, only the bromide (<b>35</b>) was a better R1 group than the chloride (<b>27</b>) by approximately 5-fold. These results also showed that although our initial hits contained active compounds with various R3 substituents, reducing the size of R3 to ethyl resulted in a significant drop in potency (compare <b>35</b> to <b>23</b>, ∼50-fold, and <b>27</b> to <b>2</b>, ∼60-fold), indicating that the hydantoin R3 group does contribute to ATAD2 binding.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. ATAD2 Potency for Fragments <b>27</b>–<b>35</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0014.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">ATAD2 pIC<sub>50</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>27</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">4.1 (1/4<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>28</b></td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left"><3.0 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>29</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left"><3.0 (10)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>30</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left"><3.0 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>31</b></td><td class="colsep0 rowsep0" align="left">NH2</td><td class="colsep0 rowsep0" align="left"><3.0 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>32</b></td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="left"><3.0 (6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>33</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left"><3.0 (4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>34</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left"><3.0 (10)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>35</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">4.8 ± 0.16 (10)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Maximum response for <b>27</b> was 45–55% at 100–300 μM.</p></div></div></div><div class="NLM_p"><b>23</b> was a validated lead molecule with activity against ATAD2 and selectivity over the BET bromodomains. The SAR indicated that bromide was the optimal R1 group and that R3 groups including the hydantoin contributed significantly to ATAD2 activity. In TR-FRET binding assays against other bromodomains, <b>23</b> had a promising selectivity profile with sub-micromolar activity against ATAD2 and ATAD2B, at least 10-fold selectivity activity against CECR2, TAF1 and BRPF3, and little binding to bromodomains such as BPTF, BRD7, BRD9, and CREBBP (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a). It also showed activity against endogenous ATAD2 in the bromosphere assay (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b). Further optimization of this series against the bromodomain of ATAD2 and also against CECR2, including screening against a broader bromodomain panel, is described in a second publication.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The remainder of this manuscript focuses on understanding the interactions of the core phenylsulfonamide moiety with ATAD2.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) pIC<sub>50</sub> of <b>23</b> in other bromodomain TR-FRET assays. (B) Replicate dose–response curves of <b>23</b> in the ATAD2 bromosphere assay (pIC<sub>50</sub> 5.4 ± 0.24, <i>n</i> = 4).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> X-ray Structural and Computational Studies</h3><div class="NLM_p">The common acetyl-lysine binding motifs of bromodomains are well understood and have been extensively reviewed.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Acetyl-lysine and most known bromodomain inhibitors form a pair of hydrogen bonds to the W1 water of a water network lying at the base of the pocket and to a conserved asparagine side chain (Asn1064 in ATAD2). However, the conserved parts of the phenylsulfonamide series resembled no previously reported acetyl-lysine mimetic, suggesting the possibility of a novel mode of interaction.</div><div class="NLM_p">We therefore tried to obtain X-ray structures of representatives of this series bound to ATAD2 by soaking compounds into protein crystals. Three structures (chloride <b>9</b>, methyl-containing <b>15</b>, and bromide analogue <b>36</b>) are shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>. <b>36</b> was prepared by the route shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> using 3-methyl piperidine. Refinement statistics and electron density maps are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S3 and Figure S9</a>, Supporting Information.</div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Crystallographic binding modes of ATAD2 bound to <b>9</b>, <b>15</b>, and <b>36</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S56">6S56</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S57">6S57</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55">6S55</a>). (A) <b>9</b> (blue) superimposed on the structure of capped acetyl-lysine (yellow, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A5N">5A5N</a>). The four water molecules W1–W4 from the KAc complex displaced by <b>9</b> are depicted with a red surface. (B–D) <b>9</b> (blue), <b>15</b> (orange), and <b>36</b> (green) ATAD2 crystal structures. (E) ATAD2/<b>36</b> complex superimposed on BRD4 (blue protein, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P5O">3P5O</a>) showing close contacts. (F) ATAD2/<b>37</b> complex (cyan, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TZ2">4TZ2</a>) superimposed on ATAD2/<b>36</b>. *Data for <b>36</b> is from a ligand displacement rather than peptide displacement TR-FRET assay. For a comparison of the two, see ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Consistent with their activity in a peptide-competitive binding assay, <b>9</b> and the other compounds from this series bind to the acetyl-lysine pocket of ATAD2. However, they lie deeper than acetyl-lysine and most other bromodomain inhibitors. The four conserved water molecules W1–W4 are all displaced by the phenylsulfonamide group, which lies at the deepest point of the pocket (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>a). The phenyl ring of <b>9</b> makes predominantly hydrophobic contacts with Val1008, Phe1009, and Val1013 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>b). The chlorine atom fills a small indentation normally occupied by the W2 water, whose hydrogen-bonding potential is usually satisfied by three other waters, including W1 and W3, and by the backbone carbonyl of Ile1056. In the absence of W1–W4, the chlorine of <b>9</b> binds in a mixed environment, making polar contacts with the backbone carbonyl of Ile1056 and the hydroxyl group of Tyr1021, as well as nonpolar contacts with the hydrophobic side chains of Phe1009, Ala1060, and Ile1056.</div><div class="NLM_p">The side-chain NH<sub>2</sub> of Asn1064 normally donates a hydrogen bond to inhibitor acceptor groups. In the complex with <b>9</b>, this group is straddled by the two sulfonamide oxygen atoms (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>b) but the heavy atom distance is over 3.5 Å, which is longer than most hydrogen bonds. One of the sulfonamide oxygens forms a hydrogen bond to the side-chain hydroxyl of Tyr1021, and in this respect, it acts as a mimetic of the W1 water molecule. The hydrogen-bonding potential of the second sulfonamide oxygen is unsatisfied, with carbon atoms of the side chains of Phe1009, Ala1060, and Ile1074 its closest contacts. The sulfonamide dimethylamine group also makes further hydrophobic contacts with the walls of the site, including Val1008 and Ile1074.</div><div class="NLM_p">The remaining polar interaction visible in this crystal structure is a hydrogen bond between the amide NH of <b>9</b> and the backbone carbonyl of Lys1011 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>b). Density for the whole ligand was clear up to this amide but was sparse for the hydantoin group, which was modeled into the space available to avoid protein contacts. In this position, one of the hydantoin carbonyl oxygens would be well placed to form a hydrogen bond to the Asp1014 backbone NH.</div><div class="NLM_p">The binding mode of <b>15</b>, shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>c, is very similar to that of <b>9</b>. The azepane ring fills more of the site, making more extensive contacts with the ZA loop residues Tyr1021 and Tyr1063. The methyl group of <b>15</b> at the bottom of the pocket superimposes precisely over the chlorine of <b>9</b>. Again, the hydantoin was not well resolved and could only be modeled.</div><div class="NLM_p">The bromide analogue <b>36</b> also binds in a very similar way (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>d) but this structure revealed more density for the hydantoin, which permitted its modeling with confidence. One of the two hydantoin carbonyl groups does indeed form a hydrogen bond to the backbone NH of Asp1014, as modeled for <b>9</b> and <b>15</b>, although due to the symmetrical shape of this ring, it is impossible to say if there is any preference for which carbonyl makes this interaction. In the complex with <b>36</b>, the side chain of Arg1007 adopts a conformation able to form a hydrogen bond to the hydantoin second carbonyl. However, Arg1007 has previously been seen to be highly mobile in complexes with other inhibitors. The density for Arg1007 is similar in the complexes with <b>36</b> and <b>15</b>, but not with <b>9</b>, suggesting that this interaction is weak and dependent on crystal forces. Alternatively, its formation may depend on the substituents on the hydantoin, but the hydantoin substituents seem to make little difference to ATAD2 potency (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_p">The R2 group of <b>36</b> is a mixture of piperidine stereoisomers. The S-enantiomer fitted the density best and so this was modeled, with the methyl piperidine filling similar volume to the azepane of <b>15</b> and extending toward Tyr1063.</div><div class="NLM_p last">The residues whose side chains make contact with <b>36</b> are generally conserved or conservatively substituted between ATAD2 and the BET family bromodomains (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S4</a>, Supporting Information). When <b>36</b> was modeled into the BRD4 BD1 site by superimposing the structure of the ATAD2 complex (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>e), the BRD4 side chains of Leu92, Leu94, and Cys136 clash with the inhibitor. Close contacts with Leu92 and Leu94 appear the most significant and are most likely the main reason for the selectivity of the series over the BET bromodomains. These differences arise from the length of the ZA loop, which is two residues longer in the BET bromodomains relative to that in ATAD2.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Comparison with <b>37</b></h3><div class="NLM_p">While several inhibitors are known that can displace one or more of the conserved acetyl-lysine site water molecules of the BET family and other bromodomains,<a onclick="showRef(event, 'ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39">(37−39)</a> the only previous ATAD2 inhibitor known to displace all four is the triazole <b>37</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>), whose <i>K</i><sub>d</sub> was estimated to be 350 μM by NMR.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Beyond the common feature that both displace the W1–W4 waters with a substituted phenyl ring, the binding modes of <b>37</b> and <b>36</b> are quite distinct (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>f). <b>37</b> makes a closer hydrogen bond to the side chain of Asn1064 through its triazole nitrogen than <b>36</b> but does not interact with residues of the ZA loop such as Tyr1021 or Lys1011. The aniline nitrogen of <b>37</b> lies close to the Ile1056 carbonyl oxygen, although the geometry of this interaction is suboptimal for a hydrogen bond. It is striking how closely the bromine of <b>36</b> superimposes on the aniline of <b>37</b>. This bromine extends deeper into the pocket than the chlorine of <b>9</b> and the methyl of <b>15</b>, pushing close to the backbone carbonyl oxygen of Ile1056. The Br–O contact distance (3.1 Å) is less than the sum of their van der Waals radii (3.37 Å). The interaction is also relatively linear (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>f). The distance and angles are consistent with examples of halogen bonding in protein–ligand complex X-ray structures (C–Br–O 149°, Br–O═C 130°, R<sub>Br–O</sub> 0.91, see also <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">S10a</a>).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Halogen bonds are electrostatic interactions involving a positively charged region of the polarizable bromine atom opposite the C–Br bond (the “sigma hole”), which can form favorable close-range contacts with oxygen atoms (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S10a</a>, Supporting Information).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Such interactions are rare but have been reported in biological systems, for example, in the complex between BI201335 and HCV NS3-NS4A protease.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> A more detailed analysis can be found below.</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0010.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Inhibitors were truncated to the fragment shown and optimized with restraints on the red substructure. (B) Total fragment molecular orbital (FMO) binding energy (kcal/mol) between the inhibitor fragments and each residue of the surrounding site. Interactions between the core of inhibitor <b>36</b> and neighboring residues of interest: (C) Ile1056, (D) Pro1012/Lys1011, (E) Met1029/Asp1030, and (F) Phe1009.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Quantum Mechanical Analysis of Binding Interactions</h3><div class="NLM_p">Since this series binds to ATAD2 in a novel way, we carried out calculations based on the crystal structures to investigate the contributions to binding. Molecular mechanics force fields do not consider polarization and charge transfer and cannot accurately handle anisotropic electrostatic interactions such as halogen bonds, so quantum mechanical methods were employed starting with fragment molecular orbital (FMO) calculations.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> This is a relatively rapid method applicable to protein–ligand complexes in which the protein is divided into fragments, typically single amino acids (although it is important to realize that residues are fragmented between the C and Cα atoms to retain the delocalized character of peptide bonds). Pairwise interaction energies are then calculated between all fragments. Pair interaction energy decomposition analysis (PIEDA) was used to divide the fragment interaction energy into contributions from electrostatic, exchange–repulsion, charge transfer, and correlation/dispersion terms.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> This calculation does not attempt to estimate the total energy of binding including desolvation and entropic contributions; rather, it describes the quantum mechanical interactions in the bound state. However, this can still provide useful insights into the nature of protein–ligand interactions.<a onclick="showRef(event, 'ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47">(45−47)</a></div><div class="NLM_p">To focus on the interactions between the core of <b>9</b>, <b>15</b>, and <b>36</b> and their surrounding residues, the inhibitors were trimmed back to the respective methyl-, chloro-, or bromo-phenylsulfonamide fragments, as shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>a. The total FMO binding energies between each fragment and each surrounding residue, including two nearby water molecules W5 and W6, are shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>b, and the PIEDA energetic decomposition terms are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S11a</a>, Supporting Information.</div><div class="NLM_p">The profile of interactions for all three inhibitors is very similar, with each analysis highlighting the same strong interactions. The biggest contribution to binding is labeled as Pro1012. Because of the way the protein residues are fragmented, as explained above, this actually represents the hydrogen bond between the inhibitor amide NH and the peptide carbonyl of Lys1011 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>d and <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a>d). Desolvation energy would probably reduce the contribution of this interaction in solution.</div><div class="NLM_p">Another residue that forms strong interactions with all three inhibitors is Asp1030 (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>e). The PIEDA calculation suggests that this is driven mainly by electrostatic contributions (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S11a</a>, Supporting Information). Although nonpolar interactions exist between the side chain of Asp1030 and the phenyl ring of the inhibitor, there are no obvious strong polar interactions. However, again, because of the fragmentation scheme, this interaction includes the contribution from the carbonyl of Met1029, which lies within 3.4 Å of the closest aromatic CH of the inhibitors, which may be polarized by the neighboring bromine.</div><div class="NLM_p">The third strongest interaction is with Phe1009, whose side chain approaches the bromine of <b>36</b> in an edge-on orientation (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>f). This is the interaction whose magnitude differs most between inhibitors (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>b). As this variation is captured within the electrostatic and dispersion PIEDA components (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S11a</a>, Supporting Information), polarization by the inhibitor halogen substituent may play a part. An electrostatic potential map of the ligand surface near Phe1009 is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S10b</a>, Supporting Information.</div><div class="NLM_p">Despite the geometry between the bromine of <b>36</b> and the carbonyl of Ile1056, which is consistent with halogen bonding, the FMO calculations show similar and small interaction energies between Ile1056 and the Br-, Cl-, and Me-substituted fragments (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>b). This suggests that this is not a highly favorable interaction. These calculations did not use high-level basis functions, which are necessary for the accurate treatment of halogen bonding, since these present challenges for FMO.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> Calculations were therefore carried out to look at the Ile1056 interaction with the three fragments, using density functional theory (DFT) with higher-level basis sets using the Counterpoise method.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Using this approach, the interaction energies between each fragment and Ile1056 are all favorable. The bromide interaction is larger than chloride or methyl, but the magnitude of all three is rather small (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>; also see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S11b</a>, Supporting Information).</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Counterpoise Interaction Energies (M06-2X-D3/LACV3P**<sup>++</sup>, kJ/mol) Calculated between Bromide, Chloride, and Methyl Fragments and Ile1056 and Phe1009 Using Jaguar</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="−" /></col><col align="char" char="−" /></col><col align="char" char="−" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="−">Br</th><th class="colsep0 rowsep0" align="center" char="−">Cl</th><th class="colsep0 rowsep0" align="center" char="−">Me</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ile1056</td><td class="colsep0 rowsep0" align="char" char="−">–3.39</td><td class="colsep0 rowsep0" align="char" char="−">–0.96</td><td class="colsep0 rowsep0" align="char" char="−">–3.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Phe1009</td><td class="colsep0 rowsep0" align="char" char="−">–18.91</td><td class="colsep0 rowsep0" align="char" char="−">–16.36</td><td class="colsep0 rowsep0" align="char" char="−">–16.11</td></tr></tbody></table></div></div><div class="NLM_p last">The interaction energies with Phe1009 were also calculated and found to be greater than those with Ile1056 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S11b</a>, Supporting Information). Again, the binding energy of the bromide analogue is slightly more favorable than chloride. The geometry and energetics therefore suggest that halogen bonding does exist in the complex between <b>36</b> and Ile1056 and that this pairwise interaction is stronger with the bromide than with the chloride or methyl analogues. However, the energetic contribution of the halogen bond is relatively small and does not appear to be among the most significant contributions to binding. Instead, the increased potency of the bromine-substituted analogues seems to be due to contributions from a wider range of interactions including other nearby residues such as Phe1009.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78145" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78145" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We set out to discover inhibitors of the ATAD2 bromodomain by high-throughput screening, expecting that the low biological tractability of this bromodomain relative to others would translate into a low hit rate. This was the case, but this was masked by a very high hit rate of compounds acting through nonprogressible mechanisms in our assay, which greatly outnumbered the few progressible hits within the screening collection. While this is one of the more challenging examples of this behavior we have seen, it is by no means unusual, highlighting the need to rigorously confirm all screening hits, especially for novel targets whose behavior is unknown. It also showed the value of testing the performance of the screening cascade before the main screen is triggered, so that unanticipated problems can be identified and solved.</div><div class="NLM_p">To proceed with confidence, it was essential to have robust orthogonal biophysical confirmation assays. Because these had relatively limited capacity, we also needed to change the primary screen to reduce the hit rate to a manageable level. Resorting to computational triage would have been ineffective in this case: the problematic compounds are diverse and did not match known frequent-hitter structural filters. A clustering or diversity-based selection would have had an insufficient impact, while biasing selection toward known bromodomain pharmacophores would have compromised our ability to find novel modes of action.</div><div class="NLM_p">The incidence of nonprogressible hits was reduced by redesigning the ATAD2 bromodomain protein used in the assay. Unexpectedly, for the assays we deployed (NMR, SPR, and TR-FRET), each performed most robustly with a different protein construct. This highlights the need to test the protein for each assay format, ensuring not just reproducible pharmacology with tool compounds but low nonprogressible hit rates and the ability to configure the assay to determine characteristics that can differentiate desirable from nondesirable binding behavior. For example, in the SPR assay, we found that the use of a mutant protein control was very helpful.</div><div class="NLM_p">To our knowledge, this is the first screen of a large unbiased compound library against ATAD2 with the exception of one previous report in which DNA-encoded libraries were screened against the GST-tagged bromodomain.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Following the capture of the protein using a glutathione matrix, hits were identified by DNA sequencing. Like the work we report here, to confirm binding of the hits when resynthesized off-DNA, a panel of follow-up assays was employed, including a peptide-based TR-FRET assay, protein NMR, microscale thermophoresis, and a thermal shift assay.</div><div class="NLM_p">After validation of all steps of the triage, we proceeded to run the HTS. This identified a single series of active and progressible hits that was robustly active in multiple hit qualification assays. Although the initial hit rate from our optimized assay was relatively low at 0.6%, the progressible hit rate was a mere 0.001% of the HTS collection. Despite our efforts to minimize unprogressible mechanisms, specific binding could not be confirmed for 99.8% of the initial hits and 95% of the more potent hits with TR-FRET IC<sub>50</sub> < 30 μM. These statistics underline the importance of the careful use of orthogonal hit qualification assays both before and during hit identification and should be a warning to researchers that great caution is needed before trusting reported hits that have not been subjected to a similar rigorous confirmation, especially for low-tractability targets.</div><div class="NLM_p">X-ray structures of the hits and analogues show that this series binds in a novel mode, displacing the active-site water network and making different active-site interactions to those usually observed in bromodomain inhibitor complexes. It is unlikely that this series would have been found using prior knowledge, as was also the case for the reported DNA-encoded library screen,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> showing that diversity screening can add value even to a well-studied protein class like bromodomains.</div><div class="NLM_p">The HTS discovered reasonably potent ATAD2 ligands that displace the active-site waters, but to our surprise it did not find any conventionally binding inhibitors. We know that the TR-FRET assay can detect such compounds, as we showed in our report of screening a bromodomain-targeted compound set.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> This may simply be due to chance, given the extremely low tractability of the target. However, it is possible that the water network in ATAD2 is less stable than other bromodomains, a suggestion supported by molecular dynamics simulations, which ranked the water network of ATAD2 as one of the least stable among bromodomains studied.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> It may indeed be that the penalty for displacing the active-site waters varies between bromodomains, explaining the selectivity of the phenylsulfonamides over the BET family, which bind their waters more tightly in simulations. One caveat is that the simulations were based upon the crystal structure of the ATAD2 981–1108 truncate. We showed above that the length of the truncated ATAD2 bromodomain sequence strongly influenced the nonprogressible hit rate in our binding assay. Though these hits do not bind to the ATAD2 acetyl-lysine site, it would not be surprising if the water network stability, and hence the ability to detect and quantify the potency of inhibitors with different binding modes, also varied with construct length. The influence of construct length upon the stability of bromodomain active-site water networks has not been investigated. In any event, the sulfonamide hits do compete with peptide for binding to full-length endogenous ATAD2 in the bromosphere assay and so should be physiologically relevant.</div><div class="NLM_p">The lead molecules contain a bromine atom that engages the ATAD2 bromodomain in a halogen-bonding interaction not previously described in bromodomains. Its binding partner is the backbone carbonyl of Ile1056 in a structurally conserved helix present across the family including the well-studied BET family. Since the carbonyl is only exposed for binding to inhibitors after displacement of the water network, differential sensitivity to water displacement may confer some selectivity for ATAD2, as described above. The SAR shows that bromine improves the binding of the series by up to 10-fold relative to chlorine. However, this was not needed to find the original HTS hits, which only contained chlorine or methyl groups. Quantum mechanics calculations suggest that the contribution of the bromine halogen bond to the overall binding energy is small and that interactions between the bromine and other residues of the site also contribute as much if not more to binding. When describing crystallographic complexes, it is often tempting to give great significance to rare and interesting interactions but this may not always be warranted.</div><div class="NLM_p last">In summary, we have presented above the discovery of the phenylsulfonamide series of ATAD2 inhibitors by high-throughput screening. The optimization of the series against the bromodomains of ATAD2 and CECR2 will be the subject of a further manuscript.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68574" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68574" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Chemistry</h3><div class="NLM_p last">All solvents were purchased from Sigma-Aldrich (Hy-Dry anhydrous solvents), and commercially available reagents were used as received. All reactions were followed by thin-layer chromatography (TLC) analysis (TLC plates GF254, Merck) or liquid chromatography/mass spectrometry (LC/MS) using a Waters ZQ instrument. NMR spectra were recorded on a Bruker NanoBay 400 MHz or a Bruker AVII+ 600 MHz spectrometer and are referenced as follows: <sup>1</sup>H NMR (400 or 600 MHz), internal standard tetramethylsilane at δ = 0.00; <sup>13</sup>C NMR (100.6 or 150.9 MHz), internal standard CDCl<sub>3</sub> at δ = 77.23 or dimethyl sulfoxide (DMSO)-<i>d</i><sub>6</sub> at δ = 39.70. Column chromatography was performed on prepacked silica gel columns (30–90 mesh, IST) using a Biotage SP4. Mass spectra were recorded on Waters ZQ (electrospray ionization-MS) and Q-Tof 2 [high-resolution mass spectrometry (HRMS)] spectrometers. Mass-directed Auto Prep (MDAP) was performed on a Waters 2767 with a MicroMass ZQ mass spectrometer using a Supelco LCABZ++ column.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Synthetic Methods and Characterization of Compounds</h3><div class="NLM_p last">Abbreviations for multiplicities observed in NMR spectra: s; singlet; br s, broad singlet; d, doublet; t, triplet; q, quadruplet; p, pentuplet; spt, septuplet; m, multiplet. The purity of all compounds was determined by LC/MS and <sup>1</sup>H NMR and was always >95%.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> LC/MS</h3><div class="NLM_p">Ultraperformance liquid chromatography (UPLC) analysis was conducted on an Acquity UPLC BEH C18 column (50 mm × 2.1 mm, i.d. 1.7 μm packing diameter) at 40 °C. Formate method: solvents employed were A = 0.1% v/v solution of formic acid in water; B = 0.1% v/v solution of formic acid in acetonitrile. High pH method: the solvents employed were A = 10 mM ammonium hydrogen carbonate in water adjusted to pH 10 with ammonia solution; B = acetonitrile. For both methods, the gradient employed was as follows (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>).</div><div class="NLM_table-wrap" id="tbl8">Table 8<div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="−" /></col><col align="char" char="−" /></col><col align="char" char="−" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">time (min)</th><th class="colsep0 rowsep0" align="center" char="−">flow rate (mL/min)</th><th class="colsep0 rowsep0" align="center" char="−">%A</th><th class="colsep0 rowsep0" align="center" char="−">%B</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="char" char="−">1</td><td class="colsep0 rowsep0" align="char" char="−">99</td><td class="colsep0 rowsep0" align="char" char="−">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="char" char="−">1</td><td class="colsep0 rowsep0" align="char" char="−">3</td><td class="colsep0 rowsep0" align="char" char="−">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="char" char="−">1</td><td class="colsep0 rowsep0" align="char" char="−">3</td><td class="colsep0 rowsep0" align="char" char="−">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="char" char="−">1</td><td class="colsep0 rowsep0" align="char" char="−">0</td><td class="colsep0 rowsep0" align="char" char="−">100</td></tr></tbody></table></div></div><div class="NLM_p last">The UV detection was a summed signal from a wavelength of 210–350 nm. Mass spectra were obtained on a Waters ZQ instrument; ionization mode, alternate-scan positive and negative electrospray; scan range 100–1000 amu; scan time: 0.27 s; interscan delay: 0.10 s. LC/MS traces are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S13</a>, Supporting Information.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> MDAP</h3><div class="NLM_p">Formate method: high-performance liquid chromatography (HPLC) analysis was conducted on either a Sunfire C18 column (100 mm × 19 mm, i.d. 5 μm packing diameter) or a Sunfire C18 column (150 mm × 30 mm, i.d. 5 μm packing diameter) at ambient temperature. The solvents employed were A = 0.1% v/v solution of formic acid in water; B = 0.1% v/v solution of formic acid in acetonitrile. It was run as a gradient over either 15 or 25 min (extended run) with a flow rate of 20 mL/min (100 mm × 19 mm, i.d 5 μm packing diameter) or 40 mL/min (150 mm × 30 mm, i.d. 5 μm packing diameter). High pH method: HPLC analysis was conducted on either an Xbridge C18 column (100 mm × 19 mm, i.d. 5 μm packing diameter) or an Xbridge C18 column (100 mm × 30 mm, i.d. 5 μm packing diameter) at ambient temperature. The solvents employed were A = 10 mM ammonium bicarbonate in water, adjusted to pH 10 with ammonia solution; B = acetonitrile. It was run as a gradient over either 15 or 25 min (extended run) with a flow rate of 20 mL/min (100 mm × 19 mm, i.d. 5 μm packing diameter) or 40 mL/min (100 mm × 30 mm, i.d. 5 μm packing diameter). For both methods, UV detection was a summed signal from a wavelength of 210–350 nm. Mass spectra were obtained on a Waters ZQ instrument; ionization mode, alternate-scan positive and negative electrospray; scan range 100–1000 amu; scan time: 0.50 s; interscan delay: 0.20 s.</div><div class="NLM_p">Compounds <b>2</b>–<b>17</b>, <b>21</b>, <b>22</b>, <b>27</b>, and <b>33</b> were commercially available. Compound <b>2</b> was resynthesized for further characterization.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (1-Methyl-6-(4-methylpiperazin-1-yl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-yl)(phenyl)methanone (<b>1</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.84–10.96 (m, 1H), 8.17 (s, 1H), 8.01–8.12 (m, 2H), 7.71–7.83 (m, 1H), 7.53–7.66 (m, 2H), 4.64–4.90 (m, 2H), 3.95 (s, 3H), 3.43–3.59 (m, 4H), 2.99–3.20 (m, 2H), 2.74–2.85 (m, 3H).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-(4-Chloro-3-(piperidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>2</b>)</h4><div class="NLM_p">A 20 mL microwave vial was charged with K<sub>2</sub>CO<sub>3</sub> (79 mg, 0.57 mmol), 2-chloro-<i>N</i>-(4-chloro-3-(piperidin-1-ylsulfonyl)phenyl)acetamide (commercially available from Enamine, 100 mg, 0.285 mmol), and 5-cyclopropyl-5-methylimidazolidine-2,4-dione (43.9 mg, 0.285 mmol) and was then filled with acetone (10 mL). The resulting mixture was degassed for 20 min with nitrogen and then stirred at 80 °C for 30 min under microwave irradiations before being cooled down to room temperature and concentrated in vacuo. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (20 mL) and water (20 mL), and the layers were separated. The organic phase was dried using a phase separator and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (25 g column, gradient: 0–100% CH<sub>2</sub>Cl<sub>2</sub> in cyclohexane) gave <i>N</i>-(4-chloro-3-(piperidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (92 mg, 69%) as a white solid. LC/MS (method high pH): retention time 1.04 min, [M + H]<sup>+</sup> = 469.21.</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.71 (s, 1H), 8.23 (d, <i>J</i> = 2.6 Hz, 1H), 8.17 (s, 1H), 7.83 (dd, <i>J</i> = 8.6, 2.8 Hz, 1H), 7.64 (d, <i>J</i> = 8.8 Hz, 1H), 4.15–4.24 (m, 2H), 3.13–3.20 (m, 4H), 1.46–1.58 (m, 6H), 1.40 (s, 3H), 1.16 (tt, <i>J</i> = 8.3, 5.3 Hz, 1H), 0.41–0.51 (m, 2H), 0.33–0.39 (m, 1H), 0.26–0.33 (m, 1H).</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (<i>R</i>)-<i>N</i>-(4-Chloro-3-(piperidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide and (<i>S</i>)-<i>N</i>-(4-Chloro-3-(piperidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>18</b> and <b>19</b>)</h4><div class="NLM_p last"><i>N</i>-(4-Chloro-3-(piperidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>2</b>, 25 mg) was separated into its two enantiomers by chiral chromatography. Analytical method: approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 20 μL was injected onto the column. Eluant: 20% EtOH in heptane, flow <i>f</i> = 1.0 mL/min, wavelength 215 nm. Column 4.6 mm × 25 cm i.d. Chiralpak AD-H. Preparative method: approximately 25 mg of material was dissolved in EtOH (1 mL) and heptane (1 mL). A solution of 1 mL was injected onto the column. Eluant: 20% EtOH in heptane, flow = 30 mL/min, wavelength 215 nm. Column 30 mm × 25 cm Chiralpak AD-H. The total number of injections is 2. The fractions containing products were combined and concentrated in vacuo to give compound <b>18</b> (10 mg, 80%) as the fastest running enantiomer and compound <b>19</b> (11 mg, 88%) as the slowest running enantiomer. LC/MS (method high pH): retention time 1.04 min, [M + H]<sup>+</sup> = 469.1.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 2-(4-Cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)-<i>N</i>-(3-(piperidin-1-ylsulfonyl)-4-(trifluoromethyl)phenyl)acetamide (<b>20</b>)</h4><div class="NLM_p last">A mixture of <i>N</i>-(4-bromo-3-(piperidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>23</b>, 300 mg, 0.584 mmol) and CuI (334 mg, 1.75 mmol) in DMF (2 mL) at room temperature was treated with methyl fluorosulfonyl difluoroacetate (0.594 mL, 4.67 mmol) in DMF (2 mL) dropwise over 1 min. The resulting mixture was stirred at 80 °C for 12 h and then cooled down to room temperature and diluted with EtOAc. The organic phase was washed with water (25 mL) and then brine (25 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by preparative HPLC gave 2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)-<i>N</i>-(3-(piperidin-1-ylsulfonyl)-4-(trifluoromethyl)phenyl)acetamide (43 mg, 15%) as an off-white solid. LC/MS (method formic): retention time 1.02 min, [M + H]<sup>+</sup> = 503.4. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.28 (d, <i>J</i> = 1.8 Hz, 1H), 8.22 (s, 1H), 7.93–8.08 (m, 2H), 4.17–4.36 (m, 2H), 3.14–3.25 (m, 5H), 1.49–1.70 (m, 7H), 1.44 (s, 3H), 0.43–0.55 (m, 2H), 0.28–0.43 (m, 2H).</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-(4-Bromo-3-(piperidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (<b>23</b>)</h4><div class="NLM_p last">Step 1: 2-bromo-5-nitrobenzenesulfonyl chloride (commercially available, 1.10 g, 3.66 mmol) was taken up in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) under nitrogen and cooled to 0 °C. DIPEA (1.41 mL, 8.05 mmol) was added, and the reaction was stirred for 5 min before piperidine (0.43 mL, 4.39 mmol) was added. The reaction was stirred at 0 °C for 1 h. The reaction mixture was quenched with a saturated NaHCO<sub>3</sub> aqueous solution (50 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 50 mL). The combined organics were filtered through a hydrophobic frit and concentrated in vacuo to afford 1-((2-bromo-5-nitrophenyl)sulfonyl)piperidine (1.24 g, 97%) as an orange gum, which was used in the next step without further purifiction. LC/MS (method formic): retention time 1.24 min, [M + H]<sup>+</sup> = 349.0 (1Br). HRMS (C<sub>11</sub>H<sub>13</sub>BrN<sub>2</sub>O<sub>4</sub>S): [M + H]<sup>+</sup> calcd 348.9858, found 348.9844; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.91 (d, <i>J</i> = 2.7 Hz, 1H), 8.22 (dd, <i>J</i> = 8.6, 2.7 Hz, 1H), 7.96 (d, <i>J</i> = 8.6 Hz, 1H), 3.32–3.40 (m, 4H), 1.57–1.73 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 146.8, 140.8, 137.0, 127.7, 127.2, 126.7, 46.8, 25.6, 23.7. Step 2: 1-((2-bromo-5-nitrophenyl)sulfonyl)piperidine (1.10 g, 3.15 mmol), NH<sub>4</sub>Cl (0.25 g, 4.73 mmol), and Fe(0) (0.53 g, 9.45 mmol) were dissolved in a 3:1 mixture of EtOH (5.0 mL) and water (1.7 mL). The resulting solution was heated to 70 °C for 2 h. The reaction was allowed to cool down to room temperature and then filtered through a plug of celite, washing with MeOH (20 mL). The resulting solution was concentrated in vacuo and then partitioned between a saturated NaHCO<sub>3</sub> aqueous solution (20 mL) and CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated in vacuo to afford the crude product. The crude product was purified by flash chromatography on silica gel, eluting with 0–50% EtOAc/cyclohexane. The pure fractions were combined and concentrated in vacuo to afford 4-bromo-3-(piperidin-1-ylsulfonyl)aniline (0.75 g, 74%) as an orange gum. LC/MS (method formic): retention time 1.02 min, [M + H]<sup>+</sup> = 319.1 (1Br). HRMS (C<sub>11</sub>H<sub>15</sub>BrN<sub>2</sub>O<sub>2</sub>S): [M + H]<sup>+</sup> calcd 319.0116, found 319.0112; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.39–7.48 (m, 2H), 6.67 (dd, <i>J</i> = 8.6, 2.9 Hz, 1H), 3.93 (br s, 2H), 3.23–3.30 (m, 4H), 1.52–1.69 (m, 6H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 145.9, 138.3, 136.2, 119.4, 118.3, 106.9, 46.5, 25.5, 23.8; IR ν<sub>max</sub> (cm<sup>–1</sup>) 3374, 2938, 1626, 1590, 1463, 1309, 1140, 719, 578. Step 3: 2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetic acid (commercially available, 319 mg, 1.50 mmol), 4-bromo-3-(piperidin-1-ylsulfonyl)aniline (400 mg, 1.25 mmol), and HATU (572 mg, 1.50 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (10 mL). DIPEA (0.657 mL, 3.76 mmol) was added, and the resulting solution was stirred at room temperature for 2 h. The reaction was quenched with a saturated NaHCO<sub>3</sub> aqueous solution (20 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 20 mL). The combined organics were passed through a hydrophobic frit and concentrated in vacuo to afford the crude product. The crude product was purified by flash chromatography on silica gel, eluting with 0–60% EtOAc/cyclohexane. The pure fractions were combined and concentrated in vacuo to afford <i>N</i>-(4-bromo-3-(piperidin-1-ylsulfonyl)phenyl)-2-(4-cyclopropyl-4-methyl-2,5-dioxoimidazolidin-1-yl)acetamide (405 mg, 63%) as a cream solid. LC/MS (method formic): retention time 1.05 min, [M + H]<sup>+</sup> = 513.2 (1Br). HRMS (C<sub>20</sub>H<sub>25</sub>BrN<sub>4</sub>O<sub>5</sub>S): [M + H]<sup>+</sup> calcd 513.0807, found 513.0807; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.71 (s, 1H), 8.98 (s, 1H), 7.91–8.01 (m, 2H), 7.62 (d, <i>J</i> = 8.6 Hz, 1H), 4.19 (s, 2H), 3.15–3.21 (m, 4H), 1.48–1.56 (m, 6H), 1.40 (s, 3H), 1.12–1.19 (m, 1H), 0.41–0.48 (m, 2H), 0.34–0.39 (m, 1H), 0.26–0.33 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 176.4, 164.6, 156.5, 138.1, 137.5, 136.3, 124.5, 122.8, 113.9, 62.0, 46.6, 41.5, 25.5, 23.7, 23.0, 17.3, 1.1, 0.3; m.p. 221.5 – 225.0 °C; IR ν<sub>max</sub> (cm<sup>–1</sup>) 3308, 1712, 1535, 1454, 1307, 579.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i>-(3-(Piperidin-1-ylsulfonyl)-4-(trifluoromethyl)phenyl)propionamide (<b>28</b>)</h4><div class="NLM_p last">A solution of <i>N</i>-(4-bromo-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>32</b>, 200 mg, 0.533 mmol), copper iodide (304 mg, 1.60 mmol), and methyl fluorosulfonyl difluoroacetate (819 mg, 4.26 mmol) in DMF (2 mL) was stirred at 80 °C for 26 h and then cooled down to room temperature, and diluted with EtOAc. The organic phase was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by preparative HPLC (column Xbridge C8 (250 mm × 19 mm<sup>2</sup>, i.d. 10 μm), mobile phase 0.1% trifluoroacetyl (TFA) in water and CH<sub>3</sub>CN) gave <i>N</i>-(3-(piperidin-1-ylsulfonyl)-4-(trifluoromethyl)phenyl)propionamide (19 mg, 10%) as an off-white solid. LC/MS (method formic): retention time 1.08 min, [M + H]<sup>+</sup> = 365.31. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.12 (d, <i>J</i> = 2.2 Hz, 1H), 8.02–8.08 (m, 1H), 7.81 (d, <i>J</i> = 8.6 Hz, 1H), 7.58–7.67 (m, 1H), 3.20–3.30 (m, 4H), 2.46 (q, <i>J</i> = 7.6 Hz, 2H), 1.49–1.71 (m, 6H), 1.27 (t, <i>J</i> = 7.6 Hz, 3H).</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-(4-Fluoro-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>29</b>)</h4><div class="NLM_p last">Step 1: a solution of piperidine (0.231 mL, 2.33 mmol) and NEt<sub>3</sub> (0.974 mL, 6.99 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) at 0 °C was treated with 2-fluoro-5-nitrobenzene-1-sulfonyl chloride (available from Combi Block, 0.67 g, 2.8 mmol) dropwise. The resulting mixture was stirred at room temperature for 16 h and then treated with ice-cold water (50 mL). The layers were separated. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 30 mL), and the combined organics were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (20 g column, gradient: 5–12% EtOAc in petroleum ether) gave 1-((2-fluoro-5-nitrophenyl)sulfonyl)piperidine (0.3 g, 37%) as pale yellow solid. Step 2: a suspension of 1-((2-fluoro-5-nitrophenyl)sulfonyl)piperidine (0.30 g, 1.0 mmol) in EtOH (15 mL) and water (5 mL) was treated at room temperature with Fe(0) (0.291 g, 5.20 mmol) and NH<sub>4</sub>Cl (0.056 g, 1.0 mmol), and the resulting mixture was stirred at 70 °C for 1 h and then cooled down to room temperature. The insolubles were filtered off using celite and washed with MeOH (50 mL). The phases were combined, and most of the volatiles were removed in vacuo. The residue was diluted with ice-cold water, and the aqueous phase was extracted with EtOAc (2 × 40 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Trituration of the residue with pentane gave 4-fluoro-3-(piperidin-1-ylsulfonyl)aniline (0.18 g, 67%) as a gray sticky solid, which was used in the next step without further purification. Step 3: a solution of 4-fluoro-3-(piperidin-1-ylsulfonyl)aniline (0.26 g, 1.0 mmol) and NEt<sub>3</sub> (0.281 mL, 2.01 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) at 0 °C was treated with propionyl chloride (0.105 mL, 1.208 mmol) dropwise. The reaction was warmed to room temperature, stirred for 1 h, and then treated with ice-cold water (30 mL). The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 30 mL). The combined organics were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. The residue was purified by preparative TLC (50% EtOAc in petroleum ether) to give <i>N</i>-(4-fluoro-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (123 mg, 39%) as an off-white solid. LC/MS (method formic): retention time 0.91 min, [M + H]<sup>+</sup> = 315.08. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.09 (dt, <i>J</i> = 8.4, 3.5 Hz, 1H), 7.68 (dd, <i>J</i> = 5.9, 2.9 Hz, 1H), 7.45–7.59 (m, 1H), 7.17 (t, <i>J</i> = 9.2 Hz, 1H), 3.07–3.27 (m, 4H), 2.42 (q, <i>J</i> = 7.7 Hz, 2H), 1.64 (quin, <i>J</i> = 5.6 Hz, 4H), 1.45–1.53 (m, 2H), 1.26 (t, <i>J</i> =7.6 Hz, 3H).</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(4-Hydroxy-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>30</b>)</h4><div class="NLM_p last">A solution of <i>N</i>-(4-bromo-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>32</b>, 400 mg, 1.07 mmol) in 1, 4-dioxane (2.5 mL) and water (2.5 mL) at room temperature was treated with KOH (179 mg, 3.20 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (19.5 mg, 0.0210 mmol), and <i>t</i>-butylXphos (9 mg, 0.021 mmol). The resulting mixture was stirred at 100 °C for 12 h, then cooled down to room temperature, and diluted with water (20 mL). The aqueous phase was extracted with EtOAc (2 × 50 mL), and the combined organics were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (gradient: 40% EtOAc in hexanes) gave a residue, which was further purified by preparative HPLC (column Packed C18 (250 mm × 19 mm, i.d. 5 μm), mobile phase: (A) 0.1% TFA in water, (B) acetonitrile) to give <i>N</i>-(4-hydroxy-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (70 mg, 21%) as a brown solid. LC/MS (method formic): retention time 0.80 min, [M + H]<sup>+</sup> = 313.11. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.54–8.95 (m, 1H), 7.73 (d, <i>J</i> = 2.4 Hz, 1H), 7.50 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.08 (br s, 1H), 7.00 (d, <i>J</i> = 8.8 Hz, 1H), 3.03–3.17 (m, 4H), 2.40 (q, <i>J</i> = 7.7 Hz, 2H), 1.66 (dt, <i>J</i> = 11.3, 5.8 Hz, 4H), 1.41–1.53 (m, 2H), 1.26 (t, <i>J</i> = 7.6 Hz, 3H).</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-(4-Amino-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>31</b>)</h4><div class="NLM_p last">A solution of <i>N</i>-(4-bromo-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>35</b>, 200 mg, 0.533 mmol), K<sub>2</sub>CO<sub>3</sub> (221 mg, 1.60 mmol), and <span class="smallcaps smallerCapital">l</span>-proline (123 mg, 1.07 mmol) in 25% w/w ammonia (7.96 mL, 53.3 mmol) at room temperature was treated with CuI (203 mg, 1.07 mmol), and the resulting mixture was stirred at 110 °C for 12 h in a sealed tube, then cooled down to room temperature, and diluted with water (50 mL). The aqueus phase was extracted with EtOAc (2 × 50 mL), and the combined organics were washed with brine (20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by preparative TLC gave <i>N</i>-(4-amino-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (37 mg, 22%) as an off-white solid. LC/MS (method formic): retention time 0.78 min, [M + H]<sup>+</sup> = 312.12. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.67 (s, 1H), 7.71 (d, <i>J</i> = 2.6 Hz, 1H), 7.48 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 6.79 (d, <i>J</i> = 8.8 Hz, 1H), 5.79 (s, 2H), 2.91–3.08 (m, 4H), 2.25 (q, <i>J</i> = 7.7 Hz, 2H), 1.46–1.60 (m, 4H), 1.33–1.43 (m, 2H), 1.06 (t, <i>J</i> = 7.6 Hz, 3H).</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-(4-Cyano-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>32</b>)</h4><div class="NLM_p last">A solution of <i>N</i>-(4-bromo-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>35</b>, 200 mg, 0.533 mmol) in NMP (2 mL) at room temperature under nitrogen was treated with copper(I) cyanide (71.6 mg, 0.799 mmol), and the resulting mixture was stirred at 160 °C for 1 h, then cooled down to room temperature, and diluted with water (50 mL). The aqueous phase was extracted with EtOAc (2 × 100 mL), and the combined organics were washed with brine (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (gradient 60% EtOAc in hexanes) gave <i>N</i>-(4-cyano-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (107 mg, 62%) as an off-white solid. LC/MS (method formic): retention time 0.92 min, [M + H]<sup>+</sup> = 322.31. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.28 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 7.96 (d, <i>J</i> = 2.2 Hz, 1H), 7.86 (m, 2H), 3.26–3.31 (m, 4H), 2.52 (q, <i>J</i> = 7.5 Hz, 2H), 1.70 (quin, <i>J</i> = 5.6 Hz, 4H), 1.52–1.62 (m, 2H), 1.31 (t, <i>J</i> = 7.6 Hz, 3H).</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-(4-Ethyl-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>34</b>)</h4><div class="NLM_p last">Step 1: a mixture of 1-ethyl-4-nitrobenzene (1.79 mL, 13.2 mmol) and sulfurochloridic acid (3.23 mL, 39.7 mmol) was heated at 100 °C for 4 h, then cooled down to room temperature, and concentrated in vacuo to give 2-ethyl-5-nitrobenzene-1-sulfonyl chloride (1.8 g, 54%) as a gray solid, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.93 (d, <i>J</i> = 2.4 Hz, 1H), 8.49 (dd, <i>J</i> = 8.6, 2.4 Hz, 1H), 7.71 (d, <i>J</i> = 8.6 Hz, 1H), 3.31 (q, <i>J</i> = 7.5 Hz, 2H), 1.42 (t, <i>J</i> = 7.6 Hz, 3H). Step 2: a suspension of 2-ethyl-5-nitrobenzene-1-sulfonyl chloride (1.8 g, 7.2 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) at 0 °C was treated with NEt<sub>3</sub> (2.51 mL, 18.0 mmol) and then piperidine (0.595 mL, 6.01 mmol), and the resulting mixture was stirred at room temperature for 1 h before being treated with ice-cold water (100 mL). The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 100 mL). The combined organics were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Tritration of the residue with Et<sub>2</sub>O and pentane gave 1-((2-ethyl-5-nitrophenyl)sulfonyl)piperidine (1.1 g, 61%) as a yellow solid, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.49 (d, <i>J</i> = 2.4 Hz, 1H), 8.42 (dd, <i>J</i> = 8.4, 2.5 Hz, 1H), 7.82 (d, <i>J</i> = 8.6 Hz, 1H), 3.10–3.17 (m, 4H), 3.06 (q, <i>J</i> = 7.5 Hz, 2H), 1.43–1.60 (m, 6H), 1.25 (t, <i>J</i> = 7.5 Hz, 3H). Step 3: a solution of 1-((2-ethyl-5-nitrophenyl)sulfonyl)piperidine (1.1 g, 4.4 mmol) in EtOH (15 mL) and water (5 mL) at room temperature was treated with Fe(0) (1.23 g, 22.0 mmol) and NH<sub>4</sub>Cl (0.354 g, 6.61 mmol), and the resulting mixture was stirred at 70 °C for 1 h and then cooled down to room temperature. The insolubles were filtered off through celite and rinsed with MeOH (100 mL), and most of the volatiles were removed in vacuo. The residue was diluted with ice-cold water (100 mL), and the aqueous phase was extracted with EtOAc (2 × 150 mL). The combined organics were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Trituration of the residue with pentane gave 4-ethyl-3-(piperidin-1-ylsulfonyl)aniline (0.8 g, 68%) as a yellow sticky solid, which was used in the next step without further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.08 (d, <i>J</i> = 8.3 Hz, 1H), 7.01 (d, <i>J</i> = 2.4 Hz, 1H), 6.75 (dd, <i>J</i> = 8.3, 2.4 Hz, 1H), 5.35 (s, 2H), 2.94–3.04 (m, 4H), 2.75 (q, <i>J</i> = 7.4 Hz, 2H), 1.43–1.60 (m, 6H), 1.11 (t, <i>J</i> = 7.5 Hz, 3H). Step 4: a solution of 4-ethyl-3-(piperidin-1-ylsulfonyl)aniline (0.80 g, 3.0 mmol) and NEt<sub>3</sub> (0.831 mL, 5.96 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (30 mL) at 0 °C under nitrogen was treated with propionyl chloride (0.305 mL, 3.58 mmol) dropwise. The mixture was stirred at room temperature for 30 min and then treated with ice-cold water (50 mL). The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 50 mL). The combined organics were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. Trituration of the residue with Et<sub>2</sub>O gave <i>N</i>-(4-ethyl-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (412 mg, 42%) as a white solid. LC/MS (method formic): retention time 1.01 min, [M + H]<sup>+</sup> = 325.12. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>), δ 10.08 (s, 1H), 8.08 (d, <i>J</i> = 2.2 Hz, 1H), 7.79 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.40 (d, <i>J</i> = 8.6 Hz, 1H), 2.99–3.13 (m, 4H), 2.88 (q, <i>J</i> = 7.5 Hz, 2H), 2.22–2.40 (m, 2H), 1.33–1.62 (m, 6H), 1.17 (t, <i>J</i> = 7.5 Hz, 3H), 1.08 (t, <i>J</i> = 7.6 Hz, 3H).</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>-(4-Bromo-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (<b>35</b>)</h4><div class="NLM_p last">A solution of 4-bromo-3-(piperidin-1-ylsulfonyl)aniline (3.60 g, 11.3 mmol) and NEt<sub>3</sub> (4.72 mL, 33.8 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (40 mL) at 0 °C was treated with propionyl chloride (1.25 g, 13.5 mmol), and the resulting mixture was stirred at this temperature for 1 h and then quenched with water (100 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 250 mL). The combined organics were washed with brine (200 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. Purification of the residue by flash chromatography on silica gel (gradient 0–100% EtOAc in hexanes) gave <i>N</i>-(4-bromo-3-(piperidin-1-ylsulfonyl)phenyl)propionamide (3.3 g, 76%) as an off-white solid. LC/MS (method formic): retention time 1.00 min, [M + H]<sup>+</sup> = 376.99 (1Br). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.06 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 7.99 (d, <i>J</i> = 2.6 Hz, 1H), 7.66–7.78 (m, 2H), 3.27–3.36 (m, 4H), 2.48 (q, <i>J</i> = 7.6 Hz, 2H), 1.54–1.73 (m, 6H), 1.30 (t, <i>J</i> = 7.6 Hz, 3H).</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-(4-Bromo-3-((3-methylpiperidin-1-yl)sulfonyl)phenyl)-2-(2,5-dioxoimidazolidin-1-yl)acetamide (<b>36</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (br s, 1H), 8.27 (d, <i>J</i> = 2.6 Hz, 1H), 8.17 (s, 1H), 7.76–7.85 (m, 1H), 7.72 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 4.20 (s, 2H), 4.02 (s, 2H), 3.45–3.65 (m, 2H), 2.63–2.80 (m, 1H), 2.40 (dd, <i>J</i> = 12.3, 10.3 Hz, 1H), 1.52–1.77 (m, 3H), 1.34–1.52 (m, 1H), 0.92–1.11 (m, 1H), 0.84 (d, <i>J</i> = 6.6 Hz, 3H).</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 2-(4-((Dimethylamino)methyl)phenyl)-5,6-dimethylpyrimidin-4(3<i>H</i>)-one (<b>38</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 12.45 (br s, 1H), 8.05 (d, <i>J</i> = 8.3 Hz, 2H), 7.41 (d, <i>J</i> = 8.3 Hz, 2H), 3.45 (s, 2H), 2.29 (s, 3H), 2.16 (s, 6H), 1.98 (s, 3H).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Protein Cloning, Expression, and Purification</h3><div class="NLM_p">Methods for the production of 6His-FLAG-Tev-ATAD2(950–1148), FLAG-6His-Tev-ATAD2(981–1121), and <sup>15</sup>N labeled and unlabeled ATAD2(981–1108) have all been published.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a></div><div class="NLM_p">For the new constructs listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S1</a> (Supporting Information), all except for construct 7 were cloned into the pET26b vector using restriction enzymes <i>Nde</i>I and <i>Xho</i>I with the FLAG-6H-tev tag added with the 5′ primer. Construct 7 was cloned into the pET11c vector using <i>Nde</i>I and <i>Bam</i>HI. Plasmid DNA was transformed into <i>Escherichia coli</i> BL21 (DE3)-competent cells (Invitrogen C6000-03) and recombinant colonies selected on Luria-Bertani (LB)-agar plates containing the relevant antibiotic. For small-scale expression studies, 5 mL of LB medium containing the relevant antibiotic was inoculated with a single colony and grown overnight at 37 °C with shaking at 180 rpm. One milliliter of the overnight culture was used to inoculate 99 mL of the culture medium (LB, 1% (v/v) glycerol or Turbo (USBiological Life Sciences, cat no. P9102-40B), and 1% (v/v) glycerol) and relevant antibiotic, and growth was maintained at 37 °C until absorbance (600 nm) was measured approximately 0.5. The cultures were induced with 0.5 mM isopropyl β-<span class="smallcaps smallerCapital">d</span>-thiogalactoside and further incubated at 18 °C overnight. The cultures were harvested by centrifugation, and the pellet was stored at −80 °C. The pellet was resuspended with one-fifth of the expression culture volume buffer A (50 mM Tris–HCl pH 7.5, 300 mM NaCl, 2 mM β-mercaptoethanol). The lysate was sonicated, on ice, for a total of 2 min, at 40% amplitude, and clarified by centrifugation at 39 191<i>g</i>, 4 °C, for 30 min.</div><div class="NLM_p last">Ni-NTA resin (Qiagen cat. no. 70666) was equilibrated with buffer A. The lysate supernatant was contacted with the resin at 4 °C for 1 h. The nonabsorbed fraction was collected via a gravity column, and the column was washed with 10 column volumes buffer A. The column was further washed with 10 column volumes buffer A containing 20 mM Imidazole. Bound ATAD2 protein was eluted with a step gradient of buffer B (50 mM Tris–HCl pH 7.5, 300 mM NaCl, 100 mM ethylenediaminetetraacetic acid, and 5 mM β-mercaptoethanol) over 10 column volumes. Protein identity was confirmed by peptide mass fingerprinting and LC/MS analysis.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Assays</h3><div class="NLM_p last">Methods for the Flag-6H-tev-ATAD2(981–1121) peptide competition time-resolved fluorescence resonance energy transfer (TR-FRET) binding assay, SPR binding assay, <sup>15</sup>N–<sup>1</sup>H HSQC NMR (50 μM protein), and bromosphere assay, as well as the mutant BET bromodomain TR-FRET assays, are available elsewhere.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Evaluation of Nonbiotinylated H4 Peptide Binding to Newly Prepared ATAD2 Constructs by TR-FRET</h3><div class="NLM_p last">Variations of the previously described Flag-6H-tev-ATAD2(981–1121) peptide competition TR-FRET assay were used. Each ATAD2 construct (25 nM) was prepared in an assay buffer [composed of 50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES), pH 7.4, 150 mM NaCl, 5% glycerol, 1 mM dithiothreitol, and 1 mM 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate] prior to addition to an equivalent volume of 50 nM biotinylated triacetylated histone H4 peptide (SGRG-K(Ac)-GG-K(Ac)-GLG-K(Ac)-GGAKRHRKV-biotin, CRB Discovery), diluted in the assay buffer. Titrations of nonbiotinylated, triacetylated histone H4 peptide were prepared, from a starting concentration of 1.25 mM, before adding to each of the ATAD2 construct–histone H4 peptide mixtures in a low-volume black 384-well microplate (Greiner Bio-One). Microplates were then centrifuged for 1 min at 1000 rpm before incubation at room temperature for 30 min, taking care to minimize both evaporation and exposure to light. Detection solution, prepared in assay buffer and consisting of 1.25 nM europium chelate-labeled anti-6His antibody (PerkinElmer, W1024, AD0111) and 40 nM LANCE APC surelight–streptavidin (PerkinElmer, CR130-100), was added to each well. Competition between biotinylated and nonbiotinylated histone H4 peptide for binding to each of the ATAD2 constructs was detected on a TRF laser equipped with a PerkinElmer Envision multimode plate reader by excitation at 337 nm, before measuring fluorescence emission at both 615 and 665 nm. The TR-FRET ratio was calculated using the equation: ratio = ((acceptor fluorescence at 665 nm)/(donor fluorescence at 615 nm)) × 1000.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Evaluation of Small-Molecule Binding to Newly Prepared ATAD2 Constructs by TR-FRET</h3><div class="NLM_p last">The following assay protocol was used to screen the robustness and bromodomain toolsets against ATAD2 constructs 1–10. Fifty nL of 1 mM test compound in 100% DMSO was added to a low-volume 384-well microplate using a Labcyte Echo 555 acoustic dispenser. A Thermo Scientific Multidrop Combi was used to dispense 5 μL of a premixed solution of ATAD2 bromodomain construct and triacetylated histone H4 peptide, prepared in assay buffer (described above). The concentrations of ATAD2 protein and triacetylated histone H4 peptide were determined from the nonbiotinylated peptide competition studies and are detailed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S5</a>, Supporting Information. After equilibrating the ATAD2 bromodomain construct and triacetylated histone H4 peptide reagents with a test compound for 30 min in the dark at room temperature, small-molecule competition with the bromodomain protein/biotinylated H4 peptide interaction was monitored using TR-FRET. Detection solution (described above) was added to the 384-well microplate in an equal volume to the protein/peptide solution using the same liquid-handling dispenser. The TR-FRET ratio was measured and calculated as described above. To normalize the assay-specific response in compound screening experiments, the means of the specific microplate positive and negative controls were utilized to convert the response to a percentage inhibition (%<i>I</i>) by applying the equation %<i>I</i> = ((μ<sub>1</sub> – <i>X</i>)/(μ<sub>1</sub> – μ<sub>2</sub>)) × 100 (where <i>X</i> is the assay-specific response, μ<sub>1</sub> is the mean of the negative, or uninhibited control, and μ<sub>2</sub> is the mean of the positive, or inhibited control). Compound screening assay quality was monitored using two control populations on every microplate tested to calculate a “<i>Z</i>′-factor”, defined by the equation <i>Z</i>′ = 1 – (((3 × σ<sub>1</sub>) + (3 × σ<sub>2</sub>))/(μ<sub>1</sub> – μ<sub>2</sub>)) (where σ<sub>1</sub> and σ<sub>2</sub> are the standard deviations of the positive and negative controls, respectively, μ<sub>1</sub> is the mean of the negative, or uninhibited control, and μ<sub>2</sub> is the mean of the positive, or inhibited control).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Isothermal Titration Calorimetry</h3><div class="NLM_p">ATAD2 protein was dialyzed into 50 mM HEPES pH 7.5, 500 mM NaCl, 1 mM tris(2-carboxyethyl)phosphine (TCEP) to give ∼100 μM working solutions. Triacetylated histone H4 peptides were diluted at least 10-fold from a high-concentration aqueous stock solution into 50 mM HEPES pH 7.5, 500 mM NaCl, 1 mM TCEP to give 0.5–2 mM working solutions. For protein constructs and peptide sequences, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S1</a>, Supporting Information.</div><div class="NLM_p">ITC data were acquired at 15 °C on a MicroCal auto-iTC. ATAD2 solutions (350 μL) and 140 μL of the peptide solutions were added to wells of a NUNC 96 × 1 mL deep-well plate. The plate was transferred to the plate tray of the auto-iTC instrument at 10 °C. A total of 19 injections of the peptide into ATAD2 protein were made at a reference power of 6 μCal/s, with an initial delay of 60 s and a stirring speed of 500 rpm, in a high-feedback mode. The first injection was 0.5 μL of duration 1 s, with a 120 s spacing and a 2 s filter period. All subsequent injections used 2 μL volumes with a duration of 4 s and a spacing of 120 and 5 s filter periods.</div><div class="NLM_p last">Raw data binding isotherms were exported into Microcal Origin and fitted to a one-site binding model.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> X-ray Crystallography</h3><div class="NLM_p last">Apo crystals of detagged ATAD2(981–1108) were typically grown at 4 °C in 96-well MRC plates using well solutions under a range of conditions spanning 0.1 M Tris–HCl pH 7.0–8.0, 1.2–1.5 M ammonium sulfate, and 20–25% poly(ethylene glycol)3350 at a protein concentration of 5–6 mg/mL. The sitting drops were made up of a 3:1 ratio of protein/well solution, and the hexagonal crystals often grew from phase-separated drops. To obtain crystal structures of ATAD2 complexed with inhibitors by soaking, compounds were soaked into apo crystals overnight either from solid or using 1–5 mM ligand diluted from a 100–200 mM DMSO stock solution. Soaked crystals were harvested in a cryo-loop and briefly transferred into a cryoprotectant buffer of 20% ethylene glycol + 80% well solution before being plunge-frozen in liquid nitrogen. Data from single crystals were collected at 100 K at the Diamond Light Source (DLS, Oxford) or on an in-house RIGAKU FR-E+ SUPERBRIGHT/Saturn A200 detector/ACTOR robotic system. Data processing was achieved using XDS<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> within AUTOPROC (Global Phasing Limited) and scaled using SCALA<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> or AIMLESS<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> within the CCP4 programming suite.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Structures were solved by Fourier synthesis using REFMAC<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> via CCP4 starting from a previously determined in-house structure. Model building was performed using COOT<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> and refined using REFMAC. Statistics for the data collection and refined coordinates, as well as density maps, are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Table S3 and Figure S9</a>, Supporting Information. Crystal structures have been deposited in the Protein Data Bank with accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55">6S55</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S56">6S56</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S57">6S57</a>.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Quantum Mechanics Calculations</h3><div class="NLM_p last">Crystal structures were prepared using the Schrödinger Maestro protein preparation tool.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Water molecules apart from the two closest to the inhibitor (W5 and W6, <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>e) were deleted, and hydrogen placement and H-bond assignment were performed using PROPKA at pH 7.0. Water orientations were sampled, and the preferred orientations were checked for consistency between the three complexes. Minimization was then carried out under the OPLS3 force field with heavy atoms restrained to 0.3 Å root-mean-square deviation. Residues apart from those shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>b were deleted, and the remainder capped with methyl groups. The three ligands were truncated to the common substructure shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>a, extracted from the protein, and optimized in Jaguar (M06-2X/CC-PVTZ-PP), keeping the red substructure atoms fixed before merging them back into the protein structure.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Frontier molecular orbital (FMO) calculations were set up using Facio (v19.2.1), and energy decomposition analysis was performed using GAMESS (v11) using SCS-MP2/6-31G**.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61,62)</a> Counterpoise binding energies between each ligand and selected binding site residues were calculated using a posteriori-corrected DFT and basis sets listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">Figure S11b</a>, Supporting Information. Electrostatic potential maps for inhibitor fragments and protein residues were generated using single-point gas-phase calculations (M06-2X-D3/LACV3P**<sup>++</sup>).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00673" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66787" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66787" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00673" class="ext-link">10.1021/acs.jmedchem.9b00673</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">ATAD2 constructs and evaluation; full ATAD2 and BET TR-FRET assay data; X-ray refinement statistics and density maps; peptide binding data; robustness set and bromodomain toolset %<i>I</i> summaries; SPR and NMR qualification of representative hits; electrostatic potential surfaces; QM-calculated fragment interaction energies; and LC/MS spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf">jm9b00673_si_001.pdf (3.35 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_002.csv">jm9b00673_si_002.csv (2.22 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Crystallography structures are deposited as <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55">6S55</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S56">6S56</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S57">6S57</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00673" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65785" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65785" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Bamborough</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#82f2e3f7eeace3ace0e3efe0edf0edf7e5eac2e5f1e9ace1edef"><span class="__cf_email__" data-cfemail="631302160f4d024d01020e010c110c16040b230410084d000c0e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#187b706d76356f793670367b706d767f587f6b73367b7775"><span class="__cf_email__" data-cfemail="ccafa4b9a2e1bbade2a4e2afa4b9a2ab8cabbfa7e2afa3a1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emmanuel H. Demont</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Angela M. Bridges</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter D. Craggs</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David P. Dixon</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Francis</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rebecca C. Furze</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Research Unit and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Grandi</span> - <span class="hlFld-Affiliation affiliation">Cellzome, Meyerhofstrasse 1, Heidelberg 69117, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma J. Jones</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bhumika Karamshi</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kelly Locke</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon C. C. Lucas</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne-Marie Michon</span> - <span class="hlFld-Affiliation affiliation">Cellzome, Meyerhofstrasse 1, Heidelberg 69117, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren J. Mitchell</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Pogány</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Research Unit and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christina Rau</span> - <span class="hlFld-Affiliation affiliation">Cellzome, Meyerhofstrasse 1, Heidelberg 69117, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ana Maria Roa</span> - <span class="hlFld-Affiliation affiliation">GlaxoSmithKline
Tres Cantos, 28760 Tres Cantos, Madrid, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew D. Roberts</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Sheppard</span> - <span class="hlFld-Affiliation affiliation">Epigenetics Research Unit and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span>; 
    <span>Present Address:
                        Medicinal Chemistry, Cardiovascular, Renal, and Metabolic Diseases, IMED Biotech Unit, AstraZeneca, Gothenburg 431 50, Sweden (R.J.S.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Watson</span> - <span class="hlFld-Affiliation affiliation">Medicinal Science and Technology and , GlaxoSmithKline Research and Development, Medicines
Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, United Kingdom</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors are present or former employees of GlaxoSmithKline.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3785-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12040" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12040" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Alberto Bejarano, Gonzalo Colmenarajo, María Eugenia Cruz, and Nicola Richmond for running and analyzing the HTS, Tony Cooper and Heather Barnett of the arrays group, and Amanda Emmons, Paul White, and all of the rest of the ATAD2 team.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ANCCA</td><td class="NLM_def"><p class="first last">AAA nuclear coregulator cancer-associated protein</p></td></tr><tr><td class="NLM_term">ATAD2</td><td class="NLM_def"><p class="first last">ATPase family AAA domain-containing 2</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extraterminal domain</p></td></tr><tr><td class="NLM_term">BRD<i>x</i></td><td class="NLM_def"><p class="first last">bromodomain-containing protein <i>x</i></p></td></tr><tr><td class="NLM_term">BPTF</td><td class="NLM_def"><p class="first last">bromodomain PHD finger transcription factor</p></td></tr><tr><td class="NLM_term">BRPF1/2/3</td><td class="NLM_def"><p class="first last">bromodomain and PHD finger-containing 1/2/3</p></td></tr><tr><td class="NLM_term">CECR2</td><td class="NLM_def"><p class="first last">cat eye syndrome chromosome region, candidate 2</p></td></tr><tr><td class="NLM_term">CREBBP</td><td class="NLM_def"><p class="first last">CREB binding protein</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetyl lysine</p></td></tr><tr><td class="NLM_term">pIC<sub>50</sub></td><td class="NLM_def"><p class="first last">–log 10 IC<sub>50</sub></p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">Surface plasmon resonance</p></td></tr><tr><td class="NLM_term">TAF</td><td class="NLM_def"><p class="first last">TATA box binding protein (TBP)-associated factor</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72870" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72870" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 62 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.04.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.addr.2016.04.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=27154268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=34-41&author=C.+A.+Lipinski&title=Rule+of+five+in+2015+and+beyond%3A+Target+and+ligand+structural+limitations%2C+ligand+chemistry+structure+and+drug+discovery+project+decisions&doi=10.1016%2Fj.addr.2016.04.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34-41</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The rule of five (Ro5), based on physicochem. profiles of phase II drugs, is consistent with structural limitations in protein targets and the drug target ligands.  Three of four parameters in Ro5 are fundamental to the structure of both target and drug binding sites.  The chem. structure of the drug ligand depends on the ligand chem. and design philosophy.  Two extremes of chem. structure and design philosophy exist; ligands constructed in the medicinal chem. synthesis lab. without input from natural selection and natural product (NP) metabolites biosynthesized based on evolutionary selection.  Exceptions to Ro5 are found mostly among NPs.  Chem. chameleon-like behavior of some NPs due to intra-mol. hydrogen bonding as exemplified by cyclosporine A is a strong contributor to NP Ro5 outliers.  The fragment derived, drug Navitoclax is an example of the extensive expertise, resources, time and key decisions required for the rare discovery of a non-NP Ro5 outlier.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG3g5NBQZuebVg90H21EOLACvtfcHk0lh3B9N2xCW26A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D&md5=72eeb37cd0cd2442ef5c7a199b61f1e3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.04.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.04.029%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DRule%2520of%2520five%2520in%25202015%2520and%2520beyond%253A%2520Target%2520and%2520ligand%2520structural%2520limitations%252C%2520ligand%2520chemistry%2520structure%2520and%2520drug%2520discovery%2520project%2520decisions%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2016%26volume%3D101%26spage%3D34%26epage%3D41%26doi%3D10.1016%2Fj.addr.2016.04.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiessling, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">The ecstasy and agony of assay interference compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2165</span>– <span class="NLM_lpage">2168</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00229</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00229" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtl2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2165-2168&author=C.+Aldrichauthor=C.+Bertozziauthor=G.+I.+Georgauthor=L.+Kiesslingauthor=C.+Lindsleyauthor=D.+Liottaauthor=K.+M.+Merzauthor=A.+Schepartzauthor=S.+Wang&title=The+ecstasy+and+agony+of+assay+interference+compounds&doi=10.1021%2Facs.jmedchem.7b00229"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Ecstasy and Agony of Assay Interference Compounds</span></div><div class="casAuthors">Aldrich, Courtney; Bertozzi, Carolyn; Georg, Gunda I.; Kiessling, Laura; Lindsley, Craig; Liotta, Dennis; Merz, Kenneth M.; Schepartz, Alanna; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2165-2168</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ecstasy of discovering a new hit from screening can lead to a highly productive research effort to discover new bioactive compds.  However, in too many cases this ecstasy is followed by the agony of realizing that the compds. are not active against the desired target.  Many of these false hits are Pan Assay INterference compds. (PAINS) 1 or colloidal aggregators.  Whether the screen is conducted in silico or in the lab. and whether screening libraries, natural products, or drugs are used, all discovery efforts that rely on some form of screening to identify bioactivity are susceptible to this phenomenon.  Studies that omit crit. controls against exptl. artifacts caused by PAINS may waste years of research effort as useless compds. are progressed.  The American Chem. Society (ACS) is eager to alert the scientific community to this problem and to recommend protocols that will eliminate the publication of research articles based on compds. with artificial activity.  This editorial aims to summarize relevant concepts and to set the framework by which relevant ACS journals will address this issue going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJUf5u51VEpbVg90H21EOLACvtfcHk0lgREhrK2vElFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtl2lsLY%253D&md5=64bf087659a47fe7769208217ef8c989</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00229%26sid%3Dliteratum%253Aachs%26aulast%3DAldrich%26aufirst%3DC.%26aulast%3DBertozzi%26aufirst%3DC.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DKiessling%26aufirst%3DL.%26aulast%3DLindsley%26aufirst%3DC.%26aulast%3DLiotta%26aufirst%3DD.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DSchepartz%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520ecstasy%2520and%2520agony%2520of%2520assay%2520interference%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2165%26epage%3D2168%26doi%3D10.1021%2Facs.jmedchem.7b00229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannifer, A.</span></span> <span> </span><span class="NLM_article-title">The importance of triaging in determining the quality of output from high-throughput screening</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1847</span>– <span class="NLM_lpage">1852</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.4155%2Ffmc.15.121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=26419190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1847-1852&author=P.+Jonesauthor=S.+McElroyauthor=A.+Morrisonauthor=A.+Pannifer&title=The+importance+of+triaging+in+determining+the+quality+of+output+from+high-throughput+screening&doi=10.4155%2Ffmc.15.121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The importance of triaging in determining the quality of output from high-throughput screening</span></div><div class="casAuthors">Jones, Philip; McElroy, Stuart; Morrison, Angus; Pannifer, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1847-1852</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ5lVzKTZCFrVg90H21EOLACvtfcHk0ljPbc6sprXxzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsL3K&md5=0fa12f27c4deea2e56ec799470534462</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.121%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%26aulast%3DMcElroy%26aufirst%3DS.%26aulast%3DMorrison%26aufirst%3DA.%26aulast%3DPannifer%26aufirst%3DA.%26atitle%3DThe%2520importance%2520of%2520triaging%2520in%2520determining%2520the%2520quality%2520of%2520output%2520from%2520high-throughput%2520screening%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D1847%26epage%3D1852%26doi%3D10.4155%2Ffmc.15.121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Seven year itch: pan-assay interference compounds (PAINS) in 2017-utility and limitations</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00903</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00903" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=36-44&author=J.+B.+Baellauthor=J.+W.+M.+Nissink&title=Seven+year+itch%3A+pan-assay+interference+compounds+%28PAINS%29+in+2017-utility+and+limitations&doi=10.1021%2Facschembio.7b00903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017-Utility and Limitations</span></div><div class="casAuthors">Baell, Jonathan B.; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on all necessary considerations for the appropriate use of PAINs filters formulated to process hundreds and thousands of compds. in seconds, and identify PAINS in order to exclude them from further anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocZ7E_j7zxfbVg90H21EOLACvtfcHk0livXFErzd73VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK&md5=051115704137d6a91b8271702c619682</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00903%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DSeven%2520year%2520itch%253A%2520pan-assay%2520interference%2520compounds%2520%2528PAINS%2529%2520in%25202017-utility%2520and%2520limitations%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D36%26epage%3D44%26doi%3D10.1021%2Facschembio.7b00903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Folmer, R. H.</span></span> <span> </span><span class="NLM_article-title">Integrating biophysics with HTS-driven drug discovery projects</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.drudis.2016.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=26826422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2qtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=491-498&author=R.+H.+Folmer&title=Integrating+biophysics+with+HTS-driven+drug+discovery+projects&doi=10.1016%2Fj.drudis.2016.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating biophysics with HTS-driven drug discovery projects</span></div><div class="casAuthors">Folmer, Rutger H. A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">491-498</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Over the past decade biophysics has become an established discipline in HTS hit triaging, owing to its high fidelity in detecting protein-ligand interactions.  Many pharma companies are using biophys. techniques to filter HTS output for false positives, as will be discussed in this review.  Moreover, I will demonstrate how the earlier application of biophysics, already at the HTS assay development stage, is potentially even more impactful.  Two key areas here are early mode-of-action studies and ensuring that the HTS assay and subsequent cascade are fit for purpose.  Top-level results from 20 inhouse projects are shown to underpin the impact of these studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZXQLSut8FrVg90H21EOLACvtfcHk0livXFErzd73VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2qtbs%253D&md5=88dc581aa9eb04c42453bc200a90a705</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DFolmer%26aufirst%3DR.%2BH.%26atitle%3DIntegrating%2520biophysics%2520with%2520HTS-driven%2520drug%2520discovery%2520projects%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D491%26epage%3D498%26doi%3D10.1016%2Fj.drudis.2016.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span> <span> </span><span class="NLM_article-title">Biophysics in drug discovery: impact, challenges and opportunities</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">679</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1038%2Fnrd.2016.123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=27516170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=679-698&author=J.+P.+Renaudauthor=C.+W.+Chungauthor=U.+H.+Danielsonauthor=U.+Egnerauthor=M.+Hennigauthor=R.+E.+Hubbardauthor=H.+Nar&title=Biophysics+in+drug+discovery%3A+impact%2C+challenges+and+opportunities&doi=10.1038%2Fnrd.2016.123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysics in drug discovery: impact, challenges and opportunities</span></div><div class="casAuthors">Renaud, Jean-Paul; Chung, Chun-wa; Danielson, U. Helena; Egner, Ursula; Hennig, Michael; Hubbard, Roderick E.; Nar, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">679-698</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past 25 years, biophys. technologies such as X-ray crystallog., NMR spectroscopy, surface plasmon resonance spectroscopy and isothermal titrn. calorimetry have become key components of drug discovery platforms in many pharmaceutical companies and academic labs.  There have been great improvements in the speed, sensitivity and range of possible measurements, providing high-resoln. mechanistic, kinetic, thermodn. and structural information on compd.-target interactions.  This Review provides a framework to understand this evolution by describing the key biophys. methods, the information they can provide and the ways in which they can be applied at different stages of the drug discovery process.  We also discuss the challenges for current technologies and future opportunities to use biophys. methods to solve drug discovery problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq91r76-q9prbVg90H21EOLACvtfcHk0lhY7lN93HbXNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7I&md5=1a73f14def6945e9806f57f1955f56d4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.123%26sid%3Dliteratum%253Aachs%26aulast%3DRenaud%26aufirst%3DJ.%2BP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DHennig%26aufirst%3DM.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DNar%26aufirst%3DH.%26atitle%3DBiophysics%2520in%2520drug%2520discovery%253A%2520impact%252C%2520challenges%2520and%2520opportunities%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D679%26epage%3D698%26doi%3D10.1038%2Fnrd.2016.123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERalpha, is required for coregulator occupancy and chromatin modification</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">18067</span>– <span class="NLM_lpage">18072</span>, <span class="refDoi"> DOI: 10.1073/pnas.0705814104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1073%2Fpnas.0705814104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=17998543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=18067-18072&author=J.+X.+Zouauthor=A.+S.+Revenkoauthor=L.+B.+Liauthor=A.+T.+Gemoauthor=H.+W.+Chen&title=ANCCA%2C+an+estrogen-regulated+AAA%2B+ATPase+coactivator+for+ERalpha%2C+is+required+for+coregulator+occupancy+and+chromatin+modification&doi=10.1073%2Fpnas.0705814104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERα, is required for coregulator occupancy and chromatin modification</span></div><div class="casAuthors">Zou, June X.; Revenko, Alexey S.; Li, Li B.; Gemo, Abigael T.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18067-18072</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">AAA+ proteins play crucial roles in diverse biol. processes via their ATPase-driven remodeling of macromol. complexes.  Here the authors report the authors' identification of an evolutionarily conserved AAA+ protein, ANCCA/Pro 2000, endowed with a bromodomain that is strongly induced by estrogen in human breast cancer cells and is a direct target of protooncogene ACTR/AlB1/SRC-3.  The authors found that ANCCA assocs. directly with estrogen-bound estrogen receptor (ER) α and ACTR.  It is selectively recruited, upon estrogen stimulation, to a subset of ERα target genes including cyclin D1, c-myc, and E2F1 and is required for their estrogen-induced expression as well as breast cancer cell proliferation.  Further studies indicate that ANCCA binds and hydrolyzes ATP and is crit. for recruitment of coregulator CBP and histone hyperacetylation at the ER target chromatin.  Moreover, mutations at the ATP binding motifs rendered ANCCA defective as a coactivator in mediating estrogen induction of gene expression.  Together, the authors' findings reveal an unexpected layer of regulatory mechanism in hormone signaling mediated by ANCCA and suggest that hormone-induced assembly of transcriptional coregulator complexes at chromatin is a process facilitated by AAA+ ATPase proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NFjjiBVoRbVg90H21EOLACvtfcHk0lhY7lN93HbXNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO&md5=618fcc431bf4b0a7589213fea55732ca</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0705814104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0705814104%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DLi%26aufirst%3DL.%2BB.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DANCCA%252C%2520an%2520estrogen-regulated%2520AAA%252B%2520ATPase%2520coactivator%2520for%2520ERalpha%252C%2520is%2520required%2520for%2520coregulator%2520occupancy%2520and%2520chromatin%2520modification%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D18067%26epage%3D18072%26doi%3D10.1073%2Fpnas.0705814104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardiff, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">ANCCA/ATAD2 overexpression identifies breast cancer patients with poor prognosis, acting to drive proliferation and survival of triple-negative cells through control of B-Myb and EZH2</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">9402</span>– <span class="NLM_lpage">9412</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-1199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1158%2F0008-5472.CAN-10-1199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=20864510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWmsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=9402-9412&author=E.+V.+Kalashnikovaauthor=A.+S.+Revenkoauthor=A.+T.+Gemoauthor=N.+P.+Andrewsauthor=C.+G.+Tepperauthor=J.+X.+Zouauthor=R.+D.+Cardiffauthor=A.+D.+Borowskyauthor=H.+W.+Chen&title=ANCCA%2FATAD2+overexpression+identifies+breast+cancer+patients+with+poor+prognosis%2C+acting+to+drive+proliferation+and+survival+of+triple-negative+cells+through+control+of+B-Myb+and+EZH2&doi=10.1158%2F0008-5472.CAN-10-1199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA/ATAD2 Overexpression Identifies Breast Cancer Patients with Poor Prognosis, Acting to Drive Proliferation and Survival of Triple-Negative Cells through Control of B-Myb and EZH2</span></div><div class="casAuthors">Kalashnikova, Ekaterina V.; Revenko, Alexey S.; Gemo, Abigael T.; Andrews, Nicolas P.; Tepper, Clifford G.; Zou, June X.; Cardiff, Robert D.; Borowsky, Alexander D.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9402-9412</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Chromatin coregulators are important factors in tumorigenesis and cancer progression.  ANCCA is an AAA+ ATPase and a bromodomain-contg. nuclear coactivator for the estrogen and androgen receptors that is crucial for assembly of chromatin-modifying complexes and proliferation of hormone-responsive cancer cells.  In this study, we show that ANCCA is overexpressed in >70% of breast tumors and that its high protein level correlates well with tumor histol. grades (P < 0.0001), highlighting ANCCA as a prognostic factor for poor overall survival and disease recurrence.  Strikingly, high-level ANCCA correlated with triple-neg. tumors that represent highly aggressive disease.  Anal. of ANCCA transcript levels in multiple expression profiles of breast cancer identified ANCCA as a common signature gene, indicating that elevated transcripts also strongly correlate with tumor metastasis and poor survival.  Biol. and mechanistic investigations revealed that ANCCA is crucial for proliferation and survival of triple-neg./basal-like cancer cells and that it controls the expression of B-Myb, histone methyltransferase EZH2, and an Rb-E2F core program for proliferation, along with a subset of key mitotic kinesins and cell survival genes (IRS2, VEGF, and Akt1).  In particular, ANCCA overexpression correlated strongly with EZH2 in tumors.  Our results suggest that ANCCA may integrate multiple oncogenic programs in breast cancer, serving in particular as a prognostic marker and a therapeutic target for triple-neg. cancers.  Cancer Res; 70(22); 9402-12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-adY-FJac6rVg90H21EOLACvtfcHk0lj1BiqddLjkZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWmsLnP&md5=d7f74575c9c8cf4ebebdcfa5e3cc45fc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-1199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-1199%26sid%3Dliteratum%253Aachs%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DAndrews%26aufirst%3DN.%2BP.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DCardiff%26aufirst%3DR.%2BD.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DANCCA%252FATAD2%2520overexpression%2520identifies%2520breast%2520cancer%2520patients%2520with%2520poor%2520prognosis%252C%2520acting%2520to%2520drive%2520proliferation%2520and%2520survival%2520of%2520triple-negative%2520cells%2520through%2520control%2520of%2520B-Myb%2520and%2520EZH2%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D9402%26epage%3D9412%26doi%3D10.1158%2F0008-5472.CAN-10-1199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lestrat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoffier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virolle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">5171</span>– <span class="NLM_lpage">5181</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1038%2Fonc.2010.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=20581866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=5171-5181&author=C.+Caronauthor=C.+Lestratauthor=S.+Marsalauthor=E.+Escoffierauthor=S.+Curtetauthor=V.+Virolleauthor=P.+Barbryauthor=A.+Debernardiauthor=C.+Brambillaauthor=E.+Brambillaauthor=S.+Rousseauxauthor=S.+Khochbin&title=Functional+characterization+of+ATAD2+as+a+new+cancer%2Ftestis+factor+and+a+predictor+of+poor+prognosis+in+breast+and+lung+cancers&doi=10.1038%2Fonc.2010.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span></div><div class="casAuthors">Caron, C.; Lestrat, C.; Marsal, S.; Escoffier, E.; Curtet, S.; Virolle, V.; Barbry, P.; Debernardi, A.; Brambilla, C.; Brambilla, E.; Rousseaux, S.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5171-5181</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells frequently express genes normally active in male germ cells.  ATAD2 is one of them encoding a conserved factor harboring an AAA type ATPase domain and a bromodomain.  We show here that ATAD2 is highly expressed in testis as well as in many cancers of different origins and that its high expression is a strong predictor of rapid mortality in lung and breast cancers.  These observations suggest that ATAD2 acts on upstream and basic cellular processes to enhance oncogenesis in a variety of unrelated cell types.  Accordingly, our functional studies show that ATAD2 controls chromatin dynamics, genome transcriptional activities and apoptotic cell response.  We could also highlight some of the important intrinsic properties of its two regulatory domains, including a functional cross-talk between the AAA ATPase domain and the bromodomain.  Altogether, these data indicate that ATAD2 overexpression in somatic cells, by acting on basic properties of chromatin, may contribute to malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotoD3fqS5d4LVg90H21EOLACvtfcHk0lj1BiqddLjkZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D&md5=1eaa99ca9fd31e5a486bedd4d0f6178d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.259%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DLestrat%26aufirst%3DC.%26aulast%3DMarsal%26aufirst%3DS.%26aulast%3DEscoffier%26aufirst%3DE.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVirolle%26aufirst%3DV.%26aulast%3DBarbry%26aufirst%3DP.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DBrambilla%26aufirst%3DC.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DFunctional%2520characterization%2520of%2520ATAD2%2520as%2520a%2520new%2520cancer%252Ftestis%2520factor%2520and%2520a%2520predictor%2520of%2520poor%2520prognosis%2520in%2520breast%2520and%2520lung%2520cancers%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26spage%3D5171%26epage%3D5181%26doi%3D10.1038%2Fonc.2010.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morozumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boussouar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidikia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petosa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Dieuleveult, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitte, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabatel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosset, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoeyen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emadali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guardiola, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Atad2 is a generalist facilitator of chromatin dynamics in embryonic stem cells</span>. <i>J. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1093/jmcb/mjv060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1093%2Fjmcb%2Fmjv060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=26459632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVKntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=349-362&author=Y.+Morozumiauthor=F.+Boussouarauthor=M.+Tanauthor=A.+Chaikuadauthor=M.+Jamshidikiaauthor=G.+Colakauthor=H.+Heauthor=L.+Nieauthor=C.+Petosaauthor=M.+de+Dieuleveultauthor=S.+Curtetauthor=A.+L.+Vitteauthor=C.+Rabatelauthor=A.+Debernardiauthor=F.+L.+Cossetauthor=E.+Verhoeyenauthor=A.+Emadaliauthor=N.+Schweiferauthor=D.+Gianniauthor=M.+Gutauthor=P.+Guardiolaauthor=S.+Rousseauxauthor=M.+Gerardauthor=S.+Knappauthor=Y.+Zhaoauthor=S.+Khochbin&title=Atad2+is+a+generalist+facilitator+of+chromatin+dynamics+in+embryonic+stem+cells&doi=10.1093%2Fjmcb%2Fmjv060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Atad2 is a generalist facilitator of chromatin dynamics in embryonic stem cells</span></div><div class="casAuthors">Morozumi, Yuichi; Boussouar, Faycal; Tan, Minjia; Chaikuad, Apirat; Jamshidikia, Mahya; Colak, Gozde; He, Huang; Nie, Litong; Petosa, Carlo; de Dieuleveult, Maud; Curtet, Sandrine; Vitte, Anne-Laure; Rabatel, Clothilde; Debernardi, Alexandra; Cosset, Francois-Loic; Verhoeyen, Els; Emadali, Anouk; Schweifer, Norbert; Gianni, Davide; Gut, Marta; Guardiola, Philippe; Rousseaux, Sophie; Gerard, Matthieu; Knapp, Stefan; Zhao, Yingming; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-362</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although the conserved AAA ATPase and bromodomain factor, ATAD2, has been described as a transcriptional co-activator upregulated in many cancers, its function remains poorly understood.  Here, using a combination of ChIP-seq, ChIP-proteomics, and RNA-seq expts. in embryonic stem cells where Atad2 is normally highly expressed, we found that Atad2 is an abundant nucleosome-bound protein present on active genes, assocd. with chromatin remodelling, DNA replication, and DNA repair factors.  A structural anal. of its bromodomain and subsequent investigations demonstrate that histone acetylation guides ATAD2 to chromatin, resulting in an overall increase of chromatin accessibility and histone dynamics, which is required for the proper activity of the highly expressed gene fraction of the genome.  While in exponentially growing cells Atad2 appears dispensable for cell growth, in differentiating ES cells Atad2 becomes crit. in sustaining specific gene expression programs, controlling proliferation and differentiation.  Altogether, this work defines Atad2 as a facilitator of general chromatin-templated activities such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMr0U-BpiFIbVg90H21EOLACvtfcHk0ljfhVBfSqkIuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVKntbs%253D&md5=5a84cdc73ed9b7abeb5595425d07d3eb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjv060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjv060%26sid%3Dliteratum%253Aachs%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DColak%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DNie%26aufirst%3DL.%26aulast%3DPetosa%26aufirst%3DC.%26aulast%3Dde%2BDieuleveult%26aufirst%3DM.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVitte%26aufirst%3DA.%2BL.%26aulast%3DRabatel%26aufirst%3DC.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DCosset%26aufirst%3DF.%2BL.%26aulast%3DVerhoeyen%26aufirst%3DE.%26aulast%3DEmadali%26aufirst%3DA.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DGianni%26aufirst%3DD.%26aulast%3DGut%26aufirst%3DM.%26aulast%3DGuardiola%26aufirst%3DP.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DGerard%26aufirst%3DM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DAtad2%2520is%2520a%2520generalist%2520facilitator%2520of%2520chromatin%2520dynamics%2520in%2520embryonic%2520stem%2520cells%26jtitle%3DJ.%2520Mol.%2520Cell%2520Biol.%26date%3D2016%26volume%3D8%26spage%3D349%26epage%3D362%26doi%3D10.1093%2Fjmcb%2Fmjv060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sokolov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Kinesin family deregulation coordinated by bromodomain protein ANCCA and histone methyltransferase MLL for breast cancer cell growth, survival, and tamoxifen resistance</span>. <i>Mol. Cancer Res</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-13-0459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1158%2F1541-7786.MCR-13-0459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=24391143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=539-549&author=J.+X.+Zouauthor=Z.+Duanauthor=J.+Wangauthor=A.+Sokolovauthor=J.+Xuauthor=C.+Z.+Chenauthor=J.+J.+Liauthor=H.+W.+Chen&title=Kinesin+family+deregulation+coordinated+by+bromodomain+protein+ANCCA+and+histone+methyltransferase+MLL+for+breast+cancer+cell+growth%2C+survival%2C+and+tamoxifen+resistance&doi=10.1158%2F1541-7786.MCR-13-0459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Kinesin Family Deregulation Coordinated by Bromodomain Protein ANCCA and Histone Methyltransferase MLL for Breast Cancer Cell Growth, Survival, and Tamoxifen Resistance</span></div><div class="casAuthors">Zou, June X.; Duan, Zhijian; Wang, Junjian; Sokolov, Alex; Xu, Jianzhen; Chen, Christopher Z.; Li, Jian Jian; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">539-549</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Kinesins are a superfamily of motor proteins and often deregulated in different cancers.  However, the mechanism of their deregulation has been poorly understood.  Through examg. kinesin gene family expression in estrogen receptor (ER)-pos. breast cancer cells, we found that estrogen stimulation of cancer cell proliferation involves a concerted regulation of specific kinesins.  Estrogen strongly induces expression of 19 kinesin genes such as Kif4A/4B, Kif5A/5B, Kif10, Kif11, Kif15, Kif18A/18B, Kif20A/20B, Kif21, Kif23, Kif24, Kif25, and KifC1, whereas suppresses the expression of seven others, including Kif1A, Kif1C, Kif7, and KifC3.  Interestingly, the bromodomain protein ANCCA/ATAD2, previously shown to be an estrogen-induced chromatin regulator, plays a crucial role in the up- and downregulation of kinesins by estrogen.  Its overexpression drives estrogen-independent upregulation of specific kinesins.  Mechanistically, ANCCA (AAA nuclear coregulator cancer assocd.) mediates E2-dependent recruitment of E2F and MLL1 histone methyltransferase at kinesin gene promoters for gene activation-assocd. H3K4me3 methylation.  Importantly, elevated levels of Kif4A, Kif15, Kif20A, and Kif23 correlate with that of ANCCA in the tumors and with poor relapse-free survival of patients with ER-pos. breast cancer.  Their knockdown strongly impeded proliferation and induced apoptosis of both tamoxifen-sensitive and resistant cancer cells.  Together, the study reveals ANCCA as a key mediator of kinesin family deregulation in breast cancer and the crucial role of multiple kinesins in growth and survival of the tumor cells.  Implications: These findings support the development of novel inhibitors of cancer-assocd. kinesins and their regulator ANCCA for effective treatment of cancers including tamoxifen-resistant breast cancers.  Mol Cancer Res; 12(4); 539-49. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaol_2gF7Tc7Vg90H21EOLACvtfcHk0ljfhVBfSqkIuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D&md5=aa5cda3daca4e8e08dd64827d72a313c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-13-0459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-13-0459%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSokolov%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BZ.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DKinesin%2520family%2520deregulation%2520coordinated%2520by%2520bromodomain%2520protein%2520ANCCA%2520and%2520histone%2520methyltransferase%2520MLL%2520for%2520breast%2520cancer%2520cell%2520growth%252C%2520survival%252C%2520and%2520tamoxifen%2520resistance%26jtitle%3DMol.%2520Cancer%2520Res%26date%3D2014%26volume%3D12%26spage%3D539%26epage%3D549%26doi%3D10.1158%2F1541-7786.MCR-13-0459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Chromatin loading of E2F-MLL complex by cancer-associated coregulator ANCCA via reading a specific histone mark</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">5260</span>– <span class="NLM_lpage">5272</span>, <span class="refDoi"> DOI: 10.1128/MCB.00484-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1128%2FMCB.00484-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=20855524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFOltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=5260-5272&author=A.+S.+Revenkoauthor=E.+V.+Kalashnikovaauthor=A.+T.+Gemoauthor=J.+X.+Zouauthor=H.+W.+Chen&title=Chromatin+loading+of+E2F-MLL+complex+by+cancer-associated+coregulator+ANCCA+via+reading+a+specific+histone+mark&doi=10.1128%2FMCB.00484-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin loading of E2F-MLL complex by cancer-associated coregulator ANCCA via reading a specific histone mark</span></div><div class="casAuthors">Revenko, Alexey S.; Kalashnikova, Ekaterina V.; Gemo, Abigael T.; Zou, June X.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5260-5272</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Histone modifications are regarded as the carrier of epigenetic memory through cell divisions.  How the marks facilitate cell cycle-dependent gene expression is poorly understood.  The evolutionarily conserved AAA ATPase ANCCA (AAA nuclear coregulator cancer-assocd. protein)/ATAD2 was identified as a direct target of oncogene AIB1/ACTR/SRC-3 and a transcriptional coregulator for estrogen and androgen receptors and is strongly implicated in tumorigenesis.  We report here that ANCCA directly interacts with E2F1 to E2F3 and that its N terminus interacts with both the N and C termini of E2F1.  ANCCA preferentially assocs. via its bromodomain with H3 acetylated at lysine 14 (H3K14ac) and is required for key cell cycle gene expression and cancer cell proliferation.  ANCCA assocs. with chromosomes at late mitosis, and its occupancy at E2F targets peaks at the G1-to-S transition.  Strikingly, ANCCA is required for recruitment of specific E2Fs to their targets and chromatin assembly of the host cell factor 1 (HCF-1)-MLL histone methyltransferase complex.  ANCCA depletion results in a marked decrease of the gene activation-linked H3K4me3 mark.  Bromodomain mutations disable ANCCA function as an E2F coactivator and its ability to promote cancer cell proliferation, while ANCCA overexpression in tumors correlates with tumor growth.  Together, these results suggest that ANCCA acts as a pioneer factor in E2F-dependent gene activation and that a novel mechanism involving ANCCA bromodomain may contribute to cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9jiEXxtPP3rVg90H21EOLACvtfcHk0lic8ddfepJ3kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFOltw%253D%253D&md5=958e54242a3a8c16ac4bf5b51acd80b8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FMCB.00484-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00484-10%26sid%3Dliteratum%253Aachs%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DChromatin%2520loading%2520of%2520E2F-MLL%2520complex%2520by%2520cancer-associated%2520coregulator%2520ANCCA%2520via%2520reading%2520a%2520specific%2520histone%2520mark%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2010%26volume%3D30%26spage%3D5260%26epage%3D5272%26doi%3D10.1128%2FMCB.00484-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Androgen-induced coactivator ANCCA mediates specific androgen receptor signaling in prostate cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">3339</span>– <span class="NLM_lpage">3346</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-3440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1158%2F0008-5472.CAN-08-3440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=19318566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksV2hsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=3339-3346&author=J.+X.+Zouauthor=L.+Guoauthor=A.+S.+Revenkoauthor=C.+G.+Tepperauthor=A.+T.+Gemoauthor=H.+J.+Kungauthor=H.+W.+Chen&title=Androgen-induced+coactivator+ANCCA+mediates+specific+androgen+receptor+signaling+in+prostate+cancer&doi=10.1158%2F0008-5472.CAN-08-3440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen-induced coactivator ANCCA mediates specific androgen receptor signaling in prostate cancer</span></div><div class="casAuthors">Zou, June X.; Guo, Linlang; Revenko, Alexey S.; Tepper, Clifford G.; Gemo, Abigael T.; Kung, Hsing-Jien; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3339-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Androgen receptor (AR) plays a pivotal role in prostate cancer, primarily by regulating different gene expression programs elicited by androgen, which is important for cancer cell proliferation, survival, and differentiation.  It is believed that the transcriptional function of AR is mediated largely by distinct nuclear coregulators.  We report here the identification of ANCCA (also known as ATAD2), a new member of the AAA+ ATPase family proteins, as a novel AR coactivator.  ANCCA interacts directly with AR and enhances its transcriptional activity, and is required for androgen-stimulated expression of a specific subgroup of genes including IGF1R, IRS-2, SGK1, and survivin.  Upon androgen stimulation, ANCCA together with AR is recruited to the specific AR target genes.  Suppression of ANCCA expression strongly inhibited the proliferation of androgen-responsive or androgen-independent, AR-pos. prostate cancer cells and caused a significant increase of cellular apoptosis.  Strikingly, the ANCCA gene itself, located at chromosome 8q24, is highly induced by androgen in androgen-dependent prostate cancer cells and xenograft tumors.  Although ANCCA is hardly detected in normal human prostate tissue, high levels of ANCCA are found in hormone-independent prostate cancer cell lines, xenograft tumor, and a subset of prostate cancers with high Gleason scores.  Together, these findings suggest that ANCCA plays an important role in prostate cancer by mediating specific AR functions in cancer cell survival and proliferation.  The possession of ATPase and bromodomain by ANCCA makes it an attractive target for the development of therapeutics for the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtieC3cr_jX7Vg90H21EOLACvtfcHk0lic8ddfepJ3kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksV2hsLY%253D&md5=daa42208c9c7019809bb491e5af1f152</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3440%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DKung%26aufirst%3DH.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DAndrogen-induced%2520coactivator%2520ANCCA%2520mediates%2520specific%2520androgen%2520receptor%2520signaling%2520in%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D3339%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-08-3440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, H. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ATAD2 in cancer: a pharmacologically challenging but tractable target</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1406921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1080%2F14728222.2018.1406921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29148850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=85-96&author=M.+Hussainauthor=Y.+Zhouauthor=Y.+Songauthor=H.+M.+A.+Hameedauthor=H.+Jiangauthor=Y.+Tuauthor=J.+Zhang&title=ATAD2+in+cancer%3A+a+pharmacologically+challenging+but+tractable+target&doi=10.1080%2F14728222.2018.1406921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 in cancer: a pharmacologically challenging but tractable target</span></div><div class="casAuthors">Hussain, Muzammal; Zhou, Yang; Song, Yu; Hameed, H. M. Adnan; Jiang, Hao; Tu, Yaoquan; Zhang, Jiancun</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-96</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  ATAD2 protein is an emerging oncogene that has strongly been linked to the etiol. of multiple advanced human cancers.  Therapeutically, despite the fact that genetic suppression/knockdown studies have validated it as a compelling drug target for future therapeutic development, recent druggability assessment data suggest that direct targeting of ATAD2's bromodomain (BRD) may be a very challenging task.  ATAD2's BRD has been predicted as a 'difficult to drug' or 'least druggable' target due to the concern that its binding pocket, and the areas around it, seem to be unfeasible for ligand binding.: In this review, after shedding light on the multifaceted roles of ATAD2 in normal physiol. as well as in cancer-etiol., we discuss tech. challenges rendered by ATAD2's BRD active site and the recent drug discovery efforts to find small mol. inhibitors against it.: The identification of a novel low-nanomolar semi-permeable chem. probe against ATAD2's BRD by recent drug discovery campaign has demonstrated it to be a pharmacol. tractable target.  Nevertheless, the development of high quality bioavailable inhibitors against ATAD2 is still a pending task.  Moreover, ATAD2 may also potentially be utilized as a promising target for future development of RNAi-based therapy to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0bdigfdvMjbVg90H21EOLACvtfcHk0lic8ddfepJ3kA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN&md5=5d0ff7c936bb9eff125efeeaa4d275d1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1406921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1406921%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DHameed%26aufirst%3DH.%2BM.%2BA.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DTu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DATAD2%2520in%2520cancer%253A%2520a%2520pharmacologically%2520challenging%2520but%2520tractable%2520target%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D85%26epage%3D96%26doi%3D10.1080%2F14728222.2018.1406921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Measures, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of bromodomain-acetyl-lysine interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1021/cb500996u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500996u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=22-39&author=M.+Brandauthor=A.+R.+Measuresauthor=B.+G.+Wilsonauthor=W.+A.+Cortopassiauthor=R.+Alexanderauthor=M.+Hossauthor=D.+S.+Hewingsauthor=T.+P.+Rooneyauthor=R.+S.+Patonauthor=S.+J.+Conway&title=Small+molecule+inhibitors+of+bromodomain-acetyl-lysine+interactions&doi=10.1021%2Fcb500996u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Bromodomain-Acetyl-lysine Interactions</span></div><div class="casAuthors">Brand, Michael; Measures, Angelina R.; Wilson, Brian G.; Cortopassi, Wilian A.; Alexander, Rikki; Hoss, Matthias; Hewings, David S.; Rooney, Timothy P. C.; Paton, Robert S.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-39</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Bromodomains are protein modules that bind to acetylated lysine residues.  Their interaction with histone proteins suggests that they function as "readers" of histone lysine acetylation, a component of the proposed "histone code".  Bromodomain-contg. proteins are often found as components of larger protein complexes with roles in fundamental cellular process including transcription.  The publication of two potent ligands for the BET bromodomains in 2010 demonstrated that small mols. can inhibit the bromodomain-acetyl-lysine protein-protein interaction.  These mols. display strong phenotypic effects in a no. of cell lines and affect a range of cancers in vivo.  This work stimulated intense interest in developing further ligands for the BET bromodomains and the design of ligands for non-BET bromodomains.  Here we review the recent progress in the field with particular attention paid to ligand design, the assays employed in early ligand discovery, and the use of computational approaches to inform ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1JZtWpBpqS7Vg90H21EOLACvtfcHk0lijY-mYFCw-gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D&md5=e40e85fc300109654712a00de400c8ba</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcb500996u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500996u%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DMeasures%26aufirst%3DA.%2BR.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DAlexander%26aufirst%3DR.%26aulast%3DHoss%26aufirst%3DM.%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520bromodomain-acetyl-lysine%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D22%26epage%3D39%26doi%3D10.1021%2Fcb500996u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Salvia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteller, M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitors and cancer therapy: From structures to applications</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1080/15592294.2016.1265710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1080%2F15592294.2016.1265710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=27911230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A280%3ADC%252BC2sjivVaksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=323-339&author=M.+P%C3%A9rez-Salviaauthor=M.+Esteller&title=Bromodomain+inhibitors+and+cancer+therapy%3A+From+structures+to+applications&doi=10.1080%2F15592294.2016.1265710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibitors and cancer therapy: From structures to applications</span></div><div class="casAuthors">Perez-Salvia Montserrat; Esteller Manel; Esteller Manel; Esteller Manel</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">323-339</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aberrations in the epigenetic landscape are a hallmark of cancer.  Alterations in enzymes that are "writers," "erasers," or "readers" of histone modification marks are common.  Bromodomains are "readers" that bind acetylated lysines in histone tails.  Their most important function is the regulation of gene transcription by the recruitment of different molecular partners.  Moreover, proteins containing bromodomains are also epigenetic regulators, although little is known about the specific function of these domains.  In recent years, there has been increasing interest in developing small molecules that can target specific bromodomains.  First, this has helped clarify biological functions of bromodomain-containing proteins.  Secondly, it opens a new front for combatting cancer.  In this review we will describe the structures and mechanisms associated with Bromodomain and Extra-Terminal motif (BET) inhibitors and non-BET inhibitors, their current status of development, and their promising role as anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCPfARVC-uxp6KlWarZTAzfW6udTcc2eaLtklBMGJqOLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjivVaksQ%253D%253D&md5=3ce4e19d8177ddc1705fd0d58e5fb020</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F15592294.2016.1265710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2016.1265710%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Salvia%26aufirst%3DM.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DBromodomain%2520inhibitors%2520and%2520cancer%2520therapy%253A%2520From%2520structures%2520to%2520applications%26jtitle%3DEpigenetics%26date%3D2017%26volume%3D12%26spage%3D323%26epage%3D339%26doi%3D10.1080%2F15592294.2016.1265710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+Coteauthor=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0liXupn6hER9gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauder, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Fragment-based screening of the bromodomain of ATAD2</span>. <i>J. Med. Chem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9687</span>– <span class="NLM_lpage">9692</span>, <span class="refDoi"> DOI: 10.1021/jm501035j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501035j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChtrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9687-9692&author=M.+J.+Harnerauthor=B.+A.+Chauderauthor=J.+Phanauthor=S.+W.+Fesik&title=Fragment-based+screening+of+the+bromodomain+of+ATAD2&doi=10.1021%2Fjm501035j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Screening of the Bromodomain of ATAD2</span></div><div class="casAuthors">Harner, Mary J.; Chauder, Brian A.; Phan, Jason; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9687-9692</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cellular and genetic evidence suggest that inhibition of ATAD2 (AAA domain-contg. protein 2) could be a useful strategy to treat several types of cancer.  To discover small-mol. inhibitors of the bromodomain of ATAD2, we used a fragment-based approach.  Fragment hits were identified using NMR spectroscopy, and ATAD2 was crystd. with three of the hits identified in the fragment screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3781FiAn_x7Vg90H21EOLACvtfcHk0liXupn6hER9gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChtrrE&md5=b3ba9d32e92c8d92b37132214940e9d2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm501035j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501035j%26sid%3Dliteratum%253Aachs%26aulast%3DHarner%26aufirst%3DM.%2BJ.%26aulast%3DChauder%26aufirst%3DB.%2BA.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DFragment-based%2520screening%2520of%2520the%2520bromodomain%2520of%2520ATAD2%26jtitle%3DJ.%2520Med.%2520Chem%26date%3D2014%26volume%3D57%26spage%3D9687%26epage%3D9692%26doi%3D10.1021%2Fjm501035j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based approaches towards identification of fragments for the low-druggability ATAD2 bromodomain</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1848</span>, <span class="refDoi"> DOI: 10.1039/C4MD00237G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1039%2FC4MD00237G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1843-1848&author=A.+Chaikuadauthor=A.+M.+Petrosauthor=O.+Fedorovauthor=J.+Xuauthor=S.+Knapp&title=Structure-based+approaches+towards+identification+of+fragments+for+the+low-druggability+ATAD2+bromodomain&doi=10.1039%2FC4MD00237G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC4MD00237G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00237G%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-based%2520approaches%2520towards%2520identification%2520of%2520fragments%2520for%2520the%2520low-druggability%2520ATAD2%2520bromodomain%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D1843%26epage%3D1848%26doi%3D10.1039%2FC4MD00237G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmons, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouzon, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of low-micromolar ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">5649</span>– <span class="NLM_lpage">5673</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5649-5673&author=E.+H.+Demontauthor=C.+W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=C.+Wellawayauthor=N.+Barrettauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=A.+J.+Emmonsauthor=E.+J.+Jonesauthor=B.+V.+Karamshiauthor=K.+Lockeauthor=D.+J.+Mitchellauthor=B.+H.+Mouzonauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watsonauthor=P.+Bamborough&title=Fragment-based+discovery+of+low-micromolar+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Demont, Emmanuel H.; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Wellaway, Chris; Barrett, Nathalie; Bridges, Angela M.; Craggs, Peter D.; Diallo, Hawa; Dixon, David P.; Douault, Clement; Emmons, Amanda J.; Jones, Emma J.; Karamshi, Bhumika V.; Locke, Kelly; Mitchell, Darren J.; Mouzon, Bernadette H.; Prinjha, Rab K.; Roberts, Andy D.; Sheppard, Robert J.; Watson, Robert J.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5649-5673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Overexpression of ATAD2 (ATPase family, AAA domain contg. 2) has been linked to disease severity and progression in a wide range of cancers, and is implicated in the regulation of several drivers of cancer growth.  Little is known of the dependence of these effects upon the ATAD2 bromodomain, which has been categorized as among the least tractable of its class.  The absence of any potent, selective inhibitors limits clear understanding of the therapeutic potential of the bromodomain.  Here, the authors describe the discovery of a hit from a fragment-based targeted array.  Optimization of this produced the first known micromolar inhibitors of the ATAD2 bromodomain, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5pJPrVexJbVg90H21EOLACvtfcHk0lj1W3_IlFFLUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP&md5=c133ce439365cd4e4110230827a7a0a4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00772%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DN.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DEmmons%26aufirst%3DA.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%2BV.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DMouzon%26aufirst%3DB.%2BH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520low-micromolar%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5649%26epage%3D5673%26doi%3D10.1021%2Facs.jmedchem.5b00772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of naphthyridones into potent ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&author=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-based+optimization+of+naphthyridones+into+potent+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0lj1W3_IlFFLUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-based%2520optimization%2520of%2520naphthyridones%2520into%2520potent%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A chemical probe for the ATAD2 bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11382</span>– <span class="NLM_lpage">11386</span>, <span class="refDoi"> DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&author=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+chemical+probe+for+the+ATAD2+bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0ljh7ObjG1tY8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520chemical%2520probe%2520for%2520the%2520ATAD2%2520bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Saez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjanacz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-selective ATAD2 chemical probe with novel chemical structure and unusual mode of action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&author=A.+E.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+Stockigtauthor=A.+Ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+Diaz-Saezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=J.+Hubnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorjanacz&title=Isoform-selective+ATAD2+chemical+probe+with+novel+chemical+structure+and+unusual+mode+of+action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span></div><div class="casAuthors">Fernandez-Montalvan, Amaury E.; Berger, Markus; Kuropka, Benno; Koo, Seong Joo; Badock, Volker; Weiske, Joerg; Puetter, Vera; Holton, Simon J.; Stoeckigt, Detlef; ter Laak, Antonius; Centrella, Paolo A.; Clark, Matthew A.; Dumelin, Christoph E.; Sigel, Eric A.; Soutter, Holly H.; Troast, Dawn M.; Zhang, Ying; Cuozzo, John W.; Keefe, Anthony D.; Roche, Didier; Rodeschini, Vincent; Chaikuad, Apirat; Diaz-Saez, Laura; Bennett, James M.; Fedorov, Oleg; Huber, Kilian V. M.; Huebner, Jan; Weinmann, Hilmar; Hartung, Ingo V.; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2730-2736</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 (ANCCA) is an epigenetic regulator and transcriptional cofactor, whose overexpression has been linked to the progress of various cancer types.  Here, we report a DNA-encoded library screen leading to the discovery of BAY-850, a potent and isoform selective inhibitor that specifically induces ATAD2 bromodomain dimerization and prevents interactions with acetylated histones in vitro, as well as with chromatin in cells.  These features qualify BAY-850 as a chem. probe to explore ATAD2 biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCB3nyG9YvubVg90H21EOLACvtfcHk0ljh7ObjG1tY8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL&md5=1f09000a7d7e60e30540fbbae117acfe</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DStockigt%26aufirst%3DD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DDiaz-Saez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DHubner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorjanacz%26aufirst%3DM.%26atitle%3DIsoform-selective%2520ATAD2%2520chemical%2520probe%2520with%2520novel%2520chemical%2520structure%2520and%2520unusual%2520mode%2520of%2520action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heptinstall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1850</span>, <span class="refDoi"> DOI: 10.1039/C8OB00099A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1039%2FC8OB00099A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29469144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVygsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=1843-1850&author=D.+C.+Millerauthor=M.+P.+Martinauthor=S.+Adhikariauthor=A.+Brennanauthor=J.+A.+Endicottauthor=B.+T.+Goldingauthor=I.+R.+Hardcastleauthor=A.+Heptinstallauthor=S.+Hobsonauthor=C.+Jennings&title=Identification+of+a+novel+ligand+for+the+ATAD2+bromodomain+with+selectivity+over+BRD4+through+a+fragment+growing+approach&doi=10.1039%2FC8OB00099A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach</span></div><div class="casAuthors">Miller, Duncan C.; Martin, Mathew P.; Adhikari, Santosh; Brennan, Alfie; Endicott, Jane A.; Golding, Bernard T.; Hardcastle, Ian R.; Heptinstall, Amy; Hobson, Stephen; Jennings, Claire; Molyneux, Lauren; Ng, Yvonne; Wedge, Stephen R.; Noble, Martin E. M.; Cano, Celine</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1843-1850</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">ATAD2 is an ATPase that is overexpressed in a variety of cancers and assocd. with a poor patient prognosis.  This protein has been suggested to function as a cofactor for a range of transcription factors, including proto-oncogene MYC and the androgen receptor.  ATAD2 comprises an ATPase domain, implicated in chromatin remodeling, and a bromodomain which allows it to interact with acetylated histone tails.  Dissection of the functional roles of these 2 domains would benefit from the availability of selective, cell-permeable pharmacol. probes.  An in silico evaluation of the 3D structures of various bromodomains suggested that developing small mol. ligands for the bromodomain of ATAD2 is likely to be challenging, although recent reports have shown that ATAD2 bromodomain ligands can be identified.  Here, we report a structure-guided fragment-based approach to identify lead compds. for ATAD2 bromodomain inhibitor development.  Our findings indicated that the ATAD2 bromodomain could accommodate fragment hits (mol. wt., <200) that yield productive structure-activity relations, and structure-guided design enabled the introduction of selectivity over BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKPvd0l4_jgLVg90H21EOLACvtfcHk0lh_DcHVU5iSNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVygsLo%253D&md5=606eacb758cb091e4c35bf3da13d8298</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC8OB00099A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8OB00099A%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DD.%2BC.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DA.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DHeptinstall%26aufirst%3DA.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DJennings%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520a%2520novel%2520ligand%2520for%2520the%2520ATAD2%2520bromodomain%2520with%2520selectivity%2520over%2520BRD4%2520through%2520a%2520fragment%2520growing%2520approach%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26spage%3D1843%26epage%3D1850%26doi%3D10.1039%2FC8OB00099A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Aiming to miss a moving target: Bromo and Extra Terminal Domain (BET) selectivity in constrained ATAD2 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8321</span>– <span class="NLM_lpage">8336</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00862</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00862" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslagtrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8321-8336&author=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=R.+J.+Watsonauthor=D.+J.+Mitchellauthor=H.+Barnettauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Sheppardauthor=T.+Wernerauthor=E.+H.+Demont&title=Aiming+to+miss+a+moving+target%3A+Bromo+and+Extra+Terminal+Domain+%28BET%29+selectivity+in+constrained+ATAD2+inhibitors&doi=10.1021%2Facs.jmedchem.8b00862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Aiming to Miss a Moving Target: Bromo and Extra Terminal Domain (BET) Selectivity in Constrained ATAD2 Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Watson, Robert J.; Mitchell, Darren J.; Barnett, Heather; Prinjha, Rab K.; Rau, Christina; Sheppard, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8321-8336</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of its class.  In our recent disclosure of the first chem. probe against this bromodomain, GSK8814 (6), we described the use of a conformationally constrained methoxy piperidine to gain selectivity over the BET bromodomains.  Here we describe an orthogonal conformational restriction strategy of the piperidine ring to give potent and selective tropane inhibitors and show structural insights into why this was more challenging than expected.  Greater understanding of why different rational approaches succeeded or failed should help in the future design of selectivity in the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9UnSVNIzmhbVg90H21EOLACvtfcHk0lh_DcHVU5iSNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslagtrzK&md5=0536cc9e89031962d38a5ece0fd57749</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00862%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DAiming%2520to%2520miss%2520a%2520moving%2520target%253A%2520Bromo%2520and%2520Extra%2520Terminal%2520Domain%2520%2528BET%2529%2520selectivity%2520in%2520constrained%2520ATAD2%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8321%26epage%3D8336%26doi%3D10.1021%2Facs.jmedchem.8b00862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poncet-Montange, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardenhagen, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee Iv, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geck
Do, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardozo, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span> <span> </span><span class="NLM_article-title">Observed bromodomain flexibility reveals histone peptide- and small molecule ligand-compatible forms of ATAD2A</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1042/BJ20140933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1042%2FBJ20140933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=25486442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFCqu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2015&pages=337-346&author=G.+Poncet-Montangeauthor=Y.+Zhanauthor=J.+P.+Bardenhagenauthor=A.+Petrocchiauthor=E.+Leoauthor=X.+Shiauthor=G.+R.+Lee+Ivauthor=P.+G.+Leonardauthor=M.+K.+Geck%0ADoauthor=M.+G.+Cardozoauthor=J.+N.+Andersenauthor=W.+S.+Palmerauthor=P.+Jonesauthor=J.+E.+Ladbury&title=Observed+bromodomain+flexibility+reveals+histone+peptide-+and+small+molecule+ligand-compatible+forms+of+ATAD2A&doi=10.1042%2FBJ20140933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Observed bromodomain flexibility reveals histone peptide- and small molecule ligand-compatible forms of ATAD2</span></div><div class="casAuthors">Poncet-Montange, Guillaume; Zhan, Yanai; Bardenhagen, Jennifer P.; Petrocchi, Alessia; Leo, Elisabetta; Shi, Xi; Lee, Gilbert R.; Leonard, Paul G.; Geck Do, Mary K.; Cardozo, Mario G.; Andersen, Jannik N.; Palmer, Wylie S.; Jones, Philip; Ladbury, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-346</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Preventing histone recognition by bromodomains emerges as an attractive therapeutic approach in cancer.  Overexpression of ATAD2 (ATPase family AAA domain-contg. 2 isoform A) in cancer cells is assocd. with poor prognosis making the bromodomain of ATAD2 a promising epigenetic therapeutic target.  In the development of an in vitro assay and identification of small mol. ligands, we conducted structure-guided studies which revealed a conformationally flexible ATAD2 bromodomain.  Structural studies on apo-, peptide-and small mol.-ATAD2 complexes (by co-crystn.) revealed that the bromodomain adopts a 'closed', histone-compatible conformation and a more 'open' ligand-compatible conformation of the binding site resp.  An unexpected conformational change of the conserved asparagine residue plays an important role in driving the peptide-binding conformation remodelling.  We also identified dimethylisoxazole-contg. ligands as ATAD2 binders which aided in the validation of the in vitro screen and in the anal. of these conformational studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofrj4a58a07rVg90H21EOLACvtfcHk0lh_DcHVU5iSNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFCqu7Y%253D&md5=7c900b1b2299e72dbb6a11cd8c1d7f1d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1042%2FBJ20140933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20140933%26sid%3Dliteratum%253Aachs%26aulast%3DPoncet-Montange%26aufirst%3DG.%26aulast%3DZhan%26aufirst%3DY.%26aulast%3DBardenhagen%26aufirst%3DJ.%2BP.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DLee%2BIv%26aufirst%3DG.%2BR.%26aulast%3DLeonard%26aufirst%3DP.%2BG.%26aulast%3DGeck%2BDo%26aufirst%3DM.%2BK.%26aulast%3DCardozo%26aufirst%3DM.%2BG.%26aulast%3DAndersen%26aufirst%3DJ.%2BN.%26aulast%3DPalmer%26aufirst%3DW.%2BS.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DObserved%2520bromodomain%2520flexibility%2520reveals%2520histone%2520peptide-%2520and%2520small%2520molecule%2520ligand-compatible%2520forms%2520of%2520ATAD2A%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D466%26spage%3D337%26epage%3D346%26doi%3D10.1042%2FBJ20140933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0ljxIxMaUqXLOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chakravorty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popa-Burke, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remlinger, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillmore, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span> <span> </span><span class="NLM_article-title">Nuisance compounds, PAINS filters, and dark chemical matter in the GSK HTS collection</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">532</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1177/2472555218768497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1177%2F2472555218768497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29699447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1agtrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=532-545&author=S.+J.+Chakravortyauthor=J.+Chanauthor=M.+N.+Greenwoodauthor=I.+Popa-Burkeauthor=K.+S.+Remlingerauthor=S.+D.+Pickettauthor=D.+V.+S.+Greenauthor=M.+C.+Fillmoreauthor=T.+W.+Deanauthor=J.+I.+Luengoauthor=R.+Macarron&title=Nuisance+compounds%2C+PAINS+filters%2C+and+dark+chemical+matter+in+the+GSK+HTS+collection&doi=10.1177%2F2472555218768497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Nuisance Compounds, PAINS Filters, and Dark Chemical Matter in the GSK HTS Collection</span></div><div class="casAuthors">Chakravorty, Subhas J.; Chan, James; Greenwood, Marie Nicole; Popa-Burke, Ioana; Remlinger, Katja S.; Pickett, Stephen D.; Green, Darren V. S.; Fillmore, Martin C.; Dean, Tony W.; Luengo, Juan I.; Macarron, Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">532-545</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) hits include compds. with undesirable properties.  Many filters have been described to identify such hits.  Notably, pan-assay interference compds. (PAINS) has been adopted by the community as the std. term to refer to such filters, and very useful guidelines have been adopted by the American Chem. Society (ACS) and subsequently triggered a healthy scientific debate about the pitfalls of draconian use of filters.  Using an inhibitory frequency index, we have analyzed in detail the promiscuity profile of the whole GlaxoSmithKline (GSK) HTS collection comprising more than 2 million unique compds. that have been tested in hundreds of screening assays.  We provide a comprehensive anal. of many previously published filters and newly described classes of nuisance structures that may serve as a useful source of empirical information to guide the design or growth of HTS collections and hit triaging strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ5bHsBxkwOrVg90H21EOLACvtfcHk0ljxIxMaUqXLOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1agtrnE&md5=9a620fb177aa001b0a36e90948f8e326</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1177%2F2472555218768497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555218768497%26sid%3Dliteratum%253Aachs%26aulast%3DChakravorty%26aufirst%3DS.%2BJ.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DGreenwood%26aufirst%3DM.%2BN.%26aulast%3DPopa-Burke%26aufirst%3DI.%26aulast%3DRemlinger%26aufirst%3DK.%2BS.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DFillmore%26aufirst%3DM.%2BC.%26aulast%3DDean%26aufirst%3DT.%2BW.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DMacarron%26aufirst%3DR.%26atitle%3DNuisance%2520compounds%252C%2520PAINS%2520filters%252C%2520and%2520dark%2520chemical%2520matter%2520in%2520the%2520GSK%2520HTS%2520collection%26jtitle%3DSLAS%2520Discovery%26date%3D2018%26volume%3D23%26spage%3D532%26epage%3D545%26doi%3D10.1177%2F2472555218768497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V.</span></span> <span> </span><span class="NLM_article-title">Design of a compound screening collection for use in high throughput screening</span>. <i>Comb. Chem. High Throughput Screening</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.2174/138620704772884832</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.2174%2F138620704772884832" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=14965262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=63-70&author=G.+Harperauthor=S.+D.+Pickettauthor=D.+V.+Green&title=Design+of+a+compound+screening+collection+for+use+in+high+throughput+screening&doi=10.2174%2F138620704772884832"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a compound screening collection for use in high throughput screening</span></div><div class="casAuthors">Harper, G.; Pickett, S. D.; Green, D. V. S.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-71</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">In this paper we introduce a quant. model that relates chem. structural similarity to biol. activity, and in particular to the activity of lead series of compds. in high-throughput assays.  From this model we derive the optimal screening collection make up for a given fixed size of screening collection, and identify the conditions under which a diverse collection of compds. or a collection focusing on particular regions of chem. space are appropriate strategies.  We derive from the model a diversity function that may be used to assess compds. for acquisition or libraries for combinatorial synthesis by their ability to complement an existing screening collection.  The diversity function is linked directly through the model to the goal of more frequent discovery of lead series from high-throughput screening.  We show how the model may also be used to derive relationships between collection size and probabilities of lead discovery in high-throughput screening, and to guide the judicious application of structural filters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpThXGuh69P87Vg90H21EOLACvtfcHk0ljxIxMaUqXLOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D&md5=b29b45b082fe5fc9d6400e1c8fd83cc0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F138620704772884832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620704772884832%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DGreen%26aufirst%3DD.%2BV.%26atitle%3DDesign%2520of%2520a%2520compound%2520screening%2520collection%2520for%2520use%2520in%2520high%2520throughput%2520screening%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2004%26volume%3D7%26spage%3D63%26epage%3D70%26doi%3D10.2174%2F138620704772884832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e1600760</span> <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&author=S.+Picaudauthor=K.+Leonardsauthor=J.+P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.+Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.+C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+targeting+of+bromodomains+by+bromosporine+identifies+BET+proteins+as+master+regulators+of+primary+transcription+response+in+leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0lj3B-7S-g2FEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520targeting%2520of%2520bromodomains%2520by%2520bromosporine%2520identifies%2520BET%2520proteins%2520as%2520master%2520regulators%2520of%2520primary%2520transcription%2520response%2520in%2520leukemia%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lj3B-7S-g2FEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gay, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckenroth, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, K. C.</span></span> <span> </span><span class="NLM_article-title">Disulfide bridge formation influences ligand recognition by the ATAD2 bromodomain</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1002/prot.25636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fprot.25636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=30520161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Sqs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2019&pages=157-167&author=J.+C.+Gayauthor=B.+E.+Eckenrothauthor=C.+M.+Evansauthor=C.+Langiniauthor=S.+Carlsonauthor=J.+T.+Lloydauthor=A.+Caflischauthor=K.+C.+Glass&title=Disulfide+bridge+formation+influences+ligand+recognition+by+the+ATAD2+bromodomain&doi=10.1002%2Fprot.25636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Disulfide bridge formation influences ligand recognition by the ATAD2 bromodomain</span></div><div class="casAuthors">Gay, Jamie C.; Eckenroth, Brian E.; Evans, Chiara M.; Langini, Cassiano; Carlson, Samuel; Lloyd, Jonathan T.; Caflisch, Amedeo; Glass, Karen C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>;
        ISSN:<span class="NLM_cas:issn">1097-0134</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The ATPase family, AAA domain-contg. protein 2 (ATAD2) has a C-terminal bromodomain, which functions as a chromatin reader domain recognizing acetylated lysine on the histone tails within the nucleosome.  ATAD2 is overexpressed in many cancers and its expression is correlated with poor patient outcomes, making it an attractive therapeutic target and potential biomarker.  We solved the crystal structure of the ATAD2 bromodomain and found that it contains a disulfide bridge near the base of the acetyllysine binding pocket (Cys1057-Cys1079).  Site-directed mutagenesis revealed that removal of a free C-terminal cysteine (C1101) residue greatly improved the soly. of the ATAD2 bromodomain in vitro.  Isothermal titrn. calorimetry expts. in combination with the Ellman's assay demonstrated that formation of an intramol. disulfide bridge neg. impacts the ligand binding affinities and alters the thermodn. parameters of the ATAD2 bromodomain interaction with a histone H4K5ac peptide as well as a small mol. bromodomain ligand.  Mol. dynamics simulations indicate that the formation of the disulfide bridge in the ATAD2 bromodomain does not alter the structure of the folded state or flexibility of the acetyllysine binding pocket.  However, consideration of this unique structural feature should be taken into account when examg. ligand-binding affinity, or in the design of new bromodomain inhibitor compds. that interact with this acetyllysine reader module.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrumAF6V6frlLVg90H21EOLACvtfcHk0lj3B-7S-g2FEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Sqs7vM&md5=4d818f3ea1da7d011523f8b2ef69821c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fprot.25636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.25636%26sid%3Dliteratum%253Aachs%26aulast%3DGay%26aufirst%3DJ.%2BC.%26aulast%3DEckenroth%26aufirst%3DB.%2BE.%26aulast%3DEvans%26aufirst%3DC.%2BM.%26aulast%3DLangini%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DS.%26aulast%3DLloyd%26aufirst%3DJ.%2BT.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DGlass%26aufirst%3DK.%2BC.%26atitle%3DDisulfide%2520bridge%2520formation%2520influences%2520ligand%2520recognition%2520by%2520the%2520ATAD2%2520bromodomain%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2019%26volume%3D87%26spage%3D157%26epage%3D167%26doi%3D10.1002%2Fprot.25636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coma, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herranz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richmond, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valmaseda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span> <span> </span><span class="NLM_article-title">Process validation and screen reproducibility in high-throughput screening</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1177/1087057108326664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1177%2F1087057108326664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=19171922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslGrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=66-76&author=I.+Comaauthor=L.+Clarkauthor=E.+Diezauthor=G.+Harperauthor=J.+Herranzauthor=G.+Hofmannauthor=M.+Lennonauthor=N.+Richmondauthor=M.+Valmasedaauthor=R.+Macarron&title=Process+validation+and+screen+reproducibility+in+high-throughput+screening&doi=10.1177%2F1087057108326664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Process validation and screen reproducibility in high-throughput screening</span></div><div class="casAuthors">Coma, Isabel; Clark, Liz; Diez, Emilio; Harper, Gavin; Herranz, Jesus; Hofmann, Glenn; Lennon, Mark; Richmond, Nicola; Valmaseda, Manuel; Macarron, Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The use of large-scale compd. screening has become a key component of drug discovery projects in both the pharmaceutical and the biotechnol. industries.  More recently, these activities have also been embraced by the academic community as a major tool for chem. genomic activities.  High-throughput screening (HTS) activities constitute a major step in the initial drug discovery efforts and involve the use of large quantities of biol. reagents, hundreds of thousands to millions of compds., and the utilization of expensive equipment.  All these factors make it very important to evaluate in advance of the HTS campaign any potential issues related to reproducibility of the experimentation and the quality of the results obtained at the end of these very costly activities.  In this article, the authors describe how GlaxoSmithKline (GSK) has addressed the need of a true validation of the HTS process before embarking in full HTS campaigns.  They present 2 different aspects of the so-called validation process: (1) optimization of the HTS workflow and its validation as a quality process and (2) the statistical evaluation of the HTS, focusing on the reproducibility of results and the ability to distinguish active from nonactive compds. in a vast collection of samples.  The authors describe a variety of reproducibility indexes that are either innovative or have been adapted from generic medical diagnostic screening strategies.  In addn., they exemplify how these validation tools have been implemented in a no. of case studies at GSK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfPaLvIL_p3rVg90H21EOLACvtfcHk0lhzKOiGixzvBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslGrtL8%253D&md5=ee5786ad5960a6089f8d7cb6346b7480</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1177%2F1087057108326664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108326664%26sid%3Dliteratum%253Aachs%26aulast%3DComa%26aufirst%3DI.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DDiez%26aufirst%3DE.%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DHerranz%26aufirst%3DJ.%26aulast%3DHofmann%26aufirst%3DG.%26aulast%3DLennon%26aufirst%3DM.%26aulast%3DRichmond%26aufirst%3DN.%26aulast%3DValmaseda%26aufirst%3DM.%26aulast%3DMacarron%26aufirst%3DR.%26atitle%3DProcess%2520validation%2520and%2520screen%2520reproducibility%2520in%2520high-throughput%2520screening%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D66%26epage%3D76%26doi%3D10.1177%2F1087057108326664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of novel monosubstituted sulfonylurea derivatives as antituberculosis agents</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2012.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.ejmech.2012.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=22326144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFGru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2012&pages=18-26&author=L.+Panauthor=Y.+Jiangauthor=Z.+Liuauthor=X.+H.+Liuauthor=Z.+Liuauthor=G.+Wangauthor=Z.+M.+Liauthor=D.+Wang&title=Synthesis+and+evaluation+of+novel+monosubstituted+sulfonylurea+derivatives+as+antituberculosis+agents&doi=10.1016%2Fj.ejmech.2012.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of novel monosubstituted sulfonylurea derivatives as antituberculosis agents</span></div><div class="casAuthors">Pan, Li; Jiang, Ying; Liu, Zhen; Liu, Xing-Hai; Liu, Zhuo; Wang, Gang; Li, Zheng-Ming; Wang, Di</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18-26</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of novel monosubstituted sulfonylurea derivs. were synthesized and characterized by 1H NMR, 13C NMR, and HRMS.  These compds. were evaluated against Mycobacterium tuberculosis H37Rv in vitro.  The results showed compds. I (R1 = vinyl, R2 = Me; R1 = Ph, R2 = Me; R1 = vinyl, R2 = OMe) exhibited moderate antituberculosis activities with MIC values in the range of 20-100 mg/L.  Compds. I (R1 = CH2Cl, R2 = Me; R1 = CH2Cl, R2 = OMe) displayed good antituberculosis activities (MIC 10 mg/L), which were comparable with that of the sulfometuron Me.  Both of the two compds. showed little cytotoxicities, with an IC50 against THP-1 cells greater than 100 mg/L.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq-Y1Xea7sCrVg90H21EOLACvtfcHk0lhzKOiGixzvBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFGru7s%253D&md5=91e3a3c06228e473bc5e48d198174308</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2012.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2012.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DX.%2BH.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DZ.%2BM.%26aulast%3DWang%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520monosubstituted%2520sulfonylurea%2520derivatives%2520as%2520antituberculosis%2520agents%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2012%26volume%3D50%26spage%3D18%26epage%3D26%26doi%3D10.1016%2Fj.ejmech.2012.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-W.</span></span> <span> </span><span class="NLM_article-title">Methyl fluorosulphonyldifluoroacetate; a new trifluoromethylating agent</span>. <i>J. Chem. Soc., Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_fpage">705</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.1039/c39890000705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1039%2Fc39890000705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADyaK3cXitVGgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1989&pages=705-706&author=Q.-Y.+Chenauthor=S.-W.+Wu&title=Methyl+fluorosulphonyldifluoroacetate%3B+a+new+trifluoromethylating+agent&doi=10.1039%2Fc39890000705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Methyl (fluorosulfonyl)difluoroacetate; a new trifluoromethylating agent</span></div><div class="casAuthors">Chen, Qingyun; Wu, Shengwen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Chemical Communications</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">705-6</span>CODEN:
                <span class="NLM_cas:coden">JCCCAT</span>;
        ISSN:<span class="NLM_cas:issn">0022-4936</span>.
    </div><div class="casAbstract">Me (fluorosulfonyl)difluoroacetate, in the presence of CuI, is used as the source of a trifluoromethyl group, which replaces the halogen in aryl, alkenyl, and alkyl halides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphEwpeqCNED7Vg90H21EOLACvtfcHk0lhzKOiGixzvBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitVGgtA%253D%253D&md5=9e937c3bd4587b28e708210d3b648fd3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2Fc39890000705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc39890000705%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DQ.-Y.%26aulast%3DWu%26aufirst%3DS.-W.%26atitle%3DMethyl%2520fluorosulphonyldifluoroacetate%253B%2520a%2520new%2520trifluoromethylating%2520agent%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Chem.%2520Commun.%26date%3D1989%26spage%3D705%26epage%3D706%26doi%3D10.1039%2Fc39890000705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. C. C.</span>; <span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Barnett, H.</span>; <span class="NLM_string-name">Chung, C.</span>; <span class="NLM_string-name">Gordon, L.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Phillipou, A.</span>; <span class="NLM_string-name">Prinjha, R.</span>; <span class="NLM_string-name">Sheppard, R. J.</span>; <span class="NLM_string-name">Tomkinson, N. C.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Demont, E. H.</span></span> in preparation.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Lucas%2C+S.+C.+C.%3B+Atkinson%2C+S.+J.%3B+Bamborough%2C+P.%3B+Barnett%2C+H.%3B+Chung%2C+C.%3B+Gordon%2C+L.%3B+Mitchell%2C+D.+J.%3B+Phillipou%2C+A.%3B+Prinjha%2C+R.%3B+Sheppard%2C+R.+J.%3B+Tomkinson%2C+N.+C.%3B+Watson%2C+R.+J.%3B+Demont%2C+E.+H.+in+preparation."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DS.%2BC.%2BC." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-lysine binding site of bromodomain-containing protein 4 (BRD4) interacts with diverse kinase inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+Emberauthor=J.+Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Acetyl-lysine+binding+site+of+bromodomain-containing+protein+4+%28BRD4%29+interacts+with+diverse+kinase+inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0lgCDwh9-WJWvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DAcetyl-lysine%2520binding%2520site%2520of%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520interacts%2520with%2520diverse%2520kinase%2520inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">Diving into the water: inducible binding conformations for BRD4, TAF1(2), BRD9, and CECR2 bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+Coteauthor=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.+R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=Diving+into+the+water%3A+inducible+binding+conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0lgCDwh9-WJWvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.%2BR.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520into%2520the%2520water%253A%2520inducible%2520binding%2520conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talluri, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for the N-terminal Bromodomain-and-Extra-Terminal-family selectivity of a dual kinase-bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9316</span>– <span class="NLM_lpage">9334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01248</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01248" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9316-9334&author=A.+Divakaranauthor=S.+K.+Talluriauthor=A.+M.+Ayoubauthor=N.+K.+Mishraauthor=H.+Cuiauthor=J.+C.+Widenauthor=N.+Berndtauthor=J.+Y.+Zhuauthor=A.+S.+Carlsonauthor=J.+J.+Topczewskiauthor=E.+K.+Schonbrunnauthor=D.+A.+Harkiauthor=W.+C.+K.+Pomerantz&title=Molecular+basis+for+the+N-terminal+Bromodomain-and-Extra-Terminal-family+selectivity+of+a+dual+kinase-bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.8b01248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase-Bromodomain Inhibitor</span></div><div class="casAuthors">Divakaran, Anand; Talluri, Siva K.; Ayoub, Alex M.; Mishra, Neeraj K.; Cui, Huarui; Widen, John C.; Berndt, Norbert; Zhu, Jin-Yi; Carlson, Angela S.; Topczewski, Joseph J.; Schonbrunn, Ernst K.; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9316-9334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis.  When dysregulated, "reader" proteins become drivers of disease.  In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging.  We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase-bromodomain inhibitors.  Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1).  Cellular efficacy was demonstrated via redn. of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 prodn. through potential synergistic inhibition of BRD4(1) and p38α.  These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZhnL9cPdDi7Vg90H21EOLACvtfcHk0lhF1s6JAO2RnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM&md5=5176ef754a898b3e03fc56e2662ab399</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DTalluri%26aufirst%3DS.%2BK.%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%2BK.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DMolecular%2520basis%2520for%2520the%2520N-terminal%2520Bromodomain-and-Extra-Terminal-family%2520selectivity%2520of%2520a%2520dual%2520kinase-bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9316%26epage%3D9334%26doi%3D10.1021%2Facs.jmedchem.8b01248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auffinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hays, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhof, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, P. S.</span></span> <span> </span><span class="NLM_article-title">Halogen bonds in biological molecules</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">16789</span>– <span class="NLM_lpage">16794</span>, <span class="refDoi"> DOI: 10.1073/pnas.0407607101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1073%2Fpnas.0407607101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=15557000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFansr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=16789-16794&author=P.+Auffingerauthor=F.+A.+Haysauthor=E.+Westhofauthor=P.+S.+Ho&title=Halogen+bonds+in+biological+molecules&doi=10.1073%2Fpnas.0407607101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen bonds in biological molecules</span></div><div class="casAuthors">Auffinger, Pascal; Hays, Franklin A.; Westhof, Eric; Ho, P. Shing</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16789-16794</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Short oxygen-halogen interactions have been known in org. chem. since the 1950s and recently have been exploited in the design of supramol. assemblies.  The present survey of protein and nucleic acid structures reveals similar halogen bonds as potentially stabilizing inter- and intramol. interactions that can affect ligand binding and mol. folding.  A halogen bond in biomols. can be defined as a short C-X···O-Y interaction (C-X is a carbon-bonded chlorine, bromine, or iodine, and O-Y is a carbonyl, hydroxyl, charged carboxylate, or phosphate group), where the X···O distance is less than or equal to the sums of the resp. van der Waals radii (3.27 Å for Cl···O, 3.37 Å for Br···O, and 3.50 Å for I···O) and can conform to the geometry seen in small mols., with the C-X···O angle ≈165° (consistent with a strong directional polarization of the halogen) and the X···O-Y angle ≈120°.  Alternative geometries can be imposed by the more complex environment found in biomols., depending on which of the two types of donor systems are involved in the interaction: (i) the lone pair electrons of oxygen (and, to a lesser extent, nitrogen and sulfur) atoms or (ii) the delocalized π-electrons of peptide bonds or carboxylate or amide groups.  Thus, the specific geometry and diversity of the interacting partners of halogen bonds offer new and versatile tools for the design of ligands as drugs and materials in nanotechnol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPMdZU91iT77Vg90H21EOLACvtfcHk0lhF1s6JAO2RnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFansr3P&md5=dfcce25d4e336c66294f6ab8ce412154</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407607101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407607101%26sid%3Dliteratum%253Aachs%26aulast%3DAuffinger%26aufirst%3DP.%26aulast%3DHays%26aufirst%3DF.%2BA.%26aulast%3DWesthof%26aufirst%3DE.%26aulast%3DHo%26aufirst%3DP.%2BS.%26atitle%3DHalogen%2520bonds%2520in%2520biological%2520molecules%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D16789%26epage%3D16794%26doi%3D10.1073%2Fpnas.0407607101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Politzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span> <span> </span><span class="NLM_article-title">σ-hole bonding: a physical interpretation</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1007/128_2014_568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1007%2F128_2014_568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=25467532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVylsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2015&pages=19-42&author=P.+Politzerauthor=J.+S.+Murrayauthor=T.+Clark&title=%CF%83-hole+bonding%3A+a+physical+interpretation&doi=10.1007%2F128_2014_568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">σ-Hole Bonding: A Physical Interpretation</span></div><div class="casAuthors">Politzer, Peter; Murray, Jane S.; Clark, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">Halogen Bonding I: Impact on Materials Chemistry and Life Sciences</span>),
    <span class="NLM_cas:pages">19-42</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">1436-5049</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">The anisotropic electronic densities of covalently-bonded Group IV-VII atoms frequently give rise to regions of pos. electrostatic potential on the extensions of covalent bonds to these atoms.  Through such pos. "σ-holes," the atoms can interact attractively and highly directionally with neg. sites such as the lone pairs of Lewis bases, anions, π electrons, etc.  In the case of Group VII this is called "halogen bonding." Hydrogen bonding can be viewed as a less directional subset of σ-hole interactions.  Since pos. σ-holes often exist in conjunction with regions of neg. potential, the atoms can also interact favorably with pos. sites.  In accordance with the Hellmann-Feynman theorem, all of these interactions are purely Coulombic in nature (which encompasses polarization and dispersion).  The strength of σ-hole bonding increases with the magnitudes of the potentials of the pos. σ-hole and the neg. site; their polarizabilities must sometimes also be taken explicitly into account.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0l1L0LKANkLVg90H21EOLACvtfcHk0lhnNlmdHvjPsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVylsL4%253D&md5=49273da6edbcfa745168c833194ee890</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F128_2014_568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2014_568%26sid%3Dliteratum%253Aachs%26aulast%3DPolitzer%26aufirst%3DP.%26aulast%3DMurray%26aufirst%3DJ.%2BS.%26aulast%3DClark%26aufirst%3DT.%26atitle%3D%25CF%2583-hole%2520bonding%253A%2520a%2520physical%2520interpretation%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2015%26volume%3D358%26spage%3D19%26epage%3D42%26doi%3D10.1007%2F128_2014_568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goudreau, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hucke, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibeault, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llinas-Brunet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, P. W.</span></span> <span> </span><span class="NLM_article-title">Combined X-ray, NMR, and kinetic analyses reveal uncommon binding characteristics of the hepatitis C virus NS3-NS4A protease inhibitor BI 201335</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">11434</span>– <span class="NLM_lpage">11443</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.211417</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1074%2Fjbc.M110.211417" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=21270126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjslKgsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=11434-11443&author=C.+T.+Lemkeauthor=N.+Goudreauauthor=S.+Zhaoauthor=O.+Huckeauthor=D.+Thibeaultauthor=M.+Llinas-Brunetauthor=P.+W.+White&title=Combined+X-ray%2C+NMR%2C+and+kinetic+analyses+reveal+uncommon+binding+characteristics+of+the+hepatitis+C+virus+NS3-NS4A+protease+inhibitor+BI+201335&doi=10.1074%2Fjbc.M110.211417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Combined X-ray, NMR, and Kinetic Analyses Reveal Uncommon Binding Characteristics of the Hepatitis C Virus NS3-NS4A Protease Inhibitor BI 201335</span></div><div class="casAuthors">Lemke, Christopher T.; Goudreau, Nathalie; Zhao, Songping; Hucke, Oliver; Thibeault, Diane; Llinas-Brunet, Montse; White, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">11434-11443</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Hepatitis C virus infection, a major cause of liver disease worldwide, is curable, but currently approved therapies have suboptimal efficacy.  Supplementing these therapies with direct-acting antiviral agents has the potential to considerably improve treatment prospects for hepatitis C virus-infected patients.  The crit. role played by the viral NS3 protease makes it an attractive target, and despite its shallow, solvent-exposed active site, several potent NS3 protease inhibitors are currently in the clinic.  BI 201335, which is progressing through Phase IIb trials, contains a unique C-terminal carboxylic acid that binds noncovalently to the active site and a bromo-quinoline substitution on its proline residue that provides significant potency.  In this work we have used stopped flow kinetics, x-ray crystallog., and NMR to characterize these distinctive features.  Key findings include: slow assocn. and dissocn. rates within a single-step binding mechanism; the crit. involvement of water mols. in acid binding; and protein side chain rearrangements, a bromine-oxygen halogen bond, and profound pKa changes within the catalytic triad assocd. with binding of the bromo-quinoline moiety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkdn8YiZZHALVg90H21EOLACvtfcHk0lhnNlmdHvjPsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjslKgsbs%253D&md5=2281c4bbfd76e34934f129fb61dcf926</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.211417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.211417%26sid%3Dliteratum%253Aachs%26aulast%3DLemke%26aufirst%3DC.%2BT.%26aulast%3DGoudreau%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DHucke%26aufirst%3DO.%26aulast%3DThibeault%26aufirst%3DD.%26aulast%3DLlinas-Brunet%26aufirst%3DM.%26aulast%3DWhite%26aufirst%3DP.%2BW.%26atitle%3DCombined%2520X-ray%252C%2520NMR%252C%2520and%2520kinetic%2520analyses%2520reveal%2520uncommon%2520binding%2520characteristics%2520of%2520the%2520hepatitis%2520C%2520virus%2520NS3-NS4A%2520protease%2520inhibitor%2520BI%2520201335%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D11434%26epage%3D11443%26doi%3D10.1074%2Fjbc.M110.211417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Exploring chemistry with the fragment molecular orbital method</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">7562</span>– <span class="NLM_lpage">7577</span>, <span class="refDoi"> DOI: 10.1039/c2cp23784a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1039%2Fc2cp23784a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=22410762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVykurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=7562-7577&author=D.+G.+Fedorovauthor=T.+Nagataauthor=K.+Kitaura&title=Exploring+chemistry+with+the+fragment+molecular+orbital+method&doi=10.1039%2Fc2cp23784a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring chemistry with the fragment molecular orbital method</span></div><div class="casAuthors">Fedorov, Dmitri G.; Nagata, Takeshi; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7562-7577</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The fragment MO (FMO) method makes possible nearly linear scaling calcns. of large mol. systems, such as water clusters, proteins and DNA.  In particular, FMO has been widely used in biochem. applications involving protein-ligand binding and drug design.  The method has been efficiently parallelized suitable for petascale computing.  Many commonly used wave functions and solvent models have been interfaced with FMO.  We review the historical background of FMO, and summarize its method development and applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_TiaN_HBr9rVg90H21EOLACvtfcHk0lhnNlmdHvjPsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVykurw%253D&md5=903e3457b1dc5749950ebae5c954960a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1039%2Fc2cp23784a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2cp23784a%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DNagata%26aufirst%3DT.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DExploring%2520chemistry%2520with%2520the%2520fragment%2520molecular%2520orbital%2520method%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2012%26volume%3D14%26spage%3D7562%26epage%3D7577%26doi%3D10.1039%2Fc2cp23784a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Pair interaction energy decomposition analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1002/jcc.20496</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fjcc.20496" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=17109433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXktVKgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=222-237&author=D.+G.+Fedorovauthor=K.+Kitaura&title=Pair+interaction+energy+decomposition+analysis&doi=10.1002%2Fjcc.20496"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Pair interaction energy decomposition analysis</span></div><div class="casAuthors">Fedorov, Dmitri G.; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">222-237</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The energy decompn. anal. (EDA) by Kitaura and Morokuma was redeveloped in the framework of the fragment MO method (FMO).  The proposed pair interaction energy decompn. anal. (PIEDA) can treat large mol. clusters and the systems in which fragments are connected by covalent bonds, such as proteins.  The interaction energy in PIEDA is divided into the same contributions as in EDA: the electrostatic, exchange-repulsion, and charge transfer energies, to which the correlation (dispersion) term was added.  The careful comparison to the ab initio EDA interaction energies for water clusters with 2-16 mols. revealed that PIEDA has the error of at most 1.2 kcal/mol (or about 1%).  The anal. was applied to (H2O)1024, the α helix, β turn, and β strand of polyalanine (ALA)10, as well as to the synthetic protein (PDB code 1L2Y) with 20 residues.  The comparative aspects of the polypeptide isomer stability are discussed in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ89I3W8YX1LVg90H21EOLACvtfcHk0lgsLYInlJNcGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXktVKgug%253D%253D&md5=759c711290fe1c44b7c86a28104ccef7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20496&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20496%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DPair%2520interaction%2520energy%2520decomposition%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D222%26epage%3D237%26doi%3D10.1002%2Fjcc.20496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heifetz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKinnon, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwan, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brookfield, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chudyk, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burch, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span> <span> </span><span class="NLM_article-title">Fragment Molecular Orbital method applied to lead optimization of novel interleukin-2 inducible T-cell kinase (ITK) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4352</span>– <span class="NLM_lpage">4363</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12jt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4352-4363&author=A.+Heifetzauthor=G.+Traniauthor=M.+Aldeghiauthor=C.+H.+MacKinnonauthor=P.+A.+McEwanauthor=F.+A.+Brookfieldauthor=E.+I.+Chudykauthor=M.+Bodkinauthor=Z.+Peiauthor=J.+D.+Burchauthor=D.+F.+Ortwine&title=Fragment+Molecular+Orbital+method+applied+to+lead+optimization+of+novel+interleukin-2+inducible+T-cell+kinase+%28ITK%29+inhibitors&doi=10.1021%2Facs.jmedchem.6b00045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Molecular Orbital Method Applied to Lead Optimization of Novel Interleukin-2 Inducible T-Cell Kinase (ITK) Inhibitors</span></div><div class="casAuthors">Heifetz, Alexander; Trani, Giancarlo; Aldeghi, Matteo; MacKinnon, Colin H.; McEwan, Paul A.; Brookfield, Frederick A.; Chudyk, Ewa I.; Bodkin, Mike; Pei, Zhonghua; Burch, Jason D.; Ortwine, Daniel F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4352-4363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of inducible T-cell kinase (ITK), a nonreceptor tyrosine kinase, may represent a novel treatment for allergic asthma.  In our previous reports, we described the discovery of sulfonylpyridine (SAP), benzothiazole (BZT), indazole (IND), and tetrahydroindazole (THI) series as novel ITK inhibitors and how computational tools such as dihedral scans and docking were used to support this process.  X-ray crystallog. and modeling were applied to provide essential insight into ITK-ligand interactions.  However, "visual inspection" traditionally used for the rationalization of protein-ligand affinity cannot always explain the full complexity of the mol. interactions.  The fragment MO (FMO) quantum-mech. (QM) method provides a complete list of the interactions formed between the ligand and protein that are often omitted from traditional structure-based descriptions.  FMO methodol. was successfully used as part of a rational structure-based drug design effort to improve the ITK potency of high-throughput screening hits, ultimately delivering ligands with potency in the subnanomolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTGF0Bo-GJM7Vg90H21EOLACvtfcHk0lgsLYInlJNcGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12jt70%253D&md5=b0958d9c29f7d61aadcc730ac00e8f90</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00045%26sid%3Dliteratum%253Aachs%26aulast%3DHeifetz%26aufirst%3DA.%26aulast%3DTrani%26aufirst%3DG.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DMacKinnon%26aufirst%3DC.%2BH.%26aulast%3DMcEwan%26aufirst%3DP.%2BA.%26aulast%3DBrookfield%26aufirst%3DF.%2BA.%26aulast%3DChudyk%26aufirst%3DE.%2BI.%26aulast%3DBodkin%26aufirst%3DM.%26aulast%3DPei%26aufirst%3DZ.%26aulast%3DBurch%26aufirst%3DJ.%2BD.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26atitle%3DFragment%2520Molecular%2520Orbital%2520method%2520applied%2520to%2520lead%2520optimization%2520of%2520novel%2520interleukin-2%2520inducible%2520T-cell%2520kinase%2520%2528ITK%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4352%26epage%3D4363%26doi%3D10.1021%2Facs.jmedchem.6b00045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Molecular recognition mechanism of FK506 binding protein: An all-electron fragment molecular orbital study</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">158</span>, <span class="refDoi"> DOI: 10.1002/prot.21389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fprot.21389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=17387719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1Oktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2007&pages=145-158&author=I.+Nakanishiauthor=D.+G.+Fedorovauthor=K.+Kitaura&title=Molecular+recognition+mechanism+of+FK506+binding+protein%3A+An+all-electron+fragment+molecular+orbital+study&doi=10.1002%2Fprot.21389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition mechanism of FK506 binding protein: an all-electron fragment molecular orbital study</span></div><div class="casAuthors">Nakanishi, Isao; Fedorov, Dmitri G.; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">145-158</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The fragment MO (FMO) method has enabled electronic structure calcns. and geometry optimizations of very large mols. with ab initio quality.  We applied the method to four FK506 binding protein (FKBP) complexes (denoted by their PDB codes 1fkb, 1fkf, 1fkg, and 1fki) contg. rapamycin, FK506, and two synthetic ligands.  The geometries of reduced complex models were optimized at the RHF (FMO-RHF) level using the 3-21G basis set, and then for a better est. of binding, the energetics were refined at a higher level of theory (2nd order Moller-Plesset perturbation theory FMO-MP2 with the 6-31G* basis set).  Thus, obtained binding energies were -103.9 (-82.0), -102.2 (-69.2), -70.1 (-57.7), and -71.3 (-55.3) kcal/mol for 1fkb, 1fkf, 1fkg, and 1fki, resp., where the correlation contribution is given in parentheses.  The results show that the electron correlation contribution to binding is extremely important, and it accounts for 70-80% of the binding energy.  The mol. recognition mechanism of FKBP was analyzed in detail based on the FMO-pair interactions between protein residues and the ligands.  Solvation effects on the protein-ligand binding were estd. using the Poisson-Boltzmann/surface area model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6aOw8ua9Pm7Vg90H21EOLACvtfcHk0lgsLYInlJNcGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1Oktb8%253D&md5=1f07d1d4804c137ced134d4fd1d0412f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fprot.21389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21389%26sid%3Dliteratum%253Aachs%26aulast%3DNakanishi%26aufirst%3DI.%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DMolecular%2520recognition%2520mechanism%2520of%2520FK506%2520binding%2520protein%253A%2520An%2520all-electron%2520fragment%2520molecular%2520orbital%2520study%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2007%26volume%3D68%26spage%3D145%26epage%3D158%26doi%3D10.1002%2Fprot.21389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, K.</span></span> <span> </span><span class="NLM_article-title">Application of the fragment molecular orbital method analysis to fragment-based drug discovery of BET (bromodomain and extra-terminal proteins) inhibitors</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2017.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.jmgm.2017.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=28365532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsl2ktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2017&pages=73-82&author=M.+Ozawaauthor=T.+Ozawaauthor=K.+Ueda&title=Application+of+the+fragment+molecular+orbital+method+analysis+to+fragment-based+drug+discovery+of+BET+%28bromodomain+and+extra-terminal+proteins%29+inhibitors&doi=10.1016%2Fj.jmgm.2017.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Application of the fragment molecular orbital method analysis to fragment-based drug discovery of BET (bromodomain and extra-terminal proteins) inhibitors</span></div><div class="casAuthors">Ozawa, Motoyasu; Ozawa, Tomonaga; Ueda, Kazuyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">73-82</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The mol. interactions of inhibitors of bromodomains (BRDs) were investigated.  BRDs are protein interaction modules that recognizing ε-N-acetyl-lysine (εAc-Lys) motifs found in histone tails and are promising protein-protein interaction (PPI) targets.  First, we analyzed a peptide ligand contg. εAc-Lys to evaluate native PPIs.  We then analyzed tetrahydroquinazoline-6-yl-benzensulfonamide derivs. found by fragment-based drug design (FBDD) and examd. their interactions with the protein compared with the peptide ligand in terms of the inter-fragment interaction energy.  In addn., we analyzed benzodiazepine derivs. that are high-affinity ligands for BRDs and examd. differences in the CH/π interactions of the amino acid residues.  We further surveyed changes in the charges of the amino acid residues among individual ligands, performed pair interaction energy decompn. anal. and estd. the water profile within the ligand binding site.  Thus, useful insights for drug design were provided.  Through these analyses and considerations, we show that the FMO method is a useful drug design tool to evaluate the process of FBDD and to explore PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU2gVSiemC4rVg90H21EOLACvtfcHk0lgJ-DnLvZ7JfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsl2ktrw%253D&md5=bf9ac28f01c2f5fa4a4e2084272ee6b3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2017.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2017.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DOzawa%26aufirst%3DM.%26aulast%3DOzawa%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DK.%26atitle%3DApplication%2520of%2520the%2520fragment%2520molecular%2520orbital%2520method%2520analysis%2520to%2520fragment-based%2520drug%2520discovery%2520of%2520BET%2520%2528bromodomain%2520and%2520extra-terminal%2520proteins%2529%2520inhibitors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2017%26volume%3D74%26spage%3D73%26epage%3D82%26doi%3D10.1016%2Fj.jmgm.2017.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Use of an auxiliary basis set to describe the polarization in the fragment molecular orbital method</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>597</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/j.cplett.2014.02.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.cplett.2014.02.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFahsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=597&publication_year=2014&pages=99-105&author=D.+G.+Fedorovauthor=K.+Kitaura&title=Use+of+an+auxiliary+basis+set+to+describe+the+polarization+in+the+fragment+molecular+orbital+method&doi=10.1016%2Fj.cplett.2014.02.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Use of an auxiliary basis set to describe the polarization in the fragment molecular orbital method</span></div><div class="casAuthors">Fedorov, Dmitri G.; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">597</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-105</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We developed a dual basis approach within the fragment MO formalism enabling efficient and accurate use of large basis sets.  The method was tested on water clusters and polypeptides and applied to perform geometry optimization of chignolin (PDB: 1UAO) in soln. at the level of DFT/6-31++G**, obtaining a structure in agreement with expt. (RMSD of 0.4526 Å).  The polarization in polypeptides is discussed with a comparison of the α-helix and β-strand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoahaZbElNpPrVg90H21EOLACvtfcHk0lgJ-DnLvZ7JfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFahsro%253D&md5=745a811aff3ea79042f9b58907c1074a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cplett.2014.02.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cplett.2014.02.029%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DUse%2520of%2520an%2520auxiliary%2520basis%2520set%2520to%2520describe%2520the%2520polarization%2520in%2520the%2520fragment%2520molecular%2520orbital%2520method%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D2014%26volume%3D597%26spage%3D99%26epage%3D105%26doi%3D10.1016%2Fj.cplett.2014.02.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozuch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. M.</span></span> <span> </span><span class="NLM_article-title">Halogen bonds: benchmarks and theoretical analysis</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1918</span>– <span class="NLM_lpage">1931</span>, <span class="refDoi"> DOI: 10.1021/ct301064t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct301064t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlyjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1918-1931&author=S.+Kozuchauthor=J.+M.+Martin&title=Halogen+bonds%3A+benchmarks+and+theoretical+analysis&doi=10.1021%2Fct301064t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen Bonds: Benchmarks and Theoretical Analysis</span></div><div class="casAuthors">Kozuch, Sebastian; Martin, Jan M. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1918-1931</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We carried out an extensive survey of wave function and DFT methods to test their accuracy on geometries and dissocn. energies of halogen bonds (XB).  For that purpose, we built two benchmark sets (XB18 and XB51).  Between the DFT methods, it was found that functionals with high exact exchange or long-range corrections were suitable for these dimers, esp. M06-2X, ωB97XD, and double hybrids.  Dispersion corrections tend to be detrimental, in spite of the fact that XB is considered a noncovalent interaction.  Wave function techniques require heavy correlated methods (i.e., CCSD(T)) or parametrized ones (SCS-MP2 or SCS(MI)MP2).  Heavy basis sets are needed to obtain high accuracy, such as aVQZ or aVTZ+CP, and ideally a CBS extrapolation.  Relativistic ECPs are also important, even for the bromine based dimers.  In addn., we explored some XB with new theor. tools, the NCI ("Non-Covalent Interactions") method and the NOFF ("Natural Orbital Fukui Functions").</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPO_Z9Idg5E7Vg90H21EOLACvtfcHk0litzxD5xt-r6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlyjs7g%253D&md5=3ca6c0b9e5047e75d73946a36fad3d40</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fct301064t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct301064t%26sid%3Dliteratum%253Aachs%26aulast%3DKozuch%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DJ.%2BM.%26atitle%3DHalogen%2520bonds%253A%2520benchmarks%2520and%2520theoretical%2520analysis%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D1918%26epage%3D1931%26doi%3D10.1021%2Fct301064t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boys, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, F.</span></span> <span> </span><span class="NLM_article-title">The calculation of small molecular interactions by the differences of separate total energies. Some procedures with reduced errors</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1080/00268977000101561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1080%2F00268977000101561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1alt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1970&pages=553-566&author=S.+F.+Boysauthor=F.+Bernardi&title=The+calculation+of+small+molecular+interactions+by+the+differences+of+separate+total+energies.+Some+procedures+with+reduced+errors&doi=10.1080%2F00268977000101561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The calculation of small molecular interactions by the differences of separate total energies. Some procedures with reduced errors</span></div><div class="casAuthors">Boys, S. F.; Bernardi, F.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">553-566</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A new direct difference method for the computation of mol. interactions has been based on a bivariational transcorrelated treatment, together with special methods for the balancing of other errors.  It appears that these new features can give a strong redn. in the error of the interaction energy, and they seem to be particularly suitable for computations in the important region near the min. energy.  It has been generally accepted that this problem is dominated by unresolved difficulties and the relation of the new methods of these apparent difficulties is analyzed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp23TXDqDyeYrVg90H21EOLACvtfcHk0litzxD5xt-r6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1alt7fM&md5=c4f847835a638b4fc449fae688c557e9</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1080%2F00268977000101561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268977000101561%26sid%3Dliteratum%253Aachs%26aulast%3DBoys%26aufirst%3DS.%2BF.%26aulast%3DBernardi%26aufirst%3DF.%26atitle%3DThe%2520calculation%2520of%2520small%2520molecular%2520interactions%2520by%2520the%2520differences%2520of%2520separate%2520total%2520energies.%2520Some%2520procedures%2520with%2520reduced%2520errors%26jtitle%3DMol.%2520Phys.%26date%3D1970%26volume%3D19%26spage%3D553%26epage%3D566%26doi%3D10.1080%2F00268977000101561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-scale analysis of water stability in bromodomain binding pockets with grand canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">19</span> <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29863194" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-scale+analysis+of+water+stability+in+bromodomain+binding+pockets+with+grand+canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-scale%2520analysis%2520of%2520water%2520stability%2520in%2520bromodomain%2520binding%2520pockets%2520with%2520grand%2520canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span> <span> </span><span class="NLM_article-title">Protein dynamics and structural waters in bromodomains</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0186570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0186570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1371%2Fjournal.pone.0186570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29077715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=X.+Zhangauthor=K.+Chenauthor=Y.+D.+Wuauthor=O.+Wiest&title=Protein+dynamics+and+structural+waters+in+bromodomains&doi=10.1371%2Fjournal.pone.0186570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics and structural waters in bromodomains</span></div><div class="casAuthors">Zhang, Xiaoxiao; Chen, Kai; Wu, Yun-Dong; Wiest, Olaf</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e0186570/1-e0186570/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers of acetylated lysines that are integral parts of histone tails.  The 61 bromodomains in humans are structurally highly conserved but specifically bind to widely varying recognition motifs, suggesting that dynamic rather than static factors are responsible for recognition selectivity.  To test this hypothesis, the dynamics of the binding sites and structural water mols. of four bromodomains (ATAD2, BAZ2B, BRD2(1) and CREBBP) representing four different subtypes is studied with 1 μs MD simulations using the RSFF2 force field.  The different dynamics of the ZA-loops and BC-loops between the four bromodomains leads to distinct patterns for the opening and closing of the binding pocket.  This in turn dets. the structural and energetic properties of the structural waters in the binding pocket, suggesting that these waters are not only important for the recognition itself, as has been proposed previously, but also contribute to the selectivity of different bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqzaoNSjS-7Vg90H21EOLACvtfcHk0ljm4k1ngfQdNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN&md5=3ed02a6025d67e5c0e8b511c3dc6756e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0186570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0186570%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DY.%2BD.%26aulast%3DWiest%26aufirst%3DO.%26atitle%3DProtein%2520dynamics%2520and%2520structural%2520waters%2520in%2520bromodomains%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0186570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljNUtBeDL81Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> In  <i>Data Reduction</i>, Proceedings of CCP4 Study Weekend, on Data Collection and Processing, <span class="NLM_year">1993</span>; pp  <span class="NLM_fpage">114</span>– <span class="NLM_lpage">122</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&pages=114-122&author=P.+R.+Evans&title=Data+Reduction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26jtitle%3DData%2520Reduction%26date%3D1993%26spage%3D114%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lheBmbaPIfHUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Collaborative
Computational Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>D50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D50&publication_year=1994&pages=760-763&author=Collaborative%0AComputational+Project%2C+Number+4&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3DD50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhl14PNr1fmjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhl14PNr1fmjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span> <i>Schrödinger
Release 2018-3: Maestro</i>; <span class="NLM_publisher-name">Schrödinger
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease+2018-3%3A+Maestro%3B+Schr%C3%B6dinger%0ALLC%3A+New+York%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSchr%25C3%25B6dinger%250ARelease%25202018-3%253A%2520Maestro%26pub%3DSchr%25C3%25B6dinger%250ALLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bochevarov, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braden, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philipp, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halls, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Jaguar: a high-performance quantum chemistry software program with strengths in life and materials sciences</span>. <i>Int. J. Quantum Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2110</span>– <span class="NLM_lpage">2142</span>, <span class="refDoi"> DOI: 10.1002/qua.24481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fqua.24481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCqs7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2013&pages=2110-2142&author=A.+D.+Bochevarovauthor=E.+Harderauthor=T.+F.+Hughesauthor=J.+R.+Greenwoodauthor=D.+A.+Bradenauthor=D.+M.+Philippauthor=D.+Rinaldoauthor=M.+D.+Hallsauthor=J.+Zhangauthor=R.+A.+Friesner&title=Jaguar%3A+a+high-performance+quantum+chemistry+software+program+with+strengths+in+life+and+materials+sciences&doi=10.1002%2Fqua.24481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Jaguar: A high-performance quantum chemistry software program with strengths in life and materials sciences</span></div><div class="casAuthors">Bochevarov, Art D.; Harder, Edward; Hughes, Thomas F.; Greenwood, Jeremy R.; Braden, Dale A.; Philipp, Dean M.; Rinaldo, David; Halls, Mathew D.; Zhang, Jing; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Quantum Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2110-2142</span>CODEN:
                <span class="NLM_cas:coden">IJQCB2</span>;
        ISSN:<span class="NLM_cas:issn">0020-7608</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Jaguar is an ab initio quantum chem. program that specializes in fast electronic structure predictions for mol. systems of medium and large size.  Jaguar focuses on computational methods with reasonable computational scaling with the size of the system, such as d. functional theory (DFT) and local second-order Moller-Plesset perturbation theory.  The favorable scaling of the methods and the high efficiency of the program make it possible to conduct routine computations involving several thousand MOs.  This performance is achieved through a utilization of the pseudospectral approxn. and several levels of parallelization.  The speed advantages are beneficial for applying Jaguar in biomol. computational modeling.  Addnl., owing to its superior wave function guess for transition-metal-contg. systems, Jaguar finds applications in inorg. and bioinorg. chem.  The emphasis on larger systems and transition metal elements paves the way toward developing Jaguar for its use in materials science modeling.  The article describes the historical and new features of Jaguar, such as improved parallelization of many modules, innovations in ab initio pKa prediction, and new semiempirical corrections for nondynamic correlation errors in DFT.  Jaguar applications in drug discovery, materials science, force field parameterization, and other areas of computational research are reviewed.  Timing benchmarks and other results obtained from the most recent Jaguar code are provided.  The article concludes with a discussion of challenges and directions for future development of the program. © 2013 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt_0dnCf0Yq7Vg90H21EOLACvtfcHk0lhl14PNr1fmjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCqs7%252FP&md5=2fc119bb9e0501e55d7292f7735c24ea</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fqua.24481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fqua.24481%26sid%3Dliteratum%253Aachs%26aulast%3DBochevarov%26aufirst%3DA.%2BD.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DHughes%26aufirst%3DT.%2BF.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DBraden%26aufirst%3DD.%2BA.%26aulast%3DPhilipp%26aufirst%3DD.%2BM.%26aulast%3DRinaldo%26aufirst%3DD.%26aulast%3DHalls%26aufirst%3DM.%2BD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DJaguar%253A%2520a%2520high-performance%2520quantum%2520chemistry%2520software%2520program%2520with%2520strengths%2520in%2520life%2520and%2520materials%2520sciences%26jtitle%3DInt.%2520J.%2520Quantum%2520Chem.%26date%3D2013%26volume%3D113%26spage%3D2110%26epage%3D2142%26doi%3D10.1002%2Fqua.24481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suenaga, M.</span></span> <span> </span><span class="NLM_article-title">Facio: new computational chemistry environment for PC GAMESS</span>. <i>J. Comput. Chem. Jpn.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.2477/jccj.4.25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.2477%2Fjccj.4.25" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtV2ktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=25-32&author=M.+Suenaga&title=Facio%3A+new+computational+chemistry+environment+for+PC+GAMESS&doi=10.2477%2Fjccj.4.25"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Facio: New computational chemistry environment for PC GAMESS</span></div><div class="casAuthors">Suenaga, Masahiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer Chemistry, Japan</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-32</span>CODEN:
                <span class="NLM_cas:coden">JCCJAG</span>;
        ISSN:<span class="NLM_cas:issn">1347-1767</span>.
    
            (<span class="NLM_cas:orgname">Nippon Konputa Kagakkai</span>)
        </div><div class="casAbstract">A new computational chem. environment (Facio) for PC GAMESS has been developed.  In the environment, mol. modeling, prepn. of GAMESS input file, execution of GAMESS calcn. and visualization of the results are facilitated with GUI (graphic user interface).  Mol. models are displayed as fine 3D graphics based on OpenGL technol.  Mol. modeling features are compared to those of com. available softwares.  Isovalue surface of MOs, total electron d. and electrostatic potentials are viewed as mesh surface.  IR/Raman intensities are represented as simulated spectra and normal modes of vibrations are shown as animated graphics.  All the features needed for computational chem. are integrated in this software.  Besides the above functions, Facio can make models of polypeptides or polynucleotides with collaboration with Tinker, the mol. mechanics and dynamics package and can make a mol. surface with the help of MSMS, the software for the calcn. of solvent-excluded surface.  Further, it can serve as PDB file viewer for biopolymers, such as protein and nucleic acid, since the native mol. format of Facio is PDB (protein data bank).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5gYQe6nfCfLVg90H21EOLACvtfcHk0lha7KwrrBBN4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtV2ktbc%253D&md5=964c411aac299863eb401868b5d011b3</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.2477%2Fjccj.4.25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2477%252Fjccj.4.25%26sid%3Dliteratum%253Aachs%26aulast%3DSuenaga%26aufirst%3DM.%26atitle%3DFacio%253A%2520new%2520computational%2520chemistry%2520environment%2520for%2520PC%2520GAMESS%26jtitle%3DJ.%2520Comput.%2520Chem.%2520Jpn.%26date%3D2005%26volume%3D4%26spage%3D25%26epage%3D32%26doi%3D10.2477%2Fjccj.4.25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldridge, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boatz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elbert, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">General atomic and molecular electronic structure system</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1002/jcc.540141112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fjcc.540141112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1993&pages=1347-1363&author=M.+W.+Schmidtauthor=K.+K.+Baldridgeauthor=J.+A.+Boatzauthor=S.+T.+Elbertauthor=M.+S.+Gordonauthor=J.+H.+Jensenauthor=S.+Kosekiauthor=N.+Matsunagaauthor=K.+A.+Nguyenauthor=S.+Su&title=General+atomic+and+molecular+electronic+structure+system&doi=10.1002%2Fjcc.540141112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">General atomic and molecular electronic structure system</span></div><div class="casAuthors">Schmidt, Michael W.; Baldridge, Kim K.; Boatz, Jerry A.; Elbert, Steven T.; Gordon, Mark S.; Jensen, Jan H.; Koseki, Shiro; Matsunaga, Nikita; Nguyen, Kiet A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1347-63</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">A description of the ab initio quantum chem. package GAMESS is presented.  Chem. systems contg. atoms through Rn can be treated with wave functions ranging from the simplest closed-shell case up to a general MCSCF case, permitting calcns. at the necessary level of sophistication.  Emphasis is given to novel features of the program.  The parallelization strategy used in the RHF, ROHF, UHF, and GVB sections of the program is described, and detailed speedup results are given.  Parallel calcns. can be run on ordinary workstations as well as dedicated parallel machines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pHblAdXU4rVg90H21EOLACvtfcHk0lha7KwrrBBN4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D&md5=4d9e37cd3ec67a92bdfb5b0c0084707c</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540141112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540141112%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DM.%2BW.%26aulast%3DBaldridge%26aufirst%3DK.%2BK.%26aulast%3DBoatz%26aufirst%3DJ.%2BA.%26aulast%3DElbert%26aufirst%3DS.%2BT.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DKoseki%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DN.%26aulast%3DNguyen%26aufirst%3DK.%2BA.%26aulast%3DSu%26aufirst%3DS.%26atitle%3DGeneral%2520atomic%2520and%2520molecular%2520electronic%2520structure%2520system%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1993%26volume%3D14%26spage%3D1347%26epage%3D1363%26doi%3D10.1002%2Fjcc.540141112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Aymeric Dolbois, Laurent Batiste, Lars Wiedmer, Jing Dong, Manuela Brütsch, Danzhi Huang, Nicholas M. Deerain, Dimitrios Spiliotopoulos, Iván Cheng-Sánchez, Eleen Laul, Cristina Nevado, Paweł Śledź, <span class="NLM_string-name hlFld-ContribAuthor">Amedeo Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Hitting a Moving Target: Simulation and Crystallography Study of ATAD2 Bromodomain Blockers. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (8)
                                     , 1573-1580. <a href="https://doi.org/10.1021/acsmedchemlett.0c00080" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00080</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00080%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DHitting%252Ba%252BMoving%252BTarget%25253A%252BSimulation%252Band%252BCrystallography%252BStudy%252Bof%252BATAD2%252BBromodomain%252BBlockers%26aulast%3DDolbois%26aufirst%3DAymeric%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29042020%26date%3D30062020%26date%3D06072020%26date%3D30062020%26volume%3D11%26issue%3D8%26spage%3D1573%26epage%3D1580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael A. Clegg, Paul Bamborough, Chun-wa Chung, Peter D. Craggs, Laurie Gordon, Paola Grandi, Melanie Leveridge, Matthew Lindon, Gemma M. Liwicki, Anne-Marie Michon, Judit Molnar, Inmaculada Rioja, Peter E. Soden, Natalie H. Theodoulou, Thilo Werner, Nicholas C. O. Tomkinson, Rab K. Prinjha, <span class="NLM_string-name hlFld-ContribAuthor">Philip G. Humphreys</span>. </span><span class="cited-content_cbyCitation_article-title">Application of Atypical Acetyl-lysine Methyl Mimetics in the Development of Selective Inhibitors of the Bromodomain-Containing Protein 7 (BRD7)/Bromodomain-Containing Protein 9 (BRD9) Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (11)
                                     , 5816-5840. <a href="https://doi.org/10.1021/acs.jmedchem.0c00075" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00075%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DApplication%252Bof%252BAtypical%252BAcetyl-lysine%252BMethyl%252BMimetics%252Bin%252Bthe%252BDevelopment%252Bof%252BSelective%252BInhibitors%252Bof%252Bthe%252BBromodomain-Containing%252BProtein%252B7%252B%252528BRD7%252529%25252FBromodomain-Containing%252BProtein%252B9%252B%252528BRD9%252529%252BBromodomains%26aulast%3DClegg%26aufirst%3DMichael%2BA.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15012020%26date%3D29052020%26date%3D15052020%26volume%3D63%26issue%3D11%26spage%3D5816%26epage%3D5840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Simon C. C. Lucas, Stephen J. Atkinson, Paul Bamborough, Heather Barnett, Chun-wa Chung, Laurie Gordon, Darren J. Mitchell, Alexander Phillipou, Rab K. Prinjha, Robert J. Sheppard, Nicholas C. O. Tomkinson, Robert J. Watson, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of Potent ATAD2 and CECR2 Bromodomain Inhibitors with an Atypical Binding Mode. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5212-5241. <a href="https://doi.org/10.1021/acs.jmedchem.0c00021" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00021%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252BPotent%252BATAD2%252Band%252BCECR2%252BBromodomain%252BInhibitors%252Bwith%252Ban%252BAtypical%252BBinding%252BMode%26aulast%3DLucas%26aufirst%3DSimon%2BC.%2BC.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06012020%26date%3D06052020%26date%3D22042020%26volume%3D63%26issue%3D10%26spage%3D5212%26epage%3D5241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Michael S. Bodnarchuk, Martin J. Packer, <span class="NLM_string-name hlFld-ContribAuthor">Alexe Haywood</span>. </span><span class="cited-content_cbyCitation_article-title">Utilizing Grand Canonical Monte Carlo Methods in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     , 77-82. <a href="https://doi.org/10.1021/acsmedchemlett.9b00499" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00499</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00499%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DUtilizing%252BGrand%252BCanonical%252BMonte%252BCarlo%252BMethods%252Bin%252BDrug%252BDiscovery%26aulast%3DBodnarchuk%26aufirst%3DMichael%2BS.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D31102019%26date%3D11122019%26date%3D20122019%26date%3D11122019%26volume%3D11%26issue%3D1%26spage%3D77%26epage%3D82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lian-Jing  Cao</span>, <span class="hlFld-ContribAuthor ">Yi-Jun  Zhang</span>, <span class="hlFld-ContribAuthor ">Si-Qi  Dong</span>, <span class="hlFld-ContribAuthor ">Xi-Zhao  Li</span>, <span class="hlFld-ContribAuthor ">Xia-Ting  Tong</span>, <span class="hlFld-ContribAuthor ">Dong  Chen</span>, <span class="hlFld-ContribAuthor ">Zi-Yi  Wu</span>, <span class="hlFld-ContribAuthor ">Xiao-Hui  Zheng</span>, <span class="hlFld-ContribAuthor ">Wen-Qiong  Xue</span>, <span class="hlFld-ContribAuthor ">Wei-Hua  Jia</span>, <span class="hlFld-ContribAuthor ">Jiang-Bo  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">ATAD2 interacts with C/EBPβ to promote esophageal squamous cell carcinoma metastasis via TGF-β1/Smad3 signaling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Experimental & Clinical Cancer Research</span><span> <strong>2021,</strong> <em>40 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13046-021-01905-x" title="DOI URL">https://doi.org/10.1186/s13046-021-01905-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13046-021-01905-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13046-021-01905-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Experimental%2520%2526%2520Clinical%2520Cancer%2520Research%26atitle%3DATAD2%252Binteracts%252Bwith%252BC%25252FEBP%2525CE%2525B2%252Bto%252Bpromote%252Besophageal%252Bsquamous%252Bcell%252Bcarcinoma%252Bmetastasis%252Bvia%252BTGF-%2525CE%2525B21%25252FSmad3%252Bsignaling%26aulast%3DCao%26aufirst%3DLian-Jing%26date%3D2021%26date%3D2021%26volume%3D40%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aditi  Nayak</span>, <span class="hlFld-ContribAuthor ">Madhuri  Dutta</span>, <span class="hlFld-ContribAuthor ">Anasuya  Roychowdhury</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging oncogene ATAD2: Signaling cascades and therapeutic initiatives. </span><span class="cited-content_cbyCitation_journal-name">Life Sciences</span><span> <strong>2021,</strong> <em>276 </em>, 119322. <a href="https://doi.org/10.1016/j.lfs.2021.119322" title="DOI URL">https://doi.org/10.1016/j.lfs.2021.119322</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lfs.2021.119322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lfs.2021.119322%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Sciences%26atitle%3DEmerging%252Boncogene%252BATAD2%25253A%252BSignaling%252Bcascades%252Band%252Btherapeutic%252Binitiatives%26aulast%3DNayak%26aufirst%3DAditi%26date%3D2021%26volume%3D276%26spage%3D119322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gang  Zhang</span>, <span class="hlFld-ContribAuthor ">Shan  Li</span>, <span class="hlFld-ContribAuthor ">Kai-Wen  Cheng</span>, <span class="hlFld-ContribAuthor ">Tsui-Fen  Chou</span>. </span><span class="cited-content_cbyCitation_article-title">AAA ATPases as therapeutic targets: Structure, functions, and small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>219 </em>, 113446. <a href="https://doi.org/10.1016/j.ejmech.2021.113446" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113446</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113446%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAAA%252BATPases%252Bas%252Btherapeutic%252Btargets%25253A%252BStructure%25252C%252Bfunctions%25252C%252Band%252Bsmall-molecule%252Binhibitors%26aulast%3DZhang%26aufirst%3DGang%26date%3D2021%26volume%3D219%26spage%3D113446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dahong  Yao</span>, <span class="hlFld-ContribAuthor ">Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinhui  Wang</span>, <span class="hlFld-ContribAuthor ">Dabo  Pan</span>, <span class="hlFld-ContribAuthor ">Zhendan  He</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel ATAD2 bromodomain inhibitors that trigger apoptosis and autophagy in breast cells by structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2020,</strong> <em>35 </em>
                                    (1)
                                     , 713-725. <a href="https://doi.org/10.1080/14756366.2020.1740924" title="DOI URL">https://doi.org/10.1080/14756366.2020.1740924</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2020.1740924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2020.1740924%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252BATAD2%252Bbromodomain%252Binhibitors%252Bthat%252Btrigger%252Bapoptosis%252Band%252Bautophagy%252Bin%252Bbreast%252Bcells%252Bby%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DYao%26aufirst%3DDahong%26date%3D2020%26date%3D2020%26volume%3D35%26issue%3D1%26spage%3D713%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Summary of key messages, and schematic of disqualification (left) vs qualification (right) triage approaches to hit confirmation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Some literature ATAD2 inhibitors and affinities reported using various techniques.<a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26">(18−26)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Typical HTS process at GSK. Following the primary screen at a single compound concentration and digital correction for plate patterns, typically ∼20 000 compounds are selected, replated, and the assay repeated in duplicate. If a second assay is available, either an orthogonal or an interference screen may be run in parallel. Following confirmation, a few thousand reproducible hits are normally progressed for full-curve IC<sub>50</sub> determination and then into hit qualification. Computational approaches (e.g., clustering, property prediction) may be applied at various stages, although over-reliance on this risks losing progressible hits. If the number of hits greatly exceeds compound-handling capacity (for example, in situations with a high, reproducible hit rate acting through nonprogressible mechanisms), then performing in silico filtering without losing the few progressible hits may be particularly challenging. (B) Refined ATAD2 HTS triage process as described below.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) TR-FRET dose–response curve for <b>1</b> in the 6H-Flag-tev-ATAD2(950–1148) peptide displacement assay. Compound <b>1</b> did not give fitted curves up to 10 μM in the optimized flag-6H-tev-ATAD2(981–1121) assay (data not shown). (B) <sup>15</sup>N–<sup>1</sup>H HSQC NMR spectrum (black in the presence of 100 μM <b>1</b> and red without) showing lack of evidence for compound binding.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. % inhibition responses at 10 μM from the diverse robustness set for ATAD2 construct 6 against (A) construct 2 and (B) construct 10. (C) Reproducibility of the 996 targeted bromodomain compounds screened in duplicate against ATAD2 construct 6.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. SPR binding responses for all hits for mutant ATAD2 (<i>X</i> axis) vs wild-type (<i>Y</i> axis). Hits were analyzed at 11 μM and signals normalized to correct for differences due to molecular weight. Hits are colored according to their <sup>15</sup>N–<sup>1</sup>H HSQC NMR binding class (red = binding, blue = nonbinding). All hits showing preferential binding to wild-type over mutant ATAD2 are the same as those that show binding by NMR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Biophysical binding data for compound <b>2</b>. (A) SPR sensorgram and concentration−response curves for mutant (red) and wild-type (black) ATAD2 bromodomains. (B) <sup>15</sup>N–<sup>1</sup>H HSQC NMR spectrum (black in the presence of 100 μM <b>2</b> and red without).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0011.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Phenysulfonamide Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) piperidine, NEt<sub>3</sub> or <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to room temperature, 37–97%; (b) Fe(0), NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O, 70 °C, 67–74%; (c) <b>26</b>, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU), DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 63%; (d) C<sub>2</sub>H<sub>5</sub>COCl, NEt<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to room temperature, 39–76%; (e) CH<sub>3</sub>CF<sub>2</sub>COOSO<sub>2</sub>F, CuI, dimethylformamide (DMF), 80 °C, 10–15%; (f) KOH, Pd<sub>2</sub>(dba)<sub>3</sub>, <i>t</i>BuXPhos, dioxane/H<sub>2</sub>O, 100 °C, 21%; (g) <span class="smallcaps smallerCapital">l</span>-proline, CuI, K<sub>2</sub>CO<sub>3</sub>, 25% w/w aqueous ammonia, 110 °C, 22%; (h) CuCN, <i>N</i>-methyl-2-pyrrolidone (NMP), 160 °C, 62%.</p></p></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) pIC<sub>50</sub> of <b>23</b> in other bromodomain TR-FRET assays. (B) Replicate dose–response curves of <b>23</b> in the ATAD2 bromosphere assay (pIC<sub>50</sub> 5.4 ± 0.24, <i>n</i> = 4).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Crystallographic binding modes of ATAD2 bound to <b>9</b>, <b>15</b>, and <b>36</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S56">6S56</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S57">6S57</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55">6S55</a>). (A) <b>9</b> (blue) superimposed on the structure of capped acetyl-lysine (yellow, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A5N">5A5N</a>). The four water molecules W1–W4 from the KAc complex displaced by <b>9</b> are depicted with a red surface. (B–D) <b>9</b> (blue), <b>15</b> (orange), and <b>36</b> (green) ATAD2 crystal structures. (E) ATAD2/<b>36</b> complex superimposed on BRD4 (blue protein, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P5O">3P5O</a>) showing close contacts. (F) ATAD2/<b>37</b> complex (cyan, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TZ2">4TZ2</a>) superimposed on ATAD2/<b>36</b>. *Data for <b>36</b> is from a ligand displacement rather than peptide displacement TR-FRET assay. For a comparison of the two, see ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/medium/jm9b00673_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0010.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Inhibitors were truncated to the fragment shown and optimized with restraints on the red substructure. (B) Total fragment molecular orbital (FMO) binding energy (kcal/mol) between the inhibitor fragments and each residue of the surrounding site. Interactions between the core of inhibitor <b>36</b> and neighboring residues of interest: (C) Ile1056, (D) Pro1012/Lys1011, (E) Met1029/Asp1030, and (F) Phe1009.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00673/20190815/images/large/jm9b00673_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00673&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i52">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74642" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74642" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 62 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.04.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.addr.2016.04.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=27154268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=34-41&author=C.+A.+Lipinski&title=Rule+of+five+in+2015+and+beyond%3A+Target+and+ligand+structural+limitations%2C+ligand+chemistry+structure+and+drug+discovery+project+decisions&doi=10.1016%2Fj.addr.2016.04.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Rule of five in 2015 and beyond: Target and ligand structural limitations, ligand chemistry structure and drug discovery project decisions</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34-41</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The rule of five (Ro5), based on physicochem. profiles of phase II drugs, is consistent with structural limitations in protein targets and the drug target ligands.  Three of four parameters in Ro5 are fundamental to the structure of both target and drug binding sites.  The chem. structure of the drug ligand depends on the ligand chem. and design philosophy.  Two extremes of chem. structure and design philosophy exist; ligands constructed in the medicinal chem. synthesis lab. without input from natural selection and natural product (NP) metabolites biosynthesized based on evolutionary selection.  Exceptions to Ro5 are found mostly among NPs.  Chem. chameleon-like behavior of some NPs due to intra-mol. hydrogen bonding as exemplified by cyclosporine A is a strong contributor to NP Ro5 outliers.  The fragment derived, drug Navitoclax is an example of the extensive expertise, resources, time and key decisions required for the rare discovery of a non-NP Ro5 outlier.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG3g5NBQZuebVg90H21EOLACvtfcHk0lj1mRcjMPnqgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xnslantr8%253D&md5=72eeb37cd0cd2442ef5c7a199b61f1e3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.04.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.04.029%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DRule%2520of%2520five%2520in%25202015%2520and%2520beyond%253A%2520Target%2520and%2520ligand%2520structural%2520limitations%252C%2520ligand%2520chemistry%2520structure%2520and%2520drug%2520discovery%2520project%2520decisions%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2016%26volume%3D101%26spage%3D34%26epage%3D41%26doi%3D10.1016%2Fj.addr.2016.04.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiessling, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">The ecstasy and agony of assay interference compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2165</span>– <span class="NLM_lpage">2168</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00229</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00229" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtl2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2165-2168&author=C.+Aldrichauthor=C.+Bertozziauthor=G.+I.+Georgauthor=L.+Kiesslingauthor=C.+Lindsleyauthor=D.+Liottaauthor=K.+M.+Merzauthor=A.+Schepartzauthor=S.+Wang&title=The+ecstasy+and+agony+of+assay+interference+compounds&doi=10.1021%2Facs.jmedchem.7b00229"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Ecstasy and Agony of Assay Interference Compounds</span></div><div class="casAuthors">Aldrich, Courtney; Bertozzi, Carolyn; Georg, Gunda I.; Kiessling, Laura; Lindsley, Craig; Liotta, Dennis; Merz, Kenneth M.; Schepartz, Alanna; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2165-2168</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ecstasy of discovering a new hit from screening can lead to a highly productive research effort to discover new bioactive compds.  However, in too many cases this ecstasy is followed by the agony of realizing that the compds. are not active against the desired target.  Many of these false hits are Pan Assay INterference compds. (PAINS) 1 or colloidal aggregators.  Whether the screen is conducted in silico or in the lab. and whether screening libraries, natural products, or drugs are used, all discovery efforts that rely on some form of screening to identify bioactivity are susceptible to this phenomenon.  Studies that omit crit. controls against exptl. artifacts caused by PAINS may waste years of research effort as useless compds. are progressed.  The American Chem. Society (ACS) is eager to alert the scientific community to this problem and to recommend protocols that will eliminate the publication of research articles based on compds. with artificial activity.  This editorial aims to summarize relevant concepts and to set the framework by which relevant ACS journals will address this issue going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJUf5u51VEpbVg90H21EOLACvtfcHk0liZTziNhLF8cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtl2lsLY%253D&md5=64bf087659a47fe7769208217ef8c989</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00229%26sid%3Dliteratum%253Aachs%26aulast%3DAldrich%26aufirst%3DC.%26aulast%3DBertozzi%26aufirst%3DC.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DKiessling%26aufirst%3DL.%26aulast%3DLindsley%26aufirst%3DC.%26aulast%3DLiotta%26aufirst%3DD.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DSchepartz%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520ecstasy%2520and%2520agony%2520of%2520assay%2520interference%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2165%26epage%3D2168%26doi%3D10.1021%2Facs.jmedchem.7b00229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannifer, A.</span></span> <span> </span><span class="NLM_article-title">The importance of triaging in determining the quality of output from high-throughput screening</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1847</span>– <span class="NLM_lpage">1852</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.4155%2Ffmc.15.121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=26419190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1847-1852&author=P.+Jonesauthor=S.+McElroyauthor=A.+Morrisonauthor=A.+Pannifer&title=The+importance+of+triaging+in+determining+the+quality+of+output+from+high-throughput+screening&doi=10.4155%2Ffmc.15.121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The importance of triaging in determining the quality of output from high-throughput screening</span></div><div class="casAuthors">Jones, Philip; McElroy, Stuart; Morrison, Angus; Pannifer, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1847-1852</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ5lVzKTZCFrVg90H21EOLACvtfcHk0liZTziNhLF8cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsL3K&md5=0fa12f27c4deea2e56ec799470534462</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.121%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%26aulast%3DMcElroy%26aufirst%3DS.%26aulast%3DMorrison%26aufirst%3DA.%26aulast%3DPannifer%26aufirst%3DA.%26atitle%3DThe%2520importance%2520of%2520triaging%2520in%2520determining%2520the%2520quality%2520of%2520output%2520from%2520high-throughput%2520screening%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D1847%26epage%3D1852%26doi%3D10.4155%2Ffmc.15.121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span> <span> </span><span class="NLM_article-title">Seven year itch: pan-assay interference compounds (PAINS) in 2017-utility and limitations</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00903</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00903" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=36-44&author=J.+B.+Baellauthor=J.+W.+M.+Nissink&title=Seven+year+itch%3A+pan-assay+interference+compounds+%28PAINS%29+in+2017-utility+and+limitations&doi=10.1021%2Facschembio.7b00903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Seven Year Itch: Pan-Assay Interference Compounds (PAINS) in 2017-Utility and Limitations</span></div><div class="casAuthors">Baell, Jonathan B.; Nissink, J. Willem M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-44</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on all necessary considerations for the appropriate use of PAINs filters formulated to process hundreds and thousands of compds. in seconds, and identify PAINS in order to exclude them from further anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocZ7E_j7zxfbVg90H21EOLACvtfcHk0liZTziNhLF8cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFSlu7bK&md5=051115704137d6a91b8271702c619682</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00903%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26atitle%3DSeven%2520year%2520itch%253A%2520pan-assay%2520interference%2520compounds%2520%2528PAINS%2529%2520in%25202017-utility%2520and%2520limitations%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D36%26epage%3D44%26doi%3D10.1021%2Facschembio.7b00903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Folmer, R. H.</span></span> <span> </span><span class="NLM_article-title">Integrating biophysics with HTS-driven drug discovery projects</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.drudis.2016.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=26826422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2qtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=491-498&author=R.+H.+Folmer&title=Integrating+biophysics+with+HTS-driven+drug+discovery+projects&doi=10.1016%2Fj.drudis.2016.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating biophysics with HTS-driven drug discovery projects</span></div><div class="casAuthors">Folmer, Rutger H. A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">491-498</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Over the past decade biophysics has become an established discipline in HTS hit triaging, owing to its high fidelity in detecting protein-ligand interactions.  Many pharma companies are using biophys. techniques to filter HTS output for false positives, as will be discussed in this review.  Moreover, I will demonstrate how the earlier application of biophysics, already at the HTS assay development stage, is potentially even more impactful.  Two key areas here are early mode-of-action studies and ensuring that the HTS assay and subsequent cascade are fit for purpose.  Top-level results from 20 inhouse projects are shown to underpin the impact of these studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZXQLSut8FrVg90H21EOLACvtfcHk0ljDWFJZ_I_0iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2qtbs%253D&md5=88dc581aa9eb04c42453bc200a90a705</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DFolmer%26aufirst%3DR.%2BH.%26atitle%3DIntegrating%2520biophysics%2520with%2520HTS-driven%2520drug%2520discovery%2520projects%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D491%26epage%3D498%26doi%3D10.1016%2Fj.drudis.2016.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span> <span> </span><span class="NLM_article-title">Biophysics in drug discovery: impact, challenges and opportunities</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">679</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1038%2Fnrd.2016.123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=27516170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=679-698&author=J.+P.+Renaudauthor=C.+W.+Chungauthor=U.+H.+Danielsonauthor=U.+Egnerauthor=M.+Hennigauthor=R.+E.+Hubbardauthor=H.+Nar&title=Biophysics+in+drug+discovery%3A+impact%2C+challenges+and+opportunities&doi=10.1038%2Fnrd.2016.123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysics in drug discovery: impact, challenges and opportunities</span></div><div class="casAuthors">Renaud, Jean-Paul; Chung, Chun-wa; Danielson, U. Helena; Egner, Ursula; Hennig, Michael; Hubbard, Roderick E.; Nar, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">679-698</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past 25 years, biophys. technologies such as X-ray crystallog., NMR spectroscopy, surface plasmon resonance spectroscopy and isothermal titrn. calorimetry have become key components of drug discovery platforms in many pharmaceutical companies and academic labs.  There have been great improvements in the speed, sensitivity and range of possible measurements, providing high-resoln. mechanistic, kinetic, thermodn. and structural information on compd.-target interactions.  This Review provides a framework to understand this evolution by describing the key biophys. methods, the information they can provide and the ways in which they can be applied at different stages of the drug discovery process.  We also discuss the challenges for current technologies and future opportunities to use biophys. methods to solve drug discovery problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq91r76-q9prbVg90H21EOLACvtfcHk0ljDWFJZ_I_0iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7I&md5=1a73f14def6945e9806f57f1955f56d4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.123%26sid%3Dliteratum%253Aachs%26aulast%3DRenaud%26aufirst%3DJ.%2BP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DHennig%26aufirst%3DM.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DNar%26aufirst%3DH.%26atitle%3DBiophysics%2520in%2520drug%2520discovery%253A%2520impact%252C%2520challenges%2520and%2520opportunities%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D679%26epage%3D698%26doi%3D10.1038%2Fnrd.2016.123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERalpha, is required for coregulator occupancy and chromatin modification</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">18067</span>– <span class="NLM_lpage">18072</span>, <span class="refDoi"> DOI: 10.1073/pnas.0705814104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1073%2Fpnas.0705814104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=17998543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=18067-18072&author=J.+X.+Zouauthor=A.+S.+Revenkoauthor=L.+B.+Liauthor=A.+T.+Gemoauthor=H.+W.+Chen&title=ANCCA%2C+an+estrogen-regulated+AAA%2B+ATPase+coactivator+for+ERalpha%2C+is+required+for+coregulator+occupancy+and+chromatin+modification&doi=10.1073%2Fpnas.0705814104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA, an estrogen-regulated AAA+ ATPase coactivator for ERα, is required for coregulator occupancy and chromatin modification</span></div><div class="casAuthors">Zou, June X.; Revenko, Alexey S.; Li, Li B.; Gemo, Abigael T.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">18067-18072</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">AAA+ proteins play crucial roles in diverse biol. processes via their ATPase-driven remodeling of macromol. complexes.  Here the authors report the authors' identification of an evolutionarily conserved AAA+ protein, ANCCA/Pro 2000, endowed with a bromodomain that is strongly induced by estrogen in human breast cancer cells and is a direct target of protooncogene ACTR/AlB1/SRC-3.  The authors found that ANCCA assocs. directly with estrogen-bound estrogen receptor (ER) α and ACTR.  It is selectively recruited, upon estrogen stimulation, to a subset of ERα target genes including cyclin D1, c-myc, and E2F1 and is required for their estrogen-induced expression as well as breast cancer cell proliferation.  Further studies indicate that ANCCA binds and hydrolyzes ATP and is crit. for recruitment of coregulator CBP and histone hyperacetylation at the ER target chromatin.  Moreover, mutations at the ATP binding motifs rendered ANCCA defective as a coactivator in mediating estrogen induction of gene expression.  Together, the authors' findings reveal an unexpected layer of regulatory mechanism in hormone signaling mediated by ANCCA and suggest that hormone-induced assembly of transcriptional coregulator complexes at chromatin is a process facilitated by AAA+ ATPase proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7NFjjiBVoRbVg90H21EOLACvtfcHk0liq6PXjhG9-OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVejtLjO&md5=618fcc431bf4b0a7589213fea55732ca</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0705814104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0705814104%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DLi%26aufirst%3DL.%2BB.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DANCCA%252C%2520an%2520estrogen-regulated%2520AAA%252B%2520ATPase%2520coactivator%2520for%2520ERalpha%252C%2520is%2520required%2520for%2520coregulator%2520occupancy%2520and%2520chromatin%2520modification%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D18067%26epage%3D18072%26doi%3D10.1073%2Fpnas.0705814104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardiff, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borowsky, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">ANCCA/ATAD2 overexpression identifies breast cancer patients with poor prognosis, acting to drive proliferation and survival of triple-negative cells through control of B-Myb and EZH2</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">9402</span>– <span class="NLM_lpage">9412</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-1199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1158%2F0008-5472.CAN-10-1199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=20864510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWmsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=9402-9412&author=E.+V.+Kalashnikovaauthor=A.+S.+Revenkoauthor=A.+T.+Gemoauthor=N.+P.+Andrewsauthor=C.+G.+Tepperauthor=J.+X.+Zouauthor=R.+D.+Cardiffauthor=A.+D.+Borowskyauthor=H.+W.+Chen&title=ANCCA%2FATAD2+overexpression+identifies+breast+cancer+patients+with+poor+prognosis%2C+acting+to+drive+proliferation+and+survival+of+triple-negative+cells+through+control+of+B-Myb+and+EZH2&doi=10.1158%2F0008-5472.CAN-10-1199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">ANCCA/ATAD2 Overexpression Identifies Breast Cancer Patients with Poor Prognosis, Acting to Drive Proliferation and Survival of Triple-Negative Cells through Control of B-Myb and EZH2</span></div><div class="casAuthors">Kalashnikova, Ekaterina V.; Revenko, Alexey S.; Gemo, Abigael T.; Andrews, Nicolas P.; Tepper, Clifford G.; Zou, June X.; Cardiff, Robert D.; Borowsky, Alexander D.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9402-9412</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Chromatin coregulators are important factors in tumorigenesis and cancer progression.  ANCCA is an AAA+ ATPase and a bromodomain-contg. nuclear coactivator for the estrogen and androgen receptors that is crucial for assembly of chromatin-modifying complexes and proliferation of hormone-responsive cancer cells.  In this study, we show that ANCCA is overexpressed in >70% of breast tumors and that its high protein level correlates well with tumor histol. grades (P < 0.0001), highlighting ANCCA as a prognostic factor for poor overall survival and disease recurrence.  Strikingly, high-level ANCCA correlated with triple-neg. tumors that represent highly aggressive disease.  Anal. of ANCCA transcript levels in multiple expression profiles of breast cancer identified ANCCA as a common signature gene, indicating that elevated transcripts also strongly correlate with tumor metastasis and poor survival.  Biol. and mechanistic investigations revealed that ANCCA is crucial for proliferation and survival of triple-neg./basal-like cancer cells and that it controls the expression of B-Myb, histone methyltransferase EZH2, and an Rb-E2F core program for proliferation, along with a subset of key mitotic kinesins and cell survival genes (IRS2, VEGF, and Akt1).  In particular, ANCCA overexpression correlated strongly with EZH2 in tumors.  Our results suggest that ANCCA may integrate multiple oncogenic programs in breast cancer, serving in particular as a prognostic marker and a therapeutic target for triple-neg. cancers.  Cancer Res; 70(22); 9402-12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-adY-FJac6rVg90H21EOLACvtfcHk0ljyHtBnt2UwIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWmsLnP&md5=d7f74575c9c8cf4ebebdcfa5e3cc45fc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-1199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-1199%26sid%3Dliteratum%253Aachs%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DAndrews%26aufirst%3DN.%2BP.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DCardiff%26aufirst%3DR.%2BD.%26aulast%3DBorowsky%26aufirst%3DA.%2BD.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DANCCA%252FATAD2%2520overexpression%2520identifies%2520breast%2520cancer%2520patients%2520with%2520poor%2520prognosis%252C%2520acting%2520to%2520drive%2520proliferation%2520and%2520survival%2520of%2520triple-negative%2520cells%2520through%2520control%2520of%2520B-Myb%2520and%2520EZH2%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D9402%26epage%3D9412%26doi%3D10.1158%2F0008-5472.CAN-10-1199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lestrat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoffier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virolle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">5171</span>– <span class="NLM_lpage">5181</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1038%2Fonc.2010.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=20581866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=5171-5181&author=C.+Caronauthor=C.+Lestratauthor=S.+Marsalauthor=E.+Escoffierauthor=S.+Curtetauthor=V.+Virolleauthor=P.+Barbryauthor=A.+Debernardiauthor=C.+Brambillaauthor=E.+Brambillaauthor=S.+Rousseauxauthor=S.+Khochbin&title=Functional+characterization+of+ATAD2+as+a+new+cancer%2Ftestis+factor+and+a+predictor+of+poor+prognosis+in+breast+and+lung+cancers&doi=10.1038%2Fonc.2010.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of ATAD2 as a new cancer/testis factor and a predictor of poor prognosis in breast and lung cancers</span></div><div class="casAuthors">Caron, C.; Lestrat, C.; Marsal, S.; Escoffier, E.; Curtet, S.; Virolle, V.; Barbry, P.; Debernardi, A.; Brambilla, C.; Brambilla, E.; Rousseaux, S.; Khochbin, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5171-5181</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells frequently express genes normally active in male germ cells.  ATAD2 is one of them encoding a conserved factor harboring an AAA type ATPase domain and a bromodomain.  We show here that ATAD2 is highly expressed in testis as well as in many cancers of different origins and that its high expression is a strong predictor of rapid mortality in lung and breast cancers.  These observations suggest that ATAD2 acts on upstream and basic cellular processes to enhance oncogenesis in a variety of unrelated cell types.  Accordingly, our functional studies show that ATAD2 controls chromatin dynamics, genome transcriptional activities and apoptotic cell response.  We could also highlight some of the important intrinsic properties of its two regulatory domains, including a functional cross-talk between the AAA ATPase domain and the bromodomain.  Altogether, these data indicate that ATAD2 overexpression in somatic cells, by acting on basic properties of chromatin, may contribute to malignant transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotoD3fqS5d4LVg90H21EOLACvtfcHk0ljyHtBnt2UwIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyktr8%253D&md5=1eaa99ca9fd31e5a486bedd4d0f6178d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.259%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DLestrat%26aufirst%3DC.%26aulast%3DMarsal%26aufirst%3DS.%26aulast%3DEscoffier%26aufirst%3DE.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVirolle%26aufirst%3DV.%26aulast%3DBarbry%26aufirst%3DP.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DBrambilla%26aufirst%3DC.%26aulast%3DBrambilla%26aufirst%3DE.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DFunctional%2520characterization%2520of%2520ATAD2%2520as%2520a%2520new%2520cancer%252Ftestis%2520factor%2520and%2520a%2520predictor%2520of%2520poor%2520prognosis%2520in%2520breast%2520and%2520lung%2520cancers%26jtitle%3DOncogene%26date%3D2010%26volume%3D29%26spage%3D5171%26epage%3D5181%26doi%3D10.1038%2Fonc.2010.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morozumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boussouar, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamshidikia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petosa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Dieuleveult, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitte, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabatel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debernardi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosset, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoeyen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emadali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guardiola, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span> <span> </span><span class="NLM_article-title">Atad2 is a generalist facilitator of chromatin dynamics in embryonic stem cells</span>. <i>J. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1093/jmcb/mjv060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1093%2Fjmcb%2Fmjv060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=26459632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVKntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=349-362&author=Y.+Morozumiauthor=F.+Boussouarauthor=M.+Tanauthor=A.+Chaikuadauthor=M.+Jamshidikiaauthor=G.+Colakauthor=H.+Heauthor=L.+Nieauthor=C.+Petosaauthor=M.+de+Dieuleveultauthor=S.+Curtetauthor=A.+L.+Vitteauthor=C.+Rabatelauthor=A.+Debernardiauthor=F.+L.+Cossetauthor=E.+Verhoeyenauthor=A.+Emadaliauthor=N.+Schweiferauthor=D.+Gianniauthor=M.+Gutauthor=P.+Guardiolaauthor=S.+Rousseauxauthor=M.+Gerardauthor=S.+Knappauthor=Y.+Zhaoauthor=S.+Khochbin&title=Atad2+is+a+generalist+facilitator+of+chromatin+dynamics+in+embryonic+stem+cells&doi=10.1093%2Fjmcb%2Fmjv060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Atad2 is a generalist facilitator of chromatin dynamics in embryonic stem cells</span></div><div class="casAuthors">Morozumi, Yuichi; Boussouar, Faycal; Tan, Minjia; Chaikuad, Apirat; Jamshidikia, Mahya; Colak, Gozde; He, Huang; Nie, Litong; Petosa, Carlo; de Dieuleveult, Maud; Curtet, Sandrine; Vitte, Anne-Laure; Rabatel, Clothilde; Debernardi, Alexandra; Cosset, Francois-Loic; Verhoeyen, Els; Emadali, Anouk; Schweifer, Norbert; Gianni, Davide; Gut, Marta; Guardiola, Philippe; Rousseaux, Sophie; Gerard, Matthieu; Knapp, Stefan; Zhao, Yingming; Khochbin, Saadi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-362</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Although the conserved AAA ATPase and bromodomain factor, ATAD2, has been described as a transcriptional co-activator upregulated in many cancers, its function remains poorly understood.  Here, using a combination of ChIP-seq, ChIP-proteomics, and RNA-seq expts. in embryonic stem cells where Atad2 is normally highly expressed, we found that Atad2 is an abundant nucleosome-bound protein present on active genes, assocd. with chromatin remodelling, DNA replication, and DNA repair factors.  A structural anal. of its bromodomain and subsequent investigations demonstrate that histone acetylation guides ATAD2 to chromatin, resulting in an overall increase of chromatin accessibility and histone dynamics, which is required for the proper activity of the highly expressed gene fraction of the genome.  While in exponentially growing cells Atad2 appears dispensable for cell growth, in differentiating ES cells Atad2 becomes crit. in sustaining specific gene expression programs, controlling proliferation and differentiation.  Altogether, this work defines Atad2 as a facilitator of general chromatin-templated activities such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMr0U-BpiFIbVg90H21EOLACvtfcHk0lgGbbC9ftkB3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVKntbs%253D&md5=5a84cdc73ed9b7abeb5595425d07d3eb</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjv060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjv060%26sid%3Dliteratum%253Aachs%26aulast%3DMorozumi%26aufirst%3DY.%26aulast%3DBoussouar%26aufirst%3DF.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DJamshidikia%26aufirst%3DM.%26aulast%3DColak%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DNie%26aufirst%3DL.%26aulast%3DPetosa%26aufirst%3DC.%26aulast%3Dde%2BDieuleveult%26aufirst%3DM.%26aulast%3DCurtet%26aufirst%3DS.%26aulast%3DVitte%26aufirst%3DA.%2BL.%26aulast%3DRabatel%26aufirst%3DC.%26aulast%3DDebernardi%26aufirst%3DA.%26aulast%3DCosset%26aufirst%3DF.%2BL.%26aulast%3DVerhoeyen%26aufirst%3DE.%26aulast%3DEmadali%26aufirst%3DA.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DGianni%26aufirst%3DD.%26aulast%3DGut%26aufirst%3DM.%26aulast%3DGuardiola%26aufirst%3DP.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DGerard%26aufirst%3DM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DKhochbin%26aufirst%3DS.%26atitle%3DAtad2%2520is%2520a%2520generalist%2520facilitator%2520of%2520chromatin%2520dynamics%2520in%2520embryonic%2520stem%2520cells%26jtitle%3DJ.%2520Mol.%2520Cell%2520Biol.%26date%3D2016%26volume%3D8%26spage%3D349%26epage%3D362%26doi%3D10.1093%2Fjmcb%2Fmjv060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sokolov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Kinesin family deregulation coordinated by bromodomain protein ANCCA and histone methyltransferase MLL for breast cancer cell growth, survival, and tamoxifen resistance</span>. <i>Mol. Cancer Res</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-13-0459</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1158%2F1541-7786.MCR-13-0459" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=24391143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=539-549&author=J.+X.+Zouauthor=Z.+Duanauthor=J.+Wangauthor=A.+Sokolovauthor=J.+Xuauthor=C.+Z.+Chenauthor=J.+J.+Liauthor=H.+W.+Chen&title=Kinesin+family+deregulation+coordinated+by+bromodomain+protein+ANCCA+and+histone+methyltransferase+MLL+for+breast+cancer+cell+growth%2C+survival%2C+and+tamoxifen+resistance&doi=10.1158%2F1541-7786.MCR-13-0459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Kinesin Family Deregulation Coordinated by Bromodomain Protein ANCCA and Histone Methyltransferase MLL for Breast Cancer Cell Growth, Survival, and Tamoxifen Resistance</span></div><div class="casAuthors">Zou, June X.; Duan, Zhijian; Wang, Junjian; Sokolov, Alex; Xu, Jianzhen; Chen, Christopher Z.; Li, Jian Jian; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">539-549</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Kinesins are a superfamily of motor proteins and often deregulated in different cancers.  However, the mechanism of their deregulation has been poorly understood.  Through examg. kinesin gene family expression in estrogen receptor (ER)-pos. breast cancer cells, we found that estrogen stimulation of cancer cell proliferation involves a concerted regulation of specific kinesins.  Estrogen strongly induces expression of 19 kinesin genes such as Kif4A/4B, Kif5A/5B, Kif10, Kif11, Kif15, Kif18A/18B, Kif20A/20B, Kif21, Kif23, Kif24, Kif25, and KifC1, whereas suppresses the expression of seven others, including Kif1A, Kif1C, Kif7, and KifC3.  Interestingly, the bromodomain protein ANCCA/ATAD2, previously shown to be an estrogen-induced chromatin regulator, plays a crucial role in the up- and downregulation of kinesins by estrogen.  Its overexpression drives estrogen-independent upregulation of specific kinesins.  Mechanistically, ANCCA (AAA nuclear coregulator cancer assocd.) mediates E2-dependent recruitment of E2F and MLL1 histone methyltransferase at kinesin gene promoters for gene activation-assocd. H3K4me3 methylation.  Importantly, elevated levels of Kif4A, Kif15, Kif20A, and Kif23 correlate with that of ANCCA in the tumors and with poor relapse-free survival of patients with ER-pos. breast cancer.  Their knockdown strongly impeded proliferation and induced apoptosis of both tamoxifen-sensitive and resistant cancer cells.  Together, the study reveals ANCCA as a key mediator of kinesin family deregulation in breast cancer and the crucial role of multiple kinesins in growth and survival of the tumor cells.  Implications: These findings support the development of novel inhibitors of cancer-assocd. kinesins and their regulator ANCCA for effective treatment of cancers including tamoxifen-resistant breast cancers.  Mol Cancer Res; 12(4); 539-49. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaol_2gF7Tc7Vg90H21EOLACvtfcHk0lhbpU5C3CTzqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVOls7w%253D&md5=aa5cda3daca4e8e08dd64827d72a313c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-13-0459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-13-0459%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSokolov%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BZ.%26aulast%3DLi%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DKinesin%2520family%2520deregulation%2520coordinated%2520by%2520bromodomain%2520protein%2520ANCCA%2520and%2520histone%2520methyltransferase%2520MLL%2520for%2520breast%2520cancer%2520cell%2520growth%252C%2520survival%252C%2520and%2520tamoxifen%2520resistance%26jtitle%3DMol.%2520Cancer%2520Res%26date%3D2014%26volume%3D12%26spage%3D539%26epage%3D549%26doi%3D10.1158%2F1541-7786.MCR-13-0459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalashnikova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Chromatin loading of E2F-MLL complex by cancer-associated coregulator ANCCA via reading a specific histone mark</span>. <i>Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">5260</span>– <span class="NLM_lpage">5272</span>, <span class="refDoi"> DOI: 10.1128/MCB.00484-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1128%2FMCB.00484-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=20855524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFOltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=5260-5272&author=A.+S.+Revenkoauthor=E.+V.+Kalashnikovaauthor=A.+T.+Gemoauthor=J.+X.+Zouauthor=H.+W.+Chen&title=Chromatin+loading+of+E2F-MLL+complex+by+cancer-associated+coregulator+ANCCA+via+reading+a+specific+histone+mark&doi=10.1128%2FMCB.00484-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin loading of E2F-MLL complex by cancer-associated coregulator ANCCA via reading a specific histone mark</span></div><div class="casAuthors">Revenko, Alexey S.; Kalashnikova, Ekaterina V.; Gemo, Abigael T.; Zou, June X.; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5260-5272</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Histone modifications are regarded as the carrier of epigenetic memory through cell divisions.  How the marks facilitate cell cycle-dependent gene expression is poorly understood.  The evolutionarily conserved AAA ATPase ANCCA (AAA nuclear coregulator cancer-assocd. protein)/ATAD2 was identified as a direct target of oncogene AIB1/ACTR/SRC-3 and a transcriptional coregulator for estrogen and androgen receptors and is strongly implicated in tumorigenesis.  We report here that ANCCA directly interacts with E2F1 to E2F3 and that its N terminus interacts with both the N and C termini of E2F1.  ANCCA preferentially assocs. via its bromodomain with H3 acetylated at lysine 14 (H3K14ac) and is required for key cell cycle gene expression and cancer cell proliferation.  ANCCA assocs. with chromosomes at late mitosis, and its occupancy at E2F targets peaks at the G1-to-S transition.  Strikingly, ANCCA is required for recruitment of specific E2Fs to their targets and chromatin assembly of the host cell factor 1 (HCF-1)-MLL histone methyltransferase complex.  ANCCA depletion results in a marked decrease of the gene activation-linked H3K4me3 mark.  Bromodomain mutations disable ANCCA function as an E2F coactivator and its ability to promote cancer cell proliferation, while ANCCA overexpression in tumors correlates with tumor growth.  Together, these results suggest that ANCCA acts as a pioneer factor in E2F-dependent gene activation and that a novel mechanism involving ANCCA bromodomain may contribute to cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9jiEXxtPP3rVg90H21EOLACvtfcHk0lhbpU5C3CTzqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFOltw%253D%253D&md5=958e54242a3a8c16ac4bf5b51acd80b8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FMCB.00484-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00484-10%26sid%3Dliteratum%253Aachs%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DKalashnikova%26aufirst%3DE.%2BV.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DChromatin%2520loading%2520of%2520E2F-MLL%2520complex%2520by%2520cancer-associated%2520coregulator%2520ANCCA%2520via%2520reading%2520a%2520specific%2520histone%2520mark%26jtitle%3DMol.%2520Cell%2520Biol.%26date%3D2010%26volume%3D30%26spage%3D5260%26epage%3D5272%26doi%3D10.1128%2FMCB.00484-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. W.</span></span> <span> </span><span class="NLM_article-title">Androgen-induced coactivator ANCCA mediates specific androgen receptor signaling in prostate cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">3339</span>– <span class="NLM_lpage">3346</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-3440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1158%2F0008-5472.CAN-08-3440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=19318566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksV2hsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=3339-3346&author=J.+X.+Zouauthor=L.+Guoauthor=A.+S.+Revenkoauthor=C.+G.+Tepperauthor=A.+T.+Gemoauthor=H.+J.+Kungauthor=H.+W.+Chen&title=Androgen-induced+coactivator+ANCCA+mediates+specific+androgen+receptor+signaling+in+prostate+cancer&doi=10.1158%2F0008-5472.CAN-08-3440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen-induced coactivator ANCCA mediates specific androgen receptor signaling in prostate cancer</span></div><div class="casAuthors">Zou, June X.; Guo, Linlang; Revenko, Alexey S.; Tepper, Clifford G.; Gemo, Abigael T.; Kung, Hsing-Jien; Chen, Hong-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3339-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Androgen receptor (AR) plays a pivotal role in prostate cancer, primarily by regulating different gene expression programs elicited by androgen, which is important for cancer cell proliferation, survival, and differentiation.  It is believed that the transcriptional function of AR is mediated largely by distinct nuclear coregulators.  We report here the identification of ANCCA (also known as ATAD2), a new member of the AAA+ ATPase family proteins, as a novel AR coactivator.  ANCCA interacts directly with AR and enhances its transcriptional activity, and is required for androgen-stimulated expression of a specific subgroup of genes including IGF1R, IRS-2, SGK1, and survivin.  Upon androgen stimulation, ANCCA together with AR is recruited to the specific AR target genes.  Suppression of ANCCA expression strongly inhibited the proliferation of androgen-responsive or androgen-independent, AR-pos. prostate cancer cells and caused a significant increase of cellular apoptosis.  Strikingly, the ANCCA gene itself, located at chromosome 8q24, is highly induced by androgen in androgen-dependent prostate cancer cells and xenograft tumors.  Although ANCCA is hardly detected in normal human prostate tissue, high levels of ANCCA are found in hormone-independent prostate cancer cell lines, xenograft tumor, and a subset of prostate cancers with high Gleason scores.  Together, these findings suggest that ANCCA plays an important role in prostate cancer by mediating specific AR functions in cancer cell survival and proliferation.  The possession of ATPase and bromodomain by ANCCA makes it an attractive target for the development of therapeutics for the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtieC3cr_jX7Vg90H21EOLACvtfcHk0lhbpU5C3CTzqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksV2hsLY%253D&md5=daa42208c9c7019809bb491e5af1f152</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3440%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%2BX.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DTepper%26aufirst%3DC.%2BG.%26aulast%3DGemo%26aufirst%3DA.%2BT.%26aulast%3DKung%26aufirst%3DH.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BW.%26atitle%3DAndrogen-induced%2520coactivator%2520ANCCA%2520mediates%2520specific%2520androgen%2520receptor%2520signaling%2520in%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D3339%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-08-3440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, H. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ATAD2 in cancer: a pharmacologically challenging but tractable target</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1406921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1080%2F14728222.2018.1406921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29148850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=85-96&author=M.+Hussainauthor=Y.+Zhouauthor=Y.+Songauthor=H.+M.+A.+Hameedauthor=H.+Jiangauthor=Y.+Tuauthor=J.+Zhang&title=ATAD2+in+cancer%3A+a+pharmacologically+challenging+but+tractable+target&doi=10.1080%2F14728222.2018.1406921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">ATAD2 in cancer: a pharmacologically challenging but tractable target</span></div><div class="casAuthors">Hussain, Muzammal; Zhou, Yang; Song, Yu; Hameed, H. M. Adnan; Jiang, Hao; Tu, Yaoquan; Zhang, Jiancun</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-96</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  ATAD2 protein is an emerging oncogene that has strongly been linked to the etiol. of multiple advanced human cancers.  Therapeutically, despite the fact that genetic suppression/knockdown studies have validated it as a compelling drug target for future therapeutic development, recent druggability assessment data suggest that direct targeting of ATAD2's bromodomain (BRD) may be a very challenging task.  ATAD2's BRD has been predicted as a 'difficult to drug' or 'least druggable' target due to the concern that its binding pocket, and the areas around it, seem to be unfeasible for ligand binding.: In this review, after shedding light on the multifaceted roles of ATAD2 in normal physiol. as well as in cancer-etiol., we discuss tech. challenges rendered by ATAD2's BRD active site and the recent drug discovery efforts to find small mol. inhibitors against it.: The identification of a novel low-nanomolar semi-permeable chem. probe against ATAD2's BRD by recent drug discovery campaign has demonstrated it to be a pharmacol. tractable target.  Nevertheless, the development of high quality bioavailable inhibitors against ATAD2 is still a pending task.  Moreover, ATAD2 may also potentially be utilized as a promising target for future development of RNAi-based therapy to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0bdigfdvMjbVg90H21EOLACvtfcHk0liahut62IZnDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFahtrrN&md5=5d0ff7c936bb9eff125efeeaa4d275d1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1406921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1406921%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DHameed%26aufirst%3DH.%2BM.%2BA.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DTu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DATAD2%2520in%2520cancer%253A%2520a%2520pharmacologically%2520challenging%2520but%2520tractable%2520target%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D85%26epage%3D96%26doi%3D10.1080%2F14728222.2018.1406921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Measures, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of bromodomain-acetyl-lysine interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1021/cb500996u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500996u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=22-39&author=M.+Brandauthor=A.+R.+Measuresauthor=B.+G.+Wilsonauthor=W.+A.+Cortopassiauthor=R.+Alexanderauthor=M.+Hossauthor=D.+S.+Hewingsauthor=T.+P.+Rooneyauthor=R.+S.+Patonauthor=S.+J.+Conway&title=Small+molecule+inhibitors+of+bromodomain-acetyl-lysine+interactions&doi=10.1021%2Fcb500996u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Bromodomain-Acetyl-lysine Interactions</span></div><div class="casAuthors">Brand, Michael; Measures, Angelina R.; Wilson, Brian G.; Cortopassi, Wilian A.; Alexander, Rikki; Hoss, Matthias; Hewings, David S.; Rooney, Timothy P. C.; Paton, Robert S.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-39</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Bromodomains are protein modules that bind to acetylated lysine residues.  Their interaction with histone proteins suggests that they function as "readers" of histone lysine acetylation, a component of the proposed "histone code".  Bromodomain-contg. proteins are often found as components of larger protein complexes with roles in fundamental cellular process including transcription.  The publication of two potent ligands for the BET bromodomains in 2010 demonstrated that small mols. can inhibit the bromodomain-acetyl-lysine protein-protein interaction.  These mols. display strong phenotypic effects in a no. of cell lines and affect a range of cancers in vivo.  This work stimulated intense interest in developing further ligands for the BET bromodomains and the design of ligands for non-BET bromodomains.  Here we review the recent progress in the field with particular attention paid to ligand design, the assays employed in early ligand discovery, and the use of computational approaches to inform ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1JZtWpBpqS7Vg90H21EOLACvtfcHk0li5yT89FKjYEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGktQ%253D%253D&md5=e40e85fc300109654712a00de400c8ba</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcb500996u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500996u%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DMeasures%26aufirst%3DA.%2BR.%26aulast%3DWilson%26aufirst%3DB.%2BG.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DAlexander%26aufirst%3DR.%26aulast%3DHoss%26aufirst%3DM.%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520bromodomain-acetyl-lysine%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D22%26epage%3D39%26doi%3D10.1021%2Fcb500996u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Salvia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteller, M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitors and cancer therapy: From structures to applications</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1080/15592294.2016.1265710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1080%2F15592294.2016.1265710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=27911230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A280%3ADC%252BC2sjivVaksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=323-339&author=M.+P%C3%A9rez-Salviaauthor=M.+Esteller&title=Bromodomain+inhibitors+and+cancer+therapy%3A+From+structures+to+applications&doi=10.1080%2F15592294.2016.1265710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibitors and cancer therapy: From structures to applications</span></div><div class="casAuthors">Perez-Salvia Montserrat; Esteller Manel; Esteller Manel; Esteller Manel</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">323-339</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aberrations in the epigenetic landscape are a hallmark of cancer.  Alterations in enzymes that are "writers," "erasers," or "readers" of histone modification marks are common.  Bromodomains are "readers" that bind acetylated lysines in histone tails.  Their most important function is the regulation of gene transcription by the recruitment of different molecular partners.  Moreover, proteins containing bromodomains are also epigenetic regulators, although little is known about the specific function of these domains.  In recent years, there has been increasing interest in developing small molecules that can target specific bromodomains.  First, this has helped clarify biological functions of bromodomain-containing proteins.  Secondly, it opens a new front for combatting cancer.  In this review we will describe the structures and mechanisms associated with Bromodomain and Extra-Terminal motif (BET) inhibitors and non-BET inhibitors, their current status of development, and their promising role as anti-cancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCPfARVC-uxp6KlWarZTAzfW6udTcc2eZe4BHytkwE-rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjivVaksQ%253D%253D&md5=3ce4e19d8177ddc1705fd0d58e5fb020</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1080%2F15592294.2016.1265710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2016.1265710%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez-Salvia%26aufirst%3DM.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DBromodomain%2520inhibitors%2520and%2520cancer%2520therapy%253A%2520From%2520structures%2520to%2520applications%26jtitle%3DEpigenetics%26date%3D2017%26volume%3D12%26spage%3D323%26epage%3D339%26doi%3D10.1080%2F15592294.2016.1265710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+Coteauthor=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lhiGGCfGai1wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauder, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Fragment-based screening of the bromodomain of ATAD2</span>. <i>J. Med. Chem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9687</span>– <span class="NLM_lpage">9692</span>, <span class="refDoi"> DOI: 10.1021/jm501035j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501035j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChtrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9687-9692&author=M.+J.+Harnerauthor=B.+A.+Chauderauthor=J.+Phanauthor=S.+W.+Fesik&title=Fragment-based+screening+of+the+bromodomain+of+ATAD2&doi=10.1021%2Fjm501035j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Screening of the Bromodomain of ATAD2</span></div><div class="casAuthors">Harner, Mary J.; Chauder, Brian A.; Phan, Jason; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9687-9692</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cellular and genetic evidence suggest that inhibition of ATAD2 (AAA domain-contg. protein 2) could be a useful strategy to treat several types of cancer.  To discover small-mol. inhibitors of the bromodomain of ATAD2, we used a fragment-based approach.  Fragment hits were identified using NMR spectroscopy, and ATAD2 was crystd. with three of the hits identified in the fragment screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3781FiAn_x7Vg90H21EOLACvtfcHk0li_be64DfyixQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChtrrE&md5=b3ba9d32e92c8d92b37132214940e9d2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm501035j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501035j%26sid%3Dliteratum%253Aachs%26aulast%3DHarner%26aufirst%3DM.%2BJ.%26aulast%3DChauder%26aufirst%3DB.%2BA.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DFragment-based%2520screening%2520of%2520the%2520bromodomain%2520of%2520ATAD2%26jtitle%3DJ.%2520Med.%2520Chem%26date%3D2014%26volume%3D57%26spage%3D9687%26epage%3D9692%26doi%3D10.1021%2Fjm501035j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based approaches towards identification of fragments for the low-druggability ATAD2 bromodomain</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1848</span>, <span class="refDoi"> DOI: 10.1039/C4MD00237G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1039%2FC4MD00237G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1843-1848&author=A.+Chaikuadauthor=A.+M.+Petrosauthor=O.+Fedorovauthor=J.+Xuauthor=S.+Knapp&title=Structure-based+approaches+towards+identification+of+fragments+for+the+low-druggability+ATAD2+bromodomain&doi=10.1039%2FC4MD00237G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC4MD00237G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00237G%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-based%2520approaches%2520towards%2520identification%2520of%2520fragments%2520for%2520the%2520low-druggability%2520ATAD2%2520bromodomain%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D1843%26epage%3D1848%26doi%3D10.1039%2FC4MD00237G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmons, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouzon, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of low-micromolar ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">5649</span>– <span class="NLM_lpage">5673</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5649-5673&author=E.+H.+Demontauthor=C.+W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=C.+Wellawayauthor=N.+Barrettauthor=A.+M.+Bridgesauthor=P.+D.+Craggsauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=A.+J.+Emmonsauthor=E.+J.+Jonesauthor=B.+V.+Karamshiauthor=K.+Lockeauthor=D.+J.+Mitchellauthor=B.+H.+Mouzonauthor=R.+K.+Prinjhaauthor=A.+D.+Robertsauthor=R.+J.+Sheppardauthor=R.+J.+Watsonauthor=P.+Bamborough&title=Fragment-based+discovery+of+low-micromolar+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Low-Micromolar ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Demont, Emmanuel H.; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Wellaway, Chris; Barrett, Nathalie; Bridges, Angela M.; Craggs, Peter D.; Diallo, Hawa; Dixon, David P.; Douault, Clement; Emmons, Amanda J.; Jones, Emma J.; Karamshi, Bhumika V.; Locke, Kelly; Mitchell, Darren J.; Mouzon, Bernadette H.; Prinjha, Rab K.; Roberts, Andy D.; Sheppard, Robert J.; Watson, Robert J.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5649-5673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Overexpression of ATAD2 (ATPase family, AAA domain contg. 2) has been linked to disease severity and progression in a wide range of cancers, and is implicated in the regulation of several drivers of cancer growth.  Little is known of the dependence of these effects upon the ATAD2 bromodomain, which has been categorized as among the least tractable of its class.  The absence of any potent, selective inhibitors limits clear understanding of the therapeutic potential of the bromodomain.  Here, the authors describe the discovery of a hit from a fragment-based targeted array.  Optimization of this produced the first known micromolar inhibitors of the ATAD2 bromodomain, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5pJPrVexJbVg90H21EOLACvtfcHk0li_be64DfyixQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCrtbbP&md5=c133ce439365cd4e4110230827a7a0a4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00772%26sid%3Dliteratum%253Aachs%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DN.%26aulast%3DBridges%26aufirst%3DA.%2BM.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DEmmons%26aufirst%3DA.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%2BV.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DMouzon%26aufirst%3DB.%2BH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRoberts%26aufirst%3DA.%2BD.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520low-micromolar%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5649%26epage%3D5673%26doi%3D10.1021%2Facs.jmedchem.5b00772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of naphthyridones into potent ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&author=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-based+optimization+of+naphthyridones+into+potent+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0lj33NxASZtpmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-based%2520optimization%2520of%2520naphthyridones%2520into%2520potent%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span> <span> </span><span class="NLM_article-title">A chemical probe for the ATAD2 bromodomain</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11382</span>– <span class="NLM_lpage">11386</span>, <span class="refDoi"> DOI: 10.1002/anie.201603928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fanie.201603928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11382-11386&author=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=R.+C.+Furzeauthor=A.+J.+Bannisterauthor=K.+H.+Cheauthor=H.+Dialloauthor=C.+Douaultauthor=P.+Grandiauthor=T.+Kouzaridesauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=S.+Robsonauthor=R.+J.+Sheppardauthor=R.+Uptonauthor=R.+J.+Watson&title=A+chemical+probe+for+the+ATAD2+bromodomain&doi=10.1002%2Fanie.201603928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Probe for the ATAD2 Bromodomain</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Furze, Rebecca C.; Bannister, Andrew J.; Che, Ka Hing; Diallo, Hawa; Douault, Clement; Grandi, Paola; Kouzarides, Tony; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Robson, Samuel; Sheppard, Robert J.; Upton, Richard; Watson, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">11382-11386</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of that target class.  Starting from a potent lead, permeability and selectivity were improved through a dual approach: (1) using CF2 as a sulfone bio-isostere to exploit the unique properties of fluorine, and (2) using 1,3-interactions to control the conformation of a piperidine ring.  This resulted in the first reported low-nanomolar, selective and cell permeable chem. probe for ATAD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqId2vaqTEeyLVg90H21EOLACvtfcHk0lj33NxASZtpmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOgu7bL&md5=7eef64dfd3c75b9c17cd7f43d368df04</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603928%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DRobson%26aufirst%3DS.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DUpton%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26atitle%3DA%2520chemical%2520probe%2520for%2520the%2520ATAD2%2520bromodomain%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11382%26epage%3D11386%26doi%3D10.1002%2Fanie.201603928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Saez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjanacz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-selective ATAD2 chemical probe with novel chemical structure and unusual mode of action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&author=A.+E.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+Stockigtauthor=A.+Ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+Diaz-Saezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=J.+Hubnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorjanacz&title=Isoform-selective+ATAD2+chemical+probe+with+novel+chemical+structure+and+unusual+mode+of+action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span></div><div class="casAuthors">Fernandez-Montalvan, Amaury E.; Berger, Markus; Kuropka, Benno; Koo, Seong Joo; Badock, Volker; Weiske, Joerg; Puetter, Vera; Holton, Simon J.; Stoeckigt, Detlef; ter Laak, Antonius; Centrella, Paolo A.; Clark, Matthew A.; Dumelin, Christoph E.; Sigel, Eric A.; Soutter, Holly H.; Troast, Dawn M.; Zhang, Ying; Cuozzo, John W.; Keefe, Anthony D.; Roche, Didier; Rodeschini, Vincent; Chaikuad, Apirat; Diaz-Saez, Laura; Bennett, James M.; Fedorov, Oleg; Huber, Kilian V. M.; Huebner, Jan; Weinmann, Hilmar; Hartung, Ingo V.; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2730-2736</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 (ANCCA) is an epigenetic regulator and transcriptional cofactor, whose overexpression has been linked to the progress of various cancer types.  Here, we report a DNA-encoded library screen leading to the discovery of BAY-850, a potent and isoform selective inhibitor that specifically induces ATAD2 bromodomain dimerization and prevents interactions with acetylated histones in vitro, as well as with chromatin in cells.  These features qualify BAY-850 as a chem. probe to explore ATAD2 biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCB3nyG9YvubVg90H21EOLACvtfcHk0li7BVnpdYV_kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL&md5=1f09000a7d7e60e30540fbbae117acfe</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DStockigt%26aufirst%3DD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DDiaz-Saez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DHubner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorjanacz%26aufirst%3DM.%26atitle%3DIsoform-selective%2520ATAD2%2520chemical%2520probe%2520with%2520novel%2520chemical%2520structure%2520and%2520unusual%2520mode%2520of%2520action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heptinstall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1843</span>– <span class="NLM_lpage">1850</span>, <span class="refDoi"> DOI: 10.1039/C8OB00099A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1039%2FC8OB00099A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29469144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivVygsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=1843-1850&author=D.+C.+Millerauthor=M.+P.+Martinauthor=S.+Adhikariauthor=A.+Brennanauthor=J.+A.+Endicottauthor=B.+T.+Goldingauthor=I.+R.+Hardcastleauthor=A.+Heptinstallauthor=S.+Hobsonauthor=C.+Jennings&title=Identification+of+a+novel+ligand+for+the+ATAD2+bromodomain+with+selectivity+over+BRD4+through+a+fragment+growing+approach&doi=10.1039%2FC8OB00099A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel ligand for the ATAD2 bromodomain with selectivity over BRD4 through a fragment growing approach</span></div><div class="casAuthors">Miller, Duncan C.; Martin, Mathew P.; Adhikari, Santosh; Brennan, Alfie; Endicott, Jane A.; Golding, Bernard T.; Hardcastle, Ian R.; Heptinstall, Amy; Hobson, Stephen; Jennings, Claire; Molyneux, Lauren; Ng, Yvonne; Wedge, Stephen R.; Noble, Martin E. M.; Cano, Celine</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1843-1850</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">ATAD2 is an ATPase that is overexpressed in a variety of cancers and assocd. with a poor patient prognosis.  This protein has been suggested to function as a cofactor for a range of transcription factors, including proto-oncogene MYC and the androgen receptor.  ATAD2 comprises an ATPase domain, implicated in chromatin remodeling, and a bromodomain which allows it to interact with acetylated histone tails.  Dissection of the functional roles of these 2 domains would benefit from the availability of selective, cell-permeable pharmacol. probes.  An in silico evaluation of the 3D structures of various bromodomains suggested that developing small mol. ligands for the bromodomain of ATAD2 is likely to be challenging, although recent reports have shown that ATAD2 bromodomain ligands can be identified.  Here, we report a structure-guided fragment-based approach to identify lead compds. for ATAD2 bromodomain inhibitor development.  Our findings indicated that the ATAD2 bromodomain could accommodate fragment hits (mol. wt., <200) that yield productive structure-activity relations, and structure-guided design enabled the introduction of selectivity over BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKPvd0l4_jgLVg90H21EOLACvtfcHk0ljYG6uY_P6ZTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivVygsLo%253D&md5=606eacb758cb091e4c35bf3da13d8298</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC8OB00099A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8OB00099A%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DD.%2BC.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DA.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DHeptinstall%26aufirst%3DA.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DJennings%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520a%2520novel%2520ligand%2520for%2520the%2520ATAD2%2520bromodomain%2520with%2520selectivity%2520over%2520BRD4%2520through%2520a%2520fragment%2520growing%2520approach%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26spage%3D1843%26epage%3D1850%26doi%3D10.1039%2FC8OB00099A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Aiming to miss a moving target: Bromo and Extra Terminal Domain (BET) selectivity in constrained ATAD2 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8321</span>– <span class="NLM_lpage">8336</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00862</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00862" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslagtrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8321-8336&author=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=R.+J.+Watsonauthor=D.+J.+Mitchellauthor=H.+Barnettauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Sheppardauthor=T.+Wernerauthor=E.+H.+Demont&title=Aiming+to+miss+a+moving+target%3A+Bromo+and+Extra+Terminal+Domain+%28BET%29+selectivity+in+constrained+ATAD2+inhibitors&doi=10.1021%2Facs.jmedchem.8b00862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Aiming to Miss a Moving Target: Bromo and Extra Terminal Domain (BET) Selectivity in Constrained ATAD2 Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Watson, Robert J.; Mitchell, Darren J.; Barnett, Heather; Prinjha, Rab K.; Rau, Christina; Sheppard, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8321-8336</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a cancer-assocd. protein whose bromodomain has been described as among the least druggable of its class.  In our recent disclosure of the first chem. probe against this bromodomain, GSK8814 (6), we described the use of a conformationally constrained methoxy piperidine to gain selectivity over the BET bromodomains.  Here we describe an orthogonal conformational restriction strategy of the piperidine ring to give potent and selective tropane inhibitors and show structural insights into why this was more challenging than expected.  Greater understanding of why different rational approaches succeeded or failed should help in the future design of selectivity in the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9UnSVNIzmhbVg90H21EOLACvtfcHk0liyU6Ci9NXtJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslagtrzK&md5=0536cc9e89031962d38a5ece0fd57749</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00862%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DAiming%2520to%2520miss%2520a%2520moving%2520target%253A%2520Bromo%2520and%2520Extra%2520Terminal%2520Domain%2520%2528BET%2529%2520selectivity%2520in%2520constrained%2520ATAD2%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8321%26epage%3D8336%26doi%3D10.1021%2Facs.jmedchem.8b00862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poncet-Montange, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardenhagen, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee Iv, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geck
Do, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardozo, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span> <span> </span><span class="NLM_article-title">Observed bromodomain flexibility reveals histone peptide- and small molecule ligand-compatible forms of ATAD2A</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1042/BJ20140933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1042%2FBJ20140933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=25486442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFCqu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2015&pages=337-346&author=G.+Poncet-Montangeauthor=Y.+Zhanauthor=J.+P.+Bardenhagenauthor=A.+Petrocchiauthor=E.+Leoauthor=X.+Shiauthor=G.+R.+Lee+Ivauthor=P.+G.+Leonardauthor=M.+K.+Geck%0ADoauthor=M.+G.+Cardozoauthor=J.+N.+Andersenauthor=W.+S.+Palmerauthor=P.+Jonesauthor=J.+E.+Ladbury&title=Observed+bromodomain+flexibility+reveals+histone+peptide-+and+small+molecule+ligand-compatible+forms+of+ATAD2A&doi=10.1042%2FBJ20140933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Observed bromodomain flexibility reveals histone peptide- and small molecule ligand-compatible forms of ATAD2</span></div><div class="casAuthors">Poncet-Montange, Guillaume; Zhan, Yanai; Bardenhagen, Jennifer P.; Petrocchi, Alessia; Leo, Elisabetta; Shi, Xi; Lee, Gilbert R.; Leonard, Paul G.; Geck Do, Mary K.; Cardozo, Mario G.; Andersen, Jannik N.; Palmer, Wylie S.; Jones, Philip; Ladbury, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-346</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Preventing histone recognition by bromodomains emerges as an attractive therapeutic approach in cancer.  Overexpression of ATAD2 (ATPase family AAA domain-contg. 2 isoform A) in cancer cells is assocd. with poor prognosis making the bromodomain of ATAD2 a promising epigenetic therapeutic target.  In the development of an in vitro assay and identification of small mol. ligands, we conducted structure-guided studies which revealed a conformationally flexible ATAD2 bromodomain.  Structural studies on apo-, peptide-and small mol.-ATAD2 complexes (by co-crystn.) revealed that the bromodomain adopts a 'closed', histone-compatible conformation and a more 'open' ligand-compatible conformation of the binding site resp.  An unexpected conformational change of the conserved asparagine residue plays an important role in driving the peptide-binding conformation remodelling.  We also identified dimethylisoxazole-contg. ligands as ATAD2 binders which aided in the validation of the in vitro screen and in the anal. of these conformational studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofrj4a58a07rVg90H21EOLACvtfcHk0liyU6Ci9NXtJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFCqu7Y%253D&md5=7c900b1b2299e72dbb6a11cd8c1d7f1d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1042%2FBJ20140933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20140933%26sid%3Dliteratum%253Aachs%26aulast%3DPoncet-Montange%26aufirst%3DG.%26aulast%3DZhan%26aufirst%3DY.%26aulast%3DBardenhagen%26aufirst%3DJ.%2BP.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DLee%2BIv%26aufirst%3DG.%2BR.%26aulast%3DLeonard%26aufirst%3DP.%2BG.%26aulast%3DGeck%2BDo%26aufirst%3DM.%2BK.%26aulast%3DCardozo%26aufirst%3DM.%2BG.%26aulast%3DAndersen%26aufirst%3DJ.%2BN.%26aulast%3DPalmer%26aufirst%3DW.%2BS.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DObserved%2520bromodomain%2520flexibility%2520reveals%2520histone%2520peptide-%2520and%2520small%2520molecule%2520ligand-compatible%2520forms%2520of%2520ATAD2A%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D466%26spage%3D337%26epage%3D346%26doi%3D10.1042%2FBJ20140933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lgU4Jn-g00sWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chakravorty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popa-Burke, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remlinger, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillmore, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span> <span> </span><span class="NLM_article-title">Nuisance compounds, PAINS filters, and dark chemical matter in the GSK HTS collection</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">532</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1177/2472555218768497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1177%2F2472555218768497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29699447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1agtrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=532-545&author=S.+J.+Chakravortyauthor=J.+Chanauthor=M.+N.+Greenwoodauthor=I.+Popa-Burkeauthor=K.+S.+Remlingerauthor=S.+D.+Pickettauthor=D.+V.+S.+Greenauthor=M.+C.+Fillmoreauthor=T.+W.+Deanauthor=J.+I.+Luengoauthor=R.+Macarron&title=Nuisance+compounds%2C+PAINS+filters%2C+and+dark+chemical+matter+in+the+GSK+HTS+collection&doi=10.1177%2F2472555218768497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Nuisance Compounds, PAINS Filters, and Dark Chemical Matter in the GSK HTS Collection</span></div><div class="casAuthors">Chakravorty, Subhas J.; Chan, James; Greenwood, Marie Nicole; Popa-Burke, Ioana; Remlinger, Katja S.; Pickett, Stephen D.; Green, Darren V. S.; Fillmore, Martin C.; Dean, Tony W.; Luengo, Juan I.; Macarron, Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">532-545</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) hits include compds. with undesirable properties.  Many filters have been described to identify such hits.  Notably, pan-assay interference compds. (PAINS) has been adopted by the community as the std. term to refer to such filters, and very useful guidelines have been adopted by the American Chem. Society (ACS) and subsequently triggered a healthy scientific debate about the pitfalls of draconian use of filters.  Using an inhibitory frequency index, we have analyzed in detail the promiscuity profile of the whole GlaxoSmithKline (GSK) HTS collection comprising more than 2 million unique compds. that have been tested in hundreds of screening assays.  We provide a comprehensive anal. of many previously published filters and newly described classes of nuisance structures that may serve as a useful source of empirical information to guide the design or growth of HTS collections and hit triaging strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ5bHsBxkwOrVg90H21EOLACvtfcHk0lgU4Jn-g00sWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1agtrnE&md5=9a620fb177aa001b0a36e90948f8e326</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1177%2F2472555218768497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555218768497%26sid%3Dliteratum%253Aachs%26aulast%3DChakravorty%26aufirst%3DS.%2BJ.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DGreenwood%26aufirst%3DM.%2BN.%26aulast%3DPopa-Burke%26aufirst%3DI.%26aulast%3DRemlinger%26aufirst%3DK.%2BS.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DFillmore%26aufirst%3DM.%2BC.%26aulast%3DDean%26aufirst%3DT.%2BW.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DMacarron%26aufirst%3DR.%26atitle%3DNuisance%2520compounds%252C%2520PAINS%2520filters%252C%2520and%2520dark%2520chemical%2520matter%2520in%2520the%2520GSK%2520HTS%2520collection%26jtitle%3DSLAS%2520Discovery%26date%3D2018%26volume%3D23%26spage%3D532%26epage%3D545%26doi%3D10.1177%2F2472555218768497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V.</span></span> <span> </span><span class="NLM_article-title">Design of a compound screening collection for use in high throughput screening</span>. <i>Comb. Chem. High Throughput Screening</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.2174/138620704772884832</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.2174%2F138620704772884832" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=14965262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2004&pages=63-70&author=G.+Harperauthor=S.+D.+Pickettauthor=D.+V.+Green&title=Design+of+a+compound+screening+collection+for+use+in+high+throughput+screening&doi=10.2174%2F138620704772884832"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a compound screening collection for use in high throughput screening</span></div><div class="casAuthors">Harper, G.; Pickett, S. D.; Green, D. V. S.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-71</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">In this paper we introduce a quant. model that relates chem. structural similarity to biol. activity, and in particular to the activity of lead series of compds. in high-throughput assays.  From this model we derive the optimal screening collection make up for a given fixed size of screening collection, and identify the conditions under which a diverse collection of compds. or a collection focusing on particular regions of chem. space are appropriate strategies.  We derive from the model a diversity function that may be used to assess compds. for acquisition or libraries for combinatorial synthesis by their ability to complement an existing screening collection.  The diversity function is linked directly through the model to the goal of more frequent discovery of lead series from high-throughput screening.  We show how the model may also be used to derive relationships between collection size and probabilities of lead discovery in high-throughput screening, and to guide the judicious application of structural filters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpThXGuh69P87Vg90H21EOLACvtfcHk0lgU4Jn-g00sWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisFWnsbg%253D&md5=b29b45b082fe5fc9d6400e1c8fd83cc0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F138620704772884832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620704772884832%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26aulast%3DGreen%26aufirst%3DD.%2BV.%26atitle%3DDesign%2520of%2520a%2520compound%2520screening%2520collection%2520for%2520use%2520in%2520high%2520throughput%2520screening%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2004%26volume%3D7%26spage%3D63%26epage%3D70%26doi%3D10.2174%2F138620704772884832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e1600760</span> <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&author=S.+Picaudauthor=K.+Leonardsauthor=J.+P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.+Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.+C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+targeting+of+bromodomains+by+bromosporine+identifies+BET+proteins+as+master+regulators+of+primary+transcription+response+in+leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0liTdfXpKESF5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520targeting%2520of%2520bromodomains%2520by%2520bromosporine%2520identifies%2520BET%2520proteins%2520as%2520master%2520regulators%2520of%2520primary%2520transcription%2520response%2520in%2520leukemia%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0liTdfXpKESF5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gay, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckenroth, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, K. C.</span></span> <span> </span><span class="NLM_article-title">Disulfide bridge formation influences ligand recognition by the ATAD2 bromodomain</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1002/prot.25636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fprot.25636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=30520161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Sqs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2019&pages=157-167&author=J.+C.+Gayauthor=B.+E.+Eckenrothauthor=C.+M.+Evansauthor=C.+Langiniauthor=S.+Carlsonauthor=J.+T.+Lloydauthor=A.+Caflischauthor=K.+C.+Glass&title=Disulfide+bridge+formation+influences+ligand+recognition+by+the+ATAD2+bromodomain&doi=10.1002%2Fprot.25636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Disulfide bridge formation influences ligand recognition by the ATAD2 bromodomain</span></div><div class="casAuthors">Gay, Jamie C.; Eckenroth, Brian E.; Evans, Chiara M.; Langini, Cassiano; Carlson, Samuel; Lloyd, Jonathan T.; Caflisch, Amedeo; Glass, Karen C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>;
        ISSN:<span class="NLM_cas:issn">1097-0134</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The ATPase family, AAA domain-contg. protein 2 (ATAD2) has a C-terminal bromodomain, which functions as a chromatin reader domain recognizing acetylated lysine on the histone tails within the nucleosome.  ATAD2 is overexpressed in many cancers and its expression is correlated with poor patient outcomes, making it an attractive therapeutic target and potential biomarker.  We solved the crystal structure of the ATAD2 bromodomain and found that it contains a disulfide bridge near the base of the acetyllysine binding pocket (Cys1057-Cys1079).  Site-directed mutagenesis revealed that removal of a free C-terminal cysteine (C1101) residue greatly improved the soly. of the ATAD2 bromodomain in vitro.  Isothermal titrn. calorimetry expts. in combination with the Ellman's assay demonstrated that formation of an intramol. disulfide bridge neg. impacts the ligand binding affinities and alters the thermodn. parameters of the ATAD2 bromodomain interaction with a histone H4K5ac peptide as well as a small mol. bromodomain ligand.  Mol. dynamics simulations indicate that the formation of the disulfide bridge in the ATAD2 bromodomain does not alter the structure of the folded state or flexibility of the acetyllysine binding pocket.  However, consideration of this unique structural feature should be taken into account when examg. ligand-binding affinity, or in the design of new bromodomain inhibitor compds. that interact with this acetyllysine reader module.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrumAF6V6frlLVg90H21EOLACvtfcHk0lg4KRsdIy-IwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Sqs7vM&md5=4d818f3ea1da7d011523f8b2ef69821c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fprot.25636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.25636%26sid%3Dliteratum%253Aachs%26aulast%3DGay%26aufirst%3DJ.%2BC.%26aulast%3DEckenroth%26aufirst%3DB.%2BE.%26aulast%3DEvans%26aufirst%3DC.%2BM.%26aulast%3DLangini%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DS.%26aulast%3DLloyd%26aufirst%3DJ.%2BT.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DGlass%26aufirst%3DK.%2BC.%26atitle%3DDisulfide%2520bridge%2520formation%2520influences%2520ligand%2520recognition%2520by%2520the%2520ATAD2%2520bromodomain%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2019%26volume%3D87%26spage%3D157%26epage%3D167%26doi%3D10.1002%2Fprot.25636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coma, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herranz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richmond, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valmaseda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span> <span> </span><span class="NLM_article-title">Process validation and screen reproducibility in high-throughput screening</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1177/1087057108326664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1177%2F1087057108326664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=19171922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslGrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=66-76&author=I.+Comaauthor=L.+Clarkauthor=E.+Diezauthor=G.+Harperauthor=J.+Herranzauthor=G.+Hofmannauthor=M.+Lennonauthor=N.+Richmondauthor=M.+Valmasedaauthor=R.+Macarron&title=Process+validation+and+screen+reproducibility+in+high-throughput+screening&doi=10.1177%2F1087057108326664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Process validation and screen reproducibility in high-throughput screening</span></div><div class="casAuthors">Coma, Isabel; Clark, Liz; Diez, Emilio; Harper, Gavin; Herranz, Jesus; Hofmann, Glenn; Lennon, Mark; Richmond, Nicola; Valmaseda, Manuel; Macarron, Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The use of large-scale compd. screening has become a key component of drug discovery projects in both the pharmaceutical and the biotechnol. industries.  More recently, these activities have also been embraced by the academic community as a major tool for chem. genomic activities.  High-throughput screening (HTS) activities constitute a major step in the initial drug discovery efforts and involve the use of large quantities of biol. reagents, hundreds of thousands to millions of compds., and the utilization of expensive equipment.  All these factors make it very important to evaluate in advance of the HTS campaign any potential issues related to reproducibility of the experimentation and the quality of the results obtained at the end of these very costly activities.  In this article, the authors describe how GlaxoSmithKline (GSK) has addressed the need of a true validation of the HTS process before embarking in full HTS campaigns.  They present 2 different aspects of the so-called validation process: (1) optimization of the HTS workflow and its validation as a quality process and (2) the statistical evaluation of the HTS, focusing on the reproducibility of results and the ability to distinguish active from nonactive compds. in a vast collection of samples.  The authors describe a variety of reproducibility indexes that are either innovative or have been adapted from generic medical diagnostic screening strategies.  In addn., they exemplify how these validation tools have been implemented in a no. of case studies at GSK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfPaLvIL_p3rVg90H21EOLACvtfcHk0ljPC5KSVuGzGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslGrtL8%253D&md5=ee5786ad5960a6089f8d7cb6346b7480</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1177%2F1087057108326664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108326664%26sid%3Dliteratum%253Aachs%26aulast%3DComa%26aufirst%3DI.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DDiez%26aufirst%3DE.%26aulast%3DHarper%26aufirst%3DG.%26aulast%3DHerranz%26aufirst%3DJ.%26aulast%3DHofmann%26aufirst%3DG.%26aulast%3DLennon%26aufirst%3DM.%26aulast%3DRichmond%26aufirst%3DN.%26aulast%3DValmaseda%26aufirst%3DM.%26aulast%3DMacarron%26aufirst%3DR.%26atitle%3DProcess%2520validation%2520and%2520screen%2520reproducibility%2520in%2520high-throughput%2520screening%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D66%26epage%3D76%26doi%3D10.1177%2F1087057108326664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of novel monosubstituted sulfonylurea derivatives as antituberculosis agents</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2012.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.ejmech.2012.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=22326144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFGru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2012&pages=18-26&author=L.+Panauthor=Y.+Jiangauthor=Z.+Liuauthor=X.+H.+Liuauthor=Z.+Liuauthor=G.+Wangauthor=Z.+M.+Liauthor=D.+Wang&title=Synthesis+and+evaluation+of+novel+monosubstituted+sulfonylurea+derivatives+as+antituberculosis+agents&doi=10.1016%2Fj.ejmech.2012.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of novel monosubstituted sulfonylurea derivatives as antituberculosis agents</span></div><div class="casAuthors">Pan, Li; Jiang, Ying; Liu, Zhen; Liu, Xing-Hai; Liu, Zhuo; Wang, Gang; Li, Zheng-Ming; Wang, Di</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18-26</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of novel monosubstituted sulfonylurea derivs. were synthesized and characterized by 1H NMR, 13C NMR, and HRMS.  These compds. were evaluated against Mycobacterium tuberculosis H37Rv in vitro.  The results showed compds. I (R1 = vinyl, R2 = Me; R1 = Ph, R2 = Me; R1 = vinyl, R2 = OMe) exhibited moderate antituberculosis activities with MIC values in the range of 20-100 mg/L.  Compds. I (R1 = CH2Cl, R2 = Me; R1 = CH2Cl, R2 = OMe) displayed good antituberculosis activities (MIC 10 mg/L), which were comparable with that of the sulfometuron Me.  Both of the two compds. showed little cytotoxicities, with an IC50 against THP-1 cells greater than 100 mg/L.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq-Y1Xea7sCrVg90H21EOLACvtfcHk0ljPC5KSVuGzGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFGru7s%253D&md5=91e3a3c06228e473bc5e48d198174308</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2012.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2012.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DX.%2BH.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DZ.%2BM.%26aulast%3DWang%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520monosubstituted%2520sulfonylurea%2520derivatives%2520as%2520antituberculosis%2520agents%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2012%26volume%3D50%26spage%3D18%26epage%3D26%26doi%3D10.1016%2Fj.ejmech.2012.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-W.</span></span> <span> </span><span class="NLM_article-title">Methyl fluorosulphonyldifluoroacetate; a new trifluoromethylating agent</span>. <i>J. Chem. Soc., Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_fpage">705</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.1039/c39890000705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1039%2Fc39890000705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADyaK3cXitVGgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1989&pages=705-706&author=Q.-Y.+Chenauthor=S.-W.+Wu&title=Methyl+fluorosulphonyldifluoroacetate%3B+a+new+trifluoromethylating+agent&doi=10.1039%2Fc39890000705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Methyl (fluorosulfonyl)difluoroacetate; a new trifluoromethylating agent</span></div><div class="casAuthors">Chen, Qingyun; Wu, Shengwen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Chemical Communications</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">705-6</span>CODEN:
                <span class="NLM_cas:coden">JCCCAT</span>;
        ISSN:<span class="NLM_cas:issn">0022-4936</span>.
    </div><div class="casAbstract">Me (fluorosulfonyl)difluoroacetate, in the presence of CuI, is used as the source of a trifluoromethyl group, which replaces the halogen in aryl, alkenyl, and alkyl halides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphEwpeqCNED7Vg90H21EOLACvtfcHk0ljPC5KSVuGzGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitVGgtA%253D%253D&md5=9e937c3bd4587b28e708210d3b648fd3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2Fc39890000705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc39890000705%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DQ.-Y.%26aulast%3DWu%26aufirst%3DS.-W.%26atitle%3DMethyl%2520fluorosulphonyldifluoroacetate%253B%2520a%2520new%2520trifluoromethylating%2520agent%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Chem.%2520Commun.%26date%3D1989%26spage%3D705%26epage%3D706%26doi%3D10.1039%2Fc39890000705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. C. C.</span>; <span class="NLM_string-name">Atkinson, S. J.</span>; <span class="NLM_string-name">Bamborough, P.</span>; <span class="NLM_string-name">Barnett, H.</span>; <span class="NLM_string-name">Chung, C.</span>; <span class="NLM_string-name">Gordon, L.</span>; <span class="NLM_string-name">Mitchell, D. J.</span>; <span class="NLM_string-name">Phillipou, A.</span>; <span class="NLM_string-name">Prinjha, R.</span>; <span class="NLM_string-name">Sheppard, R. J.</span>; <span class="NLM_string-name">Tomkinson, N. C.</span>; <span class="NLM_string-name">Watson, R. J.</span>; <span class="NLM_string-name">Demont, E. H.</span></span> in preparation.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Lucas%2C+S.+C.+C.%3B+Atkinson%2C+S.+J.%3B+Bamborough%2C+P.%3B+Barnett%2C+H.%3B+Chung%2C+C.%3B+Gordon%2C+L.%3B+Mitchell%2C+D.+J.%3B+Phillipou%2C+A.%3B+Prinjha%2C+R.%3B+Sheppard%2C+R.+J.%3B+Tomkinson%2C+N.+C.%3B+Watson%2C+R.+J.%3B+Demont%2C+E.+H.+in+preparation."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DLucas%26aufirst%3DS.%2BC.%2BC." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-lysine binding site of bromodomain-containing protein 4 (BRD4) interacts with diverse kinase inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+Emberauthor=J.+Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Acetyl-lysine+binding+site+of+bromodomain-containing+protein+4+%28BRD4%29+interacts+with+diverse+kinase+inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0liET4qni6gkiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DAcetyl-lysine%2520binding%2520site%2520of%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520interacts%2520with%2520diverse%2520kinase%2520inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Good, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span> <span> </span><span class="NLM_article-title">Diving into the water: inducible binding conformations for BRD4, TAF1(2), BRD9, and CECR2 bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5391</span>– <span class="NLM_lpage">5402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5391-5402&author=T.+D.+Crawfordauthor=V.+Tsuiauthor=E.+M.+Flynnauthor=S.+Wangauthor=A.+M.+Taylorauthor=A.+Coteauthor=J.+E.+Audiaauthor=M.+H.+Beresiniauthor=D.+J.+Burdickauthor=R.+Cummingsauthor=L.+A.+Dakinauthor=M.+Duplessisauthor=A.+C.+Goodauthor=M.+C.+Hewittauthor=H.+R.+Huangauthor=H.+Jayaramauthor=J.+R.+Kieferauthor=Y.+Jiangauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+Poyauthor=F.+A.+Romeroauthor=Y.+Tangauthor=J.+Wangauthor=Z.+Xuauthor=L.+E.+Zawadzkeauthor=X.+Zhuauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=S.+Bellonauthor=A.+G.+Cochran&title=Diving+into+the+water%3A+inducible+binding+conformations+for+BRD4%2C+TAF1%282%29%2C+BRD9%2C+and+CECR2+bromodomains&doi=10.1021%2Facs.jmedchem.6b00264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Diving into the Water: Inducible Binding Conformations for BRD4, TAF1(2), BRD9, and CECR2 Bromodomains</span></div><div class="casAuthors">Crawford, Terry D.; Tsui, Vickie; Flynn, E. Megan; Wang, Shumei; Taylor, Alexander M.; Cote, Alexandre; Audia, James E.; Beresini, Maureen H.; Burdick, Daniel J.; Cummings, Richard; Dakin, Les A.; Duplessis, Martin; Good, Andrew C.; Hewitt, Michael C.; Huang, Hon-Ren; Jayaram, Hariharan; Kiefer, James R.; Jiang, Ying; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Poy, Florence; Romero, F. Anthony; Tang, Yong; Wang, Jian; Xu, Zhaowu; Zawadzke, Laura E.; Zhu, Xiaoyu; Albrecht, Brian K.; Magnuson, Steven R.; Bellon, Steve; Cochran, Andrea G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5391-5402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. role played by non-BET bromodomains remains poorly understood, and it is therefore imperative to identify potent and highly selective inhibitors to effectively explore the biol. of individual bromodomain proteins.  A ligand-efficient nonselective bromodomain inhibitor was identified from a 6-Me pyrrolopyridone fragment.  Small hydrophobic substituents replacing the N-Me group were designed directing toward the conserved bromodomain water pocket, and two distinct binding conformations were then obsd.  The substituents either directly displaced and rearranged the conserved solvent network, as in BRD4(1) and TAF1(2), or induced a narrow hydrophobic channel adjacent to the lipophilic shelf, as in BRD9 and CECR2.  The preference of distinct substituents for individual bromodomains provided selectivity handles useful for future lead optimization efforts for selective BRD9, CECR2, and TAF1(2) inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTLnuVCIKdHbVg90H21EOLACvtfcHk0liET4qni6gkiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFSnsbs%253D&md5=bf92e94c5fd3ee36bd8441294ca85185</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00264%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DHuang%26aufirst%3DH.%2BR.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZawadzke%26aufirst%3DL.%2BE.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26atitle%3DDiving%2520into%2520the%2520water%253A%2520inducible%2520binding%2520conformations%2520for%2520BRD4%252C%2520TAF1%25282%2529%252C%2520BRD9%252C%2520and%2520CECR2%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5391%26epage%3D5402%26doi%3D10.1021%2Facs.jmedchem.6b00264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talluri, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for the N-terminal Bromodomain-and-Extra-Terminal-family selectivity of a dual kinase-bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9316</span>– <span class="NLM_lpage">9334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01248</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01248" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9316-9334&author=A.+Divakaranauthor=S.+K.+Talluriauthor=A.+M.+Ayoubauthor=N.+K.+Mishraauthor=H.+Cuiauthor=J.+C.+Widenauthor=N.+Berndtauthor=J.+Y.+Zhuauthor=A.+S.+Carlsonauthor=J.+J.+Topczewskiauthor=E.+K.+Schonbrunnauthor=D.+A.+Harkiauthor=W.+C.+K.+Pomerantz&title=Molecular+basis+for+the+N-terminal+Bromodomain-and-Extra-Terminal-family+selectivity+of+a+dual+kinase-bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.8b01248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase-Bromodomain Inhibitor</span></div><div class="casAuthors">Divakaran, Anand; Talluri, Siva K.; Ayoub, Alex M.; Mishra, Neeraj K.; Cui, Huarui; Widen, John C.; Berndt, Norbert; Zhu, Jin-Yi; Carlson, Angela S.; Topczewski, Joseph J.; Schonbrunn, Ernst K.; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9316-9334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis.  When dysregulated, "reader" proteins become drivers of disease.  In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging.  We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase-bromodomain inhibitors.  Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1).  Cellular efficacy was demonstrated via redn. of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 prodn. through potential synergistic inhibition of BRD4(1) and p38α.  These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZhnL9cPdDi7Vg90H21EOLACvtfcHk0ljDGs01vpUxtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM&md5=5176ef754a898b3e03fc56e2662ab399</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DTalluri%26aufirst%3DS.%2BK.%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%2BK.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DMolecular%2520basis%2520for%2520the%2520N-terminal%2520Bromodomain-and-Extra-Terminal-family%2520selectivity%2520of%2520a%2520dual%2520kinase-bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9316%26epage%3D9334%26doi%3D10.1021%2Facs.jmedchem.8b01248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auffinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hays, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhof, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, P. S.</span></span> <span> </span><span class="NLM_article-title">Halogen bonds in biological molecules</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">16789</span>– <span class="NLM_lpage">16794</span>, <span class="refDoi"> DOI: 10.1073/pnas.0407607101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1073%2Fpnas.0407607101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=15557000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFansr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=16789-16794&author=P.+Auffingerauthor=F.+A.+Haysauthor=E.+Westhofauthor=P.+S.+Ho&title=Halogen+bonds+in+biological+molecules&doi=10.1073%2Fpnas.0407607101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen bonds in biological molecules</span></div><div class="casAuthors">Auffinger, Pascal; Hays, Franklin A.; Westhof, Eric; Ho, P. Shing</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">16789-16794</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Short oxygen-halogen interactions have been known in org. chem. since the 1950s and recently have been exploited in the design of supramol. assemblies.  The present survey of protein and nucleic acid structures reveals similar halogen bonds as potentially stabilizing inter- and intramol. interactions that can affect ligand binding and mol. folding.  A halogen bond in biomols. can be defined as a short C-X···O-Y interaction (C-X is a carbon-bonded chlorine, bromine, or iodine, and O-Y is a carbonyl, hydroxyl, charged carboxylate, or phosphate group), where the X···O distance is less than or equal to the sums of the resp. van der Waals radii (3.27 Å for Cl···O, 3.37 Å for Br···O, and 3.50 Å for I···O) and can conform to the geometry seen in small mols., with the C-X···O angle ≈165° (consistent with a strong directional polarization of the halogen) and the X···O-Y angle ≈120°.  Alternative geometries can be imposed by the more complex environment found in biomols., depending on which of the two types of donor systems are involved in the interaction: (i) the lone pair electrons of oxygen (and, to a lesser extent, nitrogen and sulfur) atoms or (ii) the delocalized π-electrons of peptide bonds or carboxylate or amide groups.  Thus, the specific geometry and diversity of the interacting partners of halogen bonds offer new and versatile tools for the design of ligands as drugs and materials in nanotechnol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPMdZU91iT77Vg90H21EOLACvtfcHk0lhG6lb6kQ2Isg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtFansr3P&md5=dfcce25d4e336c66294f6ab8ce412154</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407607101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407607101%26sid%3Dliteratum%253Aachs%26aulast%3DAuffinger%26aufirst%3DP.%26aulast%3DHays%26aufirst%3DF.%2BA.%26aulast%3DWesthof%26aufirst%3DE.%26aulast%3DHo%26aufirst%3DP.%2BS.%26atitle%3DHalogen%2520bonds%2520in%2520biological%2520molecules%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D16789%26epage%3D16794%26doi%3D10.1073%2Fpnas.0407607101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Politzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span> <span> </span><span class="NLM_article-title">σ-hole bonding: a physical interpretation</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1007/128_2014_568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1007%2F128_2014_568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=25467532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVylsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2015&pages=19-42&author=P.+Politzerauthor=J.+S.+Murrayauthor=T.+Clark&title=%CF%83-hole+bonding%3A+a+physical+interpretation&doi=10.1007%2F128_2014_568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">σ-Hole Bonding: A Physical Interpretation</span></div><div class="casAuthors">Politzer, Peter; Murray, Jane S.; Clark, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">Halogen Bonding I: Impact on Materials Chemistry and Life Sciences</span>),
    <span class="NLM_cas:pages">19-42</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">1436-5049</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">The anisotropic electronic densities of covalently-bonded Group IV-VII atoms frequently give rise to regions of pos. electrostatic potential on the extensions of covalent bonds to these atoms.  Through such pos. "σ-holes," the atoms can interact attractively and highly directionally with neg. sites such as the lone pairs of Lewis bases, anions, π electrons, etc.  In the case of Group VII this is called "halogen bonding." Hydrogen bonding can be viewed as a less directional subset of σ-hole interactions.  Since pos. σ-holes often exist in conjunction with regions of neg. potential, the atoms can also interact favorably with pos. sites.  In accordance with the Hellmann-Feynman theorem, all of these interactions are purely Coulombic in nature (which encompasses polarization and dispersion).  The strength of σ-hole bonding increases with the magnitudes of the potentials of the pos. σ-hole and the neg. site; their polarizabilities must sometimes also be taken explicitly into account.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0l1L0LKANkLVg90H21EOLACvtfcHk0lhG6lb6kQ2Isg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVylsL4%253D&md5=49273da6edbcfa745168c833194ee890</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F128_2014_568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2014_568%26sid%3Dliteratum%253Aachs%26aulast%3DPolitzer%26aufirst%3DP.%26aulast%3DMurray%26aufirst%3DJ.%2BS.%26aulast%3DClark%26aufirst%3DT.%26atitle%3D%25CF%2583-hole%2520bonding%253A%2520a%2520physical%2520interpretation%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2015%26volume%3D358%26spage%3D19%26epage%3D42%26doi%3D10.1007%2F128_2014_568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goudreau, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hucke, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibeault, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llinas-Brunet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, P. W.</span></span> <span> </span><span class="NLM_article-title">Combined X-ray, NMR, and kinetic analyses reveal uncommon binding characteristics of the hepatitis C virus NS3-NS4A protease inhibitor BI 201335</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">11434</span>– <span class="NLM_lpage">11443</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.211417</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1074%2Fjbc.M110.211417" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=21270126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjslKgsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=11434-11443&author=C.+T.+Lemkeauthor=N.+Goudreauauthor=S.+Zhaoauthor=O.+Huckeauthor=D.+Thibeaultauthor=M.+Llinas-Brunetauthor=P.+W.+White&title=Combined+X-ray%2C+NMR%2C+and+kinetic+analyses+reveal+uncommon+binding+characteristics+of+the+hepatitis+C+virus+NS3-NS4A+protease+inhibitor+BI+201335&doi=10.1074%2Fjbc.M110.211417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Combined X-ray, NMR, and Kinetic Analyses Reveal Uncommon Binding Characteristics of the Hepatitis C Virus NS3-NS4A Protease Inhibitor BI 201335</span></div><div class="casAuthors">Lemke, Christopher T.; Goudreau, Nathalie; Zhao, Songping; Hucke, Oliver; Thibeault, Diane; Llinas-Brunet, Montse; White, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">11434-11443</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Hepatitis C virus infection, a major cause of liver disease worldwide, is curable, but currently approved therapies have suboptimal efficacy.  Supplementing these therapies with direct-acting antiviral agents has the potential to considerably improve treatment prospects for hepatitis C virus-infected patients.  The crit. role played by the viral NS3 protease makes it an attractive target, and despite its shallow, solvent-exposed active site, several potent NS3 protease inhibitors are currently in the clinic.  BI 201335, which is progressing through Phase IIb trials, contains a unique C-terminal carboxylic acid that binds noncovalently to the active site and a bromo-quinoline substitution on its proline residue that provides significant potency.  In this work we have used stopped flow kinetics, x-ray crystallog., and NMR to characterize these distinctive features.  Key findings include: slow assocn. and dissocn. rates within a single-step binding mechanism; the crit. involvement of water mols. in acid binding; and protein side chain rearrangements, a bromine-oxygen halogen bond, and profound pKa changes within the catalytic triad assocd. with binding of the bromo-quinoline moiety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkdn8YiZZHALVg90H21EOLACvtfcHk0lhVUCWHNng4Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjslKgsbs%253D&md5=2281c4bbfd76e34934f129fb61dcf926</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.211417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.211417%26sid%3Dliteratum%253Aachs%26aulast%3DLemke%26aufirst%3DC.%2BT.%26aulast%3DGoudreau%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DHucke%26aufirst%3DO.%26aulast%3DThibeault%26aufirst%3DD.%26aulast%3DLlinas-Brunet%26aufirst%3DM.%26aulast%3DWhite%26aufirst%3DP.%2BW.%26atitle%3DCombined%2520X-ray%252C%2520NMR%252C%2520and%2520kinetic%2520analyses%2520reveal%2520uncommon%2520binding%2520characteristics%2520of%2520the%2520hepatitis%2520C%2520virus%2520NS3-NS4A%2520protease%2520inhibitor%2520BI%2520201335%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D11434%26epage%3D11443%26doi%3D10.1074%2Fjbc.M110.211417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Exploring chemistry with the fragment molecular orbital method</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">7562</span>– <span class="NLM_lpage">7577</span>, <span class="refDoi"> DOI: 10.1039/c2cp23784a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1039%2Fc2cp23784a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=22410762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVykurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=7562-7577&author=D.+G.+Fedorovauthor=T.+Nagataauthor=K.+Kitaura&title=Exploring+chemistry+with+the+fragment+molecular+orbital+method&doi=10.1039%2Fc2cp23784a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring chemistry with the fragment molecular orbital method</span></div><div class="casAuthors">Fedorov, Dmitri G.; Nagata, Takeshi; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7562-7577</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The fragment MO (FMO) method makes possible nearly linear scaling calcns. of large mol. systems, such as water clusters, proteins and DNA.  In particular, FMO has been widely used in biochem. applications involving protein-ligand binding and drug design.  The method has been efficiently parallelized suitable for petascale computing.  Many commonly used wave functions and solvent models have been interfaced with FMO.  We review the historical background of FMO, and summarize its method development and applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_TiaN_HBr9rVg90H21EOLACvtfcHk0lhVUCWHNng4Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVykurw%253D&md5=903e3457b1dc5749950ebae5c954960a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1039%2Fc2cp23784a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2cp23784a%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DNagata%26aufirst%3DT.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DExploring%2520chemistry%2520with%2520the%2520fragment%2520molecular%2520orbital%2520method%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2012%26volume%3D14%26spage%3D7562%26epage%3D7577%26doi%3D10.1039%2Fc2cp23784a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Pair interaction energy decomposition analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1002/jcc.20496</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fjcc.20496" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=17109433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXktVKgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=222-237&author=D.+G.+Fedorovauthor=K.+Kitaura&title=Pair+interaction+energy+decomposition+analysis&doi=10.1002%2Fjcc.20496"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Pair interaction energy decomposition analysis</span></div><div class="casAuthors">Fedorov, Dmitri G.; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">222-237</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The energy decompn. anal. (EDA) by Kitaura and Morokuma was redeveloped in the framework of the fragment MO method (FMO).  The proposed pair interaction energy decompn. anal. (PIEDA) can treat large mol. clusters and the systems in which fragments are connected by covalent bonds, such as proteins.  The interaction energy in PIEDA is divided into the same contributions as in EDA: the electrostatic, exchange-repulsion, and charge transfer energies, to which the correlation (dispersion) term was added.  The careful comparison to the ab initio EDA interaction energies for water clusters with 2-16 mols. revealed that PIEDA has the error of at most 1.2 kcal/mol (or about 1%).  The anal. was applied to (H2O)1024, the α helix, β turn, and β strand of polyalanine (ALA)10, as well as to the synthetic protein (PDB code 1L2Y) with 20 residues.  The comparative aspects of the polypeptide isomer stability are discussed in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ89I3W8YX1LVg90H21EOLACvtfcHk0lhVUCWHNng4Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXktVKgug%253D%253D&md5=759c711290fe1c44b7c86a28104ccef7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20496&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20496%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DPair%2520interaction%2520energy%2520decomposition%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2007%26volume%3D28%26spage%3D222%26epage%3D237%26doi%3D10.1002%2Fjcc.20496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heifetz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKinnon, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwan, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brookfield, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chudyk, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burch, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span> <span> </span><span class="NLM_article-title">Fragment Molecular Orbital method applied to lead optimization of novel interleukin-2 inducible T-cell kinase (ITK) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4352</span>– <span class="NLM_lpage">4363</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12jt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4352-4363&author=A.+Heifetzauthor=G.+Traniauthor=M.+Aldeghiauthor=C.+H.+MacKinnonauthor=P.+A.+McEwanauthor=F.+A.+Brookfieldauthor=E.+I.+Chudykauthor=M.+Bodkinauthor=Z.+Peiauthor=J.+D.+Burchauthor=D.+F.+Ortwine&title=Fragment+Molecular+Orbital+method+applied+to+lead+optimization+of+novel+interleukin-2+inducible+T-cell+kinase+%28ITK%29+inhibitors&doi=10.1021%2Facs.jmedchem.6b00045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Molecular Orbital Method Applied to Lead Optimization of Novel Interleukin-2 Inducible T-Cell Kinase (ITK) Inhibitors</span></div><div class="casAuthors">Heifetz, Alexander; Trani, Giancarlo; Aldeghi, Matteo; MacKinnon, Colin H.; McEwan, Paul A.; Brookfield, Frederick A.; Chudyk, Ewa I.; Bodkin, Mike; Pei, Zhonghua; Burch, Jason D.; Ortwine, Daniel F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4352-4363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of inducible T-cell kinase (ITK), a nonreceptor tyrosine kinase, may represent a novel treatment for allergic asthma.  In our previous reports, we described the discovery of sulfonylpyridine (SAP), benzothiazole (BZT), indazole (IND), and tetrahydroindazole (THI) series as novel ITK inhibitors and how computational tools such as dihedral scans and docking were used to support this process.  X-ray crystallog. and modeling were applied to provide essential insight into ITK-ligand interactions.  However, "visual inspection" traditionally used for the rationalization of protein-ligand affinity cannot always explain the full complexity of the mol. interactions.  The fragment MO (FMO) quantum-mech. (QM) method provides a complete list of the interactions formed between the ligand and protein that are often omitted from traditional structure-based descriptions.  FMO methodol. was successfully used as part of a rational structure-based drug design effort to improve the ITK potency of high-throughput screening hits, ultimately delivering ligands with potency in the subnanomolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTGF0Bo-GJM7Vg90H21EOLACvtfcHk0lijjKHOcR2lYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12jt70%253D&md5=b0958d9c29f7d61aadcc730ac00e8f90</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00045%26sid%3Dliteratum%253Aachs%26aulast%3DHeifetz%26aufirst%3DA.%26aulast%3DTrani%26aufirst%3DG.%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DMacKinnon%26aufirst%3DC.%2BH.%26aulast%3DMcEwan%26aufirst%3DP.%2BA.%26aulast%3DBrookfield%26aufirst%3DF.%2BA.%26aulast%3DChudyk%26aufirst%3DE.%2BI.%26aulast%3DBodkin%26aufirst%3DM.%26aulast%3DPei%26aufirst%3DZ.%26aulast%3DBurch%26aufirst%3DJ.%2BD.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26atitle%3DFragment%2520Molecular%2520Orbital%2520method%2520applied%2520to%2520lead%2520optimization%2520of%2520novel%2520interleukin-2%2520inducible%2520T-cell%2520kinase%2520%2528ITK%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4352%26epage%3D4363%26doi%3D10.1021%2Facs.jmedchem.6b00045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Molecular recognition mechanism of FK506 binding protein: An all-electron fragment molecular orbital study</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">158</span>, <span class="refDoi"> DOI: 10.1002/prot.21389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fprot.21389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=17387719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1Oktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2007&pages=145-158&author=I.+Nakanishiauthor=D.+G.+Fedorovauthor=K.+Kitaura&title=Molecular+recognition+mechanism+of+FK506+binding+protein%3A+An+all-electron+fragment+molecular+orbital+study&doi=10.1002%2Fprot.21389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition mechanism of FK506 binding protein: an all-electron fragment molecular orbital study</span></div><div class="casAuthors">Nakanishi, Isao; Fedorov, Dmitri G.; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">145-158</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The fragment MO (FMO) method has enabled electronic structure calcns. and geometry optimizations of very large mols. with ab initio quality.  We applied the method to four FK506 binding protein (FKBP) complexes (denoted by their PDB codes 1fkb, 1fkf, 1fkg, and 1fki) contg. rapamycin, FK506, and two synthetic ligands.  The geometries of reduced complex models were optimized at the RHF (FMO-RHF) level using the 3-21G basis set, and then for a better est. of binding, the energetics were refined at a higher level of theory (2nd order Moller-Plesset perturbation theory FMO-MP2 with the 6-31G* basis set).  Thus, obtained binding energies were -103.9 (-82.0), -102.2 (-69.2), -70.1 (-57.7), and -71.3 (-55.3) kcal/mol for 1fkb, 1fkf, 1fkg, and 1fki, resp., where the correlation contribution is given in parentheses.  The results show that the electron correlation contribution to binding is extremely important, and it accounts for 70-80% of the binding energy.  The mol. recognition mechanism of FKBP was analyzed in detail based on the FMO-pair interactions between protein residues and the ligands.  Solvation effects on the protein-ligand binding were estd. using the Poisson-Boltzmann/surface area model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6aOw8ua9Pm7Vg90H21EOLACvtfcHk0lijjKHOcR2lYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1Oktb8%253D&md5=1f07d1d4804c137ced134d4fd1d0412f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fprot.21389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21389%26sid%3Dliteratum%253Aachs%26aulast%3DNakanishi%26aufirst%3DI.%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DMolecular%2520recognition%2520mechanism%2520of%2520FK506%2520binding%2520protein%253A%2520An%2520all-electron%2520fragment%2520molecular%2520orbital%2520study%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2007%26volume%3D68%26spage%3D145%26epage%3D158%26doi%3D10.1002%2Fprot.21389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueda, K.</span></span> <span> </span><span class="NLM_article-title">Application of the fragment molecular orbital method analysis to fragment-based drug discovery of BET (bromodomain and extra-terminal proteins) inhibitors</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2017.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.jmgm.2017.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=28365532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsl2ktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2017&pages=73-82&author=M.+Ozawaauthor=T.+Ozawaauthor=K.+Ueda&title=Application+of+the+fragment+molecular+orbital+method+analysis+to+fragment-based+drug+discovery+of+BET+%28bromodomain+and+extra-terminal+proteins%29+inhibitors&doi=10.1016%2Fj.jmgm.2017.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Application of the fragment molecular orbital method analysis to fragment-based drug discovery of BET (bromodomain and extra-terminal proteins) inhibitors</span></div><div class="casAuthors">Ozawa, Motoyasu; Ozawa, Tomonaga; Ueda, Kazuyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">73-82</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The mol. interactions of inhibitors of bromodomains (BRDs) were investigated.  BRDs are protein interaction modules that recognizing ε-N-acetyl-lysine (εAc-Lys) motifs found in histone tails and are promising protein-protein interaction (PPI) targets.  First, we analyzed a peptide ligand contg. εAc-Lys to evaluate native PPIs.  We then analyzed tetrahydroquinazoline-6-yl-benzensulfonamide derivs. found by fragment-based drug design (FBDD) and examd. their interactions with the protein compared with the peptide ligand in terms of the inter-fragment interaction energy.  In addn., we analyzed benzodiazepine derivs. that are high-affinity ligands for BRDs and examd. differences in the CH/π interactions of the amino acid residues.  We further surveyed changes in the charges of the amino acid residues among individual ligands, performed pair interaction energy decompn. anal. and estd. the water profile within the ligand binding site.  Thus, useful insights for drug design were provided.  Through these analyses and considerations, we show that the FMO method is a useful drug design tool to evaluate the process of FBDD and to explore PPI inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU2gVSiemC4rVg90H21EOLACvtfcHk0lijjKHOcR2lYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsl2ktrw%253D&md5=bf9ac28f01c2f5fa4a4e2084272ee6b3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2017.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2017.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DOzawa%26aufirst%3DM.%26aulast%3DOzawa%26aufirst%3DT.%26aulast%3DUeda%26aufirst%3DK.%26atitle%3DApplication%2520of%2520the%2520fragment%2520molecular%2520orbital%2520method%2520analysis%2520to%2520fragment-based%2520drug%2520discovery%2520of%2520BET%2520%2528bromodomain%2520and%2520extra-terminal%2520proteins%2529%2520inhibitors%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2017%26volume%3D74%26spage%3D73%26epage%3D82%26doi%3D10.1016%2Fj.jmgm.2017.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaura, K.</span></span> <span> </span><span class="NLM_article-title">Use of an auxiliary basis set to describe the polarization in the fragment molecular orbital method</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>597</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/j.cplett.2014.02.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1016%2Fj.cplett.2014.02.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFahsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=597&publication_year=2014&pages=99-105&author=D.+G.+Fedorovauthor=K.+Kitaura&title=Use+of+an+auxiliary+basis+set+to+describe+the+polarization+in+the+fragment+molecular+orbital+method&doi=10.1016%2Fj.cplett.2014.02.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Use of an auxiliary basis set to describe the polarization in the fragment molecular orbital method</span></div><div class="casAuthors">Fedorov, Dmitri G.; Kitaura, Kazuo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">597</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-105</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We developed a dual basis approach within the fragment MO formalism enabling efficient and accurate use of large basis sets.  The method was tested on water clusters and polypeptides and applied to perform geometry optimization of chignolin (PDB: 1UAO) in soln. at the level of DFT/6-31++G**, obtaining a structure in agreement with expt. (RMSD of 0.4526 Å).  The polarization in polypeptides is discussed with a comparison of the α-helix and β-strand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoahaZbElNpPrVg90H21EOLACvtfcHk0li5VSW2xnpAbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFahsro%253D&md5=745a811aff3ea79042f9b58907c1074a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cplett.2014.02.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cplett.2014.02.029%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DD.%2BG.%26aulast%3DKitaura%26aufirst%3DK.%26atitle%3DUse%2520of%2520an%2520auxiliary%2520basis%2520set%2520to%2520describe%2520the%2520polarization%2520in%2520the%2520fragment%2520molecular%2520orbital%2520method%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D2014%26volume%3D597%26spage%3D99%26epage%3D105%26doi%3D10.1016%2Fj.cplett.2014.02.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozuch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. M.</span></span> <span> </span><span class="NLM_article-title">Halogen bonds: benchmarks and theoretical analysis</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1918</span>– <span class="NLM_lpage">1931</span>, <span class="refDoi"> DOI: 10.1021/ct301064t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct301064t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlyjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1918-1931&author=S.+Kozuchauthor=J.+M.+Martin&title=Halogen+bonds%3A+benchmarks+and+theoretical+analysis&doi=10.1021%2Fct301064t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Halogen Bonds: Benchmarks and Theoretical Analysis</span></div><div class="casAuthors">Kozuch, Sebastian; Martin, Jan M. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1918-1931</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We carried out an extensive survey of wave function and DFT methods to test their accuracy on geometries and dissocn. energies of halogen bonds (XB).  For that purpose, we built two benchmark sets (XB18 and XB51).  Between the DFT methods, it was found that functionals with high exact exchange or long-range corrections were suitable for these dimers, esp. M06-2X, ωB97XD, and double hybrids.  Dispersion corrections tend to be detrimental, in spite of the fact that XB is considered a noncovalent interaction.  Wave function techniques require heavy correlated methods (i.e., CCSD(T)) or parametrized ones (SCS-MP2 or SCS(MI)MP2).  Heavy basis sets are needed to obtain high accuracy, such as aVQZ or aVTZ+CP, and ideally a CBS extrapolation.  Relativistic ECPs are also important, even for the bromine based dimers.  In addn., we explored some XB with new theor. tools, the NCI ("Non-Covalent Interactions") method and the NOFF ("Natural Orbital Fukui Functions").</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPO_Z9Idg5E7Vg90H21EOLACvtfcHk0li5VSW2xnpAbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlyjs7g%253D&md5=3ca6c0b9e5047e75d73946a36fad3d40</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fct301064t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct301064t%26sid%3Dliteratum%253Aachs%26aulast%3DKozuch%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DJ.%2BM.%26atitle%3DHalogen%2520bonds%253A%2520benchmarks%2520and%2520theoretical%2520analysis%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D1918%26epage%3D1931%26doi%3D10.1021%2Fct301064t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boys, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, F.</span></span> <span> </span><span class="NLM_article-title">The calculation of small molecular interactions by the differences of separate total energies. Some procedures with reduced errors</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1080/00268977000101561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1080%2F00268977000101561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1alt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1970&pages=553-566&author=S.+F.+Boysauthor=F.+Bernardi&title=The+calculation+of+small+molecular+interactions+by+the+differences+of+separate+total+energies.+Some+procedures+with+reduced+errors&doi=10.1080%2F00268977000101561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The calculation of small molecular interactions by the differences of separate total energies. Some procedures with reduced errors</span></div><div class="casAuthors">Boys, S. F.; Bernardi, F.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">553-566</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A new direct difference method for the computation of mol. interactions has been based on a bivariational transcorrelated treatment, together with special methods for the balancing of other errors.  It appears that these new features can give a strong redn. in the error of the interaction energy, and they seem to be particularly suitable for computations in the important region near the min. energy.  It has been generally accepted that this problem is dominated by unresolved difficulties and the relation of the new methods of these apparent difficulties is analyzed here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp23TXDqDyeYrVg90H21EOLACvtfcHk0li5VSW2xnpAbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1alt7fM&md5=c4f847835a638b4fc449fae688c557e9</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1080%2F00268977000101561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268977000101561%26sid%3Dliteratum%253Aachs%26aulast%3DBoys%26aufirst%3DS.%2BF.%26aulast%3DBernardi%26aufirst%3DF.%26atitle%3DThe%2520calculation%2520of%2520small%2520molecular%2520interactions%2520by%2520the%2520differences%2520of%2520separate%2520total%2520energies.%2520Some%2520procedures%2520with%2520reduced%2520errors%26jtitle%3DMol.%2520Phys.%26date%3D1970%26volume%3D19%26spage%3D553%26epage%3D566%26doi%3D10.1080%2F00268977000101561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldeghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodkin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">Large-scale analysis of water stability in bromodomain binding pockets with grand canonical Monte Carlo</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">19</span> <span class="refDoi"> DOI: 10.1038/s42004-018-0019-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1038%2Fs42004-018-0019-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29863194" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&author=M.+Aldeghiauthor=G.+A.+Rossauthor=M.+J.+Bodkinauthor=J.+W.+Essexauthor=S.+Knappauthor=P.+C.+Biggin&title=Large-scale+analysis+of+water+stability+in+bromodomain+binding+pockets+with+grand+canonical+Monte+Carlo&doi=10.1038%2Fs42004-018-0019-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fs42004-018-0019-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-018-0019-x%26sid%3Dliteratum%253Aachs%26aulast%3DAldeghi%26aufirst%3DM.%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodkin%26aufirst%3DM.%2BJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DLarge-scale%2520analysis%2520of%2520water%2520stability%2520in%2520bromodomain%2520binding%2520pockets%2520with%2520grand%2520canonical%2520Monte%2520Carlo%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26doi%3D10.1038%2Fs42004-018-0019-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span> <span> </span><span class="NLM_article-title">Protein dynamics and structural waters in bromodomains</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0186570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0186570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1371%2Fjournal.pone.0186570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=29077715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=X.+Zhangauthor=K.+Chenauthor=Y.+D.+Wuauthor=O.+Wiest&title=Protein+dynamics+and+structural+waters+in+bromodomains&doi=10.1371%2Fjournal.pone.0186570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics and structural waters in bromodomains</span></div><div class="casAuthors">Zhang, Xiaoxiao; Chen, Kai; Wu, Yun-Dong; Wiest, Olaf</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e0186570/1-e0186570/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic readers of acetylated lysines that are integral parts of histone tails.  The 61 bromodomains in humans are structurally highly conserved but specifically bind to widely varying recognition motifs, suggesting that dynamic rather than static factors are responsible for recognition selectivity.  To test this hypothesis, the dynamics of the binding sites and structural water mols. of four bromodomains (ATAD2, BAZ2B, BRD2(1) and CREBBP) representing four different subtypes is studied with 1 μs MD simulations using the RSFF2 force field.  The different dynamics of the ZA-loops and BC-loops between the four bromodomains leads to distinct patterns for the opening and closing of the binding pocket.  This in turn dets. the structural and energetic properties of the structural waters in the binding pocket, suggesting that these waters are not only important for the recognition itself, as has been proposed previously, but also contribute to the selectivity of different bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqzaoNSjS-7Vg90H21EOLACvtfcHk0lhHk__c3f-_sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGms7bN&md5=3ed02a6025d67e5c0e8b511c3dc6756e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0186570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0186570%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DY.%2BD.%26aulast%3DWiest%26aufirst%3DO.%26atitle%3DProtein%2520dynamics%2520and%2520structural%2520waters%2520in%2520bromodomains%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0186570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhHk__c3f-_sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> In  <i>Data Reduction</i>, Proceedings of CCP4 Study Weekend, on Data Collection and Processing, <span class="NLM_year">1993</span>; pp  <span class="NLM_fpage">114</span>– <span class="NLM_lpage">122</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&pages=114-122&author=P.+R.+Evans&title=Data+Reduction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26jtitle%3DData%2520Reduction%26date%3D1993%26spage%3D114%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgYIC_55sOTJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Collaborative
Computational Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>D50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D50&publication_year=1994&pages=760-763&author=Collaborative%0AComputational+Project%2C+Number+4&title=The+CCP4+suite%3A+programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3DD50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgYIC_55sOTJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljcflqoRDroDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span> <i>Schrödinger
Release 2018-3: Maestro</i>; <span class="NLM_publisher-name">Schrödinger
LLC</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease+2018-3%3A+Maestro%3B+Schr%C3%B6dinger%0ALLC%3A+New+York%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSchr%25C3%25B6dinger%250ARelease%25202018-3%253A%2520Maestro%26pub%3DSchr%25C3%25B6dinger%250ALLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bochevarov, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braden, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philipp, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halls, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Jaguar: a high-performance quantum chemistry software program with strengths in life and materials sciences</span>. <i>Int. J. Quantum Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2110</span>– <span class="NLM_lpage">2142</span>, <span class="refDoi"> DOI: 10.1002/qua.24481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fqua.24481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCqs7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2013&pages=2110-2142&author=A.+D.+Bochevarovauthor=E.+Harderauthor=T.+F.+Hughesauthor=J.+R.+Greenwoodauthor=D.+A.+Bradenauthor=D.+M.+Philippauthor=D.+Rinaldoauthor=M.+D.+Hallsauthor=J.+Zhangauthor=R.+A.+Friesner&title=Jaguar%3A+a+high-performance+quantum+chemistry+software+program+with+strengths+in+life+and+materials+sciences&doi=10.1002%2Fqua.24481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Jaguar: A high-performance quantum chemistry software program with strengths in life and materials sciences</span></div><div class="casAuthors">Bochevarov, Art D.; Harder, Edward; Hughes, Thomas F.; Greenwood, Jeremy R.; Braden, Dale A.; Philipp, Dean M.; Rinaldo, David; Halls, Mathew D.; Zhang, Jing; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Quantum Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2110-2142</span>CODEN:
                <span class="NLM_cas:coden">IJQCB2</span>;
        ISSN:<span class="NLM_cas:issn">0020-7608</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Jaguar is an ab initio quantum chem. program that specializes in fast electronic structure predictions for mol. systems of medium and large size.  Jaguar focuses on computational methods with reasonable computational scaling with the size of the system, such as d. functional theory (DFT) and local second-order Moller-Plesset perturbation theory.  The favorable scaling of the methods and the high efficiency of the program make it possible to conduct routine computations involving several thousand MOs.  This performance is achieved through a utilization of the pseudospectral approxn. and several levels of parallelization.  The speed advantages are beneficial for applying Jaguar in biomol. computational modeling.  Addnl., owing to its superior wave function guess for transition-metal-contg. systems, Jaguar finds applications in inorg. and bioinorg. chem.  The emphasis on larger systems and transition metal elements paves the way toward developing Jaguar for its use in materials science modeling.  The article describes the historical and new features of Jaguar, such as improved parallelization of many modules, innovations in ab initio pKa prediction, and new semiempirical corrections for nondynamic correlation errors in DFT.  Jaguar applications in drug discovery, materials science, force field parameterization, and other areas of computational research are reviewed.  Timing benchmarks and other results obtained from the most recent Jaguar code are provided.  The article concludes with a discussion of challenges and directions for future development of the program. © 2013 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt_0dnCf0Yq7Vg90H21EOLACvtfcHk0lgWtAQagkU-DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCqs7%252FP&md5=2fc119bb9e0501e55d7292f7735c24ea</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fqua.24481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fqua.24481%26sid%3Dliteratum%253Aachs%26aulast%3DBochevarov%26aufirst%3DA.%2BD.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DHughes%26aufirst%3DT.%2BF.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DBraden%26aufirst%3DD.%2BA.%26aulast%3DPhilipp%26aufirst%3DD.%2BM.%26aulast%3DRinaldo%26aufirst%3DD.%26aulast%3DHalls%26aufirst%3DM.%2BD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DJaguar%253A%2520a%2520high-performance%2520quantum%2520chemistry%2520software%2520program%2520with%2520strengths%2520in%2520life%2520and%2520materials%2520sciences%26jtitle%3DInt.%2520J.%2520Quantum%2520Chem.%26date%3D2013%26volume%3D113%26spage%3D2110%26epage%3D2142%26doi%3D10.1002%2Fqua.24481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suenaga, M.</span></span> <span> </span><span class="NLM_article-title">Facio: new computational chemistry environment for PC GAMESS</span>. <i>J. Comput. Chem. Jpn.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.2477/jccj.4.25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.2477%2Fjccj.4.25" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtV2ktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=25-32&author=M.+Suenaga&title=Facio%3A+new+computational+chemistry+environment+for+PC+GAMESS&doi=10.2477%2Fjccj.4.25"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Facio: New computational chemistry environment for PC GAMESS</span></div><div class="casAuthors">Suenaga, Masahiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer Chemistry, Japan</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-32</span>CODEN:
                <span class="NLM_cas:coden">JCCJAG</span>;
        ISSN:<span class="NLM_cas:issn">1347-1767</span>.
    
            (<span class="NLM_cas:orgname">Nippon Konputa Kagakkai</span>)
        </div><div class="casAbstract">A new computational chem. environment (Facio) for PC GAMESS has been developed.  In the environment, mol. modeling, prepn. of GAMESS input file, execution of GAMESS calcn. and visualization of the results are facilitated with GUI (graphic user interface).  Mol. models are displayed as fine 3D graphics based on OpenGL technol.  Mol. modeling features are compared to those of com. available softwares.  Isovalue surface of MOs, total electron d. and electrostatic potentials are viewed as mesh surface.  IR/Raman intensities are represented as simulated spectra and normal modes of vibrations are shown as animated graphics.  All the features needed for computational chem. are integrated in this software.  Besides the above functions, Facio can make models of polypeptides or polynucleotides with collaboration with Tinker, the mol. mechanics and dynamics package and can make a mol. surface with the help of MSMS, the software for the calcn. of solvent-excluded surface.  Further, it can serve as PDB file viewer for biopolymers, such as protein and nucleic acid, since the native mol. format of Facio is PDB (protein data bank).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5gYQe6nfCfLVg90H21EOLACvtfcHk0lgWtAQagkU-DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtV2ktbc%253D&md5=964c411aac299863eb401868b5d011b3</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.2477%2Fjccj.4.25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2477%252Fjccj.4.25%26sid%3Dliteratum%253Aachs%26aulast%3DSuenaga%26aufirst%3DM.%26atitle%3DFacio%253A%2520new%2520computational%2520chemistry%2520environment%2520for%2520PC%2520GAMESS%26jtitle%3DJ.%2520Comput.%2520Chem.%2520Jpn.%26date%3D2005%26volume%3D4%26spage%3D25%26epage%3D32%26doi%3D10.2477%2Fjccj.4.25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldridge, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boatz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elbert, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koseki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">General atomic and molecular electronic structure system</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1363</span>, <span class="refDoi"> DOI: 10.1002/jcc.540141112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=10.1002%2Fjcc.540141112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1993&pages=1347-1363&author=M.+W.+Schmidtauthor=K.+K.+Baldridgeauthor=J.+A.+Boatzauthor=S.+T.+Elbertauthor=M.+S.+Gordonauthor=J.+H.+Jensenauthor=S.+Kosekiauthor=N.+Matsunagaauthor=K.+A.+Nguyenauthor=S.+Su&title=General+atomic+and+molecular+electronic+structure+system&doi=10.1002%2Fjcc.540141112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">General atomic and molecular electronic structure system</span></div><div class="casAuthors">Schmidt, Michael W.; Baldridge, Kim K.; Boatz, Jerry A.; Elbert, Steven T.; Gordon, Mark S.; Jensen, Jan H.; Koseki, Shiro; Matsunaga, Nikita; Nguyen, Kiet A.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1347-63</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">A description of the ab initio quantum chem. package GAMESS is presented.  Chem. systems contg. atoms through Rn can be treated with wave functions ranging from the simplest closed-shell case up to a general MCSCF case, permitting calcns. at the necessary level of sophistication.  Emphasis is given to novel features of the program.  The parallelization strategy used in the RHF, ROHF, UHF, and GVB sections of the program is described, and detailed speedup results are given.  Parallel calcns. can be run on ordinary workstations as well as dedicated parallel machines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pHblAdXU4rVg90H21EOLACvtfcHk0lgC2Jw-UQ-uNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFaqtQ%253D%253D&md5=4d9e37cd3ec67a92bdfb5b0c0084707c</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540141112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540141112%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DM.%2BW.%26aulast%3DBaldridge%26aufirst%3DK.%2BK.%26aulast%3DBoatz%26aufirst%3DJ.%2BA.%26aulast%3DElbert%26aufirst%3DS.%2BT.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26aulast%3DKoseki%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DN.%26aulast%3DNguyen%26aufirst%3DK.%2BA.%26aulast%3DSu%26aufirst%3DS.%26atitle%3DGeneral%2520atomic%2520and%2520molecular%2520electronic%2520structure%2520system%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1993%26volume%3D14%26spage%3D1347%26epage%3D1363%26doi%3D10.1002%2Fjcc.540141112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DAI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DAI','PDB','3DAI'); return false;">PDB: 3DAI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S56" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S56','PDB','6S56'); return false;">PDB: 6S56</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S57" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S57','PDB','6S57'); return false;">PDB: 6S57</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55','PDB','6S55'); return false;">PDB: 6S55</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A5N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A5N','PDB','5A5N'); return false;">PDB: 5A5N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P5O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P5O','PDB','3P5O'); return false;">PDB: 3P5O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TZ2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TZ2','PDB','4TZ2'); return false;">PDB: 4TZ2</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i46"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00673">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_15401"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00673">10.1021/acs.jmedchem.9b00673</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">ATAD2 constructs and evaluation; full ATAD2 and BET TR-FRET assay data; X-ray refinement statistics and density maps; peptide binding data; robustness set and bromodomain toolset %<i>I</i> summaries; SPR and NMR qualification of representative hits; electrostatic potential surfaces; QM-calculated fragment interaction energies; and LC/MS spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_001.pdf">jm9b00673_si_001.pdf (3.35 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00673/suppl_file/jm9b00673_si_002.csv">jm9b00673_si_002.csv (2.22 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Crystallography structures are deposited as <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S55">6S55</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S56">6S56</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S57">6S57</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00673&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00673%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-16%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00673" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b2d21ee7d952","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
