
<html lang="en"     class="pb-page"  data-request-id="1001f02c-deda-4949-a94b-a6539a87bfb2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.5b01760;issue:issue:10.1021/jmcmar.2016.59.issue-6;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival" /></meta><meta name="dc.Creator" content="Jason G.  Kettle" /></meta><meta name="dc.Creator" content="Husam  Alwan" /></meta><meta name="dc.Creator" content="Michal  Bista" /></meta><meta name="dc.Creator" content="Jason  Breed" /></meta><meta name="dc.Creator" content="Nichola L.  Davies" /></meta><meta name="dc.Creator" content="Kay  Eckersley" /></meta><meta name="dc.Creator" content="Shaun  Fillery" /></meta><meta name="dc.Creator" content="Kevin M.  Foote" /></meta><meta name="dc.Creator" content="Louise  Goodwin" /></meta><meta name="dc.Creator" content="David R.  Jones" /></meta><meta name="dc.Creator" content="Helena  Käck" /></meta><meta name="dc.Creator" content="Alan  Lau" /></meta><meta name="dc.Creator" content="J. Willem M.  Nissink" /></meta><meta name="dc.Creator" content="Jon  Read" /></meta><meta name="dc.Creator" content="James S.  Scott" /></meta><meta name="dc.Creator" content="Ben  Taylor" /></meta><meta name="dc.Creator" content="Graeme  Walker" /></meta><meta name="dc.Creator" content="Lisa  Wissler" /></meta><meta name="dc.Creator" content="Marta  Wylot" /></meta><meta name="dc.Description" content="Recent literature has claimed that inhibition of the enzyme MTH1 can eradicate cancer. MTH1 is one of the “housekeeping” enzymes that are responsible for hydrolyzing damaged nucleotides in cells an..." /></meta><meta name="Description" content="Recent literature has claimed that inhibition of the enzyme MTH1 can eradicate cancer. MTH1 is one of the “housekeeping” enzymes that are responsible for hydrolyzing damaged nucleotides in cells an..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 16, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01760" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01760" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01760" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01760" /></link>
        
    
    

<title>Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01760" /></meta><meta property="og:title" content="Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0020.jpeg" /></meta><meta property="og:description" content="Recent literature has claimed that inhibition of the enzyme MTH1 can eradicate cancer. MTH1 is one of the “housekeeping” enzymes that are responsible for hydrolyzing damaged nucleotides in cells and thus prevent them from being incorporated into DNA. We have developed orthogonal and chemically distinct tool compounds to those published in the literature to allow us to test the hypothesis that inhibition of MTH1 has wide applicability in the treatment of cancer. Here we present the work that led to the discovery of three structurally different series of MTH1 inhibitors with excellent potency, selectivity, and proven target engagement in cells. None of these compounds elicited the reported cellular phenotype, and additional siRNA and CRISPR experiments further support these observations. Critically, the difference between the responses of our highly selective inhibitors and published tool compounds suggests that the effect reported for the latter may be due to off-target cytotoxic effects. As a result, we conclude that the role of MTH1 in carcinogenesis and utility of its inhibition is yet to be established." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01760"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01760">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01760&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01760&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01760&amp;href=/doi/10.1021/acs.jmedchem.5b01760" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2346-2361</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00283" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.5b00608" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jason+G.++Kettle">Jason G. Kettle</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Husam++Alwan">Husam Alwan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michal++Bista">Michal Bista</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jason++Breed">Jason Breed</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nichola+L.++Davies">Nichola L. Davies</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kay++Eckersley">Kay Eckersley</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shaun++Fillery">Shaun Fillery</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kevin+M.++Foote">Kevin M. Foote</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Louise++Goodwin">Louise Goodwin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+R.++Jones">David R. Jones</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Helena++K%C3%A4ck">Helena Käck</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alan++Lau">Alan Lau</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Willem+M.++Nissink">J. Willem M. Nissink</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jon++Read">Jon Read</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=James+S.++Scott">James S. Scott</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ben++Taylor">Ben Taylor</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Graeme++Walker">Graeme Walker</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lisa++Wissler">Lisa Wissler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marta++Wylot">Marta Wylot</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Oncology Innovative Medicines Unit, AstraZeneca, 35S47 Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Oncology Innovative Medicines Unit, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Discovery Sciences, AstraZeneca, Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Discovery Sciences, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Discovery Sciences, AstraZeneca, Pepparedsleden 1, 431 83 Mölndal, Sweden</span></div><div class="corresp-info"><strong>*</strong>Phone: +44 (0)1625 517920. Fax: +44(0)1625519749. E-mail: <a href="/cdn-cgi/l/email-protection#bcd6ddcfd3d292d7d9c8c8d0d9fcddcfc8ceddc6d9d2d9dfdd92dfd3d1"><span class="__cf_email__" data-cfemail="731912001c1d5d181607071f1633120007011209161d1610125d101c1e">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01760&amp;href=/doi/10.1021%2Facs.jmedchem.5b01760" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2346–2361</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 16, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 November 2015</li><li><span class="item_label"><b>Published</b> online</span>16 February 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 March 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01760" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01760</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editors’ Choice" /></img><span>ACS Editors’ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01760"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">9802</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">86</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01760" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Potent and Selective Inhibitors of MTH1 Probe Its Role in Cancer Cell Survival&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;G. Kettle&quot;},{&quot;first_name&quot;:&quot;Husam&quot;,&quot;last_name&quot;:&quot;Alwan&quot;},{&quot;first_name&quot;:&quot;Michal&quot;,&quot;last_name&quot;:&quot;Bista&quot;},{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;Breed&quot;},{&quot;first_name&quot;:&quot;Nichola&quot;,&quot;last_name&quot;:&quot;L. Davies&quot;},{&quot;first_name&quot;:&quot;Kay&quot;,&quot;last_name&quot;:&quot;Eckersley&quot;},{&quot;first_name&quot;:&quot;Shaun&quot;,&quot;last_name&quot;:&quot;Fillery&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;M. Foote&quot;},{&quot;first_name&quot;:&quot;Louise&quot;,&quot;last_name&quot;:&quot;Goodwin&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;R. Jones&quot;},{&quot;first_name&quot;:&quot;Helena&quot;,&quot;last_name&quot;:&quot;Käck&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;Lau&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Willem M. Nissink&quot;},{&quot;first_name&quot;:&quot;Jon&quot;,&quot;last_name&quot;:&quot;Read&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;S. Scott&quot;},{&quot;first_name&quot;:&quot;Ben&quot;,&quot;last_name&quot;:&quot;Taylor&quot;},{&quot;first_name&quot;:&quot;Graeme&quot;,&quot;last_name&quot;:&quot;Walker&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;Wissler&quot;},{&quot;first_name&quot;:&quot;Marta&quot;,&quot;last_name&quot;:&quot;Wylot&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;2346-2361&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01760&quot;},&quot;abstract&quot;:&quot;Recent literature has claimed that inhibition of the enzyme MTH1 can eradicate cancer. MTH1 is one of the “housekeeping” enzymes that are responsible for hydrolyzing damaged nucleotides in cells and thus prevent them from being incorporated into DNA. We have developed orthogonal and chemically distinct tool compounds to those published in the literature to allow us to test the hypothesis that inhibition of MTH1 has wide applicability in the treatment of cancer. Here we present the work that led to the discovery of three structurally different series of MTH1 inhibitors with excellent potency, selectivity, and proven target engagement in cells. None of these compounds elicited the reported cellular phenotype, and additional siRNA and CRISPR experiments further support these observations. Critically, the difference between the responses of our highly selective inhibitors and published tool compounds suggests that the effect reported for the latter may be due to off-target cytotoxic effects. As a result, we concl&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01760&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01760" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01760&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01760" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01760&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01760" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01760&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01760&amp;href=/doi/10.1021/acs.jmedchem.5b01760" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01760" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01760" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01760%26sid%3Dliteratum%253Aachs%26pmid%3D26878898%26genre%3Darticle%26aulast%3DKettle%26date%3D2016%26atitle%3DPotent%2Band%2BSelective%2BInhibitors%2Bof%2BMTH1%2BProbe%2BIts%2BRole%2Bin%2BCancer%2BCell%2BSurvival%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D6%26spage%3D2346%26epage%3D2361%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/jmcmar.2016.59.issue-6/20160324/jmcmar.2016.59.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Recent literature has claimed that inhibition of the enzyme MTH1 can eradicate cancer. MTH1 is one of the “housekeeping” enzymes that are responsible for hydrolyzing damaged nucleotides in cells and thus prevent them from being incorporated into DNA. We have developed orthogonal and chemically distinct tool compounds to those published in the literature to allow us to test the hypothesis that inhibition of MTH1 has wide applicability in the treatment of cancer. Here we present the work that led to the discovery of three structurally different series of MTH1 inhibitors with excellent potency, selectivity, and proven target engagement in cells. None of these compounds elicited the reported cellular phenotype, and additional siRNA and CRISPR experiments further support these observations. Critically, the difference between the responses of our highly selective inhibitors and published tool compounds suggests that the effect reported for the latter may be due to off-target cytotoxic effects. As a result, we conclude that the role of MTH1 in carcinogenesis and utility of its inhibition is yet to be established.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81777" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81777" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Within the tumor cell, high levels of reactive oxygen species lead to oxidative damage of DNA directly, but damage may also occur to the free pool of nucleotides produced within the mitochondria. Such oxidative damage is believed to play a key role in spontaneous mutagenesis because incorporation of oxidized nucleotides into DNA can cause base-pair mistranslation, potentially leading to genomic instability and cell death. When incorporated in DNA, the oxidation product of dGTP, 8-oxo-dGTP, can be translated to an adenine rather than a cytosine moiety during replication, thereby leading to a transversion.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> MTH1 is a member of the Nudix phosphohydrolase superfamily of enzymes and its role is to hydrolyze certain oxidized nucleotides, thereby preventing their incorporation into DNA, and its substrates include 8-oxo-dGTP and 2-OH-dATP. Recent papers claim a role for MTH1 in the general survival of cancer cells and describe experiments to support such a “cancer lethal” phenotype. Data are presented to show that inhibiting MTH1 function either by RNAi-mediated knock-down of expression or by small molecule inhibition in vitro using novel compounds TH588 and TH287 (<b>1</b> and <b>2</b>),<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> or by <i>S</i>-crizotinib <b>3</b><a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), the less active enantiomer of the clinically approved kinase inhibitor, leads to a cell killing effect in cancer but not nononcogenic cell lines, an effect which was associated with induction of enhanced DNA damage response. The authors tested their hypothesis using small molecule inhibitors and inducible shRNA in in vivo xenograft tumor models to show inhibition of tumor growth when targeting MTH1. Intrigued by these claims, we sought to repeat and extend the key validation work outlined and to develop novel independent chemical series with which to test the claimed effects of the compounds and target. If the role of MTH1 in tumor cell survival is as described then this would indeed be a highly significant discovery, although previous work in this area using MTH1(−/−) mice appeared somewhat contradictory.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Fortuitously, we had access to compounds that were able to serve as leads from a previous internal project that had sought agonists of Toll-like receptor 7 (TLR7) and established cross-reactivity with MTH1 as described below.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Literature inhibitors of MTH1, <b>1</b>–<b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Toll-like receptors (TLR) are pattern recognition receptors that serve to recognize microbial pathogens and fragments of bacterial DNA in order to elicit an immune response.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Such receptors are primed to recognize distinct pathogen-associated molecular patterns (PAMPs) within microbes. Imiquimod <b>4</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>), first developed by 3M Pharmaceuticals, is the only approved small molecule agonist of toll-like receptor 7 (TLR7) and is used in the topical treatment of basal cell carcinoma (BCC), genital warts, and actinic keratosis. Agonism of TLR7 results in stimulation of the innate immune system via induction of cytokines such as interferon alpha (IFNα) and IL-12 and can result in significant antitumor activity. Clearance rates in BCC following imiquimod treatment have been reported in the range 73–87%, and for many patients the therapy is effectively curative.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> During the course of historical research into agonists of TLR7,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> a photoactivatable, cross-linkable TLR7 agonist probe based on an imiquimod template, <b>5</b> (pEC<sub>50</sub> of 1 μM for the induction of IL-12 in THP-1 cells), was synthesized to identify which cellular proteins the probe interacted with. At the time of this research, such TLR7 agonists had not been shown to mediate their effects via a direct interaction with TLR7. Initial investigations were carried out on THP-1 cells as these could be differentiated to be TLR7 responsive. No displaceable binding was observed in a protein of molecular mass consistent with membrane associated TLR7, but displaceable binding was observed to a cytosolic 20 kDa protein. The interaction was observed in both differentiated and undifferentiated cells. This protein was determined to be MTH1. At this stage, a high-throughput screen was carried out against MTH1 with the intention of identifying additional novel equity against this target and several series of interest were identified.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. TLR7 agonist Imiquimod (<b>4</b>) and a photoaffinity probe TLR7 agonist (<b>5</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A working hypothesis that MTH1 may be the true effector of the efficacy of TLR7 agonists was ultimately discounted based on a number of key experiments, including the observation that TLR7 agonist activity was unaltered in splenocytes from MTH1 knockout mice and that an 85% reduction in protein expression of MTH1 via siRNA knock-down had no observable effect on TLR7 signaling. Intriguingly, further testing of TLR7 agonists from different chemical series also indicated cross-reactivity of these ligands with MTH1 and we speculate this may be due to recognition of the similar molecular architecture of the oxidized nucleotides that act as MTH1 substrates and the nucleotide bases present in the bacterial PAMPs.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> With the link to TLR7 disproven, internal research on MTH1 inhibitors was halted in 2003. When, more than a decade later, a role in oncology re-emerged for MTH1, this information proved crucial in enabling us to rapidly develop our own potent and selective cell probes with which to investigate this target as described below.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03535" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03535" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of macrocyclic MTH1 inhibitors followed closely the routes described for a related series of TLR7 agonists.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Treatment of dichloropyrimidine <b>6</b> with ammonia gave aminopyrimidine <b>7</b>, which was protected as its ethylcarbamate derivative <b>8</b> in good yield (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Alkylation of the carbamate nitrogen with a protected phenolic benzyl group to give <b>9</b> was followed by displacement of the second chlorine atom to install the key methylamino motif in <b>10</b>. Introduction of the ether side chain to give <b>12</b> was achieved following displacement of the sulfone <b>11</b>, formed by oxidation of thioether <b>10</b>, with 2,2′-oxidiethanol. Removal of the <i>tert</i>-butyldimethylsilyl protecting group gave phenol <b>13</b>, which was subject to a Mitsunobu induced macrocyclization to give <b>14</b>. Finally, formation of the imidazolone ring was achieved by iron-mediated reduction of the nitro group with concomitant ring-closure onto the carbamate to give macrocycle <b>15</b>. An analogous strategy was adopted for lactams <b>30</b>–<b>33</b> (see <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>), although in these cases the key macrocyclization step involved macrolactamization in place of Mitsunobu reaction (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_001.pdf" class="ext-link">Supporting Information</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0013.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Macrocycle MTH1 Inhibitor <b>15</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) NH<sub>3</sub>, <i>i</i>-PrOH, Et<sub>3</sub>N, 0 °C, 5 h; (ii) EtOCOCl, THF, Et<sub>3</sub>N, 0 °C, 5 h, 76% (2 steps); (iii) <i>m</i>-OTBDMS benzyl chloride, NaI, K<sub>2</sub>CO<sub>3</sub>, acetone, 20 °C, 16 h; (iv) MeNH<sub>2</sub>, THF, 20 °C, 16 h, 100% (2 steps); (v) <i>m</i>CPBA, DCM, 20 °C, 2 h; (vi) 2,2′-oxidiethanol, NaH, THF, 20 °C, 17 h, 71% (2 steps); (vii) TBAF, THF, 20 °C, 16 h, 86%; (vii) DTAD, Ph<sub>3</sub>P, THF, 20 °C, 20 h, (ix) Fe, AcOH, water, 60 °C, 4 h, 7% (2 steps).</p></p></figure><div class="NLM_p">Quinoline amides were constructed as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> from the 4-hydroxyquinoline ester <b>16</b> by conversion to the chloride <b>17</b> using phosphoryl trichloride then subsequent displacement with the requisite aniline, affording intermediates <b>18</b> and <b>20</b>. The ester functionality was then reacted with methylamine to give amide inhibitors <b>19</b> and <b>21</b>. Procedures for the synthesis of other members of this aniline series are described in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_001.pdf" class="ext-link">Supporting Information</a>. The ether linked example was made by displacement of <b>17</b> with phenol under basic conditions and then hydrolysis of the ester <b>22</b>, conversion to the acid chloride, and reaction with methylamine to give <b>23</b>. Aminoquinazoline MTH1 inhibitor <b>24</b> was made using a Suzuki reaction between 4-chloroquinazolin-2-amine and phenyldimethylacetamide 4-boronic ester as shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0014.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Quinoline Amide MTH1 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) POCl<sub>3</sub>, 100 °C, 1 h, 90%; (ii) 4-chloro-2-fluoroaniline, AcOH, <i>i</i>-PrOH, 100 °C, 16 h, 99%; (iii) MeNH<sub>2</sub>, Me<sub>3</sub>Al, THF, 60 °C, 2 h, 79%; (iv) aniline, AcOH, DMF, 100 °C, 1 h, 76%; (v) MeNH<sub>2</sub>, EtOH, 120 °C, 3 h, 81%; (vi) phenol, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 16 h, 100%; (vii) (a) NaOH, THF, 20 °C, 17 h, (b) SOCl<sub>2</sub>, 80 °C, 1 h, (c) MeNH<sub>2</sub>, DMF/THF, 20 °C, 1 h, 74%.</p></p></figure><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0015.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Aminoquinazoline MTH1 Inhibitor <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Pd-118, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 2 h, 28%.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73604" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73604" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> shows the MTH1 inhibitory potencies for a range of TLR7 modulators known in the literature. Imiquimod <b>4</b> itself has reasonable submicromolar potency, but Resiquimod, <b>25</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> known to be a much more potent TLR7 agonist, shows slightly reduced affinity for MTH1. Compound <b>26</b> is from a distinct series of orally bioavailable purinone TLR7 agonists, first reported by Dainippon Sumitomo Pharmaceuticals,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> that elicits a TLR7-mediated cellular response with similar potency to these clinical agents. Critical for this work, <i>N</i>-methylation of <b>26</b>, to give <b>27</b>, is observed to completely ablate TLR7 activity in this series and also results in a modest increase in MTH1 affinity. This observation has previously been reported for a closely related series of TLR7 agonist purinones and indicates that the exocyclic NH<sub>2</sub> motif is critical for TLR7 activity.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> In a similar fashion, it has been reported that <i>N</i>-methylation of <b>4</b> also results in complete loss of TLR7 agonist activity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> These data show that MTH1 is able to bind agonists of TLR7 as well as inactive close exemplars.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. MTH1 Potencies of Assorted TLR7 Ligands<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0016.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last"><sup>1</sup>Inhibition of MTH1 enzyme was assessed by detecting the inorganic pyrophosphate generated when the nucleoside triphosphate substrate, 8-oxo-dGTP, is hydrolyzed. <sup>2</sup>Induction of IL-12 in human PBMC cells. All data are in μM and are the mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units.</p></div></div><div></div></div><div class="NLM_p">A crystal structure of compound <b>25</b> solved in complex with full length p18 isoform MTH1 (M1–156) with an N-terminal 6-His TEV protease cleavable tag is shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>. Compound <b>25</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a) makes hydrogen bonds between the quinoline nitrogen and residue D119 and the amino group and D120. These hydrogen bonds with D119 and D120 are common features of MTH1 ligands as demonstrated in the overlay with product 8-oxo-dGMP (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c) and also feature in protein–ligand structures of <b>1</b> and <b>3</b>. Compound <b>25</b> displays an additional hydrogen bonding contact with the backbone carbonyl of T8, another interaction it has in common with 8-oxo-dGMP (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). The interaction with T8 is hypothesized to contribute to selectivity for the deoxyribose group of 8-oxo-dGMP.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The dimethylhydroxy group of <b>25</b> pushes “down” into the pocket where the ribose of the substrate/product sits (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>c). Two polar interactions made by the product 8-oxo-dGMP are not seen with inhibitors <b>25</b>; an interaction between the guanosine base and N33 and between the α-phosphate group and K23 terminal amino group. N33 is a mobile residue which can move to accommodate different chemotypes binding to MTH1. The quinoline ring of <b>25</b> occupies the same space as the nucleobase of 8-oxo-dGMP, nestling among a number of residues highlighted in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b. These residues form a pocket around the ligands making various hydrophobic and π-stacking interactions. The hydrophobic residues include W117 (above the purine binding site), L9, I70, F72, M81, and V83 (below the base binding site) and F74 and F139 (on the right-hand side of the nucleobase binding site). Finally, the side-chain of F27 is at the tip of a flexible loop (residues 23–31) that folds over the nucleobase binding site (not shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> for clarity). The mobility of this loop, along with minor shifts in other hydrophobic residues, readily accommodates different sizes and shapes of ligands. In the complex with <b>25</b>, rather unusually, the loop is open and the position that F27 normally adopts is occupied by a second molecule of <b>25</b>, which may be an artifact of the crystallographic system and high compound concentration used.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structures of human MTH1 in complex with (a) <b>25</b> (pink, resiquimod), (b) complex with <b>25</b> rotated to show the hydrophobic residues (white), and (c) complexes with <b>25</b> and 8-oxo-dGMP (orange) superimposed. Selected carbon atoms, protein backbone cartoon, and close polar contacts are represented in the complex color. Compounds are shown as sticks, and protein residue side chains are shown as thinner sticks. Close polar contacts are represented as dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The X-ray crystal structure of compound <b>27</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a) shows that it binds at the same site as compound <b>25</b> and the base of 8-oxo-dGTP, making the usual hydrogen bonding interactions with side chains of D119 and D120. The phenyl group binds in the hydrophobic site. The asparagine side chain of N33 moves in toward the binding site to occupy a similar conformation to that seen in the 8-oxo-dGMP complex. Examination of the crystal structure of <b>27</b> bound in MTH1 highlighted the relatively close proximity of the pendant methoxyethyl side chain to the <i>N</i>1-benzyl group, with the distance to the <i>meta</i>-position of the order of 3.5 Å (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a). We hypothesized that this close proximity of groups was unlikely to be a dominant conformation in solution, contributing to the moderate affinity observed. A macrocyclization strategy that connects the ether side chain to the <i>meta</i>-position of the benzyl group was expected to lock the molecule in its active binding conformation and result in significant potency gains through reduced entropic penalty of association and increased energy barrier of dissociation. Intriguingly, synthetic precedent for such a macrocyclization strategy had been reported in the patent literature for a series of purinone TLR7 agonists related to <b>26</b>.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> It is unclear given the lack of structural information for TLR7 how such a design strategy was arrived at, although homology models of TLR7 receptors have been reported.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) X-ray crystal structures of human MTH1 in complex with <b>27</b> (green). (b) Complexes with <b>27</b> (green) and <b>15</b> (yellow) overlaid. Selected carbon atoms, protein backbone cartoon, and close polar contacts are represented in the complex color. Compounds are shown as sticks, and protein residue side chains are shown as thinner sticks. Close polar contacts are represented as dotted lines. Some atoms of the loop covering the active site (residues 23–31) have been removed for clarity. (c) Example conformational ensemble for <b>15</b>. Lowest-energy conformation (green) has a ring orientation similar to that seen for <b>27</b> (cyan) in the protein–ligand structure. (d) Modeled conformation of amide-containing macrocycle <b>30</b> (yellow) overlaid on X-ray structure conformation of <b>27</b> (cyan) shows differential orientation of phenyl ring. (e) Conformational ensemble of <b>29</b> derived using NMR suggests that the dominant conformations in solution (gray) are very similar to the conformation seen in the protein–ligand structure (maroon).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Gratifyingly, macrocyclization of <b>27</b> utilizing an <i>n</i>-pentyl bis-ether linker gave a compound with sub-nM potency against MTH1, <b>28</b>, representing over a 1700-fold increase in potency, validating our hypothesis around the relatively high energy binding conformation of the former (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). This compound was relatively lipophilic, and so a contribution to this increase in potency from additional lipophilicity could not be ruled out (because compound <b>27</b> has a log <i>D</i> = 2.6, <b>28</b> is nearly a log unit more lipophilic). One consequence of this increased lipophilicity is an increase in in vitro clearance as measured in human microsomes, although in vitro permeability is high and the compound demonstrates acceptable free binding levels in human plasma. Shortening the linker to an <i>n</i>-butyl bis-ether linker, compound <b>29</b>, gave a profile that was largely similar, but the reduced lipophilicity of this compound was not enough to improve in vitro clearance. A breakthrough came when introducing a polar ether linker into the all carbon side chain of <b>28</b> to give <b>15</b>. This compound maintained its exceptional potency against MTH1 but did so at lipophilicity less than that of the uncyclized analogue <b>27</b>, representing a 1000-fold increase in potency for no increase in log <i>D</i>. With a reduction in log <i>D</i> came a significant improvement in in vitro clearance, with a profile comparable to <b>27</b>. Encouraged by this result, we sought to further improve properties by increasing the polarity of the linking group. Both secondary and tertiary amides <b>30</b> and <b>31</b> have substantially lowered log <i>D</i> (= 1.3), and this confers increased metabolic stability in vitro, together with high free levels. In each case, however, potency against MTH1 was compromised, as was permeability. The regioisomeric amides <b>32</b> and <b>33</b> were also synthesized, and the former at least demonstrated potent activity. Despite a log <i>D</i> of 1.8 for this compound, clearance remained very high, and indeed this compound proved to be unstable in human plasma. The tertiary amide analogue <b>33</b> had a similar profile but was substantially less active, more lipophilic, and similarly unstable in human plasma.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. MTH1 Potencies and in Vitro DMPK Profiles of Purinone Macrocycles<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0017.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last"><sup>1</sup>Inhibition of MTH1 enzyme was assessed by detecting the inorganic pyrophosphate generated when the nucleoside triphosphate substrate, 8-oxo-dGTP, is hydrolyzed. <sup>2</sup>log <i>D</i> at pH<sub>7.4</sub>. <sup>3</sup>In vitro clearance in human microsomes (μL/min/mg). <sup>4</sup>Percent free in human plasma. <sup>5</sup>A to B permeability in Caco2 cells at pH<sub>7.4</sub> (1 × 10<sup>–6</sup>·cm/s). <sup>6</sup>LLE = pIC<sub>50</sub> – log <i>D</i>. All activity data are reported in μM and are the mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units.</p></div></div><div></div></div><div class="NLM_p">The SAR seen for the macrocycles is likely to be closely linked to their ability to adopt the correct conformation. Modeling suggests that macrocycles with linkers of 6 and 7 atoms are able to mimic the conformation of uncyclized analogue <b>27</b> in the binding pocket (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c) to varying degrees. Linkers shorter than six atoms were considered, but their effect on conformation appeared more pronounced than with the longer linkers, and additionally, they proved less tractable chemically. Taking into account the difference in lipophilicity, the drop in potency between triether macrocycle <b>15</b> and amide-containing macrocycle <b>30</b> is approximately 30-fold. Conformational analysis of <b>30</b> showed that the preferred orientation of the phenyl of the amide macrocycle (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>d) was different from the phenyl orientation seen for <b>27</b> and <b>15</b>. However, a protein crystal structure of <b>30</b> (not shown) exhibits a phenyl orientation that is very similar to that of <b>27</b> and <b>15</b>, and we hypothesize that the energetic cost of inducing this optimal orientation of the phenyl negatively affects the potency of <b>30</b>. Such an effect is likely to play a role in the lower potency seen for <b>31</b> as well. The ligand-lipophilicity efficiency of alternative amide-containing macrocycle <b>32</b> and triether macrocycle <b>15</b> are similar, and comparison of their protein–ligand structures suggests that both are very similar in shape and conformation. The amide NH of <b>32</b> picks up an additional hydrogen bonding contact (not shown) to the backbone carbonyl of T8, but it is likely that the difference in potency between these compounds is largely caused by their difference in lipophilicity. Amide <b>33</b>, an <i>N</i>-methylated analogue of <b>32</b>, exhibits a markedly lower potency, which may be explained by abrogation of the amide hydrogen bond to T8 seen for <b>32</b> as well as conformational effects resulting from the <i>N</i>-methylation. The conformational hypothesis was corroborated by NMR experiments that showed unambiguously that conformations of the 6-linker compound <b>29</b> in solution closely resemble its modeled and protein-bound conformers (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c–e). NMR experiments on the nonmacrocyclic precursor <b>27</b> did not yield a conformational ensemble, suggesting that the unlinked chain is very flexible.</div><div class="NLM_p">In addition to the observations of cross-reactivity of TLR7 ligands with MTH1, the historical project also generated high-throughput screening data with the aim of identifying further chemical equity. Among the hits identified, a series of 3-amidoquinolines, exemplified by <b>19</b> and <b>34</b>, stood out as worthy of further profiling (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Both primary amide <b>34</b> and secondary amide <b>19</b> had high potency against MTH1 (12.5 and 0.9 nM, respectively), but the dimethyl analogue <b>35</b> showed a significant loss of potency, providing a closely related compound with significantly less activity (>130-fold lower activity of <b>35</b> vs <b>19</b>). The permeabilities of <b>19</b> and <b>35</b> as measured in a Caco-2 assay were high (<i>P</i><sub>app</sub> 24 × 10<sup>–6</sup> and 33 × 10<sup>–6</sup> cm/s, respectively) with no evidence of efflux (efflux ratios of 0.5 and 0.6, respectively), giving confidence that the molecules penetrate into cells. Further exploration of secondary amide substitution (data not shown) showed no benefits in terms of potency or LLE, with methyl judged as being optimal. Deletion of both methoxy groups on the core gave <b>36</b> and led to a small potency decrease but with LLE and properties remaining similar. Further truncation of the quinoline to the pyridyl <b>37</b> somewhat surprisingly retained potency and showed some benefits in terms of solubility, although this remained a challenge for the series. The high potency of this fragment shows the tolerance of MTH1 for binding a diverse range of structural motifs, provided key pharmacophoric elements are met, and was suggestive of a high degree of substrate promiscuity.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. MTH1 Potencies and in Vitro DMPK Profiles of 3-Amidoquinolines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0018.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">R1</th><th class="colsep0 rowsep0" align="center">R2</th><th class="colsep0 rowsep0" align="center">R3</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center" char=".">MTH1 IC<sub>50</sub><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">Cl<sub>int</sub> Hu Mics<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">Hu PPB (% free)<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char=".">Sol (μM)<a class="ref internalNav" href="#t3fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char=".">LLE<a class="ref internalNav" href="#t3fn6" aria-label="f">f</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">2-F, 4-Cl</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="char" char=".">0.0125</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">2-F, 4-Cl</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="char" char=".">0.0009</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">NMe<sub>2</sub></td><td class="colsep0 rowsep0" align="left">2-F, 4-Cl</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="char" char=".">1.22</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">0.002</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">0.009</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">8.9</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char=".">4.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="char" char=".">0.0005</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">9.1</td><td class="colsep0 rowsep0" align="char" char=".">108</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">2-(CH<sub>2</sub>)<sub>2</sub>SO<sub>2</sub>Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="char" char=".">0.0005</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="char" char=".">5.75</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">14.7</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>-4-<i>N</i>-Me-piperidine</td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="char" char=".">0.00008</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char="."> >1000</td><td class="colsep0 rowsep0" align="char" char=".">8.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Inhibition of MTH1 enzyme was assessed by detecting the inorganic pyrophosphate generated when the nucleoside triphosphate substrate, 8-oxo-dGTP, is hydrolyzed.</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">log <i>D</i> at pH<sub>7.4</sub>.</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last">In vitro clearance in human microsomes (μL/min/mg).</p></div><div class="footnote" id="t3fn4"><sup>d</sup><p class="last">Percent free in human plasma.</p></div><div class="footnote" id="t3fn5"><sup>e</sup><p class="last">Aqueous solubility at pH<sub>7.4</sub>.</p></div><div class="footnote" id="t3fn6"><sup>f</sup><p class="last">LLE = pIC<sub>50</sub> – log <i>D</i>. All activity data are reported in μM and are the mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units.</p></div></div></div><div class="NLM_p">The X-ray crystal structure of <b>19</b> in complex with MTH1 is shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 5">Figure 5</a>. The compound occupies the same site as the base of 8-oxo-dGMP. Compound <b>19</b> makes the typical hydrogen bonds with D119 and D120, and the methoxy groups sit among the hydrophobic residues. The side chain of N33 makes a hydrogen bond with the aniline nitrogen of <b>19</b>. The amide group makes an additional hydrogen bond with the backbone nitrogen of G34. A hit expansion library was carried out on lead <b>19</b> based on aniline substitution. Deletion of both halogens gave unsubstituted phenyl <b>21</b>, which was both more potent and less lipophilic, with improved LLE and physicochemical properties as a consequence. The permeability of <b>21</b> as measured in a Caco-2 assay was high (<i>P</i><sub>app</sub> 27 × 10<sup>–6</sup> cm/s) with no evidence of efflux (efflux ratio 0.4) that could limit any cellular activity. The SAR showed a clear and consistent preference for ortho > meta > para substitution (based on six matched triplets, F, Cl, Me, CN, SO<sub>2</sub>Me, and P(O)Me<sub>2</sub>, data not shown). Introduction of polar substituents in the ortho position, exemplified by ethyl sulphone <b>38</b>, allowed further lowering of log <i>D</i> while increasing LLE. Importantly, switching the aniline to an aryl ether gave <b>23</b>, which reduced activity against MTH1 by >1000-fold relative to <b>19</b>, consistent with removal of a key interaction seen in the X-ray crystal structure. In an attempt to improve solubility in this series, we investigated addition of basic functionality at the 6-position as this looked most likely to be tolerated from inspection of the X-ray crystal structure. Bases such as <b>39</b> further enhanced potency (80 pM) and improved LLE (8.4), giving highly soluble, sub-nM inhibitors of MTH1.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray crystal structures of human MTH1 in complex with <b>19</b> (green). Selected carbon atoms, protein backbone cartoon, and close polar contacts are represented in the complex color. Compounds are shown as sticks, and protein residue side chains are shown as thinner sticks. Close polar contacts are represented as dotted lines. The 2-chloro-4-fluoro-phenyl group adopts a dual conformation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A comparison of the rat pharmacokinetics of macrocycle <b>15</b>, in comparison to its uncyclized analogue <b>27</b>, is shown in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>. These data show a largely similar profile for the two compounds, with increased bioavailability seen for the macrocycle suggestive of better permeability. Clearances are moderate, with a low volume of distribution typical of uncharged molecules. The moderate clearance and low volume combine to give a relatively short half-life of around 1 h in both cases. It is clear that macrocyclization itself has had an overall neutral impact on the pharmacokinetic properties of this scaffold and a rather significant impact on potency at equivalent log <i>D</i>. The rat pharmacokinetics of quinoline <b>19</b> shows that the compound has low clearance and low volume, leading to a moderate half-life. Bioavailability of 37% is consistent with the reasonable solubility and high permeability of this compound as described above and is indicative of the potential of this series to develop tool compounds suitable for in vivo studies.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Rat Pharmacokinetics of Macrocycle <b>15</b>, Its Uncyclized Analogue <b>27</b>, and Quinoline Amide <b>19</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center" char=".">bioavailability (%)</th><th class="colsep0 rowsep0" align="center" char=".">clearance (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char=".">Vd<sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub>1/2</sub> (h)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Rat pharmacokinetics in Wistar Han rats followed a dose of 3 μMols/kg (po) and 1.5 μMols/kg (iv).</p></div></div></div><div class="NLM_p">A third series of inhibitors based on a 2-aminoquinazoline template was also identified from the HTS, with hit <b>40</b> showing modest potency against MTH1 (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). Dimethylation of the amine to give <b>41</b> resulted in reduced activity, while removal of the methyl (<b>42</b>) significantly increased potency against MTH1. A hit expansion library around <b>40</b> based on Suzuki couplings showed that <i>meta</i> and <i>para</i> substitution was favored over <i>ortho</i>. Transfering this SAR into the <i>des</i>-methyl series led to the identification of compounds such as dimethylamide <b>24</b> with enhanced MTH1 potency and acceptable physicochememical properties for further profiling. Permeability of <b>24</b> measured in a Caco-2 assay was high (<i>P</i><sub>app</sub> 34 × 10<sup>–6</sup> cm/s) with no significant efflux (efflux ratio 0.7) that would limit cellular penetration. The X-ray crystal structure of <b>24</b> in complex with MTH1 is shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 6">Figure 6</a>. The compound occupies the same site as the base of 8-oxo-dGMP. An overlay of the complex of <b>24</b> with <b>1</b> (<a class="ref internalNav" href="#fig7" aria-label="Figure 6">Figure 6</a>a) shows that the aminopyrimidine rings of the two compounds make identical typical hydrogen bonds with D119 and D120. The phenyl ring of the quinoline group sits in the same site as the cyclopropyl group among the hydrophobic residues of the nucleotide base binding pocket.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. MTH1 Potencies and in Vitro DMPK Profiles of 2-Aminoquinazolines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0019.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">R1</th><th class="colsep0 rowsep0" align="center">R2</th><th class="colsep0 rowsep0" align="center" char=".">MTH1 IC<sub>50</sub><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">Cl<sub>int</sub> Hu Mics<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">Hu PPB (% free)<a class="ref internalNav" href="#t5fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char=".">Sol (μM)<a class="ref internalNav" href="#t5fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char=".">LLE<a class="ref internalNav" href="#t5fn6" aria-label="f">f</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">3.32</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">215</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">NMe<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="char" char=".">120</td><td class="colsep0 rowsep0" align="char" char=".">0.1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">0.021</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">71</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="char" char=".">214</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>C(O)NMe<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">0.009</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="char" char=".">118</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Inhibition of MTH1 enzyme was assessed by detecting the inorganic pyrophosphate generated when the nucleoside triphosphate substrate, 8-oxo-dGTP, is hydrolyzed.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">log <i>D</i> at pH<sub>7.4</sub>.</p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">In vitro clearance in human microsomes (μL/min/mg).</p></div><div class="footnote" id="t5fn4"><sup>d</sup><p class="last">Percent free in human plasma.</p></div><div class="footnote" id="t5fn5"><sup>e</sup><p class="last">Aqueous solubility at pH<sub>7.4</sub>.</p></div><div class="footnote" id="t5fn6"><sup>f</sup><p class="last">LLE = pIC<sub>50</sub> – log <i>D</i>. All activity data are in μM and are the mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units.</p></div></div></div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) X-ray crystal structures of human MTH1 in complex with <b>24</b> (yellow). (b) Complexes with <b>24</b> and <b>1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U">4N1U</a>, cyan) overlaid. Selected carbon atoms, protein backbone cartoon, and close polar contacts are represented in the complex color. Compounds are shown as sticks, and protein residue side chains are shown as thinner sticks. Close polar contacts are represented as dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition to potent activity in an MTH1 enzyme assay, binding affinity for MTH1 protein was assessed independently by surface plasmon resonance (SPR) (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). This data confirmed the high affinities seen and correlated well with potencies as measured in the biochemical assay, giving further confidence in the data. Intriguingly, observed half-lives for dissociation from MTH1 varied significantly according to chemotype. <i>S</i>-Crizotinib <b>3</b> showed rapid dissociation (<i>t</i><sub>1/2</sub> measured in seconds), whereas <b>1</b> and the quinazoline/quinoline scaffolds described here showed moderate half-lives (of the order of minutes). The macrocyclic inhibitors, however, showed significantly extended residence time with <i>t</i><sub>1/2</sub> in excess of 1 h for both <b>15</b> and <b>28</b> in contrast with acyclic precursors like <b>27</b>, which dissociated rapidly (<i>t</i><sub>1/2</sub> in the range of seconds, beyond the resolution of SPR). On the basis of their high potencies and good permeability, examples from each series were assessed for their ability to penetrate cells and bind to MTH1 in a manner that was independent of a phenotypic end point. This is important because any phenotype which manifests itself predominantly as a cell killing effect as here may be particularly susceptible to any number of off-target effects that impact cell viability. For this, we utilized a whole cell thermal stabilization assay (CETSA)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> to assess target engagement in K562 cells, a technique that had also been employed for profiling of <b>1</b>. In this assay, cells are treated with inhibitor and subject to small increases over a temperature gradient before cells are lysed and protein quantified by Western blot. Any compound binding to its target within the cell will have a stabilizing effect on, and prevent the thermal denaturation of, the protein relative to untreated cells. These primary melt curves and ligand-induced shifts are established to aid selection of testing conditions. The compound is then tested in increasing concentration at a single temperature to establish the CETSA EC<sub>50</sub> of target engagement. As can be seen in <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>, the high enzymatic inhibitory activity of the macrocycles translated into very potent cellular target engagement with EC<sub>50</sub>s of 9 and 3 nM for <b>28</b> and <b>15</b>, respectively. These compared favorably with <b>1</b> for which in this assay we measure 10 nM (target engagement in this assay format is reported for this compound but potency is not disclosed). Nonmacrocyclic analogue <b>27</b> has a much weaker in EC<sub>50</sub> of 1.5 μM, in line with its much reduced MTH1 enzyme activity. A similar observation was made with the quinoline series, with <b>19</b> showing a potent CETSA EC<sub>50</sub> (7 nM) in marked contrast to dimethylated quinoline amide <b>35</b>, which was less active in both the enzyme assay (IC<sub>50</sub> 1200 nM) and CETSA (EC<sub>50</sub> 950 nM) assays. The aminoquinazoline <b>24</b> showed an intermediate CETSA EC<sub>50</sub> (80 nM) in accordance with the lower MTH1 enzyme activity. <a class="ref internalNav" href="#fig8" aria-label="Figure 7">Figure 7</a> shows the excellent correlation between MTH1 inhibitory potency and measured target engagement in K562 cells, across each of these described series, together with <b>1</b> and <b>3</b>. These data further support the conclusion that these diverse agents are potent, cell-active inhibitors of MTH1 function and highlight the value in pairing potent inhibitors with inactive or less active structurally related counterparts to act as negative controls in cell based experiments.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Primary Target Potencies, Lipophilicity, and Selected Tumor Cell Line Antiproliferative Effects of MTH1 Inhibitors</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">series</th><th class="colsep0 rowsep0" align="center" char=".">MTH1 IC<sub>50</sub><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">MTH1 <i>K</i><sub>d</sub><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">MTH1 <i>t</i><sub>1/2</sub> (min)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">CETSA EC<sub>50</sub><a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><a class="ref internalNav" href="#t6fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char=".">U2OS GI<sub>50</sub><a class="ref internalNav" href="#t6fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char=".">A549 GI<sub>50</sub><a class="ref internalNav" href="#t6fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char=".">H358 GI<sub>50</sub><a class="ref internalNav" href="#t6fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char=".">MCF7 GI<sub>50</sub><a class="ref internalNav" href="#t6fn6" aria-label="f">f</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">0.003</td><td class="colsep0 rowsep0" align="char" char=".">0.005</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char=".">0.010</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">0.138</td><td class="colsep0 rowsep0" align="char" char=".">0.039</td><td class="colsep0 rowsep0" align="char" char=".">0.1</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">TLR7</td><td class="colsep0 rowsep0" align="char" char=".">0.536</td><td class="colsep0 rowsep0" align="char" char=".">0.440</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">macrocycle</td><td class="colsep0 rowsep0" align="char" char=".">0.0003</td><td class="colsep0 rowsep0" align="char" char=".">0.0002</td><td class="colsep0 rowsep0" align="char" char=".">151.7</td><td class="colsep0 rowsep0" align="char" char=".">0.009</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">12.1</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">8.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">macrocycle</td><td class="colsep0 rowsep0" align="char" char=".">0.0005</td><td class="colsep0 rowsep0" align="char" char=".">0.0004</td><td class="colsep0 rowsep0" align="char" char=".">81.2</td><td class="colsep0 rowsep0" align="char" char=".">0.003</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">quinoline</td><td class="colsep0 rowsep0" align="char" char=".">0.0009</td><td class="colsep0 rowsep0" align="char" char=".">0.0005</td><td class="colsep0 rowsep0" align="char" char=".">8.7</td><td class="colsep0 rowsep0" align="char" char=".">0.007</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">12.0</td><td class="colsep0 rowsep0" align="char" char=".">13.9</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="char" char=".">14.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">quinoline</td><td class="colsep0 rowsep0" align="char" char=".">0.0006</td><td class="colsep0 rowsep0" align="char" char=".">0.0002</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char=".">NT</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">quinoline</td><td class="colsep0 rowsep0" align="char" char=".">1.22</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">0.95</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">aminoquinazoline</td><td class="colsep0 rowsep0" align="char" char=".">0.009</td><td class="colsep0 rowsep0" align="char" char=".">0.003</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">0.080</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">Inhibition of MTH1 enzyme was assessed by detecting the inorganic pyrophosphate generated when the nucleoside triphosphate substrate, 8-oxo-dGTP, is hydrolyzed.</p></div><div class="footnote" id="t6fn2"><sup>b</sup><p class="last">MTH1 affinity and halflife for dissociation as measured by surface plasmon resonance.</p></div><div class="footnote" id="t6fn3"><sup>c</sup><p class="last">Whole cell thermal stability assay in K562 cells heated at 52 °C for 3 min.</p></div><div class="footnote" id="t6fn4"><sup>d</sup><p class="last">log <i>D</i> at pH<sub>7.4</sub>.</p></div><div class="footnote" id="t6fn5"><sup>e</sup><p class="last">Growth inhibition in the indicated cell line following 5 day compound incubation.</p></div><div class="footnote" id="t6fn6"><sup>f</sup><p class="last">Growth inhibition in the indicated cell line following 7 day compound incubation. All activity data are in μM and are the mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units.</p></div></div></div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Correlation between MTH1 inhibitory potencies and target engagement in K562 cells colored by series. Straight line is line of fit.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further characterize the utility of these novel inhibitors, and to ensure any phenotype would not be confounded by off-target kinase inhibition, we profiled representative compounds <b>15</b>, <b>19</b>, <b>21</b>, and <b>24</b> against an extended panel of kinases. Given the heritage of anilinoquinolines as kinase inhibitors, it was surprising to find that <b>15</b> did not show appreciable kinase inhibition at all (<35% at 10 μM across 267 kinases). Profiling of <b>19</b>, <b>21</b>, and <b>24</b> suggested no activity against any kinase at a testing concentration of 1 μM. In addition, these compounds were profiled in a diverse panel of 153 secondary pharmacology targets and no notable off-target activities were observed that might confound phenotypic studies (these data are available as <a href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p">A selection of compounds was assessed for their ability to impact growth on a small panel of tumor cell lines (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). Consistent with published data, <b>1</b> and <b>3</b> displayed modest but consistent growth inhibition in the range 3–5 μM, even though in our hands <b>3</b> exhibits relatively weak cellular target engagement. Macrocycle <b>28</b> demonstrated much weaker activity in this panel but did have an effect on viability at the top concentrations tested. Critically, however, macrocycle <b>15</b>, which is significantly less lipophilic than <b>28</b>, and has equivalent MTH1 potency, had no impact on cell viability. In a similar fashion, initial lipophilic quinoline screening hit <b>19</b> had modest antiproliferative effects. However, lowering its lipophilicity, as in example <b>21</b>, resulted in a compound which had no such impact on cell viability for equivalent potency. These assays used an extended 5- and 7-day compound incubation, although no further changes in potency were observed when compared to more typical 3-day incubation. We conclude that the modest effects observed with <b>19</b> and <b>28</b> are due to their increased lipophilicity and represent off-target, nonspecific, and MTH1-independent effects on cell growth. Analogue <b>15</b>, in a more acceptable lipophilicity range, shows potent engagement with MTH1 in cells and yet has no effect on cell viability (analogue <b>21</b> similarly has no impact on viability). These observations were also seen across much larger cell panels. Macrocycle <b>28</b>, despite showing some activity at high concentration in this mini-panel, was largely inactive in a wider panel of 212 tumor cell lines representing a diverse array of primary cancers. From this panel, 182 of 212 lines showed no impact on growth at all, and of those where a GI<sub>50</sub> was determined, the highest potency was 6.74 μM, nearly 750-fold weaker than its measured cellular target engagement of MTH1. Quinoline <b>21</b> was tested in a panel of 135 cell lines and was inactive in 134 of them (GI<sub>50</sub> > 30 μM), showing weak activity (GI<sub>50</sub> = 27.16 μM) in a single cell line. However, consistent with the effects reported in the original disclosures, in a panel of 210 cell lines, <b>1</b> displayed broad antiproliferative activity consistent with a general cytotoxicity: moderately potent effects on cell viability across nearly all lines tested. Of the 211 cell lines assayed, only 10 showed no activity, with a mean GI<sub>50</sub> of 6.5 μM across the remaining 201 lines (range 1.5–26.4 μM). Testing of <b>3</b> gave a largely similar profile.</div><div class="NLM_p">In addition to limited effects on cell viability, cellular responses were assessed for <b>19</b> (at a concentration of 10 μM) through Western blotting of DNA damage response (DDR), p53, and apoptotic markers after 24, 48, or 72 h continual exposure. Compound <b>19</b> did not show increases in any of the DDR signaling markers tested (p-Ser1981 ATM, p-Ser15 p53, γH2AX, pSer4/8 RPA), nor did we observe induction of caspase-3 mediated apoptosis (cleaved PARP1) in the manner reported for <b>1</b> and <b>2</b> (<a class="ref internalNav" href="#fig9" aria-label="Figure 8">Figure 8</a>). Consistent with the literature data, we did observe increased p-Ser15 p53, p-Ser1981 ATM (abeit weakly), and caspase-mediated apoptosis induction (as measured by appearance of cleaved PARP1) upon treatment of U2OS cells with high concentrations of <b>1</b> (10 μM). However, we did not detect increases in other typical DNA double-strand break signaling markers such as γH2AX or pSer4/8 RPA and, somewhat confusingly, we observed robust apoptotic induction at time points (24 h) that preceded the appearance of maximal p-Ser15 p53 or p-Ser1981 ATM induction (48 h). Although the mechanisms of apoptotic induction/cell death with <b>1</b> are unclear, the difference in DDR and apoptosis induction between compounds suggest that inhibition of MTH1 may not be sufficient to induce a meaningful DNA damage response.</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0009.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Induction of DDR signaling markers and apopotosis following treatment of U2OS cells for 24, 48, or 72 h with 10 μM <b>1</b> or <b>19</b>, compared to DMSO vehicle control (Con). Treatment with <b>1</b> leads to increased p-Ser1981 ATM, p-Ser15 p53 expression, and cleaved PARP1 (apoptosis). <b>19</b> shows no impact on DDR signaling activation or apoptosis. No changes in γH2AX or pSer4/8 RPA DDR signalng were observed for either compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having observed key differences in effects on cell viability and DDR signaling for chemically diverse series of MTH1 inhibitors, we sought to revisit the genetic knockdown validation experiments disclosed in the original papers. Using a commercially available oligonucleotide targeting MTH1 that had a distinct sequence from those reported, we were able to induce complete silencing of MTH1, and yet this cell line was entirely viable, in stark contrast to the reported validation data (<a class="ref internalNav" href="#fig10" aria-label="Figure 9">Figure 9</a>a,b). Growth of this U2OS cell line was identical to that observed from one using a nontargeted siRNA and indicates that loss of MTH1 function is insufficient to affect cell viability. The original work detailed genetic ablation of protein using a sequence termed ‘oligo #3′. We obtained this siRNA and repeated these experiments. In agreement with the data reported, using oligo #3 resulted in complete knockdown of MTH1 and a profound impact on cell viability. The observation that different siRNA can lead to knockdown of MTH1 protein expression but that only one leads to cell death is suggestive of off-target, nonspecific effects of the published siRNA. In our experience, such off-target liabilities of siRNA techniques, although well established,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> are not widely appreciated, and caution should be exercised when interpreting such results, particularly for end-points focused on cell death. In addition to siRNA knockdown experiments, we also generated CRISPR-mediated MTH1 knockout SW480 cell lines (<a class="ref internalNav" href="#fig10" aria-label="Figure 9">Figure 9</a>c,d). We were able to isolate multiple, stable cell clones that completely lack MTH1 expression. One such clone is shown in <a class="ref internalNav" href="#fig10" aria-label="Figure 9">Figure 9</a>: this clone grows as efficiently as its wild-type counterpart, further demonstrating MTH1 expression is not essential for cancer cell line viability.</div><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0010.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (a) Treatment of U2OS cells with siRNA to MTH1. Incubation for 2 and 7 days with both AZ and literature (#3) siRNA leads to significant knockdown of MTH1 protein relative to nontargeted (NT) controls, but no impact on other markers of DNA-damage response (DDR) activation as measured by <i>p</i>-Ser1981 ATM and <i>p</i>-Ser15 p53 expression. (b) MTH1 knockdown results in a significant reduction in clonogenic cell survival for literature (#3) siRNA but not a distinct AZ siRNA compared to control transfected cells. (c) Isogenic gene knockouts of MTH1 using CRISPR technology in SW480 cells. (d) Clones with complete knockout of all MTH1 alleles had undetectable levels of protein expression and were readily recoverable. Clones with complete knockout of MTH1 were not impaired in growth rate compared to SW480 parental cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Finally, in a further extension of the original validation work, we treated cell lines in which MTH1 had been knocked down with MTH1 inhibitors (<a class="ref internalNav" href="#fig11" aria-label="Figure 10">Figure 10</a>). Critically, <b>1</b> and <b>3</b> were each observed to kill cells that were MTH1-null, with potencies in line with those reported for endogenous tumor lines and with potencies that matched those lines treated with scrambled siRNA. Compound <b>19</b> similarly showed no separation between these different genetic backgrounds. These experiments, which were not part of the original validation, indicate that the compounds cause cell death in an MTH1-independent manner, likely due to activity against unknown off-target mechanisms or general cytotoxicity. Consistent with this, the original validation experiments do highlight that MTH1 overexpression does not alter response to <b>3</b> and also that expression of the bacterial homologue of MTH1, MutT, could only partially rescue cells from the cytotoxic effects of <b>1</b> and <b>2</b>.</div><figure id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0011.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. MTH1 inhibitor cell viability is independent of MTH1. (a) Clonogenic survival of U2OS cells treated with <b>1</b>, <b>3</b>, or <b>19</b> following nontargeting control siRNA (NT), AZ MTH1 siRNA, or Gad et al. MTH1 siRNA #3. Values represent mean % of colonies forming ± SD in comparison to nontargeting control siRNA treated cells from at least three independent experiments unless otherwise stated. (b) Viability of SW480 cell growth in parental and SW480 MTH1<sup>–/–/–</sup> clone three cells treated with <b>1</b>, <b>3</b>, or <b>19</b>. Values represent mean % viability (CellTiter-Glo) relative to DMSO control-treated cells from at least fi veindependent experiments ± SD. Table: calculated GI<sub>50</sub> values in μM for <b>1</b>, <b>3</b>, and <b>19</b> for experiments shown in (a) and (b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33311" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33311" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have developed three distinct chemical series of MTH1 inhibitors to complement those recently described and to further probe the claimed role of this interesting enzyme. Critical to this was the knowledge this target had been tackled previously by AstraZeneca over a decade earlier. Although it may seem obvious, such continuity of information within a large organization should not automatically be assumed, not least because the original discoveries took place within a different disease-focused department, on a different site now no longer part of the AstraZeneca family, with data held on local drives in a nonsearchable format. In this instance, we relied on a small number of scientists who conducted the original programs, still present, to highlight knowledge that might easily have been lost or overlooked. In the work described herein, probes were developed through the strategies of (i) increasing potency of a weak hit through conformational restriction without increasing lipophilicity (<b>27</b> to <b>15</b>), (ii) reducing the lipophilicity of a potent inhibitor through atom deletion (<b>19</b> to <b>21</b>), or (iii) increasing potency while reducing lipophilicity through inclusion of polar functionality (<b>40</b> to <b>24</b>) as highlighted in <a class="ref internalNav" href="#fig12" aria-label="Figure 11">Figure 11</a>. It is not clear what the true target(s) of the published MTH1 inhibitors are that lead to the observed broad cell-killing effects, although it is possible that the mechanisms by which <b>1</b> and <b>3</b> achieve this are entirely different. High lipophilicity leading to general promiscuity is well documented<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and could be one such factor for <b>1</b>. Being the enantiomer of a marketed drug, <b>3</b> is in an acceptable property range, but its weaker activity, combined with its potential for additional off-target kinase activities, may contribute to the observed antiproliferative activity. Recent work highlights the value of tool compounds but also the associated pitfalls in choosing poorly characterized or promiscuous compounds with which to validate biology.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The use of a photoaffinity probe based on <b>1</b>, in much the same way as is described here for TLR7 agonists, may be one way to dissect out the targets responsible for the antiproliferative activity in this series.</div><figure id="fig12" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0012.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Plot of log <i>D</i> versus MTH1 enzyme potency for MTH1 inhibitors. Points are colored by chemical series, and arrows indicate the direction of optimization from screening hit to cell probe compound described in this study. The highlighted box is indicative of a potency and lipophilicity range deemed desirable for utility in probing target biology within cells in order to ensure potent and specific target modulation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">MTH1 inhibition has been proposed to “eradicate cancer” through a novel nononcogene addiction by targeting its function in hydrolyzing oxidized nucleotides, preventing their incorporation into DNA during replication and suppressing accumulation of otherwise lethal levels of DNA damage. However, it is not known at what level these oxidized nucleotides are expected to be lethal to cells (possibly due to an absence of reliable measures to quantify cellular levels of free oxidized nucleotides), therefore direct evidence linking MTH1 function and the consequences of their incorporation into the DNA in cancer cells remains to be understood. Indeed, deleting the bacterial MTH1 homologue or mouse MTH1 does not lead to cellular lethality despite the accumulation of thousands of mutations.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> As the Nudix family of enzymes to which MTH1 belongs comprises more than 20 genes and other DDR pathways (e.g., base-excision repair) exist which repair oxidized bases in DNA, it is possible that redundancy may allow for cellular survival, a hypothesis that remains to be tested. In summary, our data using selective small molecule inhibitors, siRNA knock-down, and CRISPR-mediated knockout of MTH1 does not support the claimed role of MTH1 being essential for cancer-cell survival.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03051" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03051" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry</h3><div class="NLM_p">Unless otherwise stated, commercially available reagents were used as supplied. All reactions requiring anhydrous conditions were conducted in dried apparatus under an atmosphere of nitrogen. <sup>1</sup>H NMR spectra were recorded using a Bruker AV300 or AV400 NMR. Chemical shifts δ are reported in ppm, and multiplicity of signals are denoted s = singlet, d = doublet, t = triplet and m = multiplet, respectively. HRMS were recorded using a Shimadzu LCMS-2020 instrument (ESI+). Reactions and intermediates were also characterized by mass spectroscopy following liquid chromatography (LCMS or UPLC); purity of tested compounds was assessed to be at least 95% by UV using LCMS or UPLC analysis unless indicated otherwise. UPLC was carried out using a Waters UPLC fitted with Waters QDa mass spectrometer (column temp 40, UV = 190–400 nm, MS = ESI with pos/neg switching) at a flow rate of 1 mL/min using a solvent system of 95% A + 5% B to 5% A to 95% B over 8 min (total runtime with equilibration back to starting conditions was 10 min), where A = 0.1% formic acid or 0.05% TFA in water (for acid work) or 0.04% ammonia in water (for base work) B = acetonitrile. For acid or base analysis, the column used was Waters Acquity BEH 1.7 μm, 2.1 mm × 100 mm. LCMS was carried out using a Shimadzu UFLC fitted with a Shimadzu LCMS-2020 mass spectrometer and a Waters BEH C18 (50 mm × 2.1 mm, 1.7 μm) or Shim-pack XR-ODS (50 mm × 3.0 mm, 2.2 μm) or Phenomenex Gemini −NX 3μ C18 110A (50 mm × 3.0 mm, 3 μm) column at a flow rate of 1.2 mL/min 95% A to 95% B over 2.0 min with a 0.6 min hold. Ion exchange purification was generally performed using a SCX-2 (Biotage, propylsulfonic acid functionalized silica, manufactured using a trifunctional silane, non-end-capped) cartridge. Column chromatography was performed on a Biotage system using a SiO<sub>2</sub> column with UV detection. Individual purification methods referred to here are detailed in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 6-Chloro-2-(methylthio)-5-nitropyrimidin-4-amine (<b>7</b>)</h4><div class="NLM_p last">Ammonia in 2-propanol (27.1 mL, 54.2 mmol) was added dropwise to 4,6-dichloro-2-(methylthio)-5-nitropyrimidine <b>6</b> (10 g, 41.7 mmol) and triethylamine (8.7 mL, 62.5 mmol) in THF (200 mL) at −20 °C under nitrogen. The resulting mixture was stirred below 0 °C for 5 h. The solvent was removed under reduced pressure and the reaction mixture diluted with water (200 mL) and extracted with ethyl acetate (3 × 200 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford 6-chloro-2-(methylthio)-5-nitropyrimidin-4-amine (9.1 g, 99%) as a yellow solid. The product was used in the next step directly without further purification. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz) δ 2.48 (3H, s), 8.11 (2H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 221; acidic, HPLC <i>t</i><sub>R</sub> = 1.023 min.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Ethyl (6-Chloro-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate (<b>8</b>)</h4><div class="NLM_p last">Ethyl carbonochloridate (4.7 g, 43.1 mmol) was added to 6-chloro-2-(methylthio)-5-nitropyrimidin-4-amine <b>7</b> (9.1 g, 41 mmol) and triethylamine (11.4 mL, 82 mmol) in THF (200 mL) at 0 °C under nitrogen. The resulting mixture was stirred at room temperature for 14 h. The reaction mixture was quenched with water (50 mL), extracted with ethyl acetate (3 × 50 mL), and the organic layer dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford a yellow solid. The crude product was purified by flash silica chromatography, elution gradient 0–15% ethyl acetate in petroleum ether. Pure fractions were evaporated to dryness to afford ethyl (6-chloro-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate (9.2 g, 77%) as a pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz) δ 1.21 (3H, t), 2.56 (3H, s), 4.14 (2H, q), 11.51 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 293; acidic, HPLC <i>t</i><sub>R</sub> = 1.129 min.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 3-((<i>tert</i>-Butyldimethylsilyl)oxy)benzaldehyde</h4><div class="NLM_p last"><i>tert</i>-Butylchlorodimethylsilane (9.3 g, 61.4 mmol) was added portionwise to 3-hydroxybenzaldehyde (5 g, 40.9 mmol), triethylamine (11.4 mL, 81.9 mmol), and DMAP (0.25 g, 2.1 mmol) in DCM (100 mL) at room temperature. The resulting mixture was stirred at room temperature for 2 h. The reaction mixture was diluted with DCM (500 mL) and washed sequentially with saturated aqueous ammonium chloride (150 mL × 2) and saturated brine (200 mL × 2). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzaldehyde (11 g) as a pale-yellow oil. The product was used in the next step directly without further purification. <sup>1</sup>H NMR (chloroform-<i>d</i>, 300 MHz) δ 0.24 (6H, s), 1.01 (9H, s), 7.08–7.12 (1H, m), 7.30–7.54 (3H, m), 9.96 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + MeCN]<sup>+</sup> = 278; acidic, HPLC <i>t</i><sub>R</sub> = 1.835 min.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (3-((<i>tert</i>-Butyldimethylsilyl)oxy)phenyl)methanol</h4><div class="NLM_p last">Sodium borohydride (0.77 g, 20.2 mmol) was added portionwise to 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzaldehyde (11 g, 40.5 mmol) in methanol (100 mL) at 0 °C. The resulting mixture was stirred at 0 °C for 15 min. The solvent was removed under reduced pressure. The crude product was purified by flash silica chromatography, elution gradient 0–20% ethyl acetate in petroleum ether. Pure fractions were evaporated to dryness to afford (3-((<i>tert</i>-butyldimethylsilyl)oxy)phenyl)methanol (9.80 g) as a colorless oil which was used without further purification. <sup>1</sup>H NMR (chloroform-<i>d</i>, 300 MHz) δ 0.21 (6H, s), 0.96 (9H, s), 1.63–1.64 (1H, m) 4.65 (2H, s), 6.78 (1H, ddd), 6.87 (1H, s), 6.94 (1H, d), 7.22 (1H, dd); <i>m</i>/<i>z</i> (ES+), [M + H – H<sub>2</sub>O]<sup>+</sup> = 221; acidic, HPLC <i>t</i><sub>R</sub> = 1.297 min.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>tert</i>-Butyl(3-(chloromethyl)phenoxy)dimethylsilane</h4><div class="NLM_p last">Thionyl chloride (4.6 mL, 62.9 mmol) was added to (3-((<i>tert</i>-butyldimethylsilyl)oxy)phenyl)methanol (10 g, 42 mmol) in DCM (150 mL) at 0 °C under nitrogen. The resulting mixture was stirred at room temperature for 5 h. The reaction mixture was quenched with ice water (100 mL) and extracted with DCM (2 × 100 mL). The organic layer was washed sequentially with saturated NaHCO<sub>3</sub> (100 mL × 3) and saturated brine (100 mL × 2), and the organic layer dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford <i>tert</i>-butyl(3-(chloromethyl)phenoxy)dimethylsilane (10 g) as a colorless liquid. <sup>1</sup>H NMR (chloroform-<i>d</i>, 300 MHz) δ 0.21 (6H, s), 0.99 (9H, s), 4.53 (2H, s), 6.78 (1H, dd), 6.80 (1H, t), 6.97 (1H, dt), 7.21 (1H, t); <i>m</i>/<i>z</i> (ES+), [M+] = 256, GCMS <i>t</i><sub>R</sub> = 8.350 min.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Ethyl 3-((<i>tert</i>-Butyldimethylsilyl)oxy)benzyl(6-chloro-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate (<b>9</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl(3-(chloromethyl)phenoxy)dimethylsilane (1.93 g, 7.5 mmol) was added to ethyl (6-chloro-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate <b>8</b> (2 g, 6.8 mmol), sodium iodide (1 g, 6.8 mmol), and K<sub>2</sub>CO<sub>3</sub> (1.42 g, 10.3 mmol) in acetone (40 mL) at 20 °C. The resulting mixture was stirred at room temperature for 16 h. The solid was filtered, and the solvent was removed under reduced pressure to afford crude ethyl 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzyl(6-chloro-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate (4 g, >100%) as a pale-yellow gum. The product was used in the next step directly without further purification; <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 513; acidic, HPLC <i>t</i><sub>R</sub> = 1.616 min.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Ethyl 3-((<i>tert</i>-Butyldimethylsilyl)oxy)benzyl(6-(methylamino)-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate (<b>10</b>)</h4><div class="NLM_p last">Methylamine (6.6 mL, 13.3 mmol) was added to ethyl 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzyl(6-chloro-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate <b>9</b> (4 g, 6.6 mmol) in THF (50 mL) at room temperature under nitrogen. The resulting mixture was stirred at room temperature for 16 h. The solvent was removed under reduced pressure, and the crude product was purified by flash silica chromatography, elution gradient 0–5% ethyl acetate in petroleum ether. Pure fractions were evaporated to dryness to afford ethyl 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzyl(6-(methylamino)-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate (3.1 g, 92%) as a yellow gum. <sup>1</sup>H NMR (chloroform-<i>d</i>, 300 MHz) δ 0.17 (6H, s), 0.96 (9H, s), 1.23 (3H, t), 2.47 (3H, s), 3.12 (3H, d), 4.15 (2H, q), 5.17 (2H, s), 6.71 (1H, dd), 6.91 (1H, d), 6.99 (1H, d), 7.15 (1H, t), 8.10 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]+ = 508; acidic, HPLC <i>t</i><sub>R</sub> = 1.595 min.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Ethyl 3-((<i>tert</i>-Butyldimethylsilyl)oxy)benzyl(6-(methylamino)-2-(methylsulfonyl)-5-nitropyrimidin-4-yl)carbamate (<b>11</b>)</h4><div class="NLM_p last"><i>m</i>CPBA (2.55 g, 14.8 mmol) was added to ethyl 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzyl(6-(methylamino)-2-(methylthio)-5-nitropyrimidin-4-yl)carbamate <b>10</b> (3 g, 5.9 mmol) in DCM (40 mL) at room temperature. The resulting mixture was stirred at room temperature for 2 h. The reaction mixture was diluted with DCM (100 mL) and washed sequentially with saturated aqueous NaHCO<sub>3</sub> (50 mL × 3) and saturated brine (50 mL × 2). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford ethyl 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzyl(6-(methylamino)-2-(methylsulfonyl)-5-nitropyrimidin-4-yl)carbamate (3.3 g, >100%) as a yellow gum. The product was used in the next step directly without further purification. <sup>1</sup>H NMR (chloroform-<i>d</i>, 300 MHz) δ 0.16 (6H, s), 0.95 (9H, s), 1.22 (3H, t), 3.14 (3H, s), 3.22 (3H, d), 4.21 (2H, q), 5.22 (2H, s), 6.71 (1H, ddd), 6.88 (1H, t), 6.96 (1H, d), 7.15 (1H, t), 7.93–8.10 (1H, m); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 540; acidic, HPLC <i>t</i><sub>R</sub> = 1.398 min.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Ethyl 3-((<i>tert</i>-Butyldimethylsilyl)oxy)benzyl(2-(2-(2-hydroxyethoxy)ethoxy)-6-(methylamino)-5-nitropyrimidin-4-yl)carbamate (<b>12</b>)</h4><div class="NLM_p last">Sodium hydride (0.73 g, 18.3 mmol) was added to 2,2′-oxidiethanol (3.89 g, 36.7 mmol) in THF (80 mL) at room temperature under nitrogen. The resulting mixture was stirred at room temperature for 1 h. Ethyl 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzyl(6-(methylamino)-2-(methylsulfonyl)-5-nitropyrimidin-4-yl)carbamate <b>11</b> (3.3 g, 6.1 mmol) was added to the mixture, and the mixture was stirred at room temperature for 16 h. The reaction was quenched with saturated aqueous ammonium chloride (2 mL). The solvent was removed under reduced pressure and the crude product purified by flash silica chromatography, elution gradient 0–5% methanol in DCM. Pure fractions were evaporated to dryness to afford ethyl 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzyl(2-(2-(2-hydroxyethoxy)ethoxy)-6-(methylamino)-5-nitropyrimidin-4-yl)carbamate (2.4 g, 69%) as a yellow gum. <sup>1</sup>H NMR (chloroform-<i>d</i>, 300 MHz) δ 0.16 (6H, s), 0.96 (9H, s), 1.19 (3H, t), 3.05–3.23 (3H, m), 3.56–3.68 (2H, m), 3.73–3.79 (4H, m), 4.17 (2H, q), 4.39–4.51 (2H, m), 5.17 (2H, s), 6.66–6.76 (1H, m), 6.87 (1H, m), 7.00 (1H, m), 7.14 (1H, td), 8.19 (1H, s), 1× OH not observed; <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 566; acidic, HPLC <i>t</i><sub>R</sub> = 1.365 min.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Ethyl 3-Hydroxybenzyl(2-(2-(2-hydroxyethoxy)ethoxy)-6-(methylamino)-5-nitropyrimidin-4-yl)carbamate (<b>13</b>)</h4><div class="NLM_p last">TBAF in THF (5.1 mL, 5.1 mmol) was added to ethyl 3-((<i>tert</i>-butyldimethylsilyl)oxy)benzyl(2-(2-(2-hydroxyethoxy)ethoxy)-6-(methylamino)-5-nitropyrimidin-4-yl)carbamate <b>12</b> (2.4 g, 4.2 mmol) in THF (40 mL) at room temperature under nitrogen. The resulting solution was stirred at room temperature for 16 h. The reaction mixture was diluted with ethyl acetate (200 mL) and washed with saturated brine (50 mL × 4). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product which was purified by flash C18-flash chromatography, elution gradient 0–60% methanol in water. Pure fractions were evaporated to dryness to afford ethyl 3-hydroxybenzyl(2-(2-(2-hydroxyethoxy)ethoxy)-6-(methylamino)-5-nitropyrimidin-4-yl)carbamate (1.64 g, 86%) as a yellow gum. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 1.08 (3H, t), 2.97 (3H, d), 3.18 (1H, s), 3.43–3.50 (4H, m), 3.68 (2H, t), 4.04 (2H, q), 4.39 (2H, t), 5.01 (2H, s), 6.55–6.66 (1H, m), 6.72–6.81 (2H, m), 7.09 (1H, t), 8.62 (1H, broad d), 9.34 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 452; acidic, HPLC <i>t</i><sub>R</sub> = 1.299 min.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Ethyl 18-(Methylamino)-19-nitro-9,12,15-trioxa-2,17,20-triazatricyclo[14.3.1.1–4,8∼]henicosa-1(20),4(21),5,7,16,18-hexaene-2-carboxylate (<b>14</b>)</h4><div class="NLM_p last">A solution of (<i>E</i>)-di-<i>tert</i>-butyl diazene-1,2-dicarboxylate (1.63 g, 7.1 mmol) in THF (20 mL) was added dropwise to a stirred solution of ethyl 3-hydroxybenzyl(2-(2-(2-hydroxyethoxy)ethoxy)-6-(methylamino)-5-nitropyrimidin-4-yl)carbamate <b>13</b> (1.6 g, 3.5 mmol) and triphenylphosphine (1.86 g, 7.1 mmol) in THF (120 mL) at room temperature under nitrogen. The resulting solution was stirred at room temperature for 20 h. The reaction mixture was quenched with water (2 mL). The solvent was removed under reduced pressure and the crude product purified by flash silica chromatography, elution gradient 0–100% DCM in petroleum ether. Pure fractions were evaporated to dryness to afford <b>14</b> (1.6 g, > 100%) as a yellow oil which solidified on standing and was used in the next step without further purification; <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 434; acidic, HPLC <i>t</i><sub>R</sub> = 1.136 min.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 20-(Methylamino)-4,5,7,8-tetrahydro-15<i>H</i>-2,19-(azenometheno)-14,10-(metheno)imidazo[5,1-<i>d</i>][1,12,15,3,5]trioxadiazacycloheptadecin-17(18<i>H</i>)-one (<b>15</b>)</h4><div class="NLM_p last">Iron (2.22 g, 39.7 mmol) was added to <b>14</b> (1.6 g, 4 mmol) in acetic acid (16 mL) and water (2 mL) at room temperature. The resulting mixture was stirred at 60 °C for 4 h. The reaction mixture was filtered through Celite and the solvent removed under reduced pressure. The crude product was purified by flash C18-flash chromatography, elution gradient 0–50% acetonitrile in water (containing 0.1% ammonia). Pure fractions were evaporated to dryness to afford <b>15</b> as a pale-yellow solid. This solid was purified by preparative HPLC (XSelect CSH Prep C18 OBD column, 5 μ silica, 19 mm diameter, 150 mm length), using decreasingly polar mixtures of water (containing 0.1% NH<sub>4</sub>HCO<sub>3</sub>) and acetonitrile as eluents. Fractions containing the desired compound were evaporated to dryness to afford <b>15</b> (0.1 g, 7%) as a white solid. <sup>1</sup>H NMR (700 MHz, DMSO) 2.88 (3H, d, <i>J</i> = 4.7 Hz), 3.58–3.60 (2H, m), 3.61–3.64 (2H, m), 4.01–4.05 (2H, m), 4.65–4.69 (2H, m), 4.82 (2H, s), 6.47–6.50 (1H, m), 6.77 (1H, dd, <i>J</i> = 2.1, 8.0 Hz), 6.95 (1H, d, <i>J</i> = 7.5 Hz), 7.16 (1H, t, <i>J</i> = 7.8 Hz), 7.74 (1H, 1H), 9.87 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 358; base, HPLC <i>t</i><sub>R</sub> = 3.20 min. HRMS ESI+ <i>m</i>/<i>z</i> observed 358.1499, C<sub>17</sub>H<sub>20</sub>N<sub>5</sub>O<sub>4</sub> requires 358.1515</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Ethyl 4-Chloro-6,7-dimethoxyquinoline-3-carboxylate (<b>17</b>)</h4><div class="NLM_p last">Phosphoryl trichloride (150 mL, 59.9 mmol) was added to ethyl 4-hydroxy-6,7-dimethoxyquinoline-3-carboxylate <b>16</b> (16.6 g, 59.9 mmol). The resulting mixture was stirred at 100 °C for 1 h. The solvent was removed under reduced pressure, and the reaction mixture was adjusted to pH = 8 with saturated NaHCO<sub>3</sub> with ice bath cooling. The reaction mixture was filtered through Celite and the solid filtered then dried in an oven under reduced pressure to afford <b>17</b> (16 g, 90%) as a gray solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz) δ 1.38 (3H, t), 3.97 (6H, s), 4.40 (2H, q), 7.50 (1H, s), 7.53 (1H, s), 8.97 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 296; acid, HPLC <i>t</i><sub>R</sub> = 1.115 min.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Ethyl 4-((4-Chloro-2-fluorophenyl)amino)-6,7-dimethoxyquinoline-3-carboxylate (<b>18</b>)</h4><div class="NLM_p last">Acetic acid (3.1 mL, 54.1 mmol) was added to ethyl 4-chloro-6,7-dimethoxyquinoline-3-carboxylate <b>17</b> (4 g, 13.5 mmol) and 4-chloro-2-fluoroaniline (2.363 g, 16.23 mmol) in 2-propanol (60 mL) at room temperature under nitrogen. The resulting mixture was stirred at 100 °C for 16 h. The solvent was removed under reduced pressure, and the crude residue was triturated with ethyl acetate/diethyl ether to give a solid which was collected by filtration and dried under vacuum to give <b>18</b> (5.4 g, 99%) as a gray solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 300 MHz) δ 1.23 (3H, t), 3.63 (3H, s), 3.94 (3H, s), 4.11 (2H, q), 7.00 (1H, t), 7.13–7.24 (2H, m), 7.37 (1H, d), 7.42–7.60 (1H, m), 8.84 (1H, s), 9.42 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 296; acidic, HPLC <i>t</i><sub>R</sub> = 1.0 min.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-((4-Chloro-2-fluorophenyl)amino)-6,7-dimethoxy-<i>N</i>-methylquinoline-3-carboxamide (<b>19</b>)</h4><div class="NLM_p last">Trimethylaluminum (2 M in hexane) (3.7 mL, 7.4 mmol) was added to ethyl 4-((4-chloro-2-fluorophenyl)amino)-6,7-dimethoxyquinoline-3-carboxylate <b>18</b> (1.5 g, 3.7 mmol) and methylamine (2 M in THF, 5.56 mL, 11.1 mmol) in THF (20 mL) at room temperature. The resulting mixture was stirred at 60 °C for 2 h. The reaction mixture was poured into ice water (100 mL), diluted with DCM:methanol (10:1) (100 mL), and the mixture was stirred at room temperature for 15 min. The mixture was filtered through a Celite pad and washed with DCM:methanol (10:1) (100 mL), extracted with DCM:methanol (10:1) (3 × 100 mL), and then the organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford a pale-yellow solid. The crude product was purified by flash C18-flash chromatography, elution gradient 0–70% acetonitrile in water (containing 0.05% ammonia). Pure fractions were evaporated to dryness to afford 4-((4-chloro-2-fluorophenyl)amino)-6,7-dimethoxy-<i>N</i>-methylquinoline-3-carboxamide <b>19</b> (1.14 g, 79%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO) 2.66 (3H, d, <i>J</i> = 4.5 Hz), 3.61 (3H, s), 3.93 (3H, s), 6.82 (1H, t, <i>J</i> = 8.9 Hz), 7.06 (1H, s), 7.08–7.15 (1H, m), 7.34 (s, 1H), 7.46 (1H, dd, <i>J</i> = 2.2, 10.9 Hz), 8.61 (1H, d, <i>J</i> = 4.5 Hz), 8.73 (1H, s), 9.92 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 390; acidic, HPLC <i>t</i><sub>R</sub> = 1.247 min. HRMS ESI+ <i>m</i>/<i>z</i> observed 390.1014, C<sub>19</sub>H<sub>18</sub>N<sub>3</sub>O<sub>3</sub>ClF requires 390.1021.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Ethyl 6,7-Dimethoxy-4-(phenylamino)quinoline-3-carboxylate (<b>20</b>)</h4><div class="NLM_p last">Acetic acid (0.097 mL, 1.7 mmol) was added to ethyl 4-chloro-6,7-dimethoxyquinoline-3-carboxylate <b>17</b> (1 g, 3.4 mmol) and aniline (0.37 mL, 4.1 mmol) in DMF (20 mL) at 25 °C. The resulting mixture was stirred at 100 °C for 1 h. The reaction was evaporated to dryness, and the crude product was purified by flash silica chromatography, elution gradient 10–100% ethyl acetate (containing 2% triethylamine) in heptane. Pure fractions were evaporated to dryness to afford <b>20</b> (0.9 g, 76%) as a white solid. <sup>1</sup>H NMR (chloroform-<i>d</i>, 400 MHz) δ 1.45 (3H, t), 3.39 (3H, s), 4.00 (3H, s), 4.43 (2H, q), 6.86 (1H, s), 7.00–7.17 (3H, m), 7.22–7.41 (3H, m), 9.13 (1H, s), 10.31 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 353.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 6,7-Dimethoxy-<i>N</i>-methyl-4-(phenylamino)quinoline-3-carboxamide (<b>21</b>)</h4><div class="NLM_p last">Methylamine (33% in ethanol, 7.0 mL, 0.96 mmol) was added to ethyl 6,7-dimethoxy-4-(phenylamino)quinoline-3-carboxylate <b>20</b> (340 mg, 0.96 mmol) at 25 °C. The resulting mixture was stirred at 120 °C for 3 h. The reaction was evaporated and the crude product was purified by flash silica chromatography, elution gradient 0–15% methanol (containing 1% NH<sub>4</sub>OH/methanol) in DCM. Pure fractions were evaporated to dryness, triturated with diethyl ether, and filtered to afford <b>21</b> (265 mg, 81%) as a cream solid. <sup>1</sup>H NMR (400 MHz, CDCl3) 3.04 (3H, d, <i>J</i> = 4.8 Hz), 3.45 (3H, s), 4.00 (3H, s), 6.36 (1H, s), 6.90 (1H, s), 6.94–7.00 (2H, m), 7.03 (1H, t, <i>J</i> = 7.4 Hz), 7.25 (3H, d, <i>J</i> = 8.6 Hz), 8.70 (1H, s), 10.26 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 338. HRMS ESI+ <i>m</i>/<i>z</i> observed 338.1492, C<sub>19</sub>H<sub>20</sub>N<sub>3</sub>O<sub>3</sub> requires 338.1505</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Ethyl 6,7-Dimethoxy-4-phenoxyquinoline-3-carboxylate (<b>22</b>)</h4><div class="NLM_p last">Potassium carbonate (140 mg, 1 mmol) was added to ethyl 4-chloro-6,7-dimethoxyquinoline-3-carboxylate <b>17</b> (150 mg, 0.5 mmol) and phenol (95 mg, 1 mmol) in acetonitrile (3 mL) at room temperature under nitrogen. The resulting mixture was stirred at 80 °C for 16 h. The solvent was removed under reduced pressure and the crude product was purified by flash silica chromatography, elution gradient 0–1% methanol in DCM. Pure fractions were evaporated to dryness to afford <b>22</b> (180 mg, 100%) as a pale-yellow gum. <sup>1</sup>H NMR (methanol-<i>d</i><sub>4</sub>, 300 MHz) δ 1.11 (3H, t), 3.82 (3H, s), 4.06 (3H, s), 4.14 (2H, q), 6.83–6.94 (2H, m), 7.07 (1H, t), 7.29–7.36 (3H, m), 7.47 (1H, s), 9.03 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 354; acidic, HPLC <i>t</i><sub>R</sub> = 1.017 min.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 6,7-Dimethoxy-<i>N</i>-methyl-4-phenoxyquinoline-3-carboxamide (<b>23</b>)</h4><div class="NLM_p last">NaOH (1.7 mL, 1.7 mmol) was added to ethyl 6,7-dimethoxy-4-phenoxyquinoline-3-carboxylate <b>22</b> (150 mg, 0.42 mmol) in THF (5 mL), and the resulting mixture was stirred at room temperature for 17 h. The reaction mixture was adjusted to pH = 6 with 2 M HCl, and then the solvent was removed under reduced pressure. The solid was collected by filtration then sulfurous dichloride (5 mL) was added and the mixture was stirred at 80 °C for 1 h. The solvent was removed under reduced pressure, and then the residue was dissolved in DMF (5 mL), methylamine (2 M in THF, 2.0 mL, 4.0 mmol) was added, and the mixture was stirred at room temperature for 1 h. The solvent was removed under reduced pressure and the crude product was purified by preparative HPLC (XSelect CSH Prep C18 OBD column, 5 μ silica, 19 mm diameter, 150 mm length), using decreasingly polar mixtures of water (containing 0.1% NH<sub>4</sub>HCO<sub>3</sub>) and acetonitrile as eluents. Fractions containing the desired compound were evaporated to dryness to afford <b>23</b> (92 mg, 74%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO) 2.58 (3H, s), 3.74 (3H, s), 3.96 (3H, s), 6.90 (2H, d, <i>J</i> = 7.8 Hz), 7.07 (1H, t, <i>J</i> = 7.4 Hz), 7.16 (1H, s), 7.26–7.38 (2H, m), 7.48 (1H, s), 8.30 (1H, d, <i>J</i> = 4.6 Hz), 8.77 (1H, s); <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 339; acidic, HPLC <i>t</i><sub>R</sub> = 1.110 min. HRMS ESI+ <i>m</i>/<i>z</i> observed 339.1363, C<sub>19</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub> requires 339.1345</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 2-(4-(2-Aminoquinazolin-4-yl)phenyl)-<i>N</i>,<i>N</i>-dimethylacetamide (<b>24</b>)</h4><div class="NLM_p last">Pd 118 (35 mg, 0.05 mmol) was added to <i>N</i>,<i>N</i>-dimethyl-2-(4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)acetamide (435 mg, 1.5 mmol), potassium carbonate (139 mg, 1 mmol), and 4-chloroquinazolin-2-amine (180 mg, 1 mmol) in 1,4-dioxane (5 mL) and water at 25 °C under nitrogen. The resulting mixture was stirred at 100 °C for 2 h. The reaction was evaporated and the crude product was purified by flash silica chromatography, elution gradient 0–10% methanol in DCM. Pure fractions were evaporated to dryness and triturated with ether, filtered, and dried to afford <b>24</b> (85 mg, 28%) as a light-brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) 3.00 (3H, s), 3.06 (3H, s), 3.83 (2H, s), 5.22 (2H, s), 7.22 (1H, ddd, <i>J</i> = 1.4, 6.7, 8.2 Hz), 7.45 (2H, d, <i>J</i> = 8.3 Hz), 7.62–7.66 (1H, m), 7.66–7.74 (3H, m), 7.83–7.9 (1H, m); <i>m</i>/<i>z</i> (ES+), [M + H]+ = 307. HRMS ESI+ <i>m</i>/<i>z</i> observed 307.1552, C<sub>18</sub>H<sub>19</sub>N<sub>4</sub>O requires 307.1559.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01760">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43181" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43181" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01760" class="ext-link">10.1021/acs.jmedchem.5b01760</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Protocols for the enzyme, SPR, NMR, and cell assays, synthetic methods for the remaining examples, together with crystallographic information, and kinase panel and secondary pharmacology selectivity data for compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_001.pdf">jm5b01760_si_001.pdf (2.56 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_002.csv">jm5b01760_si_002.csv (3.86 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01760" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81331" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81331" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason G. Kettle</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, 35S47 Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2842495b474606434d5c5c444d68495b5c5a49524d464d4b49064b4745"><span class="__cf_email__" data-cfemail="bcd6ddcfd3d292d7d9c8c8d0d9fcddcfc8ceddc6d9d2d9dfdd92dfd3d1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Husam Alwan</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, 35S47 Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michal Bista</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason Breed</span> - <span class="hlFld-Affiliation affiliation">Discovery Sciences, AstraZeneca, Unit 310,
Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nichola L. Davies</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kay Eckersley</span> - <span class="hlFld-Affiliation affiliation">Discovery Sciences, AstraZeneca, Mereside,
Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaun Fillery</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin M. Foote</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Louise Goodwin</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, 35S47 Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David R. Jones</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, 35S47 Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helena Käck</span> - <span class="hlFld-Affiliation affiliation">Discovery Sciences, AstraZeneca, Pepparedsleden
1, 431 83 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan Lau</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, 35S47 Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Willem M. Nissink</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jon Read</span> - <span class="hlFld-Affiliation affiliation">Discovery Sciences, AstraZeneca, Unit 310,
Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James S. Scott</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ben Taylor</span> - <span class="hlFld-Affiliation affiliation">Discovery Sciences, AstraZeneca, Unit 310,
Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Graeme Walker</span> - <span class="hlFld-Affiliation affiliation">Discovery Sciences, AstraZeneca, Mereside,
Alderley Park, Macclesfield, Cheshire SK10 4TG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa Wissler</span> - <span class="hlFld-Affiliation affiliation">Discovery Sciences, AstraZeneca, Pepparedsleden
1, 431 83 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marta Wylot</span> - <span class="hlFld-Affiliation affiliation">Oncology Innovative Medicines Unit, AstraZeneca, Unit 310, Darwin Building, Cambridge Science Park, Milton Road, Cambridge CB4 0WG, United Kingdom</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d46e3990-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72641" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72641" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank all past members of the original MTH1 program from AstraZeneca Charnwood without whom this work would not have been possible. We also thank David Whittaker, Calum Cook, David Perkins, and Scott Boyd for aspects of synthesis. We thank Pelago Bioscience for performing cellular thermal stability assays.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23764" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23764" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="first last">This paper was published ASAP on February 16, 2016. The Supporting Information PDF was updated with additional data. The paper was reposted on February 25, 2016.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i38" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i38"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i39" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i39"> Abbreviations Used</h2><tr><td class="NLM_term">BCC</td><td class="NLM_def"><p class="first last">basal cell carcinoma</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">CRISPR</td><td class="NLM_def"><p class="first last">clustered regularly interspaced short palindromic repeats</p></td></tr><tr><td class="NLM_term">DDR</td><td class="NLM_def"><p class="first last">DNA-damage response</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">ligand-lipophilic efficiency</p></td></tr><tr><td class="NLM_term">PAMPs</td><td class="NLM_def"><p class="first last">pathogen-associated molecular patterns</p></td></tr><tr><td class="NLM_term">PBMC</td><td class="NLM_def"><p class="first last">peripheral blood mononuclear cells</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of means</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 20 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Freudenthal, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perera, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shock, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, S. H.</span><span> </span><span class="NLM_article-title">Uncovering the polymerase-induced cytotoxicity of an oxidized nucleotide</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">517</span><span class="NLM_x">, </span> <span class="NLM_fpage">635</span><span class="NLM_x">–</span> <span class="NLM_lpage">639</span><span class="refDoi"> DOI: 10.1038/nature13886</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1038%2Fnature13886" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=25409153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGlt7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2015&pages=635-639&author=B.+D.+Freudenthalauthor=W.+A.+Beardauthor=L.+Pereraauthor=D.+D.+Shockauthor=T.+Kimauthor=T.+Schlickauthor=S.+H.+Wilson&title=Uncovering+the+polymerase-induced+cytotoxicity+of+an+oxidized+nucleotide&doi=10.1038%2Fnature13886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Uncovering the polymerase-induced cytotoxicity of an oxidized nucleotide</span></div><div class="casAuthors">Freudenthal, Bret D.; Beard, William A.; Perera, Lalith; Shock, David D.; Kim, Taejin; Schlick, Tamar; Wilson, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">517</span>
        (<span class="NLM_cas:issue">7536</span>),
    <span class="NLM_cas:pages">635-639</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oxidative stress promotes genomic instability and human diseases.  A common oxidized nucleoside is 8-oxo-7,8-dihydro-2'-deoxyguanosine, which is found both in DNA (8-oxo-G) and as a free nucleotide (8-oxo-dGTP).  Nucleotide pools are esp. vulnerable to oxidative damage.  Therefore cells encode an enzyme (MutT/MTH1) that removes free oxidized nucleotides.  This cleansing function is required for cancer cell survival and to modulate Escherichia coli antibiotic sensitivity in a DNA polymerase (pol)-dependent manner.  How polymerases discriminate between damaged and non-damaged nucleotides is not well understood.  This anal. is essential given the role of oxidized nucleotides in mutagenesis, cancer therapeutics, and bacterial antibiotics.  Even with cellular sanitizing activities, nucleotide pools contain enough 8-oxo-dGTP to promote mutagenesis.  This arises from the dual coding potential where 8-oxo-dGTP(anti) base pairs with cytosine and 8-oxo-dGTP(syn) uses its Hoogsteen edge to base pair with adenine.  Here we use time-lapse crystallog. to follow 8-oxo-dGTP insertion opposite adenine or cytosine with human pol β, to reveal that insertion is accommodated in either the syn- or anti-conformation, resp.  For 8-oxo-dGTP(anti) insertion, a novel divalent metal relieves repulsive interactions between the adducted guanine base and the triphosphate of the oxidized nucleotide.  With either templating base, hydrogen-bonding interactions between the bases are lost as the enzyme reopens after catalysis, leading to a cytotoxic nicked DNA repair intermediate.  Combining structural snapshots with kinetic and computational anal. reveals how 8-oxo-dGTP uses charge modulation during insertion that can lead to a blocked DNA repair intermediate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUp5_4v80tf7Vg90H21EOLACvtfcHk0lizrMGqi0xuKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGlt7nK&md5=cc5f601f0879b37d38e924f66cd4df06</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature13886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13886%26sid%3Dliteratum%253Aachs%26aulast%3DFreudenthal%26aufirst%3DB.%2BD.%26aulast%3DBeard%26aufirst%3DW.%2BA.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DShock%26aufirst%3DD.%2BD.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DSchlick%26aufirst%3DT.%26aulast%3DWilson%26aufirst%3DS.%2BH.%26atitle%3DUncovering%2520the%2520polymerase-induced%2520cytotoxicity%2520of%2520an%2520oxidized%2520nucleotide%26jtitle%3DNature%26date%3D2015%26volume%3D517%26spage%3D635%26epage%3D639%26doi%3D10.1038%2Fnature13886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Gad, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koolmeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jemth, A.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eshtad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ström, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svensson, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundbäck, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einarsdottir, B. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saleh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Göktürk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baranczewski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svensson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berntsson, R. P.-A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strömberg, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanjiv, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques-Cordonnier, M.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desroses, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustavsson, A.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olofsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homan, E.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loseva, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bräutigam, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Höglund, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagenkort, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pham, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altun, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaugaz, F. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vikingsson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henriksson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vallin, K. S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallner, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammarström, L. G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiita, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almlöf, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalderén, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Axelsson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djureinovic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puigvert, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Häggblad, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeppsson, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martens, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundgren, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granelli, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stenmark, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scobie, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, U. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helleday, T.</span><span> </span><span class="NLM_article-title">MTH1 inhibition eradicates cancer by preventing sanitation of the dNTP pool</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">508</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1038/nature13181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1038%2Fnature13181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=24695224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=215-221&author=H.+Gadauthor=T.+Koolmeisterauthor=A.-S.+Jemthauthor=S.+Eshtadauthor=S.+A.+Jacquesauthor=C.+E.+Str%C3%B6mauthor=L.+M.+Svenssonauthor=N.+Schultzauthor=T.+Lundb%C3%A4ckauthor=B.+O.+Einarsdottirauthor=A.+Salehauthor=C.+G%C3%B6kt%C3%BCrkauthor=P.+Baranczewskiauthor=R.+Svenssonauthor=R.+P.-A.+Berntssonauthor=R.+Gustafssonauthor=K.+Str%C3%B6mbergauthor=K.+Sanjivauthor=M.-C.+Jacques-Cordonnierauthor=M.+Desrosesauthor=A.-L.+Gustavssonauthor=R.+Olofssonauthor=F.+Johanssonauthor=E.-J.+Homanauthor=O.+Losevaauthor=L.+Br%C3%A4utigamauthor=L.+Johanssonauthor=A.+H%C3%B6glundauthor=A.+Hagenkortauthor=T.+Phamauthor=M.+Altunauthor=F.+Z.+Gaugazauthor=S.+Vikingssonauthor=B.+Eversauthor=M.+Henrikssonauthor=K.+S.+A.+Vallinauthor=O.+A.+Wallnerauthor=L.+G.+J.+Hammarstr%C3%B6mauthor=E.+Wiitaauthor=I.+Alml%C3%B6fauthor=C.+Kalder%C3%A9nauthor=H.+Axelssonauthor=T.+Djureinovicauthor=J.+C.+Puigvertauthor=M.+H%C3%A4ggbladauthor=F.+Jeppssonauthor=U.+Martensauthor=C.+Lundinauthor=B.+Lundgrenauthor=I.+Granelliauthor=A.+J.+Jensenauthor=P.+Arturssonauthor=J.+A.+Nilssonauthor=P.+Stenmarkauthor=M.+Scobieauthor=U.+W.+Berglundauthor=T.+Helleday&title=MTH1+inhibition+eradicates+cancer+by+preventing+sanitation+of+the+dNTP+pool&doi=10.1038%2Fnature13181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">MTH1 inhibition eradicates cancer by preventing sanitation of the dNTP pool</span></div><div class="casAuthors">Gad, Helge; Koolmeister, Tobias; Jemth, Ann-Sofie; Eshtad, Saeed; Jacques, Sylvain A.; Stroem, Cecilia E.; Svensson, Linda M.; Schultz, Niklas; Lundbaeck, Thomas; Einarsdottir, Berglind Osk; Saleh, Aljona; Gokturk, Camilla; Baranczewski, Pawel; Svensson, Richard; Berntsson, Ronnie P.-A.; Gustafsson, Robert; Stroemberg, Kia; Sanjiv, Kumar; Jacques-Cordonnier, Marie-Caroline; Desroses, Matthieu; Gustavsson, Anna-Lena; Olofsson, Roger; Johansson, Fredrik; Homan, Evert J.; Loseva, Olga; Braeutigam, Lars; Johansson, Lars; Hoeglund, Andreas; Hagenkort, Anna; Pham, Therese; Altun, Mikael; Gaugaz, Fabienne Z.; Vikingsson, Svante; Evers, Bastiaan; Henriksson, Martin; Vallin, Karl S. A.; Wallner, Olov A.; Hammarstroem, Lars G. J.; Wiita, Elisee; Almloef, Ingrid; Kalderen, Christina; Axelsson, Hanna; Djureinovic, Tatjana; Puigvert, Jordi Carreras; Haeggblad, Maria; Jeppsson, Fredrik; Martens, Ulf; Lundin, Cecilia; Lundgren, Bo; Granelli, Ingrid; Jensen, Annika Jenmalm; Artursson, Per; Nilsson, Jonas A.; Stenmark, Pal; Scobie, Martin; Berglund, Ulrika Warpman; Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">215-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancers have dysfunctional redox regulation resulting in reactive oxygen species prodn., damaging both DNA and free dNTPs.  The MTH1 protein sanitizes oxidized dNTP pools to prevent incorporation of damaged bases during DNA replication.  Although MTH1 is non-essential in normal cells, the authors show that cancer cells require MTH1 activity to avoid incorporation of oxidized dNTPs, resulting in DNA damage and cell death.  The authors validate MTH1 as an anticancer target in vivo and describe small mols. TH287 (I) and TH588 (II) as first-in-class nudix hydrolase family inhibitors that potently and selectively engage and inhibit the MTH1 protein in cells.  Protein co-crystal structures demonstrate that the inhibitors bind in the active site of MTH1.  The inhibitors cause incorporation of oxidized dNTPs in cancer cells, leading to DNA damage, cytotoxicity and therapeutic responses in patient-derived mouse xenografts.  This study exemplifies the non-oncogene addiction concept for anticancer treatment and validates MTH1 as being cancer phenotypic lethal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Xn3KhC40XbVg90H21EOLACvtfcHk0lgtKMvnwhBUww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D&md5=49afb4522874f4f5e216ae38d0ff7cd2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature13181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13181%26sid%3Dliteratum%253Aachs%26aulast%3DGad%26aufirst%3DH.%26aulast%3DKoolmeister%26aufirst%3DT.%26aulast%3DJemth%26aufirst%3DA.-S.%26aulast%3DEshtad%26aufirst%3DS.%26aulast%3DJacques%26aufirst%3DS.%2BA.%26aulast%3DStr%25C3%25B6m%26aufirst%3DC.%2BE.%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DEinarsdottir%26aufirst%3DB.%2BO.%26aulast%3DSaleh%26aufirst%3DA.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DBaranczewski%26aufirst%3DP.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DBerntsson%26aufirst%3DR.%2BP.-A.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DJacques-Cordonnier%26aufirst%3DM.-C.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DGustavsson%26aufirst%3DA.-L.%26aulast%3DOlofsson%26aufirst%3DR.%26aulast%3DJohansson%26aufirst%3DF.%26aulast%3DHoman%26aufirst%3DE.-J.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DBr%25C3%25A4utigam%26aufirst%3DL.%26aulast%3DJohansson%26aufirst%3DL.%26aulast%3DH%25C3%25B6glund%26aufirst%3DA.%26aulast%3DHagenkort%26aufirst%3DA.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DAltun%26aufirst%3DM.%26aulast%3DGaugaz%26aufirst%3DF.%2BZ.%26aulast%3DVikingsson%26aufirst%3DS.%26aulast%3DEvers%26aufirst%3DB.%26aulast%3DHenriksson%26aufirst%3DM.%26aulast%3DVallin%26aufirst%3DK.%2BS.%2BA.%26aulast%3DWallner%26aufirst%3DO.%2BA.%26aulast%3DHammarstr%25C3%25B6m%26aufirst%3DL.%2BG.%2BJ.%26aulast%3DWiita%26aufirst%3DE.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DKalder%25C3%25A9n%26aufirst%3DC.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DDjureinovic%26aufirst%3DT.%26aulast%3DPuigvert%26aufirst%3DJ.%2BC.%26aulast%3DH%25C3%25A4ggblad%26aufirst%3DM.%26aulast%3DJeppsson%26aufirst%3DF.%26aulast%3DMartens%26aufirst%3DU.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DLundgren%26aufirst%3DB.%26aulast%3DGranelli%26aufirst%3DI.%26aulast%3DJensen%26aufirst%3DA.%2BJ.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DNilsson%26aufirst%3DJ.%2BA.%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DMTH1%2520inhibition%2520eradicates%2520cancer%2520by%2520preventing%2520sanitation%2520of%2520the%2520dNTP%2520pool%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D215%26epage%3D221%26doi%3D10.1038%2Fnature13181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Huber, K. V. M</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salah, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radic, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gridling, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elkins, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jemth, A.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Göktürk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanjiv, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strömberg, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pham, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, U. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colinge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loizou, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helleday, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Stereospecific targeting of MTH1 by (<i>S</i>)-crizotinib as an anticancer strategy</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">508</span><span class="NLM_x">, </span> <span class="NLM_fpage">222</span><span class="NLM_x">–</span> <span class="NLM_lpage">227</span><span class="refDoi"> DOI: 10.1038/nature13194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1038%2Fnature13194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=24695225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=222-227&author=K.+V.+M+Huberauthor=E.+Salahauthor=B.+Radicauthor=M.+Gridlingauthor=J.+M.+Elkinsauthor=A.+Stukalovauthor=A.-S.+Jemthauthor=C.+G%C3%B6kt%C3%BCrkauthor=K.+Sanjivauthor=K.+Str%C3%B6mbergauthor=T.+Phamauthor=U.+W.+Berglundauthor=J.+Colingeauthor=K.+L.+Bennettauthor=J.+I.+Loizouauthor=T.+Helledayauthor=S.+Knappauthor=G.+Superti-Furga&title=Stereospecific+targeting+of+MTH1+by+%28S%29-crizotinib+as+an+anticancer+strategy&doi=10.1038%2Fnature13194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Stereospecific targeting of MTH1 by (S)-crizotinib as an anticancer strategy</span></div><div class="casAuthors">Huber, Kilian V. M.; Salah, Eidarus; Radic, Branka; Gridling, Manuela; Elkins, Jonathan M.; Stukalov, Alexey; Jemth, Ann-Sofie; Gokturk, Camilla; Sanjiv, Kumar; Stroemberg, Kia; Pham, Therese; Berglund, Ulrika Warpman; Colinge, Jacques; Bennett, Keiryn L.; Loizou, Joanna I.; Helleday, Thomas; Knapp, Stefan; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">222-227</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activated RAS GTPase signaling is a crit. driver of oncogenic transformation and malignant disease.  Cellular models of RAS-dependent cancers have been used to identify exptl. small mols., such as SCH51344, but their mol. mechanism of action remains generally unknown.  Here, using a chem. proteomic approach, the authors identify the target of SCH51344 as the human mutT homolog MTH1 (also known as NUDT1), a nucleotide pool sanitizing enzyme.  Loss-of-function of MTH1 impaired growth of KRAS tumor cells, whereas MTH1 overexpression mitigated sensitivity towards SCH51344.  Searching for more drug-like inhibitors, the authors identified the kinase inhibitor crizotinib as a nanomolar suppressor of MTH1 activity.  Surprisingly, the clin. used (R)-enantiomer of the drug was inactive, whereas the (S)-enantiomer selectively inhibited MTH1 catalytic activity.  Enzymic assays, chem. proteomic profiling, kinome-wide activity surveys and MTH1 co-crystal structures of both enantiomers provide a rationale for this remarkable stereospecificity.  Disruption of nucleotide pool homeostasis via MTH1 inhibition by (S)-crizotinib induced an increase in DNA single-strand breaks, activated DNA repair in human colon carcinoma cells, and effectively suppressed tumor growth in animal models.  The authors' results propose (S)-crizotinib as an attractive chem. entity for further pre-clin. evaluation, and small-mol. inhibitors of MTH1 in general as a promising novel class of anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbUm7UP6kIrVg90H21EOLACvtfcHk0lgtKMvnwhBUww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsb4%253D&md5=ca3df11a6782fd9140b6d8ba0c16e5ab</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature13194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13194%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DRadic%26aufirst%3DB.%26aulast%3DGridling%26aufirst%3DM.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DJemth%26aufirst%3DA.-S.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DLoizou%26aufirst%3DJ.%2BI.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DStereospecific%2520targeting%2520of%2520MTH1%2520by%2520%2528S%2529-crizotinib%2520as%2520an%2520anticancer%2520strategy%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D222%26epage%3D227%26doi%3D10.1038%2Fnature13194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Egashira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamauchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshiyama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawate, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katsuki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekiguchi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimachi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuzuki, T.</span><span> </span><span class="NLM_article-title">Mutational specificity of mice defective in the MTH1 and/or the MSH2 genes</span> <span class="citation_source-journal">DNA Repair</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">893</span><span class="refDoi"> DOI: 10.1016/S1568-7864(02)00113-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2FS1568-7864%2802%2900113-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=881-893&author=A.+Egashiraauthor=K.+Yamauchiauthor=K.+Yoshiyamaauthor=H.+Kawateauthor=M.+Katsukiauthor=M.+Sekiguchiauthor=K.+Sugimachiauthor=H.+Makiauthor=T.+Tsuzuki&title=Mutational+specificity+of+mice+defective+in+the+MTH1+and%2For+the+MSH2+genes&doi=10.1016%2FS1568-7864%2802%2900113-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS1568-7864%2802%2900113-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1568-7864%252802%252900113-1%26sid%3Dliteratum%253Aachs%26aulast%3DEgashira%26aufirst%3DA.%26aulast%3DYamauchi%26aufirst%3DK.%26aulast%3DYoshiyama%26aufirst%3DK.%26aulast%3DKawate%26aufirst%3DH.%26aulast%3DKatsuki%26aufirst%3DM.%26aulast%3DSekiguchi%26aufirst%3DM.%26aulast%3DSugimachi%26aufirst%3DK.%26aulast%3DMaki%26aufirst%3DH.%26aulast%3DTsuzuki%26aufirst%3DT.%26atitle%3DMutational%2520specificity%2520of%2520mice%2520defective%2520in%2520the%2520MTH1%2520and%252For%2520the%2520MSH2%2520genes%26jtitle%3DDNA%2520Repair%26date%3D2002%26volume%3D1%26spage%3D881%26epage%3D893%26doi%3D10.1016%2FS1568-7864%2802%2900113-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Kobold, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedemann, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothenfußer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endres, S.</span><span> </span><span class="NLM_article-title">Modes of action of TLR7 agonists in cancer therapy immunotherapy</span> <span class="citation_source-journal">Immunotherapy</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1085</span><span class="NLM_x">–</span> <span class="NLM_lpage">1095</span><span class="refDoi"> DOI: 10.2217/imt.14.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.2217%2Fimt.14.75" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1085-1095&author=S.+Koboldauthor=G.+Wiedemannauthor=S.+Rothenfu%C3%9Ferauthor=S.+Endres&title=Modes+of+action+of+TLR7+agonists+in+cancer+therapy+immunotherapy&doi=10.2217%2Fimt.14.75"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2217%2Fimt.14.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fimt.14.75%26sid%3Dliteratum%253Aachs%26aulast%3DKobold%26aufirst%3DS.%26aulast%3DWiedemann%26aufirst%3DG.%26aulast%3DRothenfu%25C3%259Fer%26aufirst%3DS.%26aulast%3DEndres%26aufirst%3DS.%26atitle%3DModes%2520of%2520action%2520of%2520TLR7%2520agonists%2520in%2520cancer%2520therapy%2520immunotherapy%26jtitle%3DImmunotherapy%26date%3D2014%26volume%3D6%26spage%3D1085%26epage%3D1095%26doi%3D10.2217%2Fimt.14.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Raasch, B.</span><span> </span><span class="NLM_article-title">Management of superficial basal cell carcinoma: focus on imiquimod</span> <span class="citation_source-journal">Clin., Cosmet. Invest. Dermatol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.2147/CCID.S3507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.2147%2FCCID.S3507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=21436969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVShtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=65-67&author=B.+Raasch&title=Management+of+superficial+basal+cell+carcinoma%3A+focus+on+imiquimod&doi=10.2147%2FCCID.S3507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Management of superficial basal cell carcinoma: focus on imiquimod</span></div><div class="casAuthors">Raasch, Beverly</div><div class="citationInfo"><span class="NLM_cas:title">Clinical, Cosmetic and Investigational Dermatology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">65-75</span>CODEN:
                <span class="NLM_cas:coden">CCIDCL</span>;
        ISSN:<span class="NLM_cas:issn">1178-7015</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press (NZ) Ltd.</span>)
        </div><div class="casAbstract">A review.  Superficial basal cell carcinoma comprise up to 25% of all histol. sub-types.  They are more likely to occur on younger persons and females and although generally more common on the trunk, also occur frequently on the exposed areas of the head and neck esp. in areas of high sun exposure.  In the last decade, new treatment options such as topical applications that modify the immune response have been trialed for effectiveness in treating these lesions.  Imiquimod 5% cream has been shown to stimulate the innate and cell mediated immune system.  The short-term success of imiquimod 5% cream in randomized controlled trials comparing different treatment regimes and dosing as a treatment for small superficial basal cell carcinoma (BCC) not on the face or neck is in the range of 82% for 5 times per wk application.  A high proportion of participants with good response rates to topical treatment (58%-92%) experience local side effects such as itching and burning, less commonly erosion and ulceration, but the proportion of participants ceasing treatment has not been high.  To date one long-term study indicates a treatment success rate of 78%-81% and that initial response is a predictor of long-term outcome.  Recurrences tend to occur within the first year after treatment.  Future research will compare this prepn. to the gold std. treatment for superficial BCC - surgical excision.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri0Dwxe6VLfrVg90H21EOLACvtfcHk0lgQJC7svDpEIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVShtL3J&md5=21eeab1a2735cc7d1b190f04c9c08e3a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2147%2FCCID.S3507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FCCID.S3507%26sid%3Dliteratum%253Aachs%26aulast%3DRaasch%26aufirst%3DB.%26atitle%3DManagement%2520of%2520superficial%2520basal%2520cell%2520carcinoma%253A%2520focus%2520on%2520imiquimod%26jtitle%3DClin.%252C%2520Cosmet.%2520Invest.%2520Dermatol.%26date%3D2009%26volume%3D2%26spage%3D65%26epage%3D67%26doi%3D10.2147%2FCCID.S3507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Greiff, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cervin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahlstroem-Emanuelsson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almqvist, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolata, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eriksson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoegestaett, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaellen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norlen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjölin, I.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widegren, H.</span><span> </span><span class="NLM_article-title">Repeated intranasal TLR7 stimulation reduces allergen responsiveness in allergic rhinitis</span> <span class="citation_source-journal">Respir. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="refDoi"> DOI: 10.1186/1465-9921-13-53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1186%2F1465-9921-13-53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=22726593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCnsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=53&author=L.+Greiffauthor=A.+Cervinauthor=C.+Ahlstroem-Emanuelssonauthor=G.+Almqvistauthor=M.+Anderssonauthor=J.+Dolataauthor=L.+Erikssonauthor=E.+Hoegestaettauthor=A.+Kaellenauthor=P.+Norlenauthor=I.-L.+Sj%C3%B6linauthor=H.+Widegren&title=Repeated+intranasal+TLR7+stimulation+reduces+allergen+responsiveness+in+allergic+rhinitis&doi=10.1186%2F1465-9921-13-53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Repeated intranasal TLR7 stimulation reduces allergen responsiveness in allergic rhinitis</span></div><div class="casAuthors">Greiff, Lennart; Cervin, Anders; Ahlstroem-Emanuelsson, Cecilia; Almqvist, Gun; Andersson, Morgan; Dolata, Jan; Eriksson, Leif; Hoegestaett, Edward; Kaellen, Anders; Norlen, Per; Sjoelin, Inga-Lisa; Widegren, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Respiratory Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53</span>CODEN:
                <span class="NLM_cas:coden">RREEBZ</span>;
        ISSN:<span class="NLM_cas:issn">1465-993X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Interactions between Th1 and Th2 immune responses are of importance to the onset and development of allergic disorders.  A Toll-like receptor 7 agonist such as AZD8848 may have potential as a treatment for allergic airway disease by skewing the immune system away from a Th2 profile.  Objective: To evaluate the efficacy and safety of intranasal AZD8848.  Methods: In a placebo-controlled single ascending dose study, AZD8848 (0.3-600 μg) was given intranasally to 48 healthy subjects and 12 patients with allergic rhinitis.  In a placebo-controlled repeat challenge/treatment study, AZD8848 (30 and 60 μg) was given once weekly for five weeks to 74 patients with allergic rhinitis out of season: starting 24 h after the final dose, daily allergen challenges were given for seven days.  Safety, tolerability, pharmacokinetics and biomarkers were monitored.  During the allergen challenge series, nasal symptoms and lavage fluid levels of tryptase and α2-macroglobulin, reflecting mast cell activity and plasma exudation, were monitored.  Results: AZD8848 produced reversible blood lymphocyte redns. and dose-dependent flu-like symptoms: 30-100 μg produced consistent yet tolerable effects.  Plasma interleukin-1 receptor antagonist was elevated after administration of AZD8848, reflecting interferon prodn. secondary to TLR7 stimulation.  At repeat challenge/treatment, AZD8848 reduced nasal symptoms recorded ten minutes after allergen challenge up to eight days after the final dose.  Tryptase and α2-macroglobulin were also reduced by AZD8848.  Conclusions: Repeated intranasal stimulation of Toll-like receptor 7 by AZD8848 was safe and produced a sustained redn. in the responsiveness to allergen in allergic rhinitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoehN4HRRB9K7Vg90H21EOLACvtfcHk0liWDHzhAX5Jww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCnsLrP&md5=1427aaddf24c0946cc1fbcedb0602d7d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1465-9921-13-53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1465-9921-13-53%26sid%3Dliteratum%253Aachs%26aulast%3DGreiff%26aufirst%3DL.%26aulast%3DCervin%26aufirst%3DA.%26aulast%3DAhlstroem-Emanuelsson%26aufirst%3DC.%26aulast%3DAlmqvist%26aufirst%3DG.%26aulast%3DAndersson%26aufirst%3DM.%26aulast%3DDolata%26aufirst%3DJ.%26aulast%3DEriksson%26aufirst%3DL.%26aulast%3DHoegestaett%26aufirst%3DE.%26aulast%3DKaellen%26aufirst%3DA.%26aulast%3DNorlen%26aufirst%3DP.%26aulast%3DSj%25C3%25B6lin%26aufirst%3DI.-L.%26aulast%3DWidegren%26aufirst%3DH.%26atitle%3DRepeated%2520intranasal%2520TLR7%2520stimulation%2520reduces%2520allergen%2520responsiveness%2520in%2520allergic%2520rhinitis%26jtitle%3DRespir.%2520Res.%26date%3D2012%26volume%3D13%26spage%3D53%26doi%3D10.1186%2F1465-9921-13-53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Diebold, S. S.</span><span> </span><span class="NLM_article-title">Recognition of viral single-stranded RNA by toll-like receptors</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">–</span> <span class="NLM_lpage">823</span><span class="refDoi"> DOI: 10.1016/j.addr.2007.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2Fj.addr.2007.11.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2008&pages=813-823&author=S.+S.+Diebold&title=Recognition+of+viral+single-stranded+RNA+by+toll-like+receptors&doi=10.1016%2Fj.addr.2007.11.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2007.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2007.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DDiebold%26aufirst%3DS.%2BS.%26atitle%3DRecognition%2520of%2520viral%2520single-stranded%2520RNA%2520by%2520toll-like%2520receptors%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2008%26volume%3D60%26spage%3D813%26epage%3D823%26doi%3D10.1016%2Fj.addr.2007.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Bonfanti, J.-F.; Fortin, J. M. C.; Muller, P.; Doublet, F. M. M.; Raboisson, P. J-M. B.; Arnoult, E. P. A.</span><span> </span><span class="NLM_article-title">Preparation of macrocyclic purines for the treatment of viral infections</span>. PCT Int. Appl. WO2014009509,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=J.-F.+Bonfanti&author=J.+M.+C.+Fortin&author=P.+Muller&author=F.+M.+M.+Doublet&author=P.+J-M.+B.+Raboisson&author=E.+P.+A.+Arnoult&title=Preparation+of+macrocyclic+purines+for+the+treatment+of+viral+infections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBonfanti%26aufirst%3DJ.-F.%26atitle%3DPreparation%2520of%2520macrocyclic%2520purines%2520for%2520the%2520treatment%2520of%2520viral%2520infections%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Meyer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockfleth, E.</span><span> </span><span class="NLM_article-title">Resiquimod, a topical drug for viral skin lesions and skin cancer</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1517/13543784.2013.749236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1517%2F13543784.2013.749236" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2013&pages=149-159&author=T.+Meyerauthor=C.+Surberauthor=L.+E.+Frenchauthor=E.+Stockfleth&title=Resiquimod%2C+a+topical+drug+for+viral+skin+lesions+and+skin+cancer&doi=10.1517%2F13543784.2013.749236"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1517%2F13543784.2013.749236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2013.749236%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DSurber%26aufirst%3DC.%26aulast%3DFrench%26aufirst%3DL.%2BE.%26aulast%3DStockfleth%26aufirst%3DE.%26atitle%3DResiquimod%252C%2520a%2520topical%2520drug%2520for%2520viral%2520skin%2520lesions%2520and%2520skin%2520cancer%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2013%26volume%3D22%26spage%3D149%26epage%3D159%26doi%3D10.1517%2F13543784.2013.749236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Kurimoto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogino, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichii, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isobe, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tobe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takaku, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sajiki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirota, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawakami, H.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of 2-substituted 8-hydroxyadenines as potent interferon inducers with improved oral bioavailabilities</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1099</span><span class="refDoi"> DOI: 10.1016/j.bmc.2003.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2Fj.bmc.2003.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=14980621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFWht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=1091-1099&author=A.+Kurimotoauthor=T.+Oginoauthor=S.+Ichiiauthor=Y.+Isobeauthor=M.+Tobeauthor=H.+Ogitaauthor=H.+Takakuauthor=H.+Sajikiauthor=K.+Hirotaauthor=H.+Kawakami&title=Synthesis+and+evaluation+of+2-substituted+8-hydroxyadenines+as+potent+interferon+inducers+with+improved+oral+bioavailabilities&doi=10.1016%2Fj.bmc.2003.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of 2-substituted 8-hydroxyadenines as potent interferon inducers with improved oral bioavailabilities</span></div><div class="casAuthors">Kurimoto, Ayumu; Ogino, Tetsuhiro; Ichii, Shinji; Isobe, Yoshiaki; Tobe, Masanori; Ogita, Haruhisa; Takaku, Haruo; Sajiki, Hironao; Hirota, Kosaku; Kawakami, Hajime</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1091-1099</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In order to create novel compds. which possess potent interferon (IFN) inducing activities with excellent oral bioavailabilities, a series of 8-hydroxyadenines, which have various alkoxy or alkylthio moieties at the adenine C(2)-position, were synthesized and evaluated.  The introduction of hydrophobic groups was not considered to be effective for potentiating the IFN-inducing activity, but several compds. having hydrophilic groups were effective.  Among the compds. tested, compd. I induced IFN from the dosage of 0.03 mg/kg, which was approx. 100-fold more potent than that of Imiquimod, and showed an excellent oral bioavailability (F=40%) which was 10-fold improved over the lead compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqY8-Ou6UMlbVg90H21EOLACvtfcHk0ljhy_H2iuIrmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFWht7g%253D&md5=7786077b1ccac729aaaa96ec4254c1cf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2003.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2003.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DKurimoto%26aufirst%3DA.%26aulast%3DOgino%26aufirst%3DT.%26aulast%3DIchii%26aufirst%3DS.%26aulast%3DIsobe%26aufirst%3DY.%26aulast%3DTobe%26aufirst%3DM.%26aulast%3DOgita%26aufirst%3DH.%26aulast%3DTakaku%26aufirst%3DH.%26aulast%3DSajiki%26aufirst%3DH.%26aulast%3DHirota%26aufirst%3DK.%26aulast%3DKawakami%26aufirst%3DH.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%25202-substituted%25208-hydroxyadenines%2520as%2520potent%2520interferon%2520inducers%2520with%2520improved%2520oral%2520bioavailabilities%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D1091%26epage%3D1099%26doi%3D10.1016%2Fj.bmc.2003.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Kazaoka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sajiki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirota, K.</span><span> </span><span class="NLM_article-title">Synthesis of 6-substituted 9-benzyl-8-hydroxypurines with potential interferon-inducing activity</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">611</span><span class="refDoi"> DOI: 10.1248/cpb.51.608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1248%2Fcpb.51.608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=12736468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVejsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2003&pages=608-611&author=K.+Kazaokaauthor=H.+Sajikiauthor=K.+Hirota&title=Synthesis+of+6-substituted+9-benzyl-8-hydroxypurines+with+potential+interferon-inducing+activity&doi=10.1248%2Fcpb.51.608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of 6-substituted 9-benzyl-8-hydroxypurines with potential interferon-inducing activity</span></div><div class="casAuthors">Kazaoka, Kazunori; Sajiki, Hironao; Hirota, Kosaku</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">608-611</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Various 6-substituted 9-benzyl-8-hydroxypurines I [R = NH2, H, SH, SMe, OH, OMe, NHR1, NHCOR2; R1 = Me, Et, CH2Et, Bu, CH2CHMe2; R2 = Me, CF3, Ph] were synthesized in order to investigate the structure-activity relationship at the 6-position of 9-benzyl-8-hydroxyadenine [I R = NH2 (II)], which is a lead compd. for the screening of interferon (IFN)-inducing activity.  6-Unsubstituted, mercapto-, methylthio- and hydroxy-substituted 9-benzyl-8-hydroxypurines I (R = H, SH, SMe, OH) were prepd. from 5-amino-1-benzyl-4-cyano-2-hydroxyimidazole.  Synthesis of a 6-methoxy analog I (R = OMe) was conducted from 5-amino-4-benzylamino-6-chloropyrimidine.  6-Alkylamino and acylaminopurines I [R = NHR1, NHCOR2] were also prepd. by reductive alkylation and acylation of II, resp.  Since these compds. indicated no activity, it was found that a free amino group of II is required for the expression of IFN-inducing activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogl5sHW5hz77Vg90H21EOLACvtfcHk0ljhy_H2iuIrmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVejsb0%253D&md5=363df4fceae05b8aa34a2a6244d72cfa</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1248%2Fcpb.51.608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.51.608%26sid%3Dliteratum%253Aachs%26aulast%3DKazaoka%26aufirst%3DK.%26aulast%3DSajiki%26aufirst%3DH.%26aulast%3DHirota%26aufirst%3DK.%26atitle%3DSynthesis%2520of%25206-substituted%25209-benzyl-8-hydroxypurines%2520with%2520potential%2520interferon-inducing%2520activity%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2003%26volume%3D51%26spage%3D608%26epage%3D611%26doi%3D10.1248%2Fcpb.51.608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Gerster, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomai, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birmachu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bomersine, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imbertson, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knafla, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maye, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikolaides, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oneyemi, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkhurst, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecore, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiter, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scribner, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testerman, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weeks, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andre, J.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lagain, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastard, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupu, M.</span><span> </span><span class="NLM_article-title">Synthesis and structure-activity-relationships of 1<i>H</i>-imidazo[4,5-<i>c</i>]quinolines that induce interferon production</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">3481</span><span class="NLM_x">–</span> <span class="NLM_lpage">3491</span><span class="refDoi"> DOI: 10.1021/jm049211v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049211v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3481-3491&author=J.+F.+Gersterauthor=K.+J.+Lindstromauthor=R.+L.+Millerauthor=M.+A.+Tomaiauthor=W.+Birmachuauthor=S.+N.+Bomersineauthor=S.+J.+Gibsonauthor=L.+M.+Imbertsonauthor=J.+R.+Jacobsonauthor=R.+T.+Knaflaauthor=P.+V.+Mayeauthor=N.+Nikolaidesauthor=F.+Y.+Oneyemiauthor=G.+W.+Parkhurstauthor=S.+E.+Pecoreauthor=M.+J.+Reiterauthor=L.+S.+Scribnerauthor=T.+L.+Testermanauthor=N.+J.+Thompsonauthor=T.+L.+Wagnerauthor=C.+E.+Weeksauthor=J.-D.+Andreauthor=D.+Lagainauthor=Y.+Bastardauthor=M.+Lupu&title=Synthesis+and+structure-activity-relationships+of+1H-imidazo%5B4%2C5-c%5Dquinolines+that+induce+interferon+production&doi=10.1021%2Fjm049211v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm049211v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049211v%26sid%3Dliteratum%253Aachs%26aulast%3DGerster%26aufirst%3DJ.%2BF.%26aulast%3DLindstrom%26aufirst%3DK.%2BJ.%26aulast%3DMiller%26aufirst%3DR.%2BL.%26aulast%3DTomai%26aufirst%3DM.%2BA.%26aulast%3DBirmachu%26aufirst%3DW.%26aulast%3DBomersine%26aufirst%3DS.%2BN.%26aulast%3DGibson%26aufirst%3DS.%2BJ.%26aulast%3DImbertson%26aufirst%3DL.%2BM.%26aulast%3DJacobson%26aufirst%3DJ.%2BR.%26aulast%3DKnafla%26aufirst%3DR.%2BT.%26aulast%3DMaye%26aufirst%3DP.%2BV.%26aulast%3DNikolaides%26aufirst%3DN.%26aulast%3DOneyemi%26aufirst%3DF.%2BY.%26aulast%3DParkhurst%26aufirst%3DG.%2BW.%26aulast%3DPecore%26aufirst%3DS.%2BE.%26aulast%3DReiter%26aufirst%3DM.%2BJ.%26aulast%3DScribner%26aufirst%3DL.%2BS.%26aulast%3DTesterman%26aufirst%3DT.%2BL.%26aulast%3DThompson%26aufirst%3DN.%2BJ.%26aulast%3DWagner%26aufirst%3DT.%2BL.%26aulast%3DWeeks%26aufirst%3DC.%2BE.%26aulast%3DAndre%26aufirst%3DJ.-D.%26aulast%3DLagain%26aufirst%3DD.%26aulast%3DBastard%26aufirst%3DY.%26aulast%3DLupu%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520structure-activity-relationships%2520of%25201H-imidazo%255B4%252C5-c%255Dquinolines%2520that%2520induce%2520interferon%2520production%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3481%26epage%3D3491%26doi%3D10.1021%2Fjm049211v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Svensson, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jemth, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desroses, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loseva, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helleday, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Högbom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stenmark, P.</span><span> </span><span class="NLM_article-title">Crystal structure of human MTH1 and the 8-oxo-dGMP product complex</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">585</span><span class="NLM_x">, </span> <span class="NLM_fpage">2617</span><span class="NLM_x">–</span> <span class="NLM_lpage">2621</span><span class="refDoi"> DOI: 10.1016/j.febslet.2011.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2Fj.febslet.2011.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=21787772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVait7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=585&publication_year=2011&pages=2617-2621&author=L.+M.+Svenssonauthor=A.+S.+Jemthauthor=M.+Desrosesauthor=O.+Losevaauthor=T.+Helledayauthor=M.+H%C3%B6gbomauthor=P.+Stenmark&title=Crystal+structure+of+human+MTH1+and+the+8-oxo-dGMP+product+complex&doi=10.1016%2Fj.febslet.2011.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of human MTH1 and the 8-oxo-dGMP product complex</span></div><div class="casAuthors">Svensson, Linda M.; Jemth, Ann-Sofie; Desroses, Matthieu; Loseva, Olga; Helleday, Thomas; Hoegbom, Martin; Stenmark, Pal</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">585</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2617-2621</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MTH1 hydrolyzes oxidized nucleotide triphosphates, thereby preventing them from being incorporated into DNA.  We here present the structures of human MTH1 (1.9 Å) and its complex with the product 8-oxo-dGMP (1.8 Å).  Unexpectedly, MTH1 binds the nucleotide in the anti conformation with no direct interaction between the 8-oxo group and the protein.  We suggest that the specificity depends on the stabilization of an enol tautomer of the 8-oxo form of dGTP.  The binding of the product induces no major structural changes.  The structures reveal the mode of nucleotide binding in MTH1 and provide the structural basis for inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9fYmwWXglD7Vg90H21EOLACvtfcHk0lgkb5or0NNEgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVait7bM&md5=711a922bab64d61641151ff0ecf6bbb1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2011.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2011.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DJemth%26aufirst%3DA.%2BS.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DH%25C3%25B6gbom%26aufirst%3DM.%26aulast%3DStenmark%26aufirst%3DP.%26atitle%3DCrystal%2520structure%2520of%2520human%2520MTH1%2520and%2520the%25208-oxo-dGMP%2520product%2520complex%26jtitle%3DFEBS%2520Lett.%26date%3D2011%26volume%3D585%26spage%3D2617%26epage%3D2621%26doi%3D10.1016%2Fj.febslet.2011.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Wei, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamitzky, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heckl, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rössle, S. C.</span><span> </span><span class="NLM_article-title">Homology modeling of human toll-like receptors TLR7, 8, and 9 ligand-binding domains</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1684</span><span class="NLM_x">–</span> <span class="NLM_lpage">1691</span><span class="refDoi"> DOI: 10.1002/pro.186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1002%2Fpro.186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=19521997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Clsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2009&pages=1684-1691&author=T.+Weiauthor=J.+Gongauthor=F.+Jamitzkyauthor=W.+M.+Hecklauthor=R.+W.+Starkauthor=S.+C.+R%C3%B6ssle&title=Homology+modeling+of+human+toll-like+receptors+TLR7%2C+8%2C+and+9+ligand-binding+domains&doi=10.1002%2Fpro.186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Homology modeling of human Toll-like receptors TLR7, 8, and 9 ligand-binding domains</span></div><div class="casAuthors">Wei, Tiandi; Gong, Jing; Jamitzky, Ferdinand; Heckl, Wolfgang M.; Stark, Robert W.; Roessle, Shaila C.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1684-1691</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Toll-like receptors (TLRs) play a key role in the innate immune system.  The TLR7, 8, and 9 compose a family of intracellularly localized TLRs that signal in response to pathogen-derived nucleic acids.  So far, there are no crystallog. structures for TLR7, 8, and 9.  For this reason, their ligand-binding mechanisms are poorly understood.  To enable first predictions of the receptor-ligand interaction sites, the authors developed three-dimensional structures for the leucine-rich repeat ectodomains of human TLR7, 8, and 9 based on homol. modeling.  To achieve a high sequence similarity between targets and templates, structural segments from all known TLR ectodomain structures (human TLR1/2/3/4 and mouse TLR3/4) were used as candidate templates for the modeling.  The resulting models support previously reported essential ligand-binding residues.  They also provide a basis to identify three potential receptor dimerization mechanisms.  Addnl., potential ligand-binding residues are identified using combined procedures.  The authors suggest further investigations of these residues through mutation expts.  This modeling approach can be extended to other members of the TLR family or other repetitive proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa9t75WYAFLbVg90H21EOLACvtfcHk0lgkb5or0NNEgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Clsrw%253D&md5=a8b8b1190ef2f70939c0d302fa5b9f4a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fpro.186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.186%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DT.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DJamitzky%26aufirst%3DF.%26aulast%3DHeckl%26aufirst%3DW.%2BM.%26aulast%3DStark%26aufirst%3DR.%2BW.%26aulast%3DR%25C3%25B6ssle%26aufirst%3DS.%2BC.%26atitle%3DHomology%2520modeling%2520of%2520human%2520toll-like%2520receptors%2520TLR7%252C%25208%252C%2520and%25209%2520ligand-binding%2520domains%26jtitle%3DProtein%2520Sci.%26date%3D2009%26volume%3D18%26spage%3D1684%26epage%3D1691%26doi%3D10.1002%2Fpro.186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Molina, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jafari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ignatushchenko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreekumar, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">341</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0lgkb5or0NNEgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Persengiev, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span> </span><span class="NLM_article-title">Nonspecific, concentration-dependent stimulation and repression of mammalian gene expression by small interfering RNAs (siRNAs)</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">18</span><span class="refDoi"> DOI: 10.1261/rna5160904</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1261%2Frna5160904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=14681580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFWguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=12-18&author=S.+P.+Persengievauthor=X.+Zhuauthor=M.+R.+Green&title=Nonspecific%2C+concentration-dependent+stimulation+and+repression+of+mammalian+gene+expression+by+small+interfering+RNAs+%28siRNAs%29&doi=10.1261%2Frna5160904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nonspecific, concentration-dependent stimulation and repression of mammalian gene expression by small interfering RNAs (siRNAs)</span></div><div class="casAuthors">Persengiev, Stephan P.; Zhu, Xiaochun; Green, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-18</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">RNA interference is an evolutionarily conserved process in which expression of a specific gene is post-transcriptionally inhibited by a small interfering RNA (siRNA), which recognizes a complementary mRNA and induces its degrdn.  Currently, RNA interference is being used extensively to inhibit expression of specific genes for exptl. and therapeutic purposes.  For applications in mammalian cells, siRNAs are designed to be <-30 base pairs to avoid nonspecific effects that arise from inducing the cellular double-stranded RNA (dsRNA)-dependent protein kinase (PKR) response.  Here we perform expression profiling in mammalian tissue-culture cells treated under std. conditions with conventional 21-bp siRNAs and find, unexpectedly, that >1000 genes involved in diverse cellular functions are nonspecifically stimulated or repressed.  The effects on gene expression are dependent upon siRNA concn. and are stable throughout the course of siRNA treatment.  Our results can be explained by previous studies showing that dsRNAs can affect multiple signaling and transcription pathways in addn. to PKR.  The potential for this widespread, nonspecific effect on mammalian gene expression must be carefully considered in the design of siRNA expts. and therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRKVQaY23i27Vg90H21EOLACvtfcHk0lhopF9vRd8jEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFWguw%253D%253D&md5=d638a7197d6dcabb9b2cdd89e1a65527</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1261%2Frna5160904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna5160904%26sid%3Dliteratum%253Aachs%26aulast%3DPersengiev%26aufirst%3DS.%2BP.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26atitle%3DNonspecific%252C%2520concentration-dependent%2520stimulation%2520and%2520repression%2520of%2520mammalian%2520gene%2520expression%2520by%2520small%2520interfering%2520RNAs%2520%2528siRNAs%2529%26jtitle%3DRNA%26date%3D2004%26volume%3D10%26spage%3D12%26epage%3D18%26doi%3D10.1261%2Frna5160904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Tarcsay, Á.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keserű, G. M.</span><span> </span><span class="NLM_article-title">Contributions of molecular properties to drug promiscuity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1789</span><span class="NLM_x">–</span> <span class="NLM_lpage">1795</span><span class="refDoi"> DOI: 10.1021/jm301514n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301514n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC3sXns1ehug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1789-1795&author=%C3%81.+Tarcsayauthor=G.+M.+Keser%C5%B1&title=Contributions+of+molecular+properties+to+drug+promiscuity&doi=10.1021%2Fjm301514n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Contributions of Molecular Properties to Drug Promiscuity</span></div><div class="casAuthors">Tarcsay, Akos; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1789-1795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In contrast to designed polypharmacol. that can result in efficient drugs for complex disorders, unintended drug promiscuity has detrimental contribution to side effects and toxicol.  Characterization of promiscuous compds. enhances the understanding of complex interaction patterns and aids the design of compds. with broader selectivity against off-targets that has a major impact on medicinal chem. outcome.  In this Miniperspective we provide insights to the effect of physicochem. parameters on promiscuity.  Information collected from recent, large-scale in vitro studies enabled us to discuss the relationships between physicochem. properties and promiscuity in detail.  In light of these data, lipophilicity and basic character have the highest influence.  On the basis of the accumulated knowledge, we propose the extensive use of pre- and postsynthesis metrics, as well as strict control of physicochem. properties during medicinal chem. optimizations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf1J_jqkK3DrVg90H21EOLACvtfcHk0lhopF9vRd8jEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXns1ehug%253D%253D&md5=c5763ee1aed897ff19e5a2344b95b9c1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm301514n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301514n%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3D%25C3%2581.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DContributions%2520of%2520molecular%2520properties%2520to%2520drug%2520promiscuity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1789%26epage%3D1795%26doi%3D10.1021%2Fjm301514n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buser-Doepner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederiksen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimshaw, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller-Fahrnow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Hagan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundström, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo-Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walpole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0liVT4IT-MDRIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Tajiri, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekiguchi, M.</span><span> </span><span class="NLM_article-title">Functional cooperation of MutT, MutM and MutY proteins in preventing mutations caused by spontaneous oxidation of guanine nucleotide in escherichia coli</span> <span class="citation_source-journal">Mutat. Res., DNA Repair</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">336</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">267</span><span class="refDoi"> DOI: 10.1016/0921-8777(94)00062-B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2F0921-8777%2894%2900062-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=7739614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADyaK2MXlsFGls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=1995&pages=257-267&author=T.+Tajiriauthor=H.+Makiauthor=M.+Sekiguchi&title=Functional+cooperation+of+MutT%2C+MutM+and+MutY+proteins+in+preventing+mutations+caused+by+spontaneous+oxidation+of+guanine+nucleotide+in+escherichia+coli&doi=10.1016%2F0921-8777%2894%2900062-B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20aR"><div class="casContent"><span class="casTitleNuber">20a</span><div class="casTitle"><span class="NLM_cas:atitle">Functional cooperation of MutT, MutM and MutY proteins in preventing mutations caused by spontaneous oxidation of guanine nucleotide in Escherichia coli</span></div><div class="casAuthors">Tajiri, Tatsurou; Maki, Hisaji; Sekiguchi, Mutsuo</div><div class="citationInfo"><span class="NLM_cas:title">Mutation Research, DNA Repair</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">257-67</span>CODEN:
                <span class="NLM_cas:coden">MRDRBE</span>;
        ISSN:<span class="NLM_cas:issn">0921-8777</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">8-Oxo-dGTP (8-oxo-7,8-dihydrodeoxyguanosine triphosphate) is a potent mutagenic substrate for DNA synthesis.  The accumulation of 8-oxo-dGTP in the nucleotide pool induces G:C→T:A transversion as well as A:T→C:G transversion, and Escherichia coli cells possess mechanisms for preventing such mutations.  The mutT gene product specifically hydrolyzes 8-oxo-dGTP to the monophosphate form while the mutM and the mutY gene products function to correct mispairs caused by incorporation of 8-oxoguanine into DNA.  From analyses of forward mutations induced in cells lacking 8-oxo-dGTPase (MutT protein) and/or repair enzymes that suppress mutations caused by 8-oxoguanine in DNA (MutM and MutY proteins), cooperative functions of these proteins in control of the spontaneous mutagenesis became evident.  In mutator strains lacking MutT and/or MutM proteins, 8-oxoguanine of DNA increased to a concn. expected from the increased rate of mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBre8HrgmQwrVg90H21EOLACvtfcHk0liVT4IT-MDRIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlsFGls7s%253D&md5=69eca7484312849d8619a6a534191c77</span></div><a href="/servlet/linkout?suffix=cit20a&amp;dbid=16384&amp;doi=10.1016%2F0921-8777%2894%2900062-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0921-8777%252894%252900062-B%26sid%3Dliteratum%253Aachs%26aulast%3DTajiri%26aufirst%3DT.%26aulast%3DMaki%26aufirst%3DH.%26aulast%3DSekiguchi%26aufirst%3DM.%26atitle%3DFunctional%2520cooperation%2520of%2520MutT%252C%2520MutM%2520and%2520MutY%2520proteins%2520in%2520preventing%2520mutations%2520caused%2520by%2520spontaneous%2520oxidation%2520of%2520guanine%2520nucleotide%2520in%2520escherichia%2520coli%26jtitle%3DMutat.%2520Res.%252C%2520DNA%2520Repair%26date%3D1995%26volume%3D336%26spage%3D257%26epage%3D267%26doi%3D10.1016%2F0921-8777%2894%2900062-B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit20b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Rotman, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuzminov, A.</span><span> </span><span class="NLM_article-title">The mutT defect does not elevate chromosomal fragmentation in escherichia coli because of the surprisingly low levels of MutM/MutY-recognized DNA modifications</span> <span class="citation_source-journal">J. Bacteriol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">189</span><span class="NLM_x">, </span> <span class="NLM_fpage">6976</span><span class="NLM_x">–</span> <span class="NLM_lpage">6988</span><span class="refDoi"> DOI: 10.1128/JB.00776-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1128%2FJB.00776-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=17616589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKmsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2007&pages=6976-6988&author=E.+Rotmanauthor=A.+Kuzminov&title=The+mutT+defect+does+not+elevate+chromosomal+fragmentation+in+escherichia+coli+because+of+the+surprisingly+low+levels+of+MutM%2FMutY-recognized+DNA+modifications&doi=10.1128%2FJB.00776-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20bR"><div class="casContent"><span class="casTitleNuber">20b</span><div class="casTitle"><span class="NLM_cas:atitle">The mutT defect does not elevate chromosomal fragmentation in Escherichia coli because of the surprisingly low levels of MutM/MutY-recognized DNA modifications</span></div><div class="casAuthors">Rotman, Ella; Kuzminov, Andrei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6976-6988</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Nucleotide pool sanitizing enzymes Dut (dUTPase), RdgB (dITPase), and MutT (8-oxo-dGTPase) of Escherichia coli hydrolyze noncanonical DNA precursors to prevent incorporation of base analogs into DNA.  Previous studies reported dramatic AT→CG mutagenesis in mutT mutants, suggesting a considerable d. of 8-oxo-G in DNA that should cause frequent excision and chromosomal fragmentation, irreparable in the absence of RecBCD-catalyzed repair and similar to the lethality of dut recBC and rdgB recBC double mutants.  In contrast, we found mutT recBC double mutants viable with no signs of chromosomal fragmentation.  Overprodn. of the MutM and MutY DNA glycosylases, both acting on DNA contg. 8-oxo-G, still yields no lethality in mutT recBC double mutants.  Plasmid DNA, extd. from mutT mutM double mutant cells and treated with MutM in vitro, shows no increased relaxation, indicating no addnl. 8-oxo-G modifications.  Our ΔmutT allele elevates the AT→CG transversion rate 27,000-fold, consistent with published reports.  However, the rate of AT→CG transversions in our mutT+ progenitor strain is some two orders of magnitude lower than in previous studies, which lowers the abs. rate of mutagenesis in ΔmutT derivs., translating into less than four 8-oxo-G modifications per genome equiv., which is too low to cause the expected effects.  Introduction of various addnl. mutations in the ΔmutT strain or treatment with oxidative agents failed to increase the mutagenesis even twofold.  We conclude that, in contrast to the previous studies, there is not enough 8-oxo-G in the DNA of mutT mutants to cause elevated excision repair that would trigger chromosomal fragmentation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriNZK-Muh6_LVg90H21EOLACvtfcHk0ljcx2TAIRXC5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKmsL3F&md5=91a568aaefcdb5a41152dc17147a50d3</span></div><a href="/servlet/linkout?suffix=cit20b&amp;dbid=16384&amp;doi=10.1128%2FJB.00776-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.00776-07%26sid%3Dliteratum%253Aachs%26aulast%3DRotman%26aufirst%3DE.%26aulast%3DKuzminov%26aufirst%3DA.%26atitle%3DThe%2520mutT%2520defect%2520does%2520not%2520elevate%2520chromosomal%2520fragmentation%2520in%2520escherichia%2520coli%2520because%2520of%2520the%2520surprisingly%2520low%2520levels%2520of%2520MutM%252FMutY-recognized%2520DNA%2520modifications%26jtitle%3DJ.%2520Bacteriol.%26date%3D2007%26volume%3D189%26spage%3D6976%26epage%3D6988%26doi%3D10.1128%2FJB.00776-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit20c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Tsuzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egashira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kura, S.</span><span> </span><span class="NLM_article-title">Analysis of MTH1 gene function in mice with targeted mutagenesis</span> <span class="citation_source-journal">Mutat. Res., Fundam. Mol. Mech. Mutagen.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">477</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">78</span><span class="refDoi"> DOI: 10.1016/S0027-5107(01)00108-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2FS0027-5107%2801%2900108-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=11376688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjslemsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2001&pages=71-78&author=T.+Tsuzukiauthor=A.+Egashiraauthor=S.+Kura&title=Analysis+of+MTH1+gene+function+in+mice+with+targeted+mutagenesis&doi=10.1016%2FS0027-5107%2801%2900108-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20cR"><div class="casContent"><span class="casTitleNuber">20c</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of MTH1 gene function in mice with targeted mutagenesis</span></div><div class="casAuthors">Tsuzuki, T.; Egashira, A.; Kura, S.</div><div class="citationInfo"><span class="NLM_cas:title">Mutation Research, Fundamental and Molecular Mechanisms of Mutagenesis</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">1,2</span>),
    <span class="NLM_cas:pages">71-78</span>CODEN:
                <span class="NLM_cas:coden">MUREAV</span>;
        ISSN:<span class="NLM_cas:issn">0027-5107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 45 refs. and emphasis on authors research.  Oxidative DNA damage is thought to contribute to carcinogenesis, aging, and neurol. degeneration.  Further, the cumulative risk of cancer increases dramatically with age in humans.  In general terms, cancer can be regarded as a degenerative disease of aging.  There is evidence for the accumulation of oxidative DNA damage with age based on studies mainly measuring an increase in 8-oxoguanine.  8-Oxo-7,8-dihydro-2'-deoxyguanosine 5'-triphosphate (8-oxo-dGTP) is formed in the nucleotide pool of a cell during normal cellular metab.  When 8-oxoguanine is incorporated into DNA causes mutation.  Organisms possess 8-oxo-dGTPase, an enzyme that specifically degrades 8-oxo-dGTP to 8-oxo-dGMP.  To analyze the function of MTH1 with 8-oxo-dGTPase activity in vivo, we generated a mouse line carrying a mutant MTH1 allele created by targeted gene disruption.  MTH1 homozygous mutant mice were found to have a phys. normal appearance, but seemed to have lost 8-oxo-dGTPase activity in liver exts.  When we examd. the susceptibility of the mutant mice to spontaneous tumorigenesis, no significant difference was obsd. in survival rate of MTH1+/+ and MTH1-/- mice.  However, pathol. examn. revealed a statistically significant difference in the incidence of tumors.  More tumors were formed in lungs, livers, and stomachs of MTH1-/- mice than in those of the wild type mice.  These studies with MTH1-null mutant mice provided an important insight into the role of this nucleotide sanitization enzyme in terms of the spontaneous tumorigenesis as well as mutagenesis caused by the oxygen-induced DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_7QRMnkJFLVg90H21EOLACvtfcHk0ljcx2TAIRXC5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjslemsr8%253D&md5=2b38b2d00ac6d0c6100cc52465312ecb</span></div><a href="/servlet/linkout?suffix=cit20c&amp;dbid=16384&amp;doi=10.1016%2FS0027-5107%2801%2900108-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0027-5107%252801%252900108-7%26sid%3Dliteratum%253Aachs%26aulast%3DTsuzuki%26aufirst%3DT.%26aulast%3DEgashira%26aufirst%3DA.%26aulast%3DKura%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520MTH1%2520gene%2520function%2520in%2520mice%2520with%2520targeted%2520mutagenesis%26jtitle%3DMutat.%2520Res.%252C%2520Fundam.%2520Mol.%2520Mech.%2520Mutagen.%26date%3D2001%26volume%3D477%26spage%3D71%26epage%3D78%26doi%3D10.1016%2FS0027-5107%2801%2900108-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20a','cit20b','cit20c']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 86 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Junjie Zou, Zhipeng Li, Shuai Liu, Chunwang Peng, Dong Fang, Xiao Wan, Zhixiong Lin, Tai-Sung Lee, Daniel P. Raleigh, Mingjun Yang, <span class="NLM_string-name hlFld-ContribAuthor">Carlos Simmerling</span>. </span><span class="cited-content_cbyCitation_article-title">Scaffold Hopping Transformations Using Auxiliary Restraints for Calculating Accurate Relative Binding Free Energies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2021,</strong> <em>17 </em>
                                    (6)
                                     , 3710-3726. <a href="https://doi.org/10.1021/acs.jctc.1c00214" title="DOI URL">https://doi.org/10.1021/acs.jctc.1c00214</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.1c00214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.1c00214%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DScaffold%252BHopping%252BTransformations%252BUsing%252BAuxiliary%252BRestraints%252Bfor%252BCalculating%252BAccurate%252BRelative%252BBinding%252BFree%252BEnergies%26aulast%3DZou%26aufirst%3DJunjie%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01032021%26date%3D24052021%26volume%3D17%26issue%3D6%26spage%3D3710%26epage%3D3726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Julie Farand, Jeffrey E. Kropf, Peter Blomgren, Jianjun Xu, Aaron C. Schmitt, Zachary E. Newby, Ting Wang, Eisuke Murakami, Ona Barauskas, Jawahar Sudhamsu, Joy Y. Feng, Anita Niedziela-Majka, Brian E. Schultz, Karen Schwartz, Serge Viatchenko-Karpinski, Dmytro Kornyeyev, Adam Kashishian, Peidong Fan, Xiaowu Chen, Eric B. Lansdon, Michael O. Ports, Kevin S. Currie, William J. Watkins, <span class="NLM_string-name hlFld-ContribAuthor">Gregory T. Notte</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent and Selective MTH1 Inhibitors for Oncology: Enabling Rapid Target (In)Validation. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (3)
                                     , 358-364. <a href="https://doi.org/10.1021/acsmedchemlett.9b00420" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00420</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00420%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BPotent%252Band%252BSelective%252BMTH1%252BInhibitors%252Bfor%252BOncology%25253A%252BEnabling%252BRapid%252BTarget%252B%252528In%252529Validation%26aulast%3DFarand%26aufirst%3DJulie%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12092019%26date%3D12112019%26date%3D19112019%26volume%3D11%26issue%3D3%26spage%3D358%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yu-ki Tahara, Anna M. Kietrys, Marian Hebenbrock, Yujeong Lee, David L. Wilson, <span class="NLM_string-name hlFld-ContribAuthor">Eric T. Kool</span>. </span><span class="cited-content_cbyCitation_article-title">Dual Inhibitors of 8-Oxoguanine Surveillance by OGG1 and NUDT1. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (12)
                                     , 2606-2615. <a href="https://doi.org/10.1021/acschembio.9b00490" title="DOI URL">https://doi.org/10.1021/acschembio.9b00490</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00490%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDual%252BInhibitors%252Bof%252B8-Oxoguanine%252BSurveillance%252Bby%252BOGG1%252Band%252BNUDT1%26aulast%3DTahara%26aufirst%3DYu-ki%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D18062019%26date%3D16102019%26date%3D29102019%26date%3D17102019%26volume%3D14%26issue%3D12%26spage%3D2606%26epage%3D2615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fredrik Rahm, Jenny Viklund, Lionel Trésaugues, Manuel Ellermann, Anja Giese, Ulrika Ericsson, Rickard Forsblom, Tobias Ginman, Judith Günther, Kenth Hallberg, Johan Lindström, Lars Boukharta Persson, Camilla Silvander, Antoine Talagas, Laura Díaz-Sáez, Oleg Fedorov, Kilian V. M. Huber, Ioanna Panagakou, Paulina Siejka, Mátyás Gorjánácz, Marcus Bauser, <span class="NLM_string-name hlFld-ContribAuthor">Martin Andersson</span>. </span><span class="cited-content_cbyCitation_article-title">Creation of a Novel Class of Potent and Selective MutT Homologue 1 (MTH1) Inhibitors Using Fragment-Based Screening and Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (6)
                                     , 2533-2551. <a href="https://doi.org/10.1021/acs.jmedchem.7b01884" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01884</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01884%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCreation%252Bof%252Ba%252BNovel%252BClass%252Bof%252BPotent%252Band%252BSelective%252BMutT%252BHomologue%252B1%252B%252528MTH1%252529%252BInhibitors%252BUsing%252BFragment-Based%252BScreening%252Band%252BStructure-Based%252BDrug%252BDesign%26aulast%3DRahm%26aufirst%3DFredrik%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D21122017%26date%3D07032018%26date%3D27022018%26volume%3D61%26issue%3D6%26spage%3D2533%26epage%3D2551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohit  Narwal</span>, <span class="hlFld-ContribAuthor ">Ann-Sofie  Jemth</span>, <span class="hlFld-ContribAuthor ">Robert  Gustafsson</span>, <span class="hlFld-ContribAuthor ">Ingrid  Almlöf</span>, <span class="hlFld-ContribAuthor ">Ulrika  Warpman Berglund</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, and <span class="hlFld-ContribAuthor ">Pål  Stenmark</span>  . </span><span class="cited-content_cbyCitation_article-title">Crystal Structures and Inhibitor Interactions of Mouse and Dog MTH1 Reveal Species-Specific Differences in Affinity. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2018,</strong> <em>57 </em>
                                    (5)
                                     , 593-603. <a href="https://doi.org/10.1021/acs.biochem.7b01163" title="DOI URL">https://doi.org/10.1021/acs.biochem.7b01163</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.7b01163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.7b01163%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DCrystal%252BStructures%252Band%252BInhibitor%252BInteractions%252Bof%252BMouse%252Band%252BDog%252BMTH1%252BReveal%252BSpecies-Specific%252BDifferences%252Bin%252BAffinity%26aulast%3DNarwal%26aufirst%3DMohit%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D16112017%26date%3D22122017%26date%3D16012018%26date%3D06022018%26date%3D27122017%26volume%3D57%26issue%3D5%26spage%3D593%26epage%3D603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haoyu S.  Yu</span>, <span class="hlFld-ContribAuthor ">Yuqing  Deng</span>, <span class="hlFld-ContribAuthor ">Yujie  Wu</span>, <span class="hlFld-ContribAuthor ">Dan  Sindhikara</span>, <span class="hlFld-ContribAuthor ">Amy R.  Rask</span>, <span class="hlFld-ContribAuthor ">Takayuki  Kimura</span>, <span class="hlFld-ContribAuthor ">Robert  Abel</span>, and <span class="hlFld-ContribAuthor ">Lingle  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">Accurate and Reliable Prediction of the Binding Affinities of Macrocycles to Their Protein Targets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2017,</strong> <em>13 </em>
                                    (12)
                                     , 6290-6300. <a href="https://doi.org/10.1021/acs.jctc.7b00885" title="DOI URL">https://doi.org/10.1021/acs.jctc.7b00885</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.7b00885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.7b00885%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DAccurate%252Band%252BReliable%252BPrediction%252Bof%252Bthe%252BBinding%252BAffinities%252Bof%252BMacrocycles%252Bto%252BTheir%252BProtein%252BTargets%26aulast%3DYu%26aufirst%3DHaoyu%2BS.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D18082017%26date%3D30112017%26date%3D12122017%26date%3D09112017%26volume%3D13%26issue%3D12%26spage%3D6290%26epage%3D6300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Axel  Rudling</span>, <span class="hlFld-ContribAuthor ">Robert  Gustafsson</span>, <span class="hlFld-ContribAuthor ">Ingrid  Almlöf</span>, <span class="hlFld-ContribAuthor ">Evert  Homan</span>, <span class="hlFld-ContribAuthor ">Martin  Scobie</span>, <span class="hlFld-ContribAuthor ">Ulrika  Warpman Berglund</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, <span class="hlFld-ContribAuthor ">Pål  Stenmark</span>, and <span class="hlFld-ContribAuthor ">Jens  Carlsson</span>  . </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Discovery and Optimization of Enzyme Inhibitors by Docking of Commercial Chemical Space. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (19)
                                     , 8160-8169. <a href="https://doi.org/10.1021/acs.jmedchem.7b01006" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01006%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-Based%252BDiscovery%252Band%252BOptimization%252Bof%252BEnzyme%252BInhibitors%252Bby%252BDocking%252Bof%252BCommercial%252BChemical%252BSpace%26aulast%3DRudling%26aufirst%3DAxel%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D17072017%26date%3D03102017%26date%3D12102017%26date%3D20092017%26volume%3D60%26issue%3D19%26spage%3D8160%26epage%3D8169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuel  Ellermann</span>, <span class="hlFld-ContribAuthor ">Ashley  Eheim</span>, <span class="hlFld-ContribAuthor ">Fredrik  Rahm</span>, <span class="hlFld-ContribAuthor ">Jenny  Viklund</span>, <span class="hlFld-ContribAuthor ">Judith  Guenther</span>, <span class="hlFld-ContribAuthor ">Martin  Andersson</span>, <span class="hlFld-ContribAuthor ">Ulrika  Ericsson</span>, <span class="hlFld-ContribAuthor ">Rickard  Forsblom</span>, <span class="hlFld-ContribAuthor ">Tobias  Ginman</span>, <span class="hlFld-ContribAuthor ">Johan  Lindström</span>, <span class="hlFld-ContribAuthor ">Camilla  Silvander</span>, <span class="hlFld-ContribAuthor ">Lionel  Trésaugues</span>, <span class="hlFld-ContribAuthor ">Anja  Giese</span>, <span class="hlFld-ContribAuthor ">Stefanie  Bunse</span>, <span class="hlFld-ContribAuthor ">Roland  Neuhaus</span>, <span class="hlFld-ContribAuthor ">Jörg  Weiske</span>, <span class="hlFld-ContribAuthor ">Maria  Quanz</span>, <span class="hlFld-ContribAuthor ">Andrea  Glasauer</span>, <span class="hlFld-ContribAuthor ">Katrin  Nowak-Reppel</span>, <span class="hlFld-ContribAuthor ">Benjamin  Bader</span>, <span class="hlFld-ContribAuthor ">Horst  Irlbacher</span>, <span class="hlFld-ContribAuthor ">Hanna  Meyer</span>, <span class="hlFld-ContribAuthor ">Nina  Queisser</span>, <span class="hlFld-ContribAuthor ">Marcus  Bauser</span>, <span class="hlFld-ContribAuthor ">Andrea  Haegebarth</span>, and <span class="hlFld-ContribAuthor ">Mátyás  Gorjánácz</span>  . </span><span class="cited-content_cbyCitation_article-title">Novel Class of Potent and Cellularly Active Inhibitors Devalidates MTH1 as Broad-Spectrum Cancer Target. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2017,</strong> <em>12 </em>
                                    (8)
                                     , 1986-1992. <a href="https://doi.org/10.1021/acschembio.7b00370" title="DOI URL">https://doi.org/10.1021/acschembio.7b00370</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.7b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.7b00370%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DNovel%252BClass%252Bof%252BPotent%252Band%252BCellularly%252BActive%252BInhibitors%252BDevalidates%252BMTH1%252Bas%252BBroad-Spectrum%252BCancer%252BTarget%26aulast%3DEllermann%26aufirst%3DManuel%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D05052017%26date%3D05072017%26date%3D12072017%26date%3D18082017%26date%3D05072017%26volume%3D12%26issue%3D8%26spage%3D1986%26epage%3D1992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ignacio  Aliagas</span>, <span class="hlFld-ContribAuthor ">Raphaëlle  Berger</span>, <span class="hlFld-ContribAuthor ">Kristin  Goldberg</span>, <span class="hlFld-ContribAuthor ">Rachel T.  Nishimura</span>, <span class="hlFld-ContribAuthor ">John  Reilly</span>, <span class="hlFld-ContribAuthor ">Paul  Richardson</span>, <span class="hlFld-ContribAuthor ">Daniel  Richter</span>, <span class="hlFld-ContribAuthor ">Edward C.  Sherer</span>, <span class="hlFld-ContribAuthor ">Brian A.  Sparling</span>, and <span class="hlFld-ContribAuthor ">Marian C.  Bryan</span>  . </span><span class="cited-content_cbyCitation_article-title">Sustainable Practices in Medicinal Chemistry Part 2: Green by Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (14)
                                     , 5955-5968. <a href="https://doi.org/10.1021/acs.jmedchem.6b01837" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01837%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSustainable%252BPractices%252Bin%252BMedicinal%252BChemistry%252BPart%252B2%25253A%252BGreen%252Bby%252BDesign%26aulast%3DAliagas%26aufirst%3DIgnacio%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D14122016%26date%3D12042017%26date%3D27072017%26date%3D04042017%26volume%3D60%26issue%3D14%26spage%3D5955%26epage%3D5968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pavel  Ryabchuk</span>, <span class="hlFld-ContribAuthor ">Jonathon P.  Matheny</span>, <span class="hlFld-ContribAuthor ">Marina  Rubina</span>, and <span class="hlFld-ContribAuthor ">Michael  Rubin</span>  . </span><span class="cited-content_cbyCitation_article-title">Templated Assembly of Chiral Medium-Sized Cyclic Ethers via 8-endo-trig Nucleophilic Cyclization of Cyclopropenes. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2016,</strong> <em>18 </em>
                                    (24)
                                     , 6272-6275. <a href="https://doi.org/10.1021/acs.orglett.6b03068" title="DOI URL">https://doi.org/10.1021/acs.orglett.6b03068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.6b03068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.6b03068%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DTemplated%252BAssembly%252Bof%252BChiral%252BMedium-Sized%252BCyclic%252BEthers%252Bvia%252B8-endo-trig%252BNucleophilic%252BCyclization%252Bof%252BCyclopropenes%26aulast%3DRyabchuk%26aufirst%3DPavel%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D12102016%26date%3D02122016%26date%3D16122016%26volume%3D18%26issue%3D24%26spage%3D6272%26epage%3D6275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Safnas F.  AbdulSalam</span>, <span class="hlFld-ContribAuthor ">Fathima Shazna  Thowfeik</span>, and <span class="hlFld-ContribAuthor ">Edward J.  Merino</span>  . </span><span class="cited-content_cbyCitation_article-title">Excessive Reactive Oxygen Species and Exotic DNA Lesions as an Exploitable Liability. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2016,</strong> <em>55 </em>
                                    (38)
                                     , 5341-5352. <a href="https://doi.org/10.1021/acs.biochem.6b00703" title="DOI URL">https://doi.org/10.1021/acs.biochem.6b00703</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.6b00703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.6b00703%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DExcessive%252BReactive%252BOxygen%252BSpecies%252Band%252BExotic%252BDNA%252BLesions%252Bas%252Ban%252BExploitable%252BLiability%26aulast%3DAbdulSalam%26aufirst%3DSafnas%2BF.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D11072016%26date%3D18082016%26date%3D13092016%26date%3D27092016%26date%3D01092016%26volume%3D55%26issue%3D38%26spage%3D5341%26epage%3D5352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Debin  Ji</span>, <span class="hlFld-ContribAuthor ">Andrew A.  Beharry</span>, <span class="hlFld-ContribAuthor ">James M.  Ford</span>, and <span class="hlFld-ContribAuthor ">Eric T.  Kool</span>  . </span><span class="cited-content_cbyCitation_article-title">A Chimeric ATP-Linked Nucleotide Enables Luminescence Signaling of Damage Surveillance by MTH1, a Cancer Target. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2016,</strong> <em>138 </em>
                                    (29)
                                     , 9005-9008. <a href="https://doi.org/10.1021/jacs.6b02895" title="DOI URL">https://doi.org/10.1021/jacs.6b02895</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.6b02895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.6b02895%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DA%252BChimeric%252BATP-Linked%252BNucleotide%252BEnables%252BLuminescence%252BSignaling%252Bof%252BDamage%252BSurveillance%252Bby%252BMTH1%25252C%252Ba%252BCancer%252BTarget%26aulast%3DJi%26aufirst%3DDebin%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D19032016%26date%3D18072016%26date%3D27072016%26date%3D14072016%26volume%3D138%26issue%3D29%26spage%3D9005%26epage%3D9008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gianluca  Papeo</span>  . </span><span class="cited-content_cbyCitation_article-title">MutT Homolog 1 (MTH1): The Silencing of a Target. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (6)
                                     , 2343-2345. <a href="https://doi.org/10.1021/acs.jmedchem.6b00283" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00283</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00283%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMutT%252BHomolog%252B1%252B%252528MTH1%252529%25253A%252BThe%252BSilencing%252Bof%252Ba%252BTarget%26aulast%3DPapeo%26aufirst%3DGianluca%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D22022016%26date%3D29022016%26date%3D24032016%26volume%3D59%26issue%3D6%26spage%3D2343%26epage%3D2345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Piotr  Raubo</span>, <span class="hlFld-ContribAuthor ">Rodrigo J.  Carbajo</span>, <span class="hlFld-ContribAuthor ">William  McCoull</span>, <span class="hlFld-ContribAuthor ">Joanna  Raubo</span>, <span class="hlFld-ContribAuthor ">Morgan  Thomas</span>. </span><span class="cited-content_cbyCitation_article-title">Diversity-orientated synthesis of macrocyclic heterocycles using a double S
              N
              Ar approach. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>19 </em>
                                    (28)
                                     , 6274-6290. <a href="https://doi.org/10.1039/D1OB00612F" title="DOI URL">https://doi.org/10.1039/D1OB00612F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1OB00612F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1OB00612F%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DDiversity-orientated%252Bsynthesis%252Bof%252Bmacrocyclic%252Bheterocycles%252Busing%252Ba%252Bdouble%252BS%252BN%252BAr%252Bapproach%26aulast%3DRaubo%26aufirst%3DPiotr%26date%3D2021%26date%3D2021%26volume%3D19%26issue%3D28%26spage%3D6274%26epage%3D6290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mu-Rong  Chao</span>, <span class="hlFld-ContribAuthor ">Mark D.  Evans</span>, <span class="hlFld-ContribAuthor ">Chiung-Wen  Hu</span>, <span class="hlFld-ContribAuthor ">Yunhee  Ji</span>, <span class="hlFld-ContribAuthor ">Peter  Møller</span>, <span class="hlFld-ContribAuthor ">Pavel  Rossner</span>, <span class="hlFld-ContribAuthor ">Marcus S.  Cooke</span>. </span><span class="cited-content_cbyCitation_article-title">Biomarkers of nucleic acid oxidation – A summary state-of-the-art. </span><span class="cited-content_cbyCitation_journal-name">Redox Biology</span><span> <strong>2021,</strong> <em>42 </em>, 101872. <a href="https://doi.org/10.1016/j.redox.2021.101872" title="DOI URL">https://doi.org/10.1016/j.redox.2021.101872</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.redox.2021.101872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.redox.2021.101872%26sid%3Dliteratum%253Aachs%26jtitle%3DRedox%2520Biology%26atitle%3DBiomarkers%252Bof%252Bnucleic%252Bacid%252Boxidation%252B%2525E2%252580%252593%252BA%252Bsummary%252Bstate-of-the-art%26aulast%3DChao%26aufirst%3DMu-Rong%26date%3D2021%26volume%3D42%26spage%3D101872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cheng  Peng</span>, <span class="hlFld-ContribAuthor ">Yu-Hsuan  Li</span>, <span class="hlFld-ContribAuthor ">Chao-Wu  Yu</span>, <span class="hlFld-ContribAuthor ">Ze-Hua  Cheng</span>, <span class="hlFld-ContribAuthor ">Jia-Rong  Liu</span>, <span class="hlFld-ContribAuthor ">Jui-Ling  Hsu</span>, <span class="hlFld-ContribAuthor ">Ling-Wei  Hsin</span>, <span class="hlFld-ContribAuthor ">Chen-Tsung  Huang</span>, <span class="hlFld-ContribAuthor ">Hsueh-Fen  Juan</span>, <span class="hlFld-ContribAuthor ">Ji-Wang  Chern</span>, <span class="hlFld-ContribAuthor ">Yi-Sheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitor development of MTH1 via high-throughput screening with fragment based library and MTH1 substrate binding cavity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>110 </em>, 104813. <a href="https://doi.org/10.1016/j.bioorg.2021.104813" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104813</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104813%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DInhibitor%252Bdevelopment%252Bof%252BMTH1%252Bvia%252Bhigh-throughput%252Bscreening%252Bwith%252Bfragment%252Bbased%252Blibrary%252Band%252BMTH1%252Bsubstrate%252Bbinding%252Bcavity%26aulast%3DPeng%26aufirst%3DCheng%26date%3D2021%26volume%3D110%26spage%3D104813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ling  Zhang</span>, <span class="hlFld-ContribAuthor ">Laura  Misiara</span>, <span class="hlFld-ContribAuthor ">Govindi J.  Samaranayake</span>, <span class="hlFld-ContribAuthor ">Nisha  Sharma</span>, <span class="hlFld-ContribAuthor ">Dao M.  Nguyen</span>, <span class="hlFld-ContribAuthor ">Yu-Ki  Tahara</span>, <span class="hlFld-ContribAuthor ">Eric T.  Kool</span>, <span class="hlFld-ContribAuthor ">Priyamvada  Rai</span>. </span><span class="cited-content_cbyCitation_article-title">OGG1 co-inhibition antagonizes the tumor-inhibitory effects of targeting MTH1. </span><span class="cited-content_cbyCitation_journal-name">Redox Biology</span><span> <strong>2021,</strong> <em>40 </em>, 101848. <a href="https://doi.org/10.1016/j.redox.2020.101848" title="DOI URL">https://doi.org/10.1016/j.redox.2020.101848</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.redox.2020.101848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.redox.2020.101848%26sid%3Dliteratum%253Aachs%26jtitle%3DRedox%2520Biology%26atitle%3DOGG1%252Bco-inhibition%252Bantagonizes%252Bthe%252Btumor-inhibitory%252Beffects%252Bof%252Btargeting%252BMTH1%26aulast%3DZhang%26aufirst%3DLing%26date%3D2021%26volume%3D40%26spage%3D101848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hui  Shi</span>, <span class="hlFld-ContribAuthor ">Ren  Ishikawa</span>, <span class="hlFld-ContribAuthor ">Choon Han  Heh</span>, <span class="hlFld-ContribAuthor ">Shigeki  Sasaki</span>, <span class="hlFld-ContribAuthor ">Yosuke  Taniguchi</span>. </span><span class="cited-content_cbyCitation_article-title">Development of MTH1-Binding Nucleotide Analogs Based on 7,8-Dihalogenated 7-Deaza-dG Derivatives. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (3)
                                     , 1274. <a href="https://doi.org/10.3390/ijms22031274" title="DOI URL">https://doi.org/10.3390/ijms22031274</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22031274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22031274%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DDevelopment%252Bof%252BMTH1-Binding%252BNucleotide%252BAnalogs%252BBased%252Bon%252B7%25252C8-Dihalogenated%252B7-Deaza-dG%252BDerivatives%26aulast%3DShi%26aufirst%3DHui%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D3%26spage%3D1274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yizhen  Yin</span>, <span class="hlFld-ContribAuthor ">Fener  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting human MutT homolog 1 (MTH1) for cancer eradication: current progress and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>10 </em>
                                    (12)
                                     , 2259-2271. <a href="https://doi.org/10.1016/j.apsb.2020.02.012" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.02.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.02.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.02.012%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DTargeting%252Bhuman%252BMutT%252Bhomolog%252B1%252B%252528MTH1%252529%252Bfor%252Bcancer%252Beradication%25253A%252Bcurrent%252Bprogress%252Band%252Bperspectives%26aulast%3DYin%26aufirst%3DYizhen%26date%3D2020%26volume%3D10%26issue%3D12%26spage%3D2259%26epage%3D2271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katarzyna D.  Arczewska</span>, <span class="hlFld-ContribAuthor ">Wanda  Krasuska</span>, <span class="hlFld-ContribAuthor ">Anna  Stachurska</span>, <span class="hlFld-ContribAuthor ">Kamila  Karpińska</span>, <span class="hlFld-ContribAuthor ">Justyna  Sikorska</span>, <span class="hlFld-ContribAuthor ">Mirosław  Kiedrowski</span>, <span class="hlFld-ContribAuthor ">Dariusz  Lange</span>, <span class="hlFld-ContribAuthor ">Tomasz  Stępień</span>, <span class="hlFld-ContribAuthor ">Barbara  Czarnocka</span>. </span><span class="cited-content_cbyCitation_article-title">hMTH1 and GPX1 expression in human thyroid tissue is interrelated to prevent oxidative DNA damage. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2020,</strong> <em>95 </em>, 102954. <a href="https://doi.org/10.1016/j.dnarep.2020.102954" title="DOI URL">https://doi.org/10.1016/j.dnarep.2020.102954</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2020.102954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2020.102954%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DhMTH1%252Band%252BGPX1%252Bexpression%252Bin%252Bhuman%252Bthyroid%252Btissue%252Bis%252Binterrelated%252Bto%252Bprevent%252Boxidative%252BDNA%252Bdamage%26aulast%3DArczewska%26aufirst%3DKatarzyna%2BD.%26date%3D2020%26volume%3D95%26spage%3D102954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huaping  Chen</span>, <span class="hlFld-ContribAuthor ">Sadia  Afrin</span>, <span class="hlFld-ContribAuthor ">Yingqiu  Guo</span>, <span class="hlFld-ContribAuthor ">Wenhua  Chu</span>, <span class="hlFld-ContribAuthor ">Tammie L.S.  Benzinger</span>, <span class="hlFld-ContribAuthor ">Buck E.  Rogers</span>, <span class="hlFld-ContribAuthor ">Joel R.  Garbow</span>, <span class="hlFld-ContribAuthor ">Joel S.  Perlmutter</span>, <span class="hlFld-ContribAuthor ">Dong  Zhou</span>, <span class="hlFld-ContribAuthor ">Jinbin  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Radiolabeled 6-(2, 3-Dichlorophenyl)-N4-methylpyrimidine-2, 4-diamine (TH287): A Potential Radiotracer for Measuring and Imaging MTH1. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (22)
                                     , 8860. <a href="https://doi.org/10.3390/ijms21228860" title="DOI URL">https://doi.org/10.3390/ijms21228860</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21228860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21228860%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DRadiolabeled%252B6-%2525282%25252C%252B3-Dichlorophenyl%252529-N4-methylpyrimidine-2%25252C%252B4-diamine%252B%252528TH287%252529%25253A%252BA%252BPotential%252BRadiotracer%252Bfor%252BMeasuring%252Band%252BImaging%252BMTH1%26aulast%3DChen%26aufirst%3DHuaping%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D22%26spage%3D8860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ann  Lin</span>, <span class="hlFld-ContribAuthor ">Jason M.  Sheltzer</span>. </span><span class="cited-content_cbyCitation_article-title">Discovering and validating cancer genetic dependencies: approaches and pitfalls. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Genetics</span><span> <strong>2020,</strong> <em>21 </em>
                                    (11)
                                     , 671-682. <a href="https://doi.org/10.1038/s41576-020-0247-7" title="DOI URL">https://doi.org/10.1038/s41576-020-0247-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41576-020-0247-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41576-020-0247-7%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Genetics%26atitle%3DDiscovering%252Band%252Bvalidating%252Bcancer%252Bgenetic%252Bdependencies%25253A%252Bapproaches%252Band%252Bpitfalls%26aulast%3DLin%26aufirst%3DAnn%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D11%26spage%3D671%26epage%3D682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sean G.  Rudd</span>, <span class="hlFld-ContribAuthor ">Helge  Gad</span>, <span class="hlFld-ContribAuthor ">Kumar  Sanjiv</span>, <span class="hlFld-ContribAuthor ">Nuno  Amaral</span>, <span class="hlFld-ContribAuthor ">Anna  Hagenkort</span>, <span class="hlFld-ContribAuthor ">Petra  Groth</span>, <span class="hlFld-ContribAuthor ">Cecilia E.  Ström</span>, <span class="hlFld-ContribAuthor ">Oliver  Mortusewicz</span>, <span class="hlFld-ContribAuthor ">Ulrika Warpman  Berglund</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>. </span><span class="cited-content_cbyCitation_article-title">MTH1 Inhibitor TH588 Disturbs Mitotic Progression and Induces Mitosis-Dependent Accumulation of Genomic 8-oxodG. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2020,</strong> <em>80 </em>
                                    (17)
                                     , 3530-3541. <a href="https://doi.org/10.1158/0008-5472.CAN-19-0883" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-19-0883</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-19-0883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-19-0883%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DMTH1%252BInhibitor%252BTH588%252BDisturbs%252BMitotic%252BProgression%252Band%252BInduces%252BMitosis-Dependent%252BAccumulation%252Bof%252BGenomic%252B8-oxodG%26aulast%3DRudd%26aufirst%3DSean%2BG.%26date%3D2020%26date%3D2020%26volume%3D80%26issue%3D17%26spage%3D3530%26epage%3D3541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ishani  Das</span>, <span class="hlFld-ContribAuthor ">Helge  Gad</span>, <span class="hlFld-ContribAuthor ">Lars  Bräutigam</span>, <span class="hlFld-ContribAuthor ">Linda  Pudelko</span>, <span class="hlFld-ContribAuthor ">Rainer  Tuominen</span>, <span class="hlFld-ContribAuthor ">Veronica  Höiom</span>, <span class="hlFld-ContribAuthor ">Ingrid  Almlöf</span>, <span class="hlFld-ContribAuthor ">Varshni  Rajagopal</span>, <span class="hlFld-ContribAuthor ">Johan  Hansson</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, <span class="hlFld-ContribAuthor ">Suzanne  Egyházi Brage</span>, <span class="hlFld-ContribAuthor ">Ulrika  Warpman Berglund</span>. </span><span class="cited-content_cbyCitation_article-title">AXL and CAV-1 play a role for MTH1 inhibitor TH1579 sensitivity in cutaneous malignant melanoma. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Differentiation</span><span> <strong>2020,</strong> <em>27 </em>
                                    (7)
                                     , 2081-2098. <a href="https://doi.org/10.1038/s41418-019-0488-1" title="DOI URL">https://doi.org/10.1038/s41418-019-0488-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41418-019-0488-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41418-019-0488-1%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Differentiation%26atitle%3DAXL%252Band%252BCAV-1%252Bplay%252Ba%252Brole%252Bfor%252BMTH1%252Binhibitor%252BTH1579%252Bsensitivity%252Bin%252Bcutaneous%252Bmalignant%252Bmelanoma%26aulast%3DDas%26aufirst%3DIshani%26date%3D2020%26date%3D2020%26volume%3D27%26issue%3D7%26spage%3D2081%26epage%3D2098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Irene  Barguilla</span>, <span class="hlFld-ContribAuthor ">Gabriela  Barszczewska</span>, <span class="hlFld-ContribAuthor ">Balasubramanyam  Annangi</span>, <span class="hlFld-ContribAuthor ">Josefa  Domenech</span>, <span class="hlFld-ContribAuthor ">Antonia  Velázquez</span>, <span class="hlFld-ContribAuthor ">Ricard  Marcos</span>, <span class="hlFld-ContribAuthor ">Alba  Hernández</span>. </span><span class="cited-content_cbyCitation_article-title">MTH1 is involved in the toxic and carcinogenic long-term effects induced by zinc oxide and cobalt nanoparticles. </span><span class="cited-content_cbyCitation_journal-name">Archives of Toxicology</span><span> <strong>2020,</strong> <em>94 </em>
                                    (6)
                                     , 1973-1984. <a href="https://doi.org/10.1007/s00204-020-02737-y" title="DOI URL">https://doi.org/10.1007/s00204-020-02737-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00204-020-02737-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00204-020-02737-y%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Toxicology%26atitle%3DMTH1%252Bis%252Binvolved%252Bin%252Bthe%252Btoxic%252Band%252Bcarcinogenic%252Blong-term%252Beffects%252Binduced%252Bby%252Bzinc%252Boxide%252Band%252Bcobalt%252Bnanoparticles%26aulast%3DBarguilla%26aufirst%3DIrene%26date%3D2020%26date%3D2020%26volume%3D94%26issue%3D6%26spage%3D1973%26epage%3D1984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maurice  Michel</span>, <span class="hlFld-ContribAuthor ">Evert J.  Homan</span>, <span class="hlFld-ContribAuthor ">Elisée  Wiita</span>, <span class="hlFld-ContribAuthor ">Kia  Pedersen</span>, <span class="hlFld-ContribAuthor ">Ingrid  Almlöf</span>, <span class="hlFld-ContribAuthor ">Anna-Lena  Gustavsson</span>, <span class="hlFld-ContribAuthor ">Thomas  Lundbäck</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, <span class="hlFld-ContribAuthor ">Ulrika  Warpman Berglund</span>. </span><span class="cited-content_cbyCitation_article-title">In silico Druggability Assessment of the NUDIX Hydrolase Protein Family as a Workflow for Target Prioritization. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.00443" title="DOI URL">https://doi.org/10.3389/fchem.2020.00443</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.00443%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DIn%252Bsilico%252BDruggability%252BAssessment%252Bof%252Bthe%252BNUDIX%252BHydrolase%252BProtein%252BFamily%252Bas%252Ba%252BWorkflow%252Bfor%252BTarget%252BPrioritization%26aulast%3DMichel%26aufirst%3DMaurice%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emma Rose  Scaletti</span>, <span class="hlFld-ContribAuthor ">Karl S.  Vallin</span>, <span class="hlFld-ContribAuthor ">Lars  Bräutigam</span>, <span class="hlFld-ContribAuthor ">Antonio  Sarno</span>, <span class="hlFld-ContribAuthor ">Ulrika  Warpman Berglund</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, <span class="hlFld-ContribAuthor ">Pål  Stenmark</span>, <span class="hlFld-ContribAuthor ">Ann-Sofie  Jemth</span>. </span><span class="cited-content_cbyCitation_article-title">MutT homologue 1 (MTH1) removes N6-methyl-dATP from the dNTP pool. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2020,</strong> <em>295 </em>
                                    (15)
                                     , 4761-4772. <a href="https://doi.org/10.1074/jbc.RA120.012636" title="DOI URL">https://doi.org/10.1074/jbc.RA120.012636</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA120.012636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA120.012636%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DMutT%252Bhomologue%252B1%252B%252528MTH1%252529%252Bremoves%252BN6-methyl-dATP%252Bfrom%252Bthe%252BdNTP%252Bpool%26aulast%3DScaletti%26aufirst%3DEmma%2BRose%26date%3D2020%26volume%3D295%26issue%3D15%26spage%3D4761%26epage%3D4772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brice  Moukengue</span>, <span class="hlFld-ContribAuthor ">Hannah K  Brown</span>, <span class="hlFld-ContribAuthor ">Céline  Charrier</span>, <span class="hlFld-ContribAuthor ">Séverine  Battaglia</span>, <span class="hlFld-ContribAuthor ">Marc  Baud'huin</span>, <span class="hlFld-ContribAuthor ">Thibaut  Quillard</span>, <span class="hlFld-ContribAuthor ">Therese M  Pham</span>, <span class="hlFld-ContribAuthor ">Ioannis S  Pateras</span>, <span class="hlFld-ContribAuthor ">Vassilis G  Gorgoulis</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, <span class="hlFld-ContribAuthor ">Dominique  Heymann</span>, <span class="hlFld-ContribAuthor ">Ulrika Warpman  Berglund</span>, <span class="hlFld-ContribAuthor ">Benjamin  Ory</span>, <span class="hlFld-ContribAuthor ">Francois  Lamoureux</span>. </span><span class="cited-content_cbyCitation_article-title">TH1579, MTH1 inhibitor, delays tumour growth and inhibits metastases development in osteosarcoma model. </span><span class="cited-content_cbyCitation_journal-name">EBioMedicine</span><span> <strong>2020,</strong> <em>53 </em>, 102704. <a href="https://doi.org/10.1016/j.ebiom.2020.102704" title="DOI URL">https://doi.org/10.1016/j.ebiom.2020.102704</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ebiom.2020.102704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ebiom.2020.102704%26sid%3Dliteratum%253Aachs%26jtitle%3DEBioMedicine%26atitle%3DTH1579%25252C%252BMTH1%252Binhibitor%25252C%252Bdelays%252Btumour%252Bgrowth%252Band%252Binhibits%252Bmetastases%252Bdevelopment%252Bin%252Bosteosarcoma%252Bmodel%26aulast%3DMoukengue%26aufirst%3DBrice%26date%3D2020%26volume%3D53%26spage%3D102704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Govindi J.  Samaranayake</span>, <span class="hlFld-ContribAuthor ">Clara I.  Troccoli</span>, <span class="hlFld-ContribAuthor ">Ling  Zhang</span>, <span class="hlFld-ContribAuthor ">Mai  Huynh</span>, <span class="hlFld-ContribAuthor ">Christina J.  Jayaraj</span>, <span class="hlFld-ContribAuthor ">Debin  Ji</span>, <span class="hlFld-ContribAuthor ">Lisa  McPherson</span>, <span class="hlFld-ContribAuthor ">Yoshiyuki  Onishi</span>, <span class="hlFld-ContribAuthor ">Dao M.  Nguyen</span>, <span class="hlFld-ContribAuthor ">David J.  Robbins</span>, <span class="hlFld-ContribAuthor ">Mahsa  Karbaschi</span>, <span class="hlFld-ContribAuthor ">Marcus S.  Cooke</span>, <span class="hlFld-ContribAuthor ">Antonio  Barrientos</span>, <span class="hlFld-ContribAuthor ">Eric T.  Kool</span>, <span class="hlFld-ContribAuthor ">Priyamvada  Rai</span>. </span><span class="cited-content_cbyCitation_article-title">The Existence of MTH1-independent 8-oxodGTPase Activity in Cancer Cells as a Compensatory Mechanism against On-target Effects of MTH1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2020,</strong> <em>19 </em>
                                    (2)
                                     , 432-446. <a href="https://doi.org/10.1158/1535-7163.MCT-19-0437" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-19-0437</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-19-0437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-19-0437%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DThe%252BExistence%252Bof%252BMTH1-independent%252B8-oxodGTPase%252BActivity%252Bin%252BCancer%252BCells%252Bas%252Ba%252BCompensatory%252BMechanism%252Bagainst%252BOn-target%252BEffects%252Bof%252BMTH1%252BInhibitors%26aulast%3DSamaranayake%26aufirst%3DGovindi%2BJ.%26date%3D2020%26date%3D2019%26volume%3D19%26issue%3D2%26spage%3D432%26epage%3D446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiayu  Wang</span>, <span class="hlFld-ContribAuthor ">Noemi  Nagy</span>, <span class="hlFld-ContribAuthor ">Maria G.  Masucci</span>. </span><span class="cited-content_cbyCitation_article-title">The Epstein–Barr virus nuclear antigen-1 upregulates the cellular antioxidant defense to enable B-cell growth transformation and immortalization. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2020,</strong> <em>39 </em>
                                    (3)
                                     , 603-616. <a href="https://doi.org/10.1038/s41388-019-1003-3" title="DOI URL">https://doi.org/10.1038/s41388-019-1003-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-019-1003-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-019-1003-3%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DThe%252BEpstein%2525E2%252580%252593Barr%252Bvirus%252Bnuclear%252Bantigen-1%252Bupregulates%252Bthe%252Bcellular%252Bantioxidant%252Bdefense%252Bto%252Benable%252BB-cell%252Bgrowth%252Btransformation%252Band%252Bimmortalization%26aulast%3DWang%26aufirst%3DJiayu%26date%3D2020%26date%3D2019%26volume%3D39%26issue%3D3%26spage%3D603%26epage%3D616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeong Woo  Lee</span>, <span class="hlFld-ContribAuthor ">Sangchul  Lee</span>, <span class="hlFld-ContribAuthor ">Jin-Nyoung  Ho</span>, <span class="hlFld-ContribAuthor ">Je-In  Youn</span>, <span class="hlFld-ContribAuthor ">Seok-Soo  Byun</span>, <span class="hlFld-ContribAuthor ">Eunsik  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Antitumor effects of MutT homolog 1 inhibitors in human bladder cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Bioscience, Biotechnology, and Biochemistry</span><span> <strong>2019,</strong> <em>83 </em>
                                    (12)
                                     , 2265-2271. <a href="https://doi.org/10.1080/09168451.2019.1648207" title="DOI URL">https://doi.org/10.1080/09168451.2019.1648207</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/09168451.2019.1648207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F09168451.2019.1648207%26sid%3Dliteratum%253Aachs%26jtitle%3DBioscience%252C%2520Biotechnology%252C%2520and%2520Biochemistry%26atitle%3DAntitumor%252Beffects%252Bof%252BMutT%252Bhomolog%252B1%252Binhibitors%252Bin%252Bhuman%252Bbladder%252Bcancer%252Bcells%26aulast%3DLee%26aufirst%3DJeong%2BWoo%26date%3D2019%26date%3D2019%26volume%3D83%26issue%3D12%26spage%3D2265%26epage%3D2271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lizet M.  van der Waals</span>, <span class="hlFld-ContribAuthor ">Jamila  Laoukili</span>, <span class="hlFld-ContribAuthor ">Jennifer M. J.  Jongen</span>, <span class="hlFld-ContribAuthor ">Danielle A.  Raats</span>, <span class="hlFld-ContribAuthor ">Inne H. M.  Borel Rinkes</span>, <span class="hlFld-ContribAuthor ">Onno  Kranenburg</span>. </span><span class="cited-content_cbyCitation_article-title">Differential anti-tumour effects of MTH1 inhibitors in patient-derived 3D colorectal cancer cultures. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-37316-w" title="DOI URL">https://doi.org/10.1038/s41598-018-37316-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-37316-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-37316-w%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DDifferential%252Banti-tumour%252Beffects%252Bof%252BMTH1%252Binhibitors%252Bin%252Bpatient-derived%252B3D%252Bcolorectal%252Bcancer%252Bcultures%26aulast%3Dvan%2Bder%2BWaals%26aufirst%3DLizet%2BM.%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nadia  Gul</span>, <span class="hlFld-ContribAuthor ">Joakim  Karlsson</span>, <span class="hlFld-ContribAuthor ">Carolina  Tängemo</span>, <span class="hlFld-ContribAuthor ">Sanna  Linsefors</span>, <span class="hlFld-ContribAuthor ">Samuel  Tuyizere</span>, <span class="hlFld-ContribAuthor ">Rosie  Perkins</span>, <span class="hlFld-ContribAuthor ">Chandu  Ala</span>, <span class="hlFld-ContribAuthor ">Zhiyuan  Zou</span>, <span class="hlFld-ContribAuthor ">Erik  Larsson</span>, <span class="hlFld-ContribAuthor ">Martin O.  Bergö</span>, <span class="hlFld-ContribAuthor ">Per  Lindahl</span>. </span><span class="cited-content_cbyCitation_article-title">The MTH1 inhibitor TH588 is a microtubule-modulating agent that eliminates cancer cells by activating the mitotic surveillance pathway. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-019-51205-w" title="DOI URL">https://doi.org/10.1038/s41598-019-51205-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-019-51205-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-019-51205-w%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DThe%252BMTH1%252Binhibitor%252BTH588%252Bis%252Ba%252Bmicrotubule-modulating%252Bagent%252Bthat%252Beliminates%252Bcancer%252Bcells%252Bby%252Bactivating%252Bthe%252Bmitotic%252Bsurveillance%252Bpathway%26aulast%3DGul%26aufirst%3DNadia%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Martinez Molina</span>, <span class="hlFld-ContribAuthor ">Michael  Dabrowski</span>. </span><span class="cited-content_cbyCitation_article-title">Label‐Free Techniques for Target Discovery and Validation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 131-152. <a href="https://doi.org/10.1002/9783527818242.ch5" title="DOI URL">https://doi.org/10.1002/9783527818242.ch5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527818242.ch5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527818242.ch5%26sid%3Dliteratum%253Aachs%26atitle%3DLabel%2525E2%252580%252590Free%252BTechniques%252Bfor%252BTarget%252BDiscovery%252Band%252BValidation%26aulast%3DMartinez%2BMolina%26aufirst%3DDaniel%26date%3D2019%26date%3D2019%26spage%3D131%26epage%3D152%26pub%3DWiley%26atitle%3DTarget%252BDiscovery%252Band%252BValidation%26aulast%3DPlowright%26aufirst%3DAlleyn%2BT%26date%3D2019%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashley  Jarvis</span>, <span class="hlFld-ContribAuthor ">Gilles  Ouvry</span>. </span><span class="cited-content_cbyCitation_article-title">Essential ingredients for rational drug design. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (20)
                                     , 126674. <a href="https://doi.org/10.1016/j.bmcl.2019.126674" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.126674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.126674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.126674%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DEssential%252Bingredients%252Bfor%252Brational%252Bdrug%252Bdesign%26aulast%3DJarvis%26aufirst%3DAshley%26date%3D2019%26volume%3D29%26issue%3D20%26spage%3D126674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher J.  Giuliano</span>, <span class="hlFld-ContribAuthor ">Ann  Lin</span>, <span class="hlFld-ContribAuthor ">Vishruth  Girish</span>, <span class="hlFld-ContribAuthor ">Jason M.  Sheltzer</span>. </span><span class="cited-content_cbyCitation_article-title">Generating Single Cell–Derived Knockout Clones in Mammalian Cells with CRISPR/Cas9. </span><span class="cited-content_cbyCitation_journal-name">Current Protocols in Molecular Biology</span><span> <strong>2019,</strong> <em>128 </em>
                                    (1)
                                     <a href="https://doi.org/10.1002/cpmb.100" title="DOI URL">https://doi.org/10.1002/cpmb.100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cpmb.100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcpmb.100%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Protocols%2520in%2520Molecular%2520Biology%26atitle%3DGenerating%252BSingle%252BCell%2525E2%252580%252593Derived%252BKnockout%252BClones%252Bin%252BMammalian%252BCells%252Bwith%252BCRISPR%25252FCas9%26aulast%3DGiuliano%26aufirst%3DChristopher%2BJ.%26date%3D2019%26date%3D2019%26volume%3D128%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lars  Wiedmer</span>, <span class="hlFld-ContribAuthor ">Claude  Schärer</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Spiliotopoulos</span>, <span class="hlFld-ContribAuthor ">Marianne  Hürzeler</span>, <span class="hlFld-ContribAuthor ">Paweł  Śledź</span>, <span class="hlFld-ContribAuthor ">Amedeo  Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand retargeting by binding site analogy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>175 </em>, 107-113. <a href="https://doi.org/10.1016/j.ejmech.2019.04.037" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.04.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.04.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.04.037%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLigand%252Bretargeting%252Bby%252Bbinding%252Bsite%252Banalogy%26aulast%3DWiedmer%26aufirst%3DLars%26date%3D2019%26volume%3D175%26spage%3D107%26epage%3D113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ilke  Ugur</span>, <span class="hlFld-ContribAuthor ">Maja  Schroft</span>, <span class="hlFld-ContribAuthor ">Antoine  Marion</span>, <span class="hlFld-ContribAuthor ">Manuel  Glaser</span>, <span class="hlFld-ContribAuthor ">Iris  Antes</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting the bioactive conformations of macrocycles: a molecular dynamics-based docking procedure with DynaDock. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2019,</strong> <em>25 </em>
                                    (7)
                                     <a href="https://doi.org/10.1007/s00894-019-4077-5" title="DOI URL">https://doi.org/10.1007/s00894-019-4077-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-019-4077-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-019-4077-5%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DPredicting%252Bthe%252Bbioactive%252Bconformations%252Bof%252Bmacrocycles%25253A%252Ba%252Bmolecular%252Bdynamics-based%252Bdocking%252Bprocedure%252Bwith%252BDynaDock%26aulast%3DUgur%26aufirst%3DIlke%26date%3D2019%26date%3D2019%26volume%3D25%26issue%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jesse C.  Patterson</span>, <span class="hlFld-ContribAuthor ">Brian A.  Joughin</span>, <span class="hlFld-ContribAuthor ">Andrea E.  Prota</span>, <span class="hlFld-ContribAuthor ">Tobias  Mühlethaler</span>, <span class="hlFld-ContribAuthor ">Oliver H.  Jonas</span>, <span class="hlFld-ContribAuthor ">Matthew A.  Whitman</span>, <span class="hlFld-ContribAuthor ">Shohreh  Varmeh</span>, <span class="hlFld-ContribAuthor ">Sen  Chen</span>, <span class="hlFld-ContribAuthor ">Steven P.  Balk</span>, <span class="hlFld-ContribAuthor ">Michel O.  Steinmetz</span>, <span class="hlFld-ContribAuthor ">Douglas A.  Lauffenburger</span>, <span class="hlFld-ContribAuthor ">Michael B.  Yaffe</span>. </span><span class="cited-content_cbyCitation_article-title">VISAGE Reveals a Targetable Mitotic Spindle Vulnerability in Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Cell Systems</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     , 74-92.e8. <a href="https://doi.org/10.1016/j.cels.2019.05.009" title="DOI URL">https://doi.org/10.1016/j.cels.2019.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cels.2019.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cels.2019.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Systems%26atitle%3DVISAGE%252BReveals%252Ba%252BTargetable%252BMitotic%252BSpindle%252BVulnerability%252Bin%252BCancer%252BCells%26aulast%3DPatterson%26aufirst%3DJesse%2BC.%26date%3D2019%26volume%3D9%26issue%3D1%26spage%3D74%26epage%3D92.e8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin-yu Zhao</span>, <span class="hlFld-ContribAuthor ">Kai  Liu</span>, <span class="hlFld-ContribAuthor ">Xiao-li  Wang</span>, <span class="hlFld-ContribAuthor ">Ri-lei  Yu</span>, <span class="hlFld-ContribAuthor ">Cong-min  Kang</span>. </span><span class="cited-content_cbyCitation_article-title">Exploration of Novel MTH1 Inhibitors Using Fragment-Based De Novo Design, Virtual Screening, and Reverse Virtual Screening Methods. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Bioorganic Chemistry</span><span> <strong>2019,</strong> <em>45 </em>
                                    (4)
                                     , 253-262. <a href="https://doi.org/10.1134/S1068162019040137" title="DOI URL">https://doi.org/10.1134/S1068162019040137</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1068162019040137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1068162019040137%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Bioorganic%2520Chemistry%26atitle%3DExploration%252Bof%252BNovel%252BMTH1%252BInhibitors%252BUsing%252BFragment-Based%252BDe%252BNovo%252BDesign%25252C%252BVirtual%252BScreening%25252C%252Band%252BReverse%252BVirtual%252BScreening%252BMethods%26aulast%3DXin-yu%2BZhao%26date%3D2019%26date%3D2019%26volume%3D45%26issue%3D4%26spage%3D253%26epage%3D262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenjuan  Zhou</span>, <span class="hlFld-ContribAuthor ">Liying  Ma</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Hui  Qiao</span>, <span class="hlFld-ContribAuthor ">Lingyu  Li</span>, <span class="hlFld-ContribAuthor ">Qian  Guo</span>, <span class="hlFld-ContribAuthor ">Jinlian  Ma</span>, <span class="hlFld-ContribAuthor ">Lijuan  Zhao</span>, <span class="hlFld-ContribAuthor ">Junwei  Wang</span>, <span class="hlFld-ContribAuthor ">Guozhong  Jiang</span>, <span class="hlFld-ContribAuthor ">Xiangbin  Wan</span>, <span class="hlFld-ContribAuthor ">Mariusz  Adam Goscinski</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Wencai  Li</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>, <span class="hlFld-ContribAuthor ">Zhenhe  Suo</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Potent and specific MTH1 inhibitors targeting gastric cancer. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Disease</span><span> <strong>2019,</strong> <em>10 </em>
                                    (6)
                                     <a href="https://doi.org/10.1038/s41419-019-1665-3" title="DOI URL">https://doi.org/10.1038/s41419-019-1665-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41419-019-1665-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41419-019-1665-3%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Disease%26atitle%3DPotent%252Band%252Bspecific%252BMTH1%252Binhibitors%252Btargeting%252Bgastric%252Bcancer%26aulast%3DZhou%26aufirst%3DWenjuan%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Priyamvada  Rai</span>, <span class="hlFld-ContribAuthor ">Robert W.  Sobol</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms of MTH1 inhibition-induced DNA strand breaks: The slippery slope from the oxidized nucleotide pool to genotoxic damage. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2019,</strong> <em>77 </em>, 18-26. <a href="https://doi.org/10.1016/j.dnarep.2019.03.001" title="DOI URL">https://doi.org/10.1016/j.dnarep.2019.03.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2019.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2019.03.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DMechanisms%252Bof%252BMTH1%252Binhibition-induced%252BDNA%252Bstrand%252Bbreaks%25253A%252BThe%252Bslippery%252Bslope%252Bfrom%252Bthe%252Boxidized%252Bnucleotide%252Bpool%252Bto%252Bgenotoxic%252Bdamage%26aulast%3DRai%26aufirst%3DPriyamvada%26date%3D2019%26volume%3D77%26spage%3D18%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Melike  Çağlayan</span>. </span><span class="cited-content_cbyCitation_article-title">Interplay between DNA Polymerases and DNA Ligases: Influence on Substrate Channeling and the Fidelity of DNA Ligation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2019,</strong> <em>431 </em>
                                    (11)
                                     , 2068-2081. <a href="https://doi.org/10.1016/j.jmb.2019.04.028" title="DOI URL">https://doi.org/10.1016/j.jmb.2019.04.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2019.04.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2019.04.028%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DInterplay%252Bbetween%252BDNA%252BPolymerases%252Band%252BDNA%252BLigases%25253A%252BInfluence%252Bon%252BSubstrate%252BChanneling%252Band%252Bthe%252BFidelity%252Bof%252BDNA%252BLigation%26aulast%3D%25C3%2587a%25C4%259Flayan%26aufirst%3DMelike%26date%3D2019%26volume%3D431%26issue%3D11%26spage%3D2068%26epage%3D2081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takeshi  Yokoyama</span>, <span class="hlFld-ContribAuthor ">Ryota  Kitakami</span>, <span class="hlFld-ContribAuthor ">Mineyuki  Mizuguchi</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a new class of MTH1 inhibitor by X-ray crystallographic screening. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 153-160. <a href="https://doi.org/10.1016/j.ejmech.2019.02.011" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.02.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.02.011%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bnew%252Bclass%252Bof%252BMTH1%252Binhibitor%252Bby%252BX-ray%252Bcrystallographic%252Bscreening%26aulast%3DYokoyama%26aufirst%3DTakeshi%26date%3D2019%26volume%3D167%26spage%3D153%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangwei  Hua</span>, <span class="hlFld-ContribAuthor ">Kumar  Sanjiv</span>, <span class="hlFld-ContribAuthor ">Helge  Gad</span>, <span class="hlFld-ContribAuthor ">Therese  Pham</span>, <span class="hlFld-ContribAuthor ">Camilla  Gokturk</span>, <span class="hlFld-ContribAuthor ">Azita  Rasti</span>, <span class="hlFld-ContribAuthor ">Zhenjun  Zhao</span>, <span class="hlFld-ContribAuthor ">Kang  He</span>, <span class="hlFld-ContribAuthor ">Mingxuan  Feng</span>, <span class="hlFld-ContribAuthor ">Yunjin  Zang</span>, <span class="hlFld-ContribAuthor ">Jianjun  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiang  Xia</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, <span class="hlFld-ContribAuthor ">Ulrika  Warpman Berglund</span>. </span><span class="cited-content_cbyCitation_article-title">Karonudib is a promising anticancer therapy in hepatocellular carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Therapeutic Advances in Medical Oncology</span><span> <strong>2019,</strong> <em>11 </em>, 175883591986696. <a href="https://doi.org/10.1177/1758835919866960" title="DOI URL">https://doi.org/10.1177/1758835919866960</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1758835919866960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1758835919866960%26sid%3Dliteratum%253Aachs%26jtitle%3DTherapeutic%2520Advances%2520in%2520Medical%2520Oncology%26atitle%3DKaronudib%252Bis%252Ba%252Bpromising%252Banticancer%252Btherapy%252Bin%252Bhepatocellular%252Bcarcinoma%26aulast%3DHua%26aufirst%3DXiangwei%26date%3D2019%26date%3D2019%26volume%3D11%26spage%3D175883591986696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sushil K.  Kashaw</span>, <span class="hlFld-ContribAuthor ">Shivangi  Agarwal</span>, <span class="hlFld-ContribAuthor ">Mitali  Mishra</span>, <span class="hlFld-ContribAuthor ">Samaresh  Sau</span>, <span class="hlFld-ContribAuthor ">Arun K.  Iyer</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Docking Analysis of Caspase-3 Activators as Potential Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name">Current Computer-Aided Drug Design</span><span> <strong>2018,</strong> <em>15 </em>
                                    (1)
                                     , 55-66. <a href="https://doi.org/10.2174/1573409914666181015150731" title="DOI URL">https://doi.org/10.2174/1573409914666181015150731</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1573409914666181015150731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1573409914666181015150731%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Computer-Aided%2520Drug%2520Design%26atitle%3DMolecular%252BDocking%252BAnalysis%252Bof%252BCaspase-3%252BActivators%252Bas%252BPotential%252BAnticancer%252BAgents%26aulast%3DKashaw%26aufirst%3DSushil%2BK.%26date%3D2018%26volume%3D15%26issue%3D1%26spage%3D55%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hussein H. K.  Abbas</span>, <span class="hlFld-ContribAuthor ">Kheloud M. H.  Alhamoudi</span>, <span class="hlFld-ContribAuthor ">Mark D.  Evans</span>, <span class="hlFld-ContribAuthor ">George D. D.  Jones</span>, <span class="hlFld-ContribAuthor ">Steven S.  Foster</span>. </span><span class="cited-content_cbyCitation_article-title">MTH1 deficiency selectively increases non-cytotoxic oxidative DNA damage in lung cancer cells: more bad news than good?. </span><span class="cited-content_cbyCitation_journal-name">BMC Cancer</span><span> <strong>2018,</strong> <em>18 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12885-018-4332-7" title="DOI URL">https://doi.org/10.1186/s12885-018-4332-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12885-018-4332-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12885-018-4332-7%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Cancer%26atitle%3DMTH1%252Bdeficiency%252Bselectively%252Bincreases%252Bnon-cytotoxic%252Boxidative%252BDNA%252Bdamage%252Bin%252Blung%252Bcancer%252Bcells%25253A%252Bmore%252Bbad%252Bnews%252Bthan%252Bgood%25253F%26aulast%3DAbbas%26aufirst%3DHussein%2BH.%2BK.%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mosche  Pompsch</span>, <span class="hlFld-ContribAuthor ">Julia  Vogel</span>, <span class="hlFld-ContribAuthor ">Fabian  Classen</span>, <span class="hlFld-ContribAuthor ">Philip  Kranz</span>, <span class="hlFld-ContribAuthor ">George  Iliakis</span>, <span class="hlFld-ContribAuthor ">Helena  Riffkin</span>, <span class="hlFld-ContribAuthor ">Ulf  Brockmeier</span>, <span class="hlFld-ContribAuthor ">Eric  Metzen</span>. </span><span class="cited-content_cbyCitation_article-title">The presumed MTH1-inhibitor TH588 sensitizes colorectal carcinoma cells to ionizing radiation in hypoxia. </span><span class="cited-content_cbyCitation_journal-name">BMC Cancer</span><span> <strong>2018,</strong> <em>18 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12885-018-5095-x" title="DOI URL">https://doi.org/10.1186/s12885-018-5095-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12885-018-5095-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12885-018-5095-x%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Cancer%26atitle%3DThe%252Bpresumed%252BMTH1-inhibitor%252BTH588%252Bsensitizes%252Bcolorectal%252Bcarcinoma%252Bcells%252Bto%252Bionizing%252Bradiation%252Bin%252Bhypoxia%26aulast%3DPompsch%26aufirst%3DMosche%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Torkild  Visnes</span>, <span class="hlFld-ContribAuthor ">Maurice  Grube</span>, <span class="hlFld-ContribAuthor ">Bishoy Magdy Fekry  Hanna</span>, <span class="hlFld-ContribAuthor ">Carlos  Benitez-Buelga</span>, <span class="hlFld-ContribAuthor ">Armando  Cázares-Körner</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting BER enzymes in cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2018,</strong> <em>71 </em>, 118-126. <a href="https://doi.org/10.1016/j.dnarep.2018.08.015" title="DOI URL">https://doi.org/10.1016/j.dnarep.2018.08.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2018.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2018.08.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DTargeting%252BBER%252Benzymes%252Bin%252Bcancer%252Btherapy%26aulast%3DVisnes%26aufirst%3DTorkild%26date%3D2018%26volume%3D71%26spage%3D118%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michelle L.  Kimberland</span>, <span class="hlFld-ContribAuthor ">Wangfang  Hou</span>, <span class="hlFld-ContribAuthor ">Adolfo  Alfonso-Pecchio</span>, <span class="hlFld-ContribAuthor ">Stephen  Wilson</span>, <span class="hlFld-ContribAuthor ">Yanhua  Rao</span>, <span class="hlFld-ContribAuthor ">Shu  Zhang</span>, <span class="hlFld-ContribAuthor ">Quinn  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Strategies for controlling CRISPR/Cas9 off-target effects and biological variations in mammalian genome editing experiments. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biotechnology</span><span> <strong>2018,</strong> <em>284 </em>, 91-101. <a href="https://doi.org/10.1016/j.jbiotec.2018.08.007" title="DOI URL">https://doi.org/10.1016/j.jbiotec.2018.08.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jbiotec.2018.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jbiotec.2018.08.007%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biotechnology%26atitle%3DStrategies%252Bfor%252Bcontrolling%252BCRISPR%25252FCas9%252Boff-target%252Beffects%252Band%252Bbiological%252Bvariations%252Bin%252Bmammalian%252Bgenome%252Bediting%252Bexperiments%26aulast%3DKimberland%26aufirst%3DMichelle%2BL.%26date%3D2018%26volume%3D284%26spage%3D91%26epage%3D101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rui-Juan  Niu</span>, <span class="hlFld-ContribAuthor ">Qing-Chuan  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong-Xing  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">The influence of residue in the position of 116 on the inhibitory potency of TH588 for MTH1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2018,</strong> <em>85 </em>, 75-83. <a href="https://doi.org/10.1016/j.jmgm.2018.08.002" title="DOI URL">https://doi.org/10.1016/j.jmgm.2018.08.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2018.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2018.08.002%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DThe%252Binfluence%252Bof%252Bresidue%252Bin%252Bthe%252Bposition%252Bof%252B116%252Bon%252Bthe%252Binhibitory%252Bpotency%252Bof%252BTH588%252Bfor%252BMTH1%26aulast%3DNiu%26aufirst%3DRui-Juan%26date%3D2018%26volume%3D85%26spage%3D75%26epage%3D83" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katarzyna D.  Arczewska</span>, <span class="hlFld-ContribAuthor ">Anna  Stachurska</span>, <span class="hlFld-ContribAuthor ">Maria  Wojewódzka</span>, <span class="hlFld-ContribAuthor ">Kamila  Karpińska</span>, <span class="hlFld-ContribAuthor ">Marcin  Kruszewski</span>, <span class="hlFld-ContribAuthor ">Hilde  Nilsen</span>, <span class="hlFld-ContribAuthor ">Barbara  Czarnocka</span>. </span><span class="cited-content_cbyCitation_article-title">hMTH1 is required for maintaining migration and invasion potential of human thyroid cancer cells. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2018,</strong> <em>69 </em>, 53-62. <a href="https://doi.org/10.1016/j.dnarep.2018.07.006" title="DOI URL">https://doi.org/10.1016/j.dnarep.2018.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2018.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2018.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DhMTH1%252Bis%252Brequired%252Bfor%252Bmaintaining%252Bmigration%252Band%252Binvasion%252Bpotential%252Bof%252Bhuman%252Bthyroid%252Bcancer%252Bcells%26aulast%3DArczewska%26aufirst%3DKatarzyna%2BD.%26date%3D2018%26volume%3D69%26spage%3D53%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Berglind O.  Einarsdottir</span>, <span class="hlFld-ContribAuthor ">Joakim  Karlsson</span>, <span class="hlFld-ContribAuthor ">Elin M. V.  Söderberg</span>, <span class="hlFld-ContribAuthor ">Mattias F.  Lindberg</span>, <span class="hlFld-ContribAuthor ">Elisa  Funck-Brentano</span>, <span class="hlFld-ContribAuthor ">Henrik  Jespersen</span>, <span class="hlFld-ContribAuthor ">Siggeir F.  Brynjolfsson</span>, <span class="hlFld-ContribAuthor ">Roger  Olofsson Bagge</span>, <span class="hlFld-ContribAuthor ">Louise  Carstam</span>, <span class="hlFld-ContribAuthor ">Martin  Scobie</span>, <span class="hlFld-ContribAuthor ">Tobias  Koolmeister</span>, <span class="hlFld-ContribAuthor ">Olof  Wallner</span>, <span class="hlFld-ContribAuthor ">Ulrika  Stierner</span>, <span class="hlFld-ContribAuthor ">Ulrika Warpman  Berglund</span>, <span class="hlFld-ContribAuthor ">Lars  Ny</span>, <span class="hlFld-ContribAuthor ">Lisa M.  Nilsson</span>, <span class="hlFld-ContribAuthor ">Erik  Larsson</span>, <span class="hlFld-ContribAuthor ">Thomas  Helleday</span>, <span class="hlFld-ContribAuthor ">Jonas A.  Nilsson</span>. </span><span class="cited-content_cbyCitation_article-title">A patient-derived xenograft pre-clinical trial reveals treatment responses and a resistance mechanism to karonudib in metastatic melanoma. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Disease</span><span> <strong>2018,</strong> <em>9 </em>
                                    (8)
                                     <a href="https://doi.org/10.1038/s41419-018-0865-6" title="DOI URL">https://doi.org/10.1038/s41419-018-0865-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41419-018-0865-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41419-018-0865-6%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Disease%26atitle%3DA%252Bpatient-derived%252Bxenograft%252Bpre-clinical%252Btrial%252Breveals%252Btreatment%252Bresponses%252Band%252Ba%252Bresistance%252Bmechanism%252Bto%252Bkaronudib%252Bin%252Bmetastatic%252Bmelanoma%26aulast%3DEinarsdottir%26aufirst%3DBerglind%2BO.%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Morgan</span>, <span class="hlFld-ContribAuthor ">Dean G.  Brown</span>, <span class="hlFld-ContribAuthor ">Simon  Lennard</span>, <span class="hlFld-ContribAuthor ">Mark J.  Anderton</span>, <span class="hlFld-ContribAuthor ">J. Carl  Barrett</span>, <span class="hlFld-ContribAuthor ">Ulf  Eriksson</span>, <span class="hlFld-ContribAuthor ">Mark  Fidock</span>, <span class="hlFld-ContribAuthor ">Bengt  Hamrén</span>, <span class="hlFld-ContribAuthor ">Anthony  Johnson</span>, <span class="hlFld-ContribAuthor ">Ruth E.  March</span>, <span class="hlFld-ContribAuthor ">James  Matcham</span>, <span class="hlFld-ContribAuthor ">Jerome  Mettetal</span>, <span class="hlFld-ContribAuthor ">David J.  Nicholls</span>, <span class="hlFld-ContribAuthor ">Stefan  Platz</span>, <span class="hlFld-ContribAuthor ">Steve  Rees</span>, <span class="hlFld-ContribAuthor ">Michael A.  Snowden</span>, <span class="hlFld-ContribAuthor ">Menelas N.  Pangalos</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of a five-dimensional framework on R&D productivity at AstraZeneca. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2018,</strong> <em>17 </em>
                                    (3)
                                     , 167-181. <a href="https://doi.org/10.1038/nrd.2017.244" title="DOI URL">https://doi.org/10.1038/nrd.2017.244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2017.244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2017.244%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DImpact%252Bof%252Ba%252Bfive-dimensional%252Bframework%252Bon%252BR%252526D%252Bproductivity%252Bat%252BAstraZeneca%26aulast%3DMorgan%26aufirst%3DPaul%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D3%26spage%3D167%26epage%3D181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">蓉蓉  狄</span>. </span><span class="cited-content_cbyCitation_article-title">The Research Progress of MTH1 Inhibitors in Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Pharmacy Information</span><span> <strong>2018,</strong> <em>07 </em>
                                    (02)
                                     , 15-20. <a href="https://doi.org/10.12677/PI.2018.72004" title="DOI URL">https://doi.org/10.12677/PI.2018.72004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.12677/PI.2018.72004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.12677%2FPI.2018.72004%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacy%2520Information%26atitle%3DThe%252BResearch%252BProgress%252Bof%252BMTH1%252BInhibitors%252Bin%252BCancer%252BTreatment%26aulast%3D%25E7%258B%2584%26aufirst%3D%25E8%2593%2589%25E8%2593%2589%26date%3D2018%26volume%3D07%26issue%3D02%26spage%3D15%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fumiyoshi  Ikejiri</span>, <span class="hlFld-ContribAuthor ">Yoshio  Honma</span>, <span class="hlFld-ContribAuthor ">Takashi  Kasukabe</span>, <span class="hlFld-ContribAuthor ">Takeshi  Urano</span>, <span class="hlFld-ContribAuthor ">Junji  Suzumiya</span>. </span><span class="cited-content_cbyCitation_article-title">TH588, an MTH1 inhibitor, enhances phenethyl isothiocyanate‑induced growth inhibition in pancreatic cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Oncology Letters</span><span> <strong>2017,</strong> <em> </em><a href="https://doi.org/10.3892/ol.2017.7713" title="DOI URL">https://doi.org/10.3892/ol.2017.7713</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/ol.2017.7713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2Fol.2017.7713%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Letters%26atitle%3DTH588%25252C%252Ban%252BMTH1%252Binhibitor%25252C%252Benhances%252Bphenethyl%252Bisothiocyanate%2525E2%252580%252591induced%252Bgrowth%252Binhibition%252Bin%252Bpancreatic%252Bcancer%252Bcells%26aulast%3DIkejiri%26aufirst%3DFumiyoshi%26date%3D2017%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shinji  Asada</span>, <span class="hlFld-ContribAuthor ">Eiko  Ohta</span>, <span class="hlFld-ContribAuthor ">Yoriko  Akimoto</span>, <span class="hlFld-ContribAuthor ">Nona  Abolhassani</span>, <span class="hlFld-ContribAuthor ">Daisuke  Tsuchimoto</span>, <span class="hlFld-ContribAuthor ">Yusaku  Nakabeppu</span>. </span><span class="cited-content_cbyCitation_article-title">2-Oxoadenosine induces cytotoxicity through intracellular accumulation of 2-oxo-ATP and depletion of ATP but not via the p38 MAPK pathway. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-06636-8" title="DOI URL">https://doi.org/10.1038/s41598-017-06636-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-06636-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-06636-8%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3D2-Oxoadenosine%252Binduces%252Bcytotoxicity%252Bthrough%252Bintracellular%252Baccumulation%252Bof%252B2-oxo-ATP%252Band%252Bdepletion%252Bof%252BATP%252Bbut%252Bnot%252Bvia%252Bthe%252Bp38%252BMAPK%252Bpathway%26aulast%3DAsada%26aufirst%3DShinji%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jingjing  Duan</span>, <span class="hlFld-ContribAuthor ">Haiyang  Zhang</span>, <span class="hlFld-ContribAuthor ">Shuang  Li</span>, <span class="hlFld-ContribAuthor ">Xinyi  Wang</span>, <span class="hlFld-ContribAuthor ">Haiou  Yang</span>, <span class="hlFld-ContribAuthor ">Shunchang  Jiao</span>, <span class="hlFld-ContribAuthor ">Yi  Ba</span>. </span><span class="cited-content_cbyCitation_article-title">The role of miR-485-5p/NUDT1 axis in gastric cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancer Cell International</span><span> <strong>2017,</strong> <em>17 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12935-017-0462-2" title="DOI URL">https://doi.org/10.1186/s12935-017-0462-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12935-017-0462-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12935-017-0462-2%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Cell%2520International%26atitle%3DThe%252Brole%252Bof%252BmiR-485-5p%25252FNUDT1%252Baxis%252Bin%252Bgastric%252Bcancer%26aulast%3DDuan%26aufirst%3DJingjing%26date%3D2017%26date%3D2017%26volume%3D17%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuanxuan  Dai</span>, <span class="hlFld-ContribAuthor ">Guilong  Guo</span>, <span class="hlFld-ContribAuthor ">Peng  Zou</span>, <span class="hlFld-ContribAuthor ">Ri  Cui</span>, <span class="hlFld-ContribAuthor ">Weiqian  Chen</span>, <span class="hlFld-ContribAuthor ">Xi  Chen</span>, <span class="hlFld-ContribAuthor ">Changtian  Yin</span>, <span class="hlFld-ContribAuthor ">Wei  He</span>, <span class="hlFld-ContribAuthor ">Rajamanickam  Vinothkumar</span>, <span class="hlFld-ContribAuthor ">Fan  Yang</span>, <span class="hlFld-ContribAuthor ">Xiaohua  Zhang</span>, <span class="hlFld-ContribAuthor ">Guang  Liang</span>. </span><span class="cited-content_cbyCitation_article-title">(S)-crizotinib induces apoptosis in human non-small cell lung cancer cells by activating ROS independent of MTH1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Experimental & Clinical Cancer Research</span><span> <strong>2017,</strong> <em>36 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13046-017-0584-3" title="DOI URL">https://doi.org/10.1186/s13046-017-0584-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13046-017-0584-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13046-017-0584-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Experimental%2520%2526%2520Clinical%2520Cancer%2520Research%26atitle%3D%252528S%252529-crizotinib%252Binduces%252Bapoptosis%252Bin%252Bhuman%252Bnon-small%252Bcell%252Blung%252Bcancer%252Bcells%252Bby%252Bactivating%252BROS%252Bindependent%252Bof%252BMTH1%26aulast%3DDai%26aufirst%3DXuanxuan%26date%3D2017%26date%3D2017%26volume%3D36%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jin  Li</span>, <span class="hlFld-ContribAuthor ">Cheng-Cheng  Yang</span>, <span class="hlFld-ContribAuthor ">Xin-Yuan  Tian</span>, <span class="hlFld-ContribAuthor ">Yun-Xuan  Li</span>, <span class="hlFld-ContribAuthor ">Ju  Cui</span>, <span class="hlFld-ContribAuthor ">Zhe  Chen</span>, <span class="hlFld-ContribAuthor ">Zhou-Lu  Deng</span>, <span class="hlFld-ContribAuthor ">Fu-Jun  Chen</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Hayakawa</span>, <span class="hlFld-ContribAuthor ">Mutsuo  Sekiguchi</span>, <span class="hlFld-ContribAuthor ">Jian-Ping  Cai</span>. </span><span class="cited-content_cbyCitation_article-title">MutT-related proteins are novel progression and prognostic markers for colorectal cancer. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (62)
                                     , 105714-105726. <a href="https://doi.org/10.18632/oncotarget.22393" title="DOI URL">https://doi.org/10.18632/oncotarget.22393</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.22393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.22393%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DMutT-related%252Bproteins%252Bare%252Bnovel%252Bprogression%252Band%252Bprognostic%252Bmarkers%252Bfor%252Bcolorectal%252Bcancer%26aulast%3DLi%26aufirst%3DJin%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D62%26spage%3D105714%26epage%3D105726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Govindi  Samaranayake</span>, <span class="hlFld-ContribAuthor ">Mai  Huynh</span>, <span class="hlFld-ContribAuthor ">Priyamvada  Rai</span>. </span><span class="cited-content_cbyCitation_article-title">MTH1 as a Chemotherapeutic Target: The Elephant in the Room. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2017,</strong> <em>9 </em>
                                    (12)
                                     , 47. <a href="https://doi.org/10.3390/cancers9050047" title="DOI URL">https://doi.org/10.3390/cancers9050047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers9050047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers9050047%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DMTH1%252Bas%252Ba%252BChemotherapeutic%252BTarget%25253A%252BThe%252BElephant%252Bin%252Bthe%252BRoom%26aulast%3DSamaranayake%26aufirst%3DGovindi%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D12%26spage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vincent  Wagner</span>, <span class="hlFld-ContribAuthor ">Linda  Jantz</span>, <span class="hlFld-ContribAuthor ">Hans  Briem</span>, <span class="hlFld-ContribAuthor ">Kai  Sommer</span>, <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>, <span class="hlFld-ContribAuthor ">Clara D.  Christ</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Macrocyclization: From de novo Macrocycle Generation to Binding Affinity Estimation. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (22)
                                     , 1866-1872. <a href="https://doi.org/10.1002/cmdc.201700478" title="DOI URL">https://doi.org/10.1002/cmdc.201700478</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700478%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DComputational%252BMacrocyclization%25253A%252BFrom%252Bde%2525E2%252580%252585novo%252BMacrocycle%252BGeneration%252Bto%252BBinding%252BAffinity%252BEstimation%26aulast%3DWagner%26aufirst%3DVincent%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D22%26spage%3D1866%26epage%3D1872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia Yu  Wang</span>, <span class="hlFld-ContribAuthor ">Guang Zhi  Liu</span>, <span class="hlFld-ContribAuthor ">James S.  Wilmott</span>, <span class="hlFld-ContribAuthor ">Ting  La</span>, <span class="hlFld-ContribAuthor ">Yu Chen  Feng</span>, <span class="hlFld-ContribAuthor ">Hamed  Yari</span>, <span class="hlFld-ContribAuthor ">Xu Guang  Yan</span>, <span class="hlFld-ContribAuthor ">Rick F.  Thorne</span>, <span class="hlFld-ContribAuthor ">Richard A.  Scolyer</span>, <span class="hlFld-ContribAuthor ">Xu Dong  Zhang</span>, <span class="hlFld-ContribAuthor ">Lei  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Skp2-Mediated Stabilization of MTH1 Promotes Survival of Melanoma Cells upon Oxidative Stress. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2017,</strong> <em>77 </em>
                                    (22)
                                     , 6226-6239. <a href="https://doi.org/10.1158/0008-5472.CAN-17-1965" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-17-1965</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-17-1965&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-17-1965%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DSkp2-Mediated%252BStabilization%252Bof%252BMTH1%252BPromotes%252BSurvival%252Bof%252BMelanoma%252BCells%252Bupon%252BOxidative%252BStress%26aulast%3DWang%26aufirst%3DJia%2BYu%26date%3D2017%26date%3D2017%26volume%3D77%26issue%3D22%26spage%3D6226%26epage%3D6239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ya  Gao</span>, <span class="hlFld-ContribAuthor ">Lihan  Zhu</span>, <span class="hlFld-ContribAuthor ">Jing  Guo</span>, <span class="hlFld-ContribAuthor ">Ting  Yuan</span>, <span class="hlFld-ContribAuthor ">Liqing  Wang</span>, <span class="hlFld-ContribAuthor ">Hua  Li</span>, <span class="hlFld-ContribAuthor ">Lixia  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Farnesyl phenolic enantiomers as natural MTH1 inhibitors from
              Ganoderma sinense. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (56)
                                     , 95865-95879. <a href="https://doi.org/10.18632/oncotarget.21430" title="DOI URL">https://doi.org/10.18632/oncotarget.21430</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.21430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.21430%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DFarnesyl%252Bphenolic%252Benantiomers%252Bas%252Bnatural%252BMTH1%252Binhibitors%252Bfrom%252BGanoderma%252Bsinense%26aulast%3DGao%26aufirst%3DYa%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D56%26spage%3D95865%26epage%3D95879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Enni  Markkanen</span>. </span><span class="cited-content_cbyCitation_article-title">Not breathing is not an option: How to deal with oxidative DNA damage. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2017,</strong> <em>59 </em>, 82-105. <a href="https://doi.org/10.1016/j.dnarep.2017.09.007" title="DOI URL">https://doi.org/10.1016/j.dnarep.2017.09.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2017.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2017.09.007%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DNot%252Bbreathing%252Bis%252Bnot%252Ban%252Boption%25253A%252BHow%252Bto%252Bdeal%252Bwith%252Boxidative%252BDNA%252Bdamage%26aulast%3DMarkkanen%26aufirst%3DEnni%26date%3D2017%26volume%3D59%26spage%3D82%26epage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>, <span class="hlFld-ContribAuthor ">Michelle  Arkin</span>, <span class="hlFld-ContribAuthor ">Yves P.  Auberson</span>, <span class="hlFld-ContribAuthor ">Henk  Timmerman</span>, <span class="hlFld-ContribAuthor ">Herbert  Waldmann</span>. </span><span class="cited-content_cbyCitation_article-title">Joining Forces: The Chemical Biology–Medicinal Chemistry Continuum. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2017,</strong> <em>24 </em>
                                    (9)
                                     , 1058-1065. <a href="https://doi.org/10.1016/j.chembiol.2017.05.019" title="DOI URL">https://doi.org/10.1016/j.chembiol.2017.05.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2017.05.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2017.05.019%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DJoining%252BForces%25253A%252BThe%252BChemical%252BBiology%2525E2%252580%252593Medicinal%252BChemistry%252BContinuum%26aulast%3DPlowright%26aufirst%3DAlleyn%2BT.%26date%3D2017%26volume%3D24%26issue%3D9%26spage%3D1058%26epage%3D1065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julian  Blagg</span>, <span class="hlFld-ContribAuthor ">Paul  Workman</span>. </span><span class="cited-content_cbyCitation_article-title">Choose and Use Your Chemical Probe Wisely to Explore Cancer Biology. </span><span class="cited-content_cbyCitation_journal-name">Cancer Cell</span><span> <strong>2017,</strong> <em>32 </em>
                                    (1)
                                     , 9-25. <a href="https://doi.org/10.1016/j.ccell.2017.06.005" title="DOI URL">https://doi.org/10.1016/j.ccell.2017.06.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ccell.2017.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ccell.2017.06.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Cell%26atitle%3DChoose%252Band%252BUse%252BYour%252BChemical%252BProbe%252BWisely%252Bto%252BExplore%252BCancer%252BBiology%26aulast%3DBlagg%26aufirst%3DJulian%26date%3D2017%26volume%3D32%26issue%3D1%26spage%3D9%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yusaku  Nakabeppu</span>, <span class="hlFld-ContribAuthor ">Eiko  Ohta</span>, <span class="hlFld-ContribAuthor ">Nona  Abolhassani</span>. </span><span class="cited-content_cbyCitation_article-title">MTH1 as a nucleotide pool sanitizing enzyme: Friend or foe?. </span><span class="cited-content_cbyCitation_journal-name">Free Radical Biology and Medicine</span><span> <strong>2017,</strong> <em>107 </em>, 151-158. <a href="https://doi.org/10.1016/j.freeradbiomed.2016.11.002" title="DOI URL">https://doi.org/10.1016/j.freeradbiomed.2016.11.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.freeradbiomed.2016.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.freeradbiomed.2016.11.002%26sid%3Dliteratum%253Aachs%26jtitle%3DFree%2520Radical%2520Biology%2520and%2520Medicine%26atitle%3DMTH1%252Bas%252Ba%252Bnucleotide%252Bpool%252Bsanitizing%252Benzyme%25253A%252BFriend%252Bor%252Bfoe%25253F%26aulast%3DNakabeppu%26aufirst%3DYusaku%26date%3D2017%26volume%3D107%26spage%3D151%26epage%3D158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duoqian  Dai</span>, <span class="hlFld-ContribAuthor ">Lu  Zhou</span>, <span class="hlFld-ContribAuthor ">Xiaohong  Zhu</span>, <span class="hlFld-ContribAuthor ">Rong  You</span>, <span class="hlFld-ContribAuthor ">Liangliang  Zhong</span>. </span><span class="cited-content_cbyCitation_article-title">Combined multi-pharmacophore, molecular docking and molecular dynamic study for discovery of promising MTH1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2017,</strong> <em>1137 </em>, 33-42. <a href="https://doi.org/10.1016/j.molstruc.2017.02.034" title="DOI URL">https://doi.org/10.1016/j.molstruc.2017.02.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2017.02.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2017.02.034%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DCombined%252Bmulti-pharmacophore%25252C%252Bmolecular%252Bdocking%252Band%252Bmolecular%252Bdynamic%252Bstudy%252Bfor%252Bdiscovery%252Bof%252Bpromising%252BMTH1%252Binhibitors%26aulast%3DDai%26aufirst%3DDuoqian%26date%3D2017%26volume%3D1137%26spage%3D33%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lenka  Munoz</span>. </span><span class="cited-content_cbyCitation_article-title">Non-kinase targets of protein kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2017,</strong> <em>16 </em>
                                    (6)
                                     , 424-440. <a href="https://doi.org/10.1038/nrd.2016.266" title="DOI URL">https://doi.org/10.1038/nrd.2016.266</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2016.266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2016.266%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DNon-kinase%252Btargets%252Bof%252Bprotein%252Bkinase%252Binhibitors%26aulast%3DMunoz%26aufirst%3DLenka%26date%3D2017%26date%3D2017%26volume%3D16%26issue%3D6%26spage%3D424%26epage%3D440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashutosh  Kumar</span>, <span class="hlFld-ContribAuthor ">Tatsuro  Kawamura</span>, <span class="hlFld-ContribAuthor ">Makoto  Kawatani</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Osada</span>, <span class="hlFld-ContribAuthor ">Kam Y. J.  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and structure-activity relationship of purine derivatives as novel MTH1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2017,</strong> <em>89 </em>
                                    (6)
                                     , 862-869. <a href="https://doi.org/10.1111/cbdd.12909" title="DOI URL">https://doi.org/10.1111/cbdd.12909</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.12909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.12909%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DIdentification%252Band%252Bstructure-activity%252Brelationship%252Bof%252Bpurine%252Bderivatives%252Bas%252Bnovel%252BMTH1%252Binhibitors%26aulast%3DKumar%26aufirst%3DAshutosh%26date%3D2017%26date%3D2016%26volume%3D89%26issue%3D6%26spage%3D862%26epage%3D869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elke Tatjana  Aristizabal Prada</span>, <span class="hlFld-ContribAuthor ">Michael  Orth</span>, <span class="hlFld-ContribAuthor ">Svenja  Nölting</span>, <span class="hlFld-ContribAuthor ">Gerald  Spöttl</span>, <span class="hlFld-ContribAuthor ">Julian  Maurer</span>, <span class="hlFld-ContribAuthor ">Christoph  Auernhammer</span>, . </span><span class="cited-content_cbyCitation_article-title">The MTH1 inhibitor TH588 demonstrates anti-tumoral effects alone and in combination with everolimus, 5-FU and gamma-irradiation in neuroendocrine tumor cells. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , e0178375. <a href="https://doi.org/10.1371/journal.pone.0178375" title="DOI URL">https://doi.org/10.1371/journal.pone.0178375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0178375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0178375%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DThe%252BMTH1%252Binhibitor%252BTH588%252Bdemonstrates%252Banti-tumoral%252Beffects%252Balone%252Band%252Bin%252Bcombination%252Bwith%252Beverolimus%25252C%252B5-FU%252Band%252Bgamma-irradiation%252Bin%252Bneuroendocrine%252Btumor%252Bcells%26aulast%3DAristizabal%2BPrada%26aufirst%3DElke%2BTatjana%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3De0178375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhipeng  Xiong</span>, <span class="hlFld-ContribAuthor ">Junfan  Niu</span>, <span class="hlFld-ContribAuthor ">Hao  Liu</span>, <span class="hlFld-ContribAuthor ">Zhihong  Xu</span>, <span class="hlFld-ContribAuthor ">Junkai  Li</span>, <span class="hlFld-ContribAuthor ">Qinglai  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and bioactivities of Phenazine-1-carboxylic acid derivatives based on the modification of PCA carboxyl group. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (9)
                                     , 2010-2013. <a href="https://doi.org/10.1016/j.bmcl.2017.03.011" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.03.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.03.011%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bbioactivities%252Bof%252BPhenazine-1-carboxylic%252Bacid%252Bderivatives%252Bbased%252Bon%252Bthe%252Bmodification%252Bof%252BPCA%252Bcarboxyl%252Bgroup%26aulast%3DXiong%26aufirst%3DZhipeng%26date%3D2017%26volume%3D27%26issue%3D9%26spage%3D2010%26epage%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yosuke  Taniguchi</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Damaged Nucleoside Mimics for Inhibition of Their Repair Enzymes. </span><span class="cited-content_cbyCitation_journal-name">YAKUGAKU ZASSHI</span><span> <strong>2017,</strong> <em>137 </em>
                                    (3)
                                     , 293-300. <a href="https://doi.org/10.1248/yakushi.16-00231-2" title="DOI URL">https://doi.org/10.1248/yakushi.16-00231-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/yakushi.16-00231-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fyakushi.16-00231-2%26sid%3Dliteratum%253Aachs%26jtitle%3DYAKUGAKU%2520ZASSHI%26atitle%3DDevelopment%252Bof%252BDamaged%252BNucleoside%252BMimics%252Bfor%252BInhibition%252Bof%252BTheir%252BRepair%252BEnzymes%26aulast%3DTaniguchi%26aufirst%3DYosuke%26date%3D2017%26volume%3D137%26issue%3D3%26spage%3D293%26epage%3D300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shaimaa  Waz</span>, <span class="hlFld-ContribAuthor ">Teruya  Nakamura</span>, <span class="hlFld-ContribAuthor ">Keisuke  Hirata</span>, <span class="hlFld-ContribAuthor ">Yukari  Koga-Ogawa</span>, <span class="hlFld-ContribAuthor ">Mami  Chirifu</span>, <span class="hlFld-ContribAuthor ">Takao  Arimori</span>, <span class="hlFld-ContribAuthor ">Taro  Tamada</span>, <span class="hlFld-ContribAuthor ">Shinji  Ikemizu</span>, <span class="hlFld-ContribAuthor ">Yusaku  Nakabeppu</span>, <span class="hlFld-ContribAuthor ">Yuriko  Yamagata</span>. </span><span class="cited-content_cbyCitation_article-title">Structural and Kinetic Studies of the Human Nudix Hydrolase MTH1 Reveal the Mechanism for Its Broad Substrate Specificity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2017,</strong> <em>292 </em>
                                    (7)
                                     , 2785-2794. <a href="https://doi.org/10.1074/jbc.M116.749713" title="DOI URL">https://doi.org/10.1074/jbc.M116.749713</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.M116.749713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.M116.749713%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DStructural%252Band%252BKinetic%252BStudies%252Bof%252Bthe%252BHuman%252BNudix%252BHydrolase%252BMTH1%252BReveal%252Bthe%252BMechanism%252Bfor%252BIts%252BBroad%252BSubstrate%252BSpecificity%26aulast%3DWaz%26aufirst%3DShaimaa%26date%3D2017%26volume%3D292%26issue%3D7%26spage%3D2785%26epage%3D2794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shingo  Akiyama</span>, <span class="hlFld-ContribAuthor ">Hiroshi  Saeki</span>, <span class="hlFld-ContribAuthor ">Yuichiro  Nakashima</span>, <span class="hlFld-ContribAuthor ">Makoto  Iimori</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kitao</span>, <span class="hlFld-ContribAuthor ">Eiji  Oki</span>, <span class="hlFld-ContribAuthor ">Yoshinao  Oda</span>, <span class="hlFld-ContribAuthor ">Yusaku  Nakabeppu</span>, <span class="hlFld-ContribAuthor ">Yoshihiro  Kakeji</span>, <span class="hlFld-ContribAuthor ">Yoshihiko  Maehara</span>. </span><span class="cited-content_cbyCitation_article-title">Prognostic impact of MutT homolog-1 expression on esophageal squamous cell carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Cancer Medicine</span><span> <strong>2017,</strong> <em>6 </em>
                                    (1)
                                     , 258-266. <a href="https://doi.org/10.1002/cam4.979" title="DOI URL">https://doi.org/10.1002/cam4.979</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cam4.979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcam4.979%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Medicine%26atitle%3DPrognostic%252Bimpact%252Bof%252BMutT%252Bhomolog-1%252Bexpression%252Bon%252Besophageal%252Bsquamous%252Bcell%252Bcarcinoma%26aulast%3DAkiyama%26aufirst%3DShingo%26date%3D2017%26date%3D2016%26volume%3D6%26issue%3D1%26spage%3D258%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sylvie M.  Guichard</span>. </span><span class="cited-content_cbyCitation_article-title">CRISPR–Cas9 for Drug Discovery in Oncology. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 61-85. <a href="https://doi.org/10.1016/bs.armc.2017.08.006" title="DOI URL">https://doi.org/10.1016/bs.armc.2017.08.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2017.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2017.08.006%26sid%3Dliteratum%253Aachs%26atitle%3DCRISPR%2525E2%252580%252593Cas9%252Bfor%252BDrug%252BDiscovery%252Bin%252BOncology%26aulast%3DGuichard%26aufirst%3DSylvie%2BM.%26date%3D2017%26spage%3D61%26epage%3D85%26pub%3DElsevier%26atitle%3DPlatform%252BTechnologies%252Bin%252BDrug%252BDiscovery%252Band%252BValidation%26date%3D2017%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. Paola  Castaldi</span>, <span class="hlFld-ContribAuthor ">Andrea  Zuhl</span>, <span class="hlFld-ContribAuthor ">Piero  Ricchiuto</span>, <span class="hlFld-ContribAuthor ">J. Adam  Hendricks</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Biology in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 335-370. <a href="https://doi.org/10.1016/bs.armc.2017.08.009" title="DOI URL">https://doi.org/10.1016/bs.armc.2017.08.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2017.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2017.08.009%26sid%3Dliteratum%253Aachs%26atitle%3DChemical%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DCastaldi%26aufirst%3DM.%2BPaola%26date%3D2017%26spage%3D335%26epage%3D370%26pub%3DElsevier%26atitle%3DPlatform%252BTechnologies%252Bin%252BDrug%252BDiscovery%252Band%252BValidation%26date%3D2017%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher K.  Mathews</span>. </span><span class="cited-content_cbyCitation_article-title">Oxidized deoxyribonucleotides, mutagenesis, and cancer. </span><span class="cited-content_cbyCitation_journal-name">The FASEB Journal</span><span> <strong>2017,</strong> <em>31 </em>
                                    (1)
                                     , 11-13. <a href="https://doi.org/10.1096/fj.201601100" title="DOI URL">https://doi.org/10.1096/fj.201601100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1096/fj.201601100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1096%2Ffj.201601100%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FASEB%2520Journal%26atitle%3DOxidized%252Bdeoxyribonucleotides%25252C%252Bmutagenesis%25252C%252Band%252Bcancer%26aulast%3DMathews%26aufirst%3DChristopher%2BK.%26date%3D2017%26date%3D2016%26volume%3D31%26issue%3D1%26spage%3D11%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elise  Fouquerel</span>, <span class="hlFld-ContribAuthor ">Justin  Lormand</span>, <span class="hlFld-ContribAuthor ">Arindam  Bose</span>, <span class="hlFld-ContribAuthor ">Hui-Ting  Lee</span>, <span class="hlFld-ContribAuthor ">Grace S  Kim</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Li</span>, <span class="hlFld-ContribAuthor ">Robert W  Sobol</span>, <span class="hlFld-ContribAuthor ">Bret D  Freudenthal</span>, <span class="hlFld-ContribAuthor ">Sua  Myong</span>, <span class="hlFld-ContribAuthor ">Patricia L  Opresko</span>. </span><span class="cited-content_cbyCitation_article-title">Oxidative guanine base damage regulates human telomerase activity. </span><span class="cited-content_cbyCitation_journal-name">Nature Structural & Molecular Biology</span><span> <strong>2016,</strong> <em>23 </em>
                                    (12)
                                     , 1092-1100. <a href="https://doi.org/10.1038/nsmb.3319" title="DOI URL">https://doi.org/10.1038/nsmb.3319</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nsmb.3319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnsmb.3319%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Structural%2520%2526%2520Molecular%2520Biology%26atitle%3DOxidative%252Bguanine%252Bbase%252Bdamage%252Bregulates%252Bhuman%252Btelomerase%252Bactivity%26aulast%3DFouquerel%26aufirst%3DElise%26date%3D2016%26date%3D2016%26volume%3D23%26issue%3D12%26spage%3D1092%26epage%3D1100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">U.  Warpman Berglund</span>, <span class="hlFld-ContribAuthor ">K.  Sanjiv</span>, <span class="hlFld-ContribAuthor ">H.  Gad</span>, <span class="hlFld-ContribAuthor ">C.  Kalderén</span>, <span class="hlFld-ContribAuthor ">T.  Koolmeister</span>, <span class="hlFld-ContribAuthor ">T.  Pham</span>, <span class="hlFld-ContribAuthor ">C.  Gokturk</span>, <span class="hlFld-ContribAuthor ">R.  Jafari</span>, <span class="hlFld-ContribAuthor ">G.  Maddalo</span>, <span class="hlFld-ContribAuthor ">B.  Seashore-Ludlow</span>, <span class="hlFld-ContribAuthor ">A.  Chernobrovkin</span>, <span class="hlFld-ContribAuthor ">A.  Manoilov</span>, <span class="hlFld-ContribAuthor ">I.S.  Pateras</span>, <span class="hlFld-ContribAuthor ">A.  Rasti</span>, <span class="hlFld-ContribAuthor ">A.-S.  Jemth</span>, <span class="hlFld-ContribAuthor ">I.  Almlöf</span>, <span class="hlFld-ContribAuthor ">O.  Loseva</span>, <span class="hlFld-ContribAuthor ">T.  Visnes</span>, <span class="hlFld-ContribAuthor ">B.O.  Einarsdottir</span>, <span class="hlFld-ContribAuthor ">F.Z.  Gaugaz</span>, <span class="hlFld-ContribAuthor ">A.  Saleh</span>, <span class="hlFld-ContribAuthor ">B.  Platzack</span>, <span class="hlFld-ContribAuthor ">O.A.  Wallner</span>, <span class="hlFld-ContribAuthor ">K.S.A.  Vallin</span>, <span class="hlFld-ContribAuthor ">M.  Henriksson</span>, <span class="hlFld-ContribAuthor ">P.  Wakchaure</span>, <span class="hlFld-ContribAuthor ">S.  Borhade</span>, <span class="hlFld-ContribAuthor ">P.  Herr</span>, <span class="hlFld-ContribAuthor ">Y.  Kallberg</span>, <span class="hlFld-ContribAuthor ">P.  Baranczewski</span>, <span class="hlFld-ContribAuthor ">E.J.  Homan</span>, <span class="hlFld-ContribAuthor ">E.  Wiita</span>, <span class="hlFld-ContribAuthor ">V.  Nagpal</span>, <span class="hlFld-ContribAuthor ">T.  Meijer</span>, <span class="hlFld-ContribAuthor ">N.  Schipper</span>, <span class="hlFld-ContribAuthor ">S.G.  Rudd</span>, <span class="hlFld-ContribAuthor ">L.  Bräutigam</span>, <span class="hlFld-ContribAuthor ">A.  Lindqvist</span>, <span class="hlFld-ContribAuthor ">A.  Filppula</span>, <span class="hlFld-ContribAuthor ">T.-C.  Lee</span>, <span class="hlFld-ContribAuthor ">P.  Artursson</span>, <span class="hlFld-ContribAuthor ">J.A.  Nilsson</span>, <span class="hlFld-ContribAuthor ">V.G.  Gorgoulis</span>, <span class="hlFld-ContribAuthor ">J.  Lehtiö</span>, <span class="hlFld-ContribAuthor ">R.A.  Zubarev</span>, <span class="hlFld-ContribAuthor ">M.  Scobie</span>, <span class="hlFld-ContribAuthor ">T.  Helleday</span>. </span><span class="cited-content_cbyCitation_article-title">Validation and development of MTH1 inhibitors for treatment of cancer. </span><span class="cited-content_cbyCitation_journal-name">Annals of Oncology</span><span> <strong>2016,</strong> <em>27 </em>
                                    (12)
                                     , 2275-2283. <a href="https://doi.org/10.1093/annonc/mdw429" title="DOI URL">https://doi.org/10.1093/annonc/mdw429</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/annonc/mdw429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fannonc%2Fmdw429%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnals%2520of%2520Oncology%26atitle%3DValidation%252Band%252Bdevelopment%252Bof%252BMTH1%252Binhibitors%252Bfor%252Btreatment%252Bof%252Bcancer%26aulast%3DWarpman%2BBerglund%26aufirst%3DU.%26date%3D2016%26volume%3D27%26issue%3D12%26spage%3D2275%26epage%3D2283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia Yu  Wang</span>, <span class="hlFld-ContribAuthor ">Lei  Jin</span>, <span class="hlFld-ContribAuthor ">Xu Guang  Yan</span>, <span class="hlFld-ContribAuthor ">Simonne  Sherwin</span>, <span class="hlFld-ContribAuthor ">Margaret  Farrelly</span>, <span class="hlFld-ContribAuthor ">Yuan Yuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Fen  Liu</span>, <span class="hlFld-ContribAuthor ">Chun Yan  Wang</span>, <span class="hlFld-ContribAuthor ">Su Tang  Guo</span>, <span class="hlFld-ContribAuthor ">Hamed  Yari</span>, <span class="hlFld-ContribAuthor ">Ting  La</span>, <span class="hlFld-ContribAuthor ">Jennifer  McFarlane</span>, <span class="hlFld-ContribAuthor ">Fu Xi  Lei</span>, <span class="hlFld-ContribAuthor ">Hessam  Tabatabaee</span>, <span class="hlFld-ContribAuthor ">Jie Zhong  Chen</span>, <span class="hlFld-ContribAuthor ">Amanda  Croft</span>, <span class="hlFld-ContribAuthor ">Chen Chen  Jiang</span>, <span class="hlFld-ContribAuthor ">Xu Dong  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Reactive Oxygen Species Dictate the Apoptotic Response of Melanoma Cells to TH588. </span><span class="cited-content_cbyCitation_journal-name">Journal of Investigative Dermatology</span><span> <strong>2016,</strong> <em>136 </em>
                                    (11)
                                     , 2277-2286. <a href="https://doi.org/10.1016/j.jid.2016.06.625" title="DOI URL">https://doi.org/10.1016/j.jid.2016.06.625</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jid.2016.06.625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jid.2016.06.625%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Investigative%2520Dermatology%26atitle%3DReactive%252BOxygen%252BSpecies%252BDictate%252Bthe%252BApoptotic%252BResponse%252Bof%252BMelanoma%252BCells%252Bto%2525C2%2525A0TH588%26aulast%3DWang%26aufirst%3DJia%2BYu%26date%3D2016%26volume%3D136%26issue%3D11%26spage%3D2277%26epage%3D2286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sebastian H.  Verne</span>, <span class="hlFld-ContribAuthor ">Marina  Perper</span>, <span class="hlFld-ContribAuthor ">Robert J.  Magno</span>, <span class="hlFld-ContribAuthor ">Ariel E.  Eber</span>, <span class="hlFld-ContribAuthor ">Adam S.  Aldahan</span>, <span class="hlFld-ContribAuthor ">Mana  AL-Harbi</span>, <span class="hlFld-ContribAuthor ">Keyvan  Nouri</span>. </span><span class="cited-content_cbyCitation_article-title">Cells to Surgery Quiz: November 2016. </span><span class="cited-content_cbyCitation_journal-name">Journal of Investigative Dermatology</span><span> <strong>2016,</strong> <em>136 </em>
                                    (11)
                                     , e117. <a href="https://doi.org/10.1016/j.jid.2016.09.002" title="DOI URL">https://doi.org/10.1016/j.jid.2016.09.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jid.2016.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jid.2016.09.002%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Investigative%2520Dermatology%26atitle%3DCells%252Bto%252BSurgery%252BQuiz%25253A%252BNovember%252B2016%26aulast%3DVerne%26aufirst%3DSebastian%2BH.%26date%3D2016%26volume%3D136%26issue%3D11%26spage%3De117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tatsuro  Kawamura</span>, <span class="hlFld-ContribAuthor ">Makoto  Kawatani</span>, <span class="hlFld-ContribAuthor ">Makoto  Muroi</span>, <span class="hlFld-ContribAuthor ">Yasumitsu  Kondoh</span>, <span class="hlFld-ContribAuthor ">Yushi  Futamura</span>, <span class="hlFld-ContribAuthor ">Harumi  Aono</span>, <span class="hlFld-ContribAuthor ">Miho  Tanaka</span>, <span class="hlFld-ContribAuthor ">Kaori  Honda</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Osada</span>. </span><span class="cited-content_cbyCitation_article-title">Proteomic profiling of small-molecule inhibitors reveals dispensability of MTH1 for cancer cell survival. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2016,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/srep26521" title="DOI URL">https://doi.org/10.1038/srep26521</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/srep26521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsrep26521%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DProteomic%252Bprofiling%252Bof%252Bsmall-molecule%252Binhibitors%252Breveals%252Bdispensability%252Bof%252BMTH1%252Bfor%252Bcancer%252Bcell%252Bsurvival%26aulast%3DKawamura%26aufirst%3DTatsuro%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rais A.  Ganai</span>, <span class="hlFld-ContribAuthor ">Erik  Johansson</span>. </span><span class="cited-content_cbyCitation_article-title">DNA Replication—A Matter of Fidelity. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2016,</strong> <em>62 </em>
                                    (5)
                                     , 745-755. <a href="https://doi.org/10.1016/j.molcel.2016.05.003" title="DOI URL">https://doi.org/10.1016/j.molcel.2016.05.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2016.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2016.05.003%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DDNA%252BReplication%2525E2%252580%252594A%252BMatter%252Bof%252BFidelity%26aulast%3DGanai%26aufirst%3DRais%25C2%25A0A.%26date%3D2016%26volume%3D62%26issue%3D5%26spage%3D745%26epage%3D755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bassam K.  Alnasleh</span>, <span class="hlFld-ContribAuthor ">Marina  Rubina</span>, <span class="hlFld-ContribAuthor ">Michael  Rubin</span>. </span><span class="cited-content_cbyCitation_article-title">Templated assembly of medium cyclic ethers via exo-trig nucleophilic cyclization of cyclopropenes. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2016,</strong> <em>52 </em>
                                    (47)
                                     , 7494-7496. <a href="https://doi.org/10.1039/C6CC02178F" title="DOI URL">https://doi.org/10.1039/C6CC02178F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6CC02178F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6CC02178F%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DTemplated%252Bassembly%252Bof%252Bmedium%252Bcyclic%252Bethers%252Bvia%252Bexo-trig%252Bnucleophilic%252Bcyclization%252Bof%252Bcyclopropenes%26aulast%3DAlnasleh%26aufirst%3DBassam%2BK.%26date%3D2016%26date%3D2016%26volume%3D52%26issue%3D47%26spage%3D7494%26epage%3D7496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Literature inhibitors of MTH1, <b>1</b>–<b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. TLR7 agonist Imiquimod (<b>4</b>) and a photoaffinity probe TLR7 agonist (<b>5</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0013.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Macrocycle MTH1 Inhibitor <b>15</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) NH<sub>3</sub>, <i>i</i>-PrOH, Et<sub>3</sub>N, 0 °C, 5 h; (ii) EtOCOCl, THF, Et<sub>3</sub>N, 0 °C, 5 h, 76% (2 steps); (iii) <i>m</i>-OTBDMS benzyl chloride, NaI, K<sub>2</sub>CO<sub>3</sub>, acetone, 20 °C, 16 h; (iv) MeNH<sub>2</sub>, THF, 20 °C, 16 h, 100% (2 steps); (v) <i>m</i>CPBA, DCM, 20 °C, 2 h; (vi) 2,2′-oxidiethanol, NaH, THF, 20 °C, 17 h, 71% (2 steps); (vii) TBAF, THF, 20 °C, 16 h, 86%; (vii) DTAD, Ph<sub>3</sub>P, THF, 20 °C, 20 h, (ix) Fe, AcOH, water, 60 °C, 4 h, 7% (2 steps).</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0014.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Quinoline Amide MTH1 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) POCl<sub>3</sub>, 100 °C, 1 h, 90%; (ii) 4-chloro-2-fluoroaniline, AcOH, <i>i</i>-PrOH, 100 °C, 16 h, 99%; (iii) MeNH<sub>2</sub>, Me<sub>3</sub>Al, THF, 60 °C, 2 h, 79%; (iv) aniline, AcOH, DMF, 100 °C, 1 h, 76%; (v) MeNH<sub>2</sub>, EtOH, 120 °C, 3 h, 81%; (vi) phenol, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 16 h, 100%; (vii) (a) NaOH, THF, 20 °C, 17 h, (b) SOCl<sub>2</sub>, 80 °C, 1 h, (c) MeNH<sub>2</sub>, DMF/THF, 20 °C, 1 h, 74%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0015.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Aminoquinazoline MTH1 Inhibitor <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Pd-118, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O, 100 °C, 2 h, 28%.</p></p></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structures of human MTH1 in complex with (a) <b>25</b> (pink, resiquimod), (b) complex with <b>25</b> rotated to show the hydrophobic residues (white), and (c) complexes with <b>25</b> and 8-oxo-dGMP (orange) superimposed. Selected carbon atoms, protein backbone cartoon, and close polar contacts are represented in the complex color. Compounds are shown as sticks, and protein residue side chains are shown as thinner sticks. Close polar contacts are represented as dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) X-ray crystal structures of human MTH1 in complex with <b>27</b> (green). (b) Complexes with <b>27</b> (green) and <b>15</b> (yellow) overlaid. Selected carbon atoms, protein backbone cartoon, and close polar contacts are represented in the complex color. Compounds are shown as sticks, and protein residue side chains are shown as thinner sticks. Close polar contacts are represented as dotted lines. Some atoms of the loop covering the active site (residues 23–31) have been removed for clarity. (c) Example conformational ensemble for <b>15</b>. Lowest-energy conformation (green) has a ring orientation similar to that seen for <b>27</b> (cyan) in the protein–ligand structure. (d) Modeled conformation of amide-containing macrocycle <b>30</b> (yellow) overlaid on X-ray structure conformation of <b>27</b> (cyan) shows differential orientation of phenyl ring. (e) Conformational ensemble of <b>29</b> derived using NMR suggests that the dominant conformations in solution (gray) are very similar to the conformation seen in the protein–ligand structure (maroon).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray crystal structures of human MTH1 in complex with <b>19</b> (green). Selected carbon atoms, protein backbone cartoon, and close polar contacts are represented in the complex color. Compounds are shown as sticks, and protein residue side chains are shown as thinner sticks. Close polar contacts are represented as dotted lines. The 2-chloro-4-fluoro-phenyl group adopts a dual conformation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) X-ray crystal structures of human MTH1 in complex with <b>24</b> (yellow). (b) Complexes with <b>24</b> and <b>1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U">4N1U</a>, cyan) overlaid. Selected carbon atoms, protein backbone cartoon, and close polar contacts are represented in the complex color. Compounds are shown as sticks, and protein residue side chains are shown as thinner sticks. Close polar contacts are represented as dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Correlation between MTH1 inhibitory potencies and target engagement in K562 cells colored by series. Straight line is line of fit.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0009.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Induction of DDR signaling markers and apopotosis following treatment of U2OS cells for 24, 48, or 72 h with 10 μM <b>1</b> or <b>19</b>, compared to DMSO vehicle control (Con). Treatment with <b>1</b> leads to increased p-Ser1981 ATM, p-Ser15 p53 expression, and cleaved PARP1 (apoptosis). <b>19</b> shows no impact on DDR signaling activation or apoptosis. No changes in γH2AX or pSer4/8 RPA DDR signalng were observed for either compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0010.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (a) Treatment of U2OS cells with siRNA to MTH1. Incubation for 2 and 7 days with both AZ and literature (#3) siRNA leads to significant knockdown of MTH1 protein relative to nontargeted (NT) controls, but no impact on other markers of DNA-damage response (DDR) activation as measured by <i>p</i>-Ser1981 ATM and <i>p</i>-Ser15 p53 expression. (b) MTH1 knockdown results in a significant reduction in clonogenic cell survival for literature (#3) siRNA but not a distinct AZ siRNA compared to control transfected cells. (c) Isogenic gene knockouts of MTH1 using CRISPR technology in SW480 cells. (d) Clones with complete knockout of all MTH1 alleles had undetectable levels of protein expression and were readily recoverable. Clones with complete knockout of MTH1 were not impaired in growth rate compared to SW480 parental cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0011.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. MTH1 inhibitor cell viability is independent of MTH1. (a) Clonogenic survival of U2OS cells treated with <b>1</b>, <b>3</b>, or <b>19</b> following nontargeting control siRNA (NT), AZ MTH1 siRNA, or Gad et al. MTH1 siRNA #3. Values represent mean % of colonies forming ± SD in comparison to nontargeting control siRNA treated cells from at least three independent experiments unless otherwise stated. (b) Viability of SW480 cell growth in parental and SW480 MTH1<sup>–/–/–</sup> clone three cells treated with <b>1</b>, <b>3</b>, or <b>19</b>. Values represent mean % viability (CellTiter-Glo) relative to DMSO control-treated cells from at least fi veindependent experiments ± SD. Table: calculated GI<sub>50</sub> values in μM for <b>1</b>, <b>3</b>, and <b>19</b> for experiments shown in (a) and (b).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/medium/jm-2015-01760p_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0012.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Plot of log <i>D</i> versus MTH1 enzyme potency for MTH1 inhibitors. Points are colored by chemical series, and arrows indicate the direction of optimization from screening hit to cell probe compound described in this study. The highlighted box is indicative of a potency and lipophilicity range deemed desirable for utility in probing target biology within cells in order to ensure potent and specific target modulation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01760/20160318/images/large/jm-2015-01760p_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01760&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44386" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44386" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 20 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Freudenthal, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perera, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shock, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, S. H.</span><span> </span><span class="NLM_article-title">Uncovering the polymerase-induced cytotoxicity of an oxidized nucleotide</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">517</span><span class="NLM_x">, </span> <span class="NLM_fpage">635</span><span class="NLM_x">–</span> <span class="NLM_lpage">639</span><span class="refDoi"> DOI: 10.1038/nature13886</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1038%2Fnature13886" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=25409153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGlt7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2015&pages=635-639&author=B.+D.+Freudenthalauthor=W.+A.+Beardauthor=L.+Pereraauthor=D.+D.+Shockauthor=T.+Kimauthor=T.+Schlickauthor=S.+H.+Wilson&title=Uncovering+the+polymerase-induced+cytotoxicity+of+an+oxidized+nucleotide&doi=10.1038%2Fnature13886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Uncovering the polymerase-induced cytotoxicity of an oxidized nucleotide</span></div><div class="casAuthors">Freudenthal, Bret D.; Beard, William A.; Perera, Lalith; Shock, David D.; Kim, Taejin; Schlick, Tamar; Wilson, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">517</span>
        (<span class="NLM_cas:issue">7536</span>),
    <span class="NLM_cas:pages">635-639</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oxidative stress promotes genomic instability and human diseases.  A common oxidized nucleoside is 8-oxo-7,8-dihydro-2'-deoxyguanosine, which is found both in DNA (8-oxo-G) and as a free nucleotide (8-oxo-dGTP).  Nucleotide pools are esp. vulnerable to oxidative damage.  Therefore cells encode an enzyme (MutT/MTH1) that removes free oxidized nucleotides.  This cleansing function is required for cancer cell survival and to modulate Escherichia coli antibiotic sensitivity in a DNA polymerase (pol)-dependent manner.  How polymerases discriminate between damaged and non-damaged nucleotides is not well understood.  This anal. is essential given the role of oxidized nucleotides in mutagenesis, cancer therapeutics, and bacterial antibiotics.  Even with cellular sanitizing activities, nucleotide pools contain enough 8-oxo-dGTP to promote mutagenesis.  This arises from the dual coding potential where 8-oxo-dGTP(anti) base pairs with cytosine and 8-oxo-dGTP(syn) uses its Hoogsteen edge to base pair with adenine.  Here we use time-lapse crystallog. to follow 8-oxo-dGTP insertion opposite adenine or cytosine with human pol β, to reveal that insertion is accommodated in either the syn- or anti-conformation, resp.  For 8-oxo-dGTP(anti) insertion, a novel divalent metal relieves repulsive interactions between the adducted guanine base and the triphosphate of the oxidized nucleotide.  With either templating base, hydrogen-bonding interactions between the bases are lost as the enzyme reopens after catalysis, leading to a cytotoxic nicked DNA repair intermediate.  Combining structural snapshots with kinetic and computational anal. reveals how 8-oxo-dGTP uses charge modulation during insertion that can lead to a blocked DNA repair intermediate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUp5_4v80tf7Vg90H21EOLACvtfcHk0liwqEOjuxCweA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGlt7nK&md5=cc5f601f0879b37d38e924f66cd4df06</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature13886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13886%26sid%3Dliteratum%253Aachs%26aulast%3DFreudenthal%26aufirst%3DB.%2BD.%26aulast%3DBeard%26aufirst%3DW.%2BA.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DShock%26aufirst%3DD.%2BD.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DSchlick%26aufirst%3DT.%26aulast%3DWilson%26aufirst%3DS.%2BH.%26atitle%3DUncovering%2520the%2520polymerase-induced%2520cytotoxicity%2520of%2520an%2520oxidized%2520nucleotide%26jtitle%3DNature%26date%3D2015%26volume%3D517%26spage%3D635%26epage%3D639%26doi%3D10.1038%2Fnature13886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Gad, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koolmeister, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jemth, A.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eshtad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ström, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svensson, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundbäck, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Einarsdottir, B. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saleh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Göktürk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baranczewski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svensson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berntsson, R. P.-A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strömberg, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanjiv, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques-Cordonnier, M.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desroses, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustavsson, A.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olofsson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Homan, E.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loseva, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bräutigam, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Höglund, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagenkort, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pham, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altun, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaugaz, F. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vikingsson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henriksson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vallin, K. S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallner, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammarström, L. G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiita, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almlöf, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalderén, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Axelsson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djureinovic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puigvert, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Häggblad, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeppsson, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martens, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundgren, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granelli, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artursson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stenmark, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scobie, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, U. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helleday, T.</span><span> </span><span class="NLM_article-title">MTH1 inhibition eradicates cancer by preventing sanitation of the dNTP pool</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">508</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1038/nature13181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1038%2Fnature13181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=24695224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=215-221&author=H.+Gadauthor=T.+Koolmeisterauthor=A.-S.+Jemthauthor=S.+Eshtadauthor=S.+A.+Jacquesauthor=C.+E.+Str%C3%B6mauthor=L.+M.+Svenssonauthor=N.+Schultzauthor=T.+Lundb%C3%A4ckauthor=B.+O.+Einarsdottirauthor=A.+Salehauthor=C.+G%C3%B6kt%C3%BCrkauthor=P.+Baranczewskiauthor=R.+Svenssonauthor=R.+P.-A.+Berntssonauthor=R.+Gustafssonauthor=K.+Str%C3%B6mbergauthor=K.+Sanjivauthor=M.-C.+Jacques-Cordonnierauthor=M.+Desrosesauthor=A.-L.+Gustavssonauthor=R.+Olofssonauthor=F.+Johanssonauthor=E.-J.+Homanauthor=O.+Losevaauthor=L.+Br%C3%A4utigamauthor=L.+Johanssonauthor=A.+H%C3%B6glundauthor=A.+Hagenkortauthor=T.+Phamauthor=M.+Altunauthor=F.+Z.+Gaugazauthor=S.+Vikingssonauthor=B.+Eversauthor=M.+Henrikssonauthor=K.+S.+A.+Vallinauthor=O.+A.+Wallnerauthor=L.+G.+J.+Hammarstr%C3%B6mauthor=E.+Wiitaauthor=I.+Alml%C3%B6fauthor=C.+Kalder%C3%A9nauthor=H.+Axelssonauthor=T.+Djureinovicauthor=J.+C.+Puigvertauthor=M.+H%C3%A4ggbladauthor=F.+Jeppssonauthor=U.+Martensauthor=C.+Lundinauthor=B.+Lundgrenauthor=I.+Granelliauthor=A.+J.+Jensenauthor=P.+Arturssonauthor=J.+A.+Nilssonauthor=P.+Stenmarkauthor=M.+Scobieauthor=U.+W.+Berglundauthor=T.+Helleday&title=MTH1+inhibition+eradicates+cancer+by+preventing+sanitation+of+the+dNTP+pool&doi=10.1038%2Fnature13181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">MTH1 inhibition eradicates cancer by preventing sanitation of the dNTP pool</span></div><div class="casAuthors">Gad, Helge; Koolmeister, Tobias; Jemth, Ann-Sofie; Eshtad, Saeed; Jacques, Sylvain A.; Stroem, Cecilia E.; Svensson, Linda M.; Schultz, Niklas; Lundbaeck, Thomas; Einarsdottir, Berglind Osk; Saleh, Aljona; Gokturk, Camilla; Baranczewski, Pawel; Svensson, Richard; Berntsson, Ronnie P.-A.; Gustafsson, Robert; Stroemberg, Kia; Sanjiv, Kumar; Jacques-Cordonnier, Marie-Caroline; Desroses, Matthieu; Gustavsson, Anna-Lena; Olofsson, Roger; Johansson, Fredrik; Homan, Evert J.; Loseva, Olga; Braeutigam, Lars; Johansson, Lars; Hoeglund, Andreas; Hagenkort, Anna; Pham, Therese; Altun, Mikael; Gaugaz, Fabienne Z.; Vikingsson, Svante; Evers, Bastiaan; Henriksson, Martin; Vallin, Karl S. A.; Wallner, Olov A.; Hammarstroem, Lars G. J.; Wiita, Elisee; Almloef, Ingrid; Kalderen, Christina; Axelsson, Hanna; Djureinovic, Tatjana; Puigvert, Jordi Carreras; Haeggblad, Maria; Jeppsson, Fredrik; Martens, Ulf; Lundin, Cecilia; Lundgren, Bo; Granelli, Ingrid; Jensen, Annika Jenmalm; Artursson, Per; Nilsson, Jonas A.; Stenmark, Pal; Scobie, Martin; Berglund, Ulrika Warpman; Helleday, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">215-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancers have dysfunctional redox regulation resulting in reactive oxygen species prodn., damaging both DNA and free dNTPs.  The MTH1 protein sanitizes oxidized dNTP pools to prevent incorporation of damaged bases during DNA replication.  Although MTH1 is non-essential in normal cells, the authors show that cancer cells require MTH1 activity to avoid incorporation of oxidized dNTPs, resulting in DNA damage and cell death.  The authors validate MTH1 as an anticancer target in vivo and describe small mols. TH287 (I) and TH588 (II) as first-in-class nudix hydrolase family inhibitors that potently and selectively engage and inhibit the MTH1 protein in cells.  Protein co-crystal structures demonstrate that the inhibitors bind in the active site of MTH1.  The inhibitors cause incorporation of oxidized dNTPs in cancer cells, leading to DNA damage, cytotoxicity and therapeutic responses in patient-derived mouse xenografts.  This study exemplifies the non-oncogene addiction concept for anticancer treatment and validates MTH1 as being cancer phenotypic lethal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Xn3KhC40XbVg90H21EOLACvtfcHk0lhAvnk_D86HrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsLc%253D&md5=49afb4522874f4f5e216ae38d0ff7cd2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature13181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13181%26sid%3Dliteratum%253Aachs%26aulast%3DGad%26aufirst%3DH.%26aulast%3DKoolmeister%26aufirst%3DT.%26aulast%3DJemth%26aufirst%3DA.-S.%26aulast%3DEshtad%26aufirst%3DS.%26aulast%3DJacques%26aufirst%3DS.%2BA.%26aulast%3DStr%25C3%25B6m%26aufirst%3DC.%2BE.%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DEinarsdottir%26aufirst%3DB.%2BO.%26aulast%3DSaleh%26aufirst%3DA.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DBaranczewski%26aufirst%3DP.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DBerntsson%26aufirst%3DR.%2BP.-A.%26aulast%3DGustafsson%26aufirst%3DR.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DJacques-Cordonnier%26aufirst%3DM.-C.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DGustavsson%26aufirst%3DA.-L.%26aulast%3DOlofsson%26aufirst%3DR.%26aulast%3DJohansson%26aufirst%3DF.%26aulast%3DHoman%26aufirst%3DE.-J.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DBr%25C3%25A4utigam%26aufirst%3DL.%26aulast%3DJohansson%26aufirst%3DL.%26aulast%3DH%25C3%25B6glund%26aufirst%3DA.%26aulast%3DHagenkort%26aufirst%3DA.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DAltun%26aufirst%3DM.%26aulast%3DGaugaz%26aufirst%3DF.%2BZ.%26aulast%3DVikingsson%26aufirst%3DS.%26aulast%3DEvers%26aufirst%3DB.%26aulast%3DHenriksson%26aufirst%3DM.%26aulast%3DVallin%26aufirst%3DK.%2BS.%2BA.%26aulast%3DWallner%26aufirst%3DO.%2BA.%26aulast%3DHammarstr%25C3%25B6m%26aufirst%3DL.%2BG.%2BJ.%26aulast%3DWiita%26aufirst%3DE.%26aulast%3DAlml%25C3%25B6f%26aufirst%3DI.%26aulast%3DKalder%25C3%25A9n%26aufirst%3DC.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DDjureinovic%26aufirst%3DT.%26aulast%3DPuigvert%26aufirst%3DJ.%2BC.%26aulast%3DH%25C3%25A4ggblad%26aufirst%3DM.%26aulast%3DJeppsson%26aufirst%3DF.%26aulast%3DMartens%26aufirst%3DU.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DLundgren%26aufirst%3DB.%26aulast%3DGranelli%26aufirst%3DI.%26aulast%3DJensen%26aufirst%3DA.%2BJ.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DNilsson%26aufirst%3DJ.%2BA.%26aulast%3DStenmark%26aufirst%3DP.%26aulast%3DScobie%26aufirst%3DM.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DHelleday%26aufirst%3DT.%26atitle%3DMTH1%2520inhibition%2520eradicates%2520cancer%2520by%2520preventing%2520sanitation%2520of%2520the%2520dNTP%2520pool%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D215%26epage%3D221%26doi%3D10.1038%2Fnature13181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Huber, K. V. M</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salah, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radic, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gridling, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elkins, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jemth, A.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Göktürk, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanjiv, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strömberg, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pham, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berglund, U. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colinge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loizou, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helleday, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Stereospecific targeting of MTH1 by (<i>S</i>)-crizotinib as an anticancer strategy</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">508</span><span class="NLM_x">, </span> <span class="NLM_fpage">222</span><span class="NLM_x">–</span> <span class="NLM_lpage">227</span><span class="refDoi"> DOI: 10.1038/nature13194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1038%2Fnature13194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=24695225" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=222-227&author=K.+V.+M+Huberauthor=E.+Salahauthor=B.+Radicauthor=M.+Gridlingauthor=J.+M.+Elkinsauthor=A.+Stukalovauthor=A.-S.+Jemthauthor=C.+G%C3%B6kt%C3%BCrkauthor=K.+Sanjivauthor=K.+Str%C3%B6mbergauthor=T.+Phamauthor=U.+W.+Berglundauthor=J.+Colingeauthor=K.+L.+Bennettauthor=J.+I.+Loizouauthor=T.+Helledayauthor=S.+Knappauthor=G.+Superti-Furga&title=Stereospecific+targeting+of+MTH1+by+%28S%29-crizotinib+as+an+anticancer+strategy&doi=10.1038%2Fnature13194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Stereospecific targeting of MTH1 by (S)-crizotinib as an anticancer strategy</span></div><div class="casAuthors">Huber, Kilian V. M.; Salah, Eidarus; Radic, Branka; Gridling, Manuela; Elkins, Jonathan M.; Stukalov, Alexey; Jemth, Ann-Sofie; Gokturk, Camilla; Sanjiv, Kumar; Stroemberg, Kia; Pham, Therese; Berglund, Ulrika Warpman; Colinge, Jacques; Bennett, Keiryn L.; Loizou, Joanna I.; Helleday, Thomas; Knapp, Stefan; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">222-227</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activated RAS GTPase signaling is a crit. driver of oncogenic transformation and malignant disease.  Cellular models of RAS-dependent cancers have been used to identify exptl. small mols., such as SCH51344, but their mol. mechanism of action remains generally unknown.  Here, using a chem. proteomic approach, the authors identify the target of SCH51344 as the human mutT homolog MTH1 (also known as NUDT1), a nucleotide pool sanitizing enzyme.  Loss-of-function of MTH1 impaired growth of KRAS tumor cells, whereas MTH1 overexpression mitigated sensitivity towards SCH51344.  Searching for more drug-like inhibitors, the authors identified the kinase inhibitor crizotinib as a nanomolar suppressor of MTH1 activity.  Surprisingly, the clin. used (R)-enantiomer of the drug was inactive, whereas the (S)-enantiomer selectively inhibited MTH1 catalytic activity.  Enzymic assays, chem. proteomic profiling, kinome-wide activity surveys and MTH1 co-crystal structures of both enantiomers provide a rationale for this remarkable stereospecificity.  Disruption of nucleotide pool homeostasis via MTH1 inhibition by (S)-crizotinib induced an increase in DNA single-strand breaks, activated DNA repair in human colon carcinoma cells, and effectively suppressed tumor growth in animal models.  The authors' results propose (S)-crizotinib as an attractive chem. entity for further pre-clin. evaluation, and small-mol. inhibitors of MTH1 in general as a promising novel class of anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbUm7UP6kIrVg90H21EOLACvtfcHk0lg2fjI-dThdUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWrsb4%253D&md5=ca3df11a6782fd9140b6d8ba0c16e5ab</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature13194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13194%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DRadic%26aufirst%3DB.%26aulast%3DGridling%26aufirst%3DM.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DJemth%26aufirst%3DA.-S.%26aulast%3DG%25C3%25B6kt%25C3%25BCrk%26aufirst%3DC.%26aulast%3DSanjiv%26aufirst%3DK.%26aulast%3DStr%25C3%25B6mberg%26aufirst%3DK.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DBerglund%26aufirst%3DU.%2BW.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DLoizou%26aufirst%3DJ.%2BI.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DStereospecific%2520targeting%2520of%2520MTH1%2520by%2520%2528S%2529-crizotinib%2520as%2520an%2520anticancer%2520strategy%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D222%26epage%3D227%26doi%3D10.1038%2Fnature13194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Egashira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamauchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshiyama, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawate, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katsuki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekiguchi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimachi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuzuki, T.</span><span> </span><span class="NLM_article-title">Mutational specificity of mice defective in the MTH1 and/or the MSH2 genes</span> <span class="citation_source-journal">DNA Repair</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">893</span><span class="refDoi"> DOI: 10.1016/S1568-7864(02)00113-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2FS1568-7864%2802%2900113-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=881-893&author=A.+Egashiraauthor=K.+Yamauchiauthor=K.+Yoshiyamaauthor=H.+Kawateauthor=M.+Katsukiauthor=M.+Sekiguchiauthor=K.+Sugimachiauthor=H.+Makiauthor=T.+Tsuzuki&title=Mutational+specificity+of+mice+defective+in+the+MTH1+and%2For+the+MSH2+genes&doi=10.1016%2FS1568-7864%2802%2900113-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS1568-7864%2802%2900113-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1568-7864%252802%252900113-1%26sid%3Dliteratum%253Aachs%26aulast%3DEgashira%26aufirst%3DA.%26aulast%3DYamauchi%26aufirst%3DK.%26aulast%3DYoshiyama%26aufirst%3DK.%26aulast%3DKawate%26aufirst%3DH.%26aulast%3DKatsuki%26aufirst%3DM.%26aulast%3DSekiguchi%26aufirst%3DM.%26aulast%3DSugimachi%26aufirst%3DK.%26aulast%3DMaki%26aufirst%3DH.%26aulast%3DTsuzuki%26aufirst%3DT.%26atitle%3DMutational%2520specificity%2520of%2520mice%2520defective%2520in%2520the%2520MTH1%2520and%252For%2520the%2520MSH2%2520genes%26jtitle%3DDNA%2520Repair%26date%3D2002%26volume%3D1%26spage%3D881%26epage%3D893%26doi%3D10.1016%2FS1568-7864%2802%2900113-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Kobold, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedemann, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothenfußer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endres, S.</span><span> </span><span class="NLM_article-title">Modes of action of TLR7 agonists in cancer therapy immunotherapy</span> <span class="citation_source-journal">Immunotherapy</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1085</span><span class="NLM_x">–</span> <span class="NLM_lpage">1095</span><span class="refDoi"> DOI: 10.2217/imt.14.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.2217%2Fimt.14.75" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1085-1095&author=S.+Koboldauthor=G.+Wiedemannauthor=S.+Rothenfu%C3%9Ferauthor=S.+Endres&title=Modes+of+action+of+TLR7+agonists+in+cancer+therapy+immunotherapy&doi=10.2217%2Fimt.14.75"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2217%2Fimt.14.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fimt.14.75%26sid%3Dliteratum%253Aachs%26aulast%3DKobold%26aufirst%3DS.%26aulast%3DWiedemann%26aufirst%3DG.%26aulast%3DRothenfu%25C3%259Fer%26aufirst%3DS.%26aulast%3DEndres%26aufirst%3DS.%26atitle%3DModes%2520of%2520action%2520of%2520TLR7%2520agonists%2520in%2520cancer%2520therapy%2520immunotherapy%26jtitle%3DImmunotherapy%26date%3D2014%26volume%3D6%26spage%3D1085%26epage%3D1095%26doi%3D10.2217%2Fimt.14.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Raasch, B.</span><span> </span><span class="NLM_article-title">Management of superficial basal cell carcinoma: focus on imiquimod</span> <span class="citation_source-journal">Clin., Cosmet. Invest. Dermatol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.2147/CCID.S3507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.2147%2FCCID.S3507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=21436969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVShtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=65-67&author=B.+Raasch&title=Management+of+superficial+basal+cell+carcinoma%3A+focus+on+imiquimod&doi=10.2147%2FCCID.S3507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Management of superficial basal cell carcinoma: focus on imiquimod</span></div><div class="casAuthors">Raasch, Beverly</div><div class="citationInfo"><span class="NLM_cas:title">Clinical, Cosmetic and Investigational Dermatology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">65-75</span>CODEN:
                <span class="NLM_cas:coden">CCIDCL</span>;
        ISSN:<span class="NLM_cas:issn">1178-7015</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press (NZ) Ltd.</span>)
        </div><div class="casAbstract">A review.  Superficial basal cell carcinoma comprise up to 25% of all histol. sub-types.  They are more likely to occur on younger persons and females and although generally more common on the trunk, also occur frequently on the exposed areas of the head and neck esp. in areas of high sun exposure.  In the last decade, new treatment options such as topical applications that modify the immune response have been trialed for effectiveness in treating these lesions.  Imiquimod 5% cream has been shown to stimulate the innate and cell mediated immune system.  The short-term success of imiquimod 5% cream in randomized controlled trials comparing different treatment regimes and dosing as a treatment for small superficial basal cell carcinoma (BCC) not on the face or neck is in the range of 82% for 5 times per wk application.  A high proportion of participants with good response rates to topical treatment (58%-92%) experience local side effects such as itching and burning, less commonly erosion and ulceration, but the proportion of participants ceasing treatment has not been high.  To date one long-term study indicates a treatment success rate of 78%-81% and that initial response is a predictor of long-term outcome.  Recurrences tend to occur within the first year after treatment.  Future research will compare this prepn. to the gold std. treatment for superficial BCC - surgical excision.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri0Dwxe6VLfrVg90H21EOLACvtfcHk0lg2fjI-dThdUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVShtL3J&md5=21eeab1a2735cc7d1b190f04c9c08e3a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2147%2FCCID.S3507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FCCID.S3507%26sid%3Dliteratum%253Aachs%26aulast%3DRaasch%26aufirst%3DB.%26atitle%3DManagement%2520of%2520superficial%2520basal%2520cell%2520carcinoma%253A%2520focus%2520on%2520imiquimod%26jtitle%3DClin.%252C%2520Cosmet.%2520Invest.%2520Dermatol.%26date%3D2009%26volume%3D2%26spage%3D65%26epage%3D67%26doi%3D10.2147%2FCCID.S3507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Greiff, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cervin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahlstroem-Emanuelsson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almqvist, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolata, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eriksson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoegestaett, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaellen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norlen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjölin, I.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widegren, H.</span><span> </span><span class="NLM_article-title">Repeated intranasal TLR7 stimulation reduces allergen responsiveness in allergic rhinitis</span> <span class="citation_source-journal">Respir. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="refDoi"> DOI: 10.1186/1465-9921-13-53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1186%2F1465-9921-13-53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=22726593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCnsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=53&author=L.+Greiffauthor=A.+Cervinauthor=C.+Ahlstroem-Emanuelssonauthor=G.+Almqvistauthor=M.+Anderssonauthor=J.+Dolataauthor=L.+Erikssonauthor=E.+Hoegestaettauthor=A.+Kaellenauthor=P.+Norlenauthor=I.-L.+Sj%C3%B6linauthor=H.+Widegren&title=Repeated+intranasal+TLR7+stimulation+reduces+allergen+responsiveness+in+allergic+rhinitis&doi=10.1186%2F1465-9921-13-53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Repeated intranasal TLR7 stimulation reduces allergen responsiveness in allergic rhinitis</span></div><div class="casAuthors">Greiff, Lennart; Cervin, Anders; Ahlstroem-Emanuelsson, Cecilia; Almqvist, Gun; Andersson, Morgan; Dolata, Jan; Eriksson, Leif; Hoegestaett, Edward; Kaellen, Anders; Norlen, Per; Sjoelin, Inga-Lisa; Widegren, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Respiratory Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">53</span>CODEN:
                <span class="NLM_cas:coden">RREEBZ</span>;
        ISSN:<span class="NLM_cas:issn">1465-993X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Interactions between Th1 and Th2 immune responses are of importance to the onset and development of allergic disorders.  A Toll-like receptor 7 agonist such as AZD8848 may have potential as a treatment for allergic airway disease by skewing the immune system away from a Th2 profile.  Objective: To evaluate the efficacy and safety of intranasal AZD8848.  Methods: In a placebo-controlled single ascending dose study, AZD8848 (0.3-600 μg) was given intranasally to 48 healthy subjects and 12 patients with allergic rhinitis.  In a placebo-controlled repeat challenge/treatment study, AZD8848 (30 and 60 μg) was given once weekly for five weeks to 74 patients with allergic rhinitis out of season: starting 24 h after the final dose, daily allergen challenges were given for seven days.  Safety, tolerability, pharmacokinetics and biomarkers were monitored.  During the allergen challenge series, nasal symptoms and lavage fluid levels of tryptase and α2-macroglobulin, reflecting mast cell activity and plasma exudation, were monitored.  Results: AZD8848 produced reversible blood lymphocyte redns. and dose-dependent flu-like symptoms: 30-100 μg produced consistent yet tolerable effects.  Plasma interleukin-1 receptor antagonist was elevated after administration of AZD8848, reflecting interferon prodn. secondary to TLR7 stimulation.  At repeat challenge/treatment, AZD8848 reduced nasal symptoms recorded ten minutes after allergen challenge up to eight days after the final dose.  Tryptase and α2-macroglobulin were also reduced by AZD8848.  Conclusions: Repeated intranasal stimulation of Toll-like receptor 7 by AZD8848 was safe and produced a sustained redn. in the responsiveness to allergen in allergic rhinitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoehN4HRRB9K7Vg90H21EOLACvtfcHk0liTmYuZJgTJcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCnsLrP&md5=1427aaddf24c0946cc1fbcedb0602d7d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1465-9921-13-53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1465-9921-13-53%26sid%3Dliteratum%253Aachs%26aulast%3DGreiff%26aufirst%3DL.%26aulast%3DCervin%26aufirst%3DA.%26aulast%3DAhlstroem-Emanuelsson%26aufirst%3DC.%26aulast%3DAlmqvist%26aufirst%3DG.%26aulast%3DAndersson%26aufirst%3DM.%26aulast%3DDolata%26aufirst%3DJ.%26aulast%3DEriksson%26aufirst%3DL.%26aulast%3DHoegestaett%26aufirst%3DE.%26aulast%3DKaellen%26aufirst%3DA.%26aulast%3DNorlen%26aufirst%3DP.%26aulast%3DSj%25C3%25B6lin%26aufirst%3DI.-L.%26aulast%3DWidegren%26aufirst%3DH.%26atitle%3DRepeated%2520intranasal%2520TLR7%2520stimulation%2520reduces%2520allergen%2520responsiveness%2520in%2520allergic%2520rhinitis%26jtitle%3DRespir.%2520Res.%26date%3D2012%26volume%3D13%26spage%3D53%26doi%3D10.1186%2F1465-9921-13-53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Diebold, S. S.</span><span> </span><span class="NLM_article-title">Recognition of viral single-stranded RNA by toll-like receptors</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">–</span> <span class="NLM_lpage">823</span><span class="refDoi"> DOI: 10.1016/j.addr.2007.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2Fj.addr.2007.11.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2008&pages=813-823&author=S.+S.+Diebold&title=Recognition+of+viral+single-stranded+RNA+by+toll-like+receptors&doi=10.1016%2Fj.addr.2007.11.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2007.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2007.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DDiebold%26aufirst%3DS.%2BS.%26atitle%3DRecognition%2520of%2520viral%2520single-stranded%2520RNA%2520by%2520toll-like%2520receptors%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2008%26volume%3D60%26spage%3D813%26epage%3D823%26doi%3D10.1016%2Fj.addr.2007.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Bonfanti, J.-F.; Fortin, J. M. C.; Muller, P.; Doublet, F. M. M.; Raboisson, P. J-M. B.; Arnoult, E. P. A.</span><span> </span><span class="NLM_article-title">Preparation of macrocyclic purines for the treatment of viral infections</span>. PCT Int. Appl. WO2014009509,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=J.-F.+Bonfanti&author=J.+M.+C.+Fortin&author=P.+Muller&author=F.+M.+M.+Doublet&author=P.+J-M.+B.+Raboisson&author=E.+P.+A.+Arnoult&title=Preparation+of+macrocyclic+purines+for+the+treatment+of+viral+infections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBonfanti%26aufirst%3DJ.-F.%26atitle%3DPreparation%2520of%2520macrocyclic%2520purines%2520for%2520the%2520treatment%2520of%2520viral%2520infections%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Meyer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockfleth, E.</span><span> </span><span class="NLM_article-title">Resiquimod, a topical drug for viral skin lesions and skin cancer</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">149</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1517/13543784.2013.749236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1517%2F13543784.2013.749236" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2013&pages=149-159&author=T.+Meyerauthor=C.+Surberauthor=L.+E.+Frenchauthor=E.+Stockfleth&title=Resiquimod%2C+a+topical+drug+for+viral+skin+lesions+and+skin+cancer&doi=10.1517%2F13543784.2013.749236"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1517%2F13543784.2013.749236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2013.749236%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DSurber%26aufirst%3DC.%26aulast%3DFrench%26aufirst%3DL.%2BE.%26aulast%3DStockfleth%26aufirst%3DE.%26atitle%3DResiquimod%252C%2520a%2520topical%2520drug%2520for%2520viral%2520skin%2520lesions%2520and%2520skin%2520cancer%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2013%26volume%3D22%26spage%3D149%26epage%3D159%26doi%3D10.1517%2F13543784.2013.749236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Kurimoto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogino, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichii, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isobe, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tobe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takaku, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sajiki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirota, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawakami, H.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of 2-substituted 8-hydroxyadenines as potent interferon inducers with improved oral bioavailabilities</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1099</span><span class="refDoi"> DOI: 10.1016/j.bmc.2003.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2Fj.bmc.2003.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=14980621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFWht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=1091-1099&author=A.+Kurimotoauthor=T.+Oginoauthor=S.+Ichiiauthor=Y.+Isobeauthor=M.+Tobeauthor=H.+Ogitaauthor=H.+Takakuauthor=H.+Sajikiauthor=K.+Hirotaauthor=H.+Kawakami&title=Synthesis+and+evaluation+of+2-substituted+8-hydroxyadenines+as+potent+interferon+inducers+with+improved+oral+bioavailabilities&doi=10.1016%2Fj.bmc.2003.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of 2-substituted 8-hydroxyadenines as potent interferon inducers with improved oral bioavailabilities</span></div><div class="casAuthors">Kurimoto, Ayumu; Ogino, Tetsuhiro; Ichii, Shinji; Isobe, Yoshiaki; Tobe, Masanori; Ogita, Haruhisa; Takaku, Haruo; Sajiki, Hironao; Hirota, Kosaku; Kawakami, Hajime</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1091-1099</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In order to create novel compds. which possess potent interferon (IFN) inducing activities with excellent oral bioavailabilities, a series of 8-hydroxyadenines, which have various alkoxy or alkylthio moieties at the adenine C(2)-position, were synthesized and evaluated.  The introduction of hydrophobic groups was not considered to be effective for potentiating the IFN-inducing activity, but several compds. having hydrophilic groups were effective.  Among the compds. tested, compd. I induced IFN from the dosage of 0.03 mg/kg, which was approx. 100-fold more potent than that of Imiquimod, and showed an excellent oral bioavailability (F=40%) which was 10-fold improved over the lead compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqY8-Ou6UMlbVg90H21EOLACvtfcHk0liTmYuZJgTJcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFWht7g%253D&md5=7786077b1ccac729aaaa96ec4254c1cf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2003.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2003.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DKurimoto%26aufirst%3DA.%26aulast%3DOgino%26aufirst%3DT.%26aulast%3DIchii%26aufirst%3DS.%26aulast%3DIsobe%26aufirst%3DY.%26aulast%3DTobe%26aufirst%3DM.%26aulast%3DOgita%26aufirst%3DH.%26aulast%3DTakaku%26aufirst%3DH.%26aulast%3DSajiki%26aufirst%3DH.%26aulast%3DHirota%26aufirst%3DK.%26aulast%3DKawakami%26aufirst%3DH.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%25202-substituted%25208-hydroxyadenines%2520as%2520potent%2520interferon%2520inducers%2520with%2520improved%2520oral%2520bioavailabilities%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D1091%26epage%3D1099%26doi%3D10.1016%2Fj.bmc.2003.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Kazaoka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sajiki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirota, K.</span><span> </span><span class="NLM_article-title">Synthesis of 6-substituted 9-benzyl-8-hydroxypurines with potential interferon-inducing activity</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">611</span><span class="refDoi"> DOI: 10.1248/cpb.51.608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1248%2Fcpb.51.608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=12736468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltVejsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2003&pages=608-611&author=K.+Kazaokaauthor=H.+Sajikiauthor=K.+Hirota&title=Synthesis+of+6-substituted+9-benzyl-8-hydroxypurines+with+potential+interferon-inducing+activity&doi=10.1248%2Fcpb.51.608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of 6-substituted 9-benzyl-8-hydroxypurines with potential interferon-inducing activity</span></div><div class="casAuthors">Kazaoka, Kazunori; Sajiki, Hironao; Hirota, Kosaku</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">608-611</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Various 6-substituted 9-benzyl-8-hydroxypurines I [R = NH2, H, SH, SMe, OH, OMe, NHR1, NHCOR2; R1 = Me, Et, CH2Et, Bu, CH2CHMe2; R2 = Me, CF3, Ph] were synthesized in order to investigate the structure-activity relationship at the 6-position of 9-benzyl-8-hydroxyadenine [I R = NH2 (II)], which is a lead compd. for the screening of interferon (IFN)-inducing activity.  6-Unsubstituted, mercapto-, methylthio- and hydroxy-substituted 9-benzyl-8-hydroxypurines I (R = H, SH, SMe, OH) were prepd. from 5-amino-1-benzyl-4-cyano-2-hydroxyimidazole.  Synthesis of a 6-methoxy analog I (R = OMe) was conducted from 5-amino-4-benzylamino-6-chloropyrimidine.  6-Alkylamino and acylaminopurines I [R = NHR1, NHCOR2] were also prepd. by reductive alkylation and acylation of II, resp.  Since these compds. indicated no activity, it was found that a free amino group of II is required for the expression of IFN-inducing activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogl5sHW5hz77Vg90H21EOLACvtfcHk0ljwUuEt4R4heg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltVejsb0%253D&md5=363df4fceae05b8aa34a2a6244d72cfa</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1248%2Fcpb.51.608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.51.608%26sid%3Dliteratum%253Aachs%26aulast%3DKazaoka%26aufirst%3DK.%26aulast%3DSajiki%26aufirst%3DH.%26aulast%3DHirota%26aufirst%3DK.%26atitle%3DSynthesis%2520of%25206-substituted%25209-benzyl-8-hydroxypurines%2520with%2520potential%2520interferon-inducing%2520activity%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2003%26volume%3D51%26spage%3D608%26epage%3D611%26doi%3D10.1248%2Fcpb.51.608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Gerster, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomai, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birmachu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bomersine, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imbertson, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knafla, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maye, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikolaides, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oneyemi, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkhurst, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pecore, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiter, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scribner, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testerman, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weeks, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andre, J.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lagain, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastard, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupu, M.</span><span> </span><span class="NLM_article-title">Synthesis and structure-activity-relationships of 1<i>H</i>-imidazo[4,5-<i>c</i>]quinolines that induce interferon production</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">3481</span><span class="NLM_x">–</span> <span class="NLM_lpage">3491</span><span class="refDoi"> DOI: 10.1021/jm049211v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049211v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3481-3491&author=J.+F.+Gersterauthor=K.+J.+Lindstromauthor=R.+L.+Millerauthor=M.+A.+Tomaiauthor=W.+Birmachuauthor=S.+N.+Bomersineauthor=S.+J.+Gibsonauthor=L.+M.+Imbertsonauthor=J.+R.+Jacobsonauthor=R.+T.+Knaflaauthor=P.+V.+Mayeauthor=N.+Nikolaidesauthor=F.+Y.+Oneyemiauthor=G.+W.+Parkhurstauthor=S.+E.+Pecoreauthor=M.+J.+Reiterauthor=L.+S.+Scribnerauthor=T.+L.+Testermanauthor=N.+J.+Thompsonauthor=T.+L.+Wagnerauthor=C.+E.+Weeksauthor=J.-D.+Andreauthor=D.+Lagainauthor=Y.+Bastardauthor=M.+Lupu&title=Synthesis+and+structure-activity-relationships+of+1H-imidazo%5B4%2C5-c%5Dquinolines+that+induce+interferon+production&doi=10.1021%2Fjm049211v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm049211v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049211v%26sid%3Dliteratum%253Aachs%26aulast%3DGerster%26aufirst%3DJ.%2BF.%26aulast%3DLindstrom%26aufirst%3DK.%2BJ.%26aulast%3DMiller%26aufirst%3DR.%2BL.%26aulast%3DTomai%26aufirst%3DM.%2BA.%26aulast%3DBirmachu%26aufirst%3DW.%26aulast%3DBomersine%26aufirst%3DS.%2BN.%26aulast%3DGibson%26aufirst%3DS.%2BJ.%26aulast%3DImbertson%26aufirst%3DL.%2BM.%26aulast%3DJacobson%26aufirst%3DJ.%2BR.%26aulast%3DKnafla%26aufirst%3DR.%2BT.%26aulast%3DMaye%26aufirst%3DP.%2BV.%26aulast%3DNikolaides%26aufirst%3DN.%26aulast%3DOneyemi%26aufirst%3DF.%2BY.%26aulast%3DParkhurst%26aufirst%3DG.%2BW.%26aulast%3DPecore%26aufirst%3DS.%2BE.%26aulast%3DReiter%26aufirst%3DM.%2BJ.%26aulast%3DScribner%26aufirst%3DL.%2BS.%26aulast%3DTesterman%26aufirst%3DT.%2BL.%26aulast%3DThompson%26aufirst%3DN.%2BJ.%26aulast%3DWagner%26aufirst%3DT.%2BL.%26aulast%3DWeeks%26aufirst%3DC.%2BE.%26aulast%3DAndre%26aufirst%3DJ.-D.%26aulast%3DLagain%26aufirst%3DD.%26aulast%3DBastard%26aufirst%3DY.%26aulast%3DLupu%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520structure-activity-relationships%2520of%25201H-imidazo%255B4%252C5-c%255Dquinolines%2520that%2520induce%2520interferon%2520production%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3481%26epage%3D3491%26doi%3D10.1021%2Fjm049211v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Svensson, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jemth, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desroses, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loseva, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helleday, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Högbom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stenmark, P.</span><span> </span><span class="NLM_article-title">Crystal structure of human MTH1 and the 8-oxo-dGMP product complex</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">585</span><span class="NLM_x">, </span> <span class="NLM_fpage">2617</span><span class="NLM_x">–</span> <span class="NLM_lpage">2621</span><span class="refDoi"> DOI: 10.1016/j.febslet.2011.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2Fj.febslet.2011.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=21787772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVait7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=585&publication_year=2011&pages=2617-2621&author=L.+M.+Svenssonauthor=A.+S.+Jemthauthor=M.+Desrosesauthor=O.+Losevaauthor=T.+Helledayauthor=M.+H%C3%B6gbomauthor=P.+Stenmark&title=Crystal+structure+of+human+MTH1+and+the+8-oxo-dGMP+product+complex&doi=10.1016%2Fj.febslet.2011.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of human MTH1 and the 8-oxo-dGMP product complex</span></div><div class="casAuthors">Svensson, Linda M.; Jemth, Ann-Sofie; Desroses, Matthieu; Loseva, Olga; Helleday, Thomas; Hoegbom, Martin; Stenmark, Pal</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">585</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2617-2621</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MTH1 hydrolyzes oxidized nucleotide triphosphates, thereby preventing them from being incorporated into DNA.  We here present the structures of human MTH1 (1.9 Å) and its complex with the product 8-oxo-dGMP (1.8 Å).  Unexpectedly, MTH1 binds the nucleotide in the anti conformation with no direct interaction between the 8-oxo group and the protein.  We suggest that the specificity depends on the stabilization of an enol tautomer of the 8-oxo form of dGTP.  The binding of the product induces no major structural changes.  The structures reveal the mode of nucleotide binding in MTH1 and provide the structural basis for inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9fYmwWXglD7Vg90H21EOLACvtfcHk0ljwUuEt4R4heg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVait7bM&md5=711a922bab64d61641151ff0ecf6bbb1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2011.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2011.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSvensson%26aufirst%3DL.%2BM.%26aulast%3DJemth%26aufirst%3DA.%2BS.%26aulast%3DDesroses%26aufirst%3DM.%26aulast%3DLoseva%26aufirst%3DO.%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DH%25C3%25B6gbom%26aufirst%3DM.%26aulast%3DStenmark%26aufirst%3DP.%26atitle%3DCrystal%2520structure%2520of%2520human%2520MTH1%2520and%2520the%25208-oxo-dGMP%2520product%2520complex%26jtitle%3DFEBS%2520Lett.%26date%3D2011%26volume%3D585%26spage%3D2617%26epage%3D2621%26doi%3D10.1016%2Fj.febslet.2011.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Wei, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamitzky, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heckl, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rössle, S. C.</span><span> </span><span class="NLM_article-title">Homology modeling of human toll-like receptors TLR7, 8, and 9 ligand-binding domains</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1684</span><span class="NLM_x">–</span> <span class="NLM_lpage">1691</span><span class="refDoi"> DOI: 10.1002/pro.186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1002%2Fpro.186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=19521997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Clsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2009&pages=1684-1691&author=T.+Weiauthor=J.+Gongauthor=F.+Jamitzkyauthor=W.+M.+Hecklauthor=R.+W.+Starkauthor=S.+C.+R%C3%B6ssle&title=Homology+modeling+of+human+toll-like+receptors+TLR7%2C+8%2C+and+9+ligand-binding+domains&doi=10.1002%2Fpro.186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Homology modeling of human Toll-like receptors TLR7, 8, and 9 ligand-binding domains</span></div><div class="casAuthors">Wei, Tiandi; Gong, Jing; Jamitzky, Ferdinand; Heckl, Wolfgang M.; Stark, Robert W.; Roessle, Shaila C.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1684-1691</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Toll-like receptors (TLRs) play a key role in the innate immune system.  The TLR7, 8, and 9 compose a family of intracellularly localized TLRs that signal in response to pathogen-derived nucleic acids.  So far, there are no crystallog. structures for TLR7, 8, and 9.  For this reason, their ligand-binding mechanisms are poorly understood.  To enable first predictions of the receptor-ligand interaction sites, the authors developed three-dimensional structures for the leucine-rich repeat ectodomains of human TLR7, 8, and 9 based on homol. modeling.  To achieve a high sequence similarity between targets and templates, structural segments from all known TLR ectodomain structures (human TLR1/2/3/4 and mouse TLR3/4) were used as candidate templates for the modeling.  The resulting models support previously reported essential ligand-binding residues.  They also provide a basis to identify three potential receptor dimerization mechanisms.  Addnl., potential ligand-binding residues are identified using combined procedures.  The authors suggest further investigations of these residues through mutation expts.  This modeling approach can be extended to other members of the TLR family or other repetitive proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa9t75WYAFLbVg90H21EOLACvtfcHk0lgCL52J4LmMPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Clsrw%253D&md5=a8b8b1190ef2f70939c0d302fa5b9f4a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fpro.186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.186%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DT.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DJamitzky%26aufirst%3DF.%26aulast%3DHeckl%26aufirst%3DW.%2BM.%26aulast%3DStark%26aufirst%3DR.%2BW.%26aulast%3DR%25C3%25B6ssle%26aufirst%3DS.%2BC.%26atitle%3DHomology%2520modeling%2520of%2520human%2520toll-like%2520receptors%2520TLR7%252C%25208%252C%2520and%25209%2520ligand-binding%2520domains%26jtitle%3DProtein%2520Sci.%26date%3D2009%26volume%3D18%26spage%3D1684%26epage%3D1691%26doi%3D10.1002%2Fpro.186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Molina, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jafari, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ignatushchenko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreekumar, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">341</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0lgCL52J4LmMPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Persengiev, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span> </span><span class="NLM_article-title">Nonspecific, concentration-dependent stimulation and repression of mammalian gene expression by small interfering RNAs (siRNAs)</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">18</span><span class="refDoi"> DOI: 10.1261/rna5160904</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1261%2Frna5160904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=14681580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFWguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=12-18&author=S.+P.+Persengievauthor=X.+Zhuauthor=M.+R.+Green&title=Nonspecific%2C+concentration-dependent+stimulation+and+repression+of+mammalian+gene+expression+by+small+interfering+RNAs+%28siRNAs%29&doi=10.1261%2Frna5160904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nonspecific, concentration-dependent stimulation and repression of mammalian gene expression by small interfering RNAs (siRNAs)</span></div><div class="casAuthors">Persengiev, Stephan P.; Zhu, Xiaochun; Green, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-18</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">RNA interference is an evolutionarily conserved process in which expression of a specific gene is post-transcriptionally inhibited by a small interfering RNA (siRNA), which recognizes a complementary mRNA and induces its degrdn.  Currently, RNA interference is being used extensively to inhibit expression of specific genes for exptl. and therapeutic purposes.  For applications in mammalian cells, siRNAs are designed to be <-30 base pairs to avoid nonspecific effects that arise from inducing the cellular double-stranded RNA (dsRNA)-dependent protein kinase (PKR) response.  Here we perform expression profiling in mammalian tissue-culture cells treated under std. conditions with conventional 21-bp siRNAs and find, unexpectedly, that >1000 genes involved in diverse cellular functions are nonspecifically stimulated or repressed.  The effects on gene expression are dependent upon siRNA concn. and are stable throughout the course of siRNA treatment.  Our results can be explained by previous studies showing that dsRNAs can affect multiple signaling and transcription pathways in addn. to PKR.  The potential for this widespread, nonspecific effect on mammalian gene expression must be carefully considered in the design of siRNA expts. and therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRKVQaY23i27Vg90H21EOLACvtfcHk0lhZaEThb3y3sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFWguw%253D%253D&md5=d638a7197d6dcabb9b2cdd89e1a65527</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1261%2Frna5160904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna5160904%26sid%3Dliteratum%253Aachs%26aulast%3DPersengiev%26aufirst%3DS.%2BP.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26atitle%3DNonspecific%252C%2520concentration-dependent%2520stimulation%2520and%2520repression%2520of%2520mammalian%2520gene%2520expression%2520by%2520small%2520interfering%2520RNAs%2520%2528siRNAs%2529%26jtitle%3DRNA%26date%3D2004%26volume%3D10%26spage%3D12%26epage%3D18%26doi%3D10.1261%2Frna5160904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Tarcsay, Á.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keserű, G. M.</span><span> </span><span class="NLM_article-title">Contributions of molecular properties to drug promiscuity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1789</span><span class="NLM_x">–</span> <span class="NLM_lpage">1795</span><span class="refDoi"> DOI: 10.1021/jm301514n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301514n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC3sXns1ehug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1789-1795&author=%C3%81.+Tarcsayauthor=G.+M.+Keser%C5%B1&title=Contributions+of+molecular+properties+to+drug+promiscuity&doi=10.1021%2Fjm301514n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Contributions of Molecular Properties to Drug Promiscuity</span></div><div class="casAuthors">Tarcsay, Akos; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1789-1795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In contrast to designed polypharmacol. that can result in efficient drugs for complex disorders, unintended drug promiscuity has detrimental contribution to side effects and toxicol.  Characterization of promiscuous compds. enhances the understanding of complex interaction patterns and aids the design of compds. with broader selectivity against off-targets that has a major impact on medicinal chem. outcome.  In this Miniperspective we provide insights to the effect of physicochem. parameters on promiscuity.  Information collected from recent, large-scale in vitro studies enabled us to discuss the relationships between physicochem. properties and promiscuity in detail.  In light of these data, lipophilicity and basic character have the highest influence.  On the basis of the accumulated knowledge, we propose the extensive use of pre- and postsynthesis metrics, as well as strict control of physicochem. properties during medicinal chem. optimizations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf1J_jqkK3DrVg90H21EOLACvtfcHk0lgj5yp6TIyYKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXns1ehug%253D%253D&md5=c5763ee1aed897ff19e5a2344b95b9c1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm301514n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301514n%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3D%25C3%2581.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DContributions%2520of%2520molecular%2520properties%2520to%2520drug%2520promiscuity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1789%26epage%3D1795%26doi%3D10.1021%2Fjm301514n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buser-Doepner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederiksen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimshaw, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller-Fahrnow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Hagan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundström, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo-Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walpole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+M.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lgj5yp6TIyYKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Tajiri, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekiguchi, M.</span><span> </span><span class="NLM_article-title">Functional cooperation of MutT, MutM and MutY proteins in preventing mutations caused by spontaneous oxidation of guanine nucleotide in escherichia coli</span> <span class="citation_source-journal">Mutat. Res., DNA Repair</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">336</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">267</span><span class="refDoi"> DOI: 10.1016/0921-8777(94)00062-B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2F0921-8777%2894%2900062-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=7739614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADyaK2MXlsFGls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=1995&pages=257-267&author=T.+Tajiriauthor=H.+Makiauthor=M.+Sekiguchi&title=Functional+cooperation+of+MutT%2C+MutM+and+MutY+proteins+in+preventing+mutations+caused+by+spontaneous+oxidation+of+guanine+nucleotide+in+escherichia+coli&doi=10.1016%2F0921-8777%2894%2900062-B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20aR"><div class="casContent"><span class="casTitleNuber">20a</span><div class="casTitle"><span class="NLM_cas:atitle">Functional cooperation of MutT, MutM and MutY proteins in preventing mutations caused by spontaneous oxidation of guanine nucleotide in Escherichia coli</span></div><div class="casAuthors">Tajiri, Tatsurou; Maki, Hisaji; Sekiguchi, Mutsuo</div><div class="citationInfo"><span class="NLM_cas:title">Mutation Research, DNA Repair</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">257-67</span>CODEN:
                <span class="NLM_cas:coden">MRDRBE</span>;
        ISSN:<span class="NLM_cas:issn">0921-8777</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">8-Oxo-dGTP (8-oxo-7,8-dihydrodeoxyguanosine triphosphate) is a potent mutagenic substrate for DNA synthesis.  The accumulation of 8-oxo-dGTP in the nucleotide pool induces G:C→T:A transversion as well as A:T→C:G transversion, and Escherichia coli cells possess mechanisms for preventing such mutations.  The mutT gene product specifically hydrolyzes 8-oxo-dGTP to the monophosphate form while the mutM and the mutY gene products function to correct mispairs caused by incorporation of 8-oxoguanine into DNA.  From analyses of forward mutations induced in cells lacking 8-oxo-dGTPase (MutT protein) and/or repair enzymes that suppress mutations caused by 8-oxoguanine in DNA (MutM and MutY proteins), cooperative functions of these proteins in control of the spontaneous mutagenesis became evident.  In mutator strains lacking MutT and/or MutM proteins, 8-oxoguanine of DNA increased to a concn. expected from the increased rate of mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBre8HrgmQwrVg90H21EOLACvtfcHk0lj44ghhc_AYyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlsFGls7s%253D&md5=69eca7484312849d8619a6a534191c77</span></div><a href="/servlet/linkout?suffix=cit20a&amp;dbid=16384&amp;doi=10.1016%2F0921-8777%2894%2900062-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0921-8777%252894%252900062-B%26sid%3Dliteratum%253Aachs%26aulast%3DTajiri%26aufirst%3DT.%26aulast%3DMaki%26aufirst%3DH.%26aulast%3DSekiguchi%26aufirst%3DM.%26atitle%3DFunctional%2520cooperation%2520of%2520MutT%252C%2520MutM%2520and%2520MutY%2520proteins%2520in%2520preventing%2520mutations%2520caused%2520by%2520spontaneous%2520oxidation%2520of%2520guanine%2520nucleotide%2520in%2520escherichia%2520coli%26jtitle%3DMutat.%2520Res.%252C%2520DNA%2520Repair%26date%3D1995%26volume%3D336%26spage%3D257%26epage%3D267%26doi%3D10.1016%2F0921-8777%2894%2900062-B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit20b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Rotman, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuzminov, A.</span><span> </span><span class="NLM_article-title">The mutT defect does not elevate chromosomal fragmentation in escherichia coli because of the surprisingly low levels of MutM/MutY-recognized DNA modifications</span> <span class="citation_source-journal">J. Bacteriol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">189</span><span class="NLM_x">, </span> <span class="NLM_fpage">6976</span><span class="NLM_x">–</span> <span class="NLM_lpage">6988</span><span class="refDoi"> DOI: 10.1128/JB.00776-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1128%2FJB.00776-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=17616589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKmsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2007&pages=6976-6988&author=E.+Rotmanauthor=A.+Kuzminov&title=The+mutT+defect+does+not+elevate+chromosomal+fragmentation+in+escherichia+coli+because+of+the+surprisingly+low+levels+of+MutM%2FMutY-recognized+DNA+modifications&doi=10.1128%2FJB.00776-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20bR"><div class="casContent"><span class="casTitleNuber">20b</span><div class="casTitle"><span class="NLM_cas:atitle">The mutT defect does not elevate chromosomal fragmentation in Escherichia coli because of the surprisingly low levels of MutM/MutY-recognized DNA modifications</span></div><div class="casAuthors">Rotman, Ella; Kuzminov, Andrei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6976-6988</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Nucleotide pool sanitizing enzymes Dut (dUTPase), RdgB (dITPase), and MutT (8-oxo-dGTPase) of Escherichia coli hydrolyze noncanonical DNA precursors to prevent incorporation of base analogs into DNA.  Previous studies reported dramatic AT→CG mutagenesis in mutT mutants, suggesting a considerable d. of 8-oxo-G in DNA that should cause frequent excision and chromosomal fragmentation, irreparable in the absence of RecBCD-catalyzed repair and similar to the lethality of dut recBC and rdgB recBC double mutants.  In contrast, we found mutT recBC double mutants viable with no signs of chromosomal fragmentation.  Overprodn. of the MutM and MutY DNA glycosylases, both acting on DNA contg. 8-oxo-G, still yields no lethality in mutT recBC double mutants.  Plasmid DNA, extd. from mutT mutM double mutant cells and treated with MutM in vitro, shows no increased relaxation, indicating no addnl. 8-oxo-G modifications.  Our ΔmutT allele elevates the AT→CG transversion rate 27,000-fold, consistent with published reports.  However, the rate of AT→CG transversions in our mutT+ progenitor strain is some two orders of magnitude lower than in previous studies, which lowers the abs. rate of mutagenesis in ΔmutT derivs., translating into less than four 8-oxo-G modifications per genome equiv., which is too low to cause the expected effects.  Introduction of various addnl. mutations in the ΔmutT strain or treatment with oxidative agents failed to increase the mutagenesis even twofold.  We conclude that, in contrast to the previous studies, there is not enough 8-oxo-G in the DNA of mutT mutants to cause elevated excision repair that would trigger chromosomal fragmentation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriNZK-Muh6_LVg90H21EOLACvtfcHk0lj44ghhc_AYyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKmsL3F&md5=91a568aaefcdb5a41152dc17147a50d3</span></div><a href="/servlet/linkout?suffix=cit20b&amp;dbid=16384&amp;doi=10.1128%2FJB.00776-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.00776-07%26sid%3Dliteratum%253Aachs%26aulast%3DRotman%26aufirst%3DE.%26aulast%3DKuzminov%26aufirst%3DA.%26atitle%3DThe%2520mutT%2520defect%2520does%2520not%2520elevate%2520chromosomal%2520fragmentation%2520in%2520escherichia%2520coli%2520because%2520of%2520the%2520surprisingly%2520low%2520levels%2520of%2520MutM%252FMutY-recognized%2520DNA%2520modifications%26jtitle%3DJ.%2520Bacteriol.%26date%3D2007%26volume%3D189%26spage%3D6976%26epage%3D6988%26doi%3D10.1128%2FJB.00776-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit20c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Tsuzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egashira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kura, S.</span><span> </span><span class="NLM_article-title">Analysis of MTH1 gene function in mice with targeted mutagenesis</span> <span class="citation_source-journal">Mutat. Res., Fundam. Mol. Mech. Mutagen.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">477</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">78</span><span class="refDoi"> DOI: 10.1016/S0027-5107(01)00108-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=10.1016%2FS0027-5107%2801%2900108-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=11376688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjslemsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2001&pages=71-78&author=T.+Tsuzukiauthor=A.+Egashiraauthor=S.+Kura&title=Analysis+of+MTH1+gene+function+in+mice+with+targeted+mutagenesis&doi=10.1016%2FS0027-5107%2801%2900108-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20cR"><div class="casContent"><span class="casTitleNuber">20c</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of MTH1 gene function in mice with targeted mutagenesis</span></div><div class="casAuthors">Tsuzuki, T.; Egashira, A.; Kura, S.</div><div class="citationInfo"><span class="NLM_cas:title">Mutation Research, Fundamental and Molecular Mechanisms of Mutagenesis</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">1,2</span>),
    <span class="NLM_cas:pages">71-78</span>CODEN:
                <span class="NLM_cas:coden">MUREAV</span>;
        ISSN:<span class="NLM_cas:issn">0027-5107</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 45 refs. and emphasis on authors research.  Oxidative DNA damage is thought to contribute to carcinogenesis, aging, and neurol. degeneration.  Further, the cumulative risk of cancer increases dramatically with age in humans.  In general terms, cancer can be regarded as a degenerative disease of aging.  There is evidence for the accumulation of oxidative DNA damage with age based on studies mainly measuring an increase in 8-oxoguanine.  8-Oxo-7,8-dihydro-2'-deoxyguanosine 5'-triphosphate (8-oxo-dGTP) is formed in the nucleotide pool of a cell during normal cellular metab.  When 8-oxoguanine is incorporated into DNA causes mutation.  Organisms possess 8-oxo-dGTPase, an enzyme that specifically degrades 8-oxo-dGTP to 8-oxo-dGMP.  To analyze the function of MTH1 with 8-oxo-dGTPase activity in vivo, we generated a mouse line carrying a mutant MTH1 allele created by targeted gene disruption.  MTH1 homozygous mutant mice were found to have a phys. normal appearance, but seemed to have lost 8-oxo-dGTPase activity in liver exts.  When we examd. the susceptibility of the mutant mice to spontaneous tumorigenesis, no significant difference was obsd. in survival rate of MTH1+/+ and MTH1-/- mice.  However, pathol. examn. revealed a statistically significant difference in the incidence of tumors.  More tumors were formed in lungs, livers, and stomachs of MTH1-/- mice than in those of the wild type mice.  These studies with MTH1-null mutant mice provided an important insight into the role of this nucleotide sanitization enzyme in terms of the spontaneous tumorigenesis as well as mutagenesis caused by the oxygen-induced DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_7QRMnkJFLVg90H21EOLACvtfcHk0ljpIOg39UvsCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjslemsr8%253D&md5=2b38b2d00ac6d0c6100cc52465312ecb</span></div><a href="/servlet/linkout?suffix=cit20c&amp;dbid=16384&amp;doi=10.1016%2FS0027-5107%2801%2900108-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0027-5107%252801%252900108-7%26sid%3Dliteratum%253Aachs%26aulast%3DTsuzuki%26aufirst%3DT.%26aulast%3DEgashira%26aufirst%3DA.%26aulast%3DKura%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520MTH1%2520gene%2520function%2520in%2520mice%2520with%2520targeted%2520mutagenesis%26jtitle%3DMutat.%2520Res.%252C%2520Fundam.%2520Mol.%2520Mech.%2520Mutagen.%26date%3D2001%26volume%3D477%26spage%3D71%26epage%3D78%26doi%3D10.1016%2FS0027-5107%2801%2900108-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N1U','PDB','4N1U'); return false;">PDB: 4N1U</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i36"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01760">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_38473"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01760">10.1021/acs.jmedchem.5b01760</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Protocols for the enzyme, SPR, NMR, and cell assays, synthetic methods for the remaining examples, together with crystallographic information, and kinase panel and secondary pharmacology selectivity data for compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_001.pdf">jm5b01760_si_001.pdf (2.56 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01760/suppl_file/jm5b01760_si_002.csv">jm5b01760_si_002.csv (3.86 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01760&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01760%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-6%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01760" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994b072f593ce2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
