
<html lang="en"     class="pb-page"  data-request-id="5b72fc96-9433-441c-a999-604303241a1b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.7b01293;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2017.60.issue-24;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Small-Molecule Inhibitors of Ubiquitin Specific Protease 7 (USP7) Using Integrated NMR and in Silico Techniques" /></meta><meta name="dc.Creator" content="Paola  Di Lello" /></meta><meta name="dc.Creator" content="Richard  Pastor" /></meta><meta name="dc.Creator" content="Jeremy M.  Murray" /></meta><meta name="dc.Creator" content="Robert A.  Blake" /></meta><meta name="dc.Creator" content="Frederick  Cohen" /></meta><meta name="dc.Creator" content="Terry D.  Crawford" /></meta><meta name="dc.Creator" content="Joy  Drobnick" /></meta><meta name="dc.Creator" content="Jason  Drummond" /></meta><meta name="dc.Creator" content="Lorna  Kategaya" /></meta><meta name="dc.Creator" content="Tracy  Kleinheinz" /></meta><meta name="dc.Creator" content="Till  Maurer" /></meta><meta name="dc.Creator" content="Lionel  Rougé" /></meta><meta name="dc.Creator" content="Xianrui  Zhao" /></meta><meta name="dc.Creator" content="Ingrid  Wertz" /></meta><meta name="dc.Creator" content="Chudi  Ndubaku" /></meta><meta name="dc.Creator" content="Vickie  Tsui" /></meta><meta name="dc.Description" content="USP7 is a deubiquitinase implicated in destabilizing the tumor suppressor p53, and for this reason it has gained increasing attention as a potential oncology target for small molecule inhibitors. H..." /></meta><meta name="Description" content="USP7 is a deubiquitinase implicated in destabilizing the tumor suppressor p53, and for this reason it has gained increasing attention as a potential oncology target for small molecule inhibitors. H..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 8, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01293" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01293" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01293" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01293" /></link>
        
    
    

<title>Discovery of Small-Molecule Inhibitors of Ubiquitin Specific Protease 7 (USP7) Using Integrated NMR and in Silico Techniques | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01293" /></meta><meta property="og:title" content="Discovery of Small-Molecule Inhibitors of Ubiquitin Specific Protease 7 (USP7) Using Integrated NMR and in Silico Techniques" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0020.jpeg" /></meta><meta property="og:description" content="USP7 is a deubiquitinase implicated in destabilizing the tumor suppressor p53, and for this reason it has gained increasing attention as a potential oncology target for small molecule inhibitors. Herein we describe the biophysical, biochemical, and computational approaches that led to the identification of 4-(2-aminopyridin-3-yl)phenol compounds described by Kategaya  (Nature 2017, 550, 534–538) as specific inhibitors of USP7. Fragment based lead discovery (FBLD) by NMR combined with virtual screening and re-mining of biochemical high-throughput screening (HTS) hits led to the discovery of a series of ligands that bind in the “palm” region of the catalytic domain of USP7 and inhibit its catalytic activity. These ligands were then optimized by structure-based design to yield cell-active molecules with reasonable physical properties. This discovery process not only involved multiple techniques working in concert but also illustrated a unique way in which hits from orthogonal screening approaches complemented each other for lead identification." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01293"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01293">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01293&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01293&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01293&amp;href=/doi/10.1021/acs.jmedchem.7b01293" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 10056-10070</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01117" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01290" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Small-Molecule Inhibitors of Ubiquitin Specific Protease 7 (USP7) Using Integrated NMR and in Silico Techniques</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Di+Lello">Paola Di Lello</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol "><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-9683-926X" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Pastor">Richard Pastor</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol "><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeremy+M.++Murray">Jeremy M. Murray</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+A.++Blake">Robert A. Blake</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Frederick++Cohen">Frederick Cohen</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Terry+D.++Crawford">Terry D. Crawford</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Joy++Drobnick">Joy Drobnick</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jason++Drummond">Jason Drummond</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lorna++Kategaya">Lorna Kategaya</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tracy++Kleinheinz">Tracy Kleinheinz</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Till++Maurer">Till Maurer</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lionel++Roug%C3%A9">Lionel Rougé</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xianrui++Zhao">Xianrui Zhao</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ingrid++Wertz">Ingrid Wertz</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chudi++Ndubaku">Chudi Ndubaku</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vickie++Tsui">Vickie Tsui</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div><div class="corresp-info"><strong>*</strong>P.D.L.: phone, +1-650- 467-4914; e-mail, <a href="/cdn-cgi/l/email-protection#1b7f72777e7777746b5b7c7e757e35787476"><span class="__cf_email__" data-cfemail="99fdf0f5fcf5f5f6e9d9fefcf7fcb7faf6f4">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>R.P.: phone, +1-650-225-8114; e-mail, <a href="/cdn-cgi/l/email-protection#adddccded9c2df83dfc4cec5ccdfc9edcac8c3c883cec2c0"><span class="__cf_email__" data-cfemail="4a3a2b393e253864382329222b382e0a2d2f242f64292527">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>V.T.; phone, +1-650-534-8426; e-mail, <a href="/cdn-cgi/l/email-protection#483e212b23212d663c3b3d2108293129663129242d662d2c3d"><span class="__cf_email__" data-cfemail="74021d171f1d115a0007011d34150d155a0d1518115a111001">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01293&amp;href=/doi/10.1021%2Facs.jmedchem.7b01293" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 10056–10070</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 22, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 August 2017</li><li><span class="item_label"><b>Published</b> online</span>8 December 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 December 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01293" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01293</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10056%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPaola%2BDi%2BLello%252C%2BRichard%2BPastor%252C%2BJeremy%2BM.%2BMurray%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D24%26contentID%3Dacs.jmedchem.7b01293%26title%3DDiscovery%2Bof%2BSmall-Molecule%2BInhibitors%2Bof%2BUbiquitin%2BSpecific%2BProtease%2B7%2B%2528USP7%2529%2BUsing%2BIntegrated%2BNMR%2Band%2Bin%2BSilico%2BTechniques%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10070%26publicationDate%3DDecember%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01293"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3896</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">28</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01293" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Small-Molecule Inhibitors of Ubiquitin Specific Protease 7 (USP7) Using Integrated NMR and in Silico Techniques&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Di Lello&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Pastor&quot;},{&quot;first_name&quot;:&quot;Jeremy&quot;,&quot;last_name&quot;:&quot;M. Murray&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;A. Blake&quot;},{&quot;first_name&quot;:&quot;Frederick&quot;,&quot;last_name&quot;:&quot;Cohen&quot;},{&quot;first_name&quot;:&quot;Terry&quot;,&quot;last_name&quot;:&quot;D. Crawford&quot;},{&quot;first_name&quot;:&quot;Joy&quot;,&quot;last_name&quot;:&quot;Drobnick&quot;},{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;Drummond&quot;},{&quot;first_name&quot;:&quot;Lorna&quot;,&quot;last_name&quot;:&quot;Kategaya&quot;},{&quot;first_name&quot;:&quot;Tracy&quot;,&quot;last_name&quot;:&quot;Kleinheinz&quot;},{&quot;first_name&quot;:&quot;Till&quot;,&quot;last_name&quot;:&quot;Maurer&quot;},{&quot;first_name&quot;:&quot;Lionel&quot;,&quot;last_name&quot;:&quot;Rougé&quot;},{&quot;first_name&quot;:&quot;Xianrui&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Ingrid&quot;,&quot;last_name&quot;:&quot;Wertz&quot;},{&quot;first_name&quot;:&quot;Chudi&quot;,&quot;last_name&quot;:&quot;Ndubaku&quot;},{&quot;first_name&quot;:&quot;Vickie&quot;,&quot;last_name&quot;:&quot;Tsui&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;10056-10070&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01293&quot;},&quot;abstract&quot;:&quot;USP7 is a deubiquitinase implicated in destabilizing the tumor suppressor p53, and for this reason it has gained increasing attention as a potential oncology target for small molecule inhibitors. Herein we describe the biophysical, biochemical, and computational approaches that led to the identification of 4-(2-aminopyridin-3-yl)phenol compounds described by Kategaya ( Nature 2017, 550, 534–538 ) as specific inhibitors of USP7. Fragment based lead discovery (FBLD) by NMR combined with virtual screening and re-mining of biochemical high-throughput screening (HTS) hits led to the discovery of a series of ligands that bind in the “palm” region of the catalytic domain of USP7 and inhibit its catalytic activity. These ligands were then optimized by structure-based design to yield cell-active molecules with reasonable physical properties. This discovery process not only involved multiple techniques working in concert but also illustrated a unique way in which hits from orthogonal screening approaches complemented e&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01293&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01293" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01293&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01293" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01293&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01293" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01293&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01293&amp;href=/doi/10.1021/acs.jmedchem.7b01293" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01293" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01293" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01293%26sid%3Dliteratum%253Aachs%26pmid%3D29166018%26genre%3Darticle%26aulast%3DDi%2BLello%26date%3D2017%26atitle%3DDiscovery%2Bof%2BSmall-Molecule%2BInhibitors%2Bof%2BUbiquitin%2BSpecific%2BProtease%2B7%2B%2528USP7%2529%2BUsing%2BIntegrated%2BNMR%2Band%2Bin%2BSilico%2BTechniques%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D24%26spage%3D10056%26epage%3D10070%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/jmcmar.2017.60.issue-24/20171228/jmcmar.2017.60.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">USP7 is a deubiquitinase implicated in destabilizing the tumor suppressor p53, and for this reason it has gained increasing attention as a potential oncology target for small molecule inhibitors. Herein we describe the biophysical, biochemical, and computational approaches that led to the identification of 4-(2-aminopyridin-3-yl)phenol compounds described by <contrib-group>Kategaya</contrib-group> ( <cite>Nature</cite> <span class="NLM_year">2017</span>, <em>550</em>, 534–538) as specific inhibitors of USP7. Fragment based lead discovery (FBLD) by NMR combined with virtual screening and re-mining of biochemical high-throughput screening (HTS) hits led to the discovery of a series of ligands that bind in the “palm” region of the catalytic domain of USP7 and inhibit its catalytic activity. These ligands were then optimized by structure-based design to yield cell-active molecules with reasonable physical properties. This discovery process not only involved multiple techniques working in concert but also illustrated a unique way in which hits from orthogonal screening approaches complemented each other for lead identification.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02370" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02370" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Ubiquitination is a key form of post-translational modification. Ubiquitination largely targets protein substrates for degradation by the 26S proteasome in a tightly regulated manner.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> As such, ubiquitination involves substrate-specific ubiquitin ligases (E3) that catalyze the formation of an isopeptide bond between the carboxyl terminus of ubiquitin and the lysine side chain of the target protein.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> A ubiquitin monomer can be attached via this mechanism to a protein substrate, and multiple ubiquitins can be linked to each other resulting in polyubiquitination. Depending on the type of linkage between the ubiquitin molecules in these chains, polyubiquitination can serve as a signal for degradation of the protein substrate.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a></div><div class="NLM_p">Opposing the function of these E3 are deubiquitinating enzymes, or DUBs. As the name suggests, a DUB removes ubiquitin from a protein substrate, another ubiquitin within a polyubiquitin chain, or a ubiquitin precursor. There are around 100 known human DUBs, consisting of five major families: ubiquitin-specific proteases (USPs), ubiquitin carboxyl-terminal hydrolases (UCHs), ovarian tumor proteases (OTUs), Machado–Joseph disease protein domain proteases (MJDs), and JAMM/MPN domain-associated metallopeptidases (JAMMs).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In addition, two families of DUBs have been recently discovered, the monocyte chemotactic protein-induced proteins (MCPIPs)<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> and a new class of cysteine proteases belonging to the MINDY family (MIU-containing novel DUB family).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> USPs are the largest family of DUBs,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and members of this family are cysteine proteases that rely on a “catalytic triad” of cysteine, aspartic/glutamic acid, and histidine to hydrolyze the isopeptide bond.</div><div class="NLM_p">Several USPs have been reported to play important biological roles that make them attractive as therapeutic targets.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> An example is USP7 or herpes virus associated USP (HAUSP), which is considered a potential oncology target due to its role in the destabilization of the tumor suppressor p53.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> As a transcription activator, p53 modulates the expression of genes like p21, Bax, or Puma that control cell cycle arrest or apoptosis in response to various cellular stresses.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> However, in the majority of human cancers p53 is inactivated by either mutations in p53 itself or alterations in the pathways that regulate its levels.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> Deubiquitination of MDM2, a substrate of USP7 and itself a ligase for p53, is one of the mechanisms by which wild-type p53 is inactivated. Therefore, inhibition of USP7 can tip the balance toward ubiquitinated MDM2, thus triggering its degradation and reactivating the p53 pathway in cancer cells that retain wild-type p53.</div><div class="NLM_p last">We recently reported a new class of highly selective USP7 inhibitors, <b>28</b> (GNE-6640) and <b>27</b> (GNE-6776), which modulated the enzymatic activity of USP7 by binding to a novel functional site within the “palm” region of the enzyme and interfering with substrate binding.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In this manuscript we describe the structure-based drug discovery strategies that led to the identification of the aforementioned new functional site on USP7 and to the discovery of the “palm” compounds. The discovery process that led to this new class of USP7 inhibitors involved the interplay between biochemical, in silico, and biophysical techniques, including nuclear magnetic resonance (NMR) and crystallography. In addition, hits from multiple small molecule libraries (fragment, HTS, and sample management libraries) were combined to generate this novel series. The detailed account of how the techniques and libraries complemented each other, as well as the structure-based optimization leading to molecules with improved biochemical potency and observable cellular activity, will be presented.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33217" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33217" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Discovery, Biophysical Characterization, and Structure-Based Optimization of the Oxadiazole Series</h3><div class="NLM_p">We have applied different screening approaches (a fragment screen by NMR and a full HTS campaign) to search for small molecule ligands that would specifically bind and inhibit the catalytic domain of USP7.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In particular, a fragment screen by NMR led to the discovery of a series of small molecules that bind in the active site near the catalytic cysteine (Cys223) and were further developed into moderately active inhibitors of the enzymatic activity (unpublished results). In a parallel approach we also used computational methods to look for additional scaffolds that could bind in the enzyme active site and potentially block its activity. Compound <b>1</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), a fragment-screen hit that binds in the enzyme active site, was used as the query molecule to carry out ligand-based virtual screening against compounds in the Genentech internal sample management library. The 500 top-scoring hits from the virtual screen were then analyzed by visual inspection, leading to the selection of 21 compounds. To verify that these 21 compounds bound to the catalytic domain of USP7 (USP7-CD), we tested them by NMR using saturation transfer difference (STD) experiments. Although many of these molecules did not confirm as ligands, a smaller subset of 12 compounds showed binding to USP7-CD. These 12 compounds were then evaluated for biochemical activity, and to further characterize their binding site, they were also tested in NMR titration experiments with labeled USP7-CD. Addition of each of these hits to labeled USP7-CD induced chemical shifts perturbations and/or line broadening of signals in the two-dimensional [<sup>1</sup>H–<sup>15</sup>N] TROSY spectrum of labeled USP7-CD, confirming an interaction with the protein. However, of the 12 compounds tested with labeled USP7-CD, three induced unspecific signals line broadening suggesting possible protein aggregation, whereas five appeared to bind in more than one site; in fact they caused changes in the resonances of residues located in the active site as well as in other regions of the protein. The remaining four compounds (<b>2</b>–<b>5</b>) induced chemical shifts perturbations only in a select subset of NMR signals, suggesting they interacted with USP7-CD at a specific site (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B). These four hits, all of which contained an oxadiazole core with a 4-phenol on one side and piperidine or pyrrolidine on the other side, were the only ones of the 12 tested that had detectable biochemical activities (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0001.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Compound <b>1</b> and compound <b>2</b> bind two distinct sites on USP7-CD. (A) Selected region of the overlay between the 2D <sup>1</sup>H–<sup>15</sup>N TROSY spectra of labeled USP7-CD apo (in black) and in the presence of 8 equiv of compound <b>1</b> (in red). (B) Selected region of the overlay between the 2D <sup>1</sup>H–<sup>15</sup>N TROSY spectra of labeled USP7-CD apo (in black) and after addition of 8 equiv of compound <b>2</b> (in red). Arrows and ovals indicate signals undergoing large chemical shifts changes. (C, D) Ribbon model of the apo USP7-CD crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M5W">4M5W</a>). Amino acids experiencing large chemical shifts perturbations upon formation of the USP7-CD/<b>1</b> complex are highlighted in orange, whereas the amino acids displaying large chemical shift changes upon formation of the USP7-CD/<b>2</b> complex are colored in yellow. The catalytic triad residues, Cys223, His464, and Asp481, are shown in sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Active-Site Binder and Biochemically Active Hits from Virtual Screening</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0014.gif" alt="" id="fx1" /></img><div></div></div><div class="NLM_p">Surprisingly, the chemical shifts perturbations that <b>2</b> and its three analogs (<b>3</b>–<b>5</b>) induced on labeled USP7-CD (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B) involved a different subset of peaks from that observed for the active site ligand <b>1</b> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A), thus pointing to a different binding site on the protein surface. Nevertheless, the <i>K</i><sub>D</sub> values of <b>2</b> and <b>3</b>, estimated by NMR to be in the range 12–27 μM for USP7-CD, were within 5-fold of the biochemical IC<sub>50</sub> values determined for full length USP7 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><div class="NLM_p">In the case of the active site ligand <b>1</b> the amino acids showing the largest chemical shifts perturbations upon addition of the ligand to labeled USP7-CD were Met292, Gln293, His294, Asp295 (located in the switching loop), Met410, Phe411, Asp412, Asn418, Ile419, Lys420, Ile421 (located in the β0−β0′ sheet) and Gly458, Asp459, Asn460, Gly462, Gly463, His464 (located in the loop connecting strands β10 and β11). When mapped onto the structure of apo USP7-CD, these amino acids clearly defined a ligand binding site (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C) that overlapped with the catalytic site. On the contrary in the case of <b>2</b> the largest chemical shifts perturbations and/or signals line-broadening were reserved for a different set of amino acids, namely, Ile320, Lys322 (helix α6), Gly326, Met328 (strand β1), Ile350, Gln351, Leu352, Ser353 (strand β3), Tyr347, Tyr348, Asp349 (located in the loop connecting strands β2 and β3), His403, Met407 (strand β8), Asp295, Gln297, Glu298, Cys300, Arg301, Leu304, Asp305, Val307, Glu308 (helix α5), and Lys281, Gly284, Trp285. When mapped onto the structure of apo USP7-CD, these amino acids clustered within the “palm site” of USP7-CD, in a region adjacent to the catalytic site (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>D), thus revealing a novel functional site.</div><div class="NLM_p">The location of the binding site identified by the NMR chemical shift perturbation studies was consistent with the crystal structure of the USP7-CD/<b>2</b> complex that was subsequently determined. The structure also revealed several critical interactions between ligand and protein. The phenol ring was buried in a hydrophobic pocket with an edge-to-face interaction with Phe324 and the phenol hydroxyl engaged in a hydrogen bond interaction with His403 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). While the oxadiazole moiety was partially solvent exposed, it was involved in a face-to-face π-stacking interaction with Tyr348. The piperidine was for the most part solvent exposed and, based on the electron density, was not involved in any interactions with the protein. SAR on the piperidine (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) indicated that it made minimal contribution to activity.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0002.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of <b>2</b> in complex with USP7-CD (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHC">5WHC</a>). Blue mesh represents electron density contoured at 0.7σ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To follow up on <b>3</b>, analogues were designed and prioritized based on the crystal structure (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Replacement of the oxadiazole core with a 2,4 substituted oxazole (<b>6</b>) resulted in no change in activity. However, employing a 2,5-substituted oxazole (<b>7</b>) led to a 3-fold potency increase. For <b>7</b> the oxygen (rather than the nitrogen in <b>6</b>) in the oxazole is closer in space to the methylene part of the Arg301 side chain, and this potentially leads to a better van der Waals interaction because of the smaller negative dipole of the oxygen relative to the nitrogen. An additional 2-fold increase in activity was observed when phenol was replaced by 3-fluorophenol, likely due to increased phenol acidity or slight changes in conformational preference (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_001.pdf" class="ext-link">Figure S1 in Supporting Information</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure-Based Optimization of the Oxadiazole Series</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0015.gif" alt="" id="fx2" /></img><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Discovery and Characterization of the Aminopyridine Series</h3><div class="NLM_p">Having discovered that the “palm” region of USP7-CD contains an additional functional site that could be exploited to modulate the enzymatic activity through small molecules binding, we applied an NMR driven scaffold-hopping strategy to search for structurally novel compounds that could bind to this site and potentially be developed into selective USP7 inhibitors.</div><div class="NLM_p">Instead of generating structurally novel compounds through a computational approach to scaffold hopping, we re-mined the hits from our original NMR fragment screen, searching for ligands that had a binding fingerprint on labeled-USP7 similar to that observed for <b>2</b> and its analogs. Using this approach, we were able to identify several heterocyclic substituted phenols (i.e., aminopyridinephenols and aminopyrazinephenols, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A) that showed perturbation maps remarkably similar to those observed for <b>2</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B,C), suggesting that they bound USP7-CD in the same site and possibly with the same binding pose as compounds of the oxadiazole series.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0003.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>2</b> and compound <b>10</b> bind to the same site on USP7-CD. (A) Re-mined fragment hits from the NMR screen. (B) 2D <sup>1</sup>H–<sup>15</sup>N TROSY spectra overlay for labeled USP7-CD in the apo form (in black) and in the presence of compound <b>10</b> (in red). Protein:ligand ratio is 1:8. (C) Overlay between the 2D <sup>1</sup>H–<sup>15</sup>N TROSY spectra of labeled USP7-CD apo (in black) and after addition of compound <b>2</b> (in red). Protein:ligand ratio is 1:8. Arrows and ovals highlight some of the signals undergoing large chemical shifts changes or line broadening upon complex formation. The chemical shifts changes induced by the two distinct series are virtually identical.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Interestingly, the aminopyridinephenol motif found in some of the hits from the fragment screen was also present in <b>14</b> and <b>15</b>, two of the 76 hits from the high-throughput screening (HTS) (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) of the Genentech proprietary library of ∼2 million compounds. However, these two HTS hits had been previously deprioritized for reconfirmation in favor of hits that were commercially available and significantly more potent.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Phenol Substituted Aminopyridines from HTS Hits<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0016.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">The asterisk (∗) indicates not screened in HTS.</p></div></div><div></div></div><div class="NLM_p">Efforts that led to the resynthesis of compounds <b>14</b> and <b>15</b> also produced compound <b>16</b> as a synthetic byproduct, and all three compounds demonstrated micromolar biochemical activities (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). A cocrystal structure of <b>16</b> was obtained, and it showed that the compound bound in a similar mode to that of <b>3</b>, as both occupied the same “palm” binding site, where significant affinity was gained by engaging His403 and Phe324 with a phenol moiety (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). However, electron density for the other components of the ligand was not well-defined, leaving its exact binding mode undetermined. Instead, four binding modes were possible based on the observed electron density as shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, with each binding mode having a distinct orientation of the solvent-exposed phenol and the amino moiety. Modeling and occupancy refinement of multiple possible binding orientations also revealed an unoccupied hydrophobic pocket, proximal to the ligand, which was targeted for affinity optimization (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). Initial medicinal chemistry efforts focused on both affinity optimization and elucidation of the inhibitory binding mode.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0004.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structure of <b>16</b> with USP7-CD (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WH7">5WH7</a>). (A) 2.8 Å electron density (blue mesh) for <b>16</b> was ambiguous and suggested that it was possible to bind in different alternative conformations. (B) Surface representation of the binding pocket highlighting the hydrophobic pocket (magenta).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0005.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. 2D representation of possible ligand binding modes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Attempts to optimize <b>16</b> began by substituting at the 2, 4, and 6 positions of the central pyridine ring (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Deletion of the 2-amino moiety (<b>17</b>) led to a complete loss of activity, whereas addition of a 6-amino group (<b>18</b>) showed comparable activity. Appending an ethyl at C4 (<b>20</b>) led to a 5-fold activity increase, whereas propyl was too long for the small pocket (<b>21</b>). C4-isopropyl was 5-fold less active than C4-ethyl (<b>22</b> vs <b>20</b>), and C4-cyano (<b>23</b>) led to complete loss of activity. This preference for small, unbranched hydrophobic groups in the C4 position was consistent with two of the possible ligand binding modes (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C and <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. C2, C4, and C6 Substituent SAR</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0017.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">R<sub>3</sub></th><th class="colsep0 rowsep0" align="center">USP7 IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">8.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">C<sub>3</sub>H<sub>7</sub></td><td class="colsep0 rowsep0" align="left">13.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left"><i>i</i>-Pr</td><td class="colsep0 rowsep0" align="left">21.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="left">>200</td></tr></tbody></table></div></div><div class="NLM_p">C3 and C5 analogue synthesis was undertaken, where one of the goals was to identify which phenol moiety made the critical His403 interaction (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). The resulting SAR is shown in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>, and unexpectedly, the SAR at both positions was very similar. At both positions, phenyl substitution led to a moderate reduction in activity, cyanophenyl substitution was well tolerated, and indazole substitution improved activity by several-fold, resulting in the most active compounds from this series. In addition, the observation that compounds without a phenol in C3 or C5 (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_001.pdf" class="ext-link">Table S1</a>) were significantly less potent demonstrated that a phenol was required for detectable activity. Taken together, the results suggested the latter two conformations in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C or <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D could be adopted to place phenol in the His403 pocket.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR at the C3 and C5 Positions</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0018.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0019.gif" alt="" id="fx6" /></img><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Cell-Based Assay Results</h3><div class="NLM_p">A representative set of our most biochemically potent and chemically diverse compounds was tested in a MDM2 MSD cell-based assay with SJSA cells, which are osteosarcoma cells with high level of MDM2 expression,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> to study their effects on levels of ubiquitinated MDM2 vs the total amount of MDM2 in a dose–response fashion (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). The rank order of activities corresponded well with biochemical potency. Compound <b>28</b> demonstrated submicromolar activity and was the most potent analog of this group, consistent with its relative biochemical activity. Compound <b>27</b> demonstrated single digit micromolar activity, while <b>18</b> and <b>31</b> both were an order of magnitude less active. The negative control <b>29</b> had EC<sub>50</sub> > 50 μM.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. MDM2 MSD Cell-Based Assay Data for Selected Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">36.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">2.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">37.5</td></tr></tbody></table></div></div><div class="NLM_p">Compounds <b>18</b>, <b>27</b>, and <b>28</b> were further tested in EOL-1 cells alongside <b>29</b> as a negative control (NC). These compounds were selected based on their EC<sub>50</sub> values in the MDM2 MSD assay. The effects of these compounds on the levels of proteins downstream of the USP7/MDM2 pathway (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A) were examined at 1.3 μM and 5 μM by Western blots (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B). By use of tubulin as a loading control, the normalized levels of p53 and p21 were quantitatively compared to DMSO and plotted in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C and <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D, respectively. As expected, the negative control had minimal effect on the levels of p53 and p21, whereas compounds <b>27</b> and <b>28</b> showed a significant increase in p53 levels. Compounds <b>27</b> and <b>28</b>, and to a lesser extent <b>18</b>, also up-regulated p21.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0006.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibiting USP7 activity increases p53 and p21 levels of EOL-1 cells. (A) Illustration of USP7 signaling through the MDM2-p53-p21 axis in the absence (left panel) and in the presence (right panel) of USP7 inhibitors. Left panel: USP7 stabilizes MDM2 by rescuing it from proteosomal degradation. MDM2 down-regulates the p53 pathway, thus leading to a reduction in p53-depended apoptosis, a reduction in p21-mediated cell cycle arrest, and an increase in cell viability. Right panel: USP7 inhibition is expected to promote MDM2 proteosomal degradation. This causes a reduction in the MDM2 levels and the consequent increase in p53 protein levels, ultimately resulting in the activation of caspases and modulation of p53-target genes, e.g., p21. An increase in caspase activity triggers apoptosis, while an increase in p21 triggers cell cycle arrest. Both result in a decrease in cell viability and cell number. (B) Western blots using antibodies against USP7, MDM2, tubulin, p53 and p21. (C, D) Graphs showing quantification of p53 and p21 Western blots normalized to tubulin. Under these conditions, compounds <b>27</b> and <b>28</b> increase relative p53 levels, whereas <b>18</b>, <b>27</b> and <b>28</b> increase relative p21 levels. Data are representative of two experiments. NC: negative control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Furthermore, the effects of USP7 inhibitors on the viability and caspase activity of EOL-1 cells were examined (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A,B) using imaging techniques. The percent confluence measured the area of each well occupied by cells, and dose-dependent decrease in cell viability was observed over 48 h when treated with each of the active inhibitors while treatment with the negative control did not have significant effects. Caspase activity was increased and the effects were dose-dependent following treatment with <b>18</b> and <b>28</b> over 48 h, whereas all doses resulted in minimal effects on caspase activation for the negative control arm. This dose-dependent effect on caspase activity was lower for <b>27</b>, leading to the hypothesis that <b>28</b> decreases cell viability via both apoptosis and cell cycle arrest (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A), whereas <b>27</b> mainly causes cell cycle arrest. Cell cycle arrest is generally triggered first in response to an insult, followed by induction of apoptosis in case the cell damage cannot be repaired. Therefore, treatment with a more potent compound like <b>28</b> can lead to apoptosis in addition to cell cycle arrest, whereas treatment with a less potent compound like <b>27</b> would mainly trigger cell cycle arrest. For compound <b>18</b>, which is the least potent in inhibiting USP7 (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> and <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>) and in modulating the levels of p53 and p21 (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C,D), the observed increase in caspase activity (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B) and the resulting decrease in cell viability (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A) could be due to off-target effects.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0007.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Inhibiting USP7 activity increases caspase activity and decreases viability of EOL-1 cells. (A) IncuCyte data showing a dose-dependent decrease in cell confluence with inhibitors relative to control. (B) IncuCyte data showing a dose-dependent increase in caspase activity with inhibitors relative to control. (C) CellTiter-Glo viability data validating IncuCyte cell confluence results. Data are representative of three experiments. NC: negative control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This hypothesis can also explain the CellTiter-Glo results (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>C) measuring ATP levels at the 48 h end point. Unlike the negative control, all three inhibitors led to decreased ATP levels as their concentrations were increased. Compound <b>28</b> was more potent, while the effects of <b>27</b> were seen at higher inhibitor concentrations. As expected, the negative control had minimal effects.</div><div class="NLM_p last">There are several reports of closely related analogs being kinase inhibitors,<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17-19)</a> and to assess potential contributions of off-target kinase activity to the cellular phenotypes that were observed, <b>27</b> and <b>28</b> were profiled against a kinase panel consisting of 219 kinases. We observed only modest inhibition, with IC<sub>50</sub> values in the low micromolar to submicromolar range, against six kinases (data not shown). Most importantly, <b>27</b> and <b>28</b>, our most potent compounds, as well as the negative control <b>29</b> were equipotent against all of these kinases, suggesting that kinase-inhibitory effects did not contribute to effects on cellular viability. This conclusion is further supported by published studies<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> demonstrating that <b>28</b> and <b>27</b> are on-pathway, USP7-selective inhibitors. Both compounds reduce cell viability in wild type HCT-116 cells but not in USP7-null HCT-116 cells, indicating that USP7 expression is required for cellular activity. Furthermore, <b>27</b>, <b>28</b>, and <b>29</b> were all profiled in 3- and 5-day viability assays in large panels of cell lines. While <b>27</b> and <b>28</b> significantly reduced viability of a number of cell lines, <b>29</b> was inactive in all cell lines evaluated. Collectively, these data indicate that the modest in vitro kinase inhibition observed for our USP7 inhibitors does not translate to cellular activity.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemistry</h3><div class="NLM_p">The oxadiazoles and oxazoles shown in <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> were prepared using a suite of heterocycle-forming reactions in both a convergent and linear manner (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch3" aria-label="3">3</a>). Condensations between acid and hydroxyamidine components promoted by carbonyldiimidazole (CDI) led to the described oxadiazoles. The acid components were commercially sourced, whereas the hydroxyamidine coupling partners were formed from nitrile precursors.</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>2</b> and <b>3</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NH<sub>2</sub>OH, CDI, 18% yield; (b) HCl, EtOAc, 99% yield; (c) HCHO, NaCNBH<sub>3</sub>, AcOH, 38% yield.</p></p></figure><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 4-OMe-benzamide, 8% yield; (b) HBr, 80% yield.</p></p></figure><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) nitromethane, KO<sup><i>t</i></sup>Bu, 62% yield; (b) 10% Pd/C, H<sub>2</sub>, quant; (c) 4-OMe-benzoic acid, HATU, TEA, 51% yield; (d) Dess–Martin reagent, 27% yield; (e) Burgess reagent, 84% yield; (f) HBr, 24% yield.</p></p></figure><div class="NLM_p">The 2,4 substituted oxazoles were generated from convergent couplings between corresponding commercially available α-bromo ketones and amide monomers. On the other hand, the 2,5 substituted oxazoles resulted from a linear sequence that began with addition of nitromethane to <i>N</i>-Boc-piperidine aldehyde to afford the 2-nitro alcohol. Nitro reduction, amine acylation, and alcohol oxidation afford the β-acylamino ketone which was then cyclized under Burgess reagent promoted conditions to form the oxazole ring.</div><div class="NLM_p">Preparation of the bis-phenol substituted aminopyridines shown in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> utilized a bis-bromination with NBS followed by a bis-Suzuki coupling with 4-anisole boronic acid on the corresponding pyridine precursors (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Methoxy demethylation with BBr<sub>3</sub> then affords <b>17</b>–<b>23</b>.</div><figure id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0011.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>17</b>–<b>23</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NBS, DCM; (b) anisole-4-boronic acid, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C, quant; (c) BBr<sub>3</sub>, DCM.</p></p></figure><div class="NLM_p">Synthesis of both the C3 and C5 phenol replacements in the aminopyridine core both involved conducting Suzuki couplings on the corresponding bromoaminopyridine late-stage intermediates <b>50</b> and <b>55</b> (<a class="ref internalNav" href="#sch5" aria-label="Schemes 5">Schemes 5</a> and <a class="ref internalNav" href="#sch6" aria-label="6">6</a>).</div><figure id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0012.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>24</b>–<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NBS, THF, 79% yield; (b) <i>n</i>-BuLi, THF, −78 °C, 93% yield; (c) anisole-4-boronic acid, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C, 73% yield; (d) NBS, DCM, 59% yield; (e) BBr<sub>3</sub>, DCM, 83% yield; (f) RB(OH)<sub>2</sub>, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C.</p></p></figure><figure id="sch6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0013.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>30</b>–<b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NBS, THF, 72% yield; (b) anisole-4-boronic acid, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C, 66% yield; (c) NBS, TFH, 67% yield; (d) BBr<sub>3</sub>, DCM, 90% yield; (e) RB(OH)<sub>2</sub>, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C.</p></p></figure><div class="NLM_p">Synthesis of <b>50</b> began with bis-bromination of commercially available 4-ethylaminopyridine followed by selective debromination at the more labile C5 position to afford <b>47</b>. Suzuki coupling to install a 4-methoxyphenyl group at C3 followed by C5 bromination and methoxy demthylation with HBr affords <b>50</b>.</div><div class="NLM_p last">Synthesis of <b>55</b> began with selective C5 bromination of 4-ethylaminopyridine with NBS. Installation of the 4-MeO-phenyl group by Suzuki coupling followed by C3 bromination and methoxy demethylation then affords <b>55</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05445" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05445" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Herein, we describe the details of how the USP7 tool compounds <b>27</b> and <b>28</b> reported by Kategaya et al.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> were discovered. These are the first small molecule allosteric inhibitors of any USP that have been fully characterized using biophysical and biochemical methods as well as cell-based assays. The lack of comprehensive data in past publications represents both the novelty and challenge of this target class,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and we could show that by combining experimental and computational methods with a variety of chemical libraries, we were able to discover these “palm” site binding series.</div><div class="NLM_p">NMR played a significant role in the discovery of lead matter, and it is no surprise that our lead matter came from a biophysical screen, since a cysteine protease like USP7 could turn up many reactive false positives in a biochemical HTS campaign. In fact, the HTS hits that were followed up by re-mining efforts described in this publication progressed farther than other HTS series. Most of the biochemical HTS hits failed to confirm upon resynthesis, did not show specific activity to the catalytic domain, or were shown to be potential aggregators. Furthermore, none of the hits from the biochemical HTS were successful in cocrystallization studies, hindering rational structure-based design.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">Our discovery process started with the application of a shape-based virtual screen using the active-site binder as a query molecule against compounds in the Genentech internal sample management library. Combining this virtual screen with a comprehensive biophysical and biochemical characterization of the virtual hits was a critical step in the discovery process. It allowed us to identify a structurally unique chemical series, the oxadiazole series, and most importantly it unveiled the presence of a second functional site, the “palm” site, besides the catalytic site. The subsequent approach of re-mining the NMR fragment hit set for additional scaffolds with similar binding fingerprints to the oxadiazole compounds but with different chemical structures was key in turning out several hits containing the aminopyridine and aminopyrazine substructures. Even though these fragment hits were not potent enough to observe biochemical activity, taking the chemical moiety back to re-examine the deprioritized HTS hits led to the successful discovery of our most potent series, the aminopyridine series.</div><div class="NLM_p">Not only were crystal structures important in the lead discovery stage, they were also essential for optimization of the aminopyridine series. Even though the electron densities provided multiple possible binding modes, medicinal chemistry optimization helped to narrow down the binding modes to two possibilities. During this process, the ethyl group was installed in a small hydrophobic pocket in the 4-position of the aminopyridine to boost potency, and unsubstituted indazoles at either the C3 or C5 position of the aminopyridine were found to be most active when coupled with a phenol on the other side. Compounds with measurable cellular EC<sub>50</sub> were obtained, and they had excellent DUB selectivity as described by Kategaya et al.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Furthermore, compounds were shown to up-regulate p21 and p53 in EOL-1 cells in a dose-dependent manner, in agreement with the pathway biology of inhibiting USP7. Increase in caspase activity and decreased cell viability were also observed, consistent with the expected outcome of USP7 inhibition.</div><div class="NLM_p">In order to progress this series as a clinical candidate, the key challenges will be to improve cellular potency and plasma protein binding properties while maintaining good pharmacokinetics. <b>27</b> demonstrated good oral bioavailability and half-life in mice but only transiently achieved target plasma exposure, due to its high plasma protein binding and lack of submicromolar cellular EC<sub>50</sub>.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Therefore, improving both these parameters will enhance the ability of this series to achieve efficacious concentrations in vivo. In addition, profiling of this series in both in vitro and in vivo toxicity assays would be required to identify potential negative side effects.</div><div class="NLM_p last">Our experience here provides a lesson for identifying lead matter for challenging targets such as DUBs, in that an integrated approach is crucial and involves multiple biophysical, biochemical, and in silico techniques, as well as a variety of compound libraries, which normally would be studied separately. The process we described to re-mine hits with various techniques and from different hit sets, as opposed to more traditional drug discovery efforts, can improve the chance of identifying potent inhibitors of new classes of enzymes and proteins otherwise considered undruggable.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35842" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35842" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Ligand-Based Virtual Screening</h3><div class="NLM_p last">Ligand-based virtual screening was carried out using fastROCS (OpenEye, Inc. <a href="http://www.eyesopen.com" class="extLink">www.eyesopen.com</a>) as implemented at Genentech. The specifics of the integration of fastROCS with a variety of databases have been described.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Compound <b>1</b> was used as the query molecule against a proprietary library of ∼100 000 compounds in Genentech’s sample management, and a maximum of 500 top-scoring compounds using default parameters were returned without postclustering. Visual inspection was carried out with consideration of leadlikeness as well as dry powder availability.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Protein Expression and Purification for NMR</h3><div class="NLM_p last">The catalytic domain of USP7 (USP7-CD), encompassing residues 208–554, was expressed as a His-Tag fusion protein in <i>E. coli</i> Rosetta 2 (DE3) cells. Unlabeled USP7-CD was produced according to the protocol reported elsewhere.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Uniformly ([<sup>15</sup>N–<sup>13</sup>C–<sup>2</sup>H], δ1[<sup>13</sup>CH<sub>3</sub>]-Ile, [<sup>13</sup>CH<sub>3</sub>]-Leu/Val, and [<sup>13</sup>CH<sub>3</sub>]-Met)-labeled USP7-CD was expressed and purified as previously described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Samples for NMR</h3><div class="NLM_p">The Genentech fragment library was screened as reported elsewhere.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">Samples for binding validation by saturation transfer difference (STD) experiments contained 250 μM ligand and 5 μM unlabeled USP7-CD in NMR buffer [PBS (pH 7.4) in 100% D<sub>2</sub>O, 50 μM 4,4-dimethyl-4-silapentane-1-sulfonic acid (DSS)].</div><div class="NLM_p">Samples for binding site mapping consisted of 250–340 μM labeled USP7-CD in PBS (pH 7.4), 300 μM to 2.5 mM ligand, 1 mM TCEP-<i>d</i><sub>16</sub>, 10% D<sub>2</sub>O, and up to 2.5% DMSO-<i>d</i><sub>6</sub>.</div><div class="NLM_p last">For the determination of the binding constants, each sample consisted of 270 μM isotopically labeled USP7-CD in PBS (pH 7.4), 1 mM TCEP-<i>d</i><sub>16</sub>, 10% D<sub>2</sub>O, and up to 2.5% DMSO-<i>d</i><sub>6</sub>. Individual ligands were added to the protein sample stepwise over a range of increasing concentrations [250 μM, 500 μM, 1000 μM, 2000 μM, and 4000 μM].</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> NMR Experiments and Data Analysis</h3><div class="NLM_p">NMR experiments were performed on 500 and 600 MHz spectrometers equipped with 1.7 mm and 5 mm cryoprobes, respectively.</div><div class="NLM_p">The backbone resonances assignment for labeled USP7-CD was obtained as previously reported.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Chemical shifts of the proton, carbon, and nitrogen nuclei were referenced externally to that of DSS at 0 ppm.</div><div class="NLM_p">The fragment library was screened using saturation transfer difference (STD) experiments<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> run at 280 K. Hits from the primary screen were further characterized by analyzing the ligand-induced chemical shifts changes of labeled USP7-CD in [<sup>1</sup>H–<sup>15</sup>N] TROSY experiments<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p">STD experiments were processed and analyzed using TOPSPIN 3.0. Data collected for binding site mapping were processed with NMRPipe/NMRDraw<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and analyzed with NMRView.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p last">The dissociation constants (<i>K</i><sub>D</sub>) were derived from the <sup>1</sup>H and <sup>15</sup>N chemical shifts changes observed in the [<sup>1</sup>H–<sup>15</sup>N] TROSY spectra of isotopically labeled USP7-CD upon addition of increasing amounts of ligand. Chemical shift changes, measured at various protein/ligand ratios, were plotted as a function of the ligand concentration and then fitted according to the equation<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/jm-2017-012934_m001.gif" alt="" /></img></span>where Δδ is the chemical shift change at various protein/ligand ratios, Δδ<sub>max</sub> is the chemical shift change at saturation, <i>K</i><sub>D</sub> is the dissociation constant, and [L] and [P] are the ligand and protein concentrations, respectively.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Biochemical Assays</h3><div class="NLM_p">Biochemical USP7 assays used ubiquitin-Rho110 as a substrate. The reaction buffer consisted of 50 mM Tris (pH 7.5), 0.01% (v/v) Triton X-100, 2.5 mM dithiothreitol, 0.1% (w/v) bovine γ globulin (Sigma catalog no. G5009-25G); USP7, full-length, native C-terminus, 0.2 nM; the substrate, ubiquitin-Rho110 (Boston Biochem catalog no. U-555), 1 μM. Reactions were carried out for 1 h at room temperature in black 20 μL volume polystyrene ProxiPlate 384 F Plus (PerkinElmer catalog no. 6008260).</div><div class="NLM_p last">Test compounds, including a control USP7 inhibitor (Ub-aldehyde, Boston Biochem catalog no. U-201) were serially diluted in DMSO, in 384-well clear V-bottom polypropylene plates (Greiner catalog no. 781280). Compounds in DMSO were diluted 10-fold into reaction buffer to achieve 3-fold the final desired concentration. The substrate, ubiquitin-Rho110 (Boston Biochem catalog no. U-555), was prepared at 3 μM (3-fold the final concentration), and 5 μL was dispensed into the reaction plate. An amount of 5 μL of the compounds (diluted in reaction buffer at 3-fold the final concentration) was transferred to the reaction plate. An amount of 5 μL of 0.6 nM USP7 (diluted in reaction buffer at 3-fold the final concentration) was transferred to the reaction plate to initiate the reaction. After 1 h incubation at room temperature the reaction was quenched by the addition of 5 μL of 400 mM acetic acid. The enzymatic product was measured by quantifying the fluorescence signal of cleaved rhodamine-110 using excitation at 485 nm and emission at 535 nm. When preincubation of USP7 with compounds was required, the order of addition of reagents was modified to premix the compounds with USP7 (with a 1 h incubation period) prior to the addition of the substrate and the initiation of the reaction period. Percentage inhibition values were calculated relative to a no enzyme control and an uninhibited enzyme control. Curve fitting and IC<sub>50</sub> calculations were carried out using Genedata Screener software.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Crystallization and Data Collection</h3><div class="NLM_p last">Crystals were grown by the hanging-drop method by mixing the USP7 catalytic domain (residues 208–554) at 15 mg/mL with an equal volume of reservoir solution containing 100 mM Tris-HCl, pH 7.0, and 20% PEG1000 (v/v). Co-structures with compounds were obtained by soaking crystals with 1 mM compound overnight. Crystals were cryoprotected with reservoir solution supplemented with 20% glycerol (v/v) and flash frozen in liquid nitrogen. Data collection and refinement statistics are detailed in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_001.pdf" class="ext-link">Table S2</a>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> MDM2 MSD Cell-Based Assays</h3><div class="NLM_p">SJSA cells were maintained in RPMI media containing 10% FBS (Sigma-Aldrich, catalog no. F6765) and 1% GlutaMAX (ThermoFisher, catalog no. 35050061). SJSA cells were seeded at 120 000 cells per well in 90 μL of low serum RPMI supplemented with 0.5% FBS and 1% GlutaMAX into a 96 well TC-treated plate (Greiner catalog no. 655090). Cells were allowed to attach to the plate for 2 h at 37 °C.</div><div class="NLM_p">Test compounds were serially diluted in DMSO in a 96-well clear V-bottom polypropylene plate (Greiner, catalog no. 651261). An amount of 5 μL of compound diluted in DMSO was added to an intermediate plate containing 95 μL/well of low serum media per well of a 96-well clear V-bottom polypropylene plate. An amount of 10 μL of intermediate dilution of compound in media was added to cell plate yielding a 1:200 final dilution of test compound. Cell plates were incubated with compound at 37 °C with 5% CO<sub>2</sub> overnight for 16 h.</div><div class="NLM_p">After compound incubation 20 μM final concentration of MG132 (Cayman Chemical, catalog no. 10012628) was added to cell plate. Plates were incubated at 37 °C for 1 h.</div><div class="NLM_p">Lysis buffer was prepared per vendor instructions. 15 μL per well 5× cold lysis buffer was added to cell plates and incubated on a shaker at 4 °C for 30 min.</div><div class="NLM_p">The 96-well MSD plates provided in ubiquitinated, total MDM2 MULTI-SPOT 96-well four spot plate kit (catalog no. N45168B-1; Meso Scale Discovery, Gaithersburg, MD) were prepared by incubation with 150 μL/well of 3% blocker buffer A (Meso Scale Discovery catalog no. R93BA-4) in 1× MSD Tris wash buffer (Meso Scale Discovery catalog no. R61TX-1) for 1 h. The plate was washed 3 times with Tris wash buffer.</div><div class="NLM_p">An amount of 100 μL of cellular lysate was transferred to each of the MSD assay plate.</div><div class="NLM_p">Cell lysates were incubated in plate for 1 h at room temperature while shaking (650 rpm) in the dark.</div><div class="NLM_p">The MSD plates were washed 3 times with 1× Tris wash buffer (Meso Scale Discovery) and then incubated with 25 μL of 1×X SULFO-TAG anti-total MDM2 antibody in antibody buffer containing blockers specified in manufacturer protocol. Antibody was incubated in plates for 1 h at room temperature while shaking (650 rpm) in the dark.</div><div class="NLM_p">The MSD plates were washed 3 times with 1× Tris wash buffer.</div><div class="NLM_p last">An amount of 150 μL of 1× read buffer T with surfactant (Meso Scale Discovery) was added to the plate, and the relative light units (RLUs) were recorded in a Sector Imager 6000 (Meso Scale Discovery). Dose response of compound induced changes to ratio of ub-MDM2/Total MDM2 was fit using Genedata Screener software.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Cell Viability Assays</h3><div class="NLM_p last">EOL-1 cells were maintained in RPMI media containing 10% FBS. 15 000 cells were seeded in 1-well of a 96-well plate (Corning cat. no. 3904). Cells were treated immediately or the next day and then monitored for 48h via live imaging using an Incucyte instrument. To monitor caspase activity, 2 μM CellEvent caspase 3/7 reagent (Life Technologies catalog no. C10423) was added to seeding media. Scans were collected every 3 h, using a 4× objective. Phase contrast was used to measure cell confluence, while green fluorescence was used to measure caspase activity. The images were analyzed using IncuCyte software (Basic Analysis parameters). All treatments were done in triplicate. 48 h after imaging, CellTiter-Glo reagent (Promega catalog no. G7570) was added to each well as an independent measure of number of remaining cells at the end of the experiment. CTG reagent was added following the vendor protocol.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Western Blots</h3><div class="NLM_p last">A total of 5 × 10<sup>6</sup> cells were seeded into 1 well of a 12-well plate and treated with DMSO, 1.3 μM or 5 μM compounds, for 4 h prior to lysis for Western blot analysis. Blots were scanned and analyzed using a Licor technology. Antibodies used were the following: USP7 (Abcam catalog no. ab84098), MDM2 (Santa Cruz catalog no. sc-965), tubulin (Licor catalog no. 926-42211), p53 (Thermo Scientific catalog no. MS738-P1), p21 (Millipore catalog no. 05-655).</div></div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Synthesis. General Methods</h3><div class="NLM_p last">All solvents and reagents were used as obtained. NMR analysis was performed in deuterated solvent on Bruker Avance 400 or 500 MHz NMR spectrometer. The spectra were referenced internally to tetramethylsilane (TMS). Chemical shifts are reported in ppm (δ) (in the NMR description, s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, and br = broad peak). All coupling constants (<i>J</i>) are reported in hertz. Mass spectra were measured with a Finnigan SSQ710C spectrometer using an ESI source coupled to a Waters 600MS high performance liquid chromatography (HPLC) system operating in reverse-phase mode with an X-bridge phenyl column of dimensions 150 mm × 2.6 mm, with 5 mm sized particles. Preparatory-scale silica gel chromatography was performed using medium-pressure liquid chromatography (MPLC) on a CombiFlash Companion (Teledyne ISCO) with RediSep normal phase silica gel (35–60 μm) columns and UV detection at 254 nm. Reverse-phase HPLC was used to purify compounds as needed by elution from a Phenomenex Gemini-NX C18 column (20.2 mm × 50 mm, 5 μm) as stationary phase using mobile phase indicated and operating at a 35 mL/min flow rate on a Waters 3100 mass-directed prep instrument. Chemical purities were >95% for all final compounds as assessed by LC/MS analysis. The following analytical method was used to determine chemical purity of final compounds: HPLC Agilent 1200, water with 0.05% TFA, acetonitrile with 0.05% TFA (buffer B), Agilent SB-C18, 1.8 mm, 2.1 mm × 30 mm, 25 °C, 3–95% buffer B in 8.5 min, 95% in 2.5 min, 400 mL/min, 220 and 254 nm, equipped with Agilent quadrupole 6140, ESI positive, 90–1300 amu.</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Compound <b>1</b> was obtained from Sigma-Aldrich (catalog no. CDS002148)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.87 (s, 2H), 7.31 (td, <i>J</i> = 8.4, 7.2 Hz, 1H), 6.87–6.71 (m, 3H), 3.88 (d, <i>J</i> = 6.3 Hz, 2H), 3.28 (dt, <i>J</i> = 12.7, 3.3 Hz, 2H), 2.88 (td, <i>J</i> = 12.7, 3.1 Hz, 2H), 2.05 (dddd, <i>J</i> = 14.2, 8.0, 6.4, 3.0 Hz, 1H), 1.90 (dd, <i>J</i> = 14.2, 3.5 Hz, 2H), 1.57–1.41 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 210.1 (M + H).</div></div><div id="sec4_11_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Synthesis of Compound <b>2</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). 4-(3-(1-Methylpiperidin-4-yl)-1,2,4-oxadiazol-5-yl)phenol (<b>2</b>)</h3><div class="NLM_p last">To a solution of compound <b>3</b> (400 mg, 1.6 mmol) and 37% HCHO (0.26 mL) in water (5.0 mL) was added AcOH (0.3 mL) followed by addition of NaBH<sub>3</sub>CN (200 mg, 3.18 mmol). The mixture was stirred at room temperature for 1 h. It was quenched with aqueous NaHCO<sub>3</sub> and concentrated to give the crude product, which was purified by prep-HPLC (FA) to give the desired product (160 mg, 38%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.52 (br, 1H), 7.98 (d, <i>J</i> = 6.8 Hz, 2H), 6.96 (d, <i>J</i> = 6.8 Hz, 2H), 3.51–3.48 (m, 2H), 3.20–3.18 (m, 3H), 2.85 (s, 3H), 2.37–2.32 (m, 2H), 2.17–2.14 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 259.9 (M + H).</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 4-(3-(Piperidin-4-yl)-1,2,4-oxadiazol-5-yl)phenol (<b>3</b>)</h3><div class="NLM_p last">To a solution of <b>35</b> (345 mg, 1.0 mmol) in EtOAc (3.0 mL) was added HCl/EtOAc (5.0 mL). The formed mixture was stirred for 1 h at room temperature. Solvent was removed to give the desired product (280 mg, 99%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.96 (d, <i>J</i> = 6.8 Hz, 2H), 6.93 (d, <i>J</i> = 6.8 Hz, 2H), 3.49–3.46 (m, 2H), 3.25–3.19 (m, 3H), 2.30 (m, 2H), 2.08–2.06 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 246.4 (M + H).</div></div><div id="sec4_12_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Compounds <b>4</b> and <b>5</b> were prepared in a manner similar to compound <b>2</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). 4-(3-(1-Methylpyrrolidin-3-yl)-1,2,4-oxadiazol-5-yl)phenol (<b>4</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.93–7.89 (m, 2H), 6.97–6.92 (m, 2H), 3.55–3.47 (m, 1H), 2.94–2.89 (m, 1H), 2.69–2.61 (m, 2H), 2.58–2.53 (m, 1H), 2.30 (s, 3H), 2.26–2.17 (m, 1H), 2.12–2.03 (m, 1H). LCMS (ESI <i>m</i>/<i>z</i>): 246.4 (M + H).</div></div><div id="sec4_12_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 4-(3-(Pyrrolidin-3-yl)-1,2,4-oxadiazol-5-yl)phenol (<b>5</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.47 (s, 1H), 9.22 (s, 1H), 7.99–7.89 (m, 2H), 7.03–6.93 (m, 2H), 3.78 (p, <i>J</i> = 7.6 Hz, 1H), 3.64 (dd, <i>J</i> = 8.2, 11.7 Hz, 1H), 3.41 (dd, <i>J</i> = 7.4, 11.7 Hz, 1H), 3.39–3.25 (m, 2H), 2.45–2.36 (m, 1H), 2.26–2.14 (m, 1H). LCMS (ESI <i>m</i>/<i>z</i>): 232.4 (M + H).</div></div><div id="sec4_13_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Synthesis of Compound <b>6</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). 4-(4-(Piperidin-4-yl)oxazol-2-yl)phenol (<b>6</b>)</h3><div class="NLM_p last">A mixture of <i>tert</i>-butyl 4-(2-(4-methoxyphenyl)oxazol-4-yl)piperidine-1-carboxylate (<b>37</b>, 0.1 g, 0.3 mmol) and hydrobromic acid (2 mL) was heated to 120 °C for 16 h under a nitrogen atmosphere. After cooling to room temperature, the mixture was concentrated in vacuo. The crude residue was purified by reverse phase chromatography (hydrochloric acid) to give the title compound (70 mg, 80%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.15 (s, 1H), 8.85 (s, 1H), 7.89 (s, 1H), 7.74 (d, <i>J</i> = 8.4 Hz, 2H), 6.87 (d, <i>J</i> = 8.4 Hz, 2H), 3.28–3.23 (m, 2H), 3.01–2.91 (m, 2H), 2.85–2.79 (m, 1H), 2.07–2.04 (m, 2H), 1.82–1.72 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>) 245.4.</div></div><div id="sec4_13_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 4-(5-(Piperidin-4-yl)oxazol-2-yl)phenol (<b>7</b>)</h3><div class="NLM_p last">A mixture of <i>tert</i>-butyl 4-(2-(4-methoxyphenyl)oxazol-5-yl)piperidine-1-carboxylate (<b>42,</b> 0.4 g, 1.06 mmol) and hydrobromic acid (10 mL) was heated to 120 °C for 6 h under a nitrogen atmosphere. After cooling to room temperature, the mixture was concentrated in vacuo. The crude residue was purified by reverse phase chromatography (formic acid) to give the title compound (66 mg, 24%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.09 (s, 1H), 9.18 (d, <i>J</i> = 9.5 Hz, 1H), 8.87 (d, <i>J</i> = 10.7 Hz, 1H), 7.91 (d, <i>J</i> = 1.0 Hz, 1H), 7.82–7.73 (m, 2H), 6.94–6.86 (m, 2H), 3.29 (dt, <i>J</i> = 12.8, 3.1 Hz, 2H), 2.99 (dtd, <i>J</i> = 12.7, 9.9, 2.7 Hz, 2H), 2.86 (tdd, <i>J</i> = 11.4, 10.4, 4.0, 2.0 Hz, 1H), 2.09 (dd, <i>J</i> = 14.3, 3.6 Hz, 2H), 1.88–1.73 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 245.4 (M + H).</div></div><div id="sec4_13_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 4-(5-(Piperidin-4-yl)oxazol-2-yl)phenol (<b>8</b>)</h3><div class="NLM_p last">Prepared in a manner similar to compound <b>7</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.36 (s, 1H), 7.75–7.68 (m, 1H), 7.00 (s, 1H), 6.73–6.65 (m, 2H), 3.22–3.15 (m, 2H), 3.04–2.98 (m, 1H), 2.90–2.83 (m, 2H), 2.05–2.00 (m, 2H), 1.74–1.63 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 263.3 (M + H).</div></div><div id="sec4_13_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 4-[5-(4-Hydroxyphenyl)-3-pyridyl]phenol (<b>17</b>)</h3><div class="NLM_p last">Prepared in a manner similar to <b>18</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.66 (s, 2H), 8.71 (d, J = 2.2 Hz, 2H), 8.09 (t, <i>J</i> = 2.2 Hz, 1H), 7.69–7.60 (m, 4H), 6.94–6.85 (m, 4H). LCMS (ESI <i>m</i>/<i>z</i>): 264.5 (M + H).</div></div><div id="sec4_14_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Synthesis of Compound <b>18</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). 4-[2,6-Diamino-5-(4-hydroxyphenyl)-3-pyridyl]phenol (<b>18</b>)</h3><div class="NLM_p">Into a 500 mL three-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen was placed a solution of <b>44</b> (12 g, 59.68 mmol, 1.00 equiv) in CH<sub>3</sub>CN (100 mL), (4-methoxyphenyl)boronic acid (11 g, 72.39 mmol, 1.20 equiv), Na<sub>2</sub>CO<sub>3</sub>(120 mL, sat.), and Pd(dppf)<sub>2</sub>Cl<sub>2</sub> (1.2 g, 1.64 mmol, 0.03 equiv). The resulting solution was stirred at 110 °C for 1 h, diluted with of 500 mL of EtOAc, and then extracted 2 × 500 mL with EtOAc. The combined organic layers were washed 3 × 200 mL with brine, dried over anhydrous sodium sulfate, and concentrated under vacuum. The crude residue was carried forward without further purification.</div><div class="NLM_p last">The crude residue was dissolved in dichloromethane (100 mL) and charged with tribromoborane (19.6 g, 78.24 mmol, 3.00 equiv) at 0 °C. The resulting solution was stirred at room temperature for 1 h and then quenched by the addition of 100 mL of NaHCO<sub>3</sub> (1M) at 0 °C. The solids were collected by filtration and then washed with 1 × 100 mL of H<sub>2</sub>O and 1 × 300 mL of EtOAc/PE (1:1) to afford 6.3 g (83%) of the title compound as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.34 (s, 2H), 7.24–7.15 (m, 4H), 6.89 (s, 1H), 6.83–6.74 (m, 4H), 5.06 (<u class="uu">s</u>, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.23, 154.35, 139.83, 129.86, 115.98, 110.55. LCMS (ESI <i>m</i>/<i>z</i>): 294.1 (M + H).</div></div><div id="sec4_15_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Compounds <b>19</b>–<b>23</b> were prepared according to the synthesis of <b>18</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). 4-[6-Amino-5-(4-hydroxyphenyl)-4-methyl-3-pyridyl]phenol (<b>19</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.48 (s, 2H), 8.14 (s, 1H), 7.71 (s, 1H), 7.14–7.00 (m, 4H), 6.92–6.84 (m, 2H), 6.84–6.75 (m, 2H), 4.95 (s, 2H), 1.81 (s, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 293.0 (M + H).</div></div><div id="sec4_15_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 4-[6-Amino-4-ethyl-5-(4-hydroxyphenyl)-3-pyridyl]phenol (<b>20</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.48 (s, 2H), 8.14 (s, 1H), 7.65 (s, 1H), 7.13–7.00 (m, 4H), 6.92–6.83 (m, 2H), 6.84–6.75 (m, 2H), 4.90 (s, 2H), 2.24 (q, <i>J</i> = 7.4 Hz, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 306.1 (M + H).</div></div><div id="sec4_15_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 4-[6-Amino-5-(4-hydroxyphenyl)-4-propyl-3-pyridyl]phenol (<b>21</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.80 (s, 1H), 9.82 (s, 1H), 9.70 (s, 1H), 7.77 (s, 1H), 7.22–7.04 (m, 6H), 7.00–6.88 (m, 2H), 6.91–6.79 (m, 2H), 1.04 (h, <i>J</i> = 7.3 Hz, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 321.1 (M + H).</div></div><div id="sec4_15_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 4-[6-Amino-5-(4-hydroxyphenyl)-4-isopropyl-3-pyridyl]phenol (<b>22</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.48 (s, 2H), 8.14 (s, 1H), 7.58 (s, 1H), 7.09–6.97 (m, 5H), 6.92–6.83 (m, 2H), 6.81–6.73 (m, 2H), 2.87 (p, <i>J</i> = 7.2 Hz, 1H), 0.79 (d, <i>J</i> = 7.2 Hz, 7H). LCMS (ESI <i>m</i>/<i>z</i>): 321.1 (M + H).</div></div><div id="sec4_15_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 2-Amino-3,5-bis(4-hydroxyphenyl)pyridine-4-carbonitrile (<b>23</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.75 (s, 2H), 8.15 (s, 1H), 8.09 (s, 1H), 7.37–7.29 (m, 2H), 7.30–7.21 (m, 2H), 6.95–6.81 (m, 4H), 5.86 (s, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 304.1 (M + H).</div></div><div id="sec4_16_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Compounds <b>24</b>–<b>27</b> were prepared according to the synthesis of compound <b>28</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). 4-(2-Amino-4-ethyl-5-phenyl-3-pyridyl)phenol (<b>24</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.52 (s, 1H), 7.70 (s, 1H), 7.46–7.27 (m, 6H), 7.10–7.02 (m, 2H), 6.93–6.84 (m, 2H), 5.01 (s, 2H), 3.27 (s, 2H), 2.26 (q, <i><u class="uu">J</u></i> = 7.4 Hz, 2H), 0.61 (t, J = 7.4 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 291.1 (M + H).</div></div><div id="sec4_16_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 3-[6-Amino-4-ethyl-5-(4-hydroxyphenyl)-3-pyridyl]benzonitrile (<b>25</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR(400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.54 (s, 1H), 7.81 (dd, <i>J</i> = 7.3, 1.5 Hz, 2H), 7.77–7.52 (m, 4H), 7.13–7.01 (m, 2H), 6.93–6.85 (m, 2H), 5.10 (s, 2H), 3.28 (s, 2H), 2.24 (q, <i>J</i> = 7.4 Hz, 2H), 0.60 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 316.1 (M + H).</div></div><div id="sec4_16_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 4-[2-Amino-4-ethyl-5-(1<i>H</i>-indazol-6-yl)-3-pyridyl]phenol (<b>26</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.07–13.02 (m, 1H), 9.55 (s, 1H), 8.09 (s, 1H), 7.81–7.74 (m, 2H), 7.42 (d, <i>J</i> = 1.3 Hz, 1H), 7.07 (td, <i>J</i> = 8.3, 1.7 Hz, 3H), 6.94–6.84 (m, 2H), 5.12 (s, 2H), 2.29 (q, <i>J</i> = 7.4 Hz, 2H), 0.62 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 331.1 (M + H).</div></div><div id="sec4_16_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 5-[6-Amino-4-ethyl-5-(4-hydroxyphenyl)-3-pyridyl]-<i>N</i>-methylpyridine-2-carboxamide (<b>27</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.56 (s, 1H), 8.80 (q, <i>J</i> = 4.7 Hz, 1H), 8.59 (dd, <i>J</i> = 2.3, 0.9 Hz, 1H), 8.07 (dd, <i>J</i> = 8.0, 0.9 Hz, 1H), 7.96 (dd, <i>J</i> = 8.0, 2.2 Hz, 1H), 7.79 (s, 1H), 7.11–7.02 (m, 2H), 6.94–6.86 (m, 2H), 5.19 (s, 2H), 2.84 (d, <i>J</i> = 4.8 Hz, 3H), 2.26 (q, <i>J</i> = 7.4 Hz, 2H), 0.62 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 349.1 (M + H).</div></div><div id="sec4_16_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 4-[2-Amino-4-ethyl-5-(1<i>H</i>-indazol-5-yl)-3-pyridyl]phenol (<b>28</b>)</h3><div class="NLM_p last">Into a 250 mL three-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen was placed <b>50</b> (1.0 g, 3.41 mmol, 1.00 equiv), 6-(tetramethyl-1,3,2-dioxaborolan-2-yl)-2<i>H</i>-indazole (880 mg, 3.41 mmol, 1.00 equiv), potassium carbonate (3.3 g, 23.88 mmol, 7.00 equiv), water (30 mL), 1,4-dioxane (25 mL), and Pd(dppf)<sub>2</sub>Cl<sub>2</sub> (200 mg, 0.3 mmol, 0.1 equiv). The resulting solution was stirred at 80 °C for 16 h, and diluted with 500 mL of H<sub>2</sub>O and 500 mL of ethyl acetate. The organic layer was washed twice with 250 mL of brine and concentrated under vacuum. The residue was purified on a silica gel column, eluting with DCM/CH<sub>3</sub>OH (20:1 to 10:1) to afford the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.07 (s, 1H), 9.52 (s, 1H), 8.07 (d, <i>J</i> = 1.0 Hz, 1H), 7.74 (s, 1H), 7.65 (dd, J = 1.6, 0.8 Hz, 1H), 7.56 (dt, <i>J</i> = 8.6, 0.9 Hz, 1H), 7.28 (dd, <i>J</i> = 8.5, 1.6 Hz, 1H), 7.10–7.04 (m, 2H), 6.92–6.86 (m, 2H), 4.94 (s, 2H), 2.26 (q, <i>J</i> = 7.4 Hz, 2H), 0.60 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 331.1 (M + H).</div></div><div id="sec4_16_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 4-[2-Amino-4-ethyl-5-(2-methylindazol-6-yl)-3-pyridyl]phenol (<b>29</b>)</h3><div class="NLM_p last">Prepared according to the synthesis of <b>28</b>. Refer to Kategaya et al. for spectral information.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div></div><div id="sec4_17_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Synthesis of Compound <b>30</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). 4-(6-Amino-4-ethyl-5-phenyl-3-pyridyl)phenol (<b>30</b>)</h3><div class="NLM_p last">A solution of 4-(6-amino-5-bromo-4-ethylpyridin-3-yl)phenol (59 mg, 0.2 mmol) in 0.5 mL of 1,4-dioxane was charged with phenylboronic acid (36 mg, 0.3 mmol), bis(diphenylphosphino)ferrocene]dichloropalladium(II) (14 mg, 0.02 mmol), and 0.5 mL of 1 M K<sub>2</sub>CO<sub>3</sub>. The mixture was then heated at 120 °C for 5 min. The mixture was then diluted with ethyl acetate and water. The layers were separated, and the organic layer was washed once with water, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was then purified by reverse-phase preparatory HPLC to afford the title compound (29 mg, 50% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.39 (s, 1H), 7.70 (s, 1H), 7.56–7.46 (m, 2H), 7.46–7.35 (m,1H), 7.30–7.22 (m, 2H), 7.14–7.06 (m, 2H), 6.84–6.75 (m, 2H), 4.90 (s, 2H), 2.22 (q, <i>J</i> = 7.5 Hz, 2H), 0.61 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 291.0 (M + H).</div></div><div id="sec4_18_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Compounds <b>31</b> and <b>32</b> were prepared according to the synthesis of compound <b>30</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). 3-[2-Amino-4-ethyl-5-(4-hydroxyphenyl)-3-pyridyl]benzonitrile (<b>31</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.43 (s, 1H), 7.87 (dt, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.78–7.63 (m, 4H), 7.61 (dt, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.15–7.02 (m, 3H), 6.86–6.76 (m, 3H), 5.19 (s, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 316.0 (M + H).</div></div><div id="sec4_18_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 4-[6-Amino-4-ethyl-5-(1<i>H</i>-indazol-5-yl)-3-pyridyl]phenol (<b>32</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.14 (s, 1H), 9.42 (s, 1H), 8.14 (s, 1H), 8.10 (d, <i>J</i> = 0.9 Hz, 1H), 7.70 (s, 1H), 7.70–7.62 (m, 2H), 7.19 (dd, <i>J</i> = 8.3, 1.7 Hz, 1H), 7.16–7.00 (m, 2H), 6.85–6.75 (m, 2H), 4.97 (s, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 331.1 (M + H).</div></div><div id="sec4_18_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> <i>tert</i>-Butyl 4-(5-(4-Hydroxyphenyl)-1,2,4-oxadiazol-3-yl)piperidine-1-carboxylate (<b>35</b>)</h3><div class="NLM_p last">To a solution of compound <b>33</b> (2.1 g, 10 mmol) in EtOH (5 mL) was added aqueous NH<sub>4</sub>OH (1.5 mL, 50%). The mixture was heated at reflux for 70 min. The mixture was concentrated in vacuo to give the crude product, which was used for the next step directly. A mixture of compound <b>34</b> (2.14 g, 10 mmol) and CDI (1.62 g, 10 mmol) in DMF (5.0 mL) was stirred for 0.5 h at room temperature. Then to this mixture was added the above crude product, and the mixture was heated to 110 °C for 2 h. Solvent was removed in vacuo and the crude product was purified by silica gel column chromatography (DCM/CH<sub>3</sub>OH = 30/1) to afford the desired product as a white solid (345 mg, 10%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.98 (d, <i>J</i> = 6.8 Hz, 2H), 6.95 (d, <i>J</i> = 6.8 Hz, 2H), 4.13–4.10 (m, 2H), 3.36–3.32 (m, 3H), 2.04 (m, 2H), 1.82–1.73 (m, 2H), 1.50 (s, 9H). LCMS (ESI <i>m</i>/<i>z</i>): 245.5 [M + H].</div></div><div id="sec4_18_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> <i>tert</i>-Butyl 4-(2-(4-Methoxyphenyl)oxazol-4-yl)piperidine-1-carboxylate (<b>37</b>)</h3><div class="NLM_p last">A mixture of 4-methoxybenzamide (0.5 g, 3.31 mmol) and <i>tert</i>-butyl 4-(2-bromoacetyl)piperidine-1-carboxylate (<b>36</b>, 2.03 g, 6.62 mmol) in toluene (10 mL) was heated to 100 °C for 16 h under a nitrogen atmosphere. After cooling to room temperature, the mixture was concentrated in vacuo. The crude residue was purified by silica gel chromatography to give the title compound (0.1 g, 8%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.15 (s, 1H), 8.85 (s, 1H), 7.89 (s, 1H), 7.74 (d, <i>J</i> = 8.4 Hz, 2H), 6.87 (d, <i>J</i> = 8.4 Hz, 2H), 3.28–3.23 (m, 2H), 3.01–2.91 (m, 2H), 2.85–2.79 (m, 1H), 2.07–2.04 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 246.4 (M + H)</div></div><div id="sec4_18_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> <i>tert</i>-Butyl 4-(1-Hydroxy-2-nitroethyl)piperidine-1-<i>tert</i>-butyl-4-(1-hydroxy-2-nitroethyl)piperidine-1-carboxylate (<b>39</b>)</h3><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-formylpiperidine-1-carboxylate (<b>38</b>, 20 g, 93.8 mmol) and nitromethane (11.45 g, 187 mmol) in THF (100 mL) and <i>t</i>-BuOH (100 mL) was added <i>t</i>-BuOK (10.52 g, 93.8 mmol). The mixture was stirred at room temperature for 2 h. The mixture was acidified with AcOH to pH 6 and then extracted with EtOAc (100 mL × 3). The combined organic layers were washed with brine (100 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuo. The crude residue was purified by silica gel chromatography to give the title compound (16 g, 62%) as a faint yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 4.19–4.01 (m, 2H), 3.28–3.24 (m, 1H), 2.89–2.86 (m, 1H), 2.55–2.64 (m, 2H), 2.22–2.13 (m, 4H), 1.84–1.79 (m, 1H), 1.56–1.41 (m, 2H), 1.43 (s, 9H), 1.28–1.17 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 275.4 (M + H).</div></div><div id="sec4_18_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> <i>tert</i>-Butyl 4-(2-Amino-1-hydroxyethyl)piperidine-1-carboxylate (<b>40</b>)</h3><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-(1-hydroxy-2-nitroethyl)piperidine-1-carboxylate (<b>39,</b> 16 g, 58 mmol) in MeOH (25 mL) was added 10% Pd/C (1.6 g). The mixture was stirred at room temperature for 12 h under a hydrogen atmosphere (50 psi). The mixture was filtered and the filtrate was concentrated in vacuo to give the title compound (12 g, crude) as a yellow solid that required no further purification. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 5.41 (d, <i>J</i> = 6.4 Hz, 1H), 4.76–4.72 (m, 1H), 4.40–4.34 (m, 1H), 3.95–3.89 (m, 3H), 2.70–2.52 (m, 2H), 1.69–1.65 (m, 1H), 1.55–1.52 (m, 2H), 1.39 (s, 9H), 1.23–1.10 (m, 2H). LCMS (ESI <i>m</i>/<i>z</i>): 246.4 (M + H).</div></div><div id="sec4_18_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> <i>tert</i>-Butyl 4-(4-Methoxybenzamido)acetyl)piperidine-1-carboxylate (<b>41</b>)</h3><div class="NLM_p">To a solution of 4-methoxybenzoic acid (1.5 g, 8.8 mmol) and <i>tert</i>-butyl 4-(2-amino-1-hydroxyethyl)piperidine-1-carboxylate (<b>40</b>, 2.58 g, 10.58 mmol) in DMF (20 mL) were added HATU (6.7 g, 17.63 mmol) and triethylamine (3.67 mL, 26.45 mmol). The mixture was stirred at room temperature for 6 h. The mixture was concentrated in vacuo. The crude residue was purified by silica gel chromatography (petroleum ether/EtOAc = 1:1) to give <i>tert</i>-butyl 4-(2-(2-fluoro-4-methoxybenzamido)-1-hydroxyethyl)piperidine-1-carboxylate (1.78 g, 51%) as a faint yellow solid.</div><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-(4-methoxybenzamido)-1-hydroxyethyl)piperidine-1-carboxylate (1.2 g, 3.03 mmol) in DCM (20 mL) was added Dess–Martin reagent (3.2 g, 7.55 mmol). The mixture was stirred at room temperature for 4 h. The mixture was concentrated in vacuo. The crude residue was purified by silica gel chromatography to give the title compound (1.1 g, 92%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.84–7.79 (m, 1H), 7.66–7.61 (m, 1H), 6.89–6.80 (m, 2H), 4.02–3.93 (m, 2H), 3.73 (s, 3H), 3.41–3.36 (m, 2H), 3.16–3.09 (m, 1H), 2.59–2.51 (m, 2H), 1.72–1.68 (m, 2H), 1.52–1.45 (m, 2H), 1.35 (s, 9H). LCMS (ESI <i>m</i>/<i>z</i>): 377.2.4 (M + H).</div></div><div id="sec4_18_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> <i>tert</i>-Butyl 4-(2-(4-Methoxyphenyl)oxazol-5-yl)piperidine-1-carboxylate (<b>42</b>)</h3><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-(2-(2(4-methoxybenzamido)acetyl)piperidine-1-carboxylate (0.5 g, 1.27 mmol) in THF (20 mL) was added Burgess reagent (906 mg, 3.8 mmol). The mixture was heated to 60 °C for 6 h under a nitrogen atmosphere. After cooling to room temperature, the mixture was concentrated in vacuo. The crude residue was purified by silica gel chromatography to give the title compound (0.4 g, 84%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.36 (s, 1H), 7.75–7.68 (m, 1H), 7.00 (s, 1H), 6.73–6.65 (m, 2H), 3.22–3.15 (m, 5H), 3.04–2.98 (m, 1H), 2.90–2.83 (m, 2H), 2.05–2.00 (m, 2H), 1.74–1.63 (m, 2H), 1.40–1.25 (m, 9H). LCMS (ESI <i>m</i>/<i>z</i>): 358.4 (M + H).</div></div><div id="sec4_18_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 3,5-Dibromopyridine-2,6-diamine (<b>44</b>, R1 and R2 = NH2, R3 = H)</h3><div class="NLM_p last">Into a 500 mL three-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen were placed 4-ethylpyridin-2-amine (10 g, 81.85 mmol, 1.00 equiv), tetrahydrofuran (200 mL), and NBS (29 g, 162.94 mmol, 2.00 equiv) at 0 °C. The resulting solution was stirred at room temperature for 15 min and then was concentrated under vacuum. The residue was purified on a silica gel column, eluting with DCM/MeOH (100:1 to 20:1) to afford 18 g (79%) of 3,5-dibromo-4-ethylpyridin-2-amine as a white solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.04 (s, 1H), 7.33–7.23 (m, 1H), 4.93 (s, 2H), 2.93 (q, <i>J</i> = 7.5 Hz, 2H), 1.17 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 246.4 (M + H).</div></div><div id="sec4_18_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 3,5-Dibromo-4-ethylpyridin-2-amine (<b>46</b>)</h3><div class="NLM_p last">Into a 500 mL three-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen were placed 4-ethylpyridin-2-amine (10 g, 81.85 mmol, 1.00 equiv), tetrahydrofuran (200 mL), and NBS (29 g, 162.94 mmol, 2.00 equiv) at 0 °C. The resulting solution was stirred at room temperature for 15 min and then was concentrated under vacuum. The residue was purified on a silica gel column eluting with DCM/MeOH (100:1 to 20:1) to afford 18 g (79%) of the title compound. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.04 (s, 1H), 4.93 (s, 2H), 2.93 (q, <i>J</i> = 7.5 Hz, 2H), 1.17 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 278.9 (M + H).</div></div><div id="sec4_18_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 3-Bromo-4-ethylpyridin-2-amine (<b>47</b>)</h3><div class="NLM_p last">Into a 500 mL three-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen was placed <b>45</b> (18 g, 64.29 mmol, 1.00 equiv) in tetrahydrofuran (300 mL). To this was added a solution of <i>n</i>-BuLi (in hexane) (58 mL, 2.00 equiv, 2.2 mol/L) at −78 °C. The resulting solution was stirred at −78 °C for 1 h, quenched by the addition of 450 mL of NH<sub>4</sub>Cl, and then extracted with 2 × 500 mL of ethyl acetate. The combined organic layers were washed with 2 × 500 mL of brine, dried over anhydrous sodium sulfate, and concentrated under vacuum. The crude product was purified by flash-prep-HPLC to afford 12 g (93%) of the title compound as a white solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.89 (d, <i>J</i> = 5.1 Hz, 1H), 6.60–6.48 (m, 1H), 4.93 (s, 2H), 2.69 (q, <i>J</i> = 7.6 Hz, 2H), 1.33–1.14 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 201.0 (M + H).</div></div><div id="sec4_18_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 4-Ethyl-3-(4methoxyphenyl)pyridn-2-amine (<b>48</b>)</h3><div class="NLM_p last">Into a 500 mL three-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen was placed a solution of 3-bromo-4-ethylpyridin-2-amine (12 g, 59.68 mmol, 1.00 equiv) in CH<sub>3</sub>CN (100 mL), (4-methoxyphenyl)boronic acid (11 g, 72.39 mmol, 1.20 equiv), Na<sub>2</sub>CO<sub>3</sub> (120 mL, sat.), and Pd(dppf)<sub>2</sub>Cl<sub>2</sub> (1.2 g, 1.64 mmol, 0.03 equiv). The resulting solution was stirred at 110 °C for 1 h, diluted with 500 mL of EA, and then extracted with of 2 × 500 mL of ethyl acetate. The combined organic layers were washed with 3 × 200 mL of brine, dried over anhydrous sodium sulfate, and concentrated under vacuum. The residue was purified on a silica gel column, eluting with ethyl acetate/petroleum ether (1:100 to 1:10) to afford 10 g (73%) of the title compound. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.06 (d, <i>J</i> = 5.1 Hz, 1H), 7.21–7.11 (m, 2H), 7.06–6.97 (m, 2H), 6.64 (d, <i>J</i> = 5.4 Hz, 1H), 4.48 (s, 2H), 3.86 (s, 3H), 2.31 (q, <i>J</i> = 7.6 Hz, 2H), 1.11–0.88 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 229.1 (M + H).</div></div><div id="sec4_18_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 5-Bromo-4-ethyl-3-(4-methoxyphenyl)pyridin-2-amine (<b>49</b>)</h3><div class="NLM_p last">Into a 250 mL three-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen were placed 4-ethyl-3-(4-methoxyphenyl)pyridn-2-amine (10 g, 43.80 mmol, 1.00 equiv), tetrahydrofuran (100 mL), followed by NBS (7.8 g, 43.83 mmol, 1.00 equiv) at 0 °C. The resulting solution was stirred at room temperature for 15 min, diluted with 500 mL of EtOAc and 500 mL of H<sub>2</sub>O. The resulting solution was extracted with 2 × 500 mL of ethyl acetate. The organic layers were combined, washed with 2 × 500 mL of brine, and concentrated under vacuum. The residue was purified on a silica gel column, eluting with ethyl acetate/petroleum ether (1:20 to 1:10) to afford 8 g (59%) of the title compound. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.09 (s, 1H), 7.22–7.09 (m, 2H), 7.09–6.98 (m, 2H), 4.55 (s, 2H), 2.47 (q, <i>J</i> = 7.5 Hz, 2H), 1.10–0.84 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI M/Z): 307.1 (M + H).</div></div><div id="sec4_18_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 4-(2-Amino-5-bromo-4-ethylpyridin-3-yl)phenol (<b>50</b>)</h3><div class="NLM_p last">Into a 250 mL three-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen was placed 5-bromo-4-ethyl-3-(4-methoxyphenyl)pyridin-2-amine as a white solid (8 g, 26.04 mmol, 1.00 equiv), dichloromethane (100 mL), followed by tribromoborane (19.6 g, 78.24 mmol, 3.00 equiv) at 0 °C. The resulting solution was stirred at room temperature for 1 h and then quenched by the addition of 100 mL of 1 M NaHCO<sub>3</sub> at 0 °C. The solids were collected by filtration and then washed with 1 × 100 mL of H<sub>2</sub>O and 1 × 300 mL of EA/PE (1:1) to afford 6.3 g (83%) of 4-(2-amino-5-bromo-4-ethylpyridin-3-yl)phenol as a white solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.09 (s, 1H), 7.22–7.09 (m, 2H), 7.09–6.98 (m, 2H), 4.55 (s, 2H), 2.47 (q, <i>J</i> = 7.5 Hz, 2H), 1.10–0.84 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 293.0 (M + H).</div></div><div id="sec4_18_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 5-Bromo-4-ethylpyridin-2-amine (<b>52</b>)</h3><div class="NLM_p last">A solution of 4-ethyl-pyridine-2-amine (<b>50</b>, 10 g, 82 mmol) in 300 mL of THF was cooled to 0 °C and charged with <i>N</i>-bromosuccinimide (14.7 g, 82 mmol). The mixture was then stirred at 0 °C for an additional 15 min. The mixture was then concentrated down and the residue was purified by silica gel chromatography (0–5% MeOH in DCM) to afford 5-bromo-4-ethylpyridin-2-amine <b>52</b> (12 g, 72% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.08 (d, <i>J</i> = 0.5 Hz, 1H), 7.32–7.21 (m, 1H), 6.40 (q, <i>J</i> = 0.6 Hz, 1H), 4.36 (s, 2H), 2.62 (qd, <i>J</i> = 7.5, 0.7 Hz, 2H), 1.21 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 201. (M + H).</div></div><div id="sec4_18_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 4-Ethyl-5-(4-methoxyphenyl)pyridin-2-amine (<b>53</b>)</h3><div class="NLM_p last">A solution of 5-bromo-4-ethylpyridin-2-amine (<b>52</b>, 1.0 g, 5.0 mmol) in 12 mL of acetonitrile was charged with 4-methoxyboronic acid (907 mg, 6.0 mmol), bis(diphenylphosphino)ferrocene]dichloropalladium(II) (364 mg, 0.5 mmol), and 12 mL of 1 M potassium carbonate. The mixture was then heated at 120 °C for 5 min. The mixture was then diluted with ethyl acetate and water. The layers were separated, and the organic layer was washed once with water, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was then purified by silica gel chromatography (1–15% MeOH in DCM) to afford 4-ethyl-5-(4-methoxyphenyl)pyridin-2-amine (900 mg, 3.9 mmol, 80%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.80 (s, 1H), 7.23–7.10 (m, 2H), 7.00–6.86 (m, 3H), 4.94 (s, 2H), 3.85 (s, 3H), 2.67 (q, <i>J</i> = 7.5 Hz, 2H), 1.05 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 229.1 (M + H).</div></div><div id="sec4_18_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> 3-Bromo-4-ethyl-5-(4-methoxyphenyl)pyridin-2-amine (<b>54</b>)</h3><div class="NLM_p last">A solution of 5-bromo-4-ethylpyridin-2-amine (9.5 g, 42 mmol) in 100 mL of THF was charged with <i>N</i>-bromosuccinimide (7.5 g, 42 mmol), and the mixture was stirred at room temperature for 15 min. The mixture was then concentrated in vacuo and the residue was purified by silica gel chromatography (0–5% MeOH in DCM) to afford the title compound (8.6 g, 67% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.80 (s, 1H), 7.23–7.10 (m, 2H), 7.00–6.86 (m, 2H), 4.94 (s, 2H), 3.85 (s, 3H), 2.67 (q, <i>J</i> = 7.5 Hz, 2H), 1.05 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 307.0 (M + H).</div></div><div id="sec4_18_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 4-(6-Amino-5-bromo-4-ethylpyridin-3-yl)phenol (<b>55</b>)</h3><div class="NLM_p last">A solution of 3-bromo-4-ethyl-5-(4-methoxyphenyl)pyridin-2-amine (<b>54</b>, 6.6 g, 21 mmol) in 40 mL of THF was charged with 64 mL of 1 M boron tribromide in DCM. After stirring at room temperature for 15 min, the mixture was then cooled to 0 °C and charged with 100 mL of saturated sodium carbonate. The layers were separated and the organic was dried over Mg<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford 4-(6-amino-5-bromo-4-ethylpyridin-3-yl)phenol (5.7 g, 90% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.80 (s, 1H), 7.23–7.10 (m, 2H), 7.00–6.86 (m, 2H), 4.94 (s, 2H), 2.67 (q, <i>J</i> = 7.5 Hz, 2H), 1.05 (t, <i>J</i> = 7.5 Hz, 3H). LCMS (ESI <i>m</i>/<i>z</i>): 293.1 (M + H).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01293">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00538" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00538" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01293" class="ext-link">10.1021/acs.jmedchem.7b01293</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Table S1 of compounds lacking phenol in C3/C5 position, Table S2 of crystallography data and refinement statistics, and Figure S1 of computational torsion scan results (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline"><sup>1</sup>H and <sup>13</sup>C 1D NMR spectra of all synthesized compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biochemical data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_001.pdf">jm7b01293_si_001.pdf (338.19 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_002.pdf">jm7b01293_si_002.pdf (1.69 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_003.csv">jm7b01293_si_003.csv (1.88 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are the following: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHC">5WHC</a> (crystal structure of <b>2</b> in complex with USP7-CD); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WH7">5WH7</a> (crystal structure of <b>16</b> with USP7-CD). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01293" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26536" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26536" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Di Lello</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9683-926X" title="Orcid link">http://orcid.org/0000-0002-9683-926X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2c484540494040435c6c4b494249024f4341"><span class="__cf_email__" data-cfemail="aecac7c2cbc2c2c1deeec9cbc0cb80cdc1c3">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Pastor</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1969786a6d766b376b707a71786b7d597e7c777c377a7674"><span class="__cf_email__" data-cfemail="f2829381869d80dc809b919a938096b295979c97dc919d9f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vickie Tsui</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#33455a50585a561d4740465a73524a521d4a525f561d565746"><span class="__cf_email__" data-cfemail="d6a0bfb5bdbfb3f8a2a5a3bf96b7afb7f8afb7bab3f8b3b2a3">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeremy M. Murray</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert A. Blake</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Frederick Cohen</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Terry D. Crawford</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joy Drobnick</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason Drummond</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lorna Kategaya</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tracy Kleinheinz</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Till Maurer</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lionel Rougé</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xianrui Zhao</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ingrid Wertz</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chudi Ndubaku</span> - <span class="hlFld-Affiliation affiliation">Genentech, Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>P.D.L. and R.P. contributed equally. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d88e3304-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00504" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00504" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Michael Hayes for compound purification and Baiwei Lin for analytical support. We also thank the Genentech Biomolecular Resources group. Use of the Stanford Synchrotron Radiation Lightsource, SLAC National Accelerator Laboratory, is supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences under Contract DE-AC02-76SF00515. The SSRL Structural Molecular Biology Program is supported by the DOE Office of Biological and Environmental Research and by the National Institutes of Health, National Institute of General Medical Sciences (including Grant P41GM103393). The contents of this publication are solely the responsibility of the authors and do not necessarily represent the official views of NIGMS or NIH.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i75" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i75"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i76" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i76"> Abbreviations Used</h2><tr><td class="NLM_term">DUB</td><td class="NLM_def"><p class="first last">deubiquitinating enzyme</p></td></tr><tr><td class="NLM_term">USP</td><td class="NLM_def"><p class="first last">ubiquitin-specific protease</p></td></tr><tr><td class="NLM_term">UCH</td><td class="NLM_def"><p class="first last">ubiquitin C-terminal hydrolase</p></td></tr><tr><td class="NLM_term">OTU</td><td class="NLM_def"><p class="first last">ovarian tumor protease</p></td></tr><tr><td class="NLM_term">HAUSP</td><td class="NLM_def"><p class="first last">herpes virus associated ubiquitin-specific protease</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">USP7-CD</td><td class="NLM_def"><p class="first last">USP7-catalytic domain</p></td></tr><tr><td class="NLM_term">STD</td><td class="NLM_def"><p class="first last">saturation transfer difference</p></td></tr><tr><td class="NLM_term">TROSY</td><td class="NLM_def"><p class="first last">transverse relaxation optimized spectroscopy</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13383" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13383" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 27 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kategaya, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Lello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drummond, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinheinz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upton, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prakash, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heideker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCleland, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritorto, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bainbridge, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwok, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stiffler, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brasher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alessi, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trost, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaishankar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hearn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renslo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arkin, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peale, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gnad, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klijn, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackwood, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forrest, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ndubaku, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwerdtfeger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wertz, I. E.</span><span> </span><span class="NLM_article-title">USP7 small molecule inhibitors interfere with ubiquitin binding</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">550</span><span class="NLM_x">, </span> <span class="NLM_fpage">534</span><span class="NLM_x">–</span> <span class="NLM_lpage">538</span><span class="refDoi"> DOI: 10.1038/nature24006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1038%2Fnature24006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=534-538&author=L.+Kategayaauthor=P.+Di+Lelloauthor=L.+Rougeauthor=R.+Pastorauthor=K.+R.+Clarkauthor=J.+Drummondauthor=T.+Kleinheinzauthor=E.+Linauthor=J.+P.+Uptonauthor=S.+Prakashauthor=J.+Heidekerauthor=M.+McClelandauthor=M.+S.+Ritortoauthor=T.+W.+Bainbridgeauthor=T.+P.+Maauthor=M.+Kwokauthor=Z.+Stifflerauthor=B.+Brasherauthor=D.+R.+Alessiauthor=M.+Trostauthor=Y.+Tangauthor=P.+Jaishankarauthor=B.+Hearnauthor=A.+R.+Rensloauthor=M.+R.+Arkinauthor=F.+Cohenauthor=K.+Yuauthor=F.+Pealeauthor=F.+Gnadauthor=M.+T.+Changauthor=C.+Klijnauthor=E.+Blackwoodauthor=S.+E.+Martinauthor=W.+F.+Forrestauthor=J.+A.+Ernstauthor=C.+Ndubakuauthor=X.+Wangauthor=M.+H.+Beresiniauthor=V.+Tsuiauthor=C.+Schwerdtfegerauthor=R.+A.+Blakeauthor=J.+Murrayauthor=T.+Maurerauthor=I.+E.+Wertz&title=USP7+small+molecule+inhibitors+interfere+with+ubiquitin+binding&doi=10.1038%2Fnature24006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature24006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24006%26sid%3Dliteratum%253Aachs%26aulast%3DKategaya%26aufirst%3DL.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DRouge%26aufirst%3DL.%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DClark%26aufirst%3DK.%2BR.%26aulast%3DDrummond%26aufirst%3DJ.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DE.%26aulast%3DUpton%26aufirst%3DJ.%2BP.%26aulast%3DPrakash%26aufirst%3DS.%26aulast%3DHeideker%26aufirst%3DJ.%26aulast%3DMcCleland%26aufirst%3DM.%26aulast%3DRitorto%26aufirst%3DM.%2BS.%26aulast%3DBainbridge%26aufirst%3DT.%2BW.%26aulast%3DMa%26aufirst%3DT.%2BP.%26aulast%3DKwok%26aufirst%3DM.%26aulast%3DStiffler%26aufirst%3DZ.%26aulast%3DBrasher%26aufirst%3DB.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DTrost%26aufirst%3DM.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DJaishankar%26aufirst%3DP.%26aulast%3DHearn%26aufirst%3DB.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DPeale%26aufirst%3DF.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DChang%26aufirst%3DM.%2BT.%26aulast%3DKlijn%26aufirst%3DC.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DS.%2BE.%26aulast%3DForrest%26aufirst%3DW.%2BF.%26aulast%3DErnst%26aufirst%3DJ.%2BA.%26aulast%3DNdubaku%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DSchwerdtfeger%26aufirst%3DC.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26atitle%3DUSP7%2520small%2520molecule%2520inhibitors%2520interfere%2520with%2520ubiquitin%2520binding%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D534%26epage%3D538%26doi%3D10.1038%2Fnature24006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Hochstrasser, M.</span><span> </span><span class="NLM_article-title">Ubiquitin-dependent protein degradation</span> <span class="citation_source-journal">Annu. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1146/annurev.genet.30.1.405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1146%2Fannurev.genet.30.1.405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=8982460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK2sXht1yk" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1996&pages=405-439&author=M.+Hochstrasser&title=Ubiquitin-dependent+protein+degradation&doi=10.1146%2Fannurev.genet.30.1.405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin-dependent protein degradation</span></div><div class="casAuthors">Hochstrasser, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Genetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">405-439</span>CODEN:
                <span class="NLM_cas:coden">ARVGB7</span>;
        ISSN:<span class="NLM_cas:issn">0066-4197</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review with 115 refs.  A growing no. of cellular regulatory mechanisms are being linked to protein modification by the polypeptide ubiquitin.  These include key transitions in the cell cycle, class I antigen processing, signal transduction pathways, and receptor-mediated endocytosis.  In most, but not all, of these examples, ubiquitination of a protein leads to its degrdn. by the 26S proteasome.  Following attachment of ubiquitin to a substrate and binding of the ubiquitinated protein to the proteasome, the bound substrate must be unfolded (and eventually deubiquitinated) and translocated through a narrow set of channels that leads to the proteasome interior, where the polypeptide is cleaved into short peptides.  Protein ubiquitination and deubiquitination are both mediated by large enzyme families, and the proteasome itself comprises a family of related but functionally distinct particles.  This diversity underlies both the high substrate specificity of the ubiquitin system and the variety of regulatory mechanisms that it serves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8i3KZXfGSRrVg90H21EOLACvtfcHk0lhxOTFvINObzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXht1yk&md5=8b3d3ec0806e94afee1e0374fae69db2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.genet.30.1.405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.genet.30.1.405%26sid%3Dliteratum%253Aachs%26aulast%3DHochstrasser%26aufirst%3DM.%26atitle%3DUbiquitin-dependent%2520protein%2520degradation%26jtitle%3DAnnu.%2520Rev.%2520Genet.%26date%3D1996%26volume%3D30%26spage%3D405%26epage%3D439%26doi%3D10.1146%2Fannurev.genet.30.1.405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Ardley, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, P. A.</span><span> </span><span class="NLM_article-title">E3 ubiquitin ligases</span> <span class="citation_source-journal">Essays Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1042/bse0410015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1042%2Fbse0410015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=16250895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1aitb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=15-30&author=H.+C.+Ardleyauthor=P.+A.+Robinson&title=E3+ubiquitin+ligases&doi=10.1042%2Fbse0410015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">E3 ubiquitin ligases</span></div><div class="casAuthors">Ardley, Helen C.; Robinson, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Essays in Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-30</span>CODEN:
                <span class="NLM_cas:coden">ESBIAV</span>;
        ISSN:<span class="NLM_cas:issn">0071-1365</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The selectivity of the ubiquitin-26 S proteasome system (UPS) for a particular substrate protein relies on the interaction between a ubiquitin-conjugating enzyme (E2, of which a cell contains relatively few) and a ubiquitin-protein ligase (E3, of which there are possibly hundreds).  Post-translational modifications of the protein substrate, such as phosphorylation or hydroxylation, are often required prior to its selection.  In this way, the precise spatio-temporal targeting and degrdn. of a given substrate can be achieved.  The E3s are a large, diverse group of proteins, characterized by one of several defining motifs.  These include a HECT (homologous to E6-assocd. protein C-terminus), RING (really interesting new gene) or U-box (a modified RING motif without the full complement of Zn2+-binding ligands) domain.  Whereas HECT E3s have a direct role in catalysis during ubiquitination, RING and U-box E3s facilitate protein ubiquitination.  These latter 2 E3 types act as adaptor-like mols.  They bring an E2 and a substrate into sufficiently close proximity to promote the substrate's ubiquitination.  Although many RING-type E3s, such as MDM2 (murine double minute clone 2 oncoprotein) and c-Cbl, can apparently act alone, others are found as components of much larger multi-protein complexes, such as the anaphase-promoting complex.  Taken together, these multifaceted properties and interactions enable E3s to provide a powerful, and specific, mechanism for protein clearance within all cells of eukaryotic organisms.  The importance of E3s is highlighted by the no. of normal cellular processes they regulate, and the no. of diseases assocd. with their loss of function or inappropriate targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodKkSLAWRHwrVg90H21EOLACvtfcHk0lhxOTFvINObzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1aitb7K&md5=497eb1d687d5a8b922282881852ddc25</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1042%2Fbse0410015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbse0410015%26sid%3Dliteratum%253Aachs%26aulast%3DArdley%26aufirst%3DH.%2BC.%26aulast%3DRobinson%26aufirst%3DP.%2BA.%26atitle%3DE3%2520ubiquitin%2520ligases%26jtitle%3DEssays%2520Biochem.%26date%3D2005%26volume%3D41%26spage%3D15%26epage%3D30%26doi%3D10.1042%2Fbse0410015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Pickart, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fushman, D.</span><span> </span><span class="NLM_article-title">Polyubiquitin chains: polymeric protein signals</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">610</span><span class="NLM_x">–</span> <span class="NLM_lpage">616</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2004.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2Fj.cbpa.2004.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=15556404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVWjtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2004&pages=610-616&author=C.+M.+Pickartauthor=D.+Fushman&title=Polyubiquitin+chains%3A+polymeric+protein+signals&doi=10.1016%2Fj.cbpa.2004.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Polyubiquitin chains: polymeric protein signals</span></div><div class="casAuthors">Pickart, Cecile M.; Fushman, David</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">610-616</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review and discussion.  The 76-residue protein, ubiquitin, exists within eukaryotic cells both as a monomer and in the form of isopeptide-linked polymers called polyubiquitin chains.  In 2 well-described cases, structurally distinct polyubiquitin chains represent functionally distinct intracellular signals.  Recently, addnl. polymeric structures have been detected in vivo and in vitro, and several large families of proteins with polyubiquitin chain-binding activity have been discovered.  Although the mol. mechanisms governing specificity in chain synthesis and recognition are still incompletely understood, the scope of signaling by polyubiquitin chains is likely to be broader than originally envisioned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpyW8mbbQxk7Vg90H21EOLACvtfcHk0lhxOTFvINObzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVWjtrrN&md5=c7d236b38bc00612688499be1fc1f808</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2004.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2004.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DPickart%26aufirst%3DC.%2BM.%26aulast%3DFushman%26aufirst%3DD.%26atitle%3DPolyubiquitin%2520chains%253A%2520polymeric%2520protein%2520signals%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2004%26volume%3D8%26spage%3D610%26epage%3D616%26doi%3D10.1016%2Fj.cbpa.2004.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Komander, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rape, M.</span><span> </span><span class="NLM_article-title">The ubiquitin code</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">203</span><span class="NLM_x">–</span> <span class="NLM_lpage">229</span><span class="refDoi"> DOI: 10.1146/annurev-biochem-060310-170328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1146%2Fannurev-biochem-060310-170328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=22524316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVGls7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2012&pages=203-229&author=D.+Komanderauthor=M.+Rape&title=The+ubiquitin+code&doi=10.1146%2Fannurev-biochem-060310-170328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin code</span></div><div class="casAuthors">Komander, David; Rape, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">203-229</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The posttranslational modification with ubiquitin, a process referred to as ubiquitylation, controls almost every process in cells.  Ubiquitin can be attached to substrate proteins as a single moiety or as polymeric chains in which successive ubiquitin mols. are connected through specific isopeptide bonds.  Reminiscent of a code, the various ubiquitin modifications adopt distinct conformations and lead to different outcomes in cells.  Here, the structure, assembly, and function of this ubiquitin code are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCbRgaWmg1IrVg90H21EOLACvtfcHk0lgbrZAJXzNOdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVGls7bJ&md5=85b2a166393de56551d541636dbda06c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060310-170328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060310-170328%26sid%3Dliteratum%253Aachs%26aulast%3DKomander%26aufirst%3DD.%26aulast%3DRape%26aufirst%3DM.%26atitle%3DThe%2520ubiquitin%2520code%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2012%26volume%3D81%26spage%3D203%26epage%3D229%26doi%3D10.1146%2Fannurev-biochem-060310-170328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Komander, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clague, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urbe, S.</span><span> </span><span class="NLM_article-title">Breaking the chains: structure and function of the deubiquitinases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">563</span><span class="refDoi"> DOI: 10.1038/nrm2731</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1038%2Fnrm2731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=19626045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovFSru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=550-563&author=D.+Komanderauthor=M.+J.+Clagueauthor=S.+Urbe&title=Breaking+the+chains%3A+structure+and+function+of+the+deubiquitinases&doi=10.1038%2Fnrm2731"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Breaking the chains: structure and function of the deubiquitinases</span></div><div class="casAuthors">Komander, David; Clague, Michael J.; Urbe, Sylvie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">550-563</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Ubiquitination is a reversible protein modification that is implicated in many cellular functions.  Recently, much progress has been made in the characterization of a superfamily of isopeptidases that remove ubiquitin: the deubiquitinases (DUBs; also known as deubiquitinating enzymes).  Far from being uniform in structure and function, these enzymes display a myriad of distinct mechanistic features.  The small no. (<100) of DUBs might at 1st suggest a low degree of selectivity; however, DUBs are subject to multiple layers of regulation that modulate both their activity and their specificity.  Due to their wide-ranging involvement in key regulatory processes, these enzymes might provide new therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8Woj6hOAP5rVg90H21EOLACvtfcHk0lgbrZAJXzNOdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovFSru74%253D&md5=9268e6c24101ff8137937a5c851d5bb8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm2731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2731%26sid%3Dliteratum%253Aachs%26aulast%3DKomander%26aufirst%3DD.%26aulast%3DClague%26aufirst%3DM.%2BJ.%26aulast%3DUrbe%26aufirst%3DS.%26atitle%3DBreaking%2520the%2520chains%253A%2520structure%2520and%2520function%2520of%2520the%2520deubiquitinases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2009%26volume%3D10%26spage%3D550%26epage%3D563%26doi%3D10.1038%2Fnrm2731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saad, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolattukudy, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, M.</span><span> </span><span class="NLM_article-title">MCP-induced protein 1 deubiquitinates TRAF proteins and negatively regulates JNK and NF-kappaB signaling</span> <span class="citation_source-journal">J. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">207</span><span class="NLM_x">, </span> <span class="NLM_fpage">2959</span><span class="NLM_x">–</span> <span class="NLM_lpage">2973</span><span class="refDoi"> DOI: 10.1084/jem.20092641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1084%2Fjem.20092641" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2010&pages=2959-2973&author=J.+Liangauthor=Y.+Saadauthor=T.+Leiauthor=J.+Wangauthor=D.+Qiauthor=Q.+Yangauthor=P.+E.+Kolattukudyauthor=M.+Fu&title=MCP-induced+protein+1+deubiquitinates+TRAF+proteins+and+negatively+regulates+JNK+and+NF-kappaB+signaling&doi=10.1084%2Fjem.20092641"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1084%2Fjem.20092641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20092641%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DSaad%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKolattukudy%26aufirst%3DP.%2BE.%26aulast%3DFu%26aufirst%3DM.%26atitle%3DMCP-induced%2520protein%25201%2520deubiquitinates%2520TRAF%2520proteins%2520and%2520negatively%2520regulates%2520JNK%2520and%2520NF-kappaB%2520signaling%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2010%26volume%3D207%26spage%3D2959%26epage%3D2973%26doi%3D10.1084%2Fjem.20092641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Fraile, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quesada, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freije, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Otin, C.</span><span> </span><span class="NLM_article-title">Deubiquitinases in cancer: new functions and therapeutic options</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">2373</span><span class="NLM_x">–</span> <span class="NLM_lpage">2388</span><span class="refDoi"> DOI: 10.1038/onc.2011.443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1038%2Fonc.2011.443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=21996736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVCjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=2373-2388&author=J.+M.+Fraileauthor=V.+Quesadaauthor=D.+Rodriguezauthor=J.+M.+Freijeauthor=C.+Lopez-Otin&title=Deubiquitinases+in+cancer%3A+new+functions+and+therapeutic+options&doi=10.1038%2Fonc.2011.443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Deubiquitinases in cancer: new functions and therapeutic options</span></div><div class="casAuthors">Fraile, J. M.; Quesada, V.; Rodriguez, D.; Freije, J. M. P.; Lopez-Otin, C.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2373-2388</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deubiquitinases (DUBs) have fundamental roles in the ubiquitin system through their ability to specifically deconjugate ubiquitin from targeted proteins.  The human genome encodes at least 98 DUBs, which can be grouped into 6 families, reflecting the need for specificity in their function.  The activity of these enzymes affects the turnover rate, activation, recycling and localization of multiple proteins, which in turn is essential for cell homeostasis, protein stability and a wide range of signaling pathways.  Consistent with this, altered DUB function has been related to several diseases, including cancer.  Thus, multiple DUBs have been classified as oncogenes or tumor suppressors because of their regulatory functions on the activity of other proteins involved in tumor development.  Therefore, recent studies have focused on pharmacol. intervention on DUB activity as a rationale to search for novel anticancer drugs.  This strategy may benefit from our current knowledge of the physiol. regulatory mechanisms of these enzymes and the fact that growth of several tumors depends on the normal activity of certain DUBs.  Further understanding of these processes may provide answers to multiple remaining questions on DUB functions and lead to the development of DUB-targeting strategies to expand the repertoire of mol. therapies against cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ3Fzatsb-GrVg90H21EOLACvtfcHk0lgbrZAJXzNOdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVCjsro%253D&md5=e31609c839729ac70591c615e06eaf44</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fonc.2011.443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2011.443%26sid%3Dliteratum%253Aachs%26aulast%3DFraile%26aufirst%3DJ.%2BM.%26aulast%3DQuesada%26aufirst%3DV.%26aulast%3DRodriguez%26aufirst%3DD.%26aulast%3DFreije%26aufirst%3DJ.%2BM.%26aulast%3DLopez-Otin%26aufirst%3DC.%26atitle%3DDeubiquitinases%2520in%2520cancer%253A%2520new%2520functions%2520and%2520therapeutic%2520options%26jtitle%3DOncogene%26date%3D2012%26volume%3D31%26spage%3D2373%26epage%3D2388%26doi%3D10.1038%2Fonc.2011.443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Abdul Rehman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristariyanto, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nkosi, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weidlich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labib, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulathu, Y.</span><span> </span><span class="NLM_article-title">MINDY-1 is a member of an evolutionarily conserved and structurally distinct new family of deubiquitinating enzymes</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span><span class="refDoi"> DOI: 10.1016/j.molcel.2016.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2Fj.molcel.2016.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=27292798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2016&pages=146-155&author=S.+A.+Abdul+Rehmanauthor=Y.+A.+Kristariyantoauthor=S.+Y.+Choiauthor=P.+J.+Nkosiauthor=S.+Weidlichauthor=K.+Labibauthor=K.+Hofmannauthor=Y.+Kulathu&title=MINDY-1+is+a+member+of+an+evolutionarily+conserved+and+structurally+distinct+new+family+of+deubiquitinating+enzymes&doi=10.1016%2Fj.molcel.2016.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">MINDY-1 Is a Member of an Evolutionarily Conserved and Structurally Distinct New Family of Deubiquitinating Enzymes</span></div><div class="casAuthors">Abdul Rehman, Syed Arif; Kristariyanto, Yosua Adi; Choi, Soo-Youn; Nkosi, Pedro, Jr.; Weidlich, Simone; Labib, Karim; Hofmann, Kay; Kulathu, Yogesh</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">146-155</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Deubiquitinating enzymes (DUBs) remove ubiquitin (Ub) from Ub-conjugated substrates to regulate the functional outcome of ubiquitylation.  Here we report the discovery of a new family of DUBs, which we have named MINDY (motif interacting with Ub-contg. novel DUB family).  Found in all eukaryotes, MINDY-family DUBs are highly selective at cleaving K48-linked polyUb, a signal that targets proteins for degrdn.  We identify the catalytic activity to be encoded within a previously unannotated domain, the crystal structure of which reveals a distinct protein fold with no homol. to any of the known DUBs.  The crystal structure of MINDY-1 (also known as FAM63A) in complex with propargylated Ub reveals conformational changes that realign the active site for catalysis.  MINDY-1 prefers cleaving long polyUb chains and works by trimming chains from the distal end.  Collectively, our results reveal a new family of DUBs that may have specialized roles in regulating proteostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPJ1o32zd56LVg90H21EOLACvtfcHk0ljgyCHU8ZPKrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWltL0%253D&md5=80c78d5cd1d2d70e0e2c71c7524f5798</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DAbdul%2BRehman%26aufirst%3DS.%2BA.%26aulast%3DKristariyanto%26aufirst%3DY.%2BA.%26aulast%3DChoi%26aufirst%3DS.%2BY.%26aulast%3DNkosi%26aufirst%3DP.%2BJ.%26aulast%3DWeidlich%26aufirst%3DS.%26aulast%3DLabib%26aufirst%3DK.%26aulast%3DHofmann%26aufirst%3DK.%26aulast%3DKulathu%26aufirst%3DY.%26atitle%3DMINDY-1%2520is%2520a%2520member%2520of%2520an%2520evolutionarily%2520conserved%2520and%2520structurally%2520distinct%2520new%2520family%2520of%2520deubiquitinating%2520enzymes%26jtitle%3DMol.%2520Cell%26date%3D2016%26volume%3D63%26spage%3D146%26epage%3D155%26doi%3D10.1016%2Fj.molcel.2016.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Daviet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colland, F.</span><span> </span><span class="NLM_article-title">Targeting ubiquitin specific proteases for drug discovery</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">270</span><span class="NLM_x">–</span> <span class="NLM_lpage">283</span><span class="refDoi"> DOI: 10.1016/j.biochi.2007.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2Fj.biochi.2007.09.013" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2008&pages=270-283&author=L.+Davietauthor=F.+Colland&title=Targeting+ubiquitin+specific+proteases+for+drug+discovery&doi=10.1016%2Fj.biochi.2007.09.013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2007.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2007.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DDaviet%26aufirst%3DL.%26aulast%3DColland%26aufirst%3DF.%26atitle%3DTargeting%2520ubiquitin%2520specific%2520proteases%2520for%2520drug%2520discovery%26jtitle%3DBiochimie%26date%3D2008%26volume%3D90%26spage%3D270%26epage%3D283%26doi%3D10.1016%2Fj.biochi.2007.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Lim, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baek, K. H.</span><span> </span><span class="NLM_article-title">Deubiquitinating enzymes as therapeutic targets in cancer</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4039</span><span class="NLM_x">–</span> <span class="NLM_lpage">4052</span><span class="refDoi"> DOI: 10.2174/1381612811319220013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.2174%2F1381612811319220013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=23181570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpslyrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=4039-4052&author=K.+H.+Limauthor=K.+H.+Baek&title=Deubiquitinating+enzymes+as+therapeutic+targets+in+cancer&doi=10.2174%2F1381612811319220013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Deubiquitinating enzymes as therapeutic targets in cancer</span></div><div class="casAuthors">Lim, Key-Hwan; Baek, Kwang-Hyun</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4039-4052</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Abnormal regulation of the ubiquitin-proteasome system (UPS) has been known to be involved in the pathogenesis of a variety of human diseases.  A no. of studies have focused on the identification of small modifiers for the UPS.  Even though the proteasome inhibitor Bortezomib (Velcade) has been approved for the therapy of multiple myeloma and mantle cell lymphoma, there are still no DUB inhibitors endorsed for clin. usage.  Since deubiquitinating enzymes (DUBs) are becoming as a new class of modifiers in the UPS, potential drugs that target specific DUBs have been investigated with the development of exptl. technologies for screening small inhibitor mols.  However, the mol. mechanisms of these mols. are poorly understood.  In order to design and develop specific small inhibitor mols. for specific DUBs, identification of specific substrates and mol. structures for each DUB is required.  Here, we review structures, substrates, and small inhibitor mols. of DUBs identified up to date, providing a clear rationale for the development of novel small inhibitor mols. of DUBs for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo48q6oLy2uCrVg90H21EOLACvtfcHk0ljgyCHU8ZPKrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpslyrtro%253D&md5=966b3bb7ccb80078050edd455aa825c2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F1381612811319220013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319220013%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DK.%2BH.%26aulast%3DBaek%26aufirst%3DK.%2BH.%26atitle%3DDeubiquitinating%2520enzymes%2520as%2520therapeutic%2520targets%2520in%2520cancer%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D4039%26epage%3D4052%26doi%3D10.2174%2F1381612811319220013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Becker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchenko, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palacios, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moll, U. M.</span><span> </span><span class="NLM_article-title">A role of HAUSP in tumor suppression in a human colon carcinoma xenograft model</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1205</span><span class="NLM_x">–</span> <span class="NLM_lpage">1213</span><span class="refDoi"> DOI: 10.4161/cc.7.9.5756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.4161%2Fcc.7.9.5756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=18418047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosVKrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=1205-1213&author=K.+Beckerauthor=N.+D.+Marchenkoauthor=G.+Palaciosauthor=U.+M.+Moll&title=A+role+of+HAUSP+in+tumor+suppression+in+a+human+colon+carcinoma+xenograft+model&doi=10.4161%2Fcc.7.9.5756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A role of HAUSP in tumor suppression in a human colon carcinoma xenograft model</span></div><div class="casAuthors">Becker, Kerstin; Marchenko, Natalia D.; Palacios, Gustavo; Moll, Ute M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1205-1213</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">The protease HAUSP is a crit. component of the p53-Mdm2 pathway and acts as a specific deubiquitinase for both p53 and Mdm2 and thus is important for p53 regulation.  In knock-down and knock-out cellular systems it was obsd. that ablation of HAUSP induces profound stabilization of p53 due to enhanced degrdn. of Mdm2.  Thus, inhibiting HAUSP by small compd. interference has been proposed as a rational therapeutic strategy to activate p53 in p53 wild type tumors.  However, HAUSP-mediated effects in the p53-Mdm2 axis are highly complex and non-linear and to date the role of HAUSP in tumor suppression in vivo remains unexplored.  Here we investigate the effect of HAUSP up and downregulation on cell proliferation, apoptosis and tumor growth in vitro and in a xenograft model in vivo, using an inducible isogenic human colon carcinoma cell system.  Importantly, in the absence of stress, both HAUSP up and downregulation inhibit cell proliferation in vitro and tumor growth in vivo due to constitutively elevated p53 levels.  Moreover, tumors with HAUSP up and downregulation respond to radiotherapy with further growth inhibition.  However, HAUSP downregulation causes resistance to Camptothecin- and irradn.-induced apoptosis, which correlates with suppressed mitochondrial translocation of p53.  Our data suggest that changes in HAUSP modulate tumor growth and apoptotic sensitivity in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYvhNCVZmgYLVg90H21EOLACvtfcHk0ljgyCHU8ZPKrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosVKrsL0%253D&md5=1e3c790603e2f80fb73d44710450d0ec</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.4161%2Fcc.7.9.5756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.7.9.5756%26sid%3Dliteratum%253Aachs%26aulast%3DBecker%26aufirst%3DK.%26aulast%3DMarchenko%26aufirst%3DN.%2BD.%26aulast%3DPalacios%26aufirst%3DG.%26aulast%3DMoll%26aufirst%3DU.%2BM.%26atitle%3DA%2520role%2520of%2520HAUSP%2520in%2520tumor%2520suppression%2520in%2520a%2520human%2520colon%2520carcinoma%2520xenograft%2520model%26jtitle%3DCell%2520Cycle%26date%3D2008%26volume%3D7%26spage%3D1205%26epage%3D1213%26doi%3D10.4161%2Fcc.7.9.5756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Levine, A. J.</span><span> </span><span class="NLM_article-title">p53, the cellular gatekeeper for growth and division</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">323</span><span class="NLM_x">–</span> <span class="NLM_lpage">331</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)81871-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2FS0092-8674%2800%2981871-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=9039259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK2sXhtFahtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1997&pages=323-331&author=A.+J.+Levine&title=p53%2C+the+cellular+gatekeeper+for+growth+and+division&doi=10.1016%2FS0092-8674%2800%2981871-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">p53, The cellular gatekeeper for growth and division</span></div><div class="casAuthors">Levine, Arnold J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">323-331</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review, with 55 refs., of p53 transcription factor protein structure, function, and roles in cell cycle regulation, apoptosis, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou4CgTSWvZVbVg90H21EOLACvtfcHk0lgR7-v48ySlng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhtFahtLg%253D&md5=bfbec429d10b9718fe9e9ea5dce9109b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981871-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981871-1%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DA.%2BJ.%26atitle%3Dp53%252C%2520the%2520cellular%2520gatekeeper%2520for%2520growth%2520and%2520division%26jtitle%3DCell%26date%3D1997%26volume%3D88%26spage%3D323%26epage%3D331%26doi%3D10.1016%2FS0092-8674%2800%2981871-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Hollstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sidransky, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogelstein, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, C. C.</span><span> </span><span class="NLM_article-title">p53 mutations in human cancers</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span><span class="refDoi"> DOI: 10.1126/science.1905840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1126%2Fscience.1905840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1905840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK3MXks1Gmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1991&pages=49-53&author=M.+Hollsteinauthor=D.+Sidranskyauthor=B.+Vogelsteinauthor=C.+C.+Harris&title=p53+mutations+in+human+cancers&doi=10.1126%2Fscience.1905840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">p53 Mutations in human cancers</span></div><div class="casAuthors">Hollstein, Monica; Sidransky, David; Vogelstein, Bert; Harris, Curtis C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">5015</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 95 refs.  Mutations in the evolutionarily conserved codons of the p53 tumor suppressor gene are common in diverse types of human cancer.  The p53 mutational spectrum differs among cancers of the colon, lung, esophagus, breast, liver, brain, reticuloendothelial tissues, and hemopoietic tissues.  Anal. of these mutations can provide clues to the etiol. of these diverse tumors and to the function of specific regions of p53.  Transitions predominate in colon, brain, and lymphoid malignancies, whereas G:C to T:A transversions are the most frequent substitutions obsd. in cancers of the lung and liver.  Mutations at A:T base pairs are seen more frequently in esophageal carcinomas than in other solid tumors.  Most transitions in colorectal carcinomas, brain tumors, leukemias, and lymphomas are at CpG dinucleotide mutational hot spots.  G to T transversions in lung, breast, and esophageal carcinomas are dispersed among numerous codons.  In liver tumors in persons from geog. areas in which both aflatoxin B1 and hepatitis B virus are cancer risk factors, most mutations are at one nucleotide pair of codon 249.  These differences may reflect the etiol. contributions of both exogenous and endogenous factors to human carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXuEpuioXy-LVg90H21EOLACvtfcHk0lgR7-v48ySlng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXks1Gmurg%253D&md5=4ff01dc67a6511e7c637d9824174f518</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.1905840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1905840%26sid%3Dliteratum%253Aachs%26aulast%3DHollstein%26aufirst%3DM.%26aulast%3DSidransky%26aufirst%3DD.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26atitle%3Dp53%2520mutations%2520in%2520human%2520cancers%26jtitle%3DScience%26date%3D1991%26volume%3D253%26spage%3D49%26epage%3D53%26doi%3D10.1126%2Fscience.1905840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Brown, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fersht, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, D. P.</span><span> </span><span class="NLM_article-title">Awakening guardian angels: drugging the p53 pathway</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">862</span><span class="NLM_x">–</span> <span class="NLM_lpage">873</span><span class="refDoi"> DOI: 10.1038/nrc2763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1038%2Fnrc2763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=19935675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=862-873&author=C.+J.+Brownauthor=S.+Lainauthor=C.+S.+Vermaauthor=A.+R.+Fershtauthor=D.+P.+Lane&title=Awakening+guardian+angels%3A+drugging+the+p53+pathway&doi=10.1038%2Fnrc2763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Awakening guardian angels: Drugging the p53 pathway</span></div><div class="casAuthors">Brown, Christopher J.; Lain, Sonia; Verma, Chandra S.; Fersht, Alan R.; Lane, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">862-873</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Currently, around 11 million people are living with a tumor that contains an inactivating mutation of TP53 (the human gene that encodes p53) and another 11 million have tumors in which the p53 pathway is partially abrogated through the inactivation of other signaling or effector components.  The p53 pathway is therefore a prime target for new cancer drug development, and several original approaches to drug discovery that could have wide applications to drug development are being used.  In one approach, mols. that activate p53 by blocking protein-protein interactions with MDM2 are in early clin. development.  Remarkable progress has also been made in the development of p53-binding mols. that can rescue the function of certain p53 mutants.  Finally, cell-based assays are being used to discover compds. that exploit the p53 pathway by either seeking targets and compds. that show synthetic lethality with TP53 mutations or by looking for non-genotoxic activators of the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZW-elIe4w6bVg90H21EOLACvtfcHk0lgR7-v48ySlng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM&md5=072f48d2ed466b2deb9d2b999cc65a53</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc2763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2763%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DC.%2BJ.%26aulast%3DLain%26aufirst%3DS.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26aulast%3DLane%26aufirst%3DD.%2BP.%26atitle%3DAwakening%2520guardian%2520angels%253A%2520drugging%2520the%2520p53%2520pathway%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D862%26epage%3D873%26doi%3D10.1038%2Fnrc2763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Tovar, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosinski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filipovic, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolinsky, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilton, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qing, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myklebost, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heimbrook, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassilev, L. T.</span><span> </span><span class="NLM_article-title">Small-molecule MDM2 antagonists reveal aberrant p53 signaling in cancer: implications for therapy</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">1888</span><span class="NLM_x">–</span> <span class="NLM_lpage">1893</span><span class="refDoi"> DOI: 10.1073/pnas.0507493103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1073%2Fpnas.0507493103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=1888-1893&author=C.+Tovarauthor=J.+Rosinskiauthor=Z.+Filipovicauthor=B.+Higginsauthor=K.+Kolinskyauthor=H.+Hiltonauthor=X.+Zhaoauthor=B.+T.+Vuauthor=W.+Qingauthor=K.+Packmanauthor=O.+Myklebostauthor=D.+C.+Heimbrookauthor=L.+T.+Vassilev&title=Small-molecule+MDM2+antagonists+reveal+aberrant+p53+signaling+in+cancer%3A+implications+for+therapy&doi=10.1073%2Fpnas.0507493103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0507493103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0507493103%26sid%3Dliteratum%253Aachs%26aulast%3DTovar%26aufirst%3DC.%26aulast%3DRosinski%26aufirst%3DJ.%26aulast%3DFilipovic%26aufirst%3DZ.%26aulast%3DHiggins%26aufirst%3DB.%26aulast%3DKolinsky%26aufirst%3DK.%26aulast%3DHilton%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DVu%26aufirst%3DB.%2BT.%26aulast%3DQing%26aufirst%3DW.%26aulast%3DPackman%26aufirst%3DK.%26aulast%3DMyklebost%26aufirst%3DO.%26aulast%3DHeimbrook%26aufirst%3DD.%2BC.%26aulast%3DVassilev%26aufirst%3DL.%2BT.%26atitle%3DSmall-molecule%2520MDM2%2520antagonists%2520reveal%2520aberrant%2520p53%2520signaling%2520in%2520cancer%253A%2520implications%2520for%2520therapy%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D1888%26epage%3D1893%26doi%3D10.1073%2Fpnas.0507493103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Ammirati, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagley, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhattacharya, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckbinder, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlo, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conrad, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dow, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Kattan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guimaraes, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaPerle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Londregan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loria, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathiowetz, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orr, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skoura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">Discovery of an in vivo tool to establish proof-of-concept for MAP4K4-based antidiabetic treatment</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1128</span><span class="NLM_x">–</span> <span class="NLM_lpage">1133</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1128-1133&author=M.+Ammiratiauthor=S.+W.+Bagleyauthor=S.+K.+Bhattacharyaauthor=L.+Buckbinderauthor=A.+A.+Carloauthor=R.+Conradauthor=C.+Cortesauthor=R.+L.+Dowauthor=M.+S.+Dowlingauthor=A.+El-Kattanauthor=K.+Fordauthor=C.+R.+Guimaraesauthor=D.+Hepworthauthor=W.+Jiaoauthor=J.+LaPerleauthor=S.+Liuauthor=A.+Londreganauthor=P.+M.+Loriaauthor=A.+M.+Mathiowetzauthor=M.+Munchhofauthor=S.+T.+Orrauthor=D.+N.+Petersenauthor=D.+A.+Priceauthor=A.+Skouraauthor=A.+C.+Smithauthor=J.+Wang&title=Discovery+of+an+in+vivo+tool+to+establish+proof-of-concept+for+MAP4K4-based+antidiabetic+treatment&doi=10.1021%2Facsmedchemlett.5b00215"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00215%26sid%3Dliteratum%253Aachs%26aulast%3DAmmirati%26aufirst%3DM.%26aulast%3DBagley%26aufirst%3DS.%2BW.%26aulast%3DBhattacharya%26aufirst%3DS.%2BK.%26aulast%3DBuckbinder%26aufirst%3DL.%26aulast%3DCarlo%26aufirst%3DA.%2BA.%26aulast%3DConrad%26aufirst%3DR.%26aulast%3DCortes%26aufirst%3DC.%26aulast%3DDow%26aufirst%3DR.%2BL.%26aulast%3DDowling%26aufirst%3DM.%2BS.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DFord%26aufirst%3DK.%26aulast%3DGuimaraes%26aufirst%3DC.%2BR.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DJiao%26aufirst%3DW.%26aulast%3DLaPerle%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLondregan%26aufirst%3DA.%26aulast%3DLoria%26aufirst%3DP.%2BM.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DMunchhof%26aufirst%3DM.%26aulast%3DOrr%26aufirst%3DS.%2BT.%26aulast%3DPetersen%26aufirst%3DD.%2BN.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DSkoura%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DA.%2BC.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520vivo%2520tool%2520to%2520establish%2520proof-of-concept%2520for%2520MAP4K4-based%2520antidiabetic%2520treatment%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D1128%26epage%3D1133%26doi%3D10.1021%2Facsmedchemlett.5b00215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Ohki, H.; Ota, M.; Shibata, Y.; Watanabe, H.; Motoki, R.; Tominaga, Y.; Jimbo, T.</span><span> </span><span class="NLM_article-title">Preparation of bicyclic pyrimidine-carboxamide compounds as inhibitors of receptor tyrosine kinase Axl</span>. WO 2013162061 A1,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=H.+Ohki&author=M.+Ota&author=Y.+Shibata&author=H.+Watanabe&author=R.+Motoki&author=Y.+Tominaga&author=T.+Jimbo&title=Preparation+of+bicyclic+pyrimidine-carboxamide+compounds+as+inhibitors+of+receptor+tyrosine+kinase+Axl"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOhki%26aufirst%3DH.%26atitle%3DPreparation%2520of%2520bicyclic%2520pyrimidine-carboxamide%2520compounds%2520as%2520inhibitors%2520of%2520receptor%2520tyrosine%2520kinase%2520Axl%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Lau, W. C.</span><span> </span><span class="NLM_article-title">Novel Fyn kinase inhibitors for therapeutic applications</span>. WO 2017044623 A1,<span class="NLM_x"> </span><span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=W.+C.+Lau&title=Novel+Fyn+kinase+inhibitors+for+therapeutic+applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DW.%2BC.%26atitle%3DNovel%2520Fyn%2520kinase%2520inhibitors%2520for%2520therapeutic%2520applications%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Ndubaku, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span> </span><span class="NLM_article-title">Inhibiting the deubiquitinating enzymes (DUBs)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1581</span><span class="NLM_x">–</span> <span class="NLM_lpage">1595</span><span class="refDoi"> DOI: 10.1021/jm501061a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501061a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGjtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1581-1595&author=C.+Ndubakuauthor=V.+Tsui&title=Inhibiting+the+deubiquitinating+enzymes+%28DUBs%29&doi=10.1021%2Fjm501061a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting the Deubiquitinating Enzymes (DUBs)</span></div><div class="casAuthors">Ndubaku, Chudi; Tsui, Vickie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1581-1595</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The diverse roles of deubiquitinating enzymes, or DUBs, in detg. the fate of specific proteins continue to unfold.  Concurrent with the revelation of DUBs as potential therapeutic targets are publications of small mol. inhibitors of these enzymes.  In this review, we summarize these mols. and their assocd. data and suggest addnl. expts. to further validate and characterize these compds.  We believe the field of drug discovery against DUBs is still in its infancy, but advances in assay development, biophys. techniques, and screening libraries hold promise for identifying suitable agents that could advance into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsiPtPfBsxNrVg90H21EOLACvtfcHk0lj0ZKT9M8Mfzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGjtLrL&md5=8ec54b033ba6fd8acb3187e4fc004296</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm501061a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501061a%26sid%3Dliteratum%253Aachs%26aulast%3DNdubaku%26aufirst%3DC.%26aulast%3DTsui%26aufirst%3DV.%26atitle%3DInhibiting%2520the%2520deubiquitinating%2520enzymes%2520%2528DUBs%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1581%26epage%3D1595%26doi%3D10.1021%2Fjm501061a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Feng, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergeron, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradley, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koehler, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortwine, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobbi, A.</span><span> </span><span class="NLM_article-title">An integrated suite of modeling tools that empower scientists in structure- and property-based drug design</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">523</span><span class="refDoi"> DOI: 10.1007/s10822-015-9845-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1007%2Fs10822-015-9845-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=511-523&author=J.+A.+Fengauthor=I.+Aliagasauthor=P.+Bergeronauthor=J.+M.+Blaneyauthor=E.+K.+Bradleyauthor=M.+F.+Koehlerauthor=M.+L.+Leeauthor=D.+F.+Ortwineauthor=V.+Tsuiauthor=J.+Wuauthor=A.+Gobbi&title=An+integrated+suite+of+modeling+tools+that+empower+scientists+in+structure-+and+property-based+drug+design&doi=10.1007%2Fs10822-015-9845-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9845-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9845-4%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DJ.%2BA.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DBergeron%26aufirst%3DP.%26aulast%3DBlaney%26aufirst%3DJ.%2BM.%26aulast%3DBradley%26aufirst%3DE.%2BK.%26aulast%3DKoehler%26aufirst%3DM.%2BF.%26aulast%3DLee%26aufirst%3DM.%2BL.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGobbi%26aufirst%3DA.%26atitle%3DAn%2520integrated%2520suite%2520of%2520modeling%2520tools%2520that%2520empower%2520scientists%2520in%2520structure-%2520and%2520property-based%2520drug%2520design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D511%26epage%3D523%26doi%3D10.1007%2Fs10822-015-9845-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Rougé, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bainbridge, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwok, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Lello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wertz, I. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span> </span><span class="NLM_article-title">Molecular understanding of USP7 substrate recognition and C-terminal activation</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1335</span><span class="NLM_x">–</span> <span class="NLM_lpage">1345</span><span class="refDoi"> DOI: 10.1016/j.str.2016.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2Fj.str.2016.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=27452404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1eju7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1335-1345&author=L.+Roug%C3%A9author=T.+W.+Bainbridgeauthor=M.+Kwokauthor=R.+Tongauthor=P.+Di+Lelloauthor=I.+E.+Wertzauthor=T.+Maurerauthor=J.+A.+Ernstauthor=J.+Murray&title=Molecular+understanding+of+USP7+substrate+recognition+and+C-terminal+activation&doi=10.1016%2Fj.str.2016.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Understanding of USP7 Substrate Recognition and C-Terminal Activation</span></div><div class="casAuthors">Rouge, Lionel; Bainbridge, Travis W.; Kwok, Michael; Tong, Raymond; Di Lello, Paola; Wertz, Ingrid E.; Maurer, Till; Ernst, James A.; Murray, Jeremy</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1335-1345</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The deubiquitinating enzyme USP7 has a pivotal role in regulating the stability of proteins involved in fundamental cellular processes of normal biol. and disease.  Despite the importance of USP7, the mechanisms underlying substrate recognition and catalytic activation are poorly understood.  Here we present structural, biochem., and biophys. analyses elucidating the mol. mechanism by which the C-terminal 19 amino acids of USP7 (residues 1084-1102) enhance the ubiquitin cleavage activity of the deubiquitinase (DUB) domain.  Our data demonstrate that the C-terminal peptide binds the activation cleft in the catalytic domain and stabilizes the catalytically competent conformation of USP7.  Addnl. structures of longer fragments of USP7, as well as soln. studies, provide insight into full-length USP7, the role of the UBL domains, and demonstrate that both substrate recognition and deubiquitinase activity are highly regulated by the catalytic and noncatalytic domains of USP7, a feature that could be essential for the proper function of multi-domain DUBs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1bahSKk_MO7Vg90H21EOLACvtfcHk0li9Ov0gt6jucw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1eju7fL&md5=8cc4de7dc38c480a1c0b1cc7976e7beb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DRoug%25C3%25A9%26aufirst%3DL.%26aulast%3DBainbridge%26aufirst%3DT.%2BW.%26aulast%3DKwok%26aufirst%3DM.%26aulast%3DTong%26aufirst%3DR.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DErnst%26aufirst%3DJ.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26atitle%3DMolecular%2520understanding%2520of%2520USP7%2520substrate%2520recognition%2520and%2520C-terminal%2520activation%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D1335%26epage%3D1345%26doi%3D10.1016%2Fj.str.2016.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Di Lello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougé, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, T.</span><span> </span><span class="NLM_article-title">(1)H, (13)C and (15)N backbone resonance assignment for the 40.5 kDa catalytic domain of Ubiquitin Specific Protease 7 (USP7)</span> <span class="citation_source-journal">Biomol. NMR Assignments</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">345</span><span class="NLM_x">–</span> <span class="NLM_lpage">349</span><span class="refDoi"> DOI: 10.1007/s12104-016-9698-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1007%2Fs12104-016-9698-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=27386854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFertLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2016&pages=345-349&author=P.+Di+Lelloauthor=L.+Roug%C3%A9author=B.+Panauthor=T.+Maurer&title=%281%29H%2C+%2813%29C+and+%2815%29N+backbone+resonance+assignment+for+the+40.5+kDa+catalytic+domain+of+Ubiquitin+Specific+Protease+7+%28USP7%29&doi=10.1007%2Fs12104-016-9698-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">1H, 13C and 15N backbone resonance assignment for the 40.5 kDa catalytic domain of Ubiquitin Specific Protease 7 (USP7)</span></div><div class="casAuthors">Di Lello, Paola; Rouge, Lionel; Pan, Borlan; Maurer, Till</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecular NMR Assignments</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">345-349</span>CODEN:
                <span class="NLM_cas:coden">BNAIBR</span>;
        ISSN:<span class="NLM_cas:issn">1874-270X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The deubiquitinase Ubiquitin Specific Protease 7 (USP7) is part of the regulatory cascade of proteins that modulates the activity of the tumor suppressor protein p53.  Deubiquitination of its target Murine Double Minute 2 (MDM2) leads to increased proteosomal degrdn. of p53.  Consequently, USP7 has emerged as an attractive oncol. target because its inhibition stabilizes p53, thereby promoting p53-dependent apoptosis in cancer cells.  Here we report the backbone resonance assignment for the 40.5 kDa catalytic domain of USP7.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY63PhaaTcnLVg90H21EOLACvtfcHk0li9Ov0gt6jucw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFertLvN&md5=158aee4ee62aa8c48d3b979c2ff0eb04</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs12104-016-9698-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12104-016-9698-3%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DRoug%25C3%25A9%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DMaurer%26aufirst%3DT.%26atitle%3D%25281%2529H%252C%2520%252813%2529C%2520and%2520%252815%2529N%2520backbone%2520resonance%2520assignment%2520for%2520the%252040.5%2520kDa%2520catalytic%2520domain%2520of%2520Ubiquitin%2520Specific%2520Protease%25207%2520%2528USP7%2529%26jtitle%3DBiomol.%2520NMR%2520Assignments%26date%3D2016%26volume%3D10%26spage%3D345%26epage%3D349%26doi%3D10.1007%2Fs12104-016-9698-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Mayer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, B.</span><span> </span><span class="NLM_article-title">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1784</span><span class="NLM_x">–</span> <span class="NLM_lpage">1788</span><span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0li9Ov0gt6jucw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Pervushin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riek, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wuthrich, K.</span><span> </span><span class="NLM_article-title">Attenuated T2 relaxation by mutual cancellation of dipole–dipole coupling and chemical shift anisotropy indicates an avenue to NMR structures of very large biological macromolecules in solution</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">12366</span><span class="NLM_x">–</span> <span class="NLM_lpage">12371</span><span class="refDoi"> DOI: 10.1073/pnas.94.23.12366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1073%2Fpnas.94.23.12366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=9356455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A280%3ADyaK1c%252FisVWgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=12366-12371&author=K.+Pervushinauthor=R.+Riekauthor=G.+Widerauthor=K.+Wuthrich&title=Attenuated+T2+relaxation+by+mutual+cancellation+of+dipole%E2%80%93dipole+coupling+and+chemical+shift+anisotropy+indicates+an+avenue+to+NMR+structures+of+very+large+biological+macromolecules+in+solution&doi=10.1073%2Fpnas.94.23.12366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Attenuated T2 relaxation by mutual cancellation of dipole-dipole coupling and chemical shift anisotropy indicates an avenue to NMR structures of very large biological macromolecules in solution</span></div><div class="casAuthors">Pervushin K; Riek R; Wider G; Wuthrich K</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">12366-71</span>
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Fast transverse relaxation of 1H, 15N, and 13C by dipole-dipole coupling (DD) and chemical shift anisotropy (CSA) modulated by rotational molecular motions has a dominant impact on the size limit for biomacromolecular structures that can be studied by NMR spectroscopy in solution.  Transverse relaxation-optimized spectroscopy (TROSY) is an approach for suppression of transverse relaxation in multidimensional NMR experiments, which is based on constructive use of interference between DD coupling and CSA.  For example, a TROSY-type two-dimensional 1H,15N-correlation experiment with a uniformly 15N-labeled protein in a DNA complex of molecular mass 17 kDa at a 1H frequency of 750 MHz showed that 15N relaxation during 15N chemical shift evolution and 1HN relaxation during signal acquisition both are significantly reduced by mutual compensation of the DD and CSA interactions.  The reduction of the linewidths when compared with a conventional two-dimensional 1H,15N-correlation experiment was 60% and 40%, respectively, and the residual linewidths were 5 Hz for 15N and 15 Hz for 1HN at 4 degrees C.  Because the ratio of the DD and CSA relaxation rates is nearly independent of the molecular size, a similar percentagewise reduction of the overall transverse relaxation rates is expected for larger proteins.  For a 15N-labeled protein of 150 kDa at 750 MHz and 20 degrees C one predicts residual linewidths of 10 Hz for 15N and 45 Hz for 1HN, and for the corresponding uniformly 15N,2H-labeled protein the residual linewidths are predicted to be smaller than 5 Hz and 15 Hz, respectively.  The TROSY principle should benefit a variety of multidimensional solution NMR experiments, especially with future use of yet somewhat higher polarizing magnetic fields than are presently available, and thus largely eliminate one of the key factors that limit work with larger molecules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSt8cd29R1zO-3MPKNEWB5RfW6udTcc2eaxGqrsoNtdJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1c%252FisVWgtQ%253D%253D&md5=ab6a5c6af04d914882d7977e67bf022b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.23.12366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.23.12366%26sid%3Dliteratum%253Aachs%26aulast%3DPervushin%26aufirst%3DK.%26aulast%3DRiek%26aufirst%3DR.%26aulast%3DWider%26aufirst%3DG.%26aulast%3DWuthrich%26aufirst%3DK.%26atitle%3DAttenuated%2520T2%2520relaxation%2520by%2520mutual%2520cancellation%2520of%2520dipole%25E2%2580%2593dipole%2520coupling%2520and%2520chemical%2520shift%2520anisotropy%2520indicates%2520an%2520avenue%2520to%2520NMR%2520structures%2520of%2520very%2520large%2520biological%2520macromolecules%2520in%2520solution%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1997%26volume%3D94%26spage%3D12366%26epage%3D12371%26doi%3D10.1073%2Fpnas.94.23.12366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Delaglio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grzesiek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuister, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeifer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bax, A.</span><span> </span><span class="NLM_article-title">NMRPipe: A multidimensional spectral processing system based on UNIX pipes</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">293</span><span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+A+multidimensional+spectral+processing+system+based+on+UNIX+pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0liBKY4ZoYMOlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520A%2520multidimensional%2520spectral%2520processing%2520system%2520based%2520on%2520UNIX%2520pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Johnson, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blevins, R. A.</span><span> </span><span class="NLM_article-title">NMRView: A computer program for the visualization and analysis of NMR data</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">603</span><span class="NLM_x">–</span> <span class="NLM_lpage">614</span><span class="refDoi"> DOI: 10.1007/BF00404272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1007%2FBF00404272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=22911360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK2cXmt1Gkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1994&pages=603-614&author=B.+A.+Johnsonauthor=R.+A.+Blevins&title=NMRView%3A+A+computer+program+for+the+visualization+and+analysis+of+NMR+data&doi=10.1007%2FBF00404272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">NMRView: a computer program for the visualization and analysis of NMR data</span></div><div class="casAuthors">Johnson, Bruce A.; Blevins, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">603-14</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    </div><div class="casAbstract">NMRView is a computer program designed for the visualization and anal. of NMR data.  It allows the user to interact with a practically unlimited no. of 2D, 3D and 4D NMR data files.  Any no. of spectral windows can be displayed on the screen in any size and location.  Automatic peak picking and facilitated peak anal. features are included to aid in the assignment of complex NMR spectra.  NMR View provides structure anal. features and data transfer to and from structural generation programs, allowing for a tight coupling between the spectral anal. and structural generation.,.  Visual correlation between structures and spectra can be done with the Mol. Data Viewer, a mol. graphics program with bidirectional communication to NMR View.  The used interface can be customized and a command language is provided to allow for the automation of various tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr12Ta6zHzfUrVg90H21EOLACvtfcHk0liBKY4ZoYMOlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmt1Gkurw%253D&md5=6a93a0d9704fbbb654abc192203c68ae</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2FBF00404272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00404272%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DB.%2BA.%26aulast%3DBlevins%26aufirst%3DR.%2BA.%26atitle%3DNMRView%253A%2520A%2520computer%2520program%2520for%2520the%2520visualization%2520and%2520analysis%2520of%2520NMR%2520data%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1994%26volume%3D4%26spage%3D603%26epage%3D614%26doi%3D10.1007%2FBF00404272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 28 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jeffrey N. Levy, Juan V. Alegre-Requena, Renrong Liu, Robert S. Paton, <span class="NLM_string-name hlFld-ContribAuthor">Andrew McNally</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Halogenation of Pyridines Using Designed Phosphine Reagents. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (25)
                                     , 11295-11305. <a href="https://doi.org/10.1021/jacs.0c04674" title="DOI URL">https://doi.org/10.1021/jacs.0c04674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c04674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c04674%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DSelective%252BHalogenation%252Bof%252BPyridines%252BUsing%252BDesigned%252BPhosphine%252BReagents%26aulast%3DLevy%26aufirst%3DJeffrey%2BN.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03052020%26date%3D10062020%26date%3D29052020%26volume%3D142%26issue%3D25%26spage%3D11295%26epage%3D11305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul R. Leger, Dennis X. Hu, Berenger Biannic, Minna Bui, Xinping Han, Emily Karbarz, Jack Maung, Akinori Okano, Maksim Osipov, Grant M. Shibuya, Kyle Young, Christopher Higgs, Betty Abraham, Delia Bradford, Cynthia Cho, Christophe Colas, Scott Jacobson, Yamini M. Ohol, Deepa Pookot, Payal Rana, Jerick Sanchez, Niket Shah, Michael Sun, Steve Wong, Dirk G. Brockstedt, Paul D. Kassner, Jacob B. Schwarz, <span class="NLM_string-name hlFld-ContribAuthor">David J. Wustrow</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent, Selective, and Orally Bioavailable Inhibitors of USP7 with In Vivo Antitumor Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (10)
                                     , 5398-5420. <a href="https://doi.org/10.1021/acs.jmedchem.0c00245" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00245</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00245%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%25252C%252BSelective%25252C%252Band%252BOrally%252BBioavailable%252BInhibitors%252Bof%252BUSP7%252Bwith%252BIn%252BVivo%252BAntitumor%252BActivity%26aulast%3DLeger%26aufirst%3DPaul%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20022020%26date%3D06052020%26date%3D17042020%26volume%3D63%26issue%3D10%26spage%3D5398%26epage%3D5420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Nathan J. Schauer, Robert S. Magin, Xiaoxi Liu, Laura M. Doherty, <span class="NLM_string-name hlFld-ContribAuthor">Sara J. Buhrlage</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in Discovering Deubiquitinating Enzyme (DUB) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (6)
                                     , 2731-2750. <a href="https://doi.org/10.1021/acs.jmedchem.9b01138" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01138</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01138%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAdvances%252Bin%252BDiscovering%252BDeubiquitinating%252BEnzyme%252B%252528DUB%252529%252BInhibitors%26aulast%3DSchauer%26aufirst%3DNathan%2BJ.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D12072019%26date%3D20112019%26date%3D04112019%26volume%3D63%26issue%3D6%26spage%3D2731%26epage%3D2750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul N. Mortenson, Daniel A. Erlanson, Iwan J. P. de Esch, Wolfgang Jahnke, <span class="NLM_string-name hlFld-ContribAuthor">Christopher N. Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2017. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (8)
                                     , 3857-3872. <a href="https://doi.org/10.1021/acs.jmedchem.8b01472" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01472</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01472%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2017%26aulast%3DMortenson%26aufirst%3DPaul%2BN.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D21092018%26date%3D21112018%26volume%3D62%26issue%3D8%26spage%3D3857%26epage%3D3872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Girija  Pawge</span>, <span class="hlFld-ContribAuthor ">Gopal L.  Khatik</span>. </span><span class="cited-content_cbyCitation_article-title">p53 regulated senescence mechanism and role of its modulators in age-related disorders. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2021,</strong> <em>190 </em>, 114651. <a href="https://doi.org/10.1016/j.bcp.2021.114651" title="DOI URL">https://doi.org/10.1016/j.bcp.2021.114651</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2021.114651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2021.114651%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3Dp53%252Bregulated%252Bsenescence%252Bmechanism%252Band%252Brole%252Bof%252Bits%252Bmodulators%252Bin%252Bage-related%252Bdisorders%26aulast%3DPawge%26aufirst%3DGirija%26date%3D2021%26volume%3D190%26spage%3D114651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert S.  Magin</span>, <span class="hlFld-ContribAuthor ">Xiaoxi  Liu</span>, <span class="hlFld-ContribAuthor ">Alejandra  Felix</span>, <span class="hlFld-ContribAuthor ">Ariana S.  Bratt</span>, <span class="hlFld-ContribAuthor ">Wai Cheung  Chan</span>, <span class="hlFld-ContribAuthor ">Sara J.  Buhrlage</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecules as tools for functional assessment of deubiquitinating enzyme function. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 1090-1100. <a href="https://doi.org/10.1016/j.chembiol.2021.04.021" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.04.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.04.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.04.021%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DSmall%252Bmolecules%252Bas%252Btools%252Bfor%252Bfunctional%252Bassessment%252Bof%252Bdeubiquitinating%252Benzyme%252Bfunction%26aulast%3DMagin%26aufirst%3DRobert%2BS.%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D1090%26epage%3D1100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabriel  LaPlante</span>, <span class="hlFld-ContribAuthor ">Wei  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the Ubiquitin-Proteasome System for Cancer Therapeutics by Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (12)
                                     , 3079. <a href="https://doi.org/10.3390/cancers13123079" title="DOI URL">https://doi.org/10.3390/cancers13123079</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13123079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13123079%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DTargeting%252Bthe%252BUbiquitin-Proteasome%252BSystem%252Bfor%252BCancer%252BTherapeutics%252Bby%252BSmall-Molecule%252BInhibitors%26aulast%3DLaPlante%26aufirst%3DGabriel%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D12%26spage%3D3079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shiyao  Chen</span>, <span class="hlFld-ContribAuthor ">Yunqi  Liu</span>, <span class="hlFld-ContribAuthor ">Huchen  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in the Development Ubiquitin-Specific Peptidase (USP) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (9)
                                     , 4546. <a href="https://doi.org/10.3390/ijms22094546" title="DOI URL">https://doi.org/10.3390/ijms22094546</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22094546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22094546%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DAdvances%252Bin%252Bthe%252BDevelopment%252BUbiquitin-Specific%252BPeptidase%252B%252528USP%252529%252BInhibitors%26aulast%3DChen%26aufirst%3DShiyao%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D9%26spage%3D4546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiabin  Lu</span>, <span class="hlFld-ContribAuthor ">He  Zhao</span>, <span class="hlFld-ContribAuthor ">Caini  Yu</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Kang</span>, <span class="hlFld-ContribAuthor ">Xiaochun  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Ubiquitin-Specific Protease 7 (USP7) in Cancer: A New Insight to Overcome Drug Resistance. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fphar.2021.648491" title="DOI URL">https://doi.org/10.3389/fphar.2021.648491</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2021.648491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2021.648491%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DTargeting%252BUbiquitin-Specific%252BProtease%252B7%252B%252528USP7%252529%252Bin%252BCancer%25253A%252BA%252BNew%252BInsight%252Bto%252BOvercome%252BDrug%252BResistance%26aulast%3DLu%26aufirst%3DJiabin%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng  Li</span>, <span class="hlFld-ContribAuthor ">Ying  Liu</span>, <span class="hlFld-ContribAuthor ">Hua  Yang</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, biological evaluation and structure-activity relationship study of quinazolin-4(3H)-one derivatives as novel USP7 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>216 </em>, 113291. <a href="https://doi.org/10.1016/j.ejmech.2021.113291" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113291</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113291%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Bbiological%252Bevaluation%252Band%252Bstructure-activity%252Brelationship%252Bstudy%252Bof%252Bquinazolin-4%2525283H%252529-one%252Bderivatives%252Bas%252Bnovel%252BUSP7%252Binhibitors%26aulast%3DLi%26aufirst%3DPeng%26date%3D2021%26volume%3D216%26spage%3D113291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin  Diethelm‐Varela</span>. </span><span class="cited-content_cbyCitation_article-title">Using NMR Spectroscopy in the Fragment‐Based Drug Discovery of Small‐Molecule Anticancer Targeted Therapies. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (5)
                                     , 725-742. <a href="https://doi.org/10.1002/cmdc.202000756" title="DOI URL">https://doi.org/10.1002/cmdc.202000756</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000756%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DUsing%252BNMR%252BSpectroscopy%252Bin%252Bthe%252BFragment%2525E2%252580%252590Based%252BDrug%252BDiscovery%252Bof%252BSmall%2525E2%252580%252590Molecule%252BAnticancer%252BTargeted%252BTherapies%26aulast%3DDiethelm%25E2%2580%2590Varela%26aufirst%3DBenjamin%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D5%26spage%3D725%26epage%3D742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Max J.  Caplin</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Foley</span>. </span><span class="cited-content_cbyCitation_article-title">Emergent synthetic methods for the modular advancement of sp
              3
              -rich fragments. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>15 </em><a href="https://doi.org/10.1039/D1SC00161B" title="DOI URL">https://doi.org/10.1039/D1SC00161B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1SC00161B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1SC00161B%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DEmergent%252Bsynthetic%252Bmethods%252Bfor%252Bthe%252Bmodular%252Badvancement%252Bof%252Bsp%252B3%252B-rich%252Bfragments%26aulast%3DCaplin%26aufirst%3DMax%2BJ.%26date%3D2021%26date%3D2021%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>. </span><span class="cited-content_cbyCitation_article-title">Many small steps towards a COVID-19 drug. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-020-18710-3" title="DOI URL">https://doi.org/10.1038/s41467-020-18710-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-020-18710-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-020-18710-3%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DMany%252Bsmall%252Bsteps%252Btowards%252Ba%252BCOVID-19%252Bdrug%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olof  Engström</span>, <span class="hlFld-ContribAuthor ">Oscar  Belda</span>, <span class="hlFld-ContribAuthor ">Mari  Kullman-Magnusson</span>, <span class="hlFld-ContribAuthor ">Mikaela  Rapp</span>, <span class="hlFld-ContribAuthor ">Kerstin  Böhm</span>, <span class="hlFld-ContribAuthor ">Ralf  Paul</span>, <span class="hlFld-ContribAuthor ">Ian  Henderson</span>, <span class="hlFld-ContribAuthor ">Dean  Derbyshire</span>, <span class="hlFld-ContribAuthor ">Sofia  Karlström</span>, <span class="hlFld-ContribAuthor ">Kevin E.B.  Parkes</span>, <span class="hlFld-ContribAuthor ">Hongtao  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of USP7 small-molecule allosteric inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (20)
                                     , 127471. <a href="https://doi.org/10.1016/j.bmcl.2020.127471" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127471%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BUSP7%252Bsmall-molecule%252Ballosteric%252Binhibitors%26aulast%3DEngstr%25C3%25B6m%26aufirst%3DOlof%26date%3D2020%26volume%3D30%26issue%3D20%26spage%3D127471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ahood  Al-Eidan</span>, <span class="hlFld-ContribAuthor ">Yihua  Wang</span>, <span class="hlFld-ContribAuthor ">Paul  Skipp</span>, <span class="hlFld-ContribAuthor ">Rob M.  Ewing</span>. </span><span class="cited-content_cbyCitation_article-title">The USP7 protein interaction network and its roles in tumorigenesis. </span><span class="cited-content_cbyCitation_journal-name">Genes & Diseases</span><span> <strong>2020,</strong> <em>1782 </em><a href="https://doi.org/10.1016/j.gendis.2020.10.004" title="DOI URL">https://doi.org/10.1016/j.gendis.2020.10.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.gendis.2020.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.gendis.2020.10.004%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%2520%2526%2520Diseases%26atitle%3DThe%252BUSP7%252Bprotein%252Binteraction%252Bnetwork%252Band%252Bits%252Broles%252Bin%252Btumorigenesis%26aulast%3DAl-Eidan%26aufirst%3DAhood%26date%3D2020%26volume%3D1782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yamini M.  Ohol</span>, <span class="hlFld-ContribAuthor ">Michael T.  Sun</span>, <span class="hlFld-ContribAuthor ">Gene  Cutler</span>, <span class="hlFld-ContribAuthor ">Paul R.  Leger</span>, <span class="hlFld-ContribAuthor ">Dennis X.  Hu</span>, <span class="hlFld-ContribAuthor ">Berenger  Biannic</span>, <span class="hlFld-ContribAuthor ">Payal  Rana</span>, <span class="hlFld-ContribAuthor ">Cynthia  Cho</span>, <span class="hlFld-ContribAuthor ">Scott  Jacobson</span>, <span class="hlFld-ContribAuthor ">Steve T.  Wong</span>, <span class="hlFld-ContribAuthor ">Jerick  Sanchez</span>, <span class="hlFld-ContribAuthor ">Niket  Shah</span>, <span class="hlFld-ContribAuthor ">Deepa  Pookot</span>, <span class="hlFld-ContribAuthor ">Betty  Abraham</span>, <span class="hlFld-ContribAuthor ">Kyle  Young</span>, <span class="hlFld-ContribAuthor ">Silpa  Suthram</span>, <span class="hlFld-ContribAuthor ">Lisa A.  Marshall</span>, <span class="hlFld-ContribAuthor ">Delia  Bradford</span>, <span class="hlFld-ContribAuthor ">Nathan  Kozon</span>, <span class="hlFld-ContribAuthor ">Xinping  Han</span>, <span class="hlFld-ContribAuthor ">Akinori  Okano</span>, <span class="hlFld-ContribAuthor ">Jack  Maung</span>, <span class="hlFld-ContribAuthor ">Christophe  Colas</span>, <span class="hlFld-ContribAuthor ">Jacob  Schwarz</span>, <span class="hlFld-ContribAuthor ">David  Wustrow</span>, <span class="hlFld-ContribAuthor ">Dirk G.  Brockstedt</span>, <span class="hlFld-ContribAuthor ">Paul D.  Kassner</span>. </span><span class="cited-content_cbyCitation_article-title">Novel, Selective Inhibitors of USP7 Uncover Multiple Mechanisms of Antitumor Activity
              In Vitro
              and
              In Vivo. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2020,</strong> <em>19 </em>
                                    (10)
                                     , 1970-1980. <a href="https://doi.org/10.1158/1535-7163.MCT-20-0184" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-20-0184</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-20-0184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-20-0184%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DNovel%25252C%252BSelective%252BInhibitors%252Bof%252BUSP7%252BUncover%252BMultiple%252BMechanisms%252Bof%252BAntitumor%252BActivity%252BIn%252BVitro%252Band%252BIn%252BVivo%26aulast%3DOhol%26aufirst%3DYamini%2BM.%26date%3D2020%26date%3D2020%26volume%3D19%26issue%3D10%26spage%3D1970%26epage%3D1980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abdul-Hamid  Emwas</span>, <span class="hlFld-ContribAuthor ">Kacper  Szczepski</span>, <span class="hlFld-ContribAuthor ">Benjamin Gabriel  Poulson</span>, <span class="hlFld-ContribAuthor ">Kousik  Chandra</span>, <span class="hlFld-ContribAuthor ">Ryan T.  McKay</span>, <span class="hlFld-ContribAuthor ">Manel  Dhahri</span>, <span class="hlFld-ContribAuthor ">Fatimah  Alahmari</span>, <span class="hlFld-ContribAuthor ">Lukasz  Jaremko</span>, <span class="hlFld-ContribAuthor ">Joanna Izabela  Lachowicz</span>, <span class="hlFld-ContribAuthor ">Mariusz  Jaremko</span>. </span><span class="cited-content_cbyCitation_article-title">NMR as a “Gold Standard” Method in Drug Design and Discovery. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (20)
                                     , 4597. <a href="https://doi.org/10.3390/molecules25204597" title="DOI URL">https://doi.org/10.3390/molecules25204597</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25204597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25204597%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNMR%252Bas%252Ba%252B%2525E2%252580%25259CGold%252BStandard%2525E2%252580%25259D%252BMethod%252Bin%252BDrug%252BDesign%252Band%252BDiscovery%26aulast%3DEmwas%26aufirst%3DAbdul-Hamid%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D20%26spage%3D4597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabrielle J.  Valles</span>, <span class="hlFld-ContribAuthor ">Irina  Bezsonova</span>, <span class="hlFld-ContribAuthor ">Roger  Woodgate</span>, <span class="hlFld-ContribAuthor ">Nicholas W.  Ashton</span>. </span><span class="cited-content_cbyCitation_article-title">USP7 Is a Master Regulator of Genome Stability. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fcell.2020.00717" title="DOI URL">https://doi.org/10.3389/fcell.2020.00717</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2020.00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2020.00717%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DUSP7%252BIs%252Ba%252BMaster%252BRegulator%252Bof%252BGenome%252BStability%26aulast%3DValles%26aufirst%3DGabrielle%2BJ.%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hai Qiu  Wu</span>, <span class="hlFld-ContribAuthor ">David  Baker</span>, <span class="hlFld-ContribAuthor ">Huib  Ovaa</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecules that target the ubiquitin system. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Society Transactions</span><span> <strong>2020,</strong> <em>48 </em>
                                    (2)
                                     , 479-497. <a href="https://doi.org/10.1042/BST20190535" title="DOI URL">https://doi.org/10.1042/BST20190535</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BST20190535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBST20190535%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Society%2520Transactions%26atitle%3DSmall%252Bmolecules%252Bthat%252Btarget%252Bthe%252Bubiquitin%252Bsystem%26aulast%3DWu%26aufirst%3DHai%2BQiu%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D2%26spage%3D479%26epage%3D497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng  Li</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in the development of ubiquitin-specific-processing protease 7 (USP7) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>191 </em>, 112107. <a href="https://doi.org/10.1016/j.ejmech.2020.112107" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112107</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112107%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252Bthe%252Bdevelopment%252Bof%252Bubiquitin-specific-processing%252Bprotease%252B7%252B%252528USP7%252529%252Binhibitors%26aulast%3DLi%26aufirst%3DPeng%26date%3D2020%26volume%3D191%26spage%3D112107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Georges</span>, <span class="hlFld-ContribAuthor ">Etienne  Coyaud</span>, <span class="hlFld-ContribAuthor ">Edyta  Marcon</span>, <span class="hlFld-ContribAuthor ">Jack  Greenblatt</span>, <span class="hlFld-ContribAuthor ">Brian  Raught</span>, <span class="hlFld-ContribAuthor ">Lori  Frappier</span>. </span><span class="cited-content_cbyCitation_article-title">USP7 Regulates Cytokinesis through FBXO38 and KIF20B. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-019-39368-y" title="DOI URL">https://doi.org/10.1038/s41598-019-39368-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-019-39368-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-019-39368-y%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DUSP7%252BRegulates%252BCytokinesis%252Bthrough%252BFBXO38%252Band%252BKIF20B%26aulast%3DGeorges%26aufirst%3DAnna%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ingrid E.  Wertz</span>, <span class="hlFld-ContribAuthor ">Jeremy M.  Murray</span>. </span><span class="cited-content_cbyCitation_article-title">Structurally-defined deubiquitinase inhibitors provide opportunities to investigate disease mechanisms. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2019,</strong> <em>31 </em>, 109-123. <a href="https://doi.org/10.1016/j.ddtec.2019.02.003" title="DOI URL">https://doi.org/10.1016/j.ddtec.2019.02.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2019.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2019.02.003%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DStructurally-defined%252Bdeubiquitinase%252Binhibitors%252Bprovide%252Bopportunities%252Bto%252Binvestigate%252Bdisease%252Bmechanisms%26aulast%3DWertz%26aufirst%3DIngrid%2BE.%26date%3D2019%26volume%3D31%26spage%3D109%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexandra  Pozhidaeva</span>, <span class="hlFld-ContribAuthor ">Irina  Bezsonova</span>. </span><span class="cited-content_cbyCitation_article-title">USP7: Structure, substrate specificity, and inhibition. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2019,</strong> <em>76 </em>, 30-39. <a href="https://doi.org/10.1016/j.dnarep.2019.02.005" title="DOI URL">https://doi.org/10.1016/j.dnarep.2019.02.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2019.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2019.02.005%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DUSP7%25253A%252BStructure%25252C%252Bsubstrate%252Bspecificity%25252C%252Band%252Binhibition%26aulast%3DPozhidaeva%26aufirst%3DAlexandra%26date%3D2019%26volume%3D76%26spage%3D30%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ingrid E.  Wertz</span>, <span class="hlFld-ContribAuthor ">Xiaojing  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">From Discovery to Bedside: Targeting the Ubiquitin System. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (2)
                                     , 156-177. <a href="https://doi.org/10.1016/j.chembiol.2018.10.022" title="DOI URL">https://doi.org/10.1016/j.chembiol.2018.10.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2018.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2018.10.022%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DFrom%252BDiscovery%252Bto%252BBedside%25253A%252BTargeting%252Bthe%252BUbiquitin%252BSystem%26aulast%3DWertz%26aufirst%3DIngrid%2BE.%26date%3D2019%26volume%3D26%26issue%3D2%26spage%3D156%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tao  Hu</span>, <span class="hlFld-ContribAuthor ">Jingyang  Zhang</span>, <span class="hlFld-ContribAuthor ">Beibei  Sha</span>, <span class="hlFld-ContribAuthor ">Miaomiao  Li</span>, <span class="hlFld-ContribAuthor ">Longhao  Wang</span>, <span class="hlFld-ContribAuthor ">Yi  Zhang</span>, <span class="hlFld-ContribAuthor ">Xingge  Liu</span>, <span class="hlFld-ContribAuthor ">Ziming  Dong</span>, <span class="hlFld-ContribAuthor ">Zhenzhen  Liu</span>, <span class="hlFld-ContribAuthor ">Pei  Li</span>, <span class="hlFld-ContribAuthor ">Ping  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the overexpressed USP7 inhibits esophageal squamous cell carcinoma cell growth by inducing NOXA‐mediated apoptosis. </span><span class="cited-content_cbyCitation_journal-name">Molecular Carcinogenesis</span><span> <strong>2019,</strong> <em>58 </em>
                                    (1)
                                     , 42-54. <a href="https://doi.org/10.1002/mc.22905" title="DOI URL">https://doi.org/10.1002/mc.22905</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/mc.22905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmc.22905%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Carcinogenesis%26atitle%3DTargeting%252Bthe%252Boverexpressed%252BUSP7%252Binhibits%252Besophageal%252Bsquamous%252Bcell%252Bcarcinoma%252Bcell%252Bgrowth%252Bby%252Binducing%252BNOXA%2525E2%252580%252590mediated%252Bapoptosis%26aulast%3DHu%26aufirst%3DTao%26date%3D2019%26date%3D2018%26volume%3D58%26issue%3D1%26spage%3D42%26epage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Georges</span>, <span class="hlFld-ContribAuthor ">Edyta  Marcon</span>, <span class="hlFld-ContribAuthor ">Jack  Greenblatt</span>, <span class="hlFld-ContribAuthor ">Lori  Frappier</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Characterization of USP7 Targets in Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-34197-x" title="DOI URL">https://doi.org/10.1038/s41598-018-34197-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-34197-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-34197-x%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DIdentification%252Band%252BCharacterization%252Bof%252BUSP7%252BTargets%252Bin%252BCancer%252BCells%26aulast%3DGeorges%26aufirst%3DAnna%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mitul  Srivastava</span>, <span class="hlFld-ContribAuthor ">Charu  Suri</span>, <span class="hlFld-ContribAuthor ">Mrityunjay  Singh</span>, <span class="hlFld-ContribAuthor ">Rajani  Mathur</span>, <span class="hlFld-ContribAuthor ">Shailendra  Asthana</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular dynamics simulation reveals the possible druggable
              hot-spots
              of USP7. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2018,</strong> <em>9 </em>
                                    (76)
                                     , 34289-34305. <a href="https://doi.org/10.18632/oncotarget.26136" title="DOI URL">https://doi.org/10.18632/oncotarget.26136</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.26136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.26136%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DMolecular%252Bdynamics%252Bsimulation%252Breveals%252Bthe%252Bpossible%252Bdruggable%252Bhot-spots%252Bof%252BUSP7%26aulast%3DSrivastava%26aufirst%3DMitul%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D76%26spage%3D34289%26epage%3D34305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Farjana  Yeasmin Khusbu</span>, <span class="hlFld-ContribAuthor ">Fang-Zhi  Chen</span>, <span class="hlFld-ContribAuthor ">Han-Chun  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting ubiquitin specific protease 7 in cancer: A deubiquitinase with great prospects. </span><span class="cited-content_cbyCitation_journal-name">Cell Biochemistry and Function</span><span> <strong>2018,</strong> <em>36 </em>
                                    (5)
                                     , 244-254. <a href="https://doi.org/10.1002/cbf.3336" title="DOI URL">https://doi.org/10.1002/cbf.3336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbf.3336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbf.3336%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Biochemistry%2520and%2520Function%26atitle%3DTargeting%252Bubiquitin%252Bspecific%252Bprotease%252B7%252Bin%252Bcancer%25253A%252BA%252Bdeubiquitinase%252Bwith%252Bgreat%252Bprospects%26aulast%3DYeasmin%2BKhusbu%26aufirst%3DFarjana%26date%3D2018%26date%3D2018%26volume%3D36%26issue%3D5%26spage%3D244%26epage%3D254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0001.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Compound <b>1</b> and compound <b>2</b> bind two distinct sites on USP7-CD. (A) Selected region of the overlay between the 2D <sup>1</sup>H–<sup>15</sup>N TROSY spectra of labeled USP7-CD apo (in black) and in the presence of 8 equiv of compound <b>1</b> (in red). (B) Selected region of the overlay between the 2D <sup>1</sup>H–<sup>15</sup>N TROSY spectra of labeled USP7-CD apo (in black) and after addition of 8 equiv of compound <b>2</b> (in red). Arrows and ovals indicate signals undergoing large chemical shifts changes. (C, D) Ribbon model of the apo USP7-CD crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M5W">4M5W</a>). Amino acids experiencing large chemical shifts perturbations upon formation of the USP7-CD/<b>1</b> complex are highlighted in orange, whereas the amino acids displaying large chemical shift changes upon formation of the USP7-CD/<b>2</b> complex are colored in yellow. The catalytic triad residues, Cys223, His464, and Asp481, are shown in sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0002.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of <b>2</b> in complex with USP7-CD (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHC">5WHC</a>). Blue mesh represents electron density contoured at 0.7σ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0003.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>2</b> and compound <b>10</b> bind to the same site on USP7-CD. (A) Re-mined fragment hits from the NMR screen. (B) 2D <sup>1</sup>H–<sup>15</sup>N TROSY spectra overlay for labeled USP7-CD in the apo form (in black) and in the presence of compound <b>10</b> (in red). Protein:ligand ratio is 1:8. (C) Overlay between the 2D <sup>1</sup>H–<sup>15</sup>N TROSY spectra of labeled USP7-CD apo (in black) and after addition of compound <b>2</b> (in red). Protein:ligand ratio is 1:8. Arrows and ovals highlight some of the signals undergoing large chemical shifts changes or line broadening upon complex formation. The chemical shifts changes induced by the two distinct series are virtually identical.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0004.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structure of <b>16</b> with USP7-CD (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WH7">5WH7</a>). (A) 2.8 Å electron density (blue mesh) for <b>16</b> was ambiguous and suggested that it was possible to bind in different alternative conformations. (B) Surface representation of the binding pocket highlighting the hydrophobic pocket (magenta).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0005.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. 2D representation of possible ligand binding modes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0006.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Inhibiting USP7 activity increases p53 and p21 levels of EOL-1 cells. (A) Illustration of USP7 signaling through the MDM2-p53-p21 axis in the absence (left panel) and in the presence (right panel) of USP7 inhibitors. Left panel: USP7 stabilizes MDM2 by rescuing it from proteosomal degradation. MDM2 down-regulates the p53 pathway, thus leading to a reduction in p53-depended apoptosis, a reduction in p21-mediated cell cycle arrest, and an increase in cell viability. Right panel: USP7 inhibition is expected to promote MDM2 proteosomal degradation. This causes a reduction in the MDM2 levels and the consequent increase in p53 protein levels, ultimately resulting in the activation of caspases and modulation of p53-target genes, e.g., p21. An increase in caspase activity triggers apoptosis, while an increase in p21 triggers cell cycle arrest. Both result in a decrease in cell viability and cell number. (B) Western blots using antibodies against USP7, MDM2, tubulin, p53 and p21. (C, D) Graphs showing quantification of p53 and p21 Western blots normalized to tubulin. Under these conditions, compounds <b>27</b> and <b>28</b> increase relative p53 levels, whereas <b>18</b>, <b>27</b> and <b>28</b> increase relative p21 levels. Data are representative of two experiments. NC: negative control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0007.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Inhibiting USP7 activity increases caspase activity and decreases viability of EOL-1 cells. (A) IncuCyte data showing a dose-dependent decrease in cell confluence with inhibitors relative to control. (B) IncuCyte data showing a dose-dependent increase in caspase activity with inhibitors relative to control. (C) CellTiter-Glo viability data validating IncuCyte cell confluence results. Data are representative of three experiments. NC: negative control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>2</b> and <b>3</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NH<sub>2</sub>OH, CDI, 18% yield; (b) HCl, EtOAc, 99% yield; (c) HCHO, NaCNBH<sub>3</sub>, AcOH, 38% yield.</p></p></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 4-OMe-benzamide, 8% yield; (b) HBr, 80% yield.</p></p></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) nitromethane, KO<sup><i>t</i></sup>Bu, 62% yield; (b) 10% Pd/C, H<sub>2</sub>, quant; (c) 4-OMe-benzoic acid, HATU, TEA, 51% yield; (d) Dess–Martin reagent, 27% yield; (e) Burgess reagent, 84% yield; (f) HBr, 24% yield.</p></p></figure><figure data-id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0011.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>17</b>–<b>23</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NBS, DCM; (b) anisole-4-boronic acid, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C, quant; (c) BBr<sub>3</sub>, DCM.</p></p></figure><figure data-id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0012.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>24</b>–<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NBS, THF, 79% yield; (b) <i>n</i>-BuLi, THF, −78 °C, 93% yield; (c) anisole-4-boronic acid, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C, 73% yield; (d) NBS, DCM, 59% yield; (e) BBr<sub>3</sub>, DCM, 83% yield; (f) RB(OH)<sub>2</sub>, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C.</p></p></figure><figure data-id="sch6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/medium/jm-2017-012934_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0013.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>30</b>–<b>32</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-24/acs.jmedchem.7b01293/20171221/images/large/jm-2017-012934_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01293&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NBS, THF, 72% yield; (b) anisole-4-boronic acid, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C, 66% yield; (c) NBS, TFH, 67% yield; (d) BBr<sub>3</sub>, DCM, 90% yield; (e) RB(OH)<sub>2</sub>, Pd(dppf)<sub>2</sub>Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, ACN, 120 °C.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i77">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63380" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63380" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 27 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kategaya, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Lello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drummond, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleinheinz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Upton, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prakash, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heideker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCleland, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritorto, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bainbridge, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwok, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stiffler, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brasher, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alessi, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trost, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaishankar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hearn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renslo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arkin, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peale, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gnad, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klijn, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackwood, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forrest, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ndubaku, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beresini, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwerdtfeger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wertz, I. E.</span><span> </span><span class="NLM_article-title">USP7 small molecule inhibitors interfere with ubiquitin binding</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">550</span><span class="NLM_x">, </span> <span class="NLM_fpage">534</span><span class="NLM_x">–</span> <span class="NLM_lpage">538</span><span class="refDoi"> DOI: 10.1038/nature24006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1038%2Fnature24006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=534-538&author=L.+Kategayaauthor=P.+Di+Lelloauthor=L.+Rougeauthor=R.+Pastorauthor=K.+R.+Clarkauthor=J.+Drummondauthor=T.+Kleinheinzauthor=E.+Linauthor=J.+P.+Uptonauthor=S.+Prakashauthor=J.+Heidekerauthor=M.+McClelandauthor=M.+S.+Ritortoauthor=T.+W.+Bainbridgeauthor=T.+P.+Maauthor=M.+Kwokauthor=Z.+Stifflerauthor=B.+Brasherauthor=D.+R.+Alessiauthor=M.+Trostauthor=Y.+Tangauthor=P.+Jaishankarauthor=B.+Hearnauthor=A.+R.+Rensloauthor=M.+R.+Arkinauthor=F.+Cohenauthor=K.+Yuauthor=F.+Pealeauthor=F.+Gnadauthor=M.+T.+Changauthor=C.+Klijnauthor=E.+Blackwoodauthor=S.+E.+Martinauthor=W.+F.+Forrestauthor=J.+A.+Ernstauthor=C.+Ndubakuauthor=X.+Wangauthor=M.+H.+Beresiniauthor=V.+Tsuiauthor=C.+Schwerdtfegerauthor=R.+A.+Blakeauthor=J.+Murrayauthor=T.+Maurerauthor=I.+E.+Wertz&title=USP7+small+molecule+inhibitors+interfere+with+ubiquitin+binding&doi=10.1038%2Fnature24006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature24006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24006%26sid%3Dliteratum%253Aachs%26aulast%3DKategaya%26aufirst%3DL.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DRouge%26aufirst%3DL.%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DClark%26aufirst%3DK.%2BR.%26aulast%3DDrummond%26aufirst%3DJ.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DE.%26aulast%3DUpton%26aufirst%3DJ.%2BP.%26aulast%3DPrakash%26aufirst%3DS.%26aulast%3DHeideker%26aufirst%3DJ.%26aulast%3DMcCleland%26aufirst%3DM.%26aulast%3DRitorto%26aufirst%3DM.%2BS.%26aulast%3DBainbridge%26aufirst%3DT.%2BW.%26aulast%3DMa%26aufirst%3DT.%2BP.%26aulast%3DKwok%26aufirst%3DM.%26aulast%3DStiffler%26aufirst%3DZ.%26aulast%3DBrasher%26aufirst%3DB.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DTrost%26aufirst%3DM.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DJaishankar%26aufirst%3DP.%26aulast%3DHearn%26aufirst%3DB.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DPeale%26aufirst%3DF.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DChang%26aufirst%3DM.%2BT.%26aulast%3DKlijn%26aufirst%3DC.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DS.%2BE.%26aulast%3DForrest%26aufirst%3DW.%2BF.%26aulast%3DErnst%26aufirst%3DJ.%2BA.%26aulast%3DNdubaku%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DSchwerdtfeger%26aufirst%3DC.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26atitle%3DUSP7%2520small%2520molecule%2520inhibitors%2520interfere%2520with%2520ubiquitin%2520binding%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D534%26epage%3D538%26doi%3D10.1038%2Fnature24006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Hochstrasser, M.</span><span> </span><span class="NLM_article-title">Ubiquitin-dependent protein degradation</span> <span class="citation_source-journal">Annu. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1146/annurev.genet.30.1.405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1146%2Fannurev.genet.30.1.405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=8982460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK2sXht1yk" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1996&pages=405-439&author=M.+Hochstrasser&title=Ubiquitin-dependent+protein+degradation&doi=10.1146%2Fannurev.genet.30.1.405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin-dependent protein degradation</span></div><div class="casAuthors">Hochstrasser, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Genetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">405-439</span>CODEN:
                <span class="NLM_cas:coden">ARVGB7</span>;
        ISSN:<span class="NLM_cas:issn">0066-4197</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review with 115 refs.  A growing no. of cellular regulatory mechanisms are being linked to protein modification by the polypeptide ubiquitin.  These include key transitions in the cell cycle, class I antigen processing, signal transduction pathways, and receptor-mediated endocytosis.  In most, but not all, of these examples, ubiquitination of a protein leads to its degrdn. by the 26S proteasome.  Following attachment of ubiquitin to a substrate and binding of the ubiquitinated protein to the proteasome, the bound substrate must be unfolded (and eventually deubiquitinated) and translocated through a narrow set of channels that leads to the proteasome interior, where the polypeptide is cleaved into short peptides.  Protein ubiquitination and deubiquitination are both mediated by large enzyme families, and the proteasome itself comprises a family of related but functionally distinct particles.  This diversity underlies both the high substrate specificity of the ubiquitin system and the variety of regulatory mechanisms that it serves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8i3KZXfGSRrVg90H21EOLACvtfcHk0lgXbpel9rB-3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXht1yk&md5=8b3d3ec0806e94afee1e0374fae69db2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.genet.30.1.405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.genet.30.1.405%26sid%3Dliteratum%253Aachs%26aulast%3DHochstrasser%26aufirst%3DM.%26atitle%3DUbiquitin-dependent%2520protein%2520degradation%26jtitle%3DAnnu.%2520Rev.%2520Genet.%26date%3D1996%26volume%3D30%26spage%3D405%26epage%3D439%26doi%3D10.1146%2Fannurev.genet.30.1.405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Ardley, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, P. A.</span><span> </span><span class="NLM_article-title">E3 ubiquitin ligases</span> <span class="citation_source-journal">Essays Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1042/bse0410015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1042%2Fbse0410015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=16250895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1aitb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=15-30&author=H.+C.+Ardleyauthor=P.+A.+Robinson&title=E3+ubiquitin+ligases&doi=10.1042%2Fbse0410015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">E3 ubiquitin ligases</span></div><div class="casAuthors">Ardley, Helen C.; Robinson, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Essays in Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-30</span>CODEN:
                <span class="NLM_cas:coden">ESBIAV</span>;
        ISSN:<span class="NLM_cas:issn">0071-1365</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The selectivity of the ubiquitin-26 S proteasome system (UPS) for a particular substrate protein relies on the interaction between a ubiquitin-conjugating enzyme (E2, of which a cell contains relatively few) and a ubiquitin-protein ligase (E3, of which there are possibly hundreds).  Post-translational modifications of the protein substrate, such as phosphorylation or hydroxylation, are often required prior to its selection.  In this way, the precise spatio-temporal targeting and degrdn. of a given substrate can be achieved.  The E3s are a large, diverse group of proteins, characterized by one of several defining motifs.  These include a HECT (homologous to E6-assocd. protein C-terminus), RING (really interesting new gene) or U-box (a modified RING motif without the full complement of Zn2+-binding ligands) domain.  Whereas HECT E3s have a direct role in catalysis during ubiquitination, RING and U-box E3s facilitate protein ubiquitination.  These latter 2 E3 types act as adaptor-like mols.  They bring an E2 and a substrate into sufficiently close proximity to promote the substrate's ubiquitination.  Although many RING-type E3s, such as MDM2 (murine double minute clone 2 oncoprotein) and c-Cbl, can apparently act alone, others are found as components of much larger multi-protein complexes, such as the anaphase-promoting complex.  Taken together, these multifaceted properties and interactions enable E3s to provide a powerful, and specific, mechanism for protein clearance within all cells of eukaryotic organisms.  The importance of E3s is highlighted by the no. of normal cellular processes they regulate, and the no. of diseases assocd. with their loss of function or inappropriate targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodKkSLAWRHwrVg90H21EOLACvtfcHk0lgXbpel9rB-3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1aitb7K&md5=497eb1d687d5a8b922282881852ddc25</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1042%2Fbse0410015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbse0410015%26sid%3Dliteratum%253Aachs%26aulast%3DArdley%26aufirst%3DH.%2BC.%26aulast%3DRobinson%26aufirst%3DP.%2BA.%26atitle%3DE3%2520ubiquitin%2520ligases%26jtitle%3DEssays%2520Biochem.%26date%3D2005%26volume%3D41%26spage%3D15%26epage%3D30%26doi%3D10.1042%2Fbse0410015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Pickart, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fushman, D.</span><span> </span><span class="NLM_article-title">Polyubiquitin chains: polymeric protein signals</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">610</span><span class="NLM_x">–</span> <span class="NLM_lpage">616</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2004.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2Fj.cbpa.2004.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=15556404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVWjtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2004&pages=610-616&author=C.+M.+Pickartauthor=D.+Fushman&title=Polyubiquitin+chains%3A+polymeric+protein+signals&doi=10.1016%2Fj.cbpa.2004.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Polyubiquitin chains: polymeric protein signals</span></div><div class="casAuthors">Pickart, Cecile M.; Fushman, David</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">610-616</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review and discussion.  The 76-residue protein, ubiquitin, exists within eukaryotic cells both as a monomer and in the form of isopeptide-linked polymers called polyubiquitin chains.  In 2 well-described cases, structurally distinct polyubiquitin chains represent functionally distinct intracellular signals.  Recently, addnl. polymeric structures have been detected in vivo and in vitro, and several large families of proteins with polyubiquitin chain-binding activity have been discovered.  Although the mol. mechanisms governing specificity in chain synthesis and recognition are still incompletely understood, the scope of signaling by polyubiquitin chains is likely to be broader than originally envisioned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpyW8mbbQxk7Vg90H21EOLACvtfcHk0lh1r_Za-PcrkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVWjtrrN&md5=c7d236b38bc00612688499be1fc1f808</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2004.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2004.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DPickart%26aufirst%3DC.%2BM.%26aulast%3DFushman%26aufirst%3DD.%26atitle%3DPolyubiquitin%2520chains%253A%2520polymeric%2520protein%2520signals%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2004%26volume%3D8%26spage%3D610%26epage%3D616%26doi%3D10.1016%2Fj.cbpa.2004.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Komander, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rape, M.</span><span> </span><span class="NLM_article-title">The ubiquitin code</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">203</span><span class="NLM_x">–</span> <span class="NLM_lpage">229</span><span class="refDoi"> DOI: 10.1146/annurev-biochem-060310-170328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1146%2Fannurev-biochem-060310-170328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=22524316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVGls7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2012&pages=203-229&author=D.+Komanderauthor=M.+Rape&title=The+ubiquitin+code&doi=10.1146%2Fannurev-biochem-060310-170328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin code</span></div><div class="casAuthors">Komander, David; Rape, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">203-229</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The posttranslational modification with ubiquitin, a process referred to as ubiquitylation, controls almost every process in cells.  Ubiquitin can be attached to substrate proteins as a single moiety or as polymeric chains in which successive ubiquitin mols. are connected through specific isopeptide bonds.  Reminiscent of a code, the various ubiquitin modifications adopt distinct conformations and lead to different outcomes in cells.  Here, the structure, assembly, and function of this ubiquitin code are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCbRgaWmg1IrVg90H21EOLACvtfcHk0lh1r_Za-PcrkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVGls7bJ&md5=85b2a166393de56551d541636dbda06c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060310-170328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060310-170328%26sid%3Dliteratum%253Aachs%26aulast%3DKomander%26aufirst%3DD.%26aulast%3DRape%26aufirst%3DM.%26atitle%3DThe%2520ubiquitin%2520code%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2012%26volume%3D81%26spage%3D203%26epage%3D229%26doi%3D10.1146%2Fannurev-biochem-060310-170328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Komander, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clague, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urbe, S.</span><span> </span><span class="NLM_article-title">Breaking the chains: structure and function of the deubiquitinases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">563</span><span class="refDoi"> DOI: 10.1038/nrm2731</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1038%2Fnrm2731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=19626045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovFSru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=550-563&author=D.+Komanderauthor=M.+J.+Clagueauthor=S.+Urbe&title=Breaking+the+chains%3A+structure+and+function+of+the+deubiquitinases&doi=10.1038%2Fnrm2731"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Breaking the chains: structure and function of the deubiquitinases</span></div><div class="casAuthors">Komander, David; Clague, Michael J.; Urbe, Sylvie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">550-563</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Ubiquitination is a reversible protein modification that is implicated in many cellular functions.  Recently, much progress has been made in the characterization of a superfamily of isopeptidases that remove ubiquitin: the deubiquitinases (DUBs; also known as deubiquitinating enzymes).  Far from being uniform in structure and function, these enzymes display a myriad of distinct mechanistic features.  The small no. (<100) of DUBs might at 1st suggest a low degree of selectivity; however, DUBs are subject to multiple layers of regulation that modulate both their activity and their specificity.  Due to their wide-ranging involvement in key regulatory processes, these enzymes might provide new therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8Woj6hOAP5rVg90H21EOLACvtfcHk0lh1r_Za-PcrkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovFSru74%253D&md5=9268e6c24101ff8137937a5c851d5bb8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm2731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2731%26sid%3Dliteratum%253Aachs%26aulast%3DKomander%26aufirst%3DD.%26aulast%3DClague%26aufirst%3DM.%2BJ.%26aulast%3DUrbe%26aufirst%3DS.%26atitle%3DBreaking%2520the%2520chains%253A%2520structure%2520and%2520function%2520of%2520the%2520deubiquitinases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2009%26volume%3D10%26spage%3D550%26epage%3D563%26doi%3D10.1038%2Fnrm2731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saad, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolattukudy, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, M.</span><span> </span><span class="NLM_article-title">MCP-induced protein 1 deubiquitinates TRAF proteins and negatively regulates JNK and NF-kappaB signaling</span> <span class="citation_source-journal">J. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">207</span><span class="NLM_x">, </span> <span class="NLM_fpage">2959</span><span class="NLM_x">–</span> <span class="NLM_lpage">2973</span><span class="refDoi"> DOI: 10.1084/jem.20092641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1084%2Fjem.20092641" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2010&pages=2959-2973&author=J.+Liangauthor=Y.+Saadauthor=T.+Leiauthor=J.+Wangauthor=D.+Qiauthor=Q.+Yangauthor=P.+E.+Kolattukudyauthor=M.+Fu&title=MCP-induced+protein+1+deubiquitinates+TRAF+proteins+and+negatively+regulates+JNK+and+NF-kappaB+signaling&doi=10.1084%2Fjem.20092641"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1084%2Fjem.20092641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20092641%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DSaad%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DQi%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKolattukudy%26aufirst%3DP.%2BE.%26aulast%3DFu%26aufirst%3DM.%26atitle%3DMCP-induced%2520protein%25201%2520deubiquitinates%2520TRAF%2520proteins%2520and%2520negatively%2520regulates%2520JNK%2520and%2520NF-kappaB%2520signaling%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2010%26volume%3D207%26spage%3D2959%26epage%3D2973%26doi%3D10.1084%2Fjem.20092641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Fraile, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quesada, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freije, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Otin, C.</span><span> </span><span class="NLM_article-title">Deubiquitinases in cancer: new functions and therapeutic options</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">2373</span><span class="NLM_x">–</span> <span class="NLM_lpage">2388</span><span class="refDoi"> DOI: 10.1038/onc.2011.443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1038%2Fonc.2011.443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=21996736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVCjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=2373-2388&author=J.+M.+Fraileauthor=V.+Quesadaauthor=D.+Rodriguezauthor=J.+M.+Freijeauthor=C.+Lopez-Otin&title=Deubiquitinases+in+cancer%3A+new+functions+and+therapeutic+options&doi=10.1038%2Fonc.2011.443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Deubiquitinases in cancer: new functions and therapeutic options</span></div><div class="casAuthors">Fraile, J. M.; Quesada, V.; Rodriguez, D.; Freije, J. M. P.; Lopez-Otin, C.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2373-2388</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deubiquitinases (DUBs) have fundamental roles in the ubiquitin system through their ability to specifically deconjugate ubiquitin from targeted proteins.  The human genome encodes at least 98 DUBs, which can be grouped into 6 families, reflecting the need for specificity in their function.  The activity of these enzymes affects the turnover rate, activation, recycling and localization of multiple proteins, which in turn is essential for cell homeostasis, protein stability and a wide range of signaling pathways.  Consistent with this, altered DUB function has been related to several diseases, including cancer.  Thus, multiple DUBs have been classified as oncogenes or tumor suppressors because of their regulatory functions on the activity of other proteins involved in tumor development.  Therefore, recent studies have focused on pharmacol. intervention on DUB activity as a rationale to search for novel anticancer drugs.  This strategy may benefit from our current knowledge of the physiol. regulatory mechanisms of these enzymes and the fact that growth of several tumors depends on the normal activity of certain DUBs.  Further understanding of these processes may provide answers to multiple remaining questions on DUB functions and lead to the development of DUB-targeting strategies to expand the repertoire of mol. therapies against cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ3Fzatsb-GrVg90H21EOLACvtfcHk0ljZ-Dsi8K3p0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVCjsro%253D&md5=e31609c839729ac70591c615e06eaf44</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fonc.2011.443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2011.443%26sid%3Dliteratum%253Aachs%26aulast%3DFraile%26aufirst%3DJ.%2BM.%26aulast%3DQuesada%26aufirst%3DV.%26aulast%3DRodriguez%26aufirst%3DD.%26aulast%3DFreije%26aufirst%3DJ.%2BM.%26aulast%3DLopez-Otin%26aufirst%3DC.%26atitle%3DDeubiquitinases%2520in%2520cancer%253A%2520new%2520functions%2520and%2520therapeutic%2520options%26jtitle%3DOncogene%26date%3D2012%26volume%3D31%26spage%3D2373%26epage%3D2388%26doi%3D10.1038%2Fonc.2011.443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Abdul Rehman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristariyanto, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nkosi, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weidlich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labib, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofmann, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulathu, Y.</span><span> </span><span class="NLM_article-title">MINDY-1 is a member of an evolutionarily conserved and structurally distinct new family of deubiquitinating enzymes</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span><span class="refDoi"> DOI: 10.1016/j.molcel.2016.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2Fj.molcel.2016.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=27292798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2016&pages=146-155&author=S.+A.+Abdul+Rehmanauthor=Y.+A.+Kristariyantoauthor=S.+Y.+Choiauthor=P.+J.+Nkosiauthor=S.+Weidlichauthor=K.+Labibauthor=K.+Hofmannauthor=Y.+Kulathu&title=MINDY-1+is+a+member+of+an+evolutionarily+conserved+and+structurally+distinct+new+family+of+deubiquitinating+enzymes&doi=10.1016%2Fj.molcel.2016.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">MINDY-1 Is a Member of an Evolutionarily Conserved and Structurally Distinct New Family of Deubiquitinating Enzymes</span></div><div class="casAuthors">Abdul Rehman, Syed Arif; Kristariyanto, Yosua Adi; Choi, Soo-Youn; Nkosi, Pedro, Jr.; Weidlich, Simone; Labib, Karim; Hofmann, Kay; Kulathu, Yogesh</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">146-155</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Deubiquitinating enzymes (DUBs) remove ubiquitin (Ub) from Ub-conjugated substrates to regulate the functional outcome of ubiquitylation.  Here we report the discovery of a new family of DUBs, which we have named MINDY (motif interacting with Ub-contg. novel DUB family).  Found in all eukaryotes, MINDY-family DUBs are highly selective at cleaving K48-linked polyUb, a signal that targets proteins for degrdn.  We identify the catalytic activity to be encoded within a previously unannotated domain, the crystal structure of which reveals a distinct protein fold with no homol. to any of the known DUBs.  The crystal structure of MINDY-1 (also known as FAM63A) in complex with propargylated Ub reveals conformational changes that realign the active site for catalysis.  MINDY-1 prefers cleaving long polyUb chains and works by trimming chains from the distal end.  Collectively, our results reveal a new family of DUBs that may have specialized roles in regulating proteostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPJ1o32zd56LVg90H21EOLACvtfcHk0ljZ-Dsi8K3p0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWltL0%253D&md5=80c78d5cd1d2d70e0e2c71c7524f5798</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DAbdul%2BRehman%26aufirst%3DS.%2BA.%26aulast%3DKristariyanto%26aufirst%3DY.%2BA.%26aulast%3DChoi%26aufirst%3DS.%2BY.%26aulast%3DNkosi%26aufirst%3DP.%2BJ.%26aulast%3DWeidlich%26aufirst%3DS.%26aulast%3DLabib%26aufirst%3DK.%26aulast%3DHofmann%26aufirst%3DK.%26aulast%3DKulathu%26aufirst%3DY.%26atitle%3DMINDY-1%2520is%2520a%2520member%2520of%2520an%2520evolutionarily%2520conserved%2520and%2520structurally%2520distinct%2520new%2520family%2520of%2520deubiquitinating%2520enzymes%26jtitle%3DMol.%2520Cell%26date%3D2016%26volume%3D63%26spage%3D146%26epage%3D155%26doi%3D10.1016%2Fj.molcel.2016.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Daviet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colland, F.</span><span> </span><span class="NLM_article-title">Targeting ubiquitin specific proteases for drug discovery</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">270</span><span class="NLM_x">–</span> <span class="NLM_lpage">283</span><span class="refDoi"> DOI: 10.1016/j.biochi.2007.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2Fj.biochi.2007.09.013" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2008&pages=270-283&author=L.+Davietauthor=F.+Colland&title=Targeting+ubiquitin+specific+proteases+for+drug+discovery&doi=10.1016%2Fj.biochi.2007.09.013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2007.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2007.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DDaviet%26aufirst%3DL.%26aulast%3DColland%26aufirst%3DF.%26atitle%3DTargeting%2520ubiquitin%2520specific%2520proteases%2520for%2520drug%2520discovery%26jtitle%3DBiochimie%26date%3D2008%26volume%3D90%26spage%3D270%26epage%3D283%26doi%3D10.1016%2Fj.biochi.2007.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Lim, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baek, K. H.</span><span> </span><span class="NLM_article-title">Deubiquitinating enzymes as therapeutic targets in cancer</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4039</span><span class="NLM_x">–</span> <span class="NLM_lpage">4052</span><span class="refDoi"> DOI: 10.2174/1381612811319220013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.2174%2F1381612811319220013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=23181570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpslyrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=4039-4052&author=K.+H.+Limauthor=K.+H.+Baek&title=Deubiquitinating+enzymes+as+therapeutic+targets+in+cancer&doi=10.2174%2F1381612811319220013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Deubiquitinating enzymes as therapeutic targets in cancer</span></div><div class="casAuthors">Lim, Key-Hwan; Baek, Kwang-Hyun</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4039-4052</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Abnormal regulation of the ubiquitin-proteasome system (UPS) has been known to be involved in the pathogenesis of a variety of human diseases.  A no. of studies have focused on the identification of small modifiers for the UPS.  Even though the proteasome inhibitor Bortezomib (Velcade) has been approved for the therapy of multiple myeloma and mantle cell lymphoma, there are still no DUB inhibitors endorsed for clin. usage.  Since deubiquitinating enzymes (DUBs) are becoming as a new class of modifiers in the UPS, potential drugs that target specific DUBs have been investigated with the development of exptl. technologies for screening small inhibitor mols.  However, the mol. mechanisms of these mols. are poorly understood.  In order to design and develop specific small inhibitor mols. for specific DUBs, identification of specific substrates and mol. structures for each DUB is required.  Here, we review structures, substrates, and small inhibitor mols. of DUBs identified up to date, providing a clear rationale for the development of novel small inhibitor mols. of DUBs for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo48q6oLy2uCrVg90H21EOLACvtfcHk0ljZ-Dsi8K3p0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpslyrtro%253D&md5=966b3bb7ccb80078050edd455aa825c2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F1381612811319220013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319220013%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DK.%2BH.%26aulast%3DBaek%26aufirst%3DK.%2BH.%26atitle%3DDeubiquitinating%2520enzymes%2520as%2520therapeutic%2520targets%2520in%2520cancer%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D4039%26epage%3D4052%26doi%3D10.2174%2F1381612811319220013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Becker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchenko, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palacios, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moll, U. M.</span><span> </span><span class="NLM_article-title">A role of HAUSP in tumor suppression in a human colon carcinoma xenograft model</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1205</span><span class="NLM_x">–</span> <span class="NLM_lpage">1213</span><span class="refDoi"> DOI: 10.4161/cc.7.9.5756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.4161%2Fcc.7.9.5756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=18418047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosVKrsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=1205-1213&author=K.+Beckerauthor=N.+D.+Marchenkoauthor=G.+Palaciosauthor=U.+M.+Moll&title=A+role+of+HAUSP+in+tumor+suppression+in+a+human+colon+carcinoma+xenograft+model&doi=10.4161%2Fcc.7.9.5756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A role of HAUSP in tumor suppression in a human colon carcinoma xenograft model</span></div><div class="casAuthors">Becker, Kerstin; Marchenko, Natalia D.; Palacios, Gustavo; Moll, Ute M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1205-1213</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">The protease HAUSP is a crit. component of the p53-Mdm2 pathway and acts as a specific deubiquitinase for both p53 and Mdm2 and thus is important for p53 regulation.  In knock-down and knock-out cellular systems it was obsd. that ablation of HAUSP induces profound stabilization of p53 due to enhanced degrdn. of Mdm2.  Thus, inhibiting HAUSP by small compd. interference has been proposed as a rational therapeutic strategy to activate p53 in p53 wild type tumors.  However, HAUSP-mediated effects in the p53-Mdm2 axis are highly complex and non-linear and to date the role of HAUSP in tumor suppression in vivo remains unexplored.  Here we investigate the effect of HAUSP up and downregulation on cell proliferation, apoptosis and tumor growth in vitro and in a xenograft model in vivo, using an inducible isogenic human colon carcinoma cell system.  Importantly, in the absence of stress, both HAUSP up and downregulation inhibit cell proliferation in vitro and tumor growth in vivo due to constitutively elevated p53 levels.  Moreover, tumors with HAUSP up and downregulation respond to radiotherapy with further growth inhibition.  However, HAUSP downregulation causes resistance to Camptothecin- and irradn.-induced apoptosis, which correlates with suppressed mitochondrial translocation of p53.  Our data suggest that changes in HAUSP modulate tumor growth and apoptotic sensitivity in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYvhNCVZmgYLVg90H21EOLACvtfcHk0ljV1KEVasjT0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosVKrsL0%253D&md5=1e3c790603e2f80fb73d44710450d0ec</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.4161%2Fcc.7.9.5756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.7.9.5756%26sid%3Dliteratum%253Aachs%26aulast%3DBecker%26aufirst%3DK.%26aulast%3DMarchenko%26aufirst%3DN.%2BD.%26aulast%3DPalacios%26aufirst%3DG.%26aulast%3DMoll%26aufirst%3DU.%2BM.%26atitle%3DA%2520role%2520of%2520HAUSP%2520in%2520tumor%2520suppression%2520in%2520a%2520human%2520colon%2520carcinoma%2520xenograft%2520model%26jtitle%3DCell%2520Cycle%26date%3D2008%26volume%3D7%26spage%3D1205%26epage%3D1213%26doi%3D10.4161%2Fcc.7.9.5756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Levine, A. J.</span><span> </span><span class="NLM_article-title">p53, the cellular gatekeeper for growth and division</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">323</span><span class="NLM_x">–</span> <span class="NLM_lpage">331</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)81871-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2FS0092-8674%2800%2981871-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=9039259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK2sXhtFahtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1997&pages=323-331&author=A.+J.+Levine&title=p53%2C+the+cellular+gatekeeper+for+growth+and+division&doi=10.1016%2FS0092-8674%2800%2981871-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">p53, The cellular gatekeeper for growth and division</span></div><div class="casAuthors">Levine, Arnold J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">323-331</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review, with 55 refs., of p53 transcription factor protein structure, function, and roles in cell cycle regulation, apoptosis, and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou4CgTSWvZVbVg90H21EOLACvtfcHk0ljV1KEVasjT0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhtFahtLg%253D&md5=bfbec429d10b9718fe9e9ea5dce9109b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981871-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981871-1%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DA.%2BJ.%26atitle%3Dp53%252C%2520the%2520cellular%2520gatekeeper%2520for%2520growth%2520and%2520division%26jtitle%3DCell%26date%3D1997%26volume%3D88%26spage%3D323%26epage%3D331%26doi%3D10.1016%2FS0092-8674%2800%2981871-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Hollstein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sidransky, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogelstein, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, C. C.</span><span> </span><span class="NLM_article-title">p53 mutations in human cancers</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">49</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span><span class="refDoi"> DOI: 10.1126/science.1905840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1126%2Fscience.1905840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1905840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK3MXks1Gmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1991&pages=49-53&author=M.+Hollsteinauthor=D.+Sidranskyauthor=B.+Vogelsteinauthor=C.+C.+Harris&title=p53+mutations+in+human+cancers&doi=10.1126%2Fscience.1905840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">p53 Mutations in human cancers</span></div><div class="casAuthors">Hollstein, Monica; Sidransky, David; Vogelstein, Bert; Harris, Curtis C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">5015</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 95 refs.  Mutations in the evolutionarily conserved codons of the p53 tumor suppressor gene are common in diverse types of human cancer.  The p53 mutational spectrum differs among cancers of the colon, lung, esophagus, breast, liver, brain, reticuloendothelial tissues, and hemopoietic tissues.  Anal. of these mutations can provide clues to the etiol. of these diverse tumors and to the function of specific regions of p53.  Transitions predominate in colon, brain, and lymphoid malignancies, whereas G:C to T:A transversions are the most frequent substitutions obsd. in cancers of the lung and liver.  Mutations at A:T base pairs are seen more frequently in esophageal carcinomas than in other solid tumors.  Most transitions in colorectal carcinomas, brain tumors, leukemias, and lymphomas are at CpG dinucleotide mutational hot spots.  G to T transversions in lung, breast, and esophageal carcinomas are dispersed among numerous codons.  In liver tumors in persons from geog. areas in which both aflatoxin B1 and hepatitis B virus are cancer risk factors, most mutations are at one nucleotide pair of codon 249.  These differences may reflect the etiol. contributions of both exogenous and endogenous factors to human carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXuEpuioXy-LVg90H21EOLACvtfcHk0ljV1KEVasjT0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXks1Gmurg%253D&md5=4ff01dc67a6511e7c637d9824174f518</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.1905840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1905840%26sid%3Dliteratum%253Aachs%26aulast%3DHollstein%26aufirst%3DM.%26aulast%3DSidransky%26aufirst%3DD.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DHarris%26aufirst%3DC.%2BC.%26atitle%3Dp53%2520mutations%2520in%2520human%2520cancers%26jtitle%3DScience%26date%3D1991%26volume%3D253%26spage%3D49%26epage%3D53%26doi%3D10.1126%2Fscience.1905840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Brown, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fersht, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, D. P.</span><span> </span><span class="NLM_article-title">Awakening guardian angels: drugging the p53 pathway</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">862</span><span class="NLM_x">–</span> <span class="NLM_lpage">873</span><span class="refDoi"> DOI: 10.1038/nrc2763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1038%2Fnrc2763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=19935675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=862-873&author=C.+J.+Brownauthor=S.+Lainauthor=C.+S.+Vermaauthor=A.+R.+Fershtauthor=D.+P.+Lane&title=Awakening+guardian+angels%3A+drugging+the+p53+pathway&doi=10.1038%2Fnrc2763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Awakening guardian angels: Drugging the p53 pathway</span></div><div class="casAuthors">Brown, Christopher J.; Lain, Sonia; Verma, Chandra S.; Fersht, Alan R.; Lane, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">862-873</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Currently, around 11 million people are living with a tumor that contains an inactivating mutation of TP53 (the human gene that encodes p53) and another 11 million have tumors in which the p53 pathway is partially abrogated through the inactivation of other signaling or effector components.  The p53 pathway is therefore a prime target for new cancer drug development, and several original approaches to drug discovery that could have wide applications to drug development are being used.  In one approach, mols. that activate p53 by blocking protein-protein interactions with MDM2 are in early clin. development.  Remarkable progress has also been made in the development of p53-binding mols. that can rescue the function of certain p53 mutants.  Finally, cell-based assays are being used to discover compds. that exploit the p53 pathway by either seeking targets and compds. that show synthetic lethality with TP53 mutations or by looking for non-genotoxic activators of the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZW-elIe4w6bVg90H21EOLACvtfcHk0ljV1KEVasjT0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM&md5=072f48d2ed466b2deb9d2b999cc65a53</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc2763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2763%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DC.%2BJ.%26aulast%3DLain%26aufirst%3DS.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26aulast%3DLane%26aufirst%3DD.%2BP.%26atitle%3DAwakening%2520guardian%2520angels%253A%2520drugging%2520the%2520p53%2520pathway%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D862%26epage%3D873%26doi%3D10.1038%2Fnrc2763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Tovar, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosinski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filipovic, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolinsky, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilton, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qing, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myklebost, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heimbrook, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassilev, L. T.</span><span> </span><span class="NLM_article-title">Small-molecule MDM2 antagonists reveal aberrant p53 signaling in cancer: implications for therapy</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">1888</span><span class="NLM_x">–</span> <span class="NLM_lpage">1893</span><span class="refDoi"> DOI: 10.1073/pnas.0507493103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1073%2Fpnas.0507493103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=1888-1893&author=C.+Tovarauthor=J.+Rosinskiauthor=Z.+Filipovicauthor=B.+Higginsauthor=K.+Kolinskyauthor=H.+Hiltonauthor=X.+Zhaoauthor=B.+T.+Vuauthor=W.+Qingauthor=K.+Packmanauthor=O.+Myklebostauthor=D.+C.+Heimbrookauthor=L.+T.+Vassilev&title=Small-molecule+MDM2+antagonists+reveal+aberrant+p53+signaling+in+cancer%3A+implications+for+therapy&doi=10.1073%2Fpnas.0507493103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0507493103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0507493103%26sid%3Dliteratum%253Aachs%26aulast%3DTovar%26aufirst%3DC.%26aulast%3DRosinski%26aufirst%3DJ.%26aulast%3DFilipovic%26aufirst%3DZ.%26aulast%3DHiggins%26aufirst%3DB.%26aulast%3DKolinsky%26aufirst%3DK.%26aulast%3DHilton%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DVu%26aufirst%3DB.%2BT.%26aulast%3DQing%26aufirst%3DW.%26aulast%3DPackman%26aufirst%3DK.%26aulast%3DMyklebost%26aufirst%3DO.%26aulast%3DHeimbrook%26aufirst%3DD.%2BC.%26aulast%3DVassilev%26aufirst%3DL.%2BT.%26atitle%3DSmall-molecule%2520MDM2%2520antagonists%2520reveal%2520aberrant%2520p53%2520signaling%2520in%2520cancer%253A%2520implications%2520for%2520therapy%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D1888%26epage%3D1893%26doi%3D10.1073%2Fpnas.0507493103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Ammirati, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagley, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhattacharya, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckbinder, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlo, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conrad, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cortes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dow, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Kattan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guimaraes, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaPerle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Londregan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loria, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathiowetz, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orr, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersen, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skoura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span> </span><span class="NLM_article-title">Discovery of an in vivo tool to establish proof-of-concept for MAP4K4-based antidiabetic treatment</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1128</span><span class="NLM_x">–</span> <span class="NLM_lpage">1133</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1128-1133&author=M.+Ammiratiauthor=S.+W.+Bagleyauthor=S.+K.+Bhattacharyaauthor=L.+Buckbinderauthor=A.+A.+Carloauthor=R.+Conradauthor=C.+Cortesauthor=R.+L.+Dowauthor=M.+S.+Dowlingauthor=A.+El-Kattanauthor=K.+Fordauthor=C.+R.+Guimaraesauthor=D.+Hepworthauthor=W.+Jiaoauthor=J.+LaPerleauthor=S.+Liuauthor=A.+Londreganauthor=P.+M.+Loriaauthor=A.+M.+Mathiowetzauthor=M.+Munchhofauthor=S.+T.+Orrauthor=D.+N.+Petersenauthor=D.+A.+Priceauthor=A.+Skouraauthor=A.+C.+Smithauthor=J.+Wang&title=Discovery+of+an+in+vivo+tool+to+establish+proof-of-concept+for+MAP4K4-based+antidiabetic+treatment&doi=10.1021%2Facsmedchemlett.5b00215"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00215%26sid%3Dliteratum%253Aachs%26aulast%3DAmmirati%26aufirst%3DM.%26aulast%3DBagley%26aufirst%3DS.%2BW.%26aulast%3DBhattacharya%26aufirst%3DS.%2BK.%26aulast%3DBuckbinder%26aufirst%3DL.%26aulast%3DCarlo%26aufirst%3DA.%2BA.%26aulast%3DConrad%26aufirst%3DR.%26aulast%3DCortes%26aufirst%3DC.%26aulast%3DDow%26aufirst%3DR.%2BL.%26aulast%3DDowling%26aufirst%3DM.%2BS.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DFord%26aufirst%3DK.%26aulast%3DGuimaraes%26aufirst%3DC.%2BR.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DJiao%26aufirst%3DW.%26aulast%3DLaPerle%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLondregan%26aufirst%3DA.%26aulast%3DLoria%26aufirst%3DP.%2BM.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DMunchhof%26aufirst%3DM.%26aulast%3DOrr%26aufirst%3DS.%2BT.%26aulast%3DPetersen%26aufirst%3DD.%2BN.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DSkoura%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DA.%2BC.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520vivo%2520tool%2520to%2520establish%2520proof-of-concept%2520for%2520MAP4K4-based%2520antidiabetic%2520treatment%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D1128%26epage%3D1133%26doi%3D10.1021%2Facsmedchemlett.5b00215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Ohki, H.; Ota, M.; Shibata, Y.; Watanabe, H.; Motoki, R.; Tominaga, Y.; Jimbo, T.</span><span> </span><span class="NLM_article-title">Preparation of bicyclic pyrimidine-carboxamide compounds as inhibitors of receptor tyrosine kinase Axl</span>. WO 2013162061 A1,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=H.+Ohki&author=M.+Ota&author=Y.+Shibata&author=H.+Watanabe&author=R.+Motoki&author=Y.+Tominaga&author=T.+Jimbo&title=Preparation+of+bicyclic+pyrimidine-carboxamide+compounds+as+inhibitors+of+receptor+tyrosine+kinase+Axl"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOhki%26aufirst%3DH.%26atitle%3DPreparation%2520of%2520bicyclic%2520pyrimidine-carboxamide%2520compounds%2520as%2520inhibitors%2520of%2520receptor%2520tyrosine%2520kinase%2520Axl%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Lau, W. C.</span><span> </span><span class="NLM_article-title">Novel Fyn kinase inhibitors for therapeutic applications</span>. WO 2017044623 A1,<span class="NLM_x"> </span><span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=W.+C.+Lau&title=Novel+Fyn+kinase+inhibitors+for+therapeutic+applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DW.%2BC.%26atitle%3DNovel%2520Fyn%2520kinase%2520inhibitors%2520for%2520therapeutic%2520applications%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Ndubaku, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span> </span><span class="NLM_article-title">Inhibiting the deubiquitinating enzymes (DUBs)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1581</span><span class="NLM_x">–</span> <span class="NLM_lpage">1595</span><span class="refDoi"> DOI: 10.1021/jm501061a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501061a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGjtLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1581-1595&author=C.+Ndubakuauthor=V.+Tsui&title=Inhibiting+the+deubiquitinating+enzymes+%28DUBs%29&doi=10.1021%2Fjm501061a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting the Deubiquitinating Enzymes (DUBs)</span></div><div class="casAuthors">Ndubaku, Chudi; Tsui, Vickie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1581-1595</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The diverse roles of deubiquitinating enzymes, or DUBs, in detg. the fate of specific proteins continue to unfold.  Concurrent with the revelation of DUBs as potential therapeutic targets are publications of small mol. inhibitors of these enzymes.  In this review, we summarize these mols. and their assocd. data and suggest addnl. expts. to further validate and characterize these compds.  We believe the field of drug discovery against DUBs is still in its infancy, but advances in assay development, biophys. techniques, and screening libraries hold promise for identifying suitable agents that could advance into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsiPtPfBsxNrVg90H21EOLACvtfcHk0liisCWV5qUBbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGjtLrL&md5=8ec54b033ba6fd8acb3187e4fc004296</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm501061a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501061a%26sid%3Dliteratum%253Aachs%26aulast%3DNdubaku%26aufirst%3DC.%26aulast%3DTsui%26aufirst%3DV.%26atitle%3DInhibiting%2520the%2520deubiquitinating%2520enzymes%2520%2528DUBs%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1581%26epage%3D1595%26doi%3D10.1021%2Fjm501061a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Feng, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergeron, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradley, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koehler, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortwine, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsui, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gobbi, A.</span><span> </span><span class="NLM_article-title">An integrated suite of modeling tools that empower scientists in structure- and property-based drug design</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">511</span><span class="NLM_x">–</span> <span class="NLM_lpage">523</span><span class="refDoi"> DOI: 10.1007/s10822-015-9845-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1007%2Fs10822-015-9845-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=511-523&author=J.+A.+Fengauthor=I.+Aliagasauthor=P.+Bergeronauthor=J.+M.+Blaneyauthor=E.+K.+Bradleyauthor=M.+F.+Koehlerauthor=M.+L.+Leeauthor=D.+F.+Ortwineauthor=V.+Tsuiauthor=J.+Wuauthor=A.+Gobbi&title=An+integrated+suite+of+modeling+tools+that+empower+scientists+in+structure-+and+property-based+drug+design&doi=10.1007%2Fs10822-015-9845-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9845-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9845-4%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DJ.%2BA.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DBergeron%26aufirst%3DP.%26aulast%3DBlaney%26aufirst%3DJ.%2BM.%26aulast%3DBradley%26aufirst%3DE.%2BK.%26aulast%3DKoehler%26aufirst%3DM.%2BF.%26aulast%3DLee%26aufirst%3DM.%2BL.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGobbi%26aufirst%3DA.%26atitle%3DAn%2520integrated%2520suite%2520of%2520modeling%2520tools%2520that%2520empower%2520scientists%2520in%2520structure-%2520and%2520property-based%2520drug%2520design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D511%26epage%3D523%26doi%3D10.1007%2Fs10822-015-9845-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Rougé, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bainbridge, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwok, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Lello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wertz, I. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J.</span><span> </span><span class="NLM_article-title">Molecular understanding of USP7 substrate recognition and C-terminal activation</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1335</span><span class="NLM_x">–</span> <span class="NLM_lpage">1345</span><span class="refDoi"> DOI: 10.1016/j.str.2016.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1016%2Fj.str.2016.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=27452404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1eju7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1335-1345&author=L.+Roug%C3%A9author=T.+W.+Bainbridgeauthor=M.+Kwokauthor=R.+Tongauthor=P.+Di+Lelloauthor=I.+E.+Wertzauthor=T.+Maurerauthor=J.+A.+Ernstauthor=J.+Murray&title=Molecular+understanding+of+USP7+substrate+recognition+and+C-terminal+activation&doi=10.1016%2Fj.str.2016.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Understanding of USP7 Substrate Recognition and C-Terminal Activation</span></div><div class="casAuthors">Rouge, Lionel; Bainbridge, Travis W.; Kwok, Michael; Tong, Raymond; Di Lello, Paola; Wertz, Ingrid E.; Maurer, Till; Ernst, James A.; Murray, Jeremy</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1335-1345</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The deubiquitinating enzyme USP7 has a pivotal role in regulating the stability of proteins involved in fundamental cellular processes of normal biol. and disease.  Despite the importance of USP7, the mechanisms underlying substrate recognition and catalytic activation are poorly understood.  Here we present structural, biochem., and biophys. analyses elucidating the mol. mechanism by which the C-terminal 19 amino acids of USP7 (residues 1084-1102) enhance the ubiquitin cleavage activity of the deubiquitinase (DUB) domain.  Our data demonstrate that the C-terminal peptide binds the activation cleft in the catalytic domain and stabilizes the catalytically competent conformation of USP7.  Addnl. structures of longer fragments of USP7, as well as soln. studies, provide insight into full-length USP7, the role of the UBL domains, and demonstrate that both substrate recognition and deubiquitinase activity are highly regulated by the catalytic and noncatalytic domains of USP7, a feature that could be essential for the proper function of multi-domain DUBs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1bahSKk_MO7Vg90H21EOLACvtfcHk0ljVejeYow6WJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1eju7fL&md5=8cc4de7dc38c480a1c0b1cc7976e7beb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DRoug%25C3%25A9%26aufirst%3DL.%26aulast%3DBainbridge%26aufirst%3DT.%2BW.%26aulast%3DKwok%26aufirst%3DM.%26aulast%3DTong%26aufirst%3DR.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DErnst%26aufirst%3DJ.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26atitle%3DMolecular%2520understanding%2520of%2520USP7%2520substrate%2520recognition%2520and%2520C-terminal%2520activation%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D1335%26epage%3D1345%26doi%3D10.1016%2Fj.str.2016.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Di Lello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougé, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, T.</span><span> </span><span class="NLM_article-title">(1)H, (13)C and (15)N backbone resonance assignment for the 40.5 kDa catalytic domain of Ubiquitin Specific Protease 7 (USP7)</span> <span class="citation_source-journal">Biomol. NMR Assignments</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">345</span><span class="NLM_x">–</span> <span class="NLM_lpage">349</span><span class="refDoi"> DOI: 10.1007/s12104-016-9698-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1007%2Fs12104-016-9698-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=27386854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFertLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2016&pages=345-349&author=P.+Di+Lelloauthor=L.+Roug%C3%A9author=B.+Panauthor=T.+Maurer&title=%281%29H%2C+%2813%29C+and+%2815%29N+backbone+resonance+assignment+for+the+40.5+kDa+catalytic+domain+of+Ubiquitin+Specific+Protease+7+%28USP7%29&doi=10.1007%2Fs12104-016-9698-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">1H, 13C and 15N backbone resonance assignment for the 40.5 kDa catalytic domain of Ubiquitin Specific Protease 7 (USP7)</span></div><div class="casAuthors">Di Lello, Paola; Rouge, Lionel; Pan, Borlan; Maurer, Till</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecular NMR Assignments</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">345-349</span>CODEN:
                <span class="NLM_cas:coden">BNAIBR</span>;
        ISSN:<span class="NLM_cas:issn">1874-270X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The deubiquitinase Ubiquitin Specific Protease 7 (USP7) is part of the regulatory cascade of proteins that modulates the activity of the tumor suppressor protein p53.  Deubiquitination of its target Murine Double Minute 2 (MDM2) leads to increased proteosomal degrdn. of p53.  Consequently, USP7 has emerged as an attractive oncol. target because its inhibition stabilizes p53, thereby promoting p53-dependent apoptosis in cancer cells.  Here we report the backbone resonance assignment for the 40.5 kDa catalytic domain of USP7.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY63PhaaTcnLVg90H21EOLACvtfcHk0ljVejeYow6WJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFertLvN&md5=158aee4ee62aa8c48d3b979c2ff0eb04</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs12104-016-9698-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12104-016-9698-3%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DRoug%25C3%25A9%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DMaurer%26aufirst%3DT.%26atitle%3D%25281%2529H%252C%2520%252813%2529C%2520and%2520%252815%2529N%2520backbone%2520resonance%2520assignment%2520for%2520the%252040.5%2520kDa%2520catalytic%2520domain%2520of%2520Ubiquitin%2520Specific%2520Protease%25207%2520%2528USP7%2529%26jtitle%3DBiomol.%2520NMR%2520Assignments%26date%3D2016%26volume%3D10%26spage%3D345%26epage%3D349%26doi%3D10.1007%2Fs12104-016-9698-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Mayer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, B.</span><span> </span><span class="NLM_article-title">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1784</span><span class="NLM_x">–</span> <span class="NLM_lpage">1788</span><span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0lgHmeOkMMb6tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Pervushin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riek, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wuthrich, K.</span><span> </span><span class="NLM_article-title">Attenuated T2 relaxation by mutual cancellation of dipole–dipole coupling and chemical shift anisotropy indicates an avenue to NMR structures of very large biological macromolecules in solution</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">12366</span><span class="NLM_x">–</span> <span class="NLM_lpage">12371</span><span class="refDoi"> DOI: 10.1073/pnas.94.23.12366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1073%2Fpnas.94.23.12366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=9356455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A280%3ADyaK1c%252FisVWgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1997&pages=12366-12371&author=K.+Pervushinauthor=R.+Riekauthor=G.+Widerauthor=K.+Wuthrich&title=Attenuated+T2+relaxation+by+mutual+cancellation+of+dipole%E2%80%93dipole+coupling+and+chemical+shift+anisotropy+indicates+an+avenue+to+NMR+structures+of+very+large+biological+macromolecules+in+solution&doi=10.1073%2Fpnas.94.23.12366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Attenuated T2 relaxation by mutual cancellation of dipole-dipole coupling and chemical shift anisotropy indicates an avenue to NMR structures of very large biological macromolecules in solution</span></div><div class="casAuthors">Pervushin K; Riek R; Wider G; Wuthrich K</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">12366-71</span>
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Fast transverse relaxation of 1H, 15N, and 13C by dipole-dipole coupling (DD) and chemical shift anisotropy (CSA) modulated by rotational molecular motions has a dominant impact on the size limit for biomacromolecular structures that can be studied by NMR spectroscopy in solution.  Transverse relaxation-optimized spectroscopy (TROSY) is an approach for suppression of transverse relaxation in multidimensional NMR experiments, which is based on constructive use of interference between DD coupling and CSA.  For example, a TROSY-type two-dimensional 1H,15N-correlation experiment with a uniformly 15N-labeled protein in a DNA complex of molecular mass 17 kDa at a 1H frequency of 750 MHz showed that 15N relaxation during 15N chemical shift evolution and 1HN relaxation during signal acquisition both are significantly reduced by mutual compensation of the DD and CSA interactions.  The reduction of the linewidths when compared with a conventional two-dimensional 1H,15N-correlation experiment was 60% and 40%, respectively, and the residual linewidths were 5 Hz for 15N and 15 Hz for 1HN at 4 degrees C.  Because the ratio of the DD and CSA relaxation rates is nearly independent of the molecular size, a similar percentagewise reduction of the overall transverse relaxation rates is expected for larger proteins.  For a 15N-labeled protein of 150 kDa at 750 MHz and 20 degrees C one predicts residual linewidths of 10 Hz for 15N and 45 Hz for 1HN, and for the corresponding uniformly 15N,2H-labeled protein the residual linewidths are predicted to be smaller than 5 Hz and 15 Hz, respectively.  The TROSY principle should benefit a variety of multidimensional solution NMR experiments, especially with future use of yet somewhat higher polarizing magnetic fields than are presently available, and thus largely eliminate one of the key factors that limit work with larger molecules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSt8cd29R1zO-3MPKNEWB5RfW6udTcc2eZGLUe5WRKFurntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1c%252FisVWgtQ%253D%253D&md5=ab6a5c6af04d914882d7977e67bf022b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.94.23.12366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.94.23.12366%26sid%3Dliteratum%253Aachs%26aulast%3DPervushin%26aufirst%3DK.%26aulast%3DRiek%26aufirst%3DR.%26aulast%3DWider%26aufirst%3DG.%26aulast%3DWuthrich%26aufirst%3DK.%26atitle%3DAttenuated%2520T2%2520relaxation%2520by%2520mutual%2520cancellation%2520of%2520dipole%25E2%2580%2593dipole%2520coupling%2520and%2520chemical%2520shift%2520anisotropy%2520indicates%2520an%2520avenue%2520to%2520NMR%2520structures%2520of%2520very%2520large%2520biological%2520macromolecules%2520in%2520solution%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1997%26volume%3D94%26spage%3D12366%26epage%3D12371%26doi%3D10.1073%2Fpnas.94.23.12366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Delaglio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grzesiek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuister, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeifer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bax, A.</span><span> </span><span class="NLM_article-title">NMRPipe: A multidimensional spectral processing system based on UNIX pipes</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">293</span><span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+A+multidimensional+spectral+processing+system+based+on+UNIX+pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0lgHmeOkMMb6tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520A%2520multidimensional%2520spectral%2520processing%2520system%2520based%2520on%2520UNIX%2520pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Johnson, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blevins, R. A.</span><span> </span><span class="NLM_article-title">NMRView: A computer program for the visualization and analysis of NMR data</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">603</span><span class="NLM_x">–</span> <span class="NLM_lpage">614</span><span class="refDoi"> DOI: 10.1007/BF00404272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=10.1007%2FBF00404272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=22911360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;key=1%3ACAS%3A528%3ADyaK2cXmt1Gkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1994&pages=603-614&author=B.+A.+Johnsonauthor=R.+A.+Blevins&title=NMRView%3A+A+computer+program+for+the+visualization+and+analysis+of+NMR+data&doi=10.1007%2FBF00404272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">NMRView: a computer program for the visualization and analysis of NMR data</span></div><div class="casAuthors">Johnson, Bruce A.; Blevins, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">603-14</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    </div><div class="casAbstract">NMRView is a computer program designed for the visualization and anal. of NMR data.  It allows the user to interact with a practically unlimited no. of 2D, 3D and 4D NMR data files.  Any no. of spectral windows can be displayed on the screen in any size and location.  Automatic peak picking and facilitated peak anal. features are included to aid in the assignment of complex NMR spectra.  NMR View provides structure anal. features and data transfer to and from structural generation programs, allowing for a tight coupling between the spectral anal. and structural generation.,.  Visual correlation between structures and spectra can be done with the Mol. Data Viewer, a mol. graphics program with bidirectional communication to NMR View.  The used interface can be customized and a command language is provided to allow for the automation of various tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr12Ta6zHzfUrVg90H21EOLACvtfcHk0lgHmeOkMMb6tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmt1Gkurw%253D&md5=6a93a0d9704fbbb654abc192203c68ae</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2FBF00404272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00404272%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DB.%2BA.%26aulast%3DBlevins%26aufirst%3DR.%2BA.%26atitle%3DNMRView%253A%2520A%2520computer%2520program%2520for%2520the%2520visualization%2520and%2520analysis%2520of%2520NMR%2520data%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1994%26volume%3D4%26spage%3D603%26epage%3D614%26doi%3D10.1007%2FBF00404272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M5W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M5W','PDB','4M5W'); return false;">PDB: 4M5W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHC','PDB','5WHC'); return false;">PDB: 5WHC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WH7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WH7','PDB','5WH7'); return false;">PDB: 5WH7</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01293">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_73230"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01293">10.1021/acs.jmedchem.7b01293</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Table S1 of compounds lacking phenol in C3/C5 position, Table S2 of crystallography data and refinement statistics, and Figure S1 of computational torsion scan results (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_001.pdf">PDF</a>)</p></li><li><p class="inline"><sup>1</sup>H and <sup>13</sup>C 1D NMR spectra of all synthesized compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biochemical data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_001.pdf">jm7b01293_si_001.pdf (338.19 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_002.pdf">jm7b01293_si_002.pdf (1.69 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01293/suppl_file/jm7b01293_si_003.csv">jm7b01293_si_003.csv (1.88 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are the following: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WHC">5WHC</a> (crystal structure of <b>2</b> in complex with USP7-CD); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WH7">5WH7</a> (crystal structure of <b>16</b> with USP7-CD). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01293&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01293%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-24%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01293" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679933420a1824cb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
